

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/24/6/635/3979487","@type":"ScholarlyArticle","name":"Comparative whole-genome analysis reveals artificial selection effects on  Ustilago esculenta  genome","datePublished":"2017-07-19","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/24/6","@type":"PublicationIssue","issueNumber":"6","datePublished":"2017-12-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsx031","keywords":["<italic>Ustilago esculenta</italic>","whole-genome analysis","fungi–host interaction","<italic>Zizania latifolia</italic>"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/24/6/635/3979487","author":[{"name":"Ye, Zihong","affiliation":"Department of Zhejiang Provincial Key Laboratory of Biometrology and Inspection &amp; Quarantine, China Jiliang University, Hangzhou, China","@type":"Person"},{"name":"Pan, Yao","affiliation":"Department of Algorithmic Bioinformatics, Heinrich Heine University, Düsseldorf, Germany  Cluster of Excellence on Plant Sciences (CEPLAS), Düsseldorf, Germany","@type":"Person"},{"name":"Zhang, Yafen","affiliation":"Department of Zhejiang Provincial Key Laboratory of Biometrology and Inspection &amp; Quarantine, China Jiliang University, Hangzhou, China","@type":"Person"},{"name":"Cui, Haifeng","affiliation":"Department of Zhejiang Provincial Key Laboratory of Biometrology and Inspection &amp; Quarantine, China Jiliang University, Hangzhou, China","@type":"Person"},{"name":"Jin, Gulei","affiliation":"Department of Agronomy &amp; Zhejiang Key Laboratory of Crop Germplasm Resources, Zhejiang University, Hangzhou, China","@type":"Person"},{"name":"McHardy, Alice C","affiliation":"Department of Algorithmic Bioinformatics, Heinrich Heine University, Düsseldorf, Germany  Cluster of Excellence on Plant Sciences (CEPLAS), Düsseldorf, Germany","@type":"Person"},{"name":"Fan, Longjiang","affiliation":"Department of Agronomy &amp; Zhejiang Key Laboratory of Crop Germplasm Resources, Zhejiang University, Hangzhou, China","@type":"Person"},{"name":"Yu, Xiaoping","affiliation":"Department of Zhejiang Provincial Key Laboratory of Biometrology and Inspection &amp; Quarantine, China Jiliang University, Hangzhou, China","@type":"Person"}],"description":"Abstract.  Ustilago esculenta, infects Zizania latifolia, and induced host stem swollen to be a popular vegetable called Jiaobai in China. It is the long-standi","pageStart":"635","pageEnd":"648","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/6/2/m_dnares_24_6cover.png?Expires=2147483647&Signature=z4yCzT-6enr7Fof9QqkUJvtPQAXDRV5XYB7R7L2HBbQZ2iaySlHxBG~hDePAa-rY6Gn-PH9W8mIlJwAyG9R2RE6Vd3RTDct~I7biyNGgXk9CP54tGn91x8VYIMO24kwuQBqahsZQMv3NxRiQKhbGI0yEDanozvBgG~-9FpaC12JFLE6A4BT6rBzjsJLh9UJqvGv0F2hWW2jLvdITi~5PoATpRIsbIK-S1wcvqp49OSN~oRepZtr6iz~107ho8gZAxoaG8nb4DvXKrvfnCuhhmbRD41NP3K2zFyztHicWtTYb1xunX52RqodNq-M3zVvUit2rSw5-mRSvm-jXb-p6nw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Comparative whole-genome analysis reveals artificial selection effects on  Ustilago esculenta  genome","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/6/2/m_dnares_24_6cover.png?Expires=2147483647&Signature=z4yCzT-6enr7Fof9QqkUJvtPQAXDRV5XYB7R7L2HBbQZ2iaySlHxBG~hDePAa-rY6Gn-PH9W8mIlJwAyG9R2RE6Vd3RTDct~I7biyNGgXk9CP54tGn91x8VYIMO24kwuQBqahsZQMv3NxRiQKhbGI0yEDanozvBgG~-9FpaC12JFLE6A4BT6rBzjsJLh9UJqvGv0F2hWW2jLvdITi~5PoATpRIsbIK-S1wcvqp49OSN~oRepZtr6iz~107ho8gZAxoaG8nb4DvXKrvfnCuhhmbRD41NP3K2zFyztHicWtTYb1xunX52RqodNq-M3zVvUit2rSw5-mRSvm-jXb-p6nw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Comparative whole-genome analysis reveals artificial selection effects on  Ustilago esculenta  genome" />
    <meta property="og:description" content="Abstract.  Ustilago esculenta, infects Zizania latifolia, and induced host stem swollen to be a popular vegetable called Jiaobai in China. It is the long-standi" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/24/6/635/3979487" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2017-07-19" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Ye, Zihong" /><meta name="citation_author_institution" content="Department of Zhejiang Provincial Key Laboratory of Biometrology and Inspection &amp; Quarantine, China Jiliang University, Hangzhou, China" /><meta name="citation_author" content="Pan, Yao" /><meta name="citation_author_institution" content="Department of Algorithmic Bioinformatics, Heinrich Heine University, Düsseldorf, Germany" /><meta name="citation_author_institution" content="Cluster of Excellence on Plant Sciences (CEPLAS), Düsseldorf, Germany" /><meta name="citation_author" content="Zhang, Yafen" /><meta name="citation_author_institution" content="Department of Zhejiang Provincial Key Laboratory of Biometrology and Inspection &amp; Quarantine, China Jiliang University, Hangzhou, China" /><meta name="citation_author" content="Cui, Haifeng" /><meta name="citation_author_institution" content="Department of Zhejiang Provincial Key Laboratory of Biometrology and Inspection &amp; Quarantine, China Jiliang University, Hangzhou, China" /><meta name="citation_author" content="Jin, Gulei" /><meta name="citation_author_institution" content="Department of Agronomy &amp; Zhejiang Key Laboratory of Crop Germplasm Resources, Zhejiang University, Hangzhou, China" /><meta name="citation_author" content="McHardy, Alice C" /><meta name="citation_author_institution" content="Department of Algorithmic Bioinformatics, Heinrich Heine University, Düsseldorf, Germany" /><meta name="citation_author_institution" content="Cluster of Excellence on Plant Sciences (CEPLAS), Düsseldorf, Germany" /><meta name="citation_author" content="Fan, Longjiang" /><meta name="citation_author_institution" content="Department of Agronomy &amp; Zhejiang Key Laboratory of Crop Germplasm Resources, Zhejiang University, Hangzhou, China" /><meta name="citation_author" content="Yu, Xiaoping" /><meta name="citation_author_institution" content="Department of Zhejiang Provincial Key Laboratory of Biometrology and Inspection &amp; Quarantine, China Jiliang University, Hangzhou, China" /><meta name="citation_title" content="Comparative whole-genome analysis reveals artificial selection effects on Ustilago esculenta genome" /><meta name="citation_firstpage" content="635" /><meta name="citation_lastpage" content="648" /><meta name="citation_doi" content="10.1093/dnares/dsx031" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="24" /><meta name="citation_issue" content="6" /><meta name="citation_publication_date" content="2017/12/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Nutritional requirements of the edible gall-producing fungus Ustilago esculenta,;  citation_journal_title=J. Biol. Sci;  citation_year=2004;  citation_volume=4;  citation_pages=246-52; " /><meta name="citation_reference" content="citation_title=Phylogeny and biogeography of the eastern Asian-North American disjunct wild-rice genus (Zizania L., Poaceae),;  citation_journal_title=Mol. Phylogenet. Evol;  citation_year=2010;  citation_volume=55;  citation_pages=1008-17; " /><meta name="citation_reference" content="citation_title=Zizania latifolia Turcz. cultivated in China,;  citation_journal_title=Genet. Resour. Crop Evol;  citation_year=2007;  citation_volume=54;  citation_pages=1211-7; " /><meta name="citation_reference" content="citation_title=Main agronomic traits, domestication and breeding of Gu (Zizania latifolia),;  citation_journal_title=J. Zhejiang Univ;  citation_year=2013;  citation_volume=39;  citation_pages=629-35; " /><meta name="citation_reference" content="citation_title=Formation and histopathology of galls induced by Ustilage esculenta in Zizania latifolia,;  citation_journal_title=Phytopathology;  citation_year=1978;  citation_volume=68;  citation_pages=1572-6; " /><meta name="citation_reference" content="citation_title=Chemical composition, angiotensin-converting enzyme-inhibitory activity and antioxidant activities of few-flower wild rice (Zizania latifolia Turcz.),;  citation_journal_title=J. Sci. Food Agric;  citation_year=2012;  citation_volume=92;  citation_pages=159-64; " /><meta name="citation_reference" content="citation_title=Plant growth and photosynthetic performance of Zizania latifolia are altered by endophytic Ustilago esculenta infection,;  citation_journal_title=Physiol. Mol. Plant Pathol;  citation_year=2013;  citation_volume=83;  citation_pages=75-83; " /><meta name="citation_reference" content="citation_title=Highly efficient and safe production of Jiaobai,;  citation_journal_title=Sci. Agric;  citation_year=2012;  citation_volume=69;  citation_pages=142-6; " /><meta name="citation_reference" content="citation_title=Origins of host-specific populations of the blast pathogen Magnaporthe oryzae in crop domestication with subsequent expansion of pandemic clones on rice and weeds of rice,;  citation_journal_title=Genetics;  citation_year=2005;  citation_volume=170;  citation_pages=613-30; " /><meta name="citation_reference" content="citation_title=Origin and domestication of the fungal wheat pathogen Mycosphaerella graminicola via sympatric speciation,;  citation_journal_title=Mol. Biol. Evol;  citation_year=2007;  citation_volume=24;  citation_pages=398-411; " /><meta name="citation_reference" content="citation_title=A bountiful harvest: genomic insights into crop domestication phenotypes,;  citation_journal_title=Annu. Rev. Plant Biol;  citation_year=2013;  citation_volume=64;  citation_pages=47-70; " /><meta name="citation_reference" content="citation_title=Crop domestication and its impact on naturally selected trophic interactions,;  citation_journal_title=Annu. Rev. Entomol;  citation_year=2015;  citation_volume=60;  citation_pages=35-58; " /><meta name="citation_reference" content="citation_title=The making of a new pathogen: insights from comparative population genomics of the domesticated wheat pathogen Mycosphaerella graminicola and its wild sister species,;  citation_journal_title=Genome Res;  citation_year=2011;  citation_volume=21;  citation_pages=2157-66; " /><meta name="citation_reference" content="citation_title=Comparative analysis of the genomes of two field isolates of the rice blast fungus Magnaporthe oryzae,;  citation_journal_title=PLoS Genet;  citation_year=2012;  citation_volume=8;  citation_pages=e1002869" /><meta name="citation_reference" content="citation_title=Evolution, selection and isolation: a genomic view of speciation in fungal plant pathogens,;  citation_journal_title=New Phytol;  citation_year=2013;  citation_volume=199;  citation_pages=895-907; " /><meta name="citation_reference" content="citation_title=Gene loss rather than gene gain is associated with a host jump from monocots to dicots in the Smut Fungus Melanopsichium pennsylvanicum,;  citation_journal_title=Genome Biol. Evol;  citation_year=2014;  citation_volume=6;  citation_pages=2034-49; " /><meta name="citation_reference" content="Heinze B.  2009 , Comparative analysis of the Maize Smut Fungi Ustilago maydis and Sporisorium reilianum. Philipps-Universität Marburg, Germany. " /><meta name="citation_reference" content="citation_title=A preliminary study on population genetic structure and phylogeography of the wild and cultivated Zizania latifolia (Poaceae) based on Adh1a sequences,;  citation_journal_title=Theor. Appl. Genet;  citation_year=2008;  citation_volume=116;  citation_pages=835-43; " /><meta name="citation_reference" content="citation_title=Sho1 and Msb2-related proteins regulate appressorium development in the smut fungus Ustilago maydis,;  citation_journal_title=Plant Cell;  citation_year=2010;  citation_volume=22;  citation_pages=2085-101; " /><meta name="citation_reference" content="citation_title=Different signalling pathways involving a G alpha protein, cAMP and a MAP kinase control germination of Botrytis cinerea conidia,;  citation_journal_title=Mol. Microbiol;  citation_year=2006;  citation_volume=59;  citation_pages=821-35; " /><meta name="citation_reference" content="citation_title=Reprogramming a maize plant: transcriptional and metabolic changes induced by the fungal biotroph Ustilago maydis,;  citation_journal_title=Plant J;  citation_year=2008;  citation_volume=56;  citation_pages=181-95; " /><meta name="citation_reference" content="citation_title=Plant surface cues prime Ustilago maydis for biotrophic development,;  citation_journal_title=PLoS Pathog;  citation_year=2014;  citation_volume=10;  citation_pages=295" /><meta name="citation_reference" content="citation_title=Characterization of the largest effector gene cluster of Ustilago maydis,;  citation_journal_title=PLoS Pathog;  citation_year=2014;  citation_volume=10;  citation_pages=191-8; " /><meta name="citation_reference" content="citation_title=Maize tumors caused by Ustilago maydis require organ-specific genes in host and pathogen,;  citation_journal_title=Science;  citation_year=2010;  citation_volume=328;  citation_pages=89-92; " /><meta name="citation_reference" content="citation_title=Genome comparison of barley and maize smut fungi reveals targeted loss of RNA silencing components and species-specific presence of transposable elements,;  citation_journal_title=Plant Cell;  citation_year=2012;  citation_volume=24;  citation_pages=1733-45; " /><meta name="citation_reference" content="citation_title=Pathogenicity determinants in smut fungi revealed by genome comparison,;  citation_journal_title=Science;  citation_year=2010;  citation_volume=330;  citation_pages=1546-8; " /><meta name="citation_reference" content="citation_title=Scaffolding pre-assembled contigs using SSPACE,;  citation_journal_title=Bioinformatics;  citation_year=2011;  citation_volume=27;  citation_pages=578-9; " /><meta name="citation_reference" content="citation_title=Toward almost closed genomes with GapFiller,;  citation_journal_title=Genome Biol;  citation_year=2012;  citation_volume=13;  citation_pages=1-9; " /><meta name="citation_reference" content="citation_title=LTR_FINDER: an efficient tool for the prediction of full-length LTR retrotransposons,;  citation_journal_title=Nucleic Acids Res;  citation_year=2007;  citation_volume=35;  citation_pages=265-8; " /><meta name="citation_reference" content="citation_title=Using native and syntenically mapped cDNA alignments to improve de novo gene finding,;  citation_journal_title=Bioinformatics;  citation_year=2008;  citation_volume=24;  citation_pages=637-44; " /><meta name="citation_reference" content="citation_title=Creating a honey bee consensus gene set,;  citation_journal_title=Genome Biol;  citation_year=2007;  citation_volume=8;  citation_pages=R13" /><meta name="citation_reference" content="citation_title=Basic local alignment search tool,;  citation_journal_title=J. Mol. Biol;  citation_year=1990;  citation_volume=215;  citation_pages=403-10; " /><meta name="citation_reference" content="citation_title=InterProScan: protein domains identifier,;  citation_journal_title=Nucleic Acids Res;  citation_year=2005;  citation_volume=33;  citation_pages=116-20; " /><meta name="citation_reference" content="citation_title=OrthoMCL: identification of ortholog groups for eukaryotic genomes,;  citation_journal_title=Genome Res;  citation_year=2003;  citation_volume=13;  citation_pages=2178-89; " /><meta name="citation_reference" content="citation_title=dbCAN: a web resource for automated carbohydrate-active enzyme annotation,;  citation_journal_title=Nucleic Acids Res;  citation_year=2012;  citation_volume=40;  citation_pages=445-51; " /><meta name="citation_reference" content="citation_title=SMURF: genomic mapping of fungal secondary metabolite clusters,;  citation_journal_title=Fungal Genet. Biol;  citation_year=2010;  citation_volume=47;  citation_pages=736-41; " /><meta name="citation_reference" content="citation_title=Identification of prokaryotic and eukaryotic signal peptides prediction of their cleavage sites,;  citation_journal_title=Protein Eng;  citation_year=1997;  citation_volume=10;  citation_pages=1-6; " /><meta name="citation_reference" content="citation_title=Predicting subcellular localization of proteins based on their N-terminal amino acid sequence,;  citation_journal_title=J. Mol. Biol;  citation_year=2000;  citation_volume=300;  citation_pages=1005-16; " /><meta name="citation_reference" content="citation_title=tRNAscan-SE: a program for improved detection of transfer RNA genes in genomic sequence,;  citation_journal_title=Nucleic Acids Res;  citation_year=1997;  citation_volume=25;  citation_pages=955-64; " /><meta name="citation_reference" content="citation_title=Infernal 1.0: inference of RNA alignments,;  citation_journal_title=Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=1335-7; " /><meta name="citation_reference" content="citation_title=Rfam 11.0: 10 years of RNA families,;  citation_journal_title=Nucleic Acids Res;  citation_year=2013;  citation_volume=41;  citation_pages=226-32; " /><meta name="citation_reference" content="citation_title=Versatile and open software for comparing large genomes,;  citation_journal_title=Genome Biol;  citation_year=2004;  citation_volume=5;  citation_pages=R12" /><meta name="citation_reference" content="citation_title=SOAP2: an improved ultrafast tool for short read alignment,;  citation_journal_title=Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=1966-7; " /><meta name="citation_reference" content="citation_title=Mapping and quantifying mammalian transcriptomes by RNA-Seq,;  citation_journal_title=Nat. Methods;  citation_year=2008;  citation_volume=5;  citation_pages=621-8; " /><meta name="citation_reference" content="citation_title=edgeR: a Bioconductor package for differential expression analysis of digital gene expression data,;  citation_journal_title=Bioinformatics;  citation_year=2010;  citation_volume=26;  citation_pages=139-40; " /><meta name="citation_reference" content="citation_title=A host plant geneme (Zizania latifolia) after a century-long endophyte infection,;  citation_journal_title=Plant J;  citation_year=2015;  citation_volume=83;  citation_pages=600-9; " /><meta name="citation_reference" content="citation_title=Fast, scalable generation of high-quality protein multiple sequence alignments using Clustal Omega,;  citation_journal_title=Mol. Syst. Biol;  citation_year=2011;  citation_volume=7;  citation_pages=1429-32; " /><meta name="citation_reference" content="citation_title=The evolution of spliceosomal introns: patterns, puzzles and progress,;  citation_journal_title=Nat. Rev. Genet;  citation_year=2006;  citation_volume=7;  citation_pages=211-21; " /><meta name="citation_reference" content="citation_title=Do TE activity and counteracting genome defenses, RNAi and methylation, shape the sex lives of smut fungi?,;  citation_journal_title=Plant Signal. Behav;  citation_year=2013;  citation_volume=8;  citation_pages=e23853" /><meta name="citation_reference" content="citation_title=Premeiotic instability of repeated sequences in Neurospora crassa,;  citation_journal_title=Annu. Rev. Genet;  citation_year=1990;  citation_volume=24;  citation_pages=579-613; " /><meta name="citation_reference" content="citation_title=Functional and evolutionary analysis of the genome of an obligate fungal symbiont,;  citation_journal_title=Genome Biol. Evol;  citation_year=2013;  citation_volume=5;  citation_pages=891-904; " /><meta name="citation_reference" content="citation_title=Interspecific sex in grass smuts and the genetic diversity of their pheromone-receptor system,;  citation_journal_title=PLoS Genet;  citation_year=2011;  citation_volume=7;  citation_pages=e1002436" /><meta name="citation_reference" content="citation_title=Mating-type locus of Cryptococcus neoformans: a step in the evolution of sex chromosomes,;  citation_journal_title=Eukaryot. Cell;  citation_year=2002;  citation_volume=1;  citation_pages=704-18; " /><meta name="citation_reference" content="citation_title=Shared forces of sex chromosome evolution in haploid-mating and diploid-mating organisms: Microbotryum violaceum and other model organisms,;  citation_journal_title=Genetics;  citation_year=2004;  citation_volume=168;  citation_pages=141-6; " /><meta name="citation_reference" content="citation_title=Evolutionary erosion of yeast sex chromosomes by mating-type switching accidents,;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2011;  citation_volume=108;  citation_pages=20024-9; " /><meta name="citation_reference" content="citation_title=The evolutionary dynamics of transposable elements in eukaryote genomes,;  citation_journal_title=Genome Dyn;  citation_year=2012;  citation_volume=7;  citation_pages=68-91; " /><meta name="citation_reference" content="citation_title=Characterization of retrotransposon sequences expressed in inflorescences of apomictic and sexual Paspalum notatum plants,;  citation_journal_title=Sex. Plant Reprod;  citation_year=2011;  citation_volume=24;  citation_pages=231-46; " /><meta name="citation_reference" content="citation_title=Evolution: transposons drive sex chromosome evolution,;  citation_journal_title=Nat. Rev. Genet;  citation_year=2014;  citation_volume=15;  citation_issue=1; " /><meta name="citation_reference" content="citation_title=The a2 mating-type locus genes lga2 and rga2 direct uniparental mitochondrial DNA (mtDNA) inheritance and constrain mtDNA recombination during sexual development of Ustilago maydis,;  citation_journal_title=Genetics;  citation_year=2009;  citation_volume=181;  citation_pages=847-60; " /><meta name="citation_reference" content="citation_title=The Ustilago maydis a2 mating-type locus genes lga2 and rga2 compromise pathogenicity in the absence of the mitochondrial p32 family protein Mrb1,;  citation_journal_title=Plant Cell;  citation_year=2004;  citation_volume=16;  citation_pages=2233-48; " /><meta name="citation_reference" content="citation_title=Atypical mitochondrial inheritance patterns in eukaryotes,;  citation_journal_title=Genome;  citation_year=2015;  citation_volume=58;  citation_pages=423-31; " /><meta name="citation_reference" content="citation_title=Environment factors can influence mitochondrial inheritance in the fungus Cryptococcus neoformans,;  citation_journal_title=Fungal Genet. Biol;  citation_year=2007;  citation_volume=44;  citation_pages=315-22; " /><meta name="citation_reference" content="citation_title=The transcription factor Rbf1 is the master regulator for b-mating type controlled pathogenic development in Ustilago maydis,;  citation_journal_title=PLoS Pathog;  citation_year=2010;  citation_volume=6;  citation_pages=761-4; " /><meta name="citation_reference" content="citation_title=Ustilago maydis as a pathogen,;  citation_journal_title=Phytopathology;  citation_year=2009;  citation_volume=47;  citation_pages=423-45; " /><meta name="citation_reference" content="citation_title=Regulation of mating and pathogenic development in Ustilago maydis,;  citation_journal_title=Curr. Opin. Microbiol;  citation_year=2004;  citation_volume=7;  citation_pages=666-72; " /><meta name="citation_reference" content="citation_title=Pheromone-regulated target genes respond differentially to MAPK phosphorylation of transcription factor Prf1,;  citation_journal_title=Mol. Microbiol;  citation_year=2008;  citation_volume=69;  citation_pages=1041-53; " /><meta name="citation_reference" content="citation_title=The Ustilago maydis Clp1 protein orchestrates pheromone and b-dependent signaling pathways to coordinate the cell cycle and pathogenic development,;  citation_journal_title=Plant Cell;  citation_year=2010;  citation_volume=22;  citation_pages=2908-22; " /><meta name="citation_reference" content="citation_title=AgSwe1p regulates mitosis in response to morphogenesis and nutrients in multinucleated Ashbya gossypii cells,;  citation_journal_title=Mol. Biol. Cell;  citation_year=2006;  citation_volume=17;  citation_pages=4494-512; " /><meta name="citation_reference" content="citation_title=A genome-based analysis of amino acid metabolism in the biotrophic plant pathogen Ustilago maydis,;  citation_journal_title=Fungal Genet. Biol;  citation_year=2008;  citation_volume=45(Suppl 1);  citation_pages=S77-87; " /><meta name="citation_reference" content="citation_title=Aphid genome expression reveals host-symbiont cooperation in the production of amino acids,;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2011;  citation_volume=108;  citation_pages=2849-54; " /><meta name="citation_reference" content="citation_title=Inoculum size effect in dimorphic fungi: extracellular control of yeast-mycelium dimorphism in Ceratocystis ulmi,;  citation_journal_title=Appl. Environ. Microbiol;  citation_year=2004;  citation_volume=70;  citation_pages=1356-9; " /><meta name="citation_reference" content="citation_title=Candida albicans induces arginine biosynthetic genes in response to host-derived reactive oxygen species,;  citation_journal_title=Eukaryot. Cell;  citation_year=2013;  citation_volume=12;  citation_pages=91-100; " /><meta name="citation_reference" content="citation_title=Compartmental and regulatory mechanisms in the arginine pathways of Neurospora crassa and Saccharomyces cerevisiae,;  citation_journal_title=Microbiol. Rev;  citation_year=1986;  citation_volume=50;  citation_pages=280-313; " /><meta name="citation_reference" content="citation_title=Ornithine-delta-aminotransferase and proline dehydrogenase genes play a role in non-host disease resistance by regulating pyrroline-5-carboxylate metabolism-induced hypersensitive response;  citation_journal_title=Plant Cell Environ;  citation_year=2012;  citation_volume=35;  citation_pages=1329-43; " /><meta name="citation_reference" content="citation_title=The wheat ethylene response factor transcription factor pathogen-induced ERF1 mediates host responses to both the necrotrophic pathogen Rhizoctonia cerealis and freezing stresses,;  citation_journal_title=Plant. Physiol;  citation_year=2014;  citation_volume=164;  citation_pages=1499-514; " /><meta name="citation_reference" content="citation_title=Methionine elicits H2O2 generation and defense gene expression in grapevine and reduces Plasmopara viticola infection,;  citation_journal_title=J. Plant Physiol;  citation_year=2013;  citation_volume=170;  citation_pages=1561-8; " /><meta name="citation_reference" content="citation_title=Homocysteine over-accumulation as the effect of potato leaves exposure to biotic stress,;  citation_journal_title=Plant Physiol. Biochem;  citation_year=2013;  citation_volume=63;  citation_pages=177-84; " /><meta name="citation_reference" content="citation_title=Lysine catabolism, amino acid transport, and systemic acquired resistance: what is the link?,;  citation_journal_title=Plant Signal. Behav;  citation_year=2014;  citation_volume=9;  citation_pages=e28933" /><meta name="citation_reference" content="citation_title=An important role of the pepper phenylalanine ammonia-lyase gene (PAL1) in salicylic acid-dependent signalling of the defence response to microbial pathogens,;  citation_journal_title=J. Exp. Bot;  citation_year=2014;  citation_volume=65;  citation_pages=2295-306; " /><meta name="citation_reference" content="citation_title=Streptomyces-induced resistance against oak powdery mildew involves host plant responses in defense, photosynthesis, and secondary metabolism pathways,;  citation_journal_title=Mol. Plant Microbe Interact;  citation_year=2014;  citation_volume=27;  citation_pages=891-900; " /><meta name="citation_reference" content="citation_title=A KS-type dehydrin and its related domains reduce Cu-promoted radical generation and the histidine residues contribute to the radical-reducing activities,;  citation_journal_title=J. Exp. Bot;  citation_year=2013;  citation_volume=64;  citation_pages=1615-24; " /><meta name="citation_reference" content="citation_title=Optical measurements of invasive forces exerted by appressoria of a plant pathogenic fungus,;  citation_journal_title=Science;  citation_year=1999;  citation_volume=285;  citation_pages=1896-9; " /><meta name="citation_reference" content="citation_title=Penetration of hard substrates by a fungus employing enormous turgor pressures,;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=1991;  citation_volume=88;  citation_pages=11281-4; " /><meta name="citation_reference" content="citation_title=Endoplasmic reticulum glucosidase II is required for pathogenicity of Ustilago maydis,;  citation_journal_title=Plant Cell;  citation_year=2005;  citation_volume=17;  citation_pages=3532-43; " /><meta name="citation_reference" content="citation_title=The Cochliobolus carbonum SNF1 gene is required for cell wall-degrading enzyme expression and virulence on maize,;  citation_journal_title=Plant Cell;  citation_year=2000;  citation_volume=12;  citation_pages=237-47; " /><meta name="citation_reference" content="citation_title=Novel G-protein-coupled receptor-like proteins in the plant pathogenic fungus Magnaporthe grisea,;  citation_journal_title=Genome Biol;  citation_year=2005;  citation_volume=6;  citation_pages=R24" /><meta name="citation_reference" content="citation_title=A maize-specifically expressed gene cluster in Ustilago maydis,;  citation_journal_title=Mol. Microbiol;  citation_year=2002;  citation_volume=43;  citation_pages=75-93; " /><meta name="citation_reference" content="citation_title=Identification of cis-active elements in Ustilago maydis mig2 promoters conferring high-level activity during pathogenic growth in maize,;  citation_journal_title=Mol. Plant Microbe Interact;  citation_year=2005;  citation_volume=18;  citation_pages=75-87; " /><meta name="citation_reference" content="citation_title=Cloning and targeted gene disruption of EXG1, encoding exo-beta 1, 3-glucanase, in the phytopathogenic fungus Cochliobolus carbonum,;  citation_journal_title=Appl. Environ. Microbiol;  citation_year=1994;  citation_volume=60;  citation_pages=594-8; " /><meta name="citation_reference" content="citation_title=Filament-specific expression of a cellulase gene in the dimorphic fungus Ustilago maydis,;  citation_journal_title=Biol. Chem. Hoppe Seyler;  citation_year=1995;  citation_volume=376;  citation_pages=617-25; " /><meta name="citation_reference" content="citation_title=Repellents have functionally replaced hydrophobins in mediating attachment to a hydrophobic surface and in formation of hydrophobic aerial hyphae in Ustilago maydis,;  citation_journal_title=Microbiology;  citation_year=2006;  citation_volume=152;  citation_pages=3607-12; " /><meta name="citation_reference" content="citation_title=A secreted Ustilago maydis effector promotes virulence by targeting anthocyanin biosynthesis in maize,;  citation_journal_title=Elife;  citation_year=2014;  citation_volume=3;  citation_pages=125-31; " /><meta name="citation_reference" content="citation_title=Insights from the genome of the biotrophic fungal plant pathogen Ustilago maydis,;  citation_journal_title=Nature;  citation_year=2006;  citation_volume=444;  citation_pages=97-101; " /><meta name="citation_reference" content="citation_title=The role of cell wall degrading enzymes in pathogenesis of Magnaporthe oryzae,;  citation_journal_title=Curr. Protein Pept. Sci;  citation_year=2016;  citation_volume=17; " /><meta name="citation_reference" content="citation_title=Genome-wide transcriptional profiling of Botrytis cinerea genes targeting plant cell walls during infections of different hosts,;  citation_journal_title=Front. Plant Sci;  citation_year=2013;  citation_volume=5;  citation_pages=435" /><meta name="citation_reference" content="citation_title=PHI-base update: additions to the pathogen host interaction database,;  citation_journal_title=Nucleic Acids Res;  citation_year=2008;  citation_volume=36;  citation_pages=572-6; " /><meta name="citation_reference" content="citation_title=Linking emerging fungal plant diseases and ecological speciation,;  citation_journal_title=Trends Ecol. Evol;  citation_year=2010;  citation_volume=25;  citation_pages=387-95; " /><meta name="citation_reference" content="citation_title=Population genetics of fungal and Oomycete effectors involved in gene-for-gene interactions,;  citation_journal_title=Mol. Plant Microbe Interact;  citation_year=2009;  citation_volume=22;  citation_pages=371-80; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/24/6/635/22295829/dsx031.pdf" /><meta name="description" content="Abstract.  Ustilago esculenta, infects Zizania latifolia, and induced host stem swollen to be a popular vegetable called Jiaobai in China. It is the long-standi" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/24/6/635/3979487" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/24/6/635/3979487" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/24/6/635/3979487");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsx031';
    </script>
<title>
	Comparative whole-genome analysis reveals artificial selection effects on Ustilago esculenta genome | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./3979487" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BcniATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iWWUsIFppaG9uZyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgWmhlamlhbmcgUHJvdmluY2lhbCBLZXkgTGFib3JhdG9yeSBvZiBCaW9tZXRyb2xvZ3kgYW5kIEluc3BlY3Rpb24gJmFtcDsgUXVhcmFudGluZSwgQ2hpbmEgSmlsaWFuZyBVbml2ZXJzaXR5LCBIYW5nemhvdSwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJQYW4sIFlhbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQWxnb3JpdGhtaWMgQmlvaW5mb3JtYXRpY3MsIEhlaW5yaWNoIEhlaW5lIFVuaXZlcnNpdHksIETDvHNzZWxkb3JmLCBHZXJtYW55IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQ2x1c3RlciBvZiBFeGNlbGxlbmNlIG9uIFBsYW50IFNjaWVuY2VzIChDRVBMQVMpLCBEw7xzc2VsZG9yZiwgR2VybWFueSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlpoYW5nLCBZYWZlbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgWmhlamlhbmcgUHJvdmluY2lhbCBLZXkgTGFib3JhdG9yeSBvZiBCaW9tZXRyb2xvZ3kgYW5kIEluc3BlY3Rpb24gJmFtcDsgUXVhcmFudGluZSwgQ2hpbmEgSmlsaWFuZyBVbml2ZXJzaXR5LCBIYW5nemhvdSwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJDdWksIEhhaWZlbmciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIFpoZWppYW5nIFByb3ZpbmNpYWwgS2V5IExhYm9yYXRvcnkgb2YgQmlvbWV0cm9sb2d5IGFuZCBJbnNwZWN0aW9uICZhbXA7IFF1YXJhbnRpbmUsIENoaW5hIEppbGlhbmcgVW5pdmVyc2l0eSwgSGFuZ3pob3UsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSmluLCBHdWxlaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQWdyb25vbXkgJmFtcDsgWmhlamlhbmcgS2V5IExhYm9yYXRvcnkgb2YgQ3JvcCBHZXJtcGxhc20gUmVzb3VyY2VzLCBaaGVqaWFuZyBVbml2ZXJzaXR5LCBIYW5nemhvdSwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJNY0hhcmR5LCBBbGljZSBDIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBBbGdvcml0aG1pYyBCaW9pbmZvcm1hdGljcywgSGVpbnJpY2ggSGVpbmUgVW5pdmVyc2l0eSwgRMO8c3NlbGRvcmYsIEdlcm1hbnkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJDbHVzdGVyIG9mIEV4Y2VsbGVuY2Ugb24gUGxhbnQgU2NpZW5jZXMgKENFUExBUyksIETDvHNzZWxkb3JmLCBHZXJtYW55IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iRmFuLCBMb25namlhbmciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEFncm9ub215ICZhbXA7IFpoZWppYW5nIEtleSBMYWJvcmF0b3J5IG9mIENyb3AgR2VybXBsYXNtIFJlc291cmNlcywgWmhlamlhbmcgVW5pdmVyc2l0eSwgSGFuZ3pob3UsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iWXUsIFhpYW9waW5nIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBaaGVqaWFuZyBQcm92aW5jaWFsIEtleSBMYWJvcmF0b3J5IG9mIEJpb21ldHJvbG9neSBhbmQgSW5zcGVjdGlvbiAmYW1wOyBRdWFyYW50aW5lLCBDaGluYSBKaWxpYW5nIFVuaXZlcnNpdHksIEhhbmd6aG91LCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iQ29tcGFyYXRpdmUgd2hvbGUtZ2Vub21lIGFuYWx5c2lzIHJldmVhbHMgYXJ0aWZpY2lhbCBzZWxlY3Rpb24gZWZmZWN0cyBvbiBVc3RpbGFnbyBlc2N1bGVudGEgZ2Vub21lIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iNjM1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSI2NDgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3gwMzEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMjQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxNy8xMi8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU51dHJpdGlvbmFsIHJlcXVpcmVtZW50cyBvZiB0aGUgZWRpYmxlIGdhbGwtcHJvZHVjaW5nIGZ1bmd1cyBVc3RpbGFnbyBlc2N1bGVudGEsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBCaW9sLiBTY2k7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTI0Ni01MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGh5bG9nZW55IGFuZCBiaW9nZW9ncmFwaHkgb2YgdGhlIGVhc3Rlcm4gQXNpYW4tTm9ydGggQW1lcmljYW4gZGlzanVuY3Qgd2lsZC1yaWNlIGdlbnVzIChaaXphbmlhIEwuLCBQb2FjZWFlKSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gUGh5bG9nZW5ldC4gRXZvbDsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT01NTsgIGNpdGF0aW9uX3BhZ2VzPTEwMDgtMTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVppemFuaWEgbGF0aWZvbGlhIFR1cmN6LiBjdWx0aXZhdGVkIGluIENoaW5hLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZXQuIFJlc291ci4gQ3JvcCBFdm9sOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTU0OyAgY2l0YXRpb25fcGFnZXM9MTIxMS03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NYWluIGFncm9ub21pYyB0cmFpdHMsIGRvbWVzdGljYXRpb24gYW5kIGJyZWVkaW5nIG9mIEd1IChaaXphbmlhIGxhdGlmb2xpYSksOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBaaGVqaWFuZyBVbml2OyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTM5OyAgY2l0YXRpb25fcGFnZXM9NjI5LTM1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Gb3JtYXRpb24gYW5kIGhpc3RvcGF0aG9sb2d5IG9mIGdhbGxzIGluZHVjZWQgYnkgVXN0aWxhZ2UgZXNjdWxlbnRhIGluIFppemFuaWEgbGF0aWZvbGlhLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGh5dG9wYXRob2xvZ3k7ICBjaXRhdGlvbl95ZWFyPTE5Nzg7ICBjaXRhdGlvbl92b2x1bWU9Njg7ICBjaXRhdGlvbl9wYWdlcz0xNTcyLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNoZW1pY2FsIGNvbXBvc2l0aW9uLCBhbmdpb3RlbnNpbi1jb252ZXJ0aW5nIGVuenltZS1pbmhpYml0b3J5IGFjdGl2aXR5IGFuZCBhbnRpb3hpZGFudCBhY3Rpdml0aWVzIG9mIGZldy1mbG93ZXIgd2lsZCByaWNlIChaaXphbmlhIGxhdGlmb2xpYSBUdXJjei4pLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gU2NpLiBGb29kIEFncmljOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTkyOyAgY2l0YXRpb25fcGFnZXM9MTU5LTY0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QbGFudCBncm93dGggYW5kIHBob3Rvc3ludGhldGljIHBlcmZvcm1hbmNlIG9mIFppemFuaWEgbGF0aWZvbGlhIGFyZSBhbHRlcmVkIGJ5IGVuZG9waHl0aWMgVXN0aWxhZ28gZXNjdWxlbnRhIGluZmVjdGlvbiw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBoeXNpb2wuIE1vbC4gUGxhbnQgUGF0aG9sOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTgzOyAgY2l0YXRpb25fcGFnZXM9NzUtODM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhpZ2hseSBlZmZpY2llbnQgYW5kIHNhZmUgcHJvZHVjdGlvbiBvZiBKaWFvYmFpLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpLiBBZ3JpYzsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT02OTsgIGNpdGF0aW9uX3BhZ2VzPTE0Mi02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1PcmlnaW5zIG9mIGhvc3Qtc3BlY2lmaWMgcG9wdWxhdGlvbnMgb2YgdGhlIGJsYXN0IHBhdGhvZ2VuIE1hZ25hcG9ydGhlIG9yeXphZSBpbiBjcm9wIGRvbWVzdGljYXRpb24gd2l0aCBzdWJzZXF1ZW50IGV4cGFuc2lvbiBvZiBwYW5kZW1pYyBjbG9uZXMgb24gcmljZSBhbmQgd2VlZHMgb2YgcmljZSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTE3MDsgIGNpdGF0aW9uX3BhZ2VzPTYxMy0zMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9T3JpZ2luIGFuZCBkb21lc3RpY2F0aW9uIG9mIHRoZSBmdW5nYWwgd2hlYXQgcGF0aG9nZW4gTXljb3NwaGFlcmVsbGEgZ3JhbWluaWNvbGEgdmlhIHN5bXBhdHJpYyBzcGVjaWF0aW9uLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBCaW9sLiBFdm9sOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTI0OyAgY2l0YXRpb25fcGFnZXM9Mzk4LTQxMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBib3VudGlmdWwgaGFydmVzdDogZ2Vub21pYyBpbnNpZ2h0cyBpbnRvIGNyb3AgZG9tZXN0aWNhdGlvbiBwaGVub3R5cGVzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QW5udS4gUmV2LiBQbGFudCBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTY0OyAgY2l0YXRpb25fcGFnZXM9NDctNzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNyb3AgZG9tZXN0aWNhdGlvbiBhbmQgaXRzIGltcGFjdCBvbiBuYXR1cmFsbHkgc2VsZWN0ZWQgdHJvcGhpYyBpbnRlcmFjdGlvbnMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Bbm51LiBSZXYuIEVudG9tb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9NjA7ICBjaXRhdGlvbl9wYWdlcz0zNS01ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIG1ha2luZyBvZiBhIG5ldyBwYXRob2dlbjogaW5zaWdodHMgZnJvbSBjb21wYXJhdGl2ZSBwb3B1bGF0aW9uIGdlbm9taWNzIG9mIHRoZSBkb21lc3RpY2F0ZWQgd2hlYXQgcGF0aG9nZW4gTXljb3NwaGFlcmVsbGEgZ3JhbWluaWNvbGEgYW5kIGl0cyB3aWxkIHNpc3RlciBzcGVjaWVzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTIxNTctNjY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBhcmF0aXZlIGFuYWx5c2lzIG9mIHRoZSBnZW5vbWVzIG9mIHR3byBmaWVsZCBpc29sYXRlcyBvZiB0aGUgcmljZSBibGFzdCBmdW5ndXMgTWFnbmFwb3J0aGUgb3J5emFlLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9ZTEwMDI4NjkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Fdm9sdXRpb24sIHNlbGVjdGlvbiBhbmQgaXNvbGF0aW9uOiBhIGdlbm9taWMgdmlldyBvZiBzcGVjaWF0aW9uIGluIGZ1bmdhbCBwbGFudCBwYXRob2dlbnMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OZXcgUGh5dG9sOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTE5OTsgIGNpdGF0aW9uX3BhZ2VzPTg5NS05MDc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmUgbG9zcyByYXRoZXIgdGhhbiBnZW5lIGdhaW4gaXMgYXNzb2NpYXRlZCB3aXRoIGEgaG9zdCBqdW1wIGZyb20gbW9ub2NvdHMgdG8gZGljb3RzIGluIHRoZSBTbXV0IEZ1bmd1cyBNZWxhbm9wc2ljaGl1bSBwZW5uc3lsdmFuaWN1bSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBCaW9sLiBFdm9sOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz0yMDM0LTQ5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJIZWluemUgQi4gIDIwMDkgLCBDb21wYXJhdGl2ZSBhbmFseXNpcyBvZiB0aGUgTWFpemUgU211dCBGdW5naSBVc3RpbGFnbyBtYXlkaXMgYW5kIFNwb3Jpc29yaXVtIHJlaWxpYW51bS4gUGhpbGlwcHMtVW5pdmVyc2l0w6R0IE1hcmJ1cmcsIEdlcm1hbnkuICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgcHJlbGltaW5hcnkgc3R1ZHkgb24gcG9wdWxhdGlvbiBnZW5ldGljIHN0cnVjdHVyZSBhbmQgcGh5bG9nZW9ncmFwaHkgb2YgdGhlIHdpbGQgYW5kIGN1bHRpdmF0ZWQgWml6YW5pYSBsYXRpZm9saWEgKFBvYWNlYWUpIGJhc2VkIG9uIEFkaDFhIHNlcXVlbmNlcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRoZW9yLiBBcHBsLiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xMTY7ICBjaXRhdGlvbl9wYWdlcz04MzUtNDM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNobzEgYW5kIE1zYjItcmVsYXRlZCBwcm90ZWlucyByZWd1bGF0ZSBhcHByZXNzb3JpdW0gZGV2ZWxvcG1lbnQgaW4gdGhlIHNtdXQgZnVuZ3VzIFVzdGlsYWdvIG1heWRpcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MjI7ICBjaXRhdGlvbl9wYWdlcz0yMDg1LTEwMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlmZmVyZW50IHNpZ25hbGxpbmcgcGF0aHdheXMgaW52b2x2aW5nIGEgRyBhbHBoYSBwcm90ZWluLCBjQU1QIGFuZCBhIE1BUCBraW5hc2UgY29udHJvbCBnZXJtaW5hdGlvbiBvZiBCb3RyeXRpcyBjaW5lcmVhIGNvbmlkaWEsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIE1pY3JvYmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT01OTsgIGNpdGF0aW9uX3BhZ2VzPTgyMS0zNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVwcm9ncmFtbWluZyBhIG1haXplIHBsYW50OiB0cmFuc2NyaXB0aW9uYWwgYW5kIG1ldGFib2xpYyBjaGFuZ2VzIGluZHVjZWQgYnkgdGhlIGZ1bmdhbCBiaW90cm9waCBVc3RpbGFnbyBtYXlkaXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBKOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTU2OyAgY2l0YXRpb25fcGFnZXM9MTgxLTk1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QbGFudCBzdXJmYWNlIGN1ZXMgcHJpbWUgVXN0aWxhZ28gbWF5ZGlzIGZvciBiaW90cm9waGljIGRldmVsb3BtZW50LDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBQYXRob2c7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz0yOTUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaGFyYWN0ZXJpemF0aW9uIG9mIHRoZSBsYXJnZXN0IGVmZmVjdG9yIGdlbmUgY2x1c3RlciBvZiBVc3RpbGFnbyBtYXlkaXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIFBhdGhvZzsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTE5MS04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NYWl6ZSB0dW1vcnMgY2F1c2VkIGJ5IFVzdGlsYWdvIG1heWRpcyByZXF1aXJlIG9yZ2FuLXNwZWNpZmljIGdlbmVzIGluIGhvc3QgYW5kIHBhdGhvZ2VuLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0zMjg7ICBjaXRhdGlvbl9wYWdlcz04OS05MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lIGNvbXBhcmlzb24gb2YgYmFybGV5IGFuZCBtYWl6ZSBzbXV0IGZ1bmdpIHJldmVhbHMgdGFyZ2V0ZWQgbG9zcyBvZiBSTkEgc2lsZW5jaW5nIGNvbXBvbmVudHMgYW5kIHNwZWNpZXMtc3BlY2lmaWMgcHJlc2VuY2Ugb2YgdHJhbnNwb3NhYmxlIGVsZW1lbnRzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTE3MzMtNDU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBhdGhvZ2VuaWNpdHkgZGV0ZXJtaW5hbnRzIGluIHNtdXQgZnVuZ2kgcmV2ZWFsZWQgYnkgZ2Vub21lIGNvbXBhcmlzb24sOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTMzMDsgIGNpdGF0aW9uX3BhZ2VzPTE1NDYtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2NhZmZvbGRpbmcgcHJlLWFzc2VtYmxlZCBjb250aWdzIHVzaW5nIFNTUEFDRSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTI3OyAgY2l0YXRpb25fcGFnZXM9NTc4LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRvd2FyZCBhbG1vc3QgY2xvc2VkIGdlbm9tZXMgd2l0aCBHYXBGaWxsZXIsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTEtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TFRSX0ZJTkRFUjogYW4gZWZmaWNpZW50IHRvb2wgZm9yIHRoZSBwcmVkaWN0aW9uIG9mIGZ1bGwtbGVuZ3RoIExUUiByZXRyb3RyYW5zcG9zb25zLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MzU7ICBjaXRhdGlvbl9wYWdlcz0yNjUtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VXNpbmcgbmF0aXZlIGFuZCBzeW50ZW5pY2FsbHkgbWFwcGVkIGNETkEgYWxpZ25tZW50cyB0byBpbXByb3ZlIGRlIG5vdm8gZ2VuZSBmaW5kaW5nLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MjQ7ICBjaXRhdGlvbl9wYWdlcz02MzctNDQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNyZWF0aW5nIGEgaG9uZXkgYmVlIGNvbnNlbnN1cyBnZW5lIHNldCw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz1SMTMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CYXNpYyBsb2NhbCBhbGlnbm1lbnQgc2VhcmNoIHRvb2wsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBNb2wuIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTE5OTA7ICBjaXRhdGlvbl92b2x1bWU9MjE1OyAgY2l0YXRpb25fcGFnZXM9NDAzLTEwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbnRlclByb1NjYW46IHByb3RlaW4gZG9tYWlucyBpZGVudGlmaWVyLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MzM7ICBjaXRhdGlvbl9wYWdlcz0xMTYtMjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU9ydGhvTUNMOiBpZGVudGlmaWNhdGlvbiBvZiBvcnRob2xvZyBncm91cHMgZm9yIGV1a2FyeW90aWMgZ2Vub21lcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz0yMTc4LTg5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1kYkNBTjogYSB3ZWIgcmVzb3VyY2UgZm9yIGF1dG9tYXRlZCBjYXJib2h5ZHJhdGUtYWN0aXZlIGVuenltZSBhbm5vdGF0aW9uLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NDA7ICBjaXRhdGlvbl9wYWdlcz00NDUtNTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNNVVJGOiBnZW5vbWljIG1hcHBpbmcgb2YgZnVuZ2FsIHNlY29uZGFyeSBtZXRhYm9saXRlIGNsdXN0ZXJzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RnVuZ2FsIEdlbmV0LiBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTQ3OyAgY2l0YXRpb25fcGFnZXM9NzM2LTQxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JZGVudGlmaWNhdGlvbiBvZiBwcm9rYXJ5b3RpYyBhbmQgZXVrYXJ5b3RpYyBzaWduYWwgcGVwdGlkZXMgcHJlZGljdGlvbiBvZiB0aGVpciBjbGVhdmFnZSBzaXRlcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb3RlaW4gRW5nOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9MS02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QcmVkaWN0aW5nIHN1YmNlbGx1bGFyIGxvY2FsaXphdGlvbiBvZiBwcm90ZWlucyBiYXNlZCBvbiB0aGVpciBOLXRlcm1pbmFsIGFtaW5vIGFjaWQgc2VxdWVuY2UsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBNb2wuIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MzAwOyAgY2l0YXRpb25fcGFnZXM9MTAwNS0xNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9dFJOQXNjYW4tU0U6IGEgcHJvZ3JhbSBmb3IgaW1wcm92ZWQgZGV0ZWN0aW9uIG9mIHRyYW5zZmVyIFJOQSBnZW5lcyBpbiBnZW5vbWljIHNlcXVlbmNlLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz05NTUtNjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUluZmVybmFsIDEuMDogaW5mZXJlbmNlIG9mIFJOQSBhbGlnbm1lbnRzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0xMzM1LTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJmYW0gMTEuMDogMTDigIl5ZWFycyBvZiBSTkEgZmFtaWxpZXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT00MTsgIGNpdGF0aW9uX3BhZ2VzPTIyNi0zMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VmVyc2F0aWxlIGFuZCBvcGVuIHNvZnR3YXJlIGZvciBjb21wYXJpbmcgbGFyZ2UgZ2Vub21lcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz1SMTIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TT0FQMjogYW4gaW1wcm92ZWQgdWx0cmFmYXN0IHRvb2wgZm9yIHNob3J0IHJlYWQgYWxpZ25tZW50LDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0xOTY2LTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1hcHBpbmcgYW5kIHF1YW50aWZ5aW5nIG1hbW1hbGlhbiB0cmFuc2NyaXB0b21lcyBieSBSTkEtU2VxLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBNZXRob2RzOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz02MjEtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9ZWRnZVI6IGEgQmlvY29uZHVjdG9yIHBhY2thZ2UgZm9yIGRpZmZlcmVudGlhbCBleHByZXNzaW9uIGFuYWx5c2lzIG9mIGRpZ2l0YWwgZ2VuZSBleHByZXNzaW9uIGRhdGEsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0yNjsgIGNpdGF0aW9uX3BhZ2VzPTEzOS00MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBob3N0IHBsYW50IGdlbmVtZSAoWml6YW5pYSBsYXRpZm9saWEpIGFmdGVyIGEgY2VudHVyeS1sb25nIGVuZG9waHl0ZSBpbmZlY3Rpb24sOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBKOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTgzOyAgY2l0YXRpb25fcGFnZXM9NjAwLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZhc3QsIHNjYWxhYmxlIGdlbmVyYXRpb24gb2YgaGlnaC1xdWFsaXR5IHByb3RlaW4gbXVsdGlwbGUgc2VxdWVuY2UgYWxpZ25tZW50cyB1c2luZyBDbHVzdGFsIE9tZWdhLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBTeXN0LiBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz0xNDI5LTMyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZXZvbHV0aW9uIG9mIHNwbGljZW9zb21hbCBpbnRyb25zOiBwYXR0ZXJucywgcHV6emxlcyBhbmQgcHJvZ3Jlc3MsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIFJldi4gR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTIxMS0yMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RG8gVEUgYWN0aXZpdHkgYW5kIGNvdW50ZXJhY3RpbmcgZ2Vub21lIGRlZmVuc2VzLCBSTkFpIGFuZCBtZXRoeWxhdGlvbiwgc2hhcGUgdGhlIHNleCBsaXZlcyBvZiBzbXV0IGZ1bmdpPyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IFNpZ25hbC4gQmVoYXY7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPWUyMzg1MyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByZW1laW90aWMgaW5zdGFiaWxpdHkgb2YgcmVwZWF0ZWQgc2VxdWVuY2VzIGluIE5ldXJvc3BvcmEgY3Jhc3NhLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QW5udS4gUmV2LiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MTk5MDsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTU3OS02MTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZ1bmN0aW9uYWwgYW5kIGV2b2x1dGlvbmFyeSBhbmFseXNpcyBvZiB0aGUgZ2Vub21lIG9mIGFuIG9ibGlnYXRlIGZ1bmdhbCBzeW1iaW9udCw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBCaW9sLiBFdm9sOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz04OTEtOTA0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbnRlcnNwZWNpZmljIHNleCBpbiBncmFzcyBzbXV0cyBhbmQgdGhlIGdlbmV0aWMgZGl2ZXJzaXR5IG9mIHRoZWlyIHBoZXJvbW9uZS1yZWNlcHRvciBzeXN0ZW0sOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz1lMTAwMjQzNiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1hdGluZy10eXBlIGxvY3VzIG9mIENyeXB0b2NvY2N1cyBuZW9mb3JtYW5zOiBhIHN0ZXAgaW4gdGhlIGV2b2x1dGlvbiBvZiBzZXggY2hyb21vc29tZXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1FdWthcnlvdC4gQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0xOyAgY2l0YXRpb25fcGFnZXM9NzA0LTE4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TaGFyZWQgZm9yY2VzIG9mIHNleCBjaHJvbW9zb21lIGV2b2x1dGlvbiBpbiBoYXBsb2lkLW1hdGluZyBhbmQgZGlwbG9pZC1tYXRpbmcgb3JnYW5pc21zOiBNaWNyb2JvdHJ5dW0gdmlvbGFjZXVtIGFuZCBvdGhlciBtb2RlbCBvcmdhbmlzbXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNjg7ICBjaXRhdGlvbl9wYWdlcz0xNDEtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXZvbHV0aW9uYXJ5IGVyb3Npb24gb2YgeWVhc3Qgc2V4IGNocm9tb3NvbWVzIGJ5IG1hdGluZy10eXBlIHN3aXRjaGluZyBhY2NpZGVudHMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDg7ICBjaXRhdGlvbl9wYWdlcz0yMDAyNC05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZXZvbHV0aW9uYXJ5IGR5bmFtaWNzIG9mIHRyYW5zcG9zYWJsZSBlbGVtZW50cyBpbiBldWthcnlvdGUgZ2Vub21lcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBEeW47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTY4LTkxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaGFyYWN0ZXJpemF0aW9uIG9mIHJldHJvdHJhbnNwb3NvbiBzZXF1ZW5jZXMgZXhwcmVzc2VkIGluIGluZmxvcmVzY2VuY2VzIG9mIGFwb21pY3RpYyBhbmQgc2V4dWFsIFBhc3BhbHVtIG5vdGF0dW0gcGxhbnRzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2V4LiBQbGFudCBSZXByb2Q7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MjQ7ICBjaXRhdGlvbl9wYWdlcz0yMzEtNDY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2b2x1dGlvbjogdHJhbnNwb3NvbnMgZHJpdmUgc2V4IGNocm9tb3NvbWUgZXZvbHV0aW9uLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBSZXYuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTE1OyAgY2l0YXRpb25faXNzdWU9MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGEyIG1hdGluZy10eXBlIGxvY3VzIGdlbmVzIGxnYTIgYW5kIHJnYTIgZGlyZWN0IHVuaXBhcmVudGFsIG1pdG9jaG9uZHJpYWwgRE5BIChtdEROQSkgaW5oZXJpdGFuY2UgYW5kIGNvbnN0cmFpbiBtdEROQSByZWNvbWJpbmF0aW9uIGR1cmluZyBzZXh1YWwgZGV2ZWxvcG1lbnQgb2YgVXN0aWxhZ28gbWF5ZGlzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTgxOyAgY2l0YXRpb25fcGFnZXM9ODQ3LTYwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgVXN0aWxhZ28gbWF5ZGlzIGEyIG1hdGluZy10eXBlIGxvY3VzIGdlbmVzIGxnYTIgYW5kIHJnYTIgY29tcHJvbWlzZSBwYXRob2dlbmljaXR5IGluIHRoZSBhYnNlbmNlIG9mIHRoZSBtaXRvY2hvbmRyaWFsIHAzMiBmYW1pbHkgcHJvdGVpbiBNcmIxLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTIyMzMtNDg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUF0eXBpY2FsIG1pdG9jaG9uZHJpYWwgaW5oZXJpdGFuY2UgcGF0dGVybnMgaW4gZXVrYXJ5b3Rlcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZTsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT01ODsgIGNpdGF0aW9uX3BhZ2VzPTQyMy0zMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RW52aXJvbm1lbnQgZmFjdG9ycyBjYW4gaW5mbHVlbmNlIG1pdG9jaG9uZHJpYWwgaW5oZXJpdGFuY2UgaW4gdGhlIGZ1bmd1cyBDcnlwdG9jb2NjdXMgbmVvZm9ybWFucyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUZ1bmdhbCBHZW5ldC4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT00NDsgIGNpdGF0aW9uX3BhZ2VzPTMxNS0yMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHRyYW5zY3JpcHRpb24gZmFjdG9yIFJiZjEgaXMgdGhlIG1hc3RlciByZWd1bGF0b3IgZm9yIGItbWF0aW5nIHR5cGUgY29udHJvbGxlZCBwYXRob2dlbmljIGRldmVsb3BtZW50IGluIFVzdGlsYWdvIG1heWRpcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgUGF0aG9nOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz03NjEtNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VXN0aWxhZ28gbWF5ZGlzIGFzIGEgcGF0aG9nZW4sOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QaHl0b3BhdGhvbG9neTsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT00NzsgIGNpdGF0aW9uX3BhZ2VzPTQyMy00NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVndWxhdGlvbiBvZiBtYXRpbmcgYW5kIHBhdGhvZ2VuaWMgZGV2ZWxvcG1lbnQgaW4gVXN0aWxhZ28gbWF5ZGlzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q3Vyci4gT3Bpbi4gTWljcm9iaW9sOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz02NjYtNzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBoZXJvbW9uZS1yZWd1bGF0ZWQgdGFyZ2V0IGdlbmVzIHJlc3BvbmQgZGlmZmVyZW50aWFsbHkgdG8gTUFQSyBwaG9zcGhvcnlsYXRpb24gb2YgdHJhbnNjcmlwdGlvbiBmYWN0b3IgUHJmMSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gTWljcm9iaW9sOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTY5OyAgY2l0YXRpb25fcGFnZXM9MTA0MS01MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIFVzdGlsYWdvIG1heWRpcyBDbHAxIHByb3RlaW4gb3JjaGVzdHJhdGVzIHBoZXJvbW9uZSBhbmQgYi1kZXBlbmRlbnQgc2lnbmFsaW5nIHBhdGh3YXlzIHRvIGNvb3JkaW5hdGUgdGhlIGNlbGwgY3ljbGUgYW5kIHBhdGhvZ2VuaWMgZGV2ZWxvcG1lbnQsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBDZWxsOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTIyOyAgY2l0YXRpb25fcGFnZXM9MjkwOC0yMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWdTd2UxcCByZWd1bGF0ZXMgbWl0b3NpcyBpbiByZXNwb25zZSB0byBtb3JwaG9nZW5lc2lzIGFuZCBudXRyaWVudHMgaW4gbXVsdGludWNsZWF0ZWQgQXNoYnlhIGdvc3N5cGlpIGNlbGxzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBCaW9sLiBDZWxsOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTE3OyAgY2l0YXRpb25fcGFnZXM9NDQ5NC01MTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgZ2Vub21lLWJhc2VkIGFuYWx5c2lzIG9mIGFtaW5vIGFjaWQgbWV0YWJvbGlzbSBpbiB0aGUgYmlvdHJvcGhpYyBwbGFudCBwYXRob2dlbiBVc3RpbGFnbyBtYXlkaXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1GdW5nYWwgR2VuZXQuIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9NDUoU3VwcGwgMSk7ICBjaXRhdGlvbl9wYWdlcz1TNzctODc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFwaGlkIGdlbm9tZSBleHByZXNzaW9uIHJldmVhbHMgaG9zdC1zeW1iaW9udCBjb29wZXJhdGlvbiBpbiB0aGUgcHJvZHVjdGlvbiBvZiBhbWlubyBhY2lkcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTEwODsgIGNpdGF0aW9uX3BhZ2VzPTI4NDktNTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlub2N1bHVtIHNpemUgZWZmZWN0IGluIGRpbW9ycGhpYyBmdW5naTogZXh0cmFjZWxsdWxhciBjb250cm9sIG9mIHllYXN0LW15Y2VsaXVtIGRpbW9ycGhpc20gaW4gQ2VyYXRvY3lzdGlzIHVsbWksOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1BcHBsLiBFbnZpcm9uLiBNaWNyb2Jpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9NzA7ICBjaXRhdGlvbl9wYWdlcz0xMzU2LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNhbmRpZGEgYWxiaWNhbnMgaW5kdWNlcyBhcmdpbmluZSBiaW9zeW50aGV0aWMgZ2VuZXMgaW4gcmVzcG9uc2UgdG8gaG9zdC1kZXJpdmVkIHJlYWN0aXZlIG94eWdlbiBzcGVjaWVzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RXVrYXJ5b3QuIENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz05MS0xMDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBhcnRtZW50YWwgYW5kIHJlZ3VsYXRvcnkgbWVjaGFuaXNtcyBpbiB0aGUgYXJnaW5pbmUgcGF0aHdheXMgb2YgTmV1cm9zcG9yYSBjcmFzc2EgYW5kIFNhY2NoYXJvbXljZXMgY2VyZXZpc2lhZSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1pY3JvYmlvbC4gUmV2OyAgY2l0YXRpb25feWVhcj0xOTg2OyAgY2l0YXRpb25fdm9sdW1lPTUwOyAgY2l0YXRpb25fcGFnZXM9MjgwLTMxMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9T3JuaXRoaW5lLWRlbHRhLWFtaW5vdHJhbnNmZXJhc2UgYW5kIHByb2xpbmUgZGVoeWRyb2dlbmFzZSBnZW5lcyBwbGF5IGEgcm9sZSBpbiBub24taG9zdCBkaXNlYXNlIHJlc2lzdGFuY2UgYnkgcmVndWxhdGluZyBweXJyb2xpbmUtNS1jYXJib3h5bGF0ZSBtZXRhYm9saXNtLWluZHVjZWQgaHlwZXJzZW5zaXRpdmUgcmVzcG9uc2U7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IENlbGwgRW52aXJvbjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0zNTsgIGNpdGF0aW9uX3BhZ2VzPTEzMjktNDM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSB3aGVhdCBldGh5bGVuZSByZXNwb25zZSBmYWN0b3IgdHJhbnNjcmlwdGlvbiBmYWN0b3IgcGF0aG9nZW4taW5kdWNlZCBFUkYxIG1lZGlhdGVzIGhvc3QgcmVzcG9uc2VzIHRvIGJvdGggdGhlIG5lY3JvdHJvcGhpYyBwYXRob2dlbiBSaGl6b2N0b25pYSBjZXJlYWxpcyBhbmQgZnJlZXppbmcgc3RyZXNzZXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudC4gUGh5c2lvbDsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNjQ7ICBjaXRhdGlvbl9wYWdlcz0xNDk5LTUxNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWV0aGlvbmluZSBlbGljaXRzIEgyTzIgZ2VuZXJhdGlvbiBhbmQgZGVmZW5zZSBnZW5lIGV4cHJlc3Npb24gaW4gZ3JhcGV2aW5lIGFuZCByZWR1Y2VzIFBsYXNtb3BhcmEgdml0aWNvbGEgaW5mZWN0aW9uLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gUGxhbnQgUGh5c2lvbDsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xNzA7ICBjaXRhdGlvbl9wYWdlcz0xNTYxLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhvbW9jeXN0ZWluZSBvdmVyLWFjY3VtdWxhdGlvbiBhcyB0aGUgZWZmZWN0IG9mIHBvdGF0byBsZWF2ZXMgZXhwb3N1cmUgdG8gYmlvdGljIHN0cmVzcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IFBoeXNpb2wuIEJpb2NoZW07ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9NjM7ICBjaXRhdGlvbl9wYWdlcz0xNzctODQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUx5c2luZSBjYXRhYm9saXNtLCBhbWlubyBhY2lkIHRyYW5zcG9ydCwgYW5kIHN5c3RlbWljIGFjcXVpcmVkIHJlc2lzdGFuY2U6IHdoYXQgaXMgdGhlIGxpbms/LDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgU2lnbmFsLiBCZWhhdjsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9ZTI4OTMzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW4gaW1wb3J0YW50IHJvbGUgb2YgdGhlIHBlcHBlciBwaGVueWxhbGFuaW5lIGFtbW9uaWEtbHlhc2UgZ2VuZSAoUEFMMSkgaW4gc2FsaWN5bGljIGFjaWQtZGVwZW5kZW50IHNpZ25hbGxpbmcgb2YgdGhlIGRlZmVuY2UgcmVzcG9uc2UgdG8gbWljcm9iaWFsIHBhdGhvZ2Vucyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEV4cC4gQm90OyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTY1OyAgY2l0YXRpb25fcGFnZXM9MjI5NS0zMDY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN0cmVwdG9teWNlcy1pbmR1Y2VkIHJlc2lzdGFuY2UgYWdhaW5zdCBvYWsgcG93ZGVyeSBtaWxkZXcgaW52b2x2ZXMgaG9zdCBwbGFudCByZXNwb25zZXMgaW4gZGVmZW5zZSwgcGhvdG9zeW50aGVzaXMsIGFuZCBzZWNvbmRhcnkgbWV0YWJvbGlzbSBwYXRod2F5cyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gUGxhbnQgTWljcm9iZSBJbnRlcmFjdDsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0yNzsgIGNpdGF0aW9uX3BhZ2VzPTg5MS05MDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgS1MtdHlwZSBkZWh5ZHJpbiBhbmQgaXRzIHJlbGF0ZWQgZG9tYWlucyByZWR1Y2UgQ3UtcHJvbW90ZWQgcmFkaWNhbCBnZW5lcmF0aW9uIGFuZCB0aGUgaGlzdGlkaW5lIHJlc2lkdWVzIGNvbnRyaWJ1dGUgdG8gdGhlIHJhZGljYWwtcmVkdWNpbmcgYWN0aXZpdGllcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEV4cC4gQm90OyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTY0OyAgY2l0YXRpb25fcGFnZXM9MTYxNS0yNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9T3B0aWNhbCBtZWFzdXJlbWVudHMgb2YgaW52YXNpdmUgZm9yY2VzIGV4ZXJ0ZWQgYnkgYXBwcmVzc29yaWEgb2YgYSBwbGFudCBwYXRob2dlbmljIGZ1bmd1cyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9Mjg1OyAgY2l0YXRpb25fcGFnZXM9MTg5Ni05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QZW5ldHJhdGlvbiBvZiBoYXJkIHN1YnN0cmF0ZXMgYnkgYSBmdW5ndXMgZW1wbG95aW5nIGVub3Jtb3VzIHR1cmdvciBwcmVzc3VyZXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MTk5MTsgIGNpdGF0aW9uX3ZvbHVtZT04ODsgIGNpdGF0aW9uX3BhZ2VzPTExMjgxLTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVuZG9wbGFzbWljIHJldGljdWx1bSBnbHVjb3NpZGFzZSBJSSBpcyByZXF1aXJlZCBmb3IgcGF0aG9nZW5pY2l0eSBvZiBVc3RpbGFnbyBtYXlkaXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBDZWxsOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTE3OyAgY2l0YXRpb25fcGFnZXM9MzUzMi00MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIENvY2hsaW9ib2x1cyBjYXJib251bSBTTkYxIGdlbmUgaXMgcmVxdWlyZWQgZm9yIGNlbGwgd2FsbC1kZWdyYWRpbmcgZW56eW1lIGV4cHJlc3Npb24gYW5kIHZpcnVsZW5jZSBvbiBtYWl6ZSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz0yMzctNDc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5vdmVsIEctcHJvdGVpbi1jb3VwbGVkIHJlY2VwdG9yLWxpa2UgcHJvdGVpbnMgaW4gdGhlIHBsYW50IHBhdGhvZ2VuaWMgZnVuZ3VzIE1hZ25hcG9ydGhlIGdyaXNlYSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz1SMjQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIG1haXplLXNwZWNpZmljYWxseSBleHByZXNzZWQgZ2VuZSBjbHVzdGVyIGluIFVzdGlsYWdvIG1heWRpcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gTWljcm9iaW9sOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTQzOyAgY2l0YXRpb25fcGFnZXM9NzUtOTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZpY2F0aW9uIG9mIGNpcy1hY3RpdmUgZWxlbWVudHMgaW4gVXN0aWxhZ28gbWF5ZGlzIG1pZzIgcHJvbW90ZXJzIGNvbmZlcnJpbmcgaGlnaC1sZXZlbCBhY3Rpdml0eSBkdXJpbmcgcGF0aG9nZW5pYyBncm93dGggaW4gbWFpemUsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIFBsYW50IE1pY3JvYmUgSW50ZXJhY3Q7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTg7ICBjaXRhdGlvbl9wYWdlcz03NS04NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2xvbmluZyBhbmQgdGFyZ2V0ZWQgZ2VuZSBkaXNydXB0aW9uIG9mIEVYRzEsIGVuY29kaW5nIGV4by1iZXRhIDEsIDMtZ2x1Y2FuYXNlLCBpbiB0aGUgcGh5dG9wYXRob2dlbmljIGZ1bmd1cyBDb2NobGlvYm9sdXMgY2FyYm9udW0sOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1BcHBsLiBFbnZpcm9uLiBNaWNyb2Jpb2w7ICBjaXRhdGlvbl95ZWFyPTE5OTQ7ICBjaXRhdGlvbl92b2x1bWU9NjA7ICBjaXRhdGlvbl9wYWdlcz01OTQtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RmlsYW1lbnQtc3BlY2lmaWMgZXhwcmVzc2lvbiBvZiBhIGNlbGx1bGFzZSBnZW5lIGluIHRoZSBkaW1vcnBoaWMgZnVuZ3VzIFVzdGlsYWdvIG1heWRpcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2wuIENoZW0uIEhvcHBlIFNleWxlcjsgIGNpdGF0aW9uX3llYXI9MTk5NTsgIGNpdGF0aW9uX3ZvbHVtZT0zNzY7ICBjaXRhdGlvbl9wYWdlcz02MTctMjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlcGVsbGVudHMgaGF2ZSBmdW5jdGlvbmFsbHkgcmVwbGFjZWQgaHlkcm9waG9iaW5zIGluIG1lZGlhdGluZyBhdHRhY2htZW50IHRvIGEgaHlkcm9waG9iaWMgc3VyZmFjZSBhbmQgaW4gZm9ybWF0aW9uIG9mIGh5ZHJvcGhvYmljIGFlcmlhbCBoeXBoYWUgaW4gVXN0aWxhZ28gbWF5ZGlzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TWljcm9iaW9sb2d5OyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTE1MjsgIGNpdGF0aW9uX3BhZ2VzPTM2MDctMTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgc2VjcmV0ZWQgVXN0aWxhZ28gbWF5ZGlzIGVmZmVjdG9yIHByb21vdGVzIHZpcnVsZW5jZSBieSB0YXJnZXRpbmcgYW50aG9jeWFuaW4gYmlvc3ludGhlc2lzIGluIG1haXplLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RWxpZmU7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MzsgIGNpdGF0aW9uX3BhZ2VzPTEyNS0zMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW5zaWdodHMgZnJvbSB0aGUgZ2Vub21lIG9mIHRoZSBiaW90cm9waGljIGZ1bmdhbCBwbGFudCBwYXRob2dlbiBVc3RpbGFnbyBtYXlkaXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NDQ0OyAgY2l0YXRpb25fcGFnZXM9OTctMTAxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgcm9sZSBvZiBjZWxsIHdhbGwgZGVncmFkaW5nIGVuenltZXMgaW4gcGF0aG9nZW5lc2lzIG9mIE1hZ25hcG9ydGhlIG9yeXphZSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUN1cnIuIFByb3RlaW4gUGVwdC4gU2NpOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTE3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSB0cmFuc2NyaXB0aW9uYWwgcHJvZmlsaW5nIG9mIEJvdHJ5dGlzIGNpbmVyZWEgZ2VuZXMgdGFyZ2V0aW5nIHBsYW50IGNlbGwgd2FsbHMgZHVyaW5nIGluZmVjdGlvbnMgb2YgZGlmZmVyZW50IGhvc3RzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RnJvbnQuIFBsYW50IFNjaTsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9NDM1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UEhJLWJhc2UgdXBkYXRlOiBhZGRpdGlvbnMgdG8gdGhlIHBhdGhvZ2VuIGhvc3QgaW50ZXJhY3Rpb24gZGF0YWJhc2UsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0zNjsgIGNpdGF0aW9uX3BhZ2VzPTU3Mi02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1MaW5raW5nIGVtZXJnaW5nIGZ1bmdhbCBwbGFudCBkaXNlYXNlcyBhbmQgZWNvbG9naWNhbCBzcGVjaWF0aW9uLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VHJlbmRzIEVjb2wuIEV2b2w7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0zODctOTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBvcHVsYXRpb24gZ2VuZXRpY3Mgb2YgZnVuZ2FsIGFuZCBPb215Y2V0ZSBlZmZlY3RvcnMgaW52b2x2ZWQgaW4gZ2VuZS1mb3ItZ2VuZSBpbnRlcmFjdGlvbnMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIFBsYW50IE1pY3JvYmUgSW50ZXJhY3Q7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MjI7ICBjaXRhdGlvbl9wYWdlcz0zNzEtODA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8yNC82LzYzNS8yMjI5NTgyOS9kc3gwMzEucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIFVzdGlsYWdvIGVzY3VsZW50YSwgaW5mZWN0cyBaaXphbmlhIGxhdGlmb2xpYSwgYW5kIGluZHVjZWQgaG9zdCBzdGVtIHN3b2xsZW4gdG8gYmUgYSBwb3B1bGFyIHZlZ2V0YWJsZSBjYWxsZWQgSmlhb2JhaSBpbiBDaGluYS4gSXQgaXMgdGhlIGxvbmctc3RhbmRpIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFcy8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj0zOTc5NDg3JnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZDwxWTtAHQx3hBIgZMduV/yno05T" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f6%2f635%2f3979487" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f6%2f635%2f3979487" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=103821" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/24/6"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/6/2/m_dnares_24_6cover.png?Expires=2147483647&amp;Signature=z4yCzT-6enr7Fof9QqkUJvtPQAXDRV5XYB7R7L2HBbQZ2iaySlHxBG~hDePAa-rY6Gn-PH9W8mIlJwAyG9R2RE6Vd3RTDct~I7biyNGgXk9CP54tGn91x8VYIMO24kwuQBqahsZQMv3NxRiQKhbGI0yEDanozvBgG~-9FpaC12JFLE6A4BT6rBzjsJLh9UJqvGv0F2hWW2jLvdITi~5PoATpRIsbIK-S1wcvqp49OSN~oRepZtr6iz~107ho8gZAxoaG8nb4DvXKrvfnCuhhmbRD41NP3K2zFyztHicWtTYb1xunX52RqodNq-M3zVvUit2rSw5-mRSvm-jXb-p6nw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/24/6">
                        <div class="volume trailing-comma">Volume 24</div>
                        <div class="issue">Issue 6</div>

            </a>

        </div>
        <div class="ii-pub-date">
December 2017        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="104086693">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#104086693">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="104086695">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#104086695">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="104086701">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#104086701">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="104086727">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#104086727">3. Results/discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="104086771">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#104086771">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="104086781">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#104086781">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/24/6/623/3953933">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/24/6/649/4077064">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Comparative whole-genome analysis reveals artificial selection effects on <em>Ustilago esculenta</em> genome
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Zihong Ye</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Zhejiang Provincial Key Laboratory of Biometrology and Inspection &amp; Quarantine, China Jiliang University, Hangzhou, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Zihong+Ye">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ye Z">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ye Z&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Zihong Ye</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yao Pan</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Algorithmic Bioinformatics, Heinrich Heine University, Düsseldorf, Germany</div><div class="aff">Cluster of Excellence on Plant Sciences (CEPLAS), Düsseldorf, Germany</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yao+Pan">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Pan Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Pan Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yao Pan</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yafen Zhang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Zhejiang Provincial Key Laboratory of Biometrology and Inspection &amp; Quarantine, China Jiliang University, Hangzhou, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yafen+Zhang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Zhang Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Zhang Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yafen Zhang</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Haifeng Cui</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Zhejiang Provincial Key Laboratory of Biometrology and Inspection &amp; Quarantine, China Jiliang University, Hangzhou, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Haifeng+Cui">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Cui H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Cui H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Haifeng Cui</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Gulei Jin</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Agronomy &amp; Zhejiang Key Laboratory of Crop Germplasm Resources, Zhejiang University, Hangzhou, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Gulei+Jin">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Jin G">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Jin G&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Gulei Jin</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Alice C McHardy</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Algorithmic Bioinformatics, Heinrich Heine University, Düsseldorf, Germany</div><div class="aff">Cluster of Excellence on Plant Sciences (CEPLAS), Düsseldorf, Germany</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Alice+C+McHardy">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=McHardy A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;McHardy A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Alice C McHardy</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Longjiang Fan</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Agronomy &amp; Zhejiang Key Laboratory of Crop Germplasm Resources, Zhejiang University, Hangzhou, China</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsx031-cor1" data-open="dsx031-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Tel. +86-571-86836006. Fax. +86-571-86914401. Email: <a href="mailto:yxp@cjlu.edu.cn" target="_blank">yxp@cjlu.edu.cn</a> (X.Y.); Tel. +86-571-88982730. Email: <a href="mailto:fanlj@zju.edu.cn" target="_blank">fanlj@zju.edu.cn</a> (L.F.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Longjiang+Fan">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Fan L">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Fan L&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Longjiang Fan</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Xiaoping Yu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Zhejiang Provincial Key Laboratory of Biometrology and Inspection &amp; Quarantine, China Jiliang University, Hangzhou, China</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsx031-cor1" data-open="dsx031-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Tel. +86-571-86836006. Fax. +86-571-86914401. Email: <a href="mailto:yxp@cjlu.edu.cn" target="_blank">yxp@cjlu.edu.cn</a> (X.Y.); Tel. +86-571-88982730. Email: <a href="mailto:fanlj@zju.edu.cn" target="_blank">fanlj@zju.edu.cn</a> (L.F.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Xiaoping+Yu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yu X">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yu X&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Xiaoping Yu</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsx031-FM1"><div class="fn"><div class="footnote-content">Zihong Ye and Yao Pan contributed equally to this work.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsx031-FM2"><div class="fn"><div class="footnote-content">Edited by Dr. Sachiko Isobe</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 24, Issue 6, 1 December 2017, Pages 635–648, <a href='https://doi.org/10.1093/dnares/dsx031'>https://doi.org/10.1093/dnares/dsx031</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">19 July 2017</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">26 December 2016</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">06 July 2017</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="3979487" href="/dnaresearch/article-pdf/24/6/635/22295829/dsx031.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Zihong Ye, Yao Pan, Yafen Zhang, Haifeng Cui, Gulei Jin, Alice C McHardy, Longjiang Fan, Xiaoping Yu;  Comparative whole-genome analysis reveals artificial selection effects on <em>Ustilago esculenta</em> genome, <em>DNA Research</em>, Volume 24, Issue 6, 1 December 2017, Pages 635–648, <a href='https://doi.org/10.1093/dnares/dsx031'>https://doi.org/10.1093/dnares/dsx031</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/3979487?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/3979487?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/3979487?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/3979487?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/3979487?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/3979487?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/3979487?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Comparative%20whole-genome%20analysis%20reveals%20artificial%20selection%20effects%20on%20%26lt%3Bem%26gt%3BUstilago%20esculenta%26lt%3B%2Fem%26gt%3B%20genome&amp;publicationDate=2017-07-19&amp;volumeNum=24&amp;issueNum=6&amp;author=Ye%2C%20Zihong%3B%20Pan%2C%20Yao&amp;startPage=635&amp;endPage=648&amp;contentId=10.1093%2Fdnares%2Fdsx031&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f6%2f635%2f3979487" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=104086693 id="104086693" class="abstract-title" >Abstract</h2>
<section class="abstract"><p><em>Ustilago esculenta</em>, infects <em>Zizania latifolia</em>, and induced host stem swollen to be a popular vegetable called <em>Jiaobai</em> in China. It is the long-standing artificial selection that maximizes the occurrence of favourable <em>Jiaobai</em>, and thus maintaining the plant–fungi interaction and modulating the fungus evolving from plant pathogen to entophyte. In this study, whole genome of <em>U. esculenta</em> was sequenced and transcriptomes of the fungi and its host were analysed. The 20.2 Mb <em>U. esculenta</em> draft genome of 6,654 predicted genes including mating, primary metabolism, secreted proteins, shared a high similarity to related Smut fungi. But <em>U. esculenta</em> prefers RNA silencing not repeat-induced point in defence and has more introns per gene, indicating relatively slow evolution rate. The fungus also lacks some genes in amino acid biosynthesis pathway which were filled by up-regulated host genes and developed distinct amino acid response mechanism to balance the infection–resistance interaction. Besides, <em>U. esculenta</em> lost some surface sensors, important virulence factors and host range-related effectors to maintain the economic endophytic life. The elucidation of the <em>U. esculenta</em> genomic information as well as expression profiles can not only contribute to more comprehensive insights into the molecular mechanism underlying artificial selection but also into smut fungi–host interactions.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Ustilago esculenta&quot;"><em>Ustilago esculenta</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;whole-genome analysis&quot;">whole-genome analysis</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;fungi–host interaction&quot;">fungi–host interaction</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Zizania latifolia&quot;"><em>Zizania latifolia</em></a></div>                            <h2 scrollto-destination=104086695 id="104086695" class="section-title" >1. Introduction</h2>
<p><em>Ustilago esculenta</em> is a basidiomycete fungus, which infects ancient wild rice, <em>Zizania latifolia</em>.<a reveal-id="dsx031-B1" data-open="dsx031-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a><sup>,</sup><em>Zizania latifolia</em> belongs to the tribe Oryzeae within the Gramineae<a reveal-id="dsx031-B2" data-open="dsx031-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> and was one of the six most important cereal crops in ancient China.<a reveal-id="dsx031-B3" data-open="dsx031-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> After infection with <em>U. esculenta</em> around 2000 yrs ago,<a reveal-id="dsx031-B4" data-open="dsx031-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a><sup>,</sup><em>Z. latifolia</em> formed a shuttle-like gall and was gradually domesticated as an aquatic vegetable called <em>Jiaobai</em> in East and Southeast Asia<a reveal-id="dsx031-B5" data-open="dsx031-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> particularly in China, which has antioxidant properties and may prevent hypertension, cardiovascular disease.<a reveal-id="dsx031-B6" data-open="dsx031-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a> Similar to other crops domestication, <em>Jiaobai</em> plants were artificially selected to increase their suitability to human requirements: taste, yield, storage and cultivation practices. In some unfavourable conditions, like high-dose radiation, water deficiency, unsuitable application of fungicide or unfavourable growing temperature, or if the plants were abandoned, the plants may produce galls being full of dark teliospores (called grey <em>Jiaobai</em>) or escape from fungal infection (called male <em>Jiaobai</em>).<a reveal-id="dsx031-B7" data-open="dsx031-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a><sup>,</sup><a reveal-id="dsx031-B8" data-open="dsx031-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> It was the long-term artificial selection that effectively maximized the occurrence of favourable <em>Jiaobai</em> and enabled <em>U. esculenta</em> consecutively to be maintained in host plants. Since <em>U. esculenta</em> was the cause of the vegetable, the process of crop domestication actually was the process of artificially selection on interaction complex of fungus and host. A key question, then, is to understand how domestication has shaped <em>U. esculenta</em> from the plant pathogen to entophyte and how, in turn, these changes may influence the interaction between the fungus and the host.</p><p>It has been shown that host domestication has driven the emergence of the rice blast pathogen <em>Magnaporthe oryzae</em><a reveal-id="dsx031-B9" data-open="dsx031-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> and the wheat pathogen <em>Zymoseptoria tritici</em>.<a reveal-id="dsx031-B10" data-open="dsx031-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> In contrast to natural evolution, domestication is due to humans as selective agents; but, just like natural selection, domestication fully depends on genetic variation, mutations, inheritance and demography.<a reveal-id="dsx031-B11" data-open="dsx031-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> The emerging consensus is that domestication causes a distinct imprint on genomes.<a reveal-id="dsx031-B12" data-open="dsx031-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> Integrated and fully analyses of the genome will give insight into the underlying effect of domestication. The comparative genomic study on <em>Mycosphaerella graminicola</em> and its wild sister species showed that speciation of <em>M. graminicola</em> was associated with adaptation to domesticated wheat and its associated agro-ecosystem.<a reveal-id="dsx031-B13" data-open="dsx031-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> And indeed, comparative genomics has revealed underlying mechanisms of genome evolution, for example, transposon-mediated gene loss driven the rapid evolution of <em>M. oryzae</em>,<a reveal-id="dsx031-B14" data-open="dsx031-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a><sup>,</sup><a reveal-id="dsx031-B15" data-open="dsx031-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> and gene loss rather than gene gain resulted in the host jump in the smut fungus <em>Melanopsichium pennsylvanicum</em>.<a reveal-id="dsx031-B16" data-open="dsx031-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a></p><p>Smut fungi are biotrophic pathogens causing characteristic sympotoms, the replacement of plant organs by black masses of teliospores, in a number of agriculturally important crop plants, mostly grass family (Grmineae). These basidiomycete fungi belong to the order Ustilaginales containing over 50 genera. Several different smut diseases in cereal crops are mostly caused not only by species of the genus Ustilago but also by other genera such as <em>Sporisorium</em>.<a reveal-id="dsx031-B17" data-open="dsx031-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a><sup>,</sup><em>U</em><em>stilago esculenta</em> together with <em>Ustilago maydis</em>, <em>Ustilago hordei</em> and <em>Sporisorium reilianum</em> are belonging to Ustilaginaceae family but owning some distinct characteristics. For instance, <em>U. esculenta</em> spend its entire life cycle in the host plant (<a reveal-id="dsx031-F1" data-open="dsx031-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>),<a reveal-id="dsx031-B3" data-open="dsx031-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a><sup>,</sup><a reveal-id="dsx031-B18" data-open="dsx031-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> while most smut fungi infect and colonize the plant with the dikaryotic mycelium and are released from the host after spore formation. There has been found that the phytopathogenic fungi developed many infection mechanisms to be successfully colonized, including surface sensors like Sho1 and Msb2,<a reveal-id="dsx031-B19" data-open="dsx031-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> intracellular signalling cascades like Cyclic Adenosine monophosphate (cAMP) and mitogen-activated protein kinase (MAPK),<a reveal-id="dsx031-B20" data-open="dsx031-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> secreted proteins to penetrate host plant and establish biotrophy<a reveal-id="dsx031-B21" data-open="dsx031-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> and transcription factors acting as key regulators of differentiation.<a reveal-id="dsx031-B22" data-open="dsx031-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> Besides, for dimorphic fungi, mating reaction and filament formation are essentials of the morphogenetic switch, which should be prepared already before penetration. Although for <em>U. esculenta</em>, there might be some special characters considering its unique entophytic lifecycle evolving from a pathogen. In addition, <em>U. esculenta</em> inhibits host inflorescence formation and cause stem enlargement, and <em>Z. latifolia</em> is its only known host<a reveal-id="dsx031-B1" data-open="dsx031-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> until now. Although many smut fungi are known to be non-host and -tissue specificity. For example, <em>U. maydis</em> cause symptom at any aboveground parts of host plant, which is coupled to secreted protein effectors<a reveal-id="dsx031-B23" data-open="dsx031-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> and their organ-specific expression.<a reveal-id="dsx031-B24" data-open="dsx031-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a><sup>,</sup><a reveal-id="dsx031-B25" data-open="dsx031-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a><sup>,</sup><em>Ustilago hordei</em> and <em>S. reilianum</em> cause the grain, inflorescence develop into smut sori, respectively.<a reveal-id="dsx031-B25" data-open="dsx031-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> Covered smut of barley and oats is caused by <em>U. hordei</em>, and head smut caused by <em>S</em>. <em>reilianum</em> is found in both maize and sorghum. Differentiated effectors being targeting different host molecules were responsible for varied infecting strategy of <em>U. maydis</em> and <em>S. reilianum</em> even they parasitized the same host.<a reveal-id="dsx031-B26" data-open="dsx031-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> And a larger repeat content at important loci, including mating-type and effector loci, were identified for <em>U. hordei</em> to explain varied genome evolution on its host when comparing with <em>U. maydis</em> and <em>S. reilianum</em>.<a reveal-id="dsx031-B26" data-open="dsx031-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a></p>                        <a id="104086699" scrollto-destination="104086699"></a>
<div id="dsx031-F1" data-id="dsx031-F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/m_dsx031f1.png?Expires=2147483647&Signature=Xoeqzb0hRR8cDa3kkTNu~v0HzH1ugcB~AwXPOeEfiTmQHuGBPf9sVrQJ7lXKJ-Y~hPhCqfJemm67Y2Z9wgy8mQRVJALyllyGYH5TkuQe4px4yxoDiQfSvwtUqSbSPeMD3vHfHmCMAjMcvxXTuMpzsz8pgg8PE0KGD79BrUd3jMMrxBXeYxIDz6QiJ6LHwBCpJ4lMuHcUi4ucs8tqMoBxIN7oBZgk3acTU39jaMLo~8ZuYBxdNOwYXGP98gPKlkId2PqyEyi8gCg~xXkcxXbP~MYhhzcOLaSMOoQAx32MMx4WLttaPNJH45RztrXs~fGByr0MCS46stBcPfIfsPrz9Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Life cycle of U. esculenta in Z. latifolia. (A) The fungus overwinters as hypha and teliospores in the rhizomes of host plant. (B) Young seedling sprouts from rhizomes. (C) Hypha invades young seedlings and inflorescence prevented in the infected plant. (D) Young seedling failed to be invaded and becomes male Jiaobai with normal inflorescence. (E) The fungus incites gall development at the basal stem. (F) Stems are full of black teliospores and called grey Jiaobai. (G) Morphology of fungus isolated from grey Jiaobai, which was T sample, observed from liquid culture (Down) and by scanning electron microscope (SEM) (Up). (H) Stems of edible Jiaobai. (I) Morphology of fungus isolated from edible Jiaobai, which was M sample, observed from liquid culture (Down) and by SEM (Up). (J) Morphology of fungus isolated from edible Jiaobai, which was MT sample, observed from liquid culture (Down) and by SEM (Up)." path-from-xml="dsx031f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086699/dsx031f1.tif" path-from-xml="dsx031f1.tif" target="_blank">View large</a><a section="104086699" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/dsx031f1.png?Expires=2147483647&Signature=TVOB5PN~xnZ1~Vtz2kB5icv8gQV4o4m1BJuGNYMte8jVbyVAiQwMlWEQ8VPrOgA-U3L5Ikcj4i~yGyus7djJGOuumFsOLJo~7uQkelTVtS5bwqZL45PSiHxvfveFLY9G0njLrE5xWalXhhd6hXXdnWlPVyjD5lgBRWm29KR5TBSO2Rj2j4Y54eQMZHu0lgwZDQ7Lcr6H6B4CSoHEGppJDXG66H1o7c2LsDsFzZmnzOVFlsa9YIPK4Tkfc3hoiG1DVry620tgKJOWQ66rQO9YfdMbe54yKDWspaTAdPg4ZqA9NhTaqYeZ0EVOi0qbrhZ3hOYFMm1N3lqFVAyHnsjWRQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086699&ar=3979487&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx031f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Life cycle of <em>U. esculenta</em> in <em>Z. latifolia</em>. (A) The fungus overwinters as hypha and teliospores in the rhizomes of host plant. (B) Young seedling sprouts from rhizomes. (C) Hypha invades young seedlings and inflorescence prevented in the infected plant. (D) Young seedling failed to be invaded and becomes male <em>Jiaobai</em> with normal inflorescence. (E) The fungus incites gall development at the basal stem. (F) Stems are full of black teliospores and called grey <em>Jiaobai</em>. (G) Morphology of fungus isolated from grey <em>Jiaobai</em>, which was T sample, observed from liquid culture (Down) and by scanning electron microscope (SEM) (Up). (H) Stems of edible <em>Jiaobai</em>. (I) Morphology of fungus isolated from edible <em>Jiaobai</em>, which was M sample, observed from liquid culture (Down) and by SEM (Up). (J) Morphology of fungus isolated from edible <em>Jiaobai</em>, which was MT sample, observed from liquid culture (Down) and by SEM (Up).</p></div></div><div content-id="dsx031-F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/m_dsx031f1.png?Expires=2147483647&Signature=Xoeqzb0hRR8cDa3kkTNu~v0HzH1ugcB~AwXPOeEfiTmQHuGBPf9sVrQJ7lXKJ-Y~hPhCqfJemm67Y2Z9wgy8mQRVJALyllyGYH5TkuQe4px4yxoDiQfSvwtUqSbSPeMD3vHfHmCMAjMcvxXTuMpzsz8pgg8PE0KGD79BrUd3jMMrxBXeYxIDz6QiJ6LHwBCpJ4lMuHcUi4ucs8tqMoBxIN7oBZgk3acTU39jaMLo~8ZuYBxdNOwYXGP98gPKlkId2PqyEyi8gCg~xXkcxXbP~MYhhzcOLaSMOoQAx32MMx4WLttaPNJH45RztrXs~fGByr0MCS46stBcPfIfsPrz9Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Life cycle of U. esculenta in Z. latifolia. (A) The fungus overwinters as hypha and teliospores in the rhizomes of host plant. (B) Young seedling sprouts from rhizomes. (C) Hypha invades young seedlings and inflorescence prevented in the infected plant. (D) Young seedling failed to be invaded and becomes male Jiaobai with normal inflorescence. (E) The fungus incites gall development at the basal stem. (F) Stems are full of black teliospores and called grey Jiaobai. (G) Morphology of fungus isolated from grey Jiaobai, which was T sample, observed from liquid culture (Down) and by scanning electron microscope (SEM) (Up). (H) Stems of edible Jiaobai. (I) Morphology of fungus isolated from edible Jiaobai, which was M sample, observed from liquid culture (Down) and by SEM (Up). (J) Morphology of fungus isolated from edible Jiaobai, which was MT sample, observed from liquid culture (Down) and by SEM (Up)." path-from-xml="dsx031f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086699/dsx031f1.tif" path-from-xml="dsx031f1.tif" target="_blank">View large</a><a section="104086699" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/dsx031f1.png?Expires=2147483647&Signature=TVOB5PN~xnZ1~Vtz2kB5icv8gQV4o4m1BJuGNYMte8jVbyVAiQwMlWEQ8VPrOgA-U3L5Ikcj4i~yGyus7djJGOuumFsOLJo~7uQkelTVtS5bwqZL45PSiHxvfveFLY9G0njLrE5xWalXhhd6hXXdnWlPVyjD5lgBRWm29KR5TBSO2Rj2j4Y54eQMZHu0lgwZDQ7Lcr6H6B4CSoHEGppJDXG66H1o7c2LsDsFzZmnzOVFlsa9YIPK4Tkfc3hoiG1DVry620tgKJOWQ66rQO9YfdMbe54yKDWspaTAdPg4ZqA9NhTaqYeZ0EVOi0qbrhZ3hOYFMm1N3lqFVAyHnsjWRQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086699&ar=3979487&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx031f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Life cycle of <em>U. esculenta</em> in <em>Z. latifolia</em>. (A) The fungus overwinters as hypha and teliospores in the rhizomes of host plant. (B) Young seedling sprouts from rhizomes. (C) Hypha invades young seedlings and inflorescence prevented in the infected plant. (D) Young seedling failed to be invaded and becomes male <em>Jiaobai</em> with normal inflorescence. (E) The fungus incites gall development at the basal stem. (F) Stems are full of black teliospores and called grey <em>Jiaobai</em>. (G) Morphology of fungus isolated from grey <em>Jiaobai</em>, which was T sample, observed from liquid culture (Down) and by scanning electron microscope (SEM) (Up). (H) Stems of edible <em>Jiaobai</em>. (I) Morphology of fungus isolated from edible <em>Jiaobai</em>, which was M sample, observed from liquid culture (Down) and by SEM (Up). (J) Morphology of fungus isolated from edible <em>Jiaobai</em>, which was MT sample, observed from liquid culture (Down) and by SEM (Up).</p></div></div><p>The availability of full genome sequences of over 50 basidiomycetes, including <em>U. maydis</em>, <em>U. hordei</em> and <em>S. reilianum</em> has accelerated research into basidiomycete genomics. <em>U</em><em>stilago esculenta</em>, which evolved as the result of crop domestication from plant pathogen to entophyte, offers the possibility to address several major questions in plant pathogen evolution. These include what general changes can be observed in genomes after a long-term domestication? To what extent are pathogenicity and the associated effectors affected by the crop domestication? What helps the smut fungi to sustain the suitable growth condition in the host with delayed teliospores formation? To address these questions, we determine the genome sequence of <em>U. esculenta</em>, highlight genomic comparison among <em>U. esculenta</em>, <em>U. maydis</em>, <em>U. hordei</em> and <em>S. reilianum</em> and perform transcriptome analysis to characterize the different stages of the <em>U. esculenta</em> life cycle in association with phenotype of host plant.</p>                            <h2 scrollto-destination=104086701 id="104086701" class="section-title" >2. Materials and methods</h2>
                            <h3 scrollto-destination=104086702 id="104086702" class="section-title" >2.1. Strains and growth conditions</h3>
<p><em>U</em><em>stilago esculenta</em> strain was isolated from edible <em>Jiaobai</em> of Longjiao No 2 in 2011 in Zhejiang province of China. Longjiao No 2 is one of the most widely cultivated <em>Jiaobai</em> varieties in east China. The galls were harvested at 160 days after planting. The <em>U. esculenta</em> strain was isolated from non-sporulating gall by slice separation method and cultured on potato dextrose agar (PDA) medium. After 6–10 days, the white mycelia that grew out from the tissue were transferred to potato dextrose broth (PDB) medium. The mycelia (M) for further analysis were collected after shaking in PDB medium for 3 days at 28°C. Teliospores were isolated from sporulating gall of grey <em>Jiaobai</em> and old gall of edible <em>Jiaobai</em> and cultured on PDA medium. After 3–5 days, haploid strains were obtained by single sporidia selection with capillary pipet under microscopy and transferred to PDB medium for additional 3 days shake-culture at 28°C. Enriched sporidia were collected for further analysis. Haploid strains from edible <em>Jiaobai</em> called MT strain with deposited number CGMCC Nos. 11841 and11842 in China Center for Type Culture Collection, haploid strains from grey <em>Jiaobai</em> called T strain with deposited number CGMCC Nos. 11843 and11844.</p>                            <h3 scrollto-destination=104086704 id="104086704" class="section-title" >2.2. Mating assay</h3>
<p>Two haploid strains with heterogametic types from <em>U. esculenta</em> were streak cultured on YEPS solid medium (1% yeast extract, 2% peptone, 2% sucrose and 1.5% agar) separately, at 28°C. After 3 days, single colonies were picked into YEPS medium for further culture (28°C and 180 r.p.m). The sporidia cells were collected when OD<sub>600</sub> reached 1 and re-suspended to OD<sub>600</sub> of 1.5. The two cultures were mixed 1:1 Vol/Vol and spotted onto YEPS solid medium and cultured at 28°C. Microscopic observation and PCR confirmation of the mating loci were performed 4 days after incubation.</p>                            <h3 scrollto-destination=104086706 id="104086706" class="section-title" >2.3. Effect of amino acids on mating and hyphal growth</h3>
<p>Mating assays were carried out and the mixed culture of two mating strains were spotted onto the basic medium (K<sub>2</sub>HPO<sub>4</sub> 1 g/l, MgSO<sub>4</sub>·7H<sub>2</sub>O 0.5 g/l, FeSO<sub>4</sub>·7H<sub>2</sub>O 0.01 g/l, KCl 0.5 g/l, glucose and agar 15 g/l) supplement with different amino acids (arginine, histidine, isoluceine, leucine, lysine, methionine, phenylalanine, proline, tyrosine or valine) as nitrogen source (20 mM) or as trace elements (0.2 mM) when KNO<sub>3</sub> as nitrogen source (20 mM/l). With an interval 12 h, mating colonies were mounted for microscopy and after 3 days, hyphal length were measured.</p>                            <h3 scrollto-destination=104086708 id="104086708" class="section-title" >2.4. Effect of arginine on strain T and MT hyphal growth</h3>
<p>Two haploid T strains and MT strains with heterogametic types were cultured and mixed as described in ‘Mating assay’ in method. The 3 µl 1:1 Vol/Vol mixed cultures were inoculated on YEPS solid medium and Arg medium (K<sub>2</sub>HPO<sub>4</sub> 1 g/l, MgSO<sub>4</sub>·7H<sub>2</sub>O 0.5 g/l, FeSO<sub>4</sub>·7H<sub>2</sub>O 0.01 g/l, KCl 0.5 g/l, Sucrose 68 g/l and arginine 13.9 g/l) individually. The cultures were incubated at 28°C. With an interval 12 h, hyphal length was measured. The measurement was repeated for three times.</p>                            <h3 scrollto-destination=104086710 id="104086710" class="section-title" >2.5. DNA, RNA preparation, genome sequencing and assembly</h3>
<p>Mycelia (M), sporidia from grey <em>Jiaobai</em> (T) and from edible <em>Jiaobai</em> (MT) were used to isolate DNA and RNA. All the materials were washed with sterile water for three times and centrifuged. After grounding in liquid nitrogen, genomic DNA was extracted using the CTAB method<a reveal-id="dsx031-B18" data-open="dsx031-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> and total RNA was extracted using Trizol (Invitrogen). The isolated RNA was then treated by RNAse-Free DNase and then subsequently treated using Illumina mRNA-Seq Prep Kit (Illumina, San Diego, CA) following the manufacturer’s instruction. Four DNA sequencing libraries with different insert lengths (170 bp, 500 bp, 6 Kb and 10 Kb) and 7.7 Gb raw sequence data (<span class="content-section supplementary-material"><a path-from-xml="sup15" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/supplementary_table_s1_dsx031.doc?Expires=2147483647&amp;Signature=uX36CZpPdsgp6fSq8CDrrqx7RrWCL9SinAkAkiNn7-LXhC7bq2ZWM7-uBYY78rMMTSiV85jy3n67040CYeXGQZSAGsXOhk6huLmwDkec5rvXvEglZgSadOiSOU6uniGQuWPwYUAk3vyc1hel786nt-IMMykT1gtFx0~3efSPVu71sIsIQWrZdWbl6FtGp~GPCdHe9IjVEDqf9ZHJeX5W3uo3Y5zQ~KX6SEO1soDP~AQbbqxeaZqWMhSod5aihTj1T94O6aGsene3X8dIm5D9~NjpPKGo-VkiJzmHCXz~9h-1OmCSbFB7mCq8Pk5~r~CRvvvx~du8OsK2c6lawR58Ng__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>) were generated using Illumina HiSeq2000 platform (BGI, Shenzhen, China).</p><p>After removing low quality and adapter sequences, the reads were <em>de novo</em> assembled using the CLC workbench 5.5.1 (CLC bio). SSPACE BASIC (version 2.0)<a reveal-id="dsx031-B27" data-open="dsx031-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> was used for scaffold construction and GapFiller (version 1.10)<a reveal-id="dsx031-B28" data-open="dsx031-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> was applied for gap closure.</p><p>Genome heterozygosity was first analysed by K-mer analysis and then confirmed by alignment between assembled scaffolds and original reads. Genome Analysis TK (Version 1.6) was used to identify Single Nucleotide Polymorphisms (SNPs).</p><p>Genome sequences of <em>U. maydis</em> were downloaded from the Broad institute of Harvard and MIT (<a class="link link-uri" href="http://www.broadinstitute.org/annotation/genome/ustilago_maydis/" target="_blank">http://www.broadinstitute.org/annotation/genome/ustilago_maydis/</a> (6 July 2017, date last accessed)). <em>Sporisorium reilianum</em> and <em>U. hordei</em> genomic sequences were downloaded and retrieved from the Munich information center for protein sequences (<a class="link link-uri" href="http://ftp://ftpmips.gsf.de/fungi/Sporisorium_reilianum/" target="_blank">ftp://ftpmips.gsf.de/fungi/Sporisorium_reilianum/</a> and <a class="link link-uri" href="http://ftp://ftpmips.gsf.de/fungi/MUHDB/" target="_blank">ftp://ftpmips.gsf.de/fungi/MUHDB/</a>).</p>                            <h3 scrollto-destination=104086715 id="104086715" class="section-title" >2.6. Repeat annotation</h3>
<p>A <em>de novo</em> repeat database of <em>U. esculenta</em> was generated using RepeatModeler (Version 1.0.7, <a class="link link-uri" href="http://www.repeatmasker.org/" target="_blank">http://www.repeatmasker.org/</a> (6 July 2017, date last accessed)). RepeatMasker (Version 3.2.7, <a class="link link-uri" href="http://www.repeatmasker.org/" target="_blank">http://www.repeatmasker.org/</a> (6 July 2017, date last accessed)) was used to identify repeats from our <em>de novo</em> database and Repbase database (<a class="link link-uri" href="http://www.girinst.org/" target="_blank">http://www.girinst.org/</a> (6 July 2017, date last accessed)). LTR_FINDER (version 1.0.5)<a reveal-id="dsx031-B29" data-open="dsx031-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> was applied to identify the long terminal repeat (LTR) elements.</p>                            <h3 scrollto-destination=104086717 id="104086717" class="section-title" >2.7. Gene annotation</h3>
<p>We predicted genes as follows: (i) <em>de novo</em> prediction. Genes were predicted using Augustus (Version 2.03)<a reveal-id="dsx031-B30" data-open="dsx031-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> with training gene sets from the <em>U. maydis</em> genome. (ii) Homologue-based prediction. We mapped the protein sequences of <em>U. maydis</em>, <em>S. reilianum</em> and <em>U. hordei</em> to the <em>U. esculenta</em> genome using tBLASTn, with a cut-off <em>E</em>-value of 10<sup>−5</sup>. (iii) RNA-seq based prediction. All contigs from RNA-seq were mapped to the <em>U. esculenta</em> genome by TopHat (Version 2.0, <a class="link link-uri" href="http://tophat.cbcb.umd.edu/" target="_blank">http://tophat.cbcb.umd.edu/</a>). (iv) All the gene predictions were combined using GLEAN (version 1.0.1)<a reveal-id="dsx031-B31" data-open="dsx031-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> to produce consensus gene sets.</p><p>Gene functions were assigned according to the best match of the alignments using BLASTp (<em>E</em>-value &lt; 10<sup>−5</sup>)<a reveal-id="dsx031-B32" data-open="dsx031-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> searching against the nucleotide database of NCBI. The motifs and domains of genes were determined by InterproScan (Version 4.5)<a reveal-id="dsx031-B33" data-open="dsx031-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> against UniProt/Swiss-Prot protein database. All genes were classified according to Gene Ontology (GO) and KEGG (Release 48.2) pathways. If the best hit of the genes in any of these processes was ‘function unknown’, the second best hit was used to assign the function until there were no more hits that met the alignment criteria, then this gene is determined as functionally unknown. OrthoMCL (version 2.0.9)<a reveal-id="dsx031-B34" data-open="dsx031-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> was used to group <em>U. esculenta</em> genes into orthologue clusters.</p><p>Genes were also analysed for carbohydrate-active enzymes (CAZymes, <a class="link link-uri" href="http://www.cazy.org" target="_blank">http://www.cazy.org</a> (6 July 2017, date last accessed)) using dbCAN.<a reveal-id="dsx031-B35" data-open="dsx031-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> Genes related to the biosynthesis of secondary metabolites were analysed using the JVI Secondary Metabolite Unique Regions Finder Web server.<a reveal-id="dsx031-B36" data-open="dsx031-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> SignalP (version 3.0)<a reveal-id="dsx031-B37" data-open="dsx031-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> was used for the prediction of secreted proteins. The candidates were analysed with ProtComp 6.0 (<a class="link link-uri" href="http://www.softberry.com" target="_blank">http://www.softberry.com</a> (6 July 2017, date last accessed)) and TargetP (version 1.1)<a reveal-id="dsx031-B38" data-open="dsx031-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> were used to predict protein localization. Information regarding the effects of virulence was retrieved from PHI-base (<a class="link link-uri" href="http://www.phi-base.org/" target="_blank">http://www.phi-base.org/</a> (6 July 2017, date last accessed)).</p><p>The tRNA genes were identified by tRNAScan-SE (version 1.21).<a reveal-id="dsx031-B39" data-open="dsx031-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> For rRNAs identification, the rRNAs from <em>U. maydis</em> were aligned against the <em>U. esculenta</em> genome using BLASTn (<em>E</em>-value &lt; 10<sup>−5</sup>) to identify possible rRNAs. Other non-coding RNAs, including miRNA and snRNA, were identified using INFERNAL (version 1.1)<a reveal-id="dsx031-B40" data-open="dsx031-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> by searching against the Rfam database.<a reveal-id="dsx031-B41" data-open="dsx031-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a></p><p>Genome syntenic analysis of smut fungi were performed with MUMmer (version 3.0),<a reveal-id="dsx031-B42" data-open="dsx031-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a> the schematic representation of the genome synteny was generated with CIRCOS (<a class="link link-uri" href="http://mkweb.bcgsc.ca/circos/" target="_blank">http://mkweb.bcgsc.ca/circos/</a>).</p>                            <h3 scrollto-destination=104086723 id="104086723" class="section-title" >2.8. Transcriptome sequencing and analysis</h3>
<p>Mycelia (M) and sporidia (MT) of MT strain and sporidia (T) of T strain were collected for transcriptome sequencing. Three samples of same culture were collected for sequencing company to guarantee the quality of RNA extracting and sequencing. The transcriptomes were sequenced using Illumina HiSeq2000 platform (BGI, Shenzhen, China). The reads from different sequenced samples (M, T and MT) were mapped to the whole-genome assembly using SOAP2.<a reveal-id="dsx031-B43" data-open="dsx031-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a> The expression quantification was calculated based on RPKM measurement.<a reveal-id="dsx031-B44" data-open="dsx031-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a> Only if the gene’s expression profile differs between two samples with a fold change &gt; 2 and a <em>P</em>-value &lt; 0.01 by edgeR,<a reveal-id="dsx031-B45" data-open="dsx031-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a> with estimateDispersions method as ‘blind’, this gene was considered to be significantly differentially expressed. The transcriptomic data have been deposited at GenBank under accession SRR1611140, SRR5234219 and SRR5234220 for M, T and MT, respectively. To understand the effect of long-term interaction between the host plant and <em>U. esculenta</em>, the transcriptomic data of stems from edible <em>Jiaobai</em> with <em>U. esculenta</em> in the plant and wild <em>Z. latifolia</em> (GenBank accession number, PRJNA187578)<a reveal-id="dsx031-B46" data-open="dsx031-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a> were downloaded and used.</p>                            <h3 scrollto-destination=104086725 id="104086725" class="section-title" >2.9. Phylogenetic analysis</h3>
<p>The phylogenetic relationships of <em>U. esculenta</em>, <em>U. maydis</em>, <em>U. hordei</em>, <em>S. reilianum</em> and <em>Cryptococcus neoformans</em> were constructed using 45 single copy gene families (<span class="content-section supplementary-material"><a path-from-xml="sup5" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/supplementary_file_s3_dsx031.xls?Expires=2147483647&Signature=QDBuwo~19ELM4sp~jq1TS0FSeZbHZm2UNm9AQWpE5eUaZhdgQC61vZnJ2gqLZ5OMjjst2nTaRkFoXu~O1l85mqJnMNUB5mNbqPie5wkYXsFjoisjSt4qtvVNqoSya-BbOtk37AeiIKotmyPazPjw-1BWwY29n0wTvisUszZj91DjqtzlaxGbPMuFuUQrfKUpk6aashvIDyAhdifYkTrSm2-gmukok~Pi1uBY70wNWaDSP0Vhb7EOQ6mrOjIbicT~mPVwziRH3YT9o5pJHOZvcaqNf7szHO0ceyBiE6RxJH5Y1eFrZZO~LjLkUGFTSIyJWl7XaNbd6Qxc8gENgQMLrg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary File S3</a></span>) by OrthoMCL. The alignments were performed using Clustal omega (version 1.2.0)<a reveal-id="dsx031-B47" data-open="dsx031-B47" class="link link-ref link-reveal xref-bibr"><sup>47</sup></a> with default parameters. One hundred bootstrap replicates were generated for each gene family alignment using the Seqboot package in PHYLIP (version 3.69, <a class="link link-uri" href="http://evolution.genetics.washington.edu/phylip/" target="_blank">http://evolution.genetics.washington.edu/phylip/</a>). For each replicate, a maximum likelyhood phylogenetic tree was constructed using Promlk package of PHYLIP under Joes–Taylor–Thornton model with testing of a molecular clock. The consensus tree was built based on 4,500 trees from the bootstrap experiments by the Consense package in PHYLIP with an extended majority rule. The percentage of support by the 4,500 individual trees was indicated on each branch of the consensus tree.</p>                            <h2 scrollto-destination=104086727 id="104086727" class="section-title" >3. Results/discussion</h2>
                            <h3 scrollto-destination=104086728 id="104086728" class="section-title" >3.1. Sequencing of the <em>U. esculenta</em> genome</h3>
<p>The collected mycelia of <em>U. esculenta</em> were sequenced using Illumina Higseq2000 platform. Four different insert libraries (170 bp–10 kb) were sequenced to generate a total of 4.54 Mb trimmed data with a genome coverage of ×139 (<span class="content-section supplementary-material"><a path-from-xml="sup15" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/supplementary_table_s1_dsx031.doc?Expires=2147483647&amp;Signature=uX36CZpPdsgp6fSq8CDrrqx7RrWCL9SinAkAkiNn7-LXhC7bq2ZWM7-uBYY78rMMTSiV85jy3n67040CYeXGQZSAGsXOhk6huLmwDkec5rvXvEglZgSadOiSOU6uniGQuWPwYUAk3vyc1hel786nt-IMMykT1gtFx0~3efSPVu71sIsIQWrZdWbl6FtGp~GPCdHe9IjVEDqf9ZHJeX5W3uo3Y5zQ~KX6SEO1soDP~AQbbqxeaZqWMhSod5aihTj1T94O6aGsene3X8dIm5D9~NjpPKGo-VkiJzmHCXz~9h-1OmCSbFB7mCq8Pk5~r~CRvvvx~du8OsK2c6lawR58Ng__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). The reads were <em>de novo</em> assembled into 1,869 contigs with a total size of 20.2 Mb (<a reveal-id="dsx031-T1" data-open="dsx031-T1" class="link link-reveal link-table xref-fig">Table 1</a>). Since the strain was diploid, we checked the heterozygosity by k-mer (<a reveal-id="dsx031-F2" data-open="dsx031-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>) and SNP analysis (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/supplementary_file_s1_dsx031.xls?Expires=2147483647&amp;Signature=LOnchKUZsFz9Wa7rvxMFb5TC5rLpt9pvMMfI8juF5HER~ivV2jeP3unCJ2D8t-OmLfjW1lwRPMjipds26lxPgP5YRFf9xeDIrqsnfEkwhSJMSFrkDmdkHFbRS74enuy8BhQghp2JkihawbZTgoSB84whiqjKQ4DgrX--9sThfQQfil5kGonaNMkDwka18e78JV-W3mhsfpYQ80V9cR88ohKWf6bP-L6heIMAD0BGLaemKJOJjsb4gV84pcQ0JSEYtfVEVlNBYdzzVEvj01oaBtnLpqKaNtAVwCzvhrZyrPCcrdCsyjoVc22urF2NlEtJbh~6Fg-uDy6zsn5r78K4XQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary File S1</a></span>). Although some random sequencing errors exist, only one distribution was found by K-mers with volume peak at 85, indicating the high homozygosity of genome. To confirm the result, we aligned the assembled scaffolds with original reads. Totally, 2,118 SNPs including 228 homologous SNPs were found. Such a low SNP frequency and density further validated the genome homozygosity. The N50 lengths of contigs and scaffolds of the assembly were 101.8 and 404.8 kb, respectively. The assembled draft genome size of <em>U. esculenta</em> is moderately larger than <em>U. maydis</em> and <em>S. reilianum</em> but slightly smaller than <em>U. hordei</em>. In total, 6,654 candidates gene were identified based on homology to known proteins and the transcriptomic data (detailed annotations see <span class="content-section supplementary-material"><a path-from-xml="sup4" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/supplementary_file_s2_dsx031.xls?Expires=2147483647&amp;Signature=C3nu5QEfgdRa~jCnGIVFbNCudwHZzuQ7Db-iD5Bi9-v5EMf2lI9s1NMQGFQV-NKFiySF1jNDyb5AYBxYTvDDPgk4SIaoWZOezeO2qqk81f5AjhnHwP8OQ4H9b-i7BJ1Cm~Uxw47idENq2jeN5CSL1curf~q3d0lzGcdk~wR3-E6dfpPP4T~TT2a-6KDrxBHJNROlKLDT8r1Uc7GSW9aAFgCi~AXUKpCXgOn7qC4m-4rQ4L-d7E2CtP~tPxxfr2EzkmkwJOZiSiHw~QC8gH1hqHiYv7glLevmTtBgJ7xsnipcOtTmbhpijvMzyZup~PHgXr2Tc-9Vxw-Z2XQV2K2sXg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary File S2</a></span>), indicating that <em>U. esculenta</em> has similar proportion of protein-coding regions as the other three smut fungi. The proportion of genes encoding secreted proteins was around 10% (663 out of 6,654 genes) of the annotated genes. This whole genome shotgun project has been deposited at DDBJ/EMBL/GenBank under the accession JTLW00000000. The version described in this article is version JTLW01000000. <div content-id="dsx031-T1" class="table-modal"><div class="table-wrap"><div id="dsx031-T1" data-id="dsx031-T1" class="table-wrap-title"><div class="label -label">Table 1</div><div class="caption"><p>The genome statistics of <em>U. esculenta</em> compared with other sequenced smut fungi</p></div></div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Genome specifics </th><th><em>U. esculenta</em> </th><th><em>U. hordei</em> </th><th><em>U. maydis</em> </th><th><em>S. reilianum</em> </th></tr></thead><tbody><tr><td colspan="5">Assembly statistics </td></tr><tr><td>  Total contig length (Mb) </td><td>20.2 </td><td>20.7 </td><td>19.7 </td><td>18.2 </td></tr><tr><td>  Total scaffold length (Mb) </td><td>21.4 </td><td>21.2 </td><td>19.8 </td><td>18.4 </td></tr><tr><td>  Average base coverage </td><td>×100 </td><td>×20 </td><td>×10 </td><td>×20 </td></tr><tr><td>  N50 contig (kb) </td><td>101.8 </td><td>48.7 </td><td>127.4 </td><td>50.3 </td></tr><tr><td>  N50 scaffold (kb) </td><td>404.8 </td><td>307.7 </td><td>817.8 </td><td>738.5 </td></tr><tr><td>  Chromosome no. </td><td> </td><td>23 </td><td>23 </td><td>23 </td></tr><tr><td>  GC content (%) </td><td>51.3 </td><td>52 </td><td>54 </td><td>59.7 </td></tr><tr><td>  Coding </td><td>56.7 </td><td>54.3 </td><td>56.3 </td><td>62.6 </td></tr><tr><td>  Non-coding </td><td>44.2 </td><td>43.4 </td><td>50.5 </td><td>54.3 </td></tr><tr><td colspan="5">Gene annotation </td></tr><tr><td>  Percentage coding (%) </td><td>61 </td><td>57.5 </td><td>61.1 </td><td>65.9 </td></tr><tr><td>  Average gene size (bp) </td><td>1,964 </td><td>1,708 </td><td>1,836 </td><td>1,858 </td></tr><tr><td>  Average gene density (kb/gene) </td><td>3.2 </td><td>3.0 </td><td>2.9 </td><td>2.8 </td></tr><tr><td>  Protein-coding genes </td><td>6,655 </td><td>7,113 </td><td>6,786 </td><td>6,648 </td></tr><tr><td>  Of homologous proteins (%) </td><td>6,045 (91) </td><td>6,325 (89) </td><td>6,214 (92) </td><td>6,394 (96) </td></tr><tr><td>  Exons </td><td>12,237 </td><td>10,907 </td><td>9,783 </td><td>9,776 </td></tr><tr><td>  Exons/gene </td><td>1.84 </td><td>1.53 </td><td>1.44 </td><td>1.47 </td></tr><tr><td>  Average exon size (bp) </td><td>994 </td><td>1,107 </td><td>1,230 </td><td>1,221 </td></tr><tr><td>  Intron </td><td>5,563 </td><td>3,161 </td><td>2,997 </td><td>3,103 </td></tr><tr><td>  Introns/gene </td><td>0.84 </td><td>0.44 </td><td>0.44 </td><td>0.46 </td></tr><tr><td>  Average intron size (bp) </td><td>185 </td><td>141 </td><td>142 </td><td>144 </td></tr><tr><td>  Average intergenic distance (bp) </td><td>1,052 </td><td>1,186 </td><td>1,127 </td><td>929 </td></tr><tr><td>  tRNA genes </td><td>152 </td><td>110 </td><td>111 </td><td>96 </td></tr><tr><td>  TEs and repeats (%) </td><td>7.5 </td><td>2.0 </td><td>8.0 </td><td>0.8 </td></tr></tbody></table></div><div content-id="dsx031-T1" class="table-modal"><table><thead name="thead" align=""><tr><th>Genome specifics </th><th><em>U. esculenta</em> </th><th><em>U. hordei</em> </th><th><em>U. maydis</em> </th><th><em>S. reilianum</em> </th></tr></thead><tbody><tr><td colspan="5">Assembly statistics </td></tr><tr><td>  Total contig length (Mb) </td><td>20.2 </td><td>20.7 </td><td>19.7 </td><td>18.2 </td></tr><tr><td>  Total scaffold length (Mb) </td><td>21.4 </td><td>21.2 </td><td>19.8 </td><td>18.4 </td></tr><tr><td>  Average base coverage </td><td>×100 </td><td>×20 </td><td>×10 </td><td>×20 </td></tr><tr><td>  N50 contig (kb) </td><td>101.8 </td><td>48.7 </td><td>127.4 </td><td>50.3 </td></tr><tr><td>  N50 scaffold (kb) </td><td>404.8 </td><td>307.7 </td><td>817.8 </td><td>738.5 </td></tr><tr><td>  Chromosome no. </td><td> </td><td>23 </td><td>23 </td><td>23 </td></tr><tr><td>  GC content (%) </td><td>51.3 </td><td>52 </td><td>54 </td><td>59.7 </td></tr><tr><td>  Coding </td><td>56.7 </td><td>54.3 </td><td>56.3 </td><td>62.6 </td></tr><tr><td>  Non-coding </td><td>44.2 </td><td>43.4 </td><td>50.5 </td><td>54.3 </td></tr><tr><td colspan="5">Gene annotation </td></tr><tr><td>  Percentage coding (%) </td><td>61 </td><td>57.5 </td><td>61.1 </td><td>65.9 </td></tr><tr><td>  Average gene size (bp) </td><td>1,964 </td><td>1,708 </td><td>1,836 </td><td>1,858 </td></tr><tr><td>  Average gene density (kb/gene) </td><td>3.2 </td><td>3.0 </td><td>2.9 </td><td>2.8 </td></tr><tr><td>  Protein-coding genes </td><td>6,655 </td><td>7,113 </td><td>6,786 </td><td>6,648 </td></tr><tr><td>  Of homologous proteins (%) </td><td>6,045 (91) </td><td>6,325 (89) </td><td>6,214 (92) </td><td>6,394 (96) </td></tr><tr><td>  Exons </td><td>12,237 </td><td>10,907 </td><td>9,783 </td><td>9,776 </td></tr><tr><td>  Exons/gene </td><td>1.84 </td><td>1.53 </td><td>1.44 </td><td>1.47 </td></tr><tr><td>  Average exon size (bp) </td><td>994 </td><td>1,107 </td><td>1,230 </td><td>1,221 </td></tr><tr><td>  Intron </td><td>5,563 </td><td>3,161 </td><td>2,997 </td><td>3,103 </td></tr><tr><td>  Introns/gene </td><td>0.84 </td><td>0.44 </td><td>0.44 </td><td>0.46 </td></tr><tr><td>  Average intron size (bp) </td><td>185 </td><td>141 </td><td>142 </td><td>144 </td></tr><tr><td>  Average intergenic distance (bp) </td><td>1,052 </td><td>1,186 </td><td>1,127 </td><td>929 </td></tr><tr><td>  tRNA genes </td><td>152 </td><td>110 </td><td>111 </td><td>96 </td></tr><tr><td>  TEs and repeats (%) </td><td>7.5 </td><td>2.0 </td><td>8.0 </td><td>0.8 </td></tr></tbody></table></div></div></div><div class="table-wrap"><div id="dsx031-T1" data-id="dsx031-T1" class="table-wrap-title"><div class="label -label">Table 1</div><div class="caption"><p>The genome statistics of <em>U. esculenta</em> compared with other sequenced smut fungi</p></div></div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Genome specifics </th><th><em>U. esculenta</em> </th><th><em>U. hordei</em> </th><th><em>U. maydis</em> </th><th><em>S. reilianum</em> </th></tr></thead><tbody><tr><td colspan="5">Assembly statistics </td></tr><tr><td>  Total contig length (Mb) </td><td>20.2 </td><td>20.7 </td><td>19.7 </td><td>18.2 </td></tr><tr><td>  Total scaffold length (Mb) </td><td>21.4 </td><td>21.2 </td><td>19.8 </td><td>18.4 </td></tr><tr><td>  Average base coverage </td><td>×100 </td><td>×20 </td><td>×10 </td><td>×20 </td></tr><tr><td>  N50 contig (kb) </td><td>101.8 </td><td>48.7 </td><td>127.4 </td><td>50.3 </td></tr><tr><td>  N50 scaffold (kb) </td><td>404.8 </td><td>307.7 </td><td>817.8 </td><td>738.5 </td></tr><tr><td>  Chromosome no. </td><td> </td><td>23 </td><td>23 </td><td>23 </td></tr><tr><td>  GC content (%) </td><td>51.3 </td><td>52 </td><td>54 </td><td>59.7 </td></tr><tr><td>  Coding </td><td>56.7 </td><td>54.3 </td><td>56.3 </td><td>62.6 </td></tr><tr><td>  Non-coding </td><td>44.2 </td><td>43.4 </td><td>50.5 </td><td>54.3 </td></tr><tr><td colspan="5">Gene annotation </td></tr><tr><td>  Percentage coding (%) </td><td>61 </td><td>57.5 </td><td>61.1 </td><td>65.9 </td></tr><tr><td>  Average gene size (bp) </td><td>1,964 </td><td>1,708 </td><td>1,836 </td><td>1,858 </td></tr><tr><td>  Average gene density (kb/gene) </td><td>3.2 </td><td>3.0 </td><td>2.9 </td><td>2.8 </td></tr><tr><td>  Protein-coding genes </td><td>6,655 </td><td>7,113 </td><td>6,786 </td><td>6,648 </td></tr><tr><td>  Of homologous proteins (%) </td><td>6,045 (91) </td><td>6,325 (89) </td><td>6,214 (92) </td><td>6,394 (96) </td></tr><tr><td>  Exons </td><td>12,237 </td><td>10,907 </td><td>9,783 </td><td>9,776 </td></tr><tr><td>  Exons/gene </td><td>1.84 </td><td>1.53 </td><td>1.44 </td><td>1.47 </td></tr><tr><td>  Average exon size (bp) </td><td>994 </td><td>1,107 </td><td>1,230 </td><td>1,221 </td></tr><tr><td>  Intron </td><td>5,563 </td><td>3,161 </td><td>2,997 </td><td>3,103 </td></tr><tr><td>  Introns/gene </td><td>0.84 </td><td>0.44 </td><td>0.44 </td><td>0.46 </td></tr><tr><td>  Average intron size (bp) </td><td>185 </td><td>141 </td><td>142 </td><td>144 </td></tr><tr><td>  Average intergenic distance (bp) </td><td>1,052 </td><td>1,186 </td><td>1,127 </td><td>929 </td></tr><tr><td>  tRNA genes </td><td>152 </td><td>110 </td><td>111 </td><td>96 </td></tr><tr><td>  TEs and repeats (%) </td><td>7.5 </td><td>2.0 </td><td>8.0 </td><td>0.8 </td></tr></tbody></table></div><div content-id="dsx031-T1" class="table-modal"><table><thead name="thead" align=""><tr><th>Genome specifics </th><th><em>U. esculenta</em> </th><th><em>U. hordei</em> </th><th><em>U. maydis</em> </th><th><em>S. reilianum</em> </th></tr></thead><tbody><tr><td colspan="5">Assembly statistics </td></tr><tr><td>  Total contig length (Mb) </td><td>20.2 </td><td>20.7 </td><td>19.7 </td><td>18.2 </td></tr><tr><td>  Total scaffold length (Mb) </td><td>21.4 </td><td>21.2 </td><td>19.8 </td><td>18.4 </td></tr><tr><td>  Average base coverage </td><td>×100 </td><td>×20 </td><td>×10 </td><td>×20 </td></tr><tr><td>  N50 contig (kb) </td><td>101.8 </td><td>48.7 </td><td>127.4 </td><td>50.3 </td></tr><tr><td>  N50 scaffold (kb) </td><td>404.8 </td><td>307.7 </td><td>817.8 </td><td>738.5 </td></tr><tr><td>  Chromosome no. </td><td> </td><td>23 </td><td>23 </td><td>23 </td></tr><tr><td>  GC content (%) </td><td>51.3 </td><td>52 </td><td>54 </td><td>59.7 </td></tr><tr><td>  Coding </td><td>56.7 </td><td>54.3 </td><td>56.3 </td><td>62.6 </td></tr><tr><td>  Non-coding </td><td>44.2 </td><td>43.4 </td><td>50.5 </td><td>54.3 </td></tr><tr><td colspan="5">Gene annotation </td></tr><tr><td>  Percentage coding (%) </td><td>61 </td><td>57.5 </td><td>61.1 </td><td>65.9 </td></tr><tr><td>  Average gene size (bp) </td><td>1,964 </td><td>1,708 </td><td>1,836 </td><td>1,858 </td></tr><tr><td>  Average gene density (kb/gene) </td><td>3.2 </td><td>3.0 </td><td>2.9 </td><td>2.8 </td></tr><tr><td>  Protein-coding genes </td><td>6,655 </td><td>7,113 </td><td>6,786 </td><td>6,648 </td></tr><tr><td>  Of homologous proteins (%) </td><td>6,045 (91) </td><td>6,325 (89) </td><td>6,214 (92) </td><td>6,394 (96) </td></tr><tr><td>  Exons </td><td>12,237 </td><td>10,907 </td><td>9,783 </td><td>9,776 </td></tr><tr><td>  Exons/gene </td><td>1.84 </td><td>1.53 </td><td>1.44 </td><td>1.47 </td></tr><tr><td>  Average exon size (bp) </td><td>994 </td><td>1,107 </td><td>1,230 </td><td>1,221 </td></tr><tr><td>  Intron </td><td>5,563 </td><td>3,161 </td><td>2,997 </td><td>3,103 </td></tr><tr><td>  Introns/gene </td><td>0.84 </td><td>0.44 </td><td>0.44 </td><td>0.46 </td></tr><tr><td>  Average intron size (bp) </td><td>185 </td><td>141 </td><td>142 </td><td>144 </td></tr><tr><td>  Average intergenic distance (bp) </td><td>1,052 </td><td>1,186 </td><td>1,127 </td><td>929 </td></tr><tr><td>  tRNA genes </td><td>152 </td><td>110 </td><td>111 </td><td>96 </td></tr><tr><td>  TEs and repeats (%) </td><td>7.5 </td><td>2.0 </td><td>8.0 </td><td>0.8 </td></tr></tbody></table></div></div></p>                        <a id="104086730" scrollto-destination="104086730"></a>
<div id="dsx031-F2" data-id="dsx031-F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/m_dsx031f2.png?Expires=2147483647&Signature=qF3jnyW0cAAqHd-0z8jUb1Omnhnzvzgc73nmAKnSyo3OPeaJrKeOB~BMZ-FfQkJi5MjSCMNwmaQaaC9xWziPrC0-7vM3CNph8AqW3rafeJ8uwmce8mkWYtpykf4xNAyt3ebmq7nGRNfRGYjJd56OK1bhGXbbv8Gsm0FNtQDLxWiPY1d6R5c65nWLJTstfUGwUN0~P~eejnUSj2dVqyg2Tv-vxzDvu8WL0LYtXQLKikn4XbaQsk3tIIPhCtab-wjNycnLTt4~2~l1wKYOTSjM23MCZW39EfMMkrU6JE4qe8souQZf5YuhyxoyrakPUfxNWzrrbut16JLTYn6mB7dyIQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The volume of K-mers is plotted against the frequency at which they occur. The left, truncated peak at low frequency and high volume represents K-mers containing essentially random sequencing errors, whereas the right distribution represents proper (putatively error-free) data. The total K-mer number is 2,424,289,422 and the volume peak is 85. The genome size can be estimated as (total K-mer number)/(volume peak), which is 28.5 Mb." path-from-xml="dsx031f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086730/dsx031f2.tif" path-from-xml="dsx031f2.tif" target="_blank">View large</a><a section="104086730" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/dsx031f2.png?Expires=2147483647&Signature=xv2t7aVZIBkQ7k~51cVp4wNGV7bcmHQGUqIs3-sRn1~7OOyzFGIenAM0Cj3vEsp-lrewolIrh9f3xQqgcIjlIednUBRWB2QirGR01IafcxgpbOql8-lDiPGdAQ~CanRJ9RC9FUMyX7Og9MH4oHJ7mNWJ1mMjjoLwfa3Yxbfc8wCnMNRSHFkQQ4LdSplpRfVv-G5QcRYwwSWfuOErYYWO-OlU0zrA3kUgWCqBu3VhLVU8q1RPtiUdTJSyQzGqWhBe-8o4NSF98j8Mjx1izrM3DP1p0H57e8G2etAcE1XB68rb5-Sw1Mdo8KhPvGHTRWLQ-HfYRABlPXYsxJFCDqjJGQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086730&ar=3979487&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx031f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The volume of K-mers is plotted against the frequency at which they occur. The left, truncated peak at low frequency and high volume represents K-mers containing essentially random sequencing errors, whereas the right distribution represents proper (putatively error-free) data. The total K-mer number is 2,424,289,422 and the volume peak is 85. The genome size can be estimated as (total K-mer number)/(volume peak), which is 28.5 Mb.</p></div></div><div content-id="dsx031-F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/m_dsx031f2.png?Expires=2147483647&Signature=qF3jnyW0cAAqHd-0z8jUb1Omnhnzvzgc73nmAKnSyo3OPeaJrKeOB~BMZ-FfQkJi5MjSCMNwmaQaaC9xWziPrC0-7vM3CNph8AqW3rafeJ8uwmce8mkWYtpykf4xNAyt3ebmq7nGRNfRGYjJd56OK1bhGXbbv8Gsm0FNtQDLxWiPY1d6R5c65nWLJTstfUGwUN0~P~eejnUSj2dVqyg2Tv-vxzDvu8WL0LYtXQLKikn4XbaQsk3tIIPhCtab-wjNycnLTt4~2~l1wKYOTSjM23MCZW39EfMMkrU6JE4qe8souQZf5YuhyxoyrakPUfxNWzrrbut16JLTYn6mB7dyIQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The volume of K-mers is plotted against the frequency at which they occur. The left, truncated peak at low frequency and high volume represents K-mers containing essentially random sequencing errors, whereas the right distribution represents proper (putatively error-free) data. The total K-mer number is 2,424,289,422 and the volume peak is 85. The genome size can be estimated as (total K-mer number)/(volume peak), which is 28.5 Mb." path-from-xml="dsx031f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086730/dsx031f2.tif" path-from-xml="dsx031f2.tif" target="_blank">View large</a><a section="104086730" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/dsx031f2.png?Expires=2147483647&Signature=xv2t7aVZIBkQ7k~51cVp4wNGV7bcmHQGUqIs3-sRn1~7OOyzFGIenAM0Cj3vEsp-lrewolIrh9f3xQqgcIjlIednUBRWB2QirGR01IafcxgpbOql8-lDiPGdAQ~CanRJ9RC9FUMyX7Og9MH4oHJ7mNWJ1mMjjoLwfa3Yxbfc8wCnMNRSHFkQQ4LdSplpRfVv-G5QcRYwwSWfuOErYYWO-OlU0zrA3kUgWCqBu3VhLVU8q1RPtiUdTJSyQzGqWhBe-8o4NSF98j8Mjx1izrM3DP1p0H57e8G2etAcE1XB68rb5-Sw1Mdo8KhPvGHTRWLQ-HfYRABlPXYsxJFCDqjJGQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086730&ar=3979487&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx031f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The volume of K-mers is plotted against the frequency at which they occur. The left, truncated peak at low frequency and high volume represents K-mers containing essentially random sequencing errors, whereas the right distribution represents proper (putatively error-free) data. The total K-mer number is 2,424,289,422 and the volume peak is 85. The genome size can be estimated as (total K-mer number)/(volume peak), which is 28.5 Mb.</p></div></div>                            <h3 scrollto-destination=104086731 id="104086731" class="section-title" >3.2. Higher intron number and varied genome defence were found in <em>U. esculenta</em></h3>
<p>We performed comparative genomic analysis on the four Ustilaginales, <em>U. maydis</em>, <em>U. hordei</em>, <em>S. reilianum</em> and <em>U. esculenta</em>. <em>Ustilago esculenta</em> shares a high degree of genomic synteny with <em>U. hordei</em>, <em>U. maydis</em> and <em>S. reilianum</em> (<a reveal-id="dsx031-F3" data-open="dsx031-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>). Orthologue comparison of <em>U. esculenta</em>, <em>U. maydis</em>, <em>U. hordei</em> and <em>S. reilianum</em> identified 3,602 genes clusters that were common to all four species (<a reveal-id="dsx031-F4" data-open="dsx031-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>). <em>Ustilago esculenta</em> had the highest orthologue similarity to <em>U. hordei</em> (88.6%, 4,166 out of 4,699). It shared 87.2% (4,102 out of 4,699) and 82.1% (3,860 out of 4,699) gene orthologue similarity with <em>U. maydis</em> and <em>S. reilianum</em>, respectively. The results are consistent with the phylogenetic analysis, which showed that <em>U. esculenta</em> is more closely related to <em>U. hordei</em> than to <em>U. maydis</em> and <em>S. reilianum</em> (<a reveal-id="dsx031-F5" data-open="dsx031-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>). Specific <em>U. esculenta</em> gene cluster (number: 399) with classifications (number: 103) were enriched in genes associated with secreted proteins (number: 62). </p>                        <a id="104086733" scrollto-destination="104086733"></a>
<div id="dsx031-F3" data-id="dsx031-F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/m_dsx031f3.png?Expires=2147483647&Signature=dSX~lDQS-VlkgbcvHbapdjm-p6EmHd0S~J1WcvVLDh6JJMuVEw4ykUjANBj6IsNAVl0fx-hcg~VmCqup-z-eQ9f1YO5Y-DtbhKT5WQIBuNaBjpZHC5r2WXdauryD4UhPMNBH5Lpf5zoLukFsrixREghrOLzl2lr0Wc1sWnuUE1sPyPchfHpqI5SsHHZEWBvmeTcUS1jRrLfJvv~0xdLkM5ieA0vnL7kkXmCR3RkrfL0D6sO48P1bGI3C1OO8YW90NQn9Zuznez5dh7j~FDwIqCxyRiLDOeQUt-rgb80ylu5948mhzHOjWvxstuP0PjXboe6XBhUCvjie7vsj~otBPQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genomic synteny of U. esculenta with three other Ustilagoinaceae species. The genomes of four fungi are depicted by different colours: U. esculenta (Ue) in red, U. maydis (Um) in orange, U. hordei (Uh) in green and S. reilianum (Sr) in blue. The scales and labels represented the genome size (Mb) are displayed outside. The synteny relationships between U. esculenta genome and the remaining three species were demonstrated by connected lines." path-from-xml="dsx031f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086733/dsx031f3.tif" path-from-xml="dsx031f3.tif" target="_blank">View large</a><a section="104086733" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/dsx031f3.png?Expires=2147483647&Signature=QQ7grja2nPAq7VzMKbQZ7LR4i7MlDJtpmTZy990HaE9lB5YEPbUm3qk0FfUQsTqubaRTUe4eFY2U9hEca3TSC9eyN-AY2IcVh9uDq9K2jPzEertyr8vIchQuY-Anz-ePrSsk31slvz7Ikzcu9S~v3kheEzlrexlmlhoxomfSSOlbXhQZkrFXN6e55bWWTrbxuBp0GZNxuxAV13bj5gFRQyIyhJmHeODFhTA1VazHgVvJv~9okUzWeAVGn2~aMmgBb5iSOz2Mg-nnN652XS4Pn0obWMiTlsP6Hmo0JwHB6foQAv3y48AOaQSGFuLi-Svktcooj4-4ufULzTx-IoJGAQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086733&ar=3979487&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx031f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genomic synteny of <em>U. esculenta</em> with three other Ustilagoinaceae species. The genomes of four fungi are depicted by different colours: <em>U. esculenta</em> (Ue) in red, <em>U. maydis</em> (Um) in orange, <em>U. hordei</em> (Uh) in green and <em>S. reilianum</em> (Sr) in blue. The scales and labels represented the genome size (Mb) are displayed outside. The synteny relationships between <em>U. esculenta</em> genome and the remaining three species were demonstrated by connected lines.</p></div></div><div content-id="dsx031-F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/m_dsx031f3.png?Expires=2147483647&Signature=dSX~lDQS-VlkgbcvHbapdjm-p6EmHd0S~J1WcvVLDh6JJMuVEw4ykUjANBj6IsNAVl0fx-hcg~VmCqup-z-eQ9f1YO5Y-DtbhKT5WQIBuNaBjpZHC5r2WXdauryD4UhPMNBH5Lpf5zoLukFsrixREghrOLzl2lr0Wc1sWnuUE1sPyPchfHpqI5SsHHZEWBvmeTcUS1jRrLfJvv~0xdLkM5ieA0vnL7kkXmCR3RkrfL0D6sO48P1bGI3C1OO8YW90NQn9Zuznez5dh7j~FDwIqCxyRiLDOeQUt-rgb80ylu5948mhzHOjWvxstuP0PjXboe6XBhUCvjie7vsj~otBPQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genomic synteny of U. esculenta with three other Ustilagoinaceae species. The genomes of four fungi are depicted by different colours: U. esculenta (Ue) in red, U. maydis (Um) in orange, U. hordei (Uh) in green and S. reilianum (Sr) in blue. The scales and labels represented the genome size (Mb) are displayed outside. The synteny relationships between U. esculenta genome and the remaining three species were demonstrated by connected lines." path-from-xml="dsx031f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086733/dsx031f3.tif" path-from-xml="dsx031f3.tif" target="_blank">View large</a><a section="104086733" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/dsx031f3.png?Expires=2147483647&Signature=QQ7grja2nPAq7VzMKbQZ7LR4i7MlDJtpmTZy990HaE9lB5YEPbUm3qk0FfUQsTqubaRTUe4eFY2U9hEca3TSC9eyN-AY2IcVh9uDq9K2jPzEertyr8vIchQuY-Anz-ePrSsk31slvz7Ikzcu9S~v3kheEzlrexlmlhoxomfSSOlbXhQZkrFXN6e55bWWTrbxuBp0GZNxuxAV13bj5gFRQyIyhJmHeODFhTA1VazHgVvJv~9okUzWeAVGn2~aMmgBb5iSOz2Mg-nnN652XS4Pn0obWMiTlsP6Hmo0JwHB6foQAv3y48AOaQSGFuLi-Svktcooj4-4ufULzTx-IoJGAQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086733&ar=3979487&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx031f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genomic synteny of <em>U. esculenta</em> with three other Ustilagoinaceae species. The genomes of four fungi are depicted by different colours: <em>U. esculenta</em> (Ue) in red, <em>U. maydis</em> (Um) in orange, <em>U. hordei</em> (Uh) in green and <em>S. reilianum</em> (Sr) in blue. The scales and labels represented the genome size (Mb) are displayed outside. The synteny relationships between <em>U. esculenta</em> genome and the remaining three species were demonstrated by connected lines.</p></div></div>                        <a id="104086734" scrollto-destination="104086734"></a>
<div id="dsx031-F4" data-id="dsx031-F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/m_dsx031f4.png?Expires=2147483647&Signature=WR52lUQJfS4IZOSIXzZ6kIQ16rM-b8ec~7mdm2O-FV84X58kBzW8a0UoIo8D0g~1k-flnBeCxCVqZ~VSUBDqBuPvIPDDoyMM6xVgpguUnAF43e1iccdbkSjlyy1Z7AUxKTPKMU~VND0fdWGirbJMJpZWpbX3AjvPnKPGtoGj7HWX67N12-kUZTok9bhG-oPiBMe5Hen26k8ccv-EnLwmJeCenK5Dyl9PR4zxFkzng-pA2LQq0di5vU0l-ffXSBsdIkXqpfkAjrV06s9HFcp1k3Bsdmm1lZgcYi1jbFD7RbM41EhekJXX5G3qmA1sCtQe4EImbJiCB-4ZwEBziLFdJg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Gene orthologue comparisons between four species of Ustilagoinaceae family. The four fungi are represented by different colours: U. esculenta (Ue) in red, U. maydis (Um) in orange, U. hordei (Uh) in green and S. reilianum (Sr) in blue." path-from-xml="dsx031f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086734/dsx031f4.tif" path-from-xml="dsx031f4.tif" target="_blank">View large</a><a section="104086734" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/dsx031f4.png?Expires=2147483647&Signature=ER-4Wf4dhCpPHSLageuV4VgIaTCt6-KLlOPTXNrTOGjfYXdg9LHVVD-QJB2PF4AV7DQwd03MEMVL81IoTrnbkDeihIYWDCvTy8V6pUclybn~2XMQN4WC6dyiTdcMPTaQKzV3le4mdaByXWsP2hx2gL7~MMnF8zMI7noTBd~8-Yi3bhd3sP7Vjb0WWocM82GRPSWnPV5xzxYaBxSV0X0qP6PNt9f6Y64iEmd5Bk3fKe4Qc8Yu6aYGEV~vmqojM58u1DL0TOn7FLmOoaIYpAFIlmWfNMBlJ6OfU6520hVOlaEqVJyvfLJFQ8ReA9T6MDDId0nsFAO8xs3pKwd8REIWxA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086734&ar=3979487&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx031f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Gene orthologue comparisons between four species of Ustilagoinaceae family. The four fungi are represented by different colours: <em>U. esculenta</em> (Ue) in red, <em>U. maydis</em> (Um) in orange, <em>U. hordei</em> (Uh) in green and <em>S. reilianum</em> (Sr) in blue.</p></div></div><div content-id="dsx031-F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/m_dsx031f4.png?Expires=2147483647&Signature=WR52lUQJfS4IZOSIXzZ6kIQ16rM-b8ec~7mdm2O-FV84X58kBzW8a0UoIo8D0g~1k-flnBeCxCVqZ~VSUBDqBuPvIPDDoyMM6xVgpguUnAF43e1iccdbkSjlyy1Z7AUxKTPKMU~VND0fdWGirbJMJpZWpbX3AjvPnKPGtoGj7HWX67N12-kUZTok9bhG-oPiBMe5Hen26k8ccv-EnLwmJeCenK5Dyl9PR4zxFkzng-pA2LQq0di5vU0l-ffXSBsdIkXqpfkAjrV06s9HFcp1k3Bsdmm1lZgcYi1jbFD7RbM41EhekJXX5G3qmA1sCtQe4EImbJiCB-4ZwEBziLFdJg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Gene orthologue comparisons between four species of Ustilagoinaceae family. The four fungi are represented by different colours: U. esculenta (Ue) in red, U. maydis (Um) in orange, U. hordei (Uh) in green and S. reilianum (Sr) in blue." path-from-xml="dsx031f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086734/dsx031f4.tif" path-from-xml="dsx031f4.tif" target="_blank">View large</a><a section="104086734" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/dsx031f4.png?Expires=2147483647&Signature=ER-4Wf4dhCpPHSLageuV4VgIaTCt6-KLlOPTXNrTOGjfYXdg9LHVVD-QJB2PF4AV7DQwd03MEMVL81IoTrnbkDeihIYWDCvTy8V6pUclybn~2XMQN4WC6dyiTdcMPTaQKzV3le4mdaByXWsP2hx2gL7~MMnF8zMI7noTBd~8-Yi3bhd3sP7Vjb0WWocM82GRPSWnPV5xzxYaBxSV0X0qP6PNt9f6Y64iEmd5Bk3fKe4Qc8Yu6aYGEV~vmqojM58u1DL0TOn7FLmOoaIYpAFIlmWfNMBlJ6OfU6520hVOlaEqVJyvfLJFQ8ReA9T6MDDId0nsFAO8xs3pKwd8REIWxA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086734&ar=3979487&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx031f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Gene orthologue comparisons between four species of Ustilagoinaceae family. The four fungi are represented by different colours: <em>U. esculenta</em> (Ue) in red, <em>U. maydis</em> (Um) in orange, <em>U. hordei</em> (Uh) in green and <em>S. reilianum</em> (Sr) in blue.</p></div></div>                        <a id="104086735" scrollto-destination="104086735"></a>
<div id="dsx031-F5" data-id="dsx031-F5" class="fig fig-section"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/m_dsx031f5.png?Expires=2147483647&Signature=pZapBVmI7uuPHY2pSeDWod9k1Ud5kke0QFjUuF6LaU48uGjbbrpSyzJsScVu4PiW2lM8qhLRvh8LE6E5MTJHZvzWMZaUQlGw-jvQYzMtwoQlJiRArkeVuezxzFkBpvHSXtHetHsGpxjFcueEkcatMyYYSQo2FICg6dQNr2xFbllgL-xXZqiEJBVyloweKxpg1mbZaDxxl0I0QdwU1GNs4uz30WSjOVZ7PxwjOLEVTRF-WLu9emR3QjPiysABpuV-oi-sqZl1xONEpmK7OgALViuYTxh-xG7jG9e53t7eFZdGZPh1yWBpVbZlHfk1S0ipyZY4~eTt3qCCu6z0i~6y6w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Molecular phylogeny of four species in Ustilagoinaceae family. Consensus phylogenetic tree was constructed from 4,500 trees (45 gene family × 100 bootstrap replicates). Trees were mid-point rooted. The numbers on the branches indicate the percentage of the 4,500 trees support the consensus tree. The fungus A. Nidulans is used as an out group." path-from-xml="dsx031f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086735/dsx031f5.tif" path-from-xml="dsx031f5.tif" target="_blank">View large</a><a section="104086735" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/dsx031f5.png?Expires=2147483647&Signature=xS5drHFjl7xA1lMTXnA4YAmfbvM0sOc3dfQJ3cxdm3KKFx4xQ5FWxn7vq7fLtyeLOzfGdJqO6INYWMemgcjFUJqfDwb4Q4~uiUYkR-UfxyH5exKcyn53wNeKmyRGeDAnJPsISD1Rsblwy3fJ2~9VwTumVtFVAgjJGrKo2CyFA5n-EEYbgPQu~3uOt5d3zZk1OhoGsz70ZRNUB3CDDo5vzehwQdBgFBl-88FZpK0ipXT7dOQe-J8QU1WRwzKjEnVydIdNdD~czIaLDE-BEL4OR--eFQOWlLNIgoromf06IcCAA8M6JTpOp6obhzQ1T7Xe8nBA5GluV9M0yqsgYsH-LA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086735&ar=3979487&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx031f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Molecular phylogeny of four species in Ustilagoinaceae family. Consensus phylogenetic tree was constructed from 4,500 trees (45 gene family × 100 bootstrap replicates). Trees were mid-point rooted. The numbers on the branches indicate the percentage of the 4,500 trees support the consensus tree. The fungus <em>A. Nidulans</em> is used as an out group.</p></div></div><div content-id="dsx031-F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/m_dsx031f5.png?Expires=2147483647&Signature=pZapBVmI7uuPHY2pSeDWod9k1Ud5kke0QFjUuF6LaU48uGjbbrpSyzJsScVu4PiW2lM8qhLRvh8LE6E5MTJHZvzWMZaUQlGw-jvQYzMtwoQlJiRArkeVuezxzFkBpvHSXtHetHsGpxjFcueEkcatMyYYSQo2FICg6dQNr2xFbllgL-xXZqiEJBVyloweKxpg1mbZaDxxl0I0QdwU1GNs4uz30WSjOVZ7PxwjOLEVTRF-WLu9emR3QjPiysABpuV-oi-sqZl1xONEpmK7OgALViuYTxh-xG7jG9e53t7eFZdGZPh1yWBpVbZlHfk1S0ipyZY4~eTt3qCCu6z0i~6y6w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Molecular phylogeny of four species in Ustilagoinaceae family. Consensus phylogenetic tree was constructed from 4,500 trees (45 gene family × 100 bootstrap replicates). Trees were mid-point rooted. The numbers on the branches indicate the percentage of the 4,500 trees support the consensus tree. The fungus A. Nidulans is used as an out group." path-from-xml="dsx031f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086735/dsx031f5.tif" path-from-xml="dsx031f5.tif" target="_blank">View large</a><a section="104086735" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/dsx031f5.png?Expires=2147483647&Signature=xS5drHFjl7xA1lMTXnA4YAmfbvM0sOc3dfQJ3cxdm3KKFx4xQ5FWxn7vq7fLtyeLOzfGdJqO6INYWMemgcjFUJqfDwb4Q4~uiUYkR-UfxyH5exKcyn53wNeKmyRGeDAnJPsISD1Rsblwy3fJ2~9VwTumVtFVAgjJGrKo2CyFA5n-EEYbgPQu~3uOt5d3zZk1OhoGsz70ZRNUB3CDDo5vzehwQdBgFBl-88FZpK0ipXT7dOQe-J8QU1WRwzKjEnVydIdNdD~czIaLDE-BEL4OR--eFQOWlLNIgoromf06IcCAA8M6JTpOp6obhzQ1T7Xe8nBA5GluV9M0yqsgYsH-LA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086735&ar=3979487&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx031f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Molecular phylogeny of four species in Ustilagoinaceae family. Consensus phylogenetic tree was constructed from 4,500 trees (45 gene family × 100 bootstrap replicates). Trees were mid-point rooted. The numbers on the branches indicate the percentage of the 4,500 trees support the consensus tree. The fungus <em>A. Nidulans</em> is used as an out group.</p></div></div><p>Unconserved regions still existed in the <em>U. esculenta</em> genome (white regions in the red Ue genome in <a reveal-id="dsx031-F3" data-open="dsx031-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>). The length of these non-syntenic regions was 3.1 Mb, 15.1% of the whole genome. GC content of these regions was 55.63%, a little higher than that of syntenic regions (51.3%). Totally, 1,589 candidate genes were predicted on these regions (<span class="content-section supplementary-material"><a path-from-xml="sup7" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/supplementary_file_s5_dsx031.xls?Expires=2147483647&Signature=KDGxjoR4DZ-wYw5wWaEOHP73FdeFM9iu0N2vG7bcFRimkLHkI9xwMzORlg7GEjmGx2WpoijgGAFEVdauXY3weS9eZ43WxRDxeJi0f~YA7O3lwbd7TGz9izpIZqwO2qYQLD~CbiiQVEsCptSqZj-OhoDDzDQ6oZPBjrpIfPn5pCBKtaw9kToh84AfxCKGzbIKK83zjUv2ToylMg3xFFnnqBI6OOTqMzMTlUH5KIa~NgOgWpSQLCM4rH9B9RMMyCLQjLLqRRhKhEKutsaCQg3BZfdaesdzc2elyyGKUTEDTNOeniambk-1-QqVGYHuhii2Sr~a5-hYHgxsfpo4Ewt7Tg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary File S5</a></span>), which were spreading in 59 pieces (each piece containing at least 4 genes). Based on the synteny analysis of orthologous genes, 243 genes were found to be orphan genes, and 761 out of 1,346 orthologous genes were annotated as hypothetical protein or uncharacterized protein. No enriched function was found according to GO analysis.</p><p>These four smut fungi have similar genome sizes, GC content and gene-coding percentages. The most obvious differences between our sequenced genome and the other three Ustilaginales are the number and average length of introns and exons. <em>Ustilago esculenta</em> has ∼2,000 more exons and introns individually compared with the other three genomes. This results in higher ratio of exons or introns to genes in <em>U. esculenta</em>. The number of intron-free genes is the lowest and the number of introns per gene (0.84 intron per gene) is the highest in <em>U. esculenta</em> among all four genomes. However in comparison to <em>Aspergillus nidulans</em> (2–3 introns per gene) or <em>C. neoformans</em> (5 introns per gene), <em>U. esculenta</em> has fewer introns. Similar to other Ustilaginales, introns of <em>U. esculenta</em> were primarily distributed at the 5′-end of genes, likely due to the selective pressure during evolution.<a reveal-id="dsx031-B5" data-open="dsx031-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> The number of introns may also be determined by the rates of evolution. Roy and Gilbert<a reveal-id="dsx031-B48" data-open="dsx031-B48" class="link link-ref link-reveal xref-bibr"><sup>48</sup></a> suggest that a large number of introns are often found in relatively earlier ancestors based on maximum likelihood reconstructions. Slow-evolving species may retain more introns than the fast-evolving species. This may explain the higher intron numbers in <em>U. esculenta</em> compared with other three Ustilaginales. Farmers keep selecting on the Ustilago–Zizania interaction complex every planting season to retain the vegetables that maintain the desired morphology of the variety. Only a few plants with the potential to produce high quality <em>Jiaobai</em> are kept for propagation. The artificial selection may reduce the selective pressure from nature, and consequently, slow down the evolution rate and result in higher intron numbers in <em>U. esculenta</em>.</p><p>Analysis for repeating sequences showed that the repeated sequences did not cluster in a specific region but were randomly distributed across the draft genome of <em>U. esculenta</em>. The coverage of repeats (7%) and transposable elements (TEs) in the assembled contigs are similar to that in <em>U. hordei</em> (7.8%).<a reveal-id="dsx031-B25" data-open="dsx031-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> It is nearly 3 and 10 times higher than in <em>U. maydis</em> and <em>S. reilianum</em>, respectively. Surprisingly, the coverage of unclassified repeats in the <em>U. esculenta</em> assembled scaffolds (2.9%) was 10 times higher than that reported in <em>U. hordei</em> (0.23%).<a reveal-id="dsx031-B25" data-open="dsx031-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> TEs expansion in fungi may increase the size of the genomes but it could also help the species adapt to a new or challenging environment.<a reveal-id="dsx031-B49" data-open="dsx031-B49" class="link link-ref link-reveal xref-bibr"><sup>49</sup></a> However, two-thirds of the <em>U. esculenta</em> TEs have no known functions. The <em>U. esculenta</em> TEs showed low similarity with their counterparts found in <em>U. hordei</em>.</p><p>Unlike <em>U. hordei</em>, we did not find a repeat-induced point (RIP) mutation mechanism in <em>U. esculenta</em>. The RIP mutation is a fungal-specific genome defence that repetitive elements mutated from CpN to TpN (GpN to ApN).<a reveal-id="dsx031-B50" data-open="dsx031-B50" class="link link-ref link-reveal xref-bibr"><sup>50</sup></a> RIP may be a limiting factor for TEs replication<a reveal-id="dsx031-B51" data-open="dsx031-B51" class="link link-ref link-reveal xref-bibr"><sup>51</sup></a> and it can control TEs activity. <em>Ustilago maydis</em> lacks genes responsible for RNA interference (RNAi) machinery. Similar to <em>U. hordei</em> and <em>S. reilianum</em>, genes related with RNAi mechanism were annotated in <em>U. esculenta</em>. Those included one sequence of <em>Ago1</em> (g3793), one sequence encoding <em>Dcl1</em> (g4220) and three sequences encoding RNA-directed RNA polymerases (<em>RdRP1</em>, g3276; <em>RdRP2</em>, g1610 and <em>RdRP3</em>, g4076). Chromodomain-coding HP1-like (<em>Chp1</em>, g238) and C5-cytosine methyltransferase (<em>DNAme</em>, g3112) that were detected in <em>U. hordei</em> and <em>S. reilianum</em> were found in <em>U. esculenta</em> as well. <em>Ustilago esculenta</em> showed highly conserved synteny with <em>S. re</em><em>i</em><em>lianum</em> and <em>U. hordei</em> in <em>Dcl1</em>, DNAme and <em>RdRP3</em>. Only <em>U. hordei</em> carried additional genes with functions associated with retrotransposition in the <em>Chp1</em> (UHOR_15241) and <em>RdRP1</em> (UHOR_13400, UHOR_08875 and UHOR_13402). The <em>RdRP1</em> locus of <em>U. esculenta</em> showed a distinct pattern compared with <em>U. hordei</em> and <em>S. reilianum</em>. There was one gene missing (homologue of UHOR_05573) in <em>U. esculenta</em>. It indicates that these four smut fungi have evolved differently regarding genome defence when they were under different selection pressure. The possession of RNAi-related genes and the lack of RIP may suggest that <em>U. esculenta</em> utilizes RNAi for genome defence to control TE proliferating and activity under high selection pressure. Although <em>U. hordei</em> balanced TE activity by combination of RNAi, methylation and RIP mutagenesis,<a reveal-id="dsx031-B49" data-open="dsx031-B49" class="link link-ref link-reveal xref-bibr"><sup>49</sup></a><sup>,</sup><em>U. maydis</em> may maintain genome stability through highly efficient recombination system for lack of universal mechanisms for TE control and heterochromatin formation.<a reveal-id="dsx031-B49" data-open="dsx031-B49" class="link link-ref link-reveal xref-bibr"><sup>49</sup></a> Genome defence mechanism of <em>S. reilianum</em>, an intermediate between <em>U. hordei</em> and <em>U. maydis</em>, remains to be determined although silencing pathway components being represented in its genome.</p>                            <h3 scrollto-destination=104086740 id="104086740" class="section-title" >3.3. DEGs between different strains</h3>
<p>To further determine changes of gene expression between different life stages and different strains, digital gene expression profiling of M, T and MT were investigated (<span class="content-section supplementary-material"><a path-from-xml="sup7" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/supplementary_file_s5_dsx031.xls?Expires=2147483647&amp;Signature=KDGxjoR4DZ-wYw5wWaEOHP73FdeFM9iu0N2vG7bcFRimkLHkI9xwMzORlg7GEjmGx2WpoijgGAFEVdauXY3weS9eZ43WxRDxeJi0f~YA7O3lwbd7TGz9izpIZqwO2qYQLD~CbiiQVEsCptSqZj-OhoDDzDQ6oZPBjrpIfPn5pCBKtaw9kToh84AfxCKGzbIKK83zjUv2ToylMg3xFFnnqBI6OOTqMzMTlUH5KIa~NgOgWpSQLCM4rH9B9RMMyCLQjLLqRRhKhEKutsaCQg3BZfdaesdzc2elyyGKUTEDTNOeniambk-1-QqVGYHuhii2Sr~a5-hYHgxsfpo4Ewt7Tg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary File S2</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup16" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/supplementary_table_s2_dsx031.doc?Expires=2147483647&amp;Signature=qfEXKZL4hpvAJCzH5H7loYPOeD3U028EvFSBZSAMTR89o5noi2NPZnb0r~Iaf8YafKKG3410omjsF4zLjml2tPrDOSX-x6KaMDwacq2253psHts4wDGeCiQhZC2ULAvN9Ir3MVmsC21bYaT6Bogh0pOGQjlrpl6r~XfGLLbw6fC6vm5zPXXHVDBleRJeKWdtUHPXc2JXLHVmRmujzBjIIntaOjzwNpr1tX~5ROFr9nTk234fHZC~GnDMfgq2bssK3KanFhK79exn9SPVWoFsFFYjOYvfXR-NmhvdI08V87u5YcvmDm6jAlVTcBoL2OgAkZimaRCKJ8dS~r2SuQmGNg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Table S2</a></span>). Totally, 6,344, 6,307 and 6,335 genes were expressed at M, T and MT, respectively, and 97.7% (6,250 out of 6,394) were constitutively expressed, while only 52 genes were strain-specific and 92 genes were expressed at two strains (<a reveal-id="dsx031-F6" data-open="dsx031-F6" class="link link-reveal link-table xref-fig">Fig. 6</a>). Four hundred and seventy-seven differentially expressed genes (DEGs) (<em>P</em>-value &lt; 0.01 and fold change of expression &gt; 2.0) between mycelia and sporidia or between sporidia of T strain and MT strain were identified (<span class="content-section supplementary-material"><a path-from-xml="sup4" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/supplementary_file_s2_dsx031.xls?Expires=2147483647&amp;Signature=C3nu5QEfgdRa~jCnGIVFbNCudwHZzuQ7Db-iD5Bi9-v5EMf2lI9s1NMQGFQV-NKFiySF1jNDyb5AYBxYTvDDPgk4SIaoWZOezeO2qqk81f5AjhnHwP8OQ4H9b-i7BJ1Cm~Uxw47idENq2jeN5CSL1curf~q3d0lzGcdk~wR3-E6dfpPP4T~TT2a-6KDrxBHJNROlKLDT8r1Uc7GSW9aAFgCi~AXUKpCXgOn7qC4m-4rQ4L-d7E2CtP~tPxxfr2EzkmkwJOZiSiHw~QC8gH1hqHiYv7glLevmTtBgJ7xsnipcOtTmbhpijvMzyZup~PHgXr2Tc-9Vxw-Z2XQV2K2sXg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary File S2</a></span>). </p>                        <a id="104086742" scrollto-destination="104086742"></a>
<div id="dsx031-F6" data-id="dsx031-F6" class="fig fig-section"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/m_dsx031f6.png?Expires=2147483647&Signature=jTvXmr-fdY~2AKFVjU5X29u7dW5pVT1F1UWJpCcDT6jZ-YmnRxRz2N-RqYS6SjjKZUF3mX8e~yuOr5MRReldwgn8Z2Wu4Ig3gxAYvxKarrDefJPzUWmOa8p7ajB0MiQV23P52iEULTDYmURnHPieRIL20YY~CEjkF9N9zPOzzcdSyjTTcKmWcieKF-MhEvK5PXsVtnDqsajLWRKpcVP4wwTJAcsZmRnhtxFCazKPBgCOEN5iesg2tSlETlQUwTxv7HTm1IzH3rtAqlyjR0jydQ6ssX27ClKKYgTcPraeKXf1G1Nd4j25JOTTbHrBOAZRKT-hZHuCYO9DXVqI0QClRw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Clusters of expressed genes in M, MT and T strains. A expressed gene number was listed for each components and strains." path-from-xml="dsx031f6.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086742/dsx031f6.tif" path-from-xml="dsx031f6.tif" target="_blank">View large</a><a section="104086742" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/dsx031f6.png?Expires=2147483647&Signature=kOMV383yKyyMGT3bmKpfpyhXDmKeaGz66kmHKVvVsDFZ98UENywfNvkAAd5h-T0v17ARk19SFSJI1ecrZyIOYRkZ4Cs-h4bKGYQ0wtsrRMVHG8Ot~gCbBfcGXFO-cTrtbrcugSdEehKnZNqAZubLW1QWLTVUqMmQRxB6lzjH4sc6xewp3X10NrHPpBo~U82QWcSExnCfzEjC9yo5XVzzFcQ54YiZ8-3SaznY4LPPEh~XsEgASD4Cko6N3HPqsT9NjVIJjEbrKTU~bpRddbT2DqNaJLeidLikw2nChLK56OGvl5qD1VyRRTJEj4VISnShA~8lDq8iyakmkZqxF24r~A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086742&ar=3979487&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx031f6.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Clusters of expressed genes in M, MT and T strains. A expressed gene number was listed for each components and strains.</p></div></div><div content-id="dsx031-F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/m_dsx031f6.png?Expires=2147483647&Signature=jTvXmr-fdY~2AKFVjU5X29u7dW5pVT1F1UWJpCcDT6jZ-YmnRxRz2N-RqYS6SjjKZUF3mX8e~yuOr5MRReldwgn8Z2Wu4Ig3gxAYvxKarrDefJPzUWmOa8p7ajB0MiQV23P52iEULTDYmURnHPieRIL20YY~CEjkF9N9zPOzzcdSyjTTcKmWcieKF-MhEvK5PXsVtnDqsajLWRKpcVP4wwTJAcsZmRnhtxFCazKPBgCOEN5iesg2tSlETlQUwTxv7HTm1IzH3rtAqlyjR0jydQ6ssX27ClKKYgTcPraeKXf1G1Nd4j25JOTTbHrBOAZRKT-hZHuCYO9DXVqI0QClRw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Clusters of expressed genes in M, MT and T strains. A expressed gene number was listed for each components and strains." path-from-xml="dsx031f6.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086742/dsx031f6.tif" path-from-xml="dsx031f6.tif" target="_blank">View large</a><a section="104086742" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/dsx031f6.png?Expires=2147483647&Signature=kOMV383yKyyMGT3bmKpfpyhXDmKeaGz66kmHKVvVsDFZ98UENywfNvkAAd5h-T0v17ARk19SFSJI1ecrZyIOYRkZ4Cs-h4bKGYQ0wtsrRMVHG8Ot~gCbBfcGXFO-cTrtbrcugSdEehKnZNqAZubLW1QWLTVUqMmQRxB6lzjH4sc6xewp3X10NrHPpBo~U82QWcSExnCfzEjC9yo5XVzzFcQ54YiZ8-3SaznY4LPPEh~XsEgASD4Cko6N3HPqsT9NjVIJjEbrKTU~bpRddbT2DqNaJLeidLikw2nChLK56OGvl5qD1VyRRTJEj4VISnShA~8lDq8iyakmkZqxF24r~A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086742&ar=3979487&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx031f6.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Clusters of expressed genes in M, MT and T strains. A expressed gene number was listed for each components and strains.</p></div></div><p>GO term enrichment analysis of the 477 DEGs indicated that the DEGs correlated with transport (transmembrane transport and transporter activity), localization, oxidation reduction, oxidoreductase activity, ion binding, cation binding, cofactor binding, co-enzyme binding, membrane and so on were enriched. Interestingly, when compared with sporidial cells, most of DEGs (394 out of 441) were up-regulated in mycelia cells. And similar pattern was observed for sporidia of T strain when compared with that of MT strain (<span class="content-section supplementary-material"><a path-from-xml="sup17" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/supplementary_table_s3_dsx031.doc?Expires=2147483647&amp;Signature=gza~8LfhvpTI6eeK5gxBdDwCA-DGJrcQCPTHr-Cf0m6PvXyGWzrswy-3RLVqZGVHIk5o4GYEJSAOH7b8dos~uJ~rDAYRa9uGkyUII-Io1E-nsNqiuoQTTDqErz77Xd0nhLEubCLLOq-EXEW2RG6q19N0V-wnsorFyjKEMgwiyPwd-EorqMBUvjECsEea5oGTZLfbfRy-B0SzkKzcr75DBVRLAH4OGGQl7zsH95FGQGboxALdsPOwwbp4YpnekynaBg3qXZgBxg2Wb-Bt5Ru6McbuxIjiMKo5jrU1ckmjdLaYvx1UskfRkfAyPUjmk9Rm851o2Vpaf--dCWqz-0HVYQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>), indicating a more active status of T strain cells.</p>                            <h3 scrollto-destination=104086744 id="104086744" class="section-title" >3.4. Structure and function of mating-type loci</h3>
<p>Genes that have a strong effect on speciation are those involved in mating, hyphal fusion or dikaryon formation, and those associated with ecological adaptation.<a reveal-id="dsx031-B15" data-open="dsx031-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> The <em>a</em> mating locus comprises a pheromone-receptor system, which is conserved in the sequenced smuts.<a reveal-id="dsx031-B52" data-open="dsx031-B52" class="link link-ref link-reveal xref-bibr"><sup>52</sup></a><em>Ustilago maydis</em> and <em>S. reilianum</em> have tetrapolar mating systems and their mating loci <em>a</em> and <em>b</em> locate in two different chromosomes. Whereas the mating loci <em>a</em> and <em>b</em> in <em>U. hordei</em> are in the same choromosome (bipolar).</p><p>A gene cluster of <em>a2</em>-locus is found in <em>U. esculenta</em> on Scaffold 34 containing the left border gene <em>lba</em> (g4094), one mating pheromone gene <em>mfa2.1</em> (g4096), the right border gene <em>rba</em> (g4097) and the <em>panC</em> (g4098) by bioinformatic analysis. Also the gene cluster of <em>a3</em>-locus is found on Scaffold 161, containing one mating pheromone gene <em>mfa3.2</em> (g6433) and one pheromone receptor gene <em>pra3</em> (g6434). Comparing to other smut fungi, the <em>a</em> locus should have at least two genes, pheromone gene and pheromone receptor gene. So we did the homologous comparison to the sequence from <em>lba</em> gene to <em>rba</em> gene on Scaffold 34, finding the pheromone receptor gene <em>pra2</em>, another mating pheromone gene <em>mfa2.3</em> and the mitochondrial protein gene <em>rga2</em> in the <em>a2</em>-locus, which was confirmed by PCR with the primers designed from <em>lba</em> and <em>rba</em>. Same works have been done in the <em>a3</em>-locus. However, the right border gene could not be found and, instead, a long sequence mainly encoding transposase were found at the end of the Scaffold 161. Additional PCR were carried out with the primers designed from <em>lba</em>, <em>rba</em> and the intermediates, and another mating pheromone gene <em>mfa3.1</em> was found nearby the <em>rba</em> gene. The two <em>a</em> loci of <em>U. esculenta</em> were syntenic to that of <em>S. reilianum</em>, respectively, with conserved gene content, order and position, except for two remarkable characters (<a reveal-id="dsx031-F7" data-open="dsx031-F7" class="link link-reveal link-table xref-fig">Fig. 7</a>). </p>                        <a id="104086747" scrollto-destination="104086747"></a>
<div id="dsx031-F7" data-id="dsx031-F7" class="fig fig-section"><div class="label fig-label">Figure 7</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/m_dsx031f7.png?Expires=2147483647&Signature=D67VeA3ytA2qqXyKaOlL3tIX0~NJMdrVUATTCrNBWnyGhoMIWJTTPfdOUIxMihHlsU-CqG7pet7B1zwAPZN8XHgr1TsbP930fho201ckVkRFuLqxbz0YWYPoV0DliHt7vYWsM9Ax0w4pzNAuRTd5AZutxx-VN-OwLXO2rmq1MY9pUc-tAD5qaa7sURoOlxe1d5iQHWcDpgL62hMGtPIkgnKes7v3TdkuDG0FqzGtBysWCek9NzD0QIu-kDx98VsrZ05sZa9lXk08vL2ZKIW2tb5efuHBhtGMz22rV0WetJBGa7WfiK-xICvri~61VkoubQqlcxJ4YyDbe4zhDgZ~jg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Mating-type loci structures of Ustilagoinaceae species. Genes are indicated by arrows. Arrows directions represent the genes orientations. The same colour indicates the same gene family and the detailed gene functions are shown at the right corners of the figures. The figures were drawn based on the real length of gene loci. (A) Indicates the loci a structures from the lba gene to the panC gene. The broken lines show the actual length is longer than it draws. (B) Indicates the loci b structures which include the two homeodomain proteins bE and bW." path-from-xml="dsx031f7.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086747/dsx031f7.tif" path-from-xml="dsx031f7.tif" target="_blank">View large</a><a section="104086747" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/dsx031f7.png?Expires=2147483647&Signature=LzQePdPGLPpAkEBmCXkUOVRVNPlj8-yNOLvHXjEg52T8qyGWDUG3YnpLLMUSD6Azj56FfBWBskTpReYjaiMe~qmNjcq69BgxeLGA9hyU8Hsa9H5kaZfbz1ps00BPqftelBNPe25awkX6kJPJTFaQ9BpgJpMBbfCcY8HL8wPzz0vH~ZeV24mYXCEFaZsHXhlsMgYxsZU-X07mUMRjTkzrlRnOeWgHjPzOsBdTOTLe1B6qlxkvPoZKSr2QKNzLbYYcVGABazvmEZ6Dz8M0AMBrj~XHalbBVpmm8bRXohcd838IIAPdxcWMOHL8QpoF1wGLiVBaQC-ZhRjIMFlAaYbxrg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086747&ar=3979487&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx031f7.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Mating-type loci structures of Ustilagoinaceae species. Genes are indicated by arrows. Arrows directions represent the genes orientations. The same colour indicates the same gene family and the detailed gene functions are shown at the right corners of the figures. The figures were drawn based on the real length of gene loci. (A) Indicates the loci a structures from the <em>lba</em> gene to the <em>panC</em> gene. The broken lines show the actual length is longer than it draws. (B) Indicates the loci b structures which include the two homeodomain proteins bE and bW.</p></div></div><div content-id="dsx031-F7" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 7</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/m_dsx031f7.png?Expires=2147483647&Signature=D67VeA3ytA2qqXyKaOlL3tIX0~NJMdrVUATTCrNBWnyGhoMIWJTTPfdOUIxMihHlsU-CqG7pet7B1zwAPZN8XHgr1TsbP930fho201ckVkRFuLqxbz0YWYPoV0DliHt7vYWsM9Ax0w4pzNAuRTd5AZutxx-VN-OwLXO2rmq1MY9pUc-tAD5qaa7sURoOlxe1d5iQHWcDpgL62hMGtPIkgnKes7v3TdkuDG0FqzGtBysWCek9NzD0QIu-kDx98VsrZ05sZa9lXk08vL2ZKIW2tb5efuHBhtGMz22rV0WetJBGa7WfiK-xICvri~61VkoubQqlcxJ4YyDbe4zhDgZ~jg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Mating-type loci structures of Ustilagoinaceae species. Genes are indicated by arrows. Arrows directions represent the genes orientations. The same colour indicates the same gene family and the detailed gene functions are shown at the right corners of the figures. The figures were drawn based on the real length of gene loci. (A) Indicates the loci a structures from the lba gene to the panC gene. The broken lines show the actual length is longer than it draws. (B) Indicates the loci b structures which include the two homeodomain proteins bE and bW." path-from-xml="dsx031f7.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086747/dsx031f7.tif" path-from-xml="dsx031f7.tif" target="_blank">View large</a><a section="104086747" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/dsx031f7.png?Expires=2147483647&Signature=LzQePdPGLPpAkEBmCXkUOVRVNPlj8-yNOLvHXjEg52T8qyGWDUG3YnpLLMUSD6Azj56FfBWBskTpReYjaiMe~qmNjcq69BgxeLGA9hyU8Hsa9H5kaZfbz1ps00BPqftelBNPe25awkX6kJPJTFaQ9BpgJpMBbfCcY8HL8wPzz0vH~ZeV24mYXCEFaZsHXhlsMgYxsZU-X07mUMRjTkzrlRnOeWgHjPzOsBdTOTLe1B6qlxkvPoZKSr2QKNzLbYYcVGABazvmEZ6Dz8M0AMBrj~XHalbBVpmm8bRXohcd838IIAPdxcWMOHL8QpoF1wGLiVBaQC-ZhRjIMFlAaYbxrg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086747&ar=3979487&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx031f7.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Mating-type loci structures of Ustilagoinaceae species. Genes are indicated by arrows. Arrows directions represent the genes orientations. The same colour indicates the same gene family and the detailed gene functions are shown at the right corners of the figures. The figures were drawn based on the real length of gene loci. (A) Indicates the loci a structures from the <em>lba</em> gene to the <em>panC</em> gene. The broken lines show the actual length is longer than it draws. (B) Indicates the loci b structures which include the two homeodomain proteins bE and bW.</p></div></div><p>First, <em>a3</em> locus was distinctively enlarged in <em>U. esculenta</em>. The <em>a2</em> locus encompassed 10-kb regions, while <em>a3</em> locus extended to over 22-kb, with a transposase flanking to <em>panC</em> and a ∼14-kb segment inserted between <em>pra3</em> and <em>mfa3.1</em> (<a reveal-id="dsx031-F7" data-open="dsx031-F7" class="link link-reveal link-table xref-fig">Fig. 7</a>). A comprehensive search of <em>a3</em> locus for repetitive elements identified seven LTR (two Gypsy, three Copia and two DIRS) occupied 31.6% of this large region. And 76.9% of LTR located at a 14-kb insertion region. This was similar to <em>MAT-1</em> of <em>U. hordei</em>, which accumulated repetitive elements (many were LTR) covering more than 50% of this region.<a reveal-id="dsx031-B25" data-open="dsx031-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> TEs, having been proved to be implicated in the mating-type rearrangement in basidiomycete<a reveal-id="dsx031-B53" data-open="dsx031-B53" class="link link-ref link-reveal xref-bibr"><sup>53</sup></a><sup>,</sup><a reveal-id="dsx031-B54" data-open="dsx031-B54" class="link link-ref link-reveal xref-bibr"><sup>54</sup></a> and yeast,<a reveal-id="dsx031-B55" data-open="dsx031-B55" class="link link-ref link-reveal xref-bibr"><sup>55</sup></a> are associated with steps in the evolution of sexual types.<a reveal-id="dsx031-B49" data-open="dsx031-B49" class="link link-ref link-reveal xref-bibr"><sup>49</sup></a> Selection pressure from the co-evolving hosts makes Ustilago and other obligate parasites continue adaptation and favour the presence of higher frequencies of TEs,<a reveal-id="dsx031-B49" data-open="dsx031-B49" class="link link-ref link-reveal xref-bibr"><sup>49</sup></a><sup>,</sup><a reveal-id="dsx031-B56" data-open="dsx031-B56" class="link link-ref link-reveal xref-bibr"><sup>56</sup></a> which may explain the extended segment of mating locus in <em>U. esculenta</em> and <em>U. hordei</em>. Recent research found that Gypsy-like elements were related to ovule development in sexual but not apomictic (asexual) genotypes of plants<a reveal-id="dsx031-B57" data-open="dsx031-B57" class="link link-ref link-reveal xref-bibr"><sup>57</sup></a> and transposons drove sex chromosome evolution with the evidences from <em>Drosophila Miranda</em><em>.</em><a reveal-id="dsx031-B58" data-open="dsx031-B58" class="link link-ref link-reveal xref-bibr"><sup>58</sup></a> Therefore, the high frequency of TEs in mating locimay help <em>U. esculenta</em> to maintain adaptive potential with regulation of the mating-type rearrangement and the sexual reproduction stage but more in-depth analysis is needed.</p><p>Second, the gene <em>lga2</em>, which ensuring uniparental mitochondrial DNA inheritance<a reveal-id="dsx031-B59" data-open="dsx031-B59" class="link link-ref link-reveal xref-bibr"><sup>59</sup></a> and being major component interfering with pathogenic,<a reveal-id="dsx031-B60" data-open="dsx031-B60" class="link link-ref link-reveal xref-bibr"><sup>60</sup></a> is missing from <em>a2</em> locus and also is not present in the genome. Uniparental inheritance of mitochondria dominates among sexual eukaryotes and is influenced by the mating-type loci in many species, such as slime mold <em>Physarum polycephalum</em>, basidiomycete fungus <em>C. neoformans</em> and <em>U. maydis</em>, green alga <em>Chlamydomonas reinhardtii</em>.<a reveal-id="dsx031-B61" data-open="dsx031-B61" class="link link-ref link-reveal xref-bibr"><sup>61</sup></a> The <em>lga2</em> and <em>rga2</em> genes are specific to the <em>a2</em> mating-type locus and direct uniparental mtDNA inheriance by mediating elimination of the <em>a1</em>-associated mtDNA in <em>U. maydis</em>. Lga2 has a negative role on mitochondrial fusion and may functions on mtDNA elimination, whereas Rga2 can counteract the mtDNA elimination from Lga2-dependent.<a reveal-id="dsx031-B59" data-open="dsx031-B59" class="link link-ref link-reveal xref-bibr"><sup>59</sup></a> Besides <em>rga2</em>, important genes involved in mitochondria inheritance were found in <em>U. esculenta</em>, for example, <em>dnm1</em> (g6223, a central component of mitochondrial fission), <em>fis1</em> (g2409, involved in mitochondrial fission), <em>fzo1</em> (g3102, key component of mitochondrial fusion) and <em>mrb1</em> (g5377, mitochondrial p32 family protein). It was implied that <em>U. esculenta</em> might mediate its mitochondrial genome integrity and inheritance in a differentiated way compared with wild <em>U. maydis</em> strain, but a similar way to <em>lga2</em> deletion strain, which leads to biparental inheritance. This may be benefit the fungi to maintain its entophytic life during artificial selection, for biparental inheritance, coupled with mitochondrial recombination, has been speculated to be a potentially adaptive strategy under challenging environmental situations.<a reveal-id="dsx031-B62" data-open="dsx031-B62" class="link link-ref link-reveal xref-bibr"><sup>62</sup></a></p><p>In addition to the <em>a</em> locus, Ustilaginales have the <em>b</em> locus encoding two transcription factors that can form an active heterodimer, if they come from compatible mating partners. Scaffold 22 contained a cluster of the <em>b</em> mating-type genes <em>bE2</em> (g2853) and <em>bW2</em> (g2854), which were significantly up-regulated in sporidia. In <em>S. reilianum</em>, <em>bW</em> is separated from the neighbouring gene <em>nat1</em> by a transposon. Similarly a transposon-related sequence was also found between <em>bE</em> and <em>c1d1</em> in <em>U. esculenta</em>. As for the <em>a</em> locus, a second copy of the <em>b</em> locus was present in the mycelial culture containing <em>bE3</em> and <em>bW3</em>. The main target of bW/bE is the transcription factor <em>rbf1</em>,<a reveal-id="dsx031-B63" data-open="dsx031-B63" class="link link-ref link-reveal xref-bibr"><sup>63</sup></a> which also has an orthologue in <em>U. esculenta</em>. To test if mating and filament induction occurs, we generated haploid sporidial cultures from <em>U. esculenta</em> teliospores. Mixing of two sporidial cultures led to induction of filaments showed compatible mating types recognized each other (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/supplementary_fig_s1_dsx031.png?Expires=2147483647&Signature=CnZcTvYT3i5XWT6EKbc3889ymAipNGHbJZMGyiU6FhWEy9eSfdWzzEO6Hv9wGkPNork~xvf0RMTJUs7FHJYzKePGrKXp0pUPW6mp314fPHnpbAvW1wRiWignpeQEpNvMxNKEc95XD08FS-FV92jsVgq62~B4vQnqFwNl9~mba-yxWJFVh3ni9eM~YSuwW4HsZ~EqVebMcNLp1PN4kt9fb8itQygLZ~x9CIrbATxzrs8u~hjoTpcDnF20iucLqbMrAMvHLBIr6wo-x967Y2wI9xB2PyQ6SR1oy7ehm-qvt4L8RLweo5-uKWgpriRxT7tEztMCPYGcgR0eSQNNssYNfQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>). This indicated that the genetic program for mating and induction of filaments was maintained in <em>U. esculenta</em>.</p><p>In our genome assembly, the <em>a</em> and <em>b</em> mating-type genes were present in different scaffolds (Scaffolds 34 and 22). Without further experiments, it is difficult to conclude if <em>U. esculenta</em> has a bipolar or tetrapolar mating system. However, we found the left border gene <em>lba</em> existed in <em>a</em>2 and <em>a3</em> loci and <em>c1d1</em> gene in <em>b2</em> locus, indicating the complete border genes existed in <em>a</em> and <em>b</em> loci in <em>U. esculenta</em>, similar to that of <em>S. reilianum</em>. Besides, two pheromone genes were detected in both <em>a2</em> and <em>a3</em> locus, indicating additional <em>a</em> locus existed to provide pheromone receptor to sense the pheromone (mfa3.1 and mfa2.1). So there should be three <em>a</em> loci in <em>U. esculenta</em>, although the third one were not found in our genome draft. Base on the mating loci comparison, we assume that the mating system of <em>U. esculenta</em> is more similar to <em>S. reilianum</em> than <em>U. hordei</em>.</p><p>Recognition of the opposite mating partner leads to activation intracellular signal transduction network of a MAPK module and cAMP signalling pathway. Both have been shown to influence mating and later stages of the pathogenic development.<a reveal-id="dsx031-B64" data-open="dsx031-B64" class="link link-ref link-reveal xref-bibr"><sup>64</sup></a><sup>,</sup><a reveal-id="dsx031-B65" data-open="dsx031-B65" class="link link-ref link-reveal xref-bibr"><sup>65</sup></a> Most genes reported to be involved in meiosis, MAPK and cAMP signalling cascades in pathogenic Ustilaginales were found in the <em>U. esculenta</em> genome. We also found orthologues of the transcription factors <em>prf1</em><a reveal-id="dsx031-B66" data-open="dsx031-B66" class="link link-ref link-reveal xref-bibr"><sup>66</sup></a> and <em>clp1</em>,<a reveal-id="dsx031-B67" data-open="dsx031-B67" class="link link-ref link-reveal xref-bibr"><sup>67</sup></a> which are required for cell fusion, filamentous growth and pathogenic development. The genes involved in the signalling network of <em>U. maydis</em> (<a reveal-id="dsx031-F8" data-open="dsx031-F8" class="link link-reveal link-table xref-fig">Fig. 8</a>) were also found in <em>U. esculenta</em> except <em>sho1</em>. All of these genes expressed and up-regulated expression were only detected for in strain M. </p>                        <a id="104086753" scrollto-destination="104086753"></a>
<div id="dsx031-F8" data-id="dsx031-F8" class="fig fig-section"><div class="label fig-label">Figure 8</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/m_dsx031f8.png?Expires=2147483647&Signature=xTVXEzCa-S5LYQ798Zfo97eqU6095u5XSerrjMvQI6Hk-iNUmxlNpOaEXoxOglXI44nHkNWm3HFo~1FZ-7v1Ul96aSXxFeWz2g0J-1wr8S0WvPEmRTTFII315obY~ZQwWncbq2omVbgeINjeRFZGMRymruvrL3vJ6anzR7dTBmzKhKkkXhLg8s0GUpAAbyUu5jA5KaPkVg55SjgpnBnG0ZxaOnF84F3KK3oWcTOwyxSnNLAKMMS1xybCAOPn78ecPf4Fdz0I7JfLegWV-M8WVwWg5I4-5VaWC~4U-jHCDno9OECajbKaM6Y-FyDfoMy7NsBFzcAfk~nQc27HFiYfqQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Networks of signalling and transcription factors regulate the pheromone response. Key components of cAMP-mediated Protein Kinase A (PKA) signalling are heterotrimeric G protein a subunit Gpa3, b subunit Bpp1, adenylate cyclase Uac1 and regulatory and catalytic PKA subunits Ubc1 and Adr1, respectively.64 Lipopeptide pheromone activates its cognate seven-transmembrane domain receptor (Pra). The activated receptor triggers a signalling pathway containing the scaffold protein Ubc2 and a pheromone responsive MAPK module (MAPKKK Kpp4, MAPKK Fuz7 and MAPK Kpp2).67 The MAPK regulates Prf1 via phosphorylation and at the transcriptional level. At least four different transcription factors, Rop1, Crk1, Hap2 and Prf1, recognize defined cis-active elements in the prf1 promoter (RRS1-3: Rop1-responsive sequence; CCAAT: CAT box; PREs: pheromone response elements; UAS: upstream activating sequence; bent arrow: transcriptional start site). In addition, MAPK activity is counteracted by phosphatase Rok1, which is activated at the transcriptional level by the MAPK module, resulting in negative feedback regulation. Prf1 leads to expression of the b-mating type locus gene bE and bW. After cell fusion, sexual and pathogenic development is orchestrated by the bE/bW-heterodimer, coordinating the key players of important developmental steps: Clp1 is required for in planta proliferation; Rbf1 is the central transcriptional regulator, controlling appressoria formation and penetration of the host plant via the transcription factor Biz1 and the MAPK Kpp6, as well as filamentous growth and the G2 cell cycle arrest by concerted action of Biz1 and Hdp1.62" path-from-xml="dsx031f8.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086753/dsx031f8.tif" path-from-xml="dsx031f8.tif" target="_blank">View large</a><a section="104086753" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/dsx031f8.png?Expires=2147483647&Signature=ef6cbhJNQmtJh6e9tVl7TK5wgYk0BvG4Z5vgbo9PRtFIsS7B2DGfkd43Q5v0UX4GsQMk8zrDVK4if4aVFXsdMAyB0h9LPciRXxy45Z~t6o4V0DMBtgTvZSpAHb7J2wCD8mV06a55xKgd5R~LJFMvRZeIMOmiNecBwjtSLE5MNjiHNlLu7htJzSYzaDJmrjmpibF12HTGMS8TadYq3IwHwiVnurop-o61wiz0WgzC4tmk6HraGlku6~0HRWCRqOkSk2-L8uR3IpcJgcOxGuseveAD29pbKwRuFjlt4FDMRi6OUMpV3LT6VkmYs8vZtVP~OsMXvGj71zV7AGkeCIfb-g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086753&ar=3979487&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx031f8.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Networks of signalling and transcription factors regulate the pheromone response. Key components of cAMP-mediated Protein Kinase A (PKA) signalling are heterotrimeric G protein a subunit Gpa3, b subunit Bpp1, adenylate cyclase Uac1 and regulatory and catalytic PKA subunits Ubc1 and Adr1, respectively.<a reveal-id="dsx031-B64" data-open="dsx031-B64" class="link link-ref link-reveal xref-bibr"><sup>64</sup></a> Lipopeptide pheromone activates its cognate seven-transmembrane domain receptor (Pra). The activated receptor triggers a signalling pathway containing the scaffold protein Ubc2 and a pheromone responsive MAPK module (MAPKKK Kpp4, MAPKK Fuz7 and MAPK Kpp2).<a reveal-id="dsx031-B67" data-open="dsx031-B67" class="link link-ref link-reveal xref-bibr"><sup>67</sup></a> The MAPK regulates Prf1 via phosphorylation and at the transcriptional level. At least four different transcription factors, Rop1, Crk1, Hap2 and Prf1, recognize defined cis-active elements in the prf1 promoter (RRS1-3: Rop1-responsive sequence; CCAAT: CAT box; PREs: pheromone response elements; UAS: upstream activating sequence; bent arrow: transcriptional start site). In addition, MAPK activity is counteracted by phosphatase Rok1, which is activated at the transcriptional level by the MAPK module, resulting in negative feedback regulation. Prf1 leads to expression of the b-mating type locus gene bE and bW. After cell fusion, sexual and pathogenic development is orchestrated by the bE/bW-heterodimer, coordinating the key players of important developmental steps: Clp1 is required for in planta proliferation; Rbf1 is the central transcriptional regulator, controlling appressoria formation and penetration of the host plant via the transcription factor Biz1 and the MAPK Kpp6, as well as filamentous growth and the G2 cell cycle arrest by concerted action of Biz1 and Hdp1.<a reveal-id="dsx031-B62" data-open="dsx031-B62" class="link link-ref link-reveal xref-bibr"><sup>62</sup></a></p></div></div><div content-id="dsx031-F8" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 8</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/m_dsx031f8.png?Expires=2147483647&Signature=xTVXEzCa-S5LYQ798Zfo97eqU6095u5XSerrjMvQI6Hk-iNUmxlNpOaEXoxOglXI44nHkNWm3HFo~1FZ-7v1Ul96aSXxFeWz2g0J-1wr8S0WvPEmRTTFII315obY~ZQwWncbq2omVbgeINjeRFZGMRymruvrL3vJ6anzR7dTBmzKhKkkXhLg8s0GUpAAbyUu5jA5KaPkVg55SjgpnBnG0ZxaOnF84F3KK3oWcTOwyxSnNLAKMMS1xybCAOPn78ecPf4Fdz0I7JfLegWV-M8WVwWg5I4-5VaWC~4U-jHCDno9OECajbKaM6Y-FyDfoMy7NsBFzcAfk~nQc27HFiYfqQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Networks of signalling and transcription factors regulate the pheromone response. Key components of cAMP-mediated Protein Kinase A (PKA) signalling are heterotrimeric G protein a subunit Gpa3, b subunit Bpp1, adenylate cyclase Uac1 and regulatory and catalytic PKA subunits Ubc1 and Adr1, respectively.64 Lipopeptide pheromone activates its cognate seven-transmembrane domain receptor (Pra). The activated receptor triggers a signalling pathway containing the scaffold protein Ubc2 and a pheromone responsive MAPK module (MAPKKK Kpp4, MAPKK Fuz7 and MAPK Kpp2).67 The MAPK regulates Prf1 via phosphorylation and at the transcriptional level. At least four different transcription factors, Rop1, Crk1, Hap2 and Prf1, recognize defined cis-active elements in the prf1 promoter (RRS1-3: Rop1-responsive sequence; CCAAT: CAT box; PREs: pheromone response elements; UAS: upstream activating sequence; bent arrow: transcriptional start site). In addition, MAPK activity is counteracted by phosphatase Rok1, which is activated at the transcriptional level by the MAPK module, resulting in negative feedback regulation. Prf1 leads to expression of the b-mating type locus gene bE and bW. After cell fusion, sexual and pathogenic development is orchestrated by the bE/bW-heterodimer, coordinating the key players of important developmental steps: Clp1 is required for in planta proliferation; Rbf1 is the central transcriptional regulator, controlling appressoria formation and penetration of the host plant via the transcription factor Biz1 and the MAPK Kpp6, as well as filamentous growth and the G2 cell cycle arrest by concerted action of Biz1 and Hdp1.62" path-from-xml="dsx031f8.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086753/dsx031f8.tif" path-from-xml="dsx031f8.tif" target="_blank">View large</a><a section="104086753" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/dsx031f8.png?Expires=2147483647&Signature=ef6cbhJNQmtJh6e9tVl7TK5wgYk0BvG4Z5vgbo9PRtFIsS7B2DGfkd43Q5v0UX4GsQMk8zrDVK4if4aVFXsdMAyB0h9LPciRXxy45Z~t6o4V0DMBtgTvZSpAHb7J2wCD8mV06a55xKgd5R~LJFMvRZeIMOmiNecBwjtSLE5MNjiHNlLu7htJzSYzaDJmrjmpibF12HTGMS8TadYq3IwHwiVnurop-o61wiz0WgzC4tmk6HraGlku6~0HRWCRqOkSk2-L8uR3IpcJgcOxGuseveAD29pbKwRuFjlt4FDMRi6OUMpV3LT6VkmYs8vZtVP~OsMXvGj71zV7AGkeCIfb-g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086753&ar=3979487&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx031f8.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Networks of signalling and transcription factors regulate the pheromone response. Key components of cAMP-mediated Protein Kinase A (PKA) signalling are heterotrimeric G protein a subunit Gpa3, b subunit Bpp1, adenylate cyclase Uac1 and regulatory and catalytic PKA subunits Ubc1 and Adr1, respectively.<a reveal-id="dsx031-B64" data-open="dsx031-B64" class="link link-ref link-reveal xref-bibr"><sup>64</sup></a> Lipopeptide pheromone activates its cognate seven-transmembrane domain receptor (Pra). The activated receptor triggers a signalling pathway containing the scaffold protein Ubc2 and a pheromone responsive MAPK module (MAPKKK Kpp4, MAPKK Fuz7 and MAPK Kpp2).<a reveal-id="dsx031-B67" data-open="dsx031-B67" class="link link-ref link-reveal xref-bibr"><sup>67</sup></a> The MAPK regulates Prf1 via phosphorylation and at the transcriptional level. At least four different transcription factors, Rop1, Crk1, Hap2 and Prf1, recognize defined cis-active elements in the prf1 promoter (RRS1-3: Rop1-responsive sequence; CCAAT: CAT box; PREs: pheromone response elements; UAS: upstream activating sequence; bent arrow: transcriptional start site). In addition, MAPK activity is counteracted by phosphatase Rok1, which is activated at the transcriptional level by the MAPK module, resulting in negative feedback regulation. Prf1 leads to expression of the b-mating type locus gene bE and bW. After cell fusion, sexual and pathogenic development is orchestrated by the bE/bW-heterodimer, coordinating the key players of important developmental steps: Clp1 is required for in planta proliferation; Rbf1 is the central transcriptional regulator, controlling appressoria formation and penetration of the host plant via the transcription factor Biz1 and the MAPK Kpp6, as well as filamentous growth and the G2 cell cycle arrest by concerted action of Biz1 and Hdp1.<a reveal-id="dsx031-B62" data-open="dsx031-B62" class="link link-ref link-reveal xref-bibr"><sup>62</sup></a></p></div></div><p>Based on the RNA-Seq data, we found that all genes involved in MAPK, cAMP and transcription factors expressed in sprodial cells and mycelia but only <em>kpp6</em> (g4166)<em>, biz1</em> (g3925) significantly up-regulated in mycelia (M). And in sporidial cells (both in T and MT) the pheromone-receptor system <em>mfa2</em>.<em>3</em>, <em>pra2</em> was higher expressed, while it was expressed lower in mycelia (M). Higher expression of the pheromone-receptor system in sporidial cells would be consistent with capacity of sporidial cells to mate resulting in mycelial growth.</p>                            <h3 scrollto-destination=104086755 id="104086755" class="section-title" >3.5. Distinct amino acids responding mechanism on balancing interaction between <em>U. esculenta</em> and <em>Z. latifolia</em></h3>
<p>Nutrition is essential prerequisite for the onset and manifestation of an infection by pathogenic microorganisms, and the plant pathogenic fungi usually delayed to digest host cells to obtain a range of nutrients.<a reveal-id="dsx031-B68" data-open="dsx031-B68" class="link link-ref link-reveal xref-bibr"><sup>68</sup></a> Biotrophic fungi, including <em>U. maydis</em>, appear to absorb the nutrients available in the apoplast surrounding living host cells for much of its pathogenic growth phase, which helped to avoid plant defence systems throughout the infection process.<a reveal-id="dsx031-B69" data-open="dsx031-B69" class="link link-ref link-reveal xref-bibr"><sup>69</sup></a> It makes the fungi have to maintain sets of genes encoding biosynthetic and degradative enzymes for primary metabolism. To fully understand relationship between <em>U. esculenta</em> and its host, we checked the genes involved in central metabolic pathways, such as glycolysis, gluconeogenesis, β-oxidation, glyoxylate cycle, tricarboxylic acid cycle and γ-aminobutyricacid (GABA) shunt (<span class="content-section supplementary-material"><a path-from-xml="sup8" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/supplementary_file_s6_dsx031.xls?Expires=2147483647&Signature=JWQJ4pECpkr6KXI49rp-iv7qTJ-P~uJw0MMNzkFDUDsJc8ZqjDZ1K5KqFaeAW3exb2jA5qOZpPMjpbaccFJK0xprsG8NdzU5ChhGLfeC-O4ZV3lGGjGYjSgWqfzHPzx4k2S~LLsp7ezDRxN07DTCNRpATjRJz5a6ZZyKbsgSPz9g1pVttpIdj0uIEB-v7qqWetVKDghHbw373-zDZZAEIGYea7XDPhDx4fI8dkr9xJcX0sYSiSKgoIYs9dsBsA4rbOdkgGyX5jk1DL2o9M5dKufeTA7Iz1v9-K3527yfCgk1Qxzvc1VXADS9jS7Ow7VlN3yxOvCnsaRG7fdwP3Gf7w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary File S6</a></span>). Nearly all enzymes can find orthologues in <em>U. esculenta</em>; however, <em>ilvC</em> (important for all branched amino acids) and certain genes for biosynthesis of tyrosine, phenylalanine, histidine, alanine and serine were missing in <em>U. esculenta</em> (<a reveal-id="dsx031-F9" data-open="dsx031-F9" class="link link-reveal link-table xref-fig">Fig. 9</a>). <em>Ustilago maydis</em> also lacks a couple of these genes but contains <em>ilvC</em>. Since <em>U. esculenta</em> was an entophyte, we checked the candidate genes of amino acid biosynthesis and its expression in the host plant, by RNA-seq analysis of stem tissues from edible <em>Jiaobai</em> and wild <em>Z. latifolia</em> (which had no <em>U. esculenta</em> inside). Remarkably, all the missing genes of amino acid biosynthesis in <em>U. esculenta</em> were found in <em>Z. latifolia</em> and most of them showed a higher expression level in edible <em>Jiaobai</em> (<a reveal-id="dsx031-F9" data-open="dsx031-F9" class="link link-reveal link-table xref-fig">Fig. 9</a>). This kind of cooperation was also found in symbiotic system of aphid host and its symbiont<a reveal-id="dsx031-B70" data-open="dsx031-B70" class="link link-ref link-reveal xref-bibr"><sup>70</sup></a>; and in aphid, host gene expression and symbiont capabilities are closely integrated within bacteriocytes, whose function as specialized organs of amino acid production. Here, we deduced that <em>U. esculenta</em> is a strict biotrophic fungus, and a partial complementary mechanism had formed after 1000 yrs of symbiosis. However unlike insects and their symbionts, it was the host plant providing the intermediate amino acid production. </p>                        <a id="104086757" scrollto-destination="104086757"></a>
<div id="dsx031-F9" data-id="dsx031-F9" class="fig fig-section"><div class="label fig-label">Figure 9</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/m_dsx031f9.png?Expires=2147483647&Signature=0aZ6V58potkpxZcLDXm2NX7cyHz2gPh5AuGgDRModTsUGmWBSysJgtZB2Jhd4XF7fLqUp85yApBiJTuy4dlROoe8TpL4ugHCDfRqjufbB32wJlmRaqAr9RsOZKkafWMj4uv0FnKcv2deAk82H46cW18Jtr4tOyO8XOUeGac2vlPEJsxik3jfVwzjvg~t7W725b4isndMenACUzAW588WV5PdkZOFIRxJ4gC6-6vc9Ibl9vmGHdDHnrb3D9HaFBABQVH5xebGWR4zzBiSREnhWwxoWxnkmj4lRQMtuoOm4FGRY~-lZjj-jt-NtU0mTSL3MiMxIWlxj48AmHs6Q5y0jQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Amino acid biosynthesis and metabolism pathway in U. esculenta. Based on the amino acid biosynthesis and metabolism pathway in plants and fungi, the putative amino acid biosynthesis and metabolism pathway map of U. esculenta was drawn. Sets of genes encoding biosynthetic and degradative enzymes, which showed differences in the genome of U. maydis, U. esculenta and Z. latifolia, were marked in blue box. Therein, the red genes ID in the box filled with yellow indicated the genes cannot find in U. esculenta, the red genes ID in the box with white bottom colour showed the genes only can find in Z. latifolia, while the green genes ID indicated the genes cannot find in U. maydis. All the genes marked with * means the up-regulated expression during stem swollen progress based on transcriptome sequencing data basic analysis." path-from-xml="dsx031f9.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086757/dsx031f9.tif" path-from-xml="dsx031f9.tif" target="_blank">View large</a><a section="104086757" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/dsx031f9.png?Expires=2147483647&Signature=YONX0C~3XimFCmMSZP89-CVwiBMhv8xSXvitQnmg0wjBLv9sb74PIztiadpT~ksC0HcUnEvrCR~XhWtxGa6S1neURNo8iBAC-JWrxl9fg8JdeXKpe6EkmspWoLdDFo7NpjKAQkst1NLTmkoOBsCGdXBa0va~~XuHgTX75ue-wx56KK3fDwXn2o~I9RKfQnnuhApCMqOBsOLdQ2M7S0PnF~lu8Mjz6etUsn7NdtPFxKfX6~h5QJWWQTLiRR8V9CsX1Mj9Yt-M4A9h2HCn4yr2oVc0bUurK~KHacEJzFXer1~8NVvN5SBQ15fRVDuZunhyo7TRIvdMycgQjrXtxxDuvQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086757&ar=3979487&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx031f9.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Amino acid biosynthesis and metabolism pathway in <em>U. esculenta</em>. Based on the amino acid biosynthesis and metabolism pathway in plants and fungi, the putative amino acid biosynthesis and metabolism pathway map of <em>U. esculenta</em> was drawn. Sets of genes encoding biosynthetic and degradative enzymes, which showed differences in the genome of <em>U. maydis</em>, <em>U. esculenta</em> and <em>Z. latifolia</em>, were marked in blue box. Therein, the red genes ID in the box filled with yellow indicated the genes cannot find in <em>U. esculenta</em>, the red genes ID in the box with white bottom colour showed the genes only can find in <em>Z. latifolia</em>, while the green genes ID indicated the genes cannot find in <em>U. maydis</em>. All the genes marked with * means the up-regulated expression during stem swollen progress based on transcriptome sequencing data basic analysis.</p></div></div><div content-id="dsx031-F9" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 9</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/m_dsx031f9.png?Expires=2147483647&Signature=0aZ6V58potkpxZcLDXm2NX7cyHz2gPh5AuGgDRModTsUGmWBSysJgtZB2Jhd4XF7fLqUp85yApBiJTuy4dlROoe8TpL4ugHCDfRqjufbB32wJlmRaqAr9RsOZKkafWMj4uv0FnKcv2deAk82H46cW18Jtr4tOyO8XOUeGac2vlPEJsxik3jfVwzjvg~t7W725b4isndMenACUzAW588WV5PdkZOFIRxJ4gC6-6vc9Ibl9vmGHdDHnrb3D9HaFBABQVH5xebGWR4zzBiSREnhWwxoWxnkmj4lRQMtuoOm4FGRY~-lZjj-jt-NtU0mTSL3MiMxIWlxj48AmHs6Q5y0jQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Amino acid biosynthesis and metabolism pathway in U. esculenta. Based on the amino acid biosynthesis and metabolism pathway in plants and fungi, the putative amino acid biosynthesis and metabolism pathway map of U. esculenta was drawn. Sets of genes encoding biosynthetic and degradative enzymes, which showed differences in the genome of U. maydis, U. esculenta and Z. latifolia, were marked in blue box. Therein, the red genes ID in the box filled with yellow indicated the genes cannot find in U. esculenta, the red genes ID in the box with white bottom colour showed the genes only can find in Z. latifolia, while the green genes ID indicated the genes cannot find in U. maydis. All the genes marked with * means the up-regulated expression during stem swollen progress based on transcriptome sequencing data basic analysis." path-from-xml="dsx031f9.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086757/dsx031f9.tif" path-from-xml="dsx031f9.tif" target="_blank">View large</a><a section="104086757" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/dsx031f9.png?Expires=2147483647&Signature=YONX0C~3XimFCmMSZP89-CVwiBMhv8xSXvitQnmg0wjBLv9sb74PIztiadpT~ksC0HcUnEvrCR~XhWtxGa6S1neURNo8iBAC-JWrxl9fg8JdeXKpe6EkmspWoLdDFo7NpjKAQkst1NLTmkoOBsCGdXBa0va~~XuHgTX75ue-wx56KK3fDwXn2o~I9RKfQnnuhApCMqOBsOLdQ2M7S0PnF~lu8Mjz6etUsn7NdtPFxKfX6~h5QJWWQTLiRR8V9CsX1Mj9Yt-M4A9h2HCn4yr2oVc0bUurK~KHacEJzFXer1~8NVvN5SBQ15fRVDuZunhyo7TRIvdMycgQjrXtxxDuvQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086757&ar=3979487&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx031f9.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Amino acid biosynthesis and metabolism pathway in <em>U. esculenta</em>. Based on the amino acid biosynthesis and metabolism pathway in plants and fungi, the putative amino acid biosynthesis and metabolism pathway map of <em>U. esculenta</em> was drawn. Sets of genes encoding biosynthetic and degradative enzymes, which showed differences in the genome of <em>U. maydis</em>, <em>U. esculenta</em> and <em>Z. latifolia</em>, were marked in blue box. Therein, the red genes ID in the box filled with yellow indicated the genes cannot find in <em>U. esculenta</em>, the red genes ID in the box with white bottom colour showed the genes only can find in <em>Z. latifolia</em>, while the green genes ID indicated the genes cannot find in <em>U. maydis</em>. All the genes marked with * means the up-regulated expression during stem swollen progress based on transcriptome sequencing data basic analysis.</p></div></div><p>The amino acid effect on mating and <em>in vitro</em> mycelium growth of <em>U. esculenta</em> and <em>U. maydis</em> are very attracting. We found when using arginine as main nitrogen source, it inhibited the fusion of yeast cells (<a reveal-id="dsx031-T2" data-open="dsx031-T2" class="link link-reveal link-table xref-fig">Table 2</a>, <span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/supplementary_fig_s2_dsx031.png?Expires=2147483647&amp;Signature=KIlbX3c6fzYQ2jkuD3wLfIzWQF8JyvU0yqkzgpxWpCDIlD21ADw8MthaJFsehsVErxc8rC1UuvRHw3BTVrWFunr1Su5Rmv2RTh6qdqPkdLfYXR81c-3zWVmXX1Zgpml09qlVUUeFrVyiVi5Tvs7aPOzXUOWV-IRwK05K2iEwvMVTQxNx~BPXdY49ejcoAOaJJXJaiQykUr7UwX8GtasZ1NVFkugoQEn0mPK82DeTFmQpNvkA42JptCOQjjgbiN115b47j-quIDedPeqrUtt67Ij9pR7AL~4DRhqNA-TgbTVPBH7ERxhAczWkibfnYNmMvHTEbRrmt2L5q2~GVtYfHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>), which was contrary to the promoting effects on mycelia formation in <em>Ceratocystis ulm</em><em>i</em><a reveal-id="dsx031-B71" data-open="dsx031-B71" class="link link-ref link-reveal xref-bibr"><sup>71</sup></a> and <em>Candida albicans</em>.<a reveal-id="dsx031-B72" data-open="dsx031-B72" class="link link-ref link-reveal xref-bibr"><sup>72</sup></a> The arginine biosynthesis pathway was up-regulated in yeast/mycelia switching,<a reveal-id="dsx031-B72" data-open="dsx031-B72" class="link link-ref link-reveal xref-bibr"><sup>72</sup></a> which was the most essential response in the progression of infection. In <em>U. esculenta</em>, the presence of exogenous arginine may exhibit feedback inhibition on arginine biosynthesis.<a reveal-id="dsx031-B73" data-open="dsx031-B73" class="link link-ref link-reveal xref-bibr"><sup>73</sup></a> The host plant may conveniently employ this feedback mechanism to modulate <em>U. esculenta</em> infection. It was consistent with the results of High Performance Liquid Chromatography (HPLC) detection for amino acids content in stems of grey <em>Jiaobai</em> and edible <em>Jiaobai</em>, that free arginine content is significantly higher in <em>Jiaobai</em> with disease symptom (grey <em>Jiaobai</em>) than in the edible <em>Jiaobai</em> (<span class="content-section supplementary-material"><a path-from-xml="sup18" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/supplementary_table_s4_dsx031.doc?Expires=2147483647&amp;Signature=eM4VlDBKbowysYVGl-J5FRQL~FX8QCoJSXTtKsCXP7YFQc7uATnZHMF0ejB2WleO~n4c5VzYXGZWhcZBr2uqxEpnOxzlyczJo4KkZHzzSnQTrpKgq22gaiVAkJTDBGTlibMhQezD4WXdfMmSz3GQoluBilL-8jkd1BSd1Ng~KWGfVEbuGYspIcF17tNbldNQPfldbthsdiEffQAaX4C4CkXj7gE0Hj3RN1TbY6j8s2Vn1horPFJCfYvILFXQVgIcmBHkA8gy-BpNOYd9iX0HPS3fFUu~ugwD3D1h70vAKOsPG6OgJmTgVT3T3d~NN0GsPcS4cgx7N4Lx6Nvi3viZ~Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>). But more studies need to be done to show how the arginine regulated the dimorphism and what signalled the regulation-feedback system. <div content-id="dsx031-T2" class="table-modal"><div class="table-wrap"><div id="dsx031-T2" data-id="dsx031-T2" class="table-wrap-title"><div class="label -label">Table 2</div><div class="caption"><p>Comparision of the mating response to the macro- or micro-amino acids effect <em>in vitro</em> between <em>U. esculenta</em> and <em>U. maydis</em></p></div></div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="3">No AA </th><th colspan="4"><em>U. esculenta</em><hr /> </th><th colspan="4"><em>U. maydis</em><hr /> </th></tr></thead><tbody><tr><td colspan="4">- </td><td colspan="4">- </td></tr><tr><td colspan="2"><hr />micro-AA </td><td colspan="2"><hr />macro-AA </td><td colspan="2"><hr />micro-AA </td><td colspan="2"><hr />macro-AA </td></tr><tr><td> </td><td> </td><td>Hyphae length (mm) </td><td> </td><td>Hyphae length (mm) </td><td> </td><td>Hyphae length (mm) </td><td> </td><td>Hyphae length (mm) </td></tr><tr><td colspan="9"><hr /> </td></tr><tr><td>Arg </td><td> ++ </td><td>328.77 ± 18.40<sup>gh</sup> </td><td>– </td><td>– </td><td> ++ </td><td>205.81 ± 56.62<sup>ij</sup> </td><td>– </td><td>– </td></tr><tr><td>Tyr </td><td> ++ </td><td>375.08 ± 41.63<sup>defgh</sup> </td><td> +++ </td><td>348.16 ± 17.77<sup>fgh</sup> </td><td> ++ </td><td>102.19 ± 73.26<sup>klm</sup> </td><td> +++ </td><td>354.32 ± 11.65<sup>fgh</sup> </td></tr><tr><td>His </td><td> ++ </td><td>348.54 ± 35.81<sup>fgh</sup> </td><td> +++ </td><td>362.12 ± 15.82<sup>efgh</sup> </td><td>+ </td><td>10.86 ± 1.96<sup>n</sup> </td><td> +++ </td><td>454.41 ± 17.12<sup>bcd</sup> </td></tr><tr><td>Ile </td><td>+ </td><td>338.62 ± 145.33<sup>fgh</sup> </td><td> ++ </td><td>391.33 ± 24.05<sup>defg</sup> </td><td> ++ </td><td>133.01 ± 54.49<sup>jkl</sup> </td><td> +++ </td><td>206.62 ± 21.55<sup>ij</sup> </td></tr><tr><td>Leu </td><td>+ </td><td>196.78 ± 135.4<sup>1j</sup> </td><td>+ </td><td>194.12 ± 56.92<sup>j</sup> </td><td> +++ </td><td>195.51 ± 22.02<sup>j</sup> </td><td> ++ </td><td>158.53 ± 30.17<sup>jk</sup> </td></tr><tr><td>Val </td><td> ++ </td><td>524.01 ± 138.44<sup>ab</sup> </td><td> ++ </td><td>290.63 ± 286.18<sup>hi</sup> </td><td> ++ </td><td>163.55 ± 30.96<sup>jk</sup> </td><td> +++ </td><td>217.12 ± 94.81<sup>ij</sup> </td></tr><tr><td>Met </td><td> +++ </td><td>580.42 ± 36.09<sup>a</sup> </td><td> +++ </td><td>423.34 ± 71.74<sup>cdef</sup> </td><td>+ </td><td>24.38 ± 8.5<sup>1mn</sup> </td><td>– </td><td>– </td></tr><tr><td>Phe </td><td> ++ </td><td>303.42 ± 15.77<sup>gh</sup> </td><td> +++ </td><td>593.88 ± 36.95<sup>a</sup> </td><td>+ </td><td>42.04 ± 11.67<sup>mn</sup> </td><td>+ </td><td>82.33 ± 17.05<sup>klmn</sup> </td></tr><tr><td>Pro </td><td> ++ </td><td>443.20 ± 18.59<sup>bcde</sup> </td><td> +++ </td><td>334.24 ± 46.49<sup>fgh</sup> </td><td>+ </td><td>140.69 ± 39.60<sup>jkl</sup> </td><td>+ </td><td>52.86 ± 26.72<sup>lmn</sup> </td></tr><tr><td>Lys </td><td> +++ </td><td>487.04 ± 35.42<sup>bc</sup> </td><td> ++ </td><td>311.14 ± 16.30<sup>gh</sup> </td><td>+ </td><td>59.13 ± 38.52<sup>lmn</sup> </td><td>– </td><td>– </td></tr></tbody></table></div><div content-id="dsx031-T2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="3">No AA </th><th colspan="4"><em>U. esculenta</em><hr /> </th><th colspan="4"><em>U. maydis</em><hr /> </th></tr></thead><tbody><tr><td colspan="4">- </td><td colspan="4">- </td></tr><tr><td colspan="2"><hr />micro-AA </td><td colspan="2"><hr />macro-AA </td><td colspan="2"><hr />micro-AA </td><td colspan="2"><hr />macro-AA </td></tr><tr><td> </td><td> </td><td>Hyphae length (mm) </td><td> </td><td>Hyphae length (mm) </td><td> </td><td>Hyphae length (mm) </td><td> </td><td>Hyphae length (mm) </td></tr><tr><td colspan="9"><hr /> </td></tr><tr><td>Arg </td><td> ++ </td><td>328.77 ± 18.40<sup>gh</sup> </td><td>– </td><td>– </td><td> ++ </td><td>205.81 ± 56.62<sup>ij</sup> </td><td>– </td><td>– </td></tr><tr><td>Tyr </td><td> ++ </td><td>375.08 ± 41.63<sup>defgh</sup> </td><td> +++ </td><td>348.16 ± 17.77<sup>fgh</sup> </td><td> ++ </td><td>102.19 ± 73.26<sup>klm</sup> </td><td> +++ </td><td>354.32 ± 11.65<sup>fgh</sup> </td></tr><tr><td>His </td><td> ++ </td><td>348.54 ± 35.81<sup>fgh</sup> </td><td> +++ </td><td>362.12 ± 15.82<sup>efgh</sup> </td><td>+ </td><td>10.86 ± 1.96<sup>n</sup> </td><td> +++ </td><td>454.41 ± 17.12<sup>bcd</sup> </td></tr><tr><td>Ile </td><td>+ </td><td>338.62 ± 145.33<sup>fgh</sup> </td><td> ++ </td><td>391.33 ± 24.05<sup>defg</sup> </td><td> ++ </td><td>133.01 ± 54.49<sup>jkl</sup> </td><td> +++ </td><td>206.62 ± 21.55<sup>ij</sup> </td></tr><tr><td>Leu </td><td>+ </td><td>196.78 ± 135.4<sup>1j</sup> </td><td>+ </td><td>194.12 ± 56.92<sup>j</sup> </td><td> +++ </td><td>195.51 ± 22.02<sup>j</sup> </td><td> ++ </td><td>158.53 ± 30.17<sup>jk</sup> </td></tr><tr><td>Val </td><td> ++ </td><td>524.01 ± 138.44<sup>ab</sup> </td><td> ++ </td><td>290.63 ± 286.18<sup>hi</sup> </td><td> ++ </td><td>163.55 ± 30.96<sup>jk</sup> </td><td> +++ </td><td>217.12 ± 94.81<sup>ij</sup> </td></tr><tr><td>Met </td><td> +++ </td><td>580.42 ± 36.09<sup>a</sup> </td><td> +++ </td><td>423.34 ± 71.74<sup>cdef</sup> </td><td>+ </td><td>24.38 ± 8.5<sup>1mn</sup> </td><td>– </td><td>– </td></tr><tr><td>Phe </td><td> ++ </td><td>303.42 ± 15.77<sup>gh</sup> </td><td> +++ </td><td>593.88 ± 36.95<sup>a</sup> </td><td>+ </td><td>42.04 ± 11.67<sup>mn</sup> </td><td>+ </td><td>82.33 ± 17.05<sup>klmn</sup> </td></tr><tr><td>Pro </td><td> ++ </td><td>443.20 ± 18.59<sup>bcde</sup> </td><td> +++ </td><td>334.24 ± 46.49<sup>fgh</sup> </td><td>+ </td><td>140.69 ± 39.60<sup>jkl</sup> </td><td>+ </td><td>52.86 ± 26.72<sup>lmn</sup> </td></tr><tr><td>Lys </td><td> +++ </td><td>487.04 ± 35.42<sup>bc</sup> </td><td> ++ </td><td>311.14 ± 16.30<sup>gh</sup> </td><td>+ </td><td>59.13 ± 38.52<sup>lmn</sup> </td><td>– </td><td>– </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>All the hyphae length data were detected 3 days after mating assays of <em>U. esculenta</em> and <em>U. maydis</em> on basic solid medium with different kind and amount amino acid added. The capital letter as a superscript showed significant differences analysis between edible and grey <em>jiaobai</em> at <em>P</em> &lt; 0.05 level, analysed by one-way ANOVA analysis with Duncan method. All the amino acids (AA) name were abbreviation listed in the table. Also, integrated mating ability analysed of the hyphae length and amount (<span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/supplementary_fig_s2_dsx031.png?Expires=2147483647&amp;Signature=KIlbX3c6fzYQ2jkuD3wLfIzWQF8JyvU0yqkzgpxWpCDIlD21ADw8MthaJFsehsVErxc8rC1UuvRHw3BTVrWFunr1Su5Rmv2RTh6qdqPkdLfYXR81c-3zWVmXX1Zgpml09qlVUUeFrVyiVi5Tvs7aPOzXUOWV-IRwK05K2iEwvMVTQxNx~BPXdY49ejcoAOaJJXJaiQykUr7UwX8GtasZ1NVFkugoQEn0mPK82DeTFmQpNvkA42JptCOQjjgbiN115b47j-quIDedPeqrUtt67Ij9pR7AL~4DRhqNA-TgbTVPBH7ERxhAczWkibfnYNmMvHTEbRrmt2L5q2~GVtYfHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>) were estimated. Therein, ‘-’ means no hyphae formation; ‘+’ means less hyphae or short hyphae length; ‘ ++’ means normal hyphae amount and length and ‘ +++’ means large amount hyphae and long hyphae length.</p></div></div></div></div></div><div class="table-wrap"><div id="dsx031-T2" data-id="dsx031-T2" class="table-wrap-title"><div class="label -label">Table 2</div><div class="caption"><p>Comparision of the mating response to the macro- or micro-amino acids effect <em>in vitro</em> between <em>U. esculenta</em> and <em>U. maydis</em></p></div></div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="3">No AA </th><th colspan="4"><em>U. esculenta</em><hr /> </th><th colspan="4"><em>U. maydis</em><hr /> </th></tr></thead><tbody><tr><td colspan="4">- </td><td colspan="4">- </td></tr><tr><td colspan="2"><hr />micro-AA </td><td colspan="2"><hr />macro-AA </td><td colspan="2"><hr />micro-AA </td><td colspan="2"><hr />macro-AA </td></tr><tr><td> </td><td> </td><td>Hyphae length (mm) </td><td> </td><td>Hyphae length (mm) </td><td> </td><td>Hyphae length (mm) </td><td> </td><td>Hyphae length (mm) </td></tr><tr><td colspan="9"><hr /> </td></tr><tr><td>Arg </td><td> ++ </td><td>328.77 ± 18.40<sup>gh</sup> </td><td>– </td><td>– </td><td> ++ </td><td>205.81 ± 56.62<sup>ij</sup> </td><td>– </td><td>– </td></tr><tr><td>Tyr </td><td> ++ </td><td>375.08 ± 41.63<sup>defgh</sup> </td><td> +++ </td><td>348.16 ± 17.77<sup>fgh</sup> </td><td> ++ </td><td>102.19 ± 73.26<sup>klm</sup> </td><td> +++ </td><td>354.32 ± 11.65<sup>fgh</sup> </td></tr><tr><td>His </td><td> ++ </td><td>348.54 ± 35.81<sup>fgh</sup> </td><td> +++ </td><td>362.12 ± 15.82<sup>efgh</sup> </td><td>+ </td><td>10.86 ± 1.96<sup>n</sup> </td><td> +++ </td><td>454.41 ± 17.12<sup>bcd</sup> </td></tr><tr><td>Ile </td><td>+ </td><td>338.62 ± 145.33<sup>fgh</sup> </td><td> ++ </td><td>391.33 ± 24.05<sup>defg</sup> </td><td> ++ </td><td>133.01 ± 54.49<sup>jkl</sup> </td><td> +++ </td><td>206.62 ± 21.55<sup>ij</sup> </td></tr><tr><td>Leu </td><td>+ </td><td>196.78 ± 135.4<sup>1j</sup> </td><td>+ </td><td>194.12 ± 56.92<sup>j</sup> </td><td> +++ </td><td>195.51 ± 22.02<sup>j</sup> </td><td> ++ </td><td>158.53 ± 30.17<sup>jk</sup> </td></tr><tr><td>Val </td><td> ++ </td><td>524.01 ± 138.44<sup>ab</sup> </td><td> ++ </td><td>290.63 ± 286.18<sup>hi</sup> </td><td> ++ </td><td>163.55 ± 30.96<sup>jk</sup> </td><td> +++ </td><td>217.12 ± 94.81<sup>ij</sup> </td></tr><tr><td>Met </td><td> +++ </td><td>580.42 ± 36.09<sup>a</sup> </td><td> +++ </td><td>423.34 ± 71.74<sup>cdef</sup> </td><td>+ </td><td>24.38 ± 8.5<sup>1mn</sup> </td><td>– </td><td>– </td></tr><tr><td>Phe </td><td> ++ </td><td>303.42 ± 15.77<sup>gh</sup> </td><td> +++ </td><td>593.88 ± 36.95<sup>a</sup> </td><td>+ </td><td>42.04 ± 11.67<sup>mn</sup> </td><td>+ </td><td>82.33 ± 17.05<sup>klmn</sup> </td></tr><tr><td>Pro </td><td> ++ </td><td>443.20 ± 18.59<sup>bcde</sup> </td><td> +++ </td><td>334.24 ± 46.49<sup>fgh</sup> </td><td>+ </td><td>140.69 ± 39.60<sup>jkl</sup> </td><td>+ </td><td>52.86 ± 26.72<sup>lmn</sup> </td></tr><tr><td>Lys </td><td> +++ </td><td>487.04 ± 35.42<sup>bc</sup> </td><td> ++ </td><td>311.14 ± 16.30<sup>gh</sup> </td><td>+ </td><td>59.13 ± 38.52<sup>lmn</sup> </td><td>– </td><td>– </td></tr></tbody></table></div><div content-id="dsx031-T2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="3">No AA </th><th colspan="4"><em>U. esculenta</em><hr /> </th><th colspan="4"><em>U. maydis</em><hr /> </th></tr></thead><tbody><tr><td colspan="4">- </td><td colspan="4">- </td></tr><tr><td colspan="2"><hr />micro-AA </td><td colspan="2"><hr />macro-AA </td><td colspan="2"><hr />micro-AA </td><td colspan="2"><hr />macro-AA </td></tr><tr><td> </td><td> </td><td>Hyphae length (mm) </td><td> </td><td>Hyphae length (mm) </td><td> </td><td>Hyphae length (mm) </td><td> </td><td>Hyphae length (mm) </td></tr><tr><td colspan="9"><hr /> </td></tr><tr><td>Arg </td><td> ++ </td><td>328.77 ± 18.40<sup>gh</sup> </td><td>– </td><td>– </td><td> ++ </td><td>205.81 ± 56.62<sup>ij</sup> </td><td>– </td><td>– </td></tr><tr><td>Tyr </td><td> ++ </td><td>375.08 ± 41.63<sup>defgh</sup> </td><td> +++ </td><td>348.16 ± 17.77<sup>fgh</sup> </td><td> ++ </td><td>102.19 ± 73.26<sup>klm</sup> </td><td> +++ </td><td>354.32 ± 11.65<sup>fgh</sup> </td></tr><tr><td>His </td><td> ++ </td><td>348.54 ± 35.81<sup>fgh</sup> </td><td> +++ </td><td>362.12 ± 15.82<sup>efgh</sup> </td><td>+ </td><td>10.86 ± 1.96<sup>n</sup> </td><td> +++ </td><td>454.41 ± 17.12<sup>bcd</sup> </td></tr><tr><td>Ile </td><td>+ </td><td>338.62 ± 145.33<sup>fgh</sup> </td><td> ++ </td><td>391.33 ± 24.05<sup>defg</sup> </td><td> ++ </td><td>133.01 ± 54.49<sup>jkl</sup> </td><td> +++ </td><td>206.62 ± 21.55<sup>ij</sup> </td></tr><tr><td>Leu </td><td>+ </td><td>196.78 ± 135.4<sup>1j</sup> </td><td>+ </td><td>194.12 ± 56.92<sup>j</sup> </td><td> +++ </td><td>195.51 ± 22.02<sup>j</sup> </td><td> ++ </td><td>158.53 ± 30.17<sup>jk</sup> </td></tr><tr><td>Val </td><td> ++ </td><td>524.01 ± 138.44<sup>ab</sup> </td><td> ++ </td><td>290.63 ± 286.18<sup>hi</sup> </td><td> ++ </td><td>163.55 ± 30.96<sup>jk</sup> </td><td> +++ </td><td>217.12 ± 94.81<sup>ij</sup> </td></tr><tr><td>Met </td><td> +++ </td><td>580.42 ± 36.09<sup>a</sup> </td><td> +++ </td><td>423.34 ± 71.74<sup>cdef</sup> </td><td>+ </td><td>24.38 ± 8.5<sup>1mn</sup> </td><td>– </td><td>– </td></tr><tr><td>Phe </td><td> ++ </td><td>303.42 ± 15.77<sup>gh</sup> </td><td> +++ </td><td>593.88 ± 36.95<sup>a</sup> </td><td>+ </td><td>42.04 ± 11.67<sup>mn</sup> </td><td>+ </td><td>82.33 ± 17.05<sup>klmn</sup> </td></tr><tr><td>Pro </td><td> ++ </td><td>443.20 ± 18.59<sup>bcde</sup> </td><td> +++ </td><td>334.24 ± 46.49<sup>fgh</sup> </td><td>+ </td><td>140.69 ± 39.60<sup>jkl</sup> </td><td>+ </td><td>52.86 ± 26.72<sup>lmn</sup> </td></tr><tr><td>Lys </td><td> +++ </td><td>487.04 ± 35.42<sup>bc</sup> </td><td> ++ </td><td>311.14 ± 16.30<sup>gh</sup> </td><td>+ </td><td>59.13 ± 38.52<sup>lmn</sup> </td><td>– </td><td>– </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>All the hyphae length data were detected 3 days after mating assays of <em>U. esculenta</em> and <em>U. maydis</em> on basic solid medium with different kind and amount amino acid added. The capital letter as a superscript showed significant differences analysis between edible and grey <em>jiaobai</em> at <em>P</em> &lt; 0.05 level, analysed by one-way ANOVA analysis with Duncan method. All the amino acids (AA) name were abbreviation listed in the table. Also, integrated mating ability analysed of the hyphae length and amount (<span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/supplementary_fig_s2_dsx031.png?Expires=2147483647&amp;Signature=KIlbX3c6fzYQ2jkuD3wLfIzWQF8JyvU0yqkzgpxWpCDIlD21ADw8MthaJFsehsVErxc8rC1UuvRHw3BTVrWFunr1Su5Rmv2RTh6qdqPkdLfYXR81c-3zWVmXX1Zgpml09qlVUUeFrVyiVi5Tvs7aPOzXUOWV-IRwK05K2iEwvMVTQxNx~BPXdY49ejcoAOaJJXJaiQykUr7UwX8GtasZ1NVFkugoQEn0mPK82DeTFmQpNvkA42JptCOQjjgbiN115b47j-quIDedPeqrUtt67Ij9pR7AL~4DRhqNA-TgbTVPBH7ERxhAczWkibfnYNmMvHTEbRrmt2L5q2~GVtYfHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>) were estimated. Therein, ‘-’ means no hyphae formation; ‘+’ means less hyphae or short hyphae length; ‘ ++’ means normal hyphae amount and length and ‘ +++’ means large amount hyphae and long hyphae length.</p></div></div></div></div></p><p>Also, results of mating assays indicated some distinct regulation mechanism of amino acids in <em>U. esculenta</em>. First, in mating progress, <em>U. esculenta</em> may reduce dependence on branched chain amino acids, which were defective in <em>U. esculenta</em> (<a reveal-id="dsx031-F9" data-open="dsx031-F9" class="link link-reveal link-table xref-fig">Fig. 9</a>, <a reveal-id="dsx031-T2" data-open="dsx031-T2" class="link link-reveal link-table xref-fig">Table 2</a>, <span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/supplementary_fig_s2_dsx031.png?Expires=2147483647&amp;Signature=KIlbX3c6fzYQ2jkuD3wLfIzWQF8JyvU0yqkzgpxWpCDIlD21ADw8MthaJFsehsVErxc8rC1UuvRHw3BTVrWFunr1Su5Rmv2RTh6qdqPkdLfYXR81c-3zWVmXX1Zgpml09qlVUUeFrVyiVi5Tvs7aPOzXUOWV-IRwK05K2iEwvMVTQxNx~BPXdY49ejcoAOaJJXJaiQykUr7UwX8GtasZ1NVFkugoQEn0mPK82DeTFmQpNvkA42JptCOQjjgbiN115b47j-quIDedPeqrUtt67Ij9pR7AL~4DRhqNA-TgbTVPBH7ERxhAczWkibfnYNmMvHTEbRrmt2L5q2~GVtYfHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup9" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/supplementary_file_s7_dsx031.xls?Expires=2147483647&amp;Signature=eQHaI44KpmhLgyut6HLGqZAARApSpoFDjjP0pfN~YaV6FXkasaP3CPJ2NTst-kYh2IAs6Uu9sh8f9GPvMb8WogC56BkdUrR2V43NVZj9B1Yd91GuQgoQKnL4fvbuzm7yWQgyRknu7zVOKh7ozAXiDc0Jtk7woLxIC0HXttLs1d4eIXt0ogwyknIr7-kJovXeVes1ZLuA-Z59kxnwH5NfU6RoTpDU-ke58ZBlQg9YYLZzOxt~M-P2qUSlDm4vqAosNN6xdybXQIYtdJ02YqxOCHokGi0M0zKDFdRAaYN4CwcDuqRqccSn9w2~WpeblZeNb4gZRgReQTbvrdKeCyU1tQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">File S7</a></span>). For adding leucine, isoleucine or valine could promote mycelia formation in <em>U. maydis</em> but had little effect on <em>U. esculenta</em> (<a reveal-id="dsx031-T2" data-open="dsx031-T2" class="link link-reveal link-table xref-fig">Table 2</a>, <span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/supplementary_fig_s2_dsx031.png?Expires=2147483647&amp;Signature=KIlbX3c6fzYQ2jkuD3wLfIzWQF8JyvU0yqkzgpxWpCDIlD21ADw8MthaJFsehsVErxc8rC1UuvRHw3BTVrWFunr1Su5Rmv2RTh6qdqPkdLfYXR81c-3zWVmXX1Zgpml09qlVUUeFrVyiVi5Tvs7aPOzXUOWV-IRwK05K2iEwvMVTQxNx~BPXdY49ejcoAOaJJXJaiQykUr7UwX8GtasZ1NVFkugoQEn0mPK82DeTFmQpNvkA42JptCOQjjgbiN115b47j-quIDedPeqrUtt67Ij9pR7AL~4DRhqNA-TgbTVPBH7ERxhAczWkibfnYNmMvHTEbRrmt2L5q2~GVtYfHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>). Second, <em>U. esculenta</em> may adapt to proline, methionine, lysine, phenylalanine and their metabolism, which were closely related to plant disease resistance reaction.<a reveal-id="dsx031-B74 dsx031-B75 dsx031-B76 dsx031-B77 dsx031-B78 dsx031-B79 dsx031-B80" data-open="dsx031-B74 dsx031-B75 dsx031-B76 dsx031-B77 dsx031-B78 dsx031-B79 dsx031-B80" class="link link-ref link-reveal xref-bibr"><sup>74–80</sup></a> In <em>U. maydis</em>, adding each of the four amino acids inhibited the mating progress, which related to fungal pathogenicity, but in <em>U. esculenta</em>, they had conspicuous promotion effect on mycelia formation (<a reveal-id="dsx031-T2" data-open="dsx031-T2" class="link link-reveal link-table xref-fig">Table 2</a>, <span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/supplementary_fig_s2_dsx031.png?Expires=2147483647&amp;Signature=KIlbX3c6fzYQ2jkuD3wLfIzWQF8JyvU0yqkzgpxWpCDIlD21ADw8MthaJFsehsVErxc8rC1UuvRHw3BTVrWFunr1Su5Rmv2RTh6qdqPkdLfYXR81c-3zWVmXX1Zgpml09qlVUUeFrVyiVi5Tvs7aPOzXUOWV-IRwK05K2iEwvMVTQxNx~BPXdY49ejcoAOaJJXJaiQykUr7UwX8GtasZ1NVFkugoQEn0mPK82DeTFmQpNvkA42JptCOQjjgbiN115b47j-quIDedPeqrUtt67Ij9pR7AL~4DRhqNA-TgbTVPBH7ERxhAczWkibfnYNmMvHTEbRrmt2L5q2~GVtYfHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>). Besides, histidine, the reactive oxygen species scavenger,<a reveal-id="dsx031-B81" data-open="dsx031-B81" class="link link-ref link-reveal xref-bibr"><sup>81</sup></a> can also promote the yeast cells fusion in <em>U. esculenta</em>.</p><p>Overall, we speculated that <em>U. esculenta</em> developed a partial complementary mechanism to gain the host’s nutrition while the hosts also utilize some nutrition metabolism pathways to defence infection. In addition, <em>in vitro</em> assays showed that His, Pro, Met, Lys and Phe, which host plant biosynthesized to respond fungus infection, were utilized by <em>U. esculenta</em> to promote its mating progress which was important for its successful infection, indicating that <em>U. esculenta</em> may change some amino acids response mechanism to regulate the balance of infection–resistance engineering for adapting the entophytic life, during 1000 yrs in <em>Z. latifolia</em>.</p>                            <h3 scrollto-destination=104086761 id="104086761" class="section-title" >3.6. Fewer surface sensors and CSEPs benefit economic entophytic life of <em>U. esculenta</em></h3>
<p>The basic requirement for pathogens to infect host is to perceive physical and chemical stimuli when contacting with host surface and then induce to penetrate host either by high turgor pressure, like <em>M. oryzae</em> and <em>Colletotrichum spp.</em>,<a reveal-id="dsx031-B82" data-open="dsx031-B82" class="link link-ref link-reveal xref-bibr"><sup>82</sup></a><sup>,</sup><a reveal-id="dsx031-B83" data-open="dsx031-B83" class="link link-ref link-reveal xref-bibr"><sup>83</sup></a> or by secretion of plant cell wall-degrading enzymes (CWDEs), like <em>U. maydis</em> and <em>Cochliobolus carbonum</em>.<a reveal-id="dsx031-B84" data-open="dsx031-B84" class="link link-ref link-reveal xref-bibr"><sup>84</sup></a><sup>,</sup><a reveal-id="dsx031-B85" data-open="dsx031-B85" class="link link-ref link-reveal xref-bibr"><sup>85</sup></a> During the process, surface sensors, secreted proteins, CAZys and CWDEs are the indispensable factors.</p><p>Sho1 and Msb2 are conserved upstream proteins of MAP kinase cascade not only in the phytopathogenic fungi <em>Fusarium oxysporum</em>, <em>M. oryzae</em> and <em>U. maydis</em> but also in model fungus <em>S. cerevisiae</em>, and human fungal pathogen <em>C. albicans</em>, having been approved to recognize the surface signals, direct the central transcriptional network towards penetration.<a reveal-id="dsx031-B22" data-open="dsx031-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> It is intriguing that there is no <em>Sho1</em> orthologue in <em>U. esculenta</em>. We further investigated 61 genes (42 of which are secreted proteins) related with <em>Sho1/Msb2</em> and also involved in filament and appressoria formation.<a reveal-id="dsx031-B22" data-open="dsx031-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> Forty-two orthologues were found in <em>U. esculenta</em>, and more than 50% of them up-regulated in mycelium when compared with sporidial. Nineteen (17 of which are secreted protein genes) out of 61 genes were not found in <em>U. esculenta</em> (<span class="content-section supplementary-material"><a path-from-xml="sup10" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/supplementary_file_s8_dsx031.xls?Expires=2147483647&amp;Signature=O1QEMkJ3OnFlt5ASQ82bp92hCFKAon9EwYkqKKL~YHSKS8eSMZOZbLTM57MIrPKZmFxycb7HRfUGDKuCicjegavh118w17gXKaZWq~YsLVarGqfpLUOuCV01KTcpBLRHnyFWpSn905mgLfVIFJRer7xW56SWgbVTiN6mppAkSR5fJEWBYiIvDFmI5sN7RTAsel4kD5q3yiv~gnkeIKX4hI8nqlf87g7szGg-lWPZBxYhPeFPTUC7Oydv9yMmIuKJzs66VX39O1UVRvJF2eo6e74z5zc6uwbMTic0s1oBC5bCJ8z6oyGMUWl5RbeE9nyHQCZwLIRmXUu2k9ZuB43NpQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary File S8</a></span>), including some important virulence factors, for example <em>pit2</em> (protease inhibitor in conjunction with Pit1 to maintain biotrophy during plant infection), <em>am1</em> (appressoria marker gene), <em>mig2-5</em>, <em>mig2-6</em>, <em>dkh6</em> (a 7TM protein, which might function in plant/pathogen interaction).<a reveal-id="dsx031-B86" data-open="dsx031-B86" class="link link-ref link-reveal xref-bibr"><sup>86</sup></a></p><p>The whole <em>mig2</em> gene cluster, a maize-specifically expressed cluster, is missing in <em>U. esculenta</em>. The <em>Ustilago scitaminea</em>, pathogen of sugar cane, appears to lack the <em>mig2</em> cluster too.<a reveal-id="dsx031-B87" data-open="dsx031-B87" class="link link-ref link-reveal xref-bibr"><sup>87</sup></a> And interestingly, some <em>U. maydis</em> strains isolated from South America lost the <em>mig2-4</em> and were unable to trigger disease symptoms in regional maize variety.<a reveal-id="dsx031-B87" data-open="dsx031-B87" class="link link-ref link-reveal xref-bibr"><sup>87</sup></a> The <em>mig2</em> cluster was related with host range,<a reveal-id="dsx031-B88" data-open="dsx031-B88" class="link link-ref link-reveal xref-bibr"><sup>88</sup></a> which may partially explain why <em>U. esculenta</em> cannot infect maize and has a narrow host range. Lanver et al.<a reveal-id="dsx031-B22" data-open="dsx031-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> identified 139 plant surface cue-induced secreted proteins in <em>U. maydis</em>. <em>Ustilago esculenta</em> shares 112 these genes including 31 CWDEs and majority of <em>b</em>-induced genes (<span class="content-section supplementary-material"><a path-from-xml="sup11" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/supplementary_file_s9_dsx031.xls?Expires=2147483647&amp;Signature=dzTBy3EHWcbAODpD58irEQwQCJ1yzhfx9Tu~L8HO4nOQu7xyE2jwQAUIOFNThl9PQbsDTvnenU-QBhQOCJs8PkYEw7YWuOWM2qo3yNiQMqm1knOruZWl4uDk8gZ7AV07raCvb8-HzvfA-D8LgnCITGPzFj5-IrCzj141o4EMBBAsLQw6WO2tLKWTs2vcQ7cYV9cTDQBwAVCLxQKVR6quxYlcew7C8LjX7~j99P0BCDnspSSziI2m0g2yr15beFD--yPml-dU-rGX0xA2XxzIndrlUmJyf-IekQAafgkrfEWQk4bR-iSltpmp9XzxO05Ut38258HtGTQQboXwYTGnPw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary File S9</a></span>), loses 27 important secreted protein such as <em>exg1</em>,<a reveal-id="dsx031-B89" data-open="dsx031-B89" class="link link-ref link-reveal xref-bibr"><sup>89</sup></a><sup>,</sup><em>egl1</em> (cellulase gene expressed only in the filament),<a reveal-id="dsx031-B90" data-open="dsx031-B90" class="link link-ref link-reveal xref-bibr"><sup>90</sup></a><sup>,</sup><em>hum2</em> (the former hydrophobin gene in aerial hyphae formation)<a reveal-id="dsx031-B91" data-open="dsx031-B91" class="link link-ref link-reveal xref-bibr"><sup>91</sup></a> and <em>tin2</em> (promotes virulence by targeting anthocyanin biosynthesis).<a reveal-id="dsx031-B92" data-open="dsx031-B92" class="link link-ref link-reveal xref-bibr"><sup>92</sup></a></p><p>Fungi produce a variety of CAZys to facilitate infection and/or gain nutrition. A total of 237 genes including auxiliary activities, glycoside hydrolases (GHs), polysaccharide lyases, carbohydrate esterases (CEs) and glycosyl transferases as well as carbohydrate-binding modules were identified in <em>U. esculenta</em> (<a reveal-id="dsx031-F10" data-open="dsx031-F10" class="link link-reveal link-table xref-fig">Fig. 10</a>). Those genes belonging to the fungal CAZy families are presumably involved in nutritional uptake and infection mechanisms. The <em>U. esculenta</em> genome contains 55 genes (including 9 CEs and 46 GHs) involved in plant cell wall-degradation, which was similar to <em>U. maydis</em>,<a reveal-id="dsx031-B93" data-open="dsx031-B93" class="link link-ref link-reveal xref-bibr"><sup>93</sup></a><sup>,</sup><em>U. hordei</em><a reveal-id="dsx031-B25" data-open="dsx031-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> and <em>S. reilianum</em>,<a reveal-id="dsx031-B26" data-open="dsx031-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> but much fewer than hemibiotrophic and necrotrophic fungi such as <em>M. oryzae</em><a reveal-id="dsx031-B94" data-open="dsx031-B94" class="link link-ref link-reveal xref-bibr"><sup>94</sup></a> and <em>Botrytis cinerea.</em><a reveal-id="dsx031-B95" data-open="dsx031-B95" class="link link-ref link-reveal xref-bibr"><sup>95</sup></a> The compositions of CEs and GHs are similar between the four Ustilagonales species. There is also no significant change in the total number or composition of enzymes coding for the digestion of the three major plant cell wall polysaccharides classes: cellulose, hemicelluloses and pectin (<span class="content-section supplementary-material"><a path-from-xml="sup12" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/supplementary_file_s10_dsx031.xls?Expires=2147483647&amp;Signature=FmcYZbBIWtQAxAf94TKUCoonCEI3xGN~2-TDfHMDM-jQqlgcmL3NzZhQEddDgBaUQehW6IkA4J4HaSexjWwAtJFvcnIYNg9-xtIfvsxxyYA9qy8GYP6InETRIoWLAePTXW503jgBhuxgPjuwH1T71yMDbWjnPM-cJoD-062ZS4hL0STZ1-gEmBq746fErS1FvpZjX1ZSAisuZg5LwothJ7lQcHIcdxysBFVvAGDbIM3PTM-3i0Va3lNXuZaujhFj1SpJMlQmmjCv2GJPMwM0nLyL1kXczuH3PZwQ9FMoK2FFS~klUhRbcRV5KB-6vrnJ3HJ8cHkq-60Vo8GRJwNcmQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary File S10</a></span>). </p>                        <a id="104086766" scrollto-destination="104086766"></a>
<div id="dsx031-F10" data-id="dsx031-F10" class="fig fig-section"><div class="label fig-label">Figure 10</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/m_dsx031f10.png?Expires=2147483647&Signature=GCCtnF-VJ1LuJM0y4HlpnwJFyzxCJOZlNkzz7mTANruAEcxQwF4YlXHIjetSidrh-cStbmmxuQQS4~vApeF1PSzopqp91fcr0EIAOxIyDJlU8vuRIncfsFkpdNzKV-qHihX0e-pqDkNIYFzaHMeAO~GnsSirxvoSfsRwe1im9tAHl2nwTOhJKll2BL3dq1WBUno3AnRkl3H56ZxBqY-aQTDMVQyG-L7S7vYc89FUbSFEYCHqsgs-oTEpy25fSzqCCoRYozJ5exUteabp9iUnk42Qx~qsSpbT~SrphzGW8-psP-Vhvp58Z01jSjxyv5u6Wjq9p4uY3OyIXbFPGsdENg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="CAZy compositions of the four Ustilagoinaceae species. The numbers represent the number of genes encoded of each CAZy category. The CAzy categories are composed of AAs: auxiliary activities; CBMs: carbohydrate-binding modules; CEs: carbohydrate esterases; GH: glycoside hydrolase; GTs: glycosyl transferases and PLs: polysaccharide lyases." path-from-xml="dsx031f10.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086766/dsx031f10.tif" path-from-xml="dsx031f10.tif" target="_blank">View large</a><a section="104086766" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/dsx031f10.png?Expires=2147483647&Signature=XerRb68eNwJLGV7gcPtDrFffNTkaBEjejc8sao5x9lXLpbHPMEXMamhx1K0y5-TU9oc9LaxmZoLTQBsLClGYjVanOb68WwnWQ1X7BYOIRoC8byEAAYxpeOx6aud2FsvRlVfg2PAJrEkBS9wufjVI93SxuTq5Ex9cE87hWvZDgbUe~yAAyPlyfW6es-CCky9TEukgLUVIQ5emzctI18eDz3kBfSqUl9IMEn6LGF-odTm0FtImNI3T1eAz46~6xJHO4m21Jc5o6SqBQI2-cehX9bgWta3Mg7s1LOreaStNh9lFUoLLI4Elt-78RKl0HHAgpRmGO9~pvEnRIQhYtfz4ZQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086766&ar=3979487&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx031f10.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>CAZy compositions of the four Ustilagoinaceae species. The numbers represent the number of genes encoded of each CAZy category. The CAzy categories are composed of AAs: auxiliary activities; CBMs: carbohydrate-binding modules; CEs: carbohydrate esterases; GH: glycoside hydrolase; GTs: glycosyl transferases and PLs: polysaccharide lyases.</p></div></div><div content-id="dsx031-F10" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 10</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/m_dsx031f10.png?Expires=2147483647&Signature=GCCtnF-VJ1LuJM0y4HlpnwJFyzxCJOZlNkzz7mTANruAEcxQwF4YlXHIjetSidrh-cStbmmxuQQS4~vApeF1PSzopqp91fcr0EIAOxIyDJlU8vuRIncfsFkpdNzKV-qHihX0e-pqDkNIYFzaHMeAO~GnsSirxvoSfsRwe1im9tAHl2nwTOhJKll2BL3dq1WBUno3AnRkl3H56ZxBqY-aQTDMVQyG-L7S7vYc89FUbSFEYCHqsgs-oTEpy25fSzqCCoRYozJ5exUteabp9iUnk42Qx~qsSpbT~SrphzGW8-psP-Vhvp58Z01jSjxyv5u6Wjq9p4uY3OyIXbFPGsdENg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="CAZy compositions of the four Ustilagoinaceae species. The numbers represent the number of genes encoded of each CAZy category. The CAzy categories are composed of AAs: auxiliary activities; CBMs: carbohydrate-binding modules; CEs: carbohydrate esterases; GH: glycoside hydrolase; GTs: glycosyl transferases and PLs: polysaccharide lyases." path-from-xml="dsx031f10.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086766/dsx031f10.tif" path-from-xml="dsx031f10.tif" target="_blank">View large</a><a section="104086766" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/dsx031f10.png?Expires=2147483647&Signature=XerRb68eNwJLGV7gcPtDrFffNTkaBEjejc8sao5x9lXLpbHPMEXMamhx1K0y5-TU9oc9LaxmZoLTQBsLClGYjVanOb68WwnWQ1X7BYOIRoC8byEAAYxpeOx6aud2FsvRlVfg2PAJrEkBS9wufjVI93SxuTq5Ex9cE87hWvZDgbUe~yAAyPlyfW6es-CCky9TEukgLUVIQ5emzctI18eDz3kBfSqUl9IMEn6LGF-odTm0FtImNI3T1eAz46~6xJHO4m21Jc5o6SqBQI2-cehX9bgWta3Mg7s1LOreaStNh9lFUoLLI4Elt-78RKl0HHAgpRmGO9~pvEnRIQhYtfz4ZQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086766&ar=3979487&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx031f10.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>CAZy compositions of the four Ustilagoinaceae species. The numbers represent the number of genes encoded of each CAZy category. The CAzy categories are composed of AAs: auxiliary activities; CBMs: carbohydrate-binding modules; CEs: carbohydrate esterases; GH: glycoside hydrolase; GTs: glycosyl transferases and PLs: polysaccharide lyases.</p></div></div><p>Effector proteins, which are typically secreted by the pathogen after contacting with host, have been recognized to govern the interaction between plants and biotrophic pathogens. Totally, 633 proteins were identified as secreted proteins. Of these secreted proteins, 300 were predicted to be candidate secreted effector proteins (CSEPs) since they could not be assigned to enzymatic functions. Compared with <em>U. maydis</em> and <em>U. hordei</em>, the genome of <em>U. esculenta</em> has nearly 20% more predicted secreted proteins and a 10∼20% fewer predicted CSEPs (554 secreted proteins and 386 CSEPs for <em>U. maydis</em>. 515 secreted proteins and 333 CSEPs for <em>U. hordei</em>).<a reveal-id="dsx031-B25" data-open="dsx031-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> Ninety-two (15%) of 633 <em>U. esculenta</em> secreted proteins were species specific. Among them, 87 were CSEPs which were 29.6% of total predicted CSEPs. Four hundred and fifteen of 633 (66%) <em>U. esculenta</em> secreted proteins are shared among <em>U. maydis</em>, <em>U. hordei</em> and <em>S. reilianum</em> (<span class="content-section supplementary-material"><a path-from-xml="sup19" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/supplementary_table_s5_dsx031.doc?Expires=2147483647&amp;Signature=SCKCjLmgMyQu7dN5bHBjv4xyxZwJ9C-tjUmi9E1vJYQ8CR2SNwJAw-ofVpyjMtO6CiAd67Kk-OZMuHcEB4gFh2hDCAKO5vzqNHsmFKp9JOxwWv7W8UZsJD19cnmijWcrkVe9TGo~vr4z~4PLRaF~vfyWrBz4JyXuViPj6Hgp6i2ZkRJInWfX68FDndyRP3N378PP6rjtB6d-CSAmMo5WpgQLzwCOf1mEIpti6roIM1Bzw0pMePsy1q1KTqUJLT0FDThlusOBYpB5tuhi7SOKV6Xq4R~GYXMP8xxJK479LW84yvBKcLV9NSCqbnnIcjVh86u4MHI4oe~ipUPCuQOZcg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>). Of them, 97 secreted proteins could be matched in pathogen–host interaction database (PHI-database)<a reveal-id="dsx031-B96" data-open="dsx031-B96" class="link link-ref link-reveal xref-bibr"><sup>96</sup></a> with an amino acid identity &gt;30%. Of all PHI hits, 55 out of total proteins were characterized as increased or reduced virulence. The remaining hits were either characterized as unaffected virulence or loss of pathogenicity.</p><p>There were 57 (11%) secreted proteins assigned to 14 clusters in <em>U. esculenta</em>, in comparison to 21, 7 and 22 clusters in <em>U. maydis</em>, in <em>U. hordei</em> and in <em>S. reilianum</em>. Seventy-six percent of Ue genes in clusters had orthologue Um genes, with amino acid identities ranging from 23.3 to 94.2%. Among the 14 clusters identified in this study, seven clusters were previously described Um effector gene clusters (<span class="content-section supplementary-material"><a path-from-xml="sup4" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/supplementary_file_s2_dsx031.xls?Expires=2147483647&Signature=C3nu5QEfgdRa~jCnGIVFbNCudwHZzuQ7Db-iD5Bi9-v5EMf2lI9s1NMQGFQV-NKFiySF1jNDyb5AYBxYTvDDPgk4SIaoWZOezeO2qqk81f5AjhnHwP8OQ4H9b-i7BJ1Cm~Uxw47idENq2jeN5CSL1curf~q3d0lzGcdk~wR3-E6dfpPP4T~TT2a-6KDrxBHJNROlKLDT8r1Uc7GSW9aAFgCi~AXUKpCXgOn7qC4m-4rQ4L-d7E2CtP~tPxxfr2EzkmkwJOZiSiHw~QC8gH1hqHiYv7glLevmTtBgJ7xsnipcOtTmbhpijvMzyZup~PHgXr2Tc-9Vxw-Z2XQV2K2sXg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary File S2</a></span>), including all clusters whose deletion reduced virulence or non-pathogenic. Fungal secreted proteins are assumed to contribute to tumour formation,<a reveal-id="dsx031-B90" data-open="dsx031-B90" class="link link-ref link-reveal xref-bibr"><sup>90</sup></a> and organ-specific effectors in both host and pathogen are required for host tumour formations.<a reveal-id="dsx031-B24" data-open="dsx031-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> In <em>U. esculenta</em>, the homologues with um3615 and um3616 secreted proteins were found, which were important genes of cluster 9A for tumour symptoms in adult tissue. When considering the responsible gene cluster 19A for tumour formation in <em>U. maydis</em> seedlings, only 4 out of 15 homologues of cluster 19A were present in <em>U. esculenta</em>. It may be the reason that <em>U. esculenta</em> could not trigger gall formation during seedlings. And more attention should be paid to find stem-specific gene clusters.</p><p>Besides, when we compared 92 CSEPs being important for successfully colonization of grass hosts and a core set of 248 CSEPs needed for pathogenicity (PSEPs) which were concluded from genome comparison among smut fungi <em>M. pennsylvanicum, U. maydis, U. hordei</em> and <em>S. reilianum</em>,<a reveal-id="dsx031-B16" data-open="dsx031-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> 55 grass host-related CSEPs and 182 PSEPs can find orthologues in <em>U. esculenta</em> (<span class="content-section supplementary-material"><a path-from-xml="sup13" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/supplementary_file_s11_dsx031.xls?Expires=2147483647&amp;Signature=QmnDxGZR55w2PZ8yTcg3Ym-0ty99sTDbMA7Zk47FsYAuQFAPBOVzsPQI~tPrnYft5Vox2khUuFDuiGIIxAWRVm7Eg-HR4IN5jOMT~s8OcJJi-NPYBaG5PS5F1SwHiL1HRcpcu~~gN73bjgya7MaqVIAEeMUz6VMKEDNNetDobLEmdYl3Pa4BXpR7BrfdCSKUejU5t2Jdl3gmKN0x3Ya08fTd2s9cj4sIdgw4KLMHb6mZIlsD4Np3MBWx1f-Ic1As1y7ZDyB~weywQ9F99UErIo-q1KWV~w2-rgC~pSm2nTvVa2JlDz0ehkKx3TBHVhy9-RsACFU8WlZK4UkOKWoH6A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Files S11</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup14" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/supplementary_file_s12_dsx031.xlsx?Expires=2147483647&amp;Signature=GLeuw~IZs~qXLUV1eoQSnZXi6QyEfwXM9mL8UjcewXluDdiruGYoBaD3onJUOT2XzFL6f0qCTICnk9U1tMnMJDOcoCrC6vy-nez63aJY-FuZv3zsNt5ZkgGInGZ2ahphsmluHYvKM2~5c0yRp~b9JRp2gZuB176zqRF40ZYFTk9JsrWGzsswfKO9Yso0BUA7y6ck28klTUBJWlPVe1XxYsICgMyZQYLmk3kOFQjQvjGlC9CMcRrqL0uIj48VUiuuC3jClOnHtTCF2uRACTg1MAbVlaEGDrxeplsPQFEV7df3zPImGFbHUS0~Uwmd0Fb22iE15w9xhba67VUHIT7m1w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">S12</a></span>). But it is remarkable that some important host-related CSEPs, including um03223 (maize-induced gene <em>mig1</em>), um12216 (related to Mig1 proteins) and 9 Ustilago-specific proteins, were not found in <em>U. esculenta</em> (<span class="content-section supplementary-material"><a path-from-xml="sup13" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/supplementary_file_s11_dsx031.xls?Expires=2147483647&amp;Signature=QmnDxGZR55w2PZ8yTcg3Ym-0ty99sTDbMA7Zk47FsYAuQFAPBOVzsPQI~tPrnYft5Vox2khUuFDuiGIIxAWRVm7Eg-HR4IN5jOMT~s8OcJJi-NPYBaG5PS5F1SwHiL1HRcpcu~~gN73bjgya7MaqVIAEeMUz6VMKEDNNetDobLEmdYl3Pa4BXpR7BrfdCSKUejU5t2Jdl3gmKN0x3Ya08fTd2s9cj4sIdgw4KLMHb6mZIlsD4Np3MBWx1f-Ic1As1y7ZDyB~weywQ9F99UErIo-q1KWV~w2-rgC~pSm2nTvVa2JlDz0ehkKx3TBHVhy9-RsACFU8WlZK4UkOKWoH6A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary File S11</a></span>). Furthermore, nearly 95% of <em>U. esculenta</em>-specific secreted proteins were CSEPs, which was one-third of total CSEPs. Missing of host-related CSEPs can further confirmed the narrow host range of <em>U. esculenta</em>, and species-specific CSEPs may give more hints in explanation. It has been proposed that effector genes, as speciation genes in fungal plant pathogens,<a reveal-id="dsx031-B97" data-open="dsx031-B97" class="link link-ref link-reveal xref-bibr"><sup>97</sup></a> co-evolve with their host targets and reflect the host adaptation.<a reveal-id="dsx031-B98" data-open="dsx031-B98" class="link link-ref link-reveal xref-bibr"><sup>98</sup></a></p><p>All in all, although the <em>U. esculenta</em> shared core set of secreted proteins with some smut fungi indicating the close relation, the fungus has evolved distinguish characters of fewer surface sensors, high percentage of species-specific CSEPs and fewer CSEPs, lost some important virulence factors and host range-related effectors. All these characters may be the nature need of maintaining lower cost endophytic life and limitation of small genome size, which was the obliged evolution under long time pressure on keeping the mycelia phase and making no need to re-infect the host.</p>                            <h2 scrollto-destination=104086771 id="104086771" class="backacknowledgements-title" >Acknowledgements</h2>
<p>The authors thank Ying Rong for the preparation of the fungus collection, Cao Qianchao for mating assay, Gui Yijie for analysing assistant with host gene expression, Michael Feldbrügge for providing plasmids. They are grateful to Vera Göhre and Michael Feldbrügge for their critical comments on the article.</p>                            <h3 scrollto-destination=104086773 id="104086773" class="section-title" >Accession number</h3>
<p>EMBARGO: No</p>                            <h3 scrollto-destination=104086775 id="104086775" class="section-title" >Supplementary data</h3>
<p><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/supplementary_fig_s1_dsx031.png?Expires=2147483647&Signature=CnZcTvYT3i5XWT6EKbc3889ymAipNGHbJZMGyiU6FhWEy9eSfdWzzEO6Hv9wGkPNork~xvf0RMTJUs7FHJYzKePGrKXp0pUPW6mp314fPHnpbAvW1wRiWignpeQEpNvMxNKEc95XD08FS-FV92jsVgq62~B4vQnqFwNl9~mba-yxWJFVh3ni9eM~YSuwW4HsZ~EqVebMcNLp1PN4kt9fb8itQygLZ~x9CIrbATxzrs8u~hjoTpcDnF20iucLqbMrAMvHLBIr6wo-x967Y2wI9xB2PyQ6SR1oy7ehm-qvt4L8RLweo5-uKWgpriRxT7tEztMCPYGcgR0eSQNNssYNfQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary data</a></span> are available at <em>DNARES</em> online.</p>                            <h3 scrollto-destination=104086777 id="104086777" class="section-title" >Funding</h3>
<p>Our work was supported by funds from the National Natural Science Foundation of China (31470785, 31600634). The funders had no role in study design, data collection and analysis, decision to publish or preparation of the article.</p>                            <h3 scrollto-destination=104086779 id="104086779" class="section-title" >Conflict of interest</h3>
<p>None declared.</p>                            <h2 scrollto-destination=104086781 id="104086781" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsx031-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chung</div> <div class="given-names">K.R.</div></div>, <div class="name string-name"><div class="surname">Tzeng</div> <div class="given-names">D.D.</div></div></div> <div class="year">2004</div>, 
<div class="article-title">Nutritional requirements of the edible gall-producing fungus <em>Ustilago esculenta</em>,</div> <div class="source">J. Biol. Sci</div> ., <div class="volume">4</div>, <div class="fpage">246</div>–<div class="lpage">52</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nutritional%20requirements%20of%20the%20edible%20gall-producing%20fungus%20Ustilago%20esculenta%2C&amp;author=K.R.%20Chung&amp;author=D.D.%20Tzeng&amp;publication_year=2004&amp;journal=J.%20Biol.%20Sci&amp;volume=4&amp;pages=246-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3923/jbs.2004.246.252" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3923%2Fjbs.2004.246.252" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3923%2Fjbs.2004.246.252"> </span></div> </div></div></div></div></div><div content-id="dsx031-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Xu</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Walters</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Antolin</div> <div class="given-names">M.F.</div></div></div>, et al.   <div class="year">2010</div>, 
<div class="article-title">Phylogeny and biogeography of the eastern Asian-North American disjunct wild-rice genus (<em>Zizania L.</em>, Poaceae),</div> <div class="source">Mol. Phylogenet. Evol</div> ., <div class="volume">55</div>, <div class="fpage">1008</div>–<div class="lpage">17</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phylogeny%20and%20biogeography%20of%20the%20eastern%20Asian-North%20American%20disjunct%20wild-rice%20genus%20(Zizania%20L.%2C%20Poaceae)%2C&amp;author=X.%20Xu&amp;author=C.%20Walters&amp;author=M.F.%20Antolin&amp;publication_year=2010&amp;journal=Mol.%20Phylogenet.%20Evol&amp;volume=55&amp;pages=1008-17" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ympev.2009.11.018" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ympev.2009.11.018" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ympev.2009.11.018"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19944174" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx031-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Guo</div> <div class="given-names">H.B.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">S.M.</div></div>, <div class="name string-name"><div class="surname">Peng</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Ke</div> <div class="given-names">W.D.</div></div></div> <div class="year">2007</div>, 
<div class="article-title"><em>Zizania latifolia</em> Turcz. cultivated in China,</div> <div class="source">Genet. Resour. Crop Evol</div> ., <div class="volume">54</div>, <div class="fpage">1211</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Zizania%20latifolia%20Turcz.%20cultivated%20in%20China%2C&amp;author=H.B.%20Guo&amp;author=S.M.%20Li&amp;author=J.%20Peng&amp;author=W.D.%20Ke&amp;publication_year=2007&amp;journal=Genet.%20Resour.%20Crop%20Evol&amp;volume=54&amp;pages=1211-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10722-006-9102-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10722-006-9102-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10722-006-9102-8"> </span></div> </div></div></div></div></div><div content-id="dsx031-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wang</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Huang</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Fan</div> <div class="given-names">L.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Main agronomic traits, domestication and breeding of Gu (<em>Zizania latifolia</em>),</div> <div class="source">J. Zhejiang Univ</div> ., <div class="volume">39</div>, <div class="fpage">629</div>–<div class="lpage">35</div>.</div></div></div></div><div content-id="dsx031-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yang</div> <div class="given-names">H.C.</div></div>, <div class="name string-name"><div class="surname">Leu</div> <div class="given-names">L.S.</div></div></div> <div class="year">1978</div>, 
<div class="article-title">Formation and histopathology of galls induced by <em>Ustilage esculenta</em> in <em>Zizania latifolia</em>,</div> <div class="source">Phytopathology</div> , <div class="volume">68</div>, <div class="fpage">1572</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Formation%20and%20histopathology%20of%20galls%20induced%20by%20Ustilage%20esculenta%20in%20Zizania%20latifolia%2C&amp;author=H.C.%20Yang&amp;author=L.S.%20Leu&amp;publication_year=1978&amp;journal=Phytopathology&amp;volume=68&amp;pages=1572-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1094/Phyto-68-1572" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1094%2FPhyto-68-1572" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1094%2FPhyto-68-1572"> </span></div> </div></div></div></div></div><div content-id="dsx031-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Qian</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Luo</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Deng</div> <div class="given-names">Y.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">Chemical composition, angiotensin-converting enzyme-inhibitory activity and antioxidant activities of few-flower wild rice (<em>Zizania latifolia</em> Turcz.),</div> <div class="source">J. Sci. Food Agric</div> ., <div class="volume">92</div>, <div class="fpage">159</div>–<div class="lpage">64</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Chemical%20composition%2C%20angiotensin-converting%20enzyme-inhibitory%20activity%20and%20antioxidant%20activities%20of%20few-flower%20wild%20rice%20(Zizania%20latifolia%20Turcz.)%2C&amp;author=B.%20Qian&amp;author=Y.%20Luo&amp;author=Y.%20Deng&amp;publication_year=2012&amp;journal=J.%20Sci.%20Food%20Agric&amp;volume=92&amp;pages=159-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/jsfa.v92.1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fjsfa.v92.1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Fjsfa.v92.1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21815157" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx031-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yan</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">X.Q.</div></div>, <div class="name string-name"><div class="surname">Xu</div> <div class="given-names">X.F.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">Plant growth and photosynthetic performance of <em>Zizania latifolia</em> are altered by endophytic <em>Ustilago esculenta</em> infection,</div> <div class="source">Physiol. Mol. Plant Pathol</div> ., <div class="volume">83</div>, <div class="fpage">75</div>–<div class="lpage">83</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Plant%20growth%20and%20photosynthetic%20performance%20of%20Zizania%20latifolia%20are%20altered%20by%20endophytic%20Ustilago%20esculenta%20infection%2C&amp;author=N.%20Yan&amp;author=X.Q.%20Wang&amp;author=X.F.%20Xu&amp;publication_year=2013&amp;journal=Physiol.%20Mol.%20Plant%20Pathol&amp;volume=83&amp;pages=75-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.pmpp.2013.05.005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.pmpp.2013.05.005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.pmpp.2013.05.005"> </span></div> </div></div></div></div></div><div content-id="dsx031-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation">Dong, S.Z., Z.G., Yu, X.P., Fu, C.H. <div class="year">2012</div>, 
<div class="article-title">Highly efficient and safe production of <em>Jiaobai</em>,</div> <div class="source">Sci. Agric</div> ., <div class="volume">69</div>, <div class="fpage">142</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1590/S0103-90162012000200009" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1590%2FS0103-90162012000200009" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1590%2FS0103-90162012000200009"> </span></div> </div></div></div></div></div><div content-id="dsx031-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Couch</div> <div class="given-names">B.C.</div></div>, <div class="name string-name"><div class="surname">Fudal</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Lebrun</div> <div class="given-names">M.H.</div></div></div>, et al.   <div class="year">2005</div>, 
<div class="article-title">Origins of host-specific populations of the blast pathogen <em>Magnaporthe oryzae</em> in crop domestication with subsequent expansion of pandemic clones on rice and weeds of rice,</div> <div class="source">Genetics</div> , <div class="volume">170</div>, <div class="fpage">613</div>–<div class="lpage">30</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Origins%20of%20host-specific%20populations%20of%20the%20blast%20pathogen%20Magnaporthe%20oryzae%20in%20crop%20domestication%20with%20subsequent%20expansion%20of%20pandemic%20clones%20on%20rice%20and%20weeds%20of%20rice%2C&amp;author=B.C.%20Couch&amp;author=I.%20Fudal&amp;author=M.H.%20Lebrun&amp;publication_year=2005&amp;journal=Genetics&amp;volume=170&amp;pages=613-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.105.041780" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.105.041780" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.105.041780"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15802503" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx031-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Stukenbrock</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Banke</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Javan-Nikkhah</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Mcdonald</div> <div class="given-names">B.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">Origin and domestication of the fungal wheat pathogen <em>Mycosphaerella graminicola</em> via sympatric speciation,</div> <div class="source">Mol. Biol. Evol</div> ., <div class="volume">24</div>, <div class="fpage">398</div>–<div class="lpage">411</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Origin%20and%20domestication%20of%20the%20fungal%20wheat%20pathogen%20Mycosphaerella%20graminicola%20via%20sympatric%20speciation%2C&amp;author=E.%20Stukenbrock&amp;author=S.%20Banke&amp;author=M.%20Javan-Nikkhah&amp;author=B.%20Mcdonald&amp;publication_year=2007&amp;journal=Mol.%20Biol.%20Evol&amp;volume=24&amp;pages=398-411" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msl169" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsl169" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsl169"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17095534" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx031-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Olsen</div> <div class="given-names">K.M.</div></div>, <div class="name string-name"><div class="surname">Wendel</div> <div class="given-names">J.F.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">A bountiful harvest: genomic insights into crop domestication phenotypes,</div> <div class="source">Annu. Rev. Plant Biol</div> ., <div class="volume">64</div>, <div class="fpage">47</div>–<div class="lpage">70</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20bountiful%20harvest%3A%20genomic%20insights%20into%20crop%20domestication%20phenotypes%2C&amp;author=K.M.%20Olsen&amp;author=J.F.%20Wendel&amp;publication_year=2013&amp;journal=Annu.%20Rev.%20Plant%20Biol&amp;volume=64&amp;pages=47-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev-arplant-050312-120048" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev-arplant-050312-120048" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev-arplant-050312-120048"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23451788" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx031-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chen</div> <div class="given-names">Y.H.</div></div>, <div class="name string-name"><div class="surname">Gols</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Benrey</div> <div class="given-names">B.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Crop domestication and its impact on naturally selected trophic interactions,</div> <div class="source">Annu. Rev. Entomol</div> ., <div class="volume">60</div>, <div class="fpage">35</div>–<div class="lpage">58</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Crop%20domestication%20and%20its%20impact%20on%20naturally%20selected%20trophic%20interactions%2C&amp;author=Y.H.%20Chen&amp;author=R.%20Gols&amp;author=B.%20Benrey&amp;publication_year=2015&amp;journal=Annu.%20Rev.%20Entomol&amp;volume=60&amp;pages=35-58" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev-ento-010814-020601" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev-ento-010814-020601" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev-ento-010814-020601"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25341108" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx031-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Stukenbrock</div> <div class="given-names">E.H.</div></div>, <div class="name string-name"><div class="surname">Bataillon</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Dutheil</div> <div class="given-names">J.Y.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">The making of a new pathogen: insights from comparative population genomics of the domesticated wheat pathogen <em>Mycosphaerella graminicola</em> and its wild sister species,</div> <div class="source">Genome Res</div> ., <div class="volume">21</div>, <div class="fpage">2157</div>–<div class="lpage">66</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20making%20of%20a%20new%20pathogen%3A%20insights%20from%20comparative%20population%20genomics%20of%20the%20domesticated%20wheat%20pathogen%20Mycosphaerella%20graminicola%20and%20its%20wild%20sister%20species%2C&amp;author=E.H.%20Stukenbrock&amp;author=T.%20Bataillon&amp;author=J.Y.%20Dutheil&amp;publication_year=2011&amp;journal=Genome%20Res&amp;volume=21&amp;pages=2157-66" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.118851.110" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.118851.110" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.118851.110"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21994252" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx031-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Xue</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Yang</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">Z.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">Comparative analysis of the genomes of two field isolates of the rice blast fungus <em>Magnaporthe oryzae</em>,</div> <div class="source">PLoS Genet</div> ., <div class="volume">8</div>, <div class="fpage">e1002869</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20analysis%20of%20the%20genomes%20of%20two%20field%20isolates%20of%20the%20rice%20blast%20fungus%20Magnaporthe%20oryzae%2C&amp;author=M.%20Xue&amp;author=J.%20Yang&amp;author=Z.%20Li&amp;publication_year=2012&amp;journal=PLoS%20Genet&amp;volume=8&amp;pages=e1002869" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1002869" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1002869" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1002869"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22876203" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx031-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Stukenbrock</div> <div class="given-names">E.H.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Evolution, selection and isolation: a genomic view of speciation in fungal plant pathogens,</div> <div class="source">New Phytol</div> ., <div class="volume">199</div>, <div class="fpage">895</div>–<div class="lpage">907</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%2C%20selection%20and%20isolation%3A%20a%20genomic%20view%20of%20speciation%20in%20fungal%20plant%20pathogens%2C&amp;author=E.H.%20Stukenbrock&amp;publication_year=2013&amp;journal=New%20Phytol&amp;volume=199&amp;pages=895-907" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/nph.12374" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fnph.12374" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fnph.12374"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23782262" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx031-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sharma</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Mishra</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Runge</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Thines</div> <div class="given-names">M.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Gene loss rather than gene gain is associated with a host jump from monocots to dicots in the Smut Fungus <em>Melanopsichium pennsylvanicum</em>,</div> <div class="source">Genome Biol. Evol</div> ., <div class="volume">6</div>, <div class="fpage">2034</div>–<div class="lpage">49</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20loss%20rather%20than%20gene%20gain%20is%20associated%20with%20a%20host%20jump%20from%20monocots%20to%20dicots%20in%20the%20Smut%20Fungus%20Melanopsichium%20pennsylvanicum%2C&amp;author=R.%20Sharma&amp;author=B.%20Mishra&amp;author=F.%20Runge&amp;author=M.%20Thines&amp;publication_year=2014&amp;journal=Genome%20Biol.%20Evol&amp;volume=6&amp;pages=2034-49" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/gbe/evu148" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fgbe%2Fevu148" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fgbe%2Fevu148"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25062916" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx031-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Heinze</div> <div class="given-names">B.</div></div></div> <div class="year">2009</div>, Comparative analysis of the Maize Smut Fungi <em>Ustilago maydis</em> and <em>Sporisorium reilianum</em>. Philipps-Universität Marburg, Germany.</div></div></div></div><div content-id="dsx031-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Xu</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Ke</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Yu</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Wen</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Ge</div> <div class="given-names">S.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">A preliminary study on population genetic structure and phylogeography of the wild and cultivated <em>Zizania latifolia</em> (Poaceae) based on Adh1a sequences,</div> <div class="source">Theor. Appl. Genet</div> ., <div class="volume">116</div>, <div class="fpage">835</div>–<div class="lpage">43</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20preliminary%20study%20on%20population%20genetic%20structure%20and%20phylogeography%20of%20the%20wild%20and%20cultivated%20Zizania%20latifolia%20(Poaceae)%20based%20on%20Adh1a%20sequences%2C&amp;author=X.%20Xu&amp;author=W.%20Ke&amp;author=X.%20Yu&amp;author=J.%20Wen&amp;author=S.%20Ge&amp;publication_year=2008&amp;journal=Theor.%20Appl.%20Genet&amp;volume=116&amp;pages=835-43" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-008-0717-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-008-0717-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-008-0717-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18283426" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx031-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lanver</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Mendoza-Mendoza</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Brachmann</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Kahmann</div> <div class="given-names">R.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Sho1 and Msb2-related proteins regulate appressorium development in the smut fungus <em>Ustilago maydis</em>,</div> <div class="source">Plant Cell</div> , <div class="volume">22</div>, <div class="fpage">2085</div>–<div class="lpage">101</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sho1%20and%20Msb2-related%20proteins%20regulate%20appressorium%20development%20in%20the%20smut%20fungus%20Ustilago%20maydis%2C&amp;author=D.%20Lanver&amp;author=A.%20Mendoza-Mendoza&amp;author=A.%20Brachmann&amp;author=R.%20Kahmann&amp;publication_year=2010&amp;journal=Plant%20Cell&amp;volume=22&amp;pages=2085-101" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.109.073734" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.109.073734" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.109.073734"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20587773" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx031-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Doehlemann</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Berndt</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Hahn</div> <div class="given-names">M.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">Different signalling pathways involving a G alpha protein, cAMP and a MAP kinase control germination of <em>Botrytis cinerea</em> conidia,</div> <div class="source">Mol. Microbiol</div> ., <div class="volume">59</div>, <div class="fpage">821</div>–<div class="lpage">35</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Different%20signalling%20pathways%20involving%20a%20G%20alpha%20protein%2C%20cAMP%20and%20a%20MAP%20kinase%20control%20germination%20of%20Botrytis%20cinerea%20conidia%2C&amp;author=G.%20Doehlemann&amp;author=P.%20Berndt&amp;author=M.%20Hahn&amp;publication_year=2006&amp;journal=Mol.%20Microbiol&amp;volume=59&amp;pages=821-35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mmi.2006.59.issue-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmmi.2006.59.issue-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmmi.2006.59.issue-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16420354" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx031-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Doehlemann</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Wahl</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Horst</div> <div class="given-names">R.J.</div></div></div>, et al.   <div class="year">2008</div>, 
<div class="article-title">Reprogramming a maize plant: transcriptional and metabolic changes induced by the fungal biotroph <em>Ustilago maydis</em>,</div> <div class="source">Plant J</div> ., <div class="volume">56</div>, <div class="fpage">181</div>–<div class="lpage">95</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Reprogramming%20a%20maize%20plant%3A%20transcriptional%20and%20metabolic%20changes%20induced%20by%20the%20fungal%20biotroph%20Ustilago%20maydis%2C&amp;author=G.%20Doehlemann&amp;author=R.%20Wahl&amp;author=R.J.%20Horst&amp;publication_year=2008&amp;journal=Plant%20J&amp;volume=56&amp;pages=181-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.2008.56.issue-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2008.56.issue-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.2008.56.issue-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18564380" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx031-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lanver</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Berndt</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Tollot</div> <div class="given-names">M.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Plant surface cues prime <em>Ustilago maydis</em> for biotrophic development,</div> <div class="source">PLoS Pathog</div> ., <div class="volume">10</div>, <div class="fpage">295</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Plant%20surface%20cues%20prime%20Ustilago%20maydis%20for%20biotrophic%20development%2C&amp;author=D.%20Lanver&amp;author=P.%20Berndt&amp;author=M.%20Tollot&amp;publication_year=2014&amp;journal=PLoS%20Pathog&amp;volume=10&amp;pages=295" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.ppat.1004272" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.ppat.1004272" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.ppat.1004272"> </span></div> </div></div></div></div></div><div content-id="dsx031-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Brefort</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Tanaka</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Neidig</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Doehlemann</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Vincon</div> <div class="given-names">V.</div></div>, <div class="name string-name"><div class="surname">Kahmann</div> <div class="given-names">R.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Characterization of the largest effector gene cluster of <em>Ustilago maydis</em>,</div> <div class="source">PLoS Pathog</div> ., <div class="volume">10</div>, <div class="fpage">191</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20the%20largest%20effector%20gene%20cluster%20of%20Ustilago%20maydis%2C&amp;author=T.%20Brefort&amp;author=S.%20Tanaka&amp;author=N.%20Neidig&amp;author=G.%20Doehlemann&amp;author=V.%20Vincon&amp;author=R.%20Kahmann&amp;publication_year=2014&amp;journal=PLoS%20Pathog&amp;volume=10&amp;pages=191-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.ppat.1003866" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.ppat.1003866" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.ppat.1003866"> </span></div> </div></div></div></div></div><div content-id="dsx031-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Skibbe</div> <div class="given-names">D.S.</div></div>, <div class="name string-name"><div class="surname">Doehlemann</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Fernandes</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Walbot</div> <div class="given-names">V.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Maize tumors caused by <em>Ustilago maydis</em> require organ-specific genes in host and pathogen,</div> <div class="source">Science</div> , <div class="volume">328</div>, <div class="fpage">89</div>–<div class="lpage">92</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Maize%20tumors%20caused%20by%20Ustilago%20maydis%20require%20organ-specific%20genes%20in%20host%20and%20pathogen%2C&amp;author=D.S.%20Skibbe&amp;author=G.%20Doehlemann&amp;author=J.%20Fernandes&amp;author=V.%20Walbot&amp;publication_year=2010&amp;journal=Science&amp;volume=328&amp;pages=89-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1185775" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1185775" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1185775"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20360107" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx031-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Laurie</div> <div class="given-names">J.D.</div></div>, <div class="name string-name"><div class="surname">Ali</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Linning</div> <div class="given-names">R.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">Genome comparison of barley and maize smut fungi reveals targeted loss of RNA silencing components and species-specific presence of transposable elements,</div> <div class="source">Plant Cell</div> , <div class="volume">24</div>, <div class="fpage">1733</div>–<div class="lpage">45</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20comparison%20of%20barley%20and%20maize%20smut%20fungi%20reveals%20targeted%20loss%20of%20RNA%20silencing%20components%20and%20species-specific%20presence%20of%20transposable%20elements%2C&amp;author=J.D.%20Laurie&amp;author=S.%20Ali&amp;author=R.%20Linning&amp;publication_year=2012&amp;journal=Plant%20Cell&amp;volume=24&amp;pages=1733-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.112.097261" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.112.097261" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.112.097261"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22623492" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx031-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Schirawski</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Mannhaupt</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Münch</div> <div class="given-names">K.</div></div></div>, et al.   <div class="year">2010</div>, 
<div class="article-title">Pathogenicity determinants in smut fungi revealed by genome comparison,</div> <div class="source">Science</div> , <div class="volume">330</div>, <div class="fpage">1546</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Pathogenicity%20determinants%20in%20smut%20fungi%20revealed%20by%20genome%20comparison%2C&amp;author=J.%20Schirawski&amp;author=G.%20Mannhaupt&amp;author=K.%20M%C3%BCnch&amp;publication_year=2010&amp;journal=Science&amp;volume=330&amp;pages=1546-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1195330" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1195330" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1195330"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21148393" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx031-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Boetzer</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Henkel</div> <div class="given-names">C.V.</div></div>, <div class="name string-name"><div class="surname">Jansen</div> <div class="given-names">H.J.</div></div>, <div class="name string-name"><div class="surname">Butler</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Pirovano</div> <div class="given-names">W.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Scaffolding pre-assembled contigs using SSPACE,</div> <div class="source">Bioinformatics</div> , <div class="volume">27</div>, <div class="fpage">578</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Scaffolding%20pre-assembled%20contigs%20using%20SSPACE%2C&amp;author=M.%20Boetzer&amp;author=C.V.%20Henkel&amp;author=H.J.%20Jansen&amp;author=D.%20Butler&amp;author=W.%20Pirovano&amp;publication_year=2011&amp;journal=Bioinformatics&amp;volume=27&amp;pages=578-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btq683" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtq683" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtq683"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21149342" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx031-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Boetzer</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Pirovano</div> <div class="given-names">W.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Toward almost closed genomes with GapFiller,</div> <div class="source">Genome Biol</div> ., <div class="volume">13</div>, <div class="fpage">1</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Toward%20almost%20closed%20genomes%20with%20GapFiller%2C&amp;author=M.%20Boetzer&amp;author=W.%20Pirovano&amp;publication_year=2012&amp;journal=Genome%20Biol&amp;volume=13&amp;pages=1-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2012-13-6-r56" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2012-13-6-r56" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2012-13-6-r56"> </span></div> </div></div></div></div></div><div content-id="dsx031-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Xu</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">H.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">LTR_FINDER: an efficient tool for the prediction of full-length LTR retrotransposons,</div> <div class="source">Nucleic Acids Res</div> ., <div class="volume">35</div>, <div class="fpage">265</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=LTR_FINDER%3A%20an%20efficient%20tool%20for%20the%20prediction%20of%20full-length%20LTR%20retrotransposons%2C&amp;author=Z.%20Xu&amp;author=H.%20Wang&amp;publication_year=2007&amp;journal=Nucleic%20Acids%20Res&amp;volume=35&amp;pages=265-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkm286" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkm286" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkm286"> </span></div> </div></div></div></div></div><div content-id="dsx031-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Stanke</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Diekhans</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Baertsch</div> <div class="given-names">R., D.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">Using native and syntenically mapped cDNA alignments to improve de novo gene finding,</div> <div class="source">Bioinformatics</div> , <div class="volume">24</div>, <div class="fpage">637</div>–<div class="lpage">44</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Using%20native%20and%20syntenically%20mapped%20cDNA%20alignments%20to%20improve%20de%20novo%20gene%20finding%2C&amp;author=M.%20Stanke&amp;author=M.%20Diekhans&amp;author=R.%2C%20D.%20Baertsch&amp;publication_year=2008&amp;journal=Bioinformatics&amp;volume=24&amp;pages=637-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btn013" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtn013" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtn013"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18218656" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx031-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Elsik</div> <div class="given-names">C.G.</div></div>, <div class="name string-name"><div class="surname">Mackey</div> <div class="given-names">A.J.</div></div>, <div class="name string-name"><div class="surname">Reese</div> <div class="given-names">J.T.</div></div>, <div class="name string-name"><div class="surname">Milshina</div> <div class="given-names">N.V.</div></div>, <div class="name string-name"><div class="surname">Roos</div> <div class="given-names">D.S.</div></div>, <div class="name string-name"><div class="surname">Weinstock</div> <div class="given-names">G.M.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">Creating a honey bee consensus gene set,</div> <div class="source">Genome Biol</div> ., <div class="volume">8</div>, <div class="fpage">R13</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Creating%20a%20honey%20bee%20consensus%20gene%20set%2C&amp;author=C.G.%20Elsik&amp;author=A.J.%20Mackey&amp;author=J.T.%20Reese&amp;author=N.V.%20Milshina&amp;author=D.S.%20Roos&amp;author=G.M.%20Weinstock&amp;publication_year=2007&amp;journal=Genome%20Biol&amp;volume=8&amp;pages=R13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2007-8-1-r13" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2007-8-1-r13" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2007-8-1-r13"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17241472" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx031-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Altschul</div> <div class="given-names">S.F.</div></div>, <div class="name string-name"><div class="surname">Gish</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Miller</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Myers</div> <div class="given-names">E.W.</div></div>, <div class="name string-name"><div class="surname">Lipman</div> <div class="given-names">D.J.</div></div></div> <div class="year">1990</div>, 
<div class="article-title">Basic local alignment search tool,</div> <div class="source">J. Mol. Biol</div> ., <div class="volume">215</div>, <div class="fpage">403</div>–<div class="lpage">10</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Basic%20local%20alignment%20search%20tool%2C&amp;author=S.F.%20Altschul&amp;author=W.%20Gish&amp;author=W.%20Miller&amp;author=E.W.%20Myers&amp;author=D.J.%20Lipman&amp;publication_year=1990&amp;journal=J.%20Mol.%20Biol&amp;volume=215&amp;pages=403-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0022-2836(05)80360-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0022-2836(05)80360-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0022-2836(05)80360-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2231712" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx031-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Quevillon</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Silventoinen</div> <div class="given-names">V.</div></div>, <div class="name string-name"><div class="surname">Pillai</div> <div class="given-names">S.</div></div></div>, et al.   <div class="year">2005</div>, 
<div class="article-title">InterProScan: protein domains identifier,</div> <div class="source">Nucleic Acids Res</div> ., <div class="volume">33</div>, <div class="fpage">116</div>–<div class="lpage">20</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=InterProScan%3A%20protein%20domains%20identifier%2C&amp;author=E.%20Quevillon&amp;author=V.%20Silventoinen&amp;author=S.%20Pillai&amp;publication_year=2005&amp;journal=Nucleic%20Acids%20Res&amp;volume=33&amp;pages=116-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gki442" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgki442" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgki442"> </span></div> </div></div></div></div></div><div content-id="dsx031-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Stoeckert</div> <div class="given-names">C.J.</div></div>, <div class="name string-name"><div class="surname">Roos</div> <div class="given-names">D.S.</div></div></div> <div class="year">2003</div>, 
<div class="article-title">OrthoMCL: identification of ortholog groups for eukaryotic genomes,</div> <div class="source">Genome Res</div> ., <div class="volume">13</div>, <div class="fpage">2178</div>–<div class="lpage">89</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=OrthoMCL%3A%20identification%20of%20ortholog%20groups%20for%20eukaryotic%20genomes%2C&amp;author=L.%20Li&amp;author=C.J.%20Stoeckert&amp;author=D.S.%20Roos&amp;publication_year=2003&amp;journal=Genome%20Res&amp;volume=13&amp;pages=2178-89" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.1224503" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.1224503" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.1224503"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12952885" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx031-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yin</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Mao</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Yang</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Mao</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Xu</div> <div class="given-names">Y.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">dbCAN: a web resource for automated carbohydrate-active enzyme annotation,</div> <div class="source">Nucleic Acids Res</div> ., <div class="volume">40</div>, <div class="fpage">445</div>–<div class="lpage">51</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=dbCAN%3A%20a%20web%20resource%20for%20automated%20carbohydrate-active%20enzyme%20annotation%2C&amp;author=Y.%20Yin&amp;author=X.%20Mao&amp;author=J.%20Yang&amp;author=X.%20Chen&amp;author=F.%20Mao&amp;author=Y.%20Xu&amp;publication_year=2012&amp;journal=Nucleic%20Acids%20Res&amp;volume=40&amp;pages=445-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gks479" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgks479" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgks479"> </span></div> </div></div></div></div></div><div content-id="dsx031-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Khaldi</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Seifuddin</div> <div class="given-names">F.T.</div></div>, <div class="name string-name"><div class="surname">Turner</div> <div class="given-names">G.</div></div></div>, et al.   <div class="year">2010</div>, 
<div class="article-title">SMURF: genomic mapping of fungal secondary metabolite clusters,</div> <div class="source">Fungal Genet. Biol</div> ., <div class="volume">47</div>, <div class="fpage">736</div>–<div class="lpage">41</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SMURF%3A%20genomic%20mapping%20of%20fungal%20secondary%20metabolite%20clusters%2C&amp;author=N.%20Khaldi&amp;author=F.T.%20Seifuddin&amp;author=G.%20Turner&amp;publication_year=2010&amp;journal=Fungal%20Genet.%20Biol&amp;volume=47&amp;pages=736-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.fgb.2010.06.003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.fgb.2010.06.003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.fgb.2010.06.003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20554054" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx031-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Nielsen</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Engelbrecht</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Brunak</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Heijne</div> <div class="given-names">G.V.</div></div></div> <div class="year">1997</div>, 
<div class="article-title">Identification of prokaryotic and eukaryotic signal peptides prediction of their cleavage sites,</div> <div class="source">Protein Eng</div> ., <div class="volume">10</div>, <div class="fpage">1</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20prokaryotic%20and%20eukaryotic%20signal%20peptides%20prediction%20of%20their%20cleavage%20sites%2C&amp;author=H.%20Nielsen&amp;author=J.%20Engelbrecht&amp;author=S.%20Brunak&amp;author=G.V.%20Heijne&amp;publication_year=1997&amp;journal=Protein%20Eng&amp;volume=10&amp;pages=1-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/protein/10.1.1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fprotein%2F10.1.1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fprotein%2F10.1.1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9051728" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx031-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Emanuelsson</div> <div class="given-names">O.</div></div>, <div class="name string-name"><div class="surname">Nielsen</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Brunak</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Heijne</div> <div class="given-names">G.V.</div></div></div> <div class="year">2000</div>, 
<div class="article-title">Predicting subcellular localization of proteins based on their N-terminal amino acid sequence,</div> <div class="source">J. Mol. Biol</div> ., <div class="volume">300</div>, <div class="fpage">1005</div>–<div class="lpage">16</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Predicting%20subcellular%20localization%20of%20proteins%20based%20on%20their%20N-terminal%20amino%20acid%20sequence%2C&amp;author=O.%20Emanuelsson&amp;author=H.%20Nielsen&amp;author=S.%20Brunak&amp;author=G.V.%20Heijne&amp;publication_year=2000&amp;journal=J.%20Mol.%20Biol&amp;volume=300&amp;pages=1005-16" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/jmbi.2000.3903" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fjmbi.2000.3903" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fjmbi.2000.3903"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10891285" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx031-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lowe</div> <div class="given-names">T.M.</div></div></div> <div class="year">1997</div>, 
<div class="article-title">tRNAscan-SE: a program for improved detection of transfer RNA genes in genomic sequence,</div> <div class="source">Nucleic Acids Res</div> ., <div class="volume">25</div>, <div class="fpage">955</div>–<div class="lpage">64</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=tRNAscan-SE%3A%20a%20program%20for%20improved%20detection%20of%20transfer%20RNA%20genes%20in%20genomic%20sequence%2C&amp;author=T.M.%20Lowe&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res&amp;volume=25&amp;pages=955-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.5.0955" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.5.0955" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.5.0955"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9023104" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx031-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Nawrocki</div> <div class="given-names">E.P.</div></div>, <div class="name string-name"><div class="surname">Kolbe</div> <div class="given-names">D.L.</div></div>, <div class="name string-name"><div class="surname">Eddy</div> <div class="given-names">S.R.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Infernal 1.0: inference of RNA alignments,</div> <div class="source">Bioinformatics</div> , <div class="volume">25</div>, <div class="fpage">1335</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Infernal%201.0%3A%20inference%20of%20RNA%20alignments%2C&amp;author=E.P.%20Nawrocki&amp;author=D.L.%20Kolbe&amp;author=S.R.%20Eddy&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=1335-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp157" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp157" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp157"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19307242" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx031-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Burge</div> <div class="given-names">S.W.</div></div>, <div class="name string-name"><div class="surname">Daub</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Eberhardt</div> <div class="given-names">R.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">Rfam 11.0: 10 years of RNA families,</div> <div class="source">Nucleic Acids Res</div> ., <div class="volume">41</div>, <div class="fpage">226</div>–<div class="lpage">32</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rfam%2011.0%3A%2010%E2%80%89years%20of%20RNA%20families%2C&amp;author=S.W.%20Burge&amp;author=J.%20Daub&amp;author=R.%20Eberhardt&amp;publication_year=2013&amp;journal=Nucleic%20Acids%20Res&amp;volume=41&amp;pages=226-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gks1005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgks1005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgks1005"> </span></div> </div></div></div></div></div><div content-id="dsx031-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kurtz</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Phillippy</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Delcher</div> <div class="given-names">A.L.</div></div></div>, et al.   <div class="year">2004</div>, 
<div class="article-title">Versatile and open software for comparing large genomes,</div> <div class="source">Genome Biol</div> ., <div class="volume">5</div>, <div class="fpage">R12</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Versatile%20and%20open%20software%20for%20comparing%20large%20genomes%2C&amp;author=S.%20Kurtz&amp;author=A.%20Phillippy&amp;author=A.L.%20Delcher&amp;publication_year=2004&amp;journal=Genome%20Biol&amp;volume=5&amp;pages=R12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2004-5-2-r12" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2004-5-2-r12" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2004-5-2-r12"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14759262" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx031-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Yu</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">Y.</div></div></div>, et al.   <div class="year">2009</div>, 
<div class="article-title">SOAP2: an improved ultrafast tool for short read alignment,</div> <div class="source">Bioinformatics</div> , <div class="volume">25</div>, <div class="fpage">1966</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SOAP2%3A%20an%20improved%20ultrafast%20tool%20for%20short%20read%20alignment%2C&amp;author=R.%20Li&amp;author=C.%20Yu&amp;author=Y.%20Li&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=1966-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp336" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp336" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp336"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19497933" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx031-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Mortazavi</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Williams</div> <div class="given-names">B.A.</div></div>, <div class="name string-name"><div class="surname">Mccue</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Schaeffer</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Wold</div> <div class="given-names">B.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">Mapping and quantifying mammalian transcriptomes by RNA-Seq,</div> <div class="source">Nat. Methods</div> , <div class="volume">5</div>, <div class="fpage">621</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mapping%20and%20quantifying%20mammalian%20transcriptomes%20by%20RNA-Seq%2C&amp;author=A.%20Mortazavi&amp;author=B.A.%20Williams&amp;author=K.%20Mccue&amp;author=L.%20Schaeffer&amp;author=B.%20Wold&amp;publication_year=2008&amp;journal=Nat.%20Methods&amp;volume=5&amp;pages=621-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.1226" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.1226" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.1226"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18516045" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx031-B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Robinson</div> <div class="given-names">M.D.</div></div>, <div class="name string-name"><div class="surname">Mccarthy</div> <div class="given-names">D.J.</div></div>, <div class="name string-name"><div class="surname">Smyth</div> <div class="given-names">G.K.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">edgeR: a Bioconductor package for differential expression analysis of digital gene expression data,</div> <div class="source">Bioinformatics</div> , <div class="volume">26</div>, <div class="fpage">139</div>–<div class="lpage">40</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=edgeR%3A%20a%20Bioconductor%20package%20for%20differential%20expression%20analysis%20of%20digital%20gene%20expression%20data%2C&amp;author=M.D.%20Robinson&amp;author=D.J.%20Mccarthy&amp;author=G.K.%20Smyth&amp;publication_year=2010&amp;journal=Bioinformatics&amp;volume=26&amp;pages=139-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp616" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp616" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp616"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19910308" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx031-B46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Guo</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Qiu</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Han</div> <div class="given-names">Z.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">A host plant geneme (<em>Zizania latifolia</em>) after a century-long endophyte infection,</div> <div class="source">Plant J</div> ., <div class="volume">83</div>, <div class="fpage">600</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20host%20plant%20geneme%20(Zizania%20latifolia)%20after%20a%20century-long%20endophyte%20infection%2C&amp;author=L.%20Guo&amp;author=J.%20Qiu&amp;author=Z.%20Han&amp;publication_year=2015&amp;journal=Plant%20J&amp;volume=83&amp;pages=600-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.12912" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.12912" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.12912"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26072920" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx031-B47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sievers</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Wilm</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Dineen</div> <div class="given-names">D.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">Fast, scalable generation of high-quality protein multiple sequence alignments using Clustal Omega,</div> <div class="source">Mol. Syst. Biol</div> ., <div class="volume">7</div>, <div class="fpage">1429</div>–<div class="lpage">32</div>.</div></div></div></div><div content-id="dsx031-B48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Roy</div> <div class="given-names">S.W.</div></div>, <div class="name string-name"><div class="surname">Gilbert</div> <div class="given-names">W.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">The evolution of spliceosomal introns: patterns, puzzles and progress,</div> <div class="source">Nat. Rev. Genet</div> ., <div class="volume">7</div>, <div class="fpage">211</div>–<div class="lpage">21</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20evolution%20of%20spliceosomal%20introns%3A%20patterns%2C%20puzzles%20and%20progress%2C&amp;author=S.W.%20Roy&amp;author=W.%20Gilbert&amp;publication_year=2006&amp;journal=Nat.%20Rev.%20Genet&amp;volume=7&amp;pages=211-21" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16485020" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx031-B49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Laurie</div> <div class="given-names">J.D.</div></div>, <div class="name string-name"><div class="surname">Linning</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Wong</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Bakkeren</div> <div class="given-names">G.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Do TE activity and counteracting genome defenses, RNAi and methylation, shape the sex lives of smut fungi?,</div> <div class="source">Plant Signal. Behav</div> ., <div class="volume">8</div>, <div class="fpage">e23853</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Do%20TE%20activity%20and%20counteracting%20genome%20defenses%2C%20RNAi%20and%20methylation%2C%20shape%20the%20sex%20lives%20of%20smut%20fungi%3F%2C&amp;author=J.D.%20Laurie&amp;author=R.%20Linning&amp;author=P.%20Wong&amp;author=G.%20Bakkeren&amp;publication_year=2013&amp;journal=Plant%20Signal.%20Behav&amp;volume=8&amp;pages=e23853" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.4161/psb.23853" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.4161%2Fpsb.23853" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.4161%2Fpsb.23853"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23425853" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx031-B50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Selker</div> <div class="given-names">E.U.</div></div></div> <div class="year">1990</div>, 
<div class="article-title">Premeiotic instability of repeated sequences in <em>Neurospora crassa</em>,</div> <div class="source">Annu. Rev. Genet</div> ., <div class="volume">24</div>, <div class="fpage">579</div>–<div class="lpage">613</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Premeiotic%20instability%20of%20repeated%20sequences%20in%20Neurospora%20crassa%2C&amp;author=E.U.%20Selker&amp;publication_year=1990&amp;journal=Annu.%20Rev.%20Genet&amp;volume=24&amp;pages=579-613" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.ge.24.120190.003051" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.ge.24.120190.003051" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.ge.24.120190.003051"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2150906" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx031-B51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Vogel</div> <div class="given-names">K.J.</div></div>, <div class="name string-name"><div class="surname">Moran</div> <div class="given-names">N.A.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Functional and evolutionary analysis of the genome of an obligate fungal symbiont,</div> <div class="source">Genome Biol. Evol</div> ., <div class="volume">5</div>, <div class="fpage">891</div>–<div class="lpage">904</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Functional%20and%20evolutionary%20analysis%20of%20the%20genome%20of%20an%20obligate%20fungal%20symbiont%2C&amp;author=K.J.%20Vogel&amp;author=N.A.%20Moran&amp;publication_year=2013&amp;journal=Genome%20Biol.%20Evol&amp;volume=5&amp;pages=891-904" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/gbe/evt054" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fgbe%2Fevt054" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fgbe%2Fevt054"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23563967" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx031-B52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B52" class="ref-content "><div class="label -label">52</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kellner</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Vollmeister</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Feldbrügge</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Begerow</div> <div class="given-names">D.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Interspecific sex in grass smuts and the genetic diversity of their pheromone-receptor system,</div> <div class="source">PLoS Genet</div> ., <div class="volume">7</div>, <div class="fpage">e1002436</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Interspecific%20sex%20in%20grass%20smuts%20and%20the%20genetic%20diversity%20of%20their%20pheromone-receptor%20system%2C&amp;author=R.%20Kellner&amp;author=E.%20Vollmeister&amp;author=M.%20Feldbr%C3%BCgge&amp;author=D.%20Begerow&amp;publication_year=2011&amp;journal=PLoS%20Genet&amp;volume=7&amp;pages=e1002436" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1002436" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1002436" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1002436"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22242007" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx031-B53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B53" class="ref-content "><div class="label -label">53</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Klaus</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Lengeler</div> <div class="given-names">D.S.F.</div></div>, <div class="name string-name"><div class="surname">Fraser</div> <div class="given-names">J.A.</div></div></div>, et al.   <div class="year">2002</div>, 
<div class="article-title">Mating-type locus of <em>Cryptococcus neoformans</em>: a step in the evolution of sex chromosomes,</div> <div class="source">Eukaryot. Cell</div> , <div class="volume">1</div>, <div class="fpage">704</div>–<div class="lpage">18</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mating-type%20locus%20of%20Cryptococcus%20neoformans%3A%20a%20step%20in%20the%20evolution%20of%20sex%20chromosomes%2C&amp;author=B.%20Klaus&amp;author=D.S.F.%20Lengeler&amp;author=J.A.%20Fraser&amp;publication_year=2002&amp;journal=Eukaryot.%20Cell&amp;volume=1&amp;pages=704-18" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/EC.1.5.704-718.2002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FEC.1.5.704-718.2002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FEC.1.5.704-718.2002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12455690" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx031-B54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B54" class="ref-content "><div class="label -label">54</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hood</div> <div class="given-names">M.E.</div></div>, <div class="name string-name"><div class="surname">Antonovics</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Koskella</div> <div class="given-names">B.</div></div></div> <div class="year">2004</div>, 
<div class="article-title">Shared forces of sex chromosome evolution in haploid-mating and diploid-mating organisms: <em>Microbotryum violaceum</em> and other model organisms,</div> <div class="source">Genetics</div> , <div class="volume">168</div>, <div class="fpage">141</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Shared%20forces%20of%20sex%20chromosome%20evolution%20in%20haploid-mating%20and%20diploid-mating%20organisms%3A%20Microbotryum%20violaceum%20and%20other%20model%20organisms%2C&amp;author=M.E.%20Hood&amp;author=J.%20Antonovics&amp;author=B.%20Koskella&amp;publication_year=2004&amp;journal=Genetics&amp;volume=168&amp;pages=141-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.104.029900" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.104.029900" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.104.029900"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15454533" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx031-B55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B55" class="ref-content "><div class="label -label">55</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gordon</div> <div class="given-names">J.L.</div></div>, <div class="name string-name"><div class="surname">Armisén</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Proux-Wéra</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Ó</div> <div class="given-names">S.S.</div></div>, <div class="name string-name"><div class="surname">Byrne</div> <div class="given-names">K.P.</div></div>, <div class="name string-name"><div class="surname">Wolfe</div> <div class="given-names">K.H.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Evolutionary erosion of yeast sex chromosomes by mating-type switching accidents,</div> <div class="source">Proc. Natl. Acad. Sci. USA</div> ., <div class="volume">108</div>, <div class="fpage">20024</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolutionary%20erosion%20of%20yeast%20sex%20chromosomes%20by%20mating-type%20switching%20accidents%2C&amp;author=J.L.%20Gordon&amp;author=D.%20Armis%C3%A9n&amp;author=E.%20Proux-W%C3%A9ra&amp;author=S.S.%20%C3%93&amp;author=K.P.%20Byrne&amp;author=K.H.%20Wolfe&amp;publication_year=2011&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=108&amp;pages=20024-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1112808108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1112808108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1112808108"> </span></div> </div></div></div></div></div><div content-id="dsx031-B56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B56" class="ref-content "><div class="label -label">56</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tollis</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Boissinot</div> <div class="given-names">S.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">The evolutionary dynamics of transposable elements in eukaryote genomes,</div> <div class="source">Genome Dyn</div> ., <div class="volume">7</div>, <div class="fpage">68</div>–<div class="lpage">91</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20evolutionary%20dynamics%20of%20transposable%20elements%20in%20eukaryote%20genomes%2C&amp;author=M.%20Tollis&amp;author=S.%20Boissinot&amp;publication_year=2012&amp;journal=Genome%20Dyn&amp;volume=7&amp;pages=68-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1159/000337126" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1159%2F000337126" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1159%2F000337126"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22759814" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx031-B57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B57" class="ref-content "><div class="label -label">57</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ochogavía</div> <div class="given-names">A.C.</div></div>, <div class="name string-name"><div class="surname">Seijo</div> <div class="given-names">J.G.</div></div>, <div class="name string-name"><div class="surname">González</div> <div class="given-names">A.M.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">Characterization of retrotransposon sequences expressed in inflorescences of apomictic and sexual <em>Paspalum notatum</em> plants,</div> <div class="source">Sex. Plant Reprod</div> ., <div class="volume">24</div>, <div class="fpage">231</div>–<div class="lpage">46</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20retrotransposon%20sequences%20expressed%20in%20inflorescences%20of%20apomictic%20and%20sexual%20Paspalum%20notatum%20plants%2C&amp;author=A.C.%20Ochogav%C3%ADa&amp;author=J.G.%20Seijo&amp;author=A.M.%20Gonz%C3%A1lez&amp;publication_year=2011&amp;journal=Sex.%20Plant%20Reprod&amp;volume=24&amp;pages=231-46" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00497-011-0165-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00497-011-0165-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00497-011-0165-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21394488" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx031-B58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B58" class="ref-content "><div class="label -label">58</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lokody</div> <div class="given-names">I.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Evolution: transposons drive sex chromosome evolution,</div> <div class="source">Nat. Rev. Genet</div> ., <div class="volume">15</div>, <div class="issue">1</div>.</div></div></div></div><div content-id="dsx031-B59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B59" class="ref-content "><div class="label -label">59</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Fedler</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Luh</div> <div class="given-names">K.S.</div></div>, <div class="name string-name"><div class="surname">Stelter</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Fernanda</div> <div class="given-names">N.J.</div></div>, <div class="name string-name"><div class="surname">Basse</div> <div class="given-names">C.W.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">The <em>a2</em> mating-type locus genes <em>lga2</em> and <em>rga2</em> direct uniparental mitochondrial DNA (mtDNA) inheritance and constrain mtDNA recombination during sexual development of <em>Ustilago maydis</em>,</div> <div class="source">Genetics</div> , <div class="volume">181</div>, <div class="fpage">847</div>–<div class="lpage">60</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20a2%20mating-type%20locus%20genes%20lga2%20and%20rga2%20direct%20uniparental%20mitochondrial%20DNA%20(mtDNA)%20inheritance%20and%20constrain%20mtDNA%20recombination%20during%20sexual%20development%20of%20Ustilago%20maydis%2C&amp;author=M.%20Fedler&amp;author=K.S.%20Luh&amp;author=K.%20Stelter&amp;author=N.J.%20Fernanda&amp;author=C.W.%20Basse&amp;publication_year=2009&amp;journal=Genetics&amp;volume=181&amp;pages=847-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.108.096859" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.108.096859" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.108.096859"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19104076" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx031-B60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B60" class="ref-content "><div class="label -label">60</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bortfeld</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Auffarth</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Kahmann</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Basse</div> <div class="given-names">C.W.</div></div></div> <div class="year">2004</div>, 
<div class="article-title">The <em>Ustilago maydis a2</em> mating-type locus genes <em>lga2</em> and <em>rga2</em> compromise pathogenicity in the absence of the mitochondrial p32 family protein Mrb1,</div> <div class="source">Plant Cell</div> , <div class="volume">16</div>, <div class="fpage">2233</div>–<div class="lpage">48</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Ustilago%20maydis%20a2%20mating-type%20locus%20genes%20lga2%20and%20rga2%20compromise%20pathogenicity%20in%20the%20absence%20of%20the%20mitochondrial%20p32%20family%20protein%20Mrb1%2C&amp;author=M.%20Bortfeld&amp;author=K.%20Auffarth&amp;author=R.%20Kahmann&amp;author=C.W.%20Basse&amp;publication_year=2004&amp;journal=Plant%20Cell&amp;volume=16&amp;pages=2233-48" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.104.022657" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.104.022657" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.104.022657"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15273296" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx031-B61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B61" class="ref-content "><div class="label -label">61</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Breton</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Stewart</div> <div class="given-names">D.T.</div></div>, <div class="name string-name"><div class="surname">Bonen</div> <div class="given-names">L.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Atypical mitochondrial inheritance patterns in eukaryotes,</div> <div class="source">Genome</div> , <div class="volume">58</div>, <div class="fpage">423</div>–<div class="lpage">31</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Atypical%20mitochondrial%20inheritance%20patterns%20in%20eukaryotes%2C&amp;author=S.%20Breton&amp;author=D.T.%20Stewart&amp;author=L.%20Bonen&amp;publication_year=2015&amp;journal=Genome&amp;volume=58&amp;pages=423-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/gen-2015-0090" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2Fgen-2015-0090" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2Fgen-2015-0090"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26501689" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx031-B62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B62" class="ref-content "><div class="label -label">62</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yan</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Sun</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Shahid</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Xu</div> <div class="given-names">J.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">Environment factors can influence mitochondrial inheritance in the fungus <em>Cryptococcus neoformans</em>,</div> <div class="source">Fungal Genet. Biol</div> ., <div class="volume">44</div>, <div class="fpage">315</div>–<div class="lpage">22</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Environment%20factors%20can%20influence%20mitochondrial%20inheritance%20in%20the%20fungus%20Cryptococcus%20neoformans%2C&amp;author=Z.%20Yan&amp;author=S.%20Sun&amp;author=M.%20Shahid&amp;author=J.%20Xu&amp;publication_year=2007&amp;journal=Fungal%20Genet.%20Biol&amp;volume=44&amp;pages=315-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.fgb.2006.10.002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.fgb.2006.10.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.fgb.2006.10.002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17092744" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx031-B63" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B63" class="ref-content "><div class="label -label">63</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Heimel</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Scherer</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Vranes</div> <div class="given-names">M.</div></div></div>, et al.   <div class="year">2010</div>, 
<div class="article-title">The transcription factor Rbf1 is the master regulator for <em>b</em>-mating type controlled pathogenic development in <em>Ustilago maydis</em>,</div> <div class="source">PLoS Pathog</div> ., <div class="volume">6</div>, <div class="fpage">761</div>–<div class="lpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20transcription%20factor%20Rbf1%20is%20the%20master%20regulator%20for%20b-mating%20type%20controlled%20pathogenic%20development%20in%20Ustilago%20maydis%2C&amp;author=K.%20Heimel&amp;author=M.%20Scherer&amp;author=M.%20Vranes&amp;publication_year=2010&amp;journal=PLoS%20Pathog&amp;volume=6&amp;pages=761-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.ppat.1001035" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.ppat.1001035" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.ppat.1001035"> </span></div> </div></div></div></div></div><div content-id="dsx031-B64" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B64" class="ref-content "><div class="label -label">64</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Brefort</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Doehlemann</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Mendoza-Mendoza</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Reissmann</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Djamei</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Kahmann</div> <div class="given-names">R.</div></div></div> <div class="year">2009</div>, 
<div class="article-title"><em>Ustilago maydis</em> as a pathogen,</div> <div class="source">Phytopathology</div> , <div class="volume">47</div>, <div class="fpage">423</div>–<div class="lpage">45</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ustilago%20maydis%20as%20a%20pathogen%2C&amp;author=T.%20Brefort&amp;author=G.%20Doehlemann&amp;author=A.%20Mendoza-Mendoza&amp;author=S.%20Reissmann&amp;author=A.%20Djamei&amp;author=R.%20Kahmann&amp;publication_year=2009&amp;journal=Phytopathology&amp;volume=47&amp;pages=423-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev-phyto-080508-081923" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev-phyto-080508-081923" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev-phyto-080508-081923"> </span></div> </div></div></div></div></div><div content-id="dsx031-B65" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B65" class="ref-content "><div class="label -label">65</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Feldbrügge</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Al</div> <div class="given-names">E.</div></div></div> <div class="year">2004</div>, 
<div class="article-title">Regulation of mating and pathogenic development in <em>Ustilago maydis</em>,</div> <div class="source">Curr. Opin. Microbiol</div> ., <div class="volume">7</div>, <div class="fpage">666</div>–<div class="lpage">72</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Regulation%20of%20mating%20and%20pathogenic%20development%20in%20Ustilago%20maydis%2C&amp;author=M.%20Feldbr%C3%BCgge&amp;author=E.%20Al&amp;publication_year=2004&amp;journal=Curr.%20Opin.%20Microbiol&amp;volume=7&amp;pages=666-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.mib.2004.10.006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.mib.2004.10.006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.mib.2004.10.006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15556041" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx031-B66" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B66" class="ref-content "><div class="label -label">66</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zarnack</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Eichhorn</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Kahmann</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Feldbrügge</div> <div class="given-names">M.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">Pheromone-regulated target genes respond differentially to MAPK phosphorylation of transcription factor Prf1,</div> <div class="source">Mol. Microbiol</div> ., <div class="volume">69</div>, <div class="fpage">1041</div>–<div class="lpage">53</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Pheromone-regulated%20target%20genes%20respond%20differentially%20to%20MAPK%20phosphorylation%20of%20transcription%20factor%20Prf1%2C&amp;author=K.%20Zarnack&amp;author=H.%20Eichhorn&amp;author=R.%20Kahmann&amp;author=M.%20Feldbr%C3%BCgge&amp;publication_year=2008&amp;journal=Mol.%20Microbiol&amp;volume=69&amp;pages=1041-53" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18627457" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx031-B67" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B67" class="ref-content "><div class="label -label">67</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Heimel</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Scherer</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Schuler</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Kämper</div> <div class="given-names">J.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">The <em>Ustilago maydis</em> Clp1 protein orchestrates pheromone and <em>b</em>-dependent signaling pathways to coordinate the cell cycle and pathogenic development,</div> <div class="source">Plant Cell</div> , <div class="volume">22</div>, <div class="fpage">2908</div>–<div class="lpage">22</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Ustilago%20maydis%20Clp1%20protein%20orchestrates%20pheromone%20and%20b-dependent%20signaling%20pathways%20to%20coordinate%20the%20cell%20cycle%20and%20pathogenic%20development%2C&amp;author=K.%20Heimel&amp;author=M.%20Scherer&amp;author=D.%20Schuler&amp;author=J.%20K%C3%A4mper&amp;publication_year=2010&amp;journal=Plant%20Cell&amp;volume=22&amp;pages=2908-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.110.076265" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.110.076265" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.110.076265"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20729384" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx031-B68" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B68" class="ref-content "><div class="label -label">68</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Helfer</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Gladfelter</div> <div class="given-names">A.S.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">AgSwe1p regulates mitosis in response to morphogenesis and nutrients in multinucleated <em>Ashbya gossypii</em> cells,</div> <div class="source">Mol. Biol. Cell</div> , <div class="volume">17</div>, <div class="fpage">4494</div>–<div class="lpage">512</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=AgSwe1p%20regulates%20mitosis%20in%20response%20to%20morphogenesis%20and%20nutrients%20in%20multinucleated%20Ashbya%20gossypii%20cells%2C&amp;author=H.%20Helfer&amp;author=A.S.%20Gladfelter&amp;publication_year=2006&amp;journal=Mol.%20Biol.%20Cell&amp;volume=17&amp;pages=4494-512" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1091/mbc.E06-03-0215" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1091%2Fmbc.E06-03-0215" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1091%2Fmbc.E06-03-0215"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16899511" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx031-B69" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B69" class="ref-content "><div class="label -label">69</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Mccann</div> <div class="given-names">M.P.</div></div>, <div class="name string-name"><div class="surname">Snetselaar</div> <div class="given-names">K.M.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">A genome-based analysis of amino acid metabolism in the biotrophic plant pathogen <em>Ustilago maydis</em>,</div> <div class="source">Fungal Genet. Biol</div> ., <div class="volume">45(Suppl 1)</div>, <div class="fpage">S77</div>–<div class="lpage">87</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20genome-based%20analysis%20of%20amino%20acid%20metabolism%20in%20the%20biotrophic%20plant%20pathogen%20Ustilago%20maydis%2C&amp;author=M.P.%20Mccann&amp;author=K.M.%20Snetselaar&amp;publication_year=2008&amp;journal=Fungal%20Genet.%20Biol&amp;volume=45(Suppl 1)&amp;pages=S77-87" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.fgb.2008.05.006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.fgb.2008.05.006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.fgb.2008.05.006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18579420" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx031-B70" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B70" class="ref-content "><div class="label -label">70</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hansen</div> <div class="given-names">A.K.</div></div>, <div class="name string-name"><div class="surname">Moran</div> <div class="given-names">N.A.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Aphid genome expression reveals host-symbiont cooperation in the production of amino acids,</div> <div class="source">Proc. Natl. Acad. Sci. USA</div> ., <div class="volume">108</div>, <div class="fpage">2849</div>–<div class="lpage">54</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Aphid%20genome%20expression%20reveals%20host-symbiont%20cooperation%20in%20the%20production%20of%20amino%20acids%2C&amp;author=A.K.%20Hansen&amp;author=N.A.%20Moran&amp;publication_year=2011&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=108&amp;pages=2849-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1013465108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1013465108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1013465108"> </span></div> </div></div></div></div></div><div content-id="dsx031-B71" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B71" class="ref-content "><div class="label -label">71</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hornby</div> <div class="given-names">J.M.</div></div>, <div class="name string-name"><div class="surname">Jacobitzkizzier</div> <div class="given-names">S.M.</div></div>, <div class="name string-name"><div class="surname">Mcneel</div> <div class="given-names">D.J.</div></div>, <div class="name string-name"><div class="surname">Jensen</div> <div class="given-names">E.C.</div></div>, <div class="name string-name"><div class="surname">Treves</div> <div class="given-names">D.S.</div></div>, <div class="name string-name"><div class="surname">Nickerson</div> <div class="given-names">K.W.</div></div></div> <div class="year">2004</div>, 
<div class="article-title">Inoculum size effect in dimorphic fungi: extracellular control of yeast-mycelium dimorphism in <em>Ceratocystis ulmi</em>,</div> <div class="source">Appl. Environ. Microbiol</div> ., <div class="volume">70</div>, <div class="fpage">1356</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Inoculum%20size%20effect%20in%20dimorphic%20fungi%3A%20extracellular%20control%20of%20yeast-mycelium%20dimorphism%20in%20Ceratocystis%20ulmi%2C&amp;author=J.M.%20Hornby&amp;author=S.M.%20Jacobitzkizzier&amp;author=D.J.%20Mcneel&amp;author=E.C.%20Jensen&amp;author=D.S.%20Treves&amp;author=K.W.%20Nickerson&amp;publication_year=2004&amp;journal=Appl.%20Environ.%20Microbiol&amp;volume=70&amp;pages=1356-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AEM.70.3.1356-1359.2004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.70.3.1356-1359.2004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FAEM.70.3.1356-1359.2004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15006753" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx031-B72" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B72" class="ref-content "><div class="label -label">72</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Jiménezlópez</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Collette</div> <div class="given-names">J.R.</div></div>, <div class="name string-name"><div class="surname">Brothers</div> <div class="given-names">K.M.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title"><em>Candida albicans</em> induces arginine biosynthetic genes in response to host-derived reactive oxygen species,</div> <div class="source">Eukaryot. Cell</div> , <div class="volume">12</div>, <div class="fpage">91</div>–<div class="lpage">100</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Candida%20albicans%20induces%20arginine%20biosynthetic%20genes%20in%20response%20to%20host-derived%20reactive%20oxygen%20species%2C&amp;author=C.%20Jim%C3%A9nezl%C3%B3pez&amp;author=J.R.%20Collette&amp;author=K.M.%20Brothers&amp;publication_year=2013&amp;journal=Eukaryot.%20Cell&amp;volume=12&amp;pages=91-100" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/EC.00290-12" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FEC.00290-12" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FEC.00290-12"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23143683" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx031-B73" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B73" class="ref-content "><div class="label -label">73</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Davis</div> <div class="given-names">R.H.</div></div></div> <div class="year">1986</div>, 
<div class="article-title">Compartmental and regulatory mechanisms in the arginine pathways of <em>Neurospora crassa</em> and <em>Saccharomyces cerevisiae</em>,</div> <div class="source">Microbiol. Rev</div> ., <div class="volume">50</div>, <div class="fpage">280</div>–<div class="lpage">313</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Compartmental%20and%20regulatory%20mechanisms%20in%20the%20arginine%20pathways%20of%20Neurospora%20crassa%20and%20Saccharomyces%20cerevisiae%2C&amp;author=R.H.%20Davis&amp;publication_year=1986&amp;journal=Microbiol.%20Rev&amp;volume=50&amp;pages=280-313" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2945985" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx031-B74" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B74" class="ref-content "><div class="label -label">74</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Senthilkumar</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Mysore</div> <div class="given-names">K.S.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Ornithine-delta-aminotransferase and proline dehydrogenase genes play a role in non-host disease resistance by regulating pyrroline-5-carboxylate metabolism-induced hypersensitive response</div>, <div class="source">Plant Cell Environ</div> ., <div class="volume">35</div>, <div class="fpage">1329</div>–<div class="lpage">43</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ornithine-delta-aminotransferase%20and%20proline%20dehydrogenase%20genes%20play%20a%20role%20in%20non-host%20disease%20resistance%20by%20regulating%20pyrroline-5-carboxylate%20metabolism-induced%20hypersensitive%20response&amp;author=M.%20Senthilkumar&amp;author=K.S.%20Mysore&amp;publication_year=2012&amp;journal=Plant%20Cell%20Environ&amp;volume=35&amp;pages=1329-43" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/pce.2012.35.issue-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fpce.2012.35.issue-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fpce.2012.35.issue-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22321246" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx031-B75" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B75" class="ref-content "><div class="label -label">75</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zhu</div> <div class="given-names">X.L.</div></div>, <div class="name string-name"><div class="surname">Qi</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">X.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">The wheat ethylene response factor transcription factor pathogen-induced ERF1 mediates host responses to both the necrotrophic pathogen <em>Rhizoctonia cerealis</em> and freezing stresses,</div> <div class="source">Plant. Physiol</div> ., <div class="volume">164</div>, <div class="fpage">1499</div>–<div class="lpage">514</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20wheat%20ethylene%20response%20factor%20transcription%20factor%20pathogen-induced%20ERF1%20mediates%20host%20responses%20to%20both%20the%20necrotrophic%20pathogen%20Rhizoctonia%20cerealis%20and%20freezing%20stresses%2C&amp;author=X.L.%20Zhu&amp;author=L.%20Qi&amp;author=X.%20Liu&amp;publication_year=2014&amp;journal=Plant.%20Physiol&amp;volume=164&amp;pages=1499-514" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.113.229575" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.113.229575" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.113.229575"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24424323" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx031-B76" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B76" class="ref-content "><div class="label -label">76</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Boubakri</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Wahab</div> <div class="given-names">M.A.</div></div>, <div class="name string-name"><div class="surname">Chong</div> <div class="given-names">J.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">Methionine elicits H<sub>2</sub>O<sub>2</sub> generation and defense gene expression in grapevine and reduces <em>Plasmopara viticola</em> infection,</div> <div class="source">J. Plant Physiol</div> ., <div class="volume">170</div>, <div class="fpage">1561</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Methionine%20elicits%20H2O2%20generation%20and%20defense%20gene%20expression%20in%20grapevine%20and%20reduces%20Plasmopara%20viticola%20infection%2C&amp;author=H.%20Boubakri&amp;author=M.A.%20Wahab&amp;author=J.%20Chong&amp;publication_year=2013&amp;journal=J.%20Plant%20Physiol&amp;volume=170&amp;pages=1561-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.jplph.2013.06.008" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.jplph.2013.06.008" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.jplph.2013.06.008"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23859561" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx031-B77" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B77" class="ref-content "><div class="label -label">77</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Arasimowicz-Jelonek</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Floryszak-Wieczorek</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Gzyl</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Chmielowska-Bąk</div> <div class="given-names">J.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Homocysteine over-accumulation as the effect of potato leaves exposure to biotic stress,</div> <div class="source">Plant Physiol. Biochem</div> ., <div class="volume">63</div>, <div class="fpage">177</div>–<div class="lpage">84</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Homocysteine%20over-accumulation%20as%20the%20effect%20of%20potato%20leaves%20exposure%20to%20biotic%20stress%2C&amp;author=M.%20Arasimowicz-Jelonek&amp;author=J.%20Floryszak-Wieczorek&amp;author=J.%20Gzyl&amp;author=J.%20Chmielowska-B%C4%85k&amp;publication_year=2013&amp;journal=Plant%20Physiol.%20Biochem&amp;volume=63&amp;pages=177-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.plaphy.2012.11.025" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.plaphy.2012.11.025" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.plaphy.2012.11.025"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23266362" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx031-B78" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B78" class="ref-content "><div class="label -label">78</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yang</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Ludewig</div> <div class="given-names">U.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Lysine catabolism, amino acid transport, and systemic acquired resistance: what is the link?,</div> <div class="source">Plant Signal. Behav</div> ., <div class="volume">9</div>, <div class="fpage">e28933</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Lysine%20catabolism%2C%20amino%20acid%20transport%2C%20and%20systemic%20acquired%20resistance%3A%20what%20is%20the%20link%3F%2C&amp;author=H.%20Yang&amp;author=U.%20Ludewig&amp;publication_year=2014&amp;journal=Plant%20Signal.%20Behav&amp;volume=9&amp;pages=e28933" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.4161/psb.28933" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.4161%2Fpsb.28933" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.4161%2Fpsb.28933"> </span></div> </div></div></div></div></div><div content-id="dsx031-B79" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B79" class="ref-content "><div class="label -label">79</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kim</div> <div class="given-names">D.S.</div></div>, <div class="name string-name"><div class="surname">Hwang</div> <div class="given-names">B.K.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">An important role of the pepper phenylalanine ammonia-lyase gene (PAL1) in salicylic acid-dependent signalling of the defence response to microbial pathogens,</div> <div class="source">J. Exp. Bot</div> ., <div class="volume">65</div>, <div class="fpage">2295</div>–<div class="lpage">306</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20important%20role%20of%20the%20pepper%20phenylalanine%20ammonia-lyase%20gene%20(PAL1)%20in%20salicylic%20acid-dependent%20signalling%20of%20the%20defence%20response%20to%20microbial%20pathogens%2C&amp;author=D.S.%20Kim&amp;author=B.K.%20Hwang&amp;publication_year=2014&amp;journal=J.%20Exp.%20Bot&amp;volume=65&amp;pages=2295-306" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jxb/eru109" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjxb%2Feru109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjxb%2Feru109"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24642849" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx031-B80" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B80" class="ref-content "><div class="label -label">80</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kurth</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Mailänder</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Bönn</div> <div class="given-names">M.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Streptomyces-induced resistance against oak powdery mildew involves host plant responses in defense, photosynthesis, and secondary metabolism pathways,</div> <div class="source">Mol. Plant Microbe Interact</div> ., <div class="volume">27</div>, <div class="fpage">891</div>–<div class="lpage">900</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Streptomyces-induced%20resistance%20against%20oak%20powdery%20mildew%20involves%20host%20plant%20responses%20in%20defense%2C%20photosynthesis%2C%20and%20secondary%20metabolism%20pathways%2C&amp;author=F.%20Kurth&amp;author=S.%20Mail%C3%A4nder&amp;author=M.%20B%C3%B6nn&amp;publication_year=2014&amp;journal=Mol.%20Plant%20Microbe%20Interact&amp;volume=27&amp;pages=891-900" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1094/MPMI-10-13-0296-R" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1094%2FMPMI-10-13-0296-R" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1094%2FMPMI-10-13-0296-R"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24779643" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx031-B81" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B81" class="ref-content "><div class="label -label">81</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hara</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Kondo</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Kato</div> <div class="given-names">T.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">A KS-type dehydrin and its related domains reduce Cu-promoted radical generation and the histidine residues contribute to the radical-reducing activities,</div> <div class="source">J. Exp. Bot</div> ., <div class="volume">64</div>, <div class="fpage">1615</div>–<div class="lpage">24</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20KS-type%20dehydrin%20and%20its%20related%20domains%20reduce%20Cu-promoted%20radical%20generation%20and%20the%20histidine%20residues%20contribute%20to%20the%20radical-reducing%20activities%2C&amp;author=M.%20Hara&amp;author=M.%20Kondo&amp;author=T.%20Kato&amp;publication_year=2013&amp;journal=J.%20Exp.%20Bot&amp;volume=64&amp;pages=1615-24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jxb/ert016" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjxb%2Fert016" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjxb%2Fert016"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23382551" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx031-B82" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B82" class="ref-content "><div class="label -label">82</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bechinger</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Giebel</div> <div class="given-names">K.F.</div></div>, <div class="name string-name"><div class="surname">Schnell</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Leiderer</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Deising</div> <div class="given-names">H.B.</div></div>, <div class="name string-name"><div class="surname">Bastmeyer</div> <div class="given-names">M.</div></div></div> <div class="year">1999</div>, 
<div class="article-title">Optical measurements of invasive forces exerted by appressoria of a plant pathogenic fungus,</div> <div class="source">Science</div> , <div class="volume">285</div>, <div class="fpage">1896</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Optical%20measurements%20of%20invasive%20forces%20exerted%20by%20appressoria%20of%20a%20plant%20pathogenic%20fungus%2C&amp;author=C.%20Bechinger&amp;author=K.F.%20Giebel&amp;author=M.%20Schnell&amp;author=P.%20Leiderer&amp;author=H.B.%20Deising&amp;author=M.%20Bastmeyer&amp;publication_year=1999&amp;journal=Science&amp;volume=285&amp;pages=1896-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.285.5435.1896" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.285.5435.1896" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.285.5435.1896"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10489364" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx031-B83" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B83" class="ref-content "><div class="label -label">83</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Howard</div> <div class="given-names">R.J.</div></div>, <div class="name string-name"><div class="surname">Ferrari</div> <div class="given-names">M.A.</div></div>, <div class="name string-name"><div class="surname">Roach</div> <div class="given-names">D.H.</div></div>, <div class="name string-name"><div class="surname">Money</div> <div class="given-names">N.P.</div></div></div> <div class="year">1991</div>, 
<div class="article-title">Penetration of hard substrates by a fungus employing enormous turgor pressures,</div> <div class="source">Proc. Natl. Acad. Sci. USA</div> ., <div class="volume">88</div>, <div class="fpage">11281</div>–<div class="lpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Penetration%20of%20hard%20substrates%20by%20a%20fungus%20employing%20enormous%20turgor%20pressures%2C&amp;author=R.J.%20Howard&amp;author=M.A.%20Ferrari&amp;author=D.H.%20Roach&amp;author=N.P.%20Money&amp;publication_year=1991&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=88&amp;pages=11281-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.88.24.11281" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.88.24.11281" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.88.24.11281"> </span></div> </div></div></div></div></div><div content-id="dsx031-B84" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B84" class="ref-content "><div class="label -label">84</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Schirawski</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Böhnert</div> <div class="given-names">H.U.</div></div>, <div class="name string-name"><div class="surname">Steinberg</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Snetselaar</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Adamikowa</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Kahmann</div> <div class="given-names">R.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">Endoplasmic reticulum glucosidase II is required for pathogenicity of <em>Ustilago maydis</em>,</div> <div class="source">Plant Cell</div> , <div class="volume">17</div>, <div class="fpage">3532</div>–<div class="lpage">43</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Endoplasmic%20reticulum%20glucosidase%20II%20is%20required%20for%20pathogenicity%20of%20Ustilago%20maydis%2C&amp;author=J.%20Schirawski&amp;author=H.U.%20B%C3%B6hnert&amp;author=G.%20Steinberg&amp;author=K.%20Snetselaar&amp;author=L.%20Adamikowa&amp;author=R.%20Kahmann&amp;publication_year=2005&amp;journal=Plant%20Cell&amp;volume=17&amp;pages=3532-43" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.105.036285" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.105.036285" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.105.036285"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16272431" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx031-B85" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B85" class="ref-content "><div class="label -label">85</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tonukari</div> <div class="given-names">N.J.</div></div>, <div class="name string-name"><div class="surname">Scott-Craig</div> <div class="given-names">J.S.</div></div>, <div class="name string-name"><div class="surname">Walton</div> <div class="given-names">J.D.</div></div></div> <div class="year">2000</div>, 
<div class="article-title">The <em>Cochliobolus carbonum SNF1</em> gene is required for cell wall-degrading enzyme expression and virulence on maize,</div> <div class="source">Plant Cell</div> , <div class="volume">12</div>, <div class="fpage">237</div>–<div class="lpage">47</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Cochliobolus%20carbonum%20SNF1%20gene%20is%20required%20for%20cell%20wall-degrading%20enzyme%20expression%20and%20virulence%20on%20maize%2C&amp;author=N.J.%20Tonukari&amp;author=J.S.%20Scott-Craig&amp;author=J.D.%20Walton&amp;publication_year=2000&amp;journal=Plant%20Cell&amp;volume=12&amp;pages=237-47" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.12.2.237" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.12.2.237" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.12.2.237"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10662860" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx031-B86" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B86" class="ref-content "><div class="label -label">86</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kulkarni</div> <div class="given-names">R.D.</div></div>, <div class="name string-name"><div class="surname">Thon</div> <div class="given-names">M.R.</div></div>, <div class="name string-name"><div class="surname">Pan</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Dean</div> <div class="given-names">R.A.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">Novel G-protein-coupled receptor-like proteins in the plant pathogenic fungus <em>Magnaporthe grisea</em>,</div> <div class="source">Genome Biol</div> ., <div class="volume">6</div>, <div class="fpage">R24</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Novel%20G-protein-coupled%20receptor-like%20proteins%20in%20the%20plant%20pathogenic%20fungus%20Magnaporthe%20grisea%2C&amp;author=R.D.%20Kulkarni&amp;author=M.R.%20Thon&amp;author=H.%20Pan&amp;author=R.A.%20Dean&amp;publication_year=2005&amp;journal=Genome%20Biol&amp;volume=6&amp;pages=R24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2005-6-3-r24" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2005-6-3-r24" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2005-6-3-r24"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15774025" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx031-B87" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B87" class="ref-content "><div class="label -label">87</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Basse</div> <div class="given-names">C.W.</div></div>, <div class="name string-name"><div class="surname">Kolb</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Kahmann</div> <div class="given-names">R.</div></div></div> <div class="year">2002</div>, 
<div class="article-title">A maize-specifically expressed gene cluster in <em>Ustilago maydis</em>,</div> <div class="source">Mol. Microbiol</div> ., <div class="volume">43</div>, <div class="fpage">75</div>–<div class="lpage">93</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20maize-specifically%20expressed%20gene%20cluster%20in%20Ustilago%20maydis%2C&amp;author=C.W.%20Basse&amp;author=S.%20Kolb&amp;author=R.%20Kahmann&amp;publication_year=2002&amp;journal=Mol.%20Microbiol&amp;volume=43&amp;pages=75-93" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-2958.2002.02742.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-2958.2002.02742.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-2958.2002.02742.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11849538" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx031-B88" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B88" class="ref-content "><div class="label -label">88</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Farfsing</div> <div class="given-names">J.W.</div></div>, <div class="name string-name"><div class="surname">Auffarth</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Basse</div> <div class="given-names">C.W.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">Identification of cis-active elements in <em>Ustilago maydis</em> mig2 promoters conferring high-level activity during pathogenic growth in maize,</div> <div class="source">Mol. Plant Microbe Interact</div> ., <div class="volume">18</div>, <div class="fpage">75</div>–<div class="lpage">87</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20cis-active%20elements%20in%20Ustilago%20maydis%20mig2%20promoters%20conferring%20high-level%20activity%20during%20pathogenic%20growth%20in%20maize%2C&amp;author=J.W.%20Farfsing&amp;author=K.%20Auffarth&amp;author=C.W.%20Basse&amp;publication_year=2005&amp;journal=Mol.%20Plant%20Microbe%20Interact&amp;volume=18&amp;pages=75-87" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1094/MPMI-18-0075" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1094%2FMPMI-18-0075" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1094%2FMPMI-18-0075"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15672821" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx031-B89" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B89" class="ref-content "><div class="label -label">89</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Schaeffer</div> <div class="given-names">H.J.</div></div>, <div class="name string-name"><div class="surname">Leykam</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Walton</div> <div class="given-names">J.D.</div></div></div> <div class="year">1994</div>, 
<div class="article-title">Cloning and targeted gene disruption of EXG1, encoding exo-beta 1, 3-glucanase, in the phytopathogenic fungus <em>Cochliobolus carbonum</em>,</div> <div class="source">Appl. Environ. Microbiol</div> ., <div class="volume">60</div>, <div class="fpage">594</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cloning%20and%20targeted%20gene%20disruption%20of%20EXG1%2C%20encoding%20exo-beta%201%2C%203-glucanase%2C%20in%20the%20phytopathogenic%20fungus%20Cochliobolus%20carbonum%2C&amp;author=H.J.%20Schaeffer&amp;author=J.%20Leykam&amp;author=J.D.%20Walton&amp;publication_year=1994&amp;journal=Appl.%20Environ.%20Microbiol&amp;volume=60&amp;pages=594-8" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8135518" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx031-B90" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B90" class="ref-content "><div class="label -label">90</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Schauwecker</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Wanner</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Kahmann</div> <div class="given-names">R.</div></div></div> <div class="year">1995</div>, 
<div class="article-title">Filament-specific expression of a cellulase gene in the dimorphic fungus <em>Ustilago maydis</em>,</div> <div class="source">Biol. Chem. Hoppe Seyler</div> , <div class="volume">376</div>, <div class="fpage">617</div>–<div class="lpage">25</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Filament-specific%20expression%20of%20a%20cellulase%20gene%20in%20the%20dimorphic%20fungus%20Ustilago%20maydis%2C&amp;author=F.%20Schauwecker&amp;author=G.%20Wanner&amp;author=R.%20Kahmann&amp;publication_year=1995&amp;journal=Biol.%20Chem.%20Hoppe%20Seyler&amp;volume=376&amp;pages=617-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1515/bchm3.1995.376.10.617" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1515%2Fbchm3.1995.376.10.617" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1515%2Fbchm3.1995.376.10.617"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8590631" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx031-B91" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B91" class="ref-content "><div class="label -label">91</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Teertstra</div> <div class="given-names">W.R.</div></div>, <div class="name string-name"><div class="surname">Deelstra</div> <div class="given-names">H.J.</div></div>, <div class="name string-name"><div class="surname">Vranes</div> <div class="given-names">M.</div></div></div>, et al.   <div class="year">2006</div>, 
<div class="article-title">Repellents have functionally replaced hydrophobins in mediating attachment to a hydrophobic surface and in formation of hydrophobic aerial hyphae in <em>Ustilago maydis</em>,</div> <div class="source">Microbiology</div> , <div class="volume">152</div>, <div class="fpage">3607</div>–<div class="lpage">12</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Repellents%20have%20functionally%20replaced%20hydrophobins%20in%20mediating%20attachment%20to%20a%20hydrophobic%20surface%20and%20in%20formation%20of%20hydrophobic%20aerial%20hyphae%20in%20Ustilago%20maydis%2C&amp;author=W.R.%20Teertstra&amp;author=H.J.%20Deelstra&amp;author=M.%20Vranes&amp;publication_year=2006&amp;journal=Microbiology&amp;volume=152&amp;pages=3607-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1099/mic.0.29034-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1099%2Fmic.0.29034-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1099%2Fmic.0.29034-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17159213" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx031-B92" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B92" class="ref-content "><div class="label -label">92</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tanaka</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Brefort</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Neidig</div> <div class="given-names">N.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">A secreted <em>Ustilago maydis</em> effector promotes virulence by targeting anthocyanin biosynthesis in maize,</div> <div class="source">Elife</div> , <div class="volume">3</div>, <div class="fpage">125</div>–<div class="lpage">31</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20secreted%20Ustilago%20maydis%20effector%20promotes%20virulence%20by%20targeting%20anthocyanin%20biosynthesis%20in%20maize%2C&amp;author=S.%20Tanaka&amp;author=T.%20Brefort&amp;author=N.%20Neidig&amp;publication_year=2014&amp;journal=Elife&amp;volume=3&amp;pages=125-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.7554/eLife.01355" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.7554%2FeLife.01355" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.7554%2FeLife.01355"> </span></div> </div></div></div></div></div><div content-id="dsx031-B93" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B93" class="ref-content "><div class="label -label">93</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kämper</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Kahmann</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Bölker</div> <div class="given-names">M.</div></div></div>, et al.   <div class="year">2006</div>, 
<div class="article-title">Insights from the genome of the biotrophic fungal plant pathogen <em>Ustilago maydis</em>,</div> <div class="source">Nature</div> , <div class="volume">444</div>, <div class="fpage">97</div>–<div class="lpage">101</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Insights%20from%20the%20genome%20of%20the%20biotrophic%20fungal%20plant%20pathogen%20Ustilago%20maydis%2C&amp;author=J.%20K%C3%A4mper&amp;author=R.%20Kahmann&amp;author=M.%20B%C3%B6lker&amp;publication_year=2006&amp;journal=Nature&amp;volume=444&amp;pages=97-101" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature05248" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature05248" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature05248"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17080091" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx031-B94" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B94" class="ref-content "><div class="label -label">94</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Quoc</div> <div class="given-names">N.B.</div></div>, <div class="name string-name"><div class="surname">Chau</div> <div class="given-names">N.N.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">The role of cell wall degrading enzymes in pathogenesis of <em>Magnaporthe oryzae</em>,</div> <div class="source">Curr. Protein Pept. Sci</div> ., <div class="volume">17</div>, [Epub ahead of print].</div></div></div></div><div content-id="dsx031-B95" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B95" class="ref-content "><div class="label -label">95</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Blanco-Ulate</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Morales-Cruz</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Amrine</div> <div class="given-names">K.C.</div></div>, <div class="name string-name"><div class="surname">Labavitch</div> <div class="given-names">J.M.</div></div>, <div class="name string-name"><div class="surname">Powell</div> <div class="given-names">A.L.</div></div>, <div class="name string-name"><div class="surname">Cantu</div> <div class="given-names">D.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Genome-wide transcriptional profiling of <em>Botrytis cinerea</em> genes targeting plant cell walls during infections of different hosts,</div> <div class="source">Front. Plant Sci</div> ., <div class="volume">5</div>, <div class="fpage">435</div>.</div></div></div></div><div content-id="dsx031-B96" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B96" class="ref-content "><div class="label -label">96</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Winnenburg</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Urban</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Beacham</div> <div class="given-names">A.</div></div></div>, et al.   <div class="year">2008</div>, 
<div class="article-title">PHI-base update: additions to the pathogen host interaction database,</div> <div class="source">Nucleic Acids Res</div> ., <div class="volume">36</div>, <div class="fpage">572</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PHI-base%20update%3A%20additions%20to%20the%20pathogen%20host%20interaction%20database%2C&amp;author=R.%20Winnenburg&amp;author=M.%20Urban&amp;author=A.%20Beacham&amp;publication_year=2008&amp;journal=Nucleic%20Acids%20Res&amp;volume=36&amp;pages=572-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkm858" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkm858" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkm858"> </span></div> </div></div></div></div></div><div content-id="dsx031-B97" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B97" class="ref-content "><div class="label -label">97</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Giraud</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Gladieux</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Gavrilets</div> <div class="given-names">S.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Linking emerging fungal plant diseases and ecological speciation,</div> <div class="source">Trends Ecol. Evol</div> ., <div class="volume">25</div>, <div class="fpage">387</div>–<div class="lpage">95</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Linking%20emerging%20fungal%20plant%20diseases%20and%20ecological%20speciation%2C&amp;author=T.%20Giraud&amp;author=P.%20Gladieux&amp;author=S.%20Gavrilets&amp;publication_year=2010&amp;journal=Trends%20Ecol.%20Evol&amp;volume=25&amp;pages=387-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tree.2010.03.006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tree.2010.03.006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tree.2010.03.006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20434790" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx031-B98" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx031-B98" class="ref-content "><div class="label -label">98</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Stukenbrock</div> <div class="given-names">E.H.</div></div>, <div class="name string-name"><div class="surname">Mcdonald</div> <div class="given-names">B.A.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Population genetics of fungal and Oomycete effectors involved in gene-for-gene interactions,</div> <div class="source">Mol. Plant Microbe Interact</div> ., <div class="volume">22</div>, <div class="fpage">371</div>–<div class="lpage">80</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Population%20genetics%20of%20fungal%20and%20Oomycete%20effectors%20involved%20in%20gene-for-gene%20interactions%2C&amp;author=E.H.%20Stukenbrock&amp;author=B.A.%20Mcdonald&amp;publication_year=2009&amp;journal=Mol.%20Plant%20Microbe%20Interact&amp;volume=22&amp;pages=371-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1094/MPMI-22-4-0371" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1094%2FMPMI-22-4-0371" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1094%2FMPMI-22-4-0371"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19271952" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2017. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=3979487&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/supplementary_fig_s1_dsx031.png?Expires=2147483647&amp;Signature=CnZcTvYT3i5XWT6EKbc3889ymAipNGHbJZMGyiU6FhWEy9eSfdWzzEO6Hv9wGkPNork~xvf0RMTJUs7FHJYzKePGrKXp0pUPW6mp314fPHnpbAvW1wRiWignpeQEpNvMxNKEc95XD08FS-FV92jsVgq62~B4vQnqFwNl9~mba-yxWJFVh3ni9eM~YSuwW4HsZ~EqVebMcNLp1PN4kt9fb8itQygLZ~x9CIrbATxzrs8u~hjoTpcDnF20iucLqbMrAMvHLBIr6wo-x967Y2wI9xB2PyQ6SR1oy7ehm-qvt4L8RLweo5-uKWgpriRxT7tEztMCPYGcgR0eSQNNssYNfQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Figure S1</div></a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/supplementary_fig_s2_dsx031.png?Expires=2147483647&amp;Signature=KIlbX3c6fzYQ2jkuD3wLfIzWQF8JyvU0yqkzgpxWpCDIlD21ADw8MthaJFsehsVErxc8rC1UuvRHw3BTVrWFunr1Su5Rmv2RTh6qdqPkdLfYXR81c-3zWVmXX1Zgpml09qlVUUeFrVyiVi5Tvs7aPOzXUOWV-IRwK05K2iEwvMVTQxNx~BPXdY49ejcoAOaJJXJaiQykUr7UwX8GtasZ1NVFkugoQEn0mPK82DeTFmQpNvkA42JptCOQjjgbiN115b47j-quIDedPeqrUtt67Ij9pR7AL~4DRhqNA-TgbTVPBH7ERxhAczWkibfnYNmMvHTEbRrmt2L5q2~GVtYfHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Figure S2</div></a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/supplementary_file_s1_dsx031.xls?Expires=2147483647&amp;Signature=LOnchKUZsFz9Wa7rvxMFb5TC5rLpt9pvMMfI8juF5HER~ivV2jeP3unCJ2D8t-OmLfjW1lwRPMjipds26lxPgP5YRFf9xeDIrqsnfEkwhSJMSFrkDmdkHFbRS74enuy8BhQghp2JkihawbZTgoSB84whiqjKQ4DgrX--9sThfQQfil5kGonaNMkDwka18e78JV-W3mhsfpYQ80V9cR88ohKWf6bP-L6heIMAD0BGLaemKJOJjsb4gV84pcQ0JSEYtfVEVlNBYdzzVEvj01oaBtnLpqKaNtAVwCzvhrZyrPCcrdCsyjoVc22urF2NlEtJbh~6Fg-uDy6zsn5r78K4XQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary File S1</div></a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/supplementary_file_s2_dsx031.xls?Expires=2147483647&amp;Signature=C3nu5QEfgdRa~jCnGIVFbNCudwHZzuQ7Db-iD5Bi9-v5EMf2lI9s1NMQGFQV-NKFiySF1jNDyb5AYBxYTvDDPgk4SIaoWZOezeO2qqk81f5AjhnHwP8OQ4H9b-i7BJ1Cm~Uxw47idENq2jeN5CSL1curf~q3d0lzGcdk~wR3-E6dfpPP4T~TT2a-6KDrxBHJNROlKLDT8r1Uc7GSW9aAFgCi~AXUKpCXgOn7qC4m-4rQ4L-d7E2CtP~tPxxfr2EzkmkwJOZiSiHw~QC8gH1hqHiYv7glLevmTtBgJ7xsnipcOtTmbhpijvMzyZup~PHgXr2Tc-9Vxw-Z2XQV2K2sXg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary File S2</div></a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/supplementary_file_s3_dsx031.xls?Expires=2147483647&amp;Signature=QDBuwo~19ELM4sp~jq1TS0FSeZbHZm2UNm9AQWpE5eUaZhdgQC61vZnJ2gqLZ5OMjjst2nTaRkFoXu~O1l85mqJnMNUB5mNbqPie5wkYXsFjoisjSt4qtvVNqoSya-BbOtk37AeiIKotmyPazPjw-1BWwY29n0wTvisUszZj91DjqtzlaxGbPMuFuUQrfKUpk6aashvIDyAhdifYkTrSm2-gmukok~Pi1uBY70wNWaDSP0Vhb7EOQ6mrOjIbicT~mPVwziRH3YT9o5pJHOZvcaqNf7szHO0ceyBiE6RxJH5Y1eFrZZO~LjLkUGFTSIyJWl7XaNbd6Qxc8gENgQMLrg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary File S3</div></a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/supplementary_file_s4_dsx031.doc?Expires=2147483647&amp;Signature=pw3dyn0M9udLtjHHIqS8EQKXqyxmJDHuskO2reXFIkm5E82UuB2uxkJSdDNyEh0WROTXzGxRW6TWkhufuJYB~-wBk72NF780jTm2BjkAdcicSDER1eGwxMACWQiPiwjTyC6j6XfxWpTRyFwCDruVahUmsMqfT4hKFL4ZPvNrjNgesKx9b0eKkqocqFhi4L0KD8JDzJeDOllspB0vq70xnI-WNsauwL97KIh-nFxL7Fv9dp3aVZlFoPIgHy3~UgIzVg3uV2sEd0qHXr4isTudbR~6DEhq0yBcHQJO9M8KlkdIZeqYOfg1yHIoJcQ7BZCSWozE6BVT8O9KnK~R59VCZQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary File S4</div></a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/supplementary_file_s5_dsx031.xls?Expires=2147483647&amp;Signature=KDGxjoR4DZ-wYw5wWaEOHP73FdeFM9iu0N2vG7bcFRimkLHkI9xwMzORlg7GEjmGx2WpoijgGAFEVdauXY3weS9eZ43WxRDxeJi0f~YA7O3lwbd7TGz9izpIZqwO2qYQLD~CbiiQVEsCptSqZj-OhoDDzDQ6oZPBjrpIfPn5pCBKtaw9kToh84AfxCKGzbIKK83zjUv2ToylMg3xFFnnqBI6OOTqMzMTlUH5KIa~NgOgWpSQLCM4rH9B9RMMyCLQjLLqRRhKhEKutsaCQg3BZfdaesdzc2elyyGKUTEDTNOeniambk-1-QqVGYHuhii2Sr~a5-hYHgxsfpo4Ewt7Tg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary File S5</div></a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/supplementary_file_s6_dsx031.xls?Expires=2147483647&amp;Signature=JWQJ4pECpkr6KXI49rp-iv7qTJ-P~uJw0MMNzkFDUDsJc8ZqjDZ1K5KqFaeAW3exb2jA5qOZpPMjpbaccFJK0xprsG8NdzU5ChhGLfeC-O4ZV3lGGjGYjSgWqfzHPzx4k2S~LLsp7ezDRxN07DTCNRpATjRJz5a6ZZyKbsgSPz9g1pVttpIdj0uIEB-v7qqWetVKDghHbw373-zDZZAEIGYea7XDPhDx4fI8dkr9xJcX0sYSiSKgoIYs9dsBsA4rbOdkgGyX5jk1DL2o9M5dKufeTA7Iz1v9-K3527yfCgk1Qxzvc1VXADS9jS7Ow7VlN3yxOvCnsaRG7fdwP3Gf7w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary File S6</div></a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/supplementary_file_s7_dsx031.xls?Expires=2147483647&amp;Signature=eQHaI44KpmhLgyut6HLGqZAARApSpoFDjjP0pfN~YaV6FXkasaP3CPJ2NTst-kYh2IAs6Uu9sh8f9GPvMb8WogC56BkdUrR2V43NVZj9B1Yd91GuQgoQKnL4fvbuzm7yWQgyRknu7zVOKh7ozAXiDc0Jtk7woLxIC0HXttLs1d4eIXt0ogwyknIr7-kJovXeVes1ZLuA-Z59kxnwH5NfU6RoTpDU-ke58ZBlQg9YYLZzOxt~M-P2qUSlDm4vqAosNN6xdybXQIYtdJ02YqxOCHokGi0M0zKDFdRAaYN4CwcDuqRqccSn9w2~WpeblZeNb4gZRgReQTbvrdKeCyU1tQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary File S7</div></a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/supplementary_file_s8_dsx031.xls?Expires=2147483647&amp;Signature=O1QEMkJ3OnFlt5ASQ82bp92hCFKAon9EwYkqKKL~YHSKS8eSMZOZbLTM57MIrPKZmFxycb7HRfUGDKuCicjegavh118w17gXKaZWq~YsLVarGqfpLUOuCV01KTcpBLRHnyFWpSn905mgLfVIFJRer7xW56SWgbVTiN6mppAkSR5fJEWBYiIvDFmI5sN7RTAsel4kD5q3yiv~gnkeIKX4hI8nqlf87g7szGg-lWPZBxYhPeFPTUC7Oydv9yMmIuKJzs66VX39O1UVRvJF2eo6e74z5zc6uwbMTic0s1oBC5bCJ8z6oyGMUWl5RbeE9nyHQCZwLIRmXUu2k9ZuB43NpQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary File S8</div></a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/supplementary_file_s9_dsx031.xls?Expires=2147483647&amp;Signature=dzTBy3EHWcbAODpD58irEQwQCJ1yzhfx9Tu~L8HO4nOQu7xyE2jwQAUIOFNThl9PQbsDTvnenU-QBhQOCJs8PkYEw7YWuOWM2qo3yNiQMqm1knOruZWl4uDk8gZ7AV07raCvb8-HzvfA-D8LgnCITGPzFj5-IrCzj141o4EMBBAsLQw6WO2tLKWTs2vcQ7cYV9cTDQBwAVCLxQKVR6quxYlcew7C8LjX7~j99P0BCDnspSSziI2m0g2yr15beFD--yPml-dU-rGX0xA2XxzIndrlUmJyf-IekQAafgkrfEWQk4bR-iSltpmp9XzxO05Ut38258HtGTQQboXwYTGnPw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary File S9</div></a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/supplementary_file_s10_dsx031.xls?Expires=2147483647&amp;Signature=FmcYZbBIWtQAxAf94TKUCoonCEI3xGN~2-TDfHMDM-jQqlgcmL3NzZhQEddDgBaUQehW6IkA4J4HaSexjWwAtJFvcnIYNg9-xtIfvsxxyYA9qy8GYP6InETRIoWLAePTXW503jgBhuxgPjuwH1T71yMDbWjnPM-cJoD-062ZS4hL0STZ1-gEmBq746fErS1FvpZjX1ZSAisuZg5LwothJ7lQcHIcdxysBFVvAGDbIM3PTM-3i0Va3lNXuZaujhFj1SpJMlQmmjCv2GJPMwM0nLyL1kXczuH3PZwQ9FMoK2FFS~klUhRbcRV5KB-6vrnJ3HJ8cHkq-60Vo8GRJwNcmQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary File S10</div></a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/supplementary_file_s11_dsx031.xls?Expires=2147483647&amp;Signature=QmnDxGZR55w2PZ8yTcg3Ym-0ty99sTDbMA7Zk47FsYAuQFAPBOVzsPQI~tPrnYft5Vox2khUuFDuiGIIxAWRVm7Eg-HR4IN5jOMT~s8OcJJi-NPYBaG5PS5F1SwHiL1HRcpcu~~gN73bjgya7MaqVIAEeMUz6VMKEDNNetDobLEmdYl3Pa4BXpR7BrfdCSKUejU5t2Jdl3gmKN0x3Ya08fTd2s9cj4sIdgw4KLMHb6mZIlsD4Np3MBWx1f-Ic1As1y7ZDyB~weywQ9F99UErIo-q1KWV~w2-rgC~pSm2nTvVa2JlDz0ehkKx3TBHVhy9-RsACFU8WlZK4UkOKWoH6A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary File S11</div></a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/supplementary_file_s12_dsx031.xlsx?Expires=2147483647&amp;Signature=GLeuw~IZs~qXLUV1eoQSnZXi6QyEfwXM9mL8UjcewXluDdiruGYoBaD3onJUOT2XzFL6f0qCTICnk9U1tMnMJDOcoCrC6vy-nez63aJY-FuZv3zsNt5ZkgGInGZ2ahphsmluHYvKM2~5c0yRp~b9JRp2gZuB176zqRF40ZYFTk9JsrWGzsswfKO9Yso0BUA7y6ck28klTUBJWlPVe1XxYsICgMyZQYLmk3kOFQjQvjGlC9CMcRrqL0uIj48VUiuuC3jClOnHtTCF2uRACTg1MAbVlaEGDrxeplsPQFEV7df3zPImGFbHUS0~Uwmd0Fb22iE15w9xhba67VUHIT7m1w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary File S12</div></a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/supplementary_table_s1_dsx031.doc?Expires=2147483647&amp;Signature=uX36CZpPdsgp6fSq8CDrrqx7RrWCL9SinAkAkiNn7-LXhC7bq2ZWM7-uBYY78rMMTSiV85jy3n67040CYeXGQZSAGsXOhk6huLmwDkec5rvXvEglZgSadOiSOU6uniGQuWPwYUAk3vyc1hel786nt-IMMykT1gtFx0~3efSPVu71sIsIQWrZdWbl6FtGp~GPCdHe9IjVEDqf9ZHJeX5W3uo3Y5zQ~KX6SEO1soDP~AQbbqxeaZqWMhSod5aihTj1T94O6aGsene3X8dIm5D9~NjpPKGo-VkiJzmHCXz~9h-1OmCSbFB7mCq8Pk5~r~CRvvvx~du8OsK2c6lawR58Ng__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S1</div></a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/supplementary_table_s2_dsx031.doc?Expires=2147483647&amp;Signature=qfEXKZL4hpvAJCzH5H7loYPOeD3U028EvFSBZSAMTR89o5noi2NPZnb0r~Iaf8YafKKG3410omjsF4zLjml2tPrDOSX-x6KaMDwacq2253psHts4wDGeCiQhZC2ULAvN9Ir3MVmsC21bYaT6Bogh0pOGQjlrpl6r~XfGLLbw6fC6vm5zPXXHVDBleRJeKWdtUHPXc2JXLHVmRmujzBjIIntaOjzwNpr1tX~5ROFr9nTk234fHZC~GnDMfgq2bssK3KanFhK79exn9SPVWoFsFFYjOYvfXR-NmhvdI08V87u5YcvmDm6jAlVTcBoL2OgAkZimaRCKJ8dS~r2SuQmGNg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S2</div></a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/supplementary_table_s3_dsx031.doc?Expires=2147483647&amp;Signature=gza~8LfhvpTI6eeK5gxBdDwCA-DGJrcQCPTHr-Cf0m6PvXyGWzrswy-3RLVqZGVHIk5o4GYEJSAOH7b8dos~uJ~rDAYRa9uGkyUII-Io1E-nsNqiuoQTTDqErz77Xd0nhLEubCLLOq-EXEW2RG6q19N0V-wnsorFyjKEMgwiyPwd-EorqMBUvjECsEea5oGTZLfbfRy-B0SzkKzcr75DBVRLAH4OGGQl7zsH95FGQGboxALdsPOwwbp4YpnekynaBg3qXZgBxg2Wb-Bt5Ru6McbuxIjiMKo5jrU1ckmjdLaYvx1UskfRkfAyPUjmk9Rm851o2Vpaf--dCWqz-0HVYQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S3</div></a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/supplementary_table_s4_dsx031.doc?Expires=2147483647&amp;Signature=eM4VlDBKbowysYVGl-J5FRQL~FX8QCoJSXTtKsCXP7YFQc7uATnZHMF0ejB2WleO~n4c5VzYXGZWhcZBr2uqxEpnOxzlyczJo4KkZHzzSnQTrpKgq22gaiVAkJTDBGTlibMhQezD4WXdfMmSz3GQoluBilL-8jkd1BSd1Ng~KWGfVEbuGYspIcF17tNbldNQPfldbthsdiEffQAaX4C4CkXj7gE0Hj3RN1TbY6j8s2Vn1horPFJCfYvILFXQVgIcmBHkA8gy-BpNOYd9iX0HPS3fFUu~ugwD3D1h70vAKOsPG6OgJmTgVT3T3d~NN0GsPcS4cgx7N4Lx6Nvi3viZ~Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S4</div></a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx031/1/supplementary_table_s5_dsx031.doc?Expires=2147483647&amp;Signature=SCKCjLmgMyQu7dN5bHBjv4xyxZwJ9C-tjUmi9E1vJYQ8CR2SNwJAw-ofVpyjMtO6CiAd67Kk-OZMuHcEB4gFh2hDCAKO5vzqNHsmFKp9JOxwWv7W8UZsJD19cnmijWcrkVe9TGo~vr4z~4PLRaF~vfyWrBz4JyXuViPj6Hgp6i2ZkRJInWfX68FDndyRP3N378PP6rjtB6d-CSAmMo5WpgQLzwCOf1mEIpti6roIM1Bzw0pMePsy1q1KTqUJLT0FDThlusOBYpB5tuhi7SOKV6Xq4R~GYXMP8xxJK479LW84yvBKcLV9NSCqbnnIcjVh86u4MHI4oe~ipUPCuQOZcg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S5</div></a> - doc file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">1,191</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000418420400008&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">4</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsx031"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="3979487"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000418420400008&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f24%2f6%2f635%2f3979487">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000418420400008&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (4)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F24%2F6%2F635%2F3979487" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/3979487'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Comparative whole-genome analysis reveals artificial selection effects on Ustilago esculenta genome" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsx031");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f24%2f6%2f635%2f3979487">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

