

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/24/1/37/2670112","@type":"ScholarlyArticle","name":"Quantum changes in  Helicobacter pylori  gene expression accompany host-adaptation","datePublished":"2016-10-31","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/24/1","@type":"PublicationIssue","issueNumber":"1","datePublished":"2017-02-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsw046","keywords":["BabA","expression","<italic>Helicobacter pylori</italic>","host adaptation","Lewis antigen"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/24/1/37/2670112","author":[{"name":"Chua, Eng-Guan","affiliation":"1 The Marshall Centre for Infectious Diseases Research and Training, School of Pathology and Laboratory Medicine, The University of Western Australia, Nedlands, Western Australia, Australia","@type":"Person"},{"name":"Wise, Michael J.","affiliation":"1 The Marshall Centre for Infectious Diseases Research and Training, School of Pathology and Laboratory Medicine, The University of Western Australia, Nedlands, Western Australia, Australia   2 School of Computer Science and Software Engineering, The University of Western Australia, Australia","@type":"Person"},{"name":"Khosravi, Yalda","affiliation":"3 Department of Medical Microbiology, University of Malaya, Kuala Lumpur, Malaysia","@type":"Person"},{"name":"Seow, Shih-Wee","affiliation":"4 National Cancer Centre of Singapore, Singapore","@type":"Person"},{"name":"Amoyo, Arlaine A.","affiliation":"4 National Cancer Centre of Singapore, Singapore","@type":"Person"},{"name":"Pettersson, Sven","affiliation":"5 Department of Microbiology, Tumor and Cell Biology (MTC), Karolinska Institutet, Stockholm, Sweden   6 LKC School of Medicine, Nanyang Technological University, Singapore   7 SCELSE Microbiome Centre, Nanyang Technological University, Singapore","@type":"Person"},{"name":"Peters, Fanny","affiliation":"1 The Marshall Centre for Infectious Diseases Research and Training, School of Pathology and Laboratory Medicine, The University of Western Australia, Nedlands, Western Australia, Australia","@type":"Person"},{"name":"Tay, Chin-Yen","affiliation":"1 The Marshall Centre for Infectious Diseases Research and Training, School of Pathology and Laboratory Medicine, The University of Western Australia, Nedlands, Western Australia, Australia","@type":"Person"},{"name":"Perkins, Timothy T.","affiliation":"1 The Marshall Centre for Infectious Diseases Research and Training, School of Pathology and Laboratory Medicine, The University of Western Australia, Nedlands, Western Australia, Australia","@type":"Person"},{"name":"Loke, Mun-Fai","affiliation":"3 Department of Medical Microbiology, University of Malaya, Kuala Lumpur, Malaysia","@type":"Person"},{"name":"Marshall, Barry J.","affiliation":"1 The Marshall Centre for Infectious Diseases Research and Training, School of Pathology and Laboratory Medicine, The University of Western Australia, Nedlands, Western Australia, Australia   8 UM Marshall Centre, High Impact Research Building, University of Malaya, 50603 Kuala Lumpur, Malaysia","@type":"Person"},{"name":"Vadivelu, Jamuna","affiliation":"3 Department of Medical Microbiology, University of Malaya, Kuala Lumpur, Malaysia","@type":"Person"}],"description":"Abstract.  Helicobacter pylori is a highly successful gastric pathogen. High genomic plasticity allows its adaptation to changing host environments. Complete ge","pageStart":"37","pageEnd":"49","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/1/1/m_dnares_24_1cover.png?Expires=2147483647&Signature=EXqRr~0bCuMnRgDilKfchgK8HfHjGx8NkkSlq7yX2eX9zgSD-cVPXfhVwj7kuo0oVIPzSixZgq5R9TFiP~OOgz8ybfHpezxNM2sixr9SShZ0K1LhBnO6322A9GuHybRZq3DsAqdkcL3vL0duBS2dzcsmON4yge6VoGCPo6kzuwcmoM-0E9z7h2t0RSYSTFF896ydP484zApd5VOOafFGZ8RyK5rTzaKEJMAeldpSZjhoijN~lS-XpQOIA9~KMdeI0frmy3d5rUnXtXBDnLYMvKmhHEY2pEtZAo9h7w-9UPONxolZtZkyNv5UUIS3z9gJT3HGO3QFQvDoa8hGk7vH6Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Quantum changes in  Helicobacter pylori  gene expression accompany host-adaptation","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/1/1/m_dnares_24_1cover.png?Expires=2147483647&Signature=EXqRr~0bCuMnRgDilKfchgK8HfHjGx8NkkSlq7yX2eX9zgSD-cVPXfhVwj7kuo0oVIPzSixZgq5R9TFiP~OOgz8ybfHpezxNM2sixr9SShZ0K1LhBnO6322A9GuHybRZq3DsAqdkcL3vL0duBS2dzcsmON4yge6VoGCPo6kzuwcmoM-0E9z7h2t0RSYSTFF896ydP484zApd5VOOafFGZ8RyK5rTzaKEJMAeldpSZjhoijN~lS-XpQOIA9~KMdeI0frmy3d5rUnXtXBDnLYMvKmhHEY2pEtZAo9h7w-9UPONxolZtZkyNv5UUIS3z9gJT3HGO3QFQvDoa8hGk7vH6Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Quantum changes in  Helicobacter pylori  gene expression accompany host-adaptation" />
    <meta property="og:description" content="Abstract.  Helicobacter pylori is a highly successful gastric pathogen. High genomic plasticity allows its adaptation to changing host environments. Complete ge" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/24/1/37/2670112" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2016-10-31" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Chua, Eng-Guan" /><meta name="citation_author_institution" content="The Marshall Centre for Infectious Diseases Research and Training, School of Pathology and Laboratory Medicine, The University of Western Australia, Nedlands, Western Australia, Australia" /><meta name="citation_author" content="Wise, Michael J." /><meta name="citation_author_institution" content="The Marshall Centre for Infectious Diseases Research and Training, School of Pathology and Laboratory Medicine, The University of Western Australia, Nedlands, Western Australia, Australia" /><meta name="citation_author_institution" content="School of Computer Science and Software Engineering, The University of Western Australia, Australia" /><meta name="citation_author" content="Khosravi, Yalda" /><meta name="citation_author_institution" content="Department of Medical Microbiology, University of Malaya, Kuala Lumpur, Malaysia" /><meta name="citation_author" content="Seow, Shih-Wee" /><meta name="citation_author_institution" content="National Cancer Centre of Singapore, Singapore" /><meta name="citation_author" content="Amoyo, Arlaine A." /><meta name="citation_author_institution" content="National Cancer Centre of Singapore, Singapore" /><meta name="citation_author" content="Pettersson, Sven" /><meta name="citation_author_institution" content="Department of Microbiology, Tumor and Cell Biology (MTC), Karolinska Institutet, Stockholm, Sweden" /><meta name="citation_author_institution" content="LKC School of Medicine, Nanyang Technological University, Singapore" /><meta name="citation_author_institution" content="SCELSE Microbiome Centre, Nanyang Technological University, Singapore" /><meta name="citation_author" content="Peters, Fanny" /><meta name="citation_author_institution" content="The Marshall Centre for Infectious Diseases Research and Training, School of Pathology and Laboratory Medicine, The University of Western Australia, Nedlands, Western Australia, Australia" /><meta name="citation_author" content="Tay, Chin-Yen" /><meta name="citation_author_institution" content="The Marshall Centre for Infectious Diseases Research and Training, School of Pathology and Laboratory Medicine, The University of Western Australia, Nedlands, Western Australia, Australia" /><meta name="citation_author" content="Perkins, Timothy T." /><meta name="citation_author_institution" content="The Marshall Centre for Infectious Diseases Research and Training, School of Pathology and Laboratory Medicine, The University of Western Australia, Nedlands, Western Australia, Australia" /><meta name="citation_author" content="Loke, Mun-Fai" /><meta name="citation_author_institution" content="Department of Medical Microbiology, University of Malaya, Kuala Lumpur, Malaysia" /><meta name="citation_author" content="Marshall, Barry J." /><meta name="citation_author_institution" content="The Marshall Centre for Infectious Diseases Research and Training, School of Pathology and Laboratory Medicine, The University of Western Australia, Nedlands, Western Australia, Australia" /><meta name="citation_author_institution" content="UM Marshall Centre, High Impact Research Building, University of Malaya, 50603 Kuala Lumpur, Malaysia" /><meta name="citation_author" content="Vadivelu, Jamuna" /><meta name="citation_author_institution" content="Department of Medical Microbiology, University of Malaya, Kuala Lumpur, Malaysia" /><meta name="citation_title" content="Quantum changes in Helicobacter pylori gene expression accompany host-adaptation" /><meta name="citation_firstpage" content="37" /><meta name="citation_lastpage" content="49" /><meta name="citation_doi" content="10.1093/dnares/dsw046" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="24" /><meta name="citation_issue" content="1" /><meta name="citation_publication_date" content="2017/02/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Unidentified curved bacilli in the stomach of patients with gastritis and peptic ulceration;  citation_journal_title=Lancet;  citation_year=1984;  citation_volume=1;  citation_pages=1311-15; " /><meta name="citation_reference" content="citation_title=The relation of Helicobacter pylori to gastric adenocarcinoma and lymphoma: pathophysiology, epidemiology, screening, clinical presentation, treatment, and prevention;  citation_journal_title=Med. Clin. North Am;  citation_year=2005;  citation_volume=89;  citation_pages=313-44; " /><meta name="citation_reference" content="citation_title=Schistosomes, liver flukes and Helicobacter pylori. IARC Working Group on the Evaluation of Carcinogenic Risks to Humans. Lyon, 7-14 June 1994;  citation_journal_title=IARC Monogr. Eval. Carcinog. Risks Hum;  citation_year=1994;  citation_volume=61;  citation_pages=1-241; " /><meta name="citation_reference" content="citation_title=Population structure of Helicobacter pylori among ethnic groups in Malaysia: recent acquisition of the bacterium by the Malay population;  citation_journal_title=BMC Microbiol;  citation_year=2009;  citation_volume=9;  citation_pages=126" /><meta name="citation_reference" content="citation_title=The complete genome sequence of the gastric pathogen Helicobacter pylori;  citation_journal_title=Nature;  citation_year=1997;  citation_volume=388;  citation_pages=539-47; " /><meta name="citation_reference" content="citation_title=Suppression of homologous and homeologous recombination by the bacterial MutS2 protein;  citation_journal_title=Mol. Cell;  citation_year=2005;  citation_volume=17;  citation_pages=113-120; " /><meta name="citation_reference" content="citation_title=Structural and functional divergence of MutS2 from bacterial MutS1 and eukaryotic MSH4-MSH5 homologs;  citation_journal_title=J. Bacteriol;  citation_year=2005;  citation_volume=187;  citation_pages=3528-37; " /><meta name="citation_reference" content="citation_title=Persistence of Helicobacter pylori Infection: Genetic and Epigenetic Diversity;  citation_journal_title=InTech;  citation_year=2014; " /><meta name="citation_reference" content="citation_title=Unexpected role for Helicobacter pylori DNA polymerase I as a source of genetic variability;  citation_journal_title=PLoS Genet;  citation_year=2011;  citation_volume=7;  citation_pages=e1002152" /><meta name="citation_reference" content="citation_title=Phase variation: how to create and coordinate population diversity;  citation_journal_title=Curr. Opin. Microbiol;  citation_year=2011;  citation_volume=14;  citation_pages=205-211; " /><meta name="citation_reference" content="citation_title=A variable homopolymeric G-repeat defines small RNA-mediated posttranscriptional regulation of a chemotaxis receptor in Helicobacter pylori;  citation_journal_title=Proc Natl. Acad. Sci. USA;  citation_year=2014;  citation_volume=111;  citation_pages=E501-10; " /><meta name="citation_reference" content="citation_title=Phase variation in Helicobacter pylori lipopolysaccharide due to changes in the lengths of poly(C) tracts in alpha3-fucosyltransferase genes;  citation_journal_title=Infect. Immun;  citation_year=1999;  citation_volume=67;  citation_pages=5361-66; " /><meta name="citation_reference" content="citation_title=A repetitive DNA element regulates expression of the Helicobacter pylori sialic acid binding adhesin by a rheostat-like mechanism;  citation_journal_title=PLoS Pathog;  citation_year=2014;  citation_volume=10;  citation_pages=e1004234" /><meta name="citation_reference" content="citation_title=Host specificity of Helicobacter pylori strains and host responses in experimentally challenged nonhuman primates;  citation_journal_title=Gastroenterology;  citation_year=1999;  citation_volume=116;  citation_pages=90-6; " /><meta name="citation_reference" content="citation_title=Characterization of virulence factors of mouse-adapted Helicobacter pylori strain SS1 and effects on gastric hydrophobicity;  citation_journal_title=Digest Dis. Sci;  citation_year=2001;  citation_volume=46;  citation_pages=1943-51; " /><meta name="citation_reference" content="citation_title=Experimental infection of Mongolian gerbils with wild-type and mutant Helicobacter pylori strains;  citation_journal_title=Infect. Immun;  citation_year=1998;  citation_volume=66;  citation_pages=4856-66; " /><meta name="citation_reference" content="citation_title=Comparing the genomes of Helicobacter pylori clinical strain UM032 and Mice-adapted derivatives;  citation_journal_title=Gut Pathog;  citation_year=2013;  citation_volume=5;  citation_pages=25" /><meta name="citation_reference" content="citation_title=The origin of the Haitian cholera outbreak strain;  citation_journal_title=New Engl. J. Med;  citation_year=2011;  citation_volume=364;  citation_pages=33-42; " /><meta name="citation_reference" content="citation_title=Choosing a benchtop sequencing machine to characterise Helicobacter pylori genomes;  citation_journal_title=PLoS One;  citation_year=2013;  citation_volume=8;  citation_pages=e67539" /><meta name="citation_reference" content="citation_title=Geneious Basic: an integrated and extendable desktop software platform for the organization and analysis of sequence data;  citation_journal_title=Bioinformatics;  citation_year=2012;  citation_volume=28;  citation_pages=1647-1649; " /><meta name="citation_reference" content="citation_title=The RAST Server: rapid annotations using subsystems technology;  citation_journal_title=BMC Genomics;  citation_year=2008;  citation_volume=9;  citation_pages=75" /><meta name="citation_reference" content="citation_title=GeneMarkS: a self-training method for prediction of gene starts in microbial genomes. Implications for finding sequence motifs in regulatory regions;  citation_journal_title=Nucleic Acids Res;  citation_year=2001;  citation_volume=29;  citation_pages=2607-18; " /><meta name="citation_reference" content="citation_title=Prodigal: prokaryotic gene recognition and translation initiation site identification;  citation_journal_title=BMC Bioinformatics;  citation_year=2010;  citation_volume=11;  citation_pages=119" /><meta name="citation_reference" content="citation_title=Identifying bacterial genes and endosymbiont DNA with Glimmer;  citation_journal_title=Bioinformatics;  citation_year=2007;  citation_volume=23;  citation_pages=673-79; " /><meta name="citation_reference" content="citation_title=Mauve: multiple alignment of conserved genomic sequence with rearrangements;  citation_journal_title=Genome Res;  citation_year=2004;  citation_volume=14;  citation_pages=1394-403; " /><meta name="citation_reference" content="citation_title=CDD: NCBI's conserved domain database;  citation_journal_title=Nucleic Acids Res;  citation_year=2015;  citation_volume=43;  citation_pages=D222-226; " /><meta name="citation_reference" content="citation_title=Analysis of relative gene expression data using real-time quantitative PCR and the 2(-Delta Delta C(T)) Method;  citation_journal_title=Methods;  citation_year=2001;  citation_volume=25;  citation_pages=402-8; " /><meta name="citation_reference" content="citation_title=A mutation burst during the acute phase of Helicobacter pylori infection in humans and rhesus macaques;  citation_journal_title=Nat. Commun;  citation_year=2014;  citation_volume=5;  citation_pages=4165" /><meta name="citation_reference" content="citation_title=Cloning and characterization of a novel membrane-associated antigenic protein of Helicobacter pylori;  citation_journal_title=Infect. Immun;  citation_year=1999;  citation_volume=67;  citation_pages=286-93; " /><meta name="citation_reference" content="citation_title=Helicobacter pylori membrane protein 1: a new carcinogenic factor of Helicobacter pylori;  citation_journal_title=Cancer Res;  citation_year=2001;  citation_volume=61;  citation_pages=6356-59; " /><meta name="citation_reference" content="citation_title=Comparative genomics of Helicobacter pylori: analysis of the outer membrane protein families;  citation_journal_title=Infect. Immun;  citation_year=2000;  citation_volume=68;  citation_pages=4155-68; " /><meta name="citation_reference" content="citation_title=Expression of the BabA adhesin during experimental infection with Helicobacter pylori;  citation_journal_title=Infect. Immun;  citation_year=2010;  citation_volume=78;  citation_pages=1593-600; " /><meta name="citation_reference" content="citation_title=Metastability of Helicobacter pylori bab adhesin genes and dynamics in Lewis b antigen binding;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2004;  citation_volume=101;  citation_pages=16923-928; " /><meta name="citation_reference" content="citation_title=Dynamics of Lewis b binding and sequence variation of the babA adhesin gene during chronic Helicobacter pylori infection in humans;  citation_journal_title=MBio;  citation_year=2014;  citation_volume=5; " /><meta name="citation_reference" content="citation_title=Genotypic profile of the outer membrane proteins BabA and BabB in clinical isolates of Helicobacter pylori;  citation_journal_title=Infect. Immun;  citation_year=2006;  citation_volume=74;  citation_pages=4375-78; " /><meta name="citation_reference" content="citation_title=Helicobacter pylori bab genes during chronic colonization;  citation_journal_title=Int. J. Mol. Epidemiol. Genet;  citation_year=2011;  citation_volume=2;  citation_pages=286-91; " /><meta name="citation_reference" content="citation_title=Helicobacter pylori adhesin binding fucosylated histo-blood group antigens revealed by retagging;  citation_journal_title=Science;  citation_year=1998;  citation_volume=279;  citation_pages=373-77; " /><meta name="citation_reference" content="citation_title=Evolution in an oncogenic bacterial species with extreme genome plasticity: Helicobacter pylori East Asian genomes;  citation_journal_title=BMC Microbiol;  citation_year=2011;  citation_volume=11;  citation_pages=104" /><meta name="citation_reference" content="citation_title=EMBOSS: the European Molecular Biology Open Software Suite;  citation_journal_title=Trends Genet;  citation_year=2000;  citation_volume=16;  citation_pages=276-77; " /><meta name="citation_reference" content="citation_title=C-terminal amino acids of Helicobacter pylori alpha1,3/4 fucosyltransferases determine type I and type II transfer;  citation_journal_title=J. Biol. Chem;  citation_year=2003;  citation_volume=278;  citation_pages=21893-900; " /><meta name="citation_reference" content="citation_title=Cloning and heterologous expression of an alpha1,3-fucosyltransferase gene from the gastric pathogen Helicobacter pylori;  citation_journal_title=J. Biol. Chem;  citation_year=1997;  citation_volume=272;  citation_pages=21357-363; " /><meta name="citation_reference" content="citation_title=Molecular cloning and functional expression of a novel Helicobacter pylori alpha-1,4 fucosyltransferase;  citation_journal_title=Glycobiology;  citation_year=2005;  citation_volume=15;  citation_pages=1076-83; " /><meta name="citation_reference" content="citation_title=Novel Helicobacter pylori alpha1,2-fucosyltransferase, a key enzyme in the synthesis of Lewis antigens;  citation_journal_title=Microbiology;  citation_year=1999;  citation_volume=145 (Pt 11);  citation_pages=3245-53; " /><meta name="citation_reference" content="citation_title=Lewis antigens in Helicobacter pylori: biosynthesis and phase variation;  citation_journal_title=Mol. Microbiol;  citation_year=2000;  citation_volume=36;  citation_pages=1187-96; " /><meta name="citation_reference" content="citation_title=Novel functions for glycosyltransferases Jhp0562 and GalT in Lewis antigen synthesis and variation in Helicobacter pylori;  citation_journal_title=Infect. Immun;  citation_year=2012;  citation_volume=80;  citation_pages=1593-605; " /><meta name="citation_reference" content="citation_title=Cloning and characterization of the alpha(1,3/4) fucosyltransferase of Helicobacter pylori;  citation_journal_title=J. Biol. Chem;  citation_year=2000;  citation_volume=275;  citation_pages=4988-94; " /><meta name="citation_reference" content="citation_title=Phase variation of the gonococcal siderophore receptor FetA;  citation_journal_title=Mol. Microbiol;  citation_year=2000;  citation_volume=36;  citation_pages=585-93; " /><meta name="citation_reference" content="citation_title=Phase variation of H. influenzae fimbriae: transcriptional control of two divergent genes through a variable combined promoter region;  citation_journal_title=Cell;  citation_year=1993;  citation_volume=73;  citation_pages=1187-96; " /><meta name="citation_reference" content="citation_title=A novel phase variation mechanism in the meningococcus driven by a ligand-responsive repressor and differential spacing of distal promoter elements;  citation_journal_title=PLoS Pathog;  citation_year=2009;  citation_volume=5;  citation_pages=e1000710" /><meta name="citation_reference" content="citation_title=Enterohemorrhagic Escherichia coli senses low biotin status in the large intestine for colonization and infection;  citation_journal_title=Nat. Commun;  citation_year=2015;  citation_volume=6;  citation_pages=6592" /><meta name="citation_reference" content="citation_title=Systematic identification of selective essential genes in Helicobacter pylori by genome prioritization and allelic replacement mutagenesis;  citation_journal_title=J. Bacteriol;  citation_year=2001;  citation_volume=183;  citation_pages=1259-68; " /><meta name="citation_reference" content="citation_title=In vivo dissection of the Helicobacter pylori Fur regulatory circuit by genome-wide location analysis;  citation_journal_title=J. Bacteriol;  citation_year=2006;  citation_volume=188;  citation_pages=4654-62; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/24/1/37/10731799/dsw046.pdf" /><meta name="description" content="Abstract.  Helicobacter pylori is a highly successful gastric pathogen. High genomic plasticity allows its adaptation to changing host environments. Complete ge" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/24/1/37/2670112" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/24/1/37/2670112" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/24/1/37/2670112");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsw046';
    </script>
<title>
	Quantum changes in Helicobacter pylori gene expression accompany host-adaptation | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./2670112" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BaWPATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQ2h1YSwgRW5nLUd1YW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJUaGUgTWFyc2hhbGwgQ2VudHJlIGZvciBJbmZlY3Rpb3VzIERpc2Vhc2VzIFJlc2VhcmNoIGFuZCBUcmFpbmluZywgU2Nob29sIG9mIFBhdGhvbG9neSBhbmQgTGFib3JhdG9yeSBNZWRpY2luZSwgVGhlIFVuaXZlcnNpdHkgb2YgV2VzdGVybiBBdXN0cmFsaWEsIE5lZGxhbmRzLCBXZXN0ZXJuIEF1c3RyYWxpYSwgQXVzdHJhbGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iV2lzZSwgTWljaGFlbCBKLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlRoZSBNYXJzaGFsbCBDZW50cmUgZm9yIEluZmVjdGlvdXMgRGlzZWFzZXMgUmVzZWFyY2ggYW5kIFRyYWluaW5nLCBTY2hvb2wgb2YgUGF0aG9sb2d5IGFuZCBMYWJvcmF0b3J5IE1lZGljaW5lLCBUaGUgVW5pdmVyc2l0eSBvZiBXZXN0ZXJuIEF1c3RyYWxpYSwgTmVkbGFuZHMsIFdlc3Rlcm4gQXVzdHJhbGlhLCBBdXN0cmFsaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJTY2hvb2wgb2YgQ29tcHV0ZXIgU2NpZW5jZSBhbmQgU29mdHdhcmUgRW5naW5lZXJpbmcsIFRoZSBVbml2ZXJzaXR5IG9mIFdlc3Rlcm4gQXVzdHJhbGlhLCBBdXN0cmFsaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLaG9zcmF2aSwgWWFsZGEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIE1lZGljYWwgTWljcm9iaW9sb2d5LCBVbml2ZXJzaXR5IG9mIE1hbGF5YSwgS3VhbGEgTHVtcHVyLCBNYWxheXNpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlNlb3csIFNoaWgtV2VlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTmF0aW9uYWwgQ2FuY2VyIENlbnRyZSBvZiBTaW5nYXBvcmUsIFNpbmdhcG9yZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkFtb3lvLCBBcmxhaW5lIEEuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTmF0aW9uYWwgQ2FuY2VyIENlbnRyZSBvZiBTaW5nYXBvcmUsIFNpbmdhcG9yZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlBldHRlcnNzb24sIFN2ZW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIE1pY3JvYmlvbG9neSwgVHVtb3IgYW5kIENlbGwgQmlvbG9neSAoTVRDKSwgS2Fyb2xpbnNrYSBJbnN0aXR1dGV0LCBTdG9ja2hvbG0sIFN3ZWRlbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkxLQyBTY2hvb2wgb2YgTWVkaWNpbmUsIE5hbnlhbmcgVGVjaG5vbG9naWNhbCBVbml2ZXJzaXR5LCBTaW5nYXBvcmUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJTQ0VMU0UgTWljcm9iaW9tZSBDZW50cmUsIE5hbnlhbmcgVGVjaG5vbG9naWNhbCBVbml2ZXJzaXR5LCBTaW5nYXBvcmUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJQZXRlcnMsIEZhbm55IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iVGhlIE1hcnNoYWxsIENlbnRyZSBmb3IgSW5mZWN0aW91cyBEaXNlYXNlcyBSZXNlYXJjaCBhbmQgVHJhaW5pbmcsIFNjaG9vbCBvZiBQYXRob2xvZ3kgYW5kIExhYm9yYXRvcnkgTWVkaWNpbmUsIFRoZSBVbml2ZXJzaXR5IG9mIFdlc3Rlcm4gQXVzdHJhbGlhLCBOZWRsYW5kcywgV2VzdGVybiBBdXN0cmFsaWEsIEF1c3RyYWxpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlRheSwgQ2hpbi1ZZW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJUaGUgTWFyc2hhbGwgQ2VudHJlIGZvciBJbmZlY3Rpb3VzIERpc2Vhc2VzIFJlc2VhcmNoIGFuZCBUcmFpbmluZywgU2Nob29sIG9mIFBhdGhvbG9neSBhbmQgTGFib3JhdG9yeSBNZWRpY2luZSwgVGhlIFVuaXZlcnNpdHkgb2YgV2VzdGVybiBBdXN0cmFsaWEsIE5lZGxhbmRzLCBXZXN0ZXJuIEF1c3RyYWxpYSwgQXVzdHJhbGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iUGVya2lucywgVGltb3RoeSBULiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlRoZSBNYXJzaGFsbCBDZW50cmUgZm9yIEluZmVjdGlvdXMgRGlzZWFzZXMgUmVzZWFyY2ggYW5kIFRyYWluaW5nLCBTY2hvb2wgb2YgUGF0aG9sb2d5IGFuZCBMYWJvcmF0b3J5IE1lZGljaW5lLCBUaGUgVW5pdmVyc2l0eSBvZiBXZXN0ZXJuIEF1c3RyYWxpYSwgTmVkbGFuZHMsIFdlc3Rlcm4gQXVzdHJhbGlhLCBBdXN0cmFsaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJMb2tlLCBNdW4tRmFpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBNZWRpY2FsIE1pY3JvYmlvbG9neSwgVW5pdmVyc2l0eSBvZiBNYWxheWEsIEt1YWxhIEx1bXB1ciwgTWFsYXlzaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJNYXJzaGFsbCwgQmFycnkgSi4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJUaGUgTWFyc2hhbGwgQ2VudHJlIGZvciBJbmZlY3Rpb3VzIERpc2Vhc2VzIFJlc2VhcmNoIGFuZCBUcmFpbmluZywgU2Nob29sIG9mIFBhdGhvbG9neSBhbmQgTGFib3JhdG9yeSBNZWRpY2luZSwgVGhlIFVuaXZlcnNpdHkgb2YgV2VzdGVybiBBdXN0cmFsaWEsIE5lZGxhbmRzLCBXZXN0ZXJuIEF1c3RyYWxpYSwgQXVzdHJhbGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iVU0gTWFyc2hhbGwgQ2VudHJlLCBIaWdoIEltcGFjdCBSZXNlYXJjaCBCdWlsZGluZywgVW5pdmVyc2l0eSBvZiBNYWxheWEsIDUwNjAzIEt1YWxhIEx1bXB1ciwgTWFsYXlzaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJWYWRpdmVsdSwgSmFtdW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBNZWRpY2FsIE1pY3JvYmlvbG9neSwgVW5pdmVyc2l0eSBvZiBNYWxheWEsIEt1YWxhIEx1bXB1ciwgTWFsYXlzaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IlF1YW50dW0gY2hhbmdlcyBpbiBIZWxpY29iYWN0ZXIgcHlsb3JpIGdlbmUgZXhwcmVzc2lvbiBhY2NvbXBhbnkgaG9zdC1hZGFwdGF0aW9uIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iMzciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjQ5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHN3MDQ2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjI0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSIxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTcvMDIvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1VbmlkZW50aWZpZWQgY3VydmVkIGJhY2lsbGkgaW4gdGhlIHN0b21hY2ggb2YgcGF0aWVudHMgd2l0aCBnYXN0cml0aXMgYW5kIHBlcHRpYyB1bGNlcmF0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1MYW5jZXQ7ICBjaXRhdGlvbl95ZWFyPTE5ODQ7ICBjaXRhdGlvbl92b2x1bWU9MTsgIGNpdGF0aW9uX3BhZ2VzPTEzMTEtMTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSByZWxhdGlvbiBvZiBIZWxpY29iYWN0ZXIgcHlsb3JpIHRvIGdhc3RyaWMgYWRlbm9jYXJjaW5vbWEgYW5kIGx5bXBob21hOiBwYXRob3BoeXNpb2xvZ3ksIGVwaWRlbWlvbG9neSwgc2NyZWVuaW5nLCBjbGluaWNhbCBwcmVzZW50YXRpb24sIHRyZWF0bWVudCwgYW5kIHByZXZlbnRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1lZC4gQ2xpbi4gTm9ydGggQW07ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9ODk7ICBjaXRhdGlvbl9wYWdlcz0zMTMtNDQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNjaGlzdG9zb21lcywgbGl2ZXIgZmx1a2VzIGFuZCBIZWxpY29iYWN0ZXIgcHlsb3JpLiBJQVJDIFdvcmtpbmcgR3JvdXAgb24gdGhlIEV2YWx1YXRpb24gb2YgQ2FyY2lub2dlbmljIFJpc2tzIHRvIEh1bWFucy4gTHlvbiwgNy0xNCBKdW5lIDE5OTQ7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUlBUkMgTW9ub2dyLiBFdmFsLiBDYXJjaW5vZy4gUmlza3MgSHVtOyAgY2l0YXRpb25feWVhcj0xOTk0OyAgY2l0YXRpb25fdm9sdW1lPTYxOyAgY2l0YXRpb25fcGFnZXM9MS0yNDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBvcHVsYXRpb24gc3RydWN0dXJlIG9mIEhlbGljb2JhY3RlciBweWxvcmkgYW1vbmcgZXRobmljIGdyb3VwcyBpbiBNYWxheXNpYTogcmVjZW50IGFjcXVpc2l0aW9uIG9mIHRoZSBiYWN0ZXJpdW0gYnkgdGhlIE1hbGF5IHBvcHVsYXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBNaWNyb2Jpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTEyNiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBjb21wbGV0ZSBnZW5vbWUgc2VxdWVuY2Ugb2YgdGhlIGdhc3RyaWMgcGF0aG9nZW4gSGVsaWNvYmFjdGVyIHB5bG9yaTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTM4ODsgIGNpdGF0aW9uX3BhZ2VzPTUzOS00NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3VwcHJlc3Npb24gb2YgaG9tb2xvZ291cyBhbmQgaG9tZW9sb2dvdXMgcmVjb21iaW5hdGlvbiBieSB0aGUgYmFjdGVyaWFsIE11dFMyIHByb3RlaW47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xNzsgIGNpdGF0aW9uX3BhZ2VzPTExMy0xMjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN0cnVjdHVyYWwgYW5kIGZ1bmN0aW9uYWwgZGl2ZXJnZW5jZSBvZiBNdXRTMiBmcm9tIGJhY3RlcmlhbCBNdXRTMSBhbmQgZXVrYXJ5b3RpYyBNU0g0LU1TSDUgaG9tb2xvZ3M7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEJhY3RlcmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xODc7ICBjaXRhdGlvbl9wYWdlcz0zNTI4LTM3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QZXJzaXN0ZW5jZSBvZiBIZWxpY29iYWN0ZXIgcHlsb3JpIEluZmVjdGlvbjogR2VuZXRpYyBhbmQgRXBpZ2VuZXRpYyBEaXZlcnNpdHk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUluVGVjaDsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VW5leHBlY3RlZCByb2xlIGZvciBIZWxpY29iYWN0ZXIgcHlsb3JpIEROQSBwb2x5bWVyYXNlIEkgYXMgYSBzb3VyY2Ugb2YgZ2VuZXRpYyB2YXJpYWJpbGl0eTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9ZTEwMDIxNTIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QaGFzZSB2YXJpYXRpb246IGhvdyB0byBjcmVhdGUgYW5kIGNvb3JkaW5hdGUgcG9wdWxhdGlvbiBkaXZlcnNpdHk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUN1cnIuIE9waW4uIE1pY3JvYmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTIwNS0yMTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgdmFyaWFibGUgaG9tb3BvbHltZXJpYyBHLXJlcGVhdCBkZWZpbmVzIHNtYWxsIFJOQS1tZWRpYXRlZCBwb3N0dHJhbnNjcmlwdGlvbmFsIHJlZ3VsYXRpb24gb2YgYSBjaGVtb3RheGlzIHJlY2VwdG9yIGluIEhlbGljb2JhY3RlciBweWxvcmk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MgTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MTExOyAgY2l0YXRpb25fcGFnZXM9RTUwMS0xMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGhhc2UgdmFyaWF0aW9uIGluIEhlbGljb2JhY3RlciBweWxvcmkgbGlwb3BvbHlzYWNjaGFyaWRlIGR1ZSB0byBjaGFuZ2VzIGluIHRoZSBsZW5ndGhzIG9mIHBvbHkoQykgdHJhY3RzIGluIGFscGhhMy1mdWNvc3lsdHJhbnNmZXJhc2UgZ2VuZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUluZmVjdC4gSW1tdW47ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9Njc7ICBjaXRhdGlvbl9wYWdlcz01MzYxLTY2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHJlcGV0aXRpdmUgRE5BIGVsZW1lbnQgcmVndWxhdGVzIGV4cHJlc3Npb24gb2YgdGhlIEhlbGljb2JhY3RlciBweWxvcmkgc2lhbGljIGFjaWQgYmluZGluZyBhZGhlc2luIGJ5IGEgcmhlb3N0YXQtbGlrZSBtZWNoYW5pc207ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgUGF0aG9nOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9ZTEwMDQyMzQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ib3N0IHNwZWNpZmljaXR5IG9mIEhlbGljb2JhY3RlciBweWxvcmkgc3RyYWlucyBhbmQgaG9zdCByZXNwb25zZXMgaW4gZXhwZXJpbWVudGFsbHkgY2hhbGxlbmdlZCBub25odW1hbiBwcmltYXRlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2FzdHJvZW50ZXJvbG9neTsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT0xMTY7ICBjaXRhdGlvbl9wYWdlcz05MC02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaGFyYWN0ZXJpemF0aW9uIG9mIHZpcnVsZW5jZSBmYWN0b3JzIG9mIG1vdXNlLWFkYXB0ZWQgSGVsaWNvYmFjdGVyIHB5bG9yaSBzdHJhaW4gU1MxIGFuZCBlZmZlY3RzIG9uIGdhc3RyaWMgaHlkcm9waG9iaWNpdHk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPURpZ2VzdCBEaXMuIFNjaTsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT00NjsgIGNpdGF0aW9uX3BhZ2VzPTE5NDMtNTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV4cGVyaW1lbnRhbCBpbmZlY3Rpb24gb2YgTW9uZ29saWFuIGdlcmJpbHMgd2l0aCB3aWxkLXR5cGUgYW5kIG11dGFudCBIZWxpY29iYWN0ZXIgcHlsb3JpIHN0cmFpbnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUluZmVjdC4gSW1tdW47ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9NjY7ICBjaXRhdGlvbl9wYWdlcz00ODU2LTY2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wYXJpbmcgdGhlIGdlbm9tZXMgb2YgSGVsaWNvYmFjdGVyIHB5bG9yaSBjbGluaWNhbCBzdHJhaW4gVU0wMzIgYW5kIE1pY2UtYWRhcHRlZCBkZXJpdmF0aXZlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R3V0IFBhdGhvZzsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9MjUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgb3JpZ2luIG9mIHRoZSBIYWl0aWFuIGNob2xlcmEgb3V0YnJlYWsgc3RyYWluOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OZXcgRW5nbC4gSi4gTWVkOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTM2NDsgIGNpdGF0aW9uX3BhZ2VzPTMzLTQyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaG9vc2luZyBhIGJlbmNodG9wIHNlcXVlbmNpbmcgbWFjaGluZSB0byBjaGFyYWN0ZXJpc2UgSGVsaWNvYmFjdGVyIHB5bG9yaSBnZW5vbWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIE9uZTsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9ZTY3NTM5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZWlvdXMgQmFzaWM6IGFuIGludGVncmF0ZWQgYW5kIGV4dGVuZGFibGUgZGVza3RvcCBzb2Z0d2FyZSBwbGF0Zm9ybSBmb3IgdGhlIG9yZ2FuaXphdGlvbiBhbmQgYW5hbHlzaXMgb2Ygc2VxdWVuY2UgZGF0YTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9Mjg7ICBjaXRhdGlvbl9wYWdlcz0xNjQ3LTE2NDk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBSQVNUIFNlcnZlcjogcmFwaWQgYW5ub3RhdGlvbnMgdXNpbmcgc3Vic3lzdGVtcyB0ZWNobm9sb2d5OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTc1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZU1hcmtTOiBhIHNlbGYtdHJhaW5pbmcgbWV0aG9kIGZvciBwcmVkaWN0aW9uIG9mIGdlbmUgc3RhcnRzIGluIG1pY3JvYmlhbCBnZW5vbWVzLiBJbXBsaWNhdGlvbnMgZm9yIGZpbmRpbmcgc2VxdWVuY2UgbW90aWZzIGluIHJlZ3VsYXRvcnkgcmVnaW9uczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9Mjk7ICBjaXRhdGlvbl9wYWdlcz0yNjA3LTE4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qcm9kaWdhbDogcHJva2FyeW90aWMgZ2VuZSByZWNvZ25pdGlvbiBhbmQgdHJhbnNsYXRpb24gaW5pdGlhdGlvbiBzaXRlIGlkZW50aWZpY2F0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz0xMTkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JZGVudGlmeWluZyBiYWN0ZXJpYWwgZ2VuZXMgYW5kIGVuZG9zeW1iaW9udCBETkEgd2l0aCBHbGltbWVyOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0yMzsgIGNpdGF0aW9uX3BhZ2VzPTY3My03OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWF1dmU6IG11bHRpcGxlIGFsaWdubWVudCBvZiBjb25zZXJ2ZWQgZ2Vub21pYyBzZXF1ZW5jZSB3aXRoIHJlYXJyYW5nZW1lbnRzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgUmVzOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9MTM5NC00MDM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNERDogTkNCSSdzIGNvbnNlcnZlZCBkb21haW4gZGF0YWJhc2U7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTQzOyAgY2l0YXRpb25fcGFnZXM9RDIyMi0yMjY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuYWx5c2lzIG9mIHJlbGF0aXZlIGdlbmUgZXhwcmVzc2lvbiBkYXRhIHVzaW5nIHJlYWwtdGltZSBxdWFudGl0YXRpdmUgUENSIGFuZCB0aGUgMigtRGVsdGEgRGVsdGEgQyhUKSkgTWV0aG9kOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1NZXRob2RzOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9NDAyLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgbXV0YXRpb24gYnVyc3QgZHVyaW5nIHRoZSBhY3V0ZSBwaGFzZSBvZiBIZWxpY29iYWN0ZXIgcHlsb3JpIGluZmVjdGlvbiBpbiBodW1hbnMgYW5kIHJoZXN1cyBtYWNhcXVlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBDb21tdW47ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPTQxNjUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DbG9uaW5nIGFuZCBjaGFyYWN0ZXJpemF0aW9uIG9mIGEgbm92ZWwgbWVtYnJhbmUtYXNzb2NpYXRlZCBhbnRpZ2VuaWMgcHJvdGVpbiBvZiBIZWxpY29iYWN0ZXIgcHlsb3JpOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1JbmZlY3QuIEltbXVuOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTY3OyAgY2l0YXRpb25fcGFnZXM9Mjg2LTkzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IZWxpY29iYWN0ZXIgcHlsb3JpIG1lbWJyYW5lIHByb3RlaW4gMTogYSBuZXcgY2FyY2lub2dlbmljIGZhY3RvciBvZiBIZWxpY29iYWN0ZXIgcHlsb3JpOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DYW5jZXIgUmVzOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTYxOyAgY2l0YXRpb25fcGFnZXM9NjM1Ni01OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyYXRpdmUgZ2Vub21pY3Mgb2YgSGVsaWNvYmFjdGVyIHB5bG9yaTogYW5hbHlzaXMgb2YgdGhlIG91dGVyIG1lbWJyYW5lIHByb3RlaW4gZmFtaWxpZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUluZmVjdC4gSW1tdW47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9Njg7ICBjaXRhdGlvbl9wYWdlcz00MTU1LTY4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FeHByZXNzaW9uIG9mIHRoZSBCYWJBIGFkaGVzaW4gZHVyaW5nIGV4cGVyaW1lbnRhbCBpbmZlY3Rpb24gd2l0aCBIZWxpY29iYWN0ZXIgcHlsb3JpOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1JbmZlY3QuIEltbXVuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTc4OyAgY2l0YXRpb25fcGFnZXM9MTU5My02MDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1ldGFzdGFiaWxpdHkgb2YgSGVsaWNvYmFjdGVyIHB5bG9yaSBiYWIgYWRoZXNpbiBnZW5lcyBhbmQgZHluYW1pY3MgaW4gTGV3aXMgYiBhbnRpZ2VuIGJpbmRpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTEwMTsgIGNpdGF0aW9uX3BhZ2VzPTE2OTIzLTkyODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RHluYW1pY3Mgb2YgTGV3aXMgYiBiaW5kaW5nIGFuZCBzZXF1ZW5jZSB2YXJpYXRpb24gb2YgdGhlIGJhYkEgYWRoZXNpbiBnZW5lIGR1cmluZyBjaHJvbmljIEhlbGljb2JhY3RlciBweWxvcmkgaW5mZWN0aW9uIGluIGh1bWFuczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TUJpbzsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vdHlwaWMgcHJvZmlsZSBvZiB0aGUgb3V0ZXIgbWVtYnJhbmUgcHJvdGVpbnMgQmFiQSBhbmQgQmFiQiBpbiBjbGluaWNhbCBpc29sYXRlcyBvZiBIZWxpY29iYWN0ZXIgcHlsb3JpOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1JbmZlY3QuIEltbXVuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTc0OyAgY2l0YXRpb25fcGFnZXM9NDM3NS03ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGVsaWNvYmFjdGVyIHB5bG9yaSBiYWIgZ2VuZXMgZHVyaW5nIGNocm9uaWMgY29sb25pemF0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1JbnQuIEouIE1vbC4gRXBpZGVtaW9sLiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0yOyAgY2l0YXRpb25fcGFnZXM9Mjg2LTkxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IZWxpY29iYWN0ZXIgcHlsb3JpIGFkaGVzaW4gYmluZGluZyBmdWNvc3lsYXRlZCBoaXN0by1ibG9vZCBncm91cCBhbnRpZ2VucyByZXZlYWxlZCBieSByZXRhZ2dpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9Mjc5OyAgY2l0YXRpb25fcGFnZXM9MzczLTc3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Fdm9sdXRpb24gaW4gYW4gb25jb2dlbmljIGJhY3RlcmlhbCBzcGVjaWVzIHdpdGggZXh0cmVtZSBnZW5vbWUgcGxhc3RpY2l0eTogSGVsaWNvYmFjdGVyIHB5bG9yaSBFYXN0IEFzaWFuIGdlbm9tZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBNaWNyb2Jpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz0xMDQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FTUJPU1M6IHRoZSBFdXJvcGVhbiBNb2xlY3VsYXIgQmlvbG9neSBPcGVuIFNvZnR3YXJlIFN1aXRlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UcmVuZHMgR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz0yNzYtNzc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUMtdGVybWluYWwgYW1pbm8gYWNpZHMgb2YgSGVsaWNvYmFjdGVyIHB5bG9yaSBhbHBoYTEsMy80IGZ1Y29zeWx0cmFuc2ZlcmFzZXMgZGV0ZXJtaW5lIHR5cGUgSSBhbmQgdHlwZSBJSSB0cmFuc2ZlcjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQmlvbC4gQ2hlbTsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0yNzg7ICBjaXRhdGlvbl9wYWdlcz0yMTg5My05MDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNsb25pbmcgYW5kIGhldGVyb2xvZ291cyBleHByZXNzaW9uIG9mIGFuIGFscGhhMSwzLWZ1Y29zeWx0cmFuc2ZlcmFzZSBnZW5lIGZyb20gdGhlIGdhc3RyaWMgcGF0aG9nZW4gSGVsaWNvYmFjdGVyIHB5bG9yaTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQmlvbC4gQ2hlbTsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT0yNzI7ICBjaXRhdGlvbl9wYWdlcz0yMTM1Ny0zNjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1vbGVjdWxhciBjbG9uaW5nIGFuZCBmdW5jdGlvbmFsIGV4cHJlc3Npb24gb2YgYSBub3ZlbCBIZWxpY29iYWN0ZXIgcHlsb3JpIGFscGhhLTEsNCBmdWNvc3lsdHJhbnNmZXJhc2U7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdseWNvYmlvbG9neTsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTEwNzYtODM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5vdmVsIEhlbGljb2JhY3RlciBweWxvcmkgYWxwaGExLDItZnVjb3N5bHRyYW5zZmVyYXNlLCBhIGtleSBlbnp5bWUgaW4gdGhlIHN5bnRoZXNpcyBvZiBMZXdpcyBhbnRpZ2VuczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TWljcm9iaW9sb2d5OyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTE0NSAoUHQgMTEpOyAgY2l0YXRpb25fcGFnZXM9MzI0NS01MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TGV3aXMgYW50aWdlbnMgaW4gSGVsaWNvYmFjdGVyIHB5bG9yaTogYmlvc3ludGhlc2lzIGFuZCBwaGFzZSB2YXJpYXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gTWljcm9iaW9sOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTM2OyAgY2l0YXRpb25fcGFnZXM9MTE4Ny05NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Tm92ZWwgZnVuY3Rpb25zIGZvciBnbHljb3N5bHRyYW5zZmVyYXNlcyBKaHAwNTYyIGFuZCBHYWxUIGluIExld2lzIGFudGlnZW4gc3ludGhlc2lzIGFuZCB2YXJpYXRpb24gaW4gSGVsaWNvYmFjdGVyIHB5bG9yaTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9SW5mZWN0LiBJbW11bjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT04MDsgIGNpdGF0aW9uX3BhZ2VzPTE1OTMtNjA1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DbG9uaW5nIGFuZCBjaGFyYWN0ZXJpemF0aW9uIG9mIHRoZSBhbHBoYSgxLDMvNCkgZnVjb3N5bHRyYW5zZmVyYXNlIG9mIEhlbGljb2JhY3RlciBweWxvcmk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEJpb2wuIENoZW07ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9Mjc1OyAgY2l0YXRpb25fcGFnZXM9NDk4OC05NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGhhc2UgdmFyaWF0aW9uIG9mIHRoZSBnb25vY29jY2FsIHNpZGVyb3Bob3JlIHJlY2VwdG9yIEZldEE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gTWljcm9iaW9sOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTM2OyAgY2l0YXRpb25fcGFnZXM9NTg1LTkzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QaGFzZSB2YXJpYXRpb24gb2YgSC4gaW5mbHVlbnphZSBmaW1icmlhZTogdHJhbnNjcmlwdGlvbmFsIGNvbnRyb2wgb2YgdHdvIGRpdmVyZ2VudCBnZW5lcyB0aHJvdWdoIGEgdmFyaWFibGUgY29tYmluZWQgcHJvbW90ZXIgcmVnaW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DZWxsOyAgY2l0YXRpb25feWVhcj0xOTkzOyAgY2l0YXRpb25fdm9sdW1lPTczOyAgY2l0YXRpb25fcGFnZXM9MTE4Ny05NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBub3ZlbCBwaGFzZSB2YXJpYXRpb24gbWVjaGFuaXNtIGluIHRoZSBtZW5pbmdvY29jY3VzIGRyaXZlbiBieSBhIGxpZ2FuZC1yZXNwb25zaXZlIHJlcHJlc3NvciBhbmQgZGlmZmVyZW50aWFsIHNwYWNpbmcgb2YgZGlzdGFsIHByb21vdGVyIGVsZW1lbnRzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIFBhdGhvZzsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9ZTEwMDA3MTAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FbnRlcm9oZW1vcnJoYWdpYyBFc2NoZXJpY2hpYSBjb2xpIHNlbnNlcyBsb3cgYmlvdGluIHN0YXR1cyBpbiB0aGUgbGFyZ2UgaW50ZXN0aW5lIGZvciBjb2xvbml6YXRpb24gYW5kIGluZmVjdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBDb21tdW47ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTY1OTIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TeXN0ZW1hdGljIGlkZW50aWZpY2F0aW9uIG9mIHNlbGVjdGl2ZSBlc3NlbnRpYWwgZ2VuZXMgaW4gSGVsaWNvYmFjdGVyIHB5bG9yaSBieSBnZW5vbWUgcHJpb3JpdGl6YXRpb24gYW5kIGFsbGVsaWMgcmVwbGFjZW1lbnQgbXV0YWdlbmVzaXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEJhY3RlcmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0xODM7ICBjaXRhdGlvbl9wYWdlcz0xMjU5LTY4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbiB2aXZvIGRpc3NlY3Rpb24gb2YgdGhlIEhlbGljb2JhY3RlciBweWxvcmkgRnVyIHJlZ3VsYXRvcnkgY2lyY3VpdCBieSBnZW5vbWUtd2lkZSBsb2NhdGlvbiBhbmFseXNpczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQmFjdGVyaW9sOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTE4ODsgIGNpdGF0aW9uX3BhZ2VzPTQ2NTQtNjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8yNC8xLzM3LzEwNzMxNzk5L2RzdzA0Ni5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgSGVsaWNvYmFjdGVyIHB5bG9yaSBpcyBhIGhpZ2hseSBzdWNjZXNzZnVsIGdhc3RyaWMgcGF0aG9nZW4uIEhpZ2ggZ2Vub21pYyBwbGFzdGljaXR5IGFsbG93cyBpdHMgYWRhcHRhdGlvbiB0byBjaGFuZ2luZyBob3N0IGVudmlyb25tZW50cy4gQ29tcGxldGUgZ2UiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVzLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTI2NzAxMTImeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2RkRmdV1h9ufsWMGzPNf4Qh7v8j2+o=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f1%2f37%2f2670112" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f1%2f37%2f2670112" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=96271" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/24/1"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/1/1/m_dnares_24_1cover.png?Expires=2147483647&amp;Signature=EXqRr~0bCuMnRgDilKfchgK8HfHjGx8NkkSlq7yX2eX9zgSD-cVPXfhVwj7kuo0oVIPzSixZgq5R9TFiP~OOgz8ybfHpezxNM2sixr9SShZ0K1LhBnO6322A9GuHybRZq3DsAqdkcL3vL0duBS2dzcsmON4yge6VoGCPo6kzuwcmoM-0E9z7h2t0RSYSTFF896ydP484zApd5VOOafFGZ8RyK5rTzaKEJMAeldpSZjhoijN~lS-XpQOIA9~KMdeI0frmy3d5rUnXtXBDnLYMvKmhHEY2pEtZAo9h7w-9UPONxolZtZkyNv5UUIS3z9gJT3HGO3QFQvDoa8hGk7vH6Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/24/1">
                        <div class="volume trailing-comma">Volume 24</div>
                        <div class="issue">Issue 1</div>

            </a>

        </div>
        <div class="ii-pub-date">
February 2017        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="61463012">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#61463012">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="61463014">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#61463014">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="61463022">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#61463022">2. Methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="61463044">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#61463044">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="61463079">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#61463079">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="61463085">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#61463085">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/24/1/25/2449750">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/24/1/51/2709597">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Quantum changes in <em>Helicobacter pylori</em> gene expression accompany host-adaptation
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                            <div class="meta-authors--limited">
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Eng-Guan Chua</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>The Marshall Centre for Infectious Diseases Research and Training, School of Pathology and Laboratory Medicine, The University of Western Australia, Nedlands, Western Australia, Australia</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsw046-cor1" data-open="dsw046-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +61893464817. Fax. +61893464816. Email: <a href="mailto:eng.chua@uwa.edu.au" target="_blank">eng.chua@uwa.edu.au</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Eng-Guan+Chua">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Chua E">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Chua E&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Eng-Guan Chua</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Michael J. Wise</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>The Marshall Centre for Infectious Diseases Research and Training, School of Pathology and Laboratory Medicine, The University of Western Australia, Nedlands, Western Australia, Australia</div><div class="aff"><div class="label -label">2</div>School of Computer Science and Software Engineering, The University of Western Australia, Australia</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Michael+J.+Wise">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Wise M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Wise M&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Michael J. Wise</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yalda Khosravi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div>Department of Medical Microbiology, University of Malaya, Kuala Lumpur, Malaysia</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yalda+Khosravi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Khosravi Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Khosravi Y&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Yalda Khosravi</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shih-Wee Seow</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div>National Cancer Centre of Singapore, Singapore</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shih-Wee+Seow">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Seow S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Seow S&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Shih-Wee Seow</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Arlaine A. Amoyo</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div>National Cancer Centre of Singapore, Singapore</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Arlaine+A.+Amoyo">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Amoyo A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Amoyo A&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Arlaine A. Amoyo</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Sven Pettersson</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">5</div>Department of Microbiology, Tumor and Cell Biology (MTC), Karolinska Institutet, Stockholm, Sweden</div><div class="aff"><div class="label -label">6</div>LKC School of Medicine, Nanyang Technological University, Singapore</div><div class="aff"><div class="label -label">7</div>SCELSE Microbiome Centre, Nanyang Technological University, Singapore</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Sven+Pettersson">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Pettersson S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Pettersson S&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Sven Pettersson</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Fanny Peters</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>The Marshall Centre for Infectious Diseases Research and Training, School of Pathology and Laboratory Medicine, The University of Western Australia, Nedlands, Western Australia, Australia</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Fanny+Peters">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Peters F">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Peters F&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Fanny Peters</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Chin-Yen Tay</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>The Marshall Centre for Infectious Diseases Research and Training, School of Pathology and Laboratory Medicine, The University of Western Australia, Nedlands, Western Australia, Australia</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Chin-Yen+Tay">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tay C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tay C&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Chin-Yen Tay</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Timothy T. Perkins</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>The Marshall Centre for Infectious Diseases Research and Training, School of Pathology and Laboratory Medicine, The University of Western Australia, Nedlands, Western Australia, Australia</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Timothy+T.+Perkins">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Perkins T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Perkins T&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Timothy T. Perkins</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Mun-Fai Loke</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div>Department of Medical Microbiology, University of Malaya, Kuala Lumpur, Malaysia</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Mun-Fai+Loke">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Loke M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Loke M&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Mun-Fai Loke</a>
                                    </span>
                                <a class="meta-authors--etal is-active" id="show-meta-authors" aria-controls="remaining-authors" href="#">
                                    <span>...</span> Show more</a>
                                <div id="remaining-authors" class="meta-authors--remaining" aria-hidden="true" aria-labeledby="show-meta-authors">
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Barry J. Marshall</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>The Marshall Centre for Infectious Diseases Research and Training, School of Pathology and Laboratory Medicine, The University of Western Australia, Nedlands, Western Australia, Australia</div><div class="aff"><div class="label -label">8</div>UM Marshall Centre, High Impact Research Building, University of Malaya, 50603 Kuala Lumpur, Malaysia</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Barry+J.+Marshall">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Marshall B">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Marshall B&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Barry J. Marshall</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jamuna Vadivelu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div>Department of Medical Microbiology, University of Malaya, Kuala Lumpur, Malaysia</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jamuna+Vadivelu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Vadivelu J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Vadivelu J&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Jamuna Vadivelu</a>
                                        </span>
                                </div>
                            </div>

                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 24, Issue 1, 1 February 2017, Pages 37–49, <a href='https://doi.org/10.1093/dnares/dsw046'>https://doi.org/10.1093/dnares/dsw046</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">31 October 2016</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">24 June 2016</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">23 September 2016</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="2670112" href="/dnaresearch/article-pdf/24/1/37/10731799/dsw046.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Eng-Guan Chua, Michael J. Wise, Yalda Khosravi, Shih-Wee Seow, Arlaine A. Amoyo, Sven Pettersson, Fanny Peters, Chin-Yen Tay, Timothy T. Perkins, Mun-Fai Loke, Barry J. Marshall, Jamuna Vadivelu;  Quantum changes in <em>Helicobacter pylori</em> gene expression accompany host-adaptation, <em>DNA Research</em>, Volume 24, Issue 1, 1 February 2017, Pages 37–49, <a href='https://doi.org/10.1093/dnares/dsw046'>https://doi.org/10.1093/dnares/dsw046</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/2670112?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/2670112?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/2670112?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/2670112?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/2670112?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/2670112?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/2670112?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Quantum%20changes%20in%20%26lt%3Bem%26gt%3BHelicobacter%20pylori%26lt%3B%2Fem%26gt%3B%20gene%20expression%20accompany%20host-adaptation&amp;publicationDate=2016-10-31&amp;volumeNum=24&amp;issueNum=1&amp;author=Chua%2C%20Eng-Guan%3B%20Wise%2C%20Michael%20J.&amp;startPage=37&amp;endPage=49&amp;contentId=10.1093%2Fdnares%2Fdsw046&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f1%2f37%2f2670112" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=61463012 id="61463012" class="abstract-title" >Abstract</h2>
<section class="abstract"><p><em>Helicobacter pylori</em> is a highly successful gastric pathogen. High genomic plasticity allows its adaptation to changing host environments. Complete genomes of <em>H. pylori</em> clinical isolate UM032 and its mice-adapted serial derivatives 298 and 299, generated using both PacBio RS and Illumina MiSeq sequencing technologies, were compared to identify novel elements responsible for host-adaptation. The acquisition of a <em>jhp0562</em>-like allele, which encodes for a galactosyltransferase, was identified in the mice-adapted strains. Our analysis implies a new β-1,4-galactosyltransferase role for this enzyme, essential for Le<sup>y</sup> antigen expression. Intragenomic recombination between <em>babA</em> and <em>babB</em> genes was also observed. Further, we expanded on the list of candidate genes whose expression patterns have been mediated by upstream homopolymer-length alterations to facilitate host adaption. Importantly, greater than four-fold reduction of mRNA levels was demonstrated in five genes. Among the down-regulated genes, three encode for outer membrane proteins, including BabA, BabB and HopD. As expected, a substantial reduction in BabA protein abundance was detected in mice-adapted strains 298 and 299 via Western analysis. Our results suggest that the expression of Le<sup>y</sup> antigen and reduced outer membrane protein expressions may facilitate <em>H. pylori</em> colonisation of mouse gastric epithelium.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="BabA">BabA</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="expression">expression</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Helicobacter pylori&quot;"><em>Helicobacter pylori</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;host adaptation&quot;">host adaptation</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Lewis antigen&quot;">Lewis antigen</a></div>                            <h2 scrollto-destination=61463014 id="61463014" class="section-title" data-legacy-id=SEC1>1. Introduction</h2>
<p><em>Helicobacter pylori</em> infects approximately half the world’s population. It is a Gram-negative microaerophilic pathogen that persistently colonises the human stomach. The bacterium is associated with chronic gastritis and peptic ulceration, and less commonly, with gastric adenocarcinoma and gastric mucosa-associated lymphoid tissue (MALT) lymphoma.<a reveal-id="dsw046-B1" data-open="dsw046-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a><sup>,</sup><a reveal-id="dsw046-B2" data-open="dsw046-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> Hence, the organism was classified as a class 1 carcinogen by World Health Organisation in 1994.<a reveal-id="dsw046-B3" data-open="dsw046-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a></p><p>Population analysis of <em>H. pylori</em> isolates from different geographical origins has revealed a high degree of genetic diversity within the species. Multi-locus sequence typing (MLST) analysis has shown that identical alleles are extremely rare in <em>H. pylori</em> even when sampling a large population of the same geographical origin.<a reveal-id="dsw046-B4" data-open="dsw046-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> This indicates that <em>H. pylori</em> is able to adapt to a new or rapidly changing environment by undergoing continuous genetic modification. Greater understanding of these modifications and the mechanisms by which they arise will improve the knowledge base from which we may develop new therapeutic targets.</p><p>The high genetic variation in <em>H. pylori</em> appears to be generated through intragenic and intergenic recombinations, plus extensive mutations attributed to the lack of a nucleotide mismatch repair system, such as that provided in other bacteria by MutS, MutH and MutL.<a reveal-id="dsw046-B5" data-open="dsw046-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> There are two families of MutS homologues: MutS1 and MutS2. The MutS1 family includes <em>Escherichia coli</em> MutS, which plays a role in post-replication mismatch repair. In the <em>H. pylori</em> genome, HP0621, a member of the MutS2 family has been identified. However, rather than participating in mismatch repairing, it suppresses homeologous recombination mediated by RecA.<a reveal-id="dsw046-B6" data-open="dsw046-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a><sup>,</sup><a reveal-id="dsw046-B7" data-open="dsw046-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> Compared to <em>E. coli</em>, <em>H. pylori</em> is also missing a number of elements responsible for direct repair of DNA damage such as Ada methyltransferase, AlkB oxidative demethylase and Phr/Spl photolyase, as well as several gylcosylases and endonucleases involved in base excision repair including MutM, Tag, AlkA, Mpg, YgjF, Nei and Nfo.<a reveal-id="dsw046-B8" data-open="dsw046-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> On the other hand, the presence of a DNA polymerase I that lacks proof-reading activity also contributes to the generation of genomic plasticity in <em>H. pylori</em>.<a reveal-id="dsw046-B9" data-open="dsw046-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a></p><p>Rapid phenotype switching is another strategy that enables pathogens to survive in diverse hostile environments and to establish persistent infections in novel hosts. Phenotypic switching can be achieved by epigenetic events, such as DNA methylation and gene silencing mediated by non-coding RNAs. It can also occur via phase-variation-associated genetic modifications, such as site-specific recombination, inversion of promoter elements, insertional inactivation mediated by insertion sequences and slipped-strand mispairing of simple sequence repeats (SSRs) including homopolymeric tracts.<a reveal-id="dsw046-B10" data-open="dsw046-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a><sup>,</sup><a reveal-id="dsw046-B11" data-open="dsw046-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a></p><p>Changes in homopolymeric tract length located in coding sequences are known to generate frameshift errors and hence result in expression of truncated and often non-functional proteins.<a reveal-id="dsw046-B12" data-open="dsw046-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> While intragenic homopolymeric tract alterations turn protein expression on or off, intergenic variations may alter mRNA level to fine-tune protein expression. This has recently been shown in the adhesin-encoding gene <em>sabA</em> of <em>H. pylori</em>. Altering the length of a poly(T) tract found adjacent to its -35 promoter region changes the mRNA transcript and protein expression levels, as well as the binding to the sialyl Le<sup>x</sup> receptor.<a reveal-id="dsw046-B13" data-open="dsw046-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a></p><p>To date, relatively little is known about <em>H. pylori</em> genetic adaptation in response to selective pressures from its host. It is important to identify these adaptive mutations to provide deeper insights into the exact mechanisms underlying <em>H. pylori</em> infection. Although <em>H. pylori</em> is highly adapted to the human gastric mucosa environment, several animal models comprising mice, gerbils and non-human primates have been used to advance understanding of <em>H. pylori</em> colonisation and pathogenesis.<a reveal-id="dsw046-B14 dsw046-B15 dsw046-B16" data-open="dsw046-B14 dsw046-B15 dsw046-B16" class="link link-ref link-reveal xref-bibr"><sup>14–16</sup></a> Whilst experimental studies in humans would be most likely to reveal clinically relevant details, studies in animal models are ethically superior to human challenge experiments, and can still tell us much about the genomic changes associated with host adaptation.</p><p>Here, we report a follow-up investigation to our previously announced complete genomes of the <em>H. pylori</em> clinical strain UM032 and its mice-adapted derivatives designated 298 and 299. These were sequenced on the Pacific Biosciences RS sequencing technology using the C2 chemistry.<a reveal-id="dsw046-B17" data-open="dsw046-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> PacBio RS sequencing technology produces extraordinarily long reads, thus increasing the likelihood of acquiring a complete genome sequence. As opposed to the current C5 chemistry, high error rates were observed in sequencing data generated using the earlier C2 chemistry.<a reveal-id="dsw046-B18" data-open="dsw046-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> Therefore, for this study all three strains were also subjected to whole-genome sequencing using Illumina’s MiSeq platform. Hence, the comparative data presented here provide comprehensive information about the changes associated with host change in <em>H. pylori</em>. We reveal <em>H. pylori</em> genetic elements potentially required for host-adaptation and epigenetic regulation of gene expression mediated by homopolymeric tracts present both in the upstream regions of coding sequences, within or in close proximity to the promoter elements, and within the coding sequences.</p>                            <h2 scrollto-destination=61463022 id="61463022" class="section-title" >2. Methods</h2>
                            <h3 scrollto-destination=61463023 id="61463023" class="section-title" >2.1. Bacterial strains</h3>
<p>The acquisition of <em>H. pylori</em> clinical strain UM032 and its mice-adapted derivatives, designated 298 and 299 respectively, was as previously described.<a reveal-id="dsw046-B17" data-open="dsw046-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a></p>                            <h3 scrollto-destination=61463025 id="61463025" class="section-title" >2.2. Illumina library preparation and sequencing</h3>
<p>Preparation of MiSeq library was performed using Illumina Nextera XT DNA sample preparation kit (Illumina, San Diego, CA, USA) as previously described with minor modifications.<a reveal-id="dsw046-B19" data-open="dsw046-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> In brief, 1 ng of genomic DNA was fragmented in 5 µl of Amplicon Tagment Mix and 10 µl of Tagment DNA buffer. Tagmentation reaction was performed by incubation at 55 °C for 5 min followed by neutralisation with 5 µl of Neutralise Tagment Buffer for 5 min. Tagmented DNA (25 µl) was indexed in a 50 µl limited-cycle PCR (12 cycles) as outlined in the Nextera XT protocol and subsequently purified using 25 µl of AMPure XP beads (Beckman Coulter Inc, Australia). The fragment size distribution of the purified DNA was analysed utilising a 2100 Bioanalyser with a High Sensitivity DNA assay kit (Agilent Technologies, Santa Clara, CA). DNA libraries were adjusted to 2 nM, pooled in equal volumes and then denatured with 0.2 N NaOH according to the Nextera protocol. The libraries were sequenced using 2 × 300 paired-end protocols on an Illumina MiSeq instrument (MiSeq Reagent Kit v3 for 600 cycles).</p>                            <h3 scrollto-destination=61463027 id="61463027" class="section-title" >2.3. Genome correction</h3>
<p>The generated MiSeq data were used for correction of homopolynucleotide errors present in the previously published <em>H. pylori</em> UM032, 298 and 299 complete genomes generated using PacBio C2 chemistry. In brief, MiSeq raw reads were mapped against the complete genome using the Geneious R7 read mapper with the medium sensitivity option.<a reveal-id="dsw046-B20" data-open="dsw046-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> Variants were subsequently called using the following parameters: minimum coverage = 10 and minimum variant frequency = 0.7. Reported errors in each reference genome sequence were corrected.</p>                            <h3 scrollto-destination=61463029 id="61463029" class="section-title" >2.4. Genome annotation</h3>
<p>Initially, the genome sequences were automatically annotated using the RAST (Rapid Annotation using Subsystem Technology) pipeline.<a reveal-id="dsw046-B21" data-open="dsw046-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> In addition to Glimmer (version 3.02) used in the pipeline, additional ORF prediction was performed using GeneMarkS (version 4.7a) and Prodigal (version 2.60).<a reveal-id="dsw046-B22 dsw046-B23 dsw046-B24" data-open="dsw046-B22 dsw046-B23 dsw046-B24" class="link link-ref link-reveal xref-bibr"><sup>22–24</sup></a> By majority ruling, the predicted start codons of ORFs predicted from multiple annotation engines were manually curated to enhance start codon accuracy. Genome annotations were further compared against gene sequences from 45 completely sequenced publicly available <em>H. pylori</em> genomes to allow identification of pseudogenes. In our context, a pseudogene can be a non-functional DNA sequence with a frameshift or premature stop codon, a sequence homologous to a protein-coding gene but without ORF or a full-length reference gene broken into two or more adjacent ORFs. Our manually curated genome annotations are available in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw046/1/dsw046_Supp.zip?Expires=2147483647&Signature=s3q4bhSX2C3MUF-jr0f2UZ1jrVjkiGXzFiaUNbhztd4DChTCgvQajffmhInT-1Vj~0SRO~Kttfa4bqexSP8TrhsM2dD3tPD9oxAo17tPZ377FPaKsD8p~u48AhZfGosl2DgYDxY~wAxQIvmlENgGz~hpjXyN~I6CdeKNliwIOlCIEQtChC1zhSWLmnuwd57p2crX2HhvAlPdEQQh2bsMNnLu1COOQRPn604GZfBH8hQezXkfEFO3oCZfpwLaMYrQ~H4BcWpJOG4ygAaGy1rYakppJlB7TfE5VrMgGP-nBVCzXm0JcV4NeHqmjB9Dbt~-D52akH052YyUn9NKOPfy5A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S1–S3</a></span>. </p><p>NCBI accession numbers of 45 public genomes used for genome annotation comparison are as follows: 35A (NC_017360), 51 (NC_017382), 52 (NC_017354), 83 (NC_017375), 908 (NC_017357), 2017 (NC_017374), 2018 (NC_017381), 26695 (NC_000915), Aklavik86 (NC_019563), Aklavik117 (NC_019560), B8 (NC_014256), B38 (NC_012973), BM012A (NC_022886), BM012S (NC_022911), Cuz20 (NC_017358), ELS37 (NC_017063), F16 (NC_017368), F30 (NC_017365), F32 (NC_017366), F57 (NC_017367), G27 (NC_011333), Gambia94/24 (NC_017371), HPAG1 (NC_008086), HUP-B14 (NC_017733), India7 (NC_017372), J99 (NC_000921), Lithuania75 (NC_017362), OK113 (NC_020508), OK310 (NC_020509), P12 (NC_011498), PeCan4 (NC_014555), PeCan18 (NC_017742), Puno120 (NC_017378), Puno135 (NC_017379), Sat464 (NC_017359), Shi112 (NC_017741), Shi169 (NC_017740), Shi417 (NC_017739), Shi470 (NC_010698), SJM180 (NC_014560), SNT49 (NC_017376), SouthAfrica7 (NC_017361), SouthAfrica20 (NC_022130), v225d (NC_017355) and XZ274 (NC_017926).</p>                            <h3 scrollto-destination=61463032 id="61463032" class="section-title" >2.5. <em>In silico</em> genome analysis</h3>
<p>The revised genome sequences of <em>H. pylori</em> UM032, 298 and 299 were compared pairwise using progressiveMauve (version 2.3.1), followed by variants calling using Geneious R7.<a reveal-id="dsw046-B25" data-open="dsw046-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a></p>                            <h3 scrollto-destination=61463034 id="61463034" class="section-title" >2.6. COG functional analysis</h3>
<p>COG of <em>H</em>. <em>pylori</em> UM032 genes with amino acid substitutions detected in the mice-adapted strains 298 and 299 was annotated by rpsblast.<a reveal-id="dsw046-B26" data-open="dsw046-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a></p>                            <h3 scrollto-destination=61463036 id="61463036" class="section-title" >2.7. Western immunodetection</h3>
<p>Whole cell lysates were prepared from bacterial cells standardised to OD<sub>600</sub> of 10. For each blot, 10 µl of sample was used for SDS-PAGE, followed by transfer to an Immobilon<sup>®</sup>-P polyvinylidene difluoride membrane (Merck Millipore). Blood group antigen-binding adhesin A (BabA) detected with polyclonal antibody, with detection by horseradish peroxidase (HRP)-conjugated goat anti-rabbit IgG antibodies (Santa Cruz Biotechnology) and visualization by chemiluminescent signal using Clarity™ Western ECL substrate (Bio-Rad) on a Fujifilm LAS-3000 imaging system. <em>H. pylori</em> UreB was detected using rabbit anti-urease B polyclonal antibodies purchased from Sigma Aldrich. <em>H. pylori</em> Le antigen phenotypes were determined using monoclonal antibodies (Abcam) to Le<sup>a</sup>, Le<sup>b</sup>, Le<sup>x</sup>, or Le<sup>y</sup>. Bound immunoglobulin M (IgM) or IgG antibodies were detected with HRP-conjugated goat anti-mouse antibodies (Sigma Aldrich) against IgM or IgG.</p>                            <h3 scrollto-destination=61463038 id="61463038" class="section-title" >2.8. Quantitative real-time polymerase chain reaction (qRT-PCR)</h3>
<p>Bacterial total RNA was isolated using NucleoSpin<sup>®</sup> RNA (Macherey-Nagel). After RNA quantification, cDNA was synthesized using 1 µg of RNA in a 20 µl reverse transcription reaction using QuantiTect Reverse Transcription Kit (Qiagen) according to manufacturer’s instructions. Real time PCR analysis was subsequently performed in 20 µl reactions in 96-well plates using cDNA from 20 ng RNA as template, LuminoCt<sup>®</sup> SYBR<sup>®</sup> Green qPCR ReadyMix<sup>TM</sup> (Sigma Aldrich) and the LightCycler 480 instrument (Roche Applied Science). Cycling conditions comprised an initial denaturation at 95 °C for 20 s, followed by 40 cycles of 95 °C for 3 s, 56 °C for 15 s and 60 °C for 30 s. Following each run a melting curve analysis was performed in which the reactions were heated slowly from 55 °C to 95 °C (0.1 °C/s). The relative expression of each target gene was calculated using the 2<sup>−</sup><sup>ΔΔCT</sup> method as described by Livak and Schmittgen<a reveal-id="dsw046-B27" data-open="dsw046-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a>, after normalisation against the <em>gyrA</em> reference gene. Primers used are listed in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw046/1/dsw046_Supp.zip?Expires=2147483647&Signature=s3q4bhSX2C3MUF-jr0f2UZ1jrVjkiGXzFiaUNbhztd4DChTCgvQajffmhInT-1Vj~0SRO~Kttfa4bqexSP8TrhsM2dD3tPD9oxAo17tPZ377FPaKsD8p~u48AhZfGosl2DgYDxY~wAxQIvmlENgGz~hpjXyN~I6CdeKNliwIOlCIEQtChC1zhSWLmnuwd57p2crX2HhvAlPdEQQh2bsMNnLu1COOQRPn604GZfBH8hQezXkfEFO3oCZfpwLaMYrQ~H4BcWpJOG4ygAaGy1rYakppJlB7TfE5VrMgGP-nBVCzXm0JcV4NeHqmjB9Dbt~-D52akH052YyUn9NKOPfy5A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>. </p>                            <h3 scrollto-destination=61463040 id="61463040" class="section-title" >2.9. Statistical analysis</h3>
<p>Paired Student’s <em>t-</em>test was used to test for differences in relative quantification of gene expression. <em>P</em>-values of less than 0.05 were considered statistically significant.</p>                            <h3 scrollto-destination=61463042 id="61463042" class="section-title" >2.10. Genome sequence update</h3>
<p>The updated complete genomes of UM032, 298 and 299 are available in GenBank under the original accession numbers: NC_021215.3 (GI:685455742), NC_021882.2 (GI:687961717) and NC_021216.3 (GI:685456130).</p>                            <h2 scrollto-destination=61463044 id="61463044" class="section-title" >3. Results and discussion</h2>
                            <h3 scrollto-destination=61463045 id="61463045" class="section-title" >3.1. Global pairwise alignment of previously published UM032, 298 and 299 genomes revealed sequence duplications and homopolymeric tract errors</h3>
                            <h4 scrollto-destination=61463046 id="61463046" class="section-title" >3.1.1. <em>Duplicate sequences</em></h4>
<div class="block-child-p">Genomic rearrangement via intragenic recombination is a powerful strategy of gene regulation employed by <em>H. pylori</em> to aid in adaption to host niches and successful colonisation of the gastric mucosa. To investigate the occurrence of chromosomal shuffling events in the mice-adapted UM032 derivatives, progressive Mauve alignment was performed after assigning a common starting point for all three genomes. No genomic rearrangement was observed. However, each genome was shown to harbour an insertion of a large DNA fragment, which measured 5659 bp, 9667 bp and 6578 bp in UM032, 298 and 299, respectively (<a reveal-id="dsw046-F1" data-open="dsw046-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>). BLASTN search demonstrated that each fragment could be further divided into two different regions that closely resemble the neighbouring sequences as depicted in <a reveal-id="dsw046-F2" data-open="dsw046-F2" class="link link-reveal link-table xref-fig">Figure 2</a>. These insertions are probably due to misassembly of the original PacBio sequences, resulting in false segmental duplications. <div data-id="dsw046-F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw046/1/m_dsw046f1.png?Expires=2147483647&amp;Signature=Cu4ay7Tynbe1MFtWpI-0I53NFC6aSbPp-yJOtfr-hc23e4aoOaGLTlEtjyWZmT9qGacd4mVwFMEO54sgNaA3sd~IcjtlNZB9wa5TntRHxaOn3iFcOfJifHMMh-~7~UIROltHBhbqMdoi~CdEstfeG3D7QZwxgRXj6oYzR~~WinNyLVnZA1YM~6HM8TxexCnwbv1OIw9bZ3fMlNfJEbf59tZ~WwEsvxmWB~jxLWTi-LIXjH83qZphMvZakV2BxXNTYirxYOKKrvrMzU8I17iE7piE3gkM6RCkP~SHF-Dpuyq4wnlMQ2vYwtMC6FvAdhoJ8wYLpGxM0GI-YW-zXeR-Yg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Whole-genome alignment of H. pylori strains UM032, 298 and 299 using Mauve indicates no genome shuffling. Insertions are visualised as gaps in the alignment." path-from-xml="dsw046f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61463047/dsw046f1.tif" path-from-xml="dsw046f1.tif" target="_blank">View large</a><a section="61463047" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw046/1/dsw046f1.png?Expires=2147483647&amp;Signature=wySufVrQEP9Rt4PQUwuW6bTsfx0rIu9zLtol6RwEl-RiKI7h1TJTj7ULi58SoYPQUwTyQfGmHABGBKzdYVdvAYs-r8i7Yh-GnBzuZCyAqplgVzx56tC0GmCazpKe-5TakCP3ty3gyphwghqX7QFc-DPNcu5cQKe6wwPbxbsUsV1cWhL7fSN2HkGFRE7yiaD5OOoZyafjhrYyJNkguYoGXo~b065gCcTuA94aj8YwPMryNZjpwJDUI-a4CuDKG2fMU726BGrnUL9gLCQP40qm6IIxVn9Abus27~Hl07d2qjF1HaOwn~Bn4XFHKXDiQ7s0Obqwc0ewzRrDiWz9UOL0jw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=61463047&amp;ar=2670112&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw046f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Whole-genome alignment of <em>H. pylori</em> strains UM032, 298 and 299 using Mauve indicates no genome shuffling. Insertions are visualised as gaps in the alignment.</p></div></div><div content-id="dsw046-F1" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw046/1/m_dsw046f1.png?Expires=2147483647&amp;Signature=Cu4ay7Tynbe1MFtWpI-0I53NFC6aSbPp-yJOtfr-hc23e4aoOaGLTlEtjyWZmT9qGacd4mVwFMEO54sgNaA3sd~IcjtlNZB9wa5TntRHxaOn3iFcOfJifHMMh-~7~UIROltHBhbqMdoi~CdEstfeG3D7QZwxgRXj6oYzR~~WinNyLVnZA1YM~6HM8TxexCnwbv1OIw9bZ3fMlNfJEbf59tZ~WwEsvxmWB~jxLWTi-LIXjH83qZphMvZakV2BxXNTYirxYOKKrvrMzU8I17iE7piE3gkM6RCkP~SHF-Dpuyq4wnlMQ2vYwtMC6FvAdhoJ8wYLpGxM0GI-YW-zXeR-Yg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Whole-genome alignment of H. pylori strains UM032, 298 and 299 using Mauve indicates no genome shuffling. Insertions are visualised as gaps in the alignment." path-from-xml="dsw046f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61463047/dsw046f1.tif" path-from-xml="dsw046f1.tif" target="_blank">View large</a><a section="61463047" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw046/1/dsw046f1.png?Expires=2147483647&amp;Signature=wySufVrQEP9Rt4PQUwuW6bTsfx0rIu9zLtol6RwEl-RiKI7h1TJTj7ULi58SoYPQUwTyQfGmHABGBKzdYVdvAYs-r8i7Yh-GnBzuZCyAqplgVzx56tC0GmCazpKe-5TakCP3ty3gyphwghqX7QFc-DPNcu5cQKe6wwPbxbsUsV1cWhL7fSN2HkGFRE7yiaD5OOoZyafjhrYyJNkguYoGXo~b065gCcTuA94aj8YwPMryNZjpwJDUI-a4CuDKG2fMU726BGrnUL9gLCQP40qm6IIxVn9Abus27~Hl07d2qjF1HaOwn~Bn4XFHKXDiQ7s0Obqwc0ewzRrDiWz9UOL0jw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=61463047&amp;ar=2670112&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw046f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Whole-genome alignment of <em>H. pylori</em> strains UM032, 298 and 299 using Mauve indicates no genome shuffling. Insertions are visualised as gaps in the alignment.</p></div></div><div data-id="dsw046-F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw046/1/m_dsw046f2.png?Expires=2147483647&amp;Signature=ADUHUz5obEG01em7Irses5EpiE0h7gydLGcC1o0boQM3q6IP6McIc3z5o~21qcsKAdfto7-4BkNg~uoCz8V5ePXZseYkSQA9fofd--c8P5tvxv0PXBBC-fCE0XWIHvD2QV18TynEEVL2NEavFhYAO-Kgs9K-qO0qlILe~GvYar1pEkadx0nnYf1w~G0YDCclrkGVy0GnIW6NG24VJHQLGpCfYTsONu-hd5rfujzLLiB3jryIl3Cb6nFMiM6oVVjs~V6B4LvxY7yAoNjlSCREPq4ZjdTfXYJi90RNXz5w0zM60wfD7rEuW4qk~LETFS19cHqxAJ0pXGqXB0nMEWdHbQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Duplicate sequences in UM032, 298 and 299 complete genomes. The ' character indicates the erroneous overlaps at each end of the original published contig." path-from-xml="dsw046f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61463047/dsw046f2.tif" path-from-xml="dsw046f2.tif" target="_blank">View large</a><a section="61463047" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw046/1/dsw046f2.png?Expires=2147483647&amp;Signature=3NyNECY-UObKvqlTe7yIJBo-6sa0AErrjWObcw2ZyMIO6Bu1JB90SXQN6F~96KNdgf-Ot2NSRgqhGc3dKnRTxVljZX3FlMhryBK51nzBwRa-Eb9HqNN4UnQZ3V1V4ok~5n0Ze4PXP7yBFL4CmhmCOV0yQ68rn9qehnGLTG99DGLAJFNzOON~Xk1H0fvwQz~0j9d9HdNMDTjfJhUTXSoQLk7KKcv-5B6H2mZlEnPuXGWAc3CJNvdVRLE3brnfV4BzMUCgFKz3NTjOsy1zZuvLC9ztPxH3ITfgnXTzh9O9Z~7u6jQ~129SHpajNJkmdBPU2rWUXn6qOQfrGi9JHM4wyw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=61463047&amp;ar=2670112&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw046f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Duplicate sequences in UM032, 298 and 299 complete genomes. The ' character indicates the erroneous overlaps at each end of the original published contig.</p></div></div><div content-id="dsw046-F2" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw046/1/m_dsw046f2.png?Expires=2147483647&amp;Signature=ADUHUz5obEG01em7Irses5EpiE0h7gydLGcC1o0boQM3q6IP6McIc3z5o~21qcsKAdfto7-4BkNg~uoCz8V5ePXZseYkSQA9fofd--c8P5tvxv0PXBBC-fCE0XWIHvD2QV18TynEEVL2NEavFhYAO-Kgs9K-qO0qlILe~GvYar1pEkadx0nnYf1w~G0YDCclrkGVy0GnIW6NG24VJHQLGpCfYTsONu-hd5rfujzLLiB3jryIl3Cb6nFMiM6oVVjs~V6B4LvxY7yAoNjlSCREPq4ZjdTfXYJi90RNXz5w0zM60wfD7rEuW4qk~LETFS19cHqxAJ0pXGqXB0nMEWdHbQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Duplicate sequences in UM032, 298 and 299 complete genomes. The ' character indicates the erroneous overlaps at each end of the original published contig." path-from-xml="dsw046f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61463047/dsw046f2.tif" path-from-xml="dsw046f2.tif" target="_blank">View large</a><a section="61463047" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw046/1/dsw046f2.png?Expires=2147483647&amp;Signature=3NyNECY-UObKvqlTe7yIJBo-6sa0AErrjWObcw2ZyMIO6Bu1JB90SXQN6F~96KNdgf-Ot2NSRgqhGc3dKnRTxVljZX3FlMhryBK51nzBwRa-Eb9HqNN4UnQZ3V1V4ok~5n0Ze4PXP7yBFL4CmhmCOV0yQ68rn9qehnGLTG99DGLAJFNzOON~Xk1H0fvwQz~0j9d9HdNMDTjfJhUTXSoQLk7KKcv-5B6H2mZlEnPuXGWAc3CJNvdVRLE3brnfV4BzMUCgFKz3NTjOsy1zZuvLC9ztPxH3ITfgnXTzh9O9Z~7u6jQ~129SHpajNJkmdBPU2rWUXn6qOQfrGi9JHM4wyw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=61463047&amp;ar=2670112&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw046f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Duplicate sequences in UM032, 298 and 299 complete genomes. The ' character indicates the erroneous overlaps at each end of the original published contig.</p></div></div></div>                            <h4 scrollto-destination=61463048 id="61463048" class="section-title" >3.1.2. <em>Homopolymeric tracts</em></h4>
<p>By using the NCBI Prokaryotic Genome Automatic Annotation Pipeline (PGAAP), a total of 167 pseudogenes have previously been annotated in UM032 genome compared to 29 in 298 and 25 in 299.<a reveal-id="dsw046-B17" data-open="dsw046-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> We performed BLASTN pairwise alignment of the pseudogenes in UM032 against their counterparts in 298 and 299, of which 146 were shown to have deletions associated with homopolymeric sequences in 298 and 299 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw046/1/dsw046_Supp.zip?Expires=2147483647&Signature=s3q4bhSX2C3MUF-jr0f2UZ1jrVjkiGXzFiaUNbhztd4DChTCgvQajffmhInT-1Vj~0SRO~Kttfa4bqexSP8TrhsM2dD3tPD9oxAo17tPZ377FPaKsD8p~u48AhZfGosl2DgYDxY~wAxQIvmlENgGz~hpjXyN~I6CdeKNliwIOlCIEQtChC1zhSWLmnuwd57p2crX2HhvAlPdEQQh2bsMNnLu1COOQRPn604GZfBH8hQezXkfEFO3oCZfpwLaMYrQ~H4BcWpJOG4ygAaGy1rYakppJlB7TfE5VrMgGP-nBVCzXm0JcV4NeHqmjB9Dbt~-D52akH052YyUn9NKOPfy5A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>), resulting in restoration of correct reading frame and thus the functional coding sequence (CDS). The presence of homopolymeric tracts in <em>H. pylori</em> is known to play an essential role in phase variation through slipped-strand mispairing, leading to differential antigenic expression that facilitates host-adaptation and persistent colonisation.<a reveal-id="dsw046-B13" data-open="dsw046-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> However, it is intriguing to observe such overwhelming mutational bias towards deletion in both 298 and 299 genomes, implying that there is potential sequencing and/or <em>de novo</em> assembly error in the Hierarchical Genome Assembly Process (HGAP) workflow (<a class="link link-uri" href="http://pacbiodevnet.com/" target="_blank">http://pacbiodevnet.com/</a>) available in Single-Molecule Real-Time (SMRT) Analysis v2.0. Sanger sequencing was performed to verify the length of the homopolymeric tract in which variation occurs in 7 randomly chosen UM032 pseudogenes and their functional counterparts. Homopolymer insertion errors detected in selected UM032 pseudogenes could be due to sequencing errors in the original genomes. </p>                            <h3 scrollto-destination=61463050 id="61463050" class="section-title" >3.2. Whole genome resequencing using MiSeq</h3>
<p>To remove homopolymer-length sequencing errors and duplications in the original sequences, all three genomes were resequenced using the MiSeq platform with at least 100× sequencing depth. The reads generated were subsequently mapped to the corresponding reference genome as described in Materials and Methods.</p>                            <h4 scrollto-destination=61463052 id="61463052" class="section-title" >3.2.1. <em>Below mean read depth across the duplicate sequences</em></h4>
<p>The analysis of read depth can be a useful approach to check for duplications. It is expected that the read depth at any location is equal to the global average coverage, given that the sequencing reads are randomly distributed across a genome. An average read depth of 135x, 122x and 174x, respectively, was determined for UM032, 298 and 299. The erroneously duplicated sequences, however, had substantially lower read depth than the global read depth and compared to that of the closely resembled upstream and downstream sequences, as summarised in <a reveal-id="dsw046-T1" data-open="dsw046-T1" class="link link-reveal link-table xref-fig">Table 1</a>, indicating they were misassembled and incorrectly present in two copies. The false duplication in each genome was therefore removed. This was followed by remapping of the sequencing reads to determine and compare the reads ratio (regional read depth divided by global read depth) of the previously duplicated sequences and their counterparts across all three genomes. These reads ratios are expected to be comparable among each other. As anticipated, the removal of duplication restored the reads ratio of these now single-copy sequences, as shown in <a reveal-id="dsw046-T2" data-open="dsw046-T2" class="link link-reveal link-table xref-fig">Table 2</a>. It is also now clear that what seemed to be the duplicated sequences are in fact near identical overlaps found at each end of the contig, which failed to be removed during sequence assembly to generate accurate circular consensus DNA sequence. <div content-id="dsw046-T1" class="table-modal"><div class="table-wrap"><div id="dsw046-T1" data-id="dsw046-T1" class="table-wrap-title"><div class="label -label">Table 1.</div><div class="caption"><p>Read depth of erroneous duplicate sequences </p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Global read depth </th><th colspan="3">Duplication sequence<hr /> </th><th colspan="3">Correct sequence<hr /> </th></tr><tr><th>Fragment<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn">*</a> </th><th>Size (bp) </th><th>Read depth </th><th>Fragment<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn">*</a> </th><th>Size (bp) </th><th>Read depth<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn">*</a> </th></tr></thead><tbody><tr><td>UM032 </td><td char="."> </td><td char="."> </td><td char="."> </td><td char="."> </td><td char="."> </td><td char="."> </td></tr><tr><td>135x </td><td char=".">A’ </td><td char=".">4,512 </td><td char=".">60  ± 38.4 </td><td char=".">A </td><td char=".">4,500 </td><td char=".">103.5  ± 41.9 </td></tr><tr><td> </td><td char=".">B’ </td><td char=".">1,147 </td><td char=".">7.7  ± 12.3 </td><td char=".">B </td><td char=".">1,139 </td><td char=".">178.5  ± 58.6 </td></tr><tr><td>298 </td><td char="."> </td><td char="."> </td><td char="."> </td><td char="."> </td><td char="."> </td><td char="."> </td></tr><tr><td>122x </td><td char=".">C’ </td><td char=".">6,774 </td><td char=".">28.9  ± 29.1 </td><td char=".">C </td><td char=".">6,744 </td><td char=".">73.3  ± 35.8 </td></tr><tr><td> </td><td char=".">D’ </td><td char=".">2,893 </td><td char=".">16.1  ± 17.8 </td><td char=".">D </td><td char=".">2,879 </td><td char=".">120.6  ± 45.6 </td></tr><tr><td>299 </td><td char="."> </td><td char="."> </td><td char="."> </td><td char="."> </td><td char="."> </td><td char="."> </td></tr><tr><td>174x </td><td char=".">E’ </td><td char=".">5,200 </td><td char=".">74.3  ± 28.4 </td><td char=".">E </td><td char=".">5,200 </td><td char=".">82.9  ± 35.5 </td></tr><tr><td> </td><td char=".">F’ </td><td char=".">1,378 </td><td char=".">32.7  ± 22.6 </td><td char=".">F </td><td char=".">1,378 </td><td char=".">81.1  ± 22.7 </td></tr></tbody></table></div><div content-id="dsw046-T1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Global read depth </th><th colspan="3">Duplication sequence<hr /> </th><th colspan="3">Correct sequence<hr /> </th></tr><tr><th>Fragment<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn">*</a> </th><th>Size (bp) </th><th>Read depth </th><th>Fragment<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn">*</a> </th><th>Size (bp) </th><th>Read depth<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn">*</a> </th></tr></thead><tbody><tr><td>UM032 </td><td char="."> </td><td char="."> </td><td char="."> </td><td char="."> </td><td char="."> </td><td char="."> </td></tr><tr><td>135x </td><td char=".">A’ </td><td char=".">4,512 </td><td char=".">60  ± 38.4 </td><td char=".">A </td><td char=".">4,500 </td><td char=".">103.5  ± 41.9 </td></tr><tr><td> </td><td char=".">B’ </td><td char=".">1,147 </td><td char=".">7.7  ± 12.3 </td><td char=".">B </td><td char=".">1,139 </td><td char=".">178.5  ± 58.6 </td></tr><tr><td>298 </td><td char="."> </td><td char="."> </td><td char="."> </td><td char="."> </td><td char="."> </td><td char="."> </td></tr><tr><td>122x </td><td char=".">C’ </td><td char=".">6,774 </td><td char=".">28.9  ± 29.1 </td><td char=".">C </td><td char=".">6,744 </td><td char=".">73.3  ± 35.8 </td></tr><tr><td> </td><td char=".">D’ </td><td char=".">2,893 </td><td char=".">16.1  ± 17.8 </td><td char=".">D </td><td char=".">2,879 </td><td char=".">120.6  ± 45.6 </td></tr><tr><td>299 </td><td char="."> </td><td char="."> </td><td char="."> </td><td char="."> </td><td char="."> </td><td char="."> </td></tr><tr><td>174x </td><td char=".">E’ </td><td char=".">5,200 </td><td char=".">74.3  ± 28.4 </td><td char=".">E </td><td char=".">5,200 </td><td char=".">82.9  ± 35.5 </td></tr><tr><td> </td><td char=".">F’ </td><td char=".">1,378 </td><td char=".">32.7  ± 22.6 </td><td char=".">F </td><td char=".">1,378 </td><td char=".">81.1  ± 22.7 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><div class="label -label">*</div><p>Please refer to <a reveal-id="dsw046-F2" data-open="dsw046-F2" class="link link-reveal link-table xref-fig">Figure 2</a> for the position of each sequence fragment described in this table.</p></div></div></div></div></div><div class="table-wrap"><div id="dsw046-T1" data-id="dsw046-T1" class="table-wrap-title"><div class="label -label">Table 1.</div><div class="caption"><p>Read depth of erroneous duplicate sequences </p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Global read depth </th><th colspan="3">Duplication sequence<hr /> </th><th colspan="3">Correct sequence<hr /> </th></tr><tr><th>Fragment<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn">*</a> </th><th>Size (bp) </th><th>Read depth </th><th>Fragment<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn">*</a> </th><th>Size (bp) </th><th>Read depth<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn">*</a> </th></tr></thead><tbody><tr><td>UM032 </td><td char="."> </td><td char="."> </td><td char="."> </td><td char="."> </td><td char="."> </td><td char="."> </td></tr><tr><td>135x </td><td char=".">A’ </td><td char=".">4,512 </td><td char=".">60  ± 38.4 </td><td char=".">A </td><td char=".">4,500 </td><td char=".">103.5  ± 41.9 </td></tr><tr><td> </td><td char=".">B’ </td><td char=".">1,147 </td><td char=".">7.7  ± 12.3 </td><td char=".">B </td><td char=".">1,139 </td><td char=".">178.5  ± 58.6 </td></tr><tr><td>298 </td><td char="."> </td><td char="."> </td><td char="."> </td><td char="."> </td><td char="."> </td><td char="."> </td></tr><tr><td>122x </td><td char=".">C’ </td><td char=".">6,774 </td><td char=".">28.9  ± 29.1 </td><td char=".">C </td><td char=".">6,744 </td><td char=".">73.3  ± 35.8 </td></tr><tr><td> </td><td char=".">D’ </td><td char=".">2,893 </td><td char=".">16.1  ± 17.8 </td><td char=".">D </td><td char=".">2,879 </td><td char=".">120.6  ± 45.6 </td></tr><tr><td>299 </td><td char="."> </td><td char="."> </td><td char="."> </td><td char="."> </td><td char="."> </td><td char="."> </td></tr><tr><td>174x </td><td char=".">E’ </td><td char=".">5,200 </td><td char=".">74.3  ± 28.4 </td><td char=".">E </td><td char=".">5,200 </td><td char=".">82.9  ± 35.5 </td></tr><tr><td> </td><td char=".">F’ </td><td char=".">1,378 </td><td char=".">32.7  ± 22.6 </td><td char=".">F </td><td char=".">1,378 </td><td char=".">81.1  ± 22.7 </td></tr></tbody></table></div><div content-id="dsw046-T1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Global read depth </th><th colspan="3">Duplication sequence<hr /> </th><th colspan="3">Correct sequence<hr /> </th></tr><tr><th>Fragment<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn">*</a> </th><th>Size (bp) </th><th>Read depth </th><th>Fragment<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn">*</a> </th><th>Size (bp) </th><th>Read depth<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn">*</a> </th></tr></thead><tbody><tr><td>UM032 </td><td char="."> </td><td char="."> </td><td char="."> </td><td char="."> </td><td char="."> </td><td char="."> </td></tr><tr><td>135x </td><td char=".">A’ </td><td char=".">4,512 </td><td char=".">60  ± 38.4 </td><td char=".">A </td><td char=".">4,500 </td><td char=".">103.5  ± 41.9 </td></tr><tr><td> </td><td char=".">B’ </td><td char=".">1,147 </td><td char=".">7.7  ± 12.3 </td><td char=".">B </td><td char=".">1,139 </td><td char=".">178.5  ± 58.6 </td></tr><tr><td>298 </td><td char="."> </td><td char="."> </td><td char="."> </td><td char="."> </td><td char="."> </td><td char="."> </td></tr><tr><td>122x </td><td char=".">C’ </td><td char=".">6,774 </td><td char=".">28.9  ± 29.1 </td><td char=".">C </td><td char=".">6,744 </td><td char=".">73.3  ± 35.8 </td></tr><tr><td> </td><td char=".">D’ </td><td char=".">2,893 </td><td char=".">16.1  ± 17.8 </td><td char=".">D </td><td char=".">2,879 </td><td char=".">120.6  ± 45.6 </td></tr><tr><td>299 </td><td char="."> </td><td char="."> </td><td char="."> </td><td char="."> </td><td char="."> </td><td char="."> </td></tr><tr><td>174x </td><td char=".">E’ </td><td char=".">5,200 </td><td char=".">74.3  ± 28.4 </td><td char=".">E </td><td char=".">5,200 </td><td char=".">82.9  ± 35.5 </td></tr><tr><td> </td><td char=".">F’ </td><td char=".">1,378 </td><td char=".">32.7  ± 22.6 </td><td char=".">F </td><td char=".">1,378 </td><td char=".">81.1  ± 22.7 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><div class="label -label">*</div><p>Please refer to <a reveal-id="dsw046-F2" data-open="dsw046-F2" class="link link-reveal link-table xref-fig">Figure 2</a> for the position of each sequence fragment described in this table.</p></div></div></div></div><div content-id="dsw046-T2" class="table-modal"><div class="table-wrap"><div id="dsw046-T2" data-id="dsw046-T2" class="table-wrap-title"><div class="label -label">Table 2.</div><div class="caption"><p>Read depth analysis of correct sequences against counterpart sequences following removal of duplications</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Strain (global read depth) </th><th colspan="3">Read depth | read ratio (read depth/global read depth)<hr /> </th></tr><tr><th>AB fragments </th><th>CD fragments </th><th>EF fragments </th></tr></thead><tbody><tr><td>UM032 (135x) </td><td>168x | 1.24 </td><td>121x | 0.90 </td><td>113x | 0.84 </td></tr><tr><td>298 (122x) </td><td>146x | 1.20 </td><td>112x | 0.92 </td><td>102x | 0.84 </td></tr><tr><td>299 (174x) </td><td>217x | 1.25 </td><td>157x | 0.90 </td><td>148x | 0.85 </td></tr></tbody></table></div><div content-id="dsw046-T2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Strain (global read depth) </th><th colspan="3">Read depth | read ratio (read depth/global read depth)<hr /> </th></tr><tr><th>AB fragments </th><th>CD fragments </th><th>EF fragments </th></tr></thead><tbody><tr><td>UM032 (135x) </td><td>168x | 1.24 </td><td>121x | 0.90 </td><td>113x | 0.84 </td></tr><tr><td>298 (122x) </td><td>146x | 1.20 </td><td>112x | 0.92 </td><td>102x | 0.84 </td></tr><tr><td>299 (174x) </td><td>217x | 1.25 </td><td>157x | 0.90 </td><td>148x | 0.85 </td></tr></tbody></table></div></div></div><div class="table-wrap"><div id="dsw046-T2" data-id="dsw046-T2" class="table-wrap-title"><div class="label -label">Table 2.</div><div class="caption"><p>Read depth analysis of correct sequences against counterpart sequences following removal of duplications</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Strain (global read depth) </th><th colspan="3">Read depth | read ratio (read depth/global read depth)<hr /> </th></tr><tr><th>AB fragments </th><th>CD fragments </th><th>EF fragments </th></tr></thead><tbody><tr><td>UM032 (135x) </td><td>168x | 1.24 </td><td>121x | 0.90 </td><td>113x | 0.84 </td></tr><tr><td>298 (122x) </td><td>146x | 1.20 </td><td>112x | 0.92 </td><td>102x | 0.84 </td></tr><tr><td>299 (174x) </td><td>217x | 1.25 </td><td>157x | 0.90 </td><td>148x | 0.85 </td></tr></tbody></table></div><div content-id="dsw046-T2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Strain (global read depth) </th><th colspan="3">Read depth | read ratio (read depth/global read depth)<hr /> </th></tr><tr><th>AB fragments </th><th>CD fragments </th><th>EF fragments </th></tr></thead><tbody><tr><td>UM032 (135x) </td><td>168x | 1.24 </td><td>121x | 0.90 </td><td>113x | 0.84 </td></tr><tr><td>298 (122x) </td><td>146x | 1.20 </td><td>112x | 0.92 </td><td>102x | 0.84 </td></tr><tr><td>299 (174x) </td><td>217x | 1.25 </td><td>157x | 0.90 </td><td>148x | 0.85 </td></tr></tbody></table></div></div></p>                            <h4 scrollto-destination=61463054 id="61463054" class="section-title" >3.2.2. <em>Variants calling in duplication-free reference genomes</em></h4>
<p>As indicated in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw046/1/dsw046_Supp.zip?Expires=2147483647&Signature=s3q4bhSX2C3MUF-jr0f2UZ1jrVjkiGXzFiaUNbhztd4DChTCgvQajffmhInT-1Vj~0SRO~Kttfa4bqexSP8TrhsM2dD3tPD9oxAo17tPZ377FPaKsD8p~u48AhZfGosl2DgYDxY~wAxQIvmlENgGz~hpjXyN~I6CdeKNliwIOlCIEQtChC1zhSWLmnuwd57p2crX2HhvAlPdEQQh2bsMNnLu1COOQRPn604GZfBH8hQezXkfEFO3oCZfpwLaMYrQ~H4BcWpJOG4ygAaGy1rYakppJlB7TfE5VrMgGP-nBVCzXm0JcV4NeHqmjB9Dbt~-D52akH052YyUn9NKOPfy5A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6</a></span>, 245 variants were detected in UM032, of which 243 were deletions found within homopolymeric runs. Surprisingly, there were significantly less errors in both 298 and 299 genomes, only seven and three variants being detected, respectively. Nonetheless, the reported errors, including all additional nucleotides found in the homopolymeric tracts, were corrected and the genomes were reannotated as stated in Methods. <a reveal-id="dsw046-T3" data-open="dsw046-T3" class="link link-reveal link-table xref-fig">Table 3</a> summarises the general features of all three genomes, prior to and after the sequence error correction and reannotation. It is important to highlight that a number of false pseudogene candidates have been efficiently removed, especially in UM032, in which the number has been significantly reduced from 167 to 42, improving the reliability of comparative analysis. <div content-id="dsw046-T3" class="table-modal"><div class="table-wrap"><div id="dsw046-T3" data-id="dsw046-T3" class="table-wrap-title"><div class="label -label">Table 3.</div><div class="caption"><p>General features of pre- and post-correction <em>H. pylori</em> UM032, 298 and 299 genomes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th colspan="6"><em>H. pylori</em><hr /> </th></tr><tr><th> </th><th colspan="2">UM032<hr /> </th><th colspan="2">298<hr /> </th><th colspan="2">299<hr /> </th></tr><tr><th>Features </th><th>Pre </th><th>Post </th><th>Pre </th><th>Post </th><th>Pre </th><th>Post </th></tr></thead><tbody><tr><td>Size (bp) </td><td>1,599,441 </td><td>1,593,537 </td><td>1,604,216 </td><td>1,594,544 </td><td>1,601,149 </td><td>1,594,569 </td></tr><tr><td>GC content (%) </td><td>38.8 </td><td>38.8 </td><td>38.8 </td><td>38.8 </td><td>38.8 </td><td>38.8 </td></tr><tr><td>CDSs </td><td>1,415 </td><td>1,458 </td><td>1,553 </td><td>1,456 </td><td>1,576 </td><td>1,457 </td></tr><tr><td>Genes </td><td>1,624 </td><td>1,543 </td><td>1,624 </td><td>1,544 </td><td>1,644 </td><td>1,544 </td></tr><tr><td>Pseudogenes </td><td>167 </td><td>43 </td><td>29 </td><td>46 </td><td>25 </td><td>45 </td></tr><tr><td>rRNAs </td><td>6 </td><td>6 </td><td>6 </td><td>6 </td><td>6 </td><td>6 </td></tr><tr><td>tRNAs </td><td>36 </td><td>36 </td><td>36 </td><td>36 </td><td>37 </td><td>36 </td></tr></tbody></table></div><div content-id="dsw046-T3" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th colspan="6"><em>H. pylori</em><hr /> </th></tr><tr><th> </th><th colspan="2">UM032<hr /> </th><th colspan="2">298<hr /> </th><th colspan="2">299<hr /> </th></tr><tr><th>Features </th><th>Pre </th><th>Post </th><th>Pre </th><th>Post </th><th>Pre </th><th>Post </th></tr></thead><tbody><tr><td>Size (bp) </td><td>1,599,441 </td><td>1,593,537 </td><td>1,604,216 </td><td>1,594,544 </td><td>1,601,149 </td><td>1,594,569 </td></tr><tr><td>GC content (%) </td><td>38.8 </td><td>38.8 </td><td>38.8 </td><td>38.8 </td><td>38.8 </td><td>38.8 </td></tr><tr><td>CDSs </td><td>1,415 </td><td>1,458 </td><td>1,553 </td><td>1,456 </td><td>1,576 </td><td>1,457 </td></tr><tr><td>Genes </td><td>1,624 </td><td>1,543 </td><td>1,624 </td><td>1,544 </td><td>1,644 </td><td>1,544 </td></tr><tr><td>Pseudogenes </td><td>167 </td><td>43 </td><td>29 </td><td>46 </td><td>25 </td><td>45 </td></tr><tr><td>rRNAs </td><td>6 </td><td>6 </td><td>6 </td><td>6 </td><td>6 </td><td>6 </td></tr><tr><td>tRNAs </td><td>36 </td><td>36 </td><td>36 </td><td>36 </td><td>37 </td><td>36 </td></tr></tbody></table></div></div></div><div class="table-wrap"><div id="dsw046-T3" data-id="dsw046-T3" class="table-wrap-title"><div class="label -label">Table 3.</div><div class="caption"><p>General features of pre- and post-correction <em>H. pylori</em> UM032, 298 and 299 genomes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th colspan="6"><em>H. pylori</em><hr /> </th></tr><tr><th> </th><th colspan="2">UM032<hr /> </th><th colspan="2">298<hr /> </th><th colspan="2">299<hr /> </th></tr><tr><th>Features </th><th>Pre </th><th>Post </th><th>Pre </th><th>Post </th><th>Pre </th><th>Post </th></tr></thead><tbody><tr><td>Size (bp) </td><td>1,599,441 </td><td>1,593,537 </td><td>1,604,216 </td><td>1,594,544 </td><td>1,601,149 </td><td>1,594,569 </td></tr><tr><td>GC content (%) </td><td>38.8 </td><td>38.8 </td><td>38.8 </td><td>38.8 </td><td>38.8 </td><td>38.8 </td></tr><tr><td>CDSs </td><td>1,415 </td><td>1,458 </td><td>1,553 </td><td>1,456 </td><td>1,576 </td><td>1,457 </td></tr><tr><td>Genes </td><td>1,624 </td><td>1,543 </td><td>1,624 </td><td>1,544 </td><td>1,644 </td><td>1,544 </td></tr><tr><td>Pseudogenes </td><td>167 </td><td>43 </td><td>29 </td><td>46 </td><td>25 </td><td>45 </td></tr><tr><td>rRNAs </td><td>6 </td><td>6 </td><td>6 </td><td>6 </td><td>6 </td><td>6 </td></tr><tr><td>tRNAs </td><td>36 </td><td>36 </td><td>36 </td><td>36 </td><td>37 </td><td>36 </td></tr></tbody></table></div><div content-id="dsw046-T3" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th colspan="6"><em>H. pylori</em><hr /> </th></tr><tr><th> </th><th colspan="2">UM032<hr /> </th><th colspan="2">298<hr /> </th><th colspan="2">299<hr /> </th></tr><tr><th>Features </th><th>Pre </th><th>Post </th><th>Pre </th><th>Post </th><th>Pre </th><th>Post </th></tr></thead><tbody><tr><td>Size (bp) </td><td>1,599,441 </td><td>1,593,537 </td><td>1,604,216 </td><td>1,594,544 </td><td>1,601,149 </td><td>1,594,569 </td></tr><tr><td>GC content (%) </td><td>38.8 </td><td>38.8 </td><td>38.8 </td><td>38.8 </td><td>38.8 </td><td>38.8 </td></tr><tr><td>CDSs </td><td>1,415 </td><td>1,458 </td><td>1,553 </td><td>1,456 </td><td>1,576 </td><td>1,457 </td></tr><tr><td>Genes </td><td>1,624 </td><td>1,543 </td><td>1,624 </td><td>1,544 </td><td>1,644 </td><td>1,544 </td></tr><tr><td>Pseudogenes </td><td>167 </td><td>43 </td><td>29 </td><td>46 </td><td>25 </td><td>45 </td></tr><tr><td>rRNAs </td><td>6 </td><td>6 </td><td>6 </td><td>6 </td><td>6 </td><td>6 </td></tr><tr><td>tRNAs </td><td>36 </td><td>36 </td><td>36 </td><td>36 </td><td>37 </td><td>36 </td></tr></tbody></table></div></div></p>                            <h3 scrollto-destination=61463056 id="61463056" class="section-title" >3.3. Whole genome comparisons of revised <em>H. pylori</em> UM032, 298 and 299</h3>
<p>The revised complete genome sequences of <em>H. pylori</em> clinical isolate UM032 and its mice-adapted derivatives 298 and 299 were compared against each other. A pairwise global alignment of the genome of UM032 with that of 298 and 299 revealed 99.9% identity at the nucleotide level. The 298 strain contained 89 variants relative to strain UM032, of which 32 were indels and the remaining were substitution mutations including single-nucleotide polymorphisms (SNPs) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw046/1/dsw046_Supp.zip?Expires=2147483647&Signature=s3q4bhSX2C3MUF-jr0f2UZ1jrVjkiGXzFiaUNbhztd4DChTCgvQajffmhInT-1Vj~0SRO~Kttfa4bqexSP8TrhsM2dD3tPD9oxAo17tPZ377FPaKsD8p~u48AhZfGosl2DgYDxY~wAxQIvmlENgGz~hpjXyN~I6CdeKNliwIOlCIEQtChC1zhSWLmnuwd57p2crX2HhvAlPdEQQh2bsMNnLu1COOQRPn604GZfBH8hQezXkfEFO3oCZfpwLaMYrQ~H4BcWpJOG4ygAaGy1rYakppJlB7TfE5VrMgGP-nBVCzXm0JcV4NeHqmjB9Dbt~-D52akH052YyUn9NKOPfy5A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S7</a></span>). Nine indels affected ORFs. In addition to tandem repeats or homopolymeric tracts and additions or deletions to the 5’ or 3’ ends of ORFs, an insertion has introduced a complete ORF, which is annotated as <em>UM298_1363</em> encoding a lipopolysaccharide (LPS) biosynthesis protein. Amongst the 57 substitutions detected, 16 were intergenic and 41 were located in protein coding sequences. Only 12 of the latter resulted in amino acid changes and the remaining were synonymous substitutions. Relative to 298, only two tandem repeat insertions and 1 deletion within a homopolymeric tract were identified in 299, of which the latter restores an open reading frame designated <em>UM299_0755</em> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw046/1/dsw046_Supp.zip?Expires=2147483647&Signature=s3q4bhSX2C3MUF-jr0f2UZ1jrVjkiGXzFiaUNbhztd4DChTCgvQajffmhInT-1Vj~0SRO~Kttfa4bqexSP8TrhsM2dD3tPD9oxAo17tPZ377FPaKsD8p~u48AhZfGosl2DgYDxY~wAxQIvmlENgGz~hpjXyN~I6CdeKNliwIOlCIEQtChC1zhSWLmnuwd57p2crX2HhvAlPdEQQh2bsMNnLu1COOQRPn604GZfBH8hQezXkfEFO3oCZfpwLaMYrQ~H4BcWpJOG4ygAaGy1rYakppJlB7TfE5VrMgGP-nBVCzXm0JcV4NeHqmjB9Dbt~-D52akH052YyUn9NKOPfy5A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S8</a></span>). We further performed a COG (clusters of orthologous groups) analysis on genes with non-synonymous changes (<a reveal-id="dsw046-T4" data-open="dsw046-T4" class="link link-reveal link-table xref-fig">Table 4</a>). Genes without an inferred COG category were also analysed. Together it was shown that genes encoding for outer membrane and lipopolysaccharide biosynthesis proteins are enriched for substitution mutations. <div content-id="dsw046-T4" class="table-modal"><div class="table-wrap"><div id="dsw046-T4" data-id="dsw046-T4" class="table-wrap-title"><div class="label -label">Table 4.</div><div class="caption"><p>COG analysis of UM032 genes with non-synonymous substitutions in its mice-adapted derivatives</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>COG </th><th>Function </th><th>Number </th><th>Locus tag </th></tr></thead><tbody><tr><td>D </td><td>Cell cycle control, cell division, chromosome partitioning </td><td>1 </td><td>UM032_0202 </td></tr><tr><td>E </td><td>Amino acid transport and metabolism </td><td>1 </td><td>UM032_0124 </td></tr><tr><td>H </td><td>Coenzyme transport and metabolism </td><td>1 </td><td>UM032_1143 </td></tr><tr><td>L </td><td>Replication, recombination and repair </td><td>1 </td><td>UM032_1365 </td></tr><tr><td>M </td><td>Cell wall/membrane/envelope biogenesis </td><td>2 </td><td>UM032_0124, UM032_1363 </td></tr><tr><td>N </td><td>Cell motility </td><td>1 </td><td>UM032_0447 </td></tr><tr><td>T </td><td>Signal transduction mechanisms </td><td>1 </td><td>UM032_0447 </td></tr></tbody></table></div><div content-id="dsw046-T4" class="table-modal"><table><thead name="thead" align=""><tr><th>COG </th><th>Function </th><th>Number </th><th>Locus tag </th></tr></thead><tbody><tr><td>D </td><td>Cell cycle control, cell division, chromosome partitioning </td><td>1 </td><td>UM032_0202 </td></tr><tr><td>E </td><td>Amino acid transport and metabolism </td><td>1 </td><td>UM032_0124 </td></tr><tr><td>H </td><td>Coenzyme transport and metabolism </td><td>1 </td><td>UM032_1143 </td></tr><tr><td>L </td><td>Replication, recombination and repair </td><td>1 </td><td>UM032_1365 </td></tr><tr><td>M </td><td>Cell wall/membrane/envelope biogenesis </td><td>2 </td><td>UM032_0124, UM032_1363 </td></tr><tr><td>N </td><td>Cell motility </td><td>1 </td><td>UM032_0447 </td></tr><tr><td>T </td><td>Signal transduction mechanisms </td><td>1 </td><td>UM032_0447 </td></tr></tbody></table></div></div></div><div class="table-wrap"><div id="dsw046-T4" data-id="dsw046-T4" class="table-wrap-title"><div class="label -label">Table 4.</div><div class="caption"><p>COG analysis of UM032 genes with non-synonymous substitutions in its mice-adapted derivatives</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>COG </th><th>Function </th><th>Number </th><th>Locus tag </th></tr></thead><tbody><tr><td>D </td><td>Cell cycle control, cell division, chromosome partitioning </td><td>1 </td><td>UM032_0202 </td></tr><tr><td>E </td><td>Amino acid transport and metabolism </td><td>1 </td><td>UM032_0124 </td></tr><tr><td>H </td><td>Coenzyme transport and metabolism </td><td>1 </td><td>UM032_1143 </td></tr><tr><td>L </td><td>Replication, recombination and repair </td><td>1 </td><td>UM032_1365 </td></tr><tr><td>M </td><td>Cell wall/membrane/envelope biogenesis </td><td>2 </td><td>UM032_0124, UM032_1363 </td></tr><tr><td>N </td><td>Cell motility </td><td>1 </td><td>UM032_0447 </td></tr><tr><td>T </td><td>Signal transduction mechanisms </td><td>1 </td><td>UM032_0447 </td></tr></tbody></table></div><div content-id="dsw046-T4" class="table-modal"><table><thead name="thead" align=""><tr><th>COG </th><th>Function </th><th>Number </th><th>Locus tag </th></tr></thead><tbody><tr><td>D </td><td>Cell cycle control, cell division, chromosome partitioning </td><td>1 </td><td>UM032_0202 </td></tr><tr><td>E </td><td>Amino acid transport and metabolism </td><td>1 </td><td>UM032_0124 </td></tr><tr><td>H </td><td>Coenzyme transport and metabolism </td><td>1 </td><td>UM032_1143 </td></tr><tr><td>L </td><td>Replication, recombination and repair </td><td>1 </td><td>UM032_1365 </td></tr><tr><td>M </td><td>Cell wall/membrane/envelope biogenesis </td><td>2 </td><td>UM032_0124, UM032_1363 </td></tr><tr><td>N </td><td>Cell motility </td><td>1 </td><td>UM032_0447 </td></tr><tr><td>T </td><td>Signal transduction mechanisms </td><td>1 </td><td>UM032_0447 </td></tr></tbody></table></div></div></p><p>Taken together, our data have demonstrated an outburst of mutations when UM032 encounters a new host environment. The second round of mice infection using the mice-adapted strain 298 however resulted only in slight genomic alterations in the output strain 299, signifying that the bacterium has undergone sufficient genomic adaptations to establish a chronic infection. It is important to mention that our findings contrast the experimental outcome of Linz <em>et al.</em> who conducted a macaque infection study using the previously macaque-adapted <em>H. pylori</em> strain J166.<a reveal-id="dsw046-B28" data-open="dsw046-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> In their study, J166<sub>output</sub> strains accumulated up to 12 SNPs within 1 week post infection. Our output strain 299, however, showed no SNPs 2 weeks after the infection. This suggests that there was a lack of acute inflammatory response, perhaps due to less variability between the gastric environments of mice, than between those of individual macaques.</p><p>Of note is that an amino acid change was detected in UM298_1343 relative to UM032_1343. This protein is known as the antigenic membrane-associated tumour necrosis factor α-inducing protein. The change replaces the isoleucine at position 111 for valine. Previous studies have shown that <em>H. pylori</em> TNFα-inducing protein is able to induce interleukin-1α, TNFα, IL-8, and macrophage inflammatory protein 1α productions in monocytes, as well as tumorigenesis in nude mice implanted with transfected Bhas 42 cells.<a reveal-id="dsw046-B29" data-open="dsw046-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a><sup>,</sup><a reveal-id="dsw046-B30" data-open="dsw046-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a></p>                            <h3 scrollto-destination=61463060 id="61463060" class="section-title" >3.4. Intragenomic recombination between <em>babA</em> and <em>babB</em></h3>
<p>Among 65 nucleotide substitutions detected in 298 and 299 relative to UM032, 24 were observed within ORF <em>UM032_1223</em>, which was identified as the <em>babA</em> gene by BLASTN search against the <em>Helicobacter</em> group in NCBI database. Such a high number of mutations could reflect the strong selective pressure for rapid adjustment of BabA-mediated adhesion to facilitate adaptation and colonisation of different host gastric niche. However, these mutations were all synonymous substitutions. Furthermore, rather than being arbitrarily distributed across the 2226 bp gene, they were confined within the C-terminal region, prompting us to investigate whether the mutations could arise due to a recombination event.</p><p>Paralogous to <em>babA</em> are both <em>babB</em> and <em>babC</em> genes. Due to their extensive sequence identity at 5’ and 3’ regions, intragenomic recombination can occur within the <em>babABC</em> family, resulting in phenotypic change of Lewis b antigen binding capacity.<a reveal-id="dsw046-B31 dsw046-B32 dsw046-B33 dsw046-B34" data-open="dsw046-B31 dsw046-B32 dsw046-B33 dsw046-B34" class="link link-ref link-reveal xref-bibr"><sup>31–34</sup></a> The <em>bab</em> genes reside in three different chromosomal locations, downstream of the <em>hypD</em> gene (locus A), the <em>rpsR</em> gene (locus B) and <em>hp0318</em> in <em>H. pylori</em> 26695 (locus C), respectively.<a reveal-id="dsw046-B31" data-open="dsw046-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a><sup>,</sup><a reveal-id="dsw046-B35" data-open="dsw046-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> Nevertheless not all <em>bab</em> genes are present within each <em>H. pylori</em> strain. There are some strains which do not possess the <em>babA</em> or <em>babC</em> gene and some harbour two copies of <em>babA</em> or <em>babB</em> gene.<a reveal-id="dsw046-B36" data-open="dsw046-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a><sup>,</sup><a reveal-id="dsw046-B37" data-open="dsw046-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> There has also been an interesting observation by Kawai <em>et al.</em> that the <em>babC</em> locus, corresponding to <em>hp0317</em> in <em>H. pylori</em> 26695, is empty in all the hpEastAsia strains as a result of reductive evolution in the outer membrane protein families.<a reveal-id="dsw046-B38" data-open="dsw046-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a></p><div class="block-child-p">To test for a recombination event, we performed a BLASTN search of the C-terminal sequence fragment in UM032 genome and identified a complete match in the C-terminal region of a silent <em>babB</em> locus designated <em>UM032_0908</em>. Furthermore, pairwise nucleotide sequence alignment of UM032 <em>babA</em> and <em>babB</em>, and 298 <em>babA</em>, along with their downstream sequence indicated the presence of a double crossover recombination event as highlighted in <a reveal-id="dsw046-F3" data-open="dsw046-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>. As there is no conversion of any amino acid residue, no functional change is deduced. It is however unclear whether the synonymous mutations may affect the <em>in vivo</em> translational speed and thus the expression of BabA in strains 298 and 299. To evaluate the possible translational outcome of these synonymous changes in <em>babA</em>, we compared the codon usage of the <em>babA</em> genes to the codon usage of highly expressed genes, specifically the ribosomal genes and the elongation factors, in the measurement of codon adaptation index (CAI) computed using EMBOSS CAI application.<a reveal-id="dsw046-B39" data-open="dsw046-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> The UM032 <em>babA</em> gene achieved a CAI score of 0.693, compared to a slightly lower value of 0.689 in the recombined version found in 298 and 299. Under such condition, the latter is less optimally encoded, by 0.4%, suggesting that there would only be marginal effect on protein expression levels. <div data-id="dsw046-F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw046/1/m_dsw046f3.png?Expires=2147483647&Signature=nJjR7uNofB0PuQfKeL6e9TZFjmtGuaTHu-4mVXnXTmGAw~0DikFaDUT5WdfeLYUyTQWutdrydRe-r2-zX0RRTuV41VsyJZx3rymssVnY7RcIQ5IVodq-QI-60nrGa~TvdK1x4q2YFMkxXnBZquINVhFcX5ihCyKWtnAaWAUwF620M5VS5zN3KaZCqYMSEJDB5GO1rZRQl1tyTMKKCbGSPn3xxz0zG~1cuKUfH3zTCBmyB5Pn9LmjKwOyCS~H-aSEkIngcGQfRWRcLIl1kYB6dV-Fo4t6o530GzuIBDY2HDyfH4~pmRtkMkkb5iAFn8HgoKqMR2y~cc6mfT7jMP6yBA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Recombination between babA and babB sequences." path-from-xml="dsw046f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61463063/dsw046f3.tif" path-from-xml="dsw046f3.tif" target="_blank">View large</a><a section="61463063" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw046/1/dsw046f3.png?Expires=2147483647&Signature=M238TFb6kskhluA8njIhkHFFWl9BoExiTELQcTmlqhmC3urqUR2fMgtCi8WLBSQe3BP2mZiH0~5dGVNuB18YB1klDyiaNe4qyXinrN~Lglpq2ItghkAhkjsXHdGLa1sEoutKqGIxmSWtaycIAtVlWz1QO5sYBsbB10IRf4elv5d925gwZQ3aFIOk-CoS-3a67IN1pfmzDHSKGQXklD~2uUedXKYQeLYfeJhUL433hvG4c3eVSwf-OpV~~JMitWQpjw44xA69MNCnpHPo0FExK0r91ei8u9O2AaTPE1W3c973T1bsJS8H0uAcF~rABv1O~4bEER59dRqMwbUSp-GXUQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=61463063&ar=2670112&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw046f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Recombination between <em>babA</em> and <em>babB</em> sequences.</p></div></div><div content-id="dsw046-F3" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw046/1/m_dsw046f3.png?Expires=2147483647&Signature=nJjR7uNofB0PuQfKeL6e9TZFjmtGuaTHu-4mVXnXTmGAw~0DikFaDUT5WdfeLYUyTQWutdrydRe-r2-zX0RRTuV41VsyJZx3rymssVnY7RcIQ5IVodq-QI-60nrGa~TvdK1x4q2YFMkxXnBZquINVhFcX5ihCyKWtnAaWAUwF620M5VS5zN3KaZCqYMSEJDB5GO1rZRQl1tyTMKKCbGSPn3xxz0zG~1cuKUfH3zTCBmyB5Pn9LmjKwOyCS~H-aSEkIngcGQfRWRcLIl1kYB6dV-Fo4t6o530GzuIBDY2HDyfH4~pmRtkMkkb5iAFn8HgoKqMR2y~cc6mfT7jMP6yBA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Recombination between babA and babB sequences." path-from-xml="dsw046f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61463063/dsw046f3.tif" path-from-xml="dsw046f3.tif" target="_blank">View large</a><a section="61463063" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw046/1/dsw046f3.png?Expires=2147483647&Signature=M238TFb6kskhluA8njIhkHFFWl9BoExiTELQcTmlqhmC3urqUR2fMgtCi8WLBSQe3BP2mZiH0~5dGVNuB18YB1klDyiaNe4qyXinrN~Lglpq2ItghkAhkjsXHdGLa1sEoutKqGIxmSWtaycIAtVlWz1QO5sYBsbB10IRf4elv5d925gwZQ3aFIOk-CoS-3a67IN1pfmzDHSKGQXklD~2uUedXKYQeLYfeJhUL433hvG4c3eVSwf-OpV~~JMitWQpjw44xA69MNCnpHPo0FExK0r91ei8u9O2AaTPE1W3c973T1bsJS8H0uAcF~rABv1O~4bEER59dRqMwbUSp-GXUQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=61463063&ar=2670112&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw046f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Recombination between <em>babA</em> and <em>babB</em> sequences.</p></div></div></div>                            <h3 scrollto-destination=61463064 id="61463064" class="section-title" >3.5. Mutations in LPS biosynthesis genes</h3>
<p>Among the 89 total variants identified in 298 relative to UM032, 10 were located in two genes designated <em>UM298_1086</em> and <em>UM298_1395.</em> There is no genotypic difference present in these genes between strains 298 and 299. BLASTN comparison revealed that both genes encode for fucosyltranferase (FucT) that plays a substantial role in the synthesis of Lewis (Le) antigens of which the former gene product exhibits 75.9% amino acid sequence identity to HP0651 (FutB) whilst the latter harbours 78.3% identical amino acids to that of HP0379 (FutA). In <em>H. pylori</em> genomes, there are three phase-variable FucT-encoding genes, termed <em>futA, futB</em> and <em>futC</em>. FutA and FutB, both are paralogous and can fucosylate either Le<sup>c</sup> antigen (type I carbohydrate backbone) in a α-(1,4) linkage to generate Le<sup>a</sup> antigen, and/or N-acetyl-lactosamine (LacNAc) (type II carbohydrate backbone) in a α-(1,3) linkage to create Le<sup>x</sup> antigen.<a reveal-id="dsw046-B40 dsw046-B41 dsw046-B42" data-open="dsw046-B40 dsw046-B41 dsw046-B42" class="link link-ref link-reveal xref-bibr"><sup>40–42</sup></a> Both Le<sup>a</sup> and Le<sup>x</sup> antigens can be further fucosylated by FutC in a α-(1,2) manner to create difucoslylated Le<sup>b</sup> or Le<sup>y</sup> (<a reveal-id="dsw046-F5" data-open="dsw046-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>).<a reveal-id="dsw046-B43" data-open="dsw046-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a></p><p>In addition to the nucleotide variations found in FucT encoding genes, a 1-kb insertion was identified upstream of the <em>UM032_1363</em> counterpart in both the 298 and the 299 genomes, designated <em>UM298_1364</em> and <em>UM299_1364</em> respectively. <em>UM032_1363</em> shares approximately 80% nucleotide sequence identity with <em>HP0619</em> in <em>H. pylori</em> strain 26695, and <em>jhp0563</em> in <em>H. pylori</em> strain J99. <em>Jhp0563</em> is a <em>β-(1,3)galT</em> gene with a product essential for the expression of Le<sup>c</sup>.<a reveal-id="dsw046-B44" data-open="dsw046-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a> The upstream homologous glycosyltransferase-encoding gene <em>jhp0562</em>, is however absent in several <em>H. pylori</em> strains including 26695.<a reveal-id="dsw046-B45" data-open="dsw046-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a> Due to a high degree of shared nucleotide sequence identity, it has been demonstrated that both <em>jhp0562</em> and <em>β-(1,3)galT</em> genes can undergo intragenic recombination within a single strain to generate functional chimeric alleles. This explains why certain strains do not possess a <em>jhp0562</em> allele.<a reveal-id="dsw046-B45" data-open="dsw046-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a> Interestingly, the 1-kb insertion has introduced a gene which is 90.7% similar to <em>jhp0562</em> in the mice-derivative strains, designated 298_1363 and 299_1363 respectively. In this study, the mice were inoculated intragastrically with a pool consisting of 12 <em>H. pylori</em> clinical strains including UM032. The <em>jhp0562</em>-like allele was probably acquired by homologous recombination from one of the other strains inoculated.</p><div class="block-child-p">To further examine how the genotypic changes above would affect Le antigen phenotypic expression, immunoblots were performed on proteinase K-treated whole cell lysates with antibodies detecting Le<sup>a</sup>, Le<sup>b</sup>, Le<sup>x</sup> and Le<sup>y</sup> (<a reveal-id="dsw046-F4" data-open="dsw046-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>). All three strains reacted with the Le<sup>b</sup> antibody but differed in the sizes of decorated O-antigen chains. The immunoblot experiments also detected strong Le<sup>y</sup> expression in strains 298 and 299. However, only a trace level of Le<sup>y</sup> was detected in input strain UM032. <a reveal-id="dsw046-F5" data-open="dsw046-F5" class="link link-reveal link-table xref-fig">Figure 5</a> shows the Le antigen synthesis model in UM032 with all anticipated glycosyltransferase-encoding genes listed. The detection of both Le<sup>b</sup> and Le<sup>y</sup> indicates the presence of the α-1,2-fucosylation process, which is expected since all three strains had an identical in-frame <em>futC</em> gene. Last but not least, none of these strains revealed any detectable levels of Le<sup>a</sup> and Le<sup>x</sup> (data not shown), suggesting that there is a relatively high activity of FutC in these strains, leading to complete α-1,2-fucosylation of these substrates once presented. In addition, terminal fucosylation could have masked internal Le<sup>a</sup> or Le<sup>x</sup> and therefore hindered antibody detection. <div data-id="dsw046-F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw046/1/m_dsw046f4.png?Expires=2147483647&amp;Signature=bPG91Ljtr5nQVTlVt~MZoa9jxa0cH9Mc71GTHkRtYMjJ~F0hDhgyPTa54tJlnTZR3DCIzrkibooP5cKpGtapwVoHHvB-bw1bdPcE-g21YFzkNKnspREitxQF8C3pFKyJgjKQXcNUUGgLRDp6F8Lf75piuj~Ubziz1xuiXLsT0t5gkQyN-stEAAUBakLXq3iyHR6zCihqER5qq254I1ugMEYkEDOtkkcyz9ZMB4oAKxh505ZH9D1-o2LcbRpS6UzZVb8hPxumvOaRRmcvso43xOaVVP~wktVVOl4j7MYFPz6gPK0pspF3~fMo3~1pAjO3VOHNMGld4q1OO79qCQljoA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Immunoblot analysis of H. pylori strains UM032, 298 and 299 whole cell lysates with anti-Ley and anti-Leb antibodies." path-from-xml="dsw046f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61463067/dsw046f4.tif" path-from-xml="dsw046f4.tif" target="_blank">View large</a><a section="61463067" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw046/1/dsw046f4.png?Expires=2147483647&amp;Signature=LH6gU8HPqgp6gyZ8AlZDIILz63dR7upfxYb4SVyNYxG4otOy1YCIrFDou1iBOdImzz1gQupNW4nFG8jYs7GOwee5gbNbY6xrbRLxUvwsuwtC9eD6rX1gAZsM0fopdQ9K5f4b5Ozq~fFdwVnAOdjVnCmm6OYZY0acaxjVOFYiam3MAg8vhjziNCKs9-MOl9vnWeMXE-6rVpU59WAOoxZZ61roIQLgPXblw4qSOD9luvXwncvMMwfa6lYg6CAKp1fx-Qugrzflnk-uxdoA-2vWiUAltwFpD15hScmvPzuJx~UP4Q5Bq8Nt7ub8vQ81AwkIqj5Cf0MRvmwKXNtj4~9sPg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=61463067&amp;ar=2670112&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw046f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Immunoblot analysis of <em>H. pylori</em> strains UM032, 298 and 299 whole cell lysates with anti-Le<sup>y</sup> and anti-Le<sup>b</sup> antibodies.</p></div></div><div content-id="dsw046-F4" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw046/1/m_dsw046f4.png?Expires=2147483647&amp;Signature=bPG91Ljtr5nQVTlVt~MZoa9jxa0cH9Mc71GTHkRtYMjJ~F0hDhgyPTa54tJlnTZR3DCIzrkibooP5cKpGtapwVoHHvB-bw1bdPcE-g21YFzkNKnspREitxQF8C3pFKyJgjKQXcNUUGgLRDp6F8Lf75piuj~Ubziz1xuiXLsT0t5gkQyN-stEAAUBakLXq3iyHR6zCihqER5qq254I1ugMEYkEDOtkkcyz9ZMB4oAKxh505ZH9D1-o2LcbRpS6UzZVb8hPxumvOaRRmcvso43xOaVVP~wktVVOl4j7MYFPz6gPK0pspF3~fMo3~1pAjO3VOHNMGld4q1OO79qCQljoA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Immunoblot analysis of H. pylori strains UM032, 298 and 299 whole cell lysates with anti-Ley and anti-Leb antibodies." path-from-xml="dsw046f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61463067/dsw046f4.tif" path-from-xml="dsw046f4.tif" target="_blank">View large</a><a section="61463067" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw046/1/dsw046f4.png?Expires=2147483647&amp;Signature=LH6gU8HPqgp6gyZ8AlZDIILz63dR7upfxYb4SVyNYxG4otOy1YCIrFDou1iBOdImzz1gQupNW4nFG8jYs7GOwee5gbNbY6xrbRLxUvwsuwtC9eD6rX1gAZsM0fopdQ9K5f4b5Ozq~fFdwVnAOdjVnCmm6OYZY0acaxjVOFYiam3MAg8vhjziNCKs9-MOl9vnWeMXE-6rVpU59WAOoxZZ61roIQLgPXblw4qSOD9luvXwncvMMwfa6lYg6CAKp1fx-Qugrzflnk-uxdoA-2vWiUAltwFpD15hScmvPzuJx~UP4Q5Bq8Nt7ub8vQ81AwkIqj5Cf0MRvmwKXNtj4~9sPg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=61463067&amp;ar=2670112&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw046f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Immunoblot analysis of <em>H. pylori</em> strains UM032, 298 and 299 whole cell lysates with anti-Le<sup>y</sup> and anti-Le<sup>b</sup> antibodies.</p></div></div><div data-id="dsw046-F5" class="fig fig-section"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw046/1/m_dsw046f5.png?Expires=2147483647&amp;Signature=ySs9KfZrtRpNhrwqnFtQtVXVIV76YuFE8McWRfIlwtQNA9wtX2K-8-I5qqfZoXtGcxprnZQjHhXJxqK2ZtWlQ3FL~-7TpRpxlAtSvf96Cd0p5WCjjzOoAeSkCZ9B-rJVBEm9mSKqsPVQEpSToa6eEJUIKdkCOvTUBVxE8X-bRDklBt788nQlW8Dp2mGIrlazgYP2slqouZ0~gB8PLdDzHOlcQuim1K~c0SR~S9IKJoTDqzDKMkILerjloQveVaiqc6uze66a3t1A0mBUguy-8nj2h~kOmr57yArS23J2hY1LCY5v9novYU~xpU8IsL9uCT0ksYniHXweXD5O7peGTg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="A schematic diagram of type I and type II Lewis antigen biosynthesis pathways in H. pylori strain UM032. GlcNAc, N-acetylglucosamine; LacNAc, N-acetyl-D-lactosamine." path-from-xml="dsw046f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61463067/dsw046f5.tif" path-from-xml="dsw046f5.tif" target="_blank">View large</a><a section="61463067" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw046/1/dsw046f5.png?Expires=2147483647&amp;Signature=B6YWDubMT6sbQN84jowHCWIMtnKeI4CdezrWZeImG38J9Z3qhv2X6gton32bGN28ZPzTmI0jLO82dVRfspgdoC1mPN2D-IiS-MVR5v22-jEoedYRV0Dv~TE-6Z6dcOSA715Uqvc5dELY3zz2yawoGTfNh7U11IkmVI6PTwIFKRR4ngc8sxK85u4bZa9sPLdLFYKho5dCHLS8FDpiuvy4UWvhc9fExrYEjTx1vvVvq4OYBX1kP3OgHbKvxXVYKjEUuFhpU0abSPSHkiZGG7murK3ngSKyFZ5KMjswbNayKKnfsHe099bnUg3uN54D9-2vt6pIRoxsMoi8yWaHZ8G8mQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=61463067&amp;ar=2670112&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw046f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>A schematic diagram of type I and type II Lewis antigen biosynthesis pathways in <em>H. pylori</em> strain UM032. GlcNAc, N-acetylglucosamine; LacNAc, N-acetyl-D-lactosamine.</p></div></div><div content-id="dsw046-F5" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw046/1/m_dsw046f5.png?Expires=2147483647&amp;Signature=ySs9KfZrtRpNhrwqnFtQtVXVIV76YuFE8McWRfIlwtQNA9wtX2K-8-I5qqfZoXtGcxprnZQjHhXJxqK2ZtWlQ3FL~-7TpRpxlAtSvf96Cd0p5WCjjzOoAeSkCZ9B-rJVBEm9mSKqsPVQEpSToa6eEJUIKdkCOvTUBVxE8X-bRDklBt788nQlW8Dp2mGIrlazgYP2slqouZ0~gB8PLdDzHOlcQuim1K~c0SR~S9IKJoTDqzDKMkILerjloQveVaiqc6uze66a3t1A0mBUguy-8nj2h~kOmr57yArS23J2hY1LCY5v9novYU~xpU8IsL9uCT0ksYniHXweXD5O7peGTg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="A schematic diagram of type I and type II Lewis antigen biosynthesis pathways in H. pylori strain UM032. GlcNAc, N-acetylglucosamine; LacNAc, N-acetyl-D-lactosamine." path-from-xml="dsw046f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61463067/dsw046f5.tif" path-from-xml="dsw046f5.tif" target="_blank">View large</a><a section="61463067" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw046/1/dsw046f5.png?Expires=2147483647&amp;Signature=B6YWDubMT6sbQN84jowHCWIMtnKeI4CdezrWZeImG38J9Z3qhv2X6gton32bGN28ZPzTmI0jLO82dVRfspgdoC1mPN2D-IiS-MVR5v22-jEoedYRV0Dv~TE-6Z6dcOSA715Uqvc5dELY3zz2yawoGTfNh7U11IkmVI6PTwIFKRR4ngc8sxK85u4bZa9sPLdLFYKho5dCHLS8FDpiuvy4UWvhc9fExrYEjTx1vvVvq4OYBX1kP3OgHbKvxXVYKjEUuFhpU0abSPSHkiZGG7murK3ngSKyFZ5KMjswbNayKKnfsHe099bnUg3uN54D9-2vt6pIRoxsMoi8yWaHZ8G8mQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=61463067&amp;ar=2670112&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw046f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>A schematic diagram of type I and type II Lewis antigen biosynthesis pathways in <em>H. pylori</em> strain UM032. GlcNAc, N-acetylglucosamine; LacNAc, N-acetyl-D-lactosamine.</p></div></div></div><p>The minute expression of Le<sup>y</sup> in strain UM032 might be either due to low level of endogenous α-1,3-FucT activity or a low LacNAc content and thus a reduced expression of Le<sup>x</sup>. In strain UM032, the <em>futA</em> gene (<em>UM032_1394</em>) is switched off because its intragenic poly(C) tract located near the ATG start codon lacks one nucleotide, resulting in out-of-frame translation. The <em>futB</em> gene, by contrast, is in-frame, yielding correct full-length FucT enzyme in each strain. Given that FutA is inactive, the presence of both Le<sup>b</sup> and Le<sup>y</sup> in strain UM032 indicates that its FutB enzyme, encoded by <em>UM032_1086</em>, must contain both α-1,3 and α-1,4 properties. This is in agreement with our prediction by pairwise comparison that UM032_1086 displays extensive amino acid identity of 93.1% with a previously documented α-1,3/4-FucT found in <em>H. pylori</em> strain UA948 (GenBank accession number AF194963) (<a reveal-id="dsw046-T5" data-open="dsw046-T5" class="link link-reveal link-table xref-fig">Table 5</a>).<a reveal-id="dsw046-B40" data-open="dsw046-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a><sup>,</sup><a reveal-id="dsw046-B46" data-open="dsw046-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a><div content-id="dsw046-T5" class="table-modal"><div class="table-wrap"><div id="dsw046-T5" data-id="dsw046-T5" class="table-wrap-title"><div class="label -label">Table 5.</div><div class="caption"><p>Amino acid sequence identities between <em>H. pylori</em> fucosyltransferases</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Fucosylation </th><th><em>H. pylori</em> FucT </th><th>NCTC11639 </th><th>UA948 </th><th>DSM6709 </th><th>UM032_1086 </th><th>UM298_1086 </th><th>UM298_1395 </th></tr></thead><tbody><tr><td> </td><td> </td><td colspan="6"><hr />Distance </td></tr><tr><td>α-1,3 </td><td>NCTC11639 </td><td> </td><td char=".">0.22 </td><td char=".">0.14 </td><td char=".">0.22 </td><td char=".">0.22 </td><td char=".">0.2 </td></tr><tr><td>α-1,3 and α-1,4 </td><td>UA948 </td><td>76.19 </td><td> </td><td char=".">0.17 </td><td char=".">0.05 </td><td char=".">0.05 </td><td char=".">0.09 </td></tr><tr><td>α-1,4 </td><td>DSM6709 </td><td>77.34 </td><td char=".">77.47 </td><td> </td><td char=".">0.19 </td><td char=".">0.18 </td><td char=".">0.19 </td></tr><tr><td>α-1,3 and α-1,4 </td><td>UM032_1086<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn">*</a> </td><td>75.16 </td><td char=".">93.1 </td><td char=".">77.78 </td><td> </td><td char=".">0.01 </td><td char=".">0.06 </td></tr><tr><td>α-1,3 and α-1,4 </td><td>UM298_1086<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn">*</a> </td><td>71.43 </td><td char=".">89.22 </td><td char=".">75.82 </td><td char=".">94.53 </td><td> </td><td char=".">0.05 </td></tr><tr><td>α-1,3 and α-1,4 </td><td>UM298_1395<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn">*</a> </td><td>73.03 </td><td char=".">83.94 </td><td char=".">75.55 </td><td char=".">88.26 </td><td char=".">93.39 </td><td> </td></tr><tr><td> </td><td> </td><td colspan="6">% Identity </td></tr></tbody></table></div><div content-id="dsw046-T5" class="table-modal"><table><thead name="thead" align=""><tr><th>Fucosylation </th><th><em>H. pylori</em> FucT </th><th>NCTC11639 </th><th>UA948 </th><th>DSM6709 </th><th>UM032_1086 </th><th>UM298_1086 </th><th>UM298_1395 </th></tr></thead><tbody><tr><td> </td><td> </td><td colspan="6"><hr />Distance </td></tr><tr><td>α-1,3 </td><td>NCTC11639 </td><td> </td><td char=".">0.22 </td><td char=".">0.14 </td><td char=".">0.22 </td><td char=".">0.22 </td><td char=".">0.2 </td></tr><tr><td>α-1,3 and α-1,4 </td><td>UA948 </td><td>76.19 </td><td> </td><td char=".">0.17 </td><td char=".">0.05 </td><td char=".">0.05 </td><td char=".">0.09 </td></tr><tr><td>α-1,4 </td><td>DSM6709 </td><td>77.34 </td><td char=".">77.47 </td><td> </td><td char=".">0.19 </td><td char=".">0.18 </td><td char=".">0.19 </td></tr><tr><td>α-1,3 and α-1,4 </td><td>UM032_1086<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn">*</a> </td><td>75.16 </td><td char=".">93.1 </td><td char=".">77.78 </td><td> </td><td char=".">0.01 </td><td char=".">0.06 </td></tr><tr><td>α-1,3 and α-1,4 </td><td>UM298_1086<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn">*</a> </td><td>71.43 </td><td char=".">89.22 </td><td char=".">75.82 </td><td char=".">94.53 </td><td> </td><td char=".">0.05 </td></tr><tr><td>α-1,3 and α-1,4 </td><td>UM298_1395<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn">*</a> </td><td>73.03 </td><td char=".">83.94 </td><td char=".">75.55 </td><td char=".">88.26 </td><td char=".">93.39 </td><td> </td></tr><tr><td> </td><td> </td><td colspan="6">% Identity </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn2"><div class="fn"><div class="label -label">*</div><p>The enzymatic activity is predicted based on amino acid sequence comparison.</p></div></div></div></div></div><div class="table-wrap"><div id="dsw046-T5" data-id="dsw046-T5" class="table-wrap-title"><div class="label -label">Table 5.</div><div class="caption"><p>Amino acid sequence identities between <em>H. pylori</em> fucosyltransferases</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Fucosylation </th><th><em>H. pylori</em> FucT </th><th>NCTC11639 </th><th>UA948 </th><th>DSM6709 </th><th>UM032_1086 </th><th>UM298_1086 </th><th>UM298_1395 </th></tr></thead><tbody><tr><td> </td><td> </td><td colspan="6"><hr />Distance </td></tr><tr><td>α-1,3 </td><td>NCTC11639 </td><td> </td><td char=".">0.22 </td><td char=".">0.14 </td><td char=".">0.22 </td><td char=".">0.22 </td><td char=".">0.2 </td></tr><tr><td>α-1,3 and α-1,4 </td><td>UA948 </td><td>76.19 </td><td> </td><td char=".">0.17 </td><td char=".">0.05 </td><td char=".">0.05 </td><td char=".">0.09 </td></tr><tr><td>α-1,4 </td><td>DSM6709 </td><td>77.34 </td><td char=".">77.47 </td><td> </td><td char=".">0.19 </td><td char=".">0.18 </td><td char=".">0.19 </td></tr><tr><td>α-1,3 and α-1,4 </td><td>UM032_1086<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn">*</a> </td><td>75.16 </td><td char=".">93.1 </td><td char=".">77.78 </td><td> </td><td char=".">0.01 </td><td char=".">0.06 </td></tr><tr><td>α-1,3 and α-1,4 </td><td>UM298_1086<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn">*</a> </td><td>71.43 </td><td char=".">89.22 </td><td char=".">75.82 </td><td char=".">94.53 </td><td> </td><td char=".">0.05 </td></tr><tr><td>α-1,3 and α-1,4 </td><td>UM298_1395<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn">*</a> </td><td>73.03 </td><td char=".">83.94 </td><td char=".">75.55 </td><td char=".">88.26 </td><td char=".">93.39 </td><td> </td></tr><tr><td> </td><td> </td><td colspan="6">% Identity </td></tr></tbody></table></div><div content-id="dsw046-T5" class="table-modal"><table><thead name="thead" align=""><tr><th>Fucosylation </th><th><em>H. pylori</em> FucT </th><th>NCTC11639 </th><th>UA948 </th><th>DSM6709 </th><th>UM032_1086 </th><th>UM298_1086 </th><th>UM298_1395 </th></tr></thead><tbody><tr><td> </td><td> </td><td colspan="6"><hr />Distance </td></tr><tr><td>α-1,3 </td><td>NCTC11639 </td><td> </td><td char=".">0.22 </td><td char=".">0.14 </td><td char=".">0.22 </td><td char=".">0.22 </td><td char=".">0.2 </td></tr><tr><td>α-1,3 and α-1,4 </td><td>UA948 </td><td>76.19 </td><td> </td><td char=".">0.17 </td><td char=".">0.05 </td><td char=".">0.05 </td><td char=".">0.09 </td></tr><tr><td>α-1,4 </td><td>DSM6709 </td><td>77.34 </td><td char=".">77.47 </td><td> </td><td char=".">0.19 </td><td char=".">0.18 </td><td char=".">0.19 </td></tr><tr><td>α-1,3 and α-1,4 </td><td>UM032_1086<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn">*</a> </td><td>75.16 </td><td char=".">93.1 </td><td char=".">77.78 </td><td> </td><td char=".">0.01 </td><td char=".">0.06 </td></tr><tr><td>α-1,3 and α-1,4 </td><td>UM298_1086<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn">*</a> </td><td>71.43 </td><td char=".">89.22 </td><td char=".">75.82 </td><td char=".">94.53 </td><td> </td><td char=".">0.05 </td></tr><tr><td>α-1,3 and α-1,4 </td><td>UM298_1395<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn">*</a> </td><td>73.03 </td><td char=".">83.94 </td><td char=".">75.55 </td><td char=".">88.26 </td><td char=".">93.39 </td><td> </td></tr><tr><td> </td><td> </td><td colspan="6">% Identity </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn2"><div class="fn"><div class="label -label">*</div><p>The enzymatic activity is predicted based on amino acid sequence comparison.</p></div></div></div></div></p><div class="block-child-p">However, it has been demonstrated that UA948FucT exhibits a greater than 5-fold affinity towards the type II chain precursor to generate Le<sup>x</sup>. This does not support the hypothesis that a low level of α-1,3 activity is present within strain UM032. We next tested whether the minor changes in the amino acid composition could have altered the acceptor substrate affinity to make UM032_1086 favour the type I over the type II chain. To address this question, we performed protein structure modelling comparing both UA948FucT and UM032_1086. <a reveal-id="dsw046-F6" data-open="dsw046-F6" class="link link-reveal link-table xref-fig">Figure 6</a> shows the models of UA948FucT and UM032_1086, each model based on the crystal structure of uncomplexed FucT (PDB ID: 2NZW). The catalytic sites accounting for interaction with the donor substrate, GDP-fucose, are highlighted. While it is accepted that structure modelling can be insensitive to minor amino acid changes, no structural differences were observed between the two models. Thus, it seems unlikely that there is distinct acceptor substrate specificity between UA948FucT and UM032_1086. <div data-id="dsw046-F6" class="fig fig-section"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw046/1/m_dsw046f6.png?Expires=2147483647&Signature=Fe5NUuCrP8cpeesdskkpN0S4~wR~wbhGH~AyH1UxJJSDzWrjNUENYtueeziUJsOn~8Im5Im~mIn261tlOvQlKWHZ-5tqnDn-ANXd-rFLDkSbJGYBaa7OjLQuJKJMH~zQGzT1ZEiphS7RaRk5BIWgUQssrXBhkQpAVwaBuKBe~6DdGE9UkcSK9sBkbv38mLsJsf9OBslry8iPQF9K4Grx8KtMEOY9FlMwtUJahYS42rmevOSv6zeh9KTQdlJVEWcxgUlP2F-UZ5x60TRyAkoiVLXHZiCac5HrbgC2sWtP7AbsejSD7p~mO0ST1vjfn-zxGSZulIgeWM1vkXNc6PUx2A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Protein structure modelling comparing both UA948FucT and UM032_1086. The catalytic sites accounting for interaction with the donor substrate, GDP-fucose, are highlighted." path-from-xml="dsw046f6.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61463069/dsw046f6.tif" path-from-xml="dsw046f6.tif" target="_blank">View large</a><a section="61463069" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw046/1/dsw046f6.png?Expires=2147483647&Signature=Vg-8bueNT2PerZ8WH1BfzxQNmtflssMncnwMwPcyVJ7eJDPc4jvgAXQW0PM-jOCX9b-On2Kt3KhoyU78HYc9IfvU3wu4kPz9h-G83xGOaTZBqrTMtu0iQdINmusH9vgzNFn-OxbGH4yuC3-VjU4K5FwnEuyrlsJ1ZSgjcYZff35obswbw5TIYV92GxsFW8~~DQLAlU3GPZJ3zCPZ3pcZzhUb3zRjfz4F8EDZKDCPAJW6RoVJNg1g6Wi1dGfMV7JlpBuWDixF1vZDO3NhlRqOiL6VBplyB4xYcaaF6EPcs5s82Ra~YGsF5AQoklF3WpVlVfrmhxHGpCa4aC4GZ2aCBA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=61463069&ar=2670112&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw046f6.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Protein structure modelling comparing both UA948FucT and UM032_1086. The catalytic sites accounting for interaction with the donor substrate, GDP-fucose, are highlighted.</p></div></div><div content-id="dsw046-F6" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw046/1/m_dsw046f6.png?Expires=2147483647&Signature=Fe5NUuCrP8cpeesdskkpN0S4~wR~wbhGH~AyH1UxJJSDzWrjNUENYtueeziUJsOn~8Im5Im~mIn261tlOvQlKWHZ-5tqnDn-ANXd-rFLDkSbJGYBaa7OjLQuJKJMH~zQGzT1ZEiphS7RaRk5BIWgUQssrXBhkQpAVwaBuKBe~6DdGE9UkcSK9sBkbv38mLsJsf9OBslry8iPQF9K4Grx8KtMEOY9FlMwtUJahYS42rmevOSv6zeh9KTQdlJVEWcxgUlP2F-UZ5x60TRyAkoiVLXHZiCac5HrbgC2sWtP7AbsejSD7p~mO0ST1vjfn-zxGSZulIgeWM1vkXNc6PUx2A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Protein structure modelling comparing both UA948FucT and UM032_1086. The catalytic sites accounting for interaction with the donor substrate, GDP-fucose, are highlighted." path-from-xml="dsw046f6.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61463069/dsw046f6.tif" path-from-xml="dsw046f6.tif" target="_blank">View large</a><a section="61463069" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw046/1/dsw046f6.png?Expires=2147483647&Signature=Vg-8bueNT2PerZ8WH1BfzxQNmtflssMncnwMwPcyVJ7eJDPc4jvgAXQW0PM-jOCX9b-On2Kt3KhoyU78HYc9IfvU3wu4kPz9h-G83xGOaTZBqrTMtu0iQdINmusH9vgzNFn-OxbGH4yuC3-VjU4K5FwnEuyrlsJ1ZSgjcYZff35obswbw5TIYV92GxsFW8~~DQLAlU3GPZJ3zCPZ3pcZzhUb3zRjfz4F8EDZKDCPAJW6RoVJNg1g6Wi1dGfMV7JlpBuWDixF1vZDO3NhlRqOiL6VBplyB4xYcaaF6EPcs5s82Ra~YGsF5AQoklF3WpVlVfrmhxHGpCa4aC4GZ2aCBA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=61463069&ar=2670112&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw046f6.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Protein structure modelling comparing both UA948FucT and UM032_1086. The catalytic sites accounting for interaction with the donor substrate, GDP-fucose, are highlighted.</p></div></div></div><p>An alternative hypothesis is that the amount of LacNAc is inherently low within the UM032 strain, permitting FutB to utilise Le<sup>c</sup> for Le<sup>a</sup> synthesis once the LacNAc reservoir is depleted. To enhance LacNAc production and thus Le<sup>y</sup> expression in strains 298 and 299, an additional copy of gylcosyltransferase gene must be present. This is consistent with the acquisition of a <em>jhp0562</em>-like allele in both strains. <em>jhp0562</em> has been demonstrated in previous mutagenesis and complementation studies as an essential component in both type I and type II Le synthesis pathways.<a reveal-id="dsw046-B45" data-open="dsw046-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a> Despite displaying only 21% nucleotide sequence similarity to known <em>β-(1,4)galT</em> genes including <em>jhp0765</em> and <em>HP0826</em>, our findings offer further support for the idea that the product of this acquired <em>jhp0562</em>-like allele contains β-1,4-galactosyltransferase activity for the conversion of N-acetyl-glucosamine (GlyNAc) to LacNAc. Subsequently the expression of full-length FutA, which is expected to exhibit similar functions to that of FutB by conserving approximately 93.4% amino acids, in strains 298 and 299, provides additional α-1,3 enzymatic activity to produce Le<sup>x</sup> that is immediately converted into Le<sup>y</sup> by FutC.</p><p>The occurrence of genotypic and phenotypic changes in Le antigens in strain 298 relative to the input strain UM032, but not between strains 298 and 299, reflects an initial event of host-driven Le antigen expression adaptation taking place in <em>H. pylori</em> upon its first encounter to a new host species. This would aid the bacterium in establishment of long-term colonisation. The acquisition of <em>jhp0562</em>-like allele further suggests that the product of this non-phase-variable allele may further confer an increase in competitive advantage among <em>H. pylori</em> strains.</p>                            <h3 scrollto-destination=61463072 id="61463072" class="section-title" >3.6. Intergenic homopolymeric tract length alterations affect gene expression</h3>
<div class="block-child-p">In this study, a number of intragenic and intergenic homopolymeric tract length changes were detected in strains 298 and 299 (<a reveal-id="dsw046-T6" data-open="dsw046-T6" class="link link-reveal link-table xref-fig">Tables 6</a> and <a reveal-id="dsw046-T7" data-open="dsw046-T7" class="link link-reveal link-table xref-fig">7</a>). In order to verify if these variations in intergenic homopolymeric tract lengths influence the expression of adjacent genes, we performed qRT-PCR analysis to better understand how these transcriptional changes, if any, can possibly facilitate novel host colonisation. Data are shown as fold change relative to the transcriptional level of each corresponding gene in strain UM032, as depicted in <a reveal-id="dsw046-F7" data-open="dsw046-F7" class="link link-reveal link-table xref-fig">Figure 7</a>. Among nine tested gene candidates, five were significantly greater than four-fold down-regulated, including <em>UM032_0212</em> (hypothetical protein), <em>UM032_0548</em> (HopD), <em>UM032_0781</em> (biotin synthase), <em>UM032_0908</em> (BabB) and <em>UM032_1223</em> (BabA). A less than two-fold change was observed between <em>UM032_0213</em> (CTP synthase) and its counterparts. No amplification was detected for <em>UM032_0025</em>, <em>UM032_0547</em> and <em>UM032_1372</em> (data not shown), suggesting that these genes were either not expressed or were expressed at very low levels. <div data-id="dsw046-F7" class="fig fig-section"><div class="label fig-label">Figure 7</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw046/1/m_dsw046f7.png?Expires=2147483647&Signature=MiDOjW7bEZAtARwsBZKu~l5HMzE6qfENSOjh7p-Eo~GaRmgmnl9v1svhxXLubWsauXrr6Qjmgyegd87egvWtV7RKyL6jBS3T8YUeVqMw77-cX8B55nNMT0a1TASvit~C~nqxp8RjVEcGgMeYr~KxtKSonyahylZnoYbmTEszhcGF17tIwNEylB5HYcbfZmPlUPu5wDKo-~SiWH7PATheSav2gyxmTdSb9-F~LfRcEAJammjXoklwIENlGxSiaDGFxbP6j8VPRC~VXwp8~8d5IS0X3hrGvW5~2KGROAeloAwKJnRmwDcpbIBlFZTAmNlRegnppe6O3mZ4E6sZTOs1uw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Real-time quantitation of genes identified with modified intergenic homopolymer-length. Data are expressed as fold change relative to strain UM032. The symbol * indicates statistical significance where p&lt;0.05." path-from-xml="dsw046f7.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61463073/dsw046f7.tif" path-from-xml="dsw046f7.tif" target="_blank">View large</a><a section="61463073" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw046/1/dsw046f7.png?Expires=2147483647&Signature=0DNR5jc8Vu0BZrTUMwRm82mb1P0zdiTb-FyOwL1UXfTEC3Vj2YNfCq3ER58hM6AfrGUdBqlfWRZrZQ4fbRx4r4ZujFKzscwafOwtDIsCF5YqDaVQhMEDzUP0gmw8R1KZHMcAJ5cyVUvEFrDOZp5PWKQEQ9OQcMgU1QlNzJWqbeMq9WUJI9bOxXV1SaSpLIbZWs-8EodboIT~S5gkkP8QU8CvPuWY0XjIH1FFMjhkvJ1N-UB1KOtn2tNimK~nHVcZoZXq3wvzFpdc~C1dyIz0-FwWSbIZkMBpdlZFXLfylDAqGbrYlEWjL3FSIc2Lk7IRs3c4ZND0Zj9sJdBbgNgrHw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=61463073&ar=2670112&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw046f7.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Real-time quantitation of genes identified with modified intergenic homopolymer-length. Data are expressed as fold change relative to strain UM032. The symbol * indicates statistical significance where <em>p</em>&lt;0.05.</p></div></div><div content-id="dsw046-F7" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 7</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw046/1/m_dsw046f7.png?Expires=2147483647&Signature=MiDOjW7bEZAtARwsBZKu~l5HMzE6qfENSOjh7p-Eo~GaRmgmnl9v1svhxXLubWsauXrr6Qjmgyegd87egvWtV7RKyL6jBS3T8YUeVqMw77-cX8B55nNMT0a1TASvit~C~nqxp8RjVEcGgMeYr~KxtKSonyahylZnoYbmTEszhcGF17tIwNEylB5HYcbfZmPlUPu5wDKo-~SiWH7PATheSav2gyxmTdSb9-F~LfRcEAJammjXoklwIENlGxSiaDGFxbP6j8VPRC~VXwp8~8d5IS0X3hrGvW5~2KGROAeloAwKJnRmwDcpbIBlFZTAmNlRegnppe6O3mZ4E6sZTOs1uw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Real-time quantitation of genes identified with modified intergenic homopolymer-length. Data are expressed as fold change relative to strain UM032. The symbol * indicates statistical significance where p&lt;0.05." path-from-xml="dsw046f7.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61463073/dsw046f7.tif" path-from-xml="dsw046f7.tif" target="_blank">View large</a><a section="61463073" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw046/1/dsw046f7.png?Expires=2147483647&Signature=0DNR5jc8Vu0BZrTUMwRm82mb1P0zdiTb-FyOwL1UXfTEC3Vj2YNfCq3ER58hM6AfrGUdBqlfWRZrZQ4fbRx4r4ZujFKzscwafOwtDIsCF5YqDaVQhMEDzUP0gmw8R1KZHMcAJ5cyVUvEFrDOZp5PWKQEQ9OQcMgU1QlNzJWqbeMq9WUJI9bOxXV1SaSpLIbZWs-8EodboIT~S5gkkP8QU8CvPuWY0XjIH1FFMjhkvJ1N-UB1KOtn2tNimK~nHVcZoZXq3wvzFpdc~C1dyIz0-FwWSbIZkMBpdlZFXLfylDAqGbrYlEWjL3FSIc2Lk7IRs3c4ZND0Zj9sJdBbgNgrHw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=61463073&ar=2670112&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw046f7.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Real-time quantitation of genes identified with modified intergenic homopolymer-length. Data are expressed as fold change relative to strain UM032. The symbol * indicates statistical significance where <em>p</em>&lt;0.05.</p></div></div><div content-id="dsw046-T6" class="table-modal"><div class="table-wrap"><div id="dsw046-T6" data-id="dsw046-T6" class="table-wrap-title"><div class="label -label">Table 6.</div><div class="caption"><p>List of altered intragenic homopolymeric tracts</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th colspan="2">UM032<hr /> </th><th colspan="2">298<hr /> </th><th colspan="2">299<hr /> </th></tr><tr><th>Product </th><th>Gene </th><th>Pseudo </th><th>Gene </th><th>Pseudo </th><th>Gene </th><th>Pseudo </th></tr></thead><tbody><tr><td>Tetratricopeptide repeat family protein </td><td>UM032_0224 </td><td>No </td><td>UM298_0224 </td><td>Yes </td><td>UM298_0224 </td><td>Yes </td></tr><tr><td>Oligopeptide transport system permease protein OppC </td><td>UM032_0307 </td><td>No </td><td>UM298_0307 </td><td>Yes </td><td>UM299_0307 </td><td>Yes </td></tr><tr><td>Putative metal-dependent hydrolase fragment 1 </td><td>UM032_0607 </td><td>No </td><td>UM298_0607 </td><td>Yes </td><td>UM299_0607 </td><td>Yes </td></tr><tr><td>Hypothetical protein </td><td>UM032_0755 </td><td>No </td><td>UM298_0755 </td><td>Yes </td><td>UM299_0755 </td><td>No </td></tr><tr><td>α-(1,3)-fucosyltransferase </td><td>UM032_1394 </td><td>Yes </td><td>UM298_1395 </td><td>No </td><td>UM299_1395 </td><td>No </td></tr></tbody></table></div><div content-id="dsw046-T6" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th colspan="2">UM032<hr /> </th><th colspan="2">298<hr /> </th><th colspan="2">299<hr /> </th></tr><tr><th>Product </th><th>Gene </th><th>Pseudo </th><th>Gene </th><th>Pseudo </th><th>Gene </th><th>Pseudo </th></tr></thead><tbody><tr><td>Tetratricopeptide repeat family protein </td><td>UM032_0224 </td><td>No </td><td>UM298_0224 </td><td>Yes </td><td>UM298_0224 </td><td>Yes </td></tr><tr><td>Oligopeptide transport system permease protein OppC </td><td>UM032_0307 </td><td>No </td><td>UM298_0307 </td><td>Yes </td><td>UM299_0307 </td><td>Yes </td></tr><tr><td>Putative metal-dependent hydrolase fragment 1 </td><td>UM032_0607 </td><td>No </td><td>UM298_0607 </td><td>Yes </td><td>UM299_0607 </td><td>Yes </td></tr><tr><td>Hypothetical protein </td><td>UM032_0755 </td><td>No </td><td>UM298_0755 </td><td>Yes </td><td>UM299_0755 </td><td>No </td></tr><tr><td>α-(1,3)-fucosyltransferase </td><td>UM032_1394 </td><td>Yes </td><td>UM298_1395 </td><td>No </td><td>UM299_1395 </td><td>No </td></tr></tbody></table></div></div></div><div class="table-wrap"><div id="dsw046-T6" data-id="dsw046-T6" class="table-wrap-title"><div class="label -label">Table 6.</div><div class="caption"><p>List of altered intragenic homopolymeric tracts</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th colspan="2">UM032<hr /> </th><th colspan="2">298<hr /> </th><th colspan="2">299<hr /> </th></tr><tr><th>Product </th><th>Gene </th><th>Pseudo </th><th>Gene </th><th>Pseudo </th><th>Gene </th><th>Pseudo </th></tr></thead><tbody><tr><td>Tetratricopeptide repeat family protein </td><td>UM032_0224 </td><td>No </td><td>UM298_0224 </td><td>Yes </td><td>UM298_0224 </td><td>Yes </td></tr><tr><td>Oligopeptide transport system permease protein OppC </td><td>UM032_0307 </td><td>No </td><td>UM298_0307 </td><td>Yes </td><td>UM299_0307 </td><td>Yes </td></tr><tr><td>Putative metal-dependent hydrolase fragment 1 </td><td>UM032_0607 </td><td>No </td><td>UM298_0607 </td><td>Yes </td><td>UM299_0607 </td><td>Yes </td></tr><tr><td>Hypothetical protein </td><td>UM032_0755 </td><td>No </td><td>UM298_0755 </td><td>Yes </td><td>UM299_0755 </td><td>No </td></tr><tr><td>α-(1,3)-fucosyltransferase </td><td>UM032_1394 </td><td>Yes </td><td>UM298_1395 </td><td>No </td><td>UM299_1395 </td><td>No </td></tr></tbody></table></div><div content-id="dsw046-T6" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th colspan="2">UM032<hr /> </th><th colspan="2">298<hr /> </th><th colspan="2">299<hr /> </th></tr><tr><th>Product </th><th>Gene </th><th>Pseudo </th><th>Gene </th><th>Pseudo </th><th>Gene </th><th>Pseudo </th></tr></thead><tbody><tr><td>Tetratricopeptide repeat family protein </td><td>UM032_0224 </td><td>No </td><td>UM298_0224 </td><td>Yes </td><td>UM298_0224 </td><td>Yes </td></tr><tr><td>Oligopeptide transport system permease protein OppC </td><td>UM032_0307 </td><td>No </td><td>UM298_0307 </td><td>Yes </td><td>UM299_0307 </td><td>Yes </td></tr><tr><td>Putative metal-dependent hydrolase fragment 1 </td><td>UM032_0607 </td><td>No </td><td>UM298_0607 </td><td>Yes </td><td>UM299_0607 </td><td>Yes </td></tr><tr><td>Hypothetical protein </td><td>UM032_0755 </td><td>No </td><td>UM298_0755 </td><td>Yes </td><td>UM299_0755 </td><td>No </td></tr><tr><td>α-(1,3)-fucosyltransferase </td><td>UM032_1394 </td><td>Yes </td><td>UM298_1395 </td><td>No </td><td>UM299_1395 </td><td>No </td></tr></tbody></table></div></div><div content-id="dsw046-T7" class="table-modal"><div class="table-wrap"><div id="dsw046-T7" data-id="dsw046-T7" class="table-wrap-title"><div class="label -label">Table 7.</div><div class="caption"><p>Loci in strain UM032 with altered intergenic homopolymeric tracts relative to strains 298 and 299</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Locus tag </th><th rowspan="2">Gene product </th><th rowspan="2">Length changes in output strains UM298 &amp; UM299 </th><th rowspan="2">Position </th><th colspan="3">Tract length comparison in 49 <em>H. pylori</em> complete genomes excluding strains UM032, 298 and 299<hr /> </th></tr><tr><th>Observed frequency </th><th>Maximum length (bp) </th><th>Minimum length (bp) </th></tr></thead><tbody><tr><td>UM032_0025 </td><td>Hypothetical protein </td><td>(A)14→13 </td><td char=".">≪ −35 </td><td>48/49 </td><td>19 </td><td>7 </td></tr><tr><td>UM032_0212 </td><td>Hypothetical protein </td><td>(A)15→16 </td><td char="."> &lt; −35 </td><td>49/49 </td><td>22 </td><td>11 </td></tr><tr><td>UM032_0213 </td><td>CTP synthase </td><td>(T)15→16 </td><td char=".">≪ −35 </td><td>49/49 </td><td>22 </td><td>11 </td></tr><tr><td>UM032_0547 </td><td>Putative endonuclease G </td><td>(A)16→15 </td><td char=".">−35/−10 </td><td>15/49 </td><td>17 </td><td>11 </td></tr><tr><td>UM032_0548 </td><td>Outer membrane protein HopD </td><td>(T)16→15 </td><td char="."> &lt; −35 </td><td>49/49 </td><td>21 </td><td>8 </td></tr><tr><td>UM032_0781 </td><td>Biotin synthase </td><td>(G)12→13 </td><td char=".">≪ −35 </td><td>46/49 </td><td>14 </td><td>7 </td></tr><tr><td>UM032_0908 </td><td>Outer membrane protein BabB </td><td>(T)14→13 </td><td char=".">≪ −35 </td><td>27/49 </td><td>22 </td><td>7 </td></tr><tr><td>UM032_1223 </td><td>Outer membrane protein BabA </td><td>(A)12→13 </td><td char=".">−35/−10 </td><td>44/49 </td><td>15 </td><td>8 </td></tr><tr><td>UM032_1372 </td><td>Hypothetical protein </td><td>(T)12→10 </td><td>N/A<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn">*</a> </td><td>49/49 </td><td>18 </td><td>8 </td></tr></tbody></table></div><div content-id="dsw046-T7" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Locus tag </th><th rowspan="2">Gene product </th><th rowspan="2">Length changes in output strains UM298 &amp; UM299 </th><th rowspan="2">Position </th><th colspan="3">Tract length comparison in 49 <em>H. pylori</em> complete genomes excluding strains UM032, 298 and 299<hr /> </th></tr><tr><th>Observed frequency </th><th>Maximum length (bp) </th><th>Minimum length (bp) </th></tr></thead><tbody><tr><td>UM032_0025 </td><td>Hypothetical protein </td><td>(A)14→13 </td><td char=".">≪ −35 </td><td>48/49 </td><td>19 </td><td>7 </td></tr><tr><td>UM032_0212 </td><td>Hypothetical protein </td><td>(A)15→16 </td><td char="."> &lt; −35 </td><td>49/49 </td><td>22 </td><td>11 </td></tr><tr><td>UM032_0213 </td><td>CTP synthase </td><td>(T)15→16 </td><td char=".">≪ −35 </td><td>49/49 </td><td>22 </td><td>11 </td></tr><tr><td>UM032_0547 </td><td>Putative endonuclease G </td><td>(A)16→15 </td><td char=".">−35/−10 </td><td>15/49 </td><td>17 </td><td>11 </td></tr><tr><td>UM032_0548 </td><td>Outer membrane protein HopD </td><td>(T)16→15 </td><td char="."> &lt; −35 </td><td>49/49 </td><td>21 </td><td>8 </td></tr><tr><td>UM032_0781 </td><td>Biotin synthase </td><td>(G)12→13 </td><td char=".">≪ −35 </td><td>46/49 </td><td>14 </td><td>7 </td></tr><tr><td>UM032_0908 </td><td>Outer membrane protein BabB </td><td>(T)14→13 </td><td char=".">≪ −35 </td><td>27/49 </td><td>22 </td><td>7 </td></tr><tr><td>UM032_1223 </td><td>Outer membrane protein BabA </td><td>(A)12→13 </td><td char=".">−35/−10 </td><td>44/49 </td><td>15 </td><td>8 </td></tr><tr><td>UM032_1372 </td><td>Hypothetical protein </td><td>(T)12→10 </td><td>N/A<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn">*</a> </td><td>49/49 </td><td>18 </td><td>8 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn3"><div class="fn"><div class="label -label">*</div><p>This ORF is the last gene in an operon.</p></div></div></div></div></div><div class="table-wrap"><div id="dsw046-T7" data-id="dsw046-T7" class="table-wrap-title"><div class="label -label">Table 7.</div><div class="caption"><p>Loci in strain UM032 with altered intergenic homopolymeric tracts relative to strains 298 and 299</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Locus tag </th><th rowspan="2">Gene product </th><th rowspan="2">Length changes in output strains UM298 &amp; UM299 </th><th rowspan="2">Position </th><th colspan="3">Tract length comparison in 49 <em>H. pylori</em> complete genomes excluding strains UM032, 298 and 299<hr /> </th></tr><tr><th>Observed frequency </th><th>Maximum length (bp) </th><th>Minimum length (bp) </th></tr></thead><tbody><tr><td>UM032_0025 </td><td>Hypothetical protein </td><td>(A)14→13 </td><td char=".">≪ −35 </td><td>48/49 </td><td>19 </td><td>7 </td></tr><tr><td>UM032_0212 </td><td>Hypothetical protein </td><td>(A)15→16 </td><td char="."> &lt; −35 </td><td>49/49 </td><td>22 </td><td>11 </td></tr><tr><td>UM032_0213 </td><td>CTP synthase </td><td>(T)15→16 </td><td char=".">≪ −35 </td><td>49/49 </td><td>22 </td><td>11 </td></tr><tr><td>UM032_0547 </td><td>Putative endonuclease G </td><td>(A)16→15 </td><td char=".">−35/−10 </td><td>15/49 </td><td>17 </td><td>11 </td></tr><tr><td>UM032_0548 </td><td>Outer membrane protein HopD </td><td>(T)16→15 </td><td char="."> &lt; −35 </td><td>49/49 </td><td>21 </td><td>8 </td></tr><tr><td>UM032_0781 </td><td>Biotin synthase </td><td>(G)12→13 </td><td char=".">≪ −35 </td><td>46/49 </td><td>14 </td><td>7 </td></tr><tr><td>UM032_0908 </td><td>Outer membrane protein BabB </td><td>(T)14→13 </td><td char=".">≪ −35 </td><td>27/49 </td><td>22 </td><td>7 </td></tr><tr><td>UM032_1223 </td><td>Outer membrane protein BabA </td><td>(A)12→13 </td><td char=".">−35/−10 </td><td>44/49 </td><td>15 </td><td>8 </td></tr><tr><td>UM032_1372 </td><td>Hypothetical protein </td><td>(T)12→10 </td><td>N/A<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn">*</a> </td><td>49/49 </td><td>18 </td><td>8 </td></tr></tbody></table></div><div content-id="dsw046-T7" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Locus tag </th><th rowspan="2">Gene product </th><th rowspan="2">Length changes in output strains UM298 &amp; UM299 </th><th rowspan="2">Position </th><th colspan="3">Tract length comparison in 49 <em>H. pylori</em> complete genomes excluding strains UM032, 298 and 299<hr /> </th></tr><tr><th>Observed frequency </th><th>Maximum length (bp) </th><th>Minimum length (bp) </th></tr></thead><tbody><tr><td>UM032_0025 </td><td>Hypothetical protein </td><td>(A)14→13 </td><td char=".">≪ −35 </td><td>48/49 </td><td>19 </td><td>7 </td></tr><tr><td>UM032_0212 </td><td>Hypothetical protein </td><td>(A)15→16 </td><td char="."> &lt; −35 </td><td>49/49 </td><td>22 </td><td>11 </td></tr><tr><td>UM032_0213 </td><td>CTP synthase </td><td>(T)15→16 </td><td char=".">≪ −35 </td><td>49/49 </td><td>22 </td><td>11 </td></tr><tr><td>UM032_0547 </td><td>Putative endonuclease G </td><td>(A)16→15 </td><td char=".">−35/−10 </td><td>15/49 </td><td>17 </td><td>11 </td></tr><tr><td>UM032_0548 </td><td>Outer membrane protein HopD </td><td>(T)16→15 </td><td char="."> &lt; −35 </td><td>49/49 </td><td>21 </td><td>8 </td></tr><tr><td>UM032_0781 </td><td>Biotin synthase </td><td>(G)12→13 </td><td char=".">≪ −35 </td><td>46/49 </td><td>14 </td><td>7 </td></tr><tr><td>UM032_0908 </td><td>Outer membrane protein BabB </td><td>(T)14→13 </td><td char=".">≪ −35 </td><td>27/49 </td><td>22 </td><td>7 </td></tr><tr><td>UM032_1223 </td><td>Outer membrane protein BabA </td><td>(A)12→13 </td><td char=".">−35/−10 </td><td>44/49 </td><td>15 </td><td>8 </td></tr><tr><td>UM032_1372 </td><td>Hypothetical protein </td><td>(T)12→10 </td><td>N/A<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn">*</a> </td><td>49/49 </td><td>18 </td><td>8 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn3"><div class="fn"><div class="label -label">*</div><p>This ORF is the last gene in an operon.</p></div></div></div></div></div><p><em>UM032_0213</em> mRNA levels remained unaffected by the length change in a poly(T) tract located 30 nucleotides upstream of the -35 element. However, <em>UM032_0212</em> transcription was significantly down-regulated at the poly(A) tract positioned 3 nucleotides upstream of the -35 element thatwas altered in length. The latter result was consistent with the findings of Åberg <em>et al.</em> that variation in the length of homopolymeric tracts located adjacent to the −35 element modulates promoter activity by changing local DNA structure and thereby binding of the RNA polymerase.<a reveal-id="dsw046-B13" data-open="dsw046-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> Similarly, transcriptional activity was also reduced in both outer membrane protein-encoding <em>UM032_0548</em> and <em>UM32_0908</em> as the poly(T) tract situated ∼20 nucleotides upstream of each −35 element was reduced by single base pair in both strains 298 and 299. This indicates that regulation via variation in homopolymeric tract is a fairly general mechanism in <em>H. pylori</em>.</p><div class="block-child-p">On the other hand, BabA-encoding <em>UM032_1223</em> had an altered poly(A) tract located between the -35 and -10 elements. A Western immunoblot was further conducted using anti-BabA antibody and as expected, BabA expression was nearly lost in strains 298 and 299 (<a reveal-id="dsw046-F8" data-open="dsw046-F8" class="link link-reveal link-table xref-fig">Fig. 8</a>). We also performed another Western analysis with commercial polyclonal antibody against <em>H. pylori</em> urease B (UreB) to demonstrate that comparable amounts of proteins are loaded in each lane (<a reveal-id="dsw046-F9" data-open="dsw046-F9" class="link link-reveal link-table xref-fig">Fig. 9</a>). It has been demonstrated in a number of previous studies that variation in the distance between these two consensus elements can affect RNA polymerase binding and thus the efficiency of gene transcription.<a reveal-id="dsw046-B47" data-open="dsw046-B47" class="link link-ref link-reveal xref-bibr"><sup>47</sup></a><sup>,</sup><a reveal-id="dsw046-B48" data-open="dsw046-B48" class="link link-ref link-reveal xref-bibr"><sup>48</sup></a> Last but not least, the altered poly(G) tract of down-regulated <em>UM032_0781</em> was found to lie more than 150 nucleotides upstream of the -35 region. This is probably due to the presence of an unknown distal negative cis-acting element, similar to that found in the phase variable NadA adhesin of <em>Neisseria meningitides</em>.<a reveal-id="dsw046-B49" data-open="dsw046-B49" class="link link-ref link-reveal xref-bibr"><sup>49</sup></a><div data-id="dsw046-F8" class="fig fig-section"><div class="label fig-label">Figure 8</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw046/1/m_dsw046f8.png?Expires=2147483647&amp;Signature=oBwij9DNRI2wOaPouWy8tCSr4PbqRNQxU5Kn6cLTMjjBDkOq0TL7wUchUTt-2NRR3sB8Z85O0n9mZ~PZbH3zuTfBsrF6gcPfkgtoT~qMHhXGef3WnPb3V562td~ZV~cIRIfrPgOXqpbXEvS6pmdZXk~kEbePyNzIhhuh5DJ952jW1KShlfsp-5b4bGoWTLI094u5J0YhBE-RrnqJyOf1w9AgxCbdWj6HnhMlgw8KwxcYGkOjoD4m8CesG2Iemh4JVTs6UR7tEzEA48kDZ9MmcwHuFVzH5UZSTXGN-oNaf0dYUBlq98EwZ8JAYx-phJ7ZY1NgtX082t-X90a6oA88gw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Western immunodetection of BabA in H. pylori strains UM032, 298 and 299. G27 and 26695 served as the positive and negative controls, respectively, in this assay." path-from-xml="dsw046f8.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61463075/dsw046f8.tif" path-from-xml="dsw046f8.tif" target="_blank">View large</a><a section="61463075" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw046/1/dsw046f8.png?Expires=2147483647&amp;Signature=idwWl5OrwE9lwD3F5-mhXapfoCIbzwMpfwYdAWFZsUqHbgqCkp~q916OUOfA8rzANgesmQqgL4YB2tTMC-Mt~PCtMvEuYUgzcmHsLONJiPCN8SqkF-k4Z80tLuvX5vLRSPS98x4BwYG-sr3cSNyVZURSQXDDYytAJBvAjJHHb~til1pcaAgk3yQM4F6VqBkpIPyMONSJ9AygQddIqboOlw3Lp-gh1hLvEU8Yk0aZh4JgNXyemIDynojDn1pcfzm7BV9J3f0sgjBnCLTnxX~FhcNCahYpFbVWYz1dDmBAWXl9lHcwdNUupAShTnIuyDzzALmca1C0Zi-NPYeT~0IB7g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=61463075&amp;ar=2670112&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw046f8.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Western immunodetection of BabA in <em>H. pylori</em> strains UM032, 298 and 299. G27 and 26695 served as the positive and negative controls, respectively, in this assay.</p></div></div><div content-id="dsw046-F8" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 8</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw046/1/m_dsw046f8.png?Expires=2147483647&amp;Signature=oBwij9DNRI2wOaPouWy8tCSr4PbqRNQxU5Kn6cLTMjjBDkOq0TL7wUchUTt-2NRR3sB8Z85O0n9mZ~PZbH3zuTfBsrF6gcPfkgtoT~qMHhXGef3WnPb3V562td~ZV~cIRIfrPgOXqpbXEvS6pmdZXk~kEbePyNzIhhuh5DJ952jW1KShlfsp-5b4bGoWTLI094u5J0YhBE-RrnqJyOf1w9AgxCbdWj6HnhMlgw8KwxcYGkOjoD4m8CesG2Iemh4JVTs6UR7tEzEA48kDZ9MmcwHuFVzH5UZSTXGN-oNaf0dYUBlq98EwZ8JAYx-phJ7ZY1NgtX082t-X90a6oA88gw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Western immunodetection of BabA in H. pylori strains UM032, 298 and 299. G27 and 26695 served as the positive and negative controls, respectively, in this assay." path-from-xml="dsw046f8.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61463075/dsw046f8.tif" path-from-xml="dsw046f8.tif" target="_blank">View large</a><a section="61463075" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw046/1/dsw046f8.png?Expires=2147483647&amp;Signature=idwWl5OrwE9lwD3F5-mhXapfoCIbzwMpfwYdAWFZsUqHbgqCkp~q916OUOfA8rzANgesmQqgL4YB2tTMC-Mt~PCtMvEuYUgzcmHsLONJiPCN8SqkF-k4Z80tLuvX5vLRSPS98x4BwYG-sr3cSNyVZURSQXDDYytAJBvAjJHHb~til1pcaAgk3yQM4F6VqBkpIPyMONSJ9AygQddIqboOlw3Lp-gh1hLvEU8Yk0aZh4JgNXyemIDynojDn1pcfzm7BV9J3f0sgjBnCLTnxX~FhcNCahYpFbVWYz1dDmBAWXl9lHcwdNUupAShTnIuyDzzALmca1C0Zi-NPYeT~0IB7g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=61463075&amp;ar=2670112&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw046f8.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Western immunodetection of BabA in <em>H. pylori</em> strains UM032, 298 and 299. G27 and 26695 served as the positive and negative controls, respectively, in this assay.</p></div></div><div data-id="dsw046-F9" class="fig fig-section"><div class="label fig-label">Figure 9</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw046/1/m_dsw046f9.png?Expires=2147483647&amp;Signature=b6ZhuhqeniGWBkCxdBVBL70LZ-g2J8gtgWZIrCj-XGYLGw3b5DF~OMuSZkqbpo3Rns0BkFErwYyaHHi9SZ4NHxLrPtttIXLYOPIfoSgQ321oOss83CohxYJOPJOZEQfZm4TVtfdtW~Q3UJYfgiQJwJm1OgGEbC0YUxPVMW65NUuT1kGQDKqN~G05TwH2E7L2~SIIMzAAy1cfL-IqbaCOrDY5dD5Gb8H5Mn1nwlpSVeR19XZpkt0Z73c8EsZ5Ml1N-bF73MtLYGSIenCvMxLLx7z9IeYyeim7L5FAZ034WCizfe17GABNuUtYzjxnXPIB7T6UR5nJNL9qE5x1YIPs9A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Western immunodetection of UreB in H. pylori strains UM032, 298 and 299." path-from-xml="dsw046f9.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61463075/dsw046f9.tif" path-from-xml="dsw046f9.tif" target="_blank">View large</a><a section="61463075" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw046/1/dsw046f9.png?Expires=2147483647&amp;Signature=APF7JwRnzUdKnleSUf3l9IGNPhYTWn3CXjxftzEyW7iykBEjB8Azxn7gfKI6aLLa74utQ3ZYy4mdbZgEMQOVMdSQeCLJojSsl5kIp2hK0Dme-bteyFCdrz69qmIWKp1-7n5bKppqAfZLb6UTfF8iUUTmeeUqfIPbJcfDVHO-OEKUoePlDbDPNu6TYO1lUgDY1o9AaQeDZrh6niUn2INiqpDqtF7eID9B1O1xaeZVJ5d9u96-lVAOx7q1TjARXxP5yX3UkxzGmegjU4CQJu6xh2f9akvfj006885XK3LUCnKEe57K9QWG0nBwZESQkEEAsAjmYkQOGb2P~QastFJkTg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=61463075&amp;ar=2670112&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw046f9.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Western immunodetection of UreB in <em>H. pylori</em> strains UM032, 298 and 299.</p></div></div><div content-id="dsw046-F9" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 9</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw046/1/m_dsw046f9.png?Expires=2147483647&amp;Signature=b6ZhuhqeniGWBkCxdBVBL70LZ-g2J8gtgWZIrCj-XGYLGw3b5DF~OMuSZkqbpo3Rns0BkFErwYyaHHi9SZ4NHxLrPtttIXLYOPIfoSgQ321oOss83CohxYJOPJOZEQfZm4TVtfdtW~Q3UJYfgiQJwJm1OgGEbC0YUxPVMW65NUuT1kGQDKqN~G05TwH2E7L2~SIIMzAAy1cfL-IqbaCOrDY5dD5Gb8H5Mn1nwlpSVeR19XZpkt0Z73c8EsZ5Ml1N-bF73MtLYGSIenCvMxLLx7z9IeYyeim7L5FAZ034WCizfe17GABNuUtYzjxnXPIB7T6UR5nJNL9qE5x1YIPs9A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Western immunodetection of UreB in H. pylori strains UM032, 298 and 299." path-from-xml="dsw046f9.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61463075/dsw046f9.tif" path-from-xml="dsw046f9.tif" target="_blank">View large</a><a section="61463075" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw046/1/dsw046f9.png?Expires=2147483647&amp;Signature=APF7JwRnzUdKnleSUf3l9IGNPhYTWn3CXjxftzEyW7iykBEjB8Azxn7gfKI6aLLa74utQ3ZYy4mdbZgEMQOVMdSQeCLJojSsl5kIp2hK0Dme-bteyFCdrz69qmIWKp1-7n5bKppqAfZLb6UTfF8iUUTmeeUqfIPbJcfDVHO-OEKUoePlDbDPNu6TYO1lUgDY1o9AaQeDZrh6niUn2INiqpDqtF7eID9B1O1xaeZVJ5d9u96-lVAOx7q1TjARXxP5yX3UkxzGmegjU4CQJu6xh2f9akvfj006885XK3LUCnKEe57K9QWG0nBwZESQkEEAsAjmYkQOGb2P~QastFJkTg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=61463075&amp;ar=2670112&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw046f9.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Western immunodetection of UreB in <em>H. pylori</em> strains UM032, 298 and 299.</p></div></div></div><p>It is unclear what functional roles biotin synthase and the hypothetical protein play in <em>H. pylori</em> host colonisation. Recently, it was reported that enterohemorrhagic <em>Escherichia coli</em> (EHEC) is able to regulate its adherence to intestinal epithelial cells by sensing surrounding biotin level.<a reveal-id="dsw046-B50" data-open="dsw046-B50" class="link link-ref link-reveal xref-bibr"><sup>50</sup></a> Upon its arrival in the low-biotin large intestine, EHEC down-regulates the expression of its biotin protein ligase BirA to remove the repression on the global regulator Fur, thereby activating LEE (locus of enterocyte effacement) genes to promote bacterial adherence. In <em>H. pylori</em>, both BirA and Fur are also present (designated HP1140 and HP1027 in strain 26695, respectively).<a reveal-id="dsw046-B51" data-open="dsw046-B51" class="link link-ref link-reveal xref-bibr"><sup>51</sup></a> Nevertheless, it is uncertain in <em>H. pylori</em> if BirA may interact with Fur in a similar paradigm to that of <em>E. coli</em> as it lacks the N-terminal winged helix-turn-helix regulatory domain. It is tempting to hopothesise that the down-regulated expression of biotin synthase could possibly result in low intracellular biotin concentration, thus allowing derepression of Fur by BirA, which in turns activates genes involved in motility and chemotaxis to facilitate host colonisation.<a reveal-id="dsw046-B52" data-open="dsw046-B52" class="link link-ref link-reveal xref-bibr"><sup>52</sup></a> This warrants further investigation. The down-regulation of outer membrane protein genes, especially <em>babA</em>, however, is thought to increase bacterial dispersion and colonisation by preventing autoaggregation which might occur as a result of binding to LPS Le<sup>b</sup> antigen.</p>                            <h3 scrollto-destination=61463077 id="61463077" class="section-title" >3.7. Concluding remarks</h3>
<p>When <em>H. pylori</em> encounters a hostile foreign environment, the bacterium rapidly adapts to the new environment via a series of genomic alterations. Here, we demonstrated that a host change led to modification of the Lewis antigen profile of <em>H. pylori</em> lipopolysaccharide via acquisition of a <em>jhp0562</em>-like allele. In addition, expression levels of outer membrane proteins including BabA, BabB and HopD changed via altered homopolymeric tract lengths. These observations provide further evidence that rapid changes in membrane associated protein expressions play a major role in the early adaptation of bacterial populations to an individual host and these components are one of the key factors in <em>H. pylori’s</em> success as a pathogen.</p>                            <h2 scrollto-destination=61463079 id="61463079" class="backacknowledgements-title" >Acknowledgements</h2>
<p>This project was supported by the University of Malaya-Ministry of Education (UM-MoE) High Impact Research (HIR) grant (reference UM.C/625/1/HIR/MoE/CHAN13/3; Account No. H-50001-A000030), the National Health and Medical Research Council (grant no. 572723), the Vice Chancellor of the University of Western Australia, and the Western Australian Department of Commerce and Department of Health. SP was supported by SCELCE Microbiome Centre and grants from LKC School of Medicine, NTU University, Singapore. We thank Dr K. Mary Webberley for providing critical comments on this manuscript. We also thank Dr Hong Li (West China Hospital, Sichuan University) for providing blood group antigen-binding adhesin A (BabA) polyclonal antibody. We would also like to acknowledge Susana Wang, Primo Baybayan and Meredith Ashby of PacBio Biosciences (USA) and Siddarth Singh of PacBio Singapore for sequencing and assembling the original complete genomes of UM032, 298 and 299.</p>                            <h3 scrollto-destination=61463081 id="61463081" class="section-title" >Conflict of interest</h3>
<p>None declared.</p>                            <h3 scrollto-destination=61463083 id="61463083" class="section-title" >Supplementary data</h3>
<p><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw046/1/dsw046_Supp.zip?Expires=2147483647&Signature=s3q4bhSX2C3MUF-jr0f2UZ1jrVjkiGXzFiaUNbhztd4DChTCgvQajffmhInT-1Vj~0SRO~Kttfa4bqexSP8TrhsM2dD3tPD9oxAo17tPZ377FPaKsD8p~u48AhZfGosl2DgYDxY~wAxQIvmlENgGz~hpjXyN~I6CdeKNliwIOlCIEQtChC1zhSWLmnuwd57p2crX2HhvAlPdEQQh2bsMNnLu1COOQRPn604GZfBH8hQezXkfEFO3oCZfpwLaMYrQ~H4BcWpJOG4ygAaGy1rYakppJlB7TfE5VrMgGP-nBVCzXm0JcV4NeHqmjB9Dbt~-D52akH052YyUn9NKOPfy5A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary data</a></span> are available at <a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org" target="_blank">www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=61463085 id="61463085" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsw046-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw046-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Marshall</div> <div class="given-names">B.J.</div></div><div class="name string-name"><div class="surname">Warren</div> <div class="given-names">J.R.</div></div></div> <div class="year">1984</div>, 
<div class="article-title">Unidentified curved bacilli in the stomach of patients with gastritis and peptic ulceration</div>. <div class="source">Lancet</div> , <div class="volume">1</div>, <div class="fpage">1311</div>–<div class="lpage">15</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Unidentified%20curved%20bacilli%20in%20the%20stomach%20of%20patients%20with%20gastritis%20and%20peptic%20ulceration&amp;author=B.J.%20Marshall&amp;author=J.R.%20Warren&amp;publication_year=1984&amp;journal=Lancet&amp;volume=1&amp;pages=1311-15" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0140-6736(84)91816-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0140-6736(84)91816-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0140-6736(84)91816-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6145023" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw046-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw046-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Marshall</div> <div class="given-names">B.J.</div></div><div class="name string-name"><div class="surname">Windsor</div> <div class="given-names">H.M.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">The relation of <em>Helicobacter pylori</em> to gastric adenocarcinoma and lymphoma: pathophysiology, epidemiology, screening, clinical presentation, treatment, and prevention</div>. <div class="source">Med. Clin. North Am</div> ., <div class="volume">89</div>, <div class="fpage">313</div>–<div class="lpage">44</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20relation%20of%20Helicobacter%20pylori%20to%20gastric%20adenocarcinoma%20and%20lymphoma%3A%20pathophysiology%2C%20epidemiology%2C%20screening%2C%20clinical%20presentation%2C%20treatment%2C%20and%20prevention&amp;author=B.J.%20Marshall&amp;author=H.M.%20Windsor&amp;publication_year=2005&amp;journal=Med.%20Clin.%20North%20Am&amp;volume=89&amp;pages=313-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.mcna.2004.09.001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.mcna.2004.09.001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.mcna.2004.09.001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15656929" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw046-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw046-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="year">1994</div>, 
<div class="article-title">Schistosomes, liver flukes and <em>Helicobacter pylori</em>. IARC Working Group on the Evaluation of Carcinogenic Risks to Humans. Lyon, 7-14 June 1994</div>. <div class="source">IARC Monogr. Eval. Carcinog. Risks Hum</div> ., <div class="volume">61</div>, <div class="fpage">1</div>–<div class="lpage">241</div>.<div class="citation-links"><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7715068" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw046-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw046-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tay</div> <div class="given-names">C.Y.</div></div><div class="name string-name"><div class="surname">Mitchell</div> <div class="given-names">H.</div></div><div class="name string-name"><div class="surname">Dong</div> <div class="given-names">Q.</div></div><div class="name string-name"><div class="surname">Goh</div> <div class="given-names">K.L.</div></div><div class="name string-name"><div class="surname">Dawes</div> <div class="given-names">I.W.</div></div><div class="name string-name"><div class="surname">Lan</div> <div class="given-names">R.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Population structure of <em>Helicobacter pylori</em> among ethnic groups in Malaysia: recent acquisition of the bacterium by the Malay population</div>. <div class="source">BMC Microbiol</div> ., <div class="volume">9</div>, <div class="fpage">126</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Population%20structure%20of%20Helicobacter%20pylori%20among%20ethnic%20groups%20in%20Malaysia%3A%20recent%20acquisition%20of%20the%20bacterium%20by%20the%20Malay%20population&amp;author=C.Y.%20Tay&amp;author=H.%20Mitchell&amp;author=Q.%20Dong&amp;author=K.L.%20Goh&amp;author=I.W.%20Dawes&amp;author=R.%20Lan&amp;publication_year=2009&amp;journal=BMC%20Microbiol&amp;volume=9&amp;pages=126" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2180-9-126" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2180-9-126" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2180-9-126"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19538757" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw046-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw046-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tomb</div> <div class="given-names">J. F.</div></div><div class="name string-name"><div class="surname">White</div> <div class="given-names">O.</div></div><div class="name string-name"><div class="surname">Kerlavage</div> <div class="given-names">A.R.</div></div></div>, et al.  . <div class="year">1997</div>, 
<div class="article-title">The complete genome sequence of the gastric pathogen <em>Helicobacter pylori</em></div>. <div class="source">Nature</div> , <div class="volume">388</div>, <div class="fpage">539</div>–<div class="lpage">47</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20complete%20genome%20sequence%20of%20the%20gastric%20pathogen%20Helicobacter%20pylori&amp;author=J.%20F.%20Tomb&amp;author=O.%20White&amp;author=A.R.%20Kerlavage&amp;publication_year=1997&amp;journal=Nature&amp;volume=388&amp;pages=539-47" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/41483" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F41483" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F41483"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9252185" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw046-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw046-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Pinto</div> <div class="given-names">A. V.</div></div><div class="name string-name"><div class="surname">Mathieu</div> <div class="given-names">A.</div></div><div class="name string-name"><div class="surname">Marsin</div> <div class="given-names">S.</div></div></div>, et al.  . <div class="year">2005</div>, 
<div class="article-title">Suppression of homologous and homeologous recombination by the bacterial MutS2 protein</div>. <div class="source">Mol. Cell</div> , <div class="volume">17</div>, <div class="fpage">113</div>–<div class="lpage">120</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Suppression%20of%20homologous%20and%20homeologous%20recombination%20by%20the%20bacterial%20MutS2%20protein&amp;author=A.%20V.%20Pinto&amp;author=A.%20Mathieu&amp;author=S.%20Marsin&amp;publication_year=2005&amp;journal=Mol.%20Cell&amp;volume=17&amp;pages=113-120" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.molcel.2004.11.035" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.molcel.2004.11.035" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.molcel.2004.11.035"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15629722" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw046-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw046-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kang</div> <div class="given-names">J.</div></div><div class="name string-name"><div class="surname">Huang</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Blaser</div> <div class="given-names">M. J.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">Structural and functional divergence of MutS2 from bacterial MutS1 and eukaryotic MSH4-MSH5 homologs</div>. <div class="source">J. Bacteriol</div> ., <div class="volume">187</div>, <div class="fpage">3528</div>–<div class="lpage">37</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structural%20and%20functional%20divergence%20of%20MutS2%20from%20bacterial%20MutS1%20and%20eukaryotic%20MSH4-MSH5%20homologs&amp;author=J.%20Kang&amp;author=S.%20Huang&amp;author=M.%20J.%20Blaser&amp;publication_year=2005&amp;journal=J.%20Bacteriol&amp;volume=187&amp;pages=3528-37" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.187.10.3528-3537.2005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.187.10.3528-3537.2005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.187.10.3528-3537.2005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15866941" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw046-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw046-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Benghezal</div> <div class="given-names">M.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Persistence of <em>Helicobacter pylori</em> Infection: Genetic and Epigenetic Diversity</div>. <div class="source">InTech</div> .</div></div></div></div><div content-id="dsw046-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw046-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Garcia-Ortiz</div> <div class="given-names">M.V.</div></div><div class="name string-name"><div class="surname">Marsin</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Arana</div> <div class="given-names">M.E.</div></div></div>, et al.  . <div class="year">2011</div>, 
<div class="article-title">Unexpected role for <em>Helicobacter pylori</em> DNA polymerase I as a source of genetic variability</div>. <div class="source">PLoS Genet</div> ., <div class="volume">7</div>, <div class="fpage">e1002152</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Unexpected%20role%20for%20Helicobacter%20pylori%20DNA%20polymerase%20I%20as%20a%20source%20of%20genetic%20variability&amp;author=M.V.%20Garcia-Ortiz&amp;author=S.%20Marsin&amp;author=M.E.%20Arana&amp;publication_year=2011&amp;journal=PLoS%20Genet&amp;volume=7&amp;pages=e1002152" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1002152" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1002152" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1002152"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21731507" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw046-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw046-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">van der Woude</div> <div class="given-names">M.W.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Phase variation: how to create and coordinate population diversity</div>. <div class="source">Curr. Opin. Microbiol</div> ., <div class="volume">14</div>, <div class="fpage">205</div>–<div class="lpage">211</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phase%20variation%3A%20how%20to%20create%20and%20coordinate%20population%20diversity&amp;author=M.W.%20van%20der%20Woude&amp;publication_year=2011&amp;journal=Curr.%20Opin.%20Microbiol&amp;volume=14&amp;pages=205-211" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.mib.2011.01.002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.mib.2011.01.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.mib.2011.01.002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21292543" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw046-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw046-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Pernitzsch</div> <div class="given-names">S.R.</div></div><div class="name string-name"><div class="surname">Tirier</div> <div class="given-names">S.M.</div></div><div class="name string-name"><div class="surname">Beier</div> <div class="given-names">D.</div></div><div class="name string-name"><div class="surname">Sharma</div> <div class="given-names">C.M.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">A variable homopolymeric G-repeat defines small RNA-mediated posttranscriptional regulation of a chemotaxis receptor in <em>Helicobacter pylori</em></div>. <div class="source">Proc Natl. Acad. Sci. USA</div> , <div class="volume">111</div>, <div class="fpage">E501</div>–<div class="lpage">10</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20variable%20homopolymeric%20G-repeat%20defines%20small%20RNA-mediated%20posttranscriptional%20regulation%20of%20a%20chemotaxis%20receptor%20in%20Helicobacter%20pylori&amp;author=S.R.%20Pernitzsch&amp;author=S.M.%20Tirier&amp;author=D.%20Beier&amp;author=C.M.%20Sharma&amp;publication_year=2014&amp;journal=Proc%20Natl.%20Acad.%20Sci.%20USA&amp;volume=111&amp;pages=E501-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1315152111" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1315152111" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1315152111"> </span></div> </div></div></div></div></div><div content-id="dsw046-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw046-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Appelmelk</div> <div class="given-names">B. J.</div></div><div class="name string-name"><div class="surname">Martin</div> <div class="given-names">S. L.</div></div><div class="name string-name"><div class="surname">Monteiro</div> <div class="given-names">M. A.</div></div></div>, et al.  . <div class="year">1999</div>, 
<div class="article-title">Phase variation in <em>Helicobacter pylori</em> lipopolysaccharide due to changes in the lengths of poly(C) tracts in alpha3-fucosyltransferase genes</div>. <div class="source">Infect. Immun</div> ., <div class="volume">67</div>, <div class="fpage">5361</div>–<div class="lpage">66</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phase%20variation%20in%20Helicobacter%20pylori%20lipopolysaccharide%20due%20to%20changes%20in%20the%20lengths%20of%20poly(C)%20tracts%20in%20alpha3-fucosyltransferase%20genes&amp;author=B.%20J.%20Appelmelk&amp;author=S.%20L.%20Martin&amp;author=M.%20A.%20Monteiro&amp;publication_year=1999&amp;journal=Infect.%20Immun&amp;volume=67&amp;pages=5361-66" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10496917" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw046-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw046-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Aberg</div> <div class="given-names">A.</div></div><div class="name string-name"><div class="surname">Gideonsson</div> <div class="given-names">P.</div></div><div class="name string-name"><div class="surname">Vallstrom</div> <div class="given-names">A.</div></div></div>, et al.  . <div class="year">2014</div>, 
<div class="article-title">A repetitive DNA element regulates expression of the <em>Helicobacter pylori</em> sialic acid binding adhesin by a rheostat-like mechanism</div>. <div class="source">PLoS Pathog</div> ., <div class="volume">10</div>, <div class="fpage">e1004234</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20repetitive%20DNA%20element%20regulates%20expression%20of%20the%20Helicobacter%20pylori%20sialic%20acid%20binding%20adhesin%20by%20a%20rheostat-like%20mechanism&amp;author=A.%20Aberg&amp;author=P.%20Gideonsson&amp;author=A.%20Vallstrom&amp;publication_year=2014&amp;journal=PLoS%20Pathog&amp;volume=10&amp;pages=e1004234" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.ppat.1004234" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.ppat.1004234" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.ppat.1004234"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24991812" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw046-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw046-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Dubois</div> <div class="given-names">A.</div></div><div class="name string-name"><div class="surname">Berg</div> <div class="given-names">D.E.</div></div><div class="name string-name"><div class="surname">Incecik</div> <div class="given-names">E.T.</div></div></div>, et al.  . <div class="year">1999</div>, 
<div class="article-title">Host specificity of <em>Helicobacter pylori</em> strains and host responses in experimentally challenged nonhuman primates</div>. <div class="source">Gastroenterology</div> , <div class="volume">116</div>, <div class="fpage">90</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Host%20specificity%20of%20Helicobacter%20pylori%20strains%20and%20host%20responses%20in%20experimentally%20challenged%20nonhuman%20primates&amp;author=A.%20Dubois&amp;author=D.E.%20Berg&amp;author=E.T.%20Incecik&amp;publication_year=1999&amp;journal=Gastroenterology&amp;volume=116&amp;pages=90-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0016-5085(99)70232-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0016-5085(99)70232-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0016-5085(99)70232-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9869606" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw046-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw046-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Day</div> <div class="given-names">A.S.</div></div><div class="name string-name"><div class="surname">Jones</div> <div class="given-names">N.L.</div></div><div class="name string-name"><div class="surname">Policova</div> <div class="given-names">Z.</div></div></div>, et al.  . <div class="year">2001</div>, 
<div class="article-title">Characterization of virulence factors of mouse-adapted <em>Helicobacter pylori</em> strain SS1 and effects on gastric hydrophobicity</div>. <div class="source">Digest Dis. Sci</div> ., <div class="volume">46</div>, <div class="fpage">1943</div>–<div class="lpage">51</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20virulence%20factors%20of%20mouse-adapted%20Helicobacter%20pylori%20strain%20SS1%20and%20effects%20on%20gastric%20hydrophobicity&amp;author=A.S.%20Day&amp;author=N.L.%20Jones&amp;author=Z.%20Policova&amp;publication_year=2001&amp;journal=Digest%20Dis.%20Sci&amp;volume=46&amp;pages=1943-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1023/A:1010691216207" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1023%2FA:1010691216207" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1023%2FA:1010691216207"> </span></div> </div></div></div></div></div><div content-id="dsw046-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw046-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wirth</div> <div class="given-names">H.P.</div></div><div class="name string-name"><div class="surname">Beins</div> <div class="given-names">M.H.</div></div><div class="name string-name"><div class="surname">Yang</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Tham</div> <div class="given-names">K.T.</div></div><div class="name string-name"><div class="surname">Blaser</div> <div class="given-names">M.J.</div></div></div> <div class="year">1998</div>, 
<div class="article-title">Experimental infection of Mongolian gerbils with wild-type and mutant <em>Helicobacter pylori</em> strains</div>. <div class="source">Infect. Immun</div> ., <div class="volume">66</div>, <div class="fpage">4856</div>–<div class="lpage">66</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Experimental%20infection%20of%20Mongolian%20gerbils%20with%20wild-type%20and%20mutant%20Helicobacter%20pylori%20strains&amp;author=H.P.%20Wirth&amp;author=M.H.%20Beins&amp;author=M.%20Yang&amp;author=K.T.%20Tham&amp;author=M.J.%20Blaser&amp;publication_year=1998&amp;journal=Infect.%20Immun&amp;volume=66&amp;pages=4856-66" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9746590" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw046-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw046-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Khosravi</div> <div class="given-names">Y.</div></div><div class="name string-name"><div class="surname">Rehvathy</div> <div class="given-names">V.</div></div><div class="name string-name"><div class="surname">Wee</div> <div class="given-names">W.Y.</div></div></div>, et al.  . <div class="year">2013</div>, 
<div class="article-title">Comparing the genomes of <em>Helicobacter pylori</em> clinical strain UM032 and Mice-adapted derivatives</div>. <div class="source">Gut Pathog</div> ., <div class="volume">5</div>, <div class="fpage">25</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparing%20the%20genomes%20of%20Helicobacter%20pylori%20clinical%20strain%20UM032%20and%20Mice-adapted%20derivatives&amp;author=Y.%20Khosravi&amp;author=V.%20Rehvathy&amp;author=W.Y.%20Wee&amp;publication_year=2013&amp;journal=Gut%20Pathog&amp;volume=5&amp;pages=25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1757-4749-5-25" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1757-4749-5-25" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1757-4749-5-25"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23957912" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw046-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw046-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chin</div> <div class="given-names">C. S.</div></div><div class="name string-name"><div class="surname">Sorenson</div> <div class="given-names">J.</div></div><div class="name string-name"><div class="surname">Harris</div> <div class="given-names">J.B.</div></div></div>, et al.  . <div class="year">2011</div>, 
<div class="article-title">The origin of the Haitian cholera outbreak strain</div>. <div class="source">New Engl. J. Med</div> ., <div class="volume">364</div>, <div class="fpage">33</div>–<div class="lpage">42</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20origin%20of%20the%20Haitian%20cholera%20outbreak%20strain&amp;author=C.%20S.%20Chin&amp;author=J.%20Sorenson&amp;author=J.B.%20Harris&amp;publication_year=2011&amp;journal=New%20Engl.%20J.%20Med&amp;volume=364&amp;pages=33-42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1056/NEJMoa1012928" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1056%2FNEJMoa1012928" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1056%2FNEJMoa1012928"> </span></div> </div></div></div></div></div><div content-id="dsw046-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw046-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Perkins</div> <div class="given-names">T.T.</div></div><div class="name string-name"><div class="surname">Tay</div> <div class="given-names">C.Y.</div></div><div class="name string-name"><div class="surname">Thirriot</div> <div class="given-names">F.</div></div><div class="name string-name"><div class="surname">Marshall</div> <div class="given-names">B.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Choosing a benchtop sequencing machine to characterise <em>Helicobacter pylori</em> genomes</div>. <div class="source">PLoS One</div> , <div class="volume">8</div>, <div class="fpage">e67539</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Choosing%20a%20benchtop%20sequencing%20machine%20to%20characterise%20Helicobacter%20pylori%20genomes&amp;author=T.T.%20Perkins&amp;author=C.Y.%20Tay&amp;author=F.%20Thirriot&amp;author=B.%20Marshall&amp;publication_year=2013&amp;journal=PLoS%20One&amp;volume=8&amp;pages=e67539" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0067539" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0067539" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0067539"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23840736" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw046-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw046-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kearse</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Moir</div> <div class="given-names">R.</div></div><div class="name string-name"><div class="surname">Wilson</div> <div class="given-names">A.</div></div></div>, et al.  . <div class="year">2012</div>, 
<div class="article-title">Geneious Basic: an integrated and extendable desktop software platform for the organization and analysis of sequence data</div>. <div class="source">Bioinformatics</div> , <div class="volume">28</div>, <div class="fpage">1647</div>-<div class="lpage">1649</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Geneious%20Basic%3A%20an%20integrated%20and%20extendable%20desktop%20software%20platform%20for%20the%20organization%20and%20analysis%20of%20sequence%20data&amp;author=M.%20Kearse&amp;author=R.%20Moir&amp;author=A.%20Wilson&amp;publication_year=2012&amp;journal=Bioinformatics&amp;volume=28&amp;pages=1647-1649" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bts199" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbts199" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbts199"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22543367" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw046-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw046-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Aziz</div> <div class="given-names">R.K.</div></div><div class="name string-name"><div class="surname">Bartels</div> <div class="given-names">D.</div></div><div class="name string-name"><div class="surname">Best</div> <div class="given-names">A.A.</div></div></div>, et al.  . <div class="year">2008</div>, 
<div class="article-title">The RAST Server: rapid annotations using subsystems technology</div>. <div class="source">BMC Genomics</div> , <div class="volume">9</div>, <div class="fpage">75</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20RAST%20Server%3A%20rapid%20annotations%20using%20subsystems%20technology&amp;author=R.K.%20Aziz&amp;author=D.%20Bartels&amp;author=A.A.%20Best&amp;publication_year=2008&amp;journal=BMC%20Genomics&amp;volume=9&amp;pages=75" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-9-75" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-9-75" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-9-75"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18261238" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw046-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw046-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Besemer</div> <div class="given-names">J.</div></div><div class="name string-name"><div class="surname">Lomsadze</div> <div class="given-names">A.</div></div><div class="name string-name"><div class="surname">Borodovsky</div> <div class="given-names">M.</div></div></div> <div class="year">2001</div>, 
<div class="article-title">GeneMarkS: a self-training method for prediction of gene starts in microbial genomes. Implications for finding sequence motifs in regulatory regions</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">29</div>, <div class="fpage">2607</div>–<div class="lpage">18</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GeneMarkS%3A%20a%20self-training%20method%20for%20prediction%20of%20gene%20starts%20in%20microbial%20genomes.%20Implications%20for%20finding%20sequence%20motifs%20in%20regulatory%20regions&amp;author=J.%20Besemer&amp;author=A.%20Lomsadze&amp;author=M.%20Borodovsky&amp;publication_year=2001&amp;journal=Nucleic%20Acids%20Res&amp;volume=29&amp;pages=2607-18" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/29.12.2607" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F29.12.2607" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F29.12.2607"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11410670" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw046-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw046-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hyatt</div> <div class="given-names">D.</div></div><div class="name string-name"><div class="surname">Chen</div> <div class="given-names">G.L.</div></div><div class="name string-name"><div class="surname">Locascio</div> <div class="given-names">P.F.</div></div><div class="name string-name"><div class="surname">Land</div> <div class="given-names">M.L.</div></div><div class="name string-name"><div class="surname">Larimer</div> <div class="given-names">F.W.</div></div><div class="name string-name"><div class="surname">Hauser</div> <div class="given-names">L.J.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Prodigal: prokaryotic gene recognition and translation initiation site identification</div>. <div class="source">BMC Bioinformatics</div> , <div class="volume">11</div>, <div class="fpage">119</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Prodigal%3A%20prokaryotic%20gene%20recognition%20and%20translation%20initiation%20site%20identification&amp;author=D.%20Hyatt&amp;author=G.L.%20Chen&amp;author=P.F.%20Locascio&amp;author=M.L.%20Land&amp;author=F.W.%20Larimer&amp;author=L.J.%20Hauser&amp;publication_year=2010&amp;journal=BMC%20Bioinformatics&amp;volume=11&amp;pages=119" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-11-119" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-11-119" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-11-119"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20211023" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw046-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw046-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Delcher</div> <div class="given-names">A.L.</div></div><div class="name string-name"><div class="surname">Bratke</div> <div class="given-names">K.A.</div></div><div class="name string-name"><div class="surname">Powers</div> <div class="given-names">E.C.</div></div><div class="name string-name"><div class="surname">Salzberg</div> <div class="given-names">S.L.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">Identifying bacterial genes and endosymbiont DNA with Glimmer</div>. <div class="source">Bioinformatics</div> , <div class="volume">23</div>, <div class="fpage">673</div>–<div class="lpage">79</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identifying%20bacterial%20genes%20and%20endosymbiont%20DNA%20with%20Glimmer&amp;author=A.L.%20Delcher&amp;author=K.A.%20Bratke&amp;author=E.C.%20Powers&amp;author=S.L.%20Salzberg&amp;publication_year=2007&amp;journal=Bioinformatics&amp;volume=23&amp;pages=673-79" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btm009" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtm009" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtm009"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17237039" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw046-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw046-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Darling</div> <div class="given-names">A.C.</div></div><div class="name string-name"><div class="surname">Mau</div> <div class="given-names">B.</div></div><div class="name string-name"><div class="surname">Blattner</div> <div class="given-names">F.R.</div></div><div class="name string-name"><div class="surname">Perna</div> <div class="given-names">N.T.</div></div></div> <div class="year">2004</div>, 
<div class="article-title">Mauve: multiple alignment of conserved genomic sequence with rearrangements</div>. <div class="source">Genome Res</div> ., <div class="volume">14</div>, <div class="fpage">1394</div>–<div class="lpage">403</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mauve%3A%20multiple%20alignment%20of%20conserved%20genomic%20sequence%20with%20rearrangements&amp;author=A.C.%20Darling&amp;author=B.%20Mau&amp;author=F.R.%20Blattner&amp;author=N.T.%20Perna&amp;publication_year=2004&amp;journal=Genome%20Res&amp;volume=14&amp;pages=1394-403" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.2289704" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.2289704" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.2289704"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15231754" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw046-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw046-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Marchler-Bauer</div> <div class="given-names">A.</div></div><div class="name string-name"><div class="surname">Derbyshire</div> <div class="given-names">M.K.</div></div><div class="name string-name"><div class="surname">Gonzales</div> <div class="given-names">N.R.</div></div></div>, et al.  . <div class="year">2015</div>, 
<div class="article-title">CDD: NCBI's conserved domain database</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">43</div>, <div class="fpage">D222</div>-<div class="lpage">226</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CDD%3A%20NCBI's%20conserved%20domain%20database&amp;author=A.%20Marchler-Bauer&amp;author=M.K.%20Derbyshire&amp;author=N.R.%20Gonzales&amp;publication_year=2015&amp;journal=Nucleic%20Acids%20Res&amp;volume=43&amp;pages=D222-226" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gku1221" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgku1221" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgku1221"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25414356" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw046-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw046-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Livak</div> <div class="given-names">K.J.</div></div><div class="name string-name"><div class="surname">Schmittgen</div> <div class="given-names">T.D.</div></div></div> <div class="year">2001</div>, 
<div class="article-title">Analysis of relative gene expression data using real-time quantitative PCR and the 2(-Delta Delta C(T)) Method</div>. <div class="source">Methods</div> , <div class="volume">25</div>, <div class="fpage">402</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20relative%20gene%20expression%20data%20using%20real-time%20quantitative%20PCR%20and%20the%202(-Delta%20Delta%20C(T))%20Method&amp;author=K.J.%20Livak&amp;author=T.D.%20Schmittgen&amp;publication_year=2001&amp;journal=Methods&amp;volume=25&amp;pages=402-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/meth.2001.1262" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fmeth.2001.1262" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fmeth.2001.1262"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11846609" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw046-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw046-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Linz</div> <div class="given-names">B.</div></div><div class="name string-name"><div class="surname">Windsor</div> <div class="given-names">H.M.</div></div><div class="name string-name"><div class="surname">McGraw</div> <div class="given-names">J.J.</div></div></div>, et al.  . <div class="year">2014</div>, 
<div class="article-title">A mutation burst during the acute phase of <em>Helicobacter pylori</em> infection in humans and rhesus macaques</div>. <div class="source">Nat. Commun</div> ., <div class="volume">5</div>, <div class="fpage">4165</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20mutation%20burst%20during%20the%20acute%20phase%20of%20Helicobacter%20pylori%20infection%20in%20humans%20and%20rhesus%20macaques&amp;author=B.%20Linz&amp;author=H.M.%20Windsor&amp;author=J.J.%20McGraw&amp;publication_year=2014&amp;journal=Nat.%20Commun&amp;volume=5&amp;pages=4165" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ncomms5165" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fncomms5165" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fncomms5165"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24924186" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw046-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw046-B29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yoshida</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Wakatsuki</div> <div class="given-names">Y.</div></div><div class="name string-name"><div class="surname">Kobayashi</div> <div class="given-names">Y.</div></div></div>, et al.  . <div class="year">1999</div>, 
<div class="article-title">Cloning and characterization of a novel membrane-associated antigenic protein of <em>Helicobacter pylori</em></div>. <div class="source">Infect. Immun</div> ., <div class="volume">67</div>, <div class="fpage">286</div>–<div class="lpage">93</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cloning%20and%20characterization%20of%20a%20novel%20membrane-associated%20antigenic%20protein%20of%20Helicobacter%20pylori&amp;author=M.%20Yoshida&amp;author=Y.%20Wakatsuki&amp;author=Y.%20Kobayashi&amp;publication_year=1999&amp;journal=Infect.%20Immun&amp;volume=67&amp;pages=286-93" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9864228" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw046-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw046-B30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Suganuma</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Kurusu</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Okabe</div> <div class="given-names">S.</div></div></div>, et al.  . <div class="year">2001</div>, 
<div class="article-title"><em>Helicobacter pylori</em> membrane protein 1: a new carcinogenic factor of <em>Helicobacter pylori</em></div>. <div class="source">Cancer Res</div> ., <div class="volume">61</div>, <div class="fpage">6356</div>–<div class="lpage">59</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Helicobacter%20pylori%20membrane%20protein%201%3A%20a%20new%20carcinogenic%20factor%20of%20Helicobacter%20pylori&amp;author=M.%20Suganuma&amp;author=M.%20Kurusu&amp;author=S.%20Okabe&amp;publication_year=2001&amp;journal=Cancer%20Res&amp;volume=61&amp;pages=6356-59" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11522625" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw046-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw046-B31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Alm</div> <div class="given-names">R.A.</div></div><div class="name string-name"><div class="surname">Bina</div> <div class="given-names">J.</div></div><div class="name string-name"><div class="surname">Andrews</div> <div class="given-names">B.M.</div></div><div class="name string-name"><div class="surname">Doig</div> <div class="given-names">P.</div></div><div class="name string-name"><div class="surname">Hancock</div> <div class="given-names">R.E.</div></div><div class="name string-name"><div class="surname">Trust</div> <div class="given-names">T.J.</div></div></div> <div class="year">2000</div>, 
<div class="article-title">Comparative genomics of <em>Helicobacter pylori</em>: analysis of the outer membrane protein families</div>. <div class="source">Infect. Immun</div> ., <div class="volume">68</div>, <div class="fpage">4155</div>–<div class="lpage">68</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20genomics%20of%20Helicobacter%20pylori%3A%20analysis%20of%20the%20outer%20membrane%20protein%20families&amp;author=R.A.%20Alm&amp;author=J.%20Bina&amp;author=B.M.%20Andrews&amp;author=P.%20Doig&amp;author=R.E.%20Hancock&amp;author=T.J.%20Trust&amp;publication_year=2000&amp;journal=Infect.%20Immun&amp;volume=68&amp;pages=4155-68" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/IAI.68.7.4155-4168.2000" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FIAI.68.7.4155-4168.2000" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FIAI.68.7.4155-4168.2000"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10858232" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw046-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw046-B32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Styer</div> <div class="given-names">C.M.</div></div><div class="name string-name"><div class="surname">Hansen</div> <div class="given-names">L.M.</div></div><div class="name string-name"><div class="surname">Cooke</div> <div class="given-names">C.L.</div></div></div>, et al.  . <div class="year">2010</div>, 
<div class="article-title">Expression of the BabA adhesin during experimental infection with <em>Helicobacter pylori</em></div>. <div class="source">Infect. Immun</div> ., <div class="volume">78</div>, <div class="fpage">1593</div>–<div class="lpage">600</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Expression%20of%20the%20BabA%20adhesin%20during%20experimental%20infection%20with%20Helicobacter%20pylori&amp;author=C.M.%20Styer&amp;author=L.M.%20Hansen&amp;author=C.L.%20Cooke&amp;publication_year=2010&amp;journal=Infect.%20Immun&amp;volume=78&amp;pages=1593-600" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/IAI.01297-09" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FIAI.01297-09" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FIAI.01297-09"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20123715" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw046-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw046-B33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Backstrom</div> <div class="given-names">A.</div></div><div class="name string-name"><div class="surname">Lundberg</div> <div class="given-names">C.</div></div><div class="name string-name"><div class="surname">Kersulyte</div> <div class="given-names">D.</div></div><div class="name string-name"><div class="surname">Berg</div> <div class="given-names">D.E.</div></div><div class="name string-name"><div class="surname">Boren</div> <div class="given-names">T.</div></div><div class="name string-name"><div class="surname">Arnqvist</div> <div class="given-names">A.</div></div></div> <div class="year">2004</div>, 
<div class="article-title">Metastability of <em>Helicobacter pylori</em> bab adhesin genes and dynamics in Lewis b antigen binding</div>. <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="volume">101</div>, <div class="fpage">16923</div>–<div class="lpage">928</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Metastability%20of%20Helicobacter%20pylori%20bab%20adhesin%20genes%20and%20dynamics%20in%20Lewis%20b%20antigen%20binding&amp;author=A.%20Backstrom&amp;author=C.%20Lundberg&amp;author=D.%20Kersulyte&amp;author=D.E.%20Berg&amp;author=T.%20Boren&amp;author=A.%20Arnqvist&amp;publication_year=2004&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=101&amp;pages=16923-928" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0404817101" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0404817101" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0404817101"> </span></div> </div></div></div></div></div><div content-id="dsw046-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw046-B34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Nell</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Kennemann</div> <div class="given-names">L.</div></div><div class="name string-name"><div class="surname">Schwarz</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Josenhans</div> <div class="given-names">C.</div></div><div class="name string-name"><div class="surname">Suerbaum</div> <div class="given-names">S.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Dynamics of Lewis b binding and sequence variation of the babA adhesin gene during chronic <em>Helicobacter pylori</em> infection in humans</div>. <div class="source">MBio</div> , <div class="volume">5</div>.</div></div></div></div><div content-id="dsw046-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw046-B35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Colbeck</div> <div class="given-names">J.C.</div></div><div class="name string-name"><div class="surname">Hansen</div> <div class="given-names">L.M.</div></div><div class="name string-name"><div class="surname">Fong</div> <div class="given-names">J.M.</div></div><div class="name string-name"><div class="surname">Solnick</div> <div class="given-names">J.V.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">Genotypic profile of the outer membrane proteins BabA and BabB in clinical isolates of <em>Helicobacter pylori</em></div>. <div class="source">Infect. Immun</div> ., <div class="volume">74</div>, <div class="fpage">4375</div>–<div class="lpage">78</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genotypic%20profile%20of%20the%20outer%20membrane%20proteins%20BabA%20and%20BabB%20in%20clinical%20isolates%20of%20Helicobacter%20pylori&amp;author=J.C.%20Colbeck&amp;author=L.M.%20Hansen&amp;author=J.M.%20Fong&amp;author=J.V.%20Solnick&amp;publication_year=2006&amp;journal=Infect.%20Immun&amp;volume=74&amp;pages=4375-78" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/IAI.00485-06" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FIAI.00485-06" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FIAI.00485-06"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16790815" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw046-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw046-B36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Matteo</div> <div class="given-names">M.J.</div></div><div class="name string-name"><div class="surname">Armitano</div> <div class="given-names">R.I.</div></div><div class="name string-name"><div class="surname">Romeo</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Wonaga</div> <div class="given-names">A.</div></div><div class="name string-name"><div class="surname">Olmos</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Catalano</div> <div class="given-names">M.</div></div></div> <div class="year">2011</div>, 
<div class="article-title"><em>Helicobacter pylori bab</em> genes during chronic colonization</div>. <div class="source">Int. J. Mol. Epidemiol. Genet</div> ., <div class="volume">2</div>, <div class="fpage">286</div>–<div class="lpage">91</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Helicobacter%20pylori%20bab%20genes%20during%20chronic%20colonization&amp;author=M.J.%20Matteo&amp;author=R.I.%20Armitano&amp;author=M.%20Romeo&amp;author=A.%20Wonaga&amp;author=M.%20Olmos&amp;author=M.%20Catalano&amp;publication_year=2011&amp;journal=Int.%20J.%20Mol.%20Epidemiol.%20Genet&amp;volume=2&amp;pages=286-91" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21915366" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw046-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw046-B37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ilver</div> <div class="given-names">D.</div></div><div class="name string-name"><div class="surname">Arnqvist</div> <div class="given-names">A.</div></div><div class="name string-name"><div class="surname">Ogren</div> <div class="given-names">J.</div></div></div>, et al.  . <div class="year">1998</div>, 
<div class="article-title"><em>Helicobacter pylori</em> adhesin binding fucosylated histo-blood group antigens revealed by retagging</div>. <div class="source">Science</div> , <div class="volume">279</div>, <div class="fpage">373</div>–<div class="lpage">77</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Helicobacter%20pylori%20adhesin%20binding%20fucosylated%20histo-blood%20group%20antigens%20revealed%20by%20retagging&amp;author=D.%20Ilver&amp;author=A.%20Arnqvist&amp;author=J.%20Ogren&amp;publication_year=1998&amp;journal=Science&amp;volume=279&amp;pages=373-77" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.279.5349.373" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.279.5349.373" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.279.5349.373"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9430586" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw046-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw046-B38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kawai</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Furuta</div> <div class="given-names">Y.</div></div><div class="name string-name"><div class="surname">Yahara</div> <div class="given-names">K.</div></div></div>, et al.  . <div class="year">2011</div>, 
<div class="article-title">Evolution in an oncogenic bacterial species with extreme genome plasticity: <em>Helicobacter pylori</em> East Asian genomes</div>. <div class="source">BMC Microbiol</div> ., <div class="volume">11</div>, <div class="fpage">104</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20in%20an%20oncogenic%20bacterial%20species%20with%20extreme%20genome%20plasticity%3A%20Helicobacter%20pylori%20East%20Asian%20genomes&amp;author=M.%20Kawai&amp;author=Y.%20Furuta&amp;author=K.%20Yahara&amp;publication_year=2011&amp;journal=BMC%20Microbiol&amp;volume=11&amp;pages=104" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2180-11-104" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2180-11-104" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2180-11-104"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21575176" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw046-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw046-B39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Rice</div> <div class="given-names">P.</div></div><div class="name string-name"><div class="surname">Longden</div> <div class="given-names">I.</div></div><div class="name string-name"><div class="surname">Bleasby</div> <div class="given-names">A.</div></div></div> <div class="year">2000</div>, 
<div class="article-title">EMBOSS: the European Molecular Biology Open Software Suite</div>. <div class="source">Trends Genet</div> ., <div class="volume">16</div>, <div class="fpage">276</div>–<div class="lpage">77</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=EMBOSS%3A%20the%20European%20Molecular%20Biology%20Open%20Software%20Suite&amp;author=P.%20Rice&amp;author=I.%20Longden&amp;author=A.%20Bleasby&amp;publication_year=2000&amp;journal=Trends%20Genet&amp;volume=16&amp;pages=276-77" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0168-9525(00)02024-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0168-9525(00)02024-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0168-9525(00)02024-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10827456" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw046-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw046-B40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ma</div> <div class="given-names">B.</div></div><div class="name string-name"><div class="surname">Wang</div> <div class="given-names">G.</div></div><div class="name string-name"><div class="surname">Palcic</div> <div class="given-names">M.M.</div></div><div class="name string-name"><div class="surname">Hazes</div> <div class="given-names">B.</div></div><div class="name string-name"><div class="surname">Taylor</div> <div class="given-names">D.E.</div></div></div> <div class="year">2003</div>, 
<div class="article-title">C-terminal amino acids of <em>Helicobacter pylori</em> alpha1,3/4 fucosyltransferases determine type I and type II transfer</div>. <div class="source">J. Biol. Chem</div> ., <div class="volume">278</div>, <div class="fpage">21893</div>–<div class="lpage">900</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=C-terminal%20amino%20acids%20of%20Helicobacter%20pylori%20alpha1%2C3%2F4%20fucosyltransferases%20determine%20type%20I%20and%20type%20II%20transfer&amp;author=B.%20Ma&amp;author=G.%20Wang&amp;author=M.M.%20Palcic&amp;author=B.%20Hazes&amp;author=D.E.%20Taylor&amp;publication_year=2003&amp;journal=J.%20Biol.%20Chem&amp;volume=278&amp;pages=21893-900" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.M301704200" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.M301704200" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.M301704200"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12676935" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw046-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw046-B41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ge</div> <div class="given-names">Z.</div></div><div class="name string-name"><div class="surname">Chan</div> <div class="given-names">N.W.</div></div><div class="name string-name"><div class="surname">Palcic</div> <div class="given-names">M.M.</div></div><div class="name string-name"><div class="surname">Taylor</div> <div class="given-names">D.E.</div></div></div> <div class="year">1997</div>, 
<div class="article-title">Cloning and heterologous expression of an alpha1,3-fucosyltransferase gene from the gastric pathogen <em>Helicobacter pylori</em></div>. <div class="source">J. Biol. Chem</div> ., <div class="volume">272</div>, <div class="fpage">21357</div>–<div class="lpage">363</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cloning%20and%20heterologous%20expression%20of%20an%20alpha1%2C3-fucosyltransferase%20gene%20from%20the%20gastric%20pathogen%20Helicobacter%20pylori&amp;author=Z.%20Ge&amp;author=N.W.%20Chan&amp;author=M.M.%20Palcic&amp;author=D.E.%20Taylor&amp;publication_year=1997&amp;journal=J.%20Biol.%20Chem&amp;volume=272&amp;pages=21357-363" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.272.34.21357" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.272.34.21357" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.272.34.21357"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9261149" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw046-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw046-B42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Rabbani</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Miksa</div> <div class="given-names">V.</div></div><div class="name string-name"><div class="surname">Wipf</div> <div class="given-names">B.</div></div><div class="name string-name"><div class="surname">Ernst</div> <div class="given-names">B.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">Molecular cloning and functional expression of a novel <em>Helicobacter pylori</em> alpha-1,4 fucosyltransferase</div>. <div class="source">Glycobiology</div> , <div class="volume">15</div>, <div class="fpage">1076</div>–<div class="lpage">83</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20cloning%20and%20functional%20expression%20of%20a%20novel%20Helicobacter%20pylori%20alpha-1%2C4%20fucosyltransferase&amp;author=S.%20Rabbani&amp;author=V.%20Miksa&amp;author=B.%20Wipf&amp;author=B.%20Ernst&amp;publication_year=2005&amp;journal=Glycobiology&amp;volume=15&amp;pages=1076-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/glycob/cwj004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fglycob%2Fcwj004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fglycob%2Fcwj004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16000696" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw046-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw046-B43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wang</div> <div class="given-names">G.</div></div><div class="name string-name"><div class="surname">Boulton</div> <div class="given-names">P.G.</div></div><div class="name string-name"><div class="surname">Chan</div> <div class="given-names">N.W.</div></div><div class="name string-name"><div class="surname">Palcic</div> <div class="given-names">M.M.</div></div><div class="name string-name"><div class="surname">Taylor</div> <div class="given-names">D.E.</div></div></div> <div class="year">1999</div>, 
<div class="article-title">Novel <em>Helicobacter pylori</em> alpha1,2-fucosyltransferase, a key enzyme in the synthesis of Lewis antigens</div>. <div class="source">Microbiology</div> , <div class="volume">145 (Pt 11)</div>, <div class="fpage">3245</div>–<div class="lpage">53</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Novel%20Helicobacter%20pylori%20alpha1%2C2-fucosyltransferase%2C%20a%20key%20enzyme%20in%20the%20synthesis%20of%20Lewis%20antigens&amp;author=G.%20Wang&amp;author=P.G.%20Boulton&amp;author=N.W.%20Chan&amp;author=M.M.%20Palcic&amp;author=D.E.%20Taylor&amp;publication_year=1999&amp;journal=Microbiology&amp;volume=145 (Pt 11)&amp;pages=3245-53" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1099/00221287-145-11-3245" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1099%2F00221287-145-11-3245" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1099%2F00221287-145-11-3245"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10589734" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw046-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw046-B44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wang</div> <div class="given-names">G.</div></div><div class="name string-name"><div class="surname">Ge</div> <div class="given-names">Z.</div></div><div class="name string-name"><div class="surname">Rasko</div> <div class="given-names">D.A.</div></div><div class="name string-name"><div class="surname">Taylor</div> <div class="given-names">D.E.</div></div></div> <div class="year">2000</div>, 
<div class="article-title">Lewis antigens in <em>Helicobacter pylori</em>: biosynthesis and phase variation</div>. <div class="source">Mol. Microbiol</div> ., <div class="volume">36</div>, <div class="fpage">1187</div>–<div class="lpage">96</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Lewis%20antigens%20in%20Helicobacter%20pylori%3A%20biosynthesis%20and%20phase%20variation&amp;author=G.%20Wang&amp;author=Z.%20Ge&amp;author=D.A.%20Rasko&amp;author=D.E.%20Taylor&amp;publication_year=2000&amp;journal=Mol.%20Microbiol&amp;volume=36&amp;pages=1187-96" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-2958.2000.01934.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-2958.2000.01934.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-2958.2000.01934.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10931272" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw046-B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw046-B45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Pohl</div> <div class="given-names">M.A.</div></div><div class="name string-name"><div class="surname">Kienesberger</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Blaser</div> <div class="given-names">M.J.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Novel functions for glycosyltransferases Jhp0562 and GalT in Lewis antigen synthesis and variation in <em>Helicobacter pylori</em></div>. <div class="source">Infect. Immun</div> ., <div class="volume">80</div>, <div class="fpage">1593</div>–<div class="lpage">605</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Novel%20functions%20for%20glycosyltransferases%20Jhp0562%20and%20GalT%20in%20Lewis%20antigen%20synthesis%20and%20variation%20in%20Helicobacter%20pylori&amp;author=M.A.%20Pohl&amp;author=S.%20Kienesberger&amp;author=M.J.%20Blaser&amp;publication_year=2012&amp;journal=Infect.%20Immun&amp;volume=80&amp;pages=1593-605" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/IAI.00032-12" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FIAI.00032-12" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FIAI.00032-12"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22290141" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw046-B46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw046-B46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Rasko</div> <div class="given-names">D.A.</div></div><div class="name string-name"><div class="surname">Wang</div> <div class="given-names">G.</div></div><div class="name string-name"><div class="surname">Palcic</div> <div class="given-names">M.M.</div></div><div class="name string-name"><div class="surname">Taylor</div> <div class="given-names">D.E.</div></div></div> <div class="year">2000</div>, 
<div class="article-title">Cloning and characterization of the alpha(1,3/4) fucosyltransferase of <em>Helicobacter pylori</em></div>. <div class="source">J. Biol. Chem</div> ., <div class="volume">275</div>, <div class="fpage">4988</div>–<div class="lpage">94</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cloning%20and%20characterization%20of%20the%20alpha(1%2C3%2F4)%20fucosyltransferase%20of%20Helicobacter%20pylori&amp;author=D.A.%20Rasko&amp;author=G.%20Wang&amp;author=M.M.%20Palcic&amp;author=D.E.%20Taylor&amp;publication_year=2000&amp;journal=J.%20Biol.%20Chem&amp;volume=275&amp;pages=4988-94" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.275.7.4988" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.275.7.4988" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.275.7.4988"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10671538" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw046-B47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw046-B47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Carson</div> <div class="given-names">S.D.</div></div><div class="name string-name"><div class="surname">Stone</div> <div class="given-names">B.</div></div><div class="name string-name"><div class="surname">Beucher</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Fu</div> <div class="given-names">J.</div></div><div class="name string-name"><div class="surname">Sparling</div> <div class="given-names">P.F.</div></div></div> <div class="year">2000</div>, 
<div class="article-title">Phase variation of the gonococcal siderophore receptor FetA</div>. <div class="source">Mol. Microbiol</div> ., <div class="volume">36</div>, <div class="fpage">585</div>–<div class="lpage">93</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phase%20variation%20of%20the%20gonococcal%20siderophore%20receptor%20FetA&amp;author=S.D.%20Carson&amp;author=B.%20Stone&amp;author=M.%20Beucher&amp;author=J.%20Fu&amp;author=P.F.%20Sparling&amp;publication_year=2000&amp;journal=Mol.%20Microbiol&amp;volume=36&amp;pages=585-93" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-2958.2000.01873.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-2958.2000.01873.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-2958.2000.01873.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10844648" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw046-B48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw046-B48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">van Ham</div> <div class="given-names">S.M.</div></div><div class="name string-name"><div class="surname">van Alphen</div> <div class="given-names">L.</div></div><div class="name string-name"><div class="surname">Mooi</div> <div class="given-names">F.R.</div></div><div class="name string-name"><div class="surname">van Putten</div> <div class="given-names">J.P.</div></div></div> <div class="year">1993</div>, 
<div class="article-title">Phase variation of H. influenzae fimbriae: transcriptional control of two divergent genes through a variable combined promoter region</div>. <div class="source">Cell</div> , <div class="volume">73</div>, <div class="fpage">1187</div>–<div class="lpage">96</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phase%20variation%20of%20H.%20influenzae%20fimbriae%3A%20transcriptional%20control%20of%20two%20divergent%20genes%20through%20a%20variable%20combined%20promoter%20region&amp;author=S.M.%20van%20Ham&amp;author=L.%20van%20Alphen&amp;author=F.R.%20Mooi&amp;author=J.P.%20van%20Putten&amp;publication_year=1993&amp;journal=Cell&amp;volume=73&amp;pages=1187-96" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0092-8674(93)90647-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0092-8674(93)90647-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0092-8674(93)90647-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8513502" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw046-B49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw046-B49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Metruccio</div> <div class="given-names">M.M.</div></div><div class="name string-name"><div class="surname">Pigozzi</div> <div class="given-names">E.</div></div><div class="name string-name"><div class="surname">Roncarati</div> <div class="given-names">D.</div></div></div>, et al.  . <div class="year">2009</div>, 
<div class="article-title">A novel phase variation mechanism in the meningococcus driven by a ligand-responsive repressor and differential spacing of distal promoter elements</div>. <div class="source">PLoS Pathog</div> ., <div class="volume">5</div>, <div class="fpage">e1000710</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20novel%20phase%20variation%20mechanism%20in%20the%20meningococcus%20driven%20by%20a%20ligand-responsive%20repressor%20and%20differential%20spacing%20of%20distal%20promoter%20elements&amp;author=M.M.%20Metruccio&amp;author=E.%20Pigozzi&amp;author=D.%20Roncarati&amp;publication_year=2009&amp;journal=PLoS%20Pathog&amp;volume=5&amp;pages=e1000710" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.ppat.1000710" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.ppat.1000710" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.ppat.1000710"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20041170" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw046-B50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw046-B50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yang</div> <div class="given-names">B.</div></div><div class="name string-name"><div class="surname">Feng</div> <div class="given-names">L.</div></div><div class="name string-name"><div class="surname">Wang</div> <div class="given-names">F.</div></div><div class="name string-name"><div class="surname">Wang</div> <div class="given-names">L.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Enterohemorrhagic Escherichia coli senses low biotin status in the large intestine for colonization and infection</div>. <div class="source">Nat. Commun</div> ., <div class="volume">6</div>, <div class="fpage">6592</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Enterohemorrhagic%20Escherichia%20coli%20senses%20low%20biotin%20status%20in%20the%20large%20intestine%20for%20colonization%20and%20infection&amp;author=B.%20Yang&amp;author=L.%20Feng&amp;author=F.%20Wang&amp;author=L.%20Wang&amp;publication_year=2015&amp;journal=Nat.%20Commun&amp;volume=6&amp;pages=6592" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ncomms7592" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fncomms7592" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fncomms7592"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25791315" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw046-B51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw046-B51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chalker</div> <div class="given-names">A.F.</div></div><div class="name string-name"><div class="surname">Minehart</div> <div class="given-names">H.W.</div></div><div class="name string-name"><div class="surname">Hughes</div> <div class="given-names">N.J.</div></div></div>, et al.  . <div class="year">2001</div>, 
<div class="article-title">Systematic identification of selective essential genes in <em>Helicobacter pylori</em> by genome prioritization and allelic replacement mutagenesis</div>. <div class="source">J. Bacteriol</div> ., <div class="volume">183</div>, <div class="fpage">1259</div>–<div class="lpage">68</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Systematic%20identification%20of%20selective%20essential%20genes%20in%20Helicobacter%20pylori%20by%20genome%20prioritization%20and%20allelic%20replacement%20mutagenesis&amp;author=A.F.%20Chalker&amp;author=H.W.%20Minehart&amp;author=N.J.%20Hughes&amp;publication_year=2001&amp;journal=J.%20Bacteriol&amp;volume=183&amp;pages=1259-68" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.183.4.1259-1268.2001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.183.4.1259-1268.2001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.183.4.1259-1268.2001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11157938" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw046-B52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw046-B52" class="ref-content "><div class="label -label">52</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Danielli</div> <div class="given-names">A.</div></div><div class="name string-name"><div class="surname">Roncarati</div> <div class="given-names">D.</div></div><div class="name string-name"><div class="surname">Delany</div> <div class="given-names">I.</div></div><div class="name string-name"><div class="surname">Chiarini</div> <div class="given-names">V.</div></div><div class="name string-name"><div class="surname">Rappuoli</div> <div class="given-names">R.</div></div><div class="name string-name"><div class="surname">Scarlato</div> <div class="given-names">V.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">In vivo dissection of the <em>Helicobacter pylori</em> Fur regulatory circuit by genome-wide location analysis</div>. <div class="source">J. Bacteriol</div> ., <div class="volume">188</div>, <div class="fpage">4654</div>–<div class="lpage">62</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=In%20vivo%20dissection%20of%20the%20Helicobacter%20pylori%20Fur%20regulatory%20circuit%20by%20genome-wide%20location%20analysis&amp;author=A.%20Danielli&amp;author=D.%20Roncarati&amp;author=I.%20Delany&amp;author=V.%20Chiarini&amp;author=R.%20Rappuoli&amp;author=V.%20Scarlato&amp;publication_year=2006&amp;journal=J.%20Bacteriol&amp;volume=188&amp;pages=4654-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.00120-06" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.00120-06" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.00120-06"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16788174" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2016. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=2670112&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw046/1/dsw046_Supp.zip?Expires=2147483647&amp;Signature=s3q4bhSX2C3MUF-jr0f2UZ1jrVjkiGXzFiaUNbhztd4DChTCgvQajffmhInT-1Vj~0SRO~Kttfa4bqexSP8TrhsM2dD3tPD9oxAo17tPZ377FPaKsD8p~u48AhZfGosl2DgYDxY~wAxQIvmlENgGz~hpjXyN~I6CdeKNliwIOlCIEQtChC1zhSWLmnuwd57p2crX2HhvAlPdEQQh2bsMNnLu1COOQRPn604GZfBH8hQezXkfEFO3oCZfpwLaMYrQ~H4BcWpJOG4ygAaGy1rYakppJlB7TfE5VrMgGP-nBVCzXm0JcV4NeHqmjB9Dbt~-D52akH052YyUn9NKOPfy5A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">467</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
0                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsw046"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="2670112"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f24%2f1%2f37%2f2670112">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F24%2F1%2F37%2F2670112" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/2670112'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Quantum changes in Helicobacter pylori gene expression accompany host-adaptation" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsw046");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f24%2f1%2f37%2f2670112">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

