

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/13/4/155/348476","@type":"ScholarlyArticle","name":"Diverse DNA Methylation Statuses at Alternative Promoters of Human Genes in Various Tissues","datePublished":"2006-01-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/13/4","@type":"PublicationIssue","issueNumber":"4","datePublished":"2006-01-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsl008","keywords":["DNA methylation status","CpG islands"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/13/4/155/348476","author":[{"name":"Cheong, Jieun","affiliation":"1  Department of Medical Genome Sciences, Graduate School of Frontier Sciences, the University of Tokyo  5-1-5 Kashiwanoha, Kashiwa, Chiba 277-8562, Japan    2  Graduate School of Medicine, the University of Tokyo  7-3-1 Hongo, Bunkyoku, Tokyo 113-0033, Japan","@type":"Person"},{"name":"Yamada, Yoichi","affiliation":"3  Division of Electrical Engineering and Computer Science, Graduate School of Natural Science and Technology, Kanazawa University  Kakuma-machi, Kanazawa, Ishikawa 920-1192, Japan","@type":"Person"},{"name":"Yamashita, Riu","affiliation":"4  Human Genome Center, Institute of Medical Science, the University of Tokyo  4-6-1 Shirokanedai, Minatoku, 108-8639 Tokyo, Japan","@type":"Person"},{"name":"Irie, Takuma","affiliation":"1  Department of Medical Genome Sciences, Graduate School of Frontier Sciences, the University of Tokyo  5-1-5 Kashiwanoha, Kashiwa, Chiba 277-8562, Japan","@type":"Person"},{"name":"Kanai, Akinori","affiliation":"1  Department of Medical Genome Sciences, Graduate School of Frontier Sciences, the University of Tokyo  5-1-5 Kashiwanoha, Kashiwa, Chiba 277-8562, Japan","@type":"Person"},{"name":"Wakaguri, Hiroyuki","affiliation":"1  Department of Medical Genome Sciences, Graduate School of Frontier Sciences, the University of Tokyo  5-1-5 Kashiwanoha, Kashiwa, Chiba 277-8562, Japan","@type":"Person"},{"name":"Nakai, Kenta","affiliation":"1  Department of Medical Genome Sciences, Graduate School of Frontier Sciences, the University of Tokyo  5-1-5 Kashiwanoha, Kashiwa, Chiba 277-8562, Japan    4  Human Genome Center, Institute of Medical Science, the University of Tokyo  4-6-1 Shirokanedai, Minatoku, 108-8639 Tokyo, Japan","@type":"Person"},{"name":"Ito, Takashi","affiliation":"5  Department of Computational Biology, Graduate School of Frontier Sciences, the University of Tokyo  5-1-5 Kashiwanoha, Kashiwa, Chiba 277-8561, Japan","@type":"Person"},{"name":"Saito, Izumu","affiliation":"1  Department of Medical Genome Sciences, Graduate School of Frontier Sciences, the University of Tokyo  5-1-5 Kashiwanoha, Kashiwa, Chiba 277-8562, Japan    2  Graduate School of Medicine, the University of Tokyo  7-3-1 Hongo, Bunkyoku, Tokyo 113-0033, Japan","@type":"Person"},{"name":"Sugano, Sumio","affiliation":"1  Department of Medical Genome Sciences, Graduate School of Frontier Sciences, the University of Tokyo  5-1-5 Kashiwanoha, Kashiwa, Chiba 277-8562, Japan","@type":"Person"},{"name":"Suzuki, Yutaka","affiliation":"1  Department of Medical Genome Sciences, Graduate School of Frontier Sciences, the University of Tokyo  5-1-5 Kashiwanoha, Kashiwa, Chiba 277-8562, Japan","@type":"Person"}],"description":"Abstract.  We characterized the DNA methylation status at 144 tissue-biased and 37 non-tissue-biased alternative promoters of 61 human genes in five normal tiss","pageStart":"155","pageEnd":"167","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/13/4/0/m_dnaresearch13_4.cover.gif?Expires=2147483647&Signature=zpj2n5NzsMk30zA4qehj6yWrvGBU~IiLu4DURkTxdl~55txhlJ5z0a2IpdyGYQcF4vlVunu-~Wh08QUqWKvj48dNgA7n8vLabfgEOY0CoaiBxls7NyW6tCeXW6Sfd2X4lV4uoHFAXj3NLXFLZshkdiFCqF5MJYIi0qOwNjsIUdAwRNhaR9xqEQSv1J65s5eb8WSbVSZ7jKourjIePJi-0rEJRs9B7Kjh~xeLnZz~SmpYlxNUZP4MEUv8qvfsHNtVbSMWxc-iOv9niMhvvwUPs612TggXmBVQynK3LwlztmX3LfsmbR-PpnPA2Dhenm3q1yRIy9w5VQzSlMaqz8FNQA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Diverse DNA Methylation Statuses at Alternative Promoters of Human Genes in Various Tissues","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/13/4/0/m_dnaresearch13_4.cover.gif?Expires=2147483647&Signature=zpj2n5NzsMk30zA4qehj6yWrvGBU~IiLu4DURkTxdl~55txhlJ5z0a2IpdyGYQcF4vlVunu-~Wh08QUqWKvj48dNgA7n8vLabfgEOY0CoaiBxls7NyW6tCeXW6Sfd2X4lV4uoHFAXj3NLXFLZshkdiFCqF5MJYIi0qOwNjsIUdAwRNhaR9xqEQSv1J65s5eb8WSbVSZ7jKourjIePJi-0rEJRs9B7Kjh~xeLnZz~SmpYlxNUZP4MEUv8qvfsHNtVbSMWxc-iOv9niMhvvwUPs612TggXmBVQynK3LwlztmX3LfsmbR-PpnPA2Dhenm3q1yRIy9w5VQzSlMaqz8FNQA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Diverse DNA Methylation Statuses at Alternative Promoters of Human Genes in Various Tissues" />
    <meta property="og:description" content="Abstract.  We characterized the DNA methylation status at 144 tissue-biased and 37 non-tissue-biased alternative promoters of 61 human genes in five normal tiss" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/13/4/155/348476" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2006-01-01" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Cheong, Jieun" /><meta name="citation_author_institution" content="1Department of Medical Genome Sciences, Graduate School of Frontier Sciences, the University of Tokyo5-1-5 Kashiwanoha, Kashiwa, Chiba 277-8562, Japan" /><meta name="citation_author_institution" content="2Graduate School of Medicine, the University of Tokyo7-3-1 Hongo, Bunkyoku, Tokyo 113-0033, Japan" /><meta name="citation_author" content="Yamada, Yoichi" /><meta name="citation_author_institution" content="3Division of Electrical Engineering and Computer Science, Graduate School of Natural Science and Technology, Kanazawa UniversityKakuma-machi, Kanazawa, Ishikawa 920-1192, Japan" /><meta name="citation_author" content="Yamashita, Riu" /><meta name="citation_author_institution" content="4Human Genome Center, Institute of Medical Science, the University of Tokyo4-6-1 Shirokanedai, Minatoku, 108-8639 Tokyo, Japan" /><meta name="citation_author" content="Irie, Takuma" /><meta name="citation_author_institution" content="1Department of Medical Genome Sciences, Graduate School of Frontier Sciences, the University of Tokyo5-1-5 Kashiwanoha, Kashiwa, Chiba 277-8562, Japan" /><meta name="citation_author" content="Kanai, Akinori" /><meta name="citation_author_institution" content="1Department of Medical Genome Sciences, Graduate School of Frontier Sciences, the University of Tokyo5-1-5 Kashiwanoha, Kashiwa, Chiba 277-8562, Japan" /><meta name="citation_author" content="Wakaguri, Hiroyuki" /><meta name="citation_author_institution" content="1Department of Medical Genome Sciences, Graduate School of Frontier Sciences, the University of Tokyo5-1-5 Kashiwanoha, Kashiwa, Chiba 277-8562, Japan" /><meta name="citation_author" content="Nakai, Kenta" /><meta name="citation_author_institution" content="1Department of Medical Genome Sciences, Graduate School of Frontier Sciences, the University of Tokyo5-1-5 Kashiwanoha, Kashiwa, Chiba 277-8562, Japan" /><meta name="citation_author_institution" content="4Human Genome Center, Institute of Medical Science, the University of Tokyo4-6-1 Shirokanedai, Minatoku, 108-8639 Tokyo, Japan" /><meta name="citation_author" content="Ito, Takashi" /><meta name="citation_author_institution" content="5Department of Computational Biology, Graduate School of Frontier Sciences, the University of Tokyo5-1-5 Kashiwanoha, Kashiwa, Chiba 277-8561, Japan" /><meta name="citation_author" content="Saito, Izumu" /><meta name="citation_author_institution" content="1Department of Medical Genome Sciences, Graduate School of Frontier Sciences, the University of Tokyo5-1-5 Kashiwanoha, Kashiwa, Chiba 277-8562, Japan" /><meta name="citation_author_institution" content="2Graduate School of Medicine, the University of Tokyo7-3-1 Hongo, Bunkyoku, Tokyo 113-0033, Japan" /><meta name="citation_author" content="Sugano, Sumio" /><meta name="citation_author_institution" content="1Department of Medical Genome Sciences, Graduate School of Frontier Sciences, the University of Tokyo5-1-5 Kashiwanoha, Kashiwa, Chiba 277-8562, Japan" /><meta name="citation_author" content="Suzuki, Yutaka" /><meta name="citation_author_institution" content="1Department of Medical Genome Sciences, Graduate School of Frontier Sciences, the University of Tokyo5-1-5 Kashiwanoha, Kashiwa, Chiba 277-8562, Japan" /><meta name="citation_title" content="Diverse DNA Methylation Statuses at Alternative Promoters of Human Genes in Various Tissues" /><meta name="citation_firstpage" content="155" /><meta name="citation_lastpage" content="167" /><meta name="citation_doi" content="10.1093/dnares/dsl008" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="13" /><meta name="citation_issue" content="4" /><meta name="citation_publication_date" content="2006/01/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Diversification of transcriptional modulation: large-scale identification and characterization of putative alternative promoters of human genes; Genome Res.;  citation_year=2006;  citation_volume=16;  citation_pages=55-65; " /><meta name="citation_reference" content="citation_title=A high-resolution map of active promoters in the human genome; Nature;  citation_year=2005;  citation_volume=436;  citation_pages=876-880; " /><meta name="citation_reference" content="citation_title=Examples of the complex architecture of the human transcriptome revealed by RACE and high-density tiling arrays; Genome Res.;  citation_year=2005;  citation_volume=15;  citation_pages=987-997; " /><meta name="citation_reference" content="citation_title=Genome-wide analysis of mammalian promoter architecture and evolution; Nat. Genet.;  citation_year=2006;  citation_volume=38;  citation_pages=608-609; " /><meta name="citation_reference" content="citation_title=Complex controls: the role of alternative promoters in mammalian genomes; Trends Genet.;  citation_year=2003;  citation_volume=19;  citation_pages=640-648; " /><meta name="citation_reference" content="citation_title=Parallel patterns of evolution in the genomes and transcriptomes of humans and chimpanzees; Science;  citation_year=2005;  citation_volume=309;  citation_pages=1850-1854; " /><meta name="citation_reference" content="citation_title=Evolution at two levels in humans and chimpanzees; Science;  citation_year=1975;  citation_volume=188;  citation_pages=107-116; " /><meta name="citation_reference" content="citation_title=Identification and characterization of the potential promoter regions of 1031 kinds of human genes; Genome Res.;  citation_year=2001;  citation_volume=11;  citation_pages=677-684; " /><meta name="citation_reference" content="citation_title=Construction of a full-length enriched and a 5′-end enriched cDNA library using the oligo-capping method; Methods Mol. Biol.;  citation_year=2003;  citation_volume=221;  citation_pages=73-91; " /><meta name="citation_reference" content="citation_title=DBTSS: database of human transcription start sites, progress report 2006; Nucleic Acids Res.;  citation_year=2006;  citation_volume=34;  citation_pages=D86-D89; " /><meta name="citation_reference" content="citation_title=CpG islands in vertebrate genomes; J. Mol. Biol.;  citation_year=1987;  citation_volume=196;  citation_pages=261-282; " /><meta name="citation_reference" content="citation_title=The fundamental role of epigenetic events in cancer; Nat. Rev. Genet.;  citation_year=2002;  citation_volume=3;  citation_pages=415-428; " /><meta name="citation_reference" content="citation_title=Cancer epigenetics; Hum. Mol. Genet.;  citation_year=2005;  citation_volume=14;  citation_pages=R65-R76; " /><meta name="citation_reference" content="citation_title=DNA methylation and gene silencing in cancer: which is the guilty party?; Oncogene;  citation_year=2002;  citation_volume=21;  citation_pages=5380-5387; " /><meta name="citation_reference" content="citation_title=From genome to epigenome; Hum. Mol. Genet.;  citation_year=2005;  citation_volume=14;  citation_pages=R3-R10; " /><meta name="citation_reference" content="citation_title=Epigenomics: beyond CpG islands; Nat. Rev. Genet.;  citation_year=2004;  citation_volume=5;  citation_pages=446-455; " /><meta name="citation_reference" content="citation_title=High sensitivity mapping of methylated cytosines; Nucleic Acids Res.;  citation_year=1994;  citation_volume=22;  citation_pages=2990-2997; " /><meta name="citation_reference" content="citation_title=A genomic sequencing protocol that yields a positive display of 5-methylcytosine residues in individual DNA strands; Proc. Natl Acad. Sci. USA;  citation_year=1992;  citation_volume=89;  citation_pages=1827-1831; " /><meta name="citation_reference" content="citation_title=A comprehensive analysis of allelic methylation status of CpG islands on human chromosome 21q; Genome Res.;  citation_year=2004;  citation_volume=14;  citation_pages=247-266; " /><meta name="citation_reference" content="citation_title=Mutations in SEPT9 cause hereditary neuralgic amyotrophy; Nat. Genet.;  citation_year=2005;  citation_volume=37;  citation_pages=1044-1046; " /><meta name="citation_reference" content="citation_title=Genome-wide profiling of promoter methylation in human; Oncogene.;  citation_year=2006;  citation_volume=25;  citation_pages=3059-3064; " /><meta name="citation_reference" content="citation_title=Microarray analysis of promoter methylation in lung cancers; J. Hum. Genet.;  citation_year=2006;  citation_volume=51;  citation_pages=368-374; " /><meta name="citation_reference" content="citation_title=High levels of de novo methylation and altered chromatin structure at CpG islands in cell lines; Cell;  citation_year=1990;  citation_volume=62;  citation_pages=503-514; " /><meta name="citation_reference" content="citation_title=DNA methylation and histone modifications: teaming up to silence genes; Curr. Opin. Genet. Dev.;  citation_year=2005;  citation_volume=15;  citation_pages=490-495; " /><meta name="citation_reference" content="citation_title=Predicting gene expression from sequence; Cell;  citation_year=2004;  citation_volume=117;  citation_pages=185-198; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/13/4/155/998674/dsl008.pdf" /><meta name="description" content="Abstract.  We characterized the DNA methylation status at 144 tissue-biased and 37 non-tissue-biased alternative promoters of 61 human genes in five normal tiss" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/13/4/155/348476" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/13/4/155/348476" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/13/4/155/348476");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsl008';
    </script>
<title>
	Diverse DNA Methylation Statuses at Alternative Promoters of Human Genes in Various Tissues | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./348476" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BfJLPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJDaGVvbmcsIEppZXVuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iMURlcGFydG1lbnQgb2YgTWVkaWNhbCBHZW5vbWUgU2NpZW5jZXMsIEdyYWR1YXRlIFNjaG9vbCBvZiBGcm9udGllciBTY2llbmNlcywgdGhlIFVuaXZlcnNpdHkgb2YgVG9reW81LTEtNSBLYXNoaXdhbm9oYSwgS2FzaGl3YSwgQ2hpYmEgMjc3LTg1NjIsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iMkdyYWR1YXRlIFNjaG9vbCBvZiBNZWRpY2luZSwgdGhlIFVuaXZlcnNpdHkgb2YgVG9reW83LTMtMSBIb25nbywgQnVua3lva3UsIFRva3lvIDExMy0wMDMzLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IllhbWFkYSwgWW9pY2hpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iM0RpdmlzaW9uIG9mIEVsZWN0cmljYWwgRW5naW5lZXJpbmcgYW5kIENvbXB1dGVyIFNjaWVuY2UsIEdyYWR1YXRlIFNjaG9vbCBvZiBOYXR1cmFsIFNjaWVuY2UgYW5kIFRlY2hub2xvZ3ksIEthbmF6YXdhIFVuaXZlcnNpdHlLYWt1bWEtbWFjaGksIEthbmF6YXdhLCBJc2hpa2F3YSA5MjAtMTE5MiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJZYW1hc2hpdGEsIFJpdSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IjRIdW1hbiBHZW5vbWUgQ2VudGVyLCBJbnN0aXR1dGUgb2YgTWVkaWNhbCBTY2llbmNlLCB0aGUgVW5pdmVyc2l0eSBvZiBUb2t5bzQtNi0xIFNoaXJva2FuZWRhaSwgTWluYXRva3UsIDEwOC04NjM5IFRva3lvLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IklyaWUsIFRha3VtYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IjFEZXBhcnRtZW50IG9mIE1lZGljYWwgR2Vub21lIFNjaWVuY2VzLCBHcmFkdWF0ZSBTY2hvb2wgb2YgRnJvbnRpZXIgU2NpZW5jZXMsIHRoZSBVbml2ZXJzaXR5IG9mIFRva3lvNS0xLTUgS2FzaGl3YW5vaGEsIEthc2hpd2EsIENoaWJhIDI3Ny04NTYyLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkthbmFpLCBBa2lub3JpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iMURlcGFydG1lbnQgb2YgTWVkaWNhbCBHZW5vbWUgU2NpZW5jZXMsIEdyYWR1YXRlIFNjaG9vbCBvZiBGcm9udGllciBTY2llbmNlcywgdGhlIFVuaXZlcnNpdHkgb2YgVG9reW81LTEtNSBLYXNoaXdhbm9oYSwgS2FzaGl3YSwgQ2hpYmEgMjc3LTg1NjIsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iV2FrYWd1cmksIEhpcm95dWtpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iMURlcGFydG1lbnQgb2YgTWVkaWNhbCBHZW5vbWUgU2NpZW5jZXMsIEdyYWR1YXRlIFNjaG9vbCBvZiBGcm9udGllciBTY2llbmNlcywgdGhlIFVuaXZlcnNpdHkgb2YgVG9reW81LTEtNSBLYXNoaXdhbm9oYSwgS2FzaGl3YSwgQ2hpYmEgMjc3LTg1NjIsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTmFrYWksIEtlbnRhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iMURlcGFydG1lbnQgb2YgTWVkaWNhbCBHZW5vbWUgU2NpZW5jZXMsIEdyYWR1YXRlIFNjaG9vbCBvZiBGcm9udGllciBTY2llbmNlcywgdGhlIFVuaXZlcnNpdHkgb2YgVG9reW81LTEtNSBLYXNoaXdhbm9oYSwgS2FzaGl3YSwgQ2hpYmEgMjc3LTg1NjIsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iNEh1bWFuIEdlbm9tZSBDZW50ZXIsIEluc3RpdHV0ZSBvZiBNZWRpY2FsIFNjaWVuY2UsIHRoZSBVbml2ZXJzaXR5IG9mIFRva3lvNC02LTEgU2hpcm9rYW5lZGFpLCBNaW5hdG9rdSwgMTA4LTg2MzkgVG9reW8sIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSXRvLCBUYWthc2hpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iNURlcGFydG1lbnQgb2YgQ29tcHV0YXRpb25hbCBCaW9sb2d5LCBHcmFkdWF0ZSBTY2hvb2wgb2YgRnJvbnRpZXIgU2NpZW5jZXMsIHRoZSBVbml2ZXJzaXR5IG9mIFRva3lvNS0xLTUgS2FzaGl3YW5vaGEsIEthc2hpd2EsIENoaWJhIDI3Ny04NTYxLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlNhaXRvLCBJenVtdSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IjFEZXBhcnRtZW50IG9mIE1lZGljYWwgR2Vub21lIFNjaWVuY2VzLCBHcmFkdWF0ZSBTY2hvb2wgb2YgRnJvbnRpZXIgU2NpZW5jZXMsIHRoZSBVbml2ZXJzaXR5IG9mIFRva3lvNS0xLTUgS2FzaGl3YW5vaGEsIEthc2hpd2EsIENoaWJhIDI3Ny04NTYyLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IjJHcmFkdWF0ZSBTY2hvb2wgb2YgTWVkaWNpbmUsIHRoZSBVbml2ZXJzaXR5IG9mIFRva3lvNy0zLTEgSG9uZ28sIEJ1bmt5b2t1LCBUb2t5byAxMTMtMDAzMywgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTdWdhbm8sIFN1bWlvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iMURlcGFydG1lbnQgb2YgTWVkaWNhbCBHZW5vbWUgU2NpZW5jZXMsIEdyYWR1YXRlIFNjaG9vbCBvZiBGcm9udGllciBTY2llbmNlcywgdGhlIFVuaXZlcnNpdHkgb2YgVG9reW81LTEtNSBLYXNoaXdhbm9oYSwgS2FzaGl3YSwgQ2hpYmEgMjc3LTg1NjIsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU3V6dWtpLCBZdXRha2EiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSIxRGVwYXJ0bWVudCBvZiBNZWRpY2FsIEdlbm9tZSBTY2llbmNlcywgR3JhZHVhdGUgU2Nob29sIG9mIEZyb250aWVyIFNjaWVuY2VzLCB0aGUgVW5pdmVyc2l0eSBvZiBUb2t5bzUtMS01IEthc2hpd2Fub2hhLCBLYXNoaXdhLCBDaGliYSAyNzctODU2MiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IkRpdmVyc2UgRE5BIE1ldGh5bGF0aW9uIFN0YXR1c2VzIGF0IEFsdGVybmF0aXZlIFByb21vdGVycyBvZiBIdW1hbiBHZW5lcyBpbiBWYXJpb3VzIFRpc3N1ZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIxNTUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjE2NyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzbDAwOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIxMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iNCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDA2LzAxLzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGl2ZXJzaWZpY2F0aW9uIG9mIHRyYW5zY3JpcHRpb25hbCBtb2R1bGF0aW9uOiBsYXJnZS1zY2FsZSBpZGVudGlmaWNhdGlvbiBhbmQgY2hhcmFjdGVyaXphdGlvbiBvZiBwdXRhdGl2ZSBhbHRlcm5hdGl2ZSBwcm9tb3RlcnMgb2YgaHVtYW4gZ2VuZXM7IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9NTUtNjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgaGlnaC1yZXNvbHV0aW9uIG1hcCBvZiBhY3RpdmUgcHJvbW90ZXJzIGluIHRoZSBodW1hbiBnZW5vbWU7IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT00MzY7ICBjaXRhdGlvbl9wYWdlcz04NzYtODgwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FeGFtcGxlcyBvZiB0aGUgY29tcGxleCBhcmNoaXRlY3R1cmUgb2YgdGhlIGh1bWFuIHRyYW5zY3JpcHRvbWUgcmV2ZWFsZWQgYnkgUkFDRSBhbmQgaGlnaC1kZW5zaXR5IHRpbGluZyBhcnJheXM7IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTE1OyAgY2l0YXRpb25fcGFnZXM9OTg3LTk5NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lLXdpZGUgYW5hbHlzaXMgb2YgbWFtbWFsaWFuIHByb21vdGVyIGFyY2hpdGVjdHVyZSBhbmQgZXZvbHV0aW9uOyBOYXQuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0zODsgIGNpdGF0aW9uX3BhZ2VzPTYwOC02MDk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBsZXggY29udHJvbHM6IHRoZSByb2xlIG9mIGFsdGVybmF0aXZlIHByb21vdGVycyBpbiBtYW1tYWxpYW4gZ2Vub21lczsgVHJlbmRzIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xOTsgIGNpdGF0aW9uX3BhZ2VzPTY0MC02NDg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBhcmFsbGVsIHBhdHRlcm5zIG9mIGV2b2x1dGlvbiBpbiB0aGUgZ2Vub21lcyBhbmQgdHJhbnNjcmlwdG9tZXMgb2YgaHVtYW5zIGFuZCBjaGltcGFuemVlczsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0zMDk7ICBjaXRhdGlvbl9wYWdlcz0xODUwLTE4NTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2b2x1dGlvbiBhdCB0d28gbGV2ZWxzIGluIGh1bWFucyBhbmQgY2hpbXBhbnplZXM7IFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTE5NzU7ICBjaXRhdGlvbl92b2x1bWU9MTg4OyAgY2l0YXRpb25fcGFnZXM9MTA3LTExNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZmljYXRpb24gYW5kIGNoYXJhY3Rlcml6YXRpb24gb2YgdGhlIHBvdGVudGlhbCBwcm9tb3RlciByZWdpb25zIG9mIDEwMzEga2luZHMgb2YgaHVtYW4gZ2VuZXM7IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9Njc3LTY4NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29uc3RydWN0aW9uIG9mIGEgZnVsbC1sZW5ndGggZW5yaWNoZWQgYW5kIGEgNeKAsi1lbmQgZW5yaWNoZWQgY0ROQSBsaWJyYXJ5IHVzaW5nIHRoZSBvbGlnby1jYXBwaW5nIG1ldGhvZDsgTWV0aG9kcyBNb2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTIyMTsgIGNpdGF0aW9uX3BhZ2VzPTczLTkxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EQlRTUzogZGF0YWJhc2Ugb2YgaHVtYW4gdHJhbnNjcmlwdGlvbiBzdGFydCBzaXRlcywgcHJvZ3Jlc3MgcmVwb3J0IDIwMDY7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0zNDsgIGNpdGF0aW9uX3BhZ2VzPUQ4Ni1EODk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNwRyBpc2xhbmRzIGluIHZlcnRlYnJhdGUgZ2Vub21lczsgSi4gTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk4NzsgIGNpdGF0aW9uX3ZvbHVtZT0xOTY7ICBjaXRhdGlvbl9wYWdlcz0yNjEtMjgyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZnVuZGFtZW50YWwgcm9sZSBvZiBlcGlnZW5ldGljIGV2ZW50cyBpbiBjYW5jZXI7IE5hdC4gUmV2LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MzsgIGNpdGF0aW9uX3BhZ2VzPTQxNS00Mjg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNhbmNlciBlcGlnZW5ldGljczsgSHVtLiBNb2wuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPVI2NS1SNzY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUROQSBtZXRoeWxhdGlvbiBhbmQgZ2VuZSBzaWxlbmNpbmcgaW4gY2FuY2VyOiB3aGljaCBpcyB0aGUgZ3VpbHR5IHBhcnR5PzsgT25jb2dlbmU7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MjE7ICBjaXRhdGlvbl9wYWdlcz01MzgwLTUzODc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZyb20gZ2Vub21lIHRvIGVwaWdlbm9tZTsgSHVtLiBNb2wuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPVIzLVIxMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXBpZ2Vub21pY3M6IGJleW9uZCBDcEcgaXNsYW5kczsgTmF0LiBSZXYuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9NDQ2LTQ1NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGlnaCBzZW5zaXRpdml0eSBtYXBwaW5nIG9mIG1ldGh5bGF0ZWQgY3l0b3NpbmVzOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5OTQ7ICBjaXRhdGlvbl92b2x1bWU9MjI7ICBjaXRhdGlvbl9wYWdlcz0yOTkwLTI5OTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgZ2Vub21pYyBzZXF1ZW5jaW5nIHByb3RvY29sIHRoYXQgeWllbGRzIGEgcG9zaXRpdmUgZGlzcGxheSBvZiA1LW1ldGh5bGN5dG9zaW5lIHJlc2lkdWVzIGluIGluZGl2aWR1YWwgRE5BIHN0cmFuZHM7IFByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTE5OTI7ICBjaXRhdGlvbl92b2x1bWU9ODk7ICBjaXRhdGlvbl9wYWdlcz0xODI3LTE4MzE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgY29tcHJlaGVuc2l2ZSBhbmFseXNpcyBvZiBhbGxlbGljIG1ldGh5bGF0aW9uIHN0YXR1cyBvZiBDcEcgaXNsYW5kcyBvbiBodW1hbiBjaHJvbW9zb21lIDIxcTsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz0yNDctMjY2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NdXRhdGlvbnMgaW4gU0VQVDkgY2F1c2UgaGVyZWRpdGFyeSBuZXVyYWxnaWMgYW15b3Ryb3BoeTsgTmF0LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9Mzc7ICBjaXRhdGlvbl9wYWdlcz0xMDQ0LTEwNDY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIHByb2ZpbGluZyBvZiBwcm9tb3RlciBtZXRoeWxhdGlvbiBpbiBodW1hbjsgT25jb2dlbmUuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9MzA1OS0zMDY0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NaWNyb2FycmF5IGFuYWx5c2lzIG9mIHByb21vdGVyIG1ldGh5bGF0aW9uIGluIGx1bmcgY2FuY2VyczsgSi4gSHVtLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NTE7ICBjaXRhdGlvbl9wYWdlcz0zNjgtMzc0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IaWdoIGxldmVscyBvZiBkZSBub3ZvIG1ldGh5bGF0aW9uIGFuZCBhbHRlcmVkIGNocm9tYXRpbiBzdHJ1Y3R1cmUgYXQgQ3BHIGlzbGFuZHMgaW4gY2VsbCBsaW5lczsgQ2VsbDsgIGNpdGF0aW9uX3llYXI9MTk5MDsgIGNpdGF0aW9uX3ZvbHVtZT02MjsgIGNpdGF0aW9uX3BhZ2VzPTUwMy01MTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUROQSBtZXRoeWxhdGlvbiBhbmQgaGlzdG9uZSBtb2RpZmljYXRpb25zOiB0ZWFtaW5nIHVwIHRvIHNpbGVuY2UgZ2VuZXM7IEN1cnIuIE9waW4uIEdlbmV0LiBEZXYuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTE1OyAgY2l0YXRpb25fcGFnZXM9NDkwLTQ5NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJlZGljdGluZyBnZW5lIGV4cHJlc3Npb24gZnJvbSBzZXF1ZW5jZTsgQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTc7ICBjaXRhdGlvbl9wYWdlcz0xODUtMTk4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMTMvNC8xNTUvOTk4Njc0L2RzbDAwOC5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgV2UgY2hhcmFjdGVyaXplZCB0aGUgRE5BIG1ldGh5bGF0aW9uIHN0YXR1cyBhdCAxNDQgdGlzc3VlLWJpYXNlZCBhbmQgMzcgbm9uLXRpc3N1ZS1iaWFzZWQgYWx0ZXJuYXRpdmUgcHJvbW90ZXJzIG9mIDYxIGh1bWFuIGdlbmVzIGluIGZpdmUgbm9ybWFsIHRpc3MiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVyLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTM0ODQ3NiZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGSGgLozS5YWyShNtBwaPhXIXAnALQ==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f13%2f4%2f155%2f348476" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f13%2f4%2f155%2f348476" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=13440" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/13/4"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/13/4/0/m_dnaresearch13_4.cover.gif?Expires=2147483647&amp;Signature=zpj2n5NzsMk30zA4qehj6yWrvGBU~IiLu4DURkTxdl~55txhlJ5z0a2IpdyGYQcF4vlVunu-~Wh08QUqWKvj48dNgA7n8vLabfgEOY0CoaiBxls7NyW6tCeXW6Sfd2X4lV4uoHFAXj3NLXFLZshkdiFCqF5MJYIi0qOwNjsIUdAwRNhaR9xqEQSv1J65s5eb8WSbVSZ7jKourjIePJi-0rEJRs9B7Kjh~xeLnZz~SmpYlxNUZP4MEUv8qvfsHNtVbSMWxc-iOv9niMhvvwUPs612TggXmBVQynK3LwlztmX3LfsmbR-PpnPA2Dhenm3q1yRIy9w5VQzSlMaqz8FNQA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/13/4">
                        <div class="volume trailing-comma">Volume 13</div>
                        <div class="issue">Issue 4</div>

            </a>

        </div>
        <div class="ii-pub-date">
2006        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="4946909">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4946909">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4946911">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4946911">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4946917">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4946917">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4946932">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4946932">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4946958">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4946958">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4946969">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4946969">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/13/4/141/348695">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/13/4/169/348050">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Diverse DNA Methylation Statuses at Alternative Promoters of Human Genes in Various Tissues
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                            <div class="meta-authors--limited">
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jieun Cheong</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><sup>1</sup><div class="insititution">Department of Medical Genome Sciences, Graduate School of Frontier Sciences, the University of Tokyo</div><div class="addr-line">5-1-5 Kashiwanoha, Kashiwa, Chiba 277-8562, Japan</div></div><div class="aff"><div class="label -label">2</div><sup>2</sup><div class="insititution">Graduate School of Medicine, the University of Tokyo</div><div class="addr-line">7-3-1 Hongo, Bunkyoku, Tokyo 113-0033, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jieun+Cheong">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Cheong J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Cheong J&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Jieun Cheong</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yoichi Yamada</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><sup>3</sup><div class="insititution">Division of Electrical Engineering and Computer Science, Graduate School of Natural Science and Technology, Kanazawa University</div><div class="addr-line">Kakuma-machi, Kanazawa, Ishikawa 920-1192, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yoichi+Yamada">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yamada Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yamada Y&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Yoichi Yamada</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Riu Yamashita</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><sup>4</sup><div class="insititution">Human Genome Center, Institute of Medical Science, the University of Tokyo</div><div class="addr-line">4-6-1 Shirokanedai, Minatoku, 108-8639 Tokyo, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Riu+Yamashita">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yamashita R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yamashita R&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Riu Yamashita</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Takuma Irie</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><sup>1</sup><div class="insititution">Department of Medical Genome Sciences, Graduate School of Frontier Sciences, the University of Tokyo</div><div class="addr-line">5-1-5 Kashiwanoha, Kashiwa, Chiba 277-8562, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Takuma+Irie">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Irie T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Irie T&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Takuma Irie</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Akinori Kanai</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><sup>1</sup><div class="insititution">Department of Medical Genome Sciences, Graduate School of Frontier Sciences, the University of Tokyo</div><div class="addr-line">5-1-5 Kashiwanoha, Kashiwa, Chiba 277-8562, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Akinori+Kanai">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kanai A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kanai A&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Akinori Kanai</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hiroyuki Wakaguri</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><sup>1</sup><div class="insititution">Department of Medical Genome Sciences, Graduate School of Frontier Sciences, the University of Tokyo</div><div class="addr-line">5-1-5 Kashiwanoha, Kashiwa, Chiba 277-8562, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hiroyuki+Wakaguri">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Wakaguri H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Wakaguri H&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Hiroyuki Wakaguri</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kenta Nakai</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><sup>1</sup><div class="insititution">Department of Medical Genome Sciences, Graduate School of Frontier Sciences, the University of Tokyo</div><div class="addr-line">5-1-5 Kashiwanoha, Kashiwa, Chiba 277-8562, Japan</div></div><div class="aff"><div class="label -label">4</div><sup>4</sup><div class="insititution">Human Genome Center, Institute of Medical Science, the University of Tokyo</div><div class="addr-line">4-6-1 Shirokanedai, Minatoku, 108-8639 Tokyo, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kenta+Nakai">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nakai K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nakai K&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Kenta Nakai</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Takashi Ito</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">5</div><sup>5</sup><div class="insititution">Department of Computational Biology, Graduate School of Frontier Sciences, the University of Tokyo</div><div class="addr-line">5-1-5 Kashiwanoha, Kashiwa, Chiba 277-8561, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Takashi+Ito">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ito T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ito T&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Takashi Ito</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Izumu Saito</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><sup>1</sup><div class="insititution">Department of Medical Genome Sciences, Graduate School of Frontier Sciences, the University of Tokyo</div><div class="addr-line">5-1-5 Kashiwanoha, Kashiwa, Chiba 277-8562, Japan</div></div><div class="aff"><div class="label -label">2</div><sup>2</sup><div class="insititution">Graduate School of Medicine, the University of Tokyo</div><div class="addr-line">7-3-1 Hongo, Bunkyoku, Tokyo 113-0033, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Izumu+Saito">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Saito I">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Saito I&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Izumu Saito</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Sumio Sugano</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><sup>1</sup><div class="insititution">Department of Medical Genome Sciences, Graduate School of Frontier Sciences, the University of Tokyo</div><div class="addr-line">5-1-5 Kashiwanoha, Kashiwa, Chiba 277-8562, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Sumio+Sugano">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sugano S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sugano S&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Sumio Sugano</a>
                                    </span>
                                <a class="meta-authors--etal is-active" id="show-meta-authors" aria-controls="remaining-authors" href="#">
                                    <span>...</span> Show more</a>
                                <div id="remaining-authors" class="meta-authors--remaining" aria-hidden="true" aria-labeledby="show-meta-authors">
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yutaka Suzuki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><sup>1</sup><div class="insititution">Department of Medical Genome Sciences, Graduate School of Frontier Sciences, the University of Tokyo</div><div class="addr-line">5-1-5 Kashiwanoha, Kashiwa, Chiba 277-8562, Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <sup>*</sup>To whom correspondence should be addressed. Tel. +81-4-7136-3607, Fax. +81-4-7136-3607, E-mail: <a href="mailto:ysuzuki@hgc.jp" target="_blank">ysuzuki@hgc.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yutaka+Suzuki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Suzuki Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Suzuki Y&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Yutaka Suzuki</a>
                                        </span>
                                </div>
                            </div>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Communicated by Yoichi Taya</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 13, Issue 4, 1 January 2006, Pages 155–167, <a href='https://doi.org/10.1093/dnares/dsl008'>https://doi.org/10.1093/dnares/dsl008</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">01 January 2006</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">24 May 2006</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Revision Received:</div>
                                <div class="wi-date">22 August 2006</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="348476" href="/dnaresearch/article-pdf/13/4/155/998674/dsl008.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Jieun Cheong, Yoichi Yamada, Riu Yamashita, Takuma Irie, Akinori Kanai, Hiroyuki Wakaguri, Kenta Nakai, Takashi Ito, Izumu Saito, Sumio Sugano, Yutaka Suzuki;  Diverse DNA Methylation Statuses at Alternative Promoters of Human Genes in Various Tissues, <em>DNA Research</em>, Volume 13, Issue 4, 1 January 2006, Pages 155–167, <a href='https://doi.org/10.1093/dnares/dsl008'>https://doi.org/10.1093/dnares/dsl008</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/348476?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/348476?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/348476?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/348476?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/348476?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/348476?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/348476?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Diverse%20DNA%20Methylation%20Statuses%20at%20Alternative%20Promoters%20of%20Human%20Genes%20in%20Various%20Tissues&amp;publicationDate=2006-01-01&amp;volumeNum=13&amp;issueNum=4&amp;author=Cheong%2C%20Jieun%3B%20Yamada%2C%20Yoichi&amp;startPage=155&amp;endPage=167&amp;contentId=10.1093%2Fdnares%2Fdsl008&amp;oa=&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f13%2f4%2f155%2f348476" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=4946909 id="4946909" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>We characterized the DNA methylation status at 144 tissue-biased and 37 non-tissue-biased alternative promoters of 61 human genes in five normal tissues. Analysis of the collected data revealed that (i) DNA methylation status differed greatly among alternative promoters belonging to the same gene; (ii) DNA methylation status differed between tissues for the majority of the individual promoters, and (iii) 80–90% of CpG-island-containing promoters were not methylated on either allele throughout the five tissues examined. Furthermore, although the statistical significance was not as clear as for the above features, we also found that (iv) the DNA methylation patterns of tissue-biased promoters changed more drastically than those of non-tissue-biased promoters; (v) tissue-biased promoters tended to be less methylated than their respective alternative promoters in the tissues where they were preferentially expressed, and (vi) the ‘null’ methylation pattern of a given promoter was enriched in the tissues where the transcription was most active. These findings together indicate that there are dynamic physiological changes of DNA methylation. DNA methylation appears to play a significant role in differential usage of alternative promoters and may be related to functional diversification between CpG-island-containing promoters and CpG-island-less promoters.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;DNA methylation status&quot;">DNA methylation status</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;CpG islands&quot;">CpG islands</a></div>                            <h2 scrollto-destination=4946911 id="4946911" class="section-title" >1. Introduction</h2>
<p>Rapidly accumulating large-scale data produced from high-throughput analyses of the genome and transcriptome have recently revealed that the use of alternative promoters is prevalent in human genes.<a reveal-id="b1  b4" data-open="b1  b4" class="link link-ref link-reveal xref-bibr">1–4</a> The alternative use of promoters consisting of different modules of transcriptional regulatory elements enables diversified transcriptional regulation within a single locus.<a reveal-id="b5" data-open="b5" class="link link-ref link-reveal xref-bibr">5</a> It is speculated that multi-faceted use of genes may serve as a molecular basis for the complexity of highly elaborated systems in humans, such as the immune system and nervous system, despite the limited number of total genes.<a reveal-id="b6" data-open="b6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="b7" data-open="b7" class="link link-ref link-reveal xref-bibr">7</a> However, in spite of the growing interest in alternative promoters, very little is known about how and on what occasions those promoters are differentially regulated.</p><p>We have been analyzing the transcription start sites of mRNAs in humans by collective analyses of cDNAs that were isolated from full-length-enriched cDNA libraries constructed by our cap-targeted method, oligo-capping.<a reveal-id="b8  b10" data-open="b8  b10" class="link link-ref link-reveal xref-bibr">8–10</a> By analyzing 1.8 million oligo-cap cDNAs, we recently demonstrated that about half of human protein-coding genes [defined as RefSeq genes (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/RefSeq/" target="_blank"></a>)] have at least two distinct putative alternative promoters (PAPs) which are separated by &gt;500 bp. On average, there were 3.1 alternative promoters per gene. In 17% of cases, tissue-biased utilization of the alternative promoters was observed. The richest tissue sources of such tissue-biased alternative promoters were testis and brain.<a reveal-id="b1" data-open="b1" class="link link-ref link-reveal xref-bibr">1</a></p><p>Interestingly, the identified alternative promoters generally consisted of a combination of one promoter which was located in a CpG island (CGI-containing promoter)<a reveal-id="b11" data-open="b11" class="link link-ref link-reveal xref-bibr">11</a> and other promoter(s) which were not associated with CpG islands (CGI-less promoters). Although about 70% of genes were found to possess at least one CGI-containing promoter, it was rare that a particular gene had more than a single CGI-containing promoter among its alternative promoters. Even when the total number of promoters increased, the number of CGI-containing promoters did not increase proportionally.<a reveal-id="b1" data-open="b1" class="link link-ref link-reveal xref-bibr">1</a> This biased distribution with regard to the presence of CpG islands led us to hypothesize that the transcriptional regulation mediated by DNA methylation might play some role(s) in the differential regulation of the alternative promoters, since the generation and maintenance of the CpG islands have been shown to be closely associated with the methylation of DNAs.</p><p>DNA methylation consists of the addition of a methyl group to the cytosine of the dinucleotide CpG by DNA methyltransferase enzymes in mammals.<a reveal-id="b11" data-open="b11" class="link link-ref link-reveal xref-bibr">11</a> Such methylation generally represses the gene expression of the downstream gene, either by direct interference with the interaction between transcription factors and their binding sites or by altering the chromatin structure of the surrounding genomic region. The regulation of DNA methylation and demethylation is thought to be crucially important during development, and many studies have shown that disruption of proper DNA methylation has profound effects on carcinogenesis.<a reveal-id="b12" data-open="b12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="b13" data-open="b13" class="link link-ref link-reveal xref-bibr">13</a> DNA methylation is thought to be one of the major causative factors for the generation of CpG islands. Methylated cytosine can be converted to thymine by deamination, and thus, in the absence of selection pressure, CpG dinucleotides are rapidly erased from genomic sequences. Actually, CpG dinucleotides are markedly underrepresented throughout the human genome. This reduction of CpG does not occur in genome regions that are devoid of methylation. Rather, CpG dinucleotides are clustered as CpG ‘islands’, which are known to be especially abundant around promoter regions, and are thought to play an important role in proper transcriptional regulation.<a reveal-id="b14  b16" data-open="b14  b16" class="link link-ref link-reveal xref-bibr">14–16</a></p><p>In this study, in order to clarify how the differential regulation of alternative promoters is realized under various cellular circumstances, we analyzed the DNA methylation status of alternative promoters and compared the findings with the tissue-usage-bias of the promoters. Among our collection of alternative promoters, we selected genes with alternative promoters whose usage was biased between tissues, since differential promoter usage was most marked in this population. We characterized the DNA methylation status of the genomic regions proximal to the transcriptional start sites (−1 kb to +200 bp; transcriptional start site: 0) for 181 alternative promoters in 61 genes in five normal tissues in total. Here we describe our first study of the tissue-profile of DNA methylation in a wide variety of normal tissues.</p>                            <h2 scrollto-destination=4946917 id="4946917" class="section-title" >2. Materials and methods</h2>
                            <h3 scrollto-destination=4946918 id="4946918" class="section-title" >2.1. Primer design</h3>
<p>PCR primers were designed using Primer3 (<a class="link link-uri" href="http://fokker.wi.mit.edu/cgi-bin/primer3/primer3_www.cgi" target="_blank"></a>). For primers for HpaII-McrBC PCRs, the parameters were set as: primer length: 20–25 bp; melting temperature: 50–60°C; target length: 1 kb; and for primers for bisulfite genomic sequencing, the parameters were set as: 20–30 bp; melting temperature: 50–70°C; target length: 300–600 bp. The RT–PCR primers were localized inside of the first exons that were unique to the respective alternative promoters using the following parameters: primer length: 20–25 bp; melting temperature: 50–60°C; target length: 50–200 bp. The sequence of each of the designed primers is shown in Supplementary Table S1. Also note that, since in almost all cases, each of the alternative promoters contained multiple transcriptional start sites, the sites used to excise the upstream putative promoters were the sites where the largest number of the 5′-ends of the oligo-cap cDNAs were mapped. The human genome information was in accord with UCSC hg17 (<a class="link link-uri" href="http://genome.ucsc.edu/" target="_blank"></a>) except for the locations of CpG islands (see below).</p>                            <h3 scrollto-destination=4946920 id="4946920" class="section-title" >2.2. HpaII-McrBC PCR assay</h3>
<p>Normal human genomic DNAs were obtained from BioChain (BioChain; USA). Each genomic DNA sample was incubated in 1 M NaCl solution overnight at 37°C. The genomic DNAs were purified by phenol-chloroform extraction and ethanol precipitation and dissolved in distilled water. Aliquots (5–15 μg) were digested with 60 U of <em>Hap</em>II (Takara; Japan), <em>Msp</em>I (Takara; Japan) or McrBC (New England Biolabs; USA) overnight at 37°C in 100 μl of reaction mixture following the instructions of the manufacturer. To assure complete digestion, the digestion step was repeated once more. Control samples were treated in the same way without the addition of the restriction enzyme. After incubation, digested samples were purified by phenol–chloroform extraction and ethanol precipitation and dissolved in distilled water.</p><p>For each digestion, the amount of undigested DNA was measured by Quantitative (Real Time) PCR using 7900HT (ABI; USA). For PCR, 1.0 μl (20–50 ng) of digested genomic DNA was used in a 20 μl reaction solution containing: 10 μl of SYBR GREEN PCR Master mix (ABI; USA): 0.125 pmol of each primer: 4 μl of distilled water. The PCR conditions were 50°C for 2 min, 95°C for 10 min, 5 cycles of: 95°C for 30 s, 58°C for 30 s, 60°C for 1 min, 5 cycles of: 95°C for 30 s, 56°C for 30 s, 60°C for 1 min, 30 cycles of: 95°C for 30 s, 55°C for 30 s, 60°C for 1 min, and 60°C for 5 min. The integrity of the PCR products (lack of primer-dimers or non-specific amplicons) was confirmed by agarose gel electrophoresis. The integrity was also assured by measuring the monomial of the dissociation curve of the PCR products using Dissociation Curve Analysis in SDS Analytic Tools (ABI; USA).</p>                            <h3 scrollto-destination=4946923 id="4946923" class="section-title" >2.3. Bisulfite genomic sequencing</h3>
<p>Five micrograms of the normal human genomic DNAs (BioChain; USA) were treated in a 600 μl sodium bisulfite reaction solution containing: 10 mM hydroquinone, 8 M urea and 3.6 M bisulfite (Sigma; USA) at pH 5.0 (pH adjustment with 10 M NaOH) at 55°C overnight, according to the standard procedure.<a reveal-id="b17" data-open="b17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="b18" data-open="b18" class="link link-ref link-reveal xref-bibr">18</a> Fifty nanograms of the bisulfite-treated DNA was used for PCR amplification in a 10 μl reaction mixture containing: 10 nM Tris–HCl at pH 7.5, 50 mM KCl, 1.5 mM MgCl<sub>2</sub>, 1 mM DTT, 10 mM 2-mercaptoethanol, each dNTP at 0.25 mM, each primer at 0.25 μM and 2.5 U of Ex-Taq DNA polymerase (TaKaRa; Japan). The PCR conditions were: 95°C for 3 min, 5 cycles of 95°C for 30 s, 55°C for 30 s, 72°C for 1 min, 5 cycles of 95°C for 30 s, 52°C for 30 s, 72°C for 1 min, 35 cycles of 95°C for 30 s, 50°C for 30 s, 72°C for 1 min and a final incubation at 72°C for 5 min. The amplified PCR products were purified using the Wizard DNA Clean-up system (Promega; USA) and were TA-cloned into pCR2.1 vector (Invitrogen; USA). Ten individual clones were sequenced from both ends using a 3730 ABI sequencer according to the standard protocol (ABI; USA).</p>                            <h3 scrollto-destination=4946925 id="4946925" class="section-title" >2.4. Data processing</h3>
<div class="block-child-p">Bias in the usage of the alternative promoters between tissues (<a reveal-id="tbl1" data-open="tbl1" class="link link-reveal link-table xref-fig">Table 1</a>) and statistical significance of the enrichments (Tables 3 and 4 and Fig. 3B) were evaluated by calculating hyper-geometric distributions using the following equation: <a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mstyle displaystyle="true"><munderover><mo>∑</mo><mrow><mi>x</mi><mo>=</mo><mi>i</mi></mrow><mrow><mi>N</mi><mi>p</mi></mrow></munderover></mstyle><mrow><mfrac><mrow><mrow><mo>(</mo><mrow><mtable><mtr><mtd><mrow><mi>N</mi><mi>p</mi></mrow></mtd></mtr><mtr><mtd><mi>x</mi></mtd></mtr></mtable></mrow><mo>)</mo></mrow><mo> </mo><mo>·</mo><mo> </mo><mrow><mo>(</mo><mrow><mtable><mtr><mtd><mrow><mi>N</mi><mo>(</mo><mn>1</mn><mo>−</mo><mi>p</mi><mo>)</mo></mrow></mtd></mtr><mtr><mtd><mrow><mi>n</mi><mo>−</mo><mi>x</mi></mrow></mtd></mtr></mtable></mrow><mo>)</mo></mrow></mrow><mrow><mrow><mo>(</mo><mrow><mtable><mtr><mtd><mi>N</mi></mtd></mtr><mtr><mtd><mi>n</mi></mtd></mtr></mtable></mrow><mo>)</mo></mrow></mrow></mfrac></mrow><mo>,</mo></mrow></math></div></div> where <em>N</em> = total number of data, <em>p</em> = expected frequency of the feature of concern estimated from the overall data (number of feature-associated data/<em>N</em>), <em>n</em> = total number of data belonging to the subgroup, and <em>i</em> = number of the data which belong to the subgroup and which are associated with the feature of concern within the subgroup. For example, for <a reveal-id="tbl1" data-open="tbl1" class="link link-reveal link-table xref-fig">Table 1</a>: <em>N</em> = total number of cDNAs belonging to the locus, <em>p</em> = expected frequency of the cDNAs associated with a given tissue within the locus (number of associated cDNAs/N), <em>n</em> = total number of cDNAs belonging to the PAP of concern, and <em>i</em> = number of the cDNAs associated with a given tissue within the PAP.</div>                        <a id="4946927" scrollto-destination="4946927"></a>
<div content-id="tbl1" class="table-modal"><div class="table-wrap"><div id="tbl1" class="table-wrap-title" data-id="tbl1"><div class="label -label"><strong>Table 1</strong></div><div class="caption"><p>(A) Statistics of the target promoters: Number of PAPs classified by the tissue sources of their tissue-biases (upper line) and the number of genes containing the PAPs with the indicated tissue-biases (lower line), (B) Frequency of the CGI-containing PAPs among each population.</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>(A) Testis </th><th>Brain </th><th>Spleen </th><th>Stomach </th><th>Liver </th><th>Others </th><th>Non-specific </th><th>Total </th></tr></thead><tbody><tr><td>Promoters </td><td>22 </td><td>52 </td><td>4 </td><td>0 </td><td>1 </td><td>81 </td><td>21 </td><td>181 </td></tr><tr><td>Genes </td><td>19 </td><td>35 </td><td>4 </td><td>0 </td><td>1 </td><td>44 </td><td>17 </td><td>61 </td></tr></tbody></table></div><div content-id="tbl1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>(A) Testis </th><th>Brain </th><th>Spleen </th><th>Stomach </th><th>Liver </th><th>Others </th><th>Non-specific </th><th>Total </th></tr></thead><tbody><tr><td>Promoters </td><td>22 </td><td>52 </td><td>4 </td><td>0 </td><td>1 </td><td>81 </td><td>21 </td><td>181 </td></tr><tr><td>Genes </td><td>19 </td><td>35 </td><td>4 </td><td>0 </td><td>1 </td><td>44 </td><td>17 </td><td>61 </td></tr></tbody></table></div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>(B) CGI-containing PAPs </th><th>CGI-less PAPs </th><th>Total </th></tr></thead><tbody><tr><td>Tissue-biased </td><td>49 (34%) </td><td>95 (66%) </td><td>144 (100%) </td></tr><tr><td>Non-tissue-biased </td><td>10 (27%) </td><td>27 (73%) </td><td>37 (100%) </td></tr><tr><td>Total </td><td>59 (33%) </td><td>122 (67%) </td><td>181 (100%) </td></tr></tbody></table></div><div content-id="tbl1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>(B) CGI-containing PAPs </th><th>CGI-less PAPs </th><th>Total </th></tr></thead><tbody><tr><td>Tissue-biased </td><td>49 (34%) </td><td>95 (66%) </td><td>144 (100%) </td></tr><tr><td>Non-tissue-biased </td><td>10 (27%) </td><td>27 (73%) </td><td>37 (100%) </td></tr><tr><td>Total </td><td>59 (33%) </td><td>122 (67%) </td><td>181 (100%) </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Note that some of the genes contained more than two TB-PAPs, and thus the total number of genes did not correspond to the sum of the numbers counted separately.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4946927" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="tbl1" class="table-wrap-title" data-id="tbl1"><div class="label -label"><strong>Table 1</strong></div><div class="caption"><p>(A) Statistics of the target promoters: Number of PAPs classified by the tissue sources of their tissue-biases (upper line) and the number of genes containing the PAPs with the indicated tissue-biases (lower line), (B) Frequency of the CGI-containing PAPs among each population.</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>(A) Testis </th><th>Brain </th><th>Spleen </th><th>Stomach </th><th>Liver </th><th>Others </th><th>Non-specific </th><th>Total </th></tr></thead><tbody><tr><td>Promoters </td><td>22 </td><td>52 </td><td>4 </td><td>0 </td><td>1 </td><td>81 </td><td>21 </td><td>181 </td></tr><tr><td>Genes </td><td>19 </td><td>35 </td><td>4 </td><td>0 </td><td>1 </td><td>44 </td><td>17 </td><td>61 </td></tr></tbody></table></div><div content-id="tbl1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>(A) Testis </th><th>Brain </th><th>Spleen </th><th>Stomach </th><th>Liver </th><th>Others </th><th>Non-specific </th><th>Total </th></tr></thead><tbody><tr><td>Promoters </td><td>22 </td><td>52 </td><td>4 </td><td>0 </td><td>1 </td><td>81 </td><td>21 </td><td>181 </td></tr><tr><td>Genes </td><td>19 </td><td>35 </td><td>4 </td><td>0 </td><td>1 </td><td>44 </td><td>17 </td><td>61 </td></tr></tbody></table></div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>(B) CGI-containing PAPs </th><th>CGI-less PAPs </th><th>Total </th></tr></thead><tbody><tr><td>Tissue-biased </td><td>49 (34%) </td><td>95 (66%) </td><td>144 (100%) </td></tr><tr><td>Non-tissue-biased </td><td>10 (27%) </td><td>27 (73%) </td><td>37 (100%) </td></tr><tr><td>Total </td><td>59 (33%) </td><td>122 (67%) </td><td>181 (100%) </td></tr></tbody></table></div><div content-id="tbl1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>(B) CGI-containing PAPs </th><th>CGI-less PAPs </th><th>Total </th></tr></thead><tbody><tr><td>Tissue-biased </td><td>49 (34%) </td><td>95 (66%) </td><td>144 (100%) </td></tr><tr><td>Non-tissue-biased </td><td>10 (27%) </td><td>27 (73%) </td><td>37 (100%) </td></tr><tr><td>Total </td><td>59 (33%) </td><td>122 (67%) </td><td>181 (100%) </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Note that some of the genes contained more than two TB-PAPs, and thus the total number of genes did not correspond to the sum of the numbers counted separately.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4946927" target="_blank">View Large</a></div></div><p>CpG islands were defined as a genomic region that was 200 bp or greater in length with a C+G content &gt;50% and a CpG percentage that exceeded by 60% that expected in random sequences.<a reveal-id="b11" data-open="b11" class="link link-ref link-reveal xref-bibr">11</a></p><p>As for the difference in DNA methylation status between tissues or between PAPs belonging to the same genes, ‘Changed’ was defined as meaning that there were at least two different statuses of DNA methylation in the group. ‘Not Changed’ was defined as meaning that there was only one status of DNA methylation in the group. For defining ‘Not Changed’, we required that the data from at least two PAPs were not ‘n.d.’ for analyzing mutual differences between PAPs, and we required at least three PAPs for analyzing tissue profiles.</p>                            <h3 scrollto-destination=4946930 id="4946930" class="section-title" >2.5. Semi-quantitative (real-time) RT–PCR assays</h3>
<div class="block-child-p">Total RNAs (50 μg) of five normal human tissues were obtained from BioChain (BioChain; USA). We synthesized first strand cDNAs with 20 pmol of dT primer (17 mer of dT). The quality and quantity of the obtained RNA were assessed using BioAnalyzer (Agilent; USA). Real-time PCR was performed using the reagents described above. The PCR conditions were: 50°C for 2 min, 95°C for 10 min, and 30 cycles of 95°C for 30 s, 60°C for 1 min. The results from three independent experiments were averaged. The PCR products were also analyzed by agarose gel electrophoresis to assess the integrity (single-band-ness). The relative expression level was evaluated as the deviation of the Ct value (constant of threshold) from the Ct values averaged for the five tissues using the equation: <a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mtext>Expression</mtext><mo>(</mo><mtext>relative</mtext><mo>)</mo><mo>=</mo><msup><mn>2</mn><mrow><mtext>Ct</mtext><mo>/</mo><mtext>Ct</mtext><mo>(</mo><mtext>average</mtext><mo>)</mo></mrow></msup></mrow></math></div></div></div>                            <h2 scrollto-destination=4946932 id="4946932" class="section-title" >3. Results</h2>
                            <h3 scrollto-destination=4946933 id="4946933" class="section-title" >3.1. Selection of alternative promoters</h3>
<p>In our previous study, we generated a collection of 30 707 PAPs in 7674 RefSeq genes.<a reveal-id="b1" data-open="b1" class="link link-ref link-reveal xref-bibr">1</a> Among them, we identified 1731 PAPs in 1287 genes which showed biased usage between tissues or cell types [tissue-biased PAPs (TB-PAPs); <em>P</em> &lt; 0.01; statistical significance was evaluated by calculating hyper-geometric distributions using the formula shown in Materials and Methods]. We further selected the genes that had at least two TB-PAPs containing restriction enzyme sites essential for the <em>Hpa</em>II-McrBC PCR assays in the region from −1 kb to +200 bp of the transcription start sites. As a result, we obtained 61 genes that had 144 PAPs. Also, we selected an additional 37 PAPs belonging to the same genes but showing no tissue bias (non-TB-PAPs). Thus, we selected a total of 181 PAPs in 61 genes as targets for methylation analysis (<a reveal-id="tbl1" data-open="tbl1" class="link link-reveal link-table xref-fig">Table 1</a>). For a schematic representation of the selection flow chart, see Supplementary Figure S1. For detailed information abouteach of the promoters, see Supplementary Table S2. Also, the numbering of the PAPs corresponds to that used in our database, DBTSS (<a class="link link-uri" href="http://dbtss.hgc.jp/" target="_blank"></a>), so that visual inspection can be performed for each of the PAPs.</p><p>Although we selected only 144 (about 8%) TB-PAPs of 1731 for the following assays, we think that this promoter set retains the overall character of TB-PAPs. As described in our previous report, testis and brain were the richest source of the tissue-biased alternative promoters. This was also true for the present set (<a reveal-id="tbl1" data-open="tbl1" class="link link-reveal link-table xref-fig">Table 1</a>). Also, among the total set of 181 promoters, 59 (33%) contained CpG-islands and 122 (67%) were CpG-island-less (<a reveal-id="tbl1" data-open="tbl1" class="link link-reveal link-table xref-fig">Table 1</a>). This frequency was similar to that of the total 30 707 PAPs (CpG-island-containing: 34%, see Ref.<a reveal-id="b1" data-open="b1" class="link link-ref link-reveal xref-bibr">1</a>). We therefore consider the features of the current set of promoters to be representative of the overall features of alternative promoters at least in these respects.</p>                            <h3 scrollto-destination=4946936 id="4946936" class="section-title" >3.2. Measurement of the DNA methylation status of promoters in five normal tissues</h3>
<p>We examined the DNA methylation status using <em>Hpa</em>II-McrBC PCR assays (HM-PCR assays). For each PAP, the DNA methylation status in five normal tissues, namely, brain, testis, spleen, stomach and liver, was determined. According to the observed PCR band patterns, we classified the DNA methylation status into four groups: ‘complete’, ‘composite’, ‘null’ and ‘incomplete’ methylation (abbreviated CL, CP, NL and IC respectively), which are classification categories often employed in previous studies (<a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1A</a>; and see Ref. <a reveal-id="b19" data-open="b19" class="link link-ref link-reveal xref-bibr">19</a>). Typical examples of PCR band patterns assigned to each of the groups are shown in <a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1B</a>.</p>                        <a id="4946938" scrollto-destination="4946938"></a>
<div id="fig1" data-id="fig1" class="fig fig-section"><div class="label fig-label"><strong>Figure 1</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/4/10.1093/dnares/dsl008/2/m_dsl008f1a.jpeg?Expires=2147483647&amp;Signature=rP50X2bdlULxmrZXdy5BULwyvWAid7YHQyqCr7g0eQmGre6Qjr6g~fjbPePs2uDvgK9UfrO7Yb3deI1FPC4Nvd4oCfiESDx7JQx6T8Sn3uoVwYCziUgzyXmH82ALkl-XfxQIKgdB5H19~ssHCmsQ0Ua9KrT9wd3kaQzWImyO4kLzZ9DF6UD8gUa~ZnjO--~R6jvgUTU-iI1zGd1o2g6RF5gt0aZfzVx1r8BoAhtOi7LhNMceibx~t9FsoX7tsO7EW-rl-1ta4LoRI7jDcs6QcaJfq~dKaebuNwJPJ8OqbjAUFx3F2yjlyDNUqzxLrDyozy3ldQ8P-6KVF4uuaj1Fpw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="HpaII-McrBC PCR of the putative ‘tissue-biased’ alternative promoters (TB-PAPs). (A) Schematic representation of the DNA methylation patterns. Open and closed circles represent non-methylated and methylated CpG sites, respectively. (B) Typical examples of the results of HM-PCR assays. The DNA methylation status was determined by digitally examining the presence or absence of the bands in the 3rd and the 4th lanes (1st and 2nd lanes are positive and negative controls, respectively) in each assay. (C) Typical examples of the results of the bisulfite genomic sequencing assays. TSS: transcriptional start site." path-from-xml="dsl008f1a.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4946938/dsl008f1a.jpeg" path-from-xml="dsl008f1a.jpeg" target="_blank">View large</a><a section="4946938" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/4/10.1093/dnares/dsl008/2/dsl008f1a.jpeg?Expires=2147483647&amp;Signature=hr8COvXCyq8R7e~nv9Bmfc8FlIuh7M7MhwIV7SqAi5lXPDmj2nol-7aSuOay-0IA5q40KGn-Z~Sqxw~wRAsTEYDMetTyLakeIYBn~UZjPaWeyUDw-9ACE8NwzxNPaUUM9w50oUf6Lu0OiKuDLz9Pycxi3Nuwt4vxoEtpvrZmbGkuVD4m5r4ceKPL4SeNUumLw0QwoSW3mPJlT49LNbL48MiPzWCIrszpC-Nt6Uk455FNN-uuktzwBlaG0xvw4xTLE8xVfwYiQUqtVkyGDoHSzk41BaMIC-joBVGuL0RbPQW49ViCwmb6pDubmeG3cNXZoD3KLagvInyvsEQCzK-WaQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=4946938&amp;ar=348476&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsl008f1a.jpeg" target="_blank">Download slide</a></div></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/4/10.1093/dnares/dsl008/2/m_dsl008f1c.jpeg?Expires=2147483647&amp;Signature=zIKiw2CWVRQFh4PD8p7DSS-1wJkhPN68LOI9Uy79x7hprIQmPQh~t6tuCh10Tgs83IV3FsZBpx72SH0Yq6dtWpWYl8y0YY-uBdjkBgoTyCyikhD5E1SUwPKFEiyBUyw~6vMqC~iFHr6vzzUoXYmYOT7RwrGr2zVd19ojTKOd-U0pLP9Lb1Vh2bztagahr-3vSWZvwVc2PqiyC1LrN63a2mVbnDyGh-0UAf-Zva0v~KxVoZ4A2jzvQkA6U4HHzLD1Jg2tDLXYMtafsMHhULIsbYF~h~cMUUYegIRdVduVVHUVqMf8jlbhrpxs15PYm5Lg1fd6sMrwayGNG7m4Y4XMiQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="HpaII-McrBC PCR of the putative ‘tissue-biased’ alternative promoters (TB-PAPs). (A) Schematic representation of the DNA methylation patterns. Open and closed circles represent non-methylated and methylated CpG sites, respectively. (B) Typical examples of the results of HM-PCR assays. The DNA methylation status was determined by digitally examining the presence or absence of the bands in the 3rd and the 4th lanes (1st and 2nd lanes are positive and negative controls, respectively) in each assay. (C) Typical examples of the results of the bisulfite genomic sequencing assays. TSS: transcriptional start site." path-from-xml="dsl008f1c.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4946938/dsl008f1c.jpeg" path-from-xml="dsl008f1c.jpeg" target="_blank">View large</a><a section="4946938" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/4/10.1093/dnares/dsl008/2/dsl008f1c.jpeg?Expires=2147483647&amp;Signature=cX2DBeW0e-4fhxoqvFLAMXowBGdntDPeyUaiF1VUIMHllzCFILvOZ21rCimBZZ7WHzfWx0j~pc7oJT0NpyJ33gvKcYY1RTpyySwRGKfmpzCiEV5T-WGPz9dr2ToA7JSlUV0PtTR4jrvMv~dxZnE2aOajF5Y5XRq6vAth5UaS5~fTJTiDX~1P95YjwKA3yINcalwYMENUleIqRPeK8~MzZxwB0nDHCx4IeJaiB7qLmmeEZTaf4vXNuj4jLW592-CYvePzwVUwq9siqiavOXcFYextU3vu6losYO9914YhYRrZS0mU1-bfkhfQJzNdNkcvsvbxa6SwQleuuz2ASQc7MA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=4946938&amp;ar=348476&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsl008f1c.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>HpaII-McrBC PCR of the putative ‘tissue-biased’ alternative promoters (TB-PAPs). (A) Schematic representation of the DNA methylation patterns. Open and closed circles represent non-methylated and methylated CpG sites, respectively. (B) Typical examples of the results of HM-PCR assays. The DNA methylation status was determined by digitally examining the presence or absence of the bands in the 3rd and the 4th lanes (1st and 2nd lanes are positive and negative controls, respectively) in each assay. (C) Typical examples of the results of the bisulfite genomic sequencing assays. TSS: transcriptional start site.</p></div></div><div content-id="fig1" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 1</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/4/10.1093/dnares/dsl008/2/m_dsl008f1a.jpeg?Expires=2147483647&amp;Signature=rP50X2bdlULxmrZXdy5BULwyvWAid7YHQyqCr7g0eQmGre6Qjr6g~fjbPePs2uDvgK9UfrO7Yb3deI1FPC4Nvd4oCfiESDx7JQx6T8Sn3uoVwYCziUgzyXmH82ALkl-XfxQIKgdB5H19~ssHCmsQ0Ua9KrT9wd3kaQzWImyO4kLzZ9DF6UD8gUa~ZnjO--~R6jvgUTU-iI1zGd1o2g6RF5gt0aZfzVx1r8BoAhtOi7LhNMceibx~t9FsoX7tsO7EW-rl-1ta4LoRI7jDcs6QcaJfq~dKaebuNwJPJ8OqbjAUFx3F2yjlyDNUqzxLrDyozy3ldQ8P-6KVF4uuaj1Fpw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="HpaII-McrBC PCR of the putative ‘tissue-biased’ alternative promoters (TB-PAPs). (A) Schematic representation of the DNA methylation patterns. Open and closed circles represent non-methylated and methylated CpG sites, respectively. (B) Typical examples of the results of HM-PCR assays. The DNA methylation status was determined by digitally examining the presence or absence of the bands in the 3rd and the 4th lanes (1st and 2nd lanes are positive and negative controls, respectively) in each assay. (C) Typical examples of the results of the bisulfite genomic sequencing assays. TSS: transcriptional start site." path-from-xml="dsl008f1a.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4946938/dsl008f1a.jpeg" path-from-xml="dsl008f1a.jpeg" target="_blank">View large</a><a section="4946938" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/4/10.1093/dnares/dsl008/2/dsl008f1a.jpeg?Expires=2147483647&amp;Signature=hr8COvXCyq8R7e~nv9Bmfc8FlIuh7M7MhwIV7SqAi5lXPDmj2nol-7aSuOay-0IA5q40KGn-Z~Sqxw~wRAsTEYDMetTyLakeIYBn~UZjPaWeyUDw-9ACE8NwzxNPaUUM9w50oUf6Lu0OiKuDLz9Pycxi3Nuwt4vxoEtpvrZmbGkuVD4m5r4ceKPL4SeNUumLw0QwoSW3mPJlT49LNbL48MiPzWCIrszpC-Nt6Uk455FNN-uuktzwBlaG0xvw4xTLE8xVfwYiQUqtVkyGDoHSzk41BaMIC-joBVGuL0RbPQW49ViCwmb6pDubmeG3cNXZoD3KLagvInyvsEQCzK-WaQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=4946938&amp;ar=348476&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsl008f1a.jpeg" target="_blank">Download slide</a></div></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/4/10.1093/dnares/dsl008/2/m_dsl008f1c.jpeg?Expires=2147483647&amp;Signature=zIKiw2CWVRQFh4PD8p7DSS-1wJkhPN68LOI9Uy79x7hprIQmPQh~t6tuCh10Tgs83IV3FsZBpx72SH0Yq6dtWpWYl8y0YY-uBdjkBgoTyCyikhD5E1SUwPKFEiyBUyw~6vMqC~iFHr6vzzUoXYmYOT7RwrGr2zVd19ojTKOd-U0pLP9Lb1Vh2bztagahr-3vSWZvwVc2PqiyC1LrN63a2mVbnDyGh-0UAf-Zva0v~KxVoZ4A2jzvQkA6U4HHzLD1Jg2tDLXYMtafsMHhULIsbYF~h~cMUUYegIRdVduVVHUVqMf8jlbhrpxs15PYm5Lg1fd6sMrwayGNG7m4Y4XMiQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="HpaII-McrBC PCR of the putative ‘tissue-biased’ alternative promoters (TB-PAPs). (A) Schematic representation of the DNA methylation patterns. Open and closed circles represent non-methylated and methylated CpG sites, respectively. (B) Typical examples of the results of HM-PCR assays. The DNA methylation status was determined by digitally examining the presence or absence of the bands in the 3rd and the 4th lanes (1st and 2nd lanes are positive and negative controls, respectively) in each assay. (C) Typical examples of the results of the bisulfite genomic sequencing assays. TSS: transcriptional start site." path-from-xml="dsl008f1c.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4946938/dsl008f1c.jpeg" path-from-xml="dsl008f1c.jpeg" target="_blank">View large</a><a section="4946938" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/4/10.1093/dnares/dsl008/2/dsl008f1c.jpeg?Expires=2147483647&amp;Signature=cX2DBeW0e-4fhxoqvFLAMXowBGdntDPeyUaiF1VUIMHllzCFILvOZ21rCimBZZ7WHzfWx0j~pc7oJT0NpyJ33gvKcYY1RTpyySwRGKfmpzCiEV5T-WGPz9dr2ToA7JSlUV0PtTR4jrvMv~dxZnE2aOajF5Y5XRq6vAth5UaS5~fTJTiDX~1P95YjwKA3yINcalwYMENUleIqRPeK8~MzZxwB0nDHCx4IeJaiB7qLmmeEZTaf4vXNuj4jLW592-CYvePzwVUwq9siqiavOXcFYextU3vu6losYO9914YhYRrZS0mU1-bfkhfQJzNdNkcvsvbxa6SwQleuuz2ASQc7MA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=4946938&amp;ar=348476&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsl008f1c.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>HpaII-McrBC PCR of the putative ‘tissue-biased’ alternative promoters (TB-PAPs). (A) Schematic representation of the DNA methylation patterns. Open and closed circles represent non-methylated and methylated CpG sites, respectively. (B) Typical examples of the results of HM-PCR assays. The DNA methylation status was determined by digitally examining the presence or absence of the bands in the 3rd and the 4th lanes (1st and 2nd lanes are positive and negative controls, respectively) in each assay. (C) Typical examples of the results of the bisulfite genomic sequencing assays. TSS: transcriptional start site.</p></div></div><p>We observed 138 (24%) ‘complete’, 264 (45%) ‘null’, 165 (28%) ‘composite’, and 16 (3%) ‘incomplete’ methylations among the 144 TB-PAPs (<a reveal-id="tbl2" data-open="tbl2" class="link link-reveal link-table xref-fig">Table 2</a>). For these 583 cases, the classification was entirely straightforward based on the PCR band patterns (the presence or absence of the PCR bands; see <a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1B</a> and the legend). Moreover, we used real-time PCR for the quantification of the band amplification. Thus, we could assess the extent of the presence or absence of the bands in a quantitative manner (for the quantitative data, see Supplementary Figure S2). In 137 (15% of a total of 905) cases, the band patterns were ambiguous (or a mixture) between the two categories, and thus they were classified as ‘not determined (n.d.)’. (For further discussion of this issue, see Supplementary Figure S4). For the 37 non-TB-PAPs, we classified 37 (24%), 64 (42%), 47 (31%), 5 (3%) and 32 (18% of total 181) promoters as ‘complete’, ‘null’, ‘composite’, ‘incomplete’ and ‘n.d.’, respectively (<a reveal-id="tbl2" data-open="tbl2" class="link link-reveal link-table xref-fig">Table 2</a>). It was interesting that ‘composite’, which is the category in which one of the two alleles is completely methylated and the other is null methylated, was observed far more frequently than ‘incomplete’, which is the category in which both alleles are partially methylated. This suggests that the gene expression statuses from two alleles might differ for a significant population of genes, although it is also possible that the presence of mixed populations of multiple cell types within the used tissues accounted for this observation (Note that the ‘composite’ pattern can raise from cell-type difference, not from allelic difference. If both alleles are completely methylated in certain cell types but completely unmethylated in other cell types in the same tissue, this can give a composite pattern. For extensive analysis regarding this issue, see Supplementary Figure S5). This general tendency was also observed for both tissue-biased and non-tissue-biased PAPs. Altogether, we obtained data explicitly representing the DNA methylation status of 736 proximal promoters. This dataset is the largest so far collected for the DNA methylation status at promoters in normal tissues.</p>                        <a id="4946940" scrollto-destination="4946940"></a>
<div content-id="tbl2" class="table-modal"><div class="table-wrap"><div id="tbl2" class="table-wrap-title" data-id="tbl2"><div class="label -label"><strong>Table 2</strong></div><div class="caption"><p>DNA methylation status of the TB-PAPs.</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Testis </th><th>Brain </th><th>Spleen </th><th>Stomach </th><th>Liver </th></tr></thead><tbody><tr><td colspan="6">(A) Tissue-biased alternative promoters (TB-PAPs) </td></tr><tr><td>    Complete </td><td>31 (28%) </td><td>33 (27%) </td><td>42 (34%) </td><td>12 (11%) </td><td>20 (17%) </td></tr><tr><td>    Null </td><td>47 (42%) </td><td>48 (39%) </td><td>50 (41%) </td><td>52 (46%) </td><td>67 (58%) </td></tr><tr><td>    Composite </td><td>30 (27%) </td><td>39 (32%) </td><td>27 (22%) </td><td>44 (39%) </td><td>25 (22%) </td></tr><tr><td>    Incomplete </td><td>4 (3%) </td><td>2 (2%) </td><td>3 (3%) </td><td>4 (4%) </td><td>3 (3%) </td></tr><tr><td>    Subtotal </td><td>112 (100%) </td><td>122 (100%) </td><td>122 (100%) </td><td>112 (100%) </td><td>115 (100%) </td></tr><tr><td>    n.d. </td><td>32 </td><td>22 </td><td>22 </td><td>32 </td><td>29 </td></tr><tr><td>    Total </td><td>144 </td><td>144 </td><td>144 </td><td>144 </td><td>144 </td></tr><tr><td colspan="6">(B) Non-tissue-biased alternative promoters (non-TB-PAPs) </td></tr><tr><td>    Complete </td><td>8 (27%) </td><td>6 (21%) </td><td>9 (28%) </td><td>4 (13%) </td><td>10 (31%) </td></tr><tr><td>    Null </td><td>11 (37%) </td><td>13 (45%) </td><td>12 (38%) </td><td>12 (40%) </td><td>16 (50%) </td></tr><tr><td>    Composite </td><td>10 (33%) </td><td>10 (34%) </td><td>11 (34%) </td><td>12 (40%) </td><td>4 (13%) </td></tr><tr><td>    Incomplete </td><td>1 (3%) </td><td>0 (0%) </td><td>0 (0%) </td><td>2 (7%) </td><td>2 (6%) </td></tr><tr><td>    Subtotal </td><td>30 (100%) </td><td>29 (100%) </td><td>32 (100%) </td><td>30 (100%) </td><td>32 (100%) </td></tr><tr><td>    n.d. </td><td>7 </td><td>8 </td><td>5 </td><td>7 </td><td>5 </td></tr><tr><td>    Total </td><td>37 </td><td>37 </td><td>37 </td><td>37 </td><td>37 </td></tr></tbody></table></div><div content-id="tbl2" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Testis </th><th>Brain </th><th>Spleen </th><th>Stomach </th><th>Liver </th></tr></thead><tbody><tr><td colspan="6">(A) Tissue-biased alternative promoters (TB-PAPs) </td></tr><tr><td>    Complete </td><td>31 (28%) </td><td>33 (27%) </td><td>42 (34%) </td><td>12 (11%) </td><td>20 (17%) </td></tr><tr><td>    Null </td><td>47 (42%) </td><td>48 (39%) </td><td>50 (41%) </td><td>52 (46%) </td><td>67 (58%) </td></tr><tr><td>    Composite </td><td>30 (27%) </td><td>39 (32%) </td><td>27 (22%) </td><td>44 (39%) </td><td>25 (22%) </td></tr><tr><td>    Incomplete </td><td>4 (3%) </td><td>2 (2%) </td><td>3 (3%) </td><td>4 (4%) </td><td>3 (3%) </td></tr><tr><td>    Subtotal </td><td>112 (100%) </td><td>122 (100%) </td><td>122 (100%) </td><td>112 (100%) </td><td>115 (100%) </td></tr><tr><td>    n.d. </td><td>32 </td><td>22 </td><td>22 </td><td>32 </td><td>29 </td></tr><tr><td>    Total </td><td>144 </td><td>144 </td><td>144 </td><td>144 </td><td>144 </td></tr><tr><td colspan="6">(B) Non-tissue-biased alternative promoters (non-TB-PAPs) </td></tr><tr><td>    Complete </td><td>8 (27%) </td><td>6 (21%) </td><td>9 (28%) </td><td>4 (13%) </td><td>10 (31%) </td></tr><tr><td>    Null </td><td>11 (37%) </td><td>13 (45%) </td><td>12 (38%) </td><td>12 (40%) </td><td>16 (50%) </td></tr><tr><td>    Composite </td><td>10 (33%) </td><td>10 (34%) </td><td>11 (34%) </td><td>12 (40%) </td><td>4 (13%) </td></tr><tr><td>    Incomplete </td><td>1 (3%) </td><td>0 (0%) </td><td>0 (0%) </td><td>2 (7%) </td><td>2 (6%) </td></tr><tr><td>    Subtotal </td><td>30 (100%) </td><td>29 (100%) </td><td>32 (100%) </td><td>30 (100%) </td><td>32 (100%) </td></tr><tr><td>    n.d. </td><td>7 </td><td>8 </td><td>5 </td><td>7 </td><td>5 </td></tr><tr><td>    Total </td><td>37 </td><td>37 </td><td>37 </td><td>37 </td><td>37 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>The numbers of PAPs whose DNA methylation status was categorized as the corresponding patterns in the indicated tissue are shown. Frequencies of the observed patterns were calculated relative to the subtotal, which excluded “n.d.”, and are shown in parentheses.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4946940" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="tbl2" class="table-wrap-title" data-id="tbl2"><div class="label -label"><strong>Table 2</strong></div><div class="caption"><p>DNA methylation status of the TB-PAPs.</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Testis </th><th>Brain </th><th>Spleen </th><th>Stomach </th><th>Liver </th></tr></thead><tbody><tr><td colspan="6">(A) Tissue-biased alternative promoters (TB-PAPs) </td></tr><tr><td>    Complete </td><td>31 (28%) </td><td>33 (27%) </td><td>42 (34%) </td><td>12 (11%) </td><td>20 (17%) </td></tr><tr><td>    Null </td><td>47 (42%) </td><td>48 (39%) </td><td>50 (41%) </td><td>52 (46%) </td><td>67 (58%) </td></tr><tr><td>    Composite </td><td>30 (27%) </td><td>39 (32%) </td><td>27 (22%) </td><td>44 (39%) </td><td>25 (22%) </td></tr><tr><td>    Incomplete </td><td>4 (3%) </td><td>2 (2%) </td><td>3 (3%) </td><td>4 (4%) </td><td>3 (3%) </td></tr><tr><td>    Subtotal </td><td>112 (100%) </td><td>122 (100%) </td><td>122 (100%) </td><td>112 (100%) </td><td>115 (100%) </td></tr><tr><td>    n.d. </td><td>32 </td><td>22 </td><td>22 </td><td>32 </td><td>29 </td></tr><tr><td>    Total </td><td>144 </td><td>144 </td><td>144 </td><td>144 </td><td>144 </td></tr><tr><td colspan="6">(B) Non-tissue-biased alternative promoters (non-TB-PAPs) </td></tr><tr><td>    Complete </td><td>8 (27%) </td><td>6 (21%) </td><td>9 (28%) </td><td>4 (13%) </td><td>10 (31%) </td></tr><tr><td>    Null </td><td>11 (37%) </td><td>13 (45%) </td><td>12 (38%) </td><td>12 (40%) </td><td>16 (50%) </td></tr><tr><td>    Composite </td><td>10 (33%) </td><td>10 (34%) </td><td>11 (34%) </td><td>12 (40%) </td><td>4 (13%) </td></tr><tr><td>    Incomplete </td><td>1 (3%) </td><td>0 (0%) </td><td>0 (0%) </td><td>2 (7%) </td><td>2 (6%) </td></tr><tr><td>    Subtotal </td><td>30 (100%) </td><td>29 (100%) </td><td>32 (100%) </td><td>30 (100%) </td><td>32 (100%) </td></tr><tr><td>    n.d. </td><td>7 </td><td>8 </td><td>5 </td><td>7 </td><td>5 </td></tr><tr><td>    Total </td><td>37 </td><td>37 </td><td>37 </td><td>37 </td><td>37 </td></tr></tbody></table></div><div content-id="tbl2" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Testis </th><th>Brain </th><th>Spleen </th><th>Stomach </th><th>Liver </th></tr></thead><tbody><tr><td colspan="6">(A) Tissue-biased alternative promoters (TB-PAPs) </td></tr><tr><td>    Complete </td><td>31 (28%) </td><td>33 (27%) </td><td>42 (34%) </td><td>12 (11%) </td><td>20 (17%) </td></tr><tr><td>    Null </td><td>47 (42%) </td><td>48 (39%) </td><td>50 (41%) </td><td>52 (46%) </td><td>67 (58%) </td></tr><tr><td>    Composite </td><td>30 (27%) </td><td>39 (32%) </td><td>27 (22%) </td><td>44 (39%) </td><td>25 (22%) </td></tr><tr><td>    Incomplete </td><td>4 (3%) </td><td>2 (2%) </td><td>3 (3%) </td><td>4 (4%) </td><td>3 (3%) </td></tr><tr><td>    Subtotal </td><td>112 (100%) </td><td>122 (100%) </td><td>122 (100%) </td><td>112 (100%) </td><td>115 (100%) </td></tr><tr><td>    n.d. </td><td>32 </td><td>22 </td><td>22 </td><td>32 </td><td>29 </td></tr><tr><td>    Total </td><td>144 </td><td>144 </td><td>144 </td><td>144 </td><td>144 </td></tr><tr><td colspan="6">(B) Non-tissue-biased alternative promoters (non-TB-PAPs) </td></tr><tr><td>    Complete </td><td>8 (27%) </td><td>6 (21%) </td><td>9 (28%) </td><td>4 (13%) </td><td>10 (31%) </td></tr><tr><td>    Null </td><td>11 (37%) </td><td>13 (45%) </td><td>12 (38%) </td><td>12 (40%) </td><td>16 (50%) </td></tr><tr><td>    Composite </td><td>10 (33%) </td><td>10 (34%) </td><td>11 (34%) </td><td>12 (40%) </td><td>4 (13%) </td></tr><tr><td>    Incomplete </td><td>1 (3%) </td><td>0 (0%) </td><td>0 (0%) </td><td>2 (7%) </td><td>2 (6%) </td></tr><tr><td>    Subtotal </td><td>30 (100%) </td><td>29 (100%) </td><td>32 (100%) </td><td>30 (100%) </td><td>32 (100%) </td></tr><tr><td>    n.d. </td><td>7 </td><td>8 </td><td>5 </td><td>7 </td><td>5 </td></tr><tr><td>    Total </td><td>37 </td><td>37 </td><td>37 </td><td>37 </td><td>37 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>The numbers of PAPs whose DNA methylation status was categorized as the corresponding patterns in the indicated tissue are shown. Frequencies of the observed patterns were calculated relative to the subtotal, which excluded “n.d.”, and are shown in parentheses.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4946940" target="_blank">View Large</a></div></div><p>We further confirmed the results obtained from the HM-PCR assays by bisulfite genomic sequencing assays. Since the bisulfite genomic sequencing assays could not be scaled up to high-throughput assays, we selected 5, 4, 15, 11 cases as representative cases randomly selected from the cases categorized as ‘complete’, ‘composite’, ‘null’ and ‘incomplete’, respectively, by HM-PCR assays (<a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1A</a>; results from 35 PAPs are presented in Supplementary Table S3). In all cases, the classifications obtained from the two assays were consistent with each other. This strong agreement between the two assays provides support for our conclusion real-time PCR was useful for quantitatively discriminating distinct DNA methylation patterns in the HM-PCR assay.</p>                            <h3 scrollto-destination=4946942 id="4946942" class="section-title" >3.3. Variable methylation patterns between alternative promoters and between tissues</h3>
<p>At first glance, we were surprised by the variable pattern of DNA methylation at the alternative promoters. <a reveal-id="fig2" data-open="fig2" class="link link-reveal link-table xref-fig">Fig. 2</a> illustrates the case of the SEPT9 gene as an extreme example. The SEPT9 gene was identified as a causative gene of an autosomal dominant hereditary disease, neuralgic amyotrophy, and is thought to play roles in cell proliferation.<a reveal-id="b20" data-open="b20" class="link link-ref link-reveal xref-bibr">20</a> This gene possessed at least 20 alternative promoters, represented by an array of 20 mutually exclusive first exon variants (<a reveal-id="fig2" data-open="fig2" class="link link-reveal link-table xref-fig">Fig. 2A</a>). Of these, the DNA methylation status was determined for 14 in this study (<a reveal-id="fig2" data-open="fig2" class="link link-reveal link-table xref-fig">Fig. 2B</a>). A surprising aspect of the data is that no two PAPs in <a reveal-id="fig2" data-open="fig2" class="link link-reveal link-table xref-fig">Fig. 2B</a> share the same methylation pattern variation among the five tissues. For example, PAP-ID1 and 2 are separated by only 800 bases. Yet, ID1 showed the CP-CP-CL-CP-NL pattern, while ID2 showed the NL-NL-NL-NL-NL pattern. PAP-ID10 showed the CP-CL-CL-CP-CP pattern, which was similar to the pattern of PAP-ID11, namely, CP-CL-CP-CP-CP. However, these two promoters are separated by 4 kb. These observations suggest that the DNA methylation pattern may be unique for each PAP even within the same gene. This could allow very fine tuning of the gene expression. Only PAP-ID5 and PAP-ID13 showed a uniform ‘composite’ methylation pattern among the five tissues (although surprisingly, PAP-ID5 and PAP-ID13 are separated by 100 kb). PAP-ID2 also showed a uniform ‘null’ methylation pattern. For the remaining 11 PAPs (79% of the 14 PAPs), the methylation pattern clearly varied among the five tissues. Thus, variable methylation patterns among the tissues may be more common than a uniform pattern.</p>                        <a id="4946944" scrollto-destination="4946944"></a>
<div id="fig2" data-id="fig2" class="fig fig-section"><div class="label fig-label"><strong>Figure 2</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/4/10.1093/dnares/dsl008/2/m_dsl008f2.jpeg?Expires=2147483647&Signature=1B0eR2l-cmJMOH8-EIk29653TzAAUf7sw8NPuZAxnEH4oVkGxpFEu5FEywQS~6BjFRn6teURCelyM9rVGWEkVUx-tkv6JwLPSa3uLjJf52leIe5pasD6eJ8Hc4ftDL~B~X31IhOHk4RM0C8VB1FF2QXf9t61fiBkQ9XOBXGxxbEXwTfg3afnme64kKf173LVi2dDqFPX4ZARIqowA61kxYId1bSbtDf0Tk5WmGoeXfz96JH5JnG57hoIQ1TZMHDGKb~Kxqds3BNT3fTCmcgurFbWMTK47Z9a2YvuWjbJFUIthNLRTYoCABoGroe8CrcQiCSv~BYjWRC4qzvxccYkOQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="DNA Methylation Status at the PAPs of the SEPT9 gene. (A) Schematic representation of the structure of the SEPT9 gene. Fourteen PAPs which were selected from 20 and were subjected to the analyses are marked with asterisks. (B) Observed patterns of DNA methylation at each of the PAPs in each of the tissues. In the 2nd column CGI-containing PAPs are represented as 1, and CGI-less PAPs as 0. In the 3rd column, tissue-preferences of the PAPs are shown." path-from-xml="dsl008f2.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4946944/dsl008f2.jpeg" path-from-xml="dsl008f2.jpeg" target="_blank">View large</a><a section="4946944" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/4/10.1093/dnares/dsl008/2/dsl008f2.jpeg?Expires=2147483647&Signature=VaWki5uwZswoyr8CK2Dkm-klFk-IGAtu9Lnie62CCt9w5DppGXmNf6pDDbw~Sqgrz8wv~lR~0wYce8zV--AA8JpusMvES9VVJ1sVEgsj2T79uBR824UQBPZ0YhmswTDpiaP-VT6RPx0-94y3rWP1TJv86A7XygIHCrTTJs2WYLiNRhb0-f51snw~tS-EohEGbewGXbdk7~uOPqqlZtkISrg2lut18K3MdfDGyuXG9ScwCqfKAzIVzAflodYbhmSJThHs~n2ovqISeCzLVUMremY2-ddtxWkJ43w7SQjevaoehwfLnH~lkSVayM~5zz726iLio5qm3xOITtob4q3GrA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4946944&ar=348476&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl008f2.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>DNA Methylation Status at the PAPs of the SEPT9 gene. (A) Schematic representation of the structure of the SEPT9 gene. Fourteen PAPs which were selected from 20 and were subjected to the analyses are marked with asterisks. (B) Observed patterns of DNA methylation at each of the PAPs in each of the tissues. In the 2nd column CGI-containing PAPs are represented as 1, and CGI-less PAPs as 0. In the 3rd column, tissue-preferences of the PAPs are shown.</p></div></div><div content-id="fig2" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 2</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/4/10.1093/dnares/dsl008/2/m_dsl008f2.jpeg?Expires=2147483647&Signature=1B0eR2l-cmJMOH8-EIk29653TzAAUf7sw8NPuZAxnEH4oVkGxpFEu5FEywQS~6BjFRn6teURCelyM9rVGWEkVUx-tkv6JwLPSa3uLjJf52leIe5pasD6eJ8Hc4ftDL~B~X31IhOHk4RM0C8VB1FF2QXf9t61fiBkQ9XOBXGxxbEXwTfg3afnme64kKf173LVi2dDqFPX4ZARIqowA61kxYId1bSbtDf0Tk5WmGoeXfz96JH5JnG57hoIQ1TZMHDGKb~Kxqds3BNT3fTCmcgurFbWMTK47Z9a2YvuWjbJFUIthNLRTYoCABoGroe8CrcQiCSv~BYjWRC4qzvxccYkOQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="DNA Methylation Status at the PAPs of the SEPT9 gene. (A) Schematic representation of the structure of the SEPT9 gene. Fourteen PAPs which were selected from 20 and were subjected to the analyses are marked with asterisks. (B) Observed patterns of DNA methylation at each of the PAPs in each of the tissues. In the 2nd column CGI-containing PAPs are represented as 1, and CGI-less PAPs as 0. In the 3rd column, tissue-preferences of the PAPs are shown." path-from-xml="dsl008f2.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4946944/dsl008f2.jpeg" path-from-xml="dsl008f2.jpeg" target="_blank">View large</a><a section="4946944" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/4/10.1093/dnares/dsl008/2/dsl008f2.jpeg?Expires=2147483647&Signature=VaWki5uwZswoyr8CK2Dkm-klFk-IGAtu9Lnie62CCt9w5DppGXmNf6pDDbw~Sqgrz8wv~lR~0wYce8zV--AA8JpusMvES9VVJ1sVEgsj2T79uBR824UQBPZ0YhmswTDpiaP-VT6RPx0-94y3rWP1TJv86A7XygIHCrTTJs2WYLiNRhb0-f51snw~tS-EohEGbewGXbdk7~uOPqqlZtkISrg2lut18K3MdfDGyuXG9ScwCqfKAzIVzAflodYbhmSJThHs~n2ovqISeCzLVUMremY2-ddtxWkJ43w7SQjevaoehwfLnH~lkSVayM~5zz726iLio5qm3xOITtob4q3GrA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4946944&ar=348476&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl008f2.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>DNA Methylation Status at the PAPs of the SEPT9 gene. (A) Schematic representation of the structure of the SEPT9 gene. Fourteen PAPs which were selected from 20 and were subjected to the analyses are marked with asterisks. (B) Observed patterns of DNA methylation at each of the PAPs in each of the tissues. In the 2nd column CGI-containing PAPs are represented as 1, and CGI-less PAPs as 0. In the 3rd column, tissue-preferences of the PAPs are shown.</p></div></div><p>In order to obtain a more generalized picture, we compared the methylation patterns between PAPs belonging to the same gene. Although we assayed 181 PAPs for 61 genes, there was about 20% ‘n.d.’ data. Thus, there were 48 genes for which we had data for at least three tissues at two PAPs per gene. Only five of these 48 genes showed a perfect match of PAP methylation patterns in all five tissues (<a reveal-id="tbl3" data-open="tbl3" class="link link-reveal link-table xref-fig">Table 3</a>). Thus, what we observed for SEPT9 was true for all the genes we analyzed. It is noteworthy that the methylation pattern was uniformly ‘null’ for the matched PAPs. There were 10 PAPs for five genes that showed a perfect match. They were ‘null’ methylated in all five tissues (<a reveal-id="tbl3" data-open="tbl3" class="link link-reveal link-table xref-fig">Table 3</a>). The number of genes with a perfect match would only be seven even if a mismatch in one tissue were allowed. There were two additional genes possessing five PAPs that had one mismatch (<a reveal-id="tbl3" data-open="tbl3" class="link link-reveal link-table xref-fig">Table 3</a>). Two of them showed a ‘composite’ pattern in liver, while the rest were all ‘null’ in the other tissues (<a reveal-id="tbl3" data-open="tbl3" class="link link-reveal link-table xref-fig">Table 3</a>).</p>                        <a id="4946946" scrollto-destination="4946946"></a>
<div content-id="tbl3" class="table-modal"><div class="table-wrap"><div id="tbl3" class="table-wrap-title" data-id="tbl3"><div class="label -label"><strong>Table 3</strong></div><div class="caption"><p>Changes in the DNA methylation status between PAPs and between tissues, the numbers of TB-PAPs and non-TB-PAPs whose DNA methylation status was Changed (upper line) or Not-Changed (lower line) between tissues</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Refseq# or Promoter ID </th><th>CpGisland </th><th>Biased expression </th><th>Testis </th><th>Brain </th><th>Spleen </th><th>Stomach </th><th>Liver </th></tr></thead><tbody><tr><td colspan="8">(A) The exceptional cases in which the DNA methylation patterns of a PAP perfectly matched with those of the accompanying PAPs in all five tissues </td></tr><tr><td colspan="8">    NM_002065 </td></tr><tr><td>        1 </td><td>1 </td><td>Testis </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td></tr><tr><td>        2 </td><td>0 </td><td>Lung </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td></tr><tr><td colspan="8">    NM_002394 </td></tr><tr><td>        1 </td><td>1 </td><td>Other </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td></tr><tr><td>        2 </td><td>0 </td><td>Cancer </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td></tr><tr><td colspan="8">    NM_004046 </td></tr><tr><td>        1 </td><td>1 </td><td>Testis </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td></tr><tr><td>        2 </td><td>1 </td><td>Brain </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td></tr><tr><td colspan="8">    NM_005104 </td></tr><tr><td>        2 </td><td>1 </td><td>Other </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td></tr><tr><td>        3 </td><td>0 </td><td>Other </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td></tr><tr><td colspan="8">    NM_006759 </td></tr><tr><td>        1 </td><td>1 </td><td>Other </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td></tr><tr><td>        2 </td><td>1 </td><td>Other </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td></tr><tr><td colspan="8"><hr /> </td></tr><tr><td colspan="8">(B) The cases in which the DNA methylation patterns of a PAP matched with those of the accompanying PAPs in all five tissues, when one mismatch was allowed </td></tr><tr><td colspan="8">    NM_005896 </td></tr><tr><td>        1 </td><td>1 </td><td>Cancer </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td></tr><tr><td>        2 </td><td>1 </td><td>Brain </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td></tr><tr><td>        3 </td><td>0 </td><td>Other </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td><td>CP </td></tr><tr><td colspan="8">    NM_018093 </td></tr><tr><td>        1 </td><td>0 </td><td>Testis </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td><td>CP </td></tr><tr><td>        2 </td><td>1 </td><td>Non-specific </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td></tr></tbody></table></div><div content-id="tbl3" class="table-modal"><table><thead name="thead" align=""><tr><th>Refseq# or Promoter ID </th><th>CpGisland </th><th>Biased expression </th><th>Testis </th><th>Brain </th><th>Spleen </th><th>Stomach </th><th>Liver </th></tr></thead><tbody><tr><td colspan="8">(A) The exceptional cases in which the DNA methylation patterns of a PAP perfectly matched with those of the accompanying PAPs in all five tissues </td></tr><tr><td colspan="8">    NM_002065 </td></tr><tr><td>        1 </td><td>1 </td><td>Testis </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td></tr><tr><td>        2 </td><td>0 </td><td>Lung </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td></tr><tr><td colspan="8">    NM_002394 </td></tr><tr><td>        1 </td><td>1 </td><td>Other </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td></tr><tr><td>        2 </td><td>0 </td><td>Cancer </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td></tr><tr><td colspan="8">    NM_004046 </td></tr><tr><td>        1 </td><td>1 </td><td>Testis </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td></tr><tr><td>        2 </td><td>1 </td><td>Brain </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td></tr><tr><td colspan="8">    NM_005104 </td></tr><tr><td>        2 </td><td>1 </td><td>Other </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td></tr><tr><td>        3 </td><td>0 </td><td>Other </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td></tr><tr><td colspan="8">    NM_006759 </td></tr><tr><td>        1 </td><td>1 </td><td>Other </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td></tr><tr><td>        2 </td><td>1 </td><td>Other </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td></tr><tr><td colspan="8"><hr /> </td></tr><tr><td colspan="8">(B) The cases in which the DNA methylation patterns of a PAP matched with those of the accompanying PAPs in all five tissues, when one mismatch was allowed </td></tr><tr><td colspan="8">    NM_005896 </td></tr><tr><td>        1 </td><td>1 </td><td>Cancer </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td></tr><tr><td>        2 </td><td>1 </td><td>Brain </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td></tr><tr><td>        3 </td><td>0 </td><td>Other </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td><td>CP </td></tr><tr><td colspan="8">    NM_018093 </td></tr><tr><td>        1 </td><td>0 </td><td>Testis </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td><td>CP </td></tr><tr><td>        2 </td><td>1 </td><td>Non-specific </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td></tr></tbody></table></div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>TB-PAPs </th><th>Non-TB-PAPs </th></tr></thead><tbody><tr><td colspan="3">(C) </td></tr><tr><td>    Changed </td><td>77 (62%) </td><td>16 (53%) </td></tr><tr><td>    Not-Changed </td><td>47 (38%) </td><td>14 (47%) </td></tr><tr><td>    Total </td><td>124 (100%) </td><td>30 (100%) </td></tr></tbody></table></div><div content-id="tbl3" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>TB-PAPs </th><th>Non-TB-PAPs </th></tr></thead><tbody><tr><td colspan="3">(C) </td></tr><tr><td>    Changed </td><td>77 (62%) </td><td>16 (53%) </td></tr><tr><td>    Not-Changed </td><td>47 (38%) </td><td>14 (47%) </td></tr><tr><td>    Total </td><td>124 (100%) </td><td>30 (100%) </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4946946" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="tbl3" class="table-wrap-title" data-id="tbl3"><div class="label -label"><strong>Table 3</strong></div><div class="caption"><p>Changes in the DNA methylation status between PAPs and between tissues, the numbers of TB-PAPs and non-TB-PAPs whose DNA methylation status was Changed (upper line) or Not-Changed (lower line) between tissues</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Refseq# or Promoter ID </th><th>CpGisland </th><th>Biased expression </th><th>Testis </th><th>Brain </th><th>Spleen </th><th>Stomach </th><th>Liver </th></tr></thead><tbody><tr><td colspan="8">(A) The exceptional cases in which the DNA methylation patterns of a PAP perfectly matched with those of the accompanying PAPs in all five tissues </td></tr><tr><td colspan="8">    NM_002065 </td></tr><tr><td>        1 </td><td>1 </td><td>Testis </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td></tr><tr><td>        2 </td><td>0 </td><td>Lung </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td></tr><tr><td colspan="8">    NM_002394 </td></tr><tr><td>        1 </td><td>1 </td><td>Other </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td></tr><tr><td>        2 </td><td>0 </td><td>Cancer </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td></tr><tr><td colspan="8">    NM_004046 </td></tr><tr><td>        1 </td><td>1 </td><td>Testis </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td></tr><tr><td>        2 </td><td>1 </td><td>Brain </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td></tr><tr><td colspan="8">    NM_005104 </td></tr><tr><td>        2 </td><td>1 </td><td>Other </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td></tr><tr><td>        3 </td><td>0 </td><td>Other </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td></tr><tr><td colspan="8">    NM_006759 </td></tr><tr><td>        1 </td><td>1 </td><td>Other </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td></tr><tr><td>        2 </td><td>1 </td><td>Other </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td></tr><tr><td colspan="8"><hr /> </td></tr><tr><td colspan="8">(B) The cases in which the DNA methylation patterns of a PAP matched with those of the accompanying PAPs in all five tissues, when one mismatch was allowed </td></tr><tr><td colspan="8">    NM_005896 </td></tr><tr><td>        1 </td><td>1 </td><td>Cancer </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td></tr><tr><td>        2 </td><td>1 </td><td>Brain </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td></tr><tr><td>        3 </td><td>0 </td><td>Other </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td><td>CP </td></tr><tr><td colspan="8">    NM_018093 </td></tr><tr><td>        1 </td><td>0 </td><td>Testis </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td><td>CP </td></tr><tr><td>        2 </td><td>1 </td><td>Non-specific </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td></tr></tbody></table></div><div content-id="tbl3" class="table-modal"><table><thead name="thead" align=""><tr><th>Refseq# or Promoter ID </th><th>CpGisland </th><th>Biased expression </th><th>Testis </th><th>Brain </th><th>Spleen </th><th>Stomach </th><th>Liver </th></tr></thead><tbody><tr><td colspan="8">(A) The exceptional cases in which the DNA methylation patterns of a PAP perfectly matched with those of the accompanying PAPs in all five tissues </td></tr><tr><td colspan="8">    NM_002065 </td></tr><tr><td>        1 </td><td>1 </td><td>Testis </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td></tr><tr><td>        2 </td><td>0 </td><td>Lung </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td></tr><tr><td colspan="8">    NM_002394 </td></tr><tr><td>        1 </td><td>1 </td><td>Other </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td></tr><tr><td>        2 </td><td>0 </td><td>Cancer </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td></tr><tr><td colspan="8">    NM_004046 </td></tr><tr><td>        1 </td><td>1 </td><td>Testis </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td></tr><tr><td>        2 </td><td>1 </td><td>Brain </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td></tr><tr><td colspan="8">    NM_005104 </td></tr><tr><td>        2 </td><td>1 </td><td>Other </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td></tr><tr><td>        3 </td><td>0 </td><td>Other </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td></tr><tr><td colspan="8">    NM_006759 </td></tr><tr><td>        1 </td><td>1 </td><td>Other </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td></tr><tr><td>        2 </td><td>1 </td><td>Other </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td></tr><tr><td colspan="8"><hr /> </td></tr><tr><td colspan="8">(B) The cases in which the DNA methylation patterns of a PAP matched with those of the accompanying PAPs in all five tissues, when one mismatch was allowed </td></tr><tr><td colspan="8">    NM_005896 </td></tr><tr><td>        1 </td><td>1 </td><td>Cancer </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td></tr><tr><td>        2 </td><td>1 </td><td>Brain </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td></tr><tr><td>        3 </td><td>0 </td><td>Other </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td><td>CP </td></tr><tr><td colspan="8">    NM_018093 </td></tr><tr><td>        1 </td><td>0 </td><td>Testis </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td><td>CP </td></tr><tr><td>        2 </td><td>1 </td><td>Non-specific </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td><td>NL </td></tr></tbody></table></div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>TB-PAPs </th><th>Non-TB-PAPs </th></tr></thead><tbody><tr><td colspan="3">(C) </td></tr><tr><td>    Changed </td><td>77 (62%) </td><td>16 (53%) </td></tr><tr><td>    Not-Changed </td><td>47 (38%) </td><td>14 (47%) </td></tr><tr><td>    Total </td><td>124 (100%) </td><td>30 (100%) </td></tr></tbody></table></div><div content-id="tbl3" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>TB-PAPs </th><th>Non-TB-PAPs </th></tr></thead><tbody><tr><td colspan="3">(C) </td></tr><tr><td>    Changed </td><td>77 (62%) </td><td>16 (53%) </td></tr><tr><td>    Not-Changed </td><td>47 (38%) </td><td>14 (47%) </td></tr><tr><td>    Total </td><td>124 (100%) </td><td>30 (100%) </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4946946" target="_blank">View Large</a></div></div><p>Indeed, the DNA methylation status of the individual PAPs differed greatly depending on the tissue. As shown in <a reveal-id="tbl3" data-open="tbl3" class="link link-reveal link-table xref-fig">Table 3</a>, the DNA methylation status of 93 of 154 (60%) PAPs varied among the five tissues. This confirms that variable methylation patterns among the tissues may be more common than a uniform pattern. We also noted that the TB-PAPs seemed to have more highly variable methylation statuses than non-TB-PAPs. TB-PAPs had variable methylation patterns in 77 of 124 cases (62%), while only 16 of 30 (53%) non-TB-PAPs had such patterns, indicating slightly greater variability in the former, although the statistical significance was not very high (<em>P</em> &lt; 0.14; evaluated by calculating hyper-geometric distributions following the equation shown in Materials and Methods), possibly due to the limited size of the current dataset.</p>                            <h3 scrollto-destination=4946948 id="4946948" class="section-title" >3.4. Relationship between tissue-bias and DNA methylation status of alternative promoters</h3>
<p>We further examined the relationship between the tissue-preferential usage of the individual PAPs and the tissue-profiles of their DNA methylation status. We compared the DNA methylation status of the TB-PAPs between their ‘preferred’ tissues and other tissues. We focused this analysis on the testis-preferring PAPs and brain-preferring PAPs, as the largest number of TB-PAPs was obtained from these tissues (also see <a reveal-id="tbl1" data-open="tbl1" class="link link-reveal link-table xref-fig">Table 1</a>). We found that the DNA methylation status at the ‘testis-preferring’ PAPs was considerably biased to ‘null’, and ‘complete’ was extremely rare (<em>P</em> &lt; 0.003; evaluated by calculating hyper-geometric distributions of ‘null’ and ‘composite’ against their frequencies in the total population; <a reveal-id="tbl4" data-open="tbl4" class="link link-reveal link-table xref-fig">Table 4</a>). Such bias toward the ‘null’ state was not seen in other tissues for testis-preferring PAPs or for non-TB-PAPs in the testis. These observations suggest that unique regulation may be exerted for TB-PAPs in the testis. We observed no such bias for the other PAPs of the genes that had testis-preferring PAPs with ‘null’ methylation in the testis (<a reveal-id="tbl4" data-open="tbl4" class="link link-reveal link-table xref-fig">Table 4</a>).</p>                        <a id="4946950" scrollto-destination="4946950"></a>
<div content-id="tbl4" class="table-modal"><div class="table-wrap"><div id="tbl4" class="table-wrap-title" data-id="tbl4"><div class="label -label"><strong>Table 4</strong></div><div class="caption"><p>Relation between the Tissue Preference and the DNA Methylation Status of the PAPs (A) Numbers of observed DNA methylation patterns in testis-preferring PAPs in testis (1st column) and in other tissues (2nd column), those in brain-preferring PAPs in brain (3rd column) and in other tissues (4th column) and those in other-tissue-preferring PAPs in testis (5th column) and in brain (6th column) (B) Number of observed DNA methylation patterns in the PAPs whose accompanying PAPs were testis-preferring and “null” methylated in testis (population marked with an asterisk in Table 4). (C) Number of observed DNA methylation patterns in the PAPs whose accompanying PAPs were brain-preferring and “compositely” methylated in brain (population marked with double asterisks in Table 4).</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>(A) </th><th>Testis-preferring PAPs in testis (population marked with an asterisk) </th><th>Testis-preferring PAPs in other tissues </th><th>Brain-preferring PAPs in brain (population marked with double asterisks) </th><th>Brain-preferring PAPs in other tissues </th><th>Other tissue-preferring PAPs in testis </th><th>Other tissue-preferring PAPs in brain </th></tr></thead><tbody><tr><td>    Complete </td><td>1 (5%) </td><td>10 (12%) </td><td>13 (31%) </td><td>38 (23%) </td><td>38 (31%) </td><td>26 (24%) </td></tr><tr><td>    Null </td><td>12 (63%)<sup>*</sup> </td><td>43 (53%) </td><td>18 (43%) </td><td>79 (49%) </td><td>46 (37%) </td><td>43 (39%) </td></tr><tr><td>    Composite </td><td>6 (32%) </td><td>26 (32%) </td><td>11 (26%)<sup>**</sup> </td><td>41 (25%) </td><td>34 (28%) </td><td>38 (35%) </td></tr><tr><td>    Incomplete </td><td>0 (0%) </td><td>2 (3%) </td><td>0 (0%) </td><td>5 (3%) </td><td>5 (4%) </td><td>2 (2%) </td></tr><tr><td>    Total </td><td>19 (100%) </td><td>81 (100%) </td><td>42 (100%) </td><td>163 (100%) </td><td>123 (100%) </td><td>109 (100%) </td></tr></tbody></table></div><div content-id="tbl4" class="table-modal"><table><thead name="thead" align=""><tr><th>(A) </th><th>Testis-preferring PAPs in testis (population marked with an asterisk) </th><th>Testis-preferring PAPs in other tissues </th><th>Brain-preferring PAPs in brain (population marked with double asterisks) </th><th>Brain-preferring PAPs in other tissues </th><th>Other tissue-preferring PAPs in testis </th><th>Other tissue-preferring PAPs in brain </th></tr></thead><tbody><tr><td>    Complete </td><td>1 (5%) </td><td>10 (12%) </td><td>13 (31%) </td><td>38 (23%) </td><td>38 (31%) </td><td>26 (24%) </td></tr><tr><td>    Null </td><td>12 (63%)<sup>*</sup> </td><td>43 (53%) </td><td>18 (43%) </td><td>79 (49%) </td><td>46 (37%) </td><td>43 (39%) </td></tr><tr><td>    Composite </td><td>6 (32%) </td><td>26 (32%) </td><td>11 (26%)<sup>**</sup> </td><td>41 (25%) </td><td>34 (28%) </td><td>38 (35%) </td></tr><tr><td>    Incomplete </td><td>0 (0%) </td><td>2 (3%) </td><td>0 (0%) </td><td>5 (3%) </td><td>5 (4%) </td><td>2 (2%) </td></tr><tr><td>    Total </td><td>19 (100%) </td><td>81 (100%) </td><td>42 (100%) </td><td>163 (100%) </td><td>123 (100%) </td><td>109 (100%) </td></tr></tbody></table></div><div class="table-overflow"><table><thead name="thead" align=""><tr><th colspan="2">(B) Accompanying PAPs in testis when testis-preferring PAPs = Null </th></tr></thead><tbody><tr><td>    Complete </td><td>2 (13%) </td></tr><tr><td>    Null </td><td>8 (50%) </td></tr><tr><td>    Composite </td><td>2 (13%) </td></tr><tr><td>    Incomplete </td><td>0 (0%) </td></tr><tr><td>    n.d. </td><td>4 (25%) </td></tr><tr><td>    Total </td><td>16 (100%) </td></tr></tbody></table></div><div content-id="tbl4" class="table-modal"><table><thead name="thead" align=""><tr><th colspan="2">(B) Accompanying PAPs in testis when testis-preferring PAPs = Null </th></tr></thead><tbody><tr><td>    Complete </td><td>2 (13%) </td></tr><tr><td>    Null </td><td>8 (50%) </td></tr><tr><td>    Composite </td><td>2 (13%) </td></tr><tr><td>    Incomplete </td><td>0 (0%) </td></tr><tr><td>    n.d. </td><td>4 (25%) </td></tr><tr><td>    Total </td><td>16 (100%) </td></tr></tbody></table></div><div class="table-overflow"><table><thead name="thead" align=""><tr><th colspan="2">(C) Accompanying PAPs in brain when brain-preferring PAPs= Composite </th></tr></thead><tbody><tr><td>    Complete </td><td>10 (36%) </td></tr><tr><td>    Null </td><td>4 (14%) </td></tr><tr><td>    Composite </td><td>9 (32%) </td></tr><tr><td>    Incomplete </td><td>0 (0%) </td></tr><tr><td>    n.d. </td><td>5 (18%) </td></tr><tr><td>    Total </td><td>28 (100%) </td></tr></tbody></table></div><div content-id="tbl4" class="table-modal"><table><thead name="thead" align=""><tr><th colspan="2">(C) Accompanying PAPs in brain when brain-preferring PAPs= Composite </th></tr></thead><tbody><tr><td>    Complete </td><td>10 (36%) </td></tr><tr><td>    Null </td><td>4 (14%) </td></tr><tr><td>    Composite </td><td>9 (32%) </td></tr><tr><td>    Incomplete </td><td>0 (0%) </td></tr><tr><td>    n.d. </td><td>5 (18%) </td></tr><tr><td>    Total </td><td>28 (100%) </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4946950" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="tbl4" class="table-wrap-title" data-id="tbl4"><div class="label -label"><strong>Table 4</strong></div><div class="caption"><p>Relation between the Tissue Preference and the DNA Methylation Status of the PAPs (A) Numbers of observed DNA methylation patterns in testis-preferring PAPs in testis (1st column) and in other tissues (2nd column), those in brain-preferring PAPs in brain (3rd column) and in other tissues (4th column) and those in other-tissue-preferring PAPs in testis (5th column) and in brain (6th column) (B) Number of observed DNA methylation patterns in the PAPs whose accompanying PAPs were testis-preferring and “null” methylated in testis (population marked with an asterisk in Table 4). (C) Number of observed DNA methylation patterns in the PAPs whose accompanying PAPs were brain-preferring and “compositely” methylated in brain (population marked with double asterisks in Table 4).</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>(A) </th><th>Testis-preferring PAPs in testis (population marked with an asterisk) </th><th>Testis-preferring PAPs in other tissues </th><th>Brain-preferring PAPs in brain (population marked with double asterisks) </th><th>Brain-preferring PAPs in other tissues </th><th>Other tissue-preferring PAPs in testis </th><th>Other tissue-preferring PAPs in brain </th></tr></thead><tbody><tr><td>    Complete </td><td>1 (5%) </td><td>10 (12%) </td><td>13 (31%) </td><td>38 (23%) </td><td>38 (31%) </td><td>26 (24%) </td></tr><tr><td>    Null </td><td>12 (63%)<sup>*</sup> </td><td>43 (53%) </td><td>18 (43%) </td><td>79 (49%) </td><td>46 (37%) </td><td>43 (39%) </td></tr><tr><td>    Composite </td><td>6 (32%) </td><td>26 (32%) </td><td>11 (26%)<sup>**</sup> </td><td>41 (25%) </td><td>34 (28%) </td><td>38 (35%) </td></tr><tr><td>    Incomplete </td><td>0 (0%) </td><td>2 (3%) </td><td>0 (0%) </td><td>5 (3%) </td><td>5 (4%) </td><td>2 (2%) </td></tr><tr><td>    Total </td><td>19 (100%) </td><td>81 (100%) </td><td>42 (100%) </td><td>163 (100%) </td><td>123 (100%) </td><td>109 (100%) </td></tr></tbody></table></div><div content-id="tbl4" class="table-modal"><table><thead name="thead" align=""><tr><th>(A) </th><th>Testis-preferring PAPs in testis (population marked with an asterisk) </th><th>Testis-preferring PAPs in other tissues </th><th>Brain-preferring PAPs in brain (population marked with double asterisks) </th><th>Brain-preferring PAPs in other tissues </th><th>Other tissue-preferring PAPs in testis </th><th>Other tissue-preferring PAPs in brain </th></tr></thead><tbody><tr><td>    Complete </td><td>1 (5%) </td><td>10 (12%) </td><td>13 (31%) </td><td>38 (23%) </td><td>38 (31%) </td><td>26 (24%) </td></tr><tr><td>    Null </td><td>12 (63%)<sup>*</sup> </td><td>43 (53%) </td><td>18 (43%) </td><td>79 (49%) </td><td>46 (37%) </td><td>43 (39%) </td></tr><tr><td>    Composite </td><td>6 (32%) </td><td>26 (32%) </td><td>11 (26%)<sup>**</sup> </td><td>41 (25%) </td><td>34 (28%) </td><td>38 (35%) </td></tr><tr><td>    Incomplete </td><td>0 (0%) </td><td>2 (3%) </td><td>0 (0%) </td><td>5 (3%) </td><td>5 (4%) </td><td>2 (2%) </td></tr><tr><td>    Total </td><td>19 (100%) </td><td>81 (100%) </td><td>42 (100%) </td><td>163 (100%) </td><td>123 (100%) </td><td>109 (100%) </td></tr></tbody></table></div><div class="table-overflow"><table><thead name="thead" align=""><tr><th colspan="2">(B) Accompanying PAPs in testis when testis-preferring PAPs = Null </th></tr></thead><tbody><tr><td>    Complete </td><td>2 (13%) </td></tr><tr><td>    Null </td><td>8 (50%) </td></tr><tr><td>    Composite </td><td>2 (13%) </td></tr><tr><td>    Incomplete </td><td>0 (0%) </td></tr><tr><td>    n.d. </td><td>4 (25%) </td></tr><tr><td>    Total </td><td>16 (100%) </td></tr></tbody></table></div><div content-id="tbl4" class="table-modal"><table><thead name="thead" align=""><tr><th colspan="2">(B) Accompanying PAPs in testis when testis-preferring PAPs = Null </th></tr></thead><tbody><tr><td>    Complete </td><td>2 (13%) </td></tr><tr><td>    Null </td><td>8 (50%) </td></tr><tr><td>    Composite </td><td>2 (13%) </td></tr><tr><td>    Incomplete </td><td>0 (0%) </td></tr><tr><td>    n.d. </td><td>4 (25%) </td></tr><tr><td>    Total </td><td>16 (100%) </td></tr></tbody></table></div><div class="table-overflow"><table><thead name="thead" align=""><tr><th colspan="2">(C) Accompanying PAPs in brain when brain-preferring PAPs= Composite </th></tr></thead><tbody><tr><td>    Complete </td><td>10 (36%) </td></tr><tr><td>    Null </td><td>4 (14%) </td></tr><tr><td>    Composite </td><td>9 (32%) </td></tr><tr><td>    Incomplete </td><td>0 (0%) </td></tr><tr><td>    n.d. </td><td>5 (18%) </td></tr><tr><td>    Total </td><td>28 (100%) </td></tr></tbody></table></div><div content-id="tbl4" class="table-modal"><table><thead name="thead" align=""><tr><th colspan="2">(C) Accompanying PAPs in brain when brain-preferring PAPs= Composite </th></tr></thead><tbody><tr><td>    Complete </td><td>10 (36%) </td></tr><tr><td>    Null </td><td>4 (14%) </td></tr><tr><td>    Composite </td><td>9 (32%) </td></tr><tr><td>    Incomplete </td><td>0 (0%) </td></tr><tr><td>    n.d. </td><td>5 (18%) </td></tr><tr><td>    Total </td><td>28 (100%) </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4946950" target="_blank">View Large</a></div></div><p>When the same analysis was performed for brain-preferring PAPs, we did not see any bias toward ‘null’. However, interestingly, in brain, the DNA methylation patterns in accompanying PAPs of the same gene were skewed. As shown in <a reveal-id="tbl4" data-open="tbl4" class="link link-reveal link-table xref-fig">Table 4</a>, even when the brain-preferring PAPs were ‘compositely’ methylated, their counterpart PAPs rarely had ‘null’ status. Although further extensive data collection will be necessary, these observations indicate that there may be several distinct methylation mechanisms for tissue-biased gene expression depending on the tissue type. Nevertheless, it seemed that actively used PAPs are generally less extensively methylated than their accompanying PAPs.</p>                            <h3 scrollto-destination=4946952 id="4946952" class="section-title" >3.5. Relationship between DNA methylation status and mRNA expression level</h3>
<p>In order to examine the relationship between the DNA methylation status and the level of mRNA expression, we measured the relative amount of mRNAs transcribed from each of the alternative promoters in the five tissues. For this purpose, we used real-time RT–PCR assays. Based on our full-length cDNA information, RT–PCR primers were designed at unique regions inside of the first exons of the transcript variants corresponding to each of the PAPs which showed different DNA methylation statuses between tissues. Ninety-three such primer pairs were considered, and in 49 cases, positive signals from either of the tissues were observed. (For an example, see <a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3A</a>.) <a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3B</a> shows the distribution of the relative expression level calculated separately for each of the DNA methylation patterns. Although the difference in the distributions of the relative expression levels between ‘null’ and ‘composite’ populations was not highly statistically significant as a whole, we still observed that ‘null’ or relatively light methylation patterns of PAPs were enriched (<em>P</em> &lt; 0.02; hyper-geometric distributions calculated similarly as in Section 3.4.) in the tissues in which the mRNA expression was the strongest (relative expression level ≥8; also see <a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3A</a>, Supplementary Table S2). We also observed that ‘complete’ DNA methylation was the only pattern in which the mRNA expression was the weakest (relative expression level ≤1/8). It is highly possible that the changes in the DNA methylation status are responsible for the modulations of the gene expression in these cases.</p>                        <a id="4946954" scrollto-destination="4946954"></a>
<div id="fig3" data-id="fig3" class="fig fig-section"><div class="label fig-label"><strong>Figure 3</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/4/10.1093/dnares/dsl008/2/m_dsl008f3.jpeg?Expires=2147483647&Signature=kCtGarSnswj7IdCQIVZoa4GOzzoKsbyn-SQtKpLCUcwk9PTbaSPyECfQPVccq~7ndnu7wz~~P-5hEI66lAGZdq4iMKWtmNqfibphCV2g30mFMAdNcBly19tR1AsUdg~eB595PahNYxpY98hcMvHkTkGpM5PYt7eDrxIIm8cIKelQIihurxqMh2pKvsVtCO1EeI7xcbipEvg9CZ~uSSqxFaExQdA3w0DTyTObYTgKYaFCpbVMaDJOoRDa89cy8MsMXFSYMZan~o14NxxcZEQ0-Lf0ArBxi44yAdlQtFIeIQm9oTDRcISS3-8bw1R0xJdB1bjA3F4wKyqozQT2dBtjXA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Methylation Pattern and mRNA Expression in the Five Tissues. (A) Examples of the semi-quantitative (real-time) RT–PCR results. The Ct values (constant of threshold; as of the default setting of ABI7900HT) and the relative expression levels are shown in the margin. The results of a similar analysis for the GAPDH mRNA are shown in Supplementary Figure S3, although we did not use GAPDH as a normalization standard. Ct: Constant of threshold as for default setting. Also note that relatively high expression levels are correlated with relatively light methylation patterns in both cases, even if their expression patterns are non-specific. (B) Composition of the DNA methylation patterns in the population of the PAPs with the indicated relative expression level. Based on the results of the semi-quantitative (real-time) RT–PCR assays, the relative expression levels were calculated as the deviation from the averaged expression level in the five tissues [also see (A) and Materials and Methods]. The relative expression levels observed for each of the PAPs in each of the tissues were correlated with their DNA-methylation status and summed for the five tissues." path-from-xml="dsl008f3.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4946954/dsl008f3.jpeg" path-from-xml="dsl008f3.jpeg" target="_blank">View large</a><a section="4946954" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/4/10.1093/dnares/dsl008/2/dsl008f3.jpeg?Expires=2147483647&Signature=fehuTLaAmAL0VgSXhDbEuKYWpg1BEnxHT4T8d6Xs-qv5da6pA2ANH9tea3dnbCa~p5ijb1pdmZiflyK~AOkygpQmvWwkIih2CjEAd9YtdweIBueUDzPADRfkTC8OB0cAWpAiqZssX0pVWd7hXaKYSZjL2OURKhfMyInG7VPYT~grJ41iHVRqYQKGuvpJc0D-88dKEmrnZr3rdrbXZ3ylvjoWC~DoZdqDiFozE2udgdQBEHv82XbuizgzNQJwp-cAz1zipLJ0ZnuFEbQRWjzcZlOOT3NUvLHfJ4VxRWArjgUrlUlpyl61lWdLKtoq~RaQKlY-C-hp2x5tDdv9DGlEYQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4946954&ar=348476&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl008f3.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Methylation Pattern and mRNA Expression in the Five Tissues. (A) Examples of the semi-quantitative (real-time) RT–PCR results. The Ct values (constant of threshold; as of the default setting of ABI7900HT) and the relative expression levels are shown in the margin. The results of a similar analysis for the GAPDH mRNA are shown in Supplementary Figure S3, although we did not use GAPDH as a normalization standard. Ct: Constant of threshold as for default setting. Also note that relatively high expression levels are correlated with relatively light methylation patterns in both cases, even if their expression patterns are non-specific. (B) Composition of the DNA methylation patterns in the population of the PAPs with the indicated relative expression level. Based on the results of the semi-quantitative (real-time) RT–PCR assays, the relative expression levels were calculated as the deviation from the averaged expression level in the five tissues [also see (A) and Materials and Methods]. The relative expression levels observed for each of the PAPs in each of the tissues were correlated with their DNA-methylation status and summed for the five tissues.</p></div></div><div content-id="fig3" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 3</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/4/10.1093/dnares/dsl008/2/m_dsl008f3.jpeg?Expires=2147483647&Signature=kCtGarSnswj7IdCQIVZoa4GOzzoKsbyn-SQtKpLCUcwk9PTbaSPyECfQPVccq~7ndnu7wz~~P-5hEI66lAGZdq4iMKWtmNqfibphCV2g30mFMAdNcBly19tR1AsUdg~eB595PahNYxpY98hcMvHkTkGpM5PYt7eDrxIIm8cIKelQIihurxqMh2pKvsVtCO1EeI7xcbipEvg9CZ~uSSqxFaExQdA3w0DTyTObYTgKYaFCpbVMaDJOoRDa89cy8MsMXFSYMZan~o14NxxcZEQ0-Lf0ArBxi44yAdlQtFIeIQm9oTDRcISS3-8bw1R0xJdB1bjA3F4wKyqozQT2dBtjXA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Methylation Pattern and mRNA Expression in the Five Tissues. (A) Examples of the semi-quantitative (real-time) RT–PCR results. The Ct values (constant of threshold; as of the default setting of ABI7900HT) and the relative expression levels are shown in the margin. The results of a similar analysis for the GAPDH mRNA are shown in Supplementary Figure S3, although we did not use GAPDH as a normalization standard. Ct: Constant of threshold as for default setting. Also note that relatively high expression levels are correlated with relatively light methylation patterns in both cases, even if their expression patterns are non-specific. (B) Composition of the DNA methylation patterns in the population of the PAPs with the indicated relative expression level. Based on the results of the semi-quantitative (real-time) RT–PCR assays, the relative expression levels were calculated as the deviation from the averaged expression level in the five tissues [also see (A) and Materials and Methods]. The relative expression levels observed for each of the PAPs in each of the tissues were correlated with their DNA-methylation status and summed for the five tissues." path-from-xml="dsl008f3.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4946954/dsl008f3.jpeg" path-from-xml="dsl008f3.jpeg" target="_blank">View large</a><a section="4946954" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/4/10.1093/dnares/dsl008/2/dsl008f3.jpeg?Expires=2147483647&Signature=fehuTLaAmAL0VgSXhDbEuKYWpg1BEnxHT4T8d6Xs-qv5da6pA2ANH9tea3dnbCa~p5ijb1pdmZiflyK~AOkygpQmvWwkIih2CjEAd9YtdweIBueUDzPADRfkTC8OB0cAWpAiqZssX0pVWd7hXaKYSZjL2OURKhfMyInG7VPYT~grJ41iHVRqYQKGuvpJc0D-88dKEmrnZr3rdrbXZ3ylvjoWC~DoZdqDiFozE2udgdQBEHv82XbuizgzNQJwp-cAz1zipLJ0ZnuFEbQRWjzcZlOOT3NUvLHfJ4VxRWArjgUrlUlpyl61lWdLKtoq~RaQKlY-C-hp2x5tDdv9DGlEYQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4946954&ar=348476&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl008f3.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Methylation Pattern and mRNA Expression in the Five Tissues. (A) Examples of the semi-quantitative (real-time) RT–PCR results. The Ct values (constant of threshold; as of the default setting of ABI7900HT) and the relative expression levels are shown in the margin. The results of a similar analysis for the GAPDH mRNA are shown in Supplementary Figure S3, although we did not use GAPDH as a normalization standard. Ct: Constant of threshold as for default setting. Also note that relatively high expression levels are correlated with relatively light methylation patterns in both cases, even if their expression patterns are non-specific. (B) Composition of the DNA methylation patterns in the population of the PAPs with the indicated relative expression level. Based on the results of the semi-quantitative (real-time) RT–PCR assays, the relative expression levels were calculated as the deviation from the averaged expression level in the five tissues [also see (A) and Materials and Methods]. The relative expression levels observed for each of the PAPs in each of the tissues were correlated with their DNA-methylation status and summed for the five tissues.</p></div></div>                            <h3 scrollto-destination=4946955 id="4946955" class="section-title" >3.6. Relationship between DNA methylation status and the presence of CpG islands</h3>
<p>We further attempted to clarify the relationship between the DNA methylation status and the presence of CpG islands. We compared the DNA methylation status between the populations of CGI-containing PAPs and CGI-less PAPs and found a striking difference. In contrast to the so-far observed variable methylation patterns, we found that the CGI-containing PAPs were predominantly ‘null’ methylated throughout the five tissues (<em>P</em> &lt; 1.0E−55; evaluated by standard Fisher's test; <a reveal-id="tbl5" data-open="tbl5" class="link link-reveal link-table xref-fig">Table 5</a>). Consequently, the DNA methylation status was unchanged throughout the tissues, remaining constantly ‘null’. These tendencies held irrespective of whether the PAPs were TB-PAPs or non-TB-PAPs (<em>P</em> &lt; 30E−9; evaluated by standard Fisher's test; <a reveal-id="tbl5" data-open="tbl5" class="link link-reveal link-table xref-fig">Table 5</a>). In contrast, the DNA methylation in CGI-less PAPs was highly variable among tissues. When the CGI-containing PAPs were ‘null’ methylated, the pattern of the DNA methylation of the other PAPs of the same gene, which were usually CGI-less, was similar to the overall distribution patterns.</p>                        <a id="4946957" scrollto-destination="4946957"></a>
<div content-id="tbl5" class="table-modal"><div class="table-wrap"><div id="tbl5" class="table-wrap-title" data-id="tbl5"><div class="label -label"><strong>Table 5</strong></div><div class="caption"><p>Changes in the DNA Methylation Status between CGI-containing and CGI-less Promoters (A) Numbers of observed patterns of DNA methylation status in CGI-containing PAPs and CGI-less PAPs are shown. The DNA methylation patterns observed in five tissues were summed. (B) Numbers of PAPs with “Changed” or “Not-Changed” DNA methylation status in five tissues are shown. (C) Numbers of DNA methylation patterns of CGI-less PAPs when their accompanying CGI-containing PAPs were “null” methylated.</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>CGI-containing PAPs </th><th>CGI-less PAPs </th></tr><tr><td colspan="3"><hr /> </td></tr></thead><tbody><tr><td colspan="3">(A) </td></tr><tr><td colspan="3"><hr /> </td></tr><tr><td>    Complete </td><td>5 (2%) </td><td>170 (34%) </td></tr><tr><td>    Null </td><td>202 (87%) </td><td>126 (25%) </td></tr><tr><td>    Composite </td><td>23 (10%) </td><td>189 (37%) </td></tr><tr><td>    Incomplete </td><td>2 (1%) </td><td>19 (4%) </td></tr><tr><td>    Total </td><td>232 (100%) </td><td>504 (100%) </td></tr></tbody></table></div><div content-id="tbl5" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>CGI-containing PAPs </th><th>CGI-less PAPs </th></tr><tr><td colspan="3"><hr /> </td></tr></thead><tbody><tr><td colspan="3">(A) </td></tr><tr><td colspan="3"><hr /> </td></tr><tr><td>    Complete </td><td>5 (2%) </td><td>170 (34%) </td></tr><tr><td>    Null </td><td>202 (87%) </td><td>126 (25%) </td></tr><tr><td>    Composite </td><td>23 (10%) </td><td>189 (37%) </td></tr><tr><td>    Incomplete </td><td>2 (1%) </td><td>19 (4%) </td></tr><tr><td>    Total </td><td>232 (100%) </td><td>504 (100%) </td></tr></tbody></table></div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th colspan="2">CGI-containing PAPs </th><th colspan="2">CGI-less PAPs </th><th colspan="2">PAP Total </th></tr><tr><th> </th><th colspan="2"><hr /> </th><th colspan="2"><hr /> </th><th colspan="2"><hr /> </th></tr><tr><th> </th><th>TB-PAP </th><th>Non-TB-PAP </th><th>TB-PAP </th><th>Non-TB-PAP </th><th>TB-PAP </th><th>Non-TB-PAP </th></tr></thead><tbody><tr><td colspan="7">(B) </td></tr><tr><td>    Changed </td><td>10 (25%) </td><td>1 (12%) </td><td>67 (80%) </td><td>15 (68%) </td><td>77 (62%) </td><td>16 (53%) </td></tr><tr><td>    Not-Changed </td><td>30 (75%) </td><td>7 (88%) </td><td>17 (20%) </td><td>7 (32%) </td><td>47 (38%) </td><td>14 (47%) </td></tr><tr><td>    Total </td><td>40 (100%) </td><td>8 (100%) </td><td>84 (100%) </td><td>22 (100%) </td><td>124 (100%) </td><td>30 (100%) </td></tr></tbody></table></div><div content-id="tbl5" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th colspan="2">CGI-containing PAPs </th><th colspan="2">CGI-less PAPs </th><th colspan="2">PAP Total </th></tr><tr><th> </th><th colspan="2"><hr /> </th><th colspan="2"><hr /> </th><th colspan="2"><hr /> </th></tr><tr><th> </th><th>TB-PAP </th><th>Non-TB-PAP </th><th>TB-PAP </th><th>Non-TB-PAP </th><th>TB-PAP </th><th>Non-TB-PAP </th></tr></thead><tbody><tr><td colspan="7">(B) </td></tr><tr><td>    Changed </td><td>10 (25%) </td><td>1 (12%) </td><td>67 (80%) </td><td>15 (68%) </td><td>77 (62%) </td><td>16 (53%) </td></tr><tr><td>    Not-Changed </td><td>30 (75%) </td><td>7 (88%) </td><td>17 (20%) </td><td>7 (32%) </td><td>47 (38%) </td><td>14 (47%) </td></tr><tr><td>    Total </td><td>40 (100%) </td><td>8 (100%) </td><td>84 (100%) </td><td>22 (100%) </td><td>124 (100%) </td><td>30 (100%) </td></tr></tbody></table></div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Accompanying CGI-less PAPs when CGI-containing PAPs = Null </th></tr></thead><tbody><tr><td colspan="2">(C) </td></tr><tr><td>    Complete </td><td>72 (34%) </td></tr><tr><td>    Null </td><td>49 (23%) </td></tr><tr><td>    Composite </td><td>83 (40%) </td></tr><tr><td>    Incomplete </td><td>6 (3%) </td></tr><tr><td>    Total </td><td>210 (100%) </td></tr></tbody></table></div><div content-id="tbl5" class="table-modal"><table><thead name="thead" align=""><tr><th>Accompanying CGI-less PAPs when CGI-containing PAPs = Null </th></tr></thead><tbody><tr><td colspan="2">(C) </td></tr><tr><td>    Complete </td><td>72 (34%) </td></tr><tr><td>    Null </td><td>49 (23%) </td></tr><tr><td>    Composite </td><td>83 (40%) </td></tr><tr><td>    Incomplete </td><td>6 (3%) </td></tr><tr><td>    Total </td><td>210 (100%) </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>The DNA methylation patterns observed in five tissues were summed.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4946957" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="tbl5" class="table-wrap-title" data-id="tbl5"><div class="label -label"><strong>Table 5</strong></div><div class="caption"><p>Changes in the DNA Methylation Status between CGI-containing and CGI-less Promoters (A) Numbers of observed patterns of DNA methylation status in CGI-containing PAPs and CGI-less PAPs are shown. The DNA methylation patterns observed in five tissues were summed. (B) Numbers of PAPs with “Changed” or “Not-Changed” DNA methylation status in five tissues are shown. (C) Numbers of DNA methylation patterns of CGI-less PAPs when their accompanying CGI-containing PAPs were “null” methylated.</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>CGI-containing PAPs </th><th>CGI-less PAPs </th></tr><tr><td colspan="3"><hr /> </td></tr></thead><tbody><tr><td colspan="3">(A) </td></tr><tr><td colspan="3"><hr /> </td></tr><tr><td>    Complete </td><td>5 (2%) </td><td>170 (34%) </td></tr><tr><td>    Null </td><td>202 (87%) </td><td>126 (25%) </td></tr><tr><td>    Composite </td><td>23 (10%) </td><td>189 (37%) </td></tr><tr><td>    Incomplete </td><td>2 (1%) </td><td>19 (4%) </td></tr><tr><td>    Total </td><td>232 (100%) </td><td>504 (100%) </td></tr></tbody></table></div><div content-id="tbl5" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>CGI-containing PAPs </th><th>CGI-less PAPs </th></tr><tr><td colspan="3"><hr /> </td></tr></thead><tbody><tr><td colspan="3">(A) </td></tr><tr><td colspan="3"><hr /> </td></tr><tr><td>    Complete </td><td>5 (2%) </td><td>170 (34%) </td></tr><tr><td>    Null </td><td>202 (87%) </td><td>126 (25%) </td></tr><tr><td>    Composite </td><td>23 (10%) </td><td>189 (37%) </td></tr><tr><td>    Incomplete </td><td>2 (1%) </td><td>19 (4%) </td></tr><tr><td>    Total </td><td>232 (100%) </td><td>504 (100%) </td></tr></tbody></table></div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th colspan="2">CGI-containing PAPs </th><th colspan="2">CGI-less PAPs </th><th colspan="2">PAP Total </th></tr><tr><th> </th><th colspan="2"><hr /> </th><th colspan="2"><hr /> </th><th colspan="2"><hr /> </th></tr><tr><th> </th><th>TB-PAP </th><th>Non-TB-PAP </th><th>TB-PAP </th><th>Non-TB-PAP </th><th>TB-PAP </th><th>Non-TB-PAP </th></tr></thead><tbody><tr><td colspan="7">(B) </td></tr><tr><td>    Changed </td><td>10 (25%) </td><td>1 (12%) </td><td>67 (80%) </td><td>15 (68%) </td><td>77 (62%) </td><td>16 (53%) </td></tr><tr><td>    Not-Changed </td><td>30 (75%) </td><td>7 (88%) </td><td>17 (20%) </td><td>7 (32%) </td><td>47 (38%) </td><td>14 (47%) </td></tr><tr><td>    Total </td><td>40 (100%) </td><td>8 (100%) </td><td>84 (100%) </td><td>22 (100%) </td><td>124 (100%) </td><td>30 (100%) </td></tr></tbody></table></div><div content-id="tbl5" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th colspan="2">CGI-containing PAPs </th><th colspan="2">CGI-less PAPs </th><th colspan="2">PAP Total </th></tr><tr><th> </th><th colspan="2"><hr /> </th><th colspan="2"><hr /> </th><th colspan="2"><hr /> </th></tr><tr><th> </th><th>TB-PAP </th><th>Non-TB-PAP </th><th>TB-PAP </th><th>Non-TB-PAP </th><th>TB-PAP </th><th>Non-TB-PAP </th></tr></thead><tbody><tr><td colspan="7">(B) </td></tr><tr><td>    Changed </td><td>10 (25%) </td><td>1 (12%) </td><td>67 (80%) </td><td>15 (68%) </td><td>77 (62%) </td><td>16 (53%) </td></tr><tr><td>    Not-Changed </td><td>30 (75%) </td><td>7 (88%) </td><td>17 (20%) </td><td>7 (32%) </td><td>47 (38%) </td><td>14 (47%) </td></tr><tr><td>    Total </td><td>40 (100%) </td><td>8 (100%) </td><td>84 (100%) </td><td>22 (100%) </td><td>124 (100%) </td><td>30 (100%) </td></tr></tbody></table></div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Accompanying CGI-less PAPs when CGI-containing PAPs = Null </th></tr></thead><tbody><tr><td colspan="2">(C) </td></tr><tr><td>    Complete </td><td>72 (34%) </td></tr><tr><td>    Null </td><td>49 (23%) </td></tr><tr><td>    Composite </td><td>83 (40%) </td></tr><tr><td>    Incomplete </td><td>6 (3%) </td></tr><tr><td>    Total </td><td>210 (100%) </td></tr></tbody></table></div><div content-id="tbl5" class="table-modal"><table><thead name="thead" align=""><tr><th>Accompanying CGI-less PAPs when CGI-containing PAPs = Null </th></tr></thead><tbody><tr><td colspan="2">(C) </td></tr><tr><td>    Complete </td><td>72 (34%) </td></tr><tr><td>    Null </td><td>49 (23%) </td></tr><tr><td>    Composite </td><td>83 (40%) </td></tr><tr><td>    Incomplete </td><td>6 (3%) </td></tr><tr><td>    Total </td><td>210 (100%) </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>The DNA methylation patterns observed in five tissues were summed.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4946957" target="_blank">View Large</a></div></div>                            <h2 scrollto-destination=4946958 id="4946958" class="section-title" >4. Discussion</h2>
<p>In this study, we characterized the DNA methylation status at 181 PAPs in 61 genes which were identified by our previous full-length cDNA analyses. This is the first report describing the profiles of the DNA methylation of PAPs in a variety of tissues and may be the largest collection of data on the methylation of human promoters in general for normal tissues. In total, we analyzed the methylation status of 8612 CpG dinucleotides in 181 PAPs located in the proximal regions of transcriptional start sites (also see Supplementary Table S2). Although this number is small relative to the total number of CpG dinucleotides existing within the whole genome, this novel analysis of these CpG islands should be significant, because they may have direct biological consequences via modulating the transcription levels of genes.</p><p>Many of the previous studies using DNA-chip-based technologies were focused on comparing the DNA methylation status between cancerous cells and normal cells.<a reveal-id="b21" data-open="b21" class="link link-ref link-reveal xref-bibr">21</a>,<a reveal-id="b22" data-open="b22" class="link link-ref link-reveal xref-bibr">22</a> From the data we collected here, we found: (i) PAPs of the same gene tend to have different methylation pattern variation depending on the tissue; (ii) DNA methylation status differs among tissues for the majority of the individual PAPs; (iii) CGI-containing PAPs are an exception to observation, (ii) in that they tend to have a uniform ‘null’ pattern throughout the tissues examined.</p><p>The last result is somewhat of a confirmation of the findings of our previous study, although it was not previously tested on this scale. We made a comprehensive analysis of CpG islands on human chromosome 21 and observed that 84% of CpG islands located within promoter regions were ‘null’ methylated in leukocytes.<a reveal-id="b19" data-open="b19" class="link link-ref link-reveal xref-bibr">19</a> The general lack of methylation of promoter CpG islands, regardless of whether the corresponding genes are expressed or not, was also suggested by other previous studies, although the numbers of the assayed promoters and cell types were limited.<a reveal-id="b23" data-open="b23" class="link link-ref link-reveal xref-bibr">23</a> In this study, we assayed 181 PAPs in five normal tissues and observed that most of the CGI-containing PAPs were constantly ‘null’ methylated irrespective of the tissue (<a reveal-id="tbl5" data-open="tbl5" class="link link-reveal link-table xref-fig">Table 5</a>). Based on this observation, we concluded that methylation appears not to be the reason for the biased expression of most of the CGI-containing TB-PAPs.</p><p>In contrast to the case of the CGI-containing TB-PAPs, we found surprisingly wide variation in the methylation pattern among the tissues examined. We observed that the majority of TB-PAPs showed alteration of their DNA methylation status among tissues (62%; <a reveal-id="tbl5" data-open="tbl5" class="link link-reveal link-table xref-fig">Table 5</a>). If we counted only CGI-less TB-PAPs, the proportion went up to 80% (<a reveal-id="tbl5" data-open="tbl5" class="link link-reveal link-table xref-fig">Table 5</a>). We also found that TB-PAPs seemed to have more variation of their methylation patterns than non-TB-PAPs (<a reveal-id="tbl3" data-open="tbl3" class="link link-reveal link-table xref-fig">Table 3</a>). We examined only five tissues in this study. We expect that the proportion of PAPs that show a variable methylation pattern among the tissues may increase if the number of tissues assayed is increased. Since we assayed 181 PAPs that were located on various chromosomes, these results indicate that various regions in the human genome are differentially methylated among various tissues. This suggests that the methylation of promoters is more dynamically regulated from one tissue to another than previously anticipated.</p><p>Among 48 genes for which we compared PAPs within the same gene, only about 10% had the same methylation pattern in all five tissues. All of these methylation patterns were ‘null’, as described in <a reveal-id="tbl3" data-open="tbl3" class="link link-reveal link-table xref-fig">Table 3</a>. Actually, CGI-containing PAPs accounted for these uniform ‘null’ patterns. Seven PAPs contained CpG-islands out of 10 PAPs of 5 genes that showed perfect matches of the methylation patterns (<a reveal-id="tbl3" data-open="tbl3" class="link link-reveal link-table xref-fig">Table 3</a>). We could not find any simple correlation of methylation pattern variation between PAPs within the same gene. The methylation of PAPs thus seems to be individually controlled even within the same gene.</p><p>We also searched for specific patterns of methylation according to the expression pattern or expression level of the mRNA transcribed from a PAP. We observed that high levels expression correlated with the ‘null’ or relatively light methylation pattern of PAPs, although the statistical significance was not as clear as in the above cases, possibly due to the limited size of the current dataset. There seems to be a tendency for CGI-containing PAPs to be ‘null’ methylated and to show high expression levels. Our previous findings suggested that most genes have only one CGI-containing PAP among all their PAPs.<a reveal-id="b1" data-open="b1" class="link link-ref link-reveal xref-bibr">1</a> It is possible that this unique CGI-containing PAP is the major promoter of the gene, playing a key role among the group of alternative promoters.</p><p>Together, in this study, we showed that information about DNA methylation is as indispensable as the mere DNA sequences of the promoters for a thorough understanding of the regulation of the expression of the genes encoded by the human genome. Modifications of the histones, methylation of the DNA, and the chromatin structure are suggested to be related with each other.<a reveal-id="b24" data-open="b24" class="link link-ref link-reveal xref-bibr">24</a> Thus, information about DNA methylation may in general be useful as an indicator of the chromatin structure. Advances in genome and transcriptome research have opened a way for decoding the gene expression regulation of the genomic DNA sequences. Actually, in yeast, a recent report demonstrated that it was possible to predict the level of gene expression directly from the genomic sequence with some accuracy, using clustered microarray data as an education dataset.<a reveal-id="b25" data-open="b25" class="link link-ref link-reveal xref-bibr">25</a> The sequence information of the DNA together with its higher-level chromatin structure will enable us to better understand how the code of the DNA directs the complex regulation of gene expression in human cells. Based on that knowledge, we should eventually be able to understand how the alternative utilization of the promoters plays roles in the diversification of gene functions.</p><p>We are grateful to K. Abe, K. Toya and K. Imamura for technical support. We are also thankful to E. Nakajima for critical reading of the manuscript. This work was supported by grants from the New Energy and Industrial Technology Development Organization (NEDO) project of the Ministry of Economy, Trade and Industry (METI) of Japan, the Japan Key Technology Center project of METI of JAPAN and a Grant-in-Aid for Scientific Research on Priority Areas from the Ministry of Education, Science, Sports and Culture of Japan.</p><p><strong>Supplementary Data:</strong> Supplementary data are available online at <a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org" target="_blank"></a></p>                            <h2 scrollto-destination=4946969 id="4946969" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="b1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kimura</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Wakamatsu</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Suzuki</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">Diversification of transcriptional modulation: large-scale identification and characterization of putative alternative promoters of human genes</div>, <div class="source">Genome Res.</div> , <div class="year">2006</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">55</div>-<div class="lpage">65</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Diversification%20of%20transcriptional%20modulation%3A%20large-scale%20identification%20and%20characterization%20of%20putative%20alternative%20promoters%20of%20human%20genes&amp;author=K.%20Kimura&amp;author=A.%20Wakamatsu&amp;author=Y.%20Suzuki&amp;publication_year=2006&amp;journal=Genome%20Res.&amp;volume=16&amp;pages=55-65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.4039406" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.4039406" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.4039406"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16344560" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kim</div> <div class="given-names">T. H.</div></div>,  <div class="name"><div class="surname">Barrera</div> <div class="given-names">L. O.</div></div>,  <div class="name"><div class="surname">Zheng</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">A high-resolution map of active promoters in the human genome</div>, <div class="source">Nature</div> , <div class="year">2005</div>, vol. <div class="volume">436</div> (pg. <div class="fpage">876</div>-<div class="lpage">880</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20high-resolution%20map%20of%20active%20promoters%20in%20the%20human%20genome&amp;author=T.%20H.%20Kim&amp;author=L.%20O.%20Barrera&amp;author=M.%20Zheng&amp;publication_year=2005&amp;journal=Nature&amp;volume=436&amp;pages=876-880" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature03877" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature03877" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature03877"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15988478" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kapranov</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Drenkow</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Cheng</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Examples of the complex architecture of the human transcriptome revealed by RACE and high-density tiling arrays</div>, <div class="source">Genome Res.</div> , <div class="year">2005</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">987</div>-<div class="lpage">997</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Examples%20of%20the%20complex%20architecture%20of%20the%20human%20transcriptome%20revealed%20by%20RACE%20and%20high-density%20tiling%20arrays&amp;author=P.%20Kapranov&amp;author=J.%20Drenkow&amp;author=J.%20Cheng&amp;publication_year=2005&amp;journal=Genome%20Res.&amp;volume=15&amp;pages=987-997" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.3455305" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.3455305" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.3455305"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15998911" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Carninci</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Sandelin</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Lenhard</div> <div class="given-names">B.</div></div>, et al. </div><div class="article-title">Genome-wide analysis of mammalian promoter architecture and evolution</div>, <div class="source">Nat. Genet.</div> , <div class="year">2006</div>, vol. <div class="volume">38</div> (pg. <div class="fpage">608</div>-<div class="lpage">609</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20analysis%20of%20mammalian%20promoter%20architecture%20and%20evolution&amp;author=P.%20Carninci&amp;author=A.%20Sandelin&amp;author=B.%20Lenhard&amp;publication_year=2006&amp;journal=Nat.%20Genet.&amp;volume=38&amp;pages=608-609" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng1789" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng1789" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng1789"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16736012" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Landry</div> <div class="given-names">J. R.</div></div>,  <div class="name"><div class="surname">Mager</div> <div class="given-names">D. L.</div></div>,  <div class="name"><div class="surname">Wilhelm</div> <div class="given-names">B. T.</div></div>. </div><div class="article-title">Complex controls: the role of alternative promoters in mammalian genomes</div>, <div class="source">Trends Genet.</div> , <div class="year">2003</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">640</div>-<div class="lpage">648</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complex%20controls%3A%20the%20role%20of%20alternative%20promoters%20in%20mammalian%20genomes&amp;author=J.%20R.%20Landry&amp;author=D.%20L.%20Mager&amp;author=B.%20T.%20Wilhelm&amp;publication_year=2003&amp;journal=Trends%20Genet.&amp;volume=19&amp;pages=640-648" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tig.2003.09.014" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tig.2003.09.014" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tig.2003.09.014"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14585616" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Khaitovich</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Hellmann</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Enard</div> <div class="given-names">W.</div></div>, et al. </div><div class="article-title">Parallel patterns of evolution in the genomes and transcriptomes of humans and chimpanzees</div>, <div class="source">Science</div> , <div class="year">2005</div>, vol. <div class="volume">309</div> (pg. <div class="fpage">1850</div>-<div class="lpage">1854</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Parallel%20patterns%20of%20evolution%20in%20the%20genomes%20and%20transcriptomes%20of%20humans%20and%20chimpanzees&amp;author=P.%20Khaitovich&amp;author=I.%20Hellmann&amp;author=W.%20Enard&amp;publication_year=2005&amp;journal=Science&amp;volume=309&amp;pages=1850-1854" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1108296" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1108296" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1108296"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16141373" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">King</div> <div class="given-names">M. C.</div></div>,  <div class="name"><div class="surname">Wilson</div> <div class="given-names">A. C.</div></div>. </div><div class="article-title">Evolution at two levels in humans and chimpanzees</div>, <div class="source">Science</div> , <div class="year">1975</div>, vol. <div class="volume">188</div> (pg. <div class="fpage">107</div>-<div class="lpage">116</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20at%20two%20levels%20in%20humans%20and%20chimpanzees&amp;author=M.%20C.%20King&amp;author=A.%20C.%20Wilson&amp;publication_year=1975&amp;journal=Science&amp;volume=188&amp;pages=107-116" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1090005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1090005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1090005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1090005" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Suzuki</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Tsunoda</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Sese</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Identification and characterization of the potential promoter regions of 1031 kinds of human genes</div>, <div class="source">Genome Res.</div> , <div class="year">2001</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">677</div>-<div class="lpage">684</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20and%20characterization%20of%20the%20potential%20promoter%20regions%20of%201031%20kinds%20of%20human%20genes&amp;author=Y.%20Suzuki&amp;author=T.%20Tsunoda&amp;author=J.%20Sese&amp;publication_year=2001&amp;journal=Genome%20Res.&amp;volume=11&amp;pages=677-684" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.GR-1640R" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.GR-1640R" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.GR-1640R"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11337467" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Suzuki</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Sugano</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Construction of a full-length enriched and a 5′-end enriched cDNA library using the oligo-capping method</div>, <div class="source">Methods Mol. Biol.</div> , <div class="year">2003</div>, vol. <div class="volume">221</div> (pg. <div class="fpage">73</div>-<div class="lpage">91</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20a%20full-length%20enriched%20and%20a%205%E2%80%B2-end%20enriched%20cDNA%20library%20using%20the%20oligo-capping%20method&amp;author=Y.%20Suzuki&amp;author=S.%20Sugano&amp;publication_year=2003&amp;journal=Methods%20Mol.%20Biol.&amp;volume=221&amp;pages=73-91" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12703735" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yamashita</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Suzuki</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Wakaguri</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Tsuritani</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Nakai</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Sugano</div> <div class="given-names">S.</div></div>. </div><div class="article-title">DBTSS: database of human transcription start sites, progress report 2006</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2006</div>, vol. <div class="volume">34</div> (pg. <div class="fpage">D86</div>-<div class="lpage">D89</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DBTSS%3A%20database%20of%20human%20transcription%20start%20sites%2C%20progress%20report%202006&amp;author=R.%20Yamashita&amp;author=Y.%20Suzuki&amp;author=H.%20Wakaguri&amp;author=K.%20Tsuritani&amp;author=K.%20Nakai&amp;author=S.%20Sugano&amp;publication_year=2006&amp;journal=Nucleic%20Acids%20Res.&amp;volume=34&amp;pages=D86-D89" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkj129" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkj129" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkj129"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16381981" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gardiner-Garden</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Frommer</div> <div class="given-names">M.</div></div>. </div><div class="article-title">CpG islands in vertebrate genomes</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">1987</div>, vol. <div class="volume">196</div> (pg. <div class="fpage">261</div>-<div class="lpage">282</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CpG%20islands%20in%20vertebrate%20genomes&amp;author=M.%20Gardiner-Garden&amp;author=M.%20Frommer&amp;publication_year=1987&amp;journal=J.%20Mol.%20Biol.&amp;volume=196&amp;pages=261-282" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0022-2836(87)90689-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0022-2836(87)90689-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0022-2836(87)90689-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3656447" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jones</div> <div class="given-names">P. A.</div></div>,  <div class="name"><div class="surname">Baylin</div> <div class="given-names">S. B.</div></div>. </div><div class="article-title">The fundamental role of epigenetic events in cancer</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="year">2002</div>, vol. <div class="volume">3</div> (pg. <div class="fpage">415</div>-<div class="lpage">428</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20fundamental%20role%20of%20epigenetic%20events%20in%20cancer&amp;author=P.%20A.%20Jones&amp;author=S.%20B.%20Baylin&amp;publication_year=2002&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=3&amp;pages=415-428" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg962" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg962" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg962"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12042769" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Laird</div> <div class="given-names">P. W.</div></div>. </div><div class="article-title">Cancer epigenetics</div>, <div class="source">Hum. Mol. Genet.</div> , <div class="year">2005</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">R65</div>-<div class="lpage">R76</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cancer%20epigenetics&amp;author=P.%20W.%20Laird&amp;publication_year=2005&amp;journal=Hum.%20Mol.%20Genet.&amp;volume=14&amp;pages=R65-R76" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/hmg/ddi113" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fhmg%2Fddi113" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fhmg%2Fddi113"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15809275" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Clark</div> <div class="given-names">S. J.</div></div>,  <div class="name"><div class="surname">Melki</div> <div class="given-names">J.</div></div>. </div><div class="article-title">DNA methylation and gene silencing in cancer: which is the guilty party?</div>, <div class="source">Oncogene</div> , <div class="year">2002</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">5380</div>-<div class="lpage">5387</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20methylation%20and%20gene%20silencing%20in%20cancer%3A%20which%20is%20the%20guilty%20party%3F&amp;author=S.%20J.%20Clark&amp;author=J.%20Melki&amp;publication_year=2002&amp;journal=Oncogene&amp;volume=21&amp;pages=5380-5387" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/sj.onc.1205598" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsj.onc.1205598" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsj.onc.1205598"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12154400" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Murrell</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Rakyan</div> <div class="given-names">V. K.</div></div>,  <div class="name"><div class="surname">Beck</div> <div class="given-names">S.</div></div>. </div><div class="article-title">From genome to epigenome</div>, <div class="source">Hum. Mol. Genet.</div> , <div class="year">2005</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">R3</div>-<div class="lpage">R10</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=From%20genome%20to%20epigenome&amp;author=A.%20Murrell&amp;author=V.%20K.%20Rakyan&amp;author=S.%20Beck&amp;publication_year=2005&amp;journal=Hum.%20Mol.%20Genet.&amp;volume=14&amp;pages=R3-R10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/hmg/ddi110" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fhmg%2Fddi110" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fhmg%2Fddi110"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15809270" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fazzari</div> <div class="given-names">M. J.</div></div>,  <div class="name"><div class="surname">Greally</div> <div class="given-names">J. M.</div></div>. </div><div class="article-title">Epigenomics: beyond CpG islands</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="year">2004</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">446</div>-<div class="lpage">455</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Epigenomics%3A%20beyond%20CpG%20islands&amp;author=M.%20J.%20Fazzari&amp;author=J.%20M.%20Greally&amp;publication_year=2004&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=5&amp;pages=446-455" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg1349" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg1349" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg1349"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15153997" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Clark</div> <div class="given-names">S. J.</div></div>,  <div class="name"><div class="surname">Harrison</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Paul</div> <div class="given-names">C. L.</div></div>,  <div class="name"><div class="surname">Frommer</div> <div class="given-names">M.</div></div>. </div><div class="article-title">High sensitivity mapping of methylated cytosines</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1994</div>, vol. <div class="volume">22</div> (pg. <div class="fpage">2990</div>-<div class="lpage">2997</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High%20sensitivity%20mapping%20of%20methylated%20cytosines&amp;author=S.%20J.%20Clark&amp;author=J.%20Harrison&amp;author=C.%20L.%20Paul&amp;author=M.%20Frommer&amp;publication_year=1994&amp;journal=Nucleic%20Acids%20Res.&amp;volume=22&amp;pages=2990-2997" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/22.15.2990" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F22.15.2990" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F22.15.2990"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8065911" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Frommer</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">McDonald</div> <div class="given-names">L. E.</div></div>,  <div class="name"><div class="surname">Millar</div> <div class="given-names">D. S.</div></div>, et al. </div><div class="article-title">A genomic sequencing protocol that yields a positive display of 5-methylcytosine residues in individual DNA strands</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">1992</div>, vol. <div class="volume">89</div> (pg. <div class="fpage">1827</div>-<div class="lpage">1831</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20genomic%20sequencing%20protocol%20that%20yields%20a%20positive%20display%20of%205-methylcytosine%20residues%20in%20individual%20DNA%20strands&amp;author=M.%20Frommer&amp;author=L.%20E.%20McDonald&amp;author=D.%20S.%20Millar&amp;publication_year=1992&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=89&amp;pages=1827-1831" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.89.5.1827" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.89.5.1827" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.89.5.1827"> </span></div> </div></div></div></div></div><div content-id="b19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yamada</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Watanabe</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Miura</div> <div class="given-names">F.</div></div>, et al. </div><div class="article-title">A comprehensive analysis of allelic methylation status of CpG islands on human chromosome 21q</div>, <div class="source">Genome Res.</div> , <div class="year">2004</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">247</div>-<div class="lpage">266</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20comprehensive%20analysis%20of%20allelic%20methylation%20status%20of%20CpG%20islands%20on%20human%20chromosome%2021q&amp;author=Y.%20Yamada&amp;author=H.%20Watanabe&amp;author=F.%20Miura&amp;publication_year=2004&amp;journal=Genome%20Res.&amp;volume=14&amp;pages=247-266" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.1351604" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.1351604" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.1351604"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14762061" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kuhlenbaumer</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Hannibal</div> <div class="given-names">M. C.</div></div>,  <div class="name"><div class="surname">Nelis</div> <div class="given-names">E.</div></div>, et al. </div><div class="article-title">Mutations in SEPT9 cause hereditary neuralgic amyotrophy</div>, <div class="source">Nat. Genet.</div> , <div class="year">2005</div>, vol. <div class="volume">37</div> (pg. <div class="fpage">1044</div>-<div class="lpage">1046</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mutations%20in%20SEPT9%20cause%20hereditary%20neuralgic%20amyotrophy&amp;author=G.%20Kuhlenbaumer&amp;author=M.%20C.%20Hannibal&amp;author=E.%20Nelis&amp;publication_year=2005&amp;journal=Nat.%20Genet.&amp;volume=37&amp;pages=1044-1046" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng1649" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng1649" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng1649"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16186812" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hatada</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Fukasawa</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Kimura</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Genome-wide profiling of promoter methylation in human</div>, <div class="source">Oncogene.</div> , <div class="year">2006</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">3059</div>-<div class="lpage">3064</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20profiling%20of%20promoter%20methylation%20in%20human&amp;author=I.%20Hatada&amp;author=M.%20Fukasawa&amp;author=M.%20Kimura&amp;publication_year=2006&amp;journal=Oncogene.&amp;volume=25&amp;pages=3059-3064" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/sj.onc.1209331" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsj.onc.1209331" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsj.onc.1209331"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16407832" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fukasawa</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Kimura</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Morita</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Microarray analysis of promoter methylation in lung cancers</div>, <div class="source">J. Hum. Genet.</div> , <div class="year">2006</div>, vol. <div class="volume">51</div> (pg. <div class="fpage">368</div>-<div class="lpage">374</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microarray%20analysis%20of%20promoter%20methylation%20in%20lung%20cancers&amp;author=M.%20Fukasawa&amp;author=M.%20Kimura&amp;author=S.%20Morita&amp;publication_year=2006&amp;journal=J.%20Hum.%20Genet.&amp;volume=51&amp;pages=368-374" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10038-005-0355-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10038-005-0355-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10038-005-0355-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16435073" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Antequera</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Boyes</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Bird</div> <div class="given-names">A.</div></div>. </div><div class="article-title">High levels of de novo methylation and altered chromatin structure at CpG islands in cell lines</div>, <div class="source">Cell</div> , <div class="year">1990</div>, vol. <div class="volume">62</div> (pg. <div class="fpage">503</div>-<div class="lpage">514</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High%20levels%20of%20de%20novo%20methylation%20and%20altered%20chromatin%20structure%20at%20CpG%20islands%20in%20cell%20lines&amp;author=F.%20Antequera&amp;author=J.%20Boyes&amp;author=A.%20Bird&amp;publication_year=1990&amp;journal=Cell&amp;volume=62&amp;pages=503-514" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0092-8674(90)90015-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0092-8674(90)90015-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0092-8674(90)90015-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1974172" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fuks</div> <div class="given-names">F.</div></div>. </div><div class="article-title">DNA methylation and histone modifications: teaming up to silence genes</div>, <div class="source">Curr. Opin. Genet. Dev.</div> , <div class="year">2005</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">490</div>-<div class="lpage">495</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20methylation%20and%20histone%20modifications%3A%20teaming%20up%20to%20silence%20genes&amp;author=F.%20Fuks&amp;publication_year=2005&amp;journal=Curr.%20Opin.%20Genet.%20Dev.&amp;volume=15&amp;pages=490-495" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gde.2005.08.002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gde.2005.08.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gde.2005.08.002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16098738" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Beer</div> <div class="given-names">M. A.</div></div>,  <div class="name"><div class="surname">Tavazoie</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Predicting gene expression from sequence</div>, <div class="source">Cell</div> , <div class="year">2004</div>, vol. <div class="volume">117</div> (pg. <div class="fpage">185</div>-<div class="lpage">198</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Predicting%20gene%20expression%20from%20sequence&amp;author=M.%20A.%20Beer&amp;author=S.%20Tavazoie&amp;publication_year=2004&amp;journal=Cell&amp;volume=117&amp;pages=185-198" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0092-8674(04)00304-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0092-8674(04)00304-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0092-8674(04)00304-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15084257" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2006. Kazusa DNA Research Institute</div><div class="license"><div class="license-p">The online version of this article has been published under an open access model. Users are entitled to use, reproduce, disseminate, or display the open access version of this article for non-commercial purposes provided that: the original authorship is properly and fully attributed; the Journal and Oxford University Press are attributed as the original place of publication with the correct citation details given; if an article is subsequently reproduced or disseminated not in its entirety but only in part or as a derivative work this must be clearly indicated. For commercial re-use, please contact journals.permissions@oxfordjournals.org</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=348476&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/4/10.1093/dnares/dsl008/2/dsl008-Supplementary_Table_4.xls?Expires=2147483647&amp;Signature=BQxP7e-tJ4aGGEM8xglmdHEVtY2XRL3W4ON-l67T5fylibnYzZ1XwiyFZQttAg0c1HAAZvvmY4WxfW9tws5qzmSf~MOP67R7lmKUqftm34DyERfsUTwZdUozTPdVzThhfHuyH7caiaVKHOBszg~GEZvOs4ifiq4DQrhnxZE3isQJd7YxjEDaWZVMxreFRLVZtJX-i10gYQntKlSYyiaieMl0j~GvYPKOWIBjsSfJdUZs8lNWwn8jHt~rNPvefWFVy7wlKqbWLM0NqgUjMM9oNKIYpJ6WnunBMroXw6Rv9CQfgxLTiae3AumV6tMnJBdda~nt67NlCE-NY6imxxv4-g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-7</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/4/10.1093/dnares/dsl008/2/dsl008-Supplementary_Table_3.xls?Expires=2147483647&amp;Signature=LZ-XGmY7iP0OrWJXdufNSA2xUxHv-RaAHK35nXC97biIf9uPLsiO2sz72Au3SK8mKGVWPVrYBB0r1isQz5cFVUBm-Ue-HI8EZV3IJ~GvSn01t35QJGcuhww1dkzSDktpT2vxaDiFKuvhs6sfp5v3mDIgHcdYqxwPMr-emaq09X9ygPpAlgYIgLXZ1xT25btOP3PmEFKjK-GQQZTDntoeRios1lRRo70F4D2vk8CcUOEJd8A3OEm3ir6J42ur6PZtrLtXdiil9Z8GuLLV7MP8Z7jeWkZyLYj12aUFbxrVV8LhVOSHfjKm-jupaYDQ8ITCJZ1fWgckhQEo5d5U5hWsGg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-6</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/4/10.1093/dnares/dsl008/2/dsl008-Supplementary_Table_2C.xls?Expires=2147483647&amp;Signature=tUy2KU1-hj1gFWbjVptsymVW9G0M~5JWy7-Yz6qXbdFZOs5mED39Fu53M-QZEWHDIKBeRQVcOk~4nYRRIZKceNLxCnOI1nuksUXQLNoJ6kolYLE0qxR~irAnT53vp0BIvjdVzc-XCWh9Dst96rrB21vGVbfYeaFnXLuA3gTt7CmqG5RO6Xv1-Rs4Jf~-HCC6Ry8Uciqd6KB1w~Vtvg4LASZmW9PmG8ThtsPOPRJFUi4B6xIO-BTBbFGzIIhmHjH-HOEoywuM8ahB4CJtD479L5880QhaxI~QAD~BzHkd4CQaxDX2nOLUXC7obHw262NfhGn9xnKxu3h1c-CadaIEWA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-5</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/4/10.1093/dnares/dsl008/2/dsl008-Supplementary_Table_2B.xls?Expires=2147483647&amp;Signature=dltBQaOfsYFLN6EtBh1qOtE9q9cs7ERTyBr5fS8eUjKOBfM-0aeoRO1UaICtNpgZHKk5u0-V059Hug1kSxJxW6rJEbejxMgWVafhFr582D-hSxnRh2KWX~EwRaJHLfhmGmxRMB3NGzcq6yWASAWLuzSQUW-9zJnH2iyneWeOTV701JrZoYEWkJc07JKizNzUwPrv1lKnFItirJbbrjVXVkJS9e0c~qW4AQd6NgZ9vRZ6yCZbEFdw1Di7da2mPwxnev5dytAKR~Pfu968JraK95IWLZ52dEarjfd3VdDwvFx0KCOQGDEdgz5ME~DO5UNC1UThslVjg-0AzZC~5aIdFA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-4</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/4/10.1093/dnares/dsl008/2/dsl008-Supplementary_Table_2A.xls?Expires=2147483647&amp;Signature=bvTW1VdswrOfHCOXW7MoSIGup1w8tKdfQFdFp63ApA6tdkkWOhNfpYwgZgz5b7tqLDXDi29TDYDzmIQPBMsaovF-O37z9Vf38gXG0vvv9ccM8PQLEb-Wnl38XMLA9uUcC1lHdmcOpr23yowcw0rH9gmkew2VMXFtvBvMoOMNhGAE8cse2MjrfK29-14M~ywciIjTs1QaVcKQsSkgr7ZX7WPZwyfNoDsaFT15bV5h0sonOKhAGGAYv1xbotKqS8X7mJ9D84oEnBA3qzaGYJ0GSxCiynsbGqMbd1uO7Y69o6i-i5dsm-1DkR44ggfGw8tUc9OKdegYe9fkxq0CwgD4qQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/4/10.1093/dnares/dsl008/2/dsl008-Supplementary_Table_1.xls?Expires=2147483647&amp;Signature=DnxcNWYRLMHyKkGfOCn91oo2-1XJjYWqc1qKh2Y1T3oQ4ozcpL80unDhQ-9Onu~pN9prAwqF8VUfJPZO4FOEYryJyErE5EjZLZ5H5nTGzlxso5x6qceX2uKECcyuk97Dba6uo9voRooX-7iqTR5aCSmDNz0xarZZUqJNpy8P-Qh5Yud4Nn0r3dxbpMUL4EP4~C~R-H1oNin4Cm7xEB6kpxcbX9mGPflGwamqYauDaO1ZlzUBWjJbEPuCzpBT0qSc8o9Z8skQa2o~xqDMACPFvl2l3iJLu6z0BHTi4N-hH8PNeuUfWuJfEF4FsmgImigi6eqv672eA5xLyWcwsVJl~g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/4/10.1093/dnares/dsl008/2/dsl008-Supplementary_Figures.pdf?Expires=2147483647&amp;Signature=BY0fSr2tA-NgqeHiV~g08BUoX9AkjOQtEI47DjKPSoqfbq7GGQkzLjx-D7SXw9JjF546uyRiEdU2ZVuq8vICIFnGjBasy-GdM6-A6sO8IDROk0YJaJpWeoyXSHQL9FbhMn7O~OnCywa7MavqFEMm8PmFUKk9EHIKWW1bW9YMPjimRvFBwOjjbB3UOY5vorRSb1KmJoJ7hPjUjLE-ThHAX4pQSpMStzhk-vvK9kaB9veumDQo-igPC3LmjUKJlm05xBPPhqopae6RDgJiY4M2cAN86npx7EDItq1~~hQ7EszaGtBcfqjzVoubfhD3BY4hfhnMBAPL9tznGQ~lsOMWRw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">220</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000242120100003&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">19</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsl008"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="348476"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000242120100003&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f13%2f4%2f155%2f348476">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000242120100003&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (19)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F13%2F4%2F155%2F348476" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/348476'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Diverse DNA Methylation Statuses at Alternative Promoters of Human Genes in Various Tissues" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsl008");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f13%2f4%2f155%2f348476">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

