

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/22/4/269/348206","@type":"ScholarlyArticle","name":"High-fidelity target sequencing of individual molecules identified using barcode sequences:  de novo  detection and absolute quantitation of mutations in plasma cell-free DNA from cancer patients","datePublished":"2015-06-29","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/22/4","@type":"PublicationIssue","issueNumber":"4","datePublished":"2015-08-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsv010","keywords":["massively parallel DNA sequencer","circulating tumour DNA","barcode sequences"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/22/4/269/348206","author":[{"name":"Kukita, Yoji","affiliation":"1  Department of Molecular and Medical Genetics ,  Research Institute, Osaka Medical Center for Cancer and Cardiovascular Diseases ,  Osaka 537-8511 ,  Japan","@type":"Person"},{"name":"Matoba, Ryo","affiliation":"2  DNA Chip Research Inc. ,  Yokohama 230-0045 ,  Japan","@type":"Person"},{"name":"Uchida, Junji","affiliation":"3  Department of Thoracic Oncology ,  Osaka Medical Center for Cancer and Cardiovascular Diseases ,  Osaka 537-8511 ,  Japan","@type":"Person"},{"name":"Hamakawa, Takuya","affiliation":"1  Department of Molecular and Medical Genetics ,  Research Institute, Osaka Medical Center for Cancer and Cardiovascular Diseases ,  Osaka 537-8511 ,  Japan    4  Department of Gastroenterological Surgery ,  Osaka University Graduate School of Medicine ,  Osaka 565-0871 ,  Japan","@type":"Person"},{"name":"Doki, Yuichiro","affiliation":"4  Department of Gastroenterological Surgery ,  Osaka University Graduate School of Medicine ,  Osaka 565-0871 ,  Japan","@type":"Person"},{"name":"Imamura, Fumio","affiliation":"3  Department of Thoracic Oncology ,  Osaka Medical Center for Cancer and Cardiovascular Diseases ,  Osaka 537-8511 ,  Japan","@type":"Person"},{"name":"Kato, Kikuya","affiliation":"1  Department of Molecular and Medical Genetics ,  Research Institute, Osaka Medical Center for Cancer and Cardiovascular Diseases ,  Osaka 537-8511 ,  Japan","@type":"Person"}],"description":"Abstract.  Circulating tumour DNA (ctDNA) is an emerging field of cancer research. However, current ctDNA analysis is usually restricted to one or a few mutatio","pageStart":"269","pageEnd":"277","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/4/1/m_cover.gif?Expires=2147483647&Signature=Ufin3LLH~Iqb2kw-hetKqzdx2YgWiSwuSGu7FcRPyZ0YMXPxK67~EvZK~D~FaBMUMtkfx2C3nicLRXrG7LQSzXkxs1Qyu1rF6Hbn-vB-awwpaXOgx90CafYjmsJlX0TeG5UBY2DnHbv1s5YegTLh88o5c9kHiz4IICTc2SgUfnIM~hO2aQPqVNMfxHD9dsgu8Ly4qu3h4vkVcBZFQVgjv-cFeqoqOFVvCZqW6Vq3z-Hi91~oLc58RRI-PjbrcLYKmszwtDPbL2sZKRQky2J2HWAa2PZzNl-CMH92uFmvRLgkUeyANQZ-s6eHxAaArS5e4XiVxAKXHDRMAVOPl6~asA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"High-fidelity target sequencing of individual molecules identified using barcode sequences:  de novo  detection and absolute quantitation of mutations in plasma cell-free DNA from cancer patients","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/4/1/m_cover.gif?Expires=2147483647&Signature=Ufin3LLH~Iqb2kw-hetKqzdx2YgWiSwuSGu7FcRPyZ0YMXPxK67~EvZK~D~FaBMUMtkfx2C3nicLRXrG7LQSzXkxs1Qyu1rF6Hbn-vB-awwpaXOgx90CafYjmsJlX0TeG5UBY2DnHbv1s5YegTLh88o5c9kHiz4IICTc2SgUfnIM~hO2aQPqVNMfxHD9dsgu8Ly4qu3h4vkVcBZFQVgjv-cFeqoqOFVvCZqW6Vq3z-Hi91~oLc58RRI-PjbrcLYKmszwtDPbL2sZKRQky2J2HWAa2PZzNl-CMH92uFmvRLgkUeyANQZ-s6eHxAaArS5e4XiVxAKXHDRMAVOPl6~asA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="High-fidelity target sequencing of individual molecules identified using barcode sequences:  de novo  detection and absolute quantitation of mutations in plasma cell-free DNA from cancer patients" />
    <meta property="og:description" content="Abstract.  Circulating tumour DNA (ctDNA) is an emerging field of cancer research. However, current ctDNA analysis is usually restricted to one or a few mutatio" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/22/4/269/348206" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2015-06-29" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Kukita, Yoji" /><meta name="citation_author_institution" content="Department of Molecular and Medical Genetics, Research Institute, Osaka Medical Center for Cancer and Cardiovascular Diseases, Osaka 537-8511, Japan" /><meta name="citation_author" content="Matoba, Ryo" /><meta name="citation_author_institution" content="DNA Chip Research Inc., Yokohama 230-0045, Japan" /><meta name="citation_author" content="Uchida, Junji" /><meta name="citation_author_institution" content="Department of Thoracic Oncology, Osaka Medical Center for Cancer and Cardiovascular Diseases, Osaka 537-8511, Japan" /><meta name="citation_author" content="Hamakawa, Takuya" /><meta name="citation_author_institution" content="Department of Molecular and Medical Genetics, Research Institute, Osaka Medical Center for Cancer and Cardiovascular Diseases, Osaka 537-8511, Japan" /><meta name="citation_author_institution" content="Department of Gastroenterological Surgery, Osaka University Graduate School of Medicine, Osaka 565-0871, Japan" /><meta name="citation_author" content="Doki, Yuichiro" /><meta name="citation_author_institution" content="Department of Gastroenterological Surgery, Osaka University Graduate School of Medicine, Osaka 565-0871, Japan" /><meta name="citation_author" content="Imamura, Fumio" /><meta name="citation_author_institution" content="Department of Thoracic Oncology, Osaka Medical Center for Cancer and Cardiovascular Diseases, Osaka 537-8511, Japan" /><meta name="citation_author" content="Kato, Kikuya" /><meta name="citation_author_institution" content="Department of Molecular and Medical Genetics, Research Institute, Osaka Medical Center for Cancer and Cardiovascular Diseases, Osaka 537-8511, Japan" /><meta name="citation_title" content="High-fidelity target sequencing of individual molecules identified using barcode sequences: de novo detection and absolute quantitation of mutations in plasma cell-free DNA from cancer patients" /><meta name="citation_firstpage" content="269" /><meta name="citation_lastpage" content="277" /><meta name="citation_doi" content="10.1093/dnares/dsv010" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="22" /><meta name="citation_issue" content="4" /><meta name="citation_publication_date" content="2015/08/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Circulating mutant DNA to assess tumor dynamics;  citation_journal_title=Nat. Med.;  citation_year=2008;  citation_volume=14;  citation_pages=985-90; " /><meta name="citation_reference" content="citation_title=The molecular evolution of acquired resistance to targeted EGFR blockade in colorectal cancers;  citation_journal_title=Nature;  citation_year=2012;  citation_volume=486;  citation_pages=537-40; " /><meta name="citation_reference" content="citation_title=Emergence of KRAS mutations and acquired resistance to anti-EGFR therapy in colorectal cancer;  citation_journal_title=Nature;  citation_year=2012;  citation_volume=486;  citation_pages=532-6; " /><meta name="citation_reference" content="citation_title=Quantitative detection of EGFR mutations in circulating tumor DNA derived from lung adenocarcinomas;  citation_journal_title=Clin. Cancer. Res.;  citation_year=2011;  citation_volume=17;  citation_pages=7808-15; " /><meta name="citation_reference" content="citation_title=The causes and consequences of genetic heterogeneity in cancer evolution;  citation_journal_title=Nature;  citation_year=2013;  citation_volume=501;  citation_pages=338-45; " /><meta name="citation_reference" content="citation_title=An ultrasensitive method for quantitating circulating tumor DNA with broad patient coverage;  citation_journal_title=Nat. Med.;  citation_year=2014;  citation_volume=20;  citation_pages=548-54; " /><meta name="citation_reference" content="citation_title=Rapid clearance of fetal DNA from maternal plasma;  citation_journal_title=Am. J. Hum. Genet.;  citation_year=1999;  citation_volume=64;  citation_pages=218-24; " /><meta name="citation_reference" content="citation_title=Digital PCR;  citation_journal_title=Proc. Natl Acad. Sci. USA;  citation_year=1999;  citation_volume=96;  citation_pages=9236-41; " /><meta name="citation_reference" content="citation_title=Detection of circulating tumor DNA in early- and late-stage human malignancies;  citation_journal_title=Sci. Transl. Med.;  citation_year=2014;  citation_volume=6;  citation_pages=224ra224" /><meta name="citation_reference" content="citation_title=Analysis of circulating tumor DNA to monitor metastatic breast cancer;  citation_journal_title=N. Engl. J. Med.;  citation_year=2013;  citation_volume=368;  citation_pages=1199-209; " /><meta name="citation_reference" content="citation_title=A method for counting PCR template molecules with application to next-generation sequencing;  citation_journal_title=Nucleic Acids Res.;  citation_year=2011;  citation_volume=39;  citation_pages=e81" /><meta name="citation_reference" content="citation_title=Detection and quantification of rare mutations with massively parallel sequencing;  citation_journal_title=Proc. Natl Acad. Sci. USA;  citation_year=2011;  citation_volume=108;  citation_pages=9530-5; " /><meta name="citation_reference" content="citation_title=Error-correcting barcoded primers for pyrosequencing hundreds of samples in multiplex;  citation_journal_title=Nat. Methods;  citation_year=2008;  citation_volume=5;  citation_pages=235-7; " /><meta name="citation_reference" content="citation_title=Digital RNA sequencing minimizes sequence-dependent bias and amplification noise with optimized single-molecule barcodes;  citation_journal_title=Proc. Natl Acad. Sci. USA;  citation_year=2012;  citation_volume=109;  citation_pages=1347-52; " /><meta name="citation_reference" content="citation_title=Comparative analysis of mutations in the p53 and K-ras genes in pancreatic cancer;  citation_journal_title=Int. J. Cancer;  citation_year=1994;  citation_volume=58;  citation_pages=185-91; " /><meta name="citation_reference" content="citation_title=Quantitative identification of mutant alleles derived from lung cancer in plasma cell-free DNA via anomaly detection using deep sequencing data;  citation_journal_title=PLoS ONE;  citation_year=2013;  citation_volume=8;  citation_pages=e81468" /><meta name="citation_reference" content="citation_title=Fast and accurate long-read alignment with Burrows-Wheeler transform;  citation_journal_title=Bioinformatics;  citation_year=2010;  citation_volume=26;  citation_pages=589-95; " /><meta name="citation_reference" content="citation_title=The Sequence Alignment/Map format and SAMtools;  citation_journal_title=Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=2078-9; " /><meta name="citation_reference" content="citation_title=VarScan 2: somatic mutation and copy number alteration discovery in cancer by exome sequencing;  citation_journal_title=Genome Res.;  citation_year=2012;  citation_volume=22;  citation_pages=568-76; " /><meta name="citation_reference" content="citation_title=Detection of ultra-rare mutations by next-generation sequencing;  citation_journal_title=Proc. Natl Acad. Sci. USA;  citation_year=2012;  citation_volume=109;  citation_pages=14508-13; " /><meta name="citation_reference" content="citation_title=A novel technique for measuring variations in DNA copy-number: competitive genomic polymerase chain reaction;  citation_journal_title=BMC Genomics;  citation_year=2007;  citation_volume=8;  citation_pages=206" /><meta name="citation_reference" content="citation_title=Adaptor-tagged competitive PCR: a novel method for measuring relative gene expression;  citation_journal_title=Nucleic Acids Res.;  citation_year=1997;  citation_volume=25;  citation_pages=4694-6; " /><meta name="citation_reference" content="citation_title=Cancer gene expression database (CGED): a database for gene expression profiling with accompanying clinical information of human cancer tissues;  citation_journal_title=Nucleic Acids Res.;  citation_year=2005;  citation_volume=33;  citation_pages=D533-6; " /><meta name="citation_reference" content="citation_title=Description of the entire mRNA population by a 3′ end cDNA fragment generated by class IIS restriction enzymes;  citation_journal_title=Nucleic Acids Res.;  citation_year=1995;  citation_volume=23;  citation_pages=3685-90; " /><meta name="citation_reference" content="citation_title=Single molecule molecular inversion probes for targeted, high-accuracy detection of low-frequency variation;  citation_journal_title=Genome Res.;  citation_year=2013;  citation_volume=23;  citation_pages=843-54; " /><meta name="citation_reference" content="citation_title=A tale of three next generation sequencing platforms: comparison of Ion Torrent, Pacific Biosciences and Illumina MiSeq sequencers;  citation_journal_title=BMC Genomics;  citation_year=2012;  citation_volume=13;  citation_pages=341" /><meta name="citation_reference" content="citation_title=Updating benchtop sequencing performance comparison;  citation_journal_title=Nat. Biotechnol.;  citation_year=2013;  citation_volume=31;  citation_pages=294-6; " /><meta name="citation_reference" content="citation_title=Noninvasive diagnosis of actionable mutations by deep sequencing of circulating free DNA in lung cancer from never-smokers: a proof-of-concept study from BioCAST/IFCT-1002;  citation_journal_title=Clin. Cancer Res.;  citation_year=2014;  citation_volume=20;  citation_pages=4613-24; " /><meta name="citation_reference" content="citation_title=Genetic alteration of the beta-catenin gene (CTNNB1) in human lung cancer and malignant mesothelioma and identification of a new 3p21.3 homozygous deletion;  citation_journal_title=Oncogene;  citation_year=2001;  citation_volume=20;  citation_pages=4249-57; " /><meta name="citation_reference" content="citation_title=Using multiplexed assays of oncogenic drivers in lung cancers to select targeted drugs;  citation_journal_title=JAMA;  citation_year=2014;  citation_volume=311;  citation_pages=1998-2006; " /><meta name="citation_reference" content="citation_title=Impact of next-generation sequencing error on analysis of barcoded plasmid libraries of known complexity and sequence;  citation_journal_title=Nucleic Acids Res.;  citation_year=2014;  citation_volume=42;  citation_pages=e129" /><meta name="citation_reference" content="citation_title=Noninvasive identification and monitoring of cancer mutations by targeted deep sequencing of plasma DNA;  citation_journal_title=Sci. Transl. Med.;  citation_year=2012;  citation_volume=4;  citation_pages=136ra168" /><meta name="citation_reference" content="citation_title=Ultrasensitive measurement of hotspot mutations in tumor DNA in blood using error-suppressed multiplexed deep sequencing;  citation_journal_title=Cancer Res.;  citation_year=2012;  citation_volume=72;  citation_pages=3492-8; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/22/4/269/997814/dsv010.pdf" /><meta name="description" content="Abstract.  Circulating tumour DNA (ctDNA) is an emerging field of cancer research. However, current ctDNA analysis is usually restricted to one or a few mutatio" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/22/4/269/348206" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/22/4/269/348206" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/22/4/269/348206");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsv010';
    </script>
<title>
	High-fidelity target sequencing of individual molecules identified using barcode sequences: de novo detection and absolute quantitation of mutations in plasma cell-free DNA from cancer patients | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./348206" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BYBaPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLdWtpdGEsIFlvamkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIE1vbGVjdWxhciBhbmQgTWVkaWNhbCBHZW5ldGljcywgUmVzZWFyY2ggSW5zdGl0dXRlLCBPc2FrYSBNZWRpY2FsIENlbnRlciBmb3IgQ2FuY2VyIGFuZCBDYXJkaW92YXNjdWxhciBEaXNlYXNlcywgT3Nha2EgNTM3LTg1MTEsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTWF0b2JhLCBSeW8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJETkEgQ2hpcCBSZXNlYXJjaCBJbmMuLCBZb2tvaGFtYSAyMzAtMDA0NSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJVY2hpZGEsIEp1bmppIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBUaG9yYWNpYyBPbmNvbG9neSwgT3Nha2EgTWVkaWNhbCBDZW50ZXIgZm9yIENhbmNlciBhbmQgQ2FyZGlvdmFzY3VsYXIgRGlzZWFzZXMsIE9zYWthIDUzNy04NTExLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkhhbWFrYXdhLCBUYWt1eWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIE1vbGVjdWxhciBhbmQgTWVkaWNhbCBHZW5ldGljcywgUmVzZWFyY2ggSW5zdGl0dXRlLCBPc2FrYSBNZWRpY2FsIENlbnRlciBmb3IgQ2FuY2VyIGFuZCBDYXJkaW92YXNjdWxhciBEaXNlYXNlcywgT3Nha2EgNTM3LTg1MTEsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBHYXN0cm9lbnRlcm9sb2dpY2FsIFN1cmdlcnksIE9zYWthIFVuaXZlcnNpdHkgR3JhZHVhdGUgU2Nob29sIG9mIE1lZGljaW5lLCBPc2FrYSA1NjUtMDg3MSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJEb2tpLCBZdWljaGlybyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgR2FzdHJvZW50ZXJvbG9naWNhbCBTdXJnZXJ5LCBPc2FrYSBVbml2ZXJzaXR5IEdyYWR1YXRlIFNjaG9vbCBvZiBNZWRpY2luZSwgT3Nha2EgNTY1LTA4NzEsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSW1hbXVyYSwgRnVtaW8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIFRob3JhY2ljIE9uY29sb2d5LCBPc2FrYSBNZWRpY2FsIENlbnRlciBmb3IgQ2FuY2VyIGFuZCBDYXJkaW92YXNjdWxhciBEaXNlYXNlcywgT3Nha2EgNTM3LTg1MTEsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS2F0bywgS2lrdXlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBNb2xlY3VsYXIgYW5kIE1lZGljYWwgR2VuZXRpY3MsIFJlc2VhcmNoIEluc3RpdHV0ZSwgT3Nha2EgTWVkaWNhbCBDZW50ZXIgZm9yIENhbmNlciBhbmQgQ2FyZGlvdmFzY3VsYXIgRGlzZWFzZXMsIE9zYWthIDUzNy04NTExLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iSGlnaC1maWRlbGl0eSB0YXJnZXQgc2VxdWVuY2luZyBvZiBpbmRpdmlkdWFsIG1vbGVjdWxlcyBpZGVudGlmaWVkIHVzaW5nIGJhcmNvZGUgc2VxdWVuY2VzOiBkZSBub3ZvIGRldGVjdGlvbiBhbmQgYWJzb2x1dGUgcXVhbnRpdGF0aW9uIG9mIG11dGF0aW9ucyBpbiBwbGFzbWEgY2VsbC1mcmVlIEROQSBmcm9tIGNhbmNlciBwYXRpZW50cyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjI2OSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iMjc3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHN2MDEwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjIyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSI0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTUvMDgvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaXJjdWxhdGluZyBtdXRhbnQgRE5BIHRvIGFzc2VzcyB0dW1vciBkeW5hbWljczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBNZWQuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9OTg1LTkwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgbW9sZWN1bGFyIGV2b2x1dGlvbiBvZiBhY3F1aXJlZCByZXNpc3RhbmNlIHRvIHRhcmdldGVkIEVHRlIgYmxvY2thZGUgaW4gY29sb3JlY3RhbCBjYW5jZXJzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NDg2OyAgY2l0YXRpb25fcGFnZXM9NTM3LTQwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FbWVyZ2VuY2Ugb2YgS1JBUyBtdXRhdGlvbnMgYW5kIGFjcXVpcmVkIHJlc2lzdGFuY2UgdG8gYW50aS1FR0ZSIHRoZXJhcHkgaW4gY29sb3JlY3RhbCBjYW5jZXI7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT00ODY7ICBjaXRhdGlvbl9wYWdlcz01MzItNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UXVhbnRpdGF0aXZlIGRldGVjdGlvbiBvZiBFR0ZSIG11dGF0aW9ucyBpbiBjaXJjdWxhdGluZyB0dW1vciBETkEgZGVyaXZlZCBmcm9tIGx1bmcgYWRlbm9jYXJjaW5vbWFzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DbGluLiBDYW5jZXIuIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTc7ICBjaXRhdGlvbl9wYWdlcz03ODA4LTE1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgY2F1c2VzIGFuZCBjb25zZXF1ZW5jZXMgb2YgZ2VuZXRpYyBoZXRlcm9nZW5laXR5IGluIGNhbmNlciBldm9sdXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT01MDE7ICBjaXRhdGlvbl9wYWdlcz0zMzgtNDU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuIHVsdHJhc2Vuc2l0aXZlIG1ldGhvZCBmb3IgcXVhbnRpdGF0aW5nIGNpcmN1bGF0aW5nIHR1bW9yIEROQSB3aXRoIGJyb2FkIHBhdGllbnQgY292ZXJhZ2U7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gTWVkLjsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0yMDsgIGNpdGF0aW9uX3BhZ2VzPTU0OC01NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmFwaWQgY2xlYXJhbmNlIG9mIGZldGFsIEROQSBmcm9tIG1hdGVybmFsIHBsYXNtYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QW0uIEouIEh1bS4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTY0OyAgY2l0YXRpb25fcGFnZXM9MjE4LTI0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaWdpdGFsIFBDUjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT05NjsgIGNpdGF0aW9uX3BhZ2VzPTkyMzYtNDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldGVjdGlvbiBvZiBjaXJjdWxhdGluZyB0dW1vciBETkEgaW4gZWFybHktIGFuZCBsYXRlLXN0YWdlIGh1bWFuIG1hbGlnbmFuY2llczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpLiBUcmFuc2wuIE1lZC47ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTIyNHJhMjI0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW5hbHlzaXMgb2YgY2lyY3VsYXRpbmcgdHVtb3IgRE5BIHRvIG1vbml0b3IgbWV0YXN0YXRpYyBicmVhc3QgY2FuY2VyOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OLiBFbmdsLiBKLiBNZWQuOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTM2ODsgIGNpdGF0aW9uX3BhZ2VzPTExOTktMjA5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIG1ldGhvZCBmb3IgY291bnRpbmcgUENSIHRlbXBsYXRlIG1vbGVjdWxlcyB3aXRoIGFwcGxpY2F0aW9uIHRvIG5leHQtZ2VuZXJhdGlvbiBzZXF1ZW5jaW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9Mzk7ICBjaXRhdGlvbl9wYWdlcz1lODEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZXRlY3Rpb24gYW5kIHF1YW50aWZpY2F0aW9uIG9mIHJhcmUgbXV0YXRpb25zIHdpdGggbWFzc2l2ZWx5IHBhcmFsbGVsIHNlcXVlbmNpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTA4OyAgY2l0YXRpb25fcGFnZXM9OTUzMC01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FcnJvci1jb3JyZWN0aW5nIGJhcmNvZGVkIHByaW1lcnMgZm9yIHB5cm9zZXF1ZW5jaW5nIGh1bmRyZWRzIG9mIHNhbXBsZXMgaW4gbXVsdGlwbGV4OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIE1ldGhvZHM7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPTIzNS03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaWdpdGFsIFJOQSBzZXF1ZW5jaW5nIG1pbmltaXplcyBzZXF1ZW5jZS1kZXBlbmRlbnQgYmlhcyBhbmQgYW1wbGlmaWNhdGlvbiBub2lzZSB3aXRoIG9wdGltaXplZCBzaW5nbGUtbW9sZWN1bGUgYmFyY29kZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTA5OyAgY2l0YXRpb25fcGFnZXM9MTM0Ny01MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyYXRpdmUgYW5hbHlzaXMgb2YgbXV0YXRpb25zIGluIHRoZSBwNTMgYW5kIEstcmFzIGdlbmVzIGluIHBhbmNyZWF0aWMgY2FuY2VyOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1JbnQuIEouIENhbmNlcjsgIGNpdGF0aW9uX3llYXI9MTk5NDsgIGNpdGF0aW9uX3ZvbHVtZT01ODsgIGNpdGF0aW9uX3BhZ2VzPTE4NS05MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UXVhbnRpdGF0aXZlIGlkZW50aWZpY2F0aW9uIG9mIG11dGFudCBhbGxlbGVzIGRlcml2ZWQgZnJvbSBsdW5nIGNhbmNlciBpbiBwbGFzbWEgY2VsbC1mcmVlIEROQSB2aWEgYW5vbWFseSBkZXRlY3Rpb24gdXNpbmcgZGVlcCBzZXF1ZW5jaW5nIGRhdGE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgT05FOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz1lODE0NjgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GYXN0IGFuZCBhY2N1cmF0ZSBsb25nLXJlYWQgYWxpZ25tZW50IHdpdGggQnVycm93cy1XaGVlbGVyIHRyYW5zZm9ybTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MjY7ICBjaXRhdGlvbl9wYWdlcz01ODktOTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBTZXF1ZW5jZSBBbGlnbm1lbnQvTWFwIGZvcm1hdCBhbmQgU0FNdG9vbHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9MjA3OC05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1WYXJTY2FuIDI6IHNvbWF0aWMgbXV0YXRpb24gYW5kIGNvcHkgbnVtYmVyIGFsdGVyYXRpb24gZGlzY292ZXJ5IGluIGNhbmNlciBieSBleG9tZSBzZXF1ZW5jaW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0yMjsgIGNpdGF0aW9uX3BhZ2VzPTU2OC03NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGV0ZWN0aW9uIG9mIHVsdHJhLXJhcmUgbXV0YXRpb25zIGJ5IG5leHQtZ2VuZXJhdGlvbiBzZXF1ZW5jaW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTEwOTsgIGNpdGF0aW9uX3BhZ2VzPTE0NTA4LTEzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIG5vdmVsIHRlY2huaXF1ZSBmb3IgbWVhc3VyaW5nIHZhcmlhdGlvbnMgaW4gRE5BIGNvcHktbnVtYmVyOiBjb21wZXRpdGl2ZSBnZW5vbWljIHBvbHltZXJhc2UgY2hhaW4gcmVhY3Rpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9MjA2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWRhcHRvci10YWdnZWQgY29tcGV0aXRpdmUgUENSOiBhIG5vdmVsIG1ldGhvZCBmb3IgbWVhc3VyaW5nIHJlbGF0aXZlIGdlbmUgZXhwcmVzc2lvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9NDY5NC02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DYW5jZXIgZ2VuZSBleHByZXNzaW9uIGRhdGFiYXNlIChDR0VEKTogYSBkYXRhYmFzZSBmb3IgZ2VuZSBleHByZXNzaW9uIHByb2ZpbGluZyB3aXRoIGFjY29tcGFueWluZyBjbGluaWNhbCBpbmZvcm1hdGlvbiBvZiBodW1hbiBjYW5jZXIgdGlzc3VlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTMzOyAgY2l0YXRpb25fcGFnZXM9RDUzMy02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZXNjcmlwdGlvbiBvZiB0aGUgZW50aXJlIG1STkEgcG9wdWxhdGlvbiBieSBhIDPigLIgZW5kIGNETkEgZnJhZ21lbnQgZ2VuZXJhdGVkIGJ5IGNsYXNzIElJUyByZXN0cmljdGlvbiBlbnp5bWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5OTU7ICBjaXRhdGlvbl92b2x1bWU9MjM7ICBjaXRhdGlvbl9wYWdlcz0zNjg1LTkwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TaW5nbGUgbW9sZWN1bGUgbW9sZWN1bGFyIGludmVyc2lvbiBwcm9iZXMgZm9yIHRhcmdldGVkLCBoaWdoLWFjY3VyYWN5IGRldGVjdGlvbiBvZiBsb3ctZnJlcXVlbmN5IHZhcmlhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MjM7ICBjaXRhdGlvbl9wYWdlcz04NDMtNTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgdGFsZSBvZiB0aHJlZSBuZXh0IGdlbmVyYXRpb24gc2VxdWVuY2luZyBwbGF0Zm9ybXM6IGNvbXBhcmlzb24gb2YgSW9uIFRvcnJlbnQsIFBhY2lmaWMgQmlvc2NpZW5jZXMgYW5kIElsbHVtaW5hIE1pU2VxIHNlcXVlbmNlcnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTM0MSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVVwZGF0aW5nIGJlbmNodG9wIHNlcXVlbmNpbmcgcGVyZm9ybWFuY2UgY29tcGFyaXNvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBCaW90ZWNobm9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0zMTsgIGNpdGF0aW9uX3BhZ2VzPTI5NC02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ob25pbnZhc2l2ZSBkaWFnbm9zaXMgb2YgYWN0aW9uYWJsZSBtdXRhdGlvbnMgYnkgZGVlcCBzZXF1ZW5jaW5nIG9mIGNpcmN1bGF0aW5nIGZyZWUgRE5BIGluIGx1bmcgY2FuY2VyIGZyb20gbmV2ZXItc21va2VyczogYSBwcm9vZi1vZi1jb25jZXB0IHN0dWR5IGZyb20gQmlvQ0FTVC9JRkNULTEwMDI7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNsaW4uIENhbmNlciBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9NDYxMy0yNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpYyBhbHRlcmF0aW9uIG9mIHRoZSBiZXRhLWNhdGVuaW4gZ2VuZSAoQ1ROTkIxKSBpbiBodW1hbiBsdW5nIGNhbmNlciBhbmQgbWFsaWduYW50IG1lc290aGVsaW9tYSBhbmQgaWRlbnRpZmljYXRpb24gb2YgYSBuZXcgM3AyMS4zIGhvbW96eWdvdXMgZGVsZXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU9uY29nZW5lOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9NDI0OS01NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VXNpbmcgbXVsdGlwbGV4ZWQgYXNzYXlzIG9mIG9uY29nZW5pYyBkcml2ZXJzIGluIGx1bmcgY2FuY2VycyB0byBzZWxlY3QgdGFyZ2V0ZWQgZHJ1Z3M7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUpBTUE7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MzExOyAgY2l0YXRpb25fcGFnZXM9MTk5OC0yMDA2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbXBhY3Qgb2YgbmV4dC1nZW5lcmF0aW9uIHNlcXVlbmNpbmcgZXJyb3Igb24gYW5hbHlzaXMgb2YgYmFyY29kZWQgcGxhc21pZCBsaWJyYXJpZXMgb2Yga25vd24gY29tcGxleGl0eSBhbmQgc2VxdWVuY2U7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT00MjsgIGNpdGF0aW9uX3BhZ2VzPWUxMjkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ob25pbnZhc2l2ZSBpZGVudGlmaWNhdGlvbiBhbmQgbW9uaXRvcmluZyBvZiBjYW5jZXIgbXV0YXRpb25zIGJ5IHRhcmdldGVkIGRlZXAgc2VxdWVuY2luZyBvZiBwbGFzbWEgRE5BOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2kuIFRyYW5zbC4gTWVkLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9MTM2cmExNjgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1VbHRyYXNlbnNpdGl2ZSBtZWFzdXJlbWVudCBvZiBob3RzcG90IG11dGF0aW9ucyBpbiB0dW1vciBETkEgaW4gYmxvb2QgdXNpbmcgZXJyb3Itc3VwcHJlc3NlZCBtdWx0aXBsZXhlZCBkZWVwIHNlcXVlbmNpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNhbmNlciBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTcyOyAgY2l0YXRpb25fcGFnZXM9MzQ5Mi04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMjIvNC8yNjkvOTk3ODE0L2RzdjAxMC5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgQ2lyY3VsYXRpbmcgdHVtb3VyIEROQSAoY3RETkEpIGlzIGFuIGVtZXJnaW5nIGZpZWxkIG9mIGNhbmNlciByZXNlYXJjaC4gSG93ZXZlciwgY3VycmVudCBjdEROQSBhbmFseXNpcyBpcyB1c3VhbGx5IHJlc3RyaWN0ZWQgdG8gb25lIG9yIGEgZmV3IG11dGF0aW8iIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVyLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTM0ODIwNiZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGSiz36oLkJZzipKn9cjek1pfrF6cg==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f4%2f269%2f348206" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f4%2f269%2f348206" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=13443" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/22/4"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/4/1/m_cover.gif?Expires=2147483647&amp;Signature=Ufin3LLH~Iqb2kw-hetKqzdx2YgWiSwuSGu7FcRPyZ0YMXPxK67~EvZK~D~FaBMUMtkfx2C3nicLRXrG7LQSzXkxs1Qyu1rF6Hbn-vB-awwpaXOgx90CafYjmsJlX0TeG5UBY2DnHbv1s5YegTLh88o5c9kHiz4IICTc2SgUfnIM~hO2aQPqVNMfxHD9dsgu8Ly4qu3h4vkVcBZFQVgjv-cFeqoqOFVvCZqW6Vq3z-Hi91~oLc58RRI-PjbrcLYKmszwtDPbL2sZKRQky2J2HWAa2PZzNl-CMH92uFmvRLgkUeyANQZ-s6eHxAaArS5e4XiVxAKXHDRMAVOPl6~asA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/22/4">
                        <div class="volume trailing-comma">Volume 22</div>
                        <div class="issue">Issue 4</div>

            </a>

        </div>
        <div class="ii-pub-date">
August 2015        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="4941266">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4941266">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4941268">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4941268">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4941274">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4941274">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4941290">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4941290">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4941320">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4941320">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4941327">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4941327">5. Data availability</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4941329">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4941329">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4941331">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4941331">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4941333">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4941333">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4941335">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4941335">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/22/4/259/348246">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/22/4/279/348084">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">High-fidelity target sequencing of individual molecules identified using barcode sequences: <em>de novo</em> detection and absolute quantitation of mutations in plasma cell-free DNA from cancer patients
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yoji Kukita</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Molecular and Medical Genetics</div>, <div class="insititution">Research Institute, Osaka Medical Center for Cancer and Cardiovascular Diseases</div>, <div class="addr-line">Osaka 537-8511</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yoji+Kukita">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kukita Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kukita Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yoji Kukita</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ryo Matoba</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="insititution">DNA Chip Research Inc.</div>, <div class="addr-line">Yokohama 230-0045</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ryo+Matoba">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Matoba R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Matoba R&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Ryo Matoba</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Junji Uchida</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Department of Thoracic Oncology</div>, <div class="insititution">Osaka Medical Center for Cancer and Cardiovascular Diseases</div>, <div class="addr-line">Osaka 537-8511</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Junji+Uchida">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Uchida J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Uchida J&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Junji Uchida</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Takuya Hamakawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Molecular and Medical Genetics</div>, <div class="insititution">Research Institute, Osaka Medical Center for Cancer and Cardiovascular Diseases</div>, <div class="addr-line">Osaka 537-8511</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">4</div><div class="addr-line">Department of Gastroenterological Surgery</div>, <div class="insititution">Osaka University Graduate School of Medicine</div>, <div class="addr-line">Osaka 565-0871</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Takuya+Hamakawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hamakawa T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hamakawa T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Takuya Hamakawa</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yuichiro Doki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">Department of Gastroenterological Surgery</div>, <div class="insititution">Osaka University Graduate School of Medicine</div>, <div class="addr-line">Osaka 565-0871</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yuichiro+Doki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Doki Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Doki Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yuichiro Doki</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Fumio Imamura</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Department of Thoracic Oncology</div>, <div class="insititution">Osaka Medical Center for Cancer and Cardiovascular Diseases</div>, <div class="addr-line">Osaka 537-8511</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Fumio+Imamura">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Imamura F">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Imamura F&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Fumio Imamura</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kikuya Kato</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Molecular and Medical Genetics</div>, <div class="insititution">Research Institute, Osaka Medical Center for Cancer and Cardiovascular Diseases</div>, <div class="addr-line">Osaka 537-8511</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81 6-6972-1181. E-mail: <a href="mailto:katou-ki@mc.pref.osaka.jp" target="_blank">katou-ki@mc.pref.osaka.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kikuya+Kato">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kato K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kato K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Kikuya Kato</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="footnote-content">Edited by Dr Osamu Ohara</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 22, Issue 4, 1 August 2015, Pages 269–277, <a href='https://doi.org/10.1093/dnares/dsv010'>https://doi.org/10.1093/dnares/dsv010</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">29 June 2015</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">31 March 2015</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">26 May 2015</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="348206" href="/dnaresearch/article-pdf/22/4/269/997814/dsv010.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Yoji Kukita, Ryo Matoba, Junji Uchida, Takuya Hamakawa, Yuichiro Doki, Fumio Imamura, Kikuya Kato;  High-fidelity target sequencing of individual molecules identified using barcode sequences: <em>de novo</em> detection and absolute quantitation of mutations in plasma cell-free DNA from cancer patients, <em>DNA Research</em>, Volume 22, Issue 4, 1 August 2015, Pages 269–277, <a href='https://doi.org/10.1093/dnares/dsv010'>https://doi.org/10.1093/dnares/dsv010</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/348206?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/348206?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/348206?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/348206?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/348206?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/348206?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/348206?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=High-fidelity%20target%20sequencing%20of%20individual%20molecules%20identified%20using%20barcode%20sequences%3A%20%26lt%3Bem%26gt%3Bde%20novo%26lt%3B%2Fem%26gt%3B%20detection%20and%20absolute%20quantitation%20of%20mutations%20in%20plasma%20cell-free%20DNA%20from%20cancer%20patients&amp;publicationDate=2015-06-29&amp;volumeNum=22&amp;issueNum=4&amp;author=Kukita%2C%20Yoji%3B%20Matoba%2C%20Ryo&amp;startPage=269&amp;endPage=277&amp;contentId=10.1093%2Fdnares%2Fdsv010&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f4%2f269%2f348206" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=4941266 id="4941266" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Circulating tumour DNA (ctDNA) is an emerging field of cancer research. However, current ctDNA analysis is usually restricted to one or a few mutation sites due to technical limitations. In the case of massively parallel DNA sequencers, the number of false positives caused by a high read error rate is a major problem. In addition, the final sequence reads do not represent the original DNA population due to the global amplification step during the template preparation. We established a high-fidelity target sequencing system of individual molecules identified in plasma cell-free DNA using barcode sequences; this system consists of the following two steps. (i) A novel target sequencing method that adds barcode sequences by adaptor ligation. This method uses linear amplification to eliminate the errors introduced during the early cycles of polymerase chain reaction. (ii) The monitoring and removal of erroneous barcode tags. This process involves the identification of individual molecules that have been sequenced and for which the number of mutations have been absolute quantitated. Using plasma cell-free DNA from patients with gastric or lung cancer, we demonstrated that the system achieved near complete elimination of false positives and enabled <em>de novo</em> detection and absolute quantitation of mutations in plasma cell-free DNA.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;massively parallel DNA sequencer&quot;">massively parallel DNA sequencer</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;circulating tumour DNA&quot;">circulating tumour DNA</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;barcode sequences&quot;">barcode sequences</a></div>                            <h2 scrollto-destination=4941268 id="4941268" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Circulating tumour DNA (ctDNA), which is the cell-free DNA (cfDNA) released from dying cancer cells to the blood, is an emerging topic in cancer research. Several proof-of-concept studies demonstrated that ctDNA could be used as a biomarker to monitor tumour burdens<sup><a reveal-id="DSV010C1" data-open="DSV010C1" class="link link-ref link-reveal xref-bibr">1</a></sup> or acquired drug resistance.<sup><a reveal-id="DSV010C2  DSV010C4" data-open="DSV010C2  DSV010C4" class="link link-ref link-reveal xref-bibr">2–4</a></sup> From a biological viewpoint, ctDNA is regarded as a carrier that brings the genetic information of solid tumours to peripheral blood. The use of ctDNA is expected to facilitate the analysis of genetic tumour heterogeneity,<sup><a reveal-id="DSV010C5" data-open="DSV010C5" class="link link-ref link-reveal xref-bibr">5</a></sup> such as the evolution of cancer cells during the disease course, because of the difficulty in sampling recurrent or metastatic tissues. On average, cfDNA is fragmented to a size of 170 bp,<sup><a reveal-id="DSV010C6" data-open="DSV010C6" class="link link-ref link-reveal xref-bibr">6</a></sup> and its half-life is estimated to be 16.5 min.<sup><a reveal-id="DSV010C7" data-open="DSV010C7" class="link link-ref link-reveal xref-bibr">7</a></sup> One millilitre of blood contains the cfDNA from one to several thousand genomes. The rare mutations resulting from cancer cells must be detected in the vast amount of DNA from normal cells and quantitated.</p><p>Although various techniques have been used to detect ctDNA, digital polymerase chain reaction (PCR)<sup><a reveal-id="DSV010C8" data-open="DSV010C8" class="link link-ref link-reveal xref-bibr">8</a></sup> and related technologies, particularly massively parallel DNA sequencers, are becoming the method of choice. In the case of massively parallel sequencers, the high read error rate of the current technologies is a major problem. Sequencing multiple sites or genomic regions dramatically increases the number of false positives, which is one of the reasons why current ctDNA assays tend to restrict their target to one or a few mutation sites.<sup><a reveal-id="DSV010C6" data-open="DSV010C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DSV010C9" data-open="DSV010C9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="DSV010C10" data-open="DSV010C10" class="link link-ref link-reveal xref-bibr">10</a></sup> Another problem is the global amplification step during the template preparation for massively parallel sequencing. The final sequence reads do not represent the original DNA population, and the number of reads usually exceeds the number of target DNA molecules. Consequently, the quantitation of mutation alleles may be affected.</p><p>These problems can be solved using barcode sequences.<sup><a reveal-id="DSV010C11" data-open="DSV010C11" class="link link-ref link-reveal xref-bibr">11</a>,<a reveal-id="DSV010C12" data-open="DSV010C12" class="link link-ref link-reveal xref-bibr">12</a></sup> Labelling DNA fragments with barcode sequences, typically N<sub>10–15</sub>, provides discrimination between the reads from individual molecules, thus enabling grouping of the reads from each molecule. Constructing a consensus of reads leads to both high-fidelity DNA sequencing and the capability to count the number of sequenced molecules. The main problem is the read errors that are introduced into barcode sequences, which can affect the basic principle of labelling each molecule with a single unique barcode. This problem has been recognized, and small collections of barcode sequences have been designed to detect and exclude erroneous sequences.<sup><a reveal-id="DSV010C13" data-open="DSV010C13" class="link link-ref link-reveal xref-bibr">13</a>,<a reveal-id="DSV010C14" data-open="DSV010C14" class="link link-ref link-reveal xref-bibr">14</a></sup> However, this approach requires each barcode sequence to be individually manufactured. Therefore, it cannot accommodate large numbers of sequences.</p><p>As a first attempt, we focused on cancer-related genes that are frequently mutated, that is <em>TP53</em> and <em>KRAS</em>, and we developed a system for <em>de novo</em> detection and absolute quantitation of mutations in plasma cfDNA. This system consists of the following two steps:
We named the collection of final consensus reads as ‘non-overlapping integrated reads (NOIR)’. The performance of the sequencing system, named the NOIR sequencing system, was demonstrated using plasma cfDNA from patients with gastric or lung cancer.</p><ul class="roman-lower"><li><p>A novel target sequencing method that adds barcode sequences by adaptor ligation. This method uses linear amplification to eliminate the errors introduced during the early cycles of PCR.</p></li><li><p>The monitoring and removal of erroneous barcode tags. This process involves the identification of individual molecules that have been sequenced and for which the number of mutations has been absolutely quantitated.</p></li></ul>                            <h2 scrollto-destination=4941274 id="4941274" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=4941275 id="4941275" class="section-title" data-legacy-id=s2a>2.1. DNA samples</h3>
<p>We used Megapool Reference male DNA (Kreatech Biotechnology, Amsterdam, Netherlands), which is a pool of DNA from 100 normal Caucasian males, for technical development. The genomic DNA of leucocytes from normal individuals and the MIA PaCa-2 pancreatic carcinoma cell line, which has an R280W mutation in the <em>TP53</em> gene,<sup><a reveal-id="DSV010C15" data-open="DSV010C15" class="link link-ref link-reveal xref-bibr">15</a></sup> was extracted using a standard phenol/chloroform protocol. Patients with activating <em>EGFR</em> mutations in lung cancer tissues were recruited at Osaka Medical Center for Cancer and Cardiovascular Diseases.<sup><a reveal-id="DSV010C16" data-open="DSV010C16" class="link link-ref link-reveal xref-bibr">16</a></sup> A gastric cancer patient was recruited at Osaka University Hospital. Written informed consent was obtained from all patients recruited to this study. This study was approved by the ethics committee of Osaka Medical Center for Cancer and Cardiovascular Diseases and Osaka University Hospital.</p><p>Plasma was prepared via centrifugation of 4–5 ml of EDTA-treated blood at 800 <em>g</em> for 10 min at room temperature. The plasma was transferred to a fresh tube and re-centrifuged at 15,100 <em>g</em> for 10 min at room temperature. After centrifugation, the upper plasma was transferred to a fresh tube. The centrifuged liquid samples were frozen at −80°C until DNA extraction. DNA was extracted from 1.5 to 2.0 ml of a liquid sample using the QIAamp circulating nucleic acid kit (Qiagen, Hilden, Germany) according to the manufacturer's instructions. For several samples, the DNA concentration was determined using the Qubit dsDNA HS Assay Kit (Life Technologies, CA, USA).</p>                            <h3 scrollto-destination=4941278 id="4941278" class="section-title" data-legacy-id=s2b>2.2. Target regions, adaptors and region-specific primers</h3>
<p>We designed adaptors and primers to analyse the genomic regions that code for the DNA-binding domain of <em>TP53</em> and for the mutation hotspots of <em>KRAS</em> and <em>CTNNB1</em> (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S1–S3</a>).</p>                            <h3 scrollto-destination=4941280 id="4941280" class="section-title" data-legacy-id=s2c>2.3. Library construction with linear amplification of the barcoded strands</h3>
<p>Genomic DNA (5–40 ng) or cfDNA (from ∼1 ml of whole blood) was digested using multiple restriction enzymes [Set1: AlwNI and Alw26I; Set2: EarI and NcoI; SetKC: EarI and NmuCI (FastDigest enzymes, Thermo Scientific, MA, USA); <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>]. The ligation of adaptors with N<sub>12</sub> barcode sequence tags was performed using <em>Escherichia coli</em> DNA ligase (Takara Bio, Shiga, Japan). The ligation products were purified twice with a 1.2× volume of AMPureXP beads (Beckman Coulter, CA, USA). Linear amplification of the purified products was performed with a region-specific primer mixture (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S2 and S3</a>) and Q5 Hot Start High-Fidelity DNA Polymerase (NEB) by 10 thermal cycles. The purified linear amplification products were amplified with the PGM/Proton primers (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>) and Platinum Taq High Fidelity (Life Technologies). The amplification products were purified with AMPureXP beads or agarose gel electrophoresis with a MinElute Gel Extraction Kit (Qiagen). Further details and library construction for experiments with double strand labeling are provided in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Methods</a>.</p>                            <h3 scrollto-destination=4941282 id="4941282" class="section-title" data-legacy-id=s2d>2.4. Massively parallel sequencing</h3>
<p>For the Ion Torrent sequencing system, we prepared sequencing templates (emulsion PCR and bead-enrichment) from sequencing libraries using an Ion PI Template OT2 200 Kit v2 or v3 (Life Technologies) and an Ion OneTouch system (Ion OneTouch Instrument and Ion OneTouch ES, Life Technologies) according to the manufacturer's protocol. Prepared templates were sequenced using an Ion PI Sequencing 200 Kit v2 or v3 and the Proton sequencer (Life Technologies). Torrent Suite 4.0 or 4.2 (Life Technologies) was used to convert the raw signals into base calls and to extract the FASTQ files of the sequencing reads. Sequencing data from the Illumina system were generated using a MiSeq system (Illumina, CA, USA) according to the manufacturer's protocol, and the FASTQ files of single-end reads were extracted.</p>                            <h3 scrollto-destination=4941284 id="4941284" class="section-title" data-legacy-id=s2e>2.5. Data analysis</h3>
<p>Reads in FASTQ format were divided using 5-bp indices for individual assignments. Sequences between the 5-bp indexes and spacer sequences were obtained as barcode tags. When the total length of the spacer and the following sequence was &gt;70 bases, the reads were aligned to target sequences (spacer + target region) with bwa (version 0.6.2) using the bwasw mode for aligning long reads<sup><a reveal-id="DSV010C17" data-open="DSV010C17" class="link link-ref link-reveal xref-bibr">17</a></sup> and the parameter setting ‘-b5 -q2 -r1 -z10’. Reads with long unmapped ends (&gt;10% of the total read length) were discarded.</p><p>Barcode tags from mapped reads were analysed in each target region. Although we designed 12-bp barcode tags, we obtained tags that were not 12-bp in length due to insertion/deletion errors that were detected during sequencing. We discarded tags that were &lt;9 bp. To recover the maximum number of reads, 11- and 13-bp tags that only differed from a 12-bp tag by the insertion or deletion of a single base were grouped with the corresponding 12-bp tag. For example, ‘TGCATGATACG’ and ‘TGCATGGATTACG’ were combined with the barcode group ‘TGCATGATTACG’.</p><p>Reads with the same barcode sequences were grouped together, and the barcode tags were assigned into 2-read bins according to the number of reads per tag. Then, the proportion of 12-bp tags in each bin was calculated, and the value (proportion) for each bin was averaged using the 11 bins around that bin. We defined the minimum bin as that with an average proportion of 90% or greater, and we set this value as the threshold for removing erroneous barcode tags. These processes were performed using an in-house Perl script.</p><p>After the removal of erroneous tags that had fewer reads than the threshold, the reads from tags with the same barcode were combined using samtools (version 0.1.18),<sup><a reveal-id="DSV010C18" data-open="DSV010C18" class="link link-ref link-reveal xref-bibr">18</a></sup> and consensus sequences were created using VarScan (v2.2.11).<sup><a reveal-id="DSV010C19" data-open="DSV010C19" class="link link-ref link-reveal xref-bibr">19</a></sup> If &gt;50 reads were obtained, the longest 50 reads were analysed. If &gt;80% of reads had an alternative base at a position, we assigned a variant. A set of consensus sequences was converted to a FASTQ file, and we assigned ‘57’ as a quality score for all bases. A FASTQ file was aligned to the sequences of target regions as described above, and the generated mapping data were processed to obtain the per base coverage (pileup files) using samtools. Subsequently, we summarized the base counts for each base position.</p><p>The sequence error rate was calculated by dividing the number of non-reference sequenced bases by the number of all sequenced bases in the target regions. When using barcode tags, consensus sequences of multiple reads from individual molecules were analysed. Reads before constructing consensus were used for deep sequencing. A common SNP site (dbSNP number: rs1800372) was not considered for calculation.</p>                            <h2 scrollto-destination=4941290 id="4941290" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=4941291 id="4941291" class="section-title" data-legacy-id=s3a>3.1. Target sequencing method that adds barcode sequences by adaptor ligation</h3>
<p>Barcode sequences can be attached to genomic DNA and transcriptomes by adaptor ligation.<sup><a reveal-id="DSV010C11" data-open="DSV010C11" class="link link-ref link-reveal xref-bibr">11</a></sup> For target or amplicon sequencing, barcode sequences are embedded in PCR primers.<sup><a reveal-id="DSV010C12" data-open="DSV010C12" class="link link-ref link-reveal xref-bibr">12</a></sup> Schmitt et al.<sup><a reveal-id="DSV010C20" data-open="DSV010C20" class="link link-ref link-reveal xref-bibr">20</a></sup> proposed a method to label both strands of a sequence with the same barcode sequence to detect the base changes that occur in only one strand. This method can only be used for adaptors, and it cannot be applied to barcode primers.</p><p>The addition of an adaptor to a restriction enzyme site and the subsequent PCR amplification with an adaptor-primer and a single gene-specific primer constitute a robust technique that our group has extensively applied to genomic DNA<sup><a reveal-id="DSV010C21" data-open="DSV010C21" class="link link-ref link-reveal xref-bibr">21</a></sup> and RNA.<sup><a reveal-id="DSV010C22" data-open="DSV010C22" class="link link-ref link-reveal xref-bibr">22</a>,<a reveal-id="DSV010C23" data-open="DSV010C23" class="link link-ref link-reveal xref-bibr">23</a></sup> We have used this method for target sequencing with barcodes. Restriction enzymes with five-, four- or three-base protruding ends can be used with this method and are listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a>. The listed collection of enzymes covers most of the human genome. We used <em>E. coli</em> DNA ligase, which enables the sequence-specific ligation of cohesive ends generated by type IIS restriction enzymes.<sup><a reveal-id="DSV010C24" data-open="DSV010C24" class="link link-ref link-reveal xref-bibr">24</a></sup> The adaptor sequence that was used includes 5 bases for indexing individuals and N<sub>12</sub> for indexing molecules (distinguishing up to 1.7 × 10<sup>7</sup> molecules). We used two versions of the method (Fig. <a reveal-id="DSV010F1" data-open="DSV010F1" class="link link-reveal link-table xref-fig">1</a>A). The first version involves linear amplification of the barcoded strand and subsequent PCR amplification. Linear amplification is expected to minimize errors in the first round of PCR. The second version employs replacement synthesis of the complementary strand to label both strands with the same barcode. For subsequent analysis, we primarily used the first version of the method with an Ion Proton sequencer. We chose the DNA-binding domain of <em>TP53</em>, which is covered by seven regions, as a target sequence (Fig. <a reveal-id="DSV010F1" data-open="DSV010F1" class="link link-reveal link-table xref-fig">1</a>B and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). </p>                        <a id="4941294" scrollto-destination="4941294"></a>
<div id="DSV010F1" data-id="DSV010F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv010/2/m_dsv01001.png?Expires=2147483647&Signature=FVKYtLKAHlRcpIPycFgJI6yCwastN864adQKsw0a5gIpLSosIqRGTBpE8EmlnGK7eLzcAwLLg85AmAW8l~2Nl5AwsYVV4JwT5JovgP9ir~Hc~-QxSvbDuFoTT7D5mMjWbQiyjGFd-Llm9VIXQvcZHho6s1GqK~wOlcLM71WP8XhqztupzQnIEQHxVREtUMosJAyOO7FpgzEZqMBN5lUUJ6tCM4i1CO0c6I-fR1JrIo~6Ms51hftqG6dvJZTWsnXOvuKja2SUV3RdUfgmcYBcWRp1GxyIi~xKSUxww-DOsQAMbY6W-DBoThOupt0s1qZUgXEErctKU9JIPmCwea0ASQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Construction of barcoded libraries. (A) Schematic of library construction. (B) Target regions in human TP53. The cDNA structure of the coding region of the human TP53 gene is shown via a bar graph of the distribution of mutations found in COSMIC v63 (http://cancer.sanger.ac.uk/). The dotted lines indicate the boundaries of exons in the DNA-binding domain. The black bars indicate target regions defined in this study." path-from-xml="dsv01001.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4941294/dsv01001.tiff" path-from-xml="dsv01001.tiff" target="_blank">View large</a><a section="4941294" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv010/2/dsv01001.png?Expires=2147483647&Signature=sxQuk78~pFgTfTKYX-ZlEg2KViyKcR4L6ZpX~WdF1k0ylMtbtwA-DNV09-25ijHrJqIorxaFnePqHKG8aZ9iHHkPXVVb8V53oUvcmBruxeYFGSEHIYIPLi~FDwWYNgai7i6HyGavVlrI9fAD3GWaqGXg-f4alCrLzkq23laYKFXSwyDoT9uHIFtwmz3sUo-BsUgUK53glC4fr98fMQ7QD9p-TlqritQ7cvMkcrghNdQbELgppvQfBSkxkWCvmosyaY7ySa32oAFxaloEoc0pZuQE7ESOh3dkx0YEg2M~eJTjy1tClak2qGJu6tS1yRvuUEbIz9MwYsEAUibv-0SFog__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4941294&ar=348206&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv01001.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Construction of barcoded libraries. (A) Schematic of library construction. (B) Target regions in human <em>TP53</em>. The cDNA structure of the coding region of the human <em>TP53</em> gene is shown via a bar graph of the distribution of mutations found in COSMIC v63 (<a class="link link-uri" href="http://cancer.sanger.ac.uk/">http://cancer.sanger.ac.uk/</a>). The dotted lines indicate the boundaries of exons in the DNA-binding domain. The black bars indicate target regions defined in this study.</p></div></div><div content-id="DSV010F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv010/2/m_dsv01001.png?Expires=2147483647&Signature=FVKYtLKAHlRcpIPycFgJI6yCwastN864adQKsw0a5gIpLSosIqRGTBpE8EmlnGK7eLzcAwLLg85AmAW8l~2Nl5AwsYVV4JwT5JovgP9ir~Hc~-QxSvbDuFoTT7D5mMjWbQiyjGFd-Llm9VIXQvcZHho6s1GqK~wOlcLM71WP8XhqztupzQnIEQHxVREtUMosJAyOO7FpgzEZqMBN5lUUJ6tCM4i1CO0c6I-fR1JrIo~6Ms51hftqG6dvJZTWsnXOvuKja2SUV3RdUfgmcYBcWRp1GxyIi~xKSUxww-DOsQAMbY6W-DBoThOupt0s1qZUgXEErctKU9JIPmCwea0ASQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Construction of barcoded libraries. (A) Schematic of library construction. (B) Target regions in human TP53. The cDNA structure of the coding region of the human TP53 gene is shown via a bar graph of the distribution of mutations found in COSMIC v63 (http://cancer.sanger.ac.uk/). The dotted lines indicate the boundaries of exons in the DNA-binding domain. The black bars indicate target regions defined in this study." path-from-xml="dsv01001.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4941294/dsv01001.tiff" path-from-xml="dsv01001.tiff" target="_blank">View large</a><a section="4941294" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv010/2/dsv01001.png?Expires=2147483647&Signature=sxQuk78~pFgTfTKYX-ZlEg2KViyKcR4L6ZpX~WdF1k0ylMtbtwA-DNV09-25ijHrJqIorxaFnePqHKG8aZ9iHHkPXVVb8V53oUvcmBruxeYFGSEHIYIPLi~FDwWYNgai7i6HyGavVlrI9fAD3GWaqGXg-f4alCrLzkq23laYKFXSwyDoT9uHIFtwmz3sUo-BsUgUK53glC4fr98fMQ7QD9p-TlqritQ7cvMkcrghNdQbELgppvQfBSkxkWCvmosyaY7ySa32oAFxaloEoc0pZuQE7ESOh3dkx0YEg2M~eJTjy1tClak2qGJu6tS1yRvuUEbIz9MwYsEAUibv-0SFog__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4941294&ar=348206&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv01001.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Construction of barcoded libraries. (A) Schematic of library construction. (B) Target regions in human <em>TP53</em>. The cDNA structure of the coding region of the human <em>TP53</em> gene is shown via a bar graph of the distribution of mutations found in COSMIC v63 (<a class="link link-uri" href="http://cancer.sanger.ac.uk/">http://cancer.sanger.ac.uk/</a>). The dotted lines indicate the boundaries of exons in the DNA-binding domain. The black bars indicate target regions defined in this study.</p></div></div>                            <h3 scrollto-destination=4941295 id="4941295" class="section-title" data-legacy-id=s3b>3.2. Monitoring and removing erroneous barcode tags in the Ion Torrent system</h3>
<p>We sequenced four of the seven regions using genomic DNA with masses ranging from 5 to 40 ng. An example of the relationship between the number of barcode sequence tags and the number of reads grouped by the same barcode sequence tag (reads per tag) is shown in Fig. <a reveal-id="DSV010F2" data-open="DSV010F2" class="link link-reveal link-table xref-fig">2</a>A. In this experiment, the input DNA corresponded to ∼10,000 copies of the genome, but the total number of tags was &gt;400,000. The majority of these tags had a small number of reads, including single reads. However, the corresponding number of reads occupied only a small fraction of the total reads obtained (Fig. <a reveal-id="DSV010F2" data-open="DSV010F2" class="link link-reveal link-table xref-fig">2</a>B). This phenomenon was observed in previous studies.<sup><a reveal-id="DSV010C11" data-open="DSV010C11" class="link link-ref link-reveal xref-bibr">11</a>,<a reveal-id="DSV010C12" data-open="DSV010C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DSV010C25" data-open="DSV010C25" class="link link-ref link-reveal xref-bibr">25</a></sup></p>                        <a id="4941297" scrollto-destination="4941297"></a>
<div id="DSV010F2" data-id="DSV010F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv010/2/m_dsv01002.png?Expires=2147483647&Signature=O~0W16TMPgo9Mf-l~xA9M3uNEP4UN4Rq-YYaFeMWqh-lQvrhIMWKUSy9rU46mveOh~cnnJjOX9mgH8gBotPmRPcot9psghNmigfYxVfvzVAwGVmEKIzehgjkUAELRoDj9gpGCwvUH7Kfv6-okVjkbfdFZzfWHAYIQ3PxHcoVKelXdDSdkDppV6t988D3o~l2Dsi8LpM-vRkvu3TXplD1MLdsXGJzZ7EjWA017r-NfESdIjeEMaFO8Pht55y1719C6zkSOII0a7d6y8ib~LgPJBEBoPxfSpQe~YguPzyUsxUZ15Hn4vz7VSZa3gy6tHcVDHmZHRsok6krXEGxoy4xjQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Monitoring the errors in the barcode sequence tags and absolute quantitation of target molecules. Results from Ion Proton sequencer data. The target region is TK102U, except in F. (A–D); results obtained using 40 ng of genomic DNA. (A) Distribution of reads per barcode tag. Vertical axis: number of different barcode tags. Horizontal axis: number of reads per tag, given as the common logarithm of the number. (B) Cumulative reads. (C) Proportions of barcode tags by size. (D) Mean proportion of 12-bp barcode tags after including the 11-bp and 13-bp tags that only differed from a 12-bp tag by the insertion or deletion of a single base. The mean proportion represents the average of the proportions of the nearest 11 bins. (E) Estimated number of target molecules after the removal of erroneous tags. Horizontal axis: number of reads used for estimation; the reads were randomly selected from the entire population of reads (‘5 ng’: 1,457,760; ‘10 ng’: 2,251,133; ‘20 ng’: 2,245,038; and ‘40 ng’: 2,395,763 reads). (F) Correlation between the number of molecules and the amount of input DNA after removal of the erroneous tags." path-from-xml="dsv01002.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4941297/dsv01002.tiff" path-from-xml="dsv01002.tiff" target="_blank">View large</a><a section="4941297" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv010/2/dsv01002.png?Expires=2147483647&Signature=v9g7ibUUDL4guA7ZQt6gZo1YLomrCBo2R64P4CVBqHc-mp7JCTFzajt9-Te8qJOUxqI4RJr2vi~n5Uxp0-bKsS7ZyFxQETrSkICwCD9dmREagYxr9eODJWiAapmCxsNo~zTC-c6NDfx43OUS2jcC2UpLw2qM03cswOKEEx-yhux1JMxYZKd782UEbKYzZ0x3sOBHoSrCM-MBYuSmZNf~H3M0OvEoOYTyRga3OqIcDhKbtl2Ur3RMriSbJaOE3Llur1gK0NBA6sGCgefnLPtLtST~OD08p~~Dwb4kYWGBmsqIRoJI9VhWH4G-sDtJ4UFFADVnG22~hSqfotHyCPzLCw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4941297&ar=348206&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv01002.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Monitoring the errors in the barcode sequence tags and absolute quantitation of target molecules. Results from Ion Proton sequencer data. The target region is TK102U, except in F. (A–D); results obtained using 40 ng of genomic DNA. (A) Distribution of reads per barcode tag. Vertical axis: number of different barcode tags. Horizontal axis: number of reads per tag, given as the common logarithm of the number. (B) Cumulative reads. (C) Proportions of barcode tags by size. (D) Mean proportion of 12-bp barcode tags after including the 11-bp and 13-bp tags that only differed from a 12-bp tag by the insertion or deletion of a single base. The mean proportion represents the average of the proportions of the nearest 11 bins. (E) Estimated number of target molecules after the removal of erroneous tags. Horizontal axis: number of reads used for estimation; the reads were randomly selected from the entire population of reads (‘5 ng’: 1,457,760; ‘10 ng’: 2,251,133; ‘20 ng’: 2,245,038; and ‘40 ng’: 2,395,763 reads). (F) Correlation between the number of molecules and the amount of input DNA after removal of the erroneous tags.</p></div></div><div content-id="DSV010F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv010/2/m_dsv01002.png?Expires=2147483647&Signature=O~0W16TMPgo9Mf-l~xA9M3uNEP4UN4Rq-YYaFeMWqh-lQvrhIMWKUSy9rU46mveOh~cnnJjOX9mgH8gBotPmRPcot9psghNmigfYxVfvzVAwGVmEKIzehgjkUAELRoDj9gpGCwvUH7Kfv6-okVjkbfdFZzfWHAYIQ3PxHcoVKelXdDSdkDppV6t988D3o~l2Dsi8LpM-vRkvu3TXplD1MLdsXGJzZ7EjWA017r-NfESdIjeEMaFO8Pht55y1719C6zkSOII0a7d6y8ib~LgPJBEBoPxfSpQe~YguPzyUsxUZ15Hn4vz7VSZa3gy6tHcVDHmZHRsok6krXEGxoy4xjQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Monitoring the errors in the barcode sequence tags and absolute quantitation of target molecules. Results from Ion Proton sequencer data. The target region is TK102U, except in F. (A–D); results obtained using 40 ng of genomic DNA. (A) Distribution of reads per barcode tag. Vertical axis: number of different barcode tags. Horizontal axis: number of reads per tag, given as the common logarithm of the number. (B) Cumulative reads. (C) Proportions of barcode tags by size. (D) Mean proportion of 12-bp barcode tags after including the 11-bp and 13-bp tags that only differed from a 12-bp tag by the insertion or deletion of a single base. The mean proportion represents the average of the proportions of the nearest 11 bins. (E) Estimated number of target molecules after the removal of erroneous tags. Horizontal axis: number of reads used for estimation; the reads were randomly selected from the entire population of reads (‘5 ng’: 1,457,760; ‘10 ng’: 2,251,133; ‘20 ng’: 2,245,038; and ‘40 ng’: 2,395,763 reads). (F) Correlation between the number of molecules and the amount of input DNA after removal of the erroneous tags." path-from-xml="dsv01002.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4941297/dsv01002.tiff" path-from-xml="dsv01002.tiff" target="_blank">View large</a><a section="4941297" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv010/2/dsv01002.png?Expires=2147483647&Signature=v9g7ibUUDL4guA7ZQt6gZo1YLomrCBo2R64P4CVBqHc-mp7JCTFzajt9-Te8qJOUxqI4RJr2vi~n5Uxp0-bKsS7ZyFxQETrSkICwCD9dmREagYxr9eODJWiAapmCxsNo~zTC-c6NDfx43OUS2jcC2UpLw2qM03cswOKEEx-yhux1JMxYZKd782UEbKYzZ0x3sOBHoSrCM-MBYuSmZNf~H3M0OvEoOYTyRga3OqIcDhKbtl2Ur3RMriSbJaOE3Llur1gK0NBA6sGCgefnLPtLtST~OD08p~~Dwb4kYWGBmsqIRoJI9VhWH4G-sDtJ4UFFADVnG22~hSqfotHyCPzLCw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4941297&ar=348206&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv01002.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Monitoring the errors in the barcode sequence tags and absolute quantitation of target molecules. Results from Ion Proton sequencer data. The target region is TK102U, except in F. (A–D); results obtained using 40 ng of genomic DNA. (A) Distribution of reads per barcode tag. Vertical axis: number of different barcode tags. Horizontal axis: number of reads per tag, given as the common logarithm of the number. (B) Cumulative reads. (C) Proportions of barcode tags by size. (D) Mean proportion of 12-bp barcode tags after including the 11-bp and 13-bp tags that only differed from a 12-bp tag by the insertion or deletion of a single base. The mean proportion represents the average of the proportions of the nearest 11 bins. (E) Estimated number of target molecules after the removal of erroneous tags. Horizontal axis: number of reads used for estimation; the reads were randomly selected from the entire population of reads (‘5 ng’: 1,457,760; ‘10 ng’: 2,251,133; ‘20 ng’: 2,245,038; and ‘40 ng’: 2,395,763 reads). (F) Correlation between the number of molecules and the amount of input DNA after removal of the erroneous tags.</p></div></div><p>Insertion/deletion errors occupy the majority (&gt;90%) of sequencing errors in the output sequences from the Ion Torrent PGM/Proton.<sup><a reveal-id="DSV010C26" data-open="DSV010C26" class="link link-ref link-reveal xref-bibr">26</a>,<a reveal-id="DSV010C27" data-open="DSV010C27" class="link link-ref link-reveal xref-bibr">27</a></sup> Consequently, the tags generated by read errors can be monitored using the tag size. The observed fractions of tags were grouped by size and are shown in Fig. <a reveal-id="DSV010F2" data-open="DSV010F2" class="link link-reveal link-table xref-fig">2</a>C. Non-12-bp tags, that is erroneous tags, occupied the majority of tags with low numbers of reads per tag, and the fraction of 12-bp tags gradually increased as the number of reads per tag increased. This dynamic suggests that erroneous tags accumulate in the low read number fraction and can be removed by setting an appropriate threshold. To recover the maximum number of reads, we grouped the 11-bp and 13-bp tags that matched 12-bp tags with the exception of a single inserted or deleted base with their matching 12-bp tags. Then, we plotted the fraction of 12-bp tags against the number of reads (Fig. <a reveal-id="DSV010F2" data-open="DSV010F2" class="link link-reveal link-table xref-fig">2</a>D). The minimum bin, where the average of the proportion of 12-bp tags in the nearest 11 bins was &gt;90%, was defined as the threshold for removing erroneous tags. The fraction of 12-bp tags obtained exceeded 95% of the recovered fraction, and little improvement was observed when using more stringent thresholds. This threshold successfully separated the two peaks observed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Figure S2A</a>. The chosen thresholds varied depending on factors such as the total number of reads and the targeted regions: from 11 to 249 (data points of Fig. <a reveal-id="DSV010F2" data-open="DSV010F2" class="link link-reveal link-table xref-fig">2</a>E) and from 57 to 485 (Fig. <a reveal-id="DSV010F2" data-open="DSV010F2" class="link link-reveal link-table xref-fig">2</a>F). Through this process, 10–20% of the total reads were discarded (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Figure S2B</a>).</p><p>We could not estimate the number of removed error-free tags by counting 12-bp tags, because the 12-bp tags in the fraction with a small number of reads contained sequences of the original size due to multiple insertion/deletion errors. Because the right peak (Fig. <a reveal-id="DSV010F2" data-open="DSV010F2" class="link link-reveal link-table xref-fig">2</a>A) represented the peak of the distribution of the error-free tags, we removed the fraction corresponding to the tail of the distribution. When <em>M</em> is defined as the threshold value, the number of error-free tags between 0 and <em>M</em> should not exceed the number of tags between <em>M</em> and 2*<em>M</em>. From the number of 12-bp tags between <em>M</em> and 2*<em>M</em>, the estimated maximum for the removed error-free tags was 5–10% of the total error-free tags.</p><p>The number of targeted molecules can be measured using exhaustive sequencing. The number of tags obtained reached saturation at 500,000 reads (Fig. <a reveal-id="DSV010F2" data-open="DSV010F2" class="link link-reveal link-table xref-fig">2</a>E). A linear relationship between the estimated number of target molecules and the amount of input DNA was observed, and the correlation coefficient was calculated to be &gt;0.98 (Fig. <a reveal-id="DSV010F2" data-open="DSV010F2" class="link link-reveal link-table xref-fig">2</a>F). Approximately 40% of the input DNA was recovered, except for TK103D, which exhibited 15% recovery. This calculation was based on the number of sequenced molecules. The difference in ligation efficiency among ligation sites is a likely cause of recovery variation.</p><p>There was a good linearity between the mutant-to-normal allele ratios of the artificial DNA mixtures and those deduced using the above procedure (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Figure S3</a>).</p><p>Previous studies employed arbitrary criteria for removing tags with a small number of reads, such as the removal of tags with a single read. After applying a criterion to remove 1- or 2-read tags, a considerable fraction of erroneous tags remained, and the number of tags exceeded the number of target molecules estimated from the amount of input DNA. The number of tags increased with the addition of reads and did not reach saturation (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Figure S4</a>), suggesting the generation of new erroneous tags.</p>                            <h3 scrollto-destination=4941303 id="4941303" class="section-title" data-legacy-id=s3c>3.3. Monitoring and removing erroneous barcode tags in the Illumina system</h3>
<p>The read error profile of Illumina sequencers is different from that of Ion Torrent PGM/Proton sequencers. When using Illumina systems, substitution errors dominate.<sup><a reveal-id="DSV010C26" data-open="DSV010C26" class="link link-ref link-reveal xref-bibr">26</a>,<a reveal-id="DSV010C27" data-open="DSV010C27" class="link link-ref link-reveal xref-bibr">27</a></sup> However, the observed distribution pattern of barcode tags was the same with both sequencing systems (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Figure S5A</a>). To accommodate Illumina sequencers, we used ‘BDHVBDHVBDHVBDH’ as an error-detecting barcode. Here, each base position lacks one of the four bases, and the appearance of that missing base indicates a read error. We can estimate the total number of erroneous tags by multiplying the number of tags with missing bases by three (Fig. <a reveal-id="DSV010F3" data-open="DSV010F3" class="link link-reveal link-table xref-fig">3</a>A, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Figure S5C</a>). We then determined the threshold for removing erroneous tags from the distribution of error-free tags, as described above. The results obtained with Illumina sequencers were similar to those obtained using Ion Proton sequencers: 10% of the reads were removed (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Figure S5B</a>); the values of the threshold varied from 15 to 65 (data points of Fig. <a reveal-id="DSV010F3" data-open="DSV010F3" class="link link-reveal link-table xref-fig">3</a>B); the number of tags was saturated with exhaustive sequencing (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Figure S5D</a>), but it increased continuously with the use of an arbitrary threshold (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Figure S5E</a>); and linearity was observed between the estimated number of target molecules and the amount of input DNA (Fig. <a reveal-id="DSV010F3" data-open="DSV010F3" class="link link-reveal link-table xref-fig">3</a>B). </p>                        <a id="4941305" scrollto-destination="4941305"></a>
<div id="DSV010F3" data-id="DSV010F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv010/2/m_dsv01003.png?Expires=2147483647&Signature=Ujt9HgMI54hRiFkmlFQ6JGsrD03LNBIPP0W0DND55W3VnFv3wiHrFZtFR9LOGnoqgdXu4T1cuRUXOI5nrY10Uf-cd~cqWZQZFLdbMM17wFi09pgjXFp4IstmYl53jjirZUjQSJek299I5V7Xaw8kchmSAFF5grR9Oq251Px1-P~mAbQwVyvNEOHPX0sAVj2ARWQMGneofT9KvLOVYU2i-Ukd3inh2FgYPRmo1GoOWyW2NLKG3eY1SDC36AYCpWHCJ2Fi4wBd0o-vlqr~FicwIdqXOxFVl1-d5fM3guNOvPPoTIYOCEzQJC~rf0nHIfnichMAvM3YZbR003u0r83qww__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Monitoring the errors in the barcode sequence tags and absolute quantitation of target molecules. Results from MiSeq sequencer data. (A) Estimated mean proportion of error-free barcode tags. The mean proportion was calculated as in Fig. 2D. (B) Correlation between the number of molecules and the amount of input DNA after removal of the erroneous tags." path-from-xml="dsv01003.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4941305/dsv01003.tiff" path-from-xml="dsv01003.tiff" target="_blank">View large</a><a section="4941305" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv010/2/dsv01003.png?Expires=2147483647&Signature=IWPprxCRauP~uZUxyePy0jNOce7dO-Q3xJwFKkGYrJSX6z7uz2mMnIVMylg7I6vFwjA6gut8rorGSfXtvEIuJvwYfZzi4n9a-D4vIZykXsn2e89vyAW4MUToeIz9U~-3IuazESk04UorM8WhBpt0kYU2i25EHKZ3EXphJ2vqA1q2NmgN3vABTScjyl5aYn7Jo4p6C8TunVmYpCCm2WjpnWMz3XUWdcqUtuEgycyV54r8i6WQ25U9WRAqMGnDJjnnA1YbdoY2zkTApYznPeKw-ZjJl0qZEq0EMLx5FXKdzOMtSvb6JeSI~YKnisnaJ6ScICo3zs98xHAbHNZkWoaYGw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4941305&ar=348206&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv01003.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Monitoring the errors in the barcode sequence tags and absolute quantitation of target molecules. Results from MiSeq sequencer data. (A) Estimated mean proportion of error-free barcode tags. The mean proportion was calculated as in Fig. <a reveal-id="DSV010F2" data-open="DSV010F2" class="link link-reveal link-table xref-fig">2</a>D. (B) Correlation between the number of molecules and the amount of input DNA after removal of the erroneous tags.</p></div></div><div content-id="DSV010F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv010/2/m_dsv01003.png?Expires=2147483647&Signature=Ujt9HgMI54hRiFkmlFQ6JGsrD03LNBIPP0W0DND55W3VnFv3wiHrFZtFR9LOGnoqgdXu4T1cuRUXOI5nrY10Uf-cd~cqWZQZFLdbMM17wFi09pgjXFp4IstmYl53jjirZUjQSJek299I5V7Xaw8kchmSAFF5grR9Oq251Px1-P~mAbQwVyvNEOHPX0sAVj2ARWQMGneofT9KvLOVYU2i-Ukd3inh2FgYPRmo1GoOWyW2NLKG3eY1SDC36AYCpWHCJ2Fi4wBd0o-vlqr~FicwIdqXOxFVl1-d5fM3guNOvPPoTIYOCEzQJC~rf0nHIfnichMAvM3YZbR003u0r83qww__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Monitoring the errors in the barcode sequence tags and absolute quantitation of target molecules. Results from MiSeq sequencer data. (A) Estimated mean proportion of error-free barcode tags. The mean proportion was calculated as in Fig. 2D. (B) Correlation between the number of molecules and the amount of input DNA after removal of the erroneous tags." path-from-xml="dsv01003.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4941305/dsv01003.tiff" path-from-xml="dsv01003.tiff" target="_blank">View large</a><a section="4941305" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv010/2/dsv01003.png?Expires=2147483647&Signature=IWPprxCRauP~uZUxyePy0jNOce7dO-Q3xJwFKkGYrJSX6z7uz2mMnIVMylg7I6vFwjA6gut8rorGSfXtvEIuJvwYfZzi4n9a-D4vIZykXsn2e89vyAW4MUToeIz9U~-3IuazESk04UorM8WhBpt0kYU2i25EHKZ3EXphJ2vqA1q2NmgN3vABTScjyl5aYn7Jo4p6C8TunVmYpCCm2WjpnWMz3XUWdcqUtuEgycyV54r8i6WQ25U9WRAqMGnDJjnnA1YbdoY2zkTApYznPeKw-ZjJl0qZEq0EMLx5FXKdzOMtSvb6JeSI~YKnisnaJ6ScICo3zs98xHAbHNZkWoaYGw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4941305&ar=348206&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv01003.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Monitoring the errors in the barcode sequence tags and absolute quantitation of target molecules. Results from MiSeq sequencer data. (A) Estimated mean proportion of error-free barcode tags. The mean proportion was calculated as in Fig. <a reveal-id="DSV010F2" data-open="DSV010F2" class="link link-reveal link-table xref-fig">2</a>D. (B) Correlation between the number of molecules and the amount of input DNA after removal of the erroneous tags.</p></div></div>                            <h3 scrollto-destination=4941306 id="4941306" class="section-title" data-legacy-id=s3d>3.4. Increased accuracy from building a consensus of reads from individual molecules</h3>
<p>The use of barcode tags enables high-fidelity sequencing by grouping and constructing a consensus of multiple sequences generated from a single molecule. The accuracy of this method is shown in Fig. <a reveal-id="DSV010F4" data-open="DSV010F4" class="link link-reveal link-table xref-fig">4</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Figure S6</a>. The error rates using barcode tags overlapped with a 95% confidence interval due to the small number of non-reference bases. Although we compared the results of using DNA polymerases with different proofreading functions (Q5 DNA polymerase from NEB and Platinum Taq DNA polymerase High Fidelity from Life Technologies) for the PCR amplification step of library construction, there was no significant difference when using barcode tags (Fig. <a reveal-id="DSV010F4" data-open="DSV010F4" class="link link-reveal link-table xref-fig">4</a>). The second version of the barcoding method, which involved labelling both strands with the same barcode sequence, did not improve the accuracy compared with that of the single strand labelling (Fig. <a reveal-id="DSV010F4" data-open="DSV010F4" class="link link-reveal link-table xref-fig">4</a>). For the first version, we used linear amplification cycles prior to PCR, which minimized the errors obtained in the early PCR cycles. This may explain why the second version of the barcoding method did not offer advantages over the single strand labelling method. </p>                        <a id="4941308" scrollto-destination="4941308"></a>
<div id="DSV010F4" data-id="DSV010F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv010/2/m_dsv01004.png?Expires=2147483647&Signature=Lr-GnD7nVyMwCZTLvMGM5mGQXQeKD70k0lGHL~JUt4PORLiJczPycybMBl-gz-iHqKNASEYC4AFeXhaLrZEtLu8B0HotewaS2UyU3jFBabRNR5OWYxORknQ3cJwXF36ZZDf9yTsH7E5hhB-keyNmgtxcm29hz8Y8kUD3hjLX5vy4WDYgFf-IB~83vgQ1Rfu8SxiSPbJBfCaSQEA6Par4KbXAoMhhAOv8euyQ7riLacLH3wobs8oCMl2lttv5aw52svOUgc2gz68eSmfOzh7L-vLK1z54e7Orjd~vZnaavDggocp9MXGHQPPlU8jmLssf1Kp94Ctq~ZtNVtJ0jTJVbg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Sequencing error rates for the target regions. Substitution error rates with (black) and without (gray) barcode tags. Q5, single strand labelling with Q5 DNA polymerase for PCR amplification; Pt, single strand labelling with the Platinum Taq DNA polymerase High Fidelity kit for PCR amplification; DS, double strand labelling. Thirty nanograms of genomic DNA were used. The calculations were based on the sequence data from the seven (Q5, Pt) or five (except TK102 and TK103U for DS) regions obtained using an Ion Proton sequencer. Ninety-five per cent confidence intervals of the error rates are as follows: Q5 tag+, 2.8 × 10−6– 8.8 × 10−6; Pt tag+, 6.9 × 10−6– 1.3 × 10−5; DS tag+, 3.3 × 10−6– 1.6 × 10−5; Q5 tag−, 9.0 × 10−5– 9.3 × 10−5; Pt tag−, 5.7 × 10−4–5.7 × 10−4; DS tag−, 3.7 × 10−4– 3.7 × 10−4." path-from-xml="dsv01004.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4941308/dsv01004.tiff" path-from-xml="dsv01004.tiff" target="_blank">View large</a><a section="4941308" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv010/2/dsv01004.png?Expires=2147483647&Signature=JRheAIoOSyqQTP4X-RknWVaW6x9XSAOiPCIqYxE5dyZmj5b8efS36X-mJSRE4hsZ41P5dDZfc5sYNNRdPu6NMvwmg6jG8hpNBCpdli~jt0XEMvUqcXyXonAgMeljk5dl1C4BQhdj7LoLl-FM7tgOBHMPpTtCTPYfPRq63WQq1Ns-ZVHyI9aN~ZBdrfEx5239yPqmc4Sao-p3P75XJMKUruG8~r7z9PT5DL1MRUHUsJZcV1xdo0apDbiVDYSOwV26usiebl0ezeaF8f9aU~CWKC-XnmjsxyAKEzpuRft0k3kmKgV4iY14t3hNPvr0528~YmIcgwha4iXbUa3SW8KLnQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4941308&ar=348206&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv01004.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Sequencing error rates for the target regions. Substitution error rates with (black) and without (gray) barcode tags. Q5, single strand labelling with Q5 DNA polymerase for PCR amplification; Pt, single strand labelling with the Platinum Taq DNA polymerase High Fidelity kit for PCR amplification; DS, double strand labelling. Thirty nanograms of genomic DNA were used. The calculations were based on the sequence data from the seven (Q5, Pt) or five (except TK102 and TK103U for DS) regions obtained using an Ion Proton sequencer. Ninety-five per cent confidence intervals of the error rates are as follows: Q5 tag+, 2.8 × 10<sup>−6</sup>– 8.8 × 10<sup>−6</sup>; Pt tag+, 6.9 × 10<sup>−6</sup>– 1.3 × 10<sup>−5</sup>; DS tag+, 3.3 × 10<sup>−6</sup>– 1.6 × 10<sup>−5</sup>; Q5 tag−, 9.0 × 10<sup>−5</sup>– 9.3 × 10<sup>−5</sup>; Pt tag−, 5.7 × 10<sup>−4</sup>–5.7 × 10<sup>−4</sup>; DS tag−, 3.7 × 10<sup>−4</sup>– 3.7 × 10<sup>−4</sup>.</p></div></div><div content-id="DSV010F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv010/2/m_dsv01004.png?Expires=2147483647&Signature=Lr-GnD7nVyMwCZTLvMGM5mGQXQeKD70k0lGHL~JUt4PORLiJczPycybMBl-gz-iHqKNASEYC4AFeXhaLrZEtLu8B0HotewaS2UyU3jFBabRNR5OWYxORknQ3cJwXF36ZZDf9yTsH7E5hhB-keyNmgtxcm29hz8Y8kUD3hjLX5vy4WDYgFf-IB~83vgQ1Rfu8SxiSPbJBfCaSQEA6Par4KbXAoMhhAOv8euyQ7riLacLH3wobs8oCMl2lttv5aw52svOUgc2gz68eSmfOzh7L-vLK1z54e7Orjd~vZnaavDggocp9MXGHQPPlU8jmLssf1Kp94Ctq~ZtNVtJ0jTJVbg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Sequencing error rates for the target regions. Substitution error rates with (black) and without (gray) barcode tags. Q5, single strand labelling with Q5 DNA polymerase for PCR amplification; Pt, single strand labelling with the Platinum Taq DNA polymerase High Fidelity kit for PCR amplification; DS, double strand labelling. Thirty nanograms of genomic DNA were used. The calculations were based on the sequence data from the seven (Q5, Pt) or five (except TK102 and TK103U for DS) regions obtained using an Ion Proton sequencer. Ninety-five per cent confidence intervals of the error rates are as follows: Q5 tag+, 2.8 × 10−6– 8.8 × 10−6; Pt tag+, 6.9 × 10−6– 1.3 × 10−5; DS tag+, 3.3 × 10−6– 1.6 × 10−5; Q5 tag−, 9.0 × 10−5– 9.3 × 10−5; Pt tag−, 5.7 × 10−4–5.7 × 10−4; DS tag−, 3.7 × 10−4– 3.7 × 10−4." path-from-xml="dsv01004.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4941308/dsv01004.tiff" path-from-xml="dsv01004.tiff" target="_blank">View large</a><a section="4941308" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv010/2/dsv01004.png?Expires=2147483647&Signature=JRheAIoOSyqQTP4X-RknWVaW6x9XSAOiPCIqYxE5dyZmj5b8efS36X-mJSRE4hsZ41P5dDZfc5sYNNRdPu6NMvwmg6jG8hpNBCpdli~jt0XEMvUqcXyXonAgMeljk5dl1C4BQhdj7LoLl-FM7tgOBHMPpTtCTPYfPRq63WQq1Ns-ZVHyI9aN~ZBdrfEx5239yPqmc4Sao-p3P75XJMKUruG8~r7z9PT5DL1MRUHUsJZcV1xdo0apDbiVDYSOwV26usiebl0ezeaF8f9aU~CWKC-XnmjsxyAKEzpuRft0k3kmKgV4iY14t3hNPvr0528~YmIcgwha4iXbUa3SW8KLnQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4941308&ar=348206&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv01004.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Sequencing error rates for the target regions. Substitution error rates with (black) and without (gray) barcode tags. Q5, single strand labelling with Q5 DNA polymerase for PCR amplification; Pt, single strand labelling with the Platinum Taq DNA polymerase High Fidelity kit for PCR amplification; DS, double strand labelling. Thirty nanograms of genomic DNA were used. The calculations were based on the sequence data from the seven (Q5, Pt) or five (except TK102 and TK103U for DS) regions obtained using an Ion Proton sequencer. Ninety-five per cent confidence intervals of the error rates are as follows: Q5 tag+, 2.8 × 10<sup>−6</sup>– 8.8 × 10<sup>−6</sup>; Pt tag+, 6.9 × 10<sup>−6</sup>– 1.3 × 10<sup>−5</sup>; DS tag+, 3.3 × 10<sup>−6</sup>– 1.6 × 10<sup>−5</sup>; Q5 tag−, 9.0 × 10<sup>−5</sup>– 9.3 × 10<sup>−5</sup>; Pt tag−, 5.7 × 10<sup>−4</sup>–5.7 × 10<sup>−4</sup>; DS tag−, 3.7 × 10<sup>−4</sup>– 3.7 × 10<sup>−4</sup>.</p></div></div><p>The accuracy of the first version of the barcoding method using the Illumina system was 1.8 × 10<sup>−6</sup> (95% confidence interval, 3.5 × 10<sup>−8</sup>– 6.9 × 10<sup>−6</sup>).</p>                            <h3 scrollto-destination=4941310 id="4941310" class="section-title" data-legacy-id=s3e>3.5. Analysis of plasma cfDNA from a gastric cancer patient</h3>
<p>We tested the NOIR sequencing system by monitoring the disease progression of a gastric cancer patient (73-yr-old male). The clinical characteristics of this patient were as follows: distal gastrectomy for a 65-mm tumour with multiple lymph node metastases; positive intraoperative peritoneal cytology; emergence of multiple liver metastases in spite of post-operative S-1 chemotherapy. This patient's primary tumour carried a mutation in <em>TP53</em> (c.747G &gt; C), and we used TK103U (Fig. <a reveal-id="DSV010F1" data-open="DSV010F1" class="link link-reveal link-table xref-fig">1</a>) to measure the level of the mutation at five different time points during the progression of the disease (Fig. <a reveal-id="DSV010F5" data-open="DSV010F5" class="link link-reveal link-table xref-fig">5</a>A, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>). The ctDNA level represented by the <em>TP53</em> mutation was zero or low during the early period of the disease and increased during the later period when the disease had progressed. In contrast, the level of cfDNA had no significant changes. Except for the above mutation, there were no base changes in any of the consensus reads. </p>                        <a id="4941312" scrollto-destination="4941312"></a>
<div id="DSV010F5" data-id="DSV010F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv010/2/m_dsv01005.png?Expires=2147483647&Signature=aFX9hnO1ncvs3STiEvgP-npMqqjsiPPufxpbfRWFFEfQA41qAG17By0676bqZFoQZkIKBugbM7F2NSRPL195j~ZhzUy9HzFL~Vb~nzMoSaYLzs4wAU5zg-il9v2Syq2Q5RMfz6uaFSSCREYZ0HckmyuTyY~5LUxt9eJdRjonhCiWuWQ958RST7bCSVUZcpMdwC2n6lcww7YSALilDqIYqxVyTTgHsA~jnX0IG4CCi61TMkuBS4bSiewPZ6FqiR9vs7nuMelxeX-V1E6kEu2joW3yhILmeIu7RL1~RIMajCAmbD6J1yvyNypRFCO058XKHBfQPJ60VBBFqzQR54x8sQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Dynamics of ctDNA during the disease progression of a gastric cancer patient revealed using the NOIR sequencing system. (A) The levels of the TP53 mutant fraction in the cell-free DNA, that is ctDNA (%), and the number of total and mutation-positive molecules at various time points after the patient was operated on are displayed. (B and C) Fraction of calls not matching the reference sequence (Month 9): NOIR (B); deep sequencing, 1.7 total million reads (C). The red line indicates the criterion for variant detection proposed by Couraud et al." path-from-xml="dsv01005.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4941312/dsv01005.tiff" path-from-xml="dsv01005.tiff" target="_blank">View large</a><a section="4941312" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv010/2/dsv01005.png?Expires=2147483647&Signature=QXzCLhdZjv7R86zp9EfYKkKrR2iBLHenwdwiySShCAiVZdix9lMmOok4uE3wthoy0PSmehQTSntf~1fYu1s6nIJ7JXCX1b3Dn9EB59e5R164CJIdHVHA7yVjhouZlgqKhKooDdjzrxdGD44YmFG8Qdlm87tUIoKQkjqSA-BGv~XTlrZ0MhFfRV5oD~3mkiXbOELnLI5cNrmnkqJYU7WO4F5Je1Vf9y082Sm8yQEuqseRTWBXda8nAzSgdXHiVQsLUJWyAQq0W-BNxWWR8uVTVpQvPOAIGfeEgEcuvtBJTnz8yPXS080HKL6MfzMOgw1puVXR6fEg2wcDjL-oZuaucg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4941312&ar=348206&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv01005.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Dynamics of ctDNA during the disease progression of a gastric cancer patient revealed using the NOIR sequencing system. (A) The levels of the <em>TP53</em> mutant fraction in the cell-free DNA, that is ctDNA (%), and the number of total and mutation-positive molecules at various time points after the patient was operated on are displayed. (B and C) Fraction of calls not matching the reference sequence (Month 9): NOIR (B); deep sequencing, 1.7 total million reads (C). The red line indicates the criterion for variant detection proposed by Couraud et al.</p></div></div><div content-id="DSV010F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv010/2/m_dsv01005.png?Expires=2147483647&Signature=aFX9hnO1ncvs3STiEvgP-npMqqjsiPPufxpbfRWFFEfQA41qAG17By0676bqZFoQZkIKBugbM7F2NSRPL195j~ZhzUy9HzFL~Vb~nzMoSaYLzs4wAU5zg-il9v2Syq2Q5RMfz6uaFSSCREYZ0HckmyuTyY~5LUxt9eJdRjonhCiWuWQ958RST7bCSVUZcpMdwC2n6lcww7YSALilDqIYqxVyTTgHsA~jnX0IG4CCi61TMkuBS4bSiewPZ6FqiR9vs7nuMelxeX-V1E6kEu2joW3yhILmeIu7RL1~RIMajCAmbD6J1yvyNypRFCO058XKHBfQPJ60VBBFqzQR54x8sQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Dynamics of ctDNA during the disease progression of a gastric cancer patient revealed using the NOIR sequencing system. (A) The levels of the TP53 mutant fraction in the cell-free DNA, that is ctDNA (%), and the number of total and mutation-positive molecules at various time points after the patient was operated on are displayed. (B and C) Fraction of calls not matching the reference sequence (Month 9): NOIR (B); deep sequencing, 1.7 total million reads (C). The red line indicates the criterion for variant detection proposed by Couraud et al." path-from-xml="dsv01005.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4941312/dsv01005.tiff" path-from-xml="dsv01005.tiff" target="_blank">View large</a><a section="4941312" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv010/2/dsv01005.png?Expires=2147483647&Signature=QXzCLhdZjv7R86zp9EfYKkKrR2iBLHenwdwiySShCAiVZdix9lMmOok4uE3wthoy0PSmehQTSntf~1fYu1s6nIJ7JXCX1b3Dn9EB59e5R164CJIdHVHA7yVjhouZlgqKhKooDdjzrxdGD44YmFG8Qdlm87tUIoKQkjqSA-BGv~XTlrZ0MhFfRV5oD~3mkiXbOELnLI5cNrmnkqJYU7WO4F5Je1Vf9y082Sm8yQEuqseRTWBXda8nAzSgdXHiVQsLUJWyAQq0W-BNxWWR8uVTVpQvPOAIGfeEgEcuvtBJTnz8yPXS080HKL6MfzMOgw1puVXR6fEg2wcDjL-oZuaucg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4941312&ar=348206&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv01005.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Dynamics of ctDNA during the disease progression of a gastric cancer patient revealed using the NOIR sequencing system. (A) The levels of the <em>TP53</em> mutant fraction in the cell-free DNA, that is ctDNA (%), and the number of total and mutation-positive molecules at various time points after the patient was operated on are displayed. (B and C) Fraction of calls not matching the reference sequence (Month 9): NOIR (B); deep sequencing, 1.7 total million reads (C). The red line indicates the criterion for variant detection proposed by Couraud et al.</p></div></div><p>We compared the above results with those obtained by deep sequencing using reads before the construction of NOIR (Fig. <a reveal-id="DSV010F5" data-open="DSV010F5" class="link link-reveal link-table xref-fig">5</a>B and C). With the criterion of variant detection proposed by Couraud et al.,<sup><a reveal-id="DSV010C28" data-open="DSV010C28" class="link link-ref link-reveal xref-bibr">28</a></sup> &gt;20 positions were erroneously identified as variants.</p>                            <h3 scrollto-destination=4941314 id="4941314" class="section-title" data-legacy-id=s3f>3.6. Mutation screening in lung cancer</h3>
<p>We performed a model screening experiment using cfDNA from lung cancer patients. The samples were originally collected for a clinical study of <em>EGFR</em> mutations, for which the statuses of the biopsy samples were available. We used <em>KRAS</em> (a 73-bp region) and <em>CTNNB1</em> (a 63-bp region) for the target genes. <em>KRAS</em> has a hotspot region in codons 12 and 13. In lung cancer, the <em>KRAS</em> and <em>EGFR</em> mutations are exclusive and rarely co-exist in the same patients. Unlike <em>KRAS</em> and <em>EGFR, CTNNB1</em> mutations are infrequent in lung cancer.<sup><a reveal-id="DSV010C29" data-open="DSV010C29" class="link link-ref link-reveal xref-bibr">29</a></sup><em>KRAS</em> hot spot mutations should appear only in <em>EGFR</em> mutation-negative lung cancer, and no mutations should be found in samples from normal individuals.</p><div class="block-child-p">When the number of base changes in a target region is significantly higher than the average, we may attribute the changes to variant(s). We can use a statistical model to calculate the probability that a specific number of sequencing errors will occur. The average number of base changes due to sequencing errors, <em>λ</em>, is as follows:<a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">λ</mi><mo xmlns="">=</mo><mi xmlns="">l</mi><mo xmlns="">×</mo><mi xmlns="">m</mi><mo xmlns="">×</mo><mtext xmlns="">ER</mtext></math></div></div></div> where, <em>l</em>, <em>m</em> and ER are the number of base pairs in a target region, the number of sequenced molecules and the sequencing error rate, respectively. With the application of a Poisson distribution model for the sequencing error, the probability of <em>n</em> or more sequencing errors is as follows:<a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">P</mi><mo xmlns="">=</mo><mn xmlns="">1</mn><mo xmlns="">−</mo><munderover xmlns=""><mrow><mo>∑</mo></mrow><mrow><mi>k</mi><mo>=</mo><mn>0</mn></mrow><mrow><mi>n</mi><mo>−</mo><mn>1</mn></mrow></munderover><mstyle xmlns=""><mrow><mfrac><mrow><mspace width="thickmathspace"></mspace><mrow><msup><mrow><mo stretchy="false">(</mo><mi>l</mi><mo>×</mo><mi>m</mi><mo>×</mo><mtext>ER</mtext><mo stretchy="false">)</mo></mrow><mi>k</mi></msup></mrow><mrow><msup><mtext>e</mtext><mrow><mo>−</mo><mi>l</mi><mo>×</mo><mi>m</mi><mo>×</mo><mrow><mi mathvariant="normal">ER</mi></mrow></mrow></msup></mrow></mrow><mrow><mi>k</mi><mo>!</mo></mrow></mfrac></mrow></mstyle></math></div></div></div> In the following experiments, we set <em>P</em> = 10<sup>−3</sup>, that is one false positive in 1,000 samples, as the criterion for anomaly detection.<sup><a reveal-id="DSV010C16" data-open="DSV010C16" class="link link-ref link-reveal xref-bibr">16</a></sup> Cases with <em>P</em> ≤ 10<sup>−3</sup> are regarded as variant(s)-positive. ER was set to 10<sup>−5</sup> for NOIR and 5 × 10<sup>−4</sup> for deep sequencing. It should be noted that this criterion evaluates the entire target region and not individual base positions.</div><p>The number of samples was 11 of <em>EGFR</em> mutation-positive lung cancer, 19 of <em>EGFR</em> mutation-negative lung cancer, 11 of leucocyte DNA from normal individuals and 7 of cfDNA from normal individuals. There was no significant loss of template molecules with these two genes during the assay process. The results are shown in Fig. <a reveal-id="DSV010F6" data-open="DSV010F6" class="link link-reveal link-table xref-fig">6</a>A and B, and the corresponding data are provided in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S7 and S8</a>. Among the nine variant-positive cases in the <em>KRAS</em> experiment, eight were <em>EGFR</em> mutation-negative lung cancers that had substitutions in codon 12, and one was an <em>EGFR</em> mutation-positive case in which the substitutions were non-synonymous and located outside of the hotspot region. All cases with base changes in the hotspot region (codon 12) were judged as variant-positive. Among the three variant-positive cases in the <em>CTNNB1</em> experiment, two cases were <em>EGFR</em> mutation-negative, and the base changes were a non-synonymous mutation and a synonymous mutation. One variant-positive case showed normal leucocyte DNA; this is the sole false-positive case. Because redundancy in sequenced molecules was removed, we can discriminate experimental errors and true mutations from distribution patterns of base changes. Base changes in this case were regarded as experimental errors, because all eight base changes were located at different base positions. In contrast, base changes in all other variant-positive cases were accumulated at a single base position. The results agreed well with previous knowledge of the mutations. Because <em>KRAS</em> mutations have been found in 25% of lung cancers,<sup><a reveal-id="DSV010C30" data-open="DSV010C30" class="link link-ref link-reveal xref-bibr">30</a></sup> the above results suggest that most of the <em>KRAS</em> mutation-positive lung cancers were likely to be detected using plasma cfDNA. </p>                        <a id="4941318" scrollto-destination="4941318"></a>
<div id="DSV010F6" data-id="DSV010F6" class="fig fig-section"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv010/2/m_dsv01006.png?Expires=2147483647&Signature=Ee~I3Z3stcsA4f6zziInhKo67zXDUrQskdNIFQiUXYAnOhqYXG2uojw1jfJPRRVXWvF-09wN~mtZ93qoqkvXK4V1sirAZ~b2b6q01gmhvPACJb8Nh6zvs8~1pMkCAHoIfEDoSaVX6d0z1Nu7-wBcQx4Rrvekb4nqlvJoandGZbgahSVv-NnA5j2G~7DaN9j-g4poR~hI1BwfYToB1ur5AyX2wBz0NdhWdO15Znq5IojkUveOAlB5qfswSsbmJYzd4IvBepmeSwR1ip0Xvnj-qDtfjkFeadeucEOVAO7QJbcyfoXM6tWAxT30e7S6ClWl4KVNZP3okkNyBtt~y70kdg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="De novo detection of KRAS mutations in plasma cell-free DNA from lung cancer patients. The vertical axis indicates the P, that is the probability of false positives, described in the main text. Horizontal axis: normal, leucocyte or plasma cell-free DNA from normal individuals; Pwt, EGFR mutation-negative patients; Pmt, EGFR mutation-positive patients. The plots correspond to KRAS (A and C) and CTNNB1 (B and D). A and B show the results from the NOIR sequencing system. C and D show the results from randomly extracted sequence reads (500 reads from plasma samples and 4,000 reads from leucocyte samples) without the use of barcodes. The dotted lines indicate the threshold for variant detection (P = 0.001). False-positive samples in C and D are marked with grey background." path-from-xml="dsv01006.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4941318/dsv01006.tiff" path-from-xml="dsv01006.tiff" target="_blank">View large</a><a section="4941318" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv010/2/dsv01006.png?Expires=2147483647&Signature=iiZkkzfBAJBYMIqmtcU0G~KA6xV03IeeKT4f9RVV56Y7g5G6jQbXz9vD3OI8-wwtpDLlUZQ5VIcVbUR70~z9~5iILzD~P8pV9vONBNpv9qdQEiRr6-9wcZUTNz6Oxnbq3aC-4aBogYBMoGZzcCvK8pwTZU6grkg85BfKDBQFqzZC-Py31cxq5vVvxDacoYTDq52-25eXX3mv3Y0cxI6qZc29a-MCidgy4M00VonF62ADohQcVVZ4n0G~YmmvUQrnPhRDZTRk75aMl28xt648dcTOGZmWw72nFpAl4-tQyhsxoMDY3IFRnRhRxtmZhmBGidndZoKQMUjDW1xbNJw9fw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4941318&ar=348206&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv01006.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p><em>De novo</em> detection of <em>KRAS</em> mutations in plasma cell-free DNA from lung cancer patients. The vertical axis indicates the <em>P</em>, that is the probability of false positives, described in the main text. Horizontal axis: normal, leucocyte or plasma cell-free DNA from normal individuals; Pwt, <em>EGFR</em> mutation-negative patients; Pmt, <em>EGFR</em> mutation-positive patients. The plots correspond to <em>KRAS</em> (A and C) and <em>CTNNB1</em> (B and D). A and B show the results from the NOIR sequencing system. C and D show the results from randomly extracted sequence reads (500 reads from plasma samples and 4,000 reads from leucocyte samples) without the use of barcodes. The dotted lines indicate the threshold for variant detection (<em>P</em> = 0.001). False-positive samples in C and D are marked with grey background.</p></div></div><div content-id="DSV010F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv010/2/m_dsv01006.png?Expires=2147483647&Signature=Ee~I3Z3stcsA4f6zziInhKo67zXDUrQskdNIFQiUXYAnOhqYXG2uojw1jfJPRRVXWvF-09wN~mtZ93qoqkvXK4V1sirAZ~b2b6q01gmhvPACJb8Nh6zvs8~1pMkCAHoIfEDoSaVX6d0z1Nu7-wBcQx4Rrvekb4nqlvJoandGZbgahSVv-NnA5j2G~7DaN9j-g4poR~hI1BwfYToB1ur5AyX2wBz0NdhWdO15Znq5IojkUveOAlB5qfswSsbmJYzd4IvBepmeSwR1ip0Xvnj-qDtfjkFeadeucEOVAO7QJbcyfoXM6tWAxT30e7S6ClWl4KVNZP3okkNyBtt~y70kdg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="De novo detection of KRAS mutations in plasma cell-free DNA from lung cancer patients. The vertical axis indicates the P, that is the probability of false positives, described in the main text. Horizontal axis: normal, leucocyte or plasma cell-free DNA from normal individuals; Pwt, EGFR mutation-negative patients; Pmt, EGFR mutation-positive patients. The plots correspond to KRAS (A and C) and CTNNB1 (B and D). A and B show the results from the NOIR sequencing system. C and D show the results from randomly extracted sequence reads (500 reads from plasma samples and 4,000 reads from leucocyte samples) without the use of barcodes. The dotted lines indicate the threshold for variant detection (P = 0.001). False-positive samples in C and D are marked with grey background." path-from-xml="dsv01006.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4941318/dsv01006.tiff" path-from-xml="dsv01006.tiff" target="_blank">View large</a><a section="4941318" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv010/2/dsv01006.png?Expires=2147483647&Signature=iiZkkzfBAJBYMIqmtcU0G~KA6xV03IeeKT4f9RVV56Y7g5G6jQbXz9vD3OI8-wwtpDLlUZQ5VIcVbUR70~z9~5iILzD~P8pV9vONBNpv9qdQEiRr6-9wcZUTNz6Oxnbq3aC-4aBogYBMoGZzcCvK8pwTZU6grkg85BfKDBQFqzZC-Py31cxq5vVvxDacoYTDq52-25eXX3mv3Y0cxI6qZc29a-MCidgy4M00VonF62ADohQcVVZ4n0G~YmmvUQrnPhRDZTRk75aMl28xt648dcTOGZmWw72nFpAl4-tQyhsxoMDY3IFRnRhRxtmZhmBGidndZoKQMUjDW1xbNJw9fw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4941318&ar=348206&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv01006.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p><em>De novo</em> detection of <em>KRAS</em> mutations in plasma cell-free DNA from lung cancer patients. The vertical axis indicates the <em>P</em>, that is the probability of false positives, described in the main text. Horizontal axis: normal, leucocyte or plasma cell-free DNA from normal individuals; Pwt, <em>EGFR</em> mutation-negative patients; Pmt, <em>EGFR</em> mutation-positive patients. The plots correspond to <em>KRAS</em> (A and C) and <em>CTNNB1</em> (B and D). A and B show the results from the NOIR sequencing system. C and D show the results from randomly extracted sequence reads (500 reads from plasma samples and 4,000 reads from leucocyte samples) without the use of barcodes. The dotted lines indicate the threshold for variant detection (<em>P</em> = 0.001). False-positive samples in C and D are marked with grey background.</p></div></div><p>We repeated the experiment with randomly selected raw reads (reads before the consensus was built) as a model of deep sequencing<sup><a reveal-id="DSV010C16" data-open="DSV010C16" class="link link-ref link-reveal xref-bibr">16</a></sup> (Fig. <a reveal-id="DSV010F6" data-open="DSV010F6" class="link link-reveal link-table xref-fig">6</a>C and D). The <em>P</em>-value of five and seven normal samples in the KRAS and <em>CTNNB1</em> experiments was less than the threshold, and judged as variant-positive. For deep sequencing, less stringent threshold was required to eliminate these false positives.</p>                            <h2 scrollto-destination=4941320 id="4941320" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>By using the NOIR sequencing system, we successfully eliminated false positives and could detect <em>de novo</em> and quantitate the mutations in plasma cfDNA. Therefore, it would be worthwhile to further evaluate the system in prospective clinical studies.</p><p>The current system is designed for <em>KRAS</em> and the hotspot region of <em>TP53</em>, which cover up to 50% of the cancer population, but the system can easily be extended to include other genes, such as <em>PI3KCA</em>, <em>PTEN</em> and <em>EGFR</em>. Our standard assay protocol uses plasma cfDNA from 1 ml of whole blood for a single reaction,<sup><a reveal-id="DSV010C16" data-open="DSV010C16" class="link link-ref link-reveal xref-bibr">16</a></sup> and this system can process 2–4 DNA fragments that are ∼100 bp long in a single reaction. Thus, standard blood sampling (5–10 ml) can cover a genomic region of 1–4 kb. Application of multiplex nested PCR may increase the number of target genes beyond this limitation. Moreover, the principle may be applied to exome sequencing. It is important to note that the NOIR sequencing system still introduces sequencing errors. As the size of the target regions increases, the false positives will increase. Accurate evaluation of the probability of false positives is important.</p><p>In the NOIR sequencing system, the barcode technology plays the most important role. The principle for our absolute quantitation can be applied to other systems such as RNA-seq. Deakin et al.<sup><a reveal-id="DSV010C31" data-open="DSV010C31" class="link link-ref link-reveal xref-bibr">31</a></sup> examined read errors within libraries of small numbers of barcode sequences. Their analysis revealed that the erroneous barcode tags localized in the fraction containing tags with small numbers of reads, and few of these tags were found in the fraction with large numbers of reads. Although the complexity of the barcode sequences was low, the results agreed well with those obtained in our study. Additionally, two or three barcode tags out of 100 were reported to be lost during PCR. The loss was likely sufficiently small to not introduce a bias that would affect the practical application of the barcode technology.</p><p>A possible alternative to the NOIR sequencing system is Safe-SeqS,<sup><a reveal-id="DSV010C12" data-open="DSV010C12" class="link link-ref link-reveal xref-bibr">12</a></sup> which uses a barcode (unique identifier) for rare mutation detection. Its error rate is similar to ours, but a direct comparison may not be valid due to the different sequencing platforms and experimental conditions. The current version of Safe-SeqS uses two or four PCR cycles for barcode addition and does not satisfy the basic principle of labelling each molecule with a single unique barcode. Safe-SeqS does not measure the errors that are introduced into barcode sequences either. Thus, absolute quantitation is not possible, and the final consensus reads are redundant. This redundancy would reduce the statistical power of the mutation detection.</p><p>Other deep sequencing approaches had higher error rates. Background frequencies of non-reference reads in Tam-Seq were ∼0.1%,<sup><a reveal-id="DSV010C32" data-open="DSV010C32" class="link link-ref link-reveal xref-bibr">32</a></sup> because using Q30 reads. Narayan et al.<sup><a reveal-id="DSV010C33" data-open="DSV010C33" class="link link-ref link-reveal xref-bibr">33</a></sup> employed overlapping paired-end reads to reduce miscalls. They could reduce the miscall frequency to 0.014%, which was still significantly higher than those obtained with the barcode technologies.</p><p>ctDNA has two unique features that clinical parameters do not have. One is that ctDNA carries genetic information from malignant tumours, and this information can be obtained through peripheral blood sampling. Another feature is that ctDNA is released from dying cancer cells, and its level reflects tumour burden and/or effects of anti-cancer drugs. Absolute quantitation of ctDNA would provide accurate information on the ctDNA dynamics to guide patient treatment.</p>                            <h2 scrollto-destination=4941327 id="4941327" class="section-title" data-legacy-id=s5>5. Data availability</h2>
<p>The sequence data were deposited in the DDBJ Sequence Reads Archive [DDBJ:DRA002833].</p>                            <h2 scrollto-destination=4941329 id="4941329" class="section-title" data-legacy-id=s6>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available at www.dnaresearch.oxfordjournals.org</a></p>                            <h2 scrollto-destination=4941331 id="4941331" class="section-title" data-legacy-id=s7>Funding</h2>
<p>This work was partly supported by the Ministry of Education, Culture, Sports, Science and Technology, Japan [KAKENHI 25430180]. Funding to pay the Open Access publication charges for this article was provided by the Osaka Medical Center for Cancer and Cardiovascular Diseases.</p>                            <h2 scrollto-destination=4941333 id="4941333" class="backacknowledgements-title" >Acknowledgements</h2>
<p>The authors thank Ms Noriko Ito for technical assistance. The authors also thank Dr Kazuhiro Katayama for fruitful discussions.</p>                            <h2 scrollto-destination=4941335 id="4941335" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSV010C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV010C1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Diehl</div>  <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Schmidt</div>  <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Choti</div>  <div class="given-names">M.A.</div></div> et al.  </div>. <div class="year">2008</div>, <div class="article-title">Circulating mutant DNA to assess tumor dynamics</div>, <div class="source">Nat. Med.</div> , <div class="volume">14</div>, <div class="fpage">985</div>–<div class="lpage">90</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Circulating%20mutant%20DNA%20to%20assess%20tumor%20dynamics&amp;author=F.%20Diehl&amp;author=K.%20Schmidt&amp;author=M.A.%20Choti&amp;publication_year=2008&amp;journal=Nat.%20Med.&amp;volume=14&amp;pages=985-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nm.1789" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnm.1789" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnm.1789"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18670422" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV010C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV010C2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Diaz</div>  <div class="given-names">L.A.</div>Jr.</div>, <div class="name string-name"><div class="surname">Williams</div>  <div class="given-names">R.T.</div></div>, <div class="name string-name"><div class="surname">Wu</div>  <div class="given-names">J.</div></div> et al.  </div>. <div class="year">2012</div>, <div class="article-title">The molecular evolution of acquired resistance to targeted EGFR blockade in colorectal cancers</div>, <div class="source">Nature</div> , <div class="volume">486</div>, <div class="fpage">537</div>–<div class="lpage">40</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20molecular%20evolution%20of%20acquired%20resistance%20to%20targeted%20EGFR%20blockade%20in%20colorectal%20cancers&amp;author=L.A.%20Diaz&amp;author=R.T.%20Williams&amp;author=J.%20Wu&amp;publication_year=2012&amp;journal=Nature&amp;volume=486&amp;pages=537-40" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22722843" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV010C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV010C3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Misale</div>  <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Yaeger</div>  <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Hobor</div>  <div class="given-names">S.</div></div> et al.  </div>. <div class="year">2012</div>, <div class="article-title">Emergence of KRAS mutations and acquired resistance to anti-EGFR therapy in colorectal cancer</div>, <div class="source">Nature</div> , <div class="volume">486</div>, <div class="fpage">532</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Emergence%20of%20KRAS%20mutations%20and%20acquired%20resistance%20to%20anti-EGFR%20therapy%20in%20colorectal%20cancer&amp;author=S.%20Misale&amp;author=R.%20Yaeger&amp;author=S.%20Hobor&amp;publication_year=2012&amp;journal=Nature&amp;volume=486&amp;pages=532-6" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22722830" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV010C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV010C4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Taniguchi</div>  <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Uchida</div>  <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Nishino</div>  <div class="given-names">K.</div></div> et al.  </div>. <div class="year">2011</div>, <div class="article-title">Quantitative detection of EGFR mutations in circulating tumor DNA derived from lung adenocarcinomas</div>, <div class="source">Clin. Cancer. Res.</div> , <div class="volume">17</div>, <div class="fpage">7808</div>–<div class="lpage">15</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Quantitative%20detection%20of%20EGFR%20mutations%20in%20circulating%20tumor%20DNA%20derived%20from%20lung%20adenocarcinomas&amp;author=K.%20Taniguchi&amp;author=J.%20Uchida&amp;author=K.%20Nishino&amp;publication_year=2011&amp;journal=Clin.%20Cancer.%20Res.&amp;volume=17&amp;pages=7808-15" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1158/1078-0432.CCR-11-1712" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1158%2F1078-0432.CCR-11-1712" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1158%2F1078-0432.CCR-11-1712"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21976538" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV010C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV010C5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Burrell</div>  <div class="given-names">R.A.</div></div>, <div class="name string-name"><div class="surname">McGranahan</div>  <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Bartek</div>  <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Swanton</div>  <div class="given-names">C.</div></div></div> <div class="year">2013</div>, <div class="article-title">The causes and consequences of genetic heterogeneity in cancer evolution</div>, <div class="source">Nature</div> , <div class="volume">501</div>, <div class="fpage">338</div>–<div class="lpage">45</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20causes%20and%20consequences%20of%20genetic%20heterogeneity%20in%20cancer%20evolution&amp;author=R.A.%20Burrell&amp;author=N.%20McGranahan&amp;author=J.%20Bartek&amp;author=C.%20Swanton&amp;publication_year=2013&amp;journal=Nature&amp;volume=501&amp;pages=338-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature12625" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature12625" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature12625"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24048066" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV010C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV010C6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Newman</div>  <div class="given-names">A.M.</div></div>, <div class="name string-name"><div class="surname">Bratman</div>  <div class="given-names">S.V.</div></div>, <div class="name string-name"><div class="surname">To</div>  <div class="given-names">J.</div></div> et al.  </div>. <div class="year">2014</div>, <div class="article-title">An ultrasensitive method for quantitating circulating tumor DNA with broad patient coverage</div>, <div class="source">Nat. Med.</div> , <div class="volume">20</div>, <div class="fpage">548</div>–<div class="lpage">54</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20ultrasensitive%20method%20for%20quantitating%20circulating%20tumor%20DNA%20with%20broad%20patient%20coverage&amp;author=A.M.%20Newman&amp;author=S.V.%20Bratman&amp;author=J.%20To&amp;publication_year=2014&amp;journal=Nat.%20Med.&amp;volume=20&amp;pages=548-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nm.3519" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnm.3519" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnm.3519"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24705333" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV010C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV010C7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lo</div>  <div class="given-names">Y.M.</div></div>, <div class="name string-name"><div class="surname">Zhang</div>  <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Leung</div>  <div class="given-names">T.N.</div></div>, <div class="name string-name"><div class="surname">Lau</div>  <div class="given-names">T.K.</div></div>, <div class="name string-name"><div class="surname">Chang</div>  <div class="given-names">A.M.</div></div>, <div class="name string-name"><div class="surname">Hjelm</div>  <div class="given-names">N.M.</div></div></div> <div class="year">1999</div>, <div class="article-title">Rapid clearance of fetal DNA from maternal plasma</div>, <div class="source">Am. J. Hum. Genet.</div> , <div class="volume">64</div>, <div class="fpage">218</div>–<div class="lpage">24</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rapid%20clearance%20of%20fetal%20DNA%20from%20maternal%20plasma&amp;author=Y.M.%20Lo&amp;author=J.%20Zhang&amp;author=T.N.%20Leung&amp;author=T.K.%20Lau&amp;author=A.M.%20Chang&amp;author=N.M.%20Hjelm&amp;publication_year=1999&amp;journal=Am.%20J.%20Hum.%20Genet.&amp;volume=64&amp;pages=218-24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1086/302205" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1086%2F302205" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1086%2F302205"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9915961" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV010C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV010C8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Vogelstein</div>  <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Kinzler</div>  <div class="given-names">K.W.</div></div></div> <div class="year">1999</div>, <div class="article-title">Digital PCR</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="volume">96</div>, <div class="fpage">9236</div>–<div class="lpage">41</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Digital%20PCR&amp;author=B.%20Vogelstein&amp;author=K.W.%20Kinzler&amp;publication_year=1999&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=96&amp;pages=9236-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.96.16.9236" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.96.16.9236" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.96.16.9236"> </span></div> </div></div></div></div></div><div content-id="DSV010C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV010C9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bettegowda</div>  <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Sausen</div>  <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Leary</div>  <div class="given-names">R.J.</div></div> et al.  </div>. <div class="year">2014</div>, <div class="article-title">Detection of circulating tumor DNA in early- and late-stage human malignancies</div>, <div class="source">Sci. Transl. Med.</div> , <div class="volume">6</div>, <div class="fpage">224ra224</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Detection%20of%20circulating%20tumor%20DNA%20in%20early-%20and%20late-stage%20human%20malignancies&amp;author=C.%20Bettegowda&amp;author=M.%20Sausen&amp;author=R.J.%20Leary&amp;publication_year=2014&amp;journal=Sci.%20Transl.%20Med.&amp;volume=6&amp;pages=224ra224" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/scitranslmed.3007094" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscitranslmed.3007094" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscitranslmed.3007094"> </span></div> </div></div></div></div></div><div content-id="DSV010C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV010C10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Dawson</div>  <div class="given-names">S.J.</div></div>, <div class="name string-name"><div class="surname">Tsui</div>  <div class="given-names">D.W.</div></div>, <div class="name string-name"><div class="surname">Murtaza</div>  <div class="given-names">M.</div></div> et al.  </div>. <div class="year">2013</div>, <div class="article-title">Analysis of circulating tumor DNA to monitor metastatic breast cancer</div>, <div class="source">N. Engl. J. Med.</div> , <div class="volume">368</div>, <div class="fpage">1199</div>–<div class="lpage">209</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20circulating%20tumor%20DNA%20to%20monitor%20metastatic%20breast%20cancer&amp;author=S.J.%20Dawson&amp;author=D.W.%20Tsui&amp;author=M.%20Murtaza&amp;publication_year=2013&amp;journal=N.%20Engl.%20J.%20Med.&amp;volume=368&amp;pages=1199-209" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1056/NEJMoa1213261" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1056%2FNEJMoa1213261" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1056%2FNEJMoa1213261"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23484797" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV010C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV010C11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Casbon</div>  <div class="given-names">J.A.</div></div>, <div class="name string-name"><div class="surname">Osborne</div>  <div class="given-names">R.J.</div></div>, <div class="name string-name"><div class="surname">Brenner</div>  <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Lichtenstein</div>  <div class="given-names">C.P.</div></div></div> <div class="year">2011</div>, <div class="article-title">A method for counting PCR template molecules with application to next-generation sequencing</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">39</div>, <div class="fpage">e81</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20method%20for%20counting%20PCR%20template%20molecules%20with%20application%20to%20next-generation%20sequencing&amp;author=J.A.%20Casbon&amp;author=R.J.%20Osborne&amp;author=S.%20Brenner&amp;author=C.P.%20Lichtenstein&amp;publication_year=2011&amp;journal=Nucleic%20Acids%20Res.&amp;volume=39&amp;pages=e81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkr217" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr217" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkr217"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21490082" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV010C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV010C12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kinde</div>  <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Wu</div>  <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Papadopoulos</div>  <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Kinzler</div>  <div class="given-names">K.W.</div></div>, <div class="name string-name"><div class="surname">Vogelstein</div>  <div class="given-names">B.</div></div></div> <div class="year">2011</div>, <div class="article-title">Detection and quantification of rare mutations with massively parallel sequencing</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="volume">108</div>, <div class="fpage">9530</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Detection%20and%20quantification%20of%20rare%20mutations%20with%20massively%20parallel%20sequencing&amp;author=I.%20Kinde&amp;author=J.%20Wu&amp;author=N.%20Papadopoulos&amp;author=K.W.%20Kinzler&amp;author=B.%20Vogelstein&amp;publication_year=2011&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=108&amp;pages=9530-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1105422108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1105422108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1105422108"> </span></div> </div></div></div></div></div><div content-id="DSV010C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV010C13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hamady</div>  <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Walker</div>  <div class="given-names">J.J.</div></div>, <div class="name string-name"><div class="surname">Harris</div>  <div class="given-names">J.K.</div></div>, <div class="name string-name"><div class="surname">Gold</div>  <div class="given-names">N.J.</div></div>, <div class="name string-name"><div class="surname">Knight</div>  <div class="given-names">R.</div></div></div> <div class="year">2008</div>, <div class="article-title">Error-correcting barcoded primers for pyrosequencing hundreds of samples in multiplex</div>, <div class="source">Nat. Methods</div> , <div class="volume">5</div>, <div class="fpage">235</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Error-correcting%20barcoded%20primers%20for%20pyrosequencing%20hundreds%20of%20samples%20in%20multiplex&amp;author=M.%20Hamady&amp;author=J.J.%20Walker&amp;author=J.K.%20Harris&amp;author=N.J.%20Gold&amp;author=R.%20Knight&amp;publication_year=2008&amp;journal=Nat.%20Methods&amp;volume=5&amp;pages=235-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.1184" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.1184" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.1184"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18264105" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV010C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV010C14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Shiroguchi</div>  <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Jia</div>  <div class="given-names">T.Z.</div></div>, <div class="name string-name"><div class="surname">Sims</div>  <div class="given-names">P.A.</div></div>, <div class="name string-name"><div class="surname">Xie</div>  <div class="given-names">X.S.</div></div></div> <div class="year">2012</div>, <div class="article-title">Digital RNA sequencing minimizes sequence-dependent bias and amplification noise with optimized single-molecule barcodes</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="volume">109</div>, <div class="fpage">1347</div>–<div class="lpage">52</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Digital%20RNA%20sequencing%20minimizes%20sequence-dependent%20bias%20and%20amplification%20noise%20with%20optimized%20single-molecule%20barcodes&amp;author=K.%20Shiroguchi&amp;author=T.Z.%20Jia&amp;author=P.A.%20Sims&amp;author=X.S.%20Xie&amp;publication_year=2012&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=109&amp;pages=1347-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1118018109" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1118018109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1118018109"> </span></div> </div></div></div></div></div><div content-id="DSV010C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV010C15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Berrozpe</div>  <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Schaeffer</div>  <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Peinado</div>  <div class="given-names">M.A.</div></div>, <div class="name string-name"><div class="surname">Real</div>  <div class="given-names">F.X.</div></div>, <div class="name string-name"><div class="surname">Perucho</div>  <div class="given-names">M.</div></div></div> <div class="year">1994</div>, <div class="article-title">Comparative analysis of mutations in the p53 and K-ras genes in pancreatic cancer</div>, <div class="source">Int. J. Cancer</div> , <div class="volume">58</div>, <div class="fpage">185</div>–<div class="lpage">91</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20analysis%20of%20mutations%20in%20the%20p53%20and%20K-ras%20genes%20in%20pancreatic%20cancer&amp;author=G.%20Berrozpe&amp;author=J.%20Schaeffer&amp;author=M.A.%20Peinado&amp;author=F.X.%20Real&amp;author=M.%20Perucho&amp;publication_year=1994&amp;journal=Int.%20J.%20Cancer&amp;volume=58&amp;pages=185-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1097-0215" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1097-0215" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1097-0215"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8026879" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV010C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV010C16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kukita</div>  <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Uchida</div>  <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Oba</div>  <div class="given-names">S.</div></div> et al.  </div>. <div class="year">2013</div>, <div class="article-title">Quantitative identification of mutant alleles derived from lung cancer in plasma cell-free DNA via anomaly detection using deep sequencing data</div>, <div class="source">PLoS ONE</div> , <div class="volume">8</div>, <div class="fpage">e81468</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Quantitative%20identification%20of%20mutant%20alleles%20derived%20from%20lung%20cancer%20in%20plasma%20cell-free%20DNA%20via%20anomaly%20detection%20using%20deep%20sequencing%20data&amp;author=Y.%20Kukita&amp;author=J.%20Uchida&amp;author=S.%20Oba&amp;publication_year=2013&amp;journal=PLoS%20ONE&amp;volume=8&amp;pages=e81468" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0081468" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0081468" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0081468"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24278442" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV010C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV010C17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div>  <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Durbin</div>  <div class="given-names">R.</div></div></div> <div class="year">2010</div>, <div class="article-title">Fast and accurate long-read alignment with Burrows-Wheeler transform</div>, <div class="source">Bioinformatics</div> , <div class="volume">26</div>, <div class="fpage">589</div>–<div class="lpage">95</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fast%20and%20accurate%20long-read%20alignment%20with%20Burrows-Wheeler%20transform&amp;author=H.%20Li&amp;author=R.%20Durbin&amp;publication_year=2010&amp;journal=Bioinformatics&amp;volume=26&amp;pages=589-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp698" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp698" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp698"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20080505" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV010C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV010C18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div>  <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Handsaker</div>  <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Wysoker</div>  <div class="given-names">A.</div></div> et al.  </div>. <div class="year">2009</div>, <div class="article-title">The Sequence Alignment/Map format and SAMtools</div>, <div class="source">Bioinformatics</div> , <div class="volume">25</div>, <div class="fpage">2078</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Sequence%20Alignment%2FMap%20format%20and%20SAMtools&amp;author=H.%20Li&amp;author=B.%20Handsaker&amp;author=A.%20Wysoker&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=2078-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp352" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp352" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp352"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19505943" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV010C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV010C19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Koboldt</div>  <div class="given-names">D.C.</div></div>, <div class="name string-name"><div class="surname">Zhang</div>  <div class="given-names">Q.</div></div>, <div class="name string-name"><div class="surname">Larson</div>  <div class="given-names">D.E.</div></div> et al.  </div>. <div class="year">2012</div>, <div class="article-title">VarScan 2: somatic mutation and copy number alteration discovery in cancer by exome sequencing</div>, <div class="source">Genome Res.</div> , <div class="volume">22</div>, <div class="fpage">568</div>–<div class="lpage">76</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=VarScan%202%3A%20somatic%20mutation%20and%20copy%20number%20alteration%20discovery%20in%20cancer%20by%20exome%20sequencing&amp;author=D.C.%20Koboldt&amp;author=Q.%20Zhang&amp;author=D.E.%20Larson&amp;publication_year=2012&amp;journal=Genome%20Res.&amp;volume=22&amp;pages=568-76" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.129684.111" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.129684.111" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.129684.111"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22300766" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV010C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV010C20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Schmitt</div>  <div class="given-names">M.W.</div></div>, <div class="name string-name"><div class="surname">Kennedy</div>  <div class="given-names">S.R.</div></div>, <div class="name string-name"><div class="surname">Salk</div>  <div class="given-names">J.J.</div></div>, <div class="name string-name"><div class="surname">Fox</div>  <div class="given-names">E.J.</div></div>, <div class="name string-name"><div class="surname">Hiatt</div>  <div class="given-names">J.B.</div></div>, <div class="name string-name"><div class="surname">Loeb</div>  <div class="given-names">L.A.</div></div></div> <div class="year">2012</div>, <div class="article-title">Detection of ultra-rare mutations by next-generation sequencing</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="volume">109</div>, <div class="fpage">14508</div>–<div class="lpage">13</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Detection%20of%20ultra-rare%20mutations%20by%20next-generation%20sequencing&amp;author=M.W.%20Schmitt&amp;author=S.R.%20Kennedy&amp;author=J.J.%20Salk&amp;author=E.J.%20Fox&amp;author=J.B.%20Hiatt&amp;author=L.A.%20Loeb&amp;publication_year=2012&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=109&amp;pages=14508-13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1208715109" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1208715109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1208715109"> </span></div> </div></div></div></div></div><div content-id="DSV010C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV010C21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Iwao-Koizumi</div>  <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Maekawa</div>  <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Nakamura</div>  <div class="given-names">Y.</div></div> et al.  </div>. <div class="year">2007</div>, <div class="article-title">A novel technique for measuring variations in DNA copy-number: competitive genomic polymerase chain reaction</div>, <div class="source">BMC Genomics</div> , <div class="volume">8</div>, <div class="fpage">206</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20novel%20technique%20for%20measuring%20variations%20in%20DNA%20copy-number%3A%20competitive%20genomic%20polymerase%20chain%20reaction&amp;author=K.%20Iwao-Koizumi&amp;author=K.%20Maekawa&amp;author=Y.%20Nakamura&amp;publication_year=2007&amp;journal=BMC%20Genomics&amp;volume=8&amp;pages=206" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-8-206" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-8-206" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-8-206"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17601344" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV010C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV010C22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kato</div>  <div class="given-names">K.</div></div></div> <div class="year">1997</div>, <div class="article-title">Adaptor-tagged competitive PCR: a novel method for measuring relative gene expression</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">25</div>, <div class="fpage">4694</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Adaptor-tagged%20competitive%20PCR%3A%20a%20novel%20method%20for%20measuring%20relative%20gene%20expression&amp;author=K.%20Kato&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res.&amp;volume=25&amp;pages=4694-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.22.4694" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.22.4694" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.22.4694"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9358186" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV010C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV010C23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kato</div>  <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Yamashita</div>  <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Matoba</div>  <div class="given-names">R.</div></div> et al.  </div>. <div class="year">2005</div>, <div class="article-title">Cancer gene expression database (CGED): a database for gene expression profiling with accompanying clinical information of human cancer tissues</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">33</div>, <div class="fpage">D533</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cancer%20gene%20expression%20database%20(CGED)%3A%20a%20database%20for%20gene%20expression%20profiling%20with%20accompanying%20clinical%20information%20of%20human%20cancer%20tissues&amp;author=K.%20Kato&amp;author=R.%20Yamashita&amp;author=R.%20Matoba&amp;publication_year=2005&amp;journal=Nucleic%20Acids%20Res.&amp;volume=33&amp;pages=D533-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gki117" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgki117" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgki117"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15608255" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV010C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV010C24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kato</div>  <div class="given-names">K.</div></div></div> <div class="year">1995</div>, <div class="article-title">Description of the entire mRNA population by a 3′ end cDNA fragment generated by class IIS restriction enzymes</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">23</div>, <div class="fpage">3685</div>–<div class="lpage">90</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Description%20of%20the%20entire%20mRNA%20population%20by%20a%203%E2%80%B2%20end%20cDNA%20fragment%20generated%20by%20class%20IIS%20restriction%20enzymes&amp;author=K.%20Kato&amp;publication_year=1995&amp;journal=Nucleic%20Acids%20Res.&amp;volume=23&amp;pages=3685-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/23.18.3685" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F23.18.3685" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F23.18.3685"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7478997" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV010C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV010C25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hiatt</div>  <div class="given-names">J.B.</div></div>, <div class="name string-name"><div class="surname">Pritchard</div>  <div class="given-names">C.C.</div></div>, <div class="name string-name"><div class="surname">Salipante</div>  <div class="given-names">S.J.</div></div>, <div class="name string-name"><div class="surname">O'Roak</div>  <div class="given-names">B.J.</div></div>, <div class="name string-name"><div class="surname">Shendure</div>  <div class="given-names">J.</div></div></div> <div class="year">2013</div>, <div class="article-title">Single molecule molecular inversion probes for targeted, high-accuracy detection of low-frequency variation</div>, <div class="source">Genome Res.</div> , <div class="volume">23</div>, <div class="fpage">843</div>–<div class="lpage">54</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Single%20molecule%20molecular%20inversion%20probes%20for%20targeted%2C%20high-accuracy%20detection%20of%20low-frequency%20variation&amp;author=J.B.%20Hiatt&amp;author=C.C.%20Pritchard&amp;author=S.J.%20Salipante&amp;author=B.J.%20O'Roak&amp;author=J.%20Shendure&amp;publication_year=2013&amp;journal=Genome%20Res.&amp;volume=23&amp;pages=843-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.147686.112" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.147686.112" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.147686.112"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23382536" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV010C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV010C26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Quail</div>  <div class="given-names">M.A.</div></div>, <div class="name string-name"><div class="surname">Smith</div>  <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Coupland</div>  <div class="given-names">P.</div></div> et al.  </div>. <div class="year">2012</div>, <div class="article-title">A tale of three next generation sequencing platforms: comparison of Ion Torrent, Pacific Biosciences and Illumina MiSeq sequencers</div>, <div class="source">BMC Genomics</div> , <div class="volume">13</div>, <div class="fpage">341</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20tale%20of%20three%20next%20generation%20sequencing%20platforms%3A%20comparison%20of%20Ion%20Torrent%2C%20Pacific%20Biosciences%20and%20Illumina%20MiSeq%20sequencers&amp;author=M.A.%20Quail&amp;author=M.%20Smith&amp;author=P.%20Coupland&amp;publication_year=2012&amp;journal=BMC%20Genomics&amp;volume=13&amp;pages=341" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-13-341" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-13-341" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-13-341"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22827831" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV010C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV010C27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Junemann</div>  <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Sedlazeck</div>  <div class="given-names">F.J.</div></div>, <div class="name string-name"><div class="surname">Prior</div>  <div class="given-names">K.</div></div> et al.  </div>. <div class="year">2013</div>, <div class="article-title">Updating benchtop sequencing performance comparison</div>, <div class="source">Nat. Biotechnol.</div> , <div class="volume">31</div>, <div class="fpage">294</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Updating%20benchtop%20sequencing%20performance%20comparison&amp;author=S.%20Junemann&amp;author=F.J.%20Sedlazeck&amp;author=K.%20Prior&amp;publication_year=2013&amp;journal=Nat.%20Biotechnol.&amp;volume=31&amp;pages=294-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.2522" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.2522" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.2522"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23563421" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV010C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV010C28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Couraud</div>  <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Vaca-Paniagua</div>  <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Villar</div>  <div class="given-names">S.</div></div> et al.  </div>. <div class="year">2014</div>, <div class="article-title">Noninvasive diagnosis of actionable mutations by deep sequencing of circulating free DNA in lung cancer from never-smokers: a proof-of-concept study from BioCAST/IFCT-1002</div>, <div class="source">Clin. Cancer Res.</div> , <div class="volume">20</div>, <div class="fpage">4613</div>–<div class="lpage">24</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Noninvasive%20diagnosis%20of%20actionable%20mutations%20by%20deep%20sequencing%20of%20circulating%20free%20DNA%20in%20lung%20cancer%20from%20never-smokers%3A%20a%20proof-of-concept%20study%20from%20BioCAST%2FIFCT-1002&amp;author=S.%20Couraud&amp;author=F.%20Vaca-Paniagua&amp;author=S.%20Villar&amp;publication_year=2014&amp;journal=Clin.%20Cancer%20Res.&amp;volume=20&amp;pages=4613-24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1158/1078-0432.CCR-13-3063" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1158%2F1078-0432.CCR-13-3063" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1158%2F1078-0432.CCR-13-3063"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25013125" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV010C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV010C29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Shigemitsu</div>  <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Sekido</div>  <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Usami</div>  <div class="given-names">N.</div></div> et al.  </div>. <div class="year">2001</div>, <div class="article-title">Genetic alteration of the beta-catenin gene (CTNNB1) in human lung cancer and malignant mesothelioma and identification of a new 3p21.3 homozygous deletion</div>, <div class="source">Oncogene</div> , <div class="volume">20</div>, <div class="fpage">4249</div>–<div class="lpage">57</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20alteration%20of%20the%20beta-catenin%20gene%20(CTNNB1)%20in%20human%20lung%20cancer%20and%20malignant%20mesothelioma%20and%20identification%20of%20a%20new%203p21.3%20homozygous%20deletion&amp;author=K.%20Shigemitsu&amp;author=Y.%20Sekido&amp;author=N.%20Usami&amp;publication_year=2001&amp;journal=Oncogene&amp;volume=20&amp;pages=4249-57" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/sj.onc.1204557" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsj.onc.1204557" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsj.onc.1204557"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11464291" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV010C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV010C30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kris</div>  <div class="given-names">M.G.</div></div>, <div class="name string-name"><div class="surname">Johnson</div>  <div class="given-names">B.E.</div></div>, <div class="name string-name"><div class="surname">Berry</div>  <div class="given-names">L.D.</div></div> et al.  </div>. <div class="year">2014</div>, <div class="article-title">Using multiplexed assays of oncogenic drivers in lung cancers to select targeted drugs</div>, <div class="source">JAMA</div> , <div class="volume">311</div>, <div class="fpage">1998</div>–<div class="lpage">2006</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Using%20multiplexed%20assays%20of%20oncogenic%20drivers%20in%20lung%20cancers%20to%20select%20targeted%20drugs&amp;author=M.G.%20Kris&amp;author=B.E.%20Johnson&amp;author=L.D.%20Berry&amp;publication_year=2014&amp;journal=JAMA&amp;volume=311&amp;pages=1998-2006" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1001/jama.2014.3741" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1001%2Fjama.2014.3741" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1001%2Fjama.2014.3741"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24846037" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV010C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV010C31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Deakin</div>  <div class="given-names">C.T.</div></div>, <div class="name string-name"><div class="surname">Deakin</div>  <div class="given-names">J.J.</div></div>, <div class="name string-name"><div class="surname">Ginn</div>  <div class="given-names">S.L.</div></div> et al.  </div>. <div class="year">2014</div>, <div class="article-title">Impact of next-generation sequencing error on analysis of barcoded plasmid libraries of known complexity and sequence</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">42</div>, <div class="fpage">e129</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Impact%20of%20next-generation%20sequencing%20error%20on%20analysis%20of%20barcoded%20plasmid%20libraries%20of%20known%20complexity%20and%20sequence&amp;author=C.T.%20Deakin&amp;author=J.J.%20Deakin&amp;author=S.L.%20Ginn&amp;publication_year=2014&amp;journal=Nucleic%20Acids%20Res.&amp;volume=42&amp;pages=e129" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gku607" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgku607" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgku607"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25013183" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV010C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV010C32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Forshew</div>  <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Murtaza</div>  <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Parkinson</div>  <div class="given-names">C.</div></div> et al.  </div>. <div class="year">2012</div>, <div class="article-title">Noninvasive identification and monitoring of cancer mutations by targeted deep sequencing of plasma DNA</div>, <div class="source">Sci. Transl. Med.</div> , <div class="volume">4</div>, <div class="fpage">136ra168</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Noninvasive%20identification%20and%20monitoring%20of%20cancer%20mutations%20by%20targeted%20deep%20sequencing%20of%20plasma%20DNA&amp;author=T.%20Forshew&amp;author=M.%20Murtaza&amp;author=C.%20Parkinson&amp;publication_year=2012&amp;journal=Sci.%20Transl.%20Med.&amp;volume=4&amp;pages=136ra168" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/scitranslmed.3003726" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscitranslmed.3003726" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscitranslmed.3003726"> </span></div> </div></div></div></div></div><div content-id="DSV010C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV010C33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Narayan</div>  <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Carriero</div>  <div class="given-names">N.J.</div></div>, <div class="name string-name"><div class="surname">Gettinger</div>  <div class="given-names">S.N.</div></div> et al.  </div>. <div class="year">2012</div>, <div class="article-title">Ultrasensitive measurement of hotspot mutations in tumor DNA in blood using error-suppressed multiplexed deep sequencing</div>, <div class="source">Cancer Res.</div> , <div class="volume">72</div>, <div class="fpage">3492</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ultrasensitive%20measurement%20of%20hotspot%20mutations%20in%20tumor%20DNA%20in%20blood%20using%20error-suppressed%20multiplexed%20deep%20sequencing&amp;author=A.%20Narayan&amp;author=N.J.%20Carriero&amp;author=S.N.%20Gettinger&amp;publication_year=2012&amp;journal=Cancer%20Res.&amp;volume=72&amp;pages=3492-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1158/0008-5472.CAN-11-4037" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1158%2F0008-5472.CAN-11-4037" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1158%2F0008-5472.CAN-11-4037"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22581825" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2015. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<a class="link link-uri" href="http://creativecommons.org/licenses/by/4.0/" target="_blank">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=348206&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv010/2/dsv010-dsv010supp.pdf?Expires=2147483647&amp;Signature=eUOIfGsqigNMEJ8qlb0ZsNb6ylkR42LwJzUF4eEQyc-DaOzvNoXIFB0YMaK3HHSFTINjw-t46uv~11OyDc6spwxCac~SFj0PqbgedxC5JarBKQxoax1p~g-dnJ2RwzO89RMMNF-8zWgmx0yHRbTbkOH3sDxnTdO2RZi6vZ-6cnlRY0~1F3~eGstduL8t65gdREao4ezjjp7Fei2MoBLnZ8CrCDDROgFW8rMCTKNRza3qTPAmVsod7-J10LX1zzQYIbyg-9ML85OA2a3myUAlKy5tIVVFFXp3GYY1T846hCc6Xe5GJ1eM0TFtsTdixp5ofy4qa4Qs-McVJ9eIvlZFsw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">1,736</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000362346000003&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">23</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsv010"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="348206"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000362346000003&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f22%2f4%2f269%2f348206">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000362346000003&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (23)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F22%2F4%2F269%2F348206" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/348206'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="High-fidelity target sequencing of individual molecules identified using barcode sequences: de novo detection and absolute quantitation of mutations in plasma cell-free DNA from cancer patients" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsv010");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f22%2f4%2f269%2f348206">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

