

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/25/1/61/4191345","@type":"ScholarlyArticle","name":"Genome re-annotation of the wild strawberry  Fragaria vesca  using extensive Illumina- and SMRT-based RNA-seq datasets","datePublished":"2017-09-23","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/25/1","@type":"PublicationIssue","issueNumber":"1","datePublished":"2018-02-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsx038","keywords":["genome annotation","PacBio-RNAseq","Illumina-RNAseq","strawberry","<italic>Fragaria vesca</italic>"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/25/1/61/4191345","author":[{"name":"Li, Yongping","affiliation":"Key Laboratory of Horticultural Plant Biology (Ministry of Education), College of Horticulture and Forestry Sciences, Huazhong Agricultural University, Wuhan 430070, China","@type":"Person"},{"name":"Wei, Wei","affiliation":"Key Laboratory of Horticultural Plant Biology (Ministry of Education), College of Horticulture and Forestry Sciences, Huazhong Agricultural University, Wuhan 430070, China","@type":"Person"},{"name":"Feng, Jia","affiliation":"Key Laboratory of Horticultural Plant Biology (Ministry of Education), College of Horticulture and Forestry Sciences, Huazhong Agricultural University, Wuhan 430070, China","@type":"Person"},{"name":"Luo, Huifeng","affiliation":"Key Laboratory of Horticultural Plant Biology (Ministry of Education), College of Horticulture and Forestry Sciences, Huazhong Agricultural University, Wuhan 430070, China","@type":"Person"},{"name":"Pi, Mengting","affiliation":"Key Laboratory of Horticultural Plant Biology (Ministry of Education), College of Horticulture and Forestry Sciences, Huazhong Agricultural University, Wuhan 430070, China","@type":"Person"},{"name":"Liu, Zhongchi","affiliation":"Key Laboratory of Horticultural Plant Biology (Ministry of Education), College of Horticulture and Forestry Sciences, Huazhong Agricultural University, Wuhan 430070, China  Department of Cell Biology and Molecular Genetics, University of Maryland, College Park, MD 20742, USA","@type":"Person"},{"name":"Kang, Chunying","affiliation":"Key Laboratory of Horticultural Plant Biology (Ministry of Education), College of Horticulture and Forestry Sciences, Huazhong Agricultural University, Wuhan 430070, China","@type":"Person"}],"description":"Abstract.  The genome of the wild diploid strawberry species Fragaria vesca, an ideal model system of cultivated strawberry (Fragaria Ã— ananassa, octoploid) and","pageStart":"61","pageEnd":"70","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/1/3/m_dnares_25_1cover.png?Expires=2147483647&Signature=SO6YCeydUh5yJs5n-IY9wmy~LoBoAx4P6Fvm-AjuGlXgQjk9YpdSZprn-w-uvSGcgqbsut967tdIfiY3dSqjk-kYgFErDTIMFh~0aK8LiNrwVszpCD29Va86wemvVduCLQhn8b79YN7Bcvi7qFB5E0wUNp-cxt7rNByaE14ePnoDwCgR5G58Pc5wx03IVhK9jft2n0g0lvAsEdQaWr3LG7PN4prc1KFhItNPOxm0fdg7HmXK23iCiP1EUMnFfqTR655CMN~bUB0aDDAZUT18HRZXXdjkjcMUCT~fjmwOlkGzU~71oDZYn-Ji-8-K8iuA-savjer0tQoSi784gQ8L6w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Genome re-annotation of the wild strawberry  Fragaria vesca  using extensive Illumina- and SMRT-based RNA-seq datasets","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/1/3/m_dnares_25_1cover.png?Expires=2147483647&Signature=SO6YCeydUh5yJs5n-IY9wmy~LoBoAx4P6Fvm-AjuGlXgQjk9YpdSZprn-w-uvSGcgqbsut967tdIfiY3dSqjk-kYgFErDTIMFh~0aK8LiNrwVszpCD29Va86wemvVduCLQhn8b79YN7Bcvi7qFB5E0wUNp-cxt7rNByaE14ePnoDwCgR5G58Pc5wx03IVhK9jft2n0g0lvAsEdQaWr3LG7PN4prc1KFhItNPOxm0fdg7HmXK23iCiP1EUMnFfqTR655CMN~bUB0aDDAZUT18HRZXXdjkjcMUCT~fjmwOlkGzU~71oDZYn-Ji-8-K8iuA-savjer0tQoSi784gQ8L6w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Genome re-annotation of the wild strawberry  Fragaria vesca  using extensive Illumina- and SMRT-based RNA-seq datasets" />
    <meta property="og:description" content="Abstract.  The genome of the wild diploid strawberry species Fragaria vesca, an ideal model system of cultivated strawberry (Fragaria &#215; ananassa, octoploid) and" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/25/1/61/4191345" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2017-09-23" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Li, Yongping" /><meta name="citation_author_institution" content="Key Laboratory of Horticultural Plant Biology (Ministry of Education), College of Horticulture and Forestry Sciences, Huazhong Agricultural University, Wuhan 430070, China" /><meta name="citation_author" content="Wei, Wei" /><meta name="citation_author_institution" content="Key Laboratory of Horticultural Plant Biology (Ministry of Education), College of Horticulture and Forestry Sciences, Huazhong Agricultural University, Wuhan 430070, China" /><meta name="citation_author" content="Feng, Jia" /><meta name="citation_author_institution" content="Key Laboratory of Horticultural Plant Biology (Ministry of Education), College of Horticulture and Forestry Sciences, Huazhong Agricultural University, Wuhan 430070, China" /><meta name="citation_author" content="Luo, Huifeng" /><meta name="citation_author_institution" content="Key Laboratory of Horticultural Plant Biology (Ministry of Education), College of Horticulture and Forestry Sciences, Huazhong Agricultural University, Wuhan 430070, China" /><meta name="citation_author" content="Pi, Mengting" /><meta name="citation_author_institution" content="Key Laboratory of Horticultural Plant Biology (Ministry of Education), College of Horticulture and Forestry Sciences, Huazhong Agricultural University, Wuhan 430070, China" /><meta name="citation_author" content="Liu, Zhongchi" /><meta name="citation_author_institution" content="Key Laboratory of Horticultural Plant Biology (Ministry of Education), College of Horticulture and Forestry Sciences, Huazhong Agricultural University, Wuhan 430070, China" /><meta name="citation_author_institution" content="Department of Cell Biology and Molecular Genetics, University of Maryland, College Park, MD 20742, USA" /><meta name="citation_author" content="Kang, Chunying" /><meta name="citation_author_institution" content="Key Laboratory of Horticultural Plant Biology (Ministry of Education), College of Horticulture and Forestry Sciences, Huazhong Agricultural University, Wuhan 430070, China" /><meta name="citation_title" content="Genome re-annotation of the wild strawberry Fragaria vesca using extensive Illumina- and SMRT-based RNA-seq datasets" /><meta name="citation_firstpage" content="61" /><meta name="citation_lastpage" content="70" /><meta name="citation_doi" content="10.1093/dnares/dsx038" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="25" /><meta name="citation_issue" content="1" /><meta name="citation_publication_date" content="2018/02/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Fragaria: a genus with deep historical roots and ripe for evolutionary and ecological insights;  citation_journal_title=Am. J. Bot;  citation_year=2014;  citation_volume=101;  citation_pages=1686-99; " /><meta name="citation_reference" content="citation_title=The genome of woodland strawberry (Fragaria vesca);  citation_journal_title=Nat. Genet;  citation_year=2011;  citation_volume=43;  citation_pages=109-16; " /><meta name="citation_reference" content="citation_title=Genome-scale transcriptomic insights into early-stage fruit development in woodland strawberry Fragaria vesca;  citation_journal_title=Plant Cell;  citation_year=2013;  citation_volume=25;  citation_pages=1960-78; " /><meta name="citation_reference" content="citation_title=Floral transcriptomes in woodland strawberry uncover developing receptacle and anther gene networks;  citation_journal_title=Plant Physiol;  citation_year=2014;  citation_volume=165;  citation_pages=1062-75; " /><meta name="citation_reference" content="citation_title=Novel and recently evolved miRNA clusters regulate expansive F-box gene networks through phasiRNAs in wild diploid strawberry;  citation_journal_title=Plant Physiol;  citation_year=2015;  citation_volume=169;  citation_pages=594-610; " /><meta name="citation_reference" content="citation_title=Global identification and analysis of long non-coding RNAs in diploid strawberry Fragaria vesca during flower and fruit development;  citation_journal_title=BMC Genomics;  citation_year=2015;  citation_volume=16;  citation_pages=815" /><meta name="citation_reference" content="citation_title=Global identification of alternative splicing via comparative analysis of SMRT- and Illumina-based RNA-seq in strawberry;  citation_journal_title=Plant J;  citation_year=2017;  citation_volume=90;  citation_pages=164-76; " /><meta name="citation_reference" content="citation_title=Re-annotation of the woodland strawberry (Fragaria vesca) genome;  citation_journal_title=BMC Genomics;  citation_year=2015;  citation_volume=16;  citation_pages=29" /><meta name="citation_reference" content="citation_title=Evolutionary origins and dynamics of octoploid strawberry subgenomes revealed by dense targeted capture linkage maps;  citation_journal_title=Genome Biol. Evol;  citation_year=2014;  citation_volume=6;  citation_pages=3295-313; " /><meta name="citation_reference" content="citation_title=Genome-scale DNA variant analysis and functional validation of a SNP underlying yellow fruit color in wild strawberry;  citation_journal_title=Sci. Rep;  citation_year=2016;  citation_volume=6;  citation_pages=29017" /><meta name="citation_reference" content="citation_title=Exploiting single-molecule transcript sequencing for eukaryotic gene prediction;  citation_journal_title=Genome Biol;  citation_year=2015;  citation_volume=16;  citation_pages=184" /><meta name="citation_reference" content="citation_title=Single-molecule real-time transcript sequencing facilitates common wheat genome annotation and grain transcriptome research;  citation_journal_title=BMC Genomics;  citation_year=2015;  citation_volume=16;  citation_pages=1039" /><meta name="citation_reference" content="citation_title=uORF-mediated translation allows engineered plant disease resistance without fitness costs;  citation_journal_title=Nature;  citation_year=2017;  citation_volume=545;  citation_pages=491-4; " /><meta name="citation_reference" content="citation_title=Improving the Arabidopsis genome annotation using maximal transcript alignment assemblies;  citation_journal_title=Nucleic Acids Res;  citation_year=2003;  citation_volume=31;  citation_pages=5654-66; " /><meta name="citation_reference" content="citation_title=MAKER2: an annotation pipeline and genome-database management tool for second-generation genome projects;  citation_journal_title=BMC Bioinformatics;  citation_year=2011;  citation_volume=12, 491; " /><meta name="citation_reference" content="citation_title=Using native and syntenically mapped cDNA alignments to improve de novo gene finding;  citation_journal_title=Bioinformatics;  citation_year=2008;  citation_volume=24;  citation_pages=637-44; " /><meta name="citation_reference" content="citation_title=Gene prediction with a hidden Markov model and a new intron submodel;  citation_journal_title=Bioinformatics;  citation_year=2003;  citation_volume=19;  citation_pages=Ii215-25; " /><meta name="citation_reference" content="citation_title=Reprogramming of strawberry (Fragaria vesca) root transcriptome in response to phytophthora cactorum;  citation_journal_title=PLoS One;  citation_year=2016;  citation_volume=11;  citation_pages=e0161078" /><meta name="citation_reference" content="citation_title=LoRDEC: accurate and efficient long read error correction;  citation_journal_title=Bioinformatics;  citation_year=2014;  citation_volume=30;  citation_pages=3506-14; " /><meta name="citation_reference" content="citation_title=GMAP: a genomic mapping and alignment program for mRNA and EST sequences;  citation_journal_title=Bioinformatics;  citation_year=2005;  citation_volume=21;  citation_pages=1859-75; " /><meta name="citation_reference" content="citation_title=TopHat2: accurate alignment of transcriptomes in the presence of insertions, deletions and gene fusions;  citation_journal_title=Genome Biol;  citation_year=2013;  citation_volume=14;  citation_pages=R36" /><meta name="citation_reference" content="citation_title=The Sequence Alignment/Map format and SAMtools;  citation_journal_title=Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=2078-9; " /><meta name="citation_reference" content="citation_title=StringTie enables improved reconstruction of a transcriptome from RNA-seq reads;  citation_journal_title=Nat. Biotechnol;  citation_year=2015;  citation_volume=33;  citation_pages=290" /><meta name="citation_reference" content="citation_title=Full-length transcriptome assembly from RNA-Seq data without a reference genome;  citation_journal_title=Nat. Biotechnol;  citation_year=2011;  citation_volume=29;  citation_pages=644-52; " /><meta name="citation_reference" content="citation_title=Gene prediction in eukaryotes with a generalized hidden Markov model that uses hints from external sources;  citation_journal_title=BMC Bioinformatics;  citation_year=2006;  citation_volume=7;  citation_pages=62" /><meta name="citation_reference" content="citation_title=Automated generation of heuristics for biological sequence comparison;  citation_journal_title=BMC Bioinformatics;  citation_year=2005;  citation_volume=6;  citation_pages=31" /><meta name="citation_reference" content="citation_title=Automated eukaryotic gene structure annotation using EVidenceModeler and the program to assemble spliced alignments;  citation_journal_title=Genome Biol;  citation_year=2008;  citation_volume=9;  citation_pages=R7" /><meta name="citation_reference" content="citation_title=Web Apollo: a web-based genomic annotation editing platform;  citation_journal_title=Genome Biol;  citation_year=2013;  citation_volume=14;  citation_pages=R93" /><meta name="citation_reference" content="citation_title=The generic genome browser: a building block for a model organism system database;  citation_journal_title=Genome Res;  citation_year=2002;  citation_volume=12;  citation_pages=1599-610; " /><meta name="citation_reference" content="citation_title=ASTALAVISTA: dynamic and flexible analysis of alternative splicing events in custom gene datasets;  citation_journal_title=Nucleic Acids Res;  citation_year=2007;  citation_volume=35;  citation_pages=W297-9; " /><meta name="citation_reference" content="citation_title=Basic local alignment search tool;  citation_journal_title=J. Mol. Biol;  citation_year=1990;  citation_volume=215;  citation_pages=403-10; " /><meta name="citation_reference" content="citation_title=CPC: assess the protein-coding potential of transcripts using sequence features and support vector machine;  citation_journal_title=Nucleic Acids Res;  citation_year=2007;  citation_volume=35;  citation_pages=W345-9; " /><meta name="citation_reference" content="citation_title=FEELnc: a tool for long non-coding RNA annotation and its application to the dog transcriptome;  citation_journal_title=Nucleic Acids Res;  citation_year=2017;  citation_volume=45;  citation_pages=e57" /><meta name="citation_reference" content="citation_title=Cutadapt removes adapter sequences from high-throughput sequencing reads;  citation_journal_title=EMBnet J;  citation_year=2011;  citation_volume=17;  citation_pages=10-2; " /><meta name="citation_reference" content="citation_title=Aligning short sequencing reads with Bowtie;  citation_journal_title=Curr. Protoc. Bioinformatics;  citation_year=2010; " /><meta name="citation_reference" content="citation_title=ShortStack: comprehensive annotation and quantification of small RNA genes;  citation_journal_title=RNA;  citation_year=2013;  citation_volume=19;  citation_pages=740-51; " /><meta name="citation_reference" content="citation_title=Assessment of transcript reconstruction methods for RNA-seq;  citation_journal_title=Nat. Methods;  citation_year=2013;  citation_volume=10;  citation_pages=1177-84; " /><meta name="citation_reference" content="citation_title=Integrative genomics viewer;  citation_journal_title=Nat. Biotechnol;  citation_year=2011;  citation_volume=29;  citation_pages=24-26; " /><meta name="citation_reference" content="citation_title=BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs;  citation_journal_title=Bioinformatics;  citation_year=2015;  citation_volume=31;  citation_pages=3210-2; " /><meta name="citation_reference" content="citation_title=InterProScan 5: genome-scale protein function classification;  citation_journal_title=Bioinformatics;  citation_year=2014;  citation_volume=30;  citation_pages=1236-40; " /><meta name="citation_reference" content="citation_title=Blast2GO: a universal tool for annotation, visualization and analysis in functional genomics research;  citation_journal_title=Bioinformatics;  citation_year=2005;  citation_volume=21;  citation_pages=3674-6; " /><meta name="citation_reference" content="citation_title=iTAK: a program for genome-wide prediction and classification of plant transcription factors, transcriptional regulators, and protein kinases;  citation_journal_title=Mol Plant;  citation_year=2016;  citation_volume=9;  citation_pages=1667-70; " /><meta name="citation_reference" content="citation_title=RGAugury: a pipeline for genome-wide prediction of resistance gene analogs (RGAs) in plants;  citation_journal_title=BMC Genomics;  citation_year=2016;  citation_volume=17;  citation_pages=852" /><meta name="citation_reference" content="citation_title=The Fragaria vesca homolog of SUPPRESSOR OF OVEREXPRESSION OF CONSTANS1 represses flowering and promotes vegetative growth;  citation_journal_title=Plant Cell;  citation_year=2013;  citation_volume=25;  citation_pages=3296-310; " /><meta name="citation_reference" content="citation_title=The TFL1 homologue KSN is a regulator of continuous flowering in rose and strawberry;  citation_journal_title=Plant J;  citation_year=2012;  citation_volume=69;  citation_pages=116-25; " /><meta name="citation_reference" content="citation_title=Mutation in TERMINAL FLOWER1 reverses the photoperiodic requirement for flowering in the wild strawberry Fragaria vesca;  citation_journal_title=Plant Physiol;  citation_year=2012;  citation_volume=159;  citation_pages=1043-54; " /><meta name="citation_reference" content="citation_title=Mechanisms and regulation of alternative pre-mRNA splicing;  citation_journal_title=Annu. Rev. Biochem;  citation_year=2015;  citation_volume=84;  citation_pages=291-323; " /><meta name="citation_reference" content="citation_title=Complexity of the alternative splicing landscape in plants;  citation_journal_title=Plant Cell;  citation_year=2013;  citation_volume=25;  citation_pages=3657-83; " /><meta name="citation_reference" content="citation_title=Araport11: a complete reannotation of the Arabidopsis thaliana reference genome;  citation_journal_title=Plant J;  citation_year=2017;  citation_volume=89;  citation_pages=789-804; " /><meta name="citation_reference" content="citation_title=Genome and transcriptome of the porcine whipworm Trichuris suis;  citation_journal_title=Nat. Genet;  citation_year=2014;  citation_volume=46;  citation_pages=701-6; " /><meta name="citation_reference" content="citation_title=Comprehensive annotation of Physcomitrella patens small RNA loci reveals that the heterochromatic short interfering RNA pathway is largely conserved in land plants;  citation_journal_title=Plant Cell;  citation_year=2015;  citation_volume=27;  citation_pages=2148-62; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/25/1/61/24151556/dsx038.pdf" /><meta name="description" content="Abstract.  The genome of the wild diploid strawberry species Fragaria vesca, an ideal model system of cultivated strawberry (Fragaria Ã— ananassa, octoploid) and" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/25/1/61/4191345" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/25/1/61/4191345" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/25/1/61/4191345");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsx038';
    </script>
<title>
	Genome re-annotation of the wild strawberry Fragaria vesca using extensive Illumina- and SMRT-based RNA-seq datasets | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./4191345" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BZB9PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJMaSwgWW9uZ3BpbmciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLZXkgTGFib3JhdG9yeSBvZiBIb3J0aWN1bHR1cmFsIFBsYW50IEJpb2xvZ3kgKE1pbmlzdHJ5IG9mIEVkdWNhdGlvbiksIENvbGxlZ2Ugb2YgSG9ydGljdWx0dXJlIGFuZCBGb3Jlc3RyeSBTY2llbmNlcywgSHVhemhvbmcgQWdyaWN1bHR1cmFsIFVuaXZlcnNpdHksIFd1aGFuIDQzMDA3MCwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJXZWksIFdlaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IktleSBMYWJvcmF0b3J5IG9mIEhvcnRpY3VsdHVyYWwgUGxhbnQgQmlvbG9neSAoTWluaXN0cnkgb2YgRWR1Y2F0aW9uKSwgQ29sbGVnZSBvZiBIb3J0aWN1bHR1cmUgYW5kIEZvcmVzdHJ5IFNjaWVuY2VzLCBIdWF6aG9uZyBBZ3JpY3VsdHVyYWwgVW5pdmVyc2l0eSwgV3VoYW4gNDMwMDcwLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkZlbmcsIEppYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IktleSBMYWJvcmF0b3J5IG9mIEhvcnRpY3VsdHVyYWwgUGxhbnQgQmlvbG9neSAoTWluaXN0cnkgb2YgRWR1Y2F0aW9uKSwgQ29sbGVnZSBvZiBIb3J0aWN1bHR1cmUgYW5kIEZvcmVzdHJ5IFNjaWVuY2VzLCBIdWF6aG9uZyBBZ3JpY3VsdHVyYWwgVW5pdmVyc2l0eSwgV3VoYW4gNDMwMDcwLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ikx1bywgSHVpZmVuZyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IktleSBMYWJvcmF0b3J5IG9mIEhvcnRpY3VsdHVyYWwgUGxhbnQgQmlvbG9neSAoTWluaXN0cnkgb2YgRWR1Y2F0aW9uKSwgQ29sbGVnZSBvZiBIb3J0aWN1bHR1cmUgYW5kIEZvcmVzdHJ5IFNjaWVuY2VzLCBIdWF6aG9uZyBBZ3JpY3VsdHVyYWwgVW5pdmVyc2l0eSwgV3VoYW4gNDMwMDcwLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlBpLCBNZW5ndGluZyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IktleSBMYWJvcmF0b3J5IG9mIEhvcnRpY3VsdHVyYWwgUGxhbnQgQmlvbG9neSAoTWluaXN0cnkgb2YgRWR1Y2F0aW9uKSwgQ29sbGVnZSBvZiBIb3J0aWN1bHR1cmUgYW5kIEZvcmVzdHJ5IFNjaWVuY2VzLCBIdWF6aG9uZyBBZ3JpY3VsdHVyYWwgVW5pdmVyc2l0eSwgV3VoYW4gNDMwMDcwLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkxpdSwgWmhvbmdjaGkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLZXkgTGFib3JhdG9yeSBvZiBIb3J0aWN1bHR1cmFsIFBsYW50IEJpb2xvZ3kgKE1pbmlzdHJ5IG9mIEVkdWNhdGlvbiksIENvbGxlZ2Ugb2YgSG9ydGljdWx0dXJlIGFuZCBGb3Jlc3RyeSBTY2llbmNlcywgSHVhemhvbmcgQWdyaWN1bHR1cmFsIFVuaXZlcnNpdHksIFd1aGFuIDQzMDA3MCwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIENlbGwgQmlvbG9neSBhbmQgTW9sZWN1bGFyIEdlbmV0aWNzLCBVbml2ZXJzaXR5IG9mIE1hcnlsYW5kLCBDb2xsZWdlIFBhcmssIE1EIDIwNzQyLCBVU0EiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLYW5nLCBDaHVueWluZyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IktleSBMYWJvcmF0b3J5IG9mIEhvcnRpY3VsdHVyYWwgUGxhbnQgQmlvbG9neSAoTWluaXN0cnkgb2YgRWR1Y2F0aW9uKSwgQ29sbGVnZSBvZiBIb3J0aWN1bHR1cmUgYW5kIEZvcmVzdHJ5IFNjaWVuY2VzLCBIdWF6aG9uZyBBZ3JpY3VsdHVyYWwgVW5pdmVyc2l0eSwgV3VoYW4gNDMwMDcwLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iR2Vub21lIHJlLWFubm90YXRpb24gb2YgdGhlIHdpbGQgc3RyYXdiZXJyeSBGcmFnYXJpYSB2ZXNjYSB1c2luZyBleHRlbnNpdmUgSWxsdW1pbmEtIGFuZCBTTVJULWJhc2VkIFJOQS1zZXEgZGF0YXNldHMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSI2MSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iNzAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3gwMzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMjUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxOC8wMi8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZyYWdhcmlhOiBhIGdlbnVzIHdpdGggZGVlcCBoaXN0b3JpY2FsIHJvb3RzIGFuZCByaXBlIGZvciBldm9sdXRpb25hcnkgYW5kIGVjb2xvZ2ljYWwgaW5zaWdodHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFtLiBKLiBCb3Q7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MTAxOyAgY2l0YXRpb25fcGFnZXM9MTY4Ni05OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGdlbm9tZSBvZiB3b29kbGFuZCBzdHJhd2JlcnJ5IChGcmFnYXJpYSB2ZXNjYSk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NDM7ICBjaXRhdGlvbl9wYWdlcz0xMDktMTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS1zY2FsZSB0cmFuc2NyaXB0b21pYyBpbnNpZ2h0cyBpbnRvIGVhcmx5LXN0YWdlIGZydWl0IGRldmVsb3BtZW50IGluIHdvb2RsYW5kIHN0cmF3YmVycnkgRnJhZ2FyaWEgdmVzY2E7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0xOTYwLTc4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GbG9yYWwgdHJhbnNjcmlwdG9tZXMgaW4gd29vZGxhbmQgc3RyYXdiZXJyeSB1bmNvdmVyIGRldmVsb3BpbmcgcmVjZXB0YWNsZSBhbmQgYW50aGVyIGdlbmUgbmV0d29ya3M7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IFBoeXNpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MTY1OyAgY2l0YXRpb25fcGFnZXM9MTA2Mi03NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Tm92ZWwgYW5kIHJlY2VudGx5IGV2b2x2ZWQgbWlSTkEgY2x1c3RlcnMgcmVndWxhdGUgZXhwYW5zaXZlIEYtYm94IGdlbmUgbmV0d29ya3MgdGhyb3VnaCBwaGFzaVJOQXMgaW4gd2lsZCBkaXBsb2lkIHN0cmF3YmVycnk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IFBoeXNpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MTY5OyAgY2l0YXRpb25fcGFnZXM9NTk0LTYxMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2xvYmFsIGlkZW50aWZpY2F0aW9uIGFuZCBhbmFseXNpcyBvZiBsb25nIG5vbi1jb2RpbmcgUk5BcyBpbiBkaXBsb2lkIHN0cmF3YmVycnkgRnJhZ2FyaWEgdmVzY2EgZHVyaW5nIGZsb3dlciBhbmQgZnJ1aXQgZGV2ZWxvcG1lbnQ7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTgxNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdsb2JhbCBpZGVudGlmaWNhdGlvbiBvZiBhbHRlcm5hdGl2ZSBzcGxpY2luZyB2aWEgY29tcGFyYXRpdmUgYW5hbHlzaXMgb2YgU01SVC0gYW5kIElsbHVtaW5hLWJhc2VkIFJOQS1zZXEgaW4gc3RyYXdiZXJyeTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgSjsgIGNpdGF0aW9uX3llYXI9MjAxNzsgIGNpdGF0aW9uX3ZvbHVtZT05MDsgIGNpdGF0aW9uX3BhZ2VzPTE2NC03NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmUtYW5ub3RhdGlvbiBvZiB0aGUgd29vZGxhbmQgc3RyYXdiZXJyeSAoRnJhZ2FyaWEgdmVzY2EpIGdlbm9tZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9MjkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Fdm9sdXRpb25hcnkgb3JpZ2lucyBhbmQgZHluYW1pY3Mgb2Ygb2N0b3Bsb2lkIHN0cmF3YmVycnkgc3ViZ2Vub21lcyByZXZlYWxlZCBieSBkZW5zZSB0YXJnZXRlZCBjYXB0dXJlIGxpbmthZ2UgbWFwczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIEJpb2wuIEV2b2w7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTMyOTUtMzEzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtc2NhbGUgRE5BIHZhcmlhbnQgYW5hbHlzaXMgYW5kIGZ1bmN0aW9uYWwgdmFsaWRhdGlvbiBvZiBhIFNOUCB1bmRlcmx5aW5nIHllbGxvdyBmcnVpdCBjb2xvciBpbiB3aWxkIHN0cmF3YmVycnk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaS4gUmVwOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz0yOTAxNyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV4cGxvaXRpbmcgc2luZ2xlLW1vbGVjdWxlIHRyYW5zY3JpcHQgc2VxdWVuY2luZyBmb3IgZXVrYXJ5b3RpYyBnZW5lIHByZWRpY3Rpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9MTg0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2luZ2xlLW1vbGVjdWxlIHJlYWwtdGltZSB0cmFuc2NyaXB0IHNlcXVlbmNpbmcgZmFjaWxpdGF0ZXMgY29tbW9uIHdoZWF0IGdlbm9tZSBhbm5vdGF0aW9uIGFuZCBncmFpbiB0cmFuc2NyaXB0b21lIHJlc2VhcmNoOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz0xMDM5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9dU9SRi1tZWRpYXRlZCB0cmFuc2xhdGlvbiBhbGxvd3MgZW5naW5lZXJlZCBwbGFudCBkaXNlYXNlIHJlc2lzdGFuY2Ugd2l0aG91dCBmaXRuZXNzIGNvc3RzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMTc7ICBjaXRhdGlvbl92b2x1bWU9NTQ1OyAgY2l0YXRpb25fcGFnZXM9NDkxLTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUltcHJvdmluZyB0aGUgQXJhYmlkb3BzaXMgZ2Vub21lIGFubm90YXRpb24gdXNpbmcgbWF4aW1hbCB0cmFuc2NyaXB0IGFsaWdubWVudCBhc3NlbWJsaWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0zMTsgIGNpdGF0aW9uX3BhZ2VzPTU2NTQtNjY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1BS0VSMjogYW4gYW5ub3RhdGlvbiBwaXBlbGluZSBhbmQgZ2Vub21lLWRhdGFiYXNlIG1hbmFnZW1lbnQgdG9vbCBmb3Igc2Vjb25kLWdlbmVyYXRpb24gZ2Vub21lIHByb2plY3RzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTIsIDQ5MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VXNpbmcgbmF0aXZlIGFuZCBzeW50ZW5pY2FsbHkgbWFwcGVkIGNETkEgYWxpZ25tZW50cyB0byBpbXByb3ZlIGRlIG5vdm8gZ2VuZSBmaW5kaW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTYzNy00NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZSBwcmVkaWN0aW9uIHdpdGggYSBoaWRkZW4gTWFya292IG1vZGVsIGFuZCBhIG5ldyBpbnRyb24gc3VibW9kZWw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTE5OyAgY2l0YXRpb25fcGFnZXM9SWkyMTUtMjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlcHJvZ3JhbW1pbmcgb2Ygc3RyYXdiZXJyeSAoRnJhZ2FyaWEgdmVzY2EpIHJvb3QgdHJhbnNjcmlwdG9tZSBpbiByZXNwb25zZSB0byBwaHl0b3BodGhvcmEgY2FjdG9ydW07ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgT25lOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9ZTAxNjEwNzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Mb1JERUM6IGFjY3VyYXRlIGFuZCBlZmZpY2llbnQgbG9uZyByZWFkIGVycm9yIGNvcnJlY3Rpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTMwOyAgY2l0YXRpb25fcGFnZXM9MzUwNi0xNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R01BUDogYSBnZW5vbWljIG1hcHBpbmcgYW5kIGFsaWdubWVudCBwcm9ncmFtIGZvciBtUk5BIGFuZCBFU1Qgc2VxdWVuY2VzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTE4NTktNzU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRvcEhhdDI6IGFjY3VyYXRlIGFsaWdubWVudCBvZiB0cmFuc2NyaXB0b21lcyBpbiB0aGUgcHJlc2VuY2Ugb2YgaW5zZXJ0aW9ucywgZGVsZXRpb25zIGFuZCBnZW5lIGZ1c2lvbnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9UjM2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIFNlcXVlbmNlIEFsaWdubWVudC9NYXAgZm9ybWF0IGFuZCBTQU10b29sczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0yMDc4LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN0cmluZ1RpZSBlbmFibGVzIGltcHJvdmVkIHJlY29uc3RydWN0aW9uIG9mIGEgdHJhbnNjcmlwdG9tZSBmcm9tIFJOQS1zZXEgcmVhZHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gQmlvdGVjaG5vbDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0zMzsgIGNpdGF0aW9uX3BhZ2VzPTI5MCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZ1bGwtbGVuZ3RoIHRyYW5zY3JpcHRvbWUgYXNzZW1ibHkgZnJvbSBSTkEtU2VxIGRhdGEgd2l0aG91dCBhIHJlZmVyZW5jZSBnZW5vbWU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gQmlvdGVjaG5vbDsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0yOTsgIGNpdGF0aW9uX3BhZ2VzPTY0NC01MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZSBwcmVkaWN0aW9uIGluIGV1a2FyeW90ZXMgd2l0aCBhIGdlbmVyYWxpemVkIGhpZGRlbiBNYXJrb3YgbW9kZWwgdGhhdCB1c2VzIGhpbnRzIGZyb20gZXh0ZXJuYWwgc291cmNlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz02MiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUF1dG9tYXRlZCBnZW5lcmF0aW9uIG9mIGhldXJpc3RpY3MgZm9yIGJpb2xvZ2ljYWwgc2VxdWVuY2UgY29tcGFyaXNvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz0zMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUF1dG9tYXRlZCBldWthcnlvdGljIGdlbmUgc3RydWN0dXJlIGFubm90YXRpb24gdXNpbmcgRVZpZGVuY2VNb2RlbGVyIGFuZCB0aGUgcHJvZ3JhbSB0byBhc3NlbWJsZSBzcGxpY2VkIGFsaWdubWVudHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz1SNyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVdlYiBBcG9sbG86IGEgd2ViLWJhc2VkIGdlbm9taWMgYW5ub3RhdGlvbiBlZGl0aW5nIHBsYXRmb3JtOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPVI5MyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBnZW5lcmljIGdlbm9tZSBicm93c2VyOiBhIGJ1aWxkaW5nIGJsb2NrIGZvciBhIG1vZGVsIG9yZ2FuaXNtIHN5c3RlbSBkYXRhYmFzZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTE1OTktNjEwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BU1RBTEFWSVNUQTogZHluYW1pYyBhbmQgZmxleGlibGUgYW5hbHlzaXMgb2YgYWx0ZXJuYXRpdmUgc3BsaWNpbmcgZXZlbnRzIGluIGN1c3RvbSBnZW5lIGRhdGFzZXRzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0zNTsgIGNpdGF0aW9uX3BhZ2VzPVcyOTctOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QmFzaWMgbG9jYWwgYWxpZ25tZW50IHNlYXJjaCB0b29sOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBNb2wuIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTE5OTA7ICBjaXRhdGlvbl92b2x1bWU9MjE1OyAgY2l0YXRpb25fcGFnZXM9NDAzLTEwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DUEM6IGFzc2VzcyB0aGUgcHJvdGVpbi1jb2RpbmcgcG90ZW50aWFsIG9mIHRyYW5zY3JpcHRzIHVzaW5nIHNlcXVlbmNlIGZlYXR1cmVzIGFuZCBzdXBwb3J0IHZlY3RvciBtYWNoaW5lOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0zNTsgIGNpdGF0aW9uX3BhZ2VzPVczNDUtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RkVFTG5jOiBhIHRvb2wgZm9yIGxvbmcgbm9uLWNvZGluZyBSTkEgYW5ub3RhdGlvbiBhbmQgaXRzIGFwcGxpY2F0aW9uIHRvIHRoZSBkb2cgdHJhbnNjcmlwdG9tZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTc7ICBjaXRhdGlvbl92b2x1bWU9NDU7ICBjaXRhdGlvbl9wYWdlcz1lNTciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DdXRhZGFwdCByZW1vdmVzIGFkYXB0ZXIgc2VxdWVuY2VzIGZyb20gaGlnaC10aHJvdWdocHV0IHNlcXVlbmNpbmcgcmVhZHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUVNQm5ldCBKOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTE3OyAgY2l0YXRpb25fcGFnZXM9MTAtMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWxpZ25pbmcgc2hvcnQgc2VxdWVuY2luZyByZWFkcyB3aXRoIEJvd3RpZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q3Vyci4gUHJvdG9jLiBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2hvcnRTdGFjazogY29tcHJlaGVuc2l2ZSBhbm5vdGF0aW9uIGFuZCBxdWFudGlmaWNhdGlvbiBvZiBzbWFsbCBSTkEgZ2VuZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVJOQTsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xOTsgIGNpdGF0aW9uX3BhZ2VzPTc0MC01MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXNzZXNzbWVudCBvZiB0cmFuc2NyaXB0IHJlY29uc3RydWN0aW9uIG1ldGhvZHMgZm9yIFJOQS1zZXE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gTWV0aG9kczsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTExNzctODQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUludGVncmF0aXZlIGdlbm9taWNzIHZpZXdlcjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBCaW90ZWNobm9sOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTI5OyAgY2l0YXRpb25fcGFnZXM9MjQtMjY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJVU0NPOiBhc3Nlc3NpbmcgZ2Vub21lIGFzc2VtYmx5IGFuZCBhbm5vdGF0aW9uIGNvbXBsZXRlbmVzcyB3aXRoIHNpbmdsZS1jb3B5IG9ydGhvbG9nczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MzE7ICBjaXRhdGlvbl9wYWdlcz0zMjEwLTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUludGVyUHJvU2NhbiA1OiBnZW5vbWUtc2NhbGUgcHJvdGVpbiBmdW5jdGlvbiBjbGFzc2lmaWNhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MzA7ICBjaXRhdGlvbl9wYWdlcz0xMjM2LTQwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CbGFzdDJHTzogYSB1bml2ZXJzYWwgdG9vbCBmb3IgYW5ub3RhdGlvbiwgdmlzdWFsaXphdGlvbiBhbmQgYW5hbHlzaXMgaW4gZnVuY3Rpb25hbCBnZW5vbWljcyByZXNlYXJjaDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MjE7ICBjaXRhdGlvbl9wYWdlcz0zNjc0LTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPWlUQUs6IGEgcHJvZ3JhbSBmb3IgZ2Vub21lLXdpZGUgcHJlZGljdGlvbiBhbmQgY2xhc3NpZmljYXRpb24gb2YgcGxhbnQgdHJhbnNjcmlwdGlvbiBmYWN0b3JzLCB0cmFuc2NyaXB0aW9uYWwgcmVndWxhdG9ycywgYW5kIHByb3RlaW4ga2luYXNlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sIFBsYW50OyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz0xNjY3LTcwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SR0F1Z3VyeTogYSBwaXBlbGluZSBmb3IgZ2Vub21lLXdpZGUgcHJlZGljdGlvbiBvZiByZXNpc3RhbmNlIGdlbmUgYW5hbG9ncyAoUkdBcykgaW4gcGxhbnRzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9MTc7ICBjaXRhdGlvbl9wYWdlcz04NTIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgRnJhZ2FyaWEgdmVzY2EgaG9tb2xvZyBvZiBTVVBQUkVTU09SIE9GIE9WRVJFWFBSRVNTSU9OIE9GIENPTlNUQU5TMSByZXByZXNzZXMgZmxvd2VyaW5nIGFuZCBwcm9tb3RlcyB2ZWdldGF0aXZlIGdyb3d0aDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTMyOTYtMzEwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgVEZMMSBob21vbG9ndWUgS1NOIGlzIGEgcmVndWxhdG9yIG9mIGNvbnRpbnVvdXMgZmxvd2VyaW5nIGluIHJvc2UgYW5kIHN0cmF3YmVycnk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IEo7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9Njk7ICBjaXRhdGlvbl9wYWdlcz0xMTYtMjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU11dGF0aW9uIGluIFRFUk1JTkFMIEZMT1dFUjEgcmV2ZXJzZXMgdGhlIHBob3RvcGVyaW9kaWMgcmVxdWlyZW1lbnQgZm9yIGZsb3dlcmluZyBpbiB0aGUgd2lsZCBzdHJhd2JlcnJ5IEZyYWdhcmlhIHZlc2NhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBQaHlzaW9sOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTE1OTsgIGNpdGF0aW9uX3BhZ2VzPTEwNDMtNTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1lY2hhbmlzbXMgYW5kIHJlZ3VsYXRpb24gb2YgYWx0ZXJuYXRpdmUgcHJlLW1STkEgc3BsaWNpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFubnUuIFJldi4gQmlvY2hlbTsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT04NDsgIGNpdGF0aW9uX3BhZ2VzPTI5MS0zMjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBsZXhpdHkgb2YgdGhlIGFsdGVybmF0aXZlIHNwbGljaW5nIGxhbmRzY2FwZSBpbiBwbGFudHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0zNjU3LTgzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BcmFwb3J0MTE6IGEgY29tcGxldGUgcmVhbm5vdGF0aW9uIG9mIHRoZSBBcmFiaWRvcHNpcyB0aGFsaWFuYSByZWZlcmVuY2UgZ2Vub21lOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBKOyAgY2l0YXRpb25feWVhcj0yMDE3OyAgY2l0YXRpb25fdm9sdW1lPTg5OyAgY2l0YXRpb25fcGFnZXM9Nzg5LTgwNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lIGFuZCB0cmFuc2NyaXB0b21lIG9mIHRoZSBwb3JjaW5lIHdoaXB3b3JtIFRyaWNodXJpcyBzdWlzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTQ2OyAgY2l0YXRpb25fcGFnZXM9NzAxLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXByZWhlbnNpdmUgYW5ub3RhdGlvbiBvZiBQaHlzY29taXRyZWxsYSBwYXRlbnMgc21hbGwgUk5BIGxvY2kgcmV2ZWFscyB0aGF0IHRoZSBoZXRlcm9jaHJvbWF0aWMgc2hvcnQgaW50ZXJmZXJpbmcgUk5BIHBhdGh3YXkgaXMgbGFyZ2VseSBjb25zZXJ2ZWQgaW4gbGFuZCBwbGFudHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9Mjc7ICBjaXRhdGlvbl9wYWdlcz0yMTQ4LTYyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMjUvMS82MS8yNDE1MTU1Ni9kc3gwMzgucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIFRoZSBnZW5vbWUgb2YgdGhlIHdpbGQgZGlwbG9pZCBzdHJhd2JlcnJ5IHNwZWNpZXMgRnJhZ2FyaWEgdmVzY2EsIGFuIGlkZWFsIG1vZGVsIHN5c3RlbSBvZiBjdWx0aXZhdGVkIHN0cmF3YmVycnkgKEZyYWdhcmlhIMOXIGFuYW5hc3NhLCBvY3RvcGxvaWQpIGFuZCIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXMvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9NDE5MTM0NSZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGSKirgqzMqIvx5iHfyb5Q1+LDVOtA==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f1%2f61%2f4191345" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f1%2f61%2f4191345" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=107546" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/25/1"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/1/3/m_dnares_25_1cover.png?Expires=2147483647&amp;Signature=SO6YCeydUh5yJs5n-IY9wmy~LoBoAx4P6Fvm-AjuGlXgQjk9YpdSZprn-w-uvSGcgqbsut967tdIfiY3dSqjk-kYgFErDTIMFh~0aK8LiNrwVszpCD29Va86wemvVduCLQhn8b79YN7Bcvi7qFB5E0wUNp-cxt7rNByaE14ePnoDwCgR5G58Pc5wx03IVhK9jft2n0g0lvAsEdQaWr3LG7PN4prc1KFhItNPOxm0fdg7HmXK23iCiP1EUMnFfqTR655CMN~bUB0aDDAZUT18HRZXXdjkjcMUCT~fjmwOlkGzU~71oDZYn-Ji-8-K8iuA-savjer0tQoSi784gQ8L6w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/25/1">
                        <div class="volume trailing-comma">Volume 25</div>
                        <div class="issue">Issue 1</div>

            </a>

        </div>
        <div class="ii-pub-date">
February 2018        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="112482839">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#112482839">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="112482844">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#112482844">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="112482856">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#112482856">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="112482905">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#112482905">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="112482984">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#112482984">4. Conclusions</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="112482988">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#112482988">5. Data availability</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="112482993">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#112482993">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="112483010">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#112483010">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/25/1/49/4107864">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/25/1/71/4265514">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Genome re-annotation of the wild strawberry <em>Fragaria vesca</em> using extensive Illumina- and SMRT-based RNA-seq datasets
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yongping Li</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Key Laboratory of Horticultural Plant Biology (Ministry of Education), College of Horticulture and Forestry Sciences, Huazhong Agricultural University, Wuhan 430070, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yongping+Li">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Li Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Li Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yongping Li</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Wei Wei</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Key Laboratory of Horticultural Plant Biology (Ministry of Education), College of Horticulture and Forestry Sciences, Huazhong Agricultural University, Wuhan 430070, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Wei+Wei">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Wei W">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Wei W&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Wei Wei</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jia Feng</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Key Laboratory of Horticultural Plant Biology (Ministry of Education), College of Horticulture and Forestry Sciences, Huazhong Agricultural University, Wuhan 430070, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jia+Feng">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Feng J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Feng J&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Jia Feng</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Huifeng Luo</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Key Laboratory of Horticultural Plant Biology (Ministry of Education), College of Horticulture and Forestry Sciences, Huazhong Agricultural University, Wuhan 430070, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Huifeng+Luo">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Luo H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Luo H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Huifeng Luo</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Mengting Pi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Key Laboratory of Horticultural Plant Biology (Ministry of Education), College of Horticulture and Forestry Sciences, Huazhong Agricultural University, Wuhan 430070, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Mengting+Pi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Pi M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Pi M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Mengting Pi</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Zhongchi Liu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Key Laboratory of Horticultural Plant Biology (Ministry of Education), College of Horticulture and Forestry Sciences, Huazhong Agricultural University, Wuhan 430070, China</div><div class="aff">Department of Cell Biology and Molecular Genetics, University of Maryland, College Park, MD 20742, USA</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Zhongchi+Liu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Liu Z">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Liu Z&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Zhongchi Liu</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Chunying Kang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Key Laboratory of Horticultural Plant Biology (Ministry of Education), College of Horticulture and Forestry Sciences, Huazhong Agricultural University, Wuhan 430070, China</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsx038-cor1" data-open="dsx038-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Tel. +86â€‰13871595425. Fax. +86â€‰27 87282010. Email: <a href="mailto:ckang@mail.hzau.edu.cn" target="_blank">ckang@mail.hzau.edu.cn</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Chunying+Kang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kang C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kang C&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Chunying Kang</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsx038-FM1"><div class="fn"><div class="footnote-content">Edited by Dr. Sachiko Isobe</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 25, Issue 1, 1 February 2018, Pages 61â€“70, <a href='https://doi.org/10.1093/dnares/dsx038'>https://doi.org/10.1093/dnares/dsx038</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">23 September 2017</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">13 July 2017</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">29 August 2017</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="4191345" href="/dnaresearch/article-pdf/25/1/61/24151556/dsx038.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Yongping Li, Wei Wei, Jia Feng, Huifeng Luo, Mengting Pi, Zhongchi Liu, Chunying Kang;  Genome re-annotation of the wild strawberry <em>Fragaria vesca</em> using extensive Illumina- and SMRT-based RNA-seq datasets, <em>DNA Research</em>, Volume 25, Issue 1, 1 February 2018, Pages 61â€“70, <a href='https://doi.org/10.1093/dnares/dsx038'>https://doi.org/10.1093/dnares/dsx038</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/4191345?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/4191345?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/4191345?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/4191345?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/4191345?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/4191345?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/4191345?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Genome%20re-annotation%20of%20the%20wild%20strawberry%20%26lt%3Bem%26gt%3BFragaria%20vesca%26lt%3B%2Fem%26gt%3B%20using%20extensive%20Illumina-%20and%20SMRT-based%20RNA-seq%20datasets&amp;publicationDate=2017-09-23&amp;volumeNum=25&amp;issueNum=1&amp;author=Li%2C%20Yongping%3B%20Wei%2C%20Wei&amp;startPage=61&amp;endPage=70&amp;contentId=10.1093%2Fdnares%2Fdsx038&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f1%2f61%2f4191345" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=112482839 id="112482839" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>The genome of the wild diploid strawberry species <em>Fragaria vesca</em>, an ideal model system of cultivated strawberry (<em>Fragaria Ã— ananassa</em>, octoploid) and other <em>Rosaceae</em> family crops, was first published in 2011 and followed by a new assembly (Fvb). However, the annotation for Fvb mainly relied on <em>ab initio</em> predictions and included only predicted coding sequences, therefore an improved annotation is highly desirable. Here, a new annotation version named v2.0.a2 was created for the Fvb genome by a pipeline utilizing one PacBio library, 90 Illumina RNA-seq libraries, and 9 small RNA-seq libraries. Altogether, 18,641 genes (55.6% out of 33,538 genes) were augmented with information on the 5â€² and/or 3â€² UTRs, 13,168 (39.3%) protein-coding genes were modified or newly identified, and 7,370 genes were found to possess alternative isoforms. In addition, 1,938 long non-coding RNAs, 171 miRNAs, and 51,714 small RNA clusters were integrated into the annotation. This new annotation of <em>F. vesca</em> is substantially improved in both accuracy and integrity of gene predictions, beneficial to the gene functional studies in strawberry and to the comparative genomic analysis of other horticultural crops in <em>Rosaceae</em> family.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genome annotation&quot;">genome annotation</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="PacBio-RNAseq">PacBio-RNAseq</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="Illumina-RNAseq">Illumina-RNAseq</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="strawberry">strawberry</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Fragaria vesca&quot;"><em>Fragaria vesca</em></a></div>                            <h2 scrollto-destination=112482844 id="112482844" class="section-title" >1. Introduction</h2>
<p>The diploid strawberry <em>F. vesca</em>, also known as Alpine or woodland strawberry, is the most widely distributed species naturally growing in the northern hemisphere. It also contributed to one set of the octoploid genome of the cultivated strawberry <em>Fragariaâ€‰Ã—â€‰ananassa.</em><a reveal-id="dsx038-B1" data-open="dsx038-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a><sup>,</sup><em>F. vesca</em> genome (âˆ¼240â€‰Mb) is relatively small and its genome sequence was published in 2011.<a reveal-id="dsx038-B2" data-open="dsx038-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> Several attributes of <em>F. vesca</em> make it a good model species of fruit crops, such as short stature, short life cycle, recurrent flowering, and high efficiency of transformation. To facilitate the gene functional studies in strawberry, we have identified and profiled the expression of protein-coding genes, lncRNAs and miRNAs using extensive transcriptome datasets generated mainly from flower and fruit tissues at different developmental stages with &gt;80 individual libraries in <em>F. vesca.</em><a reveal-id="dsx038-B3 dsx038-B4 dsx038-B5 dsx038-B6 dsx038-B7" data-open="dsx038-B3 dsx038-B4 dsx038-B5 dsx038-B6 dsx038-B7" class="link link-ref link-reveal xref-bibr"><sup>3â€“7</sup></a> As an emerging model species, <em>F. vesca</em> is playing more and more important roles both in the fields of basic biological research and in horticultural and agricultural research.</p><p>The genome of <em>F. vesca</em> was first sequenced from a fourth-generation inbred line of Hawaii4 (<em>F. vesca ssp. vesca</em>) solely by second generation short read technologies, and its genome annotation v1.1 only contains protein-coding genes derived from <em>ab initio</em> gene predictions.<a reveal-id="dsx038-B2" data-open="dsx038-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> Then, an updated annotation version (v1.1.a2) became available using 50 RNA-seq libraries generated from 25 different fruit tissue types to improve the annotation accuracy of protein-coding genes.<a reveal-id="dsx038-B8" data-open="dsx038-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> In the meantime, a refined assembly of the <em>F. vesca</em> reference genome, named Fvb, was generated based on dense linkage maps of the North American diploid <em>F. vesca ssp. bracteata.</em><a reveal-id="dsx038-B9" data-open="dsx038-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> Compared with the original genome FvH4, Fvb features many translocations and inversions. Although Fvb still contains a large number of gaps that could be closed by utilizing long read sequencing technology, it has a much shorter unanchored pseudo-chromosome. The location of <em>FvMYB10</em>, a master regulator of anthocyanin production in strawberry, is moved from Chromosome 6 in FvH4 to Chromosome 1 in Fvb, where it is supposed to be according to genetic studies, illustrating that Fvb is of high quality.<a reveal-id="dsx038-B10" data-open="dsx038-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> However, the gene annotation of Fvb (v2.0.a1) is not updated and merely obtained from the realignment of v1.1.</p><p>Previously, we analysed the transcriptomes of floral and fruit tissues in <em>F. vesca.</em><a reveal-id="dsx038-B3" data-open="dsx038-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a><sup>,</sup><a reveal-id="dsx038-B4" data-open="dsx038-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> During data analysis and gene functional studies, we noticed that a considerable proportion of genes are mis-annotated. The RNA-seq datasets from <em>F. vesca</em> (90 individual libraries) provide a rich data resource to re-annotate the genome. In addition, the PacBio full-length transcripts obtained from the fruit receptacle in <em>F. vesca</em> are also available,<a reveal-id="dsx038-B7" data-open="dsx038-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> which were proven to significantly increase the accuracy of genome annotation without any assembly.<a reveal-id="dsx038-B11" data-open="dsx038-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a><sup>,</sup><a reveal-id="dsx038-B12" data-open="dsx038-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a></p><p>The annotation v2.0.a1 of Fvb contains only the coding regions from translation start site (ATG) to stop codon of protein-coding genes. However, recent studies highlight the importance of short ORFs (open reading frame) in 5â€² UTRs in fine-tuning gene functions.<a reveal-id="dsx038-B13" data-open="dsx038-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> The 3â€² UTR sequences are also frequently needed for designing experiments. In addition, alternative splicing is an important regulatory mechanism at the post-transcriptional level, and up to 60% multi-exon protein-coding genes in plant genomes possess more than one isoform.<a reveal-id="dsx038-B7" data-open="dsx038-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> Besides protein-coding genes, non-coding transcripts are pervasively expressed and constitute an integral part of the transcriptomes, including long non-coding RNAs (lncRNAs) and miRNAs.<a reveal-id="dsx038-B5" data-open="dsx038-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a><sup>,</sup><a reveal-id="dsx038-B6" data-open="dsx038-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a> Collectively, an updated genome annotation for <em>F. vesca</em> that marries the Fvb assembly with the broader collection of accurately predicted genes, coding and noncoding, is timely and highly desirable.</p><p>Here, we optimized the genome annotation pipeline to improve the gene predictions in Fvb. We used a combination of MAKER2, AUGUSTUS, Program to Assemble Spliced Alignments (PASA), and manual curation to annotate protein-coding genes through integrating <em>ab initio</em> gene predictions, 90 RNA-seq transcriptome libraries and one PacBio full-length transcript library.<a reveal-id="dsx038-B14 dsx038-B15 dsx038-B16 dsx038-B17" data-open="dsx038-B14 dsx038-B15 dsx038-B16 dsx038-B17" class="link link-ref link-reveal xref-bibr"><sup>14â€“17</sup></a> The new annotation named V2.0.a2 not only updates the gene models of 13,168 loci, but also augments alternatively spliced isoforms of 7,370 genes, 5â€² and 3â€² UTRs of 18,641 genes, 1,938 lncRNAs, 171 miRNAs, and 51,714 small RNA clusters. Overall, V2.0.a2 provides a much more complete annotation of the <em>F. vesca</em> genome and will better serve gene functional studies in strawberry and other <em>Rosaceae</em> species.</p>                            <h2 scrollto-destination=112482856 id="112482856" class="section-title" >2. Materials and methods</h2>
                            <h3 scrollto-destination=112482857 id="112482857" class="section-title" >2.1. Transcriptome datasets used in this study</h3>
<p>Both full-length transcripts generated by PacBio and RNAseq datasets from different <em>F. vesca</em> accessions were used as evidence for gene annotation. The PacBio reads were generated from pooled fruit receptacles at different developmental stages.<a reveal-id="dsx038-B7" data-open="dsx038-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> A total of 90 Illumina RNA-seq libraries are respectively generated from fruits, flowers, meristems, and roots.<a reveal-id="dsx038-B3" data-open="dsx038-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a><sup>,</sup><a reveal-id="dsx038-B4" data-open="dsx038-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a><sup>,</sup><a reveal-id="dsx038-B18" data-open="dsx038-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> In addition, nine small RNA-seq libraries generated from vegetative, flower, and fruit tissues were also used for small RNA identification in this study.<a reveal-id="dsx038-B5" data-open="dsx038-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> Details of these datasets are described in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx038/3/suppl_data_dsx038.zip?Expires=2147483647&Signature=I3QaHcxw2cpoXFZuHY6o8jhHqE5S6J1CUZDrtDqh8UaclmGZdWqKCS64xe4ty2JGgYsezqePsCBR~xrQPD09iLOKdFU6OiI9QEzbhUz7PTkVdtgM8EictdusqAY6pQnramoMAMCwYVPzSrate3VZD2-2gYwqf-zrenMwRBJvXVQ47UZ1K02In0OpHuC1NFyL9GD6wQgR2O5kK5V4zhpVTz21QoifdpjPaRZf~lI0AD3zzlTaOLEGQ~HTiko78fhtwWTgBu3mTObx49ZNDbH2jYLC0nPmnd9fZq-C6KMHilJRAdDcZ4200OiIz3HakDTLfKZ0kvcnd8Wq~m-iU0JrKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>.</p>                            <h3 scrollto-destination=112482860 id="112482860" class="section-title" >2.2. High-fidelity gene models identified from SMRT reads</h3>
<p>The RS_IsoSeq pipeline (v2.3) was employed to analyse the PacBio data. First, the reads longer than 500â€‰bp with the minimum full pass number of 2 and a quality score above 90 were retained. Then, the full-length reads having 5â€² primer, 3â€² primer, and a poly-A tail were further selected for downstream analysis. Next, these full-length reads were collapsed into consensus transcripts by ICE and Quiver in the isoseq_cluster panel. The LoRDEC software was used to correct the sequencing errors in the consensus transcripts using Illumina reads as the reference (parameters: -k 19 -s 3).<a reveal-id="dsx038-B19" data-open="dsx038-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> The corrected consensus transcripts were then mapped by GMAP withâ€‰&gt;85% alignment coverage andâ€‰&gt;90% alignment identity.<a reveal-id="dsx038-B20" data-open="dsx038-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> At last, the aligned transcripts were used for the identification of the best gene models using Program to Assemble Spliced Alignments (PASA).<a reveal-id="dsx038-B14" data-open="dsx038-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> The scripts gff3_to_SNAP_train.pl bundled with SNAP, gmes_petap.pl bundled with GeneMark, and optimize_augustus.pl bundled with AUGUSTUS were used for training of each tool by the high-fidelity gene models with default parameters.</p>                            <h3 scrollto-destination=112482863 id="112482863" class="section-title" >2.3. A comprehensive transcriptome built from RNA-seq datasets and SMRT reads using PASA</h3>
<p>The RNA-seq reads with more that 90% of their bases having a quality score higher than 28 were retained using fastq_quality_filter (-q 28 -p 90), and the bases with low quality score in each RNA-seq read were trimmed off using fastx_trim built in FASTX-Toolkit (<a class="link link-uri" href="http://hannonlab.cshl.edu/fastx_toolkit/" target="_blank">http://hannonlab.cshl.edu/fastx_toolkit/</a> (5 September 2017, date last accessed)). Each library was mapped individually to Fvb<a reveal-id="dsx038-B9" data-open="dsx038-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> downloaded from GDR (<a class="link link-uri" href="https://www.rosaceae.org" target="_blank">https://www.rosaceae.org</a> (5 September 2017, date last accessed)) using Tophat2<a reveal-id="dsx038-B21" data-open="dsx038-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> with the following parameters: maximum intron size, 10,000bp; minimum intron size, 20â€‰bp; fr-firststrand for strand-specific libraries.</p><p>Reference-based assembly: For flower and fruit tissues, the aligned BAM files of the two biological replicates were, respectively, merged and sorted using Samtools (version 1.2)<a reveal-id="dsx038-B22" data-open="dsx038-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> due to relatively lower sequencing depth, resulting in a total of 37 BAM files. The analyses on other libraries were carried out individually. Next, transcripts were assembled individually for each library by Stringtie<a reveal-id="dsx038-B23" data-open="dsx038-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> with default settings except that the minimum isoform fraction was set to 0.3 to remove weakly expressed isoforms.</p><p><em>De novo</em> assembly: All the RNA-seq reads were pooled together for <em>de novo</em> assembly using Trinity<a reveal-id="dsx038-B24" data-open="dsx038-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> with default parameters.</p><p>To build a comprehensive transcriptome, the ID accessions of full-length and <em>de novo</em> transcripts were first fed into PASA, and full-length, genome-guided and <em>de novo</em> transcripts generated in parallel were then collapsed, mapped back to Fvb, and reconstructed by PASA through its alignment assembly module. The comprehensive transcriptome was later used as the mRNA evidence for MAKER2 described in the following section.</p>                            <h3 scrollto-destination=112482871 id="112482871" class="section-title" >2.4. MAKER2 annotation</h3>
<p>MAKER2 was used for generating an initial gene annotation.<a reveal-id="dsx038-B15" data-open="dsx038-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> First, RepeatModeler (<a class="link link-uri" href="http://www.repeatmasker.org/" target="_blank">http://www.repeatmasker.org/</a> (5 September 2017, date last accessed)) was employed to identify and classify repeats from the genome Fvb, and then RepeatMasker (Repbase version 20160829) was used to generate a masked genome. Plant protein sequences were downloaded from UniProt (<a class="link link-uri" href="http://www.uniprot.org/" target="_blank">http://www.uniprot.org/</a> (5 September 2017, date last accessed)) on 21 January 2017. MAKER2 was run on the masked Fvb with following evidence: Fvb annotation v2.0.a1, the comprehensive transcriptome, UniProt proteins, AUGUSTUS trained models, SNAP trained models, and GeneMark trained models.</p>                            <h3 scrollto-destination=112482874 id="112482874" class="section-title" >2.5. Further improve the genome using Augustus and EVM</h3>
<p>To use Augustus,<a reveal-id="dsx038-B25" data-open="dsx038-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> the following evidence were provided: (i) intron hints converted from uniquely mapped RNA-seq reads using the script bet_to_gff.pl bundled with GeneMark; (ii) intron hints converted from PacBio full-length transcripts by GMAP (<a class="link link-uri" href="http://bioinf.uni-greifswald.de/bioinf/wiki/pmwiki.php?n=Augustus.PacBioGMAP" target="_blank">http://bioinf.uni-greifswald.de/bioinf/wiki/pmwiki.php?n=Augustus.PacBioGMAP</a> (5 September 2017, date last accessed)); (iii) Protein hints generated from mapping UniPro proteins to masked genome by Exonerate;<a reveal-id="dsx038-B26" data-open="dsx038-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> (iv) repeat hints from the RepeatMasker output (<a class="link link-uri" href="http://bioinf.uni-greifswald.de/bioinf/wiki/pmwiki.php?n=Augustus.IncorporateRepeats" target="_blank">http://bioinf.uni-greifswald.de/bioinf/wiki/pmwiki.php?n=Augustus.IncorporateRepeats</a> (5 September 2017, date last accessed)). These hints were fed into Augustus to generate the high confidence gene models.</p><p>Next, EVidenceModeler (EVM)<a reveal-id="dsx038-B27" data-open="dsx038-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> was used to combine MAKER2 gene models, Augustus gene models, transcripts from SMRT and Illumina RNA-seq, and UniProt proteins with a nonstochastic weighted value into confident consensus gene models. The weight value for each part is: six for MAKER2 gene models, eight for Augustus gene models, 10 for SMRT transcripts, eight for Illumina transcripts, and three for homologous proteins. Finally, PASA was used to improve the EVM gene models by modifying gene structures and adding UTR annotations and alternatively spliced isoforms.</p>                            <h3 scrollto-destination=112482880 id="112482880" class="section-title" >2.6. Manual curation of new annotation</h3>
<p>All the gene models were inspected one by one and manually curated using a plug-in called Apollo<a reveal-id="dsx038-B28" data-open="dsx038-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> in JBrowse Genome Viewer<a reveal-id="dsx038-B29" data-open="dsx038-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> based on the mapped reads from different RNA-seq libraries.</p>                            <h3 scrollto-destination=112482887 id="112482887" class="section-title" >2.7. Alternative splicing analysis</h3>
<p>AStalavista<a reveal-id="dsx038-B30" data-open="dsx038-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> was employed to classify the alternative splicing events in the v2.0.a2 annotation. It groups the events into four major types, namely IR, ES, AA, and AD. The event belonging to each type were extracted from the AStalavista output by following codes: IR (AS code: 1^2-,0), ES (AS code: 1- 2^, 0), AA (AS code: 1-,2-), and AD (AS code: 1^,2^).</p>                            <h3 scrollto-destination=112482892 id="112482892" class="section-title" >2.8. Prediction of lncRNAs</h3>
<p>Two approaches were used to identify lncRNAs. The transcripts combined from the assemblies of Illumina RNA-seq and PacBio full-length reads were used for screening lncRNAs. The mono-exonic transcripts were removed to reduce false positives due to transcriptional noise and ambiguous alignment. In the first approach, the transcripts with class_code â€˜uâ€™ (unknown intergenic transcript), â€˜oâ€™ (generic exonic overlap with a reference transcript), â€˜xâ€™ (natural antisense transcript, NAT), and â€˜iâ€™ (intronic transcript) were selected. Then, the transcripts with lengthâ€‰&gt;â€‰200nt and ORFâ€‰&lt;â€‰300nt were retained. Next, the left transcripts were blasted against the Swiss-Protein database by BLASTX (2.6.0)<a reveal-id="dsx038-B31" data-open="dsx038-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> with E-valuesâ€‰&lt;â€‰1e-3 to remove potential protein-coding transcripts. At last, we used Coding Potential Calculator<a reveal-id="dsx038-B32" data-open="dsx038-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> to calculate the CPC score of each transcript, and only those with a CPC scoreâ€‰&lt;â€‰0 were considered as lncRNA candidates. In the second approach, lncRNAs were identified from the same set of transcripts by FEELnc<a reveal-id="dsx038-B33" data-open="dsx038-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> using the specificity threshold 0.97. The transcripts overlapping protein-coding genes in v2.0.a2 andâ€‰&lt;â€‰200â€‰nt in length were eliminated. FEELnc used the ORF coverage (i.e. length of the longest ORF/length of the lncRNA transcript), k-mer frequency, nucleotide frequency, and codon usage to distinguish between mRNAs and lncRNAs. Finally, the consensus transcripts identified by both methods were designated as lncRNAs. These lncRNAs were grouped into two major types (genic and intergenic) and six subtypes based on the localization and the direction of transcription relative to the proximal protein-coding genes by the FEELnc classifier module called FEELncclassifier.</p>                            <h3 scrollto-destination=112482896 id="112482896" class="section-title" >2.9. Alignment of small RNAs</h3>
<p>The raw reads from 9 small RNA-seq libraries generated from tissues of <em>F. vesca</em><a reveal-id="dsx038-B5" data-open="dsx038-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> were used for the identification of small RNAs. First, the 3â€² adapters were trimmed off using Cutadapt.<a reveal-id="dsx038-B34" data-open="dsx038-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> Then, the resulting .fasta file was mapped against the Fvb genome using bowtie (version 0.12.8)<a reveal-id="dsx038-B35" data-open="dsx038-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> with following parameters: -q -v 0 -p 30 -S -a -m 50. ShortStack<a reveal-id="dsx038-B36" data-open="dsx038-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> was subsequently employed to identify small RNA clusters, and only the dominant 20- to 24- nt in length and DCL-derived small RNA loci (more possibly related to RNAi) called out by ShortStack were retained. The expression level of each small RNA cluster was normalized using the count mode in ShortStack with default settings.</p>                            <h3 scrollto-destination=112482900 id="112482900" class="section-title" >2.10. Alignment of miRNAs</h3>
<p>The stem-loop sequences of 171 miRNAs in <em>F. vesca</em> predicted by a previous report<a reveal-id="dsx038-B5" data-open="dsx038-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> were mapped to the new <em>F. vesca</em> genome assembly Fvb by bowtie2 with default parameters, then the SAM file was converted into the GFF format by an in-house python script and integrated into the v2.0.a2 annotation file.</p>                            <h3 scrollto-destination=112482903 id="112482903" class="section-title" >2.11. Experimental verification of new gene models</h3>
<p>Total RNA was isolated from the fruit receptacle of YW5AF7 (the 7th generation inbred line of Yellow Wonder, an accession of <em>F. vesca</em>) using the Plant Total RNA Isolation Kit (Sangon Biotech, Shanghai, China, No. SK8631) following manufacturerâ€™s instructions. cDNA was synthesized from 1â€‰Î¼g total RNA in 20â€‰Î¼l solution using the PrimeScript<sup>TM</sup> RT reagent kit (TaKaRa, Shiga, Japan, Cat# RR047A). KOD DNA polymerase (TOYOBO Bio-Tech, Cat# F0934K) was used to amplify the coding regions of selected genes for Sanger sequencing with primers listed in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx038/3/suppl_data_dsx038.zip?Expires=2147483647&Signature=I3QaHcxw2cpoXFZuHY6o8jhHqE5S6J1CUZDrtDqh8UaclmGZdWqKCS64xe4ty2JGgYsezqePsCBR~xrQPD09iLOKdFU6OiI9QEzbhUz7PTkVdtgM8EictdusqAY6pQnramoMAMCwYVPzSrate3VZD2-2gYwqf-zrenMwRBJvXVQ47UZ1K02In0OpHuC1NFyL9GD6wQgR2O5kK5V4zhpVTz21QoifdpjPaRZf~lI0AD3zzlTaOLEGQ~HTiko78fhtwWTgBu3mTObx49ZNDbH2jYLC0nPmnd9fZq-C6KMHilJRAdDcZ4200OiIz3HakDTLfKZ0kvcnd8Wq~m-iU0JrKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>. The sequencing results were aligned with the sequences obtained from new annotations by Clustal Omega (<a class="link link-uri" href="http://www.ebi.ac.uk/Tools/msa/clustalo/" target="_blank">http://www.ebi.ac.uk/Tools/msa/clustalo/</a> (5 September 2017, date last accessed)).</p>                            <h2 scrollto-destination=112482905 id="112482905" class="section-title" >3. Results and discussion</h2>
                            <h3 scrollto-destination=112482907 id="112482907" class="section-title" >3.1. Update of gene models of protein-coding genes in Fvb based on SMRT and Illumina RNA-seq datasets</h3>
<p>In this study, a new Fvb annotation named version 2.0.a2 (v2.0.a2) was created that includes protein-coding genes, alternatively spliced isoforms of multiexon genes, small RNAs, and lncRNAs. An optimized workflow to annotate protein-coding genes and their isoforms is shown in <a reveal-id="dsx038-F1" data-open="dsx038-F1" class="link link-reveal link-table xref-fig">Figure 1</a>. To take advantage of the extensive RNA-seq datasets, MAKER2, specifically designed for second-generation genome projects, was first used to generate an initial protein-coding gene annotation.<a reveal-id="dsx038-B15" data-open="dsx038-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> Input data for MAKER2 include (i) repeats masked genome Fvb, (ii) <em>ab initio</em> gene predictors, including SNAP, GeneMaker, and Augustus, each trained with high-fidelity gene models, (iii) plant protein sequences downloaded from UniProt databases, (iv) transcripts assembled from a total of 90 Illumina RNA-seq libraries, (v) full length transcripts obtained from one PacBio transcriptome library (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx038/3/suppl_data_dsx038.zip?Expires=2147483647&Signature=I3QaHcxw2cpoXFZuHY6o8jhHqE5S6J1CUZDrtDqh8UaclmGZdWqKCS64xe4ty2JGgYsezqePsCBR~xrQPD09iLOKdFU6OiI9QEzbhUz7PTkVdtgM8EictdusqAY6pQnramoMAMCwYVPzSrate3VZD2-2gYwqf-zrenMwRBJvXVQ47UZ1K02In0OpHuC1NFyL9GD6wQgR2O5kK5V4zhpVTz21QoifdpjPaRZf~lI0AD3zzlTaOLEGQ~HTiko78fhtwWTgBu3mTObx49ZNDbH2jYLC0nPmnd9fZq-C6KMHilJRAdDcZ4200OiIz3HakDTLfKZ0kvcnd8Wq~m-iU0JrKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). The Illumina RNA-Seq datasets were generated from a series of tissue types in <em>F. vesca</em>, including leaf, seedling, root, shoot apical meristem, flower meristem, anther and carpel from flowers at different developmental stages and different fruit tissues at five stages with an average of 30 million and a range from 12 million to 89 million reads per library (see details in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx038/3/suppl_data_dsx038.zip?Expires=2147483647&Signature=I3QaHcxw2cpoXFZuHY6o8jhHqE5S6J1CUZDrtDqh8UaclmGZdWqKCS64xe4ty2JGgYsezqePsCBR~xrQPD09iLOKdFU6OiI9QEzbhUz7PTkVdtgM8EictdusqAY6pQnramoMAMCwYVPzSrate3VZD2-2gYwqf-zrenMwRBJvXVQ47UZ1K02In0OpHuC1NFyL9GD6wQgR2O5kK5V4zhpVTz21QoifdpjPaRZf~lI0AD3zzlTaOLEGQ~HTiko78fhtwWTgBu3mTObx49ZNDbH2jYLC0nPmnd9fZq-C6KMHilJRAdDcZ4200OiIz3HakDTLfKZ0kvcnd8Wq~m-iU0JrKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). In total, there are 2.5 billion RNA-Seq short reads. The PacBio data was generated from fruit receptacles pooled from small green to mature stages in <em>F. vesca</em>, containing 82,360 full-length consensus transcripts derived from 442,601ReadofInserts. </p>                        <a id="112482911" scrollto-destination="112482911"></a>
<div id="dsx038-F1" data-id="dsx038-F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx038/3/m_dsx038f1.png?Expires=2147483647&Signature=qb3SE~-nNH7TigqYQl93IC-fp2v6xgerLykZS5bJNGI9HIHePipT1aeHJflnEuVDJyzgqlvnWDqfylIHHnP4qcq9H63lb2fejNZuRJHBcXVuZICyRZYFyIxNBsVPpq5OLYsxNA6znRdkCracpu6yXt-yWqwfHioBIEXRHJ6Jwc2Xq9pxuPiS2c3-SnqwpTQkwNemCxQtfFYhihd3SAi1s4saqXkqirMeYtw3LNhBoSwo7owmLjepwRx8rEj1yvKtzPgP51OZd8ZFbb3YjfsyeCEolfTFNQz4Oc7d2uey8O5cd6FIubGN-NZLF6BfXcb6k05msAXjLt9rPogOAcpFVQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Annotation workflow for strawberry protein-coding genes. The SMRT transcripts were used for the training of SNAP, GeneMark and Augustus. RNA-Seq libraries were used to build comprehensive transcriptomes by a combination of de novo and genome-guided assemblies. UniProt protein sequences were also provided for accurate gene annotation. The red dotted box indicates the core tools used for the annotation. In the end, manual curation was carried out to assure the accuracy." path-from-xml="dsx038f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112482911/dsx038f1.tif" path-from-xml="dsx038f1.tif" target="_blank">View large</a><a section="112482911" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx038/3/dsx038f1.png?Expires=2147483647&Signature=NAgvtXTWZYYmkiNU9Jp7l2YY1yIsIdSlnxB64fB5vZJvenUb1wsWQvObz-j~rCq6QqEnM7swrd5vw0VZhokXF~3B~2B01Qj~Ihg-iJSC5TuI9NLxtaDnfSkmu9jVhmeS9l~aVHxGExbyjr2hqtKpm3fHo1dtizLGJpPc8A653ZQTfrb6DIME3YxVJFTZzUqYlu7iZUJlGxYNBSOMASmjL7NkJwLNRdfyCZFroGjwTLmuvHWpgiD7O3Gazy6kZX-0gcGeD8PBx7AdnTHVOXbAC5AilsM3m6fMsiY9DHeUbN2HyFXFXBPiNZYLvUHxY1MHGh2fVuf0cHxXASnG0CyuVw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112482911&ar=4191345&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx038f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Annotation workflow for strawberry protein-coding genes. The SMRT transcripts were used for the training of SNAP, GeneMark and Augustus. RNA-Seq libraries were used to build comprehensive transcriptomes by a combination of de novo and genome-guided assemblies. UniProt protein sequences were also provided for accurate gene annotation. The red dotted box indicates the core tools used for the annotation. In the end, manual curation was carried out to assure the accuracy.</p></div></div><div content-id="dsx038-F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx038/3/m_dsx038f1.png?Expires=2147483647&Signature=qb3SE~-nNH7TigqYQl93IC-fp2v6xgerLykZS5bJNGI9HIHePipT1aeHJflnEuVDJyzgqlvnWDqfylIHHnP4qcq9H63lb2fejNZuRJHBcXVuZICyRZYFyIxNBsVPpq5OLYsxNA6znRdkCracpu6yXt-yWqwfHioBIEXRHJ6Jwc2Xq9pxuPiS2c3-SnqwpTQkwNemCxQtfFYhihd3SAi1s4saqXkqirMeYtw3LNhBoSwo7owmLjepwRx8rEj1yvKtzPgP51OZd8ZFbb3YjfsyeCEolfTFNQz4Oc7d2uey8O5cd6FIubGN-NZLF6BfXcb6k05msAXjLt9rPogOAcpFVQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Annotation workflow for strawberry protein-coding genes. The SMRT transcripts were used for the training of SNAP, GeneMark and Augustus. RNA-Seq libraries were used to build comprehensive transcriptomes by a combination of de novo and genome-guided assemblies. UniProt protein sequences were also provided for accurate gene annotation. The red dotted box indicates the core tools used for the annotation. In the end, manual curation was carried out to assure the accuracy." path-from-xml="dsx038f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112482911/dsx038f1.tif" path-from-xml="dsx038f1.tif" target="_blank">View large</a><a section="112482911" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx038/3/dsx038f1.png?Expires=2147483647&Signature=NAgvtXTWZYYmkiNU9Jp7l2YY1yIsIdSlnxB64fB5vZJvenUb1wsWQvObz-j~rCq6QqEnM7swrd5vw0VZhokXF~3B~2B01Qj~Ihg-iJSC5TuI9NLxtaDnfSkmu9jVhmeS9l~aVHxGExbyjr2hqtKpm3fHo1dtizLGJpPc8A653ZQTfrb6DIME3YxVJFTZzUqYlu7iZUJlGxYNBSOMASmjL7NkJwLNRdfyCZFroGjwTLmuvHWpgiD7O3Gazy6kZX-0gcGeD8PBx7AdnTHVOXbAC5AilsM3m6fMsiY9DHeUbN2HyFXFXBPiNZYLvUHxY1MHGh2fVuf0cHxXASnG0CyuVw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112482911&ar=4191345&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx038f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Annotation workflow for strawberry protein-coding genes. The SMRT transcripts were used for the training of SNAP, GeneMark and Augustus. RNA-Seq libraries were used to build comprehensive transcriptomes by a combination of de novo and genome-guided assemblies. UniProt protein sequences were also provided for accurate gene annotation. The red dotted box indicates the core tools used for the annotation. In the end, manual curation was carried out to assure the accuracy.</p></div></div><p>The ambiguity of transcript reconstruction will affect the accuracy of gene annotation when using MAKER2. To avoid assembly errors in gene prediction, it is advantageous to use the transcript information in unassembled mapped reads.<a reveal-id="dsx038-B37" data-open="dsx038-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> Augustus is a gene finder that can incorporate data from RNA-seq reads (junctions reads) directly into weighted predictions, join exons into a single gene and identify alternative transcription events.<a reveal-id="dsx038-B25" data-open="dsx038-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> Therefore, Augustus was employed to further improve gene predictions through incorporating the intron hints from RNA-seq reads and protein hints from UniProt databases. Then, the gene structures predicted by both MAKER2 and Augustus were combined into consensus gene models using EVidenceModeler (EVM),<a reveal-id="dsx038-B27" data-open="dsx038-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> and the gene models generated from EVM were updated by PASA.<a reveal-id="dsx038-B14" data-open="dsx038-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> At last, the new annotation was inspected across the entire genome in IGV<a reveal-id="dsx038-B38" data-open="dsx038-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> to evaluate the prediction and select the optimal gene models by comparing with the aligned RNA-seq reads, and about 1,000 (3%) genes were manually curated in this way.</p><p>Annotation v2.0.a2 of the genome Fvb contains a final set of 33,538 protein-coding loci with 50,738 transcripts (<a reveal-id="dsx038-T1" data-open="dsx038-T1" class="link link-reveal link-table xref-fig">Table 1</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx038/3/suppl_data_dsx038.zip?Expires=2147483647&Signature=I3QaHcxw2cpoXFZuHY6o8jhHqE5S6J1CUZDrtDqh8UaclmGZdWqKCS64xe4ty2JGgYsezqePsCBR~xrQPD09iLOKdFU6OiI9QEzbhUz7PTkVdtgM8EictdusqAY6pQnramoMAMCwYVPzSrate3VZD2-2gYwqf-zrenMwRBJvXVQ47UZ1K02In0OpHuC1NFyL9GD6wQgR2O5kK5V4zhpVTz21QoifdpjPaRZf~lI0AD3zzlTaOLEGQ~HTiko78fhtwWTgBu3mTObx49ZNDbH2jYLC0nPmnd9fZq-C6KMHilJRAdDcZ4200OiIz3HakDTLfKZ0kvcnd8Wq~m-iU0JrKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). The Locus IDs for 30,012 identical or modified genes stay the same between the older annotation v2.0a1 and this annotation (v2.0.a2), while 3,525 genes (novel or split) were newly numbered using the same nomenclature as previous annotations (i.e. GeneXXXXX). The Locus IDs in both versions for the newly modified genes are listed in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx038/3/suppl_data_dsx038.zip?Expires=2147483647&Signature=I3QaHcxw2cpoXFZuHY6o8jhHqE5S6J1CUZDrtDqh8UaclmGZdWqKCS64xe4ty2JGgYsezqePsCBR~xrQPD09iLOKdFU6OiI9QEzbhUz7PTkVdtgM8EictdusqAY6pQnramoMAMCwYVPzSrate3VZD2-2gYwqf-zrenMwRBJvXVQ47UZ1K02In0OpHuC1NFyL9GD6wQgR2O5kK5V4zhpVTz21QoifdpjPaRZf~lI0AD3zzlTaOLEGQ~HTiko78fhtwWTgBu3mTObx49ZNDbH2jYLC0nPmnd9fZq-C6KMHilJRAdDcZ4200OiIz3HakDTLfKZ0kvcnd8Wq~m-iU0JrKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>. One improvement of v2.0.a2 is that a total of 18,641 genes were augmented by inclusion of 5â€² UTRs and/or 3â€² UTRs, representing nearly 55.6% of all protein-coding genes (<a reveal-id="dsx038-F2" data-open="dsx038-F2" class="link link-reveal link-table xref-fig">Fig. 2a and b</a>). Moreover, the gene models of 13,168 protein-coding genes were modified and updated (<a reveal-id="dsx038-F2" data-open="dsx038-F2" class="link link-reveal link-table xref-fig">Fig. 2a and c</a>). More specifically, 2,692 genes were derived from splitting longer genes (previously fused genes), and 213 genes were obtained by fusing neighbouring genes (<a reveal-id="dsx038-F2" data-open="dsx038-F2" class="link link-reveal link-table xref-fig">Fig. 2a, d, and e</a>). In addition, 2,031 genes were removed, and 631 genes were added (<a reveal-id="dsx038-F2" data-open="dsx038-F2" class="link link-reveal link-table xref-fig">Fig. 2a</a>). To further evaluate the quality of the annotation, the BUSCO genes<a reveal-id="dsx038-B39" data-open="dsx038-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> were compared between v2.0.a1 and v2.0.a2. The results show a significant increase of complete BUSCOs from 88.9% for v2.0.a1 to 95.7% for v2.0.a2, while the fragmented and missing BUSCOs were reduced in v2.0.a2 (<a reveal-id="dsx038-T1" data-open="dsx038-T1" class="link link-reveal link-table xref-fig">Table 1</a>), indicating a higher quality annotation. These updates will better serve gene structure and functional studies. <div content-id="dsx038-T1" class="table-modal"><div class="table-wrap"><div id="dsx038-T1" data-id="dsx038-T1" class="table-wrap-title"><div class="label -label">Table 1</div><div class="caption"><p>Summary of the v2.0.a2 annotation</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>TypeÂ </th><th>V2.0.a1Â </th><th>V2.0.a2Â </th></tr></thead><tbody><tr><td><strong>Protein-coding genes</strong>Â </td><td>Â </td><td>Â </td></tr><tr><td>Number of genesÂ </td><td>33,673Â </td><td>33,538Â </td></tr><tr><td>Mean length of genomic lociÂ </td><td>2,863Â </td><td>2,856Â </td></tr><tr><td>Mean exon numberÂ </td><td>5.1Â </td><td>4.62Â </td></tr><tr><td>Mean CDS lengthÂ </td><td>1,188Â </td><td>1,123Â </td></tr><tr><td>Mean length of intronsÂ </td><td>408Â </td><td>320Â </td></tr><tr><td>Genes with 5â€² UTRÂ </td><td>â€“Â </td><td>17,616Â </td></tr><tr><td>Genes with 3â€² UTRÂ </td><td>â€“Â </td><td>17,971Â </td></tr><tr><td>Genes with both 5â€² and 3â€² UTRÂ </td><td>â€“Â </td><td>16,946Â </td></tr><tr><td>Mean 5â€² UTR length (bp)Â </td><td>â€“Â </td><td>320Â </td></tr><tr><td>Mean 3â€² UTR length (bp)Â </td><td>â€“Â </td><td>513Â </td></tr><tr><td>Number of genes with isoformsÂ </td><td>â€“Â </td><td>7,370Â </td></tr><tr><td>Mean isoform number per geneÂ </td><td>1.00Â </td><td>1.51Â </td></tr><tr><td>Genes with functional annotationsÂ </td><td>27,875Â </td><td>28,798Â </td></tr><tr><td>Genes with GO termsÂ </td><td>17,156Â </td><td>22,106Â </td></tr><tr><td>Complete BUSCOsÂ </td><td>88.9%Â </td><td>95.7%Â </td></tr><tr><td>Fragmented BUSCOsÂ </td><td>5.6%Â </td><td>1.9%Â </td></tr><tr><td>Missing BUSCOsÂ </td><td>5.5%Â </td><td>2.4%Â </td></tr><tr><td><strong>Non-coding genes</strong>Â </td><td>Â </td><td>Â </td></tr><tr><td>microRNAs (miRNAs)Â </td><td>â€“Â </td><td>171Â </td></tr><tr><td>Small RNA clustersÂ </td><td>â€“Â </td><td>51,714Â </td></tr><tr><td>Long non-coding RNAs (lncRNAs)Â </td><td>â€“Â </td><td>1,938Â </td></tr><tr><td><strong>Total loci</strong>Â </td><td>33,673Â </td><td>87,361Â </td></tr></tbody></table></div><div content-id="dsx038-T1" class="table-modal"><table><thead name="thead" align=""><tr><th>TypeÂ </th><th>V2.0.a1Â </th><th>V2.0.a2Â </th></tr></thead><tbody><tr><td><strong>Protein-coding genes</strong>Â </td><td>Â </td><td>Â </td></tr><tr><td>Number of genesÂ </td><td>33,673Â </td><td>33,538Â </td></tr><tr><td>Mean length of genomic lociÂ </td><td>2,863Â </td><td>2,856Â </td></tr><tr><td>Mean exon numberÂ </td><td>5.1Â </td><td>4.62Â </td></tr><tr><td>Mean CDS lengthÂ </td><td>1,188Â </td><td>1,123Â </td></tr><tr><td>Mean length of intronsÂ </td><td>408Â </td><td>320Â </td></tr><tr><td>Genes with 5â€² UTRÂ </td><td>â€“Â </td><td>17,616Â </td></tr><tr><td>Genes with 3â€² UTRÂ </td><td>â€“Â </td><td>17,971Â </td></tr><tr><td>Genes with both 5â€² and 3â€² UTRÂ </td><td>â€“Â </td><td>16,946Â </td></tr><tr><td>Mean 5â€² UTR length (bp)Â </td><td>â€“Â </td><td>320Â </td></tr><tr><td>Mean 3â€² UTR length (bp)Â </td><td>â€“Â </td><td>513Â </td></tr><tr><td>Number of genes with isoformsÂ </td><td>â€“Â </td><td>7,370Â </td></tr><tr><td>Mean isoform number per geneÂ </td><td>1.00Â </td><td>1.51Â </td></tr><tr><td>Genes with functional annotationsÂ </td><td>27,875Â </td><td>28,798Â </td></tr><tr><td>Genes with GO termsÂ </td><td>17,156Â </td><td>22,106Â </td></tr><tr><td>Complete BUSCOsÂ </td><td>88.9%Â </td><td>95.7%Â </td></tr><tr><td>Fragmented BUSCOsÂ </td><td>5.6%Â </td><td>1.9%Â </td></tr><tr><td>Missing BUSCOsÂ </td><td>5.5%Â </td><td>2.4%Â </td></tr><tr><td><strong>Non-coding genes</strong>Â </td><td>Â </td><td>Â </td></tr><tr><td>microRNAs (miRNAs)Â </td><td>â€“Â </td><td>171Â </td></tr><tr><td>Small RNA clustersÂ </td><td>â€“Â </td><td>51,714Â </td></tr><tr><td>Long non-coding RNAs (lncRNAs)Â </td><td>â€“Â </td><td>1,938Â </td></tr><tr><td><strong>Total loci</strong>Â </td><td>33,673Â </td><td>87,361Â </td></tr></tbody></table></div></div></div><div class="table-wrap"><div id="dsx038-T1" data-id="dsx038-T1" class="table-wrap-title"><div class="label -label">Table 1</div><div class="caption"><p>Summary of the v2.0.a2 annotation</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>TypeÂ </th><th>V2.0.a1Â </th><th>V2.0.a2Â </th></tr></thead><tbody><tr><td><strong>Protein-coding genes</strong>Â </td><td>Â </td><td>Â </td></tr><tr><td>Number of genesÂ </td><td>33,673Â </td><td>33,538Â </td></tr><tr><td>Mean length of genomic lociÂ </td><td>2,863Â </td><td>2,856Â </td></tr><tr><td>Mean exon numberÂ </td><td>5.1Â </td><td>4.62Â </td></tr><tr><td>Mean CDS lengthÂ </td><td>1,188Â </td><td>1,123Â </td></tr><tr><td>Mean length of intronsÂ </td><td>408Â </td><td>320Â </td></tr><tr><td>Genes with 5â€² UTRÂ </td><td>â€“Â </td><td>17,616Â </td></tr><tr><td>Genes with 3â€² UTRÂ </td><td>â€“Â </td><td>17,971Â </td></tr><tr><td>Genes with both 5â€² and 3â€² UTRÂ </td><td>â€“Â </td><td>16,946Â </td></tr><tr><td>Mean 5â€² UTR length (bp)Â </td><td>â€“Â </td><td>320Â </td></tr><tr><td>Mean 3â€² UTR length (bp)Â </td><td>â€“Â </td><td>513Â </td></tr><tr><td>Number of genes with isoformsÂ </td><td>â€“Â </td><td>7,370Â </td></tr><tr><td>Mean isoform number per geneÂ </td><td>1.00Â </td><td>1.51Â </td></tr><tr><td>Genes with functional annotationsÂ </td><td>27,875Â </td><td>28,798Â </td></tr><tr><td>Genes with GO termsÂ </td><td>17,156Â </td><td>22,106Â </td></tr><tr><td>Complete BUSCOsÂ </td><td>88.9%Â </td><td>95.7%Â </td></tr><tr><td>Fragmented BUSCOsÂ </td><td>5.6%Â </td><td>1.9%Â </td></tr><tr><td>Missing BUSCOsÂ </td><td>5.5%Â </td><td>2.4%Â </td></tr><tr><td><strong>Non-coding genes</strong>Â </td><td>Â </td><td>Â </td></tr><tr><td>microRNAs (miRNAs)Â </td><td>â€“Â </td><td>171Â </td></tr><tr><td>Small RNA clustersÂ </td><td>â€“Â </td><td>51,714Â </td></tr><tr><td>Long non-coding RNAs (lncRNAs)Â </td><td>â€“Â </td><td>1,938Â </td></tr><tr><td><strong>Total loci</strong>Â </td><td>33,673Â </td><td>87,361Â </td></tr></tbody></table></div><div content-id="dsx038-T1" class="table-modal"><table><thead name="thead" align=""><tr><th>TypeÂ </th><th>V2.0.a1Â </th><th>V2.0.a2Â </th></tr></thead><tbody><tr><td><strong>Protein-coding genes</strong>Â </td><td>Â </td><td>Â </td></tr><tr><td>Number of genesÂ </td><td>33,673Â </td><td>33,538Â </td></tr><tr><td>Mean length of genomic lociÂ </td><td>2,863Â </td><td>2,856Â </td></tr><tr><td>Mean exon numberÂ </td><td>5.1Â </td><td>4.62Â </td></tr><tr><td>Mean CDS lengthÂ </td><td>1,188Â </td><td>1,123Â </td></tr><tr><td>Mean length of intronsÂ </td><td>408Â </td><td>320Â </td></tr><tr><td>Genes with 5â€² UTRÂ </td><td>â€“Â </td><td>17,616Â </td></tr><tr><td>Genes with 3â€² UTRÂ </td><td>â€“Â </td><td>17,971Â </td></tr><tr><td>Genes with both 5â€² and 3â€² UTRÂ </td><td>â€“Â </td><td>16,946Â </td></tr><tr><td>Mean 5â€² UTR length (bp)Â </td><td>â€“Â </td><td>320Â </td></tr><tr><td>Mean 3â€² UTR length (bp)Â </td><td>â€“Â </td><td>513Â </td></tr><tr><td>Number of genes with isoformsÂ </td><td>â€“Â </td><td>7,370Â </td></tr><tr><td>Mean isoform number per geneÂ </td><td>1.00Â </td><td>1.51Â </td></tr><tr><td>Genes with functional annotationsÂ </td><td>27,875Â </td><td>28,798Â </td></tr><tr><td>Genes with GO termsÂ </td><td>17,156Â </td><td>22,106Â </td></tr><tr><td>Complete BUSCOsÂ </td><td>88.9%Â </td><td>95.7%Â </td></tr><tr><td>Fragmented BUSCOsÂ </td><td>5.6%Â </td><td>1.9%Â </td></tr><tr><td>Missing BUSCOsÂ </td><td>5.5%Â </td><td>2.4%Â </td></tr><tr><td><strong>Non-coding genes</strong>Â </td><td>Â </td><td>Â </td></tr><tr><td>microRNAs (miRNAs)Â </td><td>â€“Â </td><td>171Â </td></tr><tr><td>Small RNA clustersÂ </td><td>â€“Â </td><td>51,714Â </td></tr><tr><td>Long non-coding RNAs (lncRNAs)Â </td><td>â€“Â </td><td>1,938Â </td></tr><tr><td><strong>Total loci</strong>Â </td><td>33,673Â </td><td>87,361Â </td></tr></tbody></table></div></div></p>                        <a id="112482920" scrollto-destination="112482920"></a>
<div id="dsx038-F2" data-id="dsx038-F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx038/3/m_dsx038f2.png?Expires=2147483647&Signature=VDwq9Vn7hXLdE9S86Fww1arkTWIZUfVjS2qZG2fx4MQCmchh9nGlCea5E7rwThurXe5G1fPwIU8wJaeKijzEnYLhetEOBeONsJdVaO9E3exLSQmneNNRFk3zaxSIKxS5B2PtjumIqHAOCGXVIUV9YzSEHHyNP1F3aA82wdp8uXUep1msyrzueEqVUEbP2szQ1FcT-MHvJ~hQ1m0TaC2aslPXNWZvC6fZNuvGevJLfha5B6X8zoRtkEkPGclgjjpyTJc224abtlOSDjkGpb9S6NPY9EdUIETJu8WuzbE5CDGa9vwC4DszFjuW2r4IEg8~ffA9mo9CLk4SMF6RxjN3bg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Illustration of the modifications to protein-coding genes in v2.0.a2. (a) Summary of the major types of modifications. The examples of four types are shown in the subfigures b to e. â€˜Number of genesâ€™ indicates the number of genes with each type of modifications in v2.0.a2. (b) 5â€² and 3â€² UTRs were added for gene20640. (c) Gene12679 has different and fewer exons than the v2.0.a1 equivalent. (d) Gene20642 in v2.0.a1 was split to create two loci, gene36495 and gene36496. (e) Gene12833 and gene12832 in v2.0.a1 were fused to create a single locus, gene12832. Exons are in blue, untranslated regions (UTRs) are in dark red, and introns are indicated by the thin black lines. The modified regions were highlighted by a dotted rectangle." path-from-xml="dsx038f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112482920/dsx038f2.tif" path-from-xml="dsx038f2.tif" target="_blank">View large</a><a section="112482920" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx038/3/dsx038f2.png?Expires=2147483647&Signature=KVq9xzKW3p-X4sJ76T9DggdvA6TKsJF6lscKebk99y-dcFNugIM5mVUsr7Zw5keLZDkWTplBFKnTDinYwTWM4GZsTS88clqvUF1EWElVWkaRb~293eUDebBaJ5QIfQGyis-RZSwg2Aaql42KmrmgrF3nEUrdaumJSvT64bEV4UWHCqEusVhSHLEFssg2YGU82XiqZMrQZ3NHpkCHEWtBnYYV2MWyMV14bBnNzEpctmx4s2OL7UdbK0wvL2OUzt~240j4a3nLjexswaf2BvJ7T1jaAL20QzEC3llzOtweqCrZgmP~TSiy~sn94LhLqdVFyT8TQXAAwj9nkgQQWBC8eg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112482920&ar=4191345&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx038f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Illustration of the modifications to protein-coding genes in v2.0.a2. (a) Summary of the major types of modifications. The examples of four types are shown in the subfigures b to e. â€˜Number of genesâ€™ indicates the number of genes with each type of modifications in v2.0.a2. (b) 5â€² and 3â€² UTRs were added for gene20640. (c) Gene12679 has different and fewer exons than the v2.0.a1 equivalent. (d) Gene20642 in v2.0.a1 was split to create two loci, gene36495 and gene36496. (e) Gene12833 and gene12832 in v2.0.a1 were fused to create a single locus, gene12832. Exons are in blue, untranslated regions (UTRs) are in dark red, and introns are indicated by the thin black lines. The modified regions were highlighted by a dotted rectangle.</p></div></div><div content-id="dsx038-F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx038/3/m_dsx038f2.png?Expires=2147483647&Signature=VDwq9Vn7hXLdE9S86Fww1arkTWIZUfVjS2qZG2fx4MQCmchh9nGlCea5E7rwThurXe5G1fPwIU8wJaeKijzEnYLhetEOBeONsJdVaO9E3exLSQmneNNRFk3zaxSIKxS5B2PtjumIqHAOCGXVIUV9YzSEHHyNP1F3aA82wdp8uXUep1msyrzueEqVUEbP2szQ1FcT-MHvJ~hQ1m0TaC2aslPXNWZvC6fZNuvGevJLfha5B6X8zoRtkEkPGclgjjpyTJc224abtlOSDjkGpb9S6NPY9EdUIETJu8WuzbE5CDGa9vwC4DszFjuW2r4IEg8~ffA9mo9CLk4SMF6RxjN3bg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Illustration of the modifications to protein-coding genes in v2.0.a2. (a) Summary of the major types of modifications. The examples of four types are shown in the subfigures b to e. â€˜Number of genesâ€™ indicates the number of genes with each type of modifications in v2.0.a2. (b) 5â€² and 3â€² UTRs were added for gene20640. (c) Gene12679 has different and fewer exons than the v2.0.a1 equivalent. (d) Gene20642 in v2.0.a1 was split to create two loci, gene36495 and gene36496. (e) Gene12833 and gene12832 in v2.0.a1 were fused to create a single locus, gene12832. Exons are in blue, untranslated regions (UTRs) are in dark red, and introns are indicated by the thin black lines. The modified regions were highlighted by a dotted rectangle." path-from-xml="dsx038f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112482920/dsx038f2.tif" path-from-xml="dsx038f2.tif" target="_blank">View large</a><a section="112482920" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx038/3/dsx038f2.png?Expires=2147483647&Signature=KVq9xzKW3p-X4sJ76T9DggdvA6TKsJF6lscKebk99y-dcFNugIM5mVUsr7Zw5keLZDkWTplBFKnTDinYwTWM4GZsTS88clqvUF1EWElVWkaRb~293eUDebBaJ5QIfQGyis-RZSwg2Aaql42KmrmgrF3nEUrdaumJSvT64bEV4UWHCqEusVhSHLEFssg2YGU82XiqZMrQZ3NHpkCHEWtBnYYV2MWyMV14bBnNzEpctmx4s2OL7UdbK0wvL2OUzt~240j4a3nLjexswaf2BvJ7T1jaAL20QzEC3llzOtweqCrZgmP~TSiy~sn94LhLqdVFyT8TQXAAwj9nkgQQWBC8eg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112482920&ar=4191345&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx038f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Illustration of the modifications to protein-coding genes in v2.0.a2. (a) Summary of the major types of modifications. The examples of four types are shown in the subfigures b to e. â€˜Number of genesâ€™ indicates the number of genes with each type of modifications in v2.0.a2. (b) 5â€² and 3â€² UTRs were added for gene20640. (c) Gene12679 has different and fewer exons than the v2.0.a1 equivalent. (d) Gene20642 in v2.0.a1 was split to create two loci, gene36495 and gene36496. (e) Gene12833 and gene12832 in v2.0.a1 were fused to create a single locus, gene12832. Exons are in blue, untranslated regions (UTRs) are in dark red, and introns are indicated by the thin black lines. The modified regions were highlighted by a dotted rectangle.</p></div></div>                            <h3 scrollto-destination=112482924 id="112482924" class="section-title" >3.2. Functional annotation of protein-coding genes</h3>
<p>To update the functional annotation of novel and pre-existing protein-coding genes in v2.0.a2, all the protein sequences were blasted against known proteins in the InterPro databases using InterProScan.<a reveal-id="dsx038-B40" data-open="dsx038-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> In addition, each gene was also blasted against the NCBI non-redundant (nr) protein database to gain the Gene ontology (GO) term by Blast2go.<a reveal-id="dsx038-B41" data-open="dsx038-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a> As a result, a total of 28,798 and 22,106 genes in v2.0.a2 gained functional annotations and GO terms, respectively (<a reveal-id="dsx038-T1" data-open="dsx038-T1" class="link link-reveal link-table xref-fig">Table 1</a>). Especially, there is a great increase in the percentage of genes with GO terms, from 51% in v2.0.a1 to 66% in v2.0.a2 (<a reveal-id="dsx038-T1" data-open="dsx038-T1" class="link link-reveal link-table xref-fig">Table 1</a>). For novel genes, 527 of the 631 loci (83.5%) were assigned with functional annotations.</p><p>Transcription factors and regulators and protein kinases usually play crucial roles during signal transduction. In the iTAK pipeline<a reveal-id="dsx038-B42" data-open="dsx038-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a> that we used for the classification of these genes, transcription factors are defined as proteins that regulate the expression of target genes by binding to specific cis-elements in promoter regions, while transcriptional regulators operate indirectly via interaction with the basal transcription apparatus (e.g. transcription factors) or by altering the accessibility of DNA to TFs via chromatin remodelling to regulate the expression of target genes. As a result, we identified 1,542 transcription factors from 68 families, and 390 transcriptional regulators in the v2.0.a2 annotation by using the iTAK pipeline (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx038/3/suppl_data_dsx038.zip?Expires=2147483647&Signature=I3QaHcxw2cpoXFZuHY6o8jhHqE5S6J1CUZDrtDqh8UaclmGZdWqKCS64xe4ty2JGgYsezqePsCBR~xrQPD09iLOKdFU6OiI9QEzbhUz7PTkVdtgM8EictdusqAY6pQnramoMAMCwYVPzSrate3VZD2-2gYwqf-zrenMwRBJvXVQ47UZ1K02In0OpHuC1NFyL9GD6wQgR2O5kK5V4zhpVTz21QoifdpjPaRZf~lI0AD3zzlTaOLEGQ~HTiko78fhtwWTgBu3mTObx49ZNDbH2jYLC0nPmnd9fZq-C6KMHilJRAdDcZ4200OiIz3HakDTLfKZ0kvcnd8Wq~m-iU0JrKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>).<a reveal-id="dsx038-B42" data-open="dsx038-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a> The total number of transcription factors is comparable to that of v2.0.a1, but a few gene families have gained relatively more members, such as the B3 family from 67 to 78, MYB from 112 to 118, and bHLH from 90 to 101 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx038/3/suppl_data_dsx038.zip?Expires=2147483647&Signature=I3QaHcxw2cpoXFZuHY6o8jhHqE5S6J1CUZDrtDqh8UaclmGZdWqKCS64xe4ty2JGgYsezqePsCBR~xrQPD09iLOKdFU6OiI9QEzbhUz7PTkVdtgM8EictdusqAY6pQnramoMAMCwYVPzSrate3VZD2-2gYwqf-zrenMwRBJvXVQ47UZ1K02In0OpHuC1NFyL9GD6wQgR2O5kK5V4zhpVTz21QoifdpjPaRZf~lI0AD3zzlTaOLEGQ~HTiko78fhtwWTgBu3mTObx49ZNDbH2jYLC0nPmnd9fZq-C6KMHilJRAdDcZ4200OiIz3HakDTLfKZ0kvcnd8Wq~m-iU0JrKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>). There are 1,055 protein kinase encoding genes in v2.0.a2, which are 90 more in the older annotation (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx038/3/suppl_data_dsx038.zip?Expires=2147483647&Signature=I3QaHcxw2cpoXFZuHY6o8jhHqE5S6J1CUZDrtDqh8UaclmGZdWqKCS64xe4ty2JGgYsezqePsCBR~xrQPD09iLOKdFU6OiI9QEzbhUz7PTkVdtgM8EictdusqAY6pQnramoMAMCwYVPzSrate3VZD2-2gYwqf-zrenMwRBJvXVQ47UZ1K02In0OpHuC1NFyL9GD6wQgR2O5kK5V4zhpVTz21QoifdpjPaRZf~lI0AD3zzlTaOLEGQ~HTiko78fhtwWTgBu3mTObx49ZNDbH2jYLC0nPmnd9fZq-C6KMHilJRAdDcZ4200OiIz3HakDTLfKZ0kvcnd8Wq~m-iU0JrKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>).</p><p>Disease resistance is an important agricultural trait for strawberry as well as other fruit crops, thus we also used the RGAugury pipeline<a reveal-id="dsx038-B43" data-open="dsx038-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a> to analyse <em>R</em>-genes and pattern-recognition receptors (PRRs), collectively called resistance gene analogs (RGAs), in the two annotation versions of Fvb. The RGAs are classified into four major families based on conserved domains and motifs in this pipeline, namely NBS encoding, membrane associated RLP (receptor-like protein) and RLK (receptor-like protein kinase), and TM-CC (transmembrane-coiled-coil).<a reveal-id="dsx038-B43" data-open="dsx038-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a> The first three families were further grouped into a few subfamilies. A total of 1,190 RGAs were identified, including 340 NBS encoding genes, 122 RLPs, 590 RLKs, and 138â€‰TM-CCs in v2.0.a2 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx038/3/suppl_data_dsx038.zip?Expires=2147483647&Signature=I3QaHcxw2cpoXFZuHY6o8jhHqE5S6J1CUZDrtDqh8UaclmGZdWqKCS64xe4ty2JGgYsezqePsCBR~xrQPD09iLOKdFU6OiI9QEzbhUz7PTkVdtgM8EictdusqAY6pQnramoMAMCwYVPzSrate3VZD2-2gYwqf-zrenMwRBJvXVQ47UZ1K02In0OpHuC1NFyL9GD6wQgR2O5kK5V4zhpVTz21QoifdpjPaRZf~lI0AD3zzlTaOLEGQ~HTiko78fhtwWTgBu3mTObx49ZNDbH2jYLC0nPmnd9fZq-C6KMHilJRAdDcZ4200OiIz3HakDTLfKZ0kvcnd8Wq~m-iU0JrKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>). Although the gene number in the NBS encoding family is comparable between the two annotations, the TNL subfamily has much more members (from 17 to 79), while the TX subfamily has fewer members (from 124 to 59) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx038/3/suppl_data_dsx038.zip?Expires=2147483647&Signature=I3QaHcxw2cpoXFZuHY6o8jhHqE5S6J1CUZDrtDqh8UaclmGZdWqKCS64xe4ty2JGgYsezqePsCBR~xrQPD09iLOKdFU6OiI9QEzbhUz7PTkVdtgM8EictdusqAY6pQnramoMAMCwYVPzSrate3VZD2-2gYwqf-zrenMwRBJvXVQ47UZ1K02In0OpHuC1NFyL9GD6wQgR2O5kK5V4zhpVTz21QoifdpjPaRZf~lI0AD3zzlTaOLEGQ~HTiko78fhtwWTgBu3mTObx49ZNDbH2jYLC0nPmnd9fZq-C6KMHilJRAdDcZ4200OiIz3HakDTLfKZ0kvcnd8Wq~m-iU0JrKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>). The size of the RLP and RLK families doesnâ€™t change much. However, the gene number in the TM-CC family is significantly decreased, from 192 genes in v2.0.a1 to 138 genes in v2.0.a2.</p><p>With the advantage of strand-specific RNA-seq reads, we were able to distinguish the sense and antisense transcripts from one locus. One example of such a locus having overlapping transcripts derived from opposite strands is gene02467, which codes for a tRNA ligase. This gene is located in one intron of an extremely long gene called gene02466 that encodes the ATP-binding cassette transporter A1 (<a reveal-id="dsx038-F3" data-open="dsx038-F3" class="link link-reveal link-table xref-fig">Fig. 3a</a>). By contrast, the same locus was previously annotated to have four genes in v2.0.a1 (<a reveal-id="dsx038-F3" data-open="dsx038-F3" class="link link-reveal link-table xref-fig">Fig. 3a</a>). <em>FvSOC1</em>, the homolog of <em>AtSOC1</em> (<em>SUPPRESSOR OF THE OVEREXPRESSION OF CONSTANS</em>), is a key negative regulator of flowering and a positive regulator of runnering in <em>F. vesca.</em><a reveal-id="dsx038-B44" data-open="dsx038-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a><sup>,</sup><em>FvSOC1</em> does not exist in v2.0.a1. However, the modified gene19424 with a brand new gene model in v2.0.a2 has exactly the same sequence as the EST of <em>FvSOC1</em> (accession number in NCBI: JF806634) (<a reveal-id="dsx038-F3" data-open="dsx038-F3" class="link link-reveal link-table xref-fig">Fig. 3b</a>). In addition, some gene models were fine-tuned in v2.0.a2 to have a higher annotation quality. For instance, <em>FvTFL1</em>, another crucial floral repressor in <em>F.</em><em>vesca</em>,<a reveal-id="dsx038-B45" data-open="dsx038-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a><sup>,</sup><a reveal-id="dsx038-B46" data-open="dsx038-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a> has a new translation start site producing a shorter protein, which becomes consistent with the published sequence in NCBI (accession number: JF806631) (<a reveal-id="dsx038-F3" data-open="dsx038-F3" class="link link-reveal link-table xref-fig">Fig. 3c</a>). A few other mis-annotated genes encoding glutathione S-transferases (GSTs), the red/far-red photoreceptor phytochrome B, and a bHLH transcription factor were PCR amplified, cloned and sequenced to confirm the v2.0.a2 annotation (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx038/3/suppl_data_dsx038.zip?Expires=2147483647&Signature=I3QaHcxw2cpoXFZuHY6o8jhHqE5S6J1CUZDrtDqh8UaclmGZdWqKCS64xe4ty2JGgYsezqePsCBR~xrQPD09iLOKdFU6OiI9QEzbhUz7PTkVdtgM8EictdusqAY6pQnramoMAMCwYVPzSrate3VZD2-2gYwqf-zrenMwRBJvXVQ47UZ1K02In0OpHuC1NFyL9GD6wQgR2O5kK5V4zhpVTz21QoifdpjPaRZf~lI0AD3zzlTaOLEGQ~HTiko78fhtwWTgBu3mTObx49ZNDbH2jYLC0nPmnd9fZq-C6KMHilJRAdDcZ4200OiIz3HakDTLfKZ0kvcnd8Wq~m-iU0JrKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx038/3/suppl_data_dsx038.zip?Expires=2147483647&Signature=I3QaHcxw2cpoXFZuHY6o8jhHqE5S6J1CUZDrtDqh8UaclmGZdWqKCS64xe4ty2JGgYsezqePsCBR~xrQPD09iLOKdFU6OiI9QEzbhUz7PTkVdtgM8EictdusqAY6pQnramoMAMCwYVPzSrate3VZD2-2gYwqf-zrenMwRBJvXVQ47UZ1K02In0OpHuC1NFyL9GD6wQgR2O5kK5V4zhpVTz21QoifdpjPaRZf~lI0AD3zzlTaOLEGQ~HTiko78fhtwWTgBu3mTObx49ZNDbH2jYLC0nPmnd9fZq-C6KMHilJRAdDcZ4200OiIz3HakDTLfKZ0kvcnd8Wq~m-iU0JrKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Table S6</a></span>). Taken together, not only were a large proportion of pre-existing genes corrected, but also novel genes with potentially important roles were discovered and incorporated into the new annotation. </p>                        <a id="112482934" scrollto-destination="112482934"></a>
<div id="dsx038-F3" data-id="dsx038-F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx038/3/m_dsx038f3.png?Expires=2147483647&Signature=ruZjngCxU3l793YAH96pUn3PKhRYsM6Y-gB57Zr1l5yM7-MBn24nme506R54N-v2AgcsTF-07Z9jVHUqcYTgJjAEqXufVqhlhJskxbfo9Yb7HoJRD1xcDZP27gmBXLdHLlSBB0ts1kRlEpQ75CpLAPkJgCpt4AJ8tPUUikQ7hQvYNqQJzu-0QQg2KGPXstw1rbNL4ZlDAtDX56pFDeKiX8pZywq6-tArKgpkh4dCfUbFFQwV1WDw018qBjDJBQNy3TvOtubaLibbbMBoJzNvWiAF10acgRLB9w-3C5Jx6K0H9xnCkwbpja4AVL0gPm74mjK7GGhasK0VH3~wnymwog__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Examples of three loci with improved annotations. (a) IGV view of the RNA-seq reads for the overlapping genes (gene02466 and gene02467) with opposite transcription directions. (b) The gene model of gene19424 (FvSOC1) becomes dramatically different in new annotation. (c) The gene model of gene30276 (FvTFL1) is modified in new annotation. Grey peaks indicate the read coverage. Pink bars indicate reverse strand reads. Light blue bars indicate forward strand reads. Blue bars indicate exons in v2.0.a1. Orange bars indicate exons in the reverse strand in v2.0.a2. Green bars indicate exons in the forward strand in v2.0.a2." path-from-xml="dsx038f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112482934/dsx038f3.tif" path-from-xml="dsx038f3.tif" target="_blank">View large</a><a section="112482934" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx038/3/dsx038f3.png?Expires=2147483647&Signature=nQc8QlGLx9S21bVgUlnppD9Q2lIcpjLoRNZfw3yKM64tsKE2pHXc8kr52jVvpR~39oVXWwdvDuo8fQUwcO~8QJW--ejsQgr1pK1PykkPInY1hxAQiJbs-7KEkt4aZE4TPpzpmuik-iBwAaXbwrnhHiieKWhaNp4JjHL7myd8gZfLvqWggExWmvE5ButIaNXLnuMA5KlATKhFd0qr3dPdWFOlzVIiQxqPioxVDifg4ccTdgZsrbAXdx4~xx0u9mAkRG931bxghnuLb9wcNOazp7lLyxvxytICVD~XmL-Uwpv99BrQuaxJyoLEVpjQOG--dvlQxqUFTuHWDNXAGBxqbQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112482934&ar=4191345&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx038f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Examples of three loci with improved annotations. (a) IGV view of the RNA-seq reads for the overlapping genes (gene02466 and gene02467) with opposite transcription directions. (b) The gene model of gene19424 (<em>FvSOC1</em>) becomes dramatically different in new annotation. (c) The gene model of gene30276 (<em>FvTFL1</em>) is modified in new annotation. Grey peaks indicate the read coverage. Pink bars indicate reverse strand reads. Light blue bars indicate forward strand reads. Blue bars indicate exons in v2.0.a1. Orange bars indicate exons in the reverse strand in v2.0.a2. Green bars indicate exons in the forward strand in v2.0.a2.</p></div></div><div content-id="dsx038-F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx038/3/m_dsx038f3.png?Expires=2147483647&Signature=ruZjngCxU3l793YAH96pUn3PKhRYsM6Y-gB57Zr1l5yM7-MBn24nme506R54N-v2AgcsTF-07Z9jVHUqcYTgJjAEqXufVqhlhJskxbfo9Yb7HoJRD1xcDZP27gmBXLdHLlSBB0ts1kRlEpQ75CpLAPkJgCpt4AJ8tPUUikQ7hQvYNqQJzu-0QQg2KGPXstw1rbNL4ZlDAtDX56pFDeKiX8pZywq6-tArKgpkh4dCfUbFFQwV1WDw018qBjDJBQNy3TvOtubaLibbbMBoJzNvWiAF10acgRLB9w-3C5Jx6K0H9xnCkwbpja4AVL0gPm74mjK7GGhasK0VH3~wnymwog__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Examples of three loci with improved annotations. (a) IGV view of the RNA-seq reads for the overlapping genes (gene02466 and gene02467) with opposite transcription directions. (b) The gene model of gene19424 (FvSOC1) becomes dramatically different in new annotation. (c) The gene model of gene30276 (FvTFL1) is modified in new annotation. Grey peaks indicate the read coverage. Pink bars indicate reverse strand reads. Light blue bars indicate forward strand reads. Blue bars indicate exons in v2.0.a1. Orange bars indicate exons in the reverse strand in v2.0.a2. Green bars indicate exons in the forward strand in v2.0.a2." path-from-xml="dsx038f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112482934/dsx038f3.tif" path-from-xml="dsx038f3.tif" target="_blank">View large</a><a section="112482934" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx038/3/dsx038f3.png?Expires=2147483647&Signature=nQc8QlGLx9S21bVgUlnppD9Q2lIcpjLoRNZfw3yKM64tsKE2pHXc8kr52jVvpR~39oVXWwdvDuo8fQUwcO~8QJW--ejsQgr1pK1PykkPInY1hxAQiJbs-7KEkt4aZE4TPpzpmuik-iBwAaXbwrnhHiieKWhaNp4JjHL7myd8gZfLvqWggExWmvE5ButIaNXLnuMA5KlATKhFd0qr3dPdWFOlzVIiQxqPioxVDifg4ccTdgZsrbAXdx4~xx0u9mAkRG931bxghnuLb9wcNOazp7lLyxvxytICVD~XmL-Uwpv99BrQuaxJyoLEVpjQOG--dvlQxqUFTuHWDNXAGBxqbQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112482934&ar=4191345&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx038f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Examples of three loci with improved annotations. (a) IGV view of the RNA-seq reads for the overlapping genes (gene02466 and gene02467) with opposite transcription directions. (b) The gene model of gene19424 (<em>FvSOC1</em>) becomes dramatically different in new annotation. (c) The gene model of gene30276 (<em>FvTFL1</em>) is modified in new annotation. Grey peaks indicate the read coverage. Pink bars indicate reverse strand reads. Light blue bars indicate forward strand reads. Blue bars indicate exons in v2.0.a1. Orange bars indicate exons in the reverse strand in v2.0.a2. Green bars indicate exons in the forward strand in v2.0.a2.</p></div></div>                            <h3 scrollto-destination=112482940 id="112482940" class="section-title" >3.3. Alternatively spliced genes in v2.0.a2</h3>
<p>Alternative splicing (AS) is an important regulatory mechanism at post-transcriptional level and is prevalent in animals as well as plants.<a reveal-id="dsx038-B47" data-open="dsx038-B47" class="link link-ref link-reveal xref-bibr"><sup>47</sup></a><sup>,</sup><a reveal-id="dsx038-B48" data-open="dsx038-B48" class="link link-ref link-reveal xref-bibr"><sup>48</sup></a> However, the v2.0.a1 annotation does not provide any information about the isoforms alternatively spliced from the pre-mRNAs. The v2.0.a2 annotation takes into account and provides the specific information on the alternatively spliced transcripts predicted by our analysis pipeline (<a reveal-id="dsx038-F1" data-open="dsx038-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>). To increase the prediction accuracy, weakly expressed transcripts in the RNA-seq datasets were removed during the assembly using Stringtie.<a reveal-id="dsx038-B23" data-open="dsx038-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> A total of 16,705 alternative transcripts were detected genome-wide from 7,370 multiexon genes, accounting for 21.98% of the 33,538 protein-coding genes. The average transcript number per gene is 1.51. As we used a more stringent filtering strategy, much fewer isoforms were presented in v2.0.a2 than that of our previous study (47,000 isoforms from 13,591 alternatively spliced genes, accounting for 41.6% of all the protein coding genes).<a reveal-id="dsx038-B7" data-open="dsx038-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a></p><p>Among the AS genes, about half of them (46.85%) possess only two isoforms, 23.19% genes possess three isoforms, and 1.28% genes have &gt;10 isoforms (<a reveal-id="dsx038-F4" data-open="dsx038-F4" class="link link-reveal link-table xref-fig">Fig. 4a</a>). 7,370 AS genes showed enriched GO terms in all kinds of metabolic processes (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx038/3/suppl_data_dsx038.zip?Expires=2147483647&Signature=I3QaHcxw2cpoXFZuHY6o8jhHqE5S6J1CUZDrtDqh8UaclmGZdWqKCS64xe4ty2JGgYsezqePsCBR~xrQPD09iLOKdFU6OiI9QEzbhUz7PTkVdtgM8EictdusqAY6pQnramoMAMCwYVPzSrate3VZD2-2gYwqf-zrenMwRBJvXVQ47UZ1K02In0OpHuC1NFyL9GD6wQgR2O5kK5V4zhpVTz21QoifdpjPaRZf~lI0AD3zzlTaOLEGQ~HTiko78fhtwWTgBu3mTObx49ZNDbH2jYLC0nPmnd9fZq-C6KMHilJRAdDcZ4200OiIz3HakDTLfKZ0kvcnd8Wq~m-iU0JrKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S7</a></span>), consistent with our previous studies.<a reveal-id="dsx038-B7" data-open="dsx038-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a><sup>,</sup><a reveal-id="dsx038-B49" data-open="dsx038-B49" class="link link-ref link-reveal xref-bibr"><sup>49</sup></a> To better characterize alternative splicing (AS) in the new annotation, AStalavista was used to group the AS events into a few major types.<a reveal-id="dsx038-B30" data-open="dsx038-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> 3,434 events are of alternative 3â€² splice sites (A3), 1,351 events are of alternative 5â€² splice sites (A5), 4,245 events are due to intron retention (IR), and 1,029 events result from exon skipping (ES) (<a reveal-id="dsx038-F4" data-open="dsx038-F4" class="link link-reveal link-table xref-fig">Fig. 4b</a>). Consistent with other studies,<a reveal-id="dsx038-B7" data-open="dsx038-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a><sup>,</sup><a reveal-id="dsx038-B49" data-open="dsx038-B49" class="link link-ref link-reveal xref-bibr"><sup>49</sup></a> the percentage of IR (32.39%) is the greatest, while the percentage of ES (7.85%) is the least. </p>                        <a id="112482946" scrollto-destination="112482946"></a>
<div id="dsx038-F4" data-id="dsx038-F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx038/3/m_dsx038f4.png?Expires=2147483647&Signature=Kg-xjBXSo4EdH4AiVxIerp4IRD9fkA5WPv-312memVpJ94jKsMCkvXMtwJ9SGKV1hwuRT5Sv2TsxYvmUXnLwDSLxuAJial0QGBGDEj4yE2blvLbdgmuWhn8RmsAgQwIf6x4hB11ECsh1UwKSzJyvz2n~K7NisASsAE50LzBTqehlPBtNZQHaYWs-09ObXZZqOoKLpmCx5TCyI1j3ZJCx3wi4BOkJJBlv92n-RYMNpjotzDuMiqnVo~~hQIsrsbPNhGBhm0T7xkoxM-iYDbaQIrqY5NGIl2dvQl8tMPj3C5D5ySJ2hbRkYpTqb7JP3zsIn2VKFEGQ4yPCNKvz7ryx4w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Features of the alternative splicing events in v2.0.a2. (a) Pie chart showing the percentage of genes with different number of isoforms among the 7,370 alternatively spliced genes. Different colour indicates genes with different number of isoforms per gene. (b) Summary of the different types of alternative splicing events that generate alternative transcripts in v2.0.a2." path-from-xml="dsx038f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112482946/dsx038f4.tif" path-from-xml="dsx038f4.tif" target="_blank">View large</a><a section="112482946" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx038/3/dsx038f4.png?Expires=2147483647&Signature=fMVdBr2KBbJHCd5KCQ8r5UluwLf-X11p7jTXm6Z3SIbpGPnFIZqlZJdAA1CwO99rUpnS1iY6FV6xAHaBl-d9YTCVAOboRw-bQACBK3ZU0MvtxFjc7PJJo9m~gjyiUCe1q7GDMMwgVfqJ9sGKG~zy72mpxU~ZJVskQH26TJTLnAZSgHtJQxAbXplLzuXp9rpMWtSjhuM-sOmeSNccNk23UuM6s4a5dEDSOHO86JyGSc-MOriEy-g-Yjly-YsEQrFow26oSBcuoUpQ4EpdakKahYu48y0a3Gx0jG5Mm0ANNqlcixMCzq44peaIwMX5QsVg2sIPrkSS693QXnUnnvcTWA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112482946&ar=4191345&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx038f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Features of the alternative splicing events in v2.0.a2. (a) Pie chart showing the percentage of genes with different number of isoforms among the 7,370 alternatively spliced genes. Different colour indicates genes with different number of isoforms per gene. (b) Summary of the different types of alternative splicing events that generate alternative transcripts in v2.0.a2.</p></div></div><div content-id="dsx038-F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx038/3/m_dsx038f4.png?Expires=2147483647&Signature=Kg-xjBXSo4EdH4AiVxIerp4IRD9fkA5WPv-312memVpJ94jKsMCkvXMtwJ9SGKV1hwuRT5Sv2TsxYvmUXnLwDSLxuAJial0QGBGDEj4yE2blvLbdgmuWhn8RmsAgQwIf6x4hB11ECsh1UwKSzJyvz2n~K7NisASsAE50LzBTqehlPBtNZQHaYWs-09ObXZZqOoKLpmCx5TCyI1j3ZJCx3wi4BOkJJBlv92n-RYMNpjotzDuMiqnVo~~hQIsrsbPNhGBhm0T7xkoxM-iYDbaQIrqY5NGIl2dvQl8tMPj3C5D5ySJ2hbRkYpTqb7JP3zsIn2VKFEGQ4yPCNKvz7ryx4w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Features of the alternative splicing events in v2.0.a2. (a) Pie chart showing the percentage of genes with different number of isoforms among the 7,370 alternatively spliced genes. Different colour indicates genes with different number of isoforms per gene. (b) Summary of the different types of alternative splicing events that generate alternative transcripts in v2.0.a2." path-from-xml="dsx038f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112482946/dsx038f4.tif" path-from-xml="dsx038f4.tif" target="_blank">View large</a><a section="112482946" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx038/3/dsx038f4.png?Expires=2147483647&Signature=fMVdBr2KBbJHCd5KCQ8r5UluwLf-X11p7jTXm6Z3SIbpGPnFIZqlZJdAA1CwO99rUpnS1iY6FV6xAHaBl-d9YTCVAOboRw-bQACBK3ZU0MvtxFjc7PJJo9m~gjyiUCe1q7GDMMwgVfqJ9sGKG~zy72mpxU~ZJVskQH26TJTLnAZSgHtJQxAbXplLzuXp9rpMWtSjhuM-sOmeSNccNk23UuM6s4a5dEDSOHO86JyGSc-MOriEy-g-Yjly-YsEQrFow26oSBcuoUpQ4EpdakKahYu48y0a3Gx0jG5Mm0ANNqlcixMCzq44peaIwMX5QsVg2sIPrkSS693QXnUnnvcTWA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112482946&ar=4191345&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx038f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Features of the alternative splicing events in v2.0.a2. (a) Pie chart showing the percentage of genes with different number of isoforms among the 7,370 alternatively spliced genes. Different colour indicates genes with different number of isoforms per gene. (b) Summary of the different types of alternative splicing events that generate alternative transcripts in v2.0.a2.</p></div></div>                            <h3 scrollto-destination=112482951 id="112482951" class="section-title" >3.4. Non-coding RNAs</h3>
<p>Non-coding RNAs are the integral part of the genome. To further improve the annotation of <em>F.</em><em>vesca</em> genome, the loci producing small RNAs and lncRNAs were both integrated into the new annotation v2.0.a2.</p>                            <h4 scrollto-destination=112482955 id="112482955" class="section-title" >3.4.1. Small RNAs</h4>
<p>Thus far, most of the small RNAs with known functions are 21â€“24nt long, such as small interfering RNAs (siRNAs), miRNAs, phasiRNAs, and tasiRNAs. These small RNAs regulate the expression of target genes through transcriptional gene silencing (TGS) or post-transcriptional gene silencing (PTGS). Among them, 171 miRNAs, including 44 novel miRNAs and 127 conserved miRNAs, have been previously identified in <em>F. vesca</em>,<a reveal-id="dsx038-B5" data-open="dsx038-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> but their locations in the genome are lacking. In this study, the stem-loop precursor sequences of these miRNAs were all uniquely aligned to Fvb, and the coordinates for the stem-loop and mature miRNA sequences were integrated into v2.0.a2 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx038/3/suppl_data_dsx038.zip?Expires=2147483647&Signature=I3QaHcxw2cpoXFZuHY6o8jhHqE5S6J1CUZDrtDqh8UaclmGZdWqKCS64xe4ty2JGgYsezqePsCBR~xrQPD09iLOKdFU6OiI9QEzbhUz7PTkVdtgM8EictdusqAY6pQnramoMAMCwYVPzSrate3VZD2-2gYwqf-zrenMwRBJvXVQ47UZ1K02In0OpHuC1NFyL9GD6wQgR2O5kK5V4zhpVTz21QoifdpjPaRZf~lI0AD3zzlTaOLEGQ~HTiko78fhtwWTgBu3mTObx49ZNDbH2jYLC0nPmnd9fZq-C6KMHilJRAdDcZ4200OiIz3HakDTLfKZ0kvcnd8Wq~m-iU0JrKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S8</a></span>).</p><p>Besides, siRNAs of 24nt are most widespread in the genome.<a reveal-id="dsx038-B6" data-open="dsx038-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a> Here, we analyzed a total of 9 small RNA-seq libraries generated from open flower, 4 DPA (days post anthesis)-seed, 10 DPA-seed, 4 DPA-carpel wall, 10 DPA-carpel wall, flower bud, young leaf, young seedling, and 10 DPA-fruit receptacle in <em>F. vesca</em> from a previous study.<a reveal-id="dsx038-B5" data-open="dsx038-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> ShortStack,<a reveal-id="dsx038-B36" data-open="dsx038-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> a popular pipeline used for small RNA annotation in <em>Arabidopsis</em> and other species,<a reveal-id="dsx038-B49 dsx038-B50 dsx038-B51" data-open="dsx038-B49 dsx038-B50 dsx038-B51" class="link link-ref link-reveal xref-bibr"><sup>49â€“51</sup></a> was employed to identify small RNA clusters with 20â€“24nt small RNAs in Fvb. Finally, 51,714 small RNA clusters were identified with 24nt small RNAs as the most abundant type, accounting for 98.46% out of all small RNA clusters (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx038/3/suppl_data_dsx038.zip?Expires=2147483647&Signature=I3QaHcxw2cpoXFZuHY6o8jhHqE5S6J1CUZDrtDqh8UaclmGZdWqKCS64xe4ty2JGgYsezqePsCBR~xrQPD09iLOKdFU6OiI9QEzbhUz7PTkVdtgM8EictdusqAY6pQnramoMAMCwYVPzSrate3VZD2-2gYwqf-zrenMwRBJvXVQ47UZ1K02In0OpHuC1NFyL9GD6wQgR2O5kK5V4zhpVTz21QoifdpjPaRZf~lI0AD3zzlTaOLEGQ~HTiko78fhtwWTgBu3mTObx49ZNDbH2jYLC0nPmnd9fZq-C6KMHilJRAdDcZ4200OiIz3HakDTLfKZ0kvcnd8Wq~m-iU0JrKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S9</a></span>). Furthermore, 15,705 (30.4%) and 1,544 (2.99%) of these clusters overlap with protein-coding genes and lncRNAs, respectively. When the gene density of small RNA clusters was plotted across the seven chromosomes, it roughly negatively correlated with that of protein-coding genes in both v2.0.a1 and v2.0.a2 (<a reveal-id="dsx038-F5" data-open="dsx038-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>). </p>                        <a id="112482961" scrollto-destination="112482961"></a>
<div id="dsx038-F5" data-id="dsx038-F5" class="fig fig-section"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx038/3/m_dsx038f5.png?Expires=2147483647&Signature=1SEfviwvPL4Q9kyw2s7cnCMppNzjfb48v3UY4ufIp68b1eLkKeFGIW8qu3dD3M2YXBYvN5udX3o7B~mmlV0ZnDebRSR2LrE6~3fLr3MR1TmErPFxdTEceL1pV6lzMYwR10TwIHCBiyJZZEYtYwkooz5c94S605cHchKQNwi6he7mRs2gi3tp12~Dr2zA3b2j9XfoZRB1xnbt1UX4In5fN3Snnxa93FUoH~zs9Phl4ubGrtLx8aZKRByq6wKmwciu5O41iezGtMMH7ecbDbaytWgNRduw2Ly8jeaD1KNjynb6~F8xqyNDgQ9iRLhb3S0l8odvPT~etbqrkq~~KhrZvg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Circular representation of genome-wide distribution of the genomic features. Tracks from outer to inner circles indicate: protein-coding (PC) gene models in v2.0.a2, v2.0.a1, lncRNA, miRNA, and smallRNA loci, respectively. Colour density represents gene density. Darker colour indicates a higher density." path-from-xml="dsx038f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112482961/dsx038f5.tif" path-from-xml="dsx038f5.tif" target="_blank">View large</a><a section="112482961" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx038/3/dsx038f5.png?Expires=2147483647&Signature=LHGt~B-Lphd3r-IPOoswzjXqyuJ6R69mVc4M8CwE17NOX1ED8xpGKyOqPY-wZY3Ftg7uWlVyA7d7JIdJ5QgJ9CGss8ZtLqvO33cTFdnucMiF6E8iqAiEFjxDYqyAY2fbSfdO6IgT~eL5apzXBlwOdUMOUPa~voBeEHPJPUqv8v9vTnpnK5MympOToC4s1k~xPysxYEKPcvzv-w634S2PJoRVLQcLBuy83fSMPyPYIGsDqK5wia0ZZL5sFDd4w-Uy08rTeIO2lHyoGilrYlMwaOUh8XHlA0tOsdZk0Hlf9ydFaKavxavdLV20-PRoVNJ7hsyF1ZN9XA0V7rySXqD7jA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112482961&ar=4191345&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx038f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Circular representation of genome-wide distribution of the genomic features. Tracks from outer to inner circles indicate: protein-coding (PC) gene models in v2.0.a2, v2.0.a1, lncRNA, miRNA, and smallRNA loci, respectively. Colour density represents gene density. Darker colour indicates a higher density.</p></div></div><div content-id="dsx038-F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx038/3/m_dsx038f5.png?Expires=2147483647&Signature=1SEfviwvPL4Q9kyw2s7cnCMppNzjfb48v3UY4ufIp68b1eLkKeFGIW8qu3dD3M2YXBYvN5udX3o7B~mmlV0ZnDebRSR2LrE6~3fLr3MR1TmErPFxdTEceL1pV6lzMYwR10TwIHCBiyJZZEYtYwkooz5c94S605cHchKQNwi6he7mRs2gi3tp12~Dr2zA3b2j9XfoZRB1xnbt1UX4In5fN3Snnxa93FUoH~zs9Phl4ubGrtLx8aZKRByq6wKmwciu5O41iezGtMMH7ecbDbaytWgNRduw2Ly8jeaD1KNjynb6~F8xqyNDgQ9iRLhb3S0l8odvPT~etbqrkq~~KhrZvg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Circular representation of genome-wide distribution of the genomic features. Tracks from outer to inner circles indicate: protein-coding (PC) gene models in v2.0.a2, v2.0.a1, lncRNA, miRNA, and smallRNA loci, respectively. Colour density represents gene density. Darker colour indicates a higher density." path-from-xml="dsx038f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112482961/dsx038f5.tif" path-from-xml="dsx038f5.tif" target="_blank">View large</a><a section="112482961" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx038/3/dsx038f5.png?Expires=2147483647&Signature=LHGt~B-Lphd3r-IPOoswzjXqyuJ6R69mVc4M8CwE17NOX1ED8xpGKyOqPY-wZY3Ftg7uWlVyA7d7JIdJ5QgJ9CGss8ZtLqvO33cTFdnucMiF6E8iqAiEFjxDYqyAY2fbSfdO6IgT~eL5apzXBlwOdUMOUPa~voBeEHPJPUqv8v9vTnpnK5MympOToC4s1k~xPysxYEKPcvzv-w634S2PJoRVLQcLBuy83fSMPyPYIGsDqK5wia0ZZL5sFDd4w-Uy08rTeIO2lHyoGilrYlMwaOUh8XHlA0tOsdZk0Hlf9ydFaKavxavdLV20-PRoVNJ7hsyF1ZN9XA0V7rySXqD7jA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112482961&ar=4191345&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx038f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Circular representation of genome-wide distribution of the genomic features. Tracks from outer to inner circles indicate: protein-coding (PC) gene models in v2.0.a2, v2.0.a1, lncRNA, miRNA, and smallRNA loci, respectively. Colour density represents gene density. Darker colour indicates a higher density.</p></div></div>                            <h4 scrollto-destination=112482968 id="112482968" class="section-title" >3.4.2. LncRNAs</h4>
<p>LncRNAs are defined as non-coding RNAs with a length &gt;200â€‰bp. To annotate lncRNAs in <em>F. vesca</em>, we used two independent methods to identify them and only retained the consensus ones (see Methods). Overall, we identified a total of 1,938 lncRNA loci with more than one exon in the <em>F. vesca</em> genome, corresponding to a total of 4,042 transcripts (2.1 isoforms per locus on average). The exon number distribution of lncRNAs showed that lncRNAs have fewer exons than protein coding genes. 53% of lncRNAs possess two or three exons, while only 32% of the protein-coding genes possess two or three exons (<a reveal-id="dsx038-F6" data-open="dsx038-F6" class="link link-reveal link-table xref-fig">Fig. 6a</a>). The gene length of lncRNAs varies from 201 to 12,722â€‰bp, and the mean length is 1,479â€‰bp. In this pipeline, the lncRNAs were first classified into two major groups, intergenic and genic, based on the location relative to protein-coding genes (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx038/3/suppl_data_dsx038.zip?Expires=2147483647&Signature=I3QaHcxw2cpoXFZuHY6o8jhHqE5S6J1CUZDrtDqh8UaclmGZdWqKCS64xe4ty2JGgYsezqePsCBR~xrQPD09iLOKdFU6OiI9QEzbhUz7PTkVdtgM8EictdusqAY6pQnramoMAMCwYVPzSrate3VZD2-2gYwqf-zrenMwRBJvXVQ47UZ1K02In0OpHuC1NFyL9GD6wQgR2O5kK5V4zhpVTz21QoifdpjPaRZf~lI0AD3zzlTaOLEGQ~HTiko78fhtwWTgBu3mTObx49ZNDbH2jYLC0nPmnd9fZq-C6KMHilJRAdDcZ4200OiIz3HakDTLfKZ0kvcnd8Wq~m-iU0JrKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>). Intergenic lncRNAs can be further classified into three subgroups: same_strand, located in the same strand with the neighbouring protein-coding gene; convergent, convergent transcription with the neighbouring protein-coding gene in different strands; divergent, divergent transcription with the neighbour protein-coding gene in different strands (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx038/3/suppl_data_dsx038.zip?Expires=2147483647&Signature=I3QaHcxw2cpoXFZuHY6o8jhHqE5S6J1CUZDrtDqh8UaclmGZdWqKCS64xe4ty2JGgYsezqePsCBR~xrQPD09iLOKdFU6OiI9QEzbhUz7PTkVdtgM8EictdusqAY6pQnramoMAMCwYVPzSrate3VZD2-2gYwqf-zrenMwRBJvXVQ47UZ1K02In0OpHuC1NFyL9GD6wQgR2O5kK5V4zhpVTz21QoifdpjPaRZf~lI0AD3zzlTaOLEGQ~HTiko78fhtwWTgBu3mTObx49ZNDbH2jYLC0nPmnd9fZq-C6KMHilJRAdDcZ4200OiIz3HakDTLfKZ0kvcnd8Wq~m-iU0JrKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>). Genic lncRNAs can also be classified into three subgroups: overlapping, lncRNA is longer at one end and shorter at the other end than the overlapped protein-coding gene; containing, lncRNA is longer than the overlapped protein-coding gene at both ends; nested, lncRNA is shorter than the overlapped protein-coding gene at both ends.<a reveal-id="dsx038-B33" data-open="dsx038-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> Among the 4,042 lncRNA transcripts, the intergenic type contains 1,413 same_strand, 873 convergent, and 786 divergent lncRNA transcripts; the genic type contains 689 overlapping, 74 containing, and 207 nested transcripts (<a reveal-id="dsx038-F6" data-open="dsx038-F6" class="link link-reveal link-table xref-fig">Fig. 6b;</a><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx038/3/suppl_data_dsx038.zip?Expires=2147483647&Signature=I3QaHcxw2cpoXFZuHY6o8jhHqE5S6J1CUZDrtDqh8UaclmGZdWqKCS64xe4ty2JGgYsezqePsCBR~xrQPD09iLOKdFU6OiI9QEzbhUz7PTkVdtgM8EictdusqAY6pQnramoMAMCwYVPzSrate3VZD2-2gYwqf-zrenMwRBJvXVQ47UZ1K02In0OpHuC1NFyL9GD6wQgR2O5kK5V4zhpVTz21QoifdpjPaRZf~lI0AD3zzlTaOLEGQ~HTiko78fhtwWTgBu3mTObx49ZNDbH2jYLC0nPmnd9fZq-C6KMHilJRAdDcZ4200OiIz3HakDTLfKZ0kvcnd8Wq~m-iU0JrKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx038/3/suppl_data_dsx038.zip?Expires=2147483647&Signature=I3QaHcxw2cpoXFZuHY6o8jhHqE5S6J1CUZDrtDqh8UaclmGZdWqKCS64xe4ty2JGgYsezqePsCBR~xrQPD09iLOKdFU6OiI9QEzbhUz7PTkVdtgM8EictdusqAY6pQnramoMAMCwYVPzSrate3VZD2-2gYwqf-zrenMwRBJvXVQ47UZ1K02In0OpHuC1NFyL9GD6wQgR2O5kK5V4zhpVTz21QoifdpjPaRZf~lI0AD3zzlTaOLEGQ~HTiko78fhtwWTgBu3mTObx49ZNDbH2jYLC0nPmnd9fZq-C6KMHilJRAdDcZ4200OiIz3HakDTLfKZ0kvcnd8Wq~m-iU0JrKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Table S10</a></span>). </p>                        <a id="112482977" scrollto-destination="112482977"></a>
<div id="dsx038-F6" data-id="dsx038-F6" class="fig fig-section"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx038/3/m_dsx038f6.png?Expires=2147483647&amp;Signature=yvZQWkcK9aweCGKk789AKTsK0VHnyP2e2sIqW-AAXRDWSN0H8m5pKQJgiq8u8owfNHd2eSw6VMv~mN-5knDAq7uMNFZ1yWxyoXVAO2Q1OXoW54sqvBmXbWLW3C7PkITk6lAEqbMVhz5e6N5d4TVN26s81TuyoOc-5kprP5pisYclDv~4QE59Tsh-yz3KjORfBla1sTW9oUyTAshR4irOUVbHNlqoMV2kS5bo343BvY3a5T~af9qRRcroAoDdofGUaF6D05NxtYnSl6tnJnpJNd~m5cCx39RVmHnKgT0MDYnzaQDRvMJkwVY7OVx7jJRDAJtT8jMyAy5H1zXFwjefxA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Identification and characterization of long noncoding RNAs in Fvb. Number of exons harboured by lncRNAs and protein-coding genes. (b) Classification of lncRNAs based on the location and transcription direction relative to adjacent protein-coding genes. The lncRNAs were grouped into two types: genic and intergenic. Each type was further grouped into three subtypes. The number above each bar indicates the number of genes in each subtype. The diagrams of each subtype are shown in Supplementary Fig. S2." path-from-xml="dsx038f6.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112482977/dsx038f6.tif" path-from-xml="dsx038f6.tif" target="_blank">View large</a><a section="112482977" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx038/3/dsx038f6.png?Expires=2147483647&amp;Signature=mKJ1eU3bOfDFs8z53BdDd61~F1XZYeKrLPCiKzfLHejP0KpOkFkLaBUGlsXLvUxg-MgMKCiDQB~JpPVeTVJuQ5-ai7hmCGeZgTwCUSAHpS8uMcqEvYVAdw3E7FGWuww7G8Vc1BYseRWh~SM2qTQ1ebrTcC3Jbusr59JvfZHqgz5pvwdnhGrbuqOQGrcVFnsQbAIPNeWI4VjjCl~7aJXfg4MW~ufDt12LedpPgs-bnAt5sPjyYPb6t2nvt-K2ucy0TCTCPDvvpd-EZmtQCJyXJcEQcrPaAeGP0T1ibz69tDfO7BfrIn9ojnzve~~duaCitpbXAqb51M4A5GHj-0O2zw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=112482977&amp;ar=4191345&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsx038f6.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Identification and characterization of long noncoding RNAs in Fvb. Number of exons harboured by lncRNAs and protein-coding genes. (b) Classification of lncRNAs based on the location and transcription direction relative to adjacent protein-coding genes. The lncRNAs were grouped into two types: genic and intergenic. Each type was further grouped into three subtypes. The number above each bar indicates the number of genes in each subtype. The diagrams of each subtype are shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx038/3/suppl_data_dsx038.zip?Expires=2147483647&amp;Signature=I3QaHcxw2cpoXFZuHY6o8jhHqE5S6J1CUZDrtDqh8UaclmGZdWqKCS64xe4ty2JGgYsezqePsCBR~xrQPD09iLOKdFU6OiI9QEzbhUz7PTkVdtgM8EictdusqAY6pQnramoMAMCwYVPzSrate3VZD2-2gYwqf-zrenMwRBJvXVQ47UZ1K02In0OpHuC1NFyL9GD6wQgR2O5kK5V4zhpVTz21QoifdpjPaRZf~lI0AD3zzlTaOLEGQ~HTiko78fhtwWTgBu3mTObx49ZNDbH2jYLC0nPmnd9fZq-C6KMHilJRAdDcZ4200OiIz3HakDTLfKZ0kvcnd8Wq~m-iU0JrKw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>.</p></div></div><div content-id="dsx038-F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx038/3/m_dsx038f6.png?Expires=2147483647&amp;Signature=yvZQWkcK9aweCGKk789AKTsK0VHnyP2e2sIqW-AAXRDWSN0H8m5pKQJgiq8u8owfNHd2eSw6VMv~mN-5knDAq7uMNFZ1yWxyoXVAO2Q1OXoW54sqvBmXbWLW3C7PkITk6lAEqbMVhz5e6N5d4TVN26s81TuyoOc-5kprP5pisYclDv~4QE59Tsh-yz3KjORfBla1sTW9oUyTAshR4irOUVbHNlqoMV2kS5bo343BvY3a5T~af9qRRcroAoDdofGUaF6D05NxtYnSl6tnJnpJNd~m5cCx39RVmHnKgT0MDYnzaQDRvMJkwVY7OVx7jJRDAJtT8jMyAy5H1zXFwjefxA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Identification and characterization of long noncoding RNAs in Fvb. Number of exons harboured by lncRNAs and protein-coding genes. (b) Classification of lncRNAs based on the location and transcription direction relative to adjacent protein-coding genes. The lncRNAs were grouped into two types: genic and intergenic. Each type was further grouped into three subtypes. The number above each bar indicates the number of genes in each subtype. The diagrams of each subtype are shown in Supplementary Fig. S2." path-from-xml="dsx038f6.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112482977/dsx038f6.tif" path-from-xml="dsx038f6.tif" target="_blank">View large</a><a section="112482977" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx038/3/dsx038f6.png?Expires=2147483647&amp;Signature=mKJ1eU3bOfDFs8z53BdDd61~F1XZYeKrLPCiKzfLHejP0KpOkFkLaBUGlsXLvUxg-MgMKCiDQB~JpPVeTVJuQ5-ai7hmCGeZgTwCUSAHpS8uMcqEvYVAdw3E7FGWuww7G8Vc1BYseRWh~SM2qTQ1ebrTcC3Jbusr59JvfZHqgz5pvwdnhGrbuqOQGrcVFnsQbAIPNeWI4VjjCl~7aJXfg4MW~ufDt12LedpPgs-bnAt5sPjyYPb6t2nvt-K2ucy0TCTCPDvvpd-EZmtQCJyXJcEQcrPaAeGP0T1ibz69tDfO7BfrIn9ojnzve~~duaCitpbXAqb51M4A5GHj-0O2zw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=112482977&amp;ar=4191345&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsx038f6.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Identification and characterization of long noncoding RNAs in Fvb. Number of exons harboured by lncRNAs and protein-coding genes. (b) Classification of lncRNAs based on the location and transcription direction relative to adjacent protein-coding genes. The lncRNAs were grouped into two types: genic and intergenic. Each type was further grouped into three subtypes. The number above each bar indicates the number of genes in each subtype. The diagrams of each subtype are shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx038/3/suppl_data_dsx038.zip?Expires=2147483647&amp;Signature=I3QaHcxw2cpoXFZuHY6o8jhHqE5S6J1CUZDrtDqh8UaclmGZdWqKCS64xe4ty2JGgYsezqePsCBR~xrQPD09iLOKdFU6OiI9QEzbhUz7PTkVdtgM8EictdusqAY6pQnramoMAMCwYVPzSrate3VZD2-2gYwqf-zrenMwRBJvXVQ47UZ1K02In0OpHuC1NFyL9GD6wQgR2O5kK5V4zhpVTz21QoifdpjPaRZf~lI0AD3zzlTaOLEGQ~HTiko78fhtwWTgBu3mTObx49ZNDbH2jYLC0nPmnd9fZq-C6KMHilJRAdDcZ4200OiIz3HakDTLfKZ0kvcnd8Wq~m-iU0JrKw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>.</p></div></div>                            <h2 scrollto-destination=112482984 id="112482984" class="section-title" >4. Conclusions</h2>
<p>The <em>F. vesca</em> genome was first released with annotations of merely protein coding genes that were mainly derived from <em>ab initio</em> predictions.<a reveal-id="dsx038-B2" data-open="dsx038-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> Then, an updated genome of high quality called Fvb was re-assembled based on genetic linkage mapping with the same annotation.<a reveal-id="dsx038-B9" data-open="dsx038-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> RNA-seq datasets generated from strawberry fruits have been previously utilized for <em>F. vesca</em> genome reannotation,<a reveal-id="dsx038-B8" data-open="dsx038-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> however, it was based on limited number and type of tissue samples, short Illumina reads, and was again restricted to protein-coding genes. With the recent advancement in genome sequencing, especially the availability of third generation long read sequencing data, and accumulation of large number of RNA-seq datasets from diverse tissues, creating a new and improved re-annotation for the most recent genome assembly is timely. Here, we developed an optimized pipeline to re-annotate the strawberry Fvb genome taking advantage of PacBio full-length transcripts and an extensive dataset of RNA-seq libraries. This new annotation, v2.0.a2, updated the <em>F. vesca</em> gene models and includes new information ranging from 5â€² and 3â€² UTRs to alternatively spliced transcripts. In addition, non-coding genes, including small RNAs and lncRNAs, were also integrated into this new annotation. To conclude, our new annotation is much more complete than the older annotation and will provide a valuable resource for the strawberry and the <em>Rosaceae</em> research community as well as for comparative and functional studies in flowering plants.</p>                            <h2 scrollto-destination=112482988 id="112482988" class="section-title" >5. Data availability</h2>
<p><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx038/3/suppl_data_dsx038.zip?Expires=2147483647&Signature=I3QaHcxw2cpoXFZuHY6o8jhHqE5S6J1CUZDrtDqh8UaclmGZdWqKCS64xe4ty2JGgYsezqePsCBR~xrQPD09iLOKdFU6OiI9QEzbhUz7PTkVdtgM8EictdusqAY6pQnramoMAMCwYVPzSrate3VZD2-2gYwqf-zrenMwRBJvXVQ47UZ1K02In0OpHuC1NFyL9GD6wQgR2O5kK5V4zhpVTz21QoifdpjPaRZf~lI0AD3zzlTaOLEGQ~HTiko78fhtwWTgBu3mTObx49ZNDbH2jYLC0nPmnd9fZq-C6KMHilJRAdDcZ4200OiIz3HakDTLfKZ0kvcnd8Wq~m-iU0JrKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span> is the gtf file of the new Fvb annotation v2.0.a2, which is also available in GDR (<a class="link link-uri" href="https://www.rosaceae.org/" target="_blank">https://www.rosaceae.org/</a> (5 September 2017, date last accessed)) and SGR (<a class="link link-uri" href="http://bioinformatics.towson.edu/strawberry/" target="_blank">http://bioinformatics.towson.edu/strawberry/ (5 September 2017, date last accessed)</a>).</p>                            <h2 scrollto-destination=112482993 id="112482993" class="backacknowledgements-title" >Acknowledgements</h2>
<p>The authors would like to thank Dr. Guogui Ning for the help on establishing the analysis pipeline. This work was supported by the National Natural Science Foundation of China (31572098 to C.K.) and the Scientific and Technological Self-innovation Foundation of Huazhong Agricultural University (2014RC005 to Z.L. and 2014RC017 to C.K.).</p>                            <h3 scrollto-destination=112482997 id="112482997" class="section-title" >Author contributions</h3>
<p>Conceived and designed the experiments: Y.L. and C.K. Analysed the data: Y.L., W.W., and M.P. Performed the experiments: J.F. and H.L. Wrote the paper: C.K., Y.L., and Z.L.</p>                            <h3 scrollto-destination=112483001 id="112483001" class="section-title" >Supplementary data</h3>
<p><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx038/3/suppl_data_dsx038.zip?Expires=2147483647&Signature=I3QaHcxw2cpoXFZuHY6o8jhHqE5S6J1CUZDrtDqh8UaclmGZdWqKCS64xe4ty2JGgYsezqePsCBR~xrQPD09iLOKdFU6OiI9QEzbhUz7PTkVdtgM8EictdusqAY6pQnramoMAMCwYVPzSrate3VZD2-2gYwqf-zrenMwRBJvXVQ47UZ1K02In0OpHuC1NFyL9GD6wQgR2O5kK5V4zhpVTz21QoifdpjPaRZf~lI0AD3zzlTaOLEGQ~HTiko78fhtwWTgBu3mTObx49ZNDbH2jYLC0nPmnd9fZq-C6KMHilJRAdDcZ4200OiIz3HakDTLfKZ0kvcnd8Wq~m-iU0JrKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary data</a></span> are available at <em>DNARES</em> online.</p>                            <h3 scrollto-destination=112483005 id="112483005" class="section-title" >Conflict of interest</h3>
<p>None declared.</p>                            <h2 scrollto-destination=112483010 id="112483010" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsx038-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx038-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Liston</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Cronn</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Ashman</div> <div class="given-names">T. L.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Fragaria: a genus with deep historical roots and ripe for evolutionary and ecological insights</div>. <div class="source">Am. J. Bot</div>Â ., <div class="volume">101</div>, <div class="fpage">1686</div>â€“<div class="lpage">99</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fragaria%3A%20a%20genus%20with%20deep%20historical%20roots%20and%20ripe%20for%20evolutionary%20and%20ecological%20insights&amp;author=A.%20Liston&amp;author=R.%20Cronn&amp;author=T.%20L.%20Ashman&amp;publication_year=2014&amp;journal=Am.%20J.%20Bot&amp;volume=101&amp;pages=1686-99" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3732/ajb.1400140" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3732%2Fajb.1400140" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3732%2Fajb.1400140"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25326614" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx038-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx038-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Shulaev</div> <div class="given-names">V.</div></div>, <div class="name string-name"><div class="surname">Sargent</div> <div class="given-names">D. J.</div></div>, <div class="name string-name"><div class="surname">Crowhurst</div> <div class="given-names">R. N.</div></div></div>, et al. Â  <div class="year">2011</div>, 
<div class="article-title">The genome of woodland strawberry (<em>Fragaria vesca</em>)</div>. <div class="source">Nat. Genet</div>Â ., <div class="volume">43</div>, <div class="fpage">109</div>â€“<div class="lpage">16</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20of%20woodland%20strawberry%20(Fragaria%20vesca)&amp;author=V.%20Shulaev&amp;author=D.%20J.%20Sargent&amp;author=R.%20N.%20Crowhurst&amp;publication_year=2011&amp;journal=Nat.%20Genet&amp;volume=43&amp;pages=109-16" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.740" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.740" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.740"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21186353" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx038-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx038-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kang</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Darwish</div> <div class="given-names">O.</div></div>, <div class="name string-name"><div class="surname">Geretz</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Shahan</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Alkharouf</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">Z.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Genome-scale transcriptomic insights into early-stage fruit development in woodland strawberry <em>Fragaria vesca</em></div>. <div class="source">Plant Cell</div>Â , <div class="volume">25</div>, <div class="fpage">1960</div>â€“<div class="lpage">78</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-scale%20transcriptomic%20insights%20into%20early-stage%20fruit%20development%20in%20woodland%20strawberry%20Fragaria%20vesca&amp;author=C.%20Kang&amp;author=O.%20Darwish&amp;author=A.%20Geretz&amp;author=R.%20Shahan&amp;author=N.%20Alkharouf&amp;author=Z.%20Liu&amp;publication_year=2013&amp;journal=Plant%20Cell&amp;volume=25&amp;pages=1960-78" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.113.111732" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.113.111732" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.113.111732"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23898027" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx038-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx038-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hollender</div> <div class="given-names">C. A.</div></div>, <div class="name string-name"><div class="surname">Kang</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Darwish</div> <div class="given-names">O.</div></div></div>, et al. Â  <div class="year">2014</div>, 
<div class="article-title">Floral transcriptomes in woodland strawberry uncover developing receptacle and anther gene networks</div>. <div class="source">Plant Physiol</div>Â ., <div class="volume">165</div>, <div class="fpage">1062</div>â€“<div class="lpage">75</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Floral%20transcriptomes%20in%20woodland%20strawberry%20uncover%20developing%20receptacle%20and%20anther%20gene%20networks&amp;author=C.%20A.%20Hollender&amp;author=C.%20Kang&amp;author=O.%20Darwish&amp;publication_year=2014&amp;journal=Plant%20Physiol&amp;volume=165&amp;pages=1062-75" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.114.237529" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.114.237529" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.114.237529"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24828307" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx038-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx038-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Xia</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Ye</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Meyers</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">Z.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Novel and recently evolved miRNA clusters regulate expansive F-box gene networks through phasiRNAs in wild diploid strawberry</div>. <div class="source">Plant Physiol</div>Â ., <div class="volume">169</div>, <div class="fpage">594</div>â€“<div class="lpage">610</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Novel%20and%20recently%20evolved%20miRNA%20clusters%20regulate%20expansive%20F-box%20gene%20networks%20through%20phasiRNAs%20in%20wild%20diploid%20strawberry&amp;author=R.%20Xia&amp;author=S.%20Ye&amp;author=Z.%20Liu&amp;author=B.%20Meyers&amp;author=Z.%20Liu&amp;publication_year=2015&amp;journal=Plant%20Physiol&amp;volume=169&amp;pages=594-610" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.15.00253" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.15.00253" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.15.00253"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26143249" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx038-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx038-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kang</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">Z.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Global identification and analysis of long non-coding RNAs in diploid strawberry <em>Fragaria vesca</em> during flower and fruit development</div>. <div class="source">BMC Genomics</div>Â , <div class="volume">16</div>, <div class="fpage">815</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Global%20identification%20and%20analysis%20of%20long%20non-coding%20RNAs%20in%20diploid%20strawberry%20Fragaria%20vesca%20during%20flower%20and%20fruit%20development&amp;author=C.%20Kang&amp;author=Z.%20Liu&amp;publication_year=2015&amp;journal=BMC%20Genomics&amp;volume=16&amp;pages=815" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12864-015-2014-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12864-015-2014-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12864-015-2014-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26481460" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx038-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx038-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Dai</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Hu</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Kang</div> <div class="given-names">C.</div></div></div> <div class="year">2017</div>, 
<div class="article-title">Global identification of alternative splicing via comparative analysis of SMRT- and Illumina-based RNA-seq in strawberry</div>. <div class="source">Plant J</div>Â ., <div class="volume">90</div>, <div class="fpage">164</div>â€“<div class="lpage">76</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Global%20identification%20of%20alternative%20splicing%20via%20comparative%20analysis%20of%20SMRT-%20and%20Illumina-based%20RNA-seq%20in%20strawberry&amp;author=Y.%20Li&amp;author=C.%20Dai&amp;author=C.%20Hu&amp;author=Z.%20Liu&amp;author=C.%20Kang&amp;publication_year=2017&amp;journal=Plant%20J&amp;volume=90&amp;pages=164-76" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.2017.90.issue-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2017.90.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.2017.90.issue-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27997733" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx038-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx038-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Darwish</div> <div class="given-names">O.</div></div>, <div class="name string-name"><div class="surname">Shahan</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Slovin</div> <div class="given-names">J. P.</div></div>, <div class="name string-name"><div class="surname">Alkharouf</div> <div class="given-names">N. W.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Re-annotation of the woodland strawberry (<em>Fragaria vesca</em>) genome</div>. <div class="source">BMC Genomics</div>Â , <div class="volume">16</div>, <div class="fpage">29</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Re-annotation%20of%20the%20woodland%20strawberry%20(Fragaria%20vesca)%20genome&amp;author=O.%20Darwish&amp;author=R.%20Shahan&amp;author=Z.%20Liu&amp;author=J.%20P.%20Slovin&amp;author=N.%20W.%20Alkharouf&amp;publication_year=2015&amp;journal=BMC%20Genomics&amp;volume=16&amp;pages=29" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12864-015-1221-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12864-015-1221-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12864-015-1221-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25623424" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx038-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx038-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tennessen</div> <div class="given-names">J. A.</div></div>, <div class="name string-name"><div class="surname">Govindarajulu</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Ashman</div> <div class="given-names">T. L.</div></div>, <div class="name string-name"><div class="surname">Liston</div> <div class="given-names">A.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Evolutionary origins and dynamics of octoploid strawberry subgenomes revealed by dense targeted capture linkage maps</div>. <div class="source">Genome Biol. Evol</div>Â ., <div class="volume">6</div>, <div class="fpage">3295</div>â€“<div class="lpage">313</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolutionary%20origins%20and%20dynamics%20of%20octoploid%20strawberry%20subgenomes%20revealed%20by%20dense%20targeted%20capture%20linkage%20maps&amp;author=J.%20A.%20Tennessen&amp;author=R.%20Govindarajulu&amp;author=T.%20L.%20Ashman&amp;author=A.%20Liston&amp;publication_year=2014&amp;journal=Genome%20Biol.%20Evol&amp;volume=6&amp;pages=3295-313" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/gbe/evu261" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fgbe%2Fevu261" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fgbe%2Fevu261"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25477420" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx038-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx038-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hawkins</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Caruana</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Schiksnis</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">Z.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Genome-scale DNA variant analysis and functional validation of a SNP underlying yellow fruit color in wild strawberry</div>. <div class="source">Sci. Rep</div>Â ., <div class="volume">6</div>, <div class="fpage">29017</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-scale%20DNA%20variant%20analysis%20and%20functional%20validation%20of%20a%20SNP%20underlying%20yellow%20fruit%20color%20in%20wild%20strawberry&amp;author=C.%20Hawkins&amp;author=J.%20Caruana&amp;author=E.%20Schiksnis&amp;author=Z.%20Liu&amp;publication_year=2016&amp;journal=Sci.%20Rep&amp;volume=6&amp;pages=29017" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/srep29017" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsrep29017" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsrep29017"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27377763" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx038-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx038-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Minoche</div> <div class="given-names">A. E.</div></div>, <div class="name string-name"><div class="surname">Dohm</div> <div class="given-names">J. C.</div></div>, <div class="name string-name"><div class="surname">Schneider</div> <div class="given-names">J.</div></div></div>, et al. Â  <div class="year">2015</div>, 
<div class="article-title">Exploiting single-molecule transcript sequencing for eukaryotic gene prediction</div>. <div class="source">Genome Biol</div>Â ., <div class="volume">16</div>, <div class="fpage">184</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Exploiting%20single-molecule%20transcript%20sequencing%20for%20eukaryotic%20gene%20prediction&amp;author=A.%20E.%20Minoche&amp;author=J.%20C.%20Dohm&amp;author=J.%20Schneider&amp;publication_year=2015&amp;journal=Genome%20Biol&amp;volume=16&amp;pages=184" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s13059-015-0729-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs13059-015-0729-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs13059-015-0729-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26328666" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx038-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx038-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Dong</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">J.</div></div></div>, et al. Â  <div class="year">2015</div>, 
<div class="article-title">Single-molecule real-time transcript sequencing facilitates common wheat genome annotation and grain transcriptome research</div>. <div class="source">BMC Genomics</div>Â , <div class="volume">16</div>, <div class="fpage">1039</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Single-molecule%20real-time%20transcript%20sequencing%20facilitates%20common%20wheat%20genome%20annotation%20and%20grain%20transcriptome%20research&amp;author=L.%20Dong&amp;author=H.%20Liu&amp;author=J.%20Zhang&amp;publication_year=2015&amp;journal=BMC%20Genomics&amp;volume=16&amp;pages=1039" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12864-015-2257-y" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12864-015-2257-y" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12864-015-2257-y"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26645802" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx038-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx038-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Xu</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Yuan</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Ai</div> <div class="given-names">C.</div></div></div>, et al. Â  <div class="year">2017</div>, 
<div class="article-title">uORF-mediated translation allows engineered plant disease resistance without fitness costs</div>. <div class="source">Nature</div>Â , <div class="volume">545</div>, <div class="fpage">491</div>â€“<div class="lpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=uORF-mediated%20translation%20allows%20engineered%20plant%20disease%20resistance%20without%20fitness%20costs&amp;author=G.%20Xu&amp;author=M.%20Yuan&amp;author=C.%20Ai&amp;publication_year=2017&amp;journal=Nature&amp;volume=545&amp;pages=491-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature22372" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature22372" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature22372"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28514448" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx038-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx038-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Haas</div> <div class="given-names">B. J.</div></div>, <div class="name string-name"><div class="surname">Delcher</div> <div class="given-names">A. L.</div></div>, <div class="name string-name"><div class="surname">Mount</div> <div class="given-names">S. M.</div></div></div>, et al. Â  <div class="year">2003</div>, 
<div class="article-title">Improving the Arabidopsis genome annotation using maximal transcript alignment assemblies</div>. <div class="source">Nucleic Acids Res</div>Â ., <div class="volume">31</div>, <div class="fpage">5654</div>â€“<div class="lpage">66</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Improving%20the%20Arabidopsis%20genome%20annotation%20using%20maximal%20transcript%20alignment%20assemblies&amp;author=B.%20J.%20Haas&amp;author=A.%20L.%20Delcher&amp;author=S.%20M.%20Mount&amp;publication_year=2003&amp;journal=Nucleic%20Acids%20Res&amp;volume=31&amp;pages=5654-66" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkg770" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkg770" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkg770"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14500829" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx038-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx038-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Holt</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Yandell</div> <div class="given-names">M.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">MAKER2: an annotation pipeline and genome-database management tool for second-generation genome projects</div>. <div class="source">BMC Bioinformatics</div>Â , <div class="volume">12, 491</div>.</div></div></div></div><div content-id="dsx038-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx038-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Stanke</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Diekhans</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Baertsch</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Haussler</div> <div class="given-names">D.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">Using native and syntenically mapped cDNA alignments to improve de novo gene finding</div>. <div class="source">Bioinformatics</div>Â , <div class="volume">24</div>, <div class="fpage">637</div>â€“<div class="lpage">44</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Using%20native%20and%20syntenically%20mapped%20cDNA%20alignments%20to%20improve%20de%20novo%20gene%20finding&amp;author=M.%20Stanke&amp;author=M.%20Diekhans&amp;author=R.%20Baertsch&amp;author=D.%20Haussler&amp;publication_year=2008&amp;journal=Bioinformatics&amp;volume=24&amp;pages=637-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btn013" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtn013" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtn013"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18218656" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx038-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx038-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Stanke</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Waack</div> <div class="given-names">S.</div></div></div> <div class="year">2003</div>, 
<div class="article-title">Gene prediction with a hidden Markov model and a new intron submodel</div>. <div class="source">Bioinformatics</div>Â , <div class="volume">19</div>, <div class="fpage">Ii215</div>â€“<div class="lpage">25</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20prediction%20with%20a%20hidden%20Markov%20model%20and%20a%20new%20intron%20submodel&amp;author=M.%20Stanke&amp;author=S.%20Waack&amp;publication_year=2003&amp;journal=Bioinformatics&amp;volume=19&amp;pages=Ii215-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btg1080" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtg1080" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtg1080"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14534192" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx038-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx038-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Toljamo</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Blande</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Karenlampi</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Kokko</div> <div class="given-names">H.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Reprogramming of strawberry (<em>Fragaria vesca</em>) root transcriptome in response to phytophthora cactorum</div>. <div class="source">PLoS One</div>Â , <div class="volume">11</div>, <div class="fpage">e0161078</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Reprogramming%20of%20strawberry%20(Fragaria%20vesca)%20root%20transcriptome%20in%20response%20to%20phytophthora%20cactorum&amp;author=A.%20Toljamo&amp;author=D.%20Blande&amp;author=S.%20Karenlampi&amp;author=H.%20Kokko&amp;publication_year=2016&amp;journal=PLoS%20One&amp;volume=11&amp;pages=e0161078" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0161078" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0161078" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0161078"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27518577" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx038-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx038-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Salmela</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Rivals</div> <div class="given-names">E.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">LoRDEC: accurate and efficient long read error correction</div>. <div class="source">Bioinformatics</div>Â , <div class="volume">30</div>, <div class="fpage">3506</div>â€“<div class="lpage">14</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=LoRDEC%3A%20accurate%20and%20efficient%20long%20read%20error%20correction&amp;author=L.%20Salmela&amp;author=E.%20Rivals&amp;publication_year=2014&amp;journal=Bioinformatics&amp;volume=30&amp;pages=3506-14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btu538" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtu538" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtu538"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25165095" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx038-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx038-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wu</div> <div class="given-names">T. D.</div></div>, <div class="name string-name"><div class="surname">Watanabe</div> <div class="given-names">C. K.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">GMAP: a genomic mapping and alignment program for mRNA and EST sequences</div>. <div class="source">Bioinformatics</div>Â , <div class="volume">21</div>, <div class="fpage">1859</div>â€“<div class="lpage">75</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GMAP%3A%20a%20genomic%20mapping%20and%20alignment%20program%20for%20mRNA%20and%20EST%20sequences&amp;author=T.%20D.%20Wu&amp;author=C.%20K.%20Watanabe&amp;publication_year=2005&amp;journal=Bioinformatics&amp;volume=21&amp;pages=1859-75" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bti310" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbti310" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbti310"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15728110" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx038-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx038-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kim</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Pertea</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Trapnell</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Pimentel</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Kelley</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Salzberg</div> <div class="given-names">S. L.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">TopHat2: accurate alignment of transcriptomes in the presence of insertions, deletions and gene fusions</div>. <div class="source">Genome Biol</div>Â ., <div class="volume">14</div>, <div class="fpage">R36</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=TopHat2%3A%20accurate%20alignment%20of%20transcriptomes%20in%20the%20presence%20of%20insertions%2C%20deletions%20and%20gene%20fusions&amp;author=D.%20Kim&amp;author=G.%20Pertea&amp;author=C.%20Trapnell&amp;author=H.%20Pimentel&amp;author=R.%20Kelley&amp;author=S.%20L.%20Salzberg&amp;publication_year=2013&amp;journal=Genome%20Biol&amp;volume=14&amp;pages=R36" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2013-14-4-r36" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2013-14-4-r36" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2013-14-4-r36"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23618408" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx038-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx038-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Handsaker</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Wysoker</div> <div class="given-names">A.</div></div></div>, et al. Â  <div class="year">2009</div>, 
<div class="article-title">The Sequence Alignment/Map format and SAMtools</div>. <div class="source">Bioinformatics</div>Â , <div class="volume">25</div>, <div class="fpage">2078</div>â€“<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Sequence%20Alignment%2FMap%20format%20and%20SAMtools&amp;author=H.%20Li&amp;author=B.%20Handsaker&amp;author=A.%20Wysoker&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=2078-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp352" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp352" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp352"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19505943" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx038-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx038-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Pertea</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Pertea</div> <div class="given-names">G. M.</div></div>, <div class="name string-name"><div class="surname">Antonescu</div> <div class="given-names">C. M.</div></div>, <div class="name string-name"><div class="surname">Chang</div> <div class="given-names">T. C.</div></div>, <div class="name string-name"><div class="surname">Mendell</div> <div class="given-names">J. T.</div></div>, <div class="name string-name"><div class="surname">Salzberg</div> <div class="given-names">S. L.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">StringTie enables improved reconstruction of a transcriptome from RNA-seq reads</div>. <div class="source">Nat. Biotechnol</div>Â ., <div class="volume">33</div>, <div class="fpage">290</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=StringTie%20enables%20improved%20reconstruction%20of%20a%20transcriptome%20from%20RNA-seq%20reads&amp;author=M.%20Pertea&amp;author=G.%20M.%20Pertea&amp;author=C.%20M.%20Antonescu&amp;author=T.%20C.%20Chang&amp;author=J.%20T.%20Mendell&amp;author=S.%20L.%20Salzberg&amp;publication_year=2015&amp;journal=Nat.%20Biotechnol&amp;volume=33&amp;pages=290" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.3122" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.3122" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.3122"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25690850" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx038-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx038-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Grabherr</div> <div class="given-names">M. G.</div></div>, <div class="name string-name"><div class="surname">Haas</div> <div class="given-names">B. J.</div></div>, <div class="name string-name"><div class="surname">Yassour</div> <div class="given-names">M.</div></div></div>, et al. Â  <div class="year">2011</div>, 
<div class="article-title">Full-length transcriptome assembly from RNA-Seq data without a reference genome</div>. <div class="source">Nat. Biotechnol</div>Â ., <div class="volume">29</div>, <div class="fpage">644</div>â€“<div class="lpage">52</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Full-length%20transcriptome%20assembly%20from%20RNA-Seq%20data%20without%20a%20reference%20genome&amp;author=M.%20G.%20Grabherr&amp;author=B.%20J.%20Haas&amp;author=M.%20Yassour&amp;publication_year=2011&amp;journal=Nat.%20Biotechnol&amp;volume=29&amp;pages=644-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.1883" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.1883" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.1883"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21572440" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx038-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx038-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Stanke</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Schoffmann</div> <div class="given-names">O.</div></div>, <div class="name string-name"><div class="surname">Morgenstern</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Waack</div> <div class="given-names">S.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">Gene prediction in eukaryotes with a generalized hidden Markov model that uses hints from external sources</div>. <div class="source">BMC Bioinformatics</div>Â , <div class="volume">7</div>, <div class="fpage">62</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20prediction%20in%20eukaryotes%20with%20a%20generalized%20hidden%20Markov%20model%20that%20uses%20hints%20from%20external%20sources&amp;author=M.%20Stanke&amp;author=O.%20Schoffmann&amp;author=B.%20Morgenstern&amp;author=S.%20Waack&amp;publication_year=2006&amp;journal=BMC%20Bioinformatics&amp;volume=7&amp;pages=62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-7-62" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-7-62" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-7-62"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16469098" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx038-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx038-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Slater</div> <div class="given-names">G. S.</div></div>, <div class="name string-name"><div class="surname">Birney</div> <div class="given-names">E.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">Automated generation of heuristics for biological sequence comparison</div>. <div class="source">BMC Bioinformatics</div>Â , <div class="volume">6</div>, <div class="fpage">31</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Automated%20generation%20of%20heuristics%20for%20biological%20sequence%20comparison&amp;author=G.%20S.%20Slater&amp;author=E.%20Birney&amp;publication_year=2005&amp;journal=BMC%20Bioinformatics&amp;volume=6&amp;pages=31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-6-31" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-6-31" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-6-31"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15713233" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx038-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx038-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Haas</div> <div class="given-names">B. J.</div></div>, <div class="name string-name"><div class="surname">Salzberg</div> <div class="given-names">S. L.</div></div>, <div class="name string-name"><div class="surname">Zhu</div> <div class="given-names">W.</div></div></div>, et al. Â  <div class="year">2008</div>, 
<div class="article-title">Automated eukaryotic gene structure annotation using EVidenceModeler and the program to assemble spliced alignments</div>. <div class="source">Genome Biol</div>Â ., <div class="volume">9</div>, <div class="fpage">R7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Automated%20eukaryotic%20gene%20structure%20annotation%20using%20EVidenceModeler%20and%20the%20program%20to%20assemble%20spliced%20alignments&amp;author=B.%20J.%20Haas&amp;author=S.%20L.%20Salzberg&amp;author=W.%20Zhu&amp;publication_year=2008&amp;journal=Genome%20Biol&amp;volume=9&amp;pages=R7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2008-9-1-r7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2008-9-1-r7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2008-9-1-r7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18190707" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx038-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx038-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lee</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Helt</div> <div class="given-names">G. A.</div></div>, <div class="name string-name"><div class="surname">Reese</div> <div class="given-names">J. T.</div></div></div>, et al. Â  <div class="year">2013</div>, 
<div class="article-title">Web Apollo: a web-based genomic annotation editing platform</div>. <div class="source">Genome Biol</div>Â ., <div class="volume">14</div>, <div class="fpage">R93</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Web%20Apollo%3A%20a%20web-based%20genomic%20annotation%20editing%20platform&amp;author=E.%20Lee&amp;author=G.%20A.%20Helt&amp;author=J.%20T.%20Reese&amp;publication_year=2013&amp;journal=Genome%20Biol&amp;volume=14&amp;pages=R93" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2013-14-8-r93" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2013-14-8-r93" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2013-14-8-r93"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24000942" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx038-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx038-B29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Stein</div> <div class="given-names">L. D.</div></div>, <div class="name string-name"><div class="surname">Mungall</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Shu</div> <div class="given-names">S. Q.</div></div></div>, et al. Â  <div class="year">2002</div>, 
<div class="article-title">The generic genome browser: a building block for a model organism system database</div>. <div class="source">Genome Res</div>Â ., <div class="volume">12</div>, <div class="fpage">1599</div>â€“<div class="lpage">610</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20generic%20genome%20browser%3A%20a%20building%20block%20for%20a%20model%20organism%20system%20database&amp;author=L.%20D.%20Stein&amp;author=C.%20Mungall&amp;author=S.%20Q.%20Shu&amp;publication_year=2002&amp;journal=Genome%20Res&amp;volume=12&amp;pages=1599-610" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.403602" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.403602" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.403602"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12368253" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx038-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx038-B30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Foissac</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Sammeth</div> <div class="given-names">M.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">ASTALAVISTA: dynamic and flexible analysis of alternative splicing events in custom gene datasets</div>. <div class="source">Nucleic Acids Res</div>Â ., <div class="volume">35</div>, <div class="fpage">W297</div>â€“<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=ASTALAVISTA%3A%20dynamic%20and%20flexible%20analysis%20of%20alternative%20splicing%20events%20in%20custom%20gene%20datasets&amp;author=S.%20Foissac&amp;author=M.%20Sammeth&amp;publication_year=2007&amp;journal=Nucleic%20Acids%20Res&amp;volume=35&amp;pages=W297-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkm311" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkm311" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkm311"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17485470" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx038-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx038-B31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Altschul</div> <div class="given-names">S. F.</div></div>, <div class="name string-name"><div class="surname">Gish</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Miller</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Myers</div> <div class="given-names">E. W.</div></div>, <div class="name string-name"><div class="surname">Lipman</div> <div class="given-names">D. J.</div></div></div> <div class="year">1990</div>, 
<div class="article-title">Basic local alignment search tool</div>. <div class="source">J. Mol. Biol</div>Â ., <div class="volume">215</div>, <div class="fpage">403</div>â€“<div class="lpage">10</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Basic%20local%20alignment%20search%20tool&amp;author=S.%20F.%20Altschul&amp;author=W.%20Gish&amp;author=W.%20Miller&amp;author=E.%20W.%20Myers&amp;author=D.%20J.%20Lipman&amp;publication_year=1990&amp;journal=J.%20Mol.%20Biol&amp;volume=215&amp;pages=403-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0022-2836(05)80360-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0022-2836(05)80360-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0022-2836(05)80360-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2231712" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx038-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx038-B32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kong</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Ye</div> <div class="given-names">Z.-Q.</div></div></div>, et al. Â  <div class="year">2007</div>, 
<div class="article-title">CPC: assess the protein-coding potential of transcripts using sequence features and support vector machine</div>. <div class="source">Nucleic Acids Res</div>Â ., <div class="volume">35</div>, <div class="fpage">W345</div>â€“<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CPC%3A%20assess%20the%20protein-coding%20potential%20of%20transcripts%20using%20sequence%20features%20and%20support%20vector%20machine&amp;author=L.%20Kong&amp;author=Y.%20Zhang&amp;author=Z.-Q.%20Ye&amp;publication_year=2007&amp;journal=Nucleic%20Acids%20Res&amp;volume=35&amp;pages=W345-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkm391" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkm391" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkm391"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17631615" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx038-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx038-B33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wucher</div> <div class="given-names">V.</div></div>, <div class="name string-name"><div class="surname">Legeai</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Hedan</div> <div class="given-names">B.</div></div></div>, et al. Â  <div class="year">2017</div>, 
<div class="article-title">FEELnc: a tool for long non-coding RNA annotation and its application to the dog transcriptome</div>. <div class="source">Nucleic Acids Res</div>Â ., <div class="volume">45</div>, <div class="fpage">e57</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=FEELnc%3A%20a%20tool%20for%20long%20non-coding%20RNA%20annotation%20and%20its%20application%20to%20the%20dog%20transcriptome&amp;author=V.%20Wucher&amp;author=F.%20Legeai&amp;author=B.%20Hedan&amp;publication_year=2017&amp;journal=Nucleic%20Acids%20Res&amp;volume=45&amp;pages=e57" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28053114" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx038-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx038-B34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Martin</div> <div class="given-names">M.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Cutadapt removes adapter sequences from high-throughput sequencing reads</div>. <div class="source">EMBnet J</div>Â ., <div class="volume">17</div>, <div class="fpage">10</div>â€“<div class="lpage">2</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cutadapt%20removes%20adapter%20sequences%20from%20high-throughput%20sequencing%20reads&amp;author=M.%20Martin&amp;publication_year=2011&amp;journal=EMBnet%20J&amp;volume=17&amp;pages=10-2" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.14806/ej.17.1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.14806%2Fej.17.1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.14806%2Fej.17.1"> </span></div>Â </div></div></div></div></div><div content-id="dsx038-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx038-B35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Langmead</div> <div class="given-names">B.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Aligning short sequencing reads with Bowtie</div>. <div class="source">Curr. Protoc. Bioinformatics</div>Â , doi: 10.1002/0471250953.bi1107s32.</div></div></div></div><div content-id="dsx038-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx038-B36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Axtell</div> <div class="given-names">M. J.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">ShortStack: comprehensive annotation and quantification of small RNA genes</div>. <div class="source">RNA</div>Â , <div class="volume">19</div>, <div class="fpage">740</div>â€“<div class="lpage">51</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=ShortStack%3A%20comprehensive%20annotation%20and%20quantification%20of%20small%20RNA%20genes&amp;author=M.%20J.%20Axtell&amp;publication_year=2013&amp;journal=RNA&amp;volume=19&amp;pages=740-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1261/rna.035279.112" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1261%2Frna.035279.112" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1261%2Frna.035279.112"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23610128" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx038-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx038-B37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Steijger</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Abril</div> <div class="given-names">J. F.</div></div>, <div class="name string-name"><div class="surname">EngstrÃ¶m</div> <div class="given-names">P. G.</div></div></div>, et al. Â  <div class="year">2013</div>, 
<div class="article-title">Assessment of transcript reconstruction methods for RNA-seq</div>. <div class="source">Nat. Methods</div>Â , <div class="volume">10</div>, <div class="fpage">1177</div>â€“<div class="lpage">84</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Assessment%20of%20transcript%20reconstruction%20methods%20for%20RNA-seq&amp;author=T.%20Steijger&amp;author=J.%20F.%20Abril&amp;author=P.%20G.%20Engstr%C3%B6m&amp;publication_year=2013&amp;journal=Nat.%20Methods&amp;volume=10&amp;pages=1177-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.2714" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.2714" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.2714"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24185837" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx038-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx038-B38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Robinson</div> <div class="given-names">J. T.</div></div>, <div class="name string-name"><div class="surname">Thorvaldsdottir</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Winckler</div> <div class="given-names">W.</div></div></div>, et al. Â  <div class="year">2011</div>, 
<div class="article-title">Integrative genomics viewer</div>. <div class="source">Nat. Biotechnol</div>Â ., <div class="volume">29</div>, <div class="fpage">24</div>â€“<div class="lpage">26</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Integrative%20genomics%20viewer&amp;author=J.%20T.%20Robinson&amp;author=H.%20Thorvaldsdottir&amp;author=W.%20Winckler&amp;publication_year=2011&amp;journal=Nat.%20Biotechnol&amp;volume=29&amp;pages=24-26" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.1754" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.1754" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.1754"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21221095" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx038-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx038-B39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Simao</div> <div class="given-names">F. A.</div></div>, <div class="name string-name"><div class="surname">Waterhouse</div> <div class="given-names">R. M.</div></div>, <div class="name string-name"><div class="surname">Ioannidis</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Kriventseva</div> <div class="given-names">E. V.</div></div>, <div class="name string-name"><div class="surname">Zdobnov</div> <div class="given-names">E. M.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs</div>. <div class="source">Bioinformatics</div>Â , <div class="volume">31</div>, <div class="fpage">3210</div>â€“<div class="lpage">2</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=BUSCO%3A%20assessing%20genome%20assembly%20and%20annotation%20completeness%20with%20single-copy%20orthologs&amp;author=F.%20A.%20Simao&amp;author=R.%20M.%20Waterhouse&amp;author=P.%20Ioannidis&amp;author=E.%20V.%20Kriventseva&amp;author=E.%20M.%20Zdobnov&amp;publication_year=2015&amp;journal=Bioinformatics&amp;volume=31&amp;pages=3210-2" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btv351" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtv351" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtv351"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26059717" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx038-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx038-B40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Jones</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Binns</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Chang</div> <div class="given-names">H.-Y.</div></div></div>, et al. Â  <div class="year">2014</div>, 
<div class="article-title">InterProScan 5: genome-scale protein function classification</div>. <div class="source">Bioinformatics</div>Â , <div class="volume">30</div>, <div class="fpage">1236</div>â€“<div class="lpage">40</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=InterProScan%205%3A%20genome-scale%20protein%20function%20classification&amp;author=P.%20Jones&amp;author=D.%20Binns&amp;author=H.-Y.%20Chang&amp;publication_year=2014&amp;journal=Bioinformatics&amp;volume=30&amp;pages=1236-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btu031" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtu031" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtu031"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24451626" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx038-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx038-B41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Conesa</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">GÃ¶tz</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">GarcÃ­a-GÃ³mez</div> <div class="given-names">J. M.</div></div>, <div class="name string-name"><div class="surname">Terol</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">TalÃ³n</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Robles</div> <div class="given-names">M.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">Blast2GO: a universal tool for annotation, visualization and analysis in functional genomics research</div>. <div class="source">Bioinformatics</div>Â , <div class="volume">21</div>, <div class="fpage">3674</div>â€“<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Blast2GO%3A%20a%20universal%20tool%20for%20annotation%2C%20visualization%20and%20analysis%20in%20functional%20genomics%20research&amp;author=A.%20Conesa&amp;author=S.%20G%C3%B6tz&amp;author=J.%20M.%20Garc%C3%ADa-G%C3%B3mez&amp;author=J.%20Terol&amp;author=M.%20Tal%C3%B3n&amp;author=M.%20Robles&amp;publication_year=2005&amp;journal=Bioinformatics&amp;volume=21&amp;pages=3674-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bti610" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbti610" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbti610"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16081474" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx038-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx038-B42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zheng</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Jiao</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Sun</div> <div class="given-names">H.</div></div></div>, et al. Â  <div class="year">2016</div>, 
<div class="article-title">iTAK: a program for genome-wide prediction and classification of plant transcription factors, transcriptional regulators, and protein kinases</div>. <div class="source">Mol Plant</div>Â , <div class="volume">9</div>, <div class="fpage">1667</div>â€“<div class="lpage">70</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=iTAK%3A%20a%20program%20for%20genome-wide%20prediction%20and%20classification%20of%20plant%20transcription%20factors%2C%20transcriptional%20regulators%2C%20and%20protein%20kinases&amp;author=Y.%20Zheng&amp;author=C.%20Jiao&amp;author=H.%20Sun&amp;publication_year=2016&amp;journal=Mol%20Plant&amp;volume=9&amp;pages=1667-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.molp.2016.09.014" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.molp.2016.09.014" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.molp.2016.09.014"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27717919" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx038-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx038-B43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Quan</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Jia</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Xiao</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Cloutier</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">You</div> <div class="given-names">F. M.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">RGAugury: a pipeline for genome-wide prediction of resistance gene analogs (RGAs) in plants</div>. <div class="source">BMC Genomics</div>Â , <div class="volume">17</div>, <div class="fpage">852</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RGAugury%3A%20a%20pipeline%20for%20genome-wide%20prediction%20of%20resistance%20gene%20analogs%20(RGAs)%20in%20plants&amp;author=P.%20Li&amp;author=X.%20Quan&amp;author=G.%20Jia&amp;author=J.%20Xiao&amp;author=S.%20Cloutier&amp;author=F.%20M.%20You&amp;publication_year=2016&amp;journal=BMC%20Genomics&amp;volume=17&amp;pages=852" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12864-016-3197-x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12864-016-3197-x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12864-016-3197-x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27806688" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx038-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx038-B44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Mouhu</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Kurokura</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Koskela</div> <div class="given-names">E. A.</div></div>, <div class="name string-name"><div class="surname">Albert</div> <div class="given-names">V. A.</div></div>, <div class="name string-name"><div class="surname">Elomaa</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">HytÃ¶nen</div> <div class="given-names">T.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">The <em>Fragaria vesca</em> homolog of SUPPRESSOR OF OVEREXPRESSION OF CONSTANS1 represses flowering and promotes vegetative growth</div>. <div class="source">Plant Cell</div>Â , <div class="volume">25</div>, <div class="fpage">3296</div>â€“<div class="lpage">310</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Fragaria%20vesca%20homolog%20of%20SUPPRESSOR%20OF%20OVEREXPRESSION%20OF%20CONSTANS1%20represses%20flowering%20and%20promotes%20vegetative%20growth&amp;author=K.%20Mouhu&amp;author=T.%20Kurokura&amp;author=E.%20A.%20Koskela&amp;author=V.%20A.%20Albert&amp;author=P.%20Elomaa&amp;author=T.%20Hyt%C3%B6nen&amp;publication_year=2013&amp;journal=Plant%20Cell&amp;volume=25&amp;pages=3296-310" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.113.115055" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.113.115055" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.113.115055"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24038650" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx038-B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx038-B45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Iwata</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Gaston</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Remay</div> <div class="given-names">A.</div></div></div>, et al. Â  <div class="year">2012</div>, 
<div class="article-title">The TFL1 homologue KSN is a regulator of continuous flowering in rose and strawberry</div>. <div class="source">Plant J</div>Â ., <div class="volume">69</div>, <div class="fpage">116</div>â€“<div class="lpage">25</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20TFL1%20homologue%20KSN%20is%20a%20regulator%20of%20continuous%20flowering%20in%20rose%20and%20strawberry&amp;author=H.%20Iwata&amp;author=A.%20Gaston&amp;author=A.%20Remay&amp;publication_year=2012&amp;journal=Plant%20J&amp;volume=69&amp;pages=116-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.2011.69.issue-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2011.69.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.2011.69.issue-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21895811" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx038-B46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx038-B46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Koskela</div> <div class="given-names">E. A.</div></div>, <div class="name string-name"><div class="surname">Mouhu</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Albani</div> <div class="given-names">M. C.</div></div></div>, et al. Â  <div class="year">2012</div>, 
<div class="article-title">Mutation in TERMINAL FLOWER1 reverses the photoperiodic requirement for flowering in the wild strawberry <em>Fragaria vesca</em></div>. <div class="source">Plant Physiol</div>Â ., <div class="volume">159</div>, <div class="fpage">1043</div>â€“<div class="lpage">54</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mutation%20in%20TERMINAL%20FLOWER1%20reverses%20the%20photoperiodic%20requirement%20for%20flowering%20in%20the%20wild%20strawberry%20Fragaria%20vesca&amp;author=E.%20A.%20Koskela&amp;author=K.%20Mouhu&amp;author=M.%20C.%20Albani&amp;publication_year=2012&amp;journal=Plant%20Physiol&amp;volume=159&amp;pages=1043-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.112.196659" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.112.196659" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.112.196659"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22566495" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx038-B47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx038-B47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lee</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Rio</div> <div class="given-names">D. C.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Mechanisms and regulation of alternative pre-mRNA splicing</div>. <div class="source">Annu. Rev. Biochem</div>Â ., <div class="volume">84</div>, <div class="fpage">291</div>â€“<div class="lpage">323</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mechanisms%20and%20regulation%20of%20alternative%20pre-mRNA%20splicing&amp;author=Y.%20Lee&amp;author=D.%20C.%20Rio&amp;publication_year=2015&amp;journal=Annu.%20Rev.%20Biochem&amp;volume=84&amp;pages=291-323" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev-biochem-060614-034316" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev-biochem-060614-034316" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev-biochem-060614-034316"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25784052" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx038-B48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx038-B48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Reddy</div> <div class="given-names">A. S.</div></div>, <div class="name string-name"><div class="surname">Marquez</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Kalyna</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Barta</div> <div class="given-names">A.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Complexity of the alternative splicing landscape in plants</div>. <div class="source">Plant Cell</div>Â , <div class="volume">25</div>, <div class="fpage">3657</div>â€“<div class="lpage">83</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complexity%20of%20the%20alternative%20splicing%20landscape%20in%20plants&amp;author=A.%20S.%20Reddy&amp;author=Y.%20Marquez&amp;author=M.%20Kalyna&amp;author=A.%20Barta&amp;publication_year=2013&amp;journal=Plant%20Cell&amp;volume=25&amp;pages=3657-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.113.117523" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.113.117523" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.113.117523"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24179125" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx038-B49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx038-B49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Cheng</div> <div class="given-names">C. Y.</div></div>, <div class="name string-name"><div class="surname">Krishnakumar</div> <div class="given-names">V.</div></div>, <div class="name string-name"><div class="surname">Chan</div> <div class="given-names">A. P.</div></div>, <div class="name string-name"><div class="surname">Thibaud-Nissen</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Schobel</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Town</div> <div class="given-names">C. D.</div></div></div> <div class="year">2017</div>, 
<div class="article-title">Araport11: a complete reannotation of the <em>Arabidopsis thaliana</em> reference genome</div>. <div class="source">Plant J</div>Â ., <div class="volume">89</div>, <div class="fpage">789</div>â€“<div class="lpage">804</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Araport11%3A%20a%20complete%20reannotation%20of%20the%20Arabidopsis%20thaliana%20reference%20genome&amp;author=C.%20Y.%20Cheng&amp;author=V.%20Krishnakumar&amp;author=A.%20P.%20Chan&amp;author=F.%20Thibaud-Nissen&amp;author=S.%20Schobel&amp;author=C.%20D.%20Town&amp;publication_year=2017&amp;journal=Plant%20J&amp;volume=89&amp;pages=789-804" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.13415" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.13415" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.13415"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27862469" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx038-B50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx038-B50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Jex</div> <div class="given-names">A. R.</div></div>, <div class="name string-name"><div class="surname">Nejsum</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Schwarz</div> <div class="given-names">E. M.</div></div></div>, et al. Â  <div class="year">2014</div>, 
<div class="article-title">Genome and transcriptome of the porcine whipworm <em>Trichuris suis</em></div>. <div class="source">Nat. Genet</div>Â ., <div class="volume">46</div>, <div class="fpage">701</div>â€“<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20and%20transcriptome%20of%20the%20porcine%20whipworm%20Trichuris%20suis&amp;author=A.%20R.%20Jex&amp;author=P.%20Nejsum&amp;author=E.%20M.%20Schwarz&amp;publication_year=2014&amp;journal=Nat.%20Genet&amp;volume=46&amp;pages=701-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.3012" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.3012" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.3012"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24929829" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx038-B51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx038-B51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Coruh</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Cho</div> <div class="given-names">S. H.</div></div>, <div class="name string-name"><div class="surname">Shahid</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">Q.</div></div>, <div class="name string-name"><div class="surname">Wierzbicki</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Axtell</div> <div class="given-names">M. J.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Comprehensive annotation of <em>Physcomitrella patens</em> small RNA loci reveals that the heterochromatic short interfering RNA pathway is largely conserved in land plants</div>. <div class="source">Plant Cell</div>Â , <div class="volume">27</div>, <div class="fpage">2148</div>â€“<div class="lpage">62</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comprehensive%20annotation%20of%20Physcomitrella%20patens%20small%20RNA%20loci%20reveals%20that%20the%20heterochromatic%20short%20interfering%20RNA%20pathway%20is%20largely%20conserved%20in%20land%20plants&amp;author=C.%20Coruh&amp;author=S.%20H.%20Cho&amp;author=S.%20Shahid&amp;author=Q.%20Liu&amp;author=A.%20Wierzbicki&amp;author=M.%20J.%20Axtell&amp;publication_year=2015&amp;journal=Plant%20Cell&amp;volume=27&amp;pages=2148-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.15.00228" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.15.00228" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.15.00228"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26209555" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">Â© The Author 2017. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=4191345&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx038/3/suppl_data_dsx038.zip?Expires=2147483647&amp;Signature=I3QaHcxw2cpoXFZuHY6o8jhHqE5S6J1CUZDrtDqh8UaclmGZdWqKCS64xe4ty2JGgYsezqePsCBR~xrQPD09iLOKdFU6OiI9QEzbhUz7PTkVdtgM8EictdusqAY6pQnramoMAMCwYVPzSrate3VZD2-2gYwqf-zrenMwRBJvXVQ47UZ1K02In0OpHuC1NFyL9GD6wQgR2O5kK5V4zhpVTz21QoifdpjPaRZf~lI0AD3zzlTaOLEGQ~HTiko78fhtwWTgBu3mTObx49ZNDbH2jYLC0nPmnd9fZq-C6KMHilJRAdDcZ4200OiIz3HakDTLfKZ0kvcnd8Wq~m-iU0JrKw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">2,696</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000425954100006&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">1</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsx038"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="4191345"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000425954100006&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f25%2f1%2f61%2f4191345">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000425954100006&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (1)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F25%2F1%2F61%2F4191345" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/4191345'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transpositionâ€“duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Genome re-annotation of the wild strawberry Fragaria vesca using extensive Illumina- and SMRT-based RNA-seq datasets" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsx038");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f25%2f1%2f61%2f4191345">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

