

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" /><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info = {"beacon":"bam.nr-data.net","errorBeacon":"bam.nr-data.net","licenseKey":"8d76ca6f26","applicationID":"29565620","transactionName":"ZQBRMkVUCEtRVkINXlxKcjVnGgdKRFxVCFQcBEAWTw==","queueTime":1,"applicationTime":749,"agent":"","atts":""}</script><script type="text/javascript">(window.NREUM||(NREUM={})).loader_config={xpid:"XAYBV15WGwEJUVdXAQMH"};window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{s.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(21),s={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(s.console=!0,o.indexOf("dev")!==-1&&(s.dev=!0),o.indexOf("nr_dev")!==-1&&(s.nrDev=!0))}catch(c){}s.nrDev&&i.on("internal-error",function(t){r(t.stack)}),s.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),s.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(s,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,s){try{l?l-=1:o(s||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,c.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:c.now();i("err",[t,n])}var i=t("handle"),a=t(22),s=t("ee"),c=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;c.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),c.xhrWrappable&&t(14),d=!0)}s.on("fn-start",function(t,e,n){d&&(l+=1)}),s.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),s.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),s.on("internal-error",function(t){i("ierr",[t,c.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){M++,N=y.hash,this[u]=g.now()}function o(){M--,y.hash!==N&&i(0,!0);var t=g.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+y,e])}function a(t,e){t.on(e,function(){this[e]=g.now()})}var s="-start",c="-end",f="-body",u="fn"+s,d="fn"+c,p="cb"+s,l="cb"+c,h="jsTime",m="fetch",v="addEventListener",w=window,y=w.location,g=t("loader");if(w[v]&&g.xhrWrappable){var b=t(10),x=t(11),E=t(8),O=t(6),P=t(13),R=t(7),T=t(14),L=t(9),j=t("ee"),S=j.get("tracer");t(15),g.features.spa=!0;var N,M=0;j.on(u,r),j.on(p,r),j.on(d,o),j.on(l,o),j.buffer([u,d,"xhr-done","xhr-resolved"]),O.buffer([u]),P.buffer(["setTimeout"+c,"clearTimeout"+s,u]),T.buffer([u,"new-xhr","send-xhr"+s]),R.buffer([m+s,m+"-done",m+f+s,m+f+c]),E.buffer(["newURL"]),b.buffer([u]),x.buffer(["propagate",p,l,"executor-err","resolve"+s]),S.buffer([u,"no-"+u]),L.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(T,"send-xhr"+s),a(j,"xhr-resolved"),a(j,"xhr-done"),a(R,m+s),a(R,m+"-done"),a(L,"new-jsonp"),a(L,"jsonp-end"),a(L,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,M>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),s=t(12),c="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",y="pushState",g=t("loader");g.features.stn=!0,t(8);var b=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof b&&(this.bstStart=g.now())}),o.on(v,function(t,e){var n=t[0];n instanceof b&&i("bst",[n,e,this.bstStart,g.now()])}),a.on(m,function(t,e,n){this.bstStart=g.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),this.bstType])}),s.on(m,function(){this.bstStart=g.now()}),s.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),"requestAnimationFrame"])}),o.on(y+l,function(t){this.time=g.now(),this.startPath=location.pathname+location.hash}),o.on(y+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+c]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+c]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+c]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){s.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),s=t(24)(a,!0),c=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=c(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?s(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=r.apply(this,arguments);return o.emit(n+"start",arguments,t),t.then(function(e){return o.emit(n+"end",[null,e],t),e},function(e){throw o.emit(n+"end",[e],t),e})})}var o=t("ee").get("fetch"),i=t(21);e.exports=o;var a=window,s="fetch-",c=s+"body-",f=["arrayBuffer","blob","json","text","formData"],u=a.Request,d=a.Response,p=a.fetch,l="prototype";u&&d&&p&&(i(f,function(t,e){r(u[l],e,c),r(d[l],e,c)}),r(a,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t(24)(r);e.exports=r,o.inPlace(window.history,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){c.emit("jsonp-error",[],p),c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=s(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),c.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function s(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var c=t("ee").get("jsonp"),f=t(24)(c);if(e.exports=c,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-"),c.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t(24)(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=s(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t(24),a=t("ee").get("promise"),s=i(a),c=t(21),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;c(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),s=f.resolve(i);return s}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),c(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=s(t[0],"resolve-",this),t[1]=s(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),s.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=s(t[0],"cb-",this),t[1]=s(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t(24)(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t(24)(i),s="setTimeout",c="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[s,"setImmediate"],s+d),a.inPlace(window,[c],c+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(c+u,r),i.on(s+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",s)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,y,"fn-",s)}function i(t){g.push(t),h&&(x?x.then(a):v?v(a):(E=-E,O.data=E))}function a(){for(var t=0;t<g.length;t++)r([],g[t]);g.length&&(g=[])}function s(t,e){return e}function c(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t(24)(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",y=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],g=[];e.exports=u;var b=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(c(l,b),b.prototype=l.prototype,d.inPlace(b.prototype,["open","send"],"-xhr-",s),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var x=m&&m.resolve();if(!v&&!m){var E=1,O=document.createTextNode(E);new h(a).observe(O,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<d;r++)t.removeEventListener(u[r],this.listener,!1);if(!e.aborted){if(n.duration=a.now()-this.startTime,4===t.readyState){e.status=t.status;var i=o(t,this.lastSize);if(i&&(n.rxSize=i),this.sameOrigin){var c=t.getResponseHeader("X-NewRelic-App-Data");c&&(e.cat=c.split(", ").pop())}}else e.status=0;n.cbTime=this.cbTime,f.emit("xhr-done",[t],t),s("xhr",[e,n,this.startTime])}}}function o(t,e){var n=t.responseType;if("json"===n&&null!==e)return e;var r="arraybuffer"===n||"blob"===n||"json"===n?t.response:t.responseText;return h(r)}function i(t,e){var n=c(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.sameOrigin=n.sameOrigin}var a=t("loader");if(a.xhrWrappable){var s=t("handle"),c=t(16),f=t("ee"),u=["load","error","abort","timeout"],d=u.length,p=t("id"),l=t(19),h=t(18),m=window.XMLHttpRequest;a.features.xhr=!0,t(14),f.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,l&&(l>34||l<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),f.on("open-xhr-start",function(t){this.params={method:t[0]},i(this,t[1]),this.metrics={}}),f.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid)}),f.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=h(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"===t.type&&(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{f.emit("internal-error",[n])}catch(r){}}};for(var s=0;s<d;s++)e.addEventListener(u[s],this.listener,!1)}),f.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),f.on("xhr-load-added",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),f.on("xhr-load-removed",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),f.on("addEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-added",[t[1],t[2]],e)}),f.on("removeEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-removed",[t[1],t[2]],e)}),f.on("fn-start",function(t,e,n){e instanceof m&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),f.on("fn-end",function(t,e){this.xhrCbStart&&f.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)})}},{}],16:[function(t,e,n){e.exports=function(t){var e=document.createElement("a"),n=window.location,r={};e.href=t,r.port=e.port;var o=e.href.split("://");!r.port&&o[1]&&(r.port=o[1].split("/")[0].split("@").pop().split(":")[1]),r.port&&"0"!==r.port||(r.port="https"===o[0]?"443":"80"),r.hostname=e.hostname||n.hostname,r.pathname=e.pathname,r.protocol=o[0],"/"!==r.pathname.charAt(0)&&(r.pathname="/"+r.pathname);var i=!e.protocol||":"===e.protocol||e.protocol===n.protocol,a=e.hostname===document.domain&&e.port===n.port;return r.sameOrigin=i&&(!e.hostname||a),r}},{}],17:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(s(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(21),s=t(22),c=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(c.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw c.emit("fn-err",[arguments,this,t],n),t}finally{c.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now()])}},{}],18:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],19:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],20:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var s=navigator.userAgent,c=s.match(a);c&&s.indexOf("Chrome")===-1&&s.indexOf("Chromium")===-1&&(o="Safari",i=c[1])}e.exports={agent:o,version:i,match:r}},{}],21:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],22:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],23:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],24:[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(22),a="nr@original",s=Object.prototype.hasOwnProperty,c=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,s,c;try{a=this,r=i(arguments),s="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],s])}u(e+"start",[r,a,o],s);try{return c=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],s),d}finally{u(e+"end",[r,a,c],s)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,s,c,f="-"===o.charAt(0);for(c=0;c<e.length;c++)s=e[c],a=t[s],r(a)||(t[s]=n(a,f?s+o:o,i,s))}function u(n,r,o){if(!c||e){var i=c;c=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}c=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)s.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?c(t,s,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),s=m(n),c=s.length,f=0;f<c;f++)s[f].apply(a,r);var d=u[g[n]];return d&&d.push([b,n,r,a]),a}}function l(t,e){y[t]=m(t).concat(e)}function h(t,e){var n=y[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return y[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",g[n]=e,e in u||(u[e]=[])})}var y={},g={},b={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return b}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var s="nr@context",c=t("gos"),f=t(21),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=x.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(g,function(e,n){t[e]||(t[e]=n)}),c("mark",["onload",a()+x.offset],null,"api");var n=l.createElement("script");n.src="https://"+t.agent,e.parentNode.insertBefore(n,e)}}function o(){"complete"===l.readyState&&i()}function i(){c("mark",["domContent",a()+x.offset],null,"api")}function a(){return O.exists&&performance.now?Math.round(performance.now()):(s=Math.max((new Date).getTime(),s))-x.offset}var s=(new Date).getTime(),c=t("handle"),f=t(21),u=t("ee"),d=t(20),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var y=""+location,g={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1099.min.js"},b=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),x=e.exports={offset:s,now:a,origin:y,features:{},xhrWrappable:b,userAgent:d};t(17),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),c("mark",["firstbyte",s],null,"api");var E=0,O=t(23)},{}]},{},["loader",2,15,5,3,4]);</script>
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/21/1/41/347328","@type":"ScholarlyArticle","name":"Development of 5123 Intron-Length Polymorphic Markers for Large-Scale Genotyping Applications in Foxtail Millet","datePublished":"2013-10-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/21/1","@type":"PublicationIssue","issueNumber":"1","datePublished":"2014-02-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dst039","keywords":["comparative mapping","foxtail millet (<italic>Setaria italica</italic> L.)","intron-length polymorphism (ILP)","physical mapping","transferability"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/21/1/41/347328","author":[{"name":"Muthamilarasan, Mehanathan","affiliation":"National Institute of Plant Genome Research (NIPGR) ,  Aruna Asaf Ali Marg, New Delhi 110 067 ,  India","@type":"Person"},{"name":"Venkata Suresh, B.","affiliation":"National Institute of Plant Genome Research (NIPGR) ,  Aruna Asaf Ali Marg, New Delhi 110 067 ,  India","@type":"Person"},{"name":"Pandey, Garima","affiliation":"National Institute of Plant Genome Research (NIPGR) ,  Aruna Asaf Ali Marg, New Delhi 110 067 ,  India","@type":"Person"},{"name":"Kumari, Kajal","affiliation":"National Institute of Plant Genome Research (NIPGR) ,  Aruna Asaf Ali Marg, New Delhi 110 067 ,  India","@type":"Person"},{"name":"Parida, Swarup Kumar","affiliation":"National Institute of Plant Genome Research (NIPGR) ,  Aruna Asaf Ali Marg, New Delhi 110 067 ,  India","@type":"Person"},{"name":"Prasad, Manoj","affiliation":"National Institute of Plant Genome Research (NIPGR) ,  Aruna Asaf Ali Marg, New Delhi 110 067 ,  India","@type":"Person"}],"description":"Abstract.  Generating genomic resources in terms of molecular markers is imperative in molecular breeding for crop improvement. Though development and applicati","pageStart":"41","pageEnd":"52","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/1/1/m_cover.gif?Expires=2147483647&Signature=O6KdUwcvHbd2-JMT7ataLxhbwOeCBuxRp98Pq5MMbgeFNh0CYTmpXkkm0SI3qUm3MJ0vHXWxOm2kPBu5Rgo0fKHtDJDfZESs9V~0tNUUm6kyGHNICw1kvqdJFcMATuPRllJYdqzXipnnf-Skkbo14bB23qukbSJgkTF8pGT99rZf8sbETeNndWrSEgQxZdETKMev~I0NCAvT0v~xsxmWuK-UxMPFC5n4zZ5VTdx1iGP0wcMBg32Rgk2MsAMBA~xEVLTIrTKFiOSHU6O-J6MxiLzFiPOJsgoGHQ67KOEexoif2dgVWS4VER03BHFw9~dH0~yNMz5hZOC1eS-fW6NcsQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Development of 5123 Intron-Length Polymorphic Markers for Large-Scale Genotyping Applications in Foxtail Millet","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/1/1/m_cover.gif?Expires=2147483647&Signature=O6KdUwcvHbd2-JMT7ataLxhbwOeCBuxRp98Pq5MMbgeFNh0CYTmpXkkm0SI3qUm3MJ0vHXWxOm2kPBu5Rgo0fKHtDJDfZESs9V~0tNUUm6kyGHNICw1kvqdJFcMATuPRllJYdqzXipnnf-Skkbo14bB23qukbSJgkTF8pGT99rZf8sbETeNndWrSEgQxZdETKMev~I0NCAvT0v~xsxmWuK-UxMPFC5n4zZ5VTdx1iGP0wcMBg32Rgk2MsAMBA~xEVLTIrTKFiOSHU6O-J6MxiLzFiPOJsgoGHQ67KOEexoif2dgVWS4VER03BHFw9~dH0~yNMz5hZOC1eS-fW6NcsQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Development of 5123 Intron-Length Polymorphic Markers for Large-Scale Genotyping Applications in Foxtail Millet" />
    <meta property="og:description" content="Abstract.  Generating genomic resources in terms of molecular markers is imperative in molecular breeding for crop improvement. Though development and applicati" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/21/1/41/347328" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2013-10-01" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Muthamilarasan, Mehanathan" /><meta name="citation_author_institution" content="National Institute of Plant Genome Research (NIPGR), Aruna Asaf Ali Marg, New Delhi 110 067, India" /><meta name="citation_author" content="Venkata Suresh, B." /><meta name="citation_author_institution" content="National Institute of Plant Genome Research (NIPGR), Aruna Asaf Ali Marg, New Delhi 110 067, India" /><meta name="citation_author" content="Pandey, Garima" /><meta name="citation_author_institution" content="National Institute of Plant Genome Research (NIPGR), Aruna Asaf Ali Marg, New Delhi 110 067, India" /><meta name="citation_author" content="Kumari, Kajal" /><meta name="citation_author_institution" content="National Institute of Plant Genome Research (NIPGR), Aruna Asaf Ali Marg, New Delhi 110 067, India" /><meta name="citation_author" content="Parida, Swarup Kumar" /><meta name="citation_author_institution" content="National Institute of Plant Genome Research (NIPGR), Aruna Asaf Ali Marg, New Delhi 110 067, India" /><meta name="citation_author" content="Prasad, Manoj" /><meta name="citation_author_institution" content="National Institute of Plant Genome Research (NIPGR), Aruna Asaf Ali Marg, New Delhi 110 067, India" /><meta name="citation_title" content="Development of 5123 Intron-Length Polymorphic Markers for Large-Scale Genotyping Applications in Foxtail Millet" /><meta name="citation_firstpage" content="41" /><meta name="citation_lastpage" content="52" /><meta name="citation_doi" content="10.1093/dnares/dst039" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="21" /><meta name="citation_issue" content="1" /><meta name="citation_publication_date" content="2014/02/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Foxtail millet: a sequence-driven grass model system; Plant Physiol.;  citation_year=2009;  citation_volume=149;  citation_pages=137-41; " /><meta name="citation_reference" content="citation_title=Foxtail millet: a model crop for genetic and genomic studies in bioenergy grasses; Crit. Rev. Biotechnol;  citation_year=2013;  citation_volume=33;  citation_pages=328-43; " /><meta name="citation_reference" content="citation_title=Setaria viridis and Setaria italica, model genetic systems for Panicoid grasses; J. Exp. Bot.;  citation_year=2011;  citation_volume=62;  citation_pages=3031-7; " /><meta name="citation_reference" content="citation_title=Setaria genome sequencing: an overview; J. Plant Biochem. Biotechnol.;  citation_year=2013;  citation_volume=22;  citation_pages=257-60; " /><meta name="citation_reference" content="citation_title=Recent advances in crop genomics for ensuring food security; Curr. Sci.;  citation_year=2013;  citation_volume=105;  citation_pages=155-8; " /><meta name="citation_reference" content="citation_title=Reference genome sequence of the model plant Setaria; Nat. Biotech.;  citation_year=2012;  citation_volume=30;  citation_pages=555-61; " /><meta name="citation_reference" content="citation_title=Genome sequence of foxtail millet (Setaria italica) provides insights into grass evolution and biofuel potential; Nat. Biotech.;  citation_year=2012;  citation_volume=30;  citation_pages=549-54; " /><meta name="citation_reference" content="citation_title=Genome-wide development and use of microsatellite markers for large-scale genotyping applications in foxtail millet [Setaria italica (L.)]; DNA Res.;  citation_year=2013;  citation_volume=20;  citation_pages=197-207; " /><meta name="citation_reference" content="citation_title=Development of eSSR-markers in Setaria italica and their applicability in studying genetic diversity, cross-transferability and comparative mapping in millet and non-millet species; PLoS ONE;  citation_year=2013;  citation_volume=8;  citation_pages=e67742" /><meta name="citation_reference" content="citation_title=FmMDb: a versatile database of foxtail millet markers for millets and bioenergy grasses research; PLoS ONE;  citation_year=2013;  citation_volume=8;  citation_pages=e71418" /><meta name="citation_reference" content="citation_title=Construction of a genetic linkage map in man using restriction fragment length polymorphisms; Am. J. Hum. Genet.;  citation_year=1980;  citation_volume=32;  citation_pages=314-31; " /><meta name="citation_reference" content="citation_title=DNA polymorphisms amplified by arbitrary primers are useful as genetic markers; Nucleic Acids Res.;  citation_year=1990;  citation_volume=18;  citation_pages=6531-5; " /><meta name="citation_reference" content="citation_title=AFLP: a new technique for DNA fingerprinting; Nucleic Acids Res.;  citation_year=1995;  citation_volume=23;  citation_pages=4407-14; " /><meta name="citation_reference" content="citation_title=Barley microsatellites: allele variation and mapping; Plant Mol. Biol.;  citation_year=1995;  citation_volume=27;  citation_pages=835-45; " /><meta name="citation_reference" content="citation_title=The use of a genetic map of biallelic markers in linkage studies; Nat. Genet.;  citation_year=1997;  citation_volume=17;  citation_pages=21-4; " /><meta name="citation_reference" content="citation_title=A sequence-based genetic map of Medicago truncatula and comparison of marker colinearity with M. sativa; Genetics;  citation_year=2004;  citation_volume=166;  citation_pages=1463-502; " /><meta name="citation_reference" content="citation_title=PIP: a database for potential intron polymorphism markers; Bioinformatics;  citation_year=2007;  citation_volume=23;  citation_pages=2174-7; " /><meta name="citation_reference" content="citation_title=Genome-wide investigation of intron length polymorphisms and their potential as molecular markers in rice (Oryza sativa L.); DNA Res.;  citation_year=2005;  citation_volume=12;  citation_pages=417-27; " /><meta name="citation_reference" content="citation_title=cTBP: a successful intron length polymorphism (ILP)-based genotyping method targeted to well defined experimental needs; Diversity;  citation_year=2010;  citation_volume=2;  citation_pages=572-85; " /><meta name="citation_reference" content="citation_title=Development and utilization of novel intron length polymorphic markers in foxtail millet [Setaria italica (L.) P. Beauv.]; Genome;  citation_year=2011;  citation_volume=54;  citation_pages=586-602; " /><meta name="citation_reference" content="citation_title=Extraordinary polymorphism microsatellite DNA in barley, species diversity, chromosomal location and population dynamics; Proc. Natl Acad. Sci. USA;  citation_year=1994;  citation_volume=91;  citation_pages=5466-70; " /><meta name="citation_reference" content="citation_title=MapChart: software for the graphical presentation of linkage maps and QTLs; J. Hered.;  citation_year=2002;  citation_volume=93;  citation_pages=77-8; " /><meta name="citation_reference" content="citation_title=A web server for performing electronic PCR; Nucleic Acids Res.;  citation_year=2004;  citation_volume=32;  citation_pages=W108-12; " /><meta name="citation_reference" content="citation_title=A comparative study of molecular and morphological methods of describing relationships between perennial ryegrass (Lolium perenne L.) varieties; Theor. Appl. Genet.;  citation_year=2001;  citation_volume=103;  citation_pages=1138-50; " /><meta name="citation_reference" content="citation_title=Nouvelle recherches sur la distribution florale; Bull. Soc. Vaud. Sci. Nat.;  citation_year=1908;  citation_volume=44;  citation_pages=223-70; " /><meta name="citation_reference" content=" citation_url=http://www.exetersoftware.com/cat/ntsyspc/ntsyspc.html; citation_publisher=Exeter Software, Setauket, NY; NTSYS-pc numerical taxonomy and multivariance analysis system;  citation_year=1997; " /><meta name="citation_reference" content="citation_title=Analysis of genetic diversity in subdivided populations; Proc. Natl Acad. Sci. USA;  citation_year=1973;  citation_volume=70;  citation_pages=3321-3; " /><meta name="citation_reference" content="citation_title=PowerMarker: an integrated analysis environment for genetic marker analysis; Bioinformatics;  citation_year=2005;  citation_volume=21;  citation_pages=2128-9; " /><meta name="citation_reference" content="citation_title=Circos, an information aesthetic for comparative genomics; Genome Res.;  citation_year=2009;  citation_volume=19;  citation_pages=1639-45; " /><meta name="citation_reference" content="citation_title=Nucleic acids II: the polymerase chain reaction;  citation_edition=2nd edition; citation_publisher=Sinauer Associates Inc., Sunderland, MA; Molecular systematics;  citation_year=1995;  citation_pages=205-47; " /><meta name="citation_reference" content="citation_title=Intron polymorphism pattern in AOX1b of wild St John's wort (Hypericum perforatum) allows discrimination between individual plants; Physiol. Plant.;  citation_year=2009;  citation_volume=137;  citation_pages=520-31; " /><meta name="citation_reference" content="citation_title=Development of chickpea EST-SSR markers and analysis of allelic variation across related species; Theor. Appl. Genet.;  citation_year=2009;  citation_volume=118;  citation_pages=591-608; " /><meta name="citation_reference" content="citation_title=Development and characterization of genic SSR markers in Medicago truncatula and their transferability in leguminous and non-leguminous species; Genome;  citation_year=2009;  citation_volume=52;  citation_pages=761-71; " /><meta name="citation_reference" content="citation_title=Comparative analysis of genetic diversity and structure in rice using ILP and SSR markers; Rice Sci.;  citation_year=2010;  citation_volume=17;  citation_pages=257-68; " /><meta name="citation_reference" content="citation_title=Comparative genetic maps of foxtail millet (Setaria italica) and rice (Oryza sativa); Theor. Appl. Genet.;  citation_year=1998;  citation_volume=96;  citation_pages=63-8; " /><meta name="citation_reference" content="citation_title=Genome relationships: the grass model in current research; Plant Cell;  citation_year=2000;  citation_volume=12;  citation_pages=637-46; " /><meta name="citation_reference" content="citation_title=Updating the crop circle; Curr. Opin. Plant Biol.;  citation_year=2005;  citation_volume=8;  citation_pages=155-62; " /><meta name="citation_reference" content="citation_title=Comparative analyses reveal high levels of conserved colinearity between the finger millet and rice genomes; Theor. Appl. Genet.;  citation_year=2007;  citation_volume=115;  citation_pages=489-99; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/21/1/41/994380/dst039.pdf" /><meta name="description" content="Abstract.  Generating genomic resources in terms of molecular markers is imperative in molecular breeding for crop improvement. Though development and applicati" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/21/1/41/347328" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/21/1/41/347328" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/21/1/41/347328");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dst039';
    </script>
<title>
	Development of 5123 Intron-Length Polymorphic Markers for Large-Scale Genotyping Applications in Foxtail Millet | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./347328" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BYVXPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJNdXRoYW1pbGFyYXNhbiwgTWVoYW5hdGhhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5hdGlvbmFsIEluc3RpdHV0ZSBvZiBQbGFudCBHZW5vbWUgUmVzZWFyY2ggKE5JUEdSKSwgQXJ1bmEgQXNhZiBBbGkgTWFyZywgTmV3IERlbGhpIDExMCAwNjcsIEluZGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iVmVua2F0YSBTdXJlc2gsIEIuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTmF0aW9uYWwgSW5zdGl0dXRlIG9mIFBsYW50IEdlbm9tZSBSZXNlYXJjaCAoTklQR1IpLCBBcnVuYSBBc2FmIEFsaSBNYXJnLCBOZXcgRGVsaGkgMTEwIDA2NywgSW5kaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJQYW5kZXksIEdhcmltYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5hdGlvbmFsIEluc3RpdHV0ZSBvZiBQbGFudCBHZW5vbWUgUmVzZWFyY2ggKE5JUEdSKSwgQXJ1bmEgQXNhZiBBbGkgTWFyZywgTmV3IERlbGhpIDExMCAwNjcsIEluZGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS3VtYXJpLCBLYWphbCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5hdGlvbmFsIEluc3RpdHV0ZSBvZiBQbGFudCBHZW5vbWUgUmVzZWFyY2ggKE5JUEdSKSwgQXJ1bmEgQXNhZiBBbGkgTWFyZywgTmV3IERlbGhpIDExMCAwNjcsIEluZGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iUGFyaWRhLCBTd2FydXAgS3VtYXIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJOYXRpb25hbCBJbnN0aXR1dGUgb2YgUGxhbnQgR2Vub21lIFJlc2VhcmNoIChOSVBHUiksIEFydW5hIEFzYWYgQWxpIE1hcmcsIE5ldyBEZWxoaSAxMTAgMDY3LCBJbmRpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlByYXNhZCwgTWFub2oiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJOYXRpb25hbCBJbnN0aXR1dGUgb2YgUGxhbnQgR2Vub21lIFJlc2VhcmNoIChOSVBHUiksIEFydW5hIEFzYWYgQWxpIE1hcmcsIE5ldyBEZWxoaSAxMTAgMDY3LCBJbmRpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iRGV2ZWxvcG1lbnQgb2YgNTEyMyBJbnRyb24tTGVuZ3RoIFBvbHltb3JwaGljIE1hcmtlcnMgZm9yIExhcmdlLVNjYWxlIEdlbm90eXBpbmcgQXBwbGljYXRpb25zIGluIEZveHRhaWwgTWlsbGV0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iNDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjUyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHN0MDM5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjIxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSIxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTQvMDIvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Gb3h0YWlsIG1pbGxldDogYSBzZXF1ZW5jZS1kcml2ZW4gZ3Jhc3MgbW9kZWwgc3lzdGVtOyBQbGFudCBQaHlzaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xNDk7ICBjaXRhdGlvbl9wYWdlcz0xMzctNDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZveHRhaWwgbWlsbGV0OiBhIG1vZGVsIGNyb3AgZm9yIGdlbmV0aWMgYW5kIGdlbm9taWMgc3R1ZGllcyBpbiBiaW9lbmVyZ3kgZ3Jhc3NlczsgQ3JpdC4gUmV2LiBCaW90ZWNobm9sOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTMzOyAgY2l0YXRpb25fcGFnZXM9MzI4LTQzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZXRhcmlhIHZpcmlkaXMgYW5kIFNldGFyaWEgaXRhbGljYSwgbW9kZWwgZ2VuZXRpYyBzeXN0ZW1zIGZvciBQYW5pY29pZCBncmFzc2VzOyBKLiBFeHAuIEJvdC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NjI7ICBjaXRhdGlvbl9wYWdlcz0zMDMxLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNldGFyaWEgZ2Vub21lIHNlcXVlbmNpbmc6IGFuIG92ZXJ2aWV3OyBKLiBQbGFudCBCaW9jaGVtLiBCaW90ZWNobm9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0yMjsgIGNpdGF0aW9uX3BhZ2VzPTI1Ny02MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVjZW50IGFkdmFuY2VzIGluIGNyb3AgZ2Vub21pY3MgZm9yIGVuc3VyaW5nIGZvb2Qgc2VjdXJpdHk7IEN1cnIuIFNjaS47ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTA1OyAgY2l0YXRpb25fcGFnZXM9MTU1LTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlZmVyZW5jZSBnZW5vbWUgc2VxdWVuY2Ugb2YgdGhlIG1vZGVsIHBsYW50IFNldGFyaWE7IE5hdC4gQmlvdGVjaC47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MzA7ICBjaXRhdGlvbl9wYWdlcz01NTUtNjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZSBzZXF1ZW5jZSBvZiBmb3h0YWlsIG1pbGxldCAoU2V0YXJpYSBpdGFsaWNhKSBwcm92aWRlcyBpbnNpZ2h0cyBpbnRvIGdyYXNzIGV2b2x1dGlvbiBhbmQgYmlvZnVlbCBwb3RlbnRpYWw7IE5hdC4gQmlvdGVjaC47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MzA7ICBjaXRhdGlvbl9wYWdlcz01NDktNTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIGRldmVsb3BtZW50IGFuZCB1c2Ugb2YgbWljcm9zYXRlbGxpdGUgbWFya2VycyBmb3IgbGFyZ2Utc2NhbGUgZ2Vub3R5cGluZyBhcHBsaWNhdGlvbnMgaW4gZm94dGFpbCBtaWxsZXQgW1NldGFyaWEgaXRhbGljYSAoTC4pXTsgRE5BIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MjA7ICBjaXRhdGlvbl9wYWdlcz0xOTctMjA3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZXZlbG9wbWVudCBvZiBlU1NSLW1hcmtlcnMgaW4gU2V0YXJpYSBpdGFsaWNhIGFuZCB0aGVpciBhcHBsaWNhYmlsaXR5IGluIHN0dWR5aW5nIGdlbmV0aWMgZGl2ZXJzaXR5LCBjcm9zcy10cmFuc2ZlcmFiaWxpdHkgYW5kIGNvbXBhcmF0aXZlIG1hcHBpbmcgaW4gbWlsbGV0IGFuZCBub24tbWlsbGV0IHNwZWNpZXM7IFBMb1MgT05FOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz1lNjc3NDIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GbU1EYjogYSB2ZXJzYXRpbGUgZGF0YWJhc2Ugb2YgZm94dGFpbCBtaWxsZXQgbWFya2VycyBmb3IgbWlsbGV0cyBhbmQgYmlvZW5lcmd5IGdyYXNzZXMgcmVzZWFyY2g7IFBMb1MgT05FOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz1lNzE0MTgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db25zdHJ1Y3Rpb24gb2YgYSBnZW5ldGljIGxpbmthZ2UgbWFwIGluIG1hbiB1c2luZyByZXN0cmljdGlvbiBmcmFnbWVudCBsZW5ndGggcG9seW1vcnBoaXNtczsgQW0uIEouIEh1bS4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0xOTgwOyAgY2l0YXRpb25fdm9sdW1lPTMyOyAgY2l0YXRpb25fcGFnZXM9MzE0LTMxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1ETkEgcG9seW1vcnBoaXNtcyBhbXBsaWZpZWQgYnkgYXJiaXRyYXJ5IHByaW1lcnMgYXJlIHVzZWZ1bCBhcyBnZW5ldGljIG1hcmtlcnM7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5MDsgIGNpdGF0aW9uX3ZvbHVtZT0xODsgIGNpdGF0aW9uX3BhZ2VzPTY1MzEtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QUZMUDogYSBuZXcgdGVjaG5pcXVlIGZvciBETkEgZmluZ2VycHJpbnRpbmc7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5NTsgIGNpdGF0aW9uX3ZvbHVtZT0yMzsgIGNpdGF0aW9uX3BhZ2VzPTQ0MDctMTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJhcmxleSBtaWNyb3NhdGVsbGl0ZXM6IGFsbGVsZSB2YXJpYXRpb24gYW5kIG1hcHBpbmc7IFBsYW50IE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTU7ICBjaXRhdGlvbl92b2x1bWU9Mjc7ICBjaXRhdGlvbl9wYWdlcz04MzUtNDU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSB1c2Ugb2YgYSBnZW5ldGljIG1hcCBvZiBiaWFsbGVsaWMgbWFya2VycyBpbiBsaW5rYWdlIHN0dWRpZXM7IE5hdC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTE3OyAgY2l0YXRpb25fcGFnZXM9MjEtNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBzZXF1ZW5jZS1iYXNlZCBnZW5ldGljIG1hcCBvZiBNZWRpY2FnbyB0cnVuY2F0dWxhIGFuZCBjb21wYXJpc29uIG9mIG1hcmtlciBjb2xpbmVhcml0eSB3aXRoIE0uIHNhdGl2YTsgR2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTY2OyAgY2l0YXRpb25fcGFnZXM9MTQ2My01MDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBJUDogYSBkYXRhYmFzZSBmb3IgcG90ZW50aWFsIGludHJvbiBwb2x5bW9ycGhpc20gbWFya2VyczsgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MjM7ICBjaXRhdGlvbl9wYWdlcz0yMTc0LTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIGludmVzdGlnYXRpb24gb2YgaW50cm9uIGxlbmd0aCBwb2x5bW9ycGhpc21zIGFuZCB0aGVpciBwb3RlbnRpYWwgYXMgbW9sZWN1bGFyIG1hcmtlcnMgaW4gcmljZSAoT3J5emEgc2F0aXZhIEwuKTsgRE5BIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz00MTctMjc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPWNUQlA6IGEgc3VjY2Vzc2Z1bCBpbnRyb24gbGVuZ3RoIHBvbHltb3JwaGlzbSAoSUxQKS1iYXNlZCBnZW5vdHlwaW5nIG1ldGhvZCB0YXJnZXRlZCB0byB3ZWxsIGRlZmluZWQgZXhwZXJpbWVudGFsIG5lZWRzOyBEaXZlcnNpdHk7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MjsgIGNpdGF0aW9uX3BhZ2VzPTU3Mi04NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGV2ZWxvcG1lbnQgYW5kIHV0aWxpemF0aW9uIG9mIG5vdmVsIGludHJvbiBsZW5ndGggcG9seW1vcnBoaWMgbWFya2VycyBpbiBmb3h0YWlsIG1pbGxldCBbU2V0YXJpYSBpdGFsaWNhIChMLikgUC4gQmVhdXYuXTsgR2Vub21lOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTU0OyAgY2l0YXRpb25fcGFnZXM9NTg2LTYwMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXh0cmFvcmRpbmFyeSBwb2x5bW9ycGhpc20gbWljcm9zYXRlbGxpdGUgRE5BIGluIGJhcmxleSwgc3BlY2llcyBkaXZlcnNpdHksIGNocm9tb3NvbWFsIGxvY2F0aW9uIGFuZCBwb3B1bGF0aW9uIGR5bmFtaWNzOyBQcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0xOTk0OyAgY2l0YXRpb25fdm9sdW1lPTkxOyAgY2l0YXRpb25fcGFnZXM9NTQ2Ni03MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWFwQ2hhcnQ6IHNvZnR3YXJlIGZvciB0aGUgZ3JhcGhpY2FsIHByZXNlbnRhdGlvbiBvZiBsaW5rYWdlIG1hcHMgYW5kIFFUTHM7IEouIEhlcmVkLjsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT05MzsgIGNpdGF0aW9uX3BhZ2VzPTc3LTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgd2ViIHNlcnZlciBmb3IgcGVyZm9ybWluZyBlbGVjdHJvbmljIFBDUjsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTMyOyAgY2l0YXRpb25fcGFnZXM9VzEwOC0xMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBjb21wYXJhdGl2ZSBzdHVkeSBvZiBtb2xlY3VsYXIgYW5kIG1vcnBob2xvZ2ljYWwgbWV0aG9kcyBvZiBkZXNjcmliaW5nIHJlbGF0aW9uc2hpcHMgYmV0d2VlbiBwZXJlbm5pYWwgcnllZ3Jhc3MgKExvbGl1bSBwZXJlbm5lIEwuKSB2YXJpZXRpZXM7IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MTAzOyAgY2l0YXRpb25fcGFnZXM9MTEzOC01MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Tm91dmVsbGUgcmVjaGVyY2hlcyBzdXIgbGEgZGlzdHJpYnV0aW9uIGZsb3JhbGU7IEJ1bGwuIFNvYy4gVmF1ZC4gU2NpLiBOYXQuOyAgY2l0YXRpb25feWVhcj0xOTA4OyAgY2l0YXRpb25fdm9sdW1lPTQ0OyAgY2l0YXRpb25fcGFnZXM9MjIzLTcwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSIgY2l0YXRpb25fdXJsPWh0dHA6Ly93d3cuZXhldGVyc29mdHdhcmUuY29tL2NhdC9udHN5c3BjL250c3lzcGMuaHRtbDsgY2l0YXRpb25fcHVibGlzaGVyPUV4ZXRlciBTb2Z0d2FyZSwgU2V0YXVrZXQsIE5ZOyBOVFNZUy1wYyBudW1lcmljYWwgdGF4b25vbXkgYW5kIG11bHRpdmFyaWFuY2UgYW5hbHlzaXMgc3lzdGVtOyAgY2l0YXRpb25feWVhcj0xOTk3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbmFseXNpcyBvZiBnZW5ldGljIGRpdmVyc2l0eSBpbiBzdWJkaXZpZGVkIHBvcHVsYXRpb25zOyBQcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0xOTczOyAgY2l0YXRpb25fdm9sdW1lPTcwOyAgY2l0YXRpb25fcGFnZXM9MzMyMS0zOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qb3dlck1hcmtlcjogYW4gaW50ZWdyYXRlZCBhbmFseXNpcyBlbnZpcm9ubWVudCBmb3IgZ2VuZXRpYyBtYXJrZXIgYW5hbHlzaXM7IEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTIxOyAgY2l0YXRpb25fcGFnZXM9MjEyOC05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaXJjb3MsIGFuIGluZm9ybWF0aW9uIGFlc3RoZXRpYyBmb3IgY29tcGFyYXRpdmUgZ2Vub21pY3M7IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTE5OyAgY2l0YXRpb25fcGFnZXM9MTYzOS00NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TnVjbGVpYyBhY2lkcyBJSTogdGhlIHBvbHltZXJhc2UgY2hhaW4gcmVhY3Rpb247ICBjaXRhdGlvbl9lZGl0aW9uPTJuZCBlZGl0aW9uOyBjaXRhdGlvbl9wdWJsaXNoZXI9U2luYXVlciBBc3NvY2lhdGVzIEluYy4sIFN1bmRlcmxhbmQsIE1BOyBNb2xlY3VsYXIgc3lzdGVtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTE5OTU7ICBjaXRhdGlvbl9wYWdlcz0yMDUtNDc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUludHJvbiBwb2x5bW9ycGhpc20gcGF0dGVybiBpbiBBT1gxYiBvZiB3aWxkIFN0IEpvaG4ncyB3b3J0IChIeXBlcmljdW0gcGVyZm9yYXR1bSkgYWxsb3dzIGRpc2NyaW1pbmF0aW9uIGJldHdlZW4gaW5kaXZpZHVhbCBwbGFudHM7IFBoeXNpb2wuIFBsYW50LjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xMzc7ICBjaXRhdGlvbl9wYWdlcz01MjAtMzE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldmVsb3BtZW50IG9mIGNoaWNrcGVhIEVTVC1TU1IgbWFya2VycyBhbmQgYW5hbHlzaXMgb2YgYWxsZWxpYyB2YXJpYXRpb24gYWNyb3NzIHJlbGF0ZWQgc3BlY2llczsgVGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xMTg7ICBjaXRhdGlvbl9wYWdlcz01OTEtNjA4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZXZlbG9wbWVudCBhbmQgY2hhcmFjdGVyaXphdGlvbiBvZiBnZW5pYyBTU1IgbWFya2VycyBpbiBNZWRpY2FnbyB0cnVuY2F0dWxhIGFuZCB0aGVpciB0cmFuc2ZlcmFiaWxpdHkgaW4gbGVndW1pbm91cyBhbmQgbm9uLWxlZ3VtaW5vdXMgc3BlY2llczsgR2Vub21lOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTUyOyAgY2l0YXRpb25fcGFnZXM9NzYxLTcxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wYXJhdGl2ZSBhbmFseXNpcyBvZiBnZW5ldGljIGRpdmVyc2l0eSBhbmQgc3RydWN0dXJlIGluIHJpY2UgdXNpbmcgSUxQIGFuZCBTU1IgbWFya2VyczsgUmljZSBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTE3OyAgY2l0YXRpb25fcGFnZXM9MjU3LTY4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wYXJhdGl2ZSBnZW5ldGljIG1hcHMgb2YgZm94dGFpbCBtaWxsZXQgKFNldGFyaWEgaXRhbGljYSkgYW5kIHJpY2UgKE9yeXphIHNhdGl2YSk7IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9OTY7ICBjaXRhdGlvbl9wYWdlcz02My04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUgcmVsYXRpb25zaGlwczogdGhlIGdyYXNzIG1vZGVsIGluIGN1cnJlbnQgcmVzZWFyY2g7IFBsYW50IENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz02MzctNDY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVVwZGF0aW5nIHRoZSBjcm9wIGNpcmNsZTsgQ3Vyci4gT3Bpbi4gUGxhbnQgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPTE1NS02MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyYXRpdmUgYW5hbHlzZXMgcmV2ZWFsIGhpZ2ggbGV2ZWxzIG9mIGNvbnNlcnZlZCBjb2xpbmVhcml0eSBiZXR3ZWVuIHRoZSBmaW5nZXIgbWlsbGV0IGFuZCByaWNlIGdlbm9tZXM7IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTE1OyAgY2l0YXRpb25fcGFnZXM9NDg5LTk5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMjEvMS80MS85OTQzODAvZHN0MDM5LnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBHZW5lcmF0aW5nIGdlbm9taWMgcmVzb3VyY2VzIGluIHRlcm1zIG9mIG1vbGVjdWxhciBtYXJrZXJzIGlzIGltcGVyYXRpdmUgaW4gbW9sZWN1bGFyIGJyZWVkaW5nIGZvciBjcm9wIGltcHJvdmVtZW50LiBUaG91Z2ggZGV2ZWxvcG1lbnQgYW5kIGFwcGxpY2F0aSIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXIvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9MzQ3MzI4JnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZFWsl2KfGZLLVRmrHkjZfGPph+q9" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f1%2f41%2f347328" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f1%2f41%2f347328" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=13381" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/21/1"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/1/1/m_cover.gif?Expires=2147483647&amp;Signature=O6KdUwcvHbd2-JMT7ataLxhbwOeCBuxRp98Pq5MMbgeFNh0CYTmpXkkm0SI3qUm3MJ0vHXWxOm2kPBu5Rgo0fKHtDJDfZESs9V~0tNUUm6kyGHNICw1kvqdJFcMATuPRllJYdqzXipnnf-Skkbo14bB23qukbSJgkTF8pGT99rZf8sbETeNndWrSEgQxZdETKMev~I0NCAvT0v~xsxmWuK-UxMPFC5n4zZ5VTdx1iGP0wcMBg32Rgk2MsAMBA~xEVLTIrTKFiOSHU6O-J6MxiLzFiPOJsgoGHQ67KOEexoif2dgVWS4VER03BHFw9~dH0~yNMz5hZOC1eS-fW6NcsQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/21/1">
                        <div class="volume trailing-comma">Volume 21</div>
                        <div class="issue">Issue 1</div>

            </a>

        </div>
        <div class="ii-pub-date">
February 2014        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="4913387">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4913387">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4913389">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4913389">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4913393">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4913393">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4913407">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4913407">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4913443">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4913443">4. Conclusions</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4913445">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4913445">Supplementary Data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4913447">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4913447">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4913449">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4913449">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4913451">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4913451">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/21/1/27/347182">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/21/1/53/347430">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Development of 5123 Intron-Length Polymorphic Markers for Large-Scale Genotyping Applications in Foxtail Millet
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Mehanathan Muthamilarasan</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">National Institute of Plant Genome Research (NIPGR)</div>, <div class="addr-line">Aruna Asaf Ali Marg, New Delhi 110 067</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Mehanathan+Muthamilarasan">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Muthamilarasan M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Muthamilarasan M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Mehanathan Muthamilarasan</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">B. Venkata Suresh</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">National Institute of Plant Genome Research (NIPGR)</div>, <div class="addr-line">Aruna Asaf Ali Marg, New Delhi 110 067</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=B.+Venkata+Suresh">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Venkata Suresh B">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Venkata Suresh B&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">B. Venkata Suresh</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Garima Pandey</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">National Institute of Plant Genome Research (NIPGR)</div>, <div class="addr-line">Aruna Asaf Ali Marg, New Delhi 110 067</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Garima+Pandey">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Pandey G">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Pandey G&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Garima Pandey</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kajal Kumari</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">National Institute of Plant Genome Research (NIPGR)</div>, <div class="addr-line">Aruna Asaf Ali Marg, New Delhi 110 067</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kajal+Kumari">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kumari K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kumari K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Kajal Kumari</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Swarup Kumar Parida</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">National Institute of Plant Genome Research (NIPGR)</div>, <div class="addr-line">Aruna Asaf Ali Marg, New Delhi 110 067</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Swarup+Kumar+Parida">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Parida S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Parida S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Swarup Kumar Parida</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Manoj Prasad</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">National Institute of Plant Genome Research (NIPGR)</div>, <div class="addr-line">Aruna Asaf Ali Marg, New Delhi 110 067</div>, <div class="country">India</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +91 11-26735160. Fax. +91 11-26741658. E-mail: <a href="mailto:manoj_prasad@nipgr.ac.in" target="_blank">manoj_prasad@nipgr.ac.in</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Manoj+Prasad">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Prasad M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Prasad M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Manoj Prasad</a>
                                </span>

                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 21, Issue 1, 1 February 2014, Pages 41–52, <a href='https://doi.org/10.1093/dnares/dst039'>https://doi.org/10.1093/dnares/dst039</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">01 October 2013</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">26 May 2013</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">05 September 2013</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="347328" href="/dnaresearch/article-pdf/21/1/41/994380/dst039.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Mehanathan Muthamilarasan, B. Venkata Suresh, Garima Pandey, Kajal Kumari, Swarup Kumar Parida, Manoj Prasad;  Development of 5123 Intron-Length Polymorphic Markers for Large-Scale Genotyping Applications in Foxtail Millet, <em>DNA Research</em>, Volume 21, Issue 1, 1 February 2014, Pages 41–52, <a href='https://doi.org/10.1093/dnares/dst039'>https://doi.org/10.1093/dnares/dst039</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/347328?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/347328?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/347328?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/347328?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/347328?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/347328?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/347328?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Development%20of%205123%20Intron-Length%20Polymorphic%20Markers%20for%20Large-Scale%20Genotyping%20Applications%20in%20Foxtail%20Millet&amp;publicationDate=2013-10-01&amp;volumeNum=21&amp;issueNum=1&amp;author=Muthamilarasan%2C%20Mehanathan%3B%20Venkata%20Suresh%2C%20B.&amp;startPage=41&amp;endPage=52&amp;contentId=10.1093%2Fdnares%2Fdst039&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f1%2f41%2f347328" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=4913387 id="4913387" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Generating genomic resources in terms of molecular markers is imperative in molecular breeding for crop improvement. Though development and application of microsatellite markers in large-scale was reported in the model crop foxtail millet, no such large-scale study was conducted for intron-length polymorphic (ILP) markers. Considering this, we developed 5123 ILP markers, of which 4049 were physically mapped onto 9 chromosomes of foxtail millet. BLAST analysis of 5123 expressed sequence tags (ESTs) suggested the function for ∼71.5% ESTs and grouped them into 5 different functional categories. About 440 selected primer pairs representing the foxtail millet genome and the different functional groups showed high-level of cross-genera amplification at an average of ∼85% in eight millets and five non-millet species. The efficacy of the ILP markers for distinguishing the foxtail millet is demonstrated by observed heterozygosity (0.20) and Nei's average gene diversity (0.22). <em>In silico</em> comparative mapping of physically mapped ILP markers demonstrated substantial percentage of sequence-based orthology and syntenic relationship between foxtail millet chromosomes and sorghum (∼50%), maize (∼46%), rice (∼21%) and <em>Brachypodium</em> (∼21%) chromosomes. Hence, for the first time, we developed large-scale ILP markers in foxtail millet and demonstrated their utility in germplasm characterization, transferability, phylogenetics and comparative mapping studies in millets and bioenergy grass species.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;comparative mapping&quot;">comparative mapping</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;foxtail millet (Setaria italica L.)&quot;">foxtail millet (<em>Setaria italica</em> L.)</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;intron-length polymorphism (ILP)&quot;">intron-length polymorphism (ILP)</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;physical mapping&quot;">physical mapping</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="transferability">transferability</a></div>                            <h2 scrollto-destination=4913389 id="4913389" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Foxtail millet [(<em>Setaria italica</em> (L.) P. Beauv] has recently been regarded as a tractable model crop due to its small genome (∼ 515 Mb; 2<em>n</em> = 2<em>x</em> = 18), low amount of repetitive DNA, inbreeding nature and short life cycle.<sup><a reveal-id="DST039C1" data-open="DST039C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DST039C2" data-open="DST039C2" class="link link-ref link-reveal xref-bibr">2</a></sup> Moreover, its close relatedness to several bioenergy crops with complex genomes such as switchgrass (<em>Panicum virgatum</em>), napiergrass (<em>Pennisetum purpureum</em>) and pearl millet (<em>Pennisetum glaucum</em>) and its potential abiotic stress tolerance adds up to the merits of foxtail millet as an experimental model crop for exploring various plant architectural traits, evolutionary genomics and physiological attributes of the C<sub>4</sub> Panicoid grass crops.<sup><a reveal-id="DST039C1  DST039C3" data-open="DST039C1  DST039C3" class="link link-ref link-reveal xref-bibr">1–3</a></sup> Since foxtail millet has one of the largest sets of both cultivated and wild-type germplasm rich in phenotypic variations, it appears promising for association mapping and allele mining of elite and novel variants to be integrated in crop improvement programmes.<sup><a reveal-id="DST039C2" data-open="DST039C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DST039C4" data-open="DST039C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DST039C5" data-open="DST039C5" class="link link-ref link-reveal xref-bibr">5</a></sup> Hence, considering the importance of foxtail millet, the Joint Genome Institute (JGI) of the Department of Energy, USA, and BGI (formerly Beijing Genome Initiative), China, have recently sequenced its genome.<sup><a reveal-id="DST039C6" data-open="DST039C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DST039C7" data-open="DST039C7" class="link link-ref link-reveal xref-bibr">7</a></sup> The availability of genomic sequence has motivated the scientific community to generate genomic resources, which ultimately had resulted in the development of large-scale sequence-based genomic<sup><a reveal-id="DST039C8" data-open="DST039C8" class="link link-ref link-reveal xref-bibr">8</a></sup> and genic expressed sequence tag (EST)-derived microsatellite markers.<sup><a reveal-id="DST039C9" data-open="DST039C9" class="link link-ref link-reveal xref-bibr">9</a></sup> Noteworthy, the recently developed ‘Foxtail millet Marker Database (FmMDb)’ (<a class="link link-uri" href="http://www.nipgr.res.in/foxtail.html">http://www.nipgr.res.in/foxtail.html</a>)<sup><a reveal-id="DST039C10" data-open="DST039C10" class="link link-ref link-reveal xref-bibr">10</a></sup> encompasses all these generated genomic resources for the benefit of research community aiming in genetic improvement of target millet and its related bioenergy crop species, thus bridging the gap between the researchers and breeders.</p><p>DNA markers such as restriction fragment-length polymorphism,<sup><a reveal-id="DST039C11" data-open="DST039C11" class="link link-ref link-reveal xref-bibr">11</a></sup> random amplified polymorphic DNA,<sup><a reveal-id="DST039C12" data-open="DST039C12" class="link link-ref link-reveal xref-bibr">12</a></sup> amplified fragment-length polymorphism,<sup><a reveal-id="DST039C13" data-open="DST039C13" class="link link-ref link-reveal xref-bibr">13</a></sup> simple sequence repeat polymorphism (SSR),<sup><a reveal-id="DST039C14" data-open="DST039C14" class="link link-ref link-reveal xref-bibr">14</a></sup> single-nucleotide polymorphism<sup><a reveal-id="DST039C15" data-open="DST039C15" class="link link-ref link-reveal xref-bibr">15</a></sup> and intron-length polymorphism (ILP)<sup><a reveal-id="DST039C16" data-open="DST039C16" class="link link-ref link-reveal xref-bibr">16</a></sup> exploit the variations, or polymorphisms in DNA sequences are used in various genotyping applications in crop plants. Of these, ILP markers are unique since they are gene-specific, co-dominant, hypervariable, neutral, convenient and reliable.<sup><a reveal-id="DST039C17" data-open="DST039C17" class="link link-ref link-reveal xref-bibr">17</a></sup> ILP markers utilize the variation in the intron sequences and are the most easily recognizable type as it could be detected by PCR with primers designed on exons flanking the target intron.<sup><a reveal-id="DST039C18" data-open="DST039C18" class="link link-ref link-reveal xref-bibr">18</a></sup> Thus, these markers in spite of being derived from gene sequences showed higher intra-specific polymorphism in plant species than other kinds of markers. In addition to being sequence-tagged sites markers,<sup><a reveal-id="DST039C19" data-open="DST039C19" class="link link-ref link-reveal xref-bibr">19</a></sup> ILP markers have high transferability rates among related plant species.<sup><a reveal-id="DST039C18" data-open="DST039C18" class="link link-ref link-reveal xref-bibr">18</a>,<a reveal-id="DST039C20" data-open="DST039C20" class="link link-ref link-reveal xref-bibr">20</a></sup> In order to facilitate straightforward mining of ILP markers, Yang <em>et al</em>.<sup><a reveal-id="DST039C17" data-open="DST039C17" class="link link-ref link-reveal xref-bibr">17</a></sup> has developed a web-based database platform named PIP (potential intron polymorphism) to provide detailed information of the PIP markers and homologous relationships among PIP markers from different species.</p><p>Regardless of these advantages, very few reports are available on development of ILP markers in plant species when compared with reports on other DNA markers. Noteworthy, till now only one study has been carried out on ILP markers in foxtail millet,<sup><a reveal-id="DST039C20" data-open="DST039C20" class="link link-ref link-reveal xref-bibr">20</a></sup> where about 98 markers were developed and characterized. Hence, in view of the importance of ILPs and the availability of less significant number of ILP markers in foxtail millet, the present study was conducted aiming at: (i) developing ILP markers at large-scale from the entire set of publicly available foxtail millet ESTs, (ii) demonstrating the applicability of the ILP markers in examining genetic diversity and cross-species transferability and (iii) developing physical map for studying <em>in silico</em> ILP marker-based comparative mapping between foxtail millet and other grass species such as sorghum, maize, rice and <em>Brachypodium</em>.</p>                            <h2 scrollto-destination=4913393 id="4913393" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=4913394 id="4913394" class="section-title" data-legacy-id=s2a>2.1. Plant material and DNA isolation</h3>
<p>The details of plant materials used in the study are summarized in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>. The seeds of all the investigated species were surface-sterilized in 3% sodium hypochlorite for 20 min, rinsed with sterile distilled water and were germinated in greenhouse. The genomic DNA was isolated from the fresh young leaves using the CTAB method as described elsewhere.<sup><a reveal-id="DST039C21" data-open="DST039C21" class="link link-ref link-reveal xref-bibr">21</a></sup> The DNA was purified and then quantified on agarose gel by comparison with 50 ng/μl of standard lambda (<em>λ</em>) DNA marker (NEB).</p>                            <h3 scrollto-destination=4913396 id="4913396" class="section-title" data-legacy-id=s2b>2.2. Development of putative ILP markers</h3>
<p>The publicly available EST sequences of <em>S. italica</em> were searched and retrieved from NCBI dbEST (<a class="link link-uri" href="http://ftp://ftp.ncbi.nih.gov/blast/db/">ftp://ftp.ncbi.nih.gov/blast/db/</a>). Approximately 66 027 ESTs were used for the unigene definition using the CD-HIT (Cluster Database at High Identity with Tolerance) software tool (<a class="link link-uri" href="http://weizhong-lab.ucsd.edu/cdhit_suite/cgi-bin/index.cgi">http://weizhong-lab.ucsd.edu/cdhit_suite/cgi-bin/index.cgi</a>) for redundancy minimization and assembling of sequences. The non-redundant ESTs were used for the development of specific intron-based markers. Using rice genomic sequence as reference, PIP database (<a class="link link-uri" href="http://ibi.zju.edu.cn/pgl/pip/">http://ibi.zju.edu.cn/pgl/pip/</a>)<sup><a reveal-id="DST039C17" data-open="DST039C17" class="link link-ref link-reveal xref-bibr">17</a></sup> was used to predict intron positions in the EST sequences and then designed a pair of primers flanking the intron position. A query EST was considered to be homologous to a subject-coding sequence only if there were at least 100 bp overlapping and 80% similarity between them. The corresponding position and length of identified introns from the subject species were obtained from the PIP database.<sup><a reveal-id="DST039C17" data-open="DST039C17" class="link link-ref link-reveal xref-bibr">17</a></sup> To cross-check the primer-designing potential of ILP markers and validate the results of the PIP database, the forward and reverse primers designed for the ILP markers were BLAST-searched against latest released foxtail millet pseudomolecules of nine chromosomes (<a class="link link-uri" href="http://www.phytozome.net">http://www.phytozome.net</a>).</p>                            <h3 scrollto-destination=4913398 id="4913398" class="section-title" data-legacy-id=s2c>2.3. Functional annotation and physical mapping of ILP markers</h3>
<p>The putative functions of the ILP markers were assigned by executing BLASTX search of respective marker encompassing EST sequences against the non-redundant database at NCBI (<a class="link link-uri" href="http://blast.ncbi.nlm.nih.gov/Blast.cgi">http://blast.ncbi.nlm.nih.gov/Blast.cgi</a>) with default search parameters. The ILP markers were BLAST-searched against the whole-genome sequences of foxtail millet available at Phytozome (<a class="link link-uri" href="http://www.phytozome.net">http://www.phytozome.net</a>) and plotted individually on each of the nine foxtail millet chromosomes according to their ascending order of physical position (bp), from the short-arm telomere to the long-arm telomere and finally visualized in the MapChart software.<sup><a reveal-id="DST039C22" data-open="DST039C22" class="link link-ref link-reveal xref-bibr">22</a></sup> To further validate the BLAST results of <em>in silico</em> physical mapping, we re-analysed the forward and reverse primers of ILP markers against foxtail millet chromosome pseudomolecules using the ePCR program (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/sutils/e-pcr">www.ncbi.nlm.nih.gov/sutils/e-pcr</a>).<sup><a reveal-id="DST039C23" data-open="DST039C23" class="link link-ref link-reveal xref-bibr">23</a></sup></p>                            <h3 scrollto-destination=4913400 id="4913400" class="section-title" data-legacy-id=s2d>2.4. Validation of ILP markers</h3>
<p>The ILPs were amplified in a 25 µl total volume containing 1 unit of Taq DNA polymerase (Sigma), 50 ng of genomic DNA, 10 µmol/l of each primer, 0.5 mmol/l of each dNTPs and 2.5 µl of 10× PCR reaction buffer [500 mM KCl, 200 mM Tris–HCl (pH 8.4) and 3 mM MgCl<sub>2</sub>] in an iCycler thermal controller (Bio-Rad). The PCR profile was an initial denaturation of 3 min at 94°C, followed by 35 cycles of 60 s at 94°C, 60 s at 50–55°C and 2 min at 72°C, and a final extension of 10 min at 72°C. The amplicons were resolved on 2% agarose gel (Cambrex, USA) in Tris-borate EDTA buffer (pH 8.0), stained with ethidium bromide and analysed using GelDoc-It™ imaging system (UVP). The fragment size for each locus was determined by 100 bp standard size markers (NEB). Results were confirmed by three replicate assays.</p><p>The amplified products (alleles) from millet and non-millet species were eluted and cloned into pGEM<sup>(R)</sup>-T Easy vector (Promega) following the manufacturer's instructions. The recombinant plasmids were purified using <em>AccuPrep</em> Plasmid MiniPrep DNA Extraction Kit (Bioneer) following the manufacturer's protocol. The plasmids were sequenced in an automated sequencer (3730xI DNA Analyzer, Applied Biosystems) using M13 forward and reverse primers. The sequence information was used to construct multiple sequence alignment along with a reference <em>S. italica</em> sequence using the ClustalW2 program (<a class="link link-uri" href="http://www.ebi.ac.uk/Tools/clustalw2/index.html">http://www.ebi.ac.uk/Tools/clustalw2/index.html</a>).</p>                            <h3 scrollto-destination=4913403 id="4913403" class="section-title" data-legacy-id=s2e>2.5. Genetic diversity</h3>
<p>The ILP marker profiles amplified among 96 foxtail millet accessions were scored manually; each allele was scored as present (1) or absent (0) for each of the ILP loci. Polymorphic informative content (PIC) values were calculated according to Roldán-Ruiz <em>et al</em>.<sup><a reveal-id="DST039C24" data-open="DST039C24" class="link link-ref link-reveal xref-bibr">24</a></sup> as PIC<em><sub>i</sub></em> = 2<em>f<sub>i</sub></em>(1 − <em>f<sub>i</sub></em>), where <em>f<sub>i</sub></em> is the frequency of the amplified allele (band present), and (1 − <em>f<sub>i</sub></em>) is the frequency of the null allele (band absent) of marker <em>i.</em> Using pairwise similarity matrix of Jaccard's coefficient,<sup><a reveal-id="DST039C25" data-open="DST039C25" class="link link-ref link-reveal xref-bibr">25</a></sup> the level of genetic diversity among foxtail millet accessions was calculated and a phylogenetic tree was constructed by unweighted pair-group method of arithmetic average, neighbour-joining (NJoin) module of the NTSYS-pc software v2.02.<sup><a reveal-id="DST039C26" data-open="DST039C26" class="link link-ref link-reveal xref-bibr">26</a></sup> The genetic relationships among millets and non-millet grass species based on cross-transferability of ILP markers were determined based on Nei (1983)<sup><a reveal-id="DST039C27" data-open="DST039C27" class="link link-ref link-reveal xref-bibr">27</a></sup> diversity co-efficient, and a phylogenetic tree was constructed using the neighbour-joining (NJ) tree interface of the PowerMarker software ver2.5.<sup><a reveal-id="DST039C28" data-open="DST039C28" class="link link-ref link-reveal xref-bibr">28</a></sup> The observed heterozygosity (<em>H</em><sub>O</sub>), Nei's average gene diversity<sup><a reveal-id="DST039C27" data-open="DST039C27" class="link link-ref link-reveal xref-bibr">27</a></sup> and fixation index (<em>F</em><sub>IS</sub>) were also computed using the PowerMarker software ver2.5.<sup><a reveal-id="DST039C28" data-open="DST039C28" class="link link-ref link-reveal xref-bibr">28</a></sup> Correlation analysis among PIC and the number of alleles were examined using the GraphPad InStat software v3.10 (<a class="link link-uri" href="http://www.graphpad.com">www.graphpad.com</a>).</p>                            <h3 scrollto-destination=4913405 id="4913405" class="section-title" data-legacy-id=s2f>2.6. <em>In silico</em> comparative genome mapping</h3>
<p>The ILP markers that were physically mapped on the nine chromosomes of foxtail millet were BLASTN-searched against genome sequences of sorghum, maize, rice and <em>Brachypodium</em> (<a class="link link-uri" href="http://www.phytozome.net">www.phytozome.net</a>) to develop marker-based syntenic relationships among the chromosomes of foxtail millet and three other grass species. A cut-off bit score of 54.7 and an <em>E</em>-value of &lt;1e − 05 were considered optimum for BLASTN analysis. The marker-based syntenic relationships among foxtail millet, sorghum, maize, rice and <em>Brachypodium</em> were finally visualized with visualization blocks in the Circos software v0.55 (<a class="link link-uri" href="http://circos.ca">http://circos.ca</a>).<sup><a reveal-id="DST039C29" data-open="DST039C29" class="link link-ref link-reveal xref-bibr">29</a></sup></p>                            <h2 scrollto-destination=4913407 id="4913407" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=4913408 id="4913408" class="section-title" data-legacy-id=s3a>3.1. Development of ILP markers and physical mapping in foxtail millet genome</h3>
<p>A set of 66 027 EST sequences of <em>S. italica</em> produced 24 828 non-redundant ESTs, which were used for generating ILP markers by using rice as reference genome in the PIP database.<sup><a reveal-id="DST039C17" data-open="DST039C17" class="link link-ref link-reveal xref-bibr">17</a></sup> A total of 5123 ILP markers (20.6%) were generated out of 24 828 EST sequences with an average frequency of ∼12.6 ILP markers per megabase genomic sequences (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). BLAST analysis of 5123 ILP markers against the foxtail millet genome showed the presence of all the markers in the genome, and the determination of genomic distribution of these 5123 ILP markers on the foxtail millet genome revealed physical localization of 4049 markers on the nine chromosomes of foxtail millet with an average marker density of 9.8 markers/Mb (Fig. <a reveal-id="DST039F1" data-open="DST039F1" class="link link-reveal link-table xref-fig">1</a>; Table <a reveal-id="DST039TB1" data-open="DST039TB1" class="link link-reveal link-table xref-fig">1</a>). The average marker density was maximum (14.1/Mb) in chromosome 9, followed by chromosome 5 (11.8/Mb), and minimum in chromosome 8 (5.7/Mb). An extensive analysis of chromosome-wise distribution and frequency of these physically mapped ILP markers showed higher frequency of markers mapped on chromosome 9 (831 markers, 20.5%) and minimum on chromosome 8 (230, 5.7%) (Fig. <a reveal-id="DST039F1" data-open="DST039F1" class="link link-reveal link-table xref-fig">1</a>; Table <a reveal-id="DST039TB1" data-open="DST039TB1" class="link link-reveal link-table xref-fig">1</a>). </p>                        <a id="4913410" scrollto-destination="4913410"></a>
<div content-id="DST039TB1" class="table-modal"><div class="table-wrap"><div id="DST039TB1" class="table-wrap-title" data-id="DST039TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Summary of chromosomal distribution and average physical density of ILP markers mapped on the nine chromosomes of foxtail millet</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Chromosome </th><th>Markers mapped (%) </th><th>Physical density (markers/Mb) </th></tr></thead><tbody><tr><td>Chr.1 </td><td>457 (11.3) </td><td>10.9 </td></tr><tr><td>Chr.2 </td><td>506 (12.5) </td><td>10.3 </td></tr><tr><td>Chr.3 </td><td>487 (12) </td><td>9.6 </td></tr><tr><td>Chr.4 </td><td>325 (8) </td><td>8.1 </td></tr><tr><td>Chr.5 </td><td>558 (13.8) </td><td>11.8 </td></tr><tr><td>Chr.6 </td><td>278 (6.9) </td><td>7.7 </td></tr><tr><td>Chr.7 </td><td>377 (9.3) </td><td>10.5 </td></tr><tr><td>Chr.8 </td><td>230 (5.7) </td><td>5.7 </td></tr><tr><td>Chr.9 </td><td>831 (20.5) </td><td>14.1 </td></tr><tr><td colspan="2">Average </td><td>9.8 </td></tr></tbody></table></div><div content-id="DST039TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Chromosome </th><th>Markers mapped (%) </th><th>Physical density (markers/Mb) </th></tr></thead><tbody><tr><td>Chr.1 </td><td>457 (11.3) </td><td>10.9 </td></tr><tr><td>Chr.2 </td><td>506 (12.5) </td><td>10.3 </td></tr><tr><td>Chr.3 </td><td>487 (12) </td><td>9.6 </td></tr><tr><td>Chr.4 </td><td>325 (8) </td><td>8.1 </td></tr><tr><td>Chr.5 </td><td>558 (13.8) </td><td>11.8 </td></tr><tr><td>Chr.6 </td><td>278 (6.9) </td><td>7.7 </td></tr><tr><td>Chr.7 </td><td>377 (9.3) </td><td>10.5 </td></tr><tr><td>Chr.8 </td><td>230 (5.7) </td><td>5.7 </td></tr><tr><td>Chr.9 </td><td>831 (20.5) </td><td>14.1 </td></tr><tr><td colspan="2">Average </td><td>9.8 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4913410" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DST039TB1" class="table-wrap-title" data-id="DST039TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Summary of chromosomal distribution and average physical density of ILP markers mapped on the nine chromosomes of foxtail millet</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Chromosome </th><th>Markers mapped (%) </th><th>Physical density (markers/Mb) </th></tr></thead><tbody><tr><td>Chr.1 </td><td>457 (11.3) </td><td>10.9 </td></tr><tr><td>Chr.2 </td><td>506 (12.5) </td><td>10.3 </td></tr><tr><td>Chr.3 </td><td>487 (12) </td><td>9.6 </td></tr><tr><td>Chr.4 </td><td>325 (8) </td><td>8.1 </td></tr><tr><td>Chr.5 </td><td>558 (13.8) </td><td>11.8 </td></tr><tr><td>Chr.6 </td><td>278 (6.9) </td><td>7.7 </td></tr><tr><td>Chr.7 </td><td>377 (9.3) </td><td>10.5 </td></tr><tr><td>Chr.8 </td><td>230 (5.7) </td><td>5.7 </td></tr><tr><td>Chr.9 </td><td>831 (20.5) </td><td>14.1 </td></tr><tr><td colspan="2">Average </td><td>9.8 </td></tr></tbody></table></div><div content-id="DST039TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Chromosome </th><th>Markers mapped (%) </th><th>Physical density (markers/Mb) </th></tr></thead><tbody><tr><td>Chr.1 </td><td>457 (11.3) </td><td>10.9 </td></tr><tr><td>Chr.2 </td><td>506 (12.5) </td><td>10.3 </td></tr><tr><td>Chr.3 </td><td>487 (12) </td><td>9.6 </td></tr><tr><td>Chr.4 </td><td>325 (8) </td><td>8.1 </td></tr><tr><td>Chr.5 </td><td>558 (13.8) </td><td>11.8 </td></tr><tr><td>Chr.6 </td><td>278 (6.9) </td><td>7.7 </td></tr><tr><td>Chr.7 </td><td>377 (9.3) </td><td>10.5 </td></tr><tr><td>Chr.8 </td><td>230 (5.7) </td><td>5.7 </td></tr><tr><td>Chr.9 </td><td>831 (20.5) </td><td>14.1 </td></tr><tr><td colspan="2">Average </td><td>9.8 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4913410" target="_blank">View Large</a></div></div>                        <a id="4913411" scrollto-destination="4913411"></a>
<div id="DST039F1" data-id="DST039F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst039/2/m_dst03901.png?Expires=2147483647&Signature=inZxm3Y6Ep0Tf2mGT2DjGpIhmY0cWBzUKQSRxJk5W9xhDI3fKnHTq7XrufJSZTV-CtgxlSZf2dxXNE~1TxcptiM8bdDSeP-40Cjf9Z35vFSpZzmbpPMD0LwKWlcIAKc8ex79xkbUJUhZdjC~4q48TxEP0pQ5S9oJy-~KZxRN8Kj3jnTbrw~5aM1v~n4f5cVhpNU4g7J7Ne8VjItDowaILyBQD3VVpdDvCo6ca5TX07pUs81p6Y~3Hrtuy9IysAKIspR-MbGVqEGQsUvkT20jmuiMRX~unKhkXZFKf36-~rpPyBTnZEitCGy7LVXQocSoxpcn6btNjTiv60YEXjQYfw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Physical genomic distribution of 4049 ILP markers on the nine chromosomes of foxtail millet genome. The number on the right side of the bar denotes the identity of foxtail millet ILP markers (SiILP, Setaria italica intron-length polymorphic marker) and that on the left side indicates the physical position of mapped markers in mega base-pairs (Mb)." path-from-xml="dst03901.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4913411/dst03901.tiff" path-from-xml="dst03901.tiff" target="_blank">View large</a><a section="4913411" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst039/2/dst03901.png?Expires=2147483647&Signature=o4F7lscIyLYAfMhA2xZtVkX96JkHzlSOJD5Vv9d1I33bER8Giu~ZAO56M3pv7ZQf0tiRIohZybB3rNMWGRej7ZOdWYujjlaDvf~YMVq0te~V7dp4U4es4RCtRTiXniST36NN-xQQ-upXk006VpXI~ja92PbyUN0HjyXI4NXbu5MyTX0rrn2YWKeEY30P9n7TosRGDIzSBMTRWZZ4jzmyikeg51sdoX3d~PipcndQQy4Rns9v5q1nk0uQlG~16TRqtrvzghbU~PF7r4J-WE-VUxg7-Fa1ShLTJ4yIu3CFuU-IBY9~~PV~Ura9C1t3J3vRjCu8rG4-igDG1T-JojnL2g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4913411&ar=347328&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst03901.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Physical genomic distribution of 4049 ILP markers on the nine chromosomes of foxtail millet genome. The number on the right side of the bar denotes the identity of foxtail millet ILP markers (SiILP, <em>Setaria italica</em> intron-length polymorphic marker) and that on the left side indicates the physical position of mapped markers in mega base-pairs (Mb).</p></div></div><div content-id="DST039F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst039/2/m_dst03901.png?Expires=2147483647&Signature=inZxm3Y6Ep0Tf2mGT2DjGpIhmY0cWBzUKQSRxJk5W9xhDI3fKnHTq7XrufJSZTV-CtgxlSZf2dxXNE~1TxcptiM8bdDSeP-40Cjf9Z35vFSpZzmbpPMD0LwKWlcIAKc8ex79xkbUJUhZdjC~4q48TxEP0pQ5S9oJy-~KZxRN8Kj3jnTbrw~5aM1v~n4f5cVhpNU4g7J7Ne8VjItDowaILyBQD3VVpdDvCo6ca5TX07pUs81p6Y~3Hrtuy9IysAKIspR-MbGVqEGQsUvkT20jmuiMRX~unKhkXZFKf36-~rpPyBTnZEitCGy7LVXQocSoxpcn6btNjTiv60YEXjQYfw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Physical genomic distribution of 4049 ILP markers on the nine chromosomes of foxtail millet genome. The number on the right side of the bar denotes the identity of foxtail millet ILP markers (SiILP, Setaria italica intron-length polymorphic marker) and that on the left side indicates the physical position of mapped markers in mega base-pairs (Mb)." path-from-xml="dst03901.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4913411/dst03901.tiff" path-from-xml="dst03901.tiff" target="_blank">View large</a><a section="4913411" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst039/2/dst03901.png?Expires=2147483647&Signature=o4F7lscIyLYAfMhA2xZtVkX96JkHzlSOJD5Vv9d1I33bER8Giu~ZAO56M3pv7ZQf0tiRIohZybB3rNMWGRej7ZOdWYujjlaDvf~YMVq0te~V7dp4U4es4RCtRTiXniST36NN-xQQ-upXk006VpXI~ja92PbyUN0HjyXI4NXbu5MyTX0rrn2YWKeEY30P9n7TosRGDIzSBMTRWZZ4jzmyikeg51sdoX3d~PipcndQQy4Rns9v5q1nk0uQlG~16TRqtrvzghbU~PF7r4J-WE-VUxg7-Fa1ShLTJ4yIu3CFuU-IBY9~~PV~Ura9C1t3J3vRjCu8rG4-igDG1T-JojnL2g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4913411&ar=347328&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst03901.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Physical genomic distribution of 4049 ILP markers on the nine chromosomes of foxtail millet genome. The number on the right side of the bar denotes the identity of foxtail millet ILP markers (SiILP, <em>Setaria italica</em> intron-length polymorphic marker) and that on the left side indicates the physical position of mapped markers in mega base-pairs (Mb).</p></div></div>                            <h3 scrollto-destination=4913412 id="4913412" class="section-title" data-legacy-id=s3b>3.2. Functional annotation of ILP</h3>
<p>BLASTX analyses of the 5123 EST sequences suggested a nearly defined function for ∼71.5% of ILP markers, and 28.5% had no similarities to previously sequenced genes. Based on the function, the ILP markers with defined function (71.5%) were grouped into five major categories (Fig. <a reveal-id="DST039F2" data-open="DST039F2" class="link link-reveal link-table xref-fig">2</a>). The largest category (47.4%) contained EST sequences with hypothetical/uncharacterized/putative functions. The second largest category (26.2%) comprised stress-related transcripts. The housekeeping proteins (9.5) ranked third, followed by protein kinases (8.6%) and transcription factors (8.3%) (Fig. <a reveal-id="DST039F2" data-open="DST039F2" class="link link-reveal link-table xref-fig">2</a>). Foxtail millet being a potentially abiotic stress-tolerant crop particularly towards drought and salinity, investigating the EST sequences with hypothetical/uncharacterized/putative functions (47.4%) and those which no similarities to previously sequenced genes (28.5%) could provide novel clues on the stress tolerance mechanisms. </p>                        <a id="4913414" scrollto-destination="4913414"></a>
<div id="DST039F2" data-id="DST039F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst039/2/m_dst03902.png?Expires=2147483647&Signature=m997K8PA7WXk-sKGr7lmJwTYGXzY5~nj2LVNGNamqzsJfTKSl~m8EdrLcBI~1~H6eNlTlLbaOt19t8gsDutGPwVsiK4AL8E5fpc5XLSLoTPMhrtdu4qHJPEchiOIGNoEgzH-mFahR9diVefKq140O-rAMw1nCWFY6mSCARQkPF6hGggKAlA0NdGxpWIoA-K2PIXYnspJm9NBtDnREN7qxkuliAu8W7PjCzloGrIlNB1l6GLFws~QeP9YB3c3Xd3eSmajA3mpN-MZZZu4~WweXttVno2chokIfqB5Ow2H2sd~UgmrjgDYQ1Rc7-HhAER2V84shQs1JOUbo1zga3JEEA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Functional classification of 5123 ILP markers containing ESTs/genes. The unique genes were grouped into five functional groups." path-from-xml="dst03902.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4913414/dst03902.tiff" path-from-xml="dst03902.tiff" target="_blank">View large</a><a section="4913414" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst039/2/dst03902.png?Expires=2147483647&Signature=SgEQkrQ05xHXOM~u7Ebo~jv3J9QZnlan3VqhtyOd~5HpC7jppeg6wCStVNjcvVuSCjW29SPchlI86i85GvdmNhRPMbRWeZu2h6ebnU-d3lIHCSmI8-KXt3N0UBcFVWAmgjHcCNVxAaCeyPoa76~YMe~GgFzgtEA7tQigqcEPQ8EO2BVNm6gmJ~Zzr04qygO8edu4Br~ciwTH-09~Igz-w6z5Rm3NRaxyXKyU5~a~496Adjto42oSFOtzS0yWUhcjr2Ogro5Hjn9tRu8kqFdRJfSOARZFWYV5mwjObCzd6AWoMv1QDw-Z4mG0WKaMsUZ88Vno9zDLN7bG9W~xxcD5zA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4913414&ar=347328&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst03902.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Functional classification of 5123 ILP markers containing ESTs/genes<em>.</em> The unique genes were grouped into five functional groups.</p></div></div><div content-id="DST039F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst039/2/m_dst03902.png?Expires=2147483647&Signature=m997K8PA7WXk-sKGr7lmJwTYGXzY5~nj2LVNGNamqzsJfTKSl~m8EdrLcBI~1~H6eNlTlLbaOt19t8gsDutGPwVsiK4AL8E5fpc5XLSLoTPMhrtdu4qHJPEchiOIGNoEgzH-mFahR9diVefKq140O-rAMw1nCWFY6mSCARQkPF6hGggKAlA0NdGxpWIoA-K2PIXYnspJm9NBtDnREN7qxkuliAu8W7PjCzloGrIlNB1l6GLFws~QeP9YB3c3Xd3eSmajA3mpN-MZZZu4~WweXttVno2chokIfqB5Ow2H2sd~UgmrjgDYQ1Rc7-HhAER2V84shQs1JOUbo1zga3JEEA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Functional classification of 5123 ILP markers containing ESTs/genes. The unique genes were grouped into five functional groups." path-from-xml="dst03902.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4913414/dst03902.tiff" path-from-xml="dst03902.tiff" target="_blank">View large</a><a section="4913414" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst039/2/dst03902.png?Expires=2147483647&Signature=SgEQkrQ05xHXOM~u7Ebo~jv3J9QZnlan3VqhtyOd~5HpC7jppeg6wCStVNjcvVuSCjW29SPchlI86i85GvdmNhRPMbRWeZu2h6ebnU-d3lIHCSmI8-KXt3N0UBcFVWAmgjHcCNVxAaCeyPoa76~YMe~GgFzgtEA7tQigqcEPQ8EO2BVNm6gmJ~Zzr04qygO8edu4Br~ciwTH-09~Igz-w6z5Rm3NRaxyXKyU5~a~496Adjto42oSFOtzS0yWUhcjr2Ogro5Hjn9tRu8kqFdRJfSOARZFWYV5mwjObCzd6AWoMv1QDw-Z4mG0WKaMsUZ88Vno9zDLN7bG9W~xxcD5zA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4913414&ar=347328&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst03902.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Functional classification of 5123 ILP markers containing ESTs/genes<em>.</em> The unique genes were grouped into five functional groups.</p></div></div>                            <h3 scrollto-destination=4913415 id="4913415" class="section-title" data-legacy-id=s3c>3.3. Marker validation, cross-genera transferability and genetic basis of sequence length variation</h3>
<p>To amplify introns by PCR, the exon-primed intron-crossing PCR (EPIC-PCR) method was used,<sup><a reveal-id="DST039C30" data-open="DST039C30" class="link link-ref link-reveal xref-bibr">30</a></sup> where primers were designed in flanking exons using the PIP database. The advantages of EPIC-PCR are it is fast, reliable, reproducible and convenient, providing ready-to-use and clearly intelligible results. Hence, from the 4049 physically mapped ILP markers, 440 primer pairs flanking the exons were chosen for further analyses based on two criteria, viz. representing the whole genome of foxtail millet and the function of the EST. All the ILP markers were evidenced to produce clear, successful and reproducible amplification in <em>S. italica</em> cv. Prasad with 100% amplification potential (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). This demonstrates the significance of the developed 4049 ILP markers in expediting foxtail millet genomics and molecular breeding. About 391 (∼90%) of the 440 ILP markers amplified unique single allele, while 49 markers amplified more than one allele/multiple alleles. Thus, a total of 495 alleles were amplified by 440 ILP markers in <em>S. italica</em> cv. Prasad (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). All the 440 ILP markers have the ability to distinguish the investigated millet and non-millet species into two distinct groups (Fig. <a reveal-id="DST039F3" data-open="DST039F3" class="link link-reveal link-table xref-fig">3</a>). </p>                        <a id="4913417" scrollto-destination="4913417"></a>
<div id="DST039F3" data-id="DST039F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst039/2/m_dst03903.png?Expires=2147483647&Signature=Ye~ANaZ02JmeF6reGe56KQV541IbPT~VLkV-oHZXrnwDXJ39nZ1E179q7H0llv8Qqc8xPrbVnLBUCD49N-CZrPIVOoOeaA~Kc80rO~owEnbBvJEh3j0ceXlSxxkx4AnAQImGXeNE2pye9WoCI0JRw-4TrcVospIbMfsL73gC6wRPLa-K9Fyn7aZTuPAqav-1ae9N1SyMA3YhWeM659LyNfJu2R~WFsSvQOo~115fkKEGolEtWZMEvmI8wKxrjOG7gqdtl54gC8BJPxixchsumKuNjtIvaLRfZ~rKSHZYyrdjNwOQm3FwtvasUynCjWU-H6RzPk6M2gRX30iaQNMkCg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genetic relationships among millet and non-millet grass species based on 43 foxtail millet ILP markers, using NJoin clustering. Nine millet species including foxtail millet clearly differentiated from the five non-millet grass species and expected genetic relationships among species under study were also apparent." path-from-xml="dst03903.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4913417/dst03903.tiff" path-from-xml="dst03903.tiff" target="_blank">View large</a><a section="4913417" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst039/2/dst03903.png?Expires=2147483647&Signature=oggjnZVyTdQIn~LEwJhLPEYAKIjH0Q9H6~4-X2Mqm1ZOEQvgAvpNUCZfnM2aXhpLp~VodUmd~3mUgA2eWmfvCPsaBkbsu1iwKIpNylTZW~z0R9pFcWCAz3mNuV6PiJf7BZuzcZparkbaoublXmguci5ElVK0z4x1SMSxd7OcWkC7-~TkKgmm2w43hE1i6qAr3J-0FCypxiniiQyGyVIrpQRlpKy6SOAKyWIVcgoKybpm7n89~bY7iFPxplQhdIU3DqXaLRfuo~ptd6fyNltOxY1uNE24NnjE9zxhq3j8DI0ZFHnP5H1SJKKiaXnqq2YoxOL18FLnwJXJ9Cwpzce0QA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4913417&ar=347328&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst03903.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genetic relationships among millet and non-millet grass species based on 43 foxtail millet ILP markers, using NJoin clustering. Nine millet species including foxtail millet clearly differentiated from the five non-millet grass species and expected genetic relationships among species under study were also apparent.</p></div></div><div content-id="DST039F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst039/2/m_dst03903.png?Expires=2147483647&Signature=Ye~ANaZ02JmeF6reGe56KQV541IbPT~VLkV-oHZXrnwDXJ39nZ1E179q7H0llv8Qqc8xPrbVnLBUCD49N-CZrPIVOoOeaA~Kc80rO~owEnbBvJEh3j0ceXlSxxkx4AnAQImGXeNE2pye9WoCI0JRw-4TrcVospIbMfsL73gC6wRPLa-K9Fyn7aZTuPAqav-1ae9N1SyMA3YhWeM659LyNfJu2R~WFsSvQOo~115fkKEGolEtWZMEvmI8wKxrjOG7gqdtl54gC8BJPxixchsumKuNjtIvaLRfZ~rKSHZYyrdjNwOQm3FwtvasUynCjWU-H6RzPk6M2gRX30iaQNMkCg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genetic relationships among millet and non-millet grass species based on 43 foxtail millet ILP markers, using NJoin clustering. Nine millet species including foxtail millet clearly differentiated from the five non-millet grass species and expected genetic relationships among species under study were also apparent." path-from-xml="dst03903.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4913417/dst03903.tiff" path-from-xml="dst03903.tiff" target="_blank">View large</a><a section="4913417" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst039/2/dst03903.png?Expires=2147483647&Signature=oggjnZVyTdQIn~LEwJhLPEYAKIjH0Q9H6~4-X2Mqm1ZOEQvgAvpNUCZfnM2aXhpLp~VodUmd~3mUgA2eWmfvCPsaBkbsu1iwKIpNylTZW~z0R9pFcWCAz3mNuV6PiJf7BZuzcZparkbaoublXmguci5ElVK0z4x1SMSxd7OcWkC7-~TkKgmm2w43hE1i6qAr3J-0FCypxiniiQyGyVIrpQRlpKy6SOAKyWIVcgoKybpm7n89~bY7iFPxplQhdIU3DqXaLRfuo~ptd6fyNltOxY1uNE24NnjE9zxhq3j8DI0ZFHnP5H1SJKKiaXnqq2YoxOL18FLnwJXJ9Cwpzce0QA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4913417&ar=347328&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst03903.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genetic relationships among millet and non-millet grass species based on 43 foxtail millet ILP markers, using NJoin clustering. Nine millet species including foxtail millet clearly differentiated from the five non-millet grass species and expected genetic relationships among species under study were also apparent.</p></div></div><p>From these validated set of 440 ILP markers, about 100 markers representing the whole foxtail millet genome were chosen to evaluate its polymorphism and molecular diversity potential in a set of eight accessions of <em>Setaria</em> including five cultivated and three wild species. About 45 (45%) SiILP markers showed polymorphism and a total of 163 alleles ranging from 1 to 5 alleles were amplified by candidate SiILP markers with an average of 2.15 alleles per marker locus (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>). The polymorphic potential (∼45%) of ILP markers estimated among foxtail millet cultivated and wild accessions was higher than that reported using the ILP markers derived from foxtail millet genomic sequences.<sup><a reveal-id="DST039C20" data-open="DST039C20" class="link link-ref link-reveal xref-bibr">20</a></sup> The higher polymorphic potential of ILP markers is expected because of their development targeting the hypervariable introns, which are under less selective pressure.</p><p>In order to investigate the utility of the ILP markers in cross-genera transferability, the 100 validated set of ILP markers were used to amplify the genomic DNA of millet (barnyard millet, finger millet, kodo millet, little millet, pearl millet, proso millet, guinea grass) and non-millet species (switchgrass, sorghum, maize, rice, <em>Brachypodium</em>) (Table <a reveal-id="DST039TB2" data-open="DST039TB2" class="link link-reveal link-table xref-fig">2</a>; Fig. <a reveal-id="DST039F4" data-open="DST039F4" class="link link-reveal link-table xref-fig">4</a>). Of the 100 SiILP markers assayed, the highest transferability percentage (98%) was observed in proso millet and lowest (59.4%) in wheat, with an average percent transferability of ∼85% (Table <a reveal-id="DST039TB2" data-open="DST039TB2" class="link link-reveal link-table xref-fig">2</a>). Markers which showed a consistent amplification profile in other species were scored as being cross-transferable, thus confirming the utility of developed ILP markers for revealing high cross-genera transferability. To gain further insight into the molecular basis of cross-transferability of ILP markers, the paralogous relationships of 4049 markers orthologous between foxtail millet and rice genes were determined. Three hundred and forty-seven (8.6%) of 4049 orthologous ILP markers were present in paralogous foxtail millet genes, while 80 (2%) were found in paralogous rice genes. It inferred that 3622 ILP markers designed in this study are unique either in the foxtail millet or in the rice genome. Interestingly, the 100 ILP markers showing orthologous relationships between foxtail millet and rice genes selected for cross-transferability study did not show any paralogous relationships within either rice or foxtail millet genome. This confirms the higher cross-transferability potential of ILP markers due to orthologous relationships among species rather than paralogy within species. </p>                        <a id="4913420" scrollto-destination="4913420"></a>
<div content-id="DST039TB2" class="table-modal"><div class="table-wrap"><div id="DST039TB2" class="table-wrap-title" data-id="DST039TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Percent transferability of 100 ILP markers in different millet and non-millet species</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>No. </th><th>Investigated crop </th><th>% Transferability </th></tr></thead><tbody><tr><td>1 </td><td>Barnyard millet </td><td>94.1 </td></tr><tr><td>2 </td><td>Finger millet </td><td>81.2 </td></tr><tr><td>3 </td><td>Kodo millet </td><td>74.3 </td></tr><tr><td>4 </td><td>Little millet </td><td>97.0 </td></tr><tr><td>5 </td><td>Pearl millet </td><td>96.0 </td></tr><tr><td>6 </td><td>Proso millet </td><td>98.0 </td></tr><tr><td>7 </td><td>Guinea grass </td><td>91.1 </td></tr><tr><td>8 </td><td>Switchgrass </td><td>86.1 </td></tr><tr><td>9 </td><td><em>Brachypodium</em> </td><td>67.3 </td></tr><tr><td>10 </td><td>Sorghum </td><td>89.1 </td></tr><tr><td>11 </td><td>Wheat </td><td>59.4 </td></tr><tr><td>12 </td><td>Rice </td><td>81.2 </td></tr><tr><td>13 </td><td>Maize </td><td>88.1 </td></tr><tr><td colspan="2">Average </td><td>84.8 </td></tr></tbody></table></div><div content-id="DST039TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>No. </th><th>Investigated crop </th><th>% Transferability </th></tr></thead><tbody><tr><td>1 </td><td>Barnyard millet </td><td>94.1 </td></tr><tr><td>2 </td><td>Finger millet </td><td>81.2 </td></tr><tr><td>3 </td><td>Kodo millet </td><td>74.3 </td></tr><tr><td>4 </td><td>Little millet </td><td>97.0 </td></tr><tr><td>5 </td><td>Pearl millet </td><td>96.0 </td></tr><tr><td>6 </td><td>Proso millet </td><td>98.0 </td></tr><tr><td>7 </td><td>Guinea grass </td><td>91.1 </td></tr><tr><td>8 </td><td>Switchgrass </td><td>86.1 </td></tr><tr><td>9 </td><td><em>Brachypodium</em> </td><td>67.3 </td></tr><tr><td>10 </td><td>Sorghum </td><td>89.1 </td></tr><tr><td>11 </td><td>Wheat </td><td>59.4 </td></tr><tr><td>12 </td><td>Rice </td><td>81.2 </td></tr><tr><td>13 </td><td>Maize </td><td>88.1 </td></tr><tr><td colspan="2">Average </td><td>84.8 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4913420" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DST039TB2" class="table-wrap-title" data-id="DST039TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Percent transferability of 100 ILP markers in different millet and non-millet species</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>No. </th><th>Investigated crop </th><th>% Transferability </th></tr></thead><tbody><tr><td>1 </td><td>Barnyard millet </td><td>94.1 </td></tr><tr><td>2 </td><td>Finger millet </td><td>81.2 </td></tr><tr><td>3 </td><td>Kodo millet </td><td>74.3 </td></tr><tr><td>4 </td><td>Little millet </td><td>97.0 </td></tr><tr><td>5 </td><td>Pearl millet </td><td>96.0 </td></tr><tr><td>6 </td><td>Proso millet </td><td>98.0 </td></tr><tr><td>7 </td><td>Guinea grass </td><td>91.1 </td></tr><tr><td>8 </td><td>Switchgrass </td><td>86.1 </td></tr><tr><td>9 </td><td><em>Brachypodium</em> </td><td>67.3 </td></tr><tr><td>10 </td><td>Sorghum </td><td>89.1 </td></tr><tr><td>11 </td><td>Wheat </td><td>59.4 </td></tr><tr><td>12 </td><td>Rice </td><td>81.2 </td></tr><tr><td>13 </td><td>Maize </td><td>88.1 </td></tr><tr><td colspan="2">Average </td><td>84.8 </td></tr></tbody></table></div><div content-id="DST039TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>No. </th><th>Investigated crop </th><th>% Transferability </th></tr></thead><tbody><tr><td>1 </td><td>Barnyard millet </td><td>94.1 </td></tr><tr><td>2 </td><td>Finger millet </td><td>81.2 </td></tr><tr><td>3 </td><td>Kodo millet </td><td>74.3 </td></tr><tr><td>4 </td><td>Little millet </td><td>97.0 </td></tr><tr><td>5 </td><td>Pearl millet </td><td>96.0 </td></tr><tr><td>6 </td><td>Proso millet </td><td>98.0 </td></tr><tr><td>7 </td><td>Guinea grass </td><td>91.1 </td></tr><tr><td>8 </td><td>Switchgrass </td><td>86.1 </td></tr><tr><td>9 </td><td><em>Brachypodium</em> </td><td>67.3 </td></tr><tr><td>10 </td><td>Sorghum </td><td>89.1 </td></tr><tr><td>11 </td><td>Wheat </td><td>59.4 </td></tr><tr><td>12 </td><td>Rice </td><td>81.2 </td></tr><tr><td>13 </td><td>Maize </td><td>88.1 </td></tr><tr><td colspan="2">Average </td><td>84.8 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4913420" target="_blank">View Large</a></div></div>                        <a id="4913421" scrollto-destination="4913421"></a>
<div id="DST039F4" data-id="DST039F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst039/2/m_dst03904.png?Expires=2147483647&Signature=HlzdsmAqanbt5zQ28P1BIGDNRPOrD-7ARMhXDnMDeR3qisdtvuLm-wMXQctD2TU15vpb18SfD208Cnp6JyArV~nEeEpVFhM4zWX9mvW3t-zerVJUyIrLvABZlMhySxOIza5aUJ9EcXgkR9uRauEjIlTsWkeRbUc754o2x5jzFOqY-2iXEcqiTbl0QAMQY-egC72L7YeX6AC7FyaHBNYToa-evVrCLxNuUQHU~4bBXb4SAHosrvV3udMiMZMgQc~kkX41sym7J-4udC12MsJlhiWGvjHwIbEcinSpHbp46TDO1z0ytGdtq69~hdip-8~RobIS1OB5WLP0NBX1JX1DoA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Representative gel showing amplification profiles of one ILP marker SiILP4686 and its fragment-length polymorphism among foxtail millet and related species. The amplicons are resolved in 2% agarose gel along with 100 bp DNA size standard." path-from-xml="dst03904.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4913421/dst03904.tiff" path-from-xml="dst03904.tiff" target="_blank">View large</a><a section="4913421" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst039/2/dst03904.png?Expires=2147483647&Signature=cRMZBnwEuxHcg~MgleOZLdtLApyWaX3lO6~qkhG8d5XoaERHl3RXHW3Kw6bGSrHSftCsi9j2cveMejZg6gwoooHZ85H~eR0sfRl9sAGhK54IlpYfWGwTGQjsi43fkRC4SfiSsJ1h4PNrg6lR-8sYQc-qSJdGcVtmeA~lz77CyFnELlHyLINpVF2IKW1fljFvgyFmzG6oTlcQujs9j~Dq-j0nE7kwYiJYda38FV6zuxyTpuXEKNJ3sX2KUezcY2OFBaQ548mtBJG27R2H7MhzgFSXqSs68n5PqnPFnIpluMrssS0JXML2r5Bj5-1nXY4Caeo8pnRVf20SdEgVWy2llQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4913421&ar=347328&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst03904.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Representative gel showing amplification profiles of one ILP marker SiILP4686 and its fragment-length polymorphism among foxtail millet and related species. The amplicons are resolved in 2% agarose gel along with 100 bp DNA size standard.</p></div></div><div content-id="DST039F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst039/2/m_dst03904.png?Expires=2147483647&Signature=HlzdsmAqanbt5zQ28P1BIGDNRPOrD-7ARMhXDnMDeR3qisdtvuLm-wMXQctD2TU15vpb18SfD208Cnp6JyArV~nEeEpVFhM4zWX9mvW3t-zerVJUyIrLvABZlMhySxOIza5aUJ9EcXgkR9uRauEjIlTsWkeRbUc754o2x5jzFOqY-2iXEcqiTbl0QAMQY-egC72L7YeX6AC7FyaHBNYToa-evVrCLxNuUQHU~4bBXb4SAHosrvV3udMiMZMgQc~kkX41sym7J-4udC12MsJlhiWGvjHwIbEcinSpHbp46TDO1z0ytGdtq69~hdip-8~RobIS1OB5WLP0NBX1JX1DoA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Representative gel showing amplification profiles of one ILP marker SiILP4686 and its fragment-length polymorphism among foxtail millet and related species. The amplicons are resolved in 2% agarose gel along with 100 bp DNA size standard." path-from-xml="dst03904.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4913421/dst03904.tiff" path-from-xml="dst03904.tiff" target="_blank">View large</a><a section="4913421" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst039/2/dst03904.png?Expires=2147483647&Signature=cRMZBnwEuxHcg~MgleOZLdtLApyWaX3lO6~qkhG8d5XoaERHl3RXHW3Kw6bGSrHSftCsi9j2cveMejZg6gwoooHZ85H~eR0sfRl9sAGhK54IlpYfWGwTGQjsi43fkRC4SfiSsJ1h4PNrg6lR-8sYQc-qSJdGcVtmeA~lz77CyFnELlHyLINpVF2IKW1fljFvgyFmzG6oTlcQujs9j~Dq-j0nE7kwYiJYda38FV6zuxyTpuXEKNJ3sX2KUezcY2OFBaQ548mtBJG27R2H7MhzgFSXqSs68n5PqnPFnIpluMrssS0JXML2r5Bj5-1nXY4Caeo8pnRVf20SdEgVWy2llQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4913421&ar=347328&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst03904.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Representative gel showing amplification profiles of one ILP marker SiILP4686 and its fragment-length polymorphism among foxtail millet and related species. The amplicons are resolved in 2% agarose gel along with 100 bp DNA size standard.</p></div></div><p>To examine whether the PCR products were really amplified or homologous to the target genes in millet and non-millet species, we randomly picked up a primer pair, SiILP4686 (<em>Zea mays</em> PHD transcription factor), which amplified variant alleles from 214 to 455 bp (Fig. <a reveal-id="DST039F4" data-open="DST039F4" class="link link-reveal link-table xref-fig">4</a>). As expected, the sequences of cloned PCR products of the investigated species revealed indels and several point mutations, such as single-base insertions, deletions or translocations; in addition, polymorphism in intron length was observed (data not shown). Overall, multiple sequence alignment has shown that they were homologous to each other and comprised conserved exon regions at two end positions and non-conserved or variable intron region in the middle. Similar kind of observation was also reported in rice,<sup><a reveal-id="DST039C18" data-open="DST039C18" class="link link-ref link-reveal xref-bibr">18</a></sup><em>Hypericum perforatum</em><sup><a reveal-id="DST039C31" data-open="DST039C31" class="link link-ref link-reveal xref-bibr">31</a></sup> and foxtail millet.<sup><a reveal-id="DST039C20" data-open="DST039C20" class="link link-ref link-reveal xref-bibr">20</a></sup> Further, higher level of transferability of ILPs compared with previously identified markers reflects the conserved nature of exon positions in gene and variability in the non-coding sequences.<sup><a reveal-id="DST039C18" data-open="DST039C18" class="link link-ref link-reveal xref-bibr">18</a></sup> Hence, in our study, the high levels of cross-species amplification indicate that the foxtail millet ILP markers could be successfully useful for comparative mapping in millet and non-millet species. Similarly, it was shown that EST-SSR has higher cross-species transferability than genomic SSR markers.<sup><a reveal-id="DST039C10" data-open="DST039C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DST039C32" data-open="DST039C32" class="link link-ref link-reveal xref-bibr">32</a>,<a reveal-id="DST039C33" data-open="DST039C33" class="link link-ref link-reveal xref-bibr">33</a></sup> In addition, all the 100 ILP markers possess ability to distinguish the investigated millet and non-millet species belonging to different genera. The variations in the number of alleles per ILP marker locus in different species studied are possibly dependent upon ploidy level, nature and number of genotype sets in each species used for analysis.</p>                            <h3 scrollto-destination=4913423 id="4913423" class="section-title" data-legacy-id=s3d>3.4. Genetic diversity</h3>
<p>Besides, the validated physically mapped markers could enable one to discriminate all the 96 cultivated and wild foxtail millet accessions from each other with a level diversity from 0 to 65% (Fig. <a reveal-id="DST039F5" data-open="DST039F5" class="link link-reveal link-table xref-fig">5</a>). A core set of 89 cultivated <em>S. italica</em> accessions and 7 related wild species were used to decipher the polymorphic potential of 20 ILP markers representing the whole genome of foxtail millet. In total, 59 alleles were identified with an average of about 3 alleles per locus, varying from 2 to 5 (Table <a reveal-id="DST039TB3" data-open="DST039TB3" class="link link-reveal link-table xref-fig">3</a>). This was comparable with a recent study in foxtail millet using ILP markers,<sup><a reveal-id="DST039C20" data-open="DST039C20" class="link link-ref link-reveal xref-bibr">20</a></sup> where the average number of alleles per locus reported was 2.6. The polymorphic information content (PIC) values were extended from 0.03 to 0.47 with a mean of 0.20. The observed heterozygosity (<em>H</em><sub>O</sub>) for individual loci ranged from 0.00 to 0.32, with a mean of 0.13 (Table <a reveal-id="DST039TB3" data-open="DST039TB3" class="link link-reveal link-table xref-fig">3</a>). The PIC value was calculated in order to examine the extent of information on diversity that these markers can provide and compare these results with previous published studies. The average PIC value (0.20) reported in this study was lower than that reported for rice (0.45 and 0.44).<sup><a reveal-id="DST039C18" data-open="DST039C18" class="link link-ref link-reveal xref-bibr">18</a>,<a reveal-id="DST039C34" data-open="DST039C34" class="link link-ref link-reveal xref-bibr">34</a></sup> The probable reason for differences in results might be attributed to the difference in number of genotypes and their genetic background and number of markers used. Nei's average gene diversity (<em>Nei</em>) ranged from 0.03 to 0.52, with a mean of 0.22. Among all the loci analysed with fixation index (<em>F</em><sub>IS</sub>), 15 loci were found positive, representing excess of observed homozygotes, whereas 5 loci were negative, demonstrating heterozygotes, with a mean of 0.42 per locus (Table <a reveal-id="DST039TB3" data-open="DST039TB3" class="link link-reveal link-table xref-fig">3</a>). There was no significant correlation observed between PIC and allele number for the 20 markers investigated (data not shown). The phylogenetic tree constructed in this study using ILP markers differentiated 89 cultivated <em>S. italica</em> accessions and 7 related wild species from each other and clustered according to their taxonomic classification (Fig. <a reveal-id="DST039F5" data-open="DST039F5" class="link link-reveal link-table xref-fig">5</a>). The dendrogram constructed grouped 96 <em>Setaria</em> accessions into two distinct clusters, cluster I with 89 accessions comprising cultivated species (foxtail millet, <em>S. italica</em>) and cluster II includes the wild <em>Setaria</em> species (Fig. <a reveal-id="DST039F5" data-open="DST039F5" class="link link-reveal link-table xref-fig">5</a>). Therefore, the ILP markers with high amplification and polymorphic potential distributed over nine chromosomes of foxtail millet genome could be promisingly useful for many large-scale genotyping applications in foxtail millet. </p>                        <a id="4913425" scrollto-destination="4913425"></a>
<div content-id="DST039TB3" class="table-modal"><div class="table-wrap"><div id="DST039TB3" class="table-wrap-title" data-id="DST039TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Summary of genetic diversity estimates of 96 foxtail accessions using 20 ILP markers</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Locus name </th><th><em>N</em><sub>A</sub> </th><th><em>Nei</em> </th><th><em>H</em>o </th><th>PIC </th><th><em>F</em><sub>IS</sub> </th></tr></thead><tbody><tr><td>SiILP1028 </td><td>3 </td><td>0.09 </td><td>0.01 </td><td>0.09 </td><td>0.89 </td></tr><tr><td>SiILP1422 </td><td>3 </td><td>0.16 </td><td>0.10 </td><td>0.15 </td><td>0.34 </td></tr><tr><td>SiILP2672 </td><td>2 </td><td>0.31 </td><td>0.00 </td><td>0.26 </td><td>1.00 </td></tr><tr><td>SiILP709 </td><td>3 </td><td>0.52 </td><td>1.00 </td><td>0.40 </td><td>−0.92 </td></tr><tr><td>SiILP1077 </td><td>3 </td><td>0.10 </td><td>0.01 </td><td>0.10 </td><td>0.90 </td></tr><tr><td>SiILP1496 </td><td>3 </td><td>0.50 </td><td>0.03 </td><td>0.44 </td><td>0.94 </td></tr><tr><td>SiILP4686 </td><td>3 </td><td>0.27 </td><td>0.11 </td><td>0.25 </td><td>0.57 </td></tr><tr><td>SiILP3602 </td><td>3 </td><td>0.16 </td><td>0.17 </td><td>0.15 </td><td>−0.02 </td></tr><tr><td>SiILP431 </td><td>2 </td><td>0.05 </td><td>0.01 </td><td>0.05 </td><td>0.80 </td></tr><tr><td>SiILP996 </td><td>3 </td><td>0.08 </td><td>0.02 </td><td>0.08 </td><td>0.74 </td></tr><tr><td>SiILP1412 </td><td>4 </td><td>0.06 </td><td>0.03 </td><td>0.06 </td><td>0.49 </td></tr><tr><td>SiILP23 </td><td>4 </td><td>0.11 </td><td>0.03 </td><td>0.11 </td><td>0.72 </td></tr><tr><td>SiILP497 </td><td>3 </td><td>0.42 </td><td>0.14 </td><td>0.37 </td><td>0.67 </td></tr><tr><td>SiILP3434 </td><td>3 </td><td>0.34 </td><td>0.32 </td><td>0.30 </td><td>0.06 </td></tr><tr><td>SiILP3864 </td><td>3 </td><td>0.14 </td><td>0.15 </td><td>0.13 </td><td>−0.05 </td></tr><tr><td>SiILP45 </td><td>5 </td><td>0.51 </td><td>0.00 </td><td>0.47 </td><td>1.00 </td></tr><tr><td>SiILP159 </td><td>3 </td><td>0.21 </td><td>0.10 </td><td>0.20 </td><td>0.51 </td></tr><tr><td>SiILP34 </td><td>2 </td><td>0.03 </td><td>0.03 </td><td>0.03 </td><td>−0.01 </td></tr><tr><td>SiILP1365 </td><td>3 </td><td>0.07 </td><td>0.01 </td><td>0.07 </td><td>0.85 </td></tr><tr><td>SiILP9 </td><td>2 </td><td>0.26 </td><td>0.26 </td><td>0.22 </td><td>−0.01 </td></tr><tr><td>Average </td><td>3 </td><td>0.22 </td><td>0.13 </td><td>0.20 </td><td>0.42 </td></tr></tbody></table></div><div content-id="DST039TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Locus name </th><th><em>N</em><sub>A</sub> </th><th><em>Nei</em> </th><th><em>H</em>o </th><th>PIC </th><th><em>F</em><sub>IS</sub> </th></tr></thead><tbody><tr><td>SiILP1028 </td><td>3 </td><td>0.09 </td><td>0.01 </td><td>0.09 </td><td>0.89 </td></tr><tr><td>SiILP1422 </td><td>3 </td><td>0.16 </td><td>0.10 </td><td>0.15 </td><td>0.34 </td></tr><tr><td>SiILP2672 </td><td>2 </td><td>0.31 </td><td>0.00 </td><td>0.26 </td><td>1.00 </td></tr><tr><td>SiILP709 </td><td>3 </td><td>0.52 </td><td>1.00 </td><td>0.40 </td><td>−0.92 </td></tr><tr><td>SiILP1077 </td><td>3 </td><td>0.10 </td><td>0.01 </td><td>0.10 </td><td>0.90 </td></tr><tr><td>SiILP1496 </td><td>3 </td><td>0.50 </td><td>0.03 </td><td>0.44 </td><td>0.94 </td></tr><tr><td>SiILP4686 </td><td>3 </td><td>0.27 </td><td>0.11 </td><td>0.25 </td><td>0.57 </td></tr><tr><td>SiILP3602 </td><td>3 </td><td>0.16 </td><td>0.17 </td><td>0.15 </td><td>−0.02 </td></tr><tr><td>SiILP431 </td><td>2 </td><td>0.05 </td><td>0.01 </td><td>0.05 </td><td>0.80 </td></tr><tr><td>SiILP996 </td><td>3 </td><td>0.08 </td><td>0.02 </td><td>0.08 </td><td>0.74 </td></tr><tr><td>SiILP1412 </td><td>4 </td><td>0.06 </td><td>0.03 </td><td>0.06 </td><td>0.49 </td></tr><tr><td>SiILP23 </td><td>4 </td><td>0.11 </td><td>0.03 </td><td>0.11 </td><td>0.72 </td></tr><tr><td>SiILP497 </td><td>3 </td><td>0.42 </td><td>0.14 </td><td>0.37 </td><td>0.67 </td></tr><tr><td>SiILP3434 </td><td>3 </td><td>0.34 </td><td>0.32 </td><td>0.30 </td><td>0.06 </td></tr><tr><td>SiILP3864 </td><td>3 </td><td>0.14 </td><td>0.15 </td><td>0.13 </td><td>−0.05 </td></tr><tr><td>SiILP45 </td><td>5 </td><td>0.51 </td><td>0.00 </td><td>0.47 </td><td>1.00 </td></tr><tr><td>SiILP159 </td><td>3 </td><td>0.21 </td><td>0.10 </td><td>0.20 </td><td>0.51 </td></tr><tr><td>SiILP34 </td><td>2 </td><td>0.03 </td><td>0.03 </td><td>0.03 </td><td>−0.01 </td></tr><tr><td>SiILP1365 </td><td>3 </td><td>0.07 </td><td>0.01 </td><td>0.07 </td><td>0.85 </td></tr><tr><td>SiILP9 </td><td>2 </td><td>0.26 </td><td>0.26 </td><td>0.22 </td><td>−0.01 </td></tr><tr><td>Average </td><td>3 </td><td>0.22 </td><td>0.13 </td><td>0.20 </td><td>0.42 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><em>N</em><sub>A</sub>, number of alleles; <em>H</em><sub>O</sub>, observed heterozygosity; <em>Nei</em>, Nei's average gene diversity; <em>F</em><sub>IS</sub>, fixation index; PIC, polymorphic information content.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4913425" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DST039TB3" class="table-wrap-title" data-id="DST039TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Summary of genetic diversity estimates of 96 foxtail accessions using 20 ILP markers</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Locus name </th><th><em>N</em><sub>A</sub> </th><th><em>Nei</em> </th><th><em>H</em>o </th><th>PIC </th><th><em>F</em><sub>IS</sub> </th></tr></thead><tbody><tr><td>SiILP1028 </td><td>3 </td><td>0.09 </td><td>0.01 </td><td>0.09 </td><td>0.89 </td></tr><tr><td>SiILP1422 </td><td>3 </td><td>0.16 </td><td>0.10 </td><td>0.15 </td><td>0.34 </td></tr><tr><td>SiILP2672 </td><td>2 </td><td>0.31 </td><td>0.00 </td><td>0.26 </td><td>1.00 </td></tr><tr><td>SiILP709 </td><td>3 </td><td>0.52 </td><td>1.00 </td><td>0.40 </td><td>−0.92 </td></tr><tr><td>SiILP1077 </td><td>3 </td><td>0.10 </td><td>0.01 </td><td>0.10 </td><td>0.90 </td></tr><tr><td>SiILP1496 </td><td>3 </td><td>0.50 </td><td>0.03 </td><td>0.44 </td><td>0.94 </td></tr><tr><td>SiILP4686 </td><td>3 </td><td>0.27 </td><td>0.11 </td><td>0.25 </td><td>0.57 </td></tr><tr><td>SiILP3602 </td><td>3 </td><td>0.16 </td><td>0.17 </td><td>0.15 </td><td>−0.02 </td></tr><tr><td>SiILP431 </td><td>2 </td><td>0.05 </td><td>0.01 </td><td>0.05 </td><td>0.80 </td></tr><tr><td>SiILP996 </td><td>3 </td><td>0.08 </td><td>0.02 </td><td>0.08 </td><td>0.74 </td></tr><tr><td>SiILP1412 </td><td>4 </td><td>0.06 </td><td>0.03 </td><td>0.06 </td><td>0.49 </td></tr><tr><td>SiILP23 </td><td>4 </td><td>0.11 </td><td>0.03 </td><td>0.11 </td><td>0.72 </td></tr><tr><td>SiILP497 </td><td>3 </td><td>0.42 </td><td>0.14 </td><td>0.37 </td><td>0.67 </td></tr><tr><td>SiILP3434 </td><td>3 </td><td>0.34 </td><td>0.32 </td><td>0.30 </td><td>0.06 </td></tr><tr><td>SiILP3864 </td><td>3 </td><td>0.14 </td><td>0.15 </td><td>0.13 </td><td>−0.05 </td></tr><tr><td>SiILP45 </td><td>5 </td><td>0.51 </td><td>0.00 </td><td>0.47 </td><td>1.00 </td></tr><tr><td>SiILP159 </td><td>3 </td><td>0.21 </td><td>0.10 </td><td>0.20 </td><td>0.51 </td></tr><tr><td>SiILP34 </td><td>2 </td><td>0.03 </td><td>0.03 </td><td>0.03 </td><td>−0.01 </td></tr><tr><td>SiILP1365 </td><td>3 </td><td>0.07 </td><td>0.01 </td><td>0.07 </td><td>0.85 </td></tr><tr><td>SiILP9 </td><td>2 </td><td>0.26 </td><td>0.26 </td><td>0.22 </td><td>−0.01 </td></tr><tr><td>Average </td><td>3 </td><td>0.22 </td><td>0.13 </td><td>0.20 </td><td>0.42 </td></tr></tbody></table></div><div content-id="DST039TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Locus name </th><th><em>N</em><sub>A</sub> </th><th><em>Nei</em> </th><th><em>H</em>o </th><th>PIC </th><th><em>F</em><sub>IS</sub> </th></tr></thead><tbody><tr><td>SiILP1028 </td><td>3 </td><td>0.09 </td><td>0.01 </td><td>0.09 </td><td>0.89 </td></tr><tr><td>SiILP1422 </td><td>3 </td><td>0.16 </td><td>0.10 </td><td>0.15 </td><td>0.34 </td></tr><tr><td>SiILP2672 </td><td>2 </td><td>0.31 </td><td>0.00 </td><td>0.26 </td><td>1.00 </td></tr><tr><td>SiILP709 </td><td>3 </td><td>0.52 </td><td>1.00 </td><td>0.40 </td><td>−0.92 </td></tr><tr><td>SiILP1077 </td><td>3 </td><td>0.10 </td><td>0.01 </td><td>0.10 </td><td>0.90 </td></tr><tr><td>SiILP1496 </td><td>3 </td><td>0.50 </td><td>0.03 </td><td>0.44 </td><td>0.94 </td></tr><tr><td>SiILP4686 </td><td>3 </td><td>0.27 </td><td>0.11 </td><td>0.25 </td><td>0.57 </td></tr><tr><td>SiILP3602 </td><td>3 </td><td>0.16 </td><td>0.17 </td><td>0.15 </td><td>−0.02 </td></tr><tr><td>SiILP431 </td><td>2 </td><td>0.05 </td><td>0.01 </td><td>0.05 </td><td>0.80 </td></tr><tr><td>SiILP996 </td><td>3 </td><td>0.08 </td><td>0.02 </td><td>0.08 </td><td>0.74 </td></tr><tr><td>SiILP1412 </td><td>4 </td><td>0.06 </td><td>0.03 </td><td>0.06 </td><td>0.49 </td></tr><tr><td>SiILP23 </td><td>4 </td><td>0.11 </td><td>0.03 </td><td>0.11 </td><td>0.72 </td></tr><tr><td>SiILP497 </td><td>3 </td><td>0.42 </td><td>0.14 </td><td>0.37 </td><td>0.67 </td></tr><tr><td>SiILP3434 </td><td>3 </td><td>0.34 </td><td>0.32 </td><td>0.30 </td><td>0.06 </td></tr><tr><td>SiILP3864 </td><td>3 </td><td>0.14 </td><td>0.15 </td><td>0.13 </td><td>−0.05 </td></tr><tr><td>SiILP45 </td><td>5 </td><td>0.51 </td><td>0.00 </td><td>0.47 </td><td>1.00 </td></tr><tr><td>SiILP159 </td><td>3 </td><td>0.21 </td><td>0.10 </td><td>0.20 </td><td>0.51 </td></tr><tr><td>SiILP34 </td><td>2 </td><td>0.03 </td><td>0.03 </td><td>0.03 </td><td>−0.01 </td></tr><tr><td>SiILP1365 </td><td>3 </td><td>0.07 </td><td>0.01 </td><td>0.07 </td><td>0.85 </td></tr><tr><td>SiILP9 </td><td>2 </td><td>0.26 </td><td>0.26 </td><td>0.22 </td><td>−0.01 </td></tr><tr><td>Average </td><td>3 </td><td>0.22 </td><td>0.13 </td><td>0.20 </td><td>0.42 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><em>N</em><sub>A</sub>, number of alleles; <em>H</em><sub>O</sub>, observed heterozygosity; <em>Nei</em>, Nei's average gene diversity; <em>F</em><sub>IS</sub>, fixation index; PIC, polymorphic information content.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4913425" target="_blank">View Large</a></div></div>                        <a id="4913426" scrollto-destination="4913426"></a>
<div id="DST039F5" data-id="DST039F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst039/2/m_dst03905.png?Expires=2147483647&Signature=loNMw0YUYuLncPrGN5MCBTzeAwb0Yuu8fyrav3yXaVrGU8z-heaLwc~u0RDIQXEsDOZ18x4NtJi4M~f7AAY2jXlkKVnQ76pgz4ABPhiYf6YbU0I6QRRSeNo~9glAVBr7P6-28ET5JFOBS6PhHSAdFW0bdo2iKvgmM0Kv54ZmYBQBcGd0G48W6ZejKUp26ksyfFpJuoKuwXQZh9nu2Tm8z3YmQpMUhJ4KXUDSXI1omrdZCBNqxO0L7ogz7D6Kw6hA3U9hiqs1yATpMho0kgbJX-G6a90Hf885EqqNQcqyr7breMYLXOQJQ-LiDK8c-DjZEmrNTwrw-nMmnPNLQMp-Rw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogenetic relationships among cultivated and wild Setaria species, using 20 ILP markers. Serial numbers of the accessions correspond to Supplementary Table S1. Numbers at branch points indicate support for Setaria species clustered, and values are percent of bootstrap (&gt;50% are indicated) sample that exhibited the cluster." path-from-xml="dst03905.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4913426/dst03905.tiff" path-from-xml="dst03905.tiff" target="_blank">View large</a><a section="4913426" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst039/2/dst03905.png?Expires=2147483647&Signature=mHCuHayH4QkriIas~ZtyGYRUWn1Fo513626fou2ARzMWG9uYSvKfMGhNxtMsaemB-UohjRMuIigcyHPWkQZ-A~brEqp3h7B9FsVowlPTVwTIucQmB0d6Jdcy0bSeM7-Cm9EbgkymfiDQUrBL6cAGVNNsnNxQ5njzPHBWnJusKOGoLy1OdePA35Biu0KxVCtJltCKY2~qSYfp~EapqXLBOiuUzSTBQT61jEnGJqyysmcoa0biLj0Afqe59LPo354ILtTsNljXMoRNGShdI7eGJGp26zQjuOJfqwYnJWiih-5Kef37n3MkAG8oAlfVAVyEmU~nKVw8l69-2IjpIuFwmA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4913426&ar=347328&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst03905.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic relationships among cultivated and wild <em>Setaria</em> species, using 20 ILP markers. Serial numbers of the accessions correspond to <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>. Numbers at branch points indicate support for <em>Setaria</em> species clustered, and values are percent of bootstrap (&gt;50% are indicated) sample that exhibited the cluster.</p></div></div><div content-id="DST039F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst039/2/m_dst03905.png?Expires=2147483647&Signature=loNMw0YUYuLncPrGN5MCBTzeAwb0Yuu8fyrav3yXaVrGU8z-heaLwc~u0RDIQXEsDOZ18x4NtJi4M~f7AAY2jXlkKVnQ76pgz4ABPhiYf6YbU0I6QRRSeNo~9glAVBr7P6-28ET5JFOBS6PhHSAdFW0bdo2iKvgmM0Kv54ZmYBQBcGd0G48W6ZejKUp26ksyfFpJuoKuwXQZh9nu2Tm8z3YmQpMUhJ4KXUDSXI1omrdZCBNqxO0L7ogz7D6Kw6hA3U9hiqs1yATpMho0kgbJX-G6a90Hf885EqqNQcqyr7breMYLXOQJQ-LiDK8c-DjZEmrNTwrw-nMmnPNLQMp-Rw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogenetic relationships among cultivated and wild Setaria species, using 20 ILP markers. Serial numbers of the accessions correspond to Supplementary Table S1. Numbers at branch points indicate support for Setaria species clustered, and values are percent of bootstrap (&gt;50% are indicated) sample that exhibited the cluster." path-from-xml="dst03905.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4913426/dst03905.tiff" path-from-xml="dst03905.tiff" target="_blank">View large</a><a section="4913426" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst039/2/dst03905.png?Expires=2147483647&Signature=mHCuHayH4QkriIas~ZtyGYRUWn1Fo513626fou2ARzMWG9uYSvKfMGhNxtMsaemB-UohjRMuIigcyHPWkQZ-A~brEqp3h7B9FsVowlPTVwTIucQmB0d6Jdcy0bSeM7-Cm9EbgkymfiDQUrBL6cAGVNNsnNxQ5njzPHBWnJusKOGoLy1OdePA35Biu0KxVCtJltCKY2~qSYfp~EapqXLBOiuUzSTBQT61jEnGJqyysmcoa0biLj0Afqe59LPo354ILtTsNljXMoRNGShdI7eGJGp26zQjuOJfqwYnJWiih-5Kef37n3MkAG8oAlfVAVyEmU~nKVw8l69-2IjpIuFwmA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4913426&ar=347328&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst03905.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic relationships among cultivated and wild <em>Setaria</em> species, using 20 ILP markers. Serial numbers of the accessions correspond to <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>. Numbers at branch points indicate support for <em>Setaria</em> species clustered, and values are percent of bootstrap (&gt;50% are indicated) sample that exhibited the cluster.</p></div></div>                            <h3 scrollto-destination=4913427 id="4913427" class="section-title" data-legacy-id=s3e>3.5. <em>In silico</em> comparative genome mapping between foxtail millet and other grass species</h3>
<p>In order to substantiate that the ILP marker-based physical map constructed in this study for foxtail millet genome could be useful in comparative genome mapping, the physically mapped 4049 SiILP markers were compared with their physical location on the chromosomes of other related grass genomes, including sorghum, maize, rice and <em>Brachypodium</em> (Fig. <a reveal-id="DST039F6" data-open="DST039F6" class="link link-reveal link-table xref-fig">6</a>; Table <a reveal-id="DST039TB4" data-open="DST039TB4" class="link link-reveal link-table xref-fig">4</a>). The comparative genome mapping showed considerably significant proportion of sequence-based orthology and syntenic relationship of SiILP markers distributed over nine foxtail millet chromosomes with sorghum (∼50%, 2038), maize (∼46%, 1867), rice (∼21%, 868) and <em>Brachypodium</em> (∼21%, 845) chromosomes (Fig. <a reveal-id="DST039F6" data-open="DST039F6" class="link link-reveal link-table xref-fig">6</a>; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S5–S8</a>). </p>                        <a id="4913429" scrollto-destination="4913429"></a>
<div content-id="DST039TB4" class="table-modal"><div class="table-wrap"><div id="DST039TB4" class="table-wrap-title" data-id="DST039TB4"><div class="label -label">Table 4.</div><div class="caption"><p>A summary of ILP marker-based comparative mapping showing maximum syntenic relationships of foxtail millet chromosomes with sorghum, maize, rice and <em>Brachypodium</em> chromosomes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Foxtail millet chromosomes </th><th>Sorghum chromosomes </th><th>Maize chromosomes </th><th>Rice chromosomes </th><th><em>Brachypodium</em> chromosomes </th></tr></thead><tbody><tr><td>Chr.1 </td><td>Chr.4 (221, 92.9%) </td><td>Chr.5 (117, 53.9%), Chr.4 (85, 39.2%) </td><td>Chr.2 (75, 85.2%) </td><td>Chr.3 (88, 84.6%) </td></tr><tr><td>Chr.2 </td><td>Chr.2 (241, 87%) </td><td>Chr.7 (143, 58.6%), Chr.2 (65, 26.6%), </td><td>Chr.7 (57, 48.3%) </td><td>Chr.1 (49, 47.6%) </td></tr><tr><td>Chr.3 </td><td>Chr.9 (158, 61.7%) </td><td>Chr.6 (67, 29.8%), Chr.8 (64, 28.4%) </td><td>Chr.5 (48, 60.8%) </td><td>Chr.2 (59, 58.4%) </td></tr><tr><td>Chr.4 </td><td>Chr.10 (134, 85.4%) </td><td>Chr.9 (64, 42.4%), Chr.6 (42, 27.8%) </td><td>Chr.6 (74, 83.1%) </td><td>Chr.1 (64, 83.1%) </td></tr><tr><td>Chr.5 </td><td>Chr.3 (274, 88.7%) </td><td>Chr.3 (146, 50.7%), Chr.8 (101, 35.1%) </td><td>Chr.1 (117, 82.4%) </td><td>Chr.2 (108, 90%) </td></tr><tr><td>Chr.6 </td><td>Chr.7 (116, 85.9%) </td><td>Chr.1 (37, 32.2%), Chr.4 (36, 31.3%) </td><td>Chr.8 (42, 84%) </td><td>Chr.3 (41, 82%) </td></tr><tr><td>Chr.7 </td><td>Chr.6 (114, 66.7%) </td><td>Chr.2 (64, 39.5%), Chr.10 (50, 30.9%) </td><td>Chr.7 (51, 64.6%) </td><td>Chr.5 (53, 66.3%) </td></tr><tr><td>Chr.8 </td><td>Chr.5 (59, 75.6%) </td><td>Chr.4 (34, 44.7%), Chr.2 (25, 32.9%) </td><td>Chr.11 (29, 70.7%) </td><td>Chr.4 (29, 76.3%) </td></tr><tr><td>Chr.9 </td><td>Chr.1 (380, 91.1%) </td><td>Chr.1 (221, 56.8%), Chr.5 (73, 18.8%) </td><td>Chr.3 (123, 67.6%) </td><td>Chr.1 (130, 75.6%) </td></tr></tbody></table></div><div content-id="DST039TB4" class="table-modal"><table><thead name="thead" align=""><tr><th>Foxtail millet chromosomes </th><th>Sorghum chromosomes </th><th>Maize chromosomes </th><th>Rice chromosomes </th><th><em>Brachypodium</em> chromosomes </th></tr></thead><tbody><tr><td>Chr.1 </td><td>Chr.4 (221, 92.9%) </td><td>Chr.5 (117, 53.9%), Chr.4 (85, 39.2%) </td><td>Chr.2 (75, 85.2%) </td><td>Chr.3 (88, 84.6%) </td></tr><tr><td>Chr.2 </td><td>Chr.2 (241, 87%) </td><td>Chr.7 (143, 58.6%), Chr.2 (65, 26.6%), </td><td>Chr.7 (57, 48.3%) </td><td>Chr.1 (49, 47.6%) </td></tr><tr><td>Chr.3 </td><td>Chr.9 (158, 61.7%) </td><td>Chr.6 (67, 29.8%), Chr.8 (64, 28.4%) </td><td>Chr.5 (48, 60.8%) </td><td>Chr.2 (59, 58.4%) </td></tr><tr><td>Chr.4 </td><td>Chr.10 (134, 85.4%) </td><td>Chr.9 (64, 42.4%), Chr.6 (42, 27.8%) </td><td>Chr.6 (74, 83.1%) </td><td>Chr.1 (64, 83.1%) </td></tr><tr><td>Chr.5 </td><td>Chr.3 (274, 88.7%) </td><td>Chr.3 (146, 50.7%), Chr.8 (101, 35.1%) </td><td>Chr.1 (117, 82.4%) </td><td>Chr.2 (108, 90%) </td></tr><tr><td>Chr.6 </td><td>Chr.7 (116, 85.9%) </td><td>Chr.1 (37, 32.2%), Chr.4 (36, 31.3%) </td><td>Chr.8 (42, 84%) </td><td>Chr.3 (41, 82%) </td></tr><tr><td>Chr.7 </td><td>Chr.6 (114, 66.7%) </td><td>Chr.2 (64, 39.5%), Chr.10 (50, 30.9%) </td><td>Chr.7 (51, 64.6%) </td><td>Chr.5 (53, 66.3%) </td></tr><tr><td>Chr.8 </td><td>Chr.5 (59, 75.6%) </td><td>Chr.4 (34, 44.7%), Chr.2 (25, 32.9%) </td><td>Chr.11 (29, 70.7%) </td><td>Chr.4 (29, 76.3%) </td></tr><tr><td>Chr.9 </td><td>Chr.1 (380, 91.1%) </td><td>Chr.1 (221, 56.8%), Chr.5 (73, 18.8%) </td><td>Chr.3 (123, 67.6%) </td><td>Chr.1 (130, 75.6%) </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4913429" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DST039TB4" class="table-wrap-title" data-id="DST039TB4"><div class="label -label">Table 4.</div><div class="caption"><p>A summary of ILP marker-based comparative mapping showing maximum syntenic relationships of foxtail millet chromosomes with sorghum, maize, rice and <em>Brachypodium</em> chromosomes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Foxtail millet chromosomes </th><th>Sorghum chromosomes </th><th>Maize chromosomes </th><th>Rice chromosomes </th><th><em>Brachypodium</em> chromosomes </th></tr></thead><tbody><tr><td>Chr.1 </td><td>Chr.4 (221, 92.9%) </td><td>Chr.5 (117, 53.9%), Chr.4 (85, 39.2%) </td><td>Chr.2 (75, 85.2%) </td><td>Chr.3 (88, 84.6%) </td></tr><tr><td>Chr.2 </td><td>Chr.2 (241, 87%) </td><td>Chr.7 (143, 58.6%), Chr.2 (65, 26.6%), </td><td>Chr.7 (57, 48.3%) </td><td>Chr.1 (49, 47.6%) </td></tr><tr><td>Chr.3 </td><td>Chr.9 (158, 61.7%) </td><td>Chr.6 (67, 29.8%), Chr.8 (64, 28.4%) </td><td>Chr.5 (48, 60.8%) </td><td>Chr.2 (59, 58.4%) </td></tr><tr><td>Chr.4 </td><td>Chr.10 (134, 85.4%) </td><td>Chr.9 (64, 42.4%), Chr.6 (42, 27.8%) </td><td>Chr.6 (74, 83.1%) </td><td>Chr.1 (64, 83.1%) </td></tr><tr><td>Chr.5 </td><td>Chr.3 (274, 88.7%) </td><td>Chr.3 (146, 50.7%), Chr.8 (101, 35.1%) </td><td>Chr.1 (117, 82.4%) </td><td>Chr.2 (108, 90%) </td></tr><tr><td>Chr.6 </td><td>Chr.7 (116, 85.9%) </td><td>Chr.1 (37, 32.2%), Chr.4 (36, 31.3%) </td><td>Chr.8 (42, 84%) </td><td>Chr.3 (41, 82%) </td></tr><tr><td>Chr.7 </td><td>Chr.6 (114, 66.7%) </td><td>Chr.2 (64, 39.5%), Chr.10 (50, 30.9%) </td><td>Chr.7 (51, 64.6%) </td><td>Chr.5 (53, 66.3%) </td></tr><tr><td>Chr.8 </td><td>Chr.5 (59, 75.6%) </td><td>Chr.4 (34, 44.7%), Chr.2 (25, 32.9%) </td><td>Chr.11 (29, 70.7%) </td><td>Chr.4 (29, 76.3%) </td></tr><tr><td>Chr.9 </td><td>Chr.1 (380, 91.1%) </td><td>Chr.1 (221, 56.8%), Chr.5 (73, 18.8%) </td><td>Chr.3 (123, 67.6%) </td><td>Chr.1 (130, 75.6%) </td></tr></tbody></table></div><div content-id="DST039TB4" class="table-modal"><table><thead name="thead" align=""><tr><th>Foxtail millet chromosomes </th><th>Sorghum chromosomes </th><th>Maize chromosomes </th><th>Rice chromosomes </th><th><em>Brachypodium</em> chromosomes </th></tr></thead><tbody><tr><td>Chr.1 </td><td>Chr.4 (221, 92.9%) </td><td>Chr.5 (117, 53.9%), Chr.4 (85, 39.2%) </td><td>Chr.2 (75, 85.2%) </td><td>Chr.3 (88, 84.6%) </td></tr><tr><td>Chr.2 </td><td>Chr.2 (241, 87%) </td><td>Chr.7 (143, 58.6%), Chr.2 (65, 26.6%), </td><td>Chr.7 (57, 48.3%) </td><td>Chr.1 (49, 47.6%) </td></tr><tr><td>Chr.3 </td><td>Chr.9 (158, 61.7%) </td><td>Chr.6 (67, 29.8%), Chr.8 (64, 28.4%) </td><td>Chr.5 (48, 60.8%) </td><td>Chr.2 (59, 58.4%) </td></tr><tr><td>Chr.4 </td><td>Chr.10 (134, 85.4%) </td><td>Chr.9 (64, 42.4%), Chr.6 (42, 27.8%) </td><td>Chr.6 (74, 83.1%) </td><td>Chr.1 (64, 83.1%) </td></tr><tr><td>Chr.5 </td><td>Chr.3 (274, 88.7%) </td><td>Chr.3 (146, 50.7%), Chr.8 (101, 35.1%) </td><td>Chr.1 (117, 82.4%) </td><td>Chr.2 (108, 90%) </td></tr><tr><td>Chr.6 </td><td>Chr.7 (116, 85.9%) </td><td>Chr.1 (37, 32.2%), Chr.4 (36, 31.3%) </td><td>Chr.8 (42, 84%) </td><td>Chr.3 (41, 82%) </td></tr><tr><td>Chr.7 </td><td>Chr.6 (114, 66.7%) </td><td>Chr.2 (64, 39.5%), Chr.10 (50, 30.9%) </td><td>Chr.7 (51, 64.6%) </td><td>Chr.5 (53, 66.3%) </td></tr><tr><td>Chr.8 </td><td>Chr.5 (59, 75.6%) </td><td>Chr.4 (34, 44.7%), Chr.2 (25, 32.9%) </td><td>Chr.11 (29, 70.7%) </td><td>Chr.4 (29, 76.3%) </td></tr><tr><td>Chr.9 </td><td>Chr.1 (380, 91.1%) </td><td>Chr.1 (221, 56.8%), Chr.5 (73, 18.8%) </td><td>Chr.3 (123, 67.6%) </td><td>Chr.1 (130, 75.6%) </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4913429" target="_blank">View Large</a></div></div>                        <a id="4913430" scrollto-destination="4913430"></a>
<div id="DST039F6" data-id="DST039F6" class="fig fig-section"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst039/2/m_dst03906.png?Expires=2147483647&Signature=T8cCQ7muqujmCTRz5Xn1Dl6c0CAgtnX7DbhM9uR~5ASQV4aB9uehPDDbRP9Qy5ayF9irOMTZrzJ7bHkgO6f9hEAG1Ii-~sWYt5RQHDZgDq3bucBuguiEOrxg99khWVczzQyxT2YMPffF7s-Mi1pLOK0EfNHavTZnLCOtVz0l-SjG55ZG0WrC-mV~SK1zUfY-SL-scRkAwM1sc2s~BaR8ZN4F6ptwaCopvUbIHJw40an8J3IDeD80rxpcHesbsWpcTgjWDWwQkhVupc-24k15AV9EeDd2ypAo8ScUY~CdUJ6dGduXpgQF4vig-cP59Ye150oY2~41K491qUW5Ae1k0A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genome relationships of foxtail millet with other grass species. Comparative mapping between foxtail millet chromosomes with (a) sorghum; (b) maize; (c) rice and (d) Brachypodium chromosomes, using 4049 physically mapped foxtail millet ILP markers. Maximum syntenic relationships of foxtail millet chromosomes with sorghum chromosomes based on ILP markers was evident." path-from-xml="dst03906.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4913430/dst03906.tiff" path-from-xml="dst03906.tiff" target="_blank">View large</a><a section="4913430" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst039/2/dst03906.png?Expires=2147483647&Signature=OfJ~N24XLZOS~Qno-UcvUj3KJwu7D~~TRNole01xIy8EgZXVtZajIYVC6a8ANtIzy1P4z0z-9ITjXuzToAGbS3IntfqE2lO3cfKrLXQRWKJIn5jIvvcCSeGnhhUNKXPJKlUjD2G0PKZFu1UDlpAKiSgO9L604nKVxV5saa72kFdo9eSRAEsLAGn0JvZX0HtzwWRGa0Fz0GUvNR5KKK0Hy-P~FRihd3LUjbJ2un1~s1gJJkrngny24L4ztuu3CbxQVWHgd8ra0qViqdPIXocM6aN0NEIdpKvFt5u-h7FeEdQSk7EsQzIuZpYLOfcrlM24lnJvdLsGvP4Czqz7c-nSlA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4913430&ar=347328&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst03906.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genome relationships of foxtail millet with other grass species. Comparative mapping between foxtail millet chromosomes with (a) sorghum; (b) maize; (c) rice and (d) <em>Brachypodium</em> chromosomes, using 4049 physically mapped foxtail millet ILP markers. Maximum syntenic relationships of foxtail millet chromosomes with sorghum chromosomes based on ILP markers was evident.</p></div></div><div content-id="DST039F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst039/2/m_dst03906.png?Expires=2147483647&Signature=T8cCQ7muqujmCTRz5Xn1Dl6c0CAgtnX7DbhM9uR~5ASQV4aB9uehPDDbRP9Qy5ayF9irOMTZrzJ7bHkgO6f9hEAG1Ii-~sWYt5RQHDZgDq3bucBuguiEOrxg99khWVczzQyxT2YMPffF7s-Mi1pLOK0EfNHavTZnLCOtVz0l-SjG55ZG0WrC-mV~SK1zUfY-SL-scRkAwM1sc2s~BaR8ZN4F6ptwaCopvUbIHJw40an8J3IDeD80rxpcHesbsWpcTgjWDWwQkhVupc-24k15AV9EeDd2ypAo8ScUY~CdUJ6dGduXpgQF4vig-cP59Ye150oY2~41K491qUW5Ae1k0A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genome relationships of foxtail millet with other grass species. Comparative mapping between foxtail millet chromosomes with (a) sorghum; (b) maize; (c) rice and (d) Brachypodium chromosomes, using 4049 physically mapped foxtail millet ILP markers. Maximum syntenic relationships of foxtail millet chromosomes with sorghum chromosomes based on ILP markers was evident." path-from-xml="dst03906.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4913430/dst03906.tiff" path-from-xml="dst03906.tiff" target="_blank">View large</a><a section="4913430" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst039/2/dst03906.png?Expires=2147483647&Signature=OfJ~N24XLZOS~Qno-UcvUj3KJwu7D~~TRNole01xIy8EgZXVtZajIYVC6a8ANtIzy1P4z0z-9ITjXuzToAGbS3IntfqE2lO3cfKrLXQRWKJIn5jIvvcCSeGnhhUNKXPJKlUjD2G0PKZFu1UDlpAKiSgO9L604nKVxV5saa72kFdo9eSRAEsLAGn0JvZX0HtzwWRGa0Fz0GUvNR5KKK0Hy-P~FRihd3LUjbJ2un1~s1gJJkrngny24L4ztuu3CbxQVWHgd8ra0qViqdPIXocM6aN0NEIdpKvFt5u-h7FeEdQSk7EsQzIuZpYLOfcrlM24lnJvdLsGvP4Czqz7c-nSlA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4913430&ar=347328&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst03906.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genome relationships of foxtail millet with other grass species. Comparative mapping between foxtail millet chromosomes with (a) sorghum; (b) maize; (c) rice and (d) <em>Brachypodium</em> chromosomes, using 4049 physically mapped foxtail millet ILP markers. Maximum syntenic relationships of foxtail millet chromosomes with sorghum chromosomes based on ILP markers was evident.</p></div></div>                            <h4 scrollto-destination=4913431 id="4913431" class="section-title" data-legacy-id=s3e1>3.5.1. Foxtail millet–sorghum synteny</h4>
<p>The comparative mapping showed a syntenic relationship of 2038 ILP marker loci distributed over 9 chromosomes of foxtail millet with 2038 genomic regions on 10 chromosomes of sorghum (Fig. <a reveal-id="DST039F6" data-open="DST039F6" class="link link-reveal link-table xref-fig">6</a>a; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a>). About 50% syntenic relationship of ILP marker loci between foxtail millet and sorghum chromosomes was observed on an average, with maximum synteny between foxtail millet chromosome 9 and sorghum chromosome 1 (∼91%), followed between foxtail millet chromosome 5 and sorghum chromosome 3 (∼89%) (Fig. <a reveal-id="DST039F6" data-open="DST039F6" class="link link-reveal link-table xref-fig">6</a>a; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a>).</p>                            <h4 scrollto-destination=4913433 id="4913433" class="section-title" data-legacy-id=s3e2>3.5.2. Foxtail millet–maize synteny</h4>
<p>About 1867 ILP marker loci distributed over nine chromosomes of foxtail millet showed significant matches with 1867 genomic regions of 10 maize chromosomes (Fig. <a reveal-id="DST039F6" data-open="DST039F6" class="link link-reveal link-table xref-fig">6</a>b; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>). Interestingly, each foxtail millet chromosome showed sytenic relationship with two maize chromosomes, thus highlighting the recent whole-genome duplication in maize. All the nine foxtail millet chromosomes showed considerable and higher average frequency (∼46%) of ILP marker-based syntenic relationship with specific maize chromosomes. The physically mapped ILP markers on the foxtail millet chromosome 9 showed maximum synteny (∼57%) with maize chromosome 1 and between foxtail millet chromosome 5 and maize chromosome 3 (∼51%) (Fig. <a reveal-id="DST039F6" data-open="DST039F6" class="link link-reveal link-table xref-fig">6</a>b; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>).</p>                            <h4 scrollto-destination=4913438 id="4913438" class="section-title" data-legacy-id=s3e3>3.5.3. Foxtail millet–rice synteny</h4>
<p>The syntenic relationship of 868 ILP marker loci distributed over nine chromosomes of foxtail millet with 868 genomic regions on 12 chromosomes of rice was evidenced (Fig. <a reveal-id="DST039F6" data-open="DST039F6" class="link link-reveal link-table xref-fig">6</a>c; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S7</a>). Maximum synteny of ILP marker loci was observed between foxtail millet chromosome 9 and rice chromosome 3 (∼68%) and between foxtail millet chromosome 5 and rice chromosome 1 (∼83%) (Fig. <a reveal-id="DST039F6" data-open="DST039F6" class="link link-reveal link-table xref-fig">6</a>c; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S7</a>).</p>                            <h4 scrollto-destination=4913440 id="4913440" class="section-title" data-legacy-id=s3e4>3.5.4. Foxtail millet–<em>Brachypodium</em> synteny</h4>
<p>The ILP marker-based comparative mapping showed a similar pattern of synteny between foxtail millet and rice. <em>Brachypodium</em> chromosomes demonstrated a syntenic relationship with an average frequency of ∼21% (845 marker loci) (Fig. <a reveal-id="DST039F6" data-open="DST039F6" class="link link-reveal link-table xref-fig">6</a>d; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S8</a>). The physically mapped ILP markers on the foxtail millet chromosome 9 showed maximum synteny (∼76%) with <em>Brachypodium</em> chromosome 1 (Fig. <a reveal-id="DST039F6" data-open="DST039F6" class="link link-reveal link-table xref-fig">6</a>d; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S8</a>).</p><p>Though there are many reports showing the mapping of microsatellite markers either genetically or physically on orthologous or syntenic chromosomes of different related plant genomes,<sup><a reveal-id="DST039C7  DST039C9" data-open="DST039C7  DST039C9" class="link link-ref link-reveal xref-bibr">7–9</a>,<a reveal-id="DST039C35  DST039C38" data-open="DST039C35  DST039C38" class="link link-ref link-reveal xref-bibr">35–38</a></sup> this is the first report of comparative mapping using ILP markers. The syntenic relationships showed a higher degree of synteny between foxtail and sorghum genome followed by maize, rice and <em>Brachypodium</em>, which is possibly due to their taxonomic relationship, where foxtail millet, sorghum and maize belong to same subfamily Panicoideae, while rice belongs to Ehrhartoideae and <em>Brachypodium</em> belongs to Pooideae. The comparative mapping thus demonstrates the decrease of colinearity with increasing phylogenetic distance among plant species. These results are in accordance with the interpretations reported using genomic SSR markers<sup><a reveal-id="DST039C8" data-open="DST039C8" class="link link-ref link-reveal xref-bibr">8</a></sup> and genic EST-derived SSR markers<sup><a reveal-id="DST039C9" data-open="DST039C9" class="link link-ref link-reveal xref-bibr">9</a></sup> in foxtail millet. This shows the applicability of ILP marker-based comparative genome mapping between foxtail millet and other grass species such as sorghum, maize, rice and <em>Brachypodium</em> in translating the sequence information/candidate genes from this diploid crop to other polyploid biofuel grasses. Further, the ILP marker-based comparative mapping between foxtail millet and other grass species could enable transfer of gene-based marker information among these target species and thus would expedite map-based isolation of genes of agronomic importance in foxtail millet.</p>                            <h2 scrollto-destination=4913443 id="4913443" class="section-title" data-legacy-id=s4>4. Conclusions</h2>
<p>To the best of our knowledge, the ILP markers developed in this study are the large-scale novel set of markers in foxtail millet in addition to 98 earlier reported by Gupta <em>et al</em>.<sup><a reveal-id="DST039C20" data-open="DST039C20" class="link link-ref link-reveal xref-bibr">20</a></sup> The present study identified 5123 ILP markers from 24 828 non-redundant ESTs, of which 4049 markers were physically mapped onto 9 chromosomes of foxtail millet genome. The validation, cross-genera transferability and genetic diversity studies demonstrated expediency of these ILP markers in germplasm characterization, genome relationships in millet and non-millet species and comparative mapping. The newly developed large-scale SiILP markers will be made available to the research community through the FmMDb (<a class="link link-uri" href="http://www.nipgr.res.in/foxtail.html">http://www.nipgr.res.in/foxtail.html</a>)<sup><a reveal-id="DST039C10" data-open="DST039C10" class="link link-ref link-reveal xref-bibr">10</a></sup> and this will promisingly expedite the molecular breeding in foxtail millet and other millets and forage grass species.</p>                            <h2 scrollto-destination=4913445 id="4913445" class="section-title" data-legacy-id=s5>Supplementary Data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Data are available at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=4913447 id="4913447" class="section-title" data-legacy-id=s6>Funding</h2>
<p>The authors' work in this area was supported by the core grant of NIPGR. M.M. and G.P. acknowledge the award of Junior Research Fellowship from University Grants Commission, New Delhi.</p>                            <h2 scrollto-destination=4913449 id="4913449" class="backacknowledgements-title" >Acknowledgements</h2>
<p>Grateful thanks are due to the Director, National Institute of Plant Genome Research (NIPGR), New Delhi, India, for providing facilities. The authors also thank Mr V.V. Satyanarayana, NIPGR, for his timely assistance. We are also thankful to the National Bureau of Plant Genetic Resources, New Delhi/Hyderabad/Akola, India, and Tamil Nadu Agricultural University, India, for providing the seeds.</p>                            <h2 scrollto-destination=4913451 id="4913451" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DST039C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST039C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Doust</div> <div class="given-names">A.N.</div></div>,  <div class="name"><div class="surname">Kellogg</div> <div class="given-names">E.A.</div></div>,  <div class="name"><div class="surname">Devos</div> <div class="given-names">K.M.</div></div>,  <div class="name"><div class="surname">Bennetzen</div> <div class="given-names">J.L.</div></div>. </div><div class="article-title">Foxtail millet: a sequence-driven grass model system</div>, <div class="source">Plant Physiol.</div> , <div class="year">2009</div>, vol. <div class="volume">149</div> (pg. <div class="fpage">137</div>-<div class="lpage">41</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1104/pp.108.129627" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1104/pp.108.129627</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Foxtail%20millet%3A%20a%20sequence-driven%20grass%20model%20system&amp;author=A.N.%20Doust&amp;author=E.A.%20Kellogg&amp;author=K.M.%20Devos&amp;author=J.L.%20Bennetzen&amp;publication_year=2009&amp;journal=Plant%20Physiol.&amp;volume=149&amp;pages=137-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.108.129627" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.108.129627" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.108.129627"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19126705" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST039C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST039C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lata</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Gupta</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Prasad</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Foxtail millet: a model crop for genetic and genomic studies in bioenergy grasses</div>, <div class="source">Crit. Rev. Biotechnol</div> , <div class="year">2013</div>, vol. <div class="volume">33</div> (pg. <div class="fpage">328</div>-<div class="lpage">43</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Foxtail%20millet%3A%20a%20model%20crop%20for%20genetic%20and%20genomic%20studies%20in%20bioenergy%20grasses&amp;author=C.%20Lata&amp;author=S.%20Gupta&amp;author=M.%20Prasad&amp;publication_year=2013&amp;journal=Crit.%20Rev.%20Biotechnol&amp;volume=33&amp;pages=328-43" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3109/07388551.2012.716809" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3109%2F07388551.2012.716809" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3109%2F07388551.2012.716809"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22985089" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST039C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST039C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Brutnell</div> <div class="given-names">T.P.</div></div>. </div><div class="article-title"><em>Setaria viridis</em> and <em>Setaria italica</em>, model genetic systems for Panicoid grasses</div>, <div class="source">J. Exp. Bot.</div> , <div class="year">2011</div>, vol. <div class="volume">62</div> (pg. <div class="fpage">3031</div>-<div class="lpage">7</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/jxb/err096" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/jxb/err096</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Setaria%20viridis%20and%20Setaria%20italica%2C%20model%20genetic%20systems%20for%20Panicoid%20grasses&amp;author=P.%20Li&amp;author=T.P.%20Brutnell&amp;publication_year=2011&amp;journal=J.%20Exp.%20Bot.&amp;volume=62&amp;pages=3031-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jxb/err096" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjxb%2Ferr096" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjxb%2Ferr096"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21459768" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST039C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST039C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lata</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Prasad</div> <div class="given-names">M.</div></div>. </div><div class="article-title"><em>Setaria</em> genome sequencing: an overview</div>, <div class="source">J. Plant Biochem. Biotechnol.</div> , <div class="year">2013</div>, vol. <div class="volume">22</div> (pg. <div class="fpage">257</div>-<div class="lpage">60</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s13562-013-0216-8" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s13562-013-0216-8</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Setaria%20genome%20sequencing%3A%20an%20overview&amp;author=C.%20Lata&amp;author=M.%20Prasad&amp;publication_year=2013&amp;journal=J.%20Plant%20Biochem.%20Biotechnol.&amp;volume=22&amp;pages=257-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s13562-013-0216-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs13562-013-0216-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs13562-013-0216-8"> </span></div> </div></div></div></div></div><div content-id="DST039C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST039C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Muthamilarasan</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Theriappan</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Prasad</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Recent advances in crop genomics for ensuring food security</div>, <div class="source">Curr. Sci.</div> , <div class="year">2013</div>, vol. <div class="volume">105</div> (pg. <div class="fpage">155</div>-<div class="lpage">8</div>)</div></div></div></div><div content-id="DST039C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST039C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bennetzen</div> <div class="given-names">J.L.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">J.S.H.</div></div>,  <div class="name"><div class="surname">Percifield</div> <div class="given-names">R.</div></div>, et al. </div><div class="article-title">Reference genome sequence of the model plant <em>Setaria</em></div>, <div class="source">Nat. Biotech.</div> , <div class="year">2012</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">555</div>-<div class="lpage">61</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nbt.2196" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nbt.2196</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Reference%20genome%20sequence%20of%20the%20model%20plant%20Setaria&amp;author=J.L.%20Bennetzen&amp;author=J.S.H.%20Wang&amp;author=R.%20Percifield&amp;publication_year=2012&amp;journal=Nat.%20Biotech.&amp;volume=30&amp;pages=555-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.2196" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.2196" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.2196"> </span></div> </div></div></div></div></div><div content-id="DST039C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST039C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Liu</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Quan</div> <div class="given-names">Z.</div></div>, et al. </div><div class="article-title">Genome sequence of foxtail millet (<em>Setaria italica</em>) provides insights into grass evolution and biofuel potential</div>, <div class="source">Nat. Biotech.</div> , <div class="year">2012</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">549</div>-<div class="lpage">54</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nbt.2195" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nbt.2195</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20sequence%20of%20foxtail%20millet%20(Setaria%20italica)%20provides%20insights%20into%20grass%20evolution%20and%20biofuel%20potential&amp;author=G.%20Zhang&amp;author=X.%20Liu&amp;author=Z.%20Quan&amp;publication_year=2012&amp;journal=Nat.%20Biotech.&amp;volume=30&amp;pages=549-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.2195" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.2195" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.2195"> </span></div> </div></div></div></div></div><div content-id="DST039C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST039C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pandey</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Misra</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Kumari</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">Genome-wide development and use of microsatellite markers for large-scale genotyping applications in foxtail millet [<em>Setaria italica</em> (L.)]</div>, <div class="source">DNA Res.</div> , <div class="year">2013</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">197</div>-<div class="lpage">207</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/dnares/dst002" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/dnares/dst002</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20development%20and%20use%20of%20microsatellite%20markers%20for%20large-scale%20genotyping%20applications%20in%20foxtail%20millet%20%5BSetaria%20italica%20(L.)%5D&amp;author=G.%20Pandey&amp;author=G.%20Misra&amp;author=K.%20Kumari&amp;publication_year=2013&amp;journal=DNA%20Res.&amp;volume=20&amp;pages=197-207" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dst002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdst002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdst002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23382459" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST039C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST039C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kumari</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Muthamilarasan</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Misra</div> <div class="given-names">G.</div></div>, et al. </div><div class="article-title">Development of eSSR-markers in <em>Setaria italica</em> and their applicability in studying genetic diversity, cross-transferability and comparative mapping in millet and non-millet species</div>, <div class="source">PLoS ONE</div> , <div class="year">2013</div>, vol. <div class="volume">8</div> pg. <div class="fpage">e67742</div> <div class="pub-id-doi"><a href="https://doi.org/10.1371/journal.pone.0067742" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1371/journal.pone.0067742</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20eSSR-markers%20in%20Setaria%20italica%20and%20their%20applicability%20in%20studying%20genetic%20diversity%2C%20cross-transferability%20and%20comparative%20mapping%20in%20millet%20and%20non-millet%20species&amp;author=K.%20Kumari&amp;author=M.%20Muthamilarasan&amp;author=G.%20Misra&amp;publication_year=2013&amp;journal=PLoS%20ONE&amp;volume=8&amp;pages=e67742" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0067742" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0067742" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0067742"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23805325" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST039C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST039C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Suresh</div> <div class="given-names">V.B.</div></div>,  <div class="name"><div class="surname">Muthamilarasan</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Prasad</div> <div class="given-names">M.</div></div>. </div><div class="article-title">FmMDb: a versatile database of foxtail millet markers for millets and bioenergy grasses research</div>, <div class="source">PLoS ONE</div> , <div class="year">2013</div>, vol. <div class="volume">8</div> pg. <div class="fpage">e71418</div> <div class="pub-id-doi"><a href="https://doi.org/10.1371/journal.pone.0071418" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1371/journal.pone.0071418</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=FmMDb%3A%20a%20versatile%20database%20of%20foxtail%20millet%20markers%20for%20millets%20and%20bioenergy%20grasses%20research&amp;author=V.B.%20Suresh&amp;author=M.%20Muthamilarasan&amp;author=M.%20Prasad&amp;publication_year=2013&amp;journal=PLoS%20ONE&amp;volume=8&amp;pages=e71418" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0075152" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0075152" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0075152"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23951158" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST039C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST039C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Botstein</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">White</div> <div class="given-names">R.L.</div></div>,  <div class="name"><div class="surname">Skolnick</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Davis</div> <div class="given-names">R.W.</div></div>. </div><div class="article-title">Construction of a genetic linkage map in man using restriction fragment length polymorphisms</div>, <div class="source">Am. J. Hum. Genet.</div> , <div class="year">1980</div>, vol. <div class="volume">32</div> (pg. <div class="fpage">314</div>-<div class="lpage">31</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20a%20genetic%20linkage%20map%20in%20man%20using%20restriction%20fragment%20length%20polymorphisms&amp;author=D.%20Botstein&amp;author=R.L.%20White&amp;author=M.%20Skolnick&amp;author=R.W.%20Davis&amp;publication_year=1980&amp;journal=Am.%20J.%20Hum.%20Genet.&amp;volume=32&amp;pages=314-31" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6247908" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST039C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST039C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Williams</div> <div class="given-names">J.G.</div></div>,  <div class="name"><div class="surname">Kubelik</div> <div class="given-names">A.R.</div></div>,  <div class="name"><div class="surname">Livak</div> <div class="given-names">K.J.</div></div>,  <div class="name"><div class="surname">Rafalski</div> <div class="given-names">J.A.</div></div>,  <div class="name"><div class="surname">Tingey</div> <div class="given-names">S.V.</div></div>. </div><div class="article-title">DNA polymorphisms amplified by arbitrary primers are useful as genetic markers</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1990</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">6531</div>-<div class="lpage">5</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/18.22.6531" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/18.22.6531</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20polymorphisms%20amplified%20by%20arbitrary%20primers%20are%20useful%20as%20genetic%20markers&amp;author=J.G.%20Williams&amp;author=A.R.%20Kubelik&amp;author=K.J.%20Livak&amp;author=J.A.%20Rafalski&amp;author=S.V.%20Tingey&amp;publication_year=1990&amp;journal=Nucleic%20Acids%20Res.&amp;volume=18&amp;pages=6531-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/18.22.6531" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F18.22.6531" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F18.22.6531"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1979162" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST039C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST039C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Vos</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Hogers</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Bleeker</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">AFLP: a new technique for DNA fingerprinting</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1995</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">4407</div>-<div class="lpage">14</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/23.21.4407" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/23.21.4407</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=AFLP%3A%20a%20new%20technique%20for%20DNA%20fingerprinting&amp;author=P.%20Vos&amp;author=R.%20Hogers&amp;author=M.%20Bleeker&amp;publication_year=1995&amp;journal=Nucleic%20Acids%20Res.&amp;volume=23&amp;pages=4407-14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/23.21.4407" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F23.21.4407" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F23.21.4407"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7501463" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST039C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST039C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Becker</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Heun</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Barley microsatellites: allele variation and mapping</div>, <div class="source">Plant Mol. Biol.</div> , <div class="year">1995</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">835</div>-<div class="lpage">45</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/BF00020238" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/BF00020238</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Barley%20microsatellites%3A%20allele%20variation%20and%20mapping&amp;author=J.%20Becker&amp;author=M.%20Heun&amp;publication_year=1995&amp;journal=Plant%20Mol.%20Biol.&amp;volume=27&amp;pages=835-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00020238" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00020238" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00020238"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7727762" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST039C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST039C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kruglyak</div> <div class="given-names">L.</div></div>. </div><div class="article-title">The use of a genetic map of biallelic markers in linkage studies</div>, <div class="source">Nat. Genet.</div> , <div class="year">1997</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">21</div>-<div class="lpage">4</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/ng0997-21" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/ng0997-21</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20use%20of%20a%20genetic%20map%20of%20biallelic%20markers%20in%20linkage%20studies&amp;author=L.%20Kruglyak&amp;publication_year=1997&amp;journal=Nat.%20Genet.&amp;volume=17&amp;pages=21-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng0997-21" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng0997-21" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng0997-21"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9288093" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST039C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST039C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Choi</div> <div class="given-names">H.K.</div></div>,  <div class="name"><div class="surname">Kim</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Uhm</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">A sequence-based genetic map of <em>Medicago truncatula</em> and comparison of marker colinearity with <em>M. sativa</em></div>, <div class="source">Genetics</div> , <div class="year">2004</div>, vol. <div class="volume">166</div> (pg. <div class="fpage">1463</div>-<div class="lpage">502</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1534/genetics.166.3.1463" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1534/genetics.166.3.1463</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20sequence-based%20genetic%20map%20of%20Medicago%20truncatula%20and%20comparison%20of%20marker%20colinearity%20with%20M.%20sativa&amp;author=H.K.%20Choi&amp;author=D.%20Kim&amp;author=T.%20Uhm&amp;publication_year=2004&amp;journal=Genetics&amp;volume=166&amp;pages=1463-502" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.166.3.1463" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.166.3.1463" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.166.3.1463"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15082563" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST039C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST039C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yang</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Jin</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Zhao</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Zheng</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Xu</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Wu</div> <div class="given-names">W.</div></div>. </div><div class="article-title">PIP: a database for potential intron polymorphism markers</div>, <div class="source">Bioinformatics</div> , <div class="year">2007</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">2174</div>-<div class="lpage">7</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/bioinformatics/btm296" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/bioinformatics/btm296</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PIP%3A%20a%20database%20for%20potential%20intron%20polymorphism%20markers&amp;author=L.%20Yang&amp;author=G.%20Jin&amp;author=X.%20Zhao&amp;author=Y.%20Zheng&amp;author=Z.%20Xu&amp;author=W.%20Wu&amp;publication_year=2007&amp;journal=Bioinformatics&amp;volume=23&amp;pages=2174-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btm296" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtm296" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtm296"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17545179" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST039C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST039C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Zhao</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Zhu</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Wu</div> <div class="given-names">W.</div></div>. </div><div class="article-title">Genome-wide investigation of intron length polymorphisms and their potential as molecular markers in rice (<em>Oryza sativa</em> L.)</div>, <div class="source">DNA Res.</div> , <div class="year">2005</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">417</div>-<div class="lpage">27</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/dnares/dsi019" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/dnares/dsi019</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20investigation%20of%20intron%20length%20polymorphisms%20and%20their%20potential%20as%20molecular%20markers%20in%20rice%20(Oryza%20sativa%20L.)&amp;author=X.%20Wang&amp;author=X.%20Zhao&amp;author=J.%20Zhu&amp;author=W.%20Wu&amp;publication_year=2005&amp;journal=DNA%20Res.&amp;volume=12&amp;pages=417-27" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsi019" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsi019" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsi019"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16769698" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST039C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST039C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Braglia</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Manca</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Mastromauro</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Breviario</div> <div class="given-names">D.</div></div>. </div><div class="article-title">cTBP: a successful intron length polymorphism (ILP)-based genotyping method targeted to well defined experimental needs</div>, <div class="source">Diversity</div> , <div class="year">2010</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">572</div>-<div class="lpage">85</div>)<div class="pub-id-doi"><a href="https://doi.org/10.3390/d2040572" class="link link-pub-id-doi" target="_blank">https://doi.org/10.3390/d2040572</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=cTBP%3A%20a%20successful%20intron%20length%20polymorphism%20(ILP)-based%20genotyping%20method%20targeted%20to%20well%20defined%20experimental%20needs&amp;author=L.%20Braglia&amp;author=A.%20Manca&amp;author=F.%20Mastromauro&amp;author=D.%20Breviario&amp;publication_year=2010&amp;journal=Diversity&amp;volume=2&amp;pages=572-85" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3390/d2040572" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3390%2Fd2040572" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3390%2Fd2040572"> </span></div> </div></div></div></div></div><div content-id="DST039C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST039C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gupta</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Kumari</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Das</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Lata</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Puranik</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Prasad</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Development and utilization of novel intron length polymorphic markers in foxtail millet [<em>Setaria italica</em> (L.) P. Beauv.]</div>, <div class="source">Genome</div> , <div class="year">2011</div>, vol. <div class="volume">54</div> (pg. <div class="fpage">586</div>-<div class="lpage">602</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1139/g11-020" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1139/g11-020</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20and%20utilization%20of%20novel%20intron%20length%20polymorphic%20markers%20in%20foxtail%20millet%20%5BSetaria%20italica%20(L.)%20P.%20Beauv.%5D&amp;author=S.%20Gupta&amp;author=K.%20Kumari&amp;author=J.%20Das&amp;author=C.%20Lata&amp;author=S.%20Puranik&amp;author=M.%20Prasad&amp;publication_year=2011&amp;journal=Genome&amp;volume=54&amp;pages=586-602" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/g11-020" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2Fg11-020" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2Fg11-020"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21751869" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST039C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST039C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Saghai-Maroof</div> <div class="given-names">M.A.</div></div>,  <div class="name"><div class="surname">Biyaschev</div> <div class="given-names">R.M.</div></div>,  <div class="name"><div class="surname">Yang</div> <div class="given-names">G.P.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">Q.</div></div>,  <div class="name"><div class="surname">Allard</div> <div class="given-names">R.W.</div></div>. </div><div class="article-title">Extraordinary polymorphism microsatellite DNA in barley, species diversity, chromosomal location and population dynamics</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">1994</div>, vol. <div class="volume">91</div> (pg. <div class="fpage">5466</div>-<div class="lpage">70</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1073/pnas.91.12.5466" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1073/pnas.91.12.5466</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Extraordinary%20polymorphism%20microsatellite%20DNA%20in%20barley%2C%20species%20diversity%2C%20chromosomal%20location%20and%20population%20dynamics&amp;author=M.A.%20Saghai-Maroof&amp;author=R.M.%20Biyaschev&amp;author=G.P.%20Yang&amp;author=Q.%20Zhang&amp;author=R.W.%20Allard&amp;publication_year=1994&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=91&amp;pages=5466-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.91.12.5466" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.91.12.5466" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.91.12.5466"> </span></div> </div></div></div></div></div><div content-id="DST039C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST039C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Voorrips</div> <div class="given-names">R.E.</div></div>. </div><div class="article-title">MapChart: software for the graphical presentation of linkage maps and QTLs</div>, <div class="source">J. Hered.</div> , <div class="year">2002</div>, vol. <div class="volume">93</div> (pg. <div class="fpage">77</div>-<div class="lpage">8</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/jhered/93.1.77" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/jhered/93.1.77</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MapChart%3A%20software%20for%20the%20graphical%20presentation%20of%20linkage%20maps%20and%20QTLs&amp;author=R.E.%20Voorrips&amp;publication_year=2002&amp;journal=J.%20Hered.&amp;volume=93&amp;pages=77-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jhered/93.1.77" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjhered%2F93.1.77" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjhered%2F93.1.77"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12011185" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST039C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST039C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rotmistrovsky</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Jang</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Schuler</div> <div class="given-names">G.D.</div></div>. </div><div class="article-title">A web server for performing electronic PCR</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2004</div>, vol. <div class="volume">32</div> (pg. <div class="fpage">W108</div>-<div class="lpage">12</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/gkh450" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/gkh450</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20web%20server%20for%20performing%20electronic%20PCR&amp;author=K.%20Rotmistrovsky&amp;author=W.%20Jang&amp;author=G.D.%20Schuler&amp;publication_year=2004&amp;journal=Nucleic%20Acids%20Res.&amp;volume=32&amp;pages=W108-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkh450" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkh450" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkh450"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15215361" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST039C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST039C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Roldán-Ruiz</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Van Eeuwijk</div> <div class="given-names">F.A.</div></div>,  <div class="name"><div class="surname">Gilliland</div> <div class="given-names">T.J.</div></div>, et al. </div><div class="article-title">A comparative study of molecular and morphological methods of describing relationships between perennial ryegrass (<em>Lolium perenne</em> L.) varieties</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2001</div>, vol. <div class="volume">103</div> (pg. <div class="fpage">1138</div>-<div class="lpage">50</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s001220100571" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s001220100571</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20comparative%20study%20of%20molecular%20and%20morphological%20methods%20of%20describing%20relationships%20between%20perennial%20ryegrass%20(Lolium%20perenne%20L.)%20varieties&amp;author=I.%20Rold%C3%A1n-Ruiz&amp;author=F.A.%20Van%20Eeuwijk&amp;author=T.J.%20Gilliland&amp;publication_year=2001&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=103&amp;pages=1138-50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s001220100571" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs001220100571" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs001220100571"> </span></div> </div></div></div></div></div><div content-id="DST039C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST039C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jaccard</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Nouvelle recherches sur la distribution florale</div>, <div class="source">Bull. Soc. Vaud. Sci. Nat.</div> , <div class="year">1908</div>, vol. <div class="volume">44</div> (pg. <div class="fpage">223</div>-<div class="lpage">70</div>)</div></div></div></div><div content-id="DST039C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST039C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rohlf</div> <div class="given-names">F.J.</div></div>. </div>, <div class="source">NTSYS-pc numerical taxonomy and multivariance analysis system</div> , <div class="year">1997</div><div class="publisher-loc">Setauket, NY</div><div class="publisher-name">Exeter Software</div> <div class="comment">Version 2.02e <a class="link link-uri" href="http://www.exetersoftware.com/cat/ntsyspc/ntsyspc.html">http://www.exetersoftware.com/cat/ntsyspc/ntsyspc.html</a></div></div></div></div></div><div content-id="DST039C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST039C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nei</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Analysis of genetic diversity in subdivided populations</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">1973</div>, vol. <div class="volume">70</div> (pg. <div class="fpage">3321</div>-<div class="lpage">3</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1073/pnas.70.12.3321" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1073/pnas.70.12.3321</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20genetic%20diversity%20in%20subdivided%20populations&amp;author=M.%20Nei&amp;publication_year=1973&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=70&amp;pages=3321-3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.70.12.3321" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.70.12.3321" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.70.12.3321"> </span></div> </div></div></div></div></div><div content-id="DST039C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST039C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Liu</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Muse</div> <div class="given-names">M.</div></div>. </div><div class="article-title">PowerMarker: an integrated analysis environment for genetic marker analysis</div>, <div class="source">Bioinformatics</div> , <div class="year">2005</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">2128</div>-<div class="lpage">9</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/bioinformatics/bti282" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/bioinformatics/bti282</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PowerMarker%3A%20an%20integrated%20analysis%20environment%20for%20genetic%20marker%20analysis&amp;author=K.%20Liu&amp;author=M.%20Muse&amp;publication_year=2005&amp;journal=Bioinformatics&amp;volume=21&amp;pages=2128-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bti282" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbti282" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbti282"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15705655" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST039C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST039C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Krzywinski</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Schein</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Birol</div> <div class="given-names">I.</div></div>, et al. </div><div class="article-title">Circos, an information aesthetic for comparative genomics</div>, <div class="source">Genome Res.</div> , <div class="year">2009</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">1639</div>-<div class="lpage">45</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1101/gr.092759.109" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1101/gr.092759.109</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Circos%2C%20an%20information%20aesthetic%20for%20comparative%20genomics&amp;author=M.%20Krzywinski&amp;author=J.%20Schein&amp;author=I.%20Birol&amp;publication_year=2009&amp;journal=Genome%20Res.&amp;volume=19&amp;pages=1639-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.092759.109" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.092759.109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.092759.109"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19541911" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST039C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST039C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Palumbi</div> <div class="given-names">S.R.</div></div>. </div><div class="person-group"><div class="name"><div class="surname">Hillis</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Moritz</div> <div class="given-names">C</div></div>. </div><div class="article-title">Nucleic acids II: the polymerase chain reaction</div>, <div class="source">Molecular systematics</div> , <div class="year">1995</div><div class="edition">2nd edition</div><div class="publisher-loc">Sunderland, MA</div><div class="publisher-name">Sinauer Associates Inc.</div>(pg. <div class="fpage">205</div>-<div class="lpage">47</div>)</div></div></div></div><div content-id="DST039C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST039C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ferreira</div> <div class="given-names">A.O.</div></div>,  <div class="name"><div class="surname">Cardoso</div> <div class="given-names">H.G.</div></div>,  <div class="name"><div class="surname">Macedo</div> <div class="given-names">E.S.</div></div>,  <div class="name"><div class="surname">Breviario</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Arnholdt-Schmitt</div> <div class="given-names">B.</div></div>. </div><div class="article-title">Intron polymorphism pattern in AOX1b of wild St John's wort (<em>Hypericum perforatum</em>) allows discrimination between individual plants</div>, <div class="source">Physiol. Plant.</div> , <div class="year">2009</div>, vol. <div class="volume">137</div> (pg. <div class="fpage">520</div>-<div class="lpage">31</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1111/j.1399-3054.2009.01291.x" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1111/j.1399-3054.2009.01291.x</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Intron%20polymorphism%20pattern%20in%20AOX1b%20of%20wild%20St%20John's%20wort%20(Hypericum%20perforatum)%20allows%20discrimination%20between%20individual%20plants&amp;author=A.O.%20Ferreira&amp;author=H.G.%20Cardoso&amp;author=E.S.%20Macedo&amp;author=D.%20Breviario&amp;author=B.%20Arnholdt-Schmitt&amp;publication_year=2009&amp;journal=Physiol.%20Plant.&amp;volume=137&amp;pages=520-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/ppl.2009.137.issue-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fppl.2009.137.issue-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fppl.2009.137.issue-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19843238" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST039C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST039C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Choudhary</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Sethy</div> <div class="given-names">N.K.</div></div>,  <div class="name"><div class="surname">Shokeen</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Bhatia</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Development of chickpea EST-SSR markers and analysis of allelic variation across related species</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2009</div>, vol. <div class="volume">118</div> (pg. <div class="fpage">591</div>-<div class="lpage">608</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s00122-008-0923-z" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s00122-008-0923-z</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20chickpea%20EST-SSR%20markers%20and%20analysis%20of%20allelic%20variation%20across%20related%20species&amp;author=S.%20Choudhary&amp;author=N.K.%20Sethy&amp;author=B.%20Shokeen&amp;author=S.%20Bhatia&amp;publication_year=2009&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=118&amp;pages=591-608" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-008-0923-z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-008-0923-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-008-0923-z"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19020854" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST039C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST039C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gupta</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Prasad</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Development and characterization of genic SSR markers in <em>Medicago truncatula</em> and their transferability in leguminous and non-leguminous species</div>, <div class="source">Genome</div> , <div class="year">2009</div>, vol. <div class="volume">52</div> (pg. <div class="fpage">761</div>-<div class="lpage">71</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1139/G09-051" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1139/G09-051</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20and%20characterization%20of%20genic%20SSR%20markers%20in%20Medicago%20truncatula%20and%20their%20transferability%20in%20leguminous%20and%20non-leguminous%20species&amp;author=S.%20Gupta&amp;author=M.%20Prasad&amp;publication_year=2009&amp;journal=Genome&amp;volume=52&amp;pages=761-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/G09-051" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2FG09-051" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2FG09-051"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19935924" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST039C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST039C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ming</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Xie</div> <div class="given-names">F-M.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">L-V.</div></div>,  <div class="name"><div class="surname">Zhao</div> <div class="given-names">X-Q.</div></div>,  <div class="name"><div class="surname">Jojee</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Madonna</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Comparative analysis of genetic diversity and structure in rice using ILP and SSR markers</div>, <div class="source">Rice Sci.</div> , <div class="year">2010</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">257</div>-<div class="lpage">68</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/S1672-6308(09)60025-1" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/S1672-6308(09)60025-1</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20analysis%20of%20genetic%20diversity%20and%20structure%20in%20rice%20using%20ILP%20and%20SSR%20markers&amp;author=H.%20Ming&amp;author=F-M.%20Xie&amp;author=L-V.%20Chen&amp;author=X-Q.%20Zhao&amp;author=L.%20Jojee&amp;author=D.%20Madonna&amp;publication_year=2010&amp;journal=Rice%20Sci.&amp;volume=17&amp;pages=257-68" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S1672-6308(09)60025-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS1672-6308(09)60025-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS1672-6308(09)60025-1"> </span></div> </div></div></div></div></div><div content-id="DST039C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST039C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Devos</div> <div class="given-names">K.M.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">Z.M.</div></div>,  <div class="name"><div class="surname">Beales</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Sasaki</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Gale</div> <div class="given-names">M.D.</div></div>. </div><div class="article-title">Comparative genetic maps of foxtail millet (<em>Setaria italica</em>) and rice (<em>Oryza sativa</em>)</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">1998</div>, vol. <div class="volume">96</div> (pg. <div class="fpage">63</div>-<div class="lpage">8</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s001220050709" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s001220050709</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20genetic%20maps%20of%20foxtail%20millet%20(Setaria%20italica)%20and%20rice%20(Oryza%20sativa)&amp;author=K.M.%20Devos&amp;author=Z.M.%20Wang&amp;author=J.%20Beales&amp;author=T.%20Sasaki&amp;author=M.D.%20Gale&amp;publication_year=1998&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=96&amp;pages=63-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s001220050709" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs001220050709" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs001220050709"> </span></div> </div></div></div></div></div><div content-id="DST039C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST039C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Devos</div> <div class="given-names">K.M.</div></div>,  <div class="name"><div class="surname">Gale</div> <div class="given-names">M.D.</div></div>. </div><div class="article-title">Genome relationships: the grass model in current research</div>, <div class="source">Plant Cell</div> , <div class="year">2000</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">637</div>-<div class="lpage">46</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20relationships%3A%20the%20grass%20model%20in%20current%20research&amp;author=K.M.%20Devos&amp;author=M.D.%20Gale&amp;publication_year=2000&amp;journal=Plant%20Cell&amp;volume=12&amp;pages=637-46" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.12.5.637" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.12.5.637" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.12.5.637"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10810140" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST039C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST039C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Devos</div> <div class="given-names">K.M.</div></div>. </div><div class="article-title">Updating the crop circle</div>, <div class="source">Curr. Opin. Plant Biol.</div> , <div class="year">2005</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">155</div>-<div class="lpage">62</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.pbi.2005.01.005" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.pbi.2005.01.005</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Updating%20the%20crop%20circle&amp;author=K.M.%20Devos&amp;publication_year=2005&amp;journal=Curr.%20Opin.%20Plant%20Biol.&amp;volume=8&amp;pages=155-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.pbi.2005.01.005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.pbi.2005.01.005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.pbi.2005.01.005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15752995" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST039C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST039C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Srinivasachary</div> <div class="given-names">D.M.M.</div></div>,  <div class="name"><div class="surname">Gale</div> <div class="given-names">M.D.</div></div>,  <div class="name"><div class="surname">Devos</div> <div class="given-names">K.M.</div></div>. </div><div class="article-title">Comparative analyses reveal high levels of conserved colinearity between the finger millet and rice genomes</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2007</div>, vol. <div class="volume">115</div> (pg. <div class="fpage">489</div>-<div class="lpage">99</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s00122-007-0582-5" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s00122-007-0582-5</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20analyses%20reveal%20high%20levels%20of%20conserved%20colinearity%20between%20the%20finger%20millet%20and%20rice%20genomes&amp;author=D.M.M.%20Srinivasachary&amp;author=M.D.%20Gale&amp;author=K.M.%20Devos&amp;publication_year=2007&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=115&amp;pages=489-99" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-007-0582-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-007-0582-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-007-0582-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17619853" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div><div content-id=""><div class="fn"><div class="footnote-content">Edited by Dr Masahiro Yano</div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2013. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=347328&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst039/2/dst039-dst039supp_table8.xls?Expires=2147483647&amp;Signature=ix3ZEtV7f4SU3lv3fDNdWHdlS-5S7-9k-wIzIUjrURYun2r910eNcjw0zmg2sabUEK-l9SAmoGVp2DGaD6NBGV~7Wj2KgbcpV6gVvWXFdGVlVJCiGj5Oy-hOdo4M~Uh7zCTIDci21yAbzWSQFFMilf~yZ2ssCdmx1EGPDnu5Phhi9mMV9Mm3YfbLuzKeQSIvOsiaH~fxAgx7nTbCEUQV3duTZ9wrPsHjXjMqdAj3vjYI1lEwGK5U-ZbP1SKHRsGzS~NhbG~7XL-I4jGLylSf3VEVuwb-TrVODMFpudaE-II2337rIjopKOW~pdzVn0wfpWfzv~1rrSg0LLrQqPpEQA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-8</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst039/2/dst039-dst039supp_table7.xls?Expires=2147483647&amp;Signature=sIQPvXxGusEvki~GJEjbbHo2HW4ofp7yfTB0Ih-PbUfSUYcB9xRucgtBlxQAAJfd8uxFxSEA~MHzlNvB6gPg-lfGHakJzWsfAS9t9rFbxoEmq8H17cAhux04hZ3k7VeEMTn8Y-s64Ds9qTexcbtfBUmAtwe~5Um6yK7MWxcvXJLAW~owq3XWJ9Qf4vZWk4f4IJOid432QYhofmaR1wB06eCtKtqzm3PeQZOvH-OPg8nMjEky0u8KBtrmpthTlmjTSj3mmknokDg9u2YYPTITY5AGjpNPNco6Mb8l0TASOeLPOuUTJY~32cPk53Sc~0kmAdgMDLaZgfZtBH6uhSxmFA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-7</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst039/2/dst039-dst039supp_table6.xls?Expires=2147483647&amp;Signature=dgp2pv~PeSKGR9ITwxr0rXE~4oj8o5CV5xeqkIhG~MfhckQKlwwn~OnuZbpXkS6NerlynGb9OU~8VXGpQqQLJDj3~jhnaO5vthUt3tQLihOoGDuoHa2lmPrGgbGhuRYDJ9S9htXdPlt7BTw6U~qwDTM~Bz9-u6mKS1FJxjQxV7fHX4XUt~JwbIB14nlE5yh4HSQ5yHaVhHxvB4oG0EWDbbXrNkwPGlhxTU2coJZ0R3vHcVtICSYtpVvtzMnQ4q7Hvu~8h1d4cmDUmnkv5gHk-ZgifCz-8HzEyPVzENJ-TOQpl9aXez7LIMqxSj04kgpMwJOomsE8CpIu-VHKstG-NQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-6</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst039/2/dst039-dst039supp_table5.xls?Expires=2147483647&amp;Signature=w0AQdfbw4cWZ8~xWqwxe8AP24lbp~urG6qX18mH0PyL1eiGnCTJsBF1WfBcEoA5a2FpmCsneEHOwTf30FE8q1QmoqgRIcCdfBNehs2XDcl8sKTYfJ8T4TasFWElj6e4sK53QnqMG4Ol5MlhZdajuZG7ot-Cj6Rpeg2bu2G2Gmx-LJm8ZKLzJZquHyIdBZS4lySVBRflqRFUG9E7ALkHDWf8Zrvj8KA50f-opdyDNKf-MofM6ptZB~EiuFjTScwc4ii-6aCqgr78i9M6mceH8qCmJQGqexTMQcCeXMaYvqFZCzFQDHTLiMwLVG8Ajb9Zg9-~fjXT5vufhelV9x3xx6w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-5</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst039/2/dst039-dst039supp_table4.xls?Expires=2147483647&amp;Signature=v1U700cO6jfJ9C-e2geiHukxIYH-259rMzqVq6sgYp1nKK5wDu99oImRaCW0xEKDJZGcKDPiWy5xgjM7odLp6EOsag0vhAQk2Z-U7n6Wl7Ykgg0Y2Nxmcj~b4PfyuzYeTQQ1-KPN4twrHnR6L7KSam4MSr2KJ83exMUI7DHVqNFS6FfntUOiO47~nXqSMAfILh0-8Dh79vUcOukZanNxUxfmIFYdC~LurbNPkO8-jjxf~9b4bKw0uTfzmm1apHk3hEAztfyTJkJrWmIxP~VtGn0MlIIAWAcGBBpLoC4hLcOkaVt4ZuERDNhVgIUfCdKZxGTPun16wQbtC62aCK~SaA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-4</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst039/2/dst039-dst039supp_table3.xls?Expires=2147483647&amp;Signature=DcIFXFLIdPlozEO8yHHufy5YPR9UphrOiB~F0yiDWATGRArkYyIP~o~eh6nIFWIRiEbc30X~5hM3GtH6cdlZkDrNPO92zm8okEKafo9J4wtZ-b737dd0TB7bOXgr-N~J53Ht1yf2pGkETmfDkSDhPmCvJdCiirDtA1hVLweGU45Z-Cmn5AP8yoXiVO9EBafWZ6~GOFxCOMN9qVdrkeDaMoSPoMO7byvdbQCAQlI5x7C3lOO9X9xVxzCRNkS1CV3KMn18OyuGQjc5ejKlt5lrZkXzzYKNmSP24TPhjPx3Ef85MW9GKqDB5txToWLD664uCU-QCxkcyOPX4U7lOAuoaw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst039/2/dst039-dst039supp_table2.xls?Expires=2147483647&amp;Signature=aAzUFQRiWEv6Di2wzTpHnpsjKR-aQs88PuHdgiq8zOtAwej4ZoZi3rH8eA3j3icWPRMUoEoP72qezH4Jo43r57CZX27I51fS4KiLB4H71inqpZS8TvFcD~YDOZoZBHLPvV-4uv1x-QC9H2Ec~sIZmR480GdBcE28v0zOSC002cDAD2qEuSD32n9rUJJApZ~s9rh3SPq8RyWD7BaC8TrgTJ2k2KZSKC~trPllJlQEirRCHIzgSPHs8CMGXmeINpKNT8GdbU0gyxt05Vd8Kd7WrF9qHMQFjzR3WTo7J-4d5RZSe3qzAUe9bRAFhwB7K2oskmeAkulM-REJcnikcIOXsg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst039/2/dst039-dst039supp_table1.doc?Expires=2147483647&amp;Signature=pfy5AZYf7eGz1zZ183ztH7pjb2POv7BIex7WffbyIyE4o4xKyK6o8HXcdn8WjTiMs1e32Wyq6nqsp7f9O3RODSiopf32yOkQDJ5wLUYhwaIzhi2WxLKASoV20S35ZseiYwr6KxjXaasc02zX2Pgxu1bg09AlFao3rW9yWxD33FAJDdXFcquk4xxY-cc7ckODtM7lcwKHzXXeH3EUyllWstBVOJ9mxX4CilrSkmbGzfY414NywX1TNkCalric-cu~FfyazI1avzKvyGiVpsoY3k96IsRaLOrq1ZdQaF8vhp7-MetvYEgGkaD-he4-GIIe3oAyVvzSBefNCaAJRjYL0Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - doc file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">387</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000339897000004&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">35</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dst039"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="347328"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000339897000004&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f21%2f1%2f41%2f347328">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000339897000004&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (35)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F21%2F1%2F41%2F347328" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/347328'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Development of 5123 Intron-Length Polymorphic Markers for Large-Scale Genotyping Applications in Foxtail Millet" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dst039");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f21%2f1%2f41%2f347328">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

