

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" /><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info = {"beacon":"bam.nr-data.net","errorBeacon":"bam.nr-data.net","licenseKey":"8d76ca6f26","applicationID":"29565620","transactionName":"ZQBRMkVUCEtRVkINXlxKcjVnGgdKRFxVCFQcBEAWTw==","queueTime":1,"applicationTime":2284,"agent":"","atts":""}</script><script type="text/javascript">(window.NREUM||(NREUM={})).loader_config={xpid:"XAYBV15WGwEJUVdXAQMH"};window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{s.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(21),s={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(s.console=!0,o.indexOf("dev")!==-1&&(s.dev=!0),o.indexOf("nr_dev")!==-1&&(s.nrDev=!0))}catch(c){}s.nrDev&&i.on("internal-error",function(t){r(t.stack)}),s.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),s.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(s,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,s){try{l?l-=1:o(s||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,c.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:c.now();i("err",[t,n])}var i=t("handle"),a=t(22),s=t("ee"),c=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;c.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),c.xhrWrappable&&t(14),d=!0)}s.on("fn-start",function(t,e,n){d&&(l+=1)}),s.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),s.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),s.on("internal-error",function(t){i("ierr",[t,c.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){M++,N=y.hash,this[u]=g.now()}function o(){M--,y.hash!==N&&i(0,!0);var t=g.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+y,e])}function a(t,e){t.on(e,function(){this[e]=g.now()})}var s="-start",c="-end",f="-body",u="fn"+s,d="fn"+c,p="cb"+s,l="cb"+c,h="jsTime",m="fetch",v="addEventListener",w=window,y=w.location,g=t("loader");if(w[v]&&g.xhrWrappable){var b=t(10),x=t(11),E=t(8),O=t(6),P=t(13),R=t(7),T=t(14),L=t(9),j=t("ee"),S=j.get("tracer");t(15),g.features.spa=!0;var N,M=0;j.on(u,r),j.on(p,r),j.on(d,o),j.on(l,o),j.buffer([u,d,"xhr-done","xhr-resolved"]),O.buffer([u]),P.buffer(["setTimeout"+c,"clearTimeout"+s,u]),T.buffer([u,"new-xhr","send-xhr"+s]),R.buffer([m+s,m+"-done",m+f+s,m+f+c]),E.buffer(["newURL"]),b.buffer([u]),x.buffer(["propagate",p,l,"executor-err","resolve"+s]),S.buffer([u,"no-"+u]),L.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(T,"send-xhr"+s),a(j,"xhr-resolved"),a(j,"xhr-done"),a(R,m+s),a(R,m+"-done"),a(L,"new-jsonp"),a(L,"jsonp-end"),a(L,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,M>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),s=t(12),c="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",y="pushState",g=t("loader");g.features.stn=!0,t(8);var b=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof b&&(this.bstStart=g.now())}),o.on(v,function(t,e){var n=t[0];n instanceof b&&i("bst",[n,e,this.bstStart,g.now()])}),a.on(m,function(t,e,n){this.bstStart=g.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),this.bstType])}),s.on(m,function(){this.bstStart=g.now()}),s.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),"requestAnimationFrame"])}),o.on(y+l,function(t){this.time=g.now(),this.startPath=location.pathname+location.hash}),o.on(y+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+c]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+c]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+c]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){s.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),s=t(24)(a,!0),c=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=c(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?s(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=r.apply(this,arguments);return o.emit(n+"start",arguments,t),t.then(function(e){return o.emit(n+"end",[null,e],t),e},function(e){throw o.emit(n+"end",[e],t),e})})}var o=t("ee").get("fetch"),i=t(21);e.exports=o;var a=window,s="fetch-",c=s+"body-",f=["arrayBuffer","blob","json","text","formData"],u=a.Request,d=a.Response,p=a.fetch,l="prototype";u&&d&&p&&(i(f,function(t,e){r(u[l],e,c),r(d[l],e,c)}),r(a,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t(24)(r);e.exports=r,o.inPlace(window.history,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){c.emit("jsonp-error",[],p),c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=s(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),c.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function s(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var c=t("ee").get("jsonp"),f=t(24)(c);if(e.exports=c,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-"),c.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t(24)(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=s(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t(24),a=t("ee").get("promise"),s=i(a),c=t(21),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;c(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),s=f.resolve(i);return s}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),c(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=s(t[0],"resolve-",this),t[1]=s(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),s.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=s(t[0],"cb-",this),t[1]=s(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t(24)(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t(24)(i),s="setTimeout",c="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[s,"setImmediate"],s+d),a.inPlace(window,[c],c+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(c+u,r),i.on(s+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",s)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,y,"fn-",s)}function i(t){g.push(t),h&&(x?x.then(a):v?v(a):(E=-E,O.data=E))}function a(){for(var t=0;t<g.length;t++)r([],g[t]);g.length&&(g=[])}function s(t,e){return e}function c(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t(24)(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",y=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],g=[];e.exports=u;var b=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(c(l,b),b.prototype=l.prototype,d.inPlace(b.prototype,["open","send"],"-xhr-",s),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var x=m&&m.resolve();if(!v&&!m){var E=1,O=document.createTextNode(E);new h(a).observe(O,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<d;r++)t.removeEventListener(u[r],this.listener,!1);if(!e.aborted){if(n.duration=a.now()-this.startTime,4===t.readyState){e.status=t.status;var i=o(t,this.lastSize);if(i&&(n.rxSize=i),this.sameOrigin){var c=t.getResponseHeader("X-NewRelic-App-Data");c&&(e.cat=c.split(", ").pop())}}else e.status=0;n.cbTime=this.cbTime,f.emit("xhr-done",[t],t),s("xhr",[e,n,this.startTime])}}}function o(t,e){var n=t.responseType;if("json"===n&&null!==e)return e;var r="arraybuffer"===n||"blob"===n||"json"===n?t.response:t.responseText;return h(r)}function i(t,e){var n=c(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.sameOrigin=n.sameOrigin}var a=t("loader");if(a.xhrWrappable){var s=t("handle"),c=t(16),f=t("ee"),u=["load","error","abort","timeout"],d=u.length,p=t("id"),l=t(19),h=t(18),m=window.XMLHttpRequest;a.features.xhr=!0,t(14),f.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,l&&(l>34||l<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),f.on("open-xhr-start",function(t){this.params={method:t[0]},i(this,t[1]),this.metrics={}}),f.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid)}),f.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=h(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"===t.type&&(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{f.emit("internal-error",[n])}catch(r){}}};for(var s=0;s<d;s++)e.addEventListener(u[s],this.listener,!1)}),f.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),f.on("xhr-load-added",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),f.on("xhr-load-removed",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),f.on("addEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-added",[t[1],t[2]],e)}),f.on("removeEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-removed",[t[1],t[2]],e)}),f.on("fn-start",function(t,e,n){e instanceof m&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),f.on("fn-end",function(t,e){this.xhrCbStart&&f.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)})}},{}],16:[function(t,e,n){e.exports=function(t){var e=document.createElement("a"),n=window.location,r={};e.href=t,r.port=e.port;var o=e.href.split("://");!r.port&&o[1]&&(r.port=o[1].split("/")[0].split("@").pop().split(":")[1]),r.port&&"0"!==r.port||(r.port="https"===o[0]?"443":"80"),r.hostname=e.hostname||n.hostname,r.pathname=e.pathname,r.protocol=o[0],"/"!==r.pathname.charAt(0)&&(r.pathname="/"+r.pathname);var i=!e.protocol||":"===e.protocol||e.protocol===n.protocol,a=e.hostname===document.domain&&e.port===n.port;return r.sameOrigin=i&&(!e.hostname||a),r}},{}],17:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(s(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(21),s=t(22),c=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(c.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw c.emit("fn-err",[arguments,this,t],n),t}finally{c.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now()])}},{}],18:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],19:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],20:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var s=navigator.userAgent,c=s.match(a);c&&s.indexOf("Chrome")===-1&&s.indexOf("Chromium")===-1&&(o="Safari",i=c[1])}e.exports={agent:o,version:i,match:r}},{}],21:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],22:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],23:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],24:[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(22),a="nr@original",s=Object.prototype.hasOwnProperty,c=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,s,c;try{a=this,r=i(arguments),s="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],s])}u(e+"start",[r,a,o],s);try{return c=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],s),d}finally{u(e+"end",[r,a,c],s)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,s,c,f="-"===o.charAt(0);for(c=0;c<e.length;c++)s=e[c],a=t[s],r(a)||(t[s]=n(a,f?s+o:o,i,s))}function u(n,r,o){if(!c||e){var i=c;c=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}c=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)s.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?c(t,s,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),s=m(n),c=s.length,f=0;f<c;f++)s[f].apply(a,r);var d=u[g[n]];return d&&d.push([b,n,r,a]),a}}function l(t,e){y[t]=m(t).concat(e)}function h(t,e){var n=y[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return y[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",g[n]=e,e in u||(u[e]=[])})}var y={},g={},b={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return b}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var s="nr@context",c=t("gos"),f=t(21),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=x.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(g,function(e,n){t[e]||(t[e]=n)}),c("mark",["onload",a()+x.offset],null,"api");var n=l.createElement("script");n.src="https://"+t.agent,e.parentNode.insertBefore(n,e)}}function o(){"complete"===l.readyState&&i()}function i(){c("mark",["domContent",a()+x.offset],null,"api")}function a(){return O.exists&&performance.now?Math.round(performance.now()):(s=Math.max((new Date).getTime(),s))-x.offset}var s=(new Date).getTime(),c=t("handle"),f=t(21),u=t("ee"),d=t(20),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var y=""+location,g={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1099.min.js"},b=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),x=e.exports={offset:s,now:a,origin:y,features:{},xhrWrappable:b,userAgent:d};t(17),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),c("mark",["firstbyte",s],null,"api");var E=0,O=t(23)},{}]},{},["loader",2,15,5,3,4]);</script>
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/24/2/103/2962881","@type":"ScholarlyArticle","name":"Somatic transposition and meiotically driven elimination of an active helitron family in  Pleurotus ostreatus ","datePublished":"2017-01-29","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/24/2","@type":"PublicationIssue","issueNumber":"2","datePublished":"2017-04-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsw060","keywords":["helitrons","segregation distortion","gene conversion","transposable elements","basidiomycete"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/24/2/103/2962881","author":[{"name":"Borgognone, Alessandra","@type":"Person"},{"name":"Castanera, Raúl","@type":"Person"},{"name":"Muguerza, Elaia","@type":"Person"},{"name":"Pisabarro, Antonio G.","@type":"Person"},{"name":"Ramírez, Lucía","@type":"Person"}],"description":"Abstract.  Helitrons constitute a superfamily of DNA transposons that were discovered in silico and are widespread in most eukaryotic genomes. They are postulat","pageStart":"103","pageEnd":"115","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/2/2/m_dnares_24_2cover.png?Expires=2147483647&Signature=C8sLyIyfepNN-Oy5BR2xLoVJLx2W1jQBt~48a2lQ~mjys3EWTAKP5k2kV~t7b8sG~Qk1B4iVdz8BINFP5bR-Hc0gvwqekGHM8qavh8kDaLAfN~qNxrDj7NNeLHJ-IURMi5v1ATu23u5wB01wP27BHknbA3W9JgGnq6AVGPqeLfhK7AJcTzOYzKpGYwaedh5crkYXYHp~9GCcHuTjoInf3sKaYC2Ew-HnuA-yIL45j8ocXrNQuIxJV8rFXwx1Tt0b7I1ZJg09R4-7m-GZB9vPgdehgAXUMUNQ4gDatBY22xUt9COqnlb7RZKNUXFDo7Qe0LNJgSHwNUOJvsh-SLpXkg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Somatic transposition and meiotically driven elimination of an active helitron family in  Pleurotus ostreatus ","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/2/2/m_dnares_24_2cover.png?Expires=2147483647&Signature=C8sLyIyfepNN-Oy5BR2xLoVJLx2W1jQBt~48a2lQ~mjys3EWTAKP5k2kV~t7b8sG~Qk1B4iVdz8BINFP5bR-Hc0gvwqekGHM8qavh8kDaLAfN~qNxrDj7NNeLHJ-IURMi5v1ATu23u5wB01wP27BHknbA3W9JgGnq6AVGPqeLfhK7AJcTzOYzKpGYwaedh5crkYXYHp~9GCcHuTjoInf3sKaYC2Ew-HnuA-yIL45j8ocXrNQuIxJV8rFXwx1Tt0b7I1ZJg09R4-7m-GZB9vPgdehgAXUMUNQ4gDatBY22xUt9COqnlb7RZKNUXFDo7Qe0LNJgSHwNUOJvsh-SLpXkg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Somatic transposition and meiotically driven elimination of an active helitron family in  Pleurotus ostreatus " />
    <meta property="og:description" content="Abstract.  Helitrons constitute a superfamily of DNA transposons that were discovered in silico and are widespread in most eukaryotic genomes. They are postulat" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/24/2/103/2962881" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2017-01-29" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Borgognone, Alessandra" /><meta name="citation_author" content="Castanera, Raúl" /><meta name="citation_author" content="Muguerza, Elaia" /><meta name="citation_author" content="Pisabarro, Antonio G." /><meta name="citation_author" content="Ramírez, Lucía" /><meta name="citation_title" content="Somatic transposition and meiotically driven elimination of an active helitron family in Pleurotus ostreatus" /><meta name="citation_firstpage" content="103" /><meta name="citation_lastpage" content="115" /><meta name="citation_doi" content="10.1093/dnares/dsw060" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="24" /><meta name="citation_issue" content="2" /><meta name="citation_publication_date" content="2017/04/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Mobile elements: drivers of genome evolution;  citation_journal_title=Science;  citation_year=2004;  citation_volume=303;  citation_pages=1626-32; " /><meta name="citation_reference" content="citation_title=A unified classification system for eukaryotic transposable elements;  citation_journal_title=Nat. Rev. Genet;  citation_year=2007;  citation_volume=8;  citation_pages=973-82; " /><meta name="citation_reference" content="citation_title=Rolling-circle transposons in eukaryotes;  citation_journal_title=Proc. Natl. Acad. Sci. U. S. A;  citation_year=2001;  citation_volume=98;  citation_pages=8714-9; " /><meta name="citation_reference" content="citation_title=Vertebrate helentrons and other novel Helitrons;  citation_journal_title=Gene;  citation_year=2003;  citation_volume=313;  citation_pages=201-12; " /><meta name="citation_reference" content="citation_title=Massive amplification of rolling-circle transposons in the lineage of the bat Myotis lucifugus;  citation_journal_title=Proc. Natl. Acad. Sci. U. S. A;  citation_year=2007;  citation_volume=104;  citation_pages=1895-900; " /><meta name="citation_reference" content="citation_title=The making of a genomic parasite - the Mothra family sheds light on the evolution of Helitrons in plants;  citation_journal_title=Mob. DNA;  citation_year=2015;  citation_volume=6;  citation_pages=23" /><meta name="citation_reference" content="citation_title=Helitrons on a roll: eukaryotic rolling-circle transposons;  citation_journal_title=Trends Genet;  citation_year=2007;  citation_pages=521-9; " /><meta name="citation_reference" content="citation_title=Excision of Helitron transposons in maize;  citation_journal_title=Genetics;  citation_year=2009;  citation_volume=182;  citation_pages=399-402; " /><meta name="citation_reference" content="citation_title=Gene movement by Helitron transposons contributes to the haplotype variability of maize;  citation_journal_title=Proc. Natl. Acad. Sci. U. S. A;  citation_year=2005;  citation_volume=102;  citation_pages=9068-73; " /><meta name="citation_reference" content="citation_title=Spontaneous mutations caused by a Helitron transposon, Hel-It1, in morning glory, Ipomoea tricolor;  citation_journal_title=Plant J;  citation_year=2007;  citation_volume=49;  citation_pages=924-34; " /><meta name="citation_reference" content="citation_title=Intraspecific violation of genetic colinearity and its implications in maize;  citation_journal_title=Proc. Natl. Acad. Sci. U. S. A;  citation_year=2002;  citation_volume=99;  citation_pages=9573-8; " /><meta name="citation_reference" content="citation_title=A Helitron transposon reconstructed from bats reveals a novel mechanism of genome shuffling in eukaryotes;  citation_journal_title=Nat. Commun;  citation_year=2016;  citation_volume=7;  citation_pages=10716" /><meta name="citation_reference" content="citation_title=Helitron Transposons on the Sex Chromosomes of the Platyfish Xiphophorus maculatus and Their Evolution in Animal Genomes;  citation_journal_title=Zebrafish;  citation_year=2006;  citation_volume=3;  citation_pages=39-52; " /><meta name="citation_reference" content="citation_title=The Maize Genome Contains a Helitron Insertion;  citation_journal_title=Plant Cell Online;  citation_year=2003;  citation_volume=15;  citation_pages=381-91; " /><meta name="citation_reference" content="citation_title=HelitronScanner uncovers a large overlooked cache of Helitron transposons in many plant genomes;  citation_journal_title=Proc. Natl. Acad. Sci;  citation_year=2014;  citation_volume=111;  citation_pages=10263-8; " /><meta name="citation_reference" content="citation_title=Gene duplication and exon shuffling by helitron-like transposons generate intraspecies diversity in maize;  citation_journal_title=Nat. Genet;  citation_year=2005;  citation_volume=37;  citation_pages=997-1002; " /><meta name="citation_reference" content="citation_title=Highly expressed captured genes and cross-kingdom domains present in Helitrons create novel diversity in Pleurotus ostreatus and other fungi;  citation_journal_title=BMC Genomics;  citation_year=2014;  citation_volume=15;  citation_pages=1071" /><meta name="citation_reference" content="citation_title=The tightly regulated promoter of the xanA gene of Aspergillus nidulans is included in a helitron;  citation_journal_title=Mol. Microbiol;  citation_year=2007;  citation_volume=63;  citation_pages=1577-87; " /><meta name="citation_reference" content="citation_title=Périgord black truffle genome uncovers evolutionary origins and mechanisms of symbiosis;  citation_journal_title=Nature;  citation_year=2010;  citation_volume=464;  citation_pages=1033-8; " /><meta name="citation_reference" content="citation_title=Characterization of transposable elements in the ectomycorrhizal fungus Laccaria bicolor;  citation_journal_title=PLoS One;  citation_year=2012;  citation_volume=7; " /><meta name="citation_reference" content="citation_title=Quantitative linkage mapping of lignin-degrading enzymatic activities in Pleurotus ostreatus;  citation_journal_title=Enzyme Microb. Technol;  citation_year=2008;  citation_volume=43;  citation_pages=137-43; " /><meta name="citation_reference" content="citation_title=The genetics and cultivation of Pleurotus ostreatus;  citation_journal_title=Mycologia;  citation_year=1968;  citation_volume=60;  citation_pages=627" /><meta name="citation_reference" content="citation_title=Cultivation of Pleurotus ostreatus and other edible mushrooms;  citation_journal_title=Appl. Microbiol. Biotechnol;  citation_year=2010;  citation_volume=85;  citation_pages=1321-37; " /><meta name="citation_reference" content="citation_title=Molecular karyotype of the white rot fungus Pleurotus ostreatus;  citation_journal_title=Appl. Envir. Microbiol;  citation_year=1999;  citation_volume=65;  citation_pages=3413-7; " /><meta name="citation_reference" content="citation_title=Genetic linkage map of the edible basidiomycete Pleurotus ostreatus;  citation_journal_title=Appl. Environ. Microbiol;  citation_year=2000;  citation_volume=66;  citation_pages=5290-300; " /><meta name="citation_reference" content="citation_title=Genetic linkage map and expression analysis of genes expressed in the lamellae of the edible basidiomycete Pleurotus ostreatus;  citation_journal_title=Fungal Genet. Biol. FG B;  citation_year=2006;  citation_volume=43;  citation_pages=376-87; " /><meta name="citation_reference" content="citation_title=Transposable elements versus the fungal genome: impact on whole-genome architecture and transcriptional profiles;  citation_journal_title=PLoS Genet;  citation_year=2016;  citation_volume=12;  citation_pages=e1006108" /><meta name="citation_reference" content="citation_title=Mapping of genomic regions (quantitative trait loci) controlling production and quality in industrial cultures of the edible basidiomycete Pleurotus ostreatus;  citation_journal_title=Appl. Environ. Microbiol;  citation_year=2003;  citation_volume=69;  citation_pages=3617-25; " /><meta name="citation_reference" content="citation_title=Extensive sampling of basidiomycete genomes demonstrates inadequacy of the white-rot/brown-rot paradigm for wood decay fungi;  citation_journal_title=Proc. Natl. Acad. Sci. U. S. A;  citation_year=2014;  citation_volume=111;  citation_pages=9923-8; " /><meta name="citation_reference" content="citation_title=Comparative and transcriptional analysis of the predicted secretome in the lignocellulose-degrading basidiomycete fungus Pleurotus ostreatus. Environ;  citation_journal_title=Microbiol;  citation_year=2016; " /><meta name="citation_reference" content="citation_title=Primer3-new capabilities and interfaces;  citation_journal_title=Nucleic Acids Res;  citation_year=2012;  citation_volume=40; " /><meta name="citation_reference" content="citation_title=Basic local alignment search tool;  citation_journal_title=J. Mol. Biol;  citation_year=1990;  citation_volume=215;  citation_pages=403-10; " /><meta name="citation_reference" content="citation_title=A new mathematical model for relative quantification in real-time RT-PCR;  citation_journal_title=Nucleic Acids Res;  citation_year=2001;  citation_volume=29;  citation_pages=e45" /><meta name="citation_reference" content="citation_title=Accelerated Profile HMM Searches Pearson, W. R., (ed.),;  citation_journal_title=PLoS Comput. Biol;  citation_year=2011;  citation_volume=7;  citation_pages=e1002195" /><meta name="citation_reference" content="citation_title=Pfam: the protein families database;  citation_journal_title=Nucleic Acids Res;  citation_year=2014;  citation_volume=42;  citation_pages=D222-30; " /><meta name="citation_reference" content="citation_title=CDD: NCBI’s conserved domain database;  citation_journal_title=Nucleic Acids Res.;  citation_year=2015;  citation_volume=43;  citation_pages=D222-D226; " /><meta name="citation_reference" content="citation_title=An efficient algorithm for large-scale detection of protein families;  citation_journal_title=Nucleic Acids Res;  citation_year=2002;  citation_volume=30;  citation_pages=1575-84; " /><meta name="citation_reference" content="citation_title=Insertion Sequence-Related Genetic-Variation In Resting Escherichia Coli K-12;  citation_journal_title=Genetics;  citation_year=1994;  citation_volume=136;  citation_pages=721-30; " /><meta name="citation_reference" content="citation_title=Stress and transposable elements: co-evolution or useful parasites?;  citation_journal_title=Heredity (Edinb);  citation_year=2000;  citation_volume=85;  citation_pages=101-6; " /><meta name="citation_reference" content="citation_title=Stress-induced activation of heterochromatic transcription;  citation_journal_title=PLoS Genet;  citation_year=2010;  citation_volume=6;  citation_pages=1–11" /><meta name="citation_reference" content="citation_title=A genetic mosaic in the fruiting stage of Armillaria gallica;  citation_journal_title=Fungal Genet. Biol;  citation_year=2000;  citation_volume=29;  citation_pages=72-80; " /><meta name="citation_reference" content="citation_title=Haploid vegetative mycelia of Armillaria gallica show among-cell-line variation for growth and phenotypic plasticity;  citation_journal_title=Mycologia;  citation_year=2005;  citation_volume=97;  citation_pages=777-87; " /><meta name="citation_reference" content="citation_title=Gene conversion disparity in yeast: its extent, multiple origins, and effects on allele frequencies;  citation_journal_title=Heredity (Edinb);  citation_year=1998;  citation_volume=80;  citation_issue=Pt 5;  citation_pages=538-52; " /><meta name="citation_reference" content="citation_title=GC-Content Evolution in Mammalian Genomes: The Biased Gene Conversion Hypothesis;  citation_journal_title=Dev. Psychobiol;  citation_year=2001;  citation_volume=39;  citation_pages=251-6; " /><meta name="citation_reference" content="citation_title=A mechanism for gene conversion in fungi;  citation_journal_title=Genet. Res;  citation_year=2007;  citation_volume=89;  citation_pages=285-307; " /><meta name="citation_reference" content="citation_title=A detailed analysis of the recombination landscape of the button mushroom Agaricus bisporus var. bisporus;  citation_journal_title=Fungal Genet. Biol;  citation_year=2016;  citation_volume=93;  citation_pages=35-45; " /><meta name="citation_reference" content="citation_publisher=Boston, MA, Springer US;  citation_title=Molecular and Cellular Mechanisms of Mutagenesis;  citation_year=1982;  citation_pages=259-64; " /><meta name="citation_reference" content="citation_title=Sexual mating of Botrytis cinerea illustrates PRP8 intein HEG activity;  citation_journal_title=Fungal Genet. Biol;  citation_year=2010;  citation_volume=47;  citation_pages=392-8; " /><meta name="citation_reference" content="citation_title=Billions of basepairs of recently expanded, repetitive sequences are eliminated from the somatic genome during copepod development;  citation_journal_title=BMC Genomics;  citation_year=2014;  citation_volume=15;  citation_pages=186" /><meta name="citation_reference" content="citation_title=Programmed DNA elimination in tetrahymena: A small RNA-mediated genome surveillance mechanism;  citation_journal_title=Adv. Exp. Med. Biol;  citation_year=2011;  citation_volume=722;  citation_pages=156-73; " /><meta name="citation_reference" content="citation_title=RNA interference in fungi: Pathways, functions, and applications;  citation_journal_title=Eukaryot. Cell;  citation_year=2011;  citation_pages=1148-55; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/24/2/103/13688955/dsw060.pdf" /><meta name="description" content="Abstract.  Helitrons constitute a superfamily of DNA transposons that were discovered in silico and are widespread in most eukaryotic genomes. They are postulat" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/24/2/103/2962881" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/24/2/103/2962881" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/24/2/103/2962881");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsw060';
    </script>
<title>
	Somatic transposition and meiotically driven elimination of an active helitron family in Pleurotus ostreatus | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./2962881" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BZJtPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJCb3Jnb2dub25lLCBBbGVzc2FuZHJhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQ2FzdGFuZXJhLCBSYcO6bCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik11Z3VlcnphLCBFbGFpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlBpc2FiYXJybywgQW50b25pbyBHLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlJhbcOtcmV6LCBMdWPDrWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IlNvbWF0aWMgdHJhbnNwb3NpdGlvbiBhbmQgbWVpb3RpY2FsbHkgZHJpdmVuIGVsaW1pbmF0aW9uIG9mIGFuIGFjdGl2ZSBoZWxpdHJvbiBmYW1pbHkgaW4gUGxldXJvdHVzIG9zdHJlYXR1cyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjEwMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iMTE1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHN3MDYwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjI0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSIyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTcvMDQvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Nb2JpbGUgZWxlbWVudHM6IGRyaXZlcnMgb2YgZ2Vub21lIGV2b2x1dGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0zMDM7ICBjaXRhdGlvbl9wYWdlcz0xNjI2LTMyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHVuaWZpZWQgY2xhc3NpZmljYXRpb24gc3lzdGVtIGZvciBldWthcnlvdGljIHRyYW5zcG9zYWJsZSBlbGVtZW50czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBSZXYuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz05NzMtODI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJvbGxpbmctY2lyY2xlIHRyYW5zcG9zb25zIGluIGV1a2FyeW90ZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVS4gUy4gQTsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT05ODsgIGNpdGF0aW9uX3BhZ2VzPTg3MTQtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VmVydGVicmF0ZSBoZWxlbnRyb25zIGFuZCBvdGhlciBub3ZlbCBIZWxpdHJvbnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbmU7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MzEzOyAgY2l0YXRpb25fcGFnZXM9MjAxLTEyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NYXNzaXZlIGFtcGxpZmljYXRpb24gb2Ygcm9sbGluZy1jaXJjbGUgdHJhbnNwb3NvbnMgaW4gdGhlIGxpbmVhZ2Ugb2YgdGhlIGJhdCBNeW90aXMgbHVjaWZ1Z3VzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFUuIFMuIEE7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTA0OyAgY2l0YXRpb25fcGFnZXM9MTg5NS05MDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBtYWtpbmcgb2YgYSBnZW5vbWljIHBhcmFzaXRlIC0gdGhlIE1vdGhyYSBmYW1pbHkgc2hlZHMgbGlnaHQgb24gdGhlIGV2b2x1dGlvbiBvZiBIZWxpdHJvbnMgaW4gcGxhbnRzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2IuIEROQTsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9MjMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IZWxpdHJvbnMgb24gYSByb2xsOiBldWthcnlvdGljIHJvbGxpbmctY2lyY2xlIHRyYW5zcG9zb25zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UcmVuZHMgR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl9wYWdlcz01MjEtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXhjaXNpb24gb2YgSGVsaXRyb24gdHJhbnNwb3NvbnMgaW4gbWFpemU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTE4MjsgIGNpdGF0aW9uX3BhZ2VzPTM5OS00MDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmUgbW92ZW1lbnQgYnkgSGVsaXRyb24gdHJhbnNwb3NvbnMgY29udHJpYnV0ZXMgdG8gdGhlIGhhcGxvdHlwZSB2YXJpYWJpbGl0eSBvZiBtYWl6ZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVLiBTLiBBOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTEwMjsgIGNpdGF0aW9uX3BhZ2VzPTkwNjgtNzM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNwb250YW5lb3VzIG11dGF0aW9ucyBjYXVzZWQgYnkgYSBIZWxpdHJvbiB0cmFuc3Bvc29uLCBIZWwtSXQxLCBpbiBtb3JuaW5nIGdsb3J5LCBJcG9tb2VhIHRyaWNvbG9yOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBKOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTQ5OyAgY2l0YXRpb25fcGFnZXM9OTI0LTM0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbnRyYXNwZWNpZmljIHZpb2xhdGlvbiBvZiBnZW5ldGljIGNvbGluZWFyaXR5IGFuZCBpdHMgaW1wbGljYXRpb25zIGluIG1haXplOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFUuIFMuIEE7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9OTk7ICBjaXRhdGlvbl9wYWdlcz05NTczLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgSGVsaXRyb24gdHJhbnNwb3NvbiByZWNvbnN0cnVjdGVkIGZyb20gYmF0cyByZXZlYWxzIGEgbm92ZWwgbWVjaGFuaXNtIG9mIGdlbm9tZSBzaHVmZmxpbmcgaW4gZXVrYXJ5b3RlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBDb21tdW47ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTEwNzE2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGVsaXRyb24gVHJhbnNwb3NvbnMgb24gdGhlIFNleCBDaHJvbW9zb21lcyBvZiB0aGUgUGxhdHlmaXNoIFhpcGhvcGhvcnVzIG1hY3VsYXR1cyBhbmQgVGhlaXIgRXZvbHV0aW9uIGluIEFuaW1hbCBHZW5vbWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1aZWJyYWZpc2g7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MzsgIGNpdGF0aW9uX3BhZ2VzPTM5LTUyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgTWFpemUgR2Vub21lIENvbnRhaW5zIGEgSGVsaXRyb24gSW5zZXJ0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBDZWxsIE9ubGluZTsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTM4MS05MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGVsaXRyb25TY2FubmVyIHVuY292ZXJzIGEgbGFyZ2Ugb3Zlcmxvb2tlZCBjYWNoZSBvZiBIZWxpdHJvbiB0cmFuc3Bvc29ucyBpbiBtYW55IHBsYW50IGdlbm9tZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwuIEFjYWQuIFNjaTsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTE7ICBjaXRhdGlvbl9wYWdlcz0xMDI2My04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5lIGR1cGxpY2F0aW9uIGFuZCBleG9uIHNodWZmbGluZyBieSBoZWxpdHJvbi1saWtlIHRyYW5zcG9zb25zIGdlbmVyYXRlIGludHJhc3BlY2llcyBkaXZlcnNpdHkgaW4gbWFpemU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9Mzc7ICBjaXRhdGlvbl9wYWdlcz05OTctMTAwMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGlnaGx5IGV4cHJlc3NlZCBjYXB0dXJlZCBnZW5lcyBhbmQgY3Jvc3Mta2luZ2RvbSBkb21haW5zIHByZXNlbnQgaW4gSGVsaXRyb25zIGNyZWF0ZSBub3ZlbCBkaXZlcnNpdHkgaW4gUGxldXJvdHVzIG9zdHJlYXR1cyBhbmQgb3RoZXIgZnVuZ2k7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTEwNzEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgdGlnaHRseSByZWd1bGF0ZWQgcHJvbW90ZXIgb2YgdGhlIHhhbkEgZ2VuZSBvZiBBc3BlcmdpbGx1cyBuaWR1bGFucyBpcyBpbmNsdWRlZCBpbiBhIGhlbGl0cm9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIE1pY3JvYmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT02MzsgIGNpdGF0aW9uX3BhZ2VzPTE1NzctODc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVDDqXJpZ29yZCBibGFjayB0cnVmZmxlIGdlbm9tZSB1bmNvdmVycyBldm9sdXRpb25hcnkgb3JpZ2lucyBhbmQgbWVjaGFuaXNtcyBvZiBzeW1iaW9zaXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT00NjQ7ICBjaXRhdGlvbl9wYWdlcz0xMDMzLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNoYXJhY3Rlcml6YXRpb24gb2YgdHJhbnNwb3NhYmxlIGVsZW1lbnRzIGluIHRoZSBlY3RvbXljb3JyaGl6YWwgZnVuZ3VzIExhY2NhcmlhIGJpY29sb3I7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgT25lOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVF1YW50aXRhdGl2ZSBsaW5rYWdlIG1hcHBpbmcgb2YgbGlnbmluLWRlZ3JhZGluZyBlbnp5bWF0aWMgYWN0aXZpdGllcyBpbiBQbGV1cm90dXMgb3N0cmVhdHVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Fbnp5bWUgTWljcm9iLiBUZWNobm9sOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTQzOyAgY2l0YXRpb25fcGFnZXM9MTM3LTQzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZ2VuZXRpY3MgYW5kIGN1bHRpdmF0aW9uIG9mIFBsZXVyb3R1cyBvc3RyZWF0dXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU15Y29sb2dpYTsgIGNpdGF0aW9uX3llYXI9MTk2ODsgIGNpdGF0aW9uX3ZvbHVtZT02MDsgIGNpdGF0aW9uX3BhZ2VzPTYyNyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUN1bHRpdmF0aW9uIG9mIFBsZXVyb3R1cyBvc3RyZWF0dXMgYW5kIG90aGVyIGVkaWJsZSBtdXNocm9vbXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFwcGwuIE1pY3JvYmlvbC4gQmlvdGVjaG5vbDsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT04NTsgIGNpdGF0aW9uX3BhZ2VzPTEzMjEtMzc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1vbGVjdWxhciBrYXJ5b3R5cGUgb2YgdGhlIHdoaXRlIHJvdCBmdW5ndXMgUGxldXJvdHVzIG9zdHJlYXR1czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QXBwbC4gRW52aXIuIE1pY3JvYmlvbDsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT02NTsgIGNpdGF0aW9uX3BhZ2VzPTM0MTMtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpYyBsaW5rYWdlIG1hcCBvZiB0aGUgZWRpYmxlIGJhc2lkaW9teWNldGUgUGxldXJvdHVzIG9zdHJlYXR1czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QXBwbC4gRW52aXJvbi4gTWljcm9iaW9sOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTY2OyAgY2l0YXRpb25fcGFnZXM9NTI5MC0zMDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmV0aWMgbGlua2FnZSBtYXAgYW5kIGV4cHJlc3Npb24gYW5hbHlzaXMgb2YgZ2VuZXMgZXhwcmVzc2VkIGluIHRoZSBsYW1lbGxhZSBvZiB0aGUgZWRpYmxlIGJhc2lkaW9teWNldGUgUGxldXJvdHVzIG9zdHJlYXR1czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RnVuZ2FsIEdlbmV0LiBCaW9sLiBGRyBCOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTQzOyAgY2l0YXRpb25fcGFnZXM9Mzc2LTg3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc3Bvc2FibGUgZWxlbWVudHMgdmVyc3VzIHRoZSBmdW5nYWwgZ2Vub21lOiBpbXBhY3Qgb24gd2hvbGUtZ2Vub21lIGFyY2hpdGVjdHVyZSBhbmQgdHJhbnNjcmlwdGlvbmFsIHByb2ZpbGVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9ZTEwMDYxMDgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NYXBwaW5nIG9mIGdlbm9taWMgcmVnaW9ucyAocXVhbnRpdGF0aXZlIHRyYWl0IGxvY2kpIGNvbnRyb2xsaW5nIHByb2R1Y3Rpb24gYW5kIHF1YWxpdHkgaW4gaW5kdXN0cmlhbCBjdWx0dXJlcyBvZiB0aGUgZWRpYmxlIGJhc2lkaW9teWNldGUgUGxldXJvdHVzIG9zdHJlYXR1czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QXBwbC4gRW52aXJvbi4gTWljcm9iaW9sOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTY5OyAgY2l0YXRpb25fcGFnZXM9MzYxNy0yNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXh0ZW5zaXZlIHNhbXBsaW5nIG9mIGJhc2lkaW9teWNldGUgZ2Vub21lcyBkZW1vbnN0cmF0ZXMgaW5hZGVxdWFjeSBvZiB0aGUgd2hpdGUtcm90L2Jyb3duLXJvdCBwYXJhZGlnbSBmb3Igd29vZCBkZWNheSBmdW5naTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVLiBTLiBBOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTExMTsgIGNpdGF0aW9uX3BhZ2VzPTk5MjMtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyYXRpdmUgYW5kIHRyYW5zY3JpcHRpb25hbCBhbmFseXNpcyBvZiB0aGUgcHJlZGljdGVkIHNlY3JldG9tZSBpbiB0aGUgbGlnbm9jZWxsdWxvc2UtZGVncmFkaW5nIGJhc2lkaW9teWNldGUgZnVuZ3VzIFBsZXVyb3R1cyBvc3RyZWF0dXMuIEVudmlyb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1pY3JvYmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJpbWVyMy1uZXcgY2FwYWJpbGl0aWVzIGFuZCBpbnRlcmZhY2VzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT00MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QmFzaWMgbG9jYWwgYWxpZ25tZW50IHNlYXJjaCB0b29sOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBNb2wuIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTE5OTA7ICBjaXRhdGlvbl92b2x1bWU9MjE1OyAgY2l0YXRpb25fcGFnZXM9NDAzLTEwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIG5ldyBtYXRoZW1hdGljYWwgbW9kZWwgZm9yIHJlbGF0aXZlIHF1YW50aWZpY2F0aW9uIGluIHJlYWwtdGltZSBSVC1QQ1I7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTI5OyAgY2l0YXRpb25fcGFnZXM9ZTQ1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWNjZWxlcmF0ZWQgUHJvZmlsZSBITU0gU2VhcmNoZXMgUGVhcnNvbiwgVy4gUi4sIChlZC4pLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBDb21wdXQuIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPWUxMDAyMTk1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGZhbTogdGhlIHByb3RlaW4gZmFtaWxpZXMgZGF0YWJhc2U7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTQyOyAgY2l0YXRpb25fcGFnZXM9RDIyMi0zMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q0REOiBOQ0JJ4oCZcyBjb25zZXJ2ZWQgZG9tYWluIGRhdGFiYXNlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9NDM7ICBjaXRhdGlvbl9wYWdlcz1EMjIyLUQyMjY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuIGVmZmljaWVudCBhbGdvcml0aG0gZm9yIGxhcmdlLXNjYWxlIGRldGVjdGlvbiBvZiBwcm90ZWluIGZhbWlsaWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTE1NzUtODQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUluc2VydGlvbiBTZXF1ZW5jZS1SZWxhdGVkIEdlbmV0aWMtVmFyaWF0aW9uIEluIFJlc3RpbmcgRXNjaGVyaWNoaWEgQ29saSBLLTEyOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MTk5NDsgIGNpdGF0aW9uX3ZvbHVtZT0xMzY7ICBjaXRhdGlvbl9wYWdlcz03MjEtMzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN0cmVzcyBhbmQgdHJhbnNwb3NhYmxlIGVsZW1lbnRzOiBjby1ldm9sdXRpb24gb3IgdXNlZnVsIHBhcmFzaXRlcz87ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUhlcmVkaXR5IChFZGluYik7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9ODU7ICBjaXRhdGlvbl9wYWdlcz0xMDEtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3RyZXNzLWluZHVjZWQgYWN0aXZhdGlvbiBvZiBoZXRlcm9jaHJvbWF0aWMgdHJhbnNjcmlwdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9MeKAkzExIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBnZW5ldGljIG1vc2FpYyBpbiB0aGUgZnJ1aXRpbmcgc3RhZ2Ugb2YgQXJtaWxsYXJpYSBnYWxsaWNhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1GdW5nYWwgR2VuZXQuIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9Mjk7ICBjaXRhdGlvbl9wYWdlcz03Mi04MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGFwbG9pZCB2ZWdldGF0aXZlIG15Y2VsaWEgb2YgQXJtaWxsYXJpYSBnYWxsaWNhIHNob3cgYW1vbmctY2VsbC1saW5lIHZhcmlhdGlvbiBmb3IgZ3Jvd3RoIGFuZCBwaGVub3R5cGljIHBsYXN0aWNpdHk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU15Y29sb2dpYTsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT05NzsgIGNpdGF0aW9uX3BhZ2VzPTc3Ny04NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZSBjb252ZXJzaW9uIGRpc3Bhcml0eSBpbiB5ZWFzdDogaXRzIGV4dGVudCwgbXVsdGlwbGUgb3JpZ2lucywgYW5kIGVmZmVjdHMgb24gYWxsZWxlIGZyZXF1ZW5jaWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1IZXJlZGl0eSAoRWRpbmIpOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTgwOyAgY2l0YXRpb25faXNzdWU9UHQgNTsgIGNpdGF0aW9uX3BhZ2VzPTUzOC01MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R0MtQ29udGVudCBFdm9sdXRpb24gaW4gTWFtbWFsaWFuIEdlbm9tZXM6IFRoZSBCaWFzZWQgR2VuZSBDb252ZXJzaW9uIEh5cG90aGVzaXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPURldi4gUHN5Y2hvYmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0zOTsgIGNpdGF0aW9uX3BhZ2VzPTI1MS02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIG1lY2hhbmlzbSBmb3IgZ2VuZSBjb252ZXJzaW9uIGluIGZ1bmdpOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5ldC4gUmVzOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTg5OyAgY2l0YXRpb25fcGFnZXM9Mjg1LTMwNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBkZXRhaWxlZCBhbmFseXNpcyBvZiB0aGUgcmVjb21iaW5hdGlvbiBsYW5kc2NhcGUgb2YgdGhlIGJ1dHRvbiBtdXNocm9vbSBBZ2FyaWN1cyBiaXNwb3J1cyB2YXIuIGJpc3BvcnVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1GdW5nYWwgR2VuZXQuIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9OTM7ICBjaXRhdGlvbl9wYWdlcz0zNS00NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fcHVibGlzaGVyPUJvc3RvbiwgTUEsIFNwcmluZ2VyIFVTOyAgY2l0YXRpb25fdGl0bGU9TW9sZWN1bGFyIGFuZCBDZWxsdWxhciBNZWNoYW5pc21zIG9mIE11dGFnZW5lc2lzOyAgY2l0YXRpb25feWVhcj0xOTgyOyAgY2l0YXRpb25fcGFnZXM9MjU5LTY0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZXh1YWwgbWF0aW5nIG9mIEJvdHJ5dGlzIGNpbmVyZWEgaWxsdXN0cmF0ZXMgUFJQOCBpbnRlaW4gSEVHIGFjdGl2aXR5OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1GdW5nYWwgR2VuZXQuIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NDc7ICBjaXRhdGlvbl9wYWdlcz0zOTItODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QmlsbGlvbnMgb2YgYmFzZXBhaXJzIG9mIHJlY2VudGx5IGV4cGFuZGVkLCByZXBldGl0aXZlIHNlcXVlbmNlcyBhcmUgZWxpbWluYXRlZCBmcm9tIHRoZSBzb21hdGljIGdlbm9tZSBkdXJpbmcgY29wZXBvZCBkZXZlbG9wbWVudDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTE1OyAgY2l0YXRpb25fcGFnZXM9MTg2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJvZ3JhbW1lZCBETkEgZWxpbWluYXRpb24gaW4gdGV0cmFoeW1lbmE6IEEgc21hbGwgUk5BLW1lZGlhdGVkIGdlbm9tZSBzdXJ2ZWlsbGFuY2UgbWVjaGFuaXNtOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1BZHYuIEV4cC4gTWVkLiBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTcyMjsgIGNpdGF0aW9uX3BhZ2VzPTE1Ni03MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Uk5BIGludGVyZmVyZW5jZSBpbiBmdW5naTogUGF0aHdheXMsIGZ1bmN0aW9ucywgYW5kIGFwcGxpY2F0aW9uczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RXVrYXJ5b3QuIENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl9wYWdlcz0xMTQ4LTU1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMjQvMi8xMDMvMTM2ODg5NTUvZHN3MDYwLnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBIZWxpdHJvbnMgY29uc3RpdHV0ZSBhIHN1cGVyZmFtaWx5IG9mIEROQSB0cmFuc3Bvc29ucyB0aGF0IHdlcmUgZGlzY292ZXJlZCBpbiBzaWxpY28gYW5kIGFyZSB3aWRlc3ByZWFkIGluIG1vc3QgZXVrYXJ5b3RpYyBnZW5vbWVzLiBUaGV5IGFyZSBwb3N0dWxhdCIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXMvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9Mjk2Mjg4MSZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGR7s4SMGbHqVs1g5q3HDexQqr/FLQ==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f2%2f103%2f2962881" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f2%2f103%2f2962881" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=97104" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/24/2"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/2/2/m_dnares_24_2cover.png?Expires=2147483647&amp;Signature=C8sLyIyfepNN-Oy5BR2xLoVJLx2W1jQBt~48a2lQ~mjys3EWTAKP5k2kV~t7b8sG~Qk1B4iVdz8BINFP5bR-Hc0gvwqekGHM8qavh8kDaLAfN~qNxrDj7NNeLHJ-IURMi5v1ATu23u5wB01wP27BHknbA3W9JgGnq6AVGPqeLfhK7AJcTzOYzKpGYwaedh5crkYXYHp~9GCcHuTjoInf3sKaYC2Ew-HnuA-yIL45j8ocXrNQuIxJV8rFXwx1Tt0b7I1ZJg09R4-7m-GZB9vPgdehgAXUMUNQ4gDatBY22xUt9COqnlb7RZKNUXFDo7Qe0LNJgSHwNUOJvsh-SLpXkg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/24/2">
                        <div class="volume trailing-comma">Volume 24</div>
                        <div class="issue">Issue 2</div>

            </a>

        </div>
        <div class="ii-pub-date">
April 2017        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="78055499">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#78055499">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="78055501">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#78055501">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="78055503">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#78055503">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="78055516">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#78055516">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="78055531">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#78055531">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="78055533">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#78055533">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="78055535">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#78055535">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="78055541">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#78055541">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="empty arrow">&nbsp;</li>
            <li class="next arrow">
            <a href="/dnaresearch/article/24/2/117/2949521">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
<span class="article-flag"><a href="https://academic.oup.com/dnaresearch/search-results?f_OUPSeries=Editor's+Choice">Editor's Choice</a></span>            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Somatic transposition and meiotically driven elimination of an active helitron family in <em>Pleurotus ostreatus</em>
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Alessandra Borgognone</div>
    </div>






        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Alessandra+Borgognone">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Borgognone A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Borgognone A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Alessandra Borgognone</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ra&#250;l Castanera</div>
    </div>






        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ra%c3%bal+Castanera">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Castanera R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Castanera R&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Raúl Castanera</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Elaia Muguerza</div>
    </div>






        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Elaia+Muguerza">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Muguerza E">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Muguerza E&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Elaia Muguerza</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Antonio G. Pisabarro</div>
    </div>






        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Antonio+G.+Pisabarro">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Pisabarro A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Pisabarro A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Antonio G. Pisabarro</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Luc&#237;a Ram&#237;rez</div>
    </div>


                <div class="info-card-note">
                    <a reveal-id="dsw060-cor1" data-open="dsw060-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +34 948169130. Fax. +34 948169732. Email: <a href="mailto:lramirez@unavarra.es" target="_blank">lramirez@unavarra.es</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Luc%c3%ada+Ram%c3%adrez">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ram&#237;rez L">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ram&#237;rez L&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Lucía Ramírez</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsw060-FM1"><div class="fn"><div class="footnote-content">Edited by Prof. Takashi Ito</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 24, Issue 2, 1 April 2017, Pages 103–115, <a href='https://doi.org/10.1093/dnares/dsw060'>https://doi.org/10.1093/dnares/dsw060</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">29 January 2017</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">21 September 2016</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">14 December 2016</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="2962881" href="/dnaresearch/article-pdf/24/2/103/13688955/dsw060.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Alessandra Borgognone, Raúl Castanera, Elaia Muguerza, Antonio G. Pisabarro, Lucía Ramírez;  Somatic transposition and meiotically driven elimination of an active helitron family in <em>Pleurotus ostreatus</em>, <em>DNA Research</em>, Volume 24, Issue 2, 1 April 2017, Pages 103–115, <a href='https://doi.org/10.1093/dnares/dsw060'>https://doi.org/10.1093/dnares/dsw060</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/2962881?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/2962881?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/2962881?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/2962881?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/2962881?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/2962881?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/2962881?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Somatic%20transposition%20and%20meiotically%20driven%20elimination%20of%20an%20active%20helitron%20family%20in%20%26lt%3Bem%26gt%3BPleurotus%20ostreatus%26lt%3B%2Fem%26gt%3B&amp;publicationDate=2017-01-29&amp;volumeNum=24&amp;issueNum=2&amp;author=Borgognone%2C%20Alessandra%3B%20Castanera%2C%20Ra%C3%BAl&amp;startPage=103&amp;endPage=115&amp;contentId=10.1093%2Fdnares%2Fdsw060&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f2%2f103%2f2962881" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=78055499 id="78055499" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Helitrons constitute a superfamily of DNA transposons that were discovered <em>in silico</em> and are widespread in most eukaryotic genomes. They are postulated to mobilize through a “rolling-circle” mechanism, but the experimental evidence of their transposition has been described only recently. Here, we present the inheritance patterns of HELPO1 and HELPO2 helitron families in meiotically derived progeny of the basidiomycete <em>Pleurotus ostreatus</em>. We found distorted segregation patterns of HELPO2 helitrons that led to a strong under-representation of these elements in the progeny. Further investigation of HELPO2 flanking sites showed that gene conversion may contribute to the elimination of such repetitive elements in meiosis, favouring the presence of HELPO2 vacant loci. In addition, the analysis of HELPO2 content in a reconstructed pedigree of subclones maintained under different culture conditions revealed an event of helitron somatic transposition. Additional analyses of genome and transcriptome data indicated that <em>P. ostreatus</em> carries active RNAi machinery that could be involved in the control of transposable element proliferation. Our results provide the first evidence of helitron mobilization in the fungal kingdom and highlight the interaction between genome defence mechanisms and invasive DNA.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="helitrons">helitrons</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;segregation distortion&quot;">segregation distortion</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;gene conversion&quot;">gene conversion</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;transposable elements&quot;">transposable elements</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="basidiomycete">basidiomycete</a></div>                            <h2 scrollto-destination=78055501 id="78055501" class="section-title" data-legacy-id=SEC1>1. Introduction</h2>
<p>Transposable elements (TEs) constitute an important fraction of the eukaryotic and prokaryotic genomes. These DNA fragments are able to mobilize in the host genome, contributing to chromosomal rearrangements and transcriptional modulation.<a reveal-id="dsw060-B1" data-open="dsw060-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> On the basis of their transposition mechanisms, TEs can be grouped into two classes: Class I TEs transpose by a copy-and-paste mechanism <em>via</em> RNA intermediates, and Class II TEs proliferate via DNA intermediates using cut-and-paste transposition or rolling-circle replication.<a reveal-id="dsw060-B2" data-open="dsw060-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a><sup>,</sup><a reveal-id="dsw060-B3" data-open="dsw060-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> The vast majority of TEs produce target site duplications (TSDs) at their insertion sites, with some exceptions as Helitrons, a subclass of DNA transposons discovered in 2001 by computational analysis in <em>Arabidopsis thaliana</em>, <em>Oryza sativa</em>, and <em>Caenorhabditis elegans</em> genomes.<a reveal-id="dsw060-B3" data-open="dsw060-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> Helitrons are present in a wide range of eukaryotic genomes<a reveal-id="dsw060-B3 dsw060-B4 dsw060-B5 dsw060-B6" data-open="dsw060-B3 dsw060-B4 dsw060-B5 dsw060-B6" class="link link-ref link-reveal xref-bibr"><sup>3–6</sup></a> and show several structural and enzymatic features that differentiate them from the rest of the TEs. They do not display terminal inverted repeats as other DNA transposons do, do not generate TSDs at the insertion site, and carry conserved 5’-TC and CTRR-3’ ends and a 16–bp palindromic hairpin located approximately 10–20 nucleotides upstream of the 3’ terminus.<a reveal-id="dsw060-B3" data-open="dsw060-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a><sup>,</sup><a reveal-id="dsw060-B7" data-open="dsw060-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> Additionally, putative autonomous helitrons encode a RepHel protein containing a replication initiation (Rep) and a helicase (Hel) domain. The conservation of the Rep catalytic motifs with the replication initiators of plasmids and ssDNA viruses led to the hypothesis that these elements transpose with a rolling-circle mechanism (RC), which is consistent with the absence of TSDs. Nevertheless, a study carried out in maize haplotypes identified helitron somatic excision events based on the presence of footprints in polymorphic helitron loci (occupied <em>vs</em> vacant loci occurrence in the genome),<a reveal-id="dsw060-B8" data-open="dsw060-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> suggesting that this TE subclass may show both excision and replication based transposition mechanisms. Similar to other transposons, it has been described that helitrons can promote rearrangements in plant genomes<a reveal-id="dsw060-B9" data-open="dsw060-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a><sup>,</sup><a reveal-id="dsw060-B10" data-open="dsw060-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> and generate intra-specific variability by breaking the genetic co-linearity among haplotypes.<a reveal-id="dsw060-B9 dsw060-B10 dsw060-B11" data-open="dsw060-B9 dsw060-B10 dsw060-B11" class="link link-ref link-reveal xref-bibr"><sup>9–11</sup></a> In addition, during the transposition process, helitrons can capture and disperse gene fragments, producing chimeric transcripts.<a reveal-id="dsw060-B9" data-open="dsw060-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> After their discovery, <em>in silico</em> studies have been carried out to characterize the nature of these eukaryotic DNA TEs. Despite several helitron-related finding being described and explained, the mechanism of helitron mobilization remained unclear because, until very recently, there was no experimental evidence for their transposition. Interestingly, a novel study reports the first experimental evidence of helitron insertions in cell culture, providing insights into the transposition and gene capture mechanisms.<a reveal-id="dsw060-B12" data-open="dsw060-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> In addition, the finding of circular DNA intermediates containing head-to-tail junctions of helitron ends strongly supports the originally proposed RC transposition mechanism.<a reveal-id="dsw060-B5" data-open="dsw060-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a><sup>,</sup><a reveal-id="dsw060-B12" data-open="dsw060-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> Previous bioinformatics analyses have reported that helitrons account for 0-3% of the mammalian genome size<a reveal-id="dsw060-B5" data-open="dsw060-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a><sup>,</sup><a reveal-id="dsw060-B13" data-open="dsw060-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> and 1–5% of the genome size in insects.<a reveal-id="dsw060-B4" data-open="dsw060-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a><sup>,</sup><a reveal-id="dsw060-B7" data-open="dsw060-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> In plants, these DNA transposons constitute a variable portion, contributing between 0.01 and 6% among different species.<a reveal-id="dsw060-B3" data-open="dsw060-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a><sup>,</sup><a reveal-id="dsw060-B14 dsw060-B15 dsw060-B16" data-open="dsw060-B14 dsw060-B15 dsw060-B16" class="link link-ref link-reveal xref-bibr"><sup>14–16</sup></a> In fungi, the presence of helitrons has been reported often as part of <em>in silico</em> comprehensive TE annotations of several ascomycetes and basidiomycetes sequenced genomes.<a reveal-id="dsw060-B7" data-open="dsw060-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a><sup>,</sup><a reveal-id="dsw060-B17 dsw060-B18 dsw060-B19 dsw060-B20 dsw060-B21" data-open="dsw060-B17 dsw060-B18 dsw060-B19 dsw060-B20 dsw060-B21" class="link link-ref link-reveal xref-bibr"><sup>17–21</sup></a> In addition, a study in <em>Aspergillus nidulans</em> described the capture and duplication of a gene promoter during the generation of a non-autonomous helitron<em>.</em><a reveal-id="dsw060-B18" data-open="dsw060-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> In <em>P. ostreatus</em>, a recent study carried out in our group identified and characterized two helitron families, HELPO1 and HELPO2, which showed differential patterns of expression and distribution. <em>P. ostreatus</em> has been widely studied due to its agronomic importance and also due to its ability to selectively degrade lignin.<a reveal-id="dsw060-B22" data-open="dsw060-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a><sup>,</sup><a reveal-id="dsw060-B23" data-open="dsw060-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> The simplicity of its life cycle and its ability to grow and fructify under laboratory conditions make it an interesting model for fungal genetics. The <em>P. ostreatus</em> life cycle alternates between monokaryotic and dikaryotic mycelial phases containing haploid and diploid nuclei. Two compatible monokaryons can fuse and generate a dikaryon (the sexually competent form) in which the two parental nuclei remain separated during vegetative growth and fruit body development. The karyogamy (diploid condition) occurs only in the basidia at the end of the life cycle, immediately before the meiotic division that produces four haploid basidiospores. The two nuclei harboured in the <em>P. ostreatus</em> dikaryotic strain N001 were isolated by a de-dikaryotization process based on the protoplast obtained by an enzymatic treatment of the mycelium of the dikaryotic strain N001 and the subsequent identification of the monokaryons derived from protoplasts containing either of the two nuclei.<a reveal-id="dsw060-B24" data-open="dsw060-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> This process yielded two compatible protoclones (monokaryons PC15 and PC9) bearing each one of the non-recombined haploid nuclei. The availability of these strains, as well as a meiotic progeny derived from the N001 dikaryotic strain, enabled further genetic studies of <em>P. ostreatus</em>. Specifically, the molecular karyotype determined the presence of 11 chromosomes<a reveal-id="dsw060-B24" data-open="dsw060-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> that fit with 11 linkage groups described by our group in the genetic linkage map of <em>P.</em><em>ostreatus</em>.<a reveal-id="dsw060-B25" data-open="dsw060-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a><sup>,</sup><a reveal-id="dsw060-B26" data-open="dsw060-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> These studies supported the subsequent genome sequencing of PC9 and PC15 protoclones. Both genomes showed an overall conserved macrosynteny, but the sequencing and annotation unraveled remarkable genomic differences in regions corresponding to TEs. In a recent study, 80 TE families were identified accounting for 2.5 and 6.2% of PC9 and PC15 genome sizes, respectively. The results of this work showed that the <em>P. ostreatus</em> genome is mainly populated by LTR-retrotransposons. Nevertheless, helitrons were the most abundant DNA transposons in the genome, and the HELPO1 family was among the most expressed TE families. <em>P. ostreatus</em> helitrons carry most canonical features such as the 3' subterminal hairpin and the RepHel helicase. Nevertheless, in contrast to plants and animal helitrons, their encoded helicase does not carry RPA or zinc fingers domains.<a reveal-id="dsw060-B17" data-open="dsw060-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a><sup>,</sup><a reveal-id="dsw060-B27" data-open="dsw060-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> They represent up to 0.35% of the <em>P. ostreatus</em> genome and are frequently displayed in polymorphic loci of PC15 and PC9. In the present study, we used experimental approaches to uncover the differential amplification dynamics of helitrons in <em>P. ostreatus</em> subclones maintained on solid culture under high- and low-subculture frequencies. In addition, we analysed the pattern of inheritance of helitron TEs in the offspring of 68 monokaryotic strains meiotically obtained from the dikaryotic strain N001. We provide evidence of somatic HELPO2 transposition in one of the analysed subclones and show that the HELPO2 family is under-represented in the progeny. A detailed analysis of two HELPO2 loci suggests that helitrons could be eliminated during meiosis by gene conversion.</p>                            <h2 scrollto-destination=78055503 id="78055503" class="section-title" >2. Materials and methods</h2>
                            <h3 scrollto-destination=78055504 id="78055504" class="section-title" >2.1. Origins of the <em>P. ostreatus</em> monokaryotic and dikaryotic strains</h3>
<p>A total of 74 strains and sub-clones of <em>P. ostreatus</em> with different nuclear allelic compositions were used in this study. All of them were derived from N001, a dikaryotic strain used by our group as a model for mushroom breeding, genetics and genomics since 1994.<a reveal-id="dsw060-B28" data-open="dsw060-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> The N001 strain was de-dikaryotized in 1999, and the two corresponding haploid-monokaryotic protoclones were recovered and identified as PC9 and PC15.<a reveal-id="dsw060-B24" data-open="dsw060-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> Both monokaryotic strains have been sequenced by the Joint Genome Institute, and their genome assemblies and annotations are publicly available in the Mycocosm Database.<a reveal-id="dsw060-B27" data-open="dsw060-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a><sup>,</sup><a reveal-id="dsw060-B29" data-open="dsw060-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a><sup>,</sup><a reveal-id="dsw060-B30" data-open="dsw060-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> For each of the three strains, a subclone was deposited in the Spanish Type Culture Collection (CECT) on the dates indicated in <a reveal-id="dsw060-T1" data-open="dsw060-T1" class="link link-reveal link-table xref-fig">Table 1</a> and maintained under low subculture frequency (N001-03, PC9-99 and PC15-99, two subcultures per year); another subclone was kept under high subculture frequency (approximately 8 subcultures per year) for routine laboratory workup to the present day (N001-14, PC9-14 and PC15-14). In addition, we used a haploid-monokaryotic progeny of 68 monokaryotic strains meiotically derived from N001 in 1994 (68 mK; <a reveal-id="dsw060-T1" data-open="dsw060-T1" class="link link-reveal link-table xref-fig">Table 1</a>). This collection has been stored in our laboratory for 20 years under low subculture frequency. <div content-id="dsw060-T1" class="table-modal"><div class="table-wrap"><div id="dsw060-T1" data-id="dsw060-T1" class="table-wrap-title"><div class="label -label">Table 1</div><div class="caption"><p>Description of <em>P. ostreatus</em> strains and subclones used in this study</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Subclone </th><th>Strain </th><th>Nuclear condition (ploidy) </th><th>Subculture regime (frequency) </th><th>Use/storage </th><th>Culture period </th></tr></thead><tbody><tr><td>N001-03 </td><td>N001 </td><td>Dikaryon (n+n) </td><td>High/Low<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td>Culture Collection (CECT20600) </td><td>1994–2003/2003–2014 </td></tr><tr><td>N001–14 </td><td>N001 </td><td>Dikaryon (n+n) </td><td>High (8/year) </td><td>Laboratory work </td><td>1994–2014 </td></tr><tr><td>PC9-99 </td><td>PC9 </td><td>Monokaryon (n) </td><td>Low (2/year) </td><td>Culture Collection (CECT20311) </td><td>1999–2014 </td></tr><tr><td>PC9-14 </td><td>PC9 </td><td>Monokaryon (n) </td><td>High (8/year) </td><td>Laboratory work </td><td>1999–2014 </td></tr><tr><td>PC15-99 </td><td>PC15 </td><td>Monokaryon (n) </td><td>Low (2/year) </td><td>Culture Collection (CECT20312) </td><td>1999–2014 </td></tr><tr><td>PC15-14 </td><td>PC15 </td><td>Monokaryon (n) </td><td>High (8/year) </td><td>Laboratory work </td><td>1999–2014 </td></tr><tr><td>68 mK<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td>N001 – single spore isolates </td><td>Monokaryon (n) </td><td>Low (2/year) </td><td>Culture Collection (Laboratory) </td><td>1994–2014 </td></tr></tbody></table></div><div content-id="dsw060-T1" class="table-modal"><table><thead name="thead" align=""><tr><th>Subclone </th><th>Strain </th><th>Nuclear condition (ploidy) </th><th>Subculture regime (frequency) </th><th>Use/storage </th><th>Culture period </th></tr></thead><tbody><tr><td>N001-03 </td><td>N001 </td><td>Dikaryon (n+n) </td><td>High/Low<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td>Culture Collection (CECT20600) </td><td>1994–2003/2003–2014 </td></tr><tr><td>N001–14 </td><td>N001 </td><td>Dikaryon (n+n) </td><td>High (8/year) </td><td>Laboratory work </td><td>1994–2014 </td></tr><tr><td>PC9-99 </td><td>PC9 </td><td>Monokaryon (n) </td><td>Low (2/year) </td><td>Culture Collection (CECT20311) </td><td>1999–2014 </td></tr><tr><td>PC9-14 </td><td>PC9 </td><td>Monokaryon (n) </td><td>High (8/year) </td><td>Laboratory work </td><td>1999–2014 </td></tr><tr><td>PC15-99 </td><td>PC15 </td><td>Monokaryon (n) </td><td>Low (2/year) </td><td>Culture Collection (CECT20312) </td><td>1999–2014 </td></tr><tr><td>PC15-14 </td><td>PC15 </td><td>Monokaryon (n) </td><td>High (8/year) </td><td>Laboratory work </td><td>1999–2014 </td></tr><tr><td>68 mK<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td>N001 – single spore isolates </td><td>Monokaryon (n) </td><td>Low (2/year) </td><td>Culture Collection (Laboratory) </td><td>1994–2014 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p><sup>a</sup>This subclone was maintained under high subculture frequency from 1994 to 2003. Then, it was deposited in the CECT and maintained under low subculture frequency on solid medium until 2014.</p></div></div><div content-id="tblfn2"><div class="fn"><p><sup>b</sup>Collection of 68 mK strains.</p></div></div></div></div></div><div class="table-wrap"><div id="dsw060-T1" data-id="dsw060-T1" class="table-wrap-title"><div class="label -label">Table 1</div><div class="caption"><p>Description of <em>P. ostreatus</em> strains and subclones used in this study</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Subclone </th><th>Strain </th><th>Nuclear condition (ploidy) </th><th>Subculture regime (frequency) </th><th>Use/storage </th><th>Culture period </th></tr></thead><tbody><tr><td>N001-03 </td><td>N001 </td><td>Dikaryon (n+n) </td><td>High/Low<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td>Culture Collection (CECT20600) </td><td>1994–2003/2003–2014 </td></tr><tr><td>N001–14 </td><td>N001 </td><td>Dikaryon (n+n) </td><td>High (8/year) </td><td>Laboratory work </td><td>1994–2014 </td></tr><tr><td>PC9-99 </td><td>PC9 </td><td>Monokaryon (n) </td><td>Low (2/year) </td><td>Culture Collection (CECT20311) </td><td>1999–2014 </td></tr><tr><td>PC9-14 </td><td>PC9 </td><td>Monokaryon (n) </td><td>High (8/year) </td><td>Laboratory work </td><td>1999–2014 </td></tr><tr><td>PC15-99 </td><td>PC15 </td><td>Monokaryon (n) </td><td>Low (2/year) </td><td>Culture Collection (CECT20312) </td><td>1999–2014 </td></tr><tr><td>PC15-14 </td><td>PC15 </td><td>Monokaryon (n) </td><td>High (8/year) </td><td>Laboratory work </td><td>1999–2014 </td></tr><tr><td>68 mK<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td>N001 – single spore isolates </td><td>Monokaryon (n) </td><td>Low (2/year) </td><td>Culture Collection (Laboratory) </td><td>1994–2014 </td></tr></tbody></table></div><div content-id="dsw060-T1" class="table-modal"><table><thead name="thead" align=""><tr><th>Subclone </th><th>Strain </th><th>Nuclear condition (ploidy) </th><th>Subculture regime (frequency) </th><th>Use/storage </th><th>Culture period </th></tr></thead><tbody><tr><td>N001-03 </td><td>N001 </td><td>Dikaryon (n+n) </td><td>High/Low<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td>Culture Collection (CECT20600) </td><td>1994–2003/2003–2014 </td></tr><tr><td>N001–14 </td><td>N001 </td><td>Dikaryon (n+n) </td><td>High (8/year) </td><td>Laboratory work </td><td>1994–2014 </td></tr><tr><td>PC9-99 </td><td>PC9 </td><td>Monokaryon (n) </td><td>Low (2/year) </td><td>Culture Collection (CECT20311) </td><td>1999–2014 </td></tr><tr><td>PC9-14 </td><td>PC9 </td><td>Monokaryon (n) </td><td>High (8/year) </td><td>Laboratory work </td><td>1999–2014 </td></tr><tr><td>PC15-99 </td><td>PC15 </td><td>Monokaryon (n) </td><td>Low (2/year) </td><td>Culture Collection (CECT20312) </td><td>1999–2014 </td></tr><tr><td>PC15-14 </td><td>PC15 </td><td>Monokaryon (n) </td><td>High (8/year) </td><td>Laboratory work </td><td>1999–2014 </td></tr><tr><td>68 mK<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td>N001 – single spore isolates </td><td>Monokaryon (n) </td><td>Low (2/year) </td><td>Culture Collection (Laboratory) </td><td>1994–2014 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p><sup>a</sup>This subclone was maintained under high subculture frequency from 1994 to 2003. Then, it was deposited in the CECT and maintained under low subculture frequency on solid medium until 2014.</p></div></div><div content-id="tblfn2"><div class="fn"><p><sup>b</sup>Collection of 68 mK strains.</p></div></div></div></div></p>                            <h3 scrollto-destination=78055506 id="78055506" class="section-title" >2.2. Culture conditions and nucleic acid extraction</h3>
<p>All of the strains and subclones used in this work were cultured in liquid SMY submerged fermentation (10 g of sucrose, 10 g of malt extract, 4 g of yeast extract, 1 l H<sub>2</sub>O [pH 5.6]) in the dark at 24 °C under orbital shaking (130 rpm). The cultures were kept for eight growing days, and the mycelia were collected using vacuum filtration, ground in a sterile mortar in the presence of liquid nitrogen and stored at −80°C. Genomic DNA extractions were carried out using an E.Z.N.A. Fungal DNA Mini Kit (Omega Bio-Tek, Norcross, GA), following the manufacturer’s instructions. After additional incubation with 4 μl of RNAse A (10 mg/ml) for 30 minutes at 37°C, the DNA solutions were treated twice with phenol-chloroform (3:1), and the pellet was resuspended in 40 μl of nuclease-free water. DNA concentrations were determined using a Qubit 2.0 fluorometer (Life Technology, Carlsbad, CA). The evaluation of total DNA purity was based on the Nanodrop<sup>™</sup> 2000 A<sub>260</sub>/A<sub>280</sub> ratio (Thermo Scientific, Wilmington, DE). DNA preparations served as a template for both real-time quantitative polymerase chain reaction (qPCR) and conventional PCR approaches.</p>                            <h3 scrollto-destination=78055508 id="78055508" class="section-title" >2.3. Design and validation of primer sequences</h3>
<p>Primers were designed based on the <em>P. ostreatus</em> reference genome sequences PC15 v2.0 and PC9 v1.0 (<a class="link link-uri" href="http://genome.jgi.doe.gov/PleosPC15_2/PleosPC15_2.home.html" target="_blank">http://genome.jgi.doe.gov/PleosPC15_2/PleosPC15_2.home.html</a>), using Primer3 software.<a reveal-id="dsw060-B31" data-open="dsw060-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> Specificity of primer pairs was verified <em>in silico</em> by two approaches: (i) BLASTN searches<a reveal-id="dsw060-B32" data-open="dsw060-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> against <em>P. ostreatus</em> genomes using primer sequences as a query and (ii) manual verification of perfect matches between oligonucleotides and binding sites in PC15_PC9 alignments of the target loci. The validated primers were used to perform experimental analyses based on qPCR and conventional PCR. </p>                            <h3 scrollto-destination=78055510 id="78055510" class="section-title" >2.4. Real-time quantitative PCR analysis</h3>
<div class="block-child-p">The helitron content was estimated by qPCR using a relative quantification approach.<a reveal-id="dsw060-B33" data-open="dsw060-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> Reactions were carried out in a CFX96 thermal cycler (Bio-Rad Laboratories, S.A.). Amplification products were monitored at each cycle of PCR using SYBR green fluorescent dye. Each amplification mixture (20 μl) contained 1X IQ SYBR green Supermix (10 μl), 6 μM forward and reverse primers and 0.5 ng of genomic DNA in nuclease-free water. Primers were designed targeting family-specific regions to selectively amplify the HELPO1 and HELPO2 elements. The list of primers designed for the amplification of helitrons and reference single-copy genes is shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw060/2/dsw060_Supp.zip?Expires=2147483647&Signature=zDAtgxa4F-UL54z2if0eYAx7iiuGb6srTDL1CT7Hx7wDxFEom1ojV1lVEE9Us27ThDolTKyt9Km4jlX7dNK4MHhuRaD32ktxUaSGDSkmnZRKWib00mcJHlAT~5TrnVNjQG5nML7AyU1hHzg~KRGemgvdez0x0d46-pjP6K0JGmfKcntTePu8ATjm8~7UeUJ2ducPfWRDovHNFdYPhH6uJ8yF-~DSkp7EeFNFkhyacpRLpdfEj97SkZCC1qTS06IJaQxL19jmK1BvUZWM6~NYrK7MU-sm57TX44qf0BU5aLXa6LgFrASDYk-iu7SgJew7ClWCHfbFZ8ttSGEcp8o3-g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>. Reactions were run according to the following cycling conditions: DNA templates were denatured at 95°C for 5 min, followed by 40 cycles of amplification (95°C for 15 s, 60°C for 30 s, and 95°C for 1 min). The final melting curve was set by increasing 0.5°C every 5 sec from 65 to 95°C. Each assay was performed in triplicate in 96-well plates, and a non-template control for each primer pair was included. Relative fluorescence units and threshold cycle values were processed using Bio-Rad CFX Manager Software, and the results were exported to Microsoft Excel for further analysis. The helitron copy number was estimated as a relative value in relation to the average signal of two single-copy reference genes, according to <a href="#E1" class="sectionLink xref-sec">Equation (1</a>): <a id="E1" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="E1"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi mathvariant="bold">R</mi><mi mathvariant="bold">C</mi><mi mathvariant="bold">N</mi><mo>=</mo><msup><mn>2</mn><mrow><mo>−</mo><mtext> </mtext><mo stretchy="false">(</mo><mi mathvariant="bold">C</mi><mi mathvariant="bold">t</mi><mi>h</mi><mi>e</mi><mi mathvariant="bold">l</mi><mtext> </mtext><mo>–</mo><mtext> </mtext><mi mathvariant="bold">C</mi><mi mathvariant="bold">t</mi><mi>r</mi><mi>e</mi><mi>f</mi><mo stretchy="false">)</mo></mrow></msup></mrow></math></div></div><div class="label">(1)</div></div> where RCN stands for relative copy number, Ct<sub>hel</sub> is the threshold cycle value of the target region (helitron) and Ct<sub>ref</sub> is the average threshold cycle value of <em>sar1</em> and <em>lacc3</em> single-copy reference genes.</div>                            <h3 scrollto-destination=78055512 id="78055512" class="section-title" >2.5. Conventional PCR reactions</h3>
<p>Several PCR protocols were designed according to the amplicon size and were run on a thermal cycler PCT-200 (MJ Research, MN). Biotaq DNA Polymerase (Bioline, Luckenwalde, Germany) was used to amplify products lower than 3.5 kb. Each PCR mix was performed in a final volume of 25 μl containing 1 unit of Biotaq polymerase, 50 mM MgCl<sub>2</sub>, dNTP mix at 100 mM, and forward and reverse primers at 6 μM. Template DNA was firstly denatured at 95°C for 5 min followed by 30 cycles of amplification (95°C for 40 s, 58°C for 60 s, 72°C for 1–3 min, depending on the expected product size) and a final extension cycle at 72°C for 10 min. RANGER DNA Polymerase (Bioline, Luckenwalde, Germany) was used to amplify products longer than 3.5 kb. PCR reactions were prepared following the manufacturer’s recommendations in a final volume of 50 μl containing 4 units of RANGER DNA polymerase, 20 μM of specific primers and 20 ng of total DNA template suspended in nuclease-free water. PCR conditions comprised initial denaturation at 95°C for 3 min followed by 30 cycles of 98°C for 10 s, 57°C for 60 s and 68°C for 5–10 min (depending on the expected product size), with a final extension at 68°C for 10 min. For both protocols, negative controls were included using nuclease-free water instead of DNA template. Amplicons were resolved by electrophoresis on 0.8% w/v TAE agarose gels (1X Tris-acetate-EDTA), stained with 1:20,000 Red Safe™ (iNtRON Biotechnology) and visualized under UV Illumination. PCR products were isolated from the agarose gels, purified with an E.Z.N.A Gel Extraction Kit (Omega Bio-Tek, Norcross, GA) and sequenced using Sanger technology (Sistemas Genómicos S.L.,Valencia, Spain and Genewiz Incorporated, South Plainfield, NJ). Primer pairs used for the conventional PCR approach are listed in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw060/2/dsw060_Supp.zip?Expires=2147483647&Signature=zDAtgxa4F-UL54z2if0eYAx7iiuGb6srTDL1CT7Hx7wDxFEom1ojV1lVEE9Us27ThDolTKyt9Km4jlX7dNK4MHhuRaD32ktxUaSGDSkmnZRKWib00mcJHlAT~5TrnVNjQG5nML7AyU1hHzg~KRGemgvdez0x0d46-pjP6K0JGmfKcntTePu8ATjm8~7UeUJ2ducPfWRDovHNFdYPhH6uJ8yF-~DSkp7EeFNFkhyacpRLpdfEj97SkZCC1qTS06IJaQxL19jmK1BvUZWM6~NYrK7MU-sm57TX44qf0BU5aLXa6LgFrASDYk-iu7SgJew7ClWCHfbFZ8ttSGEcp8o3-g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S2–S5</a></span>.</p>                            <h3 scrollto-destination=78055514 id="78055514" class="section-title" >2.6. Detection of RNAi pathway genes in <em>P. ostreatus</em></h3>
<p>Key genes involved in Quelling and Meiotic Silencing of Unpaired DNA (MSUD) pathways described in <em>Neurospora crassa</em> were used as a query for BLASTP searches (cutoff e value = e<sup>−</sup><sup>5</sup>) against <em>P. ostreatus</em> PC15 v2.0 and PC9 v1.0 annotations. GenBank accessions of <em>N. crassa</em> proteins were the following: <em>qde-1</em>: EAA29811.1, <em>qde-2</em>: ESA42123.1, <em>qde-3</em>: AAF31695.1<em>, sad-1</em>: AAK31733.1<em>, sms-2</em>: EAA29350<em>, dcl-1:</em> EAA32662.1<em>, dcl-2</em>: EAA34302.3, and <em>qip</em>: XP_011393741.1<em>.</em> Conserved domains of <em>P. ostreatus</em> proteins with significant hits were identified using HMMER3 software<a reveal-id="dsw060-B34" data-open="dsw060-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> with PFAM-A target database,<a reveal-id="dsw060-B35" data-open="dsw060-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> as well as NCBI Conserved Domain Database<em>.</em><a reveal-id="dsw060-B36" data-open="dsw060-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> In parallel, proteins were clustered using all by all BLASTP (cutoff e value = e<sup><em>−</em></sup><sup>10</sup>) followed by mcl<a reveal-id="dsw060-B37" data-open="dsw060-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> (inflation value<em> </em>=2). Clusters of proteins carrying the same conserved domains as the query proteins were retained. Orthology between PC15 and PC9 proteins was inferred by reciprocal best blast hit.</p>                            <h2 scrollto-destination=78055516 id="78055516" class="section-title" >3. Results and discussion</h2>
                            <h3 scrollto-destination=78055517 id="78055517" class="section-title" >3.1. HELPO1 and HELPO2 families show different inheritance patterns in the N001 meiotic progeny</h3>
<div class="block-child-p">To analyse the inheritance patterns of the two helitron families present in <em>P. ostreatus</em>, we studied helitron segregation in a progeny of 68 monokaryons meiotically derived from the N001 strain (population 68 mK) by means of qPCR. The relative copy number of HELPO1 elements showed a normal distribution in the progeny (Shapiro-Wilks test, W = 0.978, <em>P</em><em> </em>=<em> </em>0.271), with monokaryons displaying RCN values ranging from 2 to 13 and a population peak showing 6–7 RCNs (<a reveal-id="dsw060-F1" data-open="dsw060-F1" class="link link-reveal link-table xref-fig">Fig. 1A</a>). Surprisingly, results corresponding to the HELPO2 family showed a trend that did not fit with a normal distribution (W = 0.727, <em>P</em><em> </em>=<em> </em>5.135 e<sup>−</sup><sup>10</sup>, <a reveal-id="dsw060-F1" data-open="dsw060-F1" class="link link-reveal link-table xref-fig">Fig. 1B</a>). In this case, although RCNs ranged from 0 to 9, up to 49 monokaryons exhibited only 0 or 1 helitrons, uncovering a strong bias towards the lack of HELPO2 elements in the N001 progeny. Previous studies carried out by our group indicated that the HELPO2 family amplified very recently in the protoclone PC15, whereas it is absent in the compatible strain PC9.<a reveal-id="dsw060-B17" data-open="dsw060-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> The distorted segregation of HELPO2 elements might be the consequence of a specific genome defence mechanism against the invasion of repetitive DNA. In this sense, the HELPO2 family shows elements with high similarity among them and low transcription levels,<a reveal-id="dsw060-B17" data-open="dsw060-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a><sup>,</sup><a reveal-id="dsw060-B27" data-open="dsw060-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> indicating that it might be targeted by transposon-silencing mechanisms. In contrast, HELPO1 elements are more divergent and showed very high transcription levels.<a reveal-id="dsw060-B17" data-open="dsw060-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a><sup>,</sup><a reveal-id="dsw060-B27" data-open="dsw060-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> This fact suggests that this family is not targeted by genome defense mechanisms. A possible explanation for the unique profile of HELPO2 might be the very recent amplification of RepHel-coding elements in this family, evidenced by the presence of five identical full-length copies. These elements could be detected as foreign, invasive DNA and targeted by the genome defense machinery. To understand the striking differences found in the inheritance patterns of HELPO1 and HELPO2 families in the meiotic progeny, we reconstructed the pedigree of the monokaryotic collection along with their parental strains N001, PC15 and PC9 (<a reveal-id="dsw060-F2" data-open="dsw060-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>), which have been used as a genetic model of study since 1994. <div data-id="dsw060-F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw060/2/m_dsw060f1.png?Expires=2147483647&amp;Signature=xvb1h59zMyqodzKt4Z8LdJiQGjxCj3~gPZXL22u2XlIiytpIJoCvZ2UT21xLEWomXxUqgJAhR4Yf~jxolY2W16b~~tur~lHzHMAL-a50GZJ76KqOaNMfXrDtyHDjMPNOw0pZYoQKmxfsc0WskPOv-kCVzyphPnAGGBKLol41CItTURmGU4vJCSAgi~a3wV-MUKnov4osTjxXb~fPrk26iju4hlTlxD~9bnj6foLFvQlL20nZgwd8PHks1TCzm7EEvanw67yMH1KLGlEu~-wvJQOuLWmR5e-8giF0Et~M-ZkyAfp3lvwGQ68izzOO41HRAKN2YBBMdcoSBlrn29~MIQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of HELPO1 (A) and HELPO2 (B) relative copy number in the 68-mK meiotic population. Relative copy numbers are presented as histograms with density lines." path-from-xml="dsw060f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055518/dsw060f1.tif" path-from-xml="dsw060f1.tif" target="_blank">View large</a><a section="78055518" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw060/2/dsw060f1.png?Expires=2147483647&amp;Signature=WhIZTjw0yGuYDcz-poV3Zx16z5itPvw1vptyZCQ8C8KoDmweIfAGQ5fuf1tSb2tAZ4cY2hCtabbOn4JRzrn4BySrjCbKRihnqC91qe1pa5xSB76VqSOhcNsuVJS-nLHZ~8Y2aC1iHO5REpTpgk2AHMlIgRJZSKN5-sr~Cb7tSVQJxy7WpoNegCVZMv3KMgyAicmnKdApR71-kIuGEooCRvXWRJ8~ejmnaiorqBK4rjIpicRA5Z1Me92zclNGRoWEPyKeufTH7UdfXhWENfF0pHRgAA75Hxwj2oXA6DOuNv6ED1a1336L5G3sm0RNLqwx4ssnqyBKArvVQMRUYwJ1FQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=78055518&amp;ar=2962881&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw060f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of HELPO1 (A) and HELPO2 (B) relative copy number in the 68-mK meiotic population. Relative copy numbers are presented as histograms with density lines.</p></div></div><div content-id="dsw060-F1" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw060/2/m_dsw060f1.png?Expires=2147483647&amp;Signature=xvb1h59zMyqodzKt4Z8LdJiQGjxCj3~gPZXL22u2XlIiytpIJoCvZ2UT21xLEWomXxUqgJAhR4Yf~jxolY2W16b~~tur~lHzHMAL-a50GZJ76KqOaNMfXrDtyHDjMPNOw0pZYoQKmxfsc0WskPOv-kCVzyphPnAGGBKLol41CItTURmGU4vJCSAgi~a3wV-MUKnov4osTjxXb~fPrk26iju4hlTlxD~9bnj6foLFvQlL20nZgwd8PHks1TCzm7EEvanw67yMH1KLGlEu~-wvJQOuLWmR5e-8giF0Et~M-ZkyAfp3lvwGQ68izzOO41HRAKN2YBBMdcoSBlrn29~MIQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of HELPO1 (A) and HELPO2 (B) relative copy number in the 68-mK meiotic population. Relative copy numbers are presented as histograms with density lines." path-from-xml="dsw060f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055518/dsw060f1.tif" path-from-xml="dsw060f1.tif" target="_blank">View large</a><a section="78055518" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw060/2/dsw060f1.png?Expires=2147483647&amp;Signature=WhIZTjw0yGuYDcz-poV3Zx16z5itPvw1vptyZCQ8C8KoDmweIfAGQ5fuf1tSb2tAZ4cY2hCtabbOn4JRzrn4BySrjCbKRihnqC91qe1pa5xSB76VqSOhcNsuVJS-nLHZ~8Y2aC1iHO5REpTpgk2AHMlIgRJZSKN5-sr~Cb7tSVQJxy7WpoNegCVZMv3KMgyAicmnKdApR71-kIuGEooCRvXWRJ8~ejmnaiorqBK4rjIpicRA5Z1Me92zclNGRoWEPyKeufTH7UdfXhWENfF0pHRgAA75Hxwj2oXA6DOuNv6ED1a1336L5G3sm0RNLqwx4ssnqyBKArvVQMRUYwJ1FQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=78055518&amp;ar=2962881&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw060f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of HELPO1 (A) and HELPO2 (B) relative copy number in the 68-mK meiotic population. Relative copy numbers are presented as histograms with density lines.</p></div></div><div data-id="dsw060-F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw060/2/m_dsw060f2.png?Expires=2147483647&amp;Signature=bqcVBM0qpJT~9yw5CuZmv4w1ygohnwoqSuzMZqZmX-ZikxMXg~SGvgev6Ao8DfwQwltGLQVeskFZwoJ7OHvb7x2-~x7lQoV-W~26DEwut2GUCBE4k62FZs5J5-gbOnt3NkJAun2vybTgTyN-~y2cUYIHwjZ0XXtk~lWHc-ym4yM7uJabXdQX~7QNgh-XirZncS~h9XXC3XNKoBFguyxJKBW8JYSg3B4x4twyoj8c3X2AjIDfvQFB3TePpd0njtyUb9e~hXtYYZDJfF1VkRdQUXW6-DeQp4L5wt3r8dA0sOwf0Py-jT6NqPPumXTuJG0fJTa3L7aBwfxn6c1xB6gKcQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Pedigree and related time-scale events of P. ostreatus strains and subclones used in this study. (a) (1994) The N001-94 strain was sporulated leading to the monokaryotic population 68 mK. (b) (1999) PC9 and PC15 protoclones were obtained from N001 by de-dikaryotization and deposited in CECT (PC9-99 and PC15-99). (c) (2003) N001 was deposited in CECT (N001-03). (d) (2005) PC9 and PC15 protoclones were sequenced at JGI. (e) (2014) N001-03, PC9-99 and PC15-99 were retrieved from CECT and used in this study, along with N001-14, PC9-14 and PC15-14 subclones used for laboratory routine work up to 2014." path-from-xml="dsw060f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055518/dsw060f2.tif" path-from-xml="dsw060f2.tif" target="_blank">View large</a><a section="78055518" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw060/2/dsw060f2.png?Expires=2147483647&amp;Signature=Z42Yje8Ngpo-RG5EaLBSRMdANEZFDSZ9MY-p1A1FSuJa5kAf3MJgU-goeFigggNLSV64djG2o891UVAQJJ03rj6ny5zNAEvgsCMT4uhyhfIBtGPrzpDwlzOJic1E1iQSAVhgNopk4yc5lOhen0FvRldi1d33kXjLKAqGyIas1~2iaZVi4tOtyPku5M3wJEPMpZ0-hxZ1eYt6Z5r3nmlVL~u2a1AXaBr0bWUOZRhJKp4DPBEtEn6Hgt2-eagxKm4SRqAJLPmckDFotPFD3cW8fKv4ui5eHpKIL4rEOBD25HSKJxfl1rQFYAK7FI5eMK5NnRTL3qIOXtYUCnwxDvdksQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=78055518&amp;ar=2962881&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw060f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Pedigree and related time-scale events of <em>P. ostreatus</em> strains and subclones used in this study. (a) (1994) The N001-94 strain was sporulated leading to the monokaryotic population 68 mK. (b) (1999) PC9 and PC15 protoclones were obtained from N001 by de-dikaryotization and deposited in CECT (PC9-99 and PC15-99). (c) (2003) N001 was deposited in CECT (N001-03). (d) (2005) PC9 and PC15 protoclones were sequenced at JGI. (e) (2014) N001-03, PC9-99 and PC15-99 were retrieved from CECT and used in this study, along with N001-14, PC9-14 and PC15-14 subclones used for laboratory routine work up to 2014.</p></div></div><div content-id="dsw060-F2" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw060/2/m_dsw060f2.png?Expires=2147483647&amp;Signature=bqcVBM0qpJT~9yw5CuZmv4w1ygohnwoqSuzMZqZmX-ZikxMXg~SGvgev6Ao8DfwQwltGLQVeskFZwoJ7OHvb7x2-~x7lQoV-W~26DEwut2GUCBE4k62FZs5J5-gbOnt3NkJAun2vybTgTyN-~y2cUYIHwjZ0XXtk~lWHc-ym4yM7uJabXdQX~7QNgh-XirZncS~h9XXC3XNKoBFguyxJKBW8JYSg3B4x4twyoj8c3X2AjIDfvQFB3TePpd0njtyUb9e~hXtYYZDJfF1VkRdQUXW6-DeQp4L5wt3r8dA0sOwf0Py-jT6NqPPumXTuJG0fJTa3L7aBwfxn6c1xB6gKcQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Pedigree and related time-scale events of P. ostreatus strains and subclones used in this study. (a) (1994) The N001-94 strain was sporulated leading to the monokaryotic population 68 mK. (b) (1999) PC9 and PC15 protoclones were obtained from N001 by de-dikaryotization and deposited in CECT (PC9-99 and PC15-99). (c) (2003) N001 was deposited in CECT (N001-03). (d) (2005) PC9 and PC15 protoclones were sequenced at JGI. (e) (2014) N001-03, PC9-99 and PC15-99 were retrieved from CECT and used in this study, along with N001-14, PC9-14 and PC15-14 subclones used for laboratory routine work up to 2014." path-from-xml="dsw060f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055518/dsw060f2.tif" path-from-xml="dsw060f2.tif" target="_blank">View large</a><a section="78055518" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw060/2/dsw060f2.png?Expires=2147483647&amp;Signature=Z42Yje8Ngpo-RG5EaLBSRMdANEZFDSZ9MY-p1A1FSuJa5kAf3MJgU-goeFigggNLSV64djG2o891UVAQJJ03rj6ny5zNAEvgsCMT4uhyhfIBtGPrzpDwlzOJic1E1iQSAVhgNopk4yc5lOhen0FvRldi1d33kXjLKAqGyIas1~2iaZVi4tOtyPku5M3wJEPMpZ0-hxZ1eYt6Z5r3nmlVL~u2a1AXaBr0bWUOZRhJKp4DPBEtEn6Hgt2-eagxKm4SRqAJLPmckDFotPFD3cW8fKv4ui5eHpKIL4rEOBD25HSKJxfl1rQFYAK7FI5eMK5NnRTL3qIOXtYUCnwxDvdksQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=78055518&amp;ar=2962881&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw060f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Pedigree and related time-scale events of <em>P. ostreatus</em> strains and subclones used in this study. (a) (1994) The N001-94 strain was sporulated leading to the monokaryotic population 68 mK. (b) (1999) PC9 and PC15 protoclones were obtained from N001 by de-dikaryotization and deposited in CECT (PC9-99 and PC15-99). (c) (2003) N001 was deposited in CECT (N001-03). (d) (2005) PC9 and PC15 protoclones were sequenced at JGI. (e) (2014) N001-03, PC9-99 and PC15-99 were retrieved from CECT and used in this study, along with N001-14, PC9-14 and PC15-14 subclones used for laboratory routine work up to 2014.</p></div></div></div>                            <h3 scrollto-destination=78055519 id="78055519" class="section-title" >3.2. HELPO2 dynamics diverge in subclones maintained under different subculture conditions</h3>
<div class="block-child-p">The abundance of HELPO1 and HELPO2 elements was estimated in several subclones of the monokaryons and dikaryon parental strains. These subclones have been stored under different subculture frequencies for up to 20 years, as shown in <a reveal-id="dsw060-T1" data-open="dsw060-T1" class="link link-reveal link-table xref-fig">Table 1</a> and <a reveal-id="dsw060-F2" data-open="dsw060-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>. Using the previously described qPCR approach, we observed that HELPO1 RCNs were largely conserved and consistently identical among the subclones maintained under a low subculture frequency (PC9-99, PC15-99, N001-03) and the corresponding subclones maintained in the laboratory at a high subculture frequency (PC9–14, PC15–14 and N001-14; <em>P</em> &lt; 0.05, two-tailed Student’s <em>t</em>-test, <a reveal-id="dsw060-F3" data-open="dsw060-F3" class="link link-reveal link-table xref-fig">Fig. 3A</a>). Furthermore, the RCNs observed in N001 dikaryon subclones fit with the sum of PC9 and PC15 RCNs, as expected for an additive model (N001 is a dikaryotic strain that carries the PC15 and PC9 nuclei). In contrast, the estimation of HELPO2 abundance in the monokaryotic and dikaryotic subclones revealed an unexpected scenario. Significant differences were found in HELPO2 content within PC15 and N001 subclone pairs (<a reveal-id="dsw060-F3" data-open="dsw060-F3" class="link link-reveal link-table xref-fig">Fig. 3B</a>). More specifically, PC15-99 showed a RCN value higher than PC15-14 (23 vs 15, respectively), whereas N001-14 showed a striking 5-fold increment in comparison to N001-03 (11 vs 2 RCNs). In the case of PC9, both subclones lacked HELPO2 elements. Previous bioinformatics analysis reported a widespread distribution of HELPO2 copies in the chromosomes of PC15, whereas no intact HELPO2 helitrons were detected in the genome of the PC9 strain.<a reveal-id="dsw060-B17" data-open="dsw060-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> In this sense, qPCR results confirmed the lack of HELPO2 elements in both PC9-99 and PC9-14 strains. Nevertheless, the differential HELPO2 content in the PC15 and N001 subclones suggested that these elements have been mobilized in the mycelia (somatic transposition) and accumulated in the genome as a result of a continuous subculture. The fact that the addition of PC15 and PC9 HELPO2 RCNs surpasses that of any of the N001 subclones reinforces the hypothesis of an amplification burst in both PC15 subclones. Previous studies have described an amplification burst of DNA transposons (<em>IS30</em> and <em>IS5</em>) in bacterial subclones maintained for decades in stab culture, giving rise to genetic and phenotypic diversity.<a reveal-id="dsw060-B38" data-open="dsw060-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> In addition, it is known that environmental stresses increase transposition rates.<a reveal-id="dsw060-B39" data-open="dsw060-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a><sup>,</sup><a reveal-id="dsw060-B40" data-open="dsw060-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> In this sense, the de-dikaryotization process underwent by the strain N001 in 1999 might have played a critical role in helitron increases, as protoplast isolation requires an enzymatic digestion protocol that could have triggered a stressful scenario leading to HELPO2 elements activation. We hypothesize that high subculturing rates may increase the chance of helitron transposition and accumulation. The 5-fold increment of the HELPO2 element content in N001–14 vs N001-03 supports this hypothesis, but does not explain the results observed in PC15-99 and PC9-14. In the latter case, the subclone maintained under low-subculturing frequency (PC15-99) shared a higher HELPO2 RCNs than its counterpart maintained under higher subculturing rates. This finding suggest that TE insertions without strong deleterious effects have a random chance of being passed to successive generations by clonal subculturing, thus bringing to the equation a randomness parameter. On the contrary, HELPO1 family elements did not show any sign of amplification. This finding, along with the uniform patterns of its inheritance found in the N001 mk68 meiotic progeny suggested that transposition of HELPO1 elements is infrequent in the <em>P. ostreatus</em> genome. <div data-id="dsw060-F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw060/2/m_dsw060f3.png?Expires=2147483647&Signature=cmrvxQPWIAymmazCTVrKzawmyHnBNhy4yl8tFchAEC3XYzkf7IvrfSVlt13Fk34QnT7NlB9F79mFJIJAxJ2mD4Kbash3IbO2wdJOBtF4elQbAfGLmuefEPaRaMIBIaP~45tD6hiQYvToJT26lQUTseh~iLp0dfurarubjxKRwFzYsZBToeaO6dTgijvmPjPDLz5PLKY5-09BQ8yus7-3kNGtyWzw0JhqWF8IsIZhHp4YNy3YYGzdzLgOFpPIuxHdhrDiOiDJoBiWPkf0F~nXbFudxYoarqgsb~YNTZjFTqI4pFoiDe6gqNEdgvzbkgx5~HvgI0pHQ9PULR8xXQp~vw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="HELPO1 (A) and HELPO2 (B) relative copy number in parental subclones maintained under different subculture frequencies. Relative copy number of elements belonging to HELPO1 and HELPO2 families examined in the dikaryotic (N001) and monokaryotic (PC15 and PC9) parental strains. Data are reported as the mean ± SD." path-from-xml="dsw060f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055520/dsw060f3.tif" path-from-xml="dsw060f3.tif" target="_blank">View large</a><a section="78055520" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw060/2/dsw060f3.png?Expires=2147483647&Signature=MhISYpcb6JAL~3zlNDd0PN4uhqGyPRKQRlLQlyaH3GiOQJl8Ej0LffUpsDgwL~E0EcxpzkCEcklLeWBSvWfSbdtdmn5dk9QGG4WU945dyjOrLVgtmficqbFE9T~3xAIy-RRswkEX8bXP2vZQ-HyP~~43ItEO0M9AI8rBjUKzJ4nfaZ7rFPtBgC6aIqBVy~upgRfhlwzSXN33SYkOCPmIoJgUhU2qV1hkv8kKY8lgsAzbJAAk5Pgq2JXDJqJqcDeDagXpwnG24rsfNSzm02UP5VKo11zwKI70w4kwOxsXNEM8VL55ljEUN73xWW7qYb5oZ2~uW9tMmiPsyzi1FBpoIA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=78055520&ar=2962881&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw060f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>HELPO1 (A) and HELPO2 (B) relative copy number in parental subclones maintained under different subculture frequencies. Relative copy number of elements belonging to HELPO1 and HELPO2 families examined in the dikaryotic (N001) and monokaryotic (PC15 and PC9) parental strains. Data are reported as the mean ± SD.</p></div></div><div content-id="dsw060-F3" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw060/2/m_dsw060f3.png?Expires=2147483647&Signature=cmrvxQPWIAymmazCTVrKzawmyHnBNhy4yl8tFchAEC3XYzkf7IvrfSVlt13Fk34QnT7NlB9F79mFJIJAxJ2mD4Kbash3IbO2wdJOBtF4elQbAfGLmuefEPaRaMIBIaP~45tD6hiQYvToJT26lQUTseh~iLp0dfurarubjxKRwFzYsZBToeaO6dTgijvmPjPDLz5PLKY5-09BQ8yus7-3kNGtyWzw0JhqWF8IsIZhHp4YNy3YYGzdzLgOFpPIuxHdhrDiOiDJoBiWPkf0F~nXbFudxYoarqgsb~YNTZjFTqI4pFoiDe6gqNEdgvzbkgx5~HvgI0pHQ9PULR8xXQp~vw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="HELPO1 (A) and HELPO2 (B) relative copy number in parental subclones maintained under different subculture frequencies. Relative copy number of elements belonging to HELPO1 and HELPO2 families examined in the dikaryotic (N001) and monokaryotic (PC15 and PC9) parental strains. Data are reported as the mean ± SD." path-from-xml="dsw060f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055520/dsw060f3.tif" path-from-xml="dsw060f3.tif" target="_blank">View large</a><a section="78055520" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw060/2/dsw060f3.png?Expires=2147483647&Signature=MhISYpcb6JAL~3zlNDd0PN4uhqGyPRKQRlLQlyaH3GiOQJl8Ej0LffUpsDgwL~E0EcxpzkCEcklLeWBSvWfSbdtdmn5dk9QGG4WU945dyjOrLVgtmficqbFE9T~3xAIy-RRswkEX8bXP2vZQ-HyP~~43ItEO0M9AI8rBjUKzJ4nfaZ7rFPtBgC6aIqBVy~upgRfhlwzSXN33SYkOCPmIoJgUhU2qV1hkv8kKY8lgsAzbJAAk5Pgq2JXDJqJqcDeDagXpwnG24rsfNSzm02UP5VKo11zwKI70w4kwOxsXNEM8VL55ljEUN73xWW7qYb5oZ2~uW9tMmiPsyzi1FBpoIA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=78055520&ar=2962881&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw060f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>HELPO1 (A) and HELPO2 (B) relative copy number in parental subclones maintained under different subculture frequencies. Relative copy number of elements belonging to HELPO1 and HELPO2 families examined in the dikaryotic (N001) and monokaryotic (PC15 and PC9) parental strains. Data are reported as the mean ± SD.</p></div></div></div>                            <h3 scrollto-destination=78055521 id="78055521" class="section-title" >3.3. Molecular analysis of HELPO2 polymorphisms uncovers HELPO2 transposition events</h3>
<div class="block-child-p">The results obtained by the qPCR approach strongly suggest that HELPO2 is an active family, and the RCN increments observed in PC15 and N001 suggest that new elements of this family could be populating unknown loci. Previous bioinformatics analyses identified seven HELPO2 copies encoding RepHel helicases, distributed in 5 of 11 nuclear chromosomes of PC15 (<a reveal-id="dsw060-T2" data-open="dsw060-T2" class="link link-reveal link-table xref-fig">Table 2</a>). This genome-wide helitron annotation was carried out with <em>P. ostreatus</em> PC9 v1.0 and PC15 v2.0 genome assemblies<a reveal-id="dsw060-B17" data-open="dsw060-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> produced by the JGI in 2005. Using conventional PCR, we re-analysed the parental subclone pairs to track the presence or absence of such HELPO2 loci. The objective of this approach was to confirm the bioinformatics predictions and to identify putative helitron polymorphisms between and within subclone pairs that could be the result of helitron mobilizations. A PCR strategy was designed to differentiate the two possible allelic states corresponding to the presence or absence of a helitron element in a given locus. We designed two primer pairs per locus flanking the left and right helitron ends. In each pair, one primer was designed inside the element, and the other was designed outside the helitron boundary, as shown in <a reveal-id="dsw060-F4" data-open="dsw060-F4" class="link link-reveal link-table xref-fig">Fig. 4A and B</a>. This strategy (outer/inner) allowed us to identify loci carrying HELPO2 insertions (“occupied loci”) with positive PCR products, but no amplification should occur in loci with no HELPO2 insertion (“vacant loci”). As expected, none of the PC9 subclones showed amplification in any of the HELPO2 loci (<a reveal-id="dsw060-T3" data-open="dsw060-T3" class="link link-reveal link-table xref-fig">Table 3</a>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw060/2/dsw060_Supp.zip?Expires=2147483647&amp;Signature=zDAtgxa4F-UL54z2if0eYAx7iiuGb6srTDL1CT7Hx7wDxFEom1ojV1lVEE9Us27ThDolTKyt9Km4jlX7dNK4MHhuRaD32ktxUaSGDSkmnZRKWib00mcJHlAT~5TrnVNjQG5nML7AyU1hHzg~KRGemgvdez0x0d46-pjP6K0JGmfKcntTePu8ATjm8~7UeUJ2ducPfWRDovHNFdYPhH6uJ8yF-~DSkp7EeFNFkhyacpRLpdfEj97SkZCC1qTS06IJaQxL19jmK1BvUZWM6~NYrK7MU-sm57TX44qf0BU5aLXa6LgFrASDYk-iu7SgJew7ClWCHfbFZ8ttSGEcp8o3-g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>). Interestingly, a striking difference was observed between PC15-99 and PC15-14 subclones: although the first one displayed HELPO2 insertions in the seven loci described in <a reveal-id="dsw060-T2" data-open="dsw060-T2" class="link link-reveal link-table xref-fig">Table 2</a>, the latter showed no amplification in <em>Helpo2_I</em> and in <em>Helpo2_VIII_c.</em> Intriguingly, both N001 subclones (N001-03 and N001-14) showed an identical profile to PC15-14 (<a reveal-id="dsw060-T3" data-open="dsw060-T3" class="link link-reveal link-table xref-fig">Table 3</a>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw060/2/dsw060_Supp.zip?Expires=2147483647&amp;Signature=zDAtgxa4F-UL54z2if0eYAx7iiuGb6srTDL1CT7Hx7wDxFEom1ojV1lVEE9Us27ThDolTKyt9Km4jlX7dNK4MHhuRaD32ktxUaSGDSkmnZRKWib00mcJHlAT~5TrnVNjQG5nML7AyU1hHzg~KRGemgvdez0x0d46-pjP6K0JGmfKcntTePu8ATjm8~7UeUJ2ducPfWRDovHNFdYPhH6uJ8yF-~DSkp7EeFNFkhyacpRLpdfEj97SkZCC1qTS06IJaQxL19jmK1BvUZWM6~NYrK7MU-sm57TX44qf0BU5aLXa6LgFrASDYk-iu7SgJew7ClWCHfbFZ8ttSGEcp8o3-g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figs. S1A and G</a></span>). This result leads us to hypothesize that <em>Helpo2_I</em> and <em>Helpo2_VIII_c</em> helitrons were transposed and inserted exclusively in PC15-99 subclone, after the de-dikaryotization of N001 in 1999. The experimental evidence of helitron transposition has been recently demonstrated in vitro using a reconstructed element from the bat genome.<a reveal-id="dsw060-B12" data-open="dsw060-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> Our finding represents the first evidence of a fungal helitron transposition in its host genome under laboratory conditions. PCR products corresponding to <em>Helpo2_I</em> and <em>Helpo2_VIII_c</em> were purified, sequenced by Sanger technology and aligned to PC15 v2.0 and PC9 v1.0 genome sequences. The results obtained confirmed the helitron presence in the <em>Helpo2_I</em> locus, with the alignments spanning both external and internal regions of the HELPO2 (<a reveal-id="dsw060-F5" data-open="dsw060-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>). Regarding <em>Helpo2_VIII_c</em>, it is noteworthy that the PCR product of the right flank displayed a size 0.6 kb larger than expected. In this case, sequenced PCR products could be aligned to the internal helitron regions but not to the external ones due to the low quality of the sequence. To discard that this phenomenon as a false-negative, we designed a second strategy to validate vacant sites with a positive PCR product. Using the PC15 sequence as a reference, primers were designed for regions adjacent to HELPO2 boundaries and homologous to the PC9 regions flanking the vacant site (outer/outer, <a reveal-id="dsw060-F4" data-open="dsw060-F4" class="link link-reveal link-table xref-fig">Fig. 4A</a>). The complexity of such regions (often carrying other transposons, especially <em>Gypsy</em> LTR-retrotransposons) made it difficult to find homologous regions between PC15 and PC9, yielding long expected PCR fragment sizes for vacant sites, ranging from 2,603 to 8,578 bp (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw060/2/dsw060_Supp.zip?Expires=2147483647&amp;Signature=zDAtgxa4F-UL54z2if0eYAx7iiuGb6srTDL1CT7Hx7wDxFEom1ojV1lVEE9Us27ThDolTKyt9Km4jlX7dNK4MHhuRaD32ktxUaSGDSkmnZRKWib00mcJHlAT~5TrnVNjQG5nML7AyU1hHzg~KRGemgvdez0x0d46-pjP6K0JGmfKcntTePu8ATjm8~7UeUJ2ducPfWRDovHNFdYPhH6uJ8yF-~DSkp7EeFNFkhyacpRLpdfEj97SkZCC1qTS06IJaQxL19jmK1BvUZWM6~NYrK7MU-sm57TX44qf0BU5aLXa6LgFrASDYk-iu7SgJew7ClWCHfbFZ8ttSGEcp8o3-g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>). To overcome this problem, we used a long-range DNA polymerase that allowed us to amplify vacant and occupied sites including PCR product size up to 10–15 kb (i.e. <a reveal-id="dsw060-F4" data-open="dsw060-F4" class="link link-reveal link-table xref-fig">Fig. 4C</a>). The presence of vacant sites was tested in all subclones (<a reveal-id="dsw060-T4" data-open="dsw060-T4" class="link link-reveal link-table xref-fig">Table 4</a>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw060/2/dsw060_Supp.zip?Expires=2147483647&amp;Signature=zDAtgxa4F-UL54z2if0eYAx7iiuGb6srTDL1CT7Hx7wDxFEom1ojV1lVEE9Us27ThDolTKyt9Km4jlX7dNK4MHhuRaD32ktxUaSGDSkmnZRKWib00mcJHlAT~5TrnVNjQG5nML7AyU1hHzg~KRGemgvdez0x0d46-pjP6K0JGmfKcntTePu8ATjm8~7UeUJ2ducPfWRDovHNFdYPhH6uJ8yF-~DSkp7EeFNFkhyacpRLpdfEj97SkZCC1qTS06IJaQxL19jmK1BvUZWM6~NYrK7MU-sm57TX44qf0BU5aLXa6LgFrASDYk-iu7SgJew7ClWCHfbFZ8ttSGEcp8o3-g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>) including the discordant <em>Helpo2_I</em> and <em>Helpo2_VIII_c</em> loci in the PC15-99 subclone, despite the outer/inner strategy confirming the presence of HELPO2 in both of them. Using this approach, we validated all vacant sites except PC9 <em>Helpo2_VII.</em> This particular locus was further analysed with additional primer pairs and could not be amplified in any of the PC9 subclones, probably due to the inefficacy of long-range RANGER polymerase. Remarkably, the amplification and sequencing of <em>Helpo2_I</em> and <em>Helpo2_VIII_c</em> in PC15-99 revealed products corresponding to vacant sites, even though we had previously validated the presence of HELPO2 helitrons in both loci by the outer-inner strategy. Validation of these two vacant sites was performed by sequencing and subsequent alignment to the PC15 v2.0 reference (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw060/2/dsw060_Supp.zip?Expires=2147483647&amp;Signature=zDAtgxa4F-UL54z2if0eYAx7iiuGb6srTDL1CT7Hx7wDxFEom1ojV1lVEE9Us27ThDolTKyt9Km4jlX7dNK4MHhuRaD32ktxUaSGDSkmnZRKWib00mcJHlAT~5TrnVNjQG5nML7AyU1hHzg~KRGemgvdez0x0d46-pjP6K0JGmfKcntTePu8ATjm8~7UeUJ2ducPfWRDovHNFdYPhH6uJ8yF-~DSkp7EeFNFkhyacpRLpdfEj97SkZCC1qTS06IJaQxL19jmK1BvUZWM6~NYrK7MU-sm57TX44qf0BU5aLXa6LgFrASDYk-iu7SgJew7ClWCHfbFZ8ttSGEcp8o3-g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>). These results could suggest that cells with and without HELPO2 elements (HELPO2+/HELPO2-) coexist in hyphae of the PC15-99 subclone, giving rise to a mosaic mycelium. This fact reinforces the hypothesis of a very recent transposition of both elements, which were not entirely fixed or eliminated from the PC15-99 subclone in the successive subculturing performed after their insertion. In addition, these results provide indirect evidence that helitron was mobilized by replicative or semi-replicative transposition, as HELPO2 elements increased in PC15–99 without the excision of any of the other five occupied loci. <div data-id="dsw060-F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw060/2/m_dsw060f4.png?Expires=2147483647&amp;Signature=J55FgrA749sQth4BVgiQSQk1fy8kIFyjW~BBPMvrgcJQ26mgxOxwJLuMbwmcxZYQWdo5nuGw1iQbNF7YHcNNUhbWz1Jim3Pq~qAi2NRpxDP9Vxz2tCrbr9th-bh0x2LWLklo4tV7ylZpjY3KsmHohO0jujI4LwP6wNbZhHAFt50ZLXTg-mfp~T~UWxvMPuw7KWV9qt2zwMaOgvihBLfZ~iVUVuJqqWbMVeAeoMjviRFbb3vdRKkH0VPUyubiFw79cXOGO0FWXpAJMPdMVn6hdcXHYV3RyI1XWG4-fx9wCCRYlNUzPuV7ibGTjMo79B3VwekVCJllNGICGVUUG264~Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Detection of HELPO2 polymorphisms by the outer/inner and outer/outer strategies. (A) Example of a HELPO2 (Helpo2_VIII_a) polymorphic site in PC15 and PC9 haplotypes shown as an ACT (Artemis Comparison Tool) comparison. Blocks indicate conserved regions (&gt;85% similarity). The HELPO2 element is shown as a white arrow with a grey rectangle inside the arrow representing its encoded RepHel helicase. Vertical black lines show stop codons predicted in the three forward reading frames. (B) and (C) represent the amplification profiles of a polymorphic HELPO2 locus in PC9 (vacant), PC15 (occupied) and N001 (vacant + occupied) subclones using the outer/inner and outer/outer strategies." path-from-xml="dsw060f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055522/dsw060f4.tif" path-from-xml="dsw060f4.tif" target="_blank">View large</a><a section="78055522" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw060/2/dsw060f4.png?Expires=2147483647&amp;Signature=hlo9ixrSrhxyeP0ccm0XGKuuNYbM8CCdPKOazUxv5T27YkFCzbHeELMGgyuZcL-Xbs83hu~odzUQSe46OljJ0poW8Igs4TSofVvUrZ~Ys-7VJnsORbGaRnHEmKVXqT-fa6Ab1ikXHrBNPp60Em9kDqCF7BEZSAZrV99c7u1GD1744iUYNKqNuiYrkTsoXsGSYFhc9zq2gh1qi3-TwkT8Wb4JPWPmrjgrwQFnJ35cz0mWY7qefQYxOHNTYc0mt1td1ybXmtvFWlRrpdZRNotJh8k-Bpo3etio3gpaRv1IcqAevcKGhcZu53w8JpexAgJZmzz4vPCQSAu~KWWKPetSzQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=78055522&amp;ar=2962881&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw060f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Detection of HELPO2 polymorphisms by the outer/inner and outer/outer strategies. (A) Example of a HELPO2 (<em>Helpo2_VIII_a</em>) polymorphic site in PC15 and PC9 haplotypes shown as an ACT (Artemis Comparison Tool) comparison. Blocks indicate conserved regions (&gt;85% similarity). The HELPO2 element is shown as a white arrow with a grey rectangle inside the arrow representing its encoded RepHel helicase. Vertical black lines show stop codons predicted in the three forward reading frames. (B) and (C) represent the amplification profiles of a polymorphic HELPO2 locus in PC9 (vacant), PC15 (occupied) and N001 (vacant + occupied) subclones using the outer/inner and outer/outer strategies.</p></div></div><div content-id="dsw060-F4" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw060/2/m_dsw060f4.png?Expires=2147483647&amp;Signature=J55FgrA749sQth4BVgiQSQk1fy8kIFyjW~BBPMvrgcJQ26mgxOxwJLuMbwmcxZYQWdo5nuGw1iQbNF7YHcNNUhbWz1Jim3Pq~qAi2NRpxDP9Vxz2tCrbr9th-bh0x2LWLklo4tV7ylZpjY3KsmHohO0jujI4LwP6wNbZhHAFt50ZLXTg-mfp~T~UWxvMPuw7KWV9qt2zwMaOgvihBLfZ~iVUVuJqqWbMVeAeoMjviRFbb3vdRKkH0VPUyubiFw79cXOGO0FWXpAJMPdMVn6hdcXHYV3RyI1XWG4-fx9wCCRYlNUzPuV7ibGTjMo79B3VwekVCJllNGICGVUUG264~Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Detection of HELPO2 polymorphisms by the outer/inner and outer/outer strategies. (A) Example of a HELPO2 (Helpo2_VIII_a) polymorphic site in PC15 and PC9 haplotypes shown as an ACT (Artemis Comparison Tool) comparison. Blocks indicate conserved regions (&gt;85% similarity). The HELPO2 element is shown as a white arrow with a grey rectangle inside the arrow representing its encoded RepHel helicase. Vertical black lines show stop codons predicted in the three forward reading frames. (B) and (C) represent the amplification profiles of a polymorphic HELPO2 locus in PC9 (vacant), PC15 (occupied) and N001 (vacant + occupied) subclones using the outer/inner and outer/outer strategies." path-from-xml="dsw060f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055522/dsw060f4.tif" path-from-xml="dsw060f4.tif" target="_blank">View large</a><a section="78055522" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw060/2/dsw060f4.png?Expires=2147483647&amp;Signature=hlo9ixrSrhxyeP0ccm0XGKuuNYbM8CCdPKOazUxv5T27YkFCzbHeELMGgyuZcL-Xbs83hu~odzUQSe46OljJ0poW8Igs4TSofVvUrZ~Ys-7VJnsORbGaRnHEmKVXqT-fa6Ab1ikXHrBNPp60Em9kDqCF7BEZSAZrV99c7u1GD1744iUYNKqNuiYrkTsoXsGSYFhc9zq2gh1qi3-TwkT8Wb4JPWPmrjgrwQFnJ35cz0mWY7qefQYxOHNTYc0mt1td1ybXmtvFWlRrpdZRNotJh8k-Bpo3etio3gpaRv1IcqAevcKGhcZu53w8JpexAgJZmzz4vPCQSAu~KWWKPetSzQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=78055522&amp;ar=2962881&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw060f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Detection of HELPO2 polymorphisms by the outer/inner and outer/outer strategies. (A) Example of a HELPO2 (<em>Helpo2_VIII_a</em>) polymorphic site in PC15 and PC9 haplotypes shown as an ACT (Artemis Comparison Tool) comparison. Blocks indicate conserved regions (&gt;85% similarity). The HELPO2 element is shown as a white arrow with a grey rectangle inside the arrow representing its encoded RepHel helicase. Vertical black lines show stop codons predicted in the three forward reading frames. (B) and (C) represent the amplification profiles of a polymorphic HELPO2 locus in PC9 (vacant), PC15 (occupied) and N001 (vacant + occupied) subclones using the outer/inner and outer/outer strategies.</p></div></div><div data-id="dsw060-F5" class="fig fig-section"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw060/2/m_dsw060f5.png?Expires=2147483647&amp;Signature=OSbBRetE2v8-pS8WiyBOX1oc5akvAunW~8xX5UdlzTe2nBSkaDRUp6G~GiGhYcAYSs~oO5hbrgrPJMNXfV3EPoY2JNuDEa5TH~drdmIk7MCmGyu~Zqm33LtAeScCbKC6EOhGtuzqAM6Zh~rMU395WTxNvFHjmlRjz9-2xnXqvudhCMaFlhFjd-sjpkuTXQo0I3z~59O0zy0uPP8F9Az4zVrlo~BwnM3wpn5r-6CU0DyGZOBwJeOMnzrXbStIOhYycZWYSuteBpl1twTZKW3w0etSbzyf5YWaaGzdcnSQclNxvT-rJhZ8aafTzksMyjc9BbnuLt3FvXLjOru3uqAjEw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Evidence of HELPO2 insertion in the Helpo2_I locus of the PC15-99 subclone. Left (START) and right (END) Helpo2_I boundaries amplified exclusively in PC15-99 subclone (A). Helitron insertion in the PC15-99 subclone is reported by aligning both sequenced PCR products (B and C) to the occupied and empty sites of PC15 v2.0 (Pleos15) and PC9 v1.0 (Pleos9) reference genomes. Lowercase typeface indicates helitron sequences." path-from-xml="dsw060f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055522/dsw060f5.tif" path-from-xml="dsw060f5.tif" target="_blank">View large</a><a section="78055522" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw060/2/dsw060f5.png?Expires=2147483647&amp;Signature=myG0TllGhxKgMKv-8ezQI~2~q5F-zffb~gKq9t9qIr5MSwfMbWLymj8JgZbXhatrDHtQrTpPI-Sq1Qjh8l26EX5WsOu4DyphsIHS~TR6VaqJaYdQPhxAJNP0JvaHqGRQZCBtGla-Ca-UhNGPoVyC9C0i9103YSz6~t1wBtjU2B74JKlnyCpP-ntPTs825kO2e0~XoJMjS70atbWk~2CM9oawxscLDFFXuHhBjLhtsMBA65wQNvX6XBem2KlghuKr6KVQ2IOcKAw0ptRJWuf2K9v92c1TDn5JIzsU3~91kl4AutY-DauSUbOJVBQYKH0DfP1IeZvPDcED9GSEQy8-CQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=78055522&amp;ar=2962881&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw060f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Evidence of HELPO2 insertion in the <em>Helpo2_I</em> locus of the PC15-99 subclone. Left (START) and right (END) <em>Helpo2_I</em> boundaries amplified exclusively in PC15-99 subclone (A). Helitron insertion in the PC15-99 subclone is reported by aligning both sequenced PCR products (B and C) to the occupied and empty sites of PC15 v2.0 (<em>Pleos15</em>) and PC9 v1.0 (<em>Pleos9</em>) reference genomes. Lowercase typeface indicates helitron sequences.</p></div></div><div content-id="dsw060-F5" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw060/2/m_dsw060f5.png?Expires=2147483647&amp;Signature=OSbBRetE2v8-pS8WiyBOX1oc5akvAunW~8xX5UdlzTe2nBSkaDRUp6G~GiGhYcAYSs~oO5hbrgrPJMNXfV3EPoY2JNuDEa5TH~drdmIk7MCmGyu~Zqm33LtAeScCbKC6EOhGtuzqAM6Zh~rMU395WTxNvFHjmlRjz9-2xnXqvudhCMaFlhFjd-sjpkuTXQo0I3z~59O0zy0uPP8F9Az4zVrlo~BwnM3wpn5r-6CU0DyGZOBwJeOMnzrXbStIOhYycZWYSuteBpl1twTZKW3w0etSbzyf5YWaaGzdcnSQclNxvT-rJhZ8aafTzksMyjc9BbnuLt3FvXLjOru3uqAjEw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Evidence of HELPO2 insertion in the Helpo2_I locus of the PC15-99 subclone. Left (START) and right (END) Helpo2_I boundaries amplified exclusively in PC15-99 subclone (A). Helitron insertion in the PC15-99 subclone is reported by aligning both sequenced PCR products (B and C) to the occupied and empty sites of PC15 v2.0 (Pleos15) and PC9 v1.0 (Pleos9) reference genomes. Lowercase typeface indicates helitron sequences." path-from-xml="dsw060f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055522/dsw060f5.tif" path-from-xml="dsw060f5.tif" target="_blank">View large</a><a section="78055522" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw060/2/dsw060f5.png?Expires=2147483647&amp;Signature=myG0TllGhxKgMKv-8ezQI~2~q5F-zffb~gKq9t9qIr5MSwfMbWLymj8JgZbXhatrDHtQrTpPI-Sq1Qjh8l26EX5WsOu4DyphsIHS~TR6VaqJaYdQPhxAJNP0JvaHqGRQZCBtGla-Ca-UhNGPoVyC9C0i9103YSz6~t1wBtjU2B74JKlnyCpP-ntPTs825kO2e0~XoJMjS70atbWk~2CM9oawxscLDFFXuHhBjLhtsMBA65wQNvX6XBem2KlghuKr6KVQ2IOcKAw0ptRJWuf2K9v92c1TDn5JIzsU3~91kl4AutY-DauSUbOJVBQYKH0DfP1IeZvPDcED9GSEQy8-CQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=78055522&amp;ar=2962881&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw060f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Evidence of HELPO2 insertion in the <em>Helpo2_I</em> locus of the PC15-99 subclone. Left (START) and right (END) <em>Helpo2_I</em> boundaries amplified exclusively in PC15-99 subclone (A). Helitron insertion in the PC15-99 subclone is reported by aligning both sequenced PCR products (B and C) to the occupied and empty sites of PC15 v2.0 (<em>Pleos15</em>) and PC9 v1.0 (<em>Pleos9</em>) reference genomes. Lowercase typeface indicates helitron sequences.</p></div></div><div content-id="dsw060-T2" class="table-modal"><div class="table-wrap"><div id="dsw060-T2" data-id="dsw060-T2" class="table-wrap-title"><div class="label -label">Table 2</div><div class="caption"><p>Helitrons described <em>in silico</em> in <em>P. ostreatus</em> PC15 genome assembly v2.0</p></div></div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Locus </th><th>Localization (chromosome<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>a</sup></a>) </th><th>Start (bp) </th><th>End (bp) </th><th>Length (bp) </th></tr></thead><tbody><tr><td><em>Helpo2_I</em> </td><td>I </td><td>619,761 </td><td>626,150 </td><td>6,388 </td></tr><tr><td><em>Helpo2_V</em> </td><td>V </td><td>387,607 </td><td>398,218 </td><td>10,611 </td></tr><tr><td><em>Helpo2_VI</em> </td><td>VI </td><td>1,150,050 </td><td>1,156,438 </td><td>6,388 </td></tr><tr><td><em>Helpo2_VII</em> </td><td>VII </td><td>1,635,256 </td><td>1,641,644 </td><td>6,388 </td></tr><tr><td><em>Helpo2_VIII_a</em> </td><td>VIII </td><td>1367,660 </td><td>1,374,048 </td><td>6,388 </td></tr><tr><td><em>Helpo2_VIII_b</em> </td><td>VIII </td><td>1,722,302 </td><td>1,726,240 </td><td>3,938 </td></tr><tr><td><em>Helpo2_VIII_c</em> </td><td>VIII </td><td>2,234,922 </td><td>2,241,310 </td><td>6,388 </td></tr></tbody></table></div><div content-id="dsw060-T2" class="table-modal"><table><thead name="thead" align=""><tr><th>Locus </th><th>Localization (chromosome<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>a</sup></a>) </th><th>Start (bp) </th><th>End (bp) </th><th>Length (bp) </th></tr></thead><tbody><tr><td><em>Helpo2_I</em> </td><td>I </td><td>619,761 </td><td>626,150 </td><td>6,388 </td></tr><tr><td><em>Helpo2_V</em> </td><td>V </td><td>387,607 </td><td>398,218 </td><td>10,611 </td></tr><tr><td><em>Helpo2_VI</em> </td><td>VI </td><td>1,150,050 </td><td>1,156,438 </td><td>6,388 </td></tr><tr><td><em>Helpo2_VII</em> </td><td>VII </td><td>1,635,256 </td><td>1,641,644 </td><td>6,388 </td></tr><tr><td><em>Helpo2_VIII_a</em> </td><td>VIII </td><td>1367,660 </td><td>1,374,048 </td><td>6,388 </td></tr><tr><td><em>Helpo2_VIII_b</em> </td><td>VIII </td><td>1,722,302 </td><td>1,726,240 </td><td>3,938 </td></tr><tr><td><em>Helpo2_VIII_c</em> </td><td>VIII </td><td>2,234,922 </td><td>2,241,310 </td><td>6,388 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn3"><div class="fn"><p><sup>a</sup>Chromosomes are equivalent to scaffolds in PC15 v2.0 assembly.</p></div></div></div></div></div><div class="table-wrap"><div id="dsw060-T2" data-id="dsw060-T2" class="table-wrap-title"><div class="label -label">Table 2</div><div class="caption"><p>Helitrons described <em>in silico</em> in <em>P. ostreatus</em> PC15 genome assembly v2.0</p></div></div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Locus </th><th>Localization (chromosome<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>a</sup></a>) </th><th>Start (bp) </th><th>End (bp) </th><th>Length (bp) </th></tr></thead><tbody><tr><td><em>Helpo2_I</em> </td><td>I </td><td>619,761 </td><td>626,150 </td><td>6,388 </td></tr><tr><td><em>Helpo2_V</em> </td><td>V </td><td>387,607 </td><td>398,218 </td><td>10,611 </td></tr><tr><td><em>Helpo2_VI</em> </td><td>VI </td><td>1,150,050 </td><td>1,156,438 </td><td>6,388 </td></tr><tr><td><em>Helpo2_VII</em> </td><td>VII </td><td>1,635,256 </td><td>1,641,644 </td><td>6,388 </td></tr><tr><td><em>Helpo2_VIII_a</em> </td><td>VIII </td><td>1367,660 </td><td>1,374,048 </td><td>6,388 </td></tr><tr><td><em>Helpo2_VIII_b</em> </td><td>VIII </td><td>1,722,302 </td><td>1,726,240 </td><td>3,938 </td></tr><tr><td><em>Helpo2_VIII_c</em> </td><td>VIII </td><td>2,234,922 </td><td>2,241,310 </td><td>6,388 </td></tr></tbody></table></div><div content-id="dsw060-T2" class="table-modal"><table><thead name="thead" align=""><tr><th>Locus </th><th>Localization (chromosome<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>a</sup></a>) </th><th>Start (bp) </th><th>End (bp) </th><th>Length (bp) </th></tr></thead><tbody><tr><td><em>Helpo2_I</em> </td><td>I </td><td>619,761 </td><td>626,150 </td><td>6,388 </td></tr><tr><td><em>Helpo2_V</em> </td><td>V </td><td>387,607 </td><td>398,218 </td><td>10,611 </td></tr><tr><td><em>Helpo2_VI</em> </td><td>VI </td><td>1,150,050 </td><td>1,156,438 </td><td>6,388 </td></tr><tr><td><em>Helpo2_VII</em> </td><td>VII </td><td>1,635,256 </td><td>1,641,644 </td><td>6,388 </td></tr><tr><td><em>Helpo2_VIII_a</em> </td><td>VIII </td><td>1367,660 </td><td>1,374,048 </td><td>6,388 </td></tr><tr><td><em>Helpo2_VIII_b</em> </td><td>VIII </td><td>1,722,302 </td><td>1,726,240 </td><td>3,938 </td></tr><tr><td><em>Helpo2_VIII_c</em> </td><td>VIII </td><td>2,234,922 </td><td>2,241,310 </td><td>6,388 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn3"><div class="fn"><p><sup>a</sup>Chromosomes are equivalent to scaffolds in PC15 v2.0 assembly.</p></div></div></div></div><div content-id="dsw060-T3" class="table-modal"><div class="table-wrap"><div id="dsw060-T3" data-id="dsw060-T3" class="table-wrap-title"><div class="label -label">Table 3</div><div class="caption"><p>Molecular validation of HELPO2 polymorphisms using the PCR outer/inner strategy</p></div></div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th colspan="6">Strain<hr /> </th></tr><tr><th>Locus </th><th>PC9-99 </th><th>PC9-14 </th><th>PC15-99 </th><th>PC15-14 </th><th>N001–03 </th><th>N001–14 </th></tr></thead><tbody><tr><td><em>Helpo2_I</em> </td><td>Vacant </td><td>Vacant </td><td><strong>Occupied</strong> </td><td><strong>Vacant</strong> </td><td>Vacant </td><td>Vacant </td></tr><tr><td><em>Helpo2_V</em> </td><td>Vacant </td><td>Vacant </td><td>Occupied </td><td>Occupied </td><td>Occupied </td><td>Occupied </td></tr><tr><td><em>Helpo2_VI</em> </td><td>Vacant </td><td>Vacant </td><td>Occupied </td><td>Occupied </td><td>Occupied </td><td>Occupied </td></tr><tr><td><em>Helpo2_VII</em> </td><td>Vacant </td><td>Vacant </td><td>Occupied </td><td>Occupied </td><td>Occupied </td><td>Occupied </td></tr><tr><td><em>Helpo2_VIII_a</em> </td><td>Vacant </td><td>Vacant </td><td>Occupied </td><td>Occupied </td><td>Occupied </td><td>Occupied </td></tr><tr><td><em>Helpo2_VIII_b</em> </td><td>Vacant </td><td>Vacant </td><td>Occupied </td><td>Occupied </td><td>Occupied </td><td>Occupied </td></tr><tr><td><em>Helpo2_VIII_c</em> </td><td>Vacant </td><td>Vacant </td><td><strong>Occupied<sup>a</sup></strong> </td><td><strong>Vacant</strong> </td><td>Vacant </td><td>Vacant </td></tr></tbody></table></div><div content-id="dsw060-T3" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th colspan="6">Strain<hr /> </th></tr><tr><th>Locus </th><th>PC9-99 </th><th>PC9-14 </th><th>PC15-99 </th><th>PC15-14 </th><th>N001–03 </th><th>N001–14 </th></tr></thead><tbody><tr><td><em>Helpo2_I</em> </td><td>Vacant </td><td>Vacant </td><td><strong>Occupied</strong> </td><td><strong>Vacant</strong> </td><td>Vacant </td><td>Vacant </td></tr><tr><td><em>Helpo2_V</em> </td><td>Vacant </td><td>Vacant </td><td>Occupied </td><td>Occupied </td><td>Occupied </td><td>Occupied </td></tr><tr><td><em>Helpo2_VI</em> </td><td>Vacant </td><td>Vacant </td><td>Occupied </td><td>Occupied </td><td>Occupied </td><td>Occupied </td></tr><tr><td><em>Helpo2_VII</em> </td><td>Vacant </td><td>Vacant </td><td>Occupied </td><td>Occupied </td><td>Occupied </td><td>Occupied </td></tr><tr><td><em>Helpo2_VIII_a</em> </td><td>Vacant </td><td>Vacant </td><td>Occupied </td><td>Occupied </td><td>Occupied </td><td>Occupied </td></tr><tr><td><em>Helpo2_VIII_b</em> </td><td>Vacant </td><td>Vacant </td><td>Occupied </td><td>Occupied </td><td>Occupied </td><td>Occupied </td></tr><tr><td><em>Helpo2_VIII_c</em> </td><td>Vacant </td><td>Vacant </td><td><strong>Occupied<sup>a</sup></strong> </td><td><strong>Vacant</strong> </td><td>Vacant </td><td>Vacant </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn4"><div class="fn"><p>Each term represents an amplification product. Primers and expected product sizes are shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw060/2/dsw060_Supp.zip?Expires=2147483647&amp;Signature=zDAtgxa4F-UL54z2if0eYAx7iiuGb6srTDL1CT7Hx7wDxFEom1ojV1lVEE9Us27ThDolTKyt9Km4jlX7dNK4MHhuRaD32ktxUaSGDSkmnZRKWib00mcJHlAT~5TrnVNjQG5nML7AyU1hHzg~KRGemgvdez0x0d46-pjP6K0JGmfKcntTePu8ATjm8~7UeUJ2ducPfWRDovHNFdYPhH6uJ8yF-~DSkp7EeFNFkhyacpRLpdfEj97SkZCC1qTS06IJaQxL19jmK1BvUZWM6~NYrK7MU-sm57TX44qf0BU5aLXa6LgFrASDYk-iu7SgJew7ClWCHfbFZ8ttSGEcp8o3-g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>.</p></div></div><div content-id="tblfn5"><div class="fn"><p><sup>a</sup>The right flank displayed a higher size than the bioinformatics prediction (2 kb instead of 1.38 kb).</p></div></div></div></div></div><div class="table-wrap"><div id="dsw060-T3" data-id="dsw060-T3" class="table-wrap-title"><div class="label -label">Table 3</div><div class="caption"><p>Molecular validation of HELPO2 polymorphisms using the PCR outer/inner strategy</p></div></div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th colspan="6">Strain<hr /> </th></tr><tr><th>Locus </th><th>PC9-99 </th><th>PC9-14 </th><th>PC15-99 </th><th>PC15-14 </th><th>N001–03 </th><th>N001–14 </th></tr></thead><tbody><tr><td><em>Helpo2_I</em> </td><td>Vacant </td><td>Vacant </td><td><strong>Occupied</strong> </td><td><strong>Vacant</strong> </td><td>Vacant </td><td>Vacant </td></tr><tr><td><em>Helpo2_V</em> </td><td>Vacant </td><td>Vacant </td><td>Occupied </td><td>Occupied </td><td>Occupied </td><td>Occupied </td></tr><tr><td><em>Helpo2_VI</em> </td><td>Vacant </td><td>Vacant </td><td>Occupied </td><td>Occupied </td><td>Occupied </td><td>Occupied </td></tr><tr><td><em>Helpo2_VII</em> </td><td>Vacant </td><td>Vacant </td><td>Occupied </td><td>Occupied </td><td>Occupied </td><td>Occupied </td></tr><tr><td><em>Helpo2_VIII_a</em> </td><td>Vacant </td><td>Vacant </td><td>Occupied </td><td>Occupied </td><td>Occupied </td><td>Occupied </td></tr><tr><td><em>Helpo2_VIII_b</em> </td><td>Vacant </td><td>Vacant </td><td>Occupied </td><td>Occupied </td><td>Occupied </td><td>Occupied </td></tr><tr><td><em>Helpo2_VIII_c</em> </td><td>Vacant </td><td>Vacant </td><td><strong>Occupied<sup>a</sup></strong> </td><td><strong>Vacant</strong> </td><td>Vacant </td><td>Vacant </td></tr></tbody></table></div><div content-id="dsw060-T3" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th colspan="6">Strain<hr /> </th></tr><tr><th>Locus </th><th>PC9-99 </th><th>PC9-14 </th><th>PC15-99 </th><th>PC15-14 </th><th>N001–03 </th><th>N001–14 </th></tr></thead><tbody><tr><td><em>Helpo2_I</em> </td><td>Vacant </td><td>Vacant </td><td><strong>Occupied</strong> </td><td><strong>Vacant</strong> </td><td>Vacant </td><td>Vacant </td></tr><tr><td><em>Helpo2_V</em> </td><td>Vacant </td><td>Vacant </td><td>Occupied </td><td>Occupied </td><td>Occupied </td><td>Occupied </td></tr><tr><td><em>Helpo2_VI</em> </td><td>Vacant </td><td>Vacant </td><td>Occupied </td><td>Occupied </td><td>Occupied </td><td>Occupied </td></tr><tr><td><em>Helpo2_VII</em> </td><td>Vacant </td><td>Vacant </td><td>Occupied </td><td>Occupied </td><td>Occupied </td><td>Occupied </td></tr><tr><td><em>Helpo2_VIII_a</em> </td><td>Vacant </td><td>Vacant </td><td>Occupied </td><td>Occupied </td><td>Occupied </td><td>Occupied </td></tr><tr><td><em>Helpo2_VIII_b</em> </td><td>Vacant </td><td>Vacant </td><td>Occupied </td><td>Occupied </td><td>Occupied </td><td>Occupied </td></tr><tr><td><em>Helpo2_VIII_c</em> </td><td>Vacant </td><td>Vacant </td><td><strong>Occupied<sup>a</sup></strong> </td><td><strong>Vacant</strong> </td><td>Vacant </td><td>Vacant </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn4"><div class="fn"><p>Each term represents an amplification product. Primers and expected product sizes are shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw060/2/dsw060_Supp.zip?Expires=2147483647&amp;Signature=zDAtgxa4F-UL54z2if0eYAx7iiuGb6srTDL1CT7Hx7wDxFEom1ojV1lVEE9Us27ThDolTKyt9Km4jlX7dNK4MHhuRaD32ktxUaSGDSkmnZRKWib00mcJHlAT~5TrnVNjQG5nML7AyU1hHzg~KRGemgvdez0x0d46-pjP6K0JGmfKcntTePu8ATjm8~7UeUJ2ducPfWRDovHNFdYPhH6uJ8yF-~DSkp7EeFNFkhyacpRLpdfEj97SkZCC1qTS06IJaQxL19jmK1BvUZWM6~NYrK7MU-sm57TX44qf0BU5aLXa6LgFrASDYk-iu7SgJew7ClWCHfbFZ8ttSGEcp8o3-g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>.</p></div></div><div content-id="tblfn5"><div class="fn"><p><sup>a</sup>The right flank displayed a higher size than the bioinformatics prediction (2 kb instead of 1.38 kb).</p></div></div></div></div><div content-id="dsw060-T4" class="table-modal"><div class="table-wrap"><div id="dsw060-T4" data-id="dsw060-T4" class="table-wrap-title"><div class="label -label">Table 4</div><div class="caption"><p>Molecular validation of HELPO2 polymorphisms using the PCR outer/outer strategy</p></div></div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th colspan="6">Strain<hr /> </th></tr><tr><th>Locus </th><th>PC9-99 </th><th>PC9-14 </th><th>PC15-99 </th><th>PC15-14 </th><th>N001-03 </th><th>N001-14 </th></tr></thead><tbody><tr><td><em>Helpo2_I</em> </td><td>Vacant </td><td>Vacant </td><td><strong>Vacant</strong> </td><td>Vacant </td><td>Vacant </td><td>Vacant </td></tr><tr><td><em>Helpo2_V</em> </td><td>Vacant </td><td>Vacant </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td><em>Helpo2_VI</em> </td><td>Vacant </td><td>Vacant </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td><em>Helpo2_VII</em> </td><td>N/A </td><td>N/A </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td><em>Helpo2_VIII_a</em> </td><td>Vacant </td><td>Vacant </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td><em>Helpo2_VIII_b</em> </td><td>Vacant </td><td>Vacant </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td><em>Helpo2_VIII_c</em> </td><td>Vacant </td><td>Vacant </td><td><strong>Vacant</strong> </td><td>Vacant </td><td>Vacant </td><td>Vacant </td></tr></tbody></table></div><div content-id="dsw060-T4" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th colspan="6">Strain<hr /> </th></tr><tr><th>Locus </th><th>PC9-99 </th><th>PC9-14 </th><th>PC15-99 </th><th>PC15-14 </th><th>N001-03 </th><th>N001-14 </th></tr></thead><tbody><tr><td><em>Helpo2_I</em> </td><td>Vacant </td><td>Vacant </td><td><strong>Vacant</strong> </td><td>Vacant </td><td>Vacant </td><td>Vacant </td></tr><tr><td><em>Helpo2_V</em> </td><td>Vacant </td><td>Vacant </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td><em>Helpo2_VI</em> </td><td>Vacant </td><td>Vacant </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td><em>Helpo2_VII</em> </td><td>N/A </td><td>N/A </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td><em>Helpo2_VIII_a</em> </td><td>Vacant </td><td>Vacant </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td><em>Helpo2_VIII_b</em> </td><td>Vacant </td><td>Vacant </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td><em>Helpo2_VIII_c</em> </td><td>Vacant </td><td>Vacant </td><td><strong>Vacant</strong> </td><td>Vacant </td><td>Vacant </td><td>Vacant </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn6"><div class="fn"><p>Each term represents an amplification product. Cells in normal typeface show results in agreement with the outer/inner strategy, and in bold typeface result in disagreement. Grey cells represent occupied loci previously validated by the outer/inner strategy. Primers and expected product sizes are shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw060/2/dsw060_Supp.zip?Expires=2147483647&amp;Signature=zDAtgxa4F-UL54z2if0eYAx7iiuGb6srTDL1CT7Hx7wDxFEom1ojV1lVEE9Us27ThDolTKyt9Km4jlX7dNK4MHhuRaD32ktxUaSGDSkmnZRKWib00mcJHlAT~5TrnVNjQG5nML7AyU1hHzg~KRGemgvdez0x0d46-pjP6K0JGmfKcntTePu8ATjm8~7UeUJ2ducPfWRDovHNFdYPhH6uJ8yF-~DSkp7EeFNFkhyacpRLpdfEj97SkZCC1qTS06IJaQxL19jmK1BvUZWM6~NYrK7MU-sm57TX44qf0BU5aLXa6LgFrASDYk-iu7SgJew7ClWCHfbFZ8ttSGEcp8o3-g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>.</p></div></div><div content-id="tblfn7"><div class="fn"><p>N/A, no amplification.</p></div></div></div></div></div><div class="table-wrap"><div id="dsw060-T4" data-id="dsw060-T4" class="table-wrap-title"><div class="label -label">Table 4</div><div class="caption"><p>Molecular validation of HELPO2 polymorphisms using the PCR outer/outer strategy</p></div></div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th colspan="6">Strain<hr /> </th></tr><tr><th>Locus </th><th>PC9-99 </th><th>PC9-14 </th><th>PC15-99 </th><th>PC15-14 </th><th>N001-03 </th><th>N001-14 </th></tr></thead><tbody><tr><td><em>Helpo2_I</em> </td><td>Vacant </td><td>Vacant </td><td><strong>Vacant</strong> </td><td>Vacant </td><td>Vacant </td><td>Vacant </td></tr><tr><td><em>Helpo2_V</em> </td><td>Vacant </td><td>Vacant </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td><em>Helpo2_VI</em> </td><td>Vacant </td><td>Vacant </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td><em>Helpo2_VII</em> </td><td>N/A </td><td>N/A </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td><em>Helpo2_VIII_a</em> </td><td>Vacant </td><td>Vacant </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td><em>Helpo2_VIII_b</em> </td><td>Vacant </td><td>Vacant </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td><em>Helpo2_VIII_c</em> </td><td>Vacant </td><td>Vacant </td><td><strong>Vacant</strong> </td><td>Vacant </td><td>Vacant </td><td>Vacant </td></tr></tbody></table></div><div content-id="dsw060-T4" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th colspan="6">Strain<hr /> </th></tr><tr><th>Locus </th><th>PC9-99 </th><th>PC9-14 </th><th>PC15-99 </th><th>PC15-14 </th><th>N001-03 </th><th>N001-14 </th></tr></thead><tbody><tr><td><em>Helpo2_I</em> </td><td>Vacant </td><td>Vacant </td><td><strong>Vacant</strong> </td><td>Vacant </td><td>Vacant </td><td>Vacant </td></tr><tr><td><em>Helpo2_V</em> </td><td>Vacant </td><td>Vacant </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td><em>Helpo2_VI</em> </td><td>Vacant </td><td>Vacant </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td><em>Helpo2_VII</em> </td><td>N/A </td><td>N/A </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td><em>Helpo2_VIII_a</em> </td><td>Vacant </td><td>Vacant </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td><em>Helpo2_VIII_b</em> </td><td>Vacant </td><td>Vacant </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td><em>Helpo2_VIII_c</em> </td><td>Vacant </td><td>Vacant </td><td><strong>Vacant</strong> </td><td>Vacant </td><td>Vacant </td><td>Vacant </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn6"><div class="fn"><p>Each term represents an amplification product. Cells in normal typeface show results in agreement with the outer/inner strategy, and in bold typeface result in disagreement. Grey cells represent occupied loci previously validated by the outer/inner strategy. Primers and expected product sizes are shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw060/2/dsw060_Supp.zip?Expires=2147483647&amp;Signature=zDAtgxa4F-UL54z2if0eYAx7iiuGb6srTDL1CT7Hx7wDxFEom1ojV1lVEE9Us27ThDolTKyt9Km4jlX7dNK4MHhuRaD32ktxUaSGDSkmnZRKWib00mcJHlAT~5TrnVNjQG5nML7AyU1hHzg~KRGemgvdez0x0d46-pjP6K0JGmfKcntTePu8ATjm8~7UeUJ2ducPfWRDovHNFdYPhH6uJ8yF-~DSkp7EeFNFkhyacpRLpdfEj97SkZCC1qTS06IJaQxL19jmK1BvUZWM6~NYrK7MU-sm57TX44qf0BU5aLXa6LgFrASDYk-iu7SgJew7ClWCHfbFZ8ttSGEcp8o3-g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>.</p></div></div><div content-id="tblfn7"><div class="fn"><p>N/A, no amplification.</p></div></div></div></div></div>                            <h3 scrollto-destination=78055523 id="78055523" class="section-title" >3.4. Distorted segregation patterns of HELPO2 in the N001 meiotic-derived progeny</h3>
<div class="block-child-p">We investigated the segregation patterns of the seven HELPO2 loci in 68 monokaryons using the previously described outer/inner strategy. As N001 is a dikaryon formed by the mating of PC15 and PC9 monokaryons and due to the seven loci being polymorphic (the presence of helitrons in PC15 vs absence of helitrons in PC9), it should be expected that there is a proportion of 1:1 (presence vs absence of helitrons) in the progeny. In contrast, the results revealed that the inheritance of HELPO2 elements was clearly distorted (<a reveal-id="dsw060-F6" data-open="dsw060-F6" class="link link-reveal link-table xref-fig">Fig. 6</a>). First of all, three of the seven loci carrying a HELPO2 in the PC15 strain (<em>Helpo2_I, Helpo2_VII</em> and <em>Helpo2_VIII_c</em>) were completely absent in the progeny. The fact that <em>Helpo2_I</em> and <em>Helpo2_VIII_c</em> were present in the PC15-99 subclone but absent in the PC15-14, N001-03, N001-14 and the whole N001 progeny (68 mK) reinforces the hypothesis of its unique (and recent) transposition in PC15-99, which according to this result, would have taken place after N001 de-dikaryotization (1999) but before PC15 genome sequencing (2005), as both loci carry HELPO2 elements in the PC15 v2.0 assembly. This observation demonstrated that these loci were vacant in the original PC15 and PC9 strains that mated in the wild to form N001. Following similar reasoning, a HELPO2 element must have been inserted in <em>Helpo2_VII</em> in the PC15 nucleus of the N001 dikaryon in the time that elapsed between the isolation of the meiotic progeny (in 1994 all monokaryons carry a vacant locus) and N001 de-dikaryotization (1999, PC15 carries an occupied locus; <a reveal-id="dsw060-F2" data-open="dsw060-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>). Furthermore, PC9 helitron (vacant) alleles were consistently overrepresented in a ratio close to 3:1 in the remaining four loci. These results are in concordance with the biased distribution of HELPO2 elements previously found in the monokaryotic collection by qPCR (<a reveal-id="dsw060-F1" data-open="dsw060-F1" class="link link-reveal link-table xref-fig">Fig. 1B</a>). This scenario suggests the occurrence of a preferential selection of PC9 helitron (vacant) alleles during meiosis, or, in other words, an elimination of helitrons. To rule out the possibility that such a finding was derived from an artificially skewed progeny, we revisited the <em>P. ostreatus</em> genetic linkage map. This map was constructed based on the segregation of 214 molecular markers in a population of 80 sibling monokaryons meiotically derived from N001,<a reveal-id="dsw060-B26" data-open="dsw060-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> and 68 of them were used in this study. According to the genetic map, 86% of the markers followed the expected 1:1 ratio. Additionally, the segregation of genetic markers surrounding HELPO2 loci did not show distorted segregation. Taking together, these data reinforced the fact that the segregation of HELPO2 elements is selectively skewed with respect to the expected Mendelian segregation. <div data-id="dsw060-F6" class="fig fig-section"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw060/2/m_dsw060f6.png?Expires=2147483647&Signature=HEoeliIRGmqqDJyqFGWbgmlsPEH-KZP0pPrQ0IMk-u4WZtYT6YlUE9rqe5T2IKlXEPvqzbHmTD7Cxr6GZOjdBtDlff8GfLI~J~U4786iS0V4vaVFtpTrBVwDAMEoGBCleYnAiBi16dZX33U9aJ60hRbz7R5ATLvUfiEaBCRnGNO5aA0pbkbv15t51NDJSsBE9Vo0MIvlATMjod5bUSlJuMDyYcpJYz9kZkIi~k9TwnqtBPxY3z8Yp3~N2-6ILE7eAO35fXytG9C58E9bI9Tetrkxfv17ZMxzEVVEFIhlGg6q4c507wlO4CX31C4DUo1oRo~iG4pjs4XNScaa2io0jw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="HELPO2 segregation rates in the 68-mK monokaryotic progeny. Distribution of HELPO2 helitrons in a meiotic progeny of 68 individuals is reported as stacked histograms. The seven HELPO2 loci are shown on the X-axis. The Y-axis indicates the number of monokaryons. Black and grey bars show the number of monokaryons presenting occupied and vacant loci, respectively. The frequency of helitron presence per locus is shown above each bar." path-from-xml="dsw060f6.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055524/dsw060f6.tif" path-from-xml="dsw060f6.tif" target="_blank">View large</a><a section="78055524" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw060/2/dsw060f6.png?Expires=2147483647&Signature=gQorZDeLcaRJ1s5cldy5Mez86wcaTCUxV-9hwrVuLj8YVkCp4SZgq2NnBtgXL6tg02xeI5P1N3CuM10s1vuzBWe5mZh2jC5ho3bA2fCXNh16zGdVVCmhJccjv8vujNOkDiyNlyfulisJVRfBfUcBhFQJcY8iqMilX~NRAH575GP1Dp2htDDHaxQ~Sx3jjHu7yLt9U2GJsXrkTdvRMFLEXVdj1Ehe4lUHAkHviOb5aEE1dseV-y2nxLgPOVOLth~eEaxCcNlQJAb9q5o6acN2xE6qMRwuhsg9AbgiuPsvaQk1XorkOkKwV3EZ9JzRSGEj8pHopSpZYKl06Z461KOqmA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=78055524&ar=2962881&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw060f6.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>HELPO2 segregation rates in the 68-mK monokaryotic progeny. Distribution of HELPO2 helitrons in a meiotic progeny of 68 individuals is reported as stacked histograms. The seven HELPO2 loci are shown on the X-axis. The Y-axis indicates the number of monokaryons. Black and grey bars show the number of monokaryons presenting occupied and vacant loci, respectively. The frequency of helitron presence per locus is shown above each bar.</p></div></div><div content-id="dsw060-F6" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw060/2/m_dsw060f6.png?Expires=2147483647&Signature=HEoeliIRGmqqDJyqFGWbgmlsPEH-KZP0pPrQ0IMk-u4WZtYT6YlUE9rqe5T2IKlXEPvqzbHmTD7Cxr6GZOjdBtDlff8GfLI~J~U4786iS0V4vaVFtpTrBVwDAMEoGBCleYnAiBi16dZX33U9aJ60hRbz7R5ATLvUfiEaBCRnGNO5aA0pbkbv15t51NDJSsBE9Vo0MIvlATMjod5bUSlJuMDyYcpJYz9kZkIi~k9TwnqtBPxY3z8Yp3~N2-6ILE7eAO35fXytG9C58E9bI9Tetrkxfv17ZMxzEVVEFIhlGg6q4c507wlO4CX31C4DUo1oRo~iG4pjs4XNScaa2io0jw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="HELPO2 segregation rates in the 68-mK monokaryotic progeny. Distribution of HELPO2 helitrons in a meiotic progeny of 68 individuals is reported as stacked histograms. The seven HELPO2 loci are shown on the X-axis. The Y-axis indicates the number of monokaryons. Black and grey bars show the number of monokaryons presenting occupied and vacant loci, respectively. The frequency of helitron presence per locus is shown above each bar." path-from-xml="dsw060f6.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055524/dsw060f6.tif" path-from-xml="dsw060f6.tif" target="_blank">View large</a><a section="78055524" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw060/2/dsw060f6.png?Expires=2147483647&Signature=gQorZDeLcaRJ1s5cldy5Mez86wcaTCUxV-9hwrVuLj8YVkCp4SZgq2NnBtgXL6tg02xeI5P1N3CuM10s1vuzBWe5mZh2jC5ho3bA2fCXNh16zGdVVCmhJccjv8vujNOkDiyNlyfulisJVRfBfUcBhFQJcY8iqMilX~NRAH575GP1Dp2htDDHaxQ~Sx3jjHu7yLt9U2GJsXrkTdvRMFLEXVdj1Ehe4lUHAkHviOb5aEE1dseV-y2nxLgPOVOLth~eEaxCcNlQJAb9q5o6acN2xE6qMRwuhsg9AbgiuPsvaQk1XorkOkKwV3EZ9JzRSGEj8pHopSpZYKl06Z461KOqmA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=78055524&ar=2962881&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw060f6.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>HELPO2 segregation rates in the 68-mK monokaryotic progeny. Distribution of HELPO2 helitrons in a meiotic progeny of 68 individuals is reported as stacked histograms. The seven HELPO2 loci are shown on the X-axis. The Y-axis indicates the number of monokaryons. Black and grey bars show the number of monokaryons presenting occupied and vacant loci, respectively. The frequency of helitron presence per locus is shown above each bar.</p></div></div></div>                            <h3 scrollto-destination=78055525 id="78055525" class="section-title" >3.5. Somatic transposition of HELPO2 generates mosaicism in <em>P. ostreatus</em></h3>
<div class="block-child-p">The transposition of an autonomous HELPO2 element to a new chromosomal location in PC15-99 (i.e. <em>Helpo2_I</em>, chromosome I, position 619,761-626,150) was validated by two PCR strategies and reinforced by the presence of a vacant locus in the other strains/subclones, including the monokaryotic progeny. We integrated these results in the reconstruction of the pedigree of <em>P. ostreatus’</em> strains maintained under different subculture conditions to propose a time-scale model that could explain the HELPO2 transposition in PC15-99 and the current HELPO2 profiles obtained in all the strains under study (<a reveal-id="dsw060-F7" data-open="dsw060-F7" class="link link-reveal link-table xref-fig">Fig. 7</a>). After 1999, a somatic transposition of the helitron HELPO2 took place in the nucleus of a cell of a PC15 culture plate and was integrated in the <em>Helpo2_I</em> locus by a copy and paste mechanism that could be triggered by the stress of the de-dikaryotization process. As a result, the transposition event originated a new HELPO2 copy in such a cell, which after mitotic replication led to the occurrence of mosaicism in the PC15 mycelia, displaying <em>Helpo2_I</em>+ (occupied) and <em>Helpo2_I</em>- (vacant) cells. Thus, subsequent subclones derived from the PC15 strain might have been originated from agar plugs located in different sections of the mycelium plate, thus presenting polymorphic loci. We hypothesize that the PC15-14 sample maintained during 20 years under high subculture frequency and used for routine laboratory work could have derived from a section of the PC15 <em>Helpo2_I</em>- cells. Alternatively, another likely explanation could be that <em>Helpo2_I</em>+ cells were also sampled but lost in the PC15-14 subclone after multiple subculturing replications due to lower fitness in comparison to <em>Helpo2_I</em>- or simply by the result of random drift. In contrast, PC15-99 still maintains the phenotype of a genetic mosaic. In this sense, the minimal subculturing frequency of this subclone likely favoured the preservation of both <em>Helpo2_I</em>+ and <em>Helpo2_I</em>- cells in the same mycelium net. The presence of mosaicism in basidiomycetes has been widely studied in <em>Armillaria gallica</em>, where variable numbers of nuclei genotypes have been detected in single fruiting bodies collected for decades.<a reveal-id="dsw060-B41" data-open="dsw060-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a> In this sense, it has been described that among-cell-line genetic variation in the haploid mycelium of <em>A. gallica</em> may confer plasticity to adapt to changing environmental conditions.<a reveal-id="dsw060-B42" data-open="dsw060-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a><div data-id="dsw060-F7" class="fig fig-section"><div class="label fig-label">Figure 7</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw060/2/m_dsw060f7.png?Expires=2147483647&Signature=CfBOfaink2aSNI1vWPUxRNmnQUaZNf-BhzYKXERXyhx7Ag58o6FB4UyDin8XmgKUWPW20bIGkqWKss~1e5ondPXIbuPVOpeTWD1pCKUdfuHC1RqgbkYVi1v2EIobBYrGAd~tDo3JPX-xGEhStI2XY5SjSGnzBw47MRuRBsAxzmQ88S0vUGMswRGyszFqz2eqjm5RvFTh5bFiEuy~cqFoLiIL7dhD-2wL~LU8Q9HPUCCcedPRwIKnEjgYxHrvaxKnhtQmcP8Fg4XEKaVrZzR1czFZjkuEEN~64-g19C~joZCpscVkhg2sg92mX5ouGlelBefEXlqAvgviRnLDi4lnGA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Chronological model explaining HELPO2 somatic insertions in the PC15-99 subclone. (a) (1994) N001-94 parental strain displays 4 helitron loci (Helpo2_V, Helpo2_VI, Helpo2_VIII_a and Helpo2_VIII_b), which are transmitted to the meiotic progeny 68 mK. (b) (1999). After de-dikaryotization, a HELPO2 from one of the four loci mobilizes in a cell of PC15 and integrates into a new locus (i.e. Helpo2_I), producing a mosaic mycelium displaying both occupied (Helpo2_I+) and vacant (Helpo2_I) cells (dashed arrow indicates HELPO2 helitron transposition and subsequent integration). (c) (1999–2014) A PC15 subclone presenting mosaicism is deposited in the CECT and maintained under low subculture frequency up to 2014 (PC15-99). (d) (2005) A PC15 subclone carrying Helpo2_I+ cells is used for whole genome sequencing by JGI. (e) (1999–2014) PC15 subclone used for routine laboratory work up to 2014 (PC15-14) is derived from a section of PC15 Helpo2_I- cells. Alternatively, Helpo2_I + cells were lost by random drift as a consequence of the high subculture frequency." path-from-xml="dsw060f7.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055526/dsw060f7.tif" path-from-xml="dsw060f7.tif" target="_blank">View large</a><a section="78055526" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw060/2/dsw060f7.png?Expires=2147483647&Signature=Xb-Ts4oZW2mtqs73PMfieu5m-PWrBK4Q6TR6rSB1L251liPtEAfQZv2xAOBlZQExChJLqahoUvIywg0g3ilFriprq6afd3PT7Osfl6ke6oncM~LyDA-DbvFpsVq7u6lDN8sXaStnFSGThmfgR4k5QTtQTEAm0fV4BG4rxnGe-5dG4fkigrCmoux8hwU0Qi8iLVRpG1NJM6RyxfsMitZmihStP1kgcRRHEUERBR0J3pwa5XHXPSvGtRoHaotMcRxbsxcSdMo8u1DNKDCJm9HHm1eoUnhmyPix0VVG3n204~rjLtYaDYnAegyBBNS7KKi~fI3kEpXb1yOaGhltPytDbQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=78055526&ar=2962881&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw060f7.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Chronological model explaining HELPO2 somatic insertions in the PC15-99 subclone. (a) (1994) N001-94 parental strain displays <a reveal-id="dsw060-B4" data-open="dsw060-B4" class="link link-ref link-reveal xref-bibr">4</a> helitron loci (<em>Helpo</em><a reveal-id="dsw060-B2" data-open="dsw060-B2" class="link link-ref link-reveal xref-bibr">2</a>,<em>_V</em>, <em>Helpo</em><a reveal-id="dsw060-B2" data-open="dsw060-B2" class="link link-ref link-reveal xref-bibr">2</a>,<em>_VI</em>, <em>Helpo</em><a reveal-id="dsw060-B2" data-open="dsw060-B2" class="link link-ref link-reveal xref-bibr">2</a>,<em>_VIII_a</em> and <em>Helpo</em><a reveal-id="dsw060-B2" data-open="dsw060-B2" class="link link-ref link-reveal xref-bibr">2</a>,<em>_VIII_b</em>), which are transmitted to the meiotic progeny 68 mK. (b) (1999). After de-dikaryotization, a HELPO2 from one of the four loci mobilizes in a cell of PC15 and integrates into a new locus (i.e. <em>Helpo</em><a reveal-id="dsw060-B2" data-open="dsw060-B2" class="link link-ref link-reveal xref-bibr">2</a>,<em>_I</em>), producing a mosaic mycelium displaying both occupied (<em>Helpo</em><a reveal-id="dsw060-B2" data-open="dsw060-B2" class="link link-ref link-reveal xref-bibr">2</a>,<em>_I+)</em> and vacant (<em>Helpo</em><a reveal-id="dsw060-B2" data-open="dsw060-B2" class="link link-ref link-reveal xref-bibr">2</a>,<em>_I)</em> cells (dashed arrow indicates HELPO2 helitron transposition and subsequent integration). (c) (1999–2014) A PC15 subclone presenting mosaicism is deposited in the CECT and maintained under low subculture frequency up to 2014 (PC15-99). (d) (2005) A PC15 subclone carrying <em>Helpo</em><a reveal-id="dsw060-B2" data-open="dsw060-B2" class="link link-ref link-reveal xref-bibr">2</a>,<em>_I+ </em>cells is used for whole genome sequencing by JGI. (e) (1999–2014) PC15 subclone used for routine laboratory work up to 2014 (PC15-14) is derived from a section of PC15 <em>Helpo</em><a reveal-id="dsw060-B2" data-open="dsw060-B2" class="link link-ref link-reveal xref-bibr">2</a>,<em>_I-</em> cells. Alternatively, <em>Helpo</em><a reveal-id="dsw060-B2" data-open="dsw060-B2" class="link link-ref link-reveal xref-bibr">2</a><em>_I + </em>cells were lost by random drift as a consequence of the high subculture frequency.</p></div></div><div content-id="dsw060-F7" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 7</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw060/2/m_dsw060f7.png?Expires=2147483647&Signature=CfBOfaink2aSNI1vWPUxRNmnQUaZNf-BhzYKXERXyhx7Ag58o6FB4UyDin8XmgKUWPW20bIGkqWKss~1e5ondPXIbuPVOpeTWD1pCKUdfuHC1RqgbkYVi1v2EIobBYrGAd~tDo3JPX-xGEhStI2XY5SjSGnzBw47MRuRBsAxzmQ88S0vUGMswRGyszFqz2eqjm5RvFTh5bFiEuy~cqFoLiIL7dhD-2wL~LU8Q9HPUCCcedPRwIKnEjgYxHrvaxKnhtQmcP8Fg4XEKaVrZzR1czFZjkuEEN~64-g19C~joZCpscVkhg2sg92mX5ouGlelBefEXlqAvgviRnLDi4lnGA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Chronological model explaining HELPO2 somatic insertions in the PC15-99 subclone. (a) (1994) N001-94 parental strain displays 4 helitron loci (Helpo2_V, Helpo2_VI, Helpo2_VIII_a and Helpo2_VIII_b), which are transmitted to the meiotic progeny 68 mK. (b) (1999). After de-dikaryotization, a HELPO2 from one of the four loci mobilizes in a cell of PC15 and integrates into a new locus (i.e. Helpo2_I), producing a mosaic mycelium displaying both occupied (Helpo2_I+) and vacant (Helpo2_I) cells (dashed arrow indicates HELPO2 helitron transposition and subsequent integration). (c) (1999–2014) A PC15 subclone presenting mosaicism is deposited in the CECT and maintained under low subculture frequency up to 2014 (PC15-99). (d) (2005) A PC15 subclone carrying Helpo2_I+ cells is used for whole genome sequencing by JGI. (e) (1999–2014) PC15 subclone used for routine laboratory work up to 2014 (PC15-14) is derived from a section of PC15 Helpo2_I- cells. Alternatively, Helpo2_I + cells were lost by random drift as a consequence of the high subculture frequency." path-from-xml="dsw060f7.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055526/dsw060f7.tif" path-from-xml="dsw060f7.tif" target="_blank">View large</a><a section="78055526" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw060/2/dsw060f7.png?Expires=2147483647&Signature=Xb-Ts4oZW2mtqs73PMfieu5m-PWrBK4Q6TR6rSB1L251liPtEAfQZv2xAOBlZQExChJLqahoUvIywg0g3ilFriprq6afd3PT7Osfl6ke6oncM~LyDA-DbvFpsVq7u6lDN8sXaStnFSGThmfgR4k5QTtQTEAm0fV4BG4rxnGe-5dG4fkigrCmoux8hwU0Qi8iLVRpG1NJM6RyxfsMitZmihStP1kgcRRHEUERBR0J3pwa5XHXPSvGtRoHaotMcRxbsxcSdMo8u1DNKDCJm9HHm1eoUnhmyPix0VVG3n204~rjLtYaDYnAegyBBNS7KKi~fI3kEpXb1yOaGhltPytDbQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=78055526&ar=2962881&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw060f7.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Chronological model explaining HELPO2 somatic insertions in the PC15-99 subclone. (a) (1994) N001-94 parental strain displays <a reveal-id="dsw060-B4" data-open="dsw060-B4" class="link link-ref link-reveal xref-bibr">4</a> helitron loci (<em>Helpo</em><a reveal-id="dsw060-B2" data-open="dsw060-B2" class="link link-ref link-reveal xref-bibr">2</a>,<em>_V</em>, <em>Helpo</em><a reveal-id="dsw060-B2" data-open="dsw060-B2" class="link link-ref link-reveal xref-bibr">2</a>,<em>_VI</em>, <em>Helpo</em><a reveal-id="dsw060-B2" data-open="dsw060-B2" class="link link-ref link-reveal xref-bibr">2</a>,<em>_VIII_a</em> and <em>Helpo</em><a reveal-id="dsw060-B2" data-open="dsw060-B2" class="link link-ref link-reveal xref-bibr">2</a>,<em>_VIII_b</em>), which are transmitted to the meiotic progeny 68 mK. (b) (1999). After de-dikaryotization, a HELPO2 from one of the four loci mobilizes in a cell of PC15 and integrates into a new locus (i.e. <em>Helpo</em><a reveal-id="dsw060-B2" data-open="dsw060-B2" class="link link-ref link-reveal xref-bibr">2</a>,<em>_I</em>), producing a mosaic mycelium displaying both occupied (<em>Helpo</em><a reveal-id="dsw060-B2" data-open="dsw060-B2" class="link link-ref link-reveal xref-bibr">2</a>,<em>_I+)</em> and vacant (<em>Helpo</em><a reveal-id="dsw060-B2" data-open="dsw060-B2" class="link link-ref link-reveal xref-bibr">2</a>,<em>_I)</em> cells (dashed arrow indicates HELPO2 helitron transposition and subsequent integration). (c) (1999–2014) A PC15 subclone presenting mosaicism is deposited in the CECT and maintained under low subculture frequency up to 2014 (PC15-99). (d) (2005) A PC15 subclone carrying <em>Helpo</em><a reveal-id="dsw060-B2" data-open="dsw060-B2" class="link link-ref link-reveal xref-bibr">2</a>,<em>_I+ </em>cells is used for whole genome sequencing by JGI. (e) (1999–2014) PC15 subclone used for routine laboratory work up to 2014 (PC15-14) is derived from a section of PC15 <em>Helpo</em><a reveal-id="dsw060-B2" data-open="dsw060-B2" class="link link-ref link-reveal xref-bibr">2</a>,<em>_I-</em> cells. Alternatively, <em>Helpo</em><a reveal-id="dsw060-B2" data-open="dsw060-B2" class="link link-ref link-reveal xref-bibr">2</a><em>_I + </em>cells were lost by random drift as a consequence of the high subculture frequency.</p></div></div></div>                            <h3 scrollto-destination=78055527 id="78055527" class="section-title" >3.6. Allelic gene conversion may contribute to the elimination of HELPO2 helitrons from <em>P. ostreatus</em> progeny</h3>
<div class="block-child-p">As was previously described, the PCR strategies leading to an understanding of the inheritance patterns of the HELPO2 family in the progeny of N001 uncovered an over-transmission of the HELPO2- vacant alleles from the parental PC9. To understand if such phenomena were derived from a preferential inheritance or by helitron excisions during meiosis, we proceeded as follows: <em>Helpo2_VI</em> and <em>Helpo2_VIII_b</em> loci were chosen on the basis of their clear distortion segregation ratios. All the monokaryons carrying vacant sites according to the lack of amplification by the outer/inner strategy (58 of 68 for <em>Helpo2_VI</em> and 55 of 68 for <em>Helpo2_VIII_b</em>) were subjected to additional amplification reactions using the primers shown in <a reveal-id="dsw060-F8" data-open="dsw060-F8" class="link link-reveal link-table xref-fig">Fig. 8A and B</a>. The objectives were as follows: (i) to validate the presence of vacant sites with a positive PCR product and (ii) to determine the parental (PC9 or PC15) origin of such vacant sites as well as that of the adjacent regions (i.e. if they were inherited from the parental PC15 or PC9). For this purpose, primers were designed to amplify homologous regions of the parental strains carrying small indels and single nucleotide polymorphisms (SNPs), which allowed us to identify their PC9 or PC15 origin (primer sequences shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw060/2/dsw060_Supp.zip?Expires=2147483647&amp;Signature=zDAtgxa4F-UL54z2if0eYAx7iiuGb6srTDL1CT7Hx7wDxFEom1ojV1lVEE9Us27ThDolTKyt9Km4jlX7dNK4MHhuRaD32ktxUaSGDSkmnZRKWib00mcJHlAT~5TrnVNjQG5nML7AyU1hHzg~KRGemgvdez0x0d46-pjP6K0JGmfKcntTePu8ATjm8~7UeUJ2ducPfWRDovHNFdYPhH6uJ8yF-~DSkp7EeFNFkhyacpRLpdfEj97SkZCC1qTS06IJaQxL19jmK1BvUZWM6~NYrK7MU-sm57TX44qf0BU5aLXa6LgFrASDYk-iu7SgJew7ClWCHfbFZ8ttSGEcp8o3-g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S4 and S5</a></span>). For each locus, the four amplification products shown in <a reveal-id="dsw060-F8" data-open="dsw060-F8" class="link link-reveal link-table xref-fig">Fig. 8A and B</a> were sequenced and aligned to both PC15 v2.0 and PC9 v1.0 assemblies. SNPs were manually identified in the alignments and used to assign their parental origin. In 94% of the monokaryons analysed, the four regions studied were inherited from PC9, pointing to a preferential inheritance of the vacant site instead of a HELPO2 excision during meiosis. Nevertheless, a surprising profile was found in seven monokaryons (four in <em>Helpo2_VI</em> locus and three in <em>Helpo2_VIII_b</em> locus). Although the three flanking sites were inherited from the PC15 parental strain, SNPs of the amplification product corresponding to the vacant site (1,489 bp in <em>Helpo2_VI</em> and 907 bp in <em>Helpo2_VIII_b</em>) indicated that they were transmitted from the parental PC9 (<a reveal-id="dsw060-F8" data-open="dsw060-F8" class="link link-reveal link-table xref-fig">Fig. 8A and B</a>). These results might be explained by a mechanism of allelic gene conversion, which results in the loss of HELPO2 helitrons. To verify this hypothesis, the presence of gene conversion junctions were validated in the seven monokaryons by PCR as shown in <a reveal-id="dsw060-F8" data-open="dsw060-F8" class="link link-reveal link-table xref-fig">Fig. 8C</a>. To ensure the linearity of the short regions located between the amplified regions (shown in <a reveal-id="dsw060-F8" data-open="dsw060-F8" class="link link-reveal link-table xref-fig">Fig. 8A and B</a> as white rectangles), primers were designed to amplify longer regions, spanning <em>Helpo2_VI</em> and <em>Helpo2_VIII_b</em> loci as well as their flanking sites (<a reveal-id="dsw060-F8" data-open="dsw060-F8" class="link link-reveal link-table xref-fig">Fig. 8C</a>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw060/2/dsw060_Supp.zip?Expires=2147483647&amp;Signature=zDAtgxa4F-UL54z2if0eYAx7iiuGb6srTDL1CT7Hx7wDxFEom1ojV1lVEE9Us27ThDolTKyt9Km4jlX7dNK4MHhuRaD32ktxUaSGDSkmnZRKWib00mcJHlAT~5TrnVNjQG5nML7AyU1hHzg~KRGemgvdez0x0d46-pjP6K0JGmfKcntTePu8ATjm8~7UeUJ2ducPfWRDovHNFdYPhH6uJ8yF-~DSkp7EeFNFkhyacpRLpdfEj97SkZCC1qTS06IJaQxL19jmK1BvUZWM6~NYrK7MU-sm57TX44qf0BU5aLXa6LgFrASDYk-iu7SgJew7ClWCHfbFZ8ttSGEcp8o3-g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6</a></span>). Our results showed the presence of bands corresponding to the expected size, confirming the linearity of such regions and discarding the presence of rearrangements (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw060/2/dsw060_Supp.zip?Expires=2147483647&amp;Signature=zDAtgxa4F-UL54z2if0eYAx7iiuGb6srTDL1CT7Hx7wDxFEom1ojV1lVEE9Us27ThDolTKyt9Km4jlX7dNK4MHhuRaD32ktxUaSGDSkmnZRKWib00mcJHlAT~5TrnVNjQG5nML7AyU1hHzg~KRGemgvdez0x0d46-pjP6K0JGmfKcntTePu8ATjm8~7UeUJ2ducPfWRDovHNFdYPhH6uJ8yF-~DSkp7EeFNFkhyacpRLpdfEj97SkZCC1qTS06IJaQxL19jmK1BvUZWM6~NYrK7MU-sm57TX44qf0BU5aLXa6LgFrASDYk-iu7SgJew7ClWCHfbFZ8ttSGEcp8o3-g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>). After sequencing, PCR products were aligned to both PC15 and PC9 parental strains. Interestingly, the pattern of SNPs/indels showed that gene conversion occurred precisely in the same sites in all monokaryons analysed for each locus (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw060/2/dsw060_Supp.zip?Expires=2147483647&amp;Signature=zDAtgxa4F-UL54z2if0eYAx7iiuGb6srTDL1CT7Hx7wDxFEom1ojV1lVEE9Us27ThDolTKyt9Km4jlX7dNK4MHhuRaD32ktxUaSGDSkmnZRKWib00mcJHlAT~5TrnVNjQG5nML7AyU1hHzg~KRGemgvdez0x0d46-pjP6K0JGmfKcntTePu8ATjm8~7UeUJ2ducPfWRDovHNFdYPhH6uJ8yF-~DSkp7EeFNFkhyacpRLpdfEj97SkZCC1qTS06IJaQxL19jmK1BvUZWM6~NYrK7MU-sm57TX44qf0BU5aLXa6LgFrASDYk-iu7SgJew7ClWCHfbFZ8ttSGEcp8o3-g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span>), suggesting that the mechanism leading to this recombination might have specific targets. The absence of HELPO2 helitrons in the monokaryons with profiles consistent with allelic gene conversion events suggests that this mechanism might be used to eliminate unpaired repetitive sequences (such TEs) during meiosis. Allelic gene conversion is a molecular mechanism associated with recombination in which a genomic fragment is ‘copied/pasted’ onto another homologous fragment. Little is known about gene conversion in fungi, although it is associated with non-Mendelian segregation ratios in yeast and mammals.<a reveal-id="dsw060-B43" data-open="dsw060-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a><sup>,</sup><a reveal-id="dsw060-B44" data-open="dsw060-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a> In fungi, previous studies have shown that gene conversion leads to 3:1 ratios in heteroallelic crosses.<a reveal-id="dsw060-B45" data-open="dsw060-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a> A recent analysis of the recombination landscape in the basidiomycete <em>Agaricus bisporus</em> revealed a skewed segregation ratio from the expected Mendelian 1:1 in certain loci of the haploid offspring of 139 isolated single spores. In addition, results showed that crossover events (COs) occurred almost exclusively at chromosome ends including gene conversion in the reciprocal CO context.<a reveal-id="dsw060-B46" data-open="dsw060-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a> In 1982, it was proposed that gene conversion could be involved in the elimination of selfish DNA during meiosis when hybrid DNA structures are homologous but one of them displays a DNA insert present in one parent,<a reveal-id="dsw060-B47" data-open="dsw060-B47" class="link link-ref link-reveal xref-bibr"><sup>47</sup></a> as is the case with HELPO2 loci in PC15 and PC9. In such a case, a single-strand loop could be formed, being targeted by endonucleases and resulting in gene conversion with the elimination of the insert. Events of a model of gene conversion mediated by site-specific endonucleases have also been detected in different strains of <em>Botrytis cinerea</em>. The analysis of meiotic products obtained by crossing strains carrying polymorphisms permitted determination of gene conversion tracts that corresponded to unpaired regions previously identified in the parental strains.<a reveal-id="dsw060-B48" data-open="dsw060-B48" class="link link-ref link-reveal xref-bibr"><sup>48</sup></a> Our results are the first showing evidence of the somatic transposition of a native helitron in its own host and the meiotically driven elimination of an active helitron family in basidiomycetes. <div data-id="dsw060-F8" class="fig fig-section"><div class="label fig-label">Figure 8</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw060/2/m_dsw060f8.png?Expires=2147483647&amp;Signature=DmsNtt1e6Hg8g~RkaiQfeW67h8Gq8kJMDCoQ-kCuPtbZd6FGrmCDh~oIP2txaP6B4nY34TsPiLVl3TOFeTMigR27jnFafBuFapicrimQ51clcbNwzhH~yVyS17Y4lODV3Jk1i8i28zmR3dSDrklktS-~fzmsyPMgBQic3XXLb1pF-ysERBcibh6~gmStxDE9R55-6ULPgwZcTAEoDhxV9CAyS1jisLOm7b3y4syEwQQH3isxIsdgIktA-hkapgOwhcOrnaVcatm36p-1sfzMLyEGwRapK6Whx59F1uAyWRON1bTgp4xkOfXNfyyHgUM~kINsKen0oPEqnnnSD1EPKw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Molecular validation of HELPO2 meiotic loss. Schematic representation of Helpo2_VI (A), Helpo2_VIII_b (B) loci and their flanking (homologous) regions in the parental PC15 (upper blocks) and PC9 (intermediate blocks) as well as in 7 monokaryons showing profiles compatible with gene conversion events (lower blocks). HELPO2 elements are represented as an arrow in the parental PC15, but are absent in PC9 (vertical black lines). PCR primers are represented by black arrows (designed to amplify the vacant site) and white arrows (designed to amplify flanking sites). The size of overlapping regions (bp) amplified by primers pairs is showed between two slashes in PC15 and PC9 sequences. Regions excluded to PCR amplification are indicated by slashes in PC15 and PC9 parental strains and white rectangles in monokaryons. Vertical grey lines inside blocks represent PC9-specific SNPs/indels used to assign the parental origin of such regions. (C) Validation of gene conversion event in 4 (Helpo2_VI locus) and 3 (Helpo2_VIII_b locus) monokaryons. Primers sequences are indicated by arrows. Gene conversion junctions with their associated SNPs/indels are reported in the DNA alignment. " path-from-xml="dsw060f8.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055528/dsw060f8.tif" path-from-xml="dsw060f8.tif" target="_blank">View large</a><a section="78055528" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw060/2/dsw060f8.png?Expires=2147483647&amp;Signature=WHDZJ7eu6tullDtpaqhvWCdXbhWHzRxlZRf9Z2i-13Fheu3xNpIpyZau~9~HrQ60dg4XkZ~l6NV05BjTATvoMTetQnG7rpPKoK0ixVrAUJpmb1JB0up~3T8ZlIfkC-eFT86iCCG~YJVWMV-8Tyi3FYI6JvZ9PljWfm3UA0DIH2TLa~MCX6nZ72rCXV2D3t4lPDYieiCDKsblYLDJJ3eVykwC8wnuzBkD0GPbpQMx1Ha9CIYRaO7Cgii6jvHkRJw9NHUEpoVolXbqudqaXVJ0Xyj79kWyG7-PtzMzUuctbWKn9oVlBu23NFYkMFZsbrZ-ByMGFbKcJ6CLylyaA-HkUQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=78055528&amp;ar=2962881&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw060f8.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Molecular validation of HELPO2 meiotic loss. Schematic representation of <em>Helpo2_VI</em> (A), <em>Helpo2_VIII_b</em> (B) loci and their flanking (homologous) regions in the parental PC15 (upper blocks) and PC9 (intermediate blocks) as well as in 7 monokaryons showing profiles compatible with gene conversion events (lower blocks). HELPO2 elements are represented as an arrow in the parental PC15, but are absent in PC9 (vertical black lines). PCR primers are represented by black arrows (designed to amplify the vacant site) and white arrows (designed to amplify flanking sites). The size of overlapping regions (bp) amplified by primers pairs is showed between two slashes in PC15 and PC9 sequences. Regions excluded to PCR amplification are indicated by slashes in PC15 and PC9 parental strains and white rectangles in monokaryons. Vertical grey lines inside blocks represent PC9-specific SNPs/indels used to assign the parental origin of such regions. (C) Validation of gene conversion event in 4 (<em>Helpo2_VI</em> locus) and 3 (<em>Helpo2_VIII_b</em> locus) monokaryons. Primers sequences are indicated by arrows. Gene conversion junctions with their associated SNPs/indels are reported in the DNA alignment. </p></div></div><div content-id="dsw060-F8" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 8</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw060/2/m_dsw060f8.png?Expires=2147483647&amp;Signature=DmsNtt1e6Hg8g~RkaiQfeW67h8Gq8kJMDCoQ-kCuPtbZd6FGrmCDh~oIP2txaP6B4nY34TsPiLVl3TOFeTMigR27jnFafBuFapicrimQ51clcbNwzhH~yVyS17Y4lODV3Jk1i8i28zmR3dSDrklktS-~fzmsyPMgBQic3XXLb1pF-ysERBcibh6~gmStxDE9R55-6ULPgwZcTAEoDhxV9CAyS1jisLOm7b3y4syEwQQH3isxIsdgIktA-hkapgOwhcOrnaVcatm36p-1sfzMLyEGwRapK6Whx59F1uAyWRON1bTgp4xkOfXNfyyHgUM~kINsKen0oPEqnnnSD1EPKw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Molecular validation of HELPO2 meiotic loss. Schematic representation of Helpo2_VI (A), Helpo2_VIII_b (B) loci and their flanking (homologous) regions in the parental PC15 (upper blocks) and PC9 (intermediate blocks) as well as in 7 monokaryons showing profiles compatible with gene conversion events (lower blocks). HELPO2 elements are represented as an arrow in the parental PC15, but are absent in PC9 (vertical black lines). PCR primers are represented by black arrows (designed to amplify the vacant site) and white arrows (designed to amplify flanking sites). The size of overlapping regions (bp) amplified by primers pairs is showed between two slashes in PC15 and PC9 sequences. Regions excluded to PCR amplification are indicated by slashes in PC15 and PC9 parental strains and white rectangles in monokaryons. Vertical grey lines inside blocks represent PC9-specific SNPs/indels used to assign the parental origin of such regions. (C) Validation of gene conversion event in 4 (Helpo2_VI locus) and 3 (Helpo2_VIII_b locus) monokaryons. Primers sequences are indicated by arrows. Gene conversion junctions with their associated SNPs/indels are reported in the DNA alignment. " path-from-xml="dsw060f8.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055528/dsw060f8.tif" path-from-xml="dsw060f8.tif" target="_blank">View large</a><a section="78055528" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw060/2/dsw060f8.png?Expires=2147483647&amp;Signature=WHDZJ7eu6tullDtpaqhvWCdXbhWHzRxlZRf9Z2i-13Fheu3xNpIpyZau~9~HrQ60dg4XkZ~l6NV05BjTATvoMTetQnG7rpPKoK0ixVrAUJpmb1JB0up~3T8ZlIfkC-eFT86iCCG~YJVWMV-8Tyi3FYI6JvZ9PljWfm3UA0DIH2TLa~MCX6nZ72rCXV2D3t4lPDYieiCDKsblYLDJJ3eVykwC8wnuzBkD0GPbpQMx1Ha9CIYRaO7Cgii6jvHkRJw9NHUEpoVolXbqudqaXVJ0Xyj79kWyG7-PtzMzUuctbWKn9oVlBu23NFYkMFZsbrZ-ByMGFbKcJ6CLylyaA-HkUQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=78055528&amp;ar=2962881&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw060f8.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Molecular validation of HELPO2 meiotic loss. Schematic representation of <em>Helpo2_VI</em> (A), <em>Helpo2_VIII_b</em> (B) loci and their flanking (homologous) regions in the parental PC15 (upper blocks) and PC9 (intermediate blocks) as well as in 7 monokaryons showing profiles compatible with gene conversion events (lower blocks). HELPO2 elements are represented as an arrow in the parental PC15, but are absent in PC9 (vertical black lines). PCR primers are represented by black arrows (designed to amplify the vacant site) and white arrows (designed to amplify flanking sites). The size of overlapping regions (bp) amplified by primers pairs is showed between two slashes in PC15 and PC9 sequences. Regions excluded to PCR amplification are indicated by slashes in PC15 and PC9 parental strains and white rectangles in monokaryons. Vertical grey lines inside blocks represent PC9-specific SNPs/indels used to assign the parental origin of such regions. (C) Validation of gene conversion event in 4 (<em>Helpo2_VI</em> locus) and 3 (<em>Helpo2_VIII_b</em> locus) monokaryons. Primers sequences are indicated by arrows. Gene conversion junctions with their associated SNPs/indels are reported in the DNA alignment. </p></div></div></div>                            <h3 scrollto-destination=78055529 id="78055529" class="section-title" >3.7. Helitron elimination in <em>P. ostreatus</em> progeny: genome defence against invasive DNA?</h3>
<p>Eukaryotes have developed defence mechanisms to control the proliferation of TEs and avoid the potentially harmful effects of their insertions. One of the most impressive mechanisms of genome defence is called chromatin diminution, which has been described in several eukaryotes from distinct lineages.<a reveal-id="dsw060-B49" data-open="dsw060-B49" class="link link-ref link-reveal xref-bibr"><sup>49</sup></a><sup>,</sup><a reveal-id="dsw060-B50" data-open="dsw060-B50" class="link link-ref link-reveal xref-bibr"><sup>50</sup></a> This mechanism selectively eliminates DNA and targets mainly young TEs displaying high frequencies and similarities.<a reveal-id="dsw060-B49" data-open="dsw060-B49" class="link link-ref link-reveal xref-bibr"><sup>49</sup></a> The molecular mechanism underlying chromatin diminution bears similarity to piRNA-directed transposon silencing in metazoans and requires an active RNAi machinery. In fact, it has been proposed that small RNAs could play an essential role directing such DNA elimination in the protozoan <em>Tetrahymena</em><em>thermophile</em>.<a reveal-id="dsw060-B50" data-open="dsw060-B50" class="link link-ref link-reveal xref-bibr"><sup>50</sup></a> No study has described the presence of such mechanisms in fungi, but a handful of studies have shown the presence and activity of the main components of the RNAi pathway in filamentous fungi (Dicer, Argonaute, and RNA-dependent RNA polymerase, reviewed in Ref. <a reveal-id="dsw060-B51" data-open="dsw060-B51" class="link link-ref link-reveal xref-bibr">51</a>). From the diversity of the fungal RNAi pathway, two mechanisms have evolved to control TEs at the vegetative (Quelling) and sexual (MSUD) states. The latter is of great relevance to our study as it involves the identification of unpaired DNA during meiosis by a <em>trans</em>-sensing mechanism. The origin of such unpaired regions are likely produced by TE insertions in only one of the two parental strains, such as in the case of the HELPO2 family in <em>P. ostreatus.</em> We screened PC15 and PC9 genomes for proteins homologous to those involved in <em>N. crassa</em> MSUD and Quelling and found a total of 18 genes in both strains: 5 argonaute proteins, 5 RNA-dependent RNA polymerases (RdRPs), 4 dicers, 3 RecQ helicases and 1 exonuclease. <em>P. ostreatus</em> RNAi proteins carry the corresponding conserved domains (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw060/2/dsw060_Supp.zip?Expires=2147483647&Signature=zDAtgxa4F-UL54z2if0eYAx7iiuGb6srTDL1CT7Hx7wDxFEom1ojV1lVEE9Us27ThDolTKyt9Km4jlX7dNK4MHhuRaD32ktxUaSGDSkmnZRKWib00mcJHlAT~5TrnVNjQG5nML7AyU1hHzg~KRGemgvdez0x0d46-pjP6K0JGmfKcntTePu8ATjm8~7UeUJ2ducPfWRDovHNFdYPhH6uJ8yF-~DSkp7EeFNFkhyacpRLpdfEj97SkZCC1qTS06IJaQxL19jmK1BvUZWM6~NYrK7MU-sm57TX44qf0BU5aLXa6LgFrASDYk-iu7SgJew7ClWCHfbFZ8ttSGEcp8o3-g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S7</a></span>) and are actively expressed in vegetative mycelia (<a reveal-id="dsw060-T5" data-open="dsw060-T5" class="link link-reveal link-table xref-fig">Table 5</a>). Further experimental work is needed to understand whether the transcriptional repression and meiotic elimination of HELPO2 is somehow linked to the activity of RNAi pathway genes. Due to the ability of fungi to diversify and develop ‘epigenetic solutions’ to control TE proliferation, such a possibility seems to be an interesting field for future studies. <div content-id="dsw060-T5" class="table-modal"><div class="table-wrap"><div id="dsw060-T5" data-id="dsw060-T5" class="table-wrap-title"><div class="label -label">Table 5</div><div class="caption"><p>Identification of proteins belonging to the RNAi pathway in <em>P. ostreatus</em></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>PC15-ID<a reveal-id="tblfn9" data-open="tblfn9" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th>PC9-ID<a reveal-id="tblfn9" data-open="tblfn9" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th>RPKM </th><th>RPKM </th><th>Description </th></tr></thead><tbody><tr><td>25449 </td><td>90477 </td><td>10.8 </td><td>11.1 </td><td>RNA-dependent RNA polymerase </td></tr><tr><td>1053861 </td><td>85609 </td><td>16.34 </td><td>30.2 </td><td>RNA-dependent RNA polymerase </td></tr><tr><td>1076136 </td><td>83385 </td><td>2.74 </td><td>0.6 </td><td>RNA-dependent RNA polymerase </td></tr><tr><td>1041769 </td><td>52011 </td><td>36.06 </td><td>22.6 </td><td>RNA-dependent RNA polymerase </td></tr><tr><td>154946 </td><td>85193 </td><td>53.19 </td><td>21.8 </td><td>RNA-dependent RNA polymerase </td></tr><tr><td>1060211 </td><td>89407 </td><td>87.2 </td><td>233.8 </td><td>Argonaute </td></tr><tr><td>1110274 </td><td>43687 </td><td>116.78 </td><td>59.7 </td><td>Argonaute </td></tr><tr><td>173501 </td><td>122655 </td><td>85.31 </td><td>124.2 </td><td>Argonaute </td></tr><tr><td>21640 </td><td>91748 </td><td>12.43 </td><td>20.5 </td><td>Argonaute </td></tr><tr><td>44554 </td><td>114784 </td><td>27.76 </td><td>39.9 </td><td>Argonaute </td></tr><tr><td>33722 </td><td>67302 </td><td>15.34 </td><td>30.7 </td><td>RecQ Helicase </td></tr><tr><td>11210 </td><td>24796 </td><td>12.58 </td><td>3.3 </td><td>RecQ Helicase </td></tr><tr><td>160303 </td><td>87316 </td><td>7.47 </td><td>4.8 </td><td>RecQ Helicase </td></tr><tr><td>1064031 </td><td>83509 </td><td>12.96 </td><td>6.6 </td><td>DICER </td></tr><tr><td>1033048 </td><td>81629 </td><td>9.4 </td><td>7.6 </td><td>DICER </td></tr><tr><td>1093523 </td><td>50672 </td><td>27.64 </td><td>16.8 </td><td>DICER </td></tr><tr><td>1112895 </td><td>84917 </td><td>34.41 </td><td>47.1 </td><td>DICER </td></tr><tr><td>1039826 </td><td>83395 </td><td>41.63 </td><td>54.4 </td><td>DnaQ-like exonuclease </td></tr></tbody></table></div><div content-id="dsw060-T5" class="table-modal"><table><thead name="thead" align=""><tr><th>PC15-ID<a reveal-id="tblfn9" data-open="tblfn9" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th>PC9-ID<a reveal-id="tblfn9" data-open="tblfn9" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th>RPKM </th><th>RPKM </th><th>Description </th></tr></thead><tbody><tr><td>25449 </td><td>90477 </td><td>10.8 </td><td>11.1 </td><td>RNA-dependent RNA polymerase </td></tr><tr><td>1053861 </td><td>85609 </td><td>16.34 </td><td>30.2 </td><td>RNA-dependent RNA polymerase </td></tr><tr><td>1076136 </td><td>83385 </td><td>2.74 </td><td>0.6 </td><td>RNA-dependent RNA polymerase </td></tr><tr><td>1041769 </td><td>52011 </td><td>36.06 </td><td>22.6 </td><td>RNA-dependent RNA polymerase </td></tr><tr><td>154946 </td><td>85193 </td><td>53.19 </td><td>21.8 </td><td>RNA-dependent RNA polymerase </td></tr><tr><td>1060211 </td><td>89407 </td><td>87.2 </td><td>233.8 </td><td>Argonaute </td></tr><tr><td>1110274 </td><td>43687 </td><td>116.78 </td><td>59.7 </td><td>Argonaute </td></tr><tr><td>173501 </td><td>122655 </td><td>85.31 </td><td>124.2 </td><td>Argonaute </td></tr><tr><td>21640 </td><td>91748 </td><td>12.43 </td><td>20.5 </td><td>Argonaute </td></tr><tr><td>44554 </td><td>114784 </td><td>27.76 </td><td>39.9 </td><td>Argonaute </td></tr><tr><td>33722 </td><td>67302 </td><td>15.34 </td><td>30.7 </td><td>RecQ Helicase </td></tr><tr><td>11210 </td><td>24796 </td><td>12.58 </td><td>3.3 </td><td>RecQ Helicase </td></tr><tr><td>160303 </td><td>87316 </td><td>7.47 </td><td>4.8 </td><td>RecQ Helicase </td></tr><tr><td>1064031 </td><td>83509 </td><td>12.96 </td><td>6.6 </td><td>DICER </td></tr><tr><td>1033048 </td><td>81629 </td><td>9.4 </td><td>7.6 </td><td>DICER </td></tr><tr><td>1093523 </td><td>50672 </td><td>27.64 </td><td>16.8 </td><td>DICER </td></tr><tr><td>1112895 </td><td>84917 </td><td>34.41 </td><td>47.1 </td><td>DICER </td></tr><tr><td>1039826 </td><td>83395 </td><td>41.63 </td><td>54.4 </td><td>DnaQ-like exonuclease </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn8"><div class="fn"><p>Transcription of PC15 and PC9 orthologs is shown in RPKM (reads per kilobase per million mapped reads).</p></div></div><div content-id="tblfn9"><div class="fn"><p><sup>a</sup>Protein identifiers of PC15 v2.0 and PC9 v1.0 Mycocosm databases.</p></div></div></div></div></div><div class="table-wrap"><div id="dsw060-T5" data-id="dsw060-T5" class="table-wrap-title"><div class="label -label">Table 5</div><div class="caption"><p>Identification of proteins belonging to the RNAi pathway in <em>P. ostreatus</em></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>PC15-ID<a reveal-id="tblfn9" data-open="tblfn9" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th>PC9-ID<a reveal-id="tblfn9" data-open="tblfn9" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th>RPKM </th><th>RPKM </th><th>Description </th></tr></thead><tbody><tr><td>25449 </td><td>90477 </td><td>10.8 </td><td>11.1 </td><td>RNA-dependent RNA polymerase </td></tr><tr><td>1053861 </td><td>85609 </td><td>16.34 </td><td>30.2 </td><td>RNA-dependent RNA polymerase </td></tr><tr><td>1076136 </td><td>83385 </td><td>2.74 </td><td>0.6 </td><td>RNA-dependent RNA polymerase </td></tr><tr><td>1041769 </td><td>52011 </td><td>36.06 </td><td>22.6 </td><td>RNA-dependent RNA polymerase </td></tr><tr><td>154946 </td><td>85193 </td><td>53.19 </td><td>21.8 </td><td>RNA-dependent RNA polymerase </td></tr><tr><td>1060211 </td><td>89407 </td><td>87.2 </td><td>233.8 </td><td>Argonaute </td></tr><tr><td>1110274 </td><td>43687 </td><td>116.78 </td><td>59.7 </td><td>Argonaute </td></tr><tr><td>173501 </td><td>122655 </td><td>85.31 </td><td>124.2 </td><td>Argonaute </td></tr><tr><td>21640 </td><td>91748 </td><td>12.43 </td><td>20.5 </td><td>Argonaute </td></tr><tr><td>44554 </td><td>114784 </td><td>27.76 </td><td>39.9 </td><td>Argonaute </td></tr><tr><td>33722 </td><td>67302 </td><td>15.34 </td><td>30.7 </td><td>RecQ Helicase </td></tr><tr><td>11210 </td><td>24796 </td><td>12.58 </td><td>3.3 </td><td>RecQ Helicase </td></tr><tr><td>160303 </td><td>87316 </td><td>7.47 </td><td>4.8 </td><td>RecQ Helicase </td></tr><tr><td>1064031 </td><td>83509 </td><td>12.96 </td><td>6.6 </td><td>DICER </td></tr><tr><td>1033048 </td><td>81629 </td><td>9.4 </td><td>7.6 </td><td>DICER </td></tr><tr><td>1093523 </td><td>50672 </td><td>27.64 </td><td>16.8 </td><td>DICER </td></tr><tr><td>1112895 </td><td>84917 </td><td>34.41 </td><td>47.1 </td><td>DICER </td></tr><tr><td>1039826 </td><td>83395 </td><td>41.63 </td><td>54.4 </td><td>DnaQ-like exonuclease </td></tr></tbody></table></div><div content-id="dsw060-T5" class="table-modal"><table><thead name="thead" align=""><tr><th>PC15-ID<a reveal-id="tblfn9" data-open="tblfn9" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th>PC9-ID<a reveal-id="tblfn9" data-open="tblfn9" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th>RPKM </th><th>RPKM </th><th>Description </th></tr></thead><tbody><tr><td>25449 </td><td>90477 </td><td>10.8 </td><td>11.1 </td><td>RNA-dependent RNA polymerase </td></tr><tr><td>1053861 </td><td>85609 </td><td>16.34 </td><td>30.2 </td><td>RNA-dependent RNA polymerase </td></tr><tr><td>1076136 </td><td>83385 </td><td>2.74 </td><td>0.6 </td><td>RNA-dependent RNA polymerase </td></tr><tr><td>1041769 </td><td>52011 </td><td>36.06 </td><td>22.6 </td><td>RNA-dependent RNA polymerase </td></tr><tr><td>154946 </td><td>85193 </td><td>53.19 </td><td>21.8 </td><td>RNA-dependent RNA polymerase </td></tr><tr><td>1060211 </td><td>89407 </td><td>87.2 </td><td>233.8 </td><td>Argonaute </td></tr><tr><td>1110274 </td><td>43687 </td><td>116.78 </td><td>59.7 </td><td>Argonaute </td></tr><tr><td>173501 </td><td>122655 </td><td>85.31 </td><td>124.2 </td><td>Argonaute </td></tr><tr><td>21640 </td><td>91748 </td><td>12.43 </td><td>20.5 </td><td>Argonaute </td></tr><tr><td>44554 </td><td>114784 </td><td>27.76 </td><td>39.9 </td><td>Argonaute </td></tr><tr><td>33722 </td><td>67302 </td><td>15.34 </td><td>30.7 </td><td>RecQ Helicase </td></tr><tr><td>11210 </td><td>24796 </td><td>12.58 </td><td>3.3 </td><td>RecQ Helicase </td></tr><tr><td>160303 </td><td>87316 </td><td>7.47 </td><td>4.8 </td><td>RecQ Helicase </td></tr><tr><td>1064031 </td><td>83509 </td><td>12.96 </td><td>6.6 </td><td>DICER </td></tr><tr><td>1033048 </td><td>81629 </td><td>9.4 </td><td>7.6 </td><td>DICER </td></tr><tr><td>1093523 </td><td>50672 </td><td>27.64 </td><td>16.8 </td><td>DICER </td></tr><tr><td>1112895 </td><td>84917 </td><td>34.41 </td><td>47.1 </td><td>DICER </td></tr><tr><td>1039826 </td><td>83395 </td><td>41.63 </td><td>54.4 </td><td>DnaQ-like exonuclease </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn8"><div class="fn"><p>Transcription of PC15 and PC9 orthologs is shown in RPKM (reads per kilobase per million mapped reads).</p></div></div><div content-id="tblfn9"><div class="fn"><p><sup>a</sup>Protein identifiers of PC15 v2.0 and PC9 v1.0 Mycocosm databases.</p></div></div></div></div></p>                            <h2 scrollto-destination=78055531 id="78055531" class="section-title" >Supplementary data</h2>
<p><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw060/2/dsw060_Supp.zip?Expires=2147483647&Signature=zDAtgxa4F-UL54z2if0eYAx7iiuGb6srTDL1CT7Hx7wDxFEom1ojV1lVEE9Us27ThDolTKyt9Km4jlX7dNK4MHhuRaD32ktxUaSGDSkmnZRKWib00mcJHlAT~5TrnVNjQG5nML7AyU1hHzg~KRGemgvdez0x0d46-pjP6K0JGmfKcntTePu8ATjm8~7UeUJ2ducPfWRDovHNFdYPhH6uJ8yF-~DSkp7EeFNFkhyacpRLpdfEj97SkZCC1qTS06IJaQxL19jmK1BvUZWM6~NYrK7MU-sm57TX44qf0BU5aLXa6LgFrASDYk-iu7SgJew7ClWCHfbFZ8ttSGEcp8o3-g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary data</a></span> are available at <a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org" target="_blank">www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=78055533 id="78055533" class="section-title" >Funding</h2>
<p>This work was supported by Spanish National Research Plan (Projects AGL2011-30495 and AGL2014-55971-R) and FEDER funds; Public University of Navarre (http://www.unavarra.es); AB holds a PhD scholarship from the Public University of Navarre and RC holds a FPI-PhD scholarship from the Ministry of Economy and Competitiveness.</p>                            <h2 scrollto-destination=78055535 id="78055535" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We would like to thank Dr. Matteo Pellegrini and Marco Morselli (University of California, Los Angeles) for the lab equipment support.</p>                            <h3 scrollto-destination=78055537 id="78055537" class="section-title" >Conflict of interest</h3>
<p>None declared. </p>                            <h3 scrollto-destination=78055539 id="78055539" class="section-title" >Footnotes</h3>
<p>Conceived and designed the experiments: AB RC AGP LR. Performed the experiments: AB EM. Analyzed the data: AB RC LR. Wrote the paper: AB RC. Led the project, revised and edited the manuscript: AGP LR. All been deposited in databases.</p>                            <h2 scrollto-destination=78055541 id="78055541" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsw060-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw060-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kazazian</div> <div class="given-names">H. H.</div></div></div> <div class="year">2004</div>, 
<div class="article-title">Mobile elements: drivers of genome evolution</div>. <div class="source">Science</div> , <div class="volume">303</div>, <div class="fpage">1626</div>–<div class="lpage">32</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mobile%20elements%3A%20drivers%20of%20genome%20evolution&amp;author=H.%20H.%20Kazazian&amp;publication_year=2004&amp;journal=Science&amp;volume=303&amp;pages=1626-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1089670" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1089670" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1089670"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15016989" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw060-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw060-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wicker</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Sabot</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Hua-Van</div> <div class="given-names">A.</div></div></div>, et al.  . <div class="year">2007</div>, 
<div class="article-title">A unified classification system for eukaryotic transposable elements</div>. <div class="source">Nat. Rev. Genet</div> ., <div class="volume">8</div>, <div class="fpage">973</div>–<div class="lpage">82</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20unified%20classification%20system%20for%20eukaryotic%20transposable%20elements&amp;author=T.%20Wicker&amp;author=F.%20Sabot&amp;author=A.%20Hua-Van&amp;publication_year=2007&amp;journal=Nat.%20Rev.%20Genet&amp;volume=8&amp;pages=973-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg2165" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg2165" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg2165"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17984973" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw060-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw060-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kapitonov</div> <div class="given-names">V. V</div></div>, <div class="name string-name"><div class="surname">Jurka</div> <div class="given-names">J.</div></div></div> <div class="year">2001</div>, 
<div class="article-title">Rolling-circle transposons in eukaryotes</div>. <div class="source">Proc. Natl. Acad. Sci. U. S. A</div> ., <div class="volume">98</div>, <div class="fpage">8714</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rolling-circle%20transposons%20in%20eukaryotes&amp;author=V.%20V%20Kapitonov&amp;author=J.%20Jurka&amp;publication_year=2001&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A&amp;volume=98&amp;pages=8714-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.151269298" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.151269298" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.151269298"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11447285" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw060-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw060-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Poulter</div> <div class="given-names">R. T. M.</div></div>, <div class="name string-name"><div class="surname">Goodwin</div> <div class="given-names">T. J. D.</div></div>, <div class="name string-name"><div class="surname">Butler</div> <div class="given-names">M. I.</div></div></div> <div class="year">2003</div>, 
<div class="article-title">Vertebrate helentrons and other novel Helitrons</div>. <div class="source">Gene</div> , <div class="volume">313</div>, <div class="fpage">201</div>–<div class="lpage">12</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Vertebrate%20helentrons%20and%20other%20novel%20Helitrons&amp;author=R.%20T.%20M.%20Poulter&amp;author=T.%20J.%20D.%20Goodwin&amp;author=M.%20I.%20Butler&amp;publication_year=2003&amp;journal=Gene&amp;volume=313&amp;pages=201-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0378-1119(03)00679-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0378-1119(03)00679-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0378-1119(03)00679-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12957391" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw060-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw060-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Pritham</div> <div class="given-names">E. J.</div></div>, <div class="name string-name"><div class="surname">Feschotte</div> <div class="given-names">C.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">Massive amplification of rolling-circle transposons in the lineage of the bat <em>Myotis lucifugus</em></div>. <div class="source">Proc. Natl. Acad. Sci. U. S. A</div> ., <div class="volume">104</div>, <div class="fpage">1895</div>–<div class="lpage">900</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Massive%20amplification%20of%20rolling-circle%20transposons%20in%20the%20lineage%20of%20the%20bat%20Myotis%20lucifugus&amp;author=E.%20J.%20Pritham&amp;author=C.%20Feschotte&amp;publication_year=2007&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A&amp;volume=104&amp;pages=1895-900" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0609601104" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0609601104" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0609601104"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17261799" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw060-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw060-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Roffler</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Menardo</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Wicker</div> <div class="given-names">T.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">The making of a genomic parasite - the Mothra family sheds light on the evolution of Helitrons in plants</div>. <div class="source">Mob. DNA</div> , <div class="volume">6</div>, <div class="fpage">23</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20making%20of%20a%20genomic%20parasite%20-%20the%20Mothra%20family%20sheds%20light%20on%20the%20evolution%20of%20Helitrons%20in%20plants&amp;author=S.%20Roffler&amp;author=F.%20Menardo&amp;author=T.%20Wicker&amp;publication_year=2015&amp;journal=Mob.%20DNA&amp;volume=6&amp;pages=23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s13100-015-0054-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs13100-015-0054-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs13100-015-0054-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26688693" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw060-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw060-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kapitonov</div> <div class="given-names">V. V.</div></div>, <div class="name string-name"><div class="surname">Jurka</div> <div class="given-names">J.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">Helitrons on a roll: eukaryotic rolling-circle transposons</div>. <div class="source">Trends Genet</div> ., pp. <div class="fpage">521</div>–<div class="lpage">9</div>.</div></div></div></div><div content-id="dsw060-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw060-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Dooner</div> <div class="given-names">H. K.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Excision of Helitron transposons in maize</div>. <div class="source">Genetics</div> , <div class="volume">182</div>, <div class="fpage">399</div>–<div class="lpage">402</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Excision%20of%20Helitron%20transposons%20in%20maize&amp;author=Y.%20Li&amp;author=H.%20K.%20Dooner&amp;publication_year=2009&amp;journal=Genetics&amp;volume=182&amp;pages=399-402" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.109.101527" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.109.101527" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.109.101527"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19255366" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw060-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw060-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lai</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Messing</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Dooner</div> <div class="given-names">H. K.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">Gene movement by Helitron transposons contributes to the haplotype variability of maize</div>. <div class="source">Proc. Natl. Acad. Sci. U. S. A</div> ., <div class="volume">102</div>, <div class="fpage">9068</div>–<div class="lpage">73</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20movement%20by%20Helitron%20transposons%20contributes%20to%20the%20haplotype%20variability%20of%20maize&amp;author=J.%20Lai&amp;author=Y.%20Li&amp;author=J.%20Messing&amp;author=H.%20K.%20Dooner&amp;publication_year=2005&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A&amp;volume=102&amp;pages=9068-73" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0502923102" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0502923102" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0502923102"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15951422" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw060-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw060-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Choi</div> <div class="given-names">J. D.</div></div>, <div class="name string-name"><div class="surname">Hoshino</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Park</div> <div class="given-names">K.</div>Il</div>, <div class="name string-name"><div class="surname">Park</div> <div class="given-names">I. S.</div></div>, <div class="name string-name"><div class="surname">Iida</div> <div class="given-names">S.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">Spontaneous mutations caused by a Helitron transposon, Hel-It1, in morning glory, <em>Ipomoea tricolor</em></div>. <div class="source">Plant J</div> ., <div class="volume">49</div>, <div class="fpage">924</div>–<div class="lpage">34</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Spontaneous%20mutations%20caused%20by%20a%20Helitron%20transposon%2C%20Hel-It1%2C%20in%20morning%20glory%2C%20Ipomoea%20tricolor&amp;author=J.%20D.%20Choi&amp;author=A.%20Hoshino&amp;author=K.%20Park&amp;author=I.%20S.%20Park&amp;author=S.%20Iida&amp;publication_year=2007&amp;journal=Plant%20J&amp;volume=49&amp;pages=924-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.2007.49.issue-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2007.49.issue-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.2007.49.issue-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17257169" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw060-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw060-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Fu</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Dooner</div> <div class="given-names">H. K.</div></div></div> <div class="year">2002</div>, 
<div class="article-title">Intraspecific violation of genetic colinearity and its implications in maize</div>. <div class="source">Proc. Natl. Acad. Sci. U. S. A</div> ., <div class="volume">99</div>, <div class="fpage">9573</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Intraspecific%20violation%20of%20genetic%20colinearity%20and%20its%20implications%20in%20maize&amp;author=H.%20Fu&amp;author=H.%20K.%20Dooner&amp;publication_year=2002&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A&amp;volume=99&amp;pages=9573-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.132259199" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.132259199" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.132259199"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12060715" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw060-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw060-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Grabundzija</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Messing</div> <div class="given-names">S. A.</div></div>, <div class="name string-name"><div class="surname">Thomas</div> <div class="given-names">J.</div></div></div>, et al.  . <div class="year">2016</div>, 
<div class="article-title">A Helitron transposon reconstructed from bats reveals a novel mechanism of genome shuffling in eukaryotes</div>. <div class="source">Nat. Commun</div> ., <div class="volume">7</div>, <div class="fpage">10716</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20Helitron%20transposon%20reconstructed%20from%20bats%20reveals%20a%20novel%20mechanism%20of%20genome%20shuffling%20in%20eukaryotes&amp;author=I.%20Grabundzija&amp;author=S.%20A.%20Messing&amp;author=J.%20Thomas&amp;publication_year=2016&amp;journal=Nat.%20Commun&amp;volume=7&amp;pages=10716" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ncomms10716" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fncomms10716" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fncomms10716"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26931494" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw060-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw060-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zhou</div> <div class="given-names">Q.</div></div>, <div class="name string-name"><div class="surname">Froschauer</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Schultheis</div> <div class="given-names">C.</div></div></div>, et al.  . <div class="year">2006</div>, 
<div class="article-title">Helitron Transposons on the Sex Chromosomes of the Platyfish <em>Xiphophorus maculatus</em> and Their Evolution in Animal Genomes</div>. <div class="source">Zebrafish</div> , <div class="volume">3</div>, <div class="fpage">39</div>–<div class="lpage">52</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Helitron%20Transposons%20on%20the%20Sex%20Chromosomes%20of%20the%20Platyfish%20Xiphophorus%20maculatus%20and%20Their%20Evolution%20in%20Animal%20Genomes&amp;author=Q.%20Zhou&amp;author=A.%20Froschauer&amp;author=C.%20Schultheis&amp;publication_year=2006&amp;journal=Zebrafish&amp;volume=3&amp;pages=39-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1089/zeb.2006.3.39" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1089%2Fzeb.2006.3.39" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1089%2Fzeb.2006.3.39"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18248245" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw060-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw060-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lal</div> <div class="given-names">S. K.</div></div></div> <div class="year">2003</div>, 
<div class="article-title">The Maize Genome Contains a Helitron Insertion</div>. <div class="source">Plant Cell Online</div> , <div class="volume">15</div>, <div class="fpage">381</div>–<div class="lpage">91</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Maize%20Genome%20Contains%20a%20Helitron%20Insertion&amp;author=S.%20K.%20Lal&amp;publication_year=2003&amp;journal=Plant%20Cell%20Online&amp;volume=15&amp;pages=381-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.008375" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.008375" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.008375"> </span></div> </div></div></div></div></div><div content-id="dsw060-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw060-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Xiong</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">He</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Lai</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Dooner</div> <div class="given-names">H. K.</div></div>, <div class="name string-name"><div class="surname">Du</div> <div class="given-names">C.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">HelitronScanner uncovers a large overlooked cache of Helitron transposons in many plant genomes</div>. <div class="source">Proc. Natl. Acad. Sci</div> ., <div class="volume">111</div>, <div class="fpage">10263</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=HelitronScanner%20uncovers%20a%20large%20overlooked%20cache%20of%20Helitron%20transposons%20in%20many%20plant%20genomes&amp;author=W.%20Xiong&amp;author=L.%20He&amp;author=J.%20Lai&amp;author=H.%20K.%20Dooner&amp;author=C.%20Du&amp;publication_year=2014&amp;journal=Proc.%20Natl.%20Acad.%20Sci&amp;volume=111&amp;pages=10263-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1410068111" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1410068111" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1410068111"> </span></div> </div></div></div></div></div><div content-id="dsw060-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw060-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Morgante</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Brunner</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Pea</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Fengler</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Zuccolo</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Rafalski</div> <div class="given-names">A.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">Gene duplication and exon shuffling by helitron-like transposons generate intraspecies diversity in maize</div>. <div class="source">Nat. Genet</div> ., <div class="volume">37</div>, <div class="fpage">997</div>–<div class="lpage">1002</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20duplication%20and%20exon%20shuffling%20by%20helitron-like%20transposons%20generate%20intraspecies%20diversity%20in%20maize&amp;author=M.%20Morgante&amp;author=S.%20Brunner&amp;author=G.%20Pea&amp;author=K.%20Fengler&amp;author=A.%20Zuccolo&amp;author=A.%20Rafalski&amp;publication_year=2005&amp;journal=Nat.%20Genet&amp;volume=37&amp;pages=997-1002" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng1615" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng1615" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng1615"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16056225" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw060-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw060-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Castanera</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Perez</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Lopez</div> <div class="given-names">L.</div></div></div>, et al.  . <div class="year">2014</div>, 
<div class="article-title">Highly expressed captured genes and cross-kingdom domains present in Helitrons create novel diversity in <em>Pleurotus ostreatus</em> and other fungi</div>. <div class="source">BMC Genomics</div> , <div class="volume">15</div>, <div class="fpage">1071</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Highly%20expressed%20captured%20genes%20and%20cross-kingdom%20domains%20present%20in%20Helitrons%20create%20novel%20diversity%20in%20Pleurotus%20ostreatus%20and%20other%20fungi&amp;author=R.%20Castanera&amp;author=G.%20Perez&amp;author=L.%20Lopez&amp;publication_year=2014&amp;journal=BMC%20Genomics&amp;volume=15&amp;pages=1071" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-15-1071" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-15-1071" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-15-1071"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25480150" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw060-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw060-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Cultrone</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Domínguez</div> <div class="given-names">Y. R.</div></div>, <div class="name string-name"><div class="surname">Drevet</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Scazzocchio</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Fernández-Martín</div> <div class="given-names">R.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">The tightly regulated promoter of the xanA gene of <em>Aspergillus nidulans</em> is included in a helitron</div>. <div class="source">Mol. Microbiol</div> ., <div class="volume">63</div>, <div class="fpage">1577</div>–<div class="lpage">87</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20tightly%20regulated%20promoter%20of%20the%20xanA%20gene%20of%20Aspergillus%20nidulans%20is%20included%20in%20a%20helitron&amp;author=A.%20Cultrone&amp;author=Y.%20R.%20Dom%C3%ADnguez&amp;author=C.%20Drevet&amp;author=C.%20Scazzocchio&amp;author=R.%20Fern%C3%A1ndez-Mart%C3%ADn&amp;publication_year=2007&amp;journal=Mol.%20Microbiol&amp;volume=63&amp;pages=1577-87" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1365-2958.2007.05609.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1365-2958.2007.05609.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1365-2958.2007.05609.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17367381" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw060-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw060-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Martin</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Kohler</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Murat</div> <div class="given-names">C.</div></div></div>, et al.  . <div class="year">2010</div>, 
<div class="article-title">Périgord black truffle genome uncovers evolutionary origins and mechanisms of symbiosis</div>. <div class="source">Nature</div> , <div class="volume">464</div>, <div class="fpage">1033</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=P%C3%A9rigord%20black%20truffle%20genome%20uncovers%20evolutionary%20origins%20and%20mechanisms%20of%20symbiosis&amp;author=F.%20Martin&amp;author=A.%20Kohler&amp;author=C.%20Murat&amp;publication_year=2010&amp;journal=Nature&amp;volume=464&amp;pages=1033-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature08867" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature08867" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature08867"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20348908" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw060-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw060-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Labbé</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Murat</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Morin</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Tuskan</div> <div class="given-names">G. A.</div></div>, <div class="name string-name"><div class="surname">Le Tacon</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Martin</div> <div class="given-names">F.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Characterization of transposable elements in the ectomycorrhizal fungus <em>Laccaria bicolor</em></div>. <div class="source">PLoS One</div> , <div class="volume">7</div>.</div></div></div></div><div content-id="dsw060-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw060-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Santoyo</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">González</div> <div class="given-names">A. E.</div></div>, <div class="name string-name"><div class="surname">Terrón</div> <div class="given-names">M. C.</div></div>, <div class="name string-name"><div class="surname">Ramírez</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Pisabarro</div> <div class="given-names">A. G.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">Quantitative linkage mapping of lignin-degrading enzymatic activities in <em>Pleurotus ostreatus</em></div>. <div class="source">Enzyme Microb. Technol</div> ., <div class="volume">43</div>, <div class="fpage">137</div>–<div class="lpage">43</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Quantitative%20linkage%20mapping%20of%20lignin-degrading%20enzymatic%20activities%20in%20Pleurotus%20ostreatus&amp;author=F.%20Santoyo&amp;author=A.%20E.%20Gonz%C3%A1lez&amp;author=M.%20C.%20Terr%C3%B3n&amp;author=L.%20Ram%C3%ADrez&amp;author=A.%20G.%20Pisabarro&amp;publication_year=2008&amp;journal=Enzyme%20Microb.%20Technol&amp;volume=43&amp;pages=137-43" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.enzmictec.2007.11.007" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.enzmictec.2007.11.007" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.enzmictec.2007.11.007"> </span></div> </div></div></div></div></div><div content-id="dsw060-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw060-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Eugenio</div> <div class="given-names">C. P.</div></div>, <div class="name string-name"><div class="surname">Anderson</div> <div class="given-names">N. A.</div></div></div> <div class="year">1968</div>, 
<div class="article-title">The genetics and cultivation of <em>Pleurotus ostreatus</em></div>. <div class="source">Mycologia</div> , <div class="volume">60</div>, <div class="fpage">627</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genetics%20and%20cultivation%20of%20Pleurotus%20ostreatus&amp;author=C.%20P.%20Eugenio&amp;author=N.%20A.%20Anderson&amp;publication_year=1968&amp;journal=Mycologia&amp;volume=60&amp;pages=627" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2307/3757430" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2307%2F3757430" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2307%2F3757430"> </span></div> </div></div></div></div></div><div content-id="dsw060-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw060-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sánchez</div> <div class="given-names">C.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Cultivation of <em>Pleurotus ostreatus</em> and other edible mushrooms</div>. <div class="source">Appl. Microbiol. Biotechnol</div> ., <div class="volume">85</div>, <div class="fpage">1321</div>–<div class="lpage">37</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cultivation%20of%20Pleurotus%20ostreatus%20and%20other%20edible%20mushrooms&amp;author=C.%20S%C3%A1nchez&amp;publication_year=2010&amp;journal=Appl.%20Microbiol.%20Biotechnol&amp;volume=85&amp;pages=1321-37" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00253-009-2343-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00253-009-2343-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00253-009-2343-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19956947" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw060-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw060-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Larraya</div> <div class="given-names">L. M.</div></div>, <div class="name string-name"><div class="surname">Perez</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Peñas</div> <div class="given-names">M. M.</div></div></div>, et al.  . <div class="year">1999</div>, 
<div class="article-title">Molecular karyotype of the white rot fungus <em>Pleurotus ostreatus</em></div>. <div class="source">Appl. Envir. Microbiol</div> ., <div class="volume">65</div>, <div class="fpage">3413</div>–<div class="lpage">7</div>.</div></div></div></div><div content-id="dsw060-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw060-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Larraya</div> <div class="given-names">L. M.</div></div>, <div class="name string-name"><div class="surname">Perez</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Ritter</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Pisabarro</div> <div class="given-names">A. G.</div></div>, <div class="name string-name"><div class="surname">Ramírez</div> <div class="given-names">L.</div></div></div> <div class="year">2000</div>, 
<div class="article-title">Genetic linkage map of the edible basidiomycete <em>Pleurotus ostreatus</em></div>. <div class="source">Appl. Environ. Microbiol</div> ., <div class="volume">66</div>, <div class="fpage">5290</div>–<div class="lpage">300</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20linkage%20map%20of%20the%20edible%20basidiomycete%20Pleurotus%20ostreatus&amp;author=L.%20M.%20Larraya&amp;author=G.%20Perez&amp;author=E.%20Ritter&amp;author=A.%20G.%20Pisabarro&amp;author=L.%20Ram%C3%ADrez&amp;publication_year=2000&amp;journal=Appl.%20Environ.%20Microbiol&amp;volume=66&amp;pages=5290-300" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AEM.66.12.5290-5300.2000" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.66.12.5290-5300.2000" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FAEM.66.12.5290-5300.2000"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11097904" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw060-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw060-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Park</div> <div class="given-names">S.-K.</div></div>, <div class="name string-name"><div class="surname">Peñas</div> <div class="given-names">M. M.</div></div>, <div class="name string-name"><div class="surname">Ramírez</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Pisabarro</div> <div class="given-names">A. G.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">Genetic linkage map and expression analysis of genes expressed in the lamellae of the edible basidiomycete <em>Pleurotus ostreatus</em></div>. <div class="source">Fungal Genet. Biol. FG B</div> , <div class="volume">43</div>, <div class="fpage">376</div>–<div class="lpage">87</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20linkage%20map%20and%20expression%20analysis%20of%20genes%20expressed%20in%20the%20lamellae%20of%20the%20edible%20basidiomycete%20Pleurotus%20ostreatus&amp;author=S.-K.%20Park&amp;author=M.%20M.%20Pe%C3%B1as&amp;author=L.%20Ram%C3%ADrez&amp;author=A.%20G.%20Pisabarro&amp;publication_year=2006&amp;journal=Fungal%20Genet.%20Biol.%20FG%20B&amp;volume=43&amp;pages=376-87" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.fgb.2006.01.008" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.fgb.2006.01.008" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.fgb.2006.01.008"> </span></div> </div></div></div></div></div><div content-id="dsw060-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw060-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Castanera</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">López-Varas</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Borgognone</div> <div class="given-names">A.</div></div></div>, et al.  . <div class="year">2016</div>, 
<div class="article-title">Transposable elements versus the fungal genome: impact on whole-genome architecture and transcriptional profiles</div>. <div class="source">PLoS Genet</div> ., <div class="volume">12</div>, <div class="fpage">e1006108</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transposable%20elements%20versus%20the%20fungal%20genome%3A%20impact%20on%20whole-genome%20architecture%20and%20transcriptional%20profiles&amp;author=R.%20Castanera&amp;author=L.%20L%C3%B3pez-Varas&amp;author=A.%20Borgognone&amp;publication_year=2016&amp;journal=PLoS%20Genet&amp;volume=12&amp;pages=e1006108" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1006108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1006108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1006108"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27294409" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw060-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw060-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Larraya</div> <div class="given-names">L. M.</div></div>, <div class="name string-name"><div class="surname">Alfonso</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Pisabarro</div> <div class="given-names">A. G.</div></div>, <div class="name string-name"><div class="surname">Ramírez</div> <div class="given-names">L.</div></div></div> <div class="year">2003</div>, 
<div class="article-title">Mapping of genomic regions (quantitative trait loci) controlling production and quality in industrial cultures of the edible basidiomycete <em>Pleurotus ostreatus</em></div>. <div class="source">Appl. Environ. Microbiol</div> ., <div class="volume">69</div>, <div class="fpage">3617</div>–<div class="lpage">25</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mapping%20of%20genomic%20regions%20(quantitative%20trait%20loci)%20controlling%20production%20and%20quality%20in%20industrial%20cultures%20of%20the%20edible%20basidiomycete%20Pleurotus%20ostreatus&amp;author=L.%20M.%20Larraya&amp;author=M.%20Alfonso&amp;author=A.%20G.%20Pisabarro&amp;author=L.%20Ram%C3%ADrez&amp;publication_year=2003&amp;journal=Appl.%20Environ.%20Microbiol&amp;volume=69&amp;pages=3617-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AEM.69.6.3617-3625.2003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.69.6.3617-3625.2003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FAEM.69.6.3617-3625.2003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12788770" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw060-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw060-B29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Riley</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Salamov</div> <div class="given-names">A. A.</div></div>, <div class="name string-name"><div class="surname">Brown</div> <div class="given-names">D. W.</div></div></div>, et al.  . <div class="year">2014</div>, 
<div class="article-title">Extensive sampling of basidiomycete genomes demonstrates inadequacy of the white-rot/brown-rot paradigm for wood decay fungi</div>. <div class="source">Proc. Natl. Acad. Sci. U. S. A</div> ., <div class="volume">111</div>, <div class="fpage">9923</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Extensive%20sampling%20of%20basidiomycete%20genomes%20demonstrates%20inadequacy%20of%20the%20white-rot%2Fbrown-rot%20paradigm%20for%20wood%20decay%20fungi&amp;author=R.%20Riley&amp;author=A.%20A.%20Salamov&amp;author=D.%20W.%20Brown&amp;publication_year=2014&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A&amp;volume=111&amp;pages=9923-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1400592111" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1400592111" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1400592111"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24958869" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw060-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw060-B30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Alfaro</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Castanera</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Lavín</div> <div class="given-names">J. L.</div></div></div>, et al.  . <div class="year">2016</div>, 
<div class="article-title">Comparative and transcriptional analysis of the predicted secretome in the lignocellulose-degrading basidiomycete fungus <em>Pleurotus ostreatus</em>. <em>Environ</em></div><em>.</em> <div class="source">Microbiol</div> .</div></div></div></div><div content-id="dsw060-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw060-B31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Untergasser</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Cutcutache</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Koressaar</div> <div class="given-names">T.</div></div></div>, et al.  . <div class="year">2012</div>, 
<div class="article-title">Primer3-new capabilities and interfaces</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">40</div>.</div></div></div></div><div content-id="dsw060-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw060-B32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Altschul</div> <div class="given-names">S. F.</div></div>, <div class="name string-name"><div class="surname">Gish</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Miller</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Myers</div> <div class="given-names">E. W.</div></div>, <div class="name string-name"><div class="surname">Lipman</div> <div class="given-names">D. J.</div></div></div> <div class="year">1990</div>, 
<div class="article-title">Basic local alignment search tool</div>. <div class="source">J. Mol. Biol</div> ., <div class="volume">215</div>, <div class="fpage">403</div>–<div class="lpage">10</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Basic%20local%20alignment%20search%20tool&amp;author=S.%20F.%20Altschul&amp;author=W.%20Gish&amp;author=W.%20Miller&amp;author=E.%20W.%20Myers&amp;author=D.%20J.%20Lipman&amp;publication_year=1990&amp;journal=J.%20Mol.%20Biol&amp;volume=215&amp;pages=403-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0022-2836(05)80360-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0022-2836(05)80360-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0022-2836(05)80360-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2231712" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw060-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw060-B33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Pfaffl</div> <div class="given-names">M. W.</div></div></div> <div class="year">2001</div>, 
<div class="article-title">A new mathematical model for relative quantification in real-time RT-PCR</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">29</div>, <div class="fpage">e45</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20new%20mathematical%20model%20for%20relative%20quantification%20in%20real-time%20RT-PCR&amp;author=M.%20W.%20Pfaffl&amp;publication_year=2001&amp;journal=Nucleic%20Acids%20Res&amp;volume=29&amp;pages=e45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/29.9.e45" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F29.9.e45" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F29.9.e45"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11328886" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw060-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw060-B34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Eddy</div> <div class="given-names">S. R.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Accelerated Profile HMM Searches Pearson, W. R., (ed.),</div> . <div class="source">PLoS Comput. Biol</div> ., <div class="volume">7</div>, <div class="fpage">e1002195</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Accelerated%20Profile%20HMM%20Searches%20Pearson%2C%20W.%20R.%2C%20(ed.)%2C&amp;author=S.%20R.%20Eddy&amp;publication_year=2011&amp;journal=PLoS%20Comput.%20Biol&amp;volume=7&amp;pages=e1002195" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pcbi.1002195" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pcbi.1002195" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pcbi.1002195"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22039361" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw060-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw060-B35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Finn</div> <div class="given-names">R. D.</div></div>, <div class="name string-name"><div class="surname">Bateman</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Clements</div> <div class="given-names">J.</div></div></div>, et al.  . <div class="year">2014</div>, 
<div class="article-title">Pfam: the protein families database</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">42</div>, <div class="fpage">D222</div>–<div class="lpage">30</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Pfam%3A%20the%20protein%20families%20database&amp;author=R.%20D.%20Finn&amp;author=A.%20Bateman&amp;author=J.%20Clements&amp;publication_year=2014&amp;journal=Nucleic%20Acids%20Res&amp;volume=42&amp;pages=D222-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkt1223" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkt1223" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkt1223"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24288371" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw060-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw060-B36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Marchler-Bauer</div> <div class="given-names">A.</div></div></div> et al.  . <div class="year">2015</div>, 
<div class="article-title">CDD: NCBI’s conserved domain database</div>. <div class="source">Nucleic Acids Res.</div>  <div class="volume">43</div>, <div class="fpage">D222</div>–<div class="lpage">D226</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CDD%3A%20NCBI%E2%80%99s%20conserved%20domain%20database&amp;author=A.%20Marchler-Bauer&amp;publication_year=2015&amp;journal=Nucleic%20Acids%20Res.&amp;volume=43&amp;pages=D222-D226" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gku1221" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgku1221" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgku1221"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25414356" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw060-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw060-B37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Enright</div> <div class="given-names">A. J.</div></div>, <div class="name string-name"><div class="surname">Dongen</div> <div class="given-names">S. Van</div></div>, <div class="name string-name"><div class="surname">Ouzounis</div> <div class="given-names">C. A.</div></div></div> <div class="year">2002</div>, 
<div class="article-title">An efficient algorithm for large-scale detection of protein families</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">30</div>, <div class="fpage">1575</div>–<div class="lpage">84</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20efficient%20algorithm%20for%20large-scale%20detection%20of%20protein%20families&amp;author=A.%20J.%20Enright&amp;author=S.%20Van%20Dongen&amp;author=C.%20A.%20Ouzounis&amp;publication_year=2002&amp;journal=Nucleic%20Acids%20Res&amp;volume=30&amp;pages=1575-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/30.7.1575" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F30.7.1575" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F30.7.1575"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11917018" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw060-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw060-B38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Naas</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Blot</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Fitch</div> <div class="given-names">W. M.</div></div>, <div class="name string-name"><div class="surname">Arber</div> <div class="given-names">W.</div></div></div> <div class="year">1994</div>, 
<div class="article-title">Insertion Sequence-Related Genetic-Variation In Resting <em>Escherichia Coli</em> K-12</div>. <div class="source">Genetics</div> , <div class="volume">136</div>, <div class="fpage">721</div>–<div class="lpage">30</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Insertion%20Sequence-Related%20Genetic-Variation%20In%20Resting%20Escherichia%20Coli%20K-12&amp;author=T.%20Naas&amp;author=M.%20Blot&amp;author=W.%20M.%20Fitch&amp;author=W.%20Arber&amp;publication_year=1994&amp;journal=Genetics&amp;volume=136&amp;pages=721-30" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7911771" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw060-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw060-B39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Capy</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Gasperi</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Biémont</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Bazin</div> <div class="given-names">C.</div></div></div> <div class="year">2000</div>, 
<div class="article-title">Stress and transposable elements: co-evolution or useful parasites?</div>. <div class="source">Heredity (Edinb)</div> ., <div class="volume">85</div>, <div class="fpage">101</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Stress%20and%20transposable%20elements%3A%20co-evolution%20or%20useful%20parasites%3F&amp;author=P.%20Capy&amp;author=G.%20Gasperi&amp;author=C.%20Bi%C3%A9mont&amp;author=C.%20Bazin&amp;publication_year=2000&amp;journal=Heredity%20(Edinb)&amp;volume=85&amp;pages=101-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-2540.2000.00751.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-2540.2000.00751.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-2540.2000.00751.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11012710" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw060-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw060-B40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tittel-Elmer</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Bucher</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Broger</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Mathieu</div> <div class="given-names">O.</div></div>, <div class="name string-name"><div class="surname">Paszkowski</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Vaillant</div> <div class="given-names">I.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Stress-induced activation of heterochromatic transcription</div>. <div class="source">PLoS Genet</div> ., <div class="volume">6</div>, <div class="fpage">1–11</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Stress-induced%20activation%20of%20heterochromatic%20transcription&amp;author=M.%20Tittel-Elmer&amp;author=E.%20Bucher&amp;author=L.%20Broger&amp;author=O.%20Mathieu&amp;author=J.%20Paszkowski&amp;author=I.%20Vaillant&amp;publication_year=2010&amp;journal=PLoS%20Genet&amp;volume=6&amp;pages=1–11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1001175" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1001175" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1001175"> </span></div> </div></div></div></div></div><div content-id="dsw060-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw060-B41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Peabody</div> <div class="given-names">R. B.</div></div>, <div class="name string-name"><div class="surname">Peabody</div> <div class="given-names">D. C.</div></div>, <div class="name string-name"><div class="surname">Sicard</div> <div class="given-names">K. M.</div></div></div> <div class="year">2000</div>, 
<div class="article-title">A genetic mosaic in the fruiting stage of <em>Armillaria gallica</em></div>. <div class="source">Fungal Genet. Biol</div> ., <div class="volume">29</div>, <div class="fpage">72</div>–<div class="lpage">80</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20genetic%20mosaic%20in%20the%20fruiting%20stage%20of%20Armillaria%20gallica&amp;author=R.%20B.%20Peabody&amp;author=D.%20C.%20Peabody&amp;author=K.%20M.%20Sicard&amp;publication_year=2000&amp;journal=Fungal%20Genet.%20Biol&amp;volume=29&amp;pages=72-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/fgbi.2000.1187" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Ffgbi.2000.1187" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Ffgbi.2000.1187"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10919376" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw060-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw060-B42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Peabody</div> <div class="given-names">R. B.</div></div>, <div class="name string-name"><div class="surname">Peabody</div> <div class="given-names">D. C.</div></div>, <div class="name string-name"><div class="surname">Tyrrell</div> <div class="given-names">M. G.</div></div>, <div class="name string-name"><div class="surname">Edenburn-MacQueen</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Howdy</div> <div class="given-names">R. P.</div></div>, <div class="name string-name"><div class="surname">Semelrath</div> <div class="given-names">K. M.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">Haploid vegetative mycelia of <em>Armillaria gallica</em> show among-cell-line variation for growth and phenotypic plasticity</div>. <div class="source">Mycologia</div> , <div class="volume">97</div>, <div class="fpage">777</div>–<div class="lpage">87</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Haploid%20vegetative%20mycelia%20of%20Armillaria%20gallica%20show%20among-cell-line%20variation%20for%20growth%20and%20phenotypic%20plasticity&amp;author=R.%20B.%20Peabody&amp;author=D.%20C.%20Peabody&amp;author=M.%20G.%20Tyrrell&amp;author=E.%20Edenburn-MacQueen&amp;author=R.%20P.%20Howdy&amp;author=K.%20M.%20Semelrath&amp;publication_year=2005&amp;journal=Mycologia&amp;volume=97&amp;pages=777-87" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1080/15572536.2006.11832769" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F15572536.2006.11832769" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1080%2F15572536.2006.11832769"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16457347" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw060-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw060-B43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lamb</div> <div class="given-names">B. C.</div></div></div> <div class="year">1998</div>, 
<div class="article-title">Gene conversion disparity in yeast: its extent, multiple origins, and effects on allele frequencies</div>. <div class="source">Heredity (Edinb)</div> , <div class="volume">80</div> <strong>(</strong><div class="issue">Pt 5</div><strong>)</strong>, <div class="fpage">538</div>–<div class="lpage">52</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20conversion%20disparity%20in%20yeast%3A%20its%20extent%2C%20multiple%20origins%2C%20and%20effects%20on%20allele%20frequencies&amp;author=B.%20C.%20Lamb&amp;publication_year=1998&amp;journal=Heredity%20(Edinb)&amp;volume=80&amp;pages=538-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-2540.1998.00331.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-2540.1998.00331.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-2540.1998.00331.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9650278" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw060-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw060-B44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Galtier</div> <div class="given-names">N.</div></div></div> <div class="year">2001</div>, 
<div class="article-title">GC-Content Evolution in Mammalian Genomes: The Biased Gene Conversion Hypothesis</div>. <div class="source">Dev. Psychobiol</div> ., <div class="volume">39</div>, <div class="fpage">251</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GC-Content%20Evolution%20in%20Mammalian%20Genomes%3A%20The%20Biased%20Gene%20Conversion%20Hypothesis&amp;author=N.%20Galtier&amp;publication_year=2001&amp;journal=Dev.%20Psychobiol&amp;volume=39&amp;pages=251-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1098-2302" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1098-2302" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1098-2302"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11745320" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw060-B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw060-B45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Holliday</div> <div class="given-names">R.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">A mechanism for gene conversion in fungi</div>. <div class="source">Genet. Res</div> ., <div class="volume">89</div>, <div class="fpage">285</div>–<div class="lpage">307</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20mechanism%20for%20gene%20conversion%20in%20fungi&amp;author=R.%20Holliday&amp;publication_year=2007&amp;journal=Genet.%20Res&amp;volume=89&amp;pages=285-307" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1017/S0016672308009476" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1017%2FS0016672308009476" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1017%2FS0016672308009476"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18976517" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw060-B46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw060-B46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sonnenberg</div> <div class="given-names">A. S. M.</div></div>, <div class="name string-name"><div class="surname">Gao</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Lavrijssen</div> <div class="given-names">B.</div></div></div>, et al.  . <div class="year">2016</div>, 
<div class="article-title">A detailed analysis of the recombination landscape of the button mushroom <em>Agaricus bisporus var. bisporus</em></div>. <div class="source">Fungal Genet. Biol</div> ., <div class="volume">93</div>, <div class="fpage">35</div>–<div class="lpage">45</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20detailed%20analysis%20of%20the%20recombination%20landscape%20of%20the%20button%20mushroom%20Agaricus%20bisporus%20var.%20bisporus&amp;author=A.%20S.%20M.%20Sonnenberg&amp;author=W.%20Gao&amp;author=B.%20Lavrijssen&amp;publication_year=2016&amp;journal=Fungal%20Genet.%20Biol&amp;volume=93&amp;pages=35-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.fgb.2016.06.001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.fgb.2016.06.001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.fgb.2016.06.001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27288752" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw060-B47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw060-B47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Holliday</div> <div class="given-names">R.</div></div></div> <div class="year">1982</div>, Gene Conversion: A Possible Mechanism for Eliminating Selfish DNA. <div class="source">Molecular and Cellular Mechanisms of Mutagenesis</div> . 
<div class="publisher-loc">Springer US</div>, 
<div class="publisher-name">Boston, MA</div>, pp. <div class="fpage">259</div>–<div class="lpage">64</div>.</div></div></div></div><div content-id="dsw060-B48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw060-B48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bokor</div> <div class="given-names">A. A. M.</div></div>, <div class="name string-name"><div class="surname">van Kan</div> <div class="given-names">J. A. L.</div></div>, <div class="name string-name"><div class="surname">Poulter</div> <div class="given-names">R. T. M.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Sexual mating of <em>Botrytis cinerea</em> illustrates PRP8 intein HEG activity</div>. <div class="source">Fungal Genet. Biol</div> ., <div class="volume">47</div>, <div class="fpage">392</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sexual%20mating%20of%20Botrytis%20cinerea%20illustrates%20PRP8%20intein%20HEG%20activity&amp;author=A.%20A.%20M.%20Bokor&amp;author=J.%20A.%20L.%20van%20Kan&amp;author=R.%20T.%20M.%20Poulter&amp;publication_year=2010&amp;journal=Fungal%20Genet.%20Biol&amp;volume=47&amp;pages=392-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.fgb.2010.01.003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.fgb.2010.01.003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.fgb.2010.01.003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20093192" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw060-B49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw060-B49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sun</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Wyngaard</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Walton</div> <div class="given-names">D.</div></div></div>, et al.  . <div class="year">2014</div>, 
<div class="article-title">Billions of basepairs of recently expanded, repetitive sequences are eliminated from the somatic genome during copepod development</div>. <div class="source">BMC Genomics</div> , <div class="volume">15</div>, <div class="fpage">186</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Billions%20of%20basepairs%20of%20recently%20expanded%2C%20repetitive%20sequences%20are%20eliminated%20from%20the%20somatic%20genome%20during%20copepod%20development&amp;author=C.%20Sun&amp;author=G.%20Wyngaard&amp;author=D.%20Walton&amp;publication_year=2014&amp;journal=BMC%20Genomics&amp;volume=15&amp;pages=186" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-15-186" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-15-186" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-15-186"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24618421" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw060-B50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw060-B50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kataoka</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Mochizuki</div> <div class="given-names">K.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Programmed DNA elimination in tetrahymena: A small RNA-mediated genome surveillance mechanism</div>. <div class="source">Adv. Exp. Med. Biol</div> ., <div class="volume">722</div>, <div class="fpage">156</div>–<div class="lpage">73</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Programmed%20DNA%20elimination%20in%20tetrahymena%3A%20A%20small%20RNA-mediated%20genome%20surveillance%20mechanism&amp;author=K.%20Kataoka&amp;author=K.%20Mochizuki&amp;publication_year=2011&amp;journal=Adv.%20Exp.%20Med.%20Biol&amp;volume=722&amp;pages=156-73" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21915788" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw060-B51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw060-B51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Dang</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Yang</div> <div class="given-names">Q.</div></div>, <div class="name string-name"><div class="surname">Xue</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">Y.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">RNA interference in fungi: Pathways, functions, and applications</div>. <div class="source">Eukaryot. Cell</div> , pp. <div class="fpage">1148</div>–<div class="lpage">55</div>.</div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2017. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=2962881&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw060/2/dsw060_Supp.zip?Expires=2147483647&amp;Signature=zDAtgxa4F-UL54z2if0eYAx7iiuGb6srTDL1CT7Hx7wDxFEom1ojV1lVEE9Us27ThDolTKyt9Km4jlX7dNK4MHhuRaD32ktxUaSGDSkmnZRKWib00mcJHlAT~5TrnVNjQG5nML7AyU1hHzg~KRGemgvdez0x0d46-pjP6K0JGmfKcntTePu8ATjm8~7UeUJ2ducPfWRDovHNFdYPhH6uJ8yF-~DSkp7EeFNFkhyacpRLpdfEj97SkZCC1qTS06IJaQxL19jmK1BvUZWM6~NYrK7MU-sm57TX44qf0BU5aLXa6LgFrASDYk-iu7SgJew7ClWCHfbFZ8ttSGEcp8o3-g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">694</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000400913000001&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">1</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsw060"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="2962881"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000400913000001&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f24%2f2%2f103%2f2962881">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000400913000001&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (1)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F24%2F2%2F103%2F2962881" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/2962881'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Somatic transposition and meiotically driven elimination of an active helitron family in Pleurotus ostreatus" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsw060");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f24%2f2%2f103%2f2962881">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

