

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/24/5/499/3835854","@type":"ScholarlyArticle","name":"The genome sequence of sweet cherry ( Prunus avium ) for use in genomics-assisted breeding","datePublished":"2017-05-25","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/24/5","@type":"PublicationIssue","issueNumber":"5","datePublished":"2017-10-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsx020","keywords":["draft genome","genetic map","genomics-assisted breeding","sweet cherry (<italic>Prunus avium</italic>)"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/24/5/499/3835854","author":[{"name":"Shirasawa, Kenta","affiliation":"1 Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan","@type":"Person"},{"name":"Isuzugawa, Kanji","affiliation":"2 Horticultural Experiment Station, Yamagata Integrated Agricultural Research Center, Sagae, Yamagata 991-0043, Japan","@type":"Person"},{"name":"Ikenaga, Mitsunobu","affiliation":"3 Central Agricultural Experiment Station, Agricultural Research Department, Hokkaido Research Organization, Naganuma, Hokkaido 069-1395, Japan","@type":"Person"},{"name":"Saito, Yutaro","affiliation":"2 Horticultural Experiment Station, Yamagata Integrated Agricultural Research Center, Sagae, Yamagata 991-0043, Japan","@type":"Person"},{"name":"Yamamoto, Toshiya","affiliation":"4 Institute of Fruit Tree and Tea Science, National Agriculture and Food Research Organization, Tsukuba, Ibaraki 305-8605, Japan","@type":"Person"},{"name":"Hirakawa, Hideki","affiliation":"1 Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan","@type":"Person"},{"name":"Isobe, Sachiko","affiliation":"1 Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan","@type":"Person"}],"description":"Abstract.  We determined the genome sequence of sweet cherry (Prunus avium) using next-generation sequencing technology. The total length of the assembled seque","pageStart":"499","pageEnd":"508","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/5/1/m_dnares_24_5cover.png?Expires=2147483647&Signature=jul3oxjgkP675GHMbQEy~zZ1my2m2g0xI0mr9LkJEjA-PhoC4BnM1ARo9FdXFQKdaeXjXdDG8Th7l01-PuT824GoN40EPQF0GwZOV5tGDYZCguawjvcFtz41hVQe~O6T29XL-3ITGszkj0QvmRZLajQvyRnp825zpQTjNCOtAlB0rKMc7Dcdixzjc2uv2jilEDMqD9YZCuMkOXRLrwDQftwYgb~q8EavwLngcuRRSkILOW9lFSQBCUNAqNd4mDnFW5RKty7z4w3HYGKz5wJomnNLGyCwwAyELyFWHWV4xtKHzE5xEUd8do1ZdHc6pZe4TftEycCVzz1M4dnVR5tpug__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"The genome sequence of sweet cherry ( Prunus avium ) for use in genomics-assisted breeding","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/5/1/m_dnares_24_5cover.png?Expires=2147483647&Signature=jul3oxjgkP675GHMbQEy~zZ1my2m2g0xI0mr9LkJEjA-PhoC4BnM1ARo9FdXFQKdaeXjXdDG8Th7l01-PuT824GoN40EPQF0GwZOV5tGDYZCguawjvcFtz41hVQe~O6T29XL-3ITGszkj0QvmRZLajQvyRnp825zpQTjNCOtAlB0rKMc7Dcdixzjc2uv2jilEDMqD9YZCuMkOXRLrwDQftwYgb~q8EavwLngcuRRSkILOW9lFSQBCUNAqNd4mDnFW5RKty7z4w3HYGKz5wJomnNLGyCwwAyELyFWHWV4xtKHzE5xEUd8do1ZdHc6pZe4TftEycCVzz1M4dnVR5tpug__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="genome sequence of sweet cherry ( Prunus avium ) for use in genomics-assisted breeding" />
    <meta property="og:description" content="Abstract.  We determined the genome sequence of sweet cherry (Prunus avium) using next-generation sequencing technology. The total length of the assembled seque" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/24/5/499/3835854" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2017-05-25" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Shirasawa, Kenta" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Isuzugawa, Kanji" /><meta name="citation_author_institution" content="Horticultural Experiment Station, Yamagata Integrated Agricultural Research Center, Sagae, Yamagata 991-0043, Japan" /><meta name="citation_author" content="Ikenaga, Mitsunobu" /><meta name="citation_author_institution" content="Central Agricultural Experiment Station, Agricultural Research Department, Hokkaido Research Organization, Naganuma, Hokkaido 069-1395, Japan" /><meta name="citation_author" content="Saito, Yutaro" /><meta name="citation_author_institution" content="Horticultural Experiment Station, Yamagata Integrated Agricultural Research Center, Sagae, Yamagata 991-0043, Japan" /><meta name="citation_author" content="Yamamoto, Toshiya" /><meta name="citation_author_institution" content="Institute of Fruit Tree and Tea Science, National Agriculture and Food Research Organization, Tsukuba, Ibaraki 305-8605, Japan" /><meta name="citation_author" content="Hirakawa, Hideki" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Isobe, Sachiko" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_title" content="The genome sequence of sweet cherry (Prunus avium) for use in genomics-assisted breeding" /><meta name="citation_firstpage" content="499" /><meta name="citation_lastpage" content="508" /><meta name="citation_doi" content="10.1093/dnares/dsx020" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="24" /><meta name="citation_issue" content="5" /><meta name="citation_publication_date" content="2017/10/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=The genome of the domesticated apple (Malus Ã— domestica Borkh.);  citation_journal_title=Nat. Genet;  citation_year=2010;  citation_volume=42;  citation_pages=833-9; " /><meta name="citation_reference" content="citation_title=The high-quality draft genome of peach (Prunus persica) identifies unique patterns of genetic diversity, domestication and genome evolution; citation_author=International, Peach Genome Initiative;  citation_journal_title=Nat. Genet;  citation_year=2013;  citation_volume=45;  citation_pages=487-94; " /><meta name="citation_reference" content="citation_title=The genome of the pear (Pyrus bretschneideri Rehd.);  citation_journal_title=Genome Res;  citation_year=2013;  citation_volume=23;  citation_pages=396-408; " /><meta name="citation_reference" content="citation_title=The draft genome sequence of European pear (Pyrus communis L. â€²Bartlettâ€²);  citation_journal_title=PLoS One;  citation_year=2014;  citation_volume=9;  citation_pages=e92644" /><meta name="citation_reference" content="citation_title=The genome of Prunus mume,;  citation_journal_title=Nat. Commun;  citation_year=2012;  citation_volume=3;  citation_pages=1318" /><meta name="citation_reference" content="citation_title=The genome of woodland strawberry (Fragaria vesca);  citation_journal_title=Nat. Genet;  citation_year=2011;  citation_volume=43;  citation_pages=109-16; " /><meta name="citation_reference" content="citation_title=Dissection of the octoploid strawberry genome by deep sequencing of the genomes of Fragaria species,;  citation_journal_title=DNA Res;  citation_year=2014;  citation_volume=21;  citation_pages=169-81; " /><meta name="citation_reference" content="citation_title=Progress, challenges and the future of crop genomes,;  citation_journal_title=Curr. Opin. Plant Biol;  citation_year=2015;  citation_volume=24;  citation_pages=71-81; " /><meta name="citation_reference" content="citation_title=Construction of high density sweet cherry (Prunus avium L.) linkage maps using microsatellite markers and SNPs detected by genotyping-by-sequencing (GBS),;  citation_journal_title=PLoS One;  citation_year=2015;  citation_volume=10;  citation_pages=e0127750" /><meta name="citation_reference" content="citation_title=Comparative mapping and marker-assisted selection in Rosaceae fruit crops;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2004;  citation_volume=101;  citation_pages=9891-6; " /><meta name="citation_reference" content="citation_title=Genomics of pear and other Rosaceae fruit trees,;  citation_journal_title=Breed. Sci;  citation_year=2016;  citation_volume=66;  citation_pages=148-59; " /><meta name="citation_reference" content="citation_title=Comparative genomics analysis in Prunoideae to identify biologically relevant polymorphisms,;  citation_journal_title=Plant Biotechnol. J;  citation_year=2013;  citation_volume=11;  citation_pages=883-93; " /><meta name="citation_reference" content="citation_title=Comparative analysis of rosaceous genomes and the reconstruction of a putative ancestral genome for the family,;  citation_journal_title=BMC Evol. Biol;  citation_year=2011;  citation_volume=11;  citation_pages=9" /><meta name="citation_reference" content="citation_title=Development and evaluation of a genome-wide 6K SNP array for diploid sweet cherry and tetraploid sour cherry,;  citation_journal_title=PLoS One;  citation_year=2012;  citation_volume=7;  citation_pages=e48305" /><meta name="citation_reference" content="citation_title=Illumina mate-paired DNA sequencing-library preparation using Cre-Lox recombination;  citation_journal_title=Nucl. Acids Res;  citation_year=2012;  citation_volume=40;  citation_pages=e24" /><meta name="citation_reference" content="citation_title=Quality control and preprocessing of metagenomic datasets,;  citation_journal_title=Bioinformatics;  citation_year=2011;  citation_volume=27;  citation_pages=863-4; " /><meta name="citation_reference" content="citation_title=A fast, lock-free approach for efficient parallel counting of occurrences of k-mers,;  citation_journal_title=Bioinformatics;  citation_year=2011;  citation_volume=27;  citation_pages=764-70; " /><meta name="citation_reference" content="citation_title=SOAPdenovo2: an empirically improved memory-efficient short-read de novo assembler,;  citation_journal_title=Gigascience;  citation_year=2012;  citation_volume=1;  citation_pages=18" /><meta name="citation_reference" content="citation_title=Efficient de novo assembly of highly heterozygous genomes from whole-genome shotgun short reads,;  citation_journal_title=Genome Res;  citation_year=2014;  citation_volume=24;  citation_pages=1384-95; " /><meta name="citation_reference" content="citation_title=BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs,;  citation_journal_title=Bioinformatics;  citation_year=2015;  citation_volume=31;  citation_pages=3210-2; " /><meta name="citation_reference" content="citation_title=Repbase Update, a database of eukaryotic repetitive elements,;  citation_journal_title=Cytogenet. Genome Res;  citation_year=2005;  citation_volume=110;  citation_pages=462-7; " /><meta name="citation_reference" content="citation_title=De novo identification of repeat families in large genomes,;  citation_journal_title=Bioinformatics;  citation_year=2005;  citation_volume=21;  citation_pages=i351-8; " /><meta name="citation_reference" content="  Smit A.F.A., Hubley R., Green P.  2013 -2015, RepeatMasker Open-4.0. http://www.repeatmasker.org (8 May 2017, date last accessed). " /><meta name="citation_reference" content="citation_title=PacCYP707A2 negatively regulates cherry fruit ripening while PacCYP707A1 mediates drought tolerance,;  citation_journal_title=J. Exp. Bot;  citation_year=2015;  citation_volume=66;  citation_pages=3765-74; " /><meta name="citation_reference" content="citation_title=Full-length transcriptome assembly from RNA-seq data without a reference genome,;  citation_journal_title=Nat. Biotechnol;  citation_year=2011;  citation_volume=29;  citation_pages=644-52; " /><meta name="citation_reference" content="citation_title=tRNAscan-SE: a program for improved detection of transfer RNA genes in genomic sequence,;  citation_journal_title=Nucl. Acids Res;  citation_year=1997;  citation_volume=25;  citation_pages=955-64; " /><meta name="citation_reference" content="citation_title=MAKER: an easy-to-use annotation pipeline designed for emerging model organism genomes;  citation_journal_title=Genome Res;  citation_year=2008;  citation_volume=18;  citation_pages=188-96; " /><meta name="citation_reference" content="citation_title=Evolutionary origins and dynamics of octoploid strawberry subgenomes revealed by dense targeted capture linkage maps,;  citation_journal_title=Genome. Biol. Evol;  citation_year=2014;  citation_volume=6;  citation_pages=3295-313; " /><meta name="citation_reference" content="citation_title=BRAKER1: unsupervised RNA-Seq-based genome annotation with GeneMark-ET and AUGUSTUS,;  citation_journal_title=Bioinformatics;  citation_year=2016;  citation_volume=32;  citation_pages=767-9; " /><meta name="citation_reference" content="citation_title=InterProScan: protein domains identifier,;  citation_journal_title=Nucl. Acids Res;  citation_year=2005;  citation_volume=33;  citation_pages=W116-20; " /><meta name="citation_reference" content="citation_title=New developments in the InterPro database,;  citation_journal_title=Nucl. Acids Res;  citation_year=2007;  citation_volume=35;  citation_pages=D224-8; " /><meta name="citation_reference" content="citation_title=Cd-hit: a fast program for clustering and comparing large sets of protein or nucleotide sequences,;  citation_journal_title=Bioinformatics;  citation_year=2006;  citation_volume=22;  citation_pages=1658-9; " /><meta name="citation_reference" content="citation_title=Gene Ontology: tool for the unification of biology,;  citation_journal_title=Nat. Genet;  citation_year=2000;  citation_volume=25;  citation_pages=25-9; " /><meta name="citation_reference" content="citation_title=The COG database: an updated version includes eukaryotes,;  citation_journal_title=BMC Bioinform;  citation_year=2003;  citation_volume=4;  citation_pages=41" /><meta name="citation_reference" content="citation_title=KEGG: kyoto encyclopedia of genes and genomes,;  citation_journal_title=Nucl. Acids Res;  citation_year=1999;  citation_volume=27;  citation_pages=29-34; " /><meta name="citation_reference" content="citation_title=Analytical workflow of double-digest restriction site-associated DNA sequencing based on empirical and in silico optimization in tomato,;  citation_journal_title=DNA Res;  citation_year=2016;  citation_volume=23;  citation_pages=145-53; " /><meta name="citation_reference" content="citation_title=Fast gapped-read alignment with Bowtie 2,;  citation_journal_title=Nat. Methods;  citation_year=2012;  citation_volume=9;  citation_pages=357-59; " /><meta name="citation_reference" content="citation_title=The sequence alignment/map format and SAMtools,;  citation_journal_title=Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=2078-9; " /><meta name="citation_reference" content="citation_title=Rapid and accurate haplotype phasing and missing-data inference for whole-genome association studies by use of localized haplotype clustering,;  citation_journal_title=Am. J. Hum. Genet;  citation_year=2007;  citation_volume=81;  citation_pages=1084-97; " /><meta name="citation_reference" content="citation_title=The variant call format and VCFtools,;  citation_journal_title=Bioinformatics;  citation_year=2011;  citation_volume=27;  citation_pages=2156-8; " /><meta name="citation_reference" content="citation_title=Development and variability analysis of microsatellite markers in peach,;  citation_journal_title=Plant Breed;  citation_year=2002;  citation_volume=121;  citation_pages=87-92; " /><meta name="citation_reference" content="citation_title=DNA fingerprinting of tetraploid cherry germplasm using simple sequence repeats;  citation_journal_title=J. Amer. Soc. Hort. Sci;  citation_year=2001;  citation_volume=216;  citation_pages=205-9; " /><meta name="citation_reference" content="citation_title=AC/GT and AG/CT microsatellite repeats in peach [Prunus persica (L) Batsch]: isolation, characterisation and cross-species amplification in Prunus,;  citation_journal_title=Theor. Appl. Genet;  citation_year=1999;  citation_volume=99;  citation_pages=65-72; " /><meta name="citation_reference" content="citation_title=Development and characterization of polymorphic microsatellites from Prunus avium â€²Napoleonâ€²,;  citation_journal_title=Mol. Ecol. Notes;  citation_year=2003;  citation_volume=3;  citation_pages=578-80; " /><meta name="citation_reference" content="citation_title=Development of microsatellite markers in peach [Prunus persica (L.) Batsch] and their use in genetic diversity analysis in peach and sweet cherry (Prunus avium L.);  citation_journal_title=Theor. Appl. Genet;  citation_year=2002;  citation_volume=105;  citation_pages=127-38; " /><meta name="citation_reference" content="citation_title=Development of a second generation linkage map for almond using RAPD and SSR markers,;  citation_journal_title=Genome;  citation_year=2000;  citation_volume=43;  citation_pages=649-55; " /><meta name="citation_reference" content="citation_title=GDR (Genome Database for Rosaceae): integrated web-database for Rosaceae genomics and genetics data,;  citation_journal_title=Nucl. Acids Res;  citation_year=2008;  citation_volume=36;  citation_pages=D1034-40; " /><meta name="citation_reference" content="citation_title=Identification of microsatellite loci in apricot,;  citation_journal_title=Mol. Ecol. Notes;  citation_year=2002;  citation_volume=2;  citation_pages=24-6; " /><meta name="citation_reference" content="citation_title=Development and transportability across Prunus species of 42 polymorphic almond microsatellites,;  citation_journal_title=Mol. Ecol. Resour;  citation_year=2005;  citation_volume=5;  citation_pages=531-5; " /><meta name="citation_reference" content="citation_title=Prunus microsatellite marker transferability across rosaceous crops,;  citation_journal_title=Tree Genet. Genomes;  citation_year=2010;  citation_volume=6;  citation_pages=689-700; " /><meta name="citation_reference" content="citation_title=Characterization of microsatellite markers in peach [Prunus persica (L.) Batsch];  citation_journal_title=Theor. Appl. Genet;  citation_year=2000;  citation_volume=101;  citation_pages=421-8; " /><meta name="citation_reference" content="citation_title=Analysis of sweet cherry (Prunus avium L.) cultivars using SSR and AFLP markers;  citation_journal_title=J. Am. Soc. Hor. Sci;  citation_year=2003;  citation_volume=128;  citation_pages=904-9; " /><meta name="citation_reference" content="citation_title=Microsatellite DNA in peach (Prunus persica L. Batsch) and its use in fingerprinting and testing the genetic origin of cultivars,;  citation_journal_title=Genome;  citation_year=2000;  citation_volume=43;  citation_pages=512-20; " /><meta name="citation_reference" content="citation_title=Characterization of novel microsatellites and development of multiplex PCR for large-scale population studies in wild cherry, Prunus avium,;  citation_journal_title=Mol. Ecol. Resour;  citation_year=2004;  citation_volume=4;  citation_pages=429-31; " /><meta name="citation_reference" content="citation_title=Microsatellite markers in peach [Prunus persica (L.) Batsch] derived from an enriched genomic and cDNA libraries,;  citation_journal_title=Mol. Ecol. Resour;  citation_year=2002;  citation_volume=2;  citation_pages=298-301; " /><meta name="citation_reference" content="citation_title=An integrated genetic linkage map of peach by SSR, STS, AFLP and RAPD,;  citation_journal_title=J. Japan. Soc. Hort. Sci;  citation_year=2005;  citation_volume=74;  citation_pages=204-13; " /><meta name="citation_reference" content="citation_publisher=Kyazma BV, Wageningen, The Netherlands;  citation_title=JoinMapÂ®4, software for the calculation of genetic linkage maps in experimental populations;  citation_year=2006; " /><meta name="citation_reference" content="citation_title=LPmerge: an R package for merging genetic maps by linear programming,;  citation_journal_title=Bioinformatics;  citation_year=2014;  citation_volume=30;  citation_pages=1623-4; " /><meta name="citation_reference" content="citation_title=MapChart: software for the graphical presentation of linkage maps and QTLs,;  citation_journal_title=J. Hered;  citation_year=2002;  citation_volume=93;  citation_pages=77-8; " /><meta name="citation_reference" content="citation_title=Circos: an information aesthetic for comparative genomics,;  citation_journal_title=Genome Res;  citation_year=2009;  citation_volume=19;  citation_pages=1639-45; " /><meta name="citation_reference" content="citation_title=Fast algorithms for large-scale genome alignment and comparison,;  citation_journal_title=Nucl. Acids Res;  citation_year=2002;  citation_volume=30;  citation_pages=2478-83; " /><meta name="citation_reference" content="citation_title=A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of Drosophila melanogaster strain w1118; iso-2; iso-3;  citation_journal_title=Fly;  citation_year=2012;  citation_volume=6;  citation_pages=80-92; " /><meta name="citation_reference" content="citation_title=The Sequence Ontology: a tool for the unification of genome annotations,;  citation_journal_title=Genome Biol;  citation_year=2005;  citation_volume=6;  citation_pages=R44" /><meta name="citation_reference" content="citation_title=CNV-seq, a new method to detect copy number variation using high-throughput sequencing,;  citation_journal_title=BMC Bioinform;  citation_year=2009;  citation_volume=10;  citation_pages=80" /><meta name="citation_reference" content="citation_title=SNP2CAPS: a SNP and INDEL analysis tool for CAPS marker development,;  citation_journal_title=Nucl. Acids Res;  citation_year=2004;  citation_volume=32;  citation_pages=e5" /><meta name="citation_reference" content="citation_title=SciRoKo: a new tool for whole genome microsatellite search and investigation,;  citation_journal_title=Bioinformatics;  citation_year=2007;  citation_volume=23;  citation_pages=1683-5; " /><meta name="citation_reference" content="citation_title=Primer3 on the WWW for general users and for biologist programmers,;  citation_journal_title=Methods Mol. Biol;  citation_year=2000;  citation_volume=132;  citation_pages=365-86; " /><meta name="citation_reference" content="citation_title=Nuclear DNA content of some important plant,;  citation_journal_title=Plant Mol. Biol. Rep;  citation_year=1991;  citation_volume=9;  citation_pages=208-18; " /><meta name="citation_reference" content="citation_title=Quantitative measures for the management and comparison of annotated genomes;  citation_journal_title=BMC Bioinform;  citation_year=2009;  citation_volume=10;  citation_pages=67" /><meta name="citation_reference" content="citation_title=Genome-wide association study of 12 agronomic traits in peach,;  citation_journal_title=Nat. Commun;  citation_year=2016;  citation_volume=7;  citation_pages=13246" /><meta name="citation_reference" content="citation_title=Plant Genome DataBase Japan (PGDBj): a portal website for the integration of plant genome-related databases,;  citation_journal_title=Plant Cell Physiol;  citation_year=2014;  citation_volume=55;  citation_pages=e8" /><meta name="citation_reference" content="citation_title=Genomics-assisted breeding in fruit trees;  citation_journal_title=Breed. Sci;  citation_year=2016;  citation_volume=66;  citation_pages=100-15; " /><meta name="citation_reference" content="citation_title=Deployment of new biotechnologies in plant breeding,;  citation_journal_title=Nat. Biotechnol;  citation_year=2012;  citation_volume=30;  citation_pages=231-9; " /><meta name="citation_reference" content="citation_title=Genomic prediction of trait segregation in a progeny population: a case study of Japanese pear (Pyrus pyrifolia);  citation_journal_title=BMC Genet;  citation_year=2013;  citation_volume=14;  citation_pages=81" /><meta name="citation_reference" content="citation_title=Reduced generation time of apple seedlings to within a year by means of a plant virus vector: a new plant-breeding technique with no transmission of genetic modification to the next generation,;  citation_journal_title=Plant Biotechnol. J;  citation_year=2014;  citation_volume=12;  citation_pages=60-8; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/24/5/499/22893079/dsx020.pdf" /><meta name="description" content="Abstract.  We determined the genome sequence of sweet cherry (Prunus avium) using next-generation sequencing technology. The total length of the assembled seque" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/24/5/499/3835854" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/24/5/499/3835854" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/24/5/499/3835854");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsx020';
    </script>
<title>
	genome sequence of sweet cherry (Prunus avium) for use in genomics-assisted breeding | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./3835854" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BYWmATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2hpcmFzYXdhLCBLZW50YSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSXN1enVnYXdhLCBLYW5qaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkhvcnRpY3VsdHVyYWwgRXhwZXJpbWVudCBTdGF0aW9uLCBZYW1hZ2F0YSBJbnRlZ3JhdGVkIEFncmljdWx0dXJhbCBSZXNlYXJjaCBDZW50ZXIsIFNhZ2FlLCBZYW1hZ2F0YSA5OTEtMDA0MywgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJJa2VuYWdhLCBNaXRzdW5vYnUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJDZW50cmFsIEFncmljdWx0dXJhbCBFeHBlcmltZW50IFN0YXRpb24sIEFncmljdWx0dXJhbCBSZXNlYXJjaCBEZXBhcnRtZW50LCBIb2trYWlkbyBSZXNlYXJjaCBPcmdhbml6YXRpb24sIE5hZ2FudW1hLCBIb2trYWlkbyAwNjktMTM5NSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTYWl0bywgWXV0YXJvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSG9ydGljdWx0dXJhbCBFeHBlcmltZW50IFN0YXRpb24sIFlhbWFnYXRhIEludGVncmF0ZWQgQWdyaWN1bHR1cmFsIFJlc2VhcmNoIENlbnRlciwgU2FnYWUsIFlhbWFnYXRhIDk5MS0wMDQzLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IllhbWFtb3RvLCBUb3NoaXlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSW5zdGl0dXRlIG9mIEZydWl0IFRyZWUgYW5kIFRlYSBTY2llbmNlLCBOYXRpb25hbCBBZ3JpY3VsdHVyZSBhbmQgRm9vZCBSZXNlYXJjaCBPcmdhbml6YXRpb24sIFRzdWt1YmEsIEliYXJha2kgMzA1LTg2MDUsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSGlyYWthd2EsIEhpZGVraSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSXNvYmUsIFNhY2hpa28iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZSwgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iVGhlIGdlbm9tZSBzZXF1ZW5jZSBvZiBzd2VldCBjaGVycnkgKFBydW51cyBhdml1bSkgZm9yIHVzZSBpbiBnZW5vbWljcy1hc3Npc3RlZCBicmVlZGluZyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjQ5OSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iNTA4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHN4MDIwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjI0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSI1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTcvMTAvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZ2Vub21lIG9mIHRoZSBkb21lc3RpY2F0ZWQgYXBwbGUgKE1hbHVzIMOXIGRvbWVzdGljYSBCb3JraC4pOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTQyOyAgY2l0YXRpb25fcGFnZXM9ODMzLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBoaWdoLXF1YWxpdHkgZHJhZnQgZ2Vub21lIG9mIHBlYWNoIChQcnVudXMgcGVyc2ljYSkgaWRlbnRpZmllcyB1bmlxdWUgcGF0dGVybnMgb2YgZ2VuZXRpYyBkaXZlcnNpdHksIGRvbWVzdGljYXRpb24gYW5kIGdlbm9tZSBldm9sdXRpb247IGNpdGF0aW9uX2F1dGhvcj1JbnRlcm5hdGlvbmFsLCBQZWFjaCBHZW5vbWUgSW5pdGlhdGl2ZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT00NTsgIGNpdGF0aW9uX3BhZ2VzPTQ4Ny05NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGdlbm9tZSBvZiB0aGUgcGVhciAoUHlydXMgYnJldHNjaG5laWRlcmkgUmVoZC4pOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgUmVzOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTIzOyAgY2l0YXRpb25fcGFnZXM9Mzk2LTQwODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGRyYWZ0IGdlbm9tZSBzZXF1ZW5jZSBvZiBFdXJvcGVhbiBwZWFyIChQeXJ1cyBjb21tdW5pcyBMLiDigLJCYXJ0bGV0dOKAsik7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgT25lOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz1lOTI2NDQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZ2Vub21lIG9mIFBydW51cyBtdW1lLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBDb21tdW47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MzsgIGNpdGF0aW9uX3BhZ2VzPTEzMTgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZ2Vub21lIG9mIHdvb2RsYW5kIHN0cmF3YmVycnkgKEZyYWdhcmlhIHZlc2NhKTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT00MzsgIGNpdGF0aW9uX3BhZ2VzPTEwOS0xNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlzc2VjdGlvbiBvZiB0aGUgb2N0b3Bsb2lkIHN0cmF3YmVycnkgZ2Vub21lIGJ5IGRlZXAgc2VxdWVuY2luZyBvZiB0aGUgZ2Vub21lcyBvZiBGcmFnYXJpYSBzcGVjaWVzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RE5BIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTE2OS04MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJvZ3Jlc3MsIGNoYWxsZW5nZXMgYW5kIHRoZSBmdXR1cmUgb2YgY3JvcCBnZW5vbWVzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q3Vyci4gT3Bpbi4gUGxhbnQgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTcxLTgxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db25zdHJ1Y3Rpb24gb2YgaGlnaCBkZW5zaXR5IHN3ZWV0IGNoZXJyeSAoUHJ1bnVzIGF2aXVtIEwuKSBsaW5rYWdlIG1hcHMgdXNpbmcgbWljcm9zYXRlbGxpdGUgbWFya2VycyBhbmQgU05QcyBkZXRlY3RlZCBieSBnZW5vdHlwaW5nLWJ5LXNlcXVlbmNpbmcgKEdCUyksOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIE9uZTsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPWUwMTI3NzUwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyYXRpdmUgbWFwcGluZyBhbmQgbWFya2VyLWFzc2lzdGVkIHNlbGVjdGlvbiBpbiBSb3NhY2VhZSBmcnVpdCBjcm9wczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTAxOyAgY2l0YXRpb25fcGFnZXM9OTg5MS02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWljcyBvZiBwZWFyIGFuZCBvdGhlciBSb3NhY2VhZSBmcnVpdCB0cmVlcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJyZWVkLiBTY2k7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9NjY7ICBjaXRhdGlvbl9wYWdlcz0xNDgtNTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBhcmF0aXZlIGdlbm9taWNzIGFuYWx5c2lzIGluIFBydW5vaWRlYWUgdG8gaWRlbnRpZnkgYmlvbG9naWNhbGx5IHJlbGV2YW50IHBvbHltb3JwaGlzbXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBCaW90ZWNobm9sLiBKOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9ODgzLTkzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wYXJhdGl2ZSBhbmFseXNpcyBvZiByb3NhY2VvdXMgZ2Vub21lcyBhbmQgdGhlIHJlY29uc3RydWN0aW9uIG9mIGEgcHV0YXRpdmUgYW5jZXN0cmFsIGdlbm9tZSBmb3IgdGhlIGZhbWlseSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBFdm9sLiBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9OSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldmVsb3BtZW50IGFuZCBldmFsdWF0aW9uIG9mIGEgZ2Vub21lLXdpZGUgNksgU05QIGFycmF5IGZvciBkaXBsb2lkIHN3ZWV0IGNoZXJyeSBhbmQgdGV0cmFwbG9pZCBzb3VyIGNoZXJyeSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgT25lOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz1lNDgzMDUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbGx1bWluYSBtYXRlLXBhaXJlZCBETkEgc2VxdWVuY2luZy1saWJyYXJ5IHByZXBhcmF0aW9uIHVzaW5nIENyZS1Mb3ggcmVjb21iaW5hdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbC4gQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTQwOyAgY2l0YXRpb25fcGFnZXM9ZTI0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UXVhbGl0eSBjb250cm9sIGFuZCBwcmVwcm9jZXNzaW5nIG9mIG1ldGFnZW5vbWljIGRhdGFzZXRzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9Mjc7ICBjaXRhdGlvbl9wYWdlcz04NjMtNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBmYXN0LCBsb2NrLWZyZWUgYXBwcm9hY2ggZm9yIGVmZmljaWVudCBwYXJhbGxlbCBjb3VudGluZyBvZiBvY2N1cnJlbmNlcyBvZiBrLW1lcnMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0yNzsgIGNpdGF0aW9uX3BhZ2VzPTc2NC03MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U09BUGRlbm92bzI6IGFuIGVtcGlyaWNhbGx5IGltcHJvdmVkIG1lbW9yeS1lZmZpY2llbnQgc2hvcnQtcmVhZCBkZSBub3ZvIGFzc2VtYmxlciw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdpZ2FzY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTE7ICBjaXRhdGlvbl9wYWdlcz0xOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVmZmljaWVudCBkZSBub3ZvIGFzc2VtYmx5IG9mIGhpZ2hseSBoZXRlcm96eWdvdXMgZ2Vub21lcyBmcm9tIHdob2xlLWdlbm9tZSBzaG90Z3VuIHNob3J0IHJlYWRzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTEzODQtOTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJVU0NPOiBhc3Nlc3NpbmcgZ2Vub21lIGFzc2VtYmx5IGFuZCBhbm5vdGF0aW9uIGNvbXBsZXRlbmVzcyB3aXRoIHNpbmdsZS1jb3B5IG9ydGhvbG9ncyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTMxOyAgY2l0YXRpb25fcGFnZXM9MzIxMC0yOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZXBiYXNlIFVwZGF0ZSwgYSBkYXRhYmFzZSBvZiBldWthcnlvdGljIHJlcGV0aXRpdmUgZWxlbWVudHMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DeXRvZ2VuZXQuIEdlbm9tZSBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTEwOyAgY2l0YXRpb25fcGFnZXM9NDYyLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURlIG5vdm8gaWRlbnRpZmljYXRpb24gb2YgcmVwZWF0IGZhbWlsaWVzIGluIGxhcmdlIGdlbm9tZXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPWkzNTEtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iICBTbWl0IEEuRi5BLiwgSHVibGV5IFIuLCBHcmVlbiBQLiAgMjAxMyAtMjAxNSwgUmVwZWF0TWFza2VyIE9wZW4tNC4wLiBodHRwOi8vd3d3LnJlcGVhdG1hc2tlci5vcmcgKDggTWF5IDIwMTcsIGRhdGUgbGFzdCBhY2Nlc3NlZCkuICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBhY0NZUDcwN0EyIG5lZ2F0aXZlbHkgcmVndWxhdGVzIGNoZXJyeSBmcnVpdCByaXBlbmluZyB3aGlsZSBQYWNDWVA3MDdBMSBtZWRpYXRlcyBkcm91Z2h0IHRvbGVyYW5jZSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEV4cC4gQm90OyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTY2OyAgY2l0YXRpb25fcGFnZXM9Mzc2NS03NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RnVsbC1sZW5ndGggdHJhbnNjcmlwdG9tZSBhc3NlbWJseSBmcm9tIFJOQS1zZXEgZGF0YSB3aXRob3V0IGEgcmVmZXJlbmNlIGdlbm9tZSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gQmlvdGVjaG5vbDsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0yOTsgIGNpdGF0aW9uX3BhZ2VzPTY0NC01MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9dFJOQXNjYW4tU0U6IGEgcHJvZ3JhbSBmb3IgaW1wcm92ZWQgZGV0ZWN0aW9uIG9mIHRyYW5zZmVyIFJOQSBnZW5lcyBpbiBnZW5vbWljIHNlcXVlbmNlLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbC4gQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9OTU1LTY0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NQUtFUjogYW4gZWFzeS10by11c2UgYW5ub3RhdGlvbiBwaXBlbGluZSBkZXNpZ25lZCBmb3IgZW1lcmdpbmcgbW9kZWwgb3JnYW5pc20gZ2Vub21lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xODsgIGNpdGF0aW9uX3BhZ2VzPTE4OC05NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXZvbHV0aW9uYXJ5IG9yaWdpbnMgYW5kIGR5bmFtaWNzIG9mIG9jdG9wbG9pZCBzdHJhd2JlcnJ5IHN1Ymdlbm9tZXMgcmV2ZWFsZWQgYnkgZGVuc2UgdGFyZ2V0ZWQgY2FwdHVyZSBsaW5rYWdlIG1hcHMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUuIEJpb2wuIEV2b2w7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTMyOTUtMzEzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CUkFLRVIxOiB1bnN1cGVydmlzZWQgUk5BLVNlcS1iYXNlZCBnZW5vbWUgYW5ub3RhdGlvbiB3aXRoIEdlbmVNYXJrLUVUIGFuZCBBVUdVU1RVUyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTMyOyAgY2l0YXRpb25fcGFnZXM9NzY3LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUludGVyUHJvU2NhbjogcHJvdGVpbiBkb21haW5zIGlkZW50aWZpZXIsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsLiBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MzM7ICBjaXRhdGlvbl9wYWdlcz1XMTE2LTIwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OZXcgZGV2ZWxvcG1lbnRzIGluIHRoZSBJbnRlclBybyBkYXRhYmFzZSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2wuIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0zNTsgIGNpdGF0aW9uX3BhZ2VzPUQyMjQtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2QtaGl0OiBhIGZhc3QgcHJvZ3JhbSBmb3IgY2x1c3RlcmluZyBhbmQgY29tcGFyaW5nIGxhcmdlIHNldHMgb2YgcHJvdGVpbiBvciBudWNsZW90aWRlIHNlcXVlbmNlcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTIyOyAgY2l0YXRpb25fcGFnZXM9MTY1OC05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5lIE9udG9sb2d5OiB0b29sIGZvciB0aGUgdW5pZmljYXRpb24gb2YgYmlvbG9neSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0yNS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgQ09HIGRhdGFiYXNlOiBhbiB1cGRhdGVkIHZlcnNpb24gaW5jbHVkZXMgZXVrYXJ5b3Rlcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBCaW9pbmZvcm07ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTQxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9S0VHRzoga3lvdG8gZW5jeWNsb3BlZGlhIG9mIGdlbmVzIGFuZCBnZW5vbWVzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbC4gQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTI3OyAgY2l0YXRpb25fcGFnZXM9MjktMzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuYWx5dGljYWwgd29ya2Zsb3cgb2YgZG91YmxlLWRpZ2VzdCByZXN0cmljdGlvbiBzaXRlLWFzc29jaWF0ZWQgRE5BIHNlcXVlbmNpbmcgYmFzZWQgb24gZW1waXJpY2FsIGFuZCBpbiBzaWxpY28gb3B0aW1pemF0aW9uIGluIHRvbWF0byw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUROQSBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9MjM7ICBjaXRhdGlvbl9wYWdlcz0xNDUtNTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZhc3QgZ2FwcGVkLXJlYWQgYWxpZ25tZW50IHdpdGggQm93dGllIDIsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIE1ldGhvZHM7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTM1Ny01OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHNlcXVlbmNlIGFsaWdubWVudC9tYXAgZm9ybWF0IGFuZCBTQU10b29scyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9MjA3OC05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SYXBpZCBhbmQgYWNjdXJhdGUgaGFwbG90eXBlIHBoYXNpbmcgYW5kIG1pc3NpbmctZGF0YSBpbmZlcmVuY2UgZm9yIHdob2xlLWdlbm9tZSBhc3NvY2lhdGlvbiBzdHVkaWVzIGJ5IHVzZSBvZiBsb2NhbGl6ZWQgaGFwbG90eXBlIGNsdXN0ZXJpbmcsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1BbS4gSi4gSHVtLiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT04MTsgIGNpdGF0aW9uX3BhZ2VzPTEwODQtOTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSB2YXJpYW50IGNhbGwgZm9ybWF0IGFuZCBWQ0Z0b29scyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTI3OyAgY2l0YXRpb25fcGFnZXM9MjE1Ni04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZXZlbG9wbWVudCBhbmQgdmFyaWFiaWxpdHkgYW5hbHlzaXMgb2YgbWljcm9zYXRlbGxpdGUgbWFya2VycyBpbiBwZWFjaCw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IEJyZWVkOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTEyMTsgIGNpdGF0aW9uX3BhZ2VzPTg3LTkyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1ETkEgZmluZ2VycHJpbnRpbmcgb2YgdGV0cmFwbG9pZCBjaGVycnkgZ2VybXBsYXNtIHVzaW5nIHNpbXBsZSBzZXF1ZW5jZSByZXBlYXRzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBBbWVyLiBTb2MuIEhvcnQuIFNjaTsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0yMTY7ICBjaXRhdGlvbl9wYWdlcz0yMDUtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QUMvR1QgYW5kIEFHL0NUIG1pY3Jvc2F0ZWxsaXRlIHJlcGVhdHMgaW4gcGVhY2ggW1BydW51cyBwZXJzaWNhIChMKSBCYXRzY2hdOiBpc29sYXRpb24sIGNoYXJhY3RlcmlzYXRpb24gYW5kIGNyb3NzLXNwZWNpZXMgYW1wbGlmaWNhdGlvbiBpbiBQcnVudXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UaGVvci4gQXBwbC4gR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9OTk7ICBjaXRhdGlvbl9wYWdlcz02NS03MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGV2ZWxvcG1lbnQgYW5kIGNoYXJhY3Rlcml6YXRpb24gb2YgcG9seW1vcnBoaWMgbWljcm9zYXRlbGxpdGVzIGZyb20gUHJ1bnVzIGF2aXVtIOKAsk5hcG9sZW9u4oCyLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBFY29sLiBOb3RlczsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0zOyAgY2l0YXRpb25fcGFnZXM9NTc4LTgwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZXZlbG9wbWVudCBvZiBtaWNyb3NhdGVsbGl0ZSBtYXJrZXJzIGluIHBlYWNoIFtQcnVudXMgcGVyc2ljYSAoTC4pIEJhdHNjaF0gYW5kIHRoZWlyIHVzZSBpbiBnZW5ldGljIGRpdmVyc2l0eSBhbmFseXNpcyBpbiBwZWFjaCBhbmQgc3dlZXQgY2hlcnJ5IChQcnVudXMgYXZpdW0gTC4pOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UaGVvci4gQXBwbC4gR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MTA1OyAgY2l0YXRpb25fcGFnZXM9MTI3LTM4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZXZlbG9wbWVudCBvZiBhIHNlY29uZCBnZW5lcmF0aW9uIGxpbmthZ2UgbWFwIGZvciBhbG1vbmQgdXNpbmcgUkFQRCBhbmQgU1NSIG1hcmtlcnMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWU7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9NDM7ICBjaXRhdGlvbl9wYWdlcz02NDktNTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdEUiAoR2Vub21lIERhdGFiYXNlIGZvciBSb3NhY2VhZSk6IGludGVncmF0ZWQgd2ViLWRhdGFiYXNlIGZvciBSb3NhY2VhZSBnZW5vbWljcyBhbmQgZ2VuZXRpY3MgZGF0YSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2wuIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0zNjsgIGNpdGF0aW9uX3BhZ2VzPUQxMDM0LTQwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JZGVudGlmaWNhdGlvbiBvZiBtaWNyb3NhdGVsbGl0ZSBsb2NpIGluIGFwcmljb3QsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEVjb2wuIE5vdGVzOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTI7ICBjaXRhdGlvbl9wYWdlcz0yNC02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZXZlbG9wbWVudCBhbmQgdHJhbnNwb3J0YWJpbGl0eSBhY3Jvc3MgUHJ1bnVzIHNwZWNpZXMgb2YgNDIgcG9seW1vcnBoaWMgYWxtb25kIG1pY3Jvc2F0ZWxsaXRlcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gRWNvbC4gUmVzb3VyOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz01MzEtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJ1bnVzIG1pY3Jvc2F0ZWxsaXRlIG1hcmtlciB0cmFuc2ZlcmFiaWxpdHkgYWNyb3NzIHJvc2FjZW91cyBjcm9wcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRyZWUgR2VuZXQuIEdlbm9tZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTY4OS03MDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNoYXJhY3Rlcml6YXRpb24gb2YgbWljcm9zYXRlbGxpdGUgbWFya2VycyBpbiBwZWFjaCBbUHJ1bnVzIHBlcnNpY2EgKEwuKSBCYXRzY2hdOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UaGVvci4gQXBwbC4gR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MTAxOyAgY2l0YXRpb25fcGFnZXM9NDIxLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuYWx5c2lzIG9mIHN3ZWV0IGNoZXJyeSAoUHJ1bnVzIGF2aXVtIEwuKSBjdWx0aXZhcnMgdXNpbmcgU1NSIGFuZCBBRkxQIG1hcmtlcnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEFtLiBTb2MuIEhvci4gU2NpOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTEyODsgIGNpdGF0aW9uX3BhZ2VzPTkwNC05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NaWNyb3NhdGVsbGl0ZSBETkEgaW4gcGVhY2ggKFBydW51cyBwZXJzaWNhIEwuIEJhdHNjaCkgYW5kIGl0cyB1c2UgaW4gZmluZ2VycHJpbnRpbmcgYW5kIHRlc3RpbmcgdGhlIGdlbmV0aWMgb3JpZ2luIG9mIGN1bHRpdmFycyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZTsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT00MzsgIGNpdGF0aW9uX3BhZ2VzPTUxMi0yMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2hhcmFjdGVyaXphdGlvbiBvZiBub3ZlbCBtaWNyb3NhdGVsbGl0ZXMgYW5kIGRldmVsb3BtZW50IG9mIG11bHRpcGxleCBQQ1IgZm9yIGxhcmdlLXNjYWxlIHBvcHVsYXRpb24gc3R1ZGllcyBpbiB3aWxkIGNoZXJyeSwgUHJ1bnVzIGF2aXVtLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBFY29sLiBSZXNvdXI7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTQyOS0zMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWljcm9zYXRlbGxpdGUgbWFya2VycyBpbiBwZWFjaCBbUHJ1bnVzIHBlcnNpY2EgKEwuKSBCYXRzY2hdIGRlcml2ZWQgZnJvbSBhbiBlbnJpY2hlZCBnZW5vbWljIGFuZCBjRE5BIGxpYnJhcmllcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gRWNvbC4gUmVzb3VyOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTI7ICBjaXRhdGlvbl9wYWdlcz0yOTgtMzAxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBpbnRlZ3JhdGVkIGdlbmV0aWMgbGlua2FnZSBtYXAgb2YgcGVhY2ggYnkgU1NSLCBTVFMsIEFGTFAgYW5kIFJBUEQsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBKYXBhbi4gU29jLiBIb3J0LiBTY2k7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9NzQ7ICBjaXRhdGlvbl9wYWdlcz0yMDQtMTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3B1Ymxpc2hlcj1LeWF6bWEgQlYsIFdhZ2VuaW5nZW4sIFRoZSBOZXRoZXJsYW5kczsgIGNpdGF0aW9uX3RpdGxlPUpvaW5NYXDCrjQsIHNvZnR3YXJlIGZvciB0aGUgY2FsY3VsYXRpb24gb2YgZ2VuZXRpYyBsaW5rYWdlIG1hcHMgaW4gZXhwZXJpbWVudGFsIHBvcHVsYXRpb25zOyAgY2l0YXRpb25feWVhcj0yMDA2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1MUG1lcmdlOiBhbiBSIHBhY2thZ2UgZm9yIG1lcmdpbmcgZ2VuZXRpYyBtYXBzIGJ5IGxpbmVhciBwcm9ncmFtbWluZyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTMwOyAgY2l0YXRpb25fcGFnZXM9MTYyMy00OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NYXBDaGFydDogc29mdHdhcmUgZm9yIHRoZSBncmFwaGljYWwgcHJlc2VudGF0aW9uIG9mIGxpbmthZ2UgbWFwcyBhbmQgUVRMcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEhlcmVkOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTkzOyAgY2l0YXRpb25fcGFnZXM9NzctODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2lyY29zOiBhbiBpbmZvcm1hdGlvbiBhZXN0aGV0aWMgZm9yIGNvbXBhcmF0aXZlIGdlbm9taWNzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xOTsgIGNpdGF0aW9uX3BhZ2VzPTE2MzktNDU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZhc3QgYWxnb3JpdGhtcyBmb3IgbGFyZ2Utc2NhbGUgZ2Vub21lIGFsaWdubWVudCBhbmQgY29tcGFyaXNvbiw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2wuIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTI0NzgtODM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgcHJvZ3JhbSBmb3IgYW5ub3RhdGluZyBhbmQgcHJlZGljdGluZyB0aGUgZWZmZWN0cyBvZiBzaW5nbGUgbnVjbGVvdGlkZSBwb2x5bW9ycGhpc21zLCBTbnBFZmY6IFNOUHMgaW4gdGhlIGdlbm9tZSBvZiBEcm9zb3BoaWxhIG1lbGFub2dhc3RlciBzdHJhaW4gdzExMTg7IGlzby0yOyBpc28tMzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Rmx5OyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz04MC05MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIFNlcXVlbmNlIE9udG9sb2d5OiBhIHRvb2wgZm9yIHRoZSB1bmlmaWNhdGlvbiBvZiBnZW5vbWUgYW5ub3RhdGlvbnMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9UjQ0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q05WLXNlcSwgYSBuZXcgbWV0aG9kIHRvIGRldGVjdCBjb3B5IG51bWJlciB2YXJpYXRpb24gdXNpbmcgaGlnaC10aHJvdWdocHV0IHNlcXVlbmNpbmcsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgQmlvaW5mb3JtOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9ODAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TTlAyQ0FQUzogYSBTTlAgYW5kIElOREVMIGFuYWx5c2lzIHRvb2wgZm9yIENBUFMgbWFya2VyIGRldmVsb3BtZW50LDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbC4gQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTMyOyAgY2l0YXRpb25fcGFnZXM9ZTUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TY2lSb0tvOiBhIG5ldyB0b29sIGZvciB3aG9sZSBnZW5vbWUgbWljcm9zYXRlbGxpdGUgc2VhcmNoIGFuZCBpbnZlc3RpZ2F0aW9uLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MjM7ICBjaXRhdGlvbl9wYWdlcz0xNjgzLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByaW1lcjMgb24gdGhlIFdXVyBmb3IgZ2VuZXJhbCB1c2VycyBhbmQgZm9yIGJpb2xvZ2lzdCBwcm9ncmFtbWVycyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1ldGhvZHMgTW9sLiBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTEzMjsgIGNpdGF0aW9uX3BhZ2VzPTM2NS04NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TnVjbGVhciBETkEgY29udGVudCBvZiBzb21lIGltcG9ydGFudCBwbGFudCw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IE1vbC4gQmlvbC4gUmVwOyAgY2l0YXRpb25feWVhcj0xOTkxOyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz0yMDgtMTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVF1YW50aXRhdGl2ZSBtZWFzdXJlcyBmb3IgdGhlIG1hbmFnZW1lbnQgYW5kIGNvbXBhcmlzb24gb2YgYW5ub3RhdGVkIGdlbm9tZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBCaW9pbmZvcm07ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz02NyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIGFzc29jaWF0aW9uIHN0dWR5IG9mIDEyIGFncm9ub21pYyB0cmFpdHMgaW4gcGVhY2gsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIENvbW11bjsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9MTMyNDYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QbGFudCBHZW5vbWUgRGF0YUJhc2UgSmFwYW4gKFBHREJqKTogYSBwb3J0YWwgd2Vic2l0ZSBmb3IgdGhlIGludGVncmF0aW9uIG9mIHBsYW50IGdlbm9tZS1yZWxhdGVkIGRhdGFiYXNlcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IENlbGwgUGh5c2lvbDsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT01NTsgIGNpdGF0aW9uX3BhZ2VzPWU4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21pY3MtYXNzaXN0ZWQgYnJlZWRpbmcgaW4gZnJ1aXQgdHJlZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJyZWVkLiBTY2k7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9NjY7ICBjaXRhdGlvbl9wYWdlcz0xMDAtMTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURlcGxveW1lbnQgb2YgbmV3IGJpb3RlY2hub2xvZ2llcyBpbiBwbGFudCBicmVlZGluZyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gQmlvdGVjaG5vbDsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTIzMS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWljIHByZWRpY3Rpb24gb2YgdHJhaXQgc2VncmVnYXRpb24gaW4gYSBwcm9nZW55IHBvcHVsYXRpb246IGEgY2FzZSBzdHVkeSBvZiBKYXBhbmVzZSBwZWFyIChQeXJ1cyBweXJpZm9saWEpOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz04MSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlZHVjZWQgZ2VuZXJhdGlvbiB0aW1lIG9mIGFwcGxlIHNlZWRsaW5ncyB0byB3aXRoaW4gYSB5ZWFyIGJ5IG1lYW5zIG9mIGEgcGxhbnQgdmlydXMgdmVjdG9yOiBhIG5ldyBwbGFudC1icmVlZGluZyB0ZWNobmlxdWUgd2l0aCBubyB0cmFuc21pc3Npb24gb2YgZ2VuZXRpYyBtb2RpZmljYXRpb24gdG8gdGhlIG5leHQgZ2VuZXJhdGlvbiw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IEJpb3RlY2hub2wuIEo7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz02MC04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMjQvNS80OTkvMjI4OTMwNzkvZHN4MDIwLnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBXZSBkZXRlcm1pbmVkIHRoZSBnZW5vbWUgc2VxdWVuY2Ugb2Ygc3dlZXQgY2hlcnJ5IChQcnVudXMgYXZpdW0pIHVzaW5nIG5leHQtZ2VuZXJhdGlvbiBzZXF1ZW5jaW5nIHRlY2hub2xvZ3kuIFRoZSB0b3RhbCBsZW5ndGggb2YgdGhlIGFzc2VtYmxlZCBzZXF1ZSIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXMvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9MzgzNTg1NCZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGRXRGRjb9n5NhRT1N8yaL6U4ma7Yg==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f5%2f499%2f3835854" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f5%2f499%2f3835854" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=99990" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/24/5"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/5/1/m_dnares_24_5cover.png?Expires=2147483647&amp;Signature=jul3oxjgkP675GHMbQEy~zZ1my2m2g0xI0mr9LkJEjA-PhoC4BnM1ARo9FdXFQKdaeXjXdDG8Th7l01-PuT824GoN40EPQF0GwZOV5tGDYZCguawjvcFtz41hVQe~O6T29XL-3ITGszkj0QvmRZLajQvyRnp825zpQTjNCOtAlB0rKMc7Dcdixzjc2uv2jilEDMqD9YZCuMkOXRLrwDQftwYgb~q8EavwLngcuRRSkILOW9lFSQBCUNAqNd4mDnFW5RKty7z4w3HYGKz5wJomnNLGyCwwAyELyFWHWV4xtKHzE5xEUd8do1ZdHc6pZe4TftEycCVzz1M4dnVR5tpug__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/24/5">
                        <div class="volume trailing-comma">Volume 24</div>
                        <div class="issue">Issue 5</div>

            </a>

        </div>
        <div class="ii-pub-date">
October 2017        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="106356460">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106356460">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="106356462">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106356462">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="106356466">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106356466">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="106356492">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106356492">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="106356519">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106356519">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="106356524">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106356524">5. Availability</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="106356526">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106356526">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="106356534">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106356534">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/24/5/487/3778238">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/24/5/509/3859850">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">The genome sequence of sweet cherry (<em>Prunus avium</em>) for use in genomics-assisted breeding
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kenta Shirasawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsx020-cor1" data-open="dsx020-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. 81â€‰438â€‰52â€‰3935. Fax. 81â€‰438â€‰52â€‰3934. Email: <a href="mailto:shirasaw@kazusa.or.jp" target="_blank">shirasaw@kazusa.or.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kenta+Shirasawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shirasawa K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Shirasawa K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Kenta Shirasawa</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kanji Isuzugawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div>Horticultural Experiment Station, Yamagata Integrated Agricultural Research Center, Sagae, Yamagata 991-0043, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kanji+Isuzugawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Isuzugawa K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Isuzugawa K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Kanji Isuzugawa</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Mitsunobu Ikenaga</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div>Central Agricultural Experiment Station, Agricultural Research Department, Hokkaido Research Organization, Naganuma, Hokkaido 069-1395, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Mitsunobu+Ikenaga">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ikenaga M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ikenaga M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Mitsunobu Ikenaga</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yutaro Saito</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div>Horticultural Experiment Station, Yamagata Integrated Agricultural Research Center, Sagae, Yamagata 991-0043, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yutaro+Saito">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Saito Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Saito Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yutaro Saito</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Toshiya Yamamoto</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div>Institute of Fruit Tree and Tea Science, National Agriculture and Food Research Organization, Tsukuba, Ibaraki 305-8605, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Toshiya+Yamamoto">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yamamoto T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yamamoto T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Toshiya Yamamoto</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hideki Hirakawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hideki+Hirakawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hirakawa H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hirakawa H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hideki Hirakawa</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Sachiko Isobe</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Sachiko+Isobe">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Isobe S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Isobe S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Sachiko Isobe</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsx020-FM1"><div class="fn"><div class="label -label">*</div><div class="footnote-content">Edited by Prof. Kazuhiro Sato</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 24, Issue 5, 1 October 2017, Pages 499â€“508, <a href='https://doi.org/10.1093/dnares/dsx020'>https://doi.org/10.1093/dnares/dsx020</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">25 May 2017</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">08 March 2017</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">25 April 2017</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="3835854" href="/dnaresearch/article-pdf/24/5/499/22893079/dsx020.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Kenta Shirasawa, Kanji Isuzugawa, Mitsunobu Ikenaga, Yutaro Saito, Toshiya Yamamoto, Hideki Hirakawa, Sachiko Isobe;  The genome sequence of sweet cherry (<em>Prunus avium</em>) for use in genomics-assisted breeding, <em>DNA Research</em>, Volume 24, Issue 5, 1 October 2017, Pages 499â€“508, <a href='https://doi.org/10.1093/dnares/dsx020'>https://doi.org/10.1093/dnares/dsx020</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/3835854?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/3835854?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/3835854?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/3835854?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/3835854?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/3835854?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/3835854?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=The%20genome%20sequence%20of%20sweet%20cherry%20(%26lt%3Bem%26gt%3BPrunus%20avium%26lt%3B%2Fem%26gt%3B)%20for%20use%20in%20genomics-assisted%20breeding&amp;publicationDate=2017-05-25&amp;volumeNum=24&amp;issueNum=5&amp;author=Shirasawa%2C%20Kenta%3B%20Isuzugawa%2C%20Kanji&amp;startPage=499&amp;endPage=508&amp;contentId=10.1093%2Fdnares%2Fdsx020&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f5%2f499%2f3835854" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=106356460 id="106356460" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>We determined the genome sequence of sweet cherry (<em>Prunus avium</em>) using next-generation sequencing technology. The total length of the assembled sequences was 272.4â€‰Mb, consisting of 10,148 scaffold sequences with an N50 length of 219.6â€‰kb. The sequences covered 77.8% of the 352.9â€‰Mb sweet cherry genome, as estimated by <em>k</em>-mer analysis, and includedâ€‰&gt;96.0% of the core eukaryotic genes. We predicted 43,349 complete and partial protein-encoding genes. A high-density consensus map with 2,382 loci was constructed using double-digest restriction siteâ€“associated DNA sequencing. Comparing the genetic maps of sweet cherry and peach revealed high synteny between the two genomes; thus the scaffolds were integrated into pseudomolecules using map- and synteny-based strategies. Whole-genome resequencing of six modern cultivars found 1,016,866 SNPs and 162,402 insertions/deletions, out of which 0.7% were deleterious. The sequence variants, as well as simple sequence repeats, can be used as DNA markers. The genomic information helps us to identify agronomically important genes and will accelerate genetic studies and breeding programs for sweet cherries. Further information on the genomic sequences and DNA markers is available in DBcherry (<a class="link link-uri" href="http://cherry.kazusa.or.jp" target="_blank">http://cherry.kazusa.or.jp</a> (8 May 2017, date last accessed)).</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;draft genome&quot;">draft genome</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genetic map&quot;">genetic map</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genomics-assisted breeding&quot;">genomics-assisted breeding</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;sweet cherry (Prunus avium)&quot;">sweet cherry (<em>Prunus avium</em>)</a></div>                            <h2 scrollto-destination=106356462 id="106356462" class="section-title" >1. Introduction</h2>
<p>Sweet cherry (<em>Prunus avium</em>, 2nâ€‰=â€‰2xâ€‰=â€‰16) and its tetraploid relatives (<em>Pr. cerasus</em> and <em>Pr. pseudocerasus</em>, 2nâ€‰=â€‰4xâ€‰=â€‰32) are fruit crops of the Rosaceae family, which also includes apple (<em>Malusâ€‰</em>Ã—<em>â€‰domestica</em>), peach (<em>Pr. persica</em>), Japanese apricot (<em>Pr. mume</em>), strawberry (<em>Fragaria vesca</em> and <em>F.â€‰</em>Ã—â€‰<em>ananassa</em>), and Japanese, Chinese, and European pears (<em>Pyrus pyrifolia</em>, <em>Py. bretschneideri</em>, and <em>Py. communis</em>). Because of their economic importance, e.g. the world production of 2.2 M tonnes in 2014 (FAOSTAT: <a class="link link-uri" href="http://www.fao.org/faostat" target="_blank">http://www.fao.org/faostat</a> (8 May 2017, date last accessed)), breeding programs for fruit crops are progressing all over the world. However, in general, their breeding efficiency has lagged behind that of the cereal crops and vegetables due to the time and space required to grow them. Because genomics-based breeding could overcome this drawback, whole-genome sequencing has been performed on a number of the Rosaceae crops, including apple,<a reveal-id="dsx020-B1" data-open="dsx020-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> peach,<a reveal-id="dsx020-B2" data-open="dsx020-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> pear,<a reveal-id="dsx020-B3" data-open="dsx020-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a><sup>,</sup><a reveal-id="dsx020-B4" data-open="dsx020-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> Japanese apricot,<a reveal-id="dsx020-B5" data-open="dsx020-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> and strawberry,<a reveal-id="dsx020-B6" data-open="dsx020-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a><sup>,</sup><a reveal-id="dsx020-B7" data-open="dsx020-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> as well as onâ€‰&gt;100 other plant species.<a reveal-id="dsx020-B8" data-open="dsx020-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a></p><p>Nevertheless, whole-genome sequencing of sweet cherry has not been reported despite its simple, compact genome (2nâ€‰=â€‰2xâ€‰=â€‰16, genome size of âˆ¼380â€‰Mb). According to sweet cherry genetic maps, the structure of the sweet cherry genome is predicted to be similar to that of the peach genome,<a reveal-id="dsx020-B9" data-open="dsx020-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> meaning that the order of markers is conserved between the two species. Moreover, the positions of QTLs for agronomically important traits (e.g. disease resistances, as well as flower, vegetative, and fruit or nut quality) overlap.<a reveal-id="dsx020-B10" data-open="dsx020-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> Therefore, genomic information from peaches, as well as other Rosaceae fruiting crops, has already been utilized in sweet cherry breeding.<a reveal-id="dsx020-B11" data-open="dsx020-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a></p><p>Even though the marker orders are conserved between sweet cherries and peaches, the genome sequences have diverged.<a reveal-id="dsx020-B12" data-open="dsx020-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> A conserved set of orthologous markers bridge the barrier between the genome sequences, but the number of available markers is limited,<a reveal-id="dsx020-B13" data-open="dsx020-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> forcing researchers and breeders to develop a high-throughput SNP genotyping system.<a reveal-id="dsx020-B14" data-open="dsx020-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> To enhance the breeding programs for sweet cherries and to assist future genetics and genomics studies, we established genomic resources such as whole-genome sequence data, a high-density genetic map, the sweet cherry pseudomolecule based on the genetic map and synteny with the peach genome, and DNA markers using SNPs, simple sequence repeats (SSRs), and insertions/deletions identified from whole-genome resequencing of six modern cultivars. In addition, we identified agronomically important genes for fruit color, morphology, and quality and self-incompatibility in the genome. This study will be useful for breeding programs and in genetics and genomics studies on not only sweet cherry but also other members of the Rosaceae family including the sweet cherry relatives, <em>Pr. cerasus</em> and <em>Pr. pseudocerasus</em>, both of which possess complex genomes due to tetraploidy. </p>                            <h2 scrollto-destination=106356466 id="106356466" class="section-title" >2. Materials and methods</h2>
                            <h3 scrollto-destination=106356467 id="106356467" class="section-title" >2.1. Sequencing analysis of the sweet cherry genome</h3>
<p>A Japanese leading variety of sweet cherry (<em>Pr. avium</em>) (i.e. Satonishiki) was used for genomic sequencing. For genomic diversity analysis, six Japanese varieties (Benikirari, Benisayaka, Benishuho, Benitemari, Beniyutaka, and Nanyo) were used. The pedigree of the materials is shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx020/2/dsx020_supp_sweetcherrygenome-supplementaryfigures.r1.pdf?Expires=2147483647&Signature=EpzskbJ8FdkxtePPGHzVUHR8wbz7PuhsMtIZp1mBYfuYpDnoU860lHAiOXaTih335VqggdpxNmN~DlHv9jCQjL67NgdKEe0mJf6VW~rFFNlKSOSwGF16QSU~ixooLa4MnFfPBuizWoRuWKHC5qHhWq9m7g0mIOI~a3cC4r-I38M7qqsg8TxxRMqONuq~9wlcV5s9xtt-0Ia6Fmkgv9vqIrt-f~ik0NUXpBMR6Br6pBv9tJQJvkf47kQLiRbDw4zZbNs27lnG2PNg44Zj75jgFNV8z2KYtBTZOOPF3qb1rjsE96U2cnvlArNo-eTe~6GEVJGGYM0KqyUXlBDoAbMxJA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figure S1</a></span>. Young leaves from each variety were collected from the original trees (Benikirari, Benitemari, and Beniyutaka) and clones (Satonishiki, Benisayaka, Benishuho, and Nanyo), all of which were planted in the Horticultural Experiment Station at Yamagata Integrated Agricultural Research Center, Japan.</p><p>Genomic DNA was extracted from the leaves using a DNeasy Plant Mini Kit (Qiagen, Hilden, Germany) and used for construction of a paired-end (PE) library (insert size: 500â€‰bp), in accordance with the TruSeq DNA Sample Preparation Guide (Illumina, San Diego, CA, USA). In addition, four mate-pair (MP) libraries (insert sizes of 2, 5, 10, and 15â€‰kb) were constructed with GS Titanium Library Paired End Adaptors (Roche, Basel, Switzerland).<a reveal-id="dsx020-B15" data-open="dsx020-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> The nucleotide sequences were determined using massively parallel sequencing by synthesis on a HiSeq2000 (Illumina) in PE 93â€‰bp mode.</p>                            <h3 scrollto-destination=106356470 id="106356470" class="section-title" >2.2. Genome size estimation and genome assembly</h3>
<p>Out of the obtained sequence reads for PE and MP sequencing, low-quality reads were removed and adapter sequences were trimmed using PRINSEQ<a reveal-id="dsx020-B16" data-open="dsx020-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> (version 0.20.4: parameters of -trim_right 1, -trim_qual_right 10, and -min_len 92) and fastx_clipper (parameter of -a AGATCGGAAGAGC) in the FASTX-Toolkit (version 0.0.14: <a class="link link-uri" href="http://hannonlab.cshl.edu/fastx_toolkit" target="_blank">http://hannonlab.cshl.edu/fastx_toolkit</a> (8 May 2017, date last accessed)), respectively. The filtered high-quality reads were used for genomic size estimation based on <em>k</em>-mer frequency (<em>k</em>â€‰=â€‰17) using Jellyfish<a reveal-id="dsx020-B17" data-open="dsx020-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> (version 2.1.4).</p><p>The high-quality PE reads were assembled into contigs using SOAPdenovo2<a reveal-id="dsx020-B18" data-open="dsx020-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> (version r240: parameters of -F and -R) or Platanus<a reveal-id="dsx020-B19" data-open="dsx020-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> (version 1.2.1). In the assembly with SOAPdenovo2, <em>k</em>-mer sizes from 51 to 91 were examined. After comparing the two assemblies, sequence data obtained from SOAPdenovo2 (<em>k</em>-merâ€‰=â€‰81) were chosen for scaffolding with high-quality MP reads, which was carried out using SOAPdenovo2. Gaps, represented by Ns in the sequence, were filled with the high-quality PE reads using GapCloser<a reveal-id="dsx020-B18" data-open="dsx020-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> (version 1.10: parameter of -p 31). Contaminating sequences were removed by searching with BLASTN, with an E-value cutoff of 1Eâ€‰âˆ’â€‰10 and length coverage ofâ€‰â‰¥10%, against sequences from potential contaminating sources such as organelles (chloroplasts of peach [accession number: HQ336405], Japanese apricot [accession number: KF765450], and strawberry [accession number: NC_015206], and mitochondria of <em>Arabidopsis</em> [accession number: NC_001284] and apple [accession number: NC_018554]), other organisms (bacterial and fungi genome sequences registered in NCBI [<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov" target="_blank">http://www.ncbi.nlm.nih.gov</a>], and the human genome [hg19]), and artifacts (Illumina PhiX Sequencing Control v3 and vector sequences from UniVec [<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/tools/vecscreen/univec/" target="_blank">http://www.ncbi.nlm.nih.gov/tools/vecscreen/univec/</a> (8 May 2017, date last accessed)]). The resulting sequences that wereâ€‰â‰¥1,000 bases were selected and designated PAV_r1.0. Completeness of the assembly was assessed with sets of Benchmarking Universal Single-Copy Orthologues (BUSCO)<a reveal-id="dsx020-B20" data-open="dsx020-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> (version 1.1b).</p>                            <h3 scrollto-destination=106356473 id="106356473" class="section-title" >2.3. Repetitive sequence analysis</h3>
<p>Repetitive sequences in PAV_r1.0 were identified using Repbase.<a reveal-id="dsx020-B21" data-open="dsx020-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> A <em>de novo</em> repeat library for PAV_r1.0 was built using RepeatScout<a reveal-id="dsx020-B22" data-open="dsx020-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> (version 1.0.5), and the repetitive sequences were searched for using RepeatMasker<a reveal-id="dsx020-B23" data-open="dsx020-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> (version 4.0.3) based on known repetitive sequences registered in Repbase<a reveal-id="dsx020-B21" data-open="dsx020-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> and the <em>de novo</em> repeat libraries.</p>                            <h3 scrollto-destination=106356475 id="106356475" class="section-title" >2.4. RNA sequencing and assembly</h3>
<p>Total RNA was extracted from leaves, roots, flowers, calli, brown rot fruits infected by <em>Monilinia fructicola</em>, and fruits from three different stages (32 days after full bloom [DAFB], yellow; 44 DAFB, initial red; and 54 DAFB, full red)<a reveal-id="dsx020-B24" data-open="dsx020-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> using the RNeasy Mini Kit (Qiagen) or phenol/SDS extraction, and then treated with RQ1 RNase-Free DNase (Promega, Madison, WI, USA) to remove contaminating genomic DNA. RNA libraries were constructed in accordance with the TruSeq Stranded mRNA Sample Preparation Guide (Illumina). The nucleotide sequences were determined using massively parallel sequencing by synthesis on a MiSeq (Illumina) in the PE 301â€‰bp mode. The obtained reads were treated as above to remove low-quality reads and to trim adapter sequences, and were assembled using Trinity<a reveal-id="dsx020-B25" data-open="dsx020-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> (version r20140717: parameters of â€“min_contig_length 100, â€“group_pairs_distance 400, and â€“SS_lib_type RF) to generate a UniGene set.</p>                            <h3 scrollto-destination=106356477 id="106356477" class="section-title" >2.5. Gene prediction and annotation</h3>
<p>Transfer RNA (tRNA) genes were predicted using tRNAscan-SE<a reveal-id="dsx020-B26" data-open="dsx020-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> (version 1.23) with the default parameters, whereas ribosomal RNA (rRNA) genes were predicted using BLASTN searches with an E-value cutoff of 1Eâ€‰âˆ’â€‰10, with the <em>Arabidopsis thaliana</em> 18S rRNA (accession number: X16077) and 5.8S and 25S rRNAs (accession number: X52320) used as query sequences.</p><p>To identify putative protein-encoding genes in PAV_r1.0, a MAKER pipeline<a reveal-id="dsx020-B27" data-open="dsx020-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> (version 2.31.8) including <em>ab-initio</em>-, evidence-, and homology-based gene prediction methods was used. For this prediction, the UniGene set generated from the RNA-Seq analysis and peptide sequences predicted from the genomes of Rosaceae members (e.g. <em>F. vesca</em> [Genome Database for Rosaceae, GDR, version v2.0.a1],<a reveal-id="dsx020-B28" data-open="dsx020-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a><sup>,</sup><em>Pr. persica</em> [GDR v2.0.a1]<sup>2</sup>, <em>M.</em>â€‰Ã—â€‰<em>domestica</em> [GDR v1.0p],<a reveal-id="dsx020-B1" data-open="dsx020-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> and <em>Pr. mume</em><a reveal-id="dsx020-B5" data-open="dsx020-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a>) were used as a training data set. In addition, BRAKER1<a reveal-id="dsx020-B29" data-open="dsx020-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> (version 1.3) was also used to complete the gene set for PAV_r1.0. Genes related to transposable elements (TEs) were detected using BLASTP searches against the NCBI non-redundant (nr) protein database with an E-value cutoff of 1Eâ€‰âˆ’â€‰10 and by using InterProScan<a reveal-id="dsx020-B30" data-open="dsx020-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> (version 4.8) searches against the InterPro database<a reveal-id="dsx020-B31" data-open="dsx020-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> with an E-value cutoff of 1.0. The putative genes of PAV_r1.0 were clustered using CD-hit<a reveal-id="dsx020-B32" data-open="dsx020-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> (version 4.6.1) with the UniGene set of <em>F. vesca</em> (GDR v2.0.a1),<a reveal-id="dsx020-B28" data-open="dsx020-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a><sup>,</sup><em>Pr. persica</em> (GDR v2.0.a1)<sup>2</sup>, <em>M.</em>â€‰Ã—â€‰<em>domestica</em> (GDR v1.0p),<a reveal-id="dsx020-B1" data-open="dsx020-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> and <em>Pr. mume</em><a reveal-id="dsx020-B5" data-open="dsx020-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> with the parameters câ€‰=â€‰0.6 and aLâ€‰=â€‰0.4. The genes in the plant species described above were classified into plant gene ontology (GO) slim categories<a reveal-id="dsx020-B33" data-open="dsx020-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> and euKaryotic clusters of Orthologous Groups (KOG) categories,<a reveal-id="dsx020-B34" data-open="dsx020-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> and mapped onto the Kyoto Encyclopedia of Genes and Genomes (KEGG) reference pathways.<a reveal-id="dsx020-B35" data-open="dsx020-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a></p>                            <h3 scrollto-destination=106356480 id="106356480" class="section-title" >2.6. Construction of genetic linkage maps and comparative genomics</h3>
<p>Three F1-mapping populations, shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx020/2/dsx020_supp_sweetcherrygenome-supplementaryfigures.r1.pdf?Expires=2147483647&Signature=EpzskbJ8FdkxtePPGHzVUHR8wbz7PuhsMtIZp1mBYfuYpDnoU860lHAiOXaTih335VqggdpxNmN~DlHv9jCQjL67NgdKEe0mJf6VW~rFFNlKSOSwGF16QSU~ixooLa4MnFfPBuizWoRuWKHC5qHhWq9m7g0mIOI~a3cC4r-I38M7qqsg8TxxRMqONuq~9wlcV5s9xtt-0Ia6Fmkgv9vqIrt-f~ik0NUXpBMR6Br6pBv9tJQJvkf47kQLiRbDw4zZbNs27lnG2PNg44Zj75jgFNV8z2KYtBTZOOPF3qb1rjsE96U2cnvlArNo-eTe~6GEVJGGYM0KqyUXlBDoAbMxJA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figure S1</a></span>, were used to construct genetic linkage maps: (1) C-303 (<em>n</em>â€‰=â€‰94), derived from a cross between Beniyutaka and Benikirari; (2) C-309 (<em>n</em>â€‰=â€‰84), derived from a cross between C-195-50, which is a hybrid of Benishuho and an F1 C-47-70 of Benisayakaâ€‰Ã—â€‰Rainer, and Benikirari; and (3) HRO (<em>n</em>â€‰=â€‰384), derived from a cross between Nanyo and Benisayaka. Genomic DNA extracted from the leaves of each line was subjected to double-digest restriction siteâ€“associated DNA sequencing (ddRAD-Seq) library construction.<a reveal-id="dsx020-B36" data-open="dsx020-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> The DNA was digested using two restriction enzymes, <em>PstI</em> and <em>EcoRI</em>, and DNA fragments of 300â€“900â€‰bp in length were fractionated using BluePippin (Sage Science, Beverly, MA, USA). The libraries were sequenced on a HiSeq (Illumina) in PE 93â€‰bp mode.</p><p>Primary data processing of the sequencing reads was performed as described by Shirasawa et al.<a reveal-id="dsx020-B36" data-open="dsx020-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> with minor modifications. Low-quality sequences were removed and adapters were trimmed using PRINSEQ<a reveal-id="dsx020-B16" data-open="dsx020-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> (version 0.20.4) and fastx_clipper in the FASTX-Toolkit (version 0.0.13: <a class="link link-uri" href="http://hannonlab.cshl.edu/fastx_toolkit" target="_blank">http://hannonlab.cshl.edu/fastx_toolkit</a> (8 May 2017, date last accessed)), respectively. The filtered reads were mapped onto the PAV_r1.0 reference sequence using Bowtie 2<a reveal-id="dsx020-B37" data-open="dsx020-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> (version 2.2.3). To obtain a variant call format (VCF) file including SNP information, the sequence alignment/map format (SAM) files were converted to binary sequence alignment/map format (BAM) files and subjected to SNP calling using the mpileup command of SAMtools<a reveal-id="dsx020-B38" data-open="dsx020-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> (version 0.1.19) and the view command of BCFtools.<a reveal-id="dsx020-B38" data-open="dsx020-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> Missing data were imputed using Beagle4<a reveal-id="dsx020-B39" data-open="dsx020-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> (version r1185). High-confidence SNPs were selected using VCFtools<a reveal-id="dsx020-B40" data-open="dsx020-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> (version 0.1.12b) with the following criteria: 1)â€‰â‰¥5Ã—â€‰coverage in each plant line (â€“minDP 5), 2)â€‰&gt;10 SNP quality value (â€“minQ 10), 3)â€‰â‰¥0.2 minor allele frequency (â€“maf 0.2), and 4)â€‰&lt;0.5 missing data rate (â€“max-missing 0.5).</p><p>In addition, SSR markers reported in the previous studies<a reveal-id="dsx020-B41" data-open="dsx020-B41" class="link link-ref link-reveal xref-bibr"><sup>41â€“56</sup></a> (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx020/2/dsx020_supp_sweetcherrygenome-supplementarytables.xlsx?Expires=2147483647&Signature=TiRI9J~ZD4Ek9u56ktrPJXS93YJ4qGZITEQupuOKHddrgtrVQf74I2lHj50mkRJPDXcmwnkAlCHj9HYjSBnVb8OnhoJc25aktrGAXjUDZcKUAXlvLorEGDIAClU8ElBJEjfTSI1yf5-T0oDWKmgrB9I-8PYPawIiAr1bgxbvGop8yZ-gE8uZ7G~74R34kcc7nlU~gZIU~A1VLoZ6ZuovxkZkNThTFm-2zgGuUVmFAaVx7kf5xgIIORjp0lE5fsPZfEqFgiOU-cWgn0OvhePFZCdcpGQui5BaYguk9XGf4Rdo~tcDsqyUbiIqTg8O3WduuPXBd3HFTNFWXD~CzRY5BQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>) were also employed. The polymorphism screening and genotyping were performed with an Applied Biosystems 3500 Series Genetic Analyzer (Applied BioSystems, Foster City, CA, USA). The segregated SNP and SSR data of the mapping population were prepared for the CP mode of JoinMap<a reveal-id="dsx020-B57" data-open="dsx020-B57" class="link link-ref link-reveal xref-bibr"><sup>57</sup></a> (version 4) and classified into groups using the Grouping Module of JoinMap with LOD scores of 4 to 7. The marker order and relative map distances were calculated using its regression-mapping algorithm with the following parameters: Haldaneâ€™s mapping function,â€‰â‰¤0.35 recombination frequency, andâ€‰â‰¥2.0 LOD score. LPmerge<a reveal-id="dsx020-B58" data-open="dsx020-B58" class="link link-ref link-reveal xref-bibr"><sup>58</sup></a> (version 1.5) was used to integrate the linkage maps into the consensus map. The graphical linkage maps were drawn using MapChart<a reveal-id="dsx020-B59" data-open="dsx020-B59" class="link link-ref link-reveal xref-bibr"><sup>59</sup></a> (version 2.2).</p><p>For comparing the genome of sweet cherry with those of its relatives, similarity searches between the SNP-associated sequences of PAV_r1.0 (201â€‰bp in length) and the pseudomolecule sequences of <em>Pr. persica</em> (GDR v2.0.a1),<a reveal-id="dsx020-B2" data-open="dsx020-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a><sup>,</sup><em>P. mume</em>,<a reveal-id="dsx020-B5" data-open="dsx020-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a><sup>,</sup><em>F. vesca</em> (GDR v2.0.a1),<a reveal-id="dsx020-B28" data-open="dsx020-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> and <em>P. bretschneideri</em><a reveal-id="dsx020-B3" data-open="dsx020-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> were carried out using BLASTN searches with an E-value cutoff of 1Eâ€“15. The graphical comparative maps were drawn using Circos<a reveal-id="dsx020-B60" data-open="dsx020-B60" class="link link-ref link-reveal xref-bibr"><sup>60</sup></a> (version 0.69-3).</p>                            <h3 scrollto-destination=106356485 id="106356485" class="section-title" >2.7. Pseudomolecule construction</h3>
<p>Two approaches, based on the genetic map and synteny between the genomes of sweet cherry and peach, were used to construct pseudomolecule sequences. First, the genome scaffolds were assigned to the genetic map. If more than two marker loci were mapped on a single scaffold, the scaffolds were assigned with the orientation based on the marker order. Next, sequence similarity analysis of peptide sequences predicted from PAV_r1.0 was performed against those of the peach genome using BLASTP with an E-value cutoff of 1Eâ€“5. Scaffolds having a linear relationship (<em>R</em><sup>2</sup>â€‰&gt;â€‰0.6) with at least five continuous genes between the two genomes were assigned to a chromosome with that orientation. The resulting pseudomolecule sequences were aligned to the peach genome, GDR v2.0.a1, with NUCmer of the MUMmer package<a reveal-id="dsx020-B61" data-open="dsx020-B61" class="link link-ref link-reveal xref-bibr"><sup>61</sup></a> (version 2.23).</p>                            <h3 scrollto-destination=106356487 id="106356487" class="section-title" >2.8. Whole-genome resequencing for identification of DNA polymorphism</h3>
<p>Sequence reads from the PE sequencing of six varieties, Benikirari, Benisayaka, Benishuho, Benitemari, Beniyutaka, and Nanyo, were trimmed and filtered as above, and mapped on the PAV_r1.0 reference sequence with Bowtie 2<a reveal-id="dsx020-B37" data-open="dsx020-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> (version 2.2.3: parameters of â€“minins 100, â€“no-mixed, and -k 2). The resulting BAM files were subjected to SNP calling with the mpileup command of SAMtools<a reveal-id="dsx020-B38" data-open="dsx020-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> (version 0.1.19: parameter of -Duf) and the view command of BCFtools<a reveal-id="dsx020-B38" data-open="dsx020-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> (parameter of -vcg), and filtered with VCFtools<a reveal-id="dsx020-B40" data-open="dsx020-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> (version 0.1.12b: parameters of â€“minQ 50, â€“minGQ 20, â€“minDP 10, and â€“maxDP 100). The effects of mutations on gene function were predicted with SnpEff<a reveal-id="dsx020-B62" data-open="dsx020-B62" class="link link-ref link-reveal xref-bibr"><sup>62</sup></a> (version 4.2: parameters of -no-downstream and -no-upstream). SnpEff predicted the sequence ontology<a reveal-id="dsx020-B63" data-open="dsx020-B63" class="link link-ref link-reveal xref-bibr"><sup>63</sup></a> of the mutations and assigned them to four predefined impact categories: high- (e.g. nonsense mutations and frameshift mutations), moderate- (e.g. missense mutations), modifier- (e.g. intron and intergenic mutations) and low-impact (e.g. synonymous mutations) (see <a class="link link-uri" href="http://snpeff.sourceforge.net" target="_blank">http://snpeff.sourceforge.net</a> (8 May 2017, date last accessed) for details).</p><p>Copy number variations (CNVs) were detected with CNV-seq<a reveal-id="dsx020-B64" data-open="dsx020-B64" class="link link-ref link-reveal xref-bibr"><sup>64</sup></a> (version 0.2.7: parameter of â€“genome-size 272361615) using the BAM files, in which the six varieties were used as test lines with PAV_r1.0 as a reference.</p>                            <h3 scrollto-destination=106356490 id="106356490" class="section-title" >2.9. Development of CAPS, indel, and SSR markers</h3>
<p>SNP2CAPS<a reveal-id="dsx020-B65" data-open="dsx020-B65" class="link link-ref link-reveal xref-bibr"><sup>65</sup></a> was used for developing cleaved amplified polymorphic sequence (CAPS) markers with 19 restriction enzymes: <em>Afa</em>I, <em>Alu</em>I, <em>Apa</em>I, <em>Bam</em>HI, <em>Bgl</em>II, <em>Dra</em>I, <em>Eco</em>RI, <em>Eco</em>RV, <em>Hae</em>III, <em>Hha</em>I, <em>Hin</em>dIII, <em>Kpn</em>I, <em>Mbo</em>I, <em>Msp</em>I, <em>Pst</em>I, <em>Sac</em>I, <em>Sal</em>I, <em>Xba</em>I, and <em>Xho</em>I. SSRs were identified using the mismatched variable penalty (mmvp) mode of SciRoKoCo<a reveal-id="dsx020-B66" data-open="dsx020-B66" class="link link-ref link-reveal xref-bibr"><sup>66</sup></a> to detect imperfect microsatellites. Indels were selected from the VCF file of the resequencing analysis with VCFtools<a reveal-id="dsx020-B40" data-open="dsx020-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> (version 0.1.12b: parameter of â€“keep-only-indels). Oligonucleotides for the markers were designed using PRIMER3<a reveal-id="dsx020-B67" data-open="dsx020-B67" class="link link-ref link-reveal xref-bibr"><sup>67</sup></a> (version 2.2.3).</p>                            <h2 scrollto-destination=106356492 id="106356492" class="section-title" >3. Results</h2>
                            <h3 scrollto-destination=106356493 id="106356493" class="section-title" >3.1. Sequencing and assembly of the sweet cherry genome</h3>
<p>A total of 357.5 million high-quality reads (32.9â€‰Gb) were obtained from the Satonishiki cherry PE library, which had an insert size of 500â€‰bp (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx020/2/dsx020_supp_sweetcherrygenome-supplementarytables.xlsx?Expires=2147483647&amp;Signature=TiRI9J~ZD4Ek9u56ktrPJXS93YJ4qGZITEQupuOKHddrgtrVQf74I2lHj50mkRJPDXcmwnkAlCHj9HYjSBnVb8OnhoJc25aktrGAXjUDZcKUAXlvLorEGDIAClU8ElBJEjfTSI1yf5-T0oDWKmgrB9I-8PYPawIiAr1bgxbvGop8yZ-gE8uZ7G~74R34kcc7nlU~gZIU~A1VLoZ6ZuovxkZkNThTFm-2zgGuUVmFAaVx7kf5xgIIORjp0lE5fsPZfEqFgiOU-cWgn0OvhePFZCdcpGQui5BaYguk9XGf4Rdo~tcDsqyUbiIqTg8O3WduuPXBd3HFTNFWXD~CzRY5BQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). The distribution of distinct <em>k</em>-mers (<em>k</em>â€‰=â€‰17) showed two peaks at multiplicities of 41 and 77 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx020/2/dsx020_supp_sweetcherrygenome-supplementaryfigures.r1.pdf?Expires=2147483647&amp;Signature=EpzskbJ8FdkxtePPGHzVUHR8wbz7PuhsMtIZp1mBYfuYpDnoU860lHAiOXaTih335VqggdpxNmN~DlHv9jCQjL67NgdKEe0mJf6VW~rFFNlKSOSwGF16QSU~ixooLa4MnFfPBuizWoRuWKHC5qHhWq9m7g0mIOI~a3cC4r-I38M7qqsg8TxxRMqONuq~9wlcV5s9xtt-0Ia6Fmkgv9vqIrt-f~ik0NUXpBMR6Br6pBv9tJQJvkf47kQLiRbDw4zZbNs27lnG2PNg44Zj75jgFNV8z2KYtBTZOOPF3qb1rjsE96U2cnvlArNo-eTe~6GEVJGGYM0KqyUXlBDoAbMxJA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>). The low and high peaks represent heterozygous and homozygous sequences, respectively, suggesting that the heterogeneity of the genome was low. We estimated the genome size to be 352.9â€‰Mb from the higher peak, which almost agreed with the value measured by flow cytometry, 338â€‰Mb.<a reveal-id="dsx020-B68" data-open="dsx020-B68" class="link link-ref link-reveal xref-bibr"><sup>68</sup></a></p><p>The 357.5 million PE reads were assembled into contigs using SOAPdenovo2 with five <em>k</em>-mer sizes (51, 61, 71, 81, and 91), and the obtained contigs were assembled into scaffolds with 121.3 million MP reads (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx020/2/dsx020_supp_sweetcherrygenome-supplementarytables.xlsx?Expires=2147483647&Signature=TiRI9J~ZD4Ek9u56ktrPJXS93YJ4qGZITEQupuOKHddrgtrVQf74I2lHj50mkRJPDXcmwnkAlCHj9HYjSBnVb8OnhoJc25aktrGAXjUDZcKUAXlvLorEGDIAClU8ElBJEjfTSI1yf5-T0oDWKmgrB9I-8PYPawIiAr1bgxbvGop8yZ-gE8uZ7G~74R34kcc7nlU~gZIU~A1VLoZ6ZuovxkZkNThTFm-2zgGuUVmFAaVx7kf5xgIIORjp0lE5fsPZfEqFgiOU-cWgn0OvhePFZCdcpGQui5BaYguk9XGf4Rdo~tcDsqyUbiIqTg8O3WduuPXBd3HFTNFWXD~CzRY5BQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). When a <em>k</em>-mer size of 81 was employed, the total length of the scaffolds (373.7â€‰Mb) was close to the estimated genome size and the N50 length (114.8â€‰kb) was the longest. In parallel, we investigated another assembling tool, Platanus. However, while the N50 length (462.8â€‰kb) was longer than that from SOAPdenovo2, the total length of the assembly (273.2â€‰Mb) was âˆ¼100â€‰Mb shorter than expected. Therefore, we used the assembled sequences from SOAPdenovo2 (<em>k</em>-merâ€‰=â€‰81) in further analyses. Gap sequences of 47.7 million bases, represented by Ns, were filled using the PE reads. After removing sequences from contaminating sources (1.6â€‰Mb from organelles, bacteria, fungi, and humans) and sequences that were shorter than 1,000 bases (97.0â€‰Mb) (see also the next section), the remaining 10,148 sequences were designated PAV_r1.0 (<a reveal-id="dsx020-T1" data-open="dsx020-T1" class="link link-reveal link-table xref-fig">Table 1</a>), which was 272.4â€‰Mb with an N50 length of 219.6â€‰kb (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx020/2/dsx020_supp_sweetcherrygenome-supplementarytables.xlsx?Expires=2147483647&Signature=TiRI9J~ZD4Ek9u56ktrPJXS93YJ4qGZITEQupuOKHddrgtrVQf74I2lHj50mkRJPDXcmwnkAlCHj9HYjSBnVb8OnhoJc25aktrGAXjUDZcKUAXlvLorEGDIAClU8ElBJEjfTSI1yf5-T0oDWKmgrB9I-8PYPawIiAr1bgxbvGop8yZ-gE8uZ7G~74R34kcc7nlU~gZIU~A1VLoZ6ZuovxkZkNThTFm-2zgGuUVmFAaVx7kf5xgIIORjp0lE5fsPZfEqFgiOU-cWgn0OvhePFZCdcpGQui5BaYguk9XGf4Rdo~tcDsqyUbiIqTg8O3WduuPXBd3HFTNFWXD~CzRY5BQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>). The GC content was 37.7%, and the length of ambiguous bases (Ns) was 25.6â€‰Mb. The genomic completeness of PAV_r1.0 examined with BUSCO revealed that PAV_r1.0 had 918 (96.0%) complete orthologues and 17 (1.8%) fragmented orthologues, indicating that PAV_r1.0 had good coverage of the gene space of the sweet cherry genome (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx020/2/dsx020_supp_sweetcherrygenome-supplementarytables.xlsx?Expires=2147483647&Signature=TiRI9J~ZD4Ek9u56ktrPJXS93YJ4qGZITEQupuOKHddrgtrVQf74I2lHj50mkRJPDXcmwnkAlCHj9HYjSBnVb8OnhoJc25aktrGAXjUDZcKUAXlvLorEGDIAClU8ElBJEjfTSI1yf5-T0oDWKmgrB9I-8PYPawIiAr1bgxbvGop8yZ-gE8uZ7G~74R34kcc7nlU~gZIU~A1VLoZ6ZuovxkZkNThTFm-2zgGuUVmFAaVx7kf5xgIIORjp0lE5fsPZfEqFgiOU-cWgn0OvhePFZCdcpGQui5BaYguk9XGf4Rdo~tcDsqyUbiIqTg8O3WduuPXBd3HFTNFWXD~CzRY5BQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>).</p>                            <h3 scrollto-destination=106356496 id="106356496" class="section-title" >3.2. Repetitive sequence analysis</h3>
<p>In PAV_r1.0 (273.2â€‰Mb), 119.4â€‰Mb (43.8%) of repetitive sequence was identified (e.g. transposons and retrotransposons), consisting of 34.3â€‰Mb of reported repetitive sequences and 85.1â€‰Mb of repeats unique to PAV_r1.0 (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx020/2/dsx020_supp_sweetcherrygenome-supplementarytables.xlsx?Expires=2147483647&Signature=TiRI9J~ZD4Ek9u56ktrPJXS93YJ4qGZITEQupuOKHddrgtrVQf74I2lHj50mkRJPDXcmwnkAlCHj9HYjSBnVb8OnhoJc25aktrGAXjUDZcKUAXlvLorEGDIAClU8ElBJEjfTSI1yf5-T0oDWKmgrB9I-8PYPawIiAr1bgxbvGop8yZ-gE8uZ7G~74R34kcc7nlU~gZIU~A1VLoZ6ZuovxkZkNThTFm-2zgGuUVmFAaVx7kf5xgIIORjp0lE5fsPZfEqFgiOU-cWgn0OvhePFZCdcpGQui5BaYguk9XGf4Rdo~tcDsqyUbiIqTg8O3WduuPXBd3HFTNFWXD~CzRY5BQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>). The reported sequences were predominantly LTR retrotransposons: <em>Copia</em> and <em>Gypsy</em> elements occupying 8.4 and 8.0â€‰Mb, respectively. On the other hand, repeats occupied 84.2% of the eliminated sequences, each of which wasâ€‰&lt;1,000â€‰bp in length, suggesting that this repeat richness might collapse long assemblies.</p>                            <h3 scrollto-destination=106356498 id="106356498" class="section-title" >3.3. Gene predictions and functional annotations</h3>
<p>We found 536 tRNA- and 61 rRNA-encoding genes in PAV_r1.0 (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx020/2/dsx020_supp_sweetcherrygenome-supplementarytables.xlsx?Expires=2147483647&Signature=TiRI9J~ZD4Ek9u56ktrPJXS93YJ4qGZITEQupuOKHddrgtrVQf74I2lHj50mkRJPDXcmwnkAlCHj9HYjSBnVb8OnhoJc25aktrGAXjUDZcKUAXlvLorEGDIAClU8ElBJEjfTSI1yf5-T0oDWKmgrB9I-8PYPawIiAr1bgxbvGop8yZ-gE8uZ7G~74R34kcc7nlU~gZIU~A1VLoZ6ZuovxkZkNThTFm-2zgGuUVmFAaVx7kf5xgIIORjp0lE5fsPZfEqFgiOU-cWgn0OvhePFZCdcpGQui5BaYguk9XGf4Rdo~tcDsqyUbiIqTg8O3WduuPXBd3HFTNFWXD~CzRY5BQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S6 and S7</a></span>). Subsequently, we predicted protein-encoding sequences in PAV_r1.0 using evidence-, <em>ab-initio</em>-, and homology-based methods in a MAKER pipeline. In the evidence-based method, we used 189,538 transcribed sequences (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx020/2/dsx020_supp_sweetcherrygenome-supplementarytables.xlsx?Expires=2147483647&Signature=TiRI9J~ZD4Ek9u56ktrPJXS93YJ4qGZITEQupuOKHddrgtrVQf74I2lHj50mkRJPDXcmwnkAlCHj9HYjSBnVb8OnhoJc25aktrGAXjUDZcKUAXlvLorEGDIAClU8ElBJEjfTSI1yf5-T0oDWKmgrB9I-8PYPawIiAr1bgxbvGop8yZ-gE8uZ7G~74R34kcc7nlU~gZIU~A1VLoZ6ZuovxkZkNThTFm-2zgGuUVmFAaVx7kf5xgIIORjp0lE5fsPZfEqFgiOU-cWgn0OvhePFZCdcpGQui5BaYguk9XGf4Rdo~tcDsqyUbiIqTg8O3WduuPXBd3HFTNFWXD~CzRY5BQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S8</a></span>) obtained from the assembly of 57.6 million transcript reads from eight samples (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx020/2/dsx020_supp_sweetcherrygenome-supplementarytables.xlsx?Expires=2147483647&Signature=TiRI9J~ZD4Ek9u56ktrPJXS93YJ4qGZITEQupuOKHddrgtrVQf74I2lHj50mkRJPDXcmwnkAlCHj9HYjSBnVb8OnhoJc25aktrGAXjUDZcKUAXlvLorEGDIAClU8ElBJEjfTSI1yf5-T0oDWKmgrB9I-8PYPawIiAr1bgxbvGop8yZ-gE8uZ7G~74R34kcc7nlU~gZIU~A1VLoZ6ZuovxkZkNThTFm-2zgGuUVmFAaVx7kf5xgIIORjp0lE5fsPZfEqFgiOU-cWgn0OvhePFZCdcpGQui5BaYguk9XGf4Rdo~tcDsqyUbiIqTg8O3WduuPXBd3HFTNFWXD~CzRY5BQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S9</a></span>) to predict 23,709 genes (with .mk suffix), excluding TE-like sequences. Moreover, an additional 19,964 non-TE genes, which did not overlap the 23,709 genes, were predicted using the <em>ab-initio</em> method (with .br suffix). In total, 43,349 genes plus 324 pseudogenes were predicted to be in PAV_r1.0 (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx020/2/dsx020_supp_sweetcherrygenome-supplementarytables.xlsx?Expires=2147483647&Signature=TiRI9J~ZD4Ek9u56ktrPJXS93YJ4qGZITEQupuOKHddrgtrVQf74I2lHj50mkRJPDXcmwnkAlCHj9HYjSBnVb8OnhoJc25aktrGAXjUDZcKUAXlvLorEGDIAClU8ElBJEjfTSI1yf5-T0oDWKmgrB9I-8PYPawIiAr1bgxbvGop8yZ-gE8uZ7G~74R34kcc7nlU~gZIU~A1VLoZ6ZuovxkZkNThTFm-2zgGuUVmFAaVx7kf5xgIIORjp0lE5fsPZfEqFgiOU-cWgn0OvhePFZCdcpGQui5BaYguk9XGf4Rdo~tcDsqyUbiIqTg8O3WduuPXBd3HFTNFWXD~CzRY5BQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S10</a></span>). The GC content of the coding sequences was 44.3%, and the N50 length was 1,707 bases (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx020/2/dsx020_supp_sweetcherrygenome-supplementarytables.xlsx?Expires=2147483647&Signature=TiRI9J~ZD4Ek9u56ktrPJXS93YJ4qGZITEQupuOKHddrgtrVQf74I2lHj50mkRJPDXcmwnkAlCHj9HYjSBnVb8OnhoJc25aktrGAXjUDZcKUAXlvLorEGDIAClU8ElBJEjfTSI1yf5-T0oDWKmgrB9I-8PYPawIiAr1bgxbvGop8yZ-gE8uZ7G~74R34kcc7nlU~gZIU~A1VLoZ6ZuovxkZkNThTFm-2zgGuUVmFAaVx7kf5xgIIORjp0lE5fsPZfEqFgiOU-cWgn0OvhePFZCdcpGQui5BaYguk9XGf4Rdo~tcDsqyUbiIqTg8O3WduuPXBd3HFTNFWXD~CzRY5BQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S10</a></span>).</p><p>The 43,349 genes were further annotated using GO, KOG, and KEGG. In the GO analysis, 9,256 (21.4%), 3,610 (8.3%), and 14,582 (33.6%) genes were assigned to the GO slim terms of biological process, cellular component, and molecular function, respectively, (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx020/2/dsx020_supp_sweetcherrygenome-supplementarytables.xlsx?Expires=2147483647&Signature=TiRI9J~ZD4Ek9u56ktrPJXS93YJ4qGZITEQupuOKHddrgtrVQf74I2lHj50mkRJPDXcmwnkAlCHj9HYjSBnVb8OnhoJc25aktrGAXjUDZcKUAXlvLorEGDIAClU8ElBJEjfTSI1yf5-T0oDWKmgrB9I-8PYPawIiAr1bgxbvGop8yZ-gE8uZ7G~74R34kcc7nlU~gZIU~A1VLoZ6ZuovxkZkNThTFm-2zgGuUVmFAaVx7kf5xgIIORjp0lE5fsPZfEqFgiOU-cWgn0OvhePFZCdcpGQui5BaYguk9XGf4Rdo~tcDsqyUbiIqTg8O3WduuPXBd3HFTNFWXD~CzRY5BQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S11</a></span>). In the KOG analysis, 2,829, 4,690, and 4,078 genes had significant similarity to genes involved in information storage and processing, cellular processing and signaling, and metabolism (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx020/2/dsx020_supp_sweetcherrygenome-supplementarytables.xlsx?Expires=2147483647&Signature=TiRI9J~ZD4Ek9u56ktrPJXS93YJ4qGZITEQupuOKHddrgtrVQf74I2lHj50mkRJPDXcmwnkAlCHj9HYjSBnVb8OnhoJc25aktrGAXjUDZcKUAXlvLorEGDIAClU8ElBJEjfTSI1yf5-T0oDWKmgrB9I-8PYPawIiAr1bgxbvGop8yZ-gE8uZ7G~74R34kcc7nlU~gZIU~A1VLoZ6ZuovxkZkNThTFm-2zgGuUVmFAaVx7kf5xgIIORjp0lE5fsPZfEqFgiOU-cWgn0OvhePFZCdcpGQui5BaYguk9XGf4Rdo~tcDsqyUbiIqTg8O3WduuPXBd3HFTNFWXD~CzRY5BQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S12</a></span>). Furthermore, 1,672 genes were mapped to KEGG metabolic pathways (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx020/2/dsx020_supp_sweetcherrygenome-supplementarytables.xlsx?Expires=2147483647&Signature=TiRI9J~ZD4Ek9u56ktrPJXS93YJ4qGZITEQupuOKHddrgtrVQf74I2lHj50mkRJPDXcmwnkAlCHj9HYjSBnVb8OnhoJc25aktrGAXjUDZcKUAXlvLorEGDIAClU8ElBJEjfTSI1yf5-T0oDWKmgrB9I-8PYPawIiAr1bgxbvGop8yZ-gE8uZ7G~74R34kcc7nlU~gZIU~A1VLoZ6ZuovxkZkNThTFm-2zgGuUVmFAaVx7kf5xgIIORjp0lE5fsPZfEqFgiOU-cWgn0OvhePFZCdcpGQui5BaYguk9XGf4Rdo~tcDsqyUbiIqTg8O3WduuPXBd3HFTNFWXD~CzRY5BQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S13</a></span>).</p><p>For comparing the genes predicted in PAV_r1.0 with those of other Rosaceae species, the 43,349 genes were clustered with the genes of peach, Japanese apricot, apple, and strawberry to generate 75,627 clusters. A total of 3,459 clusters, including 4,535 genes from sweet cherry, were observed in all investigated species (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx020/2/dsx020_supp_sweetcherrygenome-supplementaryfigures.r1.pdf?Expires=2147483647&Signature=EpzskbJ8FdkxtePPGHzVUHR8wbz7PuhsMtIZp1mBYfuYpDnoU860lHAiOXaTih335VqggdpxNmN~DlHv9jCQjL67NgdKEe0mJf6VW~rFFNlKSOSwGF16QSU~ixooLa4MnFfPBuizWoRuWKHC5qHhWq9m7g0mIOI~a3cC4r-I38M7qqsg8TxxRMqONuq~9wlcV5s9xtt-0Ia6Fmkgv9vqIrt-f~ik0NUXpBMR6Br6pBv9tJQJvkf47kQLiRbDw4zZbNs27lnG2PNg44Zj75jgFNV8z2KYtBTZOOPF3qb1rjsE96U2cnvlArNo-eTe~6GEVJGGYM0KqyUXlBDoAbMxJA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>). On the other hand, whereas 869 clusters were absent from only sweet cherry, 16,151 clusters, consisting of 21,642 genes, were specific to sweet cherry. The proportion of <em>ab initio</em> genes in the sweet cherry specific clusters, which annotation edit distance (AED)<a reveal-id="dsx020-B69" data-open="dsx020-B69" class="link link-ref link-reveal xref-bibr"><sup>69</sup></a> was 0.28 on average, was 68.4%, while that in other clusters (AED score of 0.16) was 22.3%.</p>                            <h3 scrollto-destination=106356502 id="106356502" class="section-title" >3.4. Construction of the consensus genetic map and comparative map</h3>
<p>To anchor the genomic sequences to the sweet cherry chromosomes, high-density genetic maps for three F1 populations, C-303, C-309, and HRO, were developed using ddRAD-Seq. Approximately 1.2 million, 1.9 million, and 1.4 million high-quality reads were obtained from ddRAD-Seq libraries for C-303, C-309, and HRO, respectively, and 90.6% of the reads across the three populations aligned to PAV_r1.0; these were used to detect SNP candidates (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx020/2/dsx020_supp_sweetcherrygenome-supplementarytables.xlsx?Expires=2147483647&amp;Signature=TiRI9J~ZD4Ek9u56ktrPJXS93YJ4qGZITEQupuOKHddrgtrVQf74I2lHj50mkRJPDXcmwnkAlCHj9HYjSBnVb8OnhoJc25aktrGAXjUDZcKUAXlvLorEGDIAClU8ElBJEjfTSI1yf5-T0oDWKmgrB9I-8PYPawIiAr1bgxbvGop8yZ-gE8uZ7G~74R34kcc7nlU~gZIU~A1VLoZ6ZuovxkZkNThTFm-2zgGuUVmFAaVx7kf5xgIIORjp0lE5fsPZfEqFgiOU-cWgn0OvhePFZCdcpGQui5BaYguk9XGf4Rdo~tcDsqyUbiIqTg8O3WduuPXBd3HFTNFWXD~CzRY5BQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S14</a></span>). After filtering out low-quality candidates, 1,384, 1,475, and 1,157 high-quality SNPs were selected for C-303, C-309, and HRO, respectively. Subsequent linkage analysis, together with 53 and 37 SSRs for C-303 and C-309, respectively, generated eight linkage groups for the parental lines of each population, except for Beniyutaka of C-303 (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx020/2/dsx020_supp_sweetcherrygenome-supplementarytables.xlsx?Expires=2147483647&amp;Signature=TiRI9J~ZD4Ek9u56ktrPJXS93YJ4qGZITEQupuOKHddrgtrVQf74I2lHj50mkRJPDXcmwnkAlCHj9HYjSBnVb8OnhoJc25aktrGAXjUDZcKUAXlvLorEGDIAClU8ElBJEjfTSI1yf5-T0oDWKmgrB9I-8PYPawIiAr1bgxbvGop8yZ-gE8uZ7G~74R34kcc7nlU~gZIU~A1VLoZ6ZuovxkZkNThTFm-2zgGuUVmFAaVx7kf5xgIIORjp0lE5fsPZfEqFgiOU-cWgn0OvhePFZCdcpGQui5BaYguk9XGf4Rdo~tcDsqyUbiIqTg8O3WduuPXBd3HFTNFWXD~CzRY5BQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S15 and S16</a></span>). The six linkage maps were integrated into a consensus linkage map consisting of 2,317 SNPs and 65 SSRs, covering a total of 1,165â€‰cM (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx020/2/dsx020_supp_sweetcherrygenome-supplementaryfigures.r1.pdf?Expires=2147483647&amp;Signature=EpzskbJ8FdkxtePPGHzVUHR8wbz7PuhsMtIZp1mBYfuYpDnoU860lHAiOXaTih335VqggdpxNmN~DlHv9jCQjL67NgdKEe0mJf6VW~rFFNlKSOSwGF16QSU~ixooLa4MnFfPBuizWoRuWKHC5qHhWq9m7g0mIOI~a3cC4r-I38M7qqsg8TxxRMqONuq~9wlcV5s9xtt-0Ia6Fmkgv9vqIrt-f~ik0NUXpBMR6Br6pBv9tJQJvkf47kQLiRbDw4zZbNs27lnG2PNg44Zj75jgFNV8z2KYtBTZOOPF3qb1rjsE96U2cnvlArNo-eTe~6GEVJGGYM0KqyUXlBDoAbMxJA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>, <span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx020/2/dsx020_supp_sweetcherrygenome-supplementarytables.xlsx?Expires=2147483647&amp;Signature=TiRI9J~ZD4Ek9u56ktrPJXS93YJ4qGZITEQupuOKHddrgtrVQf74I2lHj50mkRJPDXcmwnkAlCHj9HYjSBnVb8OnhoJc25aktrGAXjUDZcKUAXlvLorEGDIAClU8ElBJEjfTSI1yf5-T0oDWKmgrB9I-8PYPawIiAr1bgxbvGop8yZ-gE8uZ7G~74R34kcc7nlU~gZIU~A1VLoZ6ZuovxkZkNThTFm-2zgGuUVmFAaVx7kf5xgIIORjp0lE5fsPZfEqFgiOU-cWgn0OvhePFZCdcpGQui5BaYguk9XGf4Rdo~tcDsqyUbiIqTg8O3WduuPXBd3HFTNFWXD~CzRY5BQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S15 and S16</a></span>).</p><p>Using the consensus map, the genomic structure of sweet cherry was compared with those of peach (<em>Pr. persica</em>), Japanese apricot (<em>Pr. mume</em>), strawberry (<em>F. vesca</em>), and Chinese pear (<em>Py. bretschneideri</em>). Out of the 2,317 mapped SNP loci, the flanking sequences had significant similarity to 2,280 loci in the peach genome, followed by 2,194 in Japanese apricot, 847 in Chinese pear, and 556 in strawberry. The sweet cherry linkage groups were therefore numbered in accordance with the names of peach chromosomes, because there was a one-to-one correspondence between the two genomes (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx020/2/dsx020_supp_sweetcherrygenome-supplementaryfigures.r1.pdf?Expires=2147483647&Signature=EpzskbJ8FdkxtePPGHzVUHR8wbz7PuhsMtIZp1mBYfuYpDnoU860lHAiOXaTih335VqggdpxNmN~DlHv9jCQjL67NgdKEe0mJf6VW~rFFNlKSOSwGF16QSU~ixooLa4MnFfPBuizWoRuWKHC5qHhWq9m7g0mIOI~a3cC4r-I38M7qqsg8TxxRMqONuq~9wlcV5s9xtt-0Ia6Fmkgv9vqIrt-f~ik0NUXpBMR6Br6pBv9tJQJvkf47kQLiRbDw4zZbNs27lnG2PNg44Zj75jgFNV8z2KYtBTZOOPF3qb1rjsE96U2cnvlArNo-eTe~6GEVJGGYM0KqyUXlBDoAbMxJA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span>).</p>                            <h3 scrollto-destination=106356505 id="106356505" class="section-title" >3.5. Establishment of the pseudomolecules</h3>
<p>Pseudomolecules for sweet cherry were established using map- and synteny-based strategies. In the map-based strategy, 162 genomic sequences spanning 14.5â€‰Mb were aligned and ordered on the consensus map using the positions of the 2,280 SNPs as anchors. Furthermore, using the synteny-based strategy, 743 sequences (177.3â€‰Mb) were mapped on the peach genome, with the criterion that peptide sequences ofâ€‰â‰¥5 continuous genes from a scaffold sequence significantly matched those in the peach genome in the same order. In total, 905 scaffolds, spanning 191.7â€‰Mb (70.4% of the length of PAV_r1.0) and carrying 31,452 genes (72.0% of the predicted genes), were anchored to the sweet cherry chromosomes (<a reveal-id="dsx020-T2" data-open="dsx020-T2" class="link link-reveal link-table xref-fig">Table 2</a>). The scaffold sequences were concatenated with 10,000â€‰Ns into pseudomolecule sequences (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx020/2/dsx020_supp_sweetcherrygenome-supplementarytables.xlsx?Expires=2147483647&Signature=TiRI9J~ZD4Ek9u56ktrPJXS93YJ4qGZITEQupuOKHddrgtrVQf74I2lHj50mkRJPDXcmwnkAlCHj9HYjSBnVb8OnhoJc25aktrGAXjUDZcKUAXlvLorEGDIAClU8ElBJEjfTSI1yf5-T0oDWKmgrB9I-8PYPawIiAr1bgxbvGop8yZ-gE8uZ7G~74R34kcc7nlU~gZIU~A1VLoZ6ZuovxkZkNThTFm-2zgGuUVmFAaVx7kf5xgIIORjp0lE5fsPZfEqFgiOU-cWgn0OvhePFZCdcpGQui5BaYguk9XGf4Rdo~tcDsqyUbiIqTg8O3WduuPXBd3HFTNFWXD~CzRY5BQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S17</a></span>). As expected, the pseudomolecules evenly covered 60.2% of the peach genome (<a reveal-id="dsx020-F1" data-open="dsx020-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>). </p>                        <a id="106356507" scrollto-destination="106356507"></a>
<div id="dsx020-F1" data-id="dsx020-F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx020/2/m_dsx020f1.png?Expires=2147483647&Signature=I~aloaffCwwj~-UQi1KxTAWblix1QHhF4QPQI1D0qH2UUJZIaPLkSSN9rjjeuPu9998uioGODcm9oM51P7vW9li4FLA~azIKQB92tvvGSiTBTnHYbM47A1eVJ1vnHQG6qGpT3y-ayhaGjlWjgnDmAOXXlREdQAEXL2wbMqUTiycLnEmLYVIbH4VQf173B2lWf8ML2v7WB92N7l4OsIz0YCizptak~HFyy~1gjf3pRQQqA-gSwNfGdYWVUzG95~jW56cNbyxwFKNEUNtraROwX255mrokppPruJyiMLtSWsD6LhBycybuxyIM1tWEdWor4D-BXMktqXvw1BINY8Ewiw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Synteny of the genomes of sweet cherry and peach. X-axis: the genome of peach (GDR v2.0.a1); Y-axis: the genome of sweet cherry (PAV_r1.0). Sequence similarity is indicated by colors." path-from-xml="dsx020f1.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106356507/dsx020f1.png" path-from-xml="dsx020f1.png" target="_blank">View large</a><a section="106356507" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx020/2/dsx020f1.png?Expires=2147483647&Signature=FnaQlxsZtkzz5-4kGsloBt8ttKnF2ABdNM727bvFb3Czl5VwtF9B3kkYbmShA5UqkrPgnuZZ-4isRoQWxCFTtu--jVDLoUkTOaLRD0~sWAbdeMlzhslQce-vcvaWWscSDQcRIzfZ-Qkfk~bs-O7C1gUszT7jP-P26emk-~RjIo0435y30R~tQsYhEUgACQ05LMVglZo-DXKSVOJVBekxTCvnanyCTnGyzYWBHAPxJFpOdysWHofkNQVn7UovLpjZXufO5TJO~SCW5lB7lyn3JiX77BDGXHAeXXswGex8H9dbpyNybvto71on7Q7jLEGU6HsbuYJAeqILD6DVwENz3w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106356507&ar=3835854&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx020f1.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Synteny of the genomes of sweet cherry and peach. X-axis: the genome of peach (GDR v2.0.a1); Y-axis: the genome of sweet cherry (PAV_r1.0). Sequence similarity is indicated by colors.</p></div></div><div content-id="dsx020-F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx020/2/m_dsx020f1.png?Expires=2147483647&Signature=I~aloaffCwwj~-UQi1KxTAWblix1QHhF4QPQI1D0qH2UUJZIaPLkSSN9rjjeuPu9998uioGODcm9oM51P7vW9li4FLA~azIKQB92tvvGSiTBTnHYbM47A1eVJ1vnHQG6qGpT3y-ayhaGjlWjgnDmAOXXlREdQAEXL2wbMqUTiycLnEmLYVIbH4VQf173B2lWf8ML2v7WB92N7l4OsIz0YCizptak~HFyy~1gjf3pRQQqA-gSwNfGdYWVUzG95~jW56cNbyxwFKNEUNtraROwX255mrokppPruJyiMLtSWsD6LhBycybuxyIM1tWEdWor4D-BXMktqXvw1BINY8Ewiw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Synteny of the genomes of sweet cherry and peach. X-axis: the genome of peach (GDR v2.0.a1); Y-axis: the genome of sweet cherry (PAV_r1.0). Sequence similarity is indicated by colors." path-from-xml="dsx020f1.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106356507/dsx020f1.png" path-from-xml="dsx020f1.png" target="_blank">View large</a><a section="106356507" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx020/2/dsx020f1.png?Expires=2147483647&Signature=FnaQlxsZtkzz5-4kGsloBt8ttKnF2ABdNM727bvFb3Czl5VwtF9B3kkYbmShA5UqkrPgnuZZ-4isRoQWxCFTtu--jVDLoUkTOaLRD0~sWAbdeMlzhslQce-vcvaWWscSDQcRIzfZ-Qkfk~bs-O7C1gUszT7jP-P26emk-~RjIo0435y30R~tQsYhEUgACQ05LMVglZo-DXKSVOJVBekxTCvnanyCTnGyzYWBHAPxJFpOdysWHofkNQVn7UovLpjZXufO5TJO~SCW5lB7lyn3JiX77BDGXHAeXXswGex8H9dbpyNybvto71on7Q7jLEGU6HsbuYJAeqILD6DVwENz3w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106356507&ar=3835854&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx020f1.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Synteny of the genomes of sweet cherry and peach. X-axis: the genome of peach (GDR v2.0.a1); Y-axis: the genome of sweet cherry (PAV_r1.0). Sequence similarity is indicated by colors.</p></div></div>                            <h3 scrollto-destination=106356508 id="106356508" class="section-title" >3.6. Genetic diversity analysis</h3>
<p>To investigate sequence and structural variation in the sweet cherry genome, whole-genome resequencing was performed on six varieties belonging to a single pedigree (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx020/2/dsx020_supp_sweetcherrygenome-supplementaryfigures.r1.pdf?Expires=2147483647&amp;Signature=EpzskbJ8FdkxtePPGHzVUHR8wbz7PuhsMtIZp1mBYfuYpDnoU860lHAiOXaTih335VqggdpxNmN~DlHv9jCQjL67NgdKEe0mJf6VW~rFFNlKSOSwGF16QSU~ixooLa4MnFfPBuizWoRuWKHC5qHhWq9m7g0mIOI~a3cC4r-I38M7qqsg8TxxRMqONuq~9wlcV5s9xtt-0Ia6Fmkgv9vqIrt-f~ik0NUXpBMR6Br6pBv9tJQJvkf47kQLiRbDw4zZbNs27lnG2PNg44Zj75jgFNV8z2KYtBTZOOPF3qb1rjsE96U2cnvlArNo-eTe~6GEVJGGYM0KqyUXlBDoAbMxJA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>). We obtained 28.1Ã—â€‰coverage with high-quality sequence read data (9.9â€‰Gb) for each plant line, and 91.6% of the reads were mapped onto the pseudomolecule sequences (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx020/2/dsx020_supp_sweetcherrygenome-supplementarytables.xlsx?Expires=2147483647&amp;Signature=TiRI9J~ZD4Ek9u56ktrPJXS93YJ4qGZITEQupuOKHddrgtrVQf74I2lHj50mkRJPDXcmwnkAlCHj9HYjSBnVb8OnhoJc25aktrGAXjUDZcKUAXlvLorEGDIAClU8ElBJEjfTSI1yf5-T0oDWKmgrB9I-8PYPawIiAr1bgxbvGop8yZ-gE8uZ7G~74R34kcc7nlU~gZIU~A1VLoZ6ZuovxkZkNThTFm-2zgGuUVmFAaVx7kf5xgIIORjp0lE5fsPZfEqFgiOU-cWgn0OvhePFZCdcpGQui5BaYguk9XGf4Rdo~tcDsqyUbiIqTg8O3WduuPXBd3HFTNFWXD~CzRY5BQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S18</a></span>).</p><p>A total of 1,179,268 sequence variants, consisting of 1,016,866 SNPs and 162,402 insertions/deletions (up to 15â€‰bp differences), were discovered. The densities of SNPs and indels in the genome were estimated to be 412.0 and 65.8 per 100â€‰kb, respectively. Among the SNPs, the major and minor substitutions were G/C to A/T transversions (31.2%) and G/C to C/G transversions (6.0%), respectively, and the transitions/transversions ratio was 1.5 across the six varieties. Differing numbers of sequence variants with respect to PAV_r1.0 were observed, ranging from 527,049 in Benishuho to 640,683 in Benitemari (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx020/2/dsx020_supp_sweetcherrygenome-supplementarytables.xlsx?Expires=2147483647&Signature=TiRI9J~ZD4Ek9u56ktrPJXS93YJ4qGZITEQupuOKHddrgtrVQf74I2lHj50mkRJPDXcmwnkAlCHj9HYjSBnVb8OnhoJc25aktrGAXjUDZcKUAXlvLorEGDIAClU8ElBJEjfTSI1yf5-T0oDWKmgrB9I-8PYPawIiAr1bgxbvGop8yZ-gE8uZ7G~74R34kcc7nlU~gZIU~A1VLoZ6ZuovxkZkNThTFm-2zgGuUVmFAaVx7kf5xgIIORjp0lE5fsPZfEqFgiOU-cWgn0OvhePFZCdcpGQui5BaYguk9XGf4Rdo~tcDsqyUbiIqTg8O3WduuPXBd3HFTNFWXD~CzRY5BQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S19</a></span>). The density of the variants in each variety was calculated to be 245.7 variants per 100â€‰kb on average. The number of heterozygous loci was 463,240.5 on average, ranging from 405,911 in Benikirari to 528,752 in Benitemari. Particularly, chromosomes 5 and 7 of Beniyutaka had fewer heterozygous loci.</p><p>The SNPs and indels were functionally annotated and classified into four categories: modifiers (88.2%) and moderate- (6.4%), low- (4.6%), and high- (0.7%) impact mutations (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx020/2/dsx020_supp_sweetcherrygenome-supplementarytables.xlsx?Expires=2147483647&Signature=TiRI9J~ZD4Ek9u56ktrPJXS93YJ4qGZITEQupuOKHddrgtrVQf74I2lHj50mkRJPDXcmwnkAlCHj9HYjSBnVb8OnhoJc25aktrGAXjUDZcKUAXlvLorEGDIAClU8ElBJEjfTSI1yf5-T0oDWKmgrB9I-8PYPawIiAr1bgxbvGop8yZ-gE8uZ7G~74R34kcc7nlU~gZIU~A1VLoZ6ZuovxkZkNThTFm-2zgGuUVmFAaVx7kf5xgIIORjp0lE5fsPZfEqFgiOU-cWgn0OvhePFZCdcpGQui5BaYguk9XGf4Rdo~tcDsqyUbiIqTg8O3WduuPXBd3HFTNFWXD~CzRY5BQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S20</a></span>). The most prevalent were variants in intergenic regions (modifiers, 65.8%) followed by intron variants (modifiers, 19.2%), missense variants (moderate-impact, 6.1%), and synonymous variants (low-impact, 3.9%). In the high-impact category, frameshift (0.3%) and stop-gained variants (0.2%) dominated.</p><p>In addition, CNV candidates were detected over the genomes of the six lines (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx020/2/dsx020_supp_sweetcherrygenome-supplementaryfigures.r1.pdf?Expires=2147483647&Signature=EpzskbJ8FdkxtePPGHzVUHR8wbz7PuhsMtIZp1mBYfuYpDnoU860lHAiOXaTih335VqggdpxNmN~DlHv9jCQjL67NgdKEe0mJf6VW~rFFNlKSOSwGF16QSU~ixooLa4MnFfPBuizWoRuWKHC5qHhWq9m7g0mIOI~a3cC4r-I38M7qqsg8TxxRMqONuq~9wlcV5s9xtt-0Ia6Fmkgv9vqIrt-f~ik0NUXpBMR6Br6pBv9tJQJvkf47kQLiRbDw4zZbNs27lnG2PNg44Zj75jgFNV8z2KYtBTZOOPF3qb1rjsE96U2cnvlArNo-eTe~6GEVJGGYM0KqyUXlBDoAbMxJA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6</a></span>). The average length of CNVs was 2.5â€‰kb and the longest was approximately 32â€‰kb in chromosome 7 of Beniyutaka, which included eight predicted genes (Pav_sc0000496.1_g220.1.br to Pav_sc0000496.1_g310.1.br). Numbers of the CNVs were ranging from 3,341 in Benishuho to 9,074 in Benikirari.</p>                            <h3 scrollto-destination=106356513 id="106356513" class="section-title" >3.7. DNA marker development</h3>
<p>CAPS and indel markers were developed in accordance with the sequence variants identified from whole-genome resequencing. Out of the 1,016,866 SNPs, 131,679 (12.9%) were located in the recognition sequence of 19 restriction enzymes. We also designed a total of 143,223 CAPS markers (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx020/2/dsx020_supp_sweetcherrygenome-supplementarytables.xlsx?Expires=2147483647&Signature=TiRI9J~ZD4Ek9u56ktrPJXS93YJ4qGZITEQupuOKHddrgtrVQf74I2lHj50mkRJPDXcmwnkAlCHj9HYjSBnVb8OnhoJc25aktrGAXjUDZcKUAXlvLorEGDIAClU8ElBJEjfTSI1yf5-T0oDWKmgrB9I-8PYPawIiAr1bgxbvGop8yZ-gE8uZ7G~74R34kcc7nlU~gZIU~A1VLoZ6ZuovxkZkNThTFm-2zgGuUVmFAaVx7kf5xgIIORjp0lE5fsPZfEqFgiOU-cWgn0OvhePFZCdcpGQui5BaYguk9XGf4Rdo~tcDsqyUbiIqTg8O3WduuPXBd3HFTNFWXD~CzRY5BQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S21</a></span>). In parallel, 151,468 indel markers for which primers were available were developed from the 162,402 indels (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx020/2/dsx020_supp_sweetcherrygenome-supplementarytables.xlsx?Expires=2147483647&Signature=TiRI9J~ZD4Ek9u56ktrPJXS93YJ4qGZITEQupuOKHddrgtrVQf74I2lHj50mkRJPDXcmwnkAlCHj9HYjSBnVb8OnhoJc25aktrGAXjUDZcKUAXlvLorEGDIAClU8ElBJEjfTSI1yf5-T0oDWKmgrB9I-8PYPawIiAr1bgxbvGop8yZ-gE8uZ7G~74R34kcc7nlU~gZIU~A1VLoZ6ZuovxkZkNThTFm-2zgGuUVmFAaVx7kf5xgIIORjp0lE5fsPZfEqFgiOU-cWgn0OvhePFZCdcpGQui5BaYguk9XGf4Rdo~tcDsqyUbiIqTg8O3WduuPXBd3HFTNFWXD~CzRY5BQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S22</a></span>). A total of 85,731 SSR motifs were detected in PAV_r1.0, including 40,924 (47.7%) di-, 13,473 (15.7%) tri-, 10,340 (12.1%) tetra-, 13,077 (15.3%) penta-, and 7,917 (9.2%) hexa-nucleotide repeat units. The most prevalent sequences in each repeat unit were AG (25,003), AAG (3,919), AAAT (5,551), AAAAT (4,056), and AAAAAT (1,497). We found that 29,539 SSRs (34.5%) were in gene regions and the remaining 56,192 (65.5%) were in intergenic regions. Out of the SSRs, primer pairs were successfully designed for 82,852 SSR motifs (96.6%), which were registered as SSR markers (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx020/2/dsx020_supp_sweetcherrygenome-supplementarytables.xlsx?Expires=2147483647&Signature=TiRI9J~ZD4Ek9u56ktrPJXS93YJ4qGZITEQupuOKHddrgtrVQf74I2lHj50mkRJPDXcmwnkAlCHj9HYjSBnVb8OnhoJc25aktrGAXjUDZcKUAXlvLorEGDIAClU8ElBJEjfTSI1yf5-T0oDWKmgrB9I-8PYPawIiAr1bgxbvGop8yZ-gE8uZ7G~74R34kcc7nlU~gZIU~A1VLoZ6ZuovxkZkNThTFm-2zgGuUVmFAaVx7kf5xgIIORjp0lE5fsPZfEqFgiOU-cWgn0OvhePFZCdcpGQui5BaYguk9XGf4Rdo~tcDsqyUbiIqTg8O3WduuPXBd3HFTNFWXD~CzRY5BQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S23</a></span>).</p>                            <h3 scrollto-destination=106356515 id="106356515" class="section-title" >3.8. Agronomically important genes in the sweet cherry genome</h3>
<p>We compared six agronomically important genes in peach<a reveal-id="dsx020-B70" data-open="dsx020-B70" class="link link-ref link-reveal xref-bibr"><sup>70</sup></a> with the predicted coding sequences. We found that two genes, Pav_sc0000464.1_g250.1.br and Pav_sc0000493.1_g020.1.br, were putative orthologues for ppa016711m (peach skin color) and ppa027093m (peach flesh color), respectively. In addition, Pav_sc0000103.1_g380.1.mk and Pav_sc0001587.1_g070.1.mk were orthologues of ppa010316m and ppa003772m, which associate with fruit hairiness and shape, respectively. Furthermore, Pav_sc0000024.1_g440.1.mk and Pav_sc0000600.1_g890.1.mk might correspond to ppa003772m (fruit adhesion and texture) and ppa006339m (non-acid fruit).</p><p>Self-incompatibility in sweet cherry is controlled by the <em>S</em>-locus, which carries the genes for <em>S-</em>RNase and <em>S</em>-locus F-box protein (SFB) as female and male determinants, respectively. Satonishiki possesses the <em>S3</em> and <em>S6</em> haplotypes. As expected, <em>S</em>-RNase (Pav_sc0004475.1_g130.1.mk) and <em>SFB</em> (Pav_sc0004475.1_g100.1.mk) of the <em>S3</em> haplotype were identified in a single contig sequence, Pav_sc0004475.1, and those of <em>S6</em> (<em>S</em>-RNase: Pav_sc0006359.1_g040.1.mk; and <em>SFB</em>: Pav_sc0006359.1_g030.1.mk) were found in another contig, Pav_sc0006359.1. The physical distances between <em>S</em>-RNase and <em>SFB</em> were 7.4â€‰kb and 1.5â€‰kb in the <em>S3</em> and <em>S6</em> haplotypes, respectively. Neither the orientation nor the order of the genes predicted in the contigs was conserved, suggesting that the genomic structures of the <em>S</em>-loci were divergent. Subsequently, we investigated the depth of coverage of the resequencing across the six lines (<a reveal-id="dsx020-F2" data-open="dsx020-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>). The reads were evenly mapped across the contigs when the lines had an identical <em>S</em>-haplotype to Satonishiki. Otherwise, the coverage was partial, suggesting that the genome sequences were divergent across different <em>S</em>-haplotypes (e.g. <em>S1</em>, <em>S4</em>, and <em>S4â€™</em>) as well as <em>S3</em> and <em>S6</em>. </p>                        <a id="106356518" scrollto-destination="106356518"></a>
<div id="dsx020-F2" data-id="dsx020-F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx020/2/m_dsx020f2.png?Expires=2147483647&Signature=ODtYpKmA-9iAQS8ty2wRoczp3Il6dgR7m3XQAsPagCGi8JONW-tt7NqyxYe8fCBpDOgxkTCzPihV0gm8lYaQ5FP3ZdB5Q0Y1e6RHhdQhDyO82sD5wdWRb0aceqLgRqisEEyAQq8cwQgMoCZa7KneCnDdMl8P7zlEm9XbdyaCz6Ey-oxCKjQNPa9nWODvkWLJUN6pgf4lejzabpiSS3x9L~oem7H7GRKm7NURSTxipK2d1HC4KsOg-An7lRoTBK79SmcaMjDdNLK4DkmFJhESPMfMFph9s4OQcu~mzV0ITcjlAcNC9jtDcX-He2PQ-GCqgb755Gd6F2DKz0mbBxrZBg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genomic structures of the S3 and S6 loci in Satonishiki and the mapping rate of sequence reads from the six cultivated lines. " path-from-xml="dsx020f2.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106356518/dsx020f2.png" path-from-xml="dsx020f2.png" target="_blank">View large</a><a section="106356518" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx020/2/dsx020f2.png?Expires=2147483647&Signature=sfOwRGG22ApAn~N~4NPzuqTsKMEutxSxP5i4FG9AXmFVz5i~p4L9lQm0ycSW~cY-3cg-yK-4IePtw2SPXdb64WeLmSaqkXhcLZfyGxQloBIix9WrphKswobFsrJhXDVPbDaJMVoCWOW9Q4LU8LfS1sfoc~kilQfZqQ79gb5iEHK74u5K95C~RWR644iLD6QQt~2vcfzRfjpgDaCWvGnzTVUNM5fZhiL8xXYYHkg~7YuAJ2jiwxjIlXT8akNGl9Bh7DOTUnyKcDDkfgBAdCCnxJTlewDPjIMRQlMavr-rBx1hC6ZkmoD-3ZglZNFZJP7vhTWe5ehYBBAgbeLgyR2t5g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106356518&ar=3835854&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx020f2.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genomic structures of the <em>S3</em> and <em>S6</em> loci in Satonishiki and the mapping rate of sequence reads from the six cultivated lines. </p></div></div><div content-id="dsx020-F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx020/2/m_dsx020f2.png?Expires=2147483647&Signature=ODtYpKmA-9iAQS8ty2wRoczp3Il6dgR7m3XQAsPagCGi8JONW-tt7NqyxYe8fCBpDOgxkTCzPihV0gm8lYaQ5FP3ZdB5Q0Y1e6RHhdQhDyO82sD5wdWRb0aceqLgRqisEEyAQq8cwQgMoCZa7KneCnDdMl8P7zlEm9XbdyaCz6Ey-oxCKjQNPa9nWODvkWLJUN6pgf4lejzabpiSS3x9L~oem7H7GRKm7NURSTxipK2d1HC4KsOg-An7lRoTBK79SmcaMjDdNLK4DkmFJhESPMfMFph9s4OQcu~mzV0ITcjlAcNC9jtDcX-He2PQ-GCqgb755Gd6F2DKz0mbBxrZBg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genomic structures of the S3 and S6 loci in Satonishiki and the mapping rate of sequence reads from the six cultivated lines. " path-from-xml="dsx020f2.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106356518/dsx020f2.png" path-from-xml="dsx020f2.png" target="_blank">View large</a><a section="106356518" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx020/2/dsx020f2.png?Expires=2147483647&Signature=sfOwRGG22ApAn~N~4NPzuqTsKMEutxSxP5i4FG9AXmFVz5i~p4L9lQm0ycSW~cY-3cg-yK-4IePtw2SPXdb64WeLmSaqkXhcLZfyGxQloBIix9WrphKswobFsrJhXDVPbDaJMVoCWOW9Q4LU8LfS1sfoc~kilQfZqQ79gb5iEHK74u5K95C~RWR644iLD6QQt~2vcfzRfjpgDaCWvGnzTVUNM5fZhiL8xXYYHkg~7YuAJ2jiwxjIlXT8akNGl9Bh7DOTUnyKcDDkfgBAdCCnxJTlewDPjIMRQlMavr-rBx1hC6ZkmoD-3ZglZNFZJP7vhTWe5ehYBBAgbeLgyR2t5g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106356518&ar=3835854&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx020f2.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genomic structures of the <em>S3</em> and <em>S6</em> loci in Satonishiki and the mapping rate of sequence reads from the six cultivated lines. </p></div></div>                            <h2 scrollto-destination=106356519 id="106356519" class="section-title" >4. Discussion</h2>
<p>Here, we report the first draft genome sequence of sweet cherry. The sequence data were used to establish genetic linkage maps with ddRAD-Seq technology, to enable whole-genome resequencing analysis to reveal the genetic diversity of cultivated lines, and to develop genome-wide DNA markers including SNPs, indels, and SSRs. In addition, agronomically important genes were identified by comparative analysis with Rosaceae relatives. This information will further genetic and genomic studies as well as assist in sweet cherry breeding programs.</p><p>The size of the assembled genome, PAV_r1.0, was 272.4â€‰Mb, which covered 77.8% of the estimated genome size of âˆ¼350â€‰Mb (<a reveal-id="dsx020-T1" data-open="dsx020-T1" class="link link-reveal link-table xref-fig">Table 1</a>). The remaining 97â€‰Mb of sequence was eliminated from the final assembly due to short contigs (&lt;1,000â€‰bp) enriched in repetitive sequences. The genome sizes of other diploid species in Rosaceae are estimated to be 265â€‰Mb in peach,<a reveal-id="dsx020-B2" data-open="dsx020-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> 280â€‰Mb in Japanese apricot,<a reveal-id="dsx020-B5" data-open="dsx020-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> and 240â€‰Mb in strawberry,<a reveal-id="dsx020-B6" data-open="dsx020-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a> all of which are approximately 100â€‰Mb shorter than that of sweet cherry. On the other hand, the proportion of repetitive sequences in PAV_r1.0 was almost equal to that of <em>Pr. persica</em>, <em>Pr. mume</em>, and <em>F. vesca</em>: approximately 40%. Therefore, we considered that a subset of the sweet cherry genome, PAV_r1.0, might correspond to the genomes of <em>Pr. persica</em>, <em>Pr. mume</em>, and <em>F. vesca</em>. Indeed, PAV_r1.0 includedâ€‰&gt;96% of BUSCO genes (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx020/2/dsx020_supp_sweetcherrygenome-supplementarytables.xlsx?Expires=2147483647&Signature=TiRI9J~ZD4Ek9u56ktrPJXS93YJ4qGZITEQupuOKHddrgtrVQf74I2lHj50mkRJPDXcmwnkAlCHj9HYjSBnVb8OnhoJc25aktrGAXjUDZcKUAXlvLorEGDIAClU8ElBJEjfTSI1yf5-T0oDWKmgrB9I-8PYPawIiAr1bgxbvGop8yZ-gE8uZ7G~74R34kcc7nlU~gZIU~A1VLoZ6ZuovxkZkNThTFm-2zgGuUVmFAaVx7kf5xgIIORjp0lE5fsPZfEqFgiOU-cWgn0OvhePFZCdcpGQui5BaYguk9XGf4Rdo~tcDsqyUbiIqTg8O3WduuPXBd3HFTNFWXD~CzRY5BQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>), suggesting that PAV_r1.0 sufficiently covered the gene space of the sweet cherry genome. The repeat-rich 97â€‰Mb of sequences eliminated from the assembly might cause genome expansion in sweet cherry, making it larger than the other diploid species. <div content-id="dsx020-T1" class="table-modal"><div class="table-wrap"><div id="dsx020-T1" data-id="dsx020-T1" class="table-wrap-title"><div class="label -label">Table 1</div><div class="caption"><p>Assembly statistics of the sweet cherry genome</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Â </th><th>PAV_r1.0Â </th></tr></thead><tbody><tr><td>Estimated genome size (bp)Â </td><td>352,883,670Â </td></tr><tr><td># of scaffoldsÂ </td><td>10,148Â </td></tr><tr><td>Size of scaffolds (bp)Â </td><td>272,361,615Â </td></tr><tr><td>Scaffold N50 (bp)Â </td><td>219,566Â </td></tr><tr><td>Longest scaffold (bp)Â </td><td>1,460,269Â </td></tr><tr><td>GC (%)Â </td><td>37.7Â </td></tr><tr><td># of genesÂ </td><td>43,673Â </td></tr><tr><td>Mean size of genes (bp)Â </td><td>1,097Â </td></tr><tr><td>Repeat (%)Â </td><td>43.8Â </td></tr></tbody></table></div><div content-id="dsx020-T1" class="table-modal"><table><thead name="thead" align=""><tr><th>Â </th><th>PAV_r1.0Â </th></tr></thead><tbody><tr><td>Estimated genome size (bp)Â </td><td>352,883,670Â </td></tr><tr><td># of scaffoldsÂ </td><td>10,148Â </td></tr><tr><td>Size of scaffolds (bp)Â </td><td>272,361,615Â </td></tr><tr><td>Scaffold N50 (bp)Â </td><td>219,566Â </td></tr><tr><td>Longest scaffold (bp)Â </td><td>1,460,269Â </td></tr><tr><td>GC (%)Â </td><td>37.7Â </td></tr><tr><td># of genesÂ </td><td>43,673Â </td></tr><tr><td>Mean size of genes (bp)Â </td><td>1,097Â </td></tr><tr><td>Repeat (%)Â </td><td>43.8Â </td></tr></tbody></table></div></div></div><div class="table-wrap"><div id="dsx020-T1" data-id="dsx020-T1" class="table-wrap-title"><div class="label -label">Table 1</div><div class="caption"><p>Assembly statistics of the sweet cherry genome</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Â </th><th>PAV_r1.0Â </th></tr></thead><tbody><tr><td>Estimated genome size (bp)Â </td><td>352,883,670Â </td></tr><tr><td># of scaffoldsÂ </td><td>10,148Â </td></tr><tr><td>Size of scaffolds (bp)Â </td><td>272,361,615Â </td></tr><tr><td>Scaffold N50 (bp)Â </td><td>219,566Â </td></tr><tr><td>Longest scaffold (bp)Â </td><td>1,460,269Â </td></tr><tr><td>GC (%)Â </td><td>37.7Â </td></tr><tr><td># of genesÂ </td><td>43,673Â </td></tr><tr><td>Mean size of genes (bp)Â </td><td>1,097Â </td></tr><tr><td>Repeat (%)Â </td><td>43.8Â </td></tr></tbody></table></div><div content-id="dsx020-T1" class="table-modal"><table><thead name="thead" align=""><tr><th>Â </th><th>PAV_r1.0Â </th></tr></thead><tbody><tr><td>Estimated genome size (bp)Â </td><td>352,883,670Â </td></tr><tr><td># of scaffoldsÂ </td><td>10,148Â </td></tr><tr><td>Size of scaffolds (bp)Â </td><td>272,361,615Â </td></tr><tr><td>Scaffold N50 (bp)Â </td><td>219,566Â </td></tr><tr><td>Longest scaffold (bp)Â </td><td>1,460,269Â </td></tr><tr><td>GC (%)Â </td><td>37.7Â </td></tr><tr><td># of genesÂ </td><td>43,673Â </td></tr><tr><td>Mean size of genes (bp)Â </td><td>1,097Â </td></tr><tr><td>Repeat (%)Â </td><td>43.8Â </td></tr></tbody></table></div></div><div content-id="dsx020-T2" class="table-modal"><div class="table-wrap"><div id="dsx020-T2" data-id="dsx020-T2" class="table-wrap-title"><div class="label -label">Table 2</div><div class="caption"><p>Statistics of the pseudomolecules for the sweet cherry genome</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>PseudomoleculeÂ </th><th>No. of assigned scaffoldsÂ </th><th>%<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a>Â </th><th>Total size of assigned scaffolds (bp)Â </th><th>%<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a>Â </th><th>No. of predicted genesÂ </th><th>%<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a>Â </th></tr></thead><tbody><tr><td>PAV_r1.0chr1Â </td><td>161Â </td><td>1.6Â </td><td>41,632,855Â </td><td>15.3Â </td><td>6,737Â </td><td>15.4Â </td></tr><tr><td>PAV_r1.0chr2Â </td><td>111Â </td><td>1.1Â </td><td>24,154,475Â </td><td>8.9Â </td><td>3,949Â </td><td>9.0Â </td></tr><tr><td>PAV_r1.0chr3Â </td><td>86Â </td><td>0.8Â </td><td>21,763,589Â </td><td>8.0Â </td><td>3,588Â </td><td>8.2Â </td></tr><tr><td>PAV_r1.0chr4Â </td><td>128Â </td><td>1.3Â </td><td>26,009,932Â </td><td>9.5Â </td><td>4,126Â </td><td>9.4Â </td></tr><tr><td>PAV_r1.0chr5Â </td><td>57Â </td><td>0.6Â </td><td>16,460,956Â </td><td>6.0Â </td><td>2,822Â </td><td>6.5Â </td></tr><tr><td>PAV_r1.0chr6Â </td><td>159Â </td><td>1.6Â </td><td>23,031,171Â </td><td>8.5Â </td><td>3,838Â </td><td>8.8Â </td></tr><tr><td>PAV_r1.0chr7Â </td><td>85Â </td><td>0.8Â </td><td>19,052,082Â </td><td>7.0Â </td><td>3,180Â </td><td>7.3Â </td></tr><tr><td>PAV_r1.0chr8Â </td><td>118Â </td><td>1.2Â </td><td>19,599,356Â </td><td>7.2Â </td><td>3,212Â </td><td>7.4Â </td></tr><tr><td>TotalÂ </td><td>905Â </td><td>8.9Â </td><td>191,704,416Â </td><td>70.4Â </td><td>31,452Â </td><td>72.0Â </td></tr></tbody></table></div><div content-id="dsx020-T2" class="table-modal"><table><thead name="thead" align=""><tr><th>PseudomoleculeÂ </th><th>No. of assigned scaffoldsÂ </th><th>%<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a>Â </th><th>Total size of assigned scaffolds (bp)Â </th><th>%<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a>Â </th><th>No. of predicted genesÂ </th><th>%<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a>Â </th></tr></thead><tbody><tr><td>PAV_r1.0chr1Â </td><td>161Â </td><td>1.6Â </td><td>41,632,855Â </td><td>15.3Â </td><td>6,737Â </td><td>15.4Â </td></tr><tr><td>PAV_r1.0chr2Â </td><td>111Â </td><td>1.1Â </td><td>24,154,475Â </td><td>8.9Â </td><td>3,949Â </td><td>9.0Â </td></tr><tr><td>PAV_r1.0chr3Â </td><td>86Â </td><td>0.8Â </td><td>21,763,589Â </td><td>8.0Â </td><td>3,588Â </td><td>8.2Â </td></tr><tr><td>PAV_r1.0chr4Â </td><td>128Â </td><td>1.3Â </td><td>26,009,932Â </td><td>9.5Â </td><td>4,126Â </td><td>9.4Â </td></tr><tr><td>PAV_r1.0chr5Â </td><td>57Â </td><td>0.6Â </td><td>16,460,956Â </td><td>6.0Â </td><td>2,822Â </td><td>6.5Â </td></tr><tr><td>PAV_r1.0chr6Â </td><td>159Â </td><td>1.6Â </td><td>23,031,171Â </td><td>8.5Â </td><td>3,838Â </td><td>8.8Â </td></tr><tr><td>PAV_r1.0chr7Â </td><td>85Â </td><td>0.8Â </td><td>19,052,082Â </td><td>7.0Â </td><td>3,180Â </td><td>7.3Â </td></tr><tr><td>PAV_r1.0chr8Â </td><td>118Â </td><td>1.2Â </td><td>19,599,356Â </td><td>7.2Â </td><td>3,212Â </td><td>7.4Â </td></tr><tr><td>TotalÂ </td><td>905Â </td><td>8.9Â </td><td>191,704,416Â </td><td>70.4Â </td><td>31,452Â </td><td>72.0Â </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p><sup>a</sup>Percentage of PAV_r1.0.</p></div></div></div></div></div><div class="table-wrap"><div id="dsx020-T2" data-id="dsx020-T2" class="table-wrap-title"><div class="label -label">Table 2</div><div class="caption"><p>Statistics of the pseudomolecules for the sweet cherry genome</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>PseudomoleculeÂ </th><th>No. of assigned scaffoldsÂ </th><th>%<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a>Â </th><th>Total size of assigned scaffolds (bp)Â </th><th>%<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a>Â </th><th>No. of predicted genesÂ </th><th>%<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a>Â </th></tr></thead><tbody><tr><td>PAV_r1.0chr1Â </td><td>161Â </td><td>1.6Â </td><td>41,632,855Â </td><td>15.3Â </td><td>6,737Â </td><td>15.4Â </td></tr><tr><td>PAV_r1.0chr2Â </td><td>111Â </td><td>1.1Â </td><td>24,154,475Â </td><td>8.9Â </td><td>3,949Â </td><td>9.0Â </td></tr><tr><td>PAV_r1.0chr3Â </td><td>86Â </td><td>0.8Â </td><td>21,763,589Â </td><td>8.0Â </td><td>3,588Â </td><td>8.2Â </td></tr><tr><td>PAV_r1.0chr4Â </td><td>128Â </td><td>1.3Â </td><td>26,009,932Â </td><td>9.5Â </td><td>4,126Â </td><td>9.4Â </td></tr><tr><td>PAV_r1.0chr5Â </td><td>57Â </td><td>0.6Â </td><td>16,460,956Â </td><td>6.0Â </td><td>2,822Â </td><td>6.5Â </td></tr><tr><td>PAV_r1.0chr6Â </td><td>159Â </td><td>1.6Â </td><td>23,031,171Â </td><td>8.5Â </td><td>3,838Â </td><td>8.8Â </td></tr><tr><td>PAV_r1.0chr7Â </td><td>85Â </td><td>0.8Â </td><td>19,052,082Â </td><td>7.0Â </td><td>3,180Â </td><td>7.3Â </td></tr><tr><td>PAV_r1.0chr8Â </td><td>118Â </td><td>1.2Â </td><td>19,599,356Â </td><td>7.2Â </td><td>3,212Â </td><td>7.4Â </td></tr><tr><td>TotalÂ </td><td>905Â </td><td>8.9Â </td><td>191,704,416Â </td><td>70.4Â </td><td>31,452Â </td><td>72.0Â </td></tr></tbody></table></div><div content-id="dsx020-T2" class="table-modal"><table><thead name="thead" align=""><tr><th>PseudomoleculeÂ </th><th>No. of assigned scaffoldsÂ </th><th>%<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a>Â </th><th>Total size of assigned scaffolds (bp)Â </th><th>%<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a>Â </th><th>No. of predicted genesÂ </th><th>%<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a>Â </th></tr></thead><tbody><tr><td>PAV_r1.0chr1Â </td><td>161Â </td><td>1.6Â </td><td>41,632,855Â </td><td>15.3Â </td><td>6,737Â </td><td>15.4Â </td></tr><tr><td>PAV_r1.0chr2Â </td><td>111Â </td><td>1.1Â </td><td>24,154,475Â </td><td>8.9Â </td><td>3,949Â </td><td>9.0Â </td></tr><tr><td>PAV_r1.0chr3Â </td><td>86Â </td><td>0.8Â </td><td>21,763,589Â </td><td>8.0Â </td><td>3,588Â </td><td>8.2Â </td></tr><tr><td>PAV_r1.0chr4Â </td><td>128Â </td><td>1.3Â </td><td>26,009,932Â </td><td>9.5Â </td><td>4,126Â </td><td>9.4Â </td></tr><tr><td>PAV_r1.0chr5Â </td><td>57Â </td><td>0.6Â </td><td>16,460,956Â </td><td>6.0Â </td><td>2,822Â </td><td>6.5Â </td></tr><tr><td>PAV_r1.0chr6Â </td><td>159Â </td><td>1.6Â </td><td>23,031,171Â </td><td>8.5Â </td><td>3,838Â </td><td>8.8Â </td></tr><tr><td>PAV_r1.0chr7Â </td><td>85Â </td><td>0.8Â </td><td>19,052,082Â </td><td>7.0Â </td><td>3,180Â </td><td>7.3Â </td></tr><tr><td>PAV_r1.0chr8Â </td><td>118Â </td><td>1.2Â </td><td>19,599,356Â </td><td>7.2Â </td><td>3,212Â </td><td>7.4Â </td></tr><tr><td>TotalÂ </td><td>905Â </td><td>8.9Â </td><td>191,704,416Â </td><td>70.4Â </td><td>31,452Â </td><td>72.0Â </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p><sup>a</sup>Percentage of PAV_r1.0.</p></div></div></div></div></p><p>Comparative analysis of the consensus genetic map (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx020/2/dsx020_supp_sweetcherrygenome-supplementaryfigures.r1.pdf?Expires=2147483647&amp;Signature=EpzskbJ8FdkxtePPGHzVUHR8wbz7PuhsMtIZp1mBYfuYpDnoU860lHAiOXaTih335VqggdpxNmN~DlHv9jCQjL67NgdKEe0mJf6VW~rFFNlKSOSwGF16QSU~ixooLa4MnFfPBuizWoRuWKHC5qHhWq9m7g0mIOI~a3cC4r-I38M7qqsg8TxxRMqONuq~9wlcV5s9xtt-0Ia6Fmkgv9vqIrt-f~ik0NUXpBMR6Br6pBv9tJQJvkf47kQLiRbDw4zZbNs27lnG2PNg44Zj75jgFNV8z2KYtBTZOOPF3qb1rjsE96U2cnvlArNo-eTe~6GEVJGGYM0KqyUXlBDoAbMxJA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>, <span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx020/2/dsx020_supp_sweetcherrygenome-supplementarytables.xlsx?Expires=2147483647&amp;Signature=TiRI9J~ZD4Ek9u56ktrPJXS93YJ4qGZITEQupuOKHddrgtrVQf74I2lHj50mkRJPDXcmwnkAlCHj9HYjSBnVb8OnhoJc25aktrGAXjUDZcKUAXlvLorEGDIAClU8ElBJEjfTSI1yf5-T0oDWKmgrB9I-8PYPawIiAr1bgxbvGop8yZ-gE8uZ7G~74R34kcc7nlU~gZIU~A1VLoZ6ZuovxkZkNThTFm-2zgGuUVmFAaVx7kf5xgIIORjp0lE5fsPZfEqFgiOU-cWgn0OvhePFZCdcpGQui5BaYguk9XGf4Rdo~tcDsqyUbiIqTg8O3WduuPXBd3HFTNFWXD~CzRY5BQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S16</a></span>) found high chromosome-level synteny between sweet cherry and peach (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx020/2/dsx020_supp_sweetcherrygenome-supplementaryfigures.r1.pdf?Expires=2147483647&amp;Signature=EpzskbJ8FdkxtePPGHzVUHR8wbz7PuhsMtIZp1mBYfuYpDnoU860lHAiOXaTih335VqggdpxNmN~DlHv9jCQjL67NgdKEe0mJf6VW~rFFNlKSOSwGF16QSU~ixooLa4MnFfPBuizWoRuWKHC5qHhWq9m7g0mIOI~a3cC4r-I38M7qqsg8TxxRMqONuq~9wlcV5s9xtt-0Ia6Fmkgv9vqIrt-f~ik0NUXpBMR6Br6pBv9tJQJvkf47kQLiRbDw4zZbNs27lnG2PNg44Zj75jgFNV8z2KYtBTZOOPF3qb1rjsE96U2cnvlArNo-eTe~6GEVJGGYM0KqyUXlBDoAbMxJA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span>), supporting the results of previous studies.<a reveal-id="dsx020-B9" data-open="dsx020-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> Therefore, it would be possible to apply genetic knowledge from peach to sweet cherry, as proposed by Dirlewanger et al.<a reveal-id="dsx020-B10" data-open="dsx020-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> As shown above, orthologues of agronomically important genes identified by a genome-wide association study (GWAS) in peach<a reveal-id="dsx020-B70" data-open="dsx020-B70" class="link link-ref link-reveal xref-bibr"><sup>70</sup></a> were nominated in sweet cherry using sequence similarity. In addition, <em>in silico</em> mapping of QTLs and genes, which have been curated and summarized in GDR<a reveal-id="dsx020-B47" data-open="dsx020-B47" class="link link-ref link-reveal xref-bibr"><sup>47</sup></a> (<a class="link link-uri" href="http://www.rosaceae.org" target="_blank">http://www.rosaceae.org</a> (8 May 2017, date last accessed)) and PGDBj<a reveal-id="dsx020-B71" data-open="dsx020-B71" class="link link-ref link-reveal xref-bibr"><sup>71</sup></a> (<a class="link link-uri" href="http://pgdbj.jp" target="_blank">http://pgdbj.jp</a> (8 May 2017, date last accessed)), among other databases, could identify useful gene candidates for sweet cherry breeding programs. Moreover, it might be possible to identify attractive genes from the sweet cherry specific cluster (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx020/2/dsx020_supp_sweetcherrygenome-supplementaryfigures.r1.pdf?Expires=2147483647&amp;Signature=EpzskbJ8FdkxtePPGHzVUHR8wbz7PuhsMtIZp1mBYfuYpDnoU860lHAiOXaTih335VqggdpxNmN~DlHv9jCQjL67NgdKEe0mJf6VW~rFFNlKSOSwGF16QSU~ixooLa4MnFfPBuizWoRuWKHC5qHhWq9m7g0mIOI~a3cC4r-I38M7qqsg8TxxRMqONuq~9wlcV5s9xtt-0Ia6Fmkgv9vqIrt-f~ik0NUXpBMR6Br6pBv9tJQJvkf47kQLiRbDw4zZbNs27lnG2PNg44Zj75jgFNV8z2KYtBTZOOPF3qb1rjsE96U2cnvlArNo-eTe~6GEVJGGYM0KqyUXlBDoAbMxJA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>), even though most of genes had no informative functional annotations due to <em>ab initio</em> prediction. Whole-genome resequencing is one of the most effective methods for allele mining. In the future, genotype and sequence variation could be assigned to phenotypic variation using QTL studies and GWAS.</p><p>In general, whole-genome resequencing of wide-spread cultivated lines and their founders would reveal historical domestication and breeding processes. However, in this study, we targeted six cultivated lines, all of which were bred in Yamagata, Japan, and registered within 40â€‰years. It would be difficult to obtain evidence of historical events, but it is possible to gain insight into new breeding strategies. Because breeding programs for sweet cherry, as well as other fruiting trees, require time and space, it is difficult to perform high-throughput breeding. Therefore, genomics-assisted breeding<a reveal-id="dsx020-B72" data-open="dsx020-B72" class="link link-ref link-reveal xref-bibr"><sup>72</sup></a> and new plant-breeding techniques<a reveal-id="dsx020-B73" data-open="dsx020-B73" class="link link-ref link-reveal xref-bibr"><sup>73</sup></a> are more effective approaches. For example, in pear, genome-wide information about SNPs and phenotypes enables the prediction of trait segregation in a progeny population, which assists in choosing a good parental combination.<a reveal-id="dsx020-B74" data-open="dsx020-B74" class="link link-ref link-reveal xref-bibr"><sup>74</sup></a> Moreover, in apple, accelerating generation advancement has been accomplished through a plant virus vector that carries a promoter for <em>Arabidopsis FLOWERING LOCUS T</em> and a silencer for apple <em>TERMINAL FLOWER 1</em>.<a reveal-id="dsx020-B75" data-open="dsx020-B75" class="link link-ref link-reveal xref-bibr"><sup>75</sup></a> These technologies developed in Rosaceae, together with a genomic selection strategy,<a reveal-id="dsx020-B72" data-open="dsx020-B72" class="link link-ref link-reveal xref-bibr"><sup>72</sup></a> would make it possible to quickly produce excellent lines, whose phenotypes (e.g. fruit size, taste, and shelf-life) would exceed those of the current leading varieties. The genomic information obtained from this study would accelerate genetic analysis and breeding programs in sweet cherry as well as other fruiting trees.</p>                            <h2 scrollto-destination=106356524 id="106356524" class="section-title" >5. Availability</h2>
<p>The genome assembly data (scaffold and pseudomolecule sequences), annotations, gene models, genetic maps, and DNA polymorphism are available at DBcherry (<a class="link link-uri" href="http://cherry.kazusa.or.jp/" target="_blank">http://cherry.kazusa.or.jp/</a> (8 May 2017, date last accessed)).</p>                            <h2 scrollto-destination=106356526 id="106356526" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank K. Kawashima, Y. Kishida, M. Kohara, C. Minami, S. Nakayama, S. Sasamoto, and A. Watanabe (Kazusa DNA Research Institute) for their technical assistance. This work was supported by the Kazusa DNA Research Institute Foundation.</p>                            <h3 scrollto-destination=106356528 id="106356528" class="section-title" >Accession numbers</h3>
<p>The sequence reads are available from the DDBJ Sequence Read Archive (DRA) under the accession numbers DRA004760â€“DRA004765 and DRA004768â€“DRA004772. The BioProject accession number of the study is PRJDB4877. The WGS accession numbers of assembled scaffold sequences are BDGV01000001â€“BDGV01010148 (10,148 entries).</p>                            <h3 scrollto-destination=106356530 id="106356530" class="section-title" >Conflict of interest</h3>
<p>None declared.</p>                            <h3 scrollto-destination=106356532 id="106356532" class="section-title" >Supplementary data</h3>
<p><span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx020/2/dsx020_supp_sweetcherrygenome-supplementaryinformation.r1.pdf?Expires=2147483647&Signature=XFws~d4jZjNfBNsaOnnmKXsMcz4jkFjIH5zugTNI6sFgBrlDa30GJcFfXXLenHUiCDiGeYhcZ~wDXdXLwAANIlAFe6RDnfyyv6Fs7g4e633wcgVjayMVcPKVfn1J6AGX32HS05qOUga1J5SIKv0vMrUW~WBxwP82E8sMZfXVwKPVwdqxd35Xs-TgnMJrl~xZHK-4RkFO2MK3eqPyus8qRiUnDgv-qjQklxyGKzpW68wX4v1kuaZKrn7~wYECKVnx-6Dzm9NoV3YqBUhpYJBvcrfdRgvopij2lPPSzFmqnixrcvzTGeKjVPr3xA1yRbh4R~tV3gEw4viEQ7OnJm8pWg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary data</a></span> are available at <em>DNARES</em> Online.</p>                            <h2 scrollto-destination=106356534 id="106356534" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsx020-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx020-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Velasco</div>
               <div class="given-names">R.</div>
            </div>, <div class="name string-name"><div class="surname">Zharkikh</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Affourtit</div> <div class="given-names">J.</div></div></div>, et al. Â <div class="year">2010</div>, 
<div class="article-title">The genome of the domesticated apple (<em>Malus</em> Ã— <em>domestica</em> Borkh.)</div>, <div class="source">Nat. Genet</div>Â ., <div class="volume">42</div>, <div class="fpage">833</div>â€“<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20of%20the%20domesticated%20apple%20(Malus%20%C3%97%20domestica%20Borkh.)&amp;author=R.%20Velasco&amp;author=A.%20Zharkikh&amp;author=J.%20Affourtit&amp;publication_year=2010&amp;journal=Nat.%20Genet&amp;volume=42&amp;pages=833-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.654" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.654" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.654"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20802477" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx020-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx020-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation">
          <div class="collab">International, Peach Genome Initiative</div>. <div class="year">2013</div>, 
<div class="article-title">The high-quality draft genome of peach (<em>Prunus persica</em>) identifies unique patterns of genetic diversity, domestication and genome evolution</div>, <div class="source">Nat. Genet</div>Â ., <div class="volume">45</div>, <div class="fpage">487</div>â€“<div class="lpage">94</div>.<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.2586" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.2586" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.2586"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23525075" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx020-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx020-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Wu</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Shi</div> <div class="given-names">Z.</div></div></div>, et al. Â <div class="year">2013</div>, 
<div class="article-title">The genome of the pear (<em>Pyrus bretschneideri</em> Rehd.)</div>, <div class="source">Genome Res</div>Â ., <div class="volume">23</div>, <div class="fpage">396</div>â€“<div class="lpage">408</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20of%20the%20pear%20(Pyrus%20bretschneideri%20Rehd.)&amp;author=J.%20Wu&amp;author=Z.%20Wang&amp;author=Z.%20Shi&amp;publication_year=2013&amp;journal=Genome%20Res&amp;volume=23&amp;pages=396-408" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.144311.112" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.144311.112" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.144311.112"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23149293" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx020-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx020-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">ChagnÃ©</div>
               <div class="given-names">D.</div>
            </div>, <div class="name string-name"><div class="surname">Crowhurst</div> <div class="given-names">R.N.</div></div>, <div class="name string-name"><div class="surname">Pindo</div> <div class="given-names">M.</div></div></div>, et al. Â <div class="year">2014</div>, 
<div class="article-title">The draft genome sequence of European pear (<em>Pyrus communis</em> L. â€²Bartlettâ€²)</div>, <div class="source">PLoS One</div>Â , <div class="volume">9</div>, <div class="fpage">e92644</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20draft%20genome%20sequence%20of%20European%20pear%20(Pyrus%20communis%20L.%20%E2%80%B2Bartlett%E2%80%B2)&amp;author=D.%20Chagn%C3%A9&amp;author=R.N.%20Crowhurst&amp;author=M.%20Pindo&amp;publication_year=2014&amp;journal=PLoS%20One&amp;volume=9&amp;pages=e92644" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0092644" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0092644" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0092644"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24699266" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx020-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx020-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Zhang</div>
               <div class="given-names">Q.</div>
            </div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Sun</div> <div class="given-names">L.</div></div></div>, et al. Â <div class="year">2012</div>, 
<div class="article-title">The genome of <em>Prunus mume</em>,</div><div class="source">Nat. Commun</div>Â ., <div class="volume">3</div>, <div class="fpage">1318</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20of%20Prunus%20mume%2C&amp;author=Q.%20Zhang&amp;author=W.%20Chen&amp;author=L.%20Sun&amp;publication_year=2012&amp;journal=Nat.%20Commun&amp;volume=3&amp;pages=1318" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ncomms2290" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fncomms2290" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fncomms2290"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23271652" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx020-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx020-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Shulaev</div>
               <div class="given-names">V.</div>
            </div>, <div class="name string-name"><div class="surname">Sargent</div> <div class="given-names">D.J.</div></div>, <div class="name string-name"><div class="surname">Crowhurst</div> <div class="given-names">R.N.</div></div></div>, et al. Â <div class="year">2011</div>, 
<div class="article-title">The genome of woodland strawberry (<em>Fragaria vesca</em>)</div>, <div class="source">Nat. Genet</div>Â ., <div class="volume">43</div>, <div class="fpage">109</div>â€“<div class="lpage">16</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20of%20woodland%20strawberry%20(Fragaria%20vesca)&amp;author=V.%20Shulaev&amp;author=D.J.%20Sargent&amp;author=R.N.%20Crowhurst&amp;publication_year=2011&amp;journal=Nat.%20Genet&amp;volume=43&amp;pages=109-16" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.740" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.740" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.740"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21186353" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx020-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx020-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Hirakawa</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Shirasawa</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Kosugi</div> <div class="given-names">S.</div></div></div>, et al. Â <div class="year">2014</div>, 
<div class="article-title">Dissection of the octoploid strawberry genome by deep sequencing of the genomes of <em>Fragaria</em> species,</div><div class="source">DNA Res</div>Â ., <div class="volume">21</div>, <div class="fpage">169</div>â€“<div class="lpage">81</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dissection%20of%20the%20octoploid%20strawberry%20genome%20by%20deep%20sequencing%20of%20the%20genomes%20of%20Fragaria%20species%2C&amp;author=H.%20Hirakawa&amp;author=K.%20Shirasawa&amp;author=S.%20Kosugi&amp;publication_year=2014&amp;journal=DNA%20Res&amp;volume=21&amp;pages=169-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dst049" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdst049" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdst049"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24282021" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx020-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx020-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Michael</div>
               <div class="given-names">T.P.</div>
            </div>, <div class="name string-name"><div class="surname">VanBuren</div> <div class="given-names">R.</div></div></div>
          <div class="year">2015</div>, 
<div class="article-title">Progress, challenges and the future of crop genomes,</div><div class="source">Curr. Opin. Plant Biol</div>Â ., <div class="volume">24</div>, <div class="fpage">71</div>â€“<div class="lpage">81</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Progress%2C%20challenges%20and%20the%20future%20of%20crop%20genomes%2C&amp;author=T.P.%20Michael&amp;author=R.%20VanBuren&amp;publication_year=2015&amp;journal=Curr.%20Opin.%20Plant%20Biol&amp;volume=24&amp;pages=71-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.pbi.2015.02.002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.pbi.2015.02.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.pbi.2015.02.002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25703261" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx020-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx020-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Guajardo</div>
               <div class="given-names">V.</div>
            </div>, <div class="name string-name"><div class="surname">SolÃ­s</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Sagredo</div> <div class="given-names">B.</div></div></div>, et al. Â <div class="year">2015</div>, 
<div class="article-title">Construction of high density sweet cherry (<em>Prunus avium</em> L.) linkage maps using microsatellite markers and SNPs detected by genotyping-by-sequencing (GBS),</div><div class="source">PLoS One</div>Â , <div class="volume">10</div>, <div class="fpage">e0127750</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20high%20density%20sweet%20cherry%20(Prunus%20avium%20L.)%20linkage%20maps%20using%20microsatellite%20markers%20and%20SNPs%20detected%20by%20genotyping-by-sequencing%20(GBS)%2C&amp;author=V.%20Guajardo&amp;author=S.%20Sol%C3%ADs&amp;author=B.%20Sagredo&amp;publication_year=2015&amp;journal=PLoS%20One&amp;volume=10&amp;pages=e0127750" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0127750" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0127750" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0127750"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26011256" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx020-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx020-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Dirlewanger</div>
               <div class="given-names">E.</div>
            </div>, <div class="name string-name"><div class="surname">Graziano</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Joobeur</div> <div class="given-names">T.</div></div></div>, et al. Â <div class="year">2004</div>, 
<div class="article-title">Comparative mapping and marker-assisted selection in Rosaceae fruit crops</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div>Â , <div class="volume">101</div>, <div class="fpage">9891</div>â€“<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20mapping%20and%20marker-assisted%20selection%20in%20Rosaceae%20fruit%20crops&amp;author=E.%20Dirlewanger&amp;author=E.%20Graziano&amp;author=T.%20Joobeur&amp;publication_year=2004&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=101&amp;pages=9891-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0307937101" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0307937101" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0307937101"> </span></div>Â </div></div></div></div></div><div content-id="dsx020-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx020-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Yamamoto</div>
               <div class="given-names">T.</div>
            </div>, <div class="name string-name"><div class="surname">Terakami</div> <div class="given-names">S.</div></div></div>
          <div class="year">2016</div>, 
<div class="article-title">Genomics of pear and other Rosaceae fruit trees,</div><div class="source">Breed. Sci</div>Â ., <div class="volume">66</div>, <div class="fpage">148</div>â€“<div class="lpage">59</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomics%20of%20pear%20and%20other%20Rosaceae%20fruit%20trees%2C&amp;author=T.%20Yamamoto&amp;author=S.%20Terakami&amp;publication_year=2016&amp;journal=Breed.%20Sci&amp;volume=66&amp;pages=148-59" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1270/jsbbs.66.148" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1270%2Fjsbbs.66.148" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1270%2Fjsbbs.66.148"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27069399" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx020-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx020-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Koepke</div>
               <div class="given-names">T.</div>
            </div>, <div class="name string-name"><div class="surname">Schaeffer</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Harper</div> <div class="given-names">A.</div></div></div>, et al. Â <div class="year">2013</div>, 
<div class="article-title">Comparative genomics analysis in Prunoideae to identify biologically relevant polymorphisms,</div><div class="source">Plant Biotechnol. J</div>Â ., <div class="volume">11</div>, <div class="fpage">883</div>â€“<div class="lpage">93</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20genomics%20analysis%20in%20Prunoideae%20to%20identify%20biologically%20relevant%20polymorphisms%2C&amp;author=T.%20Koepke&amp;author=S.%20Schaeffer&amp;author=A.%20Harper&amp;publication_year=2013&amp;journal=Plant%20Biotechnol.%20J&amp;volume=11&amp;pages=883-93" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/pbi.2013.11.issue-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fpbi.2013.11.issue-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fpbi.2013.11.issue-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23763653" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx020-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx020-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Illa</div>
               <div class="given-names">E.</div>
            </div>, <div class="name string-name"><div class="surname">Sargent</div> <div class="given-names">D.J.</div></div>, <div class="name string-name"><div class="surname">Lopez</div></div>, <div class="name string-name"><div class="surname">Girona</div> <div class="given-names">E.</div></div></div>, et al. Â <div class="year">2011</div>, 
<div class="article-title">Comparative analysis of rosaceous genomes and the reconstruction of a putative ancestral genome for the family,</div><div class="source">BMC Evol. Biol</div>Â ., <div class="volume">11</div>, <div class="fpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20analysis%20of%20rosaceous%20genomes%20and%20the%20reconstruction%20of%20a%20putative%20ancestral%20genome%20for%20the%20family%2C&amp;author=E.%20Illa&amp;author=D.J.%20Sargent&amp;author=%20Lopez&amp;author=E.%20Girona&amp;publication_year=2011&amp;journal=BMC%20Evol.%20Biol&amp;volume=11&amp;pages=9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2148-11-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2148-11-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2148-11-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21226921" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx020-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx020-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Peace</div>
               <div class="given-names">C.</div>
            </div>, <div class="name string-name"><div class="surname">Bassil</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Main</div> <div class="given-names">D.</div></div></div>, et al. Â <div class="year">2012</div>, 
<div class="article-title">Development and evaluation of a genome-wide 6K SNP array for diploid sweet cherry and tetraploid sour cherry,</div><div class="source">PLoS One</div>Â , <div class="volume">7</div>, <div class="fpage">e48305</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20and%20evaluation%20of%20a%20genome-wide%206K%20SNP%20array%20for%20diploid%20sweet%20cherry%20and%20tetraploid%20sour%20cherry%2C&amp;author=C.%20Peace&amp;author=N.%20Bassil&amp;author=D.%20Main&amp;publication_year=2012&amp;journal=PLoS%20One&amp;volume=7&amp;pages=e48305" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0048305" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0048305" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0048305"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23284615" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx020-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx020-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Van Nieuwerburgh</div>
               <div class="given-names">F.</div>
            </div>, <div class="name string-name"><div class="surname">Thompson</div> <div class="given-names">R.C.</div></div>, <div class="name string-name"><div class="surname">Ledesma</div> <div class="given-names">J.</div></div></div>, et al. Â <div class="year">2012</div>, 
<div class="article-title">Illumina mate-paired DNA sequencing-library preparation using Cre-Lox recombination</div>. <div class="source">Nucl. Acids Res</div>Â ., <div class="volume">40</div>, <div class="fpage">e24</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Illumina%20mate-paired%20DNA%20sequencing-library%20preparation%20using%20Cre-Lox%20recombination&amp;author=F.%20Van%20Nieuwerburgh&amp;author=R.C.%20Thompson&amp;author=J.%20Ledesma&amp;publication_year=2012&amp;journal=Nucl.%20Acids%20Res&amp;volume=40&amp;pages=e24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkr1000" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr1000" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkr1000"> </span></div>Â </div></div></div></div></div><div content-id="dsx020-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx020-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Schmieder</div>
               <div class="given-names">R.</div>
            </div>, <div class="name string-name"><div class="surname">Edwards</div> <div class="given-names">R.</div></div></div>
          <div class="year">2011</div>, 
<div class="article-title">Quality control and preprocessing of metagenomic datasets,</div><div class="source">Bioinformatics</div>Â , <div class="volume">27</div>, <div class="fpage">863</div>â€“<div class="lpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Quality%20control%20and%20preprocessing%20of%20metagenomic%20datasets%2C&amp;author=R.%20Schmieder&amp;author=R.%20Edwards&amp;publication_year=2011&amp;journal=Bioinformatics&amp;volume=27&amp;pages=863-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btr026" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtr026" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtr026"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21278185" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx020-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx020-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Marcais</div>
               <div class="given-names">G.</div>
            </div>, <div class="name string-name"><div class="surname">Kingsford</div> <div class="given-names">C.</div></div></div>
          <div class="year">2011</div>, 
<div class="article-title">A fast, lock-free approach for efficient parallel counting of occurrences of <em>k</em>-mers,</div><div class="source">Bioinformatics</div>Â , <div class="volume">27</div>, <div class="fpage">764</div>â€“<div class="lpage">70</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20fast%2C%20lock-free%20approach%20for%20efficient%20parallel%20counting%20of%20occurrences%20of%20k-mers%2C&amp;author=G.%20Marcais&amp;author=C.%20Kingsford&amp;publication_year=2011&amp;journal=Bioinformatics&amp;volume=27&amp;pages=764-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btr011" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtr011" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtr011"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21217122" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx020-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx020-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Luo</div>
               <div class="given-names">R.</div>
            </div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Xie</div> <div class="given-names">Y.</div></div></div>, et al. Â <div class="year">2012</div>, 
<div class="article-title">SOAPdenovo2: an empirically improved memory-efficient short-read <em>de novo</em> assembler,</div><div class="source">Gigascience</div>Â , <div class="volume">1</div>, <div class="fpage">18</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SOAPdenovo2%3A%20an%20empirically%20improved%20memory-efficient%20short-read%20de%20novo%20assembler%2C&amp;author=R.%20Luo&amp;author=B.%20Liu&amp;author=Y.%20Xie&amp;publication_year=2012&amp;journal=Gigascience&amp;volume=1&amp;pages=18" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/2047-217X-1-18" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F2047-217X-1-18" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F2047-217X-1-18"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23587118" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx020-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx020-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kajitani</div>
               <div class="given-names">R.</div>
            </div>, <div class="name string-name"><div class="surname">Toshimoto</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Noguchi</div> <div class="given-names">H.</div></div></div>, et al. Â <div class="year">2014</div>, 
<div class="article-title">Efficient <em>de novo</em> assembly of highly heterozygous genomes from whole-genome shotgun short reads,</div><div class="source">Genome Res</div>Â ., <div class="volume">24</div>, <div class="fpage">1384</div>â€“<div class="lpage">95</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Efficient%20de%20novo%20assembly%20of%20highly%20heterozygous%20genomes%20from%20whole-genome%20shotgun%20short%20reads%2C&amp;author=R.%20Kajitani&amp;author=K.%20Toshimoto&amp;author=H.%20Noguchi&amp;publication_year=2014&amp;journal=Genome%20Res&amp;volume=24&amp;pages=1384-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.170720.113" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.170720.113" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.170720.113"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24755901" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx020-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx020-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">SimÃ£o</div>
               <div class="given-names">F.A.</div>
            </div>, <div class="name string-name"><div class="surname">Waterhouse</div> <div class="given-names">R.M.</div></div>, <div class="name string-name"><div class="surname">Ioannidis</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Kriventseva</div> <div class="given-names">E.V.</div></div>, <div class="name string-name"><div class="surname">Zdobnov</div> <div class="given-names">E.M.</div></div></div>
          <div class="year">2015</div>, 
<div class="article-title">BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs,</div><div class="source">Bioinformatics</div>Â , <div class="volume">31</div>, <div class="fpage">3210</div>â€“<div class="lpage">2</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=BUSCO%3A%20assessing%20genome%20assembly%20and%20annotation%20completeness%20with%20single-copy%20orthologs%2C&amp;author=F.A.%20Sim%C3%A3o&amp;author=R.M.%20Waterhouse&amp;author=P.%20Ioannidis&amp;author=E.V.%20Kriventseva&amp;author=E.M.%20Zdobnov&amp;publication_year=2015&amp;journal=Bioinformatics&amp;volume=31&amp;pages=3210-2" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btv351" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtv351" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtv351"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26059717" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx020-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx020-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Jurka</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Kapitonov</div> <div class="given-names">V.V.</div></div>, <div class="name string-name"><div class="surname">Pavlicek</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Klonowski</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Kohany</div> <div class="given-names">O.</div></div>, <div class="name string-name"><div class="surname">Walichiewicz</div> <div class="given-names">J.</div></div></div>
          <div class="year">2005</div>, 
<div class="article-title">Repbase Update, a database of eukaryotic repetitive elements,</div><div class="source">Cytogenet. Genome Res</div>Â ., <div class="volume">110</div>, <div class="fpage">462</div>â€“<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Repbase%20Update%2C%20a%20database%20of%20eukaryotic%20repetitive%20elements%2C&amp;author=J.%20Jurka&amp;author=V.V.%20Kapitonov&amp;author=A.%20Pavlicek&amp;author=P.%20Klonowski&amp;author=O.%20Kohany&amp;author=J.%20Walichiewicz&amp;publication_year=2005&amp;journal=Cytogenet.%20Genome%20Res&amp;volume=110&amp;pages=462-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1159/000084979" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1159%2F000084979" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1159%2F000084979"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16093699" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx020-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx020-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Price</div>
               <div class="given-names">A.L.</div>
            </div>, <div class="name string-name"><div class="surname">Jones</div> <div class="given-names">N.C.</div></div>, <div class="name string-name"><div class="surname">Pevzner</div> <div class="given-names">P.A.</div></div></div>
          <div class="year">2005</div>, 
<div class="article-title"><em>De novo</em> identification of repeat families in large genomes,</div><div class="source">Bioinformatics</div>Â , <div class="volume">21</div>, <div class="fpage">i351</div>â€“<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=De%20novo%20identification%20of%20repeat%20families%20in%20large%20genomes%2C&amp;author=A.L.%20Price&amp;author=N.C.%20Jones&amp;author=P.A.%20Pevzner&amp;publication_year=2005&amp;journal=Bioinformatics&amp;volume=21&amp;pages=i351-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bti1018" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbti1018" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbti1018"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15961478" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx020-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx020-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Smit</div>
               <div class="given-names">A.F.A.</div>
            </div>, <div class="name string-name"><div class="surname">Hubley</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Green</div> <div class="given-names">P.</div></div></div>
          <div class="year">2013</div>-2015, <em>RepeatMasker Open-4.0.</em><a class="link link-uri" href="http://www.repeatmasker.org" target="_blank">http://www.repeatmasker.org</a> (8 May 2017, date last accessed).</div></div></div></div><div content-id="dsx020-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx020-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Li</div>
               <div class="given-names">Q.</div>
            </div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Dai</div> <div class="given-names">S.</div></div></div>, et al. Â <div class="year">2015</div>, 
<div class="article-title"><em>PacCYP707A2</em> negatively regulates cherry fruit ripening while <em>PacCYP707A1</em> mediates drought tolerance,</div><div class="source">J. Exp. Bot</div>Â ., <div class="volume">66</div>, <div class="fpage">3765</div>â€“<div class="lpage">74</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PacCYP707A2%20negatively%20regulates%20cherry%20fruit%20ripening%20while%20PacCYP707A1%20mediates%20drought%20tolerance%2C&amp;author=Q.%20Li&amp;author=P.%20Chen&amp;author=S.%20Dai&amp;publication_year=2015&amp;journal=J.%20Exp.%20Bot&amp;volume=66&amp;pages=3765-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jxb/erv169" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjxb%2Ferv169" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjxb%2Ferv169"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25956880" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx020-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx020-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Grabherr</div>
               <div class="given-names">M.G.</div>
            </div>, <div class="name string-name"><div class="surname">Haas</div> <div class="given-names">B.J.</div></div>, <div class="name string-name"><div class="surname">Yassour</div> <div class="given-names">M.</div></div></div>, et al. Â <div class="year">2011</div>, 
<div class="article-title">Full-length transcriptome assembly from RNA-seq data without a reference genome,</div><div class="source">Nat. Biotechnol</div>Â ., <div class="volume">29</div>, <div class="fpage">644</div>â€“<div class="lpage">52</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Full-length%20transcriptome%20assembly%20from%20RNA-seq%20data%20without%20a%20reference%20genome%2C&amp;author=M.G.%20Grabherr&amp;author=B.J.%20Haas&amp;author=M.%20Yassour&amp;publication_year=2011&amp;journal=Nat.%20Biotechnol&amp;volume=29&amp;pages=644-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.1883" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.1883" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.1883"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21572440" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx020-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx020-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Lowe</div>
               <div class="given-names">T.M.</div>
            </div>, <div class="name string-name"><div class="surname">Eddy</div> <div class="given-names">S.R.</div></div></div>
          <div class="year">1997</div>, 
<div class="article-title">tRNAscan-SE: a program for improved detection of transfer RNA genes in genomic sequence,</div><div class="source">Nucl. Acids Res</div>Â ., <div class="volume">25</div>, <div class="fpage">955</div>â€“<div class="lpage">64</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=tRNAscan-SE%3A%20a%20program%20for%20improved%20detection%20of%20transfer%20RNA%20genes%20in%20genomic%20sequence%2C&amp;author=T.M.%20Lowe&amp;author=S.R.%20Eddy&amp;publication_year=1997&amp;journal=Nucl.%20Acids%20Res&amp;volume=25&amp;pages=955-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.5.0955" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.5.0955" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.5.0955"> </span></div>Â </div></div></div></div></div><div content-id="dsx020-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx020-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Cantarel</div>
               <div class="given-names">B.L.</div>
            </div>, <div class="name string-name"><div class="surname">Korf</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Robb</div> <div class="given-names">S.M.</div></div></div>, et al. Â <div class="year">2008</div>, 
<div class="article-title">MAKER: an easy-to-use annotation pipeline designed for emerging model organism genomes</div>, <div class="source">Genome Res</div>Â ., <div class="volume">18</div>, <div class="fpage">188</div>â€“<div class="lpage">96</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MAKER%3A%20an%20easy-to-use%20annotation%20pipeline%20designed%20for%20emerging%20model%20organism%20genomes&amp;author=B.L.%20Cantarel&amp;author=I.%20Korf&amp;author=S.M.%20Robb&amp;publication_year=2008&amp;journal=Genome%20Res&amp;volume=18&amp;pages=188-96" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.6743907" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.6743907" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.6743907"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18025269" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx020-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx020-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Tennessen</div>
               <div class="given-names">J.A.</div>
            </div>, <div class="name string-name"><div class="surname">Govindarajulu</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Ashman</div> <div class="given-names">T.L.</div></div>, <div class="name string-name"><div class="surname">Liston</div> <div class="given-names">A.</div></div></div>
          <div class="year">2014</div>, 
<div class="article-title">Evolutionary origins and dynamics of octoploid strawberry subgenomes revealed by dense targeted capture linkage maps,</div><div class="source">Genome. Biol. Evol</div>Â ., <div class="volume">6</div>, <div class="fpage">3295</div>â€“<div class="lpage">313</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolutionary%20origins%20and%20dynamics%20of%20octoploid%20strawberry%20subgenomes%20revealed%20by%20dense%20targeted%20capture%20linkage%20maps%2C&amp;author=J.A.%20Tennessen&amp;author=R.%20Govindarajulu&amp;author=T.L.%20Ashman&amp;author=A.%20Liston&amp;publication_year=2014&amp;journal=Genome.%20Biol.%20Evol&amp;volume=6&amp;pages=3295-313" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/gbe/evu261" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fgbe%2Fevu261" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fgbe%2Fevu261"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25477420" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx020-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx020-B29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Hoff</div>
               <div class="given-names">K.J.</div>
            </div>, <div class="name string-name"><div class="surname">Lange</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Lomsadze</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Borodovsky</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Stanke</div> <div class="given-names">M.</div></div></div>
          <div class="year">2016</div>, 
<div class="article-title">BRAKER1: unsupervised RNA-Seq-based genome annotation with GeneMark-ET and AUGUSTUS,</div><div class="source">Bioinformatics</div>Â , <div class="volume">32</div>, <div class="fpage">767</div>â€“<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=BRAKER1%3A%20unsupervised%20RNA-Seq-based%20genome%20annotation%20with%20GeneMark-ET%20and%20AUGUSTUS%2C&amp;author=K.J.%20Hoff&amp;author=S.%20Lange&amp;author=A.%20Lomsadze&amp;author=M.%20Borodovsky&amp;author=M.%20Stanke&amp;publication_year=2016&amp;journal=Bioinformatics&amp;volume=32&amp;pages=767-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btv661" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtv661" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtv661"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26559507" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx020-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx020-B30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Quevillon</div>
               <div class="given-names">E.</div>
            </div>, <div class="name string-name"><div class="surname">Silventoinen</div> <div class="given-names">V.</div></div>, <div class="name string-name"><div class="surname">Pillai</div> <div class="given-names">S.</div></div></div>, et al. Â <div class="year">2005</div>, 
<div class="article-title">InterProScan: protein domains identifier,</div><div class="source">Nucl. Acids Res</div>Â ., <div class="volume">33</div>, <div class="fpage">W116</div>â€“<div class="lpage">20</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=InterProScan%3A%20protein%20domains%20identifier%2C&amp;author=E.%20Quevillon&amp;author=V.%20Silventoinen&amp;author=S.%20Pillai&amp;publication_year=2005&amp;journal=Nucl.%20Acids%20Res&amp;volume=33&amp;pages=W116-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gki442" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgki442" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgki442"> </span></div>Â </div></div></div></div></div><div content-id="dsx020-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx020-B31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Mulder</div>
               <div class="given-names">N.J.</div>
            </div>, <div class="name string-name"><div class="surname">Apweiler</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Attwood</div> <div class="given-names">T.K.</div></div></div>, et al. Â <div class="year">2007</div>, 
<div class="article-title">New developments in the InterPro database,</div><div class="source">Nucl. Acids Res</div>Â ., <div class="volume">35</div>, <div class="fpage">D224</div>â€“<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=New%20developments%20in%20the%20InterPro%20database%2C&amp;author=N.J.%20Mulder&amp;author=R.%20Apweiler&amp;author=T.K.%20Attwood&amp;publication_year=2007&amp;journal=Nucl.%20Acids%20Res&amp;volume=35&amp;pages=D224-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkl841" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkl841" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkl841"> </span></div>Â </div></div></div></div></div><div content-id="dsx020-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx020-B32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Li</div>
               <div class="given-names">W.</div>
            </div>, <div class="name string-name"><div class="surname">Godzik</div> <div class="given-names">A.</div></div></div>
          <div class="year">2006</div>, 
<div class="article-title">Cd-hit: a fast program for clustering and comparing large sets of protein or nucleotide sequences,</div><div class="source">Bioinformatics</div>Â , <div class="volume">22</div>, <div class="fpage">1658</div>â€“<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cd-hit%3A%20a%20fast%20program%20for%20clustering%20and%20comparing%20large%20sets%20of%20protein%20or%20nucleotide%20sequences%2C&amp;author=W.%20Li&amp;author=A.%20Godzik&amp;publication_year=2006&amp;journal=Bioinformatics&amp;volume=22&amp;pages=1658-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btl158" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtl158" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtl158"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16731699" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx020-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx020-B33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ashburner</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Ball</div> <div class="given-names">C.A.</div></div>, <div class="name string-name"><div class="surname">Blake</div> <div class="given-names">J.A.</div></div></div>, et al. Â <div class="year">2000</div>, 
<div class="article-title">Gene Ontology: tool for the unification of biology,</div><div class="source">Nat. Genet</div>Â ., <div class="volume">25</div>, <div class="fpage">25</div>â€“<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20Ontology%3A%20tool%20for%20the%20unification%20of%20biology%2C&amp;author=M.%20Ashburner&amp;author=C.A.%20Ball&amp;author=J.A.%20Blake&amp;publication_year=2000&amp;journal=Nat.%20Genet&amp;volume=25&amp;pages=25-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/75556" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F75556" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F75556"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10802651" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx020-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx020-B34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Tatusov</div>
               <div class="given-names">R.L.</div>
            </div>, <div class="name string-name"><div class="surname">Fedorova</div> <div class="given-names">N.D.</div></div>, <div class="name string-name"><div class="surname">Jackson</div> <div class="given-names">J.D.</div></div></div>, et al. Â <div class="year">2003</div>, 
<div class="article-title">The COG database: an updated version includes eukaryotes,</div><div class="source">BMC Bioinform</div>Â ., <div class="volume">4</div>, <div class="fpage">41</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20COG%20database%3A%20an%20updated%20version%20includes%20eukaryotes%2C&amp;author=R.L.%20Tatusov&amp;author=N.D.%20Fedorova&amp;author=J.D.%20Jackson&amp;publication_year=2003&amp;journal=BMC%20Bioinform&amp;volume=4&amp;pages=41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-4-41" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-4-41" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-4-41"> </span></div>Â </div></div></div></div></div><div content-id="dsx020-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx020-B35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ogata</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Goto</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Sato</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Fujibuchi</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Bono</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Kanehisa</div> <div class="given-names">M.</div></div></div>
          <div class="year">1999</div>, 
<div class="article-title">KEGG: kyoto encyclopedia of genes and genomes,</div><div class="source">Nucl. Acids Res</div>Â ., <div class="volume">27</div>, <div class="fpage">29</div>â€“<div class="lpage">34</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=KEGG%3A%20kyoto%20encyclopedia%20of%20genes%20and%20genomes%2C&amp;author=H.%20Ogata&amp;author=S.%20Goto&amp;author=K.%20Sato&amp;author=W.%20Fujibuchi&amp;author=H.%20Bono&amp;author=M.%20Kanehisa&amp;publication_year=1999&amp;journal=Nucl.%20Acids%20Res&amp;volume=27&amp;pages=29-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/27.1.29" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F27.1.29" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F27.1.29"> </span></div>Â </div></div></div></div></div><div content-id="dsx020-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx020-B36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Shirasawa</div>
               <div class="given-names">K.</div>
            </div>, <div class="name string-name"><div class="surname">Hirakawa</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Isobe</div> <div class="given-names">S.</div></div></div>
          <div class="year">2016</div>, 
<div class="article-title">Analytical workflow of double-digest restriction site-associated DNA sequencing based on empirical and <em>in silico</em> optimization in tomato,</div><div class="source">DNA Res</div>Â ., <div class="volume">23</div>, <div class="fpage">145</div>â€“<div class="lpage">53</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analytical%20workflow%20of%20double-digest%20restriction%20site-associated%20DNA%20sequencing%20based%20on%20empirical%20and%20in%20silico%20optimization%20in%20tomato%2C&amp;author=K.%20Shirasawa&amp;author=H.%20Hirakawa&amp;author=S.%20Isobe&amp;publication_year=2016&amp;journal=DNA%20Res&amp;volume=23&amp;pages=145-53" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsw004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsw004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsw004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26932983" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx020-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx020-B37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Langmead</div>
               <div class="given-names">B.</div>
            </div>, <div class="name string-name"><div class="surname">Salzberg</div> <div class="given-names">S.L.</div></div></div>
          <div class="year">2012</div>, 
<div class="article-title">Fast gapped-read alignment with Bowtie 2,</div><div class="source">Nat. Methods</div>Â , <div class="volume">9</div>, <div class="fpage">357</div>â€“<div class="lpage">59</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fast%20gapped-read%20alignment%20with%20Bowtie%202%2C&amp;author=B.%20Langmead&amp;author=S.L.%20Salzberg&amp;publication_year=2012&amp;journal=Nat.%20Methods&amp;volume=9&amp;pages=357-59" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.1923" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.1923" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.1923"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22388286" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx020-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx020-B38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Li</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Handsaker</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Wysoker</div> <div class="given-names">A.</div></div></div>, et al. Â <div class="year">2009</div>, 
<div class="article-title">The sequence alignment/map format and SAMtools,</div><div class="source">Bioinformatics</div>Â , <div class="volume">25</div>, <div class="fpage">2078</div>â€“<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20sequence%20alignment%2Fmap%20format%20and%20SAMtools%2C&amp;author=H.%20Li&amp;author=B.%20Handsaker&amp;author=A.%20Wysoker&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=2078-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp352" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp352" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp352"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19505943" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx020-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx020-B39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Browning</div>
               <div class="given-names">S.R.</div>
            </div>, <div class="name string-name"><div class="surname">Browning</div> <div class="given-names">B.L.</div></div></div>
          <div class="year">2007</div>, 
<div class="article-title">Rapid and accurate haplotype phasing and missing-data inference for whole-genome association studies by use of localized haplotype clustering,</div><div class="source">Am. J. Hum. Genet</div>Â ., <div class="volume">81</div>, <div class="fpage">1084</div>â€“<div class="lpage">97</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rapid%20and%20accurate%20haplotype%20phasing%20and%20missing-data%20inference%20for%20whole-genome%20association%20studies%20by%20use%20of%20localized%20haplotype%20clustering%2C&amp;author=S.R.%20Browning&amp;author=B.L.%20Browning&amp;publication_year=2007&amp;journal=Am.%20J.%20Hum.%20Genet&amp;volume=81&amp;pages=1084-97" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1086/521987" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1086%2F521987" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1086%2F521987"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17924348" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx020-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx020-B40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Danecek</div>
               <div class="given-names">P.</div>
            </div>, <div class="name string-name"><div class="surname">Auton</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Abecasis</div> <div class="given-names">G.</div></div></div>, et al. Â <div class="year">2011</div>, 
<div class="article-title">The variant call format and VCFtools,</div><div class="source">Bioinformatics</div>Â , <div class="volume">27</div>, <div class="fpage">2156</div>â€“<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20variant%20call%20format%20and%20VCFtools%2C&amp;author=P.%20Danecek&amp;author=A.%20Auton&amp;author=G.%20Abecasis&amp;publication_year=2011&amp;journal=Bioinformatics&amp;volume=27&amp;pages=2156-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btr330" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtr330" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtr330"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21653522" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx020-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx020-B41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Aranzana</div>
               <div class="given-names">M.J.</div>
            </div>, <div class="name string-name"><div class="surname">Garcia-Mas</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Carbo</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Arus</div> <div class="given-names">P.</div></div></div>
          <div class="year">2002</div>, 
<div class="article-title">Development and variability analysis of microsatellite markers in peach,</div><div class="source">Plant Breed</div>Â ., <div class="volume">121</div>, <div class="fpage">87</div>â€“<div class="lpage">92</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20and%20variability%20analysis%20of%20microsatellite%20markers%20in%20peach%2C&amp;author=M.J.%20Aranzana&amp;author=J.%20Garcia-Mas&amp;author=J.%20Carbo&amp;author=P.%20Arus&amp;publication_year=2002&amp;journal=Plant%20Breed&amp;volume=121&amp;pages=87-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1439-0523.2002.00656.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1439-0523.2002.00656.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1439-0523.2002.00656.x"> </span></div>Â </div></div></div></div></div><div content-id="dsx020-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx020-B42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Cantini</div>
               <div class="given-names">C.</div>
            </div>, <div class="name string-name"><div class="surname">Iezzoni</div> <div class="given-names">A.F.</div></div>, <div class="name string-name"><div class="surname">Lamboy</div> <div class="given-names">W.F.</div></div>, <div class="name string-name"><div class="surname">Boritzki</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Struss</div> <div class="given-names">D.</div></div></div>
          <div class="year">2001</div>, 
<div class="article-title">DNA fingerprinting of tetraploid cherry germplasm using simple sequence repeats</div>, <div class="source">J. Amer. Soc. Hort. Sci</div>Â ., <div class="volume">216</div>, <div class="fpage">205</div>â€“<div class="lpage">9</div>.</div></div></div></div><div content-id="dsx020-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx020-B43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Cipriani</div>
               <div class="given-names">G.</div>
            </div>, <div class="name string-name"><div class="surname">Lot</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Huang</div> <div class="given-names">W.G.</div></div>, <div class="name string-name"><div class="surname">Marrazzo</div> <div class="given-names">M.T.</div></div>, <div class="name string-name"><div class="surname">Peterlunger</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Testolin</div> <div class="given-names">R.</div></div></div>
          <div class="year">1999</div>, 
<div class="article-title">AC/GT and AG/CT microsatellite repeats in peach [<em>Prunus persica</em> (L) Batsch]: isolation, characterisation and cross-species amplification in <em>Prunus</em>,</div><div class="source">Theor. Appl. Genet</div>Â ., <div class="volume">99</div>, <div class="fpage">65</div>â€“<div class="lpage">72</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=AC%2FGT%20and%20AG%2FCT%20microsatellite%20repeats%20in%20peach%20%5BPrunus%20persica%20(L)%20Batsch%5D%3A%20isolation%2C%20characterisation%20and%20cross-species%20amplification%20in%20Prunus%2C&amp;author=G.%20Cipriani&amp;author=G.%20Lot&amp;author=W.G.%20Huang&amp;author=M.T.%20Marrazzo&amp;author=E.%20Peterlunger&amp;author=R.%20Testolin&amp;publication_year=1999&amp;journal=Theor.%20Appl.%20Genet&amp;volume=99&amp;pages=65-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s001220051209" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs001220051209" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs001220051209"> </span></div>Â </div></div></div></div></div><div content-id="dsx020-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx020-B44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Clarke</div>
               <div class="given-names">J.B.</div>
            </div>, <div class="name string-name"><div class="surname">Tobutt</div> <div class="given-names">K.R.</div></div></div>
          <div class="year">2003</div>, 
<div class="article-title">Development and characterization of polymorphic microsatellites from <em>Prunus avium</em> â€²Napoleonâ€²,</div><div class="source">Mol. Ecol. Notes</div>Â , <div class="volume">3</div>, <div class="fpage">578</div>â€“<div class="lpage">80</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20and%20characterization%20of%20polymorphic%20microsatellites%20from%20Prunus%20avium%20%E2%80%B2Napoleon%E2%80%B2%2C&amp;author=J.B.%20Clarke&amp;author=K.R.%20Tobutt&amp;publication_year=2003&amp;journal=Mol.%20Ecol.%20Notes&amp;volume=3&amp;pages=578-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1471-8286.2003.00517.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1471-8286.2003.00517.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1471-8286.2003.00517.x"> </span></div>Â </div></div></div></div></div><div content-id="dsx020-B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx020-B45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Dirlewanger</div>
               <div class="given-names">E.</div>
            </div>, <div class="name string-name"><div class="surname">Cosson</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Tavaud</div> <div class="given-names">M.</div></div></div>, et al. Â <div class="year">2002</div>, 
<div class="article-title">Development of microsatellite markers in peach [<em>Prunus persica</em> (L.) Batsch] and their use in genetic diversity analysis in peach and sweet cherry (<em>Prunus avium</em> L.)</div>, <div class="source">Theor. Appl. Genet</div>Â ., <div class="volume">105</div>, <div class="fpage">127</div>â€“<div class="lpage">38</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20microsatellite%20markers%20in%20peach%20%5BPrunus%20persica%20(L.)%20Batsch%5D%20and%20their%20use%20in%20genetic%20diversity%20analysis%20in%20peach%20and%20sweet%20cherry%20(Prunus%20avium%20L.)&amp;author=E.%20Dirlewanger&amp;author=P.%20Cosson&amp;author=M.%20Tavaud&amp;publication_year=2002&amp;journal=Theor.%20Appl.%20Genet&amp;volume=105&amp;pages=127-38" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-002-0867-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-002-0867-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-002-0867-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12582570" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx020-B46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx020-B46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Joobeur</div>
               <div class="given-names">T.</div>
            </div>, <div class="name string-name"><div class="surname">Periam</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Vicente</div> <div class="given-names">M.C.</div></div>, <div class="name string-name"><div class="surname">King</div> <div class="given-names">G.J.</div></div>, <div class="name string-name"><div class="surname">ArÃºs</div> <div class="given-names">P.</div></div></div>
          <div class="year">2000</div>, 
<div class="article-title">Development of a second generation linkage map for almond using RAPD and SSR markers,</div><div class="source">Genome</div>Â , <div class="volume">43</div>, <div class="fpage">649</div>â€“<div class="lpage">55</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20a%20second%20generation%20linkage%20map%20for%20almond%20using%20RAPD%20and%20SSR%20markers%2C&amp;author=T.%20Joobeur&amp;author=N.%20Periam&amp;author=M.C.%20Vicente&amp;author=G.J.%20King&amp;author=P.%20Ar%C3%BAs&amp;publication_year=2000&amp;journal=Genome&amp;volume=43&amp;pages=649-55" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/g00-040" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2Fg00-040" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2Fg00-040"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10984177" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx020-B47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx020-B47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Jung</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Staton</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Lee</div> <div class="given-names">T.</div></div></div>, et al. Â <div class="year">2008</div>, 
<div class="article-title">GDR (Genome Database for Rosaceae): integrated web-database for Rosaceae genomics and genetics data,</div><div class="source">Nucl. Acids Res</div>Â ., <div class="volume">36</div>, <div class="fpage">D1034</div>â€“<div class="lpage">40</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GDR%20(Genome%20Database%20for%20Rosaceae)%3A%20integrated%20web-database%20for%20Rosaceae%20genomics%20and%20genetics%20data%2C&amp;author=S.%20Jung&amp;author=M.%20Staton&amp;author=T.%20Lee&amp;publication_year=2008&amp;journal=Nucl.%20Acids%20Res&amp;volume=36&amp;pages=D1034-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkm803" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkm803" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkm803"> </span></div>Â </div></div></div></div></div><div content-id="dsx020-B48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx020-B48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Lopes</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Sefc</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Laimer</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Machado</div> <div class="given-names">A.</div></div></div>
          <div class="year">2002</div>, 
<div class="article-title">Identification of microsatellite loci in apricot,</div><div class="source">Mol. Ecol. Notes</div>Â , <div class="volume">2</div>, <div class="fpage">24</div>â€“<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20microsatellite%20loci%20in%20apricot%2C&amp;author=M.%20Lopes&amp;author=K.%20Sefc&amp;author=M.%20Laimer&amp;author=A.%20Machado&amp;publication_year=2002&amp;journal=Mol.%20Ecol.%20Notes&amp;volume=2&amp;pages=24-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1471-8286.2002.00132.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1471-8286.2002.00132.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1471-8286.2002.00132.x"> </span></div>Â </div></div></div></div></div><div content-id="dsx020-B49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx020-B49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Mnejja</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Garcia-Mas</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Howad</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Arus</div> <div class="given-names">P.</div></div></div>
          <div class="year">2005</div>, 
<div class="article-title">Development and transportability across <em>Prunus</em> species of 42 polymorphic almond microsatellites,</div><div class="source">Mol. Ecol. Resour</div>Â ., <div class="volume">5</div>, <div class="fpage">531</div>â€“<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20and%20transportability%20across%20Prunus%20species%20of%2042%20polymorphic%20almond%20microsatellites%2C&amp;author=M.%20Mnejja&amp;author=J.%20Garcia-Mas&amp;author=W.%20Howad&amp;author=P.%20Arus&amp;publication_year=2005&amp;journal=Mol.%20Ecol.%20Resour&amp;volume=5&amp;pages=531-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/men.2005.5.issue-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmen.2005.5.issue-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmen.2005.5.issue-3"> </span></div>Â </div></div></div></div></div><div content-id="dsx020-B50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx020-B50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Mnejja</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Garcia-Mas</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Audergon</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">ArÃºs</div> <div class="given-names">P.</div></div></div>
          <div class="year">2010</div>, 
<div class="article-title"><em>Prunus</em> microsatellite marker transferability across rosaceous crops,</div><div class="source">Tree Genet. Genomes</div>Â , <div class="volume">6</div>, <div class="fpage">689</div>â€“<div class="lpage">700</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Prunus%20microsatellite%20marker%20transferability%20across%20rosaceous%20crops%2C&amp;author=M.%20Mnejja&amp;author=J.%20Garcia-Mas&amp;author=J.%20Audergon&amp;author=P.%20Ar%C3%BAs&amp;publication_year=2010&amp;journal=Tree%20Genet.%20Genomes&amp;volume=6&amp;pages=689-700" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11295-010-0284-z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11295-010-0284-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11295-010-0284-z"> </span></div>Â </div></div></div></div></div><div content-id="dsx020-B51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx020-B51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Sosinski</div>
               <div class="given-names">B.</div>
            </div>, <div class="name string-name"><div class="surname">Gannavarapu</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Hager</div> <div class="given-names">L.D.</div></div></div>, et al. Â <div class="year">2000</div>, 
<div class="article-title">Characterization of microsatellite markers in peach [<em>Prunus persica</em> (L.) Batsch]</div>, <div class="source">Theor. Appl. Genet</div>Â ., <div class="volume">101</div>, <div class="fpage">421</div>â€“<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20microsatellite%20markers%20in%20peach%20%5BPrunus%20persica%20(L.)%20Batsch%5D&amp;author=B.%20Sosinski&amp;author=M.%20Gannavarapu&amp;author=L.D.%20Hager&amp;publication_year=2000&amp;journal=Theor.%20Appl.%20Genet&amp;volume=101&amp;pages=421-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s001220051499" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs001220051499" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs001220051499"> </span></div>Â </div></div></div></div></div><div content-id="dsx020-B52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx020-B52" class="ref-content "><div class="label -label">52</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Struss</div>
               <div class="given-names">D.</div>
            </div>, <div class="name string-name"><div class="surname">Ahmad</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Southwick</div> <div class="given-names">S.M.</div></div>, <div class="name string-name"><div class="surname">Boritzki</div> <div class="given-names">M.</div></div></div>
          <div class="year">2003</div>, 
<div class="article-title">Analysis of sweet cherry (<em>Prunus avium</em> L.) cultivars using SSR and AFLP markers</div>, <div class="source">J. Am. Soc. Hor. Sci</div>Â ., <div class="volume">128</div>, <div class="fpage">904</div>â€“<div class="lpage">9</div>.</div></div></div></div><div content-id="dsx020-B53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx020-B53" class="ref-content "><div class="label -label">53</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Testolin</div>
               <div class="given-names">R.</div>
            </div>, <div class="name string-name"><div class="surname">Marrazzo</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Cipriani</div> <div class="given-names">G.</div></div></div>, et al. Â <div class="year">2000</div>, 
<div class="article-title">Microsatellite DNA in peach (<em>Prunus persica</em> L. Batsch) and its use in fingerprinting and testing the genetic origin of cultivars,</div><div class="source">Genome</div>Â , <div class="volume">43</div>, <div class="fpage">512</div>â€“<div class="lpage">20</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microsatellite%20DNA%20in%20peach%20(Prunus%20persica%20L.%20Batsch)%20and%20its%20use%20in%20fingerprinting%20and%20testing%20the%20genetic%20origin%20of%20cultivars%2C&amp;author=R.%20Testolin&amp;author=T.%20Marrazzo&amp;author=G.%20Cipriani&amp;publication_year=2000&amp;journal=Genome&amp;volume=43&amp;pages=512-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/g00-010" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2Fg00-010" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2Fg00-010"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10902716" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx020-B54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx020-B54" class="ref-content "><div class="label -label">54</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Vaughan</div>
               <div class="given-names">S.P.</div>
            </div>, <div class="name string-name"><div class="surname">Russell</div> <div class="given-names">K.</div></div></div>
          <div class="year">2004</div>, 
<div class="article-title">Characterization of novel microsatellites and development of multiplex PCR for large-scale population studies in wild cherry, <em>Prunus avium</em>,</div><div class="source">Mol. Ecol. Resour</div>Â ., <div class="volume">4</div>, <div class="fpage">429</div>â€“<div class="lpage">31</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20novel%20microsatellites%20and%20development%20of%20multiplex%20PCR%20for%20large-scale%20population%20studies%20in%20wild%20cherry%2C%20Prunus%20avium%2C&amp;author=S.P.%20Vaughan&amp;author=K.%20Russell&amp;publication_year=2004&amp;journal=Mol.%20Ecol.%20Resour&amp;volume=4&amp;pages=429-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/men.2004.4.issue-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmen.2004.4.issue-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmen.2004.4.issue-3"> </span></div>Â </div></div></div></div></div><div content-id="dsx020-B55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx020-B55" class="ref-content "><div class="label -label">55</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Yamamoto</div>
               <div class="given-names">T.</div>
            </div>, <div class="name string-name"><div class="surname">Mochida</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Imai</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Shi</div> <div class="given-names">Y.Z.</div></div>, <div class="name string-name"><div class="surname">Ogiwara</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Hayashi</div> <div class="given-names">T.</div></div></div>
          <div class="year">2002</div>, 
<div class="article-title">Microsatellite markers in peach [<em>Prunus persica</em> (L.) Batsch] derived from an enriched genomic and cDNA libraries,</div><div class="source">Mol. Ecol. Resour</div>Â ., <div class="volume">2</div>, <div class="fpage">298</div>â€“<div class="lpage">301</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microsatellite%20markers%20in%20peach%20%5BPrunus%20persica%20(L.)%20Batsch%5D%20derived%20from%20an%20enriched%20genomic%20and%20cDNA%20libraries%2C&amp;author=T.%20Yamamoto&amp;author=K.%20Mochida&amp;author=T.%20Imai&amp;author=Y.Z.%20Shi&amp;author=I.%20Ogiwara&amp;author=T.%20Hayashi&amp;publication_year=2002&amp;journal=Mol.%20Ecol.%20Resour&amp;volume=2&amp;pages=298-301" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1471-8286.2002.00242.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1471-8286.2002.00242.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1471-8286.2002.00242.x"> </span></div>Â </div></div></div></div></div><div content-id="dsx020-B56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx020-B56" class="ref-content "><div class="label -label">56</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Yamamoto</div>
               <div class="given-names">T.</div>
            </div>, <div class="name string-name"><div class="surname">Yamaguchi</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Hayashi</div> <div class="given-names">T.</div></div></div>
          <div class="year">2005</div>, 
<div class="article-title">An integrated genetic linkage map of peach by SSR, STS, AFLP and RAPD,</div><div class="source">J. Japan. Soc. Hort. Sci</div>Â ., <div class="volume">74</div>, <div class="fpage">204</div>â€“<div class="lpage">13</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20integrated%20genetic%20linkage%20map%20of%20peach%20by%20SSR%2C%20STS%2C%20AFLP%20and%20RAPD%2C&amp;author=T.%20Yamamoto&amp;author=M.%20Yamaguchi&amp;author=T.%20Hayashi&amp;publication_year=2005&amp;journal=J.%20Japan.%20Soc.%20Hort.%20Sci&amp;volume=74&amp;pages=204-13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2503/jjshs.74.204" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2503%2Fjjshs.74.204" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2503%2Fjjshs.74.204"> </span></div>Â </div></div></div></div></div><div content-id="dsx020-B57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx020-B57" class="ref-content "><div class="label -label">57</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Van</div>
               <div class="given-names">Ooijen J.W.</div>
            </div>
          </div>
          <div class="year">2006</div>, <div class="source">JoinMapÂ®4, software for the calculation of genetic linkage maps in experimental populations</div>Â . 
<div class="publisher-name">Kyazma BV</div>, 
<div class="publisher-loc">Wageningen, The Netherlands</div>.</div></div></div></div><div content-id="dsx020-B58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx020-B58" class="ref-content "><div class="label -label">58</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Endelman</div>
               <div class="given-names">J.B.</div>
            </div>, <div class="name string-name"><div class="surname">Plomion</div> <div class="given-names">C.</div></div></div>
          <div class="year">2014</div>, 
<div class="article-title">LPmerge: an R package for merging genetic maps by linear programming,</div><div class="source">Bioinformatics</div>Â , <div class="volume">30</div>, <div class="fpage">1623</div>â€“<div class="lpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=LPmerge%3A%20an%20R%20package%20for%20merging%20genetic%20maps%20by%20linear%20programming%2C&amp;author=J.B.%20Endelman&amp;author=C.%20Plomion&amp;publication_year=2014&amp;journal=Bioinformatics&amp;volume=30&amp;pages=1623-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btu091" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtu091" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtu091"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24532720" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx020-B59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx020-B59" class="ref-content "><div class="label -label">59</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Voorrips</div>
               <div class="given-names">R.E.</div>
            </div>
          </div>
          <div class="year">2002</div>, 
<div class="article-title">MapChart: software for the graphical presentation of linkage maps and QTLs,</div><div class="source">J. Hered</div>Â ., <div class="volume">93</div>, <div class="fpage">77</div>â€“<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MapChart%3A%20software%20for%20the%20graphical%20presentation%20of%20linkage%20maps%20and%20QTLs%2C&amp;author=R.E.%20Voorrips&amp;publication_year=2002&amp;journal=J.%20Hered&amp;volume=93&amp;pages=77-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jhered/93.1.77" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjhered%2F93.1.77" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjhered%2F93.1.77"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12011185" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx020-B60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx020-B60" class="ref-content "><div class="label -label">60</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Krzywinski</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Schein</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Birol</div> <div class="given-names">I.</div></div></div>, et al. Â <div class="year">2009</div>, 
<div class="article-title">Circos: an information aesthetic for comparative genomics,</div><div class="source">Genome Res</div>Â ., <div class="volume">19</div>, <div class="fpage">1639</div>â€“<div class="lpage">45</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Circos%3A%20an%20information%20aesthetic%20for%20comparative%20genomics%2C&amp;author=M.%20Krzywinski&amp;author=J.%20Schein&amp;author=I.%20Birol&amp;publication_year=2009&amp;journal=Genome%20Res&amp;volume=19&amp;pages=1639-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.092759.109" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.092759.109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.092759.109"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19541911" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx020-B61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx020-B61" class="ref-content "><div class="label -label">61</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Delcher</div>
               <div class="given-names">A.L.</div>
            </div>, <div class="name string-name"><div class="surname">Phillippy</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Carlton</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Salzberg</div> <div class="given-names">S.L.</div></div></div>
          <div class="year">2002</div>, 
<div class="article-title">Fast algorithms for large-scale genome alignment and comparison,</div><div class="source">Nucl. Acids Res</div>Â ., <div class="volume">30</div>, <div class="fpage">2478</div>â€“<div class="lpage">83</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fast%20algorithms%20for%20large-scale%20genome%20alignment%20and%20comparison%2C&amp;author=A.L.%20Delcher&amp;author=A.%20Phillippy&amp;author=J.%20Carlton&amp;author=S.L.%20Salzberg&amp;publication_year=2002&amp;journal=Nucl.%20Acids%20Res&amp;volume=30&amp;pages=2478-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/30.11.2478" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F30.11.2478" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F30.11.2478"> </span></div>Â </div></div></div></div></div><div content-id="dsx020-B62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx020-B62" class="ref-content "><div class="label -label">62</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Cingolani</div>
               <div class="given-names">P.</div>
            </div>, <div class="name string-name"><div class="surname">Platts</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">L.L.</div></div></div>, et al. Â <div class="year">2012</div>, 
<div class="article-title">A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of <em>Drosophila melanogaster</em> strain <em>w</em><sup>1118</sup>; <em>iso</em>-2; <em>iso</em>-3</div>, <div class="source">Fly</div>Â , <div class="volume">6</div>, <div class="fpage">80</div>â€“<div class="lpage">92</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20program%20for%20annotating%20and%20predicting%20the%20effects%20of%20single%20nucleotide%20polymorphisms%2C%20SnpEff%3A%20SNPs%20in%20the%20genome%20of%20Drosophila%20melanogaster%20strain%20w1118%3B%20iso-2%3B%20iso-3&amp;author=P.%20Cingolani&amp;author=A.%20Platts&amp;author=L.L.%20Wang&amp;publication_year=2012&amp;journal=Fly&amp;volume=6&amp;pages=80-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.4161/fly.19695" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.4161%2Ffly.19695" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.4161%2Ffly.19695"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22728672" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx020-B63" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx020-B63" class="ref-content "><div class="label -label">63</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Eilbeck</div>
               <div class="given-names">K.</div>
            </div>, <div class="name string-name"><div class="surname">Lewis</div> <div class="given-names">S.E.</div></div>, <div class="name string-name"><div class="surname">Mungall</div> <div class="given-names">C.J.</div></div></div>, et al. Â <div class="year">2005</div>, 
<div class="article-title">The Sequence Ontology: a tool for the unification of genome annotations,</div><div class="source">Genome Biol</div>Â ., <div class="volume">6</div>, <div class="fpage">R44</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Sequence%20Ontology%3A%20a%20tool%20for%20the%20unification%20of%20genome%20annotations%2C&amp;author=K.%20Eilbeck&amp;author=S.E.%20Lewis&amp;author=C.J.%20Mungall&amp;publication_year=2005&amp;journal=Genome%20Biol&amp;volume=6&amp;pages=R44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2005-6-5-r44" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2005-6-5-r44" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2005-6-5-r44"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15892872" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx020-B64" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx020-B64" class="ref-content "><div class="label -label">64</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Xie</div>
               <div class="given-names">C.</div>
            </div>, <div class="name string-name"><div class="surname">Tammi</div> <div class="given-names">M.T.</div></div></div>
          <div class="year">2009</div>, 
<div class="article-title">CNV-seq, a new method to detect copy number variation using high-throughput sequencing,</div><div class="source">BMC Bioinform</div>Â ., <div class="volume">10</div>, <div class="fpage">80</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CNV-seq%2C%20a%20new%20method%20to%20detect%20copy%20number%20variation%20using%20high-throughput%20sequencing%2C&amp;author=C.%20Xie&amp;author=M.T.%20Tammi&amp;publication_year=2009&amp;journal=BMC%20Bioinform&amp;volume=10&amp;pages=80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-10-80" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-10-80" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-10-80"> </span></div>Â </div></div></div></div></div><div content-id="dsx020-B65" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx020-B65" class="ref-content "><div class="label -label">65</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Thiel</div>
               <div class="given-names">T.</div>
            </div>, <div class="name string-name"><div class="surname">Kota</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Grosse</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Stein</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Graner</div> <div class="given-names">A.</div></div></div>
          <div class="year">2004</div>, 
<div class="article-title">SNP2CAPS: a SNP and INDEL analysis tool for CAPS marker development,</div><div class="source">Nucl. Acids Res</div>Â ., <div class="volume">32</div>, <div class="fpage">e5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SNP2CAPS%3A%20a%20SNP%20and%20INDEL%20analysis%20tool%20for%20CAPS%20marker%20development%2C&amp;author=T.%20Thiel&amp;author=R.%20Kota&amp;author=I.%20Grosse&amp;author=N.%20Stein&amp;author=A.%20Graner&amp;publication_year=2004&amp;journal=Nucl.%20Acids%20Res&amp;volume=32&amp;pages=e5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gnh006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgnh006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgnh006"> </span></div>Â </div></div></div></div></div><div content-id="dsx020-B66" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx020-B66" class="ref-content "><div class="label -label">66</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kofler</div>
               <div class="given-names">R.</div>
            </div>, <div class="name string-name"><div class="surname">Schlotterer</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Lelley</div> <div class="given-names">T.</div></div></div>
          <div class="year">2007</div>, 
<div class="article-title">SciRoKo: a new tool for whole genome microsatellite search and investigation,</div><div class="source">Bioinformatics</div>Â , <div class="volume">23</div>, <div class="fpage">1683</div>â€“<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SciRoKo%3A%20a%20new%20tool%20for%20whole%20genome%20microsatellite%20search%20and%20investigation%2C&amp;author=R.%20Kofler&amp;author=C.%20Schlotterer&amp;author=T.%20Lelley&amp;publication_year=2007&amp;journal=Bioinformatics&amp;volume=23&amp;pages=1683-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btm157" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtm157" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtm157"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17463017" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx020-B67" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx020-B67" class="ref-content "><div class="label -label">67</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Rozen</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Skaletsky</div> <div class="given-names">H.</div></div></div>
          <div class="year">2000</div>, 
<div class="article-title">Primer3 on the WWW for general users and for biologist programmers,</div><div class="source">Methods Mol. Biol</div>Â ., <div class="volume">132</div>, <div class="fpage">365</div>â€“<div class="lpage">86</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Primer3%20on%20the%20WWW%20for%20general%20users%20and%20for%20biologist%20programmers%2C&amp;author=S.%20Rozen&amp;author=H.%20Skaletsky&amp;publication_year=2000&amp;journal=Methods%20Mol.%20Biol&amp;volume=132&amp;pages=365-86" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10547847" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx020-B68" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx020-B68" class="ref-content "><div class="label -label">68</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Arumuganathan</div>
               <div class="given-names">K.</div>
            </div>, <div class="name string-name"><div class="surname">Earle</div> <div class="given-names">E.D.</div></div></div>
          <div class="year">1991</div>, 
<div class="article-title">Nuclear DNA content of some important plant,</div><div class="source">Plant Mol. Biol. Rep</div>Â ., <div class="volume">9</div>, <div class="fpage">208</div>â€“<div class="lpage">18</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nuclear%20DNA%20content%20of%20some%20important%20plant%2C&amp;author=K.%20Arumuganathan&amp;author=E.D.%20Earle&amp;publication_year=1991&amp;journal=Plant%20Mol.%20Biol.%20Rep&amp;volume=9&amp;pages=208-18" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF02672069" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF02672069" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF02672069"> </span></div>Â </div></div></div></div></div><div content-id="dsx020-B69" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx020-B69" class="ref-content "><div class="label -label">69</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Eilbeck</div>
               <div class="given-names">K.</div>
            </div>, <div class="name string-name"><div class="surname">Moore</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Holt</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Yandell</div> <div class="given-names">M.</div></div></div>
          <div class="year">2009</div>, 
<div class="article-title">Quantitative measures for the management and comparison of annotated genomes</div>. <div class="source">BMC Bioinform</div>Â ., <div class="volume">10</div>, <div class="fpage">67</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Quantitative%20measures%20for%20the%20management%20and%20comparison%20of%20annotated%20genomes&amp;author=K.%20Eilbeck&amp;author=B.%20Moore&amp;author=C.%20Holt&amp;author=M.%20Yandell&amp;publication_year=2009&amp;journal=BMC%20Bioinform&amp;volume=10&amp;pages=67" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-10-67" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-10-67" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-10-67"> </span></div>Â </div></div></div></div></div><div content-id="dsx020-B70" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx020-B70" class="ref-content "><div class="label -label">70</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Cao</div>
               <div class="given-names">K.</div>
            </div>, <div class="name string-name"><div class="surname">Zhou</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">Q.</div></div></div>, et al. Â <div class="year">2016</div>, 
<div class="article-title">Genome-wide association study of 12 agronomic traits in peach,</div><div class="source">Nat. Commun</div>Â ., <div class="volume">7</div>, <div class="fpage">13246</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20association%20study%20of%2012%20agronomic%20traits%20in%20peach%2C&amp;author=K.%20Cao&amp;author=Z.%20Zhou&amp;author=Q.%20Wang&amp;publication_year=2016&amp;journal=Nat.%20Commun&amp;volume=7&amp;pages=13246" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ncomms13246" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fncomms13246" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fncomms13246"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27824331" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx020-B71" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx020-B71" class="ref-content "><div class="label -label">71</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Asamizu</div>
               <div class="given-names">E.</div>
            </div>, <div class="name string-name"><div class="surname">Ichihara</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Nakaya</div> <div class="given-names">A.</div></div></div>, et al. Â <div class="year">2014</div>, 
<div class="article-title">Plant Genome DataBase Japan (PGDBj): a portal website for the integration of plant genome-related databases,</div><div class="source">Plant Cell Physiol</div>Â ., <div class="volume">55</div>, <div class="fpage">e8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Plant%20Genome%20DataBase%20Japan%20(PGDBj)%3A%20a%20portal%20website%20for%20the%20integration%20of%20plant%20genome-related%20databases%2C&amp;author=E.%20Asamizu&amp;author=H.%20Ichihara&amp;author=A.%20Nakaya&amp;publication_year=2014&amp;journal=Plant%20Cell%20Physiol&amp;volume=55&amp;pages=e8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/pcp/pct189" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fpcp%2Fpct189" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fpcp%2Fpct189"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24363285" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx020-B72" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx020-B72" class="ref-content "><div class="label -label">72</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Iwata</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Minamikawa</div> <div class="given-names">M.F.</div></div>, <div class="name string-name"><div class="surname">Kajiya-Kanegae</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Ishimori</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Hayashi</div> <div class="given-names">T.</div></div></div>
          <div class="year">2016</div>, 
<div class="article-title">Genomics-assisted breeding in fruit trees</div>, <div class="source">Breed. Sci</div>Â ., <div class="volume">66</div>, <div class="fpage">100</div>â€“<div class="lpage">15</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomics-assisted%20breeding%20in%20fruit%20trees&amp;author=H.%20Iwata&amp;author=M.F.%20Minamikawa&amp;author=H.%20Kajiya-Kanegae&amp;author=M.%20Ishimori&amp;author=T.%20Hayashi&amp;publication_year=2016&amp;journal=Breed.%20Sci&amp;volume=66&amp;pages=100-15" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1270/jsbbs.66.100" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1270%2Fjsbbs.66.100" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1270%2Fjsbbs.66.100"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27069395" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx020-B73" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx020-B73" class="ref-content "><div class="label -label">73</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Lusser</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Parisi</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Plan</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">RodrÃ­guez-Cerezo</div> <div class="given-names">E.</div></div></div>
          <div class="year">2012</div>, 
<div class="article-title">Deployment of new biotechnologies in plant breeding,</div><div class="source">Nat. Biotechnol</div>Â ., <div class="volume">30</div>, <div class="fpage">231</div>â€“<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Deployment%20of%20new%20biotechnologies%20in%20plant%20breeding%2C&amp;author=M.%20Lusser&amp;author=C.%20Parisi&amp;author=D.%20Plan&amp;author=E.%20Rodr%C3%ADguez-Cerezo&amp;publication_year=2012&amp;journal=Nat.%20Biotechnol&amp;volume=30&amp;pages=231-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.2142" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.2142" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.2142"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22398616" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx020-B74" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx020-B74" class="ref-content "><div class="label -label">74</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Iwata</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Hayashi</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Terakami</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Takada</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Saito</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Yamamoto</div> <div class="given-names">T.</div></div></div>
          <div class="year">2013</div>, 
<div class="article-title">Genomic prediction of trait segregation in a progeny population: a case study of Japanese pear (<em>Pyrus pyrifolia</em>)</div>, <div class="source">BMC Genet</div>Â ., <div class="volume">14</div>, <div class="fpage">81</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20prediction%20of%20trait%20segregation%20in%20a%20progeny%20population%3A%20a%20case%20study%20of%20Japanese%20pear%20(Pyrus%20pyrifolia)&amp;author=H.%20Iwata&amp;author=T.%20Hayashi&amp;author=S.%20Terakami&amp;author=N.%20Takada&amp;author=T.%20Saito&amp;author=T.%20Yamamoto&amp;publication_year=2013&amp;journal=BMC%20Genet&amp;volume=14&amp;pages=81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2156-14-81" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2156-14-81" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2156-14-81"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24028660" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dsx020-B75" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx020-B75" class="ref-content "><div class="label -label">75</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Yamagishi</div>
               <div class="given-names">N.</div>
            </div>, <div class="name string-name"><div class="surname">Kishigami</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Yoshikawa</div> <div class="given-names">N.</div></div></div>
          <div class="year">2014</div>, 
<div class="article-title">Reduced generation time of apple seedlings to within a year by means of a plant virus vector: a new plant-breeding technique with no transmission of genetic modification to the next generation,</div><div class="source">Plant Biotechnol. J</div>Â ., <div class="volume">12</div>, <div class="fpage">60</div>â€“<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Reduced%20generation%20time%20of%20apple%20seedlings%20to%20within%20a%20year%20by%20means%20of%20a%20plant%20virus%20vector%3A%20a%20new%20plant-breeding%20technique%20with%20no%20transmission%20of%20genetic%20modification%20to%20the%20next%20generation%2C&amp;author=N.%20Yamagishi&amp;author=R.%20Kishigami&amp;author=N.%20Yoshikawa&amp;publication_year=2014&amp;journal=Plant%20Biotechnol.%20J&amp;volume=12&amp;pages=60-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/pbi.12116" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fpbi.12116" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fpbi.12116"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23998891" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">Â© The Author 2017. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<a class="link link-uri" href="http://creativecommons.org/licenses/by/4.0/" target="_blank">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=3835854&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx020/2/dsx020_supp_sweetcherrygenome-supplementaryfigures.r1.pdf?Expires=2147483647&amp;Signature=EpzskbJ8FdkxtePPGHzVUHR8wbz7PuhsMtIZp1mBYfuYpDnoU860lHAiOXaTih335VqggdpxNmN~DlHv9jCQjL67NgdKEe0mJf6VW~rFFNlKSOSwGF16QSU~ixooLa4MnFfPBuizWoRuWKHC5qHhWq9m7g0mIOI~a3cC4r-I38M7qqsg8TxxRMqONuq~9wlcV5s9xtt-0Ia6Fmkgv9vqIrt-f~ik0NUXpBMR6Br6pBv9tJQJvkf47kQLiRbDw4zZbNs27lnG2PNg44Zj75jgFNV8z2KYtBTZOOPF3qb1rjsE96U2cnvlArNo-eTe~6GEVJGGYM0KqyUXlBDoAbMxJA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx020/2/dsx020_supp_sweetcherrygenome-supplementaryinformation.r1.pdf?Expires=2147483647&amp;Signature=XFws~d4jZjNfBNsaOnnmKXsMcz4jkFjIH5zugTNI6sFgBrlDa30GJcFfXXLenHUiCDiGeYhcZ~wDXdXLwAANIlAFe6RDnfyyv6Fs7g4e633wcgVjayMVcPKVfn1J6AGX32HS05qOUga1J5SIKv0vMrUW~WBxwP82E8sMZfXVwKPVwdqxd35Xs-TgnMJrl~xZHK-4RkFO2MK3eqPyus8qRiUnDgv-qjQklxyGKzpW68wX4v1kuaZKrn7~wYECKVnx-6Dzm9NoV3YqBUhpYJBvcrfdRgvopij2lPPSzFmqnixrcvzTGeKjVPr3xA1yRbh4R~tV3gEw4viEQ7OnJm8pWg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx020/2/dsx020_supp_sweetcherrygenome-supplementarytables.xlsx?Expires=2147483647&amp;Signature=TiRI9J~ZD4Ek9u56ktrPJXS93YJ4qGZITEQupuOKHddrgtrVQf74I2lHj50mkRJPDXcmwnkAlCHj9HYjSBnVb8OnhoJc25aktrGAXjUDZcKUAXlvLorEGDIAClU8ElBJEjfTSI1yf5-T0oDWKmgrB9I-8PYPawIiAr1bgxbvGop8yZ-gE8uZ7G~74R34kcc7nlU~gZIU~A1VLoZ6ZuovxkZkNThTFm-2zgGuUVmFAaVx7kf5xgIIORjp0lE5fsPZfEqFgiOU-cWgn0OvhePFZCdcpGQui5BaYguk9XGf4Rdo~tcDsqyUbiIqTg8O3WduuPXBd3HFTNFWXD~CzRY5BQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - xlsx file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">3,978</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000413018400005&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">12</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsx020"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="3835854"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000413018400005&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f24%2f5%2f499%2f3835854">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000413018400005&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (12)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F24%2F5%2F499%2F3835854" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/3835854'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transpositionâ€“duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="genome sequence of sweet cherry (Prunus avium) for use in genomics-assisted breeding" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsx020");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f24%2f5%2f499%2f3835854">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

