

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/24/2/193/2731026","@type":"ScholarlyArticle","name":"Function and evolution of a  Lotus japonicus  AP2/ERF family transcription factor that is required for development of infection threads","datePublished":"2016-12-21","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/24/2","@type":"PublicationIssue","issueNumber":"2","datePublished":"2017-04-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsw052","keywords":["infection thread development","ERN1","<italic>Lotus japonicus</italic>","molecular evolutionary analysis"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/24/2/193/2731026","author":[{"name":"Yano, Koji","affiliation":"1 National Institute of Agrobiological Sciences, Tsukuba, Ibaraki 305-8602, Japan   2 Division of Symbiotic Systems, National Institute for Basic Biology, National Institute for Natural Sciences, Okazaki 444-8585, Japan","@type":"Person"},{"name":"Aoki, Seishiro","affiliation":"3 Department of General Systems Studies, Graduate School of Arts and Sciences, the University of Tokyo, Meguro-ku, Tokyo 153-8902, Japan   4 Department of Biological Sciences, Graduate School of Science, the University of Tokyo, Bunkyo-ku, Tokyo 113-0032, Japan","@type":"Person"},{"name":"Liu, Meng","affiliation":"2 Division of Symbiotic Systems, National Institute for Basic Biology, National Institute for Natural Sciences, Okazaki 444-8585, Japan   5 Department of Basic Biology, School of Life Science, Graduate University for Advanced Studies (SOKENDAI), Okazaki 444-8585, Japan and","@type":"Person"},{"name":"Umehara, Yosuke","affiliation":"1 National Institute of Agrobiological Sciences, Tsukuba, Ibaraki 305-8602, Japan","@type":"Person"},{"name":"Suganuma, Norio","affiliation":"6 Department of Life Science, Aichi University of Education, Kariya, Aichi 448–8542, Japan","@type":"Person"},{"name":"Iwasaki, Wataru","affiliation":"4 Department of Biological Sciences, Graduate School of Science, the University of Tokyo, Bunkyo-ku, Tokyo 113-0032, Japan","@type":"Person"},{"name":"Sato, Shusei","affiliation":"7 Kazusa DNA Research Institute, Kisarazu, Chiba 292–0812, Japan   8 Graduate School of Life Sciences, Tohoku University, Aoba-ku, Sendai 980-8577, Japan","@type":"Person"},{"name":"Soyano, Takashi","affiliation":"2 Division of Symbiotic Systems, National Institute for Basic Biology, National Institute for Natural Sciences, Okazaki 444-8585, Japan   5 Department of Basic Biology, School of Life Science, Graduate University for Advanced Studies (SOKENDAI), Okazaki 444-8585, Japan and","@type":"Person"},{"name":"Kouchi, Hiroshi","affiliation":"1 National Institute of Agrobiological Sciences, Tsukuba, Ibaraki 305-8602, Japan","@type":"Person"},{"name":"Kawaguchi, Masayoshi","affiliation":"2 Division of Symbiotic Systems, National Institute for Basic Biology, National Institute for Natural Sciences, Okazaki 444-8585, Japan   5 Department of Basic Biology, School of Life Science, Graduate University for Advanced Studies (SOKENDAI), Okazaki 444-8585, Japan and","@type":"Person"}],"description":"Abstract.  Legume-rhizobium symbiosis is achieved by two major events evolutionarily acquired: root hair infection and organogenesis. Infection thread (IT) deve","pageStart":"193","pageEnd":"203","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/2/2/m_dnares_24_2cover.png?Expires=2147483647&Signature=C8sLyIyfepNN-Oy5BR2xLoVJLx2W1jQBt~48a2lQ~mjys3EWTAKP5k2kV~t7b8sG~Qk1B4iVdz8BINFP5bR-Hc0gvwqekGHM8qavh8kDaLAfN~qNxrDj7NNeLHJ-IURMi5v1ATu23u5wB01wP27BHknbA3W9JgGnq6AVGPqeLfhK7AJcTzOYzKpGYwaedh5crkYXYHp~9GCcHuTjoInf3sKaYC2Ew-HnuA-yIL45j8ocXrNQuIxJV8rFXwx1Tt0b7I1ZJg09R4-7m-GZB9vPgdehgAXUMUNQ4gDatBY22xUt9COqnlb7RZKNUXFDo7Qe0LNJgSHwNUOJvsh-SLpXkg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Function and evolution of a  Lotus japonicus  AP2/ERF family transcription factor that is required for development of infection threads","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/2/2/m_dnares_24_2cover.png?Expires=2147483647&Signature=C8sLyIyfepNN-Oy5BR2xLoVJLx2W1jQBt~48a2lQ~mjys3EWTAKP5k2kV~t7b8sG~Qk1B4iVdz8BINFP5bR-Hc0gvwqekGHM8qavh8kDaLAfN~qNxrDj7NNeLHJ-IURMi5v1ATu23u5wB01wP27BHknbA3W9JgGnq6AVGPqeLfhK7AJcTzOYzKpGYwaedh5crkYXYHp~9GCcHuTjoInf3sKaYC2Ew-HnuA-yIL45j8ocXrNQuIxJV8rFXwx1Tt0b7I1ZJg09R4-7m-GZB9vPgdehgAXUMUNQ4gDatBY22xUt9COqnlb7RZKNUXFDo7Qe0LNJgSHwNUOJvsh-SLpXkg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Function and evolution of a  Lotus japonicus  AP2/ERF family transcription factor that is required for development of infection threads" />
    <meta property="og:description" content="Abstract.  Legume-rhizobium symbiosis is achieved by two major events evolutionarily acquired: root hair infection and organogenesis. Infection thread (IT) deve" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/24/2/193/2731026" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2016-12-21" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Yano, Koji" /><meta name="citation_author_institution" content="National Institute of Agrobiological Sciences, Tsukuba, Ibaraki 305-8602, Japan" /><meta name="citation_author_institution" content="Division of Symbiotic Systems, National Institute for Basic Biology, National Institute for Natural Sciences, Okazaki 444-8585, Japan" /><meta name="citation_author" content="Aoki, Seishiro" /><meta name="citation_author_institution" content="Department of General Systems Studies, Graduate School of Arts and Sciences, the University of Tokyo, Meguro-ku, Tokyo 153-8902, Japan" /><meta name="citation_author_institution" content="Department of Biological Sciences, Graduate School of Science, the University of Tokyo, Bunkyo-ku, Tokyo 113-0032, Japan" /><meta name="citation_author" content="Liu, Meng" /><meta name="citation_author_institution" content="Division of Symbiotic Systems, National Institute for Basic Biology, National Institute for Natural Sciences, Okazaki 444-8585, Japan" /><meta name="citation_author_institution" content="Department of Basic Biology, School of Life Science, Graduate University for Advanced Studies (SOKENDAI), Okazaki 444-8585, Japan and " /><meta name="citation_author" content="Umehara, Yosuke" /><meta name="citation_author_institution" content="National Institute of Agrobiological Sciences, Tsukuba, Ibaraki 305-8602, Japan" /><meta name="citation_author" content="Suganuma, Norio" /><meta name="citation_author_institution" content="Department of Life Science, Aichi University of Education, Kariya, Aichi 448–8542, Japan" /><meta name="citation_author" content="Iwasaki, Wataru" /><meta name="citation_author_institution" content="Department of Biological Sciences, Graduate School of Science, the University of Tokyo, Bunkyo-ku, Tokyo 113-0032, Japan" /><meta name="citation_author" content="Sato, Shusei" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, Kisarazu, Chiba 292–0812, Japan" /><meta name="citation_author_institution" content="Graduate School of Life Sciences, Tohoku University, Aoba-ku, Sendai 980-8577, Japan" /><meta name="citation_author" content="Soyano, Takashi" /><meta name="citation_author_institution" content="Division of Symbiotic Systems, National Institute for Basic Biology, National Institute for Natural Sciences, Okazaki 444-8585, Japan" /><meta name="citation_author_institution" content="Department of Basic Biology, School of Life Science, Graduate University for Advanced Studies (SOKENDAI), Okazaki 444-8585, Japan and " /><meta name="citation_author" content="Kouchi, Hiroshi" /><meta name="citation_author_institution" content="National Institute of Agrobiological Sciences, Tsukuba, Ibaraki 305-8602, Japan" /><meta name="citation_author" content="Kawaguchi, Masayoshi" /><meta name="citation_author_institution" content="Division of Symbiotic Systems, National Institute for Basic Biology, National Institute for Natural Sciences, Okazaki 444-8585, Japan" /><meta name="citation_author_institution" content="Department of Basic Biology, School of Life Science, Graduate University for Advanced Studies (SOKENDAI), Okazaki 444-8585, Japan and " /><meta name="citation_title" content="Function and evolution of a Lotus japonicus AP2/ERF family transcription factor that is required for development of infection threads" /><meta name="citation_firstpage" content="193" /><meta name="citation_lastpage" content="203" /><meta name="citation_doi" content="10.1093/dnares/dsw052" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="24" /><meta name="citation_issue" content="2" /><meta name="citation_publication_date" content="2017/04/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Mechanism of infection thread elongation in root hairs of Medicago truncatula and dynamic interplay with associated rhizobial colonization,;  citation_journal_title=Plant Physiol;  citation_year=2008;  citation_volume=148;  citation_pages=1985-95; " /><meta name="citation_reference" content="citation_title=Actinorhizal root nodule symbioses: what is signalling telling on the origins of nodulation?,;  citation_journal_title=Curr. Opin. Plant Biol;  citation_year=2014;  citation_volume=20;  citation_pages=11-18; " /><meta name="citation_reference" content="citation_title=Plant responses to nodulation factors,;  citation_journal_title=Curr. Opin. Plant Biol;  citation_year=1999;  citation_volume=2;  citation_pages=483-9; " /><meta name="citation_reference" content="citation_title=Calcium spiking in plant root hairs responding to Rhizobium nodulation signals,;  citation_journal_title=Cell;  citation_year=1996;  citation_volume=85;  citation_pages=673-81; " /><meta name="citation_reference" content="citation_title=The NFP locus of Medicago truncatula controls an early step of Nod factor signal transduction upstream of a rapid calcium flux and root hair deformation,;  citation_journal_title=Plant J;  citation_year=2003;  citation_volume=34;  citation_pages=495-506; " /><meta name="citation_reference" content="citation_title=Nod factor-induced root hair curling: Continuous polar growth towards the point of nod factor application,;  citation_journal_title=Plant Physiol;  citation_year=2003;  citation_volume=132;  citation_pages=1982-1988; " /><meta name="citation_reference" content="citation_title=A receptor kinase gene of the LysM type is involved in legume perception of rhizobial signals,;  citation_journal_title=Nature;  citation_year=2003;  citation_volume=425;  citation_pages=637-40; " /><meta name="citation_reference" content="citation_title=Plant recognition of symbiotic bacteria requires two LysM receptor-like kinases,;  citation_journal_title=Nature;  citation_year=2003;  citation_volume=425;  citation_pages=585-92; " /><meta name="citation_reference" content="citation_title=Medicago LYK3, an entry receptor in rhizobial nodulation factor signaling,;  citation_journal_title=Plant Physiol;  citation_year=2007;  citation_volume=145;  citation_pages=183-91; " /><meta name="citation_reference" content="citation_title=Legume receptors perceive the rhizobial lipochitin oligosaccharide signal molecules by direct binding,;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2012;  citation_volume=109;  citation_pages=13859-64; " /><meta name="citation_reference" content="citation_title=Evolution of signal transduction in intracellular symbiosis,;  citation_journal_title=Trends Plant Sci;  citation_year=2002;  citation_volume=7;  citation_pages=511-8; " /><meta name="citation_reference" content="citation_title=Nodulation independent of rhizobia induced by a calcium-activated kinase lacking autoinhibition,;  citation_journal_title=Nature;  citation_year=2006;  citation_volume=441;  citation_pages=1149-52; " /><meta name="citation_reference" content="citation_title=Deregulation of a Ca2+/calmodulin-dependent kinase leads to spontaneous nodule development,;  citation_journal_title=Nature;  citation_year=2006;  citation_volume=441;  citation_pages=1153-6; " /><meta name="citation_reference" content="citation_title=A dominant function of CCaMK in intracellular accommodation of bacterial and fungal endosymbionts,;  citation_journal_title=Plant J;  citation_year=2010;  citation_volume=63;  citation_pages=141-54; " /><meta name="citation_reference" content="citation_title=The molecular network governing nodule organogenesis and infection in the model legume Lotus japonicus,;  citation_journal_title=Nat. Commun;  citation_year=2010;  citation_volume=1;  citation_pages=10" /><meta name="citation_reference" content="citation_title=Transcriptomics of actinorhizal symbioses reveals homologs of the whole common symbiotic signaling cascade,;  citation_journal_title=Plant Physiol;  citation_year=2011;  citation_volume=156;  citation_pages=700-11; " /><meta name="citation_reference" content="citation_title=Comparison of the nodule vs. root transcriptome of the actinorhizal plant Datisca glomerata: actinorhizal nodules contain a specific class of defensins,;  citation_journal_title=PLoS One;  citation_year=2013;  citation_volume=8;  citation_pages=e72442" /><meta name="citation_reference" content="citation_title=Identification of potential transcriptional regulators of actinorhizal symbioses in Casuarina glauca and Alnus glutinosa,;  citation_journal_title=BMC Plant Biol;  citation_year=2014;  citation_volume=14;  citation_pages=342" /><meta name="citation_reference" content="citation_title=SymRK defines a common genetic basis for plant root endosymbioses with arbuscular mycorrhiza fungi, rhizobia, and Frankia bacteria,;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2008;  citation_volume=105;  citation_pages=4928-32; " /><meta name="citation_reference" content="citation_title=Functional adaptation of a plant receptor-kinase paved the way for the evolution of intracellular root symbioses with bacteria,;  citation_journal_title=PLoS Biol;  citation_year=2008;  citation_volume=6;  citation_pages=e68" /><meta name="citation_reference" content="citation_title=The independent acquisition of plant root nitrogen-fixing symbiosis in Fabids recruited the same genetic pathway for nodule organogenesis,;  citation_journal_title=PLoS One;  citation_year=2013;  citation_volume=8;  citation_pages=e64515" /><meta name="citation_reference" content="citation_title=A plant regulator controlling development of symbiotic root nodules,;  citation_journal_title=Nature;  citation_year=1999;  citation_volume=402;  citation_pages=191-5; " /><meta name="citation_reference" content="citation_title=Medicago truncatula NIN is essential for rhizobial-independent nodule organogenesis induced by autoactive calcium/calmodulin-dependent protein kinase,;  citation_journal_title=Plant Physiol;  citation_year=2007;  citation_volume=144;  citation_pages=324-35; " /><meta name="citation_reference" content="citation_title=The Casuarina NIN gene is transcriptionally activated throughout Frankia root infection as well as in response to bacterial diffusible signals,;  citation_journal_title=New Phytol;  citation_year=2015;  citation_volume=208;  citation_pages=887-903; " /><meta name="citation_reference" content="citation_title=Lipochitoologosaccharides modulate plant host immunity to enable endosymbiosis,;  citation_journal_title=Annu. Rev. Phytopathol;  citation_year=2015;  citation_volume=53;  citation_pages=311-4; " /><meta name="citation_reference" content="citation_title=Evolution of a symbiotic receptor through gene duplications in the legume-rhizobium mutualism,;  citation_journal_title=New Phytol;  citation_year=2014;  citation_volume=201;  citation_pages=961-72; " /><meta name="citation_reference" content="citation_title=From β- to α-Proteobacteria: The origin and evolution of rhizobial nodulation genes nodIJ,;  citation_journal_title=Mol. Biol. Evol;  citation_year=2013;  citation_volume=30;  citation_pages=2494-508; " /><meta name="citation_reference" content="citation_title=Multiple polyploidy events in the early radiation of nodulating and nonnodulating legumes,;  citation_journal_title=Mol. Biol. Evol;  citation_year=2015;  citation_volume=32;  citation_pages=193-210; " /><meta name="citation_reference" content="citation_title=Comparative genomics suggests that an ancestral polyploidy event leads to enhanced root nodule symbiosis in the Papilionoideae,;  citation_journal_title=Mol. Biol. Evol;  citation_year=2013;  citation_volume=30;  citation_pages=2602-11; " /><meta name="citation_reference" content="citation_title=CYCLOPS, a mediator of symbiotic intracellular accommodation,;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2008;  citation_volume=105;  citation_pages=20540-5; " /><meta name="citation_reference" content="citation_title=CERBERUS, a novel U-box protein containing WD-40 repeats, is required for formation of the infection thread and nodule development in the legume-Rhizobium symbiosis,;  citation_journal_title=Plant J;  citation_year=2009;  citation_volume=60;  citation_pages=168-80; " /><meta name="citation_reference" content="citation_title=CYCLOPS, a DNA-binding transcriptional activator, orchestrates symbiotic root nodule development,;  citation_journal_title=Cell Host Microbe;  citation_year=2014;  citation_volume=15;  citation_pages=139-52; " /><meta name="citation_reference" content="citation_title=A novel nuclear protein interacts with the symbiotic DMI3 calcium- and calmodulin-dependent protein kinase of Medicago truncatula,;  citation_journal_title=Mol. Plant Microbe. Interact;  citation_year=2007;  citation_volume=20;  citation_pages=912-21; " /><meta name="citation_reference" content="citation_title=LIN, a novel type of U-box/WD40 protein, controls early infection by rhizobia in legumes,;  citation_journal_title=Plant Physiol;  citation_year=2009;  citation_volume=151;  citation_pages=1239-49; " /><meta name="citation_reference" content="citation_title=An ERF transcription factor in Medicago truncatula that is essential for nod factor signal transduction,;  citation_journal_title=Plant Cell;  citation_year=2007;  citation_volume=19;  citation_pages=1221-34; " /><meta name="citation_reference" content="citation_title=AP2-ERF transcription factors mediate Nod factor dependent MtENOD11 activation in root hairs via a novel cis-regulatory motif,;  citation_journal_title=Plant Cell;  citation_year=2007;  citation_volume=19;  citation_pages=2866-85; " /><meta name="citation_reference" content="citation_title=Medicago truncatula ERN transcription factors: Regulatory interplay with NSP1/NSP2 GRAS factors and expression dynamics throughout rhizobial infection,;  citation_journal_title=Plant Physiol;  citation_year=2012;  citation_volume=160;  citation_pages=2155-72; " /><meta name="citation_reference" content="citation_title=TOO MUCH LOVE, a root regulator associated with the long-distance control of nodulation in Lotus japonicus,;  citation_journal_title=Mol. Plant Microbe. Interact;  citation_year=2009;  citation_volume=22;  citation_pages=259-68; " /><meta name="citation_reference" content="citation_title=Root, root hair, and symbiotic mutants of the model legume Lotus japonicus,;  citation_journal_title=Mol. Plant-Microbe Interact;  citation_year=2002;  citation_volume=15;  citation_pages=17-26; " /><meta name="citation_reference" content="citation_title=Control of leghaemoglobin synthesis in snake beans,;  citation_journal_title=Biochem J;  citation_year=1971;  citation_volume=125;  citation_pages=1075-80; " /><meta name="citation_reference" content="citation_title=Gibberellin controls the nodulation signaling pathway in Lotus japonicus,;  citation_journal_title=Plant J;  citation_year=2009;  citation_volume=58;  citation_pages=183-94; " /><meta name="citation_reference" content="citation_publisher=Springer, Dordrecht;  citation_title=Lotus japonicus Handbook;  citation_year=2005;  citation_pages=261-77; " /><meta name="citation_reference" content="citation_title=MAFFT version 5: improvement in accuracy of multiple sequence alignment,;  citation_journal_title=Nucleic Acids Res;  citation_year=2005;  citation_volume=33;  citation_pages=511-8; " /><meta name="citation_reference" content="citation_title=RAxML-III: a fast program for maximum likelihood-based inference of large phylogenetic trees,;  citation_journal_title=Bioinformatics;  citation_year=2005;  citation_volume=21;  citation_pages=456-63; " /><meta name="citation_reference" content="citation_title=MRBAYES: Bayesian inference of phylogeny,;  citation_journal_title=Bioinformatics;  citation_year=2001;  citation_volume=17;  citation_pages=754-755; " /><meta name="citation_reference" content="citation_title=MEGA5: molecular evolutionary genetics analysis using maximum likelihood, evolutionary distance, and maximum parsimony methods,;  citation_journal_title=Mol. Biol. Evol;  citation_year=2011;  citation_volume=28;  citation_pages=2731-9; " /><meta name="citation_reference" content="citation_title=PAML 4: Phylogenetic analysis by maximum likelihood;  citation_journal_title=Mol. Biol. Evol;  citation_year=2007;  citation_volume=23;  citation_pages=7-9; " /><meta name="citation_reference" content="citation_title=Analysis of ENOD40 expression in alb1, a symbiotic mutant of Lotus japonicus that forms empty nodules with incompletely developed nodule vascular bundles,;  citation_journal_title=Mol. Gen. Genet;  citation_year=2000;  citation_volume=264;  citation_pages=402-10; " /><meta name="citation_reference" content="citation_title=crinkle, a novel symbiotic mutant that affects the infection thread growth and alters the root hair, trichome, and seed development in Lotus japonicus,;  citation_journal_title=Plant Physiol;  citation_year=2003;  citation_volume=131;  citation_pages=1054-63; " /><meta name="citation_reference" content="citation_title=Receptor-mediated exopolysaccharide perception controls bacterial infection,;  citation_journal_title=Nature;  citation_year=2015;  citation_volume=523;  citation_pages=308-12; " /><meta name="citation_reference" content="citation_title=Nod factor/nitrate-induced CLE genes that drive HAR1-mediated systemic regulation of nodulation,;  citation_journal_title=Plant Cell Physiol;  citation_year=2009;  citation_volume=50;  citation_pages=67-77; " /><meta name="citation_reference" content="citation_title=Early Events in the Infection of Soybean (Glycine max L. Merr) by Rhizobium japonicum: I. LOCALIZATION OF INFECTIBLE ROOT CELLS,;  citation_journal_title=Plant Physiol;  citation_year=1980;  citation_volume=66;  citation_pages=1027-1031; " /><meta name="citation_reference" content="citation_title=Down-regulation of NSP2 expression in developmentally young regions of Lotus japonicus roots in response to rhizobial inoculation,;  citation_journal_title=Plant Cell Physiol;  citation_year=2013;  citation_volume=54;  citation_pages=518-27; " /><meta name="citation_reference" content="citation_publisher=Wiley-Blackwell, Hoboken;  citation_title=Biological Nitrogen Fixation;  citation_year=2015;  citation_volume=vol. 2;  citation_pages=609-22; " /><meta name="citation_reference" content="citation_title=The symbiosis-related ERN transcription factors act in concert to coordinate rhizobial host root infection,;  citation_journal_title=Plant Physiol;  citation_year=2016;  citation_volume=171;  citation_pages=1037-54; " /><meta name="citation_reference" content="citation_title=Legume genome evolution viewed through the Medicago truncatula and Lotus japonicus genomes,;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2006;  citation_volume=103;  citation_pages=14959-64; " /><meta name="citation_reference" content="citation_title=An update of the Angiosperm Phylogeny Group classification for the orders and families of flowering plants: APG IV,; citation_author=The Angiosperm Phylogeny Group;  citation_journal_title=Bot. J. Linn. Soc.;  citation_year=2016;  citation_volume=181;  citation_pages=1095-8339; " /><meta name="citation_reference" content="citation_title=Positive selection, relaxation, and acceleration in the evolution of the human and chimp genome,;  citation_journal_title=PLoS Comput. Biol;  citation_year=2006;  citation_volume=2;  citation_pages=e38" /><meta name="citation_reference" content="citation_title=Episodic evolution and adaptation of chloroplast genomes in ancestral grasses,;  citation_journal_title=PLoS One;  citation_year=2009;  citation_volume=4;  citation_pages=e529" /><meta name="citation_reference" content="citation_title=Accelerated mutation accumulation in asexual lineages of a fresh-water snail,;  citation_journal_title=Mol. Biol. Evol;  citation_year=2010;  citation_volume=27;  citation_pages=954-63; " /><meta name="citation_reference" content="citation_title=Likelihood ratio tests for detecting positive selection and application to primate lysozyme evolution,;  citation_journal_title=Mol. Biol. Evol;  citation_year=1998;  citation_volume=15;  citation_pages=568-73; " /><meta name="citation_reference" content="citation_title=NODULE INCEPTION directly targets NF-Y subunit genes to regulate essential processes of root nodule development in Lotus japonicus,;  citation_journal_title=PLoS Genet;  citation_year=2013;  citation_volume=9;  citation_pages=e1003352" /><meta name="citation_reference" content="citation_title=Two CCAAT-box-binding transcription factors redundantly regulate early steps of the legume-rhizobia endosymbiosis,;  citation_journal_title=Plant J;  citation_year=2014;  citation_volume=79;  citation_pages=757-68; " /><meta name="citation_reference" content="citation_title=The Medicago genome provides insight into the evolution of rhizobial symbioses,;  citation_journal_title=Nature;  citation_year=2011;  citation_volume=480;  citation_pages=520-4; " /><meta name="citation_reference" content="citation_title=Rhodopsin evolution in the dark,;  citation_journal_title=Nature;  citation_year=1997;  citation_volume=387;  citation_pages=667-8; " /><meta name="citation_reference" content="citation_title=Estimating synonymous and non-synonymous substitution rates under realistic evolutionary models,;  citation_journal_title=Mol. Biol. Evol;  citation_year=2000;  citation_volume=17;  citation_pages=32-43; " /><meta name="citation_reference" content="citation_title=Genome-wide analysis of the ERF gene family in Arabidopsis and rice,;  citation_journal_title=Plant Physiol;  citation_year=2006;  citation_volume=140;  citation_pages=411-32; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/24/2/193/13688890/dsw052.pdf" /><meta name="description" content="Abstract.  Legume-rhizobium symbiosis is achieved by two major events evolutionarily acquired: root hair infection and organogenesis. Infection thread (IT) deve" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/24/2/193/2731026" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/24/2/193/2731026" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/24/2/193/2731026");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsw052';
    </script>
<title>
	Function and evolution of a Lotus japonicus AP2/ERF family transcription factor that is required for development of infection threads | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./2731026" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BaWtATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iWWFubywgS29qaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5hdGlvbmFsIEluc3RpdHV0ZSBvZiBBZ3JvYmlvbG9naWNhbCBTY2llbmNlcywgVHN1a3ViYSwgSWJhcmFraSAzMDUtODYwMiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEaXZpc2lvbiBvZiBTeW1iaW90aWMgU3lzdGVtcywgTmF0aW9uYWwgSW5zdGl0dXRlIGZvciBCYXNpYyBCaW9sb2d5LCBOYXRpb25hbCBJbnN0aXR1dGUgZm9yIE5hdHVyYWwgU2NpZW5jZXMsIE9rYXpha2kgNDQ0LTg1ODUsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQW9raSwgU2Vpc2hpcm8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEdlbmVyYWwgU3lzdGVtcyBTdHVkaWVzLCBHcmFkdWF0ZSBTY2hvb2wgb2YgQXJ0cyBhbmQgU2NpZW5jZXMsIHRoZSBVbml2ZXJzaXR5IG9mIFRva3lvLCBNZWd1cm8ta3UsIFRva3lvIDE1My04OTAyLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQmlvbG9naWNhbCBTY2llbmNlcywgR3JhZHVhdGUgU2Nob29sIG9mIFNjaWVuY2UsIHRoZSBVbml2ZXJzaXR5IG9mIFRva3lvLCBCdW5reW8ta3UsIFRva3lvIDExMy0wMDMyLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkxpdSwgTWVuZyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRpdmlzaW9uIG9mIFN5bWJpb3RpYyBTeXN0ZW1zLCBOYXRpb25hbCBJbnN0aXR1dGUgZm9yIEJhc2ljIEJpb2xvZ3ksIE5hdGlvbmFsIEluc3RpdHV0ZSBmb3IgTmF0dXJhbCBTY2llbmNlcywgT2themFraSA0NDQtODU4NSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEJhc2ljIEJpb2xvZ3ksIFNjaG9vbCBvZiBMaWZlIFNjaWVuY2UsIEdyYWR1YXRlIFVuaXZlcnNpdHkgZm9yIEFkdmFuY2VkIFN0dWRpZXMgKFNPS0VOREFJKSwgT2themFraSA0NDQtODU4NSwgSmFwYW4gYW5kICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlVtZWhhcmEsIFlvc3VrZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5hdGlvbmFsIEluc3RpdHV0ZSBvZiBBZ3JvYmlvbG9naWNhbCBTY2llbmNlcywgVHN1a3ViYSwgSWJhcmFraSAzMDUtODYwMiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTdWdhbnVtYSwgTm9yaW8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIExpZmUgU2NpZW5jZSwgQWljaGkgVW5pdmVyc2l0eSBvZiBFZHVjYXRpb24sIEthcml5YSwgQWljaGkgNDQ44oCTODU0MiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJJd2FzYWtpLCBXYXRhcnUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEJpb2xvZ2ljYWwgU2NpZW5jZXMsIEdyYWR1YXRlIFNjaG9vbCBvZiBTY2llbmNlLCB0aGUgVW5pdmVyc2l0eSBvZiBUb2t5bywgQnVua3lvLWt1LCBUb2t5byAxMTMtMDAzMiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTYXRvLCBTaHVzZWkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZSwgS2lzYXJhenUsIENoaWJhIDI5MuKAkzA4MTIsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iR3JhZHVhdGUgU2Nob29sIG9mIExpZmUgU2NpZW5jZXMsIFRvaG9rdSBVbml2ZXJzaXR5LCBBb2JhLWt1LCBTZW5kYWkgOTgwLTg1NzcsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU295YW5vLCBUYWthc2hpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGl2aXNpb24gb2YgU3ltYmlvdGljIFN5c3RlbXMsIE5hdGlvbmFsIEluc3RpdHV0ZSBmb3IgQmFzaWMgQmlvbG9neSwgTmF0aW9uYWwgSW5zdGl0dXRlIGZvciBOYXR1cmFsIFNjaWVuY2VzLCBPa2F6YWtpIDQ0NC04NTg1LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQmFzaWMgQmlvbG9neSwgU2Nob29sIG9mIExpZmUgU2NpZW5jZSwgR3JhZHVhdGUgVW5pdmVyc2l0eSBmb3IgQWR2YW5jZWQgU3R1ZGllcyAoU09LRU5EQUkpLCBPa2F6YWtpIDQ0NC04NTg1LCBKYXBhbiBhbmQgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS291Y2hpLCBIaXJvc2hpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTmF0aW9uYWwgSW5zdGl0dXRlIG9mIEFncm9iaW9sb2dpY2FsIFNjaWVuY2VzLCBUc3VrdWJhLCBJYmFyYWtpIDMwNS04NjAyLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ikthd2FndWNoaSwgTWFzYXlvc2hpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGl2aXNpb24gb2YgU3ltYmlvdGljIFN5c3RlbXMsIE5hdGlvbmFsIEluc3RpdHV0ZSBmb3IgQmFzaWMgQmlvbG9neSwgTmF0aW9uYWwgSW5zdGl0dXRlIGZvciBOYXR1cmFsIFNjaWVuY2VzLCBPa2F6YWtpIDQ0NC04NTg1LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQmFzaWMgQmlvbG9neSwgU2Nob29sIG9mIExpZmUgU2NpZW5jZSwgR3JhZHVhdGUgVW5pdmVyc2l0eSBmb3IgQWR2YW5jZWQgU3R1ZGllcyAoU09LRU5EQUkpLCBPa2F6YWtpIDQ0NC04NTg1LCBKYXBhbiBhbmQgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJGdW5jdGlvbiBhbmQgZXZvbHV0aW9uIG9mIGEgTG90dXMgamFwb25pY3VzIEFQMi9FUkYgZmFtaWx5IHRyYW5zY3JpcHRpb24gZmFjdG9yIHRoYXQgaXMgcmVxdWlyZWQgZm9yIGRldmVsb3BtZW50IG9mIGluZmVjdGlvbiB0aHJlYWRzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iMTkzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSIyMDMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3cwNTIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMjQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxNy8wNC8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1lY2hhbmlzbSBvZiBpbmZlY3Rpb24gdGhyZWFkIGVsb25nYXRpb24gaW4gcm9vdCBoYWlycyBvZiBNZWRpY2FnbyB0cnVuY2F0dWxhIGFuZCBkeW5hbWljIGludGVycGxheSB3aXRoIGFzc29jaWF0ZWQgcmhpem9iaWFsIGNvbG9uaXphdGlvbiw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IFBoeXNpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTQ4OyAgY2l0YXRpb25fcGFnZXM9MTk4NS05NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWN0aW5vcmhpemFsIHJvb3Qgbm9kdWxlIHN5bWJpb3Nlczogd2hhdCBpcyBzaWduYWxsaW5nIHRlbGxpbmcgb24gdGhlIG9yaWdpbnMgb2Ygbm9kdWxhdGlvbj8sOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DdXJyLiBPcGluLiBQbGFudCBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9MTEtMTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBsYW50IHJlc3BvbnNlcyB0byBub2R1bGF0aW9uIGZhY3RvcnMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DdXJyLiBPcGluLiBQbGFudCBCaW9sOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTI7ICBjaXRhdGlvbl9wYWdlcz00ODMtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2FsY2l1bSBzcGlraW5nIGluIHBsYW50IHJvb3QgaGFpcnMgcmVzcG9uZGluZyB0byBSaGl6b2JpdW0gbm9kdWxhdGlvbiBzaWduYWxzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q2VsbDsgIGNpdGF0aW9uX3llYXI9MTk5NjsgIGNpdGF0aW9uX3ZvbHVtZT04NTsgIGNpdGF0aW9uX3BhZ2VzPTY3My04MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIE5GUCBsb2N1cyBvZiBNZWRpY2FnbyB0cnVuY2F0dWxhIGNvbnRyb2xzIGFuIGVhcmx5IHN0ZXAgb2YgTm9kIGZhY3RvciBzaWduYWwgdHJhbnNkdWN0aW9uIHVwc3RyZWFtIG9mIGEgcmFwaWQgY2FsY2l1bSBmbHV4IGFuZCByb290IGhhaXIgZGVmb3JtYXRpb24sOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBKOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTM0OyAgY2l0YXRpb25fcGFnZXM9NDk1LTUwNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Tm9kIGZhY3Rvci1pbmR1Y2VkIHJvb3QgaGFpciBjdXJsaW5nOiBDb250aW51b3VzIHBvbGFyIGdyb3d0aCB0b3dhcmRzIHRoZSBwb2ludCBvZiBub2QgZmFjdG9yIGFwcGxpY2F0aW9uLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgUGh5c2lvbDsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMzI7ICBjaXRhdGlvbl9wYWdlcz0xOTgyLTE5ODg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgcmVjZXB0b3Iga2luYXNlIGdlbmUgb2YgdGhlIEx5c00gdHlwZSBpcyBpbnZvbHZlZCBpbiBsZWd1bWUgcGVyY2VwdGlvbiBvZiByaGl6b2JpYWwgc2lnbmFscyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT00MjU7ICBjaXRhdGlvbl9wYWdlcz02MzctNDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBsYW50IHJlY29nbml0aW9uIG9mIHN5bWJpb3RpYyBiYWN0ZXJpYSByZXF1aXJlcyB0d28gTHlzTSByZWNlcHRvci1saWtlIGtpbmFzZXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9NDI1OyAgY2l0YXRpb25fcGFnZXM9NTg1LTkyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NZWRpY2FnbyBMWUszLCBhbiBlbnRyeSByZWNlcHRvciBpbiByaGl6b2JpYWwgbm9kdWxhdGlvbiBmYWN0b3Igc2lnbmFsaW5nLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgUGh5c2lvbDsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0xNDU7ICBjaXRhdGlvbl9wYWdlcz0xODMtOTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxlZ3VtZSByZWNlcHRvcnMgcGVyY2VpdmUgdGhlIHJoaXpvYmlhbCBsaXBvY2hpdGluIG9saWdvc2FjY2hhcmlkZSBzaWduYWwgbW9sZWN1bGVzIGJ5IGRpcmVjdCBiaW5kaW5nLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTA5OyAgY2l0YXRpb25fcGFnZXM9MTM4NTktNjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2b2x1dGlvbiBvZiBzaWduYWwgdHJhbnNkdWN0aW9uIGluIGludHJhY2VsbHVsYXIgc3ltYmlvc2lzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VHJlbmRzIFBsYW50IFNjaTsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9NTExLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5vZHVsYXRpb24gaW5kZXBlbmRlbnQgb2Ygcmhpem9iaWEgaW5kdWNlZCBieSBhIGNhbGNpdW0tYWN0aXZhdGVkIGtpbmFzZSBsYWNraW5nIGF1dG9pbmhpYml0aW9uLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTQ0MTsgIGNpdGF0aW9uX3BhZ2VzPTExNDktNTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURlcmVndWxhdGlvbiBvZiBhIENhMisvY2FsbW9kdWxpbi1kZXBlbmRlbnQga2luYXNlIGxlYWRzIHRvIHNwb250YW5lb3VzIG5vZHVsZSBkZXZlbG9wbWVudCw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT00NDE7ICBjaXRhdGlvbl9wYWdlcz0xMTUzLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgZG9taW5hbnQgZnVuY3Rpb24gb2YgQ0NhTUsgaW4gaW50cmFjZWxsdWxhciBhY2NvbW1vZGF0aW9uIG9mIGJhY3RlcmlhbCBhbmQgZnVuZ2FsIGVuZG9zeW1iaW9udHMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBKOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTYzOyAgY2l0YXRpb25fcGFnZXM9MTQxLTU0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgbW9sZWN1bGFyIG5ldHdvcmsgZ292ZXJuaW5nIG5vZHVsZSBvcmdhbm9nZW5lc2lzIGFuZCBpbmZlY3Rpb24gaW4gdGhlIG1vZGVsIGxlZ3VtZSBMb3R1cyBqYXBvbmljdXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIENvbW11bjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xOyAgY2l0YXRpb25fcGFnZXM9MTAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2NyaXB0b21pY3Mgb2YgYWN0aW5vcmhpemFsIHN5bWJpb3NlcyByZXZlYWxzIGhvbW9sb2dzIG9mIHRoZSB3aG9sZSBjb21tb24gc3ltYmlvdGljIHNpZ25hbGluZyBjYXNjYWRlLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgUGh5c2lvbDsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xNTY7ICBjaXRhdGlvbl9wYWdlcz03MDAtMTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBhcmlzb24gb2YgdGhlIG5vZHVsZSB2cy4gcm9vdCB0cmFuc2NyaXB0b21lIG9mIHRoZSBhY3Rpbm9yaGl6YWwgcGxhbnQgRGF0aXNjYSBnbG9tZXJhdGE6IGFjdGlub3JoaXphbCBub2R1bGVzIGNvbnRhaW4gYSBzcGVjaWZpYyBjbGFzcyBvZiBkZWZlbnNpbnMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIE9uZTsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9ZTcyNDQyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZmljYXRpb24gb2YgcG90ZW50aWFsIHRyYW5zY3JpcHRpb25hbCByZWd1bGF0b3JzIG9mIGFjdGlub3JoaXphbCBzeW1iaW9zZXMgaW4gQ2FzdWFyaW5hIGdsYXVjYSBhbmQgQWxudXMgZ2x1dGlub3NhLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIFBsYW50IEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz0zNDIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TeW1SSyBkZWZpbmVzIGEgY29tbW9uIGdlbmV0aWMgYmFzaXMgZm9yIHBsYW50IHJvb3QgZW5kb3N5bWJpb3NlcyB3aXRoIGFyYnVzY3VsYXIgbXljb3JyaGl6YSBmdW5naSwgcmhpem9iaWEsIGFuZCBGcmFua2lhIGJhY3RlcmlhLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTA1OyAgY2l0YXRpb25fcGFnZXM9NDkyOC0zMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RnVuY3Rpb25hbCBhZGFwdGF0aW9uIG9mIGEgcGxhbnQgcmVjZXB0b3Ita2luYXNlIHBhdmVkIHRoZSB3YXkgZm9yIHRoZSBldm9sdXRpb24gb2YgaW50cmFjZWxsdWxhciByb290IHN5bWJpb3NlcyB3aXRoIGJhY3RlcmlhLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz1lNjgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgaW5kZXBlbmRlbnQgYWNxdWlzaXRpb24gb2YgcGxhbnQgcm9vdCBuaXRyb2dlbi1maXhpbmcgc3ltYmlvc2lzIGluIEZhYmlkcyByZWNydWl0ZWQgdGhlIHNhbWUgZ2VuZXRpYyBwYXRod2F5IGZvciBub2R1bGUgb3JnYW5vZ2VuZXNpcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgT25lOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz1lNjQ1MTUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHBsYW50IHJlZ3VsYXRvciBjb250cm9sbGluZyBkZXZlbG9wbWVudCBvZiBzeW1iaW90aWMgcm9vdCBub2R1bGVzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTQwMjsgIGNpdGF0aW9uX3BhZ2VzPTE5MS01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NZWRpY2FnbyB0cnVuY2F0dWxhIE5JTiBpcyBlc3NlbnRpYWwgZm9yIHJoaXpvYmlhbC1pbmRlcGVuZGVudCBub2R1bGUgb3JnYW5vZ2VuZXNpcyBpbmR1Y2VkIGJ5IGF1dG9hY3RpdmUgY2FsY2l1bS9jYWxtb2R1bGluLWRlcGVuZGVudCBwcm90ZWluIGtpbmFzZSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IFBoeXNpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTQ0OyAgY2l0YXRpb25fcGFnZXM9MzI0LTM1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgQ2FzdWFyaW5hIE5JTiBnZW5lIGlzIHRyYW5zY3JpcHRpb25hbGx5IGFjdGl2YXRlZCB0aHJvdWdob3V0IEZyYW5raWEgcm9vdCBpbmZlY3Rpb24gYXMgd2VsbCBhcyBpbiByZXNwb25zZSB0byBiYWN0ZXJpYWwgZGlmZnVzaWJsZSBzaWduYWxzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmV3IFBoeXRvbDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0yMDg7ICBjaXRhdGlvbl9wYWdlcz04ODctOTAzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1MaXBvY2hpdG9vbG9nb3NhY2NoYXJpZGVzIG1vZHVsYXRlIHBsYW50IGhvc3QgaW1tdW5pdHkgdG8gZW5hYmxlIGVuZG9zeW1iaW9zaXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Bbm51LiBSZXYuIFBoeXRvcGF0aG9sOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTUzOyAgY2l0YXRpb25fcGFnZXM9MzExLTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2b2x1dGlvbiBvZiBhIHN5bWJpb3RpYyByZWNlcHRvciB0aHJvdWdoIGdlbmUgZHVwbGljYXRpb25zIGluIHRoZSBsZWd1bWUtcmhpem9iaXVtIG11dHVhbGlzbSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5ldyBQaHl0b2w7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MjAxOyAgY2l0YXRpb25fcGFnZXM9OTYxLTcyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Gcm9tIM6yLSB0byDOsS1Qcm90ZW9iYWN0ZXJpYTogVGhlIG9yaWdpbiBhbmQgZXZvbHV0aW9uIG9mIHJoaXpvYmlhbCBub2R1bGF0aW9uIGdlbmVzIG5vZElKLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBCaW9sLiBFdm9sOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTMwOyAgY2l0YXRpb25fcGFnZXM9MjQ5NC01MDg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU11bHRpcGxlIHBvbHlwbG9pZHkgZXZlbnRzIGluIHRoZSBlYXJseSByYWRpYXRpb24gb2Ygbm9kdWxhdGluZyBhbmQgbm9ubm9kdWxhdGluZyBsZWd1bWVzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBCaW9sLiBFdm9sOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTMyOyAgY2l0YXRpb25fcGFnZXM9MTkzLTIxMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyYXRpdmUgZ2Vub21pY3Mgc3VnZ2VzdHMgdGhhdCBhbiBhbmNlc3RyYWwgcG9seXBsb2lkeSBldmVudCBsZWFkcyB0byBlbmhhbmNlZCByb290IG5vZHVsZSBzeW1iaW9zaXMgaW4gdGhlIFBhcGlsaW9ub2lkZWFlLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBCaW9sLiBFdm9sOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTMwOyAgY2l0YXRpb25fcGFnZXM9MjYwMi0xMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q1lDTE9QUywgYSBtZWRpYXRvciBvZiBzeW1iaW90aWMgaW50cmFjZWxsdWxhciBhY2NvbW1vZGF0aW9uLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTA1OyAgY2l0YXRpb25fcGFnZXM9MjA1NDAtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q0VSQkVSVVMsIGEgbm92ZWwgVS1ib3ggcHJvdGVpbiBjb250YWluaW5nIFdELTQwIHJlcGVhdHMsIGlzIHJlcXVpcmVkIGZvciBmb3JtYXRpb24gb2YgdGhlIGluZmVjdGlvbiB0aHJlYWQgYW5kIG5vZHVsZSBkZXZlbG9wbWVudCBpbiB0aGUgbGVndW1lLVJoaXpvYml1bSBzeW1iaW9zaXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBKOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTYwOyAgY2l0YXRpb25fcGFnZXM9MTY4LTgwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DWUNMT1BTLCBhIEROQS1iaW5kaW5nIHRyYW5zY3JpcHRpb25hbCBhY3RpdmF0b3IsIG9yY2hlc3RyYXRlcyBzeW1iaW90aWMgcm9vdCBub2R1bGUgZGV2ZWxvcG1lbnQsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DZWxsIEhvc3QgTWljcm9iZTsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTEzOS01MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBub3ZlbCBudWNsZWFyIHByb3RlaW4gaW50ZXJhY3RzIHdpdGggdGhlIHN5bWJpb3RpYyBETUkzIGNhbGNpdW0tIGFuZCBjYWxtb2R1bGluLWRlcGVuZGVudCBwcm90ZWluIGtpbmFzZSBvZiBNZWRpY2FnbyB0cnVuY2F0dWxhLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBQbGFudCBNaWNyb2JlLiBJbnRlcmFjdDsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0yMDsgIGNpdGF0aW9uX3BhZ2VzPTkxMi0yMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TElOLCBhIG5vdmVsIHR5cGUgb2YgVS1ib3gvV0Q0MCBwcm90ZWluLCBjb250cm9scyBlYXJseSBpbmZlY3Rpb24gYnkgcmhpem9iaWEgaW4gbGVndW1lcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IFBoeXNpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTUxOyAgY2l0YXRpb25fcGFnZXM9MTIzOS00OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW4gRVJGIHRyYW5zY3JpcHRpb24gZmFjdG9yIGluIE1lZGljYWdvIHRydW5jYXR1bGEgdGhhdCBpcyBlc3NlbnRpYWwgZm9yIG5vZCBmYWN0b3Igc2lnbmFsIHRyYW5zZHVjdGlvbiw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTk7ICBjaXRhdGlvbl9wYWdlcz0xMjIxLTM0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BUDItRVJGIHRyYW5zY3JpcHRpb24gZmFjdG9ycyBtZWRpYXRlIE5vZCBmYWN0b3IgZGVwZW5kZW50IE10RU5PRDExIGFjdGl2YXRpb24gaW4gcm9vdCBoYWlycyB2aWEgYSBub3ZlbCBjaXMtcmVndWxhdG9yeSBtb3RpZiw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTk7ICBjaXRhdGlvbl9wYWdlcz0yODY2LTg1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NZWRpY2FnbyB0cnVuY2F0dWxhIEVSTiB0cmFuc2NyaXB0aW9uIGZhY3RvcnM6IFJlZ3VsYXRvcnkgaW50ZXJwbGF5IHdpdGggTlNQMS9OU1AyIEdSQVMgZmFjdG9ycyBhbmQgZXhwcmVzc2lvbiBkeW5hbWljcyB0aHJvdWdob3V0IHJoaXpvYmlhbCBpbmZlY3Rpb24sOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBQaHlzaW9sOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTE2MDsgIGNpdGF0aW9uX3BhZ2VzPTIxNTUtNzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRPTyBNVUNIIExPVkUsIGEgcm9vdCByZWd1bGF0b3IgYXNzb2NpYXRlZCB3aXRoIHRoZSBsb25nLWRpc3RhbmNlIGNvbnRyb2wgb2Ygbm9kdWxhdGlvbiBpbiBMb3R1cyBqYXBvbmljdXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIFBsYW50IE1pY3JvYmUuIEludGVyYWN0OyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTIyOyAgY2l0YXRpb25fcGFnZXM9MjU5LTY4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Sb290LCByb290IGhhaXIsIGFuZCBzeW1iaW90aWMgbXV0YW50cyBvZiB0aGUgbW9kZWwgbGVndW1lIExvdHVzIGphcG9uaWN1cyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gUGxhbnQtTWljcm9iZSBJbnRlcmFjdDsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTE3LTI2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db250cm9sIG9mIGxlZ2hhZW1vZ2xvYmluIHN5bnRoZXNpcyBpbiBzbmFrZSBiZWFucyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2NoZW0gSjsgIGNpdGF0aW9uX3llYXI9MTk3MTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjU7ICBjaXRhdGlvbl9wYWdlcz0xMDc1LTgwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HaWJiZXJlbGxpbiBjb250cm9scyB0aGUgbm9kdWxhdGlvbiBzaWduYWxpbmcgcGF0aHdheSBpbiBMb3R1cyBqYXBvbmljdXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBKOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTU4OyAgY2l0YXRpb25fcGFnZXM9MTgzLTk0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9wdWJsaXNoZXI9U3ByaW5nZXIsIERvcmRyZWNodDsgIGNpdGF0aW9uX3RpdGxlPUxvdHVzIGphcG9uaWN1cyBIYW5kYm9vazsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3BhZ2VzPTI2MS03NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TUFGRlQgdmVyc2lvbiA1OiBpbXByb3ZlbWVudCBpbiBhY2N1cmFjeSBvZiBtdWx0aXBsZSBzZXF1ZW5jZSBhbGlnbm1lbnQsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0zMzsgIGNpdGF0aW9uX3BhZ2VzPTUxMS04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SQXhNTC1JSUk6IGEgZmFzdCBwcm9ncmFtIGZvciBtYXhpbXVtIGxpa2VsaWhvb2QtYmFzZWQgaW5mZXJlbmNlIG9mIGxhcmdlIHBoeWxvZ2VuZXRpYyB0cmVlcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTIxOyAgY2l0YXRpb25fcGFnZXM9NDU2LTYzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NUkJBWUVTOiBCYXllc2lhbiBpbmZlcmVuY2Ugb2YgcGh5bG9nZW55LDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MTc7ICBjaXRhdGlvbl9wYWdlcz03NTQtNzU1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NRUdBNTogbW9sZWN1bGFyIGV2b2x1dGlvbmFyeSBnZW5ldGljcyBhbmFseXNpcyB1c2luZyBtYXhpbXVtIGxpa2VsaWhvb2QsIGV2b2x1dGlvbmFyeSBkaXN0YW5jZSwgYW5kIG1heGltdW0gcGFyc2ltb255IG1ldGhvZHMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEJpb2wuIEV2b2w7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9Mjg7ICBjaXRhdGlvbl9wYWdlcz0yNzMxLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBBTUwgNDogUGh5bG9nZW5ldGljIGFuYWx5c2lzIGJ5IG1heGltdW0gbGlrZWxpaG9vZDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBCaW9sLiBFdm9sOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTIzOyAgY2l0YXRpb25fcGFnZXM9Ny05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbmFseXNpcyBvZiBFTk9ENDAgZXhwcmVzc2lvbiBpbiBhbGIxLCBhIHN5bWJpb3RpYyBtdXRhbnQgb2YgTG90dXMgamFwb25pY3VzIHRoYXQgZm9ybXMgZW1wdHkgbm9kdWxlcyB3aXRoIGluY29tcGxldGVseSBkZXZlbG9wZWQgbm9kdWxlIHZhc2N1bGFyIGJ1bmRsZXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEdlbi4gR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MjY0OyAgY2l0YXRpb25fcGFnZXM9NDAyLTEwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1jcmlua2xlLCBhIG5vdmVsIHN5bWJpb3RpYyBtdXRhbnQgdGhhdCBhZmZlY3RzIHRoZSBpbmZlY3Rpb24gdGhyZWFkIGdyb3d0aCBhbmQgYWx0ZXJzIHRoZSByb290IGhhaXIsIHRyaWNob21lLCBhbmQgc2VlZCBkZXZlbG9wbWVudCBpbiBMb3R1cyBqYXBvbmljdXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBQaHlzaW9sOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTEzMTsgIGNpdGF0aW9uX3BhZ2VzPTEwNTQtNjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlY2VwdG9yLW1lZGlhdGVkIGV4b3BvbHlzYWNjaGFyaWRlIHBlcmNlcHRpb24gY29udHJvbHMgYmFjdGVyaWFsIGluZmVjdGlvbiw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT01MjM7ICBjaXRhdGlvbl9wYWdlcz0zMDgtMTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5vZCBmYWN0b3Ivbml0cmF0ZS1pbmR1Y2VkIENMRSBnZW5lcyB0aGF0IGRyaXZlIEhBUjEtbWVkaWF0ZWQgc3lzdGVtaWMgcmVndWxhdGlvbiBvZiBub2R1bGF0aW9uLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgQ2VsbCBQaHlzaW9sOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTUwOyAgY2l0YXRpb25fcGFnZXM9NjctNzc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVhcmx5IEV2ZW50cyBpbiB0aGUgSW5mZWN0aW9uIG9mIFNveWJlYW4gKEdseWNpbmUgbWF4IEwuIE1lcnIpIGJ5IFJoaXpvYml1bSBqYXBvbmljdW06IEkuIExPQ0FMSVpBVElPTiBPRiBJTkZFQ1RJQkxFIFJPT1QgQ0VMTFMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBQaHlzaW9sOyAgY2l0YXRpb25feWVhcj0xOTgwOyAgY2l0YXRpb25fdm9sdW1lPTY2OyAgY2l0YXRpb25fcGFnZXM9MTAyNy0xMDMxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Eb3duLXJlZ3VsYXRpb24gb2YgTlNQMiBleHByZXNzaW9uIGluIGRldmVsb3BtZW50YWxseSB5b3VuZyByZWdpb25zIG9mIExvdHVzIGphcG9uaWN1cyByb290cyBpbiByZXNwb25zZSB0byByaGl6b2JpYWwgaW5vY3VsYXRpb24sOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBDZWxsIFBoeXNpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9NTQ7ICBjaXRhdGlvbl9wYWdlcz01MTgtMjc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3B1Ymxpc2hlcj1XaWxleS1CbGFja3dlbGwsIEhvYm9rZW47ICBjaXRhdGlvbl90aXRsZT1CaW9sb2dpY2FsIE5pdHJvZ2VuIEZpeGF0aW9uOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPXZvbC4gMjsgIGNpdGF0aW9uX3BhZ2VzPTYwOS0yMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHN5bWJpb3Npcy1yZWxhdGVkIEVSTiB0cmFuc2NyaXB0aW9uIGZhY3RvcnMgYWN0IGluIGNvbmNlcnQgdG8gY29vcmRpbmF0ZSByaGl6b2JpYWwgaG9zdCByb290IGluZmVjdGlvbiw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IFBoeXNpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9MTcxOyAgY2l0YXRpb25fcGFnZXM9MTAzNy01NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TGVndW1lIGdlbm9tZSBldm9sdXRpb24gdmlld2VkIHRocm91Z2ggdGhlIE1lZGljYWdvIHRydW5jYXR1bGEgYW5kIExvdHVzIGphcG9uaWN1cyBnZW5vbWVzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MTAzOyAgY2l0YXRpb25fcGFnZXM9MTQ5NTktNjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuIHVwZGF0ZSBvZiB0aGUgQW5naW9zcGVybSBQaHlsb2dlbnkgR3JvdXAgY2xhc3NpZmljYXRpb24gZm9yIHRoZSBvcmRlcnMgYW5kIGZhbWlsaWVzIG9mIGZsb3dlcmluZyBwbGFudHM6IEFQRyBJViw7IGNpdGF0aW9uX2F1dGhvcj1UaGUgQW5naW9zcGVybSBQaHlsb2dlbnkgR3JvdXA7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJvdC4gSi4gTGlubi4gU29jLjsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT0xODE7ICBjaXRhdGlvbl9wYWdlcz0xMDk1LTgzMzk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBvc2l0aXZlIHNlbGVjdGlvbiwgcmVsYXhhdGlvbiwgYW5kIGFjY2VsZXJhdGlvbiBpbiB0aGUgZXZvbHV0aW9uIG9mIHRoZSBodW1hbiBhbmQgY2hpbXAgZ2Vub21lLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBDb21wdXQuIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MjsgIGNpdGF0aW9uX3BhZ2VzPWUzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVwaXNvZGljIGV2b2x1dGlvbiBhbmQgYWRhcHRhdGlvbiBvZiBjaGxvcm9wbGFzdCBnZW5vbWVzIGluIGFuY2VzdHJhbCBncmFzc2VzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBPbmU7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPWU1MjkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BY2NlbGVyYXRlZCBtdXRhdGlvbiBhY2N1bXVsYXRpb24gaW4gYXNleHVhbCBsaW5lYWdlcyBvZiBhIGZyZXNoLXdhdGVyIHNuYWlsLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBCaW9sLiBFdm9sOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTI3OyAgY2l0YXRpb25fcGFnZXM9OTU0LTYzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1MaWtlbGlob29kIHJhdGlvIHRlc3RzIGZvciBkZXRlY3RpbmcgcG9zaXRpdmUgc2VsZWN0aW9uIGFuZCBhcHBsaWNhdGlvbiB0byBwcmltYXRlIGx5c296eW1lIGV2b2x1dGlvbiw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gQmlvbC4gRXZvbDsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTU2OC03MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Tk9EVUxFIElOQ0VQVElPTiBkaXJlY3RseSB0YXJnZXRzIE5GLVkgc3VidW5pdCBnZW5lcyB0byByZWd1bGF0ZSBlc3NlbnRpYWwgcHJvY2Vzc2VzIG9mIHJvb3Qgbm9kdWxlIGRldmVsb3BtZW50IGluIExvdHVzIGphcG9uaWN1cyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPWUxMDAzMzUyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHdvIENDQUFULWJveC1iaW5kaW5nIHRyYW5zY3JpcHRpb24gZmFjdG9ycyByZWR1bmRhbnRseSByZWd1bGF0ZSBlYXJseSBzdGVwcyBvZiB0aGUgbGVndW1lLXJoaXpvYmlhIGVuZG9zeW1iaW9zaXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBKOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTc5OyAgY2l0YXRpb25fcGFnZXM9NzU3LTY4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgTWVkaWNhZ28gZ2Vub21lIHByb3ZpZGVzIGluc2lnaHQgaW50byB0aGUgZXZvbHV0aW9uIG9mIHJoaXpvYmlhbCBzeW1iaW9zZXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NDgwOyAgY2l0YXRpb25fcGFnZXM9NTIwLTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJob2RvcHNpbiBldm9sdXRpb24gaW4gdGhlIGRhcmssOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9Mzg3OyAgY2l0YXRpb25fcGFnZXM9NjY3LTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVzdGltYXRpbmcgc3lub255bW91cyBhbmQgbm9uLXN5bm9ueW1vdXMgc3Vic3RpdHV0aW9uIHJhdGVzIHVuZGVyIHJlYWxpc3RpYyBldm9sdXRpb25hcnkgbW9kZWxzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBCaW9sLiBFdm9sOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTE3OyAgY2l0YXRpb25fcGFnZXM9MzItNDM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIGFuYWx5c2lzIG9mIHRoZSBFUkYgZ2VuZSBmYW1pbHkgaW4gQXJhYmlkb3BzaXMgYW5kIHJpY2UsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBQaHlzaW9sOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTE0MDsgIGNpdGF0aW9uX3BhZ2VzPTQxMS0zMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzI0LzIvMTkzLzEzNjg4ODkwL2RzdzA1Mi5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgTGVndW1lLXJoaXpvYml1bSBzeW1iaW9zaXMgaXMgYWNoaWV2ZWQgYnkgdHdvIG1ham9yIGV2ZW50cyBldm9sdXRpb25hcmlseSBhY3F1aXJlZDogcm9vdCBoYWlyIGluZmVjdGlvbiBhbmQgb3JnYW5vZ2VuZXNpcy4gSW5mZWN0aW9uIHRocmVhZCAoSVQpIGRldmUiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVzLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTI3MzEwMjYmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2RkIprodqVidht+QhFmQjrpYB7srr0=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f2%2f193%2f2731026" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f2%2f193%2f2731026" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=97104" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/24/2"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/2/2/m_dnares_24_2cover.png?Expires=2147483647&amp;Signature=C8sLyIyfepNN-Oy5BR2xLoVJLx2W1jQBt~48a2lQ~mjys3EWTAKP5k2kV~t7b8sG~Qk1B4iVdz8BINFP5bR-Hc0gvwqekGHM8qavh8kDaLAfN~qNxrDj7NNeLHJ-IURMi5v1ATu23u5wB01wP27BHknbA3W9JgGnq6AVGPqeLfhK7AJcTzOYzKpGYwaedh5crkYXYHp~9GCcHuTjoInf3sKaYC2Ew-HnuA-yIL45j8ocXrNQuIxJV8rFXwx1Tt0b7I1ZJg09R4-7m-GZB9vPgdehgAXUMUNQ4gDatBY22xUt9COqnlb7RZKNUXFDo7Qe0LNJgSHwNUOJvsh-SLpXkg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/24/2">
                        <div class="volume trailing-comma">Volume 24</div>
                        <div class="issue">Issue 2</div>

            </a>

        </div>
        <div class="ii-pub-date">
April 2017        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="78054971">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#78054971">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="78054973">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#78054973">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="78054978">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#78054978">2. Materials and Methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="78054999">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#78054999">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="78055020">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#78055020">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="78055027">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#78055027">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="78055029">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#78055029">Conflict of interest</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="78055031">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#78055031">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="78055033">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#78055033">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/24/2/179/2960054">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/24/2/205/2738679">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Function and evolution of a <em>Lotus japonicus</em> AP2/ERF family transcription factor that is required for development of infection threads
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Koji Yano</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>National Institute of Agrobiological Sciences, Tsukuba, Ibaraki 305-8602, Japan</div><div class="aff"><div class="label -label">2</div>Division of Symbiotic Systems, National Institute for Basic Biology, National Institute for Natural Sciences, Okazaki 444-8585, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Koji+Yano">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yano K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yano K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Koji Yano</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Seishiro Aoki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div>Department of General Systems Studies, Graduate School of Arts and Sciences, the University of Tokyo, Meguro-ku, Tokyo 153-8902, Japan</div><div class="aff"><div class="label -label">4</div>Department of Biological Sciences, Graduate School of Science, the University of Tokyo, Bunkyo-ku, Tokyo 113-0032, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Seishiro+Aoki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Aoki S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Aoki S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Seishiro Aoki</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Meng Liu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div>Division of Symbiotic Systems, National Institute for Basic Biology, National Institute for Natural Sciences, Okazaki 444-8585, Japan</div><div class="aff"><div class="label -label">5</div>Department of Basic Biology, School of Life Science, Graduate University for Advanced Studies (SOKENDAI), Okazaki 444-8585, Japan and </div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Meng+Liu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Liu M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Liu M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Meng Liu</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yosuke Umehara</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>National Institute of Agrobiological Sciences, Tsukuba, Ibaraki 305-8602, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yosuke+Umehara">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Umehara Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Umehara Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yosuke Umehara</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Norio Suganuma</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">6</div>Department of Life Science, Aichi University of Education, Kariya, Aichi 448–8542, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Norio+Suganuma">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Suganuma N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Suganuma N&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Norio Suganuma</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Wataru Iwasaki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div>Department of Biological Sciences, Graduate School of Science, the University of Tokyo, Bunkyo-ku, Tokyo 113-0032, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Wataru+Iwasaki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Iwasaki W">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Iwasaki W&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Wataru Iwasaki</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shusei Sato</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">7</div>Kazusa DNA Research Institute, Kisarazu, Chiba 292–0812, Japan</div><div class="aff"><div class="label -label">8</div>Graduate School of Life Sciences, Tohoku University, Aoba-ku, Sendai 980-8577, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shusei+Sato">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sato S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sato S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Shusei Sato</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Takashi Soyano</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div>Division of Symbiotic Systems, National Institute for Basic Biology, National Institute for Natural Sciences, Okazaki 444-8585, Japan</div><div class="aff"><div class="label -label">5</div>Department of Basic Biology, School of Life Science, Graduate University for Advanced Studies (SOKENDAI), Okazaki 444-8585, Japan and </div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Takashi+Soyano">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Soyano T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Soyano T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Takashi Soyano</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hiroshi Kouchi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>National Institute of Agrobiological Sciences, Tsukuba, Ibaraki 305-8602, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hiroshi+Kouchi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kouchi H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kouchi H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hiroshi Kouchi</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Masayoshi Kawaguchi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div>Division of Symbiotic Systems, National Institute for Basic Biology, National Institute for Natural Sciences, Okazaki 444-8585, Japan</div><div class="aff"><div class="label -label">5</div>Department of Basic Biology, School of Life Science, Graduate University for Advanced Studies (SOKENDAI), Okazaki 444-8585, Japan and </div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsw052-cor1" data-open="dsw052-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81 564-55-7564. Fax. +81 564-55-7564. Email: <a href="mailto:masayosi@nibb.ac.jp" target="_blank">masayosi@nibb.ac.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Masayoshi+Kawaguchi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kawaguchi M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kawaguchi M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Masayoshi Kawaguchi</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsw052-FM1"><div class="fn"><div class="footnote-content">Edited by Prof. Kazuhiro Sato</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 24, Issue 2, 1 April 2017, Pages 193–203, <a href='https://doi.org/10.1093/dnares/dsw052'>https://doi.org/10.1093/dnares/dsw052</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">21 December 2016</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">18 August 2016</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">25 October 2016</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="2731026" href="/dnaresearch/article-pdf/24/2/193/13688890/dsw052.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Koji Yano, Seishiro Aoki, Meng Liu, Yosuke Umehara, Norio Suganuma, Wataru Iwasaki, Shusei Sato, Takashi Soyano, Hiroshi Kouchi, Masayoshi Kawaguchi;  Function and evolution of a <em>Lotus japonicus</em> AP2/ERF family transcription factor that is required for development of infection threads, <em>DNA Research</em>, Volume 24, Issue 2, 1 April 2017, Pages 193–203, <a href='https://doi.org/10.1093/dnares/dsw052'>https://doi.org/10.1093/dnares/dsw052</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/2731026?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/2731026?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/2731026?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/2731026?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/2731026?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/2731026?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/2731026?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Function%20and%20evolution%20of%20a%20%26lt%3Bem%26gt%3BLotus%20japonicus%26lt%3B%2Fem%26gt%3B%20AP2%2FERF%20family%20transcription%20factor%20that%20is%20required%20for%20development%20of%20infection%20threads&amp;publicationDate=2016-12-21&amp;volumeNum=24&amp;issueNum=2&amp;author=Yano%2C%20Koji%3B%20Aoki%2C%20Seishiro&amp;startPage=193&amp;endPage=203&amp;contentId=10.1093%2Fdnares%2Fdsw052&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f2%2f193%2f2731026" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=78054971 id="78054971" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Legume-rhizobium symbiosis is achieved by two major events evolutionarily acquired: root hair infection and organogenesis. Infection thread (IT) development is a distinct element for rhizobial infection. Through ITs, rhizobia are efficiently transported from infection foci on root hairs to dividing meristematic cortical cells. To unveil this process, we performed genetic screening using <em>Lotus japonicus</em> MG-20 and isolated symbiotic mutant lines affecting nodulation, root hair morphology, and IT development. Map-based cloning identified an AP2/ERF transcription factor gene orthologous to <em>Medicago truncatula ERN1</em>. <em>LjERN1</em> was activated in response to rhizobial infection and depended on CYCLOPS and NSP2. Legumes conserve an <em>ERN1</em> homolog, <em>ERN2</em>, that functions redundantly with <em>ERN1</em> in <em>M. truncatula</em>. Phylogenetic analysis showed that the lineages of <em>ERN1</em> and <em>ERN2</em> genes originated from a gene duplication event in the common ancestor of legume plants. However, genomic analysis suggested the lack of <em>ERN2</em> gene in the <em>L. japonicus</em> genome, consistent with <em>Ljern1</em> mutants exhibited a root hair phenotype that is observed in <em>ern1/ern2</em> double mutants in <em>M. truncatula.</em> Molecular evolutionary analysis suggested that the nonsynonymous/synonymous rate ratios of legume <em>ERN1</em> genes was almost identical to that of non-legume plants, whereas the <em>ERN2</em> genes experienced a relaxed selective constraint.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;infection thread development&quot;">infection thread development</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="ERN1">ERN1</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Lotus japonicus&quot;"><em>Lotus japonicus</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;molecular evolutionary analysis&quot;">molecular evolutionary analysis</a></div>                            <h2 scrollto-destination=78054973 id="78054973" class="section-title" >1. Introduction</h2>
<p>Legumes produce root nodules as a consequence of a successful interaction with nitrogen-fixing Gram negative bacteria, collectively termed rhizobia. The interaction with bacterial symbionts enables the host plant to utilize atmospheric nitrogen. Therefore, the nodule symbiosis affects agricultural ecosystems. For establishing a symbiotic relationship, host cells guide rhizobia from the infection foci on root hair surfaces into cortical nodule primordia through membranous and tubular paths known as infection threads (ITs), which basipetally develop into infected root hairs and further penetrate into dividing cortical cells. The tip-growth of ITs is correlated with bacterial cell division and file movement.<a reveal-id="dsw052-B1" data-open="dsw052-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> Rhizobia endocytically released from ITs into cortical cells and differentiate into bacteroids and can reduce dinitrogen into ammonium ion. Thus, IT progression is important for efficiently establishing the endosymbiosis. IT development has been observed not only in the root nodule symbiosis of legumes but also in the non-legume nodule symbiosis of actinorhizal plants, which are a group of a nitrogen-fixing lineage of angiosperms and interact with nitrogen fixing actinobacteria Frankia.<a reveal-id="dsw052-B2" data-open="dsw052-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> The acquisition of nodule symbioses of legumes and actinorhizal plants is suggested to have been independent evolutionary events, further implying advantages of this infection system.</p><p>In the legume nodule symbiosis, Nod factors, lipochitoligosaccharides secreted by rhizobia, act as initial signals of nodulation.<a reveal-id="dsw052-B3" data-open="dsw052-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> The Nod factor perception by host receptors (LjNFR1, LjNFR5, MtNPF, and MtLYS3) triggers the oscillation of perinuclear Ca<sup>2+ </sup>concentration and root hair deformation to entrap rhizobia.<a reveal-id="dsw052-B4 dsw052-B5 dsw052-B6 dsw052-B7 dsw052-B8 dsw052-B9 dsw052-B10" data-open="dsw052-B4 dsw052-B5 dsw052-B6 dsw052-B7 dsw052-B8 dsw052-B9 dsw052-B10" class="link link-ref link-reveal xref-bibr"><sup>4–10</sup></a> This Ca<sup>2+ </sup>spiking requires host factors that are also involved in mycorrhizal symbioses, the so-called common SYMs,<a reveal-id="dsw052-B11" data-open="dsw052-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> which are essential for the IT development. The calcium signal is believed to be decoded by LjCCaMK (calcium and calmodulin-dependent protein kinase)/MtDMI3 because gain-of-function versions of this protein kinase suppress symbiosis-defective phenotypes observed in the loss-of-function of common SYMs that are required for the Ca<sup>2+ </sup>spiking.<a reveal-id="dsw052-B12 dsw052-B13 dsw052-B14 dsw052-B15" data-open="dsw052-B12 dsw052-B13 dsw052-B14 dsw052-B15" class="link link-ref link-reveal xref-bibr"><sup>12–15</sup></a> Transcriptome analyses of actinorhizal plants have revealed that <em>Casuarina glauca</em>, <em>Datisca glomera</em>, and <em>Alnus glutinosa</em> possess common SYM factors including SymRK and CCaMK,<a reveal-id="dsw052-B16 dsw052-B17 dsw052-B18" data-open="dsw052-B16 dsw052-B17 dsw052-B18" class="link link-ref link-reveal xref-bibr"><sup>16–18</sup></a> which are involved in the nodule symbiosis of actinorhizal plants.<a reveal-id="dsw052-B19 dsw052-B20 dsw052-B21" data-open="dsw052-B19 dsw052-B20 dsw052-B21" class="link link-ref link-reveal xref-bibr"><sup>19–21</sup></a> These findings indicate that common SYM factors also act as signaling components of actinorhizal nodule symbiosis. In addition, a recent study further showed that the <em>C.</em><em>glauca</em>, homolog of a nodulation-specific legume transcription factor, NODULE INCEPTION (NIN), is involved in the IT development,<a reveal-id="dsw052-B22 dsw052-B23 dsw052-B24" data-open="dsw052-B22 dsw052-B23 dsw052-B24" class="link link-ref link-reveal xref-bibr"><sup>22–24</sup></a> implying some nodulation-specific factors commonly required for both types of the nodule symbiosis. Homologous genes among legumes and actinorhizal plants may play a common important role in the development of nodules. Several homologs in a genome (paralogs) are also crucial for the nodulation. The above-mentioned receptors of Nod factors are such kind of paralogous genes and are believed to have originated from gene duplications.<a reveal-id="dsw052-B25" data-open="dsw052-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a><sup>,</sup><a reveal-id="dsw052-B26" data-open="dsw052-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> Duplication events in the gene- and genome-levels were suggested to associate with the evolution of symbiosis between legumes and rhizobia.<a reveal-id="dsw052-B27 dsw052-B28 dsw052-B29" data-open="dsw052-B27 dsw052-B28 dsw052-B29" class="link link-ref link-reveal xref-bibr"><sup>27–29</sup></a> Possible outcomes of functional divergence of the duplicated genes such as subfunctionalization and neofuntionalization might have adopted a key role in the legume evolution.</p><p>The above-mentioned <em>L. japonicus</em> and <em>M. truncatula</em> mutants exhibit deficiencies in IT progression and cortical cell division because these factors are responsible for early nodulation signaling pathways. Nodulation includes coordinately regulated processes of IT progression and cortical cell division producing nodule primordia. However, cortical cell division does not always associate with IT progression into the cortex. <em>L. japonicus</em> mutants, <em>cerberus</em> and <em>cyclops</em>, display infection defective phenotypes, but produce nodule primordia. <em>CERBERUS</em> and <em>CYCLOPS</em> encode a U-box protein containing WD-40 repeats and a transcription factor, respectively.<a reveal-id="dsw052-B30" data-open="dsw052-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a><sup>,</sup><a reveal-id="dsw052-B31" data-open="dsw052-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> The later protein is targeted by LjCCaMK, and its activity is regulated by phosphorylation at critical amino acid residues.<a reveal-id="dsw052-B32" data-open="dsw052-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> Similar to <em>L. japonicus</em> mutants, loss-of-functions of <em>M. truncatula</em> counterparts also exhibit infection-deficient phenotype.<a reveal-id="dsw052-B33" data-open="dsw052-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a><sup>,</sup><a reveal-id="dsw052-B34" data-open="dsw052-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> These data indicate factors specifically regulating IT progression are conserved in legumes.</p><p>Many factors regulated with nodulation have been identified to date and have been used to create a basic model of nodulation pathways. Furthermore, common genetic features related with nodulation have been clarified between legumes and actinorhizal plants. However, our knowledge regarding regulatory pathways of IT progression remains fragmented. For elucidating molecular mechanisms underlying rhizobial infection processes, identifying novel factors involved in IT progression is indispensable. For this purpose, we screened <em>L. japonicus</em> mutants exhibiting infection-deficient phenotypes but producing nodule primordia. Here, we report a novel <em>L. japonicus</em> mutant for which infection processes are remarkably affected. Map-based cloning of the mutation locus revealed that the causative gene encodes homolog of <em>M. truncatula</em> ERN1.<a reveal-id="dsw052-B35 dsw052-B36 dsw052-B37" data-open="dsw052-B35 dsw052-B36 dsw052-B37" class="link link-ref link-reveal xref-bibr"><sup>35–37</sup></a> Our genomic analysis of <em>ERN</em> in legumes showed a unique feature of <em>ERN</em> genes of <em>L. japonicus</em>. Phylogenetic analysis indicated the gene duplication of the <em>ERN</em> gene in the common ancestor of legumes. Molecular evolutionary analysis suggested the difference of functional constraint between the genes of <em>ERN1</em> and <em>ERN2</em> lineages after the duplication event.</p>                            <h2 scrollto-destination=78054978 id="78054978" class="section-title" >2. Materials and Methods</h2>
                            <h3 scrollto-destination=78054979 id="78054979" class="section-title" >2.1. Plant materials</h3>
<p>The F29 mutant was obtained by ethyl methanesulfonate (EMS) mutagenesis. Approximately 5,000 seeds were immersed in 0.4% EMS overnight and allowed to germinate. M1 plants were grown in a greenhouse, and M2 seeds were collected. A screen for mutants that form nodules but exhibit retarded plant growth was performed in approximately 25,000 M2 plants. A single individual of the M3 self progeny from the M2 candidate mutant was backcrossed with the parent MG-20. The 1699-1 mutant line was produced by carbon-ion beams. Ion-beam irradiation was conducted as previously described.<a reveal-id="dsw052-B38" data-open="dsw052-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> The energy of carbon ions was 320 MeV, and the mean linear energy transfer within the seed was estimated to be 86 keV/μm. Approximately 5,000 dry seeds of wild-type (WT) <em>L. japonicus</em> MG-20 were irradiated with carbon ions (C<sup>6+</sup>) at a dose of 80 Gy. Approximately 84,000 M2 plants derived from approximately 2,900 M1 plants were screened for the nodulation phenotype. The heredity of the mutant candidates was evaluated by segregation in the next generation. The mutants <em>nin-2</em>, <em>cyclops-3</em>, <em>cerberus-5</em>, <em>nsp2-1</em>, and <em>alb1-3</em> were all derived from <em>L. japonicus</em> Gifu and described previously.<a reveal-id="dsw052-B22" data-open="dsw052-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a><sup>,</sup><a reveal-id="dsw052-B30" data-open="dsw052-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a><sup>,</sup><a reveal-id="dsw052-B31" data-open="dsw052-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a><sup>,</sup><a reveal-id="dsw052-B39" data-open="dsw052-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a></p>                            <h3 scrollto-destination=78054981 id="78054981" class="section-title" >2.2. Observation of the rhizobial infection</h3>
<p>Seeds were scarified with sandpaper and sterilized with sodium hypochlorite. Sterilized seeds were germinated on 1.0% agar plates under dark condition at 24°C in a growth cabinet (LH-220S, NK System). After 2 days, seedlings were transferred to inoculation pods containing vermiculite supplied with Broughton &amp; Dilworth (B&amp;D) medium,<a reveal-id="dsw052-B40" data-open="dsw052-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> 10 μM potassium nitrate and <em>Mesorhizobium loti</em> MAFF 303099 constitutively expressing red fluorescent protein (<em>DsRed</em>) gene.<a reveal-id="dsw052-B41" data-open="dsw052-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a> Plants were grown under 16 h light/8 h dark cycles at 24°C in the growth cabinet. The morphology of each infection events on roots infected with <em>M. loti</em> MAFF 303099 expressing DsRed was observed by a SZX16 microscope (Olympus).</p>                            <h3 scrollto-destination=78054983 id="78054983" class="section-title" >2.3. Map-based cloning</h3>
<p>The F29 mutant line was crossed with <em>L. japonicus</em> Gifu B-129 for obtaining the F2 population. A total of 1,015 F2 plants were analysed by DNA markers flanking the F29 locus. The DNA markers are detailed below: CM0371I1 is a polymerase chain reaction (PCR) marker distinguished by an insertion/deletion. CM0371SN2 is a marker detecting single-nucleotide polymorphism by a sequence analysis. TM0371 is a simple sequence repeat marker. CM0371C1 is a cleaved amplified polymorphic sequence digested by <em>MseI</em>. Candidate genes on TAC clone LjT46E19 were predicted by GENSCAN (<a class="link link-uri" href="http://genes.mit.edu/GENSCAN.html" target="_blank">http://genes.mit.edu/GENSCAN.html</a>).</p><p>F2 seeds were scarified with sandpaper and sterilized with 10% hypochlorite. The seeds were maintained in sterile water for water absorption. The seeds were then germinated on 1.0% agar plates for 2 days. Seedlings were transferred to nitrogen-deficient soil and inoculated with <em>M. loti</em> MAFF303099. After 2 weeks, nodules were observed.</p>                            <h3 scrollto-destination=78054986 id="78054986" class="section-title" >2.4. Plasmid construction and transformation</h3>
<p>pCAMBIA1300GFP was constructed detailed below. The <em>HPTII</em> gene was replaced with the green fluorescent protein (<em>GFP</em>) gene, and an <em>AscI</em> site was introduced into the <em>SmaI</em> site in pCAMBIA1300 (Cambia). <em>ERN1</em> promoter region (3,047 bp) and terminator region (1,109 bp) were amplified by PCR using specific primer sets (<em>ERN1</em> promoter-F, AACTGCAGCGTGCATT GC AC GG ATATAC; <em>ERN1</em> promoter-R, AAGGCGCGCCCACAAATTG TTCAAATTT AGTAA T  T GAGTG; <em>ERN1</em> terminator-F, AAGGC GCGC CACTTGATCTT GAAGGTCTTAAGTTAATG; <em>ERN1</em> ter mi nator-R, TTGAGCTCT GCCTTTGAACTGTGAGCAG). The promoter fragment was digested with <em>PstI</em> and <em>AscI</em> and ligated into pCAMBIA1300GFP. The terminator fragment was ligated into the vector after digesting with <em>AscI</em> and <em>SacI</em>. To utilize Gateway technology, a destination vector named pCGERNGW was then constructed by inserting reading frame cassette A of the Gateway vector conversion system (Thermo Fisher Scientific) into the <em>AscI</em> site treated by the Klenow Fragment (Takara) of the vector. For complementation tests, <em>ERN1</em> gene amplified twice by PCR (<em>ERN1</em> gene 1st-F, AAAAAGCAGG CTCCGTGATG GAG AT TCAAT TCCAGCA ;  <em>ERN1</em> gene 1st-R, AGAAAGCTGGGT TT A A CA GAACAATGA GCACAAGG; <em>ERN1</em> gene 2nd-F, GGGGACAA G TT T G TACAAAAAAGCAGGCT; <em>ERN1</em> gene 2nd-R, GGGG AC C ACTTTGTACAAGAAAGCTGGGT) was cloned into pDONR/ZEO (Thermo Fisher Scientific) via Gateway BP reaction (Thermo Fisher Scientific). A binary vector for complementation tests was constructed by introducing the <em>ERN1</em> gene into pCGERNGW using the Gateway LR reaction (Thermo Fisher Scientific). For promoter analysis, a β-glucuronidase (GUS)-Plus was amplified from pCAMBIA1305.1 (Cambia) by two times by PCR (GUS Plus gene 1st-F, AA AAAGCAGGCTCCACCATGGTAGATCTGAGGGTAA; GUS Plus gene 1st-R, AGAAAGCTGG GTTCA CACGT GATG GTG AT GG T; GUS Plus gene 2nd-F, GGGGACAA GTTTGTACA AAAA AG C AG GCT; GUS Plus gene 2nd-R, GGGGACCACTTT GTACAAG AAA GCTGGGT). This fragment was then cloned into pDONR/ZEO (Thermo Fisher Scientific) via the Gateway BP reaction (Thermo Fisher Scientific). A binary vector for the promoter-GUS analysis was constructed by introducing GUS Plus gene into pCGERNGW using the Gateway LR reaction (Thermo Fisher Scientific).</p><p>Plasmid constructs were introduced into <em>Agrobacterium rhizogenes</em> AR1193 and transformed into WT, F29, or 1699-1 plants by the hairy root transformation method with minor modifications.<a reveal-id="dsw052-B42" data-open="dsw052-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a></p>                            <h3 scrollto-destination=78054989 id="78054989" class="section-title" >2.5. Histochemical GUS staining and quantified reverse transcription-PCR</h3>
<p>For histochemical GUS staining, transformed plants with the promoter-GUS construct were inoculated with <em>M. loti</em> MAFF303099. The plants were grown for 7 days after inoculation. Roots were stained with 0.5 mg/ml 5-bromo-4-chloro-3-indolyl β-<span class="small-caps">d</span>-glucuronic acid cyclohexylammonium salt, 5 mM potassium ferricyanide, 5 mM potassium ferrocyanide, and 10 mM ethylenediaminetetraacetic acid (EDTA) in 100 mM sodium phosphate (pH 7.0). For observation of cortical cell division, roots were cleared by a chloral hydrate/glycerol/water (8w/1v/2v) solution.</p><p>For quantified reverse transcription (qRT)-PCR analysis, plants were grown for 0, 0.5, 1, and 3 days after inoculation with <em>M. loti</em> MAFF303099, and their roots were sampled from them. Total RNA was extracted from roots using the RNeasy Plant Mini Kit (Qiagen). Reverse transcription was conducted with the QuantiTect Reverse Transcription Kit (Qiagen). qRT-PCR was performed using the ABI Prism 7000 (Applied Biosystems) with a QuantiTect SYBR Green RT-PCR Kit (Qiagen). The following primer sets were used for detection of <em>ERN1</em> and <em>ATP synthase</em> expressions (ERN1-F,TCCA AT GAAACTCTGTGGGATC; ERN1-R,GGGAAGAAACA TTTGC AG GA; ATP synthase-F, GGTGATAAGCAGAGTGAAAGCA; ATP synthase-R,AAGACCAGTGAGACCAACACG).</p>                            <h3 scrollto-destination=78054992 id="78054992" class="section-title" >2.6. Ortholog and synteny analysis</h3>
<p>The most recent versions of genome, protein, and complementary DNA (cDNA) sequences of each species were downloaded from the respective genome sequence sites as follows: <em>L. japonicus</em> (version 3.0) from the <em>L. japonicus</em> Genome Sequencing Project (<a class="link link-uri" href="http://www.kazusa.or.jp/lotus/" target="_blank">http://www.kazusa.or.jp/lotus/</a>, November 2016, date last accessed), <em>M. truncatula</em> (Mt4.0v1), and <em>Glycine</em> (Wm82.a2.v1) from Phytozome 11 (<a class="link link-uri" href="https://phytozome.jgi.doe.gov/pz/portal.html#" target="_blank">https://phytozome.jgi.doe.gov/pz/portal.html#</a>, November 2016, date last accessed).</p><p>The analysis of microsynteny across species was based on comparisons of the specific regions containing <em>ERN</em> genes. Based on the <em>LjERN1</em> and <em>ERN</em> genes in <em>M. truncatula</em>, the <em>ERN</em> genes of <em>L. japonicus</em> and <em>G. max</em> were set as the anchor points according to their physical locations. The protein-coding sequences assigned to the flanking regions of each <em>ERN</em> gene in one species were compared with those in the other species by the Basic Local Alignment Search Tool for protein databases (BLASTP) program. The reciprocal best hit method was used to define orthologous relationship among the genes in <em>L. japonicus, M. truncatula</em>, and <em>G. max</em> in the same <em>ERN</em> gene region<em>.</em> For the gene conservation among the three <em>ERN</em> gene regions, a BLASTP E-value &lt;1e−20 was considered to be significant.</p>                            <h3 scrollto-destination=78054995 id="78054995" class="section-title" >2.7. Phylogenetic analysis</h3>
<p>Sequences similar to LjERN1 were collected with similarity searches against all the 68 plant genomes in the refseq database of the National Center for Biotechnology Information (NCBI) using BLASTP on 9 April 2016. The amino acid sequences similar to MtERN1, MtERN2, MtERN3, <em>Sesamum indicum</em> XP_011075059 and XP_011089231, <em>Oryza sativa</em> XP_015645889, and <em>Amborella trichopoda</em> XP_011628828 were selected with the option-max_target_seqs 100. The resulting 213 sequences and three other sequences (LjERN1, <em>Cajanus cajan</em> KYP69442.1, and KYP62096.1) were used for the preliminary analysis. Those containing large indels or those that were highly diverged were omitted from the dataset. Finally, 36 sequences of fabids (eurosid I) were used for the phylogenetic analysis. The amino acid sequences were aligned with the MAFFT program<a reveal-id="dsw052-B43" data-open="dsw052-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a> using the strategy E-INS-i, followed by manual adjustment. The maximum likelihood (ML) analysis was conducted with the LG model implemented in the RAxML program.<a reveal-id="dsw052-B44" data-open="dsw052-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a> The best model of evolution was determined using the option –auto-prot = aic in the program. The Bayes method was performed with MrBayes 3.2.6 using a WAG + I+G model.<a reveal-id="dsw052-B45" data-open="dsw052-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a> Neighbor-Joining (NJ) analysis was conducted with the JTT model of MEGA5.2.2 using the option of partial deletion of site coverage cutoff 80%.<a reveal-id="dsw052-B46" data-open="dsw052-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a></p>                            <h3 scrollto-destination=78054997 id="78054997" class="section-title" >2.8. Molecular evolutionary analysis</h3>
<p>To examine the pattern of ω (dn/ds: nonsynonymous/synonymous substitution rate ratio) acting on the ERN1 and ERN2 homologs at lineage-specific levels, ML tests implemented in Phylogenetic Analysis by Maximum Likelihood (PAML) were applied to the 36 sequences.<a reveal-id="dsw052-B47" data-open="dsw052-B47" class="link link-ref link-reveal xref-bibr"><sup>47</sup></a> The branch-specific model of PAML includes the one-, two-, and three-ratio models to represent the different presumed hypotheses. The one-ratio model assumed a constant ratio ω across all branches. The two-ratio models assigned the ratio ω<sub>ERN1</sub>, ω<sub>ERN2</sub>, or ω<sub>ERN1 + ERN2</sub> to the branches of the ERN1, ERN2, or ERN1 and ERN2 clades, respectively, and the ratio ω<sub>outgroup</sub> to the remaining branches. The three-ratio model assigned the ratios ω<sub>ERN1</sub>, ω<sub>ERN2</sub>, and ω<sub>outgroup</sub> to the corresponding branches. For any pair of models that were nested together, the goodness of fit was tested using the likelihood ratio test (LRT) with the CODEML program in PAML. The branch-site and clade models were also used for the analysis of amino acid sites under positive selection.</p>                            <h2 scrollto-destination=78054999 id="78054999" class="section-title" >3. Results</h2>
                            <h3 scrollto-destination=78055000 id="78055000" class="section-title" >3.1. Mutant lines defective in IT progression</h3>
<div class="block-child-p">During the early process of nodulation, rhizobia are entrapped by curled or deformed root hairs to form an infection pocket. From the site as the starting point, they enter into the plant cells through the development of ITs. In <em>L. japonicus</em>, mutants such as <em>alb1, cerberus, crinkle, cyclops</em>, <em>epr3</em>, and <em>nin</em> are shown to be defective in the infection process after the curling.<a reveal-id="dsw052-B22" data-open="dsw052-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a><sup>,</sup><a reveal-id="dsw052-B30" data-open="dsw052-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a><sup>,</sup><a reveal-id="dsw052-B31" data-open="dsw052-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a><sup>,</sup><a reveal-id="dsw052-B48 dsw052-B49 dsw052-B50" data-open="dsw052-B48 dsw052-B49 dsw052-B50" class="link link-ref link-reveal xref-bibr"><sup>48–50</sup></a> To identify a new component involved in the infection process, we have performed an extensive screening of symbiotic mutants using <em>L. japonicus</em>. F29, a mutant line of infection processes that does not form effective nodules, was originally isolated by the treatment of EMS to <em>L. japonicus</em> Miyakojima MG-20. On the other hand, the 1699-1 mutant line was isolated through the screening of progeny of ion-beam-mutagenized <em>L. japonicus</em> MG-20 seeds. Both lines did not display a completely non-nodulating phenotype or form an increased number of bump-like structures on the root such as <em>alb1</em> and <em>epr3</em> (<a reveal-id="dsw052-F1" data-open="dsw052-F1" class="link link-reveal link-table xref-fig">Fig. 1A</a>). <div data-id="dsw052-F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw052/2/m_dsw052f1.png?Expires=2147483647&Signature=ziOplQw1BXmhqUtu~Cndaf3H--bXJ4WWkjU6AkBM33rngQgGFjfpM6fUXQ2b1JHfqGRi0HF-HoxVNOVuJ7VEBwfrnpKUHTtYsx~GeTawQOgfi-NCInl0hoX68bnDEaijOgmi~Ns75UksTmNuoa8fSThQaNTqA5slxBBdvxP5xDy9TMW~GfDnwMqtv7olfXWVdHMR4BbGiHi4E1Guef5uoU7YXzndFmQA9gFHZm4aCcNstbPK-ftr1LlZvwSQ5RICuiQWkbDYNu65tk-wH39VDEfYbnUmd~bzX04SPMOM-6EgzOsZAwhud856~ZONNYRGdf0OUvChobXtpxkllCCThQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Nodule phenotypes of F29 and 1699-1 mutant lines. (A) Means and SDs of nodule and bump numbers 14 DAI (n = 25), and percentages of nodules/bumps that DsRed-labeled M. loti invaded into (10 DAI) are shown. (B–E) Nodules and bumps formed on WT (B), F29 (C and D), and 1699-1 (E) roots at 10 DAI with DsRed-labeled M. loti. Upper and bottom panels are bright field images and corresponding fluorescent images. Bars = 0.5 mm." path-from-xml="dsw052f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055001/dsw052f1.tif" path-from-xml="dsw052f1.tif" target="_blank">View large</a><a section="78055001" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw052/2/dsw052f1.png?Expires=2147483647&Signature=eKk-yDfl-0QH9O4aBLYNeForBfsCAmtPJjFs4wxDtTDmpNgRAgKmd-bL-bR8fQgK7BvnmT~sEJEiI02L-XuykG0b98lWo1tK3DlE2CtB56qyze4ZCyd09Za-xOGw6l5gg~XRj9RfPrPa5ZIXUJGCBtyE~zaiE-Ag0-vVCHi4kuvaSdPgmMGKZmc~bE~WmBRzOq5Qo6RvbYS-VjPI~BsS1eKxdbo5YdRPgF12T9LTjHlezFjVvdZqh6A-CrO2cS1hKxItpfqjtNWxkayy9mksGhA7Vx6sCIc~AaeByF9GN~CreSk1KFOphIHVCXNB875cYWL8dOg1QiHVcxu3fRP90A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=78055001&ar=2731026&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw052f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Nodule phenotypes of F29 and 1699-1 mutant lines. (A) Means and SDs of nodule and bump numbers 14 DAI (<em>n</em> = 25), and percentages of nodules/bumps that DsRed-labeled <em>M. loti</em> invaded into (10 DAI) are shown. (B–E) Nodules and bumps formed on WT (B), F29 (C and D), and 1699-1 (E) roots at 10 DAI with DsRed-labeled <em>M. loti</em>. Upper and bottom panels are bright field images and corresponding fluorescent images. Bars = 0.5 mm.</p></div></div><div content-id="dsw052-F1" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw052/2/m_dsw052f1.png?Expires=2147483647&Signature=ziOplQw1BXmhqUtu~Cndaf3H--bXJ4WWkjU6AkBM33rngQgGFjfpM6fUXQ2b1JHfqGRi0HF-HoxVNOVuJ7VEBwfrnpKUHTtYsx~GeTawQOgfi-NCInl0hoX68bnDEaijOgmi~Ns75UksTmNuoa8fSThQaNTqA5slxBBdvxP5xDy9TMW~GfDnwMqtv7olfXWVdHMR4BbGiHi4E1Guef5uoU7YXzndFmQA9gFHZm4aCcNstbPK-ftr1LlZvwSQ5RICuiQWkbDYNu65tk-wH39VDEfYbnUmd~bzX04SPMOM-6EgzOsZAwhud856~ZONNYRGdf0OUvChobXtpxkllCCThQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Nodule phenotypes of F29 and 1699-1 mutant lines. (A) Means and SDs of nodule and bump numbers 14 DAI (n = 25), and percentages of nodules/bumps that DsRed-labeled M. loti invaded into (10 DAI) are shown. (B–E) Nodules and bumps formed on WT (B), F29 (C and D), and 1699-1 (E) roots at 10 DAI with DsRed-labeled M. loti. Upper and bottom panels are bright field images and corresponding fluorescent images. Bars = 0.5 mm." path-from-xml="dsw052f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055001/dsw052f1.tif" path-from-xml="dsw052f1.tif" target="_blank">View large</a><a section="78055001" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw052/2/dsw052f1.png?Expires=2147483647&Signature=eKk-yDfl-0QH9O4aBLYNeForBfsCAmtPJjFs4wxDtTDmpNgRAgKmd-bL-bR8fQgK7BvnmT~sEJEiI02L-XuykG0b98lWo1tK3DlE2CtB56qyze4ZCyd09Za-xOGw6l5gg~XRj9RfPrPa5ZIXUJGCBtyE~zaiE-Ag0-vVCHi4kuvaSdPgmMGKZmc~bE~WmBRzOq5Qo6RvbYS-VjPI~BsS1eKxdbo5YdRPgF12T9LTjHlezFjVvdZqh6A-CrO2cS1hKxItpfqjtNWxkayy9mksGhA7Vx6sCIc~AaeByF9GN~CreSk1KFOphIHVCXNB875cYWL8dOg1QiHVcxu3fRP90A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=78055001&ar=2731026&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw052f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Nodule phenotypes of F29 and 1699-1 mutant lines. (A) Means and SDs of nodule and bump numbers 14 DAI (<em>n</em> = 25), and percentages of nodules/bumps that DsRed-labeled <em>M. loti</em> invaded into (10 DAI) are shown. (B–E) Nodules and bumps formed on WT (B), F29 (C and D), and 1699-1 (E) roots at 10 DAI with DsRed-labeled <em>M. loti</em>. Upper and bottom panels are bright field images and corresponding fluorescent images. Bars = 0.5 mm.</p></div></div></div><p>To visualize infection events, we inoculated F29 and 1699-1 mutant lines with Ds-Red-labeled <em>M. loti</em> MAFF303099. The nodulation phenotype of 1699-1 mutants was relatively severe compared with that of F29 mutants. The difference was clearly detected at 10 days after inoculation (DAI), when most of bumps or nodules formed on WT MG-20 roots have already displayed IT progression and release of rhizobia into cortical cells (<a reveal-id="dsw052-F1" data-open="dsw052-F1" class="link link-reveal link-table xref-fig">Fig. 1A and B</a>). Several bumps with IT progression into cortical cells were found in F29 roots although nodule development was remarkably delayed compared with WT nodules (<a reveal-id="dsw052-F1" data-open="dsw052-F1" class="link link-reveal link-table xref-fig">Fig. 1A, C, and D</a>). On the other hand, IT progression into the cortex was not observed in bumps formed on 1699-1 roots at this time point (<a reveal-id="dsw052-F1" data-open="dsw052-F1" class="link link-reveal link-table xref-fig">Fig. 1A and E</a>). Thus, both mutant lines were defective in IT progression into the cortex and nodule development.</p><p>The mutant lines were mapped near 53 cM chromosome 1, where no known symbiotic gene is located. We, therefore, conducted the allelism test. F1 plants did not form mature nodules, but instead developed many bumps or ineffective nodules in wider range of roots. This result indicates that the two mutant loci are allelic.</p>                            <h3 scrollto-destination=78055004 id="78055004" class="section-title" >3.2. Root hair morphology and IT development in two mutant lines</h3>
<div class="block-child-p">Mutants that can form bumps but do not produce mature nodules have been reported many times. They have usually shown defects in the infection processes including IT development. Therefore, we evaluated the infection events visualized by DsRed-labeled <em>M. loti</em>. In the WT (MG-20), long ITs were observed in root hairs at 10 DAI (<a reveal-id="dsw052-F2" data-open="dsw052-F2" class="link link-reveal link-table xref-fig">Fig. 2A</a>). The WT mean number was 44.4, and shorter types were only 1.3% of total ITs (<a reveal-id="dsw052-F2" data-open="dsw052-F2" class="link link-reveal link-table xref-fig">Fig. 2C</a>). Frequencies of IT formation in F29 and 1699-1 roots were remarkably lower than WT roots (<a reveal-id="dsw052-F2" data-open="dsw052-F2" class="link link-reveal link-table xref-fig">Fig. 2C</a>). Statistical analysis showed that the numbers of ITs on F29 and 1699-1 roots were 8% and 0.8% of WT roots, respectively. Over 10% of ITs formed in F29 roots were of the shorter type (<a reveal-id="dsw052-F2" data-open="dsw052-F2" class="link link-reveal link-table xref-fig">Fig. 2B and C</a>), indicating that IT development in root hairs was inhibited in mutant lines. These results suggested that the mutated gene is essential for IT development in root hairs and in cortical cells. <div data-id="dsw052-F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw052/2/m_dsw052f2.png?Expires=2147483647&Signature=hK9zGDLIXGKSF-NbdB2PNCJWJPdn76wCETuI2sDH-mrSfC20txSp9~oCio1DcFIxdH~p076GDc8y0lC4xqz1KC8J3IdMppgcvbo3QfJkYr9a0otAGaGHn4PaOElx0Q045E-hkNHBnPwZnSRoY~QaKaBw9My3PPH~ekbT8W1RvAmuOcgOQ-FOmVIMsSb5GxfoAVjrcayJIK3kLvq3euflGeFK8kN2X4tWZWjBVZg42Bka8V6VNu6EskM7XmwehA8ieG-190ClM9tnAFY09ATP2cFdm2xAeCmOBXmjUcDmAJUxEeCJe5tSLXtUvuIYVEX~1u1c2O66w-B6xt9ZmAaZtA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Root hair phenotypes of F29 and 1699-1 mutant lines. (A and B) WT (A) and F29 (B) root hairs infected by DsRed-labeled M. loti (10 DAI). ITs were visualized by DsRed fluorescence. Bars = 50 µm. (C) Means and SDs of IT numbers at 7 DAI (n = 16) are shown. ITs and short ITs represented in (A) and (B), respectively, were counted. (D–G) Root hair morphologies of WT (D), F29 (E), and 1699-1 (F and G) roots that were inoculated with M. loti for 4 days. Typical balloon-shaped root hairs found in WT (D) and mutant lines (E and F) and typical branched root hairs (G) are shown. Bars = 50 µm. (H) Statistic analysis of root hair structures. Means and SDs of root hair numbers were shown. Over 8 plants were observed (4 DAI). One-way ANOVA followed by a Tukey Honestly Significant Difference (HSD) test of the values was performed (*P &lt; 0.05; **P &lt; 0.01)." path-from-xml="dsw052f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055005/dsw052f2.tif" path-from-xml="dsw052f2.tif" target="_blank">View large</a><a section="78055005" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw052/2/dsw052f2.png?Expires=2147483647&Signature=qyOrduoE0dI4F80fLI8ww6Hv6yR7lBJNdqh0xnSJEd5ThxEGCMZpQduvcma4pW61Ul7YFOtkBVWHRNFSOOd1Kzv04zA8YZjgxcqVhcH-r73BAFTE90nBaADCn2sOmyS7UI2ELR05~GjkKf3ofFdqqog66Ohttv~lucnMgrNDSjaWOymZJhYfTGqszDESoCge~UO5RBZ6lrJm8AOF1va5hKbG-gM1eaqkogT--4Ezm-GVEWm0dDpqN3fcurUUfJmiFqUlhwbm7NXzxyMfqAA-YmNfhYZ1TMcbTd5EGG~bPJc-fjRuJ4PCUfCNjYFJsUq9zGZfyd-ynvObpTVZK025TQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=78055005&ar=2731026&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw052f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Root hair phenotypes of F29 and 1699-1 mutant lines. (A and B) WT (A) and F29 (B) root hairs infected by DsRed-labeled <em>M. loti</em> (10 DAI). ITs were visualized by DsRed fluorescence. Bars = 50 µm. (C) Means and SDs of IT numbers at 7 DAI (<em>n</em> = 16) are shown. ITs and short ITs represented in (A) and (B), respectively, were counted. (D–G) Root hair morphologies of WT (D), F29 (E), and 1699-1 (F and G) roots that were inoculated with <em>M. loti</em> for 4 days. Typical balloon-shaped root hairs found in WT (D) and mutant lines (E and F) and typical branched root hairs (G) are shown. Bars = 50 µm. (H) Statistic analysis of root hair structures. Means and SDs of root hair numbers were shown. Over 8 plants were observed (4 DAI). One-way ANOVA followed by a Tukey Honestly Significant Difference (HSD) test of the values was performed (*<em>P</em> &lt; 0.05; **<em>P</em> &lt; 0.01).</p></div></div><div content-id="dsw052-F2" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw052/2/m_dsw052f2.png?Expires=2147483647&Signature=hK9zGDLIXGKSF-NbdB2PNCJWJPdn76wCETuI2sDH-mrSfC20txSp9~oCio1DcFIxdH~p076GDc8y0lC4xqz1KC8J3IdMppgcvbo3QfJkYr9a0otAGaGHn4PaOElx0Q045E-hkNHBnPwZnSRoY~QaKaBw9My3PPH~ekbT8W1RvAmuOcgOQ-FOmVIMsSb5GxfoAVjrcayJIK3kLvq3euflGeFK8kN2X4tWZWjBVZg42Bka8V6VNu6EskM7XmwehA8ieG-190ClM9tnAFY09ATP2cFdm2xAeCmOBXmjUcDmAJUxEeCJe5tSLXtUvuIYVEX~1u1c2O66w-B6xt9ZmAaZtA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Root hair phenotypes of F29 and 1699-1 mutant lines. (A and B) WT (A) and F29 (B) root hairs infected by DsRed-labeled M. loti (10 DAI). ITs were visualized by DsRed fluorescence. Bars = 50 µm. (C) Means and SDs of IT numbers at 7 DAI (n = 16) are shown. ITs and short ITs represented in (A) and (B), respectively, were counted. (D–G) Root hair morphologies of WT (D), F29 (E), and 1699-1 (F and G) roots that were inoculated with M. loti for 4 days. Typical balloon-shaped root hairs found in WT (D) and mutant lines (E and F) and typical branched root hairs (G) are shown. Bars = 50 µm. (H) Statistic analysis of root hair structures. Means and SDs of root hair numbers were shown. Over 8 plants were observed (4 DAI). One-way ANOVA followed by a Tukey Honestly Significant Difference (HSD) test of the values was performed (*P &lt; 0.05; **P &lt; 0.01)." path-from-xml="dsw052f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055005/dsw052f2.tif" path-from-xml="dsw052f2.tif" target="_blank">View large</a><a section="78055005" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw052/2/dsw052f2.png?Expires=2147483647&Signature=qyOrduoE0dI4F80fLI8ww6Hv6yR7lBJNdqh0xnSJEd5ThxEGCMZpQduvcma4pW61Ul7YFOtkBVWHRNFSOOd1Kzv04zA8YZjgxcqVhcH-r73BAFTE90nBaADCn2sOmyS7UI2ELR05~GjkKf3ofFdqqog66Ohttv~lucnMgrNDSjaWOymZJhYfTGqszDESoCge~UO5RBZ6lrJm8AOF1va5hKbG-gM1eaqkogT--4Ezm-GVEWm0dDpqN3fcurUUfJmiFqUlhwbm7NXzxyMfqAA-YmNfhYZ1TMcbTd5EGG~bPJc-fjRuJ4PCUfCNjYFJsUq9zGZfyd-ynvObpTVZK025TQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=78055005&ar=2731026&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw052f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Root hair phenotypes of F29 and 1699-1 mutant lines. (A and B) WT (A) and F29 (B) root hairs infected by DsRed-labeled <em>M. loti</em> (10 DAI). ITs were visualized by DsRed fluorescence. Bars = 50 µm. (C) Means and SDs of IT numbers at 7 DAI (<em>n</em> = 16) are shown. ITs and short ITs represented in (A) and (B), respectively, were counted. (D–G) Root hair morphologies of WT (D), F29 (E), and 1699-1 (F and G) roots that were inoculated with <em>M. loti</em> for 4 days. Typical balloon-shaped root hairs found in WT (D) and mutant lines (E and F) and typical branched root hairs (G) are shown. Bars = 50 µm. (H) Statistic analysis of root hair structures. Means and SDs of root hair numbers were shown. Over 8 plants were observed (4 DAI). One-way ANOVA followed by a Tukey Honestly Significant Difference (HSD) test of the values was performed (*<em>P</em> &lt; 0.05; **<em>P</em> &lt; 0.01).</p></div></div></div><p>Furthermore, we investigated root hair responses at earlier stages after inoculation because fewer ITs in F29 and 1699-1 roots might be due to abnormal symbiotic root hair responses. In WT roots, curled, branched, and balloon-shaped root hairs comprised major morphological alterations at 4 DAI. Although the mean numbers of branched root hairs were not significantly different between WT and mutant lines, curled root hairs were decreased in mutant roots compared with those of WT roots (<a reveal-id="dsw052-F2" data-open="dsw052-F2" class="link link-reveal link-table xref-fig">Fig. 2A, G, and H</a>). In contrast, the mean numbers of balloon-shaped root hairs were increased in mutant lines (<a reveal-id="dsw052-F2" data-open="dsw052-F2" class="link link-reveal link-table xref-fig">Fig. 2H</a>). These misshapen root hairs expanded to an extreme compared with balloon-shaped root hairs observed in WT roots (<a reveal-id="dsw052-F2" data-open="dsw052-F2" class="link link-reveal link-table xref-fig">Fig. 2D–F</a>), and often generated new tips from lateral sides of the expanded tip (<a reveal-id="dsw052-F2" data-open="dsw052-F2" class="link link-reveal link-table xref-fig">Fig. 2E</a>). Depolarization and repolarization of root hair tips after rhizobial infection may be disorganized in mutant lines. These results indicated that the causative gene of F29 and 1699-1 lines is involved in correct symbiotic root hair responses in addition to IT development.</p>                            <h3 scrollto-destination=78055007 id="78055007" class="section-title" >3.3. Complementation by an ERF/AP2 transcription factor</h3>
<div class="block-child-p">To generate a mapping population, the F29 mutants (Miyakojima MG-20 background) were crossed with <em>L. japonicus</em> Gifu B-129. Fine mapping using the F2 plants allowed us to encompass the causal gene between two makers, TM0371 and CM0371C1, on LjT46E19 of chromosome 1 (<a class="link link-uri" href="http://www.kazusa.or.jp" target="_blank">http://www.kazusa.or.jp</a>; <a reveal-id="dsw052-F3" data-open="dsw052-F3" class="link link-reveal link-table xref-fig">Fig. 3A</a>). Eleven open reading frames were predicted in this region, and sequence comparison of WT and the F29 mutant revealed a missense mutation from Gly to Glu in the AP2 domain of an AP2/ERF transcription factor with a sequence similar to that of <em>M. truncatula</em> ERN1 (<a reveal-id="dsw052-F3" data-open="dsw052-F3" class="link link-reveal link-table xref-fig">Fig. 3B</a>). The sequence of the AP2/ERF transcription factor of <em>L. japonicus</em> showed high similarity to the <em>M. trancatula ERN1</em> and other homologs of legumes (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw052/2/dsw052_Supp.zip?Expires=2147483647&amp;Signature=SUfZUEyOx0cNgvbTgqmG5Jlc7D-wCgPBRu~nkZYtDlnDJ3y08Dk1egjJie4Bi9r0o~3T21J6w8mfvpFrbyNAq2drb5sFojcQL96jELnU9bogfK5Fi6LNtPAWKyHiFJfKZuy9UtsMZnYsh1Y3IfDP7q7xrQpM6Tdb~N-zpw88J5aGu5UvF0EP7Aurb-xgRQ4kG3PjHn5IFQBvYeAj7yMCbHPe5dB-ORrJjjGmlosU5TPSzRe3LJ9SSIUQBp5XOPrQ-r~s3VpZ6k98RKm2Ew2J5eCHjxuifUtwtPKDFIBqI6CYZsVtizidvxoFwOjgWAfKnY5U7a7aAxOp4ZGx9gcKTg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). On the other hand, approximately 10 kb of genomic region including this AP2/ERF gene was deleted in the 1699-1 mutant. We then performed a complementation test by hairy roots via <em>A. rhizogenes</em>. For six of eight F29 plants, an ineffective nodule phenotype was rescued by introducing the <em>AP2/ERF</em> gene carrying its 3,047 bp promoter and 1,109 bp terminator (<a reveal-id="dsw052-F3" data-open="dsw052-F3" class="link link-reveal link-table xref-fig">Fig. 3C</a>) but not by introducing a <em>GUS</em> gene (<a reveal-id="dsw052-F3" data-open="dsw052-F3" class="link link-reveal link-table xref-fig">Fig. 3D</a>). Based on the results, we concluded that <em>L. japonicus ERN1</em> gene is responsible for F29 and 1699-1 mutants. Hereafter, we describe F29 and 1699-1 as <em>Ljern1-5</em> and <em>Ljern1-6</em>, respectively. <div data-id="dsw052-F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw052/2/m_dsw052f3.png?Expires=2147483647&amp;Signature=PoKCuPI0gEcw8m4jUP3RgluGCqzTTM6oa8V4FQAgYx8kyWuRLfVibqaJ5yG96W7UDN56NtKmd7fuQ88l3Zioo7OWAb42ujINyZhGlmyhplOey7DfIcQyqqJ5phrWSl84SCDk1huo~mHQEyZWZYQIQTr3xJ2BWUI0uI18AOWVhBwa3y-TETnTutwpShxWmL4N8HgFQ2xp3uBqFqOCIy34vnVkT6M~Ysuj~vhlPv0Cm4VNOJF~c6XGOhu6~XmN0GkIMn4TLBmCayHAUufHZ~469Rp8uUSOTNrFfGrztjD~XzUCuRj5lpJVKm5gJQrXIsfgVQEIkmpmTjn8pA9djNvobg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Positional cloning of F29 and 1699-1. (A) Gentic linkage map of the F29 causative locus on the linkage group 1. LjT05K01 and LjT46E19 are TAC clones. DNA markers and their positions are indicated with the numbers of recombinants (in parentheses). (B) ORFs in the F29 locus delimited by recombination events and LjERN1 protein structure predicted by the nucleotide sequence are shown. The mutation site on F29 and the large deletion found in 1699-1 was indicated by a grey arrowhead and grey parentheses. (C and D) Complementation test. F29 hairy roots transformed with an ERN1 construct (C) and with an GUS vector as a negative control (D) were inoculated with M. loti for 32 days. Upper and bottom panels are bright field images and fluorescent images from a GFP transformation marker, respectively. Bars = 2 mm." path-from-xml="dsw052f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055008/dsw052f3.tif" path-from-xml="dsw052f3.tif" target="_blank">View large</a><a section="78055008" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw052/2/dsw052f3.png?Expires=2147483647&amp;Signature=ACUl2lNjegcYiIzMLP3mGvJjYjUlcgKYtrasRZByZ7wlMWZwntMCJEU4AmWQgfdte2NRjRTq23PkB0Ri3zZlkxtqgiYKkUcHugsTTz~fvIQlsnuB8rfyDevmWYfBqhLmDz5RlfTC6O8q7auFVYtBnuaZOkWMXaqgCsIx0iUSNvxCMzHK-xtwVB4iVWp11HtjQuwjmsGTog0Lz3SFpCuKB4w8d7ccsUTSiCRCdgBiwBT3PVKY53PGV1xkzxuKgVILMTYDDr77RcwNvYoVRYi9ioqRPBw~JKFqWW~8THm2j9anA35GKQ8zP4rOjO~6n-Eg4qm-5Ny1fkzV6PnqIfqyRA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=78055008&amp;ar=2731026&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw052f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Positional cloning of F29 and 1699-1. (A) Gentic linkage map of the F29 causative locus on the linkage group 1. LjT05K01 and LjT46E19 are TAC clones. DNA markers and their positions are indicated with the numbers of recombinants (in parentheses). (B) ORFs in the F29 locus delimited by recombination events and LjERN1 protein structure predicted by the nucleotide sequence are shown. The mutation site on F29 and the large deletion found in 1699-1 was indicated by a grey arrowhead and grey parentheses. (C and D) Complementation test. F29 hairy roots transformed with an <em>ERN1</em> construct (C) and with an GUS vector as a negative control (D) were inoculated with <em>M. loti</em> for 32 days. Upper and bottom panels are bright field images and fluorescent images from a GFP transformation marker, respectively. Bars = 2 mm.</p></div></div><div content-id="dsw052-F3" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw052/2/m_dsw052f3.png?Expires=2147483647&amp;Signature=PoKCuPI0gEcw8m4jUP3RgluGCqzTTM6oa8V4FQAgYx8kyWuRLfVibqaJ5yG96W7UDN56NtKmd7fuQ88l3Zioo7OWAb42ujINyZhGlmyhplOey7DfIcQyqqJ5phrWSl84SCDk1huo~mHQEyZWZYQIQTr3xJ2BWUI0uI18AOWVhBwa3y-TETnTutwpShxWmL4N8HgFQ2xp3uBqFqOCIy34vnVkT6M~Ysuj~vhlPv0Cm4VNOJF~c6XGOhu6~XmN0GkIMn4TLBmCayHAUufHZ~469Rp8uUSOTNrFfGrztjD~XzUCuRj5lpJVKm5gJQrXIsfgVQEIkmpmTjn8pA9djNvobg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Positional cloning of F29 and 1699-1. (A) Gentic linkage map of the F29 causative locus on the linkage group 1. LjT05K01 and LjT46E19 are TAC clones. DNA markers and their positions are indicated with the numbers of recombinants (in parentheses). (B) ORFs in the F29 locus delimited by recombination events and LjERN1 protein structure predicted by the nucleotide sequence are shown. The mutation site on F29 and the large deletion found in 1699-1 was indicated by a grey arrowhead and grey parentheses. (C and D) Complementation test. F29 hairy roots transformed with an ERN1 construct (C) and with an GUS vector as a negative control (D) were inoculated with M. loti for 32 days. Upper and bottom panels are bright field images and fluorescent images from a GFP transformation marker, respectively. Bars = 2 mm." path-from-xml="dsw052f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055008/dsw052f3.tif" path-from-xml="dsw052f3.tif" target="_blank">View large</a><a section="78055008" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw052/2/dsw052f3.png?Expires=2147483647&amp;Signature=ACUl2lNjegcYiIzMLP3mGvJjYjUlcgKYtrasRZByZ7wlMWZwntMCJEU4AmWQgfdte2NRjRTq23PkB0Ri3zZlkxtqgiYKkUcHugsTTz~fvIQlsnuB8rfyDevmWYfBqhLmDz5RlfTC6O8q7auFVYtBnuaZOkWMXaqgCsIx0iUSNvxCMzHK-xtwVB4iVWp11HtjQuwjmsGTog0Lz3SFpCuKB4w8d7ccsUTSiCRCdgBiwBT3PVKY53PGV1xkzxuKgVILMTYDDr77RcwNvYoVRYi9ioqRPBw~JKFqWW~8THm2j9anA35GKQ8zP4rOjO~6n-Eg4qm-5Ny1fkzV6PnqIfqyRA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=78055008&amp;ar=2731026&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw052f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Positional cloning of F29 and 1699-1. (A) Gentic linkage map of the F29 causative locus on the linkage group 1. LjT05K01 and LjT46E19 are TAC clones. DNA markers and their positions are indicated with the numbers of recombinants (in parentheses). (B) ORFs in the F29 locus delimited by recombination events and LjERN1 protein structure predicted by the nucleotide sequence are shown. The mutation site on F29 and the large deletion found in 1699-1 was indicated by a grey arrowhead and grey parentheses. (C and D) Complementation test. F29 hairy roots transformed with an <em>ERN1</em> construct (C) and with an GUS vector as a negative control (D) were inoculated with <em>M. loti</em> for 32 days. Upper and bottom panels are bright field images and fluorescent images from a GFP transformation marker, respectively. Bars = 2 mm.</p></div></div></div>                            <h3 scrollto-destination=78055009 id="78055009" class="section-title" >3.4. CYCLOPS and NSP2-dependent gene activation of <em>ERN1</em></h3>
<div class="block-child-p">We then investigated the <em>ERN1</em> expression pattern in WT in response to <em>M. loti</em> inoculation. The <em>ERN1</em> gene expressed constitutively in roots, even in the absence of <em>M. loti</em>, and was clearly induced at 12 h after inoculation (<a reveal-id="dsw052-F4" data-open="dsw052-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>). The induction was approximately 3-fold, the transcription levels were maintained at early stages, without being elevated such as the <em>NIN</em> and <em>CLE-RS</em> genes.<a reveal-id="dsw052-B51" data-open="dsw052-B51" class="link link-ref link-reveal xref-bibr"><sup>51</sup></a> To further explore the spatial expression pattern of <em>ERN1</em>, we constructed a GUS reporter carrying the same promoter and terminator used for the complementation, and transformed to the hairy roots via <em>A. rhizogenes</em>. GUS expression from <em>LjERN1</em> promoter was detectable for whole uninoculated roots (<a reveal-id="dsw052-F5" data-open="dsw052-F5" class="link link-reveal link-table xref-fig">Fig. 5A</a>). Interestingly, <em>ProERN1::GUS::TerERN1</em> showed a spatial gradient that the signals were especially high in a developmentally younger region of the root, becoming gradually lower towards the base (<a reveal-id="dsw052-F5" data-open="dsw052-F5" class="link link-reveal link-table xref-fig">Fig. 5A</a>). The root region near the root tip is known to be a susceptible zone for rhizobial infection.<a reveal-id="dsw052-B52" data-open="dsw052-B52" class="link link-ref link-reveal xref-bibr"><sup>52</sup></a><sup>,</sup><a reveal-id="dsw052-B53" data-open="dsw052-B53" class="link link-ref link-reveal xref-bibr"><sup>53</sup></a> In the root tip region shown in <a reveal-id="dsw052-F5" data-open="dsw052-F5" class="link link-reveal link-table xref-fig">Figure 5C</a>, strong GUS signals were detected in the developing root, root hairs, and a root cap, whereas no signal was detected in the root meristem that undergoes cell division. We then evaluated the expression of <em>ProERN1::GUS::TerERN1</em> in inoculated roots at 7 DAI. The expression pattern near root tips at 7 DAI was not significantly changed when compared with uninoculated roots (<a reveal-id="dsw052-F5" data-open="dsw052-F5" class="link link-reveal link-table xref-fig">Fig. 5B</a>), whereas the GUS signal was observed on the curled root hair and was more distinct on the dividing cells in the outer cortex (<a reveal-id="dsw052-F5" data-open="dsw052-F5" class="link link-reveal link-table xref-fig">Fig. 5D</a>). These expression patterns are in good agreement with those of <em>M. truncatula ERN1</em> and <em>ERN2</em> genes.<a reveal-id="dsw052-B37" data-open="dsw052-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a><em>LjERN1</em> expression in nodule primordia resembled that of <em>MtERN1</em> rather than the other because <em>MtERN2</em> expression is closely associated with IT progression. On the other hand, regions expressing <em>LjERN1</em> were broadly expanded within primordia similar to <em>MtERN1</em>. In developing nodules, the GUS signal was restricted to the inside where the infection zone is formed, whereas it was not detected significantly in the peripheral regions of the inoculated root (<a reveal-id="dsw052-F5" data-open="dsw052-F5" class="link link-reveal link-table xref-fig">Fig. 5E</a>). <div data-id="dsw052-F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw052/2/m_dsw052f4.png?Expires=2147483647&amp;Signature=fRUVMuR5FX6zeOZD2hV-B0y2NlvucbRYVPBAlochHBimoLoDu7pl0ClCyP7ZQqpNfl6FDUMAEuBhU3M7CIbY2hSXw9IZ689P4jlWAQQAPUiwBALMBOppm6Sp-X5ZC0sk5QzL2QMarzUbO4TnLX31p9ZZdRKt60Es1wNxK-5T~w-m-C-doGZPcais0QplrFTrYjTpZH9XNEJmqfdci11xfc2bEwBlvT0STN~rBWjwlBpa1DWwx~Q9~HSuslwBOw8DGYrMZx49WhQ7QbJs1GYZojKVwacmJIfv9rdXDKILFKfzn5BBgFKAEObbmrMXSJ~fqbgzDioGi59I9wggGxoTBw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="qRT-PCR analysis of ERN1 expression in WT and symbiotic mutants. Total RNA was extracted from roots at 0, 0.5, 1, and 3 days after inoculation with M. loti MAFF303099. ATP synthase was used as an internal standard. Means and SDs of three independent experiments are shown. One-way ANOVA followed by a Tukey HSD test of the values was performed. Significant differences from WT plants at each time points (*P &lt; 0.05; **P &lt; 0.01), and from mock controls (†P &lt; 0.05; ††P &lt; 0.01) are indicated." path-from-xml="dsw052f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055010/dsw052f4.tif" path-from-xml="dsw052f4.tif" target="_blank">View large</a><a section="78055010" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw052/2/dsw052f4.png?Expires=2147483647&amp;Signature=za3F7mg908~xWVVwdORt5mbsAVId7Qtj2Py3w4NJs2uL9ksrlzqkulNYI273zRp6DY4ZpFt765H1dc-eB80pmv-9HsoIUv6e8yADNqJYgkbe9nCPube~KNNA7UlN8ubdsH3Gw7n70cvPsxxCb8-RDrF~ZhMseSCmPttYI1AAr11U-ZHsHsNNYHWcYmgZ60Dw6y54LQ3Jq68czB0HE4FTYXPw0rq2~bImrewrVXACMskUwYb2LSInjNy6tfN5KoFFnHYbQ-eBOTJynI9Y~NtK2v7pkTDIqch1PUJudfMgrLxnoyGjHDWklzwUBYWZwDr--WL3XShiA~VYTQXihzLIkA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=78055010&amp;ar=2731026&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw052f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>qRT-PCR analysis of <em>ERN1</em> expression in WT and symbiotic mutants. Total RNA was extracted from roots at 0, 0.5, 1, and 3 days after inoculation with <em>M. loti MAFF303099.</em> ATP synthase was used as an internal standard. Means and SDs of three independent experiments are shown. One-way ANOVA followed by a Tukey HSD test of the values was performed. Significant differences from WT plants at each time points (*<em>P</em> &lt; 0.05; **<em>P</em> &lt; 0.01), and from mock controls (†<em>P</em> &lt; 0.05; ††<em>P</em> &lt; 0.01) are indicated.</p></div></div><div content-id="dsw052-F4" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw052/2/m_dsw052f4.png?Expires=2147483647&amp;Signature=fRUVMuR5FX6zeOZD2hV-B0y2NlvucbRYVPBAlochHBimoLoDu7pl0ClCyP7ZQqpNfl6FDUMAEuBhU3M7CIbY2hSXw9IZ689P4jlWAQQAPUiwBALMBOppm6Sp-X5ZC0sk5QzL2QMarzUbO4TnLX31p9ZZdRKt60Es1wNxK-5T~w-m-C-doGZPcais0QplrFTrYjTpZH9XNEJmqfdci11xfc2bEwBlvT0STN~rBWjwlBpa1DWwx~Q9~HSuslwBOw8DGYrMZx49WhQ7QbJs1GYZojKVwacmJIfv9rdXDKILFKfzn5BBgFKAEObbmrMXSJ~fqbgzDioGi59I9wggGxoTBw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="qRT-PCR analysis of ERN1 expression in WT and symbiotic mutants. Total RNA was extracted from roots at 0, 0.5, 1, and 3 days after inoculation with M. loti MAFF303099. ATP synthase was used as an internal standard. Means and SDs of three independent experiments are shown. One-way ANOVA followed by a Tukey HSD test of the values was performed. Significant differences from WT plants at each time points (*P &lt; 0.05; **P &lt; 0.01), and from mock controls (†P &lt; 0.05; ††P &lt; 0.01) are indicated." path-from-xml="dsw052f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055010/dsw052f4.tif" path-from-xml="dsw052f4.tif" target="_blank">View large</a><a section="78055010" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw052/2/dsw052f4.png?Expires=2147483647&amp;Signature=za3F7mg908~xWVVwdORt5mbsAVId7Qtj2Py3w4NJs2uL9ksrlzqkulNYI273zRp6DY4ZpFt765H1dc-eB80pmv-9HsoIUv6e8yADNqJYgkbe9nCPube~KNNA7UlN8ubdsH3Gw7n70cvPsxxCb8-RDrF~ZhMseSCmPttYI1AAr11U-ZHsHsNNYHWcYmgZ60Dw6y54LQ3Jq68czB0HE4FTYXPw0rq2~bImrewrVXACMskUwYb2LSInjNy6tfN5KoFFnHYbQ-eBOTJynI9Y~NtK2v7pkTDIqch1PUJudfMgrLxnoyGjHDWklzwUBYWZwDr--WL3XShiA~VYTQXihzLIkA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=78055010&amp;ar=2731026&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw052f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>qRT-PCR analysis of <em>ERN1</em> expression in WT and symbiotic mutants. Total RNA was extracted from roots at 0, 0.5, 1, and 3 days after inoculation with <em>M. loti MAFF303099.</em> ATP synthase was used as an internal standard. Means and SDs of three independent experiments are shown. One-way ANOVA followed by a Tukey HSD test of the values was performed. Significant differences from WT plants at each time points (*<em>P</em> &lt; 0.05; **<em>P</em> &lt; 0.01), and from mock controls (†<em>P</em> &lt; 0.05; ††<em>P</em> &lt; 0.01) are indicated.</p></div></div><div data-id="dsw052-F5" class="fig fig-section"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw052/2/m_dsw052f5.png?Expires=2147483647&amp;Signature=RdG9SINjiMPJEj8ziX7osbIuVdlmGbKwiK3CttU8fxnHSD86JGEBR0yNOua0iI8y6xPAyIoJRVg9efXizISyd9-urtTNnaTunxdhL6wBilLSla6NyFPbVEzyHltVI4hccppwR0M0J-xtZ~9y1nsaJ0yIXKWFY893qUzQg~ILnvPQSswJIh5ck9JD57rDCcIZxfr-8FTSUWCnQDbng5FTh1T~2pMLgJfPDaDu4buBNIxUcKGGadIWga27t6A1YphkdsKmQCn7rTLEVVMXmvMT1NRATKJCAXJf-e5e8IWhY12B2rdoqKNkcsHUReGF5S7WBHgoMG3OoRkiiGbhqIwTUA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="GUS expression from ERN1 promoter after inoculation with M. loti. hairy roots carrying ProERN1::GUS::TerERN1 were inoculated without (A and C) or with M. loti MAFF303099 (B, D, and E) for 7 days. Bars: 1.0 mm in (A) and (B), 0.2 mm in C–E." path-from-xml="dsw052f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055010/dsw052f5.tif" path-from-xml="dsw052f5.tif" target="_blank">View large</a><a section="78055010" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw052/2/dsw052f5.png?Expires=2147483647&amp;Signature=y4cMTV7lbYelG~jxugtBl-b4CI4-UgEEbHn7d8B8aOeKJ~3gv8kGuU8phn4cR--7sEP0dpKKUR7CCPDi49-1u1bItRQlUl7-mPywr2VmKf6w~5s8mz863S5L9XaShURm2VAr41edLd~W-xeo-BOANk1Y5JZpRImM-31qPdHQQ3QWuEW3SPaLI8xmf3AFDSiLaLeiGzwygfYdTPdbfPtBSY7oeHnW0FKidLu4jaGKArvSIfkPRmYbUeCJS5MWz8a0UCwyB7uVsidPeYNo3~RVh7qF9PUWIc42npyx7H4eE5mnVtXae3Ir73Dd1V4gA~MZYgl2oBXtt0x4ehPazoCK7A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=78055010&amp;ar=2731026&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw052f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>GUS expression from <em>ERN1</em> promoter after inoculation with <em>M. loti.</em> hairy roots carrying <em>ProERN1::GUS::TerERN1</em> were inoculated without (A and C) or with <em>M. loti</em> MAFF303099 (B, D, and E) for 7 days. Bars: 1.0 mm in (A) and (B), 0.2 mm in C–E.</p></div></div><div content-id="dsw052-F5" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw052/2/m_dsw052f5.png?Expires=2147483647&amp;Signature=RdG9SINjiMPJEj8ziX7osbIuVdlmGbKwiK3CttU8fxnHSD86JGEBR0yNOua0iI8y6xPAyIoJRVg9efXizISyd9-urtTNnaTunxdhL6wBilLSla6NyFPbVEzyHltVI4hccppwR0M0J-xtZ~9y1nsaJ0yIXKWFY893qUzQg~ILnvPQSswJIh5ck9JD57rDCcIZxfr-8FTSUWCnQDbng5FTh1T~2pMLgJfPDaDu4buBNIxUcKGGadIWga27t6A1YphkdsKmQCn7rTLEVVMXmvMT1NRATKJCAXJf-e5e8IWhY12B2rdoqKNkcsHUReGF5S7WBHgoMG3OoRkiiGbhqIwTUA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="GUS expression from ERN1 promoter after inoculation with M. loti. hairy roots carrying ProERN1::GUS::TerERN1 were inoculated without (A and C) or with M. loti MAFF303099 (B, D, and E) for 7 days. Bars: 1.0 mm in (A) and (B), 0.2 mm in C–E." path-from-xml="dsw052f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055010/dsw052f5.tif" path-from-xml="dsw052f5.tif" target="_blank">View large</a><a section="78055010" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw052/2/dsw052f5.png?Expires=2147483647&amp;Signature=y4cMTV7lbYelG~jxugtBl-b4CI4-UgEEbHn7d8B8aOeKJ~3gv8kGuU8phn4cR--7sEP0dpKKUR7CCPDi49-1u1bItRQlUl7-mPywr2VmKf6w~5s8mz863S5L9XaShURm2VAr41edLd~W-xeo-BOANk1Y5JZpRImM-31qPdHQQ3QWuEW3SPaLI8xmf3AFDSiLaLeiGzwygfYdTPdbfPtBSY7oeHnW0FKidLu4jaGKArvSIfkPRmYbUeCJS5MWz8a0UCwyB7uVsidPeYNo3~RVh7qF9PUWIc42npyx7H4eE5mnVtXae3Ir73Dd1V4gA~MZYgl2oBXtt0x4ehPazoCK7A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=78055010&amp;ar=2731026&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw052f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>GUS expression from <em>ERN1</em> promoter after inoculation with <em>M. loti.</em> hairy roots carrying <em>ProERN1::GUS::TerERN1</em> were inoculated without (A and C) or with <em>M. loti</em> MAFF303099 (B, D, and E) for 7 days. Bars: 1.0 mm in (A) and (B), 0.2 mm in C–E.</p></div></div></div><p>To identify what components of the Nod factor signaling pathway activate <em>ERN1</em> induction in response to rhizobia, we performed an expression analysis of <em>ERN1</em> by qRT-PCR using five mutants defective in IT development positioned downstream of CCaMK. <em>ERN1</em> was induced at early stages in <em>nin-2</em>, <em>cerberus-5</em>, and <em>alb-3</em> mutants, up to the similar levels as the WT. The induction levels of <em>ERN1</em> in <em>nsp2-1</em> mutant were almost one-third or one-half that in the WT (<a reveal-id="dsw052-F4" data-open="dsw052-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>). Notably, no induction was observed in the <em>cyclops-3</em> mutant (<a reveal-id="dsw052-F4" data-open="dsw052-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>). These results indicate a CYCLOPS and NSP2-dependent induction of <em>ERN1</em> expression.</p>                            <h3 scrollto-destination=78055012 id="78055012" class="section-title" >3.5. Syntenic relationship of the genome regions around <em>ERN</em> genes of <em>L. japonicus, M. truncatula</em> and <em>G. max</em></h3>
<p>ERN2, an ERF/AP2 transcription factor closely related to ERN1, exists wildly in legumes.<a reveal-id="dsw052-B54" data-open="dsw052-B54" class="link link-ref link-reveal xref-bibr"><sup>54</sup></a> In <em>M. truncatula</em>, it has been shown to be able to bind to the NF box, and activate <em>ENOD11</em> transcription<a reveal-id="dsw052-B36" data-open="dsw052-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> and to possess similar biological activities with ERN1.<a reveal-id="dsw052-B54" data-open="dsw052-B54" class="link link-ref link-reveal xref-bibr"><sup>54</sup></a> The double-mutant analyses indicated that ERN1/ERN2 play essential roles for initiating root hair infection and root nodule development.<a reveal-id="dsw052-B55" data-open="dsw052-B55" class="link link-ref link-reveal xref-bibr"><sup>55</sup></a> Thus, we investigated the <em>ERN2</em> ortholog in <em>L. japonicus</em>. By searching the latest genome sequences of <em>L. japonicus</em> (Lj_ver3.0) using MtERN2 (Medtr6g029180.1) as query, a top BLASTP hit was obtained against <em>LjERN1</em> (Lj1g3v3975310.1). The same result was obtained in the tBLASTn search against the transcriptome assembly of <em>L. japonicus.</em></p><p>To confirm an orthologus relationship between <em>LjERN1</em> (Lj1g3v3975310.1) and <em>MtERN1</em> (Medtr7g085820.1), the syntenic relationships between the genome regions flanking to these genes were analysed. Using a stepwise gene-by-gene reciprocal comparison of the regions, we observed strong conserved microsynteny between these regions (<a reveal-id="dsw052-T1" data-open="dsw052-T1" class="link link-reveal link-table xref-fig">Table 1</a>). As these genome regions correspond to the large synteny block between chromosome 1 of <em>L. japonicus</em> and chromosome 7 of <em>M. truncatula</em> reported by Cannon <em>et al.</em>,<a reveal-id="dsw052-B56" data-open="dsw052-B56" class="link link-ref link-reveal xref-bibr"><sup>56</sup></a> the synteic relation extended to chromosomal level. Therefore, the orthologous relationship between <em>LjERN1</em> and <em>MtERN1</em> was confirmed from the viewpoint of genome synteny. The syntenic relationship surrounding the <em>ERN1</em> gene was also observed in the corresponding genome regions in <em>G. max</em> (flanking regions of Glyma.16G040000.1 and Glyma.19G113200.1). <div content-id="dsw052-T1" class="table-modal"><div class="table-wrap"><div id="dsw052-T1" data-id="dsw052-T1" class="table-wrap-title"><div class="label -label">Table 1.</div><div class="caption"><p>Syntenic relation in the genome regions of <em>ERN</em> genes of <em>L. japonicus, M. truncatula</em>, and <em>G. max</em></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Annotation </th><th colspan="4">ERN1<hr /> </th><th colspan="4">ERN2<hr /> </th></tr><tr><th>Lj_chr3 </th><th>Mt_chr7 </th><th>Gm_chr16 </th><th>Gm_chr19 </th><th>Lj </th><th>Mt_chr6 </th><th>Gm_chr2 </th><th>Gm_chr16 </th></tr></thead><tbody><tr><td>Transmembrane protein, putative </td><td>Lj1g3v3975270.2 </td><td>Medtr7g085790.1 </td><td>– </td><td>Glyma.19G112800.1 </td><td>– </td><td>Medtr6g029250.1 </td><td>Glyma.02G072500.1 </td><td>Glyma.16G153700.1 </td></tr><tr><td>5-Adenylylsulphate reductase-like protein </td><td>Lj1g3v3975280.1 </td><td>– </td><td>Glyma.16G040200.1 </td><td>Glyma.19G112900.1 </td><td>– </td><td>Medtr6g029240.1 </td><td>Glyma.02G072600.1 </td><td> </td></tr><tr><td>Tubulin/FtsZ family protein </td><td>Lj1g3v3975290.1 </td><td>Medtr7g085800.1 </td><td>Glyma.16G040100.1 </td><td>Glyma.19G113000.1 </td><td>– </td><td>Medtr6g029190.1 </td><td>– </td><td>Glyma.16G154000.1 </td></tr><tr><td>AP2/ERF domain transcription factor </td><td>Lj1g3v3975310.1 (<em>LjERN1</em>) </td><td>Medtr7g085810.1 (<em>MtERN1</em>) </td><td>Glyma.16G040000.1 </td><td>Glyma.19G113100.1 </td><td>– </td><td>Medtr6g029180.1 (<em>MtERN2</em>) </td><td>Glyma.02G072800.1 </td><td>Glyma.16G154100.1 </td></tr><tr><td>Importin-like protein </td><td>Lj1g3v3975340.2 </td><td>Medtr7g085820.1 </td><td>Glyma.16G039900.1 </td><td>Glyma.19G113200.1 </td><td>– </td><td>– </td><td>– </td><td>– </td></tr><tr><td>glycosyltransferase family protein </td><td>Lj1g3v3975450.1 </td><td>Medtr7g085840.1 </td><td>Glyma.16G039800.1 </td><td>Glyma.19G113300.1 </td><td>– </td><td>– </td><td>– </td><td>– </td></tr><tr><td>Hypothetical protein </td><td>Lj1g3v3975460.1 </td><td>Medtr7g085850.1 </td><td>– </td><td>Glyma.19G113400.1 </td><td>– </td><td>– </td><td>– </td><td>– </td></tr></tbody></table></div><div content-id="dsw052-T1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Annotation </th><th colspan="4">ERN1<hr /> </th><th colspan="4">ERN2<hr /> </th></tr><tr><th>Lj_chr3 </th><th>Mt_chr7 </th><th>Gm_chr16 </th><th>Gm_chr19 </th><th>Lj </th><th>Mt_chr6 </th><th>Gm_chr2 </th><th>Gm_chr16 </th></tr></thead><tbody><tr><td>Transmembrane protein, putative </td><td>Lj1g3v3975270.2 </td><td>Medtr7g085790.1 </td><td>– </td><td>Glyma.19G112800.1 </td><td>– </td><td>Medtr6g029250.1 </td><td>Glyma.02G072500.1 </td><td>Glyma.16G153700.1 </td></tr><tr><td>5-Adenylylsulphate reductase-like protein </td><td>Lj1g3v3975280.1 </td><td>– </td><td>Glyma.16G040200.1 </td><td>Glyma.19G112900.1 </td><td>– </td><td>Medtr6g029240.1 </td><td>Glyma.02G072600.1 </td><td> </td></tr><tr><td>Tubulin/FtsZ family protein </td><td>Lj1g3v3975290.1 </td><td>Medtr7g085800.1 </td><td>Glyma.16G040100.1 </td><td>Glyma.19G113000.1 </td><td>– </td><td>Medtr6g029190.1 </td><td>– </td><td>Glyma.16G154000.1 </td></tr><tr><td>AP2/ERF domain transcription factor </td><td>Lj1g3v3975310.1 (<em>LjERN1</em>) </td><td>Medtr7g085810.1 (<em>MtERN1</em>) </td><td>Glyma.16G040000.1 </td><td>Glyma.19G113100.1 </td><td>– </td><td>Medtr6g029180.1 (<em>MtERN2</em>) </td><td>Glyma.02G072800.1 </td><td>Glyma.16G154100.1 </td></tr><tr><td>Importin-like protein </td><td>Lj1g3v3975340.2 </td><td>Medtr7g085820.1 </td><td>Glyma.16G039900.1 </td><td>Glyma.19G113200.1 </td><td>– </td><td>– </td><td>– </td><td>– </td></tr><tr><td>glycosyltransferase family protein </td><td>Lj1g3v3975450.1 </td><td>Medtr7g085840.1 </td><td>Glyma.16G039800.1 </td><td>Glyma.19G113300.1 </td><td>– </td><td>– </td><td>– </td><td>– </td></tr><tr><td>Hypothetical protein </td><td>Lj1g3v3975460.1 </td><td>Medtr7g085850.1 </td><td>– </td><td>Glyma.19G113400.1 </td><td>– </td><td>– </td><td>– </td><td>– </td></tr></tbody></table></div></div></div><div class="table-wrap"><div id="dsw052-T1" data-id="dsw052-T1" class="table-wrap-title"><div class="label -label">Table 1.</div><div class="caption"><p>Syntenic relation in the genome regions of <em>ERN</em> genes of <em>L. japonicus, M. truncatula</em>, and <em>G. max</em></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Annotation </th><th colspan="4">ERN1<hr /> </th><th colspan="4">ERN2<hr /> </th></tr><tr><th>Lj_chr3 </th><th>Mt_chr7 </th><th>Gm_chr16 </th><th>Gm_chr19 </th><th>Lj </th><th>Mt_chr6 </th><th>Gm_chr2 </th><th>Gm_chr16 </th></tr></thead><tbody><tr><td>Transmembrane protein, putative </td><td>Lj1g3v3975270.2 </td><td>Medtr7g085790.1 </td><td>– </td><td>Glyma.19G112800.1 </td><td>– </td><td>Medtr6g029250.1 </td><td>Glyma.02G072500.1 </td><td>Glyma.16G153700.1 </td></tr><tr><td>5-Adenylylsulphate reductase-like protein </td><td>Lj1g3v3975280.1 </td><td>– </td><td>Glyma.16G040200.1 </td><td>Glyma.19G112900.1 </td><td>– </td><td>Medtr6g029240.1 </td><td>Glyma.02G072600.1 </td><td> </td></tr><tr><td>Tubulin/FtsZ family protein </td><td>Lj1g3v3975290.1 </td><td>Medtr7g085800.1 </td><td>Glyma.16G040100.1 </td><td>Glyma.19G113000.1 </td><td>– </td><td>Medtr6g029190.1 </td><td>– </td><td>Glyma.16G154000.1 </td></tr><tr><td>AP2/ERF domain transcription factor </td><td>Lj1g3v3975310.1 (<em>LjERN1</em>) </td><td>Medtr7g085810.1 (<em>MtERN1</em>) </td><td>Glyma.16G040000.1 </td><td>Glyma.19G113100.1 </td><td>– </td><td>Medtr6g029180.1 (<em>MtERN2</em>) </td><td>Glyma.02G072800.1 </td><td>Glyma.16G154100.1 </td></tr><tr><td>Importin-like protein </td><td>Lj1g3v3975340.2 </td><td>Medtr7g085820.1 </td><td>Glyma.16G039900.1 </td><td>Glyma.19G113200.1 </td><td>– </td><td>– </td><td>– </td><td>– </td></tr><tr><td>glycosyltransferase family protein </td><td>Lj1g3v3975450.1 </td><td>Medtr7g085840.1 </td><td>Glyma.16G039800.1 </td><td>Glyma.19G113300.1 </td><td>– </td><td>– </td><td>– </td><td>– </td></tr><tr><td>Hypothetical protein </td><td>Lj1g3v3975460.1 </td><td>Medtr7g085850.1 </td><td>– </td><td>Glyma.19G113400.1 </td><td>– </td><td>– </td><td>– </td><td>– </td></tr></tbody></table></div><div content-id="dsw052-T1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Annotation </th><th colspan="4">ERN1<hr /> </th><th colspan="4">ERN2<hr /> </th></tr><tr><th>Lj_chr3 </th><th>Mt_chr7 </th><th>Gm_chr16 </th><th>Gm_chr19 </th><th>Lj </th><th>Mt_chr6 </th><th>Gm_chr2 </th><th>Gm_chr16 </th></tr></thead><tbody><tr><td>Transmembrane protein, putative </td><td>Lj1g3v3975270.2 </td><td>Medtr7g085790.1 </td><td>– </td><td>Glyma.19G112800.1 </td><td>– </td><td>Medtr6g029250.1 </td><td>Glyma.02G072500.1 </td><td>Glyma.16G153700.1 </td></tr><tr><td>5-Adenylylsulphate reductase-like protein </td><td>Lj1g3v3975280.1 </td><td>– </td><td>Glyma.16G040200.1 </td><td>Glyma.19G112900.1 </td><td>– </td><td>Medtr6g029240.1 </td><td>Glyma.02G072600.1 </td><td> </td></tr><tr><td>Tubulin/FtsZ family protein </td><td>Lj1g3v3975290.1 </td><td>Medtr7g085800.1 </td><td>Glyma.16G040100.1 </td><td>Glyma.19G113000.1 </td><td>– </td><td>Medtr6g029190.1 </td><td>– </td><td>Glyma.16G154000.1 </td></tr><tr><td>AP2/ERF domain transcription factor </td><td>Lj1g3v3975310.1 (<em>LjERN1</em>) </td><td>Medtr7g085810.1 (<em>MtERN1</em>) </td><td>Glyma.16G040000.1 </td><td>Glyma.19G113100.1 </td><td>– </td><td>Medtr6g029180.1 (<em>MtERN2</em>) </td><td>Glyma.02G072800.1 </td><td>Glyma.16G154100.1 </td></tr><tr><td>Importin-like protein </td><td>Lj1g3v3975340.2 </td><td>Medtr7g085820.1 </td><td>Glyma.16G039900.1 </td><td>Glyma.19G113200.1 </td><td>– </td><td>– </td><td>– </td><td>– </td></tr><tr><td>glycosyltransferase family protein </td><td>Lj1g3v3975450.1 </td><td>Medtr7g085840.1 </td><td>Glyma.16G039800.1 </td><td>Glyma.19G113300.1 </td><td>– </td><td>– </td><td>– </td><td>– </td></tr><tr><td>Hypothetical protein </td><td>Lj1g3v3975460.1 </td><td>Medtr7g085850.1 </td><td>– </td><td>Glyma.19G113400.1 </td><td>– </td><td>– </td><td>– </td><td>– </td></tr></tbody></table></div></div></p><p>Regarding the genome region of the <em>ERN2</em> gene, genes orthologous to <em>MtERN2</em> were identified in the <em>G. max</em> genome (Glyma.02G072800.1 and Glyma.16G154100.1), and a syntenic relationship between <em>ERN1</em> and <em>ERN2</em> genome regions was observed (<a reveal-id="dsw052-T1" data-open="dsw052-T1" class="link link-reveal link-table xref-fig">Table 1</a>). Since the candidate gene orthologous to any of these flanking genes was not identified in <em>L. japonicus</em> genome sequences, the genome region corresponding to the <em>ERN2</em> gene would probably be deleted in the genome of <em>L. japonicus</em>.</p>                            <h3 scrollto-destination=78055016 id="78055016" class="section-title" >3.6. <em>ERN1</em> and <em>ERN2</em> clades resulting from gene duplication in legume evolution</h3>
<div class="block-child-p">We estimated the phylogenetic tree of <em>ERN</em> genes using all genomic datasets in NCBI RefSeq (<a reveal-id="dsw052-F6" data-open="dsw052-F6" class="link link-reveal link-table xref-fig">Fig. 6</a>). ML, Bayesian inference, and NJ methods estimated essentially similar tree topologies (<a reveal-id="dsw052-F6" data-open="dsw052-F6" class="link link-reveal link-table xref-fig">Fig. 6</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw052/2/dsw052_Supp.zip?Expires=2147483647&amp;Signature=SUfZUEyOx0cNgvbTgqmG5Jlc7D-wCgPBRu~nkZYtDlnDJ3y08Dk1egjJie4Bi9r0o~3T21J6w8mfvpFrbyNAq2drb5sFojcQL96jELnU9bogfK5Fi6LNtPAWKyHiFJfKZuy9UtsMZnYsh1Y3IfDP7q7xrQpM6Tdb~N-zpw88J5aGu5UvF0EP7Aurb-xgRQ4kG3PjHn5IFQBvYeAj7yMCbHPe5dB-ORrJjjGmlosU5TPSzRe3LJ9SSIUQBp5XOPrQ-r~s3VpZ6k98RKm2Ew2J5eCHjxuifUtwtPKDFIBqI6CYZsVtizidvxoFwOjgWAfKnY5U7a7aAxOp4ZGx9gcKTg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figures S1 and S2</a></span>). The estimated trees strongly suggested that a gene duplication event produced the <em>ERN1</em> and <em>ERN2</em> clades after the divergence of legume plants in Fabales from the others (rosid I)<a reveal-id="dsw052-B57" data-open="dsw052-B57" class="link link-ref link-reveal xref-bibr"><sup>57</sup></a>. It should be noted that LjERN1 of <em>L. japonicus</em> is included in the ERN1 clade. All the genome-analysed legume species harbored both the <em>ERN1</em> and <em>ERN2</em> genes except for <em>L. japonicus</em>. <div data-id="dsw052-F6" class="fig fig-section"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw052/2/m_dsw052f6.png?Expires=2147483647&amp;Signature=rt1rvKRGdZl5xNNCUK2UqXawghgeeTb3~fZvFz9evwEEOG3xaorSN6y8ADPl9AYH9xI~12EdsnFBJg99~7UAi0rXBPaegTSD2ljqgk11jEXFvEMMJKbSvgjvD26vMboYV9RoF0qYMZt1d7a27YFUkHgE1YXJ9ZHk5j60MgLoHSh0nX-qL7-YU35SpimVtDSRW0iPNFe0naJUr9O0ztlwLgXQXHGrgjekRINJUmuYJhouH4Q-Zw72pLrwvQngPWCU3bl8q88T8abs89b7VqkS98PqNOH4-l7GWoXZrhtpcoOFZoYeLiLTHQooAP0p05WSFfJcGjRE5Rx-8i9gzumt5g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogenetic tree from the homologs of ERN1 and ERN2 genes inferred by ML method with LG model using RAxML program. The arrow indicates the duplication to the ERN1 and ERN2 lineages. ERN1 and ERN2 genes of Fabaceae are indicated with red and blue letters, respectively. The first and third numbers given at the relevant nodes correspond to the bootstrap support from the ML and NJ methods, respectively, and the second to the posterior probability from the Bayesian method. Phylogenetic trees inferred by the Bayesian and NJ methods are available in supplementary figures. The assembly accession number and accession version of the genes is following to the species name. The scale bar represents substitutions per site. " path-from-xml="dsw052f6.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055017/dsw052f6.tif" path-from-xml="dsw052f6.tif" target="_blank">View large</a><a section="78055017" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw052/2/dsw052f6.png?Expires=2147483647&amp;Signature=vbA9WWx-Dfn1GcsCsApLitNYFcJBDu3gN~1lLdaDPT~V7tJldWYN~1fXJws7oFexRYUYTldqnOKAM~ZlRsjXoKRImKWHK2lnhXt91P6lqx-CqiQKYUIx3BUYqoAWXvU1XE1C3-ZqH-UYlUiO0Gsu9yJl5h7UlURL7L~rga9Iav6~1Ns3nJZGKyCcVdIa4oK2AhjGeA20uuQPO~FpwPJYX0KEGC32cedoo9l8q1fTeyKtvZAT9Bt8ZbyZusrC8ZTSK5eRPG8sYxgYb19jPpPd5j8UObibfsHjAoTd7-mKlA1eQ0ejyj6rmWUY2jCENfZyh7ih7e57RB7B9X0Ec-IrgA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=78055017&amp;ar=2731026&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw052f6.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic tree from the homologs of <em>ERN1</em> and <em>ERN2</em> genes inferred by ML method with LG model using RAxML program. The arrow indicates the duplication to the <em>ERN1</em> and <em>ERN2</em> lineages. <em>ERN1</em> and <em>ERN2</em> genes of Fabaceae are indicated with red and blue letters, respectively. The first and third numbers given at the relevant nodes correspond to the bootstrap support from the ML and NJ methods, respectively, and the second to the posterior probability from the Bayesian method. Phylogenetic trees inferred by the Bayesian and NJ methods are available in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw052/2/dsw052_Supp.zip?Expires=2147483647&amp;Signature=SUfZUEyOx0cNgvbTgqmG5Jlc7D-wCgPBRu~nkZYtDlnDJ3y08Dk1egjJie4Bi9r0o~3T21J6w8mfvpFrbyNAq2drb5sFojcQL96jELnU9bogfK5Fi6LNtPAWKyHiFJfKZuy9UtsMZnYsh1Y3IfDP7q7xrQpM6Tdb~N-zpw88J5aGu5UvF0EP7Aurb-xgRQ4kG3PjHn5IFQBvYeAj7yMCbHPe5dB-ORrJjjGmlosU5TPSzRe3LJ9SSIUQBp5XOPrQ-r~s3VpZ6k98RKm2Ew2J5eCHjxuifUtwtPKDFIBqI6CYZsVtizidvxoFwOjgWAfKnY5U7a7aAxOp4ZGx9gcKTg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">supplementary figures</a></span>. The assembly accession number and accession version of the genes is following to the species name. The scale bar represents substitutions per site. </p></div></div><div content-id="dsw052-F6" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw052/2/m_dsw052f6.png?Expires=2147483647&amp;Signature=rt1rvKRGdZl5xNNCUK2UqXawghgeeTb3~fZvFz9evwEEOG3xaorSN6y8ADPl9AYH9xI~12EdsnFBJg99~7UAi0rXBPaegTSD2ljqgk11jEXFvEMMJKbSvgjvD26vMboYV9RoF0qYMZt1d7a27YFUkHgE1YXJ9ZHk5j60MgLoHSh0nX-qL7-YU35SpimVtDSRW0iPNFe0naJUr9O0ztlwLgXQXHGrgjekRINJUmuYJhouH4Q-Zw72pLrwvQngPWCU3bl8q88T8abs89b7VqkS98PqNOH4-l7GWoXZrhtpcoOFZoYeLiLTHQooAP0p05WSFfJcGjRE5Rx-8i9gzumt5g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogenetic tree from the homologs of ERN1 and ERN2 genes inferred by ML method with LG model using RAxML program. The arrow indicates the duplication to the ERN1 and ERN2 lineages. ERN1 and ERN2 genes of Fabaceae are indicated with red and blue letters, respectively. The first and third numbers given at the relevant nodes correspond to the bootstrap support from the ML and NJ methods, respectively, and the second to the posterior probability from the Bayesian method. Phylogenetic trees inferred by the Bayesian and NJ methods are available in supplementary figures. The assembly accession number and accession version of the genes is following to the species name. The scale bar represents substitutions per site. " path-from-xml="dsw052f6.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055017/dsw052f6.tif" path-from-xml="dsw052f6.tif" target="_blank">View large</a><a section="78055017" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw052/2/dsw052f6.png?Expires=2147483647&amp;Signature=vbA9WWx-Dfn1GcsCsApLitNYFcJBDu3gN~1lLdaDPT~V7tJldWYN~1fXJws7oFexRYUYTldqnOKAM~ZlRsjXoKRImKWHK2lnhXt91P6lqx-CqiQKYUIx3BUYqoAWXvU1XE1C3-ZqH-UYlUiO0Gsu9yJl5h7UlURL7L~rga9Iav6~1Ns3nJZGKyCcVdIa4oK2AhjGeA20uuQPO~FpwPJYX0KEGC32cedoo9l8q1fTeyKtvZAT9Bt8ZbyZusrC8ZTSK5eRPG8sYxgYb19jPpPd5j8UObibfsHjAoTd7-mKlA1eQ0ejyj6rmWUY2jCENfZyh7ih7e57RB7B9X0Ec-IrgA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=78055017&amp;ar=2731026&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw052f6.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic tree from the homologs of <em>ERN1</em> and <em>ERN2</em> genes inferred by ML method with LG model using RAxML program. The arrow indicates the duplication to the <em>ERN1</em> and <em>ERN2</em> lineages. <em>ERN1</em> and <em>ERN2</em> genes of Fabaceae are indicated with red and blue letters, respectively. The first and third numbers given at the relevant nodes correspond to the bootstrap support from the ML and NJ methods, respectively, and the second to the posterior probability from the Bayesian method. Phylogenetic trees inferred by the Bayesian and NJ methods are available in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw052/2/dsw052_Supp.zip?Expires=2147483647&amp;Signature=SUfZUEyOx0cNgvbTgqmG5Jlc7D-wCgPBRu~nkZYtDlnDJ3y08Dk1egjJie4Bi9r0o~3T21J6w8mfvpFrbyNAq2drb5sFojcQL96jELnU9bogfK5Fi6LNtPAWKyHiFJfKZuy9UtsMZnYsh1Y3IfDP7q7xrQpM6Tdb~N-zpw88J5aGu5UvF0EP7Aurb-xgRQ4kG3PjHn5IFQBvYeAj7yMCbHPe5dB-ORrJjjGmlosU5TPSzRe3LJ9SSIUQBp5XOPrQ-r~s3VpZ6k98RKm2Ew2J5eCHjxuifUtwtPKDFIBqI6CYZsVtizidvxoFwOjgWAfKnY5U7a7aAxOp4ZGx9gcKTg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">supplementary figures</a></span>. The assembly accession number and accession version of the genes is following to the species name. The scale bar represents substitutions per site. </p></div></div></div>                            <h3 scrollto-destination=78055018 id="78055018" class="section-title" >3.7. Difference of functional constraint between <em>ERN1</em> and <em>ERN2</em> genes</h3>
<p>The phylogenetic tree shape of the <em>ERN</em> genes suggests divergence of evolutionary rates between the <em>ERN1</em> and <em>ERN2</em> genes (<a reveal-id="dsw052-F6" data-open="dsw052-F6" class="link link-reveal link-table xref-fig">Fig. 6</a>); the branches in the ERN2 clade are longer (i.e. more substitutions per site) than those in the ERN1 clade. In general, variations in substitution rates can be due to different nonsynonymous/synonymous rate ratios (ω = dn/ds).<a reveal-id="dsw052-B27" data-open="dsw052-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a><sup>,</sup><a reveal-id="dsw052-B58 dsw052-B59 dsw052-B60" data-open="dsw052-B58 dsw052-B59 dsw052-B60" class="link link-ref link-reveal xref-bibr"><sup>58–60</sup></a> The branch model of PAML was adopted to consider different ω in the tree and five following models were compared: (i) one ratio across the tree; (ii) two different ratios between the ERN1 clade and the other parts; (iii) two different ratios between the ERN2 clade and the other parts; (iv) two different ratios between the ERN1/ERN2 clade and the other parts; and (v) three different ratios between the ERN1, ERN2 clade, and the other parts (<a reveal-id="dsw052-T2" data-open="dsw052-T2" class="link link-reveal link-table xref-fig">Table 2</a>).<a reveal-id="dsw052-B47" data-open="dsw052-B47" class="link link-ref link-reveal xref-bibr"><sup>47</sup></a><sup>,</sup><a reveal-id="dsw052-B61" data-open="dsw052-B61" class="link link-ref link-reveal xref-bibr"><sup>61</sup></a> The LRT was conducted to compare nested models and showed no significant likelihood differences between the models 1 and 2 and between 3 and 5. On the other hand, the tests rejected the null hypotheses for all other combinations (<a reveal-id="dsw052-T2" data-open="dsw052-T2" class="link link-reveal link-table xref-fig">Table 2</a>) and Akaike's and Bayesian information criteria favored the model 3. These results suggested that the ω of the genes in the ERN2 clade is greater than the background ratio and that ω of the genes in the ERN1 clade is almost identical to that of the genes of nonlegumes. The clade model of PAML also showed similar results (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw052/2/dsw052_Supp.zip?Expires=2147483647&Signature=SUfZUEyOx0cNgvbTgqmG5Jlc7D-wCgPBRu~nkZYtDlnDJ3y08Dk1egjJie4Bi9r0o~3T21J6w8mfvpFrbyNAq2drb5sFojcQL96jELnU9bogfK5Fi6LNtPAWKyHiFJfKZuy9UtsMZnYsh1Y3IfDP7q7xrQpM6Tdb~N-zpw88J5aGu5UvF0EP7Aurb-xgRQ4kG3PjHn5IFQBvYeAj7yMCbHPe5dB-ORrJjjGmlosU5TPSzRe3LJ9SSIUQBp5XOPrQ-r~s3VpZ6k98RKm2Ew2J5eCHjxuifUtwtPKDFIBqI6CYZsVtizidvxoFwOjgWAfKnY5U7a7aAxOp4ZGx9gcKTg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). No positively selected site was not detected in the <em>ERN2</em> genes with branch-site and clade models (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw052/2/dsw052_Supp.zip?Expires=2147483647&Signature=SUfZUEyOx0cNgvbTgqmG5Jlc7D-wCgPBRu~nkZYtDlnDJ3y08Dk1egjJie4Bi9r0o~3T21J6w8mfvpFrbyNAq2drb5sFojcQL96jELnU9bogfK5Fi6LNtPAWKyHiFJfKZuy9UtsMZnYsh1Y3IfDP7q7xrQpM6Tdb~N-zpw88J5aGu5UvF0EP7Aurb-xgRQ4kG3PjHn5IFQBvYeAj7yMCbHPe5dB-ORrJjjGmlosU5TPSzRe3LJ9SSIUQBp5XOPrQ-r~s3VpZ6k98RKm2Ew2J5eCHjxuifUtwtPKDFIBqI6CYZsVtizidvxoFwOjgWAfKnY5U7a7aAxOp4ZGx9gcKTg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). <div content-id="dsw052-T2" class="table-modal"><div class="table-wrap"><div id="dsw052-T2" data-id="dsw052-T2" class="table-wrap-title"><div class="label -label">Table 2.</div><div class="caption"><p>Model comparison of <em>ERN</em> genes estimate by PAML using branch model</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Model </th><th rowspan="2">(i) One-ratio model (M0 of Yang <em>et al.</em><a reveal-id="dsw052-B47" data-open="dsw052-B47" class="link link-ref link-reveal xref-bibr"><sup>47</sup></a>) </th><th colspan="3">Two ratio model<hr /> </th><th>(v) Three ratio model </th></tr><tr><th>(ii) ERN1 </th><th>(iii) ERN2 </th><th>(iv) ERN1 + ERN2 </th><th>ERN1, ERN2 </th></tr></thead><tbody><tr><td>np </td><td>72 </td><td>73 </td><td>73 </td><td>73 </td><td>74 </td></tr><tr><td>ln L </td><td>−15088.32041 </td><td>−15087.59693 </td><td>−15063.74053 </td><td>−15076.07402 </td><td>−15063.08349 </td></tr><tr><td>kappa (ts/tv) </td><td>1.58345 </td><td>1.5825 </td><td>1.58201 </td><td>1.58408 </td><td>1.58288 </td></tr><tr><td>omega (dN/dS) </td><td>0.28104 </td><td>ω0 = 0.28866, ω1 = 0.25656 </td><td>ω0 = 0.23609, ω1 = 0.45111 </td><td>ω0 = 0.22744, ω1 = 0.34265 </td><td>ω0 = 0.22749, ω1 = 0.25621, ω2 = 0.45081 </td></tr><tr><td>LRT with 2ΔL and P value </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> Two-ratio model (ERN1) </td><td>−1.447 (P = 0.229) </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> Two-ratio model (ERN2) </td><td>−49.16 (<em>P</em> &lt; 10<sup>−11</sup>) </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> Two ratio model (ERN1+ERN2) </td><td>−24.493 (<em>P</em> &lt; 10<sup>−6</sup>) </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> Three ratio model (ERN1, ERN2) </td><td>−50.474 (<em>P</em> &lt; 10<sup>−10</sup>) </td><td>−49.027 (<em>P</em> &lt; 10<sup>−10</sup>) </td><td>−1.314 (<em>P</em> = 0.518) </td><td>−25.981 (<em>P</em> &lt; 10<sup>−5</sup>) </td><td> </td></tr><tr><td>AIC </td><td>30320.641 </td><td>30321.194 </td><td><strong><span class="underline">30273.481</span></strong> </td><td>30298.148 </td><td>30274.167 </td></tr><tr><td>BIC </td><td>30618.089 </td><td>30622.773 </td><td><strong><span class="underline">30575.060</span></strong> </td><td>30599.728 </td><td>30579.878 </td></tr></tbody></table></div><div content-id="dsw052-T2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Model </th><th rowspan="2">(i) One-ratio model (M0 of Yang <em>et al.</em><a reveal-id="dsw052-B47" data-open="dsw052-B47" class="link link-ref link-reveal xref-bibr"><sup>47</sup></a>) </th><th colspan="3">Two ratio model<hr /> </th><th>(v) Three ratio model </th></tr><tr><th>(ii) ERN1 </th><th>(iii) ERN2 </th><th>(iv) ERN1 + ERN2 </th><th>ERN1, ERN2 </th></tr></thead><tbody><tr><td>np </td><td>72 </td><td>73 </td><td>73 </td><td>73 </td><td>74 </td></tr><tr><td>ln L </td><td>−15088.32041 </td><td>−15087.59693 </td><td>−15063.74053 </td><td>−15076.07402 </td><td>−15063.08349 </td></tr><tr><td>kappa (ts/tv) </td><td>1.58345 </td><td>1.5825 </td><td>1.58201 </td><td>1.58408 </td><td>1.58288 </td></tr><tr><td>omega (dN/dS) </td><td>0.28104 </td><td>ω0 = 0.28866, ω1 = 0.25656 </td><td>ω0 = 0.23609, ω1 = 0.45111 </td><td>ω0 = 0.22744, ω1 = 0.34265 </td><td>ω0 = 0.22749, ω1 = 0.25621, ω2 = 0.45081 </td></tr><tr><td>LRT with 2ΔL and P value </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> Two-ratio model (ERN1) </td><td>−1.447 (P = 0.229) </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> Two-ratio model (ERN2) </td><td>−49.16 (<em>P</em> &lt; 10<sup>−11</sup>) </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> Two ratio model (ERN1+ERN2) </td><td>−24.493 (<em>P</em> &lt; 10<sup>−6</sup>) </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> Three ratio model (ERN1, ERN2) </td><td>−50.474 (<em>P</em> &lt; 10<sup>−10</sup>) </td><td>−49.027 (<em>P</em> &lt; 10<sup>−10</sup>) </td><td>−1.314 (<em>P</em> = 0.518) </td><td>−25.981 (<em>P</em> &lt; 10<sup>−5</sup>) </td><td> </td></tr><tr><td>AIC </td><td>30320.641 </td><td>30321.194 </td><td><strong><span class="underline">30273.481</span></strong> </td><td>30298.148 </td><td>30274.167 </td></tr><tr><td>BIC </td><td>30618.089 </td><td>30622.773 </td><td><strong><span class="underline">30575.060</span></strong> </td><td>30599.728 </td><td>30579.878 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>NOTE.—np is the number of parameters. ln L is the log-likelihood. 2ΔL is twice the log-likelihood difference between the two models. Kappa indicates the transitions/transversions ratio. Omega shows the nonsynonymous/synonymous rate ratio. The Akaike’s information criterion (AIC) and Bayesian information criterion (BIC) were estimated according to Aoki <em>et al.</em><a reveal-id="dsw052-B27" data-open="dsw052-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a>. The best model is underlined. LRT allows one to compare nested models only and is defined as minus two times the logarithm of the ratio of the likelihoods.</p></div></div></div></div></div><div class="table-wrap"><div id="dsw052-T2" data-id="dsw052-T2" class="table-wrap-title"><div class="label -label">Table 2.</div><div class="caption"><p>Model comparison of <em>ERN</em> genes estimate by PAML using branch model</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Model </th><th rowspan="2">(i) One-ratio model (M0 of Yang <em>et al.</em><a reveal-id="dsw052-B47" data-open="dsw052-B47" class="link link-ref link-reveal xref-bibr"><sup>47</sup></a>) </th><th colspan="3">Two ratio model<hr /> </th><th>(v) Three ratio model </th></tr><tr><th>(ii) ERN1 </th><th>(iii) ERN2 </th><th>(iv) ERN1 + ERN2 </th><th>ERN1, ERN2 </th></tr></thead><tbody><tr><td>np </td><td>72 </td><td>73 </td><td>73 </td><td>73 </td><td>74 </td></tr><tr><td>ln L </td><td>−15088.32041 </td><td>−15087.59693 </td><td>−15063.74053 </td><td>−15076.07402 </td><td>−15063.08349 </td></tr><tr><td>kappa (ts/tv) </td><td>1.58345 </td><td>1.5825 </td><td>1.58201 </td><td>1.58408 </td><td>1.58288 </td></tr><tr><td>omega (dN/dS) </td><td>0.28104 </td><td>ω0 = 0.28866, ω1 = 0.25656 </td><td>ω0 = 0.23609, ω1 = 0.45111 </td><td>ω0 = 0.22744, ω1 = 0.34265 </td><td>ω0 = 0.22749, ω1 = 0.25621, ω2 = 0.45081 </td></tr><tr><td>LRT with 2ΔL and P value </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> Two-ratio model (ERN1) </td><td>−1.447 (P = 0.229) </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> Two-ratio model (ERN2) </td><td>−49.16 (<em>P</em> &lt; 10<sup>−11</sup>) </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> Two ratio model (ERN1+ERN2) </td><td>−24.493 (<em>P</em> &lt; 10<sup>−6</sup>) </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> Three ratio model (ERN1, ERN2) </td><td>−50.474 (<em>P</em> &lt; 10<sup>−10</sup>) </td><td>−49.027 (<em>P</em> &lt; 10<sup>−10</sup>) </td><td>−1.314 (<em>P</em> = 0.518) </td><td>−25.981 (<em>P</em> &lt; 10<sup>−5</sup>) </td><td> </td></tr><tr><td>AIC </td><td>30320.641 </td><td>30321.194 </td><td><strong><span class="underline">30273.481</span></strong> </td><td>30298.148 </td><td>30274.167 </td></tr><tr><td>BIC </td><td>30618.089 </td><td>30622.773 </td><td><strong><span class="underline">30575.060</span></strong> </td><td>30599.728 </td><td>30579.878 </td></tr></tbody></table></div><div content-id="dsw052-T2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Model </th><th rowspan="2">(i) One-ratio model (M0 of Yang <em>et al.</em><a reveal-id="dsw052-B47" data-open="dsw052-B47" class="link link-ref link-reveal xref-bibr"><sup>47</sup></a>) </th><th colspan="3">Two ratio model<hr /> </th><th>(v) Three ratio model </th></tr><tr><th>(ii) ERN1 </th><th>(iii) ERN2 </th><th>(iv) ERN1 + ERN2 </th><th>ERN1, ERN2 </th></tr></thead><tbody><tr><td>np </td><td>72 </td><td>73 </td><td>73 </td><td>73 </td><td>74 </td></tr><tr><td>ln L </td><td>−15088.32041 </td><td>−15087.59693 </td><td>−15063.74053 </td><td>−15076.07402 </td><td>−15063.08349 </td></tr><tr><td>kappa (ts/tv) </td><td>1.58345 </td><td>1.5825 </td><td>1.58201 </td><td>1.58408 </td><td>1.58288 </td></tr><tr><td>omega (dN/dS) </td><td>0.28104 </td><td>ω0 = 0.28866, ω1 = 0.25656 </td><td>ω0 = 0.23609, ω1 = 0.45111 </td><td>ω0 = 0.22744, ω1 = 0.34265 </td><td>ω0 = 0.22749, ω1 = 0.25621, ω2 = 0.45081 </td></tr><tr><td>LRT with 2ΔL and P value </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> Two-ratio model (ERN1) </td><td>−1.447 (P = 0.229) </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> Two-ratio model (ERN2) </td><td>−49.16 (<em>P</em> &lt; 10<sup>−11</sup>) </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> Two ratio model (ERN1+ERN2) </td><td>−24.493 (<em>P</em> &lt; 10<sup>−6</sup>) </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> Three ratio model (ERN1, ERN2) </td><td>−50.474 (<em>P</em> &lt; 10<sup>−10</sup>) </td><td>−49.027 (<em>P</em> &lt; 10<sup>−10</sup>) </td><td>−1.314 (<em>P</em> = 0.518) </td><td>−25.981 (<em>P</em> &lt; 10<sup>−5</sup>) </td><td> </td></tr><tr><td>AIC </td><td>30320.641 </td><td>30321.194 </td><td><strong><span class="underline">30273.481</span></strong> </td><td>30298.148 </td><td>30274.167 </td></tr><tr><td>BIC </td><td>30618.089 </td><td>30622.773 </td><td><strong><span class="underline">30575.060</span></strong> </td><td>30599.728 </td><td>30579.878 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>NOTE.—np is the number of parameters. ln L is the log-likelihood. 2ΔL is twice the log-likelihood difference between the two models. Kappa indicates the transitions/transversions ratio. Omega shows the nonsynonymous/synonymous rate ratio. The Akaike’s information criterion (AIC) and Bayesian information criterion (BIC) were estimated according to Aoki <em>et al.</em><a reveal-id="dsw052-B27" data-open="dsw052-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a>. The best model is underlined. LRT allows one to compare nested models only and is defined as minus two times the logarithm of the ratio of the likelihoods.</p></div></div></div></div></p>                            <h2 scrollto-destination=78055020 id="78055020" class="section-title" >4. Discussion</h2>
<p>Phenotypic analyses of <em>L. japonicus ern1</em> mutants showed that this gene is essential for symbiotic root hair responses and IT development. Although <em>Ljern1</em> mutants produced nodule primordia equivalently to the WT, its development was restricted (<a reveal-id="dsw052-F1" data-open="dsw052-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>). Nodule primordia formed in <em>Ljern1</em> mutants did not develop ITs into cortical cells (<a reveal-id="dsw052-F1" data-open="dsw052-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>). Thus, the defect in IT development is a major <em>Ljern1</em> phenotype. This symbiotic phenotype of <em>Ljern1</em> mutants appeared to be intermediate between the <em>M. truncatula ern1</em> single and <em>ern1/erm2</em> double mutants. <em>Mtern1</em> produces a limited number of ITs from infection foci on root hairs. These ITs are arrested within root hairs and do not progress into nodule primordia.<a reveal-id="dsw052-B35" data-open="dsw052-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> Similar to the <em>M. truncatula</em> counterpart, IT formation was remarkably downregulated in <em>Ljern1</em> mutants (<a reveal-id="dsw052-F2" data-open="dsw052-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>). Regarding the defect in IT development and production of nodule primordia, the <em>Ljern1</em> phenotype resembles that of the <em>Mtern1</em> single mutant rather than the <em>ern1/ern2</em> double mutant, which exhibits a more severe phenotype than respective single mutants. The double mutant does not produce ITs and nodule primordia,<a reveal-id="dsw052-B55" data-open="dsw052-B55" class="link link-ref link-reveal xref-bibr"><sup>55</sup></a> suggesting the implication of ERN transcription factors in nodule organogenesis. Alternatively, cortical cells may not be stimulated due to loss of infection events (infection pockets and ITs) in the root epidermis of the double mutant. Host loss-of-function symbiotic mutants able to produce nodule primordia without any infection events in the epidermis have not been identified so far.</p><p>The loss-of-infection events in the double mutant are partially explained by aberrant symbiotic root hair responses. Unlike <em>Mtern1</em>, root hairs of the double mutants are unable to entrap rhizobia because of a defect in tip curling.<a reveal-id="dsw052-B55" data-open="dsw052-B55" class="link link-ref link-reveal xref-bibr"><sup>55</sup></a> Root hair tips of the double mutants are abnormally expanded, suggesting that its polarity is deregulated. Similar abnormality was observed in <em>Ljern1</em> single mutants after inoculation with rhizobia (<a reveal-id="dsw052-F2" data-open="dsw052-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>). The number of curled root hairs was decreased, and balloon-shaped root hairs were increased in the mutant, indicating that <em>LjERN1</em> is involved in symbiotic root hair responses besides IT development. Thus, the root hair phenotype of <em>Ljern1</em> includes the feature characteristic to <em>ern1/ern2</em> double mutants in <em>M. truncatula</em>, which is compatible with the absence of <em>ERN2</em> in <em>L. japonicus.</em> However, the <em>Ljern1</em> phenotype is weaker than that of the <em>M. truncatula</em> double mutants. Other factor may partially compensate the loss of ERN function during early symbiotic root hair responses in <em>L. japonicus</em>.</p><p>GUS reporter analysis reveals gradients of <em>LjERN1</em> expression across uninfected primary roots, namely high expression levels within root tip regions of the roots, whereas levels are lower towards the base. As the root tip region is susceptible to rhizobial infection,<a reveal-id="dsw052-B52" data-open="dsw052-B52" class="link link-ref link-reveal xref-bibr"><sup>52</sup></a> this expression gradient is believed to represent an infection potential of rhizobia; in other words, the distribution of infectable root cells. When <em>M. loti</em> was inoculated, the expression of <em>ProERN1::GUS::TerERN1</em> was observed only in root hair cells undergoing rhizobial infection (via IT development) and in cortical cells initiating cell division (<a reveal-id="dsw052-F5" data-open="dsw052-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>). On the other hand, <em>ERN1</em> expression was suppressed in the peripheral epidermal regions where few infection events occurred. These findings suggest that <em>ERN1</em> expression patterns are in good agreement with rhizobial infection potential and infection events. Strong expression of <em>ERN1</em> is also distinct in dividing cortical cells. Within this interpretation, two possibilities are conceivable: (i) ERN1 is directly involved in cortical cell division; and (ii) <em>ERN1</em> expression confers the ability to accept ITs to dividing cortical cells. In the former case, ERN1 is unlikely to be involved in universal cell division because no expression was found in root meristem where cell division is always active (<a reveal-id="dsw052-F5" data-open="dsw052-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>). In the latter case, <em>ERN1</em> expression may be involved in the navigation of developing ITs to cortical cells. Further analysis will be required to clarify what kinds of functions ERN1 has in dividing cortical cells.</p><p>Quantitative expression analysis using symbiotic mutants shows that the early induction of <em>ERN1</em> by <em>M. loti</em> inoculation is required for CYCLOPS and partially for NSP2. Recently, Cerri <em>et al</em><em>.</em><a reveal-id="dsw052-B55" data-open="dsw052-B55" class="link link-ref link-reveal xref-bibr"><sup>55</sup></a> clearly showed that NSP2 in the presence of NSP1 can act as a strong activator of <em>ERN1</em> transcription in transient expression assays in <em>Nicotiana benthamiana</em>, and our data are consistent with their results regarding NSP2. On the other hand, <em>ERN1</em> induction was clearly canceled in the <em>cyclops</em> mutant background. Although CYCLOPS was originally identified as a component of the common symbiosis pathway that directly interacts with CCaMK, it has been demonstrated that it acts as a transcription activator of the <em>NIN</em> gene via direct binding of CYCLOPS-box in its promoter region. Since NIN is required for early infection events via IT development and its target <em>NF-YA1</em> directly binds with <em>MtERN1</em> promoter,<a reveal-id="dsw052-B62" data-open="dsw052-B62" class="link link-ref link-reveal xref-bibr"><sup>62</sup></a><sup>,</sup><sup><strong>,</strong></sup><a reveal-id="dsw052-B63" data-open="dsw052-B63" class="link link-ref link-reveal xref-bibr"><sup>63</sup></a> it is possible that ERN1 acts downstream of NIN to regulate IT progression. However, our results show that ERN1 is normally induced in response to the inoculation of <em>M. loti</em> in the <em>nin</em> mutant background similar to the WT, indicating that <em>ERN1</em> activation is not controlled by NIN. NIN is likely to act parallel with ERN1 under the control of CYLOPS or may act downstream of ERN1. The balloon-shaped root hair phenotype characteristic to <em>Ljern1</em> mutants has not been reported in <em>nin</em> mutants, which display excess curled and branched root hairs in response to rhizobial infection. The frequency of branched root hairs in <em>Ljern1</em> mutants was equivalent to that of the WT. The different root hair morphology may suggest that ERN1 and NIN play distinct roles to positively regulate root hair responses and IT progression. In any case, the elucidation of transcription networks involving CYCLOPS, ERN1, and NIN is a future challenge.</p><p>The phylogenetic analyses showed that <em>ERN1</em> and <em>ERN2</em> genes were originated from the gene duplication after the divergence of Fabales. The duplication event and subsequent molecular evolution of the genes might associate with the evolution of nodulation symbiosis of legume plants although we do not know the full details of the evolutionary history of <em>ERN</em> genes because of the lack of information for plants other than symbiotic legumes of Faboideae (Fabaceae) in Fabales (e.g. Caesalpinioideae, Mimosoideae, Quillajaceae, Polygalaceae, and Surianaceae). The comparative genomic analysis of the synteny of <em>ERN</em> genes showed that <em>ERN1</em> of legumes had a similar gene order to the <em>ERN</em> genes of nonlegume plants, rather than that of <em>ERN2</em> did. In contrast to the previous suggestion that <em>ERN1</em> was derived from <em>ERN2</em> via duplication,<a reveal-id="dsw052-B54" data-open="dsw052-B54" class="link link-ref link-reveal xref-bibr"><sup>54</sup></a><sup>,</sup><a reveal-id="dsw052-B55" data-open="dsw052-B55" class="link link-ref link-reveal xref-bibr"><sup>55</sup></a><sup>,</sup><a reveal-id="dsw052-B64" data-open="dsw052-B64" class="link link-ref link-reveal xref-bibr"><sup>64</sup></a> the synteny analysis proposed that <em>ERN1</em> might have an orthologous role to <em>ERN</em> genes of plants of the outgroup. Molecular evolutionary analysis of ω ratio supported this result. The LRT failed to reject the alternative hypothesis of different ω ratio between <em>ERN1</em> and all other genes in the phylogeny of <a reveal-id="dsw052-F6" data-open="dsw052-F6" class="link link-reveal link-table xref-fig">Figure 6</a> against the null hypothesis of the same ω ratios of all genes (<a reveal-id="dsw052-T2" data-open="dsw052-T2" class="link link-reveal link-table xref-fig">Table 2</a>). The test also showed that the ω ratio of <em>ERN2</em> was significantly greater than the background ratio of <em>ERN1</em> and other <em>ERN</em> genes. It has been believed that variable nonsynonymous/synonymous rate ratios among lineages may associate with the difference of functional constraint along certain lineages.<a reveal-id="dsw052-B65" data-open="dsw052-B65" class="link link-ref link-reveal xref-bibr"><sup>65</sup></a><sup>,</sup><a reveal-id="dsw052-B66" data-open="dsw052-B66" class="link link-ref link-reveal xref-bibr"><sup>66</sup></a> According to this proposition, the functional constraint appears to be conserved between legume <em>ERN1</em> and nonlegume <em>ERN</em> genes, whereas the genes in the <em>ERN2</em> lineage may have suffered relaxed selective constraints. It is possible that <em>ERN1</em> took over the main molecular function of original <em>ERN</em> genes and that <em>ERN2</em> has subfunctionalized and/or neofunctionalized the ERN function, and is a helpful adjunct to the <em>ERN1</em> gene during nodulation. A recent article appears to support this proposition.<a reveal-id="dsw052-B55" data-open="dsw052-B55" class="link link-ref link-reveal xref-bibr"><sup>55</sup></a> Although the <em>ern1</em> mutants of <em>M. truncatula</em> were unable to nodulate, <em>ern2</em> mutants can nodulate but with signs of premature senescence. Although the <em>ern1</em> mutant still exhibited early symbiotic responses including rhizobial infection, <em>ern1/ern2</em> double mutants were completely unable to initiate infection and nodule development. <em>ERN2</em> showed partially overlapping expression patterns with <em>ERN1</em>.<a reveal-id="dsw052-B55" data-open="dsw052-B55" class="link link-ref link-reveal xref-bibr"><sup>55</sup></a></p><p>Our genomic analysis suggested the possibility of a lack of the <em>ERN2</em> gene in the genome of <em>L.japonicus</em>. LjERN1 might have a sufficient role in IT development of <em>L. japonicus</em>. Alternatively, the function and expression of the <em>LjERN1</em> may be complement in those of <em>ERN2</em> genes of other legumes. This notion is consistent with the fact that <em>Ljern1</em> single mutants display balloon-shaped root hairs that are observed in only <em>ern1/ern2</em> double mutants in <em>M. truncatula.</em></p>                            <h2 scrollto-destination=78055027 id="78055027" class="backacknowledgements-title" >Acknowledgements</h2>
<p>This work was supported by a Research Fellow of the Japan Society for the Promotion of Science (JSPS) (to K.Y.; 22·3459), Grant-in-Aid for Challenging Exploratory Research from JSPS (Grant Number 26660057), the Canon foundation (to W.I.), The Ministry of Education, Culture, Sports, Science and Technology KAKENHI Grant Number 16H06279 (to W.I.), and Grant for Basic Science Research Projects 150574 from the Sumitomo Foundation (to T.S.). </p>                            <h2 scrollto-destination=78055029 id="78055029" class="backsection-title" >Conflict of interest</h2>
<p> None declared.</p>                            <h2 scrollto-destination=78055031 id="78055031" class="backsection-title" >Supplementary data</h2>
<p><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw052/2/dsw052_Supp.zip?Expires=2147483647&Signature=SUfZUEyOx0cNgvbTgqmG5Jlc7D-wCgPBRu~nkZYtDlnDJ3y08Dk1egjJie4Bi9r0o~3T21J6w8mfvpFrbyNAq2drb5sFojcQL96jELnU9bogfK5Fi6LNtPAWKyHiFJfKZuy9UtsMZnYsh1Y3IfDP7q7xrQpM6Tdb~N-zpw88J5aGu5UvF0EP7Aurb-xgRQ4kG3PjHn5IFQBvYeAj7yMCbHPe5dB-ORrJjjGmlosU5TPSzRe3LJ9SSIUQBp5XOPrQ-r~s3VpZ6k98RKm2Ew2J5eCHjxuifUtwtPKDFIBqI6CYZsVtizidvxoFwOjgWAfKnY5U7a7aAxOp4ZGx9gcKTg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary data</a></span> are available at <a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org" target="_blank">www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=78055033 id="78055033" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsw052-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw052-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Fournier</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Timmers</div> <div class="given-names">A. C.</div></div>, <div class="name string-name"><div class="surname">Sieberer</div> <div class="given-names">B. J.</div></div>, <div class="name string-name"><div class="surname">Jauneau</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Chabaud</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Barker</div> <div class="given-names">D. G.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">Mechanism of infection thread elongation in root hairs of <em>Medicago truncatula</em> and dynamic interplay with associated rhizobial colonization,</div> <div class="source">Plant Physiol</div> ., <div class="volume">148</div>, <div class="fpage">1985</div>–<div class="lpage">95</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mechanism%20of%20infection%20thread%20elongation%20in%20root%20hairs%20of%20Medicago%20truncatula%20and%20dynamic%20interplay%20with%20associated%20rhizobial%20colonization%2C&amp;author=J.%20Fournier&amp;author=A.%20C.%20Timmers&amp;author=B.%20J.%20Sieberer&amp;author=A.%20Jauneau&amp;author=M.%20Chabaud&amp;author=D.%20G.%20Barker&amp;publication_year=2008&amp;journal=Plant%20Physiol&amp;volume=148&amp;pages=1985-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.108.125674" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.108.125674" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.108.125674"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18931145" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw052-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw052-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Svistoonoff</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Hocher</div> <div class="given-names">V.</div></div>, <div class="name string-name"><div class="surname">Gherbi</div> <div class="given-names">H.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Actinorhizal root nodule symbioses: what is signalling telling on the origins of nodulation?,</div> <div class="source">Curr. Opin. Plant Biol</div> ., <div class="volume">20</div>, <div class="fpage">11</div>–<div class="lpage">18</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Actinorhizal%20root%20nodule%20symbioses%3A%20what%20is%20signalling%20telling%20on%20the%20origins%20of%20nodulation%3F%2C&amp;author=S.%20Svistoonoff&amp;author=V.%20Hocher&amp;author=H.%20Gherbi&amp;publication_year=2014&amp;journal=Curr.%20Opin.%20Plant%20Biol&amp;volume=20&amp;pages=11-18" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.pbi.2014.03.001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.pbi.2014.03.001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.pbi.2014.03.001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24691197" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw052-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw052-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Downie</div> <div class="given-names">J. A.</div></div>, <div class="name string-name"><div class="surname">Walker</div> <div class="given-names">S. A.</div></div></div> <div class="year">1999</div>, 
<div class="article-title">Plant responses to nodulation factors,</div> <div class="source">Curr. Opin. Plant Biol</div> ., <div class="volume">2</div>, <div class="fpage">483</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Plant%20responses%20to%20nodulation%20factors%2C&amp;author=J.%20A.%20Downie&amp;author=S.%20A.%20Walker&amp;publication_year=1999&amp;journal=Curr.%20Opin.%20Plant%20Biol&amp;volume=2&amp;pages=483-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S1369-5266(99)00018-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS1369-5266(99)00018-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS1369-5266(99)00018-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10607652" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw052-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw052-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ehrhardt</div> <div class="given-names">D. W.</div></div>, <div class="name string-name"><div class="surname">Wais</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Long</div> <div class="given-names">S. R.</div></div></div> <div class="year">1996</div>, 
<div class="article-title">Calcium spiking in plant root hairs responding to Rhizobium nodulation signals,</div> <div class="source">Cell</div> , <div class="volume">85</div>, <div class="fpage">673</div>–<div class="lpage">81</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Calcium%20spiking%20in%20plant%20root%20hairs%20responding%20to%20Rhizobium%20nodulation%20signals%2C&amp;author=D.%20W.%20Ehrhardt&amp;author=R.%20Wais&amp;author=S.%20R.%20Long&amp;publication_year=1996&amp;journal=Cell&amp;volume=85&amp;pages=673-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0092-8674(00)81234-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0092-8674(00)81234-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0092-8674(00)81234-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8646776" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw052-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw052-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Amor</div> <div class="given-names">B. B.</div></div>, <div class="name string-name"><div class="surname">Shaw</div> <div class="given-names">S. L.</div></div>, <div class="name string-name"><div class="surname">Oldroyd</div> <div class="given-names">G. E.</div></div></div>, et al.  . <div class="year">2003</div>, 
<div class="article-title">The <em>NFP</em> locus of <em>Medicago truncatula</em> controls an early step of Nod factor signal transduction upstream of a rapid calcium flux and root hair deformation,</div> <div class="source">Plant J</div> ., <div class="volume">34</div>, <div class="fpage">495</div>–<div class="lpage">506</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20NFP%20locus%20of%20Medicago%20truncatula%20controls%20an%20early%20step%20of%20Nod%20factor%20signal%20transduction%20upstream%20of%20a%20rapid%20calcium%20flux%20and%20root%20hair%20deformation%2C&amp;author=B.%20B.%20Amor&amp;author=S.%20L.%20Shaw&amp;author=G.%20E.%20Oldroyd&amp;publication_year=2003&amp;journal=Plant%20J&amp;volume=34&amp;pages=495-506" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-313X.2003.01743.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-313X.2003.01743.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-313X.2003.01743.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12753588" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw052-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw052-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Esseling</div> <div class="given-names">J. J.</div></div>, <div class="name string-name"><div class="surname">Lhuissier</div> <div class="given-names">F. G.</div></div>, <div class="name string-name"><div class="surname">Emons</div> <div class="given-names">A. M.</div></div></div> <div class="year">2003</div>, 
<div class="article-title">Nod factor-induced root hair curling: Continuous polar growth towards the point of nod factor application,</div> <div class="source">Plant Physiol</div> ., <div class="volume">132</div>, <div class="fpage">1982</div>–<div class="lpage">1988</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nod%20factor-induced%20root%20hair%20curling%3A%20Continuous%20polar%20growth%20towards%20the%20point%20of%20nod%20factor%20application%2C&amp;author=J.%20J.%20Esseling&amp;author=F.%20G.%20Lhuissier&amp;author=A.%20M.%20Emons&amp;publication_year=2003&amp;journal=Plant%20Physiol&amp;volume=132&amp;pages=1982-1988" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.103.021634" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.103.021634" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.103.021634"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12913154" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw052-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw052-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Madsen</div> <div class="given-names">E. B.</div></div>, <div class="name string-name"><div class="surname">Madsen</div> <div class="given-names">L. H.</div></div>, <div class="name string-name"><div class="surname">Radutoiu</div> <div class="given-names">S.</div></div></div>, et al.  . <div class="year">2003</div>, 
<div class="article-title">A receptor kinase gene of the LysM type is involved in legume perception of rhizobial signals,</div> <div class="source">Nature</div> , <div class="volume">425</div>, <div class="fpage">637</div>–<div class="lpage">40</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20receptor%20kinase%20gene%20of%20the%20LysM%20type%20is%20involved%20in%20legume%20perception%20of%20rhizobial%20signals%2C&amp;author=E.%20B.%20Madsen&amp;author=L.%20H.%20Madsen&amp;author=S.%20Radutoiu&amp;publication_year=2003&amp;journal=Nature&amp;volume=425&amp;pages=637-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature02045" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature02045" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature02045"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14534591" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw052-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw052-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Radutoiu</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Madsen</div> <div class="given-names">L. H.</div></div>, <div class="name string-name"><div class="surname">Madsen</div> <div class="given-names">E. B.</div></div></div>, et al.  . (<div class="year">2003</div>). 
<div class="article-title">Plant recognition of symbiotic bacteria requires two LysM receptor-like kinases,</div> <div class="source">Nature</div> , <div class="volume">425</div>, <div class="fpage">585</div>–<div class="lpage">92</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Plant%20recognition%20of%20symbiotic%20bacteria%20requires%20two%20LysM%20receptor-like%20kinases%2C&amp;author=S.%20Radutoiu&amp;author=L.%20H.%20Madsen&amp;author=E.%20B.%20Madsen&amp;publication_year=2003&amp;journal=Nature&amp;volume=425&amp;pages=585-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature02039" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature02039" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature02039"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14534578" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw052-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw052-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Smit</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Limpens</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Geurts</div> <div class="given-names">R.</div></div></div>, et al.  . <div class="year">2007</div>, 
<div class="article-title"><em>Medicago</em> LYK3, an entry receptor in rhizobial nodulation factor signaling,</div> <div class="source">Plant Physiol</div> ., <div class="volume">145</div>, <div class="fpage">183</div>–<div class="lpage">91</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Medicago%20LYK3%2C%20an%20entry%20receptor%20in%20rhizobial%20nodulation%20factor%20signaling%2C&amp;author=P.%20Smit&amp;author=E.%20Limpens&amp;author=R.%20Geurts&amp;publication_year=2007&amp;journal=Plant%20Physiol&amp;volume=145&amp;pages=183-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.107.100495" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.107.100495" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.107.100495"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17586690" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw052-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw052-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Broghammer</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Krusell</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Blaise</div> <div class="given-names">M.</div></div></div>, et al.  . <div class="year">2012</div>, 
<div class="article-title">Legume receptors perceive the rhizobial lipochitin oligosaccharide signal molecules by direct binding,</div> <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="volume">109</div>, <div class="fpage">13859</div>–<div class="lpage">64</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Legume%20receptors%20perceive%20the%20rhizobial%20lipochitin%20oligosaccharide%20signal%20molecules%20by%20direct%20binding%2C&amp;author=A.%20Broghammer&amp;author=L.%20Krusell&amp;author=M.%20Blaise&amp;publication_year=2012&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=109&amp;pages=13859-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1205171109" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1205171109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1205171109"> </span></div> </div></div></div></div></div><div content-id="dsw052-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw052-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kistner</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Parniske</div> <div class="given-names">M.</div></div></div> <div class="year">2002</div>, 
<div class="article-title">Evolution of signal transduction in intracellular symbiosis,</div> <div class="source">Trends Plant Sci</div> ., <div class="volume">7</div>, <div class="fpage">511</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20of%20signal%20transduction%20in%20intracellular%20symbiosis%2C&amp;author=C.%20Kistner&amp;author=M.%20Parniske&amp;publication_year=2002&amp;journal=Trends%20Plant%20Sci&amp;volume=7&amp;pages=511-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S1360-1385(02)02356-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS1360-1385(02)02356-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS1360-1385(02)02356-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12417152" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw052-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw052-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gleason</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Chaudhuri</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Yang</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Muñoz</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Poovaiah</div> <div class="given-names">B. W.</div></div>, <div class="name string-name"><div class="surname">Oldroyd</div> <div class="given-names">G. E.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">Nodulation independent of rhizobia induced by a calcium-activated kinase lacking autoinhibition,</div> <div class="source">Nature</div> , <div class="volume">441</div>, <div class="fpage">1149</div>–<div class="lpage">52</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nodulation%20independent%20of%20rhizobia%20induced%20by%20a%20calcium-activated%20kinase%20lacking%20autoinhibition%2C&amp;author=C.%20Gleason&amp;author=S.%20Chaudhuri&amp;author=T.%20Yang&amp;author=A.%20Mu%C3%B1oz&amp;author=B.%20W.%20Poovaiah&amp;author=G.%20E.%20Oldroyd&amp;publication_year=2006&amp;journal=Nature&amp;volume=441&amp;pages=1149-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature04812" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature04812" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature04812"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16810256" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw052-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw052-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tirichine</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Imaizumi-Anraku</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Yoshida</div> <div class="given-names">S.</div></div></div>, et al.  . <div class="year">2006</div>, 
<div class="article-title">Deregulation of a Ca<sup>2+</sup>/calmodulin-dependent kinase leads to spontaneous nodule development,</div> <div class="source">Nature</div> , <div class="volume">441</div>, <div class="fpage">1153</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Deregulation%20of%20a%20Ca2%2B%2Fcalmodulin-dependent%20kinase%20leads%20to%20spontaneous%20nodule%20development%2C&amp;author=L.%20Tirichine&amp;author=H.%20Imaizumi-Anraku&amp;author=S.%20Yoshida&amp;publication_year=2006&amp;journal=Nature&amp;volume=441&amp;pages=1153-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature04862" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature04862" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature04862"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16810257" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw052-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw052-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hayashi</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Banba</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Shimoda</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Kouchi</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Hayashi</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Imaizumi-Anraku</div> <div class="given-names">H.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">A dominant function of CCaMK in intracellular accommodation of bacterial and fungal endosymbionts,</div> <div class="source">Plant J</div> ., <div class="volume">63</div>, <div class="fpage">141</div>–<div class="lpage">54</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20dominant%20function%20of%20CCaMK%20in%20intracellular%20accommodation%20of%20bacterial%20and%20fungal%20endosymbionts%2C&amp;author=T.%20Hayashi&amp;author=M.%20Banba&amp;author=Y.%20Shimoda&amp;author=H.%20Kouchi&amp;author=M.%20Hayashi&amp;author=H.%20Imaizumi-Anraku&amp;publication_year=2010&amp;journal=Plant%20J&amp;volume=63&amp;pages=141-54" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20409002" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw052-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw052-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Madsen</div> <div class="given-names">L.H.</div></div>, <div class="name string-name"><div class="surname">Tirichine</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Jurkiewicz</div> <div class="given-names">A.</div></div></div>, et al.  . <div class="year">2010</div>, 
<div class="article-title">The molecular network governing nodule organogenesis and infection in the model legume <em>Lotus japonicus</em>,</div> <div class="source">Nat. Commun</div> ., <div class="volume">1</div>, <div class="fpage">10</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20molecular%20network%20governing%20nodule%20organogenesis%20and%20infection%20in%20the%20model%20legume%20Lotus%20japonicus%2C&amp;author=L.H.%20Madsen&amp;author=L.%20Tirichine&amp;author=A.%20Jurkiewicz&amp;publication_year=2010&amp;journal=Nat.%20Commun&amp;volume=1&amp;pages=10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ncomms1009" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fncomms1009" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fncomms1009"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20975672" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw052-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw052-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hocher</div> <div class="given-names">V.</div></div>, <div class="name string-name"><div class="surname">Alloisio</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Auguy</div> <div class="given-names">F.</div></div></div>, et al.  . <div class="year">2011</div>, 
<div class="article-title">Transcriptomics of actinorhizal symbioses reveals homologs of the whole common symbiotic signaling cascade,</div> <div class="source">Plant Physiol</div> ., <div class="volume">156</div>, <div class="fpage">700</div>–<div class="lpage">11</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptomics%20of%20actinorhizal%20symbioses%20reveals%20homologs%20of%20the%20whole%20common%20symbiotic%20signaling%20cascade%2C&amp;author=V.%20Hocher&amp;author=N.%20Alloisio&amp;author=F.%20Auguy&amp;publication_year=2011&amp;journal=Plant%20Physiol&amp;volume=156&amp;pages=700-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.111.174151" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.111.174151" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.111.174151"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21464474" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw052-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw052-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Demina</div> <div class="given-names">I. V.</div></div>, <div class="name string-name"><div class="surname">Persson</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Santos</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Plaszczyca</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Pawlowski</div> <div class="given-names">K.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Comparison of the nodule vs. root transcriptome of the actinorhizal plant <em>Datisca glomerata</em>: actinorhizal nodules contain a specific class of defensins,</div> <div class="source">PLoS One</div> , <div class="volume">8</div>, <div class="fpage">e72442</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparison%20of%20the%20nodule%20vs.%20root%20transcriptome%20of%20the%20actinorhizal%20plant%20Datisca%20glomerata%3A%20actinorhizal%20nodules%20contain%20a%20specific%20class%20of%20defensins%2C&amp;author=I.%20V.%20Demina&amp;author=T.%20Persson&amp;author=P.%20Santos&amp;author=M.%20Plaszczyca&amp;author=K.%20Pawlowski&amp;publication_year=2013&amp;journal=PLoS%20One&amp;volume=8&amp;pages=e72442" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0072442" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0072442" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0072442"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24009681" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw052-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw052-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Diédhiou</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Tromas</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Cissoko</div> <div class="given-names">M.</div></div></div> et al.  . <div class="year">2014</div>, 
<div class="article-title">Identification of potential transcriptional regulators of actinorhizal symbioses in <em>Casuarina glauca</em> and <em>Alnus glutinosa</em>,</div> <div class="source">BMC Plant Biol</div> ., <div class="volume">14</div>, <div class="fpage">342</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20potential%20transcriptional%20regulators%20of%20actinorhizal%20symbioses%20in%20Casuarina%20glauca%20and%20Alnus%20glutinosa%2C&amp;author=I.%20Di%C3%A9dhiou&amp;author=A.%20Tromas&amp;author=M.%20Cissoko&amp;publication_year=2014&amp;journal=BMC%20Plant%20Biol&amp;volume=14&amp;pages=342" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12870-014-0342-z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12870-014-0342-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12870-014-0342-z"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25492470" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw052-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw052-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gherbi</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Markmann</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Svistoonoff</div> <div class="given-names">S.</div></div></div>, et al.  . <div class="year">2008</div>, 
<div class="article-title">SymRK defines a common genetic basis for plant root endosymbioses with arbuscular mycorrhiza fungi, rhizobia, and <em>Frankia</em> bacteria,</div> <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="volume">105</div>, <div class="fpage">4928</div>–<div class="lpage">32</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SymRK%20defines%20a%20common%20genetic%20basis%20for%20plant%20root%20endosymbioses%20with%20arbuscular%20mycorrhiza%20fungi%2C%20rhizobia%2C%20and%20Frankia%20bacteria%2C&amp;author=H.%20Gherbi&amp;author=K.%20Markmann&amp;author=S.%20Svistoonoff&amp;publication_year=2008&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=105&amp;pages=4928-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0710618105" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0710618105" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0710618105"> </span></div> </div></div></div></div></div><div content-id="dsw052-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw052-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Markmann</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Giczey</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Parniske</div> <div class="given-names">M.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">Functional adaptation of a plant receptor-kinase paved the way for the evolution of intracellular root symbioses with bacteria,</div> <div class="source">PLoS Biol</div> ., <div class="volume">6</div>, <div class="fpage">e68</div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Functional%20adaptation%20of%20a%20plant%20receptor-kinase%20paved%20the%20way%20for%20the%20evolution%20of%20intracellular%20root%20symbioses%20with%20bacteria%2C&amp;author=K.%20Markmann&amp;author=G.%20Giczey&amp;author=M.%20Parniske&amp;publication_year=2008&amp;journal=PLoS%20Biol&amp;volume=6&amp;pages=e68" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pbio.0060068" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pbio.0060068" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pbio.0060068"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18318603" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw052-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw052-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Svistoonoff</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Benabdoun</div> <div class="given-names">F. M.</div></div>, <div class="name string-name"><div class="surname">Nambiar-Veetil</div> <div class="given-names">M.</div></div></div>, et al.  . <div class="year">2013</div>, 
<div class="article-title">The independent acquisition of plant root nitrogen-fixing symbiosis in Fabids recruited the same genetic pathway for nodule organogenesis,</div> <div class="source">PLoS One</div> , <div class="volume">8</div>, <div class="fpage">e64515</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20independent%20acquisition%20of%20plant%20root%20nitrogen-fixing%20symbiosis%20in%20Fabids%20recruited%20the%20same%20genetic%20pathway%20for%20nodule%20organogenesis%2C&amp;author=S.%20Svistoonoff&amp;author=F.%20M.%20Benabdoun&amp;author=M.%20Nambiar-Veetil&amp;publication_year=2013&amp;journal=PLoS%20One&amp;volume=8&amp;pages=e64515" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0064515" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0064515" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0064515"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23741336" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw052-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw052-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Schauser</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Roussis</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Stiller</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Stougaard</div> <div class="given-names">J.</div></div></div> <div class="year">1999</div>, 
<div class="article-title">A plant regulator controlling development of symbiotic root nodules,</div> <div class="source">Nature</div> , <div class="volume">402</div>, <div class="fpage">191</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20plant%20regulator%20controlling%20development%20of%20symbiotic%20root%20nodules%2C&amp;author=L.%20Schauser&amp;author=A.%20Roussis&amp;author=J.%20Stiller&amp;author=J.%20Stougaard&amp;publication_year=1999&amp;journal=Nature&amp;volume=402&amp;pages=191-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/46058" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F46058" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F46058"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10647012" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw052-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw052-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Marsh</div> <div class="given-names">J. F.</div></div>, <div class="name string-name"><div class="surname">Rakocevic</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Mitra</div> <div class="given-names">R. M.</div></div></div>, et al.  . <div class="year">2007</div>, 
<div class="article-title"><em>Medicago truncatula</em> NIN is essential for rhizobial-independent nodule organogenesis induced by autoactive calcium/calmodulin-dependent protein kinase,</div> <div class="source">Plant Physiol</div> ., <div class="volume">144</div>, <div class="fpage">324</div>–<div class="lpage">35</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Medicago%20truncatula%20NIN%20is%20essential%20for%20rhizobial-independent%20nodule%20organogenesis%20induced%20by%20autoactive%20calcium%2Fcalmodulin-dependent%20protein%20kinase%2C&amp;author=J.%20F.%20Marsh&amp;author=A.%20Rakocevic&amp;author=R.%20M.%20Mitra&amp;publication_year=2007&amp;journal=Plant%20Physiol&amp;volume=144&amp;pages=324-35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.106.093021" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.106.093021" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.106.093021"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17369436" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw052-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw052-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Clavijo</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Diedhiou</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Vaissayre</div> <div class="given-names">V.</div></div></div>, et al.  . <div class="year">2015</div>, 
<div class="article-title">The <em>Casuarina NIN</em> gene is transcriptionally activated throughout Frankia root infection as well as in response to bacterial diffusible signals,</div> <div class="source">New Phytol</div> ., <div class="volume">208</div>, <div class="fpage">887</div>–<div class="lpage">903</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Casuarina%20NIN%20gene%20is%20transcriptionally%20activated%20throughout%20Frankia%20root%20infection%20as%20well%20as%20in%20response%20to%20bacterial%20diffusible%20signals%2C&amp;author=F.%20Clavijo&amp;author=I.%20Diedhiou&amp;author=V.%20Vaissayre&amp;publication_year=2015&amp;journal=New%20Phytol&amp;volume=208&amp;pages=887-903" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/nph.13506" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fnph.13506" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fnph.13506"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26096779" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw052-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw052-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Limpens</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">van Zeijl</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Geurts</div> <div class="given-names">R.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Lipochitoologosaccharides modulate plant host immunity to enable endosymbiosis,</div> <div class="source">Annu. Rev. Phytopathol</div> ., <div class="volume">53</div>, <div class="fpage">311</div>–<div class="lpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Lipochitoologosaccharides%20modulate%20plant%20host%20immunity%20to%20enable%20endosymbiosis%2C&amp;author=E.%20Limpens&amp;author=A.%20van%20Zeijl&amp;author=R.%20Geurts&amp;publication_year=2015&amp;journal=Annu.%20Rev.%20Phytopathol&amp;volume=53&amp;pages=311-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev-phyto-080614-120149" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev-phyto-080614-120149" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev-phyto-080614-120149"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26047562" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw052-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw052-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">De Mita</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Streng</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Bisseling</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Geurts</div> <div class="given-names">R.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Evolution of a symbiotic receptor through gene duplications in the legume-rhizobium mutualism,</div> <div class="source">New Phytol</div> ., <div class="volume">201</div>, <div class="fpage">961</div>–<div class="lpage">72</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20of%20a%20symbiotic%20receptor%20through%20gene%20duplications%20in%20the%20legume-rhizobium%20mutualism%2C&amp;author=S.%20De%20Mita&amp;author=A.%20Streng&amp;author=T.%20Bisseling&amp;author=R.%20Geurts&amp;publication_year=2014&amp;journal=New%20Phytol&amp;volume=201&amp;pages=961-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/nph.12549" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fnph.12549" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fnph.12549"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24400903" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw052-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw052-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Aoki</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Ito</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Iwasaki</div> <div class="given-names">W.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">From β- to α-Proteobacteria: The origin and evolution of rhizobial nodulation genes nodIJ,</div> <div class="source">Mol. Biol. Evol</div> ., <div class="volume">30</div>, <div class="fpage">2494</div>–<div class="lpage">508</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=From%20%CE%B2-%20to%20%CE%B1-Proteobacteria%3A%20The%20origin%20and%20evolution%20of%20rhizobial%20nodulation%20genes%20nodIJ%2C&amp;author=S.%20Aoki&amp;author=M.%20Ito&amp;author=W.%20Iwasaki&amp;publication_year=2013&amp;journal=Mol.%20Biol.%20Evol&amp;volume=30&amp;pages=2494-508" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/mst153" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmst153" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmst153"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24030554" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw052-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw052-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Cannon</div> <div class="given-names">S. B.</div></div>, <div class="name string-name"><div class="surname">McKain</div> <div class="given-names">M. R.</div></div></div>, Harkess, et al.  . <div class="year">2015</div>, 
<div class="article-title">Multiple polyploidy events in the early radiation of nodulating and nonnodulating legumes,</div> <div class="source">Mol. Biol. Evol</div> ., <div class="volume">32</div>, <div class="fpage">193</div>–<div class="lpage">210</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Multiple%20polyploidy%20events%20in%20the%20early%20radiation%20of%20nodulating%20and%20nonnodulating%20legumes%2C&amp;author=S.%20B.%20Cannon&amp;author=M.%20R.%20McKain&amp;publication_year=2015&amp;journal=Mol.%20Biol.%20Evol&amp;volume=32&amp;pages=193-210" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msu296" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsu296" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsu296"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25349287" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw052-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw052-B29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">Q.-G.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Dunwell</div> <div class="given-names">J. M.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">Y.-M.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Comparative genomics suggests that an ancestral polyploidy event leads to enhanced root nodule symbiosis in the Papilionoideae,</div> <div class="source">Mol. Biol. Evol</div> ., <div class="volume">30</div>, <div class="fpage">2602</div>–<div class="lpage">11</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20genomics%20suggests%20that%20an%20ancestral%20polyploidy%20event%20leads%20to%20enhanced%20root%20nodule%20symbiosis%20in%20the%20Papilionoideae%2C&amp;author=Q.-G.%20Li&amp;author=L.%20Zhang&amp;author=C.%20Li&amp;author=J.%20M.%20Dunwell&amp;author=Y.-M.%20Zhang&amp;publication_year=2013&amp;journal=Mol.%20Biol.%20Evol&amp;volume=30&amp;pages=2602-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/mst152" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmst152" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmst152"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24008584" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw052-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw052-B30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yano</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Yoshida</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Müeller</div> <div class="given-names">J.</div></div></div>, et al.  . <div class="year">2008</div>, 
<div class="article-title">CYCLOPS, a mediator of symbiotic intracellular accommodation,</div> <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="volume">105</div>, <div class="fpage">20540</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CYCLOPS%2C%20a%20mediator%20of%20symbiotic%20intracellular%20accommodation%2C&amp;author=K.%20Yano&amp;author=S.%20Yoshida&amp;author=J.%20M%C3%BCeller&amp;publication_year=2008&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=105&amp;pages=20540-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0806858105" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0806858105" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0806858105"> </span></div> </div></div></div></div></div><div content-id="dsw052-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw052-B31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yano</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Shibata</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">W. L.</div></div></div>, et al.  . <div class="year">2009</div>, 
<div class="article-title">CERBERUS, a novel U-box protein containing WD-40 repeats, is required for formation of the infection thread and nodule development in the legume-<em>Rhizobium</em> symbiosis,</div> <div class="source">Plant J</div> ., <div class="volume">60</div>, <div class="fpage">168</div>–<div class="lpage">80</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CERBERUS%2C%20a%20novel%20U-box%20protein%20containing%20WD-40%20repeats%2C%20is%20required%20for%20formation%20of%20the%20infection%20thread%20and%20nodule%20development%20in%20the%20legume-Rhizobium%20symbiosis%2C&amp;author=K.%20Yano&amp;author=S.%20Shibata&amp;author=W.%20L.%20Chen&amp;publication_year=2009&amp;journal=Plant%20J&amp;volume=60&amp;pages=168-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.2009.60.issue-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2009.60.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.2009.60.issue-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19508425" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw052-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw052-B32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Singh</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Katzer</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Lambert</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Cerri</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Parniske</div> <div class="given-names">M.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">CYCLOPS, a DNA-binding transcriptional activator, orchestrates symbiotic root nodule development,</div> <div class="source">Cell Host Microbe</div> , <div class="volume">15</div>, <div class="fpage">139</div>–<div class="lpage">52</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CYCLOPS%2C%20a%20DNA-binding%20transcriptional%20activator%2C%20orchestrates%20symbiotic%20root%20nodule%20development%2C&amp;author=S.%20Singh&amp;author=K.%20Katzer&amp;author=J.%20Lambert&amp;author=M.%20Cerri&amp;author=M.%20Parniske&amp;publication_year=2014&amp;journal=Cell%20Host%20Microbe&amp;volume=15&amp;pages=139-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.chom.2014.01.011" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.chom.2014.01.011" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.chom.2014.01.011"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24528861" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw052-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw052-B33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Messinese</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Mun</div> <div class="given-names">J. H.</div></div>, <div class="name string-name"><div class="surname">Yeun</div> <div class="given-names">L. H.</div></div></div>, et al.  . <div class="year">2007</div>, 
<div class="article-title">A novel nuclear protein interacts with the symbiotic DMI3 calcium- and calmodulin-dependent protein kinase of <em>Medicago truncatula</em>,</div> <div class="source">Mol. Plant Microbe. Interact</div> ., <div class="volume">20</div>, <div class="fpage">912</div>–<div class="lpage">21</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20novel%20nuclear%20protein%20interacts%20with%20the%20symbiotic%20DMI3%20calcium-%20and%20calmodulin-dependent%20protein%20kinase%20of%20Medicago%20truncatula%2C&amp;author=E.%20Messinese&amp;author=J.%20H.%20Mun&amp;author=L.%20H.%20Yeun&amp;publication_year=2007&amp;journal=Mol.%20Plant%20Microbe.%20Interact&amp;volume=20&amp;pages=912-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1094/MPMI-20-8-0912" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1094%2FMPMI-20-8-0912" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1094%2FMPMI-20-8-0912"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17722695" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw052-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw052-B34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kiss</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Oláh</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Kaló</div> <div class="given-names">P.</div></div></div>, et al.  . <div class="year">2009</div>, 
<div class="article-title">LIN, a novel type of U-box/WD40 protein, controls early infection by rhizobia in legumes,</div> <div class="source">Plant Physiol</div> ., <div class="volume">151</div>, <div class="fpage">1239</div>–<div class="lpage">49</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=LIN%2C%20a%20novel%20type%20of%20U-box%2FWD40%20protein%2C%20controls%20early%20infection%20by%20rhizobia%20in%20legumes%2C&amp;author=E.%20Kiss&amp;author=B.%20Ol%C3%A1h&amp;author=P.%20Kal%C3%B3&amp;publication_year=2009&amp;journal=Plant%20Physiol&amp;volume=151&amp;pages=1239-49" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.109.143933" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.109.143933" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.109.143933"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19776163" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw052-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw052-B35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Middleton</div> <div class="given-names">P. H.</div></div>, <div class="name string-name"><div class="surname">Jakab</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Penmetsa</div> <div class="given-names">R. V.</div></div></div>, et al.  . <div class="year">2007</div>, 
<div class="article-title">An ERF transcription factor in <em>Medicago truncatula</em> that is essential for nod factor signal transduction,</div> <div class="source">Plant Cell</div> , <div class="volume">19</div>, <div class="fpage">1221</div>–<div class="lpage">34</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20ERF%20transcription%20factor%20in%20Medicago%20truncatula%20that%20is%20essential%20for%20nod%20factor%20signal%20transduction%2C&amp;author=P.%20H.%20Middleton&amp;author=J.%20Jakab&amp;author=R.%20V.%20Penmetsa&amp;publication_year=2007&amp;journal=Plant%20Cell&amp;volume=19&amp;pages=1221-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.106.048264" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.106.048264" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.106.048264"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17449807" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw052-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw052-B36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Andriankaja</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Boisson-Dernier</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Frances</div> <div class="given-names">L.</div></div></div>, et al.  . <div class="year">2007</div>, 
<div class="article-title">AP2-ERF transcription factors mediate Nod factor dependent Mt<em>ENOD11</em> activation in root hairs via a novel cis-regulatory motif,</div> <div class="source">Plant Cell</div> , <div class="volume">19</div>, <div class="fpage">2866</div>–<div class="lpage">85</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=AP2-ERF%20transcription%20factors%20mediate%20Nod%20factor%20dependent%20MtENOD11%20activation%20in%20root%20hairs%20via%20a%20novel%20cis-regulatory%20motif%2C&amp;author=A.%20Andriankaja&amp;author=A.%20Boisson-Dernier&amp;author=L.%20Frances&amp;publication_year=2007&amp;journal=Plant%20Cell&amp;volume=19&amp;pages=2866-85" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.107.052944" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.107.052944" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.107.052944"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17827349" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw052-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw052-B37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Cerri</div> <div class="given-names">M. R.</div></div>, <div class="name string-name"><div class="surname">Frances</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Laloum</div> <div class="given-names">T.</div></div></div>, et al.  . <div class="year">2012</div>, 
<div class="article-title"><em>Medicago truncatula</em> ERN transcription factors: Regulatory interplay with NSP1/NSP2 GRAS factors and expression dynamics throughout rhizobial infection,</div> <div class="source">Plant Physiol</div> ., <div class="volume">160</div>, <div class="fpage">2155</div>–<div class="lpage">72</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Medicago%20truncatula%20ERN%20transcription%20factors%3A%20Regulatory%20interplay%20with%20NSP1%2FNSP2%20GRAS%20factors%20and%20expression%20dynamics%20throughout%20rhizobial%20infection%2C&amp;author=M.%20R.%20Cerri&amp;author=L.%20Frances&amp;author=T.%20Laloum&amp;publication_year=2012&amp;journal=Plant%20Physiol&amp;volume=160&amp;pages=2155-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.112.203190" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.112.203190" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.112.203190"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23077241" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw052-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw052-B38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Magori</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Oka-Kira</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Shibata</div> <div class="given-names">S.</div></div></div>, et al.  . <div class="year">2009</div>, 
<div class="article-title">TOO MUCH LOVE, a root regulator associated with the long-distance control of nodulation in <em>Lotus japonicus</em>,</div> <div class="source">Mol. Plant Microbe. Interact</div> ., <div class="volume">22</div>, <div class="fpage">259</div>–<div class="lpage">68</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=TOO%20MUCH%20LOVE%2C%20a%20root%20regulator%20associated%20with%20the%20long-distance%20control%20of%20nodulation%20in%20Lotus%20japonicus%2C&amp;author=S.%20Magori&amp;author=E.%20Oka-Kira&amp;author=S.%20Shibata&amp;publication_year=2009&amp;journal=Mol.%20Plant%20Microbe.%20Interact&amp;volume=22&amp;pages=259-68" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1094/MPMI-22-3-0259" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1094%2FMPMI-22-3-0259" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1094%2FMPMI-22-3-0259"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19245320" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw052-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw052-B39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kawaguchi</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Imaizumi-Anraku</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Koiwa</div> <div class="given-names">H.</div></div></div>, et al.  . <div class="year">2002</div>, 
<div class="article-title">Root, root hair, and symbiotic mutants of the model legume <em>Lotus japonicus</em>,</div> <div class="source">Mol. Plant-Microbe Interact</div> ., <div class="volume">15</div>, <div class="fpage">17</div>–<div class="lpage">26</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Root%2C%20root%20hair%2C%20and%20symbiotic%20mutants%20of%20the%20model%20legume%20Lotus%20japonicus%2C&amp;author=M.%20Kawaguchi&amp;author=H.%20Imaizumi-Anraku&amp;author=H.%20Koiwa&amp;publication_year=2002&amp;journal=Mol.%20Plant-Microbe%20Interact&amp;volume=15&amp;pages=17-26" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1094/MPMI.2002.15.1.17" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1094%2FMPMI.2002.15.1.17" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1094%2FMPMI.2002.15.1.17"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11843301" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw052-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw052-B40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Broughton</div> <div class="given-names">W. J.</div></div>, <div class="name string-name"><div class="surname">Dilworth</div> <div class="given-names">M. J.</div></div></div> <div class="year">1971</div>, 
<div class="article-title">Control of leghaemoglobin synthesis in snake beans,</div> <div class="source">Biochem J</div> ., <div class="volume">125</div>, <div class="fpage">1075</div>–<div class="lpage">80</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Control%20of%20leghaemoglobin%20synthesis%20in%20snake%20beans%2C&amp;author=W.%20J.%20Broughton&amp;author=M.%20J.%20Dilworth&amp;publication_year=1971&amp;journal=Biochem%20J&amp;volume=125&amp;pages=1075-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1042/bj1251075" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1042%2Fbj1251075" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1042%2Fbj1251075"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/5144223" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw052-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw052-B41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Maekawa</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Maekawa-Yoshikawa</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Takeda</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Imaizumi-Anraku</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Murooka</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Hayashi</div> <div class="given-names">M.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Gibberellin controls the nodulation signaling pathway in <em>Lotus japonicus</em>,</div> <div class="source">Plant J</div> ., <div class="volume">58</div>, <div class="fpage">183</div>–<div class="lpage">94</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gibberellin%20controls%20the%20nodulation%20signaling%20pathway%20in%20Lotus%20japonicus%2C&amp;author=T.%20Maekawa&amp;author=M.%20Maekawa-Yoshikawa&amp;author=N.%20Takeda&amp;author=H.%20Imaizumi-Anraku&amp;author=Y.%20Murooka&amp;author=M.%20Hayashi&amp;publication_year=2009&amp;journal=Plant%20J&amp;volume=58&amp;pages=183-94" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.2009.58.issue-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2009.58.issue-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.2009.58.issue-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19121107" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw052-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw052-B42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Diaz</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Gronlund</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Schlaman</div> <div class="given-names">H. R. M.</div></div>, <div class="name string-name"><div class="surname">Spaink</div> <div class="given-names">H.P.</div></div></div> <div class="year">2005</div>, Induction of hairy roots for symbiotic gene expression studies. In: <div class="person-group"><div class="name string-name"><div class="surname">Marquez</div> <div class="given-names">A.J.</div></div></div> (ed), <div class="source">Lotus japonicus Handbook</div> . 
<div class="publisher-name">Springer</div>: 
<div class="publisher-loc">Dordrecht</div>, pp. <div class="fpage">261</div>–<div class="lpage">77</div>.</div></div></div></div><div content-id="dsw052-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw052-B43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Katoh</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Kuma</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Toh</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Miyata</div> <div class="given-names">T.</div></div></div>. <div class="year">2005</div>, 
<div class="article-title">MAFFT version 5: improvement in accuracy of multiple sequence alignment,</div> <div class="source">Nucleic Acids Res</div> ., <div class="volume">33</div>, <div class="fpage">511</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MAFFT%20version%205%3A%20improvement%20in%20accuracy%20of%20multiple%20sequence%20alignment%2C&amp;author=K.%20Katoh&amp;author=K.%20Kuma&amp;author=H.%20Toh&amp;author=T.%20Miyata&amp;publication_year=2005&amp;journal=Nucleic%20Acids%20Res&amp;volume=33&amp;pages=511-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gki198" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgki198" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgki198"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15661851" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw052-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw052-B44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Stamatakis</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Ludwig</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Meier</div> <div class="given-names">H.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">RAxML-III: a fast program for maximum likelihood-based inference of large phylogenetic trees,</div> <div class="source">Bioinformatics</div> , <div class="volume">21</div>, <div class="fpage">456</div>–<div class="lpage">63</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RAxML-III%3A%20a%20fast%20program%20for%20maximum%20likelihood-based%20inference%20of%20large%20phylogenetic%20trees%2C&amp;author=A.%20Stamatakis&amp;author=T.%20Ludwig&amp;author=H.%20Meier&amp;publication_year=2005&amp;journal=Bioinformatics&amp;volume=21&amp;pages=456-63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bti191" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbti191" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbti191"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15608047" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw052-B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw052-B45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Huelsenbeck</div> <div class="given-names">J. P.</div></div>, <div class="name string-name"><div class="surname">Ronquist</div> <div class="given-names">F.</div></div></div> <div class="year">2001</div>, 
<div class="article-title">MRBAYES: Bayesian inference of phylogeny,</div> <div class="source">Bioinformatics</div> , <div class="volume">17</div>, <div class="fpage">754</div>–<div class="lpage">755</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MRBAYES%3A%20Bayesian%20inference%20of%20phylogeny%2C&amp;author=J.%20P.%20Huelsenbeck&amp;author=F.%20Ronquist&amp;publication_year=2001&amp;journal=Bioinformatics&amp;volume=17&amp;pages=754-755" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/17.8.754" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2F17.8.754" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2F17.8.754"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11524383" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw052-B46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw052-B46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tamura</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Peterson</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Peterson</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Stecher</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Nei</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Kumar</div> <div class="given-names">S.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">MEGA5: molecular evolutionary genetics analysis using maximum likelihood, evolutionary distance, and maximum parsimony methods,</div> <div class="source">Mol. Biol. Evol</div> ., <div class="volume">28</div>, <div class="fpage">2731</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MEGA5%3A%20molecular%20evolutionary%20genetics%20analysis%20using%20maximum%20likelihood%2C%20evolutionary%20distance%2C%20and%20maximum%20parsimony%20methods%2C&amp;author=K.%20Tamura&amp;author=D.%20Peterson&amp;author=N.%20Peterson&amp;author=G.%20Stecher&amp;author=M.%20Nei&amp;author=S.%20Kumar&amp;publication_year=2011&amp;journal=Mol.%20Biol.%20Evol&amp;volume=28&amp;pages=2731-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msr121" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsr121" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsr121"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21546353" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw052-B47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw052-B47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yang</div> <div class="given-names">Z.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">PAML 4: Phylogenetic analysis by maximum likelihood</div>, <div class="source">Mol. Biol. Evol</div> <em>.</em>, <div class="volume">23</div>, <div class="fpage">7</div>–<div class="lpage">9</div>.</div></div></div></div><div content-id="dsw052-B48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw052-B48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Imaizumi-Anraku</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Kouchi</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Syono</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Akao</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Kawaguchi</div> <div class="given-names">M.</div></div></div> <div class="year">2000</div>, 
<div class="article-title">Analysis of <em>ENOD40</em> expression in <em>alb1</em>, a symbiotic mutant of <em>Lotus japonicus</em> that forms empty nodules with incompletely developed nodule vascular bundles,</div> <div class="source">Mol. Gen. Genet</div> ., <div class="volume">264</div>, <div class="fpage">402</div>–<div class="lpage">10</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20ENOD40%20expression%20in%20alb1%2C%20a%20symbiotic%20mutant%20of%20Lotus%20japonicus%20that%20forms%20empty%20nodules%20with%20incompletely%20developed%20nodule%20vascular%20bundles%2C&amp;author=H.%20Imaizumi-Anraku&amp;author=H.%20Kouchi&amp;author=K.%20Syono&amp;author=S.%20Akao&amp;author=M.%20Kawaguchi&amp;publication_year=2000&amp;journal=Mol.%20Gen.%20Genet&amp;volume=264&amp;pages=402-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s004380000330" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs004380000330" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs004380000330"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11129043" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw052-B49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw052-B49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tansengco</div> <div class="given-names">M. L.</div></div>, <div class="name string-name"><div class="surname">Hayashi</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Kawaguchi</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Imaizumi-Anraku</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Murooka</div> <div class="given-names">Y.</div></div></div> <div class="year">2003</div>, 
<div class="article-title"><em>crinkle</em>, a novel symbiotic mutant that affects the infection thread growth and alters the root hair, trichome, and seed development in <em>Lotus japonicus</em>,</div> <div class="source">Plant Physiol</div> ., <div class="volume">131</div>, <div class="fpage">1054</div>–<div class="lpage">63</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=crinkle%2C%20a%20novel%20symbiotic%20mutant%20that%20affects%20the%20infection%20thread%20growth%20and%20alters%20the%20root%20hair%2C%20trichome%2C%20and%20seed%20development%20in%20Lotus%20japonicus%2C&amp;author=M.%20L.%20Tansengco&amp;author=M.%20Hayashi&amp;author=M.%20Kawaguchi&amp;author=H.%20Imaizumi-Anraku&amp;author=Y.%20Murooka&amp;publication_year=2003&amp;journal=Plant%20Physiol&amp;volume=131&amp;pages=1054-63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.102.017020" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.102.017020" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.102.017020"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12644658" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw052-B50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw052-B50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kawaharada</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Kelly</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Nielsen</div> <div class="given-names">M. W.</div></div></div>, et al.  . <div class="year">2015</div>, 
<div class="article-title">Receptor-mediated exopolysaccharide perception controls bacterial infection,</div> <div class="source">Nature</div> , <div class="volume">523</div>, <div class="fpage">308</div>–<div class="lpage">12</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Receptor-mediated%20exopolysaccharide%20perception%20controls%20bacterial%20infection%2C&amp;author=Y.%20Kawaharada&amp;author=S.%20Kelly&amp;author=M.%20W.%20Nielsen&amp;publication_year=2015&amp;journal=Nature&amp;volume=523&amp;pages=308-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature14611" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature14611" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature14611"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26153863" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw052-B51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw052-B51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Okamoto</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Ohnishi</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Sato</div> <div class="given-names">S.</div></div></div>, et al.  . <div class="year">2009</div>, 
<div class="article-title">Nod factor/nitrate-induced <em>CLE</em> genes that drive HAR1-mediated systemic regulation of nodulation,</div> <div class="source">Plant Cell Physiol</div> ., <div class="volume">50</div>, <div class="fpage">67</div>–<div class="lpage">77</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nod%20factor%2Fnitrate-induced%20CLE%20genes%20that%20drive%20HAR1-mediated%20systemic%20regulation%20of%20nodulation%2C&amp;author=S.%20Okamoto&amp;author=E.%20Ohnishi&amp;author=S.%20Sato&amp;publication_year=2009&amp;journal=Plant%20Cell%20Physiol&amp;volume=50&amp;pages=67-77" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/pcp/pcn194" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fpcp%2Fpcn194" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fpcp%2Fpcn194"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19074184" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw052-B52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw052-B52" class="ref-content "><div class="label -label">52</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bhuvaneswari</div> <div class="given-names">T. V.</div></div>, <div class="name string-name"><div class="surname">Turgeon</div> <div class="given-names">B. G.</div></div>, <div class="name string-name"><div class="surname">Bauer</div> <div class="given-names">W. D.</div></div></div> <div class="year">1980</div>, 
<div class="article-title">Early Events in the Infection of Soybean (<em>Glycine max</em> L. Merr) by <em>Rhizobium japonicum</em>: I. LOCALIZATION OF INFECTIBLE ROOT CELLS,</div> <div class="source">Plant Physiol</div> ., <div class="volume">66</div>, <div class="fpage">1027</div>–<div class="lpage">1031</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Early%20Events%20in%20the%20Infection%20of%20Soybean%20(Glycine%20max%20L.%20Merr)%20by%20Rhizobium%20japonicum%3A%20I.%20LOCALIZATION%20OF%20INFECTIBLE%20ROOT%20CELLS%2C&amp;author=T.%20V.%20Bhuvaneswari&amp;author=B.%20G.%20Turgeon&amp;author=W.%20D.%20Bauer&amp;publication_year=1980&amp;journal=Plant%20Physiol&amp;volume=66&amp;pages=1027-1031" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.66.6.1027" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.66.6.1027" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.66.6.1027"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16661570" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw052-B53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw052-B53" class="ref-content "><div class="label -label">53</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Murakami</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Yokoyama</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Fukui</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Kawaguchi</div> <div class="given-names">M.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Down-regulation of <em>NSP2</em> expression in developmentally young regions of <em>Lotus japonicus</em> roots in response to rhizobial inoculation,</div> <div class="source">Plant Cell Physiol</div> ., <div class="volume">54</div>, <div class="fpage">518</div>–<div class="lpage">27</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Down-regulation%20of%20NSP2%20expression%20in%20developmentally%20young%20regions%20of%20Lotus%20japonicus%20roots%20in%20response%20to%20rhizobial%20inoculation%2C&amp;author=Y.%20Murakami&amp;author=H.%20Yokoyama&amp;author=R.%20Fukui&amp;author=M.%20Kawaguchi&amp;publication_year=2013&amp;journal=Plant%20Cell%20Physiol&amp;volume=54&amp;pages=518-27" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/pcp/pct008" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fpcp%2Fpct008" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fpcp%2Fpct008"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23335614" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw052-B54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw052-B54" class="ref-content "><div class="label -label">54</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Cerri</div> <div class="given-names">M. R.</div></div>, <div class="name string-name"><div class="surname">Gamas</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">de Carvalho-Niebel</div> <div class="given-names">F.</div></div></div> <div class="year">2015</div>, AP2/ERF transcription factors and root nodulation. In: <div class="person-group"><div class="name string-name"><div class="surname">de Bruijn</div> <div class="given-names">F.J.</div></div></div> (ed), <div class="source">Biological Nitrogen Fixation</div> , <div class="volume">vol. 2</div>, 
<div class="publisher-name">Wiley-Blackwell</div>: 
<div class="publisher-loc">Hoboken</div>, pp. <div class="fpage">609</div>–<div class="lpage">22</div>.</div></div></div></div><div content-id="dsw052-B55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw052-B55" class="ref-content "><div class="label -label">55</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Cerri</div> <div class="given-names">M. R.</div></div>, <div class="name string-name"><div class="surname">Frances</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Kelner</div> <div class="given-names">A.</div></div></div>, et al.  . <div class="year">2016</div>, 
<div class="article-title">The symbiosis-related ERN transcription factors act in concert to coordinate rhizobial host root infection,</div> <div class="source">Plant Physiol</div> ., <div class="volume">171</div>, <div class="fpage">1037</div>–<div class="lpage">54</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20symbiosis-related%20ERN%20transcription%20factors%20act%20in%20concert%20to%20coordinate%20rhizobial%20host%20root%20infection%2C&amp;author=M.%20R.%20Cerri&amp;author=L.%20Frances&amp;author=A.%20Kelner&amp;publication_year=2016&amp;journal=Plant%20Physiol&amp;volume=171&amp;pages=1037-54" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27208242" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw052-B56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw052-B56" class="ref-content "><div class="label -label">56</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Cannon</div> <div class="given-names">S. B.</div></div>, <div class="name string-name"><div class="surname">Sterck</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Rombauts</div> <div class="given-names">S.</div></div></div>, et al.  . <div class="year">2006</div>, 
<div class="article-title">Legume genome evolution viewed through the <em>Medicago truncatula</em> and <em>Lotus japonicus</em> genomes,</div> <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="volume">103</div>, <div class="fpage">14959</div>–<div class="lpage">64</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Legume%20genome%20evolution%20viewed%20through%20the%20Medicago%20truncatula%20and%20Lotus%20japonicus%20genomes%2C&amp;author=S.%20B.%20Cannon&amp;author=L.%20Sterck&amp;author=S.%20Rombauts&amp;publication_year=2006&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=103&amp;pages=14959-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0603228103" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0603228103" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0603228103"> </span></div> </div></div></div></div></div><div content-id="dsw052-B57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw052-B57" class="ref-content "><div class="label -label">57</div><div class="citation mixed-citation"><div class="collab">The Angiosperm Phylogeny Group</div> <div class="year">2016</div> 
<div class="article-title">An update of the Angiosperm Phylogeny Group classification for the orders and families of flowering plants: APG IV,</div> <div class="source">Bot. J. Linn. Soc.</div>  <div class="volume">181</div>, <div class="fpage">1095</div>–<div class="lpage">8339</div>.</div></div></div></div><div content-id="dsw052-B58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw052-B58" class="ref-content "><div class="label -label">58</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Arbiza</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Dopazo</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Dopazo</div> <div class="given-names">H.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">Positive selection, relaxation, and acceleration in the evolution of the human and chimp genome,</div> <div class="source">PLoS Comput. Biol</div> ., <div class="volume">2</div>, <div class="fpage">e38</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Positive%20selection%2C%20relaxation%2C%20and%20acceleration%20in%20the%20evolution%20of%20the%20human%20and%20chimp%20genome%2C&amp;author=L.%20Arbiza&amp;author=J.%20Dopazo&amp;author=H.%20Dopazo&amp;publication_year=2006&amp;journal=PLoS%20Comput.%20Biol&amp;volume=2&amp;pages=e38" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pcbi.0020038" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pcbi.0020038" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pcbi.0020038"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16683019" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw052-B59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw052-B59" class="ref-content "><div class="label -label">59</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zhong</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Yonezawa</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Zhong</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Hasegawa</div> <div class="given-names">M.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Episodic evolution and adaptation of chloroplast genomes in ancestral grasses,</div> <div class="source">PLoS One</div> , <div class="volume">4</div>, <div class="fpage">e529</div>.</div></div></div></div><div content-id="dsw052-B60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw052-B60" class="ref-content "><div class="label -label">60</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Neiman</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Hehman</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Miller</div> <div class="given-names">J. T.</div></div>, <div class="name string-name"><div class="surname">Logsdon</div> <div class="given-names">J. M.</div></div>, <div class="name string-name"><div class="surname">Taylor</div> <div class="given-names">D. R.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Accelerated mutation accumulation in asexual lineages of a fresh-water snail,</div> <div class="source">Mol. Biol. Evol</div> ., <div class="volume">27</div>, <div class="fpage">954</div>–<div class="lpage">63</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Accelerated%20mutation%20accumulation%20in%20asexual%20lineages%20of%20a%20fresh-water%20snail%2C&amp;author=M.%20Neiman&amp;author=G.%20Hehman&amp;author=J.%20T.%20Miller&amp;author=J.%20M.%20Logsdon&amp;author=D.%20R.%20Taylor&amp;publication_year=2010&amp;journal=Mol.%20Biol.%20Evol&amp;volume=27&amp;pages=954-63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msp300" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsp300" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsp300"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19995828" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw052-B61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw052-B61" class="ref-content "><div class="label -label">61</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yang</div> <div class="given-names">Z.</div></div></div> <div class="year">1998</div>, 
<div class="article-title">Likelihood ratio tests for detecting positive selection and application to primate lysozyme evolution,</div> <div class="source">Mol. Biol. Evol</div> ., <div class="volume">15</div>, <div class="fpage">568</div>–<div class="lpage">73</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Likelihood%20ratio%20tests%20for%20detecting%20positive%20selection%20and%20application%20to%20primate%20lysozyme%20evolution%2C&amp;author=Z.%20Yang&amp;publication_year=1998&amp;journal=Mol.%20Biol.%20Evol&amp;volume=15&amp;pages=568-73" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a025957" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Foxfordjournals.molbev.a025957" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Foxfordjournals.molbev.a025957"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9580986" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw052-B62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw052-B62" class="ref-content "><div class="label -label">62</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Soyano</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Kouchi</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Hirota</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Hayashi</div> <div class="given-names">M.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">NODULE INCEPTION directly targets <em>NF-Y</em> subunit genes to regulate essential processes of root nodule development in <em>Lotus japonicus</em>,</div> <div class="source">PLoS Genet</div> ., <div class="volume">9</div>, <div class="fpage">e1003352</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=NODULE%20INCEPTION%20directly%20targets%20NF-Y%20subunit%20genes%20to%20regulate%20essential%20processes%20of%20root%20nodule%20development%20in%20Lotus%20japonicus%2C&amp;author=T.%20Soyano&amp;author=H.%20Kouchi&amp;author=A.%20Hirota&amp;author=M.%20Hayashi&amp;publication_year=2013&amp;journal=PLoS%20Genet&amp;volume=9&amp;pages=e1003352" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1003352" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1003352" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1003352"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23555278" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw052-B63" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw052-B63" class="ref-content "><div class="label -label">63</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Laloum</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Baudin</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Frances</div> <div class="given-names">L.</div></div></div>, et al.  . <div class="year">2014</div>, 
<div class="article-title">Two CCAAT-box-binding transcription factors redundantly regulate early steps of the legume-rhizobia endosymbiosis,</div> <div class="source">Plant J</div> ., <div class="volume">79</div>, <div class="fpage">757</div>–<div class="lpage">68</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Two%20CCAAT-box-binding%20transcription%20factors%20redundantly%20regulate%20early%20steps%20of%20the%20legume-rhizobia%20endosymbiosis%2C&amp;author=T.%20Laloum&amp;author=M.%20Baudin&amp;author=L.%20Frances&amp;publication_year=2014&amp;journal=Plant%20J&amp;volume=79&amp;pages=757-68" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.2014.79.issue-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2014.79.issue-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.2014.79.issue-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24930743" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw052-B64" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw052-B64" class="ref-content "><div class="label -label">64</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Young</div> <div class="given-names">N. D.</div></div>, <div class="name string-name"><div class="surname">Debellé</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Oldroyd</div> <div class="given-names">G. E.</div></div></div>, et al.  . <div class="year">2011</div>, 
<div class="article-title">The <em>Medicago</em> genome provides insight into the evolution of rhizobial symbioses,</div> <div class="source">Nature</div> , <div class="volume">480</div>, <div class="fpage">520</div>–<div class="lpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Medicago%20genome%20provides%20insight%20into%20the%20evolution%20of%20rhizobial%20symbioses%2C&amp;author=N.%20D.%20Young&amp;author=F.%20Debell%C3%A9&amp;author=G.%20E.%20Oldroyd&amp;publication_year=2011&amp;journal=Nature&amp;volume=480&amp;pages=520-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/480162a" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F480162a" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F480162a"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22089132" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw052-B65" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw052-B65" class="ref-content "><div class="label -label">65</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Crandall</div> <div class="given-names">K. A.</div></div>, <div class="name string-name"><div class="surname">Hillis</div> <div class="given-names">D. M.</div></div></div>. <div class="year">1997</div>, 
<div class="article-title">Rhodopsin evolution in the dark,</div> <div class="source">Nature</div> , <div class="volume">387</div>, <div class="fpage">667</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rhodopsin%20evolution%20in%20the%20dark%2C&amp;author=K.%20A.%20Crandall&amp;author=D.%20M.%20Hillis&amp;publication_year=1997&amp;journal=Nature&amp;volume=387&amp;pages=667-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/42628" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F42628" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F42628"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9192889" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw052-B66" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw052-B66" class="ref-content "><div class="label -label">66</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yang</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Nielsen</div> <div class="given-names">R.</div></div></div> <div class="year">2000</div>, 
<div class="article-title">Estimating synonymous and non-synonymous substitution rates under realistic evolutionary models,</div> <div class="source">Mol. Biol. Evol</div> ., <div class="volume">17</div>, <div class="fpage">32</div>–<div class="lpage">43</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Estimating%20synonymous%20and%20non-synonymous%20substitution%20rates%20under%20realistic%20evolutionary%20models%2C&amp;author=Z.%20Yang&amp;author=R.%20Nielsen&amp;publication_year=2000&amp;journal=Mol.%20Biol.%20Evol&amp;volume=17&amp;pages=32-43" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a026236" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Foxfordjournals.molbev.a026236" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Foxfordjournals.molbev.a026236"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10666704" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw052-B67" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw052-B67" class="ref-content "><div class="label -label">67</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Nakano</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Suzuki</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Fujimura</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Shinshi</div> <div class="given-names">H.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">Genome-wide analysis of the ERF gene family in Arabidopsis and rice,</div> <div class="source">Plant Physiol</div> ., <div class="volume">140</div>, <div class="fpage">411</div>–<div class="lpage">32</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20analysis%20of%20the%20ERF%20gene%20family%20in%20Arabidopsis%20and%20rice%2C&amp;author=T.%20Nakano&amp;author=K.%20Suzuki&amp;author=T.%20Fujimura&amp;author=H.%20Shinshi&amp;publication_year=2006&amp;journal=Plant%20Physiol&amp;volume=140&amp;pages=411-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.105.073783" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.105.073783" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.105.073783"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16407444" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2016. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=2731026&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw052/2/dsw052_Supp.zip?Expires=2147483647&amp;Signature=SUfZUEyOx0cNgvbTgqmG5Jlc7D-wCgPBRu~nkZYtDlnDJ3y08Dk1egjJie4Bi9r0o~3T21J6w8mfvpFrbyNAq2drb5sFojcQL96jELnU9bogfK5Fi6LNtPAWKyHiFJfKZuy9UtsMZnYsh1Y3IfDP7q7xrQpM6Tdb~N-zpw88J5aGu5UvF0EP7Aurb-xgRQ4kG3PjHn5IFQBvYeAj7yMCbHPe5dB-ORrJjjGmlosU5TPSzRe3LJ9SSIUQBp5XOPrQ-r~s3VpZ6k98RKm2Ew2J5eCHjxuifUtwtPKDFIBqI6CYZsVtizidvxoFwOjgWAfKnY5U7a7aAxOp4ZGx9gcKTg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">1,069</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000400913000008&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">8</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsw052"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="2731026"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000400913000008&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f24%2f2%2f193%2f2731026">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000400913000008&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (8)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F24%2F2%2F193%2F2731026" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/2731026'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Function and evolution of a Lotus japonicus AP2/ERF family transcription factor that is required for development of infection threads" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsw052");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f24%2f2%2f193%2f2731026">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

