

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/19/2/103/418941","@type":"ScholarlyArticle","name":"Unveiling  Mycoplasma hyopneumoniae  Promoters: Sequence Definition and Genomic Distribution","datePublished":"2012-02-14","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/19/2","@type":"PublicationIssue","issueNumber":"2","datePublished":"2012-04-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsr045","keywords":["<italic>Mycoplasma</italic>","promoter","transcription","sigma","matrix"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/19/2/103/418941","author":[{"name":"Weber, Shana de Souto","affiliation":"1  Centro de Biotecnologia, Programa de Pós-graduação em Biologia Celular e Molecular ,  Universidade Federal do Rio Grande do Sul (UFRGS) ,  Av. Bento Gonçalves 9500, C.P. 15005, CEP 91501-970, Porto Alegre, RS ,  Brazil","@type":"Person"},{"name":"Sant'Anna, Fernando Hayashi","affiliation":"1  Centro de Biotecnologia, Programa de Pós-graduação em Biologia Celular e Molecular ,  Universidade Federal do Rio Grande do Sul (UFRGS) ,  Av. Bento Gonçalves 9500, C.P. 15005, CEP 91501-970, Porto Alegre, RS ,  Brazil","@type":"Person"},{"name":"Schrank, Irene Silveira","affiliation":"1  Centro de Biotecnologia, Programa de Pós-graduação em Biologia Celular e Molecular ,  Universidade Federal do Rio Grande do Sul (UFRGS) ,  Av. Bento Gonçalves 9500, C.P. 15005, CEP 91501-970, Porto Alegre, RS ,  Brazil    2  Departamento de Biologia Molecular e Biotecnologia - Centro de Biotecnologia ,  Universidade Federal do Rio Grande do Sul (UFRGS) ,  Av. Bento Gonçalves 9500, C.P. 15005, CEP 91501-970, Porto Alegre, RS ,  Brazil","@type":"Person"}],"description":"Abstract.  Several Mycoplasma species have had their genome completely sequenced, including four strains of the swine pathogen Mycoplasma hyopneumoniae. Neverth","pageStart":"103","pageEnd":"115","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/19/2/1/m_cover.gif?Expires=2147483647&Signature=EMTi8MiwskfL8whcuG7gPpMQVkoTywhsN9k2fcs1qMWyjLc9OmU3zzfeux1LcC8edMnaqIc6tYNqgsUJ8hCH0UQnGgbUVvjHa69ZDqc3rwPQg3T0QzS9XdepkqXV57OA92AfnyIdwEzEggsZ18zDGYx5fC~exu1TTRcnk2bBo6TdYsVgw52BF-63NakC5-nPvHQVSqNzjaFosams6g1DZ~UchQ1c7lmvgwn6cnAg013zUUb9XUUhijVu63SQWmhw0ByRocoE2U4pz7BLDSieaJZqPdVUaKjHeYkUlMHEhZhGv0eccckPuOqL2UhMZ0FzVdPiNB6hKa39ET8qYgw8~Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Unveiling  Mycoplasma hyopneumoniae  Promoters: Sequence Definition and Genomic Distribution","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/19/2/1/m_cover.gif?Expires=2147483647&Signature=EMTi8MiwskfL8whcuG7gPpMQVkoTywhsN9k2fcs1qMWyjLc9OmU3zzfeux1LcC8edMnaqIc6tYNqgsUJ8hCH0UQnGgbUVvjHa69ZDqc3rwPQg3T0QzS9XdepkqXV57OA92AfnyIdwEzEggsZ18zDGYx5fC~exu1TTRcnk2bBo6TdYsVgw52BF-63NakC5-nPvHQVSqNzjaFosams6g1DZ~UchQ1c7lmvgwn6cnAg013zUUb9XUUhijVu63SQWmhw0ByRocoE2U4pz7BLDSieaJZqPdVUaKjHeYkUlMHEhZhGv0eccckPuOqL2UhMZ0FzVdPiNB6hKa39ET8qYgw8~Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Unveiling  Mycoplasma hyopneumoniae  Promoters: Sequence Definition and Genomic Distribution" />
    <meta property="og:description" content="Abstract.  Several Mycoplasma species have had their genome completely sequenced, including four strains of the swine pathogen Mycoplasma hyopneumoniae. Neverth" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/19/2/103/418941" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2012-02-14" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Weber, Shana de Souto" /><meta name="citation_author_institution" content="Centro de Biotecnologia, Programa de Pós-graduação em Biologia Celular e Molecular, Universidade Federal do Rio Grande do Sul (UFRGS), Av. Bento Gonçalves 9500, C.P. 15005, CEP 91501-970, Porto Alegre, RS, Brazil" /><meta name="citation_author" content="Sant'Anna, Fernando Hayashi" /><meta name="citation_author_institution" content="Centro de Biotecnologia, Programa de Pós-graduação em Biologia Celular e Molecular, Universidade Federal do Rio Grande do Sul (UFRGS), Av. Bento Gonçalves 9500, C.P. 15005, CEP 91501-970, Porto Alegre, RS, Brazil" /><meta name="citation_author" content="Schrank, Irene Silveira" /><meta name="citation_author_institution" content="Centro de Biotecnologia, Programa de Pós-graduação em Biologia Celular e Molecular, Universidade Federal do Rio Grande do Sul (UFRGS), Av. Bento Gonçalves 9500, C.P. 15005, CEP 91501-970, Porto Alegre, RS, Brazil" /><meta name="citation_author_institution" content="Departamento de Biologia Molecular e Biotecnologia - Centro de Biotecnologia, Universidade Federal do Rio Grande do Sul (UFRGS), Av. Bento Gonçalves 9500, C.P. 15005, CEP 91501-970, Porto Alegre, RS, Brazil" /><meta name="citation_title" content="Unveiling Mycoplasma hyopneumoniae Promoters: Sequence Definition and Genomic Distribution" /><meta name="citation_firstpage" content="103" /><meta name="citation_lastpage" content="115" /><meta name="citation_doi" content="10.1093/dnares/dsr045" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="19" /><meta name="citation_issue" content="2" /><meta name="citation_publication_date" content="2012/04/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_publisher=Iowa State University Press, Ames; Diseases for Swine;  citation_year=2006;  citation_pages=701-17; " /><meta name="citation_reference" content="citation_title=The genome sequence of Mycoplasma hyopneumoniae strain 232, the agent of swine mycoplasmosis; J. Bacteriol.;  citation_year=2004;  citation_volume=186;  citation_pages=7123-33; " /><meta name="citation_reference" content="citation_title=Swine and poultry pathogens: the complete genome sequences of two strains of Mycoplasma hyopneumoniae and a strain of Mycoplasma synoviae; J. Bacteriol.;  citation_year=2005;  citation_volume=187;  citation_pages=5568-77; " /><meta name="citation_reference" content="citation_title=Complete genome sequence of Mycoplasma hyopneumoniae strain 168; J. Bacteriol.;  citation_year=2011;  citation_volume=193;  citation_pages=1016-7; " /><meta name="citation_reference" content="citation_title=The minimal gene complement of Mycoplasma genitalium; Science;  citation_year=1995;  citation_volume=270;  citation_pages=397-403; " /><meta name="citation_reference" content="citation_title=The complete genome sequence of Escherichia coli K-12; Science;  citation_year=1997;  citation_volume=277;  citation_pages=1453-74; " /><meta name="citation_reference" content="citation_title=The complete genome sequence of the gram-positive bacterium Bacillus subtilis; Nature;  citation_year=1997;  citation_volume=390;  citation_pages=249-56; " /><meta name="citation_reference" content="citation_title=Transcription in Mycoplasma pneumoniae; Nucleic Acids Res.;  citation_year=2000;  citation_volume=28;  citation_pages=4488-96; " /><meta name="citation_reference" content="citation_title=Some recommendations concerning primary isolation of Mycoplasma suipneumoniae and Mycoplasma flocculare a survey; Nord. Vet. Med.;  citation_year=1975;  citation_volume=27;  citation_pages=337-9; " /><meta name="citation_reference" content="citation_publisher=Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY; Molecular Cloning: A Laboratory Manual;  citation_year=2001; " /><meta name="citation_reference" content="citation_title=Sensitive detection of bacterial transcription initiation sites and differentiation from RNA processing sites in the pheromone-induced plasmid transfer system of Enterococcus faecalis; Proc. Natl Acad. Sci. USA;  citation_year=1996;  citation_volume=93;  citation_pages=7794-9; " /><meta name="citation_reference" content="citation_title=Sequence logos: a new way to display consensus sequences; Nucleic Acids Res.;  citation_year=1990;  citation_volume=18;  citation_pages=6097-100; " /><meta name="citation_reference" content="citation_title=WebLogo: a sequence logo generator; Genome Res.;  citation_year=2004;  citation_volume=14;  citation_pages=1188-90; " /><meta name="citation_reference" content="citation_title=Promoter prediction in the rhizobia; Microbiology;  citation_year=2006;  citation_volume=152;  citation_pages=1751-63; " /><meta name="citation_reference" content="citation_title=Compilation and analysis of Escherichia coli promoter DNA sequences; Nucleic Acids Res.;  citation_year=1983;  citation_volume=11;  citation_pages=2237-55; " /><meta name="citation_reference" content="citation_title=Compilation and analysis of Bacillus subtilis sigma A-dependent promoter sequences: evidence for extended contact between RNA polymerase and upstream promoter DNA; Nucleic Acids Res.;  citation_year=1995;  citation_volume=23;  citation_pages=2351-60; " /><meta name="citation_reference" content="citation_title=Genome-wide analysis of Chlamydiae for promoters that phylogenetically footprint; Res. Microbiol.;  citation_year=2007;  citation_volume=158;  citation_pages=685-93; " /><meta name="citation_reference" content="citation_title=Experimental and computational analysis of transcriptional start sites in the cyanobacterium Prochlorococcus MED4; Nucleic Acids Res.;  citation_year=2003;  citation_volume=31;  citation_pages=2890-9; " /><meta name="citation_reference" content="citation_title=Identification of Campylobacter jejuni promoter sequences; J. Bacteriol.;  citation_year=1998;  citation_volume=180;  citation_pages=594-9; " /><meta name="citation_reference" content="citation_title=Using RSAT oligo-analysis and dyad-analysis tools to discover regulatory signals in nucleic sequences; Nat. Protoc.;  citation_year=2008;  citation_volume=3;  citation_pages=1589-603; " /><meta name="citation_reference" content="citation_title=Regulatory sequence analysis tools; Nucleic Acids Res.;  citation_year=2003;  citation_volume=31;  citation_pages=3593-6; " /><meta name="citation_reference" content="citation_title=RSAT: regulatory sequence analysis tools; Nucleic Acids Res.;  citation_year=2008;  citation_volume=36;  citation_pages=W119-27; " /><meta name="citation_reference" content="citation_title=BioEdit: a user-friendly biological sequence alignment editor and analysis program for Windows 95/98/NT; Nucleic Acids Symp. Ser.;  citation_year=1999;  citation_volume=41;  citation_pages=95-8; " /><meta name="citation_reference" content="citation_title=Fitting a mixture model by expectation maximization to discover motifs in biopolymers; Proc. Int. Conf. Intell. Syst. Mol. Biol.;  citation_year=1994;  citation_volume=2;  citation_pages=28-36; " /><meta name="citation_reference" content="citation_title=Identifying DNA and protein patterns with statistically significant alignments of multiple sequences; Bioinformatics;  citation_year=1999;  citation_volume=15;  citation_pages=563-77; " /><meta name="citation_reference" content="citation_title=Theoretical and empirical quality assessment of transcription factor-binding motifs; Nucleic Acids Res.;  citation_year=2011;  citation_volume=39;  citation_pages=808-24; " /><meta name="citation_reference" content="citation_title=Using RSAT to scan genome sequences for transcription factor binding sites and cis-regulatory modules; Nat. Protoc.;  citation_year=2008;  citation_volume=3;  citation_pages=1578-88; " /><meta name="citation_reference" content="citation_title=The -16 region, a vital sequence for the utilization of a promoter in Bacillus subtilis and Escherichia coli; Mol. Microbiol.;  citation_year=1995;  citation_volume=17;  citation_pages=271-9; " /><meta name="citation_reference" content="citation_title=The TRTGn motif stabilizes the transcription initiation open complex; J. Mol. Biol.;  citation_year=2002;  citation_volume=322;  citation_pages=521-32; " /><meta name="citation_reference" content="citation_title=The sigma54 regulon (sigmulon) of Pseudomonas putida; Environ. Microbiol.;  citation_year=2003;  citation_volume=5;  citation_pages=1281-93; " /><meta name="citation_reference" content="citation_title=Regulation of gene expression by reiterative transcription; Curr. Opin. Microbiol.;  citation_year=2011;  citation_volume=14;  citation_pages=142-7; " /><meta name="citation_reference" content="citation_title=Selection for unequal densities of sigma 70 promoter-like signals in different regions of large bacterial genomes; PLoS Genet.;  citation_year=2006;  citation_volume=2;  citation_pages=e185" /><meta name="citation_reference" content="citation_title=Sigma A recognition sites in the Bacillus subtilis genome; Microbiology;  citation_year=2001;  citation_volume=147;  citation_pages=2417-24; " /><meta name="citation_reference" content="citation_title=Identification and analysis of ‘extended -10’ promoters in Escherichia coli; Nucleic Acids Res.;  citation_year=2003;  citation_volume=31;  citation_pages=4689-95; " /><meta name="citation_reference" content="citation_title=Transcriptome complexity in a genome-reduced bacterium; Science;  citation_year=2009;  citation_volume=326;  citation_pages=1268-71; " /><meta name="citation_reference" content="citation_title=RpoD promoters in Campylobacter jejuni exhibit a strong periodic signal instead of a -35 box; J. Mol. Biol.;  citation_year=2003;  citation_volume=326;  citation_pages=1361-72; " /><meta name="citation_reference" content="citation_title=Transcription initiation by mix and match elements: flexibility for polymerase binding to bacterial promoters; Gene Regul. Syst. Bio;  citation_year=2007;  citation_volume=1;  citation_pages=275-93; " /><meta name="citation_reference" content="citation_title=Triad pattern algorithm for predicting strong promoter candidates in bacterial genomes; BMC Bioinformatics;  citation_year=2008;  citation_volume=9;  citation_pages=233" /><meta name="citation_reference" content="citation_title=Investigations of the modular structure of bacterial promoters; Biochem. Soc. Symp.;  citation_year=2006;  citation_volume=73;  citation_pages=1-10; " /><meta name="citation_reference" content="citation_title=Phage T4 early promoters are resistant to inhibition by the anti-sigma factor AsiA; Mol. Microbiol;  citation_year=2004;  citation_volume=52;  citation_pages=1013-28; " /><meta name="citation_reference" content="citation_title=Large-scale computational and statistical analyses of high transcription potentialities in 32 prokaryotic genomes; Nucleic Acids Res.;  citation_year=2008;  citation_volume=36;  citation_pages=3332-40; " /><meta name="citation_reference" content="citation_title=In vivo expression analysis of the P97 and P102 paralog families of Mycoplasma hyopneumoniae; Infect. Immun.;  citation_year=2005;  citation_volume=73;  citation_pages=7784-7; " /><meta name="citation_reference" content="citation_title=Detection and quantification of intergenic transcription in Mycoplasma hyopneumoniae; Microbiology;  citation_year=2010;  citation_volume=156;  citation_pages=2305-15; " /><meta name="citation_reference" content="citation_title=Mycoplasma hyopneumoniae transcription unit organization: genome survey and prediction; DNA Res.;  citation_year=2011;  citation_volume=18;  citation_pages=413-22; " /><meta name="citation_reference" content="citation_publisher=Elsevier, New York; RNA Polymerase and the Regulation of Transcription;  citation_year=1987;  citation_pages=105-13; " /><meta name="citation_reference" content="citation_title=Competing promoters in prokaryotic transcription; Trends Biochem. Sci.;  citation_year=1991;  citation_volume=16;  citation_pages=394-7; " /><meta name="citation_reference" content="citation_title=The sigma 70 subunit of RNA polymerase induces lacUV5 promoter-proximal pausing of transcription; Nat. Struct. Mol. Biol.;  citation_year=2004;  citation_volume=11;  citation_pages=551-7; " /><meta name="citation_reference" content="citation_title=The sigma 70 subunit of RNA polymerase mediates a promoter-proximal pause at the lac promoter; Nat. Struct. Mol. Biol.;  citation_year=2004;  citation_volume=11;  citation_pages=544-50; " /><meta name="citation_reference" content="citation_title=Transcription profiles of the bacterium Mycoplasma pneumoniae grown at different temperatures; Nucleic Acids Res.;  citation_year=2003;  citation_volume=31;  citation_pages=6306-20; " /><meta name="citation_reference" content="citation_title=Transcriptional profiling of Mycoplasma hyopneumoniae during heat shock using microarrays; Infect. Immun.;  citation_year=2006;  citation_volume=74;  citation_pages=160-6; " /><meta name="citation_reference" content="citation_title=Transcriptional profiling of Mycoplasma hyopneumoniae during iron depletion using microarrays; Microbiology;  citation_year=2006;  citation_volume=152;  citation_pages=937-44; " /><meta name="citation_reference" content="citation_title=Global transcriptional analysis of Mycoplasma hyopneumoniae following exposure to hydrogen peroxide; Microbiology;  citation_year=2007;  citation_volume=153;  citation_pages=3785-90; " /><meta name="citation_reference" content="citation_title=Global transcriptional analysis of Mycoplasma hyopneumoniae following exposure to norepinephrine; Microbiology;  citation_year=2008;  citation_volume=154;  citation_pages=2581-8; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/19/2/103/1161721/dsr045.pdf" /><meta name="description" content="Abstract.  Several Mycoplasma species have had their genome completely sequenced, including four strains of the swine pathogen Mycoplasma hyopneumoniae. Neverth" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/19/2/103/418941" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/19/2/103/418941" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/19/2/103/418941");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsr045';
    </script>
<title>
	Unveiling Mycoplasma hyopneumoniae Promoters: Sequence Definition and Genomic Distribution | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./418941" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0Bd9vPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJXZWJlciwgU2hhbmEgZGUgU291dG8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJDZW50cm8gZGUgQmlvdGVjbm9sb2dpYSwgUHJvZ3JhbWEgZGUgUMOzcy1ncmFkdWHDp8OjbyBlbSBCaW9sb2dpYSBDZWx1bGFyIGUgTW9sZWN1bGFyLCBVbml2ZXJzaWRhZGUgRmVkZXJhbCBkbyBSaW8gR3JhbmRlIGRvIFN1bCAoVUZSR1MpLCBBdi4gQmVudG8gR29uw6dhbHZlcyA5NTAwLCBDLlAuIDE1MDA1LCBDRVAgOTE1MDEtOTcwLCBQb3J0byBBbGVncmUsIFJTLCBCcmF6aWwiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTYW50J0FubmEsIEZlcm5hbmRvIEhheWFzaGkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJDZW50cm8gZGUgQmlvdGVjbm9sb2dpYSwgUHJvZ3JhbWEgZGUgUMOzcy1ncmFkdWHDp8OjbyBlbSBCaW9sb2dpYSBDZWx1bGFyIGUgTW9sZWN1bGFyLCBVbml2ZXJzaWRhZGUgRmVkZXJhbCBkbyBSaW8gR3JhbmRlIGRvIFN1bCAoVUZSR1MpLCBBdi4gQmVudG8gR29uw6dhbHZlcyA5NTAwLCBDLlAuIDE1MDA1LCBDRVAgOTE1MDEtOTcwLCBQb3J0byBBbGVncmUsIFJTLCBCcmF6aWwiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTY2hyYW5rLCBJcmVuZSBTaWx2ZWlyYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkNlbnRybyBkZSBCaW90ZWNub2xvZ2lhLCBQcm9ncmFtYSBkZSBQw7NzLWdyYWR1YcOnw6NvIGVtIEJpb2xvZ2lhIENlbHVsYXIgZSBNb2xlY3VsYXIsIFVuaXZlcnNpZGFkZSBGZWRlcmFsIGRvIFJpbyBHcmFuZGUgZG8gU3VsIChVRlJHUyksIEF2LiBCZW50byBHb27Dp2FsdmVzIDk1MDAsIEMuUC4gMTUwMDUsIENFUCA5MTUwMS05NzAsIFBvcnRvIEFsZWdyZSwgUlMsIEJyYXppbCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydGFtZW50byBkZSBCaW9sb2dpYSBNb2xlY3VsYXIgZSBCaW90ZWNub2xvZ2lhIC0gQ2VudHJvIGRlIEJpb3RlY25vbG9naWEsIFVuaXZlcnNpZGFkZSBGZWRlcmFsIGRvIFJpbyBHcmFuZGUgZG8gU3VsIChVRlJHUyksIEF2LiBCZW50byBHb27Dp2FsdmVzIDk1MDAsIEMuUC4gMTUwMDUsIENFUCA5MTUwMS05NzAsIFBvcnRvIEFsZWdyZSwgUlMsIEJyYXppbCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iVW52ZWlsaW5nIE15Y29wbGFzbWEgaHlvcG5ldW1vbmlhZSBQcm9tb3RlcnM6IFNlcXVlbmNlIERlZmluaXRpb24gYW5kIEdlbm9taWMgRGlzdHJpYnV0aW9uIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iMTAzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSIxMTUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3IwNDUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMTkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxMi8wNC8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3B1Ymxpc2hlcj1Jb3dhIFN0YXRlIFVuaXZlcnNpdHkgUHJlc3MsIEFtZXM7IERpc2Vhc2VzIGZvciBTd2luZTsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3BhZ2VzPTcwMS0xNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGdlbm9tZSBzZXF1ZW5jZSBvZiBNeWNvcGxhc21hIGh5b3BuZXVtb25pYWUgc3RyYWluIDIzMiwgdGhlIGFnZW50IG9mIHN3aW5lIG15Y29wbGFzbW9zaXM7IEouIEJhY3RlcmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTg2OyAgY2l0YXRpb25fcGFnZXM9NzEyMy0zMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3dpbmUgYW5kIHBvdWx0cnkgcGF0aG9nZW5zOiB0aGUgY29tcGxldGUgZ2Vub21lIHNlcXVlbmNlcyBvZiB0d28gc3RyYWlucyBvZiBNeWNvcGxhc21hIGh5b3BuZXVtb25pYWUgYW5kIGEgc3RyYWluIG9mIE15Y29wbGFzbWEgc3lub3ZpYWU7IEouIEJhY3RlcmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTg3OyAgY2l0YXRpb25fcGFnZXM9NTU2OC03NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGxldGUgZ2Vub21lIHNlcXVlbmNlIG9mIE15Y29wbGFzbWEgaHlvcG5ldW1vbmlhZSBzdHJhaW4gMTY4OyBKLiBCYWN0ZXJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTE5MzsgIGNpdGF0aW9uX3BhZ2VzPTEwMTYtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIG1pbmltYWwgZ2VuZSBjb21wbGVtZW50IG9mIE15Y29wbGFzbWEgZ2VuaXRhbGl1bTsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MTk5NTsgIGNpdGF0aW9uX3ZvbHVtZT0yNzA7ICBjaXRhdGlvbl9wYWdlcz0zOTctNDAzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgY29tcGxldGUgZ2Vub21lIHNlcXVlbmNlIG9mIEVzY2hlcmljaGlhIGNvbGkgSy0xMjsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT0yNzc7ICBjaXRhdGlvbl9wYWdlcz0xNDUzLTc0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgY29tcGxldGUgZ2Vub21lIHNlcXVlbmNlIG9mIHRoZSBncmFtLXBvc2l0aXZlIGJhY3Rlcml1bSBCYWNpbGx1cyBzdWJ0aWxpczsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTM5MDsgIGNpdGF0aW9uX3BhZ2VzPTI0OS01NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNjcmlwdGlvbiBpbiBNeWNvcGxhc21hIHBuZXVtb25pYWU7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0yODsgIGNpdGF0aW9uX3BhZ2VzPTQ0ODgtOTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNvbWUgcmVjb21tZW5kYXRpb25zIGNvbmNlcm5pbmcgcHJpbWFyeSBpc29sYXRpb24gb2YgTXljb3BsYXNtYSBzdWlwbmV1bW9uaWFlIGFuZCBNeWNvcGxhc21hIGZsb2NjdWxhcmUgYSBzdXJ2ZXk7IE5vcmQuIFZldC4gTWVkLjsgIGNpdGF0aW9uX3llYXI9MTk3NTsgIGNpdGF0aW9uX3ZvbHVtZT0yNzsgIGNpdGF0aW9uX3BhZ2VzPTMzNy05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9wdWJsaXNoZXI9Q29sZCBTcHJpbmcgSGFyYm9yIExhYm9yYXRvcnkgUHJlc3MsIENvbGQgU3ByaW5nIEhhcmJvciwgTlk7IE1vbGVjdWxhciBDbG9uaW5nOiBBIExhYm9yYXRvcnkgTWFudWFsOyAgY2l0YXRpb25feWVhcj0yMDAxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZW5zaXRpdmUgZGV0ZWN0aW9uIG9mIGJhY3RlcmlhbCB0cmFuc2NyaXB0aW9uIGluaXRpYXRpb24gc2l0ZXMgYW5kIGRpZmZlcmVudGlhdGlvbiBmcm9tIFJOQSBwcm9jZXNzaW5nIHNpdGVzIGluIHRoZSBwaGVyb21vbmUtaW5kdWNlZCBwbGFzbWlkIHRyYW5zZmVyIHN5c3RlbSBvZiBFbnRlcm9jb2NjdXMgZmFlY2FsaXM7IFByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTE5OTY7ICBjaXRhdGlvbl92b2x1bWU9OTM7ICBjaXRhdGlvbl9wYWdlcz03Nzk0LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNlcXVlbmNlIGxvZ29zOiBhIG5ldyB3YXkgdG8gZGlzcGxheSBjb25zZW5zdXMgc2VxdWVuY2VzOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5OTA7ICBjaXRhdGlvbl92b2x1bWU9MTg7ICBjaXRhdGlvbl9wYWdlcz02MDk3LTEwMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9V2ViTG9nbzogYSBzZXF1ZW5jZSBsb2dvIGdlbmVyYXRvcjsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz0xMTg4LTkwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qcm9tb3RlciBwcmVkaWN0aW9uIGluIHRoZSByaGl6b2JpYTsgTWljcm9iaW9sb2d5OyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTE1MjsgIGNpdGF0aW9uX3BhZ2VzPTE3NTEtNjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBpbGF0aW9uIGFuZCBhbmFseXNpcyBvZiBFc2NoZXJpY2hpYSBjb2xpIHByb21vdGVyIEROQSBzZXF1ZW5jZXM7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk4MzsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTIyMzctNTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBpbGF0aW9uIGFuZCBhbmFseXNpcyBvZiBCYWNpbGx1cyBzdWJ0aWxpcyBzaWdtYSBBLWRlcGVuZGVudCBwcm9tb3RlciBzZXF1ZW5jZXM6IGV2aWRlbmNlIGZvciBleHRlbmRlZCBjb250YWN0IGJldHdlZW4gUk5BIHBvbHltZXJhc2UgYW5kIHVwc3RyZWFtIHByb21vdGVyIEROQTsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTk1OyAgY2l0YXRpb25fdm9sdW1lPTIzOyAgY2l0YXRpb25fcGFnZXM9MjM1MS02MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lLXdpZGUgYW5hbHlzaXMgb2YgQ2hsYW15ZGlhZSBmb3IgcHJvbW90ZXJzIHRoYXQgcGh5bG9nZW5ldGljYWxseSBmb290cHJpbnQ7IFJlcy4gTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0xNTg7ICBjaXRhdGlvbl9wYWdlcz02ODUtOTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV4cGVyaW1lbnRhbCBhbmQgY29tcHV0YXRpb25hbCBhbmFseXNpcyBvZiB0cmFuc2NyaXB0aW9uYWwgc3RhcnQgc2l0ZXMgaW4gdGhlIGN5YW5vYmFjdGVyaXVtIFByb2NobG9yb2NvY2N1cyBNRUQ0OyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MzE7ICBjaXRhdGlvbl9wYWdlcz0yODkwLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZpY2F0aW9uIG9mIENhbXB5bG9iYWN0ZXIgamVqdW5pIHByb21vdGVyIHNlcXVlbmNlczsgSi4gQmFjdGVyaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT0xODA7ICBjaXRhdGlvbl9wYWdlcz01OTQtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VXNpbmcgUlNBVCBvbGlnby1hbmFseXNpcyBhbmQgZHlhZC1hbmFseXNpcyB0b29scyB0byBkaXNjb3ZlciByZWd1bGF0b3J5IHNpZ25hbHMgaW4gbnVjbGVpYyBzZXF1ZW5jZXM7IE5hdC4gUHJvdG9jLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0zOyAgY2l0YXRpb25fcGFnZXM9MTU4OS02MDM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlZ3VsYXRvcnkgc2VxdWVuY2UgYW5hbHlzaXMgdG9vbHM7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0zMTsgIGNpdGF0aW9uX3BhZ2VzPTM1OTMtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UlNBVDogcmVndWxhdG9yeSBzZXF1ZW5jZSBhbmFseXNpcyB0b29sczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTM2OyAgY2l0YXRpb25fcGFnZXM9VzExOS0yNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QmlvRWRpdDogYSB1c2VyLWZyaWVuZGx5IGJpb2xvZ2ljYWwgc2VxdWVuY2UgYWxpZ25tZW50IGVkaXRvciBhbmQgYW5hbHlzaXMgcHJvZ3JhbSBmb3IgV2luZG93cyA5NS85OC9OVDsgTnVjbGVpYyBBY2lkcyBTeW1wLiBTZXIuOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTQxOyAgY2l0YXRpb25fcGFnZXM9OTUtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Rml0dGluZyBhIG1peHR1cmUgbW9kZWwgYnkgZXhwZWN0YXRpb24gbWF4aW1pemF0aW9uIHRvIGRpc2NvdmVyIG1vdGlmcyBpbiBiaW9wb2x5bWVyczsgUHJvYy4gSW50LiBDb25mLiBJbnRlbGwuIFN5c3QuIE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTQ7ICBjaXRhdGlvbl92b2x1bWU9MjsgIGNpdGF0aW9uX3BhZ2VzPTI4LTM2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JZGVudGlmeWluZyBETkEgYW5kIHByb3RlaW4gcGF0dGVybnMgd2l0aCBzdGF0aXN0aWNhbGx5IHNpZ25pZmljYW50IGFsaWdubWVudHMgb2YgbXVsdGlwbGUgc2VxdWVuY2VzOyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTU2My03NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlb3JldGljYWwgYW5kIGVtcGlyaWNhbCBxdWFsaXR5IGFzc2Vzc21lbnQgb2YgdHJhbnNjcmlwdGlvbiBmYWN0b3ItYmluZGluZyBtb3RpZnM7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0zOTsgIGNpdGF0aW9uX3BhZ2VzPTgwOC0yNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VXNpbmcgUlNBVCB0byBzY2FuIGdlbm9tZSBzZXF1ZW5jZXMgZm9yIHRyYW5zY3JpcHRpb24gZmFjdG9yIGJpbmRpbmcgc2l0ZXMgYW5kIGNpcy1yZWd1bGF0b3J5IG1vZHVsZXM7IE5hdC4gUHJvdG9jLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0zOyAgY2l0YXRpb25fcGFnZXM9MTU3OC04ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIC0xNiByZWdpb24sIGEgdml0YWwgc2VxdWVuY2UgZm9yIHRoZSB1dGlsaXphdGlvbiBvZiBhIHByb21vdGVyIGluIEJhY2lsbHVzIHN1YnRpbGlzIGFuZCBFc2NoZXJpY2hpYSBjb2xpOyBNb2wuIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTU7ICBjaXRhdGlvbl92b2x1bWU9MTc7ICBjaXRhdGlvbl9wYWdlcz0yNzEtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIFRSVEduIG1vdGlmIHN0YWJpbGl6ZXMgdGhlIHRyYW5zY3JpcHRpb24gaW5pdGlhdGlvbiBvcGVuIGNvbXBsZXg7IEouIE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MzIyOyAgY2l0YXRpb25fcGFnZXM9NTIxLTMyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgc2lnbWE1NCByZWd1bG9uIChzaWdtdWxvbikgb2YgUHNldWRvbW9uYXMgcHV0aWRhOyBFbnZpcm9uLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz0xMjgxLTkzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZWd1bGF0aW9uIG9mIGdlbmUgZXhwcmVzc2lvbiBieSByZWl0ZXJhdGl2ZSB0cmFuc2NyaXB0aW9uOyBDdXJyLiBPcGluLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9MTQyLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNlbGVjdGlvbiBmb3IgdW5lcXVhbCBkZW5zaXRpZXMgb2Ygc2lnbWEgNzAgcHJvbW90ZXItbGlrZSBzaWduYWxzIGluIGRpZmZlcmVudCByZWdpb25zIG9mIGxhcmdlIGJhY3RlcmlhbCBnZW5vbWVzOyBQTG9TIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0yOyAgY2l0YXRpb25fcGFnZXM9ZTE4NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNpZ21hIEEgcmVjb2duaXRpb24gc2l0ZXMgaW4gdGhlIEJhY2lsbHVzIHN1YnRpbGlzIGdlbm9tZTsgTWljcm9iaW9sb2d5OyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTE0NzsgIGNpdGF0aW9uX3BhZ2VzPTI0MTctMjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZpY2F0aW9uIGFuZCBhbmFseXNpcyBvZiDigJhleHRlbmRlZCAtMTDigJkgcHJvbW90ZXJzIGluIEVzY2hlcmljaGlhIGNvbGk7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0zMTsgIGNpdGF0aW9uX3BhZ2VzPTQ2ODktOTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zY3JpcHRvbWUgY29tcGxleGl0eSBpbiBhIGdlbm9tZS1yZWR1Y2VkIGJhY3Rlcml1bTsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0zMjY7ICBjaXRhdGlvbl9wYWdlcz0xMjY4LTcxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1ScG9EIHByb21vdGVycyBpbiBDYW1weWxvYmFjdGVyIGplanVuaSBleGhpYml0IGEgc3Ryb25nIHBlcmlvZGljIHNpZ25hbCBpbnN0ZWFkIG9mIGEgLTM1IGJveDsgSi4gTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0zMjY7ICBjaXRhdGlvbl9wYWdlcz0xMzYxLTcyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2NyaXB0aW9uIGluaXRpYXRpb24gYnkgbWl4IGFuZCBtYXRjaCBlbGVtZW50czogZmxleGliaWxpdHkgZm9yIHBvbHltZXJhc2UgYmluZGluZyB0byBiYWN0ZXJpYWwgcHJvbW90ZXJzOyBHZW5lIFJlZ3VsLiBTeXN0LiBCaW87ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTsgIGNpdGF0aW9uX3BhZ2VzPTI3NS05MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJpYWQgcGF0dGVybiBhbGdvcml0aG0gZm9yIHByZWRpY3Rpbmcgc3Ryb25nIHByb21vdGVyIGNhbmRpZGF0ZXMgaW4gYmFjdGVyaWFsIGdlbm9tZXM7IEJNQyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9MjMzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW52ZXN0aWdhdGlvbnMgb2YgdGhlIG1vZHVsYXIgc3RydWN0dXJlIG9mIGJhY3RlcmlhbCBwcm9tb3RlcnM7IEJpb2NoZW0uIFNvYy4gU3ltcC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NzM7ICBjaXRhdGlvbl9wYWdlcz0xLTEwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QaGFnZSBUNCBlYXJseSBwcm9tb3RlcnMgYXJlIHJlc2lzdGFudCB0byBpbmhpYml0aW9uIGJ5IHRoZSBhbnRpLXNpZ21hIGZhY3RvciBBc2lBOyBNb2wuIE1pY3JvYmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT01MjsgIGNpdGF0aW9uX3BhZ2VzPTEwMTMtMjg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxhcmdlLXNjYWxlIGNvbXB1dGF0aW9uYWwgYW5kIHN0YXRpc3RpY2FsIGFuYWx5c2VzIG9mIGhpZ2ggdHJhbnNjcmlwdGlvbiBwb3RlbnRpYWxpdGllcyBpbiAzMiBwcm9rYXJ5b3RpYyBnZW5vbWVzOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MzY7ICBjaXRhdGlvbl9wYWdlcz0zMzMyLTQwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbiB2aXZvIGV4cHJlc3Npb24gYW5hbHlzaXMgb2YgdGhlIFA5NyBhbmQgUDEwMiBwYXJhbG9nIGZhbWlsaWVzIG9mIE15Y29wbGFzbWEgaHlvcG5ldW1vbmlhZTsgSW5mZWN0LiBJbW11bi47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9NzM7ICBjaXRhdGlvbl9wYWdlcz03Nzg0LTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldGVjdGlvbiBhbmQgcXVhbnRpZmljYXRpb24gb2YgaW50ZXJnZW5pYyB0cmFuc2NyaXB0aW9uIGluIE15Y29wbGFzbWEgaHlvcG5ldW1vbmlhZTsgTWljcm9iaW9sb2d5OyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTE1NjsgIGNpdGF0aW9uX3BhZ2VzPTIzMDUtMTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU15Y29wbGFzbWEgaHlvcG5ldW1vbmlhZSB0cmFuc2NyaXB0aW9uIHVuaXQgb3JnYW5pemF0aW9uOiBnZW5vbWUgc3VydmV5IGFuZCBwcmVkaWN0aW9uOyBETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xODsgIGNpdGF0aW9uX3BhZ2VzPTQxMy0yMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fcHVibGlzaGVyPUVsc2V2aWVyLCBOZXcgWW9yazsgUk5BIFBvbHltZXJhc2UgYW5kIHRoZSBSZWd1bGF0aW9uIG9mIFRyYW5zY3JpcHRpb247ICBjaXRhdGlvbl95ZWFyPTE5ODc7ICBjaXRhdGlvbl9wYWdlcz0xMDUtMTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBldGluZyBwcm9tb3RlcnMgaW4gcHJva2FyeW90aWMgdHJhbnNjcmlwdGlvbjsgVHJlbmRzIEJpb2NoZW0uIFNjaS47ICBjaXRhdGlvbl95ZWFyPTE5OTE7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz0zOTQtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHNpZ21hIDcwIHN1YnVuaXQgb2YgUk5BIHBvbHltZXJhc2UgaW5kdWNlcyBsYWNVVjUgcHJvbW90ZXItcHJveGltYWwgcGF1c2luZyBvZiB0cmFuc2NyaXB0aW9uOyBOYXQuIFN0cnVjdC4gTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTU1MS03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgc2lnbWEgNzAgc3VidW5pdCBvZiBSTkEgcG9seW1lcmFzZSBtZWRpYXRlcyBhIHByb21vdGVyLXByb3hpbWFsIHBhdXNlIGF0IHRoZSBsYWMgcHJvbW90ZXI7IE5hdC4gU3RydWN0LiBNb2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9NTQ0LTUwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2NyaXB0aW9uIHByb2ZpbGVzIG9mIHRoZSBiYWN0ZXJpdW0gTXljb3BsYXNtYSBwbmV1bW9uaWFlIGdyb3duIGF0IGRpZmZlcmVudCB0ZW1wZXJhdHVyZXM7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0zMTsgIGNpdGF0aW9uX3BhZ2VzPTYzMDYtMjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zY3JpcHRpb25hbCBwcm9maWxpbmcgb2YgTXljb3BsYXNtYSBoeW9wbmV1bW9uaWFlIGR1cmluZyBoZWF0IHNob2NrIHVzaW5nIG1pY3JvYXJyYXlzOyBJbmZlY3QuIEltbXVuLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT03NDsgIGNpdGF0aW9uX3BhZ2VzPTE2MC02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2NyaXB0aW9uYWwgcHJvZmlsaW5nIG9mIE15Y29wbGFzbWEgaHlvcG5ldW1vbmlhZSBkdXJpbmcgaXJvbiBkZXBsZXRpb24gdXNpbmcgbWljcm9hcnJheXM7IE1pY3JvYmlvbG9neTsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0xNTI7ICBjaXRhdGlvbl9wYWdlcz05MzctNDQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdsb2JhbCB0cmFuc2NyaXB0aW9uYWwgYW5hbHlzaXMgb2YgTXljb3BsYXNtYSBoeW9wbmV1bW9uaWFlIGZvbGxvd2luZyBleHBvc3VyZSB0byBoeWRyb2dlbiBwZXJveGlkZTsgTWljcm9iaW9sb2d5OyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTE1MzsgIGNpdGF0aW9uX3BhZ2VzPTM3ODUtOTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdsb2JhbCB0cmFuc2NyaXB0aW9uYWwgYW5hbHlzaXMgb2YgTXljb3BsYXNtYSBoeW9wbmV1bW9uaWFlIGZvbGxvd2luZyBleHBvc3VyZSB0byBub3JlcGluZXBocmluZTsgTWljcm9iaW9sb2d5OyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTE1NDsgIGNpdGF0aW9uX3BhZ2VzPTI1ODEtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzE5LzIvMTAzLzExNjE3MjEvZHNyMDQ1LnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBTZXZlcmFsIE15Y29wbGFzbWEgc3BlY2llcyBoYXZlIGhhZCB0aGVpciBnZW5vbWUgY29tcGxldGVseSBzZXF1ZW5jZWQsIGluY2x1ZGluZyBmb3VyIHN0cmFpbnMgb2YgdGhlIHN3aW5lIHBhdGhvZ2VuIE15Y29wbGFzbWEgaHlvcG5ldW1vbmlhZS4gTmV2ZXJ0aCIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXIvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9NDE4OTQxJnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZNR5iNLACkQE2tVecpw9mPdGtOMA" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f19%2f2%2f103%2f418941" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f19%2f2%2f103%2f418941" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=16354" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/19/2"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/19/2/1/m_cover.gif?Expires=2147483647&amp;Signature=EMTi8MiwskfL8whcuG7gPpMQVkoTywhsN9k2fcs1qMWyjLc9OmU3zzfeux1LcC8edMnaqIc6tYNqgsUJ8hCH0UQnGgbUVvjHa69ZDqc3rwPQg3T0QzS9XdepkqXV57OA92AfnyIdwEzEggsZ18zDGYx5fC~exu1TTRcnk2bBo6TdYsVgw52BF-63NakC5-nPvHQVSqNzjaFosams6g1DZ~UchQ1c7lmvgwn6cnAg013zUUb9XUUhijVu63SQWmhw0ByRocoE2U4pz7BLDSieaJZqPdVUaKjHeYkUlMHEhZhGv0eccckPuOqL2UhMZ0FzVdPiNB6hKa39ET8qYgw8~Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/19/2">
                        <div class="volume trailing-comma">Volume 19</div>
                        <div class="issue">Issue 2</div>

            </a>

        </div>
        <div class="ii-pub-date">
April 2012        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="6198927">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6198927">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6198929">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6198929">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6198935">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6198935">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6198955">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6198955">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6198994">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6198994">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6199010">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6199010">Supplementary Data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6199012">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6199012">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6199014">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6199014">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6199016">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6199016">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="empty arrow">&nbsp;</li>
            <li class="next arrow">
            <a href="/dnaresearch/article/19/2/117/421175">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Unveiling <em>Mycoplasma hyopneumoniae</em> Promoters: Sequence Definition and Genomic Distribution
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shana de Souto Weber</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Centro de Biotecnologia, Programa de Pós-graduação em Biologia Celular e Molecular</div>, <div class="insititution">Universidade Federal do Rio Grande do Sul (UFRGS)</div>, <div class="addr-line">Av. Bento Gonçalves 9500, C.P. 15005, CEP 91501-970, Porto Alegre, RS</div>, <div class="country">Brazil</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shana+de+Souto+Weber">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Weber S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Weber S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Shana de Souto Weber</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Fernando Hayashi Sant&#39;Anna</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Centro de Biotecnologia, Programa de Pós-graduação em Biologia Celular e Molecular</div>, <div class="insititution">Universidade Federal do Rio Grande do Sul (UFRGS)</div>, <div class="addr-line">Av. Bento Gonçalves 9500, C.P. 15005, CEP 91501-970, Porto Alegre, RS</div>, <div class="country">Brazil</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Fernando+Hayashi+Sant%27Anna">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sant&#39;Anna F">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sant&#39;Anna F&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Fernando Hayashi Sant'Anna</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Irene Silveira Schrank</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Centro de Biotecnologia, Programa de Pós-graduação em Biologia Celular e Molecular</div>, <div class="insititution">Universidade Federal do Rio Grande do Sul (UFRGS)</div>, <div class="addr-line">Av. Bento Gonçalves 9500, C.P. 15005, CEP 91501-970, Porto Alegre, RS</div>, <div class="country">Brazil</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">Departamento de Biologia Molecular e Biotecnologia - Centro de Biotecnologia</div>, <div class="insititution">Universidade Federal do Rio Grande do Sul (UFRGS)</div>, <div class="addr-line">Av. Bento Gonçalves 9500, C.P. 15005, CEP 91501-970, Porto Alegre, RS</div>, <div class="country">Brazil</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +55 51-33086055. Fax. +55 51-33087309. E-mail: <a href="mailto:irene@cbiot.ufrgs.br" target="_blank">irene@cbiot.ufrgs.br</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Irene+Silveira+Schrank">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Schrank I">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Schrank I&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Irene Silveira Schrank</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Edited by Katsumi Isono</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 19, Issue 2, 1 April 2012, Pages 103–115, <a href='https://doi.org/10.1093/dnares/dsr045'>https://doi.org/10.1093/dnares/dsr045</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">14 February 2012</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">24 July 2011</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">10 December 2011</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="418941" href="/dnaresearch/article-pdf/19/2/103/1161721/dsr045.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Shana de Souto Weber, Fernando Hayashi Sant'Anna, Irene Silveira Schrank;  Unveiling <em>Mycoplasma hyopneumoniae</em> Promoters: Sequence Definition and Genomic Distribution, <em>DNA Research</em>, Volume 19, Issue 2, 1 April 2012, Pages 103–115, <a href='https://doi.org/10.1093/dnares/dsr045'>https://doi.org/10.1093/dnares/dsr045</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/418941?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/418941?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/418941?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/418941?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/418941?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/418941?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/418941?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Unveiling%20%26lt%3Bem%26gt%3BMycoplasma%20hyopneumoniae%26lt%3B%2Fem%26gt%3B%20Promoters%3A%20Sequence%20Definition%20and%20Genomic%20Distribution&amp;publicationDate=2012-02-14&amp;volumeNum=19&amp;issueNum=2&amp;author=Weber%2C%20Shana%20de%20Souto%3B%20Sant&#39;Anna%2C%20Fernando%20Hayashi&amp;startPage=103&amp;endPage=115&amp;contentId=10.1093%2Fdnares%2Fdsr045&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f19%2f2%2f103%2f418941" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=6198927 id="6198927" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Several <em>Mycoplasma</em> species have had their genome completely sequenced, including four strains of the swine pathogen <em>Mycoplasma hyopneumoniae</em>. Nevertheless, little is known about the nucleotide sequences that control transcriptional initiation in these microorganisms. Therefore, with the objective of investigating the promoter sequences of <em>M. hyopneumoniae</em>, 23 transcriptional start sites (TSSs) of distinct genes were mapped. A pattern that resembles the σ<sup>70</sup> promoter −10 element was found upstream of the TSSs. However, no −35 element was distinguished. Instead, an AT-rich periodic signal was identified. About half of the experimentally defined promoters contained the motif 5′-TRTGn-3′, which was identical to the −16 element usually found in Gram-positive bacteria. The defined promoters were utilized to build position-specific scoring matrices in order to scan putative promoters upstream of all coding sequences (CDSs) in the <em>M. hyopneumoniae</em> genome. Two hundred and one signals were found associated with 169 CDSs. Most of these sequences were located within 100 nucleotides of the start codons. This study has shown that the number of promoter-like sequences in the <em>M. hyopneumoniae</em> genome is more frequent than expected by chance, indicating that most of the sequences detected are probably biologically functional.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="Mycoplasma"><em>Mycoplasma</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="promoter">promoter</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="transcription">transcription</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="sigma">sigma</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="matrix">matrix</a></div>                            <h2 scrollto-destination=6198929 id="6198929" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>The genus <em>Mycoplasma</em>, composed of bacteria that have no cell wall and have extremely reduced genomes, includes several species of medical or veterinary significance. <em>Mycoplasma hyopneumoniae</em> is an important swine pathogen, causing worldwide economic losses in the livestock industry.<sup><a reveal-id="DSR045C1" data-open="DSR045C1" class="link link-ref link-reveal xref-bibr">1</a></sup> In recent years, many <em>Mycoplasma</em> species have had their genomes completely sequenced, including four strains of <em>M. hyopneumoniae</em>.<sup><a reveal-id="DSR045C2  DSR045C4" data-open="DSR045C2  DSR045C4" class="link link-ref link-reveal xref-bibr">2–4</a></sup> Their genomes are ∼900 kb in length and contain ∼700 genes.</p><p>The analysis of genomic data shows that <em>Mycoplasma</em> genomes contain a small number of genes related to transcription. In the Clusters of Ortholog Groups (COG) classification, there are 20 genes implicated in this process in <em>M. hyopneumoniae</em> strain 7448, corresponding to ∼3% of the total coding sequences (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/sutils/coxik.cgi?gi=18652">http://www.ncbi.nlm.nih.gov/sutils/coxik.cgi?gi=18652</a>). Comparatively, 353 transcription-related genes are found in <em>Bacillus subtilis</em>, accounting for 7.4% of the total CDSs (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/sutils/coxik.cgi?gi=27">http://www.ncbi.nlm.nih.gov/sutils/coxik.cgi?gi=27</a>).</p><p>Like other <em>Mycoplasma</em> species, <em>M. hyopneumoniae</em> lacks many regulatory elements, including two-component systems and the transcription termination factor Rho.<sup><a reveal-id="DSR045C5" data-open="DSR045C5" class="link link-ref link-reveal xref-bibr">5</a></sup> Furthermore, only a single σ factor has been identified in all the <em>Mycoplasma</em> genomes analysed, while <em>Escherichia coli</em> has at least six σ factors<sup><a reveal-id="DSR045C6" data-open="DSR045C6" class="link link-ref link-reveal xref-bibr">6</a></sup> and <em>B. subtilis</em> has at least 18.<sup><a reveal-id="DSR045C7" data-open="DSR045C7" class="link link-ref link-reveal xref-bibr">7</a></sup> These observations suggest that mycoplasmas have transcriptional regulatory mechanisms that are unique among bacterial species.</p><p>The identification of promoter sequences is an important step towards understanding gene regulation; however, there are few studies about the nucleotide sequences that control transcriptional initiation in <em>Mycoplasma</em>. A fundamental study was published more than 10 years ago by Weiner <em>et al</em>.,<sup><a reveal-id="DSR045C8" data-open="DSR045C8" class="link link-ref link-reveal xref-bibr">8</a></sup> in which several putative <em>Mycoplasma pneumoniae</em> promoters were identified by primer extension coupled with analysis using <em>E. coli</em> σ<sup>70</sup> matrices. The defined sequences were used to derive an improved matrix for promoter prediction in this species.</p><p>In <em>M. hyopneumoniae</em>, very few promoters or transcriptional start sites (TSSs) have been determined. Therefore, with the goal of investigating <em>M. hyopneumoniae</em> promoters, 23 gene TSSs were mapped, and their adjacent upstream regions were examined for overrepresented sequences. The data gathered were then used to build species-specific position-specific scoring matrices (PSSMs), which were further evaluated in relation to their predictive performance. The best PSSM was utilized to scan for putative promoters upstream of all coding sequences of the <em>M. hyopneumoniae</em> genome.</p>                            <h2 scrollto-destination=6198935 id="6198935" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=6198936 id="6198936" class="section-title" data-legacy-id=s2a>2.1. Bacterial strains and culture conditions</h3>
<p><em>Mycoplasma hyopneumoniae</em> strain 7448 was cultured in 15-ml Falcon tubes containing 5 ml of Friis medium<sup><a reveal-id="DSR045C9" data-open="DSR045C9" class="link link-ref link-reveal xref-bibr">9</a></sup> at 37°C for ∼48 h with gentle agitation in a roller drum. <em>Escherichia coli</em> XL1-Blue was cultured at 37°C in Luria-Bertani (LB) medium, which was supplemented with 100 µg/ml of ampicillin when required. For blue/white colony selection, 40 µg/ml of X-gal (5-bromo-4-chloro-3-indolyl-β-<span class="small-caps">d</span>-galacto-pyranoside) and 0.3 mM isopropyl-β-<span class="small-caps">d</span>-thiogalactopyranoside were added to the LB agar.<sup><a reveal-id="DSR045C10" data-open="DSR045C10" class="link link-ref link-reveal xref-bibr">10</a></sup></p>                            <h3 scrollto-destination=6198938 id="6198938" class="section-title" data-legacy-id=s2b>2.2. DNA manipulations, oligonucleotides and sequence analysis</h3>
<p>DNA purifications from agarose gel bands were performed with the NucleoSpin<sup>®</sup> Extract II kit (Macherey-Nagel GmbH &amp; Co. KG, Düren, Germany) according to the manufacturer's instructions. The <em>Sma</em>I-digested plasmid pUC18 was utilized in cloning procedures. DNA ligation, transformation by electroporation, colony polymerase chain reaction (PCR), plasmid extraction and agarose gel electrophoresis were performed using standard methods.<sup><a reveal-id="DSR045C10" data-open="DSR045C10" class="link link-ref link-reveal xref-bibr">10</a></sup> The 5′ RACE adapter, the primers 5′ rapid amplification of cDNA ends (RACE) Outer and 5′ RACE Inner were provided in the First Choice RNA ligase-mediated (RLM)-RACE kit (Ambion, Inc., Austin, TX, USA). Gene-specific primers employed in the 5′ RLM-RACE analysis are listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>. The primers M13 forward and M13 reverse (Invitrogen™, Carlsbad, CA, USA) were utilized in the screening of clones and in the sequencing reactions. Sequencing was performed using the Dyenamic <em>et</em> dye terminator cycle sequencing kit (Healthcare, Waukesha, WI, USA) and a MegaBACE 1000 DNA Analysis System automated sequencer (Healthcare).</p>                            <h3 scrollto-destination=6198940 id="6198940" class="section-title" data-legacy-id=s2c>2.3. RNA isolation</h3>
<p>Total RNA was isolated from a 25 ml culture of <em>M. hyopneumoniae</em> strain 7448. Cells were harvested by centrifugation at 3360 × <em>g</em> for 15 min and resuspended in 1 ml of TRIzol (Invitrogen). The cell suspension was then processed according to the manufacturer's protocol. Subsequently, 50 µg of RNA was treated with RQ1 RNase-Free DNase (Promega Corporation, Madison, WI, USA), followed by purification and concentration with the NucleoSpin® RNA Clean-up XS kit (Macherey-Nagel GmbH &amp; Co. KG).</p>                            <h3 scrollto-destination=6198942 id="6198942" class="section-title" data-legacy-id=s2d>2.4. 5′ RLM-RACE</h3>
<p>To identify TSSs, the strategy described by Bensing <em>et al</em>.<sup><a reveal-id="DSR045C11" data-open="DSR045C11" class="link link-ref link-reveal xref-bibr">11</a></sup> was employed. This methodology was performed using the First Choice RLM-RACE kit (Ambion, Inc.) following the manufacturer's protocol, except that the calf intestinal phosphatase treatment was not carried out. Briefly, a 16 µl of reaction mixture containing 10 µg of DNA-free RNA, tobacco acid pyrophosphatase (TAP) buffer and 20 U RNase Inhibitor (Fermentas) was divided into two aliquots, one of which received 2 µl of TAP enzyme (TAP+ reaction) and the other an equal volume of water (TAP− reaction). After TAP treatment, both samples were processed identically in the 5′ RACE adapter ligation and reverse transcription steps. Once cDNA was obtained, three nested PCRs were carried out for each gene: TAP+, TAP− and the negative control. All the reactions were performed in a total volume of 25 µl containing 1.25 mM MgCl<sub>2,</sub> 1× Taq buffer, 0.02 mM of each deoxynucleotide triphosphate (dNTP), 1 U Taq DNA polymerase (Ludwig Biotec, Porto Alegre, Brazil), 10 pmol of the gene- and adaptor-specific primers and 0.5 µl of the template. The outer 5′ RLM-RACE PCR was done with cDNA as the template, the 5′ RACE outer primer and the gene-specific outer primer. The inner 5′ RLM-RACE PCR was done using an aliquot of the outer 5′ RLM-RACE PCR as the template, the 5′ RACE inner primer and the gene-specific inner primer. Amplifications were performed using the touchdown technique, and the products were analysed in 1.2–2% agarose gels. Differential DNA gel bands present in the TAP-treated samples (fragments derived from unprocessed RNA), but not in the TAP-untreated samples, were purified and cloned. Clones were screened by colony PCR for the presence of the insert and then sequenced.</p>                            <h3 scrollto-destination=6198944 id="6198944" class="section-title" data-legacy-id=s2e>2.5. Sequence logos</h3>
<p>Sequence logos were created using the WebLogo site (<a class="link link-uri" href="http://weblogo.berkeley.edu/)">http://weblogo.berkeley.edu/)</a>.<sup><a reveal-id="DSR045C12" data-open="DSR045C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DSR045C13" data-open="DSR045C13" class="link link-ref link-reveal xref-bibr">13</a></sup> Experimentally defined σ<sup>70</sup> promoter sequences of different bacteria were utilized, relying on the alignments proposed by the respective authors (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). The following numbers of promoter sequences were used to generate the logos: 25 sites of <em>Sinorhizobium meliloti</em>,<sup><a reveal-id="DSR045C14" data-open="DSR045C14" class="link link-ref link-reveal xref-bibr">14</a></sup> 59 sites of <em>E. coli</em>,<sup><a reveal-id="DSR045C15" data-open="DSR045C15" class="link link-ref link-reveal xref-bibr">15</a></sup> 142 sites of <em>B. subtilis</em>,<sup><a reveal-id="DSR045C16" data-open="DSR045C16" class="link link-ref link-reveal xref-bibr">16</a></sup> 41 sites of <em>Chlamydia trachomatis</em>,<sup><a reveal-id="DSR045C17" data-open="DSR045C17" class="link link-ref link-reveal xref-bibr">17</a></sup> 35 sites of <em>M. pneumoniae</em>,<sup><a reveal-id="DSR045C8" data-open="DSR045C8" class="link link-ref link-reveal xref-bibr">8</a></sup> 25 sites of <em>Prochlorococcus marinus</em>,<sup><a reveal-id="DSR045C18" data-open="DSR045C18" class="link link-ref link-reveal xref-bibr">18</a></sup> 21 sites of <em>Campylobacter jejuni</em><sup><a reveal-id="DSR045C19" data-open="DSR045C19" class="link link-ref link-reveal xref-bibr">19</a></sup> and 23 sites of <em>M. hyopneumoniae</em>. Genome size and G + C content were obtained from the genomes deposited in the National Center of Biotechnology Information database (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/">www.ncbi.nlm.nih.gov/</a>).</p>                            <h3 scrollto-destination=6198946 id="6198946" class="section-title" data-legacy-id=s2f>2.6. PSSMs construction</h3>
<p>The 5′ regions of the TSSs determined by RLM-RACE were examined for sequence patterns using the Local-Word-Analysis tool<sup><a reveal-id="DSR045C20" data-open="DSR045C20" class="link link-ref link-reveal xref-bibr">20</a></sup> from Regulatory Sequence Analysis Tools<sup><a reveal-id="DSR045C21" data-open="DSR045C21" class="link link-ref link-reveal xref-bibr">21</a>,<a reveal-id="DSR045C22" data-open="DSR045C22" class="link link-ref link-reveal xref-bibr">22</a></sup> (RSAT) (<a class="link link-uri" href="http://rsat.ulb.ac.be/">http://rsat.ulb.ac.be/</a>). The first 50 bases upstream of the TSSs were analysed, searching for motifs composed of six or four nucleotides, applying a window with a fixed width of 10 nts (for motifs of 6 nts) and a fixed width of 5 nts (for motifs of 4 nts), and a background model that considered all upstream regions of the <em>M. hyopneumoniae</em> strain 7448 genes, preventing overlap with upstream open reading frames (ORFs). Overrepresented motifs located four to eight bases upstream of the TSS were manually aligned with BioEdit 7.0,<sup><a reveal-id="DSR045C23" data-open="DSR045C23" class="link link-ref link-reveal xref-bibr">23</a></sup> and this alignment was used to build a weight matrix of 12 columns. In addition, other two matrices of 14 and 16 columns were derived using the matrix-building programs MEME<sup><a reveal-id="DSR045C24" data-open="DSR045C24" class="link link-ref link-reveal xref-bibr">24</a></sup> and Wconsensus<sup><a reveal-id="DSR045C25" data-open="DSR045C25" class="link link-ref link-reveal xref-bibr">25</a></sup> (<a class="link link-uri" href="http://ural.wustl.edu/consensus/">http://ural.wustl.edu/consensus/</a>), respectively. For building these matrices, 25 bases upstream of the TSSs were analysed with an undefined motif width and the Bernoulli model as the background. In order to mitigate the overfitting problem, the matrices were rebuilt eliminating repeated sites.</p>                            <h3 scrollto-destination=6198948 id="6198948" class="section-title" data-legacy-id=s2g>2.7. Data set</h3>
<p>All analyses were carried out with the sequences obtained from the complete genome of <em>M. hyopneumoniae</em> strain 7448, available at NCBI under the accession code NC_007332. The data sets used for both Matrix-Quality and Matrix-Scan procedures were extracted from all the <em>M. hyopneumoniae</em> protein-coding genes using the Retrieve Sequence tool from RSAT. The 657 extracted sequences consisted of up to 250 bases upstream (without overlap with the upstream open reading frame) and 50 bases downstream of the annotated start.</p>                            <h3 scrollto-destination=6198950 id="6198950" class="section-title" data-legacy-id=s2h>2.8. PSSMs performance evaluation</h3>
<p>The ability of each of the three PSSMs to discover functional binding sites in the data set sequences was evaluated using the Matrix-Quality<sup><a reveal-id="DSR045C26" data-open="DSR045C26" class="link link-ref link-reveal xref-bibr">26</a></sup> program from RSAT.</p><p>The following parameters were applied: one pseudo-count was used for correction of the matrix; pseudo-frequencies were set at 0.01. As background, Markov orders from 0 to 4 were tested using the whole set of upstream noncoding sequences of the <em>M. hyopneumoniae</em> strain 7448 genome. Comparative analyses of the normalized weight distribution (NWD) curves, obtained from Matrix-Quality, were carried out to decide which matrix and Markov order to use. The trade-off between the estimation of the false-positive rate (FPR) and the sensitivity of the matrix was assessed using receiver-operating characteristic (ROC) curves, containing a leave-one-out (LOO) evaluation of the positive set (sequences used to build the matrix). Finally, as an additional negative control, the empirical and theoretical distributions of the original matrix were compared with the average of 10 column-permuted PSSMs, which were obtained with the Permute-Matrix tool from RSAT.</p>                            <h3 scrollto-destination=6198953 id="6198953" class="section-title" data-legacy-id=s2i>2.9. Prediction of promoters</h3>
<p>The putative <em>M. hyopneumoniae</em> promoters were identified using the 12-column weight matrix on the sequence data set through the Matrix-Scan program.<sup><a reveal-id="DSR045C27" data-open="DSR045C27" class="link link-ref link-reveal xref-bibr">27</a></sup> The parameters were set as in Matrix-Quality, except that the Markov order was set at 1. The score threshold was determined by comparing the score distribution between the predicted promoters from the sequence data set (correct orientation) with those found in the reverse complement of the sequence data set (incorrect orientation). This analysis excluded intergenic sequences present between genes that are transcribed in divergent directions. The score that resulted in a considerable reduction in putative promoters in the incorrect orientation was selected as the threshold value.</p>                            <h2 scrollto-destination=6198955 id="6198955" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=6198956 id="6198956" class="section-title" data-legacy-id=s3a>3.1. Mapping of TSSs</h3>
<p>Initially, the genes for the study of <em>M. hyopneumoniae</em> promoters were chosen based on two criteria: (i) genes annotated as hypothetical were excluded, since it was not known whether they were transcribed and (ii) genes chosen had a divergent upstream gene, thus ensuring that they did not lie inside an operon, and that, consequently, there was a promoter immediately upstream of them. About a quarter of the 79 genes that met these criteria were selected (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). The mapping of the TSSs was performed using the 5′ RLM-RACE technique, which allows distinction between primary and processed transcripts on the basis of the phosphorylation state of their 5′ ends. In this process, based on the comparison of 5′ RLM-RACE products derived from RNA treated with TAP and from untreated RNA, it is possible to identify full-length transcripts, since TAP-treated samples include both primary and processed transcripts, while untreated samples include only the processed ones. Thus, the amplification products from TAP-treated RNA samples contained a specific or at least an enhanced signal from primary transcripts compared with untreated RNA samples. Amplification products derived from the 5′ ends of intact transcripts were cloned and sequenced (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>).</p><p>The analysis of 10 or more independent clones for each gene revealed that, in many cases, the 5′ end of the transcripts varied by a few nucleotides in length. In general, the longest sequence was the most common among the clones sequenced. One or two shorter sequences, differing by no more than six nucleotides, were also relatively frequent in eight genes (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>). These could represent alternative TSSs or could have originated from processed transcripts that were co-purified with the primary ones, since both are present in the TAP-treated samples and may have small length differences. Given the latter assumption, the 5′ nucleotide of the largest sequence of each gene was considered to be the TSS.</p><p>Five genes (<em>sips</em>, P97-like, <em>pgk</em>, <em>pyrH</em> and <em>ktrA</em>) had additional nucleotides at the 5′ end of their transcripts that were not expected from the genomic sequence (data not shown). The extra nucleotides consisted of one to six adenosines within a homopolymeric region composed of at least three adenosines. In these cases, the last 5′ templated nucleotide was considered to be the TSS.</p><p>Overall, the TSSs for 23 <em>M. hyopneumoniae</em> genes were identified (Table <a reveal-id="DSR045TB1" data-open="DSR045TB1" class="link link-reveal link-table xref-fig">1</a>). Four TSSs were found inside of their respective genes: 34 bp within <em>licA</em>, 14 bp within <em>gyrA</em> and 1 bp within MHP7448_0279 and <em>dam</em>. In these cases, the next in-frame start codon downstream of the TSS was assumed to be the true start codon. The distances between TSSs and the gene starts ranged from 143 bp in MHP7448_0360 to 1 bp in <em>ktrA</em>. The genes <em>rplJ</em> and MHP7448_0198 also had distant TSSs, 100 and 137 bp from their start codons, respectively, while the TSSs of <em>licA</em>, <em>glyA</em>, MHP7448_0279 and <em>leuS</em> were situated &lt;10 bp from their start codons. Further analysis found that 80% of the transcripts initiated with an adenosine residue. </p>                        <a id="6198961" scrollto-destination="6198961"></a>
<div content-id="DSR045TB1" class="table-modal"><div class="table-wrap"><div id="DSR045TB1" class="table-wrap-title" data-id="DSR045TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Experimentally defined promoter regions of <em>M. hyopneumoniae</em></p></div> </div><div class="table-overflow"><table><tbody><tr><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dsr045/2/m_dsr04508.gif?Expires=2147483647&Signature=SX9tp9EXXR9TUr0crN9jCpbhAbZd2xJEx~Rl8XbpZ16lvl91XKFLmasounMC6CDr0fXAgMxQbAKxFurAVsh0PaUDFRfvGb6p355Lk1Hvsq1JxLDDwVgmmthg6JIZilo7gA9HlSVKRthm3DsNX20MyGWTl4mTdAlln5kzRWd258uoXeRW3CRS1TyWkO4Fu-ya7Yj4kjAYwGtHKRyrvsVPoc7tnVaK6txS~Te7zRF93K2cNxfvKqI7y~PdtJWZemLJ7PiQj0f15Ya~ky0zfbd8KRP5C2Plu-uGOyA5lrImc4iugi5tFHHhakGTQYZn~NByvoHTI8442h4l4OJAcrQDVA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsr04508.gif" /> </td></tr></tbody></table></div><div content-id="DSR045TB1" class="table-modal"><table><tbody><tr><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dsr045/2/m_dsr04508.gif?Expires=2147483647&Signature=SX9tp9EXXR9TUr0crN9jCpbhAbZd2xJEx~Rl8XbpZ16lvl91XKFLmasounMC6CDr0fXAgMxQbAKxFurAVsh0PaUDFRfvGb6p355Lk1Hvsq1JxLDDwVgmmthg6JIZilo7gA9HlSVKRthm3DsNX20MyGWTl4mTdAlln5kzRWd258uoXeRW3CRS1TyWkO4Fu-ya7Yj4kjAYwGtHKRyrvsVPoc7tnVaK6txS~Te7zRF93K2cNxfvKqI7y~PdtJWZemLJ7PiQj0f15Ya~ky0zfbd8KRP5C2Plu-uGOyA5lrImc4iugi5tFHHhakGTQYZn~NByvoHTI8442h4l4OJAcrQDVA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsr04508.gif" /> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Black background, nucleotides that occur in more than 80% of the promoters; dark grey background, nucleotides that occur in more than 70% of the promoters; light grey background, guanines that occur in more than 40% of the promoters; dots, positions used in the construction of the different PSSMs.</p></div></div><div content-id=""><div class="fn"><p><sup>a</sup>Note that there was no obvious −35 element (TTGACA) in this region.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>Region where the −16 element was found.</p></div></div><div content-id=""><div class="fn"><p><sup>c</sup>TSSs are in bold.</p></div></div><div content-id=""><div class="fn"><p><sup>d</sup>Distance (b) between the TSS and the start codon.</p></div></div><div content-id=""><div class="fn"><p><sup>e</sup>Start codons.</p></div></div><div content-id=""><div class="fn"><p><sup>f</sup>The start codons of the genes <em>licA</em>, 0279, <em>gyrA</em> and <em>dam</em> were redefined, as their TSS were located within the original CDS annotation.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6198961" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR045TB1" class="table-wrap-title" data-id="DSR045TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Experimentally defined promoter regions of <em>M. hyopneumoniae</em></p></div> </div><div class="table-overflow"><table><tbody><tr><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dsr045/2/m_dsr04508.gif?Expires=2147483647&Signature=SX9tp9EXXR9TUr0crN9jCpbhAbZd2xJEx~Rl8XbpZ16lvl91XKFLmasounMC6CDr0fXAgMxQbAKxFurAVsh0PaUDFRfvGb6p355Lk1Hvsq1JxLDDwVgmmthg6JIZilo7gA9HlSVKRthm3DsNX20MyGWTl4mTdAlln5kzRWd258uoXeRW3CRS1TyWkO4Fu-ya7Yj4kjAYwGtHKRyrvsVPoc7tnVaK6txS~Te7zRF93K2cNxfvKqI7y~PdtJWZemLJ7PiQj0f15Ya~ky0zfbd8KRP5C2Plu-uGOyA5lrImc4iugi5tFHHhakGTQYZn~NByvoHTI8442h4l4OJAcrQDVA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsr04508.gif" /> </td></tr></tbody></table></div><div content-id="DSR045TB1" class="table-modal"><table><tbody><tr><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dsr045/2/m_dsr04508.gif?Expires=2147483647&Signature=SX9tp9EXXR9TUr0crN9jCpbhAbZd2xJEx~Rl8XbpZ16lvl91XKFLmasounMC6CDr0fXAgMxQbAKxFurAVsh0PaUDFRfvGb6p355Lk1Hvsq1JxLDDwVgmmthg6JIZilo7gA9HlSVKRthm3DsNX20MyGWTl4mTdAlln5kzRWd258uoXeRW3CRS1TyWkO4Fu-ya7Yj4kjAYwGtHKRyrvsVPoc7tnVaK6txS~Te7zRF93K2cNxfvKqI7y~PdtJWZemLJ7PiQj0f15Ya~ky0zfbd8KRP5C2Plu-uGOyA5lrImc4iugi5tFHHhakGTQYZn~NByvoHTI8442h4l4OJAcrQDVA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsr04508.gif" /> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Black background, nucleotides that occur in more than 80% of the promoters; dark grey background, nucleotides that occur in more than 70% of the promoters; light grey background, guanines that occur in more than 40% of the promoters; dots, positions used in the construction of the different PSSMs.</p></div></div><div content-id=""><div class="fn"><p><sup>a</sup>Note that there was no obvious −35 element (TTGACA) in this region.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>Region where the −16 element was found.</p></div></div><div content-id=""><div class="fn"><p><sup>c</sup>TSSs are in bold.</p></div></div><div content-id=""><div class="fn"><p><sup>d</sup>Distance (b) between the TSS and the start codon.</p></div></div><div content-id=""><div class="fn"><p><sup>e</sup>Start codons.</p></div></div><div content-id=""><div class="fn"><p><sup>f</sup>The start codons of the genes <em>licA</em>, 0279, <em>gyrA</em> and <em>dam</em> were redefined, as their TSS were located within the original CDS annotation.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6198961" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=6198962 id="6198962" class="section-title" data-legacy-id=s3b>3.2. Identification of promoter elements</h3>
<p>The 23 experimentally determined TSSs were aligned and the sequences immediately 5′ to them were examined for nucleotide patterns that could comprise promoter elements. The occurrence of locally overrepresented sequences was detected using the Local-Word-Analysis tool. When looking for motifs of six nucleotides, 21 of the 23 genes had the patterns TATAAT or TAAAAT within 5–8 nts of the TSS (Table <a reveal-id="DSR045TB1" data-open="DSR045TB1" class="link link-reveal link-table xref-fig">1</a>). Additional variants were found in the remaining two genes with multiple em for motif elicitation (MEME) and Wconsensus, which recognized the motifs AAAAAT and TACAAT in the <em>recA</em> and <em>ktrA</em> genes, respectively (Table <a reveal-id="DSR045TB1" data-open="DSR045TB1" class="link link-reveal link-table xref-fig">1</a>). Four nucleotide positions of these hexamers were invariant. However, thymidine was the first base in 22 (96%) and the third base in 16 (70%) of them. Therefore, the consensus sequence was TATAAT, which is identical to the canonical σ<sup>70</sup> promoter −10 element.<sup><a reveal-id="DSR045C15" data-open="DSR045C15" class="link link-ref link-reveal xref-bibr">15</a></sup></p><p>The alignment of the sequences using the −10 hexamers revealed additional conserved elements. The base immediately 3′ of the −10 hexamer was thymine in 73% of the sequences (Table <a reveal-id="DSR045TB1" data-open="DSR045TB1" class="link link-reveal link-table xref-fig">1</a>). Moreover, there was considerable conservation in the bases upstream of the −10 element. The Local-Word-Analysis software found the pattern TATG in eight of the genes, one nucleotide upstream of the −10 element (Table <a reveal-id="DSR045TB1" data-open="DSR045TB1" class="link link-reveal link-table xref-fig">1</a>). This motif matches the consensus 5′-TRTGn-3′, an extended −10 region commonly found in Gram-positive bacteria that is also known as the −16 element.<sup><a reveal-id="DSR045C28" data-open="DSR045C28" class="link link-ref link-reveal xref-bibr">28</a>,<a reveal-id="DSR045C29" data-open="DSR045C29" class="link link-ref link-reveal xref-bibr">29</a></sup> In addition, the dinucleotide TG (the major determinant of −10 extended elements) was found one base upstream of the −10 hexamer in another three genes, so 11 (48%) promoter sequences contained a probable extended −10 element.</p><p>While it was possible identify the putative −10 and −16 elements, no conserved pattern corresponding to a −35 element was found (Table <a reveal-id="DSR045TB1" data-open="DSR045TB1" class="link link-reveal link-table xref-fig">1</a>). Instead, a periodic AT-rich sequence was seen when a sequence logo was created (Fig. <a reveal-id="DSR045F1" data-open="DSR045F1" class="link link-reveal link-table xref-fig">1</a>). </p>                        <a id="6198966" scrollto-destination="6198966"></a>
<div id="DSR045F1" data-id="DSR045F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dsr045/2/m_dsr04501.jpeg?Expires=2147483647&Signature=DLEvbJJDxTIU9GE4-2Fo-ZS8o8lftebka9nJ6Y6phLJXB1l1y1-oNKDOIoHnSPqVSXp3IC~n~Gcyu9h3evzEhIz9H4MFDsiJVJXMVhds49Q-yMAXk7~tWEaxPHLD0U0TjADP1pwsTFI5vvDFKhz1F6mI5csuwNlyCvXvt1fJtBgskTB0C-zi~9yDa2z0lT6OC-W7dtIuL3fsI1~nrnr9U9ZXIwpqSv277JS~vraE2TG0dg9JXEJOGH20U8CuQdo~5E9z1OLt-nZX5NElHrJMhfFr3MalSTaQ3R5A8kZPuXUUKR83tjuX3ijjco4jk2x~GhkTwqSfw5Q2LNMgeNAxBA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Sequence conservation in the M. hyopneumoniae promoter region. Sequence logo derived from the alignment of the 23 defined promoter regions showing the high conservation of the −10 element (positions 59–64), the presence of a semi-conserved −16 element (positions 54–57), the absence of a −35 element and the distinct periodic AT-rich signal extending upstream of the −10 element. The region extending between positions 54 and 65 was used to construct the 12-column PSSM. The vertical axis shows information content in bits. The overall height of the stack indicates the sequence conservation at that position, whereas the height of the nucleotide within the stack indicates its relative frequency at that position." path-from-xml="dsr04501.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6198966/dsr04501.jpeg" path-from-xml="dsr04501.jpeg" target="_blank">View large</a><a section="6198966" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dsr045/2/dsr04501.jpeg?Expires=2147483647&Signature=x593RfgU-PlaKtvIYj8SQMx7YCkQpxIQwbfu6L1ErLVRBxJWS3Z-i3sP4mkQEeEr7m5z9EkE~9sDiF5zuq6i3oumDMcvdGFDkpU~pdy8JuMMqJgLruiDffLVXEF~JaW9sM0e9aq09Du1M28t1icCK-dnOOOJRLcHO8dS6Cb18Ibldzmmw-JZ7fWSDxtT-NvzzhlP6d9fCrPyk9axU-wUTo12QEw4HF0LhaCQtM2rTdbYKNl3b7g18TFrCifoCOkWRQFhryqpm1lA09eAiZmO-x9c4vicYKEiHWBmkfxa-0rhx18ECgch6ezB3NJk6MV5ZwXM5x5SAltrHenj6HReEw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6198966&ar=418941&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr04501.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Sequence conservation in the <em>M. hyopneumoniae</em> promoter region. Sequence logo derived from the alignment of the 23 defined promoter regions showing the high conservation of the −10 element (positions 59–64), the presence of a semi-conserved −16 element (positions 54–57), the absence of a −35 element and the distinct periodic AT-rich signal extending upstream of the −10 element. The region extending between positions 54 and 65 was used to construct the 12-column PSSM. The vertical axis shows information content in bits. The overall height of the stack indicates the sequence conservation at that position, whereas the height of the nucleotide within the stack indicates its relative frequency at that position.</p></div></div><div content-id="DSR045F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dsr045/2/m_dsr04501.jpeg?Expires=2147483647&Signature=DLEvbJJDxTIU9GE4-2Fo-ZS8o8lftebka9nJ6Y6phLJXB1l1y1-oNKDOIoHnSPqVSXp3IC~n~Gcyu9h3evzEhIz9H4MFDsiJVJXMVhds49Q-yMAXk7~tWEaxPHLD0U0TjADP1pwsTFI5vvDFKhz1F6mI5csuwNlyCvXvt1fJtBgskTB0C-zi~9yDa2z0lT6OC-W7dtIuL3fsI1~nrnr9U9ZXIwpqSv277JS~vraE2TG0dg9JXEJOGH20U8CuQdo~5E9z1OLt-nZX5NElHrJMhfFr3MalSTaQ3R5A8kZPuXUUKR83tjuX3ijjco4jk2x~GhkTwqSfw5Q2LNMgeNAxBA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Sequence conservation in the M. hyopneumoniae promoter region. Sequence logo derived from the alignment of the 23 defined promoter regions showing the high conservation of the −10 element (positions 59–64), the presence of a semi-conserved −16 element (positions 54–57), the absence of a −35 element and the distinct periodic AT-rich signal extending upstream of the −10 element. The region extending between positions 54 and 65 was used to construct the 12-column PSSM. The vertical axis shows information content in bits. The overall height of the stack indicates the sequence conservation at that position, whereas the height of the nucleotide within the stack indicates its relative frequency at that position." path-from-xml="dsr04501.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6198966/dsr04501.jpeg" path-from-xml="dsr04501.jpeg" target="_blank">View large</a><a section="6198966" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dsr045/2/dsr04501.jpeg?Expires=2147483647&Signature=x593RfgU-PlaKtvIYj8SQMx7YCkQpxIQwbfu6L1ErLVRBxJWS3Z-i3sP4mkQEeEr7m5z9EkE~9sDiF5zuq6i3oumDMcvdGFDkpU~pdy8JuMMqJgLruiDffLVXEF~JaW9sM0e9aq09Du1M28t1icCK-dnOOOJRLcHO8dS6Cb18Ibldzmmw-JZ7fWSDxtT-NvzzhlP6d9fCrPyk9axU-wUTo12QEw4HF0LhaCQtM2rTdbYKNl3b7g18TFrCifoCOkWRQFhryqpm1lA09eAiZmO-x9c4vicYKEiHWBmkfxa-0rhx18ECgch6ezB3NJk6MV5ZwXM5x5SAltrHenj6HReEw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6198966&ar=418941&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr04501.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Sequence conservation in the <em>M. hyopneumoniae</em> promoter region. Sequence logo derived from the alignment of the 23 defined promoter regions showing the high conservation of the −10 element (positions 59–64), the presence of a semi-conserved −16 element (positions 54–57), the absence of a −35 element and the distinct periodic AT-rich signal extending upstream of the −10 element. The region extending between positions 54 and 65 was used to construct the 12-column PSSM. The vertical axis shows information content in bits. The overall height of the stack indicates the sequence conservation at that position, whereas the height of the nucleotide within the stack indicates its relative frequency at that position.</p></div></div>                            <h3 scrollto-destination=6198967 id="6198967" class="section-title" data-legacy-id=s3c>3.3. Comparison with other σ<sup>70</sup> bacterial consensus sequences</h3>
<p><em>Mycoplasma hyopneumoniae</em> promoter sequences were compared with other σ<sup>70</sup> promoters from different microorganisms. The alignments of experimentally identified sequences were retrieved to create logos, which visually represent sequence conservation.</p><p>The results presented in Fig. <a reveal-id="DSR045F2" data-open="DSR045F2" class="link link-reveal link-table xref-fig">2</a> suggest that the occurrence of −35 elements in the σ70 promoter is related to the G + C content of the organism. The promoters of the species <em>S. meliloti</em>, <em>E. coli</em>, <em>B. subtilis</em>, <em>C. trachomatis</em> and <em>M. pneumoniae</em>, which have a genomic G + C content ≥40%, have the trinucleotide TTG of the −35 element, whereas the promoters of <em>P. marinus</em>, <em>C. jejuni</em> and <em>M. hyopneumoniae</em>, which have a genomic G + C content ≤30.8%, do not have this conserved trinucleotide. </p>                        <a id="6198970" scrollto-destination="6198970"></a>
<div id="DSR045F2" data-id="DSR045F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dsr045/2/m_dsr04502.jpeg?Expires=2147483647&Signature=5MmItJHHIIXAxka~3UNnO0DSI25Ow52jVdXJKD2xRlyzTRI4zKG33wQhyLu2bCmo9Dh0lfavQ3hEh~e72Xhz0ZqhIiuV~jowdHn42zsbxObXNrgEeX0gZ-ZjLo9p2DSq045T43YtRHIyxu3PWxZEqLn39T0A8pvSxZLm5tGIUp7q1dqbGd0yT2z0DVGbqx~qV6F~RONjwHml-xtJ7-sBtSACG8zU2p9RHGLJGk0~i1LMiieN97PKDOt2EeZPTNXXFjJVLyYMOfSKKSFM-R2FmpI~pm~1pE6thYJjhqblSXt1y0q8FNyoG3xxAtodhKjPptXzFP5O3mCApl4nMTZwJA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="σ70-like recognition sites in different bacterial species. Sequence logos showing the loss of conservation of the −35 signal as the genomic G + C content decreases. The following numbers of promoter sequences were used to generate the logos: 25 sites of S. meliloti, 59 sites of E. coli, 142 sites of B. subtilis, 41 sites of C. trachomatis, 35 sites of M. pneumoniae, 25 sites of P. marinus, 21 sites of C. jejuni and 23 sites of M. hyopneumoniae. The vertical axis shows information content in bits. The overall height of the stack indicates the sequence conservation at that position, whereas the height of the nucleotide within the stack indicates its relative frequency at that position." path-from-xml="dsr04502.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6198970/dsr04502.jpeg" path-from-xml="dsr04502.jpeg" target="_blank">View large</a><a section="6198970" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dsr045/2/dsr04502.jpeg?Expires=2147483647&Signature=jCWDFV2l0PSueAkaCIrNAx5e5o0SuyY2xrW3nrUpthIcUxGl9-MSxNyB0Hh50Qap4~UhHwWx9KGz-qjlkS3LbdYh~JA~RG22P2CfL2DUvwymPftH6OhY8DPk~bYH6ccxNTltVaagcg3pkT0rb6poZN6zDFLjyU4--S53GR6mxWZxwkZ9OW0YFJiq7lbr9AYi5xE9Myp900gaDfQJS-tGJxXlqyNv2OPwWZxc~XDWm1uIlbB7N~gztH6E07KYHFK09y5Bu0SjRS0QeJ2kT9mchgSAKqbYIWNhRdxmoB6PJXpSR-8i~6K9fS~ZJwP18Tb3B0iRJs1qCK3~3z~9rU5C7w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6198970&ar=418941&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr04502.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>σ<sup>70</sup>-like recognition sites in different bacterial species. Sequence logos showing the loss of conservation of the −35 signal as the genomic G + C content decreases. The following numbers of promoter sequences were used to generate the logos: 25 sites of <em>S. meliloti</em>, 59 sites of <em>E. coli</em>, 142 sites of <em>B. subtilis</em>, 41 sites of <em>C. trachomatis</em>, 35 sites of <em>M. pneumoniae</em>, 25 sites of <em>P. marinus</em>, 21 sites of <em>C. jejuni</em> and 23 sites of <em>M. hyopneumoniae</em>. The vertical axis shows information content in bits. The overall height of the stack indicates the sequence conservation at that position, whereas the height of the nucleotide within the stack indicates its relative frequency at that position.</p></div></div><div content-id="DSR045F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dsr045/2/m_dsr04502.jpeg?Expires=2147483647&Signature=5MmItJHHIIXAxka~3UNnO0DSI25Ow52jVdXJKD2xRlyzTRI4zKG33wQhyLu2bCmo9Dh0lfavQ3hEh~e72Xhz0ZqhIiuV~jowdHn42zsbxObXNrgEeX0gZ-ZjLo9p2DSq045T43YtRHIyxu3PWxZEqLn39T0A8pvSxZLm5tGIUp7q1dqbGd0yT2z0DVGbqx~qV6F~RONjwHml-xtJ7-sBtSACG8zU2p9RHGLJGk0~i1LMiieN97PKDOt2EeZPTNXXFjJVLyYMOfSKKSFM-R2FmpI~pm~1pE6thYJjhqblSXt1y0q8FNyoG3xxAtodhKjPptXzFP5O3mCApl4nMTZwJA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="σ70-like recognition sites in different bacterial species. Sequence logos showing the loss of conservation of the −35 signal as the genomic G + C content decreases. The following numbers of promoter sequences were used to generate the logos: 25 sites of S. meliloti, 59 sites of E. coli, 142 sites of B. subtilis, 41 sites of C. trachomatis, 35 sites of M. pneumoniae, 25 sites of P. marinus, 21 sites of C. jejuni and 23 sites of M. hyopneumoniae. The vertical axis shows information content in bits. The overall height of the stack indicates the sequence conservation at that position, whereas the height of the nucleotide within the stack indicates its relative frequency at that position." path-from-xml="dsr04502.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6198970/dsr04502.jpeg" path-from-xml="dsr04502.jpeg" target="_blank">View large</a><a section="6198970" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dsr045/2/dsr04502.jpeg?Expires=2147483647&Signature=jCWDFV2l0PSueAkaCIrNAx5e5o0SuyY2xrW3nrUpthIcUxGl9-MSxNyB0Hh50Qap4~UhHwWx9KGz-qjlkS3LbdYh~JA~RG22P2CfL2DUvwymPftH6OhY8DPk~bYH6ccxNTltVaagcg3pkT0rb6poZN6zDFLjyU4--S53GR6mxWZxwkZ9OW0YFJiq7lbr9AYi5xE9Myp900gaDfQJS-tGJxXlqyNv2OPwWZxc~XDWm1uIlbB7N~gztH6E07KYHFK09y5Bu0SjRS0QeJ2kT9mchgSAKqbYIWNhRdxmoB6PJXpSR-8i~6K9fS~ZJwP18Tb3B0iRJs1qCK3~3z~9rU5C7w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6198970&ar=418941&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr04502.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>σ<sup>70</sup>-like recognition sites in different bacterial species. Sequence logos showing the loss of conservation of the −35 signal as the genomic G + C content decreases. The following numbers of promoter sequences were used to generate the logos: 25 sites of <em>S. meliloti</em>, 59 sites of <em>E. coli</em>, 142 sites of <em>B. subtilis</em>, 41 sites of <em>C. trachomatis</em>, 35 sites of <em>M. pneumoniae</em>, 25 sites of <em>P. marinus</em>, 21 sites of <em>C. jejuni</em> and 23 sites of <em>M. hyopneumoniae</em>. The vertical axis shows information content in bits. The overall height of the stack indicates the sequence conservation at that position, whereas the height of the nucleotide within the stack indicates its relative frequency at that position.</p></div></div><p>Comparison of these sequence logos also indicated that the −10 element is more conserved in <em>M. hyopneumoniae</em> than in the other bacterial species. One noteworthy observation was that this element was preceded by the dinucleotide TG, a feature that is shared with <em>B. subtilis</em> and <em>C. jejuni</em>, indicating the existence of a −16 element. Another distinct characteristic found was the presence of periodic AT-rich sequences upstream of the −10 elements of <em>M. hyopneumoniae</em> and <em>C. jejuni</em>.</p>                            <h3 scrollto-destination=6198972 id="6198972" class="section-title" data-legacy-id=s3d>3.4. Construction of a PSSM for prediction of <em>M. hyopneumoniae</em> promoters</h3>
<p>Manual alignment of the 23 defined <em>M. hyopneumoniae</em> promoters was used to create a PSSM of 12 columns (Tables <a reveal-id="DSR045TB1" data-open="DSR045TB1" class="link link-reveal link-table xref-fig">1</a> and <a reveal-id="DSR045TB2" data-open="DSR045TB2" class="link link-reveal link-table xref-fig">2</a>). In order to validate whether this alignment and the positions included in the matrix were appropriate, two other matrices were independently constructed using MEME and the Wconsensus. Both programs included the same 12 positions used in the initial matrix to build their matrices. However, these latter matrices included a few more positions, generating PSSMs of 14 and 16 columns (Table <a reveal-id="DSR045TB1" data-open="DSR045TB1" class="link link-reveal link-table xref-fig">1</a>). </p>                        <a id="6198974" scrollto-destination="6198974"></a>
<div content-id="DSR045TB2" class="table-modal"><div class="table-wrap"><div id="DSR045TB2" class="table-wrap-title" data-id="DSR045TB2"><div class="label -label">Table 2.</div><div class="caption"><p>PSSM based on experimentally determined <em>M. hyopneumoniae</em> promoters</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>−17 </th><th>−16 </th><th>−15 </th><th>−14 </th><th>−13 </th><th>−12 </th><th>−11 </th><th>−10 </th><th>−9 </th><th>−8 </th><th>−7 </th><th>−6 </th></tr></thead><tbody><tr><td>A </td><td>2 </td><td>18 </td><td>3 </td><td>6 </td><td>5 </td><td>1 </td><td>23 </td><td>5 </td><td>23 </td><td>22 </td><td>0 </td><td>5 </td></tr><tr><td>C </td><td>0 </td><td>2 </td><td>0 </td><td>0 </td><td>3 </td><td>0 </td><td>0 </td><td>1 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td></tr><tr><td>G </td><td>1 </td><td>1 </td><td>5 </td><td>11 </td><td>10 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>1 </td></tr><tr><td>T </td><td>20 </td><td>2 </td><td>15 </td><td>6 </td><td>5 </td><td>22 </td><td>0 </td><td>17 </td><td>0 </td><td>1 </td><td>23 </td><td>17 </td></tr></tbody></table></div><div content-id="DSR045TB2" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>−17 </th><th>−16 </th><th>−15 </th><th>−14 </th><th>−13 </th><th>−12 </th><th>−11 </th><th>−10 </th><th>−9 </th><th>−8 </th><th>−7 </th><th>−6 </th></tr></thead><tbody><tr><td>A </td><td>2 </td><td>18 </td><td>3 </td><td>6 </td><td>5 </td><td>1 </td><td>23 </td><td>5 </td><td>23 </td><td>22 </td><td>0 </td><td>5 </td></tr><tr><td>C </td><td>0 </td><td>2 </td><td>0 </td><td>0 </td><td>3 </td><td>0 </td><td>0 </td><td>1 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td></tr><tr><td>G </td><td>1 </td><td>1 </td><td>5 </td><td>11 </td><td>10 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>1 </td></tr><tr><td>T </td><td>20 </td><td>2 </td><td>15 </td><td>6 </td><td>5 </td><td>22 </td><td>0 </td><td>17 </td><td>0 </td><td>1 </td><td>23 </td><td>17 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6198974" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR045TB2" class="table-wrap-title" data-id="DSR045TB2"><div class="label -label">Table 2.</div><div class="caption"><p>PSSM based on experimentally determined <em>M. hyopneumoniae</em> promoters</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>−17 </th><th>−16 </th><th>−15 </th><th>−14 </th><th>−13 </th><th>−12 </th><th>−11 </th><th>−10 </th><th>−9 </th><th>−8 </th><th>−7 </th><th>−6 </th></tr></thead><tbody><tr><td>A </td><td>2 </td><td>18 </td><td>3 </td><td>6 </td><td>5 </td><td>1 </td><td>23 </td><td>5 </td><td>23 </td><td>22 </td><td>0 </td><td>5 </td></tr><tr><td>C </td><td>0 </td><td>2 </td><td>0 </td><td>0 </td><td>3 </td><td>0 </td><td>0 </td><td>1 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td></tr><tr><td>G </td><td>1 </td><td>1 </td><td>5 </td><td>11 </td><td>10 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>1 </td></tr><tr><td>T </td><td>20 </td><td>2 </td><td>15 </td><td>6 </td><td>5 </td><td>22 </td><td>0 </td><td>17 </td><td>0 </td><td>1 </td><td>23 </td><td>17 </td></tr></tbody></table></div><div content-id="DSR045TB2" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>−17 </th><th>−16 </th><th>−15 </th><th>−14 </th><th>−13 </th><th>−12 </th><th>−11 </th><th>−10 </th><th>−9 </th><th>−8 </th><th>−7 </th><th>−6 </th></tr></thead><tbody><tr><td>A </td><td>2 </td><td>18 </td><td>3 </td><td>6 </td><td>5 </td><td>1 </td><td>23 </td><td>5 </td><td>23 </td><td>22 </td><td>0 </td><td>5 </td></tr><tr><td>C </td><td>0 </td><td>2 </td><td>0 </td><td>0 </td><td>3 </td><td>0 </td><td>0 </td><td>1 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td></tr><tr><td>G </td><td>1 </td><td>1 </td><td>5 </td><td>11 </td><td>10 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>1 </td></tr><tr><td>T </td><td>20 </td><td>2 </td><td>15 </td><td>6 </td><td>5 </td><td>22 </td><td>0 </td><td>17 </td><td>0 </td><td>1 </td><td>23 </td><td>17 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6198974" target="_blank">View Large</a></div></div><p>Once the three PSSMs were obtained, they were rebuilt, excluding the repeated sites, with the aim of minimizing the problem of overfitting, and then their predictive capacity was assessed and compared in order to choose the best one. Matrix-Quality was used to perform this evaluation. This program relies on a combined analysis of theoretical and empirical score distributions to estimate the capability of a PSSM to distinguish putative binding sites from the genomic background.<sup><a reveal-id="DSR045C26" data-open="DSR045C26" class="link link-ref link-reveal xref-bibr">26</a></sup> The theoretical distribution encompasses the matrix scores along a random sequence of infinite length generated using the background model. This indicates the probability of a site scoring above a given weight score by chance, and thus provides an estimate of the FPR.<sup><a reveal-id="DSR045C26" data-open="DSR045C26" class="link link-ref link-reveal xref-bibr">26</a></sup> The empirical distribution contains the matrix scores obtained along the sequences of interest (e.g. upstream noncoding sequences), which are composed predominantly of nonbinding sites, interspersed with a few biologically functional sites.<sup><a reveal-id="DSR045C26" data-open="DSR045C26" class="link link-ref link-reveal xref-bibr">26</a></sup> Both distributions were calculated using the three PSSMs. For the empirical distribution, the sequence set comprised up to 250 bases upstream and 50 bases downstream of the start codon from all <em>M. hyopneumoniae</em> protein-coding genes (downstream bases were also scanned because some TSSs were found within genes). As a background model, the whole set of the upstream noncoding sequences of the <em>M. hyopneumoniae</em> genome was used, testing different Markov orders (0–4), since this affects the weight score computation and, consequently, the performance of the matrices. The discriminatory capability of each matrix coupled with each Markov order was assessed by comparison of the empirical and theoretical score distributions.</p><p>The difference between the two distributions indicates the discriminative power of the matrix, which can be expressed by computation of the NWD curves.<sup><a reveal-id="DSR045C26" data-open="DSR045C26" class="link link-ref link-reveal xref-bibr">26</a></sup> In this analysis, the weight score difference (WD) between the weight scores observed in empirical and theoretical distributions is calculated at each frequency value. As larger matrices allow higher scores, the WD is divided by the number of matrix columns to obtain the NWD, which allows that matrices of different lengths to be compared. All matrices performed better using a Markov order of 1 (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). Comparison of these matrices using this background model showed that the matrix of 12 columns yielded the highest NWD values (Fig. <a reveal-id="DSR045F3" data-open="DSR045F3" class="link link-reveal link-table xref-fig">3</a>), indicating that this PSSM was the best one to discriminate putative promoter sequences from the noncoding genomic background. </p>                        <a id="6198977" scrollto-destination="6198977"></a>
<div id="DSR045F3" data-id="DSR045F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dsr045/2/m_dsr04503.jpeg?Expires=2147483647&Signature=V8gh9RDjuU5KHAzoPYFGvTVEM92LlsBpIsOa2i4Z7qqJXufHy~q3r2uIWiPatf3Bor5c3Pt0kUS9c6GEsk90hAZFLEHMfUqU5tI8Z9obYppBuAqo8j-1M6pIRGoZVBQypWl67O5ApYcvcMdYTP5IwE4C6zRGmMQUt80kU1VtYTPE7tl~4-bjOGcNX8VNHpu3ijvon716MQFbN-JogS6c6g1MW7nEtY6eZK8Fb9h6Hk1rIpK0xCIp6iYZ4VpsEs7LREhk-CCTZ49gQxNLHOivhEm-XuUKrRPhbqGa1MGbP1QyCD845zY1CxRxbZaMHOY2jDKlYGODpI0xaQk6Ug206g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Performances of the 12-, 14- and 16-column PSSMs using a Markov order of 1 as the background model. Each curve shows the normalized weight score difference (NWD) calculated from theoretical and empirical distributions obtained for each matrix using a Markov order of 1. The higher the NWD value, the better the matrix distinguished putative sites from the noncoding genomic background." path-from-xml="dsr04503.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6198977/dsr04503.jpeg" path-from-xml="dsr04503.jpeg" target="_blank">View large</a><a section="6198977" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dsr045/2/dsr04503.jpeg?Expires=2147483647&Signature=oOPvU4QL3VCu85lctTBCJBaQ1ZYnthtkOb0fXHgZXYA03feJfsuGPJsOpkJauihIOtT2sEw3svToXcB7FtfswQy08QA~HpLrduE1PJCG9x0jvJOhN9ThpIwSuoKypXY03AUCZpsPSbYwXlfIh5TBKgMZY14pbyHI-CHowgSTcqdDFurMyq8Qj~nFK39OCWcz93nxp1DZjN8EtaKb0Uml~1i3CRJbvWJ0y0Dac-GKZLNqrush9HLCDOZ7nBsNA81C~LH0xmZAQJJAlNCS7iHA28Kw6SXwmufRyF2sINg2UUWgA0xFf63M2bwGgJlhYtPGrA1NM2VuLZO41VeJ2jIrAA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6198977&ar=418941&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr04503.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Performances of the 12-, 14- and 16-column PSSMs using a Markov order of 1 as the background model. Each curve shows the normalized weight score difference (NWD) calculated from theoretical and empirical distributions obtained for each matrix using a Markov order of 1. The higher the NWD value, the better the matrix distinguished putative sites from the noncoding genomic background.</p></div></div><div content-id="DSR045F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dsr045/2/m_dsr04503.jpeg?Expires=2147483647&Signature=V8gh9RDjuU5KHAzoPYFGvTVEM92LlsBpIsOa2i4Z7qqJXufHy~q3r2uIWiPatf3Bor5c3Pt0kUS9c6GEsk90hAZFLEHMfUqU5tI8Z9obYppBuAqo8j-1M6pIRGoZVBQypWl67O5ApYcvcMdYTP5IwE4C6zRGmMQUt80kU1VtYTPE7tl~4-bjOGcNX8VNHpu3ijvon716MQFbN-JogS6c6g1MW7nEtY6eZK8Fb9h6Hk1rIpK0xCIp6iYZ4VpsEs7LREhk-CCTZ49gQxNLHOivhEm-XuUKrRPhbqGa1MGbP1QyCD845zY1CxRxbZaMHOY2jDKlYGODpI0xaQk6Ug206g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Performances of the 12-, 14- and 16-column PSSMs using a Markov order of 1 as the background model. Each curve shows the normalized weight score difference (NWD) calculated from theoretical and empirical distributions obtained for each matrix using a Markov order of 1. The higher the NWD value, the better the matrix distinguished putative sites from the noncoding genomic background." path-from-xml="dsr04503.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6198977/dsr04503.jpeg" path-from-xml="dsr04503.jpeg" target="_blank">View large</a><a section="6198977" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dsr045/2/dsr04503.jpeg?Expires=2147483647&Signature=oOPvU4QL3VCu85lctTBCJBaQ1ZYnthtkOb0fXHgZXYA03feJfsuGPJsOpkJauihIOtT2sEw3svToXcB7FtfswQy08QA~HpLrduE1PJCG9x0jvJOhN9ThpIwSuoKypXY03AUCZpsPSbYwXlfIh5TBKgMZY14pbyHI-CHowgSTcqdDFurMyq8Qj~nFK39OCWcz93nxp1DZjN8EtaKb0Uml~1i3CRJbvWJ0y0Dac-GKZLNqrush9HLCDOZ7nBsNA81C~LH0xmZAQJJAlNCS7iHA28Kw6SXwmufRyF2sINg2UUWgA0xFf63M2bwGgJlhYtPGrA1NM2VuLZO41VeJ2jIrAA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6198977&ar=418941&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr04503.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Performances of the 12-, 14- and 16-column PSSMs using a Markov order of 1 as the background model. Each curve shows the normalized weight score difference (NWD) calculated from theoretical and empirical distributions obtained for each matrix using a Markov order of 1. The higher the NWD value, the better the matrix distinguished putative sites from the noncoding genomic background.</p></div></div><p>Once the PSSM and the background model were defined, additional analyses were performed. In order to generate a complementary negative control, the same data set used for the empirical distribution was scanned using column-permuted matrices derived from the 12-column PSSM. Figure <a reveal-id="DSR045F4" data-open="DSR045F4" class="link link-reveal link-table xref-fig">4</a> shows that the mean of the score distributions of 10 permuted matrices overlapped the theoretical distribution. This confirmed that the theoretical distribution can be considered an appropriate estimate of the FPR, and that the divergence observed in the original PSSM distribution corresponded to sites specifically detected by this matrix in the genome.<sup><a reveal-id="DSR045C26" data-open="DSR045C26" class="link link-ref link-reveal xref-bibr">26</a></sup></p>                        <a id="6198979" scrollto-destination="6198979"></a>
<div id="DSR045F4" data-id="DSR045F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dsr045/2/m_dsr04504.jpeg?Expires=2147483647&Signature=axMDBaY9fZYNxDabORNs8aLdwLDyKHpBJQCsC8~2Ymb3IyVNbZBjmCtySC8v5yauQvS5XqIfT1NItOJaA7M0yRHDAaaeZLwA4W53LJn4tNHGtZrbjCUSZ1xZgFBKh2UULdQehfHTw8A2eqHGQwtn~LzQR9pFAZkLEG3-3wDNlzQGbU9KJxvnRRHUAK5KU-GwBQiNt1sIMDFqt7sSjp3spN4hfI2KPK3piFQaRemd~ATylRguBJqykXn6wUVUtsZMnijJz22-qwvIFCkogJA0Nh64uf42-P9O40i6bVOVcmWtI~Ru4ZgbEFjkWZxiQFKEHEj5nyFOUbYCjVf-vMEMJA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Weight score distributions for the 12-column PSSM. The curves of the theoretical (black; dashed-dotted line) and empirical (orange; dashed line) distributions obtained with the 12-column PSSM began to separate at a weight score of 3, indicating that the promoters were being distinguished from the genomic background. Note that the mean of the score distributions of 10 column-permuted matrices (blue; solid line) overlaps with the theoretical distribution, confirming that the theoretical distribution can be considered an appropriate estimation of the FPR. The theoretical score distribution was estimated with a Markov model of order 1 using the whole set of upstream noncoding sequences of M. hyopneumoniae. The empirical score distribution was obtained with a sequence set composed of the 250 bases upstream and 50 bases downstream of the start codon of all M. hyopneumoniae protein-coding genes. The dCDF (ordinate) indicates the probability of observing a site scoring higher than or equal to a given weight score (abscissa)." path-from-xml="dsr04504.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6198979/dsr04504.jpeg" path-from-xml="dsr04504.jpeg" target="_blank">View large</a><a section="6198979" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dsr045/2/dsr04504.jpeg?Expires=2147483647&Signature=EajEeJGbsgElxtv5uiEBJpK~p2at6c8NaHkYa5WCJ14PR8GA-tsYLoBfaLRGPQb~2QApa7TVMPrQ4k7PQVtdMDMyn2Zb3vS6VrFQSCjQqXTh5jSIKPMhZ8XCYn4SUyzQtIzGOP9DYPF2fq3h~UBnHqsu0U~L8~2k0SBpLMTHodEoJTiz06ZHOEylQnGGocWTl7LYFRzef9~8uwcynVXyoSgXm9w01oHjxGIqKHGwLxk5ZCl4Uv02GhXIMNA95p7OujkwSaSVAFEO~OFiRmlmqLWwZlOhtuju0sFSwHcYOZjRpi4hLm0q2~ZCbOX4BUYVMIihJUJGJyRc9~XUuZW7dQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6198979&ar=418941&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr04504.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Weight score distributions for the 12-column PSSM. The curves of the theoretical (black; dashed-dotted line) and empirical (orange; dashed line) distributions obtained with the 12-column PSSM began to separate at a weight score of 3, indicating that the promoters were being distinguished from the genomic background. Note that the mean of the score distributions of 10 column-permuted matrices (blue; solid line) overlaps with the theoretical distribution, confirming that the theoretical distribution can be considered an appropriate estimation of the FPR. The theoretical score distribution was estimated with a Markov model of order 1 using the whole set of upstream noncoding sequences of <em>M. hyopneumoniae</em>. The empirical score distribution was obtained with a sequence set composed of the 250 bases upstream and 50 bases downstream of the start codon of all <em>M. hyopneumoniae</em> protein-coding genes. The dCDF (ordinate) indicates the probability of observing a site scoring higher than or equal to a given weight score (abscissa).</p></div></div><div content-id="DSR045F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dsr045/2/m_dsr04504.jpeg?Expires=2147483647&Signature=axMDBaY9fZYNxDabORNs8aLdwLDyKHpBJQCsC8~2Ymb3IyVNbZBjmCtySC8v5yauQvS5XqIfT1NItOJaA7M0yRHDAaaeZLwA4W53LJn4tNHGtZrbjCUSZ1xZgFBKh2UULdQehfHTw8A2eqHGQwtn~LzQR9pFAZkLEG3-3wDNlzQGbU9KJxvnRRHUAK5KU-GwBQiNt1sIMDFqt7sSjp3spN4hfI2KPK3piFQaRemd~ATylRguBJqykXn6wUVUtsZMnijJz22-qwvIFCkogJA0Nh64uf42-P9O40i6bVOVcmWtI~Ru4ZgbEFjkWZxiQFKEHEj5nyFOUbYCjVf-vMEMJA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Weight score distributions for the 12-column PSSM. The curves of the theoretical (black; dashed-dotted line) and empirical (orange; dashed line) distributions obtained with the 12-column PSSM began to separate at a weight score of 3, indicating that the promoters were being distinguished from the genomic background. Note that the mean of the score distributions of 10 column-permuted matrices (blue; solid line) overlaps with the theoretical distribution, confirming that the theoretical distribution can be considered an appropriate estimation of the FPR. The theoretical score distribution was estimated with a Markov model of order 1 using the whole set of upstream noncoding sequences of M. hyopneumoniae. The empirical score distribution was obtained with a sequence set composed of the 250 bases upstream and 50 bases downstream of the start codon of all M. hyopneumoniae protein-coding genes. The dCDF (ordinate) indicates the probability of observing a site scoring higher than or equal to a given weight score (abscissa)." path-from-xml="dsr04504.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6198979/dsr04504.jpeg" path-from-xml="dsr04504.jpeg" target="_blank">View large</a><a section="6198979" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dsr045/2/dsr04504.jpeg?Expires=2147483647&Signature=EajEeJGbsgElxtv5uiEBJpK~p2at6c8NaHkYa5WCJ14PR8GA-tsYLoBfaLRGPQb~2QApa7TVMPrQ4k7PQVtdMDMyn2Zb3vS6VrFQSCjQqXTh5jSIKPMhZ8XCYn4SUyzQtIzGOP9DYPF2fq3h~UBnHqsu0U~L8~2k0SBpLMTHodEoJTiz06ZHOEylQnGGocWTl7LYFRzef9~8uwcynVXyoSgXm9w01oHjxGIqKHGwLxk5ZCl4Uv02GhXIMNA95p7OujkwSaSVAFEO~OFiRmlmqLWwZlOhtuju0sFSwHcYOZjRpi4hLm0q2~ZCbOX4BUYVMIihJUJGJyRc9~XUuZW7dQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6198979&ar=418941&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr04504.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Weight score distributions for the 12-column PSSM. The curves of the theoretical (black; dashed-dotted line) and empirical (orange; dashed line) distributions obtained with the 12-column PSSM began to separate at a weight score of 3, indicating that the promoters were being distinguished from the genomic background. Note that the mean of the score distributions of 10 column-permuted matrices (blue; solid line) overlaps with the theoretical distribution, confirming that the theoretical distribution can be considered an appropriate estimation of the FPR. The theoretical score distribution was estimated with a Markov model of order 1 using the whole set of upstream noncoding sequences of <em>M. hyopneumoniae</em>. The empirical score distribution was obtained with a sequence set composed of the 250 bases upstream and 50 bases downstream of the start codon of all <em>M. hyopneumoniae</em> protein-coding genes. The dCDF (ordinate) indicates the probability of observing a site scoring higher than or equal to a given weight score (abscissa).</p></div></div>                            <h3 scrollto-destination=6198980 id="6198980" class="section-title" data-legacy-id=s3e>3.5. Score threshold determination</h3>
<p>The curves of the theoretical and empirical distributions of the 12-column PSSM began to separate from each other around a weight score of 3 (Fig. <a reveal-id="DSR045F4" data-open="DSR045F4" class="link link-reveal link-table xref-fig">4</a>), which is probably indicative of the presence of functional binding sites. At this score value, the decreasing cumulative distribution function (dCDF, indicates the <em>P</em>-value, i.e. the probability to obtain by chance a weight score higher than or equal to a given value) in theoretical distribution is 4.1 × 10<sup>−3</sup> (3.86 × 10<sup>−3</sup> in the permuted matrix distribution) and in the empirical distribution is 5.8 × 10<sup>−3</sup>. It means that for ∼6 sites found in the upstream gene sequences, one could expect that ∼4 of those were false-positives. Hence, the incidence of false-positives in relation to the observed frequency of sites in the target sequences is too high at this point. However, from a score of 3 upwards, the difference between the observed and the expected frequencies gradually increased (Fig. <a reveal-id="DSR045F4" data-open="DSR045F4" class="link link-reveal link-table xref-fig">4</a>). Consequently, the choice of a score threshold that would allow comprehensive promoter identification with a relatively low FPR was necessary.</p><p>The threshold score was defined using the complementary approach described by Cases <em>et al.</em><sup><a reveal-id="DSR045C30" data-open="DSR045C30" class="link link-ref link-reveal xref-bibr">30</a></sup> This procedure compares the score distributions of predicted promoters that are ‘correctly’ oriented – that are in the same direction as the downstream gene – with those found in the reverse strand, which are, therefore, ‘incorrectly’ oriented. The assumption is that false-positives should be homogeneously distributed between both strands, whereas true positives must be correctly oriented.<sup><a reveal-id="DSR045C30" data-open="DSR045C30" class="link link-ref link-reveal xref-bibr">30</a></sup></p><p>The target sequences were composed of those from the dataset used for the determination of the empirical distribution, but the sequences located between divergent genes were excluded, as they could have had promoters in both directions. The occurrence of putative promoters in these sequences and their respective reverse complements was determined by Matrix-Scan using the 12-column PSSM and a Markov order of 1 as the background model. The distributions of the correctly and incorrectly oriented promoters are presented in Fig. <a reveal-id="DSR045F5" data-open="DSR045F5" class="link link-reveal link-table xref-fig">5</a>. The incidence of incorrectly oriented promoters considerably diminished with a weight score of 6.5, so this was used as the threshold score for posterior analyses. The estimated FPR at this score was 2.4 × 10<sup>−4</sup> (2 × 10<sup>−4</sup> in the permuted matrices distribution), whereas the dCDF in the empirical distribution was 1.42 × 10<sup>−3</sup>. </p>                        <a id="6198984" scrollto-destination="6198984"></a>
<div id="DSR045F5" data-id="DSR045F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dsr045/2/m_dsr04505.jpeg?Expires=2147483647&Signature=mdvxLLHay0ABXZD1VnqpHX0J8kn8jteclQS937qM4ZeeVuQIXh1HziDbjDyg3mtAqZkaCTiubESivUNJrUaxneYYQamZJDgitXauodZMjBacOdkC9hD-8o03DOEV~Uvp6UN3ziUdTuNcUkMIbptreizHSalW7D65GrTgwhzlvOHMH~0KruDvlb-AxhMHWBhIbWVeb0j~9SODmq5kx8sL3N65~QrW30DlL-dxvWcyXdtqpvLW8SofPA9YnvYnoszpp68-C24y0udp-bNGFPord9gy8caWkMx8NcMdjlgwei5DGpt4G5ThoGdk9O40RpjrnrHG8kt1RlNvwQ7ZQEqK4Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Weight score threshold definition. Distribution of the scores of the correctly and incorrectly oriented promoters predicted in the M. hyopneumoniae intergenic regions. Note that from score 6.5, the frequencies of incorrectly oriented promoters are much smaller than the frequencies of correctly oriented promoters." path-from-xml="dsr04505.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6198984/dsr04505.jpeg" path-from-xml="dsr04505.jpeg" target="_blank">View large</a><a section="6198984" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dsr045/2/dsr04505.jpeg?Expires=2147483647&Signature=BRxNPMXTIU3DVmD8WjhYoYW5L5pVDkNHFTfKF1oF9AjYbsS-dWCpwtZovADfrIRbvILeTG72gLEjk-FvERksl3ce0jBI9BKNAKasEW7sT-pQLfLeQkNTECXTZYeDLOtOfitUkKwlopyfFGc8rQRwy3VFEdWvohhAdLBvCmO45LzNg2f~6oxxhWF~furneo9Tv9UoFB99jZPD2Fc-VwbCvm8IHFjuXoIFBVS4Vl2ZS1amrG-J7Ma0Gmn2CC7Ry6EBC8mk~VOK-RrtwUb3YXjtPIOs~c3k7~wso1CAfOll8MOCqpqLkQ9jCI1NYzQJx7AZ4x16mAxlf9INDP8BkgMhSg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6198984&ar=418941&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr04505.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Weight score threshold definition. Distribution of the scores of the correctly and incorrectly oriented promoters predicted in the <em>M. hyopneumoniae</em> intergenic regions. Note that from score 6.5, the frequencies of incorrectly oriented promoters are much smaller than the frequencies of correctly oriented promoters.</p></div></div><div content-id="DSR045F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dsr045/2/m_dsr04505.jpeg?Expires=2147483647&Signature=mdvxLLHay0ABXZD1VnqpHX0J8kn8jteclQS937qM4ZeeVuQIXh1HziDbjDyg3mtAqZkaCTiubESivUNJrUaxneYYQamZJDgitXauodZMjBacOdkC9hD-8o03DOEV~Uvp6UN3ziUdTuNcUkMIbptreizHSalW7D65GrTgwhzlvOHMH~0KruDvlb-AxhMHWBhIbWVeb0j~9SODmq5kx8sL3N65~QrW30DlL-dxvWcyXdtqpvLW8SofPA9YnvYnoszpp68-C24y0udp-bNGFPord9gy8caWkMx8NcMdjlgwei5DGpt4G5ThoGdk9O40RpjrnrHG8kt1RlNvwQ7ZQEqK4Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Weight score threshold definition. Distribution of the scores of the correctly and incorrectly oriented promoters predicted in the M. hyopneumoniae intergenic regions. Note that from score 6.5, the frequencies of incorrectly oriented promoters are much smaller than the frequencies of correctly oriented promoters." path-from-xml="dsr04505.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6198984/dsr04505.jpeg" path-from-xml="dsr04505.jpeg" target="_blank">View large</a><a section="6198984" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dsr045/2/dsr04505.jpeg?Expires=2147483647&Signature=BRxNPMXTIU3DVmD8WjhYoYW5L5pVDkNHFTfKF1oF9AjYbsS-dWCpwtZovADfrIRbvILeTG72gLEjk-FvERksl3ce0jBI9BKNAKasEW7sT-pQLfLeQkNTECXTZYeDLOtOfitUkKwlopyfFGc8rQRwy3VFEdWvohhAdLBvCmO45LzNg2f~6oxxhWF~furneo9Tv9UoFB99jZPD2Fc-VwbCvm8IHFjuXoIFBVS4Vl2ZS1amrG-J7Ma0Gmn2CC7Ry6EBC8mk~VOK-RrtwUb3YXjtPIOs~c3k7~wso1CAfOll8MOCqpqLkQ9jCI1NYzQJx7AZ4x16mAxlf9INDP8BkgMhSg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6198984&ar=418941&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr04505.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Weight score threshold definition. Distribution of the scores of the correctly and incorrectly oriented promoters predicted in the <em>M. hyopneumoniae</em> intergenic regions. Note that from score 6.5, the frequencies of incorrectly oriented promoters are much smaller than the frequencies of correctly oriented promoters.</p></div></div><p>The trade-off between the FPR and the sensitivity of the threshold score was assessed using the ROC curve generated by Matrix-Quality analysis (Fig. <a reveal-id="DSR045F6" data-open="DSR045F6" class="link link-reveal link-table xref-fig">6</a>). The sensitivity of a PSSM is the proportion of correct sites detected above the score threshold, and it is estimated by scoring the sites used to build the matrix.<sup><a reveal-id="DSR045C26" data-open="DSR045C26" class="link link-ref link-reveal xref-bibr">26</a></sup> This estimation was also performed using the LOO validation, which corrects biases in matrix sensitivity.<sup><a reveal-id="DSR045C26" data-open="DSR045C26" class="link link-ref link-reveal xref-bibr">26</a></sup> Figure <a reveal-id="DSR045F6" data-open="DSR045F6" class="link link-reveal link-table xref-fig">6</a> shows that a FPR of 2.4 × 10<sup>−4</sup> (at a score of 6.5) is associated with a sensitivity of 0.65 for the biased curve, and 0.60 for the LOO curve. It is worth noting that the LOO curve and the unbiased curve are not distant from each other, so overfitting was insignificant. </p>                        <a id="6198986" scrollto-destination="6198986"></a>
<div id="DSR045F6" data-id="DSR045F6" class="fig fig-section"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dsr045/2/m_dsr04506.jpeg?Expires=2147483647&Signature=CJzYlTuWGlrRt8K0xbbjY0epSLtEyQy4DtFgE8PFVam7UxHWmlar13~ihRsgsaYd9xo56AbmvbcVsGX1~K3KV7EC5nayzI6nTsh6OgzoFUsb8~AUTlCiPllLNqwGCV1ZRgA8iUnN0rYfBBo0mNb-Pu5SfiVKOh65w~X6I3dsMNaOJbCugen0OGFIVIpE7jqZnBlyesstHZK0Kfzd4Xnq1Bj3yhBMCialJTE~RuSYDh5ElgFf6gYR6LiSiCGvtTA~8bM9ZxYAvMkYfYwHOQwEnfNZE-hMb~HQS2dqDsrBa-p2pmC0G1TGh~W3cCU1C1nzytRrKiTgt~R2v1OvhwLwNA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Trade-off between the sensitivity and FPR of the 12-column PSSM. (A) Score distributions of the experimentally defined sites used to build the matrix. Blue (dashed line), biased scores assigned by the matrix to the defined sites. Orange (solid line), unbiased scores obtained using the LOO procedure. The ordinate indicates the probability of observing a site scoring higher than or equal to a given weight score (abscissa). (B) ROC curve indicating the risk of false-positives associated with a specific sensitivity. Both graphs show the difference between the biased (blue; dashed line) and LOO estimations (orange; solid line). The dCDF (ordinate) indicates the sensitivity (fraction of sites detected) and the abscissa shows the corresponding FPR. Note that the dCDF (A) corresponds to the sensitivity (B)." path-from-xml="dsr04506.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6198986/dsr04506.jpeg" path-from-xml="dsr04506.jpeg" target="_blank">View large</a><a section="6198986" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dsr045/2/dsr04506.jpeg?Expires=2147483647&Signature=XjT9OINowafnEKFDVm~KpV9FQEHu3u~6vj3Xh9iop9XrVgPQCPrjC0oM3Wzl-kFY~GlxNat~Io2-8uKPFq~v38B~baGt9WfJIdfcNR1E2~LyGrS4Y~keENfwURm0NO28yGuIMsdlEXtek14o8sfSfnKWgcON4g0Vq9LTf5lVAZDTpTUlgFqJJC4auUQre04teSwa3Ws0ZCOkJCbvuV5NAfQiFCu5R3qOKZc0h4PpzdaNVbqGWZP9h~li0euIG3THqESthmxKlQGB8Zgel8u8CVSHM-Lp0WQ9gaTK2EiM-GkZmq8GMifOOj-nOsMiMPP1ReR2HqWlRJmjQdq4bYnICg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6198986&ar=418941&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr04506.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Trade-off between the sensitivity and FPR of the 12-column PSSM. (A) Score distributions of the experimentally defined sites used to build the matrix. Blue (dashed line), biased scores assigned by the matrix to the defined sites. Orange (solid line), unbiased scores obtained using the LOO procedure. The ordinate indicates the probability of observing a site scoring higher than or equal to a given weight score (abscissa). (B) ROC curve indicating the risk of false-positives associated with a specific sensitivity. Both graphs show the difference between the biased (blue; dashed line) and LOO estimations (orange; solid line). The dCDF (ordinate) indicates the sensitivity (fraction of sites detected) and the abscissa shows the corresponding FPR. Note that the dCDF (A) corresponds to the sensitivity (B).</p></div></div><div content-id="DSR045F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dsr045/2/m_dsr04506.jpeg?Expires=2147483647&Signature=CJzYlTuWGlrRt8K0xbbjY0epSLtEyQy4DtFgE8PFVam7UxHWmlar13~ihRsgsaYd9xo56AbmvbcVsGX1~K3KV7EC5nayzI6nTsh6OgzoFUsb8~AUTlCiPllLNqwGCV1ZRgA8iUnN0rYfBBo0mNb-Pu5SfiVKOh65w~X6I3dsMNaOJbCugen0OGFIVIpE7jqZnBlyesstHZK0Kfzd4Xnq1Bj3yhBMCialJTE~RuSYDh5ElgFf6gYR6LiSiCGvtTA~8bM9ZxYAvMkYfYwHOQwEnfNZE-hMb~HQS2dqDsrBa-p2pmC0G1TGh~W3cCU1C1nzytRrKiTgt~R2v1OvhwLwNA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Trade-off between the sensitivity and FPR of the 12-column PSSM. (A) Score distributions of the experimentally defined sites used to build the matrix. Blue (dashed line), biased scores assigned by the matrix to the defined sites. Orange (solid line), unbiased scores obtained using the LOO procedure. The ordinate indicates the probability of observing a site scoring higher than or equal to a given weight score (abscissa). (B) ROC curve indicating the risk of false-positives associated with a specific sensitivity. Both graphs show the difference between the biased (blue; dashed line) and LOO estimations (orange; solid line). The dCDF (ordinate) indicates the sensitivity (fraction of sites detected) and the abscissa shows the corresponding FPR. Note that the dCDF (A) corresponds to the sensitivity (B)." path-from-xml="dsr04506.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6198986/dsr04506.jpeg" path-from-xml="dsr04506.jpeg" target="_blank">View large</a><a section="6198986" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dsr045/2/dsr04506.jpeg?Expires=2147483647&Signature=XjT9OINowafnEKFDVm~KpV9FQEHu3u~6vj3Xh9iop9XrVgPQCPrjC0oM3Wzl-kFY~GlxNat~Io2-8uKPFq~v38B~baGt9WfJIdfcNR1E2~LyGrS4Y~keENfwURm0NO28yGuIMsdlEXtek14o8sfSfnKWgcON4g0Vq9LTf5lVAZDTpTUlgFqJJC4auUQre04teSwa3Ws0ZCOkJCbvuV5NAfQiFCu5R3qOKZc0h4PpzdaNVbqGWZP9h~li0euIG3THqESthmxKlQGB8Zgel8u8CVSHM-Lp0WQ9gaTK2EiM-GkZmq8GMifOOj-nOsMiMPP1ReR2HqWlRJmjQdq4bYnICg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6198986&ar=418941&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr04506.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Trade-off between the sensitivity and FPR of the 12-column PSSM. (A) Score distributions of the experimentally defined sites used to build the matrix. Blue (dashed line), biased scores assigned by the matrix to the defined sites. Orange (solid line), unbiased scores obtained using the LOO procedure. The ordinate indicates the probability of observing a site scoring higher than or equal to a given weight score (abscissa). (B) ROC curve indicating the risk of false-positives associated with a specific sensitivity. Both graphs show the difference between the biased (blue; dashed line) and LOO estimations (orange; solid line). The dCDF (ordinate) indicates the sensitivity (fraction of sites detected) and the abscissa shows the corresponding FPR. Note that the dCDF (A) corresponds to the sensitivity (B).</p></div></div>                            <h3 scrollto-destination=6198987 id="6198987" class="section-title" data-legacy-id=s3f>3.6. Predicted promoters</h3>
<p>After the optimum matrix parameters were defined, the upstream sequences of all 657 <em>M. hyopneumoniae</em> CDSs were scanned for the presence of putative promoters using Matrix-Scan. Table <a reveal-id="DSR045TB3" data-open="DSR045TB3" class="link link-reveal link-table xref-fig">3</a> shows the general results of this analysis. Using a threshold score of 6.5, 201 sites were identified upstream of 169 different genes, 26% of the total CDSs. </p>                        <a id="6198989" scrollto-destination="6198989"></a>
<div content-id="DSR045TB3" class="table-modal"><div class="table-wrap"><div id="DSR045TB3" class="table-wrap-title" data-id="DSR045TB3"><div class="label -label">Table 3.</div><div class="caption"><p><em>Mycoplasma hyopneumoniae</em> promoter prediction analysis</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th><em>N</em> </th></tr></thead><tbody><tr><td colspan="2">Genomic features </td></tr><tr><td> CDSs annotated in the genome </td><td>657 </td></tr><tr><td> CDSs that have an upstream region &lt; 15 bp </td><td>201/657 (31%) </td></tr><tr><td> CDSs that have a divergent upstream gene </td><td>142/657 (22%) </td></tr><tr><td> CDSs that have an upstream gene oriented in the same direction </td><td>515/657 (78%) </td></tr><tr><td colspan="2">Predicted promoter features (weight score ≥ 6.5) </td></tr><tr><td> Promoters </td><td>201 </td></tr><tr><td> CDSs that have at least one promoter </td><td>169/657 (26%) </td></tr><tr><td> CDSs that have: </td><td> </td></tr><tr><td>  One promoter </td><td>143/169 (84%) </td></tr><tr><td>  Two promoters </td><td>22/169 (13%) </td></tr><tr><td>  Three promoters </td><td>3/169 (2%) </td></tr><tr><td>  Five promoters </td><td>1/169 (&lt;1%) </td></tr><tr><td> CDSs that have a divergent upstream gene and have at least one promoter </td><td>76/142 (54%) </td></tr><tr><td> CDSs that have an upstream gene oriented in the same direction and have at least one promoter </td><td>93/515 (18%) </td></tr><tr><td colspan="2">Predicted promoter features (weight score ≥ 4.2) </td></tr><tr><td> Promoters </td><td>409 </td></tr><tr><td> CDSs that have at least one promoter </td><td>273/657 (42%) </td></tr><tr><td> CDSs that have a divergent upstream gene and have at least one promoter </td><td>113/142 (80%) </td></tr><tr><td> CDSs that have an upstream gene oriented in the same direction and have at least one promoter </td><td>160/515 (31%) </td></tr></tbody></table></div><div content-id="DSR045TB3" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th><em>N</em> </th></tr></thead><tbody><tr><td colspan="2">Genomic features </td></tr><tr><td> CDSs annotated in the genome </td><td>657 </td></tr><tr><td> CDSs that have an upstream region &lt; 15 bp </td><td>201/657 (31%) </td></tr><tr><td> CDSs that have a divergent upstream gene </td><td>142/657 (22%) </td></tr><tr><td> CDSs that have an upstream gene oriented in the same direction </td><td>515/657 (78%) </td></tr><tr><td colspan="2">Predicted promoter features (weight score ≥ 6.5) </td></tr><tr><td> Promoters </td><td>201 </td></tr><tr><td> CDSs that have at least one promoter </td><td>169/657 (26%) </td></tr><tr><td> CDSs that have: </td><td> </td></tr><tr><td>  One promoter </td><td>143/169 (84%) </td></tr><tr><td>  Two promoters </td><td>22/169 (13%) </td></tr><tr><td>  Three promoters </td><td>3/169 (2%) </td></tr><tr><td>  Five promoters </td><td>1/169 (&lt;1%) </td></tr><tr><td> CDSs that have a divergent upstream gene and have at least one promoter </td><td>76/142 (54%) </td></tr><tr><td> CDSs that have an upstream gene oriented in the same direction and have at least one promoter </td><td>93/515 (18%) </td></tr><tr><td colspan="2">Predicted promoter features (weight score ≥ 4.2) </td></tr><tr><td> Promoters </td><td>409 </td></tr><tr><td> CDSs that have at least one promoter </td><td>273/657 (42%) </td></tr><tr><td> CDSs that have a divergent upstream gene and have at least one promoter </td><td>113/142 (80%) </td></tr><tr><td> CDSs that have an upstream gene oriented in the same direction and have at least one promoter </td><td>160/515 (31%) </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6198989" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR045TB3" class="table-wrap-title" data-id="DSR045TB3"><div class="label -label">Table 3.</div><div class="caption"><p><em>Mycoplasma hyopneumoniae</em> promoter prediction analysis</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th><em>N</em> </th></tr></thead><tbody><tr><td colspan="2">Genomic features </td></tr><tr><td> CDSs annotated in the genome </td><td>657 </td></tr><tr><td> CDSs that have an upstream region &lt; 15 bp </td><td>201/657 (31%) </td></tr><tr><td> CDSs that have a divergent upstream gene </td><td>142/657 (22%) </td></tr><tr><td> CDSs that have an upstream gene oriented in the same direction </td><td>515/657 (78%) </td></tr><tr><td colspan="2">Predicted promoter features (weight score ≥ 6.5) </td></tr><tr><td> Promoters </td><td>201 </td></tr><tr><td> CDSs that have at least one promoter </td><td>169/657 (26%) </td></tr><tr><td> CDSs that have: </td><td> </td></tr><tr><td>  One promoter </td><td>143/169 (84%) </td></tr><tr><td>  Two promoters </td><td>22/169 (13%) </td></tr><tr><td>  Three promoters </td><td>3/169 (2%) </td></tr><tr><td>  Five promoters </td><td>1/169 (&lt;1%) </td></tr><tr><td> CDSs that have a divergent upstream gene and have at least one promoter </td><td>76/142 (54%) </td></tr><tr><td> CDSs that have an upstream gene oriented in the same direction and have at least one promoter </td><td>93/515 (18%) </td></tr><tr><td colspan="2">Predicted promoter features (weight score ≥ 4.2) </td></tr><tr><td> Promoters </td><td>409 </td></tr><tr><td> CDSs that have at least one promoter </td><td>273/657 (42%) </td></tr><tr><td> CDSs that have a divergent upstream gene and have at least one promoter </td><td>113/142 (80%) </td></tr><tr><td> CDSs that have an upstream gene oriented in the same direction and have at least one promoter </td><td>160/515 (31%) </td></tr></tbody></table></div><div content-id="DSR045TB3" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th><em>N</em> </th></tr></thead><tbody><tr><td colspan="2">Genomic features </td></tr><tr><td> CDSs annotated in the genome </td><td>657 </td></tr><tr><td> CDSs that have an upstream region &lt; 15 bp </td><td>201/657 (31%) </td></tr><tr><td> CDSs that have a divergent upstream gene </td><td>142/657 (22%) </td></tr><tr><td> CDSs that have an upstream gene oriented in the same direction </td><td>515/657 (78%) </td></tr><tr><td colspan="2">Predicted promoter features (weight score ≥ 6.5) </td></tr><tr><td> Promoters </td><td>201 </td></tr><tr><td> CDSs that have at least one promoter </td><td>169/657 (26%) </td></tr><tr><td> CDSs that have: </td><td> </td></tr><tr><td>  One promoter </td><td>143/169 (84%) </td></tr><tr><td>  Two promoters </td><td>22/169 (13%) </td></tr><tr><td>  Three promoters </td><td>3/169 (2%) </td></tr><tr><td>  Five promoters </td><td>1/169 (&lt;1%) </td></tr><tr><td> CDSs that have a divergent upstream gene and have at least one promoter </td><td>76/142 (54%) </td></tr><tr><td> CDSs that have an upstream gene oriented in the same direction and have at least one promoter </td><td>93/515 (18%) </td></tr><tr><td colspan="2">Predicted promoter features (weight score ≥ 4.2) </td></tr><tr><td> Promoters </td><td>409 </td></tr><tr><td> CDSs that have at least one promoter </td><td>273/657 (42%) </td></tr><tr><td> CDSs that have a divergent upstream gene and have at least one promoter </td><td>113/142 (80%) </td></tr><tr><td> CDSs that have an upstream gene oriented in the same direction and have at least one promoter </td><td>160/515 (31%) </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6198989" target="_blank">View Large</a></div></div><p>The vast majority of the CDSs had a single putative promoter, although there were CDSs that had additional sites. In this promoter prediction analysis, 16 of the 23 promoters experimentally mapped scored between 6.9 and 11, six scored between 4.2 and 6.3, and one, the <em>recA</em> promoter, did not score above zero. Most of them corresponded to the hit with the highest score, but those of the genes <em>uvrC</em>, MHP7448_0198 and <em>ktrA</em> were the second best hits (although none of these scored higher than 6.5).</p><p>Our analyses detected at least one promoter in 54% of the CDSs that had a divergent upstream gene and in 18% of the CDSs that had an upstream gene oriented in the same direction. However, these proportions were 80 and 31%, respectively, if the threshold score was set at 4.2, the smallest weight score obtained for the experimentally defined promoters.</p><p>The distance of the promoters from the start codon was also examined. The majority of the predicted promoters, ∼67.5%, were located between 1 and 100 bases upstream of the start codon, with a preponderance located 25–50 bases upstream (Fig. <a reveal-id="DSR045F7" data-open="DSR045F7" class="link link-reveal link-table xref-fig">7</a>). Sixteen promoters were found within the coding sequences of 14 CDSs. </p>                        <a id="6198993" scrollto-destination="6198993"></a>
<div id="DSR045F7" data-id="DSR045F7" class="fig fig-section"><div class="label fig-label">Figure 7.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dsr045/2/m_dsr04507.jpeg?Expires=2147483647&Signature=ywMiDHfeF0T1rVdf0pLR12UTLjm0HEAGzSDnPvQgu6uP4b9NWgQ8RPHlNdt7GFW0BUY6wqE95Jg0SnghsJJKx8humCSDBu28iXo3AxsXs5gm8TKITuAWu2zBuLvchq1d0jXZjvie2Hb9SBR5Gepz22vQMdMJGZOOQNXiuPqM4Qu4vs1LmhXcDDnX~hmEzImgiZAt9nacxpyqBHAHn7u73JthTDdOFB0asekqYiVZw9rpR9qMHU5Zy~2Lwx0Ci~V77vrAeZuscsiO-BLDCv8Rj9SKdxis8Wx2QqsvPuECB0DbE2bZocFYM4la~lmSSTnwb-Mkyn56RF4NgAjgLinKIQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distance of the predicted promoters from the annotated start codon of the M. hyopneumoniae CDSs. Distances were determined for the 201 predicted promoters scoring ≥6.5; they were measured from the −10 element to the start codon of the genes. Black bars indicate bases upstream of the start codon, and grey bars indicate bases downstream of the start codon." path-from-xml="dsr04507.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6198993/dsr04507.jpeg" path-from-xml="dsr04507.jpeg" target="_blank">View large</a><a section="6198993" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dsr045/2/dsr04507.jpeg?Expires=2147483647&Signature=vk0h0K9Mp1EqasSa1z5sQC1S-Xy-lnhbVFoOn5wl5lYxcoo3UELjcdLN5J8Mo~K~thxIrHqDCsiBw0ShvvXk9W0fDLpLJFO1BfSW3Z0j9kSUu7FjaulO5qgb50qI1UyN8QgnuoBS0WWb81UJkHKJECbQE9DSbIj~BT5ekOQSVw8DdU3iHVHRNsgDW3BkoUH0uwijT7Jc5U~ydJdZw63ENufOxLB7K-rY~IrQXJT44Wi9b3AFD7EY4V-xtdjl0GpXrzc0hZvLp4ED1kypk6mgW2gc0OIgcxOipwZ210cIsK3xOIYRc5hhpfrfBODR89zncGM-gSteDdYz-UnBNvJnpw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6198993&ar=418941&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr04507.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distance of the predicted promoters from the annotated start codon of the <em>M. hyopneumoniae</em> CDSs. Distances were determined for the 201 predicted promoters scoring ≥6.5; they were measured from the −10 element to the start codon of the genes. Black bars indicate bases upstream of the start codon, and grey bars indicate bases downstream of the start codon.</p></div></div><div content-id="DSR045F7" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 7.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dsr045/2/m_dsr04507.jpeg?Expires=2147483647&Signature=ywMiDHfeF0T1rVdf0pLR12UTLjm0HEAGzSDnPvQgu6uP4b9NWgQ8RPHlNdt7GFW0BUY6wqE95Jg0SnghsJJKx8humCSDBu28iXo3AxsXs5gm8TKITuAWu2zBuLvchq1d0jXZjvie2Hb9SBR5Gepz22vQMdMJGZOOQNXiuPqM4Qu4vs1LmhXcDDnX~hmEzImgiZAt9nacxpyqBHAHn7u73JthTDdOFB0asekqYiVZw9rpR9qMHU5Zy~2Lwx0Ci~V77vrAeZuscsiO-BLDCv8Rj9SKdxis8Wx2QqsvPuECB0DbE2bZocFYM4la~lmSSTnwb-Mkyn56RF4NgAjgLinKIQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distance of the predicted promoters from the annotated start codon of the M. hyopneumoniae CDSs. Distances were determined for the 201 predicted promoters scoring ≥6.5; they were measured from the −10 element to the start codon of the genes. Black bars indicate bases upstream of the start codon, and grey bars indicate bases downstream of the start codon." path-from-xml="dsr04507.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6198993/dsr04507.jpeg" path-from-xml="dsr04507.jpeg" target="_blank">View large</a><a section="6198993" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dsr045/2/dsr04507.jpeg?Expires=2147483647&Signature=vk0h0K9Mp1EqasSa1z5sQC1S-Xy-lnhbVFoOn5wl5lYxcoo3UELjcdLN5J8Mo~K~thxIrHqDCsiBw0ShvvXk9W0fDLpLJFO1BfSW3Z0j9kSUu7FjaulO5qgb50qI1UyN8QgnuoBS0WWb81UJkHKJECbQE9DSbIj~BT5ekOQSVw8DdU3iHVHRNsgDW3BkoUH0uwijT7Jc5U~ydJdZw63ENufOxLB7K-rY~IrQXJT44Wi9b3AFD7EY4V-xtdjl0GpXrzc0hZvLp4ED1kypk6mgW2gc0OIgcxOipwZ210cIsK3xOIYRc5hhpfrfBODR89zncGM-gSteDdYz-UnBNvJnpw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6198993&ar=418941&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr04507.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distance of the predicted promoters from the annotated start codon of the <em>M. hyopneumoniae</em> CDSs. Distances were determined for the 201 predicted promoters scoring ≥6.5; they were measured from the −10 element to the start codon of the genes. Black bars indicate bases upstream of the start codon, and grey bars indicate bases downstream of the start codon.</p></div></div>                            <h2 scrollto-destination=6198994 id="6198994" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>The transcripts of 23 genes of <em>M. hyopneumoniae</em> were analysed in order to map their TSSs. As is usually the case in transcripts of other bacteria,<sup><a reveal-id="DSR045C8" data-open="DSR045C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSR045C15" data-open="DSR045C15" class="link link-ref link-reveal xref-bibr">15</a>,<a reveal-id="DSR045C16" data-open="DSR045C16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="DSR045C18" data-open="DSR045C18" class="link link-ref link-reveal xref-bibr">18</a></sup> most of those from <em>M. hyopneumoniae</em> started with a purine. Our data also showed that many of its gene transcripts had variation at their 5′ ends, suggesting the occurrence of heterogeneous TSSs. The heterogeneity observed in some <em>M. hyopneumoniae</em> transcripts was due to additional untemplated nucleotides (i.e. nucleotides not expected from the genome sequence), and was probably the result of transcriptional slippage.<sup><a reveal-id="DSR045C31" data-open="DSR045C31" class="link link-ref link-reveal xref-bibr">31</a></sup> In this process, the RNA polymerase adds nucleotides, repetitively, to the 3′ end of the nascent transcript, typically within homopolymeric sequences. Differently, the 5′ end of some transcripts had length differences in which the additional nucleotides were identical to the genomic sequence. Such templated heterogeneous 5′ ends have also been seen frequently in the <em>M. pneumoniae</em> transcripts.<sup><a reveal-id="DSR045C8" data-open="DSR045C8" class="link link-ref link-reveal xref-bibr">8</a></sup></p><p>In addition, a high frequency of transcripts that have just few nucleotides in their 5′ untranslated region, reported in <em>M. pneumoniae</em>,<sup><a reveal-id="DSR045C8" data-open="DSR045C8" class="link link-ref link-reveal xref-bibr">8</a></sup> was also seen in <em>M. hyopneumoniae</em>. Translation can be initiated on the leaderless mRNAs in the three domains of life, but, although they are abundant in Archaea, they are still considered rare in bacteria. Thus, as mentioned by Weiner <em>et al</em>.,<sup><a reveal-id="DSR045C8" data-open="DSR045C8" class="link link-ref link-reveal xref-bibr">8</a></sup> this high incidence of leaderless transcripts in <em>Mycoplasma</em> could be a result of adaptation to a minimal genome with the aim of reducing the genomic space required for initiation of translation.</p><p>The only σ factor identified in the <em>M. hyopneumoniae</em> genome belongs to the σ<sup>70</sup> protein family. The σ<sup>70</sup> factors interact with archetypical promoters that are composed of two main regions: the −35 element (TTGACA) and the −10 element (TATAAT). The upstream regions of experimentally defined TSSs of the <em>M. hyopneumoniae</em> genes contained a −10 element, but no obvious −35 element, a structure shared with other low G + C content bacteria.<sup><a reveal-id="DSR045C19" data-open="DSR045C19" class="link link-ref link-reveal xref-bibr">19</a></sup> It has been suggested that organisms that have undergone massive reductions in their genome acquired a low G + C content and have also had degradation of their regulatory signals.<sup><a reveal-id="DSR045C32" data-open="DSR045C32" class="link link-ref link-reveal xref-bibr">32</a></sup></p><p>Previous studies have demonstrated that transcription can occur when only the −10 element is present, although additional elements, including activator proteins and extended −10 elements (the −16 element), may be involved.<sup><a reveal-id="DSR045C32" data-open="DSR045C32" class="link link-ref link-reveal xref-bibr">32</a></sup> Forty-eight per cent of the experimentally characterized <em>M. hyopneumoniae</em> promoters contained the −16 element. This proportion is very similar to that found in <em>B. subtilis</em>, in which ∼45% of promoters possess this element.<sup><a reveal-id="DSR045C33" data-open="DSR045C33" class="link link-ref link-reveal xref-bibr">33</a></sup> Studies suggest that the extended elements compensate for the lack of conservation in the −10 and −35 boxes of the promoters.<sup><a reveal-id="DSR045C34" data-open="DSR045C34" class="link link-ref link-reveal xref-bibr">34</a></sup> The −16 elements are also found in promoters of other species, including <em>E. coli</em> and <em>C. jejuni</em>, but are not seen in <em>M. pneumoniae</em> promoters.<sup><a reveal-id="DSR045C8" data-open="DSR045C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSR045C35" data-open="DSR045C35" class="link link-ref link-reveal xref-bibr">35</a></sup></p><p>The AT-rich stretches upstream of the −10 element in the promoters of <em>M. hyopneumoniae</em> and <em>C. jejuni</em> may result in transcriptional enhancement. Petersen <em>et al</em>.<sup><a reveal-id="DSR045C36" data-open="DSR045C36" class="link link-ref link-reveal xref-bibr">36</a></sup> suggested that they could play a role as specific binding sites or be implicated in DNA curvature. These stretches could also be related to upstream (UP) elements, which can affect promoter recognition and activity.<sup><a reveal-id="DSR045C37" data-open="DSR045C37" class="link link-ref link-reveal xref-bibr">37</a></sup> UP elements are AT-rich sequences, typically located in a region from nt −40 to nt −60 (relative to the TSS) that interacts with the C-terminal domain of the α-subunits of RNA polymerase.<sup><a reveal-id="DSR045C37" data-open="DSR045C37" class="link link-ref link-reveal xref-bibr">37</a></sup> They have been identified in several bacterial species, and their occurrence increases as the genomic G + C content of the organisms decreases.<sup><a reveal-id="DSR045C38" data-open="DSR045C38" class="link link-ref link-reveal xref-bibr">38</a></sup> UP elements can improve the activity of a TGn/−10 promoter in the absence of a good −35 element,<sup><a reveal-id="DSR045C39" data-open="DSR045C39" class="link link-ref link-reveal xref-bibr">39</a></sup> and promoters comprising only UP and −10 elements can be recognized by RNA polymerase.<sup><a reveal-id="DSR045C40" data-open="DSR045C40" class="link link-ref link-reveal xref-bibr">40</a></sup> Thus, in AT-rich organisms, such as <em>M. hyopneumoniae</em>, it is likely that the AT-rich stretches act as UP elements, which may lessen the requirement for −35 hexamers.</p><p>While the conservation of the −10 element in both <em>Mycoplasma</em> species is evident, the <em>M. hyopneumoniae</em> promoters are particularly similar to those of <em>C. jejuni</em>. Besides lacking the −35 signal and possessing the extended −10 element, they also have periodic AT-rich stretches upstream of the −10 region. Since <em>M. hyopneumoniae</em> (Tenericutes) and <em>C. jejuni</em> (Proteobacteria) are phylogenetically distant, their promoter similarities suggest evolutionary convergence, which could be consequence of their high genomic A + T content (∼70%).</p><p>PSSMs have been widely used to find conserved motifs.<sup><a reveal-id="DSR045C27" data-open="DSR045C27" class="link link-ref link-reveal xref-bibr">27</a></sup> A PSSM was defined based on the experimentally defined promoters in order to detect promoter-like sequences along the intergenic regions of the <em>M. hyopneumoniae</em> genome.</p><p>The promoter scan of <em>M. hyopneumoniae</em> sequences found that the pattern detected by the matrix occurred more frequently than expected, indicating that it did not occur by chance and that it was probably functional in initiating gene transcription. Recent studies based on <em>E. coli</em> σ<sup>70</sup> promoter data were not able to detect these patterns in <em>Mycoplasma</em> genomes,<sup><a reveal-id="DSR045C32" data-open="DSR045C32" class="link link-ref link-reveal xref-bibr">32</a>,<a reveal-id="DSR045C41" data-open="DSR045C41" class="link link-ref link-reveal xref-bibr">41</a></sup> even suggesting that the existence of promoters in these bacteria was debatable.<sup><a reveal-id="DSR045C41" data-open="DSR045C41" class="link link-ref link-reveal xref-bibr">41</a></sup> However, as demonstrated by Weiner <em>et al</em>.,<sup><a reveal-id="DSR045C8" data-open="DSR045C8" class="link link-ref link-reveal xref-bibr">8</a></sup> the identification of <em>Mycoplasma</em> promoters using an <em>E. coli</em> matrix is not efficient. Our study has improved on these previous studies by using a species-specific PSSM that accounted for the variability between bacterial species, avoiding biases that might result from using heterologous PSSMs.</p><p>Approximately 26% of the CDSs in the <em>M. hyopneumoniae</em> genome had at least one identifiable promoter in their upstream region. However, many of the upstream sequences of the CDSs were too short to contain a promoter sequence of 12 nucleotides and a spacer of four nucleotides preceding the TSS. Therefore, the coverage of CDSs that could contain a promoter was greater than estimated. Adams <em>et al</em>.<sup><a reveal-id="DSR045C42" data-open="DSR045C42" class="link link-ref link-reveal xref-bibr">42</a></sup> have suggested that the upper limits of the intergenic regions in the <em>M. hyopneumoniae</em> operons is ∼50 bases, and studies have shown that genes that are organized in tandem with intergenic distances much larger than 50 bases can be transcribed in large transcriptional units.<sup><a reveal-id="DSR045C43" data-open="DSR045C43" class="link link-ref link-reveal xref-bibr">43</a></sup> These findings indicate that many CDSs are regulated by common promoters, and therefore that not all CDSs necessarily have a promoter in their adjacent upstream regions.</p><p>Intergenic regions between divergently oriented genes are the most probable sites to find promoter-like sequences. Our analyses indicated that 54% of the genes with this organization had at least one promoter signal. In contrast, of the 515 genes oriented in tandem, only 93 (18%) had a promoter sequence upstream. The relatively small proportion of in tandem CDSs that possessed promoters was probably attributable to the organization of most of these genes in transcriptional units and, therefore, their transcription might be driven by promoters that are not in the nearest upstream intergenic region.</p><p>Although experimental studies have detected the presence of large transcripts in <em>M. hyopneumoniae</em>, which could be transcribed from the promoter upstream of the first CDS of the transcriptional unit, our study demonstrates that many internal CDSs may also contain putative promoters. For instance, in the experimentally defined transcriptional unit containing the genes <em>deoC</em>, <em>upp</em>, MHP7448_0525, <em>lon</em> and <em>tuf</em>,<sup><a reveal-id="DSR045C44" data-open="DSR045C44" class="link link-ref link-reveal xref-bibr">44</a></sup> all the genes, except MHP7448_0525, contain promoter sequences in their upstream regions (with scores varying from 8.4 to 11) (data not shown). This example corroborates the findings of Gardner <em>et al</em>.,<sup><a reveal-id="DSR045C43" data-open="DSR045C43" class="link link-ref link-reveal xref-bibr">43</a></sup> who demonstrated that, even when transcription does not cease between genes, there is evidence of independent transcriptional initiation by the promoter of the following gene.</p><p>Most of the CDSs had a single promoter sequence (84%), but CDSs with multiple promoter sequences were also detected. The <em>tuf</em> gene, for example, which is known to be highly expressed, possessed three promoters in its upstream region, two of which overlapped (data not shown). Overlapping signals could promote transcription by recruiting RNA polymerases to the primary promoter sequence.<sup><a reveal-id="DSR045C45" data-open="DSR045C45" class="link link-ref link-reveal xref-bibr">45</a></sup> In the absence of a strong promoter, overlapping sites could be noncompetitive weak promoters that could produce basal transcription of the downstream genes. On the other hand, they could also negatively regulate transcription through competition between RNA polymerases,<sup><a reveal-id="DSR045C46" data-open="DSR045C46" class="link link-ref link-reveal xref-bibr">46</a></sup> or through the induction of a pause in the early steps in elongation.<sup><a reveal-id="DSR045C47" data-open="DSR045C47" class="link link-ref link-reveal xref-bibr">47</a>,<a reveal-id="DSR045C48" data-open="DSR045C48" class="link link-ref link-reveal xref-bibr">48</a></sup></p><p>The majority of the putative promoters were found between 1 and 100 bases upstream of the start codon. This is congruent with many previous studies performed in different bacterial species.<sup><a reveal-id="DSR045C36" data-open="DSR045C36" class="link link-ref link-reveal xref-bibr">36</a></sup> Some predicted promoter sequences were found within CDSs. This could be because the start codons of these genes were not assigned correctly, or because these putative intragenic signals have an unknown regulatory function.</p><p>Although a comprehensive prediction of promoters was performed in this study, many putative signals were not detected using the criteria used for prediction. The main restraint was the threshold score of 6.5. Approximately 30% of the promoters defined experimentally in our study were not detected using this cut-off value. Even the <em>recA</em> promoter was not detected using these criteria. The lowest score for the experimentally defined promoters was 4.2; however, at this threshold, about half of the sequences identified were estimated to be false-positives. There are many promoter-like sequences in the genome with scores &gt;4.2 (dCDF = 3.46 × 10<sup>−3</sup>), raising the question of how RNA polymerase distinguishes the signals of true promoters from the false-positives. As <em>M. hyopneumoniae</em> only has a small number of known regulatory proteins,<sup><a reveal-id="DSR045C2" data-open="DSR045C2" class="link link-ref link-reveal xref-bibr">2</a></sup> one might speculate that most of the sequences that score &gt;4.2 are true promoters. Gardner <em>et al</em>.<sup><a reveal-id="DSR045C43" data-open="DSR045C43" class="link link-ref link-reveal xref-bibr">43</a></sup> found that there is transcription across the majority of the intergenic regions in <em>M. hyopneumoniae</em>. However, studies have demonstrated that this species is able to control transcription;<sup><a reveal-id="DSR045C49  DSR045C53" data-open="DSR045C49  DSR045C53" class="link link-ref link-reveal xref-bibr">49–53</a></sup> therefore, the sequence contexts in which the signals are immersed may be a determinant of transcriptional initiation.</p><p>In summary, our study has contributed to understanding of transcriptional regulation in <em>M. hyopneumoniae</em>, as it has identified basic elements involved in transcriptional initiation and verified their distribution in the upstream regions of protein-coding genes in this species. Possible applications for the PSSM defined in this study would be refinement of genome annotations and investigation of promoters in closely related species, such as <em>Mycoplasma hyorhinis</em> and <em>Mycoplasma flocculare</em>.</p>                            <h2 scrollto-destination=6199010 id="6199010" class="section-title" data-legacy-id=s5>Supplementary Data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Data are available at www.dnaresearch.oxfordjournals.org.</a></p>                            <h2 scrollto-destination=6199012 id="6199012" class="section-title" data-legacy-id=s6>Funding</h2>
<p>This work was supported by grants from the Brazilian National Research Council, the Fundação de Amparo à Pesquisa do Rio Grande do Sul and the Coordenação de Aperfeiçoamento de Pessoal de Nível Superior.</p>                            <h2 scrollto-destination=6199014 id="6199014" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We especially thank Professor Augusto Schrank for valuable suggestions and Alejandra Medina-Rivera for all support with the Matrix-Quality program. We thank Professor Arnaldo Zaha for revising the manuscript. We thank Franciele Maboni Siqueira for supplying her unpublished experimental data. We also thank Bianca Gervini Fávero Bittencourt for the <em>M. hyopneumoniae</em> cultures.</p>                            <h2 scrollto-destination=6199016 id="6199016" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSR045C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR045C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Thacker</div> <div class="given-names">E.L.</div></div>. </div><div class="person-group"><div class="name"><div class="surname">Straw</div> <div class="given-names">B.E.</div></div>,  <div class="name"><div class="surname">Zimmermann</div> <div class="given-names">J.J.</div></div>,  <div class="name"><div class="surname">D'Allaire</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Taylor</div> <div class="given-names">D.J.</div></div>. </div>, <div class="source">Diseases for Swine</div> , <div class="year">2006</div><div class="publisher-loc">Ames</div><div class="publisher-name">Iowa State University Press</div>(pg. <div class="fpage">701</div>-<div class="lpage">17</div>)</div></div></div></div><div content-id="DSR045C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR045C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Minion</div> <div class="given-names">F.C.</div></div>,  <div class="name"><div class="surname">Lefkowitz</div> <div class="given-names">E.J.</div></div>,  <div class="name"><div class="surname">Madsen</div> <div class="given-names">M.L.</div></div>,  <div class="name"><div class="surname">Cleary</div> <div class="given-names">B.J.</div></div>,  <div class="name"><div class="surname">Swartzell</div> <div class="given-names">S.M.</div></div>,  <div class="name"><div class="surname">Mahairas</div> <div class="given-names">G.G.</div></div>. </div><div class="article-title">The genome sequence of <em>Mycoplasma hyopneumoniae</em> strain 232, the agent of swine mycoplasmosis</div>, <div class="source">J. Bacteriol.</div> , <div class="year">2004</div>, vol. <div class="volume">186</div> (pg. <div class="fpage">7123</div>-<div class="lpage">33</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20sequence%20of%20Mycoplasma%20hyopneumoniae%20strain%20232%2C%20the%20agent%20of%20swine%20mycoplasmosis&amp;author=F.C.%20Minion&amp;author=E.J.%20Lefkowitz&amp;author=M.L.%20Madsen&amp;author=B.J.%20Cleary&amp;author=S.M.%20Swartzell&amp;author=G.G.%20Mahairas&amp;publication_year=2004&amp;journal=J.%20Bacteriol.&amp;volume=186&amp;pages=7123-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.186.21.7123-7133.2004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.186.21.7123-7133.2004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.186.21.7123-7133.2004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15489423" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR045C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR045C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Vasconcelos</div> <div class="given-names">A.T.</div></div>,  <div class="name"><div class="surname">Ferreira</div> <div class="given-names">H.B.</div></div>,  <div class="name"><div class="surname">Bizarro</div> <div class="given-names">C.V.</div></div>, et al. </div><div class="article-title">Swine and poultry pathogens: the complete genome sequences of two strains of <em>Mycoplasma hyopneumoniae</em> and a strain of <em>Mycoplasma synoviae</em></div>, <div class="source">J. Bacteriol.</div> , <div class="year">2005</div>, vol. <div class="volume">187</div> (pg. <div class="fpage">5568</div>-<div class="lpage">77</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Swine%20and%20poultry%20pathogens%3A%20the%20complete%20genome%20sequences%20of%20two%20strains%20of%20Mycoplasma%20hyopneumoniae%20and%20a%20strain%20of%20Mycoplasma%20synoviae&amp;author=A.T.%20Vasconcelos&amp;author=H.B.%20Ferreira&amp;author=C.V.%20Bizarro&amp;publication_year=2005&amp;journal=J.%20Bacteriol.&amp;volume=187&amp;pages=5568-77" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.187.16.5568-5577.2005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.187.16.5568-5577.2005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.187.16.5568-5577.2005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16077101" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR045C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR045C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Liu</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Feng</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Fang</div> <div class="given-names">L.</div></div>, et al. </div><div class="article-title">Complete genome sequence of <em>Mycoplasma hyopneumoniae</em> strain 168</div>, <div class="source">J. Bacteriol.</div> , <div class="year">2011</div>, vol. <div class="volume">193</div> (pg. <div class="fpage">1016</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complete%20genome%20sequence%20of%20Mycoplasma%20hyopneumoniae%20strain%20168&amp;author=W.%20Liu&amp;author=Z.%20Feng&amp;author=L.%20Fang&amp;publication_year=2011&amp;journal=J.%20Bacteriol.&amp;volume=193&amp;pages=1016-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.01305-10" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.01305-10" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.01305-10"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21148737" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR045C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR045C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fraser</div> <div class="given-names">C.M.</div></div>,  <div class="name"><div class="surname">Gocayne</div> <div class="given-names">J.D.</div></div>,  <div class="name"><div class="surname">White</div> <div class="given-names">O.</div></div>, et al. </div><div class="article-title">The minimal gene complement of <em>Mycoplasma genitalium</em></div>, <div class="source">Science</div> , <div class="year">1995</div>, vol. <div class="volume">270</div> (pg. <div class="fpage">397</div>-<div class="lpage">403</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20minimal%20gene%20complement%20of%20Mycoplasma%20genitalium&amp;author=C.M.%20Fraser&amp;author=J.D.%20Gocayne&amp;author=O.%20White&amp;publication_year=1995&amp;journal=Science&amp;volume=270&amp;pages=397-403" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.270.5235.397" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.270.5235.397" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.270.5235.397"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7569993" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR045C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR045C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Blattner</div> <div class="given-names">F.R.</div></div>,  <div class="name"><div class="surname">Plunkett</div> <div class="given-names">G.</div>III</div>,  <div class="name"><div class="surname">Bloch</div> <div class="given-names">C.A.</div></div>, et al. </div><div class="article-title">The complete genome sequence of <em>Escherichia coli</em> K-12</div>, <div class="source">Science</div> , <div class="year">1997</div>, vol. <div class="volume">277</div> (pg. <div class="fpage">1453</div>-<div class="lpage">74</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20complete%20genome%20sequence%20of%20Escherichia%20coli%20K-12&amp;author=F.R.%20Blattner&amp;author=G.%20Plunkett&amp;author=C.A.%20Bloch&amp;publication_year=1997&amp;journal=Science&amp;volume=277&amp;pages=1453-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.277.5331.1453" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.277.5331.1453" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.277.5331.1453"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9278503" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR045C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR045C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kunst</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Ogasawara</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Moszer</div> <div class="given-names">I.</div></div>, et al. </div><div class="article-title">The complete genome sequence of the gram-positive bacterium <em>Bacillus subtilis</em></div>, <div class="source">Nature</div> , <div class="year">1997</div>, vol. <div class="volume">390</div> (pg. <div class="fpage">249</div>-<div class="lpage">56</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20complete%20genome%20sequence%20of%20the%20gram-positive%20bacterium%20Bacillus%20subtilis&amp;author=F.%20Kunst&amp;author=N.%20Ogasawara&amp;author=I.%20Moszer&amp;publication_year=1997&amp;journal=Nature&amp;volume=390&amp;pages=249-56" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/36786" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F36786" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F36786"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9384377" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR045C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR045C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Weiner</div> <div class="given-names">J.</div>III</div>,  <div class="name"><div class="surname">Herrmann</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Browning</div> <div class="given-names">G.F.</div></div>. </div><div class="article-title">Transcription in <em>Mycoplasma pneumoniae</em></div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2000</div>, vol. <div class="volume">28</div> (pg. <div class="fpage">4488</div>-<div class="lpage">96</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcription%20in%20Mycoplasma%20pneumoniae&amp;author=J.%20Weiner&amp;author=R.%20Herrmann&amp;author=G.F.%20Browning&amp;publication_year=2000&amp;journal=Nucleic%20Acids%20Res.&amp;volume=28&amp;pages=4488-96" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/28.22.4488" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F28.22.4488" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F28.22.4488"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11071937" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR045C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR045C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Friis</div> <div class="given-names">N.F.</div></div>. </div><div class="article-title">Some recommendations concerning primary isolation of <em>Mycoplasma suipneumoniae</em> and <em>Mycoplasma flocculare</em> a survey</div>, <div class="source">Nord. Vet. Med.</div> , <div class="year">1975</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">337</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Some%20recommendations%20concerning%20primary%20isolation%20of%20Mycoplasma%20suipneumoniae%20and%20Mycoplasma%20flocculare%20a%20survey&amp;author=N.F.%20Friis&amp;publication_year=1975&amp;journal=Nord.%20Vet.%20Med.&amp;volume=27&amp;pages=337-9" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1098011" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR045C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR045C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sambrook</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Russell</div> <div class="given-names">D.W.</div></div>. </div>, <div class="source">Molecular Cloning: A Laboratory Manual</div> , <div class="year">2001</div><div class="publisher-loc">Cold Spring Harbor, NY</div><div class="publisher-name">Cold Spring Harbor Laboratory Press</div></div></div></div></div><div content-id="DSR045C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR045C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bensing</div> <div class="given-names">B.A.</div></div>,  <div class="name"><div class="surname">Meyer</div> <div class="given-names">B.J.</div></div>,  <div class="name"><div class="surname">Dunny</div> <div class="given-names">G.M.</div></div>. </div><div class="article-title">Sensitive detection of bacterial transcription initiation sites and differentiation from RNA processing sites in the pheromone-induced plasmid transfer system of <em>Enterococcus faecalis</em></div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">1996</div>, vol. <div class="volume">93</div> (pg. <div class="fpage">7794</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sensitive%20detection%20of%20bacterial%20transcription%20initiation%20sites%20and%20differentiation%20from%20RNA%20processing%20sites%20in%20the%20pheromone-induced%20plasmid%20transfer%20system%20of%20Enterococcus%20faecalis&amp;author=B.A.%20Bensing&amp;author=B.J.%20Meyer&amp;author=G.M.%20Dunny&amp;publication_year=1996&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=93&amp;pages=7794-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.93.15.7794" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.93.15.7794" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.93.15.7794"> </span></div> </div></div></div></div></div><div content-id="DSR045C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR045C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schneider</div> <div class="given-names">T.D.</div></div>,  <div class="name"><div class="surname">Stephens</div> <div class="given-names">R.M.</div></div>. </div><div class="article-title">Sequence logos: a new way to display consensus sequences</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1990</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">6097</div>-<div class="lpage">100</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequence%20logos%3A%20a%20new%20way%20to%20display%20consensus%20sequences&amp;author=T.D.%20Schneider&amp;author=R.M.%20Stephens&amp;publication_year=1990&amp;journal=Nucleic%20Acids%20Res.&amp;volume=18&amp;pages=6097-100" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/18.20.6097" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F18.20.6097" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F18.20.6097"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2172928" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR045C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR045C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Crooks</div> <div class="given-names">G.E.</div></div>,  <div class="name"><div class="surname">Hon</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Chandonia</div> <div class="given-names">J.M.</div></div>,  <div class="name"><div class="surname">Brenner</div> <div class="given-names">S.E.</div></div>. </div><div class="article-title">WebLogo: a sequence logo generator</div>, <div class="source">Genome Res.</div> , <div class="year">2004</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">1188</div>-<div class="lpage">90</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=WebLogo%3A%20a%20sequence%20logo%20generator&amp;author=G.E.%20Crooks&amp;author=G.%20Hon&amp;author=J.M.%20Chandonia&amp;author=S.E.%20Brenner&amp;publication_year=2004&amp;journal=Genome%20Res.&amp;volume=14&amp;pages=1188-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.849004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.849004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.849004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15173120" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR045C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR045C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">MacLellan</div> <div class="given-names">S.R.</div></div>,  <div class="name"><div class="surname">MacLean</div> <div class="given-names">A.M.</div></div>,  <div class="name"><div class="surname">Finan</div> <div class="given-names">T.M.</div></div>. </div><div class="article-title">Promoter prediction in the rhizobia</div>, <div class="source">Microbiology</div> , <div class="year">2006</div>, vol. <div class="volume">152</div> (pg. <div class="fpage">1751</div>-<div class="lpage">63</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Promoter%20prediction%20in%20the%20rhizobia&amp;author=S.R.%20MacLellan&amp;author=A.M.%20MacLean&amp;author=T.M.%20Finan&amp;publication_year=2006&amp;journal=Microbiology&amp;volume=152&amp;pages=1751-63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1099/mic.0.28743-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1099%2Fmic.0.28743-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1099%2Fmic.0.28743-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16735738" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR045C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR045C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hawley</div> <div class="given-names">D.K.</div></div>,  <div class="name"><div class="surname">McClure</div> <div class="given-names">W.R.</div></div>. </div><div class="article-title">Compilation and analysis of <em>Escherichia coli</em> promoter DNA sequences</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1983</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">2237</div>-<div class="lpage">55</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Compilation%20and%20analysis%20of%20Escherichia%20coli%20promoter%20DNA%20sequences&amp;author=D.K.%20Hawley&amp;author=W.R.%20McClure&amp;publication_year=1983&amp;journal=Nucleic%20Acids%20Res.&amp;volume=11&amp;pages=2237-55" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/11.8.2237" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F11.8.2237" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F11.8.2237"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6344016" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR045C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR045C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Helmann</div> <div class="given-names">J.D.</div></div>. </div><div class="article-title">Compilation and analysis of <em>Bacillus subtilis</em> sigma A-dependent promoter sequences: evidence for extended contact between RNA polymerase and upstream promoter DNA</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1995</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">2351</div>-<div class="lpage">60</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Compilation%20and%20analysis%20of%20Bacillus%20subtilis%20sigma%20A-dependent%20promoter%20sequences%3A%20evidence%20for%20extended%20contact%20between%20RNA%20polymerase%20and%20upstream%20promoter%20DNA&amp;author=J.D.%20Helmann&amp;publication_year=1995&amp;journal=Nucleic%20Acids%20Res.&amp;volume=23&amp;pages=2351-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/23.13.2351" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F23.13.2351" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F23.13.2351"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7630711" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR045C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR045C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Grech</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Maetschke</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Mathews</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Timms</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Genome-wide analysis of Chlamydiae for promoters that phylogenetically footprint</div>, <div class="source">Res. Microbiol.</div> , <div class="year">2007</div>, vol. <div class="volume">158</div> (pg. <div class="fpage">685</div>-<div class="lpage">93</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20analysis%20of%20Chlamydiae%20for%20promoters%20that%20phylogenetically%20footprint&amp;author=B.%20Grech&amp;author=S.%20Maetschke&amp;author=S.%20Mathews&amp;author=P.%20Timms&amp;publication_year=2007&amp;journal=Res.%20Microbiol.&amp;volume=158&amp;pages=685-93" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.resmic.2007.08.005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.resmic.2007.08.005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.resmic.2007.08.005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18039561" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR045C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR045C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Vogel</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Axmann</div> <div class="given-names">I.M.</div></div>,  <div class="name"><div class="surname">Herzel</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Hess</div> <div class="given-names">W.R.</div></div>. </div><div class="article-title">Experimental and computational analysis of transcriptional start sites in the cyanobacterium <em>Prochlorococcus</em> MED4</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2003</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">2890</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Experimental%20and%20computational%20analysis%20of%20transcriptional%20start%20sites%20in%20the%20cyanobacterium%20Prochlorococcus%20MED4&amp;author=J.%20Vogel&amp;author=I.M.%20Axmann&amp;author=H.%20Herzel&amp;author=W.R.%20Hess&amp;publication_year=2003&amp;journal=Nucleic%20Acids%20Res.&amp;volume=31&amp;pages=2890-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkg398" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkg398" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkg398"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12771216" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR045C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR045C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wosten</div> <div class="given-names">M.M.</div></div>,  <div class="name"><div class="surname">Boeve</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Koot</div> <div class="given-names">M.G.</div></div>,  <div class="name"><div class="surname">van Nuenen</div> <div class="given-names">A.C.</div></div>,  <div class="name"><div class="surname">van der Zeijst</div> <div class="given-names">B.A.</div></div>. </div><div class="article-title">Identification of <em>Campylobacter jejuni</em> promoter sequences</div>, <div class="source">J. Bacteriol.</div> , <div class="year">1998</div>, vol. <div class="volume">180</div> (pg. <div class="fpage">594</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20Campylobacter%20jejuni%20promoter%20sequences&amp;author=M.M.%20Wosten&amp;author=M.%20Boeve&amp;author=M.G.%20Koot&amp;author=A.C.%20van%20Nuenen&amp;author=B.A.%20van%20der%20Zeijst&amp;publication_year=1998&amp;journal=J.%20Bacteriol.&amp;volume=180&amp;pages=594-9" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9457862" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR045C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR045C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Defrance</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Janky</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Sand</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">van</div> <div class="given-names">H.J.</div></div>. </div><div class="article-title">Using RSAT oligo-analysis and dyad-analysis tools to discover regulatory signals in nucleic sequences</div>, <div class="source">Nat. Protoc.</div> , <div class="year">2008</div>, vol. <div class="volume">3</div> (pg. <div class="fpage">1589</div>-<div class="lpage">603</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Using%20RSAT%20oligo-analysis%20and%20dyad-analysis%20tools%20to%20discover%20regulatory%20signals%20in%20nucleic%20sequences&amp;author=M.%20Defrance&amp;author=R.%20Janky&amp;author=O.%20Sand&amp;author=H.J.%20van&amp;publication_year=2008&amp;journal=Nat.%20Protoc.&amp;volume=3&amp;pages=1589-603" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nprot.2008.98" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnprot.2008.98" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnprot.2008.98"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18802440" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR045C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR045C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">van Helden</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Regulatory sequence analysis tools</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2003</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">3593</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Regulatory%20sequence%20analysis%20tools&amp;author=J.%20van%20Helden&amp;publication_year=2003&amp;journal=Nucleic%20Acids%20Res.&amp;volume=31&amp;pages=3593-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkg567" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkg567" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkg567"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12824373" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR045C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR045C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Thomas-Chollier</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Sand</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Turatsinze</div> <div class="given-names">J.V.</div></div>, et al. </div><div class="article-title">RSAT: regulatory sequence analysis tools</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2008</div>, vol. <div class="volume">36</div> (pg. <div class="fpage">W119</div>-<div class="lpage">27</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RSAT%3A%20regulatory%20sequence%20analysis%20tools&amp;author=M.%20Thomas-Chollier&amp;author=O.%20Sand&amp;author=J.V.%20Turatsinze&amp;publication_year=2008&amp;journal=Nucleic%20Acids%20Res.&amp;volume=36&amp;pages=W119-27" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkn304" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkn304" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkn304"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18495751" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR045C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR045C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hall</div> <div class="given-names">T.A.</div></div>. </div><div class="article-title">BioEdit: a user-friendly biological sequence alignment editor and analysis program for Windows 95/98/NT</div>, <div class="source">Nucleic Acids Symp. Ser.</div> , <div class="year">1999</div>, vol. <div class="volume">41</div> (pg. <div class="fpage">95</div>-<div class="lpage">8</div>)</div></div></div></div><div content-id="DSR045C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR045C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bailey</div> <div class="given-names">T.L.</div></div>,  <div class="name"><div class="surname">Elkan</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Fitting a mixture model by expectation maximization to discover motifs in biopolymers</div>, <div class="source">Proc. Int. Conf. Intell. Syst. Mol. Biol.</div> , <div class="year">1994</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">28</div>-<div class="lpage">36</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fitting%20a%20mixture%20model%20by%20expectation%20maximization%20to%20discover%20motifs%20in%20biopolymers&amp;author=T.L.%20Bailey&amp;author=C.%20Elkan&amp;publication_year=1994&amp;journal=Proc.%20Int.%20Conf.%20Intell.%20Syst.%20Mol.%20Biol.&amp;volume=2&amp;pages=28-36" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7584402" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR045C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR045C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hertz</div> <div class="given-names">G.Z.</div></div>,  <div class="name"><div class="surname">Stormo</div> <div class="given-names">G.D.</div></div>. </div><div class="article-title">Identifying DNA and protein patterns with statistically significant alignments of multiple sequences</div>, <div class="source">Bioinformatics</div> , <div class="year">1999</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">563</div>-<div class="lpage">77</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identifying%20DNA%20and%20protein%20patterns%20with%20statistically%20significant%20alignments%20of%20multiple%20sequences&amp;author=G.Z.%20Hertz&amp;author=G.D.%20Stormo&amp;publication_year=1999&amp;journal=Bioinformatics&amp;volume=15&amp;pages=563-77" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/15.7.563" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2F15.7.563" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2F15.7.563"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10487864" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR045C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR045C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Medina-Rivera</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">breu-Goodger</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Thomas-Chollier</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Salgado</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Collado-Vides</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">van</div> <div class="given-names">H.J.</div></div>. </div><div class="article-title">Theoretical and empirical quality assessment of transcription factor-binding motifs</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2011</div>, vol. <div class="volume">39</div> (pg. <div class="fpage">808</div>-<div class="lpage">24</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Theoretical%20and%20empirical%20quality%20assessment%20of%20transcription%20factor-binding%20motifs&amp;author=A.%20Medina-Rivera&amp;author=C.%20breu-Goodger&amp;author=M.%20Thomas-Chollier&amp;author=H.%20Salgado&amp;author=J.%20Collado-Vides&amp;author=H.J.%20van&amp;publication_year=2011&amp;journal=Nucleic%20Acids%20Res.&amp;volume=39&amp;pages=808-24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkq710" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkq710" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkq710"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20923783" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR045C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR045C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Turatsinze</div> <div class="given-names">J.V.</div></div>,  <div class="name"><div class="surname">Thomas-Chollier</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Defrance</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">van</div> <div class="given-names">H.J.</div></div>. </div><div class="article-title">Using RSAT to scan genome sequences for transcription factor binding sites and cis-regulatory modules</div>, <div class="source">Nat. Protoc.</div> , <div class="year">2008</div>, vol. <div class="volume">3</div> (pg. <div class="fpage">1578</div>-<div class="lpage">88</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Using%20RSAT%20to%20scan%20genome%20sequences%20for%20transcription%20factor%20binding%20sites%20and%20cis-regulatory%20modules&amp;author=J.V.%20Turatsinze&amp;author=M.%20Thomas-Chollier&amp;author=M.%20Defrance&amp;author=H.J.%20van&amp;publication_year=2008&amp;journal=Nat.%20Protoc.&amp;volume=3&amp;pages=1578-88" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nprot.2008.97" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnprot.2008.97" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnprot.2008.97"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18802439" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR045C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR045C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Voskuil</div> <div class="given-names">M.I.</div></div>,  <div class="name"><div class="surname">Voepel</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Chambliss</div> <div class="given-names">G.H.</div></div>. </div><div class="article-title">The -16 region, a vital sequence for the utilization of a promoter in <em>Bacillus subtilis</em> and <em>Escherichia coli</em></div>, <div class="source">Mol. Microbiol.</div> , <div class="year">1995</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">271</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20-16%20region%2C%20a%20vital%20sequence%20for%20the%20utilization%20of%20a%20promoter%20in%20Bacillus%20subtilis%20and%20Escherichia%20coli&amp;author=M.I.%20Voskuil&amp;author=K.%20Voepel&amp;author=G.H.%20Chambliss&amp;publication_year=1995&amp;journal=Mol.%20Microbiol.&amp;volume=17&amp;pages=271-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mmi.1995.17.issue-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmmi.1995.17.issue-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmmi.1995.17.issue-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7494476" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR045C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR045C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Voskuil</div> <div class="given-names">M.I.</div></div>,  <div class="name"><div class="surname">Chambliss</div> <div class="given-names">G.H.</div></div>. </div><div class="article-title">The TRTGn motif stabilizes the transcription initiation open complex</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">2002</div>, vol. <div class="volume">322</div> (pg. <div class="fpage">521</div>-<div class="lpage">32</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20TRTGn%20motif%20stabilizes%20the%20transcription%20initiation%20open%20complex&amp;author=M.I.%20Voskuil&amp;author=G.H.%20Chambliss&amp;publication_year=2002&amp;journal=J.%20Mol.%20Biol.&amp;volume=322&amp;pages=521-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0022-2836(02)00802-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0022-2836(02)00802-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0022-2836(02)00802-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12225746" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR045C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR045C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cases</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Ussery</div> <div class="given-names">D.W.</div></div>,  <div class="name"><div class="surname">de Lorenzo</div> <div class="given-names">V.</div></div>. </div><div class="article-title">The sigma54 regulon (sigmulon) of <em>Pseudomonas putida</em></div>, <div class="source">Environ. Microbiol.</div> , <div class="year">2003</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">1281</div>-<div class="lpage">93</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20sigma54%20regulon%20(sigmulon)%20of%20Pseudomonas%20putida&amp;author=I.%20Cases&amp;author=D.W.%20Ussery&amp;author=V.%20de%20Lorenzo&amp;publication_year=2003&amp;journal=Environ.%20Microbiol.&amp;volume=5&amp;pages=1281-93" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1462-2920.2003.00528.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1462-2920.2003.00528.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1462-2920.2003.00528.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14641574" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR045C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR045C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Turnbough</div> <div class="given-names">C.L.</div>Jr.</div>. </div><div class="article-title">Regulation of gene expression by reiterative transcription</div>, <div class="source">Curr. Opin. Microbiol.</div> , <div class="year">2011</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">142</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Regulation%20of%20gene%20expression%20by%20reiterative%20transcription&amp;author=C.L.%20Turnbough&amp;publication_year=2011&amp;journal=Curr.%20Opin.%20Microbiol.&amp;volume=14&amp;pages=142-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.mib.2011.01.012" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.mib.2011.01.012" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.mib.2011.01.012"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21334966" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR045C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR045C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Huerta</div> <div class="given-names">A.M.</div></div>,  <div class="name"><div class="surname">Francino</div> <div class="given-names">M.P.</div></div>,  <div class="name"><div class="surname">Morett</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Collado-Vides</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Selection for unequal densities of sigma 70 promoter-like signals in different regions of large bacterial genomes</div>, <div class="source">PLoS Genet.</div> , <div class="year">2006</div>, vol. <div class="volume">2</div> pg. <div class="fpage">e185</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Selection%20for%20unequal%20densities%20of%20sigma%2070%20promoter-like%20signals%20in%20different%20regions%20of%20large%20bacterial%20genomes&amp;author=A.M.%20Huerta&amp;author=M.P.%20Francino&amp;author=E.%20Morett&amp;author=J.%20Collado-Vides&amp;publication_year=2006&amp;journal=PLoS%20Genet.&amp;volume=2&amp;pages=e185" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.0020185" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.0020185" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.0020185"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17096598" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR045C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR045C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jarmer</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Larsen</div> <div class="given-names">T.S.</div></div>,  <div class="name"><div class="surname">Krogh</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Saxild</div> <div class="given-names">H.H.</div></div>,  <div class="name"><div class="surname">Brunak</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Knudsen</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Sigma A recognition sites in the <em>Bacillus subtilis</em> genome</div>, <div class="source">Microbiology</div> , <div class="year">2001</div>, vol. <div class="volume">147</div> (pg. <div class="fpage">2417</div>-<div class="lpage">24</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sigma%20A%20recognition%20sites%20in%20the%20Bacillus%20subtilis%20genome&amp;author=H.%20Jarmer&amp;author=T.S.%20Larsen&amp;author=A.%20Krogh&amp;author=H.H.%20Saxild&amp;author=S.%20Brunak&amp;author=S.%20Knudsen&amp;publication_year=2001&amp;journal=Microbiology&amp;volume=147&amp;pages=2417-24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1099/00221287-147-9-2417" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1099%2F00221287-147-9-2417" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1099%2F00221287-147-9-2417"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11535782" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR045C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR045C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mitchell</div> <div class="given-names">J.E.</div></div>,  <div class="name"><div class="surname">Zheng</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Busby</div> <div class="given-names">S.J.</div></div>,  <div class="name"><div class="surname">Minchin</div> <div class="given-names">S.D.</div></div>. </div><div class="article-title">Identification and analysis of ‘extended -10’ promoters in <em>Escherichia coli</em></div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2003</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">4689</div>-<div class="lpage">95</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20and%20analysis%20of%20%E2%80%98extended%20-10%E2%80%99%20promoters%20in%20Escherichia%20coli&amp;author=J.E.%20Mitchell&amp;author=D.%20Zheng&amp;author=S.J.%20Busby&amp;author=S.D.%20Minchin&amp;publication_year=2003&amp;journal=Nucleic%20Acids%20Res.&amp;volume=31&amp;pages=4689-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkg694" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkg694" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkg694"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12907708" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR045C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR045C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Guell</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">van Noort</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Yus</div> <div class="given-names">E.</div></div>, et al. </div><div class="article-title">Transcriptome complexity in a genome-reduced bacterium</div>, <div class="source">Science</div> , <div class="year">2009</div>, vol. <div class="volume">326</div> (pg. <div class="fpage">1268</div>-<div class="lpage">71</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptome%20complexity%20in%20a%20genome-reduced%20bacterium&amp;author=M.%20Guell&amp;author=V.%20van%20Noort&amp;author=E.%20Yus&amp;publication_year=2009&amp;journal=Science&amp;volume=326&amp;pages=1268-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1176951" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1176951" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1176951"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19965477" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR045C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR045C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Petersen</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Larsen</div> <div class="given-names">T.S.</div></div>,  <div class="name"><div class="surname">Ussery</div> <div class="given-names">D.W.</div></div>,  <div class="name"><div class="surname">On</div> <div class="given-names">S.L.</div></div>,  <div class="name"><div class="surname">Krogh</div> <div class="given-names">A.</div></div>. </div><div class="article-title">RpoD promoters in <em>Campylobacter jejuni</em> exhibit a strong periodic signal instead of a -35 box</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">2003</div>, vol. <div class="volume">326</div> (pg. <div class="fpage">1361</div>-<div class="lpage">72</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RpoD%20promoters%20in%20Campylobacter%20jejuni%20exhibit%20a%20strong%20periodic%20signal%20instead%20of%20a%20-35%20box&amp;author=L.%20Petersen&amp;author=T.S.%20Larsen&amp;author=D.W.%20Ussery&amp;author=S.L.%20On&amp;author=A.%20Krogh&amp;publication_year=2003&amp;journal=J.%20Mol.%20Biol.&amp;volume=326&amp;pages=1361-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0022-2836(03)00034-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0022-2836(03)00034-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0022-2836(03)00034-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12595250" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR045C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR045C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hook-Barnard</div> <div class="given-names">I.G.</div></div>,  <div class="name"><div class="surname">Hinton</div> <div class="given-names">D.M.</div></div>. </div><div class="article-title">Transcription initiation by mix and match elements: flexibility for polymerase binding to bacterial promoters</div>, <div class="source">Gene Regul. Syst. Bio</div> , <div class="year">2007</div>, vol. <div class="volume">1</div> (pg. <div class="fpage">275</div>-<div class="lpage">93</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcription%20initiation%20by%20mix%20and%20match%20elements%3A%20flexibility%20for%20polymerase%20binding%20to%20bacterial%20promoters&amp;author=I.G.%20Hook-Barnard&amp;author=D.M.%20Hinton&amp;publication_year=2007&amp;journal=Gene%20Regul.%20Syst.%20Bio&amp;volume=1&amp;pages=275-93" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19119427" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR045C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR045C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dekhtyar</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Morin</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Sakanyan</div> <div class="given-names">V.</div></div>. </div><div class="article-title">Triad pattern algorithm for predicting strong promoter candidates in bacterial genomes</div>, <div class="source">BMC Bioinformatics</div> , <div class="year">2008</div>, vol. <div class="volume">9</div> pg. <div class="fpage">233</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Triad%20pattern%20algorithm%20for%20predicting%20strong%20promoter%20candidates%20in%20bacterial%20genomes&amp;author=M.%20Dekhtyar&amp;author=A.%20Morin&amp;author=V.%20Sakanyan&amp;publication_year=2008&amp;journal=BMC%20Bioinformatics&amp;volume=9&amp;pages=233" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-9-233" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-9-233" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-9-233"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18471287" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR045C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR045C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Miroslavova</div> <div class="given-names">N.S.</div></div>,  <div class="name"><div class="surname">Busby</div> <div class="given-names">S.J.</div></div>. </div><div class="article-title">Investigations of the modular structure of bacterial promoters</div>, <div class="source">Biochem. Soc. Symp.</div> , <div class="year">2006</div>, vol. <div class="volume">73</div> (pg. <div class="fpage">1</div>-<div class="lpage">10</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Investigations%20of%20the%20modular%20structure%20of%20bacterial%20promoters&amp;author=N.S.%20Miroslavova&amp;author=S.J.%20Busby&amp;publication_year=2006&amp;journal=Biochem.%20Soc.%20Symp.&amp;volume=73&amp;pages=1-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1042/bss0730001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1042%2Fbss0730001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1042%2Fbss0730001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16626282" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR045C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR045C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Orsini</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Igonet</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Pene</div> <div class="given-names">C.</div></div>, et al. </div><div class="article-title">Phage T4 early promoters are resistant to inhibition by the anti-sigma factor AsiA</div>, <div class="source">Mol. Microbiol</div> , <div class="year">2004</div>, vol. <div class="volume">52</div> (pg. <div class="fpage">1013</div>-<div class="lpage">28</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phage%20T4%20early%20promoters%20are%20resistant%20to%20inhibition%20by%20the%20anti-sigma%20factor%20AsiA&amp;author=G.%20Orsini&amp;author=S.%20Igonet&amp;author=C.%20Pene&amp;publication_year=2004&amp;journal=Mol.%20Microbiol&amp;volume=52&amp;pages=1013-28" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1365-2958.2004.04038.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1365-2958.2004.04038.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1365-2958.2004.04038.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15130121" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR045C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR045C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sinoquet</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Demey</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Braun</div> <div class="given-names">F.</div></div>. </div><div class="article-title">Large-scale computational and statistical analyses of high transcription potentialities in 32 prokaryotic genomes</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2008</div>, vol. <div class="volume">36</div> (pg. <div class="fpage">3332</div>-<div class="lpage">40</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Large-scale%20computational%20and%20statistical%20analyses%20of%20high%20transcription%20potentialities%20in%2032%20prokaryotic%20genomes&amp;author=C.%20Sinoquet&amp;author=S.%20Demey&amp;author=F.%20Braun&amp;publication_year=2008&amp;journal=Nucleic%20Acids%20Res.&amp;volume=36&amp;pages=3332-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkn135" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkn135" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkn135"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18440978" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR045C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR045C42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Adams</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Pitzer</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Minion</div> <div class="given-names">F.C.</div></div>. </div><div class="article-title">In vivo expression analysis of the P97 and P102 paralog families of <em>Mycoplasma hyopneumoniae</em></div>, <div class="source">Infect. Immun.</div> , <div class="year">2005</div>, vol. <div class="volume">73</div> (pg. <div class="fpage">7784</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=In%20vivo%20expression%20analysis%20of%20the%20P97%20and%20P102%20paralog%20families%20of%20Mycoplasma%20hyopneumoniae&amp;author=C.%20Adams&amp;author=J.%20Pitzer&amp;author=F.C.%20Minion&amp;publication_year=2005&amp;journal=Infect.%20Immun.&amp;volume=73&amp;pages=7784-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/IAI.73.11.7784-7787.2005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FIAI.73.11.7784-7787.2005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FIAI.73.11.7784-7787.2005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16239586" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR045C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR045C43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gardner</div> <div class="given-names">S.W.</div></div>,  <div class="name"><div class="surname">Minion</div> <div class="given-names">F.C.</div></div>. </div><div class="article-title">Detection and quantification of intergenic transcription in <em>Mycoplasma hyopneumoniae</em></div>, <div class="source">Microbiology</div> , <div class="year">2010</div>, vol. <div class="volume">156</div> (pg. <div class="fpage">2305</div>-<div class="lpage">15</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Detection%20and%20quantification%20of%20intergenic%20transcription%20in%20Mycoplasma%20hyopneumoniae&amp;author=S.W.%20Gardner&amp;author=F.C.%20Minion&amp;publication_year=2010&amp;journal=Microbiology&amp;volume=156&amp;pages=2305-15" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1099/mic.0.038760-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1099%2Fmic.0.038760-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1099%2Fmic.0.038760-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20430814" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR045C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR045C44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Siqueira</div> <div class="given-names">F.M.</div></div>,  <div class="name"><div class="surname">Schrank</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Schrank</div> <div class="given-names">I.S.</div></div>. </div><div class="article-title"><em>Mycoplasma hyopneumoniae</em> transcription unit organization: genome survey and prediction</div>, <div class="source">DNA Res.</div> , <div class="year">2011</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">413</div>-<div class="lpage">22</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mycoplasma%20hyopneumoniae%20transcription%20unit%20organization%3A%20genome%20survey%20and%20prediction&amp;author=F.M.%20Siqueira&amp;author=A.%20Schrank&amp;author=I.S.%20Schrank&amp;publication_year=2011&amp;journal=DNA%20Res.&amp;volume=18&amp;pages=413-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsr028" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsr028" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsr028"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22086999" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR045C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR045C45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Reznikoff</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Bertrand</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Donnelly</div> <div class="given-names">C.</div></div>. </div><div class="person-group"><div class="name"><div class="surname">Reznikoff</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Burgess</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Dahlberg</div> <div class="given-names">J.</div></div>, et al. </div>, <div class="source">RNA Polymerase and the Regulation of Transcription</div> , <div class="year">1987</div><div class="publisher-loc">New York</div><div class="publisher-name">Elsevier</div>(pg. <div class="fpage">105</div>-<div class="lpage">13</div>)</div></div></div></div><div content-id="DSR045C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR045C46" class="ref-content "><div class="label -label">46</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Goodrich</div> <div class="given-names">J.A.</div></div>,  <div class="name"><div class="surname">McClure</div> <div class="given-names">W.R.</div></div>. </div><div class="article-title">Competing promoters in prokaryotic transcription</div>, <div class="source">Trends Biochem. Sci.</div> , <div class="year">1991</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">394</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Competing%20promoters%20in%20prokaryotic%20transcription&amp;author=J.A.%20Goodrich&amp;author=W.R.%20McClure&amp;publication_year=1991&amp;journal=Trends%20Biochem.%20Sci.&amp;volume=16&amp;pages=394-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0968-0004(91)90162-O" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0968-0004(91)90162-O" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0968-0004(91)90162-O"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1776166" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR045C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR045C47" class="ref-content "><div class="label -label">47</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Brodolin</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Zenkin</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Mustaev</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Mamaeva</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Heumann</div> <div class="given-names">H.</div></div>. </div><div class="article-title">The sigma 70 subunit of RNA polymerase induces lacUV5 promoter-proximal pausing of transcription</div>, <div class="source">Nat. Struct. Mol. Biol.</div> , <div class="year">2004</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">551</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20sigma%2070%20subunit%20of%20RNA%20polymerase%20induces%20lacUV5%20promoter-proximal%20pausing%20of%20transcription&amp;author=K.%20Brodolin&amp;author=N.%20Zenkin&amp;author=A.%20Mustaev&amp;author=D.%20Mamaeva&amp;author=H.%20Heumann&amp;publication_year=2004&amp;journal=Nat.%20Struct.%20Mol.%20Biol.&amp;volume=11&amp;pages=551-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nsmb768" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnsmb768" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnsmb768"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15122346" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR045C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR045C48" class="ref-content "><div class="label -label">48</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nickels</div> <div class="given-names">B.E.</div></div>,  <div class="name"><div class="surname">Mukhopadhyay</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Garrity</div> <div class="given-names">S.J.</div></div>,  <div class="name"><div class="surname">Ebright</div> <div class="given-names">R.H.</div></div>,  <div class="name"><div class="surname">Hochschild</div> <div class="given-names">A.</div></div>. </div><div class="article-title">The sigma 70 subunit of RNA polymerase mediates a promoter-proximal pause at the lac promoter</div>, <div class="source">Nat. Struct. Mol. Biol.</div> , <div class="year">2004</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">544</div>-<div class="lpage">50</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20sigma%2070%20subunit%20of%20RNA%20polymerase%20mediates%20a%20promoter-proximal%20pause%20at%20the%20lac%20promoter&amp;author=B.E.%20Nickels&amp;author=J.%20Mukhopadhyay&amp;author=S.J.%20Garrity&amp;author=R.H.%20Ebright&amp;author=A.%20Hochschild&amp;publication_year=2004&amp;journal=Nat.%20Struct.%20Mol.%20Biol.&amp;volume=11&amp;pages=544-50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nsmb757" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnsmb757" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnsmb757"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15122345" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR045C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR045C49" class="ref-content "><div class="label -label">49</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Weiner</div> <div class="given-names">J.</div>III</div>,  <div class="name"><div class="surname">Zimmerman</div> <div class="given-names">C.U.</div></div>,  <div class="name"><div class="surname">Gohlmann</div> <div class="given-names">H.W.</div></div>,  <div class="name"><div class="surname">Herrmann</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Transcription profiles of the bacterium <em>Mycoplasma pneumoniae</em> grown at different temperatures</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2003</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">6306</div>-<div class="lpage">20</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcription%20profiles%20of%20the%20bacterium%20Mycoplasma%20pneumoniae%20grown%20at%20different%20temperatures&amp;author=J.%20Weiner&amp;author=C.U.%20Zimmerman&amp;author=H.W.%20Gohlmann&amp;author=R.%20Herrmann&amp;publication_year=2003&amp;journal=Nucleic%20Acids%20Res.&amp;volume=31&amp;pages=6306-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkg841" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkg841" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkg841"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14576319" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR045C50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR045C50" class="ref-content "><div class="label -label">50</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Madsen</div> <div class="given-names">M.L.</div></div>,  <div class="name"><div class="surname">Nettleton</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Thacker</div> <div class="given-names">E.L.</div></div>,  <div class="name"><div class="surname">Edwards</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Minion</div> <div class="given-names">F.C.</div></div>. </div><div class="article-title">Transcriptional profiling of <em>Mycoplasma hyopneumoniae</em> during heat shock using microarrays</div>, <div class="source">Infect. Immun.</div> , <div class="year">2006</div>, vol. <div class="volume">74</div> (pg. <div class="fpage">160</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptional%20profiling%20of%20Mycoplasma%20hyopneumoniae%20during%20heat%20shock%20using%20microarrays&amp;author=M.L.%20Madsen&amp;author=D.%20Nettleton&amp;author=E.L.%20Thacker&amp;author=R.%20Edwards&amp;author=F.C.%20Minion&amp;publication_year=2006&amp;journal=Infect.%20Immun.&amp;volume=74&amp;pages=160-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/IAI.74.1.160-166.2006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FIAI.74.1.160-166.2006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FIAI.74.1.160-166.2006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16368969" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR045C51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR045C51" class="ref-content "><div class="label -label">51</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Madsen</div> <div class="given-names">M.L.</div></div>,  <div class="name"><div class="surname">Nettleton</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Thacker</div> <div class="given-names">E.L.</div></div>,  <div class="name"><div class="surname">Minion</div> <div class="given-names">F.C.</div></div>. </div><div class="article-title">Transcriptional profiling of <em>Mycoplasma hyopneumoniae</em> during iron depletion using microarrays</div>, <div class="source">Microbiology</div> , <div class="year">2006</div>, vol. <div class="volume">152</div> (pg. <div class="fpage">937</div>-<div class="lpage">44</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptional%20profiling%20of%20Mycoplasma%20hyopneumoniae%20during%20iron%20depletion%20using%20microarrays&amp;author=M.L.%20Madsen&amp;author=D.%20Nettleton&amp;author=E.L.%20Thacker&amp;author=F.C.%20Minion&amp;publication_year=2006&amp;journal=Microbiology&amp;volume=152&amp;pages=937-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1099/mic.0.28674-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1099%2Fmic.0.28674-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1099%2Fmic.0.28674-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16549658" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR045C52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR045C52" class="ref-content "><div class="label -label">52</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schafer</div> <div class="given-names">E.R.</div></div>,  <div class="name"><div class="surname">Oneal</div> <div class="given-names">M.J.</div></div>,  <div class="name"><div class="surname">Madsen</div> <div class="given-names">M.L.</div></div>,  <div class="name"><div class="surname">Minion</div> <div class="given-names">F.C.</div></div>. </div><div class="article-title">Global transcriptional analysis of <em>Mycoplasma hyopneumoniae</em> following exposure to hydrogen peroxide</div>, <div class="source">Microbiology</div> , <div class="year">2007</div>, vol. <div class="volume">153</div> (pg. <div class="fpage">3785</div>-<div class="lpage">90</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Global%20transcriptional%20analysis%20of%20Mycoplasma%20hyopneumoniae%20following%20exposure%20to%20hydrogen%20peroxide&amp;author=E.R.%20Schafer&amp;author=M.J.%20Oneal&amp;author=M.L.%20Madsen&amp;author=F.C.%20Minion&amp;publication_year=2007&amp;journal=Microbiology&amp;volume=153&amp;pages=3785-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1099/mic.0.2007/011387-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1099%2Fmic.0.2007%2F011387-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1099%2Fmic.0.2007%2F011387-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17975087" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR045C53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR045C53" class="ref-content "><div class="label -label">53</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Oneal</div> <div class="given-names">M.J.</div></div>,  <div class="name"><div class="surname">Schafer</div> <div class="given-names">E.R.</div></div>,  <div class="name"><div class="surname">Madsen</div> <div class="given-names">M.L.</div></div>,  <div class="name"><div class="surname">Minion</div> <div class="given-names">F.C.</div></div>. </div><div class="article-title">Global transcriptional analysis of <em>Mycoplasma hyopneumoniae</em> following exposure to norepinephrine</div>, <div class="source">Microbiology</div> , <div class="year">2008</div>, vol. <div class="volume">154</div> (pg. <div class="fpage">2581</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Global%20transcriptional%20analysis%20of%20Mycoplasma%20hyopneumoniae%20following%20exposure%20to%20norepinephrine&amp;author=M.J.%20Oneal&amp;author=E.R.%20Schafer&amp;author=M.L.%20Madsen&amp;author=F.C.%20Minion&amp;publication_year=2008&amp;journal=Microbiology&amp;volume=154&amp;pages=2581-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1099/mic.0.2008/020230-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1099%2Fmic.0.2008%2F020230-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1099%2Fmic.0.2008%2F020230-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18757792" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2012. Published by Oxford University Press on behalf of Kazusa DNA Research Institute</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/3.0/), which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=418941&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dsr045/2/dsr045-dsr045supp.pdf?Expires=2147483647&amp;Signature=ioe7ZTUk1WK3dpb9hAqyftLziMEmia5GZU8h6QZQA1Nq6ZpVvyZ78aC6MNeM~ruwrnA8gDwVt5w2AL6RZF3r7IOx~O14eamF51nkbotCvUi1djnKGif2KSwCqemuYmfCCPxyr1YXVKoYmwcSJWXDAyoO5vFRnnji2LCsbG8GCxIDtGt6hqF~WMjOthdafxasMt3muSt7a5JOg4lUqiaQnTmIKpw07Eo0o2wmJheYlAJ3mONZMlCuNWfA56uL7QXVmdJWcx5qmmbJohyKFEVwjZMlBM661vbE22U0kgchEx8Fd~aZA2VnOKoh7guxVJo2BoGuiBsQ5HnJCgDCTzi~mw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">218</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000303290100001&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">10</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsr045"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="418941"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000303290100001&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f19%2f2%2f103%2f418941">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000303290100001&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (10)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F19%2F2%2F103%2F418941" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/418941'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Unveiling Mycoplasma hyopneumoniae Promoters: Sequence Definition and Genomic Distribution" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsr045");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f19%2f2%2f103%2f418941">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

