

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/24/3/303/2972686","@type":"ScholarlyArticle","name":"Ribo-seq enlightens codon usage bias","datePublished":"2017-02-07","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/24/3","@type":"PublicationIssue","issueNumber":"3","datePublished":"2017-06-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsw062","keywords":["codon usage","high throughput sequencing","synonymous codon","evolution","translation"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/24/3/303/2972686","author":[{"name":"Paulet, Damien","affiliation":"1 Department of Computer Science, Laboratoire d'Informatique, de Robotique et de Microélectronique de Montpellier (LIRMM), CNRS et Université de Montpellier, 34095 Montpellier Cedex 5, France   2 Institut de Biologie Computationnelle (IBC); Université de Montpellier, France","@type":"Person"},{"name":"David, Alexandre","affiliation":"3 Cancer Biology Department, Institut de Génomique Fonctionnelle, CNRS, INSERM, Université de Montpellier, F-34094 Montpellier, France","@type":"Person"},{"name":"Rivals, Eric","affiliation":"1 Department of Computer Science, Laboratoire d'Informatique, de Robotique et de Microélectronique de Montpellier (LIRMM), CNRS et Université de Montpellier, 34095 Montpellier Cedex 5, France   2 Institut de Biologie Computationnelle (IBC); Université de Montpellier, France","@type":"Person"}],"description":"Abstract.  Codon usage is biased between lowly and highly expressed genes in a genome-specific manner. This universal bias has been well assessed in some unicel","pageStart":"303","pageEnd":"210","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/3/1/m_dnares_24_3cover.png?Expires=2147483647&Signature=eoXc11KAMpHEsaqW0C6T2VF4hIzrR2AQ2xJtZ-I1SzuN8Ozcm3pfcXhSESBXVMgST8o0VjwPqT85KWv95Ep~BtVLeJis8VsXRFBVpMHhb~g9dvz54ejvU8kSReDsueZzPSdTwZuEswyhmEGVK~9fmtkzww9NnyhhDFzj14yR9QQPP3D5RCa40I2kb6S3cosdNYyeAUAcZmppIJGZquGgABfHKwfV6R3Ffo5Kcif39o76ejsFOOlH0X60OYvM4Yme1Vyjd2eLo4R5~a0mH9tf6MZMrQpFP4iVZGwKwAheMFv-ISdx3BcAb4Re6~N24GwZYpxXAywMLeRm98GmIDGpbg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Ribo-seq enlightens codon usage bias","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/3/1/m_dnares_24_3cover.png?Expires=2147483647&Signature=eoXc11KAMpHEsaqW0C6T2VF4hIzrR2AQ2xJtZ-I1SzuN8Ozcm3pfcXhSESBXVMgST8o0VjwPqT85KWv95Ep~BtVLeJis8VsXRFBVpMHhb~g9dvz54ejvU8kSReDsueZzPSdTwZuEswyhmEGVK~9fmtkzww9NnyhhDFzj14yR9QQPP3D5RCa40I2kb6S3cosdNYyeAUAcZmppIJGZquGgABfHKwfV6R3Ffo5Kcif39o76ejsFOOlH0X60OYvM4Yme1Vyjd2eLo4R5~a0mH9tf6MZMrQpFP4iVZGwKwAheMFv-ISdx3BcAb4Re6~N24GwZYpxXAywMLeRm98GmIDGpbg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Ribo-seq enlightens codon usage bias" />
    <meta property="og:description" content="Abstract.  Codon usage is biased between lowly and highly expressed genes in a genome-specific manner. This universal bias has been well assessed in some unicel" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/24/3/303/2972686" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2017-02-07" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Paulet, Damien" /><meta name="citation_author_institution" content="Department of Computer Science, Laboratoire d'Informatique, de Robotique et de Microélectronique de Montpellier (LIRMM), CNRS et Université de Montpellier, 34095 Montpellier Cedex 5, France" /><meta name="citation_author_institution" content="Institut de Biologie Computationnelle (IBC); Université de Montpellier, France" /><meta name="citation_author" content="David, Alexandre" /><meta name="citation_author_institution" content="Cancer Biology Department, Institut de Génomique Fonctionnelle, CNRS, INSERM, Université de Montpellier, F-34094 Montpellier, France" /><meta name="citation_author" content="Rivals, Eric" /><meta name="citation_author_institution" content="Department of Computer Science, Laboratoire d'Informatique, de Robotique et de Microélectronique de Montpellier (LIRMM), CNRS et Université de Montpellier, 34095 Montpellier Cedex 5, France" /><meta name="citation_author_institution" content="Institut de Biologie Computationnelle (IBC); Université de Montpellier, France" /><meta name="citation_title" content="Ribo-seq enlightens codon usage bias" /><meta name="citation_firstpage" content="303" /><meta name="citation_lastpage" content="210" /><meta name="citation_doi" content="10.1093/dnares/dsw062" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="24" /><meta name="citation_issue" content="3" /><meta name="citation_publication_date" content="2017/06/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Codon catalog usage and the genome hypothesis;  citation_journal_title=Nucleic Acids Res;  citation_year=1980;  citation_volume=8;  citation_pages=r49-62; " /><meta name="citation_reference" content="citation_title=Codon usage in bacteria: correlation with gene expressivity;  citation_journal_title=Nucleic Acids Res;  citation_year=1982;  citation_volume=10;  citation_pages=7055-74; " /><meta name="citation_reference" content="citation_title=Synonymous but not the same: the causes and consequences of codon bias;  citation_journal_title=Nat. Rev. Genet;  citation_year=2011;  citation_volume=12;  citation_pages=32-42; " /><meta name="citation_reference" content="citation_title=Codon bias as a means to fine-tune gene expression;  citation_journal_title=Mol. Cell;  citation_year=2015;  citation_volume=59;  citation_pages=149-61; " /><meta name="citation_reference" content="citation_title=Evolutionary conservation of codon optimality reveals hidden signatures of cotranslational folding,;  citation_journal_title=Nat. Struct. Mol. Biol;  citation_year=2013;  citation_volume=20;  citation_pages=237-43; " /><meta name="citation_reference" content="citation_title=Codon usage influences the local rate of translation elongation to regulate co-translational protein folding,;  citation_journal_title=Mol. Cell;  citation_year=2015;  citation_volume=59;  citation_pages=744-54; " /><meta name="citation_reference" content="citation_title=Codon usage in yeast: cluster analysis clearly differentiates highly and lowly expressed genes,;  citation_journal_title=Nucleic Acids Res;  citation_year=1986;  citation_volume=14;  citation_pages=5125-43; " /><meta name="citation_reference" content="citation_title=Genome-wide analysis in vivo of translation with nucleotide resolution using ribosome profiling,;  citation_journal_title=Science;  citation_year=2009;  citation_volume=324;  citation_pages=218-23; " /><meta name="citation_reference" content="citation_title=Recent advances of DNA sequencing via nanopore-based technologies,;  citation_journal_title=Sci. Bull;  citation_year=2014;  citation_volume=60;  citation_pages=287-95; " /><meta name="citation_reference" content="citation_title=Ribosome profiling reveals pervasive and regulated stop codon readthrough in Drosophila melanogaster,;  citation_journal_title=eLife;  citation_year=2013;  citation_volume=2;  citation_pages=e01179" /><meta name="citation_reference" content="citation_title=Detecting translational regulation by change point analysis of ribosome profiling data sets,;  citation_journal_title=RNA N. Y. N;  citation_year=2014;  citation_volume=20;  citation_pages=1507-18; " /><meta name="citation_reference" content="citation_title=Quantitative profiling of initiating ribosomes in vivo,;  citation_journal_title=Nat. Methods;  citation_year=2015;  citation_volume=12;  citation_pages=147-53; " /><meta name="citation_reference" content="citation_title=Improved ribosome-footprint and mrna measurements provide insights into dynamics and regulation of yeast translation,;  citation_journal_title=Cell Rep;  citation_year=2016;  citation_volume=14;  citation_pages=1787-99; " /><meta name="citation_reference" content="citation_publisher=Cold Spring Harbor Laboratory Press, Cold Spring Harbor (NY);  citation_title=C Elegans II;  citation_year=1997; " /><meta name="citation_reference" content="citation_title=CRAC: an integrated approach to the analysis of RNA-seq reads;  citation_journal_title=Genome Biol;  citation_year=2013;  citation_volume=14;  citation_pages=R30" /><meta name="citation_reference" content="citation_title=Wobble base-pairing slows in vivo translation elongation in metazoans,;  citation_journal_title=RNA;  citation_year=2011;  citation_volume=17;  citation_pages=2063-73; " /><meta name="citation_reference" content="citation_publisher=Sinauer Associates, Indiana University at Bloomington, ;  citation_title=The Origins of Genome Architecture;  citation_year=2007; " /><meta name="citation_reference" content="citation_title=tRNA gene number and codon usage in the C. elegans genome are co-adapted for optimal translation of highly expressed genes,;  citation_journal_title=Trends Genet;  citation_year=2000;  citation_volume=16;  citation_pages=287-9; " /><meta name="citation_reference" content="citation_title=GtRNAdb 2.0: an expanded database of transfer RNA genes identified in complete and draft genomes;  citation_journal_title=Nucleic Acids Res;  citation_year=2016;  citation_volume=44;  citation_pages=D184-9; " /><meta name="citation_reference" content="citation_title=Evolution of codon usage patterns: the extent and nature of divergence between Candida albicans and Saccharomyces cerevisiae,;  citation_journal_title=Nucleic Acids Res;  citation_year=1992;  citation_volume=20;  citation_pages=5289-95; " /><meta name="citation_reference" content="citation_title=Codon–anticodon pairing: the wobble hypothesis,;  citation_journal_title=J. Mol. Biol;  citation_year=1966;  citation_volume=19;  citation_pages=548-55; " /><meta name="citation_reference" content="citation_title=Global quantification of mammalian gene expression control,;  citation_journal_title=Nature;  citation_year=2011;  citation_volume=473;  citation_pages=337-42; " /><meta name="citation_reference" content="citation_title=Codon usage and tRNA content in unicellular and multicellular organisms,;  citation_journal_title=Mol. Biol. Evol;  citation_year=1985;  citation_volume=2;  citation_pages=13-34; " /><meta name="citation_reference" content="citation_title=Codon usage is an important determinant of gene expression levels largely through its effects on transcription;  citation_journal_title=Proc. Natl. Acad. Sci.;  citation_year=2016;  citation_volume=113;  citation_issue=41;  citation_pages=E6117-E6125; " /><meta name="citation_reference" content="citation_title=Adjacent codons act in concert to modulate translation efficiency in yeast;  citation_journal_title=Cell;  citation_year=2016;  citation_volume=166;  citation_pages=679-90; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/24/3/303/17743936/dsw062.pdf" /><meta name="description" content="Abstract.  Codon usage is biased between lowly and highly expressed genes in a genome-specific manner. This universal bias has been well assessed in some unicel" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/24/3/303/2972686" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/24/3/303/2972686" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/24/3/303/2972686");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsw062';
    </script>
<title>
	Ribo-seq enlightens codon usage bias | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./2972686" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BZFAPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJQYXVsZXQsIERhbWllbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQ29tcHV0ZXIgU2NpZW5jZSwgTGFib3JhdG9pcmUgZCdJbmZvcm1hdGlxdWUsIGRlIFJvYm90aXF1ZSBldCBkZSBNaWNyb8OpbGVjdHJvbmlxdWUgZGUgTW9udHBlbGxpZXIgKExJUk1NKSwgQ05SUyBldCBVbml2ZXJzaXTDqSBkZSBNb250cGVsbGllciwgMzQwOTUgTW9udHBlbGxpZXIgQ2VkZXggNSwgRnJhbmNlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSW5zdGl0dXQgZGUgQmlvbG9naWUgQ29tcHV0YXRpb25uZWxsZSAoSUJDKTsgVW5pdmVyc2l0w6kgZGUgTW9udHBlbGxpZXIsIEZyYW5jZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkRhdmlkLCBBbGV4YW5kcmUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJDYW5jZXIgQmlvbG9neSBEZXBhcnRtZW50LCBJbnN0aXR1dCBkZSBHw6lub21pcXVlIEZvbmN0aW9ubmVsbGUsIENOUlMsIElOU0VSTSwgVW5pdmVyc2l0w6kgZGUgTW9udHBlbGxpZXIsIEYtMzQwOTQgTW9udHBlbGxpZXIsIEZyYW5jZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlJpdmFscywgRXJpYyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQ29tcHV0ZXIgU2NpZW5jZSwgTGFib3JhdG9pcmUgZCdJbmZvcm1hdGlxdWUsIGRlIFJvYm90aXF1ZSBldCBkZSBNaWNyb8OpbGVjdHJvbmlxdWUgZGUgTW9udHBlbGxpZXIgKExJUk1NKSwgQ05SUyBldCBVbml2ZXJzaXTDqSBkZSBNb250cGVsbGllciwgMzQwOTUgTW9udHBlbGxpZXIgQ2VkZXggNSwgRnJhbmNlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSW5zdGl0dXQgZGUgQmlvbG9naWUgQ29tcHV0YXRpb25uZWxsZSAoSUJDKTsgVW5pdmVyc2l0w6kgZGUgTW9udHBlbGxpZXIsIEZyYW5jZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iUmliby1zZXEgZW5saWdodGVucyBjb2RvbiB1c2FnZSBiaWFzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iMzAzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSIyMTAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3cwNjIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMjQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxNy8wNi8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvZG9uIGNhdGFsb2cgdXNhZ2UgYW5kIHRoZSBnZW5vbWUgaHlwb3RoZXNpczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTE5ODA7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPXI0OS02MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29kb24gdXNhZ2UgaW4gYmFjdGVyaWE6IGNvcnJlbGF0aW9uIHdpdGggZ2VuZSBleHByZXNzaXZpdHk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0xOTgyOyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9NzA1NS03NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3lub255bW91cyBidXQgbm90IHRoZSBzYW1lOiB0aGUgY2F1c2VzIGFuZCBjb25zZXF1ZW5jZXMgb2YgY29kb24gYmlhczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBSZXYuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9MzItNDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvZG9uIGJpYXMgYXMgYSBtZWFucyB0byBmaW5lLXR1bmUgZ2VuZSBleHByZXNzaW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9NTk7ICBjaXRhdGlvbl9wYWdlcz0xNDktNjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2b2x1dGlvbmFyeSBjb25zZXJ2YXRpb24gb2YgY29kb24gb3B0aW1hbGl0eSByZXZlYWxzIGhpZGRlbiBzaWduYXR1cmVzIG9mIGNvdHJhbnNsYXRpb25hbCBmb2xkaW5nLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBTdHJ1Y3QuIE1vbC4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0yMDsgIGNpdGF0aW9uX3BhZ2VzPTIzNy00MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29kb24gdXNhZ2UgaW5mbHVlbmNlcyB0aGUgbG9jYWwgcmF0ZSBvZiB0cmFuc2xhdGlvbiBlbG9uZ2F0aW9uIHRvIHJlZ3VsYXRlIGNvLXRyYW5zbGF0aW9uYWwgcHJvdGVpbiBmb2xkaW5nLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBDZWxsOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTU5OyAgY2l0YXRpb25fcGFnZXM9NzQ0LTU0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db2RvbiB1c2FnZSBpbiB5ZWFzdDogY2x1c3RlciBhbmFseXNpcyBjbGVhcmx5IGRpZmZlcmVudGlhdGVzIGhpZ2hseSBhbmQgbG93bHkgZXhwcmVzc2VkIGdlbmVzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTE5ODY7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz01MTI1LTQzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBhbmFseXNpcyBpbiB2aXZvIG9mIHRyYW5zbGF0aW9uIHdpdGggbnVjbGVvdGlkZSByZXNvbHV0aW9uIHVzaW5nIHJpYm9zb21lIHByb2ZpbGluZyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MzI0OyAgY2l0YXRpb25fcGFnZXM9MjE4LTIzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZWNlbnQgYWR2YW5jZXMgb2YgRE5BIHNlcXVlbmNpbmcgdmlhIG5hbm9wb3JlLWJhc2VkIHRlY2hub2xvZ2llcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaS4gQnVsbDsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT02MDsgIGNpdGF0aW9uX3BhZ2VzPTI4Ny05NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Umlib3NvbWUgcHJvZmlsaW5nIHJldmVhbHMgcGVydmFzaXZlIGFuZCByZWd1bGF0ZWQgc3RvcCBjb2RvbiByZWFkdGhyb3VnaCBpbiBEcm9zb3BoaWxhIG1lbGFub2dhc3Rlciw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPWVMaWZlOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTI7ICBjaXRhdGlvbl9wYWdlcz1lMDExNzkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZXRlY3RpbmcgdHJhbnNsYXRpb25hbCByZWd1bGF0aW9uIGJ5IGNoYW5nZSBwb2ludCBhbmFseXNpcyBvZiByaWJvc29tZSBwcm9maWxpbmcgZGF0YSBzZXRzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Uk5BIE4uIFkuIE47ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MjA7ICBjaXRhdGlvbl9wYWdlcz0xNTA3LTE4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1RdWFudGl0YXRpdmUgcHJvZmlsaW5nIG9mIGluaXRpYXRpbmcgcmlib3NvbWVzIGluIHZpdm8sOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIE1ldGhvZHM7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz0xNDctNTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUltcHJvdmVkIHJpYm9zb21lLWZvb3RwcmludCBhbmQgbXJuYSBtZWFzdXJlbWVudHMgcHJvdmlkZSBpbnNpZ2h0cyBpbnRvIGR5bmFtaWNzIGFuZCByZWd1bGF0aW9uIG9mIHllYXN0IHRyYW5zbGF0aW9uLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q2VsbCBSZXA7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz0xNzg3LTk5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9wdWJsaXNoZXI9Q29sZCBTcHJpbmcgSGFyYm9yIExhYm9yYXRvcnkgUHJlc3MsIENvbGQgU3ByaW5nIEhhcmJvciAoTlkpOyAgY2l0YXRpb25fdGl0bGU9QyBFbGVnYW5zIElJOyAgY2l0YXRpb25feWVhcj0xOTk3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DUkFDOiBhbiBpbnRlZ3JhdGVkIGFwcHJvYWNoIHRvIHRoZSBhbmFseXNpcyBvZiBSTkEtc2VxIHJlYWRzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPVIzMCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVdvYmJsZSBiYXNlLXBhaXJpbmcgc2xvd3MgaW4gdml2byB0cmFuc2xhdGlvbiBlbG9uZ2F0aW9uIGluIG1ldGF6b2Fucyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVJOQTsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xNzsgIGNpdGF0aW9uX3BhZ2VzPTIwNjMtNzM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3B1Ymxpc2hlcj1TaW5hdWVyIEFzc29jaWF0ZXMsIEluZGlhbmEgVW5pdmVyc2l0eSBhdCBCbG9vbWluZ3RvbiwgOyAgY2l0YXRpb25fdGl0bGU9VGhlIE9yaWdpbnMgb2YgR2Vub21lIEFyY2hpdGVjdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9dFJOQSBnZW5lIG51bWJlciBhbmQgY29kb24gdXNhZ2UgaW4gdGhlIEMuIGVsZWdhbnMgZ2Vub21lIGFyZSBjby1hZGFwdGVkIGZvciBvcHRpbWFsIHRyYW5zbGF0aW9uIG9mIGhpZ2hseSBleHByZXNzZWQgZ2VuZXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UcmVuZHMgR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz0yODctOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R3RSTkFkYiAyLjA6IGFuIGV4cGFuZGVkIGRhdGFiYXNlIG9mIHRyYW5zZmVyIFJOQSBnZW5lcyBpZGVudGlmaWVkIGluIGNvbXBsZXRlIGFuZCBkcmFmdCBnZW5vbWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT00NDsgIGNpdGF0aW9uX3BhZ2VzPUQxODQtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXZvbHV0aW9uIG9mIGNvZG9uIHVzYWdlIHBhdHRlcm5zOiB0aGUgZXh0ZW50IGFuZCBuYXR1cmUgb2YgZGl2ZXJnZW5jZSBiZXR3ZWVuIENhbmRpZGEgYWxiaWNhbnMgYW5kIFNhY2NoYXJvbXljZXMgY2VyZXZpc2lhZSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0xOTkyOyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9NTI4OS05NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29kb27igJNhbnRpY29kb24gcGFpcmluZzogdGhlIHdvYmJsZSBoeXBvdGhlc2lzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gTW9sLiBCaW9sOyAgY2l0YXRpb25feWVhcj0xOTY2OyAgY2l0YXRpb25fdm9sdW1lPTE5OyAgY2l0YXRpb25fcGFnZXM9NTQ4LTU1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HbG9iYWwgcXVhbnRpZmljYXRpb24gb2YgbWFtbWFsaWFuIGdlbmUgZXhwcmVzc2lvbiBjb250cm9sLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTQ3MzsgIGNpdGF0aW9uX3BhZ2VzPTMzNy00MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29kb24gdXNhZ2UgYW5kIHRSTkEgY29udGVudCBpbiB1bmljZWxsdWxhciBhbmQgbXVsdGljZWxsdWxhciBvcmdhbmlzbXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEJpb2wuIEV2b2w7ICBjaXRhdGlvbl95ZWFyPTE5ODU7ICBjaXRhdGlvbl92b2x1bWU9MjsgIGNpdGF0aW9uX3BhZ2VzPTEzLTM0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db2RvbiB1c2FnZSBpcyBhbiBpbXBvcnRhbnQgZGV0ZXJtaW5hbnQgb2YgZ2VuZSBleHByZXNzaW9uIGxldmVscyBsYXJnZWx5IHRocm91Z2ggaXRzIGVmZmVjdHMgb24gdHJhbnNjcmlwdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bC4gQWNhZC4gU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT0xMTM7ICBjaXRhdGlvbl9pc3N1ZT00MTsgIGNpdGF0aW9uX3BhZ2VzPUU2MTE3LUU2MTI1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BZGphY2VudCBjb2RvbnMgYWN0IGluIGNvbmNlcnQgdG8gbW9kdWxhdGUgdHJhbnNsYXRpb24gZWZmaWNpZW5jeSBpbiB5ZWFzdDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q2VsbDsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT0xNjY7ICBjaXRhdGlvbl9wYWdlcz02NzktOTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8yNC8zLzMwMy8xNzc0MzkzNi9kc3cwNjIucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIENvZG9uIHVzYWdlIGlzIGJpYXNlZCBiZXR3ZWVuIGxvd2x5IGFuZCBoaWdobHkgZXhwcmVzc2VkIGdlbmVzIGluIGEgZ2Vub21lLXNwZWNpZmljIG1hbm5lci4gVGhpcyB1bml2ZXJzYWwgYmlhcyBoYXMgYmVlbiB3ZWxsIGFzc2Vzc2VkIGluIHNvbWUgdW5pY2VsIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFcy8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj0yOTcyNjg2JnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZCntssE0FFyAw3/HvuIIG0jchBiL" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f3%2f303%2f2972686" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f3%2f303%2f2972686" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=97549" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/24/3"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/3/1/m_dnares_24_3cover.png?Expires=2147483647&amp;Signature=eoXc11KAMpHEsaqW0C6T2VF4hIzrR2AQ2xJtZ-I1SzuN8Ozcm3pfcXhSESBXVMgST8o0VjwPqT85KWv95Ep~BtVLeJis8VsXRFBVpMHhb~g9dvz54ejvU8kSReDsueZzPSdTwZuEswyhmEGVK~9fmtkzww9NnyhhDFzj14yR9QQPP3D5RCa40I2kb6S3cosdNYyeAUAcZmppIJGZquGgABfHKwfV6R3Ffo5Kcif39o76ejsFOOlH0X60OYvM4Yme1Vyjd2eLo4R5~a0mH9tf6MZMrQpFP4iVZGwKwAheMFv-ISdx3BcAb4Re6~N24GwZYpxXAywMLeRm98GmIDGpbg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/24/3">
                        <div class="volume trailing-comma">Volume 24</div>
                        <div class="issue">Issue 3</div>

            </a>

        </div>
        <div class="ii-pub-date">
June 2017        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="88473278">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#88473278">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="88473280">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#88473280">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="88473284">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#88473284">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="88473296">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#88473296">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="88473318">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#88473318">4. Conclusion and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="88473324">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#88473324">Availability</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="88473326">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#88473326">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="88473334">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#88473334">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/24/3/289/2960056">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/24/3/311/2949522">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Ribo-seq enlightens codon usage bias
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Damien Paulet</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Department of Computer Science, Laboratoire d'Informatique, de Robotique et de Microélectronique de Montpellier (LIRMM), CNRS et Université de Montpellier, 34095 Montpellier Cedex 5, France</div><div class="aff"><div class="label -label">2</div>Institut de Biologie Computationnelle (IBC); Université de Montpellier, France</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Damien+Paulet">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Paulet D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Paulet D&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Damien Paulet</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Alexandre David</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div>Cancer Biology Department, Institut de Génomique Fonctionnelle, CNRS, INSERM, Université de Montpellier, F-34094 Montpellier, France</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Alexandre+David">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=David A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;David A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Alexandre David</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Eric Rivals</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Department of Computer Science, Laboratoire d'Informatique, de Robotique et de Microélectronique de Montpellier (LIRMM), CNRS et Université de Montpellier, 34095 Montpellier Cedex 5, France</div><div class="aff"><div class="label -label">2</div>Institut de Biologie Computationnelle (IBC); Université de Montpellier, France</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsw062-cor1" data-open="dsw062-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +0033 4 67 41 86 64. Fax. +0033 4 67 41 85 00. Email: <a href="mailto:rivals@lirmm.fr" target="_blank">rivals@lirmm.fr</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Eric+Rivals">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Rivals E">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Rivals E&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Eric Rivals</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsw062-FM1"><div class="fn"><div class="footnote-content">Edited by Prof. Kenta Nakai</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 24, Issue 3, 1 June 2017, Pages 303, <a href='https://doi.org/10.1093/dnares/dsw062'>https://doi.org/10.1093/dnares/dsw062</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">07 February 2017</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">20 June 2016</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">15 December 2016</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="2972686" href="/dnaresearch/article-pdf/24/3/303/17743936/dsw062.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Damien Paulet, Alexandre David, Eric Rivals;  Ribo-seq enlightens codon usage bias, <em>DNA Research</em>, Volume 24, Issue 3, 1 June 2017, Pages 303–210, <a href='https://doi.org/10.1093/dnares/dsw062'>https://doi.org/10.1093/dnares/dsw062</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/2972686?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/2972686?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/2972686?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/2972686?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/2972686?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/2972686?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/2972686?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Ribo-seq%20enlightens%20codon%20usage%20bias&amp;publicationDate=2017-02-07&amp;volumeNum=24&amp;issueNum=3&amp;author=Paulet%2C%20Damien%3B%20David%2C%20Alexandre&amp;startPage=303&amp;endPage=210&amp;contentId=10.1093%2Fdnares%2Fdsw062&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f3%2f303%2f2972686" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=88473278 id="88473278" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Codon usage is biased between lowly and highly expressed genes in a genome-specific manner. This universal bias has been well assessed in some unicellular species, but remains problematic to assess in more complex species. We propose a new method to compute codon usage bias based on genome wide translational data. A new technique based on sequencing of ribosome protected mRNA fragments (Ribo-seq) allowed us to rank genes and compute codon usage bias with high precision for a great variety of species, including mammals. Genes ranking using Ribo-Seq data confirms the influence of the tRNA pool on codon usage bias and shows a decreasing bias in multicellular species. Ribo-Seq analysis also makes possible to detect preferred codons without information on genes function.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;codon usage&quot;">codon usage</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;high throughput sequencing&quot;">high throughput sequencing</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;synonymous codon&quot;">synonymous codon</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="evolution">evolution</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="translation">translation</a></div>                            <h2 scrollto-destination=88473280 id="88473280" class="section-title" >1. Introduction</h2>
<p>Usage of synonymous codons is biased in genomes, as some codons are favoured in highly expressed genes.<a reveal-id="dsw062-B1" data-open="dsw062-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a><sup>,</sup><a reveal-id="dsw062-B2" data-open="dsw062-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> This phenomenon, called codon usage bias (CUB), has been observed in most genomes, although it seems to vary according to the complexity of genomes.<a reveal-id="dsw062-B1" data-open="dsw062-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> The choice of codons seems to depend mainly on tRNA pool: the main hypothesis is that using codons that have the greatest number of accepting tRNAs increases the efficiency or accuracy of translation of highly expressed genes.<a reveal-id="dsw062-B3" data-open="dsw062-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a><sup>,</sup><a reveal-id="dsw062-B4" data-open="dsw062-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> In addition to its impact on mRNA translation, CUB has an impact on protein folding.<a reveal-id="dsw062-B5" data-open="dsw062-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a><sup>,</sup><a reveal-id="dsw062-B6" data-open="dsw062-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a> Despite its importance, precise measures of CUB were, for years, difficult to obtain at genome wide scale. Sharp et al.<a reveal-id="dsw062-B7" data-open="dsw062-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> developed a clustering approach from a set of sequenced genes: the RSCU for Relative Synonymous Codon Usage. This measure is inferred with a statistical process: two clusters of genes are determined based on expression levels, in order to maximize the difference in RSCU between the two groups, then the RSCU is computed using the codon frequencies within each group. Consequently, RSCU accuracy relies on the number of sequenced genes. Here, we propose a new method that permits to directly compute CUB from all translated genes. A recent technique named ribosome profiling (or Ribo-seq) has revolutionized the analysis of translation and permitted to refine the picture of global gene expression control. Ribo-seq is based on deep sequencing of ribosome-protected mRNA fragments (RPF or “footprints”) and permits genome-wide analysis of translated sequences at nucleotide resolution. From a bioinformatic standpoint, Ribo-seq provides a new type of data: indeed, RPF reveal codon occupancy of active ribosomes.<a reveal-id="dsw062-B8" data-open="dsw062-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> This technique yields a precise picture of translation by quantifying the number of ribosomes at every position in a genome. Researchers often analyse jointly Ribo-seq data and RNA-seq data and compare the number of mRNAs and the number of mRNAs involved in translation, which allows them to measure the efficiency of translation.<a reveal-id="dsw062-B9" data-open="dsw062-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a><sup>,</sup><a reveal-id="dsw062-B10" data-open="dsw062-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a><sup>,</sup><a reveal-id="dsw062-B11" data-open="dsw062-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a><sup>,</sup><a reveal-id="dsw062-B12" data-open="dsw062-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a><sup>,</sup><a reveal-id="dsw062-B13" data-open="dsw062-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> We decided to use only Ribo-seq data to measure CUB, based on a simple rationale: the more observed footprints, the more ribosomes at this position. This means that genes with a high coverage are <em>highly translated</em>, and mutations that lead to a benefit in terms of translation are more prone to be fixed in those genes: as a consequence, Ribo-seq could indicate which genes have a strong CUB. Ribo-seq is often used to measure translation efficiency, as the number of ribosomes per copy of an mRNA species. Here <em>highly translated</em> (which differs from <em>efficiently translated</em>) means that all copies of one mRNA species yield, all together, a large number of proteins.</p><p>We ranked genes according to their number of footprints in a precise and limited region of their mRNA (from the 20th to the 200th codons from the start codon). We then split genes into two groups according to their number of footprints and measured codon bias within each group: this way, our two groups are not optimized for RSCU, but derived from translation experiments. Moreover, contrary to previous approaches, we used high-throughput data reflecting the number of ribosomes per sequence and use all translated genes. Therefore, our measure of CUB is directly computed from translational data.</p><p>We applied our approach on a variety of genomes from a parasite, like <em>Plasmodium</em><em>falciparum</em>, to multicellular eukaryotes like <em>Caenorhabditis</em><em>elegans</em> and <em>H</em><em>omo</em><em>sapiens</em>. We corroborate Sharp’s results on yeast and <em>C. elegans</em>,<sup>7,</sup><a reveal-id="dsw062-B14" data-open="dsw062-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> but also detect preferred codons in every species. A preferred codon of an amino acid is defined as the codon with the highest frequency in highly expressed genes. Our analysis provides novel insights on the evolution of CUB.</p>                            <h2 scrollto-destination=88473284 id="88473284" class="section-title" >2. Materials and methods</h2>
                            <h3 scrollto-destination=88473285 id="88473285" class="section-title" >2.1. ORF selection</h3>
<p>Sequences and annotations were collected from NCBI for all organisms. Positions of start codons are crucial in this study. Therefore, we first kept all genes with a unique isoform. Genes with multiple isoforms were rejected if they had different start codons. If all isoforms share the same start codon, we kept the longest common coding sequence. We excluded the mRNAs of mitochondrial genes. Finally, we collected all the spliced mRNA sequences of the selected genes.</p>                            <h3 scrollto-destination=88473287 id="88473287" class="section-title" >2.2. Ribo-seq data and mapping</h3>
<p>All Ribo-seq data come from already published data (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw062/1/dsw062_Supp.pdf?Expires=2147483647&Signature=hwp7bk5cUxNRikfSiIkl3TTPotX6N9DEEJ1zXu86eUPtNFMB~YA6Knkmbl2G3U93cBYnO4ZUn~ivFp6FWmS~uRTi4KVhxvk~1D4S5tIXKJEv~WcODSd~FPYYIrDjy6Byb~heC3TkYI6~2ibqOjma8-Kz94DphePxx5KjpUQG~J14fvlC1wcan17sCxRLVNIZ3MtSuahmFcDkVuLVlPJ1Fer-kfIIsk-3A~3b-MnRXtNkhZ0GvgxhcVEBJyt7i0htXRsjBVZwNLzuYK7v3lp-sEVD2QUJMR2h7PeEEDDnHFM0zoK256NjDv1LUPzTi7ouTIYp5VN6IRg4Bg0AikgJ8g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). We chose species belonging to opisthokonts (the Fungi/Metazoa group) for which Ribo-seq data were available and used <em>P. falciparum</em> as an outgroup because of its restricted set of tRNA genes compared to all other species considered in this study. When several datasets were available for one species, we selected datasets according to three conditions: 1/availability of a replicate, 2/ribosome profiling that closely follows the protocol of Ingolia et al.,<a reveal-id="dsw062-B8" data-open="dsw062-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> and 3/a sufficient sequencing depth after quality filtering. For each selected dataset, we plot the footprint position with respect to the start codon for all genes and determined the majority peak near the start. For all experiments, the peak turns out to be at -12 nucleotides upstream from start codon. As explained in the study by Ingolia et al.,<a reveal-id="dsw062-B8" data-open="dsw062-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> the “shift” applied to the positions of mapped reads was set to 12 nucleotides.</p><p>Reads were mapped with CRAC<a reveal-id="dsw062-B15" data-open="dsw062-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> with a k-mer of length 25: this k-mer is longer than the recommended length for RNA-seq to better fit the short length of RPF. After mapping on mRNA, we took the first position of each read and applied a shift of 12 nucleotides to assign the read to the P-site position of ribosomes.<a reveal-id="dsw062-B16" data-open="dsw062-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a></p>                            <h3 scrollto-destination=88473290 id="88473290" class="section-title" >2.3. Categorizing genes into lowly and highly translated genes</h3>
<p>For each gene, we computed the coverage by ribosome footprints from the 20th to the 200th codons from the start codon (and at least 20 codons before the stop codon). We computed the mean coverage on the selected region after removal all genes having less than 10 footprints. Then, we ranked genes according to this mean coverage. From this ranking, we create two groups that were made uniform in terms of total amount of footprints: the lowly and highly translated genes. To accentuate the contrast between those two groups, we excluded the top 5% of lowly translated genes and the bottom 5% of highly translated genes.</p>                            <h3 scrollto-destination=88473292 id="88473292" class="section-title" >2.4. RSCU computing</h3>
<p>Finally, we computed the RSCU as defined by Sharp et al.<a reveal-id="dsw062-B7" data-open="dsw062-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> within each group (lowly and highly translated genes). Instead of computing the bias of codon usage over the whole ORF, we took sequences from the 20th to the 200th codons from the start codon, i.e. the same region as the one used for gene sorting. For clarity, when the RSCU is computed based on Ribo-seq data, we named it RSCU<sub>RS</sub>. We developed a computer program in Java language to perform this computation (see the Availability section).</p>                            <h3 scrollto-destination=88473294 id="88473294" class="section-title" >2.5. Copy number of tRNA genes and clustering</h3>
<p>We use the number of tRNA genes of a species as a proxy for the size of its tRNA pool.<a reveal-id="dsw062-B17" data-open="dsw062-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a><sup>,</sup><a reveal-id="dsw062-B18" data-open="dsw062-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> We obtained the copy numbers of tRNA genes from the latest version (2.0 from 2016) of the Genomic tRNA Database (GtRNAdb)<a reveal-id="dsw062-B19" data-open="dsw062-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> for all species, but <em>Histoplasma capsulatum</em>. For the latter, we used the Genbank annotation from NCBI database. To cluster species according to their tRNA copy number and according to their average RSCU<sub>RS</sub>, we perform hierarchical clustering with the R software using the Canberra distance with UPGMA algorithm (Unweighted Pair Group Method with Arithmetic Mean). Details are provided in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw062/1/dsw062_Supp.pdf?Expires=2147483647&Signature=hwp7bk5cUxNRikfSiIkl3TTPotX6N9DEEJ1zXu86eUPtNFMB~YA6Knkmbl2G3U93cBYnO4ZUn~ivFp6FWmS~uRTi4KVhxvk~1D4S5tIXKJEv~WcODSd~FPYYIrDjy6Byb~heC3TkYI6~2ibqOjma8-Kz94DphePxx5KjpUQG~J14fvlC1wcan17sCxRLVNIZ3MtSuahmFcDkVuLVlPJ1Fer-kfIIsk-3A~3b-MnRXtNkhZ0GvgxhcVEBJyt7i0htXRsjBVZwNLzuYK7v3lp-sEVD2QUJMR2h7PeEEDDnHFM0zoK256NjDv1LUPzTi7ouTIYp5VN6IRg4Bg0AikgJ8g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material</a></span>.</p>                            <h2 scrollto-destination=88473296 id="88473296" class="section-title" >3. Results and discussion</h2>
<p>Only amino acids that have multiple encoding codons are studied: only 59 codons were used as we excluded ATG (methionine), TGG (tryptophan), and stop codons.</p>                            <h3 scrollto-destination=88473298 id="88473298" class="section-title" >3.1. CUB computation</h3>
<div class="block-child-p">Given a set of mRNA sequences, Sharp has proposed to compute the RSCU for each synonymous codon j of each amino acid i using the formula <a id="E1" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="E1"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mrow><mfrac><mrow><msub><mi>n</mi><mi>i</mi></msub><mo>×</mo><msub><mi>x</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub></mrow><mrow><munderover><mo>∑</mo><mrow><mi>j</mi><mo>=</mo><mn>1</mn></mrow><mrow><msub><mi>n</mi><mi>i</mi></msub></mrow></munderover><mrow><msub><mi>x</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub></mrow></mrow></mfrac></mrow></mrow></math></div></div></div> where n<sub>i</sub> denotes the number of synonymous codons for amino acid i, and <em>x<sub>ij</sub></em> denotes the number of occurrences of codon j in the set of sequences. Sharp has selected a subset of mRNAs that were known to be highly expressed. Our proposal is to automatically select mRNA sequences that are highly translated according to their coverage by ribosomes, which is determined by mapping Ribo-seq reads. Then, we use the formula above with counts of codons limited to a range comprised between the 20th and the 200th codons. The choice of this range is explained below.</div><p>Sharp and Bradnam<a reveal-id="dsw062-B14" data-open="dsw062-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> have computed RSCU for <em>C. elegans</em>, and we compared his results with our method (RSCU<sub>RS</sub>, for RSCU by Ribo-seq) as illustrated in <a reveal-id="dsw062-F1" data-open="dsw062-F1" class="link link-reveal link-table xref-fig">Fig. 1A</a>. The two methods split genes in two categories: highly and lowly expressed genes, but with completely different inputs: Sharp et al. computed codon bias with a statistical model based on gene sequences, while we used only Ribo-seq footprints. Nevertheless, we obtained a high correlation (&gt;0.97) between Sharp's measure and ours. This correlation is even higher within replicates. In 1992, Lloyd and Sharp<a reveal-id="dsw062-B20" data-open="dsw062-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> published RSCU values for highly expressed genes in <em>C</em><em>andida</em><em>albicans</em>. Here again, the comparison with RSCU values is shown in <a reveal-id="dsw062-F1" data-open="dsw062-F1" class="link link-reveal link-table xref-fig">Fig. 1B</a>, and we observed a very high correlation between RSCU and RSCU<sub>RS</sub> (&gt;0.98). In both experiments, we used a larger set of genes (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw062/1/dsw062_Supp.pdf?Expires=2147483647&Signature=hwp7bk5cUxNRikfSiIkl3TTPotX6N9DEEJ1zXu86eUPtNFMB~YA6Knkmbl2G3U93cBYnO4ZUn~ivFp6FWmS~uRTi4KVhxvk~1D4S5tIXKJEv~WcODSd~FPYYIrDjy6Byb~heC3TkYI6~2ibqOjma8-Kz94DphePxx5KjpUQG~J14fvlC1wcan17sCxRLVNIZ3MtSuahmFcDkVuLVlPJ1Fer-kfIIsk-3A~3b-MnRXtNkhZ0GvgxhcVEBJyt7i0htXRsjBVZwNLzuYK7v3lp-sEVD2QUJMR2h7PeEEDDnHFM0zoK256NjDv1LUPzTi7ouTIYp5VN6IRg4Bg0AikgJ8g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>A).</p>                        <a id="88473301" scrollto-destination="88473301"></a>
<div id="dsw062-F1" data-id="dsw062-F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw062/1/m_dsw062f1.png?Expires=2147483647&Signature=UAQodszJDWMbGoWX1cuHQ9tyc7uNtMkCgoJthcetnluf8WNxq1g6691Anr1SEiRsMo2gWuEPo~68Y8NV0B36PY9EN2EmzMmihTUTFSahaQ~bCY3AcRAhDv6zbob1Np2b8U1XVxXVegke6NC~8Q97FRbOra58NKXEAW~CkkCuIGYLVoG8V3ohshTmp6BOhWX2YTm6fx29zA4G1r36PXlpYWsU5yndpLKXxLdFAYoIEThyyohK9Ya5bdBiVYdg2tfqWjy7PJrOoVfrDqNRcFwb6XI-6y03IyARxgvjUB3Cn0A88e4NzdXaxu67vxHIfDzp9WbFTw35E8bX~iaKgEgZcA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of the strength of codon usage bias as measured by the RSCU as originally defined by Sharp et al., and by the RSCURS which is introduced in this work. Both measures are plotted for a subset of highly expressed genes. The RSCU is computed for each codon of an amino acid: it multiplies the relative observed frequency of this codon among all possible codons for the corresponding amino acid by the number of possible codons. In all graphs, the codons on x-axis are ordered by increasing RSCU values computed by Sharp et al. (A) Comparison of RSCU values of Sharp and RSCURS values in C. elegans (in two replicates). Both replicate curves for RSCURS are extremely close from each other, and they closely follow that from Sharp. (B) Comparison of RSCU values of Sharp and RSCURS values in C. albicans. (C) and (D). Comparison of eight RSCURS curves obtained with codons counts computed in different ranges of codons starting with 20th codon and ending between the 50th and the 400th codons, respectively, in C. elegans (C) and in C. albicans (D). Apart for the smallest range of codons (i.e. [20-50]) all curves are very close to each other, thereby showing the robustness of with respect to the range of codons taken into account. One observes that RSCURS values reach higher values in the yeast species than in the worm species. Refer to the online version for colors." path-from-xml="dsw062f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/88473301/dsw062f1.tif" path-from-xml="dsw062f1.tif" target="_blank">View large</a><a section="88473301" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw062/1/dsw062f1.png?Expires=2147483647&Signature=dC~rl10CbD2Rf5yuSO3hEa30MYkxnAhYfTKD62TIKQFJJPhIjyDT2UcVVhK1hem-O7LfSUf2nFOCedFu-NkkSuAJgIxDym1G76r1OqDpSyvPxP9XEFbSWLN3MnWcu0FXLj8OJ4sxL2kcYbS04-LEmjINGEOLDKbgLDEKH~C2-ARCqMYWGxYOmsVife73lYymk8rEPqiIF~chyNaa~nWiEueUl2izrz2EIkZzuwwO213Q3ic0VSdPhnQi0YqL-FlbSFSBtvPDo7p7C2ekRgbZ5CuRRSTPxGDQTfthsEWVjP9xUtxReK4ZgUKjtfhQ-Q-mmjw~6VtnCcDTyxwKG5Pflw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=88473301&ar=2972686&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw062f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of the strength of codon usage bias as measured by the RSCU as originally defined by Sharp et al., and by the RSCU<sub>RS</sub> which is introduced in this work. Both measures are plotted for a subset of highly expressed genes. The RSCU is computed for each codon of an amino acid: it multiplies the relative observed frequency of this codon among all possible codons for the corresponding amino acid by the number of possible codons. In all graphs, the codons on x-axis are ordered by increasing RSCU values computed by Sharp et al. (A) Comparison of RSCU values of Sharp and RSCU<sub>RS</sub> values in <em>C. elegans</em> (in two replicates). Both replicate curves for RSCU<sub>RS</sub> are extremely close from each other, and they closely follow that from Sharp. (B) Comparison of RSCU values of Sharp and RSCU<sub>RS</sub> values in <em>C. albicans</em>. (C) and (D). Comparison of eight RSCU<sub>RS</sub> curves obtained with codons counts computed in different ranges of codons starting with 20th codon and ending between the 50th and the 400th codons, respectively, in <em>C. elegans</em> (C) and in <em>C. albicans</em> (D). Apart for the smallest range of codons (i.e. [20-50]) all curves are very close to each other, thereby showing the robustness of with respect to the range of codons taken into account. One observes that RSCU<sub>RS</sub> values reach higher values in the yeast species than in the worm species. Refer to the online version for colors.</p></div></div><div content-id="dsw062-F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw062/1/m_dsw062f1.png?Expires=2147483647&Signature=UAQodszJDWMbGoWX1cuHQ9tyc7uNtMkCgoJthcetnluf8WNxq1g6691Anr1SEiRsMo2gWuEPo~68Y8NV0B36PY9EN2EmzMmihTUTFSahaQ~bCY3AcRAhDv6zbob1Np2b8U1XVxXVegke6NC~8Q97FRbOra58NKXEAW~CkkCuIGYLVoG8V3ohshTmp6BOhWX2YTm6fx29zA4G1r36PXlpYWsU5yndpLKXxLdFAYoIEThyyohK9Ya5bdBiVYdg2tfqWjy7PJrOoVfrDqNRcFwb6XI-6y03IyARxgvjUB3Cn0A88e4NzdXaxu67vxHIfDzp9WbFTw35E8bX~iaKgEgZcA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of the strength of codon usage bias as measured by the RSCU as originally defined by Sharp et al., and by the RSCURS which is introduced in this work. Both measures are plotted for a subset of highly expressed genes. The RSCU is computed for each codon of an amino acid: it multiplies the relative observed frequency of this codon among all possible codons for the corresponding amino acid by the number of possible codons. In all graphs, the codons on x-axis are ordered by increasing RSCU values computed by Sharp et al. (A) Comparison of RSCU values of Sharp and RSCURS values in C. elegans (in two replicates). Both replicate curves for RSCURS are extremely close from each other, and they closely follow that from Sharp. (B) Comparison of RSCU values of Sharp and RSCURS values in C. albicans. (C) and (D). Comparison of eight RSCURS curves obtained with codons counts computed in different ranges of codons starting with 20th codon and ending between the 50th and the 400th codons, respectively, in C. elegans (C) and in C. albicans (D). Apart for the smallest range of codons (i.e. [20-50]) all curves are very close to each other, thereby showing the robustness of with respect to the range of codons taken into account. One observes that RSCURS values reach higher values in the yeast species than in the worm species. Refer to the online version for colors." path-from-xml="dsw062f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/88473301/dsw062f1.tif" path-from-xml="dsw062f1.tif" target="_blank">View large</a><a section="88473301" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw062/1/dsw062f1.png?Expires=2147483647&Signature=dC~rl10CbD2Rf5yuSO3hEa30MYkxnAhYfTKD62TIKQFJJPhIjyDT2UcVVhK1hem-O7LfSUf2nFOCedFu-NkkSuAJgIxDym1G76r1OqDpSyvPxP9XEFbSWLN3MnWcu0FXLj8OJ4sxL2kcYbS04-LEmjINGEOLDKbgLDEKH~C2-ARCqMYWGxYOmsVife73lYymk8rEPqiIF~chyNaa~nWiEueUl2izrz2EIkZzuwwO213Q3ic0VSdPhnQi0YqL-FlbSFSBtvPDo7p7C2ekRgbZ5CuRRSTPxGDQTfthsEWVjP9xUtxReK4ZgUKjtfhQ-Q-mmjw~6VtnCcDTyxwKG5Pflw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=88473301&ar=2972686&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw062f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of the strength of codon usage bias as measured by the RSCU as originally defined by Sharp et al., and by the RSCU<sub>RS</sub> which is introduced in this work. Both measures are plotted for a subset of highly expressed genes. The RSCU is computed for each codon of an amino acid: it multiplies the relative observed frequency of this codon among all possible codons for the corresponding amino acid by the number of possible codons. In all graphs, the codons on x-axis are ordered by increasing RSCU values computed by Sharp et al. (A) Comparison of RSCU values of Sharp and RSCU<sub>RS</sub> values in <em>C. elegans</em> (in two replicates). Both replicate curves for RSCU<sub>RS</sub> are extremely close from each other, and they closely follow that from Sharp. (B) Comparison of RSCU values of Sharp and RSCU<sub>RS</sub> values in <em>C. albicans</em>. (C) and (D). Comparison of eight RSCU<sub>RS</sub> curves obtained with codons counts computed in different ranges of codons starting with 20th codon and ending between the 50th and the 400th codons, respectively, in <em>C. elegans</em> (C) and in <em>C. albicans</em> (D). Apart for the smallest range of codons (i.e. [20-50]) all curves are very close to each other, thereby showing the robustness of with respect to the range of codons taken into account. One observes that RSCU<sub>RS</sub> values reach higher values in the yeast species than in the worm species. Refer to the online version for colors.</p></div></div><p>We question whether our method, RSCU<sub>RS</sub>, is robust with respect to the range of codons used—by default [20, 200] codons. The lower limit of 20 avoids counting the accumulation of RPF due to translation initiation.<a reveal-id="dsw062-B8" data-open="dsw062-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> The upper limit of 200 was chosen as a minimum length for including sufficient counts and to maximize the number of mRNAs taken into account. Clearly, the upper limit choice is somehow arbitrary. We computed the RSCU<sub>RS</sub> for eight different upper limits ranging from 50 to 400 codons. The RSCU<sub>RS</sub> curves for each range are shown on a single graph in <a reveal-id="dsw062-F1" data-open="dsw062-F1" class="link link-reveal link-table xref-fig">Fig. 1B</a> for <em>C. elegans</em> and <a reveal-id="dsw062-F1" data-open="dsw062-F1" class="link link-reveal link-table xref-fig">Fig. 1C</a> for <em>C. albicans</em>. The agreement among all eight curves is striking for both species. Only the curves of two shortest ranges—[20-50] and [20,100] —depart slightly from the other curves, indicating that a minimum number of codons is necessary to capture a stable signal. All other curves are very close to each other, showing that the RSCU<sub>RS</sub> results are not strongly dependent of the upper limit. Hence, RSCU<sub>RS</sub> appears to be robust with respect to this parameter. Since the curve for range [20,200] is highly correlated with Sharp's curves, logically all eight curves are highly correlated with Sharp's RSCU values as well.</p>                            <h3 scrollto-destination=88473303 id="88473303" class="section-title" >3.2. CUB exists in all species</h3>
<p>As a first step, we computed the Euclidean distance of RSCU<sub>RS</sub> between highly and lowly translated genes (<a reveal-id="dsw062-F2" data-open="dsw062-F2" class="link link-reveal link-table xref-fig">Fig. 2A</a>) as a mean to evaluate the intensity of CUB for each organism: if this distance is close to 0, it means there are very few differences in codon usage between highly and lowly translated genes. Results gave two clearly distinct species groups. The first group corresponds to species with a lower CUB and comprises mammals, Drosophila, <em>P. falciparum</em>, and <em>H. capsulatum</em>. The second group of species exhibiting a higher CUB contains three yeasts and <em>C. elegans</em>.</p>                        <a id="88473305" scrollto-destination="88473305"></a>
<div id="dsw062-F2" data-id="dsw062-F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw062/1/m_dsw062f2.png?Expires=2147483647&Signature=3Q4v1C-3atTT0fByF1myoXxCLA3Nz7IZ-OAjKPDBqjK6CiV6Inn0MVzKzNbiWhLgE2Kl86O8dokaP~ncLU~rV3SDavYkgR7IvZTe3s3G6fiZPGIOBmC360PvwWwRLmixFune6IJ70vlYxhFusvqcpkYUzdFud1XuGeNTwBXYF0tU0s2sVYZkQ6aSD2BB6asKxgRHP55Skx15yx42I-5zY6jg8ER58Sl9l4hZxqbyssjwxCgmx0AgAM2w0AtAnWlAp-mJfJxOBwOI0cL6TulT6o-c88JkG-p6V7AWDVMMWqLjrOvcu2zibevDGSkBfeH7-NK8ENwyo73I3jhWXcging__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Differences in codon usage bias between lowly and highly translated genes. (A) Euclidean distance between RSCURS of lowly and highly translated genes was computed for ten species. This distance clearly partitions the species in two groups: the mean euclidean distance is 1.51 for group 1 (left, in orange), and is 6.64 for group 2 (right, in grey). Group 2 comprises C. albicans, S. cerevisiae, S. pombe and C. elegans, while group 1 comprises all vertebrates, D. melanogaster, P. falciparum and H. capsulatum. (B) For each species, comparison of the RSCURS of all codons between lowly and highly translated genes (ltg vs htg). Codons from group 1 species (containing the vertebrates) are shown as orange circles, while codons from group 2 species (that of the budding yeast) as gray circles. A point near the diagonal indicates a similar behavior in ltg and htg. The further apart the point from the diagonal, the higher the bias of that codon. Refer to the online version for colors." path-from-xml="dsw062f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/88473305/dsw062f2.tif" path-from-xml="dsw062f2.tif" target="_blank">View large</a><a section="88473305" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw062/1/dsw062f2.png?Expires=2147483647&Signature=P7gGG1QMEV1XhLmvaTC2t3s0IBTLof1qmTkqd6VCLvaULYxYOusbByZk72XVVaKK2rIVr39yy2bo7KOosaJBfmRDs5Jn3tMXy0xYru7Jz1zp4zDYXjzjwapwxcMc9hJDget-6s2LIhIbxrgTALyGHNEjwOsjXEBPqDv-lWpD7xdadK873nkSZoHIpOkx8VXBPzx-DPPyq6ECDZorGcsFTisVA8Vpd-3Ae3DRVc3hfEFcf9MPUp8iqTzwquPqt0SV0x6AFQgiq7ufAl6G~uUZWXpGPLmlCx7FZMQwbgf-Y6VK~huhTHQmu4sIW2Thz-kD~jrbImgng1hsIEYivtNbsg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=88473305&ar=2972686&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw062f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Differences in codon usage bias between lowly and highly translated genes. (A) Euclidean distance between RSCU<sub>RS</sub> of lowly and highly translated genes was computed for ten species. This distance clearly partitions the species in two groups: the mean euclidean distance is 1.51 for group 1 (left, in orange), and is 6.64 for group 2 (right, in grey). Group 2 comprises <em>C. albicans, S. cerevisiae, S. pombe</em> and <em>C. elegans</em>, while group 1 comprises all vertebrates, <em>D. melanogaster, P. falciparum</em> and <em>H. capsulatum</em>. (B) For each species, comparison of the RSCU<sub>RS</sub> of all codons between lowly and highly translated genes (ltg vs htg). Codons from group 1 species (containing the vertebrates) are shown as orange circles, while codons from group 2 species (that of the budding yeast) as gray circles. A point near the diagonal indicates a similar behavior in ltg and htg. The further apart the point from the diagonal, the higher the bias of that codon. Refer to the online version for colors.</p></div></div><div content-id="dsw062-F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw062/1/m_dsw062f2.png?Expires=2147483647&Signature=3Q4v1C-3atTT0fByF1myoXxCLA3Nz7IZ-OAjKPDBqjK6CiV6Inn0MVzKzNbiWhLgE2Kl86O8dokaP~ncLU~rV3SDavYkgR7IvZTe3s3G6fiZPGIOBmC360PvwWwRLmixFune6IJ70vlYxhFusvqcpkYUzdFud1XuGeNTwBXYF0tU0s2sVYZkQ6aSD2BB6asKxgRHP55Skx15yx42I-5zY6jg8ER58Sl9l4hZxqbyssjwxCgmx0AgAM2w0AtAnWlAp-mJfJxOBwOI0cL6TulT6o-c88JkG-p6V7AWDVMMWqLjrOvcu2zibevDGSkBfeH7-NK8ENwyo73I3jhWXcging__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Differences in codon usage bias between lowly and highly translated genes. (A) Euclidean distance between RSCURS of lowly and highly translated genes was computed for ten species. This distance clearly partitions the species in two groups: the mean euclidean distance is 1.51 for group 1 (left, in orange), and is 6.64 for group 2 (right, in grey). Group 2 comprises C. albicans, S. cerevisiae, S. pombe and C. elegans, while group 1 comprises all vertebrates, D. melanogaster, P. falciparum and H. capsulatum. (B) For each species, comparison of the RSCURS of all codons between lowly and highly translated genes (ltg vs htg). Codons from group 1 species (containing the vertebrates) are shown as orange circles, while codons from group 2 species (that of the budding yeast) as gray circles. A point near the diagonal indicates a similar behavior in ltg and htg. The further apart the point from the diagonal, the higher the bias of that codon. Refer to the online version for colors." path-from-xml="dsw062f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/88473305/dsw062f2.tif" path-from-xml="dsw062f2.tif" target="_blank">View large</a><a section="88473305" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw062/1/dsw062f2.png?Expires=2147483647&Signature=P7gGG1QMEV1XhLmvaTC2t3s0IBTLof1qmTkqd6VCLvaULYxYOusbByZk72XVVaKK2rIVr39yy2bo7KOosaJBfmRDs5Jn3tMXy0xYru7Jz1zp4zDYXjzjwapwxcMc9hJDget-6s2LIhIbxrgTALyGHNEjwOsjXEBPqDv-lWpD7xdadK873nkSZoHIpOkx8VXBPzx-DPPyq6ECDZorGcsFTisVA8Vpd-3Ae3DRVc3hfEFcf9MPUp8iqTzwquPqt0SV0x6AFQgiq7ufAl6G~uUZWXpGPLmlCx7FZMQwbgf-Y6VK~huhTHQmu4sIW2Thz-kD~jrbImgng1hsIEYivtNbsg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=88473305&ar=2972686&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw062f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Differences in codon usage bias between lowly and highly translated genes. (A) Euclidean distance between RSCU<sub>RS</sub> of lowly and highly translated genes was computed for ten species. This distance clearly partitions the species in two groups: the mean euclidean distance is 1.51 for group 1 (left, in orange), and is 6.64 for group 2 (right, in grey). Group 2 comprises <em>C. albicans, S. cerevisiae, S. pombe</em> and <em>C. elegans</em>, while group 1 comprises all vertebrates, <em>D. melanogaster, P. falciparum</em> and <em>H. capsulatum</em>. (B) For each species, comparison of the RSCU<sub>RS</sub> of all codons between lowly and highly translated genes (ltg vs htg). Codons from group 1 species (containing the vertebrates) are shown as orange circles, while codons from group 2 species (that of the budding yeast) as gray circles. A point near the diagonal indicates a similar behavior in ltg and htg. The further apart the point from the diagonal, the higher the bias of that codon. Refer to the online version for colors.</p></div></div><p>For these species, we compare the RSCU<sub>RS</sub> of all codons between lowly and highly translated genes (ltg vs htg; <a reveal-id="dsw062-F2" data-open="dsw062-F2" class="link link-reveal link-table xref-fig">Fig. 2B</a>). One notices the asymmetry of the diagram with respect to the diagonal: The part of the diagram beyond a RSCU<sub>RS</sub> of two is populated above the diagonal and almost empty below it. This confirms the existence of favoured codons in htg, and their absence in ltg. Codons located far above the diagonal indicate strongly favoured codons in htg. This illustrates the impact of selection pressure linked to the gene expression level. There also exist codons far below the diagonal (which are disfavoured in htg), but their RSCU<sub>RS</sub> remains low (&lt;2): those are either disfavoured or slightly favoured in ltg. Some codons even exhibit a nearly null RSCU<sub>RS</sub> in htg, meaning these are almost forbidden in htg. Apart from the global asymmetry, the difference between the codons of the two species groups is striking. All codons far apart from the diagonal belong to group 2 and very few codons of these species lie near the diagonal: in a species like yeast, the difference of usage between htg and ltg is strong for most codons. Note also that in group 2, some codons are disfavoured in htg and favoured in ltg, suggesting there could be some selection pressure also in ltg, albeit weaker than in htg. In species of group 1, most codons lie around the diagonal, but the bias changes with the RSCU<sub>RS</sub> value: most codons that are disfavoured in htg (dots lying below the horizontal line, i.e. with a RSCU<sub>RS</sub> in htg &lt;1) lie below the diagonal, while most codons favoured in htg (i.e. a RSCU<sub>RS</sub> in htg &gt; 1) lie above the diagonal. This suggests that codon usage preference in htg is small, but is detectable with Ribo-seq data, and thus do exist also in these species.</p>                            <h3 scrollto-destination=88473307 id="88473307" class="section-title" >3.3. Patterns of codon preferences</h3>
<p>Then, we compared the frequency of codons in highly translated genes. Higher eukaryotes have few strongly preferred codons while yeasts show marked differences (<a reveal-id="dsw062-F3" data-open="dsw062-F3" class="link link-reveal link-table xref-fig">Fig. 3A</a>): some yeasts present a highly preferred codon for Arginine, Glutamic acid and Cysteine, whereas mammals and Drosophila do not. Glutamine follows remarkable patterns: both mammals and Drosophila strongly prefer CAG, yeasts favour CAA, while <em>H. capsulatum</em> and <em>C. elegans</em> use CAA/CAG in an equivalent fashion.</p>                        <a id="88473309" scrollto-destination="88473309"></a>
<div id="dsw062-F3" data-id="dsw062-F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw062/1/m_dsw062f3.png?Expires=2147483647&Signature=rkd6b5qu4mOj8d8IWg5RKqp0mh6PyZvawro64FtyzvEcLeo7rh3IZaBTyN0vXpJ3KFWcQAsREPYAbsmabdcfr~MZeilMdzsF7LV4Joqw-C3e4uG-Br4~2LISSqHPOkoHAFc96CaYgRZaVVuCf8x~3gMN3Sass~G9ZoymIroRcuXJ2wVZ7SklWQsSfs-UaozHsjdrHGXWMR6~4Udlx0GdYYCQerFkev0BBIeLxeVfi2HWh8P1WAu1q7oX88A1T2a90dK7Tv8Iecolhdsvq~My29wmraqFcsmuACp1bl1DcCXcixqgzqAAeEgMlEFJ2pgSa2rr4HySCy2Q4p03PJKaLg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(A) Preferred codons in highly translated genes. To ease the comparison of the bias across codons, we plotted the codon frequency such that each codon has a value comprised in [0,1]. The frequency is the RSCU multiplied by the number of codons for the corresponding amino acid. The range of colors goes from blue for a frequency of 0 until red for a frequency of 1. Same species abbreviations as in Fig. 1. (B) Clustering of the ten species based on tRNA copy numbers. (C) Clustering of the 10 species based on RSCURS in highly translated genes. Both clustering were performed using UPGMA algorithm in R (see Methods section). (D) Variation of the numbers of codons lacking tRNA with the size of the species' tRNA pool. Linear regression lines are solid for codons with no tRNA (R2 = 0.69) and dotted for preferred codons with no tRNA (R2 = 0.53). H. capsulatum was excluded for both regressions. The regression line for all codons decreases sharply (slope = −0.026), while the regression line for preferred codons remains flat (slope = −0.002). Refer to the online version for colors." path-from-xml="dsw062f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/88473309/dsw062f3.tif" path-from-xml="dsw062f3.tif" target="_blank">View large</a><a section="88473309" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw062/1/dsw062f3.png?Expires=2147483647&Signature=uPS9kKcWr0ZsNQgPr94-qiyNWurmyORP96W4yvEyeGHjrEWVYCaXx7zrnHjiIzI91TJ8MYI4-dAlsb1FKtAXfYFpK9ZiqrzhUNqLPqKuH5c7-wOp79YCxh6dJ~Zaa1bBeKAPv1M0z5KutG~BhlFLxceZiNUgDp36YCM6nR8FyieMsPEsB2kiw6FfDzGN0KIicUVc9U2vcIqhIvXJ9gQoqovJs~hCH9K4NRM78A9MUiznPK1izekVq19Nsf5OZIa99Vov6xfBc3KzZ82qGPeKAJJgKD7vF-Xv63wiWQzBHxYA~ltoUcvSID-xpICYHoJwFuc7BBo-MXulOaO0Uh21Kw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=88473309&ar=2972686&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw062f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(A) Preferred codons in highly translated genes. To ease the comparison of the bias across codons, we plotted the codon frequency such that each codon has a value comprised in [0,1]. The frequency is the RSCU multiplied by the number of codons for the corresponding amino acid. The range of colors goes from blue for a frequency of 0 until red for a frequency of 1. Same species abbreviations as in <a reveal-id="dsw062-F1" data-open="dsw062-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>. (B) Clustering of the ten species based on tRNA copy numbers. (C) Clustering of the 10 species based on RSCU<sub>RS</sub> in highly translated genes. Both clustering were performed using UPGMA algorithm in R (see Methods section). (D) Variation of the numbers of codons lacking tRNA with the size of the species' tRNA pool. Linear regression lines are solid for codons with no tRNA (<em>R</em><sup>2</sup> = 0.69) and dotted for preferred codons with no tRNA (<em>R</em><sup>2</sup> = 0.53). <em>H. capsulatum</em> was excluded for both regressions. The regression line for all codons decreases sharply (slope = −0.026), while the regression line for preferred codons remains flat (slope = −0.002). Refer to the online version for colors.</p></div></div><div content-id="dsw062-F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw062/1/m_dsw062f3.png?Expires=2147483647&Signature=rkd6b5qu4mOj8d8IWg5RKqp0mh6PyZvawro64FtyzvEcLeo7rh3IZaBTyN0vXpJ3KFWcQAsREPYAbsmabdcfr~MZeilMdzsF7LV4Joqw-C3e4uG-Br4~2LISSqHPOkoHAFc96CaYgRZaVVuCf8x~3gMN3Sass~G9ZoymIroRcuXJ2wVZ7SklWQsSfs-UaozHsjdrHGXWMR6~4Udlx0GdYYCQerFkev0BBIeLxeVfi2HWh8P1WAu1q7oX88A1T2a90dK7Tv8Iecolhdsvq~My29wmraqFcsmuACp1bl1DcCXcixqgzqAAeEgMlEFJ2pgSa2rr4HySCy2Q4p03PJKaLg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(A) Preferred codons in highly translated genes. To ease the comparison of the bias across codons, we plotted the codon frequency such that each codon has a value comprised in [0,1]. The frequency is the RSCU multiplied by the number of codons for the corresponding amino acid. The range of colors goes from blue for a frequency of 0 until red for a frequency of 1. Same species abbreviations as in Fig. 1. (B) Clustering of the ten species based on tRNA copy numbers. (C) Clustering of the 10 species based on RSCURS in highly translated genes. Both clustering were performed using UPGMA algorithm in R (see Methods section). (D) Variation of the numbers of codons lacking tRNA with the size of the species' tRNA pool. Linear regression lines are solid for codons with no tRNA (R2 = 0.69) and dotted for preferred codons with no tRNA (R2 = 0.53). H. capsulatum was excluded for both regressions. The regression line for all codons decreases sharply (slope = −0.026), while the regression line for preferred codons remains flat (slope = −0.002). Refer to the online version for colors." path-from-xml="dsw062f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/88473309/dsw062f3.tif" path-from-xml="dsw062f3.tif" target="_blank">View large</a><a section="88473309" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw062/1/dsw062f3.png?Expires=2147483647&Signature=uPS9kKcWr0ZsNQgPr94-qiyNWurmyORP96W4yvEyeGHjrEWVYCaXx7zrnHjiIzI91TJ8MYI4-dAlsb1FKtAXfYFpK9ZiqrzhUNqLPqKuH5c7-wOp79YCxh6dJ~Zaa1bBeKAPv1M0z5KutG~BhlFLxceZiNUgDp36YCM6nR8FyieMsPEsB2kiw6FfDzGN0KIicUVc9U2vcIqhIvXJ9gQoqovJs~hCH9K4NRM78A9MUiznPK1izekVq19Nsf5OZIa99Vov6xfBc3KzZ82qGPeKAJJgKD7vF-Xv63wiWQzBHxYA~ltoUcvSID-xpICYHoJwFuc7BBo-MXulOaO0Uh21Kw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=88473309&ar=2972686&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw062f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(A) Preferred codons in highly translated genes. To ease the comparison of the bias across codons, we plotted the codon frequency such that each codon has a value comprised in [0,1]. The frequency is the RSCU multiplied by the number of codons for the corresponding amino acid. The range of colors goes from blue for a frequency of 0 until red for a frequency of 1. Same species abbreviations as in <a reveal-id="dsw062-F1" data-open="dsw062-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>. (B) Clustering of the ten species based on tRNA copy numbers. (C) Clustering of the 10 species based on RSCU<sub>RS</sub> in highly translated genes. Both clustering were performed using UPGMA algorithm in R (see Methods section). (D) Variation of the numbers of codons lacking tRNA with the size of the species' tRNA pool. Linear regression lines are solid for codons with no tRNA (<em>R</em><sup>2</sup> = 0.69) and dotted for preferred codons with no tRNA (<em>R</em><sup>2</sup> = 0.53). <em>H. capsulatum</em> was excluded for both regressions. The regression line for all codons decreases sharply (slope = −0.026), while the regression line for preferred codons remains flat (slope = −0.002). Refer to the online version for colors.</p></div></div><p>Our analysis revealed that some preferred codons are well conserved (<a reveal-id="dsw062-T1" data-open="dsw062-T1" class="link link-reveal link-table xref-fig">Table 1</a>): all species, but <em>P. falciparum</em>, have the same preferred codon for phenylalanine, histidine, tyrosine (that is, all aromatic amino acids that can be studied), and asparagine. On the opposite, Leucine has five different preferred codons. Indeed, unique preferred codons are rare: <em>Schizosaccharomyces</em><em>pombe</em> has one only for proline (CCT), and <em>P. falciparum</em> has the highest number of unique features with seven unique preferred codons. <div content-id="dsw062-T1" class="table-modal"><div class="table-wrap"><div id="dsw062-T1" data-id="dsw062-T1" class="table-wrap-title"><div class="label -label">Table 1</div><div class="caption"><p>Preferred codons per organism in highly (A) and lowly (B) translated genes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th colspan="11">A.<hr /> </th></tr><tr><th> </th><th>Hs </th><th>Mm </th><th>Rn </th><th>Dm </th><th>Hc </th><th>Ce </th><th>Sp </th><th>Sc </th><th>Ca </th><th>Pf </th></tr></thead><tbody><tr><td>Phe </td><td>TTC </td><td>TTC </td><td>TTC </td><td>TTC </td><td>TTC </td><td>TTC </td><td>TTC </td><td>TTC </td><td>TTC </td><td><strong>TTT*</strong> </td></tr><tr><td>His </td><td>CAC </td><td>CAC </td><td>CAC </td><td>CAC </td><td>CAC </td><td>CAC </td><td>CAC </td><td>CAC </td><td>CAC </td><td>CAT </td></tr><tr><td>Tyr </td><td>TAC </td><td>TAC </td><td>TAC </td><td>TAC </td><td>TAC </td><td>TAC </td><td>TAC </td><td>TAC </td><td>TAC </td><td><strong>TAT*</strong> </td></tr><tr><td>Asn </td><td>AAC </td><td>AAC </td><td>AAC </td><td>AAC </td><td>AAC </td><td>AAC </td><td>AAC </td><td>AAC </td><td>AAC </td><td><strong>AAT*</strong> </td></tr><tr><td>Lys </td><td>AAG </td><td>AAG </td><td>AAG </td><td>AAG </td><td>AAG </td><td>AAG </td><td>AAG </td><td>AAG </td><td>AAA </td><td>AAA </td></tr><tr><td>Cys </td><td>TGC </td><td>TGC </td><td>TGC </td><td>TGC </td><td>TGC </td><td>TGC </td><td>TGC </td><td><strong>TGT*</strong> </td><td><strong>TGT*</strong> </td><td>TGT </td></tr><tr><td>Glu </td><td>GAG </td><td>GAG </td><td>GAG </td><td>GAG </td><td>GAG </td><td>GAG </td><td>GAG </td><td>GAA </td><td>GAA </td><td>GAA </td></tr><tr><td>Ala </td><td><strong>GCC*</strong> </td><td><strong>GCC*</strong> </td><td><strong>GCC*</strong> </td><td><strong>GCC*</strong> </td><td><strong>GCC*</strong> </td><td><strong>GCC*</strong> </td><td>GCT </td><td>GCT </td><td>GCT </td><td>GCT </td></tr><tr><td>Se </td><td><strong>TCC*</strong> </td><td><strong>TCC*</strong> </td><td><strong>TCC*</strong> </td><td><strong>TCC*</strong> </td><td><strong>TCC*</strong> </td><td><strong>TCC*</strong> </td><td>TCT </td><td>TCT </td><td>TCT </td><td>TCA </td></tr><tr><td>Thr </td><td><strong>ACC*</strong> </td><td><strong>ACC*</strong> </td><td><strong>ACC*</strong> </td><td><strong>ACC*</strong> </td><td><strong>ACC*</strong> </td><td><strong>ACC*</strong> </td><td>ACT </td><td>ACT </td><td>ACT </td><td>ACA </td></tr><tr><td>Ile </td><td><strong>ATC*</strong> </td><td><strong>ATC*</strong> </td><td><strong>ATC*</strong> </td><td><strong>ATC*</strong> </td><td><strong>ATC*</strong> </td><td><strong>ATC*</strong> </td><td>ATT </td><td><strong>ATC*</strong> </td><td>ATT </td><td>ATT </td></tr><tr><td>Gly </td><td>GGC </td><td>GGC </td><td>GGC </td><td>GGC </td><td><strong>GGC*</strong> </td><td><strong>GGA*</strong> </td><td><strong>GGT*</strong> </td><td><strong>GGT*</strong> </td><td><strong>GGT*</strong> </td><td>GGA </td></tr><tr><td>Pro </td><td>CCC </td><td><strong>CCC*</strong> </td><td><strong>CCC*</strong> </td><td><strong>CCC*</strong> </td><td><strong>CCC*</strong> </td><td>CCA </td><td>CCT </td><td>CCA </td><td>CCA </td><td>CCA </td></tr><tr><td>Gln </td><td><strong>CAG*</strong> </td><td><strong>CAG*</strong> </td><td><strong>CAG*</strong> </td><td><strong>CAG*</strong> </td><td><strong>CAG*</strong> </td><td>CAA </td><td>CAA </td><td>CAA </td><td>CAA </td><td>CAA </td></tr><tr><td>Leu </td><td><strong>CTG*</strong> </td><td>CTG </td><td>CTG </td><td><strong>CTG*</strong> </td><td><strong>CTC*</strong> </td><td><strong>CTC*</strong> </td><td><strong>TTG*</strong> </td><td>TTG </td><td>TTG </td><td>TTA </td></tr><tr><td>Val </td><td>GTG </td><td>GTG </td><td><strong>GTG*</strong> </td><td>GTG </td><td><strong>GTC*</strong> </td><td><strong>GTC*</strong> </td><td>GTT </td><td>GTT </td><td>GTT </td><td>GTT </td></tr><tr><td>Arg </td><td><strong>CGC*</strong> </td><td><strong>CGC*</strong> </td><td><strong>CGC*</strong> </td><td><strong>CGC*</strong> </td><td><strong>CGC*</strong> </td><td>CGT </td><td>CGT </td><td>AGA </td><td>AGA </td><td>AGA </td></tr><tr><td>Asp </td><td><strong>GAT*</strong> </td><td>GAC </td><td>GAC </td><td>GAC </td><td><strong>GAT*</strong> </td><td>GAC </td><td><strong>GAT*</strong> </td><td>GAC </td><td><strong>GAT*</strong> </td><td><strong>GAT*</strong> </td></tr></tbody></table></div><div content-id="dsw062-T1" class="table-modal"><table><thead name="thead" align=""><tr><th colspan="11">A.<hr /> </th></tr><tr><th> </th><th>Hs </th><th>Mm </th><th>Rn </th><th>Dm </th><th>Hc </th><th>Ce </th><th>Sp </th><th>Sc </th><th>Ca </th><th>Pf </th></tr></thead><tbody><tr><td>Phe </td><td>TTC </td><td>TTC </td><td>TTC </td><td>TTC </td><td>TTC </td><td>TTC </td><td>TTC </td><td>TTC </td><td>TTC </td><td><strong>TTT*</strong> </td></tr><tr><td>His </td><td>CAC </td><td>CAC </td><td>CAC </td><td>CAC </td><td>CAC </td><td>CAC </td><td>CAC </td><td>CAC </td><td>CAC </td><td>CAT </td></tr><tr><td>Tyr </td><td>TAC </td><td>TAC </td><td>TAC </td><td>TAC </td><td>TAC </td><td>TAC </td><td>TAC </td><td>TAC </td><td>TAC </td><td><strong>TAT*</strong> </td></tr><tr><td>Asn </td><td>AAC </td><td>AAC </td><td>AAC </td><td>AAC </td><td>AAC </td><td>AAC </td><td>AAC </td><td>AAC </td><td>AAC </td><td><strong>AAT*</strong> </td></tr><tr><td>Lys </td><td>AAG </td><td>AAG </td><td>AAG </td><td>AAG </td><td>AAG </td><td>AAG </td><td>AAG </td><td>AAG </td><td>AAA </td><td>AAA </td></tr><tr><td>Cys </td><td>TGC </td><td>TGC </td><td>TGC </td><td>TGC </td><td>TGC </td><td>TGC </td><td>TGC </td><td><strong>TGT*</strong> </td><td><strong>TGT*</strong> </td><td>TGT </td></tr><tr><td>Glu </td><td>GAG </td><td>GAG </td><td>GAG </td><td>GAG </td><td>GAG </td><td>GAG </td><td>GAG </td><td>GAA </td><td>GAA </td><td>GAA </td></tr><tr><td>Ala </td><td><strong>GCC*</strong> </td><td><strong>GCC*</strong> </td><td><strong>GCC*</strong> </td><td><strong>GCC*</strong> </td><td><strong>GCC*</strong> </td><td><strong>GCC*</strong> </td><td>GCT </td><td>GCT </td><td>GCT </td><td>GCT </td></tr><tr><td>Se </td><td><strong>TCC*</strong> </td><td><strong>TCC*</strong> </td><td><strong>TCC*</strong> </td><td><strong>TCC*</strong> </td><td><strong>TCC*</strong> </td><td><strong>TCC*</strong> </td><td>TCT </td><td>TCT </td><td>TCT </td><td>TCA </td></tr><tr><td>Thr </td><td><strong>ACC*</strong> </td><td><strong>ACC*</strong> </td><td><strong>ACC*</strong> </td><td><strong>ACC*</strong> </td><td><strong>ACC*</strong> </td><td><strong>ACC*</strong> </td><td>ACT </td><td>ACT </td><td>ACT </td><td>ACA </td></tr><tr><td>Ile </td><td><strong>ATC*</strong> </td><td><strong>ATC*</strong> </td><td><strong>ATC*</strong> </td><td><strong>ATC*</strong> </td><td><strong>ATC*</strong> </td><td><strong>ATC*</strong> </td><td>ATT </td><td><strong>ATC*</strong> </td><td>ATT </td><td>ATT </td></tr><tr><td>Gly </td><td>GGC </td><td>GGC </td><td>GGC </td><td>GGC </td><td><strong>GGC*</strong> </td><td><strong>GGA*</strong> </td><td><strong>GGT*</strong> </td><td><strong>GGT*</strong> </td><td><strong>GGT*</strong> </td><td>GGA </td></tr><tr><td>Pro </td><td>CCC </td><td><strong>CCC*</strong> </td><td><strong>CCC*</strong> </td><td><strong>CCC*</strong> </td><td><strong>CCC*</strong> </td><td>CCA </td><td>CCT </td><td>CCA </td><td>CCA </td><td>CCA </td></tr><tr><td>Gln </td><td><strong>CAG*</strong> </td><td><strong>CAG*</strong> </td><td><strong>CAG*</strong> </td><td><strong>CAG*</strong> </td><td><strong>CAG*</strong> </td><td>CAA </td><td>CAA </td><td>CAA </td><td>CAA </td><td>CAA </td></tr><tr><td>Leu </td><td><strong>CTG*</strong> </td><td>CTG </td><td>CTG </td><td><strong>CTG*</strong> </td><td><strong>CTC*</strong> </td><td><strong>CTC*</strong> </td><td><strong>TTG*</strong> </td><td>TTG </td><td>TTG </td><td>TTA </td></tr><tr><td>Val </td><td>GTG </td><td>GTG </td><td><strong>GTG*</strong> </td><td>GTG </td><td><strong>GTC*</strong> </td><td><strong>GTC*</strong> </td><td>GTT </td><td>GTT </td><td>GTT </td><td>GTT </td></tr><tr><td>Arg </td><td><strong>CGC*</strong> </td><td><strong>CGC*</strong> </td><td><strong>CGC*</strong> </td><td><strong>CGC*</strong> </td><td><strong>CGC*</strong> </td><td>CGT </td><td>CGT </td><td>AGA </td><td>AGA </td><td>AGA </td></tr><tr><td>Asp </td><td><strong>GAT*</strong> </td><td>GAC </td><td>GAC </td><td>GAC </td><td><strong>GAT*</strong> </td><td>GAC </td><td><strong>GAT*</strong> </td><td>GAC </td><td><strong>GAT*</strong> </td><td><strong>GAT*</strong> </td></tr></tbody></table></div><div class="table-overflow"><table><thead name="thead" align=""><tr><th colspan="11">B.<hr /> </th></tr><tr><th> </th><th>Hs </th><th>Ms </th><th>Rn </th><th>Dm </th><th>Hc </th><th>Ce </th><th>Sp </th><th>Sc </th><th>Ca </th><th>Pf </th></tr></thead><tbody><tr><td>Phe </td><td>TTT </td><td><strong><span class="underline">TTC</span></strong> </td><td><strong><span class="underline">TTC</span></strong> </td><td><strong><span class="underline">TTC</span></strong> </td><td><strong><span class="underline">TTC</span></strong> </td><td><strong><span class="underline">TTC</span></strong> </td><td>TTT </td><td>TTT </td><td>TTT </td><td><strong><span class="underline">TTT</span></strong> </td></tr><tr><td>His </td><td><strong><span class="underline">CAC</span></strong> </td><td><strong><span class="underline">CAC</span></strong> </td><td><strong><span class="underline">CAC</span></strong> </td><td><strong><span class="underline">CAC</span></strong> </td><td>CAT </td><td>CAT </td><td>CAT </td><td>CAT </td><td>CAT </td><td><strong><span class="underline">CAT</span></strong> </td></tr><tr><td>Tyr </td><td><strong><span class="underline">TAC</span></strong> </td><td><strong><span class="underline">TAC</span></strong> </td><td><strong><span class="underline">TAC</span></strong> </td><td><strong><span class="underline">TAC</span></strong> </td><td><strong><span class="underline">TAC</span></strong> </td><td>TAT </td><td>TAT </td><td>TAT </td><td>TAT </td><td><strong><span class="underline">TAT</span></strong> </td></tr><tr><td>Asn </td><td>AAT </td><td><strong><span class="underline">AAC</span></strong> </td><td><strong><span class="underline">AAC</span></strong> </td><td><strong><span class="underline">AAC</span></strong> </td><td>AAT </td><td>AAT </td><td>AAT </td><td>AAT </td><td>AAT </td><td><strong><span class="underline">AAT</span></strong> </td></tr><tr><td>Lys </td><td><strong><span class="underline">AAG</span></strong> </td><td><strong><span class="underline">AAG</span></strong> </td><td><strong><span class="underline">AAG</span></strong> </td><td><strong><span class="underline">AAG</span></strong> </td><td><strong><span class="underline">AAG</span></strong> </td><td>AAA </td><td>AAA </td><td>AAA </td><td><strong><span class="underline">AAA</span></strong> </td><td><strong><span class="underline">AAA</span></strong> </td></tr><tr><td>Cys </td><td><strong><span class="underline">TGC</span></strong> </td><td><strong><span class="underline">TGC</span></strong> </td><td><strong><span class="underline">TGC</span></strong> </td><td><strong><span class="underline">TGC</span></strong> </td><td><strong><span class="underline">TGC</span></strong> </td><td>TGT </td><td>TGT </td><td><strong><span class="underline">TGT</span></strong> </td><td><strong><span class="underline">TGT</span></strong> </td><td><strong><span class="underline">TGT</span></strong> </td></tr><tr><td>Glu </td><td><strong><span class="underline">GAG</span></strong> </td><td><strong><span class="underline">GAG</span></strong> </td><td><strong><span class="underline">GAG</span></strong> </td><td><strong><span class="underline">GAG</span></strong> </td><td>GAA </td><td>GAA </td><td>GAA </td><td><strong><span class="underline">GAA</span></strong> </td><td><strong><span class="underline">GAA</span></strong> </td><td><strong><span class="underline">GAA</span></strong> </td></tr><tr><td>Ala </td><td><strong><span class="underline">GCC</span></strong> </td><td><strong><span class="underline">GCC</span></strong> </td><td><strong><span class="underline">GCC</span></strong> </td><td><strong><span class="underline">GCC</span></strong> </td><td><strong><span class="underline">GCC</span></strong> </td><td>GCT </td><td><strong><span class="underline">GCT</span></strong> </td><td><strong><span class="underline">GCT</span></strong> </td><td><strong><span class="underline">GCT</span></strong> </td><td><strong><span class="underline">GCT</span></strong> </td></tr><tr><td>Ser </td><td>AGC </td><td>AGC </td><td>AGC </td><td>AGC </td><td>TCC </td><td>TCA </td><td><strong><span class="underline">TCT</span></strong> </td><td><strong><span class="underline">TCT</span></strong> </td><td>TCA </td><td>AGT </td></tr><tr><td>Thr </td><td><strong><span class="underline">ACC</span></strong> </td><td><strong><span class="underline">ACC</span></strong> </td><td><strong><span class="underline">ACC</span></strong> </td><td><strong><span class="underline">ACC</span></strong> </td><td><strong><span class="underline">ACC</span></strong> </td><td>ACA </td><td><strong><span class="underline">ACT</span></strong> </td><td><strong><span class="underline">ACT</span></strong> </td><td><strong><span class="underline">ACT</span></strong> </td><td><strong><span class="underline">ACA</span></strong> </td></tr><tr><td>Ile </td><td><strong><span class="underline">ATC</span></strong> </td><td><strong><span class="underline">ATC</span></strong> </td><td><strong><span class="underline">ATC</span></strong> </td><td><strong><span class="underline">ATC</span></strong> </td><td><strong><span class="underline">ATC</span></strong> </td><td>ATT </td><td><strong><span class="underline">ATT</span></strong> </td><td>ATT </td><td><strong><span class="underline">ATT</span></strong> </td><td>ATA </td></tr><tr><td>Gly </td><td><strong><span class="underline">GGC</span></strong> </td><td><strong><span class="underline">GGC</span></strong> </td><td><strong><span class="underline">GGC</span></strong> </td><td><strong><span class="underline">GGC</span></strong> </td><td><strong><span class="underline">GGC</span></strong> </td><td><strong><span class="underline">GGA</span></strong> </td><td><strong><span class="underline">GGT</span></strong> </td><td><strong><span class="underline">GGT</span></strong> </td><td><strong><span class="underline">GGT</span></strong> </td><td><strong><span class="underline">GGA</span></strong> </td></tr><tr><td>Pro </td><td><strong><span class="underline">CCC</span></strong> </td><td><strong><span class="underline">CCC</span></strong> </td><td><strong><span class="underline">CCC</span></strong> </td><td><strong><span class="underline">CCC</span></strong> </td><td>CCA </td><td><strong><span class="underline">CCA</span></strong> </td><td><strong><span class="underline">CCT</span></strong> </td><td><strong><span class="underline">CCA</span></strong> </td><td><strong><span class="underline">CCA</span></strong> </td><td><strong><span class="underline">CCA</span></strong> </td></tr><tr><td>Gln </td><td><strong><span class="underline">CAG</span></strong> </td><td><strong><span class="underline">CAG</span></strong> </td><td><strong><span class="underline">CAG</span></strong> </td><td><strong><span class="underline">CAG</span></strong> </td><td><strong><span class="underline">CAG</span></strong> </td><td><strong><span class="underline">CAA</span></strong> </td><td><strong><span class="underline">CAA</span></strong> </td><td><strong><span class="underline">CAA</span></strong> </td><td><strong><span class="underline">CAA</span></strong> </td><td><strong><span class="underline">CAA</span></strong> </td></tr><tr><td>Leu </td><td><strong><span class="underline">CTG</span></strong> </td><td><strong><span class="underline">CTG</span></strong> </td><td><strong><span class="underline">CTG</span></strong> </td><td><strong><span class="underline">CTG</span></strong> </td><td><strong><span class="underline">CTC</span></strong> </td><td>CTT </td><td>TTA </td><td><strong><span class="underline">TTG</span></strong> </td><td>TTA </td><td>TTA </td></tr><tr><td>Val </td><td><strong><span class="underline">GTG</span></strong> </td><td><strong><span class="underline">GTG</span></strong> </td><td><strong><span class="underline">GTG</span></strong> </td><td><strong><span class="underline">GTG</span></strong> </td><td><strong><span class="underline">GTC</span></strong> </td><td>GTT </td><td><strong><span class="underline">GTT</span></strong> </td><td><strong><span class="underline">GTT</span></strong> </td><td><strong><span class="underline">GTT</span></strong> </td><td>GTA </td></tr><tr><td>Arg </td><td><strong><span class="underline">CGG</span></strong> </td><td><strong><span class="underline">CGG</span></strong> </td><td><strong><span class="underline">CGG</span></strong> </td><td><strong><span class="underline">CGG</span></strong> </td><td><strong><span class="underline">CGG</span></strong> </td><td>AGA </td><td><strong><span class="underline">CGT</span></strong> </td><td><strong><span class="underline">AGA</span></strong> </td><td><strong><span class="underline">AGA</span></strong> </td><td><strong><span class="underline">AGA</span></strong> </td></tr><tr><td>Asp </td><td><strong><span class="underline">GAT</span></strong> </td><td><strong><span class="underline">GAC</span></strong> </td><td><strong><span class="underline">GAC</span></strong> </td><td><strong><span class="underline">GAC</span></strong> </td><td><strong><span class="underline">GAT</span></strong> </td><td>GAT </td><td><strong><span class="underline">GAT</span></strong> </td><td>GAT </td><td><strong><span class="underline">GAT</span></strong> </td><td><strong><span class="underline">GAT</span></strong> </td></tr></tbody></table></div><div content-id="dsw062-T1" class="table-modal"><table><thead name="thead" align=""><tr><th colspan="11">B.<hr /> </th></tr><tr><th> </th><th>Hs </th><th>Ms </th><th>Rn </th><th>Dm </th><th>Hc </th><th>Ce </th><th>Sp </th><th>Sc </th><th>Ca </th><th>Pf </th></tr></thead><tbody><tr><td>Phe </td><td>TTT </td><td><strong><span class="underline">TTC</span></strong> </td><td><strong><span class="underline">TTC</span></strong> </td><td><strong><span class="underline">TTC</span></strong> </td><td><strong><span class="underline">TTC</span></strong> </td><td><strong><span class="underline">TTC</span></strong> </td><td>TTT </td><td>TTT </td><td>TTT </td><td><strong><span class="underline">TTT</span></strong> </td></tr><tr><td>His </td><td><strong><span class="underline">CAC</span></strong> </td><td><strong><span class="underline">CAC</span></strong> </td><td><strong><span class="underline">CAC</span></strong> </td><td><strong><span class="underline">CAC</span></strong> </td><td>CAT </td><td>CAT </td><td>CAT </td><td>CAT </td><td>CAT </td><td><strong><span class="underline">CAT</span></strong> </td></tr><tr><td>Tyr </td><td><strong><span class="underline">TAC</span></strong> </td><td><strong><span class="underline">TAC</span></strong> </td><td><strong><span class="underline">TAC</span></strong> </td><td><strong><span class="underline">TAC</span></strong> </td><td><strong><span class="underline">TAC</span></strong> </td><td>TAT </td><td>TAT </td><td>TAT </td><td>TAT </td><td><strong><span class="underline">TAT</span></strong> </td></tr><tr><td>Asn </td><td>AAT </td><td><strong><span class="underline">AAC</span></strong> </td><td><strong><span class="underline">AAC</span></strong> </td><td><strong><span class="underline">AAC</span></strong> </td><td>AAT </td><td>AAT </td><td>AAT </td><td>AAT </td><td>AAT </td><td><strong><span class="underline">AAT</span></strong> </td></tr><tr><td>Lys </td><td><strong><span class="underline">AAG</span></strong> </td><td><strong><span class="underline">AAG</span></strong> </td><td><strong><span class="underline">AAG</span></strong> </td><td><strong><span class="underline">AAG</span></strong> </td><td><strong><span class="underline">AAG</span></strong> </td><td>AAA </td><td>AAA </td><td>AAA </td><td><strong><span class="underline">AAA</span></strong> </td><td><strong><span class="underline">AAA</span></strong> </td></tr><tr><td>Cys </td><td><strong><span class="underline">TGC</span></strong> </td><td><strong><span class="underline">TGC</span></strong> </td><td><strong><span class="underline">TGC</span></strong> </td><td><strong><span class="underline">TGC</span></strong> </td><td><strong><span class="underline">TGC</span></strong> </td><td>TGT </td><td>TGT </td><td><strong><span class="underline">TGT</span></strong> </td><td><strong><span class="underline">TGT</span></strong> </td><td><strong><span class="underline">TGT</span></strong> </td></tr><tr><td>Glu </td><td><strong><span class="underline">GAG</span></strong> </td><td><strong><span class="underline">GAG</span></strong> </td><td><strong><span class="underline">GAG</span></strong> </td><td><strong><span class="underline">GAG</span></strong> </td><td>GAA </td><td>GAA </td><td>GAA </td><td><strong><span class="underline">GAA</span></strong> </td><td><strong><span class="underline">GAA</span></strong> </td><td><strong><span class="underline">GAA</span></strong> </td></tr><tr><td>Ala </td><td><strong><span class="underline">GCC</span></strong> </td><td><strong><span class="underline">GCC</span></strong> </td><td><strong><span class="underline">GCC</span></strong> </td><td><strong><span class="underline">GCC</span></strong> </td><td><strong><span class="underline">GCC</span></strong> </td><td>GCT </td><td><strong><span class="underline">GCT</span></strong> </td><td><strong><span class="underline">GCT</span></strong> </td><td><strong><span class="underline">GCT</span></strong> </td><td><strong><span class="underline">GCT</span></strong> </td></tr><tr><td>Ser </td><td>AGC </td><td>AGC </td><td>AGC </td><td>AGC </td><td>TCC </td><td>TCA </td><td><strong><span class="underline">TCT</span></strong> </td><td><strong><span class="underline">TCT</span></strong> </td><td>TCA </td><td>AGT </td></tr><tr><td>Thr </td><td><strong><span class="underline">ACC</span></strong> </td><td><strong><span class="underline">ACC</span></strong> </td><td><strong><span class="underline">ACC</span></strong> </td><td><strong><span class="underline">ACC</span></strong> </td><td><strong><span class="underline">ACC</span></strong> </td><td>ACA </td><td><strong><span class="underline">ACT</span></strong> </td><td><strong><span class="underline">ACT</span></strong> </td><td><strong><span class="underline">ACT</span></strong> </td><td><strong><span class="underline">ACA</span></strong> </td></tr><tr><td>Ile </td><td><strong><span class="underline">ATC</span></strong> </td><td><strong><span class="underline">ATC</span></strong> </td><td><strong><span class="underline">ATC</span></strong> </td><td><strong><span class="underline">ATC</span></strong> </td><td><strong><span class="underline">ATC</span></strong> </td><td>ATT </td><td><strong><span class="underline">ATT</span></strong> </td><td>ATT </td><td><strong><span class="underline">ATT</span></strong> </td><td>ATA </td></tr><tr><td>Gly </td><td><strong><span class="underline">GGC</span></strong> </td><td><strong><span class="underline">GGC</span></strong> </td><td><strong><span class="underline">GGC</span></strong> </td><td><strong><span class="underline">GGC</span></strong> </td><td><strong><span class="underline">GGC</span></strong> </td><td><strong><span class="underline">GGA</span></strong> </td><td><strong><span class="underline">GGT</span></strong> </td><td><strong><span class="underline">GGT</span></strong> </td><td><strong><span class="underline">GGT</span></strong> </td><td><strong><span class="underline">GGA</span></strong> </td></tr><tr><td>Pro </td><td><strong><span class="underline">CCC</span></strong> </td><td><strong><span class="underline">CCC</span></strong> </td><td><strong><span class="underline">CCC</span></strong> </td><td><strong><span class="underline">CCC</span></strong> </td><td>CCA </td><td><strong><span class="underline">CCA</span></strong> </td><td><strong><span class="underline">CCT</span></strong> </td><td><strong><span class="underline">CCA</span></strong> </td><td><strong><span class="underline">CCA</span></strong> </td><td><strong><span class="underline">CCA</span></strong> </td></tr><tr><td>Gln </td><td><strong><span class="underline">CAG</span></strong> </td><td><strong><span class="underline">CAG</span></strong> </td><td><strong><span class="underline">CAG</span></strong> </td><td><strong><span class="underline">CAG</span></strong> </td><td><strong><span class="underline">CAG</span></strong> </td><td><strong><span class="underline">CAA</span></strong> </td><td><strong><span class="underline">CAA</span></strong> </td><td><strong><span class="underline">CAA</span></strong> </td><td><strong><span class="underline">CAA</span></strong> </td><td><strong><span class="underline">CAA</span></strong> </td></tr><tr><td>Leu </td><td><strong><span class="underline">CTG</span></strong> </td><td><strong><span class="underline">CTG</span></strong> </td><td><strong><span class="underline">CTG</span></strong> </td><td><strong><span class="underline">CTG</span></strong> </td><td><strong><span class="underline">CTC</span></strong> </td><td>CTT </td><td>TTA </td><td><strong><span class="underline">TTG</span></strong> </td><td>TTA </td><td>TTA </td></tr><tr><td>Val </td><td><strong><span class="underline">GTG</span></strong> </td><td><strong><span class="underline">GTG</span></strong> </td><td><strong><span class="underline">GTG</span></strong> </td><td><strong><span class="underline">GTG</span></strong> </td><td><strong><span class="underline">GTC</span></strong> </td><td>GTT </td><td><strong><span class="underline">GTT</span></strong> </td><td><strong><span class="underline">GTT</span></strong> </td><td><strong><span class="underline">GTT</span></strong> </td><td>GTA </td></tr><tr><td>Arg </td><td><strong><span class="underline">CGG</span></strong> </td><td><strong><span class="underline">CGG</span></strong> </td><td><strong><span class="underline">CGG</span></strong> </td><td><strong><span class="underline">CGG</span></strong> </td><td><strong><span class="underline">CGG</span></strong> </td><td>AGA </td><td><strong><span class="underline">CGT</span></strong> </td><td><strong><span class="underline">AGA</span></strong> </td><td><strong><span class="underline">AGA</span></strong> </td><td><strong><span class="underline">AGA</span></strong> </td></tr><tr><td>Asp </td><td><strong><span class="underline">GAT</span></strong> </td><td><strong><span class="underline">GAC</span></strong> </td><td><strong><span class="underline">GAC</span></strong> </td><td><strong><span class="underline">GAC</span></strong> </td><td><strong><span class="underline">GAT</span></strong> </td><td>GAT </td><td><strong><span class="underline">GAT</span></strong> </td><td>GAT </td><td><strong><span class="underline">GAT</span></strong> </td><td><strong><span class="underline">GAT</span></strong> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>Color code: on a blue background codons shared with <em>M. musculus</em>, on a red background, codons shared with <em>C. albicans.</em> A: Bolded codons with a star are preferred codons for which a codon with more acceptors exists. B: bolded and underlined codons are identical in lowly and highly translated genes. The following abbreviations of species names are used throughout this article: H. sapiens (Hs), <em>H. sapiens (Hs)</em>, <em>M. musculus (Mm)</em>, <em>R. norvegicus (Rn)</em>, <em>D. melanogaster (Dm)</em>, <em>C. elegans (Ce), H. capsulatum (Hc), S. pombe (Sp)</em>, <em>S. cerevisiae (Sc), C. albicans (Ca</em>), and <em>P. falciparum (Pf)</em>. Refer to the online version for colors.</p></div></div></div></div></div><div class="table-wrap"><div id="dsw062-T1" data-id="dsw062-T1" class="table-wrap-title"><div class="label -label">Table 1</div><div class="caption"><p>Preferred codons per organism in highly (A) and lowly (B) translated genes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th colspan="11">A.<hr /> </th></tr><tr><th> </th><th>Hs </th><th>Mm </th><th>Rn </th><th>Dm </th><th>Hc </th><th>Ce </th><th>Sp </th><th>Sc </th><th>Ca </th><th>Pf </th></tr></thead><tbody><tr><td>Phe </td><td>TTC </td><td>TTC </td><td>TTC </td><td>TTC </td><td>TTC </td><td>TTC </td><td>TTC </td><td>TTC </td><td>TTC </td><td><strong>TTT*</strong> </td></tr><tr><td>His </td><td>CAC </td><td>CAC </td><td>CAC </td><td>CAC </td><td>CAC </td><td>CAC </td><td>CAC </td><td>CAC </td><td>CAC </td><td>CAT </td></tr><tr><td>Tyr </td><td>TAC </td><td>TAC </td><td>TAC </td><td>TAC </td><td>TAC </td><td>TAC </td><td>TAC </td><td>TAC </td><td>TAC </td><td><strong>TAT*</strong> </td></tr><tr><td>Asn </td><td>AAC </td><td>AAC </td><td>AAC </td><td>AAC </td><td>AAC </td><td>AAC </td><td>AAC </td><td>AAC </td><td>AAC </td><td><strong>AAT*</strong> </td></tr><tr><td>Lys </td><td>AAG </td><td>AAG </td><td>AAG </td><td>AAG </td><td>AAG </td><td>AAG </td><td>AAG </td><td>AAG </td><td>AAA </td><td>AAA </td></tr><tr><td>Cys </td><td>TGC </td><td>TGC </td><td>TGC </td><td>TGC </td><td>TGC </td><td>TGC </td><td>TGC </td><td><strong>TGT*</strong> </td><td><strong>TGT*</strong> </td><td>TGT </td></tr><tr><td>Glu </td><td>GAG </td><td>GAG </td><td>GAG </td><td>GAG </td><td>GAG </td><td>GAG </td><td>GAG </td><td>GAA </td><td>GAA </td><td>GAA </td></tr><tr><td>Ala </td><td><strong>GCC*</strong> </td><td><strong>GCC*</strong> </td><td><strong>GCC*</strong> </td><td><strong>GCC*</strong> </td><td><strong>GCC*</strong> </td><td><strong>GCC*</strong> </td><td>GCT </td><td>GCT </td><td>GCT </td><td>GCT </td></tr><tr><td>Se </td><td><strong>TCC*</strong> </td><td><strong>TCC*</strong> </td><td><strong>TCC*</strong> </td><td><strong>TCC*</strong> </td><td><strong>TCC*</strong> </td><td><strong>TCC*</strong> </td><td>TCT </td><td>TCT </td><td>TCT </td><td>TCA </td></tr><tr><td>Thr </td><td><strong>ACC*</strong> </td><td><strong>ACC*</strong> </td><td><strong>ACC*</strong> </td><td><strong>ACC*</strong> </td><td><strong>ACC*</strong> </td><td><strong>ACC*</strong> </td><td>ACT </td><td>ACT </td><td>ACT </td><td>ACA </td></tr><tr><td>Ile </td><td><strong>ATC*</strong> </td><td><strong>ATC*</strong> </td><td><strong>ATC*</strong> </td><td><strong>ATC*</strong> </td><td><strong>ATC*</strong> </td><td><strong>ATC*</strong> </td><td>ATT </td><td><strong>ATC*</strong> </td><td>ATT </td><td>ATT </td></tr><tr><td>Gly </td><td>GGC </td><td>GGC </td><td>GGC </td><td>GGC </td><td><strong>GGC*</strong> </td><td><strong>GGA*</strong> </td><td><strong>GGT*</strong> </td><td><strong>GGT*</strong> </td><td><strong>GGT*</strong> </td><td>GGA </td></tr><tr><td>Pro </td><td>CCC </td><td><strong>CCC*</strong> </td><td><strong>CCC*</strong> </td><td><strong>CCC*</strong> </td><td><strong>CCC*</strong> </td><td>CCA </td><td>CCT </td><td>CCA </td><td>CCA </td><td>CCA </td></tr><tr><td>Gln </td><td><strong>CAG*</strong> </td><td><strong>CAG*</strong> </td><td><strong>CAG*</strong> </td><td><strong>CAG*</strong> </td><td><strong>CAG*</strong> </td><td>CAA </td><td>CAA </td><td>CAA </td><td>CAA </td><td>CAA </td></tr><tr><td>Leu </td><td><strong>CTG*</strong> </td><td>CTG </td><td>CTG </td><td><strong>CTG*</strong> </td><td><strong>CTC*</strong> </td><td><strong>CTC*</strong> </td><td><strong>TTG*</strong> </td><td>TTG </td><td>TTG </td><td>TTA </td></tr><tr><td>Val </td><td>GTG </td><td>GTG </td><td><strong>GTG*</strong> </td><td>GTG </td><td><strong>GTC*</strong> </td><td><strong>GTC*</strong> </td><td>GTT </td><td>GTT </td><td>GTT </td><td>GTT </td></tr><tr><td>Arg </td><td><strong>CGC*</strong> </td><td><strong>CGC*</strong> </td><td><strong>CGC*</strong> </td><td><strong>CGC*</strong> </td><td><strong>CGC*</strong> </td><td>CGT </td><td>CGT </td><td>AGA </td><td>AGA </td><td>AGA </td></tr><tr><td>Asp </td><td><strong>GAT*</strong> </td><td>GAC </td><td>GAC </td><td>GAC </td><td><strong>GAT*</strong> </td><td>GAC </td><td><strong>GAT*</strong> </td><td>GAC </td><td><strong>GAT*</strong> </td><td><strong>GAT*</strong> </td></tr></tbody></table></div><div content-id="dsw062-T1" class="table-modal"><table><thead name="thead" align=""><tr><th colspan="11">A.<hr /> </th></tr><tr><th> </th><th>Hs </th><th>Mm </th><th>Rn </th><th>Dm </th><th>Hc </th><th>Ce </th><th>Sp </th><th>Sc </th><th>Ca </th><th>Pf </th></tr></thead><tbody><tr><td>Phe </td><td>TTC </td><td>TTC </td><td>TTC </td><td>TTC </td><td>TTC </td><td>TTC </td><td>TTC </td><td>TTC </td><td>TTC </td><td><strong>TTT*</strong> </td></tr><tr><td>His </td><td>CAC </td><td>CAC </td><td>CAC </td><td>CAC </td><td>CAC </td><td>CAC </td><td>CAC </td><td>CAC </td><td>CAC </td><td>CAT </td></tr><tr><td>Tyr </td><td>TAC </td><td>TAC </td><td>TAC </td><td>TAC </td><td>TAC </td><td>TAC </td><td>TAC </td><td>TAC </td><td>TAC </td><td><strong>TAT*</strong> </td></tr><tr><td>Asn </td><td>AAC </td><td>AAC </td><td>AAC </td><td>AAC </td><td>AAC </td><td>AAC </td><td>AAC </td><td>AAC </td><td>AAC </td><td><strong>AAT*</strong> </td></tr><tr><td>Lys </td><td>AAG </td><td>AAG </td><td>AAG </td><td>AAG </td><td>AAG </td><td>AAG </td><td>AAG </td><td>AAG </td><td>AAA </td><td>AAA </td></tr><tr><td>Cys </td><td>TGC </td><td>TGC </td><td>TGC </td><td>TGC </td><td>TGC </td><td>TGC </td><td>TGC </td><td><strong>TGT*</strong> </td><td><strong>TGT*</strong> </td><td>TGT </td></tr><tr><td>Glu </td><td>GAG </td><td>GAG </td><td>GAG </td><td>GAG </td><td>GAG </td><td>GAG </td><td>GAG </td><td>GAA </td><td>GAA </td><td>GAA </td></tr><tr><td>Ala </td><td><strong>GCC*</strong> </td><td><strong>GCC*</strong> </td><td><strong>GCC*</strong> </td><td><strong>GCC*</strong> </td><td><strong>GCC*</strong> </td><td><strong>GCC*</strong> </td><td>GCT </td><td>GCT </td><td>GCT </td><td>GCT </td></tr><tr><td>Se </td><td><strong>TCC*</strong> </td><td><strong>TCC*</strong> </td><td><strong>TCC*</strong> </td><td><strong>TCC*</strong> </td><td><strong>TCC*</strong> </td><td><strong>TCC*</strong> </td><td>TCT </td><td>TCT </td><td>TCT </td><td>TCA </td></tr><tr><td>Thr </td><td><strong>ACC*</strong> </td><td><strong>ACC*</strong> </td><td><strong>ACC*</strong> </td><td><strong>ACC*</strong> </td><td><strong>ACC*</strong> </td><td><strong>ACC*</strong> </td><td>ACT </td><td>ACT </td><td>ACT </td><td>ACA </td></tr><tr><td>Ile </td><td><strong>ATC*</strong> </td><td><strong>ATC*</strong> </td><td><strong>ATC*</strong> </td><td><strong>ATC*</strong> </td><td><strong>ATC*</strong> </td><td><strong>ATC*</strong> </td><td>ATT </td><td><strong>ATC*</strong> </td><td>ATT </td><td>ATT </td></tr><tr><td>Gly </td><td>GGC </td><td>GGC </td><td>GGC </td><td>GGC </td><td><strong>GGC*</strong> </td><td><strong>GGA*</strong> </td><td><strong>GGT*</strong> </td><td><strong>GGT*</strong> </td><td><strong>GGT*</strong> </td><td>GGA </td></tr><tr><td>Pro </td><td>CCC </td><td><strong>CCC*</strong> </td><td><strong>CCC*</strong> </td><td><strong>CCC*</strong> </td><td><strong>CCC*</strong> </td><td>CCA </td><td>CCT </td><td>CCA </td><td>CCA </td><td>CCA </td></tr><tr><td>Gln </td><td><strong>CAG*</strong> </td><td><strong>CAG*</strong> </td><td><strong>CAG*</strong> </td><td><strong>CAG*</strong> </td><td><strong>CAG*</strong> </td><td>CAA </td><td>CAA </td><td>CAA </td><td>CAA </td><td>CAA </td></tr><tr><td>Leu </td><td><strong>CTG*</strong> </td><td>CTG </td><td>CTG </td><td><strong>CTG*</strong> </td><td><strong>CTC*</strong> </td><td><strong>CTC*</strong> </td><td><strong>TTG*</strong> </td><td>TTG </td><td>TTG </td><td>TTA </td></tr><tr><td>Val </td><td>GTG </td><td>GTG </td><td><strong>GTG*</strong> </td><td>GTG </td><td><strong>GTC*</strong> </td><td><strong>GTC*</strong> </td><td>GTT </td><td>GTT </td><td>GTT </td><td>GTT </td></tr><tr><td>Arg </td><td><strong>CGC*</strong> </td><td><strong>CGC*</strong> </td><td><strong>CGC*</strong> </td><td><strong>CGC*</strong> </td><td><strong>CGC*</strong> </td><td>CGT </td><td>CGT </td><td>AGA </td><td>AGA </td><td>AGA </td></tr><tr><td>Asp </td><td><strong>GAT*</strong> </td><td>GAC </td><td>GAC </td><td>GAC </td><td><strong>GAT*</strong> </td><td>GAC </td><td><strong>GAT*</strong> </td><td>GAC </td><td><strong>GAT*</strong> </td><td><strong>GAT*</strong> </td></tr></tbody></table></div><div class="table-overflow"><table><thead name="thead" align=""><tr><th colspan="11">B.<hr /> </th></tr><tr><th> </th><th>Hs </th><th>Ms </th><th>Rn </th><th>Dm </th><th>Hc </th><th>Ce </th><th>Sp </th><th>Sc </th><th>Ca </th><th>Pf </th></tr></thead><tbody><tr><td>Phe </td><td>TTT </td><td><strong><span class="underline">TTC</span></strong> </td><td><strong><span class="underline">TTC</span></strong> </td><td><strong><span class="underline">TTC</span></strong> </td><td><strong><span class="underline">TTC</span></strong> </td><td><strong><span class="underline">TTC</span></strong> </td><td>TTT </td><td>TTT </td><td>TTT </td><td><strong><span class="underline">TTT</span></strong> </td></tr><tr><td>His </td><td><strong><span class="underline">CAC</span></strong> </td><td><strong><span class="underline">CAC</span></strong> </td><td><strong><span class="underline">CAC</span></strong> </td><td><strong><span class="underline">CAC</span></strong> </td><td>CAT </td><td>CAT </td><td>CAT </td><td>CAT </td><td>CAT </td><td><strong><span class="underline">CAT</span></strong> </td></tr><tr><td>Tyr </td><td><strong><span class="underline">TAC</span></strong> </td><td><strong><span class="underline">TAC</span></strong> </td><td><strong><span class="underline">TAC</span></strong> </td><td><strong><span class="underline">TAC</span></strong> </td><td><strong><span class="underline">TAC</span></strong> </td><td>TAT </td><td>TAT </td><td>TAT </td><td>TAT </td><td><strong><span class="underline">TAT</span></strong> </td></tr><tr><td>Asn </td><td>AAT </td><td><strong><span class="underline">AAC</span></strong> </td><td><strong><span class="underline">AAC</span></strong> </td><td><strong><span class="underline">AAC</span></strong> </td><td>AAT </td><td>AAT </td><td>AAT </td><td>AAT </td><td>AAT </td><td><strong><span class="underline">AAT</span></strong> </td></tr><tr><td>Lys </td><td><strong><span class="underline">AAG</span></strong> </td><td><strong><span class="underline">AAG</span></strong> </td><td><strong><span class="underline">AAG</span></strong> </td><td><strong><span class="underline">AAG</span></strong> </td><td><strong><span class="underline">AAG</span></strong> </td><td>AAA </td><td>AAA </td><td>AAA </td><td><strong><span class="underline">AAA</span></strong> </td><td><strong><span class="underline">AAA</span></strong> </td></tr><tr><td>Cys </td><td><strong><span class="underline">TGC</span></strong> </td><td><strong><span class="underline">TGC</span></strong> </td><td><strong><span class="underline">TGC</span></strong> </td><td><strong><span class="underline">TGC</span></strong> </td><td><strong><span class="underline">TGC</span></strong> </td><td>TGT </td><td>TGT </td><td><strong><span class="underline">TGT</span></strong> </td><td><strong><span class="underline">TGT</span></strong> </td><td><strong><span class="underline">TGT</span></strong> </td></tr><tr><td>Glu </td><td><strong><span class="underline">GAG</span></strong> </td><td><strong><span class="underline">GAG</span></strong> </td><td><strong><span class="underline">GAG</span></strong> </td><td><strong><span class="underline">GAG</span></strong> </td><td>GAA </td><td>GAA </td><td>GAA </td><td><strong><span class="underline">GAA</span></strong> </td><td><strong><span class="underline">GAA</span></strong> </td><td><strong><span class="underline">GAA</span></strong> </td></tr><tr><td>Ala </td><td><strong><span class="underline">GCC</span></strong> </td><td><strong><span class="underline">GCC</span></strong> </td><td><strong><span class="underline">GCC</span></strong> </td><td><strong><span class="underline">GCC</span></strong> </td><td><strong><span class="underline">GCC</span></strong> </td><td>GCT </td><td><strong><span class="underline">GCT</span></strong> </td><td><strong><span class="underline">GCT</span></strong> </td><td><strong><span class="underline">GCT</span></strong> </td><td><strong><span class="underline">GCT</span></strong> </td></tr><tr><td>Ser </td><td>AGC </td><td>AGC </td><td>AGC </td><td>AGC </td><td>TCC </td><td>TCA </td><td><strong><span class="underline">TCT</span></strong> </td><td><strong><span class="underline">TCT</span></strong> </td><td>TCA </td><td>AGT </td></tr><tr><td>Thr </td><td><strong><span class="underline">ACC</span></strong> </td><td><strong><span class="underline">ACC</span></strong> </td><td><strong><span class="underline">ACC</span></strong> </td><td><strong><span class="underline">ACC</span></strong> </td><td><strong><span class="underline">ACC</span></strong> </td><td>ACA </td><td><strong><span class="underline">ACT</span></strong> </td><td><strong><span class="underline">ACT</span></strong> </td><td><strong><span class="underline">ACT</span></strong> </td><td><strong><span class="underline">ACA</span></strong> </td></tr><tr><td>Ile </td><td><strong><span class="underline">ATC</span></strong> </td><td><strong><span class="underline">ATC</span></strong> </td><td><strong><span class="underline">ATC</span></strong> </td><td><strong><span class="underline">ATC</span></strong> </td><td><strong><span class="underline">ATC</span></strong> </td><td>ATT </td><td><strong><span class="underline">ATT</span></strong> </td><td>ATT </td><td><strong><span class="underline">ATT</span></strong> </td><td>ATA </td></tr><tr><td>Gly </td><td><strong><span class="underline">GGC</span></strong> </td><td><strong><span class="underline">GGC</span></strong> </td><td><strong><span class="underline">GGC</span></strong> </td><td><strong><span class="underline">GGC</span></strong> </td><td><strong><span class="underline">GGC</span></strong> </td><td><strong><span class="underline">GGA</span></strong> </td><td><strong><span class="underline">GGT</span></strong> </td><td><strong><span class="underline">GGT</span></strong> </td><td><strong><span class="underline">GGT</span></strong> </td><td><strong><span class="underline">GGA</span></strong> </td></tr><tr><td>Pro </td><td><strong><span class="underline">CCC</span></strong> </td><td><strong><span class="underline">CCC</span></strong> </td><td><strong><span class="underline">CCC</span></strong> </td><td><strong><span class="underline">CCC</span></strong> </td><td>CCA </td><td><strong><span class="underline">CCA</span></strong> </td><td><strong><span class="underline">CCT</span></strong> </td><td><strong><span class="underline">CCA</span></strong> </td><td><strong><span class="underline">CCA</span></strong> </td><td><strong><span class="underline">CCA</span></strong> </td></tr><tr><td>Gln </td><td><strong><span class="underline">CAG</span></strong> </td><td><strong><span class="underline">CAG</span></strong> </td><td><strong><span class="underline">CAG</span></strong> </td><td><strong><span class="underline">CAG</span></strong> </td><td><strong><span class="underline">CAG</span></strong> </td><td><strong><span class="underline">CAA</span></strong> </td><td><strong><span class="underline">CAA</span></strong> </td><td><strong><span class="underline">CAA</span></strong> </td><td><strong><span class="underline">CAA</span></strong> </td><td><strong><span class="underline">CAA</span></strong> </td></tr><tr><td>Leu </td><td><strong><span class="underline">CTG</span></strong> </td><td><strong><span class="underline">CTG</span></strong> </td><td><strong><span class="underline">CTG</span></strong> </td><td><strong><span class="underline">CTG</span></strong> </td><td><strong><span class="underline">CTC</span></strong> </td><td>CTT </td><td>TTA </td><td><strong><span class="underline">TTG</span></strong> </td><td>TTA </td><td>TTA </td></tr><tr><td>Val </td><td><strong><span class="underline">GTG</span></strong> </td><td><strong><span class="underline">GTG</span></strong> </td><td><strong><span class="underline">GTG</span></strong> </td><td><strong><span class="underline">GTG</span></strong> </td><td><strong><span class="underline">GTC</span></strong> </td><td>GTT </td><td><strong><span class="underline">GTT</span></strong> </td><td><strong><span class="underline">GTT</span></strong> </td><td><strong><span class="underline">GTT</span></strong> </td><td>GTA </td></tr><tr><td>Arg </td><td><strong><span class="underline">CGG</span></strong> </td><td><strong><span class="underline">CGG</span></strong> </td><td><strong><span class="underline">CGG</span></strong> </td><td><strong><span class="underline">CGG</span></strong> </td><td><strong><span class="underline">CGG</span></strong> </td><td>AGA </td><td><strong><span class="underline">CGT</span></strong> </td><td><strong><span class="underline">AGA</span></strong> </td><td><strong><span class="underline">AGA</span></strong> </td><td><strong><span class="underline">AGA</span></strong> </td></tr><tr><td>Asp </td><td><strong><span class="underline">GAT</span></strong> </td><td><strong><span class="underline">GAC</span></strong> </td><td><strong><span class="underline">GAC</span></strong> </td><td><strong><span class="underline">GAC</span></strong> </td><td><strong><span class="underline">GAT</span></strong> </td><td>GAT </td><td><strong><span class="underline">GAT</span></strong> </td><td>GAT </td><td><strong><span class="underline">GAT</span></strong> </td><td><strong><span class="underline">GAT</span></strong> </td></tr></tbody></table></div><div content-id="dsw062-T1" class="table-modal"><table><thead name="thead" align=""><tr><th colspan="11">B.<hr /> </th></tr><tr><th> </th><th>Hs </th><th>Ms </th><th>Rn </th><th>Dm </th><th>Hc </th><th>Ce </th><th>Sp </th><th>Sc </th><th>Ca </th><th>Pf </th></tr></thead><tbody><tr><td>Phe </td><td>TTT </td><td><strong><span class="underline">TTC</span></strong> </td><td><strong><span class="underline">TTC</span></strong> </td><td><strong><span class="underline">TTC</span></strong> </td><td><strong><span class="underline">TTC</span></strong> </td><td><strong><span class="underline">TTC</span></strong> </td><td>TTT </td><td>TTT </td><td>TTT </td><td><strong><span class="underline">TTT</span></strong> </td></tr><tr><td>His </td><td><strong><span class="underline">CAC</span></strong> </td><td><strong><span class="underline">CAC</span></strong> </td><td><strong><span class="underline">CAC</span></strong> </td><td><strong><span class="underline">CAC</span></strong> </td><td>CAT </td><td>CAT </td><td>CAT </td><td>CAT </td><td>CAT </td><td><strong><span class="underline">CAT</span></strong> </td></tr><tr><td>Tyr </td><td><strong><span class="underline">TAC</span></strong> </td><td><strong><span class="underline">TAC</span></strong> </td><td><strong><span class="underline">TAC</span></strong> </td><td><strong><span class="underline">TAC</span></strong> </td><td><strong><span class="underline">TAC</span></strong> </td><td>TAT </td><td>TAT </td><td>TAT </td><td>TAT </td><td><strong><span class="underline">TAT</span></strong> </td></tr><tr><td>Asn </td><td>AAT </td><td><strong><span class="underline">AAC</span></strong> </td><td><strong><span class="underline">AAC</span></strong> </td><td><strong><span class="underline">AAC</span></strong> </td><td>AAT </td><td>AAT </td><td>AAT </td><td>AAT </td><td>AAT </td><td><strong><span class="underline">AAT</span></strong> </td></tr><tr><td>Lys </td><td><strong><span class="underline">AAG</span></strong> </td><td><strong><span class="underline">AAG</span></strong> </td><td><strong><span class="underline">AAG</span></strong> </td><td><strong><span class="underline">AAG</span></strong> </td><td><strong><span class="underline">AAG</span></strong> </td><td>AAA </td><td>AAA </td><td>AAA </td><td><strong><span class="underline">AAA</span></strong> </td><td><strong><span class="underline">AAA</span></strong> </td></tr><tr><td>Cys </td><td><strong><span class="underline">TGC</span></strong> </td><td><strong><span class="underline">TGC</span></strong> </td><td><strong><span class="underline">TGC</span></strong> </td><td><strong><span class="underline">TGC</span></strong> </td><td><strong><span class="underline">TGC</span></strong> </td><td>TGT </td><td>TGT </td><td><strong><span class="underline">TGT</span></strong> </td><td><strong><span class="underline">TGT</span></strong> </td><td><strong><span class="underline">TGT</span></strong> </td></tr><tr><td>Glu </td><td><strong><span class="underline">GAG</span></strong> </td><td><strong><span class="underline">GAG</span></strong> </td><td><strong><span class="underline">GAG</span></strong> </td><td><strong><span class="underline">GAG</span></strong> </td><td>GAA </td><td>GAA </td><td>GAA </td><td><strong><span class="underline">GAA</span></strong> </td><td><strong><span class="underline">GAA</span></strong> </td><td><strong><span class="underline">GAA</span></strong> </td></tr><tr><td>Ala </td><td><strong><span class="underline">GCC</span></strong> </td><td><strong><span class="underline">GCC</span></strong> </td><td><strong><span class="underline">GCC</span></strong> </td><td><strong><span class="underline">GCC</span></strong> </td><td><strong><span class="underline">GCC</span></strong> </td><td>GCT </td><td><strong><span class="underline">GCT</span></strong> </td><td><strong><span class="underline">GCT</span></strong> </td><td><strong><span class="underline">GCT</span></strong> </td><td><strong><span class="underline">GCT</span></strong> </td></tr><tr><td>Ser </td><td>AGC </td><td>AGC </td><td>AGC </td><td>AGC </td><td>TCC </td><td>TCA </td><td><strong><span class="underline">TCT</span></strong> </td><td><strong><span class="underline">TCT</span></strong> </td><td>TCA </td><td>AGT </td></tr><tr><td>Thr </td><td><strong><span class="underline">ACC</span></strong> </td><td><strong><span class="underline">ACC</span></strong> </td><td><strong><span class="underline">ACC</span></strong> </td><td><strong><span class="underline">ACC</span></strong> </td><td><strong><span class="underline">ACC</span></strong> </td><td>ACA </td><td><strong><span class="underline">ACT</span></strong> </td><td><strong><span class="underline">ACT</span></strong> </td><td><strong><span class="underline">ACT</span></strong> </td><td><strong><span class="underline">ACA</span></strong> </td></tr><tr><td>Ile </td><td><strong><span class="underline">ATC</span></strong> </td><td><strong><span class="underline">ATC</span></strong> </td><td><strong><span class="underline">ATC</span></strong> </td><td><strong><span class="underline">ATC</span></strong> </td><td><strong><span class="underline">ATC</span></strong> </td><td>ATT </td><td><strong><span class="underline">ATT</span></strong> </td><td>ATT </td><td><strong><span class="underline">ATT</span></strong> </td><td>ATA </td></tr><tr><td>Gly </td><td><strong><span class="underline">GGC</span></strong> </td><td><strong><span class="underline">GGC</span></strong> </td><td><strong><span class="underline">GGC</span></strong> </td><td><strong><span class="underline">GGC</span></strong> </td><td><strong><span class="underline">GGC</span></strong> </td><td><strong><span class="underline">GGA</span></strong> </td><td><strong><span class="underline">GGT</span></strong> </td><td><strong><span class="underline">GGT</span></strong> </td><td><strong><span class="underline">GGT</span></strong> </td><td><strong><span class="underline">GGA</span></strong> </td></tr><tr><td>Pro </td><td><strong><span class="underline">CCC</span></strong> </td><td><strong><span class="underline">CCC</span></strong> </td><td><strong><span class="underline">CCC</span></strong> </td><td><strong><span class="underline">CCC</span></strong> </td><td>CCA </td><td><strong><span class="underline">CCA</span></strong> </td><td><strong><span class="underline">CCT</span></strong> </td><td><strong><span class="underline">CCA</span></strong> </td><td><strong><span class="underline">CCA</span></strong> </td><td><strong><span class="underline">CCA</span></strong> </td></tr><tr><td>Gln </td><td><strong><span class="underline">CAG</span></strong> </td><td><strong><span class="underline">CAG</span></strong> </td><td><strong><span class="underline">CAG</span></strong> </td><td><strong><span class="underline">CAG</span></strong> </td><td><strong><span class="underline">CAG</span></strong> </td><td><strong><span class="underline">CAA</span></strong> </td><td><strong><span class="underline">CAA</span></strong> </td><td><strong><span class="underline">CAA</span></strong> </td><td><strong><span class="underline">CAA</span></strong> </td><td><strong><span class="underline">CAA</span></strong> </td></tr><tr><td>Leu </td><td><strong><span class="underline">CTG</span></strong> </td><td><strong><span class="underline">CTG</span></strong> </td><td><strong><span class="underline">CTG</span></strong> </td><td><strong><span class="underline">CTG</span></strong> </td><td><strong><span class="underline">CTC</span></strong> </td><td>CTT </td><td>TTA </td><td><strong><span class="underline">TTG</span></strong> </td><td>TTA </td><td>TTA </td></tr><tr><td>Val </td><td><strong><span class="underline">GTG</span></strong> </td><td><strong><span class="underline">GTG</span></strong> </td><td><strong><span class="underline">GTG</span></strong> </td><td><strong><span class="underline">GTG</span></strong> </td><td><strong><span class="underline">GTC</span></strong> </td><td>GTT </td><td><strong><span class="underline">GTT</span></strong> </td><td><strong><span class="underline">GTT</span></strong> </td><td><strong><span class="underline">GTT</span></strong> </td><td>GTA </td></tr><tr><td>Arg </td><td><strong><span class="underline">CGG</span></strong> </td><td><strong><span class="underline">CGG</span></strong> </td><td><strong><span class="underline">CGG</span></strong> </td><td><strong><span class="underline">CGG</span></strong> </td><td><strong><span class="underline">CGG</span></strong> </td><td>AGA </td><td><strong><span class="underline">CGT</span></strong> </td><td><strong><span class="underline">AGA</span></strong> </td><td><strong><span class="underline">AGA</span></strong> </td><td><strong><span class="underline">AGA</span></strong> </td></tr><tr><td>Asp </td><td><strong><span class="underline">GAT</span></strong> </td><td><strong><span class="underline">GAC</span></strong> </td><td><strong><span class="underline">GAC</span></strong> </td><td><strong><span class="underline">GAC</span></strong> </td><td><strong><span class="underline">GAT</span></strong> </td><td>GAT </td><td><strong><span class="underline">GAT</span></strong> </td><td>GAT </td><td><strong><span class="underline">GAT</span></strong> </td><td><strong><span class="underline">GAT</span></strong> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>Color code: on a blue background codons shared with <em>M. musculus</em>, on a red background, codons shared with <em>C. albicans.</em> A: Bolded codons with a star are preferred codons for which a codon with more acceptors exists. B: bolded and underlined codons are identical in lowly and highly translated genes. The following abbreviations of species names are used throughout this article: H. sapiens (Hs), <em>H. sapiens (Hs)</em>, <em>M. musculus (Mm)</em>, <em>R. norvegicus (Rn)</em>, <em>D. melanogaster (Dm)</em>, <em>C. elegans (Ce), H. capsulatum (Hc), S. pombe (Sp)</em>, <em>S. cerevisiae (Sc), C. albicans (Ca</em>), and <em>P. falciparum (Pf)</em>. Refer to the online version for colors.</p></div></div></div></div></p>                            <h3 scrollto-destination=88473311 id="88473311" class="section-title" >3.4. Preferred codons and tRNA copy number</h3>
<p>It is assumed that a codon with the highest copy number of corresponding tRNAs is prone to be the preferred codon. This result is confirmed in all species, but <em>H. capsulatum</em> (<a reveal-id="dsw062-T2" data-open="dsw062-T2" class="link link-reveal link-table xref-fig">Table 2</a>). Nevertheless, species do not equally observe this tendency: in <em>S. pombe</em> 15 out 18 amino acids prefer the codon with the highest copy number of tRNA, while only 9-12 amino acids behave the same in mammals. Non-majority preferred codons are shared among species, like species preferring GGT for glycine (<em>S. pombe</em>, <em>Saccharomyces</em><em>cerevisiae</em> and <em>C.</em><em>albicans</em>) or preferring GCC for Alanine (<em>H. sapiens</em>, <em>M</em><em>us</em><em>musculus</em>, <em>R</em><em>attus</em><em>norvegicus</em>, <em>D</em><em>rosophila</em><em>melanogaster</em>, <em>C. elegans</em> and <em>H. capsulatum</em>). <div content-id="dsw062-T2" class="table-modal"><div class="table-wrap"><div id="dsw062-T2" data-id="dsw062-T2" class="table-wrap-title"><div class="label -label">Table 2</div><div class="caption"><p>Comparison of favourite codons and tRNA copy numbers</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Species </th><th>Number of tRNAs </th><th>Number of favourite codons having the highest tRNA copy number (out of 18 cases) </th><th> (%) </th><th>Number of favorite codons lacking tRNA </th><th>Number of codons lacking tRNA </th></tr></thead><tbody><tr><td><em>H. sapiens</em> </td><td>589 </td><td>9 </td><td>50.0 </td><td>2 </td><td>6 </td></tr><tr><td><em>C. elegans</em> </td><td>563 </td><td>12 </td><td>66.7 </td><td>6 </td><td>15 </td></tr><tr><td><em>M. musculus</em> </td><td>441 </td><td>12 </td><td>66.7 </td><td>2 </td><td>8 </td></tr><tr><td><em>R. norvegicus</em> </td><td>385 </td><td>10 </td><td>55.6 </td><td>5 </td><td>13 </td></tr><tr><td><em>D. melanogaster</em> </td><td>268 </td><td>12 </td><td>66.7 </td><td>6 </td><td>18 </td></tr><tr><td><em>S. cerevisiae</em> </td><td>259 </td><td>15 </td><td>83.3 </td><td>3 </td><td>20 </td></tr><tr><td><em>S. pombe</em> </td><td>158 </td><td>15 </td><td>83.3 </td><td>2 </td><td>16 </td></tr><tr><td><em>C. albicans</em> </td><td>119 </td><td>15 </td><td>83.3 </td><td>3 </td><td>20 </td></tr><tr><td><em>H. capsulatum</em> </td><td>71 </td><td>8 </td><td>44.4 </td><td>5 </td><td>16 </td></tr><tr><td><em>P. falciparum</em> </td><td>33 </td><td>14 </td><td>77.8 </td><td>7 </td><td>26 </td></tr></tbody></table></div><div content-id="dsw062-T2" class="table-modal"><table><thead name="thead" align=""><tr><th>Species </th><th>Number of tRNAs </th><th>Number of favourite codons having the highest tRNA copy number (out of 18 cases) </th><th> (%) </th><th>Number of favorite codons lacking tRNA </th><th>Number of codons lacking tRNA </th></tr></thead><tbody><tr><td><em>H. sapiens</em> </td><td>589 </td><td>9 </td><td>50.0 </td><td>2 </td><td>6 </td></tr><tr><td><em>C. elegans</em> </td><td>563 </td><td>12 </td><td>66.7 </td><td>6 </td><td>15 </td></tr><tr><td><em>M. musculus</em> </td><td>441 </td><td>12 </td><td>66.7 </td><td>2 </td><td>8 </td></tr><tr><td><em>R. norvegicus</em> </td><td>385 </td><td>10 </td><td>55.6 </td><td>5 </td><td>13 </td></tr><tr><td><em>D. melanogaster</em> </td><td>268 </td><td>12 </td><td>66.7 </td><td>6 </td><td>18 </td></tr><tr><td><em>S. cerevisiae</em> </td><td>259 </td><td>15 </td><td>83.3 </td><td>3 </td><td>20 </td></tr><tr><td><em>S. pombe</em> </td><td>158 </td><td>15 </td><td>83.3 </td><td>2 </td><td>16 </td></tr><tr><td><em>C. albicans</em> </td><td>119 </td><td>15 </td><td>83.3 </td><td>3 </td><td>20 </td></tr><tr><td><em>H. capsulatum</em> </td><td>71 </td><td>8 </td><td>44.4 </td><td>5 </td><td>16 </td></tr><tr><td><em>P. falciparum</em> </td><td>33 </td><td>14 </td><td>77.8 </td><td>7 </td><td>26 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn2"><div class="fn"><p>(The codons for Stop, Met and Trp are excluded). tRNA: transfer RNA. Copy numbers of tRNA genes come from GtRNAdb 2.0<a reveal-id="dsw062-B19" data-open="dsw062-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a>. The percentage of favourite codons (one per amino acid) that have the highest tRNA copy number lies above 50% for all species but <em>H. capsulatum</em>.</p></div></div></div></div></div><div class="table-wrap"><div id="dsw062-T2" data-id="dsw062-T2" class="table-wrap-title"><div class="label -label">Table 2</div><div class="caption"><p>Comparison of favourite codons and tRNA copy numbers</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Species </th><th>Number of tRNAs </th><th>Number of favourite codons having the highest tRNA copy number (out of 18 cases) </th><th> (%) </th><th>Number of favorite codons lacking tRNA </th><th>Number of codons lacking tRNA </th></tr></thead><tbody><tr><td><em>H. sapiens</em> </td><td>589 </td><td>9 </td><td>50.0 </td><td>2 </td><td>6 </td></tr><tr><td><em>C. elegans</em> </td><td>563 </td><td>12 </td><td>66.7 </td><td>6 </td><td>15 </td></tr><tr><td><em>M. musculus</em> </td><td>441 </td><td>12 </td><td>66.7 </td><td>2 </td><td>8 </td></tr><tr><td><em>R. norvegicus</em> </td><td>385 </td><td>10 </td><td>55.6 </td><td>5 </td><td>13 </td></tr><tr><td><em>D. melanogaster</em> </td><td>268 </td><td>12 </td><td>66.7 </td><td>6 </td><td>18 </td></tr><tr><td><em>S. cerevisiae</em> </td><td>259 </td><td>15 </td><td>83.3 </td><td>3 </td><td>20 </td></tr><tr><td><em>S. pombe</em> </td><td>158 </td><td>15 </td><td>83.3 </td><td>2 </td><td>16 </td></tr><tr><td><em>C. albicans</em> </td><td>119 </td><td>15 </td><td>83.3 </td><td>3 </td><td>20 </td></tr><tr><td><em>H. capsulatum</em> </td><td>71 </td><td>8 </td><td>44.4 </td><td>5 </td><td>16 </td></tr><tr><td><em>P. falciparum</em> </td><td>33 </td><td>14 </td><td>77.8 </td><td>7 </td><td>26 </td></tr></tbody></table></div><div content-id="dsw062-T2" class="table-modal"><table><thead name="thead" align=""><tr><th>Species </th><th>Number of tRNAs </th><th>Number of favourite codons having the highest tRNA copy number (out of 18 cases) </th><th> (%) </th><th>Number of favorite codons lacking tRNA </th><th>Number of codons lacking tRNA </th></tr></thead><tbody><tr><td><em>H. sapiens</em> </td><td>589 </td><td>9 </td><td>50.0 </td><td>2 </td><td>6 </td></tr><tr><td><em>C. elegans</em> </td><td>563 </td><td>12 </td><td>66.7 </td><td>6 </td><td>15 </td></tr><tr><td><em>M. musculus</em> </td><td>441 </td><td>12 </td><td>66.7 </td><td>2 </td><td>8 </td></tr><tr><td><em>R. norvegicus</em> </td><td>385 </td><td>10 </td><td>55.6 </td><td>5 </td><td>13 </td></tr><tr><td><em>D. melanogaster</em> </td><td>268 </td><td>12 </td><td>66.7 </td><td>6 </td><td>18 </td></tr><tr><td><em>S. cerevisiae</em> </td><td>259 </td><td>15 </td><td>83.3 </td><td>3 </td><td>20 </td></tr><tr><td><em>S. pombe</em> </td><td>158 </td><td>15 </td><td>83.3 </td><td>2 </td><td>16 </td></tr><tr><td><em>C. albicans</em> </td><td>119 </td><td>15 </td><td>83.3 </td><td>3 </td><td>20 </td></tr><tr><td><em>H. capsulatum</em> </td><td>71 </td><td>8 </td><td>44.4 </td><td>5 </td><td>16 </td></tr><tr><td><em>P. falciparum</em> </td><td>33 </td><td>14 </td><td>77.8 </td><td>7 </td><td>26 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn2"><div class="fn"><p>(The codons for Stop, Met and Trp are excluded). tRNA: transfer RNA. Copy numbers of tRNA genes come from GtRNAdb 2.0<a reveal-id="dsw062-B19" data-open="dsw062-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a>. The percentage of favourite codons (one per amino acid) that have the highest tRNA copy number lies above 50% for all species but <em>H. capsulatum</em>.</p></div></div></div></div></p><p>We inferred two species trees based on the species' proximity either in terms of tRNA copy number, or in terms of average RSCU<sub>RS</sub> over highly translated genes (<a reveal-id="dsw062-F3" data-open="dsw062-F3" class="link link-reveal link-table xref-fig">Fig. 3B and C</a>). These trees are similar, but not equal. tRNA copy number groups the mammals with Drosophila and <em>S. cerevisiae</em>, and makes a clade with <em>C. albicans</em>, <em>S. pombe</em> and <em>C. elegans</em>, while both <em>H. capsulatum and P. falciparum</em> form an outgroup (which reflects their small tRNA gene copy numbers; see <a reveal-id="dsw062-T2" data-open="dsw062-T2" class="link link-reveal link-table xref-fig">Table 2</a>). With CUB (<a reveal-id="dsw062-F3" data-open="dsw062-F3" class="link link-reveal link-table xref-fig">Fig. 3C</a>), the clade grouping the mammals and Drosophila remains opposed to the branches containing the yeast species and <em>P. falciparum</em>. The striking feature is the position of <em>H. capsulatum</em>, which, due to its low tRNA gene number, is located at the base the tRNA-tree, but it is grouped with the mammals in the CUB-tree. Despite its basal position in the tree, <em>H. capsulatum</em> is closer to the group of yeasts (Euclidian distance = 29.9) than to the group of mammals (Euclidian distance = 86.5).</p>                            <h3 scrollto-destination=88473314 id="88473314" class="section-title" >3.5. Preferred codons, codons lacking tRNA and GC content</h3>
<p>Wobbling corresponds to a codon that is decoded without following Watson–Crick base pair rules.<a reveal-id="dsw062-B21" data-open="dsw062-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> Wobbling is complex to study, but codons lacking tRNA are necessarily subject to it. In <a reveal-id="dsw062-F3" data-open="dsw062-F3" class="link link-reveal link-table xref-fig">Fig. 3D</a>, we compared the numbers of codons lacking tRNA, and of preferred codons lacking tRNA for all species ordered by the size of their tRNA pool. As expected, when the tRNA pool increases, the number of codons lacking tRNA decreases steeply (slope equals −0.026642). However, the number of preferred codons lacking tRNA remains stable and appears not to depend on the size of the tRNA pool (slope dashed curve equals −0.002812).</p><p>The last base of a codon is supposed to correlate with the GC content of a genome. This result is partially confirmed in <a reveal-id="dsw062-F4" data-open="dsw062-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>. On one extremity, <em>P. falciparum</em> has a very low GC content and no preferred codon ending with a G or C. At the other extremity, all species with a high GC content strongly prefer codons ending with a G or C. For intermediate values of GC content, results are more puzzling: <em>C. elegans</em> has an intermediate GC content compared to <em>C. albicans</em>, <em>S. pombe</em> and <em>S. cerevisiae</em>, but has a much stronger preference for codons ending with a G or C.</p>                        <a id="88473317" scrollto-destination="88473317"></a>
<div id="dsw062-F4" data-id="dsw062-F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw062/1/m_dsw062f4.png?Expires=2147483647&Signature=xOsPlYXsdvWg-1Ene105CPkMH1yQbDCxiYO-Cw3cbrA218aAe1cBX9Si6pFPWmjIBtZWtwfz1cJ80ecBAlJXlpmN2MynEHsNMLvNwqPUMFH325y2KvpQB7QYB1F1xZPEcat8-7YAlg0AqjJwcbppwxiYIDv3PqyZghj7FTwozFk-md0-UVrzp7s~Kdfx77CV~hPxH3xTeItkMOeQkEq8MWK2HpARSqZzCpSjz~M~ZHMaSOgMLIgJYlRMT8inY4d8YPFWClu-k65YCci7aPh1pbdgbitnMlSQgFItiY9cg9wiiqIMo-lPfC9CuPeyOHKekY4gno4JcVbtKtfHHRh4kA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genome percentage of GC bases (GC %) versus GC % of last base of preferred codons. Same species abbreviations as in Fig. 1." path-from-xml="dsw062f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/88473317/dsw062f4.tif" path-from-xml="dsw062f4.tif" target="_blank">View large</a><a section="88473317" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw062/1/dsw062f4.png?Expires=2147483647&Signature=sribdIgOGbbKf2GqYjS5gfw07AcIuzjXh1nH6YtHiUK9uCSG4UXfAOOQ9uccBw-vdzU5nwKUWUWniKQ80cHhtdwFs~TIYQ~YvAUYW26VmLBszgKzuth6lM2tiv2SCCqtfHlYWK65roiLUpUBHujQbAN9VsS5MHSoyXIZh00f88yfkzAk8NqpM9s6jk-OPquRI9LDsr7lFLNgfWHaCk0XIsuGelyUwyz0rv6iGZWGjzD3L4Bl44T5GgpBZQulibl5W12C7Q72FnScMO~Lt6BTF4eptNJP1bPavtwv9rROAOVN-wAnDDm722UBaEytIwivml13NDaG6a8d6XjSFUTuVw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=88473317&ar=2972686&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw062f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genome percentage of GC bases (GC %) versus GC % of last base of preferred codons. Same species abbreviations as in <a reveal-id="dsw062-F1" data-open="dsw062-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>.</p></div></div><div content-id="dsw062-F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw062/1/m_dsw062f4.png?Expires=2147483647&Signature=xOsPlYXsdvWg-1Ene105CPkMH1yQbDCxiYO-Cw3cbrA218aAe1cBX9Si6pFPWmjIBtZWtwfz1cJ80ecBAlJXlpmN2MynEHsNMLvNwqPUMFH325y2KvpQB7QYB1F1xZPEcat8-7YAlg0AqjJwcbppwxiYIDv3PqyZghj7FTwozFk-md0-UVrzp7s~Kdfx77CV~hPxH3xTeItkMOeQkEq8MWK2HpARSqZzCpSjz~M~ZHMaSOgMLIgJYlRMT8inY4d8YPFWClu-k65YCci7aPh1pbdgbitnMlSQgFItiY9cg9wiiqIMo-lPfC9CuPeyOHKekY4gno4JcVbtKtfHHRh4kA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genome percentage of GC bases (GC %) versus GC % of last base of preferred codons. Same species abbreviations as in Fig. 1." path-from-xml="dsw062f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/88473317/dsw062f4.tif" path-from-xml="dsw062f4.tif" target="_blank">View large</a><a section="88473317" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw062/1/dsw062f4.png?Expires=2147483647&Signature=sribdIgOGbbKf2GqYjS5gfw07AcIuzjXh1nH6YtHiUK9uCSG4UXfAOOQ9uccBw-vdzU5nwKUWUWniKQ80cHhtdwFs~TIYQ~YvAUYW26VmLBszgKzuth6lM2tiv2SCCqtfHlYWK65roiLUpUBHujQbAN9VsS5MHSoyXIZh00f88yfkzAk8NqpM9s6jk-OPquRI9LDsr7lFLNgfWHaCk0XIsuGelyUwyz0rv6iGZWGjzD3L4Bl44T5GgpBZQulibl5W12C7Q72FnScMO~Lt6BTF4eptNJP1bPavtwv9rROAOVN-wAnDDm722UBaEytIwivml13NDaG6a8d6XjSFUTuVw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=88473317&ar=2972686&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw062f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genome percentage of GC bases (GC %) versus GC % of last base of preferred codons. Same species abbreviations as in <a reveal-id="dsw062-F1" data-open="dsw062-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>.</p></div></div>                            <h2 scrollto-destination=88473318 id="88473318" class="section-title" >4. Conclusion and discussion</h2>
<p>CUB measures the difference in the frequency of translation of each possible codon for an amino acid. CUB is detected in highly expressed genes because the selection pressure to choose optimal codons is stronger in those genes. First, it is known that numerous mRNAs are finally not translated<a reveal-id="dsw062-B22" data-open="dsw062-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> and have thus no impact on fitness. Hence, to measure CUB, Ribo-seq seems more appropriate to select highly expressed genes than RNA-seq data. Second, various forces and processes (such as mutation bias) impact the nucleotidic composition of a gene.<a reveal-id="dsw062-B17" data-open="dsw062-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> Because it captures the presence of ribosomes, Ribo-seq allows to measure the sole translational advantage of a codon rather than the impact of evolutionary forces unrelated to translation.</p><p>Our method based on Ribo-seq—rather than RNA-seq—data allows us to compute a precise and direct measure of CUB. Although Ribo-seq was not originally designed for CUB studies, we show that our method assesses the CUB with high precision. Using translation data provides other benefits: first, all translated genes can be included even if functional information is lacking. Second, our method avoids any statistical optimization as CUB is directly computed from translation data. Third, Ribo-seq data allows to compute CUB and to detect preferred codons in a wide variety of species. Fourth, our method may help to measure variation of CUB across conditions or tissues. Nevertheless, the number of Ribo-seq footprints per gene depends on the sequencing depth, which may introduce some bias if depth is low. To correctly classify genes, this method also requires a good sequencing depth in order to get at least some footprints on lowly translated genes: for example, we can grossely estimate that it requires at least 3,000 covered genes to study human's CUB.</p><p>Our method was validated by comparison to previous results and our results confirm the current knowledge on CUB. First, the CUB is stronger in highly translated genes.<a reveal-id="dsw062-B7" data-open="dsw062-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> Second, most higher eukaryotes exhibit a weaker CUB when compared to unicellular species, like yeasts.<a reveal-id="dsw062-B17" data-open="dsw062-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> Third, tRNA copy number has a strong influence on preferred codons, as they indicate preferred codons for most of amino acids.<a reveal-id="dsw062-B18" data-open="dsw062-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> Nevertheless, the tRNA copy number is not the only criterion: the GC content also influences preferred codons, but to a lesser extent than the tRNA pool. The clustering of pluricellular species in <a reveal-id="dsw062-F3" data-open="dsw062-F3" class="link link-reveal link-table xref-fig">Fig. 3B and C</a> suggests that translation can adapt to a changing GC content. Surprisingly, an increasing tRNA pool size does not diminish the number of preferred codons lacking tRNA.</p><p>Ribo-seq data on a larger set of species would help to get a better understanding of CUB evolution, but to date, only very few species have been subjected to Ribo-seq experiments. Nevertheless, comparison of codon bias between species seems to reveal a common story that is partially adapted by every organism. In yeast, the level of ribosome occupancy measured using an adequate protocol was found to be negatively correlated with tRNA copy numbers, which is used as a proxy of cognate tRNA abundances.<a reveal-id="dsw062-B13" data-open="dsw062-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> Our results suggest that the tRNA pool is the key factor for every amino acid in a wide range of species, and that some preferred codons are conserved across the tree of life. However, we clearly distinguished two species groups when comparing the intensities of CUB. The low-intensity group contains mammals and drosophila (plus <em>P. falciparum</em> that has a very restricted tRNA pool, which may be due to the difficulty of tRNA gene prediction, and <em>H. capsulatum</em> which we discussed later) versus a three-time higher CUB intensity group formed by <em>C. elegans</em>, <em>S. pombe</em>, <em>C. albicans</em> and <em>S. cerevisiae</em>. Surprisingly, <em>C. elegans</em> does not belong to the low intensity group, while mammals and drosophila do. It has common features with both groups: some amino acids with a yeast-like preferences (like valine or proline), some with a mammals-like preferences (like glutamic acid or aspartic acid) and some with a unique profile (glycine and leucine). A possible scenario could be a decreasing intensity of CUB in multicellular species: nonetheless, this evolution would be achieved progressively, amino acid by amino acid (often through a substitution at the third base of a codon<a reveal-id="dsw062-B23" data-open="dsw062-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a>—see examples below). Highly translated genes are more subjected to CUB than lowly translated genes and, as shown in <a reveal-id="dsw062-T1" data-open="dsw062-T1" class="link link-reveal link-table xref-fig">Table 1</a>, a general pattern appears from yeasts to mammals: an evolution CAT to CAC for Histidine, from GCT to GCC in Alanine, for example. This general pattern admits some exceptions: for instance, <em>H. capsulatum</em> and <em>C. elegans</em> favour GTC for valine whereas the common pattern favours an evolution from GTT to GTG. Besides the general tendencies, CUB evolution is to some degree dependent of the species, as suggested by some species specific preferred codons. Including more species would help to further unravel the evolution of CUB, or to investigate the generality of its impacts that were detected only in some species<a reveal-id="dsw062-B24" data-open="dsw062-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a><sup>,</sup><a reveal-id="dsw062-B25" data-open="dsw062-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a>; thus we hope that Ribosome profiling will be applied to an increasing number of species.</p><p>The most striking results were obtained for the pathogenic yeast, <em>H. capsulatum</em>. Its tRNA pool is closer to that of other yeasts, but its CUB much closer to that of mammals. This suggests that its CUB is quite independent of its tRNA pool. As <em>H. capsulatum</em> is a pathogen of mammals, it is tempting to say that it adapts to its host, but we lack information to conclude. Is <em>H. capsulatum</em> able to use the translational machinery of its host (ribosomes and/or tRNAs) or do more tRNA genes remain to be annotated in its genome? Including other pathogenic species in the comparison, such as the fungi Aspergillus, would help to determine whether this feature is specific to <em>H. capsulatum</em> or widespread in pathogens. Nevertheless, pathogens could represent an exception in which the CUB depends more on the host's tRNA pool than on their own tRNA pool. </p>                            <h2 scrollto-destination=88473324 id="88473324" class="section-title" >Availability</h2>
<p>The program for computing the codon usage bias from Ribo-seq is freely available at: <a class="link link-uri" href="http://www.lirmm.fr/&amp;sim;rivals/rscu" target="_blank">http://www.lirmm.fr/∼rivals/rscu</a></p>                            <h2 scrollto-destination=88473326 id="88473326" class="backacknowledgements-title" >Acknowledgements</h2>
<p>The authors want to thank France Génomique and the ATGC platform for the logistic support they provided. We thank Nicolas Galtier for his insightful comments on the manuscript.</p>                            <h3 scrollto-destination=88473328 id="88473328" class="section-title" >Conflict of interest</h3>
<p>None declared.</p>                            <h3 scrollto-destination=88473330 id="88473330" class="section-title" >Supplementary data</h3>
<p><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw062/1/dsw062_Supp.pdf?Expires=2147483647&Signature=hwp7bk5cUxNRikfSiIkl3TTPotX6N9DEEJ1zXu86eUPtNFMB~YA6Knkmbl2G3U93cBYnO4ZUn~ivFp6FWmS~uRTi4KVhxvk~1D4S5tIXKJEv~WcODSd~FPYYIrDjy6Byb~heC3TkYI6~2ibqOjma8-Kz94DphePxx5KjpUQG~J14fvlC1wcan17sCxRLVNIZ3MtSuahmFcDkVuLVlPJ1Fer-kfIIsk-3A~3b-MnRXtNkhZ0GvgxhcVEBJyt7i0htXRsjBVZwNLzuYK7v3lp-sEVD2QUJMR2h7PeEEDDnHFM0zoK256NjDv1LUPzTi7ouTIYp5VN6IRg4Bg0AikgJ8g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary data</a></span> are available at <a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org" target="_blank">www.dnaresearch.oxfordjournals.org</a>.</p>                            <h3 scrollto-destination=88473332 id="88473332" class="section-title" >Funding</h3>
<p>This work is supported by the Fondation pour la Recherche Médicale, grant DBI20131228574, ANR grant (ANR-11-BINF-0002).</p>                            <h2 scrollto-destination=88473334 id="88473334" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsw062-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw062-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Grantham</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Gautier</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Gouy</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Mercier</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Pavé</div> <div class="given-names">A.</div></div></div> <div class="year">1980</div>, 
<div class="article-title">Codon catalog usage and the genome hypothesis</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">8</div>, <div class="fpage">r49</div>–<div class="lpage">62</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20catalog%20usage%20and%20the%20genome%20hypothesis&amp;author=R.%20Grantham&amp;author=C.%20Gautier&amp;author=M.%20Gouy&amp;author=R.%20Mercier&amp;author=A.%20Pav%C3%A9&amp;publication_year=1980&amp;journal=Nucleic%20Acids%20Res&amp;volume=8&amp;pages=r49-62" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6986610" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw062-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw062-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gouy</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Gautier</div> <div class="given-names">C.</div></div></div> <div class="year">1982</div>, 
<div class="article-title">Codon usage in bacteria: correlation with gene expressivity</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">10</div>, <div class="fpage">7055</div>–<div class="lpage">74</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20usage%20in%20bacteria%3A%20correlation%20with%20gene%20expressivity&amp;author=M.%20Gouy&amp;author=C.%20Gautier&amp;publication_year=1982&amp;journal=Nucleic%20Acids%20Res&amp;volume=10&amp;pages=7055-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/10.22.7055" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F10.22.7055" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F10.22.7055"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6760125" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw062-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw062-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Plotkin</div> <div class="given-names">J. B.</div></div>, <div class="name string-name"><div class="surname">Kudla</div> <div class="given-names">G.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Synonymous but not the same: the causes and consequences of codon bias</div>. <div class="source">Nat. Rev. Genet</div> ., <div class="volume">12</div>, <div class="fpage">32</div>–<div class="lpage">42</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Synonymous%20but%20not%20the%20same%3A%20the%20causes%20and%20consequences%20of%20codon%20bias&amp;author=J.%20B.%20Plotkin&amp;author=G.%20Kudla&amp;publication_year=2011&amp;journal=Nat.%20Rev.%20Genet&amp;volume=12&amp;pages=32-42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg2899" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg2899" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg2899"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21102527" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw062-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw062-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Quax</div> <div class="given-names">T. E. F.</div></div>, <div class="name string-name"><div class="surname">Claassens</div> <div class="given-names">N. J.</div></div>, <div class="name string-name"><div class="surname">Söll</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">van der Oost</div> <div class="given-names">J.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Codon bias as a means to fine-tune gene expression</div>, <div class="source">Mol. Cell</div> , <div class="volume">59</div>, <div class="fpage">149</div>–<div class="lpage">61</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20bias%20as%20a%20means%20to%20fine-tune%20gene%20expression&amp;author=T.%20E.%20F.%20Quax&amp;author=N.%20J.%20Claassens&amp;author=D.%20S%C3%B6ll&amp;author=J.%20van%20der%20Oost&amp;publication_year=2015&amp;journal=Mol.%20Cell&amp;volume=59&amp;pages=149-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.molcel.2015.05.035" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.molcel.2015.05.035" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.molcel.2015.05.035"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26186290" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw062-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw062-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Pechmann</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Frydman</div> <div class="given-names">J.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Evolutionary conservation of codon optimality reveals hidden signatures of cotranslational folding,</div> <div class="source">Nat. Struct. Mol. Biol</div> ., <div class="volume">20</div>, <div class="fpage">237</div>–<div class="lpage">43</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolutionary%20conservation%20of%20codon%20optimality%20reveals%20hidden%20signatures%20of%20cotranslational%20folding%2C&amp;author=S.%20Pechmann&amp;author=J.%20Frydman&amp;publication_year=2013&amp;journal=Nat.%20Struct.%20Mol.%20Biol&amp;volume=20&amp;pages=237-43" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nsmb.2466" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnsmb.2466" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnsmb.2466"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23262490" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw062-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw062-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yu</div> <div class="given-names">C.-H.</div></div>, <div class="name string-name"><div class="surname">Dang</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Zhou</div> <div class="given-names">Z.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Codon usage influences the local rate of translation elongation to regulate co-translational protein folding,</div> <div class="source">Mol. Cell</div> , <div class="volume">59</div>, <div class="fpage">744</div>–<div class="lpage">54</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20usage%20influences%20the%20local%20rate%20of%20translation%20elongation%20to%20regulate%20co-translational%20protein%20folding%2C&amp;author=C.-H.%20Yu&amp;author=Y.%20Dang&amp;author=Z.%20Zhou&amp;publication_year=2015&amp;journal=Mol.%20Cell&amp;volume=59&amp;pages=744-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.molcel.2015.07.018" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.molcel.2015.07.018" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.molcel.2015.07.018"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26321254" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw062-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw062-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sharp</div> <div class="given-names">P. M.</div></div>, <div class="name string-name"><div class="surname">Tuohy</div> <div class="given-names">T. M.</div></div>, <div class="name string-name"><div class="surname">Mosurski</div> <div class="given-names">K. R.</div></div></div> <div class="year">1986</div>, 
<div class="article-title">Codon usage in yeast: cluster analysis clearly differentiates highly and lowly expressed genes,</div> <div class="source">Nucleic Acids Res</div> ., <div class="volume">14</div>, <div class="fpage">5125</div>–<div class="lpage">43</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20usage%20in%20yeast%3A%20cluster%20analysis%20clearly%20differentiates%20highly%20and%20lowly%20expressed%20genes%2C&amp;author=P.%20M.%20Sharp&amp;author=T.%20M.%20Tuohy&amp;author=K.%20R.%20Mosurski&amp;publication_year=1986&amp;journal=Nucleic%20Acids%20Res&amp;volume=14&amp;pages=5125-43" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/14.13.5125" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F14.13.5125" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F14.13.5125"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3526280" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw062-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw062-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ingolia</div> <div class="given-names">N. T.</div></div>, <div class="name string-name"><div class="surname">Ghaemmaghami</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Newman</div> <div class="given-names">J. R. S.</div></div>, <div class="name string-name"><div class="surname">Weissman</div> <div class="given-names">J. S.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Genome-wide analysis in vivo of translation with nucleotide resolution using ribosome profiling,</div> <div class="source">Science</div> , <div class="volume">324</div>, <div class="fpage">218</div>–<div class="lpage">23</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20analysis%20in%20vivo%20of%20translation%20with%20nucleotide%20resolution%20using%20ribosome%20profiling%2C&amp;author=N.%20T.%20Ingolia&amp;author=S.%20Ghaemmaghami&amp;author=J.%20R.%20S.%20Newman&amp;author=J.%20S.%20Weissman&amp;publication_year=2009&amp;journal=Science&amp;volume=324&amp;pages=218-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1168978" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1168978" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1168978"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19213877" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw062-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw062-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Guo</div> <div class="given-names">B.-Y.</div></div>, <div class="name string-name"><div class="surname">Zeng</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Wu</div> <div class="given-names">H.-C.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Recent advances of DNA sequencing via nanopore-based technologies,</div> <div class="source">Sci. Bull</div> ., <div class="volume">60</div>, <div class="fpage">287</div>–<div class="lpage">95</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Recent%20advances%20of%20DNA%20sequencing%20via%20nanopore-based%20technologies%2C&amp;author=B.-Y.%20Guo&amp;author=T.%20Zeng&amp;author=H.-C.%20Wu&amp;publication_year=2014&amp;journal=Sci.%20Bull&amp;volume=60&amp;pages=287-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11434-014-0707-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11434-014-0707-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11434-014-0707-6"> </span></div> </div></div></div></div></div><div content-id="dsw062-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw062-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Dunn</div> <div class="given-names">J. G.</div></div>, <div class="name string-name"><div class="surname">Foo</div> <div class="given-names">C. K.</div></div>, <div class="name string-name"><div class="surname">Belletier</div> <div class="given-names">N. G.</div></div>, <div class="name string-name"><div class="surname">Gavis</div> <div class="given-names">E. R.</div></div>, <div class="name string-name"><div class="surname">Weissman</div> <div class="given-names">J. S.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Ribosome profiling reveals pervasive and regulated stop codon readthrough in Drosophila melanogaster,</div> <div class="source">eLife</div> , <div class="volume">2</div>, <div class="fpage">e01179</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ribosome%20profiling%20reveals%20pervasive%20and%20regulated%20stop%20codon%20readthrough%20in%20Drosophila%20melanogaster%2C&amp;author=J.%20G.%20Dunn&amp;author=C.%20K.%20Foo&amp;author=N.%20G.%20Belletier&amp;author=E.%20R.%20Gavis&amp;author=J.%20S.%20Weissman&amp;publication_year=2013&amp;journal=eLife&amp;volume=2&amp;pages=e01179" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.7554/eLife.01179" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.7554%2FeLife.01179" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.7554%2FeLife.01179"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24302569" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw062-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw062-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zupanic</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Meplan</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Grellscheid</div> <div class="given-names">S. N.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Detecting translational regulation by change point analysis of ribosome profiling data sets,</div> <div class="source">RNA N. Y. N</div> , <div class="volume">20</div>, <div class="fpage">1507</div>–<div class="lpage">18</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Detecting%20translational%20regulation%20by%20change%20point%20analysis%20of%20ribosome%20profiling%20data%20sets%2C&amp;author=A.%20Zupanic&amp;author=C.%20Meplan&amp;author=S.%20N.%20Grellscheid&amp;publication_year=2014&amp;journal=RNA%20N.%20Y.%20N&amp;volume=20&amp;pages=1507-18" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1261/rna.045286.114" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1261%2Frna.045286.114" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1261%2Frna.045286.114"> </span></div> </div></div></div></div></div><div content-id="dsw062-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw062-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gao</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Wan</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Ma</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Shen</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Qian</div> <div class="given-names">S.-B.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Quantitative profiling of initiating ribosomes in vivo,</div> <div class="source">Nat. Methods</div> , <div class="volume">12</div>, <div class="fpage">147</div>–<div class="lpage">53</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Quantitative%20profiling%20of%20initiating%20ribosomes%20in%20vivo%2C&amp;author=X.%20Gao&amp;author=J.%20Wan&amp;author=B.%20Liu&amp;author=M.%20Ma&amp;author=B.%20Shen&amp;author=S.-B.%20Qian&amp;publication_year=2015&amp;journal=Nat.%20Methods&amp;volume=12&amp;pages=147-53" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.3208" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.3208" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.3208"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25486063" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw062-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw062-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Weinberg</div> <div class="given-names">D. E.</div></div>, <div class="name string-name"><div class="surname">Shah</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Eichhorn</div> <div class="given-names">S. W.</div></div>, <div class="name string-name"><div class="surname">Hussmann</div> <div class="given-names">J. A.</div></div>, <div class="name string-name"><div class="surname">Plotkin</div> <div class="given-names">J. B.</div></div>, <div class="name string-name"><div class="surname">Bartel</div> <div class="given-names">D. P.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Improved ribosome-footprint and mrna measurements provide insights into dynamics and regulation of yeast translation,</div> <div class="source">Cell Rep</div> ., <div class="volume">14</div>, <div class="fpage">1787</div>–<div class="lpage">99</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Improved%20ribosome-footprint%20and%20mrna%20measurements%20provide%20insights%20into%20dynamics%20and%20regulation%20of%20yeast%20translation%2C&amp;author=D.%20E.%20Weinberg&amp;author=P.%20Shah&amp;author=S.%20W.%20Eichhorn&amp;author=J.%20A.%20Hussmann&amp;author=J.%20B.%20Plotkin&amp;author=D.%20P.%20Bartel&amp;publication_year=2016&amp;journal=Cell%20Rep&amp;volume=14&amp;pages=1787-99" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.celrep.2016.01.043" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.celrep.2016.01.043" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.celrep.2016.01.043"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26876183" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw062-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw062-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sharp</div> <div class="given-names">P. M.</div></div>, <div class="name string-name"><div class="surname">Bradnam</div> <div class="given-names">K. R.</div></div></div> <div class="year">1997</div>, Appendix 3 Codon Usage in <em>C. elegans</em>. In: <div class="person-group"><div class="name string-name"><div class="surname">Riddle</div> <div class="given-names">D.L.</div></div>, <div class="name string-name"><div class="surname">Meyer</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Blumenthal</div> <div class="given-names">B.J. Al</div></div></div>, editors. <div class="source">C Elegans II</div> . <div class="edition">2nd Ed.</div> 
<div class="publisher-name">Cold Spring Harbor Laboratory Press</div>: 
<div class="publisher-loc">Cold Spring Harbor (NY)</div>.</div></div></div></div><div content-id="dsw062-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw062-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Philippe</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Salson</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Commes</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Rivals</div> <div class="given-names">E.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">CRAC: an integrated approach to the analysis of RNA-seq reads</div>. <div class="source">Genome Biol</div> ., <div class="volume">14</div>, <div class="fpage">R30</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CRAC%3A%20an%20integrated%20approach%20to%20the%20analysis%20of%20RNA-seq%20reads&amp;author=N.%20Philippe&amp;author=M.%20Salson&amp;author=T.%20Commes&amp;author=E.%20Rivals&amp;publication_year=2013&amp;journal=Genome%20Biol&amp;volume=14&amp;pages=R30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2013-14-3-r30" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2013-14-3-r30" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2013-14-3-r30"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23537109" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw062-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw062-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Stadler</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Fire</div> <div class="given-names">A.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Wobble base-pairing slows in vivo translation elongation in metazoans,</div> <div class="source">RNA</div> , <div class="volume">17</div>, <div class="fpage">2063</div>–<div class="lpage">73</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Wobble%20base-pairing%20slows%20in%20vivo%20translation%20elongation%20in%20metazoans%2C&amp;author=M.%20Stadler&amp;author=A.%20Fire&amp;publication_year=2011&amp;journal=RNA&amp;volume=17&amp;pages=2063-73" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1261/rna.02890211" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1261%2Frna.02890211" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1261%2Frna.02890211"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22045228" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw062-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw062-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lynch</div> <div class="given-names">M.</div></div></div> <div class="year">2007</div>, <div class="source">The Origins of Genome Architecture</div> . 
<div class="publisher-name">Sinauer Associates, Indiana University at Bloomington</div>.</div></div></div></div><div content-id="dsw062-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw062-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Duret</div> <div class="given-names">L.</div></div></div> <div class="year">2000</div>, 
<div class="article-title">tRNA gene number and codon usage in the <em>C. elegans</em> genome are co-adapted for optimal translation of highly expressed genes,</div> <div class="source">Trends Genet</div> ., <div class="volume">16</div>, <div class="fpage">287</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=tRNA%20gene%20number%20and%20codon%20usage%20in%20the%20C.%20elegans%20genome%20are%20co-adapted%20for%20optimal%20translation%20of%20highly%20expressed%20genes%2C&amp;author=L.%20Duret&amp;publication_year=2000&amp;journal=Trends%20Genet&amp;volume=16&amp;pages=287-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0168-9525(00)02041-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0168-9525(00)02041-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0168-9525(00)02041-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10858656" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw062-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw062-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chan</div> <div class="given-names">P. P.</div></div>, <div class="name string-name"><div class="surname">Lowe</div> <div class="given-names">T. M.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">GtRNAdb 2.0: an expanded database of transfer RNA genes identified in complete and draft genomes</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">44</div>, <div class="fpage">D184</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GtRNAdb%202.0%3A%20an%20expanded%20database%20of%20transfer%20RNA%20genes%20identified%20in%20complete%20and%20draft%20genomes&amp;author=P.%20P.%20Chan&amp;author=T.%20M.%20Lowe&amp;publication_year=2016&amp;journal=Nucleic%20Acids%20Res&amp;volume=44&amp;pages=D184-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkv1309" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkv1309" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkv1309"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26673694" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw062-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw062-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lloyd</div> <div class="given-names">A. T.</div></div>, <div class="name string-name"><div class="surname">Sharp</div> <div class="given-names">P. M.</div></div></div> <div class="year">1992</div>, 
<div class="article-title">Evolution of codon usage patterns: the extent and nature of divergence between <em>Candida albicans</em> and <em>Saccharomyces cerevisiae</em>,</div> <div class="source">Nucleic Acids Res</div> ., <div class="volume">20</div>, <div class="fpage">5289</div>–<div class="lpage">95</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20of%20codon%20usage%20patterns%3A%20the%20extent%20and%20nature%20of%20divergence%20between%20Candida%20albicans%20and%20Saccharomyces%20cerevisiae%2C&amp;author=A.%20T.%20Lloyd&amp;author=P.%20M.%20Sharp&amp;publication_year=1992&amp;journal=Nucleic%20Acids%20Res&amp;volume=20&amp;pages=5289-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/20.20.5289" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F20.20.5289" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F20.20.5289"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1437548" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw062-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw062-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Crick</div> <div class="given-names">F. H.</div></div></div> <div class="year">1966</div>, 
<div class="article-title">Codon–anticodon pairing: the wobble hypothesis,</div> <div class="source">J. Mol. Biol</div> ., <div class="volume">19</div>, <div class="fpage">548</div>–<div class="lpage">55</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%E2%80%93anticodon%20pairing%3A%20the%20wobble%20hypothesis%2C&amp;author=F.%20H.%20Crick&amp;publication_year=1966&amp;journal=J.%20Mol.%20Biol&amp;volume=19&amp;pages=548-55" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0022-2836(66)80022-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0022-2836(66)80022-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0022-2836(66)80022-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/5969078" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw062-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw062-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Schwanhäusser</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Busse</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">N.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">Global quantification of mammalian gene expression control,</div> <div class="source">Nature</div> , <div class="volume">473</div>, <div class="fpage">337</div>–<div class="lpage">42</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Global%20quantification%20of%20mammalian%20gene%20expression%20control%2C&amp;author=B.%20Schwanh%C3%A4usser&amp;author=D.%20Busse&amp;author=N.%20Li&amp;publication_year=2011&amp;journal=Nature&amp;volume=473&amp;pages=337-42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature10098" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature10098" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature10098"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21593866" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw062-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw062-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ikemura</div> <div class="given-names">T.</div></div></div> <div class="year">1985</div>, 
<div class="article-title">Codon usage and tRNA content in unicellular and multicellular organisms,</div> <div class="source">Mol. Biol. Evol</div> ., <div class="volume">2</div>, <div class="fpage">13</div>–<div class="lpage">34</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20usage%20and%20tRNA%20content%20in%20unicellular%20and%20multicellular%20organisms%2C&amp;author=T.%20Ikemura&amp;publication_year=1985&amp;journal=Mol.%20Biol.%20Evol&amp;volume=2&amp;pages=13-34" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3916708" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw062-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw062-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zhou</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Dang</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Zhou</div> <div class="given-names">M.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">Codon usage is an important determinant of gene expression levels largely through its effects on transcription</div>, <div class="source">Proc. Natl. Acad. Sci.</div> , <div class="volume">113</div>(<div class="issue">41</div>), <div class="fpage">E6117</div>–<div class="lpage">E6125</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20usage%20is%20an%20important%20determinant%20of%20gene%20expression%20levels%20largely%20through%20its%20effects%20on%20transcription&amp;author=Z.%20Zhou&amp;author=Y.%20Dang&amp;author=M.%20Zhou&amp;publication_year=2016&amp;journal=Proc.%20Natl.%20Acad.%20Sci.&amp;volume=113&amp;pages=E6117-E6125" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1606724113" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1606724113" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1606724113"> </span></div> </div></div></div></div></div><div content-id="dsw062-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw062-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gamble</div> <div class="given-names">C. E.</div></div>, <div class="name string-name"><div class="surname">Brule</div> <div class="given-names">C. E.</div></div>, <div class="name string-name"><div class="surname">Dean</div> <div class="given-names">K. M.</div></div>, <div class="name string-name"><div class="surname">Fields</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Grayhack</div> <div class="given-names">E. J.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Adjacent codons act in concert to modulate translation efficiency in yeast</div>, <div class="source">Cell</div> , <div class="volume">166</div>, <div class="fpage">679</div>–<div class="lpage">90</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Adjacent%20codons%20act%20in%20concert%20to%20modulate%20translation%20efficiency%20in%20yeast&amp;author=C.%20E.%20Gamble&amp;author=C.%20E.%20Brule&amp;author=K.%20M.%20Dean&amp;author=S.%20Fields&amp;author=E.%20J.%20Grayhack&amp;publication_year=2016&amp;journal=Cell&amp;volume=166&amp;pages=679-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2016.05.070" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2016.05.070" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2016.05.070"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27374328" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2017. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<a class="link link-uri" href="http://creativecommons.org/licenses/by/4.0/" target="_blank">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=2972686&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw062/1/dsw062_Supp.pdf?Expires=2147483647&amp;Signature=hwp7bk5cUxNRikfSiIkl3TTPotX6N9DEEJ1zXu86eUPtNFMB~YA6Knkmbl2G3U93cBYnO4ZUn~ivFp6FWmS~uRTi4KVhxvk~1D4S5tIXKJEv~WcODSd~FPYYIrDjy6Byb~heC3TkYI6~2ibqOjma8-Kz94DphePxx5KjpUQG~J14fvlC1wcan17sCxRLVNIZ3MtSuahmFcDkVuLVlPJ1Fer-kfIIsk-3A~3b-MnRXtNkhZ0GvgxhcVEBJyt7i0htXRsjBVZwNLzuYK7v3lp-sEVD2QUJMR2h7PeEEDDnHFM0zoK256NjDv1LUPzTi7ouTIYp5VN6IRg4Bg0AikgJ8g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">1,036</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000404439600008&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">2</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsw062"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="2972686"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000404439600008&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f24%2f3%2f303%2f2972686">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000404439600008&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (2)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F24%2F3%2F303%2F2972686" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/2972686'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Ribo-seq enlightens codon usage bias" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsw062");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f24%2f3%2f303%2f2972686">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

