

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/13/6/245/464398","@type":"ScholarlyArticle","name":"Exploration and Grading of Possible Genes from 183 Bacterial Strains by a Common Protocol to Identification of New Genes: Gene Trek in Prokaryote Space (GTPS)","datePublished":"2006-01-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/13/6","@type":"PublicationIssue","issueNumber":"6","datePublished":"2006-01-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsl014","keywords":["comparative genome","gene prediction","annotation","gene grading","bacterial genome"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/13/6/245/464398","author":[{"name":"Kosuge, Takehide","affiliation":"1  Center for Information Biology, DNA Data Bank of Japan, National Institute of Genetics, The Graduate University for Advanced Studies (SOKENDAI)  Mishima, Shizuoka 411-8540, Japan","@type":"Person"},{"name":"Abe, Takashi","affiliation":"1  Center for Information Biology, DNA Data Bank of Japan, National Institute of Genetics, The Graduate University for Advanced Studies (SOKENDAI)  Mishima, Shizuoka 411-8540, Japan","@type":"Person"},{"name":"Okido, Toshihisa","affiliation":"1  Center for Information Biology, DNA Data Bank of Japan, National Institute of Genetics, The Graduate University for Advanced Studies (SOKENDAI)  Mishima, Shizuoka 411-8540, Japan","@type":"Person"},{"name":"Tanaka, Naoto","affiliation":"1  Center for Information Biology, DNA Data Bank of Japan, National Institute of Genetics, The Graduate University for Advanced Studies (SOKENDAI)  Mishima, Shizuoka 411-8540, Japan    2  Institute for Bioinformatics Research and Development (BIRD), Japan Science and Technology Corporation (JST)  Chiyoda-ku, Tokyo 102-8666, Japan    3  Faculty of Pharmaceutical Science, Tokyo University of Science, Noda  Chiba 278-8510, Japan","@type":"Person"},{"name":"Hirahata, Masaki","affiliation":"1  Center for Information Biology, DNA Data Bank of Japan, National Institute of Genetics, The Graduate University for Advanced Studies (SOKENDAI)  Mishima, Shizuoka 411-8540, Japan    2  Institute for Bioinformatics Research and Development (BIRD), Japan Science and Technology Corporation (JST)  Chiyoda-ku, Tokyo 102-8666, Japan","@type":"Person"},{"name":"Maruyama, Yutaka","affiliation":"1  Center for Information Biology, DNA Data Bank of Japan, National Institute of Genetics, The Graduate University for Advanced Studies (SOKENDAI)  Mishima, Shizuoka 411-8540, Japan    2  Institute for Bioinformatics Research and Development (BIRD), Japan Science and Technology Corporation (JST)  Chiyoda-ku, Tokyo 102-8666, Japan","@type":"Person"},{"name":"Mashima, Jun","affiliation":"1  Center for Information Biology, DNA Data Bank of Japan, National Institute of Genetics, The Graduate University for Advanced Studies (SOKENDAI)  Mishima, Shizuoka 411-8540, Japan","@type":"Person"},{"name":"Tomiki, Aki","affiliation":"1  Center for Information Biology, DNA Data Bank of Japan, National Institute of Genetics, The Graduate University for Advanced Studies (SOKENDAI)  Mishima, Shizuoka 411-8540, Japan","@type":"Person"},{"name":"Kurokawa, Motoyoshi","affiliation":"4  Advanced Center for Computing and Communication, RIKEN, Wako  Saitama 351-0198, Japan","@type":"Person"},{"name":"Himeno, Ryutaro","affiliation":"4  Advanced Center for Computing and Communication, RIKEN, Wako  Saitama 351-0198, Japan","@type":"Person"},{"name":"Fukuchi, Satoshi","affiliation":"1  Center for Information Biology, DNA Data Bank of Japan, National Institute of Genetics, The Graduate University for Advanced Studies (SOKENDAI)  Mishima, Shizuoka 411-8540, Japan","@type":"Person"},{"name":"Miyazaki, Satoru","affiliation":"1  Center for Information Biology, DNA Data Bank of Japan, National Institute of Genetics, The Graduate University for Advanced Studies (SOKENDAI)  Mishima, Shizuoka 411-8540, Japan    3  Faculty of Pharmaceutical Science, Tokyo University of Science, Noda  Chiba 278-8510, Japan","@type":"Person"},{"name":"Gojobori, Takashi","affiliation":"1  Center for Information Biology, DNA Data Bank of Japan, National Institute of Genetics, The Graduate University for Advanced Studies (SOKENDAI)  Mishima, Shizuoka 411-8540, Japan","@type":"Person"},{"name":"Tateno, Yoshio","affiliation":"1  Center for Information Biology, DNA Data Bank of Japan, National Institute of Genetics, The Graduate University for Advanced Studies (SOKENDAI)  Mishima, Shizuoka 411-8540, Japan","@type":"Person"},{"name":"Sugawara, Hideaki","affiliation":"1  Center for Information Biology, DNA Data Bank of Japan, National Institute of Genetics, The Graduate University for Advanced Studies (SOKENDAI)  Mishima, Shizuoka 411-8540, Japan","@type":"Person"}],"description":"Abstract.  A large number of complete microorganism genomes has been sequenced and submitted to the public database and then incorporated into our complete geno","pageStart":"245","pageEnd":"254","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/13/6/0/m_dnaresearch13_6.cover.gif?Expires=2147483647&Signature=fcRvTZS-Esu1iTIRpBwNtJBvy2AJRBCdQJyTXwh60M7JIQlqTrbnOTGA7L-7i6L4UYLG4JUSar16AKIY53IuiLWx-2mOh3JMXxdjEKlHYOAqNyuibso-D4bG3vOwDPxfhcZLg45cv2YcwYaGQxhzHrL5TjKyhZ4mwYb7GjG0uGA0KhpFZicZMiPYJGB~oynsMfd94bPAIrtQQPPmGzVGv0FtPTWthom~pPxQD8WgJhn5dGBE8EuXCW7bGuh87zOMErdi4RM6l1hIpsCmjH99FUAMlBlnn1Obqs6yCaDyJuJdsESPq4-uwP4YJIZZyz6U843ECQydjMQgSO4nkuzNbQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Exploration and Grading of Possible Genes from 183 Bacterial Strains by a Common Protocol to Identification of New Genes: Gene Trek in Prokaryote Space (GTPS)","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/13/6/0/m_dnaresearch13_6.cover.gif?Expires=2147483647&Signature=fcRvTZS-Esu1iTIRpBwNtJBvy2AJRBCdQJyTXwh60M7JIQlqTrbnOTGA7L-7i6L4UYLG4JUSar16AKIY53IuiLWx-2mOh3JMXxdjEKlHYOAqNyuibso-D4bG3vOwDPxfhcZLg45cv2YcwYaGQxhzHrL5TjKyhZ4mwYb7GjG0uGA0KhpFZicZMiPYJGB~oynsMfd94bPAIrtQQPPmGzVGv0FtPTWthom~pPxQD8WgJhn5dGBE8EuXCW7bGuh87zOMErdi4RM6l1hIpsCmjH99FUAMlBlnn1Obqs6yCaDyJuJdsESPq4-uwP4YJIZZyz6U843ECQydjMQgSO4nkuzNbQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Exploration and Grading of Possible Genes from 183 Bacterial Strains by a Common Protocol to Identification of New Genes: Gene Trek in Prokaryote Space (GTPS)" />
    <meta property="og:description" content="Abstract.  A large number of complete microorganism genomes has been sequenced and submitted to the public database and then incorporated into our complete geno" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/13/6/245/464398" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2006-01-01" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Kosuge, Takehide" /><meta name="citation_author_institution" content="1Center for Information Biology, DNA Data Bank of Japan, National Institute of Genetics, The Graduate University for Advanced Studies (SOKENDAI)Mishima, Shizuoka 411-8540, Japan" /><meta name="citation_author" content="Abe, Takashi" /><meta name="citation_author_institution" content="1Center for Information Biology, DNA Data Bank of Japan, National Institute of Genetics, The Graduate University for Advanced Studies (SOKENDAI)Mishima, Shizuoka 411-8540, Japan" /><meta name="citation_author" content="Okido, Toshihisa" /><meta name="citation_author_institution" content="1Center for Information Biology, DNA Data Bank of Japan, National Institute of Genetics, The Graduate University for Advanced Studies (SOKENDAI)Mishima, Shizuoka 411-8540, Japan" /><meta name="citation_author" content="Tanaka, Naoto" /><meta name="citation_author_institution" content="1Center for Information Biology, DNA Data Bank of Japan, National Institute of Genetics, The Graduate University for Advanced Studies (SOKENDAI)Mishima, Shizuoka 411-8540, Japan" /><meta name="citation_author_institution" content="2Institute for Bioinformatics Research and Development (BIRD), Japan Science and Technology Corporation (JST)Chiyoda-ku, Tokyo 102-8666, Japan" /><meta name="citation_author_institution" content="3Faculty of Pharmaceutical Science, Tokyo University of Science, NodaChiba 278-8510, Japan" /><meta name="citation_author" content="Hirahata, Masaki" /><meta name="citation_author_institution" content="1Center for Information Biology, DNA Data Bank of Japan, National Institute of Genetics, The Graduate University for Advanced Studies (SOKENDAI)Mishima, Shizuoka 411-8540, Japan" /><meta name="citation_author_institution" content="2Institute for Bioinformatics Research and Development (BIRD), Japan Science and Technology Corporation (JST)Chiyoda-ku, Tokyo 102-8666, Japan" /><meta name="citation_author" content="Maruyama, Yutaka" /><meta name="citation_author_institution" content="1Center for Information Biology, DNA Data Bank of Japan, National Institute of Genetics, The Graduate University for Advanced Studies (SOKENDAI)Mishima, Shizuoka 411-8540, Japan" /><meta name="citation_author_institution" content="2Institute for Bioinformatics Research and Development (BIRD), Japan Science and Technology Corporation (JST)Chiyoda-ku, Tokyo 102-8666, Japan" /><meta name="citation_author" content="Mashima, Jun" /><meta name="citation_author_institution" content="1Center for Information Biology, DNA Data Bank of Japan, National Institute of Genetics, The Graduate University for Advanced Studies (SOKENDAI)Mishima, Shizuoka 411-8540, Japan" /><meta name="citation_author" content="Tomiki, Aki" /><meta name="citation_author_institution" content="1Center for Information Biology, DNA Data Bank of Japan, National Institute of Genetics, The Graduate University for Advanced Studies (SOKENDAI)Mishima, Shizuoka 411-8540, Japan" /><meta name="citation_author" content="Kurokawa, Motoyoshi" /><meta name="citation_author_institution" content="4Advanced Center for Computing and Communication, RIKEN, WakoSaitama 351-0198, Japan" /><meta name="citation_author" content="Himeno, Ryutaro" /><meta name="citation_author_institution" content="4Advanced Center for Computing and Communication, RIKEN, WakoSaitama 351-0198, Japan" /><meta name="citation_author" content="Fukuchi, Satoshi" /><meta name="citation_author_institution" content="1Center for Information Biology, DNA Data Bank of Japan, National Institute of Genetics, The Graduate University for Advanced Studies (SOKENDAI)Mishima, Shizuoka 411-8540, Japan" /><meta name="citation_author" content="Miyazaki, Satoru" /><meta name="citation_author_institution" content="1Center for Information Biology, DNA Data Bank of Japan, National Institute of Genetics, The Graduate University for Advanced Studies (SOKENDAI)Mishima, Shizuoka 411-8540, Japan" /><meta name="citation_author_institution" content="3Faculty of Pharmaceutical Science, Tokyo University of Science, NodaChiba 278-8510, Japan" /><meta name="citation_author" content="Gojobori, Takashi" /><meta name="citation_author_institution" content="1Center for Information Biology, DNA Data Bank of Japan, National Institute of Genetics, The Graduate University for Advanced Studies (SOKENDAI)Mishima, Shizuoka 411-8540, Japan" /><meta name="citation_author" content="Tateno, Yoshio" /><meta name="citation_author_institution" content="1Center for Information Biology, DNA Data Bank of Japan, National Institute of Genetics, The Graduate University for Advanced Studies (SOKENDAI)Mishima, Shizuoka 411-8540, Japan" /><meta name="citation_author" content="Sugawara, Hideaki" /><meta name="citation_author_institution" content="1Center for Information Biology, DNA Data Bank of Japan, National Institute of Genetics, The Graduate University for Advanced Studies (SOKENDAI)Mishima, Shizuoka 411-8540, Japan" /><meta name="citation_title" content="Exploration and Grading of Possible Genes from 183 Bacterial Strains by a Common Protocol to Identification of New Genes: Gene Trek in Prokaryote Space (GTPS)" /><meta name="citation_firstpage" content="245" /><meta name="citation_lastpage" content="254" /><meta name="citation_doi" content="10.1093/dnares/dsl014" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="13" /><meta name="citation_issue" content="6" /><meta name="citation_publication_date" content="2006/01/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Whole-genome random sequencing and assembly of Haemophilus influenzae Rd; Science;  citation_year=1995;  citation_volume=269;  citation_pages=496-512; " /><meta name="citation_reference" content="citation_title=Genomic mapping by fingerprinting random clones: a mathematical analysis; Genomics;  citation_year=1988;  citation_volume=2;  citation_pages=231-239; " /><meta name="citation_reference" content="citation_title=Genome Information Broker (GIB): data retrieval and comparative analysis system for completed microbial genomes and more; Nucleic Acids Res.;  citation_year=2002;  citation_volume=30;  citation_pages=66-68; " /><meta name="citation_reference" content="citation_title=Phylogenetic identification and in situ detection of individual microbial cells without cultivation; Microbiol. Rev.;  citation_year=1995;  citation_volume=59;  citation_pages=143-169; " /><meta name="citation_reference" content="citation_title=Identifying microbial diversity in the natural environment: a molecular phylogenetic approach; Trends Biotechnol.;  citation_year=1996;  citation_volume=14;  citation_pages=190-197; " /><meta name="citation_reference" content="citation_title=Estimating prokaryotic diversity and its limits; Proc. Natl Acad. Sci. USA.;  citation_year=2002;  citation_volume=99;  citation_pages=10494-10499; " /><meta name="citation_reference" content="citation_title=The complete genome sequence of Escherichia coli, K-12; Science;  citation_year=1997;  citation_volume=277;  citation_pages=1453-1474; " /><meta name="citation_reference" content="citation_title=Escherichia coli K-12: a cooperatively developed annotation snapshot—2005; Nucleic Acids Res.;  citation_year=2006;  citation_volume=34;  citation_pages=1-9; " /><meta name="citation_reference" content="citation_title=The c-sis gene encodes a precursor of the B chain of platelet-derived growth factor; EMBO J.;  citation_year=1984;  citation_volume=3;  citation_pages=921-928; " /><meta name="citation_reference" content="citation_title=Basic local alignment search tool; J. Mol. Biol.;  citation_year=1990;  citation_volume=215;  citation_pages=403-410; " /><meta name="citation_reference" content="citation_title=Comparative genomics of prokaryotic GTP-binding proteins [the Era, Obg, EngA, ThdF (TrmE), YchF and YihA families] and their relationship to eukaryotic GTP-binding proteins (the DRG, ARF, RAB, RAN, RAS and RHO families); J. Mol. Microbiol. Biotechnol.;  citation_year=2001;  citation_volume=3;  citation_pages=21-35; " /><meta name="citation_reference" content="citation_title=Towards a genome-based taxonomy for prokaryotes; J. Bacteriol.;  citation_year=2005;  citation_volume=187;  citation_pages=6258-6264; " /><meta name="citation_reference" content="citation_title=Searching for hypothetical proteins: theory and practice based upon original data and literature; Prog. Neurobiol.;  citation_year=2005;  citation_volume=77;  citation_pages=90-127; " /><meta name="citation_reference" content="citation_title=Virulence insights from the Paracoccidioides brasiliensis transcriptome; Genet. Mol. Res.;  citation_year=2005;  citation_volume=4;  citation_pages=372-389; " /><meta name="citation_reference" content="citation_title=Identifying protein function—a call for community action; PLoS Biol.;  citation_year=2004;  citation_volume=2;  citation_pages=E42" /><meta name="citation_reference" content="citation_title=The phylogenetic distribution and evolutionary origins of endocannabinoid signalling; Handb. Exp. Pharmacol.;  citation_year=2005;  citation_pages=283-297; " /><meta name="citation_reference" content="citation_title=Bacterial genomics and pathogen evolution; Cell;  citation_year=2006;  citation_volume=124;  citation_pages=703-714; " /><meta name="citation_reference" content="citation_title=Insights on biology and evolution from microbial genome sequencing; Genome Res.;  citation_year=2005;  citation_volume=15;  citation_pages=1603-1610; " /><meta name="citation_reference" content="citation_title=Comparative genomics of emerging human ehrlichiosis agents; PLoS Genet.;  citation_year=2006;  citation_volume=2;  citation_pages=e21" /><meta name="citation_reference" content="citation_title=tRNAscan-SE: a program for improved detection of transfer RNA genes in genomic sequence; Nucleic Acids Res.;  citation_year=1997;  citation_volume=25;  citation_pages=955-964; " /><meta name="citation_reference" content="citation_title=Rfam: an RNA family database; Nucleic Acids Res.;  citation_year=2003;  citation_volume=31;  citation_pages=439-441; " /><meta name="citation_reference" content="citation_title=Microbial gene identification using interpolated Markov models; Nucleic Acids Res.;  citation_year=1998;  citation_volume=26;  citation_pages=544-548; " /><meta name="citation_reference" content="citation_title=Improved microbial gene identification with GLIMMER; Nucleic Acids Res.;  citation_year=1999;  citation_volume=27;  citation_pages=4636-4641; " /><meta name="citation_reference" content="citation_title=Ab initio gene identification: prokaryote genome annotation with GeneScan and GLIMMER; J. Biosci.;  citation_year=2002;  citation_volume=27;  citation_pages=7-14; " /><meta name="citation_reference" content="citation_title=The Universal Protein Resource (UniProt); Nucleic Acids Res.;  citation_year=2005;  citation_volume=33;  citation_pages=D154-D159; " /><meta name="citation_reference" content="citation_title=A probabilistic method for identifying start codons in bacterial genomes; Bioinformatics;  citation_year=2001;  citation_volume=17;  citation_pages=1123-1130; " /><meta name="citation_reference" content="citation_title=The complete genome sequence of the gram-positive bacterium Bacillus subtilis; Nature;  citation_year=1997;  citation_volume=390;  citation_pages=249-256; " /><meta name="citation_reference" content="citation_title=Multivariate entropy distance method for prokaryotic gene identification; J. Bioinformatics and Comp. Biol.;  citation_year=2004;  citation_volume=2;  citation_pages=353-373; " /><meta name="citation_reference" content="citation_title=The InterPro database, an integrated documentation resource for protein families, domains and functional sites; Nucleic Acids Res.;  citation_year=2001;  citation_volume=29;  citation_pages=37-40; " /><meta name="citation_reference" content="citation_title=GTOP: a database of protein structures predicted from genome sequences; Nucleic Acids Res.;  citation_year=2002;  citation_volume=30;  citation_pages=294-298; " /><meta name="citation_reference" content="citation_title=Highly accurate genome sequences of Escherichia coli K-12 strains MG1655 and W3110; Mol. Syst. Biol.;  citation_year=2006;  citation_volume=2;  citation_pages=0007" /><meta name="citation_reference" content="citation_title=Clustering of highly homologous sequences to reduce the size of large protein databases; Bioinformatics;  citation_year=2001;  citation_volume=17;  citation_pages=282-283; " /><meta name="citation_reference" content="citation_title=Tolerating some redundancy significantly speeds up clustering of large protein databases; Bioinformatics;  citation_year=2002;  citation_volume=18;  citation_pages=77-82; " /><meta name="citation_reference" content="citation_title=Environmental genome shotgun sequencing of the Sargasso Sea; Science;  citation_year=2004;  citation_volume=304;  citation_pages=66-74; " /><meta name="citation_reference" content="citation_title=Genome sequencing in microfabricated high-density picolitre reactors; Nature;  citation_year=2005;  citation_volume=437;  citation_pages=376-380; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/13/6/245/1259659/dsl014.pdf" /><meta name="description" content="Abstract.  A large number of complete microorganism genomes has been sequenced and submitted to the public database and then incorporated into our complete geno" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/13/6/245/464398" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/13/6/245/464398" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/13/6/245/464398");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsl014';
    </script>
<title>
	Exploration and Grading of Possible Genes from 183 Bacterial Strains by a Common Protocol to Identification of New Genes: Gene Trek in Prokaryote Space (GTPS) | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./464398" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BfNrPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLb3N1Z2UsIFRha2VoaWRlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iMUNlbnRlciBmb3IgSW5mb3JtYXRpb24gQmlvbG9neSwgRE5BIERhdGEgQmFuayBvZiBKYXBhbiwgTmF0aW9uYWwgSW5zdGl0dXRlIG9mIEdlbmV0aWNzLCBUaGUgR3JhZHVhdGUgVW5pdmVyc2l0eSBmb3IgQWR2YW5jZWQgU3R1ZGllcyAoU09LRU5EQUkpTWlzaGltYSwgU2hpenVva2EgNDExLTg1NDAsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQWJlLCBUYWthc2hpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iMUNlbnRlciBmb3IgSW5mb3JtYXRpb24gQmlvbG9neSwgRE5BIERhdGEgQmFuayBvZiBKYXBhbiwgTmF0aW9uYWwgSW5zdGl0dXRlIG9mIEdlbmV0aWNzLCBUaGUgR3JhZHVhdGUgVW5pdmVyc2l0eSBmb3IgQWR2YW5jZWQgU3R1ZGllcyAoU09LRU5EQUkpTWlzaGltYSwgU2hpenVva2EgNDExLTg1NDAsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iT2tpZG8sIFRvc2hpaGlzYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IjFDZW50ZXIgZm9yIEluZm9ybWF0aW9uIEJpb2xvZ3ksIEROQSBEYXRhIEJhbmsgb2YgSmFwYW4sIE5hdGlvbmFsIEluc3RpdHV0ZSBvZiBHZW5ldGljcywgVGhlIEdyYWR1YXRlIFVuaXZlcnNpdHkgZm9yIEFkdmFuY2VkIFN0dWRpZXMgKFNPS0VOREFJKU1pc2hpbWEsIFNoaXp1b2thIDQxMS04NTQwLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlRhbmFrYSwgTmFvdG8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSIxQ2VudGVyIGZvciBJbmZvcm1hdGlvbiBCaW9sb2d5LCBETkEgRGF0YSBCYW5rIG9mIEphcGFuLCBOYXRpb25hbCBJbnN0aXR1dGUgb2YgR2VuZXRpY3MsIFRoZSBHcmFkdWF0ZSBVbml2ZXJzaXR5IGZvciBBZHZhbmNlZCBTdHVkaWVzIChTT0tFTkRBSSlNaXNoaW1hLCBTaGl6dW9rYSA0MTEtODU0MCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSIySW5zdGl0dXRlIGZvciBCaW9pbmZvcm1hdGljcyBSZXNlYXJjaCBhbmQgRGV2ZWxvcG1lbnQgKEJJUkQpLCBKYXBhbiBTY2llbmNlIGFuZCBUZWNobm9sb2d5IENvcnBvcmF0aW9uIChKU1QpQ2hpeW9kYS1rdSwgVG9reW8gMTAyLTg2NjYsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iM0ZhY3VsdHkgb2YgUGhhcm1hY2V1dGljYWwgU2NpZW5jZSwgVG9reW8gVW5pdmVyc2l0eSBvZiBTY2llbmNlLCBOb2RhQ2hpYmEgMjc4LTg1MTAsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSGlyYWhhdGEsIE1hc2FraSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IjFDZW50ZXIgZm9yIEluZm9ybWF0aW9uIEJpb2xvZ3ksIEROQSBEYXRhIEJhbmsgb2YgSmFwYW4sIE5hdGlvbmFsIEluc3RpdHV0ZSBvZiBHZW5ldGljcywgVGhlIEdyYWR1YXRlIFVuaXZlcnNpdHkgZm9yIEFkdmFuY2VkIFN0dWRpZXMgKFNPS0VOREFJKU1pc2hpbWEsIFNoaXp1b2thIDQxMS04NTQwLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IjJJbnN0aXR1dGUgZm9yIEJpb2luZm9ybWF0aWNzIFJlc2VhcmNoIGFuZCBEZXZlbG9wbWVudCAoQklSRCksIEphcGFuIFNjaWVuY2UgYW5kIFRlY2hub2xvZ3kgQ29ycG9yYXRpb24gKEpTVClDaGl5b2RhLWt1LCBUb2t5byAxMDItODY2NiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJNYXJ1eWFtYSwgWXV0YWthIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iMUNlbnRlciBmb3IgSW5mb3JtYXRpb24gQmlvbG9neSwgRE5BIERhdGEgQmFuayBvZiBKYXBhbiwgTmF0aW9uYWwgSW5zdGl0dXRlIG9mIEdlbmV0aWNzLCBUaGUgR3JhZHVhdGUgVW5pdmVyc2l0eSBmb3IgQWR2YW5jZWQgU3R1ZGllcyAoU09LRU5EQUkpTWlzaGltYSwgU2hpenVva2EgNDExLTg1NDAsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iMkluc3RpdHV0ZSBmb3IgQmlvaW5mb3JtYXRpY3MgUmVzZWFyY2ggYW5kIERldmVsb3BtZW50IChCSVJEKSwgSmFwYW4gU2NpZW5jZSBhbmQgVGVjaG5vbG9neSBDb3Jwb3JhdGlvbiAoSlNUKUNoaXlvZGEta3UsIFRva3lvIDEwMi04NjY2LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik1hc2hpbWEsIEp1biIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IjFDZW50ZXIgZm9yIEluZm9ybWF0aW9uIEJpb2xvZ3ksIEROQSBEYXRhIEJhbmsgb2YgSmFwYW4sIE5hdGlvbmFsIEluc3RpdHV0ZSBvZiBHZW5ldGljcywgVGhlIEdyYWR1YXRlIFVuaXZlcnNpdHkgZm9yIEFkdmFuY2VkIFN0dWRpZXMgKFNPS0VOREFJKU1pc2hpbWEsIFNoaXp1b2thIDQxMS04NTQwLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlRvbWlraSwgQWtpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iMUNlbnRlciBmb3IgSW5mb3JtYXRpb24gQmlvbG9neSwgRE5BIERhdGEgQmFuayBvZiBKYXBhbiwgTmF0aW9uYWwgSW5zdGl0dXRlIG9mIEdlbmV0aWNzLCBUaGUgR3JhZHVhdGUgVW5pdmVyc2l0eSBmb3IgQWR2YW5jZWQgU3R1ZGllcyAoU09LRU5EQUkpTWlzaGltYSwgU2hpenVva2EgNDExLTg1NDAsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS3Vyb2thd2EsIE1vdG95b3NoaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IjRBZHZhbmNlZCBDZW50ZXIgZm9yIENvbXB1dGluZyBhbmQgQ29tbXVuaWNhdGlvbiwgUklLRU4sIFdha29TYWl0YW1hIDM1MS0wMTk4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkhpbWVubywgUnl1dGFybyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IjRBZHZhbmNlZCBDZW50ZXIgZm9yIENvbXB1dGluZyBhbmQgQ29tbXVuaWNhdGlvbiwgUklLRU4sIFdha29TYWl0YW1hIDM1MS0wMTk4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkZ1a3VjaGksIFNhdG9zaGkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSIxQ2VudGVyIGZvciBJbmZvcm1hdGlvbiBCaW9sb2d5LCBETkEgRGF0YSBCYW5rIG9mIEphcGFuLCBOYXRpb25hbCBJbnN0aXR1dGUgb2YgR2VuZXRpY3MsIFRoZSBHcmFkdWF0ZSBVbml2ZXJzaXR5IGZvciBBZHZhbmNlZCBTdHVkaWVzIChTT0tFTkRBSSlNaXNoaW1hLCBTaGl6dW9rYSA0MTEtODU0MCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJNaXlhemFraSwgU2F0b3J1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iMUNlbnRlciBmb3IgSW5mb3JtYXRpb24gQmlvbG9neSwgRE5BIERhdGEgQmFuayBvZiBKYXBhbiwgTmF0aW9uYWwgSW5zdGl0dXRlIG9mIEdlbmV0aWNzLCBUaGUgR3JhZHVhdGUgVW5pdmVyc2l0eSBmb3IgQWR2YW5jZWQgU3R1ZGllcyAoU09LRU5EQUkpTWlzaGltYSwgU2hpenVva2EgNDExLTg1NDAsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iM0ZhY3VsdHkgb2YgUGhhcm1hY2V1dGljYWwgU2NpZW5jZSwgVG9reW8gVW5pdmVyc2l0eSBvZiBTY2llbmNlLCBOb2RhQ2hpYmEgMjc4LTg1MTAsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iR29qb2JvcmksIFRha2FzaGkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSIxQ2VudGVyIGZvciBJbmZvcm1hdGlvbiBCaW9sb2d5LCBETkEgRGF0YSBCYW5rIG9mIEphcGFuLCBOYXRpb25hbCBJbnN0aXR1dGUgb2YgR2VuZXRpY3MsIFRoZSBHcmFkdWF0ZSBVbml2ZXJzaXR5IGZvciBBZHZhbmNlZCBTdHVkaWVzIChTT0tFTkRBSSlNaXNoaW1hLCBTaGl6dW9rYSA0MTEtODU0MCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJUYXRlbm8sIFlvc2hpbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IjFDZW50ZXIgZm9yIEluZm9ybWF0aW9uIEJpb2xvZ3ksIEROQSBEYXRhIEJhbmsgb2YgSmFwYW4sIE5hdGlvbmFsIEluc3RpdHV0ZSBvZiBHZW5ldGljcywgVGhlIEdyYWR1YXRlIFVuaXZlcnNpdHkgZm9yIEFkdmFuY2VkIFN0dWRpZXMgKFNPS0VOREFJKU1pc2hpbWEsIFNoaXp1b2thIDQxMS04NTQwLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlN1Z2F3YXJhLCBIaWRlYWtpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iMUNlbnRlciBmb3IgSW5mb3JtYXRpb24gQmlvbG9neSwgRE5BIERhdGEgQmFuayBvZiBKYXBhbiwgTmF0aW9uYWwgSW5zdGl0dXRlIG9mIEdlbmV0aWNzLCBUaGUgR3JhZHVhdGUgVW5pdmVyc2l0eSBmb3IgQWR2YW5jZWQgU3R1ZGllcyAoU09LRU5EQUkpTWlzaGltYSwgU2hpenVva2EgNDExLTg1NDAsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJFeHBsb3JhdGlvbiBhbmQgR3JhZGluZyBvZiBQb3NzaWJsZSBHZW5lcyBmcm9tIDE4MyBCYWN0ZXJpYWwgU3RyYWlucyBieSBhIENvbW1vbiBQcm90b2NvbCB0byBJZGVudGlmaWNhdGlvbiBvZiBOZXcgR2VuZXM6IEdlbmUgVHJlayBpbiBQcm9rYXJ5b3RlIFNwYWNlIChHVFBTKSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjI0NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iMjU0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHNsMDE0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjEzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSI2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMDYvMDEvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1XaG9sZS1nZW5vbWUgcmFuZG9tIHNlcXVlbmNpbmcgYW5kIGFzc2VtYmx5IG9mIEhhZW1vcGhpbHVzIGluZmx1ZW56YWUgUmQ7IFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTE5OTU7ICBjaXRhdGlvbl92b2x1bWU9MjY5OyAgY2l0YXRpb25fcGFnZXM9NDk2LTUxMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21pYyBtYXBwaW5nIGJ5IGZpbmdlcnByaW50aW5nIHJhbmRvbSBjbG9uZXM6IGEgbWF0aGVtYXRpY2FsIGFuYWx5c2lzOyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MTk4ODsgIGNpdGF0aW9uX3ZvbHVtZT0yOyAgY2l0YXRpb25fcGFnZXM9MjMxLTIzOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lIEluZm9ybWF0aW9uIEJyb2tlciAoR0lCKTogZGF0YSByZXRyaWV2YWwgYW5kIGNvbXBhcmF0aXZlIGFuYWx5c2lzIHN5c3RlbSBmb3IgY29tcGxldGVkIG1pY3JvYmlhbCBnZW5vbWVzIGFuZCBtb3JlOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MzA7ICBjaXRhdGlvbl9wYWdlcz02Ni02ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGh5bG9nZW5ldGljIGlkZW50aWZpY2F0aW9uIGFuZCBpbiBzaXR1IGRldGVjdGlvbiBvZiBpbmRpdmlkdWFsIG1pY3JvYmlhbCBjZWxscyB3aXRob3V0IGN1bHRpdmF0aW9uOyBNaWNyb2Jpb2wuIFJldi47ICBjaXRhdGlvbl95ZWFyPTE5OTU7ICBjaXRhdGlvbl92b2x1bWU9NTk7ICBjaXRhdGlvbl9wYWdlcz0xNDMtMTY5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JZGVudGlmeWluZyBtaWNyb2JpYWwgZGl2ZXJzaXR5IGluIHRoZSBuYXR1cmFsIGVudmlyb25tZW50OiBhIG1vbGVjdWxhciBwaHlsb2dlbmV0aWMgYXBwcm9hY2g7IFRyZW5kcyBCaW90ZWNobm9sLjsgIGNpdGF0aW9uX3llYXI9MTk5NjsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTE5MC0xOTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVzdGltYXRpbmcgcHJva2FyeW90aWMgZGl2ZXJzaXR5IGFuZCBpdHMgbGltaXRzOyBQcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBLjsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT05OTsgIGNpdGF0aW9uX3BhZ2VzPTEwNDk0LTEwNDk5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgY29tcGxldGUgZ2Vub21lIHNlcXVlbmNlIG9mIEVzY2hlcmljaGlhIGNvbGksIEstMTI7IFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9Mjc3OyAgY2l0YXRpb25fcGFnZXM9MTQ1My0xNDc0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Fc2NoZXJpY2hpYSBjb2xpIEstMTI6IGEgY29vcGVyYXRpdmVseSBkZXZlbG9wZWQgYW5ub3RhdGlvbiBzbmFwc2hvdOKAlDIwMDU7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0zNDsgIGNpdGF0aW9uX3BhZ2VzPTEtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGMtc2lzIGdlbmUgZW5jb2RlcyBhIHByZWN1cnNvciBvZiB0aGUgQiBjaGFpbiBvZiBwbGF0ZWxldC1kZXJpdmVkIGdyb3d0aCBmYWN0b3I7IEVNQk8gSi47ICBjaXRhdGlvbl95ZWFyPTE5ODQ7ICBjaXRhdGlvbl92b2x1bWU9MzsgIGNpdGF0aW9uX3BhZ2VzPTkyMS05Mjg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJhc2ljIGxvY2FsIGFsaWdubWVudCBzZWFyY2ggdG9vbDsgSi4gTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5MDsgIGNpdGF0aW9uX3ZvbHVtZT0yMTU7ICBjaXRhdGlvbl9wYWdlcz00MDMtNDEwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wYXJhdGl2ZSBnZW5vbWljcyBvZiBwcm9rYXJ5b3RpYyBHVFAtYmluZGluZyBwcm90ZWlucyBbdGhlIEVyYSwgT2JnLCBFbmdBLCBUaGRGIChUcm1FKSwgWWNoRiBhbmQgWWloQSBmYW1pbGllc10gYW5kIHRoZWlyIHJlbGF0aW9uc2hpcCB0byBldWthcnlvdGljIEdUUC1iaW5kaW5nIHByb3RlaW5zICh0aGUgRFJHLCBBUkYsIFJBQiwgUkFOLCBSQVMgYW5kIFJITyBmYW1pbGllcyk7IEouIE1vbC4gTWljcm9iaW9sLiBCaW90ZWNobm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0zOyAgY2l0YXRpb25fcGFnZXM9MjEtMzU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRvd2FyZHMgYSBnZW5vbWUtYmFzZWQgdGF4b25vbXkgZm9yIHByb2thcnlvdGVzOyBKLiBCYWN0ZXJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTE4NzsgIGNpdGF0aW9uX3BhZ2VzPTYyNTgtNjI2NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2VhcmNoaW5nIGZvciBoeXBvdGhldGljYWwgcHJvdGVpbnM6IHRoZW9yeSBhbmQgcHJhY3RpY2UgYmFzZWQgdXBvbiBvcmlnaW5hbCBkYXRhIGFuZCBsaXRlcmF0dXJlOyBQcm9nLiBOZXVyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTc3OyAgY2l0YXRpb25fcGFnZXM9OTAtMTI3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1WaXJ1bGVuY2UgaW5zaWdodHMgZnJvbSB0aGUgUGFyYWNvY2NpZGlvaWRlcyBicmFzaWxpZW5zaXMgdHJhbnNjcmlwdG9tZTsgR2VuZXQuIE1vbC4gUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9MzcyLTM4OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZnlpbmcgcHJvdGVpbiBmdW5jdGlvbuKAlGEgY2FsbCBmb3IgY29tbXVuaXR5IGFjdGlvbjsgUExvUyBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0yOyAgY2l0YXRpb25fcGFnZXM9RTQyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHBoeWxvZ2VuZXRpYyBkaXN0cmlidXRpb24gYW5kIGV2b2x1dGlvbmFyeSBvcmlnaW5zIG9mIGVuZG9jYW5uYWJpbm9pZCBzaWduYWxsaW5nOyBIYW5kYi4gRXhwLiBQaGFybWFjb2wuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fcGFnZXM9MjgzLTI5NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QmFjdGVyaWFsIGdlbm9taWNzIGFuZCBwYXRob2dlbiBldm9sdXRpb247IENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MTI0OyAgY2l0YXRpb25fcGFnZXM9NzAzLTcxNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW5zaWdodHMgb24gYmlvbG9neSBhbmQgZXZvbHV0aW9uIGZyb20gbWljcm9iaWFsIGdlbm9tZSBzZXF1ZW5jaW5nOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTE2MDMtMTYxMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyYXRpdmUgZ2Vub21pY3Mgb2YgZW1lcmdpbmcgaHVtYW4gZWhybGljaGlvc2lzIGFnZW50czsgUExvUyBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MjsgIGNpdGF0aW9uX3BhZ2VzPWUyMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPXRSTkFzY2FuLVNFOiBhIHByb2dyYW0gZm9yIGltcHJvdmVkIGRldGVjdGlvbiBvZiB0cmFuc2ZlciBSTkEgZ2VuZXMgaW4gZ2Vub21pYyBzZXF1ZW5jZTsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9OTU1LTk2NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmZhbTogYW4gUk5BIGZhbWlseSBkYXRhYmFzZTsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTMxOyAgY2l0YXRpb25fcGFnZXM9NDM5LTQ0MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWljcm9iaWFsIGdlbmUgaWRlbnRpZmljYXRpb24gdXNpbmcgaW50ZXJwb2xhdGVkIE1hcmtvdiBtb2RlbHM7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT0yNjsgIGNpdGF0aW9uX3BhZ2VzPTU0NC01NDg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUltcHJvdmVkIG1pY3JvYmlhbCBnZW5lIGlkZW50aWZpY2F0aW9uIHdpdGggR0xJTU1FUjsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTI3OyAgY2l0YXRpb25fcGFnZXM9NDYzNi00NjQxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BYiBpbml0aW8gZ2VuZSBpZGVudGlmaWNhdGlvbjogcHJva2FyeW90ZSBnZW5vbWUgYW5ub3RhdGlvbiB3aXRoIEdlbmVTY2FuIGFuZCBHTElNTUVSOyBKLiBCaW9zY2kuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTI3OyAgY2l0YXRpb25fcGFnZXM9Ny0xNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIFVuaXZlcnNhbCBQcm90ZWluIFJlc291cmNlIChVbmlQcm90KTsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTMzOyAgY2l0YXRpb25fcGFnZXM9RDE1NC1EMTU5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHByb2JhYmlsaXN0aWMgbWV0aG9kIGZvciBpZGVudGlmeWluZyBzdGFydCBjb2RvbnMgaW4gYmFjdGVyaWFsIGdlbm9tZXM7IEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTE3OyAgY2l0YXRpb25fcGFnZXM9MTEyMy0xMTMwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgY29tcGxldGUgZ2Vub21lIHNlcXVlbmNlIG9mIHRoZSBncmFtLXBvc2l0aXZlIGJhY3Rlcml1bSBCYWNpbGx1cyBzdWJ0aWxpczsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTM5MDsgIGNpdGF0aW9uX3BhZ2VzPTI0OS0yNTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU11bHRpdmFyaWF0ZSBlbnRyb3B5IGRpc3RhbmNlIG1ldGhvZCBmb3IgcHJva2FyeW90aWMgZ2VuZSBpZGVudGlmaWNhdGlvbjsgSi4gQmlvaW5mb3JtYXRpY3MgYW5kIENvbXAuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTI7ICBjaXRhdGlvbl9wYWdlcz0zNTMtMzczOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgSW50ZXJQcm8gZGF0YWJhc2UsIGFuIGludGVncmF0ZWQgZG9jdW1lbnRhdGlvbiByZXNvdXJjZSBmb3IgcHJvdGVpbiBmYW1pbGllcywgZG9tYWlucyBhbmQgZnVuY3Rpb25hbCBzaXRlczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTI5OyAgY2l0YXRpb25fcGFnZXM9MzctNDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdUT1A6IGEgZGF0YWJhc2Ugb2YgcHJvdGVpbiBzdHJ1Y3R1cmVzIHByZWRpY3RlZCBmcm9tIGdlbm9tZSBzZXF1ZW5jZXM7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTI5NC0yOTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhpZ2hseSBhY2N1cmF0ZSBnZW5vbWUgc2VxdWVuY2VzIG9mIEVzY2hlcmljaGlhIGNvbGkgSy0xMiBzdHJhaW5zIE1HMTY1NSBhbmQgVzMxMTA7IE1vbC4gU3lzdC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MjsgIGNpdGF0aW9uX3BhZ2VzPTAwMDciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DbHVzdGVyaW5nIG9mIGhpZ2hseSBob21vbG9nb3VzIHNlcXVlbmNlcyB0byByZWR1Y2UgdGhlIHNpemUgb2YgbGFyZ2UgcHJvdGVpbiBkYXRhYmFzZXM7IEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTE3OyAgY2l0YXRpb25fcGFnZXM9MjgyLTI4MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VG9sZXJhdGluZyBzb21lIHJlZHVuZGFuY3kgc2lnbmlmaWNhbnRseSBzcGVlZHMgdXAgY2x1c3RlcmluZyBvZiBsYXJnZSBwcm90ZWluIGRhdGFiYXNlczsgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MTg7ICBjaXRhdGlvbl9wYWdlcz03Ny04MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RW52aXJvbm1lbnRhbCBnZW5vbWUgc2hvdGd1biBzZXF1ZW5jaW5nIG9mIHRoZSBTYXJnYXNzbyBTZWE7IFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MzA0OyAgY2l0YXRpb25fcGFnZXM9NjYtNzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZSBzZXF1ZW5jaW5nIGluIG1pY3JvZmFicmljYXRlZCBoaWdoLWRlbnNpdHkgcGljb2xpdHJlIHJlYWN0b3JzOyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9NDM3OyAgY2l0YXRpb25fcGFnZXM9Mzc2LTM4MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzEzLzYvMjQ1LzEyNTk2NTkvZHNsMDE0LnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBBIGxhcmdlIG51bWJlciBvZiBjb21wbGV0ZSBtaWNyb29yZ2FuaXNtIGdlbm9tZXMgaGFzIGJlZW4gc2VxdWVuY2VkIGFuZCBzdWJtaXR0ZWQgdG8gdGhlIHB1YmxpYyBkYXRhYmFzZSBhbmQgdGhlbiBpbmNvcnBvcmF0ZWQgaW50byBvdXIgY29tcGxldGUgZ2VubyIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXIvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9NDY0Mzk4JnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZC5VCdhzzICL8R+D8WEFPebofd7E" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f13%2f6%2f245%2f464398" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f13%2f6%2f245%2f464398" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=18198" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/13/6"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/13/6/0/m_dnaresearch13_6.cover.gif?Expires=2147483647&amp;Signature=fcRvTZS-Esu1iTIRpBwNtJBvy2AJRBCdQJyTXwh60M7JIQlqTrbnOTGA7L-7i6L4UYLG4JUSar16AKIY53IuiLWx-2mOh3JMXxdjEKlHYOAqNyuibso-D4bG3vOwDPxfhcZLg45cv2YcwYaGQxhzHrL5TjKyhZ4mwYb7GjG0uGA0KhpFZicZMiPYJGB~oynsMfd94bPAIrtQQPPmGzVGv0FtPTWthom~pPxQD8WgJhn5dGBE8EuXCW7bGuh87zOMErdi4RM6l1hIpsCmjH99FUAMlBlnn1Obqs6yCaDyJuJdsESPq4-uwP4YJIZZyz6U843ECQydjMQgSO4nkuzNbQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/13/6">
                        <div class="volume trailing-comma">Volume 13</div>
                        <div class="issue">Issue 6</div>

            </a>

        </div>
        <div class="ii-pub-date">
2006        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="6973909">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6973909">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6973912">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6973912">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6973922">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6973922">2. Materials and Methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6973969">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6973969">3. Results and Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6974005">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6974005">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6974009">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6974009">REFERENCES</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="empty arrow">&nbsp;</li>
            <li class="next arrow">
            <a href="/dnaresearch/article/13/6/255/464686">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Exploration and Grading of Possible Genes from 183 Bacterial Strains by a Common Protocol to Identification of New Genes: Gene Trek in Prokaryote Space (GTPS)
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                            <div class="meta-authors--limited">
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Takehide Kosuge</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><sup>1</sup><div class="insititution">Center for Information Biology, DNA Data Bank of Japan, National Institute of Genetics, The Graduate University for Advanced Studies (SOKENDAI)</div><div class="addr-line">Mishima, Shizuoka 411-8540, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Takehide+Kosuge">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kosuge T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kosuge T&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Takehide Kosuge</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Takashi Abe</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><sup>1</sup><div class="insititution">Center for Information Biology, DNA Data Bank of Japan, National Institute of Genetics, The Graduate University for Advanced Studies (SOKENDAI)</div><div class="addr-line">Mishima, Shizuoka 411-8540, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Takashi+Abe">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Abe T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Abe T&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Takashi Abe</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Toshihisa Okido</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><sup>1</sup><div class="insititution">Center for Information Biology, DNA Data Bank of Japan, National Institute of Genetics, The Graduate University for Advanced Studies (SOKENDAI)</div><div class="addr-line">Mishima, Shizuoka 411-8540, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Toshihisa+Okido">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Okido T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Okido T&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Toshihisa Okido</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Naoto Tanaka</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><sup>1</sup><div class="insititution">Center for Information Biology, DNA Data Bank of Japan, National Institute of Genetics, The Graduate University for Advanced Studies (SOKENDAI)</div><div class="addr-line">Mishima, Shizuoka 411-8540, Japan</div></div><div class="aff"><div class="label -label">2</div><sup>2</sup><div class="insititution">Institute for Bioinformatics Research and Development (BIRD), Japan Science and Technology Corporation (JST)</div><div class="addr-line">Chiyoda-ku, Tokyo 102-8666, Japan</div></div><div class="aff"><div class="label -label">3</div><sup>3</sup><div class="insititution">Faculty of Pharmaceutical Science, Tokyo University of Science, Noda</div><div class="addr-line">Chiba 278-8510, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Naoto+Tanaka">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tanaka N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tanaka N&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Naoto Tanaka</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Masaki Hirahata</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><sup>1</sup><div class="insititution">Center for Information Biology, DNA Data Bank of Japan, National Institute of Genetics, The Graduate University for Advanced Studies (SOKENDAI)</div><div class="addr-line">Mishima, Shizuoka 411-8540, Japan</div></div><div class="aff"><div class="label -label">2</div><sup>2</sup><div class="insititution">Institute for Bioinformatics Research and Development (BIRD), Japan Science and Technology Corporation (JST)</div><div class="addr-line">Chiyoda-ku, Tokyo 102-8666, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Masaki+Hirahata">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hirahata M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hirahata M&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Masaki Hirahata</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yutaka Maruyama</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><sup>1</sup><div class="insititution">Center for Information Biology, DNA Data Bank of Japan, National Institute of Genetics, The Graduate University for Advanced Studies (SOKENDAI)</div><div class="addr-line">Mishima, Shizuoka 411-8540, Japan</div></div><div class="aff"><div class="label -label">2</div><sup>2</sup><div class="insititution">Institute for Bioinformatics Research and Development (BIRD), Japan Science and Technology Corporation (JST)</div><div class="addr-line">Chiyoda-ku, Tokyo 102-8666, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yutaka+Maruyama">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Maruyama Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Maruyama Y&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Yutaka Maruyama</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jun Mashima</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><sup>1</sup><div class="insititution">Center for Information Biology, DNA Data Bank of Japan, National Institute of Genetics, The Graduate University for Advanced Studies (SOKENDAI)</div><div class="addr-line">Mishima, Shizuoka 411-8540, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jun+Mashima">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Mashima J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Mashima J&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Jun Mashima</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Aki Tomiki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><sup>1</sup><div class="insititution">Center for Information Biology, DNA Data Bank of Japan, National Institute of Genetics, The Graduate University for Advanced Studies (SOKENDAI)</div><div class="addr-line">Mishima, Shizuoka 411-8540, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Aki+Tomiki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tomiki A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tomiki A&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Aki Tomiki</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Motoyoshi Kurokawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><sup>4</sup><div class="insititution">Advanced Center for Computing and Communication, RIKEN, Wako</div><div class="addr-line">Saitama 351-0198, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Motoyoshi+Kurokawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kurokawa M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kurokawa M&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Motoyoshi Kurokawa</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ryutaro Himeno</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><sup>4</sup><div class="insititution">Advanced Center for Computing and Communication, RIKEN, Wako</div><div class="addr-line">Saitama 351-0198, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ryutaro+Himeno">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Himeno R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Himeno R&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Ryutaro Himeno</a>
                                    </span>
                                <a class="meta-authors--etal is-active" id="show-meta-authors" aria-controls="remaining-authors" href="#">
                                    <span>...</span> Show more</a>
                                <div id="remaining-authors" class="meta-authors--remaining" aria-hidden="true" aria-labeledby="show-meta-authors">
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Satoshi Fukuchi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><sup>1</sup><div class="insititution">Center for Information Biology, DNA Data Bank of Japan, National Institute of Genetics, The Graduate University for Advanced Studies (SOKENDAI)</div><div class="addr-line">Mishima, Shizuoka 411-8540, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Satoshi+Fukuchi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Fukuchi S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Fukuchi S&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Satoshi Fukuchi</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Satoru Miyazaki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><sup>1</sup><div class="insititution">Center for Information Biology, DNA Data Bank of Japan, National Institute of Genetics, The Graduate University for Advanced Studies (SOKENDAI)</div><div class="addr-line">Mishima, Shizuoka 411-8540, Japan</div></div><div class="aff"><div class="label -label">3</div><sup>3</sup><div class="insititution">Faculty of Pharmaceutical Science, Tokyo University of Science, Noda</div><div class="addr-line">Chiba 278-8510, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Satoru+Miyazaki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Miyazaki S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Miyazaki S&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Satoru Miyazaki</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Takashi Gojobori</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><sup>1</sup><div class="insititution">Center for Information Biology, DNA Data Bank of Japan, National Institute of Genetics, The Graduate University for Advanced Studies (SOKENDAI)</div><div class="addr-line">Mishima, Shizuoka 411-8540, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Takashi+Gojobori">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Gojobori T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Gojobori T&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Takashi Gojobori</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yoshio Tateno</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><sup>1</sup><div class="insititution">Center for Information Biology, DNA Data Bank of Japan, National Institute of Genetics, The Graduate University for Advanced Studies (SOKENDAI)</div><div class="addr-line">Mishima, Shizuoka 411-8540, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yoshio+Tateno">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tateno Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tateno Y&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Yoshio Tateno</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hideaki Sugawara</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><sup>1</sup><div class="insititution">Center for Information Biology, DNA Data Bank of Japan, National Institute of Genetics, The Graduate University for Advanced Studies (SOKENDAI)</div><div class="addr-line">Mishima, Shizuoka 411-8540, Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <sup>*</sup>To whom correspondence should be addressed. Tel. +81-55-981-6895. Fax +81-55-981-6896, E-mail: <a href="mailto:hsugawar@genes.nig.ac.jp" target="_blank">hsugawar@genes.nig.ac.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hideaki+Sugawara">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sugawara H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sugawara H&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Hideaki Sugawara</a>
                                        </span>
                                </div>
                            </div>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Communicated by Dr Katsumi Isono</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 13, Issue 6, 1 January 2006, Pages 245–254, <a href='https://doi.org/10.1093/dnares/dsl014'>https://doi.org/10.1093/dnares/dsl014</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">01 January 2006</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">29 June 2006</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Revision Received:</div>
                                <div class="wi-date">09 November 2006</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="464398" href="/dnaresearch/article-pdf/13/6/245/1259659/dsl014.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Takehide Kosuge, Takashi Abe, Toshihisa Okido, Naoto Tanaka, Masaki Hirahata, Yutaka Maruyama, Jun Mashima, Aki Tomiki, Motoyoshi Kurokawa, Ryutaro Himeno, Satoshi Fukuchi, Satoru Miyazaki, Takashi Gojobori, Yoshio Tateno, Hideaki Sugawara;  Exploration and Grading of Possible Genes from 183 Bacterial Strains by a Common Protocol to Identification of New Genes: Gene Trek in Prokaryote Space (GTPS), <em>DNA Research</em>, Volume 13, Issue 6, 1 January 2006, Pages 245–254, <a href='https://doi.org/10.1093/dnares/dsl014'>https://doi.org/10.1093/dnares/dsl014</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/464398?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/464398?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/464398?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/464398?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/464398?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/464398?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/464398?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Exploration%20and%20Grading%20of%20Possible%20Genes%20from%20183%20Bacterial%20Strains%20by%20a%20Common%20Protocol%20to%20Identification%20of%20New%20Genes%3A%20Gene%20Trek%20in%20Prokaryote%20Space%20(GTPS)&amp;publicationDate=2006-01-01&amp;volumeNum=13&amp;issueNum=6&amp;author=Kosuge%2C%20Takehide%3B%20Abe%2C%20Takashi&amp;startPage=245&amp;endPage=254&amp;contentId=10.1093%2Fdnares%2Fdsl014&amp;oa=&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f13%2f6%2f245%2f464398" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=6973909 id="6973909" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>A large number of complete microorganism genomes has been sequenced and submitted to the public database and then incorporated into our complete genome database, Genome Information Broker (GIB, <a class="link link-uri" href="http://gib.genes.nig.ac.jp/" target="_blank"></a>). However, when comparative genomics is carried out, researchers must be aware that there are protein-coding genes not confirmed by homology or motif search and that reliable protein-coding genes are missing. Therefore, we developed a protocol (Gene Trek in Prokaryote Space, GTPS) for finding possible protein-coding genes in bacterial genomes. GTPS assigns a degree of reliability to predicted protein-coding genes. We first systematically applied the protocol to the complete genomes of all 123 bacterial species and strains that were publicly available as of July 2003, and then to those of 183 species and strains available as of September 2004. We found a number of incorrect genes and several new ones in the genome data in question. We also found a way to estimate the total number of orthologous genes in the bacterial world.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;comparative genome&quot;">comparative genome</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;gene prediction&quot;">gene prediction</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="annotation">annotation</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;gene grading&quot;">gene grading</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;bacterial genome&quot;">bacterial genome</a></div>                            <h2 scrollto-destination=6973912 id="6973912" class="section-title" >1. Introduction</h2>
<p>It is imperative to study and understand bacteria because they are integrated with, and mutually dependent on, all other living systems and the ecosystem. Although there are many approaches to scientific understanding of bacteria; it may be natural to start with genes. After all, it is the bacterial genome that makes a microorganism our friend, foe or otherwise.</p><p>Since complete sequencing of the <em>Haemophilus influenzae</em> genome,<a reveal-id="b1" data-open="b1" class="link link-ref link-reveal xref-bibr">1</a> the whole genome sequencing method<a reveal-id="b2" data-open="b2" class="link link-ref link-reveal xref-bibr">2</a> has been applied extensively to bacterial species, and complete genome sequences have been determined one after another. Currently, whole genome sequences are available for &gt;300 bacterial strains from the Genome Information Broker (GIB, <a class="link link-uri" href="http://gib.genes.nig.ac.jp/" target="_blank"></a>)<a reveal-id="b3" data-open="b3" class="link link-ref link-reveal xref-bibr">3</a> website of the DNA Data Bank of Japan (DDBJ). The tens of thousands of genes stored in GIB reflect the fact that bacteria are remarkably diverse, as well as, flexible so that they may survive in various environmental conditions. Furthermore, innumerable bacterial species remain unidentified.<a reveal-id="b4  b6" data-open="b4  b6" class="link link-ref link-reveal xref-bibr">4–6</a> Nevertheless, even though bacterial species are countless, there should be a limit to the number of their genes. We thus ponder the question: how many genes are there in the bacterial world?</p><p>As the reports of complete genome sequences have accumulated, another question has become more serious and urgent: how real are the genes that reportedly exist in the genome of a bacterial species? The most thoroughly studied species is <em>Escherichia coli</em> K-12, which is reported to possess 4302 protein-coding genes in its genome.<a reveal-id="b7" data-open="b7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="b8" data-open="b8" class="link link-ref link-reveal xref-bibr">8</a> However, no one has yet confirmed that every one of these purported genes functions and produces the corresponding RNA and protein. This extends to other species as well. There may be several reasons for this frustrating reality, one being that researchers find it difficult to examine all genes experimentally and thus resort to computerized homology searches.<a reveal-id="b9  b13" data-open="b9  b13" class="link link-ref link-reveal xref-bibr">9–13</a> Homology searching is an unquestionably powerful method for exploring gene functions,<a reveal-id="b14" data-open="b14" class="link link-ref link-reveal xref-bibr">14</a> but it is confined to the <em>in silico</em> domain where the parameters particular to a software tool determine the outcome. It is generally observed that different sets of parameters lead to different results for the same data input.</p><p>In the case of software tools for finding genes, the situation may be even worse because we have to deal not only with software parameters but also with algorithms. We can thus argue that a significant percentage of the computationally deduced genes of the bacterial species displayed by GIB and other databases will not be reproduced exactly by using one gene finder tool with a certain set of parameters. This includes doubly or triply inferred genes from the literature. If we allow this practice to continue, we will be forced into dealing with many invalid genes in the near future. We are presented with two choices. The first is to begin experimentally proving or disproving every gene in question, as Roberts proposed.<a reveal-id="b15" data-open="b15" class="link link-ref link-reveal xref-bibr">15</a> The second, more feasible choice is to apply a closely examined tool to all genome sequence data available from GIB and elsewhere and to deduce genes all over again. With this approach, we can expect to reduce inconsistency and invalidity and to reproduce more reliable data. This will also make it possible to classify the deduced genes in the order of validity on the basis of defined criteria. Such classification will certainly assist us in experimentally examining suspicious genes.</p><p>Herein, we introduce steps to be taken in the second approach to make comparative genomics of microbes<a reveal-id="b11" data-open="b11" class="link link-ref link-reveal xref-bibr">11</a>,<a reveal-id="b16  b19" data-open="b16  b19" class="link link-ref link-reveal xref-bibr">16–19</a> efficient and meaningful. We have developed an annotation protocol (Gene Trek in Prokaryote Space, GTPS) for finding possible protein-coding genes in bacterial genomes published by 2003, and we have constructed a database of possible protein-coding genes with the degrees of reliability. After 2003, the entire GTPS procedure was refined and for the most part automated. We then applied the new GTPS to bacterial genomes published as of 2004, and we found new protein-coding genes that had not been reported in the published genome data. The GTPS analysis of 303 bacterial species and strains published as of February 2006 is now in progress for GTPS ver 2005. In this report, we show that GTPS can systematically evaluate published protein-coding genes and find new genes. We also suggest a way to determine the total number of orthologous genes in the bacterial world.</p>                            <h2 scrollto-destination=6973922 id="6973922" class="section-title" >2. Materials and Methods</h2>
                            <h3 scrollto-destination=6973924 id="6973924" class="section-title" >2.1. Genome sequences used in this study</h3>
<p>We have provided for GIB to continuously receive and install complete bacterial genome sequences released by the International Nucleotide Sequence Database Collaboration (INSDC, i.e. DDBJ/EMBL/GenBank, <a class="link link-uri" href="http://www.insdc.org/" target="_blank"></a>). We examined complete bacterial genome data stored in GIB for 123 species and strains released by July 2003 (ver. 2003). Thereafter, we added and updated data for the 183 species and strains that were released by September 2004 (ver. 2004). Analysis of the 303 species and strains (ver. 2005) is now in progress. An open reading frame (ORF) was defined in the protocol as a possible protein-coding region.</p>                            <h3 scrollto-destination=6973927 id="6973927" class="section-title" >2.2. Steps in the exploration and evaluation of ORFs</h3>
<p>Analysis involves three main steps (<a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1</a>): </p><ul class="roman-lower"><li><p>Search for and masking of RNA coding genes in the genome sequence.</p></li><li><p>Prediction of boundaries for ORFs.</p></li><li><p>Evaluation of the predicted ORFs based on the results of homology and motif searches.</p></li></ul>                        <a id="6973931" scrollto-destination="6973931"></a>
<div id="fig1" data-id="fig1" class="fig fig-section"><div class="label fig-label"><strong>Figure 1</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/6/10.1093/dnares/dsl014/2/m_dsl014f1.jpeg?Expires=2147483647&Signature=wX7AYgQCSGebzs4FTCpA9Avmck2hRVRHNkcgqS6u3WO5GfF7Em1QsIsQ~uM70LDNmTMu2z4Adx62yCVkRQYsRIcyisWoTJ6-Ezwy-acgg4qealfMfONWpUMyPJQqN59-ckTLTnXBS1IeE6t5uRDuyDj2mwlqYTx-AjDVl-8hkxuqGb1TPTR~RlbXhL35x3u78H4uJj6u~hz0ShePkHLvEZF7xtSqJxFdSfK-1TdpnxACxsry6aFQgVyGC7ckfMXVqYS4oJ1yY-fq0aa53eC7OUnA2Djc5VmqjxUHHsJ~Mw9alyswN4XUfTexcB1Jce56Xoa7o962AVNFOFqnlulcVg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The workflow of GTPS. Steps in the workflow are described in the Section Materials and Methods." path-from-xml="dsl014f1.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6973931/dsl014f1.jpeg" path-from-xml="dsl014f1.jpeg" target="_blank">View large</a><a section="6973931" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/6/10.1093/dnares/dsl014/2/dsl014f1.jpeg?Expires=2147483647&Signature=oPvJm~eJxvO8xLlyZBhZewtWc~Uw8kNqEO9XDz8jbiVHbZukYLvMDs5KaTceBF4-i-ME671lMh~KG~wn3SusftQvT2ssIUcGYzlNeT3iXke0ZSWf77QI~Ftytx7vpDntAZ~MFhc27D4cAR6kYc56Cj29k5M-BDORXOVkEkGta~TdECRbZTFV0MTeLnb9NLinaFhGW0pPrFUXY3lafFhu0S3lO~8ORZi1fugndrqLqKg3XkfFkJhHh3~KvIeUIuD9XWvaqivCNNjk861hk9SQGJ6BkA0FhGfKzGvKHH13Ay2BVcjow4N9dqBEymCZa5oLrx40KlBTkiT-xowE2eEpgg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6973931&ar=464398&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl014f1.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The workflow of GTPS. Steps in the workflow are described in the Section Materials and Methods.</p></div></div><div content-id="fig1" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 1</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/6/10.1093/dnares/dsl014/2/m_dsl014f1.jpeg?Expires=2147483647&Signature=wX7AYgQCSGebzs4FTCpA9Avmck2hRVRHNkcgqS6u3WO5GfF7Em1QsIsQ~uM70LDNmTMu2z4Adx62yCVkRQYsRIcyisWoTJ6-Ezwy-acgg4qealfMfONWpUMyPJQqN59-ckTLTnXBS1IeE6t5uRDuyDj2mwlqYTx-AjDVl-8hkxuqGb1TPTR~RlbXhL35x3u78H4uJj6u~hz0ShePkHLvEZF7xtSqJxFdSfK-1TdpnxACxsry6aFQgVyGC7ckfMXVqYS4oJ1yY-fq0aa53eC7OUnA2Djc5VmqjxUHHsJ~Mw9alyswN4XUfTexcB1Jce56Xoa7o962AVNFOFqnlulcVg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The workflow of GTPS. Steps in the workflow are described in the Section Materials and Methods." path-from-xml="dsl014f1.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6973931/dsl014f1.jpeg" path-from-xml="dsl014f1.jpeg" target="_blank">View large</a><a section="6973931" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/6/10.1093/dnares/dsl014/2/dsl014f1.jpeg?Expires=2147483647&Signature=oPvJm~eJxvO8xLlyZBhZewtWc~Uw8kNqEO9XDz8jbiVHbZukYLvMDs5KaTceBF4-i-ME671lMh~KG~wn3SusftQvT2ssIUcGYzlNeT3iXke0ZSWf77QI~Ftytx7vpDntAZ~MFhc27D4cAR6kYc56Cj29k5M-BDORXOVkEkGta~TdECRbZTFV0MTeLnb9NLinaFhGW0pPrFUXY3lafFhu0S3lO~8ORZi1fugndrqLqKg3XkfFkJhHh3~KvIeUIuD9XWvaqivCNNjk861hk9SQGJ6BkA0FhGfKzGvKHH13Ay2BVcjow4N9dqBEymCZa5oLrx40KlBTkiT-xowE2eEpgg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6973931&ar=464398&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl014f1.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The workflow of GTPS. Steps in the workflow are described in the Section Materials and Methods.</p></div></div><p>These steps are described in detail in the following sections.</p>                            <h4 scrollto-destination=6973935 id="6973935" class="section-title" >2.2.1. Search for and masking of RNA coding genes in the genome sequence</h4>
<p>RNA coding genes (ribosomal RNA [rRNA], transfer RNA [tRNA] and other non-coding RNA [ncRNA]) were first searched for and masked. For rRNA, the regions annotated as rRNA in the microbial genome as released by GIB were searched for and masked. For tRNA, we used the tRNAscan-SE (ver. 1.23)<a reveal-id="b20" data-open="b20" class="link link-ref link-reveal xref-bibr">20</a> for prediction of the tRNA region. This program has been commonly used in previously reported studies. ‘−B’ option and ‘−A’ option were used for bacterial and archaeal genomes, respectively. The other parameters were set to the default values. A region thus detected by tRNAscan-SE as a non-pseudo tRNA was determined to be a tRNA gene. For ncRNAs, the regions that perfectly matched the sequences registered on Rfam,<a reveal-id="b21" data-open="b21" class="link link-ref link-reveal xref-bibr">21</a> a well-known ncRNA database, were searched for and masked in the genome.</p>                            <h4 scrollto-destination=6973939 id="6973939" class="section-title" >2.2.2. Prediction of boundaries for ORFs</h4>
<p>From various ORF prediction programs applied to microbial genomes (<a reveal-id="fig2" data-open="fig2" class="link link-reveal link-table xref-fig">Fig. 2</a>), we chose GLIMMER 2.0,<a reveal-id="b22" data-open="b22" class="link link-ref link-reveal xref-bibr">22</a>,<a reveal-id="b23" data-open="b23" class="link link-ref link-reveal xref-bibr">23</a> the latest version as of September 2004. Because GLIMMER 2.0 can produce a learning dataset for ORF prediction by using the input genomic sequence in question only, it makes a highly accurate prediction of ORFs.<a reveal-id="b24" data-open="b24" class="link link-ref link-reveal xref-bibr">24</a> The important parameter in using GLIMMER 2.0 is the minimum length of the ORF region. In predicting ORFs for the microbial genomes that have been reported to date, each annotation team sets the minimum length differently (Supplementary <a reveal-id="tbl1" data-open="tbl1" class="link link-reveal link-table xref-fig">Table 1</a>). Consequently, the predicted results are greatly affected by the differences in the minimum length that is set. This explains in part why there are inconsistencies in the annotation information. To determine the optimum setting for the minimum ORF length, GLIMMER 2.0 was applied to 183 strains by using four minimum ORF lengths, 15AA, 30AA, 60AA and 90AA (Supplementary <a reveal-id="tbl2" data-open="tbl2" class="link link-reveal link-table xref-fig">Table 2</a>). In comparison to the ORFs in the INSDC entries, the predicted numbers were too large at the 15AA and 30AA settings and too small at the 90AA setting. The highest rate of concordance with the existing annotations is obtained by setting the minimum length at 60AA. (Supplementary Tables 1 and 2 are available at <a class="link link-uri" href="http://www.dnares.oxfordjournals.org" target="_blank"></a>)</p>                        <a id="6973941" scrollto-destination="6973941"></a>
<div id="fig2" data-id="fig2" class="fig fig-section"><div class="label fig-label"><strong>Figure 2</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/6/10.1093/dnares/dsl014/2/m_dsl014f2.jpeg?Expires=2147483647&Signature=but9WWNrkxicno-eK8EngBRVs2-Ko1RJOwqmMyArgB3aWD3Kx0SCK1U60cP8c9neNKoQlahd~g~3H46WUfwR7bsbqcVOEE-~vwQWf--DbWbqG~2GUeTcY0J1Ak-1IY3nQ9r~RPv1kk0BuUVWH4sXbW8akmR6EZcuNgOEl7Kwecjs25iZV96l2XU3H7UvZYRWo0-snIIN1VO0Bi8Nakvov0xzf9OXxcnjDVyV9t7Pl4-r2WG0ljpvaIN1facmEq-2MHZm6GNTfCZsVJdaPVVzOLEqch0quE5KfCguEtFMnWnQxPsEfKdCAQAjLPiPetcp5eF~xJjB1dyd9A1TGiICcw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Prediction programs. A variety of programs used for the prediction of ORFs among the 183 complete genomes that were released by GIB as of September 2004, were investigated." path-from-xml="dsl014f2.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6973941/dsl014f2.jpeg" path-from-xml="dsl014f2.jpeg" target="_blank">View large</a><a section="6973941" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/6/10.1093/dnares/dsl014/2/dsl014f2.jpeg?Expires=2147483647&Signature=r1s4rhnZrTGYLqTeAtMGDbhTeXxbam47zx7IQFiA59u2wxV0Ne9xZ01cYNi0tVuXPVrPaN4T-i~MTv8CBMehLCypxpvvxcXnQeweOjF5j-Xf-V-ppl6k8-8GX6ezfQUnd-eMIH2loEQd7rJ3StrIsddDoJGXCB3fuvUg1MBTHGRJUM8t-smC~DVLa5eqq1FfILuFSbjlMC8EwU6iowcmlGoFrs9rMUsf6SbFQA1-ZK11whZ4NZ2Grx0jjodb5zH1iSNFG4-47zMtGOE8y5XnBTNpPJuCdZ7xi-ulp4MBs7v6sgxuI~vnzKAh8My2ZUvqDsg9Ixm9I2IPx2YHF2JB5w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6973941&ar=464398&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl014f2.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Prediction programs. A variety of programs used for the prediction of ORFs among the 183 complete genomes that were released by GIB as of September 2004, were investigated.</p></div></div><div content-id="fig2" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 2</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/6/10.1093/dnares/dsl014/2/m_dsl014f2.jpeg?Expires=2147483647&Signature=but9WWNrkxicno-eK8EngBRVs2-Ko1RJOwqmMyArgB3aWD3Kx0SCK1U60cP8c9neNKoQlahd~g~3H46WUfwR7bsbqcVOEE-~vwQWf--DbWbqG~2GUeTcY0J1Ak-1IY3nQ9r~RPv1kk0BuUVWH4sXbW8akmR6EZcuNgOEl7Kwecjs25iZV96l2XU3H7UvZYRWo0-snIIN1VO0Bi8Nakvov0xzf9OXxcnjDVyV9t7Pl4-r2WG0ljpvaIN1facmEq-2MHZm6GNTfCZsVJdaPVVzOLEqch0quE5KfCguEtFMnWnQxPsEfKdCAQAjLPiPetcp5eF~xJjB1dyd9A1TGiICcw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Prediction programs. A variety of programs used for the prediction of ORFs among the 183 complete genomes that were released by GIB as of September 2004, were investigated." path-from-xml="dsl014f2.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6973941/dsl014f2.jpeg" path-from-xml="dsl014f2.jpeg" target="_blank">View large</a><a section="6973941" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/6/10.1093/dnares/dsl014/2/dsl014f2.jpeg?Expires=2147483647&Signature=r1s4rhnZrTGYLqTeAtMGDbhTeXxbam47zx7IQFiA59u2wxV0Ne9xZ01cYNi0tVuXPVrPaN4T-i~MTv8CBMehLCypxpvvxcXnQeweOjF5j-Xf-V-ppl6k8-8GX6ezfQUnd-eMIH2loEQd7rJ3StrIsddDoJGXCB3fuvUg1MBTHGRJUM8t-smC~DVLa5eqq1FfILuFSbjlMC8EwU6iowcmlGoFrs9rMUsf6SbFQA1-ZK11whZ4NZ2Grx0jjodb5zH1iSNFG4-47zMtGOE8y5XnBTNpPJuCdZ7xi-ulp4MBs7v6sgxuI~vnzKAh8My2ZUvqDsg9Ixm9I2IPx2YHF2JB5w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6973941&ar=464398&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl014f2.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Prediction programs. A variety of programs used for the prediction of ORFs among the 183 complete genomes that were released by GIB as of September 2004, were investigated.</p></div></div>                        <a id="6973945" scrollto-destination="6973945"></a>
<div content-id="tbl1" class="table-modal"><div class="table-wrap"><div id="tbl1" class="table-wrap-title" data-id="tbl1"><div class="label -label"><strong>Table 1</strong></div><div class="caption"><p>Grades of ORFs predicted by GTPS</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Grades </th><th>Ver. 2003 </th><th>Ver. 2004 </th></tr></thead><tbody><tr><td>AAAA–A </td><td>283 247 </td><td>431 672 </td></tr><tr><td>BBBB–B </td><td>7208 </td><td>10 250 </td></tr><tr><td>C </td><td>4680 </td><td>7511 </td></tr><tr><td>D </td><td>79 779 </td><td>1 07 382 </td></tr><tr><td>E </td><td>6788 </td><td>10 225 </td></tr><tr><td>X </td><td>466 681 </td><td>687 110 </td></tr><tr><td>Total (AAAA–X) </td><td>848 383 </td><td>1 254 150 </td></tr><tr><td>Potential genes (AAAA1–D3) </td><td>370 876 </td><td>551 246 </td></tr><tr><td>INSDC </td><td>362 828 </td><td>537 312 </td></tr></tbody></table></div><div content-id="tbl1" class="table-modal"><table><thead name="thead" align=""><tr><th>Grades </th><th>Ver. 2003 </th><th>Ver. 2004 </th></tr></thead><tbody><tr><td>AAAA–A </td><td>283 247 </td><td>431 672 </td></tr><tr><td>BBBB–B </td><td>7208 </td><td>10 250 </td></tr><tr><td>C </td><td>4680 </td><td>7511 </td></tr><tr><td>D </td><td>79 779 </td><td>1 07 382 </td></tr><tr><td>E </td><td>6788 </td><td>10 225 </td></tr><tr><td>X </td><td>466 681 </td><td>687 110 </td></tr><tr><td>Total (AAAA–X) </td><td>848 383 </td><td>1 254 150 </td></tr><tr><td>Potential genes (AAAA1–D3) </td><td>370 876 </td><td>551 246 </td></tr><tr><td>INSDC </td><td>362 828 </td><td>537 312 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6973945" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="tbl1" class="table-wrap-title" data-id="tbl1"><div class="label -label"><strong>Table 1</strong></div><div class="caption"><p>Grades of ORFs predicted by GTPS</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Grades </th><th>Ver. 2003 </th><th>Ver. 2004 </th></tr></thead><tbody><tr><td>AAAA–A </td><td>283 247 </td><td>431 672 </td></tr><tr><td>BBBB–B </td><td>7208 </td><td>10 250 </td></tr><tr><td>C </td><td>4680 </td><td>7511 </td></tr><tr><td>D </td><td>79 779 </td><td>1 07 382 </td></tr><tr><td>E </td><td>6788 </td><td>10 225 </td></tr><tr><td>X </td><td>466 681 </td><td>687 110 </td></tr><tr><td>Total (AAAA–X) </td><td>848 383 </td><td>1 254 150 </td></tr><tr><td>Potential genes (AAAA1–D3) </td><td>370 876 </td><td>551 246 </td></tr><tr><td>INSDC </td><td>362 828 </td><td>537 312 </td></tr></tbody></table></div><div content-id="tbl1" class="table-modal"><table><thead name="thead" align=""><tr><th>Grades </th><th>Ver. 2003 </th><th>Ver. 2004 </th></tr></thead><tbody><tr><td>AAAA–A </td><td>283 247 </td><td>431 672 </td></tr><tr><td>BBBB–B </td><td>7208 </td><td>10 250 </td></tr><tr><td>C </td><td>4680 </td><td>7511 </td></tr><tr><td>D </td><td>79 779 </td><td>1 07 382 </td></tr><tr><td>E </td><td>6788 </td><td>10 225 </td></tr><tr><td>X </td><td>466 681 </td><td>687 110 </td></tr><tr><td>Total (AAAA–X) </td><td>848 383 </td><td>1 254 150 </td></tr><tr><td>Potential genes (AAAA1–D3) </td><td>370 876 </td><td>551 246 </td></tr><tr><td>INSDC </td><td>362 828 </td><td>537 312 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6973945" target="_blank">View Large</a></div></div>                        <a id="6973947" scrollto-destination="6973947"></a>
<div content-id="tbl2" class="table-modal"><div class="table-wrap"><div id="tbl2" class="table-wrap-title" data-id="tbl2"><div class="label -label"><strong>Table 2</strong></div><div class="caption"><p>GTPS and INSDC annotations</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th colspan="2">ver. 2003 </th><th colspan="2">ver. 2004 </th></tr><tr><th> </th><th colspan="2"><hr /> </th><th colspan="2"><hr /> </th></tr><tr><th> </th><th>Number </th><th>(%) </th><th>Number </th><th>(%) </th></tr></thead><tbody><tr><td>Identical </td><td>261 720 </td><td>70.6 </td><td>390 557 </td><td>70.8 </td></tr><tr><td>3′ matched </td><td>92 206 </td><td>24.9 </td><td>133 146 </td><td>24.2 </td></tr><tr><td>New ORFs (only predicted in this study) </td><td>14 954 </td><td>4.0 </td><td>23 935 </td><td>4.3 </td></tr><tr><td>Not predicted by Glimmer </td><td>1996 </td><td>0.5 </td><td>3608 </td><td>0.7 </td></tr><tr><td>Total (potential genes) </td><td>370 876 </td><td> </td><td>551 246 </td><td> </td></tr></tbody></table></div><div content-id="tbl2" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th colspan="2">ver. 2003 </th><th colspan="2">ver. 2004 </th></tr><tr><th> </th><th colspan="2"><hr /> </th><th colspan="2"><hr /> </th></tr><tr><th> </th><th>Number </th><th>(%) </th><th>Number </th><th>(%) </th></tr></thead><tbody><tr><td>Identical </td><td>261 720 </td><td>70.6 </td><td>390 557 </td><td>70.8 </td></tr><tr><td>3′ matched </td><td>92 206 </td><td>24.9 </td><td>133 146 </td><td>24.2 </td></tr><tr><td>New ORFs (only predicted in this study) </td><td>14 954 </td><td>4.0 </td><td>23 935 </td><td>4.3 </td></tr><tr><td>Not predicted by Glimmer </td><td>1996 </td><td>0.5 </td><td>3608 </td><td>0.7 </td></tr><tr><td>Total (potential genes) </td><td>370 876 </td><td> </td><td>551 246 </td><td> </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6973947" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="tbl2" class="table-wrap-title" data-id="tbl2"><div class="label -label"><strong>Table 2</strong></div><div class="caption"><p>GTPS and INSDC annotations</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th colspan="2">ver. 2003 </th><th colspan="2">ver. 2004 </th></tr><tr><th> </th><th colspan="2"><hr /> </th><th colspan="2"><hr /> </th></tr><tr><th> </th><th>Number </th><th>(%) </th><th>Number </th><th>(%) </th></tr></thead><tbody><tr><td>Identical </td><td>261 720 </td><td>70.6 </td><td>390 557 </td><td>70.8 </td></tr><tr><td>3′ matched </td><td>92 206 </td><td>24.9 </td><td>133 146 </td><td>24.2 </td></tr><tr><td>New ORFs (only predicted in this study) </td><td>14 954 </td><td>4.0 </td><td>23 935 </td><td>4.3 </td></tr><tr><td>Not predicted by Glimmer </td><td>1996 </td><td>0.5 </td><td>3608 </td><td>0.7 </td></tr><tr><td>Total (potential genes) </td><td>370 876 </td><td> </td><td>551 246 </td><td> </td></tr></tbody></table></div><div content-id="tbl2" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th colspan="2">ver. 2003 </th><th colspan="2">ver. 2004 </th></tr><tr><th> </th><th colspan="2"><hr /> </th><th colspan="2"><hr /> </th></tr><tr><th> </th><th>Number </th><th>(%) </th><th>Number </th><th>(%) </th></tr></thead><tbody><tr><td>Identical </td><td>261 720 </td><td>70.6 </td><td>390 557 </td><td>70.8 </td></tr><tr><td>3′ matched </td><td>92 206 </td><td>24.9 </td><td>133 146 </td><td>24.2 </td></tr><tr><td>New ORFs (only predicted in this study) </td><td>14 954 </td><td>4.0 </td><td>23 935 </td><td>4.3 </td></tr><tr><td>Not predicted by Glimmer </td><td>1996 </td><td>0.5 </td><td>3608 </td><td>0.7 </td></tr><tr><td>Total (potential genes) </td><td>370 876 </td><td> </td><td>551 246 </td><td> </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6973947" target="_blank">View Large</a></div></div><p>However, we noticed that the minimum length of the protein sequences in the UniProtKB/Swiss-Prot<a reveal-id="b25" data-open="b25" class="link link-ref link-reveal xref-bibr">25</a> database was 17AA (O52748, Heterocyst inhibition signaling peptide). Therefore, to improve accuracy of the ORF prediction, we set the minimum length at 60AA and at 15AA. When two ORFs predicted with each of the two minimum lengths were matched at the 3′ ends, the longer ORF was chosen for the following steps.</p><p>To confirm the 5′ end of each ORF in a circular genome, we debugged and significantly modified the source code of RBSfinder.<a reveal-id="b26" data-open="b26" class="link link-ref link-reveal xref-bibr">26</a> We then carefully determined the seed sequence of 5 bp for the revised RBSfinder for each species as follows. First, the 3′ end of the 16S rRNA gene of the species in question was determined by evaluating the alignment of the gene with that of the reference sequence, <em>rrnO</em> gene of <em>Bacillus subtilis</em> 168.<a reveal-id="b27" data-open="b27" class="link link-ref link-reveal xref-bibr">27</a> Second, for the determined region of 15 bp, pentamers were gathered starting from the 3′ end of the region and moving upward by 1 bp until entire region was covered, and a list of the pentamers was prepared. Third, by using this list, a histogram was made for the frequency at which a pentamer matched a pentamer in the region 20 bp upstream from the predicted ORF. Finally, the seed sequence was determined by visual inspection of the histogram. The seed sequence was the listed pentamer that most frequently matched the region (Supplementary <a reveal-id="tbl3" data-open="tbl3" class="link link-reveal link-table xref-fig">Table 3</a>; Supplementary Table 3 is available at <a class="link link-uri" href="http://www.dnares.oxfordjournals.org" target="_blank"></a>). For some genomes, RBSfinder was not executed because appropriate seed sequences could not be found in this study.</p>                        <a id="6973951" scrollto-destination="6973951"></a>
<div content-id="tbl3" class="table-modal"><div class="table-wrap"><div id="tbl3" class="table-wrap-title" data-id="tbl3"><div class="label -label"><strong>Table 3</strong></div><div class="caption"><p>Comparison of <em>E. coli</em> ORFs in INSDC with those in ver. 2003 and ver. 2004</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Ver. 2003 </th><th>ver. 2004 </th></tr></thead><tbody><tr><td>Common with INSDC </td><td>3281 </td><td>3128 </td></tr><tr><td>3′-matched </td><td>721 </td><td>1054 </td></tr><tr><td>New ORFs (only predicted in this study) </td><td>674 </td><td>476 </td></tr></tbody></table></div><div content-id="tbl3" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Ver. 2003 </th><th>ver. 2004 </th></tr></thead><tbody><tr><td>Common with INSDC </td><td>3281 </td><td>3128 </td></tr><tr><td>3′-matched </td><td>721 </td><td>1054 </td></tr><tr><td>New ORFs (only predicted in this study) </td><td>674 </td><td>476 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6973951" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="tbl3" class="table-wrap-title" data-id="tbl3"><div class="label -label"><strong>Table 3</strong></div><div class="caption"><p>Comparison of <em>E. coli</em> ORFs in INSDC with those in ver. 2003 and ver. 2004</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Ver. 2003 </th><th>ver. 2004 </th></tr></thead><tbody><tr><td>Common with INSDC </td><td>3281 </td><td>3128 </td></tr><tr><td>3′-matched </td><td>721 </td><td>1054 </td></tr><tr><td>New ORFs (only predicted in this study) </td><td>674 </td><td>476 </td></tr></tbody></table></div><div content-id="tbl3" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Ver. 2003 </th><th>ver. 2004 </th></tr></thead><tbody><tr><td>Common with INSDC </td><td>3281 </td><td>3128 </td></tr><tr><td>3′-matched </td><td>721 </td><td>1054 </td></tr><tr><td>New ORFs (only predicted in this study) </td><td>674 </td><td>476 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6973951" target="_blank">View Large</a></div></div><p>GLIMMER 3<a reveal-id="b22" data-open="b22" class="link link-ref link-reveal xref-bibr">22</a>,<a reveal-id="b23" data-open="b23" class="link link-ref link-reveal xref-bibr">23</a>,<a reveal-id="b28" data-open="b28" class="link link-ref link-reveal xref-bibr">28</a> has been released after ver. 2004. We now evaluate the parameter setting of GLIMMER 3 for analysis of the entire genomes of 303 bacterial species and strains published as of February 2006.</p>                            <h4 scrollto-destination=6973954 id="6973954" class="section-title" >2.2.3. Evaluation of the predicted ORFs based on results of homology and motif searches</h4>
<p>In this study, protein-coding regions, such as for a leader peptide and intron-containing genes, were taken from INSDC and put into predicted ORFs. When GLIMMER 2.0 could not predict ORFs in the corresponding location in the INSDC data where a protein-coding sequence existed, the coding sequence was regarded one of our predicted ORFs. After predicting all ORFs for all species, we performed a blastp 2.2.6<a reveal-id="b10" data-open="b10" class="link link-ref link-reveal xref-bibr">10</a> search for them against the genes of BCT division in the DDBJ Amino Acid Database (DAD), release 24, for ver. 2003 and DDBJ, release 28, for ver. 2004. The DAD is a database that contains all the translated amino acid sequences from the corresponding DNA sequences in INSDC. In the blastp search, we used the following three conditions as the threshold for the blast hit: I (<em>E</em>-value ≤10<sup>−40</sup>; sequence identity of aligned residues ≥30%; mutual coverage ≥70% between subject and query), II (<em>E</em>-value ≤10<sup>−40</sup>; sequence identity of aligned residues (80%; mutual coverage (80% between subject and query), and III (<em>E</em>-value ignored; sequence identity of aligned residues (90%; mutual coverage (90% between subject and query). For the blast analysis, we used a 16-node PC cluster system (CPU: Dual Xeon 3.2 GHz). The computation time for the analysis was 8 h for the 123 genomes in ver. 2003 and 12.5 h for the 183 genomes in ver. 2004. Next, we performed a motif search using InterProScan 3.2 (InterPro 6.2) for ver. 2003 and 3.3 (InterPro 7.2) for ver. 2004.<a reveal-id="b29" data-open="b29" class="link link-ref link-reveal xref-bibr">29</a> A motif hit was assigned when the region predicted to have InterPro motifs was &gt;30% of the ORF length. In this study we used 64-CPU RIKEN Super Combined Cluster System. The computation time for InterProScan was 420 h for the 123 genomes in ver. 2003 and 624 h for the 183 genomes in ver. 2004.</p><p>Flags were assigned to all of the ORFs to indicate the minimum length in GLIMMER 2.0 prediction, existence of Shine-Dalgarno sequence and results of the homology and motif searches. Details of the flag information are provided as Supplementary Table 4. (Supplementary Table 4 is available at <a class="link link-uri" href="http://www.dnares.oxfordjournals.org" target="_blank"></a>) The ORFs were compared with the corresponding genes in INSDC annotation. When the corresponding gene had a ‘/pseudo’ qualifier and the ORF was included in the gene or overlapped with the gene in-frame, a pseudo flag was assigned to the ORF. In addition, the ORFs were graded on an A–E or X scale to classify them according to the blastp and InterProScan results (<a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3</a>). Grade A included highly reliable ORFs that are well supported by blastp and motif scans. Grades B and C included ORFs with poor homology and motif results and are thus less reliable than Grade A. Grades A and B were each divided into four sub-grades (AAAA, AAA, AA and A; BBBB, BBB, BB and B) with respect to the InterProScan results. Grade D included hypothetical proteins with no motifs. Grades A, B and C were used to indicate reliable genes for the computer simulation of gene divergence in the bacterial world.</p>                        <a id="6973959" scrollto-destination="6973959"></a>
<div id="fig3" data-id="fig3" class="fig fig-section"><div class="label fig-label"><strong>Figure 3</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/6/10.1093/dnares/dsl014/2/m_dsl014f3.jpeg?Expires=2147483647&Signature=VqDpGPbL3TQG4NyPYYMwq9wQ2Arz6wI~6S~T9hPhva2NM0-bqLaRVR-LAKPAJv96LzfK~qfsBlb3cAYJKOb8kvRKISjTAEq~3qguSdVKLFMDCdOO7CcdiHUWln5V6lwwPuGPgTg3GLDPuc4HaTgdWlhcCWYtTAuNwsV4BsYzmayhc97oA0SjVfLHd4Dn8B6HzQrLm~KSEJpokt81LhzHBLY2Yk-wvwT6suZqlKAp37Zz9B6Zw9fzHpKmDStRstb6J7Lri6jRGAeCcQqXNXpg77Ks3cXWyPQfc7uEwrjNXOnY5W-7WTQQE9LlqST-SCG1HjXVmRISapi3QL3KQPMlgQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="ORFs grades. ORFs were graded as follows. Grades A–C: functional information available; Grade D: hypothetical protein; Grade E: hypothetical protein supported by poor coverage in blastp and Grade X: no information available. An unknown protein is a hypothetical protein for which the product and notes are given in Supplementary Table 7. A putative membrane protein is a protein for which the product is given in Supplementary Table 8. An unknown motif is a motif that includes ‘unknown’ in its InterPro description. A significant motif is a known motif. Potential genes were defined as ORFs of Grades AAAA1–D3. (Supplementary Tables 7 and 8 are available at )" path-from-xml="dsl014f3.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6973959/dsl014f3.jpeg" path-from-xml="dsl014f3.jpeg" target="_blank">View large</a><a section="6973959" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/6/10.1093/dnares/dsl014/2/dsl014f3.jpeg?Expires=2147483647&Signature=MB5CXcvYplrmmn5u4jtooZTyXgxkkDvxXSTvZxVhR9vYfpRvlod89HOHH7Q~~tWtLj3GpFkVuKbLIlxP7Xh9OmwaP-wFhYUfTP5VNjb~Gcwmavp6q864Bbgp2rBmf3y~nZprDxJFZPSGkgvmf36cXNIvVKEzTY4CNG1ikXZND0jQgUxEuq7jbwLbqkeKxF5L~HtL-LEhE2EgqZ28bYoA~kM0EtKpox3aFx-Wn~57Ds8trofwRTnsq8gbqB~zGsBpvKzgAbWKLE7GOluGV2h4fiYg~JbBTOeebpf3KiseVZ4Y0DGfGbpalZZ8EqRaDXXH-TPt3zbNlV8y6HYija8Nwg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6973959&ar=464398&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl014f3.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>ORFs grades. ORFs were graded as follows. Grades A–C: functional information available; Grade D: hypothetical protein; Grade E: hypothetical protein supported by poor coverage in blastp and Grade X: no information available. An unknown protein is a hypothetical protein for which the product and notes are given in Supplementary Table 7. A putative membrane protein is a protein for which the product is given in Supplementary Table 8. An unknown motif is a motif that includes ‘unknown’ in its InterPro description. A significant motif is a known motif. Potential genes were defined as ORFs of Grades AAAA1–D3. (Supplementary Tables 7 and 8 are available at <a class="link link-uri" href="http://www.dnares.oxfordjournals.org" target="_blank"></a>)</p></div></div><div content-id="fig3" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 3</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/6/10.1093/dnares/dsl014/2/m_dsl014f3.jpeg?Expires=2147483647&Signature=VqDpGPbL3TQG4NyPYYMwq9wQ2Arz6wI~6S~T9hPhva2NM0-bqLaRVR-LAKPAJv96LzfK~qfsBlb3cAYJKOb8kvRKISjTAEq~3qguSdVKLFMDCdOO7CcdiHUWln5V6lwwPuGPgTg3GLDPuc4HaTgdWlhcCWYtTAuNwsV4BsYzmayhc97oA0SjVfLHd4Dn8B6HzQrLm~KSEJpokt81LhzHBLY2Yk-wvwT6suZqlKAp37Zz9B6Zw9fzHpKmDStRstb6J7Lri6jRGAeCcQqXNXpg77Ks3cXWyPQfc7uEwrjNXOnY5W-7WTQQE9LlqST-SCG1HjXVmRISapi3QL3KQPMlgQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="ORFs grades. ORFs were graded as follows. Grades A–C: functional information available; Grade D: hypothetical protein; Grade E: hypothetical protein supported by poor coverage in blastp and Grade X: no information available. An unknown protein is a hypothetical protein for which the product and notes are given in Supplementary Table 7. A putative membrane protein is a protein for which the product is given in Supplementary Table 8. An unknown motif is a motif that includes ‘unknown’ in its InterPro description. A significant motif is a known motif. Potential genes were defined as ORFs of Grades AAAA1–D3. (Supplementary Tables 7 and 8 are available at )" path-from-xml="dsl014f3.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6973959/dsl014f3.jpeg" path-from-xml="dsl014f3.jpeg" target="_blank">View large</a><a section="6973959" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/6/10.1093/dnares/dsl014/2/dsl014f3.jpeg?Expires=2147483647&Signature=MB5CXcvYplrmmn5u4jtooZTyXgxkkDvxXSTvZxVhR9vYfpRvlod89HOHH7Q~~tWtLj3GpFkVuKbLIlxP7Xh9OmwaP-wFhYUfTP5VNjb~Gcwmavp6q864Bbgp2rBmf3y~nZprDxJFZPSGkgvmf36cXNIvVKEzTY4CNG1ikXZND0jQgUxEuq7jbwLbqkeKxF5L~HtL-LEhE2EgqZ28bYoA~kM0EtKpox3aFx-Wn~57Ds8trofwRTnsq8gbqB~zGsBpvKzgAbWKLE7GOluGV2h4fiYg~JbBTOeebpf3KiseVZ4Y0DGfGbpalZZ8EqRaDXXH-TPt3zbNlV8y6HYija8Nwg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6973959&ar=464398&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl014f3.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>ORFs grades. ORFs were graded as follows. Grades A–C: functional information available; Grade D: hypothetical protein; Grade E: hypothetical protein supported by poor coverage in blastp and Grade X: no information available. An unknown protein is a hypothetical protein for which the product and notes are given in Supplementary Table 7. A putative membrane protein is a protein for which the product is given in Supplementary Table 8. An unknown motif is a motif that includes ‘unknown’ in its InterPro description. A significant motif is a known motif. Potential genes were defined as ORFs of Grades AAAA1–D3. (Supplementary Tables 7 and 8 are available at <a class="link link-uri" href="http://www.dnares.oxfordjournals.org" target="_blank"></a>)</p></div></div><p>The predicted ORFs were also graded with respect to the degree of matching to INSDC ORFs as follows: 1 (identical), 2 (3′-matched), 3 (newly predicted in this study) and 4 (not predicted by GLIMMER 2.0). Finally whole ORFs were graded for reliability and the degree of matching to INSDC in combination (e.g. AAAA1, BB2, C3 or D4). Potential genes were defined as ORFs of Grades AAAA1–D3 (<a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3</a>). All false positives obtained by GLIMMER 2.0 and taken from the INSDC data, which were not predicted by GLIMMER 2.0, were graded as E or X.</p><p>In the final step, a protein product name was automatically assigned to each ORF according to the method described in Supplementary Table 5. (Supplementary Table 5 is available at <a class="link link-uri" href="http://www.dnares.oxfordjournals.org" target="_blank"></a>) It took about 3 months to complete the entire job though all resources were dedicated to this project.</p>                            <h3 scrollto-destination=6973965 id="6973965" class="section-title" >2.3. Development of the GTPS viewer</h3>
<p>We developed a system for browsing the results of our GTPS analysis. This browser is available on the Web (GTPS Viewer, <a class="link link-uri" href="http://gtps.ddbj.nig.ac.jp/" target="_blank"></a>). Users can view the results of our ver. 2003 and ver. 2004 analyses, and the database will be annually updated and expanded. GTPS can show gene information for each genome, homology analysis results, motif search results, reliability grades and link to Genomes-TO-Protein structures and functions (GTOP).<a reveal-id="b30" data-open="b30" class="link link-ref link-reveal xref-bibr">30</a> GTPS can also show candidate orthologous gene sets based on homology searching. The results can be downloaded for each genome in DDBJ flat file and tab-delimited formats.</p>                            <h2 scrollto-destination=6973969 id="6973969" class="section-title" >3. Results and Discussion</h2>
                            <h3 scrollto-destination=6973971 id="6973971" class="section-title" >3.1. ORFs predicted by GTPS analysis</h3>
<p><a reveal-id="tbl1" data-open="tbl1" class="link link-reveal link-table xref-fig">Table 1</a> shows the classification of ORFs according to our grading system, A (best) to X (worst). The number of the potential genes (AAAA1–D3) registered in ver. 2003 was 370 876 and in ver. 2004 was 551 246. The number of ORFs registered in ver. 2003 INSDC was 362 828 and in ver. 2004 INSDC was 537 312. Of the ORFs registered in INSDC (6690 and 9657 genes for ver. 2003 and ver. 2004, respectively) ∼18% were not classified into possible protein-coding genes in this study. Comparison of ORFs between GTPS and INSDC is shown in <a reveal-id="tbl2" data-open="tbl2" class="link link-reveal link-table xref-fig">Table 2</a>. Of the possible protein-coding genes predicted by GTPS, ∼70% were identical to INSDC genes and ∼4% were newly found by GTPS. Researchers who submitted data to INSDC did not find these new ORFs. We also used the UniProtKB/Swiss-Prot database to confirm our new ORFs. The new ORFs overlapped those found by using UniProtKB/Swiss-Prot. For example, a newly found ORF between 8457 and 8678 bp of the <em>Aquifex aeolicus</em> VF5 genome was the same as one coding for 50S ribosomal protein L29 with accession no. P56613 in UniProtKB/Swiss-Prot. In addition to this, many new ORFs showed significant identities to those coding for the proteins registered in UniProtKB/Swiss-Prot (<a reveal-id="fig4" data-open="fig4" class="link link-reveal link-table xref-fig">Fig. 4</a>).</p>                        <a id="6973974" scrollto-destination="6973974"></a>
<div id="fig4" data-id="fig4" class="fig fig-section"><div class="label fig-label"><strong>Figure 4</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/6/10.1093/dnares/dsl014/2/m_dsl014f4.jpeg?Expires=2147483647&Signature=rGoNM44cr~bvjri7QNg0TiEcovCejjG5ZT7rAv6Aclrj8UM4qvE0XZ9yCu918nNkueMgtHea6tLflMy44M1iT2V1fBKJpxofVix4Fs3eMD6Tm-fu8RqZYEG2eb1xf10g4osrhTF4XoFQkIYaRJOlb8lteVJAHLilxyHD1DfA~DET-uYJQKMkWnEKnli-V29T-OmYu-OxhyMCMkBQ62MqlL2GGRu69ACfgAA7~gqd9XqGjKh28wErVb1eribCozGyyz6g0HIGxfnDpa9O9a4dS4qNHeDiOuknoRs2N8~6qg-rUIu56WyMpOFBjY8xrMYTucaZdwT7Kd00M-X7Uhe~mw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Verification of newly predicted ORFs obtained by the GTPS. Nine genes of eight organisms are shown for illustration. Yellow arrow shows an ORF that was not annotated in the INSDC data but was newly found in the GTPS. Positions on the genome, GTPS ID, grade, product name in the GTPS and homology search result against UniProtKB/Swiss-Prot or InterProScan are indicated under the each ORF." path-from-xml="dsl014f4.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6973974/dsl014f4.jpeg" path-from-xml="dsl014f4.jpeg" target="_blank">View large</a><a section="6973974" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/6/10.1093/dnares/dsl014/2/dsl014f4.jpeg?Expires=2147483647&Signature=3uRpoz8vMjQcsDmKmVDclegv0x6Iau35N8ZTqFZJ1YHcUfMXYjTbJsc-8eniVNkjxhTWlWFFmV7HSGnuL-ckhBKwBLqRznItyTKzl1rellRnXFSxAsZan6utyvnnKiAp844zx23uBuoVTv-GSbH8dcEOwFMTEtriB92ioImUOIlrPEWGlgeTmNMSDkCHeHWqr1YEAXitKGK-YD738QcWlWwvDcxztA3rH2kQ~X3CkTADqhJzooBaUjuhdGCxzPehy96SpzYXTk1fUxJg3TikpGyaBkJ9JNFkfSCRwlgdu1ID5jzR6tAu7TBEBBNU2sWXEd~ceInhRkdvSicnqA7qNA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6973974&ar=464398&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl014f4.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Verification of newly predicted ORFs obtained by the GTPS. Nine genes of eight organisms are shown for illustration. Yellow arrow shows an ORF that was not annotated in the INSDC data but was newly found in the GTPS. Positions on the genome, GTPS ID, grade, product name in the GTPS and homology search result against UniProtKB/Swiss-Prot or InterProScan are indicated under the each ORF.</p></div></div><div content-id="fig4" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 4</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/6/10.1093/dnares/dsl014/2/m_dsl014f4.jpeg?Expires=2147483647&Signature=rGoNM44cr~bvjri7QNg0TiEcovCejjG5ZT7rAv6Aclrj8UM4qvE0XZ9yCu918nNkueMgtHea6tLflMy44M1iT2V1fBKJpxofVix4Fs3eMD6Tm-fu8RqZYEG2eb1xf10g4osrhTF4XoFQkIYaRJOlb8lteVJAHLilxyHD1DfA~DET-uYJQKMkWnEKnli-V29T-OmYu-OxhyMCMkBQ62MqlL2GGRu69ACfgAA7~gqd9XqGjKh28wErVb1eribCozGyyz6g0HIGxfnDpa9O9a4dS4qNHeDiOuknoRs2N8~6qg-rUIu56WyMpOFBjY8xrMYTucaZdwT7Kd00M-X7Uhe~mw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Verification of newly predicted ORFs obtained by the GTPS. Nine genes of eight organisms are shown for illustration. Yellow arrow shows an ORF that was not annotated in the INSDC data but was newly found in the GTPS. Positions on the genome, GTPS ID, grade, product name in the GTPS and homology search result against UniProtKB/Swiss-Prot or InterProScan are indicated under the each ORF." path-from-xml="dsl014f4.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6973974/dsl014f4.jpeg" path-from-xml="dsl014f4.jpeg" target="_blank">View large</a><a section="6973974" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/6/10.1093/dnares/dsl014/2/dsl014f4.jpeg?Expires=2147483647&Signature=3uRpoz8vMjQcsDmKmVDclegv0x6Iau35N8ZTqFZJ1YHcUfMXYjTbJsc-8eniVNkjxhTWlWFFmV7HSGnuL-ckhBKwBLqRznItyTKzl1rellRnXFSxAsZan6utyvnnKiAp844zx23uBuoVTv-GSbH8dcEOwFMTEtriB92ioImUOIlrPEWGlgeTmNMSDkCHeHWqr1YEAXitKGK-YD738QcWlWwvDcxztA3rH2kQ~X3CkTADqhJzooBaUjuhdGCxzPehy96SpzYXTk1fUxJg3TikpGyaBkJ9JNFkfSCRwlgdu1ID5jzR6tAu7TBEBBNU2sWXEd~ceInhRkdvSicnqA7qNA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6973974&ar=464398&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl014f4.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Verification of newly predicted ORFs obtained by the GTPS. Nine genes of eight organisms are shown for illustration. Yellow arrow shows an ORF that was not annotated in the INSDC data but was newly found in the GTPS. Positions on the genome, GTPS ID, grade, product name in the GTPS and homology search result against UniProtKB/Swiss-Prot or InterProScan are indicated under the each ORF.</p></div></div><p>The number of bacterial genes is expected correlate with the size of the bacterial genome, because a bacterial genome is packed with genes without large intergenic regions and introns. We plotted the correlation between them using ver. 2004 of GTPS and that of INSDC (<a reveal-id="fig5" data-open="fig5" class="link link-reveal link-table xref-fig">Fig. 5</a>). It is clear that the number of genes correlates with the genome size. We thus consider that bacterial genomes have evolved in coordination between genetic complexity and genome size. For example, bacterial parasites such as <em>Buchnera</em> sp. do not possess many genes, and the genome is small. Therefore, using the linear relationship in <a reveal-id="fig5" data-open="fig5" class="link link-reveal link-table xref-fig">Fig. 5</a>, one can estimate the number of genes in a genome of a given size, or <em>vice versa</em>. In the GTPS study, the <em>Aeropyrum pernix</em> K1 genome decreased by 1006 genes, whereas the <em>Mycobacterium leprae</em> TN geneome increased by 549 genes. On the whole linearity holds better for GTPS than for INSDC, suggesting that the different researchers who submitted data used different tools and parameters for finding ORFs. Examples of less efficient gene predictions for the three organisms mentioned above are shown in <a reveal-id="fig6" data-open="fig6" class="link link-reveal link-table xref-fig">Fig. 6</a>. The GTPS moved ORFs without showing homology and motif data to lower grades. In the case of <em>M. leprae</em> TN, most of the 549 genes were to code for hypothetical proteins that were predicted on the flanking regions of pseudogenes. One such example is shown in <a reveal-id="fig4" data-open="fig4" class="link link-reveal link-table xref-fig">Fig. 4</a>.</p>                        <a id="6973977" scrollto-destination="6973977"></a>
<div id="fig5" data-id="fig5" class="fig fig-section"><div class="label fig-label"><strong>Figure 5</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/6/10.1093/dnares/dsl014/2/m_dsl014f5.jpeg?Expires=2147483647&Signature=PBEGlM9bAIOmiBgWMR3rvWa7-ckmHOnj62ZAoJmUn4K9-Dv2~REa2ak1iemqmeeYjLmns8RoHzKMIaWf-ttPLL6zz~~m3OdQ0y9UpItwomzQHH8YFXSCQ0ENPc6wC-N~tO1fNfGbvNWgaL29KKX1vYi~JAauZg79L9RU0hopPvNPQJwAZ329BdVAlnIiFCa4LSiLJfPKEYh1tsHICzwmrf4Qf80XLJTziGBZGzYlyoq3--40WJ6QYNI9R94kLg341fQya9htIc5oAR6keQfyfVouP6jW3qJ3fMYTpM6pLgzQOHRbBo8Jr9k5G62s57bnsiahcKjyUziCRjymzSdw-w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Correlation between the number of ORFs and genome size. The number of potential genes predicted by GTPS (Grades AAAA1–D3) was compared with the number of ORFs deposited in INSDC. The number of reliable genes decreased for Aeropyrum pernix K1 (red arrow) and increased for M. leprae TN (blue arrow)." path-from-xml="dsl014f5.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6973977/dsl014f5.jpeg" path-from-xml="dsl014f5.jpeg" target="_blank">View large</a><a section="6973977" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/6/10.1093/dnares/dsl014/2/dsl014f5.jpeg?Expires=2147483647&Signature=grR-hiWjR3Wug-BijxRDB1ky3KH-ZqloMz7BxXhhNMrtcbP-kbKzm1mO3-RHDsboiuUdNtbANUTtWm9emmky7WOzIKVJz3i7TyJwWYfSavKy2V9SvbwohHnfaccy17j9l0RwAcU8YRyWyk2v4UTREQ5ZgPlICGifsxYDMRjDkZD9UbJzHV0tpcXz3SkAgGyJUKCs2A8-7ZIm6hbP2crSQXpbDfUgdSSZJPh18xrSJLJQvesOx8xvbpLm7A3g6RdEjahz9IIpdrf4lJ0tHONkE9rhvDHuLCkd9Xr8X82uZqeqrgy-RzPdiwXoCzKBX4mvXtsSevw22UYWJ4pTgfgFJw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6973977&ar=464398&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl014f5.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Correlation between the number of ORFs and genome size. The number of potential genes predicted by GTPS (Grades AAAA1–D3) was compared with the number of ORFs deposited in INSDC. The number of reliable genes decreased for <em>Aeropyrum pernix</em> K1 (red arrow) and increased for <em>M. leprae</em> TN (blue arrow).</p></div></div><div content-id="fig5" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 5</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/6/10.1093/dnares/dsl014/2/m_dsl014f5.jpeg?Expires=2147483647&Signature=PBEGlM9bAIOmiBgWMR3rvWa7-ckmHOnj62ZAoJmUn4K9-Dv2~REa2ak1iemqmeeYjLmns8RoHzKMIaWf-ttPLL6zz~~m3OdQ0y9UpItwomzQHH8YFXSCQ0ENPc6wC-N~tO1fNfGbvNWgaL29KKX1vYi~JAauZg79L9RU0hopPvNPQJwAZ329BdVAlnIiFCa4LSiLJfPKEYh1tsHICzwmrf4Qf80XLJTziGBZGzYlyoq3--40WJ6QYNI9R94kLg341fQya9htIc5oAR6keQfyfVouP6jW3qJ3fMYTpM6pLgzQOHRbBo8Jr9k5G62s57bnsiahcKjyUziCRjymzSdw-w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Correlation between the number of ORFs and genome size. The number of potential genes predicted by GTPS (Grades AAAA1–D3) was compared with the number of ORFs deposited in INSDC. The number of reliable genes decreased for Aeropyrum pernix K1 (red arrow) and increased for M. leprae TN (blue arrow)." path-from-xml="dsl014f5.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6973977/dsl014f5.jpeg" path-from-xml="dsl014f5.jpeg" target="_blank">View large</a><a section="6973977" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/6/10.1093/dnares/dsl014/2/dsl014f5.jpeg?Expires=2147483647&Signature=grR-hiWjR3Wug-BijxRDB1ky3KH-ZqloMz7BxXhhNMrtcbP-kbKzm1mO3-RHDsboiuUdNtbANUTtWm9emmky7WOzIKVJz3i7TyJwWYfSavKy2V9SvbwohHnfaccy17j9l0RwAcU8YRyWyk2v4UTREQ5ZgPlICGifsxYDMRjDkZD9UbJzHV0tpcXz3SkAgGyJUKCs2A8-7ZIm6hbP2crSQXpbDfUgdSSZJPh18xrSJLJQvesOx8xvbpLm7A3g6RdEjahz9IIpdrf4lJ0tHONkE9rhvDHuLCkd9Xr8X82uZqeqrgy-RzPdiwXoCzKBX4mvXtsSevw22UYWJ4pTgfgFJw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6973977&ar=464398&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl014f5.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Correlation between the number of ORFs and genome size. The number of potential genes predicted by GTPS (Grades AAAA1–D3) was compared with the number of ORFs deposited in INSDC. The number of reliable genes decreased for <em>Aeropyrum pernix</em> K1 (red arrow) and increased for <em>M. leprae</em> TN (blue arrow).</p></div></div>                        <a id="6973979" scrollto-destination="6973979"></a>
<div id="fig6" data-id="fig6" class="fig fig-section"><div class="label fig-label"><strong>Figure 6</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/6/10.1093/dnares/dsl014/2/m_dsl014f6.jpeg?Expires=2147483647&Signature=iaRs3DdW6m840ZMrrwXGdQRpSbVQqrQXzUW04MxEIGufW2Al0fJnPVte0RykSwFz57JT2z5SnJsRXtBQb8y6psDlvGXPLEtUCgioJaD6rvF2wLXYvFGSZKINdJKzf-9i-6WgLOF-QcgU2QqGumh-c8qHDkkzbwAZRvI7~Y9ZZ-T3xl3ljEuyUskB-JdRqGZG7ocrwn6v97ANtp5FKc71uM8PuoHdeiFTIGYUkReQBVdA8soD8nlgTXhFDQk0i-oXYn9wp2saXAHiCg0VY8F8hVBpo9JCgxNxX-FDfXkogS7h~rDto~ZJSCg6tb4nBQLmuJ63xzZdAjERVr-O4avi5g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Examples of annotation data for the three organisms. A number of ORFs that were annotated in INSDC data were sorted to Grades D, E and X. Each arrow represents an ORF. Green ORFs are potential genes (Grades AAAA–D3). Purple ORF (Grade D4) and gray ORF (Grade X), which were annotated in INSDC, were not classified as potential genes in the GTPS." path-from-xml="dsl014f6.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6973979/dsl014f6.jpeg" path-from-xml="dsl014f6.jpeg" target="_blank">View large</a><a section="6973979" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/6/10.1093/dnares/dsl014/2/dsl014f6.jpeg?Expires=2147483647&Signature=yRKFznsm4Vu~SnDyauB-xwvBhilvrKlgvtZ1T0GtATfBX2lQTkqKWImOiZRyppwCoelFT59iflTg5UzJpPxu~Cj5o4ZC53hS1DgtFz5mhiiGAQeuUWvNM3xuEpNrGTt5jDe0BULw9i8ntXaLvniLkHlOiS0GxUZJRJQRzuwmyF-oObk4jVIrIHDwZ7CnXX7RfuitqYNpNQJxLxnOt4FmXPIe1zXiYr52GBuZGXHd-gboF6zCktaZwB6t6S1q08U9xkpiX-qx0OBTiDnAi7LKG9lUg9HjK2I6ZrARu~x-BLf7PlLxSYFyG5T54NWwFWgug76Jpc4w-lTSPabdyvHDfg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6973979&ar=464398&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl014f6.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Examples of annotation data for the three organisms. A number of ORFs that were annotated in INSDC data were sorted to Grades D, E and X. Each arrow represents an ORF. Green ORFs are potential genes (Grades AAAA–D3). Purple ORF (Grade D4) and gray ORF (Grade X), which were annotated in INSDC, were not classified as potential genes in the GTPS.</p></div></div><div content-id="fig6" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 6</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/6/10.1093/dnares/dsl014/2/m_dsl014f6.jpeg?Expires=2147483647&Signature=iaRs3DdW6m840ZMrrwXGdQRpSbVQqrQXzUW04MxEIGufW2Al0fJnPVte0RykSwFz57JT2z5SnJsRXtBQb8y6psDlvGXPLEtUCgioJaD6rvF2wLXYvFGSZKINdJKzf-9i-6WgLOF-QcgU2QqGumh-c8qHDkkzbwAZRvI7~Y9ZZ-T3xl3ljEuyUskB-JdRqGZG7ocrwn6v97ANtp5FKc71uM8PuoHdeiFTIGYUkReQBVdA8soD8nlgTXhFDQk0i-oXYn9wp2saXAHiCg0VY8F8hVBpo9JCgxNxX-FDfXkogS7h~rDto~ZJSCg6tb4nBQLmuJ63xzZdAjERVr-O4avi5g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Examples of annotation data for the three organisms. A number of ORFs that were annotated in INSDC data were sorted to Grades D, E and X. Each arrow represents an ORF. Green ORFs are potential genes (Grades AAAA–D3). Purple ORF (Grade D4) and gray ORF (Grade X), which were annotated in INSDC, were not classified as potential genes in the GTPS." path-from-xml="dsl014f6.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6973979/dsl014f6.jpeg" path-from-xml="dsl014f6.jpeg" target="_blank">View large</a><a section="6973979" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/6/10.1093/dnares/dsl014/2/dsl014f6.jpeg?Expires=2147483647&Signature=yRKFznsm4Vu~SnDyauB-xwvBhilvrKlgvtZ1T0GtATfBX2lQTkqKWImOiZRyppwCoelFT59iflTg5UzJpPxu~Cj5o4ZC53hS1DgtFz5mhiiGAQeuUWvNM3xuEpNrGTt5jDe0BULw9i8ntXaLvniLkHlOiS0GxUZJRJQRzuwmyF-oObk4jVIrIHDwZ7CnXX7RfuitqYNpNQJxLxnOt4FmXPIe1zXiYr52GBuZGXHd-gboF6zCktaZwB6t6S1q08U9xkpiX-qx0OBTiDnAi7LKG9lUg9HjK2I6ZrARu~x-BLf7PlLxSYFyG5T54NWwFWgug76Jpc4w-lTSPabdyvHDfg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6973979&ar=464398&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl014f6.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Examples of annotation data for the three organisms. A number of ORFs that were annotated in INSDC data were sorted to Grades D, E and X. Each arrow represents an ORF. Green ORFs are potential genes (Grades AAAA–D3). Purple ORF (Grade D4) and gray ORF (Grade X), which were annotated in INSDC, were not classified as potential genes in the GTPS.</p></div></div><p>We made a histogram of the corresponding number of amino acids in the new ORFs predicted by applying GTPS to the data of ver. 2004 to know if the difference in the number of ORFs between the GTPS and INSDC predictions is due to gene size. At first, the newly predicted ORFs were selected from Grades AAAA3, AAA3, AA3, A3, BBBB3, BBB3, BB3, B3 and C3. ORFs with corresponding locations in the INSDC sequence data for which any information such as gene feature, repeat_region feature, or other characteristics were then removed. The remaining 9088 ORFs with no information were used for making the histogram. As seen in <a reveal-id="fig7" data-open="fig7" class="link link-reveal link-table xref-fig">Fig. 7</a>, 6739 new ORFs from the GTPS prediction correspond to short proteins with 20–100 amino acids. This indicates that the minimum number of nucleotides in ORF prediction was set at &gt;100 amino acids in some genome projects. Supplementary <a reveal-id="tbl1" data-open="tbl1" class="link link-reveal link-table xref-fig">Table 1</a> also supports this suggestion. (Supplementary Table 1 is available at <a class="link link-uri" href="http://www.dnares.oxfordjournals.org" target="_blank"></a>).</p>                        <a id="6973984" scrollto-destination="6973984"></a>
<div id="fig7" data-id="fig7" class="fig fig-section"><div class="label fig-label"><strong>Figure 7</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/6/10.1093/dnares/dsl014/2/m_dsl014f7.jpeg?Expires=2147483647&Signature=m1R4QkBFX19zxOMlfe4HEIU4IgEhy76xpJE~VHBTHB4nX5ONuEuxE1B5nMVS0R~j0zOrJypD2AQaMLV-JJlcZO7KRJ8B0G0MtDcOh5Gkavx~JDyn2m7pCDfZTXla87BYqCwMtjaDbCfmlST6RGvzq3wsXw2JvfC73nGBFAPW2rowheDNOz4OnE3u8K5AztM7g5tObqInGPDB02GCK44p883mdVnBCbFz8F9f4WEtm1b2rgUsQygSJCKM~xkhEyc77Z19uUj5FQOXE2LGQGP6DKIbKmbjlIEwgITRmeJAvgsg03ASG7nkarkC~IVNc5vWUKLDyGLdPCjzYZZD1CVIfg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of amino acid sequence lengths of new genes predicted by GTPS. The newly predicted ORFs, which were not annotated in INSDC, were selected from Grades AAAA3, AAA3, AA3, A3, BBBB3, BBB3, BB3, B3 and C3 of ver. 2004 and used for drawing a histogram. The number of newly predicted ORFs and their amino acid sequence lengths are shown." path-from-xml="dsl014f7.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6973984/dsl014f7.jpeg" path-from-xml="dsl014f7.jpeg" target="_blank">View large</a><a section="6973984" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/6/10.1093/dnares/dsl014/2/dsl014f7.jpeg?Expires=2147483647&Signature=2ACfhmb0cmz-5seKJHIppqcDqjXR-kZKf9YSzC44dTEzCkzkgeQlzz1ExeflpMDtD0wyfZ0tfju4vo2zSv8vVuMcwNMj4X6tEWrZygfkSK4zGj2nGX962CUlqY7XGT-V0dVnhUpYr5o3KdosEtxkOdK05IaHHqib1pvigRaQG9cVVGPxFA3RWSsrfNwSQNj9a3eUPGmP2M89HYI-V2wUb4EyFT12TVZpqSt3OKkGLUgnoCt~DXnDZRr1jvcvLMBPm-CpYhpO6POpQWsl14vGUOfE8HlHDcyKnrvFtXIk5qI8miJNRhhcZm1PDnLH3JA662W9aWY4eF6aUzRicIlEBQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6973984&ar=464398&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl014f7.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of amino acid sequence lengths of new genes predicted by GTPS. The newly predicted ORFs, which were not annotated in INSDC, were selected from Grades AAAA3, AAA3, AA3, A3, BBBB3, BBB3, BB3, B3 and C3 of ver. 2004 and used for drawing a histogram. The number of newly predicted ORFs and their amino acid sequence lengths are shown.</p></div></div><div content-id="fig7" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 7</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/6/10.1093/dnares/dsl014/2/m_dsl014f7.jpeg?Expires=2147483647&Signature=m1R4QkBFX19zxOMlfe4HEIU4IgEhy76xpJE~VHBTHB4nX5ONuEuxE1B5nMVS0R~j0zOrJypD2AQaMLV-JJlcZO7KRJ8B0G0MtDcOh5Gkavx~JDyn2m7pCDfZTXla87BYqCwMtjaDbCfmlST6RGvzq3wsXw2JvfC73nGBFAPW2rowheDNOz4OnE3u8K5AztM7g5tObqInGPDB02GCK44p883mdVnBCbFz8F9f4WEtm1b2rgUsQygSJCKM~xkhEyc77Z19uUj5FQOXE2LGQGP6DKIbKmbjlIEwgITRmeJAvgsg03ASG7nkarkC~IVNc5vWUKLDyGLdPCjzYZZD1CVIfg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of amino acid sequence lengths of new genes predicted by GTPS. The newly predicted ORFs, which were not annotated in INSDC, were selected from Grades AAAA3, AAA3, AA3, A3, BBBB3, BBB3, BB3, B3 and C3 of ver. 2004 and used for drawing a histogram. The number of newly predicted ORFs and their amino acid sequence lengths are shown." path-from-xml="dsl014f7.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6973984/dsl014f7.jpeg" path-from-xml="dsl014f7.jpeg" target="_blank">View large</a><a section="6973984" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/6/10.1093/dnares/dsl014/2/dsl014f7.jpeg?Expires=2147483647&Signature=2ACfhmb0cmz-5seKJHIppqcDqjXR-kZKf9YSzC44dTEzCkzkgeQlzz1ExeflpMDtD0wyfZ0tfju4vo2zSv8vVuMcwNMj4X6tEWrZygfkSK4zGj2nGX962CUlqY7XGT-V0dVnhUpYr5o3KdosEtxkOdK05IaHHqib1pvigRaQG9cVVGPxFA3RWSsrfNwSQNj9a3eUPGmP2M89HYI-V2wUb4EyFT12TVZpqSt3OKkGLUgnoCt~DXnDZRr1jvcvLMBPm-CpYhpO6POpQWsl14vGUOfE8HlHDcyKnrvFtXIk5qI8miJNRhhcZm1PDnLH3JA662W9aWY4eF6aUzRicIlEBQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6973984&ar=464398&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl014f7.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of amino acid sequence lengths of new genes predicted by GTPS. The newly predicted ORFs, which were not annotated in INSDC, were selected from Grades AAAA3, AAA3, AA3, A3, BBBB3, BBB3, BB3, B3 and C3 of ver. 2004 and used for drawing a histogram. The number of newly predicted ORFs and their amino acid sequence lengths are shown.</p></div></div><p>In the GTPS prediction for the ver. 2004 data, 581 490 ORFs coding for 20–100 amino acids were predicted, but most of them fell into the worst or second-worst grades (5125 ORFs in Grade E and 500 433 ORFs in Grade X). There were 39 538 ORFs classified into Grade D that were hypothetical proteins. There were 36 394 ORFs classified into Grades AAAA–C according to results of homology or motif searches, 6739 of these were revealed to be new genes. As shown in <a reveal-id="fig4" data-open="fig4" class="link link-reveal link-table xref-fig">Fig. 4</a>, some of the new ORFs coding for 20–100 amino acids show high identity to proteins in UniProtKB/Swiss-Prot. GTPS with no size limitation was able to predict ORF coding for 2159 amino acids in the <em>Bordetella bronchiseptica</em> RB50 and <em>Bordetella parapertussis</em> 12 822 strains.</p><p>In this study, ORFs of complete genomes deposited in INSDC and stored in GIB were re-evaluated. In the GTPS analysis, all complete bacterial genomes are simultaneously re-evaluated by blastp and InterProScan analyses for the same version of a database. Note that we cannot compare ORF prediction results one with another if they were obtained by using different versions of reference databases (e.g. InterPro) and different versions of gene prediction tools. Neither can we assign consistent grades of reliability to predicted ORFs. GTPS leads to more consistent, comprehensive and reliable results in ORF prediction than those obtained by any extant method.</p>                            <h3 scrollto-destination=6973992 id="6973992" class="section-title" >3.2. E. coli genome annotation</h3>
<p><em>E. coli</em> is a model organism, and it had been studied extensively before the whole genome was sequenced. In this section, we compare annotation of the <em>E. coli</em> genome in ver. 2003, ver. 2004 and INSDC. It is noted that by updating <em>E. coli</em> genome data, the number of bases in the genome increased from 4 639 221 in ver. 2003 to 4 639 675 in ver. 2004.<a reveal-id="b31" data-open="b31" class="link link-ref link-reveal xref-bibr">31</a> However, the total number of the predicted ORFs decreased from 9845 (ver 2003) to 9768 (ver 2004). The number of ORFs in Grades AAAA–D3 also decreased from 4695 (ver. 2003) to 4688 (ver. 2004). The ratio of the number of ORFs in AAAA–D3 to the total number of ORFs was 48% in both ver. 2003 and ver. 2004. In ver. 2003 125 ORFs turned out to code for a different amino acid sequence than those in INSDC, as did 111 ORFs in ver. 2004. However, it was not feasible for GTPS to clarify whether the changes were due to sequence errors or other causes. As we continue the annual GTPS analysis, it will be possible for us to detect hidden sequence errors.</p><p>Comparison of the <em>E. coli</em> ORFs in the two versions and those in INSDC is summarized in <a reveal-id="tbl3" data-open="tbl3" class="link link-reveal link-table xref-fig">Table 3</a>. Whereas the number of the new ORFs decreased in ver. 2004, the number of 3′-matched ORFs increased. In addition, 193 new ORFs in ver. 2003 matched the 3′ terminals of INSDC ORFs, indicating that GTPS predicted the 193 ORFs in advance of the INSDC ORFs update. An ORF from the present comparison was adopted to the ORF of ECK4368 in an international community annotation.<a reveal-id="b8" data-open="b8" class="link link-ref link-reveal xref-bibr">8</a></p>                            <h3 scrollto-destination=6973995 id="6973995" class="section-title" >3.3. Simulation of the wide range of gene divergence in the bacterial world</h3>
<p>An interesting question is whether or not the number of reliable ORFs will increase according to the increasing number of new bacterial genomes to be sequenced and submitted in the future. To answer this question, we conducted computer simulations focusing on the diversity of gene functions in bacterial genomes. Homologous ORFs with (90% identity were defined as having the same function. The percentage of homology was obtained by using CD-HIT.<a reveal-id="b32" data-open="b32" class="link link-ref link-reveal xref-bibr">32</a>,<a reveal-id="b33" data-open="b33" class="link link-ref link-reveal xref-bibr">33</a> When the ORFs are clustered into groups by homology, the number of groups indicates the degree of divergence in gene function. If the number of groups becomes saturated despite an increase in the number of genomes, we will be able to regard the saturated number as the number of unique genes in the bacterial world. With this in mind, we conducted the following simulation.</p><p>First, we randomly sampled 10 genomes from the 183 genomes and clustered their predicted ORFs by using CD-HIT. We then computed the ratio of the clustered ORFs to all ORFs among the sampled genomes was obtained. This procedure was repeated 10 times to calculate the average ratio over the repeats. Next, the whole procedure was repeated, but the number of sampled genomes was increased by 10 at a time until the number reached 180. These 180 comprised the simulation set. We conducted two simulations for (90 and 100% homology in clusterings. The results are shown in <a reveal-id="fig8" data-open="fig8" class="link link-reveal link-table xref-fig">Fig. 8</a>.</p>                        <a id="6973998" scrollto-destination="6973998"></a>
<div id="fig8" data-id="fig8" class="fig fig-section"><div class="label fig-label"><strong>Figure 8</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/6/10.1093/dnares/dsl014/2/m_dsl014f8.jpeg?Expires=2147483647&Signature=a0deshS07nb38IcC1r-K3n~53JyFGrPqzp-gwo0h-1TMdWkFVgvwpbwBMUiglS1PlM0dvzi~X3JHCGSzpRhPMOPSuf7SSkztMeSuncw~XDvjJdk-GEy5sIC4kxf2emUIiYjI6btZ58sXUnl7GINuGyw6PiLd1Lxa3vzvn9tSV0nprmIzeL09krOgILo5N1Aj7~w7hpeTZls6ERY2hENqjn3QCf1a5EH3ULgYPGJGOrDLlJYwQ5XhsKJrh7~dY2aXeE2rbleAZGk2bS7oSRoISrQiTvU0lRdtvAkIlTRiPimB6A2FQXTYXhP9Z2ZsoyhGaQzpe~COsBYZJTEwEYeeFQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Correlation between the number of genomes and the ratio of clustered genes. Each number of genomes (x-axis) was chosen by random selection from 183 genomes. The ratio of clustered genes indicates the average value when CD-HIT was performed 10 times for a different genome set. ID100 (square) and ID90 (triangle) show the ratio of clustered genes based on 100 and 90% identity, respectively, under the condition of ≥90% coverage." path-from-xml="dsl014f8.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6973998/dsl014f8.jpeg" path-from-xml="dsl014f8.jpeg" target="_blank">View large</a><a section="6973998" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/6/10.1093/dnares/dsl014/2/dsl014f8.jpeg?Expires=2147483647&Signature=E9N1SUrV8zs0zkOBx2t5e0wkIPc-A-M9iZ7ePl2irIYiw7i24CZbTm9fLmX2KxZJMD5~wSD1T2KXZzjB3XyYdAOMoBas8Gj~YTFfILhkTexD4it1CdxKqOffF83C5up01xxYk63BJfo9ly2nzJp~BxzxtAgjMZCSxX9iewkgNc7iovssWZW9FVbrJocuUuoUrKVUQQEXQMMa~NgJqNuWAOX6sS70E3YiHk-eC3FUOBNC4nQN0B27IUBivVtkp084QGx08j6uojmElwsM65EUAK2DDGSe34vGZ4O2XA-aLWI0h0k3M~b688Eo~w-27QV~GatybRQrQhqF~CSRHDHNOg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6973998&ar=464398&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl014f8.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Correlation between the number of genomes and the ratio of clustered genes. Each number of genomes (<em>x</em>-axis) was chosen by random selection from 183 genomes. The ratio of clustered genes indicates the average value when CD-HIT was performed 10 times for a different genome set. ID100 (square) and ID90 (triangle) show the ratio of clustered genes based on 100 and 90% identity, respectively, under the condition of ≥90% coverage.</p></div></div><div content-id="fig8" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 8</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/6/10.1093/dnares/dsl014/2/m_dsl014f8.jpeg?Expires=2147483647&Signature=a0deshS07nb38IcC1r-K3n~53JyFGrPqzp-gwo0h-1TMdWkFVgvwpbwBMUiglS1PlM0dvzi~X3JHCGSzpRhPMOPSuf7SSkztMeSuncw~XDvjJdk-GEy5sIC4kxf2emUIiYjI6btZ58sXUnl7GINuGyw6PiLd1Lxa3vzvn9tSV0nprmIzeL09krOgILo5N1Aj7~w7hpeTZls6ERY2hENqjn3QCf1a5EH3ULgYPGJGOrDLlJYwQ5XhsKJrh7~dY2aXeE2rbleAZGk2bS7oSRoISrQiTvU0lRdtvAkIlTRiPimB6A2FQXTYXhP9Z2ZsoyhGaQzpe~COsBYZJTEwEYeeFQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Correlation between the number of genomes and the ratio of clustered genes. Each number of genomes (x-axis) was chosen by random selection from 183 genomes. The ratio of clustered genes indicates the average value when CD-HIT was performed 10 times for a different genome set. ID100 (square) and ID90 (triangle) show the ratio of clustered genes based on 100 and 90% identity, respectively, under the condition of ≥90% coverage." path-from-xml="dsl014f8.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6973998/dsl014f8.jpeg" path-from-xml="dsl014f8.jpeg" target="_blank">View large</a><a section="6973998" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/6/10.1093/dnares/dsl014/2/dsl014f8.jpeg?Expires=2147483647&Signature=E9N1SUrV8zs0zkOBx2t5e0wkIPc-A-M9iZ7ePl2irIYiw7i24CZbTm9fLmX2KxZJMD5~wSD1T2KXZzjB3XyYdAOMoBas8Gj~YTFfILhkTexD4it1CdxKqOffF83C5up01xxYk63BJfo9ly2nzJp~BxzxtAgjMZCSxX9iewkgNc7iovssWZW9FVbrJocuUuoUrKVUQQEXQMMa~NgJqNuWAOX6sS70E3YiHk-eC3FUOBNC4nQN0B27IUBivVtkp084QGx08j6uojmElwsM65EUAK2DDGSe34vGZ4O2XA-aLWI0h0k3M~b688Eo~w-27QV~GatybRQrQhqF~CSRHDHNOg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6973998&ar=464398&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl014f8.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Correlation between the number of genomes and the ratio of clustered genes. Each number of genomes (<em>x</em>-axis) was chosen by random selection from 183 genomes. The ratio of clustered genes indicates the average value when CD-HIT was performed 10 times for a different genome set. ID100 (square) and ID90 (triangle) show the ratio of clustered genes based on 100 and 90% identity, respectively, under the condition of ≥90% coverage.</p></div></div><p>As shown in <a reveal-id="fig8" data-open="fig8" class="link link-reveal link-table xref-fig">Fig. 8</a>, the ratio increased with the increase in the number of genomes. The ratio is not yet saturated at 180 genomes (29.5% of the genes among 180 genomes were clustered and presumed to have the same function.). Therefore, we could not determine the total number of bacterial genes by using 183 genomes. We need to have genome sequences of strains from every branch of the phylogenetic tree for bacteria. We will then be able to infer the total number of gene functions from the bacterial world.</p><p>Essentially, we estimated the total number of ‘orthologous’ genes in the prokaryote world. An orthologous gene of a species is defined as one that has diverged from the gene of its common ancestor by speciation. Therefore, an orthologous gene, like cytochrome <em>c</em> gene, functions equally in all species that carry it. Most orthologous genes submitted to INSDC were predicted to be ORFs by software tools for gene prediction or homology searching. It is thus natural to treat such an orthologous gene in different species by the same standard or criteria when one examines and evaluates its authenticity. In our GTPS analysis, we used only bacterial genome sequence data, and we disregarded the attached biological information or annotation. What we did is also called annotation, but it may differ from what is usually used. We believe that our criteria and the results of our examination and evaluation of as many bacterial genes as available at INSDC based on the same criteria will be quite useful and helpful, particularly for those who carry out homology searches against INSDC in their studies including genome sequencing projects and for those who try to experimentally examine the authenticity of a hypothetical gene.</p>                            <h3 scrollto-destination=6974001 id="6974001" class="section-title" >3.4. Future aspects of GTPS</h3>
<p>Users of our GTPS database can see the details of protocols and look into lines of evidence that support our prediction and grading of ORFs on the basis of blastp and InterProScan analyses (visit <a class="link link-uri" href="http://gtps.ddbj.nig.ac.jp/" target="_blank"></a>). For example, while some researchers want to use only Grade A ORFs, others may include Grade D ORFs. Therefore, researchers can have confidence in selecting and using candidate genes from the GTPS database without visiting multiple sites on the Internet and repeating large-scale computation. Even genome sequencing projects may use the GTPS database for annotation of newly sequenced genomes. It is noted that GTPS will reflect results of proteome analysis through quality-controlled secondary databases including InterPro, because we continue GTPS analysis by using the latest tools and databases, and we update our GTPS database at least once a year. We will increase the frequency of updates by improving each step of the GTPS analysis and using more powerful computer resources.</p><p>Venter <em>et al</em>.<a reveal-id="b34" data-open="b34" class="link link-ref link-reveal xref-bibr">34</a> reported and submitted massive amounts of data on 800 000 sequences and 1 500 000 genes into the whole genome shotgun (WGS) section of INSDC. The amount was almost equivalent to the number of bacterial genes that had been published at that time. The number of genomic sequences deposited into WGS has been increasing, and the number of WGS entries exceeded the total number of the other INSDC entries in 2005. However, finding useful genes in the WGS entries has not been straightforward due to heterogeneity of the annotations. Therefore, we will carry out GTPS analysis for the WGS sequences to extract genes with consistent reliability scores. With the emergence of new sequencing technologies such as the 454 sequencing system,<a reveal-id="b35" data-open="b35" class="link link-ref link-reveal xref-bibr">35</a> meta-genome data will be produced and submitted to INSDC in succession. GTPS will also be useful for annotating a large number of possible genes in the submitted meta-genome data.</p><p>The results of re-annotation of complete microbial genomes are freely available at <a class="link link-uri" href="http://gtps.ddbj.nig.ac.jp/" target="_blank"></a>.</p>                            <h2 scrollto-destination=6974005 id="6974005" class="section-title" >Supplementary data</h2>
<p>Supplementary data are available online at <a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org" target="_blank"></a></p><p>The authors express their sincere thanks to Yasumasa Shigemoto of Fujitsu Co., Ltd., Yoshikazu Kuwana of Tokai Soft Co., Ltd, Shigetaka Sakamoto of HOLONICS Co., Ltd and Mitsui Knowledge Industry Co., Ltd for their part in the development of the GTPS database. This work was supported in part by the Institute for Bioinformatics Research and Development of the Japan Science and Technology Agency and the Japanese Bioportal Project of Special Coordination Funds for Promoting Science and Technology. Computation was done with the RIKEN Super Combined Cluster System.</p>                            <h2 scrollto-destination=6974009 id="6974009" class="backreferences-title" >REFERENCES</h2>
<div class="ref-list"><div content-id="b1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fleischmann</div> <div class="given-names">R. D.</div></div>,  <div class="name"><div class="surname">Adams</div> <div class="given-names">M. D.</div></div>,  <div class="name"><div class="surname">White</div> <div class="given-names">O.</div></div>, et al. </div><div class="article-title">Whole-genome random sequencing and assembly of <em>Haemophilus influenzae</em> Rd</div>, <div class="source">Science</div> , <div class="year">1995</div>, vol. <div class="volume">269</div> (pg. <div class="fpage">496</div>-<div class="lpage">512</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Whole-genome%20random%20sequencing%20and%20assembly%20of%20Haemophilus%20influenzae%20Rd&amp;author=R.%20D.%20Fleischmann&amp;author=M.%20D.%20Adams&amp;author=O.%20White&amp;publication_year=1995&amp;journal=Science&amp;volume=269&amp;pages=496-512" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.7542800" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.7542800" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.7542800"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7542800" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lander</div> <div class="given-names">E. S.</div></div>,  <div class="name"><div class="surname">Waterman</div> <div class="given-names">M. S.</div></div>. </div><div class="article-title">Genomic mapping by fingerprinting random clones: a mathematical analysis</div>, <div class="source">Genomics</div> , <div class="year">1988</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">231</div>-<div class="lpage">239</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20mapping%20by%20fingerprinting%20random%20clones%3A%20a%20mathematical%20analysis&amp;author=E.%20S.%20Lander&amp;author=M.%20S.%20Waterman&amp;publication_year=1988&amp;journal=Genomics&amp;volume=2&amp;pages=231-239" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0888-7543(88)90007-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0888-7543(88)90007-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0888-7543(88)90007-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3294162" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fumoto</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Miyazaki</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Sugawara</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Genome Information Broker (GIB): data retrieval and comparative analysis system for completed microbial genomes and more</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2002</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">66</div>-<div class="lpage">68</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20Information%20Broker%20(GIB)%3A%20data%20retrieval%20and%20comparative%20analysis%20system%20for%20completed%20microbial%20genomes%20and%20more&amp;author=M.%20Fumoto&amp;author=S.%20Miyazaki&amp;author=H.%20Sugawara&amp;publication_year=2002&amp;journal=Nucleic%20Acids%20Res.&amp;volume=30&amp;pages=66-68" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/30.1.66" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F30.1.66" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F30.1.66"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11752256" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Amann</div> <div class="given-names">R. I.</div></div>,  <div class="name"><div class="surname">Ludwig</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Schleifer</div> <div class="given-names">K. H.</div></div>. </div><div class="article-title">Phylogenetic identification and in situ detection of individual microbial cells without cultivation</div>, <div class="source">Microbiol. Rev.</div> , <div class="year">1995</div>, vol. <div class="volume">59</div> (pg. <div class="fpage">143</div>-<div class="lpage">169</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phylogenetic%20identification%20and%20in%20situ%20detection%20of%20individual%20microbial%20cells%20without%20cultivation&amp;author=R.%20I.%20Amann&amp;author=W.%20Ludwig&amp;author=K.%20H.%20Schleifer&amp;publication_year=1995&amp;journal=Microbiol.%20Rev.&amp;volume=59&amp;pages=143-169" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7535888" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hugenholtz</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Pace</div> <div class="given-names">N. R.</div></div>. </div><div class="article-title">Identifying microbial diversity in the natural environment: a molecular phylogenetic approach</div>, <div class="source">Trends Biotechnol.</div> , <div class="year">1996</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">190</div>-<div class="lpage">197</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identifying%20microbial%20diversity%20in%20the%20natural%20environment%3A%20a%20molecular%20phylogenetic%20approach&amp;author=P.%20Hugenholtz&amp;author=N.%20R.%20Pace&amp;publication_year=1996&amp;journal=Trends%20Biotechnol.&amp;volume=14&amp;pages=190-197" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0167-7799(96)10025-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0167-7799(96)10025-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0167-7799(96)10025-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8663938" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Curtis</div> <div class="given-names">T. P.</div></div>,  <div class="name"><div class="surname">Sloan</div> <div class="given-names">W. T.</div></div>,  <div class="name"><div class="surname">Scannell</div> <div class="given-names">J. W.</div></div>. </div><div class="article-title">Estimating prokaryotic diversity and its limits</div>, <div class="source">Proc. Natl Acad. Sci. USA.</div> , <div class="year">2002</div>, vol. <div class="volume">99</div> (pg. <div class="fpage">10494</div>-<div class="lpage">10499</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Estimating%20prokaryotic%20diversity%20and%20its%20limits&amp;author=T.%20P.%20Curtis&amp;author=W.%20T.%20Sloan&amp;author=J.%20W.%20Scannell&amp;publication_year=2002&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA.&amp;volume=99&amp;pages=10494-10499" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.142680199" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.142680199" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.142680199"> </span></div> </div></div></div></div></div><div content-id="b7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Blattner</div> <div class="given-names">F. R.</div></div>,  <div class="name"><div class="surname">Plunkett</div> <div class="given-names">G.</div>III</div>,  <div class="name"><div class="surname">Bloch</div> <div class="given-names">C. A.</div></div>, et al. </div><div class="article-title">The complete genome sequence of Escherichia coli, K-12</div>, <div class="source">Science</div> , <div class="year">1997</div>, vol. <div class="volume">277</div> (pg. <div class="fpage">1453</div>-<div class="lpage">1474</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20complete%20genome%20sequence%20of%20Escherichia%20coli%2C%20K-12&amp;author=F.%20R.%20Blattner&amp;author=G.%20Plunkett&amp;author=C.%20A.%20Bloch&amp;publication_year=1997&amp;journal=Science&amp;volume=277&amp;pages=1453-1474" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.277.5331.1453" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.277.5331.1453" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.277.5331.1453"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9278503" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Riley</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Abe</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Arnaud</div> <div class="given-names">M. B.</div></div>, et al. </div><div class="article-title">Escherichia coli K-12: a cooperatively developed annotation snapshot—2005</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2006</div>, vol. <div class="volume">34</div> (pg. <div class="fpage">1</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Escherichia%20coli%20K-12%3A%20a%20cooperatively%20developed%20annotation%20snapshot%E2%80%942005&amp;author=M.%20Riley&amp;author=T.%20Abe&amp;author=M.%20B.%20Arnaud&amp;publication_year=2006&amp;journal=Nucleic%20Acids%20Res.&amp;volume=34&amp;pages=1-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkj405" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkj405" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkj405"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16397293" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Johnsson</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Heldin</div> <div class="given-names">C. H.</div></div>,  <div class="name"><div class="surname">Wasteson</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">The c-sis gene encodes a precursor of the B chain of platelet-derived growth factor</div>, <div class="source">EMBO J.</div> , <div class="year">1984</div>, vol. <div class="volume">3</div> (pg. <div class="fpage">921</div>-<div class="lpage">928</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20c-sis%20gene%20encodes%20a%20precursor%20of%20the%20B%20chain%20of%20platelet-derived%20growth%20factor&amp;author=A.%20Johnsson&amp;author=C.%20H.%20Heldin&amp;author=A.%20Wasteson&amp;publication_year=1984&amp;journal=EMBO%20J.&amp;volume=3&amp;pages=921-928" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6329745" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Altschul</div> <div class="given-names">S. F.</div></div>,  <div class="name"><div class="surname">Gish</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Miller</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Myers</div> <div class="given-names">E. W.</div></div>,  <div class="name"><div class="surname">Lipman</div> <div class="given-names">D. J.</div></div>. </div><div class="article-title">Basic local alignment search tool</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">1990</div>, vol. <div class="volume">215</div> (pg. <div class="fpage">403</div>-<div class="lpage">410</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Basic%20local%20alignment%20search%20tool&amp;author=S.%20F.%20Altschul&amp;author=W.%20Gish&amp;author=W.%20Miller&amp;author=E.%20W.%20Myers&amp;author=D.%20J.%20Lipman&amp;publication_year=1990&amp;journal=J.%20Mol.%20Biol.&amp;volume=215&amp;pages=403-410" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0022-2836(05)80360-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0022-2836(05)80360-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0022-2836(05)80360-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2231712" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mittenhuber</div> <div class="given-names">G.</div></div>. </div><div class="article-title">Comparative genomics of prokaryotic GTP-binding proteins [the Era, Obg, EngA, ThdF (TrmE), YchF and YihA families] and their relationship to eukaryotic GTP-binding proteins (the DRG, ARF, RAB, RAN, RAS and RHO families)</div>, <div class="source">J. Mol. Microbiol. Biotechnol.</div> , <div class="year">2001</div>, vol. <div class="volume">3</div> (pg. <div class="fpage">21</div>-<div class="lpage">35</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20genomics%20of%20prokaryotic%20GTP-binding%20proteins%20%5Bthe%20Era%2C%20Obg%2C%20EngA%2C%20ThdF%20(TrmE)%2C%20YchF%20and%20YihA%20families%5D%20and%20their%20relationship%20to%20eukaryotic%20GTP-binding%20proteins%20(the%20DRG%2C%20ARF%2C%20RAB%2C%20RAN%2C%20RAS%20and%20RHO%20families)&amp;author=G.%20Mittenhuber&amp;publication_year=2001&amp;journal=J.%20Mol.%20Microbiol.%20Biotechnol.&amp;volume=3&amp;pages=21-35" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11200227" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Konstantinidis</div> <div class="given-names">K. T.</div></div>,  <div class="name"><div class="surname">Tiedje</div> <div class="given-names">J. M.</div></div>. </div><div class="article-title">Towards a genome-based taxonomy for prokaryotes</div>, <div class="source">J. Bacteriol.</div> , <div class="year">2005</div>, vol. <div class="volume">187</div> (pg. <div class="fpage">6258</div>-<div class="lpage">6264</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Towards%20a%20genome-based%20taxonomy%20for%20prokaryotes&amp;author=K.%20T.%20Konstantinidis&amp;author=J.%20M.%20Tiedje&amp;publication_year=2005&amp;journal=J.%20Bacteriol.&amp;volume=187&amp;pages=6258-6264" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.187.18.6258-6264.2005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.187.18.6258-6264.2005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.187.18.6258-6264.2005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16159757" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lubec</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Afjehi</div> <div class="given-names">S. L.</div></div>,  <div class="name"><div class="surname">Yang</div> <div class="given-names">J. W.</div></div>,  <div class="name"><div class="surname">John</div> <div class="given-names">J. P.</div></div>. </div><div class="article-title">Searching for hypothetical proteins: theory and practice based upon original data and literature</div>, <div class="source">Prog. Neurobiol.</div> , <div class="year">2005</div>, vol. <div class="volume">77</div> (pg. <div class="fpage">90</div>-<div class="lpage">127</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Searching%20for%20hypothetical%20proteins%3A%20theory%20and%20practice%20based%20upon%20original%20data%20and%20literature&amp;author=G.%20Lubec&amp;author=S.%20L.%20Afjehi&amp;author=J.%20W.%20Yang&amp;author=J.%20P.%20John&amp;publication_year=2005&amp;journal=Prog.%20Neurobiol.&amp;volume=77&amp;pages=90-127" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.pneurobio.2005.10.001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.pneurobio.2005.10.001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.pneurobio.2005.10.001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16271823" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tavares</div> <div class="given-names">A. H.</div></div>,  <div class="name"><div class="surname">Silva</div> <div class="given-names">S. S.</div></div>,  <div class="name"><div class="surname">Bernardes</div> <div class="given-names">V. V.</div></div>, et al. </div><div class="article-title">Virulence insights from the Paracoccidioides brasiliensis transcriptome</div>, <div class="source">Genet. Mol. Res.</div> , <div class="year">2005</div>, vol. <div class="volume">4</div> (pg. <div class="fpage">372</div>-<div class="lpage">389</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Virulence%20insights%20from%20the%20Paracoccidioides%20brasiliensis%20transcriptome&amp;author=A.%20H.%20Tavares&amp;author=S.%20S.%20Silva&amp;author=V.%20V.%20Bernardes&amp;publication_year=2005&amp;journal=Genet.%20Mol.%20Res.&amp;volume=4&amp;pages=372-389" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16110452" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Roberts</div> <div class="given-names">R. J.</div></div>. </div><div class="article-title">Identifying protein function—a call for community action</div>, <div class="source">PLoS Biol.</div> , <div class="year">2004</div>, vol. <div class="volume">2</div> pg. <div class="fpage">E42</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identifying%20protein%20function%E2%80%94a%20call%20for%20community%20action&amp;author=R.%20J.%20Roberts&amp;publication_year=2004&amp;journal=PLoS%20Biol.&amp;volume=2&amp;pages=E42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pbio.0020042" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pbio.0020042" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pbio.0020042"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15024411" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Elphick</div> <div class="given-names">M. R.</div></div>,  <div class="name"><div class="surname">Egertova</div> <div class="given-names">M.</div></div>. </div><div class="article-title">The phylogenetic distribution and evolutionary origins of endocannabinoid signalling</div>, <div class="source">Handb. Exp. Pharmacol.</div> , <div class="year">2005</div>(pg. <div class="fpage">283</div>-<div class="lpage">297</div>)</div></div></div></div><div content-id="b17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Raskin</div> <div class="given-names">D. M.</div></div>,  <div class="name"><div class="surname">Seshadri</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Pukatzki</div> <div class="given-names">S. U.</div></div>,  <div class="name"><div class="surname">Mekalanos</div> <div class="given-names">J. J.</div></div>. </div><div class="article-title">Bacterial genomics and pathogen evolution</div>, <div class="source">Cell</div> , <div class="year">2006</div>, vol. <div class="volume">124</div> (pg. <div class="fpage">703</div>-<div class="lpage">714</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bacterial%20genomics%20and%20pathogen%20evolution&amp;author=D.%20M.%20Raskin&amp;author=R.%20Seshadri&amp;author=S.%20U.%20Pukatzki&amp;author=J.%20J.%20Mekalanos&amp;publication_year=2006&amp;journal=Cell&amp;volume=124&amp;pages=703-714" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2006.02.002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2006.02.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2006.02.002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16497582" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fraser-Liggett</div> <div class="given-names">C. M.</div></div>. </div><div class="article-title">Insights on biology and evolution from microbial genome sequencing</div>, <div class="source">Genome Res.</div> , <div class="year">2005</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">1603</div>-<div class="lpage">1610</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Insights%20on%20biology%20and%20evolution%20from%20microbial%20genome%20sequencing&amp;author=C.%20M.%20Fraser-Liggett&amp;publication_year=2005&amp;journal=Genome%20Res.&amp;volume=15&amp;pages=1603-1610" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.3724205" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.3724205" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.3724205"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16339357" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hotopp</div> <div class="given-names">J. C.</div></div>,  <div class="name"><div class="surname">Lin</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Madupu</div> <div class="given-names">R.</div></div>, et al. </div><div class="article-title">Comparative genomics of emerging human ehrlichiosis agents</div>, <div class="source">PLoS Genet.</div> , <div class="year">2006</div>, vol. <div class="volume">2</div> pg. <div class="fpage">e21</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20genomics%20of%20emerging%20human%20ehrlichiosis%20agents&amp;author=J.%20C.%20Hotopp&amp;author=M.%20Lin&amp;author=R.%20Madupu&amp;publication_year=2006&amp;journal=PLoS%20Genet.&amp;volume=2&amp;pages=e21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.0020021" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.0020021" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.0020021"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16482227" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lowe</div> <div class="given-names">T. M.</div></div>,  <div class="name"><div class="surname">Eddy</div> <div class="given-names">S. R.</div></div>. </div><div class="article-title">tRNAscan-SE: a program for improved detection of transfer RNA genes in genomic sequence</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1997</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">955</div>-<div class="lpage">964</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=tRNAscan-SE%3A%20a%20program%20for%20improved%20detection%20of%20transfer%20RNA%20genes%20in%20genomic%20sequence&amp;author=T.%20M.%20Lowe&amp;author=S.%20R.%20Eddy&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res.&amp;volume=25&amp;pages=955-964" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.5.0955" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.5.0955" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.5.0955"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9023104" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Griffiths</div> <div class="given-names">J. S</div></div>,  <div class="name"><div class="surname">Bateman</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Marshall</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Khanna</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Eddy</div> <div class="given-names">S. R.</div></div>. </div><div class="article-title">Rfam: an RNA family database</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2003</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">439</div>-<div class="lpage">441</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rfam%3A%20an%20RNA%20family%20database&amp;author=J.%20S%20Griffiths&amp;author=A.%20Bateman&amp;author=M.%20Marshall&amp;author=A.%20Khanna&amp;author=S.%20R.%20Eddy&amp;publication_year=2003&amp;journal=Nucleic%20Acids%20Res.&amp;volume=31&amp;pages=439-441" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkg006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkg006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkg006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12520045" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Salzberg</div> <div class="given-names">S. L.</div></div>,  <div class="name"><div class="surname">Delcher</div> <div class="given-names">A. L.</div></div>,  <div class="name"><div class="surname">Kasif</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">White</div> <div class="given-names">O.</div></div>. </div><div class="article-title">Microbial gene identification using interpolated Markov models</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1998</div>, vol. <div class="volume">26</div> (pg. <div class="fpage">544</div>-<div class="lpage">548</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microbial%20gene%20identification%20using%20interpolated%20Markov%20models&amp;author=S.%20L.%20Salzberg&amp;author=A.%20L.%20Delcher&amp;author=S.%20Kasif&amp;author=O.%20White&amp;publication_year=1998&amp;journal=Nucleic%20Acids%20Res.&amp;volume=26&amp;pages=544-548" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/26.2.544" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F26.2.544" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F26.2.544"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9421513" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Delcher</div> <div class="given-names">A. L.</div></div>,  <div class="name"><div class="surname">Harmon</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Kasif</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">White</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Salzberg</div> <div class="given-names">S. L.</div></div>. </div><div class="article-title">Improved microbial gene identification with GLIMMER</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1999</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">4636</div>-<div class="lpage">4641</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Improved%20microbial%20gene%20identification%20with%20GLIMMER&amp;author=A.%20L.%20Delcher&amp;author=D.%20Harmon&amp;author=S.%20Kasif&amp;author=O.%20White&amp;author=S.%20L.%20Salzberg&amp;publication_year=1999&amp;journal=Nucleic%20Acids%20Res.&amp;volume=27&amp;pages=4636-4641" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/27.23.4636" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F27.23.4636" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F27.23.4636"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10556321" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Aggarwal</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Ramaswamy</div> <div class="given-names">R.</div></div>. </div><div class="article-title"><em>Ab initio</em> gene identification: prokaryote genome annotation with GeneScan and GLIMMER</div>, <div class="source">J. Biosci.</div> , <div class="year">2002</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">7</div>-<div class="lpage">14</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ab%20initio%20gene%20identification%3A%20prokaryote%20genome%20annotation%20with%20GeneScan%20and%20GLIMMER&amp;author=G.%20Aggarwal&amp;author=R.%20Ramaswamy&amp;publication_year=2002&amp;journal=J.%20Biosci.&amp;volume=27&amp;pages=7-14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF02703679" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF02703679" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF02703679"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11927773" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bairoch</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Apweiler</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Wu</div> <div class="given-names">C.H.</div></div>, et al. </div><div class="article-title">The Universal Protein Resource (UniProt)</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2005</div>, vol. <div class="volume">33</div> (pg. <div class="fpage">D154</div>-<div class="lpage">D159</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Universal%20Protein%20Resource%20(UniProt)&amp;author=A.%20Bairoch&amp;author=R.%20Apweiler&amp;author=C.H.%20Wu&amp;publication_year=2005&amp;journal=Nucleic%20Acids%20Res.&amp;volume=33&amp;pages=D154-D159" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gki070" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgki070" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgki070"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15608167" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Suzek</div> <div class="given-names">B. E.</div></div>,  <div class="name"><div class="surname">Ermolaeva</div> <div class="given-names">M. D.</div></div>,  <div class="name"><div class="surname">Schreiber</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Salzberg</div> <div class="given-names">S. L.</div></div>. </div><div class="article-title">A probabilistic method for identifying start codons in bacterial genomes</div>, <div class="source">Bioinformatics</div> , <div class="year">2001</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">1123</div>-<div class="lpage">1130</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20probabilistic%20method%20for%20identifying%20start%20codons%20in%20bacterial%20genomes&amp;author=B.%20E.%20Suzek&amp;author=M.%20D.%20Ermolaeva&amp;author=M.%20Schreiber&amp;author=S.%20L.%20Salzberg&amp;publication_year=2001&amp;journal=Bioinformatics&amp;volume=17&amp;pages=1123-1130" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/17.12.1123" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2F17.12.1123" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2F17.12.1123"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11751220" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kunst</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Ogasawara</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Moszer</div> <div class="given-names">I.</div></div>, et al. </div><div class="article-title">The complete genome sequence of the gram-positive bacterium <em>Bacillus subtilis</em></div>, <div class="source">Nature</div> , <div class="year">1997</div>, vol. <div class="volume">390</div> (pg. <div class="fpage">249</div>-<div class="lpage">256</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20complete%20genome%20sequence%20of%20the%20gram-positive%20bacterium%20Bacillus%20subtilis&amp;author=F.%20Kunst&amp;author=N.%20Ogasawara&amp;author=I.%20Moszer&amp;publication_year=1997&amp;journal=Nature&amp;volume=390&amp;pages=249-256" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/36786" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F36786" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F36786"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9384377" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ouyang</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Zhu</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">She</div> <div class="given-names">S. Z.</div></div>. </div><div class="article-title">Multivariate entropy distance method for prokaryotic gene identification</div>, <div class="source">J. Bioinformatics and Comp. Biol.</div> , <div class="year">2004</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">353</div>-<div class="lpage">373</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Multivariate%20entropy%20distance%20method%20for%20prokaryotic%20gene%20identification&amp;author=Z.%20Ouyang&amp;author=H.%20Zhu&amp;author=J.%20Wang&amp;author=S.%20Z.%20She&amp;publication_year=2004&amp;journal=J.%20Bioinformatics%20and%20Comp.%20Biol.&amp;volume=2&amp;pages=353-373" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1142/S0219720004000624" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1142%2FS0219720004000624" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1142%2FS0219720004000624"> </span></div> </div></div></div></div></div><div content-id="b29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Apweiler</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Attwood</div> <div class="given-names">T. K.</div></div>,  <div class="name"><div class="surname">Bairoch</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">The InterPro database, an integrated documentation resource for protein families, domains and functional sites</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2001</div>, vol. <div class="volume">29</div> (pg. <div class="fpage">37</div>-<div class="lpage">40</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20InterPro%20database%2C%20an%20integrated%20documentation%20resource%20for%20protein%20families%2C%20domains%20and%20functional%20sites&amp;author=R.%20Apweiler&amp;author=T.%20K.%20Attwood&amp;author=A.%20Bairoch&amp;publication_year=2001&amp;journal=Nucleic%20Acids%20Res.&amp;volume=29&amp;pages=37-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/29.1.37" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F29.1.37" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F29.1.37"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11125043" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kawabata</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Fukuchi</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Homma</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">GTOP: a database of protein structures predicted from genome sequences</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2002</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">294</div>-<div class="lpage">298</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GTOP%3A%20a%20database%20of%20protein%20structures%20predicted%20from%20genome%20sequences&amp;author=T.%20Kawabata&amp;author=S.%20Fukuchi&amp;author=K.%20Homma&amp;publication_year=2002&amp;journal=Nucleic%20Acids%20Res.&amp;volume=30&amp;pages=294-298" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/30.1.294" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F30.1.294" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F30.1.294"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11752318" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hayashi</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Morooka</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Yamamoto</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">Highly accurate genome sequences of <em>Escherichia coli</em> K-12 strains MG1655 and W3110</div>, <div class="source">Mol. Syst. Biol.</div> , <div class="year">2006</div>, vol. <div class="volume">2</div> pg. <div class="fpage">0007</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Highly%20accurate%20genome%20sequences%20of%20Escherichia%20coli%20K-12%20strains%20MG1655%20and%20W3110&amp;author=K.%20Hayashi&amp;author=N.%20Morooka&amp;author=Y.%20Yamamoto&amp;publication_year=2006&amp;journal=Mol.%20Syst.%20Biol.&amp;volume=2&amp;pages=0007" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/msb4100049" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fmsb4100049" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fmsb4100049"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16738553" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Jaroszewski</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Godzik</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Clustering of highly homologous sequences to reduce the size of large protein databases</div>, <div class="source">Bioinformatics</div> , <div class="year">2001</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">282</div>-<div class="lpage">283</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Clustering%20of%20highly%20homologous%20sequences%20to%20reduce%20the%20size%20of%20large%20protein%20databases&amp;author=W.%20Li&amp;author=L.%20Jaroszewski&amp;author=A.%20Godzik&amp;publication_year=2001&amp;journal=Bioinformatics&amp;volume=17&amp;pages=282-283" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/17.3.282" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2F17.3.282" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2F17.3.282"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11294794" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Jaroszewski</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Godzik</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Tolerating some redundancy significantly speeds up clustering of large protein databases</div>, <div class="source">Bioinformatics</div> , <div class="year">2002</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">77</div>-<div class="lpage">82</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Tolerating%20some%20redundancy%20significantly%20speeds%20up%20clustering%20of%20large%20protein%20databases&amp;author=W.%20Li&amp;author=L.%20Jaroszewski&amp;author=A.%20Godzik&amp;publication_year=2002&amp;journal=Bioinformatics&amp;volume=18&amp;pages=77-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/18.1.77" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2F18.1.77" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2F18.1.77"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11836214" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Venter</div> <div class="given-names">J. C.</div></div>,  <div class="name"><div class="surname">Remington</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Heidelberg</div> <div class="given-names">J. F.</div></div>, et al. </div><div class="article-title">Environmental genome shotgun sequencing of the Sargasso Sea</div>, <div class="source">Science</div> , <div class="year">2004</div>, vol. <div class="volume">304</div> (pg. <div class="fpage">66</div>-<div class="lpage">74</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Environmental%20genome%20shotgun%20sequencing%20of%20the%20Sargasso%20Sea&amp;author=J.%20C.%20Venter&amp;author=K.%20Remington&amp;author=J.%20F.%20Heidelberg&amp;publication_year=2004&amp;journal=Science&amp;volume=304&amp;pages=66-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1093857" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1093857" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1093857"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15001713" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Margulies</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Egholm</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Altman</div> <div class="given-names">W. E.</div></div>, et al. </div><div class="article-title">Genome sequencing in microfabricated high-density picolitre reactors</div>, <div class="source">Nature</div> , <div class="year">2005</div>, vol. <div class="volume">437</div> (pg. <div class="fpage">376</div>-<div class="lpage">380</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20sequencing%20in%20microfabricated%20high-density%20picolitre%20reactors&amp;author=M.%20Margulies&amp;author=M.%20Egholm&amp;author=W.%20E.%20Altman&amp;publication_year=2005&amp;journal=Nature&amp;volume=437&amp;pages=376-380" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16056220" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2006. Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">The online version of this article has been published under an open access model. Users are entitled to use, reproduce, disseminate, or display the open access version of this article for non-commercial purposes provided that: the original authorship is properly and fully attributed; the Journal and Oxford University Press are attributed as the original place of publication with the correct citation details given; if an article is subsequently reproduced or disseminated not in its entirety but only in part or as a derivative work this must be clearly indicated. For commercial re-use, please contact journals.permissions@oxfordjournals.org</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=464398&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/6/10.1093/dnares/dsl014/2/dsl014-dsl014__SupplementaryTable3.xls?Expires=2147483647&amp;Signature=oPhx-zs7SHK93j-UCO8rjl-8ZeKPUlpNdfpfKzSR-7qYTQXJ6oEXh3MAA7BjL0G8H8yzVboAin4LENqnvwmaE9Dxgl6YxGT~FdnB3kNi1837A1zJEd6zvf2rZsrHa0e0js0A68wT3h-ZZdRhFQI8kfB2FTgDMGGfTYKwKsoT-tuHK5xH9Kb7XT9skDTs-b-IbKwJCtXQwvSRLSGkj3CL5IHnJOprq9u3w4pf2wIQsWPObdJhXx4Mys325LDry--3Uor7GhVrhCyubfy8aiaSZMwE3VGT5E5gIclU5lGl5asotIR~vjct16wop8kUIUWeIomMi6PKPqeClUyQm4dRHA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-8</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/6/10.1093/dnares/dsl014/2/dsl014-dsl014__SupplementaryTable2.xls?Expires=2147483647&amp;Signature=m6~7nG6X3cswoEhUL~bOUJ5NV1L~NcZr7Yff950YYuiTAlrOnNB34GHiBeoVf6c70a6bxpCmqnJZ8WpxxqWQoW5y3L5mcjXULt7KtjpGu7kpnHXHKeb8KZMy5aMH5wh8jSjlS4UMa4-RUmPccrAaRdFt0K2knTsYX0jl8Rt-K0SswbJwnFwxkT65LNmp2GnvnzRtgJQgq-HLaBK0lXiijByfsOyfPcZhJLIo6gAnNcd3PzF9DSOYPtEV364CaffYk-o3hMvDUf12adGRtIOM46NV-YC4QUO10b5X6od3GDRkpFddMmInQegwTqLKBFEvl36YW8q71dW99acE46Xk9Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-7</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/6/10.1093/dnares/dsl014/2/dsl014-dsl014__SupplementaryTable1.doc?Expires=2147483647&amp;Signature=ZOKuEubH6pBHrKku8~qbhsPiNHhGtVlX5eD99eA52-YvwTpcD0iw7skgJpwl5SceI2rL4UsgV5YKcN0E3o~-bfvJFqr1fj0fGJdPAuUeJGVHgD6hHzutWcseoxsn0oSq5SLhAthTUq7XeoOui48byj9awJ9X6Hn9RzLbt8g9lTn5-lF1QSau3lPGmwTbnuHGEo3dZzaX146Mp4NsxuuWad0cas-X0Qx3xVdCkDuhVL5zN51Zv-yDKQcGjUyZKOYRdfBvFLinutNXD8hgw0nSIo97jbR5KKxxSKzDFSnZUaaWxqkzieoSTla8Iv7vFQCg~aKtz5xZdjMp3lnvCZ0FtQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-6</a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/6/10.1093/dnares/dsl014/2/dsl014-dsl014_SupplementaryTable_6.xls?Expires=2147483647&amp;Signature=r25cjqrfH3Q~BGQ63XWEbzhrCww0GgolWSddqM2fzxGdNFaRXXtbjAEFRu8ULTnobw7xlgokYTkKN61Bi00GbFatSVLz4GLHtsnyS~MJgbtmwxg18frC3mZWQUTj7HWr8KO7OfVtTdyi4Lqnsxi6vDWkSdbUvLUlN0NdosHJcx2Lb-AngYzj9FH2O1TnQr~lUKdzbxaSI8GOkVyuCiOobN7mTk7zXXS9uFBTlqSU1Ba4E0rpYYExOzl~zhuhHgPMiqacZZzxFZEzns9dlAT0ye-OcPRS7YBOzFUjPzhfKXZdEpokgwm5W6ViEuVdZSHgWhfkJYsY23Atg6wLBfJjSA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-5</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/6/10.1093/dnares/dsl014/2/dsl014-dsl014_SupplementaryTable7.xls?Expires=2147483647&amp;Signature=2yJr1UldZBzeG2sNQU0qU5dS3eHLjzbb47sNVZPuGauQXnCPeg2qSXRe4KmVZNUUlV3aD1GvyIzUvWYfi4~0RXs8FCv6RhEmFYkTH6TqB7zZJnr2yp8ww-BordFIix5kfIfA7GwfTQnByyrmT6f1lxHSwYZmHIZTuYmu7g0sKWnwzXnlhW2da7ZSpds9FbmAXyzeVFtRkyWgnD6W3ja7hiHKj5ALUCCrJ4nZ37qJ0GvWrqU8J6QFVIDxaYs427D~POrRLJ11R9ob9roeifqMwM0Vg2AsbWMHwrDRvTi2YbFQEY-LV5mfzqshJpqR51gc2Kz5Cfm8ovxyAlJtWYOsDg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-4</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/6/10.1093/dnares/dsl014/2/dsl014-dsl014_Supplementary8.pdf?Expires=2147483647&amp;Signature=bT9jzFaur978NWOlLl8nEDhKXwAAA93YL0t1vLJgDQ-WQkCH2DXQAIX63RDvQ2JHy~gEaEedZcfoHb8YhbcT9qBLqwAeBO3B4pPxaH3zIukkhnAKAHmuFrWvzGPMJWHrNhm0lUJU5p60ckOoyQsesALu-TCRiLgj8NhTk1kgmMuZtQg5vQD1lBMxxRkBka3~fB4mlMyM9lsjxlQKGMvjrS~V5ud2x4tM0toOsqinlrS774MMhxCsFwkMPSUxwDN~erMk5BwH4V7BcDCQwDtB3GppdStsfwu4eu8g8xoMTO3Ig12CeORDWWOdF8PBzblRCRwQ92r0MzBUZbjA9-KxRQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/6/10.1093/dnares/dsl014/2/dsl014-dsl014_Supplementary5.doc?Expires=2147483647&amp;Signature=IJycv1D0tYkkfE31AvLeHC2QgAkPM0~cvpGS3DmVLxSqKGEnHZKRR4AWYWoCE-gtkCc0OS6vUsETAAk4pJRXXvyHroxE5aJm0t6-oLLK78kEQTP~Bn810d1FMf9QQtFtotzTRerMrKhu-j~-mQFtOuymRJ9-3rEC3QH-Dj6PDAACjH-VgptAMVJOIjsLwDDAOyubou4JFv-VVAU28RvcDZZK-6XjpFImT~8RjddLQ43r60vvK7S5ZffnOlKxiYZ4CkrE4zgp-6veXZnEB~1EdGrIFq0ZwEAJTLyeFKN1gZvCsVnT6x0il0GJ46BG1vRD5OGYdCnKhmj8qxlWuNFRYQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/6/10.1093/dnares/dsl014/2/dsl014-dsl014_Supplementary4.doc?Expires=2147483647&amp;Signature=HZOuB1jj31Cxa2iTzfKDiD-9pBvQhos1KQSjFy62qkBoMoLWAIMwbnyI8qeSVmL42GXjPsJhhqA0J4bdYIVDCEDdwZz06~XtVnwG~nnJDBb67ykNcnzAfgh2FZ8o5f2kvk1MBnXT0uIvqczSDK2WDeaKlInVw1emOIXEhElOJqAWspVxicZo9J8b7TmwMQDkqrNtdOXkSY2NJZkrh7sB2LqnI1bbL3E3YPKFwsfAYUX4XAA~HopKRN4zjyLm0t1nXKLZhyUJAsWZwLbKVySMUJDc0-cLOAzz4EAv~9qKnBCwFoVa9Y4SMLtf~~lBzX01fDPafw3KatPfX5bRY3YGVQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - doc file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">164</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000244930400001&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">13</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsl014"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="464398"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000244930400001&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f13%2f6%2f245%2f464398">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000244930400001&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (13)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F13%2F6%2F245%2F464398" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/464398'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Exploration and Grading of Possible Genes from 183 Bacterial Strains by a Common Protocol to Identification of New Genes: Gene Trek in Prokaryote Space (GTPS)" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsl014");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f13%2f6%2f245%2f464398">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

