

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/20/1/17/382150","@type":"ScholarlyArticle","name":"A Simple Sequence Repeat- and Single-Nucleotide Polymorphism-Based Genetic Linkage Map of the Brown Planthopper,  Nilaparvata lugens ","datePublished":"2012-11-29","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/20/1","@type":"PublicationIssue","issueNumber":"1","datePublished":"2013-02-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dss030","keywords":["<italic>Nilaparvata lugens</italic>","brown planthopper","genetic linkage map","SSR","SNP"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/20/1/17/382150","author":[{"name":"Jairin, Jirapong","affiliation":"1  Ubon Ratchathani Rice Research Center ,  PO Box 65, Muang, Ubon Ratchathani 34000 ,  Thailand    2  Plant Breeding Laboratory, Graduate School, Faculty of Agriculture,  Kyushu University ,  6-10-1, Hakozaki, Higashi-ku, Fukuoka 812-8581 ,  Japan","@type":"Person"},{"name":"Kobayashi, Tetsuya","affiliation":"3  National Institute of Agrobiological Sciences ,  1-2, O-washi, Tsukuba, Ibaraki 305-8634 ,  Japan","@type":"Person"},{"name":"Yamagata, Yoshiyuki","affiliation":"2  Plant Breeding Laboratory, Graduate School, Faculty of Agriculture,  Kyushu University ,  6-10-1, Hakozaki, Higashi-ku, Fukuoka 812-8581 ,  Japan","@type":"Person"},{"name":"Sanada-Morimura, Sachiyo","affiliation":"4  Kyushu Okinawa Agricultural Research Center ,  National Agriculture and Food Research Organization ,  Kumamoto 861-1192 ,  Japan","@type":"Person"},{"name":"Mori, Kazuki","affiliation":"5  Laboratory of Molecular Gene Technology, Division of Molecular Biosciences, Department of Bioscience and Biotechnology, Faculty of Agriculture ,  Kyushu University ,  6-10-1 Hakozaki, Higashi-ku, Fukuoka 812-8581 ,  Japan","@type":"Person"},{"name":"Tashiro, Kosuke","affiliation":"5  Laboratory of Molecular Gene Technology, Division of Molecular Biosciences, Department of Bioscience and Biotechnology, Faculty of Agriculture ,  Kyushu University ,  6-10-1 Hakozaki, Higashi-ku, Fukuoka 812-8581 ,  Japan","@type":"Person"},{"name":"Kuhara, Satoru","affiliation":"5  Laboratory of Molecular Gene Technology, Division of Molecular Biosciences, Department of Bioscience and Biotechnology, Faculty of Agriculture ,  Kyushu University ,  6-10-1 Hakozaki, Higashi-ku, Fukuoka 812-8581 ,  Japan","@type":"Person"},{"name":"Kuwazaki, Seigo","affiliation":"3  National Institute of Agrobiological Sciences ,  1-2, O-washi, Tsukuba, Ibaraki 305-8634 ,  Japan","@type":"Person"},{"name":"Urio, Masahiro","affiliation":"3  National Institute of Agrobiological Sciences ,  1-2, O-washi, Tsukuba, Ibaraki 305-8634 ,  Japan","@type":"Person"},{"name":"Suetsugu, Yoshitaka","affiliation":"3  National Institute of Agrobiological Sciences ,  1-2, O-washi, Tsukuba, Ibaraki 305-8634 ,  Japan","@type":"Person"},{"name":"Yamamoto, Kimiko","affiliation":"3  National Institute of Agrobiological Sciences ,  1-2, O-washi, Tsukuba, Ibaraki 305-8634 ,  Japan","@type":"Person"},{"name":"Matsumura, Masaya","affiliation":"4  Kyushu Okinawa Agricultural Research Center ,  National Agriculture and Food Research Organization ,  Kumamoto 861-1192 ,  Japan","@type":"Person"},{"name":"Yasui, Hideshi","affiliation":"2  Plant Breeding Laboratory, Graduate School, Faculty of Agriculture,  Kyushu University ,  6-10-1, Hakozaki, Higashi-ku, Fukuoka 812-8581 ,  Japan","@type":"Person"}],"description":"Abstract.  In this study, we developed the first genetic linkage map for the major rice insect pest, the brown planthopper (BPH, Nilaparvata lugens). The linkag","pageStart":"17","pageEnd":"30","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/1/1/m_cover.gif?Expires=2147483647&Signature=K1v8cGQnstZYdECB9GA3oc5Ko3ohhQ9lF83XAKOVlOLlyAqR0JX0Vn18cJ3Fh6Ni4coRVii1nVVwFuzd9baa8EpiNPdDbUxg77BFyv3S5jO4sVroBj5PfHgZd5D14mPzL5KK6FHcweE5FD2RLiGEj~ct-3TIgAM5vP7r7dob4djMEW-dmZ2mnE0M4g6gvpwd4meZtXx6YM6PJ-qadpYPQyOJp7E7APzkRfsaUptwFx9azq6qh9BweylqnGS-k05la9PhL7T57M1047efztU-bcJDc0lmA7wiGh4dEYA5ldVPoM0OpeZ~9-tJ2gANZpruMQldUUZX51aSqOwvf70tSQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"A Simple Sequence Repeat- and Single-Nucleotide Polymorphism-Based Genetic Linkage Map of the Brown Planthopper,  Nilaparvata lugens ","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/1/1/m_cover.gif?Expires=2147483647&Signature=K1v8cGQnstZYdECB9GA3oc5Ko3ohhQ9lF83XAKOVlOLlyAqR0JX0Vn18cJ3Fh6Ni4coRVii1nVVwFuzd9baa8EpiNPdDbUxg77BFyv3S5jO4sVroBj5PfHgZd5D14mPzL5KK6FHcweE5FD2RLiGEj~ct-3TIgAM5vP7r7dob4djMEW-dmZ2mnE0M4g6gvpwd4meZtXx6YM6PJ-qadpYPQyOJp7E7APzkRfsaUptwFx9azq6qh9BweylqnGS-k05la9PhL7T57M1047efztU-bcJDc0lmA7wiGh4dEYA5ldVPoM0OpeZ~9-tJ2gANZpruMQldUUZX51aSqOwvf70tSQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Simple Sequence Repeat- and Single-Nucleotide Polymorphism-Based Genetic Linkage Map of the Brown Planthopper,  Nilaparvata lugens " />
    <meta property="og:description" content="Abstract.  In this study, we developed the first genetic linkage map for the major rice insect pest, the brown planthopper (BPH, Nilaparvata lugens). The linkag" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/20/1/17/382150" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2012-11-29" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Jairin, Jirapong" /><meta name="citation_author_institution" content="Ubon Ratchathani Rice Research Center, PO Box 65, Muang, Ubon Ratchathani 34000, Thailand" /><meta name="citation_author_institution" content="Plant Breeding Laboratory, Graduate School, Faculty of Agriculture,Kyushu University, 6-10-1, Hakozaki, Higashi-ku, Fukuoka 812-8581, Japan" /><meta name="citation_author" content="Kobayashi, Tetsuya" /><meta name="citation_author_institution" content="National Institute of Agrobiological Sciences, 1-2, O-washi, Tsukuba, Ibaraki 305-8634, Japan" /><meta name="citation_author" content="Yamagata, Yoshiyuki" /><meta name="citation_author_institution" content="Plant Breeding Laboratory, Graduate School, Faculty of Agriculture,Kyushu University, 6-10-1, Hakozaki, Higashi-ku, Fukuoka 812-8581, Japan" /><meta name="citation_author" content="Sanada-Morimura, Sachiyo" /><meta name="citation_author_institution" content="Kyushu Okinawa Agricultural Research Center, National Agriculture and Food Research Organization, Kumamoto 861-1192, Japan" /><meta name="citation_author" content="Mori, Kazuki" /><meta name="citation_author_institution" content="Laboratory of Molecular Gene Technology, Division of Molecular Biosciences, Department of Bioscience and Biotechnology, Faculty of Agriculture, Kyushu University, 6-10-1 Hakozaki, Higashi-ku, Fukuoka 812-8581, Japan" /><meta name="citation_author" content="Tashiro, Kosuke" /><meta name="citation_author_institution" content="Laboratory of Molecular Gene Technology, Division of Molecular Biosciences, Department of Bioscience and Biotechnology, Faculty of Agriculture, Kyushu University, 6-10-1 Hakozaki, Higashi-ku, Fukuoka 812-8581, Japan" /><meta name="citation_author" content="Kuhara, Satoru" /><meta name="citation_author_institution" content="Laboratory of Molecular Gene Technology, Division of Molecular Biosciences, Department of Bioscience and Biotechnology, Faculty of Agriculture, Kyushu University, 6-10-1 Hakozaki, Higashi-ku, Fukuoka 812-8581, Japan" /><meta name="citation_author" content="Kuwazaki, Seigo" /><meta name="citation_author_institution" content="National Institute of Agrobiological Sciences, 1-2, O-washi, Tsukuba, Ibaraki 305-8634, Japan" /><meta name="citation_author" content="Urio, Masahiro" /><meta name="citation_author_institution" content="National Institute of Agrobiological Sciences, 1-2, O-washi, Tsukuba, Ibaraki 305-8634, Japan" /><meta name="citation_author" content="Suetsugu, Yoshitaka" /><meta name="citation_author_institution" content="National Institute of Agrobiological Sciences, 1-2, O-washi, Tsukuba, Ibaraki 305-8634, Japan" /><meta name="citation_author" content="Yamamoto, Kimiko" /><meta name="citation_author_institution" content="National Institute of Agrobiological Sciences, 1-2, O-washi, Tsukuba, Ibaraki 305-8634, Japan" /><meta name="citation_author" content="Matsumura, Masaya" /><meta name="citation_author_institution" content="Kyushu Okinawa Agricultural Research Center, National Agriculture and Food Research Organization, Kumamoto 861-1192, Japan" /><meta name="citation_author" content="Yasui, Hideshi" /><meta name="citation_author_institution" content="Plant Breeding Laboratory, Graduate School, Faculty of Agriculture,Kyushu University, 6-10-1, Hakozaki, Higashi-ku, Fukuoka 812-8581, Japan" /><meta name="citation_title" content="A Simple Sequence Repeat- and Single-Nucleotide Polymorphism-Based Genetic Linkage Map of the Brown Planthopper, Nilaparvata lugens" /><meta name="citation_firstpage" content="17" /><meta name="citation_lastpage" content="30" /><meta name="citation_doi" content="10.1093/dnares/dss030" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="20" /><meta name="citation_issue" content="1" /><meta name="citation_publication_date" content="2013/02/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Resurrecting the ghost of green revolutions past: the brown planthopper as a recurring threat to high-yielding rice production in tropical Asia; J. Asia-Pacific Entomol.;  citation_year=2012;  citation_volume=15;  citation_pages=122-40; " /><meta name="citation_reference" content="citation_publisher=IRRI, Los Baños, Philippines; Planthoppers: New Threats to the Sustainability of Intensive Rice Production Systems in Asia;  citation_year=2009;  citation_pages=191-220; " /><meta name="citation_reference" content="citation_title=Are planthopper problems caused by a breakdown in ecosystem services?; citation_publisher=IRRI, Los Baños, Philippines; Planthoppers: New Threats to the Sustainability of Intensive Rice Production Systems in Asia;  citation_year=2009;  citation_pages=221-31; " /><meta name="citation_reference" content="citation_title=Judicial use of insecticides deter planthopper outbreaks and extend the life of resistant varieties in Southwest Asian rice.; citation_publisher=Chapman and Hall, New York; Planthoppers: Their Ecology and Management;  citation_year=1994;  citation_pages=599-614; " /><meta name="citation_reference" content="citation_title=Rice viruses transmitted by the brown planthopper Nilaparvata lugens Stål.; citation_publisher=IRRI, Los Baños, Philippines; Planthoppers: New Threats to the Sustainability of Intensive Rice Production Systems in Asia;  citation_year=2009;  citation_pages=357-68; " /><meta name="citation_reference" content="citation_title=Status and prospects for an integrated approach to the control of rice planthoppers; citation_publisher=Chapman and Hall, New York; Planthoppers: Their Ecology and Management;  citation_year=1994;  citation_pages=615-32; " /><meta name="citation_reference" content="citation_title=Quantitative genetic analysis of biotypes of the brown planthopper Nilaparvata lugens: heritability of virulence to resistant rice varieties; Entomologia Experimentalis et Applicata;  citation_year=1999;  citation_volume=90;  citation_pages=279-87; " /><meta name="citation_reference" content="citation_title=The chromosomes of Nilaparvata lugens Stål. and some other Auchenorrhyncha; Cytologia;  citation_year=1982;  citation_volume=47;  citation_pages=227-36; " /><meta name="citation_reference" content="citation_title=Brain cells and chromosomes of the brown planthopper Nilaparvata lugens (Stål); Cytologia;  citation_year=1987;  citation_volume=52;  citation_pages=859-65; " /><meta name="citation_reference" content="citation_title=Re-examination of chromosomes of three species of rice planthoppers (Homoptera: Delphacidae); Appl. Entomol. Zool.;  citation_year=1990;  citation_volume=4;  citation_pages=538-40; " /><meta name="citation_reference" content="citation_title=Chitin synthase 1 gene and its two alternative splicing variants from two sap-sucking insects, Nilaparvata lugens and Laodelphax striatellus (Hemiptera: Delphacidae); Insect Biochem. Mol. Biol.;  citation_year=2012;  citation_volume=42;  citation_pages=637-46; " /><meta name="citation_reference" content="citation_title=Annotated ESTs from various tissues of the brown planthopper Nilaparvata lugens: a genomic resource for studying agricultural pests; BMC Genomics;  citation_year=2008;  citation_volume=9;  citation_pages=117" /><meta name="citation_reference" content="citation_title=Transcriptome analysis of the brown planthopper Nilaparvata lugens; PLoS One;  citation_year=2010;  citation_volume=5;  citation_pages=e14233" /><meta name="citation_reference" content="citation_title=Genomic resources for the brown planthopper, Nilaparvata lugens: transcriptome pyrosequencing and microarray design; Insect Sci.;  citation_year=2012;  citation_volume=9;  citation_pages=1-12; " /><meta name="citation_reference" content="citation_title=Pyrosequencing the midgut transcriptome of the brown planthopper, Nilaparvata lugens; Insect Mol. Biol.;  citation_year=2011;  citation_volume=20;  citation_pages=745-62; " /><meta name="citation_reference" content="citation_title=De novo intestine-specific transcriptome of the brown planthopper Nilaparvata lugens revealed potential functions in digestion, detoxification and immune response; Genomics;  citation_year=2012;  citation_volume=99;  citation_pages=256-64; " /><meta name="citation_reference" content="citation_title=Development and use of e-SSR markers for assessing genetic diversity in the brown planthopper (Nilaparvata lugens Stål); Bull. Entomol. Res.;  citation_year=2011;  citation_volume=102;  citation_pages=113-22; " /><meta name="citation_reference" content="citation_title=Permanent genetic resources added to molecular ecology resources database 1 January 2009–30 April 2009; citation_author=Molecular Ecology Resources Primer Development Consortium; Mol. Ecol. Resources;  citation_year=2009;  citation_volume=9;  citation_pages=1375-429; " /><meta name="citation_reference" content="citation_title=Mining and characterization of sequence tagged microsatellites from the brown planthopper Nilaparvata lugens; J. Insect Sci.;  citation_year=2011;  citation_volume=11;  citation_pages=134" /><meta name="citation_reference" content="citation_title=Isolation and characterization of microsatellite markers in brown planthopper (Nilaparvata lugens Stål); Int. J. Mol. Sci.;  citation_year=2012;  citation_volume=13;  citation_pages=9527-33; " /><meta name="citation_reference" content="citation_title=Development of virulence to resistant rice varieties in the brown planthopper, Nilaparvata lugens (Homoptera: Delphacidae), immigrating into Japan; Appl. Entomol. Zool.;  citation_year=2000;  citation_volume=35;  citation_pages=529-33; " /><meta name="citation_reference" content="citation_title=A migration analysis of the rice planthopper Nilaparvata lugens from the Philippines to East Asia with three-dimensional computer simulations; Popul. Ecol.;  citation_year=2005;  citation_volume=47;  citation_pages=143-50; " /><meta name="citation_reference" content="citation_title=A plant DNA minipreparation: version II; Plant Mol. Biol. Rep.;  citation_year=1983;  citation_volume=1;  citation_pages=19-21; " /><meta name="citation_reference" content="citation_title=Bacterial symbionts of the brown planthopper, Nilaparvata lugens (Homoptera: Delphacidae); Appl. Environ. Microbiol.;  citation_year=2010;  citation_volume=76;  citation_pages=1740-5; " /><meta name="citation_reference" content="citation_title=Polymerase and multiply-primed rolling circle amplification rapid amplification of plasmid and phage DNA using Phi29 DNA; Genome Res.;  citation_year=2001;  citation_volume=11;  citation_pages=1095-9; " /><meta name="citation_reference" content="citation_title=Isolating microsatellite DNA loci; citation_publisher=Academic Press, San Diego, CA; Methods in Enzymology;  citation_year=2005;  citation_volume=vol. 395;  citation_pages=202-22; " /><meta name="citation_reference" content="citation_title=Strategies for microsatellite isolation: a review; Mol. Ecol.;  citation_year=2002;  citation_volume=11;  citation_pages=1-16; " /><meta name="citation_reference" content="citation_title=Computational and experimental analysis of microsatellites in rice (Oryza sativa L.): frequency, length variation, transposon associations, and genetic marker potential; Genome Res.;  citation_year=2001;  citation_volume=11;  citation_pages=1441-52; " /><meta name="citation_reference" content="citation_title=Primer3 on the WWW for general users and for biologist programmers; Methods Mol. Biol.;  citation_year=2000;  citation_volume=132;  citation_pages=365-86; " /><meta name="citation_reference" content="citation_title=Aggressive assembly of pyrosequencing reads with mates; Bioinformatics;  citation_year=2008;  citation_volume=24;  citation_pages=2818-24; " /><meta name="citation_reference" content="citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform; Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=1754-60; " /><meta name="citation_reference" content="citation_title=The sequence alignment/map (SAM) format and SAM tools; Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=2078-9; " /><meta name="citation_reference" content="citation_title=Tablet–next generation sequence assembly visualization; Bioinformatics;  citation_year=2010;  citation_volume=26;  citation_pages=401-2; " /><meta name="citation_reference" content="citation_publisher=Kyazma, B.V, Wageningen, Netherlands; JoinMap®4, Software for the Calculation of Genetic Linkage Maps in Experimental Populations;  citation_year=2006; " /><meta name="citation_reference" content="citation_title=The estimation of map distances from recombination values; Ann. Eugen.;  citation_year=1944;  citation_volume=12;  citation_pages=172-5; " /><meta name="citation_reference" content="citation_title=MapChart: software for the graphical presentation of linkage maps and QTLs; J. Hered.;  citation_year=2002;  citation_volume=93;  citation_pages=77-8; " /><meta name="citation_reference" content="citation_title=A maximum likelihood method for estimating genome length using genetic linkage data; Genetics;  citation_year=1991;  citation_volume=128;  citation_pages=175-82; " /><meta name="citation_reference" content="citation_title=Isolation and characterization of microsatellites in the brown planthopper, Nilaparvata lugenes Stål; Korea J. Appl. Entomol.;  citation_year=1994;  citation_volume=43;  citation_pages=311-5; " /><meta name="citation_reference" content="citation_title=Genotype and SNP calling from next generation sequencing data; Nature Rev.;  citation_year=2011;  citation_volume=12;  citation_pages=443-51; " /><meta name="citation_reference" content="citation_title=A microsatellite linkage map of the European sea bass Dicentrarchus labrax L.; Genetics;  citation_year=2005;  citation_volume=170;  citation_pages=1821-6; " /><meta name="citation_reference" content="citation_title=A microsatellite linkage map of barramundi, Lates calcarifer; Genetics;  citation_year=2007;  citation_volume=175;  citation_pages=907-15; " /><meta name="citation_reference" content="citation_title=Two-phase analysis in consensus genetic mapping; G3;  citation_year=2012;  citation_volume=2;  citation_pages=537-49; " /><meta name="citation_reference" content="citation_title=An amplified fragment length polymorphism map of the silkworm; Genetics;  citation_year=2001;  citation_volume=157;  citation_pages=1277-84; " /><meta name="citation_reference" content="citation_title=Construction of AFLP-based genetic linkage map for Zhikong scallop, Chlamys farreri and mapping of sex-linked markers; Aquaculture;  citation_year=2005;  citation_volume=245;  citation_pages=63-73; " /><meta name="citation_reference" content="citation_title=Genetic mapping and QTL analysis of growth-related traits in the Pacific oyster; Mar. Biotechnol.;  citation_year=2012;  citation_volume=14;  citation_pages=218-26; " /><meta name="citation_reference" content="citation_title=Fixation, segregation and linkage of allozyme loci in inbred families of the Pacific oyster Crassostrea giga (Thunberg): implications for the causes of inbreeding depression; Genetics;  citation_year=1997;  citation_volume=146;  citation_pages=321-34; " /><meta name="citation_reference" content="citation_title=Sex-ratio segregation distortion associated with reproductive isolation in Drosophila; Proc. Nat. Acad. Sci. USA;  citation_year=2001;  citation_volume=98;  citation_pages=13183-8; " /><meta name="citation_reference" content="citation_title=Cheaters sometimes prosper: distortion of mendelian segregation by meiotic drive; Trends Genet.;  citation_year=1993;  citation_volume=9;  citation_pages=205-10; " /><meta name="citation_reference" content="citation_title=Estimating the effects of population size and type on the accuracy of genetic maps; Genet. Mol. Biol.;  citation_year=2006;  citation_volume=29;  citation_pages=182-92; " /><meta name="citation_reference" content="citation_title=AFLP-based genetic linkage map of the Colorado potato beetle Leptinotarsa decemlineata: sex chromosomes and a pyrethroid-resistance candidate gene; Genetics;  citation_year=2001;  citation_volume=158;  citation_pages=695-700; " /><meta name="citation_reference" content="citation_title=An AFLP-based genetic linkage map of channel catfish (Ictalurus punctatus) constructed by using an interspecific hybrid resource family; Genetics;  citation_year=2003;  citation_volume=165;  citation_pages=687-94; " /><meta name="citation_reference" content="citation_title=First-generation linkage map of the warningly colored butterfly Heliconius erato; Heredity;  citation_year=2005;  citation_volume=94;  citation_pages=408-17; " /><meta name="citation_reference" content="citation_title=A microsatellite marker linkage map of the housefly, Musca domestica: evidence for male recombination; Insect Mol. Biol.;  citation_year=2010;  citation_volume=19;  citation_pages=575-81; " /><meta name="citation_reference" content="citation_title=A microsatellite linkage map of rainbow trout (Oncorhynchus mykiss) characterized by large sex-specific differences in recombination rates; Genetics;  citation_year=2000;  citation_volume=155;  citation_pages=1331-45; " /><meta name="citation_reference" content="citation_title=A dense SNP-based linkage map for Atlantic salmon (Salmo salar) reveals extended chromosome homologies and striking differences in sex-specific recombination patterns; BMC Genomics;  citation_year=2011;  citation_volume=12;  citation_pages=615" /><meta name="citation_reference" content="citation_title=Construction of an integrated map of Haliotis diversicolor using microsatellite markers; Mar. Biotechnol.;  citation_year=2012;  citation_volume=14;  citation_pages=79-86; " /><meta name="citation_reference" content="citation_title=Simple sequence repeat-based consensus linkage map of Bombyx mori; Proc. Nat. Acad. Sci. USA;  citation_year=2005;  citation_volume=102;  citation_pages=16303-8; " /><meta name="citation_reference" content="citation_title=A microsatellite linkage map for Drosophila montana shows large variation in recombination rates, and a courtship song trait maps to an area of low recombination; J. Evol. Biol.;  citation_year=2010;  citation_volume=23;  citation_pages=518-27; " /><meta name="citation_reference" content="citation_title=A microsatellite-based linkage map of the honeybee, Apis mellifera L.; Genetics;  citation_year=2004;  citation_volume=167;  citation_pages=253-62; " /><meta name="citation_reference" content="citation_title=AFLP-based genetic linkage map for the red flour beetle (Tribolium castaneum); J. Heredity;  citation_year=2004;  citation_volume=95;  citation_pages=53-61; " /><meta name="citation_reference" content="citation_title=Virulence of long-term laboratory populations of the brown planthopper, Nilaparvata lugens (Stål), and whitebacked planthopper, Sogatella furcifera (Horváth) (Homoptera: Delphacidae), on rice differential varieties; Appl. Entomol. Zool.;  citation_year=2009;  citation_volume=44;  citation_pages=149-53; " /><meta name="citation_reference" content="citation_title=Construction of a single nucleotide polymorphism linkage map for the silkworm, Bombyx mori, based on bacterial artificial chromosome end sequences; Genetics;  citation_year=2006;  citation_volume=173;  citation_pages=151-61; " /><meta name="citation_reference" content="citation_title=Identification of Y chromosomal PCR marker and production of a selected strain for molecular sexing in the brown planthopper, Nilaparvata lugens; Arch. Insect Biochem. Physiol.;  citation_year=2007;  citation_volume=65;  citation_pages=1-10; " /><meta name="citation_reference" content="citation_title=The isolation and characterization of Drosophila yolk protein genes; Cell;  citation_year=1980;  citation_volume=21;  citation_pages=729-38; " /><meta name="citation_reference" content="citation_title=Isolation of two Drosophila melanogaster genes abundantly expressed in the ovary during vitelline membrane synthesis; Dev. Biol.;  citation_year=1984;  citation_volume=105;  citation_pages=155-65; " /><meta name="citation_reference" content="citation_title=Structural analysis of the three vitellogenin genes in Drosophila melanogaster; Nucl. Acids Res.;  citation_year=1981;  citation_volume=9;  citation_pages=1323-8; " /><meta name="citation_reference" content="citation_title=X-linkage of a vitellogenin gene in Locusta migratoria; Chromosoma;  citation_year=1983;  citation_volume=88;  citation_pages=190-3; " /><meta name="citation_reference" content="citation_title=Characterization of regions of the Caenorhabditis elegans X chromosome containing vitellogenin genes; J. Mol. Biol.;  citation_year=1986;  citation_volume=188;  citation_pages=301-12; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/20/1/17/1083835/dss030.pdf" /><meta name="description" content="Abstract.  In this study, we developed the first genetic linkage map for the major rice insect pest, the brown planthopper (BPH, Nilaparvata lugens). The linkag" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/20/1/17/382150" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/20/1/17/382150" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/20/1/17/382150");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dss030';
    </script>
<title>
	Simple Sequence Repeat- and Single-Nucleotide Polymorphism-Based Genetic Linkage Map of the Brown Planthopper, Nilaparvata lugens | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./382150" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BfymATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSmFpcmluLCBKaXJhcG9uZyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlVib24gUmF0Y2hhdGhhbmkgUmljZSBSZXNlYXJjaCBDZW50ZXIsIFBPIEJveCA2NSwgTXVhbmcsIFVib24gUmF0Y2hhdGhhbmkgMzQwMDAsIFRoYWlsYW5kIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iUGxhbnQgQnJlZWRpbmcgTGFib3JhdG9yeSwgR3JhZHVhdGUgU2Nob29sLCBGYWN1bHR5IG9mIEFncmljdWx0dXJlLEt5dXNodSBVbml2ZXJzaXR5LCA2LTEwLTEsIEhha296YWtpLCBIaWdhc2hpLWt1LCBGdWt1b2thIDgxMi04NTgxLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IktvYmF5YXNoaSwgVGV0c3V5YSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5hdGlvbmFsIEluc3RpdHV0ZSBvZiBBZ3JvYmlvbG9naWNhbCBTY2llbmNlcywgMS0yLCBPLXdhc2hpLCBUc3VrdWJhLCBJYmFyYWtpIDMwNS04NjM0LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IllhbWFnYXRhLCBZb3NoaXl1a2kiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJQbGFudCBCcmVlZGluZyBMYWJvcmF0b3J5LCBHcmFkdWF0ZSBTY2hvb2wsIEZhY3VsdHkgb2YgQWdyaWN1bHR1cmUsS3l1c2h1IFVuaXZlcnNpdHksIDYtMTAtMSwgSGFrb3pha2ksIEhpZ2FzaGkta3UsIEZ1a3Vva2EgODEyLTg1ODEsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2FuYWRhLU1vcmltdXJhLCBTYWNoaXlvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS3l1c2h1IE9raW5hd2EgQWdyaWN1bHR1cmFsIFJlc2VhcmNoIENlbnRlciwgTmF0aW9uYWwgQWdyaWN1bHR1cmUgYW5kIEZvb2QgUmVzZWFyY2ggT3JnYW5pemF0aW9uLCBLdW1hbW90byA4NjEtMTE5MiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJNb3JpLCBLYXp1a2kiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJMYWJvcmF0b3J5IG9mIE1vbGVjdWxhciBHZW5lIFRlY2hub2xvZ3ksIERpdmlzaW9uIG9mIE1vbGVjdWxhciBCaW9zY2llbmNlcywgRGVwYXJ0bWVudCBvZiBCaW9zY2llbmNlIGFuZCBCaW90ZWNobm9sb2d5LCBGYWN1bHR5IG9mIEFncmljdWx0dXJlLCBLeXVzaHUgVW5pdmVyc2l0eSwgNi0xMC0xIEhha296YWtpLCBIaWdhc2hpLWt1LCBGdWt1b2thIDgxMi04NTgxLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlRhc2hpcm8sIEtvc3VrZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkxhYm9yYXRvcnkgb2YgTW9sZWN1bGFyIEdlbmUgVGVjaG5vbG9neSwgRGl2aXNpb24gb2YgTW9sZWN1bGFyIEJpb3NjaWVuY2VzLCBEZXBhcnRtZW50IG9mIEJpb3NjaWVuY2UgYW5kIEJpb3RlY2hub2xvZ3ksIEZhY3VsdHkgb2YgQWdyaWN1bHR1cmUsIEt5dXNodSBVbml2ZXJzaXR5LCA2LTEwLTEgSGFrb3pha2ksIEhpZ2FzaGkta3UsIEZ1a3Vva2EgODEyLTg1ODEsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS3VoYXJhLCBTYXRvcnUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJMYWJvcmF0b3J5IG9mIE1vbGVjdWxhciBHZW5lIFRlY2hub2xvZ3ksIERpdmlzaW9uIG9mIE1vbGVjdWxhciBCaW9zY2llbmNlcywgRGVwYXJ0bWVudCBvZiBCaW9zY2llbmNlIGFuZCBCaW90ZWNobm9sb2d5LCBGYWN1bHR5IG9mIEFncmljdWx0dXJlLCBLeXVzaHUgVW5pdmVyc2l0eSwgNi0xMC0xIEhha296YWtpLCBIaWdhc2hpLWt1LCBGdWt1b2thIDgxMi04NTgxLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ikt1d2F6YWtpLCBTZWlnbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5hdGlvbmFsIEluc3RpdHV0ZSBvZiBBZ3JvYmlvbG9naWNhbCBTY2llbmNlcywgMS0yLCBPLXdhc2hpLCBUc3VrdWJhLCBJYmFyYWtpIDMwNS04NjM0LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlVyaW8sIE1hc2FoaXJvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTmF0aW9uYWwgSW5zdGl0dXRlIG9mIEFncm9iaW9sb2dpY2FsIFNjaWVuY2VzLCAxLTIsIE8td2FzaGksIFRzdWt1YmEsIEliYXJha2kgMzA1LTg2MzQsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU3VldHN1Z3UsIFlvc2hpdGFrYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5hdGlvbmFsIEluc3RpdHV0ZSBvZiBBZ3JvYmlvbG9naWNhbCBTY2llbmNlcywgMS0yLCBPLXdhc2hpLCBUc3VrdWJhLCBJYmFyYWtpIDMwNS04NjM0LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IllhbWFtb3RvLCBLaW1pa28iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJOYXRpb25hbCBJbnN0aXR1dGUgb2YgQWdyb2Jpb2xvZ2ljYWwgU2NpZW5jZXMsIDEtMiwgTy13YXNoaSwgVHN1a3ViYSwgSWJhcmFraSAzMDUtODYzNCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJNYXRzdW11cmEsIE1hc2F5YSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ikt5dXNodSBPa2luYXdhIEFncmljdWx0dXJhbCBSZXNlYXJjaCBDZW50ZXIsIE5hdGlvbmFsIEFncmljdWx0dXJlIGFuZCBGb29kIFJlc2VhcmNoIE9yZ2FuaXphdGlvbiwgS3VtYW1vdG8gODYxLTExOTIsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iWWFzdWksIEhpZGVzaGkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJQbGFudCBCcmVlZGluZyBMYWJvcmF0b3J5LCBHcmFkdWF0ZSBTY2hvb2wsIEZhY3VsdHkgb2YgQWdyaWN1bHR1cmUsS3l1c2h1IFVuaXZlcnNpdHksIDYtMTAtMSwgSGFrb3pha2ksIEhpZ2FzaGkta3UsIEZ1a3Vva2EgODEyLTg1ODEsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJBIFNpbXBsZSBTZXF1ZW5jZSBSZXBlYXQtIGFuZCBTaW5nbGUtTnVjbGVvdGlkZSBQb2x5bW9ycGhpc20tQmFzZWQgR2VuZXRpYyBMaW5rYWdlIE1hcCBvZiB0aGUgQnJvd24gUGxhbnRob3BwZXIsIE5pbGFwYXJ2YXRhIGx1Z2VucyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjE3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSIzMCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzczAzMCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIyMCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDEzLzAyLzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVzdXJyZWN0aW5nIHRoZSBnaG9zdCBvZiBncmVlbiByZXZvbHV0aW9ucyBwYXN0OiB0aGUgYnJvd24gcGxhbnRob3BwZXIgYXMgYSByZWN1cnJpbmcgdGhyZWF0IHRvIGhpZ2gteWllbGRpbmcgcmljZSBwcm9kdWN0aW9uIGluIHRyb3BpY2FsIEFzaWE7IEouIEFzaWEtUGFjaWZpYyBFbnRvbW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTEyMi00MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fcHVibGlzaGVyPUlSUkksIExvcyBCYcOxb3MsIFBoaWxpcHBpbmVzOyBQbGFudGhvcHBlcnM6IE5ldyBUaHJlYXRzIHRvIHRoZSBTdXN0YWluYWJpbGl0eSBvZiBJbnRlbnNpdmUgUmljZSBQcm9kdWN0aW9uIFN5c3RlbXMgaW4gQXNpYTsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3BhZ2VzPTE5MS0yMjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFyZSBwbGFudGhvcHBlciBwcm9ibGVtcyBjYXVzZWQgYnkgYSBicmVha2Rvd24gaW4gZWNvc3lzdGVtIHNlcnZpY2VzPzsgY2l0YXRpb25fcHVibGlzaGVyPUlSUkksIExvcyBCYcOxb3MsIFBoaWxpcHBpbmVzOyBQbGFudGhvcHBlcnM6IE5ldyBUaHJlYXRzIHRvIHRoZSBTdXN0YWluYWJpbGl0eSBvZiBJbnRlbnNpdmUgUmljZSBQcm9kdWN0aW9uIFN5c3RlbXMgaW4gQXNpYTsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3BhZ2VzPTIyMS0zMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SnVkaWNpYWwgdXNlIG9mIGluc2VjdGljaWRlcyBkZXRlciBwbGFudGhvcHBlciBvdXRicmVha3MgYW5kIGV4dGVuZCB0aGUgbGlmZSBvZiByZXNpc3RhbnQgdmFyaWV0aWVzIGluIFNvdXRod2VzdCBBc2lhbiByaWNlLjsgY2l0YXRpb25fcHVibGlzaGVyPUNoYXBtYW4gYW5kIEhhbGwsIE5ldyBZb3JrOyBQbGFudGhvcHBlcnM6IFRoZWlyIEVjb2xvZ3kgYW5kIE1hbmFnZW1lbnQ7ICBjaXRhdGlvbl95ZWFyPTE5OTQ7ICBjaXRhdGlvbl9wYWdlcz01OTktNjE0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SaWNlIHZpcnVzZXMgdHJhbnNtaXR0ZWQgYnkgdGhlIGJyb3duIHBsYW50aG9wcGVyIE5pbGFwYXJ2YXRhIGx1Z2VucyBTdMOlbC47IGNpdGF0aW9uX3B1Ymxpc2hlcj1JUlJJLCBMb3MgQmHDsW9zLCBQaGlsaXBwaW5lczsgUGxhbnRob3BwZXJzOiBOZXcgVGhyZWF0cyB0byB0aGUgU3VzdGFpbmFiaWxpdHkgb2YgSW50ZW5zaXZlIFJpY2UgUHJvZHVjdGlvbiBTeXN0ZW1zIGluIEFzaWE7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl9wYWdlcz0zNTctNjg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN0YXR1cyBhbmQgcHJvc3BlY3RzIGZvciBhbiBpbnRlZ3JhdGVkIGFwcHJvYWNoIHRvIHRoZSBjb250cm9sIG9mIHJpY2UgcGxhbnRob3BwZXJzOyBjaXRhdGlvbl9wdWJsaXNoZXI9Q2hhcG1hbiBhbmQgSGFsbCwgTmV3IFlvcms7IFBsYW50aG9wcGVyczogVGhlaXIgRWNvbG9neSBhbmQgTWFuYWdlbWVudDsgIGNpdGF0aW9uX3llYXI9MTk5NDsgIGNpdGF0aW9uX3BhZ2VzPTYxNS0zMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UXVhbnRpdGF0aXZlIGdlbmV0aWMgYW5hbHlzaXMgb2YgYmlvdHlwZXMgb2YgdGhlIGJyb3duIHBsYW50aG9wcGVyIE5pbGFwYXJ2YXRhIGx1Z2VuczogaGVyaXRhYmlsaXR5IG9mIHZpcnVsZW5jZSB0byByZXNpc3RhbnQgcmljZSB2YXJpZXRpZXM7IEVudG9tb2xvZ2lhIEV4cGVyaW1lbnRhbGlzIGV0IEFwcGxpY2F0YTsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT05MDsgIGNpdGF0aW9uX3BhZ2VzPTI3OS04NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGNocm9tb3NvbWVzIG9mIE5pbGFwYXJ2YXRhIGx1Z2VucyBTdMOlbC4gYW5kIHNvbWUgb3RoZXIgQXVjaGVub3JyaHluY2hhOyBDeXRvbG9naWE7ICBjaXRhdGlvbl95ZWFyPTE5ODI7ICBjaXRhdGlvbl92b2x1bWU9NDc7ICBjaXRhdGlvbl9wYWdlcz0yMjctMzY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJyYWluIGNlbGxzIGFuZCBjaHJvbW9zb21lcyBvZiB0aGUgYnJvd24gcGxhbnRob3BwZXIgTmlsYXBhcnZhdGEgbHVnZW5zIChTdMOlbCk7IEN5dG9sb2dpYTsgIGNpdGF0aW9uX3llYXI9MTk4NzsgIGNpdGF0aW9uX3ZvbHVtZT01MjsgIGNpdGF0aW9uX3BhZ2VzPTg1OS02NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmUtZXhhbWluYXRpb24gb2YgY2hyb21vc29tZXMgb2YgdGhyZWUgc3BlY2llcyBvZiByaWNlIHBsYW50aG9wcGVycyAoSG9tb3B0ZXJhOiBEZWxwaGFjaWRhZSk7IEFwcGwuIEVudG9tb2wuIFpvb2wuOyAgY2l0YXRpb25feWVhcj0xOTkwOyAgY2l0YXRpb25fdm9sdW1lPTQ7ICBjaXRhdGlvbl9wYWdlcz01MzgtNDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNoaXRpbiBzeW50aGFzZSAxIGdlbmUgYW5kIGl0cyB0d28gYWx0ZXJuYXRpdmUgc3BsaWNpbmcgdmFyaWFudHMgZnJvbSB0d28gc2FwLXN1Y2tpbmcgaW5zZWN0cywgTmlsYXBhcnZhdGEgbHVnZW5zIGFuZCBMYW9kZWxwaGF4IHN0cmlhdGVsbHVzIChIZW1pcHRlcmE6IERlbHBoYWNpZGFlKTsgSW5zZWN0IEJpb2NoZW0uIE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NDI7ICBjaXRhdGlvbl9wYWdlcz02MzctNDY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFubm90YXRlZCBFU1RzIGZyb20gdmFyaW91cyB0aXNzdWVzIG9mIHRoZSBicm93biBwbGFudGhvcHBlciBOaWxhcGFydmF0YSBsdWdlbnM6IGEgZ2Vub21pYyByZXNvdXJjZSBmb3Igc3R1ZHlpbmcgYWdyaWN1bHR1cmFsIHBlc3RzOyBCTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTExNyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zY3JpcHRvbWUgYW5hbHlzaXMgb2YgdGhlIGJyb3duIHBsYW50aG9wcGVyIE5pbGFwYXJ2YXRhIGx1Z2VuczsgUExvUyBPbmU7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPWUxNDIzMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9taWMgcmVzb3VyY2VzIGZvciB0aGUgYnJvd24gcGxhbnRob3BwZXIsIE5pbGFwYXJ2YXRhIGx1Z2VuczogdHJhbnNjcmlwdG9tZSBweXJvc2VxdWVuY2luZyBhbmQgbWljcm9hcnJheSBkZXNpZ247IEluc2VjdCBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz0xLTEyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QeXJvc2VxdWVuY2luZyB0aGUgbWlkZ3V0IHRyYW5zY3JpcHRvbWUgb2YgdGhlIGJyb3duIHBsYW50aG9wcGVyLCBOaWxhcGFydmF0YSBsdWdlbnM7IEluc2VjdCBNb2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9NzQ1LTYyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZSBub3ZvIGludGVzdGluZS1zcGVjaWZpYyB0cmFuc2NyaXB0b21lIG9mIHRoZSBicm93biBwbGFudGhvcHBlciBOaWxhcGFydmF0YSBsdWdlbnMgcmV2ZWFsZWQgcG90ZW50aWFsIGZ1bmN0aW9ucyBpbiBkaWdlc3Rpb24sIGRldG94aWZpY2F0aW9uIGFuZCBpbW11bmUgcmVzcG9uc2U7IEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTk5OyAgY2l0YXRpb25fcGFnZXM9MjU2LTY0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZXZlbG9wbWVudCBhbmQgdXNlIG9mIGUtU1NSIG1hcmtlcnMgZm9yIGFzc2Vzc2luZyBnZW5ldGljIGRpdmVyc2l0eSBpbiB0aGUgYnJvd24gcGxhbnRob3BwZXIgKE5pbGFwYXJ2YXRhIGx1Z2VucyBTdMOlbCk7IEJ1bGwuIEVudG9tb2wuIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTAyOyAgY2l0YXRpb25fcGFnZXM9MTEzLTIyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QZXJtYW5lbnQgZ2VuZXRpYyByZXNvdXJjZXMgYWRkZWQgdG8gbW9sZWN1bGFyIGVjb2xvZ3kgcmVzb3VyY2VzIGRhdGFiYXNlIDEgSmFudWFyeSAyMDA54oCTMzAgQXByaWwgMjAwOTsgY2l0YXRpb25fYXV0aG9yPU1vbGVjdWxhciBFY29sb2d5IFJlc291cmNlcyBQcmltZXIgRGV2ZWxvcG1lbnQgQ29uc29ydGl1bTsgTW9sLiBFY29sLiBSZXNvdXJjZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTEzNzUtNDI5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NaW5pbmcgYW5kIGNoYXJhY3Rlcml6YXRpb24gb2Ygc2VxdWVuY2UgdGFnZ2VkIG1pY3Jvc2F0ZWxsaXRlcyBmcm9tIHRoZSBicm93biBwbGFudGhvcHBlciBOaWxhcGFydmF0YSBsdWdlbnM7IEouIEluc2VjdCBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9MTM0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SXNvbGF0aW9uIGFuZCBjaGFyYWN0ZXJpemF0aW9uIG9mIG1pY3Jvc2F0ZWxsaXRlIG1hcmtlcnMgaW4gYnJvd24gcGxhbnRob3BwZXIgKE5pbGFwYXJ2YXRhIGx1Z2VucyBTdMOlbCk7IEludC4gSi4gTW9sLiBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9OTUyNy0zMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGV2ZWxvcG1lbnQgb2YgdmlydWxlbmNlIHRvIHJlc2lzdGFudCByaWNlIHZhcmlldGllcyBpbiB0aGUgYnJvd24gcGxhbnRob3BwZXIsIE5pbGFwYXJ2YXRhIGx1Z2VucyAoSG9tb3B0ZXJhOiBEZWxwaGFjaWRhZSksIGltbWlncmF0aW5nIGludG8gSmFwYW47IEFwcGwuIEVudG9tb2wuIFpvb2wuOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTM1OyAgY2l0YXRpb25fcGFnZXM9NTI5LTMzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIG1pZ3JhdGlvbiBhbmFseXNpcyBvZiB0aGUgcmljZSBwbGFudGhvcHBlciBOaWxhcGFydmF0YSBsdWdlbnMgZnJvbSB0aGUgUGhpbGlwcGluZXMgdG8gRWFzdCBBc2lhIHdpdGggdGhyZWUtZGltZW5zaW9uYWwgY29tcHV0ZXIgc2ltdWxhdGlvbnM7IFBvcHVsLiBFY29sLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT00NzsgIGNpdGF0aW9uX3BhZ2VzPTE0My01MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBwbGFudCBETkEgbWluaXByZXBhcmF0aW9uOiB2ZXJzaW9uIElJOyBQbGFudCBNb2wuIEJpb2wuIFJlcC47ICBjaXRhdGlvbl95ZWFyPTE5ODM7ICBjaXRhdGlvbl92b2x1bWU9MTsgIGNpdGF0aW9uX3BhZ2VzPTE5LTIxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CYWN0ZXJpYWwgc3ltYmlvbnRzIG9mIHRoZSBicm93biBwbGFudGhvcHBlciwgTmlsYXBhcnZhdGEgbHVnZW5zIChIb21vcHRlcmE6IERlbHBoYWNpZGFlKTsgQXBwbC4gRW52aXJvbi4gTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT03NjsgIGNpdGF0aW9uX3BhZ2VzPTE3NDAtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UG9seW1lcmFzZSBhbmQgbXVsdGlwbHktcHJpbWVkIHJvbGxpbmcgY2lyY2xlIGFtcGxpZmljYXRpb24gcmFwaWQgYW1wbGlmaWNhdGlvbiBvZiBwbGFzbWlkIGFuZCBwaGFnZSBETkEgdXNpbmcgUGhpMjkgRE5BOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTEwOTUtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SXNvbGF0aW5nIG1pY3Jvc2F0ZWxsaXRlIEROQSBsb2NpOyBjaXRhdGlvbl9wdWJsaXNoZXI9QWNhZGVtaWMgUHJlc3MsIFNhbiBEaWVnbywgQ0E7IE1ldGhvZHMgaW4gRW56eW1vbG9neTsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT12b2wuIDM5NTsgIGNpdGF0aW9uX3BhZ2VzPTIwMi0yMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3RyYXRlZ2llcyBmb3IgbWljcm9zYXRlbGxpdGUgaXNvbGF0aW9uOiBhIHJldmlldzsgTW9sLiBFY29sLjsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTEtMTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXB1dGF0aW9uYWwgYW5kIGV4cGVyaW1lbnRhbCBhbmFseXNpcyBvZiBtaWNyb3NhdGVsbGl0ZXMgaW4gcmljZSAoT3J5emEgc2F0aXZhIEwuKTogZnJlcXVlbmN5LCBsZW5ndGggdmFyaWF0aW9uLCB0cmFuc3Bvc29uIGFzc29jaWF0aW9ucywgYW5kIGdlbmV0aWMgbWFya2VyIHBvdGVudGlhbDsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz0xNDQxLTUyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QcmltZXIzIG9uIHRoZSBXV1cgZm9yIGdlbmVyYWwgdXNlcnMgYW5kIGZvciBiaW9sb2dpc3QgcHJvZ3JhbW1lcnM7IE1ldGhvZHMgTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMzI7ICBjaXRhdGlvbl9wYWdlcz0zNjUtODY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFnZ3Jlc3NpdmUgYXNzZW1ibHkgb2YgcHlyb3NlcXVlbmNpbmcgcmVhZHMgd2l0aCBtYXRlczsgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MjQ7ICBjaXRhdGlvbl9wYWdlcz0yODE4LTI0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GYXN0IGFuZCBhY2N1cmF0ZSBzaG9ydCByZWFkIGFsaWdubWVudCB3aXRoIEJ1cnJvd3MtV2hlZWxlciB0cmFuc2Zvcm07IEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9MTc1NC02MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHNlcXVlbmNlIGFsaWdubWVudC9tYXAgKFNBTSkgZm9ybWF0IGFuZCBTQU0gdG9vbHM7IEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9MjA3OC05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UYWJsZXTigJNuZXh0IGdlbmVyYXRpb24gc2VxdWVuY2UgYXNzZW1ibHkgdmlzdWFsaXphdGlvbjsgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MjY7ICBjaXRhdGlvbl9wYWdlcz00MDEtMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fcHVibGlzaGVyPUt5YXptYSwgQi5WLCBXYWdlbmluZ2VuLCBOZXRoZXJsYW5kczsgSm9pbk1hcMKuNCwgU29mdHdhcmUgZm9yIHRoZSBDYWxjdWxhdGlvbiBvZiBHZW5ldGljIExpbmthZ2UgTWFwcyBpbiBFeHBlcmltZW50YWwgUG9wdWxhdGlvbnM7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBlc3RpbWF0aW9uIG9mIG1hcCBkaXN0YW5jZXMgZnJvbSByZWNvbWJpbmF0aW9uIHZhbHVlczsgQW5uLiBFdWdlbi47ICBjaXRhdGlvbl95ZWFyPTE5NDQ7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz0xNzItNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWFwQ2hhcnQ6IHNvZnR3YXJlIGZvciB0aGUgZ3JhcGhpY2FsIHByZXNlbnRhdGlvbiBvZiBsaW5rYWdlIG1hcHMgYW5kIFFUTHM7IEouIEhlcmVkLjsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT05MzsgIGNpdGF0aW9uX3BhZ2VzPTc3LTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgbWF4aW11bSBsaWtlbGlob29kIG1ldGhvZCBmb3IgZXN0aW1hdGluZyBnZW5vbWUgbGVuZ3RoIHVzaW5nIGdlbmV0aWMgbGlua2FnZSBkYXRhOyBHZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MTk5MTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjg7ICBjaXRhdGlvbl9wYWdlcz0xNzUtODI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlzb2xhdGlvbiBhbmQgY2hhcmFjdGVyaXphdGlvbiBvZiBtaWNyb3NhdGVsbGl0ZXMgaW4gdGhlIGJyb3duIHBsYW50aG9wcGVyLCBOaWxhcGFydmF0YSBsdWdlbmVzIFN0w6VsOyBLb3JlYSBKLiBBcHBsLiBFbnRvbW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5NDsgIGNpdGF0aW9uX3ZvbHVtZT00MzsgIGNpdGF0aW9uX3BhZ2VzPTMxMS01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vdHlwZSBhbmQgU05QIGNhbGxpbmcgZnJvbSBuZXh0IGdlbmVyYXRpb24gc2VxdWVuY2luZyBkYXRhOyBOYXR1cmUgUmV2LjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTQ0My01MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBtaWNyb3NhdGVsbGl0ZSBsaW5rYWdlIG1hcCBvZiB0aGUgRXVyb3BlYW4gc2VhIGJhc3MgRGljZW50cmFyY2h1cyBsYWJyYXggTC47IEdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTE3MDsgIGNpdGF0aW9uX3BhZ2VzPTE4MjEtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBtaWNyb3NhdGVsbGl0ZSBsaW5rYWdlIG1hcCBvZiBiYXJyYW11bmRpLCBMYXRlcyBjYWxjYXJpZmVyOyBHZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0xNzU7ICBjaXRhdGlvbl9wYWdlcz05MDctMTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVR3by1waGFzZSBhbmFseXNpcyBpbiBjb25zZW5zdXMgZ2VuZXRpYyBtYXBwaW5nOyBHMzsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0yOyAgY2l0YXRpb25fcGFnZXM9NTM3LTQ5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBhbXBsaWZpZWQgZnJhZ21lbnQgbGVuZ3RoIHBvbHltb3JwaGlzbSBtYXAgb2YgdGhlIHNpbGt3b3JtOyBHZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0xNTc7ICBjaXRhdGlvbl9wYWdlcz0xMjc3LTg0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db25zdHJ1Y3Rpb24gb2YgQUZMUC1iYXNlZCBnZW5ldGljIGxpbmthZ2UgbWFwIGZvciBaaGlrb25nIHNjYWxsb3AsIENobGFteXMgZmFycmVyaSBhbmQgbWFwcGluZyBvZiBzZXgtbGlua2VkIG1hcmtlcnM7IEFxdWFjdWx0dXJlOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTI0NTsgIGNpdGF0aW9uX3BhZ2VzPTYzLTczOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5ldGljIG1hcHBpbmcgYW5kIFFUTCBhbmFseXNpcyBvZiBncm93dGgtcmVsYXRlZCB0cmFpdHMgaW4gdGhlIFBhY2lmaWMgb3lzdGVyOyBNYXIuIEJpb3RlY2hub2wuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9MjE4LTI2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GaXhhdGlvbiwgc2VncmVnYXRpb24gYW5kIGxpbmthZ2Ugb2YgYWxsb3p5bWUgbG9jaSBpbiBpbmJyZWQgZmFtaWxpZXMgb2YgdGhlIFBhY2lmaWMgb3lzdGVyIENyYXNzb3N0cmVhIGdpZ2EgKFRodW5iZXJnKTogaW1wbGljYXRpb25zIGZvciB0aGUgY2F1c2VzIG9mIGluYnJlZWRpbmcgZGVwcmVzc2lvbjsgR2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9MTQ2OyAgY2l0YXRpb25fcGFnZXM9MzIxLTM0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZXgtcmF0aW8gc2VncmVnYXRpb24gZGlzdG9ydGlvbiBhc3NvY2lhdGVkIHdpdGggcmVwcm9kdWN0aXZlIGlzb2xhdGlvbiBpbiBEcm9zb3BoaWxhOyBQcm9jLiBOYXQuIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTk4OyAgY2l0YXRpb25fcGFnZXM9MTMxODMtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2hlYXRlcnMgc29tZXRpbWVzIHByb3NwZXI6IGRpc3RvcnRpb24gb2YgbWVuZGVsaWFuIHNlZ3JlZ2F0aW9uIGJ5IG1laW90aWMgZHJpdmU7IFRyZW5kcyBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTE5OTM7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTIwNS0xMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXN0aW1hdGluZyB0aGUgZWZmZWN0cyBvZiBwb3B1bGF0aW9uIHNpemUgYW5kIHR5cGUgb24gdGhlIGFjY3VyYWN5IG9mIGdlbmV0aWMgbWFwczsgR2VuZXQuIE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9Mjk7ICBjaXRhdGlvbl9wYWdlcz0xODItOTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFGTFAtYmFzZWQgZ2VuZXRpYyBsaW5rYWdlIG1hcCBvZiB0aGUgQ29sb3JhZG8gcG90YXRvIGJlZXRsZSBMZXB0aW5vdGFyc2EgZGVjZW1saW5lYXRhOiBzZXggY2hyb21vc29tZXMgYW5kIGEgcHlyZXRocm9pZC1yZXNpc3RhbmNlIGNhbmRpZGF0ZSBnZW5lOyBHZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0xNTg7ICBjaXRhdGlvbl9wYWdlcz02OTUtNzAwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBBRkxQLWJhc2VkIGdlbmV0aWMgbGlua2FnZSBtYXAgb2YgY2hhbm5lbCBjYXRmaXNoIChJY3RhbHVydXMgcHVuY3RhdHVzKSBjb25zdHJ1Y3RlZCBieSB1c2luZyBhbiBpbnRlcnNwZWNpZmljIGh5YnJpZCByZXNvdXJjZSBmYW1pbHk7IEdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTE2NTsgIGNpdGF0aW9uX3BhZ2VzPTY4Ny05NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Rmlyc3QtZ2VuZXJhdGlvbiBsaW5rYWdlIG1hcCBvZiB0aGUgd2FybmluZ2x5IGNvbG9yZWQgYnV0dGVyZmx5IEhlbGljb25pdXMgZXJhdG87IEhlcmVkaXR5OyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTk0OyAgY2l0YXRpb25fcGFnZXM9NDA4LTE3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIG1pY3Jvc2F0ZWxsaXRlIG1hcmtlciBsaW5rYWdlIG1hcCBvZiB0aGUgaG91c2VmbHksIE11c2NhIGRvbWVzdGljYTogZXZpZGVuY2UgZm9yIG1hbGUgcmVjb21iaW5hdGlvbjsgSW5zZWN0IE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTk7ICBjaXRhdGlvbl9wYWdlcz01NzUtODE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgbWljcm9zYXRlbGxpdGUgbGlua2FnZSBtYXAgb2YgcmFpbmJvdyB0cm91dCAoT25jb3JoeW5jaHVzIG15a2lzcykgY2hhcmFjdGVyaXplZCBieSBsYXJnZSBzZXgtc3BlY2lmaWMgZGlmZmVyZW5jZXMgaW4gcmVjb21iaW5hdGlvbiByYXRlczsgR2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MTU1OyAgY2l0YXRpb25fcGFnZXM9MTMzMS00NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBkZW5zZSBTTlAtYmFzZWQgbGlua2FnZSBtYXAgZm9yIEF0bGFudGljIHNhbG1vbiAoU2FsbW8gc2FsYXIpIHJldmVhbHMgZXh0ZW5kZWQgY2hyb21vc29tZSBob21vbG9naWVzIGFuZCBzdHJpa2luZyBkaWZmZXJlbmNlcyBpbiBzZXgtc3BlY2lmaWMgcmVjb21iaW5hdGlvbiBwYXR0ZXJuczsgQk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9NjE1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29uc3RydWN0aW9uIG9mIGFuIGludGVncmF0ZWQgbWFwIG9mIEhhbGlvdGlzIGRpdmVyc2ljb2xvciB1c2luZyBtaWNyb3NhdGVsbGl0ZSBtYXJrZXJzOyBNYXIuIEJpb3RlY2hub2wuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9NzktODY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNpbXBsZSBzZXF1ZW5jZSByZXBlYXQtYmFzZWQgY29uc2Vuc3VzIGxpbmthZ2UgbWFwIG9mIEJvbWJ5eCBtb3JpOyBQcm9jLiBOYXQuIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTEwMjsgIGNpdGF0aW9uX3BhZ2VzPTE2MzAzLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgbWljcm9zYXRlbGxpdGUgbGlua2FnZSBtYXAgZm9yIERyb3NvcGhpbGEgbW9udGFuYSBzaG93cyBsYXJnZSB2YXJpYXRpb24gaW4gcmVjb21iaW5hdGlvbiByYXRlcywgYW5kIGEgY291cnRzaGlwIHNvbmcgdHJhaXQgbWFwcyB0byBhbiBhcmVhIG9mIGxvdyByZWNvbWJpbmF0aW9uOyBKLiBFdm9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0yMzsgIGNpdGF0aW9uX3BhZ2VzPTUxOC0yNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBtaWNyb3NhdGVsbGl0ZS1iYXNlZCBsaW5rYWdlIG1hcCBvZiB0aGUgaG9uZXliZWUsIEFwaXMgbWVsbGlmZXJhIEwuOyBHZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNjc7ICBjaXRhdGlvbl9wYWdlcz0yNTMtNjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFGTFAtYmFzZWQgZ2VuZXRpYyBsaW5rYWdlIG1hcCBmb3IgdGhlIHJlZCBmbG91ciBiZWV0bGUgKFRyaWJvbGl1bSBjYXN0YW5ldW0pOyBKLiBIZXJlZGl0eTsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT05NTsgIGNpdGF0aW9uX3BhZ2VzPTUzLTYxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1WaXJ1bGVuY2Ugb2YgbG9uZy10ZXJtIGxhYm9yYXRvcnkgcG9wdWxhdGlvbnMgb2YgdGhlIGJyb3duIHBsYW50aG9wcGVyLCBOaWxhcGFydmF0YSBsdWdlbnMgKFN0w6VsKSwgYW5kIHdoaXRlYmFja2VkIHBsYW50aG9wcGVyLCBTb2dhdGVsbGEgZnVyY2lmZXJhIChIb3J2w6F0aCkgKEhvbW9wdGVyYTogRGVscGhhY2lkYWUpLCBvbiByaWNlIGRpZmZlcmVudGlhbCB2YXJpZXRpZXM7IEFwcGwuIEVudG9tb2wuIFpvb2wuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTQ0OyAgY2l0YXRpb25fcGFnZXM9MTQ5LTUzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db25zdHJ1Y3Rpb24gb2YgYSBzaW5nbGUgbnVjbGVvdGlkZSBwb2x5bW9ycGhpc20gbGlua2FnZSBtYXAgZm9yIHRoZSBzaWxrd29ybSwgQm9tYnl4IG1vcmksIGJhc2VkIG9uIGJhY3RlcmlhbCBhcnRpZmljaWFsIGNocm9tb3NvbWUgZW5kIHNlcXVlbmNlczsgR2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MTczOyAgY2l0YXRpb25fcGFnZXM9MTUxLTYxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JZGVudGlmaWNhdGlvbiBvZiBZIGNocm9tb3NvbWFsIFBDUiBtYXJrZXIgYW5kIHByb2R1Y3Rpb24gb2YgYSBzZWxlY3RlZCBzdHJhaW4gZm9yIG1vbGVjdWxhciBzZXhpbmcgaW4gdGhlIGJyb3duIHBsYW50aG9wcGVyLCBOaWxhcGFydmF0YSBsdWdlbnM7IEFyY2guIEluc2VjdCBCaW9jaGVtLiBQaHlzaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT02NTsgIGNpdGF0aW9uX3BhZ2VzPTEtMTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBpc29sYXRpb24gYW5kIGNoYXJhY3Rlcml6YXRpb24gb2YgRHJvc29waGlsYSB5b2xrIHByb3RlaW4gZ2VuZXM7IENlbGw7ICBjaXRhdGlvbl95ZWFyPTE5ODA7ICBjaXRhdGlvbl92b2x1bWU9MjE7ICBjaXRhdGlvbl9wYWdlcz03MjktMzg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlzb2xhdGlvbiBvZiB0d28gRHJvc29waGlsYSBtZWxhbm9nYXN0ZXIgZ2VuZXMgYWJ1bmRhbnRseSBleHByZXNzZWQgaW4gdGhlIG92YXJ5IGR1cmluZyB2aXRlbGxpbmUgbWVtYnJhbmUgc3ludGhlc2lzOyBEZXYuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTg0OyAgY2l0YXRpb25fdm9sdW1lPTEwNTsgIGNpdGF0aW9uX3BhZ2VzPTE1NS02NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3RydWN0dXJhbCBhbmFseXNpcyBvZiB0aGUgdGhyZWUgdml0ZWxsb2dlbmluIGdlbmVzIGluIERyb3NvcGhpbGEgbWVsYW5vZ2FzdGVyOyBOdWNsLiBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTgxOyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz0xMzIzLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVgtbGlua2FnZSBvZiBhIHZpdGVsbG9nZW5pbiBnZW5lIGluIExvY3VzdGEgbWlncmF0b3JpYTsgQ2hyb21vc29tYTsgIGNpdGF0aW9uX3llYXI9MTk4MzsgIGNpdGF0aW9uX3ZvbHVtZT04ODsgIGNpdGF0aW9uX3BhZ2VzPTE5MC0zOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaGFyYWN0ZXJpemF0aW9uIG9mIHJlZ2lvbnMgb2YgdGhlIENhZW5vcmhhYmRpdGlzIGVsZWdhbnMgWCBjaHJvbW9zb21lIGNvbnRhaW5pbmcgdml0ZWxsb2dlbmluIGdlbmVzOyBKLiBNb2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTg2OyAgY2l0YXRpb25fdm9sdW1lPTE4ODsgIGNpdGF0aW9uX3BhZ2VzPTMwMS0xMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzIwLzEvMTcvMTA4MzgzNS9kc3MwMzAucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIEluIHRoaXMgc3R1ZHksIHdlIGRldmVsb3BlZCB0aGUgZmlyc3QgZ2VuZXRpYyBsaW5rYWdlIG1hcCBmb3IgdGhlIG1ham9yIHJpY2UgaW5zZWN0IHBlc3QsIHRoZSBicm93biBwbGFudGhvcHBlciAoQlBILCBOaWxhcGFydmF0YSBsdWdlbnMpLiBUaGUgbGlua2FnIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFci8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj0zODIxNTAmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2RklZCRwgXerq6f7P9BtZ0972mdd/E=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f1%2f17%2f382150" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f1%2f17%2f382150" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=14916" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/20/1"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/1/1/m_cover.gif?Expires=2147483647&amp;Signature=K1v8cGQnstZYdECB9GA3oc5Ko3ohhQ9lF83XAKOVlOLlyAqR0JX0Vn18cJ3Fh6Ni4coRVii1nVVwFuzd9baa8EpiNPdDbUxg77BFyv3S5jO4sVroBj5PfHgZd5D14mPzL5KK6FHcweE5FD2RLiGEj~ct-3TIgAM5vP7r7dob4djMEW-dmZ2mnE0M4g6gvpwd4meZtXx6YM6PJ-qadpYPQyOJp7E7APzkRfsaUptwFx9azq6qh9BweylqnGS-k05la9PhL7T57M1047efztU-bcJDc0lmA7wiGh4dEYA5ldVPoM0OpeZ~9-tJ2gANZpruMQldUUZX51aSqOwvf70tSQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/20/1">
                        <div class="volume trailing-comma">Volume 20</div>
                        <div class="issue">Issue 1</div>

            </a>

        </div>
        <div class="ii-pub-date">
February 2013        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="5594872">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5594872">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5594874">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5594874">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5594886">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5594886">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5594958">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5594958">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5594974">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5594974">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5594990">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5594990">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5594992">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5594992">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5594994">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5594994">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5594996">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5594996">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/20/1/1/381937">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/20/1/31/382624">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">A Simple Sequence Repeat- and Single-Nucleotide Polymorphism-Based Genetic Linkage Map of the Brown Planthopper, <em>Nilaparvata lugens</em>
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                            <div class="meta-authors--limited">
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jirapong Jairin</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Ubon Ratchathani Rice Research Center</div>, <div class="addr-line">PO Box 65, Muang, Ubon Ratchathani 34000</div>, <div class="country">Thailand</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">Plant Breeding Laboratory, Graduate School, Faculty of Agriculture,</div><div class="insititution">Kyushu University</div>, <div class="addr-line">6-10-1, Hakozaki, Higashi-ku, Fukuoka 812-8581</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jirapong+Jairin">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Jairin J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Jairin J&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Jirapong Jairin</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tetsuya Kobayashi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="insititution">National Institute of Agrobiological Sciences</div>, <div class="addr-line">1-2, O-washi, Tsukuba, Ibaraki 305-8634</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tetsuya+Kobayashi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kobayashi T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kobayashi T&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Tetsuya Kobayashi</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yoshiyuki Yamagata</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Plant Breeding Laboratory, Graduate School, Faculty of Agriculture,</div><div class="insititution">Kyushu University</div>, <div class="addr-line">6-10-1, Hakozaki, Higashi-ku, Fukuoka 812-8581</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yoshiyuki+Yamagata">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yamagata Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yamagata Y&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Yoshiyuki Yamagata</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Sachiyo Sanada-Morimura</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">Kyushu Okinawa Agricultural Research Center</div>, <div class="insititution">National Agriculture and Food Research Organization</div>, <div class="addr-line">Kumamoto 861-1192</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Sachiyo+Sanada-Morimura">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sanada-Morimura S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sanada-Morimura S&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Sachiyo Sanada-Morimura</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kazuki Mori</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">5</div><div class="addr-line">Laboratory of Molecular Gene Technology, Division of Molecular Biosciences, Department of Bioscience and Biotechnology, Faculty of Agriculture</div>, <div class="insititution">Kyushu University</div>, <div class="addr-line">6-10-1 Hakozaki, Higashi-ku, Fukuoka 812-8581</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kazuki+Mori">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Mori K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Mori K&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Kazuki Mori</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kosuke Tashiro</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">5</div><div class="addr-line">Laboratory of Molecular Gene Technology, Division of Molecular Biosciences, Department of Bioscience and Biotechnology, Faculty of Agriculture</div>, <div class="insititution">Kyushu University</div>, <div class="addr-line">6-10-1 Hakozaki, Higashi-ku, Fukuoka 812-8581</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kosuke+Tashiro">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tashiro K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tashiro K&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Kosuke Tashiro</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Satoru Kuhara</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">5</div><div class="addr-line">Laboratory of Molecular Gene Technology, Division of Molecular Biosciences, Department of Bioscience and Biotechnology, Faculty of Agriculture</div>, <div class="insititution">Kyushu University</div>, <div class="addr-line">6-10-1 Hakozaki, Higashi-ku, Fukuoka 812-8581</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Satoru+Kuhara">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kuhara S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kuhara S&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Satoru Kuhara</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Seigo Kuwazaki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="insititution">National Institute of Agrobiological Sciences</div>, <div class="addr-line">1-2, O-washi, Tsukuba, Ibaraki 305-8634</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Seigo+Kuwazaki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kuwazaki S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kuwazaki S&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Seigo Kuwazaki</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Masahiro Urio</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="insititution">National Institute of Agrobiological Sciences</div>, <div class="addr-line">1-2, O-washi, Tsukuba, Ibaraki 305-8634</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Masahiro+Urio">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Urio M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Urio M&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Masahiro Urio</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yoshitaka Suetsugu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="insititution">National Institute of Agrobiological Sciences</div>, <div class="addr-line">1-2, O-washi, Tsukuba, Ibaraki 305-8634</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yoshitaka+Suetsugu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Suetsugu Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Suetsugu Y&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Yoshitaka Suetsugu</a>
                                    </span>
                                <a class="meta-authors--etal is-active" id="show-meta-authors" aria-controls="remaining-authors" href="#">
                                    <span>...</span> Show more</a>
                                <div id="remaining-authors" class="meta-authors--remaining" aria-hidden="true" aria-labeledby="show-meta-authors">
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kimiko Yamamoto</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="insititution">National Institute of Agrobiological Sciences</div>, <div class="addr-line">1-2, O-washi, Tsukuba, Ibaraki 305-8634</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kimiko+Yamamoto">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yamamoto K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yamamoto K&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Kimiko Yamamoto</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Masaya Matsumura</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">Kyushu Okinawa Agricultural Research Center</div>, <div class="insititution">National Agriculture and Food Research Organization</div>, <div class="addr-line">Kumamoto 861-1192</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Masaya+Matsumura">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Matsumura M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Matsumura M&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Masaya Matsumura</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hideshi Yasui</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Plant Breeding Laboratory, Graduate School, Faculty of Agriculture,</div><div class="insititution">Kyushu University</div>, <div class="addr-line">6-10-1, Hakozaki, Higashi-ku, Fukuoka 812-8581</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81 92-642-2821. Fax. +81 92-642-2822. Email: <a href="mailto:hyasui@agr.kyushu-u.ac.jp" target="_blank">hyasui@agr.kyushu-u.ac.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hideshi+Yasui">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yasui H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yasui H&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Hideshi Yasui</a>
                                        </span>
                                </div>
                            </div>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">These authors contributed equally to this work.</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 20, Issue 1, 1 February 2013, Pages 17–30, <a href='https://doi.org/10.1093/dnares/dss030'>https://doi.org/10.1093/dnares/dss030</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">29 November 2012</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">01 September 2012</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">28 October 2012</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="382150" href="/dnaresearch/article-pdf/20/1/17/1083835/dss030.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Jirapong Jairin, Tetsuya Kobayashi, Yoshiyuki Yamagata, Sachiyo Sanada-Morimura, Kazuki Mori, Kosuke Tashiro, Satoru Kuhara, Seigo Kuwazaki, Masahiro Urio, Yoshitaka Suetsugu, Kimiko Yamamoto, Masaya Matsumura, Hideshi Yasui;  A Simple Sequence Repeat- and Single-Nucleotide Polymorphism-Based Genetic Linkage Map of the Brown Planthopper, <em>Nilaparvata lugens</em>, <em>DNA Research</em>, Volume 20, Issue 1, 1 February 2013, Pages 17–30, <a href='https://doi.org/10.1093/dnares/dss030'>https://doi.org/10.1093/dnares/dss030</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/382150?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/382150?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/382150?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/382150?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/382150?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/382150?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/382150?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=A%20Simple%20Sequence%20Repeat-%20and%20Single-Nucleotide%20Polymorphism-Based%20Genetic%20Linkage%20Map%20of%20the%20Brown%20Planthopper%2C%20%26lt%3Bem%26gt%3BNilaparvata%20lugens%26lt%3B%2Fem%26gt%3B&amp;publicationDate=2012-11-29&amp;volumeNum=20&amp;issueNum=1&amp;author=Jairin%2C%20Jirapong%3B%20Kobayashi%2C%20Tetsuya&amp;startPage=17&amp;endPage=30&amp;contentId=10.1093%2Fdnares%2Fdss030&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f1%2f17%2f382150" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=5594872 id="5594872" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>In this study, we developed the first genetic linkage map for the major rice insect pest, the brown planthopper (BPH, <em>Nilaparvata lugens</em>). The linkage map was constructed by integrating linkage data from two backcross populations derived from three inbred BPH strains. The consensus map consists of 474 simple sequence repeats, 43 single-nucleotide polymorphisms, and 1 sequence-tagged site, for a total of 518 markers at 472 unique positions in 17 linkage groups. The linkage groups cover 1093.9 cM, with an average distance of 2.3 cM between loci. The average number of marker loci per linkage group was 27.8. The sex-linkage group was identified by exploiting X-linked and Y-specific markers. Our linkage map and the newly developed markers used to create it constitute an essential resource and a useful framework for future genetic analyses in BPH.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Nilaparvata lugens&quot;"><em>Nilaparvata lugens</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;brown planthopper&quot;">brown planthopper</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genetic linkage map&quot;">genetic linkage map</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="SSR">SSR</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="SNP">SNP</a></div>                            <h2 scrollto-destination=5594874 id="5594874" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>The brown planthopper (BPH), <em>Nilaparvata lugens</em> Stål (Homoptera: Delphacidae), is one of the most economically important insect pests of rice (<em>Oryza sativa</em> L.) throughout Asia. The widespread damage caused by BPH leads to substantial and unpredictable decreases in rice yield. Continuous rice cultivation, accompanied by extensive use and frequent misuse of insecticides along with a high rate of nitrogen fertilizer application, often causes BPH outbreaks in rice fields.<sup><a reveal-id="DSS030C1" data-open="DSS030C1" class="link link-ref link-reveal xref-bibr">1</a></sup> More than 50 million ha of rice fields throughout Asia have been heavily affected by this insect in the past decade.<sup><a reveal-id="DSS030C2  DSS030C4" data-open="DSS030C2  DSS030C4" class="link link-ref link-reveal xref-bibr">2–4</a></sup> In addition to directly feeding on rice plants, BPH also causes indirect damage by transmitting viruses that cause the ragged stunt and grassy stunt diseases of rice.<sup><a reveal-id="DSS030C5" data-open="DSS030C5" class="link link-ref link-reveal xref-bibr">5</a></sup></p><p>Host-plant resistance offers a practical approach for BPH control, and a succession of resistant rice varieties has been bred and cultivated widely in Asia. Nevertheless, the appearance of BPH with new virulence characteristics can overcome their resistance soon after the varieties have been released.<sup><a reveal-id="DSS030C4" data-open="DSS030C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DSS030C6" data-open="DSS030C6" class="link link-ref link-reveal xref-bibr">6</a></sup> The virulence, which refers to the ability to adapt to resistant rice varieties, has been reported to be under genetic control and is inherited as a quantitative trait.<sup><a reveal-id="DSS030C7" data-open="DSS030C7" class="link link-ref link-reveal xref-bibr">7</a></sup> Understanding the genetic basis of the virulence against rice cultivars will provide significant information for future BPH management strategies. A genetic linkage map would provide an important framework for identifying quantitative trait loci (QTL) related to the virulence characteristics. Unfortunately, no detailed genetic linkage map based on molecular markers has yet been developed for BPH or for other closely related delphacid species.</p><p>BPH has a diploid chromosome number of 30 (28 autosomes combined with the sex determination system XY and XX in males and females, respectively).<sup><a reveal-id="DSS030C8  DSS030C10" data-open="DSS030C8  DSS030C10" class="link link-ref link-reveal xref-bibr">8–10</a></sup> The genome size of BPH is estimated to be 1.2 Gbp.<sup><a reveal-id="DSS030C11" data-open="DSS030C11" class="link link-ref link-reveal xref-bibr">11</a></sup> Recently, a number of genomic resources for BPH have been made available, and they provide highly useful information to support experimental genetics studies.<sup><a reveal-id="DSS030C12  DSS030C16" data-open="DSS030C12  DSS030C16" class="link link-ref link-reveal xref-bibr">12–16</a></sup> More than 350 expressed sequence tag (EST)-derived simple sequence repeat (e-SSR) markers for BPH have been successfully developed subsequent to the availability of large EST datasets.<sup><a reveal-id="DSS030C17  DSS030C19" data-open="DSS030C17  DSS030C19" class="link link-ref link-reveal xref-bibr">17–19</a></sup> In addition, a total of 136 SSRs have been isolated from BPH genomic DNA.<sup><a reveal-id="DSS030C20" data-open="DSS030C20" class="link link-ref link-reveal xref-bibr">20</a></sup> These newly developed e-SSRs and genomic SSR (g-SSR) markers have been used to assess the genetic diversity of BPH populations.<sup><a reveal-id="DSS030C17" data-open="DSS030C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DSS030C20" data-open="DSS030C20" class="link link-ref link-reveal xref-bibr">20</a></sup> However, the exact genomic locations of the newly developed SSR markers have not yet been determined by genetic mapping.</p><p>Genetic maps are valuable genomic resources and represent a powerful research tool for identifying the genetic basis of phenotypic variation. The purpose of the present study was to construct a baseline genetic linkage map for BPH using both new and existing SSRs and single-nucleotide polymorphisms (SNPs). G-SSR markers were generated from the sequence information of repeat regions derived from high-throughput microsatellite isolation procedures. We also developed SNP markers using next-generation sequencing technologies. The newly developed and public markers were successfully applied to construct a consensus linkage map. This map will provide initial information for identifying the genetic basis of BPH virulence and other important traits, including insecticide resistance, virus transmission, and wing polymorphisms. In addition, the newly developed markers will support studies of the morphology, physiology, evolution, and ecology of this significant insect pest.</p>                            <h2 scrollto-destination=5594886 id="5594886" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=5594893 id="5594893" class="section-title" data-legacy-id=s2a>2.1. Insect strains and crosses</h3>
<p>Four BPH strains collected in Japan were used in this study. (i) Hadano-66 was collected in Hadano, Kanagawa Prefecture, in 1966 and was maintained in a laboratory colony for ∼500 generations, (ii) Chikugo-89 was collected in Chikugo, Fukuoka Prefecture, in 1989 and was maintained in a laboratory colony for more than 250 generations, (iii) Koshi-10 was collected in Koshi, Kumamoto Prefecture, in 2010 and was maintained for ∼20 generations in a laboratory colony and (iv) Izumo-87 was collected in Izumo, Shimane Prefecture, in 1987 and was maintained in a laboratory colony for ∼270 generations. These BPH strains immigrated into Japan from East Asian regions.<sup><a reveal-id="DSS030C21" data-open="DSS030C21" class="link link-ref link-reveal xref-bibr">21</a>,<a reveal-id="DSS030C22" data-open="DSS030C22" class="link link-ref link-reveal xref-bibr">22</a></sup> The Izumo-87 strain was used for whole-genome sequencing to generate a reference sequence<sup><a reveal-id="DSS030C12" data-open="DSS030C12" class="link link-ref link-reveal xref-bibr">12</a></sup> and detect SNPs. Hadano-66, Chikugo-89, and Koshi-10 strains were used to generate the inbred BPH strains (IBSs) and the mapping populations. One pair of insects from each BPH strain was selected for crossing to generate the IBSs H66 for Hadano-66, C89 for Chikugo-89, and K10 for Koshi-10. The IBSs were prepared without outcrossing through three generations of single-pair, full-sibling mating (Fig. <a reveal-id="DSS030F1" data-open="DSS030F1" class="link link-reveal link-table xref-fig">1</a>). All BPH strains were reared on seedlings of cv. ‘Reiho’, a <em>japonica</em> rice variety, and maintained in the laboratory at 25 ± 2°C under a 16L:8D photoperiod. To generate the mapping populations, a pair of insects from two different IBSs was selected and crossed using single-pair mating. Two backcross populations (BC<sub>1</sub>F<sub>1</sub>) were generated from crosses (♀ × ♂) of K10 × H66 with backcrossing with H66 (referred to as the KH population) and C89 × K10 with backcrossing with C89 (referred to as the CK population). </p>                        <a id="5594902" scrollto-destination="5594902"></a>
<div id="DSS030F1" data-id="DSS030F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss030/2/m_dss03001.jpeg?Expires=2147483647&Signature=AJbkyxDZpzj0urgqV2X2Bc5EtpDcWgUeaIBR71a1ybyB54pj~HQbf0USYiZoi2glZ05uXOhYVFeV9-8h~95Qt8lHKUPrmqQVMw~m265Y7hZzfxiT0B4qOIvDVRGBOCIpI~ETPiho44l39gByzXzJKByF8eYtwZ7ObGvf5yUEMew-I0~ySwlElm4~3g~U2WQgg0p~v8P9poduAhWMPZGe7DdNzaKeshj7cXOpCXiGEsXTP4sWuwOjKU8RBgZAUKrmwj-pNNtxEc1juIKdL-D~mjrq7tm2U-2X~Ea4JCuX65NDZvQLhSBz3fEvagwlS-AUycZDdb9gxy2d5622Tllfqw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Mating scheme of an experiment perform to obtain IBSs and mapping populations." path-from-xml="dss03001.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5594902/dss03001.jpeg" path-from-xml="dss03001.jpeg" target="_blank">View large</a><a section="5594902" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss030/2/dss03001.jpeg?Expires=2147483647&Signature=gxXoPJ-gL1DpLiPpaa6kC3dTul1w6R3FmMV03T~FL8is9I9tzfd6SPihzj7-abngH1WqrqzPEjUNLAmgU5rMWUouov0lffLGaM897W3ZajwsQIulbeWWRRslH5h5dA-XJECe-hLymTMd9g~XrnhNNrCuQQpegVguFH6UnA-cn7tKevo-9R0mciLUdZD1nI4vLwZZLyEgqkqQU94Je-lxovrtktBSuygzjxrSwodId2z7qfxVMBAXS1XMskbSIi6B~w9nOtgb0-m6VCRevWntVdgf-6hASj9KATAwnGMa-gXgzSrRQNeLkDhRhI8ff7cQw9qWcYjqH-9l3cbXcaah0A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5594902&ar=382150&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss03001.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Mating scheme of an experiment perform to obtain IBSs and mapping populations.</p></div></div><div content-id="DSS030F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss030/2/m_dss03001.jpeg?Expires=2147483647&Signature=AJbkyxDZpzj0urgqV2X2Bc5EtpDcWgUeaIBR71a1ybyB54pj~HQbf0USYiZoi2glZ05uXOhYVFeV9-8h~95Qt8lHKUPrmqQVMw~m265Y7hZzfxiT0B4qOIvDVRGBOCIpI~ETPiho44l39gByzXzJKByF8eYtwZ7ObGvf5yUEMew-I0~ySwlElm4~3g~U2WQgg0p~v8P9poduAhWMPZGe7DdNzaKeshj7cXOpCXiGEsXTP4sWuwOjKU8RBgZAUKrmwj-pNNtxEc1juIKdL-D~mjrq7tm2U-2X~Ea4JCuX65NDZvQLhSBz3fEvagwlS-AUycZDdb9gxy2d5622Tllfqw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Mating scheme of an experiment perform to obtain IBSs and mapping populations." path-from-xml="dss03001.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5594902/dss03001.jpeg" path-from-xml="dss03001.jpeg" target="_blank">View large</a><a section="5594902" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss030/2/dss03001.jpeg?Expires=2147483647&Signature=gxXoPJ-gL1DpLiPpaa6kC3dTul1w6R3FmMV03T~FL8is9I9tzfd6SPihzj7-abngH1WqrqzPEjUNLAmgU5rMWUouov0lffLGaM897W3ZajwsQIulbeWWRRslH5h5dA-XJECe-hLymTMd9g~XrnhNNrCuQQpegVguFH6UnA-cn7tKevo-9R0mciLUdZD1nI4vLwZZLyEgqkqQU94Je-lxovrtktBSuygzjxrSwodId2z7qfxVMBAXS1XMskbSIi6B~w9nOtgb0-m6VCRevWntVdgf-6hASj9KATAwnGMa-gXgzSrRQNeLkDhRhI8ff7cQw9qWcYjqH-9l3cbXcaah0A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5594902&ar=382150&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss03001.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Mating scheme of an experiment perform to obtain IBSs and mapping populations.</p></div></div><p>The crosses were performed by placing a virgin female and a male from each IBS on ‘Reiho’ seedlings in a glass tube. After mating and oviposition, the parental insects were collected and stored in 95% ethanol. The F<sub>1</sub> offspring from each cross was cultured in separate containers. Newly hatched nymphs of F<sub>1</sub> progeny from each cross were transferred into a plastic 800-ml container (dimensions: 8.5 cm wide by 16 cm tall) containing 8–10 seedlings and reared up to the fifth-instar nymph stage. An individual fifth-instar nymph of each sex was selected and reared in a separate glass tube to obtain unmated adults. F<sub>1</sub> females were backcrossed with their parent family to generate the BC<sub>1</sub>F<sub>1</sub> generation (Fig. <a reveal-id="DSS030F1" data-open="DSS030F1" class="link link-reveal link-table xref-fig">1</a>). BC<sub>1</sub>F<sub>1</sub> offspring from each cross were reared up to the second- and third-instar nymph stages. Although it is not possible to sex the insects at this stage, using early instars greatly accelerates the analytical process and avoids viability problems that may occur at later stages. All offspring were then collected and stored in 95% ethanol. We randomly selected 91 KH offspring and 89 CK offspring for the linkage analysis. All crossing experiments were conducted in the insectary at the Kyushu Okinawa Agricultural Research Center, Kumamoto, Japan.</p>                            <h3 scrollto-destination=5594915 id="5594915" class="section-title" data-legacy-id=s2b>2.2. Total DNA extraction and whole-genome amplification</h3>
<p>Genomic DNA of the preserved samples (grandparents, parents, and offspring) was isolated individually using a modified version of the potassium acetate procedure of Dellaporta <em>et al</em>.<sup><a reveal-id="DSS030C23" data-open="DSS030C23" class="link link-ref link-reveal xref-bibr">23</a></sup> For the adult insects, only the head and thorax were used for DNA extraction to reduce the possibility of contamination by fertilized eggs and bacterial symbionts.<sup><a reveal-id="DSS030C24" data-open="DSS030C24" class="link link-ref link-reveal xref-bibr">24</a></sup> The whole-genome amplification technique<sup><a reveal-id="DSS030C25" data-open="DSS030C25" class="link link-ref link-reveal xref-bibr">25</a></sup> was used to produce large amounts of DNA from individual insects for large-scale genotyping.</p>                            <h3 scrollto-destination=5594929 id="5594929" class="section-title" data-legacy-id=s2c>2.3. Microsatellite isolation, sequencing, and primer design</h3>
<p>Two SSR-enrichment genomic libraries were prepared from Hadano-66 genomic DNA that was extracted from ∼50 insects, including nymphs, adults, males, and females. The first library (referred to as the SREL library) was produced by the Savannah River Ecology Laboratory (University of Georgia), following the method of Glenn and Schable.<sup><a reveal-id="DSS030C26" data-open="DSS030C26" class="link link-ref link-reveal xref-bibr">26</a></sup> Briefly, 2 μg of BPH genomic DNA was digested with <em>Rsa</em>I. The digested DNA was ligated with double-stranded SuperSNX linkers (5′–GTTTAAGGCCTAGCTAGCAGAATC–3′/5′–GATTCTGCTAGCTAGGC CTTAAACAAAA–3′). The linker-ligated DNA was hybridized with seven biotinylated oligo probes: (AG)<sub>12</sub>, (TG)<sub>12</sub>, (AAC)<sub>6</sub>, (AAG)<sub>8</sub>, (AAT)<sub>12</sub>, (ACT)<sub>12</sub>, and (ATC)<sub>8</sub>. The DNA fragments with a microsatellite repeat were captured using Dynabeads (Dynal, Oslo, Norway) under a magnetic field. The amount of eluted DNA was increased by using the polymerase chain reaction (PCR) to recover enriched DNA fragments using the SuperSNX primer pairs.</p><p>The fast isolation by amplified fragment length polymorphism of sequences containing repeats (FIASCO) procedure described by Zane <em>et al</em>.<sup><a reveal-id="DSS030C27" data-open="DSS030C27" class="link link-ref link-reveal xref-bibr">27</a></sup> was used to isolate the second library (referred to as the KU library) at the Plant Breeding Laboratory (Kyushu University). Briefly, genomic DNA was digested with <em>Mse</em>I and ligated to <em>Mse</em>I adaptors (5′–TACTCAGGACTCAT–3/5–GACGATGAGTCCTGAG–3′). The mixture was amplified with an <em>Mse</em>I adaptor primer (5′–GATGAGTCCTGAGTAAN–3′). The PCR product was then hybridized with six biotinylated probes: (CT)<sub>10</sub>, (CA)<sub>10</sub>, (CAC)<sub>7</sub>, (AAG)<sub>7</sub>, (TCC)<sub>7</sub>, and (GACA)<sub>5</sub>. The DNA molecules that were hybridized to biotinylated probes were captured using streptavidin magnetic particles (Roche Applied Science, Mannheim, Germany) using a magnetic field. PCR was performed to recover enriched DNA fragments using the <em>Mse</em>I adaptor primer.</p><p>The SREL and KU libraries were sequenced using the Roche 454 GS FLX Titanium system (454 Life Sciences, Branford, CT, USA). SSRs in the read data were detected using a Perl script in the Simple Sequence Repeat Identification Tool (SSRIT),<sup><a reveal-id="DSS030C28" data-open="DSS030C28" class="link link-ref link-reveal xref-bibr">28</a></sup> with minimum thresholds of nine, six, five, five, five, and five repeat units for di-, tri-, tetra-, penta-, hexa-, and heptanucleotide repeats, respectively. The terminal sequences derived from the ligated adapters were eliminated in both directions. SSR regions were masked by continuous N letters with the same length as the detected SSRs. The masked sequences were then <em>de novo</em> assembled using the Newbler software (454 Life Sciences). A flanking sequence of an appropriate length for primer design was calculated using a Perl script and Primer3.<sup><a reveal-id="DSS030C29" data-open="DSS030C29" class="link link-ref link-reveal xref-bibr">29</a></sup></p><p>To develop e-SSR markers, we obtained the EST sequences contributed by Noda <em>et al</em>.<sup><a reveal-id="DSS030C12" data-open="DSS030C12" class="link link-ref link-reveal xref-bibr">12</a></sup> from the NCBI database (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/">http://www.ncbi.nlm.nih.gov/</a>). The EST sequences were analysed for SSR discovery using SSRIT with minimum thresholds of six, five, five, three, and three repeat units for di-, tri-, tetra-, penta-, and hexanucleotide SSRs, respectively. All EST sequences that contained SSRs were assembled using the CodonCode Aligner software (CodonCode Corporation, <a class="link link-uri" href="http://www.codoncode.com/">http://www.codoncode.com/</a>). Primer3 was used to design the primers. We designed 341 primer pairs, but only 50 primer pairs were randomly selected and used for the polymorphism test.</p><p>The newly developed g-SSR and e-SSR markers were named using the prefixes NLGS (<em>Nilaparvata lugens</em> g-SSR) and NLES (<em>Nilaparvata lugens</em> e-SSR), respectively, followed by the number of the marker. For the primers from Jing <em>et al</em>.,<sup><a reveal-id="DSS030C17" data-open="DSS030C17" class="link link-ref link-reveal xref-bibr">17</a></sup> we retained the original published names. Primer functionality and polymorphism were tested on the parents of the backcross populations. Only clearly polymorphic markers were employed to genotype individuals from the mapping populations. The PCR products were electrophoresed on 4% agarose gels in 0.5× Tris–borate–EDTA buffer, stained with ethidium bromide, and visualized under ultraviolet light. All PCR conditions are summarized in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S1–S3</a>.</p>                            <h3 scrollto-destination=5594948 id="5594948" class="section-title" data-legacy-id=s2d>2.4. SNP discovery and analysis</h3>
<p>SNPs were discovered by comparing whole-genome sequence data from two BPH strains, Izumo-87 (I87)<sup><a reveal-id="DSS030C12" data-open="DSS030C12" class="link link-ref link-reveal xref-bibr">12</a></sup> and IBS-C89. The reference genomic sequences of I87 were acquired using the Roche 454 GS FLX pyrosequencing platform (H. Noda, National Institute of Agrobiological Sciences, unpublished data). To obtain reference genomic contigs, the short-read next-generation sequencing data were subjected to <em>de novo</em> assembly using the Celera Assembler software (v6.1).<sup><a reveal-id="DSS030C30" data-open="DSS030C30" class="link link-ref link-reveal xref-bibr">30</a></sup> The genome sequence of IBS-C89 was performed using the Illumina HiSeq 2000 platform, and a total of about 77.3 million reads were generated. The read data were mapped to reference genomic contigs using the BWA software (v0.6.1).<sup><a reveal-id="DSS030C31" data-open="DSS030C31" class="link link-ref link-reveal xref-bibr">31</a></sup> SNPs between I87 and IBS-C89 were called using the SAMtools/BCFtools software (v0.1.12a)<sup><a reveal-id="DSS030C32" data-open="DSS030C32" class="link link-ref link-reveal xref-bibr">32</a></sup> and visualized using the Tablet software (v1.12).<sup><a reveal-id="DSS030C33" data-open="DSS030C33" class="link link-ref link-reveal xref-bibr">33</a></sup> We randomly selected 234 SNPs that were homogeneous within the strain. PCR primers were designed to amplify 500- to 600-bp fragments carrying each potential SNP. The fragments were sequenced using a 3730 DNA analyzer (Applied Biosystems, Foster City, CA, USA) to screen for polymorphic SNPs between the reference strains (I87 and IBS-C89) and two of the parents of the KH mapping populations (K10 and H66). The selected polymorphic SNPs were used to genotype individual offspring of the KH population. The developed SNP markers were named using the prefixes NLSP (<em>Nilaparvata lugens</em> SNP), followed by the number of the marker. The primers and the PCR conditions are summarized in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>.</p>                            <h3 scrollto-destination=5594953 id="5594953" class="section-title" data-legacy-id=s2e>2.5. Genetic linkage analysis and map construction</h3>
<p>A genetic linkage map of each mapping population was analysed using the JoinMap software (v4.1).<sup><a reveal-id="DSS030C34" data-open="DSS030C34" class="link link-ref link-reveal xref-bibr">34</a></sup> All markers were tested for significant deviation from a normal Mendelian segregation ratio to assess the goodness-of-fit to the expected ratios of 1:1, 1:1:1:1, and 1:2:1 for each marker at a 5% significance level. The ‘outcross pollinated’ (CP) option was used as the population type in this analysis. Grouping of markers was performed using a minimum-independence logarithm of odds (LOD) threshold of 5.0. Groups were converted into maps using the regression algorithm provided by JoinMap with a recombination frequency smaller than 0.45 used for the linkages, and then Kosambi's mapping function<sup><a reveal-id="DSS030C35" data-open="DSS030C35" class="link link-ref link-reveal xref-bibr">35</a></sup> was used to calculate map distances. Distorted markers were retained for mapping, if their presence did not alter the surrounding marker order in a given linkage group (LG).</p><p>On completion of the linkage maps for KH and CK, the individual maps were integrated into a consensus map by applying the ‘combine groups for map integration’ module of JoinMap. An integrated map was created by merging bridging-SSR markers in homologous LGs into a single consensus map. Linkage maps were drawn using the MapChart software for Windows.<sup><a reveal-id="DSS030C36" data-open="DSS030C36" class="link link-ref link-reveal xref-bibr">36</a></sup> The expected genome coverage was estimated according to the method of Chakravarti <em>et al</em>.,<sup><a reveal-id="DSS030C37" data-open="DSS030C37" class="link link-ref link-reveal xref-bibr">37</a></sup> in which the length of each LG was multiplied by (<em>m</em> + 1)/(<em>m</em> − 1), where <em>m</em> is the marker number on each LG.</p>                            <h2 scrollto-destination=5594958 id="5594958" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=5594959 id="5594959" class="section-title" data-legacy-id=s3a>3.1. SSR development and genotyping</h3>
<p>SSR development and primer design: High-throughput sequencing by the 454 GS FLX Titanium generated ∼23.6 Mbp of read data with 87 068 reads from the SREL library. The sequence data were <em>de novo</em> assembled into 1906 contigs. After the assembly, the contigs had an average length of 303 bp. Among the 1906 unique sequences, 970 had flanking regions that were large enough for primer design. For the KU library, ∼24.1 Mbp of read data were generated. From 83 630 reads, 4490 unique contigs were produced, with an average length of 314 bp after the assembly. Of the 4490 contigs, 2556 had flanking regions that were large enough for primer design. Finally, 860 and 2300 primer pairs were designed from the read data for the SREL and KU libraries, respectively.</p><p>SSR polymorphism: A total of 3160 g-SSR primer pairs from the SSR enrichments, 50 newly developed e-SSRs, and 351 publicly available e-SSRs were used to screen for polymorphisms in the parents of the KH and CK populations. Of the 3160 g-SSR markers, 2048 (64.8%) were successfully amplified by PCR. Of these 2048 SSR primer pairs, 922 (45.0%) amplified fragments were polymorphic between the parents. From these polymorphic g-SSRs, only 418 (45.3%) were informative in the mapping populations. The remaining 504 (54.7%) polymorphic markers could not be scored, mostly due to banding patterns that were too difficult to interpret, including multiple-banding patterns, bands that amplified poorly in the offspring, and bands that did not show clear polymorphism on the 4% agarose gel. Of the 50 newly developed e-SSR markers, 15 (30.0%) amplified fragments were polymorphic between the parents. Of the 351 published e-SSR markers that we tested, 317 (90.3%) were successfully amplified by PCR and 100 of the 317 (31.5%) showed polymorphism. Of the total of 1037 polymorphic markers, including g-SSRs, public e-SSRs, and newly developed e-SSRs, only 408 and 213 primer pairs were selected to analyse offspring of the KH and CK populations, respectively. Details of the polymorphic SSR markers are summarized in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S1–S3</a>.</p>                            <h3 scrollto-destination=5594962 id="5594962" class="section-title" data-legacy-id=s3b>3.2. SNP discovery and genotyping</h3>
<p>Of the 234 primer pairs designed to amplify SNP target sequences, 164 (70.1%) were successfully used to amplify a single amplicon of the expected size. Among them, 79 loci (48.2%) were polymorphic between the survey strains (I87 and IBS-C89), and 48 loci (29.3%) were polymorphic between the parents of the KH population. Six SNP markers that were not heterozygous in the offspring of the KH population were excluded from the analysis, leaving 42 loci. After adding 4 SNPs from a preliminary experiment, 46 SNPs were selected and used to genotype the offspring of the KH mapping population. The SNP markers and PCR conditions are summarized in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>.</p>                            <h3 scrollto-destination=5594964 id="5594964" class="section-title" data-legacy-id=s3c>3.3. Linkage analysis and map construction</h3>
<p>Individual genetic linkage maps: The polymorphic markers used for linkage map construction consisted of e-SSR, g-SSR, SNP, and sequence-tagged site (STS) markers. Based on the results of the LG analysis, we constructed individual linkage maps for the KH and CK populations. The KH linkage map was developed from g-SSR, e-SSR, and SNP markers with an LOD score of 5.0. The map consisted of 18 LGs with 357 unique positions (excluding duplicated markers), derived from 348 SSR and 44 SNP markers and covered 870.2 cM. The average distance between the markers was 2.8 cM (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5, Fig. S1</a>). The LGs ranged in length from 3.8 to 83.2 cM. Of these markers, 21 were unlinked or unsuccessfully positioned. Of the 21 unmapped markers, 13 were not linked to another marker and the remaining 8 markers were assigned to LGs, but no map positions were determined. Chi-square analysis indicated that more than 10% of the markers deviated from Mendelian segregation at <em>P</em> &lt; 0.01.</p><p>Only 180 g-SSR markers, 32 e-SSR markers, and 1 STS marker that showed polymorphism among the parents (for a total of 213 markers) were selected to analyse the offspring of the CK population. A linkage map was constructed with a recombination threshold of 0.45, and the minimum number of markers per group was set to two. Linkage analysis was performed at a minimum LOD of 5.0. The linkage map consisted of 207 marker loci in 16 LGs that covered 770.7 cM. (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6, Fig. S1</a>). The LGs ranged in length from 1.4 to 85.9 cM, with an average distance of 4.4 cM between markers. We positioned 43 distorted loci (20.8%) in LGs 1 to 15. The majority was clustered in LG16. We found four unlinked markers and excluded two other distorted markers from the analysis. The Y chromosomal marker PMn3 was mapped at the end of the LG arm next to NLGS201 in LG16 of the CK population.</p><p>Consensus linkage map: The consensus map was constructed by integrating linkage data from the two backcross populations (KH and CK). The ‘combine groups for map integration’ module of JoinMap was used to integrate the two individual maps into a consensus map. A total of 73 common SSR markers between the homologous LGs were used as bridges to integrate the individual maps. The order of the bridged markers was consistent between the two individual maps, except for 16 marker inversions in LG2 to LG8, in LG10, and in LG12 to LG13 (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). The final consensus map contained 400 g-SSR, 74 e-SSR, 43 SNP, and 1 STS markers (for a total of 518), with 472 unique positions in 17 LGs. Table <a reveal-id="DSS030TB1" data-open="DSS030TB1" class="link link-reveal link-table xref-fig">1</a> summarizes the statistics for the consensus LGs, and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S7</a> reports the locus name and its position for all loci in the 17 LGs in the consensus map. The length of the 17 LGs ranged from 19.0 cM (LG17) to 98.1 cM (LG13). The average genetic distance between loci was 2.3 cM. Gaps larger than 20 cM were found in LG 8 to LG11 and in LG14 (Fig. <a reveal-id="DSS030F2" data-open="DSS030F2" class="link link-reveal link-table xref-fig">2</a>). The largest gap in the framework map was 43.9 cM and occurred in LG8. The large gaps observed in the individual maps were reduced in the consensus map. In total, 30 markers remained unlinked. The number of integrated (consensus) LGs was close to the actual number of haploid chromosomes in BPH (<em>n</em> = 15). The 17 LGs contained 518 loci that encompassed 1093.9 cM of the total map distance (excluding the distance from PM3n to the distal end of LG11) and covered 92.3% of the estimated genome size. According to the estimated genome size of BPH (1200 Mbp),<sup><a reveal-id="DSS030C11" data-open="DSS030C11" class="link link-ref link-reveal xref-bibr">11</a></sup> the recombination rate is ∼0.91 cM/Mbp. </p>                        <a id="5594968" scrollto-destination="5594968"></a>
<div content-id="DSS030TB1" class="table-modal"><div class="table-wrap"><div id="DSS030TB1" class="table-wrap-title" data-id="DSS030TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Description of the integrated consensus map</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">LG </th><th rowspan="2">Total length (cM) </th><th rowspan="2">No. of loci </th><th rowspan="2">No. of markers </th><th rowspan="2">Average distance (cM) </th><th rowspan="2">No. of distorted markers (<em>P</em> &lt; 0.01) </th><th rowspan="2">Estimated genome size (cM) </th><th rowspan="2">Genome coverage (%) </th><th rowspan="2">Largest interval (cM) </th><th rowspan="2">No. of bridging markers </th><th colspan="4">DNA markers<hr /> </th></tr><tr><th>e-SSR </th><th>g-SSR </th><th>SNP </th><th>STS </th></tr></thead><tbody><tr><td>LG1 </td><td>45.7 </td><td>42 </td><td>42 </td><td>1.1 </td><td>8 </td><td>48.0 </td><td>95.3 </td><td>5.7 </td><td>3 </td><td>4 </td><td>32 </td><td>6 </td><td>0 </td></tr><tr><td>LG2 </td><td>67.1 </td><td>37 </td><td>46 </td><td>1.8 </td><td>2 </td><td>70.1 </td><td>95.7 </td><td>12.3 </td><td>7 </td><td>10 </td><td>30 </td><td>6 </td><td>0 </td></tr><tr><td>LG3 </td><td>75.0 </td><td>39 </td><td>40 </td><td>1.9 </td><td>3 </td><td>78.8 </td><td>95.1 </td><td>8.5 </td><td>6 </td><td>13 </td><td>24 </td><td>3 </td><td>0 </td></tr><tr><td>LG4 </td><td>81.0 </td><td>32 </td><td>37 </td><td>2.5 </td><td>3 </td><td>85.5 </td><td>94.7 </td><td>18.7 </td><td>7 </td><td>2 </td><td>34 </td><td>1 </td><td>0 </td></tr><tr><td>LG5 </td><td>43.5 </td><td>35 </td><td>36 </td><td>1.2 </td><td>1 </td><td>46.0 </td><td>94.6 </td><td>6.5 </td><td>10 </td><td>3 </td><td>33 </td><td>0 </td><td>0 </td></tr><tr><td>LG6 </td><td>58.6 </td><td>42 </td><td>46 </td><td>1.4 </td><td>2 </td><td>61.2 </td><td>95.7 </td><td>10.1 </td><td>5 </td><td>5 </td><td>40 </td><td>1 </td><td>0 </td></tr><tr><td>LG7 </td><td>19.2 </td><td>24 </td><td>40 </td><td>0.8 </td><td>6 </td><td>20.2 </td><td>95.1 </td><td>5.4 </td><td>6 </td><td>4 </td><td>34 </td><td>2 </td><td>0 </td></tr><tr><td>LG8 </td><td>97.9 </td><td>29 </td><td>32 </td><td>3.4 </td><td>5 </td><td>104.2 </td><td>93.9 </td><td>44.0 </td><td>2 </td><td>3 </td><td>25 </td><td>4 </td><td>0 </td></tr><tr><td>LG9 </td><td>69.7 </td><td>20 </td><td>22 </td><td>3.5 </td><td>0 </td><td>76.4 </td><td>91.3 </td><td>34.8 </td><td>0 </td><td>5 </td><td>14 </td><td>3 </td><td>0 </td></tr><tr><td>LG10 </td><td>74.6 </td><td>29 </td><td>29 </td><td>2.6 </td><td>4 </td><td>79.9 </td><td>93.3 </td><td>21.7 </td><td>8 </td><td>1 </td><td>28 </td><td>0 </td><td>0 </td></tr><tr><td>LG11 </td><td>53.5 </td><td>27 </td><td>27 </td><td>2.0 </td><td>22 </td><td>57.8 </td><td>92.6 </td><td>16.3 </td><td>3 </td><td>8 </td><td>11 </td><td>7 </td><td>1 </td></tr><tr><td>LG12 </td><td>75.6 </td><td>26 </td><td>26 </td><td>2.9 </td><td>3 </td><td>81.6 </td><td>92.6 </td><td>9.0 </td><td>5 </td><td>6 </td><td>17 </td><td>3 </td><td>0 </td></tr><tr><td>LG13 </td><td>98.1 </td><td>23 </td><td>24 </td><td>4.3 </td><td>5 </td><td>106.6 </td><td>92.0 </td><td>19.8 </td><td>3 </td><td>5 </td><td>16 </td><td>3 </td><td>0 </td></tr><tr><td>LG14 </td><td>77.0 </td><td>26 </td><td>27 </td><td>3.0 </td><td>0 </td><td>82.9 </td><td>92.9 </td><td>23.4 </td><td>3 </td><td>1 </td><td>25 </td><td>1 </td><td>0 </td></tr><tr><td>LG15 </td><td>52.4 </td><td>18 </td><td>18 </td><td>2.9 </td><td>0 </td><td>58.5 </td><td>89.5 </td><td>11.4 </td><td>3 </td><td>3 </td><td>12 </td><td>3 </td><td>0 </td></tr><tr><td>LG16 </td><td>86.0 </td><td>16 </td><td>19 </td><td>5.4 </td><td>3 </td><td>95.6 </td><td>90.0 </td><td>19.2 </td><td>2 </td><td>1 </td><td>18 </td><td>0 </td><td>0 </td></tr><tr><td>LG17 </td><td>19.0 </td><td>7 </td><td>7 </td><td>2.7 </td><td>0 </td><td>25.3 </td><td>75.0 </td><td>6.5 </td><td>0 </td><td>0 </td><td>7 </td><td>0 </td><td>0 </td></tr><tr><td>Total </td><td>1093.9 </td><td>472 </td><td>518 </td><td> </td><td>67.0 </td><td>1178.6 </td><td> </td><td>273.0 </td><td>73 </td><td>74 </td><td>400 </td><td>43 </td><td>1 </td></tr><tr><td>Average </td><td>64.3 </td><td>27.8 </td><td>30.5 </td><td>2.3 </td><td>3.9 </td><td>69.3 </td><td>92.3 </td><td>16.1 </td><td>4.3 </td><td>4.4 </td><td>23.5 </td><td>2.5 </td><td>0.1 </td></tr></tbody></table></div><div content-id="DSS030TB1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">LG </th><th rowspan="2">Total length (cM) </th><th rowspan="2">No. of loci </th><th rowspan="2">No. of markers </th><th rowspan="2">Average distance (cM) </th><th rowspan="2">No. of distorted markers (<em>P</em> &lt; 0.01) </th><th rowspan="2">Estimated genome size (cM) </th><th rowspan="2">Genome coverage (%) </th><th rowspan="2">Largest interval (cM) </th><th rowspan="2">No. of bridging markers </th><th colspan="4">DNA markers<hr /> </th></tr><tr><th>e-SSR </th><th>g-SSR </th><th>SNP </th><th>STS </th></tr></thead><tbody><tr><td>LG1 </td><td>45.7 </td><td>42 </td><td>42 </td><td>1.1 </td><td>8 </td><td>48.0 </td><td>95.3 </td><td>5.7 </td><td>3 </td><td>4 </td><td>32 </td><td>6 </td><td>0 </td></tr><tr><td>LG2 </td><td>67.1 </td><td>37 </td><td>46 </td><td>1.8 </td><td>2 </td><td>70.1 </td><td>95.7 </td><td>12.3 </td><td>7 </td><td>10 </td><td>30 </td><td>6 </td><td>0 </td></tr><tr><td>LG3 </td><td>75.0 </td><td>39 </td><td>40 </td><td>1.9 </td><td>3 </td><td>78.8 </td><td>95.1 </td><td>8.5 </td><td>6 </td><td>13 </td><td>24 </td><td>3 </td><td>0 </td></tr><tr><td>LG4 </td><td>81.0 </td><td>32 </td><td>37 </td><td>2.5 </td><td>3 </td><td>85.5 </td><td>94.7 </td><td>18.7 </td><td>7 </td><td>2 </td><td>34 </td><td>1 </td><td>0 </td></tr><tr><td>LG5 </td><td>43.5 </td><td>35 </td><td>36 </td><td>1.2 </td><td>1 </td><td>46.0 </td><td>94.6 </td><td>6.5 </td><td>10 </td><td>3 </td><td>33 </td><td>0 </td><td>0 </td></tr><tr><td>LG6 </td><td>58.6 </td><td>42 </td><td>46 </td><td>1.4 </td><td>2 </td><td>61.2 </td><td>95.7 </td><td>10.1 </td><td>5 </td><td>5 </td><td>40 </td><td>1 </td><td>0 </td></tr><tr><td>LG7 </td><td>19.2 </td><td>24 </td><td>40 </td><td>0.8 </td><td>6 </td><td>20.2 </td><td>95.1 </td><td>5.4 </td><td>6 </td><td>4 </td><td>34 </td><td>2 </td><td>0 </td></tr><tr><td>LG8 </td><td>97.9 </td><td>29 </td><td>32 </td><td>3.4 </td><td>5 </td><td>104.2 </td><td>93.9 </td><td>44.0 </td><td>2 </td><td>3 </td><td>25 </td><td>4 </td><td>0 </td></tr><tr><td>LG9 </td><td>69.7 </td><td>20 </td><td>22 </td><td>3.5 </td><td>0 </td><td>76.4 </td><td>91.3 </td><td>34.8 </td><td>0 </td><td>5 </td><td>14 </td><td>3 </td><td>0 </td></tr><tr><td>LG10 </td><td>74.6 </td><td>29 </td><td>29 </td><td>2.6 </td><td>4 </td><td>79.9 </td><td>93.3 </td><td>21.7 </td><td>8 </td><td>1 </td><td>28 </td><td>0 </td><td>0 </td></tr><tr><td>LG11 </td><td>53.5 </td><td>27 </td><td>27 </td><td>2.0 </td><td>22 </td><td>57.8 </td><td>92.6 </td><td>16.3 </td><td>3 </td><td>8 </td><td>11 </td><td>7 </td><td>1 </td></tr><tr><td>LG12 </td><td>75.6 </td><td>26 </td><td>26 </td><td>2.9 </td><td>3 </td><td>81.6 </td><td>92.6 </td><td>9.0 </td><td>5 </td><td>6 </td><td>17 </td><td>3 </td><td>0 </td></tr><tr><td>LG13 </td><td>98.1 </td><td>23 </td><td>24 </td><td>4.3 </td><td>5 </td><td>106.6 </td><td>92.0 </td><td>19.8 </td><td>3 </td><td>5 </td><td>16 </td><td>3 </td><td>0 </td></tr><tr><td>LG14 </td><td>77.0 </td><td>26 </td><td>27 </td><td>3.0 </td><td>0 </td><td>82.9 </td><td>92.9 </td><td>23.4 </td><td>3 </td><td>1 </td><td>25 </td><td>1 </td><td>0 </td></tr><tr><td>LG15 </td><td>52.4 </td><td>18 </td><td>18 </td><td>2.9 </td><td>0 </td><td>58.5 </td><td>89.5 </td><td>11.4 </td><td>3 </td><td>3 </td><td>12 </td><td>3 </td><td>0 </td></tr><tr><td>LG16 </td><td>86.0 </td><td>16 </td><td>19 </td><td>5.4 </td><td>3 </td><td>95.6 </td><td>90.0 </td><td>19.2 </td><td>2 </td><td>1 </td><td>18 </td><td>0 </td><td>0 </td></tr><tr><td>LG17 </td><td>19.0 </td><td>7 </td><td>7 </td><td>2.7 </td><td>0 </td><td>25.3 </td><td>75.0 </td><td>6.5 </td><td>0 </td><td>0 </td><td>7 </td><td>0 </td><td>0 </td></tr><tr><td>Total </td><td>1093.9 </td><td>472 </td><td>518 </td><td> </td><td>67.0 </td><td>1178.6 </td><td> </td><td>273.0 </td><td>73 </td><td>74 </td><td>400 </td><td>43 </td><td>1 </td></tr><tr><td>Average </td><td>64.3 </td><td>27.8 </td><td>30.5 </td><td>2.3 </td><td>3.9 </td><td>69.3 </td><td>92.3 </td><td>16.1 </td><td>4.3 </td><td>4.4 </td><td>23.5 </td><td>2.5 </td><td>0.1 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5594968" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSS030TB1" class="table-wrap-title" data-id="DSS030TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Description of the integrated consensus map</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">LG </th><th rowspan="2">Total length (cM) </th><th rowspan="2">No. of loci </th><th rowspan="2">No. of markers </th><th rowspan="2">Average distance (cM) </th><th rowspan="2">No. of distorted markers (<em>P</em> &lt; 0.01) </th><th rowspan="2">Estimated genome size (cM) </th><th rowspan="2">Genome coverage (%) </th><th rowspan="2">Largest interval (cM) </th><th rowspan="2">No. of bridging markers </th><th colspan="4">DNA markers<hr /> </th></tr><tr><th>e-SSR </th><th>g-SSR </th><th>SNP </th><th>STS </th></tr></thead><tbody><tr><td>LG1 </td><td>45.7 </td><td>42 </td><td>42 </td><td>1.1 </td><td>8 </td><td>48.0 </td><td>95.3 </td><td>5.7 </td><td>3 </td><td>4 </td><td>32 </td><td>6 </td><td>0 </td></tr><tr><td>LG2 </td><td>67.1 </td><td>37 </td><td>46 </td><td>1.8 </td><td>2 </td><td>70.1 </td><td>95.7 </td><td>12.3 </td><td>7 </td><td>10 </td><td>30 </td><td>6 </td><td>0 </td></tr><tr><td>LG3 </td><td>75.0 </td><td>39 </td><td>40 </td><td>1.9 </td><td>3 </td><td>78.8 </td><td>95.1 </td><td>8.5 </td><td>6 </td><td>13 </td><td>24 </td><td>3 </td><td>0 </td></tr><tr><td>LG4 </td><td>81.0 </td><td>32 </td><td>37 </td><td>2.5 </td><td>3 </td><td>85.5 </td><td>94.7 </td><td>18.7 </td><td>7 </td><td>2 </td><td>34 </td><td>1 </td><td>0 </td></tr><tr><td>LG5 </td><td>43.5 </td><td>35 </td><td>36 </td><td>1.2 </td><td>1 </td><td>46.0 </td><td>94.6 </td><td>6.5 </td><td>10 </td><td>3 </td><td>33 </td><td>0 </td><td>0 </td></tr><tr><td>LG6 </td><td>58.6 </td><td>42 </td><td>46 </td><td>1.4 </td><td>2 </td><td>61.2 </td><td>95.7 </td><td>10.1 </td><td>5 </td><td>5 </td><td>40 </td><td>1 </td><td>0 </td></tr><tr><td>LG7 </td><td>19.2 </td><td>24 </td><td>40 </td><td>0.8 </td><td>6 </td><td>20.2 </td><td>95.1 </td><td>5.4 </td><td>6 </td><td>4 </td><td>34 </td><td>2 </td><td>0 </td></tr><tr><td>LG8 </td><td>97.9 </td><td>29 </td><td>32 </td><td>3.4 </td><td>5 </td><td>104.2 </td><td>93.9 </td><td>44.0 </td><td>2 </td><td>3 </td><td>25 </td><td>4 </td><td>0 </td></tr><tr><td>LG9 </td><td>69.7 </td><td>20 </td><td>22 </td><td>3.5 </td><td>0 </td><td>76.4 </td><td>91.3 </td><td>34.8 </td><td>0 </td><td>5 </td><td>14 </td><td>3 </td><td>0 </td></tr><tr><td>LG10 </td><td>74.6 </td><td>29 </td><td>29 </td><td>2.6 </td><td>4 </td><td>79.9 </td><td>93.3 </td><td>21.7 </td><td>8 </td><td>1 </td><td>28 </td><td>0 </td><td>0 </td></tr><tr><td>LG11 </td><td>53.5 </td><td>27 </td><td>27 </td><td>2.0 </td><td>22 </td><td>57.8 </td><td>92.6 </td><td>16.3 </td><td>3 </td><td>8 </td><td>11 </td><td>7 </td><td>1 </td></tr><tr><td>LG12 </td><td>75.6 </td><td>26 </td><td>26 </td><td>2.9 </td><td>3 </td><td>81.6 </td><td>92.6 </td><td>9.0 </td><td>5 </td><td>6 </td><td>17 </td><td>3 </td><td>0 </td></tr><tr><td>LG13 </td><td>98.1 </td><td>23 </td><td>24 </td><td>4.3 </td><td>5 </td><td>106.6 </td><td>92.0 </td><td>19.8 </td><td>3 </td><td>5 </td><td>16 </td><td>3 </td><td>0 </td></tr><tr><td>LG14 </td><td>77.0 </td><td>26 </td><td>27 </td><td>3.0 </td><td>0 </td><td>82.9 </td><td>92.9 </td><td>23.4 </td><td>3 </td><td>1 </td><td>25 </td><td>1 </td><td>0 </td></tr><tr><td>LG15 </td><td>52.4 </td><td>18 </td><td>18 </td><td>2.9 </td><td>0 </td><td>58.5 </td><td>89.5 </td><td>11.4 </td><td>3 </td><td>3 </td><td>12 </td><td>3 </td><td>0 </td></tr><tr><td>LG16 </td><td>86.0 </td><td>16 </td><td>19 </td><td>5.4 </td><td>3 </td><td>95.6 </td><td>90.0 </td><td>19.2 </td><td>2 </td><td>1 </td><td>18 </td><td>0 </td><td>0 </td></tr><tr><td>LG17 </td><td>19.0 </td><td>7 </td><td>7 </td><td>2.7 </td><td>0 </td><td>25.3 </td><td>75.0 </td><td>6.5 </td><td>0 </td><td>0 </td><td>7 </td><td>0 </td><td>0 </td></tr><tr><td>Total </td><td>1093.9 </td><td>472 </td><td>518 </td><td> </td><td>67.0 </td><td>1178.6 </td><td> </td><td>273.0 </td><td>73 </td><td>74 </td><td>400 </td><td>43 </td><td>1 </td></tr><tr><td>Average </td><td>64.3 </td><td>27.8 </td><td>30.5 </td><td>2.3 </td><td>3.9 </td><td>69.3 </td><td>92.3 </td><td>16.1 </td><td>4.3 </td><td>4.4 </td><td>23.5 </td><td>2.5 </td><td>0.1 </td></tr></tbody></table></div><div content-id="DSS030TB1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">LG </th><th rowspan="2">Total length (cM) </th><th rowspan="2">No. of loci </th><th rowspan="2">No. of markers </th><th rowspan="2">Average distance (cM) </th><th rowspan="2">No. of distorted markers (<em>P</em> &lt; 0.01) </th><th rowspan="2">Estimated genome size (cM) </th><th rowspan="2">Genome coverage (%) </th><th rowspan="2">Largest interval (cM) </th><th rowspan="2">No. of bridging markers </th><th colspan="4">DNA markers<hr /> </th></tr><tr><th>e-SSR </th><th>g-SSR </th><th>SNP </th><th>STS </th></tr></thead><tbody><tr><td>LG1 </td><td>45.7 </td><td>42 </td><td>42 </td><td>1.1 </td><td>8 </td><td>48.0 </td><td>95.3 </td><td>5.7 </td><td>3 </td><td>4 </td><td>32 </td><td>6 </td><td>0 </td></tr><tr><td>LG2 </td><td>67.1 </td><td>37 </td><td>46 </td><td>1.8 </td><td>2 </td><td>70.1 </td><td>95.7 </td><td>12.3 </td><td>7 </td><td>10 </td><td>30 </td><td>6 </td><td>0 </td></tr><tr><td>LG3 </td><td>75.0 </td><td>39 </td><td>40 </td><td>1.9 </td><td>3 </td><td>78.8 </td><td>95.1 </td><td>8.5 </td><td>6 </td><td>13 </td><td>24 </td><td>3 </td><td>0 </td></tr><tr><td>LG4 </td><td>81.0 </td><td>32 </td><td>37 </td><td>2.5 </td><td>3 </td><td>85.5 </td><td>94.7 </td><td>18.7 </td><td>7 </td><td>2 </td><td>34 </td><td>1 </td><td>0 </td></tr><tr><td>LG5 </td><td>43.5 </td><td>35 </td><td>36 </td><td>1.2 </td><td>1 </td><td>46.0 </td><td>94.6 </td><td>6.5 </td><td>10 </td><td>3 </td><td>33 </td><td>0 </td><td>0 </td></tr><tr><td>LG6 </td><td>58.6 </td><td>42 </td><td>46 </td><td>1.4 </td><td>2 </td><td>61.2 </td><td>95.7 </td><td>10.1 </td><td>5 </td><td>5 </td><td>40 </td><td>1 </td><td>0 </td></tr><tr><td>LG7 </td><td>19.2 </td><td>24 </td><td>40 </td><td>0.8 </td><td>6 </td><td>20.2 </td><td>95.1 </td><td>5.4 </td><td>6 </td><td>4 </td><td>34 </td><td>2 </td><td>0 </td></tr><tr><td>LG8 </td><td>97.9 </td><td>29 </td><td>32 </td><td>3.4 </td><td>5 </td><td>104.2 </td><td>93.9 </td><td>44.0 </td><td>2 </td><td>3 </td><td>25 </td><td>4 </td><td>0 </td></tr><tr><td>LG9 </td><td>69.7 </td><td>20 </td><td>22 </td><td>3.5 </td><td>0 </td><td>76.4 </td><td>91.3 </td><td>34.8 </td><td>0 </td><td>5 </td><td>14 </td><td>3 </td><td>0 </td></tr><tr><td>LG10 </td><td>74.6 </td><td>29 </td><td>29 </td><td>2.6 </td><td>4 </td><td>79.9 </td><td>93.3 </td><td>21.7 </td><td>8 </td><td>1 </td><td>28 </td><td>0 </td><td>0 </td></tr><tr><td>LG11 </td><td>53.5 </td><td>27 </td><td>27 </td><td>2.0 </td><td>22 </td><td>57.8 </td><td>92.6 </td><td>16.3 </td><td>3 </td><td>8 </td><td>11 </td><td>7 </td><td>1 </td></tr><tr><td>LG12 </td><td>75.6 </td><td>26 </td><td>26 </td><td>2.9 </td><td>3 </td><td>81.6 </td><td>92.6 </td><td>9.0 </td><td>5 </td><td>6 </td><td>17 </td><td>3 </td><td>0 </td></tr><tr><td>LG13 </td><td>98.1 </td><td>23 </td><td>24 </td><td>4.3 </td><td>5 </td><td>106.6 </td><td>92.0 </td><td>19.8 </td><td>3 </td><td>5 </td><td>16 </td><td>3 </td><td>0 </td></tr><tr><td>LG14 </td><td>77.0 </td><td>26 </td><td>27 </td><td>3.0 </td><td>0 </td><td>82.9 </td><td>92.9 </td><td>23.4 </td><td>3 </td><td>1 </td><td>25 </td><td>1 </td><td>0 </td></tr><tr><td>LG15 </td><td>52.4 </td><td>18 </td><td>18 </td><td>2.9 </td><td>0 </td><td>58.5 </td><td>89.5 </td><td>11.4 </td><td>3 </td><td>3 </td><td>12 </td><td>3 </td><td>0 </td></tr><tr><td>LG16 </td><td>86.0 </td><td>16 </td><td>19 </td><td>5.4 </td><td>3 </td><td>95.6 </td><td>90.0 </td><td>19.2 </td><td>2 </td><td>1 </td><td>18 </td><td>0 </td><td>0 </td></tr><tr><td>LG17 </td><td>19.0 </td><td>7 </td><td>7 </td><td>2.7 </td><td>0 </td><td>25.3 </td><td>75.0 </td><td>6.5 </td><td>0 </td><td>0 </td><td>7 </td><td>0 </td><td>0 </td></tr><tr><td>Total </td><td>1093.9 </td><td>472 </td><td>518 </td><td> </td><td>67.0 </td><td>1178.6 </td><td> </td><td>273.0 </td><td>73 </td><td>74 </td><td>400 </td><td>43 </td><td>1 </td></tr><tr><td>Average </td><td>64.3 </td><td>27.8 </td><td>30.5 </td><td>2.3 </td><td>3.9 </td><td>69.3 </td><td>92.3 </td><td>16.1 </td><td>4.3 </td><td>4.4 </td><td>23.5 </td><td>2.5 </td><td>0.1 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5594968" target="_blank">View Large</a></div></div>                        <a id="5594969" scrollto-destination="5594969"></a>
<div id="DSS030F2" data-id="DSS030F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss030/2/m_dss03002a.jpeg?Expires=2147483647&amp;Signature=YKnVtb7X2vcSGPiwSNQmNSxW3oaw8zWdw~8~JVqfGOVY59pMEzI616NNwR6UaX7Rjk4d2G1pAV2vRGZwFiDPxpZwo6vPDXhQQMkrtLOzXEOhcRUW-v5AmBCRyyqEC3ntxTN-vxFo3~UeLLDVlU42ADjhD~hS4OfVkR2uNQbq8g8F4c75EvF4zFVd0ovpO8c4uEsUUhjbpYea1DPm4LY2-DDPf0JIx7M7QG4vvovMPlbQsaUkTR1NoRpiM~3LqZ2c3Ov6P509S1zfXlMi7e2QCcSbEaBX0jVkZwztir9vUb2apLaVdjrC42Jf2GrOzAfu4ExiCPXZDyeFso2-3LFJNA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="A genetic linkage map of BPH. The LGs were constructed by integrating linkage data from two backcross populations (KH and CK). Grouping of markers was performed using an independence LOD threshold of five. Groups were converted to maps using the regression algorithm provided by JoinMap, with a recombination frequency smaller than 0.45 and Kosambi's mapping function applied for calculation of map distances. The LGs have been arbitrarily numbered LG1–LG17 based on the order generated by JoinMap (v4.1). Distances in centimorgans are indicated at the left of each LG. Marker names starting with NLGS (black) represent g-SSRs, those starting with NLES and BM represent e-SSRs (blue), and those starting with NLSP (red) represent SNP markers. PM3n is an STS marker located on the Y chromosome. A gap between the marker BM500 and PM3n on the LG11 is expected to include the differentiated region of the Y chromosome. e-SSR markers BM13, BM14, BM204, BM464, BM499, and BM500 are for vitellogenin genes that are genes with female-specific expression located on the X chromosome. Segregation-distorted loci indicate different significant levels; *P &lt; 0.01 and **P &lt; 0.001." path-from-xml="dss03002a.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5594969/dss03002a.jpeg" path-from-xml="dss03002a.jpeg" target="_blank">View large</a><a section="5594969" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss030/2/dss03002a.jpeg?Expires=2147483647&amp;Signature=wixgsmMcB85RiQhK01EDLFm9nLr2ab3HhaptWYrXRfqk~c4ZpLJlMtu6oIyk61otiyIynGlPV~ZjBwMK-H7bOx3cCTFoOFt6Iwa40poLizu-EN7YoZZjRZl5pCOoHuFfYKo9mRgEVO5Qf-aVs23Qprad4NRutp3hRDLA1PuhXPEr1fLPzHXOLOAxtb8~qGqlZKSpQDDjFYK0RO2Z~7hogXbI~MID5XqMYWbJfvIBL5uoWNNkomCTLPFkjqcZ2i-emV0dk8NcjvjSgnTPqKzpurx-~qc0jpYJu~onTMivnQFAr64wdm-bXgQ8RDTPxG4VWw5aOhhKtRcKtaQ5Ec7K4A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=5594969&amp;ar=382150&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dss03002a.jpeg" target="_blank">Download slide</a></div></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss030/2/m_dss03002b.jpeg?Expires=2147483647&amp;Signature=mMx87WbwdUrzwM-uKTAHPxEdZWN4XYLxOKdQtePysrKGUUtWIybdwmaUFwGRBRoXvGnzReMq5rAsWf981To4ha30Zr7yP1FOkAbbJIr6Dyg8jOZzFzxzbtZVNgknuHUN~fDjlogoli58fKmQAJUkmSVo5T5gOn8XhUj1CV~E5kQl2lh1U9QeiugOLDFruR7M9coHUEDbQDyx7~tE1q5DVwOyy2HsLheKpGw8D16925qHHeo1Upq1-ffRBX3~V3Pzjd~LJpgkpeqSHVyJ6E5GjwGPE13LrBsJmdYz5XTwYaof95Hmp~o8F25yywmSIUmiPGMk1nHOtYu7mZGszIMI8w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="A genetic linkage map of BPH. The LGs were constructed by integrating linkage data from two backcross populations (KH and CK). Grouping of markers was performed using an independence LOD threshold of five. Groups were converted to maps using the regression algorithm provided by JoinMap, with a recombination frequency smaller than 0.45 and Kosambi's mapping function applied for calculation of map distances. The LGs have been arbitrarily numbered LG1–LG17 based on the order generated by JoinMap (v4.1). Distances in centimorgans are indicated at the left of each LG. Marker names starting with NLGS (black) represent g-SSRs, those starting with NLES and BM represent e-SSRs (blue), and those starting with NLSP (red) represent SNP markers. PM3n is an STS marker located on the Y chromosome. A gap between the marker BM500 and PM3n on the LG11 is expected to include the differentiated region of the Y chromosome. e-SSR markers BM13, BM14, BM204, BM464, BM499, and BM500 are for vitellogenin genes that are genes with female-specific expression located on the X chromosome. Segregation-distorted loci indicate different significant levels; *P &lt; 0.01 and **P &lt; 0.001." path-from-xml="dss03002b.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5594969/dss03002b.jpeg" path-from-xml="dss03002b.jpeg" target="_blank">View large</a><a section="5594969" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss030/2/dss03002b.jpeg?Expires=2147483647&amp;Signature=cN5sXFNH0lQOvvhEUHUEo5VZ6iSlgPcL4f1pCLA-RsaR6jWYPWSUbnKr~0frDMICXQFeZ6bHYAAfmSVFZiN-2aEmt1qDLzfptXSiN8Pr2oqitTNiaRsXCo5K~BgFXlv8VDSiIwuzURaksg54kTvXaEpk-QRnbW2SLL~l2r32CbBMkW~8X0nFjcOQRi7ORaA1F8dNcQbpDOdHRkFti8dqqED0ecEUm6j9pX57SrmwkIBiTXVH4se7p0jsztiIkWyBrewkOnvA74lxp8hMx~yG4A~LFSPH~4oFGEwJzSnGrTrm-pnx8TCJaLX6aetfLoC79nTmUDbVw5GDn8Q3pyCoGw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=5594969&amp;ar=382150&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dss03002b.jpeg" target="_blank">Download slide</a></div></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss030/2/m_dss03002c.jpeg?Expires=2147483647&amp;Signature=qhbWikYHTXeZm5PJDYGtShcZUgmjjxGomcr~l-RHLQbqm7WwPwNNc-azxZ05E6g23Sfzy~EgT71IlFKVjdRNjv8lEw0xaDVUayTXev3AZStOMBNwpYe6T4olGlkmI~sDtEYVJMMczAQIckqX0EFmiupZMuodQGMU9OuBQ7rwTzhro7F8lIHtocbQ8oSs~Vy3NaSRk1G7ZfcodhFGBRv-1N0PW-65xmLhYmh13kTR9ffkjxAuM-ANpXjPiCsevnR-yZnJbwEo5olj4thj30tfJPjP5IgXArlJZKvQA8QFbOcy1BqxZnhZ3PKqjcqq3Cy8Lg~oCqNA5q~impGd7vISjA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="A genetic linkage map of BPH. The LGs were constructed by integrating linkage data from two backcross populations (KH and CK). Grouping of markers was performed using an independence LOD threshold of five. Groups were converted to maps using the regression algorithm provided by JoinMap, with a recombination frequency smaller than 0.45 and Kosambi's mapping function applied for calculation of map distances. The LGs have been arbitrarily numbered LG1–LG17 based on the order generated by JoinMap (v4.1). Distances in centimorgans are indicated at the left of each LG. Marker names starting with NLGS (black) represent g-SSRs, those starting with NLES and BM represent e-SSRs (blue), and those starting with NLSP (red) represent SNP markers. PM3n is an STS marker located on the Y chromosome. A gap between the marker BM500 and PM3n on the LG11 is expected to include the differentiated region of the Y chromosome. e-SSR markers BM13, BM14, BM204, BM464, BM499, and BM500 are for vitellogenin genes that are genes with female-specific expression located on the X chromosome. Segregation-distorted loci indicate different significant levels; *P &lt; 0.01 and **P &lt; 0.001." path-from-xml="dss03002c.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5594969/dss03002c.jpeg" path-from-xml="dss03002c.jpeg" target="_blank">View large</a><a section="5594969" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss030/2/dss03002c.jpeg?Expires=2147483647&amp;Signature=S70-xFyzsoMva3DiShCel3fd5gNgZGVwxObJVdFNpN3TSJkAr5bWUYxJkzDvumIEtxBXtxvbzSkq~uEnuoaqvRyvX7VnW82wsdrAVb3888ivP6JK5gAjt3IwmkBzoyLofa50tQjC4jmVPFHPpBzmQSVW24-nfsm1bOWN3aU~QfAG2~yThvRKHvvSWXWBQP3OOhYaZ6HruKYYiwOAKFS3vFJUg4IojqyspojyiJbPFXmNLE2Ft9tQa~X8JIJCWAKCd2cpNB7RQyDPvdFKriIv0t6SBnIdUrQSZsvPVpqpK17TgrBByJPjPrMpl1XSkTFWdtBH-XVu1o0WVgER3IhKsg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=5594969&amp;ar=382150&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dss03002c.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>A genetic linkage map of BPH. The LGs were constructed by integrating linkage data from two backcross populations (KH and CK). Grouping of markers was performed using an independence LOD threshold of five. Groups were converted to maps using the regression algorithm provided by JoinMap, with a recombination frequency smaller than 0.45 and Kosambi's mapping function applied for calculation of map distances. The LGs have been arbitrarily numbered LG1–LG17 based on the order generated by JoinMap (v4.1). Distances in centimorgans are indicated at the left of each LG. Marker names starting with NLGS (black) represent g-SSRs, those starting with NLES and BM represent e-SSRs (blue), and those starting with NLSP (red) represent SNP markers. PM3n is an STS marker located on the Y chromosome. A gap between the marker BM500 and PM3n on the LG11 is expected to include the differentiated region of the Y chromosome. e-SSR markers BM13, BM14, BM204, BM464, BM499, and BM500 are for vitellogenin genes that are genes with female-specific expression located on the X chromosome. Segregation-distorted loci indicate different significant levels; *<em>P</em> &lt; 0.01 and **<em>P</em> &lt; 0.001.</p></div></div><div content-id="DSS030F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss030/2/m_dss03002a.jpeg?Expires=2147483647&amp;Signature=YKnVtb7X2vcSGPiwSNQmNSxW3oaw8zWdw~8~JVqfGOVY59pMEzI616NNwR6UaX7Rjk4d2G1pAV2vRGZwFiDPxpZwo6vPDXhQQMkrtLOzXEOhcRUW-v5AmBCRyyqEC3ntxTN-vxFo3~UeLLDVlU42ADjhD~hS4OfVkR2uNQbq8g8F4c75EvF4zFVd0ovpO8c4uEsUUhjbpYea1DPm4LY2-DDPf0JIx7M7QG4vvovMPlbQsaUkTR1NoRpiM~3LqZ2c3Ov6P509S1zfXlMi7e2QCcSbEaBX0jVkZwztir9vUb2apLaVdjrC42Jf2GrOzAfu4ExiCPXZDyeFso2-3LFJNA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="A genetic linkage map of BPH. The LGs were constructed by integrating linkage data from two backcross populations (KH and CK). Grouping of markers was performed using an independence LOD threshold of five. Groups were converted to maps using the regression algorithm provided by JoinMap, with a recombination frequency smaller than 0.45 and Kosambi's mapping function applied for calculation of map distances. The LGs have been arbitrarily numbered LG1–LG17 based on the order generated by JoinMap (v4.1). Distances in centimorgans are indicated at the left of each LG. Marker names starting with NLGS (black) represent g-SSRs, those starting with NLES and BM represent e-SSRs (blue), and those starting with NLSP (red) represent SNP markers. PM3n is an STS marker located on the Y chromosome. A gap between the marker BM500 and PM3n on the LG11 is expected to include the differentiated region of the Y chromosome. e-SSR markers BM13, BM14, BM204, BM464, BM499, and BM500 are for vitellogenin genes that are genes with female-specific expression located on the X chromosome. Segregation-distorted loci indicate different significant levels; *P &lt; 0.01 and **P &lt; 0.001." path-from-xml="dss03002a.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5594969/dss03002a.jpeg" path-from-xml="dss03002a.jpeg" target="_blank">View large</a><a section="5594969" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss030/2/dss03002a.jpeg?Expires=2147483647&amp;Signature=wixgsmMcB85RiQhK01EDLFm9nLr2ab3HhaptWYrXRfqk~c4ZpLJlMtu6oIyk61otiyIynGlPV~ZjBwMK-H7bOx3cCTFoOFt6Iwa40poLizu-EN7YoZZjRZl5pCOoHuFfYKo9mRgEVO5Qf-aVs23Qprad4NRutp3hRDLA1PuhXPEr1fLPzHXOLOAxtb8~qGqlZKSpQDDjFYK0RO2Z~7hogXbI~MID5XqMYWbJfvIBL5uoWNNkomCTLPFkjqcZ2i-emV0dk8NcjvjSgnTPqKzpurx-~qc0jpYJu~onTMivnQFAr64wdm-bXgQ8RDTPxG4VWw5aOhhKtRcKtaQ5Ec7K4A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=5594969&amp;ar=382150&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dss03002a.jpeg" target="_blank">Download slide</a></div></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss030/2/m_dss03002b.jpeg?Expires=2147483647&amp;Signature=mMx87WbwdUrzwM-uKTAHPxEdZWN4XYLxOKdQtePysrKGUUtWIybdwmaUFwGRBRoXvGnzReMq5rAsWf981To4ha30Zr7yP1FOkAbbJIr6Dyg8jOZzFzxzbtZVNgknuHUN~fDjlogoli58fKmQAJUkmSVo5T5gOn8XhUj1CV~E5kQl2lh1U9QeiugOLDFruR7M9coHUEDbQDyx7~tE1q5DVwOyy2HsLheKpGw8D16925qHHeo1Upq1-ffRBX3~V3Pzjd~LJpgkpeqSHVyJ6E5GjwGPE13LrBsJmdYz5XTwYaof95Hmp~o8F25yywmSIUmiPGMk1nHOtYu7mZGszIMI8w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="A genetic linkage map of BPH. The LGs were constructed by integrating linkage data from two backcross populations (KH and CK). Grouping of markers was performed using an independence LOD threshold of five. Groups were converted to maps using the regression algorithm provided by JoinMap, with a recombination frequency smaller than 0.45 and Kosambi's mapping function applied for calculation of map distances. The LGs have been arbitrarily numbered LG1–LG17 based on the order generated by JoinMap (v4.1). Distances in centimorgans are indicated at the left of each LG. Marker names starting with NLGS (black) represent g-SSRs, those starting with NLES and BM represent e-SSRs (blue), and those starting with NLSP (red) represent SNP markers. PM3n is an STS marker located on the Y chromosome. A gap between the marker BM500 and PM3n on the LG11 is expected to include the differentiated region of the Y chromosome. e-SSR markers BM13, BM14, BM204, BM464, BM499, and BM500 are for vitellogenin genes that are genes with female-specific expression located on the X chromosome. Segregation-distorted loci indicate different significant levels; *P &lt; 0.01 and **P &lt; 0.001." path-from-xml="dss03002b.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5594969/dss03002b.jpeg" path-from-xml="dss03002b.jpeg" target="_blank">View large</a><a section="5594969" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss030/2/dss03002b.jpeg?Expires=2147483647&amp;Signature=cN5sXFNH0lQOvvhEUHUEo5VZ6iSlgPcL4f1pCLA-RsaR6jWYPWSUbnKr~0frDMICXQFeZ6bHYAAfmSVFZiN-2aEmt1qDLzfptXSiN8Pr2oqitTNiaRsXCo5K~BgFXlv8VDSiIwuzURaksg54kTvXaEpk-QRnbW2SLL~l2r32CbBMkW~8X0nFjcOQRi7ORaA1F8dNcQbpDOdHRkFti8dqqED0ecEUm6j9pX57SrmwkIBiTXVH4se7p0jsztiIkWyBrewkOnvA74lxp8hMx~yG4A~LFSPH~4oFGEwJzSnGrTrm-pnx8TCJaLX6aetfLoC79nTmUDbVw5GDn8Q3pyCoGw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=5594969&amp;ar=382150&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dss03002b.jpeg" target="_blank">Download slide</a></div></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss030/2/m_dss03002c.jpeg?Expires=2147483647&amp;Signature=qhbWikYHTXeZm5PJDYGtShcZUgmjjxGomcr~l-RHLQbqm7WwPwNNc-azxZ05E6g23Sfzy~EgT71IlFKVjdRNjv8lEw0xaDVUayTXev3AZStOMBNwpYe6T4olGlkmI~sDtEYVJMMczAQIckqX0EFmiupZMuodQGMU9OuBQ7rwTzhro7F8lIHtocbQ8oSs~Vy3NaSRk1G7ZfcodhFGBRv-1N0PW-65xmLhYmh13kTR9ffkjxAuM-ANpXjPiCsevnR-yZnJbwEo5olj4thj30tfJPjP5IgXArlJZKvQA8QFbOcy1BqxZnhZ3PKqjcqq3Cy8Lg~oCqNA5q~impGd7vISjA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="A genetic linkage map of BPH. The LGs were constructed by integrating linkage data from two backcross populations (KH and CK). Grouping of markers was performed using an independence LOD threshold of five. Groups were converted to maps using the regression algorithm provided by JoinMap, with a recombination frequency smaller than 0.45 and Kosambi's mapping function applied for calculation of map distances. The LGs have been arbitrarily numbered LG1–LG17 based on the order generated by JoinMap (v4.1). Distances in centimorgans are indicated at the left of each LG. Marker names starting with NLGS (black) represent g-SSRs, those starting with NLES and BM represent e-SSRs (blue), and those starting with NLSP (red) represent SNP markers. PM3n is an STS marker located on the Y chromosome. A gap between the marker BM500 and PM3n on the LG11 is expected to include the differentiated region of the Y chromosome. e-SSR markers BM13, BM14, BM204, BM464, BM499, and BM500 are for vitellogenin genes that are genes with female-specific expression located on the X chromosome. Segregation-distorted loci indicate different significant levels; *P &lt; 0.01 and **P &lt; 0.001." path-from-xml="dss03002c.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5594969/dss03002c.jpeg" path-from-xml="dss03002c.jpeg" target="_blank">View large</a><a section="5594969" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss030/2/dss03002c.jpeg?Expires=2147483647&amp;Signature=S70-xFyzsoMva3DiShCel3fd5gNgZGVwxObJVdFNpN3TSJkAr5bWUYxJkzDvumIEtxBXtxvbzSkq~uEnuoaqvRyvX7VnW82wsdrAVb3888ivP6JK5gAjt3IwmkBzoyLofa50tQjC4jmVPFHPpBzmQSVW24-nfsm1bOWN3aU~QfAG2~yThvRKHvvSWXWBQP3OOhYaZ6HruKYYiwOAKFS3vFJUg4IojqyspojyiJbPFXmNLE2Ft9tQa~X8JIJCWAKCd2cpNB7RQyDPvdFKriIv0t6SBnIdUrQSZsvPVpqpK17TgrBByJPjPrMpl1XSkTFWdtBH-XVu1o0WVgER3IhKsg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=5594969&amp;ar=382150&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dss03002c.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>A genetic linkage map of BPH. The LGs were constructed by integrating linkage data from two backcross populations (KH and CK). Grouping of markers was performed using an independence LOD threshold of five. Groups were converted to maps using the regression algorithm provided by JoinMap, with a recombination frequency smaller than 0.45 and Kosambi's mapping function applied for calculation of map distances. The LGs have been arbitrarily numbered LG1–LG17 based on the order generated by JoinMap (v4.1). Distances in centimorgans are indicated at the left of each LG. Marker names starting with NLGS (black) represent g-SSRs, those starting with NLES and BM represent e-SSRs (blue), and those starting with NLSP (red) represent SNP markers. PM3n is an STS marker located on the Y chromosome. A gap between the marker BM500 and PM3n on the LG11 is expected to include the differentiated region of the Y chromosome. e-SSR markers BM13, BM14, BM204, BM464, BM499, and BM500 are for vitellogenin genes that are genes with female-specific expression located on the X chromosome. Segregation-distorted loci indicate different significant levels; *<em>P</em> &lt; 0.01 and **<em>P</em> &lt; 0.001.</p></div></div>                            <h3 scrollto-destination=5594970 id="5594970" class="section-title" data-legacy-id=s3d>3.4. Identification of markers linked with the sex chromosomes</h3>
<p>We found that most SSR and SNP markers in LG16 of the CK population deviated significantly from the expected Mendelian segregation ratio of 1:2:1 (homozygous parental: heterozygous: homozygous parental). The segregation of all marker loci in LG16 was exploited to distinguish the X chromosome from the autosomes. If the genotypes of the F<sub>1</sub> maternal and BC<sub>1</sub> paternal parents are designated as X<sub>1</sub>X<sub>2</sub> and X<sub>1</sub>Y<sub>1</sub>, respectively, there would be four equal proportions of BC<sub>1</sub>F<sub>1</sub> genotypes in the offspring in which the females are X<sub>1</sub>X<sub>1</sub> or X<sub>1</sub>X<sub>2</sub> and the males are X<sub>2</sub>Y<sub>1</sub> or X<sub>1</sub>Y<sub>1</sub>. In addition, a ratio of 2:1:1 [homozygous/hemizygous paternal bands (X<sub>1</sub>X<sub>1</sub>/X<sub>1</sub>Y<sub>1</sub>): heterozygous (X<sub>1</sub>X<sub>2</sub>): homozygous maternal band (X<sub>2</sub>Y<sub>1</sub>)] would be expected from the amplified DNA bands. The Y-specific dominant marker PM3n, which was informative in the CK population segregated in a 1:1 ratio, suggested that the CK population had an equal number of males (43) and females (44) in the BC<sub>1</sub>F<sub>1</sub> generation. The observed ratio for all SSR markers in LG16 of CK population matched the expected ratio of 2:1:1, suggesting that these markers are located on the X chromosome (Table <a reveal-id="DSS030TB2" data-open="DSS030TB2" class="link link-reveal link-table xref-fig">2</a>). </p>                        <a id="5594972" scrollto-destination="5594972"></a>
<div content-id="DSS030TB2" class="table-modal"><div class="table-wrap"><div id="DSS030TB2" class="table-wrap-title" data-id="DSS030TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Segregation of marker loci in the X-LGs of the CK and KH populations</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Marker locus </th><th colspan="3">Banding pattern<hr /> </th><th rowspan="2">No. of insects </th><th colspan="2"><em>P</em>-value<hr /> </th></tr><tr><th>X<sub>1</sub>X<sub>1</sub>/X<sub>1</sub>X<sub>2</sub> </th><th>X<sub>2</sub>Y<sub>1</sub> </th><th>X<sub>1</sub>Y<sub>1</sub> </th><th>1:2:1<sup>a</sup> </th><th>2:1:1<sup>b</sup> </th></tr></thead><tbody><tr><td colspan="7">CK population (codominant marker) </td></tr><tr><td> NLGS225 </td><td>39 </td><td>26 </td><td>21 </td><td>86 </td><td>&lt;0.001** </td><td>0.52<sup>c</sup> </td></tr><tr><td> BM13 </td><td>40 </td><td>28 </td><td>20 </td><td>88 </td><td>&lt;0.001** </td><td>0.34<sup>c</sup> </td></tr><tr><td> BM25 </td><td>38 </td><td>28 </td><td>22 </td><td>88 </td><td>&lt;0.001** </td><td>0.29<sup>c</sup> </td></tr><tr><td> NLGS3014 </td><td>40 </td><td>24 </td><td>23 </td><td>87 </td><td>&lt;0.001** </td><td>0.75<sup>c</sup> </td></tr><tr><td> NLGS953 </td><td>35 </td><td>29 </td><td>25 </td><td>89 </td><td>&lt;0.001** </td><td>0.11<sup>c</sup> </td></tr><tr><td> NLGS825 </td><td>38 </td><td>28 </td><td>23 </td><td>89 </td><td>&lt;0.001** </td><td>0.29<sup>c</sup> </td></tr><tr><td> BM201 </td><td>37 </td><td>27 </td><td>25 </td><td>89 </td><td>&lt;0.001** </td><td>0.27<sup>c</sup> </td></tr><tr><td colspan="7">KH population (codominant marker) </td></tr><tr><td> NLSP0007 </td><td>45 </td><td>18 </td><td>24 </td><td>87 </td><td>&lt;0.001** </td><td>0.63<sup>c</sup> </td></tr><tr><td> NLSP0011 </td><td>45 </td><td>21 </td><td>24 </td><td>90 </td><td>&lt;0.001** </td><td>0.90<sup>c</sup> </td></tr><tr><td> NLSP0018 </td><td>50 </td><td>14 </td><td>24 </td><td>88 </td><td>&lt;0.001** </td><td>0.14<sup>c</sup> </td></tr><tr><td> NLSP0022 </td><td>48 </td><td>19 </td><td>22 </td><td>89 </td><td>&lt;0.001** </td><td>0.69<sup>c</sup> </td></tr><tr><td> NLSP0024 </td><td>45 </td><td>20 </td><td>23 </td><td>88 </td><td>&lt;0.001** </td><td>0.88<sup>c</sup> </td></tr><tr><td> NLSP0031 </td><td>49 </td><td>18 </td><td>24 </td><td>91 </td><td>&lt;0.001** </td><td>0.51<sup>c</sup> </td></tr><tr><td> NLSP0033 </td><td>43 </td><td>19 </td><td>23 </td><td>85 </td><td>&lt;0.001** </td><td>0.82<sup>c</sup> </td></tr><tr><td> BM464 </td><td>50 </td><td>20 </td><td>21 </td><td>91 </td><td>&lt;0.001** </td><td>0.63<sup>c</sup> </td></tr><tr><td> BM204 </td><td>49 </td><td>20 </td><td>22 </td><td>91 </td><td>&lt;0.001** </td><td>0.73<sup>c</sup> </td></tr><tr><td> BM499 </td><td>47 </td><td>18 </td><td>26 </td><td>91 </td><td>&lt;0.001** </td><td>0.47<sup>c</sup> </td></tr><tr><td> BM500 </td><td>55 </td><td>17 </td><td>18 </td><td>90 </td><td>&lt;0.001** </td><td>0.11<sup>c</sup> </td></tr><tr><td> BM13 </td><td>45 </td><td>20 </td><td>24 </td><td>89 </td><td>&lt;0.001** </td><td>0.83<sup>c</sup> </td></tr><tr><td colspan="4">KH population (dominant marker) </td><td> </td><td>3:1<sup>a</sup> </td><td>3:1<sup>b</sup> </td></tr><tr><td> NLGS74 </td><td colspan="2">68 </td><td>23 </td><td>91 </td><td>1.00<sup>c</sup> </td><td>1.00<sup>c</sup> </td></tr><tr><td> NLGS225 </td><td colspan="2">66 </td><td>25 </td><td>91 </td><td>0.96<sup>c</sup> </td><td>0.96<sup>c</sup> </td></tr><tr><td> NLGS307 </td><td colspan="2">64 </td><td>27 </td><td>91 </td><td>0.97<sup>c</sup> </td><td>0.97<sup>c</sup> </td></tr><tr><td> NLGS399 </td><td colspan="2">64 </td><td>27 </td><td>91 </td><td>0.94<sup>c</sup> </td><td>0.94<sup>c</sup> </td></tr><tr><td> NLGS809 </td><td colspan="2">66 </td><td>22 </td><td>88 </td><td>0.98<sup>c</sup> </td><td>0.98<sup>c</sup> </td></tr><tr><td> NLGS1060 </td><td colspan="2">67 </td><td>23 </td><td>90 </td><td>1.00<sup>c</sup> </td><td>1.00<sup>c</sup> </td></tr><tr><td> NLGS3014 </td><td colspan="2">62 </td><td>27 </td><td>89 </td><td>0.80<sup>c</sup> </td><td>0.80<sup>c</sup> </td></tr><tr><td> NLGS59 </td><td colspan="2">60 </td><td>30 </td><td>90 </td><td>0.90<sup>c</sup> </td><td>0.90<sup>c</sup> </td></tr><tr><td> NLGS808 </td><td colspan="2">61 </td><td>29 </td><td>90 </td><td>0.73<sup>c</sup> </td><td>0.73<sup>c</sup> </td></tr><tr><td> BM14 </td><td colspan="2">50 </td><td>35 </td><td>85 </td><td>0.23<sup>c</sup> </td><td>0.23<sup>c</sup> </td></tr></tbody></table></div><div content-id="DSS030TB2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Marker locus </th><th colspan="3">Banding pattern<hr /> </th><th rowspan="2">No. of insects </th><th colspan="2"><em>P</em>-value<hr /> </th></tr><tr><th>X<sub>1</sub>X<sub>1</sub>/X<sub>1</sub>X<sub>2</sub> </th><th>X<sub>2</sub>Y<sub>1</sub> </th><th>X<sub>1</sub>Y<sub>1</sub> </th><th>1:2:1<sup>a</sup> </th><th>2:1:1<sup>b</sup> </th></tr></thead><tbody><tr><td colspan="7">CK population (codominant marker) </td></tr><tr><td> NLGS225 </td><td>39 </td><td>26 </td><td>21 </td><td>86 </td><td>&lt;0.001** </td><td>0.52<sup>c</sup> </td></tr><tr><td> BM13 </td><td>40 </td><td>28 </td><td>20 </td><td>88 </td><td>&lt;0.001** </td><td>0.34<sup>c</sup> </td></tr><tr><td> BM25 </td><td>38 </td><td>28 </td><td>22 </td><td>88 </td><td>&lt;0.001** </td><td>0.29<sup>c</sup> </td></tr><tr><td> NLGS3014 </td><td>40 </td><td>24 </td><td>23 </td><td>87 </td><td>&lt;0.001** </td><td>0.75<sup>c</sup> </td></tr><tr><td> NLGS953 </td><td>35 </td><td>29 </td><td>25 </td><td>89 </td><td>&lt;0.001** </td><td>0.11<sup>c</sup> </td></tr><tr><td> NLGS825 </td><td>38 </td><td>28 </td><td>23 </td><td>89 </td><td>&lt;0.001** </td><td>0.29<sup>c</sup> </td></tr><tr><td> BM201 </td><td>37 </td><td>27 </td><td>25 </td><td>89 </td><td>&lt;0.001** </td><td>0.27<sup>c</sup> </td></tr><tr><td colspan="7">KH population (codominant marker) </td></tr><tr><td> NLSP0007 </td><td>45 </td><td>18 </td><td>24 </td><td>87 </td><td>&lt;0.001** </td><td>0.63<sup>c</sup> </td></tr><tr><td> NLSP0011 </td><td>45 </td><td>21 </td><td>24 </td><td>90 </td><td>&lt;0.001** </td><td>0.90<sup>c</sup> </td></tr><tr><td> NLSP0018 </td><td>50 </td><td>14 </td><td>24 </td><td>88 </td><td>&lt;0.001** </td><td>0.14<sup>c</sup> </td></tr><tr><td> NLSP0022 </td><td>48 </td><td>19 </td><td>22 </td><td>89 </td><td>&lt;0.001** </td><td>0.69<sup>c</sup> </td></tr><tr><td> NLSP0024 </td><td>45 </td><td>20 </td><td>23 </td><td>88 </td><td>&lt;0.001** </td><td>0.88<sup>c</sup> </td></tr><tr><td> NLSP0031 </td><td>49 </td><td>18 </td><td>24 </td><td>91 </td><td>&lt;0.001** </td><td>0.51<sup>c</sup> </td></tr><tr><td> NLSP0033 </td><td>43 </td><td>19 </td><td>23 </td><td>85 </td><td>&lt;0.001** </td><td>0.82<sup>c</sup> </td></tr><tr><td> BM464 </td><td>50 </td><td>20 </td><td>21 </td><td>91 </td><td>&lt;0.001** </td><td>0.63<sup>c</sup> </td></tr><tr><td> BM204 </td><td>49 </td><td>20 </td><td>22 </td><td>91 </td><td>&lt;0.001** </td><td>0.73<sup>c</sup> </td></tr><tr><td> BM499 </td><td>47 </td><td>18 </td><td>26 </td><td>91 </td><td>&lt;0.001** </td><td>0.47<sup>c</sup> </td></tr><tr><td> BM500 </td><td>55 </td><td>17 </td><td>18 </td><td>90 </td><td>&lt;0.001** </td><td>0.11<sup>c</sup> </td></tr><tr><td> BM13 </td><td>45 </td><td>20 </td><td>24 </td><td>89 </td><td>&lt;0.001** </td><td>0.83<sup>c</sup> </td></tr><tr><td colspan="4">KH population (dominant marker) </td><td> </td><td>3:1<sup>a</sup> </td><td>3:1<sup>b</sup> </td></tr><tr><td> NLGS74 </td><td colspan="2">68 </td><td>23 </td><td>91 </td><td>1.00<sup>c</sup> </td><td>1.00<sup>c</sup> </td></tr><tr><td> NLGS225 </td><td colspan="2">66 </td><td>25 </td><td>91 </td><td>0.96<sup>c</sup> </td><td>0.96<sup>c</sup> </td></tr><tr><td> NLGS307 </td><td colspan="2">64 </td><td>27 </td><td>91 </td><td>0.97<sup>c</sup> </td><td>0.97<sup>c</sup> </td></tr><tr><td> NLGS399 </td><td colspan="2">64 </td><td>27 </td><td>91 </td><td>0.94<sup>c</sup> </td><td>0.94<sup>c</sup> </td></tr><tr><td> NLGS809 </td><td colspan="2">66 </td><td>22 </td><td>88 </td><td>0.98<sup>c</sup> </td><td>0.98<sup>c</sup> </td></tr><tr><td> NLGS1060 </td><td colspan="2">67 </td><td>23 </td><td>90 </td><td>1.00<sup>c</sup> </td><td>1.00<sup>c</sup> </td></tr><tr><td> NLGS3014 </td><td colspan="2">62 </td><td>27 </td><td>89 </td><td>0.80<sup>c</sup> </td><td>0.80<sup>c</sup> </td></tr><tr><td> NLGS59 </td><td colspan="2">60 </td><td>30 </td><td>90 </td><td>0.90<sup>c</sup> </td><td>0.90<sup>c</sup> </td></tr><tr><td> NLGS808 </td><td colspan="2">61 </td><td>29 </td><td>90 </td><td>0.73<sup>c</sup> </td><td>0.73<sup>c</sup> </td></tr><tr><td> BM14 </td><td colspan="2">50 </td><td>35 </td><td>85 </td><td>0.23<sup>c</sup> </td><td>0.23<sup>c</sup> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Expected ratio for autosomes.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>Expected ratio for the X chromosome.</p></div></div><div content-id=""><div class="fn"><p><sup>c</sup>Non-significant deviation from the expected ratio.</p></div></div><div content-id=""><div class="fn"><p>**Significant at the 0.1% level.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5594972" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSS030TB2" class="table-wrap-title" data-id="DSS030TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Segregation of marker loci in the X-LGs of the CK and KH populations</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Marker locus </th><th colspan="3">Banding pattern<hr /> </th><th rowspan="2">No. of insects </th><th colspan="2"><em>P</em>-value<hr /> </th></tr><tr><th>X<sub>1</sub>X<sub>1</sub>/X<sub>1</sub>X<sub>2</sub> </th><th>X<sub>2</sub>Y<sub>1</sub> </th><th>X<sub>1</sub>Y<sub>1</sub> </th><th>1:2:1<sup>a</sup> </th><th>2:1:1<sup>b</sup> </th></tr></thead><tbody><tr><td colspan="7">CK population (codominant marker) </td></tr><tr><td> NLGS225 </td><td>39 </td><td>26 </td><td>21 </td><td>86 </td><td>&lt;0.001** </td><td>0.52<sup>c</sup> </td></tr><tr><td> BM13 </td><td>40 </td><td>28 </td><td>20 </td><td>88 </td><td>&lt;0.001** </td><td>0.34<sup>c</sup> </td></tr><tr><td> BM25 </td><td>38 </td><td>28 </td><td>22 </td><td>88 </td><td>&lt;0.001** </td><td>0.29<sup>c</sup> </td></tr><tr><td> NLGS3014 </td><td>40 </td><td>24 </td><td>23 </td><td>87 </td><td>&lt;0.001** </td><td>0.75<sup>c</sup> </td></tr><tr><td> NLGS953 </td><td>35 </td><td>29 </td><td>25 </td><td>89 </td><td>&lt;0.001** </td><td>0.11<sup>c</sup> </td></tr><tr><td> NLGS825 </td><td>38 </td><td>28 </td><td>23 </td><td>89 </td><td>&lt;0.001** </td><td>0.29<sup>c</sup> </td></tr><tr><td> BM201 </td><td>37 </td><td>27 </td><td>25 </td><td>89 </td><td>&lt;0.001** </td><td>0.27<sup>c</sup> </td></tr><tr><td colspan="7">KH population (codominant marker) </td></tr><tr><td> NLSP0007 </td><td>45 </td><td>18 </td><td>24 </td><td>87 </td><td>&lt;0.001** </td><td>0.63<sup>c</sup> </td></tr><tr><td> NLSP0011 </td><td>45 </td><td>21 </td><td>24 </td><td>90 </td><td>&lt;0.001** </td><td>0.90<sup>c</sup> </td></tr><tr><td> NLSP0018 </td><td>50 </td><td>14 </td><td>24 </td><td>88 </td><td>&lt;0.001** </td><td>0.14<sup>c</sup> </td></tr><tr><td> NLSP0022 </td><td>48 </td><td>19 </td><td>22 </td><td>89 </td><td>&lt;0.001** </td><td>0.69<sup>c</sup> </td></tr><tr><td> NLSP0024 </td><td>45 </td><td>20 </td><td>23 </td><td>88 </td><td>&lt;0.001** </td><td>0.88<sup>c</sup> </td></tr><tr><td> NLSP0031 </td><td>49 </td><td>18 </td><td>24 </td><td>91 </td><td>&lt;0.001** </td><td>0.51<sup>c</sup> </td></tr><tr><td> NLSP0033 </td><td>43 </td><td>19 </td><td>23 </td><td>85 </td><td>&lt;0.001** </td><td>0.82<sup>c</sup> </td></tr><tr><td> BM464 </td><td>50 </td><td>20 </td><td>21 </td><td>91 </td><td>&lt;0.001** </td><td>0.63<sup>c</sup> </td></tr><tr><td> BM204 </td><td>49 </td><td>20 </td><td>22 </td><td>91 </td><td>&lt;0.001** </td><td>0.73<sup>c</sup> </td></tr><tr><td> BM499 </td><td>47 </td><td>18 </td><td>26 </td><td>91 </td><td>&lt;0.001** </td><td>0.47<sup>c</sup> </td></tr><tr><td> BM500 </td><td>55 </td><td>17 </td><td>18 </td><td>90 </td><td>&lt;0.001** </td><td>0.11<sup>c</sup> </td></tr><tr><td> BM13 </td><td>45 </td><td>20 </td><td>24 </td><td>89 </td><td>&lt;0.001** </td><td>0.83<sup>c</sup> </td></tr><tr><td colspan="4">KH population (dominant marker) </td><td> </td><td>3:1<sup>a</sup> </td><td>3:1<sup>b</sup> </td></tr><tr><td> NLGS74 </td><td colspan="2">68 </td><td>23 </td><td>91 </td><td>1.00<sup>c</sup> </td><td>1.00<sup>c</sup> </td></tr><tr><td> NLGS225 </td><td colspan="2">66 </td><td>25 </td><td>91 </td><td>0.96<sup>c</sup> </td><td>0.96<sup>c</sup> </td></tr><tr><td> NLGS307 </td><td colspan="2">64 </td><td>27 </td><td>91 </td><td>0.97<sup>c</sup> </td><td>0.97<sup>c</sup> </td></tr><tr><td> NLGS399 </td><td colspan="2">64 </td><td>27 </td><td>91 </td><td>0.94<sup>c</sup> </td><td>0.94<sup>c</sup> </td></tr><tr><td> NLGS809 </td><td colspan="2">66 </td><td>22 </td><td>88 </td><td>0.98<sup>c</sup> </td><td>0.98<sup>c</sup> </td></tr><tr><td> NLGS1060 </td><td colspan="2">67 </td><td>23 </td><td>90 </td><td>1.00<sup>c</sup> </td><td>1.00<sup>c</sup> </td></tr><tr><td> NLGS3014 </td><td colspan="2">62 </td><td>27 </td><td>89 </td><td>0.80<sup>c</sup> </td><td>0.80<sup>c</sup> </td></tr><tr><td> NLGS59 </td><td colspan="2">60 </td><td>30 </td><td>90 </td><td>0.90<sup>c</sup> </td><td>0.90<sup>c</sup> </td></tr><tr><td> NLGS808 </td><td colspan="2">61 </td><td>29 </td><td>90 </td><td>0.73<sup>c</sup> </td><td>0.73<sup>c</sup> </td></tr><tr><td> BM14 </td><td colspan="2">50 </td><td>35 </td><td>85 </td><td>0.23<sup>c</sup> </td><td>0.23<sup>c</sup> </td></tr></tbody></table></div><div content-id="DSS030TB2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Marker locus </th><th colspan="3">Banding pattern<hr /> </th><th rowspan="2">No. of insects </th><th colspan="2"><em>P</em>-value<hr /> </th></tr><tr><th>X<sub>1</sub>X<sub>1</sub>/X<sub>1</sub>X<sub>2</sub> </th><th>X<sub>2</sub>Y<sub>1</sub> </th><th>X<sub>1</sub>Y<sub>1</sub> </th><th>1:2:1<sup>a</sup> </th><th>2:1:1<sup>b</sup> </th></tr></thead><tbody><tr><td colspan="7">CK population (codominant marker) </td></tr><tr><td> NLGS225 </td><td>39 </td><td>26 </td><td>21 </td><td>86 </td><td>&lt;0.001** </td><td>0.52<sup>c</sup> </td></tr><tr><td> BM13 </td><td>40 </td><td>28 </td><td>20 </td><td>88 </td><td>&lt;0.001** </td><td>0.34<sup>c</sup> </td></tr><tr><td> BM25 </td><td>38 </td><td>28 </td><td>22 </td><td>88 </td><td>&lt;0.001** </td><td>0.29<sup>c</sup> </td></tr><tr><td> NLGS3014 </td><td>40 </td><td>24 </td><td>23 </td><td>87 </td><td>&lt;0.001** </td><td>0.75<sup>c</sup> </td></tr><tr><td> NLGS953 </td><td>35 </td><td>29 </td><td>25 </td><td>89 </td><td>&lt;0.001** </td><td>0.11<sup>c</sup> </td></tr><tr><td> NLGS825 </td><td>38 </td><td>28 </td><td>23 </td><td>89 </td><td>&lt;0.001** </td><td>0.29<sup>c</sup> </td></tr><tr><td> BM201 </td><td>37 </td><td>27 </td><td>25 </td><td>89 </td><td>&lt;0.001** </td><td>0.27<sup>c</sup> </td></tr><tr><td colspan="7">KH population (codominant marker) </td></tr><tr><td> NLSP0007 </td><td>45 </td><td>18 </td><td>24 </td><td>87 </td><td>&lt;0.001** </td><td>0.63<sup>c</sup> </td></tr><tr><td> NLSP0011 </td><td>45 </td><td>21 </td><td>24 </td><td>90 </td><td>&lt;0.001** </td><td>0.90<sup>c</sup> </td></tr><tr><td> NLSP0018 </td><td>50 </td><td>14 </td><td>24 </td><td>88 </td><td>&lt;0.001** </td><td>0.14<sup>c</sup> </td></tr><tr><td> NLSP0022 </td><td>48 </td><td>19 </td><td>22 </td><td>89 </td><td>&lt;0.001** </td><td>0.69<sup>c</sup> </td></tr><tr><td> NLSP0024 </td><td>45 </td><td>20 </td><td>23 </td><td>88 </td><td>&lt;0.001** </td><td>0.88<sup>c</sup> </td></tr><tr><td> NLSP0031 </td><td>49 </td><td>18 </td><td>24 </td><td>91 </td><td>&lt;0.001** </td><td>0.51<sup>c</sup> </td></tr><tr><td> NLSP0033 </td><td>43 </td><td>19 </td><td>23 </td><td>85 </td><td>&lt;0.001** </td><td>0.82<sup>c</sup> </td></tr><tr><td> BM464 </td><td>50 </td><td>20 </td><td>21 </td><td>91 </td><td>&lt;0.001** </td><td>0.63<sup>c</sup> </td></tr><tr><td> BM204 </td><td>49 </td><td>20 </td><td>22 </td><td>91 </td><td>&lt;0.001** </td><td>0.73<sup>c</sup> </td></tr><tr><td> BM499 </td><td>47 </td><td>18 </td><td>26 </td><td>91 </td><td>&lt;0.001** </td><td>0.47<sup>c</sup> </td></tr><tr><td> BM500 </td><td>55 </td><td>17 </td><td>18 </td><td>90 </td><td>&lt;0.001** </td><td>0.11<sup>c</sup> </td></tr><tr><td> BM13 </td><td>45 </td><td>20 </td><td>24 </td><td>89 </td><td>&lt;0.001** </td><td>0.83<sup>c</sup> </td></tr><tr><td colspan="4">KH population (dominant marker) </td><td> </td><td>3:1<sup>a</sup> </td><td>3:1<sup>b</sup> </td></tr><tr><td> NLGS74 </td><td colspan="2">68 </td><td>23 </td><td>91 </td><td>1.00<sup>c</sup> </td><td>1.00<sup>c</sup> </td></tr><tr><td> NLGS225 </td><td colspan="2">66 </td><td>25 </td><td>91 </td><td>0.96<sup>c</sup> </td><td>0.96<sup>c</sup> </td></tr><tr><td> NLGS307 </td><td colspan="2">64 </td><td>27 </td><td>91 </td><td>0.97<sup>c</sup> </td><td>0.97<sup>c</sup> </td></tr><tr><td> NLGS399 </td><td colspan="2">64 </td><td>27 </td><td>91 </td><td>0.94<sup>c</sup> </td><td>0.94<sup>c</sup> </td></tr><tr><td> NLGS809 </td><td colspan="2">66 </td><td>22 </td><td>88 </td><td>0.98<sup>c</sup> </td><td>0.98<sup>c</sup> </td></tr><tr><td> NLGS1060 </td><td colspan="2">67 </td><td>23 </td><td>90 </td><td>1.00<sup>c</sup> </td><td>1.00<sup>c</sup> </td></tr><tr><td> NLGS3014 </td><td colspan="2">62 </td><td>27 </td><td>89 </td><td>0.80<sup>c</sup> </td><td>0.80<sup>c</sup> </td></tr><tr><td> NLGS59 </td><td colspan="2">60 </td><td>30 </td><td>90 </td><td>0.90<sup>c</sup> </td><td>0.90<sup>c</sup> </td></tr><tr><td> NLGS808 </td><td colspan="2">61 </td><td>29 </td><td>90 </td><td>0.73<sup>c</sup> </td><td>0.73<sup>c</sup> </td></tr><tr><td> BM14 </td><td colspan="2">50 </td><td>35 </td><td>85 </td><td>0.23<sup>c</sup> </td><td>0.23<sup>c</sup> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Expected ratio for autosomes.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>Expected ratio for the X chromosome.</p></div></div><div content-id=""><div class="fn"><p><sup>c</sup>Non-significant deviation from the expected ratio.</p></div></div><div content-id=""><div class="fn"><p>**Significant at the 0.1% level.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5594972" target="_blank">View Large</a></div></div><p>Based on LG11 in the integrated map, the bridged markers in LG16 of the CK population were located in LG11 of the KH population. Of the 22 markers in LG11 of the KH population, 12 loci were codominant markers. Because the PM3n marker was not informative for the KH population, the sexes of BC<sub>1</sub>F<sub>1</sub> were not determined. If we assume that the KH population had an equal number of males and females in the BC<sub>1</sub>F<sub>1</sub> generation, the ratio of 2:1:1 would be expected for codominant markers. The codominant markers, including seven SNPs and five SSRs, segregated in a 2:1:1 ratio of homozygous/hemizygous paternal bands: heterozygous: homozygous maternal band in the KH offspring. Using 4% agarose gel to visualize the amplified fragments, we could not distinguish between heterozygous and homozygous genotypes for the paternal genotype of the 10 SSR markers in LG11. These markers were dominant markers, and segregated in a 3:1 ratio for all paternal band types: homozygous maternal band types. These loci also segregated in the ratio expected for a sex-linked marker (Table <a reveal-id="DSS030TB2" data-open="DSS030TB2" class="link link-reveal link-table xref-fig">2</a>).</p>                            <h2 scrollto-destination=5594974 id="5594974" class="section-title" data-legacy-id=s4>4. Discussion</h2>
                            <h3 scrollto-destination=5594975 id="5594975" class="section-title" data-legacy-id=s4a>4.1. Development and analysis of SSR markers</h3>
<p>Early research focused on the development of SSR markers from BPH genomic DNA.<sup><a reveal-id="DSS030C38" data-open="DSS030C38" class="link link-ref link-reveal xref-bibr">38</a></sup> Because of a lack of appropriate strategies at that time, few markers were developed. Later, after an enormous number of BPH ESTs became available,<sup><a reveal-id="DSS030C12" data-open="DSS030C12" class="link link-ref link-reveal xref-bibr">12</a></sup> more than 350 e-SSR markers were developed to support genetic studies.<sup><a reveal-id="DSS030C17  DSS030C19" data-open="DSS030C17  DSS030C19" class="link link-ref link-reveal xref-bibr">17–19</a></sup> Recently, 30 new g-SSR markers were developed via the FIASCO method.<sup><a reveal-id="DSS030C20" data-open="DSS030C20" class="link link-ref link-reveal xref-bibr">20</a></sup> In the present study, we combined enrichment-based SSR isolation with next-generation sequencing approaches and demonstrated that this approach was a powerful tool for developing SSR markers from genomic DNA. Through these strategies, we developed 3160 g-SSR markers, of which 922 (29.2%) were successfully amplified and showed polymorphism among the BPH strains used as parents for the mapping. As the polymorphisms were detected on a simple agarose gel, only 418 SSR markers (45.3%) were informative. The remaining 54.7% were not informative, mostly due to banding patterns that were too difficult to interpret or bands that were too close together. The number of informative markers could be increased using more powerful fragment separation systems such as fluorescently labelled-fragment capillary arrays.</p><p>The newly developed e-SSR primers in this study were less polymorphic (30.0%) when compared with the g-SSRs (45.0%). Likewise, the polymorphisms of the publicly available e-SSRs also were &lt;32%. It might be because of greater DNA sequence conservation in transcribed regions. However, e-SSR markers produced a high proportion of high-quality markers with stronger and clearer amplified bands.</p>                            <h3 scrollto-destination=5594978 id="5594978" class="section-title" data-legacy-id=s4b>4.2. Development and analysis of SNP markers</h3>
<p>We found a large number of candidate SNPs (&gt;100 000) between the two survey strains in the next-generation sequencing data. However, the frequency of valid SNPs for linkage analysis was low because of several difficulties in developing informative markers using the strategies described in this paper. The major reason for these difficulties was the lack of a reliable reference genome sequence for BPH. The draft assembly of the BPH genome contains a considerable number of intraspecies sequence polymorphisms, leading to insufficiently precise calling of SNPs. Although generating pure IBSs is an effective strategy for reducing heterogeneity, repetitive inbreeding is not easy because it decreases the fecundity and fitness of the BPH populations (T. Kobayashi, unpublished data). Differences between the strains used for the SNP survey (I87 vs. IBS-C89) and the linkage analysis (K10 vs. H66) also reduced the number of valid SNPs. To increase SNP markers in this species, high-quality reference genome sequences and a better strategy to evaluate potential SNPs will be necessary.<sup><a reveal-id="DSS030C39" data-open="DSS030C39" class="link link-ref link-reveal xref-bibr">39</a></sup></p>                            <h3 scrollto-destination=5594980 id="5594980" class="section-title" data-legacy-id=s4c>4.3. Genetic linkage map</h3>
<p>Based on the results of our study, we have constructed an SSR- and SNP-based linkage map for BPH. The genetic map consists of 518 markers that cover 1093.9 cM, with an average marker spacing of 2.3 cM. The majority of the mapped e-SSR and SNP markers were widely distributed among several LGs. Disagreement in the number of LGs between the two populations (KH and CK) might have resulted from the large portion of markers in some LGs (LG8 and LG11) and the existence of one small LG (LG17), possibly due to the relatively small number of BC<sub>1</sub>F<sub>1</sub> individuals (&lt;100) used for mapping and the lack of informative markers among LGs belonging to the same chromosome.<sup><a reveal-id="DSS030C40" data-open="DSS030C40" class="link link-ref link-reveal xref-bibr">40</a>,<a reveal-id="DSS030C41" data-open="DSS030C41" class="link link-ref link-reveal xref-bibr">41</a></sup> The quality and accuracy of marker order in such maps depend on numerous factors, including segregation distortion, the population size, and scoring errors. In some cases, the quality can be improved and the problem can be solved.<sup><a reveal-id="DSS030C42" data-open="DSS030C42" class="link link-ref link-reveal xref-bibr">42</a></sup> Numerous examples of a high level of deviations from Mendelian segregation ratios have been reported in previous studies (<em>Bombyx mori</em>,<sup><a reveal-id="DSS030C43" data-open="DSS030C43" class="link link-ref link-reveal xref-bibr">43</a></sup><em>Chlamys farreri</em>,<sup><a reveal-id="DSS030C44" data-open="DSS030C44" class="link link-ref link-reveal xref-bibr">44</a></sup> and <em>Crassostrea gigas</em><sup><a reveal-id="DSS030C45" data-open="DSS030C45" class="link link-ref link-reveal xref-bibr">45</a></sup>). In our study, segregation distortion (<em>P</em> &lt; 0.01) was observed for 8.7% of the total marker loci that we analysed. This may have several possible explanations, including inbreeding depression,<sup><a reveal-id="DSS030C46" data-open="DSS030C46" class="link link-ref link-reveal xref-bibr">46</a></sup> sex-biased ratio distortion,<sup><a reveal-id="DSS030C47" data-open="DSS030C47" class="link link-ref link-reveal xref-bibr">47</a></sup> erroneous scoring, or linkage between molecular markers and distorting factors such as recessive lethal genes or incompatible alleles.<sup><a reveal-id="DSS030C48" data-open="DSS030C48" class="link link-ref link-reveal xref-bibr">48</a></sup></p><p>Population size is one of the factors associated with the accuracy of detecting recombination events, and it can affect the efficiency of the mapping process.<sup><a reveal-id="DSS030C49" data-open="DSS030C49" class="link link-ref link-reveal xref-bibr">49</a></sup> Because we used small population sizes, inaccurate marker order may have occurred due to inversions and marker distance. However, numerous linkage maps have been successfully produced from small mapping populations (50–94 individuals) in several species.<sup><a reveal-id="DSS030C40" data-open="DSS030C40" class="link link-ref link-reveal xref-bibr">40</a>,<a reveal-id="DSS030C50  DSS030C52" data-open="DSS030C50  DSS030C52" class="link link-ref link-reveal xref-bibr">50–52</a></sup> To obtain accurate estimates of marker positions in the map, we only selected clearly polymorphic markers and carefully scored them to reduce errors and ensure the accuracy of the linkage map. Nonetheless, inversions of some markers were observed in several LGs of the consensus map (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). A more accurate marker order can be acquired by means of map validation, when a draft genome sequence for BPH becomes available.</p><p>Recombination frequencies in mapping populations of some vertebrate and arthropod species show distinct differences between males and females.<sup><a reveal-id="DSS030C53  DSS030C57" data-open="DSS030C53  DSS030C57" class="link link-ref link-reveal xref-bibr">53–57</a></sup> Although the mechanism responsible for these differences in recombination rates between the sexes is not well understood, the phenomenon is common. The conventional way to construct a linkage map is usually to analyse the two sexes separately. Because we collected the offspring of the mapping populations during their early growth stages, the sex of the offspring could not be identified. In addition, the number of females and males in the populations was too small to enable mapping separately within each sex. Consequently, we included both sexes in our linkage analysis to produce a single map. Developing separate linkage maps for the two sexes in future mapping experiments will provide a better understanding of differences in the recombination rates between the sexes of BPH.</p><p>Mapping with multiple populations is advantageous because a larger number of markers can be placed on a single map, thereby providing greater genomic coverage. Integrated linkage maps have been constructed in various plant and animal species, including insects.<sup><a reveal-id="DSS030C52" data-open="DSS030C52" class="link link-ref link-reveal xref-bibr">52</a>,<a reveal-id="DSS030C58  DSS030C60" data-open="DSS030C58  DSS030C60" class="link link-ref link-reveal xref-bibr">58–60</a></sup> We combined two genetic linkage maps from different genetic backgrounds derived from three IBSs that have different virulence against rice differential varieties to BPH,<sup><a reveal-id="DSS030C61" data-open="DSS030C61" class="link link-ref link-reveal xref-bibr">61</a></sup> into one integrated map. The numbers of markers and the genomic coverage were increased by this approach, and the gaps between markers were reduced in the integrated map. However, the marker density of the final linkage map was still low when compared with the denser linkage maps of other model insect species (e.g. <em>B. mori</em><sup><a reveal-id="DSS030C57" data-open="DSS030C57" class="link link-ref link-reveal xref-bibr">57</a>,<a reveal-id="DSS030C62" data-open="DSS030C62" class="link link-ref link-reveal xref-bibr">62</a></sup> and <em>Apis mellifera</em><sup><a reveal-id="DSS030C59" data-open="DSS030C59" class="link link-ref link-reveal xref-bibr">59</a></sup>). The construction of a highly saturated linkage map with full genome coverage will be possible in future research with more SNP markers. Nevertheless, the new SSR markers and the consensus map developed in the present study provide useful genetic and genomic tools for molecular analysis of the BPH genome and will facilitate the construction of a higher density second-generation linkage map.</p>                            <h3 scrollto-destination=5594985 id="5594985" class="section-title" data-legacy-id=s4d>4.4. Determination of the sex-LG</h3>
<p>As we could not identify the sex of individual nymphs in our mapping populations, we used the Y-specific marker PM3n<sup><a reveal-id="DSS030C63" data-open="DSS030C63" class="link link-ref link-reveal xref-bibr">63</a></sup> as a sex-determining locus (<em>Sex</em>). In the consensus map, the putative location of PM3n was mapped at the distal end of LG11. Additional evidence to confirm the sex-LG would require X-linked loci. Because the e-SSRs were developed from the transcribed regions of the genome, mapping e-SSR markers directly shows the location of the corresponding genes within the linkage map. Vitellogenin genes exhibit female-specific expression and are generally located on the X chromosome in various organisms, including insects.<sup><a reveal-id="DSS030C64  DSS030C68" data-open="DSS030C64  DSS030C68" class="link link-ref link-reveal xref-bibr">64–68</a></sup> An interesting feature of the present map is that six e-SSRs derived from vitellogenin genes<sup><a reveal-id="DSS030C17" data-open="DSS030C17" class="link link-ref link-reveal xref-bibr">17</a></sup> were gathered in only LG11 of the consensus map. We tested the segregation of all marker loci on LG11 and found that all the markers fitted the segregation ratio expected for genes located on the X chromosome. The results indicated that LG11 corresponds to the X-LG in BPH. The X-linked markers would be useful in future genetic analysis of the sex chromosome.</p>                            <h3 scrollto-destination=5594987 id="5594987" class="section-title" data-legacy-id=s4e>4.5. Conclusions and future perspectives</h3>
<p>In this paper, we report the development of g-SSR, e-SSR, and SNP markers and present a preliminary genetic map for BPH based on these markers. Two individual linkage maps from the KH and CK populations were integrated into a consensus map to increase locus density and provide more dense coverage of the genome. The consensus map consisted of 518 markers that covered a total of 1093.9 cM. Denser coverage is needed to reduce the gaps and group the smallest LGs so that the consensus map matches the actual number (15) of chromosomes. Additional SNP identification and mapping should be performed in future research. It should be possible to extend the genome coverage in this manner. The mean marker distance of 2.3 cM in the consensus map may offer enough resolution for the genetic dissection of QTLs for agriculturally important traits, and particularly the virulence of BPH against the host plant. An understanding of the genetics responsible for virulence is a crucial aspect for future management of BPH. In addition, the newly developed markers can be applied in various studies of the genetics and ecology of this species at a molecular level (e.g. population genetics, migration sources, and the evolution of BPH).</p><p>The primer information is available at <a class="link link-uri" href="http://sogo.dna.affrc.go.jp/cgi-bin/sogo.cgi?class=unka">http://sogo.dna.affrc.go.jp/cgi-bin/sogo.cgi?class=unka</a>. All the sequence data used for the generation of the SSR markers and the whole-genome sequence data were submitted to the DDBJ BioProject Database under ID PRJDB683.</p>                            <h2 scrollto-destination=5594990 id="5594990" class="section-title" data-legacy-id=s5>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available at www.dnaresearch.oxfordjournals.org.</a></p>                            <h2 scrollto-destination=5594992 id="5594992" class="section-title" data-legacy-id=s6>Funding</h2>
<p>This work was supported by the Grant-in-Aid for fellows of the Japan Society for the Promotion of Science (No. P10400) and the NIAS Strategic Research Fund. This work was also partially supported by the Grant-in-Aid for Scientific Research from the Ministry of Education, Culture, Sports, Science and Technology of Japan (No. 23580009). A Postdoctoral Fellowship for Foreign Researchers was awarded to J.J.</p>                            <h2 scrollto-destination=5594994 id="5594994" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank the Savannah River Ecology laboratory (The University of Georgia) for preparing the enriched genomic DNA. We thank Dr S. Fukuoka and Ms T. Ando (National Institute of Agrobiological Sciences) for technical support and SSR genotyping. We are grateful to Prof. A. Yoshimura (Kyushu University) for his highly valuable suggestions on linkage map construction. Many thanks to our colleagues at the Plant Breeding laboratory (Kyushu University), the staff of the Kyushu Okinawa Agricultural Research Center, and the National Institute of Agrobiological Sciences for their assistance and encouragement.</p>                            <h2 scrollto-destination=5594996 id="5594996" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSS030C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS030C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bottrell</div> <div class="given-names">D.G.</div></div>,  <div class="name"><div class="surname">Schoenly</div> <div class="given-names">K.G.</div></div>. </div><div class="article-title">Resurrecting the ghost of green revolutions past: the brown planthopper as a recurring threat to high-yielding rice production in tropical Asia</div>, <div class="source">J. Asia-Pacific Entomol.</div> , <div class="year">2012</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">122</div>-<div class="lpage">40</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Resurrecting%20the%20ghost%20of%20green%20revolutions%20past%3A%20the%20brown%20planthopper%20as%20a%20recurring%20threat%20to%20high-yielding%20rice%20production%20in%20tropical%20Asia&amp;author=D.G.%20Bottrell&amp;author=K.G.%20Schoenly&amp;publication_year=2012&amp;journal=J.%20Asia-Pacific%20Entomol.&amp;volume=15&amp;pages=122-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.aspen.2011.09.004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.aspen.2011.09.004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.aspen.2011.09.004"> </span></div> </div></div></div></div></div><div content-id="DSS030C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS030C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Catindig</div> <div class="given-names">J.L.A.</div></div>,  <div class="name"><div class="surname">Arida</div> <div class="given-names">G.S.</div></div>,  <div class="name"><div class="surname">Baehaki</div> <div class="given-names">S.E.</div></div>, et al. </div><div class="person-group"><div class="name"><div class="surname">Heong</div> <div class="given-names">K.L.</div></div>,  <div class="name"><div class="surname">Hardy</div> <div class="given-names">B.</div></div>. </div>, <div class="source">Planthoppers: New Threats to the Sustainability of Intensive Rice Production Systems in Asia</div> , <div class="year">2009</div><div class="publisher-loc">Los Baños, Philippines</div><div class="publisher-name">IRRI</div>(pg. <div class="fpage">191</div>-<div class="lpage">220</div>)</div></div></div></div><div content-id="DSS030C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS030C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Heong</div> <div class="given-names">K.L.</div></div>. </div><div class="person-group"><div class="name"><div class="surname">Heong</div> <div class="given-names">K.L.</div></div>,  <div class="name"><div class="surname">Hardy</div> <div class="given-names">B.</div></div>. </div><div class="article-title">Are planthopper problems caused by a breakdown in ecosystem services?</div>, <div class="source">Planthoppers: New Threats to the Sustainability of Intensive Rice Production Systems in Asia</div> , <div class="year">2009</div><div class="publisher-loc">Los Baños, Philippines</div><div class="publisher-name">IRRI</div>(pg. <div class="fpage">221</div>-<div class="lpage">31</div>)</div></div></div></div><div content-id="DSS030C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS030C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gallagher</div> <div class="given-names">K.D.</div></div>,  <div class="name"><div class="surname">Kenmore</div> <div class="given-names">P.E.</div></div>,  <div class="name"><div class="surname">Sogawa</div> <div class="given-names">K.</div></div>. </div><div class="person-group"><div class="name"><div class="surname">Denno</div> <div class="given-names">R.F.</div></div>,  <div class="name"><div class="surname">Perfect</div> <div class="given-names">T.J.</div></div>. </div><div class="article-title">Judicial use of insecticides deter planthopper outbreaks and extend the life of resistant varieties in Southwest Asian rice.</div>, <div class="source">Planthoppers: Their Ecology and Management</div> , <div class="year">1994</div><div class="publisher-loc">New York</div><div class="publisher-name">Chapman and Hall</div>(pg. <div class="fpage">599</div>-<div class="lpage">614</div>)</div></div></div></div><div content-id="DSS030C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS030C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cabauatan</div> <div class="given-names">P.Q.</div></div>,  <div class="name"><div class="surname">Cabunagan</div> <div class="given-names">R.C.</div></div>,  <div class="name"><div class="surname">Choi</div> <div class="given-names">I.R.</div></div>. </div><div class="person-group"><div class="name"><div class="surname">Heong</div> <div class="given-names">K.L.</div></div>,  <div class="name"><div class="surname">Hardy</div> <div class="given-names">B.</div></div>. </div><div class="article-title">Rice viruses transmitted by the brown planthopper <em>Nilaparvata lugens</em> Stål.</div>, <div class="source">Planthoppers: New Threats to the Sustainability of Intensive Rice Production Systems in Asia</div> , <div class="year">2009</div><div class="publisher-loc">Los Baños, Philippines</div><div class="publisher-name">IRRI</div>(pg. <div class="fpage">357</div>-<div class="lpage">68</div>)</div></div></div></div><div content-id="DSS030C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS030C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hare</div> <div class="given-names">J.D.</div></div>. </div><div class="person-group"><div class="name"><div class="surname">Denno</div> <div class="given-names">R.F.</div></div>,  <div class="name"><div class="surname">Perfect</div> <div class="given-names">T.J.</div></div>. </div><div class="article-title">Status and prospects for an integrated approach to the control of rice planthoppers</div>, <div class="source">Planthoppers: Their Ecology and Management</div> , <div class="year">1994</div><div class="publisher-loc">New York</div><div class="publisher-name">Chapman and Hall</div>(pg. <div class="fpage">615</div>-<div class="lpage">32</div>)</div></div></div></div><div content-id="DSS030C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS030C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tanaka</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Quantitative genetic analysis of biotypes of the brown planthopper <em>Nilaparvata lugens</em>: heritability of virulence to resistant rice varieties</div>, <div class="source">Entomologia Experimentalis et Applicata</div> , <div class="year">1999</div>, vol. <div class="volume">90</div> (pg. <div class="fpage">279</div>-<div class="lpage">87</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Quantitative%20genetic%20analysis%20of%20biotypes%20of%20the%20brown%20planthopper%20Nilaparvata%20lugens%3A%20heritability%20of%20virulence%20to%20resistant%20rice%20varieties&amp;author=K.%20Tanaka&amp;publication_year=1999&amp;journal=Entomologia%20Experimentalis%20et%20Applicata&amp;volume=90&amp;pages=279-87" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1570-7458.1999.00448.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1570-7458.1999.00448.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1570-7458.1999.00448.x"> </span></div> </div></div></div></div></div><div content-id="DSS030C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS030C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Den Hollander</div> <div class="given-names">J.</div></div>. </div><div class="article-title">The chromosomes of <em>Nilaparvata lugens</em> Stål. and some other Auchenorrhyncha</div>, <div class="source">Cytologia</div> , <div class="year">1982</div>, vol. <div class="volume">47</div> (pg. <div class="fpage">227</div>-<div class="lpage">36</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20chromosomes%20of%20Nilaparvata%20lugens%20St%C3%A5l.%20and%20some%20other%20Auchenorrhyncha&amp;author=J.%20Den%20Hollander&amp;publication_year=1982&amp;journal=Cytologia&amp;volume=47&amp;pages=227-36" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1508/cytologia.47.227" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1508%2Fcytologia.47.227" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1508%2Fcytologia.47.227"> </span></div> </div></div></div></div></div><div content-id="DSS030C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS030C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Saxena</div> <div class="given-names">R.C.</div></div>,  <div class="name"><div class="surname">Barrion</div> <div class="given-names">A.A.</div></div>. </div><div class="article-title">Brain cells and chromosomes of the brown planthopper <em>Nilaparvata lugens</em> (Stål)</div>, <div class="source">Cytologia</div> , <div class="year">1987</div>, vol. <div class="volume">52</div> (pg. <div class="fpage">859</div>-<div class="lpage">65</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Brain%20cells%20and%20chromosomes%20of%20the%20brown%20planthopper%20Nilaparvata%20lugens%20(St%C3%A5l)&amp;author=R.C.%20Saxena&amp;author=A.A.%20Barrion&amp;publication_year=1987&amp;journal=Cytologia&amp;volume=52&amp;pages=859-65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1508/cytologia.52.859" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1508%2Fcytologia.52.859" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1508%2Fcytologia.52.859"> </span></div> </div></div></div></div></div><div content-id="DSS030C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS030C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Noda</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Tatewaki</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Re-examination of chromosomes of three species of rice planthoppers (Homoptera: Delphacidae)</div>, <div class="source">Appl. Entomol. Zool.</div> , <div class="year">1990</div>, vol. <div class="volume">4</div> (pg. <div class="fpage">538</div>-<div class="lpage">40</div>)</div></div></div></div><div content-id="DSS030C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS030C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Fan</div> <div class="given-names">H.W.</div></div>,  <div class="name"><div class="surname">Huang</div> <div class="given-names">H.J.</div></div>, et al. </div><div class="article-title">Chitin synthase 1 gene and its two alternative splicing variants from two sap-sucking insects, <em>Nilaparvata lugens</em> and <em>Laodelphax striatellus</em> (Hemiptera: Delphacidae)</div>, <div class="source">Insect Biochem. Mol. Biol.</div> , <div class="year">2012</div>, vol. <div class="volume">42</div> (pg. <div class="fpage">637</div>-<div class="lpage">46</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Chitin%20synthase%201%20gene%20and%20its%20two%20alternative%20splicing%20variants%20from%20two%20sap-sucking%20insects%2C%20Nilaparvata%20lugens%20and%20Laodelphax%20striatellus%20(Hemiptera%3A%20Delphacidae)&amp;author=Y.%20Wang&amp;author=H.W.%20Fan&amp;author=H.J.%20Huang&amp;publication_year=2012&amp;journal=Insect%20Biochem.%20Mol.%20Biol.&amp;volume=42&amp;pages=637-46" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ibmb.2012.04.009" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ibmb.2012.04.009" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ibmb.2012.04.009"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22634163" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS030C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS030C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Noda</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Kawai</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Koizumi</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">Annotated ESTs from various tissues of the brown planthopper <em>Nilaparvata lugens</em>: a genomic resource for studying agricultural pests</div>, <div class="source">BMC Genomics</div> , <div class="year">2008</div>, vol. <div class="volume">9</div> pg. <div class="fpage">117</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Annotated%20ESTs%20from%20various%20tissues%20of%20the%20brown%20planthopper%20Nilaparvata%20lugens%3A%20a%20genomic%20resource%20for%20studying%20agricultural%20pests&amp;author=H.%20Noda&amp;author=S.%20Kawai&amp;author=Y.%20Koizumi&amp;publication_year=2008&amp;journal=BMC%20Genomics&amp;volume=9&amp;pages=117" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-9-117" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-9-117" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-9-117"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18315884" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS030C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS030C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Xue</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Bao</div> <div class="given-names">Y.Y.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">B.</div></div>, et al. </div><div class="article-title">Transcriptome analysis of the brown planthopper <em>Nilaparvata lugens</em></div>, <div class="source">PLoS One</div> , <div class="year">2010</div>, vol. <div class="volume">5</div> pg. <div class="fpage">e14233</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptome%20analysis%20of%20the%20brown%20planthopper%20Nilaparvata%20lugens&amp;author=J.%20Xue&amp;author=Y.Y.%20Bao&amp;author=B.%20Li&amp;publication_year=2010&amp;journal=PLoS%20One&amp;volume=5&amp;pages=e14233" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0014233" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0014233" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0014233"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21151909" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS030C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS030C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bass</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Hebsgaard</div> <div class="given-names">M.B.</div></div>,  <div class="name"><div class="surname">Hughes</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Genomic resources for the brown planthopper, <em>Nilaparvata lugens</em>: transcriptome pyrosequencing and microarray design</div>, <div class="source">Insect Sci.</div> , <div class="year">2012</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">1</div>-<div class="lpage">12</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20resources%20for%20the%20brown%20planthopper%2C%20Nilaparvata%20lugens%3A%20transcriptome%20pyrosequencing%20and%20microarray%20design&amp;author=C.%20Bass&amp;author=M.B.%20Hebsgaard&amp;author=J.%20Hughes&amp;publication_year=2012&amp;journal=Insect%20Sci.&amp;volume=9&amp;pages=1-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/ins.2012.19.issue-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fins.2012.19.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fins.2012.19.issue-1"> </span></div> </div></div></div></div></div><div content-id="DSS030C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS030C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Peng</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Zha</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">He</div> <div class="given-names">R.</div></div>, et al. </div><div class="article-title">Pyrosequencing the midgut transcriptome of the brown planthopper, <em>Nilaparvata lugens</em></div>, <div class="source">Insect Mol. Biol.</div> , <div class="year">2011</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">745</div>-<div class="lpage">62</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Pyrosequencing%20the%20midgut%20transcriptome%20of%20the%20brown%20planthopper%2C%20Nilaparvata%20lugens&amp;author=X.%20Peng&amp;author=W.%20Zha&amp;author=R.%20He&amp;publication_year=2011&amp;journal=Insect%20Mol.%20Biol.&amp;volume=20&amp;pages=745-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/imb.2011.20.issue-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fimb.2011.20.issue-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fimb.2011.20.issue-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21919985" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS030C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS030C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bao</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Wu</div> <div class="given-names">W.</div></div>, et al. </div><div class="article-title"><em>De novo</em> intestine-specific transcriptome of the brown planthopper <em>Nilaparvata lugens</em> revealed potential functions in digestion, detoxification and immune response</div>, <div class="source">Genomics</div> , <div class="year">2012</div>, vol. <div class="volume">99</div> (pg. <div class="fpage">256</div>-<div class="lpage">64</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=De%20novo%20intestine-specific%20transcriptome%20of%20the%20brown%20planthopper%20Nilaparvata%20lugens%20revealed%20potential%20functions%20in%20digestion%2C%20detoxification%20and%20immune%20response&amp;author=Y.%20Bao&amp;author=Y.%20Wang&amp;author=W.%20Wu&amp;publication_year=2012&amp;journal=Genomics&amp;volume=99&amp;pages=256-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ygeno.2012.02.002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ygeno.2012.02.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ygeno.2012.02.002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22361737" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS030C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS030C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jing</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Liu</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Peng</div> <div class="given-names">L.</div></div>, et al. </div><div class="article-title">Development and use of e-SSR markers for assessing genetic diversity in the brown planthopper (<em>Nilaparvata lugens</em> Stål)</div>, <div class="source">Bull. Entomol. Res.</div> , <div class="year">2011</div>, vol. <div class="volume">102</div> (pg. <div class="fpage">113</div>-<div class="lpage">22</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20and%20use%20of%20e-SSR%20markers%20for%20assessing%20genetic%20diversity%20in%20the%20brown%20planthopper%20(Nilaparvata%20lugens%20St%C3%A5l)&amp;author=S.%20Jing&amp;author=B.%20Liu&amp;author=L.%20Peng&amp;publication_year=2011&amp;journal=Bull.%20Entomol.%20Res.&amp;volume=102&amp;pages=113-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1017/S0007485311000435" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1017%2FS0007485311000435" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1017%2FS0007485311000435"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21896240" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS030C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS030C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Abercrombie</div> <div class="given-names">L.G.</div></div>,  <div class="name"><div class="surname">Anderson</div> <div class="given-names">C.M.</div></div>, et al. </div><div class="collab">Molecular Ecology Resources Primer Development Consortium</div><div class="article-title">Permanent genetic resources added to molecular ecology resources database 1 January 2009–30 April 2009</div>, <div class="source">Mol. Ecol. Resources</div> , <div class="year">2009</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">1375</div>-<div class="lpage">429</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Permanent%20genetic%20resources%20added%20to%20molecular%20ecology%20resources%20database%201%20January%202009%E2%80%9330%20April%202009&amp;author=L.G.%20Abercrombie&amp;author=C.M.%20Anderson&amp;publication_year=2009&amp;journal=Mol.%20Ecol.%20Resources&amp;volume=9&amp;pages=1375-429" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/men.2009.9.issue-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmen.2009.9.issue-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmen.2009.9.issue-5"> </span></div> </div></div></div></div></div><div content-id="DSS030C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS030C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sun</div> <div class="given-names">J.T.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">Y.K.</div></div>,  <div class="name"><div class="surname">Ge</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Hong</div> <div class="given-names">X.Y.</div></div>. </div><div class="article-title">Mining and characterization of sequence tagged microsatellites from the brown planthopper <em>Nilaparvata lugens</em></div>, <div class="source">J. Insect Sci.</div> , <div class="year">2011</div>, vol. <div class="volume">11</div> pg. <div class="fpage">134</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mining%20and%20characterization%20of%20sequence%20tagged%20microsatellites%20from%20the%20brown%20planthopper%20Nilaparvata%20lugens&amp;author=J.T.%20Sun&amp;author=Y.K.%20Zhang&amp;author=C.%20Ge&amp;author=X.Y.%20Hong&amp;publication_year=2011&amp;journal=J.%20Insect%20Sci.&amp;volume=11&amp;pages=134" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1673/031.011.13401" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1673%2F031.011.13401" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1673%2F031.011.13401"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22243416" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS030C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS030C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jing</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Zhou</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Yu</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">Isolation and characterization of microsatellite markers in brown planthopper (<em>Nilaparvata lugens</em> Stål)</div>, <div class="source">Int. J. Mol. Sci.</div> , <div class="year">2012</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">9527</div>-<div class="lpage">33</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Isolation%20and%20characterization%20of%20microsatellite%20markers%20in%20brown%20planthopper%20(Nilaparvata%20lugens%20St%C3%A5l)&amp;author=S.%20Jing&amp;author=X.%20Zhou&amp;author=H.%20Yu&amp;publication_year=2012&amp;journal=Int.%20J.%20Mol.%20Sci.&amp;volume=13&amp;pages=9527-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3390/ijms13089527" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3390%2Fijms13089527" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3390%2Fijms13089527"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22949813" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS030C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS030C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tanaka</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Matsumura</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Development of virulence to resistant rice varieties in the brown planthopper, <em>Nilaparvata lugens</em> (Homoptera: Delphacidae), immigrating into Japan</div>, <div class="source">Appl. Entomol. Zool.</div> , <div class="year">2000</div>, vol. <div class="volume">35</div> (pg. <div class="fpage">529</div>-<div class="lpage">33</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20virulence%20to%20resistant%20rice%20varieties%20in%20the%20brown%20planthopper%2C%20Nilaparvata%20lugens%20(Homoptera%3A%20Delphacidae)%2C%20immigrating%20into%20Japan&amp;author=K.%20Tanaka&amp;author=M.%20Matsumura&amp;publication_year=2000&amp;journal=Appl.%20Entomol.%20Zool.&amp;volume=35&amp;pages=529-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1303/aez.2000.529" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1303%2Faez.2000.529" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1303%2Faez.2000.529"> </span></div> </div></div></div></div></div><div content-id="DSS030C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS030C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Otuka</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Watanabe</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Suzuki</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Matsumura</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Furuno</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Chino</div> <div class="given-names">M.</div></div>. </div><div class="article-title">A migration analysis of the rice planthopper <em>Nilaparvata lugens</em> from the Philippines to East Asia with three-dimensional computer simulations</div>, <div class="source">Popul. Ecol.</div> , <div class="year">2005</div>, vol. <div class="volume">47</div> (pg. <div class="fpage">143</div>-<div class="lpage">50</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20migration%20analysis%20of%20the%20rice%20planthopper%20Nilaparvata%20lugens%20from%20the%20Philippines%20to%20East%20Asia%20with%20three-dimensional%20computer%20simulations&amp;author=A.%20Otuka&amp;author=T.%20Watanabe&amp;author=Y.%20Suzuki&amp;author=M.%20Matsumura&amp;author=A.%20Furuno&amp;author=M.%20Chino&amp;publication_year=2005&amp;journal=Popul.%20Ecol.&amp;volume=47&amp;pages=143-50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10144-005-0216-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10144-005-0216-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10144-005-0216-1"> </span></div> </div></div></div></div></div><div content-id="DSS030C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS030C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dellaporta</div> <div class="given-names">S.L.</div></div>,  <div class="name"><div class="surname">Wood</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Hicks</div> <div class="given-names">J.R.</div></div>. </div><div class="article-title">A plant DNA minipreparation: version II</div>, <div class="source">Plant Mol. Biol. Rep.</div> , <div class="year">1983</div>, vol. <div class="volume">1</div> (pg. <div class="fpage">19</div>-<div class="lpage">21</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20plant%20DNA%20minipreparation%3A%20version%20II&amp;author=S.L.%20Dellaporta&amp;author=J.%20Wood&amp;author=J.R.%20Hicks&amp;publication_year=1983&amp;journal=Plant%20Mol.%20Biol.%20Rep.&amp;volume=1&amp;pages=19-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF02712670" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF02712670" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF02712670"> </span></div> </div></div></div></div></div><div content-id="DSS030C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS030C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tang</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Lv</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Jing</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Zhu</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">He</div> <div class="given-names">G.</div></div>. </div><div class="article-title">Bacterial symbionts of the brown planthopper, <em>Nilaparvata lugens</em> (Homoptera: Delphacidae)</div>, <div class="source">Appl. Environ. Microbiol.</div> , <div class="year">2010</div>, vol. <div class="volume">76</div> (pg. <div class="fpage">1740</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bacterial%20symbionts%20of%20the%20brown%20planthopper%2C%20Nilaparvata%20lugens%20(Homoptera%3A%20Delphacidae)&amp;author=M.%20Tang&amp;author=L.%20Lv&amp;author=S.%20Jing&amp;author=L.%20Zhu&amp;author=G.%20He&amp;publication_year=2010&amp;journal=Appl.%20Environ.%20Microbiol.&amp;volume=76&amp;pages=1740-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AEM.02240-09" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.02240-09" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FAEM.02240-09"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20097822" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS030C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS030C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dean</div> <div class="given-names">F.B.</div></div>,  <div class="name"><div class="surname">Nelson</div> <div class="given-names">J.R.</div></div>,  <div class="name"><div class="surname">Giesler</div> <div class="given-names">T.L.</div></div>,  <div class="name"><div class="surname">Lasken</div> <div class="given-names">R.S.</div></div>. </div><div class="article-title">Polymerase and multiply-primed rolling circle amplification rapid amplification of plasmid and phage DNA using <em>Phi</em>29 DNA</div>, <div class="source">Genome Res.</div> , <div class="year">2001</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">1095</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Polymerase%20and%20multiply-primed%20rolling%20circle%20amplification%20rapid%20amplification%20of%20plasmid%20and%20phage%20DNA%20using%20Phi29%20DNA&amp;author=F.B.%20Dean&amp;author=J.R.%20Nelson&amp;author=T.L.%20Giesler&amp;author=R.S.%20Lasken&amp;publication_year=2001&amp;journal=Genome%20Res.&amp;volume=11&amp;pages=1095-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.180501" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.180501" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.180501"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11381035" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS030C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS030C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Glenn</div> <div class="given-names">T.C.</div></div>,  <div class="name"><div class="surname">Schable</div> <div class="given-names">N.A.</div></div>. </div><div class="person-group"><div class="name"><div class="surname">Zimmer</div> <div class="given-names">E.A.</div></div>,  <div class="name"><div class="surname">Roalson</div> <div class="given-names">E.H.</div></div>. </div><div class="article-title">Isolating microsatellite DNA loci</div>, <div class="source">Methods in Enzymology</div> , <div class="year">2005</div>, vol. <div class="volume">vol. 395</div> <div class="publisher-loc">San Diego, CA</div><div class="publisher-name">Academic Press</div>(pg. <div class="fpage">202</div>-<div class="lpage">22</div>)</div></div></div></div><div content-id="DSS030C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS030C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zane</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Bargelloni</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Patarnello</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Strategies for microsatellite isolation: a review</div>, <div class="source">Mol. Ecol.</div> , <div class="year">2002</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">1</div>-<div class="lpage">16</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Strategies%20for%20microsatellite%20isolation%3A%20a%20review&amp;author=L.%20Zane&amp;author=L.%20Bargelloni&amp;author=T.%20Patarnello&amp;publication_year=2002&amp;journal=Mol.%20Ecol.&amp;volume=11&amp;pages=1-16" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.0962-1083.2001.01418.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.0962-1083.2001.01418.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.0962-1083.2001.01418.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11903900" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS030C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS030C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Temnykh</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">DeClerck</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Lukashova</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Lipovich</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Cartinhour</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">McCouch</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Computational and experimental analysis of microsatellites in rice (<em>Oryza sativa</em> L.): frequency, length variation, transposon associations, and genetic marker potential</div>, <div class="source">Genome Res.</div> , <div class="year">2001</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">1441</div>-<div class="lpage">52</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Computational%20and%20experimental%20analysis%20of%20microsatellites%20in%20rice%20(Oryza%20sativa%20L.)%3A%20frequency%2C%20length%20variation%2C%20transposon%20associations%2C%20and%20genetic%20marker%20potential&amp;author=S.%20Temnykh&amp;author=G.%20DeClerck&amp;author=A.%20Lukashova&amp;author=L.%20Lipovich&amp;author=S.%20Cartinhour&amp;author=S.%20McCouch&amp;publication_year=2001&amp;journal=Genome%20Res.&amp;volume=11&amp;pages=1441-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.184001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.184001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.184001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11483586" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS030C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS030C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rozen</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Skaletsky</div> <div class="given-names">H.J.</div></div>. </div><div class="article-title">Primer3 on the WWW for general users and for biologist programmers</div>, <div class="source">Methods Mol. Biol.</div> , <div class="year">2000</div>, vol. <div class="volume">132</div> (pg. <div class="fpage">365</div>-<div class="lpage">86</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Primer3%20on%20the%20WWW%20for%20general%20users%20and%20for%20biologist%20programmers&amp;author=S.%20Rozen&amp;author=H.J.%20Skaletsky&amp;publication_year=2000&amp;journal=Methods%20Mol.%20Biol.&amp;volume=132&amp;pages=365-86" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10547847" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS030C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS030C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Miller</div> <div class="given-names">J.R.</div></div>,  <div class="name"><div class="surname">Delcher</div> <div class="given-names">A.L.</div></div>,  <div class="name"><div class="surname">Koren</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Aggressive assembly of pyrosequencing reads with mates</div>, <div class="source">Bioinformatics</div> , <div class="year">2008</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">2818</div>-<div class="lpage">24</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Aggressive%20assembly%20of%20pyrosequencing%20reads%20with%20mates&amp;author=J.R.%20Miller&amp;author=A.L.%20Delcher&amp;author=S.%20Koren&amp;publication_year=2008&amp;journal=Bioinformatics&amp;volume=24&amp;pages=2818-24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btn548" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtn548" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtn548"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18952627" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS030C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS030C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Durbin</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Fast and accurate short read alignment with Burrows-Wheeler transform</div>, <div class="source">Bioinformatics</div> , <div class="year">2009</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">1754</div>-<div class="lpage">60</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fast%20and%20accurate%20short%20read%20alignment%20with%20Burrows-Wheeler%20transform&amp;author=H.%20Li&amp;author=R.%20Durbin&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=1754-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp324" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp324" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp324"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19451168" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS030C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS030C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Handsaker</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Wysoker</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">The sequence alignment/map (SAM) format and SAM tools</div>, <div class="source">Bioinformatics</div> , <div class="year">2009</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">2078</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20sequence%20alignment%2Fmap%20(SAM)%20format%20and%20SAM%20tools&amp;author=H.%20Li&amp;author=B.%20Handsaker&amp;author=A.%20Wysoker&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=2078-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp352" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp352" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp352"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19505943" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS030C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS030C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Milne</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Bayer</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Cardle</div> <div class="given-names">L.</div></div>, et al. </div><div class="article-title">Tablet–next generation sequence assembly visualization</div>, <div class="source">Bioinformatics</div> , <div class="year">2010</div>, vol. <div class="volume">26</div> (pg. <div class="fpage">401</div>-<div class="lpage">2</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Tablet%E2%80%93next%20generation%20sequence%20assembly%20visualization&amp;author=I.%20Milne&amp;author=M.%20Bayer&amp;author=L.%20Cardle&amp;publication_year=2010&amp;journal=Bioinformatics&amp;volume=26&amp;pages=401-2" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp666" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp666" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp666"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19965881" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS030C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS030C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Van Ooijen</div> <div class="given-names">J.W.</div></div>. </div>, <div class="source">JoinMap®4, Software for the Calculation of Genetic Linkage Maps in Experimental Populations</div> , <div class="year">2006</div><div class="publisher-loc">Wageningen, Netherlands</div><div class="publisher-name">Kyazma, B.V</div></div></div></div></div><div content-id="DSS030C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS030C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kosambi</div> <div class="given-names">D.D.</div></div>. </div><div class="article-title">The estimation of map distances from recombination values</div>, <div class="source">Ann. Eugen.</div> , <div class="year">1944</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">172</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20estimation%20of%20map%20distances%20from%20recombination%20values&amp;author=D.D.%20Kosambi&amp;publication_year=1944&amp;journal=Ann.%20Eugen.&amp;volume=12&amp;pages=172-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1469-1809.1943.tb02321.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1469-1809.1943.tb02321.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1469-1809.1943.tb02321.x"> </span></div> </div></div></div></div></div><div content-id="DSS030C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS030C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Voorrips</div> <div class="given-names">R.E.</div></div>. </div><div class="article-title">MapChart: software for the graphical presentation of linkage maps and QTLs</div>, <div class="source">J. Hered.</div> , <div class="year">2002</div>, vol. <div class="volume">93</div> (pg. <div class="fpage">77</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MapChart%3A%20software%20for%20the%20graphical%20presentation%20of%20linkage%20maps%20and%20QTLs&amp;author=R.E.%20Voorrips&amp;publication_year=2002&amp;journal=J.%20Hered.&amp;volume=93&amp;pages=77-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jhered/93.1.77" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjhered%2F93.1.77" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjhered%2F93.1.77"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12011185" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS030C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS030C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chakravarti</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Lasher</div> <div class="given-names">L.K.</div></div>,  <div class="name"><div class="surname">Reefer</div> <div class="given-names">J.E.</div></div>. </div><div class="article-title">A maximum likelihood method for estimating genome length using genetic linkage data</div>, <div class="source">Genetics</div> , <div class="year">1991</div>, vol. <div class="volume">128</div> (pg. <div class="fpage">175</div>-<div class="lpage">82</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20maximum%20likelihood%20method%20for%20estimating%20genome%20length%20using%20genetic%20linkage%20data&amp;author=A.%20Chakravarti&amp;author=L.K.%20Lasher&amp;author=J.E.%20Reefer&amp;publication_year=1991&amp;journal=Genetics&amp;volume=128&amp;pages=175-82" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2060775" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS030C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS030C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mun</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Song</div> <div class="given-names">Y.H.</div></div>,  <div class="name"><div class="surname">Roderick</div> <div class="given-names">G.K.</div></div>. </div><div class="article-title">Isolation and characterization of microsatellites in the brown planthopper, <em>Nilaparvata lugenes</em> Stål</div>, <div class="source">Korea J. Appl. Entomol.</div> , <div class="year">1994</div>, vol. <div class="volume">43</div> (pg. <div class="fpage">311</div>-<div class="lpage">5</div>)</div></div></div></div><div content-id="DSS030C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS030C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nielsen</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Paul</div> <div class="given-names">J.S.</div></div>,  <div class="name"><div class="surname">Albrechtsen</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Song</div> <div class="given-names">Y.S.</div></div>. </div><div class="article-title">Genotype and SNP calling from next generation sequencing data</div>, <div class="source">Nature Rev.</div> , <div class="year">2011</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">443</div>-<div class="lpage">51</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genotype%20and%20SNP%20calling%20from%20next%20generation%20sequencing%20data&amp;author=R.%20Nielsen&amp;author=J.S.%20Paul&amp;author=A.%20Albrechtsen&amp;author=Y.S.%20Song&amp;publication_year=2011&amp;journal=Nature%20Rev.&amp;volume=12&amp;pages=443-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg2986" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg2986" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg2986"> </span></div> </div></div></div></div></div><div content-id="DSS030C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS030C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chistiakov</div> <div class="given-names">D.A.</div></div>,  <div class="name"><div class="surname">Hellemans</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Haley</div> <div class="given-names">C.S.</div></div>, et al. </div><div class="article-title">A microsatellite linkage map of the European sea bass <em>Dicentrarchus labrax</em> L.</div>, <div class="source">Genetics</div> , <div class="year">2005</div>, vol. <div class="volume">170</div> (pg. <div class="fpage">1821</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20microsatellite%20linkage%20map%20of%20the%20European%20sea%20bass%20Dicentrarchus%20labrax%20L.&amp;author=D.A.%20Chistiakov&amp;author=B.%20Hellemans&amp;author=C.S.%20Haley&amp;publication_year=2005&amp;journal=Genetics&amp;volume=170&amp;pages=1821-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.104.039719" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.104.039719" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.104.039719"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15937133" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS030C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS030C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">C.M.</div></div>,  <div class="name"><div class="surname">Zhu</div> <div class="given-names">Z.Y.</div></div>,  <div class="name"><div class="surname">Lo</div> <div class="given-names">L.C.</div></div>, et al. </div><div class="article-title">A microsatellite linkage map of barramundi, <em>Lates calcarifer</em></div>, <div class="source">Genetics</div> , <div class="year">2007</div>, vol. <div class="volume">175</div> (pg. <div class="fpage">907</div>-<div class="lpage">15</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20microsatellite%20linkage%20map%20of%20barramundi%2C%20Lates%20calcarifer&amp;author=C.M.%20Wang&amp;author=Z.Y.%20Zhu&amp;author=L.C.%20Lo&amp;publication_year=2007&amp;journal=Genetics&amp;volume=175&amp;pages=907-15" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.106.059972" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.106.059972" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.106.059972"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17179079" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS030C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS030C42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ronin</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Mester</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Minkov</div> <div class="given-names">D.</div></div>, et al. </div><div class="article-title">Two-phase analysis in consensus genetic mapping</div>, <div class="source">G3</div> , <div class="year">2012</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">537</div>-<div class="lpage">49</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Two-phase%20analysis%20in%20consensus%20genetic%20mapping&amp;author=Y.%20Ronin&amp;author=D.%20Mester&amp;author=D.%20Minkov&amp;publication_year=2012&amp;journal=G3&amp;volume=2&amp;pages=537-49" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/g3.112.002428" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fg3.112.002428" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fg3.112.002428"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22670224" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS030C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS030C43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tan</div> <div class="given-names">Y.D.</div></div>,  <div class="name"><div class="surname">Wan</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Zhu</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">An amplified fragment length polymorphism map of the silkworm</div>, <div class="source">Genetics</div> , <div class="year">2001</div>, vol. <div class="volume">157</div> (pg. <div class="fpage">1277</div>-<div class="lpage">84</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20amplified%20fragment%20length%20polymorphism%20map%20of%20the%20silkworm&amp;author=Y.D.%20Tan&amp;author=C.%20Wan&amp;author=Y.%20Zhu&amp;publication_year=2001&amp;journal=Genetics&amp;volume=157&amp;pages=1277-84" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11238411" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS030C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS030C44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Xiang</div> <div class="given-names">J.H.</div></div>,  <div class="name"><div class="surname">Liu</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Dong</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">X.</div></div>. </div><div class="article-title">Construction of AFLP-based genetic linkage map for Zhikong scallop, <em>Chlamys farreri</em> and mapping of sex-linked markers</div>, <div class="source">Aquaculture</div> , <div class="year">2005</div>, vol. <div class="volume">245</div> (pg. <div class="fpage">63</div>-<div class="lpage">73</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20AFLP-based%20genetic%20linkage%20map%20for%20Zhikong%20scallop%2C%20Chlamys%20farreri%20and%20mapping%20of%20sex-linked%20markers&amp;author=L.%20Li&amp;author=J.H.%20Xiang&amp;author=X.%20Liu&amp;author=Y.%20Zhang&amp;author=B.%20Dong&amp;author=X.%20Zhang&amp;publication_year=2005&amp;journal=Aquaculture&amp;volume=245&amp;pages=63-73" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.aquaculture.2004.12.015" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.aquaculture.2004.12.015" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.aquaculture.2004.12.015"> </span></div> </div></div></div></div></div><div content-id="DSS030C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS030C45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Guo</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">Q.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">Q.Z.</div></div>,  <div class="name"><div class="surname">Kong</div> <div class="given-names">L.F.</div></div>. </div><div class="article-title">Genetic mapping and QTL analysis of growth-related traits in the Pacific oyster</div>, <div class="source">Mar. Biotechnol.</div> , <div class="year">2012</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">218</div>-<div class="lpage">26</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20mapping%20and%20QTL%20analysis%20of%20growth-related%20traits%20in%20the%20Pacific%20oyster&amp;author=X.%20Guo&amp;author=Q.%20Li&amp;author=Q.Z.%20Wang&amp;author=L.F.%20Kong&amp;publication_year=2012&amp;journal=Mar.%20Biotechnol.&amp;volume=14&amp;pages=218-26" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10126-011-9405-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10126-011-9405-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10126-011-9405-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21932055" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS030C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS030C46" class="ref-content "><div class="label -label">46</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">McColdrick</div> <div class="given-names">D.J.</div></div>,  <div class="name"><div class="surname">Hedgecock</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Fixation, segregation and linkage of allozyme loci in inbred families of the Pacific oyster <em>Crassostrea giga</em> (Thunberg): implications for the causes of inbreeding depression</div>, <div class="source">Genetics</div> , <div class="year">1997</div>, vol. <div class="volume">146</div> (pg. <div class="fpage">321</div>-<div class="lpage">34</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fixation%2C%20segregation%20and%20linkage%20of%20allozyme%20loci%20in%20inbred%20families%20of%20the%20Pacific%20oyster%20Crassostrea%20giga%20(Thunberg)%3A%20implications%20for%20the%20causes%20of%20inbreeding%20depression&amp;author=D.J.%20McColdrick&amp;author=D.%20Hedgecock&amp;publication_year=1997&amp;journal=Genetics&amp;volume=146&amp;pages=321-34" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9136021" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS030C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS030C47" class="ref-content "><div class="label -label">47</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tao</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Hart</div> <div class="given-names">D.L.</div></div>,  <div class="name"><div class="surname">Laurie</div> <div class="given-names">C.C.</div></div>. </div><div class="article-title">Sex-ratio segregation distortion associated with reproductive isolation in <em>Drosophila</em></div>, <div class="source">Proc. Nat. Acad. Sci. USA</div> , <div class="year">2001</div>, vol. <div class="volume">98</div> (pg. <div class="fpage">13183</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sex-ratio%20segregation%20distortion%20associated%20with%20reproductive%20isolation%20in%20Drosophila&amp;author=Y.%20Tao&amp;author=D.L.%20Hart&amp;author=C.C.%20Laurie&amp;publication_year=2001&amp;journal=Proc.%20Nat.%20Acad.%20Sci.%20USA&amp;volume=98&amp;pages=13183-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.231478798" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.231478798" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.231478798"> </span></div> </div></div></div></div></div><div content-id="DSS030C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS030C48" class="ref-content "><div class="label -label">48</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lyttle</div> <div class="given-names">T.W.</div></div>. </div><div class="article-title">Cheaters sometimes prosper: distortion of mendelian segregation by meiotic drive</div>, <div class="source">Trends Genet.</div> , <div class="year">1993</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">205</div>-<div class="lpage">10</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cheaters%20sometimes%20prosper%3A%20distortion%20of%20mendelian%20segregation%20by%20meiotic%20drive&amp;author=T.W.%20Lyttle&amp;publication_year=1993&amp;journal=Trends%20Genet.&amp;volume=9&amp;pages=205-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0168-9525(93)90120-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0168-9525(93)90120-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0168-9525(93)90120-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8337761" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS030C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS030C49" class="ref-content "><div class="label -label">49</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ferreira</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">da Silva</div> <div class="given-names">M.F.</div></div>,  <div class="name"><div class="surname">da Costa e Silva</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Cruz</div> <div class="given-names">C.D.</div></div>. </div><div class="article-title">Estimating the effects of population size and type on the accuracy of genetic maps</div>, <div class="source">Genet. Mol. Biol.</div> , <div class="year">2006</div>, vol. <div class="volume">29</div> (pg. <div class="fpage">182</div>-<div class="lpage">92</div>)</div></div></div></div><div content-id="DSS030C50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS030C50" class="ref-content "><div class="label -label">50</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hawthorne</div> <div class="given-names">D.J.</div></div>. </div><div class="article-title">AFLP-based genetic linkage map of the Colorado potato beetle <em>Leptinotarsa decemlineata</em>: sex chromosomes and a pyrethroid-resistance candidate gene</div>, <div class="source">Genetics</div> , <div class="year">2001</div>, vol. <div class="volume">158</div> (pg. <div class="fpage">695</div>-<div class="lpage">700</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=AFLP-based%20genetic%20linkage%20map%20of%20the%20Colorado%20potato%20beetle%20Leptinotarsa%20decemlineata%3A%20sex%20chromosomes%20and%20a%20pyrethroid-resistance%20candidate%20gene&amp;author=D.J.%20Hawthorne&amp;publication_year=2001&amp;journal=Genetics&amp;volume=158&amp;pages=695-700" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11404333" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS030C51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS030C51" class="ref-content "><div class="label -label">51</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Liu</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Karsi</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Cao</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Dunham</div> <div class="given-names">R.</div></div>. </div><div class="article-title">An AFLP-based genetic linkage map of channel catfish (<em>Ictalurus punctatus</em>) constructed by using an interspecific hybrid resource family</div>, <div class="source">Genetics</div> , <div class="year">2003</div>, vol. <div class="volume">165</div> (pg. <div class="fpage">687</div>-<div class="lpage">94</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20AFLP-based%20genetic%20linkage%20map%20of%20channel%20catfish%20(Ictalurus%20punctatus)%20constructed%20by%20using%20an%20interspecific%20hybrid%20resource%20family&amp;author=Z.%20Liu&amp;author=A.%20Karsi&amp;author=P.%20Li&amp;author=D.%20Cao&amp;author=R.%20Dunham&amp;publication_year=2003&amp;journal=Genetics&amp;volume=165&amp;pages=687-94" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14573480" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS030C52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS030C52" class="ref-content "><div class="label -label">52</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tobler</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Kapan</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Flanagan</div> <div class="given-names">N.S.</div></div>, et al. </div><div class="article-title">First-generation linkage map of the warningly colored butterfly <em>Heliconius erato</em></div>, <div class="source">Heredity</div> , <div class="year">2005</div>, vol. <div class="volume">94</div> (pg. <div class="fpage">408</div>-<div class="lpage">17</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=First-generation%20linkage%20map%20of%20the%20warningly%20colored%20butterfly%20Heliconius%20erato&amp;author=A.%20Tobler&amp;author=D.%20Kapan&amp;author=N.S.%20Flanagan&amp;publication_year=2005&amp;journal=Heredity&amp;volume=94&amp;pages=408-17" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/sj.hdy.6800619" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsj.hdy.6800619" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsj.hdy.6800619"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15592446" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS030C53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS030C53" class="ref-content "><div class="label -label">53</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Feldmeyer</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Pen</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Beukeboom</div> <div class="given-names">L.W.</div></div>. </div><div class="article-title">A microsatellite marker linkage map of the housefly, <em>Musca domestica</em>: evidence for male recombination</div>, <div class="source">Insect Mol. Biol.</div> , <div class="year">2010</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">575</div>-<div class="lpage">81</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20microsatellite%20marker%20linkage%20map%20of%20the%20housefly%2C%20Musca%20domestica%3A%20evidence%20for%20male%20recombination&amp;author=B.%20Feldmeyer&amp;author=I.%20Pen&amp;author=L.W.%20Beukeboom&amp;publication_year=2010&amp;journal=Insect%20Mol.%20Biol.&amp;volume=19&amp;pages=575-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1365-2583.2010.01016.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1365-2583.2010.01016.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1365-2583.2010.01016.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20491981" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS030C54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS030C54" class="ref-content "><div class="label -label">54</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sakamoto</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Danzmann</div> <div class="given-names">R.G.</div></div>,  <div class="name"><div class="surname">Gharbi</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">A microsatellite linkage map of rainbow trout (<em>Oncorhynchus mykiss</em>) characterized by large sex-specific differences in recombination rates</div>, <div class="source">Genetics</div> , <div class="year">2000</div>, vol. <div class="volume">155</div> (pg. <div class="fpage">1331</div>-<div class="lpage">45</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20microsatellite%20linkage%20map%20of%20rainbow%20trout%20(Oncorhynchus%20mykiss)%20characterized%20by%20large%20sex-specific%20differences%20in%20recombination%20rates&amp;author=T.%20Sakamoto&amp;author=R.G.%20Danzmann&amp;author=K.%20Gharbi&amp;publication_year=2000&amp;journal=Genetics&amp;volume=155&amp;pages=1331-45" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10880492" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS030C55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS030C55" class="ref-content "><div class="label -label">55</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lien</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Gidskehaug</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Moen</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">A dense SNP-based linkage map for Atlantic salmon (<em>Salmo salar</em>) reveals extended chromosome homologies and striking differences in sex-specific recombination patterns</div>, <div class="source">BMC Genomics</div> , <div class="year">2011</div>, vol. <div class="volume">12</div> pg. <div class="fpage">615</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20dense%20SNP-based%20linkage%20map%20for%20Atlantic%20salmon%20(Salmo%20salar)%20reveals%20extended%20chromosome%20homologies%20and%20striking%20differences%20in%20sex-specific%20recombination%20patterns&amp;author=S.%20Lien&amp;author=L.%20Gidskehaug&amp;author=T.%20Moen&amp;publication_year=2011&amp;journal=BMC%20Genomics&amp;volume=12&amp;pages=615" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-12-615" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-12-615" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-12-615"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22182215" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS030C56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS030C56" class="ref-content "><div class="label -label">56</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhan</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Fan</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">You</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Yu</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Ke</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Construction of an integrated map of <em>Haliotis diversicolor</em> using microsatellite markers</div>, <div class="source">Mar. Biotechnol.</div> , <div class="year">2012</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">79</div>-<div class="lpage">86</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20an%20integrated%20map%20of%20Haliotis%20diversicolor%20using%20microsatellite%20markers&amp;author=X.%20Zhan&amp;author=F.%20Fan&amp;author=W.%20You&amp;author=J.%20Yu&amp;author=C.%20Ke&amp;publication_year=2012&amp;journal=Mar.%20Biotechnol.&amp;volume=14&amp;pages=79-86" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10126-011-9390-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10126-011-9390-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10126-011-9390-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21617897" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS030C57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS030C57" class="ref-content "><div class="label -label">57</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Miao</div> <div class="given-names">X.X.</div></div>,  <div class="name"><div class="surname">Xu</div> <div class="given-names">S.J.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">M.H.</div></div>, et al. </div><div class="article-title">Simple sequence repeat-based consensus linkage map of <em>Bombyx mori</em></div>, <div class="source">Proc. Nat. Acad. Sci. USA</div> , <div class="year">2005</div>, vol. <div class="volume">102</div> (pg. <div class="fpage">16303</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Simple%20sequence%20repeat-based%20consensus%20linkage%20map%20of%20Bombyx%20mori&amp;author=X.X.%20Miao&amp;author=S.J.%20Xu&amp;author=M.H.%20Li&amp;publication_year=2005&amp;journal=Proc.%20Nat.%20Acad.%20Sci.%20USA&amp;volume=102&amp;pages=16303-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0507794102" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0507794102" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0507794102"> </span></div> </div></div></div></div></div><div content-id="DSS030C58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS030C58" class="ref-content "><div class="label -label">58</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schafer</div> <div class="given-names">M.A.</div></div>,  <div class="name"><div class="surname">Mazzi</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Klappert</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">A microsatellite linkage map for <em>Drosophila montana</em> shows large variation in recombination rates, and a courtship song trait maps to an area of low recombination</div>, <div class="source">J. Evol. Biol.</div> , <div class="year">2010</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">518</div>-<div class="lpage">27</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20microsatellite%20linkage%20map%20for%20Drosophila%20montana%20shows%20large%20variation%20in%20recombination%20rates%2C%20and%20a%20courtship%20song%20trait%20maps%20to%20an%20area%20of%20low%20recombination&amp;author=M.A.%20Schafer&amp;author=D.%20Mazzi&amp;author=K.%20Klappert&amp;publication_year=2010&amp;journal=J.%20Evol.%20Biol.&amp;volume=23&amp;pages=518-27" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/jeb.2010.23.issue-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fjeb.2010.23.issue-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fjeb.2010.23.issue-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20040000" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS030C59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS030C59" class="ref-content "><div class="label -label">59</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Solignac</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Vautrin</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Baudry</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Mougel</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Loiseau</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Cornuet</div> <div class="given-names">J.M.</div></div>. </div><div class="article-title">A microsatellite-based linkage map of the honeybee, <em>Apis mellifera</em> L.</div>, <div class="source">Genetics</div> , <div class="year">2004</div>, vol. <div class="volume">167</div> (pg. <div class="fpage">253</div>-<div class="lpage">62</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20microsatellite-based%20linkage%20map%20of%20the%20honeybee%2C%20Apis%20mellifera%20L.&amp;author=M.%20Solignac&amp;author=D.%20Vautrin&amp;author=E.%20Baudry&amp;author=F.%20Mougel&amp;author=A.%20Loiseau&amp;author=J.M.%20Cornuet&amp;publication_year=2004&amp;journal=Genetics&amp;volume=167&amp;pages=253-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.167.1.253" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.167.1.253" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.167.1.253"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15166152" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS030C60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS030C60" class="ref-content "><div class="label -label">60</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhong</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Pai</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Yan</div> <div class="given-names">G.</div></div>. </div><div class="article-title">AFLP-based genetic linkage map for the red flour beetle (<em>Tribolium castaneum</em>)</div>, <div class="source">J. Heredity</div> , <div class="year">2004</div>, vol. <div class="volume">95</div> (pg. <div class="fpage">53</div>-<div class="lpage">61</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=AFLP-based%20genetic%20linkage%20map%20for%20the%20red%20flour%20beetle%20(Tribolium%20castaneum)&amp;author=D.%20Zhong&amp;author=A.%20Pai&amp;author=G.%20Yan&amp;publication_year=2004&amp;journal=J.%20Heredity&amp;volume=95&amp;pages=53-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jhered/esh012" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjhered%2Fesh012" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjhered%2Fesh012"> </span></div> </div></div></div></div></div><div content-id="DSS030C61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS030C61" class="ref-content "><div class="label -label">61</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Myint</div> <div class="given-names">K.K.M.</div></div>,  <div class="name"><div class="surname">Yasui</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Takagi</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Matsumura</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Virulence of long-term laboratory populations of the brown planthopper, <em>Nilaparvata lugens</em> (Stål), and whitebacked planthopper, <em>Sogatella furcifera</em> (Horváth) (Homoptera: Delphacidae), on rice differential varieties</div>, <div class="source">Appl. Entomol. Zool.</div> , <div class="year">2009</div>, vol. <div class="volume">44</div> (pg. <div class="fpage">149</div>-<div class="lpage">53</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Virulence%20of%20long-term%20laboratory%20populations%20of%20the%20brown%20planthopper%2C%20Nilaparvata%20lugens%20(St%C3%A5l)%2C%20and%20whitebacked%20planthopper%2C%20Sogatella%20furcifera%20(Horv%C3%A1th)%20(Homoptera%3A%20Delphacidae)%2C%20on%20rice%20differential%20varieties&amp;author=K.K.M.%20Myint&amp;author=H.%20Yasui&amp;author=M.%20Takagi&amp;author=M.%20Matsumura&amp;publication_year=2009&amp;journal=Appl.%20Entomol.%20Zool.&amp;volume=44&amp;pages=149-53" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1303/aez.2009.149" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1303%2Faez.2009.149" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1303%2Faez.2009.149"> </span></div> </div></div></div></div></div><div content-id="DSS030C62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS030C62" class="ref-content "><div class="label -label">62</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yamamoto</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Narukawa</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Kadono-Okuda</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">Construction of a single nucleotide polymorphism linkage map for the silkworm, <em>Bombyx mori</em>, based on bacterial artificial chromosome end sequences</div>, <div class="source">Genetics</div> , <div class="year">2006</div>, vol. <div class="volume">173</div> (pg. <div class="fpage">151</div>-<div class="lpage">61</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20a%20single%20nucleotide%20polymorphism%20linkage%20map%20for%20the%20silkworm%2C%20Bombyx%20mori%2C%20based%20on%20bacterial%20artificial%20chromosome%20end%20sequences&amp;author=K.%20Yamamoto&amp;author=J.%20Narukawa&amp;author=K.%20Kadono-Okuda&amp;publication_year=2006&amp;journal=Genetics&amp;volume=173&amp;pages=151-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.105.053801" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.105.053801" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.105.053801"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16547112" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS030C63" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS030C63" class="ref-content "><div class="label -label">63</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kobayashi</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Noda</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Identification of Y chromosomal PCR marker and production of a selected strain for molecular sexing in the brown planthopper, <em>Nilaparvata lugens</em></div>, <div class="source">Arch. Insect Biochem. Physiol.</div> , <div class="year">2007</div>, vol. <div class="volume">65</div> (pg. <div class="fpage">1</div>-<div class="lpage">10</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20Y%20chromosomal%20PCR%20marker%20and%20production%20of%20a%20selected%20strain%20for%20molecular%20sexing%20in%20the%20brown%20planthopper%2C%20Nilaparvata%20lugens&amp;author=T.%20Kobayashi&amp;author=H.%20Noda&amp;publication_year=2007&amp;journal=Arch.%20Insect%20Biochem.%20Physiol.&amp;volume=65&amp;pages=1-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1520-6327" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1520-6327" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1520-6327"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17427932" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS030C64" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS030C64" class="ref-content "><div class="label -label">64</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Barnett</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Pachl</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Gergen</div> <div class="given-names">J.P.</div></div>,  <div class="name"><div class="surname">Wensink</div> <div class="given-names">P.C.</div></div>. </div><div class="article-title">The isolation and characterization of <em>Drosophila</em> yolk protein genes</div>, <div class="source">Cell</div> , <div class="year">1980</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">729</div>-<div class="lpage">38</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20isolation%20and%20characterization%20of%20Drosophila%20yolk%20protein%20genes&amp;author=T.%20Barnett&amp;author=C.%20Pachl&amp;author=J.P.%20Gergen&amp;author=P.C.%20Wensink&amp;publication_year=1980&amp;journal=Cell&amp;volume=21&amp;pages=729-38" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0092-8674(80)90436-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0092-8674(80)90436-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0092-8674(80)90436-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6777047" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS030C65" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS030C65" class="ref-content "><div class="label -label">65</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Higgins</div> <div class="given-names">M.J.</div></div>,  <div class="name"><div class="surname">Walker</div> <div class="given-names">V.K.</div></div>,  <div class="name"><div class="surname">Holden</div> <div class="given-names">J.A.</div></div>,  <div class="name"><div class="surname">White</div> <div class="given-names">B.N.</div></div>. </div><div class="article-title">Isolation of two <em>Drosophila melanogaster</em> genes abundantly expressed in the ovary during vitelline membrane synthesis</div>, <div class="source">Dev. Biol.</div> , <div class="year">1984</div>, vol. <div class="volume">105</div> (pg. <div class="fpage">155</div>-<div class="lpage">65</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Isolation%20of%20two%20Drosophila%20melanogaster%20genes%20abundantly%20expressed%20in%20the%20ovary%20during%20vitelline%20membrane%20synthesis&amp;author=M.J.%20Higgins&amp;author=V.K.%20Walker&amp;author=J.A.%20Holden&amp;author=B.N.%20White&amp;publication_year=1984&amp;journal=Dev.%20Biol.&amp;volume=105&amp;pages=155-65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0012-1606(84)90271-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0012-1606(84)90271-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0012-1606(84)90271-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6088330" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS030C66" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS030C66" class="ref-content "><div class="label -label">66</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Riddell</div> <div class="given-names">D.C.</div></div>,  <div class="name"><div class="surname">Higgins</div> <div class="given-names">M.J.</div></div>,  <div class="name"><div class="surname">McMillan</div> <div class="given-names">B.J.</div></div>,  <div class="name"><div class="surname">White</div> <div class="given-names">B.N.</div></div>. </div><div class="article-title">Structural analysis of the three vitellogenin genes in <em>Drosophila melanogaster</em></div>, <div class="source">Nucl. Acids Res.</div> , <div class="year">1981</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">1323</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structural%20analysis%20of%20the%20three%20vitellogenin%20genes%20in%20Drosophila%20melanogaster&amp;author=D.C.%20Riddell&amp;author=M.J.%20Higgins&amp;author=B.J.%20McMillan&amp;author=B.N.%20White&amp;publication_year=1981&amp;journal=Nucl.%20Acids%20Res.&amp;volume=9&amp;pages=1323-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/9.6.1323" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F9.6.1323" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F9.6.1323"> </span></div> </div></div></div></div></div><div content-id="DSS030C67" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS030C67" class="ref-content "><div class="label -label">67</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bradfield</div> <div class="given-names">J.Y.</div></div>,  <div class="name"><div class="surname">Wyatt</div> <div class="given-names">G.R.</div></div>. </div><div class="article-title">X-linkage of a vitellogenin gene in <em>Locusta migratoria</em></div>, <div class="source">Chromosoma</div> , <div class="year">1983</div>, vol. <div class="volume">88</div> (pg. <div class="fpage">190</div>-<div class="lpage">3</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=X-linkage%20of%20a%20vitellogenin%20gene%20in%20Locusta%20migratoria&amp;author=J.Y.%20Bradfield&amp;author=G.R.%20Wyatt&amp;publication_year=1983&amp;journal=Chromosoma&amp;volume=88&amp;pages=190-3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00285619" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00285619" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00285619"> </span></div> </div></div></div></div></div><div content-id="DSS030C68" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS030C68" class="ref-content "><div class="label -label">68</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Heine</div> <div class="given-names">U.</div></div>,  <div class="name"><div class="surname">Blumenthal</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Characterization of regions of the <em>Caenorhabditis elegans</em> X chromosome containing vitellogenin genes</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">1986</div>, vol. <div class="volume">188</div> (pg. <div class="fpage">301</div>-<div class="lpage">12</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20regions%20of%20the%20Caenorhabditis%20elegans%20X%20chromosome%20containing%20vitellogenin%20genes&amp;author=U.%20Heine&amp;author=T.%20Blumenthal&amp;publication_year=1986&amp;journal=J.%20Mol.%20Biol.&amp;volume=188&amp;pages=301-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0022-2836(86)90156-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0022-2836(86)90156-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0022-2836(86)90156-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3735423" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div><div content-id=""><div class="fn"><div class="footnote-content">Edited by Dr Satoshi Tabata</div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2012. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/3.0/), which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=382150&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss030/2/dss030-dss030supp_Table7.xlsx?Expires=2147483647&amp;Signature=CzHUGHUZHSCkKefI60u19pB1AveHurOcmotqtH0bHxaI6VA7A26aaLfa4hBWSCczFhdvK9pbaZJqyQMDETUVEs0YRY0MP2Ez4l~FRELZdO1Pky938~dUpi-XdJeADR-UV-AGDLx-MOlWu9CjXIbWigGal0mi8bk2A8KRX9wk28-jM4bOhN2q9MPqVEUBFhUrsk7MwHJ1~OWiuc4uY9w6hRCUTu6DGE9arqH-QGWe8B4C0A1FVBnkjKvs3LHlJylfRnURSNcU9ZIiQlKOAf1dI0ZFBMKzd4TluC~13s-RYpiv0KlMuYNXWq20oc6E08FGkZpIxCgFhe6Y1gT9P9HJkQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-8</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss030/2/dss030-dss030supp_Table6.xlsx?Expires=2147483647&amp;Signature=YIkbvj-JMIHqaHg-FF4thgmKy1iNZHoN9H-3QdYiRnrcWO0PP0TvyL5Se~zWHfciSpNFM9wZGnhLBlMh20pODz2UUij3y85gYiYQf7bqsj9jK9ikxh3CdPZzCRJ350ukqf3BkjuC~3KkM4aT4jGIrwyIHR1VSqJINf-Mw0SiC0yWZDsVha4nFFc6WnTzIGHQPs2O-7gK1H032wL8pGnl0fhQB30SU3Ckx1L-Qc3NLfoEMk4lCbyLW3Q0MK97MI7dOZdq4Rz96yyjy0D~N~i7ErErdnZ60Emi27v02kvkT81YlNHSbmJBeebcg4xlRcbhWtef2fhS2dElwbhl36W7zg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-7</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss030/2/dss030-dss030supp_Table5.xlsx?Expires=2147483647&amp;Signature=l2quPjiWFB9a9KiTo4srB097s~575KZDZANOMmgio~uH9Bk2aPno1wR09rzcNDpVmHkPkon82GE~gek5f6M5eAjzdzINrAnriEf9mM2lmRFicK5ERruzasau~L9pFKiI0wyCIUZNECu5pvTWoXiu~5boUTkmnPwPtlSZmKQUICI52a8uM3YUkoGCukM8eVranPFSBvORyjSdCy883puio6Hhn3mneeXnIJAZaNCmIeBDr5DgeTraJ6Z8zBRVH3t03xHyacj~q1LNKkAHfSVS1r0SG2ath9RSzE-UMcoowpaleHGubooVY-BLp9kW1Lm5wvmkZjDS7QgFl-P8GomY7w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-6</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss030/2/dss030-dss030supp_Table4.xlsx?Expires=2147483647&amp;Signature=2UF4MnjskO00vAKyf-BcFcQ9teWEiKxh5Z9rSzBneXIFgz9lQCjOGJ0wYHquWNEeggeQCQ61cbI88C3Re~nYZTIQ-A1DjqYotZrVuypReU6DBxqhQ3ZJ2BxJV-3LXM6uEwE2c41r2sgW4y7km7OUmfLYSDMTiaQi6CjzOMo9GSDFP~76Wm-sJRyGo1Cn24gzg-FEgpg7YTNsf6g1T7VES-FqnYp2LqL~1FqlyGWpACVHbzXQW3xmdPJyC0BjubRspC4KwPn~kmq8x26~i9S9m5zbpSWFeHQs06ioZJNd5N63rNHETYFpj77FzfqUIS8vqXM76Fl2V~hB3sm9dWc-Vw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-5</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss030/2/dss030-dss030supp_Table3.xlsx?Expires=2147483647&amp;Signature=RTFTANtUEDj5JgFOyxAFb0lkbcEExrPl33v6bsF8zU2wP~mi9BrxETr4bihu0xUzApPA0sJcQFGSd6TTZd~9vFj0M1AJHgzNZkQNgr0230Yb4Dqyi9Pt12FZX42IPowepttlt1HXk2U3fp~fDmSGdpMdD6Qk02D2Utlk9FpX4yU6HIZ13DsYwFaUI5bG5cNQcmGtR5bsfj2SDX3mNgFbswgJ4mmUaPxVgjAahNA9k9NUQZblMz~CgFV-zsu-pr-7loTx0pD3lqq9tw5PQIBH13eW5w1dDX7SqzPNmm~VH8vDfTOmUUC~d0ORLBXgns6GNQg0~m~kzcseicUXgEOAMA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-4</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss030/2/dss030-dss030supp_Table2.xlsx?Expires=2147483647&amp;Signature=qKmkAkRYWyuSMtNxvW8rObFS-FRN4uj2snadmVtbVK2NaZRzAB36DICvN0MDJwA2cwTxvbQUTfgZCWDCAnhiQa62M6iNxwx48SDJ9NbKo2XUAeDQ6t881oFgRlmShBb1K2S4maDHb9Y6R9eL9PAIgpqRcK1jMJzi7GE7UF2KJHwF-VFDUQB6Iv1o15Il9tkVEzu3iq5e7bxjLfP-AlJ7fnX04CQJzLPeZjFwoR0tlV8eVFXSIdYCGI2yFZu4tMSy86cqgln1yb9-b4oh3kL57jNtbrAdy2DyvCkS7ZlTK0xUcBsM5sUQ7iM8RVhCvau1z-pxFjwq2gMlUwUrp5KVoQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss030/2/dss030-dss030supp_Table1.xlsx?Expires=2147483647&amp;Signature=AsCye42UqpiCjEdtoR~N5Y2PVWjS4SDnuLYjs~FPe1SU6LodtU00qZouseUVZkQZ5grzPT25V-~0hsBawyUX9~qwCT9L0FA4MpuOaoB2BFQ23NEfQCE2kl7jxr~343SD8iIZsnLByoxhOtdY0Jn~wNr-Wf8pKxncCrUhz7J0lr9AocsGjEG2HUmFSyYPuDmX6n8ZrHINZf3puv5coa-elGpn39kA0A-mZwiUh2-Izr0qWKmc6zF6YEt6Ov7-LuvwW8B6g0qQpyB8xPT-aq4UBMABMlLxoHypAabfl8ALEqN66PqEYawQN7BQnxCErf7K7~Te7tkB~vze1QUMkQB5pA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss030/2/dss030-dss030supp_fig.ppt?Expires=2147483647&amp;Signature=t9mDRGmiC4IuXxgxXpfPtP2Ek0L1SCsmPs2wYt6XqSwW8CF5~M2e0zauFXyOxYTRdtPEarXchv3BDJxNBZKnAMfMhTg6kjGCPFgHl1FVDR9jI~vyiZUTpDwmQt1rxURG~~qbjX3dKvmqbcT2dCw9b7GEJtcpR22sQd8VLvcfnNv7QA5cJWW6d1Caf4hn0WBYii6d1Rz8k29sw0IJcTwwFOKF5wAlun7JlFqouenu6lp9nuVOrlmPQz7ISDyyngKrocNcTp0E~XT5A97myBF3q5EJGZEn-yGK~pGCcQPi6PAGIMdp30cjc8TsrLoJTxGOH~x-tcXvWBWwx0n5XGg8XQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - ppt file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">288</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000315666200002&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">10</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dss030"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="382150"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000315666200002&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f20%2f1%2f17%2f382150">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000315666200002&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (10)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F20%2F1%2F17%2F382150" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/382150'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Simple Sequence Repeat- and Single-Nucleotide Polymorphism-Based Genetic Linkage Map of the Brown Planthopper, Nilaparvata lugens" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dss030");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f20%2f1%2f17%2f382150">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

