

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/18/2/93/365332","@type":"ScholarlyArticle","name":"The Complete Chloroplast Genome of 17 Individuals of Pest Species  Jacobaea vulgaris : SNPs, Microsatellites and Barcoding Markers for Population and Phylogenetic Studies","datePublished":"2011-03-28","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/18/2","@type":"PublicationIssue","issueNumber":"2","datePublished":"2011-04-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsr002","keywords":["phylogenetic markers","<italic>Jacobaea vulgaris</italic>","Asteraceae","cpDNA polymorphisms","inverted repeat"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/18/2/93/365332","author":[{"name":"Doorduin, Leonie","affiliation":"1  Institute of Biology, Section Plant Ecology and Phytochemistry ,  Leiden University ,  PO Box 9505, 2300 RA Leiden ,  The Netherlands","@type":"Person"},{"name":"Gravendeel, Barbara","affiliation":"1  Institute of Biology, Section Plant Ecology and Phytochemistry ,  Leiden University ,  PO Box 9505, 2300 RA Leiden ,  The Netherlands    2  NCB Naturalis - NHN Leiden University ,  Einsteinweg 2, 2333 CC Leiden ,  The Netherlands","@type":"Person"},{"name":"Lammers, Youri","affiliation":"1  Institute of Biology, Section Plant Ecology and Phytochemistry ,  Leiden University ,  PO Box 9505, 2300 RA Leiden ,  The Netherlands","@type":"Person"},{"name":"Ariyurek, Yavuz","affiliation":"3  Leiden Genome Technology Center, Human and Clinical Genetics ,  Leiden University Medical Center ,  PO box 9600, 2300 RC Leiden ,  The Netherlands","@type":"Person"},{"name":"Chin-A-Woeng, Thomas","affiliation":"4  ServiceXS ,  Plesmanlaan 1d, 2333 BZ Leiden ,  The Netherlands","@type":"Person"},{"name":"Vrieling, Klaas","affiliation":"1  Institute of Biology, Section Plant Ecology and Phytochemistry ,  Leiden University ,  PO Box 9505, 2300 RA Leiden ,  The Netherlands","@type":"Person"}],"description":"Abstract.  Invasive individuals from the pest species Jacobaea vulgaris show different allocation patterns in defence and growth compared with native individual","pageStart":"93","pageEnd":"105","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/18/2/1/m_cover.gif?Expires=2147483647&Signature=zPrik-vQsfP8tHq7QCbk4-ZAM~7ALug60QU9E2a0HTM3jWpw4mhYvM5ZjIfk8gvh0~D~8ZHzMVsbLbVV9I~imi38eK1-WbEZj8UJvCZsVN9ocsj3MzknfjxMwhJRMN7OhfXTufW~YjTAGZRrjum43uomBF~AyUhUCSuUuosVsuEI7xRAk7wYC1Ni9rgonZdcOzQdfiuitUvT5RG~wLQZYhdUWA83IS9zyFUehXeSL1v2Ik301pcwodAx4daAPn-F6iCk6UwJJGptSyDThKiltCKPrcoMzV3V1WqixkUPlkjoJdqdq8RIeTplSpeuudNseKNAwRJXHEN70rQsx2BmAw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"The Complete Chloroplast Genome of 17 Individuals of Pest Species  Jacobaea vulgaris : SNPs, Microsatellites and Barcoding Markers for Population and Phylogenetic Studies","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/18/2/1/m_cover.gif?Expires=2147483647&Signature=zPrik-vQsfP8tHq7QCbk4-ZAM~7ALug60QU9E2a0HTM3jWpw4mhYvM5ZjIfk8gvh0~D~8ZHzMVsbLbVV9I~imi38eK1-WbEZj8UJvCZsVN9ocsj3MzknfjxMwhJRMN7OhfXTufW~YjTAGZRrjum43uomBF~AyUhUCSuUuosVsuEI7xRAk7wYC1Ni9rgonZdcOzQdfiuitUvT5RG~wLQZYhdUWA83IS9zyFUehXeSL1v2Ik301pcwodAx4daAPn-F6iCk6UwJJGptSyDThKiltCKPrcoMzV3V1WqixkUPlkjoJdqdq8RIeTplSpeuudNseKNAwRJXHEN70rQsx2BmAw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Complete Chloroplast Genome of 17 Individuals of Pest Species  Jacobaea vulgaris : SNPs, Microsatellites and Barcoding Markers for Population and Phylogenetic Studies" />
    <meta property="og:description" content="Abstract.  Invasive individuals from the pest species Jacobaea vulgaris show different allocation patterns in defence and growth compared with native individual" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/18/2/93/365332" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2011-03-28" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Doorduin, Leonie" /><meta name="citation_author_institution" content="Institute of Biology, Section Plant Ecology and Phytochemistry, Leiden University, PO Box 9505, 2300 RA Leiden, The Netherlands" /><meta name="citation_author" content="Gravendeel, Barbara" /><meta name="citation_author_institution" content="Institute of Biology, Section Plant Ecology and Phytochemistry, Leiden University, PO Box 9505, 2300 RA Leiden, The Netherlands" /><meta name="citation_author_institution" content="NCB Naturalis - NHN Leiden University, Einsteinweg 2, 2333 CC Leiden, The Netherlands" /><meta name="citation_author" content="Lammers, Youri" /><meta name="citation_author_institution" content="Institute of Biology, Section Plant Ecology and Phytochemistry, Leiden University, PO Box 9505, 2300 RA Leiden, The Netherlands" /><meta name="citation_author" content="Ariyurek, Yavuz" /><meta name="citation_author_institution" content="Leiden Genome Technology Center, Human and Clinical Genetics, Leiden University Medical Center, PO box 9600, 2300 RC Leiden, The Netherlands" /><meta name="citation_author" content="Chin-A-Woeng, Thomas" /><meta name="citation_author_institution" content="ServiceXS, Plesmanlaan 1d, 2333 BZ Leiden, The Netherlands" /><meta name="citation_author" content="Vrieling, Klaas" /><meta name="citation_author_institution" content="Institute of Biology, Section Plant Ecology and Phytochemistry, Leiden University, PO Box 9505, 2300 RA Leiden, The Netherlands" /><meta name="citation_title" content="The Complete Chloroplast Genome of 17 Individuals of Pest Species Jacobaea vulgaris: SNPs, Microsatellites and Barcoding Markers for Population and Phylogenetic Studies" /><meta name="citation_firstpage" content="93" /><meta name="citation_lastpage" content="105" /><meta name="citation_doi" content="10.1093/dnares/dsr002" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="18" /><meta name="citation_issue" content="2" /><meta name="citation_publication_date" content="2011/04/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=A phylogenetic study of the order Asterales using one morphological and three molecular data sets; Int. J. Plant Sci.;  citation_year=2003;  citation_volume=164;  citation_pages=553-78; " /><meta name="citation_reference" content="citation_title=Tribal interrelationships and phylogeny of the Asteraceae; Aliso;  citation_year=1976;  citation_volume=8;  citation_pages=465-92; " /><meta name="citation_reference" content="citation_title=Evolutionary origins and natural functions of fructans—a climatological, biogeographic and mechanistic appraisal; New Phytologist.;  citation_year=1993;  citation_volume=123;  citation_pages=3-14; " /><meta name="citation_reference" content="citation_title=Multiple paleopolyploidizations during the evolution of the Compositae reveal parallel patterns of duplicate gene retention after millions of years; Mol. Biol. Evol.;  citation_year=2008;  citation_volume=25;  citation_pages=2445-55; " /><meta name="citation_reference" content="citation_title=Two chloroplast DNA inversions originated simultaneously during the early evolution of the sunflower family (Asteraceae); Mol. Biol. Evol.;  citation_year=2005;  citation_volume=22;  citation_pages=1783-92; " /><meta name="citation_reference" content="citation_publisher=CAB International: Wallingford, UK; Biological Control of Weeds: A World Catalogue of Agents and their Target Weeds;  citation_year=1987; " /><meta name="citation_reference" content="citation_title=Ecological effects of the invasive weed species Senecio jacobaea L (ragwort) in a New Zealand pasture; Agric. Ecosyst. Environ.;  citation_year=1995;  citation_volume=56;  citation_pages=19-28; " /><meta name="citation_reference" content="citation_title=The value of sampling anomalous taxa in phylogenetic studies: major clades of the Asteraceae revealed; Mol. Phylogenet. Evol.;  citation_year=2008;  citation_volume=47;  citation_pages=757-82; " /><meta name="citation_reference" content="citation_title=An ITS phylogeny of tribe Senecioneae (Asteraceae) and a new delimitation of Senecio L; Taxon;  citation_year=2007;  citation_volume=56;  citation_pages=1077-104; " /><meta name="citation_reference" content="citation_title=Patterns and causes of incongruence between plastid and nuclear Senecioneae (Asteraceae) phylogenies; Am. J. Bot.;  citation_year=2010;  citation_volume=97;  citation_pages=856-73; " /><meta name="citation_reference" content="citation_title=Biology of Canadian weeds 96 Senecio jacobaea L; Can. J. Plant Sci.;  citation_year=1991;  citation_volume=71;  citation_pages=127-40; " /><meta name="citation_reference" content="citation_title=Botanical aspects of ragwort (Senecio jacobaea L.) control; Bulletin of the New Zealand Dept. of Sci. and Indust. Research;  citation_year=1940;  citation_volume=82;  citation_pages=2-61; " /><meta name="citation_reference" content="citation_title=Biology and control of ragwort, Senecio jacobaea L, in Victoria, Australia; Weed Res.;  citation_year=1972;  citation_volume=12;  citation_pages=37-45; " /><meta name="citation_reference" content="citation_publisher=Oregon State College, Corvallis; Weeds of the Pacific Northwest;  citation_year=1957; " /><meta name="citation_reference" content="citation_title=Chemotypes of 2 pyrrolizidine alkaloid-containing Senecio species; Phytochemistry;  citation_year=1992;  citation_volume=31;  citation_pages=559-65; " /><meta name="citation_reference" content="citation_title=Population ecology of Senecio jacobaea in a sand dune system 1. Reproductive strategy and the biennial habit; Journal of Ecology;  citation_year=1979;  citation_volume=67;  citation_pages=131-53; " /><meta name="citation_reference" content="citation_publisher=Department of Natural Resources and Environment, Melbourne; The Ragwort Management Handbook;  citation_year=1997; " /><meta name="citation_reference" content="citation_title=The enemy release and EICA hypothesis revisited: incorporating the fundamental difference between specialist and generalist herbivores; Ecol. Lett.;  citation_year=2005;  citation_volume=8;  citation_pages=704-14; " /><meta name="citation_reference" content="citation_title=History, chance and adaptation during biological invasion: separating stochastic phenotypic evolution from response to selection; Ecol. Lett.;  citation_year=2008;  citation_volume=11;  citation_pages=852-66; " /><meta name="citation_reference" content="citation_title=The lack of genetic bottleneck in invasive Tansy ragwort populations suggests multiple source populations; Basic Appl. Ecol.;  citation_year=2010;  citation_volume=11;  citation_pages=244-50; " /><meta name="citation_reference" content="citation_title=Blood group O alleles innative Americans: implications in the peopling of the Americas; Am. J. Phys. Anthropol.;  citation_year=2010;  citation_volume=142;  citation_pages=85-94; " /><meta name="citation_reference" content="citation_title=Geographical distribution, genetic diversity and social organization of a new European termite, Reticulitermes urbis (Isoptera: Rhinotermitidae); Biol. Invasions;  citation_year=2010;  citation_volume=12;  citation_pages=1389-402; " /><meta name="citation_reference" content="citation_title=Highly reduced genetic diversity of Rosa rubiginosa L. populations in the invasive range; Int. J. Plant Sci.;  citation_year=2010;  citation_volume=171;  citation_pages=435-46; " /><meta name="citation_reference" content="citation_title=Next-generation DNA sequencing; Nat. Biotechnol.;  citation_year=2008;  citation_volume=26;  citation_pages=1135-45; " /><meta name="citation_reference" content="citation_title=Rates of nucleotide substitution vary greatly among plant mitochondrial, chloroplast, and nuclear DNAs; Proc. Natl Acad. Sci. USA;  citation_year=1987;  citation_volume=84;  citation_pages=9054-8; " /><meta name="citation_reference" content="citation_title=Complete chloroplast genome sequence of a major allogamous forage species, perennial ryegrass (Lolium perenne L.); DNA Res.;  citation_year=2009;  citation_volume=16;  citation_pages=165-76; " /><meta name="citation_reference" content="citation_title=Comparative analysis of the complete sequence of the plastid genome of Parthenium argentatum and identification of DNA barcodes to differentiate Parthenium species and lines; BMC Plant Biol.;  citation_year=2009;  citation_volume=9; " /><meta name="citation_reference" content="citation_title=Complete chloroplast genome of Oncidium Gower Ramsey and evaluation of molecular markers for identification and breeding in Oncidiinae; BMC Plant Biol.;  citation_year=2010;  citation_volume=10; " /><meta name="citation_reference" content="citation_title=The spatial distribution of chloroplast DNA and allozyme polymorphisms within a population of Silene alba (Caryophyllaceae).; Am. J. Bot.;  citation_year=1996;  citation_volume=83;  citation_pages=727-31; " /><meta name="citation_reference" content="citation_title=Use of nuclear genes for phylogeny reconstruction in plants; Aust. Syst. Bot.;  citation_year=2004;  citation_volume=17;  citation_pages=145-70; " /><meta name="citation_reference" content="citation_title=A method for isolation of chloroplast DNA and mitochondrial DNA from sunflower; Plant Mol. Biol. Rep.;  citation_year=1998;  citation_volume=16;  citation_pages=183-9; " /><meta name="citation_reference" content="citation_title=A rapid DNA isolation procedure for small quantities of fresh leaf tissue; Phytochem. Bull.;  citation_year=1987;  citation_volume=19;  citation_pages=11-5; " /><meta name="citation_reference" content="citation_title=Primer3 on the WWW for general users and for biologist programmers; Methods Mol. Biol.;  citation_year=2000;  citation_volume=132;  citation_pages=365-86; " /><meta name="citation_reference" content="citation_title=Velvet: algorithms for de novo short read assembly using de Bruijn graphs; Genome Res.;  citation_year=2008;  citation_volume=18;  citation_pages=821-9; " /><meta name="citation_reference" content="citation_title=Versatile and open software for comparing large genomes; Genome Biol.;  citation_year=2004;  citation_volume=5; " /><meta name="citation_reference" content="citation_title=Automatic annotation of organellar genomes with DOGMA; Bioinformatics;  citation_year=2004;  citation_volume=20;  citation_pages=3252-5; " /><meta name="citation_reference" content="citation_title=GenomeVx: simple web-based creation of editable circular chromosome maps; Bioinformatics;  citation_year=2008;  citation_volume=24;  citation_pages=861-2; " /><meta name="citation_reference" content="citation_publisher=Sinauer Associates, Sunderland, MA, USA; MacClade Version 4.06 OSX;  citation_year=2003; " /><meta name="citation_reference" content=" citation_edition=10th edition; citation_publisher=Sinauer Associated, Sunderland; Phylogenetic Analysis Using Parsimony 4.0b;  citation_year=2002; " /><meta name="citation_reference" content="citation_publisher=University of Washington, ; PHYLIP Phylogenetic Inference Programs Version 3.572;  citation_year=1995; " /><meta name="citation_reference" content="citation_title=MapView: visualization of short reads alignment on a desktop computer; Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=1554-5; " /><meta name="citation_reference" content="citation_title=Multiplex sequencing of plant chloroplast genomes using Solexa sequencing-by-synthesis technology; Nucleic Acids Res.;  citation_year=2008;  citation_volume=36; " /><meta name="citation_reference" content="citation_title=A comparative analysis of the Lactuca and Helianthus (Asteraceae) plastid genomes: identification of divergent regions and categorization of shared repeats; Am. J. Bot.;  citation_year=2007;  citation_volume=94;  citation_pages=302-12; " /><meta name="citation_reference" content="citation_title=Establishing genomic tools and resources for Guizotia abyssinica; the development of a library of expressed sequence tags, microsatellite loci, and the sequencing of its chloroplast genome; Mol. Ecol. Resour.;  citation_year=2010;  citation_volume=10;  citation_pages=1048-58; " /><meta name="citation_reference" content="citation_title=Tortoise and the hare: choosing between noncoding plastome and nuclear ADH sequences for phylogeny reconstruction in a recently diverged plant group; Am. J. Bot.;  citation_year=1998;  citation_volume=85;  citation_pages=1301-15; " /><meta name="citation_reference" content="citation_title=Contrasting modes and tempos of genome evolution in land plant organelles; Trends Genet.;  citation_year=1990;  citation_volume=6;  citation_pages=115-20; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/18/2/93/1042957/dsr002.pdf" /><meta name="description" content="Abstract.  Invasive individuals from the pest species Jacobaea vulgaris show different allocation patterns in defence and growth compared with native individual" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/18/2/93/365332" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/18/2/93/365332" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/18/2/93/365332");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsr002';
    </script>
<title>
	Complete Chloroplast Genome of 17 Individuals of Pest Species Jacobaea vulgaris: SNPs, Microsatellites and Barcoding Markers for Population and Phylogenetic Studies | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./365332" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BY5oPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJEb29yZHVpbiwgTGVvbmllIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSW5zdGl0dXRlIG9mIEJpb2xvZ3ksIFNlY3Rpb24gUGxhbnQgRWNvbG9neSBhbmQgUGh5dG9jaGVtaXN0cnksIExlaWRlbiBVbml2ZXJzaXR5LCBQTyBCb3ggOTUwNSwgMjMwMCBSQSBMZWlkZW4sIFRoZSBOZXRoZXJsYW5kcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkdyYXZlbmRlZWwsIEJhcmJhcmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJJbnN0aXR1dGUgb2YgQmlvbG9neSwgU2VjdGlvbiBQbGFudCBFY29sb2d5IGFuZCBQaHl0b2NoZW1pc3RyeSwgTGVpZGVuIFVuaXZlcnNpdHksIFBPIEJveCA5NTA1LCAyMzAwIFJBIExlaWRlbiwgVGhlIE5ldGhlcmxhbmRzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTkNCIE5hdHVyYWxpcyAtIE5ITiBMZWlkZW4gVW5pdmVyc2l0eSwgRWluc3RlaW53ZWcgMiwgMjMzMyBDQyBMZWlkZW4sIFRoZSBOZXRoZXJsYW5kcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkxhbW1lcnMsIFlvdXJpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSW5zdGl0dXRlIG9mIEJpb2xvZ3ksIFNlY3Rpb24gUGxhbnQgRWNvbG9neSBhbmQgUGh5dG9jaGVtaXN0cnksIExlaWRlbiBVbml2ZXJzaXR5LCBQTyBCb3ggOTUwNSwgMjMwMCBSQSBMZWlkZW4sIFRoZSBOZXRoZXJsYW5kcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkFyaXl1cmVrLCBZYXZ1eiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkxlaWRlbiBHZW5vbWUgVGVjaG5vbG9neSBDZW50ZXIsIEh1bWFuIGFuZCBDbGluaWNhbCBHZW5ldGljcywgTGVpZGVuIFVuaXZlcnNpdHkgTWVkaWNhbCBDZW50ZXIsIFBPIGJveCA5NjAwLCAyMzAwIFJDIExlaWRlbiwgVGhlIE5ldGhlcmxhbmRzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQ2hpbi1BLVdvZW5nLCBUaG9tYXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJTZXJ2aWNlWFMsIFBsZXNtYW5sYWFuIDFkLCAyMzMzIEJaIExlaWRlbiwgVGhlIE5ldGhlcmxhbmRzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iVnJpZWxpbmcsIEtsYWFzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSW5zdGl0dXRlIG9mIEJpb2xvZ3ksIFNlY3Rpb24gUGxhbnQgRWNvbG9neSBhbmQgUGh5dG9jaGVtaXN0cnksIExlaWRlbiBVbml2ZXJzaXR5LCBQTyBCb3ggOTUwNSwgMjMwMCBSQSBMZWlkZW4sIFRoZSBOZXRoZXJsYW5kcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iVGhlIENvbXBsZXRlIENobG9yb3BsYXN0IEdlbm9tZSBvZiAxNyBJbmRpdmlkdWFscyBvZiBQZXN0IFNwZWNpZXMgSmFjb2JhZWEgdnVsZ2FyaXM6IFNOUHMsIE1pY3Jvc2F0ZWxsaXRlcyBhbmQgQmFyY29kaW5nIE1hcmtlcnMgZm9yIFBvcHVsYXRpb24gYW5kIFBoeWxvZ2VuZXRpYyBTdHVkaWVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iOTMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjEwNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzcjAwMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIxOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDExLzA0LzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBwaHlsb2dlbmV0aWMgc3R1ZHkgb2YgdGhlIG9yZGVyIEFzdGVyYWxlcyB1c2luZyBvbmUgbW9ycGhvbG9naWNhbCBhbmQgdGhyZWUgbW9sZWN1bGFyIGRhdGEgc2V0czsgSW50LiBKLiBQbGFudCBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTE2NDsgIGNpdGF0aW9uX3BhZ2VzPTU1My03ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJpYmFsIGludGVycmVsYXRpb25zaGlwcyBhbmQgcGh5bG9nZW55IG9mIHRoZSBBc3RlcmFjZWFlOyBBbGlzbzsgIGNpdGF0aW9uX3llYXI9MTk3NjsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9NDY1LTkyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Fdm9sdXRpb25hcnkgb3JpZ2lucyBhbmQgbmF0dXJhbCBmdW5jdGlvbnMgb2YgZnJ1Y3RhbnPigJRhIGNsaW1hdG9sb2dpY2FsLCBiaW9nZW9ncmFwaGljIGFuZCBtZWNoYW5pc3RpYyBhcHByYWlzYWw7IE5ldyBQaHl0b2xvZ2lzdC47ICBjaXRhdGlvbl95ZWFyPTE5OTM7ICBjaXRhdGlvbl92b2x1bWU9MTIzOyAgY2l0YXRpb25fcGFnZXM9My0xNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TXVsdGlwbGUgcGFsZW9wb2x5cGxvaWRpemF0aW9ucyBkdXJpbmcgdGhlIGV2b2x1dGlvbiBvZiB0aGUgQ29tcG9zaXRhZSByZXZlYWwgcGFyYWxsZWwgcGF0dGVybnMgb2YgZHVwbGljYXRlIGdlbmUgcmV0ZW50aW9uIGFmdGVyIG1pbGxpb25zIG9mIHllYXJzOyBNb2wuIEJpb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9MjQ0NS01NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHdvIGNobG9yb3BsYXN0IEROQSBpbnZlcnNpb25zIG9yaWdpbmF0ZWQgc2ltdWx0YW5lb3VzbHkgZHVyaW5nIHRoZSBlYXJseSBldm9sdXRpb24gb2YgdGhlIHN1bmZsb3dlciBmYW1pbHkgKEFzdGVyYWNlYWUpOyBNb2wuIEJpb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTIyOyAgY2l0YXRpb25fcGFnZXM9MTc4My05MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fcHVibGlzaGVyPUNBQiBJbnRlcm5hdGlvbmFsOiBXYWxsaW5nZm9yZCwgVUs7IEJpb2xvZ2ljYWwgQ29udHJvbCBvZiBXZWVkczogQSBXb3JsZCBDYXRhbG9ndWUgb2YgQWdlbnRzIGFuZCB0aGVpciBUYXJnZXQgV2VlZHM7ICBjaXRhdGlvbl95ZWFyPTE5ODc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVjb2xvZ2ljYWwgZWZmZWN0cyBvZiB0aGUgaW52YXNpdmUgd2VlZCBzcGVjaWVzIFNlbmVjaW8gamFjb2JhZWEgTCAocmFnd29ydCkgaW4gYSBOZXcgWmVhbGFuZCBwYXN0dXJlOyBBZ3JpYy4gRWNvc3lzdC4gRW52aXJvbi47ICBjaXRhdGlvbl95ZWFyPTE5OTU7ICBjaXRhdGlvbl92b2x1bWU9NTY7ICBjaXRhdGlvbl9wYWdlcz0xOS0yODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHZhbHVlIG9mIHNhbXBsaW5nIGFub21hbG91cyB0YXhhIGluIHBoeWxvZ2VuZXRpYyBzdHVkaWVzOiBtYWpvciBjbGFkZXMgb2YgdGhlIEFzdGVyYWNlYWUgcmV2ZWFsZWQ7IE1vbC4gUGh5bG9nZW5ldC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9NDc7ICBjaXRhdGlvbl9wYWdlcz03NTctODI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuIElUUyBwaHlsb2dlbnkgb2YgdHJpYmUgU2VuZWNpb25lYWUgKEFzdGVyYWNlYWUpIGFuZCBhIG5ldyBkZWxpbWl0YXRpb24gb2YgU2VuZWNpbyBMOyBUYXhvbjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT01NjsgIGNpdGF0aW9uX3BhZ2VzPTEwNzctMTA0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QYXR0ZXJucyBhbmQgY2F1c2VzIG9mIGluY29uZ3J1ZW5jZSBiZXR3ZWVuIHBsYXN0aWQgYW5kIG51Y2xlYXIgU2VuZWNpb25lYWUgKEFzdGVyYWNlYWUpIHBoeWxvZ2VuaWVzOyBBbS4gSi4gQm90LjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT05NzsgIGNpdGF0aW9uX3BhZ2VzPTg1Ni03MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QmlvbG9neSBvZiBDYW5hZGlhbiB3ZWVkcyA5NiBTZW5lY2lvIGphY29iYWVhIEw7IENhbi4gSi4gUGxhbnQgU2NpLjsgIGNpdGF0aW9uX3llYXI9MTk5MTsgIGNpdGF0aW9uX3ZvbHVtZT03MTsgIGNpdGF0aW9uX3BhZ2VzPTEyNy00MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Qm90YW5pY2FsIGFzcGVjdHMgb2YgcmFnd29ydCAoU2VuZWNpbyBqYWNvYmFlYSBMLikgY29udHJvbDsgQnVsbGV0aW4gb2YgdGhlIE5ldyBaZWFsYW5kIERlcHQuIG9mIFNjaS4gYW5kIEluZHVzdC4gUmVzZWFyY2g7ICBjaXRhdGlvbl95ZWFyPTE5NDA7ICBjaXRhdGlvbl92b2x1bWU9ODI7ICBjaXRhdGlvbl9wYWdlcz0yLTYxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CaW9sb2d5IGFuZCBjb250cm9sIG9mIHJhZ3dvcnQsIFNlbmVjaW8gamFjb2JhZWEgTCwgaW4gVmljdG9yaWEsIEF1c3RyYWxpYTsgV2VlZCBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTcyOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9MzctNDU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3B1Ymxpc2hlcj1PcmVnb24gU3RhdGUgQ29sbGVnZSwgQ29ydmFsbGlzOyBXZWVkcyBvZiB0aGUgUGFjaWZpYyBOb3J0aHdlc3Q7ICBjaXRhdGlvbl95ZWFyPTE5NTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNoZW1vdHlwZXMgb2YgMiBweXJyb2xpemlkaW5lIGFsa2Fsb2lkLWNvbnRhaW5pbmcgU2VuZWNpbyBzcGVjaWVzOyBQaHl0b2NoZW1pc3RyeTsgIGNpdGF0aW9uX3llYXI9MTk5MjsgIGNpdGF0aW9uX3ZvbHVtZT0zMTsgIGNpdGF0aW9uX3BhZ2VzPTU1OS02NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UG9wdWxhdGlvbiBlY29sb2d5IG9mIFNlbmVjaW8gamFjb2JhZWEgaW4gYSBzYW5kIGR1bmUgc3lzdGVtIDEuIFJlcHJvZHVjdGl2ZSBzdHJhdGVneSBhbmQgdGhlIGJpZW5uaWFsIGhhYml0OyBKb3VybmFsIG9mIEVjb2xvZ3k7ICBjaXRhdGlvbl95ZWFyPTE5Nzk7ICBjaXRhdGlvbl92b2x1bWU9Njc7ICBjaXRhdGlvbl9wYWdlcz0xMzEtNTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3B1Ymxpc2hlcj1EZXBhcnRtZW50IG9mIE5hdHVyYWwgUmVzb3VyY2VzIGFuZCBFbnZpcm9ubWVudCwgTWVsYm91cm5lOyBUaGUgUmFnd29ydCBNYW5hZ2VtZW50IEhhbmRib29rOyAgY2l0YXRpb25feWVhcj0xOTk3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZW5lbXkgcmVsZWFzZSBhbmQgRUlDQSBoeXBvdGhlc2lzIHJldmlzaXRlZDogaW5jb3Jwb3JhdGluZyB0aGUgZnVuZGFtZW50YWwgZGlmZmVyZW5jZSBiZXR3ZWVuIHNwZWNpYWxpc3QgYW5kIGdlbmVyYWxpc3QgaGVyYml2b3JlczsgRWNvbC4gTGV0dC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPTcwNC0xNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGlzdG9yeSwgY2hhbmNlIGFuZCBhZGFwdGF0aW9uIGR1cmluZyBiaW9sb2dpY2FsIGludmFzaW9uOiBzZXBhcmF0aW5nIHN0b2NoYXN0aWMgcGhlbm90eXBpYyBldm9sdXRpb24gZnJvbSByZXNwb25zZSB0byBzZWxlY3Rpb247IEVjb2wuIExldHQuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9ODUyLTY2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgbGFjayBvZiBnZW5ldGljIGJvdHRsZW5lY2sgaW4gaW52YXNpdmUgVGFuc3kgcmFnd29ydCBwb3B1bGF0aW9ucyBzdWdnZXN0cyBtdWx0aXBsZSBzb3VyY2UgcG9wdWxhdGlvbnM7IEJhc2ljIEFwcGwuIEVjb2wuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9MjQ0LTUwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CbG9vZCBncm91cCBPIGFsbGVsZXMgaW5uYXRpdmUgQW1lcmljYW5zOiBpbXBsaWNhdGlvbnMgaW4gdGhlIHBlb3BsaW5nIG9mIHRoZSBBbWVyaWNhczsgQW0uIEouIFBoeXMuIEFudGhyb3BvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTQyOyAgY2l0YXRpb25fcGFnZXM9ODUtOTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlb2dyYXBoaWNhbCBkaXN0cmlidXRpb24sIGdlbmV0aWMgZGl2ZXJzaXR5IGFuZCBzb2NpYWwgb3JnYW5pemF0aW9uIG9mIGEgbmV3IEV1cm9wZWFuIHRlcm1pdGUsIFJldGljdWxpdGVybWVzIHVyYmlzIChJc29wdGVyYTogUmhpbm90ZXJtaXRpZGFlKTsgQmlvbC4gSW52YXNpb25zOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9MTM4OS00MDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhpZ2hseSByZWR1Y2VkIGdlbmV0aWMgZGl2ZXJzaXR5IG9mIFJvc2EgcnViaWdpbm9zYSBMLiBwb3B1bGF0aW9ucyBpbiB0aGUgaW52YXNpdmUgcmFuZ2U7IEludC4gSi4gUGxhbnQgU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xNzE7ICBjaXRhdGlvbl9wYWdlcz00MzUtNDY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5leHQtZ2VuZXJhdGlvbiBETkEgc2VxdWVuY2luZzsgTmF0LiBCaW90ZWNobm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0yNjsgIGNpdGF0aW9uX3BhZ2VzPTExMzUtNDU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJhdGVzIG9mIG51Y2xlb3RpZGUgc3Vic3RpdHV0aW9uIHZhcnkgZ3JlYXRseSBhbW9uZyBwbGFudCBtaXRvY2hvbmRyaWFsLCBjaGxvcm9wbGFzdCwgYW5kIG51Y2xlYXIgRE5BczsgUHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MTk4NzsgIGNpdGF0aW9uX3ZvbHVtZT04NDsgIGNpdGF0aW9uX3BhZ2VzPTkwNTQtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGxldGUgY2hsb3JvcGxhc3QgZ2Vub21lIHNlcXVlbmNlIG9mIGEgbWFqb3IgYWxsb2dhbW91cyBmb3JhZ2Ugc3BlY2llcywgcGVyZW5uaWFsIHJ5ZWdyYXNzIChMb2xpdW0gcGVyZW5uZSBMLik7IEROQSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9MTY1LTc2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wYXJhdGl2ZSBhbmFseXNpcyBvZiB0aGUgY29tcGxldGUgc2VxdWVuY2Ugb2YgdGhlIHBsYXN0aWQgZ2Vub21lIG9mIFBhcnRoZW5pdW0gYXJnZW50YXR1bSBhbmQgaWRlbnRpZmljYXRpb24gb2YgRE5BIGJhcmNvZGVzIHRvIGRpZmZlcmVudGlhdGUgUGFydGhlbml1bSBzcGVjaWVzIGFuZCBsaW5lczsgQk1DIFBsYW50IEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBsZXRlIGNobG9yb3BsYXN0IGdlbm9tZSBvZiBPbmNpZGl1bSBHb3dlciBSYW1zZXkgYW5kIGV2YWx1YXRpb24gb2YgbW9sZWN1bGFyIG1hcmtlcnMgZm9yIGlkZW50aWZpY2F0aW9uIGFuZCBicmVlZGluZyBpbiBPbmNpZGlpbmFlOyBCTUMgUGxhbnQgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBzcGF0aWFsIGRpc3RyaWJ1dGlvbiBvZiBjaGxvcm9wbGFzdCBETkEgYW5kIGFsbG96eW1lIHBvbHltb3JwaGlzbXMgd2l0aGluIGEgcG9wdWxhdGlvbiBvZiBTaWxlbmUgYWxiYSAoQ2FyeW9waHlsbGFjZWFlKS47IEFtLiBKLiBCb3QuOyAgY2l0YXRpb25feWVhcj0xOTk2OyAgY2l0YXRpb25fdm9sdW1lPTgzOyAgY2l0YXRpb25fcGFnZXM9NzI3LTMxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Vc2Ugb2YgbnVjbGVhciBnZW5lcyBmb3IgcGh5bG9nZW55IHJlY29uc3RydWN0aW9uIGluIHBsYW50czsgQXVzdC4gU3lzdC4gQm90LjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNzsgIGNpdGF0aW9uX3BhZ2VzPTE0NS03MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBtZXRob2QgZm9yIGlzb2xhdGlvbiBvZiBjaGxvcm9wbGFzdCBETkEgYW5kIG1pdG9jaG9uZHJpYWwgRE5BIGZyb20gc3VuZmxvd2VyOyBQbGFudCBNb2wuIEJpb2wuIFJlcC47ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz0xODMtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSByYXBpZCBETkEgaXNvbGF0aW9uIHByb2NlZHVyZSBmb3Igc21hbGwgcXVhbnRpdGllcyBvZiBmcmVzaCBsZWFmIHRpc3N1ZTsgUGh5dG9jaGVtLiBCdWxsLjsgIGNpdGF0aW9uX3llYXI9MTk4NzsgIGNpdGF0aW9uX3ZvbHVtZT0xOTsgIGNpdGF0aW9uX3BhZ2VzPTExLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByaW1lcjMgb24gdGhlIFdXVyBmb3IgZ2VuZXJhbCB1c2VycyBhbmQgZm9yIGJpb2xvZ2lzdCBwcm9ncmFtbWVyczsgTWV0aG9kcyBNb2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTEzMjsgIGNpdGF0aW9uX3BhZ2VzPTM2NS04NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VmVsdmV0OiBhbGdvcml0aG1zIGZvciBkZSBub3ZvIHNob3J0IHJlYWQgYXNzZW1ibHkgdXNpbmcgZGUgQnJ1aWpuIGdyYXBoczsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTg7ICBjaXRhdGlvbl9wYWdlcz04MjEtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VmVyc2F0aWxlIGFuZCBvcGVuIHNvZnR3YXJlIGZvciBjb21wYXJpbmcgbGFyZ2UgZ2Vub21lczsgR2Vub21lIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUF1dG9tYXRpYyBhbm5vdGF0aW9uIG9mIG9yZ2FuZWxsYXIgZ2Vub21lcyB3aXRoIERPR01BOyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0yMDsgIGNpdGF0aW9uX3BhZ2VzPTMyNTItNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lVng6IHNpbXBsZSB3ZWItYmFzZWQgY3JlYXRpb24gb2YgZWRpdGFibGUgY2lyY3VsYXIgY2hyb21vc29tZSBtYXBzOyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTg2MS0yOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9wdWJsaXNoZXI9U2luYXVlciBBc3NvY2lhdGVzLCBTdW5kZXJsYW5kLCBNQSwgVVNBOyBNYWNDbGFkZSBWZXJzaW9uIDQuMDYgT1NYOyAgY2l0YXRpb25feWVhcj0yMDAzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSIgY2l0YXRpb25fZWRpdGlvbj0xMHRoIGVkaXRpb247IGNpdGF0aW9uX3B1Ymxpc2hlcj1TaW5hdWVyIEFzc29jaWF0ZWQsIFN1bmRlcmxhbmQ7IFBoeWxvZ2VuZXRpYyBBbmFseXNpcyBVc2luZyBQYXJzaW1vbnkgNC4wYjsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fcHVibGlzaGVyPVVuaXZlcnNpdHkgb2YgV2FzaGluZ3RvbiwgOyBQSFlMSVAgUGh5bG9nZW5ldGljIEluZmVyZW5jZSBQcm9ncmFtcyBWZXJzaW9uIDMuNTcyOyAgY2l0YXRpb25feWVhcj0xOTk1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NYXBWaWV3OiB2aXN1YWxpemF0aW9uIG9mIHNob3J0IHJlYWRzIGFsaWdubWVudCBvbiBhIGRlc2t0b3AgY29tcHV0ZXI7IEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9MTU1NC01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NdWx0aXBsZXggc2VxdWVuY2luZyBvZiBwbGFudCBjaGxvcm9wbGFzdCBnZW5vbWVzIHVzaW5nIFNvbGV4YSBzZXF1ZW5jaW5nLWJ5LXN5bnRoZXNpcyB0ZWNobm9sb2d5OyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MzY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgY29tcGFyYXRpdmUgYW5hbHlzaXMgb2YgdGhlIExhY3R1Y2EgYW5kIEhlbGlhbnRodXMgKEFzdGVyYWNlYWUpIHBsYXN0aWQgZ2Vub21lczogaWRlbnRpZmljYXRpb24gb2YgZGl2ZXJnZW50IHJlZ2lvbnMgYW5kIGNhdGVnb3JpemF0aW9uIG9mIHNoYXJlZCByZXBlYXRzOyBBbS4gSi4gQm90LjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT05NDsgIGNpdGF0aW9uX3BhZ2VzPTMwMi0xMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXN0YWJsaXNoaW5nIGdlbm9taWMgdG9vbHMgYW5kIHJlc291cmNlcyBmb3IgR3Vpem90aWEgYWJ5c3NpbmljYTsgdGhlIGRldmVsb3BtZW50IG9mIGEgbGlicmFyeSBvZiBleHByZXNzZWQgc2VxdWVuY2UgdGFncywgbWljcm9zYXRlbGxpdGUgbG9jaSwgYW5kIHRoZSBzZXF1ZW5jaW5nIG9mIGl0cyBjaGxvcm9wbGFzdCBnZW5vbWU7IE1vbC4gRWNvbC4gUmVzb3VyLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTEwNDgtNTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRvcnRvaXNlIGFuZCB0aGUgaGFyZTogY2hvb3NpbmcgYmV0d2VlbiBub25jb2RpbmcgcGxhc3RvbWUgYW5kIG51Y2xlYXIgQURIIHNlcXVlbmNlcyBmb3IgcGh5bG9nZW55IHJlY29uc3RydWN0aW9uIGluIGEgcmVjZW50bHkgZGl2ZXJnZWQgcGxhbnQgZ3JvdXA7IEFtLiBKLiBCb3QuOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTg1OyAgY2l0YXRpb25fcGFnZXM9MTMwMS0xNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29udHJhc3RpbmcgbW9kZXMgYW5kIHRlbXBvcyBvZiBnZW5vbWUgZXZvbHV0aW9uIGluIGxhbmQgcGxhbnQgb3JnYW5lbGxlczsgVHJlbmRzIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MTk5MDsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9MTE1LTIwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMTgvMi85My8xMDQyOTU3L2RzcjAwMi5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgSW52YXNpdmUgaW5kaXZpZHVhbHMgZnJvbSB0aGUgcGVzdCBzcGVjaWVzIEphY29iYWVhIHZ1bGdhcmlzIHNob3cgZGlmZmVyZW50IGFsbG9jYXRpb24gcGF0dGVybnMgaW4gZGVmZW5jZSBhbmQgZ3Jvd3RoIGNvbXBhcmVkIHdpdGggbmF0aXZlIGluZGl2aWR1YWwiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVyLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTM2NTMzMiZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGSv75M+gaNTBJs5wabVqe1WcSSEfg==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f18%2f2%2f93%2f365332" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f18%2f2%2f93%2f365332" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=14229" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/18/2"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/18/2/1/m_cover.gif?Expires=2147483647&amp;Signature=zPrik-vQsfP8tHq7QCbk4-ZAM~7ALug60QU9E2a0HTM3jWpw4mhYvM5ZjIfk8gvh0~D~8ZHzMVsbLbVV9I~imi38eK1-WbEZj8UJvCZsVN9ocsj3MzknfjxMwhJRMN7OhfXTufW~YjTAGZRrjum43uomBF~AyUhUCSuUuosVsuEI7xRAk7wYC1Ni9rgonZdcOzQdfiuitUvT5RG~wLQZYhdUWA83IS9zyFUehXeSL1v2Ik301pcwodAx4daAPn-F6iCk6UwJJGptSyDThKiltCKPrcoMzV3V1WqixkUPlkjoJdqdq8RIeTplSpeuudNseKNAwRJXHEN70rQsx2BmAw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/18/2">
                        <div class="volume trailing-comma">Volume 18</div>
                        <div class="issue">Issue 2</div>

            </a>

        </div>
        <div class="ii-pub-date">
April 2011        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="5299637">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5299637">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5299639">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5299639">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5299648">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5299648">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5299671">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5299671">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5299705">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5299705">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5299707">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5299707">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5299709">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5299709">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/18/2/77/365047">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/18/2/107/365532">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">The Complete Chloroplast Genome of 17 Individuals of Pest Species <em>Jacobaea vulgaris</em>: SNPs, Microsatellites and Barcoding Markers for Population and Phylogenetic Studies
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Leonie Doorduin</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Institute of Biology, Section Plant Ecology and Phytochemistry</div>, <div class="insititution">Leiden University</div>, <div class="addr-line">PO Box 9505, 2300 RA Leiden</div>, <div class="country">The Netherlands</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +31 (0) 715275117. Fax. +31 (0) 715274900. E-mail: <a href="mailto:l.j.doorduin@biology.leidenuniv.nl" target="_blank">l.j.doorduin@biology.leidenuniv.nl</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Leonie+Doorduin">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Doorduin L">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Doorduin L&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Leonie Doorduin</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Barbara Gravendeel</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Institute of Biology, Section Plant Ecology and Phytochemistry</div>, <div class="insititution">Leiden University</div>, <div class="addr-line">PO Box 9505, 2300 RA Leiden</div>, <div class="country">The Netherlands</div></div><div class="aff"><div class="label -label">2</div><div class="insititution">NCB Naturalis - NHN Leiden University</div>, <div class="addr-line">Einsteinweg 2, 2333 CC Leiden</div>, <div class="country">The Netherlands</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Barbara+Gravendeel">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Gravendeel B">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Gravendeel B&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Barbara Gravendeel</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Youri Lammers</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Institute of Biology, Section Plant Ecology and Phytochemistry</div>, <div class="insititution">Leiden University</div>, <div class="addr-line">PO Box 9505, 2300 RA Leiden</div>, <div class="country">The Netherlands</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Youri+Lammers">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Lammers Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Lammers Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Youri Lammers</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yavuz Ariyurek</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Leiden Genome Technology Center, Human and Clinical Genetics</div>, <div class="insititution">Leiden University Medical Center</div>, <div class="addr-line">PO box 9600, 2300 RC Leiden</div>, <div class="country">The Netherlands</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yavuz+Ariyurek">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ariyurek Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ariyurek Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yavuz Ariyurek</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Thomas Chin-A-Woeng</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="insititution">ServiceXS</div>, <div class="addr-line">Plesmanlaan 1d, 2333 BZ Leiden</div>, <div class="country">The Netherlands</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Thomas+Chin-A-Woeng">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Chin-A-Woeng T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Chin-A-Woeng T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Thomas Chin-A-Woeng</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Klaas Vrieling</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Institute of Biology, Section Plant Ecology and Phytochemistry</div>, <div class="insititution">Leiden University</div>, <div class="addr-line">PO Box 9505, 2300 RA Leiden</div>, <div class="country">The Netherlands</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Klaas+Vrieling">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Vrieling K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Vrieling K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Klaas Vrieling</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Edited by Satoshi Tabata</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 18, Issue 2, 1 April 2011, Pages 93–105, <a href='https://doi.org/10.1093/dnares/dsr002'>https://doi.org/10.1093/dnares/dsr002</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">28 March 2011</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">29 November 2010</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">02 February 2011</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="365332" href="/dnaresearch/article-pdf/18/2/93/1042957/dsr002.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Leonie Doorduin, Barbara Gravendeel, Youri Lammers, Yavuz Ariyurek, Thomas Chin-A-Woeng, Klaas Vrieling;  The Complete Chloroplast Genome of 17 Individuals of Pest Species <em>Jacobaea vulgaris</em>: SNPs, Microsatellites and Barcoding Markers for Population and Phylogenetic Studies, <em>DNA Research</em>, Volume 18, Issue 2, 1 April 2011, Pages 93–105, <a href='https://doi.org/10.1093/dnares/dsr002'>https://doi.org/10.1093/dnares/dsr002</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/365332?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/365332?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/365332?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/365332?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/365332?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/365332?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/365332?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=The%20Complete%20Chloroplast%20Genome%20of%2017%20Individuals%20of%20Pest%20Species%20%26lt%3Bem%26gt%3BJacobaea%20vulgaris%26lt%3B%2Fem%26gt%3B%3A%20SNPs%2C%20Microsatellites%20and%20Barcoding%20Markers%20for%20Population%20and%20Phylogenetic%20Studies&amp;publicationDate=2011-03-28&amp;volumeNum=18&amp;issueNum=2&amp;author=Doorduin%2C%20Leonie%3B%20Gravendeel%2C%20Barbara&amp;startPage=93&amp;endPage=105&amp;contentId=10.1093%2Fdnares%2Fdsr002&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f18%2f2%2f93%2f365332" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=5299637 id="5299637" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Invasive individuals from the pest species <em>Jacobaea vulgaris</em> show different allocation patterns in defence and growth compared with native individuals. To examine if these changes are caused by fast evolution, it is necessary to identify native source populations and compare these with invasive populations. For this purpose, we are in need of intraspecific polymorphic markers. We therefore sequenced the complete chloroplast genomes of 12 native and 5 invasive individuals of <em>J. vulgaris</em> with next generation sequencing and discovered single-nucleotide polymorphisms (SNPs) and microsatellites. This is the first study in which the chloroplast genome of that many individuals within a single species was sequenced. Thirty-two SNPs and 34 microsatellite regions were found. For none of the individuals, differences were found between the inverted repeats. Furthermore, being the first chloroplast genome sequenced in the Senecioneae clade, we compared it with four other members of the Asteraceae family to identify new regions for phylogentic inference within this clade and also within the Asteraceae family. Five markers (<em>ndhC-trnV</em>, <em>ndhC-atpE</em>, <em>rps18-rpl20</em>, <em>clpP</em> and <em>psbM-trnD</em>) contained parsimony-informative characters higher than 2%. Finally, we compared two procedures of preparing chloroplast DNA for next generation sequencing.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;phylogenetic markers&quot;">phylogenetic markers</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Jacobaea vulgaris&quot;"><em>Jacobaea vulgaris</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="Asteraceae">Asteraceae</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;cpDNA polymorphisms&quot;">cpDNA polymorphisms</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;inverted repeat&quot;">inverted repeat</a></div>                            <h2 scrollto-destination=5299639 id="5299639" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Comprising one-tenth of all flowering plants and containing over 20 000 species, the Asteraceae are one of the largest vascular plant families.<sup><a reveal-id="DSR002C1" data-open="DSR002C1" class="link link-ref link-reveal xref-bibr">1</a></sup> With the exception of Antarctica, the Asteraceae are distributed on all continents. Species in this family are extremely various in secondary chemistry,<sup><a reveal-id="DSR002C2" data-open="DSR002C2" class="link link-ref link-reveal xref-bibr">2</a></sup> inflorescence morphology<sup><a reveal-id="DSR002C3" data-open="DSR002C3" class="link link-ref link-reveal xref-bibr">3</a></sup> and chromosome numbers.<sup><a reveal-id="DSR002C4" data-open="DSR002C4" class="link link-ref link-reveal xref-bibr">4</a></sup> This huge variation provides great opportunities to acquire insight into the diversification process in this family, which began 42–36 million years ago.<sup><a reveal-id="DSR002C5" data-open="DSR002C5" class="link link-ref link-reveal xref-bibr">5</a></sup> The Asteraceae are not only interesting because of their phenotypic and species diversity, but this family also includes members of economically important food crops, herbal species, ornamentals and plants for the cut-flower industry. Other members such as <em>Jacobaea vulgaris</em>, <em>Senecio vulgaris</em> and <em>Taraxacum officinale</em> are weedy and have an economical and ecological impact.<sup><a reveal-id="DSR002C6" data-open="DSR002C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DSR002C7" data-open="DSR002C7" class="link link-ref link-reveal xref-bibr">7</a></sup></p><p>We sequenced the complete chloroplast genome of <em>J. vulgaris</em> with next generation sequencing techniques to find new genetic markers that are phylogenetically informative and to discover intraspecific polymorphic markers for population studies. The conservative structure of the chloroplast genome makes it easy to compare with other members of the Asteraceae family. In a recent study of Panero and Funk,<sup><a reveal-id="DSR002C8" data-open="DSR002C8" class="link link-ref link-reveal xref-bibr">8</a></sup> 12 major lineages of Asteraceae were found with the Bayesian and maximum parsimony methods by combining 10 chloroplast loci from 108 taxa. Within the subfamily Asteroideae, strong statistical support was found for tribal relationships except for the Senecioneae tribe. In the Bayesian analysis, this tribe was unresolved, and in the maximum parsimony analysis, it was placed as a sister group to Calenduleae without strong statistical support (52% bootstrap proportions). In other studies of Pelser <em>et al</em>.,<sup><a reveal-id="DSR002C9" data-open="DSR002C9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="DSR002C10" data-open="DSR002C10" class="link link-ref link-reveal xref-bibr">10</a></sup> a phylogenetic analysis of the nuclear ribosomal (nr) internal spacers and external spacer and five chloroplast loci were done to clarify intergeneric relationships within Senecioneae and to delimitate the genus <em>Senecio</em>. Although these phylogenies gave more insight, they still lacked strong statistical support and resolution.</p><p>No chloroplast genome has been previously sequenced from any species in the Senecioneae clade, and the chloroplast genome sequence of <em>J. vulgaris</em> can yield more information about variation within this clade, as well as between clades of the Asteroideae subfamily. In this study, the chloroplast genome of <em>J. vulgaris</em> (tribe Jacobaea) was compared with <em>Guizotia abyssinica</em>, <em>Helianthus annuus</em>, <em>Parthenium argentatum</em> (all belonging to tribe Heliantheae) and <em>Lactuca sativa</em> (tribe Lactuceae). To guide future phylogenetic studies within the Asteraceae family, we identified new phylogenetically informative chloroplast markers by finding differences within and between genome organization.</p><p><em>Jacobaea vulgaris</em> is a troublesome weed that belongs to the Asteraceae family and is native to Europe and western Asia, ranging from Norway through Turkey, and from Great Britain to Siberia. It was first reported in the 1850s in Canada,<sup><a reveal-id="DSR002C11" data-open="DSR002C11" class="link link-ref link-reveal xref-bibr">11</a></sup> in 1875 in New Zealand<sup><a reveal-id="DSR002C12" data-open="DSR002C12" class="link link-ref link-reveal xref-bibr">12</a></sup> and shortly thereafter in Australia<sup><a reveal-id="DSR002C13" data-open="DSR002C13" class="link link-ref link-reveal xref-bibr">13</a></sup> and in 1900 at the west coast of North America.<sup><a reveal-id="DSR002C14" data-open="DSR002C14" class="link link-ref link-reveal xref-bibr">14</a></sup> In introduced areas, <em>J. vulgaris</em> is a pest species, outcompeting local plants and containing pyrrolizidine alkaloids which are toxic to herbivores.<sup><a reveal-id="DSR002C15" data-open="DSR002C15" class="link link-ref link-reveal xref-bibr">15</a></sup> Control is difficult, since the lifecycle can vary from annual to short-lived perennial, depending on the genotype. Moreover, seeds remain viable in the soil for several years.<sup><a reveal-id="DSR002C16" data-open="DSR002C16" class="link link-ref link-reveal xref-bibr">16</a></sup><em>Jacobaea vulgaris</em> causes four million dollar losses annually to cattle poisoning and control in Australia alone.<sup><a reveal-id="DSR002C17" data-open="DSR002C17" class="link link-ref link-reveal xref-bibr">17</a></sup></p><p>Joshi and Vrieling<sup><a reveal-id="DSR002C18" data-open="DSR002C18" class="link link-ref link-reveal xref-bibr">18</a></sup> compared <em>J. vulgaris</em> plants from the invasive areas with plants from the native area and found that invasive individuals contained higher pyrrolizidine alkaloid levels, have a 30% higher reproductive effort, are more susceptible to attack by specialist herbivores and less susceptible to generalist herbivores. These results suggest that selection pressures in the invasive area shaped the different allocation patterns in <em>J. vulgaris</em> in the invasive areas within 70 generations. However, it is possible that introduced populations were derived from native European populations that happened to express pyrrolizidine alkaloid and allocation patterns that are similar to those currently observed in invasive ranges.</p><p>To exclude the null hypothesis that these patterns are observed as a result of genetic drift rather than natural selection, native source populations need to be identified and compared with invasive populations.<sup><a reveal-id="DSR002C19" data-open="DSR002C19" class="link link-ref link-reveal xref-bibr">19</a></sup> Source populations can be pinpointed by using neutral molecular markers such as amplification fragment length polymorphisms (AFLPs). A previous study on <em>J. vulgaris</em>, based on nuclear AFLP data, did not show a difference in the amount of variation between native and invasive individuals. These findings suggest that introductions from multiple source populations have occurred.<sup><a reveal-id="DSR002C20" data-open="DSR002C20" class="link link-ref link-reveal xref-bibr">20</a></sup> Other neutral markers are single-nucleotide polymorphisms (SNPs) and microsatellite markers in the chloroplast genome.<sup><a reveal-id="DSR002C21  DSR002C23" data-open="DSR002C21  DSR002C23" class="link link-ref link-reveal xref-bibr">21–23</a></sup></p><p>Next generation sequencing can produce DNA sequences cheaply and quickly,<sup><a reveal-id="DSR002C24" data-open="DSR002C24" class="link link-ref link-reveal xref-bibr">24</a></sup> facilitating the rapid sequencing of nuclear and organellar genomes. Chloroplast genomes are known for their conservative rates of evolution.<sup><a reveal-id="DSR002C25" data-open="DSR002C25" class="link link-ref link-reveal xref-bibr">25</a></sup> With an average size of 150 kb, chloroplast genomes are sufficiently large to find differences between and within species.<sup><a reveal-id="DSR002C26  DSR002C28" data-open="DSR002C26  DSR002C28" class="link link-ref link-reveal xref-bibr">26–28</a></sup> The absence of recombination and maternal transmission of the chloroplast genome (limiting gene flow to seed dispersal only) makes cpDNA markers useful for tracing source population(s).<sup><a reveal-id="DSR002C29" data-open="DSR002C29" class="link link-ref link-reveal xref-bibr">29</a>,<a reveal-id="DSR002C30" data-open="DSR002C30" class="link link-ref link-reveal xref-bibr">30</a></sup></p><p>In this study, we sequenced the chloroplast genome of 17 <em>J. vulgaris</em> individuals by using the Illumina genome analyzer platform. This is the first study sequencing multiple individuals of the same species with next generation sequencing. Multiple individuals were sequenced to reveal intraspecific variation (SNPs and microsatellite loci). Finally, we compared two different procedures of preparation for sequencing the chloroplast genome, namely direct extraction of the chloroplast DNA and amplifying the cpDNA with a long-range PCR.</p>                            <h2 scrollto-destination=5299648 id="5299648" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=5299649 id="5299649" class="section-title" data-legacy-id=s2a>2.1. Extraction of chloroplasts and isolation of DNA from chloroplasts</h3>
<p>By using the chloroplast extraction kit of Sigma-Aldrich (CP-ISO) and following the manufacturer's protocol, chloroplasts from sample nr. 17 (Table <a reveal-id="DSR002TB1" data-open="DSR002TB1" class="link link-reveal link-table xref-fig">1</a>) were isolated out of 30 g of fresh leaf material. To remove unwanted whole cells and cell wall debris, the blended leaf material with the chloroplast isolation buffer was centrifuged. To separate the intact from the broken chloroplasts, a 40% Percoll layer was used. Before DNA extraction, the intact chloroplasts were treated with ST buffer (400 mM sucrose, 50 mM Tris, pH 7.8, 0.1% bovine serum albumin) with a final concentration of 25 μg/ml DNAse-1 (Sigma-Aldrich) per gram of leaf material to digest DNA outside the intact chloroplasts. After centrifuging, the chloroplast pellet was resuspended in a Tris EDTA NaCl buffer (100 mM Tris, pH 7.2, 50 mM ethylenediaminetetraacetic acid (EDTA), 100 mM NaCl, 0.2% β-mercaptoethanol). To extract the DNA from the chloroplasts, the chloroplasts were lysed with 1% sodium dodecyl sulphate followed by a phenol/chloroform step to remove proteins. The DNA was precipitated overnight with 1/10 vol. of 5 M ammonium acetate and 1 vol. of isopropanol. After centrifuging, the pellet was washed with 70% ethanol and redissolved in TE buffer (1 M Tris–HCl, pH = 8.0, 0.5 M EDTA).<sup><a reveal-id="DSR002C31" data-open="DSR002C31" class="link link-ref link-reveal xref-bibr">31</a></sup></p>                        <a id="5299651" scrollto-destination="5299651"></a>
<div content-id="DSR002TB1" class="table-modal"><div class="table-wrap"><div id="DSR002TB1" class="table-wrap-title" data-id="DSR002TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Geographical information, percentage of the chloroplast genome sequenced, method used for preparing the template for Illumina sequencing, lane number on the Illumina platform and reads obtained from the 17 individuals of <em>J. vulgaris</em> that were sequenced</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Sample </th><th>Country </th><th>Location </th><th>Latitude/longitude </th><th>% cp genome sequenced </th><th>Template sequencing </th><th>Illumina lane </th></tr></thead><tbody><tr><td>1 </td><td>New Zealand </td><td>Haast (South Island) </td><td>43°S 169°E </td><td>89.9 </td><td>Long-range PCR </td><td>2 (776) </td></tr><tr><td>2 </td><td>Ireland </td><td>Caherdaniel </td><td>51°N 10°W </td><td>88.5 </td><td>Long-range PCR </td><td>2 (545) </td></tr><tr><td>3 </td><td>Norway </td><td>Malvik </td><td>63°N 10°E </td><td>83.4 </td><td>Long-range PCR </td><td>2 (543) </td></tr><tr><td>4 </td><td>Canada </td><td>Cardigan </td><td>46°N 62°W </td><td>89.8 </td><td>Long-range PCR </td><td>2 (838) </td></tr><tr><td>5 </td><td>UK </td><td>Padstow </td><td>50°N 4°W </td><td>98.3 </td><td>Long-range PCR </td><td>2 (1043) </td></tr><tr><td>6 </td><td>Poland </td><td>Warsaw </td><td>52°N 18°E </td><td>94.3 </td><td>Long-range PCR </td><td>2 (650) </td></tr><tr><td>7 </td><td>Spain </td><td>Covadonga </td><td>43°N 04°W </td><td>91.5 </td><td>Long-range PCR </td><td>2 (457) </td></tr><tr><td>8 </td><td>France </td><td>Perrogney </td><td>47°N 05°E </td><td>89.9 </td><td>Long-range PCR </td><td>2 (558) </td></tr><tr><td>9 </td><td>Hungary </td><td>Lénárddaróc </td><td>48°N 20°E </td><td>86.7 </td><td>Long-range PCR </td><td>2 (80) </td></tr><tr><td>10 </td><td>The Netherlands </td><td>Ameland </td><td>53°N 05°E </td><td>88.6 </td><td>Long-range PCR </td><td>2 (468) </td></tr><tr><td>11 </td><td>Australia </td><td>Barramonga </td><td>38°S 143°E </td><td>90.6 </td><td>Long-range PCR </td><td>2 (680) </td></tr><tr><td>12 </td><td>Australia </td><td>Franklin (Tasmania) </td><td>43°S 147°E </td><td>91.8 </td><td>Long-range PCR </td><td>2 (465) </td></tr><tr><td>13 </td><td>UK </td><td>Portsmouth </td><td>50°N 01°W </td><td>98.9 </td><td>Long-range PCR </td><td>2 (1102) </td></tr><tr><td>14 </td><td>Sweden </td><td>Kapellskär </td><td>59°N 53°E </td><td>99.9 </td><td>Long-range PCR </td><td>3 (11 084) </td></tr><tr><td>15 </td><td>New Zealand </td><td>Opunake (North Island) </td><td>39°S 173°E </td><td>94.7 </td><td>Long-range PCR </td><td>2 (691) </td></tr><tr><td>16 </td><td>Germany </td><td>Halle </td><td>51°N 11°E </td><td>98.7 </td><td>Long-range PCR </td><td>2 (805) </td></tr><tr><td>17 </td><td>Spain </td><td>Covadonga </td><td>43°N 04°W </td><td>99.9 </td><td>Chloroplast DNA extract </td><td>1 (18 646)<sup>a</sup> </td></tr></tbody></table></div><div content-id="DSR002TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Sample </th><th>Country </th><th>Location </th><th>Latitude/longitude </th><th>% cp genome sequenced </th><th>Template sequencing </th><th>Illumina lane </th></tr></thead><tbody><tr><td>1 </td><td>New Zealand </td><td>Haast (South Island) </td><td>43°S 169°E </td><td>89.9 </td><td>Long-range PCR </td><td>2 (776) </td></tr><tr><td>2 </td><td>Ireland </td><td>Caherdaniel </td><td>51°N 10°W </td><td>88.5 </td><td>Long-range PCR </td><td>2 (545) </td></tr><tr><td>3 </td><td>Norway </td><td>Malvik </td><td>63°N 10°E </td><td>83.4 </td><td>Long-range PCR </td><td>2 (543) </td></tr><tr><td>4 </td><td>Canada </td><td>Cardigan </td><td>46°N 62°W </td><td>89.8 </td><td>Long-range PCR </td><td>2 (838) </td></tr><tr><td>5 </td><td>UK </td><td>Padstow </td><td>50°N 4°W </td><td>98.3 </td><td>Long-range PCR </td><td>2 (1043) </td></tr><tr><td>6 </td><td>Poland </td><td>Warsaw </td><td>52°N 18°E </td><td>94.3 </td><td>Long-range PCR </td><td>2 (650) </td></tr><tr><td>7 </td><td>Spain </td><td>Covadonga </td><td>43°N 04°W </td><td>91.5 </td><td>Long-range PCR </td><td>2 (457) </td></tr><tr><td>8 </td><td>France </td><td>Perrogney </td><td>47°N 05°E </td><td>89.9 </td><td>Long-range PCR </td><td>2 (558) </td></tr><tr><td>9 </td><td>Hungary </td><td>Lénárddaróc </td><td>48°N 20°E </td><td>86.7 </td><td>Long-range PCR </td><td>2 (80) </td></tr><tr><td>10 </td><td>The Netherlands </td><td>Ameland </td><td>53°N 05°E </td><td>88.6 </td><td>Long-range PCR </td><td>2 (468) </td></tr><tr><td>11 </td><td>Australia </td><td>Barramonga </td><td>38°S 143°E </td><td>90.6 </td><td>Long-range PCR </td><td>2 (680) </td></tr><tr><td>12 </td><td>Australia </td><td>Franklin (Tasmania) </td><td>43°S 147°E </td><td>91.8 </td><td>Long-range PCR </td><td>2 (465) </td></tr><tr><td>13 </td><td>UK </td><td>Portsmouth </td><td>50°N 01°W </td><td>98.9 </td><td>Long-range PCR </td><td>2 (1102) </td></tr><tr><td>14 </td><td>Sweden </td><td>Kapellskär </td><td>59°N 53°E </td><td>99.9 </td><td>Long-range PCR </td><td>3 (11 084) </td></tr><tr><td>15 </td><td>New Zealand </td><td>Opunake (North Island) </td><td>39°S 173°E </td><td>94.7 </td><td>Long-range PCR </td><td>2 (691) </td></tr><tr><td>16 </td><td>Germany </td><td>Halle </td><td>51°N 11°E </td><td>98.7 </td><td>Long-range PCR </td><td>2 (805) </td></tr><tr><td>17 </td><td>Spain </td><td>Covadonga </td><td>43°N 04°W </td><td>99.9 </td><td>Chloroplast DNA extract </td><td>1 (18 646)<sup>a</sup> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Numbers given in parenthesis are the number of single-end reads × 1000.</p></div></div><div content-id=""><div class="fn"><p><sup>a</sup>Paired-end reads.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5299651" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR002TB1" class="table-wrap-title" data-id="DSR002TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Geographical information, percentage of the chloroplast genome sequenced, method used for preparing the template for Illumina sequencing, lane number on the Illumina platform and reads obtained from the 17 individuals of <em>J. vulgaris</em> that were sequenced</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Sample </th><th>Country </th><th>Location </th><th>Latitude/longitude </th><th>% cp genome sequenced </th><th>Template sequencing </th><th>Illumina lane </th></tr></thead><tbody><tr><td>1 </td><td>New Zealand </td><td>Haast (South Island) </td><td>43°S 169°E </td><td>89.9 </td><td>Long-range PCR </td><td>2 (776) </td></tr><tr><td>2 </td><td>Ireland </td><td>Caherdaniel </td><td>51°N 10°W </td><td>88.5 </td><td>Long-range PCR </td><td>2 (545) </td></tr><tr><td>3 </td><td>Norway </td><td>Malvik </td><td>63°N 10°E </td><td>83.4 </td><td>Long-range PCR </td><td>2 (543) </td></tr><tr><td>4 </td><td>Canada </td><td>Cardigan </td><td>46°N 62°W </td><td>89.8 </td><td>Long-range PCR </td><td>2 (838) </td></tr><tr><td>5 </td><td>UK </td><td>Padstow </td><td>50°N 4°W </td><td>98.3 </td><td>Long-range PCR </td><td>2 (1043) </td></tr><tr><td>6 </td><td>Poland </td><td>Warsaw </td><td>52°N 18°E </td><td>94.3 </td><td>Long-range PCR </td><td>2 (650) </td></tr><tr><td>7 </td><td>Spain </td><td>Covadonga </td><td>43°N 04°W </td><td>91.5 </td><td>Long-range PCR </td><td>2 (457) </td></tr><tr><td>8 </td><td>France </td><td>Perrogney </td><td>47°N 05°E </td><td>89.9 </td><td>Long-range PCR </td><td>2 (558) </td></tr><tr><td>9 </td><td>Hungary </td><td>Lénárddaróc </td><td>48°N 20°E </td><td>86.7 </td><td>Long-range PCR </td><td>2 (80) </td></tr><tr><td>10 </td><td>The Netherlands </td><td>Ameland </td><td>53°N 05°E </td><td>88.6 </td><td>Long-range PCR </td><td>2 (468) </td></tr><tr><td>11 </td><td>Australia </td><td>Barramonga </td><td>38°S 143°E </td><td>90.6 </td><td>Long-range PCR </td><td>2 (680) </td></tr><tr><td>12 </td><td>Australia </td><td>Franklin (Tasmania) </td><td>43°S 147°E </td><td>91.8 </td><td>Long-range PCR </td><td>2 (465) </td></tr><tr><td>13 </td><td>UK </td><td>Portsmouth </td><td>50°N 01°W </td><td>98.9 </td><td>Long-range PCR </td><td>2 (1102) </td></tr><tr><td>14 </td><td>Sweden </td><td>Kapellskär </td><td>59°N 53°E </td><td>99.9 </td><td>Long-range PCR </td><td>3 (11 084) </td></tr><tr><td>15 </td><td>New Zealand </td><td>Opunake (North Island) </td><td>39°S 173°E </td><td>94.7 </td><td>Long-range PCR </td><td>2 (691) </td></tr><tr><td>16 </td><td>Germany </td><td>Halle </td><td>51°N 11°E </td><td>98.7 </td><td>Long-range PCR </td><td>2 (805) </td></tr><tr><td>17 </td><td>Spain </td><td>Covadonga </td><td>43°N 04°W </td><td>99.9 </td><td>Chloroplast DNA extract </td><td>1 (18 646)<sup>a</sup> </td></tr></tbody></table></div><div content-id="DSR002TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Sample </th><th>Country </th><th>Location </th><th>Latitude/longitude </th><th>% cp genome sequenced </th><th>Template sequencing </th><th>Illumina lane </th></tr></thead><tbody><tr><td>1 </td><td>New Zealand </td><td>Haast (South Island) </td><td>43°S 169°E </td><td>89.9 </td><td>Long-range PCR </td><td>2 (776) </td></tr><tr><td>2 </td><td>Ireland </td><td>Caherdaniel </td><td>51°N 10°W </td><td>88.5 </td><td>Long-range PCR </td><td>2 (545) </td></tr><tr><td>3 </td><td>Norway </td><td>Malvik </td><td>63°N 10°E </td><td>83.4 </td><td>Long-range PCR </td><td>2 (543) </td></tr><tr><td>4 </td><td>Canada </td><td>Cardigan </td><td>46°N 62°W </td><td>89.8 </td><td>Long-range PCR </td><td>2 (838) </td></tr><tr><td>5 </td><td>UK </td><td>Padstow </td><td>50°N 4°W </td><td>98.3 </td><td>Long-range PCR </td><td>2 (1043) </td></tr><tr><td>6 </td><td>Poland </td><td>Warsaw </td><td>52°N 18°E </td><td>94.3 </td><td>Long-range PCR </td><td>2 (650) </td></tr><tr><td>7 </td><td>Spain </td><td>Covadonga </td><td>43°N 04°W </td><td>91.5 </td><td>Long-range PCR </td><td>2 (457) </td></tr><tr><td>8 </td><td>France </td><td>Perrogney </td><td>47°N 05°E </td><td>89.9 </td><td>Long-range PCR </td><td>2 (558) </td></tr><tr><td>9 </td><td>Hungary </td><td>Lénárddaróc </td><td>48°N 20°E </td><td>86.7 </td><td>Long-range PCR </td><td>2 (80) </td></tr><tr><td>10 </td><td>The Netherlands </td><td>Ameland </td><td>53°N 05°E </td><td>88.6 </td><td>Long-range PCR </td><td>2 (468) </td></tr><tr><td>11 </td><td>Australia </td><td>Barramonga </td><td>38°S 143°E </td><td>90.6 </td><td>Long-range PCR </td><td>2 (680) </td></tr><tr><td>12 </td><td>Australia </td><td>Franklin (Tasmania) </td><td>43°S 147°E </td><td>91.8 </td><td>Long-range PCR </td><td>2 (465) </td></tr><tr><td>13 </td><td>UK </td><td>Portsmouth </td><td>50°N 01°W </td><td>98.9 </td><td>Long-range PCR </td><td>2 (1102) </td></tr><tr><td>14 </td><td>Sweden </td><td>Kapellskär </td><td>59°N 53°E </td><td>99.9 </td><td>Long-range PCR </td><td>3 (11 084) </td></tr><tr><td>15 </td><td>New Zealand </td><td>Opunake (North Island) </td><td>39°S 173°E </td><td>94.7 </td><td>Long-range PCR </td><td>2 (691) </td></tr><tr><td>16 </td><td>Germany </td><td>Halle </td><td>51°N 11°E </td><td>98.7 </td><td>Long-range PCR </td><td>2 (805) </td></tr><tr><td>17 </td><td>Spain </td><td>Covadonga </td><td>43°N 04°W </td><td>99.9 </td><td>Chloroplast DNA extract </td><td>1 (18 646)<sup>a</sup> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Numbers given in parenthesis are the number of single-end reads × 1000.</p></div></div><div content-id=""><div class="fn"><p><sup>a</sup>Paired-end reads.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5299651" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=5299652 id="5299652" class="section-title" data-legacy-id=s2b>2.2. Total DNA extraction</h3>
<p>Total DNA extractions from samples 1 to 16 of <em>J. vulgaris</em> (Table <a reveal-id="DSR002TB1" data-open="DSR002TB1" class="link link-reveal link-table xref-fig">1</a>) were carried out on five leaf punches of 1 cm diameter each, using the CTAB extraction protocol of Doyle and Doyle.<sup><a reveal-id="DSR002C32" data-open="DSR002C32" class="link link-ref link-reveal xref-bibr">32</a></sup></p>                            <h3 scrollto-destination=5299655 id="5299655" class="section-title" data-legacy-id=s2c>2.3. Long-range PCR</h3>
<p>To develop primers for a long-range PCR, the sequences of <em>H. annuus</em> (NC007977), <em>L. sativa</em> (DQ383816) and <em>G. abyssinica</em> (EU549769) were aligned with BioEdit. With the aid of this alignment and the annotation of <em>H. annuus</em>, primers were designed in conserved regions of genes. A total of 18 primer pairs was designed by Primer3 software,<sup><a reveal-id="DSR002C33" data-open="DSR002C33" class="link link-ref link-reveal xref-bibr">33</a></sup> which collectively amplified the total chloroplast genome of <em>J. vulgaris</em> with overlapping fragments resulting in amplicons between 5808 and 11 110 bp (see <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a> for primer sequences). For amplification, the Takara La Taq kit (Takara Bio Inc., Otsu, Shiga, Japan) was used. PCR was carried out in a total volume of 20 μl containing 8–80 ng of DNA, 2.5 mM MgCl<sub>2</sub>, 2.5 mM of each dNTP, 0.7 μM of each primer and 1 U <em>Taq</em> DNA polymerase. The PCR cycling conditions were as follows: 1 min at 94°C; 30 cycles of 10 s at 98°C and 12 min at 69°C; followed by 10 min at 72°C. PCR products were loaded on a 1.5% agarose gel, stained with ethidium bromide and visualized under UV light to check for amplification. If the PCR products contained more than one band, the total product was always loaded on a 1% agarose gel and bands of the right size were cut out of the gel. To extract and purify the DNA fragments from the gel, the Wizard SV gel and PCR Clean-Up System of Promega was used. All cleaned PCR products were run on a gel to estimate the amount of product, and in addition, the amount of DNA was quantified with an ND-1000 spectrophotometer (Nanodrop Technologies). All 18 amplicons for each individual sample were pooled in equal molar ratios containing roughly 200–300 ng of DNA resulting in 16 pooled samples of 75 μl each.</p>                            <h3 scrollto-destination=5299657 id="5299657" class="section-title" data-legacy-id=s2d>2.4. Sequencing</h3>
<p>For sequencing of the cpDNA, three lanes on an Illumina sequencer (Illumina 1G/Solexa, Illumina Inc., San Diego, CA, USA) were used. Sequencing was carried out at the Leiden Genome Technology Center. In the first lane, the DNA isolated from the chloroplasts of sample 17 was run with paired-end reads of 32 bp. In the second lane, the pooled long-range PCR products of samples 1–13, 15 and 16 were run and in the third lane sample 14. Both were single-end runs of 35 bp (Table <a reveal-id="DSR002TB1" data-open="DSR002TB1" class="link link-reveal link-table xref-fig">1</a>). Sample 14 was run in a separate lane because of its low DNA concentration. Preparation of all products was done following the protocol of Illumina kits with minor modifications. For sample 17, DNA was fragmented by a nebulizer using 32 psi N<sub>2</sub> for 6 min. After purification, the DNA was eluted in 15 μl elution buffer. The samples were blunt-ended with T4 DNA polymerase, Klenow polymerase and T4 polynucleotide kinase. After purification, an A-residue was added to the 3′ end of the DNA fragments using Klenow fragment (3′–5′ exo-minus). Purification was done with a Qiagen MinElute column. Adapters of the paired-end adapter oligo mix were ligated to the DNA fragments. After purification with a Qiagen MinElute column, adapter-ligated DNAs in the range of 200–250 bp were size selected using agarose electrophoresis. Products were isolated from the gel using a QIAquick Gel Extraction Kit and after purification a PCR was done.</p><p>For samples 1–16 (Table <a reveal-id="DSR002TB1" data-open="DSR002TB1" class="link link-reveal link-table xref-fig">1</a>), sonication with a Bioruptor was used to fragment the DNA. This machine was placed in a room at 4°C and was kept cool by adding ice. For a total of 15 min, the machine was set on 30 s active and 30 s inactive. This sonication step was repeated four times. All other steps were the same as done for sample 17 except for the PCR step. Unique index tags of six bases provided in the Multiplexing Sample Preparation Oligonucleotide Kit were added in the PCR step to discriminate between the 16 samples. The amplified libraries were quantified by lab-on-a-chip (Agilent Technologies) followed by equimolar mixing of 10 nM per sample. Cluster generation was performed after applying 6 pM of each sample to the individual lanes of the Illumina flow cell, and sequencing was carried out on the Illumina Genome Analyzer according to the manufacturer's instructions. Image analysis and base calling were performed using the Illumina Pipeline 1.3.2, where sequence tags were obtained after purity filtering. This was followed by an alignment using MAQ.</p>                            <h3 scrollto-destination=5299660 id="5299660" class="section-title" data-legacy-id=s2e>2.5. Data filtering and genome assembly</h3>
<p>Sample 17 from the first lane was used to assemble a draft chloroplast (cp) genome of <em>J. vulgaris</em>. The software package MAQ v0.5.0 was used to map all quality-filtered paired reads of the first run against the chloroplast genome of <em>H. annuus</em>. To solve gaps in this consensus sequence, a <em>de novo</em> assembly was done with the same data using the software package Velvet v 0.6<sup><a reveal-id="DSR002C34" data-open="DSR002C34" class="link link-ref link-reveal xref-bibr">34</a></sup> (parameters: hash length = 21), which produced 37 747 contigs. To find contigs with homology to the reference, these contigs were aligned to the <em>H. annuus</em> reference sequence with the program Mummer v3.0.<sup><a reveal-id="DSR002C35" data-open="DSR002C35" class="link link-ref link-reveal xref-bibr">35</a></sup> The contigs having homology to the reference were extended by using the original reads with Velvet. These extended contigs were aligned to the reference of <em>H. annuus</em> with Mummer once again, and the contigs which assembled properly were saved. These final contigs were aligned against the consensus sequence; as a result, some of the gaps in the consensus were solved. A new MAQ alignment was performed, mapping all the Illumina reads against the last consensus sequence made, to produce the draft sequence.</p>                            <h3 scrollto-destination=5299662 id="5299662" class="section-title" data-legacy-id=s2f>2.6. Bridging the gaps that were still in the draft sequence</h3>
<p>The draft sequence still contained 23 gaps with an average gap length of 394 bp. Gaps were bridged by adding the data from the runs of the cpDNA amplified by a long-range PCR of 16 individuals. These data were used in Velvet to produce a <em>de novo</em> sequence (parameters: hash length = 21, short-fastq reads). The resulting <em>de novo</em> contigs were aligned against the draft sequence in the Blast's bl2seq multiple sequence aligner. In this way, five gaps with a total of 1822 bp were bridged. The last 18 gaps were bridged by developing primers around the gaps, and traditional Sanger sequencing to yield the final complete cp genome.</p>                            <h3 scrollto-destination=5299664 id="5299664" class="section-title" data-legacy-id=s2g>2.7. Annotation</h3>
<p>The program DOGMA<sup><a reveal-id="DSR002C36" data-open="DSR002C36" class="link link-ref link-reveal xref-bibr">36</a></sup> was used for annotating all genes and to identify rRNAs and tRNAs. A circular cp genome map (Fig. <a reveal-id="DSR002F1" data-open="DSR002F1" class="link link-reveal link-table xref-fig">1</a>) was drawn using the program GenomeVx.<sup><a reveal-id="DSR002C37" data-open="DSR002C37" class="link link-ref link-reveal xref-bibr">37</a></sup></p>                        <a id="5299666" scrollto-destination="5299666"></a>
<div id="DSR002F1" data-id="DSR002F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/2/10.1093/dnares/dsr002/2/m_dsr00201.jpeg?Expires=2147483647&Signature=1bmgxdyg4a0EJjKPsit3wApg3M01tHkOQv0QTpjpb30QQWwjF75razbFFjpWthamqCO2YGaiG28s-LAdKyHz3qqJ1AlqIJ-XDOpap0JrwtA~vJYEar6HHqA-61C19yki2B~jSWuSxiklhAkYfHIWQNj9EIklMCracLRssE4ZC4~mxEQID1aFztn5yE-i0tN3jomHbAmL6WMylmMD6IGdHA0CszM-K0b6xdbS9DfcMuyk0jzEq-kbaPTrgUlAAdL0jlPhIjB5GL04O1Xn7kCNFLI11kPsyG8F9AlpITAxFOJdxGTihN5Y~fMWIa1znnyqUKxx4ksOfVib-xpgphgKiw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Representative map of the chloroplast genome of J. vulgaris (GenBank accession HQ234669)." path-from-xml="dsr00201.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5299666/dsr00201.jpeg" path-from-xml="dsr00201.jpeg" target="_blank">View large</a><a section="5299666" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/2/10.1093/dnares/dsr002/2/dsr00201.jpeg?Expires=2147483647&Signature=CzKbeTmdAe17v1dCRVv1w5M5Wp4rUISWRaNFd9IsjHIeHJvFOcfJj3j3mOa7j3ynvC9ZZpXlQP0WD7vepXUQJfaIklGX0hqrLPOfq1awEY~19rkAdIwk4qzutgF2-CF3wGF7vuK-xnzo8ePYlT0WZIEF6RTwUTM6fjT4eLEXPzSeWmyVhhd9XAEl6A7Swoi8js9Aog1jE5V~Ct60H34VCGuQ2i3Xe~21wmnBKzFsAqtriA3B8v8beIoglmAhemZNKlACW7XL~XlUcAiidHr-cdCcXfjiiNqZ6Hg8vHbenipbprSzRyxViAKK2a06D5Ew6DI~SFrWyLAgXQvZK-0ZZA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5299666&ar=365332&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr00201.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Representative map of the chloroplast genome of <em>J. vulgaris</em> (GenBank accession HQ234669).</p></div></div><div content-id="DSR002F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/2/10.1093/dnares/dsr002/2/m_dsr00201.jpeg?Expires=2147483647&Signature=1bmgxdyg4a0EJjKPsit3wApg3M01tHkOQv0QTpjpb30QQWwjF75razbFFjpWthamqCO2YGaiG28s-LAdKyHz3qqJ1AlqIJ-XDOpap0JrwtA~vJYEar6HHqA-61C19yki2B~jSWuSxiklhAkYfHIWQNj9EIklMCracLRssE4ZC4~mxEQID1aFztn5yE-i0tN3jomHbAmL6WMylmMD6IGdHA0CszM-K0b6xdbS9DfcMuyk0jzEq-kbaPTrgUlAAdL0jlPhIjB5GL04O1Xn7kCNFLI11kPsyG8F9AlpITAxFOJdxGTihN5Y~fMWIa1znnyqUKxx4ksOfVib-xpgphgKiw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Representative map of the chloroplast genome of J. vulgaris (GenBank accession HQ234669)." path-from-xml="dsr00201.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5299666/dsr00201.jpeg" path-from-xml="dsr00201.jpeg" target="_blank">View large</a><a section="5299666" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/2/10.1093/dnares/dsr002/2/dsr00201.jpeg?Expires=2147483647&Signature=CzKbeTmdAe17v1dCRVv1w5M5Wp4rUISWRaNFd9IsjHIeHJvFOcfJj3j3mOa7j3ynvC9ZZpXlQP0WD7vepXUQJfaIklGX0hqrLPOfq1awEY~19rkAdIwk4qzutgF2-CF3wGF7vuK-xnzo8ePYlT0WZIEF6RTwUTM6fjT4eLEXPzSeWmyVhhd9XAEl6A7Swoi8js9Aog1jE5V~Ct60H34VCGuQ2i3Xe~21wmnBKzFsAqtriA3B8v8beIoglmAhemZNKlACW7XL~XlUcAiidHr-cdCcXfjiiNqZ6Hg8vHbenipbprSzRyxViAKK2a06D5Ew6DI~SFrWyLAgXQvZK-0ZZA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5299666&ar=365332&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr00201.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Representative map of the chloroplast genome of <em>J. vulgaris</em> (GenBank accession HQ234669).</p></div></div>                            <h3 scrollto-destination=5299667 id="5299667" class="section-title" data-legacy-id=s2h>2.8. Comparison of the chloroplast DNA of J. vulgaris with other Asteraceae genomes analysed</h3>
<p>A total of 22 conserved protein-coding genes from five species, extracted from all available complete chloroplast genomes from Asteraceae deposited at NCBI GenBank (<em>H. annuus</em>, NC007977; <em>L. sativa</em>, DQ383816; <em>P. argentatum</em>, GU120098; <em>G. abyssinica</em>, EU549769 and <em>J. vulgaris</em>, HQ234669), were aligned using the pairwise automatic alignment tool in MacClade 4.06<sup><a reveal-id="DSR002C38" data-open="DSR002C38" class="link link-ref link-reveal xref-bibr">38</a></sup> with further adjustment by hand. To get insight in the informative character of the selected protein-coding genes, maximum parsimony analyses were run on the individual alignments comprising a total of 33 669 bp with PAUP* 4.0b10<sup><a reveal-id="DSR002C39" data-open="DSR002C39" class="link link-ref link-reveal xref-bibr">39</a></sup> using heuristic search, random addition with 100 replicates and tree bisection-reconnection (TBR) swapping. The relative robustness for clades found in all single most parsimonious trees (MPTs) was assessed by performing 1000 replicates of bootstrapping<sup><a reveal-id="DSR002C40" data-open="DSR002C40" class="link link-ref link-reveal xref-bibr">40</a></sup> using fast, stepwise additions, TBR branch-swapping with 10 random taxon additions per replicate, MULTREES on and holding 100 trees per replicate. We also calculated tree lengths and consistency index (CI) and retention index (RI) values measuring the extent of homoplasy.</p>                            <h3 scrollto-destination=5299669 id="5299669" class="section-title" data-legacy-id=s2i>2.9. Detection of polymorphic loci</h3>
<p>For visualizing the output of all reads, Mapview was used.<sup><a reveal-id="DSR002C41" data-open="DSR002C41" class="link link-ref link-reveal xref-bibr">41</a></sup> This program visualizes all reads that are mapped against the reference genome. Furthermore, it can produce a SNP list. The final assembled cp genome was used as a reference. To find SNPs, genomes of individuals 1–17 were used. SNPs were only added to the list if at least one individual that varied from the reference genome had a coverage of at least 30 reads traversing that particular nucleotide and only when SNPs were located outside A and T polymer regions. Potential microsatellite regions were tracked by looking for 10 or more repeats of A and T nucleotides.</p>                            <h2 scrollto-destination=5299671 id="5299671" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=5299672 id="5299672" class="section-title" data-legacy-id=s3a>3.1. Construction of the chloroplast genome of J. vulgaris</h3>
<p>The chloroplast genome of <em>J. vulgaris</em> is 150 686 bp in length. The genome contains two inverted repeat (IR) regions of 24 777 bp each. The IRs are separated by a large single-copy (LSC) and a small single-copy (SSC) region of 82 855 and 18 277 bp, respectively. The genome comprises 81 protein-coding genes of which seven are located in the IRs. <em>Ycf1</em> lies partly in the IR and the single-copy region. The four rRNA genes are all located in the IR. There are 29 unique tRNA genes. Twenty-two tRNA genes are located in the single-copy region, whereas the others are located in the IR (Fig. <a reveal-id="DSR002F1" data-open="DSR002F1" class="link link-reveal link-table xref-fig">1</a>).</p><p>The single lane on Illumina yielded sufficient reads to map more than 99.9% of the complete cp genome of <em>J. vulgaris</em>. For the pooled individuals, on average 92% of the whole genome was mapped. There was a highly significant correlation between the number of reads and percentage of the genome mapped (Fig. <a reveal-id="DSR002F2" data-open="DSR002F2" class="link link-reveal link-table xref-fig">2</a>). From the figure, it is estimated that ∼1 300 000 single-end Illumina reads of 32 bp are needed to reach a mapping percentage higher than 99.9% of the cp genome of <em>J. vulgaris</em>. </p>                        <a id="5299675" scrollto-destination="5299675"></a>
<div id="DSR002F2" data-id="DSR002F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/2/10.1093/dnares/dsr002/2/m_dsr00202.jpeg?Expires=2147483647&Signature=rYpHE5mm4oQOvfC8QZRVsWkbR~S2lXjTwFKTQEuhmGHHnM0gftwzw1o1IPzENIahsUYuplGNxYwAd14SjWcvE94S47NxibiccB5YfS5ET~FJPfhxUVKmNr1nqCncFljHQJoIX23fCGejLBvwX9kldiRlj39y7Ec8iTNm-1145-rS-emKU~MSHbn7RkCmFRNg~Yu0H5NqoFiSxUAlZ4KX0ISxPJs-fU8G9EPxbKonIB7Aihqj~YgZqd7mCiIfmE3DdRjjBiY5jdfYyy9rtiaWYugn3oLocuhHV1CMXkgjtbcESvgF1iPT1m0yGJHoN2jl23g194l4rsxAz1vjCGoZ-w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Number of Illumina sequencing reads plotted against percentage of the chloroplast genome mapped for 17 individuals of J. vulgaris." path-from-xml="dsr00202.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5299675/dsr00202.jpeg" path-from-xml="dsr00202.jpeg" target="_blank">View large</a><a section="5299675" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/2/10.1093/dnares/dsr002/2/dsr00202.jpeg?Expires=2147483647&Signature=JKp8-jQoW~zgRH8YV2JnIaMloKQkRTq08QRtkoetiqwZF6fu~kE9JcwYtGrBFYpQKWP7fMkv0AgGPz-87MxMcQ2WfDlYBURN67muIJK9BMAWk7FQV3-a9cl3J6sxqMjOL-O2Ez1aCTXZsNzkBnYYm~DJQncnwTMRpaexcrjC1aZe9o4G~BHSu-1ZTlLb-ZlPlN1g0PMKadeWnTWQU4Nk3JFzpL9YNSAyKEsgM6zYNtrYO1cyRcLmcT4XXGsZy1GqBTWk~Ds4hmpfXVTZnoND451wLBiAMVe3Z~Z37MoG~p6m50Cs4-vNyc9AwdFRLc2662eU5gX45OcKNS12QBgpjA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5299675&ar=365332&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr00202.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Number of Illumina sequencing reads plotted against percentage of the chloroplast genome mapped for 17 individuals of <em>J. vulgaris</em>.</p></div></div><div content-id="DSR002F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/2/10.1093/dnares/dsr002/2/m_dsr00202.jpeg?Expires=2147483647&Signature=rYpHE5mm4oQOvfC8QZRVsWkbR~S2lXjTwFKTQEuhmGHHnM0gftwzw1o1IPzENIahsUYuplGNxYwAd14SjWcvE94S47NxibiccB5YfS5ET~FJPfhxUVKmNr1nqCncFljHQJoIX23fCGejLBvwX9kldiRlj39y7Ec8iTNm-1145-rS-emKU~MSHbn7RkCmFRNg~Yu0H5NqoFiSxUAlZ4KX0ISxPJs-fU8G9EPxbKonIB7Aihqj~YgZqd7mCiIfmE3DdRjjBiY5jdfYyy9rtiaWYugn3oLocuhHV1CMXkgjtbcESvgF1iPT1m0yGJHoN2jl23g194l4rsxAz1vjCGoZ-w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Number of Illumina sequencing reads plotted against percentage of the chloroplast genome mapped for 17 individuals of J. vulgaris." path-from-xml="dsr00202.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5299675/dsr00202.jpeg" path-from-xml="dsr00202.jpeg" target="_blank">View large</a><a section="5299675" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/2/10.1093/dnares/dsr002/2/dsr00202.jpeg?Expires=2147483647&Signature=JKp8-jQoW~zgRH8YV2JnIaMloKQkRTq08QRtkoetiqwZF6fu~kE9JcwYtGrBFYpQKWP7fMkv0AgGPz-87MxMcQ2WfDlYBURN67muIJK9BMAWk7FQV3-a9cl3J6sxqMjOL-O2Ez1aCTXZsNzkBnYYm~DJQncnwTMRpaexcrjC1aZe9o4G~BHSu-1ZTlLb-ZlPlN1g0PMKadeWnTWQU4Nk3JFzpL9YNSAyKEsgM6zYNtrYO1cyRcLmcT4XXGsZy1GqBTWk~Ds4hmpfXVTZnoND451wLBiAMVe3Z~Z37MoG~p6m50Cs4-vNyc9AwdFRLc2662eU5gX45OcKNS12QBgpjA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5299675&ar=365332&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr00202.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Number of Illumina sequencing reads plotted against percentage of the chloroplast genome mapped for 17 individuals of <em>J. vulgaris</em>.</p></div></div>                            <h3 scrollto-destination=5299676 id="5299676" class="section-title" data-legacy-id=s3b>3.2. Comparison of the sequencing success of cpDNA extracted from chloroplasts with amplified cpDNA using long-range PCR</h3>
<p>For the first lane with cpDNA extracted from isolated chloroplasts, a paired-end run was carried out on the Illumina platform, yielding 582 Mb of sequence with a read length of 32 bp. Of all reads, only 2.1% (391 604 reads) mapped against the chloroplast genome of <em>H. annuus</em>. The obtained reads covered 99.9% of the cp genome of <em>J. vulgaris</em> (Table <a reveal-id="DSR002TB1" data-open="DSR002TB1" class="link link-reveal link-table xref-fig">1</a>, Fig. <a reveal-id="DSR002F3" data-open="DSR002F3" class="link link-reveal link-table xref-fig">3</a>). The average coverage was 83 with a coefficient of variation of 0.34 (Fig. <a reveal-id="DSR002F4" data-open="DSR002F4" class="link link-reveal link-table xref-fig">4</a>). </p>                        <a id="5299678" scrollto-destination="5299678"></a>
<div id="DSR002F3" data-id="DSR002F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/2/10.1093/dnares/dsr002/2/m_dsr00203.jpeg?Expires=2147483647&Signature=uFLc4LCKqFAzUD~8LL~a9dM7iDmoV2uZoCoxKveoA8CxmmH5otLrly8NESfRrL8zJJ8Qg2DwAc9wcwqj6qdRgVa84FAVauoVHqhxM~iYrPS0vStu8bipK7MpHt~vCPpT7yEsTLQQMUKeiBaYS~o4iQDfcNEgrBGImJXMoJu89u08wW2ofBNo7A0DCZeeOSzkPgJvfSgp8~o7UAIDs5uSxdj6v~-aRFW9BOZ7Ka2n5J5XypckZ1ZC5cBfNMdawG73QOop~IO~8wcfvUYgYVn3LgZd9Hp1CGxkjTQASW2haE1KFyt4hT~XjZ4e3emnGDqGM~5p8G5MrtGFf8xTAwia6w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Coverage of the chloroplast genome amplified with a long-range PCR for 16 individuals of J. vulgaris." path-from-xml="dsr00203.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5299678/dsr00203.jpeg" path-from-xml="dsr00203.jpeg" target="_blank">View large</a><a section="5299678" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/2/10.1093/dnares/dsr002/2/dsr00203.jpeg?Expires=2147483647&Signature=efVidovWFnNFTEF8uRv~75gSuLAWBXNCsQqlnIjUe~dlZOXZH~bHcLEfb0mue~I59LEWLUz3ryYaQgkxJkQ7LlxCNeh~oBR8zmN183mhvY0wg6hruAt4NviZ2I0KgG-tTUfNGUCaSJaOgXtl33HN75KEXBuDwA8sqx~gQQmYLVQUV87MbLMGUL5kYwWn6S8gp39mlznt6HGvjlbFwgTBUqy7SSHNCh7iDeegnCng1zUZSsu8~L6pkDPZVoxrfn6Y-XJWWYF6LiEbP9Wy5igAy0by0F2VZ-yJQs9gLnj83en6zzOqWXuT1bVh5XRmDTCoK4XOAcbnw-qKfnxTfvWKdQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5299678&ar=365332&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr00203.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Coverage of the chloroplast genome amplified with a long-range PCR for 16 individuals of <em>J. vulgaris</em>.</p></div></div><div content-id="DSR002F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/2/10.1093/dnares/dsr002/2/m_dsr00203.jpeg?Expires=2147483647&Signature=uFLc4LCKqFAzUD~8LL~a9dM7iDmoV2uZoCoxKveoA8CxmmH5otLrly8NESfRrL8zJJ8Qg2DwAc9wcwqj6qdRgVa84FAVauoVHqhxM~iYrPS0vStu8bipK7MpHt~vCPpT7yEsTLQQMUKeiBaYS~o4iQDfcNEgrBGImJXMoJu89u08wW2ofBNo7A0DCZeeOSzkPgJvfSgp8~o7UAIDs5uSxdj6v~-aRFW9BOZ7Ka2n5J5XypckZ1ZC5cBfNMdawG73QOop~IO~8wcfvUYgYVn3LgZd9Hp1CGxkjTQASW2haE1KFyt4hT~XjZ4e3emnGDqGM~5p8G5MrtGFf8xTAwia6w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Coverage of the chloroplast genome amplified with a long-range PCR for 16 individuals of J. vulgaris." path-from-xml="dsr00203.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5299678/dsr00203.jpeg" path-from-xml="dsr00203.jpeg" target="_blank">View large</a><a section="5299678" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/2/10.1093/dnares/dsr002/2/dsr00203.jpeg?Expires=2147483647&Signature=efVidovWFnNFTEF8uRv~75gSuLAWBXNCsQqlnIjUe~dlZOXZH~bHcLEfb0mue~I59LEWLUz3ryYaQgkxJkQ7LlxCNeh~oBR8zmN183mhvY0wg6hruAt4NviZ2I0KgG-tTUfNGUCaSJaOgXtl33HN75KEXBuDwA8sqx~gQQmYLVQUV87MbLMGUL5kYwWn6S8gp39mlznt6HGvjlbFwgTBUqy7SSHNCh7iDeegnCng1zUZSsu8~L6pkDPZVoxrfn6Y-XJWWYF6LiEbP9Wy5igAy0by0F2VZ-yJQs9gLnj83en6zzOqWXuT1bVh5XRmDTCoK4XOAcbnw-qKfnxTfvWKdQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5299678&ar=365332&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr00203.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Coverage of the chloroplast genome amplified with a long-range PCR for 16 individuals of <em>J. vulgaris</em>.</p></div></div>                        <a id="5299679" scrollto-destination="5299679"></a>
<div id="DSR002F4" data-id="DSR002F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/2/10.1093/dnares/dsr002/2/m_dsr00204.jpeg?Expires=2147483647&Signature=0MDZY9FVuhDuPLyzGvQIZALnwRSL2nmart12wakyqurXygY7OhRvjMG602pBSEIbnHofPJCnoZAhX-wONQmKifW8oH~xWWiDtUSANrmdkQ~oPLM8~CupsLk0SjHsqJf2ZXNUr6ayncyhPdvI59mQz9E5oySEONwrpm7LOucpM26WjKNWxlFLaLBjw0Pxm7qR7CpuVkr3TyiHdFCcBp~qhigr3QVF8S7Mx1-njoFAeuWMczUNWdVKNLM0-9nA1HDNbWkdpf91CzQeeKmUyT8w8Y3YXlWoYYW3PIwF6bdrqVslgaO1nwWf8uxO2u4REc9DWZdo436tKi21BGDtANMU-w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(A) Whole chloroplast genome coverage plotted for individual 17 of J. vulgaris, of which DNA was obtained by using the chloroplast extraction method. (B) Whole chloroplast genome coverage plotted for 16 individuals of J. vulgaris run in two lanes total, of which DNA was obtained by using the long-range PCR method." path-from-xml="dsr00204.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5299679/dsr00204.jpeg" path-from-xml="dsr00204.jpeg" target="_blank">View large</a><a section="5299679" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/2/10.1093/dnares/dsr002/2/dsr00204.jpeg?Expires=2147483647&Signature=hUGJD0R9ovZMjBJfSVVTl1g82tSKwZBSe8YKAah7YxhEUz4ngT8uvcKvtKtxsMJvRVjdICwJdZz4WiyuHC7Tp-dmZLeysvgi~HWn1JHuaq~GTnojJLNM56QRJSm38SN~HaR6ZaLHuejmT6RnFOKxaOM~u1K3M1~AJzXrSSmwJGDqybt6ydtJcKdY8vKYwIwHLyiaym99~crZwfkDkUY8tcAJ-5WT~Ziyv3ROhNWmmsGCuFVPIFqmKcZzCJ64L3KQIE2GhMXk~3JgPjaQiF9r5~Icag0gqYKEKAmZG7Ve259BEFnNav5wa5goJSdfQf~uci4uftefFvy4kgQIwzkbnw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5299679&ar=365332&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr00204.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(<strong>A</strong>) Whole chloroplast genome coverage plotted for individual 17 of <em>J. vulgaris</em>, of which DNA was obtained by using the chloroplast extraction method. (<strong>B</strong>) Whole chloroplast genome coverage plotted for 16 individuals of <em>J. vulgaris</em> run in two lanes total, of which DNA was obtained by using the long-range PCR method.</p></div></div><div content-id="DSR002F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/2/10.1093/dnares/dsr002/2/m_dsr00204.jpeg?Expires=2147483647&Signature=0MDZY9FVuhDuPLyzGvQIZALnwRSL2nmart12wakyqurXygY7OhRvjMG602pBSEIbnHofPJCnoZAhX-wONQmKifW8oH~xWWiDtUSANrmdkQ~oPLM8~CupsLk0SjHsqJf2ZXNUr6ayncyhPdvI59mQz9E5oySEONwrpm7LOucpM26WjKNWxlFLaLBjw0Pxm7qR7CpuVkr3TyiHdFCcBp~qhigr3QVF8S7Mx1-njoFAeuWMczUNWdVKNLM0-9nA1HDNbWkdpf91CzQeeKmUyT8w8Y3YXlWoYYW3PIwF6bdrqVslgaO1nwWf8uxO2u4REc9DWZdo436tKi21BGDtANMU-w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(A) Whole chloroplast genome coverage plotted for individual 17 of J. vulgaris, of which DNA was obtained by using the chloroplast extraction method. (B) Whole chloroplast genome coverage plotted for 16 individuals of J. vulgaris run in two lanes total, of which DNA was obtained by using the long-range PCR method." path-from-xml="dsr00204.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5299679/dsr00204.jpeg" path-from-xml="dsr00204.jpeg" target="_blank">View large</a><a section="5299679" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/2/10.1093/dnares/dsr002/2/dsr00204.jpeg?Expires=2147483647&Signature=hUGJD0R9ovZMjBJfSVVTl1g82tSKwZBSe8YKAah7YxhEUz4ngT8uvcKvtKtxsMJvRVjdICwJdZz4WiyuHC7Tp-dmZLeysvgi~HWn1JHuaq~GTnojJLNM56QRJSm38SN~HaR6ZaLHuejmT6RnFOKxaOM~u1K3M1~AJzXrSSmwJGDqybt6ydtJcKdY8vKYwIwHLyiaym99~crZwfkDkUY8tcAJ-5WT~Ziyv3ROhNWmmsGCuFVPIFqmKcZzCJ64L3KQIE2GhMXk~3JgPjaQiF9r5~Icag0gqYKEKAmZG7Ve259BEFnNav5wa5goJSdfQf~uci4uftefFvy4kgQIwzkbnw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5299679&ar=365332&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr00204.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(<strong>A</strong>) Whole chloroplast genome coverage plotted for individual 17 of <em>J. vulgaris</em>, of which DNA was obtained by using the chloroplast extraction method. (<strong>B</strong>) Whole chloroplast genome coverage plotted for 16 individuals of <em>J. vulgaris</em> run in two lanes total, of which DNA was obtained by using the long-range PCR method.</p></div></div><p>For the other two lanes, containing long-range PCR products of 15 individuals in one lane and the long-range PCR products of one individual in a separate lane, a single-end run was carried out on the Illumina platform. This run yielded reads of 35 bp resulting in 339 and 388 Mb of sequence, respectively. For both lanes, more than 99.9% of the reads (96 894 177 and 11 075 400, respectively) mapped against the chloroplast genome of <em>H. annuus</em>. In both lanes, the reads obtained covered more than 99.9% of the cp genome of <em>J. vulgaris</em> (Table <a reveal-id="DSR002TB1" data-open="DSR002TB1" class="link link-reveal link-table xref-fig">1</a>, Fig. <a reveal-id="DSR002F3" data-open="DSR002F3" class="link link-reveal link-table xref-fig">3</a>). The average coverage obtained for both lanes combined was 4920×. Average coverage varied largely between primer pairs, with average coverage ranging from 542× for the lowest to 19 755× for the highest primer pair (Fig. <a reveal-id="DSR002F4" data-open="DSR002F4" class="link link-reveal link-table xref-fig">4</a>). The average coefficient of variation of coverage within primer pairs, averaged over all primer pairs, is 1.04 (Fig. <a reveal-id="DSR002F4" data-open="DSR002F4" class="link link-reveal link-table xref-fig">4</a>). In summary, the variation was three times higher than that obtained with the direct cpDNA extraction.</p><p>However, extraction of chloroplasts and subsequent extraction of DNA from these chloroplasts were not very efficient for sequencing the complete chloroplast genome. The cpDNA extract still contained around 98% of non-cpDNA. The low efficiency of the chloroplast extraction method might be due to the fact that (nuclear) DNA sticks to the surface of the chloroplast<sup><a reveal-id="DSR002C31" data-open="DSR002C31" class="link link-ref link-reveal xref-bibr">31</a></sup> or to a shortage of DNAases to remove DNA in the intact chloroplast solution. Furthermore, the low efficiency can be caused by poor lysis of the chloroplasts. In contrast, the cpDNA amplified with a long-range PCR contained less than 1% non-cpDNA. Apparently, the long-range PCR worked very efficiently in <em>J. vulgaris</em>, and the results were much better than the results obtained with the same method for <em>Pinus</em> cpDNA sequencing<sup><a reveal-id="DSR002C42" data-open="DSR002C42" class="link link-ref link-reveal xref-bibr">42</a></sup> where non-cpDNA ranged from 19% to 24%.</p><p>Although the number of cpDNA reads obtained with the chloroplast extraction method was far lower than that obtained with the long-range PCR methods, the variation in coverage over the total chloroplast genome was approximately three times lower (Fig. <a reveal-id="DSR002F4" data-open="DSR002F4" class="link link-reveal link-table xref-fig">4</a>). Moreover, the variation in the coverage of the long-range PCR products was primer-dependent (Fig. <a reveal-id="DSR002F4" data-open="DSR002F4" class="link link-reveal link-table xref-fig">4</a>B). Despite the higher variation in coverage, using the long-range PCR products as templates for Illumina sequencing was far more efficient than using cpDNA directly. Moreover, the cpDNA extraction method proved to be cumbersome because we needed 30 g of fresh material per individual.</p>                            <h3 scrollto-destination=5299683 id="5299683" class="section-title" data-legacy-id=s3c>3.3. Comparison of the chloroplast DNA of J. vulgaris with other Asteraceae genomes analysed</h3>
<p>When comparing the full chloroplast genome of <em>J. vulgaris</em> with all complete Asteraceae chloroplast genomes (including those from <em>G. abyssinica</em>, <em>H. annuus</em>, <em>L. sativa</em> and <em>P. argentatum</em>), a few regions (<em>trnS-trnC</em> and <em>trnE-rpoB</em>) could not be aligned because these regions were absent in <em>P. argentatum</em>, and most other regions showed almost no sequence divergence. Regions that could be aligned and that showed moderate sequence divergence between these five species are listed in Table <a reveal-id="DSR002TB2" data-open="DSR002TB2" class="link link-reveal link-table xref-fig">2</a>. Five markers (<em>ndhC-trnV</em>, <em>ndhC-atpE</em>, <em>rps18-rpl20</em>, <em>clpP</em> and <em>psbM-trnD</em>) contained parsimony-informative characters higher than 2% and contained equally high phylogenetic information when compared with other phylogenetic markers that are frequently applied among Asteraceae species such as <em>trnL-trnF</em> (6.9%), <em>trnH-psbA</em> (1.7%), <em>rbcL</em> (1.4%), <em>rps16</em> (0.5%) and <em>ndhF</em> (0.4%). In Fig. <a reveal-id="DSR002F5" data-open="DSR002F5" class="link link-reveal link-table xref-fig">5</a>, the corresponding single MPTs are depicted. </p>                        <a id="5299685" scrollto-destination="5299685"></a>
<div content-id="DSR002TB2" class="table-modal"><div class="table-wrap"><div id="DSR002TB2" class="table-wrap-title" data-id="DSR002TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Promising regions identified for molecular phylogenetic studies of Asteraceae by comparison of the full chloroplast genomes of <em>G. abyssinica</em>, <em>H. annuus</em>, <em>Jacobaea vulgaris</em>, <em>L. sativa</em> and <em>P. argentatum</em></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Region </th><th>Length (bp) </th><th>Tree length </th><th>CI </th><th>RI length </th><th>Pars. inf. char. (%) </th><th>Topologies gene versus species tree </th></tr></thead><tbody><tr><td><em>trnL-trnF</em><sup>a</sup> </td><td>360 </td><td>100 </td><td>0.91 </td><td>0.64 </td><td>6.9 </td><td>Incongruent </td></tr><tr><td><em>ndhC-trnV</em> </td><td>1189 </td><td>520 </td><td>0.89 </td><td>0.88 </td><td>4 </td><td>Congruent </td></tr><tr><td><em>ndhC-atpE</em> </td><td>2376 </td><td>665 </td><td>0.96 </td><td>0.75 </td><td>3.5 </td><td>Congruent </td></tr><tr><td><em>rps18-rpl20</em> </td><td>282 </td><td>50 </td><td>0.96 </td><td>0.78 </td><td>3 </td><td>Congruent </td></tr><tr><td><em>clpP</em> </td><td>889 </td><td>181 </td><td>0.97 </td><td>0.79 </td><td>2.6 </td><td>Incongruent </td></tr><tr><td><em>psbM-trnD</em> </td><td>800 </td><td>114 </td><td>0.92 </td><td>0.55 </td><td>2.5 </td><td>Incongruent </td></tr><tr><td><em>petN-psbM</em> </td><td>569 </td><td>92 </td><td>0.97 </td><td>0.83 </td><td>2 </td><td>Congruent </td></tr><tr><td><em>rps8-rps14</em> </td><td>219 </td><td>29 </td><td>0.96 </td><td>0.75 </td><td>2 </td><td>Incongruent </td></tr><tr><td><em>ycf1</em> </td><td>5811 </td><td>878 </td><td>0.94 </td><td>0.59 </td><td>2 </td><td>Congruent </td></tr><tr><td><em>ycf3-trnS</em> </td><td>1075 </td><td>232 </td><td>0.76 </td><td>0.67 </td><td>2 </td><td>Congruent </td></tr><tr><td>combined regions </td><td>40 449 </td><td>7719 </td><td>0.97 </td><td>0.62 </td><td>1.8 </td><td>Congruent </td></tr><tr><td><em>ndhA</em> </td><td>2317 </td><td>208 </td><td>0.94 </td><td>0.70 </td><td>1.7 </td><td>Congruent </td></tr><tr><td><em>trnH-psbA</em><sup>a</sup> </td><td>1571 </td><td>172 </td><td>0.92 </td><td>0.52 </td><td>1.7 </td><td>Congruent </td></tr><tr><td><em>petD</em> </td><td>1266 </td><td>108 </td><td>0.97 </td><td>0.86 </td><td>1.6 </td><td>Congruent </td></tr><tr><td><em>rbcL</em><sup>a</sup> </td><td>1458 </td><td>96 </td><td>0.95 </td><td>0.76 </td><td>1.4 </td><td>Congruent </td></tr><tr><td><em>petB</em> </td><td>1490 </td><td>115 </td><td>0.96 </td><td>0.75 </td><td>1.3 </td><td>Congruent </td></tr><tr><td><em>ndhI</em> </td><td>547 </td><td>241 </td><td>0.95 </td><td>0.83 </td><td>1 </td><td>Congruent </td></tr><tr><td><em>rps8-rps3</em> </td><td>2451 </td><td>262 </td><td>0.94 </td><td>0.50 </td><td>1 </td><td>Congruent </td></tr><tr><td><em>rps15</em> </td><td>338 </td><td>27 </td><td>0.93 </td><td>0.50 </td><td>1 </td><td>Incongruent </td></tr><tr><td><em>rpoC1</em> </td><td>780 </td><td>82 </td><td>0.97 </td><td>0.80 </td><td>1 </td><td>Congruent </td></tr><tr><td><em>psbB</em> </td><td>1561 </td><td>78 </td><td>0.99 </td><td>0.93 </td><td>0.8 </td><td>Congruent </td></tr><tr><td><em>rpoC2</em> </td><td>4609 </td><td>260 </td><td>0.97 </td><td>0.81 </td><td>0.8 </td><td>Congruent </td></tr><tr><td><em>ndhG</em> </td><td>540 </td><td>31 </td><td>1.00 </td><td>1.00 </td><td>0.7 </td><td>Congruent </td></tr><tr><td><em>rpoB</em> </td><td>3606 </td><td>133 </td><td>0.97 </td><td>0.83 </td><td>0.6 </td><td>Congruent </td></tr><tr><td><em>rps16</em><sup>a</sup> </td><td>1159 </td><td>101 </td><td>0.99 </td><td>0.83 </td><td>0.5 </td><td>Congruent </td></tr><tr><td><em>cemA</em> </td><td>690 </td><td>47 </td><td>0.80 </td><td>0.75 </td><td>0.4 </td><td>Congruent </td></tr><tr><td><em>psaC</em> </td><td>264 </td><td>10 </td><td>1.00 </td><td>1.00 </td><td>0.4 </td><td>Congruent </td></tr><tr><td><em>ndhF</em><sup>a</sup> </td><td>2232 </td><td>156 </td><td>0.98 </td><td>0.67 </td><td>0.4 </td><td>Congruent </td></tr></tbody></table></div><div content-id="DSR002TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Region </th><th>Length (bp) </th><th>Tree length </th><th>CI </th><th>RI length </th><th>Pars. inf. char. (%) </th><th>Topologies gene versus species tree </th></tr></thead><tbody><tr><td><em>trnL-trnF</em><sup>a</sup> </td><td>360 </td><td>100 </td><td>0.91 </td><td>0.64 </td><td>6.9 </td><td>Incongruent </td></tr><tr><td><em>ndhC-trnV</em> </td><td>1189 </td><td>520 </td><td>0.89 </td><td>0.88 </td><td>4 </td><td>Congruent </td></tr><tr><td><em>ndhC-atpE</em> </td><td>2376 </td><td>665 </td><td>0.96 </td><td>0.75 </td><td>3.5 </td><td>Congruent </td></tr><tr><td><em>rps18-rpl20</em> </td><td>282 </td><td>50 </td><td>0.96 </td><td>0.78 </td><td>3 </td><td>Congruent </td></tr><tr><td><em>clpP</em> </td><td>889 </td><td>181 </td><td>0.97 </td><td>0.79 </td><td>2.6 </td><td>Incongruent </td></tr><tr><td><em>psbM-trnD</em> </td><td>800 </td><td>114 </td><td>0.92 </td><td>0.55 </td><td>2.5 </td><td>Incongruent </td></tr><tr><td><em>petN-psbM</em> </td><td>569 </td><td>92 </td><td>0.97 </td><td>0.83 </td><td>2 </td><td>Congruent </td></tr><tr><td><em>rps8-rps14</em> </td><td>219 </td><td>29 </td><td>0.96 </td><td>0.75 </td><td>2 </td><td>Incongruent </td></tr><tr><td><em>ycf1</em> </td><td>5811 </td><td>878 </td><td>0.94 </td><td>0.59 </td><td>2 </td><td>Congruent </td></tr><tr><td><em>ycf3-trnS</em> </td><td>1075 </td><td>232 </td><td>0.76 </td><td>0.67 </td><td>2 </td><td>Congruent </td></tr><tr><td>combined regions </td><td>40 449 </td><td>7719 </td><td>0.97 </td><td>0.62 </td><td>1.8 </td><td>Congruent </td></tr><tr><td><em>ndhA</em> </td><td>2317 </td><td>208 </td><td>0.94 </td><td>0.70 </td><td>1.7 </td><td>Congruent </td></tr><tr><td><em>trnH-psbA</em><sup>a</sup> </td><td>1571 </td><td>172 </td><td>0.92 </td><td>0.52 </td><td>1.7 </td><td>Congruent </td></tr><tr><td><em>petD</em> </td><td>1266 </td><td>108 </td><td>0.97 </td><td>0.86 </td><td>1.6 </td><td>Congruent </td></tr><tr><td><em>rbcL</em><sup>a</sup> </td><td>1458 </td><td>96 </td><td>0.95 </td><td>0.76 </td><td>1.4 </td><td>Congruent </td></tr><tr><td><em>petB</em> </td><td>1490 </td><td>115 </td><td>0.96 </td><td>0.75 </td><td>1.3 </td><td>Congruent </td></tr><tr><td><em>ndhI</em> </td><td>547 </td><td>241 </td><td>0.95 </td><td>0.83 </td><td>1 </td><td>Congruent </td></tr><tr><td><em>rps8-rps3</em> </td><td>2451 </td><td>262 </td><td>0.94 </td><td>0.50 </td><td>1 </td><td>Congruent </td></tr><tr><td><em>rps15</em> </td><td>338 </td><td>27 </td><td>0.93 </td><td>0.50 </td><td>1 </td><td>Incongruent </td></tr><tr><td><em>rpoC1</em> </td><td>780 </td><td>82 </td><td>0.97 </td><td>0.80 </td><td>1 </td><td>Congruent </td></tr><tr><td><em>psbB</em> </td><td>1561 </td><td>78 </td><td>0.99 </td><td>0.93 </td><td>0.8 </td><td>Congruent </td></tr><tr><td><em>rpoC2</em> </td><td>4609 </td><td>260 </td><td>0.97 </td><td>0.81 </td><td>0.8 </td><td>Congruent </td></tr><tr><td><em>ndhG</em> </td><td>540 </td><td>31 </td><td>1.00 </td><td>1.00 </td><td>0.7 </td><td>Congruent </td></tr><tr><td><em>rpoB</em> </td><td>3606 </td><td>133 </td><td>0.97 </td><td>0.83 </td><td>0.6 </td><td>Congruent </td></tr><tr><td><em>rps16</em><sup>a</sup> </td><td>1159 </td><td>101 </td><td>0.99 </td><td>0.83 </td><td>0.5 </td><td>Congruent </td></tr><tr><td><em>cemA</em> </td><td>690 </td><td>47 </td><td>0.80 </td><td>0.75 </td><td>0.4 </td><td>Congruent </td></tr><tr><td><em>psaC</em> </td><td>264 </td><td>10 </td><td>1.00 </td><td>1.00 </td><td>0.4 </td><td>Congruent </td></tr><tr><td><em>ndhF</em><sup>a</sup> </td><td>2232 </td><td>156 </td><td>0.98 </td><td>0.67 </td><td>0.4 </td><td>Congruent </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>The CI and RI were calculated with autapomorphic characters excluded.</p></div></div><div content-id=""><div class="fn"><p><sup>a</sup>Commonly used phylogenetic markers included for comparison.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5299685" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR002TB2" class="table-wrap-title" data-id="DSR002TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Promising regions identified for molecular phylogenetic studies of Asteraceae by comparison of the full chloroplast genomes of <em>G. abyssinica</em>, <em>H. annuus</em>, <em>Jacobaea vulgaris</em>, <em>L. sativa</em> and <em>P. argentatum</em></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Region </th><th>Length (bp) </th><th>Tree length </th><th>CI </th><th>RI length </th><th>Pars. inf. char. (%) </th><th>Topologies gene versus species tree </th></tr></thead><tbody><tr><td><em>trnL-trnF</em><sup>a</sup> </td><td>360 </td><td>100 </td><td>0.91 </td><td>0.64 </td><td>6.9 </td><td>Incongruent </td></tr><tr><td><em>ndhC-trnV</em> </td><td>1189 </td><td>520 </td><td>0.89 </td><td>0.88 </td><td>4 </td><td>Congruent </td></tr><tr><td><em>ndhC-atpE</em> </td><td>2376 </td><td>665 </td><td>0.96 </td><td>0.75 </td><td>3.5 </td><td>Congruent </td></tr><tr><td><em>rps18-rpl20</em> </td><td>282 </td><td>50 </td><td>0.96 </td><td>0.78 </td><td>3 </td><td>Congruent </td></tr><tr><td><em>clpP</em> </td><td>889 </td><td>181 </td><td>0.97 </td><td>0.79 </td><td>2.6 </td><td>Incongruent </td></tr><tr><td><em>psbM-trnD</em> </td><td>800 </td><td>114 </td><td>0.92 </td><td>0.55 </td><td>2.5 </td><td>Incongruent </td></tr><tr><td><em>petN-psbM</em> </td><td>569 </td><td>92 </td><td>0.97 </td><td>0.83 </td><td>2 </td><td>Congruent </td></tr><tr><td><em>rps8-rps14</em> </td><td>219 </td><td>29 </td><td>0.96 </td><td>0.75 </td><td>2 </td><td>Incongruent </td></tr><tr><td><em>ycf1</em> </td><td>5811 </td><td>878 </td><td>0.94 </td><td>0.59 </td><td>2 </td><td>Congruent </td></tr><tr><td><em>ycf3-trnS</em> </td><td>1075 </td><td>232 </td><td>0.76 </td><td>0.67 </td><td>2 </td><td>Congruent </td></tr><tr><td>combined regions </td><td>40 449 </td><td>7719 </td><td>0.97 </td><td>0.62 </td><td>1.8 </td><td>Congruent </td></tr><tr><td><em>ndhA</em> </td><td>2317 </td><td>208 </td><td>0.94 </td><td>0.70 </td><td>1.7 </td><td>Congruent </td></tr><tr><td><em>trnH-psbA</em><sup>a</sup> </td><td>1571 </td><td>172 </td><td>0.92 </td><td>0.52 </td><td>1.7 </td><td>Congruent </td></tr><tr><td><em>petD</em> </td><td>1266 </td><td>108 </td><td>0.97 </td><td>0.86 </td><td>1.6 </td><td>Congruent </td></tr><tr><td><em>rbcL</em><sup>a</sup> </td><td>1458 </td><td>96 </td><td>0.95 </td><td>0.76 </td><td>1.4 </td><td>Congruent </td></tr><tr><td><em>petB</em> </td><td>1490 </td><td>115 </td><td>0.96 </td><td>0.75 </td><td>1.3 </td><td>Congruent </td></tr><tr><td><em>ndhI</em> </td><td>547 </td><td>241 </td><td>0.95 </td><td>0.83 </td><td>1 </td><td>Congruent </td></tr><tr><td><em>rps8-rps3</em> </td><td>2451 </td><td>262 </td><td>0.94 </td><td>0.50 </td><td>1 </td><td>Congruent </td></tr><tr><td><em>rps15</em> </td><td>338 </td><td>27 </td><td>0.93 </td><td>0.50 </td><td>1 </td><td>Incongruent </td></tr><tr><td><em>rpoC1</em> </td><td>780 </td><td>82 </td><td>0.97 </td><td>0.80 </td><td>1 </td><td>Congruent </td></tr><tr><td><em>psbB</em> </td><td>1561 </td><td>78 </td><td>0.99 </td><td>0.93 </td><td>0.8 </td><td>Congruent </td></tr><tr><td><em>rpoC2</em> </td><td>4609 </td><td>260 </td><td>0.97 </td><td>0.81 </td><td>0.8 </td><td>Congruent </td></tr><tr><td><em>ndhG</em> </td><td>540 </td><td>31 </td><td>1.00 </td><td>1.00 </td><td>0.7 </td><td>Congruent </td></tr><tr><td><em>rpoB</em> </td><td>3606 </td><td>133 </td><td>0.97 </td><td>0.83 </td><td>0.6 </td><td>Congruent </td></tr><tr><td><em>rps16</em><sup>a</sup> </td><td>1159 </td><td>101 </td><td>0.99 </td><td>0.83 </td><td>0.5 </td><td>Congruent </td></tr><tr><td><em>cemA</em> </td><td>690 </td><td>47 </td><td>0.80 </td><td>0.75 </td><td>0.4 </td><td>Congruent </td></tr><tr><td><em>psaC</em> </td><td>264 </td><td>10 </td><td>1.00 </td><td>1.00 </td><td>0.4 </td><td>Congruent </td></tr><tr><td><em>ndhF</em><sup>a</sup> </td><td>2232 </td><td>156 </td><td>0.98 </td><td>0.67 </td><td>0.4 </td><td>Congruent </td></tr></tbody></table></div><div content-id="DSR002TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Region </th><th>Length (bp) </th><th>Tree length </th><th>CI </th><th>RI length </th><th>Pars. inf. char. (%) </th><th>Topologies gene versus species tree </th></tr></thead><tbody><tr><td><em>trnL-trnF</em><sup>a</sup> </td><td>360 </td><td>100 </td><td>0.91 </td><td>0.64 </td><td>6.9 </td><td>Incongruent </td></tr><tr><td><em>ndhC-trnV</em> </td><td>1189 </td><td>520 </td><td>0.89 </td><td>0.88 </td><td>4 </td><td>Congruent </td></tr><tr><td><em>ndhC-atpE</em> </td><td>2376 </td><td>665 </td><td>0.96 </td><td>0.75 </td><td>3.5 </td><td>Congruent </td></tr><tr><td><em>rps18-rpl20</em> </td><td>282 </td><td>50 </td><td>0.96 </td><td>0.78 </td><td>3 </td><td>Congruent </td></tr><tr><td><em>clpP</em> </td><td>889 </td><td>181 </td><td>0.97 </td><td>0.79 </td><td>2.6 </td><td>Incongruent </td></tr><tr><td><em>psbM-trnD</em> </td><td>800 </td><td>114 </td><td>0.92 </td><td>0.55 </td><td>2.5 </td><td>Incongruent </td></tr><tr><td><em>petN-psbM</em> </td><td>569 </td><td>92 </td><td>0.97 </td><td>0.83 </td><td>2 </td><td>Congruent </td></tr><tr><td><em>rps8-rps14</em> </td><td>219 </td><td>29 </td><td>0.96 </td><td>0.75 </td><td>2 </td><td>Incongruent </td></tr><tr><td><em>ycf1</em> </td><td>5811 </td><td>878 </td><td>0.94 </td><td>0.59 </td><td>2 </td><td>Congruent </td></tr><tr><td><em>ycf3-trnS</em> </td><td>1075 </td><td>232 </td><td>0.76 </td><td>0.67 </td><td>2 </td><td>Congruent </td></tr><tr><td>combined regions </td><td>40 449 </td><td>7719 </td><td>0.97 </td><td>0.62 </td><td>1.8 </td><td>Congruent </td></tr><tr><td><em>ndhA</em> </td><td>2317 </td><td>208 </td><td>0.94 </td><td>0.70 </td><td>1.7 </td><td>Congruent </td></tr><tr><td><em>trnH-psbA</em><sup>a</sup> </td><td>1571 </td><td>172 </td><td>0.92 </td><td>0.52 </td><td>1.7 </td><td>Congruent </td></tr><tr><td><em>petD</em> </td><td>1266 </td><td>108 </td><td>0.97 </td><td>0.86 </td><td>1.6 </td><td>Congruent </td></tr><tr><td><em>rbcL</em><sup>a</sup> </td><td>1458 </td><td>96 </td><td>0.95 </td><td>0.76 </td><td>1.4 </td><td>Congruent </td></tr><tr><td><em>petB</em> </td><td>1490 </td><td>115 </td><td>0.96 </td><td>0.75 </td><td>1.3 </td><td>Congruent </td></tr><tr><td><em>ndhI</em> </td><td>547 </td><td>241 </td><td>0.95 </td><td>0.83 </td><td>1 </td><td>Congruent </td></tr><tr><td><em>rps8-rps3</em> </td><td>2451 </td><td>262 </td><td>0.94 </td><td>0.50 </td><td>1 </td><td>Congruent </td></tr><tr><td><em>rps15</em> </td><td>338 </td><td>27 </td><td>0.93 </td><td>0.50 </td><td>1 </td><td>Incongruent </td></tr><tr><td><em>rpoC1</em> </td><td>780 </td><td>82 </td><td>0.97 </td><td>0.80 </td><td>1 </td><td>Congruent </td></tr><tr><td><em>psbB</em> </td><td>1561 </td><td>78 </td><td>0.99 </td><td>0.93 </td><td>0.8 </td><td>Congruent </td></tr><tr><td><em>rpoC2</em> </td><td>4609 </td><td>260 </td><td>0.97 </td><td>0.81 </td><td>0.8 </td><td>Congruent </td></tr><tr><td><em>ndhG</em> </td><td>540 </td><td>31 </td><td>1.00 </td><td>1.00 </td><td>0.7 </td><td>Congruent </td></tr><tr><td><em>rpoB</em> </td><td>3606 </td><td>133 </td><td>0.97 </td><td>0.83 </td><td>0.6 </td><td>Congruent </td></tr><tr><td><em>rps16</em><sup>a</sup> </td><td>1159 </td><td>101 </td><td>0.99 </td><td>0.83 </td><td>0.5 </td><td>Congruent </td></tr><tr><td><em>cemA</em> </td><td>690 </td><td>47 </td><td>0.80 </td><td>0.75 </td><td>0.4 </td><td>Congruent </td></tr><tr><td><em>psaC</em> </td><td>264 </td><td>10 </td><td>1.00 </td><td>1.00 </td><td>0.4 </td><td>Congruent </td></tr><tr><td><em>ndhF</em><sup>a</sup> </td><td>2232 </td><td>156 </td><td>0.98 </td><td>0.67 </td><td>0.4 </td><td>Congruent </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>The CI and RI were calculated with autapomorphic characters excluded.</p></div></div><div content-id=""><div class="fn"><p><sup>a</sup>Commonly used phylogenetic markers included for comparison.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5299685" target="_blank">View Large</a></div></div>                        <a id="5299686" scrollto-destination="5299686"></a>
<div id="DSR002F5" data-id="DSR002F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/2/10.1093/dnares/dsr002/2/m_dsr00205.jpeg?Expires=2147483647&Signature=GjHgUSOPjiuDYz~DE2u8Id1gVT4724fpWnNYNoYW6F1A0Tlnt-4YI0xVkmKf3QK8o7FlxFZTSQbQC3C7Ak34KJx1cPOefZzf8TLU24smUC5T3-4ndvDaPjJrQy4038foQ2TaZSjFNP4XBwY-dP8~IahP61xLIMLvvz1zwQV3p~sYKbZnMpoUDuNPHTQ7uxOJST7kt4hXd3juXQlqVfJFjJGf02TuXrHGtiwFOdqmjTTVzta0ZcXJyBy5l9CC-PESEgOYHpAyj68b6DwU-B5Fz~igURoYjydaHG6Rv5oOIKQhbGT1CWO-0CTptNvfUvlzdvKKqPMptQSZhZnoKcejRg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylograms derived from maximum parsimony (MP) analysis of alignments of DNA sequences of five different Asteraceae species of a total of 27 individual chloroplast regions indicated below the trees. The phylogram called ‘combined regions’ in the middle is derived from MP analysis of all 27 regions together." path-from-xml="dsr00205.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5299686/dsr00205.jpeg" path-from-xml="dsr00205.jpeg" target="_blank">View large</a><a section="5299686" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/2/10.1093/dnares/dsr002/2/dsr00205.jpeg?Expires=2147483647&Signature=rFgiI5x3UHc9ASk1WcgNed8n3mKCULHdSasbJA43HYYCI~AoqULFQ-oY9wSMOnhshtmMFhkgWBF8rHF-IYORK7kfFiiG5xQbmfFX~~RUmCVXmfh2~WeYL0hsXZF-cOAoMnerTHoVsRun8y3ma2FqsXax~fIebrktm7D1x6X0oafl19GA89c0aY1CtqV1Fy-ajFbuXC58RXJQgWulUBiHSiaBYHwS6c3XaiDTxtRdrhfvw8Rn-GIzyv4vPtztrS20dAHIWSy89beFQ8dfyfsE74B~sDH9zwtbhrbAiZ~HEhqxqcSsMN8fcPb3fHXOdTZmtpEeK2cCOW~HFMPPYiE0Uw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5299686&ar=365332&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr00205.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylograms derived from maximum parsimony (MP) analysis of alignments of DNA sequences of five different Asteraceae species of a total of 27 individual chloroplast regions indicated below the trees. The phylogram called ‘combined regions’ in the middle is derived from MP analysis of all 27 regions together.</p></div></div><div content-id="DSR002F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/2/10.1093/dnares/dsr002/2/m_dsr00205.jpeg?Expires=2147483647&Signature=GjHgUSOPjiuDYz~DE2u8Id1gVT4724fpWnNYNoYW6F1A0Tlnt-4YI0xVkmKf3QK8o7FlxFZTSQbQC3C7Ak34KJx1cPOefZzf8TLU24smUC5T3-4ndvDaPjJrQy4038foQ2TaZSjFNP4XBwY-dP8~IahP61xLIMLvvz1zwQV3p~sYKbZnMpoUDuNPHTQ7uxOJST7kt4hXd3juXQlqVfJFjJGf02TuXrHGtiwFOdqmjTTVzta0ZcXJyBy5l9CC-PESEgOYHpAyj68b6DwU-B5Fz~igURoYjydaHG6Rv5oOIKQhbGT1CWO-0CTptNvfUvlzdvKKqPMptQSZhZnoKcejRg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylograms derived from maximum parsimony (MP) analysis of alignments of DNA sequences of five different Asteraceae species of a total of 27 individual chloroplast regions indicated below the trees. The phylogram called ‘combined regions’ in the middle is derived from MP analysis of all 27 regions together." path-from-xml="dsr00205.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5299686/dsr00205.jpeg" path-from-xml="dsr00205.jpeg" target="_blank">View large</a><a section="5299686" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/2/10.1093/dnares/dsr002/2/dsr00205.jpeg?Expires=2147483647&Signature=rFgiI5x3UHc9ASk1WcgNed8n3mKCULHdSasbJA43HYYCI~AoqULFQ-oY9wSMOnhshtmMFhkgWBF8rHF-IYORK7kfFiiG5xQbmfFX~~RUmCVXmfh2~WeYL0hsXZF-cOAoMnerTHoVsRun8y3ma2FqsXax~fIebrktm7D1x6X0oafl19GA89c0aY1CtqV1Fy-ajFbuXC58RXJQgWulUBiHSiaBYHwS6c3XaiDTxtRdrhfvw8Rn-GIzyv4vPtztrS20dAHIWSy89beFQ8dfyfsE74B~sDH9zwtbhrbAiZ~HEhqxqcSsMN8fcPb3fHXOdTZmtpEeK2cCOW~HFMPPYiE0Uw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5299686&ar=365332&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr00205.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylograms derived from maximum parsimony (MP) analysis of alignments of DNA sequences of five different Asteraceae species of a total of 27 individual chloroplast regions indicated below the trees. The phylogram called ‘combined regions’ in the middle is derived from MP analysis of all 27 regions together.</p></div></div><p>In a former comparison with <em>H. annuus</em> against <em>L. sativa</em><sup><a reveal-id="DSR002C43" data-open="DSR002C43" class="link link-ref link-reveal xref-bibr">43</a></sup> and with <em>H. annuus</em> against <em>G. abyssinica</em>,<sup><a reveal-id="DSR002C44" data-open="DSR002C44" class="link link-ref link-reveal xref-bibr">44</a></sup> the regions <em>ndhC-trnV</em> and <em>clpP</em> were already identified as divergent regions within the Asteraceae. CIs of the newly discovered phylogenetic markers, indicating homoplasy, of the newly discovered markers were all in the same range as the commonly used markers except for <em>ycf3-trnS</em> and <em>cemA</em>, which had slightly lower values. RI values ranged from 0.52 to 0.83 for the commonly used markers and from 0.50 to 1.00 for the newly discovered markers.</p><p>Analysis of all 27 regions combined resulted in a congruent topology with high support for all internal nodes. Gene trees can be incongruent with species trees when evolution of genes and species did not occur congruently.<sup><a reveal-id="DSR002C10" data-open="DSR002C10" class="link link-ref link-reveal xref-bibr">10</a></sup> Gene trees of five regions (<em>trnL-trnF</em>, <em>clpP</em>, <em>psbM-trnD</em>, <em>rps8-rps14</em> and <em>rps15</em>) were found to be incongruent with the generally inferred species tree of the Asteraceae species analysed (Table <a reveal-id="DSR002TB2" data-open="DSR002TB2" class="link link-reveal link-table xref-fig">2</a>; Fig. <a reveal-id="DSR002F5" data-open="DSR002F5" class="link link-reveal link-table xref-fig">5</a>).</p><p>With a length of 150 686 bp, <em>J. vulgaris</em> has the smallest chloroplast genome compared with the four other Asteraceae cp genomes sequenced so far. The length is 2215 bp less than the largest cp genome of <em>P. argentatum</em>. The genome is identical in gene content to <em>H. annuus</em> and <em>L. sativa</em> and differs in gene number with <em>G. abyssinica</em> (which has one gene less) and <em>P. argentatum</em> (which has four genes more). Although the similarity in gene content was high, few non-coding regions showed a high sequence divergence between the five Asteraceae species. A number of regions showing sequence divergence between these species contained a high phylogenetic content compared with the standard applied phylogenetic markers used in the Asteraceae. Those regions seem promising for development of universal primers to further investigate clades in molecular phylogenies of Asteraceae hitherto unresolved. Furthermore, many of these regions are not yet used in angiosperm molecular phylogenetic studies<sup><a reveal-id="DSR002C45" data-open="DSR002C45" class="link link-ref link-reveal xref-bibr">45</a></sup> and seem worthwhile to investigate further.</p>                            <h3 scrollto-destination=5299690 id="5299690" class="section-title" data-legacy-id=s3d>3.4. Detection of polymorphic loci</h3>
                            <h4 scrollto-destination=5299691 id="5299691" class="section-title" data-legacy-id=s3d1>3.4.1. Single-nucleotide polymorphisms</h4>
<p>The 17 individuals of <em>J. vulgaris</em> yielded a total of 32 SNPs (Table <a reveal-id="DSR002TB3" data-open="DSR002TB3" class="link link-reveal link-table xref-fig">3</a>), which is on average one SNP per 4705 bp. In 66% of the cases, an SNP allele was found only in a single individual. Fifty-nine per cent of the SNP polymorphisms where substitutions from a purine to a pyrimidine or vice versa. No SNPs were found in tRNAs (Table <a reveal-id="DSR002TB4" data-open="DSR002TB4" class="link link-reveal link-table xref-fig">4</a>). Within the single-copy region (LSC and SSC), SNPs were almost equally divided over coding DNA (tRNA + exons + genic) (13) and intergenic spacers and introns (19). However, in the coding DNA, on average one SNP every 4573 bp was found compared with one SNP on average for every 2780 bp in intergenic and intron spacers (Table <a reveal-id="DSR002TB4" data-open="DSR002TB4" class="link link-reveal link-table xref-fig">4</a>). Within the genes, two SNPs were located in introns, this is on average one SNP per 3439 bp compared with one SNP per 4811 bp located in coding gene sequences (genes + exons; Table <a reveal-id="DSR002TB4" data-open="DSR002TB4" class="link link-reveal link-table xref-fig">4</a>). Of the 13 SNPs found in coding DNA, three resulted in non-synonymous substitutions (Table <a reveal-id="DSR002TB3" data-open="DSR002TB3" class="link link-reveal link-table xref-fig">3</a>). </p>                        <a id="5299693" scrollto-destination="5299693"></a>
<div content-id="DSR002TB3" class="table-modal"><div class="table-wrap"><div id="DSR002TB3" class="table-wrap-title" data-id="DSR002TB3"><div class="label -label">Table 3.</div><div class="caption"><p>List of positions and variants of SNPs and microsatellites in the chloroplast genome of <em>J. Vulgaris</em>: SNP positions, alleles with the most occurring allele first, frequency of the least occurring allele in 17 individuals of the cp genome of <em>J. vulgaris</em> and region and locus of these SNPs</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Position </th><th>Alleles </th><th>Freq. </th><th>Region </th><th>Locus </th><th>Position </th><th>Alleles </th><th>Freq. </th><th>Region </th><th>Locus </th></tr></thead><tbody><tr><td>165 </td><td>T/A </td><td>0.13 </td><td>Intergenic </td><td><em>trnH-GUG/psbA</em> </td><td>61 436 </td><td>C/T </td><td>0.31 </td><td>Genic </td><td><em>petA</em> </td></tr><tr><td><strong>4032</strong> </td><td><strong>A/C</strong> </td><td><strong>0.06</strong> </td><td><strong>Intergenic</strong> </td><td><strong><em>matK/trnK-UUU</em></strong> </td><td>65 579 </td><td>G/C </td><td>0.06 </td><td>Intergenic </td><td><em>trnP-UGG/psaJ</em> </td></tr><tr><td><strong>5555</strong> </td><td><strong>A/T</strong> </td><td><strong>0.13</strong> </td><td><strong>Intron</strong> </td><td><strong><em>rps16</em></strong> </td><td>66 056 </td><td>T/G </td><td>0.19 </td><td>Intergenic </td><td><em>psaJ/rpl33</em> </td></tr><tr><td><strong>7837</strong> </td><td><strong>A/C</strong> </td><td><strong>0.06</strong> </td><td><strong>Intergenic</strong> </td><td><strong><em>psbK/psbL</em></strong> </td><td>67 055 </td><td>G/A </td><td>0.25 </td><td>Intergenic </td><td><em>Rps18/rpl20</em> </td></tr><tr><td><strong>11 353</strong> </td><td><strong>C/A</strong> </td><td><strong>0.06</strong> </td><td><strong>Intergenic</strong> </td><td><strong><em>trnY-GUA</em>/<em>trnE-UUC</em></strong> </td><td>67 963 </td><td>T/C </td><td>0.69 </td><td>Intergenic </td><td><em>Rpl20/rps12</em> </td></tr><tr><td><strong>18 287</strong> </td><td><strong>A/C</strong> </td><td><strong>0.13</strong> </td><td><strong>Exon</strong> </td><td><strong><em>rpoC1</em></strong> </td><td>69 567** </td><td>T/C </td><td>0.06 </td><td>Exon </td><td><em>clpP</em> </td></tr><tr><td>22 648 </td><td>C/T </td><td>0.06 </td><td>Genic </td><td><em>rpoC2</em> </td><td>70 234 </td><td>T/G </td><td>0.06 </td><td>Intron </td><td><em>clpP</em> </td></tr><tr><td>24 906 </td><td>T/G </td><td>0.38 </td><td>Intergenic </td><td><em>atpI-atpH</em> </td><td>92 417* </td><td>C/T </td><td>0.06 </td><td>Intergenic </td><td><em>trnL-CAA/ndhB</em> </td></tr><tr><td>31 299 </td><td>C/A </td><td>0.06 </td><td>Intergenic </td><td><em>trnT-GGU/psbD</em> </td><td>97 496* </td><td>C/A </td><td>0.06 </td><td>Intergenic </td><td><em>Rps7/ycf15</em> </td></tr><tr><td>39 790 </td><td>A/G </td><td>0.44 </td><td>Genic </td><td><em>psaA</em> </td><td>106 663* </td><td>T/G </td><td>0.06 </td><td>Intergenic </td><td><em>trnR-ACG/trnN-GUU</em> </td></tr><tr><td>39 829 </td><td>G/A </td><td>0.13 </td><td>Genic </td><td><em>psaA</em> </td><td>106 664* </td><td>C/A </td><td>0.06 </td><td>Intergenic </td><td><em>trnR-ACG/trnN-GUU</em> </td></tr><tr><td>43 765 </td><td>C/T </td><td>0.06 </td><td>Intergenic </td><td><em>Ycf3/trnS-GCA</em> </td><td>108 200** </td><td>G/C </td><td>0.25 </td><td>Genic </td><td><em>Ycf1</em> </td></tr><tr><td>47 181 </td><td>G/C </td><td>0.06 </td><td>Intergenic </td><td><em>trnL-UAA/trnF-GAA</em> </td><td><strong>118 779</strong> </td><td><strong>C/G</strong> </td><td><strong>0.06</strong> </td><td><strong>Genic</strong> </td><td><strong><em>ndhD</em></strong> </td></tr><tr><td>49 751 </td><td>C/T </td><td>0.06 </td><td>Genic </td><td><em>ndhC</em> </td><td>123 423 </td><td>A/C </td><td>0.06 </td><td>Intergenic </td><td><em>Rpl32/ndhF</em> </td></tr><tr><td>53 025 </td><td>G/A </td><td>0.06 </td><td>Genic </td><td><em>atpB</em> </td><td>124 027 </td><td>C/T </td><td>0.06 </td><td>Genic </td><td><em>ndhF</em> </td></tr><tr><td><strong>60 245</strong> </td><td><strong>C/T</strong> </td><td><strong>0.06</strong> </td><td><strong>Genic</strong> </td><td><strong><em>cemA</em></strong> </td><td>124 035** </td><td>C/T </td><td>0.06 </td><td>Genic </td><td><em>ndhF</em> </td></tr></tbody></table></div><div content-id="DSR002TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Position </th><th>Alleles </th><th>Freq. </th><th>Region </th><th>Locus </th><th>Position </th><th>Alleles </th><th>Freq. </th><th>Region </th><th>Locus </th></tr></thead><tbody><tr><td>165 </td><td>T/A </td><td>0.13 </td><td>Intergenic </td><td><em>trnH-GUG/psbA</em> </td><td>61 436 </td><td>C/T </td><td>0.31 </td><td>Genic </td><td><em>petA</em> </td></tr><tr><td><strong>4032</strong> </td><td><strong>A/C</strong> </td><td><strong>0.06</strong> </td><td><strong>Intergenic</strong> </td><td><strong><em>matK/trnK-UUU</em></strong> </td><td>65 579 </td><td>G/C </td><td>0.06 </td><td>Intergenic </td><td><em>trnP-UGG/psaJ</em> </td></tr><tr><td><strong>5555</strong> </td><td><strong>A/T</strong> </td><td><strong>0.13</strong> </td><td><strong>Intron</strong> </td><td><strong><em>rps16</em></strong> </td><td>66 056 </td><td>T/G </td><td>0.19 </td><td>Intergenic </td><td><em>psaJ/rpl33</em> </td></tr><tr><td><strong>7837</strong> </td><td><strong>A/C</strong> </td><td><strong>0.06</strong> </td><td><strong>Intergenic</strong> </td><td><strong><em>psbK/psbL</em></strong> </td><td>67 055 </td><td>G/A </td><td>0.25 </td><td>Intergenic </td><td><em>Rps18/rpl20</em> </td></tr><tr><td><strong>11 353</strong> </td><td><strong>C/A</strong> </td><td><strong>0.06</strong> </td><td><strong>Intergenic</strong> </td><td><strong><em>trnY-GUA</em>/<em>trnE-UUC</em></strong> </td><td>67 963 </td><td>T/C </td><td>0.69 </td><td>Intergenic </td><td><em>Rpl20/rps12</em> </td></tr><tr><td><strong>18 287</strong> </td><td><strong>A/C</strong> </td><td><strong>0.13</strong> </td><td><strong>Exon</strong> </td><td><strong><em>rpoC1</em></strong> </td><td>69 567** </td><td>T/C </td><td>0.06 </td><td>Exon </td><td><em>clpP</em> </td></tr><tr><td>22 648 </td><td>C/T </td><td>0.06 </td><td>Genic </td><td><em>rpoC2</em> </td><td>70 234 </td><td>T/G </td><td>0.06 </td><td>Intron </td><td><em>clpP</em> </td></tr><tr><td>24 906 </td><td>T/G </td><td>0.38 </td><td>Intergenic </td><td><em>atpI-atpH</em> </td><td>92 417* </td><td>C/T </td><td>0.06 </td><td>Intergenic </td><td><em>trnL-CAA/ndhB</em> </td></tr><tr><td>31 299 </td><td>C/A </td><td>0.06 </td><td>Intergenic </td><td><em>trnT-GGU/psbD</em> </td><td>97 496* </td><td>C/A </td><td>0.06 </td><td>Intergenic </td><td><em>Rps7/ycf15</em> </td></tr><tr><td>39 790 </td><td>A/G </td><td>0.44 </td><td>Genic </td><td><em>psaA</em> </td><td>106 663* </td><td>T/G </td><td>0.06 </td><td>Intergenic </td><td><em>trnR-ACG/trnN-GUU</em> </td></tr><tr><td>39 829 </td><td>G/A </td><td>0.13 </td><td>Genic </td><td><em>psaA</em> </td><td>106 664* </td><td>C/A </td><td>0.06 </td><td>Intergenic </td><td><em>trnR-ACG/trnN-GUU</em> </td></tr><tr><td>43 765 </td><td>C/T </td><td>0.06 </td><td>Intergenic </td><td><em>Ycf3/trnS-GCA</em> </td><td>108 200** </td><td>G/C </td><td>0.25 </td><td>Genic </td><td><em>Ycf1</em> </td></tr><tr><td>47 181 </td><td>G/C </td><td>0.06 </td><td>Intergenic </td><td><em>trnL-UAA/trnF-GAA</em> </td><td><strong>118 779</strong> </td><td><strong>C/G</strong> </td><td><strong>0.06</strong> </td><td><strong>Genic</strong> </td><td><strong><em>ndhD</em></strong> </td></tr><tr><td>49 751 </td><td>C/T </td><td>0.06 </td><td>Genic </td><td><em>ndhC</em> </td><td>123 423 </td><td>A/C </td><td>0.06 </td><td>Intergenic </td><td><em>Rpl32/ndhF</em> </td></tr><tr><td>53 025 </td><td>G/A </td><td>0.06 </td><td>Genic </td><td><em>atpB</em> </td><td>124 027 </td><td>C/T </td><td>0.06 </td><td>Genic </td><td><em>ndhF</em> </td></tr><tr><td><strong>60 245</strong> </td><td><strong>C/T</strong> </td><td><strong>0.06</strong> </td><td><strong>Genic</strong> </td><td><strong><em>cemA</em></strong> </td><td>124 035** </td><td>C/T </td><td>0.06 </td><td>Genic </td><td><em>ndhF</em> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>SNPs that were tested for multiple individuals with high-resolution melting are indicated by bold typeface.</p></div></div><div content-id=""><div class="fn"><p>*SNPs located in the IR.</p></div></div><div content-id=""><div class="fn"><p>**Non-synonymous substitutions.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5299693" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR002TB3" class="table-wrap-title" data-id="DSR002TB3"><div class="label -label">Table 3.</div><div class="caption"><p>List of positions and variants of SNPs and microsatellites in the chloroplast genome of <em>J. Vulgaris</em>: SNP positions, alleles with the most occurring allele first, frequency of the least occurring allele in 17 individuals of the cp genome of <em>J. vulgaris</em> and region and locus of these SNPs</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Position </th><th>Alleles </th><th>Freq. </th><th>Region </th><th>Locus </th><th>Position </th><th>Alleles </th><th>Freq. </th><th>Region </th><th>Locus </th></tr></thead><tbody><tr><td>165 </td><td>T/A </td><td>0.13 </td><td>Intergenic </td><td><em>trnH-GUG/psbA</em> </td><td>61 436 </td><td>C/T </td><td>0.31 </td><td>Genic </td><td><em>petA</em> </td></tr><tr><td><strong>4032</strong> </td><td><strong>A/C</strong> </td><td><strong>0.06</strong> </td><td><strong>Intergenic</strong> </td><td><strong><em>matK/trnK-UUU</em></strong> </td><td>65 579 </td><td>G/C </td><td>0.06 </td><td>Intergenic </td><td><em>trnP-UGG/psaJ</em> </td></tr><tr><td><strong>5555</strong> </td><td><strong>A/T</strong> </td><td><strong>0.13</strong> </td><td><strong>Intron</strong> </td><td><strong><em>rps16</em></strong> </td><td>66 056 </td><td>T/G </td><td>0.19 </td><td>Intergenic </td><td><em>psaJ/rpl33</em> </td></tr><tr><td><strong>7837</strong> </td><td><strong>A/C</strong> </td><td><strong>0.06</strong> </td><td><strong>Intergenic</strong> </td><td><strong><em>psbK/psbL</em></strong> </td><td>67 055 </td><td>G/A </td><td>0.25 </td><td>Intergenic </td><td><em>Rps18/rpl20</em> </td></tr><tr><td><strong>11 353</strong> </td><td><strong>C/A</strong> </td><td><strong>0.06</strong> </td><td><strong>Intergenic</strong> </td><td><strong><em>trnY-GUA</em>/<em>trnE-UUC</em></strong> </td><td>67 963 </td><td>T/C </td><td>0.69 </td><td>Intergenic </td><td><em>Rpl20/rps12</em> </td></tr><tr><td><strong>18 287</strong> </td><td><strong>A/C</strong> </td><td><strong>0.13</strong> </td><td><strong>Exon</strong> </td><td><strong><em>rpoC1</em></strong> </td><td>69 567** </td><td>T/C </td><td>0.06 </td><td>Exon </td><td><em>clpP</em> </td></tr><tr><td>22 648 </td><td>C/T </td><td>0.06 </td><td>Genic </td><td><em>rpoC2</em> </td><td>70 234 </td><td>T/G </td><td>0.06 </td><td>Intron </td><td><em>clpP</em> </td></tr><tr><td>24 906 </td><td>T/G </td><td>0.38 </td><td>Intergenic </td><td><em>atpI-atpH</em> </td><td>92 417* </td><td>C/T </td><td>0.06 </td><td>Intergenic </td><td><em>trnL-CAA/ndhB</em> </td></tr><tr><td>31 299 </td><td>C/A </td><td>0.06 </td><td>Intergenic </td><td><em>trnT-GGU/psbD</em> </td><td>97 496* </td><td>C/A </td><td>0.06 </td><td>Intergenic </td><td><em>Rps7/ycf15</em> </td></tr><tr><td>39 790 </td><td>A/G </td><td>0.44 </td><td>Genic </td><td><em>psaA</em> </td><td>106 663* </td><td>T/G </td><td>0.06 </td><td>Intergenic </td><td><em>trnR-ACG/trnN-GUU</em> </td></tr><tr><td>39 829 </td><td>G/A </td><td>0.13 </td><td>Genic </td><td><em>psaA</em> </td><td>106 664* </td><td>C/A </td><td>0.06 </td><td>Intergenic </td><td><em>trnR-ACG/trnN-GUU</em> </td></tr><tr><td>43 765 </td><td>C/T </td><td>0.06 </td><td>Intergenic </td><td><em>Ycf3/trnS-GCA</em> </td><td>108 200** </td><td>G/C </td><td>0.25 </td><td>Genic </td><td><em>Ycf1</em> </td></tr><tr><td>47 181 </td><td>G/C </td><td>0.06 </td><td>Intergenic </td><td><em>trnL-UAA/trnF-GAA</em> </td><td><strong>118 779</strong> </td><td><strong>C/G</strong> </td><td><strong>0.06</strong> </td><td><strong>Genic</strong> </td><td><strong><em>ndhD</em></strong> </td></tr><tr><td>49 751 </td><td>C/T </td><td>0.06 </td><td>Genic </td><td><em>ndhC</em> </td><td>123 423 </td><td>A/C </td><td>0.06 </td><td>Intergenic </td><td><em>Rpl32/ndhF</em> </td></tr><tr><td>53 025 </td><td>G/A </td><td>0.06 </td><td>Genic </td><td><em>atpB</em> </td><td>124 027 </td><td>C/T </td><td>0.06 </td><td>Genic </td><td><em>ndhF</em> </td></tr><tr><td><strong>60 245</strong> </td><td><strong>C/T</strong> </td><td><strong>0.06</strong> </td><td><strong>Genic</strong> </td><td><strong><em>cemA</em></strong> </td><td>124 035** </td><td>C/T </td><td>0.06 </td><td>Genic </td><td><em>ndhF</em> </td></tr></tbody></table></div><div content-id="DSR002TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Position </th><th>Alleles </th><th>Freq. </th><th>Region </th><th>Locus </th><th>Position </th><th>Alleles </th><th>Freq. </th><th>Region </th><th>Locus </th></tr></thead><tbody><tr><td>165 </td><td>T/A </td><td>0.13 </td><td>Intergenic </td><td><em>trnH-GUG/psbA</em> </td><td>61 436 </td><td>C/T </td><td>0.31 </td><td>Genic </td><td><em>petA</em> </td></tr><tr><td><strong>4032</strong> </td><td><strong>A/C</strong> </td><td><strong>0.06</strong> </td><td><strong>Intergenic</strong> </td><td><strong><em>matK/trnK-UUU</em></strong> </td><td>65 579 </td><td>G/C </td><td>0.06 </td><td>Intergenic </td><td><em>trnP-UGG/psaJ</em> </td></tr><tr><td><strong>5555</strong> </td><td><strong>A/T</strong> </td><td><strong>0.13</strong> </td><td><strong>Intron</strong> </td><td><strong><em>rps16</em></strong> </td><td>66 056 </td><td>T/G </td><td>0.19 </td><td>Intergenic </td><td><em>psaJ/rpl33</em> </td></tr><tr><td><strong>7837</strong> </td><td><strong>A/C</strong> </td><td><strong>0.06</strong> </td><td><strong>Intergenic</strong> </td><td><strong><em>psbK/psbL</em></strong> </td><td>67 055 </td><td>G/A </td><td>0.25 </td><td>Intergenic </td><td><em>Rps18/rpl20</em> </td></tr><tr><td><strong>11 353</strong> </td><td><strong>C/A</strong> </td><td><strong>0.06</strong> </td><td><strong>Intergenic</strong> </td><td><strong><em>trnY-GUA</em>/<em>trnE-UUC</em></strong> </td><td>67 963 </td><td>T/C </td><td>0.69 </td><td>Intergenic </td><td><em>Rpl20/rps12</em> </td></tr><tr><td><strong>18 287</strong> </td><td><strong>A/C</strong> </td><td><strong>0.13</strong> </td><td><strong>Exon</strong> </td><td><strong><em>rpoC1</em></strong> </td><td>69 567** </td><td>T/C </td><td>0.06 </td><td>Exon </td><td><em>clpP</em> </td></tr><tr><td>22 648 </td><td>C/T </td><td>0.06 </td><td>Genic </td><td><em>rpoC2</em> </td><td>70 234 </td><td>T/G </td><td>0.06 </td><td>Intron </td><td><em>clpP</em> </td></tr><tr><td>24 906 </td><td>T/G </td><td>0.38 </td><td>Intergenic </td><td><em>atpI-atpH</em> </td><td>92 417* </td><td>C/T </td><td>0.06 </td><td>Intergenic </td><td><em>trnL-CAA/ndhB</em> </td></tr><tr><td>31 299 </td><td>C/A </td><td>0.06 </td><td>Intergenic </td><td><em>trnT-GGU/psbD</em> </td><td>97 496* </td><td>C/A </td><td>0.06 </td><td>Intergenic </td><td><em>Rps7/ycf15</em> </td></tr><tr><td>39 790 </td><td>A/G </td><td>0.44 </td><td>Genic </td><td><em>psaA</em> </td><td>106 663* </td><td>T/G </td><td>0.06 </td><td>Intergenic </td><td><em>trnR-ACG/trnN-GUU</em> </td></tr><tr><td>39 829 </td><td>G/A </td><td>0.13 </td><td>Genic </td><td><em>psaA</em> </td><td>106 664* </td><td>C/A </td><td>0.06 </td><td>Intergenic </td><td><em>trnR-ACG/trnN-GUU</em> </td></tr><tr><td>43 765 </td><td>C/T </td><td>0.06 </td><td>Intergenic </td><td><em>Ycf3/trnS-GCA</em> </td><td>108 200** </td><td>G/C </td><td>0.25 </td><td>Genic </td><td><em>Ycf1</em> </td></tr><tr><td>47 181 </td><td>G/C </td><td>0.06 </td><td>Intergenic </td><td><em>trnL-UAA/trnF-GAA</em> </td><td><strong>118 779</strong> </td><td><strong>C/G</strong> </td><td><strong>0.06</strong> </td><td><strong>Genic</strong> </td><td><strong><em>ndhD</em></strong> </td></tr><tr><td>49 751 </td><td>C/T </td><td>0.06 </td><td>Genic </td><td><em>ndhC</em> </td><td>123 423 </td><td>A/C </td><td>0.06 </td><td>Intergenic </td><td><em>Rpl32/ndhF</em> </td></tr><tr><td>53 025 </td><td>G/A </td><td>0.06 </td><td>Genic </td><td><em>atpB</em> </td><td>124 027 </td><td>C/T </td><td>0.06 </td><td>Genic </td><td><em>ndhF</em> </td></tr><tr><td><strong>60 245</strong> </td><td><strong>C/T</strong> </td><td><strong>0.06</strong> </td><td><strong>Genic</strong> </td><td><strong><em>cemA</em></strong> </td><td>124 035** </td><td>C/T </td><td>0.06 </td><td>Genic </td><td><em>ndhF</em> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>SNPs that were tested for multiple individuals with high-resolution melting are indicated by bold typeface.</p></div></div><div content-id=""><div class="fn"><p>*SNPs located in the IR.</p></div></div><div content-id=""><div class="fn"><p>**Non-synonymous substitutions.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5299693" target="_blank">View Large</a></div></div>                        <a id="5299694" scrollto-destination="5299694"></a>
<div content-id="DSR002TB4" class="table-modal"><div class="table-wrap"><div id="DSR002TB4" class="table-wrap-title" data-id="DSR002TB4"><div class="label -label">Table 4.</div><div class="caption"><p>Summary of number of basepairs, number of SNPs, number of basepairs per SNP, number of microsatellite regions and number of basepairs per microsatellite region</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th colspan="2">nr of bp<hr /> </th><th colspan="2">nr of SNPs<hr /> </th><th colspan="2">nr of bp/nr of SNPs<hr /> </th><th colspan="2">nr of ms<hr /> </th><th colspan="2">nr of bp/nr of ms<hr /> </th></tr><tr><th> </th><th>SCR </th><th>IR </th><th>SCR </th><th>IR </th><th>SCR </th><th>IR </th><th>SCR </th><th>IR </th><th>SCR </th><th>IR </th></tr></thead><tbody><tr><td>Non-coding DNA </td><td>41 688 </td><td>8574 </td><td>16 </td><td>4 </td><td>2606 </td><td>2144 </td><td>28 </td><td>1 </td><td>1489 </td><td>8574 </td></tr><tr><td>Coding DNA </td><td>59 445 </td><td>11 688 </td><td>12 </td><td>0 </td><td>4954 </td><td> </td><td>5 </td><td>0 </td><td>11 889 </td><td> </td></tr><tr><td>rRNA </td><td>0 </td><td>4515 </td><td>0 </td><td>0 </td><td> </td><td> </td><td>0 </td><td>0 </td><td> </td><td> </td></tr><tr><td>Non-coding gene </td><td>6877 </td><td>1339 </td><td>2 </td><td>0 </td><td>3439 </td><td> </td><td>5 </td><td>0 </td><td>1375 </td><td> </td></tr><tr><td>Coding gene </td><td>57 733 </td><td>11 174 </td><td>12 </td><td>0 </td><td>4811 </td><td> </td><td>6 </td><td>0 </td><td>9622 </td><td> </td></tr></tbody></table></div><div content-id="DSR002TB4" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th colspan="2">nr of bp<hr /> </th><th colspan="2">nr of SNPs<hr /> </th><th colspan="2">nr of bp/nr of SNPs<hr /> </th><th colspan="2">nr of ms<hr /> </th><th colspan="2">nr of bp/nr of ms<hr /> </th></tr><tr><th> </th><th>SCR </th><th>IR </th><th>SCR </th><th>IR </th><th>SCR </th><th>IR </th><th>SCR </th><th>IR </th><th>SCR </th><th>IR </th></tr></thead><tbody><tr><td>Non-coding DNA </td><td>41 688 </td><td>8574 </td><td>16 </td><td>4 </td><td>2606 </td><td>2144 </td><td>28 </td><td>1 </td><td>1489 </td><td>8574 </td></tr><tr><td>Coding DNA </td><td>59 445 </td><td>11 688 </td><td>12 </td><td>0 </td><td>4954 </td><td> </td><td>5 </td><td>0 </td><td>11 889 </td><td> </td></tr><tr><td>rRNA </td><td>0 </td><td>4515 </td><td>0 </td><td>0 </td><td> </td><td> </td><td>0 </td><td>0 </td><td> </td><td> </td></tr><tr><td>Non-coding gene </td><td>6877 </td><td>1339 </td><td>2 </td><td>0 </td><td>3439 </td><td> </td><td>5 </td><td>0 </td><td>1375 </td><td> </td></tr><tr><td>Coding gene </td><td>57 733 </td><td>11 174 </td><td>12 </td><td>0 </td><td>4811 </td><td> </td><td>6 </td><td>0 </td><td>9622 </td><td> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Non-coding DNA, intergenic spacers and introns; coding DNA, tRNA, genes and exons; non-coding gene, introns; coding gene, genes and exons. All comparisons are made for the single copy region (SCR) and for one IR.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5299694" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR002TB4" class="table-wrap-title" data-id="DSR002TB4"><div class="label -label">Table 4.</div><div class="caption"><p>Summary of number of basepairs, number of SNPs, number of basepairs per SNP, number of microsatellite regions and number of basepairs per microsatellite region</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th colspan="2">nr of bp<hr /> </th><th colspan="2">nr of SNPs<hr /> </th><th colspan="2">nr of bp/nr of SNPs<hr /> </th><th colspan="2">nr of ms<hr /> </th><th colspan="2">nr of bp/nr of ms<hr /> </th></tr><tr><th> </th><th>SCR </th><th>IR </th><th>SCR </th><th>IR </th><th>SCR </th><th>IR </th><th>SCR </th><th>IR </th><th>SCR </th><th>IR </th></tr></thead><tbody><tr><td>Non-coding DNA </td><td>41 688 </td><td>8574 </td><td>16 </td><td>4 </td><td>2606 </td><td>2144 </td><td>28 </td><td>1 </td><td>1489 </td><td>8574 </td></tr><tr><td>Coding DNA </td><td>59 445 </td><td>11 688 </td><td>12 </td><td>0 </td><td>4954 </td><td> </td><td>5 </td><td>0 </td><td>11 889 </td><td> </td></tr><tr><td>rRNA </td><td>0 </td><td>4515 </td><td>0 </td><td>0 </td><td> </td><td> </td><td>0 </td><td>0 </td><td> </td><td> </td></tr><tr><td>Non-coding gene </td><td>6877 </td><td>1339 </td><td>2 </td><td>0 </td><td>3439 </td><td> </td><td>5 </td><td>0 </td><td>1375 </td><td> </td></tr><tr><td>Coding gene </td><td>57 733 </td><td>11 174 </td><td>12 </td><td>0 </td><td>4811 </td><td> </td><td>6 </td><td>0 </td><td>9622 </td><td> </td></tr></tbody></table></div><div content-id="DSR002TB4" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th colspan="2">nr of bp<hr /> </th><th colspan="2">nr of SNPs<hr /> </th><th colspan="2">nr of bp/nr of SNPs<hr /> </th><th colspan="2">nr of ms<hr /> </th><th colspan="2">nr of bp/nr of ms<hr /> </th></tr><tr><th> </th><th>SCR </th><th>IR </th><th>SCR </th><th>IR </th><th>SCR </th><th>IR </th><th>SCR </th><th>IR </th><th>SCR </th><th>IR </th></tr></thead><tbody><tr><td>Non-coding DNA </td><td>41 688 </td><td>8574 </td><td>16 </td><td>4 </td><td>2606 </td><td>2144 </td><td>28 </td><td>1 </td><td>1489 </td><td>8574 </td></tr><tr><td>Coding DNA </td><td>59 445 </td><td>11 688 </td><td>12 </td><td>0 </td><td>4954 </td><td> </td><td>5 </td><td>0 </td><td>11 889 </td><td> </td></tr><tr><td>rRNA </td><td>0 </td><td>4515 </td><td>0 </td><td>0 </td><td> </td><td> </td><td>0 </td><td>0 </td><td> </td><td> </td></tr><tr><td>Non-coding gene </td><td>6877 </td><td>1339 </td><td>2 </td><td>0 </td><td>3439 </td><td> </td><td>5 </td><td>0 </td><td>1375 </td><td> </td></tr><tr><td>Coding gene </td><td>57 733 </td><td>11 174 </td><td>12 </td><td>0 </td><td>4811 </td><td> </td><td>6 </td><td>0 </td><td>9622 </td><td> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Non-coding DNA, intergenic spacers and introns; coding DNA, tRNA, genes and exons; non-coding gene, introns; coding gene, genes and exons. All comparisons are made for the single copy region (SCR) and for one IR.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5299694" target="_blank">View Large</a></div></div><p>Reads derived from the IRs are distributed randomly to IRa or IRb by the assembly software. However, if IRa is different from IRb by an indel or SNP, this would be observed as a polymorphism within an individual. That was, however, never observed as we specifically checked for this. In the one case where we found that the sequence of the IR of individual 11 was deviating from other individuals for four positions, these positions within individual 11 were fully homozygous in both IRa and IRb. All SNPs found in the IRs, 2 × 4 in total, were located in the intergenic spacers of individual 11. The four SNPs found in individual 11 in IRa were found in exactly the same place and the same mutation as in IRb. This suggests ‘concerted evolution’ or gene conversion for the IR region. On average, one SNP in every 1808 bp was found in the intergenic spacers in the IR. For a subset of 11 SNPs, primers were developed (Table <a reveal-id="DSR002TB3" data-open="DSR002TB3" class="link link-reveal link-table xref-fig">3</a>) and several individuals were genotyped using high-resolution melting. For all these individuals, the SNPs were confirmed.</p><p>The number of SNPs that were found in this study might be slightly underestimated because the whole cp genome was not mapped with sufficient coverage to detect all SNPs in the 17 individuals analysed. Although the number of synonymous substitutions in chloroplast genes is on average at least three times lower than that of nuclear genes,<sup><a reveal-id="DSR002C24" data-open="DSR002C24" class="link link-ref link-reveal xref-bibr">24</a></sup> we still found SNPs using chloroplast genomes of 17 individuals of <em>J. vulgaris</em> originating from different populations. We found that SNPs were 1.8 times more frequent in intergenic spacers and introns when compared with DNA coding genes. These findings are in line with the assumption that coding DNA generally evolves more slowly than non-coding regions.<sup><a reveal-id="DSR002C30" data-open="DSR002C30" class="link link-ref link-reveal xref-bibr">30</a></sup></p><p>The result that individual 11 has four SNPs in both the IR regions suggests that a mechanism is present that provides simultaneous mutations in both IRa and IRb. In all 17 individuals, the sequences of IRa and IRb did not differ from each other by a single base. The gene <em>Ycf</em>1 starts at the end of IRb and extends into small single copy (SSC) to yield the full <em>Ycf</em>1 sequence. In IRa, the <em>Ycf</em>1 gene starts but is not extended into SSC yielding a non-functional sequence. It suggests that there is a selective force that prevents that the IR regions start to deviate from each other even when all the mutations are located in the intergenic spacer or non-functional genes. As a consequence, the IRs may contribute to the structural stability of the cp genome. Two plant groups, legumes and conifers, lost their IR and comparative sequence studies showed that these chloroplasts experienced a 4-fold increase in silent substitutions compared with chloroplasts containing the IR.<sup><a reveal-id="DSR002C46" data-open="DSR002C46" class="link link-ref link-reveal xref-bibr">46</a></sup></p>                            <h4 scrollto-destination=5299698 id="5299698" class="section-title" data-legacy-id=s3d2>3.4.2. Microsatellites</h4>
<p>A total of 34 microsatellite regions were found with A/T repeats longer than nine repeats, which is one microsatellite per 4432 bp. Only one microsatellite region was found with 11 G repeats and no repeats of 10 or more Cs were found in the chloroplast genome of <em>J. vulgaris</em>.</p><p>Within the single-copy region, 5.6 times as much microsatellite regions were found in intergenic spacers and introns compared with coding DNA (28 against 5, respectively). No microsatellites were found in the tRNA and rRNA. We found on average one microsatellite region every 1489 bp in intergenic spacers and introns against 1 of 11 889 bp in coding DNA (Table <a reveal-id="DSR002TB4" data-open="DSR002TB4" class="link link-reveal link-table xref-fig">4</a>). Within the genes, microsatellite regions were almost equally divided over exons and genes (6) and introns (5). This is on average 1 of 1375 bp for introns against 1 of 96 222 bp for exons and genes (Table <a reveal-id="DSR002TB4" data-open="DSR002TB4" class="link link-reveal link-table xref-fig">4</a>). This is not in accordance with the data of SNPs where the number of SNPs per base pair was relatively almost the same for exons + genes and introns. An insertion or a deletion in an exon or gene will lead to a frame shift and therefore likely leads to a non-functional protein. Both microsatellite regions and SNPs occur less in DNA coding regions (exons + genes + tRNA) compared with non-coding regions (intergenic spacers + intron). However, this difference is more marked for microsatellite regions than SNPs.</p><p>Of the 34 microsatellite regions, only one was located on IRb in an intergenic spacer. This is surprising because concerted evolution, as earlier suggested, should lead to exact sequence duplication in IRa compared with IRb, and therefore both IRs should contain the same number of nucleotide repeats. Indeed, we found a microsatellite region at the same place on both IRs, but this repeat was only 8 bp on IRa and is therefore not included in Table <a reveal-id="DSR002TB5" data-open="DSR002TB5" class="link link-reveal link-table xref-fig">5</a>. For 10 repeat regions, primers were developed and multiple individuals from different populations were genotyped (Table <a reveal-id="DSR002TB5" data-open="DSR002TB5" class="link link-reveal link-table xref-fig">5</a>). Optimization failed for one primer pair, but the other nine regions were amplified and they were all polymorphic. We tested 93 <em>J. vulgaris</em> individuals in total and found that all were polymorphic with the number of alleles per locus varying from two to six with an average of 3.3 alleles per locus. </p>                        <a id="5299702" scrollto-destination="5299702"></a>
<div content-id="DSR002TB5" class="table-modal"><div class="table-wrap"><div id="DSR002TB5" class="table-wrap-title" data-id="DSR002TB5"><div class="label -label">Table 5.</div><div class="caption"><p>List of positions and variants of SNPs and microsatellites in the chloroplast genome of <em>J. Vulgaris</em>: potential microsatellite loci, repeat, repeat length in the consensus chloroplast sequence and the region and locus of these repeats in the cp genome of <em>J. vulgaris</em></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Position of repeat </th><th>Repeat </th><th>Repeat length of consensus </th><th>Region </th><th>Locus </th></tr></thead><tbody><tr><td><strong>6705</strong> </td><td><strong>A</strong> </td><td><strong>11</strong> </td><td><strong>Intergenic</strong> </td><td><strong><em>rps16/trnQ-UUG</em></strong> </td></tr><tr><td><strong>12 459</strong> </td><td><strong>T</strong> </td><td><strong>14</strong> </td><td><strong>Intergenic</strong> </td><td><strong><em>trnE-UUC/rpoB</em></strong> </td></tr><tr><td>13 143 </td><td>A </td><td>10 </td><td>Genic </td><td><em>rpoB</em> </td></tr><tr><td>16 413 </td><td>T </td><td>10 </td><td>Intron </td><td><em>rpoC1</em> </td></tr><tr><td>17 759 </td><td>A </td><td>10 </td><td>Exon </td><td><em>rpoC1</em> </td></tr><tr><td>18 185 </td><td>A </td><td>10 </td><td>Exon </td><td><em>rpoC1</em> </td></tr><tr><td><strong>24 848</strong> </td><td><strong>A</strong> </td><td><strong>17</strong> </td><td><strong>Intergenic</strong> </td><td><strong><em>atpL/atpH</em></strong> </td></tr><tr><td><strong>27 760</strong> </td><td><strong>T</strong> </td><td><strong>15</strong> </td><td><strong>Intergenic</strong> </td><td><strong><em>atpF/atpA</em></strong> </td></tr><tr><td>27 776 </td><td>A </td><td>11 </td><td>Intergenic </td><td><em>atpF/atpA</em> </td></tr><tr><td>34 901 </td><td>A </td><td>10 </td><td>Intergenic </td><td><em>trnS-UGA/psbZ</em> </td></tr><tr><td><strong>41 459</strong> </td><td><strong>T</strong> </td><td><strong>10</strong> </td><td><strong>Intergenic</strong> </td><td><strong><em>psaA-ycf3</em></strong> </td></tr><tr><td>41 471 </td><td>A </td><td>13 </td><td>Intergenic </td><td><em>psaA-ycf3</em> </td></tr><tr><td>46 228 </td><td>A </td><td>14 </td><td>Intergenic </td><td><em>trnT-UGU/trnL-UAA</em> </td></tr><tr><td>49 996 </td><td>G </td><td>11 </td><td>Intergenic </td><td><em>ndhC/trnV-UAC</em> </td></tr><tr><td>53 630 </td><td>A </td><td>10 </td><td>Intergenic </td><td><em>atpB/rbcL</em> </td></tr><tr><td><strong>54 013</strong> </td><td><strong>T</strong> </td><td><strong>18</strong> </td><td><strong>Intergenic</strong> </td><td><strong><em>atpB/rbcL</em></strong> </td></tr><tr><td>58 662 </td><td>T </td><td>10 </td><td>Intergenic </td><td><em>psaL/ycf4</em> </td></tr><tr><td>64 247 </td><td>A </td><td>11 </td><td>Intergenic </td><td><em>psbIE/petL</em> </td></tr><tr><td>69 969 </td><td>A </td><td>11 </td><td>Intron </td><td><em>clpP</em> </td></tr><tr><td>70 312 </td><td>A </td><td>10 </td><td>Intron </td><td><em>clpP</em> </td></tr><tr><td>72 916 </td><td>A </td><td>11 </td><td>Genic/Intergenic </td><td><em>psbT/psbN</em> </td></tr><tr><td>74 047 </td><td>A </td><td>11 </td><td>Intron </td><td><em>petB</em> </td></tr><tr><td><strong>76 775</strong> </td><td><strong>T</strong> </td><td><strong>17</strong> </td><td><strong>Genic</strong> </td><td><strong><em>rpoA</em></strong> </td></tr><tr><td>79 191 </td><td>T </td><td>13 </td><td>Intergenic </td><td><em>rps8/rpl14</em> </td></tr><tr><td><strong>79 774</strong> </td><td><strong>A</strong> </td><td><strong>10</strong> </td><td><strong>Intergenic</strong> </td><td><strong><em>rpl14/rpl16</em></strong> </td></tr><tr><td>81 396 </td><td>T </td><td>10 </td><td>Intergenic </td><td><em>rpl16/rps3</em> </td></tr><tr><td>82 909* </td><td>T </td><td>10 </td><td>Intergenic </td><td><em>rps19/rpl2</em> </td></tr><tr><td><strong>109 743</strong> </td><td><strong>A</strong> </td><td><strong>10</strong> </td><td><strong>Genic</strong> </td><td><strong><em>ycf1</em></strong> </td></tr><tr><td>112 000 </td><td>A </td><td>11 </td><td>Genic </td><td><em>ycf1</em> </td></tr><tr><td>114 539 </td><td>T </td><td>10 </td><td>Intron </td><td><em>ndhA</em> </td></tr><tr><td>121 458 </td><td>A </td><td>11 </td><td>Intergenic </td><td><em>ccsA/trnL-UAG</em> </td></tr><tr><td>121 889 </td><td>T </td><td>11 </td><td>Intergenic </td><td><em>trnL-UAG/rpl32</em> </td></tr><tr><td>123 661 </td><td>A </td><td>10 </td><td>Intergenic </td><td><em>rpl32/ndhF</em> </td></tr><tr><td>150 626* </td><td>A </td><td>10 </td><td>Intergenic </td><td><em>rpl2/tnH-GUG</em> </td></tr></tbody></table></div><div content-id="DSR002TB5" class="table-modal"><table><thead name="thead" align=""><tr><th>Position of repeat </th><th>Repeat </th><th>Repeat length of consensus </th><th>Region </th><th>Locus </th></tr></thead><tbody><tr><td><strong>6705</strong> </td><td><strong>A</strong> </td><td><strong>11</strong> </td><td><strong>Intergenic</strong> </td><td><strong><em>rps16/trnQ-UUG</em></strong> </td></tr><tr><td><strong>12 459</strong> </td><td><strong>T</strong> </td><td><strong>14</strong> </td><td><strong>Intergenic</strong> </td><td><strong><em>trnE-UUC/rpoB</em></strong> </td></tr><tr><td>13 143 </td><td>A </td><td>10 </td><td>Genic </td><td><em>rpoB</em> </td></tr><tr><td>16 413 </td><td>T </td><td>10 </td><td>Intron </td><td><em>rpoC1</em> </td></tr><tr><td>17 759 </td><td>A </td><td>10 </td><td>Exon </td><td><em>rpoC1</em> </td></tr><tr><td>18 185 </td><td>A </td><td>10 </td><td>Exon </td><td><em>rpoC1</em> </td></tr><tr><td><strong>24 848</strong> </td><td><strong>A</strong> </td><td><strong>17</strong> </td><td><strong>Intergenic</strong> </td><td><strong><em>atpL/atpH</em></strong> </td></tr><tr><td><strong>27 760</strong> </td><td><strong>T</strong> </td><td><strong>15</strong> </td><td><strong>Intergenic</strong> </td><td><strong><em>atpF/atpA</em></strong> </td></tr><tr><td>27 776 </td><td>A </td><td>11 </td><td>Intergenic </td><td><em>atpF/atpA</em> </td></tr><tr><td>34 901 </td><td>A </td><td>10 </td><td>Intergenic </td><td><em>trnS-UGA/psbZ</em> </td></tr><tr><td><strong>41 459</strong> </td><td><strong>T</strong> </td><td><strong>10</strong> </td><td><strong>Intergenic</strong> </td><td><strong><em>psaA-ycf3</em></strong> </td></tr><tr><td>41 471 </td><td>A </td><td>13 </td><td>Intergenic </td><td><em>psaA-ycf3</em> </td></tr><tr><td>46 228 </td><td>A </td><td>14 </td><td>Intergenic </td><td><em>trnT-UGU/trnL-UAA</em> </td></tr><tr><td>49 996 </td><td>G </td><td>11 </td><td>Intergenic </td><td><em>ndhC/trnV-UAC</em> </td></tr><tr><td>53 630 </td><td>A </td><td>10 </td><td>Intergenic </td><td><em>atpB/rbcL</em> </td></tr><tr><td><strong>54 013</strong> </td><td><strong>T</strong> </td><td><strong>18</strong> </td><td><strong>Intergenic</strong> </td><td><strong><em>atpB/rbcL</em></strong> </td></tr><tr><td>58 662 </td><td>T </td><td>10 </td><td>Intergenic </td><td><em>psaL/ycf4</em> </td></tr><tr><td>64 247 </td><td>A </td><td>11 </td><td>Intergenic </td><td><em>psbIE/petL</em> </td></tr><tr><td>69 969 </td><td>A </td><td>11 </td><td>Intron </td><td><em>clpP</em> </td></tr><tr><td>70 312 </td><td>A </td><td>10 </td><td>Intron </td><td><em>clpP</em> </td></tr><tr><td>72 916 </td><td>A </td><td>11 </td><td>Genic/Intergenic </td><td><em>psbT/psbN</em> </td></tr><tr><td>74 047 </td><td>A </td><td>11 </td><td>Intron </td><td><em>petB</em> </td></tr><tr><td><strong>76 775</strong> </td><td><strong>T</strong> </td><td><strong>17</strong> </td><td><strong>Genic</strong> </td><td><strong><em>rpoA</em></strong> </td></tr><tr><td>79 191 </td><td>T </td><td>13 </td><td>Intergenic </td><td><em>rps8/rpl14</em> </td></tr><tr><td><strong>79 774</strong> </td><td><strong>A</strong> </td><td><strong>10</strong> </td><td><strong>Intergenic</strong> </td><td><strong><em>rpl14/rpl16</em></strong> </td></tr><tr><td>81 396 </td><td>T </td><td>10 </td><td>Intergenic </td><td><em>rpl16/rps3</em> </td></tr><tr><td>82 909* </td><td>T </td><td>10 </td><td>Intergenic </td><td><em>rps19/rpl2</em> </td></tr><tr><td><strong>109 743</strong> </td><td><strong>A</strong> </td><td><strong>10</strong> </td><td><strong>Genic</strong> </td><td><strong><em>ycf1</em></strong> </td></tr><tr><td>112 000 </td><td>A </td><td>11 </td><td>Genic </td><td><em>ycf1</em> </td></tr><tr><td>114 539 </td><td>T </td><td>10 </td><td>Intron </td><td><em>ndhA</em> </td></tr><tr><td>121 458 </td><td>A </td><td>11 </td><td>Intergenic </td><td><em>ccsA/trnL-UAG</em> </td></tr><tr><td>121 889 </td><td>T </td><td>11 </td><td>Intergenic </td><td><em>trnL-UAG/rpl32</em> </td></tr><tr><td>123 661 </td><td>A </td><td>10 </td><td>Intergenic </td><td><em>rpl32/ndhF</em> </td></tr><tr><td>150 626* </td><td>A </td><td>10 </td><td>Intergenic </td><td><em>rpl2/tnH-GUG</em> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Microsatellites that were tested for polymorphisms in multiple individuals are indicated by bold typeface.</p></div></div><div content-id=""><div class="fn"><p>*Microsatellites located in the IR.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5299702" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR002TB5" class="table-wrap-title" data-id="DSR002TB5"><div class="label -label">Table 5.</div><div class="caption"><p>List of positions and variants of SNPs and microsatellites in the chloroplast genome of <em>J. Vulgaris</em>: potential microsatellite loci, repeat, repeat length in the consensus chloroplast sequence and the region and locus of these repeats in the cp genome of <em>J. vulgaris</em></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Position of repeat </th><th>Repeat </th><th>Repeat length of consensus </th><th>Region </th><th>Locus </th></tr></thead><tbody><tr><td><strong>6705</strong> </td><td><strong>A</strong> </td><td><strong>11</strong> </td><td><strong>Intergenic</strong> </td><td><strong><em>rps16/trnQ-UUG</em></strong> </td></tr><tr><td><strong>12 459</strong> </td><td><strong>T</strong> </td><td><strong>14</strong> </td><td><strong>Intergenic</strong> </td><td><strong><em>trnE-UUC/rpoB</em></strong> </td></tr><tr><td>13 143 </td><td>A </td><td>10 </td><td>Genic </td><td><em>rpoB</em> </td></tr><tr><td>16 413 </td><td>T </td><td>10 </td><td>Intron </td><td><em>rpoC1</em> </td></tr><tr><td>17 759 </td><td>A </td><td>10 </td><td>Exon </td><td><em>rpoC1</em> </td></tr><tr><td>18 185 </td><td>A </td><td>10 </td><td>Exon </td><td><em>rpoC1</em> </td></tr><tr><td><strong>24 848</strong> </td><td><strong>A</strong> </td><td><strong>17</strong> </td><td><strong>Intergenic</strong> </td><td><strong><em>atpL/atpH</em></strong> </td></tr><tr><td><strong>27 760</strong> </td><td><strong>T</strong> </td><td><strong>15</strong> </td><td><strong>Intergenic</strong> </td><td><strong><em>atpF/atpA</em></strong> </td></tr><tr><td>27 776 </td><td>A </td><td>11 </td><td>Intergenic </td><td><em>atpF/atpA</em> </td></tr><tr><td>34 901 </td><td>A </td><td>10 </td><td>Intergenic </td><td><em>trnS-UGA/psbZ</em> </td></tr><tr><td><strong>41 459</strong> </td><td><strong>T</strong> </td><td><strong>10</strong> </td><td><strong>Intergenic</strong> </td><td><strong><em>psaA-ycf3</em></strong> </td></tr><tr><td>41 471 </td><td>A </td><td>13 </td><td>Intergenic </td><td><em>psaA-ycf3</em> </td></tr><tr><td>46 228 </td><td>A </td><td>14 </td><td>Intergenic </td><td><em>trnT-UGU/trnL-UAA</em> </td></tr><tr><td>49 996 </td><td>G </td><td>11 </td><td>Intergenic </td><td><em>ndhC/trnV-UAC</em> </td></tr><tr><td>53 630 </td><td>A </td><td>10 </td><td>Intergenic </td><td><em>atpB/rbcL</em> </td></tr><tr><td><strong>54 013</strong> </td><td><strong>T</strong> </td><td><strong>18</strong> </td><td><strong>Intergenic</strong> </td><td><strong><em>atpB/rbcL</em></strong> </td></tr><tr><td>58 662 </td><td>T </td><td>10 </td><td>Intergenic </td><td><em>psaL/ycf4</em> </td></tr><tr><td>64 247 </td><td>A </td><td>11 </td><td>Intergenic </td><td><em>psbIE/petL</em> </td></tr><tr><td>69 969 </td><td>A </td><td>11 </td><td>Intron </td><td><em>clpP</em> </td></tr><tr><td>70 312 </td><td>A </td><td>10 </td><td>Intron </td><td><em>clpP</em> </td></tr><tr><td>72 916 </td><td>A </td><td>11 </td><td>Genic/Intergenic </td><td><em>psbT/psbN</em> </td></tr><tr><td>74 047 </td><td>A </td><td>11 </td><td>Intron </td><td><em>petB</em> </td></tr><tr><td><strong>76 775</strong> </td><td><strong>T</strong> </td><td><strong>17</strong> </td><td><strong>Genic</strong> </td><td><strong><em>rpoA</em></strong> </td></tr><tr><td>79 191 </td><td>T </td><td>13 </td><td>Intergenic </td><td><em>rps8/rpl14</em> </td></tr><tr><td><strong>79 774</strong> </td><td><strong>A</strong> </td><td><strong>10</strong> </td><td><strong>Intergenic</strong> </td><td><strong><em>rpl14/rpl16</em></strong> </td></tr><tr><td>81 396 </td><td>T </td><td>10 </td><td>Intergenic </td><td><em>rpl16/rps3</em> </td></tr><tr><td>82 909* </td><td>T </td><td>10 </td><td>Intergenic </td><td><em>rps19/rpl2</em> </td></tr><tr><td><strong>109 743</strong> </td><td><strong>A</strong> </td><td><strong>10</strong> </td><td><strong>Genic</strong> </td><td><strong><em>ycf1</em></strong> </td></tr><tr><td>112 000 </td><td>A </td><td>11 </td><td>Genic </td><td><em>ycf1</em> </td></tr><tr><td>114 539 </td><td>T </td><td>10 </td><td>Intron </td><td><em>ndhA</em> </td></tr><tr><td>121 458 </td><td>A </td><td>11 </td><td>Intergenic </td><td><em>ccsA/trnL-UAG</em> </td></tr><tr><td>121 889 </td><td>T </td><td>11 </td><td>Intergenic </td><td><em>trnL-UAG/rpl32</em> </td></tr><tr><td>123 661 </td><td>A </td><td>10 </td><td>Intergenic </td><td><em>rpl32/ndhF</em> </td></tr><tr><td>150 626* </td><td>A </td><td>10 </td><td>Intergenic </td><td><em>rpl2/tnH-GUG</em> </td></tr></tbody></table></div><div content-id="DSR002TB5" class="table-modal"><table><thead name="thead" align=""><tr><th>Position of repeat </th><th>Repeat </th><th>Repeat length of consensus </th><th>Region </th><th>Locus </th></tr></thead><tbody><tr><td><strong>6705</strong> </td><td><strong>A</strong> </td><td><strong>11</strong> </td><td><strong>Intergenic</strong> </td><td><strong><em>rps16/trnQ-UUG</em></strong> </td></tr><tr><td><strong>12 459</strong> </td><td><strong>T</strong> </td><td><strong>14</strong> </td><td><strong>Intergenic</strong> </td><td><strong><em>trnE-UUC/rpoB</em></strong> </td></tr><tr><td>13 143 </td><td>A </td><td>10 </td><td>Genic </td><td><em>rpoB</em> </td></tr><tr><td>16 413 </td><td>T </td><td>10 </td><td>Intron </td><td><em>rpoC1</em> </td></tr><tr><td>17 759 </td><td>A </td><td>10 </td><td>Exon </td><td><em>rpoC1</em> </td></tr><tr><td>18 185 </td><td>A </td><td>10 </td><td>Exon </td><td><em>rpoC1</em> </td></tr><tr><td><strong>24 848</strong> </td><td><strong>A</strong> </td><td><strong>17</strong> </td><td><strong>Intergenic</strong> </td><td><strong><em>atpL/atpH</em></strong> </td></tr><tr><td><strong>27 760</strong> </td><td><strong>T</strong> </td><td><strong>15</strong> </td><td><strong>Intergenic</strong> </td><td><strong><em>atpF/atpA</em></strong> </td></tr><tr><td>27 776 </td><td>A </td><td>11 </td><td>Intergenic </td><td><em>atpF/atpA</em> </td></tr><tr><td>34 901 </td><td>A </td><td>10 </td><td>Intergenic </td><td><em>trnS-UGA/psbZ</em> </td></tr><tr><td><strong>41 459</strong> </td><td><strong>T</strong> </td><td><strong>10</strong> </td><td><strong>Intergenic</strong> </td><td><strong><em>psaA-ycf3</em></strong> </td></tr><tr><td>41 471 </td><td>A </td><td>13 </td><td>Intergenic </td><td><em>psaA-ycf3</em> </td></tr><tr><td>46 228 </td><td>A </td><td>14 </td><td>Intergenic </td><td><em>trnT-UGU/trnL-UAA</em> </td></tr><tr><td>49 996 </td><td>G </td><td>11 </td><td>Intergenic </td><td><em>ndhC/trnV-UAC</em> </td></tr><tr><td>53 630 </td><td>A </td><td>10 </td><td>Intergenic </td><td><em>atpB/rbcL</em> </td></tr><tr><td><strong>54 013</strong> </td><td><strong>T</strong> </td><td><strong>18</strong> </td><td><strong>Intergenic</strong> </td><td><strong><em>atpB/rbcL</em></strong> </td></tr><tr><td>58 662 </td><td>T </td><td>10 </td><td>Intergenic </td><td><em>psaL/ycf4</em> </td></tr><tr><td>64 247 </td><td>A </td><td>11 </td><td>Intergenic </td><td><em>psbIE/petL</em> </td></tr><tr><td>69 969 </td><td>A </td><td>11 </td><td>Intron </td><td><em>clpP</em> </td></tr><tr><td>70 312 </td><td>A </td><td>10 </td><td>Intron </td><td><em>clpP</em> </td></tr><tr><td>72 916 </td><td>A </td><td>11 </td><td>Genic/Intergenic </td><td><em>psbT/psbN</em> </td></tr><tr><td>74 047 </td><td>A </td><td>11 </td><td>Intron </td><td><em>petB</em> </td></tr><tr><td><strong>76 775</strong> </td><td><strong>T</strong> </td><td><strong>17</strong> </td><td><strong>Genic</strong> </td><td><strong><em>rpoA</em></strong> </td></tr><tr><td>79 191 </td><td>T </td><td>13 </td><td>Intergenic </td><td><em>rps8/rpl14</em> </td></tr><tr><td><strong>79 774</strong> </td><td><strong>A</strong> </td><td><strong>10</strong> </td><td><strong>Intergenic</strong> </td><td><strong><em>rpl14/rpl16</em></strong> </td></tr><tr><td>81 396 </td><td>T </td><td>10 </td><td>Intergenic </td><td><em>rpl16/rps3</em> </td></tr><tr><td>82 909* </td><td>T </td><td>10 </td><td>Intergenic </td><td><em>rps19/rpl2</em> </td></tr><tr><td><strong>109 743</strong> </td><td><strong>A</strong> </td><td><strong>10</strong> </td><td><strong>Genic</strong> </td><td><strong><em>ycf1</em></strong> </td></tr><tr><td>112 000 </td><td>A </td><td>11 </td><td>Genic </td><td><em>ycf1</em> </td></tr><tr><td>114 539 </td><td>T </td><td>10 </td><td>Intron </td><td><em>ndhA</em> </td></tr><tr><td>121 458 </td><td>A </td><td>11 </td><td>Intergenic </td><td><em>ccsA/trnL-UAG</em> </td></tr><tr><td>121 889 </td><td>T </td><td>11 </td><td>Intergenic </td><td><em>trnL-UAG/rpl32</em> </td></tr><tr><td>123 661 </td><td>A </td><td>10 </td><td>Intergenic </td><td><em>rpl32/ndhF</em> </td></tr><tr><td>150 626* </td><td>A </td><td>10 </td><td>Intergenic </td><td><em>rpl2/tnH-GUG</em> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Microsatellites that were tested for polymorphisms in multiple individuals are indicated by bold typeface.</p></div></div><div content-id=""><div class="fn"><p>*Microsatellites located in the IR.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5299702" target="_blank">View Large</a></div></div><p>The number of microsatellite regions is promising for investigating allele frequencies in populations and eventually, together with the SNP data, tracing the source population(s) of non-native <em>J. vulgaris</em>. The number of variable microsatellites might be higher since we arbitrarily decided to include only mononucleotide repeats that were at least 10 bp long. We found that potential microsatellite regions were 4.7 times more located in intergenic regions and introns compared with coding regions. Because SNPs were only 1.8 times more located in intergenic regions and introns compared with coding regions, we conclude that point mutations are more frequent in coding DNA than indels leading to frame shifts immediately. Although the location of potential microsatellite loci is certain, the repeat length is an approximation. During the long-range PCR and PCR steps in the sample preparation steps for the Illumina platform, indels can occur in microsatellite loci, leading to less or more repeats. Consequently, the Illumina reads for microsatellite loci differed, making it hard to deduce the repeat length. This could also be the explanation for finding a difference in repeat length of a potential microsatellite locus between the IRs.</p><p>In conclusion, we found promising regions for development of universal primers that can be used for further investigation of clades in molecular phylogenies of Asteraceae. Considering the number of SNPs and microsatellites found in this study, we recommend screening of the complete chloroplast genome to find differences within a species. Despite the higher variation in coverage, using the long-range PCR products as templates for Illumina sequencing seemed to be far more efficient than using cpDNA directly.</p>                            <h2 scrollto-destination=5299705 id="5299705" class="section-title" data-legacy-id=s4>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=5299707 id="5299707" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank T. Morley, J. Ireson, H. Auge, M. Bartelheimer, U. Schaffner, P. Olejnicrak, A. Balogh, E. van der Meijden and N. Sletvold for collecting seeds. Furthermore, we would like to acknowledge P. Pelser for sharing information about phylogenetic markers currently employed within the Senecioninae. At last, we are very thankful to H. Kirk for revising the English.</p>                            <h2 scrollto-destination=5299709 id="5299709" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSR002C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR002C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lundberg</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Bremer</div> <div class="given-names">K.</div></div>. </div><div class="article-title">A phylogenetic study of the order Asterales using one morphological and three molecular data sets</div>, <div class="source">Int. J. Plant Sci.</div> , <div class="year">2003</div>, vol. <div class="volume">164</div> (pg. <div class="fpage">553</div>-<div class="lpage">78</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20phylogenetic%20study%20of%20the%20order%20Asterales%20using%20one%20morphological%20and%20three%20molecular%20data%20sets&amp;author=J.%20Lundberg&amp;author=K.%20Bremer&amp;publication_year=2003&amp;journal=Int.%20J.%20Plant%20Sci.&amp;volume=164&amp;pages=553-78" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1086/374829" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1086%2F374829" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1086%2F374829"> </span></div> </div></div></div></div></div><div content-id="DSR002C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR002C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Carlquist</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Tribal interrelationships and phylogeny of the Asteraceae</div>, <div class="source">Aliso</div> , <div class="year">1976</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">465</div>-<div class="lpage">92</div>)</div></div></div></div><div content-id="DSR002C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR002C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hendry</div> <div class="given-names">G.A.F.</div></div>. </div><div class="article-title">Evolutionary origins and natural functions of fructans—a climatological, biogeographic and mechanistic appraisal</div>, <div class="source">New Phytologist.</div> , <div class="year">1993</div>, vol. <div class="volume">123</div> (pg. <div class="fpage">3</div>-<div class="lpage">14</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolutionary%20origins%20and%20natural%20functions%20of%20fructans%E2%80%94a%20climatological%2C%20biogeographic%20and%20mechanistic%20appraisal&amp;author=G.A.F.%20Hendry&amp;publication_year=1993&amp;journal=New%20Phytologist.&amp;volume=123&amp;pages=3-14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1469-8137.1993.tb04525.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1469-8137.1993.tb04525.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1469-8137.1993.tb04525.x"> </span></div> </div></div></div></div></div><div content-id="DSR002C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR002C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Barker</div> <div class="given-names">M.S.</div></div>,  <div class="name"><div class="surname">Kane</div> <div class="given-names">N.C.</div></div>,  <div class="name"><div class="surname">Matvienko</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Multiple paleopolyploidizations during the evolution of the Compositae reveal parallel patterns of duplicate gene retention after millions of years</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2008</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">2445</div>-<div class="lpage">55</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Multiple%20paleopolyploidizations%20during%20the%20evolution%20of%20the%20Compositae%20reveal%20parallel%20patterns%20of%20duplicate%20gene%20retention%20after%20millions%20of%20years&amp;author=M.S.%20Barker&amp;author=N.C.%20Kane&amp;author=M.%20Matvienko&amp;publication_year=2008&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=25&amp;pages=2445-55" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msn187" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsn187" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsn187"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18728074" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR002C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR002C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kim</div> <div class="given-names">K.M.</div></div>,  <div class="name"><div class="surname">Choi</div> <div class="given-names">K.S.</div></div>,  <div class="name"><div class="surname">Jansen</div> <div class="given-names">R.K.</div></div>. </div><div class="article-title">Two chloroplast DNA inversions originated simultaneously during the early evolution of the sunflower family (Asteraceae)</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2005</div>, vol. <div class="volume">22</div> (pg. <div class="fpage">1783</div>-<div class="lpage">92</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Two%20chloroplast%20DNA%20inversions%20originated%20simultaneously%20during%20the%20early%20evolution%20of%20the%20sunflower%20family%20(Asteraceae)&amp;author=K.M.%20Kim&amp;author=K.S.%20Choi&amp;author=R.K.%20Jansen&amp;publication_year=2005&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=22&amp;pages=1783-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msi174" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsi174" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsi174"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15917497" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR002C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR002C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Julien</div> <div class="given-names">M.</div></div>. </div>, <div class="source">Biological Control of Weeds: A World Catalogue of Agents and their Target Weeds</div> , <div class="year">1987</div><div class="publisher-loc">CAB International: Wallingford, UK</div></div></div></div></div><div content-id="DSR002C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR002C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wardle</div> <div class="given-names">D.A.</div></div>,  <div class="name"><div class="surname">Nicholson</div> <div class="given-names">K.S.</div></div>,  <div class="name"><div class="surname">Rahman</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Ecological effects of the invasive weed species <em>Senecio jacobaea</em> L (ragwort) in a New Zealand pasture</div>, <div class="source">Agric. Ecosyst. Environ.</div> , <div class="year">1995</div>, vol. <div class="volume">56</div> (pg. <div class="fpage">19</div>-<div class="lpage">28</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ecological%20effects%20of%20the%20invasive%20weed%20species%20Senecio%20jacobaea%20L%20(ragwort)%20in%20a%20New%20Zealand%20pasture&amp;author=D.A.%20Wardle&amp;author=K.S.%20Nicholson&amp;author=A.%20Rahman&amp;publication_year=1995&amp;journal=Agric.%20Ecosyst.%20Environ.&amp;volume=56&amp;pages=19-28" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0167-8809(95)00636-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0167-8809(95)00636-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0167-8809(95)00636-2"> </span></div> </div></div></div></div></div><div content-id="DSR002C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR002C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Panero</div> <div class="given-names">J.L.</div></div>,  <div class="name"><div class="surname">Funk</div> <div class="given-names">V.A.</div></div>. </div><div class="article-title">The value of sampling anomalous taxa in phylogenetic studies: major clades of the Asteraceae revealed</div>, <div class="source">Mol. Phylogenet. Evol.</div> , <div class="year">2008</div>, vol. <div class="volume">47</div> (pg. <div class="fpage">757</div>-<div class="lpage">82</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20value%20of%20sampling%20anomalous%20taxa%20in%20phylogenetic%20studies%3A%20major%20clades%20of%20the%20Asteraceae%20revealed&amp;author=J.L.%20Panero&amp;author=V.A.%20Funk&amp;publication_year=2008&amp;journal=Mol.%20Phylogenet.%20Evol.&amp;volume=47&amp;pages=757-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ympev.2008.02.011" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ympev.2008.02.011" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ympev.2008.02.011"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18375151" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR002C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR002C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pelser</div> <div class="given-names">P.B.</div></div>,  <div class="name"><div class="surname">Nordenstam</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Kadereit</div> <div class="given-names">J.W.</div></div>,  <div class="name"><div class="surname">Watson</div> <div class="given-names">L.E.</div></div>. </div><div class="article-title">An ITS phylogeny of tribe Senecioneae (Asteraceae) and a new delimitation of <em>Senecio</em> L</div>, <div class="source">Taxon</div> , <div class="year">2007</div>, vol. <div class="volume">56</div> (pg. <div class="fpage">1077</div>-<div class="lpage">104</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20ITS%20phylogeny%20of%20tribe%20Senecioneae%20(Asteraceae)%20and%20a%20new%20delimitation%20of%20Senecio%20L&amp;author=P.B.%20Pelser&amp;author=B.%20Nordenstam&amp;author=J.W.%20Kadereit&amp;author=L.E.%20Watson&amp;publication_year=2007&amp;journal=Taxon&amp;volume=56&amp;pages=1077-104" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2307/25065905" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2307%2F25065905" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2307%2F25065905"> </span></div> </div></div></div></div></div><div content-id="DSR002C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR002C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pelser</div> <div class="given-names">P.B.</div></div>,  <div class="name"><div class="surname">Kennedy</div> <div class="given-names">A.H.</div></div>,  <div class="name"><div class="surname">Tepe</div> <div class="given-names">E.J.</div></div>, et al. </div><div class="article-title">Patterns and causes of incongruence between plastid and nuclear Senecioneae (Asteraceae) phylogenies</div>, <div class="source">Am. J. Bot.</div> , <div class="year">2010</div>, vol. <div class="volume">97</div> (pg. <div class="fpage">856</div>-<div class="lpage">73</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Patterns%20and%20causes%20of%20incongruence%20between%20plastid%20and%20nuclear%20Senecioneae%20(Asteraceae)%20phylogenies&amp;author=P.B.%20Pelser&amp;author=A.H.%20Kennedy&amp;author=E.J.%20Tepe&amp;publication_year=2010&amp;journal=Am.%20J.%20Bot.&amp;volume=97&amp;pages=856-73" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3732/ajb.0900287" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3732%2Fajb.0900287" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3732%2Fajb.0900287"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21622451" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR002C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR002C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bain</div> <div class="given-names">J.F.</div></div>. </div><div class="article-title">Biology of Canadian weeds 96 <em>Senecio jacobaea</em> L</div>, <div class="source">Can. J. Plant Sci.</div> , <div class="year">1991</div>, vol. <div class="volume">71</div> (pg. <div class="fpage">127</div>-<div class="lpage">40</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Biology%20of%20Canadian%20weeds%2096%20Senecio%20jacobaea%20L&amp;author=J.F.%20Bain&amp;publication_year=1991&amp;journal=Can.%20J.%20Plant%20Sci.&amp;volume=71&amp;pages=127-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.4141/cjps91-014" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.4141%2Fcjps91-014" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.4141%2Fcjps91-014"> </span></div> </div></div></div></div></div><div content-id="DSR002C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR002C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Poole</div> <div class="given-names">A.L.</div></div>,  <div class="name"><div class="surname">Cairns</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Botanical aspects of ragwort (<em>Senecio jacobaea</em> L.) control</div>, <div class="source">Bulletin of the New Zealand Dept. of Sci. and Indust. Research</div> , <div class="year">1940</div>, vol. <div class="volume">82</div> (pg. <div class="fpage">2</div>-<div class="lpage">61</div>)</div></div></div></div><div content-id="DSR002C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR002C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schmidl</div> <div class="given-names">L.</div></div>. </div><div class="article-title">Biology and control of ragwort, <em>Senecio jacobaea</em> L, in Victoria, Australia</div>, <div class="source">Weed Res.</div> , <div class="year">1972</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">37</div>-<div class="lpage">45</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Biology%20and%20control%20of%20ragwort%2C%20Senecio%20jacobaea%20L%2C%20in%20Victoria%2C%20Australia&amp;author=L.%20Schmidl&amp;publication_year=1972&amp;journal=Weed%20Res.&amp;volume=12&amp;pages=37-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/wre.1972.12.issue-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fwre.1972.12.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fwre.1972.12.issue-1"> </span></div> </div></div></div></div></div><div content-id="DSR002C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR002C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gilkey</div> <div class="given-names">H.M.</div></div>. </div>, <div class="source">Weeds of the Pacific Northwest</div> , <div class="year">1957</div><div class="publisher-loc">Corvallis</div><div class="publisher-name">Oregon State College</div></div></div></div></div><div content-id="DSR002C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR002C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Witte</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Ernst</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Adam</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Hartmann</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Chemotypes of 2 pyrrolizidine alkaloid-containing <em>Senecio</em> species</div>, <div class="source">Phytochemistry</div> , <div class="year">1992</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">559</div>-<div class="lpage">65</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Chemotypes%20of%202%20pyrrolizidine%20alkaloid-containing%20Senecio%20species&amp;author=L.%20Witte&amp;author=L.%20Ernst&amp;author=H.%20Adam&amp;author=T.%20Hartmann&amp;publication_year=1992&amp;journal=Phytochemistry&amp;volume=31&amp;pages=559-65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0031-9422(92)90038-R" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0031-9422(92)90038-R" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0031-9422(92)90038-R"> </span></div> </div></div></div></div></div><div content-id="DSR002C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR002C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Van der meijden</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Van der waalskooi</div> <div class="given-names">R.E.</div></div>. </div><div class="article-title">Population ecology of <em>Senecio jacobaea</em> in a sand dune system 1. Reproductive strategy and the biennial habit</div>, <div class="source">Journal of Ecology</div> , <div class="year">1979</div>, vol. <div class="volume">67</div> (pg. <div class="fpage">131</div>-<div class="lpage">53</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Population%20ecology%20of%20Senecio%20jacobaea%20in%20a%20sand%20dune%20system%201.%20Reproductive%20strategy%20and%20the%20biennial%20habit&amp;author=E.%20Van%20der%20meijden&amp;author=R.E.%20Van%20der%20waalskooi&amp;publication_year=1979&amp;journal=Journal%20of%20Ecology&amp;volume=67&amp;pages=131-53" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2307/2259341" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2307%2F2259341" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2307%2F2259341"> </span></div> </div></div></div></div></div><div content-id="DSR002C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR002C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">McLaren</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Mickan</div> <div class="given-names">F.</div></div>. </div>, <div class="source">The Ragwort Management Handbook</div> , <div class="year">1997</div><div class="publisher-loc">Melbourne</div><div class="publisher-name">Department of Natural Resources and Environment</div></div></div></div></div><div content-id="DSR002C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR002C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Joshi</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Vrieling</div> <div class="given-names">K.</div></div>. </div><div class="article-title">The enemy release and EICA hypothesis revisited: incorporating the fundamental difference between specialist and generalist herbivores</div>, <div class="source">Ecol. Lett.</div> , <div class="year">2005</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">704</div>-<div class="lpage">14</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20enemy%20release%20and%20EICA%20hypothesis%20revisited%3A%20incorporating%20the%20fundamental%20difference%20between%20specialist%20and%20generalist%20herbivores&amp;author=J.%20Joshi&amp;author=K.%20Vrieling&amp;publication_year=2005&amp;journal=Ecol.%20Lett.&amp;volume=8&amp;pages=704-14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/ele.2005.8.issue-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fele.2005.8.issue-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fele.2005.8.issue-7"> </span></div> </div></div></div></div></div><div content-id="DSR002C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR002C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Keller</div> <div class="given-names">S.R.</div></div>,  <div class="name"><div class="surname">Taylor</div> <div class="given-names">D.R.</div></div>. </div><div class="article-title">History, chance and adaptation during biological invasion: separating stochastic phenotypic evolution from response to selection</div>, <div class="source">Ecol. Lett.</div> , <div class="year">2008</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">852</div>-<div class="lpage">66</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=History%2C%20chance%20and%20adaptation%20during%20biological%20invasion%3A%20separating%20stochastic%20phenotypic%20evolution%20from%20response%20to%20selection&amp;author=S.R.%20Keller&amp;author=D.R.%20Taylor&amp;publication_year=2008&amp;journal=Ecol.%20Lett.&amp;volume=11&amp;pages=852-66" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/ele.2008.11.issue-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fele.2008.11.issue-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fele.2008.11.issue-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18422638" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR002C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR002C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Doorduin</div> <div class="given-names">L.J.</div></div>,  <div class="name"><div class="surname">van den Hof</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Vrieling</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Joshi</div> <div class="given-names">J.</div></div>. </div><div class="article-title">The lack of genetic bottleneck in invasive Tansy ragwort populations suggests multiple source populations</div>, <div class="source">Basic Appl. Ecol.</div> , <div class="year">2010</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">244</div>-<div class="lpage">50</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20lack%20of%20genetic%20bottleneck%20in%20invasive%20Tansy%20ragwort%20populations%20suggests%20multiple%20source%20populations&amp;author=L.J.%20Doorduin&amp;author=K.%20van%20den%20Hof&amp;author=K.%20Vrieling&amp;author=J.%20Joshi&amp;publication_year=2010&amp;journal=Basic%20Appl.%20Ecol.&amp;volume=11&amp;pages=244-50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.baae.2009.12.007" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.baae.2009.12.007" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.baae.2009.12.007"> </span></div> </div></div></div></div></div><div content-id="DSR002C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR002C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Estrada-Mena</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Estrada</div> <div class="given-names">F.J.</div></div>,  <div class="name"><div class="surname">Ulloa-Arvizu</div> <div class="given-names">R.</div></div>, et al. </div><div class="article-title">Blood group O alleles innative Americans: implications in the peopling of the Americas</div>, <div class="source">Am. J. Phys. Anthropol.</div> , <div class="year">2010</div>, vol. <div class="volume">142</div> (pg. <div class="fpage">85</div>-<div class="lpage">94</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Blood%20group%20O%20alleles%20innative%20Americans%3A%20implications%20in%20the%20peopling%20of%20the%20Americas&amp;author=B.%20Estrada-Mena&amp;author=F.J.%20Estrada&amp;author=R.%20Ulloa-Arvizu&amp;publication_year=2010&amp;journal=Am.%20J.%20Phys.%20Anthropol.&amp;volume=142&amp;pages=85-94" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19862808" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR002C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR002C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Leniaud</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Dedeine</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Pichon</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Dupont</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Bagneres</div> <div class="given-names">A.G.</div></div>. </div><div class="article-title">Geographical distribution, genetic diversity and social organization of a new European termite, <em>Reticulitermes urbis</em> (Isoptera: Rhinotermitidae)</div>, <div class="source">Biol. Invasions</div> , <div class="year">2010</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">1389</div>-<div class="lpage">402</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Geographical%20distribution%2C%20genetic%20diversity%20and%20social%20organization%20of%20a%20new%20European%20termite%2C%20Reticulitermes%20urbis%20(Isoptera%3A%20Rhinotermitidae)&amp;author=L.%20Leniaud&amp;author=F.%20Dedeine&amp;author=A.%20Pichon&amp;author=S.%20Dupont&amp;author=A.G.%20Bagneres&amp;publication_year=2010&amp;journal=Biol.%20Invasions&amp;volume=12&amp;pages=1389-402" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10530-009-9555-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10530-009-9555-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10530-009-9555-8"> </span></div> </div></div></div></div></div><div content-id="DSR002C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR002C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zimmermann</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Ritz</div> <div class="given-names">C.M.</div></div>,  <div class="name"><div class="surname">Hirsch</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Renison</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Wesche</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Hensen</div> <div class="given-names">I.</div></div>. </div><div class="article-title">Highly reduced genetic diversity of <em>Rosa rubiginosa</em> L. populations in the invasive range</div>, <div class="source">Int. J. Plant Sci.</div> , <div class="year">2010</div>, vol. <div class="volume">171</div> (pg. <div class="fpage">435</div>-<div class="lpage">46</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Highly%20reduced%20genetic%20diversity%20of%20Rosa%20rubiginosa%20L.%20populations%20in%20the%20invasive%20range&amp;author=H.%20Zimmermann&amp;author=C.M.%20Ritz&amp;author=H.%20Hirsch&amp;author=D.%20Renison&amp;author=K.%20Wesche&amp;author=I.%20Hensen&amp;publication_year=2010&amp;journal=Int.%20J.%20Plant%20Sci.&amp;volume=171&amp;pages=435-46" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1086/651244" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1086%2F651244" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1086%2F651244"> </span></div> </div></div></div></div></div><div content-id="DSR002C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR002C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shendure</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Ji</div> <div class="given-names">H.L.</div></div>. </div><div class="article-title">Next-generation DNA sequencing</div>, <div class="source">Nat. Biotechnol.</div> , <div class="year">2008</div>, vol. <div class="volume">26</div> (pg. <div class="fpage">1135</div>-<div class="lpage">45</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Next-generation%20DNA%20sequencing&amp;author=J.%20Shendure&amp;author=H.L.%20Ji&amp;publication_year=2008&amp;journal=Nat.%20Biotechnol.&amp;volume=26&amp;pages=1135-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt1486" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt1486" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt1486"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18846087" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR002C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR002C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wolfe</div> <div class="given-names">K.H.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">W.H.</div></div>,  <div class="name"><div class="surname">Sharp</div> <div class="given-names">P.M.</div></div>. </div><div class="article-title">Rates of nucleotide substitution vary greatly among plant mitochondrial, chloroplast, and nuclear DNAs</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">1987</div>, vol. <div class="volume">84</div> (pg. <div class="fpage">9054</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rates%20of%20nucleotide%20substitution%20vary%20greatly%20among%20plant%20mitochondrial%2C%20chloroplast%2C%20and%20nuclear%20DNAs&amp;author=K.H.%20Wolfe&amp;author=W.H.%20Li&amp;author=P.M.%20Sharp&amp;publication_year=1987&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=84&amp;pages=9054-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.84.24.9054" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.84.24.9054" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.84.24.9054"> </span></div> </div></div></div></div></div><div content-id="DSR002C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR002C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Diekmann</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Hodkinson</div> <div class="given-names">T.R.</div></div>,  <div class="name"><div class="surname">Wolfe</div> <div class="given-names">K.H.</div></div>,  <div class="name"><div class="surname">van den Bekerom</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Dix</div> <div class="given-names">P.J.</div></div>,  <div class="name"><div class="surname">Barth</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Complete chloroplast genome sequence of a major allogamous forage species, perennial ryegrass (<em>Lolium perenne</em> L.)</div>, <div class="source">DNA Res.</div> , <div class="year">2009</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">165</div>-<div class="lpage">76</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complete%20chloroplast%20genome%20sequence%20of%20a%20major%20allogamous%20forage%20species%2C%20perennial%20ryegrass%20(Lolium%20perenne%20L.)&amp;author=K.%20Diekmann&amp;author=T.R.%20Hodkinson&amp;author=K.H.%20Wolfe&amp;author=R.%20van%20den%20Bekerom&amp;author=P.J.%20Dix&amp;author=S.%20Barth&amp;publication_year=2009&amp;journal=DNA%20Res.&amp;volume=16&amp;pages=165-76" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsp008" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsp008" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsp008"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19414502" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR002C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR002C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kumar</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Hahn</div> <div class="given-names">F.M.</div></div>,  <div class="name"><div class="surname">McMahan</div> <div class="given-names">C.M.</div></div>,  <div class="name"><div class="surname">Cornish</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Whalen</div> <div class="given-names">M.C.</div></div>. </div><div class="article-title">Comparative analysis of the complete sequence of the plastid genome of <em>Parthenium argentatum</em> and identification of DNA barcodes to differentiate <em>Parthenium</em> species and lines</div>, <div class="source">BMC Plant Biol.</div> , <div class="year">2009</div>, vol. <div class="volume">9</div> </div></div></div></div><div content-id="DSR002C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR002C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wu</div> <div class="given-names">F.H.</div></div>,  <div class="name"><div class="surname">Chan</div> <div class="given-names">M.T.</div></div>,  <div class="name"><div class="surname">Liao</div> <div class="given-names">D.C.</div></div>, et al. </div><div class="article-title">Complete chloroplast genome of Oncidium Gower Ramsey and evaluation of molecular markers for identification and breeding in Oncidiinae</div>, <div class="source">BMC Plant Biol.</div> , <div class="year">2010</div>, vol. <div class="volume">10</div> </div></div></div></div><div content-id="DSR002C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR002C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">McCauley</div> <div class="given-names">D.E.</div></div>,  <div class="name"><div class="surname">Stevens</div> <div class="given-names">J.E.</div></div>,  <div class="name"><div class="surname">Peroni</div> <div class="given-names">P.A.</div></div>,  <div class="name"><div class="surname">Raveill</div> <div class="given-names">J.A.</div></div>. </div><div class="article-title">The spatial distribution of chloroplast DNA and allozyme polymorphisms within a population of <em>Silene alba</em> (Caryophyllaceae).</div>, <div class="source">Am. J. Bot.</div> , <div class="year">1996</div>, vol. <div class="volume">83</div> (pg. <div class="fpage">727</div>-<div class="lpage">31</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20spatial%20distribution%20of%20chloroplast%20DNA%20and%20allozyme%20polymorphisms%20within%20a%20population%20of%20Silene%20alba%20(Caryophyllaceae).&amp;author=D.E.%20McCauley&amp;author=J.E.%20Stevens&amp;author=P.A.%20Peroni&amp;author=J.A.%20Raveill&amp;publication_year=1996&amp;journal=Am.%20J.%20Bot.&amp;volume=83&amp;pages=727-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2307/2445849" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2307%2F2445849" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2307%2F2445849"> </span></div> </div></div></div></div></div><div content-id="DSR002C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR002C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Small</div> <div class="given-names">R.L.</div></div>,  <div class="name"><div class="surname">Cronn</div> <div class="given-names">R.C.</div></div>,  <div class="name"><div class="surname">Wendel</div> <div class="given-names">J.F.</div></div>. </div><div class="article-title">Use of nuclear genes for phylogeny reconstruction in plants</div>, <div class="source">Aust. Syst. Bot.</div> , <div class="year">2004</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">145</div>-<div class="lpage">70</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Use%20of%20nuclear%20genes%20for%20phylogeny%20reconstruction%20in%20plants&amp;author=R.L.%20Small&amp;author=R.C.%20Cronn&amp;author=J.F.%20Wendel&amp;publication_year=2004&amp;journal=Aust.%20Syst.%20Bot.&amp;volume=17&amp;pages=145-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1071/SB03015" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1071%2FSB03015" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1071%2FSB03015"> </span></div> </div></div></div></div></div><div content-id="DSR002C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR002C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Triboush</div> <div class="given-names">S.O.</div></div>,  <div class="name"><div class="surname">Danilenko</div> <div class="given-names">N.G.</div></div>,  <div class="name"><div class="surname">Davydenko</div> <div class="given-names">O.G.</div></div>. </div><div class="article-title">A method for isolation of chloroplast DNA and mitochondrial DNA from sunflower</div>, <div class="source">Plant Mol. Biol. Rep.</div> , <div class="year">1998</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">183</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20method%20for%20isolation%20of%20chloroplast%20DNA%20and%20mitochondrial%20DNA%20from%20sunflower&amp;author=S.O.%20Triboush&amp;author=N.G.%20Danilenko&amp;author=O.G.%20Davydenko&amp;publication_year=1998&amp;journal=Plant%20Mol.%20Biol.%20Rep.&amp;volume=16&amp;pages=183-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1023/A:1007487806583" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1023%2FA:1007487806583" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1023%2FA:1007487806583"> </span></div> </div></div></div></div></div><div content-id="DSR002C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR002C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Doyle</div> <div class="given-names">J.J.</div></div>,  <div class="name"><div class="surname">Doyle</div> <div class="given-names">J.L.</div></div>. </div><div class="article-title">A rapid DNA isolation procedure for small quantities of fresh leaf tissue</div>, <div class="source">Phytochem. Bull.</div> , <div class="year">1987</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">11</div>-<div class="lpage">5</div>)</div></div></div></div><div content-id="DSR002C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR002C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rozen</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Skaletsky</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Primer3 on the WWW for general users and for biologist programmers</div>, <div class="source">Methods Mol. Biol.</div> , <div class="year">2000</div>, vol. <div class="volume">132</div> (pg. <div class="fpage">365</div>-<div class="lpage">86</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Primer3%20on%20the%20WWW%20for%20general%20users%20and%20for%20biologist%20programmers&amp;author=S.%20Rozen&amp;author=H.%20Skaletsky&amp;publication_year=2000&amp;journal=Methods%20Mol.%20Biol.&amp;volume=132&amp;pages=365-86" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10547847" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR002C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR002C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zerbino</div> <div class="given-names">D.R.</div></div>,  <div class="name"><div class="surname">Birney</div> <div class="given-names">E.</div></div>. </div><div class="article-title">Velvet: algorithms for de novo short read assembly using de Bruijn graphs</div>, <div class="source">Genome Res.</div> , <div class="year">2008</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">821</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Velvet%3A%20algorithms%20for%20de%20novo%20short%20read%20assembly%20using%20de%20Bruijn%20graphs&amp;author=D.R.%20Zerbino&amp;author=E.%20Birney&amp;publication_year=2008&amp;journal=Genome%20Res.&amp;volume=18&amp;pages=821-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.074492.107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.074492.107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.074492.107"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18349386" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR002C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR002C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kurtz</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Phillippy</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Declcher</div> <div class="given-names">A.L.</div></div>, et al. </div><div class="article-title">Versatile and open software for comparing large genomes</div>, <div class="source">Genome Biol.</div> , <div class="year">2004</div>, vol. <div class="volume">5</div> </div></div></div></div><div content-id="DSR002C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR002C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wyman</div> <div class="given-names">S.K.</div></div>,  <div class="name"><div class="surname">Jansen</div> <div class="given-names">R.K.</div></div>,  <div class="name"><div class="surname">Boore</div> <div class="given-names">J.L.</div></div>. </div><div class="article-title">Automatic annotation of organellar genomes with DOGMA</div>, <div class="source">Bioinformatics</div> , <div class="year">2004</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">3252</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Automatic%20annotation%20of%20organellar%20genomes%20with%20DOGMA&amp;author=S.K.%20Wyman&amp;author=R.K.%20Jansen&amp;author=J.L.%20Boore&amp;publication_year=2004&amp;journal=Bioinformatics&amp;volume=20&amp;pages=3252-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bth352" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbth352" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbth352"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15180927" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR002C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR002C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Conant</div> <div class="given-names">G.C.</div></div>,  <div class="name"><div class="surname">Wolfe</div> <div class="given-names">K.H.</div></div>. </div><div class="article-title">GenomeVx: simple web-based creation of editable circular chromosome maps</div>, <div class="source">Bioinformatics</div> , <div class="year">2008</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">861</div>-<div class="lpage">2</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GenomeVx%3A%20simple%20web-based%20creation%20of%20editable%20circular%20chromosome%20maps&amp;author=G.C.%20Conant&amp;author=K.H.%20Wolfe&amp;publication_year=2008&amp;journal=Bioinformatics&amp;volume=24&amp;pages=861-2" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btm598" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtm598" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtm598"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18227121" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR002C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR002C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Maddison</div> <div class="given-names">W.P.</div></div>,  <div class="name"><div class="surname">Maddison</div> <div class="given-names">D.R.</div></div>. </div>, <div class="source">MacClade Version 4.06 OSX</div> , <div class="year">2003</div><div class="publisher-loc">Sunderland, MA, USA</div><div class="publisher-name">Sinauer Associates</div></div></div></div></div><div content-id="DSR002C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR002C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Swofford</div> <div class="given-names">D.L.</div></div>. </div>, <div class="source">Phylogenetic Analysis Using Parsimony 4.0b</div> , <div class="year">2002</div><div class="edition">10th edition</div><div class="publisher-loc">Sunderland</div><div class="publisher-name">Sinauer Associated</div></div></div></div></div><div content-id="DSR002C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR002C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Felsenstein</div> <div class="given-names">J.</div></div>. </div>, <div class="source">PHYLIP Phylogenetic Inference Programs Version 3.572</div> , <div class="year">1995</div><div class="publisher-name">University of Washington</div></div></div></div></div><div content-id="DSR002C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR002C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bao</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Guo</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">J.W.</div></div>,  <div class="name"><div class="surname">Zhou</div> <div class="given-names">R.C.</div></div>,  <div class="name"><div class="surname">Lu</div> <div class="given-names">X.M.</div></div>,  <div class="name"><div class="surname">Shi</div> <div class="given-names">S.H.</div></div>. </div><div class="article-title">MapView: visualization of short reads alignment on a desktop computer</div>, <div class="source">Bioinformatics</div> , <div class="year">2009</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">1554</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MapView%3A%20visualization%20of%20short%20reads%20alignment%20on%20a%20desktop%20computer&amp;author=H.%20Bao&amp;author=H.%20Guo&amp;author=J.W.%20Wang&amp;author=R.C.%20Zhou&amp;author=X.M.%20Lu&amp;author=S.H.%20Shi&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=1554-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp255" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp255" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp255"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19369497" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR002C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR002C42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cronn</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Liston</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Parks</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Gernandt</div> <div class="given-names">D.S.</div></div>,  <div class="name"><div class="surname">Shen</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Mockler</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Multiplex sequencing of plant chloroplast genomes using Solexa sequencing-by-synthesis technology</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2008</div>, vol. <div class="volume">36</div> </div></div></div></div><div content-id="DSR002C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR002C43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Timme</div> <div class="given-names">R.E.</div></div>,  <div class="name"><div class="surname">Kuehl</div> <div class="given-names">J.V.</div></div>,  <div class="name"><div class="surname">Boore</div> <div class="given-names">J.L.</div></div>,  <div class="name"><div class="surname">Jansen</div> <div class="given-names">R.K.</div></div>. </div><div class="article-title">A comparative analysis of the <em>Lactuca</em> and <em>Helianthus</em> (Asteraceae) plastid genomes: identification of divergent regions and categorization of shared repeats</div>, <div class="source">Am. J. Bot.</div> , <div class="year">2007</div>, vol. <div class="volume">94</div> (pg. <div class="fpage">302</div>-<div class="lpage">12</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20comparative%20analysis%20of%20the%20Lactuca%20and%20Helianthus%20(Asteraceae)%20plastid%20genomes%3A%20identification%20of%20divergent%20regions%20and%20categorization%20of%20shared%20repeats&amp;author=R.E.%20Timme&amp;author=J.V.%20Kuehl&amp;author=J.L.%20Boore&amp;author=R.K.%20Jansen&amp;publication_year=2007&amp;journal=Am.%20J.%20Bot.&amp;volume=94&amp;pages=302-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3732/ajb.94.3.302" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3732%2Fajb.94.3.302" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3732%2Fajb.94.3.302"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21636403" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR002C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR002C44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dempewolf</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Kane</div> <div class="given-names">N.C.</div></div>,  <div class="name"><div class="surname">Ostevik</div> <div class="given-names">K.L.</div></div>, et al. </div><div class="article-title">Establishing genomic tools and resources for <em>Guizotia abyssinica</em>; the development of a library of expressed sequence tags, microsatellite loci, and the sequencing of its chloroplast genome</div>, <div class="source">Mol. Ecol. Resour.</div> , <div class="year">2010</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">1048</div>-<div class="lpage">58</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Establishing%20genomic%20tools%20and%20resources%20for%20Guizotia%20abyssinica%3B%20the%20development%20of%20a%20library%20of%20expressed%20sequence%20tags%2C%20microsatellite%20loci%2C%20and%20the%20sequencing%20of%20its%20chloroplast%20genome&amp;author=H.%20Dempewolf&amp;author=N.C.%20Kane&amp;author=K.L.%20Ostevik&amp;publication_year=2010&amp;journal=Mol.%20Ecol.%20Resour.&amp;volume=10&amp;pages=1048-58" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/men.2010.10.issue-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmen.2010.10.issue-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmen.2010.10.issue-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21565115" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR002C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR002C45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Small</div> <div class="given-names">R.L.</div></div>,  <div class="name"><div class="surname">Ryburn</div> <div class="given-names">J.A.</div></div>,  <div class="name"><div class="surname">Cronn</div> <div class="given-names">R.C.</div></div>,  <div class="name"><div class="surname">Seelanan</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Wendel</div> <div class="given-names">J.F.</div></div>. </div><div class="article-title">Tortoise and the hare: choosing between noncoding plastome and nuclear ADH sequences for phylogeny reconstruction in a recently diverged plant group</div>, <div class="source">Am. J. Bot.</div> , <div class="year">1998</div>, vol. <div class="volume">85</div> (pg. <div class="fpage">1301</div>-<div class="lpage">15</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Tortoise%20and%20the%20hare%3A%20choosing%20between%20noncoding%20plastome%20and%20nuclear%20ADH%20sequences%20for%20phylogeny%20reconstruction%20in%20a%20recently%20diverged%20plant%20group&amp;author=R.L.%20Small&amp;author=J.A.%20Ryburn&amp;author=R.C.%20Cronn&amp;author=T.%20Seelanan&amp;author=J.F.%20Wendel&amp;publication_year=1998&amp;journal=Am.%20J.%20Bot.&amp;volume=85&amp;pages=1301-15" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2307/2446640" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2307%2F2446640" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2307%2F2446640"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21685016" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR002C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR002C46" class="ref-content "><div class="label -label">46</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Palmer</div> <div class="given-names">J.D.</div></div>. </div><div class="article-title">Contrasting modes and tempos of genome evolution in land plant organelles</div>, <div class="source">Trends Genet.</div> , <div class="year">1990</div>, vol. <div class="volume">6</div> (pg. <div class="fpage">115</div>-<div class="lpage">20</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Contrasting%20modes%20and%20tempos%20of%20genome%20evolution%20in%20land%20plant%20organelles&amp;author=J.D.%20Palmer&amp;publication_year=1990&amp;journal=Trends%20Genet.&amp;volume=6&amp;pages=115-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0168-9525(90)90125-P" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0168-9525(90)90125-P" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0168-9525(90)90125-P"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2132730" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2011. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/2.5), which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=365332&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/2/10.1093/dnares/dsr002/2/dsr002-dsr002supp_Table1.doc?Expires=2147483647&amp;Signature=Nk7xyBOwN0XLx-C18f5L5xUTyTHIJCbR-JrLx721g3vMCugdo83KZ9QOgOfjzx1XTdOpqoRqSqvAxEo3C1JcI6ByyjZHNr2O33IIMvs2fD3LxX85YyNptgKRA6qu~mA~jt5kDHLzz7eARkIPMmTp8PjZePBH3AoJFsuBKiR01~H732L1TU0~HGkz9uzR7unegxNFUy4cM8TluB~QO0k2hMx4NDwkAQpBvhmmib-CNe76yLdheo-Ewpr22JyC6Iw-QXaNdEkcyOn1xWCJbameqQ~13XQj-RGA~AeIXlInxdvPfUOWdt7NDjrsysebEdqEAlSvmFaZn-zbYbxt8OSJpw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - doc file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">496</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000290061400002&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">61</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsr002"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="365332"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000290061400002&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f18%2f2%2f93%2f365332">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000290061400002&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (61)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F18%2F2%2F93%2F365332" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/365332'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Complete Chloroplast Genome of 17 Individuals of Pest Species Jacobaea vulgaris: SNPs, Microsatellites and Barcoding Markers for Population and Phylogenetic Studies" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsr002");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f18%2f2%2f93%2f365332">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

