

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" /><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info = {"beacon":"bam.nr-data.net","errorBeacon":"bam.nr-data.net","licenseKey":"8d76ca6f26","applicationID":"29565620","transactionName":"ZQBRMkVUCEtRVkINXlxKcjVnGgdKRFxVCFQcBEAWTw==","queueTime":0,"applicationTime":1624,"agent":"","atts":""}</script><script type="text/javascript">(window.NREUM||(NREUM={})).loader_config={xpid:"XAYBV15WGwEJUVdXAQMH"};window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{s.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(21),s={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(s.console=!0,o.indexOf("dev")!==-1&&(s.dev=!0),o.indexOf("nr_dev")!==-1&&(s.nrDev=!0))}catch(c){}s.nrDev&&i.on("internal-error",function(t){r(t.stack)}),s.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),s.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(s,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,s){try{l?l-=1:o(s||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,c.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:c.now();i("err",[t,n])}var i=t("handle"),a=t(22),s=t("ee"),c=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;c.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),c.xhrWrappable&&t(14),d=!0)}s.on("fn-start",function(t,e,n){d&&(l+=1)}),s.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),s.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),s.on("internal-error",function(t){i("ierr",[t,c.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){M++,N=y.hash,this[u]=g.now()}function o(){M--,y.hash!==N&&i(0,!0);var t=g.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+y,e])}function a(t,e){t.on(e,function(){this[e]=g.now()})}var s="-start",c="-end",f="-body",u="fn"+s,d="fn"+c,p="cb"+s,l="cb"+c,h="jsTime",m="fetch",v="addEventListener",w=window,y=w.location,g=t("loader");if(w[v]&&g.xhrWrappable){var b=t(10),x=t(11),E=t(8),O=t(6),P=t(13),R=t(7),T=t(14),L=t(9),j=t("ee"),S=j.get("tracer");t(15),g.features.spa=!0;var N,M=0;j.on(u,r),j.on(p,r),j.on(d,o),j.on(l,o),j.buffer([u,d,"xhr-done","xhr-resolved"]),O.buffer([u]),P.buffer(["setTimeout"+c,"clearTimeout"+s,u]),T.buffer([u,"new-xhr","send-xhr"+s]),R.buffer([m+s,m+"-done",m+f+s,m+f+c]),E.buffer(["newURL"]),b.buffer([u]),x.buffer(["propagate",p,l,"executor-err","resolve"+s]),S.buffer([u,"no-"+u]),L.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(T,"send-xhr"+s),a(j,"xhr-resolved"),a(j,"xhr-done"),a(R,m+s),a(R,m+"-done"),a(L,"new-jsonp"),a(L,"jsonp-end"),a(L,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,M>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),s=t(12),c="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",y="pushState",g=t("loader");g.features.stn=!0,t(8);var b=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof b&&(this.bstStart=g.now())}),o.on(v,function(t,e){var n=t[0];n instanceof b&&i("bst",[n,e,this.bstStart,g.now()])}),a.on(m,function(t,e,n){this.bstStart=g.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),this.bstType])}),s.on(m,function(){this.bstStart=g.now()}),s.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),"requestAnimationFrame"])}),o.on(y+l,function(t){this.time=g.now(),this.startPath=location.pathname+location.hash}),o.on(y+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+c]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+c]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+c]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){s.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),s=t(24)(a,!0),c=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=c(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?s(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=r.apply(this,arguments);return o.emit(n+"start",arguments,t),t.then(function(e){return o.emit(n+"end",[null,e],t),e},function(e){throw o.emit(n+"end",[e],t),e})})}var o=t("ee").get("fetch"),i=t(21);e.exports=o;var a=window,s="fetch-",c=s+"body-",f=["arrayBuffer","blob","json","text","formData"],u=a.Request,d=a.Response,p=a.fetch,l="prototype";u&&d&&p&&(i(f,function(t,e){r(u[l],e,c),r(d[l],e,c)}),r(a,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t(24)(r);e.exports=r,o.inPlace(window.history,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){c.emit("jsonp-error",[],p),c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=s(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),c.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function s(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var c=t("ee").get("jsonp"),f=t(24)(c);if(e.exports=c,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-"),c.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t(24)(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=s(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t(24),a=t("ee").get("promise"),s=i(a),c=t(21),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;c(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),s=f.resolve(i);return s}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),c(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=s(t[0],"resolve-",this),t[1]=s(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),s.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=s(t[0],"cb-",this),t[1]=s(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t(24)(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t(24)(i),s="setTimeout",c="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[s,"setImmediate"],s+d),a.inPlace(window,[c],c+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(c+u,r),i.on(s+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",s)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,y,"fn-",s)}function i(t){g.push(t),h&&(x?x.then(a):v?v(a):(E=-E,O.data=E))}function a(){for(var t=0;t<g.length;t++)r([],g[t]);g.length&&(g=[])}function s(t,e){return e}function c(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t(24)(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",y=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],g=[];e.exports=u;var b=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(c(l,b),b.prototype=l.prototype,d.inPlace(b.prototype,["open","send"],"-xhr-",s),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var x=m&&m.resolve();if(!v&&!m){var E=1,O=document.createTextNode(E);new h(a).observe(O,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<d;r++)t.removeEventListener(u[r],this.listener,!1);if(!e.aborted){if(n.duration=a.now()-this.startTime,4===t.readyState){e.status=t.status;var i=o(t,this.lastSize);if(i&&(n.rxSize=i),this.sameOrigin){var c=t.getResponseHeader("X-NewRelic-App-Data");c&&(e.cat=c.split(", ").pop())}}else e.status=0;n.cbTime=this.cbTime,f.emit("xhr-done",[t],t),s("xhr",[e,n,this.startTime])}}}function o(t,e){var n=t.responseType;if("json"===n&&null!==e)return e;var r="arraybuffer"===n||"blob"===n||"json"===n?t.response:t.responseText;return h(r)}function i(t,e){var n=c(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.sameOrigin=n.sameOrigin}var a=t("loader");if(a.xhrWrappable){var s=t("handle"),c=t(16),f=t("ee"),u=["load","error","abort","timeout"],d=u.length,p=t("id"),l=t(19),h=t(18),m=window.XMLHttpRequest;a.features.xhr=!0,t(14),f.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,l&&(l>34||l<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),f.on("open-xhr-start",function(t){this.params={method:t[0]},i(this,t[1]),this.metrics={}}),f.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid)}),f.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=h(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"===t.type&&(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{f.emit("internal-error",[n])}catch(r){}}};for(var s=0;s<d;s++)e.addEventListener(u[s],this.listener,!1)}),f.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),f.on("xhr-load-added",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),f.on("xhr-load-removed",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),f.on("addEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-added",[t[1],t[2]],e)}),f.on("removeEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-removed",[t[1],t[2]],e)}),f.on("fn-start",function(t,e,n){e instanceof m&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),f.on("fn-end",function(t,e){this.xhrCbStart&&f.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)})}},{}],16:[function(t,e,n){e.exports=function(t){var e=document.createElement("a"),n=window.location,r={};e.href=t,r.port=e.port;var o=e.href.split("://");!r.port&&o[1]&&(r.port=o[1].split("/")[0].split("@").pop().split(":")[1]),r.port&&"0"!==r.port||(r.port="https"===o[0]?"443":"80"),r.hostname=e.hostname||n.hostname,r.pathname=e.pathname,r.protocol=o[0],"/"!==r.pathname.charAt(0)&&(r.pathname="/"+r.pathname);var i=!e.protocol||":"===e.protocol||e.protocol===n.protocol,a=e.hostname===document.domain&&e.port===n.port;return r.sameOrigin=i&&(!e.hostname||a),r}},{}],17:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(s(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(21),s=t(22),c=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(c.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw c.emit("fn-err",[arguments,this,t],n),t}finally{c.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now()])}},{}],18:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],19:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],20:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var s=navigator.userAgent,c=s.match(a);c&&s.indexOf("Chrome")===-1&&s.indexOf("Chromium")===-1&&(o="Safari",i=c[1])}e.exports={agent:o,version:i,match:r}},{}],21:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],22:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],23:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],24:[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(22),a="nr@original",s=Object.prototype.hasOwnProperty,c=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,s,c;try{a=this,r=i(arguments),s="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],s])}u(e+"start",[r,a,o],s);try{return c=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],s),d}finally{u(e+"end",[r,a,c],s)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,s,c,f="-"===o.charAt(0);for(c=0;c<e.length;c++)s=e[c],a=t[s],r(a)||(t[s]=n(a,f?s+o:o,i,s))}function u(n,r,o){if(!c||e){var i=c;c=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}c=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)s.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?c(t,s,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),s=m(n),c=s.length,f=0;f<c;f++)s[f].apply(a,r);var d=u[g[n]];return d&&d.push([b,n,r,a]),a}}function l(t,e){y[t]=m(t).concat(e)}function h(t,e){var n=y[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return y[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",g[n]=e,e in u||(u[e]=[])})}var y={},g={},b={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return b}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var s="nr@context",c=t("gos"),f=t(21),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=x.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(g,function(e,n){t[e]||(t[e]=n)}),c("mark",["onload",a()+x.offset],null,"api");var n=l.createElement("script");n.src="https://"+t.agent,e.parentNode.insertBefore(n,e)}}function o(){"complete"===l.readyState&&i()}function i(){c("mark",["domContent",a()+x.offset],null,"api")}function a(){return O.exists&&performance.now?Math.round(performance.now()):(s=Math.max((new Date).getTime(),s))-x.offset}var s=(new Date).getTime(),c=t("handle"),f=t(21),u=t("ee"),d=t(20),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var y=""+location,g={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1099.min.js"},b=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),x=e.exports={offset:s,now:a,origin:y,features:{},xhrWrappable:b,userAgent:d};t(17),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),c("mark",["firstbyte",s],null,"api");var E=0,O=t(23)},{}]},{},["loader",2,15,5,3,4]);</script>
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/13/3/89/347450","@type":"ScholarlyArticle","name":"Search and Classification of Potential Minisatellite Sequences from Bacterial Genomes","datePublished":"2006-01-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/13/3","@type":"PublicationIssue","issueNumber":"3","datePublished":"2006-01-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsl004","keywords":["information decomposition","modified profile analysis","minisatellites","bacterial genomes"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/13/3/89/347450","author":[{"name":"Shelenkov, Andrew","affiliation":"Bioengineering Centre of Russian Academy of Sciences, Prospect 60-tya Oktyabrya  7/1, 117312 Moscow, Russia","@type":"Person"},{"name":"Skryabin, Konstantin","affiliation":"Bioengineering Centre of Russian Academy of Sciences, Prospect 60-tya Oktyabrya  7/1, 117312 Moscow, Russia","@type":"Person"},{"name":"Korotkov, Eugene","affiliation":"Bioengineering Centre of Russian Academy of Sciences, Prospect 60-tya Oktyabrya  7/1, 117312 Moscow, Russia","@type":"Person"}],"description":"Abstract.  We used the method of Information Decomposition developed by us to identify the latent dinucleotide periodicity regions in bacterial genomes. The num","pageStart":"89","pageEnd":"102","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/13/3/0/m_dnaresearch13_3.cover.gif?Expires=2147483647&Signature=anx2J~8v0IVYzlePG5tUbncUtEx2PsYUKWb~Qpjo1jhMHlu0islS2eGWU9i~f0nzFtaqNwAgI5-no5gavB~5K2oqu5oELPuFU9PB0RbXx30rT5CtFH7Z7Tak8QE4~Qv5CaQ-S-laJYNwnkzL6WJ228M7M2qJZAZbrbUpUHT7RG37R-U5f~bJ7ij8Hr1oUW8kL1EexLVBqSK~E8j95q-vzg7TEqouIZZVhlOy-gxfExmOp8ieedz1ReyVM4Msqvq9iUEHn2KT3PqF7FeNdH-Lqt5xtcPFEJYTrL4VFii2sM~Knj8tJAAT5R0NldIHOqOze-~nk5Mk6oyjGclZXc9hCA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Search and Classification of Potential Minisatellite Sequences from Bacterial Genomes","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/13/3/0/m_dnaresearch13_3.cover.gif?Expires=2147483647&Signature=anx2J~8v0IVYzlePG5tUbncUtEx2PsYUKWb~Qpjo1jhMHlu0islS2eGWU9i~f0nzFtaqNwAgI5-no5gavB~5K2oqu5oELPuFU9PB0RbXx30rT5CtFH7Z7Tak8QE4~Qv5CaQ-S-laJYNwnkzL6WJ228M7M2qJZAZbrbUpUHT7RG37R-U5f~bJ7ij8Hr1oUW8kL1EexLVBqSK~E8j95q-vzg7TEqouIZZVhlOy-gxfExmOp8ieedz1ReyVM4Msqvq9iUEHn2KT3PqF7FeNdH-Lqt5xtcPFEJYTrL4VFii2sM~Knj8tJAAT5R0NldIHOqOze-~nk5Mk6oyjGclZXc9hCA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Search and Classification of Potential Minisatellite Sequences from Bacterial Genomes" />
    <meta property="og:description" content="Abstract.  We used the method of Information Decomposition developed by us to identify the latent dinucleotide periodicity regions in bacterial genomes. The num" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/13/3/89/347450" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2006-01-01" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Shelenkov, Andrew" /><meta name="citation_author_institution" content="Bioengineering Centre of Russian Academy of Sciences, Prospect 60-tya Oktyabrya7/1, 117312 Moscow, Russia" /><meta name="citation_author" content="Skryabin, Konstantin" /><meta name="citation_author_institution" content="Bioengineering Centre of Russian Academy of Sciences, Prospect 60-tya Oktyabrya7/1, 117312 Moscow, Russia" /><meta name="citation_author" content="Korotkov, Eugene" /><meta name="citation_author_institution" content="Bioengineering Centre of Russian Academy of Sciences, Prospect 60-tya Oktyabrya7/1, 117312 Moscow, Russia" /><meta name="citation_title" content="Search and Classification of Potential Minisatellite Sequences from Bacterial Genomes" /><meta name="citation_firstpage" content="89" /><meta name="citation_lastpage" content="102" /><meta name="citation_doi" content="10.1093/dnares/dsl004" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="13" /><meta name="citation_issue" content="3" /><meta name="citation_publication_date" content="2006/01/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Molecular basis of genetic instability of triplet repeats; J. Biol. Chem.;  citation_year=1996;  citation_volume=271;  citation_pages=2875-2878; " /><meta name="citation_reference" content="citation_title=DNA secondary structures and the evolution of hypervariable tandem arrays; J. Biol. Chem.;  citation_year=1997;  citation_volume=272;  citation_pages=9517-9523; " /><meta name="citation_reference" content="citation_title=Fragile X syndrome unstable element, p(CCG)n, and other simple tandem repeat sequences are binding sites for specific nuclear proteins; Hum. Mol. Genet.;  citation_year=1993;  citation_volume=2;  citation_pages=1429-1435; " /><meta name="citation_reference" content="citation_title=(CT)n (GA)n repeats and heat shock elements have distinct roles in chromatin structure and transcriptional activation of the Drosophila hsp26 gene; Mol. Cell. Biol.;  citation_year=1993;  citation_volume=13;  citation_pages=2802-2814; " /><meta name="citation_reference" content="citation_title=Multiple-locus variable-number tandem repeat analysis reveals genetic relationships within Bacillus anthracis; J. Bacteriol.;  citation_year=2000;  citation_volume=182;  citation_pages=2928-2936; " /><meta name="citation_reference" content="citation_title=Genetic diversity in the Mycobacterium tuberculosis complex based on variable numbers of tandem DNA repeats; Microbiology;  citation_year=1998;  citation_volume=144;  citation_pages=1189-1196; " /><meta name="citation_reference" content="citation_title=Variable human minisatellite-like regions in the Mycobacterium tuberculosis genome; Mol. Microbiol.;  citation_year=2000;  citation_volume=36;  citation_pages=762-771; " /><meta name="citation_reference" content="citation_title=A tandem repeats database for bacterial genomes: application to the genotyping of Yersinia pestis and Bacillus anthracis; BMC Microbiology;  citation_year=2001;  citation_volume=1;  citation_pages=2" /><meta name="citation_reference" content="citation_title=Microsatellites in different eukaryotic genomes: survey and analysis; Genome Res.;  citation_year=2000;  citation_volume=10;  citation_pages=967-981; " /><meta name="citation_reference" content="citation_title=Simple sequence repeats in Escherichia coli: abundance, distribution, composition, and polymorphism; Genome Res.;  citation_year=2000;  citation_volume=10;  citation_pages=62-71; " /><meta name="citation_reference" content="citation_title=A comprehensive genetic map of the human genome based on 5,264 microsatellites; Nature;  citation_year=1996;  citation_volume=380;  citation_pages=149-152; " /><meta name="citation_reference" content="citation_title=Variable number of tandem repeats in clinical strains of Haemophilus influenzae; Infect. Immun.;  citation_year=1997;  citation_volume=65;  citation_pages=5017-5027; " /><meta name="citation_reference" content="citation_title=Diversity in a variable-number tandem repeat from Yersinia pestis; J. Clin. Microbiol.;  citation_year=2000;  citation_volume=38;  citation_pages=1516-1519; " /><meta name="citation_reference" content="citation_title=Tandem cyclic alignment, Proceedings of the 12th annual symposium on combinatorial pattern matching; LNCS;  citation_year=2001;  citation_volume=2089;  citation_pages=118-130; " /><meta name="citation_reference" content="citation_title=Tandem repeats finder: a program to analyze DNA sequences; Nucleic Acids Res.;  citation_year=1999;  citation_volume=27;  citation_pages=573-580; " /><meta name="citation_reference" content="citation_title=mreps: efficient and flexible detection of tandem repeats in DNA; Nucleic Acids Res.;  citation_year=2003;  citation_volume=31;  citation_pages=3672-3678; " /><meta name="citation_reference" content="citation_title=STRBase: a short tandem repeat DNA database for the human identity testing community; Nucleic Acids Res.;  citation_year=2001;  citation_volume=29;  citation_pages=320-322; " /><meta name="citation_reference" content="citation_title=TRbase: a database relating tandem repeats to disease genes for the human genome; Bioinformatics;  citation_year=2005;  citation_volume=21;  citation_pages=811-816; " /><meta name="citation_reference" content="citation_title=Information decomposition method to analyze symbolical sequences; Phys. Lett. A;  citation_year=2003;  citation_volume=312;  citation_pages=198-210; " /><meta name="citation_reference" content="citation_title=Evolution of tRNA-like sequences and genome variability; Gene;  citation_year=2004;  citation_volume=335;  citation_pages=57-71; " /><meta name="citation_reference" content="citation_title=MIR: family of repeats common for vertebrate genomes; Mol. Biol. (Mosk);  citation_year=2000;  citation_volume=34;  citation_pages=553-559; " /><meta name="citation_reference" content="citation_title=Fourier and wavelet transform analysis, a tool for visualizing regular patterns in DNA sequences; J. Theor. Biol.;  citation_year=2000;  citation_volume=206;  citation_pages=323-326; " /><meta name="citation_reference" content="citation_title=Latent periodicity of protein sequences; J. Mol. Model.;  citation_year=1999;  citation_volume=5;  citation_pages=103-115; " /><meta name="citation_reference" content="citation_title=Latent sequence periodicity of some oncogenes and DNA-binding protein genes; CABIOS;  citation_year=1997;  citation_volume=13;  citation_pages=37-44; " /><meta name="citation_reference" content="citation_title=Method revealing latent periodicity of the nucleotide sequences modified for a case of small samples; DNA Res.;  citation_year=1999;  citation_volume=6;  citation_pages=153-163; " /><meta name="citation_reference" content="citation_title=Information Theory and Statistics; citation_publisher=John Wiley &amp; Sons, New York;  citation_year=1959; " /><meta name="citation_reference" content="citation_title=Hierarchical grouping to optimize an objective function; J. Amer. Stat. Assoc.;  citation_year=1963;  citation_volume=58;  citation_pages=236-244; " /><meta name="citation_reference" content="citation_title=Introduction to Computational Biology; citation_publisher=Map, Sequences and Genomes, Chapman &amp; Hall, London;  citation_year=1995; " /><meta name="citation_reference" content="citation_title=Profile analysis: detection of distantly related proteins; Proc. Natl Acad. Sci. USA;  citation_year=1987;  citation_volume=84;  citation_pages=4355-4358; " /><meta name="citation_reference" content="citation_title=Weighting aligned protein or nucleic acid sequences to correct for unequal representation; J. Mol. Biol.;  citation_year=1990;  citation_volume=216;  citation_pages=813-818; " /><meta name="citation_reference" content="citation_title=Identification of common molecular subsequences; J. Mol. Biol.;  citation_year=1981;  citation_volume=147;  citation_pages=195-197; " /><meta name="citation_reference" content="citation_title=Estimation of P-values for global alignments of protein sequences; Bioinformatics;  citation_year=2001;  citation_volume=17;  citation_pages=1158-1167; " /><meta name="citation_reference" content="citation_title=DNA tertiary structures formed in vitro by misaligned hybridization of multiple tandem repeat sequences; Nucleic Acids Res.;  citation_year=1989;  citation_volume=17;  citation_pages=7417-7426; " /><meta name="citation_reference" content="citation_title=EMBOSS: The European molecular biology open software suite; Trends Genet.;  citation_year=2000;  citation_volume=16;  citation_pages=276-277; " /><meta name="citation_reference" content="citation_title=An algorithm for locating nonoverlapping regions of maximum alignment score; SIAM J. Comput.;  citation_year=1996;  citation_volume=25;  citation_pages=648-662; " /><meta name="citation_reference" content="citation_title=A space efficient algorithm for finding the best nonoverlapping alignment score; Theor. Comput. Sci.;  citation_year=1995;  citation_volume=145;  citation_pages=357-369; " /><meta name="citation_reference" content="citation_title=All highest scoring paths in weighted grid graphs and their application to finding all approximate repeats in strings; SIAM J. Comput.;  citation_year=1998;  citation_volume=27;  citation_pages=972-992; " /><meta name="citation_reference" content="citation_title=Latent periodicity of serine–threonine and tyrosine protein kinases and other protein families; Comput. Biol. Chem.;  citation_year=2005;  citation_volume=29;  citation_pages=229-243; " /><meta name="citation_reference" content="citation_title=Locating probable genes using fourier transform approach; Bioinformatics;  citation_year=2002;  citation_volume=18;  citation_pages=196-197; " /><meta name="citation_reference" content="citation_title=Nucleosome units and hidden periodicities in DNA sequences; J. Biomol. Struct. Dyn.;  citation_year=1998;  citation_volume=15;  citation_pages=937-947; " /><meta name="citation_reference" content="citation_title=Vectors of Shannon information from Fourier signals characterizing base periodicity in genes and genomes; Biochem. Biophys. Res. Commun.;  citation_year=2000;  citation_volume=268;  citation_pages=289-292; " /><meta name="citation_reference" content="citation_title=Discovering simple DNA sequences by the algorithmic significance method; CABIOS;  citation_year=1993;  citation_volume=9;  citation_pages=407-411; " /><meta name="citation_reference" content="citation_title=An algorithm for approximate tandem repeats; J. Comp. Biol.;  citation_year=2001;  citation_volume=8;  citation_pages=1-18; " /><meta name="citation_reference" content="citation_title=Genome-wide analysis of microsatellite repeats in humans: their abundance and density in specific genomic regions; Genome Biol.;  citation_year=2003;  citation_volume=4;  citation_pages=R13" /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/13/3/89/17462509/dsl004.pdf" /><meta name="description" content="Abstract.  We used the method of Information Decomposition developed by us to identify the latent dinucleotide periodicity regions in bacterial genomes. The num" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/13/3/89/347450" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/13/3/89/347450" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/13/3/89/347450");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsl004';
    </script>
<title>
	Search and Classification of Potential Minisatellite Sequences from Bacterial Genomes | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./347450" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BZpaPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTaGVsZW5rb3YsIEFuZHJldyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkJpb2VuZ2luZWVyaW5nIENlbnRyZSBvZiBSdXNzaWFuIEFjYWRlbXkgb2YgU2NpZW5jZXMsIFByb3NwZWN0IDYwLXR5YSBPa3R5YWJyeWE3LzEsIDExNzMxMiBNb3Njb3csIFJ1c3NpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlNrcnlhYmluLCBLb25zdGFudGluIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQmlvZW5naW5lZXJpbmcgQ2VudHJlIG9mIFJ1c3NpYW4gQWNhZGVteSBvZiBTY2llbmNlcywgUHJvc3BlY3QgNjAtdHlhIE9rdHlhYnJ5YTcvMSwgMTE3MzEyIE1vc2NvdywgUnVzc2lhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS29yb3Rrb3YsIEV1Z2VuZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkJpb2VuZ2luZWVyaW5nIENlbnRyZSBvZiBSdXNzaWFuIEFjYWRlbXkgb2YgU2NpZW5jZXMsIFByb3NwZWN0IDYwLXR5YSBPa3R5YWJyeWE3LzEsIDExNzMxMiBNb3Njb3csIFJ1c3NpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iU2VhcmNoIGFuZCBDbGFzc2lmaWNhdGlvbiBvZiBQb3RlbnRpYWwgTWluaXNhdGVsbGl0ZSBTZXF1ZW5jZXMgZnJvbSBCYWN0ZXJpYWwgR2Vub21lcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9Ijg5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSIxMDIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc2wwMDQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMTMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAwNi8wMS8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1vbGVjdWxhciBiYXNpcyBvZiBnZW5ldGljIGluc3RhYmlsaXR5IG9mIHRyaXBsZXQgcmVwZWF0czsgSi4gQmlvbC4gQ2hlbS47ICBjaXRhdGlvbl95ZWFyPTE5OTY7ICBjaXRhdGlvbl92b2x1bWU9MjcxOyAgY2l0YXRpb25fcGFnZXM9Mjg3NS0yODc4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1ETkEgc2Vjb25kYXJ5IHN0cnVjdHVyZXMgYW5kIHRoZSBldm9sdXRpb24gb2YgaHlwZXJ2YXJpYWJsZSB0YW5kZW0gYXJyYXlzOyBKLiBCaW9sLiBDaGVtLjsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT0yNzI7ICBjaXRhdGlvbl9wYWdlcz05NTE3LTk1MjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZyYWdpbGUgWCBzeW5kcm9tZSB1bnN0YWJsZSBlbGVtZW50LCBwKENDRyluLCBhbmQgb3RoZXIgc2ltcGxlIHRhbmRlbSByZXBlYXQgc2VxdWVuY2VzIGFyZSBiaW5kaW5nIHNpdGVzIGZvciBzcGVjaWZpYyBudWNsZWFyIHByb3RlaW5zOyBIdW0uIE1vbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0xOTkzOyAgY2l0YXRpb25fdm9sdW1lPTI7ICBjaXRhdGlvbl9wYWdlcz0xNDI5LTE0MzU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPShDVCluIChHQSluIHJlcGVhdHMgYW5kIGhlYXQgc2hvY2sgZWxlbWVudHMgaGF2ZSBkaXN0aW5jdCByb2xlcyBpbiBjaHJvbWF0aW4gc3RydWN0dXJlIGFuZCB0cmFuc2NyaXB0aW9uYWwgYWN0aXZhdGlvbiBvZiB0aGUgRHJvc29waGlsYSBoc3AyNiBnZW5lOyBNb2wuIENlbGwuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTkzOyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9MjgwMi0yODE0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NdWx0aXBsZS1sb2N1cyB2YXJpYWJsZS1udW1iZXIgdGFuZGVtIHJlcGVhdCBhbmFseXNpcyByZXZlYWxzIGdlbmV0aWMgcmVsYXRpb25zaGlwcyB3aXRoaW4gQmFjaWxsdXMgYW50aHJhY2lzOyBKLiBCYWN0ZXJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTE4MjsgIGNpdGF0aW9uX3BhZ2VzPTI5MjgtMjkzNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpYyBkaXZlcnNpdHkgaW4gdGhlIE15Y29iYWN0ZXJpdW0gdHViZXJjdWxvc2lzIGNvbXBsZXggYmFzZWQgb24gdmFyaWFibGUgbnVtYmVycyBvZiB0YW5kZW0gRE5BIHJlcGVhdHM7IE1pY3JvYmlvbG9neTsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT0xNDQ7ICBjaXRhdGlvbl9wYWdlcz0xMTg5LTExOTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVZhcmlhYmxlIGh1bWFuIG1pbmlzYXRlbGxpdGUtbGlrZSByZWdpb25zIGluIHRoZSBNeWNvYmFjdGVyaXVtIHR1YmVyY3Vsb3NpcyBnZW5vbWU7IE1vbC4gTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0zNjsgIGNpdGF0aW9uX3BhZ2VzPTc2Mi03NzE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgdGFuZGVtIHJlcGVhdHMgZGF0YWJhc2UgZm9yIGJhY3RlcmlhbCBnZW5vbWVzOiBhcHBsaWNhdGlvbiB0byB0aGUgZ2Vub3R5cGluZyBvZiBZZXJzaW5pYSBwZXN0aXMgYW5kIEJhY2lsbHVzIGFudGhyYWNpczsgQk1DIE1pY3JvYmlvbG9neTsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0xOyAgY2l0YXRpb25fcGFnZXM9MiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1pY3Jvc2F0ZWxsaXRlcyBpbiBkaWZmZXJlbnQgZXVrYXJ5b3RpYyBnZW5vbWVzOiBzdXJ2ZXkgYW5kIGFuYWx5c2lzOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTk2Ny05ODE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNpbXBsZSBzZXF1ZW5jZSByZXBlYXRzIGluIEVzY2hlcmljaGlhIGNvbGk6IGFidW5kYW5jZSwgZGlzdHJpYnV0aW9uLCBjb21wb3NpdGlvbiwgYW5kIHBvbHltb3JwaGlzbTsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz02Mi03MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBjb21wcmVoZW5zaXZlIGdlbmV0aWMgbWFwIG9mIHRoZSBodW1hbiBnZW5vbWUgYmFzZWQgb24gNSwyNjQgbWljcm9zYXRlbGxpdGVzOyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTE5OTY7ICBjaXRhdGlvbl92b2x1bWU9MzgwOyAgY2l0YXRpb25fcGFnZXM9MTQ5LTE1MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VmFyaWFibGUgbnVtYmVyIG9mIHRhbmRlbSByZXBlYXRzIGluIGNsaW5pY2FsIHN0cmFpbnMgb2YgSGFlbW9waGlsdXMgaW5mbHVlbnphZTsgSW5mZWN0LiBJbW11bi47ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9NjU7ICBjaXRhdGlvbl9wYWdlcz01MDE3LTUwMjc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpdmVyc2l0eSBpbiBhIHZhcmlhYmxlLW51bWJlciB0YW5kZW0gcmVwZWF0IGZyb20gWWVyc2luaWEgcGVzdGlzOyBKLiBDbGluLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTM4OyAgY2l0YXRpb25fcGFnZXM9MTUxNi0xNTE5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UYW5kZW0gY3ljbGljIGFsaWdubWVudCwgUHJvY2VlZGluZ3Mgb2YgdGhlIDEydGggYW5udWFsIHN5bXBvc2l1bSBvbiBjb21iaW5hdG9yaWFsIHBhdHRlcm4gbWF0Y2hpbmc7IExOQ1M7ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MjA4OTsgIGNpdGF0aW9uX3BhZ2VzPTExOC0xMzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRhbmRlbSByZXBlYXRzIGZpbmRlcjogYSBwcm9ncmFtIHRvIGFuYWx5emUgRE5BIHNlcXVlbmNlczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTI3OyAgY2l0YXRpb25fcGFnZXM9NTczLTU4MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9bXJlcHM6IGVmZmljaWVudCBhbmQgZmxleGlibGUgZGV0ZWN0aW9uIG9mIHRhbmRlbSByZXBlYXRzIGluIEROQTsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTMxOyAgY2l0YXRpb25fcGFnZXM9MzY3Mi0zNjc4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TVFJCYXNlOiBhIHNob3J0IHRhbmRlbSByZXBlYXQgRE5BIGRhdGFiYXNlIGZvciB0aGUgaHVtYW4gaWRlbnRpdHkgdGVzdGluZyBjb21tdW5pdHk7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0yOTsgIGNpdGF0aW9uX3BhZ2VzPTMyMC0zMjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRSYmFzZTogYSBkYXRhYmFzZSByZWxhdGluZyB0YW5kZW0gcmVwZWF0cyB0byBkaXNlYXNlIGdlbmVzIGZvciB0aGUgaHVtYW4gZ2Vub21lOyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTgxMS04MTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUluZm9ybWF0aW9uIGRlY29tcG9zaXRpb24gbWV0aG9kIHRvIGFuYWx5emUgc3ltYm9saWNhbCBzZXF1ZW5jZXM7IFBoeXMuIExldHQuIEE7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MzEyOyAgY2l0YXRpb25fcGFnZXM9MTk4LTIxMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXZvbHV0aW9uIG9mIHRSTkEtbGlrZSBzZXF1ZW5jZXMgYW5kIGdlbm9tZSB2YXJpYWJpbGl0eTsgR2VuZTsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0zMzU7ICBjaXRhdGlvbl9wYWdlcz01Ny03MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TUlSOiBmYW1pbHkgb2YgcmVwZWF0cyBjb21tb24gZm9yIHZlcnRlYnJhdGUgZ2Vub21lczsgTW9sLiBCaW9sLiAoTW9zayk7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MzQ7ICBjaXRhdGlvbl9wYWdlcz01NTMtNTU5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Gb3VyaWVyIGFuZCB3YXZlbGV0IHRyYW5zZm9ybSBhbmFseXNpcywgYSB0b29sIGZvciB2aXN1YWxpemluZyByZWd1bGFyIHBhdHRlcm5zIGluIEROQSBzZXF1ZW5jZXM7IEouIFRoZW9yLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0yMDY7ICBjaXRhdGlvbl9wYWdlcz0zMjMtMzI2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1MYXRlbnQgcGVyaW9kaWNpdHkgb2YgcHJvdGVpbiBzZXF1ZW5jZXM7IEouIE1vbC4gTW9kZWwuOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz0xMDMtMTE1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1MYXRlbnQgc2VxdWVuY2UgcGVyaW9kaWNpdHkgb2Ygc29tZSBvbmNvZ2VuZXMgYW5kIEROQS1iaW5kaW5nIHByb3RlaW4gZ2VuZXM7IENBQklPUzsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTM3LTQ0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NZXRob2QgcmV2ZWFsaW5nIGxhdGVudCBwZXJpb2RpY2l0eSBvZiB0aGUgbnVjbGVvdGlkZSBzZXF1ZW5jZXMgbW9kaWZpZWQgZm9yIGEgY2FzZSBvZiBzbWFsbCBzYW1wbGVzOyBETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9MTUzLTE2MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW5mb3JtYXRpb24gVGhlb3J5IGFuZCBTdGF0aXN0aWNzOyBjaXRhdGlvbl9wdWJsaXNoZXI9Sm9obiBXaWxleSAmYW1wOyBTb25zLCBOZXcgWW9yazsgIGNpdGF0aW9uX3llYXI9MTk1OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGllcmFyY2hpY2FsIGdyb3VwaW5nIHRvIG9wdGltaXplIGFuIG9iamVjdGl2ZSBmdW5jdGlvbjsgSi4gQW1lci4gU3RhdC4gQXNzb2MuOyAgY2l0YXRpb25feWVhcj0xOTYzOyAgY2l0YXRpb25fdm9sdW1lPTU4OyAgY2l0YXRpb25fcGFnZXM9MjM2LTI0NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW50cm9kdWN0aW9uIHRvIENvbXB1dGF0aW9uYWwgQmlvbG9neTsgY2l0YXRpb25fcHVibGlzaGVyPU1hcCwgU2VxdWVuY2VzIGFuZCBHZW5vbWVzLCBDaGFwbWFuICZhbXA7IEhhbGwsIExvbmRvbjsgIGNpdGF0aW9uX3llYXI9MTk5NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJvZmlsZSBhbmFseXNpczogZGV0ZWN0aW9uIG9mIGRpc3RhbnRseSByZWxhdGVkIHByb3RlaW5zOyBQcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0xOTg3OyAgY2l0YXRpb25fdm9sdW1lPTg0OyAgY2l0YXRpb25fcGFnZXM9NDM1NS00MzU4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1XZWlnaHRpbmcgYWxpZ25lZCBwcm90ZWluIG9yIG51Y2xlaWMgYWNpZCBzZXF1ZW5jZXMgdG8gY29ycmVjdCBmb3IgdW5lcXVhbCByZXByZXNlbnRhdGlvbjsgSi4gTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5MDsgIGNpdGF0aW9uX3ZvbHVtZT0yMTY7ICBjaXRhdGlvbl9wYWdlcz04MTMtODE4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JZGVudGlmaWNhdGlvbiBvZiBjb21tb24gbW9sZWN1bGFyIHN1YnNlcXVlbmNlczsgSi4gTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk4MTsgIGNpdGF0aW9uX3ZvbHVtZT0xNDc7ICBjaXRhdGlvbl9wYWdlcz0xOTUtMTk3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Fc3RpbWF0aW9uIG9mIFAtdmFsdWVzIGZvciBnbG9iYWwgYWxpZ25tZW50cyBvZiBwcm90ZWluIHNlcXVlbmNlczsgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MTc7ICBjaXRhdGlvbl9wYWdlcz0xMTU4LTExNjc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUROQSB0ZXJ0aWFyeSBzdHJ1Y3R1cmVzIGZvcm1lZCBpbiB2aXRybyBieSBtaXNhbGlnbmVkIGh5YnJpZGl6YXRpb24gb2YgbXVsdGlwbGUgdGFuZGVtIHJlcGVhdCBzZXF1ZW5jZXM7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk4OTsgIGNpdGF0aW9uX3ZvbHVtZT0xNzsgIGNpdGF0aW9uX3BhZ2VzPTc0MTctNzQyNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RU1CT1NTOiBUaGUgRXVyb3BlYW4gbW9sZWN1bGFyIGJpb2xvZ3kgb3BlbiBzb2Z0d2FyZSBzdWl0ZTsgVHJlbmRzIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTI3Ni0yNzc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuIGFsZ29yaXRobSBmb3IgbG9jYXRpbmcgbm9ub3ZlcmxhcHBpbmcgcmVnaW9ucyBvZiBtYXhpbXVtIGFsaWdubWVudCBzY29yZTsgU0lBTSBKLiBDb21wdXQuOyAgY2l0YXRpb25feWVhcj0xOTk2OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9NjQ4LTY2MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBzcGFjZSBlZmZpY2llbnQgYWxnb3JpdGhtIGZvciBmaW5kaW5nIHRoZSBiZXN0IG5vbm92ZXJsYXBwaW5nIGFsaWdubWVudCBzY29yZTsgVGhlb3IuIENvbXB1dC4gU2NpLjsgIGNpdGF0aW9uX3llYXI9MTk5NTsgIGNpdGF0aW9uX3ZvbHVtZT0xNDU7ICBjaXRhdGlvbl9wYWdlcz0zNTctMzY5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbGwgaGlnaGVzdCBzY29yaW5nIHBhdGhzIGluIHdlaWdodGVkIGdyaWQgZ3JhcGhzIGFuZCB0aGVpciBhcHBsaWNhdGlvbiB0byBmaW5kaW5nIGFsbCBhcHByb3hpbWF0ZSByZXBlYXRzIGluIHN0cmluZ3M7IFNJQU0gSi4gQ29tcHV0LjsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT0yNzsgIGNpdGF0aW9uX3BhZ2VzPTk3Mi05OTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxhdGVudCBwZXJpb2RpY2l0eSBvZiBzZXJpbmXigJN0aHJlb25pbmUgYW5kIHR5cm9zaW5lIHByb3RlaW4ga2luYXNlcyBhbmQgb3RoZXIgcHJvdGVpbiBmYW1pbGllczsgQ29tcHV0LiBCaW9sLiBDaGVtLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0yOTsgIGNpdGF0aW9uX3BhZ2VzPTIyOS0yNDM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxvY2F0aW5nIHByb2JhYmxlIGdlbmVzIHVzaW5nIGZvdXJpZXIgdHJhbnNmb3JtIGFwcHJvYWNoOyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0xODsgIGNpdGF0aW9uX3BhZ2VzPTE5Ni0xOTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU51Y2xlb3NvbWUgdW5pdHMgYW5kIGhpZGRlbiBwZXJpb2RpY2l0aWVzIGluIEROQSBzZXF1ZW5jZXM7IEouIEJpb21vbC4gU3RydWN0LiBEeW4uOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTE1OyAgY2l0YXRpb25fcGFnZXM9OTM3LTk0NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VmVjdG9ycyBvZiBTaGFubm9uIGluZm9ybWF0aW9uIGZyb20gRm91cmllciBzaWduYWxzIGNoYXJhY3Rlcml6aW5nIGJhc2UgcGVyaW9kaWNpdHkgaW4gZ2VuZXMgYW5kIGdlbm9tZXM7IEJpb2NoZW0uIEJpb3BoeXMuIFJlcy4gQ29tbXVuLjsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0yNjg7ICBjaXRhdGlvbl9wYWdlcz0yODktMjkyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaXNjb3ZlcmluZyBzaW1wbGUgRE5BIHNlcXVlbmNlcyBieSB0aGUgYWxnb3JpdGhtaWMgc2lnbmlmaWNhbmNlIG1ldGhvZDsgQ0FCSU9TOyAgY2l0YXRpb25feWVhcj0xOTkzOyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz00MDctNDExOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBhbGdvcml0aG0gZm9yIGFwcHJveGltYXRlIHRhbmRlbSByZXBlYXRzOyBKLiBDb21wLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9MS0xODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lLXdpZGUgYW5hbHlzaXMgb2YgbWljcm9zYXRlbGxpdGUgcmVwZWF0cyBpbiBodW1hbnM6IHRoZWlyIGFidW5kYW5jZSBhbmQgZGVuc2l0eSBpbiBzcGVjaWZpYyBnZW5vbWljIHJlZ2lvbnM7IEdlbm9tZSBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9UjEzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzEzLzMvODkvMTc0NjI1MDkvZHNsMDA0LnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBXZSB1c2VkIHRoZSBtZXRob2Qgb2YgSW5mb3JtYXRpb24gRGVjb21wb3NpdGlvbiBkZXZlbG9wZWQgYnkgdXMgdG8gaWRlbnRpZnkgdGhlIGxhdGVudCBkaW51Y2xlb3RpZGUgcGVyaW9kaWNpdHkgcmVnaW9ucyBpbiBiYWN0ZXJpYWwgZ2Vub21lcy4gVGhlIG51bSIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXIvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9MzQ3NDUwJnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZDWX9Iu135tylx+/T4jyiPuqNa32" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f13%2f3%2f89%2f347450" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f13%2f3%2f89%2f347450" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=13406" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/13/3"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/13/3/0/m_dnaresearch13_3.cover.gif?Expires=2147483647&amp;Signature=anx2J~8v0IVYzlePG5tUbncUtEx2PsYUKWb~Qpjo1jhMHlu0islS2eGWU9i~f0nzFtaqNwAgI5-no5gavB~5K2oqu5oELPuFU9PB0RbXx30rT5CtFH7Z7Tak8QE4~Qv5CaQ-S-laJYNwnkzL6WJ228M7M2qJZAZbrbUpUHT7RG37R-U5f~bJ7ij8Hr1oUW8kL1EexLVBqSK~E8j95q-vzg7TEqouIZZVhlOy-gxfExmOp8ieedz1ReyVM4Msqvq9iUEHn2KT3PqF7FeNdH-Lqt5xtcPFEJYTrL4VFii2sM~Knj8tJAAT5R0NldIHOqOze-~nk5Mk6oyjGclZXc9hCA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/13/3">
                        <div class="volume trailing-comma">Volume 13</div>
                        <div class="issue">Issue 3</div>

            </a>

        </div>
        <div class="ii-pub-date">
2006        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="87194392">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87194392">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87194394">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87194394">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87194404">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87194404">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87194459">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87194459">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87194488">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87194488">4. Conclusion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87194493">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87194493">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="empty arrow">&nbsp;</li>
            <li class="next arrow">
            <a href="/dnaresearch/article/13/3/103/347233">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Search and Classification of Potential Minisatellite Sequences from Bacterial Genomes
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Andrew Shelenkov</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">Bioengineering Centre of Russian Academy of Sciences, Prospect 60-tya Oktyabrya</div><div class="addr-line">7/1, 117312 Moscow, Russia</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <sup>*</sup>To whom correspondence should be addressed. Tel. +7-495-135-2161. Fax +7-495-135-2161. E-mail: <a href="mailto:fallandar@mail.ru" target="_blank">fallandar@mail.ru</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Andrew+Shelenkov">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shelenkov A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Shelenkov A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Andrew Shelenkov</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Konstantin Skryabin</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">Bioengineering Centre of Russian Academy of Sciences, Prospect 60-tya Oktyabrya</div><div class="addr-line">7/1, 117312 Moscow, Russia</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Konstantin+Skryabin">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Skryabin K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Skryabin K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Konstantin Skryabin</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Eugene Korotkov</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">Bioengineering Centre of Russian Academy of Sciences, Prospect 60-tya Oktyabrya</div><div class="addr-line">7/1, 117312 Moscow, Russia</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Eugene+Korotkov">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Korotkov E">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Korotkov E&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Eugene Korotkov</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Communicated by Kenta Nakai</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 13, Issue 3, 1 January 2006, Pages 89–102, <a href='https://doi.org/10.1093/dnares/dsl004'>https://doi.org/10.1093/dnares/dsl004</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">01 January 2006</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">12 April 2006</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Revision Received:</div>
                                <div class="wi-date">01 June 2006</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="347450" href="/dnaresearch/article-pdf/13/3/89/17462509/dsl004.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Andrew Shelenkov, Konstantin Skryabin, Eugene Korotkov;  Search and Classification of Potential Minisatellite Sequences from Bacterial Genomes, <em>DNA Research</em>, Volume 13, Issue 3, 1 January 2006, Pages 89–102, <a href='https://doi.org/10.1093/dnares/dsl004'>https://doi.org/10.1093/dnares/dsl004</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/347450?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/347450?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/347450?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/347450?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/347450?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/347450?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/347450?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Search%20and%20Classification%20of%20Potential%20Minisatellite%20Sequences%20from%20Bacterial%20Genomes&amp;publicationDate=2006-01-01&amp;volumeNum=13&amp;issueNum=3&amp;author=Shelenkov%2C%20Andrew%3B%20Skryabin%2C%20Konstantin&amp;startPage=89&amp;endPage=102&amp;contentId=10.1093%2Fdnares%2Fdsl004&amp;oa=&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f13%2f3%2f89%2f347450" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=87194392 id="87194392" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>We used the method of Information Decomposition developed by us to identify the latent dinucleotide periodicity regions in bacterial genomes. The number of potential minisatellite sequences obtained at high level of statistical significance was 454. Then we classified the periodicity matrices and obtained 45 classes. We used the other new method developed by us—Modified Profile Analysis—to reveal more periodic sequences in the presence of indels using the classes obtained. The number of sequences found by combination of these two methods was 3949. Most of them cannot be revealed by other methods including dynamic programming and Fourier transformation.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;information decomposition&quot;">information decomposition</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;modified profile analysis&quot;">modified profile analysis</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="minisatellites">minisatellites</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;bacterial genomes&quot;">bacterial genomes</a></div>                            <h2 scrollto-destination=87194394 id="87194394" class="section-title" >1. Introduction</h2>
<p>The presence of repeated sequences is a common feature for both eukaryotic and prokaryotic genomes. It has been suggested that the repeats themselves produce unusual physical structures in the DNA, causing polymerase slippage and the resulting amplification.<a reveal-id="b1" data-open="b1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="b2" data-open="b2" class="link link-ref link-reveal xref-bibr">2</a> The other potential role for tandem repeats is gene regulation, in which the repeats may interact with transcription factors, alter the structure of the chromatin or act as protein binding sites.<a reveal-id="b3" data-open="b3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="b4" data-open="b4" class="link link-ref link-reveal xref-bibr">4</a> In the last few years, tandem repeats have been increasingly recognized as markers of choice for genotyping a number of pathogens.<a reveal-id="b5  b7" data-open="b5  b7" class="link link-ref link-reveal xref-bibr">5–7</a> The rapid evolution of these structures appears to contribute to the phenotypic flexibility of pathogens.</p><p>Tandem repeats are usually classified among satellites (spanning megabases of DNA, associated with heterochromatin), minisatellites (repeat units in the range 6–10 bp, spanning hundreds of base pairs) and microsatellites (repeat units in the range 2–5 bp, spanning a few tens of nucleotides).<a reveal-id="b8" data-open="b8" class="link link-ref link-reveal xref-bibr">8</a> Microsatellites or simple sequence repeats (SSRs) are tandemly repeated DNA sequences found in varying abundance in most genomes.<a reveal-id="b9" data-open="b9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="b10" data-open="b10" class="link link-ref link-reveal xref-bibr">10</a> These repeats have been extensively used for genetic mapping and population studies.<a reveal-id="b11" data-open="b11" class="link link-ref link-reveal xref-bibr">11</a> Microsatellites are frequently polymorphic with the number of repeat units varying between organisms. The polymorphism associated with tandem repeats has been used in mammalian genetics for the construction of genetic maps and still is the basis of DNA fingerprinting in forensic applications. Polymorphic minisatelites are also found in bacterial genomes.<a reveal-id="b8" data-open="b8" class="link link-ref link-reveal xref-bibr">8</a></p><p>The availability of whole-genome sequences has opened the way to the systematic evaluation of tandem repeats diversity and application to epidemiological studies.<a reveal-id="b8" data-open="b8" class="link link-ref link-reveal xref-bibr">8</a> More recently, a number of studies<a reveal-id="b12" data-open="b12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="b13" data-open="b13" class="link link-ref link-reveal xref-bibr">13</a> have confirmed the idea that tandem repeats reminiscent of minisatellites and microsatellites are likely to be a significant source of very informative markers for the identification of pathogenic bacteria.</p><p>Once repeats are identified, the central task becomes the clustering of tandem repeats into families, i.e. repeats that occur in different locations in a genome but have identical or very similar underlying patterns. Grouping these repeats will facilitate identification and study of their common properties. Tandem repeat families have been detected in both prokaryotes and eukaryotes, including the <em>Escherichia coli</em>, <em>Saccharomyces cerevisiae</em>, <em>Caenorhabditis elegans</em> and human genomes.<a reveal-id="b14" data-open="b14" class="link link-ref link-reveal xref-bibr">14</a> Analyzing the mutational history of tandem repeats requires utilizing the pattern of mutations among adjacent copies to describe the interwoven progression of substitutions, indels and duplication/excision events leading from a single copy of the pattern to the present day sequence. Such histories can suggest how the boundaries and size of the duplication unit vary and may reveal details about the duplication mechanism.<a reveal-id="b15" data-open="b15" class="link link-ref link-reveal xref-bibr">15</a></p><p>Among the best programs for finding the tandem repeats are Tandem Repeats Finder<a reveal-id="b15" data-open="b15" class="link link-ref link-reveal xref-bibr">15</a> and mreps.<a reveal-id="b16" data-open="b16" class="link link-ref link-reveal xref-bibr">16</a> Although they overcome some limitations of other algorithms, they have their own, limitations, namely the size of the pattern (Tandem Repeat Finder) and too rigid pattern definition (mreps). Since the results of these methods depend strongly on the homology of the DNA regions, they cannot be used to identify the fuzzy periodicity and ancient minisatellites.</p><p>A number of public tandem repeat databases are available now. To name a few, a database of bacterial genomes,<a reveal-id="b8" data-open="b8" class="link link-ref link-reveal xref-bibr">8</a> a short tandem repeat DNA database,<a reveal-id="b17" data-open="b17" class="link link-ref link-reveal xref-bibr">17</a> TRDB<a reveal-id="b15" data-open="b15" class="link link-ref link-reveal xref-bibr">15</a> and TRbase.<a reveal-id="b18" data-open="b18" class="link link-ref link-reveal xref-bibr">18</a> The methods and programs described in the previous paragraphs are used to fill such databases in most cases. In Section 3 we will discuss in detail the advantages and drawbacks of the existing methods.</p><p>This paper has the following goals. The first goal is to identify the potential minisatellites with the method of Information Decomposition (ID),<a reveal-id="b19" data-open="b19" class="link link-ref link-reveal xref-bibr">19</a> to calculate the periodicity matrices and to classify them. The second goal is to find the new periodic regions (in the presence of insertions and deletions) with the Modified Profile Analysis (MPA)<a reveal-id="b20" data-open="b20" class="link link-ref link-reveal xref-bibr">20</a> approach using the periodicity class matrices found on the previous step.</p><p>We have chosen bacterial genomes as a target of our study for the following reasons. First, bacterial minisatellites can be used as markers for PCR, so there is a practical application for this work. Second, the mutational history of bacterial genomes is evidently more prolonged than that of eukaryotic genomes, so search for the fuzzy repeats is more complicated, but at the same time can give more insights concerning the biological role of ‘ancient’ minisatellites. Third, the comparatively small number of available sequences could result in less number of classes, thus making it easier to explore their properties and to reveal the biological principles standing behind the results of classification process.</p><p>As we will show in Section 3, our method can reveal the periodical sequences that were not found by the existing software packages.</p>                            <h2 scrollto-destination=87194404 id="87194404" class="section-title" >2. Materials and methods</h2>
<p>Our search for the potential minisatellites was focused on the bacterial genomes, since their size allows finishing the computations in a reasonable amount of time; since, the periodical sequences found yield the evident practical application, they can be used in PCR analysis. In bacterial genomes it is possible to find the very old or ‘ancient’ microsatellites possessing very fuzzy periodicity, so they can be passed through by the mathematical methods of tandem repeats finding. Yet these ancient sequences are of great biological interest since they are usually highly polymorphous and, thus, can be used as genetic markers.</p><p>The method proposed in this paper consists of three parts—usage of ID method<a reveal-id="b19" data-open="b19" class="link link-ref link-reveal xref-bibr">19</a> to obtain the initial data, classification of the results found and the usage of MPA for searching the sequences with indels.<a reveal-id="b21" data-open="b21" class="link link-ref link-reveal xref-bibr">21</a> The usage of periodicity classes rather than all periodic sequences found dramatically decreases the searching time for the MPA method. The advantages of using the combination of these methods are: no limitations are placed on the size of the sequence containing the repeated pattern, so the search becomes more versatile; usage of the classification matrices allows us to find even distantly related sequences; although we use the alignment matrices, we do not fill them entirely, which speeds up the calculations even more. And the main advantage is that by using this combination of methods we can find the distantly related repeats and ancient minisatellites possessing very fuzzy periodicity that cannot be revealed by other methods.</p>                            <h3 scrollto-destination=87194407 id="87194407" class="section-title" >2.1. Information decomposition</h3>
<p>The method of ID is described in detail in ref. (<a reveal-id="b19" data-open="b19" class="link link-ref link-reveal xref-bibr">19</a>). Since this method plays an important role in the present study, we also give its thorough description.</p><p>At first, let us define the concept of latent periodicity of a symbolical sequence.</p><div class="block-child-p">Suppose that we have a sequence <em>S</em>, which consists of <em>N</em> subsequences <em>S<sup>i</sup></em> (<em>i</em> = 1,2,…,<em>N</em>) of equal length <em>L</em>: <a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>S</mi><mo>≡</mo><msup><mi>S</mi><mn>1</mn></msup><msup><mi>S</mi><mn>2</mn></msup><mn>…</mn><msup><mi>S</mi><mi>N</mi></msup><mo>≡</mo><mrow><mo>{</mo><mrow><msubsup><mi>s</mi><mn>1</mn><mn>1</mn></msubsup><msubsup><mi>s</mi><mn>2</mn><mn>1</mn></msubsup><mn>…</mn><msubsup><mi>s</mi><mi>L</mi><mn>1</mn></msubsup><msubsup><mi>s</mi><mn>1</mn><mn>2</mn></msubsup><msubsup><mi>s</mi><mn>2</mn><mn>2</mn></msubsup><mn>…</mn><msubsup><mi>s</mi><mi>L</mi><mn>2</mn></msubsup><mn>…</mn><msubsup><mi>s</mi><mn>1</mn><mi>N</mi></msubsup><msubsup><mi>s</mi><mn>2</mn><mi>N</mi></msubsup><mn>…</mn><msubsup><mi>s</mi><mi>L</mi><mi>N</mi></msubsup></mrow><mo>}</mo></mrow></mrow></math></div></div> where <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msubsup><mi>s</mi><mi>i</mi><mi>j</mi></msubsup></mrow></math></span> are DNA bases. Suppose we are looking for a period of the length <em>L</em> in the sequence <em>S</em> ignoring possibilities of insertions and deletions of symbols. To find such a period we should evaluate global homology between subsequences <em>S<sup>i</sup></em> (<em>i</em> = 1,2,…,<em>N</em>). If this homology between subsequences <em>S<sup>i</sup></em> is statistically significant, we can conclude that symbol periodicity with period <em>L</em> exists in sequence <em>S</em>. Possibilities for finding periodicity in the sequence <em>S</em> will depend on the mode of insertion of the quantitative measure determining similarity of subsequences <em>S<sup>i</sup></em>. At the present time, the method of dynamic programming and Fourier transformation are often used for these purposes. To introduce the quantitative measure for global homology of subsequences <em>S<sup>i</sup></em> these methods use a search for homology between these subsequences. While using the method of dynamic programming, the search for homology is set by BLAST or Identity matrices; in these matrices, weight of coinciding bases is always higher than the weight of non-coinciding ones, and, when using the Fourier transformation, the search for homology is set by laws of autocorrelation function construction.<a reveal-id="b22" data-open="b22" class="link link-ref link-reveal xref-bibr">22</a> Earlier we have shown<a reveal-id="b19" data-open="b19" class="link link-ref link-reveal xref-bibr">19</a> that the homology search that uses quantitative measures could miss a hidden periodicity of length <em>L</em> in sequence <em>S</em> because of the lack of statistically significant homology between subsequences <em>S<sup>i</sup></em>; also we have shown that periodicity can be recognized only if a sufficient number of <em>S<sup>i</sup></em> periods is available.</div><div class="block-child-p">Let us consider a notion of latent periodicity in more detail. For introduction of the quantitative measure of subsequences <em>S<sup>i</sup></em> homology, we need to construct a multiple alignment without inserts and deletions of symbols and arrange these subsequences in sequential order. The total weight of this multiple alignment, which can be considered as the quantitative measure of similarity of subsequences <em>S<sup>i</sup></em>, can be introduced as the sum of weights for all positions: <a id="e1" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="e1"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>W</mi><mo>=</mo><mstyle displaystyle="true"><munderover><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mi>L</mi></munderover></mstyle><mrow><msub><mi>W</mi><mi>i</mi></msub></mrow></mrow></math></div><div class="label">(1)</div></div> The traditional approach calculates position weight as the sum of weights of all possible pairs of DNA bases that could appear while comparing the sequences: <a id="e2" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="e2"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>W</mi><mi>i</mi></msub><mo>=</mo><mstyle displaystyle="true"><munder><mo>∑</mo><mi>α</mi></munder></mstyle><mrow><mstyle displaystyle="true"><munder><mo>∑</mo><mrow><mi>β</mi><mo>&gt;</mo><mi>α</mi></mrow></munder></mstyle><mrow><mi>P</mi><mrow><mo>(</mo><mrow><msubsup><mi>s</mi><mi>i</mi><mi>α</mi></msubsup><mo>,</mo><msubsup><mi>s</mi><mi>i</mi><mi>β</mi></msubsup></mrow><mo>)</mo></mrow></mrow></mrow></mrow></math></div><div class="label">(2)</div></div> where α and β show numbers of subsequences <em>S<sup>i</sup></em>; <em>P</em> is some weight matrix such as BLAST or Identity matrices. This expression can also be introduced as the following sum: <a id="e3" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="e3"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>W</mi><mi>i</mi></msub><mo>=</mo><mfrac><mn>1</mn><mn>2</mn></mfrac><mstyle displaystyle="true"><munder><mo>∑</mo><mrow><mi>l</mi><mo>,</mo><mi>k</mi></mrow></munder></mstyle><mrow><mi>m</mi><mo>(</mo><mi>i</mi><mo>,</mo><mi>l</mi><mo>)</mo><mrow><mo>(</mo><mrow><mi>m</mi><mo>(</mo><mi>i</mi><mo>,</mo><mi>k</mi><mo>)</mo><mo>−</mo><msubsup><mi>δ</mi><mi>l</mi><mi>k</mi></msubsup></mrow><mo>)</mo></mrow><mi>P</mi><mo>(</mo><mi>l</mi><mo>,</mo><mi>k</mi><mo>)</mo></mrow></mrow></math></div><div class="label">(3)</div></div> where <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msubsup><mi>δ</mi><mi>l</mi><mi>k</mi></msubsup></mrow></math></span> is 1 at <em>l</em> = <em>k</em> and 0 in all other cases. The function <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msubsup><mi>δ</mi><mi>l</mi><mi>k</mi></msubsup></mrow></math></span> is introduced to exclude from consideration the similarity of the subsequence <em>S<sup>i</sup></em> to itself. Variable values <em>l</em> and <em>k</em> show a type of DNA bases; <em>m</em>(<em>i</em>, <em>l</em>) represents amount of base type <em>l</em> in the position <em>i</em> of multiple alignment. Earlier we proposed another measure of similarity,<a reveal-id="b23  b25" data-open="b23  b25" class="link link-ref link-reveal xref-bibr">23–25</a> which may be defined as the ‘information content’<a reveal-id="b26" data-open="b26" class="link link-ref link-reveal xref-bibr">26</a>: <a id="e4" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="e4"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msubsup><mi>W</mi><mi>i</mi><mo>′</mo></msubsup><mo>=</mo><mstyle displaystyle="true"><munderover><mo>∑</mo><mrow><mi>l</mi><mo>=</mo><mn>1</mn></mrow><mn>4</mn></munderover></mstyle><mrow><mi>m</mi><mo>(</mo><mi>i</mi><mo>,</mo><mi>l</mi><mo>)</mo><mtext>ln</mtext><mfrac><mrow><mi>K</mi><mi>m</mi><mo>(</mo><mi>i</mi><mo>,</mo><mi>l</mi><mo>)</mo></mrow><mrow><mi>x</mi><mo>(</mo><mi>i</mi><mo>)</mo><mi>y</mi><mo>(</mo><mi>l</mi><mo>)</mo></mrow></mfrac></mrow></mrow></math></div><div class="label">(4)</div></div> where <em>K</em> = <em>NL</em>, <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>x</mi><mi>i</mi></msub><mo>=</mo><mstyle displaystyle="false"><munderover><mo>∑</mo><mrow><mi>l</mi><mo>=</mo><mn>1</mn></mrow><mn>4</mn></munderover></mstyle><mrow><mi>m</mi><mo>(</mo><mi>i</mi><mo>,</mo><mi>l</mi><mo>)</mo></mrow></mrow></math></span> and <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>y</mi><mi>l</mi></msub><mo>=</mo><mstyle displaystyle="false"><munderover><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mi>L</mi></munderover></mstyle><mrow><mi>m</mi><mo>(</mo><mi>i</mi><mo>,</mo><mi>l</mi><mo>)</mo></mrow></mrow></math>⁠</span>.</div><div class="block-child-p">It is clear that the measures of homology of subsequences <em>S<sup>i</sup></em> determined by formulas (3) and (4) are different and, thus, an alignment could have higher weight using formulas (1) and (4) and lower weight using formulas (1) and (3), and <em>vice versa</em>. However, the term ‘high weight’ is less informative especially during comparison of weights determined by means of different mathematical measures. For each of the introduced measures we should determine the probability <em>P</em> that the weight (higher or equal to <em>W</em>) would be found during alignment of purely random sequences. In the case of periodicity search for length <em>L</em> in <em>R</em> independent sequences (e.g. analysis of <em>R</em> sequences from the Genbank database) the probability <em>f</em> should be considered instead of probability <em>P</em>: <a id="e5" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="e5"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>f</mi><mo>=</mo><mn>1</mn><mo>−</mo><msup><mrow><mo>(</mo><mn>1</mn><mo>−</mo><mi>p</mi><mo>)</mo></mrow><mi>R</mi></msup></mrow></math></div><div class="label">(5)</div></div> For evaluation of probability <em>P</em> during a search for the period of length <em>L</em> in the sequence <em>S</em> using the measures determined by formulas (3) and (4), we can randomly mix initial sequence <em>S</em> and create many random sequences <em>Q<sub>i</sub></em> (<em>i</em> = 1,2,…) with length equal to the length of sequence <em>S</em>. Using many such sequences <em>Q<sub>i</sub></em>, we can determine the value <em>Z</em> as: <a id="e6" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="e6"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>Z</mi><mo>=</mo><mfrac><mrow><mi>W</mi><mo>−</mo><mi>E</mi><mo>(</mo><mi>W</mi><mo>)</mo></mrow><mrow><msqrt><mrow><mi>D</mi><mo>(</mo><mi>W</mi><mo>)</mo></mrow></msqrt></mrow></mfrac></mrow></math></div><div class="label">(6)</div></div> where <em>E</em>(<em>W</em>) and <em>D</em>(<em>W</em>) are the mean and variance of the weight <em>W</em>, respectively; they are calculated for many random sequences <em>Q<sub>i</sub></em>. High values of <em>Z</em> correspond to all lower values of probability <em>P</em>; they suggest the existence of significant similarity between subsequences <em>S</em><sup>1</sup>, <em>S</em><sup>2</sup>,…, <em>S<sup>N</sup></em>. For evaluation of only non-accidental similarity, some threshold value of <em>Z</em> corresponding to the probability <em>P</em> &lt; 0.05 is usually defined. Homologies with <em>Z-</em>value exceeding threshold level are considered as non-random.</div><p>As it was mentioned above, different weights and different <em>Z</em>-values could result in differences in homology level. In some cases, periodicity could be evident using the information measure [formula (4)] and at the same time could be missed by the methods of homology search [formula (3)]. Let us define the probability <em>P</em> that corresponds to <em>Z-</em>value calculated by formulas (3), (1), and (6) as α and define the probability <em>P</em> that corresponds to <em>Z-</em>value calculated by formulas (4), (1), and (6) as β. Let us also assume that the sequence <em>S</em> contains hidden periodicity of length <em>L</em> provided that the probability α &gt; 0.05 and the probability β &lt; 0.05. Suppose that the sequence <em>S</em> contains periodicity related to homology between subsequences <em>S<sup>i</sup></em> at α &lt; 0.05 irrespectively of the probability values β. Let us also suppose that the sequence <em>S</em> lacks periodicity of the length <em>L</em> at α and β &gt; 0.05.</p><p>In practice such a difference in probabilities α and β can be found quite often while analyzing rather long sequence when some set of symbols, not only one, appears at each position of the alignment. In this case, the number of homologous coincidences can be relatively small for each position. Since the weights of homologous coincidences in BLAST or Identity matrices are significantly higher than the weights of non-homologous coincidences, the final value of <em>W</em> can be relatively small for small number of homologous coincidences; this will provide rather high value of α. At the same time, β is determined on the basis of deviations of DNA base frequencies for each position <em>i</em> from the DNA base frequencies determined for the whole sequence <em>S</em> being analyzed. Such deviations can be significant and will result in a small and statistically significant value of β for a high and statistically insignificant value of α.</p><p>As an example, let us consider two DNA sequences; one of them possesses perfect periodicity whereas the other one has latent periodicity. We determine α and β probabilities using the method of Monte Carlo. For each DNA sequence, we generate 500 random sequences with the same base composition as in the initial ones by means of random shuffling of bases throughout the whole DNA sequence. Using formulas (3) and (1), we calculate the weight <em>W</em> for each of the 500 randomly generated sequences, then determine <em>E</em>(<em>W</em>) and <em>D</em>(<em>W</em>) and finally calculate <em>Z-</em>value by formula (6). We perform the same calculations for the weight determined by formulas (4) and (1). As the result, we will have two <em>Z-</em>values for each DNA sequence: one value has been calculated by formulas (3) and (1) and the other one by formulas (4) and (1).</p><p>Let us take a DNA sequence containing 20 (<em>N</em> = 20) tandem repeats {atcgagt} as the first sequence. In subsequent consideration, this sequence of 140 bases in length will play a role of the sequence <em>S</em> while the length of subsequence <em>S<sup>i</sup></em> is equal to seven DNA bases; this means that we are looking for the period of seven DNA bases (<a reveal-id="tbl1" data-open="tbl1" class="link link-reveal link-table xref-fig">Table 1</a>). For the sequence <em>S</em>, we generate multiple alignment of periods; here they play the role of subsequence <em>S<sup>i</sup></em>. These periods are positioned one under the other. For this multiple alignment of periods, <em>Z-</em>value calculated by formulas (3), (1) and (6) using BLAST matrix is equal to 68.5 ± 3.5, whereas <em>Z-</em>value calculated by formulas (4), (1) and (6) is 55.2 ± 1.9. These calculations show that in the case of perfect periodicity both methods of weight function calculation give very similar results that are indistinguishable within the error of calculation by the Monte Carlo method.</p>                        <a id="87194417" scrollto-destination="87194417"></a>
<div content-id="tbl1" class="table-modal"><div class="table-wrap"><div id="tbl1" class="table-wrap-title" data-id="tbl1"><div class="label -label"><strong>Table 1</strong></div><div class="caption"><p>Matrix used for generation of artificial sequence with hidden periodicity of seven DNA bases in length</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Position of period </th><th>A set of DNA bases that could appear in a given position of the period </th></tr></thead><tbody><tr><td>1 </td><td>atc </td></tr><tr><td>2 </td><td>agct </td></tr><tr><td>3 </td><td>ctg </td></tr><tr><td>4 </td><td>agct </td></tr><tr><td>5 </td><td>cta </td></tr><tr><td>6 </td><td>ct </td></tr><tr><td>7 </td><td>gat </td></tr></tbody></table></div><div content-id="tbl1" class="table-modal"><table><thead name="thead" align=""><tr><th>Position of period </th><th>A set of DNA bases that could appear in a given position of the period </th></tr></thead><tbody><tr><td>1 </td><td>atc </td></tr><tr><td>2 </td><td>agct </td></tr><tr><td>3 </td><td>ctg </td></tr><tr><td>4 </td><td>agct </td></tr><tr><td>5 </td><td>cta </td></tr><tr><td>6 </td><td>ct </td></tr><tr><td>7 </td><td>gat </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>In each position of the period probabilities of selection of any base are equal to each other.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87194417" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="tbl1" class="table-wrap-title" data-id="tbl1"><div class="label -label"><strong>Table 1</strong></div><div class="caption"><p>Matrix used for generation of artificial sequence with hidden periodicity of seven DNA bases in length</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Position of period </th><th>A set of DNA bases that could appear in a given position of the period </th></tr></thead><tbody><tr><td>1 </td><td>atc </td></tr><tr><td>2 </td><td>agct </td></tr><tr><td>3 </td><td>ctg </td></tr><tr><td>4 </td><td>agct </td></tr><tr><td>5 </td><td>cta </td></tr><tr><td>6 </td><td>ct </td></tr><tr><td>7 </td><td>gat </td></tr></tbody></table></div><div content-id="tbl1" class="table-modal"><table><thead name="thead" align=""><tr><th>Position of period </th><th>A set of DNA bases that could appear in a given position of the period </th></tr></thead><tbody><tr><td>1 </td><td>atc </td></tr><tr><td>2 </td><td>agct </td></tr><tr><td>3 </td><td>ctg </td></tr><tr><td>4 </td><td>agct </td></tr><tr><td>5 </td><td>cta </td></tr><tr><td>6 </td><td>ct </td></tr><tr><td>7 </td><td>gat </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>In each position of the period probabilities of selection of any base are equal to each other.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87194417" target="_blank">View Large</a></div></div><p>As a second example, we consider a DNA sequence given below, which is characterized by the existence of latent period of seven symbols in length (<em>L</em> = 7, <em>N</em> = 20). Let us assume that DNA bases listed in <a reveal-id="tbl2" data-open="tbl2" class="link link-reveal link-table xref-fig">Table 2</a> could appear in each position of the period with equal probability. One of possible sequences in which such periodicity is actually observed is the following:</p>                        <a id="87194419" scrollto-destination="87194419"></a>
<div content-id="tbl2" class="table-modal"><div class="table-wrap"><div id="tbl2" class="table-wrap-title" data-id="tbl2"><div class="label -label"><strong>Table 2</strong></div><div class="caption"><p>The distribution of periodic sequences found by the groups of organisms to which they belong</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Category </th><th>Subcategory </th><th>Number of sequences </th></tr></thead><tbody><tr><td>Archaea </td><td>Crenarchaeota </td><td>9 </td></tr><tr><td> </td><td>Euryarchaeota </td><td>20 </td></tr><tr><td>Bacteria </td><td>Actinobacteria </td><td>61 </td></tr><tr><td> </td><td>Bacteroid </td><td>6 </td></tr><tr><td> </td><td>Chlamydiae </td><td>4 </td></tr><tr><td> </td><td>Chlorobi </td><td>2 </td></tr><tr><td> </td><td>Cyanobacteria </td><td>7 </td></tr><tr><td> </td><td>Deinococcus-Thermus </td><td>1 </td></tr><tr><td> </td><td>Thermotogae </td><td>1 </td></tr><tr><td> </td><td colspan="2">Firmicutes </td></tr><tr><td> </td><td>    Firmicutes Bacillales </td><td>33 </td></tr><tr><td> </td><td>    Firmicutes Clostridia </td><td>8 </td></tr><tr><td> </td><td>    Firmicutes Lactobacillales </td><td>22 </td></tr><tr><td> </td><td>    Firmicutes Mollicutes </td><td>4 </td></tr><tr><td> </td><td>Planctomyces </td><td>8 </td></tr><tr><td> </td><td>Proteobacteria </td><td> </td></tr><tr><td> </td><td>    Alphaproteobacteria </td><td>43 </td></tr><tr><td> </td><td>    Betaproteobacteria </td><td>26 </td></tr><tr><td> </td><td>    Deltaproteobacteria </td><td>2 </td></tr><tr><td> </td><td>    Epsilonproteobacteria </td><td>21 </td></tr><tr><td> </td><td>    Gammaproteobacteria </td><td>158 </td></tr><tr><td> </td><td>Spirochaetes </td><td>14 </td></tr><tr><td> </td><td>Uncultured bacterium </td><td>3 </td></tr><tr><td>Plasmids </td><td>Plasmids </td><td>1 </td></tr></tbody></table></div><div content-id="tbl2" class="table-modal"><table><thead name="thead" align=""><tr><th>Category </th><th>Subcategory </th><th>Number of sequences </th></tr></thead><tbody><tr><td>Archaea </td><td>Crenarchaeota </td><td>9 </td></tr><tr><td> </td><td>Euryarchaeota </td><td>20 </td></tr><tr><td>Bacteria </td><td>Actinobacteria </td><td>61 </td></tr><tr><td> </td><td>Bacteroid </td><td>6 </td></tr><tr><td> </td><td>Chlamydiae </td><td>4 </td></tr><tr><td> </td><td>Chlorobi </td><td>2 </td></tr><tr><td> </td><td>Cyanobacteria </td><td>7 </td></tr><tr><td> </td><td>Deinococcus-Thermus </td><td>1 </td></tr><tr><td> </td><td>Thermotogae </td><td>1 </td></tr><tr><td> </td><td colspan="2">Firmicutes </td></tr><tr><td> </td><td>    Firmicutes Bacillales </td><td>33 </td></tr><tr><td> </td><td>    Firmicutes Clostridia </td><td>8 </td></tr><tr><td> </td><td>    Firmicutes Lactobacillales </td><td>22 </td></tr><tr><td> </td><td>    Firmicutes Mollicutes </td><td>4 </td></tr><tr><td> </td><td>Planctomyces </td><td>8 </td></tr><tr><td> </td><td>Proteobacteria </td><td> </td></tr><tr><td> </td><td>    Alphaproteobacteria </td><td>43 </td></tr><tr><td> </td><td>    Betaproteobacteria </td><td>26 </td></tr><tr><td> </td><td>    Deltaproteobacteria </td><td>2 </td></tr><tr><td> </td><td>    Epsilonproteobacteria </td><td>21 </td></tr><tr><td> </td><td>    Gammaproteobacteria </td><td>158 </td></tr><tr><td> </td><td>Spirochaetes </td><td>14 </td></tr><tr><td> </td><td>Uncultured bacterium </td><td>3 </td></tr><tr><td>Plasmids </td><td>Plasmids </td><td>1 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87194419" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="tbl2" class="table-wrap-title" data-id="tbl2"><div class="label -label"><strong>Table 2</strong></div><div class="caption"><p>The distribution of periodic sequences found by the groups of organisms to which they belong</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Category </th><th>Subcategory </th><th>Number of sequences </th></tr></thead><tbody><tr><td>Archaea </td><td>Crenarchaeota </td><td>9 </td></tr><tr><td> </td><td>Euryarchaeota </td><td>20 </td></tr><tr><td>Bacteria </td><td>Actinobacteria </td><td>61 </td></tr><tr><td> </td><td>Bacteroid </td><td>6 </td></tr><tr><td> </td><td>Chlamydiae </td><td>4 </td></tr><tr><td> </td><td>Chlorobi </td><td>2 </td></tr><tr><td> </td><td>Cyanobacteria </td><td>7 </td></tr><tr><td> </td><td>Deinococcus-Thermus </td><td>1 </td></tr><tr><td> </td><td>Thermotogae </td><td>1 </td></tr><tr><td> </td><td colspan="2">Firmicutes </td></tr><tr><td> </td><td>    Firmicutes Bacillales </td><td>33 </td></tr><tr><td> </td><td>    Firmicutes Clostridia </td><td>8 </td></tr><tr><td> </td><td>    Firmicutes Lactobacillales </td><td>22 </td></tr><tr><td> </td><td>    Firmicutes Mollicutes </td><td>4 </td></tr><tr><td> </td><td>Planctomyces </td><td>8 </td></tr><tr><td> </td><td>Proteobacteria </td><td> </td></tr><tr><td> </td><td>    Alphaproteobacteria </td><td>43 </td></tr><tr><td> </td><td>    Betaproteobacteria </td><td>26 </td></tr><tr><td> </td><td>    Deltaproteobacteria </td><td>2 </td></tr><tr><td> </td><td>    Epsilonproteobacteria </td><td>21 </td></tr><tr><td> </td><td>    Gammaproteobacteria </td><td>158 </td></tr><tr><td> </td><td>Spirochaetes </td><td>14 </td></tr><tr><td> </td><td>Uncultured bacterium </td><td>3 </td></tr><tr><td>Plasmids </td><td>Plasmids </td><td>1 </td></tr></tbody></table></div><div content-id="tbl2" class="table-modal"><table><thead name="thead" align=""><tr><th>Category </th><th>Subcategory </th><th>Number of sequences </th></tr></thead><tbody><tr><td>Archaea </td><td>Crenarchaeota </td><td>9 </td></tr><tr><td> </td><td>Euryarchaeota </td><td>20 </td></tr><tr><td>Bacteria </td><td>Actinobacteria </td><td>61 </td></tr><tr><td> </td><td>Bacteroid </td><td>6 </td></tr><tr><td> </td><td>Chlamydiae </td><td>4 </td></tr><tr><td> </td><td>Chlorobi </td><td>2 </td></tr><tr><td> </td><td>Cyanobacteria </td><td>7 </td></tr><tr><td> </td><td>Deinococcus-Thermus </td><td>1 </td></tr><tr><td> </td><td>Thermotogae </td><td>1 </td></tr><tr><td> </td><td colspan="2">Firmicutes </td></tr><tr><td> </td><td>    Firmicutes Bacillales </td><td>33 </td></tr><tr><td> </td><td>    Firmicutes Clostridia </td><td>8 </td></tr><tr><td> </td><td>    Firmicutes Lactobacillales </td><td>22 </td></tr><tr><td> </td><td>    Firmicutes Mollicutes </td><td>4 </td></tr><tr><td> </td><td>Planctomyces </td><td>8 </td></tr><tr><td> </td><td>Proteobacteria </td><td> </td></tr><tr><td> </td><td>    Alphaproteobacteria </td><td>43 </td></tr><tr><td> </td><td>    Betaproteobacteria </td><td>26 </td></tr><tr><td> </td><td>    Deltaproteobacteria </td><td>2 </td></tr><tr><td> </td><td>    Epsilonproteobacteria </td><td>21 </td></tr><tr><td> </td><td>    Gammaproteobacteria </td><td>158 </td></tr><tr><td> </td><td>Spirochaetes </td><td>14 </td></tr><tr><td> </td><td>Uncultured bacterium </td><td>3 </td></tr><tr><td>Plasmids </td><td>Plasmids </td><td>1 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87194419" target="_blank">View Large</a></div></div><p>ACCTACATGGGTTTTAGTATGTTCTACTCGGACTACACCCTATTCTCCGCCTCTTGTGGTTCTTGTGCCG TGCCCCTTCTTACTTACCCCTTTTGCCGTATGCTAAAGATCGAATCCTGTCTAACTTTGAATTAAGTATT</p><p>Since this period is a hidden one, it is impossible to reveal the periodicity ‘visually’ by homology between separate periods. Let us evaluate probability values α and β for this sequence. For this sequence, <em>Z-</em>values determined by the Monte Carlo method are 2.5 ± 0.6 and 8.0 ± 0.7. Using normal distribution for the evaluation of distribution of <em>Z-</em>value, we obtain the value of probability α &gt; 0.05 and the value of probability β &lt; 10<sup>−9</sup>. Thus, it is clear that such periodicity will be statistically insignificant if we use the weight introduced on the basis of BLAST matrix [formulas (3) and (1)] and that it is revealed at a statistically significant level using the information measure introduced by formulas (4) and (1).</p><div class="block-child-p">To make a search of the latent periods in symbolical sequences we are going to use the idea of mutual information. This idea is the keystone of the ID method. ID is a spectrum representing the statistical significance of mutual information for periods of various lengths in the analyzed symbolical sequence. Mutual information between the sequence of interest and artificial symbolical periodic sequences can be used to obtain an ID spectrum. Let the sequence under consideration have a length <em>L</em>. We generate random sequences possessing the periodicity with a period length equal to from 2 to <em>L</em>/2 using numbers as symbols. The artificial sequence with period length equal to <em>n</em> symbols can be presented as: 1,2,…,<em>n</em>, 1,2…<em>n</em>…. Further, we can determine the mutual information between the analyzed sequence and each of the artificial periodic sequences. To do this, we fill the (<em>n</em> × <em>k</em>) matrix <em>M</em> where <em>n</em> shows the period length of the artificial periodic sequence used, and <em>k</em> is the size of the alphabet of the sequence under study. The elements of this matrix are equal the numbers of coincidences of <em>ij</em> (<em>I</em> = 1,2,…,<em>n</em>; <em>j</em> = 1,2,…,<em>k</em>) type between sequences compared. <em>L</em> is the length of the analyzed symbolical sequence, <em>x</em>(<em>i</em>), <em>i</em> = 1,2,…,<em>n</em> are the frequencies of symbols 1,2,…,<em>n</em> in the artificial periodic symbolical sequence; <em>y</em>(<em>j</em>), <em>j</em> = 1,2,…,<em>k</em> are the frequencies of symbols in the analyzed symbolical sequence. The value of the mutual information is calculated using formula <a id="e7" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="e7"><math xmlns="http://www.w3.org/1998/Math/MathML"><mtable columnalign="left" xmlns=""><mtr><mtd><maligngroup></maligngroup><mi>I</mi><mo>=</mo><malignmark></malignmark><mstyle displaystyle="true"><munderover><mo>∑</mo><mtext>l</mtext><mi>n</mi></munderover></mstyle><mrow><mstyle displaystyle="true"><munderover><mo>∑</mo><mtext>l</mtext><mi>k</mi></munderover></mstyle><mrow><mi>M</mi><mo>(</mo><mi>i</mi><mo>,</mo><mi>j</mi><mo>)</mo><mtext>ln </mtext><mi>M</mi><mo>(</mo><mi>i</mi><mo>,</mo><mi>j</mi><mo>)</mo></mrow></mrow><mo>−</mo><mstyle displaystyle="true"><munderover><mo>∑</mo><mtext>l</mtext><mi>n</mi></munderover></mstyle><mrow><mi>x</mi><mo>(</mo><mi>i</mi><mo>)</mo><mtext>ln </mtext><mi>x</mi><mo>(</mo><mi>i</mi><mo>)</mo></mrow></mtd></mtr><mtr><mtd><maligngroup></maligngroup><malignmark></malignmark><mo>−</mo><mstyle displaystyle="true"><munderover><mo>∑</mo><mtext>l</mtext><mi>k</mi></munderover></mstyle><mrow><mi>y</mi><mo>(</mo><mi>j</mi><mo>)</mo><mtext>ln </mtext><mi>y</mi><mo>(</mo><mi>j</mi><mo>)</mo></mrow><mo>+</mo><mi>L</mi><mtext>ln </mtext><mi>L</mi></mtd></mtr></mtable></math></div><div class="label">(7)</div></div></div><div class="block-child-p">For ID construction it is necessary to take into account that the value 2I(<em>n</em>,<em>k</em>) is distributed as χ<sup>2</sup> with (<em>n</em> − 1)(<em>k</em> − 1) degrees of freedom.<a reveal-id="b26" data-open="b26" class="link link-ref link-reveal xref-bibr">26</a> To estimate the statistical significance of I(<em>n</em>,<em>k</em>) the Monte Carlo method is used by means of <em>Z</em>(<em>n</em>,<em>k</em>) calculation using formula <a id="e8" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="e8"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>Z</mi><mrow><mo>(</mo><mi>n</mi><mo>,</mo><mi>k</mi><mo>)</mo></mrow><mo>=</mo><mfrac><mrow><mrow><mo>{</mo><mrow><mtext>I</mtext><mo>(</mo><mi>n</mi><mo>,</mo><mi>k</mi><mo>)</mo></mrow><mo>−</mo><mstyle displaystyle="true"><mover accent="true"><mrow><mtext>I</mtext><mo>(</mo><mi>n</mi><mo>,</mo><mi>k</mi><mo>)</mo></mrow><mo>¯</mo></mover></mstyle><mo>}</mo></mrow></mrow><mrow><msqrt><mrow><mi>D</mi><mrow><mo>(</mo><mrow><mtext>I</mtext><mo>(</mo><mi>n</mi><mo>,</mo><mi>k</mi><mo>)</mo></mrow><mo>)</mo></mrow></mrow></msqrt></mrow></mfrac></mrow></math></div><div class="label">(8)</div></div> where <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mstyle displaystyle="true"><mover accent="true"><mrow><mi>I</mi><mo>(</mo><mi>n</mi><mo>,</mo><mi>k</mi><mo>)</mo></mrow><mo>¯</mo></mover></mstyle></mrow></math></span> and <em>D</em>(I(<em>n</em>,<em>k</em>)) show the mean value and variance of the I(<em>n</em>,<em>k</em>) value, for a set of random matrices with the same sums <em>x</em>(<em>i</em>) and <em>y</em>(<em>j</em>) as in the initial matrix M(<em>n</em>,<em>k</em>).</div><p>The spectrum <em>Z</em>(<em>n</em>,<em>k</em>) is similar to a spectrum of Fourier transformation for numerical sequences but has the following advantages: (i) the calculation of the spectrum does not require any transformation of a symbolical sequence to numerical sequences; (ii) ID allows the revealing of both obvious periodicity and the latent periodicity of a symbolical sequence in which there is no statistically important similarity between any two periods; (iii) the statistical significance of long periods is not spread onto the statistical significance of shorter periods; (iv) on the basis of the matrix M it is possible to determine the type of periodicity (<a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1</a>).</p>                        <a id="87194425" scrollto-destination="87194425"></a>
<div id="fig1" data-id="fig1" class="fig fig-section"><div class="label fig-label"><strong>Figure 1</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/3/10.1093_dnares_dsl004/1/m_dsl004f1.jpeg?Expires=2147483647&Signature=Z1cT6Vorezuzt91tPcVsiebohMnIM1BlbQd6voP5~pBf1AbXvTWeRcHgP6DDcRcDL-YceUajzwMOmJaaVulj~HkGqf4ov601OzxmpJUVOK-VHwVAWhy95LiI0nhS5PbqgYafkeyRGwf8zFhj0LyaBPI0CJQQTtXiWuK3LcyOJCR~C39nDM7hCBS8D55XxCxZ9UP6XC-1HOvnM6BGdsOzPA0HsdRcaG2n9Ufo-RhU3OTZlMGajJSEfvAvHTbDXRrzARfW6Z~TNyvVy9YXzklYeleEr087R~qC~dCUkN0Yu2dU3U7Ka-DaQFxBBdod~868a9dKnVDrqK04yeVX08U~MQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Matrix T shows the symbol frequencies of the sequence with the latent periodicity." path-from-xml="dsl004f1.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87194425/dsl004f1.jpeg" path-from-xml="dsl004f1.jpeg" target="_blank">View large</a><a section="87194425" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/3/10.1093_dnares_dsl004/1/dsl004f1.jpeg?Expires=2147483647&Signature=XjYtbEydxUs2sXw4nErgyGwyxQWpJwrk2XNneZpcHY92Eqncps2h89T9~6f5Rdej2ax2X6A7X5C~YSVIViXxeWfp51SL3lXa8t494z4sSa2~WDixGzTVxWUYu37Iq3CtL6CLp~9No0~5mwgm-CtJuqkBVHZAAH4Pyuf~ijKh6999oiyapecgiO3PILZnLPnAwbQ9jy4MyNp1f5AHgPQ8GzkBGMHCN0W0OVFHSkgQ9WqclwDFUqcmwXTRwO4zYZ4-esoZxhJtsazPggluj7QpjxzypVjEimrbUzIkFYiwJiaahb2S2KHQTIAm-9DnUeDbIiFZFWSzqgtABGcFHHYTSg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87194425&ar=347450&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl004f1.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Matrix <em>T</em> shows the symbol frequencies of the sequence with the latent periodicity.</p></div></div><div content-id="fig1" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 1</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/3/10.1093_dnares_dsl004/1/m_dsl004f1.jpeg?Expires=2147483647&Signature=Z1cT6Vorezuzt91tPcVsiebohMnIM1BlbQd6voP5~pBf1AbXvTWeRcHgP6DDcRcDL-YceUajzwMOmJaaVulj~HkGqf4ov601OzxmpJUVOK-VHwVAWhy95LiI0nhS5PbqgYafkeyRGwf8zFhj0LyaBPI0CJQQTtXiWuK3LcyOJCR~C39nDM7hCBS8D55XxCxZ9UP6XC-1HOvnM6BGdsOzPA0HsdRcaG2n9Ufo-RhU3OTZlMGajJSEfvAvHTbDXRrzARfW6Z~TNyvVy9YXzklYeleEr087R~qC~dCUkN0Yu2dU3U7Ka-DaQFxBBdod~868a9dKnVDrqK04yeVX08U~MQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Matrix T shows the symbol frequencies of the sequence with the latent periodicity." path-from-xml="dsl004f1.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87194425/dsl004f1.jpeg" path-from-xml="dsl004f1.jpeg" target="_blank">View large</a><a section="87194425" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/3/10.1093_dnares_dsl004/1/dsl004f1.jpeg?Expires=2147483647&Signature=XjYtbEydxUs2sXw4nErgyGwyxQWpJwrk2XNneZpcHY92Eqncps2h89T9~6f5Rdej2ax2X6A7X5C~YSVIViXxeWfp51SL3lXa8t494z4sSa2~WDixGzTVxWUYu37Iq3CtL6CLp~9No0~5mwgm-CtJuqkBVHZAAH4Pyuf~ijKh6999oiyapecgiO3PILZnLPnAwbQ9jy4MyNp1f5AHgPQ8GzkBGMHCN0W0OVFHSkgQ9WqclwDFUqcmwXTRwO4zYZ4-esoZxhJtsazPggluj7QpjxzypVjEimrbUzIkFYiwJiaahb2S2KHQTIAm-9DnUeDbIiFZFWSzqgtABGcFHHYTSg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87194425&ar=347450&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl004f1.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Matrix <em>T</em> shows the symbol frequencies of the sequence with the latent periodicity.</p></div></div><p>The window with the size equal to 2000 nt was used for scanning the sequence. The length <em>L</em> of the sequence under consideration was varied in order to find the region of genomic sequence possessing the highest level of statistical significance. The maximum possible value of the length <em>L</em> was equal to the window size.</p><p>The region of the sequence under study is considered to be periodic if the statistical significance <em>Z</em> for this region is greater than some threshold value.</p>                            <h3 scrollto-destination=87194428 id="87194428" class="section-title" >2.2. Algorithm of the latent periodicity classification</h3>
<p>Each periodic region found by ID corresponds to its positional frequency matrix <em>M</em>′. Since we consider the periods of length equal to 2, this matrix can be represented as a vector as in <a reveal-id="fig2" data-open="fig2" class="link link-reveal link-table xref-fig">Fig. 2</a>.</p>                        <a id="87194430" scrollto-destination="87194430"></a>
<div id="fig2" data-id="fig2" class="fig fig-section"><div class="label fig-label"><strong>Figure 2</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/3/10.1093_dnares_dsl004/1/m_dsl004f2.jpeg?Expires=2147483647&Signature=FtM735Uf-QLh8ciDpi9IOhXRR~Mq4t2CmaVcn9zC3aOap23Wl~rXs5594XYjg7XEjH~ktgVg29LJUcH8ZpeQhrJiVopAmrF1Tti0QMH5DFYQehxVOdjt1ZMybyLSV~HcyuPpEtz7RBx0yYi8QqUWxkmf3FFfazQ2p4NEi0iHesC-B41QndJLczjDg076Ejfx3TFjxr4myQhBktYwIlp9VslyZV7~wYciFds66Isxn4ugkLimNAx4~W16pICfkGeg6FiBDsxswZd51ez6fNg-BV-Loojj6M7gXoLtXFyjU5VpAlCfaaLhnmvYkTRclrFOcqEjXPPMxdXeCU6SBGlf5Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Periodicity matrix represented as a vector. Ki (K = {A,T,C,G}, i = {1,2}) represents the frequency (number of times the symbol is observed) of symbol K in i-th position of the period for the region found." path-from-xml="dsl004f2.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87194430/dsl004f2.jpeg" path-from-xml="dsl004f2.jpeg" target="_blank">View large</a><a section="87194430" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/3/10.1093_dnares_dsl004/1/dsl004f2.jpeg?Expires=2147483647&Signature=txbVxvN74XPr3aIWHZIxvKh9cdiVsSaDnd5c-EpI~n0hoEFkxtVDKDAh4PJP9tXMwqDAbHDtAX3x-Y7EY2m2tEFmzaQm8W3EN4-PdU80KVvy5Xwpsx4pw3MATNhU0vrp77wTzHDUoMn-9V5tnGM5R2WX2019mQWUwiznIq4Eep6S~MdFsya1IGoQHPJ43523ySbn-6rKCCBtRNS7qh7McZZov5DqBknILFi4VFY1IP11MWqC3xj36gxK4JR8uaqMUwzgXdMd4f4-Ww50DWtKiejWkvhn~9vi284kxwNnuT2g7WSkb7B1lvThH1dr3Mo-tp7F-nK8kZMJWoCAE3MplA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87194430&ar=347450&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl004f2.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Periodicity matrix represented as a vector. <em>K<sub>i</sub></em> (<em>K</em> = {A,T,C,G}, <em>i</em> = {1,2}) represents the frequency (number of times the symbol is observed) of symbol K in <em>i</em>-th position of the period for the region found.</p></div></div><div content-id="fig2" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 2</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/3/10.1093_dnares_dsl004/1/m_dsl004f2.jpeg?Expires=2147483647&Signature=FtM735Uf-QLh8ciDpi9IOhXRR~Mq4t2CmaVcn9zC3aOap23Wl~rXs5594XYjg7XEjH~ktgVg29LJUcH8ZpeQhrJiVopAmrF1Tti0QMH5DFYQehxVOdjt1ZMybyLSV~HcyuPpEtz7RBx0yYi8QqUWxkmf3FFfazQ2p4NEi0iHesC-B41QndJLczjDg076Ejfx3TFjxr4myQhBktYwIlp9VslyZV7~wYciFds66Isxn4ugkLimNAx4~W16pICfkGeg6FiBDsxswZd51ez6fNg-BV-Loojj6M7gXoLtXFyjU5VpAlCfaaLhnmvYkTRclrFOcqEjXPPMxdXeCU6SBGlf5Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Periodicity matrix represented as a vector. Ki (K = {A,T,C,G}, i = {1,2}) represents the frequency (number of times the symbol is observed) of symbol K in i-th position of the period for the region found." path-from-xml="dsl004f2.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87194430/dsl004f2.jpeg" path-from-xml="dsl004f2.jpeg" target="_blank">View large</a><a section="87194430" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/3/10.1093_dnares_dsl004/1/dsl004f2.jpeg?Expires=2147483647&Signature=txbVxvN74XPr3aIWHZIxvKh9cdiVsSaDnd5c-EpI~n0hoEFkxtVDKDAh4PJP9tXMwqDAbHDtAX3x-Y7EY2m2tEFmzaQm8W3EN4-PdU80KVvy5Xwpsx4pw3MATNhU0vrp77wTzHDUoMn-9V5tnGM5R2WX2019mQWUwiznIq4Eep6S~MdFsya1IGoQHPJ43523ySbn-6rKCCBtRNS7qh7McZZov5DqBknILFi4VFY1IP11MWqC3xj36gxK4JR8uaqMUwzgXdMd4f4-Ww50DWtKiejWkvhn~9vi284kxwNnuT2g7WSkb7B1lvThH1dr3Mo-tp7F-nK8kZMJWoCAE3MplA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87194430&ar=347450&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl004f2.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Periodicity matrix represented as a vector. <em>K<sub>i</sub></em> (<em>K</em> = {A,T,C,G}, <em>i</em> = {1,2}) represents the frequency (number of times the symbol is observed) of symbol K in <em>i</em>-th position of the period for the region found.</p></div></div><p>A block diagram for the algorithm of the latent periodicity classification is shown in <a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3</a>.</p>                        <a id="87194432" scrollto-destination="87194432"></a>
<div id="fig3" data-id="fig3" class="fig fig-section"><div class="label fig-label"><strong>Figure 3</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/3/10.1093_dnares_dsl004/1/m_dsl004f3.jpeg?Expires=2147483647&Signature=ODPYa13Q~VCsQ5TCd35-nazoD~Vr8MZ70Kq5blsAayPyBhuNuVNYtcFK3eV5cPEvcDEv1Hd6aQYGZlHjJp0DX-kHrFf8MkM70SJbSTzdLinLPfCXivOYdeQttNDCj9OfCFyl1SC07VFv1vkOHkDkBvvjM9ofaMbatbmeGXUTh7wk2-9fF8l9ioTJbP9QgUrOzSrgYRjf53T8d9SugJZTxEZ6HXf~WlQwfkEOhtANKZ92r~HOHPss4FJjeC1NjoZbenZSU9jFqAGE-YRqmpzV~dv2dTtBvXWDAD0dML0-DZk15mHd42c~JzFve-SRg2pStR7FWMYpzmspVA4ZHBpw5A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Block diagram of the latent periodicity matrices classification." path-from-xml="dsl004f3.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87194432/dsl004f3.jpeg" path-from-xml="dsl004f3.jpeg" target="_blank">View large</a><a section="87194432" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/3/10.1093_dnares_dsl004/1/dsl004f3.jpeg?Expires=2147483647&Signature=jx5bV7lMqFCZhyKn71Z6T5exmQOp75CyC8VP7OEDtgPYa~7EVJNT4-eOXQYmsPexU9AvGD0hvdUdifngnOiywtgoke9WX0W9J6NICK9sYe0Ac4sNWK2R4pYEO64p3kqs8~Lw36GRqQgldStInhHW48pAG937ftejDy33jfdmaPUCclQuq9LErgWPYSaOvHaaCuqMN~xv9rYYfubU8ghpKp~Qu9r2~A43SG1t8ubeQ7~myK8pmvn6pen6kFk8ErFFQewhCPonl7FXrbFy06HmxPmhzYLzbtzhc4qBILhmvyA0FQhMxgQWcmEbNj2KPIq0VaKY8eQqAK8YMeQA~325bg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87194432&ar=347450&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl004f3.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Block diagram of the latent periodicity matrices classification.</p></div></div><div content-id="fig3" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 3</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/3/10.1093_dnares_dsl004/1/m_dsl004f3.jpeg?Expires=2147483647&Signature=ODPYa13Q~VCsQ5TCd35-nazoD~Vr8MZ70Kq5blsAayPyBhuNuVNYtcFK3eV5cPEvcDEv1Hd6aQYGZlHjJp0DX-kHrFf8MkM70SJbSTzdLinLPfCXivOYdeQttNDCj9OfCFyl1SC07VFv1vkOHkDkBvvjM9ofaMbatbmeGXUTh7wk2-9fF8l9ioTJbP9QgUrOzSrgYRjf53T8d9SugJZTxEZ6HXf~WlQwfkEOhtANKZ92r~HOHPss4FJjeC1NjoZbenZSU9jFqAGE-YRqmpzV~dv2dTtBvXWDAD0dML0-DZk15mHd42c~JzFve-SRg2pStR7FWMYpzmspVA4ZHBpw5A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Block diagram of the latent periodicity matrices classification." path-from-xml="dsl004f3.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87194432/dsl004f3.jpeg" path-from-xml="dsl004f3.jpeg" target="_blank">View large</a><a section="87194432" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/3/10.1093_dnares_dsl004/1/dsl004f3.jpeg?Expires=2147483647&Signature=jx5bV7lMqFCZhyKn71Z6T5exmQOp75CyC8VP7OEDtgPYa~7EVJNT4-eOXQYmsPexU9AvGD0hvdUdifngnOiywtgoke9WX0W9J6NICK9sYe0Ac4sNWK2R4pYEO64p3kqs8~Lw36GRqQgldStInhHW48pAG937ftejDy33jfdmaPUCclQuq9LErgWPYSaOvHaaCuqMN~xv9rYYfubU8ghpKp~Qu9r2~A43SG1t8ubeQ7~myK8pmvn6pen6kFk8ErFFQewhCPonl7FXrbFy06HmxPmhzYLzbtzhc4qBILhmvyA0FQhMxgQWcmEbNj2KPIq0VaKY8eQqAK8YMeQA~325bg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87194432&ar=347450&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl004f3.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Block diagram of the latent periodicity matrices classification.</p></div></div><p>Since the regions of the latent periodicity were of different length, all compared matrices have been normalized to unity. We denoted the period length as <em>N</em> (<em>N</em> = 2), each matrix of the latent periodicity has been represented as a vector of nucleotides' frequencies distributed over 4<em>N</em> ranks. We chose the Pearson statistics as a measure of the similarity of two vectors (or as a distance between them). To use this statistics, we build two matrices—M1 and M2.</p><p>Let us denote a matrix obtained from a combination of such two vectors as M1. It has the marginal frequencies <em>X</em>(<em>i</em>) = Σ<sub>j</sub> M1(<em>i,j</em>), and <em>Y</em>(<em>j</em>) = Σ<sub>i</sub> M1(<em>i,j</em>), where Σ<em><sub>i</sub>X</em>(<em>i</em>) = Σ<em><sub>j</sub>Y</em>(<em>j</em>) = 2.</p><div class="block-child-p">A matrix M2 has been constructed as the expected one over a set of the random matrices, having the same marginal quantities <em>X</em>(<em>i</em>) and <em>Y</em>(<em>j</em>) as M1. Its elements are defined as M2(<em>i,j</em>) = (1/2)<em>X</em>(<em>i</em>) × <em>Y</em>(<em>j</em>). The Pearson statistics, whose values' distribution follows the χ<sup>2</sup>, allows the estimation of the deviation of quantities in matrix M1 from expected ones in M2 matrix: <a id="e9" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="e9"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>U</mi><mo>=</mo><msup><mi>χ</mi><mn>2</mn></msup><mo>=</mo><mfrac><mrow><msub><mo>∑</mo><mrow><mi>i</mi><mo>,</mo><mi>j</mi></mrow></msub><mrow><mrow><mo>{</mo><mrow><msup><mrow><mrow><mo>(</mo><mrow><mtext>M1</mtext><mo>(</mo><mi>i</mi><mo>,</mo><mi>j</mi><mo>)</mo><mo>−</mo><mtext>M</mtext><mn>2</mn><mo>(</mo><mi>i</mi><mo>,</mo><mi>j</mi><mo>)</mo></mrow><mo>)</mo></mrow></mrow><mn>2</mn></msup></mrow><mo>}</mo></mrow></mrow></mrow><mrow><mi>M</mi><mn>2</mn><mo>(</mo><mi>i</mi><mo>,</mo><mi>j</mi><mo>)</mo></mrow></mfrac></mrow></math></div><div class="label">(9)</div></div> The number of the χ<sup>2</sup> degrees of freedom was equal to (2 × 4<em>N</em>) − 1, that is, the number of comparison ranks (the number of matrix M1 or M2 elements) minus the number of independent linkages—a single claim on constancy of marginal elements: <em>X</em>(1) = <em>X</em>(2) = 1.</div><p>The pairwise comparison has been made between the vectors shown in <a reveal-id="fig4" data-open="fig4" class="link link-reveal link-table xref-fig">Fig. 4</a>. The lower index in <a reveal-id="fig4" data-open="fig4" class="link link-reveal link-table xref-fig">Fig. 4</a> corresponds to the period position; the upper one reflects an ordinal number of the compared vector.</p>                        <a id="87194437" scrollto-destination="87194437"></a>
<div id="fig4" data-id="fig4" class="fig fig-section"><div class="label fig-label"><strong>Figure 4</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/3/10.1093_dnares_dsl004/1/m_dsl004f4.jpeg?Expires=2147483647&Signature=TP096M1umTW6yOFGNPswDjKI4Ptl7mqsKCoEtPsH6Nrc676XeurO3mwreMilC9OSaQLELcJj7CchcciyREnQ4ifblXJjRWxQS6uD9Nnl5ToxPvi1EEKi6L75Pmp82o-EvvuhRFOYN5bcUjvsbpeO-kio4YMcx9~rWJJ7aeynymdYmyPmREI-J0Re0r3RfiRAk2GNfk-YEkOpciFHy-FVRY~ngpn2yaHpJ6F97GP1K8aXm~5L4itdpI1h2xx9qtmBD~PGTyxDSWv2tnJpBYjiazwWO7dheEbF0BwuVEI1SSVbfu47wAt4SMnaEdOf1DsYRFnmN5OFPDDTIeqTPveKtA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="A scheme of comparison between the two latent dinucleotide periodicity matrices is shown. Both matrices are presented as 4N-dimensional vectors. X and Y are marginal frequencies, X(i) = Σj M1(i,j), and Y(j) = Σi M1(i,j), where ΣiX(i) = ΣjY(j) = 2." path-from-xml="dsl004f4.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87194437/dsl004f4.jpeg" path-from-xml="dsl004f4.jpeg" target="_blank">View large</a><a section="87194437" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/3/10.1093_dnares_dsl004/1/dsl004f4.jpeg?Expires=2147483647&Signature=0HWp9I7NgcME6UNpBr33jv8rK053R7zSIWTy2bnfR~21xX043UjX1gWBSmtILHRw~lzV3I6DGjrAElHTCYAXEBJYEOT3C3ifIS2SOm4ms1W7AIqX7lTMDHA7HA0SyO4f7hFn63sbV~FDKfmIRukwe2IapDr4jo7t6z76w4uCRWHOPUWKDAknG8YrQDDC6l9T4shHlHFDdBYNbaDXaAWBS9gOGhLsLy5k0EXQeCyMsUiLFbjfgruZcJBjgQTVqpNG92zkM6mYeOIYPO5nOInh1ZICEXgWng8Yej5nan6Mco477sCatyS3jJPzNmN2Q~k~VstX8qg4d6R4mw6NUxnWOA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87194437&ar=347450&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl004f4.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>A scheme of comparison between the two latent dinucleotide periodicity matrices is shown. Both matrices are presented as 4<em>N</em>-dimensional vectors. <em>X</em> and <em>Y</em> are marginal frequencies, <em>X</em>(<em>i</em>) = Σ<sub>j</sub> M1(<em>i,j</em>), and <em>Y</em>(<em>j</em>) = Σ<sub>i</sub> M1(<em>i,j</em>), where Σ<sub>i</sub><em>X</em>(<em>i</em>) = Σ<sub>j</sub><em>Y</em>(<em>j</em>) = 2.</p></div></div><div content-id="fig4" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 4</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/3/10.1093_dnares_dsl004/1/m_dsl004f4.jpeg?Expires=2147483647&Signature=TP096M1umTW6yOFGNPswDjKI4Ptl7mqsKCoEtPsH6Nrc676XeurO3mwreMilC9OSaQLELcJj7CchcciyREnQ4ifblXJjRWxQS6uD9Nnl5ToxPvi1EEKi6L75Pmp82o-EvvuhRFOYN5bcUjvsbpeO-kio4YMcx9~rWJJ7aeynymdYmyPmREI-J0Re0r3RfiRAk2GNfk-YEkOpciFHy-FVRY~ngpn2yaHpJ6F97GP1K8aXm~5L4itdpI1h2xx9qtmBD~PGTyxDSWv2tnJpBYjiazwWO7dheEbF0BwuVEI1SSVbfu47wAt4SMnaEdOf1DsYRFnmN5OFPDDTIeqTPveKtA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="A scheme of comparison between the two latent dinucleotide periodicity matrices is shown. Both matrices are presented as 4N-dimensional vectors. X and Y are marginal frequencies, X(i) = Σj M1(i,j), and Y(j) = Σi M1(i,j), where ΣiX(i) = ΣjY(j) = 2." path-from-xml="dsl004f4.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87194437/dsl004f4.jpeg" path-from-xml="dsl004f4.jpeg" target="_blank">View large</a><a section="87194437" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/3/10.1093_dnares_dsl004/1/dsl004f4.jpeg?Expires=2147483647&Signature=0HWp9I7NgcME6UNpBr33jv8rK053R7zSIWTy2bnfR~21xX043UjX1gWBSmtILHRw~lzV3I6DGjrAElHTCYAXEBJYEOT3C3ifIS2SOm4ms1W7AIqX7lTMDHA7HA0SyO4f7hFn63sbV~FDKfmIRukwe2IapDr4jo7t6z76w4uCRWHOPUWKDAknG8YrQDDC6l9T4shHlHFDdBYNbaDXaAWBS9gOGhLsLy5k0EXQeCyMsUiLFbjfgruZcJBjgQTVqpNG92zkM6mYeOIYPO5nOInh1ZICEXgWng8Yej5nan6Mco477sCatyS3jJPzNmN2Q~k~VstX8qg4d6R4mw6NUxnWOA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87194437&ar=347450&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl004f4.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>A scheme of comparison between the two latent dinucleotide periodicity matrices is shown. Both matrices are presented as 4<em>N</em>-dimensional vectors. <em>X</em> and <em>Y</em> are marginal frequencies, <em>X</em>(<em>i</em>) = Σ<sub>j</sub> M1(<em>i,j</em>), and <em>Y</em>(<em>j</em>) = Σ<sub>i</sub> M1(<em>i,j</em>), where Σ<sub>i</sub><em>X</em>(<em>i</em>) = Σ<sub>j</sub><em>Y</em>(<em>j</em>) = 2.</p></div></div><p>General comparison scheme between the vectors was as follows. At first we performed the pairwise comparison of all the initial periodicity vectors. For each pair of vectors we calculated the Pearson statistics. While making a comparison, we took into consideration all cyclic permutations of vectors' columns, which was necessary because of uncertainty of the period start position. A possibility of classic DNA inversions was also considered in this calculation. To achieve this, we replaced one of the vectors being compared by its complementary and inverse variant.</p><p>In a second step, we chose the two vectors for which the value of the Pearson statistics was minimal. If this value corresponded to accidental probability of less than or equal to 5%, then these two vectors were combined via recapitulation of their elements. The elements of a new vector were calculated as weighted sums of the elements of two source vectors. The contribution of the specific vector to the sum was greater, the higher the number of vectors that had been already merged into it. A cyclic permutation, fixed inverse and complementary transformation were considered while making the vectors' combination. Such a new vector was then normalized again to unity. After this we returned to the first step, but we have excluded two merged vectors from the set and replaced them by one vector representing their combination.</p><p>The process of vector comparison and merging had been continued until the minimal value of the Pearson statistics for the set of vectors became greater than the χ<sup>2</sup> value corresponding to 5% level. The vectors that were left up to this moment were considered as the periodicity classes.</p><p>The classes obtained were then represented on a tree diagram in <a reveal-id="fig5" data-open="fig5" class="link link-reveal link-table xref-fig">Fig. 5</a>. The χ<sup>2</sup> value was chosen as the measure of dissimilarity between the class matrices in pairwise comparison. We used the minimum-variance method<a reveal-id="b27" data-open="b27" class="link link-ref link-reveal xref-bibr">27</a> to build the tree diagram of classes' similarity.</p>                        <a id="87194442" scrollto-destination="87194442"></a>
<div id="fig5" data-id="fig5" class="fig fig-section"><div class="label fig-label"><strong>Figure 5</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/3/10.1093_dnares_dsl004/1/m_dsl004f5.jpeg?Expires=2147483647&Signature=kBqYmGpGTyOdTJ3qqXxvONqB6y~wxVvP9cddkP5-Btd6b2hjnhOkyNsF4AzcNJOXvTaTpXS4lKCTtkZ9cQ7Pt5pGtU7KoebTsTT5JPB-EDvM7longcwbpauDO6DIVIhL0q9mbbMmH4kot6ziUU1nJpdRtu0SS-HOWHrcxr4Et2QRp9VLMGWiYlumP5W8ZOHT~ITfoAHz-PlvgVp67y90VrHHO2kuvZEL0acsvExr~66kbJikg-Cd1o4dSkrf3iOMZeA4ZCEjFMDt8VUsoJacVeBLlkhkMeiDpav8ZsaYlsBd3Tp5d2-O~8d4YrO9agdDd5SjMQUd-7NIs2xVfWWy7Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="A tree diagram for the 45 classes of dinucleotide periodicity. The χ2 value was chosen as the measure of dissimilarity between the class matrices in pairwise comparison. We used the minimum-variance method to build the tree diagram of classes' similarity." path-from-xml="dsl004f5.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87194442/dsl004f5.jpeg" path-from-xml="dsl004f5.jpeg" target="_blank">View large</a><a section="87194442" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/3/10.1093_dnares_dsl004/1/dsl004f5.jpeg?Expires=2147483647&Signature=YXJDK6-~6UVA14gAx0q2wnwIiv9qocjfqObRSeFTpR1sN9sJPbFMbMTgkl13BrG-GDnxV15B29absDpteGJFHkvVPyJKnlRiy9sFLL6Fp3p~lKAbU9Km4i-rdO~AuNY~wiC4Zpn0p~inSOZyXS3TKnJG78-l0xOIsgapxIfaP~AHHL~bubfIKusBHMKScy9HjQKC0pIy4DGEid140Dm2fu9mMtpeZJ9AqaH~M5AHytzX3nn8rx-7iGzre2kpmUJLbpCK8l19tX8Feh~W2KsRwA9QC~m8VDuQaN6pgzj4Ef1K4UfmHswSoiuI~OoxXR8KY5I-CaoVZwtdMUUK1NYquA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87194442&ar=347450&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl004f5.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>A tree diagram for the 45 classes of dinucleotide periodicity. The χ<sup>2</sup> value was chosen as the measure of dissimilarity between the class matrices in pairwise comparison. We used the minimum-variance method to build the tree diagram of classes' similarity.</p></div></div><div content-id="fig5" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 5</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/3/10.1093_dnares_dsl004/1/m_dsl004f5.jpeg?Expires=2147483647&Signature=kBqYmGpGTyOdTJ3qqXxvONqB6y~wxVvP9cddkP5-Btd6b2hjnhOkyNsF4AzcNJOXvTaTpXS4lKCTtkZ9cQ7Pt5pGtU7KoebTsTT5JPB-EDvM7longcwbpauDO6DIVIhL0q9mbbMmH4kot6ziUU1nJpdRtu0SS-HOWHrcxr4Et2QRp9VLMGWiYlumP5W8ZOHT~ITfoAHz-PlvgVp67y90VrHHO2kuvZEL0acsvExr~66kbJikg-Cd1o4dSkrf3iOMZeA4ZCEjFMDt8VUsoJacVeBLlkhkMeiDpav8ZsaYlsBd3Tp5d2-O~8d4YrO9agdDd5SjMQUd-7NIs2xVfWWy7Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="A tree diagram for the 45 classes of dinucleotide periodicity. The χ2 value was chosen as the measure of dissimilarity between the class matrices in pairwise comparison. We used the minimum-variance method to build the tree diagram of classes' similarity." path-from-xml="dsl004f5.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87194442/dsl004f5.jpeg" path-from-xml="dsl004f5.jpeg" target="_blank">View large</a><a section="87194442" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/3/10.1093_dnares_dsl004/1/dsl004f5.jpeg?Expires=2147483647&Signature=YXJDK6-~6UVA14gAx0q2wnwIiv9qocjfqObRSeFTpR1sN9sJPbFMbMTgkl13BrG-GDnxV15B29absDpteGJFHkvVPyJKnlRiy9sFLL6Fp3p~lKAbU9Km4i-rdO~AuNY~wiC4Zpn0p~inSOZyXS3TKnJG78-l0xOIsgapxIfaP~AHHL~bubfIKusBHMKScy9HjQKC0pIy4DGEid140Dm2fu9mMtpeZJ9AqaH~M5AHytzX3nn8rx-7iGzre2kpmUJLbpCK8l19tX8Feh~W2KsRwA9QC~m8VDuQaN6pgzj4Ef1K4UfmHswSoiuI~OoxXR8KY5I-CaoVZwtdMUUK1NYquA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87194442&ar=347450&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl004f5.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>A tree diagram for the 45 classes of dinucleotide periodicity. The χ<sup>2</sup> value was chosen as the measure of dissimilarity between the class matrices in pairwise comparison. We used the minimum-variance method to build the tree diagram of classes' similarity.</p></div></div><p>Let us note that critical level of the χ<sup>2</sup> value was estimated as the result of all 2<em>N</em> trials in searching for pairwise vector similarity. An accidental probability of similarity found in 2<em>N</em> trials: α = 1 − (1 − <em>P</em>)<sup>2</sup><em><sup>N</sup></em> should be less than or equal to 5%. From this point a critical level of accidental probability in one trial <em>P</em> was calculated by using the inverse χ<sup>2</sup> function.</p>                            <h3 scrollto-destination=87194444 id="87194444" class="section-title" >2.3. Modified profile analysis</h3>
<p>We have used the dynamic profile alignment approach,<a reveal-id="b21" data-open="b21" class="link link-ref link-reveal xref-bibr">21</a> which takes into account divergence of sequences as due to spot mutations and also indels. The method unites algorithms of dynamic programming for finding the best alignment<a reveal-id="b28" data-open="b28" class="link link-ref link-reveal xref-bibr">28</a> with analysis of position specific nucleotides as in a profile.<a reveal-id="b29" data-open="b29" class="link link-ref link-reveal xref-bibr">29</a> Since optimal alignment of analysed sequence has been built against a class matrix, that represents distribution of base weights over consensus positions, here we will use a term of alignment against position-specific weight matrix (PSWM). The procedure for building the PSWM is similar in a way with the one described in ref. (<a reveal-id="b20" data-open="b20" class="link link-ref link-reveal xref-bibr">20</a>), but there are some differences that will be described below.</p><p>In order to find periodic sequences in GenBank we have built position-specific frequency matrixes of the nucleotides (PSMs) for each of the periodicity classes obtained in the previous step. To prevent distortion of base frequencies in isotypical periodic sequences, (effect of unequal representation) we have used the Sibbald and Argos algorithm<a reveal-id="b30" data-open="b30" class="link link-ref link-reveal xref-bibr">30</a> to weigh up each sequence inversely proportional to the number of sequences that are highly similar to this sequence.</p><div class="block-child-p">Obtained PSMs have been converted to PSWMs in accordance with the formulae: <a id="e10" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="e10"><math xmlns="http://www.w3.org/1998/Math/MathML"><mtable columnalign="left" xmlns=""><mtr><mtd><maligngroup></maligngroup><malignmark></malignmark><mrow><msup><mi>w</mi><mo>′</mo></msup><mo>(</mo><mi>S</mi><mo>,</mo><mi>j</mi><mo>)</mo><mo>=</mo><mi>f</mi><mo>(</mo><mi>S</mi><mo>,</mo><mi>j</mi><mo>)</mo><mtext>ln</mtext><mrow><mo>{</mo><mrow><mi>f</mi><mo>(</mo><mi>S</mi><mo>,</mo><mi>j</mi><mo>)</mo><mo>/</mo><mi>p</mi><mo>(</mo><mi>S</mi><mo>)</mo></mrow><mo>}</mo></mrow></mrow></mtd></mtr><mtr><mtd><maligngroup></maligngroup><malignmark></malignmark><mrow><mi>w</mi><mo>(</mo><mi>S</mi><mo>,</mo><mi>j</mi><mo>)</mo><mo>=</mo><msup><mi>w</mi><mo>′</mo></msup><mo>(</mo><mi>S</mi><mo>,</mo><mi>j</mi><mo>)</mo><mo>−</mo><msup><mover accent="true"><mi>w</mi><mo>∼</mo></mover><mo>′</mo></msup><mo>(</mo><mi>j</mi><mo>)</mo></mrow></mtd></mtr></mtable></math></div><div class="label">(10)</div></div> where <em>S</em> = {A,T,C,G}, <em>f</em>(<em>S,j</em>)—element of PSM matrix, <em>P</em>(<em>S</em>)=∑<sub><em>j</em></sub><em>f</em>(<em>S</em>, <em>j</em>), <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msup><mover accent="true"><mi>w</mi><mo>∼</mo></mover><mo>′</mo></msup><mrow><mo>(</mo><mi>j</mi><mo>)</mo></mrow><mo>=</mo><mn>0.25</mn><mstyle displaystyle="false"><munder><mo>∑</mo><mi>S</mi></munder></mstyle><mrow><msup><mi>w</mi><mo>′</mo></msup><mrow><mo>(</mo><mrow><mi>S</mi><mo>,</mo><mi>j</mi></mrow><mo>)</mo></mrow></mrow></mrow></math></span>—the mean weight of the bases in the <em>j</em>-column of the PSM matrix, <em>w</em>(<em>S,j</em>)—a weight of base <em>S</em> in the PSWM matrix.</div><p>If we use <em>w</em>′ only, rare nucleotides would have almost zero weight. To avoid this, we found mean weight for each position and subtracted it from <em>w</em>′ to obtain the modified weight <em>w</em>. In this case, even very rare nucleotides would have negative weight, though the function <em>w</em> at small frequencies <em>f</em>(<em>S, j</em>) is somewhat non-monotomic.</p><p>Therefore the transition to PSWM assists in assigning the higher weight to rarely occurring bases in the periodicity class when they have high occurrence frequency at the given position in the PSM and vice versa, decreasing the weight of bases with low occurrence frequency at the given position.<a reveal-id="b20" data-open="b20" class="link link-ref link-reveal xref-bibr">20</a></p>                            <h3 scrollto-destination=87194450 id="87194450" class="section-title" >2.4. Algorithm of dynamic alignment of PSWM</h3>
<div class="block-child-p">We have used a dynamic algorithm for finding local similarity,<a reveal-id="b28" data-open="b28" class="link link-ref link-reveal xref-bibr">28</a> also known as the Smith–Waterman algorithm,<a reveal-id="b31" data-open="b31" class="link link-ref link-reveal xref-bibr">31</a> to align the GenBank sequence being analysed against PSWM. Elements of the alignment matrix have been defined in accordance with formulae: <a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><div class="fig-graphic"><a class="jumplink-placeholder" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/3/10.1093_dnares_dsl004/1/dsl004a1.jpeg?Expires=2147483647&Signature=V4Usn~XOFKNk0AQeU8g3RsFkDAN~sTrSDvUmnPRK~RJd2cWawd-YXHKbnwVNk8v9KScct-Q4D2VhJ3LhOvgH8Rq5i8ADOnSCWYphLVhujrUPzJdyv7KhFxBaEUoGtU5Pc66lnVk9EAKwsnbJqDessm~or6fpqKNo~Ai9E7j8FctcPoyCVZ8oqTV0fauWy-6S8~kLCtux2ry5c1jQmTeC847y1BeXQY-XfZCe-muovtzs6Ghf7XMOM2lfentxUMrTuzN6G0oSZfwkLD3SXcBGPd~43JOHQ74MNDD2ZZ1bvAx0mSDSx2mkjAL3VbVNs~XsbxiYqEOKawKR-Z0sLJTOIg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" id="dsl004a1.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/3/10.1093_dnares_dsl004/1/m_dsl004a1.jpeg?Expires=2147483647&Signature=GAJlEsPx8CyH4mjFzjvRYMmilzMx8nL~CA7CJBMNUnQ5UbxE~DkPCFAFIdCsJ7kVv9ZE33d-0-1nYQr-W8~0xEPzWAXivnbBRxoxq1ThN5TjAfPfZH8oaHqq1ZQOW6vv5hq6sz5rEz1Ybn16es1kdkTv9BulIdI09Yqt8J04wgWyKHrxvkjUNJUn4h97f-MT8eqa6YpIwH52oySRimxp1ZuZsun9Qak1FBGkp-I7zr4X2fb-j7jrREknvQpDMFRrVfhL24SlthTW-0ETkQbYDZZ2bxQ9fQSAKIH6gIdlyzbH1lWHqiA0QAuDq5-5-5At9d-EN84EROVJxKFaQ3~e4A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsl004a1.jpeg" /></a></div></div></div><a id="e11" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="e11"><math xmlns="http://www.w3.org/1998/Math/MathML"><mtable columnalign="left" xmlns=""><mtr><mtd><mrow><mi>F</mi><mrow><mo>(</mo><mrow><mn>0</mn><mo>,</mo><mn>0</mn></mrow><mo>)</mo></mrow></mrow><mrow><mi>F</mi><mrow><mo>(</mo><mrow><mn>0</mn><mo>,</mo><mn>0</mn></mrow><mo>)</mo></mrow><mo>=</mo><mn>0.0</mn><mo>;</mo><mi>F</mi><mrow><mo>(</mo><mrow><mi>i</mi><mo>,</mo><mn>0</mn></mrow><mo>)</mo></mrow><mo>=</mo><mi>F</mi><mrow><mo>(</mo><mrow><mn>0</mn><mo>,</mo><mn>0</mn></mrow><mo>)</mo></mrow><mo>−</mo><msub><mi>v</mi><mi>d</mi></msub><mrow><mo>(</mo><mrow><mn>1</mn><mo>+</mo><mo>log</mo><mrow><mo>(</mo><mi>i</mi><mo>)</mo></mrow></mrow><mo>)</mo></mrow><mo>;</mo></mrow></mtd></mtr><mtr><mtd><mrow><mi>F</mi><mrow><mo>(</mo><mrow><mn>0</mn><mo>,</mo><mi>j</mi></mrow><mo>)</mo></mrow><mo>=</mo><mi>F</mi><mrow><mo>(</mo><mrow><mn>0</mn><mo>,</mo><mn>0</mn></mrow><mo>)</mo></mrow><mo>−</mo><msub><mi>v</mi><mi>d</mi></msub><mrow><mo>(</mo><mrow><mn>1</mn><mo>+</mo><mtext>log</mtext><mrow><mo>(</mo><mi>j</mi><mo>)</mo></mrow></mrow><mo>)</mo></mrow></mrow></mtd></mtr></mtable></math></div><div class="label">(11)</div></div> where <em>i</em> is the position of the base in the analysed sequence, <em>j</em> is the position in consensus, <em>d</em> max = 40 is the maximal analysed length of indels, <em>v<sub>d</sub></em> = 1.0 is the penalty for opening indel, and <em>w</em>(<em>S</em>(<em>i</em>),<em>j</em>) is the PSWM element, calculated through formulae (10), where <em>S</em>(<em>i</em>) is a base at position <em>i</em> of the sequence being analysed. Scanning of PSWM through GenBank has been carried out with a step of 20 bases while the scanning window size (and thus the maximal possible length of subsequence) was equal to 1000 nt. The periodicity class consensus was reproduced as many times as it was required to match the length of maximal subsequence. At each step an analysed sequence has been aligned against PSWM. We filled the matrix <em>F</em>(<em>k,j</em>) completely and then found its maximal element <em>f</em><sub>max</sub>(<em>k<sub>m</sub>,j<sub>m</sub></em>). Depending on the <em>f</em><sub>max</sub> position, we determined the optimal alignment as the way from the maximal element to the first zero with coordinates (<em>k</em><sub>0</sub>,<em>j</em><sub>0</sub>). This allows us to find the ‘maximal subsequence’.<a reveal-id="b28" data-open="b28" class="link link-ref link-reveal xref-bibr">28</a> An alignment obtained shows this maximal subsequence <em>S<sub>m</sub></em>, and the location of related PSWM is marked by the nucleotides of higher weight. In order to speed up the calculation, we limited the maximal nucleotide subsequence to the range |∣<em>k</em> − <em>j</em>∣ &lt; 30. The cumulative weight of the found alignment was defined as: <a id="e12" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="e12"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>W</mi><mo>=</mo><mstyle displaystyle="true"><munderover><mo>∑</mo><mrow><msub><mi>j</mi><mn>0</mn></msub></mrow><mrow><msub><mi>j</mi><mi>m</mi></msub></mrow></munderover></mstyle><mrow><mi>w</mi><mrow><mo>(</mo><mrow><mi>S</mi><mrow><mo>(</mo><mi>i</mi><mo>)</mo></mrow><mo>,</mo><mi>j</mi></mrow><mo>)</mo></mrow></mrow></mrow></math></div><div class="label">(12)</div></div></div>                            <h3 scrollto-destination=87194452 id="87194452" class="section-title" >2.5. Statistical significance of similarity</h3>
<div class="block-child-p">To determine the statistical significance of the found alignment of the GenBank sequence, the probability of alignment of the given PSWM against random sequences with the same symbol composition needs to be estimated. To do this, the alignment matrix <em>F</em>′ was filled using the formulae <a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><div class="fig-graphic"><a class="jumplink-placeholder" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/3/10.1093_dnares_dsl004/1/dsl004a2.jpeg?Expires=2147483647&Signature=BQNN81onPlqsL7eU0rRcrbWRgCag85~FRElIx-3LYfAcWjcGztgQmiK-BvI1SjpH8o3Ch8ULU37ag-v779Zccdi0E3Qzl8yvDdhXoWjhFBalEbjCptol7z0GnBW51bIOSx8zuj8NTZrH-vXVoAB~P1gX~s4XQhx2XK3HDvxFwTS3bWD~5jIBywNwrlWsJIQm3aR1WgRG0kfTC6PPK2Gw-EHBj507mx27ybUOOiMo1Qz-2mNIJ60mdlvtKuYgbbZEPgGdA9M62pShMdCtstFQKRbD0vRj96Sxnkwh8uqsB8kLCJ4zFy4SRcY3De5t049wrpDLWZg~6wMgTls3xtWUjg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" id="dsl004a2.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/3/10.1093_dnares_dsl004/1/m_dsl004a2.jpeg?Expires=2147483647&Signature=ZQsDEEYiGKuhIubKk8FQT-1ZYI8UBqirg5Q5kc3WX8jYXeeRt~VGNPXZAQhfUDM7Vcj-VLjieDjNbw9yAmEIvMOecTVKotAzu9u8tEEEPb~1nmGP3VYldTwMulp~NS1aiE5QQUlwa4Tr6sU4VjpM6Msn~xbE8IlxRSRcZR9GTn4AbEsj~2kBtXDJATIhRd5UunyjBqGjwb2fjIqc7dEfR0NTntT~GmQB1xwXj7BeamHogPuYlN-6fno9imqq2qecFvZ6Y6y2fUx0hTEY4hkMVhOCgeR~quixR-A7hV1B90eMH~tPNQlTbZZgfX9Gko2cigd4SGMS7J9RxbseEcBRhQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsl004a2.jpeg" /></a></div></div></div><a id="e13" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="e13"><math xmlns="http://www.w3.org/1998/Math/MathML"><mtable columnalign="left" xmlns=""><mtr><mtd><mrow><msup><mi>F</mi><mo>′</mo></msup><mrow><mo>(</mo><mrow><mn>0</mn><mo>,</mo><mn>0</mn></mrow><mo>)</mo></mrow></mrow><mrow><msup><mi>F</mi><mo>′</mo></msup></mrow><mrow><mo>(</mo><mrow><mn>0</mn><mo>,</mo><mn>0</mn></mrow><mo>)</mo></mrow><mo>=</mo><mn>0.0</mn><mo>;</mo><msup><mi>F</mi><mo>′</mo></msup><mrow><mo>(</mo><mrow><mi>i</mi><mo>,</mo><mn>0</mn></mrow><mo>)</mo></mrow><mo>=</mo><msup><mi>F</mi><mo>′</mo></msup><mrow><mo>(</mo><mrow><mn>0</mn><mo>,</mo><mn>0</mn></mrow><mo>)</mo></mrow><mo>−</mo><msub><mi>v</mi><mi>d</mi></msub><mrow><mo>(</mo><mrow><mn>1</mn><mo>+</mo><mo>log</mo><mrow><mo>(</mo><mi>i</mi><mo>)</mo></mrow></mrow><mo>)</mo></mrow><mo>;</mo></mtd></mtr><mtr><mtd><mrow><mi>F</mi><mrow><mo>(</mo><mrow><mn>0</mn><mo>,</mo><mi>j</mi></mrow><mo>)</mo></mrow></mrow><mo>=</mo><msup><mi>F</mi><mo>′</mo></msup><mrow><mo>(</mo><mrow><mn>0</mn><mo>,</mo><mn>0</mn></mrow><mo>)</mo></mrow><mo>−</mo><msub><mi>v</mi><mi>d</mi></msub><mrow><mo>(</mo><mrow><mn>1</mn><mo>+</mo><mo>log</mo><mrow><mo>(</mo><mi>j</mi><mo>)</mo></mrow></mrow><mo>)</mo></mrow></mtd></mtr></mtable></math></div><div class="label">(13)</div></div> e.g. in a same way as in formulae (11) but without considering zero values for choosing the maximum. We then calculated <em>d</em> = <em>f</em>(<em>k<sub>m</sub>,j<sub>m</sub></em>) − <em>f</em>(<em>k</em><sub>0</sub>,<em>j</em><sub>0</sub>). If we use formulae (11), the distribution of <em>d</em> is close to normal.<a reveal-id="b32" data-open="b32" class="link link-ref link-reveal xref-bibr">32</a> We confirmed this by using the imitation modeling (Monte Carlo method). We generated 100 sequences with the same symbol composition (i.e. symbol frequencies and triplet correlation) as in the sequence under consideration and then filled <em>F</em>′ matrix for each such sequence. In other words, to determine the statistical significance of the found local alignment we have assumed it to be a global one (by fixing its co-ordinates) during imitation modeling. In such a case most calculations could be carried out before the databank scanning that greatly speeds up the calculations.</div><div class="block-child-p">As a measure of statistical significance, we have determined the <em>Z</em>-score; that is, the normalized deviation of the found sequence alignment weight from the average weight of random sequences alignment against PSWM: <a id="e14" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="e14"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>Z</mi><mo>=</mo><mfrac><mrow><mrow><mo>(</mo><mrow><mi>W</mi><mo>− </mo><mi>M</mi><mrow><mo>( </mo><mrow><msub><mi>W</mi><mtext>rnd</mtext></msub></mrow><mo>)</mo></mrow></mrow><mo>)</mo></mrow></mrow><mrow><mi>σ</mi><mrow><mo>( </mo><mrow><msub><mi>W</mi><mtext>rnd</mtext></msub></mrow><mo>)</mo></mrow></mrow></mfrac><mo>,</mo></mrow></math></div><div class="label">(14)</div></div> where <em>W</em>—found alignment weight, <em>W</em><sub>rnd</sub>—random sequence alignment weight, <em>M</em> and σ—average value and standard variance of <em>W</em><sub>rnd</sub>, respectively.</div><p>We set up the minimum length of a periodical sequence to be found equal to 40 bases.</p><p>All significant results revealed were filtered in order to exclude the overlapping sequences. If an overlapped region exceeded 30% of the length of the smaller of two overlapped alignments, only the alignment with the greater <em>Z</em>-score has been left.</p>                            <h3 scrollto-destination=87194457 id="87194457" class="section-title" >2.6. Statistical test for the periodicity</h3>
<p>Since the method of MPA ensures only the statistical significance of sequence similarity, not its periodicity, we had to perform additional statistical test for the sequences found by this method. To do this, we have calculated the ID spectrum for all the sequences found by MPA in the same way as it was described in Section 2.1. We used the Monte Carlo method to calculate the statistical significance. For each sequence we have generated 200 sequences by randomly shuffling its symbols and then calculated mean, variance and, finally, <em>Z</em>-score as it was described above. The sequence was considered to be periodic with period length equal to 2 if the value corresponding to this length in ID spectrum was maximal and also was equal to or greater than 7.0. This test ensures that the sequences showing the given parameters possess the dinucleotide periodicity at statistically significant level.</p>                            <h2 scrollto-destination=87194459 id="87194459" class="section-title" >3. Results and discussion</h2>
                            <h3 scrollto-destination=87194460 id="87194460" class="section-title" >3.1. Classification results for bacterial genomes</h3>
<p>Making a search for the latent dinucleotide periodicity in prokaryotic genomes from the GenBank-137 by using the ID method, we have found 454 sequences possessing the periodicity of such a type at level of <em>Z</em>-score ≥ 5. In order to find non-random sequences of maximal length it is essential to choose <em>Z</em>-score providing &lt;5% probability of finding a random sequence with <em>Z</em> greater than this threshold score. We found such a value by applying the Monte Carlo method to a random set of symbols with a length ∼10 times greater than the bacterial sequences presented in GenBank. So we can conclude that the number of ‘noisy’ sequences (i.e. the sequences that are not significantly periodic) in the set of the sequences found with <em>Z</em> ≥ 5 is &lt;5%. The distribution of the sequences found by the group of organisms they belong to is shown in <a reveal-id="tbl2" data-open="tbl2" class="link link-reveal link-table xref-fig">Table 2</a>. Rather a big number of sequences belongs to Gammaproteobacteria (158) and Actinobacteria (61). Together they account for more than a half of the periodical sequences. The organisms whose sequences include the biggest number of periodical sequences are as follows: <em>E. coli</em> (31), <em>Bradyrhizobium japonicum</em> (18), <em>Xanthomonas campestris</em> (17), <em>Helicobacter pylori</em> (16) and <em>Xanthomonas axonopodis</em> (16), from which the last three belong to the pathogenic bacteria and two others are symbiotic ones.</p><p>The length distribution of dinucleotide periodic sequences found in prokaryotic genomes is presented in <a reveal-id="fig6" data-open="fig6" class="link link-reveal link-table xref-fig">Fig. 6</a>. The length of most of the sequences (∼400) falls into the range 1–300, with the shortest one being equal to 28, but there are some examples of lengthy periodic sequences having length of more than 1000 nt.</p>                        <a id="87194463" scrollto-destination="87194463"></a>
<div id="fig6" data-id="fig6" class="fig fig-section"><div class="label fig-label"><strong>Figure 6</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/3/10.1093_dnares_dsl004/1/m_dsl004f6.jpeg?Expires=2147483647&Signature=bby3bwgfd2HDkoRqSqOiSCRJaoG1qPLRCMbVXw8lEaMfcjFIz5PiBhGr7wpsKYjjQSET-QYGgRsc3LWWWs3zZ6PjIXge8gBqoBjs8CpvJuu4GVr8oXZTtLfW8Uc5z1bU~ErespLhgOrbsmXtqTZgugMeWVQcMyRWc3hJsOkGkKpekM5CaEl3fz4hwik-VW~WcNdPlsxnefTJcikVL2BLPvVe9fsmLRx4NHz9frfpikuEMMDZIAD1ZUeUnjZMwVOoa-ro8wF2UFTk~dykGjFKne0U-hgmUlkvvcd9sBvFXu1k34auK8vsJMRQIrsB0J9zbMDLhxDkEw9b8HzZRSxyUA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Length distribution of dinucleotide periodic sequences found in prokaryotic genomes." path-from-xml="dsl004f6.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87194463/dsl004f6.jpeg" path-from-xml="dsl004f6.jpeg" target="_blank">View large</a><a section="87194463" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/3/10.1093_dnares_dsl004/1/dsl004f6.jpeg?Expires=2147483647&Signature=kYU2Bz07LfnUL44d3zy31UCbQTZzXN5Ub-18kExXeaQgD8v6od33riYQC2oLeDJlfdFPU-3Bpuz9OmvqrYPc7DkGTfe-wkGa4Njus~FaPRK~zbfooQhFacKCgrm33hlj8NK3OgAxXVKNNOmicdcphl0jYvFeIicmg65NkcjIoMUq~sRM6LxYZ4XnZCzs6FLMYl2tP18wRhE2dLGXu3WSKyc4B5oTKTc8eEneaMu-r7GpsT~kxwlWdNTx5~KaJr16g5-E54ukqv1UCm~bes4dsqxsAjxjk3hcCzlmj-OqFgI1~0dzH-NPJh3GtU0zVwgt~e0zn027N-MkNeatNacA9A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87194463&ar=347450&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl004f6.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Length distribution of dinucleotide periodic sequences found in prokaryotic genomes.</p></div></div><div content-id="fig6" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 6</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/3/10.1093_dnares_dsl004/1/m_dsl004f6.jpeg?Expires=2147483647&Signature=bby3bwgfd2HDkoRqSqOiSCRJaoG1qPLRCMbVXw8lEaMfcjFIz5PiBhGr7wpsKYjjQSET-QYGgRsc3LWWWs3zZ6PjIXge8gBqoBjs8CpvJuu4GVr8oXZTtLfW8Uc5z1bU~ErespLhgOrbsmXtqTZgugMeWVQcMyRWc3hJsOkGkKpekM5CaEl3fz4hwik-VW~WcNdPlsxnefTJcikVL2BLPvVe9fsmLRx4NHz9frfpikuEMMDZIAD1ZUeUnjZMwVOoa-ro8wF2UFTk~dykGjFKne0U-hgmUlkvvcd9sBvFXu1k34auK8vsJMRQIrsB0J9zbMDLhxDkEw9b8HzZRSxyUA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Length distribution of dinucleotide periodic sequences found in prokaryotic genomes." path-from-xml="dsl004f6.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87194463/dsl004f6.jpeg" path-from-xml="dsl004f6.jpeg" target="_blank">View large</a><a section="87194463" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/3/10.1093_dnares_dsl004/1/dsl004f6.jpeg?Expires=2147483647&Signature=kYU2Bz07LfnUL44d3zy31UCbQTZzXN5Ub-18kExXeaQgD8v6od33riYQC2oLeDJlfdFPU-3Bpuz9OmvqrYPc7DkGTfe-wkGa4Njus~FaPRK~zbfooQhFacKCgrm33hlj8NK3OgAxXVKNNOmicdcphl0jYvFeIicmg65NkcjIoMUq~sRM6LxYZ4XnZCzs6FLMYl2tP18wRhE2dLGXu3WSKyc4B5oTKTc8eEneaMu-r7GpsT~kxwlWdNTx5~KaJr16g5-E54ukqv1UCm~bes4dsqxsAjxjk3hcCzlmj-OqFgI1~0dzH-NPJh3GtU0zVwgt~e0zn027N-MkNeatNacA9A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87194463&ar=347450&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl004f6.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Length distribution of dinucleotide periodic sequences found in prokaryotic genomes.</p></div></div><p>All 454 sequences were classified by the latent period type as it was described above in Section 2. As a result, 45 classes have been discriminated; each of them combined three or more sequences of dinucleotide periodicity. The total number of sequences belonging to classes was 219. The distribution of the number of sequences contained in the classes is shown in <a reveal-id="tbl3" data-open="tbl3" class="link link-reveal link-table xref-fig">Table 3</a>.</p>                        <a id="87194465" scrollto-destination="87194465"></a>
<div content-id="tbl3" class="table-modal"><div class="table-wrap"><div id="tbl3" class="table-wrap-title" data-id="tbl3"><div class="label -label"><strong>Table 3</strong></div><div class="caption"><p>Distribution of the number of sequences in the classes obtained</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Number of sequences </th><th>Number of classes containing the given number of sequences </th></tr></thead><tbody><tr><td>3 </td><td>22 </td></tr><tr><td>4 </td><td>5 </td></tr><tr><td>5 </td><td>5 </td></tr><tr><td>6 </td><td>5 </td></tr><tr><td>7 </td><td>2 </td></tr><tr><td>8 </td><td>2 </td></tr><tr><td>9 </td><td>2 </td></tr><tr><td>11 </td><td>1 </td></tr><tr><td>19 </td><td>1 </td></tr></tbody></table></div><div content-id="tbl3" class="table-modal"><table><thead name="thead" align=""><tr><th>Number of sequences </th><th>Number of classes containing the given number of sequences </th></tr></thead><tbody><tr><td>3 </td><td>22 </td></tr><tr><td>4 </td><td>5 </td></tr><tr><td>5 </td><td>5 </td></tr><tr><td>6 </td><td>5 </td></tr><tr><td>7 </td><td>2 </td></tr><tr><td>8 </td><td>2 </td></tr><tr><td>9 </td><td>2 </td></tr><tr><td>11 </td><td>1 </td></tr><tr><td>19 </td><td>1 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87194465" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="tbl3" class="table-wrap-title" data-id="tbl3"><div class="label -label"><strong>Table 3</strong></div><div class="caption"><p>Distribution of the number of sequences in the classes obtained</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Number of sequences </th><th>Number of classes containing the given number of sequences </th></tr></thead><tbody><tr><td>3 </td><td>22 </td></tr><tr><td>4 </td><td>5 </td></tr><tr><td>5 </td><td>5 </td></tr><tr><td>6 </td><td>5 </td></tr><tr><td>7 </td><td>2 </td></tr><tr><td>8 </td><td>2 </td></tr><tr><td>9 </td><td>2 </td></tr><tr><td>11 </td><td>1 </td></tr><tr><td>19 </td><td>1 </td></tr></tbody></table></div><div content-id="tbl3" class="table-modal"><table><thead name="thead" align=""><tr><th>Number of sequences </th><th>Number of classes containing the given number of sequences </th></tr></thead><tbody><tr><td>3 </td><td>22 </td></tr><tr><td>4 </td><td>5 </td></tr><tr><td>5 </td><td>5 </td></tr><tr><td>6 </td><td>5 </td></tr><tr><td>7 </td><td>2 </td></tr><tr><td>8 </td><td>2 </td></tr><tr><td>9 </td><td>2 </td></tr><tr><td>11 </td><td>1 </td></tr><tr><td>19 </td><td>1 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87194465" target="_blank">View Large</a></div></div><p>We see that more than a half of the sequences that form the classes fall into two large classes, so the type of periodicity represented by the latter is quite common.</p><p>The tree diagram of class similarity is shown in <a reveal-id="fig5" data-open="fig5" class="link link-reveal link-table xref-fig">Fig. 5</a>.</p><p>The largest class combined 19 loci of latent dinucleotide periodicity (m × 10 on the tree diagram), next two largest classes contained 11 loci each (m × 32 and m × 38 on the tree diagram). The latent period type of the largest class is shown in <a reveal-id="fig7" data-open="fig7" class="link link-reveal link-table xref-fig">Fig. 7</a>. As we can see, cytosine and guanine are clearly predominated in both positions. Thus the period consensus may be conventionally described as {c,g}{c,g}. Classes of latent dinucleotide periodicity shown in <a reveal-id="fig5" data-open="fig5" class="link link-reveal link-table xref-fig">Fig. 5</a> are combined according to the similarity in period type. Let us consider, for example, two extreme left and right groups of classes. The first of them combines the classes m × 45, m × 14, m × 5, m × 31, m × 19, m × 4, and the second combines m × 38, m × 10, m × 13, m × 11, m × 35, m × 1. The aggregation of classes in the first group takes place due to the significant frequency of adenine appearance in the first period position. The conventional consensus of combination is {}{<em>n</em>}, where <em>n</em> – any nucleotide from the set (a,t,c,g). The aggregation process in the second group is caused by the significant values of frequencies of cytosine and guanine in the first position of the period. The combination conventional consensus in this case is {}{<em>n</em>}.</p>                        <a id="87194469" scrollto-destination="87194469"></a>
<div id="fig7" data-id="fig7" class="fig fig-section"><div class="label fig-label"><strong>Figure 7</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/3/10.1093_dnares_dsl004/1/m_dsl004f7.jpeg?Expires=2147483647&Signature=IINoL3VZKhTKXnEQXN4~6z3taOuD8VNLw1FAEcPGWtkwFUU21gsosHVUT3tUQEMNO3SpDvrgK4JEF4tqTa7BFbonSZqzp31jAkmdYPhM2n3uQuRZ-dzq4gyZ7A2NsdxzRqHbl59Ii3T-Cb5r2D-3WmezU8~Ui9U-h9CoMYe2tcfuDBT4k9cMbMYHPNKolNg-HrxfbPlzJ-OtHGvHxW6MEH61VEHMrGgdv9p9JuV-vyi7nZKtqBtlcSQViyqIq2Qc2AytD4oi~Q1VB9ogBWsGiZVgBEgsSTi66rGyjnItWsC2leE21AVfh2-f~VrqgMK2B7r-bKCduojlEErfOalL8A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Non-normalized vector and normalized matrix of the latent periodicity class containing the biggest number of sequences which is equal to 19. Elements of the vector equal to the frequencies of appearance of a, t, c and g nucleotides in the first and second position of the period. The decimal numbers show the frequencies of corresponding nucleotides in period positions." path-from-xml="dsl004f7.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87194469/dsl004f7.jpeg" path-from-xml="dsl004f7.jpeg" target="_blank">View large</a><a section="87194469" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/3/10.1093_dnares_dsl004/1/dsl004f7.jpeg?Expires=2147483647&Signature=eDsl-48b1SV~pRCfF0cy95EsqAs1gUDUIuYpBqc4U8OnyxgR9XkVoR~zbbWNJ9a0LmdnyDUIOTZCxL2H5ij9-Ea4uaQu-2R1HuRGuYwtwPfUxF892jj~-SjEAd6wBjfUI4OC4thmpHY8CLC8oW67ntSn4FChPLR469gKq0qnWXlIedwgFbQ3ZxdZmp8N~FaeRhoHKjvzwDS50K614fbkIe56MDHtTQOGwRnOuMRQ-ZU2gsSOeA8iIq2eBtXHeGCHvl8VwrkDvaU8w8vrN2HOCGrei730DTVkufDbFWPRvYTdC8w3jF1FNL-97-b8fkWC9Jwsm3DD~O8ZlkU6pDFpVg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87194469&ar=347450&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl004f7.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Non-normalized vector and normalized matrix of the latent periodicity class containing the biggest number of sequences which is equal to 19. Elements of the vector equal to the frequencies of appearance of a, t, c and g nucleotides in the first and second position of the period. The decimal numbers show the frequencies of corresponding nucleotides in period positions.</p></div></div><div content-id="fig7" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 7</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/3/10.1093_dnares_dsl004/1/m_dsl004f7.jpeg?Expires=2147483647&Signature=IINoL3VZKhTKXnEQXN4~6z3taOuD8VNLw1FAEcPGWtkwFUU21gsosHVUT3tUQEMNO3SpDvrgK4JEF4tqTa7BFbonSZqzp31jAkmdYPhM2n3uQuRZ-dzq4gyZ7A2NsdxzRqHbl59Ii3T-Cb5r2D-3WmezU8~Ui9U-h9CoMYe2tcfuDBT4k9cMbMYHPNKolNg-HrxfbPlzJ-OtHGvHxW6MEH61VEHMrGgdv9p9JuV-vyi7nZKtqBtlcSQViyqIq2Qc2AytD4oi~Q1VB9ogBWsGiZVgBEgsSTi66rGyjnItWsC2leE21AVfh2-f~VrqgMK2B7r-bKCduojlEErfOalL8A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Non-normalized vector and normalized matrix of the latent periodicity class containing the biggest number of sequences which is equal to 19. Elements of the vector equal to the frequencies of appearance of a, t, c and g nucleotides in the first and second position of the period. The decimal numbers show the frequencies of corresponding nucleotides in period positions." path-from-xml="dsl004f7.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87194469/dsl004f7.jpeg" path-from-xml="dsl004f7.jpeg" target="_blank">View large</a><a section="87194469" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/3/10.1093_dnares_dsl004/1/dsl004f7.jpeg?Expires=2147483647&Signature=eDsl-48b1SV~pRCfF0cy95EsqAs1gUDUIuYpBqc4U8OnyxgR9XkVoR~zbbWNJ9a0LmdnyDUIOTZCxL2H5ij9-Ea4uaQu-2R1HuRGuYwtwPfUxF892jj~-SjEAd6wBjfUI4OC4thmpHY8CLC8oW67ntSn4FChPLR469gKq0qnWXlIedwgFbQ3ZxdZmp8N~FaeRhoHKjvzwDS50K614fbkIe56MDHtTQOGwRnOuMRQ-ZU2gsSOeA8iIq2eBtXHeGCHvl8VwrkDvaU8w8vrN2HOCGrei730DTVkufDbFWPRvYTdC8w3jF1FNL-97-b8fkWC9Jwsm3DD~O8ZlkU6pDFpVg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87194469&ar=347450&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl004f7.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Non-normalized vector and normalized matrix of the latent periodicity class containing the biggest number of sequences which is equal to 19. Elements of the vector equal to the frequencies of appearance of a, t, c and g nucleotides in the first and second position of the period. The decimal numbers show the frequencies of corresponding nucleotides in period positions.</p></div></div><p>The presence of lengthy regions of latent dinucleotide periodicity in prokaryotic genes allows one to think that many prokaryotic genes show the high variability in those regions which have little (if any) influence on the functionality of the protein being coded. The mechanism of microsatellites origin seems to be connected with DNA strand sliding and with mispairing of neighboring repeats at the time of replication.<a reveal-id="b33" data-open="b33" class="link link-ref link-reveal xref-bibr">33</a> Owing to the short cycle of prokaryotic organism reproduction, the number of dinucleotide repeats grows rapidly, which leads to the appearance of the lengthy tracts. The nucleotide mutation rate should be high on the dinucleotide periodicity tracts because of the lack of selective constraints, and thus the dinucleotide periodicity, erodes rapidly becoming the latent periodicity. In addition, the existence of lengthy dinucleotide periodicity tracts can facilitate the improvement of the genome's physical properties, e.g. the raise of its flexibility, or, conversely, the rigidity, for certain DNA regions. Thus, the latent dinucleotide periodicity could be stabilized under the influence of yet other forces of natural selection.</p><p>All the class matrices obtained are available online as Supplementary Table 1 at <a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org" target="_blank"></a>.</p>                            <h3 scrollto-destination=87194472 id="87194472" class="section-title" >3.2. Search with the MPA</h3>
<p>We made the search using the method of MPA for 45 classes obtained in the previous step. Each class matrix was used for scanning through all bacterial loci in GenBank. Totally 27 087 corresponding to the frequency matrices of periodicity classes have been found. Among them there were both sequences possessing dinucleotide periodicity and many sequences possessing 6 nt periods of similar type (3 periods of 2 nt each). The presence of 6 symbol periodicity is caused by triplet periodicity of coding DNA regions (2 periods of 3 symbols each make a 6 symbol periodicity). Therefore, we filtered the overlapped sequences as it was described in the Section 2 and then use the statistical test described in the Section 2.6 to find whether the sequences found possess the dinucleotide periodicity at significant level. The number of filtered sequences that have passed the test was 3949. Such a big number in comparison with the 454 sequences that were found originally gives a reason to suggest that these sequences consist of strongly diverged tandem duplications. The periodicity of these possible ancient microsatellites is fuzzy, so it cannot be found by ID only.</p><p>All sequences found were aligned against the consensus sequence corresponding to the periodicity matrix. Examples of the alignments obtained are shown in <a reveal-id="tbl4" data-open="tbl4" class="link link-reveal link-table xref-fig">Table 4</a>. The ID spectra for the sequences from <a reveal-id="tbl4" data-open="tbl4" class="link link-reveal link-table xref-fig">Table 4</a> are shown in <a reveal-id="fig8" data-open="fig8" class="link link-reveal link-table xref-fig">Fig. 8</a>. From this figure it is obvious that the maximal value corresponds to the period length equal to 2 and this value is &gt;7. So these sequences possess the dinucleotide periodicity at statistically significant level. None of these sequences were found by other software packages.<a reveal-id="b15" data-open="b15" class="link link-ref link-reveal xref-bibr">15</a>,<a reveal-id="b16" data-open="b16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="b34" data-open="b34" class="link link-ref link-reveal xref-bibr">34</a></p>                        <a id="87194475" scrollto-destination="87194475"></a>
<div id="fig8" data-id="fig8" class="fig fig-section"><div class="label fig-label"><strong>Figure 8</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/3/10.1093_dnares_dsl004/1/m_dsl004f8.jpeg?Expires=2147483647&Signature=F4BH2FdYWZFqgFpsRS2CUC7Q1igfAoPNG~7RYrlF~bj1jli4jTlKsDC2Y8eJMyw1MO0XwLaS0TdpN~ROaJh4yYsy4zeIHqRc-dM-Z-tJ3gG-IQPuNPZEVt9FWAaidXQX52hm4Be4xojmsC-8NkAaBLS9LpJh~XLRpI1HkOYHbPV2ljJbOukLBO-4125EJVcu5Gvl29PsLc1yvcUalKyruECBngjCwaPQn4w3x4DvX4MrCTQCfvmvzwULH1sVhD-BDUA4YTec68UklLsuwHfp8HOVd2~YlJYGuL5Pb442hMJ4Sw-~L4Fq1HNzBxi-jOfmB0fIm-~MvyzYcsiu~LbQXg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Information decomposition spectra for AE012486 (a); AF453480 (b); CJ11168X1, positions 176412–176535 (c); and CJ11168X1, positions 165728–165785 (d). The maximum at period length equal to 2 is clearly visible." path-from-xml="dsl004f8.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87194475/dsl004f8.jpeg" path-from-xml="dsl004f8.jpeg" target="_blank">View large</a><a section="87194475" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/3/10.1093_dnares_dsl004/1/dsl004f8.jpeg?Expires=2147483647&Signature=YKVTaqgNfs-TrKxoc5Z9JneRpECRtaX8JGe9Tm1aKT85ekwhKd9dkCMU1XjgPlRFzK186Tdah4jffPFqrCdHm90npojLPUx4mpE~XyMI5loUfObYi0A3RviwicJn~z5JLYUD7ZsBtARmy5rPiXCsOO40Y2hj7s8aNJmgxLFWzU9M~zDUtR~J9cc6sRi52fzqv9w6CkfRkFa9DI0RV3S9Bnjlernpv~tU6gxaiei~1XSWAY53zXddbvDtG6DgEs4-txAfmp3n7upjeEbEjk3iuC4yON9EVO4rMcPBIiqfLEjeIvEcl2jtcrXPdI~jmch7uZmhZ8~dYHwoPrMQcRxcWA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87194475&ar=347450&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl004f8.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Information decomposition spectra for AE012486 (<strong>a</strong>); AF453480 (<strong>b</strong>); CJ11168X1, positions 176412–176535 (<strong>c</strong>); and CJ11168X1, positions 165728–165785 (<strong>d</strong>). The maximum at period length equal to 2 is clearly visible.</p></div></div><div content-id="fig8" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 8</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/3/10.1093_dnares_dsl004/1/m_dsl004f8.jpeg?Expires=2147483647&Signature=F4BH2FdYWZFqgFpsRS2CUC7Q1igfAoPNG~7RYrlF~bj1jli4jTlKsDC2Y8eJMyw1MO0XwLaS0TdpN~ROaJh4yYsy4zeIHqRc-dM-Z-tJ3gG-IQPuNPZEVt9FWAaidXQX52hm4Be4xojmsC-8NkAaBLS9LpJh~XLRpI1HkOYHbPV2ljJbOukLBO-4125EJVcu5Gvl29PsLc1yvcUalKyruECBngjCwaPQn4w3x4DvX4MrCTQCfvmvzwULH1sVhD-BDUA4YTec68UklLsuwHfp8HOVd2~YlJYGuL5Pb442hMJ4Sw-~L4Fq1HNzBxi-jOfmB0fIm-~MvyzYcsiu~LbQXg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Information decomposition spectra for AE012486 (a); AF453480 (b); CJ11168X1, positions 176412–176535 (c); and CJ11168X1, positions 165728–165785 (d). The maximum at period length equal to 2 is clearly visible." path-from-xml="dsl004f8.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87194475/dsl004f8.jpeg" path-from-xml="dsl004f8.jpeg" target="_blank">View large</a><a section="87194475" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/3/10.1093_dnares_dsl004/1/dsl004f8.jpeg?Expires=2147483647&Signature=YKVTaqgNfs-TrKxoc5Z9JneRpECRtaX8JGe9Tm1aKT85ekwhKd9dkCMU1XjgPlRFzK186Tdah4jffPFqrCdHm90npojLPUx4mpE~XyMI5loUfObYi0A3RviwicJn~z5JLYUD7ZsBtARmy5rPiXCsOO40Y2hj7s8aNJmgxLFWzU9M~zDUtR~J9cc6sRi52fzqv9w6CkfRkFa9DI0RV3S9Bnjlernpv~tU6gxaiei~1XSWAY53zXddbvDtG6DgEs4-txAfmp3n7upjeEbEjk3iuC4yON9EVO4rMcPBIiqfLEjeIvEcl2jtcrXPdI~jmch7uZmhZ8~dYHwoPrMQcRxcWA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87194475&ar=347450&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl004f8.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Information decomposition spectra for AE012486 (<strong>a</strong>); AF453480 (<strong>b</strong>); CJ11168X1, positions 176412–176535 (<strong>c</strong>); and CJ11168X1, positions 165728–165785 (<strong>d</strong>). The maximum at period length equal to 2 is clearly visible.</p></div></div>                        <a id="87194476" scrollto-destination="87194476"></a>
<div content-id="tbl4" class="table-modal"><div class="table-wrap"><div id="tbl4" class="table-wrap-title" data-id="tbl4"><div class="label -label"><strong>Table 4</strong></div><div class="caption"><p>Examples of alignments of periodic sequences found</p></div> </div><div class="fig-graphic"><a class="jumplink-placeholder" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/3/10.1093_dnares_dsl004/1/dsl004t1.jpeg?Expires=2147483647&Signature=uLjTdRdhLYAaFbHNLeGA~0QHpKEIjJCGhXOEgkjq9pUWMuJ-jpmcPB399mkrqCweKh0NIbUAAlGZEDkD8n4JQp-QZnMx7vtU4UFL63VliMXlv5k~95FmjGnqXvqQTWRQdXhST9aJV7kKEvkPZpLD7QoL7eWkgEijhxZkv~NaqatLCo5kb0hUvbu~xNuF7hThO1pZptGYRjtIv3EnFWKYTdwiRG5UShb7aws~h56ZV9nFUZO-JFNp29zXRcz-s~ByBGqUUKSNtckEwJo2HIDf4un4i6dSqBieqG4GIqAwNYiv2K4IJ0hBx7-JY7iBNWJvH4ulP8EZVHkYIUsunG2rfw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" id="dsl004t1.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/3/10.1093_dnares_dsl004/1/m_dsl004t1.jpeg?Expires=2147483647&Signature=zV7e6HlkQ7QM~ZeeLIU5nuzD9DLrAML08DxcIqzerboQ2uk1pCkjzDS~0EHVHFVzMlmvXJDNTBGVXuei05f6h~jb1c7w8Z66rklN~D7z0dM8j8FR5XuXVm0CY9u4vwWXkZ8Bb1UMfDRs5IFeWcl3gyS1tIUoeNRIpSVh7L8Er9yHpi1CoaIPLUosuZFnjETD5zrl8VRGOb-CN5m8mE36u1jTlx5syWLLazWtTHaASm~osgfYtO0KeOr~xx-o6vw6x2Vu5aWgMNJBtFahkOR6T4tSwgWmclIGmAc~ykJAWUPE5VYHWWG2WYohtsj6~JERtykXxhh3eznKTj8HBmB1KQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Examples of alignments of periodic sequences found" path-from-xml="dsl004t1.jpeg" /></a></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87194476" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="tbl4" class="table-wrap-title" data-id="tbl4"><div class="label -label"><strong>Table 4</strong></div><div class="caption"><p>Examples of alignments of periodic sequences found</p></div> </div><div class="fig-graphic"><a class="jumplink-placeholder" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/3/10.1093_dnares_dsl004/1/dsl004t1.jpeg?Expires=2147483647&Signature=uLjTdRdhLYAaFbHNLeGA~0QHpKEIjJCGhXOEgkjq9pUWMuJ-jpmcPB399mkrqCweKh0NIbUAAlGZEDkD8n4JQp-QZnMx7vtU4UFL63VliMXlv5k~95FmjGnqXvqQTWRQdXhST9aJV7kKEvkPZpLD7QoL7eWkgEijhxZkv~NaqatLCo5kb0hUvbu~xNuF7hThO1pZptGYRjtIv3EnFWKYTdwiRG5UShb7aws~h56ZV9nFUZO-JFNp29zXRcz-s~ByBGqUUKSNtckEwJo2HIDf4un4i6dSqBieqG4GIqAwNYiv2K4IJ0hBx7-JY7iBNWJvH4ulP8EZVHkYIUsunG2rfw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" id="dsl004t1.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/3/10.1093_dnares_dsl004/1/m_dsl004t1.jpeg?Expires=2147483647&Signature=zV7e6HlkQ7QM~ZeeLIU5nuzD9DLrAML08DxcIqzerboQ2uk1pCkjzDS~0EHVHFVzMlmvXJDNTBGVXuei05f6h~jb1c7w8Z66rklN~D7z0dM8j8FR5XuXVm0CY9u4vwWXkZ8Bb1UMfDRs5IFeWcl3gyS1tIUoeNRIpSVh7L8Er9yHpi1CoaIPLUosuZFnjETD5zrl8VRGOb-CN5m8mE36u1jTlx5syWLLazWtTHaASm~osgfYtO0KeOr~xx-o6vw6x2Vu5aWgMNJBtFahkOR6T4tSwgWmclIGmAc~ykJAWUPE5VYHWWG2WYohtsj6~JERtykXxhh3eznKTj8HBmB1KQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Examples of alignments of periodic sequences found" path-from-xml="dsl004t1.jpeg" /></a></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87194476" target="_blank">View Large</a></div></div><p>To get some biological insight, it is interesting to explore which functional elements of DNA contain periodicity and, inversely, which periodical sequences belong to which functional elements. We used the GenBank FEATURES field as a source of information for our study. Only the sequences with high statistical significance of their periodicity (<em>Z</em> ≥ 7.0) were selected. The sequence was considered to belong to some functional element if its overlap with the region of corresponding feature was equal or greater than 30%.</p><p>The distribution of the number of periodical sequences overlapping with some functional elements of genome is shown in <a reveal-id="tbl5" data-open="tbl5" class="link link-reveal link-table xref-fig">Table 5</a>.</p>                        <a id="87194479" scrollto-destination="87194479"></a>
<div content-id="tbl5" class="table-modal"><div class="table-wrap"><div id="tbl5" class="table-wrap-title" data-id="tbl5"><div class="label -label"><strong>Table 5</strong></div><div class="caption"><p>Distribution of some periodical sequences revealed in previously characterized DNA regions</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>GenBank feature </th><th>Number of sequences overlapping with the feature region </th></tr></thead><tbody><tr><td>3′-UTR </td><td>1 </td></tr><tr><td>mRNA </td><td>10 </td></tr><tr><td>Promoter </td><td>14 </td></tr><tr><td>rep_origin </td><td>26 </td></tr><tr><td>repeat_region </td><td>115 </td></tr><tr><td>repeat_unit </td><td>7 </td></tr><tr><td>rRNA </td><td>51 </td></tr><tr><td>sig_peptide </td><td>2 </td></tr><tr><td>stem_loop </td><td>2 </td></tr><tr><td>tRNA </td><td>60 </td></tr></tbody></table></div><div content-id="tbl5" class="table-modal"><table><thead name="thead" align=""><tr><th>GenBank feature </th><th>Number of sequences overlapping with the feature region </th></tr></thead><tbody><tr><td>3′-UTR </td><td>1 </td></tr><tr><td>mRNA </td><td>10 </td></tr><tr><td>Promoter </td><td>14 </td></tr><tr><td>rep_origin </td><td>26 </td></tr><tr><td>repeat_region </td><td>115 </td></tr><tr><td>repeat_unit </td><td>7 </td></tr><tr><td>rRNA </td><td>51 </td></tr><tr><td>sig_peptide </td><td>2 </td></tr><tr><td>stem_loop </td><td>2 </td></tr><tr><td>tRNA </td><td>60 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87194479" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="tbl5" class="table-wrap-title" data-id="tbl5"><div class="label -label"><strong>Table 5</strong></div><div class="caption"><p>Distribution of some periodical sequences revealed in previously characterized DNA regions</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>GenBank feature </th><th>Number of sequences overlapping with the feature region </th></tr></thead><tbody><tr><td>3′-UTR </td><td>1 </td></tr><tr><td>mRNA </td><td>10 </td></tr><tr><td>Promoter </td><td>14 </td></tr><tr><td>rep_origin </td><td>26 </td></tr><tr><td>repeat_region </td><td>115 </td></tr><tr><td>repeat_unit </td><td>7 </td></tr><tr><td>rRNA </td><td>51 </td></tr><tr><td>sig_peptide </td><td>2 </td></tr><tr><td>stem_loop </td><td>2 </td></tr><tr><td>tRNA </td><td>60 </td></tr></tbody></table></div><div content-id="tbl5" class="table-modal"><table><thead name="thead" align=""><tr><th>GenBank feature </th><th>Number of sequences overlapping with the feature region </th></tr></thead><tbody><tr><td>3′-UTR </td><td>1 </td></tr><tr><td>mRNA </td><td>10 </td></tr><tr><td>Promoter </td><td>14 </td></tr><tr><td>rep_origin </td><td>26 </td></tr><tr><td>repeat_region </td><td>115 </td></tr><tr><td>repeat_unit </td><td>7 </td></tr><tr><td>rRNA </td><td>51 </td></tr><tr><td>sig_peptide </td><td>2 </td></tr><tr><td>stem_loop </td><td>2 </td></tr><tr><td>tRNA </td><td>60 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87194479" target="_blank">View Large</a></div></div><p>Since the sequences of interest were prokaryotic, it is not surprising that most of them belong to the gene regions, so these data are not shown in the <a reveal-id="tbl5" data-open="tbl5" class="link link-reveal link-table xref-fig">Table 5</a>. Also one can see that more than 100 sequences overlapped with sequence repeats were already detected empirically, so our method is proven to find such kind of repeats. What is really interesting is that periodical sequences were also found in promoters. The expected value for the number of promoters in the periodic sequences found is 32. It was calculated as a product of number of promoters in all bacterial genomes available and the fraction of periodic sequences in these genomes. Since the value obtained is 14, the abundance of promoters in periodic sequences is evidently not statistically significant. Nevertheless, the study of the periodic sequences found in promoters can be interesting since these sequences could appear to be ancient minisatellites.</p>                            <h3 scrollto-destination=87194481 id="87194481" class="section-title" >3.3. Comparison with related works and discussion</h3>
<p>A number of algorithms have already been proposed that either directly or indirectly detect tandem repeats. All of them suffer from significant limitations. One group of algorithms is based on computing the alignment matrices.<a reveal-id="b35  b37" data-open="b35  b37" class="link link-ref link-reveal xref-bibr">35–37</a> Their major drawback is excessive running time. But what is more important is that the methods that are based on using similarity matrices are able to identify only the repeats with high level of homology between repeat units. Similar phenomenon is described in ref. (<a reveal-id="b38" data-open="b38" class="link link-ref link-reveal xref-bibr">38</a>). The methods for searching the periodicity using Fourier transformation<a reveal-id="b39  b41" data-open="b39  b41" class="link link-ref link-reveal xref-bibr">39–41</a> are not able to identify the periodicity in presence of insertions and deletions in sequence and they do not produce the periodicity matrix that can be used for further analysis. Another group of algorithms finds tandem repeats indirectly using methods from the field of data compression. An algorithm by Milosavljevic and Jurka<a reveal-id="b42" data-open="b42" class="link link-ref link-reveal xref-bibr">42</a> detects ‘simple sequences’, i.e. mixtures of fragments that occur elsewhere. Simple sequences may or may not contain tandem repeats and this algorithm makes no attempt to deduce a repeated pattern. Some of the algorithms are sensitive dramatically to the insertions and deletions in sequence and, thus, can identify only the repeats obeying very strict rules.<a reveal-id="b43" data-open="b43" class="link link-ref link-reveal xref-bibr">43</a>,<a reveal-id="b44" data-open="b44" class="link link-ref link-reveal xref-bibr">44</a> The software programs for finding tandem repeats in genomic sequences available in the EMBOSS package<a reveal-id="b34" data-open="b34" class="link link-ref link-reveal xref-bibr">34</a> identify tandem repeats of a very limited type (certain microsatellites).</p><p>The major advantages of using the combination of our methods are that the size of the periodic sequence is not limited and is not specified beforehand; the methods are able to identify even very fuzzy repeats; the alignment matrices used are not filled entirely to speed the calculation. Though other methods developed share some of these advantages, to our knowledge, they are not able to identify the potential minisatellites found by using our approach.</p><p>The sequences shown in <a reveal-id="tbl4" data-open="tbl4" class="link link-reveal link-table xref-fig">Table 4</a> have not been found to be periodic or to be minisatellites by other programs and methods available. We found 3949 possible minisatellite sequences in bacterial genomes; this is remarkably greater than the number of sequences already found. Because of the statistical threshold defined, we can conclude that &gt;95% of the sequences found are periodic at statistically significant level.</p><p>As a first step, we use the ID method to find periodical sequences within bacterial genomes. Then we obtained classes of periodicity (defined by frequency matrix) to get some insight into the common patterns of periodicity for different organisms or group of organisms. The patterns, though rather fuzzy, do exist and we described this fact in details in the Results section. Since the repeats in bacterial genomes show high variability, most of them cannot be revealed by traditional methods and by the ID. MPA gives much better results, but it would take too much time to make a search for the periodicity with indels using all frequency matrices obtained using the ID.</p><p>If we take class matrices rather than all periodicity matrices, the time needed for the computations decreases dramatically. For bacterial genomes, using class matrices allows to speed up the calculation 10 times (45 class matrices versus 454 initial matrices). On the other hand, classes reflect more common properties of genome than individual matrices and, thus, can be used to reveal more periodical sequences. As we have shown in the Results section, some classes contain more than 10 sequences belonging to different organisms, so the genome properties they reflect are quite common. The number of sequences found by our method proves these suggestions.</p><p>Albeit the revealing of periodical sequences in genomes is an interesting and challenging task, the more important thing is to correlate the periodicity with possible biological functions and/or evolutional features. We have discussed above the difficulties in identification of ancient microsatellites and their importance for PCR analysis because of their highly polymorphic nature. In the Results section we showed some examples of periodical sequences found within the regions that had been already described as polymorphous. For each of these sequences we found the others defined by the same frequency matrix and, thus, possibly having the same nature. And it is possible to use the periodical sequences found with indels as a starting point for the PCR analysis, because they can turn out to be highly polymorphous ones. The study of possible ancient minisatellites may also be helpful for evolutionary analysis of genomes.</p>                            <h2 scrollto-destination=87194488 id="87194488" class="section-title" >4. Conclusion</h2>
<p>In this paper, we have presented a new method and algorithms for <em>de novo</em> identification of latent periodical sequences, which can be considered as potential microsatellites and minisatellites. A remarkable feature of this method is its ability to identify fuzzy or loose repeats (e.g. possible ancient microsatellites) that cannot be revealed by other methods.</p><p>The work presented in this paper was supported in part by the Russian Federal Ministry for Science and Education in the context of the ‘System Biology’ project (2005–2006).</p><p><strong>Supplementary Data:</strong> Supplementary data is available online at <a class="link link-uri" href="http://dnaresearch.oxfordjournals.org" target="_blank"></a>.</p>                            <h2 scrollto-destination=87194493 id="87194493" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="b1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wells</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Molecular basis of genetic instability of triplet repeats</div>, <div class="source">J. Biol. Chem.</div> , <div class="year">1996</div>, vol. <div class="volume">271</div> (pg. <div class="fpage">2875</div>-<div class="lpage">2878</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20basis%20of%20genetic%20instability%20of%20triplet%20repeats&amp;author=R.%20Wells&amp;publication_year=1996&amp;journal=J.%20Biol.%20Chem.&amp;volume=271&amp;pages=2875-2878" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.271.6.2875" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.271.6.2875" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.271.6.2875"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8621672" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Weitzmann</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Woodford</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Usdin</div> <div class="given-names">K.</div></div>. </div><div class="article-title">DNA secondary structures and the evolution of hypervariable tandem arrays</div>, <div class="source">J. Biol. Chem.</div> , <div class="year">1997</div>, vol. <div class="volume">272</div> (pg. <div class="fpage">9517</div>-<div class="lpage">9523</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20secondary%20structures%20and%20the%20evolution%20of%20hypervariable%20tandem%20arrays&amp;author=M.%20Weitzmann&amp;author=K.%20Woodford&amp;author=K.%20Usdin&amp;publication_year=1997&amp;journal=J.%20Biol.%20Chem.&amp;volume=272&amp;pages=9517-9523" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.272.14.9517" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.272.14.9517" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.272.14.9517"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9083093" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Richards</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Holman</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Yu</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Sutherland</div> <div class="given-names">G.</div></div>. </div><div class="article-title">Fragile X syndrome unstable element, p(CCG)n, and other simple tandem repeat sequences are binding sites for specific nuclear proteins</div>, <div class="source">Hum. Mol. Genet.</div> , <div class="year">1993</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">1429</div>-<div class="lpage">1435</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fragile%20X%20syndrome%20unstable%20element%2C%20p(CCG)n%2C%20and%20other%20simple%20tandem%20repeat%20sequences%20are%20binding%20sites%20for%20specific%20nuclear%20proteins&amp;author=R.%20Richards&amp;author=K.%20Holman&amp;author=S.%20Yu&amp;author=G.%20Sutherland&amp;publication_year=1993&amp;journal=Hum.%20Mol.%20Genet.&amp;volume=2&amp;pages=1429-1435" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/hmg/2.9.1429" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fhmg%2F2.9.1429" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fhmg%2F2.9.1429"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8242066" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lu</div> <div class="given-names">Q.</div></div>,  <div class="name"><div class="surname">Wallrath</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Granok</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Elgin</div> <div class="given-names">S.</div></div>. </div><div class="article-title">(CT)n (GA)n repeats and heat shock elements have distinct roles in chromatin structure and transcriptional activation of the Drosophila hsp26 gene</div>, <div class="source">Mol. Cell. Biol.</div> , <div class="year">1993</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">2802</div>-<div class="lpage">2814</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=(CT)n%20(GA)n%20repeats%20and%20heat%20shock%20elements%20have%20distinct%20roles%20in%20chromatin%20structure%20and%20transcriptional%20activation%20of%20the%20Drosophila%20hsp26%20gene&amp;author=Q.%20Lu&amp;author=L.%20Wallrath&amp;author=H.%20Granok&amp;author=S.%20Elgin&amp;publication_year=1993&amp;journal=Mol.%20Cell.%20Biol.&amp;volume=13&amp;pages=2802-2814" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/MCB.13.5.2802" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FMCB.13.5.2802" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FMCB.13.5.2802"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8474442" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Keim</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Price</div> <div class="given-names">L. B.</div></div>,  <div class="name"><div class="surname">Klevytska</div> <div class="given-names">A. M.</div></div>, et al. </div><div class="article-title">Multiple-locus variable-number tandem repeat analysis reveals genetic relationships within <em>Bacillus anthracis</em></div>, <div class="source">J. Bacteriol.</div> , <div class="year">2000</div>, vol. <div class="volume">182</div> (pg. <div class="fpage">2928</div>-<div class="lpage">2936</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Multiple-locus%20variable-number%20tandem%20repeat%20analysis%20reveals%20genetic%20relationships%20within%20Bacillus%20anthracis&amp;author=P.%20Keim&amp;author=L.%20B.%20Price&amp;author=A.%20M.%20Klevytska&amp;publication_year=2000&amp;journal=J.%20Bacteriol.&amp;volume=182&amp;pages=2928-2936" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.182.10.2928-2936.2000" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.182.10.2928-2936.2000" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.182.10.2928-2936.2000"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10781564" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Frothingham</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Meeker-O'Connell</div> <div class="given-names">W. A.</div></div>. </div><div class="article-title">Genetic diversity in the <em>Mycobacterium tuberculosis</em> complex based on variable numbers of tandem DNA repeats</div>, <div class="source">Microbiology</div> , <div class="year">1998</div>, vol. <div class="volume">144</div> (pg. <div class="fpage">1189</div>-<div class="lpage">1196</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20diversity%20in%20the%20Mycobacterium%20tuberculosis%20complex%20based%20on%20variable%20numbers%20of%20tandem%20DNA%20repeats&amp;author=R.%20Frothingham&amp;author=W.%20A.%20Meeker-O'Connell&amp;publication_year=1998&amp;journal=Microbiology&amp;volume=144&amp;pages=1189-1196" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1099/00221287-144-5-1189" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1099%2F00221287-144-5-1189" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1099%2F00221287-144-5-1189"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9611793" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Supply</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Mazars</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Lesjean</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Vincent</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Gicquel</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Locht</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Variable human minisatellite-like regions in the <em>Mycobacterium tuberculosis</em> genome</div>, <div class="source">Mol. Microbiol.</div> , <div class="year">2000</div>, vol. <div class="volume">36</div> (pg. <div class="fpage">762</div>-<div class="lpage">771</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Variable%20human%20minisatellite-like%20regions%20in%20the%20Mycobacterium%20tuberculosis%20genome&amp;author=P.%20Supply&amp;author=E.%20Mazars&amp;author=S.%20Lesjean&amp;author=V.%20Vincent&amp;author=B.%20Gicquel&amp;author=C.%20Locht&amp;publication_year=2000&amp;journal=Mol.%20Microbiol.&amp;volume=36&amp;pages=762-771" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-2958.2000.01905.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-2958.2000.01905.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-2958.2000.01905.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10844663" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Le Fleche</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Hauck</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Onteniente</div> <div class="given-names">L.</div></div>, et al. </div><div class="article-title">A tandem repeats database for bacterial genomes: application to the genotyping of <em>Yersinia pestis</em> and <em>Bacillus anthracis</em></div>, <div class="source">BMC Microbiology</div> , <div class="year">2001</div>, vol. <div class="volume">1</div> pg. <div class="fpage">2</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20tandem%20repeats%20database%20for%20bacterial%20genomes%3A%20application%20to%20the%20genotyping%20of%20Yersinia%20pestis%20and%20Bacillus%20anthracis&amp;author=P.%20Le%20Fleche&amp;author=Y.%20Hauck&amp;author=L.%20Onteniente&amp;publication_year=2001&amp;journal=BMC%20Microbiology&amp;volume=1&amp;pages=2" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2180-1-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2180-1-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2180-1-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11299044" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Toth</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Gaspari</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Jurka</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Microsatellites in different eukaryotic genomes: survey and analysis</div>, <div class="source">Genome Res.</div> , <div class="year">2000</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">967</div>-<div class="lpage">981</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microsatellites%20in%20different%20eukaryotic%20genomes%3A%20survey%20and%20analysis&amp;author=G.%20Toth&amp;author=Z.%20Gaspari&amp;author=J.%20Jurka&amp;publication_year=2000&amp;journal=Genome%20Res.&amp;volume=10&amp;pages=967-981" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.10.7.967" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.10.7.967" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.10.7.967"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10899146" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gur-Arie</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Cohen</div> <div class="given-names">C. J.</div></div>,  <div class="name"><div class="surname">Eitan</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Shelef</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Hallerman</div> <div class="given-names">E. M.</div></div>,  <div class="name"><div class="surname">and Kashi</div> <div class="given-names">Y.</div></div>. </div><div class="article-title">Simple sequence repeats in <em>Escherichia coli</em>: abundance, distribution, composition, and polymorphism</div>, <div class="source">Genome Res.</div> , <div class="year">2000</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">62</div>-<div class="lpage">71</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Simple%20sequence%20repeats%20in%20Escherichia%20coli%3A%20abundance%2C%20distribution%2C%20composition%2C%20and%20polymorphism&amp;author=R.%20Gur-Arie&amp;author=C.%20J.%20Cohen&amp;author=Y.%20Eitan&amp;author=L.%20Shelef&amp;author=E.%20M.%20Hallerman&amp;author=Y.%20and%20Kashi&amp;publication_year=2000&amp;journal=Genome%20Res.&amp;volume=10&amp;pages=62-71" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10645951" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dib</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Faure</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Fizames</div> <div class="given-names">C.</div></div>, et al. </div><div class="article-title">A comprehensive genetic map of the human genome based on 5,264 microsatellites</div>, <div class="source">Nature</div> , <div class="year">1996</div>, vol. <div class="volume">380</div> (pg. <div class="fpage">149</div>-<div class="lpage">152</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20comprehensive%20genetic%20map%20of%20the%20human%20genome%20based%20on%205%2C264%20microsatellites&amp;author=C.%20Dib&amp;author=S.%20Faure&amp;author=C.%20Fizames&amp;publication_year=1996&amp;journal=Nature&amp;volume=380&amp;pages=149-152" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/380152a0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F380152a0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F380152a0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8600386" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">van Belkum</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Scherer</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">van Leeuwen</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Willemse</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">van Alphen</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Verbrugh</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Variable number of tandem repeats in clinical strains of <em>Haemophilus influenzae</em></div>, <div class="source">Infect. Immun.</div> , <div class="year">1997</div>, vol. <div class="volume">65</div> (pg. <div class="fpage">5017</div>-<div class="lpage">5027</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Variable%20number%20of%20tandem%20repeats%20in%20clinical%20strains%20of%20Haemophilus%20influenzae&amp;author=A.%20van%20Belkum&amp;author=S.%20Scherer&amp;author=W.%20van%20Leeuwen&amp;author=D.%20Willemse&amp;author=L.%20van%20Alphen&amp;author=H.%20Verbrugh&amp;publication_year=1997&amp;journal=Infect.%20Immun.&amp;volume=65&amp;pages=5017-5027" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9393791" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Adair</div> <div class="given-names">D. M.</div></div>,  <div class="name"><div class="surname">Worsham</div> <div class="given-names">P. L.</div></div>,  <div class="name"><div class="surname">Hill</div> <div class="given-names">K. K.</div></div>, et al. </div><div class="article-title">Diversity in a variable-number tandem repeat from Yersinia pestis</div>, <div class="source">J. Clin. Microbiol.</div> , <div class="year">2000</div>, vol. <div class="volume">38</div> (pg. <div class="fpage">1516</div>-<div class="lpage">1519</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Diversity%20in%20a%20variable-number%20tandem%20repeat%20from%20Yersinia%20pestis&amp;author=D.%20M.%20Adair&amp;author=P.%20L.%20Worsham&amp;author=K.%20K.%20Hill&amp;publication_year=2000&amp;journal=J.%20Clin.%20Microbiol.&amp;volume=38&amp;pages=1516-1519" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10747136" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Benson</div> <div class="given-names">G.</div></div>. </div><div class="article-title">Tandem cyclic alignment, Proceedings of the 12th annual symposium on combinatorial pattern matching</div>, <div class="source">LNCS</div> , <div class="year">2001</div>, vol. <div class="volume">2089</div> (pg. <div class="fpage">118</div>-<div class="lpage">130</div>)</div></div></div></div><div content-id="b15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Benson</div> <div class="given-names">G.</div></div>. </div><div class="article-title">Tandem repeats finder: a program to analyze DNA sequences</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1999</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">573</div>-<div class="lpage">580</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Tandem%20repeats%20finder%3A%20a%20program%20to%20analyze%20DNA%20sequences&amp;author=G.%20Benson&amp;publication_year=1999&amp;journal=Nucleic%20Acids%20Res.&amp;volume=27&amp;pages=573-580" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/27.2.573" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F27.2.573" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F27.2.573"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9862982" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kolpakov</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Bana</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Kucherov</div> <div class="given-names">G.</div></div>. </div><div class="article-title">mreps: efficient and flexible detection of tandem repeats in DNA</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2003</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">3672</div>-<div class="lpage">3678</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=mreps%3A%20efficient%20and%20flexible%20detection%20of%20tandem%20repeats%20in%20DNA&amp;author=R.%20Kolpakov&amp;author=G.%20Bana&amp;author=G.%20Kucherov&amp;publication_year=2003&amp;journal=Nucleic%20Acids%20Res.&amp;volume=31&amp;pages=3672-3678" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkg617" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkg617" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkg617"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12824391" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ruitberg</div> <div class="given-names">C. M.</div></div>,  <div class="name"><div class="surname">Reeder</div> <div class="given-names">D. J.</div></div>,  <div class="name"><div class="surname">Butler</div> <div class="given-names">J. M.</div></div>. </div><div class="article-title">STRBase: a short tandem repeat DNA database for the human identity testing community</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2001</div>, vol. <div class="volume">29</div> (pg. <div class="fpage">320</div>-<div class="lpage">322</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=STRBase%3A%20a%20short%20tandem%20repeat%20DNA%20database%20for%20the%20human%20identity%20testing%20community&amp;author=C.%20M.%20Ruitberg&amp;author=D.%20J.%20Reeder&amp;author=J.%20M.%20Butler&amp;publication_year=2001&amp;journal=Nucleic%20Acids%20Res.&amp;volume=29&amp;pages=320-322" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/29.1.320" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F29.1.320" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F29.1.320"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11125125" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Boby</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Patch</div> <div class="given-names">A.-M.</div></div>,  <div class="name"><div class="surname">Aves</div> <div class="given-names">S. J.</div></div>. </div><div class="article-title">TRbase: a database relating tandem repeats to disease genes for the human genome</div>, <div class="source">Bioinformatics</div> , <div class="year">2005</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">811</div>-<div class="lpage">816</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=TRbase%3A%20a%20database%20relating%20tandem%20repeats%20to%20disease%20genes%20for%20the%20human%20genome&amp;author=T.%20Boby&amp;author=A.-M.%20Patch&amp;author=S.%20J.%20Aves&amp;publication_year=2005&amp;journal=Bioinformatics&amp;volume=21&amp;pages=811-816" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bti059" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbti059" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbti059"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15479712" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Korotkov</div> <div class="given-names">E. V.</div></div>,  <div class="name"><div class="surname">Korotkova</div> <div class="given-names">M. A.</div></div>,  <div class="name"><div class="surname">Kudryashov</div> <div class="given-names">N. A.</div></div>. </div><div class="article-title">Information decomposition method to analyze symbolical sequences</div>, <div class="source">Phys. Lett. A</div> , <div class="year">2003</div>, vol. <div class="volume">312</div> (pg. <div class="fpage">198</div>-<div class="lpage">210</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Information%20decomposition%20method%20to%20analyze%20symbolical%20sequences&amp;author=E.%20V.%20Korotkov&amp;author=M.%20A.%20Korotkova&amp;author=N.%20A.%20Kudryashov&amp;publication_year=2003&amp;journal=Phys.%20Lett.%20A&amp;volume=312&amp;pages=198-210" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0375-9601(03)00641-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0375-9601(03)00641-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0375-9601(03)00641-8"> </span></div> </div></div></div></div></div><div content-id="b20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Frenkel</div> <div class="given-names">F. E.</div></div>,  <div class="name"><div class="surname">Chaley</div> <div class="given-names">M. B.</div></div>,  <div class="name"><div class="surname">Korotkov</div> <div class="given-names">E. V.</div></div>,  <div class="name"><div class="surname">Skryabin</div> <div class="given-names">K. G.</div></div>. </div><div class="article-title">Evolution of tRNA-like sequences and genome variability</div>, <div class="source">Gene</div> , <div class="year">2004</div>, vol. <div class="volume">335</div> (pg. <div class="fpage">57</div>-<div class="lpage">71</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20of%20tRNA-like%20sequences%20and%20genome%20variability&amp;author=F.%20E.%20Frenkel&amp;author=M.%20B.%20Chaley&amp;author=E.%20V.%20Korotkov&amp;author=K.%20G.%20Skryabin&amp;publication_year=2004&amp;journal=Gene&amp;volume=335&amp;pages=57-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gene.2004.03.005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gene.2004.03.005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gene.2004.03.005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15194190" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Korotkov</div> <div class="given-names">E. V.</div></div>,  <div class="name"><div class="surname">Korotkova</div> <div class="given-names">M. A.</div></div>,  <div class="name"><div class="surname">Rudenko</div> <div class="given-names">V. M.</div></div>. </div><div class="article-title">MIR: family of repeats common for vertebrate genomes</div>, <div class="source">Mol. Biol. (Mosk)</div> , <div class="year">2000</div>, vol. <div class="volume">34</div> (pg. <div class="fpage">553</div>-<div class="lpage">559</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MIR%3A%20family%20of%20repeats%20common%20for%20vertebrate%20genomes&amp;author=E.%20V.%20Korotkov&amp;author=M.%20A.%20Korotkova&amp;author=V.%20M.%20Rudenko&amp;publication_year=2000&amp;journal=Mol.%20Biol.%20(Mosk)&amp;volume=34&amp;pages=553-559" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF02759556" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF02759556" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF02759556"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11042848" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dodin</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Vandergheynst</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Levoir</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Cordier</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Marcourt</div> <div class="given-names">L.</div></div>. </div><div class="article-title">Fourier and wavelet transform analysis, a tool for visualizing regular patterns in DNA sequences</div>, <div class="source">J. Theor. Biol.</div> , <div class="year">2000</div>, vol. <div class="volume">206</div> (pg. <div class="fpage">323</div>-<div class="lpage">326</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fourier%20and%20wavelet%20transform%20analysis%2C%20a%20tool%20for%20visualizing%20regular%20patterns%20in%20DNA%20sequences&amp;author=G.%20Dodin&amp;author=P.%20Vandergheynst&amp;author=P.%20Levoir&amp;author=C.%20Cordier&amp;author=L.%20Marcourt&amp;publication_year=2000&amp;journal=J.%20Theor.%20Biol.&amp;volume=206&amp;pages=323-326" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/jtbi.2000.2127" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fjtbi.2000.2127" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fjtbi.2000.2127"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10988018" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Korotkova</div> <div class="given-names">M. A.</div></div>,  <div class="name"><div class="surname">Korotkov</div> <div class="given-names">E. V.</div></div>,  <div class="name"><div class="surname">Rudenko</div> <div class="given-names">V. M.</div></div>. </div><div class="article-title">Latent periodicity of protein sequences</div>, <div class="source">J. Mol. Model.</div> , <div class="year">1999</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">103</div>-<div class="lpage">115</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Latent%20periodicity%20of%20protein%20sequences&amp;author=M.%20A.%20Korotkova&amp;author=E.%20V.%20Korotkov&amp;author=V.%20M.%20Rudenko&amp;publication_year=1999&amp;journal=J.%20Mol.%20Model.&amp;volume=5&amp;pages=103-115" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s008940050122" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs008940050122" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs008940050122"> </span></div> </div></div></div></div></div><div content-id="b24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Korotkov</div> <div class="given-names">E. V.</div></div>,  <div class="name"><div class="surname">Korotkova</div> <div class="given-names">M. A.</div></div>,  <div class="name"><div class="surname">Tulko</div> <div class="given-names">J. S.</div></div>. </div><div class="article-title">Latent sequence periodicity of some oncogenes and DNA-binding protein genes</div>, <div class="source">CABIOS</div> , <div class="year">1997</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">37</div>-<div class="lpage">44</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Latent%20sequence%20periodicity%20of%20some%20oncogenes%20and%20DNA-binding%20protein%20genes&amp;author=E.%20V.%20Korotkov&amp;author=M.%20A.%20Korotkova&amp;author=J.%20S.%20Tulko&amp;publication_year=1997&amp;journal=CABIOS&amp;volume=13&amp;pages=37-44" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9088707" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chaley</div> <div class="given-names">M. B.</div></div>,  <div class="name"><div class="surname">Korotkov</div> <div class="given-names">E. V.</div></div>,  <div class="name"><div class="surname">Skryabin</div> <div class="given-names">K. G.</div></div>. </div><div class="article-title">Method revealing latent periodicity of the nucleotide sequences modified for a case of small samples</div>, <div class="source">DNA Res.</div> , <div class="year">1999</div>, vol. <div class="volume">6</div> (pg. <div class="fpage">153</div>-<div class="lpage">163</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Method%20revealing%20latent%20periodicity%20of%20the%20nucleotide%20sequences%20modified%20for%20a%20case%20of%20small%20samples&amp;author=M.%20B.%20Chaley&amp;author=E.%20V.%20Korotkov&amp;author=K.%20G.%20Skryabin&amp;publication_year=1999&amp;journal=DNA%20Res.&amp;volume=6&amp;pages=153-163" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/6.3.153" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F6.3.153" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2F6.3.153"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10470846" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kullback</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Information Theory and Statistics</div>, <div class="year">1959</div><div class="publisher-loc">New York</div><div class="publisher-name">John Wiley &amp; Sons</div></div></div></div></div><div content-id="b27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ward</div> <div class="given-names">J. H.</div></div>. </div><div class="article-title">Hierarchical grouping to optimize an objective function</div>, <div class="source">J. Amer. Stat. Assoc.</div> , <div class="year">1963</div>, vol. <div class="volume">58</div> (pg. <div class="fpage">236</div>-<div class="lpage">244</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Hierarchical%20grouping%20to%20optimize%20an%20objective%20function&amp;author=J.%20H.%20Ward&amp;publication_year=1963&amp;journal=J.%20Amer.%20Stat.%20Assoc.&amp;volume=58&amp;pages=236-244" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1080/01621459.1963.10500845" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F01621459.1963.10500845" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1080%2F01621459.1963.10500845"> </span></div> </div></div></div></div></div><div content-id="b28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Waterman</div> <div class="given-names">M. S.</div></div>. </div><div class="article-title">Introduction to Computational Biology</div>, <div class="year">1995</div><div class="publisher-loc">London</div><div class="publisher-name">Map, Sequences and Genomes, Chapman &amp; Hall</div></div></div></div></div><div content-id="b29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gribskov</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">McLachlan</div> <div class="given-names">A. D.</div></div>,  <div class="name"><div class="surname">and Eisenberg</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Profile analysis: detection of distantly related proteins</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">1987</div>, vol. <div class="volume">84</div> (pg. <div class="fpage">4355</div>-<div class="lpage">4358</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Profile%20analysis%3A%20detection%20of%20distantly%20related%20proteins&amp;author=M.%20Gribskov&amp;author=A.%20D.%20McLachlan&amp;author=D.%20and%20Eisenberg&amp;publication_year=1987&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=84&amp;pages=4355-4358" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.84.13.4355" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.84.13.4355" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.84.13.4355"> </span></div> </div></div></div></div></div><div content-id="b30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sibbald</div> <div class="given-names">P. R.</div></div>,  <div class="name"><div class="surname">Argos</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Weighting aligned protein or nucleic acid sequences to correct for unequal representation</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">1990</div>, vol. <div class="volume">216</div> (pg. <div class="fpage">813</div>-<div class="lpage">818</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Weighting%20aligned%20protein%20or%20nucleic%20acid%20sequences%20to%20correct%20for%20unequal%20representation&amp;author=P.%20R.%20Sibbald&amp;author=P.%20Argos&amp;publication_year=1990&amp;journal=J.%20Mol.%20Biol.&amp;volume=216&amp;pages=813-818" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0022-2836(99)80003-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0022-2836(99)80003-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0022-2836(99)80003-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2176240" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Smith</div> <div class="given-names">T. F.</div></div>,  <div class="name"><div class="surname">Waterman</div> <div class="given-names">M. S.</div></div>. </div><div class="article-title">Identification of common molecular subsequences</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">1981</div>, vol. <div class="volume">147</div> (pg. <div class="fpage">195</div>-<div class="lpage">197</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20common%20molecular%20subsequences&amp;author=T.%20F.%20Smith&amp;author=M.%20S.%20Waterman&amp;publication_year=1981&amp;journal=J.%20Mol.%20Biol.&amp;volume=147&amp;pages=195-197" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0022-2836(81)90087-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0022-2836(81)90087-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0022-2836(81)90087-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7265238" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Webber</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Barton</div> <div class="given-names">G. J.</div></div>. </div><div class="article-title">Estimation of P-values for global alignments of protein sequences</div>, <div class="source">Bioinformatics</div> , <div class="year">2001</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">1158</div>-<div class="lpage">1167</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Estimation%20of%20P-values%20for%20global%20alignments%20of%20protein%20sequences&amp;author=C.%20Webber&amp;author=G.%20J.%20Barton&amp;publication_year=2001&amp;journal=Bioinformatics&amp;volume=17&amp;pages=1158-1167" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/17.12.1158" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2F17.12.1158" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2F17.12.1158"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11751224" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Coggins</div> <div class="given-names">L. W.</div></div>,  <div class="name"><div class="surname">O'Prey</div> <div class="given-names">M.</div></div>. </div><div class="article-title">DNA tertiary structures formed in vitro by misaligned hybridization of multiple tandem repeat sequences</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1989</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">7417</div>-<div class="lpage">7426</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20tertiary%20structures%20formed%20in%20vitro%20by%20misaligned%20hybridization%20of%20multiple%20tandem%20repeat%20sequences&amp;author=L.%20W.%20Coggins&amp;author=M.%20O'Prey&amp;publication_year=1989&amp;journal=Nucleic%20Acids%20Res.&amp;volume=17&amp;pages=7417-7426" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/17.18.7417" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F17.18.7417" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F17.18.7417"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2677997" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rice</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Longden</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Bleasby</div> <div class="given-names">A.</div></div>. </div><div class="article-title">EMBOSS: The European molecular biology open software suite</div>, <div class="source">Trends Genet.</div> , <div class="year">2000</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">276</div>-<div class="lpage">277</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=EMBOSS%3A%20The%20European%20molecular%20biology%20open%20software%20suite&amp;author=P.%20Rice&amp;author=I.%20Longden&amp;author=A.%20Bleasby&amp;publication_year=2000&amp;journal=Trends%20Genet.&amp;volume=16&amp;pages=276-277" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0168-9525(00)02024-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0168-9525(00)02024-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0168-9525(00)02024-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10827456" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kannan</div> <div class="given-names">S. K.</div></div>,  <div class="name"><div class="surname">Myers</div> <div class="given-names">E. W.</div></div>. </div><div class="article-title">An algorithm for locating nonoverlapping regions of maximum alignment score</div>, <div class="source">SIAM J. Comput.</div> , <div class="year">1996</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">648</div>-<div class="lpage">662</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20algorithm%20for%20locating%20nonoverlapping%20regions%20of%20maximum%20alignment%20score&amp;author=S.%20K.%20Kannan&amp;author=E.%20W.%20Myers&amp;publication_year=1996&amp;journal=SIAM%20J.%20Comput.&amp;volume=25&amp;pages=648-662" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1137/S0097539794262677" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1137%2FS0097539794262677" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1137%2FS0097539794262677"> </span></div> </div></div></div></div></div><div content-id="b36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Benson</div> <div class="given-names">G.</div></div>. </div><div class="article-title">A space efficient algorithm for finding the best nonoverlapping alignment score</div>, <div class="source">Theor. Comput. Sci.</div> , <div class="year">1995</div>, vol. <div class="volume">145</div> (pg. <div class="fpage">357</div>-<div class="lpage">369</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20space%20efficient%20algorithm%20for%20finding%20the%20best%20nonoverlapping%20alignment%20score&amp;author=G.%20Benson&amp;publication_year=1995&amp;journal=Theor.%20Comput.%20Sci.&amp;volume=145&amp;pages=357-369" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0304-3975(95)92848-R" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0304-3975(95)92848-R" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0304-3975(95)92848-R"> </span></div> </div></div></div></div></div><div content-id="b37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schmidt</div> <div class="given-names">J. P.</div></div>. </div><div class="article-title">All highest scoring paths in weighted grid graphs and their application to finding all approximate repeats in strings</div>, <div class="source">SIAM J. Comput.</div> , <div class="year">1998</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">972</div>-<div class="lpage">992</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=All%20highest%20scoring%20paths%20in%20weighted%20grid%20graphs%20and%20their%20application%20to%20finding%20all%20approximate%20repeats%20in%20strings&amp;author=J.%20P.%20Schmidt&amp;publication_year=1998&amp;journal=SIAM%20J.%20Comput.&amp;volume=27&amp;pages=972-992" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1137/S0097539795288489" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1137%2FS0097539795288489" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1137%2FS0097539795288489"> </span></div> </div></div></div></div></div><div content-id="b38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Laskin</div> <div class="given-names">A. A.</div></div>,  <div class="name"><div class="surname">Kudryashov</div> <div class="given-names">N. A.</div></div>,  <div class="name"><div class="surname">Skryabin</div> <div class="given-names">K. G.</div></div>,  <div class="name"><div class="surname">Korotkov</div> <div class="given-names">E. V.</div></div>. </div><div class="article-title">Latent periodicity of serine–threonine and tyrosine protein kinases and other protein families</div>, <div class="source">Comput. Biol. Chem.</div> , <div class="year">2005</div>, vol. <div class="volume">29</div> (pg. <div class="fpage">229</div>-<div class="lpage">243</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Latent%20periodicity%20of%20serine%E2%80%93threonine%20and%20tyrosine%20protein%20kinases%20and%20other%20protein%20families&amp;author=A.%20A.%20Laskin&amp;author=N.%20A.%20Kudryashov&amp;author=K.%20G.%20Skryabin&amp;author=E.%20V.%20Korotkov&amp;publication_year=2005&amp;journal=Comput.%20Biol.%20Chem.&amp;volume=29&amp;pages=229-243" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.compbiolchem.2005.04.003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.compbiolchem.2005.04.003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.compbiolchem.2005.04.003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15979043" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Issac</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Singh</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Kaur</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Raghava</div> <div class="given-names">G. P. S.</div></div>. </div><div class="article-title">Locating probable genes using fourier transform approach</div>, <div class="source">Bioinformatics</div> , <div class="year">2002</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">196</div>-<div class="lpage">197</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Locating%20probable%20genes%20using%20fourier%20transform%20approach&amp;author=B.%20Issac&amp;author=H.%20Singh&amp;author=H.%20Kaur&amp;author=G.%20P.%20S.%20Raghava&amp;publication_year=2002&amp;journal=Bioinformatics&amp;volume=18&amp;pages=196-197" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/18.1.196" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2F18.1.196" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2F18.1.196"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11836230" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chechetkin</div> <div class="given-names">V. R.</div></div>,  <div class="name"><div class="surname">Lobzin</div> <div class="given-names">V. V.</div></div>. </div><div class="article-title">Nucleosome units and hidden periodicities in DNA sequences</div>, <div class="source">J. Biomol. Struct. Dyn.</div> , <div class="year">1998</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">937</div>-<div class="lpage">947</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nucleosome%20units%20and%20hidden%20periodicities%20in%20DNA%20sequences&amp;author=V.%20R.%20Chechetkin&amp;author=V.%20V.%20Lobzin&amp;publication_year=1998&amp;journal=J.%20Biomol.%20Struct.%20Dyn.&amp;volume=15&amp;pages=937-947" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1080/07391102.1998.10508214" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F07391102.1998.10508214" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1080%2F07391102.1998.10508214"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9619515" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jackson</div> <div class="given-names">J. H.</div></div>,  <div class="name"><div class="surname">George</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Herring</div> <div class="given-names">P. A.</div></div>. </div><div class="article-title">Vectors of Shannon information from Fourier signals characterizing base periodicity in genes and genomes</div>, <div class="source">Biochem. Biophys. Res. Commun.</div> , <div class="year">2000</div>, vol. <div class="volume">268</div> (pg. <div class="fpage">289</div>-<div class="lpage">292</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Vectors%20of%20Shannon%20information%20from%20Fourier%20signals%20characterizing%20base%20periodicity%20in%20genes%20and%20genomes&amp;author=J.%20H.%20Jackson&amp;author=R.%20George&amp;author=P.%20A.%20Herring&amp;publication_year=2000&amp;journal=Biochem.%20Biophys.%20Res.%20Commun.&amp;volume=268&amp;pages=289-292" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/bbrc.2000.2112" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fbbrc.2000.2112" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fbbrc.2000.2112"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10679195" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Milosavljevic</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Jurka</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Discovering simple DNA sequences by the algorithmic significance method</div>, <div class="source">CABIOS</div> , <div class="year">1993</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">407</div>-<div class="lpage">411</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Discovering%20simple%20DNA%20sequences%20by%20the%20algorithmic%20significance%20method&amp;author=A.%20Milosavljevic&amp;author=J.%20Jurka&amp;publication_year=1993&amp;journal=CABIOS&amp;volume=9&amp;pages=407-411" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8402207" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Landau</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Schmidt</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Sokol</div> <div class="given-names">D.</div></div>. </div><div class="article-title">An algorithm for approximate tandem repeats</div>, <div class="source">J. Comp. Biol.</div> , <div class="year">2001</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">1</div>-<div class="lpage">18</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20algorithm%20for%20approximate%20tandem%20repeats&amp;author=G.%20Landau&amp;author=J.%20Schmidt&amp;author=D.%20Sokol&amp;publication_year=2001&amp;journal=J.%20Comp.%20Biol.&amp;volume=8&amp;pages=1-18" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1089/106652701300099038" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1089%2F106652701300099038" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1089%2F106652701300099038"> </span></div> </div></div></div></div></div><div content-id="b44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Subramanian</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Mishra</div> <div class="given-names">R. K.</div></div>,  <div class="name"><div class="surname">Singh</div> <div class="given-names">L.</div></div>. </div><div class="article-title">Genome-wide analysis of microsatellite repeats in humans: their abundance and density in specific genomic regions</div>, <div class="source">Genome Biol.</div> , <div class="year">2003</div>, vol. <div class="volume">4</div> pg. <div class="fpage">R13</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20analysis%20of%20microsatellite%20repeats%20in%20humans%3A%20their%20abundance%20and%20density%20in%20specific%20genomic%20regions&amp;author=S.%20Subramanian&amp;author=R.%20K.%20Mishra&amp;author=L.%20Singh&amp;publication_year=2003&amp;journal=Genome%20Biol.&amp;volume=4&amp;pages=R13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2003-4-2-r13" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2003-4-2-r13" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2003-4-2-r13"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12620123" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2006. Kazusa DNA Research Institute</div><div class="license"><div class="license-p">The online version of this article has been published under an open access model. Users are entitled to use, reproduce, disseminate, or display the open access version of this article for non-commercial purposes provided that: the original authorship is properly and fully attributed; the Journal and Oxford University Press are attributed as the original place of publication with the correct citation details given; if an article is subsequently reproduced or disseminated not in its entirety but only in part or as a derivative work this must be clearly indicated. For commercial re-use, please contact journals.permissions@oxfordjournals.org</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=347450&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/3/10.1093_dnares_dsl004/1/dsl004_Supplementary_Data.zip?Expires=2147483647&amp;Signature=uCWjbALvhjgOOLh-v7MhIk54H3Q1npvWRpLNeJairtqfF1w6Vhe1fKj7jSqi-0j~5VKNO70UqV5qi~noop1VRslmXebBxip0i9ryUEFHgf6AS2rX1Irf7pDWNDrBl2FrUgNFVm4Se5Ag1951izv19LRviepyr8zX5FMsjivV8BSOQzJI6~cr9iuGJnn9HX6ZIQQ1VX8VzTl7m1fjJMAX2H8vp5yQ2EJJvkpjfnEOpObS5ZJguUJf~9ncuh9-w-2-~7RnaE-aqJfsee4QZ-JCkSGTPSgR4pvS6qBtzzkq2~MwlI83dW-mBVGqm2G3I6qh0mQQm~C2yGd-wvHjYIrGyQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">122</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000241913100001&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">11</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsl004"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="347450"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000241913100001&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f13%2f3%2f89%2f347450">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000241913100001&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (11)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F13%2F3%2F89%2F347450" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/347450'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Search and Classification of Potential Minisatellite Sequences from Bacterial Genomes" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsl004");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f13%2f3%2f89%2f347450">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

