

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/12/4/257/621446","@type":"ScholarlyArticle","name":"Preparation of a Set of Expression-Ready Clones of Mammalian Long cDNAs Encoding Large Proteins by the ORF Trap Cloning Method","datePublished":"2005-01-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/12/4","@type":"PublicationIssue","issueNumber":"4","datePublished":"2005-01-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsi010","keywords":["large protein","cDNA","expression clone","proteomics","subcellular localization"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/12/4/257/621446","author":[{"name":"Nakajima, Daisuke","affiliation":"1  Department of Human Gene Research, Kazusa DNA Research Institute  2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan","@type":"Person"},{"name":"Saito, Kenji","affiliation":"2  Laboratory of Pharmacogenomics, Graduate School of Pharmaceutical Sciences, Chiba University  2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan","@type":"Person"},{"name":"Yamakawa, Hisashi","affiliation":"1  Department of Human Gene Research, Kazusa DNA Research Institute  2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan","@type":"Person"},{"name":"Kikuno, Reiko F.","affiliation":"1  Department of Human Gene Research, Kazusa DNA Research Institute  2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan","@type":"Person"},{"name":"Nakayama, Manabu","affiliation":"1  Department of Human Gene Research, Kazusa DNA Research Institute  2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan    2  Laboratory of Pharmacogenomics, Graduate School of Pharmaceutical Sciences, Chiba University  2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan","@type":"Person"},{"name":"Ohara, Reiko","affiliation":"1  Department of Human Gene Research, Kazusa DNA Research Institute  2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan","@type":"Person"},{"name":"Okazaki, Noriko","affiliation":"1  Department of Human Gene Research, Kazusa DNA Research Institute  2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan","@type":"Person"},{"name":"Koga, Hisashi","affiliation":"1  Department of Human Gene Research, Kazusa DNA Research Institute  2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan    3  Chiba Industry Advancement Center  2-6 Nakase, Mihama-ku, Chiba 261-7126, Japan","@type":"Person"},{"name":"Nagase, Takahiro","affiliation":"1  Department of Human Gene Research, Kazusa DNA Research Institute  2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan","@type":"Person"},{"name":"Ohara, Osamu","affiliation":"1  Department of Human Gene Research, Kazusa DNA Research Institute  2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan    2  Laboratory of Pharmacogenomics, Graduate School of Pharmaceutical Sciences, Chiba University  2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan    4  RIKEN Research Center for Allergy and Immunology  1-7-22 Suehiro, Tsurumi-ku, Yokohama, Kanagawa 230-0045, Japan","@type":"Person"}],"description":"Abstract.  Although we have so far identified and sequenced &gt;2000 human long cDNAs, known as KIAA cDNAs, half of them have yet to be functionally annotated. ","pageStart":"257","pageEnd":"267","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/12/4/0/m_dnaresearch12_4.cover.gif?Expires=2147483647&Signature=3uNJBvpswUl8ckNzhRD6LiG0MySGXqGkcTQYLlRIvfld1eLLpkZoI4Ma5qe4h0TQIrhjk1-qwhKlhwYLpQpMu3ntM421fXasS8~G0V0MwH6cjjfM9fdJn0EH2kkIRqpR9dzDT~TKvzm8K-R~6ztHb07HPr6KoeUx6Rgs8s~OeOw2mLTVUNaBcNQn8k14DbREAYbOxque3El9Lplnm2ngB~AQ0rwKokc~WvC44rQAaAltU0RG~GjoIdSMmocCDEXHDpbpBtxuUw6A9Y5q2U4JLWP3XC47XRtYoHhAyJWDFgU30hB9Q-HhZ4XJLURWeo2hPNBzP0pbMYzBduIAxMvdMQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Preparation of a Set of Expression-Ready Clones of Mammalian Long cDNAs Encoding Large Proteins by the ORF Trap Cloning Method","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/12/4/0/m_dnaresearch12_4.cover.gif?Expires=2147483647&Signature=3uNJBvpswUl8ckNzhRD6LiG0MySGXqGkcTQYLlRIvfld1eLLpkZoI4Ma5qe4h0TQIrhjk1-qwhKlhwYLpQpMu3ntM421fXasS8~G0V0MwH6cjjfM9fdJn0EH2kkIRqpR9dzDT~TKvzm8K-R~6ztHb07HPr6KoeUx6Rgs8s~OeOw2mLTVUNaBcNQn8k14DbREAYbOxque3El9Lplnm2ngB~AQ0rwKokc~WvC44rQAaAltU0RG~GjoIdSMmocCDEXHDpbpBtxuUw6A9Y5q2U4JLWP3XC47XRtYoHhAyJWDFgU30hB9Q-HhZ4XJLURWeo2hPNBzP0pbMYzBduIAxMvdMQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Preparation of a Set of Expression-Ready Clones of Mammalian Long cDNAs Encoding Large Proteins by the ORF Trap Cloning Method" />
    <meta property="og:description" content="Abstract.  Although we have so far identified and sequenced &amp;gt;2000 human long cDNAs, known as KIAA cDNAs, half of them have yet to be functionally annotated. " />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/12/4/257/621446" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2005-01-01" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Nakajima, Daisuke" /><meta name="citation_author_institution" content="1Department of Human Gene Research, Kazusa DNA Research Institute2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Saito, Kenji" /><meta name="citation_author_institution" content="2Laboratory of Pharmacogenomics, Graduate School of Pharmaceutical Sciences, Chiba University2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Yamakawa, Hisashi" /><meta name="citation_author_institution" content="1Department of Human Gene Research, Kazusa DNA Research Institute2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Kikuno, Reiko F." /><meta name="citation_author_institution" content="1Department of Human Gene Research, Kazusa DNA Research Institute2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Nakayama, Manabu" /><meta name="citation_author_institution" content="1Department of Human Gene Research, Kazusa DNA Research Institute2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author_institution" content="2Laboratory of Pharmacogenomics, Graduate School of Pharmaceutical Sciences, Chiba University2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Ohara, Reiko" /><meta name="citation_author_institution" content="1Department of Human Gene Research, Kazusa DNA Research Institute2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Okazaki, Noriko" /><meta name="citation_author_institution" content="1Department of Human Gene Research, Kazusa DNA Research Institute2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Koga, Hisashi" /><meta name="citation_author_institution" content="1Department of Human Gene Research, Kazusa DNA Research Institute2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author_institution" content="3Chiba Industry Advancement Center2-6 Nakase, Mihama-ku, Chiba 261-7126, Japan" /><meta name="citation_author" content="Nagase, Takahiro" /><meta name="citation_author_institution" content="1Department of Human Gene Research, Kazusa DNA Research Institute2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Ohara, Osamu" /><meta name="citation_author_institution" content="1Department of Human Gene Research, Kazusa DNA Research Institute2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author_institution" content="2Laboratory of Pharmacogenomics, Graduate School of Pharmaceutical Sciences, Chiba University2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author_institution" content="4RIKEN Research Center for Allergy and Immunology1-7-22 Suehiro, Tsurumi-ku, Yokohama, Kanagawa 230-0045, Japan" /><meta name="citation_title" content="Preparation of a Set of Expression-Ready Clones of Mammalian Long cDNAs Encoding Large Proteins by the ORF Trap Cloning Method" /><meta name="citation_firstpage" content="257" /><meta name="citation_lastpage" content="267" /><meta name="citation_doi" content="10.1093/dnares/dsi010" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="12" /><meta name="citation_issue" content="4" /><meta name="citation_publication_date" content="2005/01/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=The complete set of predicted genes from Saccharomyces cerevisiae in a readily usable form; Genome Res.;  citation_year=1997;  citation_volume=7;  citation_pages=1169-1173; " /><meta name="citation_reference" content="citation_title=Global analysis of protein activities using proteome chips; Science;  citation_year=2001;  citation_volume=293;  citation_pages=2101-2105; " /><meta name="citation_reference" content="citation_title=Systematic identification of protein complexes in Saccharomyces cerevisiae by mass spectrometry; Nature;  citation_year=2002;  citation_volume=415;  citation_pages=180-183; " /><meta name="citation_reference" content="citation_title=C. elegans ORFeome version 1.1: Experimental verification of the genome annotation and resource for proteome-scale protein expression; Nature Genet.;  citation_year=2003;  citation_volume=34;  citation_pages=35-41; " /><meta name="citation_reference" content="citation_title=Protein analysis on a proteomic scale; Nature;  citation_year=2003;  citation_volume=422;  citation_pages=208-215; " /><meta name="citation_reference" content="citation_title=Human ORFeome version 1.1: A platform for reverse proteomics; Genome Res.;  citation_year=2004;  citation_volume=14;  citation_pages=2128-2135; " /><meta name="citation_reference" content="citation_title=Protein expression clone repositories for functional proteomics; Curr. Opin. Chem. Biol.;  citation_year=2004;  citation_volume=8;  citation_pages=98-102; " /><meta name="citation_reference" content="citation_title=Systematic subcellular localization of novel proteins identified by large-scale cDNA sequencing; EMBO Rep.;  citation_year=2000;  citation_volume=3;  citation_pages=287-292; " /><meta name="citation_reference" content="citation_title=Protein–protein interaction panel using mouse full-length cDNAs; Genome Res.;  citation_year=2001;  citation_volume=11;  citation_pages=1758-1765; " /><meta name="citation_reference" content="citation_title=From ORFeome to Biology: A functional genomics pipeline; Genome Res.;  citation_year=2004;  citation_volume=14;  citation_pages=2136-2144; " /><meta name="citation_reference" content="citation_title=Prediction of the coding sequences of unidentified human genes. I. The coding sequences of 40 new genes (KIAA0001–KIAA0040) deduced by analysis of randomly sampled cDNA clones from human immature myeloid cell line KG-1; DNA Res.;  citation_year=1994;  citation_volume=1;  citation_pages=27-35; " /><meta name="citation_reference" content="citation_title=The mammalian gene collection; Science;  citation_year=1999;  citation_volume=286;  citation_pages=455-457; " /><meta name="citation_reference" content="citation_title=Toward a catalog of human genes and proteins: sequencing and analysis of 500 novel complete protein coding human cDNAs; Genome Res.;  citation_year=2001;  citation_volume=11;  citation_pages=422-435; " /><meta name="citation_reference" content="citation_title=Functional annotation of a full-length mouse cDNA collection; Nature;  citation_year=2001;  citation_volume=409;  citation_pages=685-690; " /><meta name="citation_reference" content="citation_title=Generation and initial analysis of more than 15,000 full-length human and mouse cDNA sequences; citation_author=Mammalian Gene Collection (MGC) Program Team; Proc. Natl Acad. Sci. USA;  citation_year=2002;  citation_volume=99;  citation_pages=16899-16903; " /><meta name="citation_reference" content="citation_title=Analysis of the mouse transcriptome based on functional annotation of 60,770 full-length cDNAs; Nature;  citation_year=2002;  citation_volume=420;  citation_pages=563-573; " /><meta name="citation_reference" content="citation_title=Prediction of the coding sequences of mouse homologues of KIAA gene: IV. The complete nucleotide sequences of 500 mouse KIAA-homologous cDNAs identified by screening of terminal sequences of cDNA clones randomly sampled from size-fractionated libraries; DNA Res.;  citation_year=2004;  citation_volume=11;  citation_pages=205-218; " /><meta name="citation_reference" content="citation_title=Complete sequencing and characterization of 21,243 full-length human cDNAs; Nature Genet.;  citation_year=2004;  citation_volume=36;  citation_pages=40-45; " /><meta name="citation_reference" content="citation_title=Construction and characterization of human brain cDNA libraries suitable for analysis of cDNA clones encoding relatively large proteins; DNA Res.;  citation_year=1997;  citation_volume=4;  citation_pages=53-59; " /><meta name="citation_reference" content="citation_title=The Kazusa cDNA project for identification of unknown human transcripts; C. R. Biol.;  citation_year=2003;  citation_volume=326;  citation_pages=959-966; " /><meta name="citation_reference" content="citation_title=Construction of expression-ready cDNA clones for KIAA genes: Manual curation of 330 KIAA cDNA clones; DNA Res.;  citation_year=2002;  citation_volume=9;  citation_pages=99-106; " /><meta name="citation_reference" content="citation_title=Prediction of the coding sequences of mouse homologues of KIAA gene: I. The complete nucleotide sequences of 100 mouse KIAA-homologous cDNAs identified by screening of terminal sequences of cDNA clones randomly sampled from size-fractionated libraries; DNA Res.;  citation_year=2002;  citation_volume=9;  citation_pages=179-188; " /><meta name="citation_reference" content="citation_title=High-throughput production of recombinant antigens for mouse KIAA proteins in Escherichia coli: computational allocation of plasmids of glutathione-S-transferase-fused antigens by an in vitro recombination-assisted method; DNA Res.;  citation_year=2003;  citation_volume=10;  citation_pages=129-136; " /><meta name="citation_reference" content="citation_title=A comprehensive approach for establishment of the platform to analyze functions of KIAA proteins: Generation and evaluation of anti-mKIAA antibodies; Proteomics;  citation_year=2004;  citation_volume=4;  citation_pages=1412-1416; " /><meta name="citation_reference" content="citation_title=HUGE: a database for human KIAA proteins, a 2004 update integrating HUGEppi and ROUGE; Nucleic Acids Res.;  citation_year=2004;  citation_volume=32;  citation_pages=D502-D504; " /><meta name="citation_reference" content="citation_title=The Gene Ontology Annotation (GOA) project: Implementation of GO in SWISS-PROT, TrEMBL, and InterPro; Genome Res.;  citation_year=2003;  citation_volume=13;  citation_pages=662-672; " /><meta name="citation_reference" content="citation_title=Genetic analysis of the RecE pathway of genetic recombination in Escherichia coli K-12; J. Bacteriol.;  citation_year=1981;  citation_volume=145;  citation_pages=521-532; " /><meta name="citation_reference" content="citation_title=In vivo cloning of PCR products in E. coli; Nucleic Acids Res.;  citation_year=1993;  citation_volume=21;  citation_pages=5192-5197; " /><meta name="citation_reference" content="citation_title=A new logic for DNA engineering using recombination in Escherichia coli; Nature Genet.;  citation_year=1998;  citation_volume=20;  citation_pages=123-128; " /><meta name="citation_reference" content="citation_title=GATEWAY recombinational cloning: application to the cloning of large numbers of open reading frames or ORFeomes; Meth. Enzymol.;  citation_year=2000;  citation_volume=328;  citation_pages=575-592; " /><meta name="citation_reference" content="citation_title=Techniques: Recombinogenic engineering—new options for cloning and manipulating DNA; Trends Biochem. Sci.;  citation_year=2001;  citation_volume=26;  citation_pages=325-331; " /><meta name="citation_reference" content="citation_title=Anomalous behavior of human leukocyte interferon subtypes on polyacrylamide gel electrophoresis in the presence of dodecyl sulfate; FEBS Lett.;  citation_year=1987;  citation_volume=211;  citation_pages=78-82; " /><meta name="citation_reference" content="citation_title=Interaction of peroxisomes with microtubules; Eur. J. Biochem.;  citation_year=2000;  citation_volume=267;  citation_pages=6264-6275; " /><meta name="citation_reference" content="citation_title=A comprehensive approach for establishment of the platform to analyze functions of KIAA proteins II: public release of inaugural version of InGaP database containing gene/protein expression profiles for 127 mouse KIAA genes/proteins; DNA Res.;  citation_year=2004;  citation_volume=11;  citation_pages=293-304; " /><meta name="citation_reference" content="citation_title=ORFeome cloning and systems biology: Standardized mass production of the parts from the parts-list; Genome Res.;  citation_year=2004;  citation_volume=14;  citation_pages=2001-2009; " /><meta name="citation_reference" content="citation_title=Oligo-capping: a simple method to replace the cap structure of eukaryotic mRNAs with oligoribonucleotides; Gene;  citation_year=1994;  citation_volume=138;  citation_pages=171-174; " /><meta name="citation_reference" content="citation_title=High-efficiency full-length cDNA cloning by biotinylated CAP trapper; Genomics;  citation_year=1996;  citation_volume=37;  citation_pages=327-336; " /><meta name="citation_reference" content="citation_title=Cap analysis gene expression for high-throughput analysis of transcriptional starting point and identification of promoter usage; Proc. Natl Acad. Sci. USA.;  citation_year=2003;  citation_volume=100;  citation_pages=15776-15781; " /><meta name="citation_reference" content="citation_title=5′-end SAGE for the analysis of transcriptional start sites; Nature Biotech.;  citation_year=2004;  citation_volume=22;  citation_pages=1146-1149; " /><meta name="citation_reference" content="citation_title=5′ Long serial analysis of gene expression (LongSAGE) and 3′LongSAGE for transcriptome characterization and genome annotation; Proc. Natl Acad. Sci.;  citation_year=2004;  citation_volume=101;  citation_pages=11701-11706; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/12/4/257/1719566/dsi010.pdf" /><meta name="description" content="Abstract.  Although we have so far identified and sequenced &gt;2000 human long cDNAs, known as KIAA cDNAs, half of them have yet to be functionally annotated. " />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/12/4/257/621446" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/12/4/257/621446" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/12/4/257/621446");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsi010';
    </script>
<title>
	Preparation of a Set of Expression-Ready Clones of Mammalian Long cDNAs Encoding Large Proteins by the ORF Trap Cloning Method | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./621446" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BZJuPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJOYWthamltYSwgRGFpc3VrZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IjFEZXBhcnRtZW50IG9mIEh1bWFuIEdlbmUgUmVzZWFyY2gsIEthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlMi02LTcgS2F6dXNhLUthbWF0YXJpLCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2FpdG8sIEtlbmppIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iMkxhYm9yYXRvcnkgb2YgUGhhcm1hY29nZW5vbWljcywgR3JhZHVhdGUgU2Nob29sIG9mIFBoYXJtYWNldXRpY2FsIFNjaWVuY2VzLCBDaGliYSBVbml2ZXJzaXR5Mi02LTcgS2F6dXNhLUthbWF0YXJpLCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iWWFtYWthd2EsIEhpc2FzaGkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSIxRGVwYXJ0bWVudCBvZiBIdW1hbiBHZW5lIFJlc2VhcmNoLCBLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZTItNi03IEthenVzYS1LYW1hdGFyaSwgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Iktpa3VubywgUmVpa28gRi4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSIxRGVwYXJ0bWVudCBvZiBIdW1hbiBHZW5lIFJlc2VhcmNoLCBLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZTItNi03IEthenVzYS1LYW1hdGFyaSwgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik5ha2F5YW1hLCBNYW5hYnUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSIxRGVwYXJ0bWVudCBvZiBIdW1hbiBHZW5lIFJlc2VhcmNoLCBLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZTItNi03IEthenVzYS1LYW1hdGFyaSwgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IjJMYWJvcmF0b3J5IG9mIFBoYXJtYWNvZ2Vub21pY3MsIEdyYWR1YXRlIFNjaG9vbCBvZiBQaGFybWFjZXV0aWNhbCBTY2llbmNlcywgQ2hpYmEgVW5pdmVyc2l0eTItNi03IEthenVzYS1LYW1hdGFyaSwgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik9oYXJhLCBSZWlrbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IjFEZXBhcnRtZW50IG9mIEh1bWFuIEdlbmUgUmVzZWFyY2gsIEthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlMi02LTcgS2F6dXNhLUthbWF0YXJpLCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iT2themFraSwgTm9yaWtvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iMURlcGFydG1lbnQgb2YgSHVtYW4gR2VuZSBSZXNlYXJjaCwgS2F6dXNhIEROQSBSZXNlYXJjaCBJbnN0aXR1dGUyLTYtNyBLYXp1c2EtS2FtYXRhcmksIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLb2dhLCBIaXNhc2hpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iMURlcGFydG1lbnQgb2YgSHVtYW4gR2VuZSBSZXNlYXJjaCwgS2F6dXNhIEROQSBSZXNlYXJjaCBJbnN0aXR1dGUyLTYtNyBLYXp1c2EtS2FtYXRhcmksIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSIzQ2hpYmEgSW5kdXN0cnkgQWR2YW5jZW1lbnQgQ2VudGVyMi02IE5ha2FzZSwgTWloYW1hLWt1LCBDaGliYSAyNjEtNzEyNiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJOYWdhc2UsIFRha2FoaXJvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iMURlcGFydG1lbnQgb2YgSHVtYW4gR2VuZSBSZXNlYXJjaCwgS2F6dXNhIEROQSBSZXNlYXJjaCBJbnN0aXR1dGUyLTYtNyBLYXp1c2EtS2FtYXRhcmksIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJPaGFyYSwgT3NhbXUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSIxRGVwYXJ0bWVudCBvZiBIdW1hbiBHZW5lIFJlc2VhcmNoLCBLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZTItNi03IEthenVzYS1LYW1hdGFyaSwgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IjJMYWJvcmF0b3J5IG9mIFBoYXJtYWNvZ2Vub21pY3MsIEdyYWR1YXRlIFNjaG9vbCBvZiBQaGFybWFjZXV0aWNhbCBTY2llbmNlcywgQ2hpYmEgVW5pdmVyc2l0eTItNi03IEthenVzYS1LYW1hdGFyaSwgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IjRSSUtFTiBSZXNlYXJjaCBDZW50ZXIgZm9yIEFsbGVyZ3kgYW5kIEltbXVub2xvZ3kxLTctMjIgU3VlaGlybywgVHN1cnVtaS1rdSwgWW9rb2hhbWEsIEthbmFnYXdhIDIzMC0wMDQ1LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iUHJlcGFyYXRpb24gb2YgYSBTZXQgb2YgRXhwcmVzc2lvbi1SZWFkeSBDbG9uZXMgb2YgTWFtbWFsaWFuIExvbmcgY0ROQXMgRW5jb2RpbmcgTGFyZ2UgUHJvdGVpbnMgYnkgdGhlIE9SRiBUcmFwIENsb25pbmcgTWV0aG9kIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iMjU3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSIyNjciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc2kwMTAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMTIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAwNS8wMS8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBjb21wbGV0ZSBzZXQgb2YgcHJlZGljdGVkIGdlbmVzIGZyb20gU2FjY2hhcm9teWNlcyBjZXJldmlzaWFlIGluIGEgcmVhZGlseSB1c2FibGUgZm9ybTsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTExNjktMTE3MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2xvYmFsIGFuYWx5c2lzIG9mIHByb3RlaW4gYWN0aXZpdGllcyB1c2luZyBwcm90ZW9tZSBjaGlwczsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0yOTM7ICBjaXRhdGlvbl9wYWdlcz0yMTAxLTIxMDU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN5c3RlbWF0aWMgaWRlbnRpZmljYXRpb24gb2YgcHJvdGVpbiBjb21wbGV4ZXMgaW4gU2FjY2hhcm9teWNlcyBjZXJldmlzaWFlIGJ5IG1hc3Mgc3BlY3Ryb21ldHJ5OyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9NDE1OyAgY2l0YXRpb25fcGFnZXM9MTgwLTE4MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Qy4gZWxlZ2FucyBPUkZlb21lIHZlcnNpb24gMS4xOiBFeHBlcmltZW50YWwgdmVyaWZpY2F0aW9uIG9mIHRoZSBnZW5vbWUgYW5ub3RhdGlvbiBhbmQgcmVzb3VyY2UgZm9yIHByb3Rlb21lLXNjYWxlIHByb3RlaW4gZXhwcmVzc2lvbjsgTmF0dXJlIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0zNDsgIGNpdGF0aW9uX3BhZ2VzPTM1LTQxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qcm90ZWluIGFuYWx5c2lzIG9uIGEgcHJvdGVvbWljIHNjYWxlOyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9NDIyOyAgY2l0YXRpb25fcGFnZXM9MjA4LTIxNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SHVtYW4gT1JGZW9tZSB2ZXJzaW9uIDEuMTogQSBwbGF0Zm9ybSBmb3IgcmV2ZXJzZSBwcm90ZW9taWNzOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTIxMjgtMjEzNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJvdGVpbiBleHByZXNzaW9uIGNsb25lIHJlcG9zaXRvcmllcyBmb3IgZnVuY3Rpb25hbCBwcm90ZW9taWNzOyBDdXJyLiBPcGluLiBDaGVtLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9OTgtMTAyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TeXN0ZW1hdGljIHN1YmNlbGx1bGFyIGxvY2FsaXphdGlvbiBvZiBub3ZlbCBwcm90ZWlucyBpZGVudGlmaWVkIGJ5IGxhcmdlLXNjYWxlIGNETkEgc2VxdWVuY2luZzsgRU1CTyBSZXAuOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz0yODctMjkyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qcm90ZWlu4oCTcHJvdGVpbiBpbnRlcmFjdGlvbiBwYW5lbCB1c2luZyBtb3VzZSBmdWxsLWxlbmd0aCBjRE5BczsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz0xNzU4LTE3NjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZyb20gT1JGZW9tZSB0byBCaW9sb2d5OiBBIGZ1bmN0aW9uYWwgZ2Vub21pY3MgcGlwZWxpbmU7IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9MjEzNi0yMTQ0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QcmVkaWN0aW9uIG9mIHRoZSBjb2Rpbmcgc2VxdWVuY2VzIG9mIHVuaWRlbnRpZmllZCBodW1hbiBnZW5lcy4gSS4gVGhlIGNvZGluZyBzZXF1ZW5jZXMgb2YgNDAgbmV3IGdlbmVzIChLSUFBMDAwMeKAk0tJQUEwMDQwKSBkZWR1Y2VkIGJ5IGFuYWx5c2lzIG9mIHJhbmRvbWx5IHNhbXBsZWQgY0ROQSBjbG9uZXMgZnJvbSBodW1hbiBpbW1hdHVyZSBteWVsb2lkIGNlbGwgbGluZSBLRy0xOyBETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5NDsgIGNpdGF0aW9uX3ZvbHVtZT0xOyAgY2l0YXRpb25fcGFnZXM9MjctMzU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBtYW1tYWxpYW4gZ2VuZSBjb2xsZWN0aW9uOyBTY2llbmNlOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTI4NjsgIGNpdGF0aW9uX3BhZ2VzPTQ1NS00NTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRvd2FyZCBhIGNhdGFsb2cgb2YgaHVtYW4gZ2VuZXMgYW5kIHByb3RlaW5zOiBzZXF1ZW5jaW5nIGFuZCBhbmFseXNpcyBvZiA1MDAgbm92ZWwgY29tcGxldGUgcHJvdGVpbiBjb2RpbmcgaHVtYW4gY0ROQXM7IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9NDIyLTQzNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RnVuY3Rpb25hbCBhbm5vdGF0aW9uIG9mIGEgZnVsbC1sZW5ndGggbW91c2UgY0ROQSBjb2xsZWN0aW9uOyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9NDA5OyAgY2l0YXRpb25fcGFnZXM9Njg1LTY5MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXJhdGlvbiBhbmQgaW5pdGlhbCBhbmFseXNpcyBvZiBtb3JlIHRoYW4gMTUsMDAwIGZ1bGwtbGVuZ3RoIGh1bWFuIGFuZCBtb3VzZSBjRE5BIHNlcXVlbmNlczsgY2l0YXRpb25fYXV0aG9yPU1hbW1hbGlhbiBHZW5lIENvbGxlY3Rpb24gKE1HQykgUHJvZ3JhbSBUZWFtOyBQcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTk5OyAgY2l0YXRpb25fcGFnZXM9MTY4OTktMTY5MDM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuYWx5c2lzIG9mIHRoZSBtb3VzZSB0cmFuc2NyaXB0b21lIGJhc2VkIG9uIGZ1bmN0aW9uYWwgYW5ub3RhdGlvbiBvZiA2MCw3NzAgZnVsbC1sZW5ndGggY0ROQXM7IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT00MjA7ICBjaXRhdGlvbl9wYWdlcz01NjMtNTczOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QcmVkaWN0aW9uIG9mIHRoZSBjb2Rpbmcgc2VxdWVuY2VzIG9mIG1vdXNlIGhvbW9sb2d1ZXMgb2YgS0lBQSBnZW5lOiBJVi4gVGhlIGNvbXBsZXRlIG51Y2xlb3RpZGUgc2VxdWVuY2VzIG9mIDUwMCBtb3VzZSBLSUFBLWhvbW9sb2dvdXMgY0ROQXMgaWRlbnRpZmllZCBieSBzY3JlZW5pbmcgb2YgdGVybWluYWwgc2VxdWVuY2VzIG9mIGNETkEgY2xvbmVzIHJhbmRvbWx5IHNhbXBsZWQgZnJvbSBzaXplLWZyYWN0aW9uYXRlZCBsaWJyYXJpZXM7IEROQSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9MjA1LTIxODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGxldGUgc2VxdWVuY2luZyBhbmQgY2hhcmFjdGVyaXphdGlvbiBvZiAyMSwyNDMgZnVsbC1sZW5ndGggaHVtYW4gY0ROQXM7IE5hdHVyZSBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MzY7ICBjaXRhdGlvbl9wYWdlcz00MC00NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29uc3RydWN0aW9uIGFuZCBjaGFyYWN0ZXJpemF0aW9uIG9mIGh1bWFuIGJyYWluIGNETkEgbGlicmFyaWVzIHN1aXRhYmxlIGZvciBhbmFseXNpcyBvZiBjRE5BIGNsb25lcyBlbmNvZGluZyByZWxhdGl2ZWx5IGxhcmdlIHByb3RlaW5zOyBETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9NTMtNTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBLYXp1c2EgY0ROQSBwcm9qZWN0IGZvciBpZGVudGlmaWNhdGlvbiBvZiB1bmtub3duIGh1bWFuIHRyYW5zY3JpcHRzOyBDLiBSLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0zMjY7ICBjaXRhdGlvbl9wYWdlcz05NTktOTY2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db25zdHJ1Y3Rpb24gb2YgZXhwcmVzc2lvbi1yZWFkeSBjRE5BIGNsb25lcyBmb3IgS0lBQSBnZW5lczogTWFudWFsIGN1cmF0aW9uIG9mIDMzMCBLSUFBIGNETkEgY2xvbmVzOyBETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9OTktMTA2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QcmVkaWN0aW9uIG9mIHRoZSBjb2Rpbmcgc2VxdWVuY2VzIG9mIG1vdXNlIGhvbW9sb2d1ZXMgb2YgS0lBQSBnZW5lOiBJLiBUaGUgY29tcGxldGUgbnVjbGVvdGlkZSBzZXF1ZW5jZXMgb2YgMTAwIG1vdXNlIEtJQUEtaG9tb2xvZ291cyBjRE5BcyBpZGVudGlmaWVkIGJ5IHNjcmVlbmluZyBvZiB0ZXJtaW5hbCBzZXF1ZW5jZXMgb2YgY0ROQSBjbG9uZXMgcmFuZG9tbHkgc2FtcGxlZCBmcm9tIHNpemUtZnJhY3Rpb25hdGVkIGxpYnJhcmllczsgRE5BIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTE3OS0xODg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhpZ2gtdGhyb3VnaHB1dCBwcm9kdWN0aW9uIG9mIHJlY29tYmluYW50IGFudGlnZW5zIGZvciBtb3VzZSBLSUFBIHByb3RlaW5zIGluIEVzY2hlcmljaGlhIGNvbGk6IGNvbXB1dGF0aW9uYWwgYWxsb2NhdGlvbiBvZiBwbGFzbWlkcyBvZiBnbHV0YXRoaW9uZS1TLXRyYW5zZmVyYXNlLWZ1c2VkIGFudGlnZW5zIGJ5IGFuIGluIHZpdHJvIHJlY29tYmluYXRpb24tYXNzaXN0ZWQgbWV0aG9kOyBETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTEyOS0xMzY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgY29tcHJlaGVuc2l2ZSBhcHByb2FjaCBmb3IgZXN0YWJsaXNobWVudCBvZiB0aGUgcGxhdGZvcm0gdG8gYW5hbHl6ZSBmdW5jdGlvbnMgb2YgS0lBQSBwcm90ZWluczogR2VuZXJhdGlvbiBhbmQgZXZhbHVhdGlvbiBvZiBhbnRpLW1LSUFBIGFudGlib2RpZXM7IFByb3Rlb21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTE0MTItMTQxNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SFVHRTogYSBkYXRhYmFzZSBmb3IgaHVtYW4gS0lBQSBwcm90ZWlucywgYSAyMDA0IHVwZGF0ZSBpbnRlZ3JhdGluZyBIVUdFcHBpIGFuZCBST1VHRTsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTMyOyAgY2l0YXRpb25fcGFnZXM9RDUwMi1ENTA0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgR2VuZSBPbnRvbG9neSBBbm5vdGF0aW9uIChHT0EpIHByb2plY3Q6IEltcGxlbWVudGF0aW9uIG9mIEdPIGluIFNXSVNTLVBST1QsIFRyRU1CTCwgYW5kIEludGVyUHJvOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTY2Mi02NzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmV0aWMgYW5hbHlzaXMgb2YgdGhlIFJlY0UgcGF0aHdheSBvZiBnZW5ldGljIHJlY29tYmluYXRpb24gaW4gRXNjaGVyaWNoaWEgY29saSBLLTEyOyBKLiBCYWN0ZXJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTgxOyAgY2l0YXRpb25fdm9sdW1lPTE0NTsgIGNpdGF0aW9uX3BhZ2VzPTUyMS01MzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUluIHZpdm8gY2xvbmluZyBvZiBQQ1IgcHJvZHVjdHMgaW4gRS4gY29saTsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTkzOyAgY2l0YXRpb25fdm9sdW1lPTIxOyAgY2l0YXRpb25fcGFnZXM9NTE5Mi01MTk3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIG5ldyBsb2dpYyBmb3IgRE5BIGVuZ2luZWVyaW5nIHVzaW5nIHJlY29tYmluYXRpb24gaW4gRXNjaGVyaWNoaWEgY29saTsgTmF0dXJlIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT0yMDsgIGNpdGF0aW9uX3BhZ2VzPTEyMy0xMjg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdBVEVXQVkgcmVjb21iaW5hdGlvbmFsIGNsb25pbmc6IGFwcGxpY2F0aW9uIHRvIHRoZSBjbG9uaW5nIG9mIGxhcmdlIG51bWJlcnMgb2Ygb3BlbiByZWFkaW5nIGZyYW1lcyBvciBPUkZlb21lczsgTWV0aC4gRW56eW1vbC47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MzI4OyAgY2l0YXRpb25fcGFnZXM9NTc1LTU5MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGVjaG5pcXVlczogUmVjb21iaW5vZ2VuaWMgZW5naW5lZXJpbmfigJRuZXcgb3B0aW9ucyBmb3IgY2xvbmluZyBhbmQgbWFuaXB1bGF0aW5nIEROQTsgVHJlbmRzIEJpb2NoZW0uIFNjaS47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MjY7ICBjaXRhdGlvbl9wYWdlcz0zMjUtMzMxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Bbm9tYWxvdXMgYmVoYXZpb3Igb2YgaHVtYW4gbGV1a29jeXRlIGludGVyZmVyb24gc3VidHlwZXMgb24gcG9seWFjcnlsYW1pZGUgZ2VsIGVsZWN0cm9waG9yZXNpcyBpbiB0aGUgcHJlc2VuY2Ugb2YgZG9kZWN5bCBzdWxmYXRlOyBGRUJTIExldHQuOyAgY2l0YXRpb25feWVhcj0xOTg3OyAgY2l0YXRpb25fdm9sdW1lPTIxMTsgIGNpdGF0aW9uX3BhZ2VzPTc4LTgyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbnRlcmFjdGlvbiBvZiBwZXJveGlzb21lcyB3aXRoIG1pY3JvdHVidWxlczsgRXVyLiBKLiBCaW9jaGVtLjsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0yNjc7ICBjaXRhdGlvbl9wYWdlcz02MjY0LTYyNzU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgY29tcHJlaGVuc2l2ZSBhcHByb2FjaCBmb3IgZXN0YWJsaXNobWVudCBvZiB0aGUgcGxhdGZvcm0gdG8gYW5hbHl6ZSBmdW5jdGlvbnMgb2YgS0lBQSBwcm90ZWlucyBJSTogcHVibGljIHJlbGVhc2Ugb2YgaW5hdWd1cmFsIHZlcnNpb24gb2YgSW5HYVAgZGF0YWJhc2UgY29udGFpbmluZyBnZW5lL3Byb3RlaW4gZXhwcmVzc2lvbiBwcm9maWxlcyBmb3IgMTI3IG1vdXNlIEtJQUEgZ2VuZXMvcHJvdGVpbnM7IEROQSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9MjkzLTMwNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9T1JGZW9tZSBjbG9uaW5nIGFuZCBzeXN0ZW1zIGJpb2xvZ3k6IFN0YW5kYXJkaXplZCBtYXNzIHByb2R1Y3Rpb24gb2YgdGhlIHBhcnRzIGZyb20gdGhlIHBhcnRzLWxpc3Q7IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9MjAwMS0yMDA5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1PbGlnby1jYXBwaW5nOiBhIHNpbXBsZSBtZXRob2QgdG8gcmVwbGFjZSB0aGUgY2FwIHN0cnVjdHVyZSBvZiBldWthcnlvdGljIG1STkFzIHdpdGggb2xpZ29yaWJvbnVjbGVvdGlkZXM7IEdlbmU7ICBjaXRhdGlvbl95ZWFyPTE5OTQ7ICBjaXRhdGlvbl92b2x1bWU9MTM4OyAgY2l0YXRpb25fcGFnZXM9MTcxLTE3NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGlnaC1lZmZpY2llbmN5IGZ1bGwtbGVuZ3RoIGNETkEgY2xvbmluZyBieSBiaW90aW55bGF0ZWQgQ0FQIHRyYXBwZXI7IEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0xOTk2OyAgY2l0YXRpb25fdm9sdW1lPTM3OyAgY2l0YXRpb25fcGFnZXM9MzI3LTMzNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2FwIGFuYWx5c2lzIGdlbmUgZXhwcmVzc2lvbiBmb3IgaGlnaC10aHJvdWdocHV0IGFuYWx5c2lzIG9mIHRyYW5zY3JpcHRpb25hbCBzdGFydGluZyBwb2ludCBhbmQgaWRlbnRpZmljYXRpb24gb2YgcHJvbW90ZXIgdXNhZ2U7IFByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0EuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTEwMDsgIGNpdGF0aW9uX3BhZ2VzPTE1Nzc2LTE1NzgxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT014oCyLWVuZCBTQUdFIGZvciB0aGUgYW5hbHlzaXMgb2YgdHJhbnNjcmlwdGlvbmFsIHN0YXJ0IHNpdGVzOyBOYXR1cmUgQmlvdGVjaC47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MjI7ICBjaXRhdGlvbl9wYWdlcz0xMTQ2LTExNDk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPTXigLIgTG9uZyBzZXJpYWwgYW5hbHlzaXMgb2YgZ2VuZSBleHByZXNzaW9uIChMb25nU0FHRSkgYW5kIDPigLJMb25nU0FHRSBmb3IgdHJhbnNjcmlwdG9tZSBjaGFyYWN0ZXJpemF0aW9uIGFuZCBnZW5vbWUgYW5ub3RhdGlvbjsgUHJvYy4gTmF0bCBBY2FkLiBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTEwMTsgIGNpdGF0aW9uX3BhZ2VzPTExNzAxLTExNzA2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMTIvNC8yNTcvMTcxOTU2Ni9kc2kwMTAucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIEFsdGhvdWdoIHdlIGhhdmUgc28gZmFyIGlkZW50aWZpZWQgYW5kIHNlcXVlbmNlZCAmZ3Q7MjAwMCBodW1hbiBsb25nIGNETkFzLCBrbm93biBhcyBLSUFBIGNETkFzLCBoYWxmIG9mIHRoZW0gaGF2ZSB5ZXQgdG8gYmUgZnVuY3Rpb25hbGx5IGFubm90YXRlZC4gIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFci8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj02MjE0NDYmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2RkFnDtldpE3RL4a+vTM4uGIXWUG6s=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f12%2f4%2f257%2f621446" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f12%2f4%2f257%2f621446" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=25342" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/12/4"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/12/4/0/m_dnaresearch12_4.cover.gif?Expires=2147483647&amp;Signature=3uNJBvpswUl8ckNzhRD6LiG0MySGXqGkcTQYLlRIvfld1eLLpkZoI4Ma5qe4h0TQIrhjk1-qwhKlhwYLpQpMu3ntM421fXasS8~G0V0MwH6cjjfM9fdJn0EH2kkIRqpR9dzDT~TKvzm8K-R~6ztHb07HPr6KoeUx6Rgs8s~OeOw2mLTVUNaBcNQn8k14DbREAYbOxque3El9Lplnm2ngB~AQ0rwKokc~WvC44rQAaAltU0RG~GjoIdSMmocCDEXHDpbpBtxuUw6A9Y5q2U4JLWP3XC47XRtYoHhAyJWDFgU30hB9Q-HhZ4XJLURWeo2hPNBzP0pbMYzBduIAxMvdMQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/12/4">
                        <div class="volume trailing-comma">Volume 12</div>
                        <div class="issue">Issue 4</div>

            </a>

        </div>
        <div class="ii-pub-date">
2005        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="10427746">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#10427746">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="10427748">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#10427748">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="10427752">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#10427752">2. Materials and Methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="10427767">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#10427767">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="10427780">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#10427780">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="10427785">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#10427785">Supplementary Material</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="10427789">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#10427789">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/12/4/247/621603">&lt; Previous</a>
        </li>        
            <li class="empty arrow">&nbsp;</li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Preparation of a Set of Expression-Ready Clones of Mammalian Long cDNAs Encoding Large Proteins by the ORF Trap Cloning Method
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Daisuke Nakajima</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><sup>1</sup><div class="insititution">Department of Human Gene Research, Kazusa DNA Research Institute</div><div class="addr-line">2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Daisuke+Nakajima">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nakajima D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nakajima D&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Daisuke Nakajima</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kenji Saito</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><sup>2</sup><div class="insititution">Laboratory of Pharmacogenomics, Graduate School of Pharmaceutical Sciences, Chiba University</div><div class="addr-line">2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kenji+Saito">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Saito K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Saito K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Kenji Saito</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hisashi Yamakawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><sup>1</sup><div class="insititution">Department of Human Gene Research, Kazusa DNA Research Institute</div><div class="addr-line">2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hisashi+Yamakawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yamakawa H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yamakawa H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hisashi Yamakawa</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Reiko F. Kikuno</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><sup>1</sup><div class="insititution">Department of Human Gene Research, Kazusa DNA Research Institute</div><div class="addr-line">2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Reiko+F.+Kikuno">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kikuno R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kikuno R&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Reiko F. Kikuno</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Manabu Nakayama</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><sup>1</sup><div class="insititution">Department of Human Gene Research, Kazusa DNA Research Institute</div><div class="addr-line">2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan</div></div><div class="aff"><div class="label -label">2</div><sup>2</sup><div class="insititution">Laboratory of Pharmacogenomics, Graduate School of Pharmaceutical Sciences, Chiba University</div><div class="addr-line">2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Manabu+Nakayama">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nakayama M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nakayama M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Manabu Nakayama</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Reiko Ohara</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><sup>1</sup><div class="insititution">Department of Human Gene Research, Kazusa DNA Research Institute</div><div class="addr-line">2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Reiko+Ohara">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ohara R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ohara R&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Reiko Ohara</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Noriko Okazaki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><sup>1</sup><div class="insititution">Department of Human Gene Research, Kazusa DNA Research Institute</div><div class="addr-line">2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Noriko+Okazaki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Okazaki N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Okazaki N&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Noriko Okazaki</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hisashi Koga</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><sup>1</sup><div class="insititution">Department of Human Gene Research, Kazusa DNA Research Institute</div><div class="addr-line">2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan</div></div><div class="aff"><div class="label -label">3</div><sup>3</sup><div class="insititution">Chiba Industry Advancement Center</div><div class="addr-line">2-6 Nakase, Mihama-ku, Chiba 261-7126, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hisashi+Koga">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Koga H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Koga H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hisashi Koga</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Takahiro Nagase</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><sup>1</sup><div class="insititution">Department of Human Gene Research, Kazusa DNA Research Institute</div><div class="addr-line">2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <sup>*</sup>To whom correspondence should be addressed. Tel. +81 438 52 3930, Fax. +81 438 52 3931, E-mail: <a href="mailto:nagase@kazusa.or.jp" target="_blank">nagase@kazusa.or.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Takahiro+Nagase">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nagase T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nagase T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Takahiro Nagase</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Osamu Ohara</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><sup>1</sup><div class="insititution">Department of Human Gene Research, Kazusa DNA Research Institute</div><div class="addr-line">2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan</div></div><div class="aff"><div class="label -label">2</div><sup>2</sup><div class="insititution">Laboratory of Pharmacogenomics, Graduate School of Pharmaceutical Sciences, Chiba University</div><div class="addr-line">2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan</div></div><div class="aff"><div class="label -label">4</div><sup>4</sup><div class="insititution">RIKEN Research Center for Allergy and Immunology</div><div class="addr-line">1-7-22 Suehiro, Tsurumi-ku, Yokohama, Kanagawa 230-0045, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Osamu+Ohara">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ohara O">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ohara O&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Osamu Ohara</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Communicated by Michio Oishi</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 12, Issue 4, 1 January 2005, Pages 257–267, <a href='https://doi.org/10.1093/dnares/dsi010'>https://doi.org/10.1093/dnares/dsi010</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">01 January 2005</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">09 March 2005</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Revision Received:</div>
                                <div class="wi-date">22 July 2005</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="621446" href="/dnaresearch/article-pdf/12/4/257/1719566/dsi010.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Daisuke Nakajima, Kenji Saito, Hisashi Yamakawa, Reiko F. Kikuno, Manabu Nakayama, Reiko Ohara, Noriko Okazaki, Hisashi Koga, Takahiro Nagase, Osamu Ohara;  Preparation of a Set of Expression-Ready Clones of Mammalian Long cDNAs Encoding Large Proteins by the ORF Trap Cloning Method, <em>DNA Research</em>, Volume 12, Issue 4, 1 January 2005, Pages 257–267, <a href='https://doi.org/10.1093/dnares/dsi010'>https://doi.org/10.1093/dnares/dsi010</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/621446?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/621446?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/621446?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/621446?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/621446?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/621446?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/621446?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Preparation%20of%20a%20Set%20of%20Expression-Ready%20Clones%20of%20Mammalian%20Long%20cDNAs%20Encoding%20Large%20Proteins%20by%20the%20ORF%20Trap%20Cloning%20Method&amp;publicationDate=2005-01-01&amp;volumeNum=12&amp;issueNum=4&amp;author=Nakajima%2C%20Daisuke%3B%20Saito%2C%20Kenji&amp;startPage=257&amp;endPage=267&amp;contentId=10.1093%2Fdnares%2Fdsi010&amp;oa=&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f12%2f4%2f257%2f621446" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=10427746 id="10427746" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Although we have so far identified and sequenced &gt;2000 human long cDNAs, known as KIAA cDNAs, half of them have yet to be functionally annotated. Expression-ready cDNA clones derived from these genes, where the open reading frame (ORF) of the gene of interest is placed under the control of an appropriate promoter, are critical for functional characterization of these gene products. In this study, we attempted to systematically convert original cDNA clones to expression-ready forms for native and fusion proteins. For this purpose, we developed a new method for ORF cloning based on a homologous recombination in <em>Escherichia coli</em> to avoid laborious manipulations and artificial introduction of mutations in ORF. Using 1589 putative full-length ORFs (from 1002 KIAA genes, 119 human known genes and 468 mouse genes) with an average size of 2.8 kb, we successfully prepared expression plasmids for 1463 native proteins and for 1343 fusion proteins by this method. The resultant expression-ready clones were examined using an <em>in vitro</em> transcription/translation system followed by SDS–polyacrylamide gel electrophoresis and by transient expression of GFP-fusion proteins in human embryonic kidney (HEK) 293 cells. This set of expression-ready clones of long cDNAs encoding large proteins would open a new route to experimentally analyze their functions on a proteomic scale, since unavailability of expression-ready clones for mammalian large proteins has been a major obstacle to the functional analysis of these cDNAs.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;large protein&quot;">large protein</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="cDNA">cDNA</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;expression clone&quot;">expression clone</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="proteomics">proteomics</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;subcellular localization&quot;">subcellular localization</a></div>                            <h2 scrollto-destination=10427748 id="10427748" class="section-title" >1. Introduction</h2>
<p>In the last decade, whole genome sequences have been revealed from bacteria to humans in a complete or a draft form. This invaluable information is expected to shift the paradigm of molecular biology by making it possible to predict a complete catalog of protein-coding genes in an organism. However, the genome sequence information alone is not sufficient to achieve this. Information about transcribed sequences is indispensable for deducing protein primary structures encoded by the mammalian genome due to difficult-to-predict exon–intron structure. Furthermore, because the protein primary structures do not necessarily enable us to deduce their biological functions, we need various types of genomic resources to experimentally explore them in a genome-wide manner. In particular, a set of expression plasmids for all the proteins in an organism serves as an indispensable reagent for functional genomics. Thus, many efforts have been made to prepare sets of defined protein-expression clones, so-called ‘ORFeome cloning’, to analyze protein function on a proteomic scale.<a reveal-id="b1  b7" data-open="b1  b7" class="link link-ref link-reveal xref-bibr">1–7</a> Although open reading frame (ORF) is different from the protein-coding sequence (CDS) by definition, we hereafter use ORF rather than CDS for protein-coding sequence to avoid unnecessary confusion, contrary to the conventions used by other researchers.</p><p>Although ORF cloning into expression vectors has already been attempted in mammals,<a reveal-id="b6" data-open="b6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="b8  b10" data-open="b8  b10" class="link link-ref link-reveal xref-bibr">8–10</a> it involves many technical difficulties at the ORFeome level. The most serious problem is the difficulty in predicting the ORFeome from the mammalian genomic information. Thus, to achieve reliable prediction of protein-coding genes, a large volume of sequence information of cDNAs was accumulated to complement the genomic sequence information in the public domain.<a reveal-id="b11  b18" data-open="b11  b18" class="link link-ref link-reveal xref-bibr">11–18</a> Furthermore, rapidly growing data of genomic and transcriptomic sequences from several different mammals allow us to make comparisons and thereby assign mammalian ORFeomes more convincingly than was possible a few years ago. Therefore, we believe that ORFeome cloning in mammals is a research area that should be accorded high priority.</p><p>Our cDNA sequencing project was unique, focusing on long cDNAs (&gt;4 kb) encoding relatively large proteins from the brain. This was because of our interest in cDNA clones encoding multidomain proteins, many of which play crucial roles, such as signal transduction, cell–cell communication, cell structure/motility and gene regulation, in multicellular organisms.<a reveal-id="b19" data-open="b19" class="link link-ref link-reveal xref-bibr">19</a>,<a reveal-id="b20" data-open="b20" class="link link-ref link-reveal xref-bibr">20</a> More than 2000 newly isolated cDNAs have been entirely sequenced (referred to as KIAA plus a 4-digit number; the average number of amino acid residues in KIAA ORFs was 949). The KIAA cDNA sequences and clones are continuously updated if their ORFs were spuriously interrupted or truncated.<a reveal-id="b21" data-open="b21" class="link link-ref link-reveal xref-bibr">21</a> We have recently started to collect cDNA clones for mouse homologs of human KIAA (mKIAA) genes and polyclonal antibodies against mKIAA proteins to explore the function of KIAA gene products using an animal model <em>in vivo</em>.<a reveal-id="b17" data-open="b17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="b22  b24" data-open="b22  b24" class="link link-ref link-reveal xref-bibr">22–24</a> The information about KIAA and mKIAA genes is available through our databases (HUGE for human KIAA genes; <a class="link link-uri" href="http://www.kazusa.or.jp/huge" target="_blank"></a> and ROUGE for mouse KIAA genes; <a class="link link-uri" href="http://www.kazusa.or.jp/rouge" target="_blank"></a>).<a reveal-id="b25" data-open="b25" class="link link-ref link-reveal xref-bibr">25</a> Approximately half of the KIAA proteins have not been functionally annotated by Gene Ontology.<a reveal-id="b26" data-open="b26" class="link link-ref link-reveal xref-bibr">26</a> To overcome this situation, it is essential to express KIAA proteins in cells and analyze their functions. Thus, we considered that preparation of cDNA clones in an expression-ready format to accelerate the functional study of KIAA gene products with unknown function on a molecular level was urgently needed. A set of KIAA ORF clones would constitute an indispensable subset of the human ORFeome collection. To achieve this, we needed to solve some technical problems in transferring long ORF to an expression vector in a systematic manner. In this study, we developed a new method, termed as the ORF trap method, for ORFeome cloning of long cDNAs. We describe the details of the ORF trap method, its application to long ORF cloning, and characterization of the protein products of the resultant ORF clones. The results indicate that this method worked effectively and the resultant clone set could be successfully used as a versatile reagent for functional characterization of large proteins.</p>                            <h2 scrollto-destination=10427752 id="10427752" class="section-title" >2. Materials and Methods</h2>
                            <h3 scrollto-destination=10427753 id="10427753" class="section-title" >2.1. Materials</h3>
<p>pTRAP1 plasmid DNA was constructed by the Gateway BP reaction with pDONR201 plasmid (Invitrogen, USA) and the <em>att</em>B1-SD-Kozak-Not I-<em>att</em>B2 fragment (5′-GGGGACAAGTTTGTACAAAAAAGCAGGCTTCGAAGGAGATAGAACCA TGGGGGATGTGAAGCTGGTTGCCTCGTCACACATTTCCAAAACCTCCCT CAGTGTGGCGGCCGCCTCATTGTGGAAACGATGGAGGAAGGTGAAGGG GAAGGGGAAGAGGAAGAAGAGTGGAACCCAGCTTTCTTGTACAAAGTGGTCCCC-3′) according to the instructions provided by the supplier, followed by insertion of the DNA fragment harboring the chloramphenicol resistance and <em>ccd</em>B genes amplified by PCR with pDONR201 into the NotI site. The expression clones including various ORFs were constructed by the Gateway LR reaction with the entry clones amplified by the TempliPhi Amplification Kit (Amersham Biosciences, UK) and pcDNA-DEST47 for expression in an <em>in vitro</em> transcription/translation system or in mammalian cells.</p>                            <h3 scrollto-destination=10427755 id="10427755" class="section-title" >2.2. Cultured cells</h3>
<p>The human embryonic kidney (HEK) 293 cell line was obtained from the Health Science Research Resources Bank (HSRRB) of the Japan Health Sciences Foundation. The cell line was grown in DMEM supplemented with 10% of Tet System Approved Fetal Bovine Serum (BD Biosciences, USA).</p>                            <h3 scrollto-destination=10427757 id="10427757" class="section-title" >2.3. Preparation of JC8679 competent cells for electroporation</h3>
<p><em>E. coli</em> strain JC8679 (<em>recB21</em>, <em>recC22</em>, <em>sbcA23</em>, <em>thr-1</em>, <em>leuB6</em>, <em>phi-1</em>, <em>lacY1</em>, <em>galK2</em>, <em>ara-14</em>, <em>Xyl-5</em>, <em>mtl-1</em>, <em>proA2</em>, <em>his-4</em>, <em>argE3</em>, <em>rpsL31</em>, <em>tsx-33</em>, <em>supE44</em>, <em>his-328</em>) was obtained from HSRRB of Japan Health Sciences Foundation and used for bacterial transformation. The competent bacterial cells were prepared as follows: a single colony of JC8679 was picked and grown in 30 ml of SOB medium overnight at 37°C. After 5 ml of the cultured bacteria were inoculated into 500 ml of SOB medium, they were cultured with agitation at 37°C until the cells reached OD<sub>600</sub> = 0.35–0.4. The cells were then chilled on ice for 15 min and harvested by centrifuging of 2300 <em>g</em> for 10 min at 2°C. After discarding the supernatants, the cells were suspended in 5 ml of ice-cold H<sub>2</sub>O, then 500 ml of ice-cold H<sub>2</sub>O was added and the cells were centrifuged and then washed twice. After washing with 20 ml of ice-cold 10% glycerol, the cells were suspended in 1.25 ml of ice-cold 10% glycerol. Finally, the cells were divided and placed in separate tubes, frozen in dry ice-ethanol and stored in −80°C freezer.</p>                            <h3 scrollto-destination=10427759 id="10427759" class="section-title" >2.4. Homologous recombination</h3>
<p>A linear plasmid containing gene-specific sequences was generated by PCR in a 50 µl reaction including 2.5 units LA Taq DNA polymerase (Takara, Japan), 0.25 mM each of four dNTPs and 5 ng of plasmid template, pTRAP1 (94°C for 5 min; 5 cycles of 94°C for 30 s, 40°C for 30 s and 72°C for 2 min; 20 cycles of 94°C for 30 s, 55°C for 30 s and 72°C for 2 min) using 2 µM each of a gene-specific orfn-primer (5′-N<sub>41</sub>CATGGTTCTATC-3′, where N<sub>41</sub> indicates a 41 nt-anti-sense sequence after the start codon of ORF) and a gene-specific orfc-primer (5′-N′<sub>42</sub>XXXXXXXACCCAGCTTTC-3′, where N′<sub>42</sub> indicates a 42-nt-sense sequence before the stop codon of ORF). A consecutive X is a discriminator for a native or fusion-type of construct which contains one of the following sequences: <span class="underline">TA</span>A/<span class="underline">CGTA</span>G, <span class="underline">TG</span>A/<span class="underline">CGCA</span>G, TGA/<span class="underline">TCTAG(A)</span> or TA/<span class="underline">CGTACG</span> where the underlined sequences can be digested by SnaBI, FspI, XbaI or BsiWI restriction endonucleases, respectively, where the construct is a fusion-type. Hereafter, PCR products thus amplified are tentatively referred to as trap vectors. The linear trap vector was then purified with 5–10 µg of a plasmid DNA containing cDNA corresponding to respective gene-specific primer sequences (original clones) by Wizard SV 96 PCR Clean-Up System (Promega, USA) and finally dissolved in 5 µl of H<sub>2</sub>O. JC8679 (20 µl) competent cells were transformed with a mixture of linear trap vector and circular or linear original clone (2 µl) in a cuvette (Bio-Rad, USA, 0.1 cm electrode) by electroporation using a Bio-Rad Gene Pulser set at 1.67 kV, 200 ohms and 25 µF. The transformed cells were incubated in 500 µl of SOC at 37°C for 2 h with agitation and were plated on LB-kanamycin (50 µg/ml) agar. To select recombinant clones, plasmid DNAs were prepared from at least eight colonies per electroporation by the nucleic acid purification system, MFX-9600-Magnia with MagExtractor (Toyobo, Japan) in a 96-well format and the recombined regions were confirmed by single-pass sequencing. Single-pass sequencing was performed from both ends of the insert DNA with GatewaySeqL-A (5′-TCGCGTTAACGCTAGCATGGATCTC-3′) and GatewaySeqL-B (5′-GTAACATCAGAGATTTTGAGACAC-3′) primers and ABI BigDye Terminator Cycle Sequencing Kit Ver. 3.1 (Applied Biosystems, USA) using ABI3700 DNA sequencer. After arraying the plasmid clones correctly recombined in a new 96-well plate, the plasmid DNAs were introduced into competent DH10B cells to resolve plasmid multimers formed in JC8679 and were amplified. At the same time, single-pass sequencing was performed from both ends of the insert DNA again and the sizes of the plasmid DNAs in a covalently closed circular form were estimated by agarose gel electrophoresis with a molecular weight marker for supercoiled DNA (Supercoiled DNA Ladder, Invitrogen, USA) followed by an ethidium bromide-staining and the FragmeNT analysis of gel image data obtained by FluorImager SI (Molecular Dynamics, USA). Although most of the original clones whose backbones were pBluescript II SK (+) vector were adapted into linear form, the cDNA fragment cloned in the pSPORT1 vector were cut out with appropriate restriction enzymes before the recombination reaction to prevent undesired recombination. For some cDNA clones with truncation of a few amino acid residues at the predicted N-terminal sequence, entry clones were constructed by using gene-specific primers containing complementing sequences to make full-length ORFs.</p>                            <h3 scrollto-destination=10427761 id="10427761" class="section-title" >2.5. <em>In vitro</em> transcription/translation assay</h3>
<p>pcDNA-DEST47 destination vector (Invitrogen, USA) was used for the Gateway LR reaction with the entry clones we constructed to produce proteins by an <em>in vitro</em> transcription/translation system using rabbit reticulocyte lysate. For the fusion type of expression clones, C-terminal Cycle 3 green fluorescent protein (GFP)-fusion vector was used. Plasmid DNAs used in an <em>in vitro</em> expression were prepared by MFX-9600-Magnia in a 96-well format from overnight-cultured bacteria and were purified using MultiScreen, MAFB NOB (Millipore, USA) according to the instructions provided by the manufacturer. Plasmids (100–200 ng) were subjected to an <em>in vitro</em> transcription/translation system (TNT T7 Quick Coupled Reticulocyte Lysate System, Promega, USA) in the presence of 4 µl of Quick Mix, 0.2 µl of FluoroTect™ Green<sub>Lys</sub> tRNA (Promega, USA) and 0.1 µl of 1 M methionine in a final reaction volume of 5.3 µl. The products were resolved on MDG-267 Real Gel Plate (concentration gradient: 5–10%, Biocraft, Japan). Prestained protein size markers (BenchMark size markers, Invitrogen, USA) were used for estimation of the apparent molecular masses of the <em>in vitro</em> products. When multiple discrete bands were observed from a single clone, the size of the largest band was estimated.</p>                            <h3 scrollto-destination=10427763 id="10427763" class="section-title" >2.6. Subcellular localization of GFP-fusion protein</h3>
<p>GFP-fusion protein expression clones used for subcellular localization were constructed and prepared as described above. Resultant purified DNAs (100–200 ng) were transfected into HEK293 cells using FUGENE 6 Transfection Reagent (Roche, USA) in an 8-well chambered coverglass (Nalge Nunc International, USA). Cells (200 µl; 4 × 10<sup>4</sup>) were plated 24 h before the transfection experiment. After 40–50 h of transfection, the subcellular localization of the GFP-fusion proteins was observed by fluorescent microscope (Axiovert S100, Zeiss, Germany) and recorded (DP70, Olympus, Tokyo).</p>                            <h3 scrollto-destination=10427765 id="10427765" class="section-title" >2.7. Western blotting</h3>
<p>For western blot analysis of GFP-fusion proteins, expression clones were constructed as described above and prepared using QIAGEN Plasmid Kit (Qiagen, Germany). HEK293 (1.5 × 10<sup>5</sup>) cells were plated in 24-well culture plate (Becton Dickinson, San Jose, CA) 24 h before transfection. GFP-fusion protein expression clones (250 ng) were used for the transfection per well. Cell extracts were prepared 48 h after the transfection by washing cells with PBS and dissolving with 50 µl of the 2× SDS sample buffer [100 mM Tris-Cl (pH 6.8), 4% SDS, 20% glycerol, 0.1% BPB]. After brief sonication, 2-mercaptoethanol was added to the samples to a final concentration of 144 mM and the samples were heated at 95°C for 5 min. The sample (1–5 µl) was applied for SDS–Polyacrylamide gel electrophoresis (SDS–PAGE) (MDG-267 Real Gel Plate, concentration gradient: 5–10%, BIOCRAFT, Japan). At a time, some expression clones were subjected to an <em>in vitro</em> transcription/translation system according to the above-described method except a labeling using FluoroTect™ Green<sub>Lys</sub> tRNA and 5 µl of the sample was used for SDS–PAGE. After the electrophoresis, gels were immersed into the transfer buffer [25 mM Tris-Cl (pH 8.3), 192 mM glycine, 20% (v/v) methanol] for 15 min and the separated proteins were electrophoretically transferred onto PVDF membrane (FluoroTrans W, Pall, USA) using the transfer buffer with the BIOCRAFT BE-300 semidry transfer device. Detection of GFP-fusion protein was performed at room temperature as follows. The transfer membrane was washed with TBS [20 mM Tris-Cl (pH 7.5) and 150 mM NaCl] including 0.05% Tween-20 (TBST) buffer with gentle agitation for 10 min followed by incubation with blocking solution (TBST containing 5% skim milk) for 60 min. The membrane was then incubated with 1:1000 anti-GFP mouse IgG1-k antibody (Nacalai Tesque, Japan) in blocking solution for 60 min. After washing with TBST for 5 min four times, the membrane was incubated with 1:4000 horseradish-peroxidase (HRP)-conjugated anti-mouse IgG antibody (Dako Corporation, Carpinteria, CA) in blocking solution for 60 min. GFP-fusion proteins on the membrane were finally detected by ECL plus (Amersham Biosciences, UK) according to the instructions provided by the manufacturer after washing the membrane with TBST for 5 min four times and exposing it to Biomax X-ray films (Kodak, USA) or recording by Luminescent Image Analyzer LAS1000 (FUJIFILM, Japan). Prestained protein size markers (Magic Mark XP, Invitrogen, USA) were used for estimation of the apparent molecular masses of the GFP-fusion proteins.</p>                            <h2 scrollto-destination=10427767 id="10427767" class="section-title" >3. Results</h2>
                            <h3 scrollto-destination=10427768 id="10427768" class="section-title" >3.1. Cloning of ORFs by homologous recombination in <em>E. coli</em></h3>
<p>In conventional ORF cloning, PCR has been extensively used as a convenient method for excising ORF from parental cDNA clones. Although care was taken to minimize artificial introduction of mutations in ORF during PCR, accidental isolation of artificially mutated clones was always a possibility. This risk obviously increases as the size of ORF increases. Thus, we were strongly motivated to develop an alternative way to isolate ORF clones, because our ORFs of interest were considerably longer than those manipulated in previous reports.<a reveal-id="b1  b4" data-open="b1  b4" class="link link-ref link-reveal xref-bibr">1–4</a>,<a reveal-id="b6" data-open="b6" class="link link-ref link-reveal xref-bibr">6</a> In this study, we used a homologous recombination system in <em>E. coli</em>, JC8679, which has genotype of <em>rec</em>BC and <em>sbc</em>A,<a reveal-id="b27  b29" data-open="b27  b29" class="link link-ref link-reveal xref-bibr">27–29</a> for ORF cloning. This recombination cloning system enabled us to construct multiple clones harboring only ORFs (in practice, from the initiation codon to the termination codon) in parallel through the following three major steps: (i) generation of gene-specific trap vector fragments by PCR using pairs of gene-specific primers and pTRAP1 plasmid DNA as a template; (ii) transformation of <em>E. coli</em> JC8679 cells with a gene-specific trap vector fragment and a plasmid DNA harboring the ORF of interest by electroporation; (iii) clone selection/confirmation by single-pass sequencing after growth on agar plates containing appropriate antibiotics. An outline of the ORF trap cloning method is schematically shown in <a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1A</a>. In brief, pTRAP1 vector was amplified by PCR using the orfn and orfc primers, which contained gene-specific sequences corresponding to a 41-nt anti-sense sequence after the start codon of ORFs and a 42-nt sense sequence before the stop codon of ORFs, respectively, which serve as tags of homologous recombination. The gene-specific sequence of orfn was flanked by the Shine–Dalgarno (SD)–Kozak sequence while orfc was a mixture of oligomers carrying the gene-specific sequence followed by a termination codon and a non-termination codon. The structures of orfn and orfc primers are shown in <a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1B</a>. Resultant linear trap-vector fragments, which have gene-specific sequences at the extremities, were co-introduced into <em>E. coli</em> JC8679 cells with the corresponding cDNA clones. From clones appearing on agar plates containing antibiotics, the flanking sequences of the recombination sites of multiple independent clones were confirmed by single-pass sequencing using vector primers to confirm the recombination at an appropriate site and to remove clones with mutations derived from small amounts of incorrectly synthesized primers and/or PCR errors. Mutations introduced in the vector backbone sequence did not matter because the vector backbone was eventually replaced by the expression vectors. In particular, we designed this cloning method so as to be able to simultaneously obtain ORF clones both for production of C-terminal fusion protein and for native protein. For this purpose, the orfc primer was a mixture of two primers with and without the ORF termination codon. As shown in <a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1B</a>, to discriminate native and C-terminal fusion type of expression clones, a unique restriction endonuclease cleavage site was introduced in the fusion type of clone at the region where the termination codon was removed. As a result, any tag sequence of interest can be introduced in the unique restriction cleavage site. For maximum versatility, the resultant ORF clones are in a compatible format with an <em>in vitro</em> lambda phage recombinational cloning system, known as the Gateway system.<a reveal-id="b30" data-open="b30" class="link link-ref link-reveal xref-bibr">30</a> Once the ORF clones were successfully obtained, we were able to prepare expression plasmids for their native, N-terminal fusion and C-terminal fusion proteins in various expression systems on demand using the Gateway recombination cloning system. Using 1589 putative full-length ORFs (from 1121 human and 468 mouse genes) with an average size of 2.8 kb, we successfully prepared expression plasmids for 1463 native proteins and for 1343 fusion proteins by this method. Putative full-length ORFs were chosen on the basis of sequence information on homologous genes and GENSCAN predicted CDSs. Information about the entry clones thus prepared is available through Supplementary Table (available at <a class="link link-uri" href="http://www.dnares.oxfordjournals.org" target="_blank"></a>) and the HUGE protein database.</p>                        <a id="10427770" scrollto-destination="10427770"></a>
<div id="fig1" data-id="fig1" class="fig fig-section"><div class="label fig-label"><strong>Figure 1</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/4/10.1093/dnares/dsi010/2/m_dsi010f1.jpeg?Expires=2147483647&Signature=hEi-FlarCuTJIooK6mJJOogt0ujj~c8m0a6Od-B0IfYFqGHyrIRXf8~17BicKMrSoWfZ07qwrIaXryoqoO6yMbDH9U9oqgRktHNy3Zd5yOS7s6f0PxYCcAWsBirh3X8RT8SedHNfa72rccqDDQgRbtoxzh6Bnk3h4mvb2duRWo1qU0oO7nlVaIIT2S~drawMjGWCGHRs5Pw9joQbhK2j37U21ux4KhYwvdfY9SBVyyqQuzJita6LiHXUels-645fZfohSqkT6sWP6ahP4fro2tetwCxAsVqRPId5BduJLnTbCXuIR8LgnY1F4TaX9XQh7BRmx9w9xJTnEKMah-VyRQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Schematic representation of the ORF trap cloning. (A) Strategy for cloning of ORFs. Trap vector is prepared by PCR using the gene-specific orfn and orfc primers with pTRAP1 vector as a template. The trap vector is introduced into JC8679 cells with the original clones corresponding to the gene-specific primers to induce homologous recombination. The resultant recombinant clones termed pENT can be used for Gateway recombination cloning to produce a variety of expression vectors. Kanamycin-resistant (KanR), chloramphenicol-resistant (CmR) and ampicillin-resistant (AmpR) genes are boxed. Shine–Dalgarno (SD) and Kozak consensus sequences, stop codon (stop) or non-stop codon (fusion), ccdB gene and tag sequences are also indicated in the open boxes. Gateway recombination sequences (attL) are shown in the closed boxes. ORFs and untranslated sequences are indicated as gray and open boxes. (B) Sequences of orfn and orfc primers. The consensus sequences of gene-specific primers and the corresponding vector sequences are boxed. SD, Kozak and stop codon sequences are indicated in boldface. The gene-specific sequences are shown as N(41) and N(42) (see text for details). Discriminator sequences in the orfc primer are indicated as X and the respective discriminators for the fusion-type clones are shown with a unique restriction endonuclease site, in which the degenerated nucleotides are underlined." path-from-xml="dsi010f1.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/10427770/dsi010f1.jpeg" path-from-xml="dsi010f1.jpeg" target="_blank">View large</a><a section="10427770" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/4/10.1093/dnares/dsi010/2/dsi010f1.jpeg?Expires=2147483647&Signature=SSgn-LtHfpvIIjWf1UQJYX6rdxgp-g3q6r1MuvTipE3D7YiX8TjhOLJntKMukMZ919KzZBjZBvCyZPLLgCq2uHgY3EGdvjQl0wH7NdEK3bT2XlKDqf05O-7CbABhXcOmXo28x24MrI64axyL3SgRaoYcOJ7V0YPm6~lzm8GIIT5BxfHZgWMApwNa9REfPE-Uw5f2d4dGB4pGh0NLUAdtddIKtifWg8WZaon4bvN6cz05qnT2mTwOAsQI9RqjMdOyd8NCytHGb8dRVLp7awywXpyOWfUM2KVXXcTBqIJ558R44TbRybLtWPM~8GUUZ4AReRExXl2XpCHGysWKWid8nw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=10427770&ar=621446&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi010f1.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Schematic representation of the ORF trap cloning. (A) Strategy for cloning of ORFs. Trap vector is prepared by PCR using the gene-specific orfn and orfc primers with pTRAP1 vector as a template. The trap vector is introduced into JC8679 cells with the original clones corresponding to the gene-specific primers to induce homologous recombination. The resultant recombinant clones termed pENT can be used for Gateway recombination cloning to produce a variety of expression vectors. Kanamycin-resistant (KanR), chloramphenicol-resistant (CmR) and ampicillin-resistant (AmpR) genes are boxed. Shine–Dalgarno (SD) and Kozak consensus sequences, stop codon (stop) or non-stop codon (fusion), ccdB gene and tag sequences are also indicated in the open boxes. Gateway recombination sequences (attL) are shown in the closed boxes. ORFs and untranslated sequences are indicated as gray and open boxes. (B) Sequences of orfn and orfc primers. The consensus sequences of gene-specific primers and the corresponding vector sequences are boxed. SD, Kozak and stop codon sequences are indicated in boldface. The gene-specific sequences are shown as N(41) and N(42) (see text for details). Discriminator sequences in the orfc primer are indicated as X and the respective discriminators for the fusion-type clones are shown with a unique restriction endonuclease site, in which the degenerated nucleotides are underlined.</p></div></div><div content-id="fig1" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 1</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/4/10.1093/dnares/dsi010/2/m_dsi010f1.jpeg?Expires=2147483647&Signature=hEi-FlarCuTJIooK6mJJOogt0ujj~c8m0a6Od-B0IfYFqGHyrIRXf8~17BicKMrSoWfZ07qwrIaXryoqoO6yMbDH9U9oqgRktHNy3Zd5yOS7s6f0PxYCcAWsBirh3X8RT8SedHNfa72rccqDDQgRbtoxzh6Bnk3h4mvb2duRWo1qU0oO7nlVaIIT2S~drawMjGWCGHRs5Pw9joQbhK2j37U21ux4KhYwvdfY9SBVyyqQuzJita6LiHXUels-645fZfohSqkT6sWP6ahP4fro2tetwCxAsVqRPId5BduJLnTbCXuIR8LgnY1F4TaX9XQh7BRmx9w9xJTnEKMah-VyRQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Schematic representation of the ORF trap cloning. (A) Strategy for cloning of ORFs. Trap vector is prepared by PCR using the gene-specific orfn and orfc primers with pTRAP1 vector as a template. The trap vector is introduced into JC8679 cells with the original clones corresponding to the gene-specific primers to induce homologous recombination. The resultant recombinant clones termed pENT can be used for Gateway recombination cloning to produce a variety of expression vectors. Kanamycin-resistant (KanR), chloramphenicol-resistant (CmR) and ampicillin-resistant (AmpR) genes are boxed. Shine–Dalgarno (SD) and Kozak consensus sequences, stop codon (stop) or non-stop codon (fusion), ccdB gene and tag sequences are also indicated in the open boxes. Gateway recombination sequences (attL) are shown in the closed boxes. ORFs and untranslated sequences are indicated as gray and open boxes. (B) Sequences of orfn and orfc primers. The consensus sequences of gene-specific primers and the corresponding vector sequences are boxed. SD, Kozak and stop codon sequences are indicated in boldface. The gene-specific sequences are shown as N(41) and N(42) (see text for details). Discriminator sequences in the orfc primer are indicated as X and the respective discriminators for the fusion-type clones are shown with a unique restriction endonuclease site, in which the degenerated nucleotides are underlined." path-from-xml="dsi010f1.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/10427770/dsi010f1.jpeg" path-from-xml="dsi010f1.jpeg" target="_blank">View large</a><a section="10427770" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/4/10.1093/dnares/dsi010/2/dsi010f1.jpeg?Expires=2147483647&Signature=SSgn-LtHfpvIIjWf1UQJYX6rdxgp-g3q6r1MuvTipE3D7YiX8TjhOLJntKMukMZ919KzZBjZBvCyZPLLgCq2uHgY3EGdvjQl0wH7NdEK3bT2XlKDqf05O-7CbABhXcOmXo28x24MrI64axyL3SgRaoYcOJ7V0YPm6~lzm8GIIT5BxfHZgWMApwNa9REfPE-Uw5f2d4dGB4pGh0NLUAdtddIKtifWg8WZaon4bvN6cz05qnT2mTwOAsQI9RqjMdOyd8NCytHGb8dRVLp7awywXpyOWfUM2KVXXcTBqIJ558R44TbRybLtWPM~8GUUZ4AReRExXl2XpCHGysWKWid8nw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=10427770&ar=621446&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi010f1.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Schematic representation of the ORF trap cloning. (A) Strategy for cloning of ORFs. Trap vector is prepared by PCR using the gene-specific orfn and orfc primers with pTRAP1 vector as a template. The trap vector is introduced into JC8679 cells with the original clones corresponding to the gene-specific primers to induce homologous recombination. The resultant recombinant clones termed pENT can be used for Gateway recombination cloning to produce a variety of expression vectors. Kanamycin-resistant (KanR), chloramphenicol-resistant (CmR) and ampicillin-resistant (AmpR) genes are boxed. Shine–Dalgarno (SD) and Kozak consensus sequences, stop codon (stop) or non-stop codon (fusion), ccdB gene and tag sequences are also indicated in the open boxes. Gateway recombination sequences (attL) are shown in the closed boxes. ORFs and untranslated sequences are indicated as gray and open boxes. (B) Sequences of orfn and orfc primers. The consensus sequences of gene-specific primers and the corresponding vector sequences are boxed. SD, Kozak and stop codon sequences are indicated in boldface. The gene-specific sequences are shown as N(41) and N(42) (see text for details). Discriminator sequences in the orfc primer are indicated as X and the respective discriminators for the fusion-type clones are shown with a unique restriction endonuclease site, in which the degenerated nucleotides are underlined.</p></div></div><p>Unlike PCR, homologous recombination-based cloning systems are not expected to introduce point mutations easily because the cloned sequence is amplified <em>in vivo</em> by the endogenous <em>E. coli</em> replication machinery.<a reveal-id="b31" data-open="b31" class="link link-ref link-reveal xref-bibr">31</a> In fact, the sequencing analysis of coding regions derived from 14 cDNA clones, which totaled ∼50 kb, revealed no nucleotide substitutions, insertions or deletions after homologous recombination-based cloning (data not shown).</p>                            <h3 scrollto-destination=10427772 id="10427772" class="section-title" >3.2. Evaluation of recombinant proteins produced <em>in vitro</em> by SDS–PAGE</h3>
<p>To characterize the recombinant proteins, 1530 native type and 1442 GFP-fusion proteins were synthesized <em>in vitro</em> and evaluated by SDS–PAGE. Although some protein products migrated faster or slower than expected, apparent molecular mass estimated by SDS–PAGE is known to vary considerably according to the amino acid sequences and/or post-translational modifications on proteins.<a reveal-id="b32" data-open="b32" class="link link-ref link-reveal xref-bibr">32</a> Thus, we focused our efforts on comparing the size-differences between the native and the fusion protein to evaluate the recombinant products, rather than on their absolute apparent molecular masses. Thus, apparent molecular masses of 1334 (87%) native recombinant proteins and 1283 (89%) GFP-fusion proteins were estimated in a range from 70 to 160% against molecular weights predicted by SDS–PAGE (Supplementary Table is available at <a class="link link-uri" href="http://www.dnares.oxfordjournals.org" target="_blank"></a>). <a reveal-id="fig2" data-open="fig2" class="link link-reveal link-table xref-fig">Fig. 2A</a> shows examples of the analysis of the fluorescence-labeled proteins (clones described in ORFT16 and ORFT17 plates in the Supplementary Table which is available at <a class="link link-uri" href="http://www.dnares.oxfordjournals.org" target="_blank"></a>). Although some small bands, probably resulting from degradation or accidental translation from an internal methionine codon, were seen in some lanes, the sizes of the largest proteins were always compared with the predicted molecular masses. Comparison of the apparent molecular masses of 1179 pairs of native and GFP-fusion proteins are shown in <a reveal-id="fig2" data-open="fig2" class="link link-reveal link-table xref-fig">Fig. 2B</a>. The average size of fluorescence-labeled native proteins was 111 kDa and the average size-difference between native and GFP-fusion proteins was 29.6 kDa with a standard deviation of 5.3. Because fusion of the GFP moiety resulted in an increase in the molecular mass of the recombinant proteins by 26.8 kDa, the increase of apparent molecular mass of the GFP-fusion protein by ∼30 kDa on SDS–PAGE indicated that the reading frame of the recombinant protein was correctly maintained after the ORF trap cloning.</p>                        <a id="10427774" scrollto-destination="10427774"></a>
<div id="fig2" data-id="fig2" class="fig fig-section"><div class="label fig-label"><strong>Figure 2</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/4/10.1093/dnares/dsi010/2/m_dsi010f2.jpeg?Expires=2147483647&Signature=aEoNYt4gM7DmvZ6U7Do9JNHjoYpXi8cbl3dzd-sdECqRnuVgrM-C0WFet7h7rhFQ9RR0Ti4Ek50Yue7woGc3wT1qJDob2BRMO1-oAZW9Bim3G6udlRS98Vh88rg8sEvLEDO8Jh0VnLKPuJSyuJ9GS56InS6ROK24qwd1UaZ56izAg0pWAOl9FLaD6OKt9LUM7l4j5nurS~9OVpcCOhYkXyZCoFBHxPE2u~ZEexpQ7PH4piFO0aRSd0p10Z2i0A4U~JJ4arM1KAprgmL6rk0m1NH6JBPFwk1sZabuLAud9v~Usu1DZCfWyj5XtlASUGjvI6njpYlBsbF4aKz5qfVCMA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Evaluation of ORF trap clones by estimation of the size of their recombinant proteins. (A) Fluorescence-labeled recombinant proteins produced in an in vitro transcription/translation system. 303 native (N) and GFP-fusion (F) proteins for respective genes were separated by SDS–polyacrylamide gel as a pair and detected with FluorImager (Molecular Device, USA). The positions of BenchMark protein size markers are indicated on the left. (B) Apparent molecular masses of 1179 pairs of native and GFP-fusion proteins. The apparent molecular masses of recombinant clones for native (blue) and GFP-fusion proteins (red) estimated by SDS–PAGE are shown next to each other. The vertical axis indicates the apparent molecular masses in kDa. The horizontal axis indicates the number of protein pairs." path-from-xml="dsi010f2.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/10427774/dsi010f2.jpeg" path-from-xml="dsi010f2.jpeg" target="_blank">View large</a><a section="10427774" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/4/10.1093/dnares/dsi010/2/dsi010f2.jpeg?Expires=2147483647&Signature=LmTi6VoH36Hp7EMpQNfFi~8uBUo6fB~2TNJ3FY2iy--OUtVtk0RWCKQGueGWyIdUFcVKFCkt3jN~R7Sj6apXDl1KhLyuOcbsSqY0SMzpUMu7WfSNWV57ajkDCNl5R8tPyWMS2nVfSkr9ts-w75IAQCY4-i9uElBFaIQHXYy7bIGgybgiJceWPzpbGieGlYs3QDU8Yd7w8sD9srGk821j42pezBu-6MPHOk1wyAE~wFZKwg8TvtFfUaHeywsFk8d8N294U78~ZuWe-GXdOHif~9DK17D-kxG586wdXcoSZGFU2wTlkCp5l4cn~53SVFaq7rvbcNy5yqL1a3EZNdRsmA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=10427774&ar=621446&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi010f2.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Evaluation of ORF trap clones by estimation of the size of their recombinant proteins. (A) Fluorescence-labeled recombinant proteins produced in an <em>in vitro</em> transcription/translation system. 303 native (N) and GFP-fusion (F) proteins for respective genes were separated by SDS–polyacrylamide gel as a pair and detected with FluorImager (Molecular Device, USA). The positions of BenchMark protein size markers are indicated on the left. (B) Apparent molecular masses of 1179 pairs of native and GFP-fusion proteins. The apparent molecular masses of recombinant clones for native (blue) and GFP-fusion proteins (red) estimated by SDS–PAGE are shown next to each other. The vertical axis indicates the apparent molecular masses in kDa. The horizontal axis indicates the number of protein pairs.</p></div></div><div content-id="fig2" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 2</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/4/10.1093/dnares/dsi010/2/m_dsi010f2.jpeg?Expires=2147483647&Signature=aEoNYt4gM7DmvZ6U7Do9JNHjoYpXi8cbl3dzd-sdECqRnuVgrM-C0WFet7h7rhFQ9RR0Ti4Ek50Yue7woGc3wT1qJDob2BRMO1-oAZW9Bim3G6udlRS98Vh88rg8sEvLEDO8Jh0VnLKPuJSyuJ9GS56InS6ROK24qwd1UaZ56izAg0pWAOl9FLaD6OKt9LUM7l4j5nurS~9OVpcCOhYkXyZCoFBHxPE2u~ZEexpQ7PH4piFO0aRSd0p10Z2i0A4U~JJ4arM1KAprgmL6rk0m1NH6JBPFwk1sZabuLAud9v~Usu1DZCfWyj5XtlASUGjvI6njpYlBsbF4aKz5qfVCMA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Evaluation of ORF trap clones by estimation of the size of their recombinant proteins. (A) Fluorescence-labeled recombinant proteins produced in an in vitro transcription/translation system. 303 native (N) and GFP-fusion (F) proteins for respective genes were separated by SDS–polyacrylamide gel as a pair and detected with FluorImager (Molecular Device, USA). The positions of BenchMark protein size markers are indicated on the left. (B) Apparent molecular masses of 1179 pairs of native and GFP-fusion proteins. The apparent molecular masses of recombinant clones for native (blue) and GFP-fusion proteins (red) estimated by SDS–PAGE are shown next to each other. The vertical axis indicates the apparent molecular masses in kDa. The horizontal axis indicates the number of protein pairs." path-from-xml="dsi010f2.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/10427774/dsi010f2.jpeg" path-from-xml="dsi010f2.jpeg" target="_blank">View large</a><a section="10427774" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/4/10.1093/dnares/dsi010/2/dsi010f2.jpeg?Expires=2147483647&Signature=LmTi6VoH36Hp7EMpQNfFi~8uBUo6fB~2TNJ3FY2iy--OUtVtk0RWCKQGueGWyIdUFcVKFCkt3jN~R7Sj6apXDl1KhLyuOcbsSqY0SMzpUMu7WfSNWV57ajkDCNl5R8tPyWMS2nVfSkr9ts-w75IAQCY4-i9uElBFaIQHXYy7bIGgybgiJceWPzpbGieGlYs3QDU8Yd7w8sD9srGk821j42pezBu-6MPHOk1wyAE~wFZKwg8TvtFfUaHeywsFk8d8N294U78~ZuWe-GXdOHif~9DK17D-kxG586wdXcoSZGFU2wTlkCp5l4cn~53SVFaq7rvbcNy5yqL1a3EZNdRsmA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=10427774&ar=621446&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi010f2.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Evaluation of ORF trap clones by estimation of the size of their recombinant proteins. (A) Fluorescence-labeled recombinant proteins produced in an <em>in vitro</em> transcription/translation system. 303 native (N) and GFP-fusion (F) proteins for respective genes were separated by SDS–polyacrylamide gel as a pair and detected with FluorImager (Molecular Device, USA). The positions of BenchMark protein size markers are indicated on the left. (B) Apparent molecular masses of 1179 pairs of native and GFP-fusion proteins. The apparent molecular masses of recombinant clones for native (blue) and GFP-fusion proteins (red) estimated by SDS–PAGE are shown next to each other. The vertical axis indicates the apparent molecular masses in kDa. The horizontal axis indicates the number of protein pairs.</p></div></div>                            <h3 scrollto-destination=10427775 id="10427775" class="section-title" >3.3. Analysis of transiently expressed GFP-fusion proteins in cultured cells</h3>
<p>To demonstrate the usefulness of the set of long ORF clones, we first examined the subcellular localization of transiently expressed GFP-fusion proteins in living HEK293 cells along with the systematic analysis for subcellular localization of expressed fusion proteins developed by Simpson and coworkers.<a reveal-id="b8" data-open="b8" class="link link-ref link-reveal xref-bibr">8</a> Example of subcellular localizations of transiently expressed GFP-fusion proteins were shown in <a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3A</a>. We identified the subcellular localizations for 719 GFP-fusion proteins in HEK293 cells with a success rate of 82%, and they were tentatively classified into 13 groups as shown in <a reveal-id="tbl1" data-open="tbl1" class="link link-reveal link-table xref-fig">Table 1</a>. Nearly 34% of the fusion proteins were dispersed in whole cytoplasm, while some proteins were localized in unspecified organelles within the cytoplasm in a speckled distribution (the ‘cytoplasmic organelle’ group in <a reveal-id="tbl1" data-open="tbl1" class="link link-reveal link-table xref-fig">Table 1</a>). Some speckles had the potential to translocate with detectable speed in the cytoplasm, suggesting that their organization is dependent on a microtubular system similar to that of peroxisomes (data not shown).<a reveal-id="b33" data-open="b33" class="link link-ref link-reveal xref-bibr">33</a> The next most frequently observed localization was the nucleus (12.6%). Although some fusion proteins were localized as interchromatin granule clusters in the nucleus (the ‘nuclear vesicle’ group in <a reveal-id="tbl1" data-open="tbl1" class="link link-reveal link-table xref-fig">Table 1</a>), their detailed localization has yet to be studied. The fluorescent images of HEK293 cells expressing GFP-fusion proteins are accessible through the InGaP database (<a class="link link-uri" href="http://www.kazusa.or.jp/ingap" target="_blank"></a>).<a reveal-id="b34" data-open="b34" class="link link-ref link-reveal xref-bibr">34</a></p>                        <a id="10427777" scrollto-destination="10427777"></a>
<div id="fig3" data-id="fig3" class="fig fig-section"><div class="label fig-label"><strong>Figure 3</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/4/10.1093/dnares/dsi010/2/m_dsi010f3.jpeg?Expires=2147483647&Signature=U23D0pxFnpTbSl4h8HflzhhEJ-5CFFAfjvMQZ-RHD1ZmC~~viYf5WGgjXe9xg21o5v3huAP70SSFv8UIIsdoOAh4DuCtw45MBt~Lksb57TMpf0xUnbs2UDry1kxKsPi4FiY4k4t3c2AWEaUytmjdsvVNd4z6YwJLjArYonmy6VH5mYiWPlUS6TKxpvECYTJUf95pgjZDz~BoaO7WH4KGKXuGmAUj6K749ODyNdW9s4sL1ZguJNQxwUmXm6YEFDbcYEPf3c-2BJxLJEvIEPmuJ930Y3emw~qoEuluMPZ0XTrlReaJVJfMjJw0nJxwI-IHiatUebf9BsbTYZCUzXXIUw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Recombinant proteins expressed in HEK293 cells. Twelve GFP-fusion protein expression clones were analyzed as described below: lane 1, L1CAM: neural cell adhesion molecule L1 precursor; lane 2, PCDHGA8/KIAA0588: protocadherin gamma A12 precursor; lane 3, MCLC/KIAA0761: mid-1-related chloride channel; lane 4, KCND2/KIAA1044: potassium voltage-gated channel subfamily D member 2; lane 5, SCN3B/KIAA1158: sodium channel beta-3 subunit precursor; lane 6, TPCN1/KIAA1169: two pore segment channel 1; lane 7, C11orf11/KIAA0659: neural stem cell-derived dendrite regulator; lane 8, NRXN2/KIAA0921: neurexin 2-beta precursor; lane 9, PLXND1/KIAA0620: plexin D1 precursor; lane 10, PLXNB3/KIAA1206: plexin B3 precursor; lane 11, ERR3/KIAA0832: estrogen-related receptor gamma isoform 2; and lane 12, PDZRN3/KIAA1095, PDZ domain containing RING finger 3. (A) Subcellular localizations of transiently expressed the GFP-fusion proteins. Fluorescence image (left) shows localization of GFP-fusion proteins in white; the phase contrast image (right) shows cell shape. A scale bar is indicated in the panel of L1CAM as a white bar. Gene names are shown on the downside of photos. (B) Western blots of transiently expressed GFP-fusion proteins: 12 fusion type entry clones were transferred into pcDNA-DEST47 vector. Resultant GFP-fusion protein expression clones were transfected into HEK293 cells. The cell extracts (5 µl of the sample) were electrophoretically separated by SDS–PAGE and the GFP-fusion proteins were detected by ECL plus using the anti-GFP antibody after transfer to the membrane and by exposing to X-ray films. The lane numbers correspond to the expression clone described above. Multiple bands that migrated slower than expected are indicated by white closing brackets. (C) Comparison of the apparent molecular masses between GFP-fusion proteins produced in HEK293 cells and rabbit reticulocyte lysates. The lane numbers correspond to the expression clone described above. The GFP-fusion proteins produced in the cultured cells (Cell Ext.: 1 µl of the sample) and in rabbit reticulocyte lysate (RRL: 1 µl of the sample) were electrophoretically separated by SDS–PAGE and detected as described above except the data were recorded by Luminescent Image Analyzer LAS1000. Major products detected by the antibody in the lanes of RRL are indicated by arrows. Molecular masses determined by the protein-size marker are shown in kDa on the left-hand side of the panels in (B) and (C)." path-from-xml="dsi010f3.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/10427777/dsi010f3.jpeg" path-from-xml="dsi010f3.jpeg" target="_blank">View large</a><a section="10427777" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/4/10.1093/dnares/dsi010/2/dsi010f3.jpeg?Expires=2147483647&Signature=TMqgjU~cxIdjHpzcSy04mV48LAx5ditwKjeJ3G1CO4RVXDyLaJ7bvbAe59V~5eFdEMx7FcGkRPFGDofsf3Ev3BRu1i7AR-6wBCHmt-Wtkc7O~-ZuKKYBmVVqPfqQsD3suYqPDSYsgKcAX~8PTaWogs5aC9vQolOoOrOVajdhDZIuxcwvY6ZRcUg8bJMfAE-gsruVWQ2O6ZF3AAlfO0vL-avYKMDL6nC1P9XNklRFZ5XGm5PTdwpfmlZERhI-SFxW7PwguW50sTtPo7GfA7mJy320wBsZioG-R-iMmY5OFmwXWAX8mY-YdLCFR5JdaoYW~ZNr66DcnRXR~c-GG-i6wg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=10427777&ar=621446&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi010f3.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Recombinant proteins expressed in HEK293 cells. Twelve GFP-fusion protein expression clones were analyzed as described below: lane 1, L1CAM: neural cell adhesion molecule L1 precursor; lane 2, PCDHGA8/KIAA0588: protocadherin gamma A12 precursor; lane 3, MCLC/KIAA0761: mid-1-related chloride channel; lane 4, KCND2/KIAA1044: potassium voltage-gated channel subfamily D member 2; lane 5, SCN3B/KIAA1158: sodium channel beta-3 subunit precursor; lane 6, TPCN1/KIAA1169: two pore segment channel 1; lane 7, C11orf11/KIAA0659: neural stem cell-derived dendrite regulator; lane 8, NRXN2/KIAA0921: neurexin 2-beta precursor; lane 9, PLXND1/KIAA0620: plexin D1 precursor; lane 10, PLXNB3/KIAA1206: plexin B3 precursor; lane 11, ERR3/KIAA0832: estrogen-related receptor gamma isoform 2; and lane 12, PDZRN3/KIAA1095, PDZ domain containing RING finger 3. (A) Subcellular localizations of transiently expressed the GFP-fusion proteins. Fluorescence image (left) shows localization of GFP-fusion proteins in white; the phase contrast image (right) shows cell shape. A scale bar is indicated in the panel of L1CAM as a white bar. Gene names are shown on the downside of photos. (B) Western blots of transiently expressed GFP-fusion proteins: 12 fusion type entry clones were transferred into pcDNA-DEST47 vector. Resultant GFP-fusion protein expression clones were transfected into HEK293 cells. The cell extracts (5 µl of the sample) were electrophoretically separated by SDS–PAGE and the GFP-fusion proteins were detected by ECL plus using the anti-GFP antibody after transfer to the membrane and by exposing to X-ray films. The lane numbers correspond to the expression clone described above. Multiple bands that migrated slower than expected are indicated by white closing brackets. (C) Comparison of the apparent molecular masses between GFP-fusion proteins produced in HEK293 cells and rabbit reticulocyte lysates. The lane numbers correspond to the expression clone described above. The GFP-fusion proteins produced in the cultured cells (Cell Ext.: 1 µl of the sample) and in rabbit reticulocyte lysate (RRL: 1 µl of the sample) were electrophoretically separated by SDS–PAGE and detected as described above except the data were recorded by Luminescent Image Analyzer LAS1000. Major products detected by the antibody in the lanes of RRL are indicated by arrows. Molecular masses determined by the protein-size marker are shown in kDa on the left-hand side of the panels in (B) and (C).</p></div></div><div content-id="fig3" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 3</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/4/10.1093/dnares/dsi010/2/m_dsi010f3.jpeg?Expires=2147483647&Signature=U23D0pxFnpTbSl4h8HflzhhEJ-5CFFAfjvMQZ-RHD1ZmC~~viYf5WGgjXe9xg21o5v3huAP70SSFv8UIIsdoOAh4DuCtw45MBt~Lksb57TMpf0xUnbs2UDry1kxKsPi4FiY4k4t3c2AWEaUytmjdsvVNd4z6YwJLjArYonmy6VH5mYiWPlUS6TKxpvECYTJUf95pgjZDz~BoaO7WH4KGKXuGmAUj6K749ODyNdW9s4sL1ZguJNQxwUmXm6YEFDbcYEPf3c-2BJxLJEvIEPmuJ930Y3emw~qoEuluMPZ0XTrlReaJVJfMjJw0nJxwI-IHiatUebf9BsbTYZCUzXXIUw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Recombinant proteins expressed in HEK293 cells. Twelve GFP-fusion protein expression clones were analyzed as described below: lane 1, L1CAM: neural cell adhesion molecule L1 precursor; lane 2, PCDHGA8/KIAA0588: protocadherin gamma A12 precursor; lane 3, MCLC/KIAA0761: mid-1-related chloride channel; lane 4, KCND2/KIAA1044: potassium voltage-gated channel subfamily D member 2; lane 5, SCN3B/KIAA1158: sodium channel beta-3 subunit precursor; lane 6, TPCN1/KIAA1169: two pore segment channel 1; lane 7, C11orf11/KIAA0659: neural stem cell-derived dendrite regulator; lane 8, NRXN2/KIAA0921: neurexin 2-beta precursor; lane 9, PLXND1/KIAA0620: plexin D1 precursor; lane 10, PLXNB3/KIAA1206: plexin B3 precursor; lane 11, ERR3/KIAA0832: estrogen-related receptor gamma isoform 2; and lane 12, PDZRN3/KIAA1095, PDZ domain containing RING finger 3. (A) Subcellular localizations of transiently expressed the GFP-fusion proteins. Fluorescence image (left) shows localization of GFP-fusion proteins in white; the phase contrast image (right) shows cell shape. A scale bar is indicated in the panel of L1CAM as a white bar. Gene names are shown on the downside of photos. (B) Western blots of transiently expressed GFP-fusion proteins: 12 fusion type entry clones were transferred into pcDNA-DEST47 vector. Resultant GFP-fusion protein expression clones were transfected into HEK293 cells. The cell extracts (5 µl of the sample) were electrophoretically separated by SDS–PAGE and the GFP-fusion proteins were detected by ECL plus using the anti-GFP antibody after transfer to the membrane and by exposing to X-ray films. The lane numbers correspond to the expression clone described above. Multiple bands that migrated slower than expected are indicated by white closing brackets. (C) Comparison of the apparent molecular masses between GFP-fusion proteins produced in HEK293 cells and rabbit reticulocyte lysates. The lane numbers correspond to the expression clone described above. The GFP-fusion proteins produced in the cultured cells (Cell Ext.: 1 µl of the sample) and in rabbit reticulocyte lysate (RRL: 1 µl of the sample) were electrophoretically separated by SDS–PAGE and detected as described above except the data were recorded by Luminescent Image Analyzer LAS1000. Major products detected by the antibody in the lanes of RRL are indicated by arrows. Molecular masses determined by the protein-size marker are shown in kDa on the left-hand side of the panels in (B) and (C)." path-from-xml="dsi010f3.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/10427777/dsi010f3.jpeg" path-from-xml="dsi010f3.jpeg" target="_blank">View large</a><a section="10427777" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/4/10.1093/dnares/dsi010/2/dsi010f3.jpeg?Expires=2147483647&Signature=TMqgjU~cxIdjHpzcSy04mV48LAx5ditwKjeJ3G1CO4RVXDyLaJ7bvbAe59V~5eFdEMx7FcGkRPFGDofsf3Ev3BRu1i7AR-6wBCHmt-Wtkc7O~-ZuKKYBmVVqPfqQsD3suYqPDSYsgKcAX~8PTaWogs5aC9vQolOoOrOVajdhDZIuxcwvY6ZRcUg8bJMfAE-gsruVWQ2O6ZF3AAlfO0vL-avYKMDL6nC1P9XNklRFZ5XGm5PTdwpfmlZERhI-SFxW7PwguW50sTtPo7GfA7mJy320wBsZioG-R-iMmY5OFmwXWAX8mY-YdLCFR5JdaoYW~ZNr66DcnRXR~c-GG-i6wg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=10427777&ar=621446&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi010f3.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Recombinant proteins expressed in HEK293 cells. Twelve GFP-fusion protein expression clones were analyzed as described below: lane 1, L1CAM: neural cell adhesion molecule L1 precursor; lane 2, PCDHGA8/KIAA0588: protocadherin gamma A12 precursor; lane 3, MCLC/KIAA0761: mid-1-related chloride channel; lane 4, KCND2/KIAA1044: potassium voltage-gated channel subfamily D member 2; lane 5, SCN3B/KIAA1158: sodium channel beta-3 subunit precursor; lane 6, TPCN1/KIAA1169: two pore segment channel 1; lane 7, C11orf11/KIAA0659: neural stem cell-derived dendrite regulator; lane 8, NRXN2/KIAA0921: neurexin 2-beta precursor; lane 9, PLXND1/KIAA0620: plexin D1 precursor; lane 10, PLXNB3/KIAA1206: plexin B3 precursor; lane 11, ERR3/KIAA0832: estrogen-related receptor gamma isoform 2; and lane 12, PDZRN3/KIAA1095, PDZ domain containing RING finger 3. (A) Subcellular localizations of transiently expressed the GFP-fusion proteins. Fluorescence image (left) shows localization of GFP-fusion proteins in white; the phase contrast image (right) shows cell shape. A scale bar is indicated in the panel of L1CAM as a white bar. Gene names are shown on the downside of photos. (B) Western blots of transiently expressed GFP-fusion proteins: 12 fusion type entry clones were transferred into pcDNA-DEST47 vector. Resultant GFP-fusion protein expression clones were transfected into HEK293 cells. The cell extracts (5 µl of the sample) were electrophoretically separated by SDS–PAGE and the GFP-fusion proteins were detected by ECL plus using the anti-GFP antibody after transfer to the membrane and by exposing to X-ray films. The lane numbers correspond to the expression clone described above. Multiple bands that migrated slower than expected are indicated by white closing brackets. (C) Comparison of the apparent molecular masses between GFP-fusion proteins produced in HEK293 cells and rabbit reticulocyte lysates. The lane numbers correspond to the expression clone described above. The GFP-fusion proteins produced in the cultured cells (Cell Ext.: 1 µl of the sample) and in rabbit reticulocyte lysate (RRL: 1 µl of the sample) were electrophoretically separated by SDS–PAGE and detected as described above except the data were recorded by Luminescent Image Analyzer LAS1000. Major products detected by the antibody in the lanes of RRL are indicated by arrows. Molecular masses determined by the protein-size marker are shown in kDa on the left-hand side of the panels in (B) and (C).</p></div></div>                        <a id="10427778" scrollto-destination="10427778"></a>
<div content-id="tbl1" class="table-modal"><div class="table-wrap"><div id="tbl1" class="table-wrap-title" data-id="tbl1"><div class="label -label"><strong>Table 1</strong></div><div class="caption"><p>Subcellular localization of exogenously expressed GFP-fusion proteins in 293 cells</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Subcellular localization<a reveal-id="tf1-1" data-open="tf1-1" class="link link-ref link-reveal xref-fn">a</a> </th><th>Number of localizations<a reveal-id="tf1-2" data-open="tf1-2" class="link link-ref link-reveal xref-fn">b</a> </th><th>% </th></tr></thead><tbody><tr><td>Cytoplasm </td><td>254 </td><td char=".">33.6 </td></tr><tr><td>Nucleus </td><td>96 </td><td char=".">12.6 </td></tr><tr><td>Cytoplasmic organelle </td><td>78 </td><td char=".">10.4 </td></tr><tr><td>Endoplasmic reticulum </td><td>78 </td><td char=".">10.3 </td></tr><tr><td>Plasma membrane </td><td>78 </td><td char=".">10.3 </td></tr><tr><td>Nucleolus </td><td>39 </td><td char=".">5.1 </td></tr><tr><td>Nuclear envelope </td><td>35 </td><td char=".">4.6 </td></tr><tr><td>Mitochondria </td><td>26 </td><td char=".">3.4 </td></tr><tr><td>Whole cell </td><td>25 </td><td char=".">3.3 </td></tr><tr><td>Nuclear vesicle </td><td>25 </td><td char=".">3.3 </td></tr><tr><td>Golgi </td><td>12 </td><td char=".">1.6 </td></tr><tr><td>Peroxisomes </td><td>8 </td><td char=".">1.1 </td></tr><tr><td>Microtubules </td><td>3 </td><td char=".">0.4 </td></tr><tr><td>Total </td><td>757 </td><td> </td></tr></tbody></table></div><div content-id="tbl1" class="table-modal"><table><thead name="thead" align=""><tr><th>Subcellular localization<a reveal-id="tf1-1" data-open="tf1-1" class="link link-ref link-reveal xref-fn">a</a> </th><th>Number of localizations<a reveal-id="tf1-2" data-open="tf1-2" class="link link-ref link-reveal xref-fn">b</a> </th><th>% </th></tr></thead><tbody><tr><td>Cytoplasm </td><td>254 </td><td char=".">33.6 </td></tr><tr><td>Nucleus </td><td>96 </td><td char=".">12.6 </td></tr><tr><td>Cytoplasmic organelle </td><td>78 </td><td char=".">10.4 </td></tr><tr><td>Endoplasmic reticulum </td><td>78 </td><td char=".">10.3 </td></tr><tr><td>Plasma membrane </td><td>78 </td><td char=".">10.3 </td></tr><tr><td>Nucleolus </td><td>39 </td><td char=".">5.1 </td></tr><tr><td>Nuclear envelope </td><td>35 </td><td char=".">4.6 </td></tr><tr><td>Mitochondria </td><td>26 </td><td char=".">3.4 </td></tr><tr><td>Whole cell </td><td>25 </td><td char=".">3.3 </td></tr><tr><td>Nuclear vesicle </td><td>25 </td><td char=".">3.3 </td></tr><tr><td>Golgi </td><td>12 </td><td char=".">1.6 </td></tr><tr><td>Peroxisomes </td><td>8 </td><td char=".">1.1 </td></tr><tr><td>Microtubules </td><td>3 </td><td char=".">0.4 </td></tr><tr><td>Total </td><td>757 </td><td> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tf1-1"><div class="fn"><p><sup>a</sup>Subcellular localization is tentatively classified into 13 groups.</p></div></div><div content-id="tf1-2"><div class="fn"><p><sup>b</sup>Number of subcellular localizations observed for the 589 exogenously expressed proteins are indicated in each group.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/10427778" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="tbl1" class="table-wrap-title" data-id="tbl1"><div class="label -label"><strong>Table 1</strong></div><div class="caption"><p>Subcellular localization of exogenously expressed GFP-fusion proteins in 293 cells</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Subcellular localization<a reveal-id="tf1-1" data-open="tf1-1" class="link link-ref link-reveal xref-fn">a</a> </th><th>Number of localizations<a reveal-id="tf1-2" data-open="tf1-2" class="link link-ref link-reveal xref-fn">b</a> </th><th>% </th></tr></thead><tbody><tr><td>Cytoplasm </td><td>254 </td><td char=".">33.6 </td></tr><tr><td>Nucleus </td><td>96 </td><td char=".">12.6 </td></tr><tr><td>Cytoplasmic organelle </td><td>78 </td><td char=".">10.4 </td></tr><tr><td>Endoplasmic reticulum </td><td>78 </td><td char=".">10.3 </td></tr><tr><td>Plasma membrane </td><td>78 </td><td char=".">10.3 </td></tr><tr><td>Nucleolus </td><td>39 </td><td char=".">5.1 </td></tr><tr><td>Nuclear envelope </td><td>35 </td><td char=".">4.6 </td></tr><tr><td>Mitochondria </td><td>26 </td><td char=".">3.4 </td></tr><tr><td>Whole cell </td><td>25 </td><td char=".">3.3 </td></tr><tr><td>Nuclear vesicle </td><td>25 </td><td char=".">3.3 </td></tr><tr><td>Golgi </td><td>12 </td><td char=".">1.6 </td></tr><tr><td>Peroxisomes </td><td>8 </td><td char=".">1.1 </td></tr><tr><td>Microtubules </td><td>3 </td><td char=".">0.4 </td></tr><tr><td>Total </td><td>757 </td><td> </td></tr></tbody></table></div><div content-id="tbl1" class="table-modal"><table><thead name="thead" align=""><tr><th>Subcellular localization<a reveal-id="tf1-1" data-open="tf1-1" class="link link-ref link-reveal xref-fn">a</a> </th><th>Number of localizations<a reveal-id="tf1-2" data-open="tf1-2" class="link link-ref link-reveal xref-fn">b</a> </th><th>% </th></tr></thead><tbody><tr><td>Cytoplasm </td><td>254 </td><td char=".">33.6 </td></tr><tr><td>Nucleus </td><td>96 </td><td char=".">12.6 </td></tr><tr><td>Cytoplasmic organelle </td><td>78 </td><td char=".">10.4 </td></tr><tr><td>Endoplasmic reticulum </td><td>78 </td><td char=".">10.3 </td></tr><tr><td>Plasma membrane </td><td>78 </td><td char=".">10.3 </td></tr><tr><td>Nucleolus </td><td>39 </td><td char=".">5.1 </td></tr><tr><td>Nuclear envelope </td><td>35 </td><td char=".">4.6 </td></tr><tr><td>Mitochondria </td><td>26 </td><td char=".">3.4 </td></tr><tr><td>Whole cell </td><td>25 </td><td char=".">3.3 </td></tr><tr><td>Nuclear vesicle </td><td>25 </td><td char=".">3.3 </td></tr><tr><td>Golgi </td><td>12 </td><td char=".">1.6 </td></tr><tr><td>Peroxisomes </td><td>8 </td><td char=".">1.1 </td></tr><tr><td>Microtubules </td><td>3 </td><td char=".">0.4 </td></tr><tr><td>Total </td><td>757 </td><td> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tf1-1"><div class="fn"><p><sup>a</sup>Subcellular localization is tentatively classified into 13 groups.</p></div></div><div content-id="tf1-2"><div class="fn"><p><sup>b</sup>Number of subcellular localizations observed for the 589 exogenously expressed proteins are indicated in each group.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/10427778" target="_blank">View Large</a></div></div><p>To confirm biochemically whether exogenous gene products were produced in HEK293 cells, we next analyzed the GFP-fusion proteins transiently expressed in HEK293 cells by protein blot analysis using an anti-GFP antibody. The production levels of the GFP-fusion proteins varied considerably from gene to gene, although all of them were placed under the control of an identical CMV promoter (<a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3B</a>, lanes 1–12). Among 12 GFP-fusion proteins examined in our study, some of them showed the extra bands that were slower to migrate on the SDS–PAGE (white closing brackets in <a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3B</a>, lanes 1, 4–7) than expected bands with the sizes estimated by <em>in vitro</em> products. Multiple bands that were faster to migrate on the gel were also detected in some lanes. To compare the apparent molecular masses of the recombinant proteins with the extra bands in HEK293 cells that migrated slowly (Cell Ext. in <a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3C</a>) with those of the corresponding <em>in vitro</em> products, the GFP-fusion proteins produced in a rabbit reticulocyte lysate (RRL in <a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3C</a>) were simultaneously analyzed by the protein blot analysis using an anti-GFP antibody. The positions of their major products produced in rabbit reticulocyte lysate are indicated by black arrows in <a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3C</a>. These results can reveal the occurrence of post-translational events such as modification or processing on certain proteins in certain cells. In the human L1 cell adhesion molecule (<em>L1CAM</em>), the size difference appeared to be caused by protein modifications such as glycosylation because this gene encoded an integral membrane glycoprotein in the immunoglobulin superfamily (<a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3C</a>, lane 1). KIAA1044, KIAA1158, KIAA1169 and KIAA0659 GFP-fusion proteins, which are potassium channel KV4.2, sodium channel beta-3 subunit precursor, two-pore channel 1 and neural stem cell-derived dendrite regulator, respectively, also showed distinct SDS–PAGE patterns when they were produced <em>in vivo</em> and <em>in vitro</em> (<a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3C</a>, lanes 4–7).</p>                            <h2 scrollto-destination=10427780 id="10427780" class="section-title" >4. Discussion</h2>
<p>It is widely accepted that full-length cDNA clones in an expression-ready form are essential for functional analysis of mammalian genes.<a reveal-id="b35" data-open="b35" class="link link-ref link-reveal xref-bibr">35</a> For comprehensive functional analysis, it is an urgent task to prepare a complete set of full-length genes, which can direct the synthesis of authentic proteins encoded by the genome, because they serve as indispensable and versatile reagents for production of recombinant proteins. In mammalian systems, cDNA clones are extensively used for this purpose. There are many sophisticated methods for constructing full-length clone-enriched cDNA libraries, but none of them guarantees that every cDNA clone thus generated is full-length.<a reveal-id="b36" data-open="b36" class="link link-ref link-reveal xref-bibr">36</a>,<a reveal-id="b37" data-open="b37" class="link link-ref link-reveal xref-bibr">37</a> Therefore, the cDNA clones must be carefully evaluated by multiple independent criteria to ensure they are full-length. Since sequence information of orthologous genes among different mammals has become available recently, comparison of the genomic structures of these orthologues enables us to judge whether a cDNA clone encodes full-length ORF or not with increased confidence. The long cDNA clones manipulated in the present study were considered as full-length in this way.<a reveal-id="b21" data-open="b21" class="link link-ref link-reveal xref-bibr">21</a> Recent technical improvements in comprehensively assigning 5′-extreme sequences could eventually confirm the 5′-end structures of human cDNAs,<a reveal-id="b38  b40" data-open="b38  b40" class="link link-ref link-reveal xref-bibr">38–40</a> and, if necessary, these ORF clones should be revised accordingly. This is highly critical in ORF clones because even a single amino acid change could have a dramatic effect on protein function.</p><p>Large cDNAs are generally difficult targets to manipulate, particularly in a systematic manner. Although PCR is often used for ORF cloning of short cDNAs, we considered that for long cDNAs, the amplification step of PCR should be avoided if possible. We developed a homologous recombination system in <em>E. coli</em> to clone large ORF regions into a vector to prevent mutations in the protein-coding sequences in this study. So far, we have attempted to construct 1121 clones for human genes and 468 clones for mouse genes including KIAA and mKIAA genes, which encode relatively large proteins. Most of the clones were confirmed by single-pass sequencing of recombination sites and by evaluation of their expressed proteins. Our experience indicates that this method is quite useful for ORFeome cloning in general.</p><p>When considering the role of gene products, it is useful to elucidate their subcellular distribution even though this information is often obtained from experiments using exogenous gene products. According to the UniProt Knowledgebase (<a class="link link-uri" href="http://www.uniprot.org" target="_blank"></a>), there are only ∼80 reports concerning subcellular localizations of KIAA genes in the citation information of the databases. Determination of the subcellular localization of the remaining unanalyzed KIAA gene products is urgently needed. We have been analyzing the subcellular localizations and the records will be available through our InGaP database (<a class="link link-uri" href="http://www.kazusa.or.jp/ingap" target="_blank"></a>).<a reveal-id="b34" data-open="b34" class="link link-ref link-reveal xref-bibr">34</a> This database also contains immunohistochemical data using antibodies against mKIAA proteins, which we accumulated in another project.<a reveal-id="b24" data-open="b24" class="link link-ref link-reveal xref-bibr">24</a> The comparison of subcellular localizations of mKIAA-GFP fused proteins and antibody-detected mKIAA proteins would provide many biological insights regarding the functions of mKIAA proteins. It should be noted that we analyzed 12 GFP-fusion proteins by protein blotting in parallel with fluorescent imaging analysis. Since we accumulated SDS–PAGE patterns for respective GFP-fusion proteins produced <em>in vitro</em> as described in this study, the comparison of SDS–PAGE patterns for transiently expressed GFP-fusion proteins in certain cells with those of the corresponding <em>in vitro</em> products could reveal occurrence of post-translational modifications (e.g. processing by protease, glycosylation and phosphorylation). Together with the information about their subcellular distribution, these sources of information may provide comprehensive insights into the biological function of KIAA/mKIAA proteins.</p><p>According to the Ensembl database (build34, <a class="link link-uri" href="http://www.ensembl.org" target="_blank"></a>), 9.5% of human genes (2108 genes) encode proteins composed of ≥1000 amino acid residues. While each cDNA is not always full-length, ∼70% of the entirely sequenced cDNAs corresponding to those genes that encode large proteins (most of which are derived from the human brain) are collected at present in our Institute. On the other hand, of the 1002 putative full-length ORFs for KIAA genes treated in this study, only 20% of KIAA cDNA clones are listed in human ORFeome Version 1.1 (Open Biosystems, 8266 human cDNAs)<a reveal-id="b6" data-open="b6" class="link link-ref link-reveal xref-bibr">6</a> as other full-length cDNA clones, based on structural comparison by FASTA analysis (ungapped identities and coverage against amino acid sequences of KIAA proteins were &gt;95% and &gt;90%, respectively, data not shown). This implies that availability of ORF clones for large proteins is quite low in the research community. As a research group making efforts to identify unknown large cDNAs, we intend to collect as many as possible cDNAs encoding relatively large proteins in a full-length expression-ready form. The collection of mammalian long ORF clones will keep growing and will be an invaluable resource for the advancement of molecular biology and medical science.</p>                            <h2 scrollto-destination=10427785 id="10427785" class="section-title" >Supplementary Material</h2>
<p>Supplementary material is available online at <a class="link link-uri" href="http://dnaresearch.oxfordjournals.org" target="_blank"></a></p><p>This project was supported by grants from the Kazusa DNA Research Institute. We thank Kazuko Yamada, Keishi Ozawa, Kiyoe Sumi, Nobue Kashima, Emiko Suzuki and Masatoshi Murakami for their technical assistance.</p>                            <h2 scrollto-destination=10427789 id="10427789" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="b1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hudson</div> <div class="given-names">R. J.</div>Jr.</div>,  <div class="name"><div class="surname">Dawson</div> <div class="given-names">E. P.</div></div>,  <div class="name"><div class="surname">Rushing</div> <div class="given-names">K. L.</div></div>, et al. </div><div class="article-title">The complete set of predicted genes from <em>Saccharomyces cerevisiae</em> in a readily usable form</div>, <div class="source">Genome Res.</div> , <div class="year">1997</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">1169</div>-<div class="lpage">1173</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20complete%20set%20of%20predicted%20genes%20from%20Saccharomyces%20cerevisiae%20in%20a%20readily%20usable%20form&amp;author=R.%20J.%20Hudson&amp;author=E.%20P.%20Dawson&amp;author=K.%20L.%20Rushing&amp;publication_year=1997&amp;journal=Genome%20Res.&amp;volume=7&amp;pages=1169-1173" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9414322" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhu</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Bilgin</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Bangham</div> <div class="given-names">R.</div></div>, et al. </div><div class="article-title">Global analysis of protein activities using proteome chips</div>, <div class="source">Science</div> , <div class="year">2001</div>, vol. <div class="volume">293</div> (pg. <div class="fpage">2101</div>-<div class="lpage">2105</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Global%20analysis%20of%20protein%20activities%20using%20proteome%20chips&amp;author=H.%20Zhu&amp;author=M.%20Bilgin&amp;author=R.%20Bangham&amp;publication_year=2001&amp;journal=Science&amp;volume=293&amp;pages=2101-2105" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1062191" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1062191" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1062191"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11474067" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ho</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Gruhler</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Heilbut</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Systematic identification of protein complexes in <em>Saccharomyces cerevisiae</em> by mass spectrometry</div>, <div class="source">Nature</div> , <div class="year">2002</div>, vol. <div class="volume">415</div> (pg. <div class="fpage">180</div>-<div class="lpage">183</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Systematic%20identification%20of%20protein%20complexes%20in%20Saccharomyces%20cerevisiae%20by%20mass%20spectrometry&amp;author=Y.%20Ho&amp;author=A.%20Gruhler&amp;author=A.%20Heilbut&amp;publication_year=2002&amp;journal=Nature&amp;volume=415&amp;pages=180-183" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/415180a" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F415180a" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F415180a"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11805837" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Reboul</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Vaglio</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Rual</div> <div class="given-names">J.-F.</div></div>, et al. </div><div class="article-title"><em>C. elegans</em> ORFeome version 1.1: Experimental verification of the genome annotation and resource for proteome-scale protein expression</div>, <div class="source">Nature Genet.</div> , <div class="year">2003</div>, vol. <div class="volume">34</div> (pg. <div class="fpage">35</div>-<div class="lpage">41</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=C.%20elegans%20ORFeome%20version%201.1%3A%20Experimental%20verification%20of%20the%20genome%20annotation%20and%20resource%20for%20proteome-scale%20protein%20expression&amp;author=J.%20Reboul&amp;author=P.%20Vaglio&amp;author=J.-F.%20Rual&amp;publication_year=2003&amp;journal=Nature%20Genet.&amp;volume=34&amp;pages=35-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng1140" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng1140" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng1140"> </span></div> </div></div></div></div></div><div content-id="b5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Phizicky</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Bastiaens</div> <div class="given-names">P. I. H.</div></div>,  <div class="name"><div class="surname">Zhu</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Snyder</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Fields</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Protein analysis on a proteomic scale</div>, <div class="source">Nature</div> , <div class="year">2003</div>, vol. <div class="volume">422</div> (pg. <div class="fpage">208</div>-<div class="lpage">215</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Protein%20analysis%20on%20a%20proteomic%20scale&amp;author=E.%20Phizicky&amp;author=P.%20I.%20H.%20Bastiaens&amp;author=H.%20Zhu&amp;author=M.%20Snyder&amp;author=S.%20Fields&amp;publication_year=2003&amp;journal=Nature&amp;volume=422&amp;pages=208-215" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature01512" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature01512" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature01512"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12634794" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rual</div> <div class="given-names">J.-F.</div></div>,  <div class="name"><div class="surname">Hirozane-Kishikawa</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Hao</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Human ORFeome version 1.1: A platform for reverse proteomics</div>, <div class="source">Genome Res.</div> , <div class="year">2004</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">2128</div>-<div class="lpage">2135</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Human%20ORFeome%20version%201.1%3A%20A%20platform%20for%20reverse%20proteomics&amp;author=J.-F.%20Rual&amp;author=T.%20Hirozane-Kishikawa&amp;author=T.%20Hao&amp;publication_year=2004&amp;journal=Genome%20Res.&amp;volume=14&amp;pages=2128-2135" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.2973604" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.2973604" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.2973604"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15489335" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pearlberg</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">LaBaer</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Protein expression clone repositories for functional proteomics</div>, <div class="source">Curr. Opin. Chem. Biol.</div> , <div class="year">2004</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">98</div>-<div class="lpage">102</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Protein%20expression%20clone%20repositories%20for%20functional%20proteomics&amp;author=J.%20Pearlberg&amp;author=J.%20LaBaer&amp;publication_year=2004&amp;journal=Curr.%20Opin.%20Chem.%20Biol.&amp;volume=8&amp;pages=98-102" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cbpa.2003.12.010" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cbpa.2003.12.010" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cbpa.2003.12.010"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15036163" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Simpson</div> <div class="given-names">J. C.</div></div>,  <div class="name"><div class="surname">Wellenreuther</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Poustka</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Pepperkok</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Wiemann</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Systematic subcellular localization of novel proteins identified by large-scale cDNA sequencing</div>, <div class="source">EMBO Rep.</div> , <div class="year">2000</div>, vol. <div class="volume">3</div> (pg. <div class="fpage">287</div>-<div class="lpage">292</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Systematic%20subcellular%20localization%20of%20novel%20proteins%20identified%20by%20large-scale%20cDNA%20sequencing&amp;author=J.%20C.%20Simpson&amp;author=R.%20Wellenreuther&amp;author=A.%20Poustka&amp;author=R.%20Pepperkok&amp;author=S.%20Wiemann&amp;publication_year=2000&amp;journal=EMBO%20Rep.&amp;volume=3&amp;pages=287-292" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/embo-reports/kvd058" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fembo-reports%2Fkvd058" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fembo-reports%2Fkvd058"> </span></div> </div></div></div></div></div><div content-id="b9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Suzuki</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Fukunishi</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Kagawa</div> <div class="given-names">I.</div></div>, et al. </div><div class="article-title">Protein–protein interaction panel using mouse full-length cDNAs</div>, <div class="source">Genome Res.</div> , <div class="year">2001</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">1758</div>-<div class="lpage">1765</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Protein%E2%80%93protein%20interaction%20panel%20using%20mouse%20full-length%20cDNAs&amp;author=H.%20Suzuki&amp;author=Y.%20Fukunishi&amp;author=I.%20Kagawa&amp;publication_year=2001&amp;journal=Genome%20Res.&amp;volume=11&amp;pages=1758-1765" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.180101" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.180101" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.180101"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11591653" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wiemann</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Arlt</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Huber</div> <div class="given-names">W.</div></div>, et al. </div><div class="article-title">From ORFeome to Biology: A functional genomics pipeline</div>, <div class="source">Genome Res.</div> , <div class="year">2004</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">2136</div>-<div class="lpage">2144</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=From%20ORFeome%20to%20Biology%3A%20A%20functional%20genomics%20pipeline&amp;author=S.%20Wiemann&amp;author=D.%20Arlt&amp;author=W.%20Huber&amp;publication_year=2004&amp;journal=Genome%20Res.&amp;volume=14&amp;pages=2136-2144" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.2576704" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.2576704" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.2576704"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15489336" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nomura</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Miyajima</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Sazuka</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Prediction of the coding sequences of unidentified human genes. I. The coding sequences of 40 new genes (KIAA0001–KIAA0040) deduced by analysis of randomly sampled cDNA clones from human immature myeloid cell line KG-1</div>, <div class="source">DNA Res.</div> , <div class="year">1994</div>, vol. <div class="volume">1</div> (pg. <div class="fpage">27</div>-<div class="lpage">35</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Prediction%20of%20the%20coding%20sequences%20of%20unidentified%20human%20genes.%20I.%20The%20coding%20sequences%20of%2040%20new%20genes%20(KIAA0001%E2%80%93KIAA0040)%20deduced%20by%20analysis%20of%20randomly%20sampled%20cDNA%20clones%20from%20human%20immature%20myeloid%20cell%20line%20KG-1&amp;author=N.%20Nomura&amp;author=N.%20Miyajima&amp;author=T.%20Sazuka&amp;publication_year=1994&amp;journal=DNA%20Res.&amp;volume=1&amp;pages=27-35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/1.1.27" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F1.1.27" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2F1.1.27"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7584026" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Strausberg</div> <div class="given-names">R. L.</div></div>,  <div class="name"><div class="surname">Feingold</div> <div class="given-names">E. A.</div></div>,  <div class="name"><div class="surname">Klausner</div> <div class="given-names">R. D.</div></div>,  <div class="name"><div class="surname">Collins</div> <div class="given-names">F. S.</div></div>. </div><div class="article-title">The mammalian gene collection</div>, <div class="source">Science</div> , <div class="year">1999</div>, vol. <div class="volume">286</div> (pg. <div class="fpage">455</div>-<div class="lpage">457</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20mammalian%20gene%20collection&amp;author=R.%20L.%20Strausberg&amp;author=E.%20A.%20Feingold&amp;author=R.%20D.%20Klausner&amp;author=F.%20S.%20Collins&amp;publication_year=1999&amp;journal=Science&amp;volume=286&amp;pages=455-457" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.286.5439.455" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.286.5439.455" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.286.5439.455"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10521335" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wiemann</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Weil</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Wellenreuther</div> <div class="given-names">R.</div></div>, et al. </div><div class="article-title">Toward a catalog of human genes and proteins: sequencing and analysis of 500 novel complete protein coding human cDNAs</div>, <div class="source">Genome Res.</div> , <div class="year">2001</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">422</div>-<div class="lpage">435</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Toward%20a%20catalog%20of%20human%20genes%20and%20proteins%3A%20sequencing%20and%20analysis%20of%20500%20novel%20complete%20protein%20coding%20human%20cDNAs&amp;author=S.%20Wiemann&amp;author=B.%20Weil&amp;author=R.%20Wellenreuther&amp;publication_year=2001&amp;journal=Genome%20Res.&amp;volume=11&amp;pages=422-435" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.GR1547R" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.GR1547R" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.GR1547R"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11230166" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kawai</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Shinagawa</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Shibata</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">Functional annotation of a full-length mouse cDNA collection</div>, <div class="source">Nature</div> , <div class="year">2001</div>, vol. <div class="volume">409</div> (pg. <div class="fpage">685</div>-<div class="lpage">690</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Functional%20annotation%20of%20a%20full-length%20mouse%20cDNA%20collection&amp;author=J.%20Kawai&amp;author=A.%20Shinagawa&amp;author=K.%20Shibata&amp;publication_year=2001&amp;journal=Nature&amp;volume=409&amp;pages=685-690" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/35055500" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F35055500" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F35055500"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11217851" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="collab">Mammalian Gene Collection (MGC) Program Team</div><div class="article-title">Generation and initial analysis of more than 15,000 full-length human and mouse cDNA sequences</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2002</div>, vol. <div class="volume">99</div> (pg. <div class="fpage">16899</div>-<div class="lpage">16903</div>)<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.242603899" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.242603899" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.242603899"> </span></div> </div></div></div></div></div><div content-id="b16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Okazaki</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Furuno</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Kasukawa</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Analysis of the mouse transcriptome based on functional annotation of 60,770 full-length cDNAs</div>, <div class="source">Nature</div> , <div class="year">2002</div>, vol. <div class="volume">420</div> (pg. <div class="fpage">563</div>-<div class="lpage">573</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20the%20mouse%20transcriptome%20based%20on%20functional%20annotation%20of%2060%2C770%20full-length%20cDNAs&amp;author=Y.%20Okazaki&amp;author=M.%20Furuno&amp;author=T.%20Kasukawa&amp;publication_year=2002&amp;journal=Nature&amp;volume=420&amp;pages=563-573" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature01266" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature01266" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature01266"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12466851" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Okazaki</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Kikuno</div> <div class="given-names">R. F.</div></div>,  <div class="name"><div class="surname">Ohara</div> <div class="given-names">R.</div></div>, et al. </div><div class="article-title">Prediction of the coding sequences of mouse homologues of KIAA gene: IV. The complete nucleotide sequences of 500 mouse KIAA-homologous cDNAs identified by screening of terminal sequences of cDNA clones randomly sampled from size-fractionated libraries</div>, <div class="source">DNA Res.</div> , <div class="year">2004</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">205</div>-<div class="lpage">218</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Prediction%20of%20the%20coding%20sequences%20of%20mouse%20homologues%20of%20KIAA%20gene%3A%20IV.%20The%20complete%20nucleotide%20sequences%20of%20500%20mouse%20KIAA-homologous%20cDNAs%20identified%20by%20screening%20of%20terminal%20sequences%20of%20cDNA%20clones%20randomly%20sampled%20from%20size-fractionated%20libraries&amp;author=N.%20Okazaki&amp;author=R.%20F.%20Kikuno&amp;author=R.%20Ohara&amp;publication_year=2004&amp;journal=DNA%20Res.&amp;volume=11&amp;pages=205-218" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/11.3.205" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F11.3.205" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2F11.3.205"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15368895" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ota</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Suzuki</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Nishikawa</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Complete sequencing and characterization of 21,243 full-length human cDNAs</div>, <div class="source">Nature Genet.</div> , <div class="year">2004</div>, vol. <div class="volume">36</div> (pg. <div class="fpage">40</div>-<div class="lpage">45</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complete%20sequencing%20and%20characterization%20of%2021%2C243%20full-length%20human%20cDNAs&amp;author=T.%20Ota&amp;author=Y.%20Suzuki&amp;author=T.%20Nishikawa&amp;publication_year=2004&amp;journal=Nature%20Genet.&amp;volume=36&amp;pages=40-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng1285" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng1285" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng1285"> </span></div> </div></div></div></div></div><div content-id="b19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ohara</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Nagase</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Ishikawa</div> <div class="given-names">K.-I.</div></div>, et al. </div><div class="article-title">Construction and characterization of human brain cDNA libraries suitable for analysis of cDNA clones encoding relatively large proteins</div>, <div class="source">DNA Res.</div> , <div class="year">1997</div>, vol. <div class="volume">4</div> (pg. <div class="fpage">53</div>-<div class="lpage">59</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20and%20characterization%20of%20human%20brain%20cDNA%20libraries%20suitable%20for%20analysis%20of%20cDNA%20clones%20encoding%20relatively%20large%20proteins&amp;author=O.%20Ohara&amp;author=T.%20Nagase&amp;author=K.-I.%20Ishikawa&amp;publication_year=1997&amp;journal=DNA%20Res.&amp;volume=4&amp;pages=53-59" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/4.1.53" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F4.1.53" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2F4.1.53"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9179496" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nagase</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Kikuno</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Ohara</div> <div class="given-names">O.</div></div>. </div><div class="article-title">The Kazusa cDNA project for identification of unknown human transcripts</div>, <div class="source">C. R. Biol.</div> , <div class="year">2003</div>, vol. <div class="volume">326</div> (pg. <div class="fpage">959</div>-<div class="lpage">966</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Kazusa%20cDNA%20project%20for%20identification%20of%20unknown%20human%20transcripts&amp;author=T.%20Nagase&amp;author=R.%20Kikuno&amp;author=O.%20Ohara&amp;publication_year=2003&amp;journal=C.%20R.%20Biol.&amp;volume=326&amp;pages=959-966" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.crvi.2003.09.020" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.crvi.2003.09.020" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.crvi.2003.09.020"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14744101" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nakajima</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Okazaki</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Yamakawa</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Kikuno</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Ohara</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Nagase</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Construction of expression-ready cDNA clones for KIAA genes: Manual curation of 330 KIAA cDNA clones</div>, <div class="source">DNA Res.</div> , <div class="year">2002</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">99</div>-<div class="lpage">106</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20expression-ready%20cDNA%20clones%20for%20KIAA%20genes%3A%20Manual%20curation%20of%20330%20KIAA%20cDNA%20clones&amp;author=D.%20Nakajima&amp;author=N.%20Okazaki&amp;author=H.%20Yamakawa&amp;author=R.%20Kikuno&amp;author=O.%20Ohara&amp;author=T.%20Nagase&amp;publication_year=2002&amp;journal=DNA%20Res.&amp;volume=9&amp;pages=99-106" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/9.3.99" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F9.3.99" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2F9.3.99"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12168954" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Okazaki</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Kikuno</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Ohara</div> <div class="given-names">R.</div></div>, et al. </div><div class="article-title">Prediction of the coding sequences of mouse homologues of KIAA gene: I. The complete nucleotide sequences of 100 mouse KIAA-homologous cDNAs identified by screening of terminal sequences of cDNA clones randomly sampled from size-fractionated libraries</div>, <div class="source">DNA Res.</div> , <div class="year">2002</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">179</div>-<div class="lpage">188</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Prediction%20of%20the%20coding%20sequences%20of%20mouse%20homologues%20of%20KIAA%20gene%3A%20I.%20The%20complete%20nucleotide%20sequences%20of%20100%20mouse%20KIAA-homologous%20cDNAs%20identified%20by%20screening%20of%20terminal%20sequences%20of%20cDNA%20clones%20randomly%20sampled%20from%20size-fractionated%20libraries&amp;author=N.%20Okazaki&amp;author=R.%20Kikuno&amp;author=R.%20Ohara&amp;publication_year=2002&amp;journal=DNA%20Res.&amp;volume=9&amp;pages=179-188" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/9.5.179" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F9.5.179" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2F9.5.179"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12465718" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hara</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Shimada</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Kohga</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Ohara</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Koga</div> <div class="given-names">H.</div></div>. </div><div class="article-title">High-throughput production of recombinant antigens for mouse KIAA proteins in <em>Escherichia coli</em>: computational allocation of plasmids of glutathione-S-transferase-fused antigens by an <em>in vitro</em> recombination-assisted method</div>, <div class="source">DNA Res.</div> , <div class="year">2003</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">129</div>-<div class="lpage">136</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-throughput%20production%20of%20recombinant%20antigens%20for%20mouse%20KIAA%20proteins%20in%20Escherichia%20coli%3A%20computational%20allocation%20of%20plasmids%20of%20glutathione-S-transferase-fused%20antigens%20by%20an%20in%20vitro%20recombination-assisted%20method&amp;author=Y.%20Hara&amp;author=K.%20Shimada&amp;author=H.%20Kohga&amp;author=O.%20Ohara&amp;author=H.%20Koga&amp;publication_year=2003&amp;journal=DNA%20Res.&amp;volume=10&amp;pages=129-136" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/10.3.129" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F10.3.129" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2F10.3.129"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12886955" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Koga</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Shimada</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Hara</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">A comprehensive approach for establishment of the platform to analyze functions of KIAA proteins: Generation and evaluation of anti-mKIAA antibodies</div>, <div class="source">Proteomics</div> , <div class="year">2004</div>, vol. <div class="volume">4</div> (pg. <div class="fpage">1412</div>-<div class="lpage">1416</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20comprehensive%20approach%20for%20establishment%20of%20the%20platform%20to%20analyze%20functions%20of%20KIAA%20proteins%3A%20Generation%20and%20evaluation%20of%20anti-mKIAA%20antibodies&amp;author=H.%20Koga&amp;author=K.%20Shimada&amp;author=Y.%20Hara&amp;publication_year=2004&amp;journal=Proteomics&amp;volume=4&amp;pages=1412-1416" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1615-9861" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1615-9861" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1615-9861"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15188409" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kikuno</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Nagase</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Nakayama</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">HUGE: a database for human KIAA proteins, a 2004 update integrating HUGEppi and ROUGE</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2004</div>, vol. <div class="volume">32</div> (pg. <div class="fpage">D502</div>-<div class="lpage">D504</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=HUGE%3A%20a%20database%20for%20human%20KIAA%20proteins%2C%20a%202004%20update%20integrating%20HUGEppi%20and%20ROUGE&amp;author=R.%20Kikuno&amp;author=T.%20Nagase&amp;author=M.%20Nakayama&amp;publication_year=2004&amp;journal=Nucleic%20Acids%20Res.&amp;volume=32&amp;pages=D502-D504" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkh035" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkh035" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkh035"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14681467" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Camon</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Magrane</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Barrell</div> <div class="given-names">D.</div></div>, et al. </div><div class="article-title">The Gene Ontology Annotation (GOA) project: Implementation of GO in SWISS-PROT, TrEMBL, and InterPro</div>, <div class="source">Genome Res.</div> , <div class="year">2003</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">662</div>-<div class="lpage">672</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Gene%20Ontology%20Annotation%20(GOA)%20project%3A%20Implementation%20of%20GO%20in%20SWISS-PROT%2C%20TrEMBL%2C%20and%20InterPro&amp;author=E.%20Camon&amp;author=M.%20Magrane&amp;author=D.%20Barrell&amp;publication_year=2003&amp;journal=Genome%20Res.&amp;volume=13&amp;pages=662-672" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.461403" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.461403" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.461403"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12654719" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gillen</div> <div class="given-names">J. R.</div></div>,  <div class="name"><div class="surname">Willis</div> <div class="given-names">D. K.</div></div>,  <div class="name"><div class="surname">Clark</div> <div class="given-names">A. J.</div></div>. </div><div class="article-title">Genetic analysis of the RecE pathway of genetic recombination in <em>Escherichia coli</em> K-12</div>, <div class="source">J. Bacteriol.</div> , <div class="year">1981</div>, vol. <div class="volume">145</div> (pg. <div class="fpage">521</div>-<div class="lpage">532</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20analysis%20of%20the%20RecE%20pathway%20of%20genetic%20recombination%20in%20Escherichia%20coli%20K-12&amp;author=J.%20R.%20Gillen&amp;author=D.%20K.%20Willis&amp;author=A.%20J.%20Clark&amp;publication_year=1981&amp;journal=J.%20Bacteriol.&amp;volume=145&amp;pages=521-532" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6257642" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Oliner</div> <div class="given-names">J. D.</div></div>,  <div class="name"><div class="surname">Kinzler</div> <div class="given-names">K. W.</div></div>,  <div class="name"><div class="surname">Vogelstein</div> <div class="given-names">B.</div></div>. </div><div class="article-title"><em>In vivo</em> cloning of PCR products in <em>E. coli</em></div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1993</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">5192</div>-<div class="lpage">5197</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=In%20vivo%20cloning%20of%20PCR%20products%20in%20E.%20coli&amp;author=J.%20D.%20Oliner&amp;author=K.%20W.%20Kinzler&amp;author=B.%20Vogelstein&amp;publication_year=1993&amp;journal=Nucleic%20Acids%20Res.&amp;volume=21&amp;pages=5192-5197" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/21.22.5192" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F21.22.5192" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F21.22.5192"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8255776" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Buchholz</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Muyrers</div> <div class="given-names">J. P.</div></div>,  <div class="name"><div class="surname">P., Stewart</div> <div class="given-names">A. F</div></div>. </div><div class="article-title">A new logic for DNA engineering using recombination in <em>Escherichia coli</em></div>, <div class="source">Nature Genet.</div> , <div class="year">1998</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">123</div>-<div class="lpage">128</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20new%20logic%20for%20DNA%20engineering%20using%20recombination%20in%20Escherichia%20coli&amp;author=Y.%20Zhang&amp;author=F.%20Buchholz&amp;author=J.%20P.%20Muyrers&amp;author=A.%20F%20P.%2C%20Stewart&amp;publication_year=1998&amp;journal=Nature%20Genet.&amp;volume=20&amp;pages=123-128" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/2417" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F2417" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F2417"> </span></div> </div></div></div></div></div><div content-id="b30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Walhout</div> <div class="given-names">A. J. M.</div></div>,  <div class="name"><div class="surname">Temple</div> <div class="given-names">G. F.</div></div>,  <div class="name"><div class="surname">Brasch</div> <div class="given-names">M. A.</div></div>, et al. </div><div class="article-title">GATEWAY recombinational cloning: application to the cloning of large numbers of open reading frames or ORFeomes</div>, <div class="source">Meth. Enzymol.</div> , <div class="year">2000</div>, vol. <div class="volume">328</div> (pg. <div class="fpage">575</div>-<div class="lpage">592</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GATEWAY%20recombinational%20cloning%3A%20application%20to%20the%20cloning%20of%20large%20numbers%20of%20open%20reading%20frames%20or%20ORFeomes&amp;author=A.%20J.%20M.%20Walhout&amp;author=G.%20F.%20Temple&amp;author=M.%20A.%20Brasch&amp;publication_year=2000&amp;journal=Meth.%20Enzymol.&amp;volume=328&amp;pages=575-592" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11075367" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Muyrers</div> <div class="given-names">J. P. P.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Stewart</div> <div class="given-names">F.</div></div>. </div><div class="article-title">Techniques: Recombinogenic engineering—new options for cloning and manipulating DNA</div>, <div class="source">Trends Biochem. Sci.</div> , <div class="year">2001</div>, vol. <div class="volume">26</div> (pg. <div class="fpage">325</div>-<div class="lpage">331</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Techniques%3A%20Recombinogenic%20engineering%E2%80%94new%20options%20for%20cloning%20and%20manipulating%20DNA&amp;author=J.%20P.%20P.%20Muyrers&amp;author=Y.%20Zhang&amp;author=F.%20Stewart&amp;publication_year=2001&amp;journal=Trends%20Biochem.%20Sci.&amp;volume=26&amp;pages=325-331" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0968-0004(00)01757-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0968-0004(00)01757-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0968-0004(00)01757-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11343926" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ohara</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Teraoka</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Anomalous behavior of human leukocyte interferon subtypes on polyacrylamide gel electrophoresis in the presence of dodecyl sulfate</div>, <div class="source">FEBS Lett.</div> , <div class="year">1987</div>, vol. <div class="volume">211</div> (pg. <div class="fpage">78</div>-<div class="lpage">82</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Anomalous%20behavior%20of%20human%20leukocyte%20interferon%20subtypes%20on%20polyacrylamide%20gel%20electrophoresis%20in%20the%20presence%20of%20dodecyl%20sulfate&amp;author=O.%20Ohara&amp;author=H.%20Teraoka&amp;publication_year=1987&amp;journal=FEBS%20Lett.&amp;volume=211&amp;pages=78-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0014-5793(87)81278-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0014-5793(87)81278-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0014-5793(87)81278-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3803589" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Thiemann</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Schrader</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Volkl</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Baumgart</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Fahimi</div> <div class="given-names">H. D</div></div>. </div><div class="article-title">Interaction of peroxisomes with microtubules</div>, <div class="source">Eur. J. Biochem.</div> , <div class="year">2000</div>, vol. <div class="volume">267</div> (pg. <div class="fpage">6264</div>-<div class="lpage">6275</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Interaction%20of%20peroxisomes%20with%20microtubules&amp;author=M.%20Thiemann&amp;author=M.%20Schrader&amp;author=A.%20Volkl&amp;author=E.%20Baumgart&amp;author=H.%20D%20Fahimi&amp;publication_year=2000&amp;journal=Eur.%20J.%20Biochem.&amp;volume=267&amp;pages=6264-6275" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1432-1327.2000.01713.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1432-1327.2000.01713.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1432-1327.2000.01713.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11012681" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Koga</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Yuasa</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Nagase</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">A comprehensive approach for establishment of the platform to analyze functions of KIAA proteins II: public release of inaugural version of InGaP database containing gene/protein expression profiles for 127 mouse KIAA genes/proteins</div>, <div class="source">DNA Res.</div> , <div class="year">2004</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">293</div>-<div class="lpage">304</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20comprehensive%20approach%20for%20establishment%20of%20the%20platform%20to%20analyze%20functions%20of%20KIAA%20proteins%20II%3A%20public%20release%20of%20inaugural%20version%20of%20InGaP%20database%20containing%20gene%2Fprotein%20expression%20profiles%20for%20127%20mouse%20KIAA%20genes%2Fproteins&amp;author=H.%20Koga&amp;author=S.%20Yuasa&amp;author=T.%20Nagase&amp;publication_year=2004&amp;journal=DNA%20Res.&amp;volume=11&amp;pages=293-304" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/11.4.293" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F11.4.293" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2F11.4.293"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15500254" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Brasch</div> <div class="given-names">M. A.</div></div>,  <div class="name"><div class="surname">Hartley</div> <div class="given-names">J. L.</div></div>,  <div class="name"><div class="surname">Vidal</div> <div class="given-names">M.</div></div>. </div><div class="article-title">ORFeome cloning and systems biology: Standardized mass production of the parts from the parts-list</div>, <div class="source">Genome Res.</div> , <div class="year">2004</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">2001</div>-<div class="lpage">2009</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=ORFeome%20cloning%20and%20systems%20biology%3A%20Standardized%20mass%20production%20of%20the%20parts%20from%20the%20parts-list&amp;author=M.%20A.%20Brasch&amp;author=J.%20L.%20Hartley&amp;author=M.%20Vidal&amp;publication_year=2004&amp;journal=Genome%20Res.&amp;volume=14&amp;pages=2001-2009" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.2769804" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.2769804" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.2769804"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15489318" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Maruyama</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Sugano</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Oligo-capping: a simple method to replace the cap structure of eukaryotic mRNAs with oligoribonucleotides</div>, <div class="source">Gene</div> , <div class="year">1994</div>, vol. <div class="volume">138</div> (pg. <div class="fpage">171</div>-<div class="lpage">174</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Oligo-capping%3A%20a%20simple%20method%20to%20replace%20the%20cap%20structure%20of%20eukaryotic%20mRNAs%20with%20oligoribonucleotides&amp;author=K.%20Maruyama&amp;author=S.%20Sugano&amp;publication_year=1994&amp;journal=Gene&amp;volume=138&amp;pages=171-174" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0378-1119(94)90802-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0378-1119(94)90802-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0378-1119(94)90802-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8125298" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Carninci</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Kvam</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Kitamura</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">High-efficiency full-length cDNA cloning by biotinylated CAP trapper</div>, <div class="source">Genomics</div> , <div class="year">1996</div>, vol. <div class="volume">37</div> (pg. <div class="fpage">327</div>-<div class="lpage">336</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-efficiency%20full-length%20cDNA%20cloning%20by%20biotinylated%20CAP%20trapper&amp;author=P.%20Carninci&amp;author=C.%20Kvam&amp;author=A.%20Kitamura&amp;publication_year=1996&amp;journal=Genomics&amp;volume=37&amp;pages=327-336" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/geno.1996.0567" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fgeno.1996.0567" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fgeno.1996.0567"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8938445" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shiraki</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Kondo</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Katayama</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Cap analysis gene expression for high-throughput analysis of transcriptional starting point and identification of promoter usage</div>, <div class="source">Proc. Natl Acad. Sci. USA.</div> , <div class="year">2003</div>, vol. <div class="volume">100</div> (pg. <div class="fpage">15776</div>-<div class="lpage">15781</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cap%20analysis%20gene%20expression%20for%20high-throughput%20analysis%20of%20transcriptional%20starting%20point%20and%20identification%20of%20promoter%20usage&amp;author=T.%20Shiraki&amp;author=S.%20Kondo&amp;author=S.%20Katayama&amp;publication_year=2003&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA.&amp;volume=100&amp;pages=15776-15781" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.2136655100" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.2136655100" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.2136655100"> </span></div> </div></div></div></div></div><div content-id="b39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hashimoto</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Suzuki</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Kasai</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">5′-end SAGE for the analysis of transcriptional start sites</div>, <div class="source">Nature Biotech.</div> , <div class="year">2004</div>, vol. <div class="volume">22</div> (pg. <div class="fpage">1146</div>-<div class="lpage">1149</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=5%E2%80%B2-end%20SAGE%20for%20the%20analysis%20of%20transcriptional%20start%20sites&amp;author=S.%20Hashimoto&amp;author=Y.%20Suzuki&amp;author=Y.%20Kasai&amp;publication_year=2004&amp;journal=Nature%20Biotech.&amp;volume=22&amp;pages=1146-1149" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt998" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt998" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt998"> </span></div> </div></div></div></div></div><div content-id="b40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wei</div> <div class="given-names">C.-L.</div></div>,  <div class="name"><div class="surname">Ng</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Chiu</div> <div class="given-names">K. P</div></div>, et al. </div><div class="article-title">5′ Long serial analysis of gene expression (LongSAGE) and 3′LongSAGE for transcriptome characterization and genome annotation</div>, <div class="source">Proc. Natl Acad. Sci.</div> , <div class="year">2004</div>, vol. <div class="volume">101</div> (pg. <div class="fpage">11701</div>-<div class="lpage">11706</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=5%E2%80%B2%20Long%20serial%20analysis%20of%20gene%20expression%20(LongSAGE)%20and%203%E2%80%B2LongSAGE%20for%20transcriptome%20characterization%20and%20genome%20annotation&amp;author=C.-L.%20Wei&amp;author=P.%20Ng&amp;author=K.%20P%20Chiu&amp;publication_year=2004&amp;journal=Proc.%20Natl%20Acad.%20Sci.&amp;volume=101&amp;pages=11701-11706" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0403514101" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0403514101" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0403514101"> </span></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2006. Kazusa DNA Research Institute</div><div class="license"><div class="license-p">The online version of this article has been published under an open access model. Users are entitled to use, reproduce, disseminate, or display the open access version of this article for non-commercial purposes provided that: the original authorship is properly and fully attributed; the Journal and Oxford University Press are attributed as the original place of publication with the correct citation details given; if an article is subsequently reproduced or disseminated not in its entirety but only in part or as a derivative work this must be clearly indicated. For commercial re-use, please contact journals.permissions@oxfordjournals.org</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=621446&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/4/10.1093/dnares/dsi010/2/dsi010-dsi010_Supplementary_Table.xls?Expires=2147483647&amp;Signature=F8XiriJ-Bzvy3O0Wbyxv2c3UIs2OCCI1JATDTt1zI93ROj6Ye6Cjx0WhLyg-NuY5ZHkCOUX~B-VstScmBIqZ1XGKT-nQIWurvmlD6dH1ZlKd2iPI6i6qoOCF9IxDIoCRbqp7HconVYNf3IRT4XdxE4MFIEgyHnGH-qMILQI~mzQL9R-xoXZQjXt-dLixmoJvZy7zE9CZ5nPFVvMTzzXORk9wyhV76DFvo5UcgtZA-uSt6tvqVpUVEMjhAvpwn8SaYmvFsWVObhteTrhwQzpRVDX9-6g8ROrEj3gFn2mXYE9kg-JGkNG8yhL-4wGCRnsIfbAny7nFYIEt8xBKI9UPEw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - xls file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">244</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000242119700004&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">10</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsi010"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="621446"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000242119700004&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f12%2f4%2f257%2f621446">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000242119700004&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (10)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F12%2F4%2F257%2F621446" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/621446'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Preparation of a Set of Expression-Ready Clones of Mammalian Long cDNAs Encoding Large Proteins by the ORF Trap Cloning Method" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsi010");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f12%2f4%2f257%2f621446">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

