

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/17/5/261/353723","@type":"ScholarlyArticle","name":"Protein Connectivity and Protein Complexity Promotes Human Gene Duplicability in a Mutually Exclusive Manner","datePublished":"2010-09-09","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/17/5","@type":"PublicationIssue","issueNumber":"5","datePublished":"2010-10-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsq019","keywords":["protein–protein interaction network","protein complexity","haploinsufficient","organismal complexity","human"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/17/5/261/353723","author":[{"name":"Bhattacharya, Tanusree","affiliation":"Bioinformatics Centre ,  Bose Institute ,  P 1/12, C.I.T. Scheme VII M, Kolkata 700 054 ,  India","@type":"Person"},{"name":"Ghosh, Tapash Chandra","affiliation":"Bioinformatics Centre ,  Bose Institute ,  P 1/12, C.I.T. Scheme VII M, Kolkata 700 054 ,  India","@type":"Person"}],"description":"Abstract.  It has previously been reported that protein complexity (i.e. number of subunits in a protein complex) is negatively correlated to gene duplicability","pageStart":"261","pageEnd":"270","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/17/5/1/m_cover.gif?Expires=2147483647&Signature=XCQlUD9LK7XFUkpUfRn~YGtjsVRpmrT9eb4syTPTn6rTzboWNlQlH-siiO3TXM5GrRqmN7FUTnXpB3myvmk93syOV9bWEsz2vGx6yN~HeBizlfceSxcyWv64eGdML9C9SGPBdE~eItnEEBaSgpFCn0R~HTuQmXRsioxnSdebpn-~JqHg7eHzflvfN4sXgzFZGJPEuIGC5YLTuLUzh7LpgtxWSMJLCp-arnHh1smvnR4mJrGafdSyV48rumRp74lot6SiJXr5I0jIo5wItyB0fT8rkkkvVM3SAdqxuBdwVyTCOfmwlFn0~mbY3SumXub6CMCFckG66ON0FXK5WQIs7Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Protein Connectivity and Protein Complexity Promotes Human Gene Duplicability in a Mutually Exclusive Manner","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/17/5/1/m_cover.gif?Expires=2147483647&Signature=XCQlUD9LK7XFUkpUfRn~YGtjsVRpmrT9eb4syTPTn6rTzboWNlQlH-siiO3TXM5GrRqmN7FUTnXpB3myvmk93syOV9bWEsz2vGx6yN~HeBizlfceSxcyWv64eGdML9C9SGPBdE~eItnEEBaSgpFCn0R~HTuQmXRsioxnSdebpn-~JqHg7eHzflvfN4sXgzFZGJPEuIGC5YLTuLUzh7LpgtxWSMJLCp-arnHh1smvnR4mJrGafdSyV48rumRp74lot6SiJXr5I0jIo5wItyB0fT8rkkkvVM3SAdqxuBdwVyTCOfmwlFn0~mbY3SumXub6CMCFckG66ON0FXK5WQIs7Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Protein Connectivity and Protein Complexity Promotes Human Gene Duplicability in a Mutually Exclusive Manner" />
    <meta property="og:description" content="Abstract.  It has previously been reported that protein complexity (i.e. number of subunits in a protein complex) is negatively correlated to gene duplicability" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/17/5/261/353723" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2010-09-09" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Bhattacharya, Tanusree" /><meta name="citation_author_institution" content="Bioinformatics Centre, Bose Institute, P 1/12, C.I.T. Scheme VII M, Kolkata 700 054, India" /><meta name="citation_author" content="Ghosh, Tapash Chandra" /><meta name="citation_author_institution" content="Bioinformatics Centre, Bose Institute, P 1/12, C.I.T. Scheme VII M, Kolkata 700 054, India" /><meta name="citation_title" content="Protein Connectivity and Protein Complexity Promotes Human Gene Duplicability in a Mutually Exclusive Manner" /><meta name="citation_firstpage" content="261" /><meta name="citation_lastpage" content="270" /><meta name="citation_doi" content="10.1093/dnares/dsq019" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="17" /><meta name="citation_issue" content="5" /><meta name="citation_publication_date" content="2010/10/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_publisher=Springer, Berlin; Evolution by Gene Duplication;  citation_year=1970;  citation_pages=160" /><meta name="citation_reference" content="citation_title=Segmental duplications and the evolution of the primate genome; Nat. Rev. Genet.;  citation_year=2002;  citation_volume=3;  citation_pages=65-72; " /><meta name="citation_reference" content="citation_publisher=Sinauer Associates, Sunderland, MA; Molecular Evolution;  citation_year=1997;  citation_pages=432" /><meta name="citation_reference" content="citation_title=Molecular evidence for an ancient duplication of the entire yeast genome; Nature;  citation_year=1997;  citation_volume=387;  citation_pages=708-13; " /><meta name="citation_reference" content="citation_title=Sequencing and comparison of yeast species to identify genes and regulatory elements; Nature;  citation_year=2003;  citation_volume=423;  citation_pages=241-54; " /><meta name="citation_reference" content="citation_title=Comparable rates of gene loss and functional divergence after genome duplications early in vertebrate evolution; Genetics;  citation_year=1997;  citation_volume=147;  citation_pages=1259-66; " /><meta name="citation_reference" content="citation_title=Evolutionary analyses of the human genome; Nature;  citation_year=2001;  citation_volume=409;  citation_pages=847-9; " /><meta name="citation_reference" content="citation_title=Gene essentiality, gene duplicability and protein connectivity in human and mouse; Trends Genet.;  citation_year=2007;  citation_volume=23;  citation_pages=375-8; " /><meta name="citation_reference" content="citation_title=Organismal complexity, protein complexity, and gene duplicability; Proc. Natl. Acad Sci. USA;  citation_year=2003;  citation_volume=100;  citation_pages=15661-5; " /><meta name="citation_reference" content="citation_title=Protein function, connectivity, and duplicability in yeast; Mol. Biol. Evol.;  citation_year=2006;  citation_volume=23;  citation_pages=30-9; " /><meta name="citation_reference" content="citation_title=Gene dosage and gene duplicability; Genetics;  citation_year=2008;  citation_volume=179;  citation_pages=2319-24; " /><meta name="citation_reference" content="citation_title=A common framework for understanding the origin of genetic dominance and evolutionary fates of gene duplications; Trends Genet.;  citation_year=2004;  citation_volume=20;  citation_pages=287-91; " /><meta name="citation_reference" content="citation_title=Dosage sensitivity and the evolution of gene families in yeast; Nature;  citation_year=2003;  citation_volume=424;  citation_pages=194-7; " /><meta name="citation_reference" content="citation_title=Divergence in the spatial pattern of gene expression between human duplicate genes; Genome Res.;  citation_year=2003;  citation_volume=13;  citation_pages=1638-45; " /><meta name="citation_reference" content="citation_title=Splitting pairs: the diverging fates of duplicated genes; Nat. Rev. Genet.;  citation_year=2002;  citation_volume=3;  citation_pages=827-37; " /><meta name="citation_reference" content="citation_title=Evolutionary constraints on hub and non-hub proteins in human protein interaction network: insight from protein connectivity and intrinsic disorder; Gene;  citation_year=2009;  citation_volume=434;  citation_pages=50-5; " /><meta name="citation_reference" content="citation_title=Human Protein Reference Database–2009 update; Nucleic Acids Res.;  citation_year=2009;  citation_volume=37;  citation_pages=767-72; " /><meta name="citation_reference" content="citation_title=What properties characterize the hub proteins of the protein–protein interaction network of Saccharomyces cerevisiae?; Genome Biol.;  citation_year=2006;  citation_volume=7;  citation_pages=R45" /><meta name="citation_reference" content="citation_title=Gaining confidence in high throughput protein interaction networks; Nat. Biotechnol.;  citation_year=2004;  citation_volume=22;  citation_pages=78-85; " /><meta name="citation_reference" content="citation_title=CORUM: the comprehensive resource of mammalian protein complexes; Nucleic Acids Res.;  citation_year=2008;  citation_volume=36;  citation_pages=D646-D650; " /><meta name="citation_reference" content="citation_title=Proteome survey reveals modularity of the yeast cell machinery; Nature;  citation_year=2006;  citation_volume=440;  citation_pages=631-6; " /><meta name="citation_reference" content="citation_title=Identification of human haploinsufficient genes and their genomic proximity to segmental duplications; Eur. J. Hum. Genet.;  citation_year=2008;  citation_volume=16;  citation_pages=1350-7; " /><meta name="citation_reference" content="citation_title=Mechanisms of haploinsufficiency revealed by genome-wide profiling in yeast; Genetics;  citation_year=2005;  citation_volume=169;  citation_pages=1915-25; " /><meta name="citation_reference" content="citation_title=EnsemblCompara GeneTrees: Complete, duplication-aware phylogenetic trees in vertebrates; Genome Res.;  citation_year=2009;  citation_volume=19;  citation_pages=327-35; " /><meta name="citation_reference" content="citation_title=Detection of gene duplications and block duplications in eukaryotic genomes; J Struct. Funct. Genomics;  citation_year=2003;  citation_volume=3;  citation_pages=27-34; " /><meta name="citation_reference" content="citation_title=Comparative genomics of centrality and essentiality in three eukaryotic protein-interaction networks; Mol. Biol. Evol.;  citation_year=2005;  citation_volume=22;  citation_pages=803-6; " /><meta name="citation_reference" content="citation_title=Exploring the etiology of haploinsufficiency; Bioessays;  citation_year=2002;  citation_volume=24;  citation_pages=175-84; " /><meta name="citation_reference" content="citation_title=Cancer biology. A matter of dosage; Science;  citation_year=2002;  citation_volume=298;  citation_pages=761-3; " /><meta name="citation_reference" content="citation_title=Higher duplicability of less important genes in yeast genomes; Mol. Biol. Evol.;  citation_year=2006;  citation_volume=23;  citation_pages=144-51; " /><meta name="citation_reference" content="citation_title=Complex topology rather than complex membership is a determinant of protein dosage sensitivity; Mol. Syst. Biol.;  citation_year=2009;  citation_volume=5;  citation_pages=253" /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/17/5/261/1011658/dsq019.pdf" /><meta name="description" content="Abstract.  It has previously been reported that protein complexity (i.e. number of subunits in a protein complex) is negatively correlated to gene duplicability" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/17/5/261/353723" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/17/5/261/353723" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/17/5/261/353723");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsq019';
    </script>
<title>
	Protein Connectivity and Protein Complexity Promotes Human Gene Duplicability in a Mutually Exclusive Manner | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./353723" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0Bb09PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJCaGF0dGFjaGFyeWEsIFRhbnVzcmVlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQmlvaW5mb3JtYXRpY3MgQ2VudHJlLCBCb3NlIEluc3RpdHV0ZSwgUCAxLzEyLCBDLkkuVC4gU2NoZW1lIFZJSSBNLCBLb2xrYXRhIDcwMCAwNTQsIEluZGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iR2hvc2gsIFRhcGFzaCBDaGFuZHJhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQmlvaW5mb3JtYXRpY3MgQ2VudHJlLCBCb3NlIEluc3RpdHV0ZSwgUCAxLzEyLCBDLkkuVC4gU2NoZW1lIFZJSSBNLCBLb2xrYXRhIDcwMCAwNTQsIEluZGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJQcm90ZWluIENvbm5lY3Rpdml0eSBhbmQgUHJvdGVpbiBDb21wbGV4aXR5IFByb21vdGVzIEh1bWFuIEdlbmUgRHVwbGljYWJpbGl0eSBpbiBhIE11dHVhbGx5IEV4Y2x1c2l2ZSBNYW5uZXIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIyNjEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjI3MCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzcTAxOSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIxNyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDEwLzEwLzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fcHVibGlzaGVyPVNwcmluZ2VyLCBCZXJsaW47IEV2b2x1dGlvbiBieSBHZW5lIER1cGxpY2F0aW9uOyAgY2l0YXRpb25feWVhcj0xOTcwOyAgY2l0YXRpb25fcGFnZXM9MTYwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2VnbWVudGFsIGR1cGxpY2F0aW9ucyBhbmQgdGhlIGV2b2x1dGlvbiBvZiB0aGUgcHJpbWF0ZSBnZW5vbWU7IE5hdC4gUmV2LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MzsgIGNpdGF0aW9uX3BhZ2VzPTY1LTcyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9wdWJsaXNoZXI9U2luYXVlciBBc3NvY2lhdGVzLCBTdW5kZXJsYW5kLCBNQTsgTW9sZWN1bGFyIEV2b2x1dGlvbjsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3BhZ2VzPTQzMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1vbGVjdWxhciBldmlkZW5jZSBmb3IgYW4gYW5jaWVudCBkdXBsaWNhdGlvbiBvZiB0aGUgZW50aXJlIHllYXN0IGdlbm9tZTsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTM4NzsgIGNpdGF0aW9uX3BhZ2VzPTcwOC0xMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2VxdWVuY2luZyBhbmQgY29tcGFyaXNvbiBvZiB5ZWFzdCBzcGVjaWVzIHRvIGlkZW50aWZ5IGdlbmVzIGFuZCByZWd1bGF0b3J5IGVsZW1lbnRzOyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9NDIzOyAgY2l0YXRpb25fcGFnZXM9MjQxLTU0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wYXJhYmxlIHJhdGVzIG9mIGdlbmUgbG9zcyBhbmQgZnVuY3Rpb25hbCBkaXZlcmdlbmNlIGFmdGVyIGdlbm9tZSBkdXBsaWNhdGlvbnMgZWFybHkgaW4gdmVydGVicmF0ZSBldm9sdXRpb247IEdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTE0NzsgIGNpdGF0aW9uX3BhZ2VzPTEyNTktNjY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2b2x1dGlvbmFyeSBhbmFseXNlcyBvZiB0aGUgaHVtYW4gZ2Vub21lOyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9NDA5OyAgY2l0YXRpb25fcGFnZXM9ODQ3LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmUgZXNzZW50aWFsaXR5LCBnZW5lIGR1cGxpY2FiaWxpdHkgYW5kIHByb3RlaW4gY29ubmVjdGl2aXR5IGluIGh1bWFuIGFuZCBtb3VzZTsgVHJlbmRzIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0yMzsgIGNpdGF0aW9uX3BhZ2VzPTM3NS04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1PcmdhbmlzbWFsIGNvbXBsZXhpdHksIHByb3RlaW4gY29tcGxleGl0eSwgYW5kIGdlbmUgZHVwbGljYWJpbGl0eTsgUHJvYy4gTmF0bC4gQWNhZCBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMDA7ICBjaXRhdGlvbl9wYWdlcz0xNTY2MS01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qcm90ZWluIGZ1bmN0aW9uLCBjb25uZWN0aXZpdHksIGFuZCBkdXBsaWNhYmlsaXR5IGluIHllYXN0OyBNb2wuIEJpb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTIzOyAgY2l0YXRpb25fcGFnZXM9MzAtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZSBkb3NhZ2UgYW5kIGdlbmUgZHVwbGljYWJpbGl0eTsgR2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTc5OyAgY2l0YXRpb25fcGFnZXM9MjMxOS0yNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBjb21tb24gZnJhbWV3b3JrIGZvciB1bmRlcnN0YW5kaW5nIHRoZSBvcmlnaW4gb2YgZ2VuZXRpYyBkb21pbmFuY2UgYW5kIGV2b2x1dGlvbmFyeSBmYXRlcyBvZiBnZW5lIGR1cGxpY2F0aW9uczsgVHJlbmRzIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0yMDsgIGNpdGF0aW9uX3BhZ2VzPTI4Ny05MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RG9zYWdlIHNlbnNpdGl2aXR5IGFuZCB0aGUgZXZvbHV0aW9uIG9mIGdlbmUgZmFtaWxpZXMgaW4geWVhc3Q7IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT00MjQ7ICBjaXRhdGlvbl9wYWdlcz0xOTQtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGl2ZXJnZW5jZSBpbiB0aGUgc3BhdGlhbCBwYXR0ZXJuIG9mIGdlbmUgZXhwcmVzc2lvbiBiZXR3ZWVuIGh1bWFuIGR1cGxpY2F0ZSBnZW5lczsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz0xNjM4LTQ1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TcGxpdHRpbmcgcGFpcnM6IHRoZSBkaXZlcmdpbmcgZmF0ZXMgb2YgZHVwbGljYXRlZCBnZW5lczsgTmF0LiBSZXYuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0zOyAgY2l0YXRpb25fcGFnZXM9ODI3LTM3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Fdm9sdXRpb25hcnkgY29uc3RyYWludHMgb24gaHViIGFuZCBub24taHViIHByb3RlaW5zIGluIGh1bWFuIHByb3RlaW4gaW50ZXJhY3Rpb24gbmV0d29yazogaW5zaWdodCBmcm9tIHByb3RlaW4gY29ubmVjdGl2aXR5IGFuZCBpbnRyaW5zaWMgZGlzb3JkZXI7IEdlbmU7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9NDM0OyAgY2l0YXRpb25fcGFnZXM9NTAtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SHVtYW4gUHJvdGVpbiBSZWZlcmVuY2UgRGF0YWJhc2XigJMyMDA5IHVwZGF0ZTsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTM3OyAgY2l0YXRpb25fcGFnZXM9NzY3LTcyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1XaGF0IHByb3BlcnRpZXMgY2hhcmFjdGVyaXplIHRoZSBodWIgcHJvdGVpbnMgb2YgdGhlIHByb3RlaW7igJNwcm90ZWluIGludGVyYWN0aW9uIG5ldHdvcmsgb2YgU2FjY2hhcm9teWNlcyBjZXJldmlzaWFlPzsgR2Vub21lIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz1SNDUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HYWluaW5nIGNvbmZpZGVuY2UgaW4gaGlnaCB0aHJvdWdocHV0IHByb3RlaW4gaW50ZXJhY3Rpb24gbmV0d29ya3M7IE5hdC4gQmlvdGVjaG5vbC47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MjI7ICBjaXRhdGlvbl9wYWdlcz03OC04NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q09SVU06IHRoZSBjb21wcmVoZW5zaXZlIHJlc291cmNlIG9mIG1hbW1hbGlhbiBwcm90ZWluIGNvbXBsZXhlczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTM2OyAgY2l0YXRpb25fcGFnZXM9RDY0Ni1ENjUwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qcm90ZW9tZSBzdXJ2ZXkgcmV2ZWFscyBtb2R1bGFyaXR5IG9mIHRoZSB5ZWFzdCBjZWxsIG1hY2hpbmVyeTsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTQ0MDsgIGNpdGF0aW9uX3BhZ2VzPTYzMS02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JZGVudGlmaWNhdGlvbiBvZiBodW1hbiBoYXBsb2luc3VmZmljaWVudCBnZW5lcyBhbmQgdGhlaXIgZ2Vub21pYyBwcm94aW1pdHkgdG8gc2VnbWVudGFsIGR1cGxpY2F0aW9uczsgRXVyLiBKLiBIdW0uIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTEzNTAtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWVjaGFuaXNtcyBvZiBoYXBsb2luc3VmZmljaWVuY3kgcmV2ZWFsZWQgYnkgZ2Vub21lLXdpZGUgcHJvZmlsaW5nIGluIHllYXN0OyBHZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xNjk7ICBjaXRhdGlvbl9wYWdlcz0xOTE1LTI1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FbnNlbWJsQ29tcGFyYSBHZW5lVHJlZXM6IENvbXBsZXRlLCBkdXBsaWNhdGlvbi1hd2FyZSBwaHlsb2dlbmV0aWMgdHJlZXMgaW4gdmVydGVicmF0ZXM7IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTE5OyAgY2l0YXRpb25fcGFnZXM9MzI3LTM1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZXRlY3Rpb24gb2YgZ2VuZSBkdXBsaWNhdGlvbnMgYW5kIGJsb2NrIGR1cGxpY2F0aW9ucyBpbiBldWthcnlvdGljIGdlbm9tZXM7IEogU3RydWN0LiBGdW5jdC4gR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MzsgIGNpdGF0aW9uX3BhZ2VzPTI3LTM0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wYXJhdGl2ZSBnZW5vbWljcyBvZiBjZW50cmFsaXR5IGFuZCBlc3NlbnRpYWxpdHkgaW4gdGhyZWUgZXVrYXJ5b3RpYyBwcm90ZWluLWludGVyYWN0aW9uIG5ldHdvcmtzOyBNb2wuIEJpb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTIyOyAgY2l0YXRpb25fcGFnZXM9ODAzLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV4cGxvcmluZyB0aGUgZXRpb2xvZ3kgb2YgaGFwbG9pbnN1ZmZpY2llbmN5OyBCaW9lc3NheXM7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MjQ7ICBjaXRhdGlvbl9wYWdlcz0xNzUtODQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNhbmNlciBiaW9sb2d5LiBBIG1hdHRlciBvZiBkb3NhZ2U7IFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9Mjk4OyAgY2l0YXRpb25fcGFnZXM9NzYxLTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhpZ2hlciBkdXBsaWNhYmlsaXR5IG9mIGxlc3MgaW1wb3J0YW50IGdlbmVzIGluIHllYXN0IGdlbm9tZXM7IE1vbC4gQmlvbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MjM7ICBjaXRhdGlvbl9wYWdlcz0xNDQtNTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBsZXggdG9wb2xvZ3kgcmF0aGVyIHRoYW4gY29tcGxleCBtZW1iZXJzaGlwIGlzIGEgZGV0ZXJtaW5hbnQgb2YgcHJvdGVpbiBkb3NhZ2Ugc2Vuc2l0aXZpdHk7IE1vbC4gU3lzdC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPTI1MyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8xNy81LzI2MS8xMDExNjU4L2RzcTAxOS5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgSXQgaGFzIHByZXZpb3VzbHkgYmVlbiByZXBvcnRlZCB0aGF0IHByb3RlaW4gY29tcGxleGl0eSAoaS5lLiBudW1iZXIgb2Ygc3VidW5pdHMgaW4gYSBwcm90ZWluIGNvbXBsZXgpIGlzIG5lZ2F0aXZlbHkgY29ycmVsYXRlZCB0byBnZW5lIGR1cGxpY2FiaWxpdHkiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVyLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTM1MzcyMyZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGRYpuNbMUlnjCPlY0dhatB8HpvJ7A==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f17%2f5%2f261%2f353723" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f17%2f5%2f261%2f353723" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=13694" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/17/5"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/17/5/1/m_cover.gif?Expires=2147483647&amp;Signature=XCQlUD9LK7XFUkpUfRn~YGtjsVRpmrT9eb4syTPTn6rTzboWNlQlH-siiO3TXM5GrRqmN7FUTnXpB3myvmk93syOV9bWEsz2vGx6yN~HeBizlfceSxcyWv64eGdML9C9SGPBdE~eItnEEBaSgpFCn0R~HTuQmXRsioxnSdebpn-~JqHg7eHzflvfN4sXgzFZGJPEuIGC5YLTuLUzh7LpgtxWSMJLCp-arnHh1smvnR4mJrGafdSyV48rumRp74lot6SiJXr5I0jIo5wItyB0fT8rkkkvVM3SAdqxuBdwVyTCOfmwlFn0~mbY3SumXub6CMCFckG66ON0FXK5WQIs7Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/17/5">
                        <div class="volume trailing-comma">Volume 17</div>
                        <div class="issue">Issue 5</div>

            </a>

        </div>
        <div class="ii-pub-date">
October 2010        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="5036631">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5036631">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5036638">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5036638">1 Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5036661">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5036661">2 Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5036717">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5036717">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5036774">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5036774">4. Discussions</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5036779">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5036779">Supplementary Data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5036781">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5036781">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5036783">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5036783">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5036787">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5036787">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="empty arrow">&nbsp;</li>
            <li class="next arrow">
            <a href="/dnaresearch/article/17/5/271/353434">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Protein Connectivity and Protein Complexity Promotes Human Gene Duplicability in a Mutually Exclusive Manner
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tanusree Bhattacharya</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Bioinformatics Centre</div>, <div class="insititution">Bose Institute</div>, <div class="addr-line">P 1/12, C.I.T. Scheme VII M, Kolkata 700 054</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tanusree+Bhattacharya">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Bhattacharya T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Bhattacharya T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Tanusree Bhattacharya</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tapash Chandra Ghosh</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Bioinformatics Centre</div>, <div class="insititution">Bose Institute</div>, <div class="addr-line">P 1/12, C.I.T. Scheme VII M, Kolkata 700 054</div>, <div class="country">India</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +91 33 23556626. Fax. <div class="fax">+91 33 23553886</div>. E-mail: <a href="mailto:tapash@boseinst.ernet.in" target="_blank">tapash@boseinst.ernet.in</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tapash+Chandra+Ghosh">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ghosh T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ghosh T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Tapash Chandra Ghosh</a>
                                </span>

                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 17, Issue 5, 1 October 2010, Pages 261–270, <a href='https://doi.org/10.1093/dnares/dsq019'>https://doi.org/10.1093/dnares/dsq019</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">09 September 2010</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">28 January 2010</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">26 July 2010</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="353723" href="/dnaresearch/article-pdf/17/5/261/1011658/dsq019.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Tanusree Bhattacharya, Tapash Chandra Ghosh;  Protein Connectivity and Protein Complexity Promotes Human Gene Duplicability in a Mutually Exclusive Manner, <em>DNA Research</em>, Volume 17, Issue 5, 1 October 2010, Pages 261–270, <a href='https://doi.org/10.1093/dnares/dsq019'>https://doi.org/10.1093/dnares/dsq019</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/353723?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/353723?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/353723?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/353723?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/353723?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/353723?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/353723?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Protein%20Connectivity%20and%20Protein%20Complexity%20Promotes%20Human%20Gene%20Duplicability%20in%20a%20Mutually%20Exclusive%20Manner&amp;publicationDate=2010-09-09&amp;volumeNum=17&amp;issueNum=5&amp;author=Bhattacharya%2C%20Tanusree%3B%20Ghosh%2C%20Tapash%20Chandra&amp;startPage=261&amp;endPage=270&amp;contentId=10.1093%2Fdnares%2Fdsq019&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f17%2f5%2f261%2f353723" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=5036631 id="5036631" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>It has previously been reported that protein complexity (i.e. number of subunits in a protein complex) is negatively correlated to gene duplicability in yeast as well as in humans. However, unlike in yeast, protein connectivity in a protein–protein interaction network has a positive correlation with gene duplicability in human genes. In the present study, we have analyzed 1732 human and 1269 yeast proteins that are present both in a protein–protein interaction network as well as in a protein complex network. In the human case, we observed that both protein connectivity and protein complexity complement each other in a mutually exclusive manner over gene duplicability in a positive direction. Analysis of human haploinsufficient proteins and large protein complexes (complex size &gt;10) shows that when protein connectivity does not have any direct association with gene duplicability, there exists a positive correlation between gene duplicability and protein complexity. The same trend, however, is not found in case of yeast, where both protein connectivity and protein complexity independently guide gene duplicability in the negative direction. We conclude that the higher rate of duplication of human genes may be attributed to organismal complexity either by increasing connectivity in the protein–protein interaction network or by increasing protein complexity.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;protein–protein interaction network&quot;">protein–protein interaction network</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;protein complexity&quot;">protein complexity</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="haploinsufficient">haploinsufficient</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;organismal complexity&quot;">organismal complexity</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="human">human</a></div>                            <h2 scrollto-destination=5036638 id="5036638" class="section-title" data-legacy-id=s1>1 Introduction</h2>
<p>Gene duplication is the primary mechanism for generating new genes and biological processes that facilitates complex organisms to evolve more rapidly than the primitive ones.<sup><a reveal-id="DSQ019C1  DSQ019C3" data-open="DSQ019C1  DSQ019C3" class="link link-ref link-reveal xref-bibr">1–3</a></sup> In general, eukaryotic genomes contain a large fraction of gene duplicates not only from the single gene or segmental duplications but also from a whole genome duplication (WGD) event. <em>Saccharomyces cereviceae</em> is the major example where WGD has occurred ∼100 million years ago.<sup><a reveal-id="DSQ019C4" data-open="DSQ019C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DSQ019C5" data-open="DSQ019C5" class="link link-ref link-reveal xref-bibr">5</a></sup> Genomic instability and massive gene loss promptly followed WGD and purged most of the newly formed gene copies from the yeast genome, retaining ∼10% of them.<sup><a reveal-id="DSQ019C4" data-open="DSQ019C4" class="link link-ref link-reveal xref-bibr">4</a></sup> In case of vertebrates, half of all duplicated genes have been maintained.<sup><a reveal-id="DSQ019C6" data-open="DSQ019C6" class="link link-ref link-reveal xref-bibr">6</a></sup> The analysis of the human genome has shown that most of the human genes are duplicated.<sup><a reveal-id="DSQ019C7" data-open="DSQ019C7" class="link link-ref link-reveal xref-bibr">7</a></sup> Nevertheless, what exactly guides gene duplicability for a particular organism still remains unclear. Two of the most well-known factors that guide gene duplication are (i) protein connectivity, defined as the number of links that a protein node has to other nodes in the protein–protein interaction network and (ii) protein complexity, defined as the number of subunits in a protein complex.<sup><a reveal-id="DSQ019C8" data-open="DSQ019C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSQ019C9" data-open="DSQ019C9" class="link link-ref link-reveal xref-bibr">9</a></sup></p><p>It has been previously found that protein connectivity is negatively correlated with gene duplicability (generally defined as the number of paralogs per gene family) in the case of yeast. That is, highly connected proteins (hubs) have a lower rate of duplication compared with lowly connected proteins (non-hub).<sup><a reveal-id="DSQ019C10" data-open="DSQ019C10" class="link link-ref link-reveal xref-bibr">10</a></sup> However, this trend is opposite in the human case, where protein connectivity and gene duplicability are positively correlated.<sup><a reveal-id="DSQ019C8" data-open="DSQ019C8" class="link link-ref link-reveal xref-bibr">8</a></sup> The reason behind this difference is that due to higher protein dosage in yeast, retention of gene duplicates might have a deleterious effect on the organism. But, in the case of mammals, due to a higher rate of diversification in the function of the gene duplicates, it becomes more robust against dosage increase.<sup><a reveal-id="DSQ019C11" data-open="DSQ019C11" class="link link-ref link-reveal xref-bibr">11</a>,<a reveal-id="DSQ019C12" data-open="DSQ019C12" class="link link-ref link-reveal xref-bibr">12</a></sup></p><p>Protein complexity might also be another important factor that influences gene duplicability. According to their fates, duplicated genes found within protein complex networks belong to one of the three categories: (i) intra-complex paralogs that remain within the same protein complex, (ii) bi-complex paralog when resulting genes function within two separate complexes and (iii) overhang, when the resulting genes posses no general association with a known protein complex.<sup><a reveal-id="DSQ019C13" data-open="DSQ019C13" class="link link-ref link-reveal xref-bibr">13</a></sup> Duplication of a subunit in a protein complex might also cause dosage imbalance if rapid sub-functionalization or neo-functionalization does not occur to the newly arisen genes.<sup><a reveal-id="DSQ019C14" data-open="DSQ019C14" class="link link-ref link-reveal xref-bibr">14</a></sup> Therefore, duplication of a subunit of a protein complex is less likely to be successful than the duplication of a monomer.<sup><a reveal-id="DSQ019C14" data-open="DSQ019C14" class="link link-ref link-reveal xref-bibr">14</a></sup> The proportion of unduplicated genes is high (≥65%) for both monomers and multimers in yeast, whereas it is less (≤30%) in humans<sup><a reveal-id="DSQ019C9" data-open="DSQ019C9" class="link link-ref link-reveal xref-bibr">9</a></sup> since complex organisms are more robust against dosage increase. Detailed studies on yeast protein complexes showed that most of the participating subunits of multiprotein complexes are under tightly regulated gene dosage. These proteins are encoded mainly by haploinsufficient (wild type recessive) genes.<sup><a reveal-id="DSQ019C14" data-open="DSQ019C14" class="link link-ref link-reveal xref-bibr">14</a></sup> This is in agreement with the central prediction of the physiological theory which states that, genes encoding proteins whose functions tend to be insensitive to protein dosage should typically be haplosufficient (dominant wild type).<sup><a reveal-id="DSQ019C15" data-open="DSQ019C15" class="link link-ref link-reveal xref-bibr">15</a></sup> Eventually, human haploinsufficient genes have, on the average, more paralogs than haplosufficient ones because additional products of these genes probably lead to increased fitness.<sup><a reveal-id="DSQ019C15" data-open="DSQ019C15" class="link link-ref link-reveal xref-bibr">15</a></sup></p><p>Recently it has been observed that protein connectivity and protein complexity complement each other in guiding the evolutionary rates of human proteins.<sup><a reveal-id="DSQ019C16" data-open="DSQ019C16" class="link link-ref link-reveal xref-bibr">16</a></sup> Moreover, it has been reported that gene duplicability decreases with the increase in the number of subunits in protein complex, though this rate of decrease is fairly slow for humans compared with yeast.<sup><a reveal-id="DSQ019C8" data-open="DSQ019C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSQ019C9" data-open="DSQ019C9" class="link link-ref link-reveal xref-bibr">9</a></sup> In the present study, we investigated the interrelationship among the three features viz. gene duplicability, protein connectivity and protein complexity, in proteins that are present both in protein complexes network as well as in the protein–protein interaction network. Our analysis reveals that the human protein–protein interaction network has a larger influence on a gene to be duplicated. Moreover, we found that non-hub proteins in the complex are duplicated as well as hub proteins, indicating that protein complexity can also increase gene duplicability, in contrast to previous reports.<sup><a reveal-id="DSQ019C9" data-open="DSQ019C9" class="link link-ref link-reveal xref-bibr">9</a></sup> Analyses of haploinsufficient proteins of both humans and yeast suggest that organismal complexity (defined as a measure of the number of different cell types in an organism), as well as the higher rate of functional divergence of human protein complexes, has a large influence on higher duplicability.</p>                            <h2 scrollto-destination=5036661 id="5036661" class="section-title" data-legacy-id=s2>2 Materials and methods</h2>
                            <h3 scrollto-destination=5036668 id="5036668" class="section-title" data-legacy-id=s2a>2.1. Protein–protein interaction data</h3>
<p>All human protein–protein interaction data were taken from the Human Protein Reference Database (HPRD) release 7 (<a class="link link-uri" href="http://www.hprd.org/download">http://www.hprd.org/download</a>), which is essentially a database covering thousands of protein–protein interactions, posttranslational modifications, enzyme/substrates relationships, disease associations, tissue expression and subcellular localization. The data were extracted from the literature manually by biologists who read and interpreted &gt;300 000 published articles during the annotation process<sup><a reveal-id="DSQ019C17" data-open="DSQ019C17" class="link link-ref link-reveal xref-bibr">17</a></sup> where a total of 9386 unique protein interactors were found. Proteins with more than five interactions were considered as hub proteins and proteins with one or two interacting partners were considered as non-hub proteins.<sup><a reveal-id="DSQ019C16" data-open="DSQ019C16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="DSQ019C18" data-open="DSQ019C18" class="link link-ref link-reveal xref-bibr">18</a></sup> All yeast protein–protein interaction data were collected from Li <em>et al.</em>,<sup><a reveal-id="DSQ019C10" data-open="DSQ019C10" class="link link-ref link-reveal xref-bibr">10</a></sup> where protein–protein interaction pairs were collected from various high throughput experiments and databases. This collection was combined with another high throughput dataset of Bader <em>et al.</em><sup><a reveal-id="DSQ019C19" data-open="DSQ019C19" class="link link-ref link-reveal xref-bibr">19</a></sup> Subsequently, many small-scale experiments were performed to obtain a larger high confidence dataset.<sup><a reveal-id="DSQ019C10" data-open="DSQ019C10" class="link link-ref link-reveal xref-bibr">10</a></sup> This high confidence dataset (<a class="link link-uri" href="http://mbe.oxfordjournals.org/content/vol0/issue2005/images/data/msi249/DC1/msi249supp2.txt">http://mbe.oxfordjournals.org/content/vol0/issue2005/images/data/msi249/DC1/msi249supp2.txt</a>) is taken for our analysis of yeast proteins. Finally, we considered 1732 human (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>) and 1269 yeast proteins (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>) for which both protein–protein interaction data as well as protein complex data are available.</p>                            <h3 scrollto-destination=5036680 id="5036680" class="section-title" data-legacy-id=s2b>2.2. Identification of protein complexes</h3>
<p>We obtained the list of human protein complexes from <a class="link link-uri" href="http://mips.gsf.de/genre/proj/corum">http://mips.gsf.de/genre/proj/corum</a>. A total of 1345 number of protein complexes<sup><a reveal-id="DSQ019C20" data-open="DSQ019C20" class="link link-ref link-reveal xref-bibr">20</a></sup> were found (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). The protein complex data for yeast was collected from Gavin <em>et al</em>.<sup><a reveal-id="DSQ019C21" data-open="DSQ019C21" class="link link-ref link-reveal xref-bibr">21</a></sup> (<a class="link link-uri" href="http://www.nature.com/nature/journal/v440/n7084/extref/nature04532-s3.pdf">http://www.nature.com/nature/journal/v440/n7084/extref/nature04532-s3.pdf</a>; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>).</p><p>The protein complexes having &gt;10 subunits were classified as large complexes and complexes having less than 10 subunits were considered as small complex.</p>                            <h3 scrollto-destination=5036694 id="5036694" class="section-title" data-legacy-id=s2c>2.3. Identification of haploinsufficient genes</h3>
<p>Human haploinsufficient gene sequences were taken from Dang <em>et al</em>.<sup><a reveal-id="DSQ019C22" data-open="DSQ019C22" class="link link-ref link-reveal xref-bibr">22</a></sup> where haploinsufficient genes have mainly been collected from PubMed and OMIM search results with one of the major keyword being haploinsufficient and spurious hits were gradually removed by comparing against known human gene names and symbols<sup><a reveal-id="DSQ019C22" data-open="DSQ019C22" class="link link-ref link-reveal xref-bibr">22</a></sup> (<a class="link link-uri" href="http://www.nature.com/ejhg/journal/v16/n11/suppinfo/ejhg2008111s1.html?url=/ejhg/journal/v16/n11/abs/ejhg2008111a.html">http://www.nature.com/ejhg/journal/v16/n11/suppinfo/ejhg2008111s1.html?url=/ejhg/journal/v16/n11/abs/ejhg2008111a.html</a>, Yeast haploinsufficient genes are collected from Deutschbauer <em>et al</em>.<sup><a reveal-id="DSQ019C23" data-open="DSQ019C23" class="link link-ref link-reveal xref-bibr">23</a></sup> (<a class="link link-uri" href="http://www.genetics.org/cgi/content/full/genetics.104.036871/DC1">http://www.genetics.org/cgi/content/full/genetics.104.036871/DC1</a>). These haploinsufficient proteins are generally collected from yeast deletion strain experiments on YPD and minimal media.</p>                            <h3 scrollto-destination=5036701 id="5036701" class="section-title" data-legacy-id=s2d>2.4. Identification of paralogs</h3>
<p>The paralogs for both humans and yeast were taken from ensemble paralog database version 52. The general steps followed by the ensemble<sup><a reveal-id="DSQ019C24" data-open="DSQ019C24" class="link link-ref link-reveal xref-bibr">24</a></sup> for identifying paralogs sequences are based mainly on the construction of the gene tree reconciled with the species tree formed by the cluster of aligned sequences obtained from BLASTP. True paralogy for every pair of genes in the gene tree has been inferred by calculating the <em>d</em><sub>N</sub>/<em>d</em><sub>S</sub> ratio. This method not only enables the detection of true paralogs but also helps discard misleaded paralogs obtained from domain fusion. Finally, true paralog sets in humans were downloaded from the ensemble using 30% similarity between two sequences, and the alignable region between two sequences is &gt;80% of the longer protein.<sup><a reveal-id="DSQ019C19" data-open="DSQ019C19" class="link link-ref link-reveal xref-bibr">19</a></sup> Thus, a total set of 9609 number of duplicated proteins in humans and 1604 duplicated proteins in yeast were identified. Among these 9609 human paralogs only 664 genes (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a>) having paralogs are present both in the protein–protein interaction network as well as protein complex, network and in the case of yeast, 182 genes (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>) having paralog have been identified, present in both networks.</p>                            <h3 scrollto-destination=5036711 id="5036711" class="section-title" data-legacy-id=s2e>2.5. Statistical analysis</h3>
<p>All statistical analyses were performed by the software SPSS and TANAGRA. We measured Spearman rank correlation for both bivariate and partial correlations, as our data values were mostly repetitive, and Spearman rank correlations have been calculated after finding rank of the values. As a result, extreme variations in values have less control over the correlation. Simple bivariate correlation may be biased due to the presence of other factors and therefore Spearman's partial correlation coefficient was calculated after eliminating the confounded factors. Additionally, we performed Principal Component Analysis (PCA) through SPSS.</p>                            <h2 scrollto-destination=5036717 id="5036717" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=5036721 id="5036721" class="section-title" data-legacy-id=s3a>3.1. Protein connectivity, protein complexity and gene duplicability</h3>
<p>In order to examine how the gene duplicability in humans as well as in yeast are influenced by protein connectivity and protein complexity, we collected 1732 human and 1269 yeast proteins for which both protein–protein interaction data and protein complex annotation were available. Non-parametric Spearman's correlations for both protein connectivity and protein complexity with the gene duplicability of human genes reveal that, gene duplicability is negatively correlated (Spearman rank test; <em>R</em> = −0.111, <em>P</em> &lt; 0.001) with protein complexity and positively correlated (Spearman rank test; <em>R</em> = 0.169, <em>P</em> &lt; 0.001) with protein connectivity. In order to examine whether these two factors independently influence gene duplicability, we computed Spearman's partial correlation analysis between gene duplicability and one of the two factors by controlling the other. We observed that correlation between protein complexity and gene duplicability diminished drastically when protein connectivity was kept controlled (ρ = −0.080, <em>P</em> &lt; 0.001), whereas positive correlation existed between protein connectivity and gene duplicability when protein complexity was controlled (ρ = 0.150, <em>P</em> &lt; 0.001). The same trend was also observed by computing the Pearson partial correlation coefficient which showed that protein complexity did not have any correlation with gene duplicability when protein connectivity was controlled (ρ = 0.035, <em>P</em> = 7.6 × 10<sup>−2</sup>), but the effect of protein connectivity remained when protein complexity was controlled (ρ = 0.158, <em>P</em> &lt; 0.001). These results suggest that protein connectivity is the more important factor for gene duplicability in humans, although the effect of protein complexity over gene duplicability is guided mainly by protein connectivity. Moreover, a significant negative correlation (<em>R</em> = −0.201, <em>P</em> &lt; 0.001) was found between protein connectivity and protein complexity. This correlation holds even when gene duplicability is controlled (ρ = −0.186, <em>P</em> &lt; 0.001).</p><p>To test the independence of the sample points we removed those paralogs which are present in the protein complex and performed the same analysis. We obtained the same trend, that is, gene duplicability is negatively correlated with protein complexity (<em>R</em> = −0.121, <em>P</em> &lt; 0.001) whereas it is positively correlated with protein connectivity (<em>R</em> = 0.065, <em>P</em> = 4.7 × 10<sup>−2</sup>) and protein connectivity and protein complexity are negatively correlated with each other (<em>R</em> = −0.225, <em>P</em> &lt; 0.001). This indicates that shared gene duplicability does not have any bias on our analysis.</p><p>All the above analyses have been applied in the case of yeast genes. Non-parametric Spearman's correlation between protein complexity and gene duplicability revealed that gene duplicability is negatively correlated (<em>R</em> = −0.236, <em>P</em> &lt; 0.001) with protein complexity. The same trend (though with reduced correlation value) was observed when Spearman's correlation was determined between protein connectivity and gene duplicability (<em>R</em> = −0.074, <em>P</em> = 2.8 × 10<sup>−2</sup>). However, there is no significant correlation between protein connectivity and protein complexity. Spearman's partial correlation analysis demonstrates that both protein connectivity (ρ = −0.077, <em>P</em> = 2.37 × 10<sup>−3</sup>) and protein complexity (ρ = −0.237, <em>P</em> &lt; 0.001) affect gene duplicability independently. This has been further supported by Pearson partial correlation which also finds independent correlation for protein connectivity (ρ = −0.136, <em>P</em> &lt; 0.001) as well as for protein complexity (ρ = −0.172, <em>P</em> &lt; 0.001).</p><p>In order to evaluate how the position of a protein in the protein–protein interaction network influences gene duplicability, we correlated gene duplicability with the protein's centrality which is measured in terms of closeness (average number of nodes connecting a protein to all other proteins) and betweenness (measures the frequency with which a node lies on the shortest path between all other nodes) of a protein in the protein–protein interaction network.<sup><a reveal-id="DSQ019C26" data-open="DSQ019C26" class="link link-ref link-reveal xref-bibr">26</a></sup> In both the human and yeast genes, we found that protein connectivity is positively correlated with both these parameters (<em>R</em> = 0.791, <em>P</em> &lt; 0.001) and (<em>R</em> = 0.901, <em>P</em> &lt; 0.001), respectively, in case of humans and (<em>R</em> = 0.779, <em>P</em> &lt; 0.001) and (<em>R</em> = 0.560, <em>P</em> &lt; 0.001), respectively, in case of yeast. These results indicate that the relationship between protein connectivity and gene duplicability is in the same direction as that of protein's centrality, that is, closeness and betweenness (<em>R</em> = 0.220, <em>P</em> &lt; 0.001) and (<em>R</em> = 0.160, <em>P</em> &lt; 0.001), respectively, in case of humans and (<em>R</em> = −0.118, <em>P</em> = 9 × 10<sup>−3</sup>, <em>R</em> = −0.183, <em>P</em> &lt; 0.001), respectively, in case of yeast.</p><p>At this point, it is worthwhile to mention that the protein–protein interaction data come from a variety of methods and the percentage of interactions determined by various methods may differ between yeasts and human. Therefore, it is important to make sure that the differing trends are not simply due to different types of methods used in determining protein connectivity in protein–protein interaction network. In order to test this we have collected all the 7328 interaction data of humans as well as 4127 interaction data of yeast from yeast two-hybrid experiments of which 561 human proteins and 1032 yeast proteins are part of both protein complex and the protein–protein interaction network. We found the same trend when we performed our analysis on genes from yeast two-hybrid experiments (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S7</a>).</p><p>Moreover, it has been reported that human protein complexes are more robust against dosage than yeast protein complexes due to their high organismal complexity. Therefore, we have investigated the effect of gene dosage over human and yeast protein complexes.</p>                            <h3 scrollto-destination=5036743 id="5036743" class="section-title" data-legacy-id=s3b>3.2. Combinatorial effects of protein dosage, gene duplicability, protein connectivity and protein complexity</h3>
<p>The role of protein complex network on gene duplication can only be estimated by evaluating the known dose sensitive complex proteins, i.e. the haploinsufficient proteins<sup><a reveal-id="DSQ019C27" data-open="DSQ019C27" class="link link-ref link-reveal xref-bibr">27</a></sup> present in the protein complex. Haploinsufficiency occurs when a diploid organism only has a single functional copy of a gene (with the other copy inactivated by mutation) and the single functional copy of the gene does not produce enough gene product (typically a protein) to bring about a wild type condition. Studies show that mutation or loss of a single allele may be sufficient to exert diseased cellular phenotypes.<sup><a reveal-id="DSQ019C28" data-open="DSQ019C28" class="link link-ref link-reveal xref-bibr">28</a></sup> This gene dosage effect results in haploinsufficiency. Thus, haploinsufficient gene needs both of its alleles to be functional in order to express the wild type.</p><p>Analyzing 299 human haploinsufficint genes obtained from Dang <em>et al</em>.<sup><a reveal-id="DSQ019C22" data-open="DSQ019C22" class="link link-ref link-reveal xref-bibr">22</a></sup> it has been found that 59 proteins are present in both the protein complex and the protein–protein interaction network, of which 21 (36%) are duplicated. A detailed study of these proteins shows that protein connectivity is not at all correlated with gene duplicability whereas protein complexity has a positive correlation (<em>R</em> = 0.228, <em>P</em> = 3.5 × 10<sup>−2</sup>) with gene duplicability. Again we have also collected all the haploinsufficient data for humans from the OMIM to get a larger dataset. In that dataset where we have collected a total of 410 human haploinsufficient proteins of which 32 duplicated haploinsufficient proteins that are present in both the protein complex as well as in the protein–protein interaction network among the 83 (∼39%) proteins present in both protein–protein interaction as well as protein complex network. In this dataset also we have obtained the same result as the previous, that is, protein complexity is positively correlated (<em>R</em> = 0.213, <em>P</em> = 2.18 × 10<sup>−2</sup>) whereas no correlation exists between protein connectivity and gene duplicability (<em>R</em> = 0.008, <em>P</em> = 9.24 × 10<sup>−1</sup>). This trend has also been found in the protein–protein interaction dataset verified by the yeast two-hybrid experimental method where protein complexity is also positively correlated with gene duplicability (<em>R</em> = 0.247, <em>P</em> = 2.7 × 10<sup>−2</sup>). However, the trend is the opposite in the case of <em>Saccharomyces cerevisiae</em>, where out of 184 haploinsufficient proteins collected from Deutschbauer <em>et al</em>.,<sup><a reveal-id="DSQ019C23" data-open="DSQ019C23" class="link link-ref link-reveal xref-bibr">23</a></sup> 79 are found to be present in both the protein–protein interaction network as well as in the protein complex, of which only 12 (15%) are duplicated. After analysis it has been found that protein complexity is negatively correlated (<em>R</em> = −0.183, <em>P</em> = 2.8 × 10<sup>−2</sup>) with gene duplicability whereas protein connectivity is not correlated with gene duplicability (<em>R</em> = 0.0661, <em>P</em> = 4.32 × 10<sup>−1</sup>). The same trend has been observed for protein–protein interaction data obtained by the yeast two-hybrid experiment case of protein complexity (<em>R</em> = −0.154, <em>P</em> = 3.8 × 10<sup>−2</sup>).</p><p>But in both the cases of humans and yeast, the correlation between protein complexity and the number of paralogs per gene exists when connectivity is controlled (ρ = 0.247, <em>P</em> = 1.15 × 10<sup>−2</sup>) for human and (ρ = −0.1775, <em>P</em> = 3.4 × 10<sup>−2</sup>) for yeast whereas, protein connectivity does not have any significant correlation when complexity is controlled (ρ = 0.1917, <em>P</em> = 7.05 × 10<sup>−2</sup>). Partial correlation with our new data also suggest the same trend, that is, protein complexity is positively correlated (ρ = 0.231, <em>P</em> = 1.27 × 10<sup>−2</sup>) and protein connectivity does not have any correlation at all (ρ = 0.0941, <em>P</em> = 3.17 × 10<sup>−2</sup>). Even Pearson partial correlation also shows the positive trend of protein complexity towards gene duplicability even when protein connectivity is controlled (ρ = 0.068 <em>P</em> = 5.34 × 10<sup>−1</sup>). Moreover, a significant negative correlation (<em>R</em> = −0.269, <em>P</em> = 1.2 × 10<sup>−2</sup>) was found between protein connectivity and protein complexity in human haploinsufficient proteins which remains while gene duplicability is controlled (ρ = −0.307, <em>P</em> = 4.16 × 10<sup>−3</sup>). This negative correlation is even increased in the analysis with the new data (<em>R</em> = −0.361, <em>P</em> &lt; 0.001). The above results from humans suggest that, if haploinsufficient genes are present in the protein complex, protein connectivity is not the most influencing factor, but it helps protein complexity to play. However, the inverse correlation between gene duplicability and protein complexity in yeast haploinsufficient proteins, might be due to their differential organismal complexity between two organisms.<sup><a reveal-id="DSQ019C9" data-open="DSQ019C9" class="link link-ref link-reveal xref-bibr">9</a></sup></p><p>The data from the yeast haploinsufficient proteins supports the ‘balance hypotheses’ as well as the existence of strong dose sensitivity of this organism.<sup><a reveal-id="DSQ019C14" data-open="DSQ019C14" class="link link-ref link-reveal xref-bibr">14</a></sup></p><p>However, in the analysis of complex proteins without any haploinsufficient genes it has been observed that protein complexity is negatively correlated with gene duplicability in both yeast and humans as observed earlier (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S8</a>). This result suggests that in protein complexes where haploinsufficient genes are absent, the effect of protein connectivity within the protein–protein interaction network is more predominant than that of protein complexity on gene duplication.</p>                            <h3 scrollto-destination=5036756 id="5036756" class="section-title" data-legacy-id=s3c>3.3. Effect of protein connectivity, and protein complexity on gene duplicability in humans</h3>
<p>The differences in the correlations of the two different types of connectivities with gene duplicability prompted us to study the differences of additional parameters namely, protein complexity, protein connectivity and gene duplicability in these two groups of proteins (Table <a reveal-id="DSQ019TB1" data-open="DSQ019TB1" class="link link-reveal link-table xref-fig">1</a>). From the Table <a reveal-id="DSQ019TB1" data-open="DSQ019TB1" class="link link-reveal link-table xref-fig">1</a> it is evident that there is no significant difference in average protein connectivity as well as in average number of paralogs per gene in haploinsufficient proteins and in proteins where haploinsufficient proteins are absent. However, in one case protein connectivity is positively correlated (where haploinsufficint proteins are absent) and in the other case protein connectivity does not have any correlation (in haploinsufficient proteins present in both the protein–protein interaction network and protein complex network) with gene duplicability. At the same time the differences between complex sizes in these two groups are also apparent (Table <a reveal-id="DSQ019TB1" data-open="DSQ019TB1" class="link link-reveal link-table xref-fig">1</a>). But, in both the cases protein connectivity and protein complexity are negatively correlated. </p>                        <a id="5036760" scrollto-destination="5036760"></a>
<div content-id="DSQ019TB1" class="table-modal"><div class="table-wrap"><div id="DSQ019TB1" class="table-wrap-title" data-id="DSQ019TB1"><div class="label -label">Table 1</div><div class="caption"><p>Connectivity, complexity and average number of paralogs per gene across haploinsufficient proteins and after removal of haploinsufficient proteins in human</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Haploinsufficint proteins present in both the protein–protein interaction network and protein complex network </th><th>After removing haploinsufficient proteins from proteins present in both the protein–protein interaction network and protein complex network </th><th>Level of significance </th></tr></thead><tbody><tr><td>Average complexity </td><td>8.6511 </td><td>12.9321 </td><td><em>P</em> &lt; 0.0001 </td></tr><tr><td>Average interaction </td><td>26.3720 </td><td>27.2717 </td><td><em>P</em> = 0.261 </td></tr><tr><td>Average number of paralogs per gene </td><td>1.8720 </td><td>2.2857 </td><td><em>P</em> = 0.464 </td></tr></tbody></table></div><div content-id="DSQ019TB1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Haploinsufficint proteins present in both the protein–protein interaction network and protein complex network </th><th>After removing haploinsufficient proteins from proteins present in both the protein–protein interaction network and protein complex network </th><th>Level of significance </th></tr></thead><tbody><tr><td>Average complexity </td><td>8.6511 </td><td>12.9321 </td><td><em>P</em> &lt; 0.0001 </td></tr><tr><td>Average interaction </td><td>26.3720 </td><td>27.2717 </td><td><em>P</em> = 0.261 </td></tr><tr><td>Average number of paralogs per gene </td><td>1.8720 </td><td>2.2857 </td><td><em>P</em> = 0.464 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Note. Average complexity means average number of subunits per complex in a particular group of proteins and average interactions indicate average protein connectivity in the protein–protein interaction network of a particular group of proteins.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5036760" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSQ019TB1" class="table-wrap-title" data-id="DSQ019TB1"><div class="label -label">Table 1</div><div class="caption"><p>Connectivity, complexity and average number of paralogs per gene across haploinsufficient proteins and after removal of haploinsufficient proteins in human</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Haploinsufficint proteins present in both the protein–protein interaction network and protein complex network </th><th>After removing haploinsufficient proteins from proteins present in both the protein–protein interaction network and protein complex network </th><th>Level of significance </th></tr></thead><tbody><tr><td>Average complexity </td><td>8.6511 </td><td>12.9321 </td><td><em>P</em> &lt; 0.0001 </td></tr><tr><td>Average interaction </td><td>26.3720 </td><td>27.2717 </td><td><em>P</em> = 0.261 </td></tr><tr><td>Average number of paralogs per gene </td><td>1.8720 </td><td>2.2857 </td><td><em>P</em> = 0.464 </td></tr></tbody></table></div><div content-id="DSQ019TB1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Haploinsufficint proteins present in both the protein–protein interaction network and protein complex network </th><th>After removing haploinsufficient proteins from proteins present in both the protein–protein interaction network and protein complex network </th><th>Level of significance </th></tr></thead><tbody><tr><td>Average complexity </td><td>8.6511 </td><td>12.9321 </td><td><em>P</em> &lt; 0.0001 </td></tr><tr><td>Average interaction </td><td>26.3720 </td><td>27.2717 </td><td><em>P</em> = 0.261 </td></tr><tr><td>Average number of paralogs per gene </td><td>1.8720 </td><td>2.2857 </td><td><em>P</em> = 0.464 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Note. Average complexity means average number of subunits per complex in a particular group of proteins and average interactions indicate average protein connectivity in the protein–protein interaction network of a particular group of proteins.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5036760" target="_blank">View Large</a></div></div><p>Thus, when we have estimated the average number of paralogs for both highly connected (hub) (445 number of proteins) and lowly connected (non-hub) (96 number of proteins) proteins present in the protein complex (excluding 117 proteins which have an intermediate connectivity between 3 and 5), we found, surprisingly, that both hub and non-hub proteins present in protein complex show nearly equal (Mann–Whitney U-test; <em>P</em> = 0.128) average number of paralogs (Fig. <a reveal-id="DSQ019F1" data-open="DSQ019F1" class="link link-reveal link-table xref-fig">1</a>). Again we have also verified this trend by remodeling hub and non-hub data, that is, taking hub proteins of connectivity &gt;9 and non-hub proteins as connectivity 1 and we have obtained the same result, that is, both hub and non-hub proteins present in the complex show nearly the same gene duplicability (Mann–Whitney U-test; <em>P</em> = 0.533). But the significant difference in average number of paralogs between hub and non-hub proteins in both the methods reappear (Mann–Whitney U-test; <em>P</em> = 0.012, <em>P</em> = 0.015) after removing both of the hub and non-hub proteins present in the protein complex. These results suggest that protein connectivity influences more protein complexity that guides gene duplicability. But when this factor fails to correlate directly, then the other factor, i.e. protein complexity, plays its role. </p>                        <a id="5036762" scrollto-destination="5036762"></a>
<div id="DSQ019F1" data-id="DSQ019F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/5/10.1093/dnares/dsq019/2/m_dsq01901.jpeg?Expires=2147483647&Signature=ti1i6nlTV0l7dMT4K18MpsmEeSea0c~UxPN-9vhbSQ~2mh5CxnqKJq8~VWtIQgc~fi4EYLFsrGsMPxpChDmuCkR406BhrOmVNldEvtkYAtGPu3NZeOC217ShECjZUFneYd6CUathS-OCQCmDj4gh1DE00VsBiTGxK223dPv7g9gMOTGdIZoK8XgcICcNkuzU3ntyP4MVOZo3Uu4Evy5zS-19~Ij9DJ~xFlFNswsYe8sqUbb-uiQU4TzP~4gsTzCDCLiCZMuCNiLw2n6SYvSFFEvHt-HSjGN8n7iubs3vzEEEB2Cjy62aehZ9GX9ws0CXTzVuuAMHOpp2s0q2950PuQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Average number of paralogs per gene of highly connected (hub) and lowly connected (non-hub) protein present in the protein complex." path-from-xml="dsq01901.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5036762/dsq01901.jpeg" path-from-xml="dsq01901.jpeg" target="_blank">View large</a><a section="5036762" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/5/10.1093/dnares/dsq019/2/dsq01901.jpeg?Expires=2147483647&Signature=vsO~-HxfWp6PVoW0nlw2yKQedIxK97gdXG8CrqMEv4V~zQUg4Dlt89szWsxtnjBvqLSm8Z~rT0zxLjLhEqb~CQ4myKvLx~jNZg9hFW3QH91buXc0GPABEo8vIuZLGBogizM5MBcpM-P6eotTauLqnT26fvhAAAz4Tuxtw1suNv3CkbsQJRS-uo-GTVvloAI4yrSVvOh2yfWtLUNgzxsO7b9xJ~UhjCHzZ7TV6Ta2t8LjV1N0fDRA3JcdGieGxTQ7hFKQ60KG72-IgASS1OSZvclTjtTM~61xb8e5jwqVex77WajCgKd8F7vOfOASZtpDDXrk3OOsZrY0DbgCwIbLDw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5036762&ar=353723&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq01901.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Average number of paralogs per gene of highly connected (hub) and lowly connected (non-hub) protein present in the protein complex.</p></div></div><div content-id="DSQ019F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/5/10.1093/dnares/dsq019/2/m_dsq01901.jpeg?Expires=2147483647&Signature=ti1i6nlTV0l7dMT4K18MpsmEeSea0c~UxPN-9vhbSQ~2mh5CxnqKJq8~VWtIQgc~fi4EYLFsrGsMPxpChDmuCkR406BhrOmVNldEvtkYAtGPu3NZeOC217ShECjZUFneYd6CUathS-OCQCmDj4gh1DE00VsBiTGxK223dPv7g9gMOTGdIZoK8XgcICcNkuzU3ntyP4MVOZo3Uu4Evy5zS-19~Ij9DJ~xFlFNswsYe8sqUbb-uiQU4TzP~4gsTzCDCLiCZMuCNiLw2n6SYvSFFEvHt-HSjGN8n7iubs3vzEEEB2Cjy62aehZ9GX9ws0CXTzVuuAMHOpp2s0q2950PuQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Average number of paralogs per gene of highly connected (hub) and lowly connected (non-hub) protein present in the protein complex." path-from-xml="dsq01901.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5036762/dsq01901.jpeg" path-from-xml="dsq01901.jpeg" target="_blank">View large</a><a section="5036762" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/5/10.1093/dnares/dsq019/2/dsq01901.jpeg?Expires=2147483647&Signature=vsO~-HxfWp6PVoW0nlw2yKQedIxK97gdXG8CrqMEv4V~zQUg4Dlt89szWsxtnjBvqLSm8Z~rT0zxLjLhEqb~CQ4myKvLx~jNZg9hFW3QH91buXc0GPABEo8vIuZLGBogizM5MBcpM-P6eotTauLqnT26fvhAAAz4Tuxtw1suNv3CkbsQJRS-uo-GTVvloAI4yrSVvOh2yfWtLUNgzxsO7b9xJ~UhjCHzZ7TV6Ta2t8LjV1N0fDRA3JcdGieGxTQ7hFKQ60KG72-IgASS1OSZvclTjtTM~61xb8e5jwqVex77WajCgKd8F7vOfOASZtpDDXrk3OOsZrY0DbgCwIbLDw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5036762&ar=353723&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq01901.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Average number of paralogs per gene of highly connected (hub) and lowly connected (non-hub) protein present in the protein complex.</p></div></div><p>This explains the higher duplicability of non-hub proteins in the protein complex. In this case since these proteins are poorly connected, the protein's complexity guides the duplicability. Since most of the non-hub proteins are part a of large complex (average complexity 26.75–27) we aimed to analyze what exactly happens to protein connectivity and gene duplicability in large and small protein complexes and, interestingly, we found that there is a significant difference in average connectivity as well as average complex size between the large and small complexes [average connectivity of large complex is 18 and average connectivity of small complex is 32 (Mann–Whitney U-test; <em>P</em> &lt; 0.001] and the average complexity of the large complex is 30 and average complexity of the small complex is 5 (Mann–Whitney U-test; <em>P</em> &lt; 0.001)). However, there is no significant difference in the paralog number of the large and small complexes (average paralog number of large complex is 2.17 while average paralog number of the small complex is 2.31, respectively). From the above result it is evident that though there is a significant difference in protein connectivity between large and small protein complexes there exist no significant differences in average paralog numbers between these two groups of proteins. So, if protein connectivity is the most effective factor for the duplication of a gene, then the average paralog number of small complexes has to be greater than that of large complexes. But this is not the case. This indicates that there is a significant role of protein complexity over gene duplicability. The results indicate that protein complexity has a positive role in increasing gene duplicability. This result has further been validated by analyzing proteins present in the large complex (complex size is ≥10). In the large complex group, we surprisingly found that, both protein connectivity and protein complexity are positively correlated with gene duplicability. These correlations hold true even when both the connectivity and complexity are controlled separately (Table <a reveal-id="DSQ019TB2" data-open="DSQ019TB2" class="link link-reveal link-table xref-fig">2</a>). The actual distribution of gene duplicability with respect to protein complexity and protein connectivity in the case of human is shown in Fig. <a reveal-id="DSQ019F2" data-open="DSQ019F2" class="link link-reveal link-table xref-fig">2</a>. </p>                        <a id="5036764" scrollto-destination="5036764"></a>
<div content-id="DSQ019TB2" class="table-modal"><div class="table-wrap"><div id="DSQ019TB2" class="table-wrap-title" data-id="DSQ019TB2"><div class="label -label">Table 2</div><div class="caption"><p>Correlation between protein connectivity and gene duplicability as well as between gene duplicability and protein complexity in large complexes in human genes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Spearman rank correlation for Gene duplicability </th><th>Level of significance </th><th>Partial correlation for gene duplicability with control </th><th>Level of significance </th></tr></thead><tbody><tr><td>Protein complexity </td><td>0.150 </td><td><em>P</em> &lt; 0.001 </td><td>0.179 (Connectivity control) </td><td><em>P</em> &lt; 0.001 </td></tr><tr><td>Protein connectivity </td><td>0.075 </td><td><em>P</em> = 3.30 × 10<sup>−2</sup> </td><td>0.124 (Complexity control) </td><td><em>P</em> &lt; 0.001 </td></tr></tbody></table></div><div content-id="DSQ019TB2" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Spearman rank correlation for Gene duplicability </th><th>Level of significance </th><th>Partial correlation for gene duplicability with control </th><th>Level of significance </th></tr></thead><tbody><tr><td>Protein complexity </td><td>0.150 </td><td><em>P</em> &lt; 0.001 </td><td>0.179 (Connectivity control) </td><td><em>P</em> &lt; 0.001 </td></tr><tr><td>Protein connectivity </td><td>0.075 </td><td><em>P</em> = 3.30 × 10<sup>−2</sup> </td><td>0.124 (Complexity control) </td><td><em>P</em> &lt; 0.001 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5036764" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSQ019TB2" class="table-wrap-title" data-id="DSQ019TB2"><div class="label -label">Table 2</div><div class="caption"><p>Correlation between protein connectivity and gene duplicability as well as between gene duplicability and protein complexity in large complexes in human genes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Spearman rank correlation for Gene duplicability </th><th>Level of significance </th><th>Partial correlation for gene duplicability with control </th><th>Level of significance </th></tr></thead><tbody><tr><td>Protein complexity </td><td>0.150 </td><td><em>P</em> &lt; 0.001 </td><td>0.179 (Connectivity control) </td><td><em>P</em> &lt; 0.001 </td></tr><tr><td>Protein connectivity </td><td>0.075 </td><td><em>P</em> = 3.30 × 10<sup>−2</sup> </td><td>0.124 (Complexity control) </td><td><em>P</em> &lt; 0.001 </td></tr></tbody></table></div><div content-id="DSQ019TB2" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Spearman rank correlation for Gene duplicability </th><th>Level of significance </th><th>Partial correlation for gene duplicability with control </th><th>Level of significance </th></tr></thead><tbody><tr><td>Protein complexity </td><td>0.150 </td><td><em>P</em> &lt; 0.001 </td><td>0.179 (Connectivity control) </td><td><em>P</em> &lt; 0.001 </td></tr><tr><td>Protein connectivity </td><td>0.075 </td><td><em>P</em> = 3.30 × 10<sup>−2</sup> </td><td>0.124 (Complexity control) </td><td><em>P</em> &lt; 0.001 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5036764" target="_blank">View Large</a></div></div>                        <a id="5036765" scrollto-destination="5036765"></a>
<div id="DSQ019F2" data-id="DSQ019F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/5/10.1093/dnares/dsq019/2/m_dsq01902.jpeg?Expires=2147483647&Signature=2miP~AvZEs8fOKL0hOKVH7ip0G7TL4EwtJ~o~84BeGxMs4yncxWsX4I2ClXLq7qJ6dv5sM~MUQJNgFiUlS08xL9KAusVBK4GzEXBZFrpnBKGU4pBIwezlXN58KO26oG7JzaLUmMhGSYPRDtdOLMCyDbYLy2cVUaFvC7jxEexiAd2IT4gLoFX5KAp76pLF4KrsS3Fw4FnmSIpIJ1A1iIbCIFLj4o-FHKyoXtKMTCsaODsefuGiGjOhGKpNkpY-GxKJcYzGAIW4VaExHmz7eqa5cIU2CG4Dse4zlua1gIJpqnoP-2tMn1YOh4~rFZYf6BHDanMrIq8iyQsa4p1UYsvPQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The distribution showing the relation of gene duplicability in overall complex, large complex and protein connectivity in human." path-from-xml="dsq01902.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5036765/dsq01902.jpeg" path-from-xml="dsq01902.jpeg" target="_blank">View large</a><a section="5036765" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/5/10.1093/dnares/dsq019/2/dsq01902.jpeg?Expires=2147483647&Signature=f-18CdhRf7HZO74Q3GXhWQ3HKeyZnElEsg9Xb4eocFNsqwCMa~uQ2xEXjxXgzmvmdyUkt61HVpa2gqvCxG3jsdQL61up-saSAOwmhXodU3iXMmHQDr62b2sjY4-nowOPCPKI9Seh8hL60g0nwZ~XxYYv4xtUGRZWlns3n16oBdxPMuwq3yQEVlwA3B4C2vZJTdf5~DT0wOFd5tvGKeE6sUWOVdaSMQ~tZtc~oFVEIZ3U~uUwwBuhcC7agtjZV29RZZZSvFShI4hbYpTx4gY0NSJa72FwsMLcAXKpgtfzFpEUVjIaB0a0YAypj7UATxTCDi5LMpO2YmzwFqrsQm-9Sg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5036765&ar=353723&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq01902.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The distribution showing the relation of gene duplicability in overall complex, large complex and protein connectivity in human.</p></div></div><div content-id="DSQ019F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/5/10.1093/dnares/dsq019/2/m_dsq01902.jpeg?Expires=2147483647&Signature=2miP~AvZEs8fOKL0hOKVH7ip0G7TL4EwtJ~o~84BeGxMs4yncxWsX4I2ClXLq7qJ6dv5sM~MUQJNgFiUlS08xL9KAusVBK4GzEXBZFrpnBKGU4pBIwezlXN58KO26oG7JzaLUmMhGSYPRDtdOLMCyDbYLy2cVUaFvC7jxEexiAd2IT4gLoFX5KAp76pLF4KrsS3Fw4FnmSIpIJ1A1iIbCIFLj4o-FHKyoXtKMTCsaODsefuGiGjOhGKpNkpY-GxKJcYzGAIW4VaExHmz7eqa5cIU2CG4Dse4zlua1gIJpqnoP-2tMn1YOh4~rFZYf6BHDanMrIq8iyQsa4p1UYsvPQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The distribution showing the relation of gene duplicability in overall complex, large complex and protein connectivity in human." path-from-xml="dsq01902.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5036765/dsq01902.jpeg" path-from-xml="dsq01902.jpeg" target="_blank">View large</a><a section="5036765" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/5/10.1093/dnares/dsq019/2/dsq01902.jpeg?Expires=2147483647&Signature=f-18CdhRf7HZO74Q3GXhWQ3HKeyZnElEsg9Xb4eocFNsqwCMa~uQ2xEXjxXgzmvmdyUkt61HVpa2gqvCxG3jsdQL61up-saSAOwmhXodU3iXMmHQDr62b2sjY4-nowOPCPKI9Seh8hL60g0nwZ~XxYYv4xtUGRZWlns3n16oBdxPMuwq3yQEVlwA3B4C2vZJTdf5~DT0wOFd5tvGKeE6sUWOVdaSMQ~tZtc~oFVEIZ3U~uUwwBuhcC7agtjZV29RZZZSvFShI4hbYpTx4gY0NSJa72FwsMLcAXKpgtfzFpEUVjIaB0a0YAypj7UATxTCDi5LMpO2YmzwFqrsQm-9Sg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5036765&ar=353723&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq01902.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The distribution showing the relation of gene duplicability in overall complex, large complex and protein connectivity in human.</p></div></div><p>We used the PCA using a correlation coefficient matrix in order to disentangle the contributions of protein complexity and protein connectivity to gene duplicability. The dominant eigen vectors (taken as equal or &gt;1) that emerge from this analysis can be interpreted as the one of the contributors guiding gene duplication. The first principal component accounts for 59.37% of the total variance and both protein complexity and protein connectivity have equal contributions on this factor.</p><p>We have also checked the results for the protein–protein interaction data collected by the yeast two-hybrid experiment and we found that both protein connectivity and protein complexity are positively correlated with gene duplicability (<em>R</em> = 0.132, <em>P</em> &lt; 0.001; <em>R</em> = 0.208, <em>P</em> &lt; 0.001), respectively.</p><p>Increase in one component of protein complex can change the stoichiometry of other participating proteins of that complex and hence the net effect may be deleterious for that complex. Therefore, the fixation of the duplicate gene can only happen if rapid sub-functionalization or neo-functionalization can occur in the duplicated genes.</p><p>To find what actually happens to the duplicated genes of large complexes, we have analyzed two large complexes separately namely, Splicesome complex and Nop56p-associated pre-rRNA complex. It has been found that for the Spliceosome complex out of 148 subunits only 36 subunits are duplicated, i.e. 24.34% (36/148) subunits are duplicated which gives rise to total 47 unique number of paralogs, of which 18 remain in the complex and most of these proteins are bi-complex paralogs whereas, 14 out of these 18 proteins have a tendency to remain within the same complex. So, due to the duplication 61.70% of overhangs are produced from the Spliceosome complex (Table <a reveal-id="DSQ019TB3" data-open="DSQ019TB3" class="link link-reveal link-table xref-fig">3</a>). </p>                        <a id="5036770" scrollto-destination="5036770"></a>
<div content-id="DSQ019TB3" class="table-modal"><div class="table-wrap"><div id="DSQ019TB3" class="table-wrap-title" data-id="DSQ019TB3"><div class="label -label">Table 3</div><div class="caption"><p>Analysis of human Spliceosome, Nop56p-associated pre-rRNA and overall complex to estimate the percentage of overhang generated in each case</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Name </th><th>Total subunit </th><th>Total subunit duplicated </th><th>Number of paralogs produced </th><th>Percentage of overhang produced (%) </th></tr></thead><tbody><tr><td>Spliceosome </td><td>148 </td><td>36 </td><td>47 </td><td>61.70 </td></tr><tr><td>Nop56p-associated pre-rRNA </td><td>104 </td><td>50 </td><td>134 </td><td>89.6 </td></tr><tr><td>Overall complex </td><td>2106 </td><td>720 </td><td>1290 </td><td>71.78 </td></tr></tbody></table></div><div content-id="DSQ019TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Name </th><th>Total subunit </th><th>Total subunit duplicated </th><th>Number of paralogs produced </th><th>Percentage of overhang produced (%) </th></tr></thead><tbody><tr><td>Spliceosome </td><td>148 </td><td>36 </td><td>47 </td><td>61.70 </td></tr><tr><td>Nop56p-associated pre-rRNA </td><td>104 </td><td>50 </td><td>134 </td><td>89.6 </td></tr><tr><td>Overall complex </td><td>2106 </td><td>720 </td><td>1290 </td><td>71.78 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5036770" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSQ019TB3" class="table-wrap-title" data-id="DSQ019TB3"><div class="label -label">Table 3</div><div class="caption"><p>Analysis of human Spliceosome, Nop56p-associated pre-rRNA and overall complex to estimate the percentage of overhang generated in each case</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Name </th><th>Total subunit </th><th>Total subunit duplicated </th><th>Number of paralogs produced </th><th>Percentage of overhang produced (%) </th></tr></thead><tbody><tr><td>Spliceosome </td><td>148 </td><td>36 </td><td>47 </td><td>61.70 </td></tr><tr><td>Nop56p-associated pre-rRNA </td><td>104 </td><td>50 </td><td>134 </td><td>89.6 </td></tr><tr><td>Overall complex </td><td>2106 </td><td>720 </td><td>1290 </td><td>71.78 </td></tr></tbody></table></div><div content-id="DSQ019TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Name </th><th>Total subunit </th><th>Total subunit duplicated </th><th>Number of paralogs produced </th><th>Percentage of overhang produced (%) </th></tr></thead><tbody><tr><td>Spliceosome </td><td>148 </td><td>36 </td><td>47 </td><td>61.70 </td></tr><tr><td>Nop56p-associated pre-rRNA </td><td>104 </td><td>50 </td><td>134 </td><td>89.6 </td></tr><tr><td>Overall complex </td><td>2106 </td><td>720 </td><td>1290 </td><td>71.78 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5036770" target="_blank">View Large</a></div></div><p>In the case of Nop56p-associated pre-rRNA complex the same trend is followed. Out of 104 subunits of the complex 50 subunits are duplicated to give a total 134 unique number of paralogs, of which 14 are retained in the complex (4 retain in same complex). So only 10.44% paralogs are part of the protein complexes and the rest remain as overhang outside the complex (Table <a reveal-id="DSQ019TB3" data-open="DSQ019TB3" class="link link-reveal link-table xref-fig">3</a>).</p><p>Moreover, analysis of the duplicated genes of the whole human protein complex reveals that most of the duplicated complex proteins are overhangs (926/1290) × 100 = 71.78%, i.e. they remain outside the complex (Table <a reveal-id="DSQ019TB3" data-open="DSQ019TB3" class="link link-reveal link-table xref-fig">3</a>).</p><p>This result is just the opposite of yeast, where very few overhangs are produced.<sup><a reveal-id="DSQ019C13" data-open="DSQ019C13" class="link link-ref link-reveal xref-bibr">13</a></sup> This may be the effect of higher rate of functional divergence of duplicate genes in case of humans, which is not so rapid for a simple organism like yeast. Moreover, human protein complexes are more robust against dosage than yeast protein complexes.<sup><a reveal-id="DSQ019C9" data-open="DSQ019C9" class="link link-ref link-reveal xref-bibr">9</a></sup> All these results lead us to conclude that organismal complexity is the important factor which causes higher duplicability of mammalian genes sometimes by protein connectivity within protein–protein interaction network or by proteins belongingness to complexes.</p>                            <h2 scrollto-destination=5036774 id="5036774" class="section-title" data-legacy-id=s4>4. Discussions</h2>
<p>In our earlier studies, while investigating the role of evolutionary rate and intrinsic disorder on protein connectivity in the protein–protein interaction network as well as in protein complex assembly in humans, we have found that both protein connectivity and protein complexity complement each other in a mutually exclusive manner.<sup><a reveal-id="DSQ019C16" data-open="DSQ019C16" class="link link-ref link-reveal xref-bibr">16</a></sup> But, the role of protein connectivity in the protein–protein interaction network as well as in protein complex assembly on gene duplicability was yet to be understood. In earlier studies it has been reported that in the case of humans, gene duplicability is positively correlated with protein connectivity in the protein–protein interaction network, but in the case of yeast gene duplicability is negatively correlated with protein connectivity.<sup><a reveal-id="DSQ019C8" data-open="DSQ019C8" class="link link-ref link-reveal xref-bibr">8</a></sup> These may be the reasons for rapid functional divergence of the duplicate genes in the case of humans, which is not true in a simple organism like yeast as proposed by Yang <em>et al</em>. (2003)<sup><a reveal-id="DSQ019C9" data-open="DSQ019C9" class="link link-ref link-reveal xref-bibr">9</a></sup> where they have shown a strong positive correlation between proportion of duplicate gene pairs with the divergence of gene expression and both synonymous and non-synonymous divergence.<sup><a reveal-id="DSQ019C14" data-open="DSQ019C14" class="link link-ref link-reveal xref-bibr">14</a></sup> Higher duplicability of essential genes in the case of humans may promote organismal complexity, as gene essentiality is positively correlated with protein connectivity.<sup><a reveal-id="DSQ019C8" data-open="DSQ019C8" class="link link-ref link-reveal xref-bibr">8</a></sup> But, in the case of yeast, higher duplicability of less-important genes was found.<sup><a reveal-id="DSQ019C29" data-open="DSQ019C29" class="link link-ref link-reveal xref-bibr">29</a></sup> These may cause duplicability to be increased with protein connectivity in case of human. If this phenomenon is true for humans, then protein connectivity within protein complex can also promote higher gene duplicability in humans. We have obtained equal rate of duplication for both hub and non-hub proteins present in protein complex. From the higher duplicability of non-hub proteins we may say that protein's belongingness to a complex may also promote gene duplicability in the positive direction. But till date it has been concluded that the higher the belongingness in a complex, lesser is the duplication. This is true for both humans and yeast. This motivated us to study the combined effect of protein connectivity and protein complexity over gene duplicability in the case of humans. Our study reveals that when combined, although apparently it seems that protein connectivity has a significant positive correlation with gene duplicability and protein complexity has a significant negative correlation with gene duplicability, detailed studies show that protein connectivity has an immense effect on protein complexity and hence acts as a modulating factor for increasing gene duplicability. So when protein connectivity increases, protein complexity decreases, and both of these phenomena will lead to higher duplicability of a gene which is schematically explained further by Fig. <a reveal-id="DSQ019F3" data-open="DSQ019F3" class="link link-reveal link-table xref-fig">3</a>A. But, eventually what happens to that case where protein connectivity fails to control duplicability directly? That is, in the case of haploinsufficient genes which are present in the protein complex. It has been found that in case of human haploinsufficient genes protein complexity is positively correlated with gene duplicability whereas no significant correlation has been found with protein connectivity in the protein–protein interaction network. This signifies when the connectivity within the protein–protein interaction network is absent; then connectivity within the complex plays the same role of connectivity within protein–protein interaction network, which is schematically explained further by Fig. <a reveal-id="DSQ019F3" data-open="DSQ019F3" class="link link-reveal link-table xref-fig">3</a>B. This positive correlation becomes more prominent when we consider the paralogs which are produced from proteins of large complexes. </p>                        <a id="5036776" scrollto-destination="5036776"></a>
<div id="DSQ019F3" data-id="DSQ019F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/5/10.1093/dnares/dsq019/2/m_dsq01903.jpeg?Expires=2147483647&Signature=xU8BSZGItpMxfivTB07J7roA-SQ2s7yMBsUYO9wuUOaH1sfBDtOtmrMke3TI2YIpI7-z6xmraXAc1CLphw69XRsekSic87WgLd5uBMErI5X6mZxUEtIGut4qXmw3ask2sRZ6FmI6-bWLLIGPvZZB3A6SOxQiLunhEgvae-BrIWi1X0SVe6ZJku-xoDbOsLNvsdAnbYdv~-fQS3gv2XMgMnRYWFvh~W4Gx1k5WTjRziA~Ppb4apsALd84nyp4ljHvEPXeuCQobjt-82b-0qgT-f~028WLtywhS85q6w-a293pWOgmoMdwBXVLlThUmChsVkYFKf6gQq6TtaIgiU67UA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of relationship between protein connectivity, protein complexity over gene duplicability (A) when both are influencing gene duplication; (B) when only complexity influencing gene duplication." path-from-xml="dsq01903.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5036776/dsq01903.jpeg" path-from-xml="dsq01903.jpeg" target="_blank">View large</a><a section="5036776" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/5/10.1093/dnares/dsq019/2/dsq01903.jpeg?Expires=2147483647&Signature=MYgpZsmtgbhH-ak425mmIAO-pPetj4fCOYPhgsaynj69Oaw5ZP2dsBPUT1pNMpoKTtFfyuP20sTRgZq0fQy8Sp1Y5fZLKH6g8sNUFNQWmbUbzNxsWJMeBCz7-effAP9DjmLI-~NhsMcdg-zebGFgZo23Zf~qLufFkkT-2st0tOg9CgYxe1mAUoOlV9RLwnxF2L~4RVAbilOOs-mj3uIcxxa0Z6~r~T75gYpwm6IRKX-xoXAjY~RUFW9rEvOt0UPNcghna9b5vpPDQscX~NqDBfG~LS3TaQUQqAni5TeJcqIzvHGoRHbwz~cuk1~hQD~TZRYuglrhSmfIbiK8QOEOnw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5036776&ar=353723&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq01903.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of relationship between protein connectivity, protein complexity over gene duplicability (A) when both are influencing gene duplication; (B) when only complexity influencing gene duplication.</p></div></div><div content-id="DSQ019F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/5/10.1093/dnares/dsq019/2/m_dsq01903.jpeg?Expires=2147483647&Signature=xU8BSZGItpMxfivTB07J7roA-SQ2s7yMBsUYO9wuUOaH1sfBDtOtmrMke3TI2YIpI7-z6xmraXAc1CLphw69XRsekSic87WgLd5uBMErI5X6mZxUEtIGut4qXmw3ask2sRZ6FmI6-bWLLIGPvZZB3A6SOxQiLunhEgvae-BrIWi1X0SVe6ZJku-xoDbOsLNvsdAnbYdv~-fQS3gv2XMgMnRYWFvh~W4Gx1k5WTjRziA~Ppb4apsALd84nyp4ljHvEPXeuCQobjt-82b-0qgT-f~028WLtywhS85q6w-a293pWOgmoMdwBXVLlThUmChsVkYFKf6gQq6TtaIgiU67UA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of relationship between protein connectivity, protein complexity over gene duplicability (A) when both are influencing gene duplication; (B) when only complexity influencing gene duplication." path-from-xml="dsq01903.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5036776/dsq01903.jpeg" path-from-xml="dsq01903.jpeg" target="_blank">View large</a><a section="5036776" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/5/10.1093/dnares/dsq019/2/dsq01903.jpeg?Expires=2147483647&Signature=MYgpZsmtgbhH-ak425mmIAO-pPetj4fCOYPhgsaynj69Oaw5ZP2dsBPUT1pNMpoKTtFfyuP20sTRgZq0fQy8Sp1Y5fZLKH6g8sNUFNQWmbUbzNxsWJMeBCz7-effAP9DjmLI-~NhsMcdg-zebGFgZo23Zf~qLufFkkT-2st0tOg9CgYxe1mAUoOlV9RLwnxF2L~4RVAbilOOs-mj3uIcxxa0Z6~r~T75gYpwm6IRKX-xoXAjY~RUFW9rEvOt0UPNcghna9b5vpPDQscX~NqDBfG~LS3TaQUQqAni5TeJcqIzvHGoRHbwz~cuk1~hQD~TZRYuglrhSmfIbiK8QOEOnw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5036776&ar=353723&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq01903.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of relationship between protein connectivity, protein complexity over gene duplicability (A) when both are influencing gene duplication; (B) when only complexity influencing gene duplication.</p></div></div><p>This also confirms our earlier findings that non-hub proteins in the complex are as duplicated as hub proteins in the complex. Eventually, we have found that non-hub proteins are mostly part of large complexes. So, this also supports the idea that protein connectivity within the protein complex plays the same role as protein connectivity within the protein–protein interaction network, and the negative correlation enables them to work in a mutually exclusive manner. But, the theory can only be supported if the protein product produced from the participating gene of the complex undergoes the following fates. Firstly, if rapid functional divergence occurs to these proteins so that the duplicated protein is not part of the complex at all, then this would not affect the dosage of the participating complex. Secondly, if the duplicated protein enhances the topology of the complex which may enhance the total complex assembly, then the enhanced dosage of the particular gene would have no effect over the protein complex.<sup><a reveal-id="DSQ019C8" data-open="DSQ019C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSQ019C30" data-open="DSQ019C30" class="link link-ref link-reveal xref-bibr">30</a></sup> Thirdly, a multimer protein might tend to be involved in more functions than a monomer. So the rapid functional divergence of the multimer protein may promote functional divergence more rapidly. So, if the duplicated gene does undergo through the above fates, then the enhanced dosage of that particular gene may have a deleterious effect on the whole complex. Essentially, our analysis reveals that most of the proteins produced from complexes are not part of the complex and this higher rate of diversification is not found in case of yeast. That is why, in the case of yeast the higher the association within complex lesser is the duplicability.</p><p>So, in case of both humans and yeast, protein connectivity in the protein–protein interaction network as well as in the protein complex behaves in the same manner. But, in addition to this, in the case of human there exists a negative correlation between these two sorts of connectivities, which is absent in the case of yeast. In yeast these two connectivities act as two different forces that control in the same negative direction. But in case of a complex organism like humans the higher rate of duplication for a particular gene may be the cause of either connectivity in the protein–protein interaction network or perhaps connectivity within the protein complex. Hence, organismal complexity promotes protein complexity as well as protein connectivity to behave differently due to different dosage effect on these two organisms. In both the cases the higher divergence of the duplicated protein does not affect network assembly. This may be the effect of organismal complexity which has a major effect on gene duplicability.</p>                            <h2 scrollto-destination=5036779 id="5036779" class="section-title" data-legacy-id=s5>Supplementary Data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Data are available at www.dnaresearch.oxfordjournals.org.</a></p>                            <h2 scrollto-destination=5036781 id="5036781" class="section-title" data-legacy-id=s6>Funding</h2>
<p>Department of Biotechnology, Govt of India (Sanction no 102/IFD/SAN/PR-1860/2008-2009).</p>                            <h2 scrollto-destination=5036783 id="5036783" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank the Department of Biotechnology, Govt. of India for financial help. We are also thankful to Mr Sanjib K. Gupta and Mrs Sujata Roy for their technical help. We are grateful to Professor Pradip Kumar Parrack for his critical reading of the manuscript. We are thankful to two anonymous referees for their helpful comments in improving the manuscript.</p>                            <h2 scrollto-destination=5036787 id="5036787" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSQ019C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ019C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ohno</div> <div class="given-names">S.</div></div>. </div>, <div class="source">Evolution by Gene Duplication</div> , <div class="year">1970</div><div class="publisher-loc">Berlin</div><div class="publisher-name">Springer</div>pg. <div class="fpage">160</div> </div></div></div></div><div content-id="DSQ019C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ019C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Samonte</div> <div class="given-names">R.V.</div></div>,  <div class="name"><div class="surname">Eichler</div> <div class="given-names">E.E.</div></div>. </div><div class="article-title">Segmental duplications and the evolution of the primate genome</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="year">2002</div>, vol. <div class="volume">3</div> (pg. <div class="fpage">65</div>-<div class="lpage">72</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nrg705" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nrg705</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Segmental%20duplications%20and%20the%20evolution%20of%20the%20primate%20genome&amp;author=R.V.%20Samonte&amp;author=E.E.%20Eichler&amp;publication_year=2002&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=3&amp;pages=65-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg705" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg705" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg705"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11823792" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ019C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ019C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">W.H.</div></div>. </div>, <div class="source">Molecular Evolution</div> , <div class="year">1997</div><div class="publisher-loc">Sunderland, MA</div><div class="publisher-name">Sinauer Associates</div>pg. <div class="fpage">432</div> </div></div></div></div><div content-id="DSQ019C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ019C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wolfe</div> <div class="given-names">K.H.</div></div>,  <div class="name"><div class="surname">Shields</div> <div class="given-names">D.C.</div></div>. </div><div class="article-title">Molecular evidence for an ancient duplication of the entire yeast genome</div>, <div class="source">Nature</div> , <div class="year">1997</div>, vol. <div class="volume">387</div> (pg. <div class="fpage">708</div>-<div class="lpage">13</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/42711" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/42711</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20evidence%20for%20an%20ancient%20duplication%20of%20the%20entire%20yeast%20genome&amp;author=K.H.%20Wolfe&amp;author=D.C.%20Shields&amp;publication_year=1997&amp;journal=Nature&amp;volume=387&amp;pages=708-13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/42711" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F42711" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F42711"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9192896" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ019C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ019C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kellis</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Patterson</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Endrizzi</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Birren</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Lander</div> <div class="given-names">E.S.</div></div>. </div><div class="article-title">Sequencing and comparison of yeast species to identify genes and regulatory elements</div>, <div class="source">Nature</div> , <div class="year">2003</div>, vol. <div class="volume">423</div> (pg. <div class="fpage">241</div>-<div class="lpage">54</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nature01644" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nature01644</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequencing%20and%20comparison%20of%20yeast%20species%20to%20identify%20genes%20and%20regulatory%20elements&amp;author=M.%20Kellis&amp;author=N.%20Patterson&amp;author=M.%20Endrizzi&amp;author=B.%20Birren&amp;author=E.S.%20Lander&amp;publication_year=2003&amp;journal=Nature&amp;volume=423&amp;pages=241-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature01644" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature01644" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature01644"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12748633" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ019C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ019C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nadeau</div> <div class="given-names">J.H.</div></div>,  <div class="name"><div class="surname">Sankoff</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Comparable rates of gene loss and functional divergence after genome duplications early in vertebrate evolution</div>, <div class="source">Genetics</div> , <div class="year">1997</div>, vol. <div class="volume">147</div> (pg. <div class="fpage">1259</div>-<div class="lpage">66</div>)<div class="pub-id">
                [<a href="http://www.ncbi.nlm.nih.gov/pubmed/9383068" class="link link-pub-id" target="_blank">PubMed</a>]
            </div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparable%20rates%20of%20gene%20loss%20and%20functional%20divergence%20after%20genome%20duplications%20early%20in%20vertebrate%20evolution&amp;author=J.H.%20Nadeau&amp;author=D.%20Sankoff&amp;publication_year=1997&amp;journal=Genetics&amp;volume=147&amp;pages=1259-66" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9383068" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ019C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ019C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">W.H.</div></div>,  <div class="name"><div class="surname">Gu</div> <div class="given-names">Z.L.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">H.D.</div></div>,  <div class="name"><div class="surname">Nekrutenko</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Evolutionary analyses of the human genome</div>, <div class="source">Nature</div> , <div class="year">2001</div>, vol. <div class="volume">409</div> (pg. <div class="fpage">847</div>-<div class="lpage">9</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/35057039" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/35057039</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolutionary%20analyses%20of%20the%20human%20genome&amp;author=W.H.%20Li&amp;author=Z.L.%20Gu&amp;author=H.D.%20Wang&amp;author=A.%20Nekrutenko&amp;publication_year=2001&amp;journal=Nature&amp;volume=409&amp;pages=847-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/35057039" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F35057039" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F35057039"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11237007" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ019C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ019C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Liang</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">W.H.</div></div>. </div><div class="article-title">Gene essentiality, gene duplicability and protein connectivity in human and mouse</div>, <div class="source">Trends Genet.</div> , <div class="year">2007</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">375</div>-<div class="lpage">8</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.tig.2007.04.005" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.tig.2007.04.005</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20essentiality%2C%20gene%20duplicability%20and%20protein%20connectivity%20in%20human%20and%20mouse&amp;author=H.%20Liang&amp;author=W.H.%20Li&amp;publication_year=2007&amp;journal=Trends%20Genet.&amp;volume=23&amp;pages=375-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tig.2007.04.005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tig.2007.04.005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tig.2007.04.005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17512629" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ019C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ019C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yang</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Lusk</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">W.H.</div></div>. </div><div class="article-title">Organismal complexity, protein complexity, and gene duplicability</div>, <div class="source">Proc. Natl. Acad Sci. USA</div> , <div class="year">2003</div>, vol. <div class="volume">100</div> (pg. <div class="fpage">15661</div>-<div class="lpage">5</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1073/pnas.2536672100" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1073/pnas.2536672100</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Organismal%20complexity%2C%20protein%20complexity%2C%20and%20gene%20duplicability&amp;author=J.%20Yang&amp;author=R.%20Lusk&amp;author=W.H.%20Li&amp;publication_year=2003&amp;journal=Proc.%20Natl.%20Acad%20Sci.%20USA&amp;volume=100&amp;pages=15661-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.2536672100" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.2536672100" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.2536672100"> </span></div> </div></div></div></div></div><div content-id="DSQ019C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ019C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Prachumwat</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">W.H.</div></div>. </div><div class="article-title">Protein function, connectivity, and duplicability in yeast</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2006</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">30</div>-<div class="lpage">9</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/molbev/msi249" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/molbev/msi249</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Protein%20function%2C%20connectivity%2C%20and%20duplicability%20in%20yeast&amp;author=A.%20Prachumwat&amp;author=W.H.%20Li&amp;publication_year=2006&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=23&amp;pages=30-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msi249" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsi249" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsi249"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16120800" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ019C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ019C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Qian</div> <div class="given-names">W.F.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">J.Z.</div></div>. </div><div class="article-title">Gene dosage and gene duplicability</div>, <div class="source">Genetics</div> , <div class="year">2008</div>, vol. <div class="volume">179</div> (pg. <div class="fpage">2319</div>-<div class="lpage">24</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1534/genetics.108.090936" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1534/genetics.108.090936</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20dosage%20and%20gene%20duplicability&amp;author=W.F.%20Qian&amp;author=J.Z.%20Zhang&amp;publication_year=2008&amp;journal=Genetics&amp;volume=179&amp;pages=2319-24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.108.090936" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.108.090936" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.108.090936"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18689880" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ019C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ019C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kondrashov</div> <div class="given-names">F.A.</div></div>,  <div class="name"><div class="surname">Koonin</div> <div class="given-names">E.V.</div></div>. </div><div class="article-title">A common framework for understanding the origin of genetic dominance and evolutionary fates of gene duplications</div>, <div class="source">Trends Genet.</div> , <div class="year">2004</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">287</div>-<div class="lpage">91</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.tig.2004.05.001" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.tig.2004.05.001</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20common%20framework%20for%20understanding%20the%20origin%20of%20genetic%20dominance%20and%20evolutionary%20fates%20of%20gene%20duplications&amp;author=F.A.%20Kondrashov&amp;author=E.V.%20Koonin&amp;publication_year=2004&amp;journal=Trends%20Genet.&amp;volume=20&amp;pages=287-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tig.2004.05.001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tig.2004.05.001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tig.2004.05.001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15219392" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ019C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ019C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Papp</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Pal</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Hurst</div> <div class="given-names">L.D.</div></div>. </div><div class="article-title">Dosage sensitivity and the evolution of gene families in yeast</div>, <div class="source">Nature</div> , <div class="year">2003</div>, vol. <div class="volume">424</div> (pg. <div class="fpage">194</div>-<div class="lpage">7</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nature01771" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nature01771</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dosage%20sensitivity%20and%20the%20evolution%20of%20gene%20families%20in%20yeast&amp;author=B.%20Papp&amp;author=C.%20Pal&amp;author=L.D.%20Hurst&amp;publication_year=2003&amp;journal=Nature&amp;volume=424&amp;pages=194-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature01771" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature01771" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature01771"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12853957" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ019C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ019C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Makova</div> <div class="given-names">K.D.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">W.H.</div></div>. </div><div class="article-title">Divergence in the spatial pattern of gene expression between human duplicate genes</div>, <div class="source">Genome Res.</div> , <div class="year">2003</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">1638</div>-<div class="lpage">45</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1101/gr.1133803" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1101/gr.1133803</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Divergence%20in%20the%20spatial%20pattern%20of%20gene%20expression%20between%20human%20duplicate%20genes&amp;author=K.D.%20Makova&amp;author=W.H.%20Li&amp;publication_year=2003&amp;journal=Genome%20Res.&amp;volume=13&amp;pages=1638-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.1133803" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.1133803" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.1133803"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12840042" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ019C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ019C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Prince</div> <div class="given-names">V.E.</div></div>,  <div class="name"><div class="surname">Pickett</div> <div class="given-names">F.B.</div></div>. </div><div class="article-title">Splitting pairs: the diverging fates of duplicated genes</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="year">2002</div>, vol. <div class="volume">3</div> (pg. <div class="fpage">827</div>-<div class="lpage">37</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nrg928" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nrg928</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Splitting%20pairs%3A%20the%20diverging%20fates%20of%20duplicated%20genes&amp;author=V.E.%20Prince&amp;author=F.B.%20Pickett&amp;publication_year=2002&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=3&amp;pages=827-37" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg928" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg928" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg928"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12415313" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ019C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ019C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Manna</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Bhattacharya</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Kahali</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Ghosh</div> <div class="given-names">T.C.</div></div>. </div><div class="article-title">Evolutionary constraints on hub and non-hub proteins in human protein interaction network: insight from protein connectivity and intrinsic disorder</div>, <div class="source">Gene</div> , <div class="year">2009</div>, vol. <div class="volume">434</div> (pg. <div class="fpage">50</div>-<div class="lpage">5</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.gene.2008.12.013" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.gene.2008.12.013</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolutionary%20constraints%20on%20hub%20and%20non-hub%20proteins%20in%20human%20protein%20interaction%20network%3A%20insight%20from%20protein%20connectivity%20and%20intrinsic%20disorder&amp;author=B.%20Manna&amp;author=T.%20Bhattacharya&amp;author=B.%20Kahali&amp;author=T.C.%20Ghosh&amp;publication_year=2009&amp;journal=Gene&amp;volume=434&amp;pages=50-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gene.2008.12.013" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gene.2008.12.013" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gene.2008.12.013"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19185053" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ019C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ019C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Keshava Prasad</div> <div class="given-names">T.S.</div></div>,  <div class="name"><div class="surname">Goel</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Kandasamy</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">Human Protein Reference Database–2009 update</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2009</div>, vol. <div class="volume">37</div> (pg. <div class="fpage">767</div>-<div class="lpage">72</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Human%20Protein%20Reference%20Database%E2%80%932009%20update&amp;author=T.S.%20Keshava%20Prasad&amp;author=R.%20Goel&amp;author=K.%20Kandasamy&amp;publication_year=2009&amp;journal=Nucleic%20Acids%20Res.&amp;volume=37&amp;pages=767-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkn892" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkn892" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkn892"> </span></div> </div></div></div></div></div><div content-id="DSQ019C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ019C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ekman</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Light</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Bjorklund</div> <div class="given-names">A.K.</div></div>,  <div class="name"><div class="surname">Elofsson</div> <div class="given-names">A.</div></div>. </div><div class="article-title">What properties characterize the hub proteins of the protein–protein interaction network of <em>Saccharomyces cerevisiae</em>?</div>, <div class="source">Genome Biol.</div> , <div class="year">2006</div>, vol. <div class="volume">7</div> pg. <div class="fpage">R45</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/gb-2006-7-6-r45" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/gb-2006-7-6-r45</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=What%20properties%20characterize%20the%20hub%20proteins%20of%20the%20protein%E2%80%93protein%20interaction%20network%20of%20Saccharomyces%20cerevisiae%3F&amp;author=D.%20Ekman&amp;author=S.%20Light&amp;author=A.K.%20Bjorklund&amp;author=A.%20Elofsson&amp;publication_year=2006&amp;journal=Genome%20Biol.&amp;volume=7&amp;pages=R45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2006-7-6-r45" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2006-7-6-r45" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2006-7-6-r45"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16780599" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ019C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ019C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bader</div> <div class="given-names">J.S.</div></div>,  <div class="name"><div class="surname">Chaudhuri</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Rothberg</div> <div class="given-names">J.M.</div></div>,  <div class="name"><div class="surname">Chant</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Gaining confidence in high throughput protein interaction networks</div>, <div class="source">Nat. Biotechnol.</div> , <div class="year">2004</div>, vol. <div class="volume">22</div> (pg. <div class="fpage">78</div>-<div class="lpage">85</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nbt924" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nbt924</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gaining%20confidence%20in%20high%20throughput%20protein%20interaction%20networks&amp;author=J.S.%20Bader&amp;author=A.%20Chaudhuri&amp;author=J.M.%20Rothberg&amp;author=J.%20Chant&amp;publication_year=2004&amp;journal=Nat.%20Biotechnol.&amp;volume=22&amp;pages=78-85" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt924" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt924" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt924"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14704708" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ019C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ019C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ruepp</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Brauner</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Dunger-Kaltenbach</div> <div class="given-names">I.</div></div>, et al. </div><div class="article-title">CORUM: the comprehensive resource of mammalian protein complexes</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2008</div>, vol. <div class="volume">36</div> (pg. <div class="fpage">D646</div>-<div class="lpage">D650</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/gkm936" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/gkm936</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CORUM%3A%20the%20comprehensive%20resource%20of%20mammalian%20protein%20complexes&amp;author=A.%20Ruepp&amp;author=B.%20Brauner&amp;author=I.%20Dunger-Kaltenbach&amp;publication_year=2008&amp;journal=Nucleic%20Acids%20Res.&amp;volume=36&amp;pages=D646-D650" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkm936" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkm936" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkm936"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17965090" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ019C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ019C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gavin</div> <div class="given-names">A.C.</div></div>,  <div class="name"><div class="surname">Aloy</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Grandi</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Krause</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Boesche</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Proteome survey reveals modularity of the yeast cell machinery</div>, <div class="source">Nature</div> , <div class="year">2006</div>, vol. <div class="volume">440</div> (pg. <div class="fpage">631</div>-<div class="lpage">6</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nature04532" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nature04532</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Proteome%20survey%20reveals%20modularity%20of%20the%20yeast%20cell%20machinery&amp;author=A.C.%20Gavin&amp;author=P.%20Aloy&amp;author=P.%20Grandi&amp;author=R.%20Krause&amp;author=M.%20Boesche&amp;publication_year=2006&amp;journal=Nature&amp;volume=440&amp;pages=631-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature04532" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature04532" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature04532"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16429126" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ019C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ019C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dang</div> <div class="given-names">V.T.</div></div>,  <div class="name"><div class="surname">Kassahn</div> <div class="given-names">K.S.</div></div>,  <div class="name"><div class="surname">Marcos</div> <div class="given-names">A.E.</div></div>,  <div class="name"><div class="surname">Ragan</div> <div class="given-names">M.A.</div></div>. </div><div class="article-title">Identification of human haploinsufficient genes and their genomic proximity to segmental duplications</div>, <div class="source">Eur. J. Hum. Genet.</div> , <div class="year">2008</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">1350</div>-<div class="lpage">7</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/ejhg.2008.111" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/ejhg.2008.111</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20human%20haploinsufficient%20genes%20and%20their%20genomic%20proximity%20to%20segmental%20duplications&amp;author=V.T.%20Dang&amp;author=K.S.%20Kassahn&amp;author=A.E.%20Marcos&amp;author=M.A.%20Ragan&amp;publication_year=2008&amp;journal=Eur.%20J.%20Hum.%20Genet.&amp;volume=16&amp;pages=1350-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ejhg.2008.111" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fejhg.2008.111" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fejhg.2008.111"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18523451" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ019C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ019C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Deutschbauer</div> <div class="given-names">A.M.</div></div>,  <div class="name"><div class="surname">Jaramillo</div> <div class="given-names">D.F.</div></div>,  <div class="name"><div class="surname">Proctor</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Mechanisms of haploinsufficiency revealed by genome-wide profiling in yeast</div>, <div class="source">Genetics</div> , <div class="year">2005</div>, vol. <div class="volume">169</div> (pg. <div class="fpage">1915</div>-<div class="lpage">25</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1534/genetics.104.036871" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1534/genetics.104.036871</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mechanisms%20of%20haploinsufficiency%20revealed%20by%20genome-wide%20profiling%20in%20yeast&amp;author=A.M.%20Deutschbauer&amp;author=D.F.%20Jaramillo&amp;author=M.%20Proctor&amp;publication_year=2005&amp;journal=Genetics&amp;volume=169&amp;pages=1915-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.104.036871" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.104.036871" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.104.036871"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15716499" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ019C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ019C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Vilella</div> <div class="given-names">A.J.</div></div>,  <div class="name"><div class="surname">Severin</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Ureta-Vidal</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">EnsemblCompara GeneTrees: Complete, duplication-aware phylogenetic trees in vertebrates</div>, <div class="source">Genome Res.</div> , <div class="year">2009</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">327</div>-<div class="lpage">35</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1101/gr.073585.107" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1101/gr.073585.107</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=EnsemblCompara%20GeneTrees%3A%20Complete%2C%20duplication-aware%20phylogenetic%20trees%20in%20vertebrates&amp;author=A.J.%20Vilella&amp;author=J.%20Severin&amp;author=A.%20Ureta-Vidal&amp;publication_year=2009&amp;journal=Genome%20Res.&amp;volume=19&amp;pages=327-35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.073585.107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.073585.107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.073585.107"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19029536" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ019C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ019C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">W.H.</div></div>,  <div class="name"><div class="surname">Gu</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Cavalcanti</div> <div class="given-names">A.R.</div></div>,  <div class="name"><div class="surname">Nekrutenko</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Detection of gene duplications and block duplications in eukaryotic genomes</div>, <div class="source">J Struct. Funct. Genomics</div> , <div class="year">2003</div>, vol. <div class="volume">3</div> (pg. <div class="fpage">27</div>-<div class="lpage">34</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1023/A:1022644628861" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1023/A:1022644628861</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Detection%20of%20gene%20duplications%20and%20block%20duplications%20in%20eukaryotic%20genomes&amp;author=W.H.%20Li&amp;author=Z.%20Gu&amp;author=A.R.%20Cavalcanti&amp;author=A.%20Nekrutenko&amp;publication_year=2003&amp;journal=J%20Struct.%20Funct.%20Genomics&amp;volume=3&amp;pages=27-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1023/A:1022644628861" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1023%2FA:1022644628861" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1023%2FA:1022644628861"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12836682" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ019C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ019C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hahn</div> <div class="given-names">M.W.</div></div>,  <div class="name"><div class="surname">Kern</div> <div class="given-names">A.D.</div></div>. </div><div class="article-title">Comparative genomics of centrality and essentiality in three eukaryotic protein-interaction networks</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2005</div>, vol. <div class="volume">22</div> (pg. <div class="fpage">803</div>-<div class="lpage">6</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/molbev/msi072" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/molbev/msi072</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20genomics%20of%20centrality%20and%20essentiality%20in%20three%20eukaryotic%20protein-interaction%20networks&amp;author=M.W.%20Hahn&amp;author=A.D.%20Kern&amp;publication_year=2005&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=22&amp;pages=803-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msi072" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsi072" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsi072"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15616139" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ019C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ019C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Veitia</div> <div class="given-names">R.A.</div></div>. </div><div class="article-title">Exploring the etiology of haploinsufficiency</div>, <div class="source">Bioessays</div> , <div class="year">2002</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">175</div>-<div class="lpage">84</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1002/bies.10023" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1002/bies.10023</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Exploring%20the%20etiology%20of%20haploinsufficiency&amp;author=R.A.%20Veitia&amp;publication_year=2002&amp;journal=Bioessays&amp;volume=24&amp;pages=175-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1521-1878" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1521-1878" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1521-1878"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11835282" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ019C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ019C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fodde</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Smits</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Cancer biology. A matter of dosage</div>, <div class="source">Science</div> , <div class="year">2002</div>, vol. <div class="volume">298</div> (pg. <div class="fpage">761</div>-<div class="lpage">3</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1126/science.1077707" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1126/science.1077707</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cancer%20biology.%20A%20matter%20of%20dosage&amp;author=R.%20Fodde&amp;author=R.%20Smits&amp;publication_year=2002&amp;journal=Science&amp;volume=298&amp;pages=761-3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1077707" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1077707" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1077707"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12399571" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ019C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ019C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">He</div> <div class="given-names">X.L.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">J.Z.</div></div>. </div><div class="article-title">Higher duplicability of less important genes in yeast genomes</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2006</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">144</div>-<div class="lpage">51</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/molbev/msj015" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/molbev/msj015</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Higher%20duplicability%20of%20less%20important%20genes%20in%20yeast%20genomes&amp;author=X.L.%20He&amp;author=J.Z.%20Zhang&amp;publication_year=2006&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=23&amp;pages=144-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msj015" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsj015" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsj015"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16151181" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ019C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ019C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Oberdorf</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Kortemme</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Complex topology rather than complex membership is a determinant of protein dosage sensitivity</div>, <div class="source">Mol. Syst. Biol.</div> , <div class="year">2009</div>, vol. <div class="volume">5</div> pg. <div class="fpage">253</div> <div class="pub-id">
                [<a href="http://www.ncbi.nlm.nih.gov/pubmed/19293832" class="link link-pub-id" target="_blank">PubMed</a>]
            </div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complex%20topology%20rather%20than%20complex%20membership%20is%20a%20determinant%20of%20protein%20dosage%20sensitivity&amp;author=R.%20Oberdorf&amp;author=T.%20Kortemme&amp;publication_year=2009&amp;journal=Mol.%20Syst.%20Biol.&amp;volume=5&amp;pages=253" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/msb.2009.9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fmsb.2009.9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fmsb.2009.9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19293832" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div><div content-id=""><div class="fn"><div class="footnote-content">Edited by Osamu Ohara</div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2010. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/2.5), which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=353723&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/5/10.1093/dnares/dsq019/2/dsq019-dsq019supp_table8.pdf?Expires=2147483647&amp;Signature=idL2t62NtIy7LUCv6cl1jNRu6~oxVrvq7QFt8QJQUdlSilsDSv8HTJcmGJHfgDuOICTR~yUTA9L1XoQvJ-GZCkidoO3DH13g0i3Hl2NWBxMCrCyjbaBeD7EThzWPH6BPAdd676RZ8lobWPkZz4oUMdDQXymhavttVNqRMoBlcrmYVywea2nnaIjM1bfSt2IM~GsNsbmf5h2ZGNDJnXQFYzojYgqdAQLd2emuKDA7wQNqDw40FPpLkMeyBmxftpm8~A4EqFjR4E3xl5TA3yiw7i09ArbpLFJwoJNiQe2bEM2jGmdfGa2AoT0TuPFwRpBrnPSlEXioTer~gsQFVnfmrg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-8</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/5/10.1093/dnares/dsq019/2/dsq019-dsq019supp_table7.pdf?Expires=2147483647&amp;Signature=IoUaSFxGhie2CqYBAg7ym-xzOKjCLk6iB70bEs2FFlFOifLZUkGRW2LisPd5uSChhLurXljfEf8WgRdMQ74WUOR9Bns5RT79vCjUv4l5s659qbtje-qqs0n5jK1nMlMVNcHnf~ctKrWeoFmtblXBLT3Dl7njV3LRyiSobK0NRdAcLqwuMjT5GqvWQ-JvzxCwfykaPjwjg4iYUBmnpxEYmG22JtkKEcllD4WN7rgDjfQnRhPUgWqLvUuM1ikyiC24hT2lw51tSElHDqTsoKQIQt3-Y9l~BYnSZLdNQcAZry4UXSpknzqmbovvHuiMPqQc2oBngvnsAVbzrQI~2fUwAQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-7</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/5/10.1093/dnares/dsq019/2/dsq019-dsq019supp_table6.xls?Expires=2147483647&amp;Signature=xWpKl7b-Ka8Ix39Ryn5sHnIBRYwABE1zWK0~OZ~-j8DN2ujwZKqW9URJaDEchbAPedOqlhTxqAn-8v8iTHiyiAjs6EHu-ATBMOXNMow52oM0A57LAs3ayAKVzOqkqh3Wr6mR4sjbx8uuX4YyZMuCpfPKeFyvrM1eEGK7F02NuTtgxnKLUolRStsRbYHzBDcpQnOp~LXVY-aK5KwYFuWKEj~Ma7AV6w9wTScEtxwpbHXzVV0Enojl3bQJRn9zsE4W1YEYpMUEI-VJ3X5RptSc955cQ7x5gsrfVJp2Igt39z69SgfwucpXlpJbF0TPVRWkw9w6D-QqIa7GSINU8VsYWQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-6</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/5/10.1093/dnares/dsq019/2/dsq019-dsq019supp_table5.xls?Expires=2147483647&amp;Signature=LW~XpS9XX4BE0ZHNqAmoIe4sSaW3-wGdrrJ0E5l4hw5RLwMURTR2Z7FOtmxj6rAUZclA1-foYVMQrNNePHFlGr8zlYdhsPKYSBNquF2F4X1FgLmwb3bRcn2~2xbFiWassNFk6NQHbDz1AzQ-hr-L-ErAFuenC-7z4uZxUOmbtJsEyccGT1uzddphV5UolQEmRYd4u8Os0t1cQ1pkDUSCNeyD4LRDLuc5Ic9lEmcO--VgU-wY7p823wv1zu7Lm9LD7jk7yPbtXXWRxzxRwTrJhW~ITWbgQ1VQDcFwoPaql0HzLgSXa6BwjvrEhkOOHWx9Jpdx2Ovn7YhGHli0MTQyfQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-5</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/5/10.1093/dnares/dsq019/2/dsq019-dsq019supp_table4.xls?Expires=2147483647&amp;Signature=K8cUd6qGRouXiTJ3tZPULhLnpJVIC6i2c1U8zlGnCp309BPFTcLt9AOfiMGRU1bDT8nqzkb7to1vFJsorUURg-V5CuYW5E8a5WpQVatzWM2JbgSI-Kpg9cBAPv6nZy9pPqL8vDmHQnojcixjzu-wSKXqkX8BWbtNmspzcdiUSFIr-I7zSB4XEfttg2NWiqowYyjn13uT2Ue9sqOUgnHAHLl4k~NsBFFbDaXUemWdGq~w7WB5WB9~DI2wju4xtDpcgQNi9pSU4iSuENY2E0M3av~ZNLeo59Po4zYyd-MMwIfYOgrAYX7gC4Ib7O04E7p9PxyUbjYJKxZ2qx78nat5lg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-4</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/5/10.1093/dnares/dsq019/2/dsq019-dsq019supp_table3.xls?Expires=2147483647&amp;Signature=fD408QGNWd-PVx-yIWy9~UaAdGHqYJ6qZS7nI-NKQENBTfDLMTuES1YnPap2NF3AqrdRs3BlLK1q0rbEgNk~rvYtU2j9hxBwouJpSlshHbIE4QCgso-ql2rTUnDx-xhcftKsSgflRw1Kqax56mKWnvAK0lZNuUdwbIF7WTzKW3gV1ujrC-fGOQgixcO92WBwhRetLGbqR1aywfi0ozmsXU8vNYbAbWvT-fN-98ZEnjSSP4EoutL3gy8bAO~d1LyrXNqzeaYOFD3rAsGzODtLbEdmsUbTRRIHGKnItlRhHlln42DJSrB-HAJRzR6~5cOTrXGeZL-CtmOBgIu4NLwTzw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/5/10.1093/dnares/dsq019/2/dsq019-dsq019supp_table2.xls?Expires=2147483647&amp;Signature=vg0FQqIWRRt82JF0n9svcGWeQXksodxRsyvunZjw0NkxcKEL33xe8wyzr2KMKizwK8nmZtyMbRH022a-nzHLSnXijll0GNsDYVZZQIMnCKgqs0OUN6emgEgRxBWaeyS37xchbrfjpEIyzlOcIbupUpLCb1U3fPMXkDOE0j7g5UJO6XNsM6x0DqHhBPQ8Dzn-3AC8T3MmckdfIQbecnFPAFPOfP1uuxSJke6uZkkDFU-dHOkouJ4CoFMNrB4wXGfuON4FJuBgk3Ehh5ObbGEvF5r16ccQb22tY-JpUbKIrZIye6TyDtln8Xn67rSTxiQ705WcIz18hsV6JTtAGpbh6g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/5/10.1093/dnares/dsq019/2/dsq019-dsq019supp_table1.xls?Expires=2147483647&amp;Signature=pRWF-wNPwusPiN~1vZRU0Z2oxOk9hmh0vPrGXzjjgkN6TgAzeFofP2gC1IsGyoRsDa1LRqy7O3PpRxj2mC43OMqo1gjl-JkYLk5uWH32noebSB-lCCfUtnI2BnHnL88WEzNLzkCuExMSNiOe4OkoTfpEgHisDfiogz9URcBO-XTN313jEXJQCC8BkPKd3gBFCmneMO6iLRX9XI-2txrDRxxFjSi8t4bGefvYbhK3jVbOGUYs~EECDFHiqZ694DeGZjR9~FdtFvhLVNF-JqY5NW8dkIJ5C5zzr0nyR3W3dyAGVA89gJgOwnB8Vm6XXarRS4oWuKC-M52ZGK8RQOx9ow__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - xls file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">123</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000283631300001&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">2</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsq019"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="353723"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000283631300001&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f17%2f5%2f261%2f353723">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000283631300001&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (2)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F17%2F5%2F261%2F353723" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/353723'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Protein Connectivity and Protein Complexity Promotes Human Gene Duplicability in a Mutually Exclusive Manner" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsq019");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f17%2f5%2f261%2f353723">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

