

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/22/6/495/2462510","@type":"ScholarlyArticle","name":"Identification of minimal eukaryotic introns through GeneBase, a user-friendly tool for parsing the NCBI Gene databank","datePublished":"2015-11-17","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/22/6","@type":"PublicationIssue","issueNumber":"6","datePublished":"2015-12-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsv028","keywords":["NCBI Gene","gene data parsing","computational biology","minimal intron","personal computer software"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/22/6/495/2462510","author":[{"name":"Piovesan, Allison","affiliation":"1  Department of Experimental, Diagnostic and Specialty Medicine (DIMES), Unit of Histology, Embryology and Applied Biology ,  University of Bologna ,  Bologna, BO 40126 ,  Italy","@type":"Person"},{"name":"Caracausi, Maria","affiliation":"1  Department of Experimental, Diagnostic and Specialty Medicine (DIMES), Unit of Histology, Embryology and Applied Biology ,  University of Bologna ,  Bologna, BO 40126 ,  Italy","@type":"Person"},{"name":"Ricci, Marco","affiliation":"2  Department of Biological, Geological and Environmental Sciences (BIGeA) ,  University of Bologna ,  Bologna, BO 40126 ,  Italy","@type":"Person"},{"name":"Strippoli, Pierluigi","affiliation":"1  Department of Experimental, Diagnostic and Specialty Medicine (DIMES), Unit of Histology, Embryology and Applied Biology ,  University of Bologna ,  Bologna, BO 40126 ,  Italy","@type":"Person"},{"name":"Vitale, Lorenza","affiliation":"1  Department of Experimental, Diagnostic and Specialty Medicine (DIMES), Unit of Histology, Embryology and Applied Biology ,  University of Bologna ,  Bologna, BO 40126 ,  Italy","@type":"Person"},{"name":"Pelleri, Maria Chiara","affiliation":"1  Department of Experimental, Diagnostic and Specialty Medicine (DIMES), Unit of Histology, Embryology and Applied Biology ,  University of Bologna ,  Bologna, BO 40126 ,  Italy","@type":"Person"}],"description":"Abstract.  We have developed GeneBase, a full parser of the National Center for Biotechnology Information (NCBI) Gene database, which generates a fully structur","pageStart":"495","pageEnd":"503","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/6/1/m_cover.gif?Expires=2147483647&Signature=083aE4wPido26a6~EOoRT3Nb4fPpp5HV1MZzrjjay94a3evfhfRTvESRMhc7ROqyA2UIz2umFef89XL3Lu7hUZFod2Ix~QtkT4CP~8MDkYY61O0I1KLQQcYNpXVti4-0uwuVQWc0Ik7X46YCFzJAu2cTqwHZ2B0OBVFR-OZWKIwha86xHbqGB8R5L83t0zl8dqx1CkCPppI95mus6SUAdvZ0-wLyrVhsfEErb~4S7v8v0rjWvz1OZ1A2b6wWGyo1cjX3sRRxlgjkSlblmiowk2fs-R4agAPGw2bGiz2-0n1EnfsgKN8CSyFRqrRfdmccMFKJr5L5jh5C1s7L6OJB0g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Identification of minimal eukaryotic introns through GeneBase, a user-friendly tool for parsing the NCBI Gene databank","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/6/1/m_cover.gif?Expires=2147483647&Signature=083aE4wPido26a6~EOoRT3Nb4fPpp5HV1MZzrjjay94a3evfhfRTvESRMhc7ROqyA2UIz2umFef89XL3Lu7hUZFod2Ix~QtkT4CP~8MDkYY61O0I1KLQQcYNpXVti4-0uwuVQWc0Ik7X46YCFzJAu2cTqwHZ2B0OBVFR-OZWKIwha86xHbqGB8R5L83t0zl8dqx1CkCPppI95mus6SUAdvZ0-wLyrVhsfEErb~4S7v8v0rjWvz1OZ1A2b6wWGyo1cjX3sRRxlgjkSlblmiowk2fs-R4agAPGw2bGiz2-0n1EnfsgKN8CSyFRqrRfdmccMFKJr5L5jh5C1s7L6OJB0g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Identification of minimal eukaryotic introns through GeneBase, a user-friendly tool for parsing the NCBI Gene databank" />
    <meta property="og:description" content="Abstract.  We have developed GeneBase, a full parser of the National Center for Biotechnology Information (NCBI) Gene database, which generates a fully structur" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/22/6/495/2462510" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2015-11-17" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Piovesan, Allison" /><meta name="citation_author_institution" content="Department of Experimental, Diagnostic and Specialty Medicine (DIMES), Unit of Histology, Embryology and Applied Biology, University of Bologna, Bologna, BO 40126, Italy" /><meta name="citation_author" content="Caracausi, Maria" /><meta name="citation_author_institution" content="Department of Experimental, Diagnostic and Specialty Medicine (DIMES), Unit of Histology, Embryology and Applied Biology, University of Bologna, Bologna, BO 40126, Italy" /><meta name="citation_author" content="Ricci, Marco" /><meta name="citation_author_institution" content="Department of Biological, Geological and Environmental Sciences (BIGeA), University of Bologna, Bologna, BO 40126, Italy" /><meta name="citation_author" content="Strippoli, Pierluigi" /><meta name="citation_author_institution" content="Department of Experimental, Diagnostic and Specialty Medicine (DIMES), Unit of Histology, Embryology and Applied Biology, University of Bologna, Bologna, BO 40126, Italy" /><meta name="citation_author" content="Vitale, Lorenza" /><meta name="citation_author_institution" content="Department of Experimental, Diagnostic and Specialty Medicine (DIMES), Unit of Histology, Embryology and Applied Biology, University of Bologna, Bologna, BO 40126, Italy" /><meta name="citation_author" content="Pelleri, Maria Chiara" /><meta name="citation_author_institution" content="Department of Experimental, Diagnostic and Specialty Medicine (DIMES), Unit of Histology, Embryology and Applied Biology, University of Bologna, Bologna, BO 40126, Italy" /><meta name="citation_title" content="Identification of minimal eukaryotic introns through GeneBase, a user-friendly tool for parsing the NCBI Gene databank" /><meta name="citation_firstpage" content="495" /><meta name="citation_lastpage" content="503" /><meta name="citation_doi" content="10.1093/dnares/dsv028" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="22" /><meta name="citation_issue" content="6" /><meta name="citation_publication_date" content="2015/12/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=The history of the genetic sequence databases;  citation_journal_title=Genomics;  citation_year=1990;  citation_volume=6;  citation_pages=701-7; " /><meta name="citation_reference" content="citation_title=From private data to public knowledge; citation_publisher=Portland Press Ltd, London, UK;  citation_year=1997; " /><meta name="citation_reference" content="citation_title=Gene: a gene-centered information resource at NCBI;  citation_journal_title=Nucleic Acids Res.;  citation_year=2014;  citation_volume=43;  citation_pages=D36-42; " /><meta name="citation_reference" content="citation_title=Text-based database searching;  citation_journal_title=TRENDS Biotechnol.;  citation_year=1998;  citation_volume=16;  citation_pages=3-5; " /><meta name="citation_reference" content="citation_title=SORTEZ: a relational translator for NCBI's ASN.1 database;  citation_journal_title=Comput. Appl. Biosci.;  citation_year=1994;  citation_volume=10;  citation_pages=369-78; " /><meta name="citation_reference" content="citation_title=Fast parsers for Entrez Gene;  citation_journal_title=Bioinformatics;  citation_year=2005;  citation_volume=21;  citation_pages=3189-90; " /><meta name="citation_reference" content="citation_title=Atlas—a data warehouse for integrative bioinformatics;  citation_journal_title=BMC Bioinformatics;  citation_year=2005;  citation_volume=6;  citation_pages=34" /><meta name="citation_reference" content="citation_title=BIOSPIDA: a relational database translator for NCBI;  citation_journal_title=AMIA Annu. Symp. Proc.;  citation_year=2010;  citation_volume=20,10;  citation_pages=422-6; " /><meta name="citation_reference" content="citation_title=Intron-exon structures of eukaryotic model organisms;  citation_journal_title=Nucleic Acids Res.;  citation_year=1999;  citation_volume=27;  citation_pages=3219-28; " /><meta name="citation_reference" content="citation_title=Systematic analysis of intron size and abundance parameters in diverse lineages;  citation_journal_title=Sci. China. Life Sci.;  citation_year=2013;  citation_volume=56;  citation_pages=968-74; " /><meta name="citation_reference" content="citation_title=A computational analysis of sequence features involved in recognition of short introns;  citation_journal_title=Proc. Natl Acad. Sci. USA;  citation_year=2001;  citation_volume=98;  citation_pages=11193-8; " /><meta name="citation_reference" content="citation_title=Minimal introns are not ‘junk;  citation_journal_title=Genome Res.;  citation_year=2002;  citation_volume=12;  citation_pages=1185-9; " /><meta name="citation_reference" content="citation_title=Information content of Caenorhabditis elegans splice site sequences varies with intron length;  citation_journal_title=Nucleic Acids Res.;  citation_year=1990;  citation_volume=18;  citation_pages=1509-12; " /><meta name="citation_reference" content="citation_title=Splicing signals in Drosophila: intron size, information content, and consensus sequences;  citation_journal_title=Nucleic Acids Res.;  citation_year=1992;  citation_volume=20;  citation_pages=4255-62; " /><meta name="citation_reference" content="citation_title=A novel role for minimal introns: routing mRNAs to the cytosol;  citation_journal_title=PLoS One;  citation_year=2010;  citation_volume=5;  citation_pages=e10144" /><meta name="citation_reference" content="citation_publisher=Garland Science, New York;  citation_title=Human molecular genetics;  citation_year=2010; " /><meta name="citation_reference" content="citation_title=Mammalian introns: when the junk generates molecular diversity;  citation_journal_title=Int. J. Mol. Sci.;  citation_year=2015;  citation_volume=16;  citation_pages=4429-52; " /><meta name="citation_reference" content="citation_title=UniGene Tabulator: a full parser for the UniGene format;  citation_journal_title=Bioinformatics;  citation_year=2006;  citation_volume=22;  citation_pages=2570-1; " /><meta name="citation_reference" content="citation_title=TRAM (Transcriptome Mapper): database-driven creation and analysis of transcriptome maps from multiple sources;  citation_journal_title=BMC Genomics;  citation_year=2011;  citation_volume=12;  citation_pages=121" /><meta name="citation_reference" content="citation_title=Database resources of the National Center for Biotechnology Information;  citation_journal_title=Nucleic Acids Res.;  citation_year=2015;  citation_volume=43;  citation_pages=D6-17; " /><meta name="citation_reference" content="citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs;  citation_journal_title=Nucleic Acids Res.;  citation_year=1997;  citation_volume=25;  citation_pages=3389-402; " /><meta name="citation_reference" content="citation_title=Gene Ontology annotations and resources; citation_author=Gene Ontology Consortium;  citation_journal_title=Nucleic Acids Res.;  citation_year=2013;  citation_volume=41;  citation_pages=D530-5; " /><meta name="citation_reference" content="citation_title=Analysis of canonical and non-canonical splice sites in mammalian genomes;  citation_journal_title=Nucleic Acids Res.;  citation_year=2000;  citation_volume=28;  citation_pages=4364-75; " /><meta name="citation_reference" content="citation_publisher=Garland Science, New York;  citation_title=Molecular biology of the cell;  citation_year=2014; " /><meta name="citation_reference" content="citation_title=Identification and validation of evolutionarily conserved unusually short pre-mRNA introns in the human genome;  citation_journal_title=Int. J. Mol. Sci.;  citation_year=2015;  citation_volume=16;  citation_pages=10376-88; " /><meta name="citation_reference" content="citation_title=Genome-scale analysis of human mRNA 5′ coding sequences based on expressed sequence tag (EST) database;  citation_journal_title=Genomics;  citation_year=2012;  citation_volume=100;  citation_pages=125-30; " /><meta name="citation_reference" content="citation_title=Improving mRNA 5′ coding sequence determination in the mouse genome;  citation_journal_title=Mamm. Genome;  citation_year=2014;  citation_volume=25;  citation_pages=149-59; " /><meta name="citation_reference" content="citation_title=SRS: information retrieval system for molecular biology data banks;  citation_journal_title=Methods Enzymol.;  citation_year=1996;  citation_volume=266;  citation_pages=114-28; " /><meta name="citation_reference" content="citation_title=GeneRecords: a relational database for GenBank flat file parsing and data manipulation in personal computers;  citation_journal_title=Bioinformatics;  citation_year=2004;  citation_volume=20;  citation_pages=2883-5; " /><meta name="citation_reference" content="citation_title=Statistical features of human exons and their flanking regions;  citation_journal_title=Hum. Mol. Genet.;  citation_year=1998;  citation_volume=7;  citation_pages=919-32; " /><meta name="citation_reference" content="citation_title=Truncating mutations in the last exon of NOTCH2 cause a rare skeletal disorder with osteoporosis;  citation_journal_title=Nat. Genet.;  citation_year=2011;  citation_volume=43;  citation_pages=306-8; " /><meta name="citation_reference" content="citation_title=EID: the Exon-Intron Database-an exhaustive database of protein-coding intron-containing genes;  citation_journal_title=Nucleic Acids Res.;  citation_year=2000;  citation_volume=28;  citation_pages=185-90; " /><meta name="citation_reference" content="citation_title=IE-Kb: intron exon knowledge base;  citation_journal_title=Bioinformatics;  citation_year=2000;  citation_volume=16;  citation_pages=1151-2; " /><meta name="citation_reference" content="citation_title=The IDB and IEDB: intron sequence and evolution databases;  citation_journal_title=Nucleic Acids Res.;  citation_year=2000;  citation_volume=28;  citation_pages=181-4; " /><meta name="citation_reference" content="citation_title=ExInt: an exon intron database;  citation_journal_title=Nucleic Acids Res.;  citation_year=2002;  citation_volume=30;  citation_pages=191-4; " /><meta name="citation_reference" content="citation_title=SEGE: a database on ‘intron less/single exonic’ genes from eukaryotes;  citation_journal_title=Bioinformatics;  citation_year=2002;  citation_volume=18;  citation_pages=1266-7; " /><meta name="citation_reference" content="citation_title=Xpro: database of eukaryotic protein-encoding genes;  citation_journal_title=Nucleic Acids Res.;  citation_year=2004;  citation_volume=32;  citation_pages=D59-63; " /><meta name="citation_reference" content="citation_title=Advances in the exon-intron database (EID);  citation_journal_title=Brief Bioinform.;  citation_year=2006;  citation_volume=7;  citation_pages=178-85; " /><meta name="citation_reference" content="citation_title=Mechanistic insights into human pre-mRNA splicing of human ultra-short introns: potential unusual mechanism identifies G-rich introns;  citation_journal_title=Biochem. Biophys. Res. Commun.;  citation_year=2012;  citation_volume=423;  citation_pages=289-94; " /><meta name="citation_reference" content="citation_title=Genome-wide bioinformatic and molecular analysis of introns in Saccharomyces cerevisiae;  citation_journal_title=RNA;  citation_year=1999;  citation_volume=5;  citation_pages=221-34; " /><meta name="citation_reference" content="citation_title=The Arabidopsis Information Resource (TAIR): a comprehensive database and web-based information retrieval, analysis, and visualization system for a model plant;  citation_journal_title=Nucleic Acids Res.;  citation_year=2001;  citation_volume=29;  citation_pages=102-5; " /><meta name="citation_reference" content="citation_title=Splicing of precursors to mRNAs by the spliceosomes; citation_publisher=Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY;  citation_year=1999;  citation_pages=525-60; " /><meta name="citation_reference" content="citation_title=The GENCODE pseudogene resource;  citation_journal_title=Genome Biol.;  citation_year=2012;  citation_volume=13;  citation_pages=R51" /><meta name="citation_reference" content="citation_title=TassDB2—a comprehensive database of subtle alternative splicing events;  citation_journal_title=BMC Bioinformatics;  citation_year=2010;  citation_volume=11;  citation_pages=216" /><meta name="citation_reference" content="citation_title=Widespread occurrence of alternative splicing at NAGNAG acceptors contributes to proteome plasticity;  citation_journal_title=Nat. Genet.;  citation_year=2004;  citation_volume=36;  citation_pages=1255-7; " /><meta name="citation_reference" content="citation_title=Differential expression of alternatively spliced mRNA forms of the insulin-like growth factor 1 receptor in human neuroendocrine tumors;  citation_journal_title=Oncol. Rep.;  citation_year=2006;  citation_volume=15;  citation_pages=1249-56; " /><meta name="citation_reference" content="citation_title=Sequence, ‘subtle’ alternative splicing and expression of the CYYR1 (cysteine/tyrosine-rich 1) mRNA in human neuroendocrine tumors;  citation_journal_title=BMC Cancer;  citation_year=2007;  citation_volume=7;  citation_pages=66" /><meta name="citation_reference" content="citation_title=Noisy splicing drives mRNA isoform diversity in human cells;  citation_journal_title=PLoS Genet.;  citation_year=2010;  citation_volume=6;  citation_pages=e1001236" /><meta name="citation_reference" content="citation_title=Alternative splicing at GYNNGY 5′ splice sites: more noise, less regulation;  citation_journal_title=Nucleic Acids Res.;  citation_year=2014;  citation_volume=42;  citation_pages=13969-80; " /><meta name="citation_reference" content="citation_title=In silico prediction of splice-altering single nucleotide variants in the human genome;  citation_journal_title=Nucleic Acids Res.;  citation_year=2014;  citation_volume=42;  citation_pages=13534-44; " /><meta name="citation_reference" content="citation_title=A comprehensive survey of non-canonical splice sites in the human transcriptome;  citation_journal_title=Nucleic Acids Res.;  citation_year=2014;  citation_volume=42;  citation_pages=10564-78; " /><meta name="citation_reference" content="citation_title=Computational identification of functional introns: high positional conservation of introns that harbor RNA genes;  citation_journal_title=Nucleic Acids Res.;  citation_year=2013;  citation_volume=41;  citation_pages=5604-13; " /><meta name="citation_reference" content="citation_title=Selection for short introns in highly expressed genes;  citation_journal_title=Nat. Genet.;  citation_year=2002;  citation_volume=31;  citation_pages=415-8; " /><meta name="citation_reference" content="citation_title=Intron size in mammals: complexity comes to terms with economy;  citation_journal_title=Trends Genet.;  citation_year=2007;  citation_volume=23;  citation_pages=20-4; " /><meta name="citation_reference" content="citation_title=Evolution of gene structure in the conifer Picea glauca: a comparative analysis of the impact of intron size;  citation_journal_title=BMC Plant Biol.;  citation_year=2014;  citation_volume=14;  citation_pages=95" /><meta name="citation_reference" content="citation_title=Universal tight correlation of codon bias and pool of RNA codons (codonome): the genome is optimized to allow any distribution of gene expression values in the transcriptome from bacteria to humans;  citation_journal_title=Genomics;  citation_year=2013;  citation_volume=101;  citation_pages=282-9; " /><meta name="citation_reference" content="citation_title=A quantitative transcriptome reference map of the normal human brain;  citation_journal_title=Neurogenetics;  citation_year=2014;  citation_volume=15;  citation_pages=267-87; " /><meta name="citation_reference" content="citation_title=Integrated differential transcriptome maps of Acute Megakaryoblastic Leukemia (AMKL) in children with or without Down Syndrome (DS);  citation_journal_title=BMC Med. Genomics;  citation_year=2014;  citation_volume=7;  citation_pages=63" /><meta name="citation_reference" content="citation_title=An integrated route to identifying new pathogenesis-based therapeutic approaches for trisomy 21 (Down Syndrome) following the thought of Jérôme Lejeune;  citation_journal_title=Sci. Postprint;  citation_year=2013;  citation_volume=1;  citation_pages=e00010" /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/22/6/495/17463400/dsv028.pdf" /><meta name="description" content="Abstract.  We have developed GeneBase, a full parser of the National Center for Biotechnology Information (NCBI) Gene database, which generates a fully structur" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/22/6/495/2462510" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/22/6/495/2462510" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/22/6/495/2462510");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsv028';
    </script>
<title>
	Identification of minimal eukaryotic introns through GeneBase, a user-friendly tool for parsing the NCBI Gene databank | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./2462510" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BcCFATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iUGlvdmVzYW4sIEFsbGlzb24iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEV4cGVyaW1lbnRhbCwgRGlhZ25vc3RpYyBhbmQgU3BlY2lhbHR5IE1lZGljaW5lIChESU1FUyksIFVuaXQgb2YgSGlzdG9sb2d5LCBFbWJyeW9sb2d5IGFuZCBBcHBsaWVkIEJpb2xvZ3ksIFVuaXZlcnNpdHkgb2YgQm9sb2duYSwgQm9sb2duYSwgQk8gNDAxMjYsIEl0YWx5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQ2FyYWNhdXNpLCBNYXJpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgRXhwZXJpbWVudGFsLCBEaWFnbm9zdGljIGFuZCBTcGVjaWFsdHkgTWVkaWNpbmUgKERJTUVTKSwgVW5pdCBvZiBIaXN0b2xvZ3ksIEVtYnJ5b2xvZ3kgYW5kIEFwcGxpZWQgQmlvbG9neSwgVW5pdmVyc2l0eSBvZiBCb2xvZ25hLCBCb2xvZ25hLCBCTyA0MDEyNiwgSXRhbHkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJSaWNjaSwgTWFyY28iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEJpb2xvZ2ljYWwsIEdlb2xvZ2ljYWwgYW5kIEVudmlyb25tZW50YWwgU2NpZW5jZXMgKEJJR2VBKSwgVW5pdmVyc2l0eSBvZiBCb2xvZ25hLCBCb2xvZ25hLCBCTyA0MDEyNiwgSXRhbHkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTdHJpcHBvbGksIFBpZXJsdWlnaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgRXhwZXJpbWVudGFsLCBEaWFnbm9zdGljIGFuZCBTcGVjaWFsdHkgTWVkaWNpbmUgKERJTUVTKSwgVW5pdCBvZiBIaXN0b2xvZ3ksIEVtYnJ5b2xvZ3kgYW5kIEFwcGxpZWQgQmlvbG9neSwgVW5pdmVyc2l0eSBvZiBCb2xvZ25hLCBCb2xvZ25hLCBCTyA0MDEyNiwgSXRhbHkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJWaXRhbGUsIExvcmVuemEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEV4cGVyaW1lbnRhbCwgRGlhZ25vc3RpYyBhbmQgU3BlY2lhbHR5IE1lZGljaW5lIChESU1FUyksIFVuaXQgb2YgSGlzdG9sb2d5LCBFbWJyeW9sb2d5IGFuZCBBcHBsaWVkIEJpb2xvZ3ksIFVuaXZlcnNpdHkgb2YgQm9sb2duYSwgQm9sb2duYSwgQk8gNDAxMjYsIEl0YWx5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iUGVsbGVyaSwgTWFyaWEgQ2hpYXJhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBFeHBlcmltZW50YWwsIERpYWdub3N0aWMgYW5kIFNwZWNpYWx0eSBNZWRpY2luZSAoRElNRVMpLCBVbml0IG9mIEhpc3RvbG9neSwgRW1icnlvbG9neSBhbmQgQXBwbGllZCBCaW9sb2d5LCBVbml2ZXJzaXR5IG9mIEJvbG9nbmEsIEJvbG9nbmEsIEJPIDQwMTI2LCBJdGFseSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iSWRlbnRpZmljYXRpb24gb2YgbWluaW1hbCBldWthcnlvdGljIGludHJvbnMgdGhyb3VnaCBHZW5lQmFzZSwgYSB1c2VyLWZyaWVuZGx5IHRvb2wgZm9yIHBhcnNpbmcgdGhlIE5DQkkgR2VuZSBkYXRhYmFuayIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjQ5NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iNTAzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHN2MDI4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjIyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSI2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTUvMTIvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgaGlzdG9yeSBvZiB0aGUgZ2VuZXRpYyBzZXF1ZW5jZSBkYXRhYmFzZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0xOTkwOyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz03MDEtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RnJvbSBwcml2YXRlIGRhdGEgdG8gcHVibGljIGtub3dsZWRnZTsgY2l0YXRpb25fcHVibGlzaGVyPVBvcnRsYW5kIFByZXNzIEx0ZCwgTG9uZG9uLCBVSzsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZTogYSBnZW5lLWNlbnRlcmVkIGluZm9ybWF0aW9uIHJlc291cmNlIGF0IE5DQkk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT00MzsgIGNpdGF0aW9uX3BhZ2VzPUQzNi00MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGV4dC1iYXNlZCBkYXRhYmFzZSBzZWFyY2hpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRSRU5EUyBCaW90ZWNobm9sLjsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTMtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U09SVEVaOiBhIHJlbGF0aW9uYWwgdHJhbnNsYXRvciBmb3IgTkNCSSdzIEFTTi4xIGRhdGFiYXNlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Db21wdXQuIEFwcGwuIEJpb3NjaS47ICBjaXRhdGlvbl95ZWFyPTE5OTQ7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz0zNjktNzg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZhc3QgcGFyc2VycyBmb3IgRW50cmV6IEdlbmU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTIxOyAgY2l0YXRpb25fcGFnZXM9MzE4OS05MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXRsYXPigJRhIGRhdGEgd2FyZWhvdXNlIGZvciBpbnRlZ3JhdGl2ZSBiaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz0zNCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJJT1NQSURBOiBhIHJlbGF0aW9uYWwgZGF0YWJhc2UgdHJhbnNsYXRvciBmb3IgTkNCSTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QU1JQSBBbm51LiBTeW1wLiBQcm9jLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0yMCwxMDsgIGNpdGF0aW9uX3BhZ2VzPTQyMi02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbnRyb24tZXhvbiBzdHJ1Y3R1cmVzIG9mIGV1a2FyeW90aWMgbW9kZWwgb3JnYW5pc21zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9Mjc7ICBjaXRhdGlvbl9wYWdlcz0zMjE5LTI4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TeXN0ZW1hdGljIGFuYWx5c2lzIG9mIGludHJvbiBzaXplIGFuZCBhYnVuZGFuY2UgcGFyYW1ldGVycyBpbiBkaXZlcnNlIGxpbmVhZ2VzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2kuIENoaW5hLiBMaWZlIFNjaS47ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9NTY7ICBjaXRhdGlvbl9wYWdlcz05NjgtNzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgY29tcHV0YXRpb25hbCBhbmFseXNpcyBvZiBzZXF1ZW5jZSBmZWF0dXJlcyBpbnZvbHZlZCBpbiByZWNvZ25pdGlvbiBvZiBzaG9ydCBpbnRyb25zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTk4OyAgY2l0YXRpb25fcGFnZXM9MTExOTMtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWluaW1hbCBpbnRyb25zIGFyZSBub3Qg4oCYanVuazsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz0xMTg1LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUluZm9ybWF0aW9uIGNvbnRlbnQgb2YgQ2Flbm9yaGFiZGl0aXMgZWxlZ2FucyBzcGxpY2Ugc2l0ZSBzZXF1ZW5jZXMgdmFyaWVzIHdpdGggaW50cm9uIGxlbmd0aDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTkwOyAgY2l0YXRpb25fdm9sdW1lPTE4OyAgY2l0YXRpb25fcGFnZXM9MTUwOS0xMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3BsaWNpbmcgc2lnbmFscyBpbiBEcm9zb3BoaWxhOiBpbnRyb24gc2l6ZSwgaW5mb3JtYXRpb24gY29udGVudCwgYW5kIGNvbnNlbnN1cyBzZXF1ZW5jZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5MjsgIGNpdGF0aW9uX3ZvbHVtZT0yMDsgIGNpdGF0aW9uX3BhZ2VzPTQyNTUtNjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgbm92ZWwgcm9sZSBmb3IgbWluaW1hbCBpbnRyb25zOiByb3V0aW5nIG1STkFzIHRvIHRoZSBjeXRvc29sOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIE9uZTsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9ZTEwMTQ0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fcHVibGlzaGVyPUdhcmxhbmQgU2NpZW5jZSwgTmV3IFlvcms7ICBjaXRhdGlvbl90aXRsZT1IdW1hbiBtb2xlY3VsYXIgZ2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1hbW1hbGlhbiBpbnRyb25zOiB3aGVuIHRoZSBqdW5rIGdlbmVyYXRlcyBtb2xlY3VsYXIgZGl2ZXJzaXR5OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1JbnQuIEouIE1vbC4gU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTQ0MjktNTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVVuaUdlbmUgVGFidWxhdG9yOiBhIGZ1bGwgcGFyc2VyIGZvciB0aGUgVW5pR2VuZSBmb3JtYXQ7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTIyOyAgY2l0YXRpb25fcGFnZXM9MjU3MC0xOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UUkFNIChUcmFuc2NyaXB0b21lIE1hcHBlcik6IGRhdGFiYXNlLWRyaXZlbiBjcmVhdGlvbiBhbmQgYW5hbHlzaXMgb2YgdHJhbnNjcmlwdG9tZSBtYXBzIGZyb20gbXVsdGlwbGUgc291cmNlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9MTIxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGF0YWJhc2UgcmVzb3VyY2VzIG9mIHRoZSBOYXRpb25hbCBDZW50ZXIgZm9yIEJpb3RlY2hub2xvZ3kgSW5mb3JtYXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT00MzsgIGNpdGF0aW9uX3BhZ2VzPUQ2LTE3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HYXBwZWQgQkxBU1QgYW5kIFBTSS1CTEFTVDogYSBuZXcgZ2VuZXJhdGlvbiBvZiBwcm90ZWluIGRhdGFiYXNlIHNlYXJjaCBwcm9ncmFtczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9MzM4OS00MDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmUgT250b2xvZ3kgYW5ub3RhdGlvbnMgYW5kIHJlc291cmNlczsgY2l0YXRpb25fYXV0aG9yPUdlbmUgT250b2xvZ3kgQ29uc29ydGl1bTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTQxOyAgY2l0YXRpb25fcGFnZXM9RDUzMC01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbmFseXNpcyBvZiBjYW5vbmljYWwgYW5kIG5vbi1jYW5vbmljYWwgc3BsaWNlIHNpdGVzIGluIG1hbW1hbGlhbiBnZW5vbWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9Mjg7ICBjaXRhdGlvbl9wYWdlcz00MzY0LTc1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9wdWJsaXNoZXI9R2FybGFuZCBTY2llbmNlLCBOZXcgWW9yazsgIGNpdGF0aW9uX3RpdGxlPU1vbGVjdWxhciBiaW9sb2d5IG9mIHRoZSBjZWxsOyAgY2l0YXRpb25feWVhcj0yMDE0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JZGVudGlmaWNhdGlvbiBhbmQgdmFsaWRhdGlvbiBvZiBldm9sdXRpb25hcmlseSBjb25zZXJ2ZWQgdW51c3VhbGx5IHNob3J0IHByZS1tUk5BIGludHJvbnMgaW4gdGhlIGh1bWFuIGdlbm9tZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9SW50LiBKLiBNb2wuIFNjaS47ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz0xMDM3Ni04ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lLXNjYWxlIGFuYWx5c2lzIG9mIGh1bWFuIG1STkEgNeKAsiBjb2Rpbmcgc2VxdWVuY2VzIGJhc2VkIG9uIGV4cHJlc3NlZCBzZXF1ZW5jZSB0YWcgKEVTVCkgZGF0YWJhc2U7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTEwMDsgIGNpdGF0aW9uX3BhZ2VzPTEyNS0zMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW1wcm92aW5nIG1STkEgNeKAsiBjb2Rpbmcgc2VxdWVuY2UgZGV0ZXJtaW5hdGlvbiBpbiB0aGUgbW91c2UgZ2Vub21lOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1NYW1tLiBHZW5vbWU7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0xNDktNTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNSUzogaW5mb3JtYXRpb24gcmV0cmlldmFsIHN5c3RlbSBmb3IgbW9sZWN1bGFyIGJpb2xvZ3kgZGF0YSBiYW5rczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TWV0aG9kcyBFbnp5bW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5NjsgIGNpdGF0aW9uX3ZvbHVtZT0yNjY7ICBjaXRhdGlvbl9wYWdlcz0xMTQtMjg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmVSZWNvcmRzOiBhIHJlbGF0aW9uYWwgZGF0YWJhc2UgZm9yIEdlbkJhbmsgZmxhdCBmaWxlIHBhcnNpbmcgYW5kIGRhdGEgbWFuaXB1bGF0aW9uIGluIHBlcnNvbmFsIGNvbXB1dGVyczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MjA7ICBjaXRhdGlvbl9wYWdlcz0yODgzLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN0YXRpc3RpY2FsIGZlYXR1cmVzIG9mIGh1bWFuIGV4b25zIGFuZCB0aGVpciBmbGFua2luZyByZWdpb25zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1IdW0uIE1vbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz05MTktMzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRydW5jYXRpbmcgbXV0YXRpb25zIGluIHRoZSBsYXN0IGV4b24gb2YgTk9UQ0gyIGNhdXNlIGEgcmFyZSBza2VsZXRhbCBkaXNvcmRlciB3aXRoIG9zdGVvcG9yb3NpczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NDM7ICBjaXRhdGlvbl9wYWdlcz0zMDYtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RUlEOiB0aGUgRXhvbi1JbnRyb24gRGF0YWJhc2UtYW4gZXhoYXVzdGl2ZSBkYXRhYmFzZSBvZiBwcm90ZWluLWNvZGluZyBpbnRyb24tY29udGFpbmluZyBnZW5lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTI4OyAgY2l0YXRpb25fcGFnZXM9MTg1LTkwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JRS1LYjogaW50cm9uIGV4b24ga25vd2xlZGdlIGJhc2U7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9MTE1MS0yOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgSURCIGFuZCBJRURCOiBpbnRyb24gc2VxdWVuY2UgYW5kIGV2b2x1dGlvbiBkYXRhYmFzZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0yODsgIGNpdGF0aW9uX3BhZ2VzPTE4MS00OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FeEludDogYW4gZXhvbiBpbnRyb24gZGF0YWJhc2U7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTE5MS00OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TRUdFOiBhIGRhdGFiYXNlIG9uIOKAmGludHJvbiBsZXNzL3NpbmdsZSBleG9uaWPigJkgZ2VuZXMgZnJvbSBldWthcnlvdGVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0xODsgIGNpdGF0aW9uX3BhZ2VzPTEyNjYtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9WHBybzogZGF0YWJhc2Ugb2YgZXVrYXJ5b3RpYyBwcm90ZWluLWVuY29kaW5nIGdlbmVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MzI7ICBjaXRhdGlvbl9wYWdlcz1ENTktNjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFkdmFuY2VzIGluIHRoZSBleG9uLWludHJvbiBkYXRhYmFzZSAoRUlEKTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QnJpZWYgQmlvaW5mb3JtLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9MTc4LTg1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NZWNoYW5pc3RpYyBpbnNpZ2h0cyBpbnRvIGh1bWFuIHByZS1tUk5BIHNwbGljaW5nIG9mIGh1bWFuIHVsdHJhLXNob3J0IGludHJvbnM6IHBvdGVudGlhbCB1bnVzdWFsIG1lY2hhbmlzbSBpZGVudGlmaWVzIEctcmljaCBpbnRyb25zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9jaGVtLiBCaW9waHlzLiBSZXMuIENvbW11bi47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NDIzOyAgY2l0YXRpb25fcGFnZXM9Mjg5LTk0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBiaW9pbmZvcm1hdGljIGFuZCBtb2xlY3VsYXIgYW5hbHlzaXMgb2YgaW50cm9ucyBpbiBTYWNjaGFyb215Y2VzIGNlcmV2aXNpYWU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVJOQTsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9MjIxLTM0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgQXJhYmlkb3BzaXMgSW5mb3JtYXRpb24gUmVzb3VyY2UgKFRBSVIpOiBhIGNvbXByZWhlbnNpdmUgZGF0YWJhc2UgYW5kIHdlYi1iYXNlZCBpbmZvcm1hdGlvbiByZXRyaWV2YWwsIGFuYWx5c2lzLCBhbmQgdmlzdWFsaXphdGlvbiBzeXN0ZW0gZm9yIGEgbW9kZWwgcGxhbnQ7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0yOTsgIGNpdGF0aW9uX3BhZ2VzPTEwMi01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TcGxpY2luZyBvZiBwcmVjdXJzb3JzIHRvIG1STkFzIGJ5IHRoZSBzcGxpY2Vvc29tZXM7IGNpdGF0aW9uX3B1Ymxpc2hlcj1Db2xkIFNwcmluZyBIYXJib3IgTGFib3JhdG9yeSBQcmVzcywgQ29sZCBTcHJpbmcgSGFyYm9yLCBOWTsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3BhZ2VzPTUyNS02MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIEdFTkNPREUgcHNldWRvZ2VuZSByZXNvdXJjZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9UjUxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGFzc0RCMuKAlGEgY29tcHJlaGVuc2l2ZSBkYXRhYmFzZSBvZiBzdWJ0bGUgYWx0ZXJuYXRpdmUgc3BsaWNpbmcgZXZlbnRzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz0yMTYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1XaWRlc3ByZWFkIG9jY3VycmVuY2Ugb2YgYWx0ZXJuYXRpdmUgc3BsaWNpbmcgYXQgTkFHTkFHIGFjY2VwdG9ycyBjb250cmlidXRlcyB0byBwcm90ZW9tZSBwbGFzdGljaXR5OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0zNjsgIGNpdGF0aW9uX3BhZ2VzPTEyNTUtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlmZmVyZW50aWFsIGV4cHJlc3Npb24gb2YgYWx0ZXJuYXRpdmVseSBzcGxpY2VkIG1STkEgZm9ybXMgb2YgdGhlIGluc3VsaW4tbGlrZSBncm93dGggZmFjdG9yIDEgcmVjZXB0b3IgaW4gaHVtYW4gbmV1cm9lbmRvY3JpbmUgdHVtb3JzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1PbmNvbC4gUmVwLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTEyNDktNTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNlcXVlbmNlLCDigJhzdWJ0bGXigJkgYWx0ZXJuYXRpdmUgc3BsaWNpbmcgYW5kIGV4cHJlc3Npb24gb2YgdGhlIENZWVIxIChjeXN0ZWluZS90eXJvc2luZS1yaWNoIDEpIG1STkEgaW4gaHVtYW4gbmV1cm9lbmRvY3JpbmUgdHVtb3JzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgQ2FuY2VyOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz02NiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5vaXN5IHNwbGljaW5nIGRyaXZlcyBtUk5BIGlzb2Zvcm0gZGl2ZXJzaXR5IGluIGh1bWFuIGNlbGxzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9ZTEwMDEyMzYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbHRlcm5hdGl2ZSBzcGxpY2luZyBhdCBHWU5OR1kgNeKAsiBzcGxpY2Ugc2l0ZXM6IG1vcmUgbm9pc2UsIGxlc3MgcmVndWxhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTQyOyAgY2l0YXRpb25fcGFnZXM9MTM5NjktODA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUluIHNpbGljbyBwcmVkaWN0aW9uIG9mIHNwbGljZS1hbHRlcmluZyBzaW5nbGUgbnVjbGVvdGlkZSB2YXJpYW50cyBpbiB0aGUgaHVtYW4gZ2Vub21lOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9NDI7ICBjaXRhdGlvbl9wYWdlcz0xMzUzNC00NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBjb21wcmVoZW5zaXZlIHN1cnZleSBvZiBub24tY2Fub25pY2FsIHNwbGljZSBzaXRlcyBpbiB0aGUgaHVtYW4gdHJhbnNjcmlwdG9tZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTQyOyAgY2l0YXRpb25fcGFnZXM9MTA1NjQtNzg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXB1dGF0aW9uYWwgaWRlbnRpZmljYXRpb24gb2YgZnVuY3Rpb25hbCBpbnRyb25zOiBoaWdoIHBvc2l0aW9uYWwgY29uc2VydmF0aW9uIG9mIGludHJvbnMgdGhhdCBoYXJib3IgUk5BIGdlbmVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9NDE7ICBjaXRhdGlvbl9wYWdlcz01NjA0LTEzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZWxlY3Rpb24gZm9yIHNob3J0IGludHJvbnMgaW4gaGlnaGx5IGV4cHJlc3NlZCBnZW5lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MzE7ICBjaXRhdGlvbl9wYWdlcz00MTUtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW50cm9uIHNpemUgaW4gbWFtbWFsczogY29tcGxleGl0eSBjb21lcyB0byB0ZXJtcyB3aXRoIGVjb25vbXk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRyZW5kcyBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MjM7ICBjaXRhdGlvbl9wYWdlcz0yMC00OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Fdm9sdXRpb24gb2YgZ2VuZSBzdHJ1Y3R1cmUgaW4gdGhlIGNvbmlmZXIgUGljZWEgZ2xhdWNhOiBhIGNvbXBhcmF0aXZlIGFuYWx5c2lzIG9mIHRoZSBpbXBhY3Qgb2YgaW50cm9uIHNpemU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBQbGFudCBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTk1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VW5pdmVyc2FsIHRpZ2h0IGNvcnJlbGF0aW9uIG9mIGNvZG9uIGJpYXMgYW5kIHBvb2wgb2YgUk5BIGNvZG9ucyAoY29kb25vbWUpOiB0aGUgZ2Vub21lIGlzIG9wdGltaXplZCB0byBhbGxvdyBhbnkgZGlzdHJpYnV0aW9uIG9mIGdlbmUgZXhwcmVzc2lvbiB2YWx1ZXMgaW4gdGhlIHRyYW5zY3JpcHRvbWUgZnJvbSBiYWN0ZXJpYSB0byBodW1hbnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTEwMTsgIGNpdGF0aW9uX3BhZ2VzPTI4Mi05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHF1YW50aXRhdGl2ZSB0cmFuc2NyaXB0b21lIHJlZmVyZW5jZSBtYXAgb2YgdGhlIG5vcm1hbCBodW1hbiBicmFpbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmV1cm9nZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTI2Ny04NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW50ZWdyYXRlZCBkaWZmZXJlbnRpYWwgdHJhbnNjcmlwdG9tZSBtYXBzIG9mIEFjdXRlIE1lZ2FrYXJ5b2JsYXN0aWMgTGV1a2VtaWEgKEFNS0wpIGluIGNoaWxkcmVuIHdpdGggb3Igd2l0aG91dCBEb3duIFN5bmRyb21lIChEUyk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBNZWQuIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz02MyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuIGludGVncmF0ZWQgcm91dGUgdG8gaWRlbnRpZnlpbmcgbmV3IHBhdGhvZ2VuZXNpcy1iYXNlZCB0aGVyYXBldXRpYyBhcHByb2FjaGVzIGZvciB0cmlzb215IDIxIChEb3duIFN5bmRyb21lKSBmb2xsb3dpbmcgdGhlIHRob3VnaHQgb2YgSsOpcsO0bWUgTGVqZXVuZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpLiBQb3N0cHJpbnQ7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTsgIGNpdGF0aW9uX3BhZ2VzPWUwMDAxMCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8yMi82LzQ5NS8xNzQ2MzQwMC9kc3YwMjgucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIFdlIGhhdmUgZGV2ZWxvcGVkIEdlbmVCYXNlLCBhIGZ1bGwgcGFyc2VyIG9mIHRoZSBOYXRpb25hbCBDZW50ZXIgZm9yIEJpb3RlY2hub2xvZ3kgSW5mb3JtYXRpb24gKE5DQkkpIEdlbmUgZGF0YWJhc2UsIHdoaWNoIGdlbmVyYXRlcyBhIGZ1bGx5IHN0cnVjdHVyIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFcy8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj0yNDYyNTEwJnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZDMNFhkCmnKFoqDp0xAK+nR8JUBp" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f6%2f495%2f2462510" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f6%2f495%2f2462510" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=83834" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/22/6"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/6/1/m_cover.gif?Expires=2147483647&amp;Signature=083aE4wPido26a6~EOoRT3Nb4fPpp5HV1MZzrjjay94a3evfhfRTvESRMhc7ROqyA2UIz2umFef89XL3Lu7hUZFod2Ix~QtkT4CP~8MDkYY61O0I1KLQQcYNpXVti4-0uwuVQWc0Ik7X46YCFzJAu2cTqwHZ2B0OBVFR-OZWKIwha86xHbqGB8R5L83t0zl8dqx1CkCPppI95mus6SUAdvZ0-wLyrVhsfEErb~4S7v8v0rjWvz1OZ1A2b6wWGyo1cjX3sRRxlgjkSlblmiowk2fs-R4agAPGw2bGiz2-0n1EnfsgKN8CSyFRqrRfdmccMFKJr5L5jh5C1s7L6OJB0g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/22/6">
                        <div class="volume trailing-comma">Volume 22</div>
                        <div class="issue">Issue 6</div>

            </a>

        </div>
        <div class="ii-pub-date">
December 2015        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="87201610">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87201610">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87201612">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87201612">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87201618">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87201618">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87201637">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87201637">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87201654">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87201654">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87201665">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87201665">Availability</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87201667">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87201667">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87201669">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87201669">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87201671">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87201671">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/22/6/485/2462549">&lt; Previous</a>
        </li>        
            <li class="empty arrow">&nbsp;</li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Identification of minimal eukaryotic introns through GeneBase, a user-friendly tool for parsing the NCBI Gene databank
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Allison Piovesan</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Experimental, Diagnostic and Specialty Medicine (DIMES), Unit of Histology, Embryology and Applied Biology</div>, <div class="insititution">University of Bologna</div>, <div class="addr-line">Bologna, BO 40126</div>, <div class="country">Italy</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Allison+Piovesan">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Piovesan A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Piovesan A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Allison Piovesan</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Maria Caracausi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Experimental, Diagnostic and Specialty Medicine (DIMES), Unit of Histology, Embryology and Applied Biology</div>, <div class="insititution">University of Bologna</div>, <div class="addr-line">Bologna, BO 40126</div>, <div class="country">Italy</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Maria+Caracausi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Caracausi M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Caracausi M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Maria Caracausi</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Marco Ricci</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Department of Biological, Geological and Environmental Sciences (BIGeA)</div>, <div class="insititution">University of Bologna</div>, <div class="addr-line">Bologna, BO 40126</div>, <div class="country">Italy</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Marco+Ricci">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ricci M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ricci M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Marco Ricci</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Pierluigi Strippoli</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Experimental, Diagnostic and Specialty Medicine (DIMES), Unit of Histology, Embryology and Applied Biology</div>, <div class="insititution">University of Bologna</div>, <div class="addr-line">Bologna, BO 40126</div>, <div class="country">Italy</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Pierluigi+Strippoli">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Strippoli P">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Strippoli P&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Pierluigi Strippoli</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Lorenza Vitale</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Experimental, Diagnostic and Specialty Medicine (DIMES), Unit of Histology, Embryology and Applied Biology</div>, <div class="insititution">University of Bologna</div>, <div class="addr-line">Bologna, BO 40126</div>, <div class="country">Italy</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +39 0512094097. Fax. +39 0512094110. E-mail: <a href="mailto:lorenza.vitale@unibo.it" target="_blank">lorenza.vitale@unibo.it</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Lorenza+Vitale">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Vitale L">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Vitale L&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Lorenza Vitale</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Maria Chiara Pelleri</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Experimental, Diagnostic and Specialty Medicine (DIMES), Unit of Histology, Embryology and Applied Biology</div>, <div class="insititution">University of Bologna</div>, <div class="addr-line">Bologna, BO 40126</div>, <div class="country">Italy</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Maria+Chiara+Pelleri">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Pelleri M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Pelleri M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Maria Chiara Pelleri</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="footnote-content">Edited by Prof. Kenta Nakai</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 22, Issue 6, 1 December 2015, Pages 495–503, <a href='https://doi.org/10.1093/dnares/dsv028'>https://doi.org/10.1093/dnares/dsv028</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">17 November 2015</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">15 June 2015</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">07 October 2015</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="2462510" href="/dnaresearch/article-pdf/22/6/495/17463400/dsv028.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Allison Piovesan, Maria Caracausi, Marco Ricci, Pierluigi Strippoli, Lorenza Vitale, Maria Chiara Pelleri;  Identification of minimal eukaryotic introns through GeneBase, a user-friendly tool for parsing the NCBI Gene databank, <em>DNA Research</em>, Volume 22, Issue 6, 1 December 2015, Pages 495–503, <a href='https://doi.org/10.1093/dnares/dsv028'>https://doi.org/10.1093/dnares/dsv028</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/2462510?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/2462510?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/2462510?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/2462510?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/2462510?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/2462510?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/2462510?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Identification%20of%20minimal%20eukaryotic%20introns%20through%20GeneBase%2C%20a%20user-friendly%20tool%20for%20parsing%20the%20NCBI%20Gene%20databank&amp;publicationDate=2015-11-17&amp;volumeNum=22&amp;issueNum=6&amp;author=Piovesan%2C%20Allison%3B%20Caracausi%2C%20Maria&amp;startPage=495&amp;endPage=503&amp;contentId=10.1093%2Fdnares%2Fdsv028&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f6%2f495%2f2462510" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=87201610 id="87201610" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>We have developed GeneBase, a full parser of the National Center for Biotechnology Information (NCBI) Gene database, which generates a fully structured local database with an intuitive user-friendly graphic interface for personal computers. Features of all the annotated eukaryotic genes are accessible through three main software tables, including for each entry details such as the gene summary, the gene exon/intron structure and the specific Gene Ontology attributions. The structuring of the data, the creation of additional calculation fields and the integration with nucleotide sequences allow users to make many types of comparisons and calculations that are useful for data retrieval and analysis. We provide an original example analysis of the existing introns across all the available species, through which the classic biological problem of the ‘minimal intron’ may find a solution using available data. Based on all currently available data, we can define the shortest known eukaryotic GT-AG intron length, setting the physical limit at the 30 base pair intron belonging to the human <em>MST1L</em> gene. This ‘model intron’ will shed light on the minimal requirement elements of recognition used for conventional splicing functioning. Remarkably, this size is indeed consistent with the sum of the splicing consensus sequence lengths.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;NCBI Gene&quot;">NCBI Gene</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;gene data parsing&quot;">gene data parsing</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;computational biology&quot;">computational biology</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;minimal intron&quot;">minimal intron</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;personal computer software&quot;">personal computer software</a></div>                            <h2 scrollto-destination=87201612 id="87201612" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>The automation of sequencing techniques and the spread of computer use gave rise to a flourishing number of new molecular structures and sequences and to a proliferation of new databases in which to store them.<sup><a reveal-id="DSV028C1" data-open="DSV028C1" class="link link-ref link-reveal xref-bibr">1</a></sup> The public availability of databases is of inestimable value, because the collective use of data leads to the discovery of new knowledge that goes beyond the results yielded by individual experiments.<sup><a reveal-id="DSV028C2" data-open="DSV028C2" class="link link-ref link-reveal xref-bibr">2</a></sup></p><p>The National Center for Biotechnology Information (NCBI) Gene database (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/gene">http://www.ncbi.nlm.nih.gov/gene</a>) is a collection of gene records accessible as web page entries. Among the other existing genome browsers, such as the University of California at Santa Cruz (UCSC) Genome Browser (<a class="link link-uri" href="http://genome-euro.ucsc.edu/cgi-bin/hgGateway">http://genome-euro.ucsc.edu/cgi-bin/hgGateway</a>) and Ensembl (<a class="link link-uri" href="http://www.ensembl.org/index.html">http://www.ensembl.org/index.html</a>), NCBI Gene is the most complete, containing information about more than 2 million genes from more than 350 species (eukaryotic genes with nuclear genome annotations; the total is over 11 million genes from almost 13,000 species). It provides information about gene nomenclature, chromosomal localization, gene transcripts and products, as well as a series of useful links to, among other things, sequences, maps, citations, phenotypes, variation details, interactions and external databases.<sup><a reveal-id="DSV028C3" data-open="DSV028C3" class="link link-ref link-reveal xref-bibr">3</a></sup> As well as browsing the data, a very efficient way of accessing information is by performing a text search. NCBI Gene is a text searchable database through indexed fields using specific term queries (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/books/NBK3841/">http://www.ncbi.nlm.nih.gov/books/NBK3841/</a>), although unfortunately not all fields of the database are available for searching.<sup><a reveal-id="DSV028C4" data-open="DSV028C4" class="link link-ref link-reveal xref-bibr">4</a></sup> Furthermore, NCBI makes some Entrez Programming Utilities (E-Utilities) available that can be combined to form customized data pipelines to extract the desired information from NCBI Gene (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/books/NBK25497/">http://www.ncbi.nlm.nih.gov/books/NBK25497/</a>). The retrieved matching results can be downloaded in different formats: text, abstract syntax notation one (ASN.1) and extensible markup language (XML), allowing further analyses on a local computer. Parsers able to transform ASN.1- or XML-formatted NCBI Gene data to a relational database exist. They usually give output information that can only be accessed through the structured query language (SQL) without a graphical interface; and some of them also require high-level programming and querying skills.<sup><a reveal-id="DSV028C5  DSV028C8" data-open="DSV028C5  DSV028C8" class="link link-ref link-reveal xref-bibr">5–8</a></sup></p><p>To address these problems, we have developed GeneBase, a full parser of the NCBI Gene database, which generates a fully structured local and relational database combined with an intuitive user-friendly graphical interface for personal computers. It allows users to do original searches, calculations and analyses of the main information about genes which are fully annotated with the ‘Gene Table’ section in NCBI Gene, i.e. eukaryotic genes. Furthermore, for a subset of gene records, we integrated nucleotide sequences useful for additional elaboration with the corresponding gene-associated meta-information.</p><p>GeneBase database contains a wealth of interesting biological information, and we provide an original analysis of the classic biological problem of the ‘minimal intron’ (the minimal DNA sequence element that can function as an intron) across all the available species as an example. A limitation in minimum length below which an intron cannot be spliced must exist.<sup><a reveal-id="DSV028C9" data-open="DSV028C9" class="link link-ref link-reveal xref-bibr">9</a></sup> In the literature, the minimal intron length is usually given as a range, an average or as a length lesser than a certain number, often without a primary reference, accession number or gene name. Short introns were defined as those not longer than 116 base pairs (bp) in <em>Arabidopsis thaliana</em> (Taxonomy Identifier or ID: 3702).<sup><a reveal-id="DSV028C10" data-open="DSV028C10" class="link link-ref link-reveal xref-bibr">10</a></sup> In <em>Saccharomyces cerevisiae</em> (Taxonomy ID: 4932), short intron length is ≤191 bp,<sup><a reveal-id="DSV028C11" data-open="DSV028C11" class="link link-ref link-reveal xref-bibr">11</a></sup> with an average of 92 ± 20 and 49 ± 11 bp, as in <em>Schizosaccharomyces pombe</em> (Taxonomy ID: 4896).<sup><a reveal-id="DSV028C12" data-open="DSV028C12" class="link link-ref link-reveal xref-bibr">12</a></sup> In <em>Caenorhabditis elegans</em> (Taxonomy ID: 6239), short intron length was on average 51.5 bp,<sup><a reveal-id="DSV028C13" data-open="DSV028C13" class="link link-ref link-reveal xref-bibr">13</a></sup> and confirmed later with a length of ≤60 bp,<sup><a reveal-id="DSV028C11" data-open="DSV028C11" class="link link-ref link-reveal xref-bibr">11</a></sup> with a minimum of 48 bp.<sup><a reveal-id="DSV028C10" data-open="DSV028C10" class="link link-ref link-reveal xref-bibr">10</a></sup> In <em>Drosophila</em> (Taxonomy ID: 7215), the minimum length is 63 bp,<sup><a reveal-id="DSV028C10" data-open="DSV028C10" class="link link-ref link-reveal xref-bibr">10</a></sup> but the minimum experimentally verified is 74 bp.<sup><a reveal-id="DSV028C14" data-open="DSV028C14" class="link link-ref link-reveal xref-bibr">14</a></sup> For mouse and human (Taxonomy IDs: 10088 and 9606, respectively), one of the most recent studies defined the minimal intron length range as between 50 and 150 bp, corresponding to the peak value of the intron length distribution,<sup><a reveal-id="DSV028C10" data-open="DSV028C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DSV028C15" data-open="DSV028C15" class="link link-ref link-reveal xref-bibr">15</a></sup> in contrast with the length &lt;30 bp in <em>Homo sapiens</em> (Taxonomy ID: 9606) hypothesized by Strachan and Read.<sup><a reveal-id="DSV028C16" data-open="DSV028C16" class="link link-ref link-reveal xref-bibr">16</a></sup></p><p>We show that the intron length problem, which still raises researchers' interest,<sup><a reveal-id="DSV028C17" data-open="DSV028C17" class="link link-ref link-reveal xref-bibr">17</a></sup> may find a solution, regarding all currently available data and canonical introns, through a new tool like GeneBase, which is especially useful for retrieving data with numerical range constraints and with the corresponding gene-associated meta-information. Introns &lt;30 bp were not found in any of the species analysed, shedding light on the minimal sequence requirement elements used by the cell for conventional splicing functioning. Remarkably, the 30 bp size is indeed consistent with the sum of the known 5′/3′ splicing consensus sequence lengths.</p>                            <h2 scrollto-destination=87201618 id="87201618" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=87201619 id="87201619" class="section-title" data-legacy-id=s2a>2.1. Database construction</h3>
<p>GeneBase was developed within the FileMaker Pro Advanced environment (FileMaker, Santa Clara, CA, USA), which has already been proved useful for complex parsing of genomic data.<sup><a reveal-id="DSV028C18" data-open="DSV028C18" class="link link-ref link-reveal xref-bibr">18</a>,<a reveal-id="DSV028C19" data-open="DSV028C19" class="link link-ref link-reveal xref-bibr">19</a></sup> This is a database management system with an intuitive user-friendly graphical interface for both Macintosh (Mac OS X) and Windows operating systems. Minimum system requirements are: Mac OS X 10.6, Intel-based Mac CPU (Central Processing Unit), 1 GigaByte (GB) of RAM (Random Access Memory), 1024 × 768 or higher resolution video adapter and display; Windows XP Professional, Home Edition (Service Pack 3), 700 MegaHertz (MHz) CPU or faster, 256 MegaBytes (MB) of RAM, 1024 × 768 or higher resolution video adapter and display.</p><p>The pre-loaded version of GeneBase was obtained by first downloading all the available Animalia (Metazoa, Taxonomy ID: 33208), Fungi (Taxonomy ID: 4751) and Plant (Viridiplantae, Taxonomy ID: 33090) kingdom gene entries from NCBI Gene. Specific text queries were used to fragment the download according to the three kingdoms and to retrieve all current (alive/live) records with a genomic gene source, excluding gene models (generated by annotation pipelines), as described in detail in the GeneBase guide. The initial download was performed on 22 April 2015 choosing the ASN.1 format, as it is the data reference representation format used by NCBI, providing smaller file sizes, fewer errors and complete data, while avoiding problems encountered by the FileMaker Pro XML parsing engine with large data files. We have developed a Python (<a class="link link-uri" href="http://www.python.org/">http://www.python.org/</a>, version 2.7) executable script to quickly parse ASN.1-formatted downloaded gene entries and thus obtain three tab-delimited files suitable for import into the three main related tables of GeneBase (corresponding to NCBI sections): ‘Gene_Summary’, ‘Gene_Table’ and ‘Gene_Ontology’. ‘Gene_Summary’ table contains one record for each gene and collects details such as the official gene symbol, the official gene full name, the organism's name and a brief summary description of the gene and its cellular localization and function (when available). ‘Gene_Table’ consists of one record for each exon including the corresponding intron if an intron follows that exon, representing the exon/intron structure of each transcript isoform as annotated on the indicated genomic Reference Sequence (RefSeq).<sup><a reveal-id="DSV028C20" data-open="DSV028C20" class="link link-ref link-reveal xref-bibr">20</a></sup> Each record contains details such as RefSeq GenBank accession numbers of chromosome, messenger RNA (mRNA) and protein (when available), and the genomic coordinates of exons and introns. Additional calculation fields were created to extract further data not present in the original NCBI gene entries from the available information and are highlighted in red text. Furthermore, additional boxes were created to show useful related fields of other related software tables, giving the opportunity to perform crossed searches. Different buttons were developed to facilitate navigation through the software tables, the retrieval of features in popular databases and the launch of online BLAST (Basic Local Alignment Search Tool)<sup><a reveal-id="DSV028C21" data-open="DSV028C21" class="link link-ref link-reveal xref-bibr">21</a></sup> comparisons. Finally, text fields were specifically designed to contain exon and intron sequences. ‘Gene_Ontology’ table contains the specific Gene Ontology (GO) attributions for each gene when available.<sup><a reveal-id="DSV028C22" data-open="DSV028C22" class="link link-ref link-reveal xref-bibr">22</a></sup> A specific table named ‘Transcripts’ is available, showing the RefSeq status provided for each gene as well as for each of its individual transcripts. An additional table named ‘Reports’ is generated to provide statistics such as the mean length of exons and introns. In agreement with general criteria in database design, we decided to fragment information into distinct fields as much as possible, to facilitate an independent management of data. All database fields are indexed to ensure efficient data retrieval through the query options.</p><p>A specific feature of GeneBase is the integration with nucleotide sequences. To achieve this, a Python executable script was developed to extract exon (both coding and not coding) and intron sequences of each entry from any chromosome sequences in FASTA format and, thus, obtain a tab-delimited file suitable for import into GeneBase, if desired (Fig. <a reveal-id="DSV028F1" data-open="DSV028F1" class="link link-reveal link-table xref-fig">1</a>). </p>                        <a id="87201623" scrollto-destination="87201623"></a>
<div id="DSV028F1" data-id="DSV028F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv028/1/m_dsv02801.png?Expires=2147483647&Signature=aTiOj11fSqJLq6n7glqYAmF8hsPR313Btudv0n8w5KNu~zUGqcWHbTHtvLXaiYImpLRT8-UUAShIhIw15Oxo-nDOm39wI~z3Khg4V2aZjVRLfDOOWSwmnip~uvFFulxzVEoeV7VI1sNQ46Ga2cjifvzeSn1Sqgao5lVKE9zg4wacYMLrhCxPySirRrvmDf-22BrTRGmvCFrU33WToqXa7oMy2IEncjpcVtNQaf30Ev62UiT~QHRFb6yrW1ShfMximxKq3x7siQqUBNk6Vmzou5qRFNNt6ubw18xWEp5kvoG93rYBCRMZtj~xS6a9C2r~YT5ZLUIQz1~ayRyyj1cIxA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Flow diagram illustrating the data parsing involved in the GeneBase development. ‘Gene_Summary’, ‘Gene_Table’ and ‘Gene_Ontology’ are the three main related software tables." path-from-xml="dsv02801.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87201623/dsv02801.tiff" path-from-xml="dsv02801.tiff" target="_blank">View large</a><a section="87201623" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv028/1/dsv02801.png?Expires=2147483647&Signature=PeDfjSa6kQ3qAdEUByicX1DDbywAXUfpfzHIhFjgQvCIIDvGf5QV2FCFdhK2jr7bEhZzvusoi5dhhUIzbfdFKhx09LmRaSklmIXnhZYTppwyLM-LZP4dh0RuzDYtNm1EEBC9lVXU9r36KwoLMAP-JCFB9u5vbNxHFCWQgVYAw2pTZzKFjN-x8soMS5K~ctLTBrwT-ZfLEh9OtgFgb71kt90BobaTn-t9dl-MAGUE~qzujebs90G9VnX5G9mXcr8SVDn6ASeBtDgKeptQ5OqTd7augm9FYPgP8UDbgA6qKXqOJZ36YTUKz5TvuUW2PVxKrcB9q4ztcg4POudMfjpV~Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87201623&ar=2462510&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02801.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Flow diagram illustrating the data parsing involved in the GeneBase development. ‘Gene_Summary’, ‘Gene_Table’ and ‘Gene_Ontology’ are the three main related software tables.</p></div></div><div content-id="DSV028F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv028/1/m_dsv02801.png?Expires=2147483647&Signature=aTiOj11fSqJLq6n7glqYAmF8hsPR313Btudv0n8w5KNu~zUGqcWHbTHtvLXaiYImpLRT8-UUAShIhIw15Oxo-nDOm39wI~z3Khg4V2aZjVRLfDOOWSwmnip~uvFFulxzVEoeV7VI1sNQ46Ga2cjifvzeSn1Sqgao5lVKE9zg4wacYMLrhCxPySirRrvmDf-22BrTRGmvCFrU33WToqXa7oMy2IEncjpcVtNQaf30Ev62UiT~QHRFb6yrW1ShfMximxKq3x7siQqUBNk6Vmzou5qRFNNt6ubw18xWEp5kvoG93rYBCRMZtj~xS6a9C2r~YT5ZLUIQz1~ayRyyj1cIxA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Flow diagram illustrating the data parsing involved in the GeneBase development. ‘Gene_Summary’, ‘Gene_Table’ and ‘Gene_Ontology’ are the three main related software tables." path-from-xml="dsv02801.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87201623/dsv02801.tiff" path-from-xml="dsv02801.tiff" target="_blank">View large</a><a section="87201623" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv028/1/dsv02801.png?Expires=2147483647&Signature=PeDfjSa6kQ3qAdEUByicX1DDbywAXUfpfzHIhFjgQvCIIDvGf5QV2FCFdhK2jr7bEhZzvusoi5dhhUIzbfdFKhx09LmRaSklmIXnhZYTppwyLM-LZP4dh0RuzDYtNm1EEBC9lVXU9r36KwoLMAP-JCFB9u5vbNxHFCWQgVYAw2pTZzKFjN-x8soMS5K~ctLTBrwT-ZfLEh9OtgFgb71kt90BobaTn-t9dl-MAGUE~qzujebs90G9VnX5G9mXcr8SVDn6ASeBtDgKeptQ5OqTd7augm9FYPgP8UDbgA6qKXqOJZ36YTUKz5TvuUW2PVxKrcB9q4ztcg4POudMfjpV~Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87201623&ar=2462510&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02801.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Flow diagram illustrating the data parsing involved in the GeneBase development. ‘Gene_Summary’, ‘Gene_Table’ and ‘Gene_Ontology’ are the three main related software tables.</p></div></div><p>For the example application that we have presented here, we have selected ‘Gene_Table’ exon/intron records of GeneBase database belonging to genes with a ‘REVIEWED’ or ‘VALIDATED’ RefSeq status, whose corresponding RNA has an ‘NM_’ or ‘NR_’ RefSeq accession number, to exclude ‘XM_’ or ‘XR_’ model Refseq records generated by automated pipelines.<sup><a reveal-id="DSV028C20" data-open="DSV028C20" class="link link-ref link-reveal xref-bibr">20</a></sup> We downloaded the corresponding chromosome sequences from the NCBI Nucleotide database using Batch Entrez (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/sites/batchentrez">http://www.ncbi.nlm.nih.gov/sites/batchentrez</a>) in FASTA format on 5 May 2015. Then the tab-delimited file containing the corresponding exon and intron sequences obtained from script parsing was imported into the specific ‘Gene_Table’ fields.</p><p>We provide here a version of GeneBase pre-loaded with eukaryotic gene data updated on 22 April 2015, along with an empty template that may be used at any time to load <em>ab initio</em> the latest version or any desired subset of NCBI Gene data by any user, following parsing by our scripts. The import of parsed NCBI Gene entries can be performed also using a pre-loaded version of GeneBase without deleting previously imported records, if desired. In any case, we plan to release an updated version of our eukaryote GeneBase each year, for the convenience of users.</p><p>We made stand-alone software (of both GeneBase pre-loaded and empty versions), including the FileMaker runtime with a user guide included and the relative Python scripts for the initial data pre-processing and sequence calculations, freely available to basic users at <a class="link link-uri" href="http://apollo11.isto.unibo.it/software/">http://apollo11.isto.unibo.it/software/</a>. The freely distributed licensed runtime application allows full data import, records export in diverse file formats, as well as full record management and analysis and script execution. The user can also export sequence data contained in GeneBase in an automatically generated FASTA formatted text file, allowing further processing of these data, e.g. their use as a target database by a locally installed version of BLAST. The downloading, parsing and import of gene entries, the downloading of chromosome sequences and the calculation of exon and intron sequences are described in detail in the software documentation.</p><p>Only for the creation of new fields, further calculation or additional relationship definition an original copy of FileMaker Pro version 12 (or higher) is required.</p>                            <h3 scrollto-destination=87201628 id="87201628" class="section-title" data-legacy-id=s2b>2.2. Example application</h3>
<p>The fully structured local database obtained (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv028/1/dsv028_Supplementary_Data.zip?Expires=2147483647&Signature=aRYnjW0Rb4P1z~LqsvpBA4F8yDeKnemALjuIs5dIu7b78ZIyCC9vStwlmcaLpwYHHynJbVrUGwOnQs3sBQ7xB6WMmPBg1XPcJaNy2tx~WiUOnNubYatD8rrHP4-b0kqD26V-9w-T5~7ViNvHPhtETEQ4Qe0MRM7nMeiJQfNtapr3~Sd2ReGizplxkP2zCfwUUBhYjaeRaKj4EGiQwmBeXrBErc~TaNkS~uGM0gU0lcrArfMkDjnNsYKFTvojR-CLBO9iYZel7PwBnhK6Iu2q-GunrXWXN3nleQzH4Vl5aPuE12cNg9dJn3hbWN8XW-hFI-vG6tJ~WhfJ0dVGoR3IGQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>) allows users to make many types of comparisons and do a variety of calculations that are useful for data retrieval and analysis.</p><p>As an example of how to use GeneBase, we have provided an original analysis of the existing introns across all the available species. To define the minimal intron (the minimal known DNA sequence element that can function as an intron), we queried GeneBase for introns of between 1 and 40 bp, considering only genes currently annotated on the most recent genome annotation (excluding records with ‘Genome_Annontation_Status’: ‘not in current annotation release’). Among the retrieved records, we only considered for bioinformatic validation introns belonging to gene entries with a ‘REVIEWED’ or ‘VALIDATED’ RefSeq status, with an ‘NM_’ or ‘NR_’ type of corresponding RefSeq RNA accession number and with the canonical splicing sites (GT and AG for donor and acceptor sites, respectively), thus focusing on the conventional splicing mechanism.<sup><a reveal-id="DSV028C23" data-open="DSV028C23" class="link link-ref link-reveal xref-bibr">23</a></sup> We excluded from the analysis sequences predicted only by annotation of genomic sequence and thus lacking experimental evidence. We used BLASTN, the standard Nucleotide BLAST (<a class="link link-uri" href="https://blast.ncbi.nlm.nih.gov/?Blast.cgiCMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=References">https://blast.ncbi.nlm.nih.gov/?Blast.cgiCMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=References</a>),<sup><a reveal-id="DSV028C21" data-open="DSV028C21" class="link link-ref link-reveal xref-bibr">21</a></sup> to find at least two independent expressed sequence tag (EST) sequences, or one manually curated RNA sequence, that encompass the intron, as a proof of existence of intron sequences. We queried the chromosome accession of each considered intron, using the previous exon start and the following exon end coordinate range as ‘Query subrange’, limiting the searches to the ‘Nucleotide collection nr/nt’ and then to the ‘Expressed sequence tags (est)’ databases, and to the organism to which the intron sequence belongs. The analysis was carried out in May 2015.</p><p>The validated intron sequences were studied for the presence of the well-known functionally important sites, e.g. the branch point and the poly-pyrimidine tract, using the following eukaryotic consensus reference sequence: MAGGTRAGT…YNYYRAYY…YYYYYYYYYYYNYCAGG, where M = A/C, R = A/G, Y = C/T, N = any base.<sup><a reveal-id="DSV028C16" data-open="DSV028C16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="DSV028C24" data-open="DSV028C24" class="link link-ref link-reveal xref-bibr">24</a></sup></p><p>Furthermore, the 20 shortest human canonical intron sequences available in GeneBase, with a ‘REVIEWED’ or ‘VALIDATED’ RefSeq status and an ‘NM_’ or ‘NR_’ type of corresponding RefSeq RNA accession number, were analysed for the guanine (G) content.<sup><a reveal-id="DSV028C25" data-open="DSV028C25" class="link link-ref link-reveal xref-bibr">25</a></sup></p><p>Where indicated, the number of organisms was calculated by sorting records by organism and then exporting them to generate a summary report of the organisms present in the database.</p><p>The total number of currently annotated exon, coding exon or intron records, with a ‘REVIEWED’ or ‘VALIDATED’ RefSeq status and with an ‘NM_’ or ‘NR_’ type of corresponding RefSeq RNA accession number, were retrieved typing: ‘REVIEWED’ or ‘VALIDATED’ in the ‘RefSeq_Status’ field of ‘Gene_Table’ table, ‘N*’ in the ‘RefSeq_RNA_Accession’ field, ‘&gt;0’ in the corresponding length fields and omitting entries marked as ‘not in current annotation release’ in the ‘Genome_Annotation_Status’ field. A set of non-redundant intron records can be retrieved typing ‘Yes’ in the ‘Non_Redundant_Intron’ field (Fig. <a reveal-id="DSV028F2" data-open="DSV028F2" class="link link-reveal link-table xref-fig">2</a>). </p>                        <a id="87201635" scrollto-destination="87201635"></a>
<div id="DSV028F2" data-id="DSV028F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv028/1/m_dsv02802.png?Expires=2147483647&Signature=YDk2LchdlgZSbhA8OWAUenQWET6DQh2XpMW-Sd-aUWttrvSc22IIBRWCasP6vatSw8EJ5Npo~9mulX2YOghlk5DXtcw4di6kWijPLaGO6EIF~QCBjIrfXbiE4SrmojGvPfTMLcogqWHFirXxWzZ8na98H6qk1rwOvy5LYZKqXbknfrrGF4UiTDKIhCM5DVPnJ7hGENFfm1CHw4lvvjvObmj7OMhsf~WGUOFR0tQeKPGr9J6yJTEJ0vPPaFL9oHLacQLV-D4kUkMX2hbO4nvg5OdHgIRQPnaiXYegUCY6EE1VRHyCmw5ftgIikQ8Iz0bi7HIQb-5NaWDpS7DAVaC-3g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Screen shot of GeneBase ‘Gene_Table’ interface. Fields are described in detail in the software documentation. Exon and coding exon sequences are in distinct fields to allow an independent management of data (see Materials and methods). This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv02802.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87201635/dsv02802.tiff" path-from-xml="dsv02802.tiff" target="_blank">View large</a><a section="87201635" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv028/1/dsv02802.png?Expires=2147483647&Signature=c2ZNmX27-KNPbBYBl5byE6GSEiNxLYYqQLTqQwIVeBUbLyciA0~Bv5G7xEbd2dL8z3~5lcBsOndvO9je2toSfeRO4vKfCqC50j66XfqCMKaz1NWYRBNQO3cCqfTa0Iw-k~yc3LK3GQxOGPdyj~NGktwWutGKuRyYkiNFsEnC2Zcji6l0CE28JjhjN2XbAipe9BWACrHFmeQKh4Lj3s5jf~v2taa5SmMYBZJ9fsZD7h5WSa87GFKqM4X9LUukx0QEZHTfMhhNkBKFVCeKi5XqqwKUAB~ynUVj2ZTWrQMKZCznKZ9NG-WYmTZdItThxhi0M1n7UoX4fKVRNDL24TlWfg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87201635&ar=2462510&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02802.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Screen shot of GeneBase ‘Gene_Table’ interface. Fields are described in detail in the software documentation. Exon and coding exon sequences are in distinct fields to allow an independent management of data (see Materials and methods). This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSV028F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv028/1/m_dsv02802.png?Expires=2147483647&Signature=YDk2LchdlgZSbhA8OWAUenQWET6DQh2XpMW-Sd-aUWttrvSc22IIBRWCasP6vatSw8EJ5Npo~9mulX2YOghlk5DXtcw4di6kWijPLaGO6EIF~QCBjIrfXbiE4SrmojGvPfTMLcogqWHFirXxWzZ8na98H6qk1rwOvy5LYZKqXbknfrrGF4UiTDKIhCM5DVPnJ7hGENFfm1CHw4lvvjvObmj7OMhsf~WGUOFR0tQeKPGr9J6yJTEJ0vPPaFL9oHLacQLV-D4kUkMX2hbO4nvg5OdHgIRQPnaiXYegUCY6EE1VRHyCmw5ftgIikQ8Iz0bi7HIQb-5NaWDpS7DAVaC-3g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Screen shot of GeneBase ‘Gene_Table’ interface. Fields are described in detail in the software documentation. Exon and coding exon sequences are in distinct fields to allow an independent management of data (see Materials and methods). This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv02802.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87201635/dsv02802.tiff" path-from-xml="dsv02802.tiff" target="_blank">View large</a><a section="87201635" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv028/1/dsv02802.png?Expires=2147483647&Signature=c2ZNmX27-KNPbBYBl5byE6GSEiNxLYYqQLTqQwIVeBUbLyciA0~Bv5G7xEbd2dL8z3~5lcBsOndvO9je2toSfeRO4vKfCqC50j66XfqCMKaz1NWYRBNQO3cCqfTa0Iw-k~yc3LK3GQxOGPdyj~NGktwWutGKuRyYkiNFsEnC2Zcji6l0CE28JjhjN2XbAipe9BWACrHFmeQKh4Lj3s5jf~v2taa5SmMYBZJ9fsZD7h5WSa87GFKqM4X9LUukx0QEZHTfMhhNkBKFVCeKi5XqqwKUAB~ynUVj2ZTWrQMKZCznKZ9NG-WYmTZdItThxhi0M1n7UoX4fKVRNDL24TlWfg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87201635&ar=2462510&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02802.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Screen shot of GeneBase ‘Gene_Table’ interface. Fields are described in detail in the software documentation. Exon and coding exon sequences are in distinct fields to allow an independent management of data (see Materials and methods). This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><p>Intron length distribution graphs and correlation analyses between intron and genome lengths among selected organisms were done using the R package (<a class="link link-uri" href="http://www.r-project.org/">http://www.r-project.org/</a>).</p>                            <h2 scrollto-destination=87201637 id="87201637" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=87201638 id="87201638" class="section-title" data-legacy-id=s3a>3.1. Database construction</h3>
<p>We downloaded from NCBI Gene all current (alive/live) eukaryotic records with a genomic gene source (excluding gene models) available up to 22 April 2015. We obtained 679,451 entries for Animalia (Metazoa, Taxonomy ID: 33208), 1,203,082 for Fungi (Taxonomy ID: 4751) and 534,875 for Plants (Viridiplantae, Taxonomy ID: 33090), for a total of 359 organisms. Among the 2,417,408 total gene entries, 76,182 are ‘REVIEWED’, 41,862 ‘VALIDATED’, 2,245,205 ‘PROVISIONAL’, 31,464 ‘INFERRED’, 22,691 ‘PREDICTED’, 1 ‘MODEL’ and 1 ‘WITHDRAWN’ (despite the gene model exclusion performed using the web search described in the GeneBase guide; the remaining two gene entries are not specified).</p><p>After the initial parsing and importing steps, the three main tables in GeneBase database are constituted as follows: ‘Gene_Summary’ contains 2,417,408 records (one for each NCBI Gene entry), ‘Gene_Table’ (Fig. <a reveal-id="DSV028F2" data-open="DSV028F2" class="link link-reveal link-table xref-fig">2</a>) contains 13,824,965 records (one record for each gene exon, included the corresponding intron if an intron follows that exon) and ‘Gene_Ontology’ contains 149,064 records in all (one for each gene with GO information available). Due to the lack of annotated transcribed products, a gene structure was not available for 86,824 Gene IDs (explaining the presence of ‘Gene_Summary’ gene entries without or with incomplete corresponding ‘Gene_Table’ records, for example Gene ID: 105667210 and Gene ID: 100121140, respectively).</p><p>Among the total gene entries, 2,368,726 are protein-coding, 25,796 pseudogenes (pseudo), 21,247 non-coding RNA (ncRNA), 527 coding for small nucleolar RNA (snoRNA), 137 for small nuclear RNA (snRNA), 86 for ribosomal RNA (rRNA) and 6 for cytoplasmic RNA genes (scRNA) (the remaining are not specified).</p><p>Then, to integrate nucleotide sequences, from the ‘Gene_Table’ table of our database, we selected 861,550 records with the ‘REVIEWED’ RefSeq status and 534,578 with the ‘VALIDATED’ RefSeq status (in both cases having an ‘NM_’ or ‘NR_’ type of RefSeq RNA accession number, to exclude ‘XM_’ or ‘XR_’ model Refseq records generated by automated pipelines) for a total of 1,396,128 exon entries. Using Batch Entrez, we were able to retrieve and download 1,336 records out of the 1,338 corresponding chromosome sequences. This selection gave rise to a total of 1,385,944 ‘Gene_Table’ records, which represent 10% of all available entries, updated with exon, coding exon (for protein-coding genes) and the corresponding downstream intron sequences up to 5 May 2015. The whole database including sequences has a size of 25.1 GB following decompression.</p>                            <h3 scrollto-destination=87201643 id="87201643" class="section-title" data-legacy-id=s3b>3.2. Statistics</h3>
<p>In GeneBase, an additional table named ‘Reports’ is generated to easily provide statistics such as the mean length of exons (both coding and not coding) and introns (Table <a reveal-id="DSV028TB1" data-open="DSV028TB1" class="link link-reveal link-table xref-fig">1</a>). These values were obtained considering all the available records in the ‘Gene_Table’ table of GeneBase database belonging to gene entries currently annotated on the most recent genome annotation, with a ‘REVIEWED’ or ‘VALIDATED’ RefSeq status and with an ‘NM_’ or ‘NR_’ type of the corresponding RefSeq RNA entry accession number. The mean exon length for all organisms is 308 base pairs (bp) with a standard deviation (SD) of 613 (range 1–91,671). In total, 1,252,462 GeneBase records are related to coding exons. The mean coding exon length is 206 bp with a SD of 325 (range 1–27,708). The mean intron length for all organisms is 3,820 bp with a SD of 15,693 (range 1–1,160,411), and the overall intron length distribution is shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv028/1/dsv028_Supplementary_Data.zip?Expires=2147483647&Signature=aRYnjW0Rb4P1z~LqsvpBA4F8yDeKnemALjuIs5dIu7b78ZIyCC9vStwlmcaLpwYHHynJbVrUGwOnQs3sBQ7xB6WMmPBg1XPcJaNy2tx~WiUOnNubYatD8rrHP4-b0kqD26V-9w-T5~7ViNvHPhtETEQ4Qe0MRM7nMeiJQfNtapr3~Sd2ReGizplxkP2zCfwUUBhYjaeRaKj4EGiQwmBeXrBErc~TaNkS~uGM0gU0lcrArfMkDjnNsYKFTvojR-CLBO9iYZel7PwBnhK6Iu2q-GunrXWXN3nleQzH4Vl5aPuE12cNg9dJn3hbWN8XW-hFI-vG6tJ~WhfJ0dVGoR3IGQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>. Other intron length statistics for some representative organisms are provided in Table <a reveal-id="DSV028TB2" data-open="DSV028TB2" class="link link-reveal link-table xref-fig">2</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv028/1/dsv028_Supplementary_Data.zip?Expires=2147483647&Signature=aRYnjW0Rb4P1z~LqsvpBA4F8yDeKnemALjuIs5dIu7b78ZIyCC9vStwlmcaLpwYHHynJbVrUGwOnQs3sBQ7xB6WMmPBg1XPcJaNy2tx~WiUOnNubYatD8rrHP4-b0kqD26V-9w-T5~7ViNvHPhtETEQ4Qe0MRM7nMeiJQfNtapr3~Sd2ReGizplxkP2zCfwUUBhYjaeRaKj4EGiQwmBeXrBErc~TaNkS~uGM0gU0lcrArfMkDjnNsYKFTvojR-CLBO9iYZel7PwBnhK6Iu2q-GunrXWXN3nleQzH4Vl5aPuE12cNg9dJn3hbWN8XW-hFI-vG6tJ~WhfJ0dVGoR3IGQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>, considering a non-redundant set. The correlation coefficient between intron and genome lengths among these organisms is 0.957 with a <em>P</em>-value of 5.302e<sup>−05</sup> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv028/1/dsv028_Supplementary_Data.zip?Expires=2147483647&Signature=aRYnjW0Rb4P1z~LqsvpBA4F8yDeKnemALjuIs5dIu7b78ZIyCC9vStwlmcaLpwYHHynJbVrUGwOnQs3sBQ7xB6WMmPBg1XPcJaNy2tx~WiUOnNubYatD8rrHP4-b0kqD26V-9w-T5~7ViNvHPhtETEQ4Qe0MRM7nMeiJQfNtapr3~Sd2ReGizplxkP2zCfwUUBhYjaeRaKj4EGiQwmBeXrBErc~TaNkS~uGM0gU0lcrArfMkDjnNsYKFTvojR-CLBO9iYZel7PwBnhK6Iu2q-GunrXWXN3nleQzH4Vl5aPuE12cNg9dJn3hbWN8XW-hFI-vG6tJ~WhfJ0dVGoR3IGQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>) while, considering <em>Vertebrata</em> (Taxonomy ID: 7742), the correlation coefficient is 0.968 with a <em>P</em>-value of 0.007 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv028/1/dsv028_Supplementary_Data.zip?Expires=2147483647&Signature=aRYnjW0Rb4P1z~LqsvpBA4F8yDeKnemALjuIs5dIu7b78ZIyCC9vStwlmcaLpwYHHynJbVrUGwOnQs3sBQ7xB6WMmPBg1XPcJaNy2tx~WiUOnNubYatD8rrHP4-b0kqD26V-9w-T5~7ViNvHPhtETEQ4Qe0MRM7nMeiJQfNtapr3~Sd2ReGizplxkP2zCfwUUBhYjaeRaKj4EGiQwmBeXrBErc~TaNkS~uGM0gU0lcrArfMkDjnNsYKFTvojR-CLBO9iYZel7PwBnhK6Iu2q-GunrXWXN3nleQzH4Vl5aPuE12cNg9dJn3hbWN8XW-hFI-vG6tJ~WhfJ0dVGoR3IGQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span>). </p>                        <a id="87201645" scrollto-destination="87201645"></a>
<div content-id="DSV028TB1" class="table-modal"><div class="table-wrap"><div id="DSV028TB1" class="table-wrap-title" data-id="DSV028TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Statistical analysis of exon and intron lengths</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Number of records<sup>a</sup> </th><th>Mean length (bp) </th><th>Standard deviation (bp) </th><th>Minimum length (bp)<sup>b</sup> </th><th>Maximum length (bp) </th></tr></thead><tbody><tr><td>Exons </td><td>1,396,026 </td><td>308 </td><td>613 </td><td>2 </td><td>91,671 </td></tr><tr><td>Coding exons </td><td>1,252,462 </td><td>206 </td><td>325 </td><td>2 </td><td>27,708 </td></tr><tr><td>Introns </td><td>1,219,806 </td><td>3,820 </td><td>15,693 </td><td>30 </td><td>1,160,411 </td></tr></tbody></table></div><div content-id="DSV028TB1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Number of records<sup>a</sup> </th><th>Mean length (bp) </th><th>Standard deviation (bp) </th><th>Minimum length (bp)<sup>b</sup> </th><th>Maximum length (bp) </th></tr></thead><tbody><tr><td>Exons </td><td>1,396,026 </td><td>308 </td><td>613 </td><td>2 </td><td>91,671 </td></tr><tr><td>Coding exons </td><td>1,252,462 </td><td>206 </td><td>325 </td><td>2 </td><td>27,708 </td></tr><tr><td>Introns </td><td>1,219,806 </td><td>3,820 </td><td>15,693 </td><td>30 </td><td>1,160,411 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>The analysis was carried out considering only ‘Gene_Table’ records belonging to gene entries with a ‘REVIEWED’ or ‘VALIDATED’ RefSeq status and with an ‘NM_’ or ‘NR_’ type of corresponding RefSeq RNA accession number, omitting entries marked as ‘not in current annotation release’ in the ‘Genome_Annotation_Status’ field (see Materials and methods). Mean and standard deviation values were obtained from the ‘Reports’ database table calculation fields. Lengths are given in base pairs (bp).</p></div></div><div content-id="tblfn2"><div class="fn"><p><sup>a</sup>Common exons and introns belonging to multiple transcript variants are counted multiple times. The existence of intronless genes and the fact that terminal exons are not followed by an intron account for a reduced number of introns in comparison with exons.</p></div></div><div content-id="tblfn3"><div class="fn"><p><sup>b</sup>Minimum exon and intron length determination is subject to the annotation artifacts described in the text, so only the manually verified data are shown here.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87201645" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSV028TB1" class="table-wrap-title" data-id="DSV028TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Statistical analysis of exon and intron lengths</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Number of records<sup>a</sup> </th><th>Mean length (bp) </th><th>Standard deviation (bp) </th><th>Minimum length (bp)<sup>b</sup> </th><th>Maximum length (bp) </th></tr></thead><tbody><tr><td>Exons </td><td>1,396,026 </td><td>308 </td><td>613 </td><td>2 </td><td>91,671 </td></tr><tr><td>Coding exons </td><td>1,252,462 </td><td>206 </td><td>325 </td><td>2 </td><td>27,708 </td></tr><tr><td>Introns </td><td>1,219,806 </td><td>3,820 </td><td>15,693 </td><td>30 </td><td>1,160,411 </td></tr></tbody></table></div><div content-id="DSV028TB1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Number of records<sup>a</sup> </th><th>Mean length (bp) </th><th>Standard deviation (bp) </th><th>Minimum length (bp)<sup>b</sup> </th><th>Maximum length (bp) </th></tr></thead><tbody><tr><td>Exons </td><td>1,396,026 </td><td>308 </td><td>613 </td><td>2 </td><td>91,671 </td></tr><tr><td>Coding exons </td><td>1,252,462 </td><td>206 </td><td>325 </td><td>2 </td><td>27,708 </td></tr><tr><td>Introns </td><td>1,219,806 </td><td>3,820 </td><td>15,693 </td><td>30 </td><td>1,160,411 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>The analysis was carried out considering only ‘Gene_Table’ records belonging to gene entries with a ‘REVIEWED’ or ‘VALIDATED’ RefSeq status and with an ‘NM_’ or ‘NR_’ type of corresponding RefSeq RNA accession number, omitting entries marked as ‘not in current annotation release’ in the ‘Genome_Annotation_Status’ field (see Materials and methods). Mean and standard deviation values were obtained from the ‘Reports’ database table calculation fields. Lengths are given in base pairs (bp).</p></div></div><div content-id="tblfn2"><div class="fn"><p><sup>a</sup>Common exons and introns belonging to multiple transcript variants are counted multiple times. The existence of intronless genes and the fact that terminal exons are not followed by an intron account for a reduced number of introns in comparison with exons.</p></div></div><div content-id="tblfn3"><div class="fn"><p><sup>b</sup>Minimum exon and intron length determination is subject to the annotation artifacts described in the text, so only the manually verified data are shown here.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87201645" target="_blank">View Large</a></div></div>                        <a id="87201646" scrollto-destination="87201646"></a>
<div content-id="DSV028TB2" class="table-modal"><div class="table-wrap"><div id="DSV028TB2" class="table-wrap-title" data-id="DSV028TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Statistical analysis of intron length of some representative organisms</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Organism (Taxonomy ID) </th><th>Number of introns </th><th>Mean length (bp) </th><th>Standard deviation (bp) </th><th>Minimum length (bp)<sup>a</sup> </th><th>Maximum length (bp) </th></tr></thead><tbody><tr><td><em>Arabidopsis thaliana</em> (3702) </td><td>124,533 </td><td>169 </td><td>194 </td><td>— </td><td>11,602 </td></tr><tr><td><em>Caenorhabditis elegans</em> (6239) </td><td>107,605 </td><td>324 </td><td>803 </td><td>39 </td><td>100,913 </td></tr><tr><td><em>Tribolium castaneum</em> (7070) </td><td>434 </td><td>1,883 </td><td>5,592 </td><td>35 </td><td>53,400 </td></tr><tr><td><em>Drosophila melanogaster</em> (7227) </td><td>58,480 </td><td>1,657 </td><td>5,841 </td><td>40 </td><td>189,627 </td></tr><tr><td><em>Xenopus tropicalis</em> (8364) </td><td>4,174 </td><td>2,849 </td><td>7,646 </td><td>— </td><td>160,644 </td></tr><tr><td><em>Danio rerio</em> (7955) </td><td>10,306 </td><td>2,281 </td><td>5,578 </td><td>— </td><td>170,685 </td></tr><tr><td><em>Gallus gallus</em> (9031) </td><td>14,706 </td><td>2,856 </td><td>9,719 </td><td>— </td><td>351,090 </td></tr><tr><td><em>Mus musculus</em> (10090) </td><td>85,507 </td><td>5,622 </td><td>20,369 </td><td>— </td><td>1,041,985 </td></tr><tr><td><em>Homo sapiens</em> (9606) </td><td>155,222 </td><td>7,386 </td><td>24,002 </td><td>30 </td><td>1,160,411 </td></tr></tbody></table></div><div content-id="DSV028TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Organism (Taxonomy ID) </th><th>Number of introns </th><th>Mean length (bp) </th><th>Standard deviation (bp) </th><th>Minimum length (bp)<sup>a</sup> </th><th>Maximum length (bp) </th></tr></thead><tbody><tr><td><em>Arabidopsis thaliana</em> (3702) </td><td>124,533 </td><td>169 </td><td>194 </td><td>— </td><td>11,602 </td></tr><tr><td><em>Caenorhabditis elegans</em> (6239) </td><td>107,605 </td><td>324 </td><td>803 </td><td>39 </td><td>100,913 </td></tr><tr><td><em>Tribolium castaneum</em> (7070) </td><td>434 </td><td>1,883 </td><td>5,592 </td><td>35 </td><td>53,400 </td></tr><tr><td><em>Drosophila melanogaster</em> (7227) </td><td>58,480 </td><td>1,657 </td><td>5,841 </td><td>40 </td><td>189,627 </td></tr><tr><td><em>Xenopus tropicalis</em> (8364) </td><td>4,174 </td><td>2,849 </td><td>7,646 </td><td>— </td><td>160,644 </td></tr><tr><td><em>Danio rerio</em> (7955) </td><td>10,306 </td><td>2,281 </td><td>5,578 </td><td>— </td><td>170,685 </td></tr><tr><td><em>Gallus gallus</em> (9031) </td><td>14,706 </td><td>2,856 </td><td>9,719 </td><td>— </td><td>351,090 </td></tr><tr><td><em>Mus musculus</em> (10090) </td><td>85,507 </td><td>5,622 </td><td>20,369 </td><td>— </td><td>1,041,985 </td></tr><tr><td><em>Homo sapiens</em> (9606) </td><td>155,222 </td><td>7,386 </td><td>24,002 </td><td>30 </td><td>1,160,411 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn4"><div class="fn"><p>The analysis was carried out considering a set of non-redundant ‘Gene_Table’ records belonging to gene entries with a ‘REVIEWED’ or ‘VALIDATED’ RefSeq status and with an ‘NM_’ or ‘NR_’ type of corresponding RefSeq RNA accession number, omitting entries marked as ‘not in current annotation release’ in the ‘Genome_Annotation_Status’ field (see Materials and methods). Mean and standard deviation values were obtained from the ‘Reports’ database table calculation fields. Lengths are given in base pairs (bp).</p></div></div><div content-id="tblfn5"><div class="fn"><p><sup>a</sup>Minimum length determination is subject to the annotation artifacts described in the text, so only the manually verified data for <em>C. elegans</em>, <em>T. castaneum</em>, <em>D. melanogaster</em> and <em>H. sapiens</em> are shown here.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87201646" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSV028TB2" class="table-wrap-title" data-id="DSV028TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Statistical analysis of intron length of some representative organisms</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Organism (Taxonomy ID) </th><th>Number of introns </th><th>Mean length (bp) </th><th>Standard deviation (bp) </th><th>Minimum length (bp)<sup>a</sup> </th><th>Maximum length (bp) </th></tr></thead><tbody><tr><td><em>Arabidopsis thaliana</em> (3702) </td><td>124,533 </td><td>169 </td><td>194 </td><td>— </td><td>11,602 </td></tr><tr><td><em>Caenorhabditis elegans</em> (6239) </td><td>107,605 </td><td>324 </td><td>803 </td><td>39 </td><td>100,913 </td></tr><tr><td><em>Tribolium castaneum</em> (7070) </td><td>434 </td><td>1,883 </td><td>5,592 </td><td>35 </td><td>53,400 </td></tr><tr><td><em>Drosophila melanogaster</em> (7227) </td><td>58,480 </td><td>1,657 </td><td>5,841 </td><td>40 </td><td>189,627 </td></tr><tr><td><em>Xenopus tropicalis</em> (8364) </td><td>4,174 </td><td>2,849 </td><td>7,646 </td><td>— </td><td>160,644 </td></tr><tr><td><em>Danio rerio</em> (7955) </td><td>10,306 </td><td>2,281 </td><td>5,578 </td><td>— </td><td>170,685 </td></tr><tr><td><em>Gallus gallus</em> (9031) </td><td>14,706 </td><td>2,856 </td><td>9,719 </td><td>— </td><td>351,090 </td></tr><tr><td><em>Mus musculus</em> (10090) </td><td>85,507 </td><td>5,622 </td><td>20,369 </td><td>— </td><td>1,041,985 </td></tr><tr><td><em>Homo sapiens</em> (9606) </td><td>155,222 </td><td>7,386 </td><td>24,002 </td><td>30 </td><td>1,160,411 </td></tr></tbody></table></div><div content-id="DSV028TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Organism (Taxonomy ID) </th><th>Number of introns </th><th>Mean length (bp) </th><th>Standard deviation (bp) </th><th>Minimum length (bp)<sup>a</sup> </th><th>Maximum length (bp) </th></tr></thead><tbody><tr><td><em>Arabidopsis thaliana</em> (3702) </td><td>124,533 </td><td>169 </td><td>194 </td><td>— </td><td>11,602 </td></tr><tr><td><em>Caenorhabditis elegans</em> (6239) </td><td>107,605 </td><td>324 </td><td>803 </td><td>39 </td><td>100,913 </td></tr><tr><td><em>Tribolium castaneum</em> (7070) </td><td>434 </td><td>1,883 </td><td>5,592 </td><td>35 </td><td>53,400 </td></tr><tr><td><em>Drosophila melanogaster</em> (7227) </td><td>58,480 </td><td>1,657 </td><td>5,841 </td><td>40 </td><td>189,627 </td></tr><tr><td><em>Xenopus tropicalis</em> (8364) </td><td>4,174 </td><td>2,849 </td><td>7,646 </td><td>— </td><td>160,644 </td></tr><tr><td><em>Danio rerio</em> (7955) </td><td>10,306 </td><td>2,281 </td><td>5,578 </td><td>— </td><td>170,685 </td></tr><tr><td><em>Gallus gallus</em> (9031) </td><td>14,706 </td><td>2,856 </td><td>9,719 </td><td>— </td><td>351,090 </td></tr><tr><td><em>Mus musculus</em> (10090) </td><td>85,507 </td><td>5,622 </td><td>20,369 </td><td>— </td><td>1,041,985 </td></tr><tr><td><em>Homo sapiens</em> (9606) </td><td>155,222 </td><td>7,386 </td><td>24,002 </td><td>30 </td><td>1,160,411 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn4"><div class="fn"><p>The analysis was carried out considering a set of non-redundant ‘Gene_Table’ records belonging to gene entries with a ‘REVIEWED’ or ‘VALIDATED’ RefSeq status and with an ‘NM_’ or ‘NR_’ type of corresponding RefSeq RNA accession number, omitting entries marked as ‘not in current annotation release’ in the ‘Genome_Annotation_Status’ field (see Materials and methods). Mean and standard deviation values were obtained from the ‘Reports’ database table calculation fields. Lengths are given in base pairs (bp).</p></div></div><div content-id="tblfn5"><div class="fn"><p><sup>a</sup>Minimum length determination is subject to the annotation artifacts described in the text, so only the manually verified data for <em>C. elegans</em>, <em>T. castaneum</em>, <em>D. melanogaster</em> and <em>H. sapiens</em> are shown here.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87201646" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=87201647 id="87201647" class="section-title" data-legacy-id=s3c>3.3. Search for the minimal intron</h3>
<p>We found 118,942 ‘Gene_Table’ records related to an intron sequence with a length range between 1 and 40 bp (Table <a reveal-id="DSV028TB3" data-open="DSV028TB3" class="link link-reveal link-table xref-fig">3</a>). Among the entries with ‘REVIEWED’ or ‘VALIDATED’ RefSeq status, 587 intron records belong to an RNA with an ‘NM_’ or ‘NR_’ RefSeq type accession number (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv028/1/dsv028_Supplementary_Data.zip?Expires=2147483647&Signature=aRYnjW0Rb4P1z~LqsvpBA4F8yDeKnemALjuIs5dIu7b78ZIyCC9vStwlmcaLpwYHHynJbVrUGwOnQs3sBQ7xB6WMmPBg1XPcJaNy2tx~WiUOnNubYatD8rrHP4-b0kqD26V-9w-T5~7ViNvHPhtETEQ4Qe0MRM7nMeiJQfNtapr3~Sd2ReGizplxkP2zCfwUUBhYjaeRaKj4EGiQwmBeXrBErc~TaNkS~uGM0gU0lcrArfMkDjnNsYKFTvojR-CLBO9iYZel7PwBnhK6Iu2q-GunrXWXN3nleQzH4Vl5aPuE12cNg9dJn3hbWN8XW-hFI-vG6tJ~WhfJ0dVGoR3IGQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). Considering the presence of the canonical GT-AG splicing sites as a prerequisite, a subset of 170 introns was used for the bioinformatic validation of the minimal intron (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv028/1/dsv028_Supplementary_Data.zip?Expires=2147483647&Signature=aRYnjW0Rb4P1z~LqsvpBA4F8yDeKnemALjuIs5dIu7b78ZIyCC9vStwlmcaLpwYHHynJbVrUGwOnQs3sBQ7xB6WMmPBg1XPcJaNy2tx~WiUOnNubYatD8rrHP4-b0kqD26V-9w-T5~7ViNvHPhtETEQ4Qe0MRM7nMeiJQfNtapr3~Sd2ReGizplxkP2zCfwUUBhYjaeRaKj4EGiQwmBeXrBErc~TaNkS~uGM0gU0lcrArfMkDjnNsYKFTvojR-CLBO9iYZel7PwBnhK6Iu2q-GunrXWXN3nleQzH4Vl5aPuE12cNg9dJn3hbWN8XW-hFI-vG6tJ~WhfJ0dVGoR3IGQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S1 and S2</a></span>). Through BLASTN software, we were able to find at least two independent EST sequences, or one manually curated RNA sequence, encompassing the intron as validation of the existence of intron sequences, for 15 introns only (Table <a reveal-id="DSV028TB4" data-open="DSV028TB4" class="link link-reveal link-table xref-fig">4</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv028/1/dsv028_Supplementary_Data.zip?Expires=2147483647&Signature=aRYnjW0Rb4P1z~LqsvpBA4F8yDeKnemALjuIs5dIu7b78ZIyCC9vStwlmcaLpwYHHynJbVrUGwOnQs3sBQ7xB6WMmPBg1XPcJaNy2tx~WiUOnNubYatD8rrHP4-b0kqD26V-9w-T5~7ViNvHPhtETEQ4Qe0MRM7nMeiJQfNtapr3~Sd2ReGizplxkP2zCfwUUBhYjaeRaKj4EGiQwmBeXrBErc~TaNkS~uGM0gU0lcrArfMkDjnNsYKFTvojR-CLBO9iYZel7PwBnhK6Iu2q-GunrXWXN3nleQzH4Vl5aPuE12cNg9dJn3hbWN8XW-hFI-vG6tJ~WhfJ0dVGoR3IGQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S2 and S3</a></span>). </p>                        <a id="87201649" scrollto-destination="87201649"></a>
<div content-id="DSV028TB3" class="table-modal"><div class="table-wrap"><div id="DSV028TB3" class="table-wrap-title" data-id="DSV028TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Number of retrieved GeneBase records and corresponding intron lengths</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Intron length queried (bp) </th><th colspan="6">Number of retrieved records<hr /> </th></tr><tr><th>REVIEWED </th><th>VALIDATED </th><th>PREDICTED </th><th>PROVISIONAL </th><th>INFERRED </th><th>EMPTY </th></tr></thead><tbody><tr><td>1–10 </td><td>21 </td><td>5 </td><td>2 </td><td>2,822 </td><td>1 </td><td>954 </td></tr><tr><td>11–20 </td><td>22 </td><td>1 </td><td>4 </td><td>8,250 </td><td>1 </td><td>7,291 </td></tr><tr><td>21–30 </td><td>63 </td><td>13 </td><td>37 </td><td>32,487 </td><td>2 </td><td>4,269 </td></tr><tr><td>31–40 </td><td>442 </td><td>20 </td><td>38 </td><td>58,242 </td><td>8 </td><td>3,947 </td></tr><tr><td>Total </td><td>548 </td><td>39 </td><td>81 </td><td>101,801 </td><td>12 </td><td>16,461 </td></tr></tbody></table></div><div content-id="DSV028TB3" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Intron length queried (bp) </th><th colspan="6">Number of retrieved records<hr /> </th></tr><tr><th>REVIEWED </th><th>VALIDATED </th><th>PREDICTED </th><th>PROVISIONAL </th><th>INFERRED </th><th>EMPTY </th></tr></thead><tbody><tr><td>1–10 </td><td>21 </td><td>5 </td><td>2 </td><td>2,822 </td><td>1 </td><td>954 </td></tr><tr><td>11–20 </td><td>22 </td><td>1 </td><td>4 </td><td>8,250 </td><td>1 </td><td>7,291 </td></tr><tr><td>21–30 </td><td>63 </td><td>13 </td><td>37 </td><td>32,487 </td><td>2 </td><td>4,269 </td></tr><tr><td>31–40 </td><td>442 </td><td>20 </td><td>38 </td><td>58,242 </td><td>8 </td><td>3,947 </td></tr><tr><td>Total </td><td>548 </td><td>39 </td><td>81 </td><td>101,801 </td><td>12 </td><td>16,461 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn6"><div class="fn"><p>Lengths are given in base pairs (bp). The total number of ‘Gene_Table’ records with a currently annotated intron length between 1 and 40 bp is 118,942. Common introns belonging to multiple transcript variants are counted multiple times.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87201649" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSV028TB3" class="table-wrap-title" data-id="DSV028TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Number of retrieved GeneBase records and corresponding intron lengths</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Intron length queried (bp) </th><th colspan="6">Number of retrieved records<hr /> </th></tr><tr><th>REVIEWED </th><th>VALIDATED </th><th>PREDICTED </th><th>PROVISIONAL </th><th>INFERRED </th><th>EMPTY </th></tr></thead><tbody><tr><td>1–10 </td><td>21 </td><td>5 </td><td>2 </td><td>2,822 </td><td>1 </td><td>954 </td></tr><tr><td>11–20 </td><td>22 </td><td>1 </td><td>4 </td><td>8,250 </td><td>1 </td><td>7,291 </td></tr><tr><td>21–30 </td><td>63 </td><td>13 </td><td>37 </td><td>32,487 </td><td>2 </td><td>4,269 </td></tr><tr><td>31–40 </td><td>442 </td><td>20 </td><td>38 </td><td>58,242 </td><td>8 </td><td>3,947 </td></tr><tr><td>Total </td><td>548 </td><td>39 </td><td>81 </td><td>101,801 </td><td>12 </td><td>16,461 </td></tr></tbody></table></div><div content-id="DSV028TB3" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Intron length queried (bp) </th><th colspan="6">Number of retrieved records<hr /> </th></tr><tr><th>REVIEWED </th><th>VALIDATED </th><th>PREDICTED </th><th>PROVISIONAL </th><th>INFERRED </th><th>EMPTY </th></tr></thead><tbody><tr><td>1–10 </td><td>21 </td><td>5 </td><td>2 </td><td>2,822 </td><td>1 </td><td>954 </td></tr><tr><td>11–20 </td><td>22 </td><td>1 </td><td>4 </td><td>8,250 </td><td>1 </td><td>7,291 </td></tr><tr><td>21–30 </td><td>63 </td><td>13 </td><td>37 </td><td>32,487 </td><td>2 </td><td>4,269 </td></tr><tr><td>31–40 </td><td>442 </td><td>20 </td><td>38 </td><td>58,242 </td><td>8 </td><td>3,947 </td></tr><tr><td>Total </td><td>548 </td><td>39 </td><td>81 </td><td>101,801 </td><td>12 </td><td>16,461 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn6"><div class="fn"><p>Lengths are given in base pairs (bp). The total number of ‘Gene_Table’ records with a currently annotated intron length between 1 and 40 bp is 118,942. Common introns belonging to multiple transcript variants are counted multiple times.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87201649" target="_blank">View Large</a></div></div>                        <a id="87201650" scrollto-destination="87201650"></a>
<div content-id="DSV028TB4" class="table-modal"><div class="table-wrap"><div id="DSV028TB4" class="table-wrap-title" data-id="DSV028TB4"><div class="label -label">Table 4.</div><div class="caption"><p>Minimal introns validated through bioinformatic analysis (one representative intron for each available organism; see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv028/1/dsv028_Supplementary_Data.zip?Expires=2147483647&Signature=aRYnjW0Rb4P1z~LqsvpBA4F8yDeKnemALjuIs5dIu7b78ZIyCC9vStwlmcaLpwYHHynJbVrUGwOnQs3sBQ7xB6WMmPBg1XPcJaNy2tx~WiUOnNubYatD8rrHP4-b0kqD26V-9w-T5~7ViNvHPhtETEQ4Qe0MRM7nMeiJQfNtapr3~Sd2ReGizplxkP2zCfwUUBhYjaeRaKj4EGiQwmBeXrBErc~TaNkS~uGM0gU0lcrArfMkDjnNsYKFTvojR-CLBO9iYZel7PwBnhK6Iu2q-GunrXWXN3nleQzH4Vl5aPuE12cNg9dJn3hbWN8XW-hFI-vG6tJ~WhfJ0dVGoR3IGQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S1, S2 and S3</a></span> for more details)</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Gene symbol (Gene ID) </th><th>Organism (Taxonomy ID) </th><th>RefSeq RNA accession number </th><th>Intron number </th><th>Position<sup>a</sup> </th><th>Intron length (bp) </th><th>Previous exon 3′ </th><th>Intron sequence 5′-3′<sup>b</sup> </th><th>Following exon 5′ </th><th>GenBank accession numbers<sup>c</sup> </th></tr></thead><tbody><tr><td><em>MST1L</em> (11223) </td><td><em>Homo sapiens</em> (9606) </td><td>NM_001271733.1 </td><td>9 </td><td>CDS </td><td>30 </td><td>gca </td><td>gtgagtccctggtgctcccggccccgccag
******   ***** </td><td>g<br />* </td><td>AY192149.1 </td></tr><tr><td><em>nAChRb1</em> (657999) </td><td><em>Tribolium castaneum</em> (7070) </td><td>NM_001162528.1 </td><td>8 </td><td>CDS </td><td>35 </td><td>aag<br />*** </td><td>gtaaaaatctaatcacatacccacccccattcaag
**** ****:**  ** </td><td>c </td><td>EU937812.1 </td></tr><tr><td><em>CELE_B0348.6</em> (178536) </td><td><em>Caenorhabditis elegans</em> (6239) </td><td>NM_070722.3 </td><td>2 </td><td>CDS </td><td>39 </td><td>caa<br />** </td><td>gtacgttttgagaaatatattttattcaatgaatcatag
*** **  *** :** * ** </td><td>a </td><td>CK586343.1 CK586324.1 CK586322.1 CK579517.1 </td></tr><tr><td><em>bcd</em> (40830) </td><td><em>Drosophila melanogaster</em> (7227) </td><td>NM_176410.3 </td><td>2 </td><td>CDS </td><td>40 </td><td>cag<br />*** </td><td>gtgagctcaaagccaacaaagtcagccatcgtcttatcag
*****  ** * :** ***** </td><td>a </td><td>X14459.1 BT021332.1<br />CO301329.1 BI163743.1<br />AA941658.2 </td></tr></tbody></table></div><div content-id="DSV028TB4" class="table-modal"><table><thead name="thead" align=""><tr><th>Gene symbol (Gene ID) </th><th>Organism (Taxonomy ID) </th><th>RefSeq RNA accession number </th><th>Intron number </th><th>Position<sup>a</sup> </th><th>Intron length (bp) </th><th>Previous exon 3′ </th><th>Intron sequence 5′-3′<sup>b</sup> </th><th>Following exon 5′ </th><th>GenBank accession numbers<sup>c</sup> </th></tr></thead><tbody><tr><td><em>MST1L</em> (11223) </td><td><em>Homo sapiens</em> (9606) </td><td>NM_001271733.1 </td><td>9 </td><td>CDS </td><td>30 </td><td>gca </td><td>gtgagtccctggtgctcccggccccgccag
******   ***** </td><td>g<br />* </td><td>AY192149.1 </td></tr><tr><td><em>nAChRb1</em> (657999) </td><td><em>Tribolium castaneum</em> (7070) </td><td>NM_001162528.1 </td><td>8 </td><td>CDS </td><td>35 </td><td>aag<br />*** </td><td>gtaaaaatctaatcacatacccacccccattcaag
**** ****:**  ** </td><td>c </td><td>EU937812.1 </td></tr><tr><td><em>CELE_B0348.6</em> (178536) </td><td><em>Caenorhabditis elegans</em> (6239) </td><td>NM_070722.3 </td><td>2 </td><td>CDS </td><td>39 </td><td>caa<br />** </td><td>gtacgttttgagaaatatattttattcaatgaatcatag
*** **  *** :** * ** </td><td>a </td><td>CK586343.1 CK586324.1 CK586322.1 CK579517.1 </td></tr><tr><td><em>bcd</em> (40830) </td><td><em>Drosophila melanogaster</em> (7227) </td><td>NM_176410.3 </td><td>2 </td><td>CDS </td><td>40 </td><td>cag<br />*** </td><td>gtgagctcaaagccaacaaagtcagccatcgtcttatcag
*****  ** * :** ***** </td><td>a </td><td>X14459.1 BT021332.1<br />CO301329.1 BI163743.1<br />AA941658.2 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn7"><div class="fn"><p><sup>a</sup>CDS: coding sequence.</p></div></div><div content-id="tblfn8"><div class="fn"><p><sup>b</sup>Bold: donor (5′) and acceptor (3′) splice sites. Underline: possible pyrimidine-rich region. Colon: possible branch point. Asterisks: eukaryotic conserved bases (eukaryotic consensus sequence taken as reference: MAG<strong>GT</strong>RAGT…YNYYRAYY…YYYYYYYYYYYNYC<strong>AG</strong>G, where M = A/C, R = A/G, Y = C/T, N = any base).<sup><a reveal-id="DSV028C16" data-open="DSV028C16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="DSV028C24" data-open="DSV028C24" class="link link-ref link-reveal xref-bibr">24</a></sup></p></div></div><div content-id="tblfn9"><div class="fn"><p><sup>c</sup>Some example independent RNA sequences encompassing the intron and thus validating its existence.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87201650" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSV028TB4" class="table-wrap-title" data-id="DSV028TB4"><div class="label -label">Table 4.</div><div class="caption"><p>Minimal introns validated through bioinformatic analysis (one representative intron for each available organism; see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv028/1/dsv028_Supplementary_Data.zip?Expires=2147483647&Signature=aRYnjW0Rb4P1z~LqsvpBA4F8yDeKnemALjuIs5dIu7b78ZIyCC9vStwlmcaLpwYHHynJbVrUGwOnQs3sBQ7xB6WMmPBg1XPcJaNy2tx~WiUOnNubYatD8rrHP4-b0kqD26V-9w-T5~7ViNvHPhtETEQ4Qe0MRM7nMeiJQfNtapr3~Sd2ReGizplxkP2zCfwUUBhYjaeRaKj4EGiQwmBeXrBErc~TaNkS~uGM0gU0lcrArfMkDjnNsYKFTvojR-CLBO9iYZel7PwBnhK6Iu2q-GunrXWXN3nleQzH4Vl5aPuE12cNg9dJn3hbWN8XW-hFI-vG6tJ~WhfJ0dVGoR3IGQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S1, S2 and S3</a></span> for more details)</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Gene symbol (Gene ID) </th><th>Organism (Taxonomy ID) </th><th>RefSeq RNA accession number </th><th>Intron number </th><th>Position<sup>a</sup> </th><th>Intron length (bp) </th><th>Previous exon 3′ </th><th>Intron sequence 5′-3′<sup>b</sup> </th><th>Following exon 5′ </th><th>GenBank accession numbers<sup>c</sup> </th></tr></thead><tbody><tr><td><em>MST1L</em> (11223) </td><td><em>Homo sapiens</em> (9606) </td><td>NM_001271733.1 </td><td>9 </td><td>CDS </td><td>30 </td><td>gca </td><td>gtgagtccctggtgctcccggccccgccag
******   ***** </td><td>g<br />* </td><td>AY192149.1 </td></tr><tr><td><em>nAChRb1</em> (657999) </td><td><em>Tribolium castaneum</em> (7070) </td><td>NM_001162528.1 </td><td>8 </td><td>CDS </td><td>35 </td><td>aag<br />*** </td><td>gtaaaaatctaatcacatacccacccccattcaag
**** ****:**  ** </td><td>c </td><td>EU937812.1 </td></tr><tr><td><em>CELE_B0348.6</em> (178536) </td><td><em>Caenorhabditis elegans</em> (6239) </td><td>NM_070722.3 </td><td>2 </td><td>CDS </td><td>39 </td><td>caa<br />** </td><td>gtacgttttgagaaatatattttattcaatgaatcatag
*** **  *** :** * ** </td><td>a </td><td>CK586343.1 CK586324.1 CK586322.1 CK579517.1 </td></tr><tr><td><em>bcd</em> (40830) </td><td><em>Drosophila melanogaster</em> (7227) </td><td>NM_176410.3 </td><td>2 </td><td>CDS </td><td>40 </td><td>cag<br />*** </td><td>gtgagctcaaagccaacaaagtcagccatcgtcttatcag
*****  ** * :** ***** </td><td>a </td><td>X14459.1 BT021332.1<br />CO301329.1 BI163743.1<br />AA941658.2 </td></tr></tbody></table></div><div content-id="DSV028TB4" class="table-modal"><table><thead name="thead" align=""><tr><th>Gene symbol (Gene ID) </th><th>Organism (Taxonomy ID) </th><th>RefSeq RNA accession number </th><th>Intron number </th><th>Position<sup>a</sup> </th><th>Intron length (bp) </th><th>Previous exon 3′ </th><th>Intron sequence 5′-3′<sup>b</sup> </th><th>Following exon 5′ </th><th>GenBank accession numbers<sup>c</sup> </th></tr></thead><tbody><tr><td><em>MST1L</em> (11223) </td><td><em>Homo sapiens</em> (9606) </td><td>NM_001271733.1 </td><td>9 </td><td>CDS </td><td>30 </td><td>gca </td><td>gtgagtccctggtgctcccggccccgccag
******   ***** </td><td>g<br />* </td><td>AY192149.1 </td></tr><tr><td><em>nAChRb1</em> (657999) </td><td><em>Tribolium castaneum</em> (7070) </td><td>NM_001162528.1 </td><td>8 </td><td>CDS </td><td>35 </td><td>aag<br />*** </td><td>gtaaaaatctaatcacatacccacccccattcaag
**** ****:**  ** </td><td>c </td><td>EU937812.1 </td></tr><tr><td><em>CELE_B0348.6</em> (178536) </td><td><em>Caenorhabditis elegans</em> (6239) </td><td>NM_070722.3 </td><td>2 </td><td>CDS </td><td>39 </td><td>caa<br />** </td><td>gtacgttttgagaaatatattttattcaatgaatcatag
*** **  *** :** * ** </td><td>a </td><td>CK586343.1 CK586324.1 CK586322.1 CK579517.1 </td></tr><tr><td><em>bcd</em> (40830) </td><td><em>Drosophila melanogaster</em> (7227) </td><td>NM_176410.3 </td><td>2 </td><td>CDS </td><td>40 </td><td>cag<br />*** </td><td>gtgagctcaaagccaacaaagtcagccatcgtcttatcag
*****  ** * :** ***** </td><td>a </td><td>X14459.1 BT021332.1<br />CO301329.1 BI163743.1<br />AA941658.2 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn7"><div class="fn"><p><sup>a</sup>CDS: coding sequence.</p></div></div><div content-id="tblfn8"><div class="fn"><p><sup>b</sup>Bold: donor (5′) and acceptor (3′) splice sites. Underline: possible pyrimidine-rich region. Colon: possible branch point. Asterisks: eukaryotic conserved bases (eukaryotic consensus sequence taken as reference: MAG<strong>GT</strong>RAGT…YNYYRAYY…YYYYYYYYYYYNYC<strong>AG</strong>G, where M = A/C, R = A/G, Y = C/T, N = any base).<sup><a reveal-id="DSV028C16" data-open="DSV028C16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="DSV028C24" data-open="DSV028C24" class="link link-ref link-reveal xref-bibr">24</a></sup></p></div></div><div content-id="tblfn9"><div class="fn"><p><sup>c</sup>Some example independent RNA sequences encompassing the intron and thus validating its existence.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87201650" target="_blank">View Large</a></div></div><p>The shortest intron sequence is 30 bp long and belongs to the <em>MST1L</em> gene of <em>H. sapiens</em> (Gene ID: 11223, Taxonomy ID: 9606), encoding for the putative macrophage stimulating 1-like protein. The following validated intron length is 35 bp and belongs to the <em>nAChRb1</em> gene of <em>Tribolium castaneum</em> (Gene ID: 657999, Taxonomy ID: 7070), encoding for the nicotinic acetylcholine receptor beta-1 subunit. Regarding other organisms, also validated were the 39 bp intron of <em>CELE_B0348.6</em> gene of <em>C. elegans</em> (Gene ID: 178536, Taxonomy ID: 6239) and the 40 bp intron of <em>bcd</em> gene of <em>Drosophila melanogaster</em> (Gene ID: 40830, Taxonomy ID: 7227), encoding for the eukaryotic translation initiation factor 4E-3 and for bicoid, respectively.</p><p>The analysis of the validated intron sequences revealed the presence of many conserved bases compared with the well-known consensus sequence (Table <a reveal-id="DSV028TB4" data-open="DSV028TB4" class="link link-reveal link-table xref-fig">4</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv028/1/dsv028_Supplementary_Data.zip?Expires=2147483647&Signature=aRYnjW0Rb4P1z~LqsvpBA4F8yDeKnemALjuIs5dIu7b78ZIyCC9vStwlmcaLpwYHHynJbVrUGwOnQs3sBQ7xB6WMmPBg1XPcJaNy2tx~WiUOnNubYatD8rrHP4-b0kqD26V-9w-T5~7ViNvHPhtETEQ4Qe0MRM7nMeiJQfNtapr3~Sd2ReGizplxkP2zCfwUUBhYjaeRaKj4EGiQwmBeXrBErc~TaNkS~uGM0gU0lcrArfMkDjnNsYKFTvojR-CLBO9iYZel7PwBnhK6Iu2q-GunrXWXN3nleQzH4Vl5aPuE12cNg9dJn3hbWN8XW-hFI-vG6tJ~WhfJ0dVGoR3IGQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>).</p><p>The G content of the 20 shortest ‘REVIEWED’ or ‘VALIDATED’ human canonical intron sequences is shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv028/1/dsv028_Supplementary_Data.zip?Expires=2147483647&Signature=aRYnjW0Rb4P1z~LqsvpBA4F8yDeKnemALjuIs5dIu7b78ZIyCC9vStwlmcaLpwYHHynJbVrUGwOnQs3sBQ7xB6WMmPBg1XPcJaNy2tx~WiUOnNubYatD8rrHP4-b0kqD26V-9w-T5~7ViNvHPhtETEQ4Qe0MRM7nMeiJQfNtapr3~Sd2ReGizplxkP2zCfwUUBhYjaeRaKj4EGiQwmBeXrBErc~TaNkS~uGM0gU0lcrArfMkDjnNsYKFTvojR-CLBO9iYZel7PwBnhK6Iu2q-GunrXWXN3nleQzH4Vl5aPuE12cNg9dJn3hbWN8XW-hFI-vG6tJ~WhfJ0dVGoR3IGQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>.</p>                            <h2 scrollto-destination=87201654 id="87201654" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>We have described a full parsing of all eukaryotic gene entries annotated in NCBI Gene and generated a relational and fully indexed local database named GeneBase. It consists of three main related tables containing information—such as gene nomenclature, structure and transcripts—in indexed fields about more than 2 million genes from more than 350 species, ranging from Plants and Fungi to Animals. This compares very favourably to, for example, the 91 species available through the Gene Table download utility provided by UCSC (<a class="link link-uri" href="http://genome-euro.ucsc.edu/cgi-bin/hgTables">http://genome-euro.ucsc.edu/cgi-bin/hgTables</a>) or the 69 available in Ensembl through the download tool BioMart (<a class="link link-uri" href="http://www.ensembl.org/biomart/">http://www.ensembl.org/biomart/</a>). The choice we made this time of considering NCBI Gene data, in contrast with our other sequence parsing tools based on UCSC data,<sup><a reveal-id="DSV028C26" data-open="DSV028C26" class="link link-ref link-reveal xref-bibr">26</a>,<a reveal-id="DSV028C27" data-open="DSV028C27" class="link link-ref link-reveal xref-bibr">27</a></sup> also gave us maximum flexibility in the choice of search parameters. However, the parsing of such a high number of gene entries inevitably needs a considerable amount of disk space and time to perform calculations. We chose not to provide a GeneBase web tool, because FileMaker Pro is particularly useful due to its structure and interface but has limitations for web publication of full features of the local file. In addition, a web version would not be able to give users the freedom to customize the database and apply it to local files. Therefore, along with the eukaryote pre-loaded version of the database (for users interested in all the available entries), we have made an empty version available. It can be loaded by any user by importing subsequent versions or desired subsets of NCBI Gene data downloaded using specific text queries and parsed by our provided scripts, according to the GeneBase guide.</p><p>Among all the genes that are present in the pre-loaded version of GeneBase (based on NCBI Gene records available up to 22 April 2015), only 3.15 and 1.73% are ‘REVIEWED’ and ‘VALIDATED’ records, respectively, while almost all of the entries are ‘PROVISIONAL’ (92.88%), according to the entry status as provided by RefSeq. Among all the ‘Gene_Table’ exon/intron records, 16.9% are derived from an ‘NM_’ or ‘NR_’ RefSeq RNA type sequence. This implies that not only ‘REVIEWED’ and ‘VALIDATED’ but also ‘PROVISIONAL’, ‘INFERRED’ or ‘PREDICTED’ genes can present a gene product derived from these types of sequences. In addition, there are some discrepancies regarding the ‘live’ status provided by NCBI even for genes not annotated on the most recent genome annotation, addressed in GeneBase with the ‘Genome_Annotation_Status’ field, which is useful to filter out these entries. We also observed that in some cases, a ‘REVIEWED’ gene (for example Gene ID: 405306) with an ‘NM_’ RNA type product has an intron of 0 bp (or even −1 bp) length which is biologically not sound, but combined with the ‘low-quality sequence region’ markup in the corresponding RefSeq genome sequence, it indicates a compensation for genome assembly defects. The structuration of data of our database (e.g. numbers are seen as numeric fields) along with an intuitive user-friendly graphic interface allows users to perform many types of searches, even to exclude these kind of artifacts, and to make comparisons and calculations that are useful for data retrieval and analysis. The use of numeric fields has the specific advantage of making it possible to search for the exact number (or range), thus avoiding, querying ‘1’, for example, other unwanted results, such as 10, 11, 12 and so forth.<sup><a reveal-id="DSV028C28" data-open="DSV028C28" class="link link-ref link-reveal xref-bibr">28</a>,<a reveal-id="DSV028C29" data-open="DSV028C29" class="link link-ref link-reveal xref-bibr">29</a></sup> In addition, calculation fields were created to generate further data that are not present in the original NCBI gene entries or in other databases, making the analysis capability of GeneBase unique. For example, the 3′-untranslated or 3′-translated and untranslated terminal exons<sup><a reveal-id="DSV028C30" data-open="DSV028C30" class="link link-ref link-reveal xref-bibr">30</a></sup> are well known for their particular characteristics (e.g. the presence of the termination codon, the polyadenylation site and an average length) and sometimes for their implications in pathological nonsense mutations escaping nonsense-mediated decay.<sup><a reveal-id="DSV028C31" data-open="DSV028C31" class="link link-ref link-reveal xref-bibr">31</a></sup> They cannot usually be retrieved with other databases, while GeneBase shows specific indexed calculation fields, labelling these exons as ‘Last_Exon’ and ‘Last_Coding_Exon’ (Fig. <a reveal-id="DSV028F2" data-open="DSV028F2" class="link link-reveal link-table xref-fig">2</a>), to highlight their importance and to make them easily retrievable. In addition, specific fields showing related information from relational software tables allow users to perform original crossed searches of many entry features in a single organism of interest as well as across a full range of species. Finally, the implementation with exon (both coding and not coding) and intron sequences (Fig. <a reveal-id="DSV028F2" data-open="DSV028F2" class="link link-reveal link-table xref-fig">2</a>) gives rise to original and previously unexplored gene information and sequence correlations.</p><p>GeneBase database contains a wealth of interesting biological information, and we provide an original analysis of the existing introns across all the available species as an example application to confirm its usefulness.</p><p>Although several exon and/or intron databases<sup><a reveal-id="DSV028C32  DSV028C38" data-open="DSV028C32  DSV028C38" class="link link-ref link-reveal xref-bibr">32–38</a></sup> and NCBI Gene parsers<sup><a reveal-id="DSV028C5  DSV028C8" data-open="DSV028C5  DSV028C8" class="link link-ref link-reveal xref-bibr">5–8</a></sup> exist, to our knowledge none is able to dynamically generate a fully structured relational database that could be updated (regenerated) by any user, with an intuitive user-friendly graphic interface that allows any type of further calculations and analysis. Use of NCBI Gene data has the advantage of allowing users to avoid GenBank redundancy purging procedures used to develop an Exon-Intron Database (EID),<sup><a reveal-id="DSV028C32" data-open="DSV028C32" class="link link-ref link-reveal xref-bibr">32</a></sup> the Intron Exon-Knowledge base (IE-Kb),<sup><a reveal-id="DSV028C33" data-open="DSV028C33" class="link link-ref link-reveal xref-bibr">33</a></sup> another Exon-Intron Database (ExInt)<sup><a reveal-id="DSV028C35" data-open="DSV028C35" class="link link-ref link-reveal xref-bibr">35</a></sup> and the database of eukaryotic protein-encoding genes (Xpro).<sup><a reveal-id="DSV028C37" data-open="DSV028C37" class="link link-ref link-reveal xref-bibr">37</a></sup> In the particular case of multiple transcript variants presenting the same exon or intron, the ‘Non_Redundant’ fields are useful for searching for unique exons and introns. On the other hand, the decision to keep the exon/intron structure of each transcript isoform for each gene, allows the option of, for example, alternative splicing studies.<sup><a reveal-id="DSV028C32" data-open="DSV028C32" class="link link-ref link-reveal xref-bibr">32</a>,<a reveal-id="DSV028C37" data-open="DSV028C37" class="link link-ref link-reveal xref-bibr">37</a>,<a reveal-id="DSV028C38" data-open="DSV028C38" class="link link-ref link-reveal xref-bibr">38</a></sup> In addition, unlike more specialised databases such as, among others, the database on intron less/single exonic genes (SEGE)<sup><a reveal-id="DSV028C36" data-open="DSV028C36" class="link link-ref link-reveal xref-bibr">36</a></sup> and the Intron and Intron Evolution Database (IDB and IEDB),<sup><a reveal-id="DSV028C34" data-open="DSV028C34" class="link link-ref link-reveal xref-bibr">34</a></sup> GeneBase contains exon/intron structure of 5′UTR (untranslated region), coding region and 3′UTR of coding and non-coding nuclear genes,<sup><a reveal-id="DSV028C33" data-open="DSV028C33" class="link link-ref link-reveal xref-bibr">33</a>,<a reveal-id="DSV028C34" data-open="DSV028C34" class="link link-ref link-reveal xref-bibr">34</a>,<a reveal-id="DSV028C37" data-open="DSV028C37" class="link link-ref link-reveal xref-bibr">37</a></sup> as well as intronless genes.<sup><a reveal-id="DSV028C36" data-open="DSV028C36" class="link link-ref link-reveal xref-bibr">36</a></sup> In the NCBI Gene subset parsed and loaded into GeneBase, the majority of gene entries does not belong to <em>C. elegans</em> or to <em>A. thaliana</em> (Taxonomy IDs: 6239 and 3702, respectively), unlike for example in ExInt<sup><a reveal-id="DSV028C35" data-open="DSV028C35" class="link link-ref link-reveal xref-bibr">35</a></sup> and in IDB and IEDB,<sup><a reveal-id="DSV028C34" data-open="DSV028C34" class="link link-ref link-reveal xref-bibr">34</a></sup> but to <em>H. sapiens</em> (Taxonomy ID: 9606), in agreement with the current choice to exclude gene entries with a ‘MODEL’ RefSeq status.<sup><a reveal-id="DSV028C32" data-open="DSV028C32" class="link link-ref link-reveal xref-bibr">32</a></sup> In any case, the user can select and download any NCBI Gene entry set for parsing.</p><p>Exploiting our ‘Gene_Table’ database table, which collects exon/intron structure for each gene product, our aim was to find the minimal intron, namely the minimal known DNA sequence element that can function as an intron. The overall distribution of intron (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv028/1/dsv028_Supplementary_Data.zip?Expires=2147483647&Signature=aRYnjW0Rb4P1z~LqsvpBA4F8yDeKnemALjuIs5dIu7b78ZIyCC9vStwlmcaLpwYHHynJbVrUGwOnQs3sBQ7xB6WMmPBg1XPcJaNy2tx~WiUOnNubYatD8rrHP4-b0kqD26V-9w-T5~7ViNvHPhtETEQ4Qe0MRM7nMeiJQfNtapr3~Sd2ReGizplxkP2zCfwUUBhYjaeRaKj4EGiQwmBeXrBErc~TaNkS~uGM0gU0lcrArfMkDjnNsYKFTvojR-CLBO9iYZel7PwBnhK6Iu2q-GunrXWXN3nleQzH4Vl5aPuE12cNg9dJn3hbWN8XW-hFI-vG6tJ~WhfJ0dVGoR3IGQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>) and the high standard deviation (Tables <a reveal-id="DSV028TB1" data-open="DSV028TB1" class="link link-reveal link-table xref-fig">1</a> and <a reveal-id="DSV028TB2" data-open="DSV028TB2" class="link link-reveal link-table xref-fig">2</a>) reflects the extreme variation in intron lengths.<sup><a reveal-id="DSV028C33" data-open="DSV028C33" class="link link-ref link-reveal xref-bibr">33</a></sup> Regarding the chosen representative organisms, <em>A. thaliana</em>, <em>C. elegans</em>, <em>T. castaneum</em> and <em>D. melanogaster</em> (Taxonomy IDs: 3702, 6239, 7070 and 7227, respectively) have only one major narrow peak around 100 bp, while <em>Danio rerio</em> (Taxonomy ID: 7955) has two similar peaks (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv028/1/dsv028_Supplementary_Data.zip?Expires=2147483647&Signature=aRYnjW0Rb4P1z~LqsvpBA4F8yDeKnemALjuIs5dIu7b78ZIyCC9vStwlmcaLpwYHHynJbVrUGwOnQs3sBQ7xB6WMmPBg1XPcJaNy2tx~WiUOnNubYatD8rrHP4-b0kqD26V-9w-T5~7ViNvHPhtETEQ4Qe0MRM7nMeiJQfNtapr3~Sd2ReGizplxkP2zCfwUUBhYjaeRaKj4EGiQwmBeXrBErc~TaNkS~uGM0gU0lcrArfMkDjnNsYKFTvojR-CLBO9iYZel7PwBnhK6Iu2q-GunrXWXN3nleQzH4Vl5aPuE12cNg9dJn3hbWN8XW-hFI-vG6tJ~WhfJ0dVGoR3IGQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>). Other chosen representative vertebrates (<em>Xenopus tropicalis</em>, <em>Gallus gallus</em>, <em>Mus musculus</em> and <em>H. sapiens</em>, Taxonomy IDs: 8364, 9031, 10090 and 9606, respectively) have one initial minor narrow peak and a subsequent higher and wider one (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv028/1/dsv028_Supplementary_Data.zip?Expires=2147483647&Signature=aRYnjW0Rb4P1z~LqsvpBA4F8yDeKnemALjuIs5dIu7b78ZIyCC9vStwlmcaLpwYHHynJbVrUGwOnQs3sBQ7xB6WMmPBg1XPcJaNy2tx~WiUOnNubYatD8rrHP4-b0kqD26V-9w-T5~7ViNvHPhtETEQ4Qe0MRM7nMeiJQfNtapr3~Sd2ReGizplxkP2zCfwUUBhYjaeRaKj4EGiQwmBeXrBErc~TaNkS~uGM0gU0lcrArfMkDjnNsYKFTvojR-CLBO9iYZel7PwBnhK6Iu2q-GunrXWXN3nleQzH4Vl5aPuE12cNg9dJn3hbWN8XW-hFI-vG6tJ~WhfJ0dVGoR3IGQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>). It is interesting to note that <em>M. musculus</em> and <em>H. sapiens</em> (Taxonomy ID: 9606) trends are remarkably overlapping (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv028/1/dsv028_Supplementary_Data.zip?Expires=2147483647&Signature=aRYnjW0Rb4P1z~LqsvpBA4F8yDeKnemALjuIs5dIu7b78ZIyCC9vStwlmcaLpwYHHynJbVrUGwOnQs3sBQ7xB6WMmPBg1XPcJaNy2tx~WiUOnNubYatD8rrHP4-b0kqD26V-9w-T5~7ViNvHPhtETEQ4Qe0MRM7nMeiJQfNtapr3~Sd2ReGizplxkP2zCfwUUBhYjaeRaKj4EGiQwmBeXrBErc~TaNkS~uGM0gU0lcrArfMkDjnNsYKFTvojR-CLBO9iYZel7PwBnhK6Iu2q-GunrXWXN3nleQzH4Vl5aPuE12cNg9dJn3hbWN8XW-hFI-vG6tJ~WhfJ0dVGoR3IGQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>), in accordance with an earlier report.<sup><a reveal-id="DSV028C10" data-open="DSV028C10" class="link link-ref link-reveal xref-bibr">10</a></sup> Mean intron lengths of these organisms significantly correlate with their genome length and a noteworthy correlation can be found considering only <em>Vertebrata</em> (Taxonomy ID: 7742, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv028/1/dsv028_Supplementary_Data.zip?Expires=2147483647&Signature=aRYnjW0Rb4P1z~LqsvpBA4F8yDeKnemALjuIs5dIu7b78ZIyCC9vStwlmcaLpwYHHynJbVrUGwOnQs3sBQ7xB6WMmPBg1XPcJaNy2tx~WiUOnNubYatD8rrHP4-b0kqD26V-9w-T5~7ViNvHPhtETEQ4Qe0MRM7nMeiJQfNtapr3~Sd2ReGizplxkP2zCfwUUBhYjaeRaKj4EGiQwmBeXrBErc~TaNkS~uGM0gU0lcrArfMkDjnNsYKFTvojR-CLBO9iYZel7PwBnhK6Iu2q-GunrXWXN3nleQzH4Vl5aPuE12cNg9dJn3hbWN8XW-hFI-vG6tJ~WhfJ0dVGoR3IGQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figs S3 and S4</a></span>).</p><p>The intron length problem, which currently raises researchers’ interest,<sup><a reveal-id="DSV028C17" data-open="DSV028C17" class="link link-ref link-reveal xref-bibr">17</a></sup> has recently also been addressed by Sasaki-Haraguchi et al.,<sup><a reveal-id="DSV028C39" data-open="DSV028C39" class="link link-ref link-reveal xref-bibr">39</a></sup> limiting their study to the human genome. They demonstrate an alternative splicing leading to low efficiency to the removal of an ultra-short 43 bp intron of the <em>ESRP2</em> gene (Gene ID: 80004), following transfection of a construct containing the intron and the two flanking exons in human cultured neoplastic cells. This 43 bp intron is not currently annotated in the NCBI Gene database.</p><p>Uncertainty in defining the minimal intron length can be due to sequencing errors in genomes, to artifacts in intron size determination by annotation pipelines and to the lack of a tool like GeneBase, which is especially useful for retrieving data with numerical range constraints. As other authors previously supposed<sup><a reveal-id="DSV028C40" data-open="DSV028C40" class="link link-ref link-reveal xref-bibr">40</a></sup> and verified,<sup><a reveal-id="DSV028C38" data-open="DSV028C38" class="link link-ref link-reveal xref-bibr">38</a></sup> annotation errors and mismatches between the sequenced transcript and the reference genome can even generate one-nucleotide long introns, which we still came across too (Table <a reveal-id="DSV028TB3" data-open="DSV028TB3" class="link link-reveal link-table xref-fig">3</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv028/1/dsv028_Supplementary_Data.zip?Expires=2147483647&Signature=aRYnjW0Rb4P1z~LqsvpBA4F8yDeKnemALjuIs5dIu7b78ZIyCC9vStwlmcaLpwYHHynJbVrUGwOnQs3sBQ7xB6WMmPBg1XPcJaNy2tx~WiUOnNubYatD8rrHP4-b0kqD26V-9w-T5~7ViNvHPhtETEQ4Qe0MRM7nMeiJQfNtapr3~Sd2ReGizplxkP2zCfwUUBhYjaeRaKj4EGiQwmBeXrBErc~TaNkS~uGM0gU0lcrArfMkDjnNsYKFTvojR-CLBO9iYZel7PwBnhK6Iu2q-GunrXWXN3nleQzH4Vl5aPuE12cNg9dJn3hbWN8XW-hFI-vG6tJ~WhfJ0dVGoR3IGQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). Despite the additional exclusion procedure of gene ‘MODEL’ and the decision to consider only ‘REVIEWED’ or ‘VALIDATED’ gene entries with an ‘NM_’ or ‘NR_’ type of corresponding RefSeq RNA accession number, we were compelled to validate our results through a further bioinformatic analysis,<sup><a reveal-id="DSV028C37" data-open="DSV028C37" class="link link-ref link-reveal xref-bibr">37</a>,<a reveal-id="DSV028C38" data-open="DSV028C38" class="link link-ref link-reveal xref-bibr">38</a></sup> which yielded 8.8% of actual intron sequences in this subset (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv028/1/dsv028_Supplementary_Data.zip?Expires=2147483647&Signature=aRYnjW0Rb4P1z~LqsvpBA4F8yDeKnemALjuIs5dIu7b78ZIyCC9vStwlmcaLpwYHHynJbVrUGwOnQs3sBQ7xB6WMmPBg1XPcJaNy2tx~WiUOnNubYatD8rrHP4-b0kqD26V-9w-T5~7ViNvHPhtETEQ4Qe0MRM7nMeiJQfNtapr3~Sd2ReGizplxkP2zCfwUUBhYjaeRaKj4EGiQwmBeXrBErc~TaNkS~uGM0gU0lcrArfMkDjnNsYKFTvojR-CLBO9iYZel7PwBnhK6Iu2q-GunrXWXN3nleQzH4Vl5aPuE12cNg9dJn3hbWN8XW-hFI-vG6tJ~WhfJ0dVGoR3IGQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). The validation process was also hindered by genomic sequence repetitions, but with manual analysis, it was possible to identify intron sequences that had been erroneously annotated (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv028/1/dsv028_Supplementary_Data.zip?Expires=2147483647&Signature=aRYnjW0Rb4P1z~LqsvpBA4F8yDeKnemALjuIs5dIu7b78ZIyCC9vStwlmcaLpwYHHynJbVrUGwOnQs3sBQ7xB6WMmPBg1XPcJaNy2tx~WiUOnNubYatD8rrHP4-b0kqD26V-9w-T5~7ViNvHPhtETEQ4Qe0MRM7nMeiJQfNtapr3~Sd2ReGizplxkP2zCfwUUBhYjaeRaKj4EGiQwmBeXrBErc~TaNkS~uGM0gU0lcrArfMkDjnNsYKFTvojR-CLBO9iYZel7PwBnhK6Iu2q-GunrXWXN3nleQzH4Vl5aPuE12cNg9dJn3hbWN8XW-hFI-vG6tJ~WhfJ0dVGoR3IGQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). In addition, 141 sequences derived from an annotated genomic sequences belonging to <em>A. thaliana</em> (Taxonomy ID: 3702). These lack experimental evidence as indicated in RefSeq Nucleotide entries and by the gene confidence ranking provided by TAIR (The Arabidopsis Information Resource at <a class="link link-uri" href="https://www.arabidopsis.org/index.jsp">https://www.arabidopsis.org/index.jsp</a>)<sup><a reveal-id="DSV028C41" data-open="DSV028C41" class="link link-ref link-reveal xref-bibr">41</a></sup> and were not considered further.</p><p>Following our analyses based on all currently available data, the shortest eukaryotic GT-AG intron length belongs to the human <em>MST1L</em> gene (<em>MST1-like</em>, Gene ID: 11223), encoding for the putative macrophage stimulating 1-like protein, setting the physical limit of the intron size at 30 bp (Table <a reveal-id="DSV028TB4" data-open="DSV028TB4" class="link link-reveal link-table xref-fig">4</a>). Remarkably, the 30 bp size is indeed consistent with the sum of the 5′/3′ splicing consensus site lengths.<sup><a reveal-id="DSV028C16" data-open="DSV028C16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="DSV028C24" data-open="DSV028C24" class="link link-ref link-reveal xref-bibr">24</a>,<a reveal-id="DSV028C42" data-open="DSV028C42" class="link link-ref link-reveal xref-bibr">42</a></sup> Unlike <em>MST1</em> (Gene ID: 4485), for which 153 orthologs are reported in NCBI Gene, <em>MST1L</em> (Gene ID: 11223, which is the <em>MST1</em> human paralog) lacks experimentally confirmed orthologs in other species. This is probably due to the fact that <em>MST1L</em> (Gene ID: 11223) appears to be an interesting case of ‘resurrected’ pseudogene in humans, so that it is now classified as a protein-coding gene with mass spectrometry evidence for the polypeptide product.<sup><a reveal-id="DSV028C43" data-open="DSV028C43" class="link link-ref link-reveal xref-bibr">43</a></sup> Another human minimal intron validated through our application is the <em>AQP12A</em> 37 bp intron (Gene ID: 375318, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv028/1/dsv028_Supplementary_Data.zip?Expires=2147483647&Signature=aRYnjW0Rb4P1z~LqsvpBA4F8yDeKnemALjuIs5dIu7b78ZIyCC9vStwlmcaLpwYHHynJbVrUGwOnQs3sBQ7xB6WMmPBg1XPcJaNy2tx~WiUOnNubYatD8rrHP4-b0kqD26V-9w-T5~7ViNvHPhtETEQ4Qe0MRM7nMeiJQfNtapr3~Sd2ReGizplxkP2zCfwUUBhYjaeRaKj4EGiQwmBeXrBErc~TaNkS~uGM0gU0lcrArfMkDjnNsYKFTvojR-CLBO9iYZel7PwBnhK6Iu2q-GunrXWXN3nleQzH4Vl5aPuE12cNg9dJn3hbWN8XW-hFI-vG6tJ~WhfJ0dVGoR3IGQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>). Its sequence existence was also confirmed by a work published while this study was being submitted for publication.<sup><a reveal-id="DSV028C25" data-open="DSV028C25" class="link link-ref link-reveal xref-bibr">25</a></sup> Finally, the inclusion in our analysis of organisms not present in other genome browsers also gave us the opportunity to validate four introns shorter than 40 bp of <em>T. castaneum</em> (Taxonomy ID: 7070).</p><p>Thanks to their small size, our ‘model introns’ will help identify the basic and crucial minimal requirement elements of recognition used by the cell for conventional splicing functioning. We were able to locate possible pyrimidine-rich regions and possible consensus branch points, although only one intron (<em>LOC656453</em>, Gene ID: 656453, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv028/1/dsv028_Supplementary_Data.zip?Expires=2147483647&Signature=aRYnjW0Rb4P1z~LqsvpBA4F8yDeKnemALjuIs5dIu7b78ZIyCC9vStwlmcaLpwYHHynJbVrUGwOnQs3sBQ7xB6WMmPBg1XPcJaNy2tx~WiUOnNubYatD8rrHP4-b0kqD26V-9w-T5~7ViNvHPhtETEQ4Qe0MRM7nMeiJQfNtapr3~Sd2ReGizplxkP2zCfwUUBhYjaeRaKj4EGiQwmBeXrBErc~TaNkS~uGM0gU0lcrArfMkDjnNsYKFTvojR-CLBO9iYZel7PwBnhK6Iu2q-GunrXWXN3nleQzH4Vl5aPuE12cNg9dJn3hbWN8XW-hFI-vG6tJ~WhfJ0dVGoR3IGQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>) presents the whole classic eukaryotic consensus sequence YNYYRAYY.<sup><a reveal-id="DSV028C16" data-open="DSV028C16" class="link link-ref link-reveal xref-bibr">16</a></sup> The shortest validated human introns (<em>MST1L</em> 30 bp intron, Gene ID: 11223 and <em>AQP12A</em> 37 bp intron, Gene ID: 375318) present a G-rich sequence, as shown for other ultra-short human introns (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv028/1/dsv028_Supplementary_Data.zip?Expires=2147483647&Signature=aRYnjW0Rb4P1z~LqsvpBA4F8yDeKnemALjuIs5dIu7b78ZIyCC9vStwlmcaLpwYHHynJbVrUGwOnQs3sBQ7xB6WMmPBg1XPcJaNy2tx~WiUOnNubYatD8rrHP4-b0kqD26V-9w-T5~7ViNvHPhtETEQ4Qe0MRM7nMeiJQfNtapr3~Sd2ReGizplxkP2zCfwUUBhYjaeRaKj4EGiQwmBeXrBErc~TaNkS~uGM0gU0lcrArfMkDjnNsYKFTvojR-CLBO9iYZel7PwBnhK6Iu2q-GunrXWXN3nleQzH4Vl5aPuE12cNg9dJn3hbWN8XW-hFI-vG6tJ~WhfJ0dVGoR3IGQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>).<sup><a reveal-id="DSV028C25" data-open="DSV028C25" class="link link-ref link-reveal xref-bibr">25</a></sup> The typical alternative splicing sites GYNGYN<sup><a reveal-id="DSV028C44" data-open="DSV028C44" class="link link-ref link-reveal xref-bibr">44</a></sup> or NAGNAG for ‘subtle splicing’<sup><a reveal-id="DSV028C45  DSV028C47" data-open="DSV028C45  DSV028C47" class="link link-ref link-reveal xref-bibr">45–47</a></sup> were not identified. Recent advances in the study of alternative splicing show that part of the maturation process of the primary transcript produces errors and creates stochastic noise.<sup><a reveal-id="DSV028C48" data-open="DSV028C48" class="link link-ref link-reveal xref-bibr">48</a>,<a reveal-id="DSV028C49" data-open="DSV028C49" class="link link-ref link-reveal xref-bibr">49</a></sup> This would increase the difficulties faced by intron prediction tools taking into account all the well-known factors: 5′/3′ splice site, branch point, splicing regulatory elements such as exonic/intronic splicing enhancers/silencers, the spliceosome and other trans-acting elements.<sup><a reveal-id="DSV028C50" data-open="DSV028C50" class="link link-ref link-reveal xref-bibr">50</a></sup> In the light of the observed data, the formation of secondary structures of the primary transcript might intervene in intron identification by the cell.<sup><a reveal-id="DSV028C51" data-open="DSV028C51" class="link link-ref link-reveal xref-bibr">51</a></sup> More in-depth analysis of minimal introns to test this possibility will be necessary in the future.</p><p>Our analysis presented here confirms the difficulties still encountered in working with genomic sequences and is a starting point for further studies. Furthermore, it depends on the chosen gene entry subset and on the RefSeq classification system and is subject to the accuracy of the input dataset. On the other hand, our example application shows how a simple biological question such as how long the minimal GT-AG intron is (a numeric datum combined with a sequence feature) in all eukaryotic validated genes (which means a selection of a common gene characteristic in different organisms) is easily achievable with a single search, thanks to the GeneBase architecture. To our knowledge, GeneBase is a unique example of a database which relationally correlates and allows the complete elaboration of gene-associated meta-information data and the corresponding sequences across different organisms. Our tool's strength is also to allow large-scale analysis of genes, considerably increasing the possibilities for the study of the 2.4 million genes available in the NCBI Gene databank. Furthermore, GeneBase is useful for studying other characteristic intron lengths and sequences such as, for example, the unusual length of 5′-end first introns<sup><a reveal-id="DSV028C52" data-open="DSV028C52" class="link link-ref link-reveal xref-bibr">52</a></sup> in terms of evolution and gene expression levels.<sup><a reveal-id="DSV028C53  DSV028C55" data-open="DSV028C53  DSV028C55" class="link link-ref link-reveal xref-bibr">53–55</a></sup> The implementations of different databases using the same platform (FileMaker) recently led to intriguing results.<sup><a reveal-id="DSV028C56" data-open="DSV028C56" class="link link-ref link-reveal xref-bibr">56</a></sup> In conjunction with quantitative transcriptome mapping in normal<sup><a reveal-id="DSV028C57" data-open="DSV028C57" class="link link-ref link-reveal xref-bibr">57</a></sup> and pathological<sup><a reveal-id="DSV028C58" data-open="DSV028C58" class="link link-ref link-reveal xref-bibr">58</a></sup> human cell types, GeneBase may represent the nucleus for a novel relational multi-purpose and user-friendly modular platform for the analysis of biological data, from meta-information of genes to their sequences and expression values. It will especially be used in the context of our current reanalysis of human chromosome 21 gene content to identify new targets for trisomy 21.<sup><a reveal-id="DSV028C59" data-open="DSV028C59" class="link link-ref link-reveal xref-bibr">59</a></sup></p>                            <h2 scrollto-destination=87201665 id="87201665" class="section-title" data-legacy-id=s5>Availability</h2>
<p>GeneBase (both pre-loaded and empty versions), the ‘Database Design Report’, the user guide and the relative Python scripts for the initial data pre-processing and sequence calculations are publicly available at <a class="link link-uri" href="http://apollo11.isto.unibo.it/software/">http://apollo11.isto.unibo.it/software/</a>.</p>                            <h2 scrollto-destination=87201667 id="87201667" class="section-title" data-legacy-id=s7>Funding</h2>
<p>This work was funded by ‘RFO’ grants from Alma Mater Studiorum—University of Bologna to P.S. and L.V. M.C.'s and A.P.'s fellowships were supported by Fondazione Umano Progresso, Milano, Italy. The software was run on the Apple Mac Pro ‘Multiprocessor Server’ available at the DIMES Department and funded by ‘Fondazione CARISBO’, Bologna, Italy. Funding to pay the Open Access publication charges for this article was provided by donations to our Laboratory of Genomics for the study of trisomy 21.</p>                            <h2 scrollto-destination=87201669 id="87201669" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We specially thank the Fondazione Umano Progresso, Milano, Italy, for supporting the research on trisomy 21 conducted at the DIMES Department. We are grateful to Danielle Mitzman for her expert revision of the manuscript.</p>                            <h2 scrollto-destination=87201671 id="87201671" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSV028C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV028C1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Smith</div>
               <div class="given-names">T.F.</div>
            </div>
          </div>
          <div class="year">1990</div>, <div class="article-title">The history of the genetic sequence databases</div>, <div class="source">Genomics</div> , <div class="volume">6</div>, <div class="fpage">701</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20history%20of%20the%20genetic%20sequence%20databases&amp;author=T.F.%20Smith&amp;publication_year=1990&amp;journal=Genomics&amp;volume=6&amp;pages=701-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0888-7543(90)90509-S" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0888-7543(90)90509-S" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0888-7543(90)90509-S"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2187795" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV028C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV028C2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kennard</div>
               <div class="given-names">O.</div>
            </div>
          </div>
          <div class="year">1997</div>, <div class="article-title">From private data to public knowledge</div>. In: <div class="person-group"><div class="name string-name"><div class="surname">Butterworth</div> <div class="given-names">I.</div></div></div>, (ed.), <div class="comment">The Impact of Electronic Publishing on the Academic Community, an International Workshop Organised by the Academia Europaea and the Wenner-Gren Foundation, Wenner-Gren Center, Stockholm, 16–20 April, 1997</div>. <div class="publisher-name">Portland Press Ltd</div>, <div class="publisher-loc">London, UK</div>.</div></div></div></div><div content-id="DSV028C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV028C3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Brown</div>
               <div class="given-names">G.R.</div>
            </div>, <div class="name string-name"><div class="surname">Hem</div> <div class="given-names">V.</div></div>, <div class="name string-name"><div class="surname">Katz</div> <div class="given-names">K.S.</div></div>et al.  </div>. <div class="year">2014</div>, <div class="article-title">Gene: a gene-centered information resource at NCBI</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">43</div>, <div class="fpage">D36</div>–<div class="lpage">42</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%3A%20a%20gene-centered%20information%20resource%20at%20NCBI&amp;author=G.R.%20Brown&amp;author=V.%20Hem&amp;author=K.S.%20Katz&amp;publication_year=2014&amp;journal=Nucleic%20Acids%20Res.&amp;volume=43&amp;pages=D36-42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gku1055" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgku1055" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgku1055"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25355515" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV028C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV028C4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Lewitter</div>
               <div class="given-names">F.</div>
            </div>
          </div>
          <div class="year">1998</div>, <div class="article-title">Text-based database searching</div>, <div class="source">TRENDS Biotechnol.</div> , <div class="volume">16</div>, <div class="fpage">3</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Text-based%20database%20searching&amp;author=F.%20Lewitter&amp;publication_year=1998&amp;journal=TRENDS%20Biotechnol.&amp;volume=16&amp;pages=3-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0167-7799(98)00126-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0167-7799(98)00126-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0167-7799(98)00126-7"> </span></div> </div></div></div></div></div><div content-id="DSV028C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV028C5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Hart</div>
               <div class="given-names">K.W.</div>
            </div>, <div class="name string-name"><div class="surname">Searls</div> <div class="given-names">D.B.</div></div>, <div class="name string-name"><div class="surname">Overton</div> <div class="given-names">G.C.</div></div></div>
          <div class="year">1994</div>, <div class="article-title">SORTEZ: a relational translator for NCBI's ASN.1 database</div>, <div class="source">Comput. Appl. Biosci.</div> , <div class="volume">10</div>, <div class="fpage">369</div>–<div class="lpage">78</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SORTEZ%3A%20a%20relational%20translator%20for%20NCBI's%20ASN.1%20database&amp;author=K.W.%20Hart&amp;author=D.B.%20Searls&amp;author=G.C.%20Overton&amp;publication_year=1994&amp;journal=Comput.%20Appl.%20Biosci.&amp;volume=10&amp;pages=369-78" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7804870" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV028C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV028C6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Liu</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Grigoriev</div> <div class="given-names">A.</div></div></div>
          <div class="year">2005</div>, <div class="article-title">Fast parsers for Entrez Gene</div>, <div class="source">Bioinformatics</div> , <div class="volume">21</div>, <div class="fpage">3189</div>–<div class="lpage">90</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fast%20parsers%20for%20Entrez%20Gene&amp;author=M.%20Liu&amp;author=A.%20Grigoriev&amp;publication_year=2005&amp;journal=Bioinformatics&amp;volume=21&amp;pages=3189-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bti488" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbti488" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbti488"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15879451" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV028C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV028C7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Shah</div>
               <div class="given-names">S.P.</div>
            </div>, <div class="name string-name"><div class="surname">Huang</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Xu</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Yuen</div> <div class="given-names">M.M.S.</div></div>, <div class="name string-name"><div class="surname">Ling</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Ouellette</div> <div class="given-names">B.F.F.</div></div></div>
          <div class="year">2005</div>, <div class="article-title">Atlas—a data warehouse for integrative bioinformatics</div>, <div class="source">BMC Bioinformatics</div> , <div class="volume">6</div>, <div class="fpage">34</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Atlas%E2%80%94a%20data%20warehouse%20for%20integrative%20bioinformatics&amp;author=S.P.%20Shah&amp;author=Y.%20Huang&amp;author=T.%20Xu&amp;author=M.M.S.%20Yuen&amp;author=J.%20Ling&amp;author=B.F.F.%20Ouellette&amp;publication_year=2005&amp;journal=BMC%20Bioinformatics&amp;volume=6&amp;pages=34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-6-34" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-6-34" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-6-34"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15723693" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV028C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV028C8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Hagen</div>
               <div class="given-names">M.S.</div>
            </div>, <div class="name string-name"><div class="surname">Lee</div> <div class="given-names">E.K.</div></div></div>
          <div class="year">2010</div>, <div class="article-title">BIOSPIDA: a relational database translator for NCBI</div>, <div class="source">AMIA Annu. Symp. Proc.</div> , <div class="volume">20,10</div>, <div class="fpage">422</div>–<div class="lpage">6</div>.</div></div></div></div><div content-id="DSV028C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV028C9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Deutsch</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Long</div> <div class="given-names">M.</div></div></div>
          <div class="year">1999</div>, <div class="article-title">Intron-exon structures of eukaryotic model organisms</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">27</div>, <div class="fpage">3219</div>–<div class="lpage">28</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Intron-exon%20structures%20of%20eukaryotic%20model%20organisms&amp;author=M.%20Deutsch&amp;author=M.%20Long&amp;publication_year=1999&amp;journal=Nucleic%20Acids%20Res.&amp;volume=27&amp;pages=3219-28" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/27.15.3219" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F27.15.3219" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F27.15.3219"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10454621" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV028C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV028C10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Wu</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Xiao</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">L.</div></div>et al.  </div>. <div class="year">2013</div>, <div class="article-title">Systematic analysis of intron size and abundance parameters in diverse lineages</div>, <div class="source">Sci. China. Life Sci.</div> , <div class="volume">56</div>, <div class="fpage">968</div>–<div class="lpage">74</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Systematic%20analysis%20of%20intron%20size%20and%20abundance%20parameters%20in%20diverse%20lineages&amp;author=J.%20Wu&amp;author=J.%20Xiao&amp;author=L.%20Wang&amp;publication_year=2013&amp;journal=Sci.%20China.%20Life%20Sci.&amp;volume=56&amp;pages=968-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11427-013-4540-y" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11427-013-4540-y" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11427-013-4540-y"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24022126" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV028C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV028C11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Lim</div>
               <div class="given-names">L.P.</div>
            </div>, <div class="name string-name"><div class="surname">Burge</div> <div class="given-names">C.B.</div></div></div>
          <div class="year">2001</div>, <div class="article-title">A computational analysis of sequence features involved in recognition of short introns</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="volume">98</div>, <div class="fpage">11193</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20computational%20analysis%20of%20sequence%20features%20involved%20in%20recognition%20of%20short%20introns&amp;author=L.P.%20Lim&amp;author=C.B.%20Burge&amp;publication_year=2001&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=98&amp;pages=11193-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.201407298" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.201407298" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.201407298"> </span></div> </div></div></div></div></div><div content-id="DSV028C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV028C12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Yu</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Yang</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Kibukawa</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Paddock</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Passey</div> <div class="given-names">D.A.</div></div>, <div class="name string-name"><div class="surname">Wong</div> <div class="given-names">G.K.-S.</div></div></div>
          <div class="year">2002</div>, <div class="article-title">Minimal introns are not ‘junk</div>’, <div class="source">Genome Res.</div> , <div class="volume">12</div>, <div class="fpage">1185</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Minimal%20introns%20are%20not%20%E2%80%98junk&amp;author=J.%20Yu&amp;author=Z.%20Yang&amp;author=M.%20Kibukawa&amp;author=M.%20Paddock&amp;author=D.A.%20Passey&amp;author=G.K.-S.%20Wong&amp;publication_year=2002&amp;journal=Genome%20Res.&amp;volume=12&amp;pages=1185-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.224602" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.224602" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.224602"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12176926" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV028C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV028C13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Fields</div>
               <div class="given-names">C.</div>
            </div>
          </div>
          <div class="year">1990</div>, <div class="article-title">Information content of Caenorhabditis elegans splice site sequences varies with intron length</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">18</div>, <div class="fpage">1509</div>–<div class="lpage">12</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Information%20content%20of%20Caenorhabditis%20elegans%20splice%20site%20sequences%20varies%20with%20intron%20length&amp;author=C.%20Fields&amp;publication_year=1990&amp;journal=Nucleic%20Acids%20Res.&amp;volume=18&amp;pages=1509-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/18.6.1509" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F18.6.1509" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F18.6.1509"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2326191" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV028C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV028C14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Mount</div>
               <div class="given-names">S.M.</div>
            </div>, <div class="name string-name"><div class="surname">Burks</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Hertz</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Stormo</div> <div class="given-names">G.D.</div></div>, <div class="name string-name"><div class="surname">White</div> <div class="given-names">O.</div></div>, <div class="name string-name"><div class="surname">Fields</div> <div class="given-names">C.</div></div></div>
          <div class="year">1992</div>, <div class="article-title">Splicing signals in Drosophila: intron size, information content, and consensus sequences</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">20</div>, <div class="fpage">4255</div>–<div class="lpage">62</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Splicing%20signals%20in%20Drosophila%3A%20intron%20size%2C%20information%20content%2C%20and%20consensus%20sequences&amp;author=S.M.%20Mount&amp;author=C.%20Burks&amp;author=G.%20Hertz&amp;author=G.D.%20Stormo&amp;author=O.%20White&amp;author=C.%20Fields&amp;publication_year=1992&amp;journal=Nucleic%20Acids%20Res.&amp;volume=20&amp;pages=4255-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/20.16.4255" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F20.16.4255" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F20.16.4255"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1508718" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV028C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV028C15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Zhu</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">He</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">D.</div></div>et al.  </div>. <div class="year">2010</div>, <div class="article-title">A novel role for minimal introns: routing mRNAs to the cytosol</div>, <div class="source">PLoS One</div> , <div class="volume">5</div>, <div class="fpage">e10144</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20novel%20role%20for%20minimal%20introns%3A%20routing%20mRNAs%20to%20the%20cytosol&amp;author=J.%20Zhu&amp;author=F.%20He&amp;author=D.%20Wang&amp;publication_year=2010&amp;journal=PLoS%20One&amp;volume=5&amp;pages=e10144" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0010144" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0010144" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0010144"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20419085" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV028C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV028C16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Strachan</div>
               <div class="given-names">T.</div>
            </div>, <div class="name string-name"><div class="surname">Read</div> <div class="given-names">A.</div></div></div>
          <div class="year">2010</div>, <div class="source">Human molecular genetics</div> , <div class="edition">4th edition</div>. <div class="publisher-name">Garland Science</div>: <div class="publisher-loc">New York</div>.</div></div></div></div><div content-id="DSV028C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV028C17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Hubé</div>
               <div class="given-names">F.</div>
            </div>, <div class="name string-name"><div class="surname">Francastel</div> <div class="given-names">C.</div></div></div>
          <div class="year">2015</div>, <div class="article-title">Mammalian introns: when the junk generates molecular diversity</div>, <div class="source">Int. J. Mol. Sci.</div> , <div class="volume">16</div>, <div class="fpage">4429</div>–<div class="lpage">52</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mammalian%20introns%3A%20when%20the%20junk%20generates%20molecular%20diversity&amp;author=F.%20Hub%C3%A9&amp;author=C.%20Francastel&amp;publication_year=2015&amp;journal=Int.%20J.%20Mol.%20Sci.&amp;volume=16&amp;pages=4429-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3390/ijms16034429" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3390%2Fijms16034429" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3390%2Fijms16034429"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25710723" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV028C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV028C18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Lenzi</div>
               <div class="given-names">L.</div>
            </div>, <div class="name string-name"><div class="surname">Frabetti</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Facchin</div> <div class="given-names">F.</div></div>et al.  </div>. <div class="year">2006</div>, <div class="article-title">UniGene Tabulator: a full parser for the UniGene format</div>, <div class="source">Bioinformatics</div> , <div class="volume">22</div>, <div class="fpage">2570</div>–<div class="lpage">1</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=UniGene%20Tabulator%3A%20a%20full%20parser%20for%20the%20UniGene%20format&amp;author=L.%20Lenzi&amp;author=F.%20Frabetti&amp;author=F.%20Facchin&amp;publication_year=2006&amp;journal=Bioinformatics&amp;volume=22&amp;pages=2570-1" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btl425" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtl425" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtl425"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16895929" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV028C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV028C19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Lenzi</div>
               <div class="given-names">L.</div>
            </div>, <div class="name string-name"><div class="surname">Facchin</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Piva</div> <div class="given-names">F.</div></div>et al.  </div>. <div class="year">2011</div>, <div class="article-title">TRAM (Transcriptome Mapper): database-driven creation and analysis of transcriptome maps from multiple sources</div>, <div class="source">BMC Genomics</div> , <div class="volume">12</div>, <div class="fpage">121</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=TRAM%20(Transcriptome%20Mapper)%3A%20database-driven%20creation%20and%20analysis%20of%20transcriptome%20maps%20from%20multiple%20sources&amp;author=L.%20Lenzi&amp;author=F.%20Facchin&amp;author=F.%20Piva&amp;publication_year=2011&amp;journal=BMC%20Genomics&amp;volume=12&amp;pages=121" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-12-121" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-12-121" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-12-121"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21333005" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV028C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV028C20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Agarwala</div>
               <div class="given-names">R.</div>
            </div>, <div class="name string-name"><div class="surname">Barrett</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Beck</div> <div class="given-names">J.</div></div>et al.  </div>. <div class="year">2015</div>, <div class="article-title">Database resources of the National Center for Biotechnology Information</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">43</div>, <div class="fpage">D6</div>–<div class="lpage">17</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Database%20resources%20of%20the%20National%20Center%20for%20Biotechnology%20Information&amp;author=R.%20Agarwala&amp;author=T.%20Barrett&amp;author=J.%20Beck&amp;publication_year=2015&amp;journal=Nucleic%20Acids%20Res.&amp;volume=43&amp;pages=D6-17" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gku1130" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgku1130" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgku1130"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25398906" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV028C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV028C21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Altschul</div>
               <div class="given-names">S.F.</div>
            </div>, <div class="name string-name"><div class="surname">Madden</div> <div class="given-names">T.L.</div></div>, <div class="name string-name"><div class="surname">Schäffer</div> <div class="given-names">A.A.</div></div>et al.  </div>. <div class="year">1997</div>, <div class="article-title">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">25</div>, <div class="fpage">3389</div>–<div class="lpage">402</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gapped%20BLAST%20and%20PSI-BLAST%3A%20a%20new%20generation%20of%20protein%20database%20search%20programs&amp;author=S.F.%20Altschul&amp;author=T.L.%20Madden&amp;author=A.A.%20Sch%C3%A4ffer&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res.&amp;volume=25&amp;pages=3389-402" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.17.3389" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.17.3389" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.17.3389"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9254694" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV028C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV028C22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation">
          <div class="collab">Gene Ontology Consortium</div>
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Blake</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Dolan</div> <div class="given-names">M.</div></div>et al.  </div>. <div class="year">2013</div>, <div class="article-title">Gene Ontology annotations and resources</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">41</div>, <div class="fpage">D530</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20Ontology%20annotations%20and%20resources&amp;author=J.%20Blake&amp;author=M.%20Dolan&amp;publication_year=2013&amp;journal=Nucleic%20Acids%20Res.&amp;volume=41&amp;pages=D530-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gks1050" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgks1050" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgks1050"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23161678" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV028C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV028C23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Burset</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Seledtsov</div> <div class="given-names">I.A.</div></div>, <div class="name string-name"><div class="surname">Solovyev</div> <div class="given-names">V.V.</div></div></div>
          <div class="year">2000</div>, <div class="article-title">Analysis of canonical and non-canonical splice sites in mammalian genomes</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">28</div>, <div class="fpage">4364</div>–<div class="lpage">75</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20canonical%20and%20non-canonical%20splice%20sites%20in%20mammalian%20genomes&amp;author=M.%20Burset&amp;author=I.A.%20Seledtsov&amp;author=V.V.%20Solovyev&amp;publication_year=2000&amp;journal=Nucleic%20Acids%20Res.&amp;volume=28&amp;pages=4364-75" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/28.21.4364" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F28.21.4364" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F28.21.4364"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11058137" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV028C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV028C24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Alberts</div>
               <div class="given-names">B.</div>
            </div>, <div class="name string-name"><div class="surname">Johnson</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Lewis</div> <div class="given-names">J.</div></div>et al.  </div>. <div class="year">2014</div>, <div class="source">Molecular biology of the cell</div> , <div class="edition">6th edition</div>. <div class="publisher-name">Garland Science</div>: <div class="publisher-loc">New York</div>.</div></div></div></div><div content-id="DSV028C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV028C25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Shimada</div>
               <div class="given-names">M.K.</div>
            </div>, <div class="name string-name"><div class="surname">Sasaki-Haraguchi</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Mayeda</div> <div class="given-names">A.</div></div></div>
          <div class="year">2015</div>, <div class="article-title">Identification and validation of evolutionarily conserved unusually short pre-mRNA introns in the human genome</div>, <div class="source">Int. J. Mol. Sci.</div> , <div class="volume">16</div>, <div class="fpage">10376</div>–<div class="lpage">88</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20and%20validation%20of%20evolutionarily%20conserved%20unusually%20short%20pre-mRNA%20introns%20in%20the%20human%20genome&amp;author=M.K.%20Shimada&amp;author=N.%20Sasaki-Haraguchi&amp;author=A.%20Mayeda&amp;publication_year=2015&amp;journal=Int.%20J.%20Mol.%20Sci.&amp;volume=16&amp;pages=10376-88" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3390/ijms160510376" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3390%2Fijms160510376" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3390%2Fijms160510376"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25961948" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV028C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV028C26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Casadei</div>
               <div class="given-names">R.</div>
            </div>, <div class="name string-name"><div class="surname">Piovesan</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Vitale</div> <div class="given-names">L.</div></div>et al.  </div>. <div class="year">2012</div>, <div class="article-title">Genome-scale analysis of human mRNA 5′ coding sequences based on expressed sequence tag (EST) database</div>, <div class="source">Genomics</div> , <div class="volume">100</div>, <div class="fpage">125</div>–<div class="lpage">30</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-scale%20analysis%20of%20human%20mRNA%205%E2%80%B2%20coding%20sequences%20based%20on%20expressed%20sequence%20tag%20(EST)%20database&amp;author=R.%20Casadei&amp;author=A.%20Piovesan&amp;author=L.%20Vitale&amp;publication_year=2012&amp;journal=Genomics&amp;volume=100&amp;pages=125-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ygeno.2012.05.012" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ygeno.2012.05.012" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ygeno.2012.05.012"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22659028" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV028C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV028C27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Piovesan</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Caracausi</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Pelleri</div> <div class="given-names">M.C.</div></div>et al.  </div>. <div class="year">2014</div>, <div class="article-title">Improving mRNA 5′ coding sequence determination in the mouse genome</div>, <div class="source">Mamm. Genome</div> , <div class="volume">25</div>, <div class="fpage">149</div>–<div class="lpage">59</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Improving%20mRNA%205%E2%80%B2%20coding%20sequence%20determination%20in%20the%20mouse%20genome&amp;author=A.%20Piovesan&amp;author=M.%20Caracausi&amp;author=M.C.%20Pelleri&amp;publication_year=2014&amp;journal=Mamm.%20Genome&amp;volume=25&amp;pages=149-59" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00335-013-9498-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00335-013-9498-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00335-013-9498-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24504701" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV028C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV028C28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Etzold</div>
               <div class="given-names">T.</div>
            </div>, <div class="name string-name"><div class="surname">Ulyanov</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Argos</div> <div class="given-names">P.</div></div></div>
          <div class="year">1996</div>, <div class="article-title">SRS: information retrieval system for molecular biology data banks</div>, <div class="source">Methods Enzymol.</div> , <div class="volume">266</div>, <div class="fpage">114</div>–<div class="lpage">28</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SRS%3A%20information%20retrieval%20system%20for%20molecular%20biology%20data%20banks&amp;author=T.%20Etzold&amp;author=A.%20Ulyanov&amp;author=P.%20Argos&amp;publication_year=1996&amp;journal=Methods%20Enzymol.&amp;volume=266&amp;pages=114-28" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8743681" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV028C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV028C29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">D'Addabbo</div>
               <div class="given-names">P.</div>
            </div>, <div class="name string-name"><div class="surname">Lenzi</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Facchin</div> <div class="given-names">F.</div></div>et al.  </div>. <div class="year">2004</div>, <div class="article-title">GeneRecords: a relational database for GenBank flat file parsing and data manipulation in personal computers</div>, <div class="source">Bioinformatics</div> , <div class="volume">20</div>, <div class="fpage">2883</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GeneRecords%3A%20a%20relational%20database%20for%20GenBank%20flat%20file%20parsing%20and%20data%20manipulation%20in%20personal%20computers&amp;author=P.%20D'Addabbo&amp;author=L.%20Lenzi&amp;author=F.%20Facchin&amp;publication_year=2004&amp;journal=Bioinformatics&amp;volume=20&amp;pages=2883-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bth321" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbth321" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbth321"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15145799" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV028C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV028C30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Zhang</div>
               <div class="given-names">M.Q.</div>
            </div>
          </div>
          <div class="year">1998</div>, <div class="article-title">Statistical features of human exons and their flanking regions</div>, <div class="source">Hum. Mol. Genet.</div> , <div class="volume">7</div>, <div class="fpage">919</div>–<div class="lpage">32</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Statistical%20features%20of%20human%20exons%20and%20their%20flanking%20regions&amp;author=M.Q.%20Zhang&amp;publication_year=1998&amp;journal=Hum.%20Mol.%20Genet.&amp;volume=7&amp;pages=919-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/hmg/7.5.919" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fhmg%2F7.5.919" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fhmg%2F7.5.919"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9536098" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV028C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV028C31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Isidor</div>
               <div class="given-names">B.</div>
            </div>, <div class="name string-name"><div class="surname">Lindenbaum</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Pichon</div> <div class="given-names">O.</div></div>et al.  </div>. <div class="year">2011</div>, <div class="article-title">Truncating mutations in the last exon of NOTCH2 cause a rare skeletal disorder with osteoporosis</div>, <div class="source">Nat. Genet.</div> , <div class="volume">43</div>, <div class="fpage">306</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Truncating%20mutations%20in%20the%20last%20exon%20of%20NOTCH2%20cause%20a%20rare%20skeletal%20disorder%20with%20osteoporosis&amp;author=B.%20Isidor&amp;author=P.%20Lindenbaum&amp;author=O.%20Pichon&amp;publication_year=2011&amp;journal=Nat.%20Genet.&amp;volume=43&amp;pages=306-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.778" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.778" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.778"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21378989" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV028C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV028C32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Saxonov</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Daizadeh</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Fedorov</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Gilbert</div> <div class="given-names">W.</div></div></div>
          <div class="year">2000</div>, <div class="article-title">EID: the Exon-Intron Database-an exhaustive database of protein-coding intron-containing genes</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">28</div>, <div class="fpage">185</div>–<div class="lpage">90</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=EID%3A%20the%20Exon-Intron%20Database-an%20exhaustive%20database%20of%20protein-coding%20intron-containing%20genes&amp;author=S.%20Saxonov&amp;author=I.%20Daizadeh&amp;author=A.%20Fedorov&amp;author=W.%20Gilbert&amp;publication_year=2000&amp;journal=Nucleic%20Acids%20Res.&amp;volume=28&amp;pages=185-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/28.1.185" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F28.1.185" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F28.1.185"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10592221" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV028C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV028C33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Sakharkar</div>
               <div class="given-names">M.K.</div>
            </div>, <div class="name string-name"><div class="surname">Kangueane</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Woon</div> <div class="given-names">T.W.</div></div>et al.  </div>. <div class="year">2000</div>, <div class="article-title">IE-Kb: intron exon knowledge base</div>, <div class="source">Bioinformatics</div> , <div class="volume">16</div>, <div class="fpage">1151</div>–<div class="lpage">2</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=IE-Kb%3A%20intron%20exon%20knowledge%20base&amp;author=M.K.%20Sakharkar&amp;author=P.%20Kangueane&amp;author=T.W.%20Woon&amp;publication_year=2000&amp;journal=Bioinformatics&amp;volume=16&amp;pages=1151-2" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/16.12.1151" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2F16.12.1151" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2F16.12.1151"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11159334" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV028C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV028C34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Schisler</div>
               <div class="given-names">N.J.</div>
            </div>, <div class="name string-name"><div class="surname">Palmer</div> <div class="given-names">J.D.</div></div></div>
          <div class="year">2000</div>, <div class="article-title">The IDB and IEDB: intron sequence and evolution databases</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">28</div>, <div class="fpage">181</div>–<div class="lpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20IDB%20and%20IEDB%3A%20intron%20sequence%20and%20evolution%20databases&amp;author=N.J.%20Schisler&amp;author=J.D.%20Palmer&amp;publication_year=2000&amp;journal=Nucleic%20Acids%20Res.&amp;volume=28&amp;pages=181-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/28.1.181" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F28.1.181" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F28.1.181"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10592220" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV028C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV028C35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Sakharkar</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Passetti</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">de Souza</div> <div class="given-names">J.E.</div></div>, <div class="name string-name"><div class="surname">Long</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">de Souza</div> <div class="given-names">S.J.</div></div></div>
          <div class="year">2002</div>, <div class="article-title">ExInt: an exon intron database</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">30</div>, <div class="fpage">191</div>–<div class="lpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=ExInt%3A%20an%20exon%20intron%20database&amp;author=M.%20Sakharkar&amp;author=F.%20Passetti&amp;author=J.E.%20de%20Souza&amp;author=M.%20Long&amp;author=S.J.%20de%20Souza&amp;publication_year=2002&amp;journal=Nucleic%20Acids%20Res.&amp;volume=30&amp;pages=191-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/30.1.191" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F30.1.191" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F30.1.191"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11752290" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV028C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV028C36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Sakharkar</div>
               <div class="given-names">M.K.</div>
            </div>, <div class="name string-name"><div class="surname">Kangueane</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Petrov</div> <div class="given-names">D.A.</div></div>, <div class="name string-name"><div class="surname">Kolaskar</div> <div class="given-names">A.S.</div></div>, <div class="name string-name"><div class="surname">Subbiah</div> <div class="given-names">S.</div></div></div>
          <div class="year">2002</div>, <div class="article-title">SEGE: a database on ‘intron less/single exonic’ genes from eukaryotes</div>, <div class="source">Bioinformatics</div> , <div class="volume">18</div>, <div class="fpage">1266</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SEGE%3A%20a%20database%20on%20%E2%80%98intron%20less%2Fsingle%20exonic%E2%80%99%20genes%20from%20eukaryotes&amp;author=M.K.%20Sakharkar&amp;author=P.%20Kangueane&amp;author=D.A.%20Petrov&amp;author=A.S.%20Kolaskar&amp;author=S.%20Subbiah&amp;publication_year=2002&amp;journal=Bioinformatics&amp;volume=18&amp;pages=1266-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/18.9.1266" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2F18.9.1266" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2F18.9.1266"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12217920" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV028C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV028C37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Gopalan</div>
               <div class="given-names">V.</div>
            </div>, <div class="name string-name"><div class="surname">Tan</div> <div class="given-names">T.W.</div></div>, <div class="name string-name"><div class="surname">Lee</div> <div class="given-names">B.T.K.</div></div>, <div class="name string-name"><div class="surname">Ranganathan</div> <div class="given-names">S.</div></div></div>
          <div class="year">2004</div>, <div class="article-title">Xpro: database of eukaryotic protein-encoding genes</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">32</div>, <div class="fpage">D59</div>–<div class="lpage">63</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Xpro%3A%20database%20of%20eukaryotic%20protein-encoding%20genes&amp;author=V.%20Gopalan&amp;author=T.W.%20Tan&amp;author=B.T.K.%20Lee&amp;author=S.%20Ranganathan&amp;publication_year=2004&amp;journal=Nucleic%20Acids%20Res.&amp;volume=32&amp;pages=D59-63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkh051" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkh051" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkh051"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14681359" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV028C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV028C38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Shepelev</div>
               <div class="given-names">V.</div>
            </div>, <div class="name string-name"><div class="surname">Fedorov</div> <div class="given-names">A.</div></div></div>
          <div class="year">2006</div>, <div class="article-title">Advances in the exon-intron database (EID)</div>, <div class="source">Brief Bioinform.</div> , <div class="volume">7</div>, <div class="fpage">178</div>–<div class="lpage">85</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Advances%20in%20the%20exon-intron%20database%20(EID)&amp;author=V.%20Shepelev&amp;author=A.%20Fedorov&amp;publication_year=2006&amp;journal=Brief%20Bioinform.&amp;volume=7&amp;pages=178-85" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bib/bbl003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbib%2Fbbl003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbib%2Fbbl003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16772261" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV028C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV028C39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Sasaki-Haraguchi</div>
               <div class="given-names">N.</div>
            </div>, <div class="name string-name"><div class="surname">Shimada</div> <div class="given-names">M.K.</div></div>, <div class="name string-name"><div class="surname">Taniguchi</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Ohno</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Mayeda</div> <div class="given-names">A.</div></div></div>
          <div class="year">2012</div>, <div class="article-title">Mechanistic insights into human pre-mRNA splicing of human ultra-short introns: potential unusual mechanism identifies G-rich introns</div>, <div class="source">Biochem. Biophys. Res. Commun.</div> , <div class="volume">423</div>, <div class="fpage">289</div>–<div class="lpage">94</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mechanistic%20insights%20into%20human%20pre-mRNA%20splicing%20of%20human%20ultra-short%20introns%3A%20potential%20unusual%20mechanism%20identifies%20G-rich%20introns&amp;author=N.%20Sasaki-Haraguchi&amp;author=M.K.%20Shimada&amp;author=I.%20Taniguchi&amp;author=M.%20Ohno&amp;author=A.%20Mayeda&amp;publication_year=2012&amp;journal=Biochem.%20Biophys.%20Res.%20Commun.&amp;volume=423&amp;pages=289-94" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.bbrc.2012.05.112" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.bbrc.2012.05.112" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.bbrc.2012.05.112"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22640740" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV028C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV028C40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Spingola</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Grate</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Haussler</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Ares</div> <div class="given-names">M.</div></div></div>
          <div class="year">1999</div>, <div class="article-title">Genome-wide bioinformatic and molecular analysis of introns in <em>Saccharomyces cerevisiae</em></div>, <div class="source">RNA</div> , <div class="volume">5</div>, <div class="fpage">221</div>–<div class="lpage">34</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20bioinformatic%20and%20molecular%20analysis%20of%20introns%20in%20Saccharomyces%20cerevisiae&amp;author=M.%20Spingola&amp;author=L.%20Grate&amp;author=D.%20Haussler&amp;author=M.%20Ares&amp;publication_year=1999&amp;journal=RNA&amp;volume=5&amp;pages=221-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1017/S1355838299981682" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1017%2FS1355838299981682" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1017%2FS1355838299981682"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10024174" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV028C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV028C41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Huala</div>
               <div class="given-names">E.</div>
            </div>, <div class="name string-name"><div class="surname">Dickerman</div> <div class="given-names">A.W.</div></div>, <div class="name string-name"><div class="surname">Garcia-Hernandez</div> <div class="given-names">M.</div></div>et al.  </div>. <div class="year">2001</div>, <div class="article-title">The Arabidopsis Information Resource (TAIR): a comprehensive database and web-based information retrieval, analysis, and visualization system for a model plant</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">29</div>, <div class="fpage">102</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Arabidopsis%20Information%20Resource%20(TAIR)%3A%20a%20comprehensive%20database%20and%20web-based%20information%20retrieval%2C%20analysis%2C%20and%20visualization%20system%20for%20a%20model%20plant&amp;author=E.%20Huala&amp;author=A.W.%20Dickerman&amp;author=M.%20Garcia-Hernandez&amp;publication_year=2001&amp;journal=Nucleic%20Acids%20Res.&amp;volume=29&amp;pages=102-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/29.1.102" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F29.1.102" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F29.1.102"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11125061" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV028C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV028C42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Burge</div>
               <div class="given-names">C.B.</div>
            </div>, <div class="name string-name"><div class="surname">Tuschl</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Sharp</div> <div class="given-names">T.A.</div></div></div>
          <div class="year">1999</div>, <div class="article-title">Splicing of precursors to mRNAs by the spliceosomes</div>. In: <div class="person-group"><div class="name string-name"><div class="surname">Gesteland</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Cech</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Atkins</div> <div class="given-names">J.</div></div></div> (eds), <div class="comment">The RNA World</div>, <div class="edition">2nd edition</div>. <div class="publisher-name">Cold Spring Harbor Laboratory Press</div>: <div class="publisher-loc">Cold Spring Harbor, NY</div>, pp.<div class="fpage">525</div>–<div class="lpage">60</div>.</div></div></div></div><div content-id="DSV028C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV028C43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Pei</div>
               <div class="given-names">B.</div>
            </div>, <div class="name string-name"><div class="surname">Sisu</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Frankish</div> <div class="given-names">A.</div></div>et al.  </div>. <div class="year">2012</div>, <div class="article-title">The GENCODE pseudogene resource</div>, <div class="source">Genome Biol.</div> , <div class="volume">13</div>, <div class="fpage">R51</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20GENCODE%20pseudogene%20resource&amp;author=B.%20Pei&amp;author=C.%20Sisu&amp;author=A.%20Frankish&amp;publication_year=2012&amp;journal=Genome%20Biol.&amp;volume=13&amp;pages=R51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2012-13-9-r51" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2012-13-9-r51" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2012-13-9-r51"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22951037" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV028C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV028C44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Sinha</div>
               <div class="given-names">R.</div>
            </div>, <div class="name string-name"><div class="surname">Lenser</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Jahn</div> <div class="given-names">N.</div></div>et al.  </div>. <div class="year">2010</div>, <div class="article-title">TassDB2—a comprehensive database of subtle alternative splicing events</div>, <div class="source">BMC Bioinformatics</div> , <div class="volume">11</div>, <div class="fpage">216</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=TassDB2%E2%80%94a%20comprehensive%20database%20of%20subtle%20alternative%20splicing%20events&amp;author=R.%20Sinha&amp;author=T.%20Lenser&amp;author=N.%20Jahn&amp;publication_year=2010&amp;journal=BMC%20Bioinformatics&amp;volume=11&amp;pages=216" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-11-216" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-11-216" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-11-216"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20429909" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV028C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV028C45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Hiller</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Huse</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Szafranski</div> <div class="given-names">K.</div></div>et al.  </div>. <div class="year">2004</div>, <div class="article-title">Widespread occurrence of alternative splicing at NAGNAG acceptors contributes to proteome plasticity</div>, <div class="source">Nat. Genet.</div> , <div class="volume">36</div>, <div class="fpage">1255</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Widespread%20occurrence%20of%20alternative%20splicing%20at%20NAGNAG%20acceptors%20contributes%20to%20proteome%20plasticity&amp;author=M.%20Hiller&amp;author=K.%20Huse&amp;author=K.%20Szafranski&amp;publication_year=2004&amp;journal=Nat.%20Genet.&amp;volume=36&amp;pages=1255-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng1469" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng1469" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng1469"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15516930" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV028C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV028C46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Vitale</div>
               <div class="given-names">L.</div>
            </div>, <div class="name string-name"><div class="surname">Lenzi</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Huntsman</div> <div class="given-names">S.A.</div></div>et al.  </div>. <div class="year">2006</div>, <div class="article-title">Differential expression of alternatively spliced mRNA forms of the insulin-like growth factor 1 receptor in human neuroendocrine tumors</div>, <div class="source">Oncol. Rep.</div> , <div class="volume">15</div>, <div class="fpage">1249</div>–<div class="lpage">56</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Differential%20expression%20of%20alternatively%20spliced%20mRNA%20forms%20of%20the%20insulin-like%20growth%20factor%201%20receptor%20in%20human%20neuroendocrine%20tumors&amp;author=L.%20Vitale&amp;author=L.%20Lenzi&amp;author=S.A.%20Huntsman&amp;publication_year=2006&amp;journal=Oncol.%20Rep.&amp;volume=15&amp;pages=1249-56" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16596194" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV028C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV028C47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Vitale</div>
               <div class="given-names">L.</div>
            </div>, <div class="name string-name"><div class="surname">Frabetti</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Huntsman</div> <div class="given-names">S.A.</div></div>et al.  </div>. <div class="year">2007</div>, <div class="article-title">Sequence, ‘subtle’ alternative splicing and expression of the CYYR1 (cysteine/tyrosine-rich 1) mRNA in human neuroendocrine tumors</div>, <div class="source">BMC Cancer</div> , <div class="volume">7</div>, <div class="fpage">66</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequence%2C%20%E2%80%98subtle%E2%80%99%20alternative%20splicing%20and%20expression%20of%20the%20CYYR1%20(cysteine%2Ftyrosine-rich%201)%20mRNA%20in%20human%20neuroendocrine%20tumors&amp;author=L.%20Vitale&amp;author=F.%20Frabetti&amp;author=S.A.%20Huntsman&amp;publication_year=2007&amp;journal=BMC%20Cancer&amp;volume=7&amp;pages=66" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2407-7-66" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2407-7-66" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2407-7-66"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17442112" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV028C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV028C48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Pickrell</div>
               <div class="given-names">J.K.</div>
            </div>, <div class="name string-name"><div class="surname">Pai</div> <div class="given-names">A.A.</div></div>, <div class="name string-name"><div class="surname">Gilad</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Pritchard</div> <div class="given-names">J.K.</div></div></div>
          <div class="year">2010</div>, <div class="article-title">Noisy splicing drives mRNA isoform diversity in human cells</div>, <div class="source">PLoS Genet.</div> , <div class="volume">6</div>, <div class="fpage">e1001236</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Noisy%20splicing%20drives%20mRNA%20isoform%20diversity%20in%20human%20cells&amp;author=J.K.%20Pickrell&amp;author=A.A.%20Pai&amp;author=Y.%20Gilad&amp;author=J.K.%20Pritchard&amp;publication_year=2010&amp;journal=PLoS%20Genet.&amp;volume=6&amp;pages=e1001236" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1001236" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1001236" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1001236"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21151575" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV028C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV028C49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Wang</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Shu</div> <div class="given-names">Y.</div></div>et al.  </div>. <div class="year">2014</div>, <div class="article-title">Alternative splicing at GYNNGY 5′ splice sites: more noise, less regulation</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">42</div>, <div class="fpage">13969</div>–<div class="lpage">80</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Alternative%20splicing%20at%20GYNNGY%205%E2%80%B2%20splice%20sites%3A%20more%20noise%2C%20less%20regulation&amp;author=M.%20Wang&amp;author=P.%20Zhang&amp;author=Y.%20Shu&amp;publication_year=2014&amp;journal=Nucleic%20Acids%20Res.&amp;volume=42&amp;pages=13969-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gku1253" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgku1253" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgku1253"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25428370" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV028C50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV028C50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Jian</div>
               <div class="given-names">X.</div>
            </div>, <div class="name string-name"><div class="surname">Boerwinkle</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">X.</div></div></div>
          <div class="year">2014</div>, <div class="article-title">In silico prediction of splice-altering single nucleotide variants in the human genome</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">42</div>, <div class="fpage">13534</div>–<div class="lpage">44</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=In%20silico%20prediction%20of%20splice-altering%20single%20nucleotide%20variants%20in%20the%20human%20genome&amp;author=X.%20Jian&amp;author=E.%20Boerwinkle&amp;author=X.%20Liu&amp;publication_year=2014&amp;journal=Nucleic%20Acids%20Res.&amp;volume=42&amp;pages=13534-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gku1206" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgku1206" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgku1206"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25416802" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV028C51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV028C51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Parada</div>
               <div class="given-names">G.E.</div>
            </div>, <div class="name string-name"><div class="surname">Munita</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Cerda</div> <div class="given-names">C.A.</div></div>, <div class="name string-name"><div class="surname">Gysling</div> <div class="given-names">K.</div></div></div>
          <div class="year">2014</div>, <div class="article-title">A comprehensive survey of non-canonical splice sites in the human transcriptome</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">42</div>, <div class="fpage">10564</div>–<div class="lpage">78</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20comprehensive%20survey%20of%20non-canonical%20splice%20sites%20in%20the%20human%20transcriptome&amp;author=G.E.%20Parada&amp;author=R.%20Munita&amp;author=C.A.%20Cerda&amp;author=K.%20Gysling&amp;publication_year=2014&amp;journal=Nucleic%20Acids%20Res.&amp;volume=42&amp;pages=10564-78" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gku744" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgku744" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgku744"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25123659" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV028C52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV028C52" class="ref-content "><div class="label -label">52</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Chorev</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Carmel</div> <div class="given-names">L.</div></div></div>
          <div class="year">2013</div>, <div class="article-title">Computational identification of functional introns: high positional conservation of introns that harbor RNA genes</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">41</div>, <div class="fpage">5604</div>–<div class="lpage">13</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Computational%20identification%20of%20functional%20introns%3A%20high%20positional%20conservation%20of%20introns%20that%20harbor%20RNA%20genes&amp;author=M.%20Chorev&amp;author=L.%20Carmel&amp;publication_year=2013&amp;journal=Nucleic%20Acids%20Res.&amp;volume=41&amp;pages=5604-13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkt244" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkt244" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkt244"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23605046" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV028C53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV028C53" class="ref-content "><div class="label -label">53</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Castillo-Davis</div>
               <div class="given-names">C.I.</div>
            </div>, <div class="name string-name"><div class="surname">Mekhedov</div> <div class="given-names">S.L.</div></div>, <div class="name string-name"><div class="surname">Hartl</div> <div class="given-names">D.L.</div></div>, <div class="name string-name"><div class="surname">Koonin</div> <div class="given-names">E.V.</div></div>, <div class="name string-name"><div class="surname">Kondrashov</div> <div class="given-names">F.A.</div></div></div>
          <div class="year">2002</div>, <div class="article-title">Selection for short introns in highly expressed genes</div>, <div class="source">Nat. Genet.</div> , <div class="volume">31</div>, <div class="fpage">415</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Selection%20for%20short%20introns%20in%20highly%20expressed%20genes&amp;author=C.I.%20Castillo-Davis&amp;author=S.L.%20Mekhedov&amp;author=D.L.%20Hartl&amp;author=E.V.%20Koonin&amp;author=F.A.%20Kondrashov&amp;publication_year=2002&amp;journal=Nat.%20Genet.&amp;volume=31&amp;pages=415-8" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12134150" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV028C54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV028C54" class="ref-content "><div class="label -label">54</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Pozzoli</div>
               <div class="given-names">U.</div>
            </div>, <div class="name string-name"><div class="surname">Menozzi</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Comi</div> <div class="given-names">G.P.</div></div>, <div class="name string-name"><div class="surname">Cagliani</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Bresolin</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Sironi</div> <div class="given-names">M.</div></div></div>
          <div class="year">2007</div>, <div class="article-title">Intron size in mammals: complexity comes to terms with economy</div>, <div class="source">Trends Genet.</div> , <div class="volume">23</div>, <div class="fpage">20</div>–<div class="lpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Intron%20size%20in%20mammals%3A%20complexity%20comes%20to%20terms%20with%20economy&amp;author=U.%20Pozzoli&amp;author=G.%20Menozzi&amp;author=G.P.%20Comi&amp;author=R.%20Cagliani&amp;author=N.%20Bresolin&amp;author=M.%20Sironi&amp;publication_year=2007&amp;journal=Trends%20Genet.&amp;volume=23&amp;pages=20-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tig.2006.10.003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tig.2006.10.003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tig.2006.10.003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17070957" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV028C55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV028C55" class="ref-content "><div class="label -label">55</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Stival Sena</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Giguère</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Boyle</div> <div class="given-names">B.</div></div>et al.  </div>. <div class="year">2014</div>, <div class="article-title">Evolution of gene structure in the conifer Picea glauca: a comparative analysis of the impact of intron size</div>, <div class="source">BMC Plant Biol.</div> , <div class="volume">14</div>, <div class="fpage">95</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20of%20gene%20structure%20in%20the%20conifer%20Picea%20glauca%3A%20a%20comparative%20analysis%20of%20the%20impact%20of%20intron%20size&amp;author=J.%20Stival%20Sena&amp;author=I.%20Gigu%C3%A8re&amp;author=B.%20Boyle&amp;publication_year=2014&amp;journal=BMC%20Plant%20Biol.&amp;volume=14&amp;pages=95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2229-14-95" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2229-14-95" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2229-14-95"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24734980" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV028C56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV028C56" class="ref-content "><div class="label -label">56</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Piovesan</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Vitale</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Pelleri</div> <div class="given-names">M.C.</div></div>, <div class="name string-name"><div class="surname">Strippoli</div> <div class="given-names">P.</div></div></div>
          <div class="year">2013</div>, <div class="article-title">Universal tight correlation of codon bias and pool of RNA codons (codonome): the genome is optimized to allow any distribution of gene expression values in the transcriptome from bacteria to humans</div>, <div class="source">Genomics</div> , <div class="volume">101</div>, <div class="fpage">282</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Universal%20tight%20correlation%20of%20codon%20bias%20and%20pool%20of%20RNA%20codons%20(codonome)%3A%20the%20genome%20is%20optimized%20to%20allow%20any%20distribution%20of%20gene%20expression%20values%20in%20the%20transcriptome%20from%20bacteria%20to%20humans&amp;author=A.%20Piovesan&amp;author=L.%20Vitale&amp;author=M.C.%20Pelleri&amp;author=P.%20Strippoli&amp;publication_year=2013&amp;journal=Genomics&amp;volume=101&amp;pages=282-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ygeno.2013.02.009" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ygeno.2013.02.009" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ygeno.2013.02.009"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23466472" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV028C57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV028C57" class="ref-content "><div class="label -label">57</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Caracausi</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Vitale</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Pelleri</div> <div class="given-names">M.C.</div></div>, <div class="name string-name"><div class="surname">Piovesan</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Bruno</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Strippoli</div> <div class="given-names">P.</div></div></div>
          <div class="year">2014</div>, <div class="article-title">A quantitative transcriptome reference map of the normal human brain</div>, <div class="source">Neurogenetics</div> , <div class="volume">15</div>, <div class="fpage">267</div>–<div class="lpage">87</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20quantitative%20transcriptome%20reference%20map%20of%20the%20normal%20human%20brain&amp;author=M.%20Caracausi&amp;author=L.%20Vitale&amp;author=M.C.%20Pelleri&amp;author=A.%20Piovesan&amp;author=S.%20Bruno&amp;author=P.%20Strippoli&amp;publication_year=2014&amp;journal=Neurogenetics&amp;volume=15&amp;pages=267-87" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10048-014-0419-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10048-014-0419-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10048-014-0419-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25185649" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV028C58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV028C58" class="ref-content "><div class="label -label">58</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Pelleri</div>
               <div class="given-names">M.C.</div>
            </div>, <div class="name string-name"><div class="surname">Piovesan</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Caracausi</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Berardi</div> <div class="given-names">A.C.</div></div>, <div class="name string-name"><div class="surname">Vitale</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Strippoli</div> <div class="given-names">P.</div></div></div>
          <div class="year">2014</div>, <div class="article-title">Integrated differential transcriptome maps of Acute Megakaryoblastic Leukemia (AMKL) in children with or without Down Syndrome (DS)</div>, <div class="source">BMC Med. Genomics</div> , <div class="volume">7</div>, <div class="fpage">63</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Integrated%20differential%20transcriptome%20maps%20of%20Acute%20Megakaryoblastic%20Leukemia%20(AMKL)%20in%20children%20with%20or%20without%20Down%20Syndrome%20(DS)&amp;author=M.C.%20Pelleri&amp;author=A.%20Piovesan&amp;author=M.%20Caracausi&amp;author=A.C.%20Berardi&amp;author=L.%20Vitale&amp;author=P.%20Strippoli&amp;publication_year=2014&amp;journal=BMC%20Med.%20Genomics&amp;volume=7&amp;pages=63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12920-014-0063-z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12920-014-0063-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12920-014-0063-z"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25476127" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV028C59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV028C59" class="ref-content "><div class="label -label">59</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Strippoli</div>
               <div class="given-names">P.</div>
            </div>, <div class="name string-name"><div class="surname">Pelleri</div> <div class="given-names">M.C.</div></div>, <div class="name string-name"><div class="surname">Caracausi</div> <div class="given-names">M.</div></div>et al.  </div>. <div class="year">2013</div>, <div class="article-title">An integrated route to identifying new pathogenesis-based therapeutic approaches for trisomy 21 (Down Syndrome) following the thought of Jérôme Lejeune</div>, <div class="source">Sci. Postprint</div> , <div class="volume">1</div>, <div class="fpage">e00010</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20integrated%20route%20to%20identifying%20new%20pathogenesis-based%20therapeutic%20approaches%20for%20trisomy%2021%20(Down%20Syndrome)%20following%20the%20thought%20of%20J%C3%A9r%C3%B4me%20Lejeune&amp;author=P.%20Strippoli&amp;author=M.C.%20Pelleri&amp;author=M.%20Caracausi&amp;publication_year=2013&amp;journal=Sci.%20Postprint&amp;volume=1&amp;pages=e00010" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.14340/spp.2013.12R0005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.14340%2Fspp.2013.12R0005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.14340%2Fspp.2013.12R0005"> </span></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2015. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=2462510&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv028/1/dsv028_Supplementary_Data.zip?Expires=2147483647&amp;Signature=aRYnjW0Rb4P1z~LqsvpBA4F8yDeKnemALjuIs5dIu7b78ZIyCC9vStwlmcaLpwYHHynJbVrUGwOnQs3sBQ7xB6WMmPBg1XPcJaNy2tx~WiUOnNubYatD8rrHP4-b0kqD26V-9w-T5~7ViNvHPhtETEQ4Qe0MRM7nMeiJQfNtapr3~Sd2ReGizplxkP2zCfwUUBhYjaeRaKj4EGiQwmBeXrBErc~TaNkS~uGM0gU0lcrArfMkDjnNsYKFTvojR-CLBO9iYZel7PwBnhK6Iu2q-GunrXWXN3nleQzH4Vl5aPuE12cNg9dJn3hbWN8XW-hFI-vG6tJ~WhfJ0dVGoR3IGQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">278</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000366631700010&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">10</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsv028"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="2462510"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000366631700010&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f22%2f6%2f495%2f2462510">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000366631700010&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (10)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F22%2F6%2F495%2F2462510" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/2462510'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Identification of minimal eukaryotic introns through GeneBase, a user-friendly tool for parsing the NCBI Gene databank" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsv028");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f22%2f6%2f495%2f2462510">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

