

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/12/4/221/621324","@type":"ScholarlyArticle","name":"Characterization of Mononucleotide Repeats in Sequenced Prokaryotic Genomes","datePublished":"2005-01-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/12/4","@type":"PublicationIssue","issueNumber":"4","datePublished":"2005-01-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsi009","keywords":["mononucleotide repeats","simple sequence repeats","comparative genomics","genome evolution"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/12/4/221/621324","author":[{"name":"Coenye, Tom","affiliation":"Laboratorium voor Microbiologie, Universiteit Gent  K.L. Ledeganckstraat 35, B-9000 Gent, Belgium","@type":"Person"},{"name":"Vandamme, Peter","affiliation":"Laboratorium voor Microbiologie, Universiteit Gent  K.L. Ledeganckstraat 35, B-9000 Gent, Belgium","@type":"Person"}],"description":"Abstract.  The increasing availability of prokaryotic genome sequences has shown that simple sequence repeats (SSRs) are widespread in prokaryotes and that ther","pageStart":"221","pageEnd":"233","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/12/4/0/m_dnaresearch12_4.cover.gif?Expires=2147483647&Signature=3uNJBvpswUl8ckNzhRD6LiG0MySGXqGkcTQYLlRIvfld1eLLpkZoI4Ma5qe4h0TQIrhjk1-qwhKlhwYLpQpMu3ntM421fXasS8~G0V0MwH6cjjfM9fdJn0EH2kkIRqpR9dzDT~TKvzm8K-R~6ztHb07HPr6KoeUx6Rgs8s~OeOw2mLTVUNaBcNQn8k14DbREAYbOxque3El9Lplnm2ngB~AQ0rwKokc~WvC44rQAaAltU0RG~GjoIdSMmocCDEXHDpbpBtxuUw6A9Y5q2U4JLWP3XC47XRtYoHhAyJWDFgU30hB9Q-HhZ4XJLURWeo2hPNBzP0pbMYzBduIAxMvdMQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Characterization of Mononucleotide Repeats in Sequenced Prokaryotic Genomes","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/12/4/0/m_dnaresearch12_4.cover.gif?Expires=2147483647&Signature=3uNJBvpswUl8ckNzhRD6LiG0MySGXqGkcTQYLlRIvfld1eLLpkZoI4Ma5qe4h0TQIrhjk1-qwhKlhwYLpQpMu3ntM421fXasS8~G0V0MwH6cjjfM9fdJn0EH2kkIRqpR9dzDT~TKvzm8K-R~6ztHb07HPr6KoeUx6Rgs8s~OeOw2mLTVUNaBcNQn8k14DbREAYbOxque3El9Lplnm2ngB~AQ0rwKokc~WvC44rQAaAltU0RG~GjoIdSMmocCDEXHDpbpBtxuUw6A9Y5q2U4JLWP3XC47XRtYoHhAyJWDFgU30hB9Q-HhZ4XJLURWeo2hPNBzP0pbMYzBduIAxMvdMQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Characterization of Mononucleotide Repeats in Sequenced Prokaryotic Genomes" />
    <meta property="og:description" content="Abstract.  The increasing availability of prokaryotic genome sequences has shown that simple sequence repeats (SSRs) are widespread in prokaryotes and that ther" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/12/4/221/621324" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2005-01-01" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Coenye, Tom" /><meta name="citation_author_institution" content="Laboratorium voor Microbiologie, Universiteit GentK.L. Ledeganckstraat 35, B-9000 Gent, Belgium" /><meta name="citation_author" content="Vandamme, Peter" /><meta name="citation_author_institution" content="Laboratorium voor Microbiologie, Universiteit GentK.L. Ledeganckstraat 35, B-9000 Gent, Belgium" /><meta name="citation_title" content="Characterization of Mononucleotide Repeats in Sequenced Prokaryotic Genomes" /><meta name="citation_firstpage" content="221" /><meta name="citation_lastpage" content="233" /><meta name="citation_doi" content="10.1093/dnares/dsi009" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="12" /><meta name="citation_issue" content="4" /><meta name="citation_publication_date" content="2005/01/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Short-sequence DNA repeats in prokaryotic genomes; Microbiol. Mol. Biol. Rev.;  citation_year=1998;  citation_volume=62;  citation_pages=275-293; " /><meta name="citation_reference" content="citation_title=Tandem repeats in complete bacterial genomes sequences. Sequence and structural analyses for comparative studies; Res. Microbiol.;  citation_year=1999;  citation_volume=150;  citation_pages=745-754; " /><meta name="citation_reference" content="citation_title=Slipped-strand mispairing: a major mechanism for DNA sequence evolution; Mol. Biol. Evol.;  citation_year=1987;  citation_volume=4;  citation_pages=203-221; " /><meta name="citation_reference" content="citation_title=Occurrence and structure–function relationship of pentameric short sequence repeats in microbial genomes; Res. Microbiol.;  citation_year=1999;  citation_volume=150;  citation_pages=617-626; " /><meta name="citation_reference" content="citation_title=Insertions, deletions, and the origin of microsatellites; Genet. Res.;  citation_year=2000;  citation_volume=76;  citation_pages=227-236; " /><meta name="citation_reference" content="citation_title=Two distinct modes of microsatellite mutation processes: Evidence from the complete genomic sequences of nine species; Genome Res.;  citation_year=2003;  citation_volume=13;  citation_pages=2242-2251; " /><meta name="citation_reference" content="citation_title=Promoters responsive to DNA bending: a common theme in prokaryotic gene expression; Microbiol. Rev.;  citation_year=1994;  citation_volume=58;  citation_pages=268-290; " /><meta name="citation_reference" content="citation_title=Transcription through a simple DNA repeat blocks replication elongation; EMBO J.;  citation_year=1998;  citation_volume=17;  citation_pages=5095-5102; " /><meta name="citation_reference" content="citation_title=Molecular switches—the ON and OFF of bacterial phase variation; Mol. Microbiol.;  citation_year=1999;  citation_volume=33;  citation_pages=919-932; " /><meta name="citation_reference" content="citation_title=Over-representation of repeats in stress response genes:a strategy to increase versatility under stressful conditions?; Nucleic Acids Res.;  citation_year=2002;  citation_volume=30;  citation_pages=1886-1894; " /><meta name="citation_reference" content="citation_title=Instability of repetitive DNA sequences: The role of replication in multiple mechanisms; Proc. Natl Acad. Sci. USA;  citation_year=2001;  citation_volume=98;  citation_pages=8319-8325; " /><meta name="citation_reference" content="citation_title=An appraisal of the potential for illegitimate recombination in bacterial genomes and its consequences: from duplications to genome reduction; Genome Res.;  citation_year=2003;  citation_volume=13;  citation_pages=1123-1132; " /><meta name="citation_reference" content="citation_title=Phylogenetic assignment of length variation at a microsatellite locus; Proc. Natl Acad. Sci. USA;  citation_year=1997;  citation_volume=94;  citation_pages=10745-10749; " /><meta name="citation_reference" content="citation_title=Association of chromosomal RNA with repetitive DNA; Proc. Natl. Acad. Sci. USA;  citation_year=1971;  citation_volume=68;  citation_pages=387-389; " /><meta name="citation_reference" content="citation_title=Tandem inverted repeats in mitochondrial DNA of petite mutants of Saccharomyces cerevisiae; Proc. Natl. Acad. Sci. USA;  citation_year=1974;  citation_volume=71;  citation_pages=1366-1370; " /><meta name="citation_reference" content="citation_title=A novel repeated element with Z-DNA-forming potential is widely found in evolutionarily diverse eukaryotic genomes; Proc. Natl Acad. Sci. USA;  citation_year=1982;  citation_volume=79;  citation_pages=6465-6469; " /><meta name="citation_reference" content="citation_title=Differential distribution of simple sequence repeats in eukaryotic genome sequences; Mol. Biol. Evol.;  citation_year=2001;  citation_volume=18;  citation_pages=1161-1167; " /><meta name="citation_reference" content="citation_title=Hypervariability of simple sequences as a general source for polymorphic DNA markers; Nucleic Acids Res.;  citation_year=1989;  citation_volume=17;  citation_pages=6463-6471; " /><meta name="citation_reference" content="citation_title=Simple sequences; Curr. Opin. Genet. Dev.;  citation_year=1994;  citation_volume=4;  citation_pages=832-837; " /><meta name="citation_reference" content="citation_title=Characteristic enrichment of DNA repeats in different genomes; Proc. Natl Acad. Sci. USA;  citation_year=1997;  citation_volume=94;  citation_pages=5237-5242; " /><meta name="citation_reference" content="citation_title=Abundant microsatellite polymorphisms in Saccharomyces cerevisiae, and the different distributions of microsatellites in eight prokaryotes and S. cerevisiae, result from strong mutation pressures and a variety of selective forces; Proc. Natl Acad. Sci. USA;  citation_year=1998;  citation_volume=95;  citation_pages=1647-1652; " /><meta name="citation_reference" content="citation_title=Cryptic simplicity is a major source of genetic variation; Nature;  citation_year=1986;  citation_volume=322;  citation_pages=652-656; " /><meta name="citation_reference" content="citation_title=Distinct frequency-distributions of homopolymeric DNA tracts in different genomes; Nucleic Acids Res.;  citation_year=1998;  citation_volume=26;  citation_pages=4056-4062; " /><meta name="citation_reference" content="citation_title=Simple sequence repeats in Escherichia coli: abundance, distribution, composition, and polymorphism; Genome Res.;  citation_year=2000;  citation_volume=10;  citation_pages=62-71; " /><meta name="citation_reference" content="citation_title=Simple sequence repeats and compositional bias in the bipartite Ralstonia solanacearum GMI1000 genome; BMC Genomics;  citation_year=2003;  citation_volume=4;  citation_pages=10" /><meta name="citation_reference" content="citation_title=Identification and characterisation of simple sequence repeats in the genomes of Shigella species; Gene;  citation_year=2003;  citation_volume=322;  citation_pages=85-92; " /><meta name="citation_reference" content="citation_title=The number of repeats expected in random nucleic acid sequences and found in genes; J. Theor. Biol.;  citation_year=1981;  citation_volume=91;  citation_pages=71-98; " /><meta name="citation_reference" content="citation_title=Artemis Sequence visualisation and annotation; Bioinformatics;  citation_year=2000;  citation_volume=16;  citation_pages=944-945; " /><meta name="citation_reference" content="citation_title=Gene duplication and biased functional retention of paralogs in bacterial genomes; Trends Microbiol.;  citation_year=2004;  citation_volume=12;  citation_pages=148-154; " /><meta name="citation_reference" content="citation_title=Biased biological functions of horizontally transferred genes in prokaryotic genomes; Nature Genet.;  citation_year=2004;  citation_volume=36;  citation_pages=760-766; " /><meta name="citation_reference" content="citation_title=Comparative genomic structure of prokaryotes; Annu. Rev. Genet.;  citation_year=2004;  citation_volume=38;  citation_pages=771-791; " /><meta name="citation_reference" content="citation_title=Genome update: Correlation of bacterial genomic properties; Microbiology;  citation_year=2004;  citation_volume=150;  citation_pages=3899-3903; " /><meta name="citation_reference" content="citation_title=Phase and antigenic variation in bacteria; Clin. Microbiol. Rev.;  citation_year=2004;  citation_volume=17;  citation_pages=581-611; " /><meta name="citation_reference" content="citation_title=Simple sequence repeats in the Helicobacter pylori genome; Mol. Microbiol.;  citation_year=1998;  citation_volume=27;  citation_pages=1091-1098; " /><meta name="citation_reference" content="citation_title=The diversity within an expanded and redefined repertoire of phase-variable genes in Helicobacter pylori; Microbiology;  citation_year=2004;  citation_volume=150;  citation_pages=817-830; " /><meta name="citation_reference" content="citation_title=Deciphering Campylobacter jejuni cell surface interactions from the genome sequence; Curr. Opin. Microbiol.;  citation_year=2001;  citation_volume=4;  citation_pages=35-40; " /><meta name="citation_reference" content="citation_title=Genome deterioration: loss of repeated sequences and accumulation of junk DNA; Genetica;  citation_year=2002;  citation_volume=115;  citation_pages=1-12; " /><meta name="citation_reference" content="citation_title=Molecular biology of mycoplasmas; Annu. Rev. Microbiol.;  citation_year=1996;  citation_volume=50;  citation_pages=25-57; " /><meta name="citation_reference" content="citation_title=Genome evolution in bacterial endosymbionts of insects; Nature Rev. Gen.;  citation_year=2002;  citation_volume=3;  citation_pages=850-861; " /><meta name="citation_reference" content="citation_title=Molecular basis of genetic instability of triplet repeats; J. Biol. Chem.;  citation_year=1996;  citation_volume=271;  citation_pages=2875-2878; " /><meta name="citation_reference" content="citation_title=Hypermutability of homonucleotide runs in mismatch repair and DNA polymerase proofreading yeast mutants; Mol. Cell. Biol.;  citation_year=1997;  citation_volume=17;  citation_pages=2859-2865; " /><meta name="citation_reference" content="citation_title=Simple sequence repeats as a source of quantitative genetic variation; Trends Genet.;  citation_year=1997;  citation_volume=13;  citation_pages=74-78; " /><meta name="citation_reference" content="citation_title=A DNA structural atlas for Escherichia coli; J. Mol. Biol.;  citation_year=2000;  citation_volume=299;  citation_pages=907-930; " /><meta name="citation_reference" content="citation_title=Bias of purine stretches in sequenced chromosomes; Comp. Chem.;  citation_year=2002;  citation_volume=26;  citation_pages=531-541; " /><meta name="citation_reference" content="citation_title=How the genome readies itself for evolution; Science;  citation_year=1998;  citation_volume=281;  citation_pages=1131-1134; " /><meta name="citation_reference" content="citation_title=Evolutionary tuning knob; Endeavour;  citation_year=1997;  citation_volume=21;  citation_pages=36-40; " /><meta name="citation_reference" content="citation_title=Indirect selection on the mutational landscape: An evolved role for the mutability of repetitive DNA?; citation_text=Evolution 2000, Indiana University Conferences, Bloomington, IN;  citation_year=2000; " /><meta name="citation_reference" content="citation_title=Amelioration of bacterial genomes: Rates of change and exchange; J. Mol. Evol.;  citation_year=1997;  citation_volume=44;  citation_pages=383-397; " /><meta name="citation_reference" content="citation_title=Genetic control of microsatellite stability; Mutat.Res.;  citation_year=1997;  citation_volume=383;  citation_pages=61-70; " /><meta name="citation_reference" content="citation_title=Base composition bias might result from competition for metabolic resources; Trends Genet.;  citation_year=2002;  citation_volume=18;  citation_pages=291-294; " /><meta name="citation_reference" content="citation_title=Occurrence of potential cruciform and H-DNA forming sequences in genomic DNA; Nucleic Acids Res.;  citation_year=1995;  citation_volume=23;  citation_pages=1977-1983; " /><meta name="citation_reference" content="citation_title=Searching genomes for sequences with the potential to form intrastrand triple helices; J. Mol. Biol.;  citation_year=2000;  citation_volume=302;  citation_pages=797-809; " /><meta name="citation_reference" content="citation_title=A triplex-mediated knot between separated polypurine-polypyrimidine tracts in circular DNA blocks transcription by Escherichia coli RNA polymerase; DNA Cell Biol.;  citation_year=2000;  citation_volume=19;  citation_pages=235-341; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/12/4/221/1719082/dsi009.pdf" /><meta name="description" content="Abstract.  The increasing availability of prokaryotic genome sequences has shown that simple sequence repeats (SSRs) are widespread in prokaryotes and that ther" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/12/4/221/621324" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/12/4/221/621324" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/12/4/221/621324");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsi009';
    </script>
<title>
	Characterization of Mononucleotide Repeats in Sequenced Prokaryotic Genomes | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./621324" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BbloPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJDb2VueWUsIFRvbSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkxhYm9yYXRvcml1bSB2b29yIE1pY3JvYmlvbG9naWUsIFVuaXZlcnNpdGVpdCBHZW50Sy5MLiBMZWRlZ2FuY2tzdHJhYXQgMzUsIEItOTAwMCBHZW50LCBCZWxnaXVtIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iVmFuZGFtbWUsIFBldGVyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTGFib3JhdG9yaXVtIHZvb3IgTWljcm9iaW9sb2dpZSwgVW5pdmVyc2l0ZWl0IEdlbnRLLkwuIExlZGVnYW5ja3N0cmFhdCAzNSwgQi05MDAwIEdlbnQsIEJlbGdpdW0iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IkNoYXJhY3Rlcml6YXRpb24gb2YgTW9ub251Y2xlb3RpZGUgUmVwZWF0cyBpbiBTZXF1ZW5jZWQgUHJva2FyeW90aWMgR2Vub21lcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjIyMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iMjMzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHNpMDA5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjEyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSI0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMDUvMDEvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TaG9ydC1zZXF1ZW5jZSBETkEgcmVwZWF0cyBpbiBwcm9rYXJ5b3RpYyBnZW5vbWVzOyBNaWNyb2Jpb2wuIE1vbC4gQmlvbC4gUmV2LjsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT02MjsgIGNpdGF0aW9uX3BhZ2VzPTI3NS0yOTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRhbmRlbSByZXBlYXRzIGluIGNvbXBsZXRlIGJhY3RlcmlhbCBnZW5vbWVzIHNlcXVlbmNlcy4gU2VxdWVuY2UgYW5kIHN0cnVjdHVyYWwgYW5hbHlzZXMgZm9yIGNvbXBhcmF0aXZlIHN0dWRpZXM7IFJlcy4gTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT0xNTA7ICBjaXRhdGlvbl9wYWdlcz03NDUtNzU0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TbGlwcGVkLXN0cmFuZCBtaXNwYWlyaW5nOiBhIG1ham9yIG1lY2hhbmlzbSBmb3IgRE5BIHNlcXVlbmNlIGV2b2x1dGlvbjsgTW9sLiBCaW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MTk4NzsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9MjAzLTIyMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9T2NjdXJyZW5jZSBhbmQgc3RydWN0dXJl4oCTZnVuY3Rpb24gcmVsYXRpb25zaGlwIG9mIHBlbnRhbWVyaWMgc2hvcnQgc2VxdWVuY2UgcmVwZWF0cyBpbiBtaWNyb2JpYWwgZ2Vub21lczsgUmVzLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTE1MDsgIGNpdGF0aW9uX3BhZ2VzPTYxNy02MjY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUluc2VydGlvbnMsIGRlbGV0aW9ucywgYW5kIHRoZSBvcmlnaW4gb2YgbWljcm9zYXRlbGxpdGVzOyBHZW5ldC4gUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT03NjsgIGNpdGF0aW9uX3BhZ2VzPTIyNy0yMzY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVR3byBkaXN0aW5jdCBtb2RlcyBvZiBtaWNyb3NhdGVsbGl0ZSBtdXRhdGlvbiBwcm9jZXNzZXM6IEV2aWRlbmNlIGZyb20gdGhlIGNvbXBsZXRlIGdlbm9taWMgc2VxdWVuY2VzIG9mIG5pbmUgc3BlY2llczsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz0yMjQyLTIyNTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByb21vdGVycyByZXNwb25zaXZlIHRvIEROQSBiZW5kaW5nOiBhIGNvbW1vbiB0aGVtZSBpbiBwcm9rYXJ5b3RpYyBnZW5lIGV4cHJlc3Npb247IE1pY3JvYmlvbC4gUmV2LjsgIGNpdGF0aW9uX3llYXI9MTk5NDsgIGNpdGF0aW9uX3ZvbHVtZT01ODsgIGNpdGF0aW9uX3BhZ2VzPTI2OC0yOTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zY3JpcHRpb24gdGhyb3VnaCBhIHNpbXBsZSBETkEgcmVwZWF0IGJsb2NrcyByZXBsaWNhdGlvbiBlbG9uZ2F0aW9uOyBFTUJPIEouOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTE3OyAgY2l0YXRpb25fcGFnZXM9NTA5NS01MTAyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Nb2xlY3VsYXIgc3dpdGNoZXPigJR0aGUgT04gYW5kIE9GRiBvZiBiYWN0ZXJpYWwgcGhhc2UgdmFyaWF0aW9uOyBNb2wuIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9MzM7ICBjaXRhdGlvbl9wYWdlcz05MTktOTMyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1PdmVyLXJlcHJlc2VudGF0aW9uIG9mIHJlcGVhdHMgaW4gc3RyZXNzIHJlc3BvbnNlIGdlbmVzOmEgc3RyYXRlZ3kgdG8gaW5jcmVhc2UgdmVyc2F0aWxpdHkgdW5kZXIgc3RyZXNzZnVsIGNvbmRpdGlvbnM/OyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MzA7ICBjaXRhdGlvbl9wYWdlcz0xODg2LTE4OTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUluc3RhYmlsaXR5IG9mIHJlcGV0aXRpdmUgRE5BIHNlcXVlbmNlczogVGhlIHJvbGUgb2YgcmVwbGljYXRpb24gaW4gbXVsdGlwbGUgbWVjaGFuaXNtczsgUHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT05ODsgIGNpdGF0aW9uX3BhZ2VzPTgzMTktODMyNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW4gYXBwcmFpc2FsIG9mIHRoZSBwb3RlbnRpYWwgZm9yIGlsbGVnaXRpbWF0ZSByZWNvbWJpbmF0aW9uIGluIGJhY3RlcmlhbCBnZW5vbWVzIGFuZCBpdHMgY29uc2VxdWVuY2VzOiBmcm9tIGR1cGxpY2F0aW9ucyB0byBnZW5vbWUgcmVkdWN0aW9uOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTExMjMtMTEzMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGh5bG9nZW5ldGljIGFzc2lnbm1lbnQgb2YgbGVuZ3RoIHZhcmlhdGlvbiBhdCBhIG1pY3Jvc2F0ZWxsaXRlIGxvY3VzOyBQcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTk0OyAgY2l0YXRpb25fcGFnZXM9MTA3NDUtMTA3NDk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFzc29jaWF0aW9uIG9mIGNocm9tb3NvbWFsIFJOQSB3aXRoIHJlcGV0aXRpdmUgRE5BOyBQcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MTk3MTsgIGNpdGF0aW9uX3ZvbHVtZT02ODsgIGNpdGF0aW9uX3BhZ2VzPTM4Ny0zODk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRhbmRlbSBpbnZlcnRlZCByZXBlYXRzIGluIG1pdG9jaG9uZHJpYWwgRE5BIG9mIHBldGl0ZSBtdXRhbnRzIG9mIFNhY2NoYXJvbXljZXMgY2VyZXZpc2lhZTsgUHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTE5NzQ7ICBjaXRhdGlvbl92b2x1bWU9NzE7ICBjaXRhdGlvbl9wYWdlcz0xMzY2LTEzNzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgbm92ZWwgcmVwZWF0ZWQgZWxlbWVudCB3aXRoIFotRE5BLWZvcm1pbmcgcG90ZW50aWFsIGlzIHdpZGVseSBmb3VuZCBpbiBldm9sdXRpb25hcmlseSBkaXZlcnNlIGV1a2FyeW90aWMgZ2Vub21lczsgUHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MTk4MjsgIGNpdGF0aW9uX3ZvbHVtZT03OTsgIGNpdGF0aW9uX3BhZ2VzPTY0NjUtNjQ2OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlmZmVyZW50aWFsIGRpc3RyaWJ1dGlvbiBvZiBzaW1wbGUgc2VxdWVuY2UgcmVwZWF0cyBpbiBldWthcnlvdGljIGdlbm9tZSBzZXF1ZW5jZXM7IE1vbC4gQmlvbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MTg7ICBjaXRhdGlvbl9wYWdlcz0xMTYxLTExNjc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUh5cGVydmFyaWFiaWxpdHkgb2Ygc2ltcGxlIHNlcXVlbmNlcyBhcyBhIGdlbmVyYWwgc291cmNlIGZvciBwb2x5bW9ycGhpYyBETkEgbWFya2VyczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTg5OyAgY2l0YXRpb25fdm9sdW1lPTE3OyAgY2l0YXRpb25fcGFnZXM9NjQ2My02NDcxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TaW1wbGUgc2VxdWVuY2VzOyBDdXJyLiBPcGluLiBHZW5ldC4gRGV2LjsgIGNpdGF0aW9uX3llYXI9MTk5NDsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9ODMyLTgzNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2hhcmFjdGVyaXN0aWMgZW5yaWNobWVudCBvZiBETkEgcmVwZWF0cyBpbiBkaWZmZXJlbnQgZ2Vub21lczsgUHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT05NDsgIGNpdGF0aW9uX3BhZ2VzPTUyMzctNTI0MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWJ1bmRhbnQgbWljcm9zYXRlbGxpdGUgcG9seW1vcnBoaXNtcyBpbiBTYWNjaGFyb215Y2VzIGNlcmV2aXNpYWUsIGFuZCB0aGUgZGlmZmVyZW50IGRpc3RyaWJ1dGlvbnMgb2YgbWljcm9zYXRlbGxpdGVzIGluIGVpZ2h0IHByb2thcnlvdGVzIGFuZCBTLiBjZXJldmlzaWFlLCByZXN1bHQgZnJvbSBzdHJvbmcgbXV0YXRpb24gcHJlc3N1cmVzIGFuZCBhIHZhcmlldHkgb2Ygc2VsZWN0aXZlIGZvcmNlczsgUHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT05NTsgIGNpdGF0aW9uX3BhZ2VzPTE2NDctMTY1MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q3J5cHRpYyBzaW1wbGljaXR5IGlzIGEgbWFqb3Igc291cmNlIG9mIGdlbmV0aWMgdmFyaWF0aW9uOyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTE5ODY7ICBjaXRhdGlvbl92b2x1bWU9MzIyOyAgY2l0YXRpb25fcGFnZXM9NjUyLTY1NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlzdGluY3QgZnJlcXVlbmN5LWRpc3RyaWJ1dGlvbnMgb2YgaG9tb3BvbHltZXJpYyBETkEgdHJhY3RzIGluIGRpZmZlcmVudCBnZW5vbWVzOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9MjY7ICBjaXRhdGlvbl9wYWdlcz00MDU2LTQwNjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNpbXBsZSBzZXF1ZW5jZSByZXBlYXRzIGluIEVzY2hlcmljaGlhIGNvbGk6IGFidW5kYW5jZSwgZGlzdHJpYnV0aW9uLCBjb21wb3NpdGlvbiwgYW5kIHBvbHltb3JwaGlzbTsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz02Mi03MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2ltcGxlIHNlcXVlbmNlIHJlcGVhdHMgYW5kIGNvbXBvc2l0aW9uYWwgYmlhcyBpbiB0aGUgYmlwYXJ0aXRlIFJhbHN0b25pYSBzb2xhbmFjZWFydW0gR01JMTAwMCBnZW5vbWU7IEJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9MTAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JZGVudGlmaWNhdGlvbiBhbmQgY2hhcmFjdGVyaXNhdGlvbiBvZiBzaW1wbGUgc2VxdWVuY2UgcmVwZWF0cyBpbiB0aGUgZ2Vub21lcyBvZiBTaGlnZWxsYSBzcGVjaWVzOyBHZW5lOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTMyMjsgIGNpdGF0aW9uX3BhZ2VzPTg1LTkyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgbnVtYmVyIG9mIHJlcGVhdHMgZXhwZWN0ZWQgaW4gcmFuZG9tIG51Y2xlaWMgYWNpZCBzZXF1ZW5jZXMgYW5kIGZvdW5kIGluIGdlbmVzOyBKLiBUaGVvci4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5ODE7ICBjaXRhdGlvbl92b2x1bWU9OTE7ICBjaXRhdGlvbl9wYWdlcz03MS05ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXJ0ZW1pcyBTZXF1ZW5jZSB2aXN1YWxpc2F0aW9uIGFuZCBhbm5vdGF0aW9uOyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTk0NC05NDU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmUgZHVwbGljYXRpb24gYW5kIGJpYXNlZCBmdW5jdGlvbmFsIHJldGVudGlvbiBvZiBwYXJhbG9ncyBpbiBiYWN0ZXJpYWwgZ2Vub21lczsgVHJlbmRzIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz0xNDgtMTU0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CaWFzZWQgYmlvbG9naWNhbCBmdW5jdGlvbnMgb2YgaG9yaXpvbnRhbGx5IHRyYW5zZmVycmVkIGdlbmVzIGluIHByb2thcnlvdGljIGdlbm9tZXM7IE5hdHVyZSBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MzY7ICBjaXRhdGlvbl9wYWdlcz03NjAtNzY2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wYXJhdGl2ZSBnZW5vbWljIHN0cnVjdHVyZSBvZiBwcm9rYXJ5b3RlczsgQW5udS4gUmV2LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9Mzg7ICBjaXRhdGlvbl9wYWdlcz03NzEtNzkxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUgdXBkYXRlOiBDb3JyZWxhdGlvbiBvZiBiYWN0ZXJpYWwgZ2Vub21pYyBwcm9wZXJ0aWVzOyBNaWNyb2Jpb2xvZ3k7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTUwOyAgY2l0YXRpb25fcGFnZXM9Mzg5OS0zOTAzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QaGFzZSBhbmQgYW50aWdlbmljIHZhcmlhdGlvbiBpbiBiYWN0ZXJpYTsgQ2xpbi4gTWljcm9iaW9sLiBSZXYuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTE3OyAgY2l0YXRpb25fcGFnZXM9NTgxLTYxMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2ltcGxlIHNlcXVlbmNlIHJlcGVhdHMgaW4gdGhlIEhlbGljb2JhY3RlciBweWxvcmkgZ2Vub21lOyBNb2wuIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9Mjc7ICBjaXRhdGlvbl9wYWdlcz0xMDkxLTEwOTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBkaXZlcnNpdHkgd2l0aGluIGFuIGV4cGFuZGVkIGFuZCByZWRlZmluZWQgcmVwZXJ0b2lyZSBvZiBwaGFzZS12YXJpYWJsZSBnZW5lcyBpbiBIZWxpY29iYWN0ZXIgcHlsb3JpOyBNaWNyb2Jpb2xvZ3k7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTUwOyAgY2l0YXRpb25fcGFnZXM9ODE3LTgzMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGVjaXBoZXJpbmcgQ2FtcHlsb2JhY3RlciBqZWp1bmkgY2VsbCBzdXJmYWNlIGludGVyYWN0aW9ucyBmcm9tIHRoZSBnZW5vbWUgc2VxdWVuY2U7IEN1cnIuIE9waW4uIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTM1LTQwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUgZGV0ZXJpb3JhdGlvbjogbG9zcyBvZiByZXBlYXRlZCBzZXF1ZW5jZXMgYW5kIGFjY3VtdWxhdGlvbiBvZiBqdW5rIEROQTsgR2VuZXRpY2E7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MTE1OyAgY2l0YXRpb25fcGFnZXM9MS0xMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9sZWN1bGFyIGJpb2xvZ3kgb2YgbXljb3BsYXNtYXM7IEFubnUuIFJldi4gTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5NjsgIGNpdGF0aW9uX3ZvbHVtZT01MDsgIGNpdGF0aW9uX3BhZ2VzPTI1LTU3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUgZXZvbHV0aW9uIGluIGJhY3RlcmlhbCBlbmRvc3ltYmlvbnRzIG9mIGluc2VjdHM7IE5hdHVyZSBSZXYuIEdlbi47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MzsgIGNpdGF0aW9uX3BhZ2VzPTg1MC04NjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1vbGVjdWxhciBiYXNpcyBvZiBnZW5ldGljIGluc3RhYmlsaXR5IG9mIHRyaXBsZXQgcmVwZWF0czsgSi4gQmlvbC4gQ2hlbS47ICBjaXRhdGlvbl95ZWFyPTE5OTY7ICBjaXRhdGlvbl92b2x1bWU9MjcxOyAgY2l0YXRpb25fcGFnZXM9Mjg3NS0yODc4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IeXBlcm11dGFiaWxpdHkgb2YgaG9tb251Y2xlb3RpZGUgcnVucyBpbiBtaXNtYXRjaCByZXBhaXIgYW5kIEROQSBwb2x5bWVyYXNlIHByb29mcmVhZGluZyB5ZWFzdCBtdXRhbnRzOyBNb2wuIENlbGwuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTE3OyAgY2l0YXRpb25fcGFnZXM9Mjg1OS0yODY1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TaW1wbGUgc2VxdWVuY2UgcmVwZWF0cyBhcyBhIHNvdXJjZSBvZiBxdWFudGl0YXRpdmUgZ2VuZXRpYyB2YXJpYXRpb247IFRyZW5kcyBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz03NC03ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBETkEgc3RydWN0dXJhbCBhdGxhcyBmb3IgRXNjaGVyaWNoaWEgY29saTsgSi4gTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0yOTk7ICBjaXRhdGlvbl9wYWdlcz05MDctOTMwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CaWFzIG9mIHB1cmluZSBzdHJldGNoZXMgaW4gc2VxdWVuY2VkIGNocm9tb3NvbWVzOyBDb21wLiBDaGVtLjsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0yNjsgIGNpdGF0aW9uX3BhZ2VzPTUzMS01NDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhvdyB0aGUgZ2Vub21lIHJlYWRpZXMgaXRzZWxmIGZvciBldm9sdXRpb247IFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9MjgxOyAgY2l0YXRpb25fcGFnZXM9MTEzMS0xMTM0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Fdm9sdXRpb25hcnkgdHVuaW5nIGtub2I7IEVuZGVhdm91cjsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTM2LTQwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbmRpcmVjdCBzZWxlY3Rpb24gb24gdGhlIG11dGF0aW9uYWwgbGFuZHNjYXBlOiBBbiBldm9sdmVkIHJvbGUgZm9yIHRoZSBtdXRhYmlsaXR5IG9mIHJlcGV0aXRpdmUgRE5BPzsgY2l0YXRpb25fdGV4dD1Fdm9sdXRpb24gMjAwMCwgSW5kaWFuYSBVbml2ZXJzaXR5IENvbmZlcmVuY2VzLCBCbG9vbWluZ3RvbiwgSU47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFtZWxpb3JhdGlvbiBvZiBiYWN0ZXJpYWwgZ2Vub21lczogUmF0ZXMgb2YgY2hhbmdlIGFuZCBleGNoYW5nZTsgSi4gTW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT00NDsgIGNpdGF0aW9uX3BhZ2VzPTM4My0zOTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmV0aWMgY29udHJvbCBvZiBtaWNyb3NhdGVsbGl0ZSBzdGFiaWxpdHk7IE11dGF0LlJlcy47ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9MzgzOyAgY2l0YXRpb25fcGFnZXM9NjEtNzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJhc2UgY29tcG9zaXRpb24gYmlhcyBtaWdodCByZXN1bHQgZnJvbSBjb21wZXRpdGlvbiBmb3IgbWV0YWJvbGljIHJlc291cmNlczsgVHJlbmRzIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0xODsgIGNpdGF0aW9uX3BhZ2VzPTI5MS0yOTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU9jY3VycmVuY2Ugb2YgcG90ZW50aWFsIGNydWNpZm9ybSBhbmQgSC1ETkEgZm9ybWluZyBzZXF1ZW5jZXMgaW4gZ2Vub21pYyBETkE7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5NTsgIGNpdGF0aW9uX3ZvbHVtZT0yMzsgIGNpdGF0aW9uX3BhZ2VzPTE5NzctMTk4MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2VhcmNoaW5nIGdlbm9tZXMgZm9yIHNlcXVlbmNlcyB3aXRoIHRoZSBwb3RlbnRpYWwgdG8gZm9ybSBpbnRyYXN0cmFuZCB0cmlwbGUgaGVsaWNlczsgSi4gTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0zMDI7ICBjaXRhdGlvbl9wYWdlcz03OTctODA5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHRyaXBsZXgtbWVkaWF0ZWQga25vdCBiZXR3ZWVuIHNlcGFyYXRlZCBwb2x5cHVyaW5lLXBvbHlweXJpbWlkaW5lIHRyYWN0cyBpbiBjaXJjdWxhciBETkEgYmxvY2tzIHRyYW5zY3JpcHRpb24gYnkgRXNjaGVyaWNoaWEgY29saSBSTkEgcG9seW1lcmFzZTsgRE5BIENlbGwgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MTk7ICBjaXRhdGlvbl9wYWdlcz0yMzUtMzQxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMTIvNC8yMjEvMTcxOTA4Mi9kc2kwMDkucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIFRoZSBpbmNyZWFzaW5nIGF2YWlsYWJpbGl0eSBvZiBwcm9rYXJ5b3RpYyBnZW5vbWUgc2VxdWVuY2VzIGhhcyBzaG93biB0aGF0IHNpbXBsZSBzZXF1ZW5jZSByZXBlYXRzIChTU1JzKSBhcmUgd2lkZXNwcmVhZCBpbiBwcm9rYXJ5b3RlcyBhbmQgdGhhdCB0aGVyIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFci8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj02MjEzMjQmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2RkfqgvmvAui62Y0hrgTQ8zgj8cusg=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f12%2f4%2f221%2f621324" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f12%2f4%2f221%2f621324" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=25342" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/12/4"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/12/4/0/m_dnaresearch12_4.cover.gif?Expires=2147483647&amp;Signature=3uNJBvpswUl8ckNzhRD6LiG0MySGXqGkcTQYLlRIvfld1eLLpkZoI4Ma5qe4h0TQIrhjk1-qwhKlhwYLpQpMu3ntM421fXasS8~G0V0MwH6cjjfM9fdJn0EH2kkIRqpR9dzDT~TKvzm8K-R~6ztHb07HPr6KoeUx6Rgs8s~OeOw2mLTVUNaBcNQn8k14DbREAYbOxque3El9Lplnm2ngB~AQ0rwKokc~WvC44rQAaAltU0RG~GjoIdSMmocCDEXHDpbpBtxuUw6A9Y5q2U4JLWP3XC47XRtYoHhAyJWDFgU30hB9Q-HhZ4XJLURWeo2hPNBzP0pbMYzBduIAxMvdMQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/12/4">
                        <div class="volume trailing-comma">Volume 12</div>
                        <div class="issue">Issue 4</div>

            </a>

        </div>
        <div class="ii-pub-date">
2005        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="10424224">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#10424224">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="10424226">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#10424226">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="10424231">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#10424231">2. Material and Methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="10424242">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#10424242">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="10424291">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#10424291">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="10424307">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#10424307">Supplementary Data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="10424315">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#10424315">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="empty arrow">&nbsp;</li>
            <li class="next arrow">
            <a href="/dnaresearch/article/12/4/235/621215">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Characterization of Mononucleotide Repeats in Sequenced Prokaryotic Genomes
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tom Coenye</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">Laboratorium voor Microbiologie, Universiteit Gent</div><div class="addr-line">K.L. Ledeganckstraat 35, B-9000 Gent, Belgium</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <sup>*</sup>To whom correspondence should be addressed. Tel. +32-9-2645128, Fax. +32-9-2645092, Email: <a href="mailto:Tom.Coenye@UGent.be" target="_blank">Tom.Coenye@UGent.be</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tom+Coenye">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Coenye T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Coenye T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Tom Coenye</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Peter Vandamme</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">Laboratorium voor Microbiologie, Universiteit Gent</div><div class="addr-line">K.L. Ledeganckstraat 35, B-9000 Gent, Belgium</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Peter+Vandamme">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Vandamme P">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Vandamme P&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Peter Vandamme</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Communicated by Naotaka Ogasawara</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 12, Issue 4, 1 January 2005, Pages 221–233, <a href='https://doi.org/10.1093/dnares/dsi009'>https://doi.org/10.1093/dnares/dsi009</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">01 January 2005</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">17 January 2005</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Revision Received:</div>
                                <div class="wi-date">07 July 2005</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="621324" href="/dnaresearch/article-pdf/12/4/221/1719082/dsi009.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Tom Coenye, Peter Vandamme;  Characterization of Mononucleotide Repeats in Sequenced Prokaryotic Genomes, <em>DNA Research</em>, Volume 12, Issue 4, 1 January 2005, Pages 221–233, <a href='https://doi.org/10.1093/dnares/dsi009'>https://doi.org/10.1093/dnares/dsi009</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/621324?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/621324?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/621324?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/621324?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/621324?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/621324?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/621324?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Characterization%20of%20Mononucleotide%20Repeats%20in%20Sequenced%20Prokaryotic%20Genomes&amp;publicationDate=2005-01-01&amp;volumeNum=12&amp;issueNum=4&amp;author=Coenye%2C%20Tom%3B%20Vandamme%2C%20Peter&amp;startPage=221&amp;endPage=233&amp;contentId=10.1093%2Fdnares%2Fdsi009&amp;oa=&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f12%2f4%2f221%2f621324" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=10424224 id="10424224" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>The increasing availability of prokaryotic genome sequences has shown that simple sequence repeats (SSRs) are widespread in prokaryotes and that there is extensive variation in their length, number and distribution. Considering their potential importance in generating genomic diversity, we determined the distribution of a specific group of SSRs, mononucleotide repeats of size between 5 and 13 nt, in 157 sequenced prokaryotic genomes. The data obtained in the present study show that (i) a large number of mononucleotide SSRs is present in all prokaryotic genomes investigated, (ii) shorter repeats are much more abundant than longer repeats, and (iii) in the majority of the genomes, longer mononucleotide SSRs are excluded from coding regions although we identified several organisms where mononucleotide SSRs are not excluded from the coding regions. We also observed that some genomes contain more mononucleotide SSRs than expected, while others contain significantly less. Bacterial genomes that contain much less mononucleotide SSRs than expected are generally larger and more GC-rich, while bacterial genomes that contain much more mononucleotide SSRs than expected are in general smaller and more AT-rich. Finally, we also noted that genomes that contain a high fraction of horizontally transferred genes have a lower mononucleotide SSR density and that A and T are generally overrepresented in mononucleotide SSRs.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;mononucleotide repeats&quot;">mononucleotide repeats</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;simple sequence repeats&quot;">simple sequence repeats</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;comparative genomics&quot;">comparative genomics</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genome evolution&quot;">genome evolution</a></div>                            <h2 scrollto-destination=10424226 id="10424226" class="section-title" >1. Introduction</h2>
<p>Repetitive DNA consists of homopolymeric tracts of single nucleotides or of small or large numbers of multimeric classes of repeats. These can either be homogeneous (i.e. built from identical units) or heterogeneous (i.e. built from mixed units).<a reveal-id="b1" data-open="b1" class="link link-ref link-reveal xref-bibr">1</a> A special category of repeats are tandem repeats, which are made up of monomeric sequences of variable length, repeated periodically, with contiguous monomers arranged in a head-to-tail fashion.<a reveal-id="b2" data-open="b2" class="link link-ref link-reveal xref-bibr">2</a> The variability in repeat number of these small tandem repeats (also called simple sequence repeats or SSRs) is caused by slipped-strand mispairing, in which the tertiary structure of the repetitive DNA allows mismatching of the neighbouring repeats and repeats can then be inserted or deleted during DNA polymerase-mediated DNA duplication.<a reveal-id="b1" data-open="b1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="b3" data-open="b3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="b4" data-open="b4" class="link link-ref link-reveal xref-bibr">4</a> Recently, an additional process, called indel slippage, was proposed.<a reveal-id="b5" data-open="b5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="b6" data-open="b6" class="link link-ref link-reveal xref-bibr">6</a> Unlike DNA replication slippage, this process is independent of the length of the repeat array and is thought to operate at repeats with few repetitions.</p><p>Several functional roles have been proposed for SSRs. First of all, they are thought to be involved in various mechanisms of regulation of gene expression.<a reveal-id="b1" data-open="b1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="b2" data-open="b2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="b4" data-open="b4" class="link link-ref link-reveal xref-bibr">4</a> It is thought that variation in SSR numbers can alter the spacing between structurally important domains (like the −35 and −10 promoter regions) and in this way can affect promoter strength.<a reveal-id="b2" data-open="b2" class="link link-ref link-reveal xref-bibr">2</a> Differences in bending of the DNA molecule caused by unusual repeat structures can also affect promoter strength.<a reveal-id="b7" data-open="b7" class="link link-ref link-reveal xref-bibr">7</a> Another example is the blocking of DNA replication elongation by transcription through SSRs.<a reveal-id="b8" data-open="b8" class="link link-ref link-reveal xref-bibr">8</a> Second, the loss or gain of repeats can affect the integrity of ORFs (open reading frames) and can result in phase variation, in which shifting in and out of frame (relative to the start of translation) leads to on–off switching of the associated gene products.<a reveal-id="b9" data-open="b9" class="link link-ref link-reveal xref-bibr">9</a> The observation of a high number of short close repeats in 296 <em>Escherichia coli</em> genes related to repair, recombination and physiological adaptations to different stresses suggests that these repeats may reflect a strategy to cope with stress.<a reveal-id="b10" data-open="b10" class="link link-ref link-reveal xref-bibr">10</a> In addition, rearrangements between tandemly repeated sequences are a major source of genomic change as they can result in deletion or duplication of the DNA flanked by the repeated sequences<a reveal-id="b11" data-open="b11" class="link link-ref link-reveal xref-bibr">11</a> and it has been shown that genomes containing a higher repeat density have higher rates of rearrangements leading to an accelerated loss of gene order.<a reveal-id="b12" data-open="b12" class="link link-ref link-reveal xref-bibr">12</a> Finally, it has been shown that regions bordering SSR loci are more susceptible to mutagenic events.<a reveal-id="b13" data-open="b13" class="link link-ref link-reveal xref-bibr">13</a></p><p>It has been known for over 30 years that eukaryotic genomes contain a significant fraction of repeated sequences<a reveal-id="b14  b17" data-open="b14  b17" class="link link-ref link-reveal xref-bibr">14–17</a> and SSRs have also been known to exist in prokaryotes for quite some time.<a reveal-id="b3" data-open="b3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="b18  b21" data-open="b18  b21" class="link link-ref link-reveal xref-bibr">18–21</a> Early analysis<a reveal-id="b22" data-open="b22" class="link link-ref link-reveal xref-bibr">22</a> showed that nearly all SSRs were considerably overrepresented in genomic sequence fragments compared with that in their randomized counterparts; it also appeared that they were most frequent in eukaryotes and more rare in prokaryotes (probably due to more economical use of DNA, i.e. a higher coding density, in the latter). This was later confirmed by more in-depth analyses.<a reveal-id="b20" data-open="b20" class="link link-ref link-reveal xref-bibr">20</a>,<a reveal-id="b21" data-open="b21" class="link link-ref link-reveal xref-bibr">21</a>,<a reveal-id="b23" data-open="b23" class="link link-ref link-reveal xref-bibr">23</a></p><p>The increasing availability of prokaryotic genome sequences has shown that SSRs are also widespread in prokaryotes and that there is extensive variation in their length, number and distribution, in for example <em>Escherichia coli</em>, <em>Shigella</em> sp. and <em>Ralstonia solanacearum</em>.<a reveal-id="b20" data-open="b20" class="link link-ref link-reveal xref-bibr">20</a>,<a reveal-id="b21" data-open="b21" class="link link-ref link-reveal xref-bibr">21</a>,<a reveal-id="b24  b26" data-open="b24  b26" class="link link-ref link-reveal xref-bibr">24–26</a> However, to date there are no studies which have systematically addressed and compared the distribution of SSRs in a large number of bacterial and/or archaeal genomes. Considering the potential importance of SSRs in generating genomic diversity and regulating gene expression, we determined the distribution of a specific group of SSRs, mononucleotide repeats (i.e. sequences consisting of a single repeated nucleotide) of size between 5 and 13 nt, in 157 sequenced prokaryotic genomes.</p>                            <h2 scrollto-destination=10424231 id="10424231" class="section-title" >2. Material and Methods</h2>
                            <h3 scrollto-destination=10424232 id="10424232" class="section-title" >2.1. DNA sequences</h3>
<p>We included the sequences of 157 prokaryotic genomes in the present study. These included 19 archaeal and 138 bacterial genomes (<a reveal-id="tbl1" data-open="tbl1" class="link link-reveal link-table xref-fig">Tables 1</a> and <a reveal-id="tbl2" data-open="tbl2" class="link link-reveal link-table xref-fig">2</a>). Most were downloaded from the GenBank database, but we also included a few unpublished genomes that were produced by the Pathogen Sequencing Group at the Sanger Institute (<a class="link link-uri" href="http://www.sanger.ac.uk/Projects/Microbes/" target="_blank"></a>) or the University of Oklahoma (<a class="link link-uri" href="http://www.genome.ou.edu/gono.html" target="_blank"></a>).</p>                        <a id="10424234" scrollto-destination="10424234"></a>
<div content-id="tbl1" class="table-modal"><div class="table-wrap"><div id="tbl1" class="table-wrap-title" data-id="tbl1"><div class="label -label"><strong>Table 1</strong></div><div class="caption"><p>Overview of the archaeal genomes compared in the present study, the density of mononucleotide SSRs in the genome, the number of mononucleotide SSRs observed (O), the number of mononucleotide SSRs expected (based on base composition) (E) and the ratio O/E.</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Strain designation </th><th>Accession no. </th><th>mol% GC </th><th>Genome size (bp) </th><th>Repeat density </th><th>Observed no. of repeats </th><th>Expected no. of repeats </th><th>O/E </th></tr></thead><tbody><tr><td><em>Aeropyrum pernix</em> Kl </td><td>NC_000854 </td><td char=".">56.3 </td><td>1 669 695 </td><td char=".">0.0249 </td><td>7861 </td><td>5541 </td><td char=".">1.42 </td></tr><tr><td><em>Archaeglobus fulgidus</em> DSM 4304 </td><td>NC_000917 </td><td char=".">48.6 </td><td>2 178 400 </td><td char=".">0.0268 </td><td>11 058 </td><td>6424 </td><td char=".">1.72 </td></tr><tr><td><em>Halobacterium</em> sp. NRC-1 </td><td>NC_002607 </td><td char=".">67.9 </td><td>2 014 239 </td><td char=".">0.0114 </td><td>4431 </td><td>12 360 </td><td char=".">0.36 </td></tr><tr><td><em>Methanocaldococcus jannaschii</em> DSM 2661 </td><td>NC_000909 </td><td char=".">31.4 </td><td>1 664 970 </td><td char=".">0.0600 </td><td>18 306 </td><td>10 654 </td><td char=".">1.72 </td></tr><tr><td><em>Methanococcus maripaludis</em> S2 </td><td>NC_005791 </td><td char=".">33.1 </td><td>1 661 137 </td><td char=".">0.0741 </td><td>22 237 </td><td>9603 </td><td char=".">2.32 </td></tr><tr><td><em>Methanopyrus kandleri</em> AVI 9 </td><td>NC_003551 </td><td char=".">61.2 </td><td>1 694 969 </td><td char=".">0.0115 </td><td>3797 </td><td>7063 </td><td char=".">0.54 </td></tr><tr><td><em>Methanosarcina</em> acetivorans C2A </td><td>NC_003552 </td><td char=".">42.7 </td><td>5 751 492 </td><td char=".">0.0455 </td><td>48 027 </td><td>19 856 </td><td char=".">2.42 </td></tr><tr><td><em>Methanosarcina mazei</em> Goel </td><td>NC_003901 </td><td char=".">41.5 </td><td>4 096 345 </td><td char=".">0.0476 </td><td>35 828 </td><td>14 908 </td><td char=".">2.40 </td></tr><tr><td><em>Methanothermobacter thermoautotrophicus</em> DeltaH </td><td>NC_000916 </td><td char=".">49.5 </td><td>1 751 377 </td><td char=".">0.0226 </td><td>7449 </td><td>5135 </td><td char=".">1.45 </td></tr><tr><td><em>Nanoarchaeum equitans</em> Kin4-M </td><td>NC_005213 </td><td char=".">31.6 </td><td>490 885 </td><td char=".">0.0638 </td><td>5779 </td><td>3104 </td><td char=".">1.86 </td></tr><tr><td><em>Picrophilus torridus</em> DSM 9790 </td><td>NC_005877 </td><td char=".">36.0 </td><td>1 545 895 </td><td char=".">0.0432 </td><td>12 314 </td><td>7533 </td><td char=".">1.63 </td></tr><tr><td><em>Pyrobaculum aerophilum</em> IM2 </td><td>NC_003364 </td><td char=".">51.4 </td><td>2 222 430 </td><td char=".">0.0340 </td><td>14 301 </td><td>6554 </td><td char=".">2.18 </td></tr><tr><td><em>Pyrococcus abyssi</em> GE5 </td><td>NC_003413 </td><td char=".">40.8 </td><td>1 908 256 </td><td char=".">0.0315 </td><td>11 433 </td><td>7178 </td><td char=".">1.59 </td></tr><tr><td><em>Pyrococcus horikoshii</em> OT3 </td><td>NC_000961 </td><td char=".">41.9 </td><td>1 738 505 </td><td char=".">0.0245 </td><td>8148 </td><td>6213 </td><td char=".">1.31 </td></tr><tr><td><em>Sulfolobus solfataricus</em> P2 </td><td>NC_002754 </td><td char=".">35.8 </td><td>2 992 245 </td><td char=".">0.0274 </td><td>15 539 </td><td>14 751 </td><td char=".">1.05 </td></tr><tr><td><em>Sulfolobus tokodaii</em> 7 </td><td>NC_003106 </td><td char=".">32.8 </td><td>2 694 765 </td><td char=".">0.0373 </td><td>18 933 </td><td>15 859 </td><td char=".">1.19 </td></tr><tr><td><em>Thermoplasma acidophilum</em> DSM 1728 </td><td>NC_002578 </td><td char=".">46.0 </td><td>1 564 906 </td><td char=".">0.0158 </td><td>4660 </td><td>4829 </td><td char=".">0.96 </td></tr><tr><td><em>Thermoplasma volcanium</em> GSS1 </td><td>NC_002578 </td><td char=".">39.9 </td><td>1 584 804 </td><td char=".">0.0273 </td><td>8029 </td><td>6234 </td><td char=".">1.29 </td></tr></tbody></table></div><div content-id="tbl1" class="table-modal"><table><thead name="thead" align=""><tr><th>Strain designation </th><th>Accession no. </th><th>mol% GC </th><th>Genome size (bp) </th><th>Repeat density </th><th>Observed no. of repeats </th><th>Expected no. of repeats </th><th>O/E </th></tr></thead><tbody><tr><td><em>Aeropyrum pernix</em> Kl </td><td>NC_000854 </td><td char=".">56.3 </td><td>1 669 695 </td><td char=".">0.0249 </td><td>7861 </td><td>5541 </td><td char=".">1.42 </td></tr><tr><td><em>Archaeglobus fulgidus</em> DSM 4304 </td><td>NC_000917 </td><td char=".">48.6 </td><td>2 178 400 </td><td char=".">0.0268 </td><td>11 058 </td><td>6424 </td><td char=".">1.72 </td></tr><tr><td><em>Halobacterium</em> sp. NRC-1 </td><td>NC_002607 </td><td char=".">67.9 </td><td>2 014 239 </td><td char=".">0.0114 </td><td>4431 </td><td>12 360 </td><td char=".">0.36 </td></tr><tr><td><em>Methanocaldococcus jannaschii</em> DSM 2661 </td><td>NC_000909 </td><td char=".">31.4 </td><td>1 664 970 </td><td char=".">0.0600 </td><td>18 306 </td><td>10 654 </td><td char=".">1.72 </td></tr><tr><td><em>Methanococcus maripaludis</em> S2 </td><td>NC_005791 </td><td char=".">33.1 </td><td>1 661 137 </td><td char=".">0.0741 </td><td>22 237 </td><td>9603 </td><td char=".">2.32 </td></tr><tr><td><em>Methanopyrus kandleri</em> AVI 9 </td><td>NC_003551 </td><td char=".">61.2 </td><td>1 694 969 </td><td char=".">0.0115 </td><td>3797 </td><td>7063 </td><td char=".">0.54 </td></tr><tr><td><em>Methanosarcina</em> acetivorans C2A </td><td>NC_003552 </td><td char=".">42.7 </td><td>5 751 492 </td><td char=".">0.0455 </td><td>48 027 </td><td>19 856 </td><td char=".">2.42 </td></tr><tr><td><em>Methanosarcina mazei</em> Goel </td><td>NC_003901 </td><td char=".">41.5 </td><td>4 096 345 </td><td char=".">0.0476 </td><td>35 828 </td><td>14 908 </td><td char=".">2.40 </td></tr><tr><td><em>Methanothermobacter thermoautotrophicus</em> DeltaH </td><td>NC_000916 </td><td char=".">49.5 </td><td>1 751 377 </td><td char=".">0.0226 </td><td>7449 </td><td>5135 </td><td char=".">1.45 </td></tr><tr><td><em>Nanoarchaeum equitans</em> Kin4-M </td><td>NC_005213 </td><td char=".">31.6 </td><td>490 885 </td><td char=".">0.0638 </td><td>5779 </td><td>3104 </td><td char=".">1.86 </td></tr><tr><td><em>Picrophilus torridus</em> DSM 9790 </td><td>NC_005877 </td><td char=".">36.0 </td><td>1 545 895 </td><td char=".">0.0432 </td><td>12 314 </td><td>7533 </td><td char=".">1.63 </td></tr><tr><td><em>Pyrobaculum aerophilum</em> IM2 </td><td>NC_003364 </td><td char=".">51.4 </td><td>2 222 430 </td><td char=".">0.0340 </td><td>14 301 </td><td>6554 </td><td char=".">2.18 </td></tr><tr><td><em>Pyrococcus abyssi</em> GE5 </td><td>NC_003413 </td><td char=".">40.8 </td><td>1 908 256 </td><td char=".">0.0315 </td><td>11 433 </td><td>7178 </td><td char=".">1.59 </td></tr><tr><td><em>Pyrococcus horikoshii</em> OT3 </td><td>NC_000961 </td><td char=".">41.9 </td><td>1 738 505 </td><td char=".">0.0245 </td><td>8148 </td><td>6213 </td><td char=".">1.31 </td></tr><tr><td><em>Sulfolobus solfataricus</em> P2 </td><td>NC_002754 </td><td char=".">35.8 </td><td>2 992 245 </td><td char=".">0.0274 </td><td>15 539 </td><td>14 751 </td><td char=".">1.05 </td></tr><tr><td><em>Sulfolobus tokodaii</em> 7 </td><td>NC_003106 </td><td char=".">32.8 </td><td>2 694 765 </td><td char=".">0.0373 </td><td>18 933 </td><td>15 859 </td><td char=".">1.19 </td></tr><tr><td><em>Thermoplasma acidophilum</em> DSM 1728 </td><td>NC_002578 </td><td char=".">46.0 </td><td>1 564 906 </td><td char=".">0.0158 </td><td>4660 </td><td>4829 </td><td char=".">0.96 </td></tr><tr><td><em>Thermoplasma volcanium</em> GSS1 </td><td>NC_002578 </td><td char=".">39.9 </td><td>1 584 804 </td><td char=".">0.0273 </td><td>8029 </td><td>6234 </td><td char=".">1.29 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Repeat density is shown as the percentage of the genome that is located in mononucleotide repeats &gt;5 nt.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/10424234" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="tbl1" class="table-wrap-title" data-id="tbl1"><div class="label -label"><strong>Table 1</strong></div><div class="caption"><p>Overview of the archaeal genomes compared in the present study, the density of mononucleotide SSRs in the genome, the number of mononucleotide SSRs observed (O), the number of mononucleotide SSRs expected (based on base composition) (E) and the ratio O/E.</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Strain designation </th><th>Accession no. </th><th>mol% GC </th><th>Genome size (bp) </th><th>Repeat density </th><th>Observed no. of repeats </th><th>Expected no. of repeats </th><th>O/E </th></tr></thead><tbody><tr><td><em>Aeropyrum pernix</em> Kl </td><td>NC_000854 </td><td char=".">56.3 </td><td>1 669 695 </td><td char=".">0.0249 </td><td>7861 </td><td>5541 </td><td char=".">1.42 </td></tr><tr><td><em>Archaeglobus fulgidus</em> DSM 4304 </td><td>NC_000917 </td><td char=".">48.6 </td><td>2 178 400 </td><td char=".">0.0268 </td><td>11 058 </td><td>6424 </td><td char=".">1.72 </td></tr><tr><td><em>Halobacterium</em> sp. NRC-1 </td><td>NC_002607 </td><td char=".">67.9 </td><td>2 014 239 </td><td char=".">0.0114 </td><td>4431 </td><td>12 360 </td><td char=".">0.36 </td></tr><tr><td><em>Methanocaldococcus jannaschii</em> DSM 2661 </td><td>NC_000909 </td><td char=".">31.4 </td><td>1 664 970 </td><td char=".">0.0600 </td><td>18 306 </td><td>10 654 </td><td char=".">1.72 </td></tr><tr><td><em>Methanococcus maripaludis</em> S2 </td><td>NC_005791 </td><td char=".">33.1 </td><td>1 661 137 </td><td char=".">0.0741 </td><td>22 237 </td><td>9603 </td><td char=".">2.32 </td></tr><tr><td><em>Methanopyrus kandleri</em> AVI 9 </td><td>NC_003551 </td><td char=".">61.2 </td><td>1 694 969 </td><td char=".">0.0115 </td><td>3797 </td><td>7063 </td><td char=".">0.54 </td></tr><tr><td><em>Methanosarcina</em> acetivorans C2A </td><td>NC_003552 </td><td char=".">42.7 </td><td>5 751 492 </td><td char=".">0.0455 </td><td>48 027 </td><td>19 856 </td><td char=".">2.42 </td></tr><tr><td><em>Methanosarcina mazei</em> Goel </td><td>NC_003901 </td><td char=".">41.5 </td><td>4 096 345 </td><td char=".">0.0476 </td><td>35 828 </td><td>14 908 </td><td char=".">2.40 </td></tr><tr><td><em>Methanothermobacter thermoautotrophicus</em> DeltaH </td><td>NC_000916 </td><td char=".">49.5 </td><td>1 751 377 </td><td char=".">0.0226 </td><td>7449 </td><td>5135 </td><td char=".">1.45 </td></tr><tr><td><em>Nanoarchaeum equitans</em> Kin4-M </td><td>NC_005213 </td><td char=".">31.6 </td><td>490 885 </td><td char=".">0.0638 </td><td>5779 </td><td>3104 </td><td char=".">1.86 </td></tr><tr><td><em>Picrophilus torridus</em> DSM 9790 </td><td>NC_005877 </td><td char=".">36.0 </td><td>1 545 895 </td><td char=".">0.0432 </td><td>12 314 </td><td>7533 </td><td char=".">1.63 </td></tr><tr><td><em>Pyrobaculum aerophilum</em> IM2 </td><td>NC_003364 </td><td char=".">51.4 </td><td>2 222 430 </td><td char=".">0.0340 </td><td>14 301 </td><td>6554 </td><td char=".">2.18 </td></tr><tr><td><em>Pyrococcus abyssi</em> GE5 </td><td>NC_003413 </td><td char=".">40.8 </td><td>1 908 256 </td><td char=".">0.0315 </td><td>11 433 </td><td>7178 </td><td char=".">1.59 </td></tr><tr><td><em>Pyrococcus horikoshii</em> OT3 </td><td>NC_000961 </td><td char=".">41.9 </td><td>1 738 505 </td><td char=".">0.0245 </td><td>8148 </td><td>6213 </td><td char=".">1.31 </td></tr><tr><td><em>Sulfolobus solfataricus</em> P2 </td><td>NC_002754 </td><td char=".">35.8 </td><td>2 992 245 </td><td char=".">0.0274 </td><td>15 539 </td><td>14 751 </td><td char=".">1.05 </td></tr><tr><td><em>Sulfolobus tokodaii</em> 7 </td><td>NC_003106 </td><td char=".">32.8 </td><td>2 694 765 </td><td char=".">0.0373 </td><td>18 933 </td><td>15 859 </td><td char=".">1.19 </td></tr><tr><td><em>Thermoplasma acidophilum</em> DSM 1728 </td><td>NC_002578 </td><td char=".">46.0 </td><td>1 564 906 </td><td char=".">0.0158 </td><td>4660 </td><td>4829 </td><td char=".">0.96 </td></tr><tr><td><em>Thermoplasma volcanium</em> GSS1 </td><td>NC_002578 </td><td char=".">39.9 </td><td>1 584 804 </td><td char=".">0.0273 </td><td>8029 </td><td>6234 </td><td char=".">1.29 </td></tr></tbody></table></div><div content-id="tbl1" class="table-modal"><table><thead name="thead" align=""><tr><th>Strain designation </th><th>Accession no. </th><th>mol% GC </th><th>Genome size (bp) </th><th>Repeat density </th><th>Observed no. of repeats </th><th>Expected no. of repeats </th><th>O/E </th></tr></thead><tbody><tr><td><em>Aeropyrum pernix</em> Kl </td><td>NC_000854 </td><td char=".">56.3 </td><td>1 669 695 </td><td char=".">0.0249 </td><td>7861 </td><td>5541 </td><td char=".">1.42 </td></tr><tr><td><em>Archaeglobus fulgidus</em> DSM 4304 </td><td>NC_000917 </td><td char=".">48.6 </td><td>2 178 400 </td><td char=".">0.0268 </td><td>11 058 </td><td>6424 </td><td char=".">1.72 </td></tr><tr><td><em>Halobacterium</em> sp. NRC-1 </td><td>NC_002607 </td><td char=".">67.9 </td><td>2 014 239 </td><td char=".">0.0114 </td><td>4431 </td><td>12 360 </td><td char=".">0.36 </td></tr><tr><td><em>Methanocaldococcus jannaschii</em> DSM 2661 </td><td>NC_000909 </td><td char=".">31.4 </td><td>1 664 970 </td><td char=".">0.0600 </td><td>18 306 </td><td>10 654 </td><td char=".">1.72 </td></tr><tr><td><em>Methanococcus maripaludis</em> S2 </td><td>NC_005791 </td><td char=".">33.1 </td><td>1 661 137 </td><td char=".">0.0741 </td><td>22 237 </td><td>9603 </td><td char=".">2.32 </td></tr><tr><td><em>Methanopyrus kandleri</em> AVI 9 </td><td>NC_003551 </td><td char=".">61.2 </td><td>1 694 969 </td><td char=".">0.0115 </td><td>3797 </td><td>7063 </td><td char=".">0.54 </td></tr><tr><td><em>Methanosarcina</em> acetivorans C2A </td><td>NC_003552 </td><td char=".">42.7 </td><td>5 751 492 </td><td char=".">0.0455 </td><td>48 027 </td><td>19 856 </td><td char=".">2.42 </td></tr><tr><td><em>Methanosarcina mazei</em> Goel </td><td>NC_003901 </td><td char=".">41.5 </td><td>4 096 345 </td><td char=".">0.0476 </td><td>35 828 </td><td>14 908 </td><td char=".">2.40 </td></tr><tr><td><em>Methanothermobacter thermoautotrophicus</em> DeltaH </td><td>NC_000916 </td><td char=".">49.5 </td><td>1 751 377 </td><td char=".">0.0226 </td><td>7449 </td><td>5135 </td><td char=".">1.45 </td></tr><tr><td><em>Nanoarchaeum equitans</em> Kin4-M </td><td>NC_005213 </td><td char=".">31.6 </td><td>490 885 </td><td char=".">0.0638 </td><td>5779 </td><td>3104 </td><td char=".">1.86 </td></tr><tr><td><em>Picrophilus torridus</em> DSM 9790 </td><td>NC_005877 </td><td char=".">36.0 </td><td>1 545 895 </td><td char=".">0.0432 </td><td>12 314 </td><td>7533 </td><td char=".">1.63 </td></tr><tr><td><em>Pyrobaculum aerophilum</em> IM2 </td><td>NC_003364 </td><td char=".">51.4 </td><td>2 222 430 </td><td char=".">0.0340 </td><td>14 301 </td><td>6554 </td><td char=".">2.18 </td></tr><tr><td><em>Pyrococcus abyssi</em> GE5 </td><td>NC_003413 </td><td char=".">40.8 </td><td>1 908 256 </td><td char=".">0.0315 </td><td>11 433 </td><td>7178 </td><td char=".">1.59 </td></tr><tr><td><em>Pyrococcus horikoshii</em> OT3 </td><td>NC_000961 </td><td char=".">41.9 </td><td>1 738 505 </td><td char=".">0.0245 </td><td>8148 </td><td>6213 </td><td char=".">1.31 </td></tr><tr><td><em>Sulfolobus solfataricus</em> P2 </td><td>NC_002754 </td><td char=".">35.8 </td><td>2 992 245 </td><td char=".">0.0274 </td><td>15 539 </td><td>14 751 </td><td char=".">1.05 </td></tr><tr><td><em>Sulfolobus tokodaii</em> 7 </td><td>NC_003106 </td><td char=".">32.8 </td><td>2 694 765 </td><td char=".">0.0373 </td><td>18 933 </td><td>15 859 </td><td char=".">1.19 </td></tr><tr><td><em>Thermoplasma acidophilum</em> DSM 1728 </td><td>NC_002578 </td><td char=".">46.0 </td><td>1 564 906 </td><td char=".">0.0158 </td><td>4660 </td><td>4829 </td><td char=".">0.96 </td></tr><tr><td><em>Thermoplasma volcanium</em> GSS1 </td><td>NC_002578 </td><td char=".">39.9 </td><td>1 584 804 </td><td char=".">0.0273 </td><td>8029 </td><td>6234 </td><td char=".">1.29 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Repeat density is shown as the percentage of the genome that is located in mononucleotide repeats &gt;5 nt.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/10424234" target="_blank">View Large</a></div></div>                        <a id="10424235" scrollto-destination="10424235"></a>
<div content-id="tbl2" class="table-modal"><div class="table-wrap"><div id="tbl2" class="table-wrap-title" data-id="tbl2"><div class="label -label"><strong>Table 2</strong></div><div class="caption"><p>Overview of the bacterial genomes compared in the present study, the density of mononucleotide SSRs in the genome, the number of mononucleotide SSRs observed (O), the number of mononucleotide SSRs expected (based on base composition) (E) and the ratio O/E.</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Strain designation </th><th>Accession no. </th><th>mol% GC </th><th>Genome size (bp) </th><th>Repeat density </th><th>Observed no. of repeats </th><th>Expected (E) no. of repeats </th><th>O/E </th></tr></thead><tbody><tr><td><em>Acinetobacter</em> sp. ADP1 </td><td>NC_005966 </td><td>40.4 </td><td>3 598 621 </td><td char=".">0.0366 </td><td>24 075 </td><td>13 805 </td><td char=".">1.74 </td></tr><tr><td><em>Agrobacterium tumefaciens</em> C58 </td><td>NC_003304 </td><td>59.3 </td><td>4 917 050 </td><td char=".">0.0120 </td><td>11 180 </td><td>18 587 </td><td char=".">0.60 </td></tr><tr><td><em>Aquifex aeolicus</em> VF5 </td><td>NC_000918 </td><td>43.5 </td><td>1 551 335 </td><td char=".">0.0374 </td><td>10 932 </td><td>5187 </td><td char=".">2.11 </td></tr><tr><td><em>Bacillus anthracis Ames</em> </td><td>NC_003997 </td><td>35.4 </td><td>5 227 293 </td><td char=".">0.0454 </td><td>43 728 </td><td>26 377 </td><td char=".">1.66 </td></tr><tr><td><em>Bacillus cereus</em> ATCC 14579 </td><td>NC_004722 </td><td>35.3 </td><td>5 411 809 </td><td char=".">0.0455 </td><td>45 299 </td><td>27 468 </td><td char=".">1.65 </td></tr><tr><td><em>Bacillus halodurans</em> C-125 </td><td>NC_002570 </td><td>43.7 </td><td>4 202 353 </td><td char=".">0.0367 </td><td>28 385 </td><td>13 945 </td><td char=".">2.04 </td></tr><tr><td><em>Bacillus licheniformis</em> ATCC 14580 </td><td>NC_006270 </td><td>46.2 </td><td>4 222 336 </td><td char=".">0.0438 </td><td>33 815 </td><td>12 966 </td><td char=".">2.61 </td></tr><tr><td><em>Bacillus subtilis</em> 168 </td><td>NC_000964 </td><td>43.5 </td><td>4 214 630 </td><td char=".">0.0423 </td><td>32 621 </td><td>14 092 </td><td char=".">2.31 </td></tr><tr><td><em>Bacillus thuringiensis</em> 97-27 </td><td>NC_005957 </td><td>35.4 </td><td>5 327 682 </td><td char=".">0.0450 </td><td>44 116 </td><td>26 884 </td><td char=".">1.64 </td></tr><tr><td><em>Bacteriovorax marinus</em> </td><td>Unpublished<a reveal-id="tf2-1" data-open="tf2-1" class="link link-ref link-reveal xref-fn">a</a> </td><td>36.7 </td><td>3 435 933 </td><td char=".">0.0278 </td><td>17 861 </td><td>16 084 </td><td char=".">1.11 </td></tr><tr><td><em>Bacteroides fragilis</em> NCTC 9343 </td><td>Unpublished </td><td>43.1 </td><td>5 205 140 </td><td char=".">0.0269 </td><td>25 896 </td><td>17 679 </td><td char=".">1.46 </td></tr><tr><td><em>Bacteroides thetaiotamicron</em> VPI-5482 </td><td>NC_004663 </td><td>42.8 </td><td>6 260 361 </td><td char=".">0.0237 </td><td>27 662 </td><td>21 523 </td><td char=".">1.29 </td></tr><tr><td><em>Bartonella henselae</em> Houston- 1 </td><td>NC_005956 </td><td>38.2 </td><td>1 931 047 </td><td char=".">0.0568 </td><td>19 800 </td><td>8312 </td><td char=".">2.38 </td></tr><tr><td><em>Bartonella quintana</em> Toulose </td><td>NC_005955 </td><td>38.8 </td><td>1 581 384 </td><td char=".">0.0545 </td><td>15 587 </td><td>6589 </td><td char=".">2.37 </td></tr><tr><td><em>Bdellovibrio bacteriovorus</em> HD100 </td><td>NC_005363 </td><td>50.6 </td><td>3 782 950 </td><td char=".">0.0298 </td><td>21 025 </td><td>11 096 </td><td char=".">1.89 </td></tr><tr><td><em>Bifidobacterium longum</em> NCC 2705 </td><td>NC_004307 </td><td>60.1 </td><td>2 256 646 </td><td char=".">0.0064 </td><td>2761 </td><td>8877 </td><td char=".">0.31 </td></tr><tr><td><em>Blochmannia floridanus</em> </td><td>NC_005061 </td><td>27.4 </td><td>705 557 </td><td char=".">0.0595 </td><td>7353 </td><td>5724 </td><td char=".">1.28 </td></tr><tr><td><em>Bordetella avium</em> </td><td>Unpublished </td><td>69.2 </td><td>3 732 255 </td><td char=".">0.0144 </td><td>9986 </td><td>24 753 </td><td char=".">0.40 </td></tr><tr><td><em>Bordetella bronchiseptica</em> RB50 </td><td>NC_002927 </td><td>68.1 </td><td>5 339 179 </td><td char=".">0.0070 </td><td>7041 </td><td>33 158 </td><td char=".">0.21 </td></tr><tr><td><em>Bordetella parapertussis</em> 12822 </td><td>NC_002928 </td><td>68.1 </td><td>4 773 551 </td><td char=".">0.0068 </td><td>6134 </td><td>29 645 </td><td char=".">0.21 </td></tr><tr><td><em>Bordetella pertussis</em> Tohamal </td><td>NC_002929 </td><td>67.7 </td><td>4 086 189 </td><td char=".">0.0074 </td><td>5684 </td><td>24 777 </td><td char=".">0.23 </td></tr><tr><td><em>Borrelia burgdorferi</em> B3 1 </td><td>NC_001318 </td><td>28.6 </td><td>910 724 </td><td char=".">0.0922 </td><td>14 886 </td><td>6884 </td><td char=".">2.16 </td></tr><tr><td><em>Borrelia garinii</em> </td><td>NC_006156 </td><td>28.3 </td><td>904 246 </td><td char=".">0.0915 </td><td>14 614 </td><td>6957 </td><td char=".">2.10 </td></tr><tr><td><em>Bradyrhizobium japonicum</em> USDA 110 </td><td>NC_004463 </td><td>64.1 </td><td>9 105 828 </td><td char=".">0.0060 </td><td>10 568 </td><td>44 631 </td><td char=".">0.24 </td></tr><tr><td><em>Brucella melitensis</em> 16M </td><td>NC_003317 </td><td>57.2 </td><td>3 294 931 </td><td char=".">0.0132 </td><td>8324 </td><td>11 328 </td><td char=".">0.73 </td></tr><tr><td><em>Brucella suis</em> 1330 </td><td>NC_004310 </td><td>57.2 </td><td>3 315 173 </td><td char=".">0.0132 </td><td>8337 </td><td>11 398 </td><td char=".">0.73 </td></tr><tr><td><em>Buchnera aphidicola</em> Bp </td><td>NC_004545 </td><td>25.3 </td><td>615 980 </td><td char=".">0.0946 </td><td>10 043 </td><td>5644 </td><td char=".">1.78 </td></tr><tr><td><em>Burkholderia cenocepacia</em> J2315 </td><td>Unpublished </td><td>67.0 </td><td>7 963 121 </td><td char=".">0.0051 </td><td>7659 </td><td>46 309 </td><td char=".">0.17 </td></tr><tr><td><em>Burkholderia mallei</em> ATCC 23344 </td><td>NC_006349 </td><td>68.3 </td><td>5 835 527 </td><td char=".">0.0056 </td><td>6154 </td><td>36 676 </td><td char=".">0.17 </td></tr><tr><td><em>Burkholderia pseudomallei</em> K96243 </td><td>NC_006350 </td><td>68.1 </td><td>7 247 547 </td><td char=".">0.0058 </td><td>7897 </td><td>45 009 </td><td char=".">0.18 </td></tr><tr><td><em>Campylobacter jejuni</em> NCTC 11168 </td><td>NC_002163 </td><td>30.5 </td><td>1 641 481 </td><td char=".">0.0898 </td><td>26 642 </td><td>11 083 </td><td char=".">2.40 </td></tr><tr><td><em>Caulobacter crescentus</em> CB15 </td><td>NC_002996 </td><td>67.2 </td><td>4 016 947 </td><td char=".">0.0086 </td><td>6681 </td><td>23 641 </td><td char=".">0.28 </td></tr><tr><td><em>Chlamydia muridarum</em> MoPn </td><td>NC_002620 </td><td>40.3 </td><td>1 072 950 </td><td char=".">0.0460 </td><td>8978 </td><td>4137 </td><td char=".">2.17 </td></tr><tr><td><em>Chlamydia trachomatis</em> D/UW-3/CX </td><td>NC_000117 </td><td>41.3 </td><td>1 072 950 </td><td char=".">0.0383 </td><td>7473 </td><td>3941 </td><td char=".">1.90 </td></tr><tr><td><em>Chlamydophila abortus</em> S26/3 </td><td>Unpublished </td><td>39.9 </td><td>1 144 377 </td><td char=".">0.0408 </td><td>8530 </td><td>4502 </td><td char=".">1.89 </td></tr><tr><td><em>Chlamydophila caviae</em> GPIC </td><td>NC_003361 </td><td>39.2 </td><td>1 173 390 </td><td char=".">0.0405 </td><td>8712 </td><td>4786 </td><td char=".">1.82 </td></tr><tr><td><em>Chlamydophila pneumoniae</em> AR39 </td><td>NC_002179 </td><td>40.6 </td><td>1 229 853 </td><td char=".">0.0400 </td><td>8972 </td><td>4672 </td><td char=".">1.92 </td></tr><tr><td><em>Chlorobium tepidum</em> TLS </td><td>NC_002932 </td><td>56.5 </td><td>2 154 946 </td><td char=".">0.0213 </td><td>8449 </td><td>7205 </td><td char=".">1.17 </td></tr><tr><td><em>Chromobacterium violaceum</em> ATCC 12472 </td><td>NC_005085 </td><td>64.8 </td><td>4 751 080 </td><td char=".">0.0088 </td><td>7808 </td><td>24 256 </td><td char=".">0.32 </td></tr><tr><td><em>Clostridium acetobutylicum</em> ATCC 824 </td><td>NC_003030 </td><td>30.9 </td><td>3 940 880 </td><td char=".">0.0542 </td><td>38 709 </td><td>25 981 </td><td char=".">1.49 </td></tr><tr><td><em>Clostridium botulinum</em> ATCC 3502 </td><td>Unpublished </td><td>28.2 </td><td>3 886 916 </td><td char=".">0.0639 </td><td>45 111 </td><td>30 083 </td><td char=".">1.50 </td></tr><tr><td><em>Clostridium difficile</em> 630 </td><td>Unpublished </td><td>29.1 </td><td>4 290 252 </td><td char=".">0.0570 </td><td>44 286 </td><td>31 485 </td><td char=".">1.41 </td></tr><tr><td><em>Clostridium perfringens</em> 13 </td><td>NC_003366 </td><td>28.6 </td><td>3 031 430 </td><td char=".">0.0558 </td><td>31 018 </td><td>22 915 </td><td char=".">1.35 </td></tr><tr><td><em>Clostridium tetani</em> E88 </td><td>NC_004557 </td><td>28.7 </td><td>2 799 251 </td><td char=".">0.0612 </td><td>31 149 </td><td>21 035 </td><td char=".">1.48 </td></tr><tr><td><em>Corynebacterium diphtheriae</em> NCTC 13129 </td><td>NC_002935 </td><td>53.5 </td><td>2 488 635 </td><td char=".">0.0171 </td><td>8230 </td><td>7589 </td><td char=".">1.08 </td></tr><tr><td><em>Corynebacterium efficiens</em> YS-314 </td><td>NC_004369 </td><td>63.1 </td><td>3 147 090 </td><td char=".">0.0150 </td><td>9230 </td><td>14 565 </td><td char=".">0.63 </td></tr><tr><td><em>Corynebacterium glutamicum</em> ATCC 13032 </td><td>NC_003450 </td><td>53.8 </td><td>3 309 401 </td><td char=".">0.0142 </td><td>9101 </td><td>10 163 </td><td char=".">0.90 </td></tr><tr><td><em>Coxiella burnettii</em> RSA 493 </td><td>NC_002971 </td><td>42.7 </td><td>1 995 275 </td><td char=".">0.0510 </td><td>18 486 </td><td>6888 </td><td char=".">2.68 </td></tr><tr><td><em>Deinococcus radiodurans</em> Rl </td><td>NC_001263 </td><td>66.9 </td><td>3 060 986 </td><td char=".">0.0202 </td><td>11 805 </td><td>17 695 </td><td char=".">0.67 </td></tr><tr><td><em>Desulfotalea psychrophila</em> LSv54 </td><td>NC_006138 </td><td>46.8 </td><td>3 523 383 </td><td char=".">0.0396 </td><td>25 157 </td><td>10 675 </td><td char=".">2.36 </td></tr><tr><td><em>Desulfovibrio vulgaris</em> Hildenborough </td><td>NC_002937 </td><td>63.1 </td><td>3 570 858 </td><td char=".">0.0156 </td><td>10 447 </td><td>16 526 </td><td char=".">0.63 </td></tr><tr><td><em>Enterococcus faecalis</em> V583 </td><td>NC_004668 </td><td>37.5 </td><td>3 218 031 </td><td char=".">0.0547 </td><td>31 958 </td><td>14 398 </td><td char=".">2.22 </td></tr><tr><td><em>Erwinia carotovora</em> SCR11043 </td><td>NC_004547 </td><td>51 </td><td>5 064 019 </td><td char=".">0.0213 </td><td>20 025 </td><td>14 885 </td><td char=".">1.35 </td></tr><tr><td><em>Escherichia coli</em> K12 </td><td>NC_000913 </td><td>50.8 </td><td>4 639 221 </td><td char=".">0.0222 </td><td>19 237 </td><td>13 620 </td><td char=".">1.41 </td></tr><tr><td><em>Fusobacterium nucleatum</em> ATCC 25586 </td><td>NC_003454 </td><td>27.1 </td><td>2 174 500 </td><td char=".">0.0809 </td><td>31 627 </td><td>17 952 </td><td char=".">1.76 </td></tr><tr><td><em>Geobacter sulfurreducens</em> PCA </td><td>NC_002939 </td><td>60.9 </td><td>3 814 139 </td><td char=".">0.0235 </td><td>16 741 </td><td>15 641 </td><td char=".">1.07 </td></tr><tr><td><em>Gloeobacter violaceus</em> PCC7421 </td><td>NC_005125 </td><td>62 </td><td>4 659 019 </td><td char=".">0.0238 </td><td>20 847 </td><td>20 275 </td><td char=".">1.03 </td></tr><tr><td><em>Haemophilus ducreyi</em> 35000HP </td><td>NC_002940 </td><td>38.2 </td><td>1 698 955 </td><td char=".">0.0410 </td><td>12 790 </td><td>7313 </td><td char=".">1.75 </td></tr><tr><td><em>Haemophilus influenzae</em> Rd KW20 </td><td>NC_000907 </td><td>38.1 </td><td>1 830 138 </td><td char=".">0.0459 </td><td>15 499 </td><td>7921 </td><td char=".">1.96 </td></tr><tr><td><em>Helicobacter hepaticus</em> ATCC 51449 </td><td>NC_004917 </td><td>35.9 </td><td>1 799 146 </td><td char=".">0.0546 </td><td>17 933 </td><td>8818 </td><td char=".">2.03 </td></tr><tr><td><em>Helicobacter pylori</em> 26695 </td><td>NC_000915 </td><td>38.9 </td><td>1 667 867 </td><td char=".">0.0793 </td><td>24 106 </td><td>6913 </td><td char=".">3.49 </td></tr><tr><td><em>Helicobacter pylori</em> J99 </td><td>NC_000921 </td><td>39.2 </td><td>1 643 831 </td><td char=".">0.0785 </td><td>23 551 </td><td>6705 </td><td char=".">3.51 </td></tr><tr><td><em>Lactobacillus johnsonii</em> NCC 533 </td><td>NC_005362 </td><td>34.6 </td><td>1 992 676 </td><td char=".">0.0450 </td><td>16 570 </td><td>10 538 </td><td char=".">1.57 </td></tr><tr><td><em>Lactobacillus plantarum</em> WCFS1 </td><td>NC_004567 </td><td>44.5 </td><td>3 308 274 </td><td char=".">0.0227 </td><td>14 021 </td><td>10 672 </td><td char=".">1.31 </td></tr><tr><td><em>Lactococcus lactis</em> IL1403 </td><td>NC_002662 </td><td>35.3 </td><td>2 365 589 </td><td char=".">0.0573 </td><td>24 747 </td><td>12 007 </td><td char=".">2.06 </td></tr><tr><td><em>Leifsonia xyli</em> CTCB07 </td><td>NC_006087 </td><td>67.7 </td><td>2 584 158 </td><td char=".">0.0100 </td><td>4989 </td><td>15 669 </td><td char=".">0.32 </td></tr><tr><td><em>Leptospira interrogans</em> Fiocruz LI-130 </td><td>NC_004342 </td><td>35 </td><td>4 277 185 </td><td char=".">0.0755 </td><td>58 065 </td><td>22 094 </td><td char=".">2.63 </td></tr><tr><td><em>Listeria innocua</em> Clip 11262 </td><td>NC_003212 </td><td>37.4 </td><td>3 011 208 </td><td char=".">0.0513 </td><td>28 276 </td><td>13 549 </td><td char=".">2.09 </td></tr><tr><td><em>Listeria monocytogenes</em> EGD-e </td><td>NC_003210 </td><td>38 </td><td>2 944 528 </td><td char=".">0.0496 </td><td>26 798 </td><td>12 814 </td><td char=".">2.09 </td></tr><tr><td><em>Mesoplasma florum</em> LI </td><td>NC_006055 </td><td>27 </td><td>793 224 </td><td char=".">0.0738 </td><td>10 545 </td><td>6587 </td><td char=".">1.60 </td></tr><tr><td><em>Mesorhizobium loti</em> MAFF303099 </td><td>NC_002678 </td><td>62.7 </td><td>7 036 074 </td><td char=".">0.0070 </td><td>9388 </td><td>31 836 </td><td char=".">0.29 </td></tr><tr><td><em>Mycobacterium avium</em> k10 </td><td>NC_002944 </td><td>69.3 </td><td>4 829 781 </td><td char=".">0.0075 </td><td>7045 </td><td>32 224 </td><td char=".">0.22 </td></tr><tr><td><em>Mycobacterium bovis</em> AF2122/97 </td><td>NC_002945 </td><td>65.6 </td><td>4 345 492 </td><td char=".">0.0080 </td><td>6710 </td><td>23 254 </td><td char=".">0.29 </td></tr><tr><td><em>Mycobacterium leprae</em> TN </td><td>NC_002677 </td><td>57.8 </td><td>3 268 203 </td><td char=".">0.0095 </td><td>5925 </td><td>11 526 </td><td char=".">0.51 </td></tr><tr><td><em>Mycobacterium tuberculosis</em> H37Rv </td><td>NC_002755 </td><td>65.6 </td><td>4 411 532 </td><td char=".">0.0079 </td><td>6767 </td><td>23 608 </td><td char=".">0.29 </td></tr><tr><td><em>Mycoplasma gallisepticum</em> Rd KW20 </td><td>NC_004829 </td><td>31.5 </td><td>996 422 </td><td char=".">0.0518 </td><td>9347 </td><td>6338 </td><td char=".">1.47 </td></tr><tr><td><em>Mycoplasma genitalium</em> G-37 </td><td>NC_000908 </td><td>31.7 </td><td>580 074 </td><td char=".">0.0709 </td><td>7428 </td><td>3646 </td><td char=".">2.04 </td></tr><tr><td><em>Mycoplasma mobile</em> 163K </td><td>NC_006908 </td><td>25 </td><td>777 079 </td><td char=".">0.1034 </td><td>14 323 </td><td>7244 </td><td char=".">1.98 </td></tr><tr><td><em>Mycoplasma mycoides</em> PG1 </td><td>NC_005364 </td><td>24 </td><td>1 211 703 </td><td char=".">0.0914 </td><td>19 889 </td><td>11 956 </td><td char=".">1.66 </td></tr><tr><td><em>Mycoplasma penetrans</em> HF-2 </td><td>NC_004432 </td><td>25.7 </td><td>1 358 633 </td><td char=".">0.0764 </td><td>18 734 </td><td>12 166 </td><td char=".">1.54 </td></tr><tr><td><em>Mycoplasma pneumoniae</em> Ml29 </td><td>NC_000912 </td><td>40 </td><td>816 394 </td><td char=".">0.0432 </td><td>6584 </td><td>3195 </td><td char=".">2.06 </td></tr><tr><td><em>Mycoplasma pulmonis</em> UAB CTIP </td><td>NC_002771 </td><td>26.6 </td><td>963 879 </td><td char=".">0.1149 </td><td>19 740 </td><td>8193 </td><td char=".">2.41 </td></tr><tr><td><em>Neisseria gonorrhoeae</em> </td><td>Unpublished </td><td>52.7 </td><td>2 146 877 </td><td char=".">0.0326 </td><td>12 995 </td><td>6443 </td><td char=".">2.02 </td></tr><tr><td><em>Neisseria meningitidis</em> MC58 </td><td>NC_003112 </td><td>51.5 </td><td>2 272 351 </td><td char=".">0.0331 </td><td>14 001 </td><td>6707 </td><td char=".">2.09 </td></tr><tr><td><em>Neisseria meningitidis</em> Z2491 </td><td>NC_003116 </td><td>51.8 </td><td>2184 406 </td><td char=".">0.0328 </td><td>13 304 </td><td>6469 </td><td char=".">2.06 </td></tr><tr><td><em>Nitrosomonas europeae</em> ATCC 19718 </td><td>NC_004757 </td><td>50.7 </td><td>2 812094 </td><td char=".">0.0209 </td><td>10 934 </td><td>8252 </td><td char=".">1.33 </td></tr><tr><td><em>Nostoc</em> sp. PCC 7120 </td><td>NC_003272 </td><td>41.3 </td><td>6 413 771 </td><td char=".">0.0315 </td><td>37 738 </td><td>23 559 </td><td char=".">1.60 </td></tr><tr><td><em>Oceanobacillus iheyensis</em> HTE831 </td><td>NC_004193 </td><td>35.7 </td><td>3 630 528 </td><td char=".">0.0388 </td><td>25 862 </td><td>18 002 </td><td char=".">1.44 </td></tr><tr><td><em>Parachlamydia</em> sp. UWE25 </td><td>NC_005861 </td><td>34.7 </td><td>2 414 465 </td><td char=".">0.0649 </td><td>27 883 </td><td>12 694 </td><td char=".">2.20 </td></tr><tr><td><em>Pasteurella multocida</em> Pm70 </td><td>NC_002632 </td><td>40.4 </td><td>2 257 487 </td><td char=".">0.0417 </td><td>17 297 </td><td>8660 </td><td char=".">2.00 </td></tr><tr><td><em>Photorhabdus luminescens</em> TTO1 </td><td>NC_005126 </td><td>42.8 </td><td>5 688 987 </td><td char=".">0.0314 </td><td>32 937 </td><td>19 559 </td><td char=".">1.68 </td></tr><tr><td><em>Phytoplasma asteris</em> OY-M </td><td>NC_005303 </td><td>27.7 </td><td>860 631 </td><td char=".">0.1108 </td><td>16 757 </td><td>6860 </td><td char=".">2.44 </td></tr><tr><td><em>Pirelulla</em> sp. 1 </td><td>NC_005027 </td><td>55.4 </td><td>7 145 576 </td><td char=".">0.0101 </td><td>13 771 </td><td>22 974 </td><td char=".">0.60 </td></tr><tr><td><em>Porphyromonas gingivalis</em> W83 </td><td>NC_002950 </td><td>48.3 </td><td>2 343 476 </td><td char=".">0.0221 </td><td>9614 </td><td>6932 </td><td char=".">1.39 </td></tr><tr><td><em>Prochlorococcus marinus</em> MIT 9313 </td><td>NC_005071 </td><td>50.7 </td><td>2 410 873 </td><td char=".">0.0169 </td><td>7592 </td><td>7075 </td><td char=".">1.07 </td></tr><tr><td><em>Propionibacterium acnes</em> KPA171202 </td><td>NC_006085 </td><td>60 </td><td>2 560 265 </td><td char=".">0.0105 </td><td>5250 </td><td>10 021 </td><td char=".">0.52 </td></tr><tr><td><em>Pseudomonas aeruginosa</em> PAO1 </td><td>NC_002516 </td><td>66.6 </td><td>6 264 403 </td><td char=".">0.0063 </td><td>7472 </td><td>35 572 </td><td char=".">0.21 </td></tr><tr><td><em>Pseudomonas putida</em> KT2440 </td><td>NC_002947 </td><td>61.5 </td><td>6 181 863 </td><td char=".">0.0103 </td><td>12 079 </td><td>26 178 </td><td char=".">0.46 </td></tr><tr><td><em>Pseudomonas syringae</em> DC3000 </td><td>NC_004578 </td><td>58.4 </td><td>6 397 126 </td><td char=".">0.0125 </td><td>14 975 </td><td>23 174 </td><td char=".">0.65 </td></tr><tr><td><em>Ralstonia solanacearum</em> GMI1000 </td><td>NC_003295 </td><td>67 </td><td>5 810 922 </td><td char=".">0.0077 </td><td>8463 </td><td>33 793 </td><td char=".">0.25 </td></tr><tr><td><em>Rhodopseudomonas palustris</em> CGA009 </td><td>NC_005296 </td><td>65 </td><td>5 459 213 </td><td char=".">0.0043 </td><td>4511 </td><td>28 200 </td><td char=".">0.16 </td></tr><tr><td><em>Rickettsia conorii</em> Malish 7 </td><td>NC_003103 </td><td>32.4 </td><td>1 268 755 </td><td char=".">0.0525 </td><td>12 127 </td><td>7647 </td><td char=".">1.59 </td></tr><tr><td><em>Rickettsia prowazekii</em> Madrid E </td><td>NC_000963 </td><td>29 </td><td>1 111 523 </td><td char=".">0.0560 </td><td>11 282 </td><td>8206 </td><td char=".">1.37 </td></tr><tr><td><em>Rickettsia typhi</em> Wilmington </td><td>NC_006142 </td><td>28.3 </td><td>1 111 496 </td><td char=".">0.0552 </td><td>11 128 </td><td>8552 </td><td char=".">1.30 </td></tr><tr><td><em>Salmonella enterica</em> CT18 </td><td>NC_003198 </td><td>52.1 </td><td>4 809 037 </td><td char=".">0.0240 </td><td>21 454 </td><td>14 296 </td><td char=".">1.50 </td></tr><tr><td><em>Salmonella typhimurium</em> LT2 </td><td>NC_003197 </td><td>52.2 </td><td>4 857 432 </td><td char=".">0.0239 </td><td>21 602 </td><td>14 460 </td><td char=".">1.49 </td></tr><tr><td><em>Serratia marcescens</em> Dbl 1 </td><td>Unpublished </td><td>59.5 </td><td>5 113 802 </td><td char=".">0.0155 </td><td>14 776 </td><td>19 521 </td><td char=".">0.76 </td></tr><tr><td><em>Shewanella oneidensis</em> MR-1 </td><td>NC_004347 </td><td>46 </td><td>4 969 803 </td><td char=".">0.0249 </td><td>23 135 </td><td>15 337 </td><td char=".">1.51 </td></tr><tr><td><em>Shigella flexneri</em> 2457T </td><td>NC_004741 </td><td>50.9 </td><td>4 599 354 </td><td char=".">0.0222 </td><td>19 024 </td><td>13 511 </td><td char=".">1.41 </td></tr><tr><td><em>Sinorhizobium meliloti</em> 1021 </td><td>NC_003047 </td><td>62.7 </td><td>3 654 135 </td><td char=".">0.0075 </td><td>5228 </td><td>16 534 </td><td char=".">0.32 </td></tr><tr><td><em>Staphylococcus aureus</em> MW2 </td><td>NC_003923 </td><td>32.8 </td><td>2 820 462 </td><td char=".">0.0389 </td><td>20 991 </td><td>16 599 </td><td char=".">1.26 </td></tr><tr><td><em>Staphylococcus epidermidis</em> ATCC 12228 </td><td>NC_004461 </td><td>32.1 </td><td>2 499 279 </td><td char=".">0.0424 </td><td>19 544 </td><td>15 337 </td><td char=".">1.27 </td></tr><tr><td><em>Streptococcus agalactiae</em> 2603V/R </td><td>NC_004416 </td><td>35.7 </td><td>2 160 267 </td><td char=".">0.0446 </td><td>17 709 </td><td>10 712 </td><td char=".">1.65 </td></tr><tr><td><em>Streptococcus mutans</em> UA159 </td><td>NC_004350 </td><td>36.8 </td><td>2 030 921 </td><td char=".">0.0483 </td><td>18 109 </td><td>9453 </td><td char=".">1.92 </td></tr><tr><td><em>Streptococcus pneumoniae</em> TIGR4 </td><td>NC_003028 </td><td>39.7 </td><td>2 160 837 </td><td char=".">0.0374 </td><td>15 003 </td><td>8588 </td><td char=".">1.75 </td></tr><tr><td><em>Streptococcus pyogenes</em> MGAS315 </td><td>NC_004070 </td><td>38.6 </td><td>1 900 521 </td><td char=".">0.0453 </td><td>15 743 </td><td>8005 </td><td char=".">1.97 </td></tr><tr><td><em>Streptomyces avermitilis</em> MA-4680 </td><td>NC_003155 </td><td>70.7 </td><td>9 025 608 </td><td char=".">0.0125 </td><td>21 960 </td><td>65 454 </td><td char=".">0.34 </td></tr><tr><td><em>Streptomyces coelicolor</em> A3 (2) </td><td>NC_003888 </td><td>72.1 </td><td>8 667 507 </td><td char=".">0.0128 </td><td>21 627 </td><td>68 279 </td><td char=".">0.32 </td></tr><tr><td><em>Synechococcus</em> sp. WH8102 </td><td>NC_005070 </td><td>59.4 </td><td>2 434 428 </td><td char=".">0.0153 </td><td>7002 </td><td>9247 </td><td char=".">0.76 </td></tr><tr><td><em>Synechocystis</em> sp. PCC 6803 </td><td>NC_000911 </td><td>47.7 </td><td>3 573 470 </td><td char=".">0.0457 </td><td>30 442 </td><td>10 654 </td><td char=".">2.86 </td></tr><tr><td><em>Thermoanaerobacter tengcongensis</em> </td><td>NC_003869 </td><td>37.6 </td><td>2 689 445 </td><td char=".">0.0568 </td><td>28 261 </td><td>11 966 </td><td char=".">2.36 </td></tr><tr><td><em>Thermosynechococcus elongatus</em> BP-1 </td><td>NC_004113 </td><td>53.9 </td><td>2 593 857 </td><td char=".">0.0325 </td><td>15 850 </td><td>7985 </td><td char=".">1.99 </td></tr><tr><td><em>Thermotoga maritima</em> MSB 8 </td><td>NC_000853 </td><td>46.2 </td><td>1 860 725 </td><td char=".">0.0298 </td><td>10 394 </td><td>5714 </td><td char=".">1.82 </td></tr><tr><td><em>Thermus thermophilus</em> HB27 </td><td>NC_005835 </td><td>69.4 </td><td>1 894 877 </td><td char=".">0.0533 </td><td>18 935 </td><td>12 718 </td><td char=".">1.49 </td></tr><tr><td><em>Treponema denticola</em> ATCC 35405 </td><td>NC_002967 </td><td>37.9 </td><td>2 843 201 </td><td char=".">0.0737 </td><td>37 216 </td><td>12 442 </td><td char=".">2.99 </td></tr><tr><td><em>Treponema pallidum</em> Nichols </td><td>NC_000919 </td><td>52.8 </td><td>1 138 011 </td><td char=".">0.0320 </td><td>6669 </td><td>3421 </td><td char=".">1.95 </td></tr><tr><td><em>Tropheryma whipplei</em> TW08/27 </td><td>NC_004551 </td><td>46.3 </td><td>925 938 </td><td char=".">0.0286 </td><td>4942 </td><td>2837 </td><td char=".">1.74 </td></tr><tr><td><em>Ureaplasma parvum</em> ATCC 700970 </td><td>NC_002162 </td><td>25.5 </td><td>751 719 </td><td char=".">0.0833 </td><td>11 146 </td><td>6809 </td><td char=".">1.64 </td></tr><tr><td><em>Vibrio cholerae</em> Nl696l </td><td>NC_002505 </td><td>47.5 </td><td>4 033 464 </td><td char=".">0.0238 </td><td>17 969 </td><td>12 063 </td><td char=".">1.49 </td></tr><tr><td><em>Vibrio parahaemolyticus</em> RMID 2210633 </td><td>NC_004603 </td><td>45.4 </td><td>5 165 770 </td><td char=".">0.0228 </td><td>22 140 </td><td>16 203 </td><td char=".">1.37 </td></tr><tr><td><em>Vibrio vulnificus</em> CMCP6 </td><td>NC_004459 </td><td>46.7 </td><td>5 211 578 </td><td char=".">0.0240 </td><td>23 472 </td><td>15 823 </td><td char=".">1.48 </td></tr><tr><td><em>Wigglesworthia glossinidia</em> </td><td>NC_004344 </td><td>22.5 </td><td>697 721 </td><td char=".">0.1241 </td><td>14 619 </td><td>7488 </td><td char=".">1.95 </td></tr><tr><td><em>Wolinella succinogenes</em> DSM 1740 </td><td>NC_005090 </td><td>48.5 </td><td>2 110 355 </td><td char=".">0.0434 </td><td>17 143 </td><td>6229 </td><td char=".">2.75 </td></tr><tr><td><em>Xanthomonas axonopodis</em> 306 </td><td>NC_003919 </td><td>64.8 </td><td>5 175 554 </td><td char=".">0.0060 </td><td>5855 </td><td>26 423 </td><td char=".">0.22 </td></tr><tr><td><em>Xanthomonas campestris</em> ATCC 33913 </td><td>NC_003902 </td><td>65.1 </td><td>5 076 188 </td><td char=".">0.0058 </td><td>5558 </td><td>26 376 </td><td char=".">0.21 </td></tr><tr><td><em>Xylella fastidiosa</em> 9a5c </td><td>NC_002488 </td><td>52.7 </td><td>2 679 306 </td><td char=".">0.0187 </td><td>9276 </td><td>8040 </td><td char=".">1.15 </td></tr><tr><td><em>Yersinia enterocolitica</em> 8081 </td><td>Unpublished </td><td>43.9 </td><td>4 615 899 </td><td char=".">0.0240 </td><td>20 592 </td><td>15 205 </td><td char=".">1.35 </td></tr><tr><td><em>Yersinia pestis</em> KIM </td><td>NC_004088 </td><td>47.6 </td><td>4 600 755 </td><td char=".">0.0270 </td><td>22 980 </td><td>13 738 </td><td char=".">1.67 </td></tr></tbody></table></div><div content-id="tbl2" class="table-modal"><table><thead name="thead" align=""><tr><th>Strain designation </th><th>Accession no. </th><th>mol% GC </th><th>Genome size (bp) </th><th>Repeat density </th><th>Observed no. of repeats </th><th>Expected (E) no. of repeats </th><th>O/E </th></tr></thead><tbody><tr><td><em>Acinetobacter</em> sp. ADP1 </td><td>NC_005966 </td><td>40.4 </td><td>3 598 621 </td><td char=".">0.0366 </td><td>24 075 </td><td>13 805 </td><td char=".">1.74 </td></tr><tr><td><em>Agrobacterium tumefaciens</em> C58 </td><td>NC_003304 </td><td>59.3 </td><td>4 917 050 </td><td char=".">0.0120 </td><td>11 180 </td><td>18 587 </td><td char=".">0.60 </td></tr><tr><td><em>Aquifex aeolicus</em> VF5 </td><td>NC_000918 </td><td>43.5 </td><td>1 551 335 </td><td char=".">0.0374 </td><td>10 932 </td><td>5187 </td><td char=".">2.11 </td></tr><tr><td><em>Bacillus anthracis Ames</em> </td><td>NC_003997 </td><td>35.4 </td><td>5 227 293 </td><td char=".">0.0454 </td><td>43 728 </td><td>26 377 </td><td char=".">1.66 </td></tr><tr><td><em>Bacillus cereus</em> ATCC 14579 </td><td>NC_004722 </td><td>35.3 </td><td>5 411 809 </td><td char=".">0.0455 </td><td>45 299 </td><td>27 468 </td><td char=".">1.65 </td></tr><tr><td><em>Bacillus halodurans</em> C-125 </td><td>NC_002570 </td><td>43.7 </td><td>4 202 353 </td><td char=".">0.0367 </td><td>28 385 </td><td>13 945 </td><td char=".">2.04 </td></tr><tr><td><em>Bacillus licheniformis</em> ATCC 14580 </td><td>NC_006270 </td><td>46.2 </td><td>4 222 336 </td><td char=".">0.0438 </td><td>33 815 </td><td>12 966 </td><td char=".">2.61 </td></tr><tr><td><em>Bacillus subtilis</em> 168 </td><td>NC_000964 </td><td>43.5 </td><td>4 214 630 </td><td char=".">0.0423 </td><td>32 621 </td><td>14 092 </td><td char=".">2.31 </td></tr><tr><td><em>Bacillus thuringiensis</em> 97-27 </td><td>NC_005957 </td><td>35.4 </td><td>5 327 682 </td><td char=".">0.0450 </td><td>44 116 </td><td>26 884 </td><td char=".">1.64 </td></tr><tr><td><em>Bacteriovorax marinus</em> </td><td>Unpublished<a reveal-id="tf2-1" data-open="tf2-1" class="link link-ref link-reveal xref-fn">a</a> </td><td>36.7 </td><td>3 435 933 </td><td char=".">0.0278 </td><td>17 861 </td><td>16 084 </td><td char=".">1.11 </td></tr><tr><td><em>Bacteroides fragilis</em> NCTC 9343 </td><td>Unpublished </td><td>43.1 </td><td>5 205 140 </td><td char=".">0.0269 </td><td>25 896 </td><td>17 679 </td><td char=".">1.46 </td></tr><tr><td><em>Bacteroides thetaiotamicron</em> VPI-5482 </td><td>NC_004663 </td><td>42.8 </td><td>6 260 361 </td><td char=".">0.0237 </td><td>27 662 </td><td>21 523 </td><td char=".">1.29 </td></tr><tr><td><em>Bartonella henselae</em> Houston- 1 </td><td>NC_005956 </td><td>38.2 </td><td>1 931 047 </td><td char=".">0.0568 </td><td>19 800 </td><td>8312 </td><td char=".">2.38 </td></tr><tr><td><em>Bartonella quintana</em> Toulose </td><td>NC_005955 </td><td>38.8 </td><td>1 581 384 </td><td char=".">0.0545 </td><td>15 587 </td><td>6589 </td><td char=".">2.37 </td></tr><tr><td><em>Bdellovibrio bacteriovorus</em> HD100 </td><td>NC_005363 </td><td>50.6 </td><td>3 782 950 </td><td char=".">0.0298 </td><td>21 025 </td><td>11 096 </td><td char=".">1.89 </td></tr><tr><td><em>Bifidobacterium longum</em> NCC 2705 </td><td>NC_004307 </td><td>60.1 </td><td>2 256 646 </td><td char=".">0.0064 </td><td>2761 </td><td>8877 </td><td char=".">0.31 </td></tr><tr><td><em>Blochmannia floridanus</em> </td><td>NC_005061 </td><td>27.4 </td><td>705 557 </td><td char=".">0.0595 </td><td>7353 </td><td>5724 </td><td char=".">1.28 </td></tr><tr><td><em>Bordetella avium</em> </td><td>Unpublished </td><td>69.2 </td><td>3 732 255 </td><td char=".">0.0144 </td><td>9986 </td><td>24 753 </td><td char=".">0.40 </td></tr><tr><td><em>Bordetella bronchiseptica</em> RB50 </td><td>NC_002927 </td><td>68.1 </td><td>5 339 179 </td><td char=".">0.0070 </td><td>7041 </td><td>33 158 </td><td char=".">0.21 </td></tr><tr><td><em>Bordetella parapertussis</em> 12822 </td><td>NC_002928 </td><td>68.1 </td><td>4 773 551 </td><td char=".">0.0068 </td><td>6134 </td><td>29 645 </td><td char=".">0.21 </td></tr><tr><td><em>Bordetella pertussis</em> Tohamal </td><td>NC_002929 </td><td>67.7 </td><td>4 086 189 </td><td char=".">0.0074 </td><td>5684 </td><td>24 777 </td><td char=".">0.23 </td></tr><tr><td><em>Borrelia burgdorferi</em> B3 1 </td><td>NC_001318 </td><td>28.6 </td><td>910 724 </td><td char=".">0.0922 </td><td>14 886 </td><td>6884 </td><td char=".">2.16 </td></tr><tr><td><em>Borrelia garinii</em> </td><td>NC_006156 </td><td>28.3 </td><td>904 246 </td><td char=".">0.0915 </td><td>14 614 </td><td>6957 </td><td char=".">2.10 </td></tr><tr><td><em>Bradyrhizobium japonicum</em> USDA 110 </td><td>NC_004463 </td><td>64.1 </td><td>9 105 828 </td><td char=".">0.0060 </td><td>10 568 </td><td>44 631 </td><td char=".">0.24 </td></tr><tr><td><em>Brucella melitensis</em> 16M </td><td>NC_003317 </td><td>57.2 </td><td>3 294 931 </td><td char=".">0.0132 </td><td>8324 </td><td>11 328 </td><td char=".">0.73 </td></tr><tr><td><em>Brucella suis</em> 1330 </td><td>NC_004310 </td><td>57.2 </td><td>3 315 173 </td><td char=".">0.0132 </td><td>8337 </td><td>11 398 </td><td char=".">0.73 </td></tr><tr><td><em>Buchnera aphidicola</em> Bp </td><td>NC_004545 </td><td>25.3 </td><td>615 980 </td><td char=".">0.0946 </td><td>10 043 </td><td>5644 </td><td char=".">1.78 </td></tr><tr><td><em>Burkholderia cenocepacia</em> J2315 </td><td>Unpublished </td><td>67.0 </td><td>7 963 121 </td><td char=".">0.0051 </td><td>7659 </td><td>46 309 </td><td char=".">0.17 </td></tr><tr><td><em>Burkholderia mallei</em> ATCC 23344 </td><td>NC_006349 </td><td>68.3 </td><td>5 835 527 </td><td char=".">0.0056 </td><td>6154 </td><td>36 676 </td><td char=".">0.17 </td></tr><tr><td><em>Burkholderia pseudomallei</em> K96243 </td><td>NC_006350 </td><td>68.1 </td><td>7 247 547 </td><td char=".">0.0058 </td><td>7897 </td><td>45 009 </td><td char=".">0.18 </td></tr><tr><td><em>Campylobacter jejuni</em> NCTC 11168 </td><td>NC_002163 </td><td>30.5 </td><td>1 641 481 </td><td char=".">0.0898 </td><td>26 642 </td><td>11 083 </td><td char=".">2.40 </td></tr><tr><td><em>Caulobacter crescentus</em> CB15 </td><td>NC_002996 </td><td>67.2 </td><td>4 016 947 </td><td char=".">0.0086 </td><td>6681 </td><td>23 641 </td><td char=".">0.28 </td></tr><tr><td><em>Chlamydia muridarum</em> MoPn </td><td>NC_002620 </td><td>40.3 </td><td>1 072 950 </td><td char=".">0.0460 </td><td>8978 </td><td>4137 </td><td char=".">2.17 </td></tr><tr><td><em>Chlamydia trachomatis</em> D/UW-3/CX </td><td>NC_000117 </td><td>41.3 </td><td>1 072 950 </td><td char=".">0.0383 </td><td>7473 </td><td>3941 </td><td char=".">1.90 </td></tr><tr><td><em>Chlamydophila abortus</em> S26/3 </td><td>Unpublished </td><td>39.9 </td><td>1 144 377 </td><td char=".">0.0408 </td><td>8530 </td><td>4502 </td><td char=".">1.89 </td></tr><tr><td><em>Chlamydophila caviae</em> GPIC </td><td>NC_003361 </td><td>39.2 </td><td>1 173 390 </td><td char=".">0.0405 </td><td>8712 </td><td>4786 </td><td char=".">1.82 </td></tr><tr><td><em>Chlamydophila pneumoniae</em> AR39 </td><td>NC_002179 </td><td>40.6 </td><td>1 229 853 </td><td char=".">0.0400 </td><td>8972 </td><td>4672 </td><td char=".">1.92 </td></tr><tr><td><em>Chlorobium tepidum</em> TLS </td><td>NC_002932 </td><td>56.5 </td><td>2 154 946 </td><td char=".">0.0213 </td><td>8449 </td><td>7205 </td><td char=".">1.17 </td></tr><tr><td><em>Chromobacterium violaceum</em> ATCC 12472 </td><td>NC_005085 </td><td>64.8 </td><td>4 751 080 </td><td char=".">0.0088 </td><td>7808 </td><td>24 256 </td><td char=".">0.32 </td></tr><tr><td><em>Clostridium acetobutylicum</em> ATCC 824 </td><td>NC_003030 </td><td>30.9 </td><td>3 940 880 </td><td char=".">0.0542 </td><td>38 709 </td><td>25 981 </td><td char=".">1.49 </td></tr><tr><td><em>Clostridium botulinum</em> ATCC 3502 </td><td>Unpublished </td><td>28.2 </td><td>3 886 916 </td><td char=".">0.0639 </td><td>45 111 </td><td>30 083 </td><td char=".">1.50 </td></tr><tr><td><em>Clostridium difficile</em> 630 </td><td>Unpublished </td><td>29.1 </td><td>4 290 252 </td><td char=".">0.0570 </td><td>44 286 </td><td>31 485 </td><td char=".">1.41 </td></tr><tr><td><em>Clostridium perfringens</em> 13 </td><td>NC_003366 </td><td>28.6 </td><td>3 031 430 </td><td char=".">0.0558 </td><td>31 018 </td><td>22 915 </td><td char=".">1.35 </td></tr><tr><td><em>Clostridium tetani</em> E88 </td><td>NC_004557 </td><td>28.7 </td><td>2 799 251 </td><td char=".">0.0612 </td><td>31 149 </td><td>21 035 </td><td char=".">1.48 </td></tr><tr><td><em>Corynebacterium diphtheriae</em> NCTC 13129 </td><td>NC_002935 </td><td>53.5 </td><td>2 488 635 </td><td char=".">0.0171 </td><td>8230 </td><td>7589 </td><td char=".">1.08 </td></tr><tr><td><em>Corynebacterium efficiens</em> YS-314 </td><td>NC_004369 </td><td>63.1 </td><td>3 147 090 </td><td char=".">0.0150 </td><td>9230 </td><td>14 565 </td><td char=".">0.63 </td></tr><tr><td><em>Corynebacterium glutamicum</em> ATCC 13032 </td><td>NC_003450 </td><td>53.8 </td><td>3 309 401 </td><td char=".">0.0142 </td><td>9101 </td><td>10 163 </td><td char=".">0.90 </td></tr><tr><td><em>Coxiella burnettii</em> RSA 493 </td><td>NC_002971 </td><td>42.7 </td><td>1 995 275 </td><td char=".">0.0510 </td><td>18 486 </td><td>6888 </td><td char=".">2.68 </td></tr><tr><td><em>Deinococcus radiodurans</em> Rl </td><td>NC_001263 </td><td>66.9 </td><td>3 060 986 </td><td char=".">0.0202 </td><td>11 805 </td><td>17 695 </td><td char=".">0.67 </td></tr><tr><td><em>Desulfotalea psychrophila</em> LSv54 </td><td>NC_006138 </td><td>46.8 </td><td>3 523 383 </td><td char=".">0.0396 </td><td>25 157 </td><td>10 675 </td><td char=".">2.36 </td></tr><tr><td><em>Desulfovibrio vulgaris</em> Hildenborough </td><td>NC_002937 </td><td>63.1 </td><td>3 570 858 </td><td char=".">0.0156 </td><td>10 447 </td><td>16 526 </td><td char=".">0.63 </td></tr><tr><td><em>Enterococcus faecalis</em> V583 </td><td>NC_004668 </td><td>37.5 </td><td>3 218 031 </td><td char=".">0.0547 </td><td>31 958 </td><td>14 398 </td><td char=".">2.22 </td></tr><tr><td><em>Erwinia carotovora</em> SCR11043 </td><td>NC_004547 </td><td>51 </td><td>5 064 019 </td><td char=".">0.0213 </td><td>20 025 </td><td>14 885 </td><td char=".">1.35 </td></tr><tr><td><em>Escherichia coli</em> K12 </td><td>NC_000913 </td><td>50.8 </td><td>4 639 221 </td><td char=".">0.0222 </td><td>19 237 </td><td>13 620 </td><td char=".">1.41 </td></tr><tr><td><em>Fusobacterium nucleatum</em> ATCC 25586 </td><td>NC_003454 </td><td>27.1 </td><td>2 174 500 </td><td char=".">0.0809 </td><td>31 627 </td><td>17 952 </td><td char=".">1.76 </td></tr><tr><td><em>Geobacter sulfurreducens</em> PCA </td><td>NC_002939 </td><td>60.9 </td><td>3 814 139 </td><td char=".">0.0235 </td><td>16 741 </td><td>15 641 </td><td char=".">1.07 </td></tr><tr><td><em>Gloeobacter violaceus</em> PCC7421 </td><td>NC_005125 </td><td>62 </td><td>4 659 019 </td><td char=".">0.0238 </td><td>20 847 </td><td>20 275 </td><td char=".">1.03 </td></tr><tr><td><em>Haemophilus ducreyi</em> 35000HP </td><td>NC_002940 </td><td>38.2 </td><td>1 698 955 </td><td char=".">0.0410 </td><td>12 790 </td><td>7313 </td><td char=".">1.75 </td></tr><tr><td><em>Haemophilus influenzae</em> Rd KW20 </td><td>NC_000907 </td><td>38.1 </td><td>1 830 138 </td><td char=".">0.0459 </td><td>15 499 </td><td>7921 </td><td char=".">1.96 </td></tr><tr><td><em>Helicobacter hepaticus</em> ATCC 51449 </td><td>NC_004917 </td><td>35.9 </td><td>1 799 146 </td><td char=".">0.0546 </td><td>17 933 </td><td>8818 </td><td char=".">2.03 </td></tr><tr><td><em>Helicobacter pylori</em> 26695 </td><td>NC_000915 </td><td>38.9 </td><td>1 667 867 </td><td char=".">0.0793 </td><td>24 106 </td><td>6913 </td><td char=".">3.49 </td></tr><tr><td><em>Helicobacter pylori</em> J99 </td><td>NC_000921 </td><td>39.2 </td><td>1 643 831 </td><td char=".">0.0785 </td><td>23 551 </td><td>6705 </td><td char=".">3.51 </td></tr><tr><td><em>Lactobacillus johnsonii</em> NCC 533 </td><td>NC_005362 </td><td>34.6 </td><td>1 992 676 </td><td char=".">0.0450 </td><td>16 570 </td><td>10 538 </td><td char=".">1.57 </td></tr><tr><td><em>Lactobacillus plantarum</em> WCFS1 </td><td>NC_004567 </td><td>44.5 </td><td>3 308 274 </td><td char=".">0.0227 </td><td>14 021 </td><td>10 672 </td><td char=".">1.31 </td></tr><tr><td><em>Lactococcus lactis</em> IL1403 </td><td>NC_002662 </td><td>35.3 </td><td>2 365 589 </td><td char=".">0.0573 </td><td>24 747 </td><td>12 007 </td><td char=".">2.06 </td></tr><tr><td><em>Leifsonia xyli</em> CTCB07 </td><td>NC_006087 </td><td>67.7 </td><td>2 584 158 </td><td char=".">0.0100 </td><td>4989 </td><td>15 669 </td><td char=".">0.32 </td></tr><tr><td><em>Leptospira interrogans</em> Fiocruz LI-130 </td><td>NC_004342 </td><td>35 </td><td>4 277 185 </td><td char=".">0.0755 </td><td>58 065 </td><td>22 094 </td><td char=".">2.63 </td></tr><tr><td><em>Listeria innocua</em> Clip 11262 </td><td>NC_003212 </td><td>37.4 </td><td>3 011 208 </td><td char=".">0.0513 </td><td>28 276 </td><td>13 549 </td><td char=".">2.09 </td></tr><tr><td><em>Listeria monocytogenes</em> EGD-e </td><td>NC_003210 </td><td>38 </td><td>2 944 528 </td><td char=".">0.0496 </td><td>26 798 </td><td>12 814 </td><td char=".">2.09 </td></tr><tr><td><em>Mesoplasma florum</em> LI </td><td>NC_006055 </td><td>27 </td><td>793 224 </td><td char=".">0.0738 </td><td>10 545 </td><td>6587 </td><td char=".">1.60 </td></tr><tr><td><em>Mesorhizobium loti</em> MAFF303099 </td><td>NC_002678 </td><td>62.7 </td><td>7 036 074 </td><td char=".">0.0070 </td><td>9388 </td><td>31 836 </td><td char=".">0.29 </td></tr><tr><td><em>Mycobacterium avium</em> k10 </td><td>NC_002944 </td><td>69.3 </td><td>4 829 781 </td><td char=".">0.0075 </td><td>7045 </td><td>32 224 </td><td char=".">0.22 </td></tr><tr><td><em>Mycobacterium bovis</em> AF2122/97 </td><td>NC_002945 </td><td>65.6 </td><td>4 345 492 </td><td char=".">0.0080 </td><td>6710 </td><td>23 254 </td><td char=".">0.29 </td></tr><tr><td><em>Mycobacterium leprae</em> TN </td><td>NC_002677 </td><td>57.8 </td><td>3 268 203 </td><td char=".">0.0095 </td><td>5925 </td><td>11 526 </td><td char=".">0.51 </td></tr><tr><td><em>Mycobacterium tuberculosis</em> H37Rv </td><td>NC_002755 </td><td>65.6 </td><td>4 411 532 </td><td char=".">0.0079 </td><td>6767 </td><td>23 608 </td><td char=".">0.29 </td></tr><tr><td><em>Mycoplasma gallisepticum</em> Rd KW20 </td><td>NC_004829 </td><td>31.5 </td><td>996 422 </td><td char=".">0.0518 </td><td>9347 </td><td>6338 </td><td char=".">1.47 </td></tr><tr><td><em>Mycoplasma genitalium</em> G-37 </td><td>NC_000908 </td><td>31.7 </td><td>580 074 </td><td char=".">0.0709 </td><td>7428 </td><td>3646 </td><td char=".">2.04 </td></tr><tr><td><em>Mycoplasma mobile</em> 163K </td><td>NC_006908 </td><td>25 </td><td>777 079 </td><td char=".">0.1034 </td><td>14 323 </td><td>7244 </td><td char=".">1.98 </td></tr><tr><td><em>Mycoplasma mycoides</em> PG1 </td><td>NC_005364 </td><td>24 </td><td>1 211 703 </td><td char=".">0.0914 </td><td>19 889 </td><td>11 956 </td><td char=".">1.66 </td></tr><tr><td><em>Mycoplasma penetrans</em> HF-2 </td><td>NC_004432 </td><td>25.7 </td><td>1 358 633 </td><td char=".">0.0764 </td><td>18 734 </td><td>12 166 </td><td char=".">1.54 </td></tr><tr><td><em>Mycoplasma pneumoniae</em> Ml29 </td><td>NC_000912 </td><td>40 </td><td>816 394 </td><td char=".">0.0432 </td><td>6584 </td><td>3195 </td><td char=".">2.06 </td></tr><tr><td><em>Mycoplasma pulmonis</em> UAB CTIP </td><td>NC_002771 </td><td>26.6 </td><td>963 879 </td><td char=".">0.1149 </td><td>19 740 </td><td>8193 </td><td char=".">2.41 </td></tr><tr><td><em>Neisseria gonorrhoeae</em> </td><td>Unpublished </td><td>52.7 </td><td>2 146 877 </td><td char=".">0.0326 </td><td>12 995 </td><td>6443 </td><td char=".">2.02 </td></tr><tr><td><em>Neisseria meningitidis</em> MC58 </td><td>NC_003112 </td><td>51.5 </td><td>2 272 351 </td><td char=".">0.0331 </td><td>14 001 </td><td>6707 </td><td char=".">2.09 </td></tr><tr><td><em>Neisseria meningitidis</em> Z2491 </td><td>NC_003116 </td><td>51.8 </td><td>2184 406 </td><td char=".">0.0328 </td><td>13 304 </td><td>6469 </td><td char=".">2.06 </td></tr><tr><td><em>Nitrosomonas europeae</em> ATCC 19718 </td><td>NC_004757 </td><td>50.7 </td><td>2 812094 </td><td char=".">0.0209 </td><td>10 934 </td><td>8252 </td><td char=".">1.33 </td></tr><tr><td><em>Nostoc</em> sp. PCC 7120 </td><td>NC_003272 </td><td>41.3 </td><td>6 413 771 </td><td char=".">0.0315 </td><td>37 738 </td><td>23 559 </td><td char=".">1.60 </td></tr><tr><td><em>Oceanobacillus iheyensis</em> HTE831 </td><td>NC_004193 </td><td>35.7 </td><td>3 630 528 </td><td char=".">0.0388 </td><td>25 862 </td><td>18 002 </td><td char=".">1.44 </td></tr><tr><td><em>Parachlamydia</em> sp. UWE25 </td><td>NC_005861 </td><td>34.7 </td><td>2 414 465 </td><td char=".">0.0649 </td><td>27 883 </td><td>12 694 </td><td char=".">2.20 </td></tr><tr><td><em>Pasteurella multocida</em> Pm70 </td><td>NC_002632 </td><td>40.4 </td><td>2 257 487 </td><td char=".">0.0417 </td><td>17 297 </td><td>8660 </td><td char=".">2.00 </td></tr><tr><td><em>Photorhabdus luminescens</em> TTO1 </td><td>NC_005126 </td><td>42.8 </td><td>5 688 987 </td><td char=".">0.0314 </td><td>32 937 </td><td>19 559 </td><td char=".">1.68 </td></tr><tr><td><em>Phytoplasma asteris</em> OY-M </td><td>NC_005303 </td><td>27.7 </td><td>860 631 </td><td char=".">0.1108 </td><td>16 757 </td><td>6860 </td><td char=".">2.44 </td></tr><tr><td><em>Pirelulla</em> sp. 1 </td><td>NC_005027 </td><td>55.4 </td><td>7 145 576 </td><td char=".">0.0101 </td><td>13 771 </td><td>22 974 </td><td char=".">0.60 </td></tr><tr><td><em>Porphyromonas gingivalis</em> W83 </td><td>NC_002950 </td><td>48.3 </td><td>2 343 476 </td><td char=".">0.0221 </td><td>9614 </td><td>6932 </td><td char=".">1.39 </td></tr><tr><td><em>Prochlorococcus marinus</em> MIT 9313 </td><td>NC_005071 </td><td>50.7 </td><td>2 410 873 </td><td char=".">0.0169 </td><td>7592 </td><td>7075 </td><td char=".">1.07 </td></tr><tr><td><em>Propionibacterium acnes</em> KPA171202 </td><td>NC_006085 </td><td>60 </td><td>2 560 265 </td><td char=".">0.0105 </td><td>5250 </td><td>10 021 </td><td char=".">0.52 </td></tr><tr><td><em>Pseudomonas aeruginosa</em> PAO1 </td><td>NC_002516 </td><td>66.6 </td><td>6 264 403 </td><td char=".">0.0063 </td><td>7472 </td><td>35 572 </td><td char=".">0.21 </td></tr><tr><td><em>Pseudomonas putida</em> KT2440 </td><td>NC_002947 </td><td>61.5 </td><td>6 181 863 </td><td char=".">0.0103 </td><td>12 079 </td><td>26 178 </td><td char=".">0.46 </td></tr><tr><td><em>Pseudomonas syringae</em> DC3000 </td><td>NC_004578 </td><td>58.4 </td><td>6 397 126 </td><td char=".">0.0125 </td><td>14 975 </td><td>23 174 </td><td char=".">0.65 </td></tr><tr><td><em>Ralstonia solanacearum</em> GMI1000 </td><td>NC_003295 </td><td>67 </td><td>5 810 922 </td><td char=".">0.0077 </td><td>8463 </td><td>33 793 </td><td char=".">0.25 </td></tr><tr><td><em>Rhodopseudomonas palustris</em> CGA009 </td><td>NC_005296 </td><td>65 </td><td>5 459 213 </td><td char=".">0.0043 </td><td>4511 </td><td>28 200 </td><td char=".">0.16 </td></tr><tr><td><em>Rickettsia conorii</em> Malish 7 </td><td>NC_003103 </td><td>32.4 </td><td>1 268 755 </td><td char=".">0.0525 </td><td>12 127 </td><td>7647 </td><td char=".">1.59 </td></tr><tr><td><em>Rickettsia prowazekii</em> Madrid E </td><td>NC_000963 </td><td>29 </td><td>1 111 523 </td><td char=".">0.0560 </td><td>11 282 </td><td>8206 </td><td char=".">1.37 </td></tr><tr><td><em>Rickettsia typhi</em> Wilmington </td><td>NC_006142 </td><td>28.3 </td><td>1 111 496 </td><td char=".">0.0552 </td><td>11 128 </td><td>8552 </td><td char=".">1.30 </td></tr><tr><td><em>Salmonella enterica</em> CT18 </td><td>NC_003198 </td><td>52.1 </td><td>4 809 037 </td><td char=".">0.0240 </td><td>21 454 </td><td>14 296 </td><td char=".">1.50 </td></tr><tr><td><em>Salmonella typhimurium</em> LT2 </td><td>NC_003197 </td><td>52.2 </td><td>4 857 432 </td><td char=".">0.0239 </td><td>21 602 </td><td>14 460 </td><td char=".">1.49 </td></tr><tr><td><em>Serratia marcescens</em> Dbl 1 </td><td>Unpublished </td><td>59.5 </td><td>5 113 802 </td><td char=".">0.0155 </td><td>14 776 </td><td>19 521 </td><td char=".">0.76 </td></tr><tr><td><em>Shewanella oneidensis</em> MR-1 </td><td>NC_004347 </td><td>46 </td><td>4 969 803 </td><td char=".">0.0249 </td><td>23 135 </td><td>15 337 </td><td char=".">1.51 </td></tr><tr><td><em>Shigella flexneri</em> 2457T </td><td>NC_004741 </td><td>50.9 </td><td>4 599 354 </td><td char=".">0.0222 </td><td>19 024 </td><td>13 511 </td><td char=".">1.41 </td></tr><tr><td><em>Sinorhizobium meliloti</em> 1021 </td><td>NC_003047 </td><td>62.7 </td><td>3 654 135 </td><td char=".">0.0075 </td><td>5228 </td><td>16 534 </td><td char=".">0.32 </td></tr><tr><td><em>Staphylococcus aureus</em> MW2 </td><td>NC_003923 </td><td>32.8 </td><td>2 820 462 </td><td char=".">0.0389 </td><td>20 991 </td><td>16 599 </td><td char=".">1.26 </td></tr><tr><td><em>Staphylococcus epidermidis</em> ATCC 12228 </td><td>NC_004461 </td><td>32.1 </td><td>2 499 279 </td><td char=".">0.0424 </td><td>19 544 </td><td>15 337 </td><td char=".">1.27 </td></tr><tr><td><em>Streptococcus agalactiae</em> 2603V/R </td><td>NC_004416 </td><td>35.7 </td><td>2 160 267 </td><td char=".">0.0446 </td><td>17 709 </td><td>10 712 </td><td char=".">1.65 </td></tr><tr><td><em>Streptococcus mutans</em> UA159 </td><td>NC_004350 </td><td>36.8 </td><td>2 030 921 </td><td char=".">0.0483 </td><td>18 109 </td><td>9453 </td><td char=".">1.92 </td></tr><tr><td><em>Streptococcus pneumoniae</em> TIGR4 </td><td>NC_003028 </td><td>39.7 </td><td>2 160 837 </td><td char=".">0.0374 </td><td>15 003 </td><td>8588 </td><td char=".">1.75 </td></tr><tr><td><em>Streptococcus pyogenes</em> MGAS315 </td><td>NC_004070 </td><td>38.6 </td><td>1 900 521 </td><td char=".">0.0453 </td><td>15 743 </td><td>8005 </td><td char=".">1.97 </td></tr><tr><td><em>Streptomyces avermitilis</em> MA-4680 </td><td>NC_003155 </td><td>70.7 </td><td>9 025 608 </td><td char=".">0.0125 </td><td>21 960 </td><td>65 454 </td><td char=".">0.34 </td></tr><tr><td><em>Streptomyces coelicolor</em> A3 (2) </td><td>NC_003888 </td><td>72.1 </td><td>8 667 507 </td><td char=".">0.0128 </td><td>21 627 </td><td>68 279 </td><td char=".">0.32 </td></tr><tr><td><em>Synechococcus</em> sp. WH8102 </td><td>NC_005070 </td><td>59.4 </td><td>2 434 428 </td><td char=".">0.0153 </td><td>7002 </td><td>9247 </td><td char=".">0.76 </td></tr><tr><td><em>Synechocystis</em> sp. PCC 6803 </td><td>NC_000911 </td><td>47.7 </td><td>3 573 470 </td><td char=".">0.0457 </td><td>30 442 </td><td>10 654 </td><td char=".">2.86 </td></tr><tr><td><em>Thermoanaerobacter tengcongensis</em> </td><td>NC_003869 </td><td>37.6 </td><td>2 689 445 </td><td char=".">0.0568 </td><td>28 261 </td><td>11 966 </td><td char=".">2.36 </td></tr><tr><td><em>Thermosynechococcus elongatus</em> BP-1 </td><td>NC_004113 </td><td>53.9 </td><td>2 593 857 </td><td char=".">0.0325 </td><td>15 850 </td><td>7985 </td><td char=".">1.99 </td></tr><tr><td><em>Thermotoga maritima</em> MSB 8 </td><td>NC_000853 </td><td>46.2 </td><td>1 860 725 </td><td char=".">0.0298 </td><td>10 394 </td><td>5714 </td><td char=".">1.82 </td></tr><tr><td><em>Thermus thermophilus</em> HB27 </td><td>NC_005835 </td><td>69.4 </td><td>1 894 877 </td><td char=".">0.0533 </td><td>18 935 </td><td>12 718 </td><td char=".">1.49 </td></tr><tr><td><em>Treponema denticola</em> ATCC 35405 </td><td>NC_002967 </td><td>37.9 </td><td>2 843 201 </td><td char=".">0.0737 </td><td>37 216 </td><td>12 442 </td><td char=".">2.99 </td></tr><tr><td><em>Treponema pallidum</em> Nichols </td><td>NC_000919 </td><td>52.8 </td><td>1 138 011 </td><td char=".">0.0320 </td><td>6669 </td><td>3421 </td><td char=".">1.95 </td></tr><tr><td><em>Tropheryma whipplei</em> TW08/27 </td><td>NC_004551 </td><td>46.3 </td><td>925 938 </td><td char=".">0.0286 </td><td>4942 </td><td>2837 </td><td char=".">1.74 </td></tr><tr><td><em>Ureaplasma parvum</em> ATCC 700970 </td><td>NC_002162 </td><td>25.5 </td><td>751 719 </td><td char=".">0.0833 </td><td>11 146 </td><td>6809 </td><td char=".">1.64 </td></tr><tr><td><em>Vibrio cholerae</em> Nl696l </td><td>NC_002505 </td><td>47.5 </td><td>4 033 464 </td><td char=".">0.0238 </td><td>17 969 </td><td>12 063 </td><td char=".">1.49 </td></tr><tr><td><em>Vibrio parahaemolyticus</em> RMID 2210633 </td><td>NC_004603 </td><td>45.4 </td><td>5 165 770 </td><td char=".">0.0228 </td><td>22 140 </td><td>16 203 </td><td char=".">1.37 </td></tr><tr><td><em>Vibrio vulnificus</em> CMCP6 </td><td>NC_004459 </td><td>46.7 </td><td>5 211 578 </td><td char=".">0.0240 </td><td>23 472 </td><td>15 823 </td><td char=".">1.48 </td></tr><tr><td><em>Wigglesworthia glossinidia</em> </td><td>NC_004344 </td><td>22.5 </td><td>697 721 </td><td char=".">0.1241 </td><td>14 619 </td><td>7488 </td><td char=".">1.95 </td></tr><tr><td><em>Wolinella succinogenes</em> DSM 1740 </td><td>NC_005090 </td><td>48.5 </td><td>2 110 355 </td><td char=".">0.0434 </td><td>17 143 </td><td>6229 </td><td char=".">2.75 </td></tr><tr><td><em>Xanthomonas axonopodis</em> 306 </td><td>NC_003919 </td><td>64.8 </td><td>5 175 554 </td><td char=".">0.0060 </td><td>5855 </td><td>26 423 </td><td char=".">0.22 </td></tr><tr><td><em>Xanthomonas campestris</em> ATCC 33913 </td><td>NC_003902 </td><td>65.1 </td><td>5 076 188 </td><td char=".">0.0058 </td><td>5558 </td><td>26 376 </td><td char=".">0.21 </td></tr><tr><td><em>Xylella fastidiosa</em> 9a5c </td><td>NC_002488 </td><td>52.7 </td><td>2 679 306 </td><td char=".">0.0187 </td><td>9276 </td><td>8040 </td><td char=".">1.15 </td></tr><tr><td><em>Yersinia enterocolitica</em> 8081 </td><td>Unpublished </td><td>43.9 </td><td>4 615 899 </td><td char=".">0.0240 </td><td>20 592 </td><td>15 205 </td><td char=".">1.35 </td></tr><tr><td><em>Yersinia pestis</em> KIM </td><td>NC_004088 </td><td>47.6 </td><td>4 600 755 </td><td char=".">0.0270 </td><td>22 980 </td><td>13 738 </td><td char=".">1.67 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Repeat density is shown as the percentage of the genome that is located in mononucleotide repeats &gt;5 nt.</p></div></div><div content-id="tf2-1"><div class="fn"><p><sup>a</sup>Unpublished genome sequences were produced by the Pathogen Sequencing Group at the Sanger Institute (<a class="link link-uri" href="http://www.sanger.ac.uk/Projects/Microbes/" target="_blank"></a>) (<em>Bacteriovorax marinus</em>, <em>Bacteroides fragilis</em>, <em>Bordetella avium</em>, <em>Burkholderia cenocepacia</em>, <em>Chlamydophila abortus</em>, <em>Clostridium botulinum</em>, <em>Clostridium difficile</em>, <em>Serratia marcescens</em> and <em>Yersinia enterocolitica</em>) or the University of Oklahoma (<a class="link link-uri" href="http://www.genome.ou.edu/gono.html" target="_blank"></a>) (<em>Neisseria gonorrhoeae</em>).</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/10424235" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="tbl2" class="table-wrap-title" data-id="tbl2"><div class="label -label"><strong>Table 2</strong></div><div class="caption"><p>Overview of the bacterial genomes compared in the present study, the density of mononucleotide SSRs in the genome, the number of mononucleotide SSRs observed (O), the number of mononucleotide SSRs expected (based on base composition) (E) and the ratio O/E.</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Strain designation </th><th>Accession no. </th><th>mol% GC </th><th>Genome size (bp) </th><th>Repeat density </th><th>Observed no. of repeats </th><th>Expected (E) no. of repeats </th><th>O/E </th></tr></thead><tbody><tr><td><em>Acinetobacter</em> sp. ADP1 </td><td>NC_005966 </td><td>40.4 </td><td>3 598 621 </td><td char=".">0.0366 </td><td>24 075 </td><td>13 805 </td><td char=".">1.74 </td></tr><tr><td><em>Agrobacterium tumefaciens</em> C58 </td><td>NC_003304 </td><td>59.3 </td><td>4 917 050 </td><td char=".">0.0120 </td><td>11 180 </td><td>18 587 </td><td char=".">0.60 </td></tr><tr><td><em>Aquifex aeolicus</em> VF5 </td><td>NC_000918 </td><td>43.5 </td><td>1 551 335 </td><td char=".">0.0374 </td><td>10 932 </td><td>5187 </td><td char=".">2.11 </td></tr><tr><td><em>Bacillus anthracis Ames</em> </td><td>NC_003997 </td><td>35.4 </td><td>5 227 293 </td><td char=".">0.0454 </td><td>43 728 </td><td>26 377 </td><td char=".">1.66 </td></tr><tr><td><em>Bacillus cereus</em> ATCC 14579 </td><td>NC_004722 </td><td>35.3 </td><td>5 411 809 </td><td char=".">0.0455 </td><td>45 299 </td><td>27 468 </td><td char=".">1.65 </td></tr><tr><td><em>Bacillus halodurans</em> C-125 </td><td>NC_002570 </td><td>43.7 </td><td>4 202 353 </td><td char=".">0.0367 </td><td>28 385 </td><td>13 945 </td><td char=".">2.04 </td></tr><tr><td><em>Bacillus licheniformis</em> ATCC 14580 </td><td>NC_006270 </td><td>46.2 </td><td>4 222 336 </td><td char=".">0.0438 </td><td>33 815 </td><td>12 966 </td><td char=".">2.61 </td></tr><tr><td><em>Bacillus subtilis</em> 168 </td><td>NC_000964 </td><td>43.5 </td><td>4 214 630 </td><td char=".">0.0423 </td><td>32 621 </td><td>14 092 </td><td char=".">2.31 </td></tr><tr><td><em>Bacillus thuringiensis</em> 97-27 </td><td>NC_005957 </td><td>35.4 </td><td>5 327 682 </td><td char=".">0.0450 </td><td>44 116 </td><td>26 884 </td><td char=".">1.64 </td></tr><tr><td><em>Bacteriovorax marinus</em> </td><td>Unpublished<a reveal-id="tf2-1" data-open="tf2-1" class="link link-ref link-reveal xref-fn">a</a> </td><td>36.7 </td><td>3 435 933 </td><td char=".">0.0278 </td><td>17 861 </td><td>16 084 </td><td char=".">1.11 </td></tr><tr><td><em>Bacteroides fragilis</em> NCTC 9343 </td><td>Unpublished </td><td>43.1 </td><td>5 205 140 </td><td char=".">0.0269 </td><td>25 896 </td><td>17 679 </td><td char=".">1.46 </td></tr><tr><td><em>Bacteroides thetaiotamicron</em> VPI-5482 </td><td>NC_004663 </td><td>42.8 </td><td>6 260 361 </td><td char=".">0.0237 </td><td>27 662 </td><td>21 523 </td><td char=".">1.29 </td></tr><tr><td><em>Bartonella henselae</em> Houston- 1 </td><td>NC_005956 </td><td>38.2 </td><td>1 931 047 </td><td char=".">0.0568 </td><td>19 800 </td><td>8312 </td><td char=".">2.38 </td></tr><tr><td><em>Bartonella quintana</em> Toulose </td><td>NC_005955 </td><td>38.8 </td><td>1 581 384 </td><td char=".">0.0545 </td><td>15 587 </td><td>6589 </td><td char=".">2.37 </td></tr><tr><td><em>Bdellovibrio bacteriovorus</em> HD100 </td><td>NC_005363 </td><td>50.6 </td><td>3 782 950 </td><td char=".">0.0298 </td><td>21 025 </td><td>11 096 </td><td char=".">1.89 </td></tr><tr><td><em>Bifidobacterium longum</em> NCC 2705 </td><td>NC_004307 </td><td>60.1 </td><td>2 256 646 </td><td char=".">0.0064 </td><td>2761 </td><td>8877 </td><td char=".">0.31 </td></tr><tr><td><em>Blochmannia floridanus</em> </td><td>NC_005061 </td><td>27.4 </td><td>705 557 </td><td char=".">0.0595 </td><td>7353 </td><td>5724 </td><td char=".">1.28 </td></tr><tr><td><em>Bordetella avium</em> </td><td>Unpublished </td><td>69.2 </td><td>3 732 255 </td><td char=".">0.0144 </td><td>9986 </td><td>24 753 </td><td char=".">0.40 </td></tr><tr><td><em>Bordetella bronchiseptica</em> RB50 </td><td>NC_002927 </td><td>68.1 </td><td>5 339 179 </td><td char=".">0.0070 </td><td>7041 </td><td>33 158 </td><td char=".">0.21 </td></tr><tr><td><em>Bordetella parapertussis</em> 12822 </td><td>NC_002928 </td><td>68.1 </td><td>4 773 551 </td><td char=".">0.0068 </td><td>6134 </td><td>29 645 </td><td char=".">0.21 </td></tr><tr><td><em>Bordetella pertussis</em> Tohamal </td><td>NC_002929 </td><td>67.7 </td><td>4 086 189 </td><td char=".">0.0074 </td><td>5684 </td><td>24 777 </td><td char=".">0.23 </td></tr><tr><td><em>Borrelia burgdorferi</em> B3 1 </td><td>NC_001318 </td><td>28.6 </td><td>910 724 </td><td char=".">0.0922 </td><td>14 886 </td><td>6884 </td><td char=".">2.16 </td></tr><tr><td><em>Borrelia garinii</em> </td><td>NC_006156 </td><td>28.3 </td><td>904 246 </td><td char=".">0.0915 </td><td>14 614 </td><td>6957 </td><td char=".">2.10 </td></tr><tr><td><em>Bradyrhizobium japonicum</em> USDA 110 </td><td>NC_004463 </td><td>64.1 </td><td>9 105 828 </td><td char=".">0.0060 </td><td>10 568 </td><td>44 631 </td><td char=".">0.24 </td></tr><tr><td><em>Brucella melitensis</em> 16M </td><td>NC_003317 </td><td>57.2 </td><td>3 294 931 </td><td char=".">0.0132 </td><td>8324 </td><td>11 328 </td><td char=".">0.73 </td></tr><tr><td><em>Brucella suis</em> 1330 </td><td>NC_004310 </td><td>57.2 </td><td>3 315 173 </td><td char=".">0.0132 </td><td>8337 </td><td>11 398 </td><td char=".">0.73 </td></tr><tr><td><em>Buchnera aphidicola</em> Bp </td><td>NC_004545 </td><td>25.3 </td><td>615 980 </td><td char=".">0.0946 </td><td>10 043 </td><td>5644 </td><td char=".">1.78 </td></tr><tr><td><em>Burkholderia cenocepacia</em> J2315 </td><td>Unpublished </td><td>67.0 </td><td>7 963 121 </td><td char=".">0.0051 </td><td>7659 </td><td>46 309 </td><td char=".">0.17 </td></tr><tr><td><em>Burkholderia mallei</em> ATCC 23344 </td><td>NC_006349 </td><td>68.3 </td><td>5 835 527 </td><td char=".">0.0056 </td><td>6154 </td><td>36 676 </td><td char=".">0.17 </td></tr><tr><td><em>Burkholderia pseudomallei</em> K96243 </td><td>NC_006350 </td><td>68.1 </td><td>7 247 547 </td><td char=".">0.0058 </td><td>7897 </td><td>45 009 </td><td char=".">0.18 </td></tr><tr><td><em>Campylobacter jejuni</em> NCTC 11168 </td><td>NC_002163 </td><td>30.5 </td><td>1 641 481 </td><td char=".">0.0898 </td><td>26 642 </td><td>11 083 </td><td char=".">2.40 </td></tr><tr><td><em>Caulobacter crescentus</em> CB15 </td><td>NC_002996 </td><td>67.2 </td><td>4 016 947 </td><td char=".">0.0086 </td><td>6681 </td><td>23 641 </td><td char=".">0.28 </td></tr><tr><td><em>Chlamydia muridarum</em> MoPn </td><td>NC_002620 </td><td>40.3 </td><td>1 072 950 </td><td char=".">0.0460 </td><td>8978 </td><td>4137 </td><td char=".">2.17 </td></tr><tr><td><em>Chlamydia trachomatis</em> D/UW-3/CX </td><td>NC_000117 </td><td>41.3 </td><td>1 072 950 </td><td char=".">0.0383 </td><td>7473 </td><td>3941 </td><td char=".">1.90 </td></tr><tr><td><em>Chlamydophila abortus</em> S26/3 </td><td>Unpublished </td><td>39.9 </td><td>1 144 377 </td><td char=".">0.0408 </td><td>8530 </td><td>4502 </td><td char=".">1.89 </td></tr><tr><td><em>Chlamydophila caviae</em> GPIC </td><td>NC_003361 </td><td>39.2 </td><td>1 173 390 </td><td char=".">0.0405 </td><td>8712 </td><td>4786 </td><td char=".">1.82 </td></tr><tr><td><em>Chlamydophila pneumoniae</em> AR39 </td><td>NC_002179 </td><td>40.6 </td><td>1 229 853 </td><td char=".">0.0400 </td><td>8972 </td><td>4672 </td><td char=".">1.92 </td></tr><tr><td><em>Chlorobium tepidum</em> TLS </td><td>NC_002932 </td><td>56.5 </td><td>2 154 946 </td><td char=".">0.0213 </td><td>8449 </td><td>7205 </td><td char=".">1.17 </td></tr><tr><td><em>Chromobacterium violaceum</em> ATCC 12472 </td><td>NC_005085 </td><td>64.8 </td><td>4 751 080 </td><td char=".">0.0088 </td><td>7808 </td><td>24 256 </td><td char=".">0.32 </td></tr><tr><td><em>Clostridium acetobutylicum</em> ATCC 824 </td><td>NC_003030 </td><td>30.9 </td><td>3 940 880 </td><td char=".">0.0542 </td><td>38 709 </td><td>25 981 </td><td char=".">1.49 </td></tr><tr><td><em>Clostridium botulinum</em> ATCC 3502 </td><td>Unpublished </td><td>28.2 </td><td>3 886 916 </td><td char=".">0.0639 </td><td>45 111 </td><td>30 083 </td><td char=".">1.50 </td></tr><tr><td><em>Clostridium difficile</em> 630 </td><td>Unpublished </td><td>29.1 </td><td>4 290 252 </td><td char=".">0.0570 </td><td>44 286 </td><td>31 485 </td><td char=".">1.41 </td></tr><tr><td><em>Clostridium perfringens</em> 13 </td><td>NC_003366 </td><td>28.6 </td><td>3 031 430 </td><td char=".">0.0558 </td><td>31 018 </td><td>22 915 </td><td char=".">1.35 </td></tr><tr><td><em>Clostridium tetani</em> E88 </td><td>NC_004557 </td><td>28.7 </td><td>2 799 251 </td><td char=".">0.0612 </td><td>31 149 </td><td>21 035 </td><td char=".">1.48 </td></tr><tr><td><em>Corynebacterium diphtheriae</em> NCTC 13129 </td><td>NC_002935 </td><td>53.5 </td><td>2 488 635 </td><td char=".">0.0171 </td><td>8230 </td><td>7589 </td><td char=".">1.08 </td></tr><tr><td><em>Corynebacterium efficiens</em> YS-314 </td><td>NC_004369 </td><td>63.1 </td><td>3 147 090 </td><td char=".">0.0150 </td><td>9230 </td><td>14 565 </td><td char=".">0.63 </td></tr><tr><td><em>Corynebacterium glutamicum</em> ATCC 13032 </td><td>NC_003450 </td><td>53.8 </td><td>3 309 401 </td><td char=".">0.0142 </td><td>9101 </td><td>10 163 </td><td char=".">0.90 </td></tr><tr><td><em>Coxiella burnettii</em> RSA 493 </td><td>NC_002971 </td><td>42.7 </td><td>1 995 275 </td><td char=".">0.0510 </td><td>18 486 </td><td>6888 </td><td char=".">2.68 </td></tr><tr><td><em>Deinococcus radiodurans</em> Rl </td><td>NC_001263 </td><td>66.9 </td><td>3 060 986 </td><td char=".">0.0202 </td><td>11 805 </td><td>17 695 </td><td char=".">0.67 </td></tr><tr><td><em>Desulfotalea psychrophila</em> LSv54 </td><td>NC_006138 </td><td>46.8 </td><td>3 523 383 </td><td char=".">0.0396 </td><td>25 157 </td><td>10 675 </td><td char=".">2.36 </td></tr><tr><td><em>Desulfovibrio vulgaris</em> Hildenborough </td><td>NC_002937 </td><td>63.1 </td><td>3 570 858 </td><td char=".">0.0156 </td><td>10 447 </td><td>16 526 </td><td char=".">0.63 </td></tr><tr><td><em>Enterococcus faecalis</em> V583 </td><td>NC_004668 </td><td>37.5 </td><td>3 218 031 </td><td char=".">0.0547 </td><td>31 958 </td><td>14 398 </td><td char=".">2.22 </td></tr><tr><td><em>Erwinia carotovora</em> SCR11043 </td><td>NC_004547 </td><td>51 </td><td>5 064 019 </td><td char=".">0.0213 </td><td>20 025 </td><td>14 885 </td><td char=".">1.35 </td></tr><tr><td><em>Escherichia coli</em> K12 </td><td>NC_000913 </td><td>50.8 </td><td>4 639 221 </td><td char=".">0.0222 </td><td>19 237 </td><td>13 620 </td><td char=".">1.41 </td></tr><tr><td><em>Fusobacterium nucleatum</em> ATCC 25586 </td><td>NC_003454 </td><td>27.1 </td><td>2 174 500 </td><td char=".">0.0809 </td><td>31 627 </td><td>17 952 </td><td char=".">1.76 </td></tr><tr><td><em>Geobacter sulfurreducens</em> PCA </td><td>NC_002939 </td><td>60.9 </td><td>3 814 139 </td><td char=".">0.0235 </td><td>16 741 </td><td>15 641 </td><td char=".">1.07 </td></tr><tr><td><em>Gloeobacter violaceus</em> PCC7421 </td><td>NC_005125 </td><td>62 </td><td>4 659 019 </td><td char=".">0.0238 </td><td>20 847 </td><td>20 275 </td><td char=".">1.03 </td></tr><tr><td><em>Haemophilus ducreyi</em> 35000HP </td><td>NC_002940 </td><td>38.2 </td><td>1 698 955 </td><td char=".">0.0410 </td><td>12 790 </td><td>7313 </td><td char=".">1.75 </td></tr><tr><td><em>Haemophilus influenzae</em> Rd KW20 </td><td>NC_000907 </td><td>38.1 </td><td>1 830 138 </td><td char=".">0.0459 </td><td>15 499 </td><td>7921 </td><td char=".">1.96 </td></tr><tr><td><em>Helicobacter hepaticus</em> ATCC 51449 </td><td>NC_004917 </td><td>35.9 </td><td>1 799 146 </td><td char=".">0.0546 </td><td>17 933 </td><td>8818 </td><td char=".">2.03 </td></tr><tr><td><em>Helicobacter pylori</em> 26695 </td><td>NC_000915 </td><td>38.9 </td><td>1 667 867 </td><td char=".">0.0793 </td><td>24 106 </td><td>6913 </td><td char=".">3.49 </td></tr><tr><td><em>Helicobacter pylori</em> J99 </td><td>NC_000921 </td><td>39.2 </td><td>1 643 831 </td><td char=".">0.0785 </td><td>23 551 </td><td>6705 </td><td char=".">3.51 </td></tr><tr><td><em>Lactobacillus johnsonii</em> NCC 533 </td><td>NC_005362 </td><td>34.6 </td><td>1 992 676 </td><td char=".">0.0450 </td><td>16 570 </td><td>10 538 </td><td char=".">1.57 </td></tr><tr><td><em>Lactobacillus plantarum</em> WCFS1 </td><td>NC_004567 </td><td>44.5 </td><td>3 308 274 </td><td char=".">0.0227 </td><td>14 021 </td><td>10 672 </td><td char=".">1.31 </td></tr><tr><td><em>Lactococcus lactis</em> IL1403 </td><td>NC_002662 </td><td>35.3 </td><td>2 365 589 </td><td char=".">0.0573 </td><td>24 747 </td><td>12 007 </td><td char=".">2.06 </td></tr><tr><td><em>Leifsonia xyli</em> CTCB07 </td><td>NC_006087 </td><td>67.7 </td><td>2 584 158 </td><td char=".">0.0100 </td><td>4989 </td><td>15 669 </td><td char=".">0.32 </td></tr><tr><td><em>Leptospira interrogans</em> Fiocruz LI-130 </td><td>NC_004342 </td><td>35 </td><td>4 277 185 </td><td char=".">0.0755 </td><td>58 065 </td><td>22 094 </td><td char=".">2.63 </td></tr><tr><td><em>Listeria innocua</em> Clip 11262 </td><td>NC_003212 </td><td>37.4 </td><td>3 011 208 </td><td char=".">0.0513 </td><td>28 276 </td><td>13 549 </td><td char=".">2.09 </td></tr><tr><td><em>Listeria monocytogenes</em> EGD-e </td><td>NC_003210 </td><td>38 </td><td>2 944 528 </td><td char=".">0.0496 </td><td>26 798 </td><td>12 814 </td><td char=".">2.09 </td></tr><tr><td><em>Mesoplasma florum</em> LI </td><td>NC_006055 </td><td>27 </td><td>793 224 </td><td char=".">0.0738 </td><td>10 545 </td><td>6587 </td><td char=".">1.60 </td></tr><tr><td><em>Mesorhizobium loti</em> MAFF303099 </td><td>NC_002678 </td><td>62.7 </td><td>7 036 074 </td><td char=".">0.0070 </td><td>9388 </td><td>31 836 </td><td char=".">0.29 </td></tr><tr><td><em>Mycobacterium avium</em> k10 </td><td>NC_002944 </td><td>69.3 </td><td>4 829 781 </td><td char=".">0.0075 </td><td>7045 </td><td>32 224 </td><td char=".">0.22 </td></tr><tr><td><em>Mycobacterium bovis</em> AF2122/97 </td><td>NC_002945 </td><td>65.6 </td><td>4 345 492 </td><td char=".">0.0080 </td><td>6710 </td><td>23 254 </td><td char=".">0.29 </td></tr><tr><td><em>Mycobacterium leprae</em> TN </td><td>NC_002677 </td><td>57.8 </td><td>3 268 203 </td><td char=".">0.0095 </td><td>5925 </td><td>11 526 </td><td char=".">0.51 </td></tr><tr><td><em>Mycobacterium tuberculosis</em> H37Rv </td><td>NC_002755 </td><td>65.6 </td><td>4 411 532 </td><td char=".">0.0079 </td><td>6767 </td><td>23 608 </td><td char=".">0.29 </td></tr><tr><td><em>Mycoplasma gallisepticum</em> Rd KW20 </td><td>NC_004829 </td><td>31.5 </td><td>996 422 </td><td char=".">0.0518 </td><td>9347 </td><td>6338 </td><td char=".">1.47 </td></tr><tr><td><em>Mycoplasma genitalium</em> G-37 </td><td>NC_000908 </td><td>31.7 </td><td>580 074 </td><td char=".">0.0709 </td><td>7428 </td><td>3646 </td><td char=".">2.04 </td></tr><tr><td><em>Mycoplasma mobile</em> 163K </td><td>NC_006908 </td><td>25 </td><td>777 079 </td><td char=".">0.1034 </td><td>14 323 </td><td>7244 </td><td char=".">1.98 </td></tr><tr><td><em>Mycoplasma mycoides</em> PG1 </td><td>NC_005364 </td><td>24 </td><td>1 211 703 </td><td char=".">0.0914 </td><td>19 889 </td><td>11 956 </td><td char=".">1.66 </td></tr><tr><td><em>Mycoplasma penetrans</em> HF-2 </td><td>NC_004432 </td><td>25.7 </td><td>1 358 633 </td><td char=".">0.0764 </td><td>18 734 </td><td>12 166 </td><td char=".">1.54 </td></tr><tr><td><em>Mycoplasma pneumoniae</em> Ml29 </td><td>NC_000912 </td><td>40 </td><td>816 394 </td><td char=".">0.0432 </td><td>6584 </td><td>3195 </td><td char=".">2.06 </td></tr><tr><td><em>Mycoplasma pulmonis</em> UAB CTIP </td><td>NC_002771 </td><td>26.6 </td><td>963 879 </td><td char=".">0.1149 </td><td>19 740 </td><td>8193 </td><td char=".">2.41 </td></tr><tr><td><em>Neisseria gonorrhoeae</em> </td><td>Unpublished </td><td>52.7 </td><td>2 146 877 </td><td char=".">0.0326 </td><td>12 995 </td><td>6443 </td><td char=".">2.02 </td></tr><tr><td><em>Neisseria meningitidis</em> MC58 </td><td>NC_003112 </td><td>51.5 </td><td>2 272 351 </td><td char=".">0.0331 </td><td>14 001 </td><td>6707 </td><td char=".">2.09 </td></tr><tr><td><em>Neisseria meningitidis</em> Z2491 </td><td>NC_003116 </td><td>51.8 </td><td>2184 406 </td><td char=".">0.0328 </td><td>13 304 </td><td>6469 </td><td char=".">2.06 </td></tr><tr><td><em>Nitrosomonas europeae</em> ATCC 19718 </td><td>NC_004757 </td><td>50.7 </td><td>2 812094 </td><td char=".">0.0209 </td><td>10 934 </td><td>8252 </td><td char=".">1.33 </td></tr><tr><td><em>Nostoc</em> sp. PCC 7120 </td><td>NC_003272 </td><td>41.3 </td><td>6 413 771 </td><td char=".">0.0315 </td><td>37 738 </td><td>23 559 </td><td char=".">1.60 </td></tr><tr><td><em>Oceanobacillus iheyensis</em> HTE831 </td><td>NC_004193 </td><td>35.7 </td><td>3 630 528 </td><td char=".">0.0388 </td><td>25 862 </td><td>18 002 </td><td char=".">1.44 </td></tr><tr><td><em>Parachlamydia</em> sp. UWE25 </td><td>NC_005861 </td><td>34.7 </td><td>2 414 465 </td><td char=".">0.0649 </td><td>27 883 </td><td>12 694 </td><td char=".">2.20 </td></tr><tr><td><em>Pasteurella multocida</em> Pm70 </td><td>NC_002632 </td><td>40.4 </td><td>2 257 487 </td><td char=".">0.0417 </td><td>17 297 </td><td>8660 </td><td char=".">2.00 </td></tr><tr><td><em>Photorhabdus luminescens</em> TTO1 </td><td>NC_005126 </td><td>42.8 </td><td>5 688 987 </td><td char=".">0.0314 </td><td>32 937 </td><td>19 559 </td><td char=".">1.68 </td></tr><tr><td><em>Phytoplasma asteris</em> OY-M </td><td>NC_005303 </td><td>27.7 </td><td>860 631 </td><td char=".">0.1108 </td><td>16 757 </td><td>6860 </td><td char=".">2.44 </td></tr><tr><td><em>Pirelulla</em> sp. 1 </td><td>NC_005027 </td><td>55.4 </td><td>7 145 576 </td><td char=".">0.0101 </td><td>13 771 </td><td>22 974 </td><td char=".">0.60 </td></tr><tr><td><em>Porphyromonas gingivalis</em> W83 </td><td>NC_002950 </td><td>48.3 </td><td>2 343 476 </td><td char=".">0.0221 </td><td>9614 </td><td>6932 </td><td char=".">1.39 </td></tr><tr><td><em>Prochlorococcus marinus</em> MIT 9313 </td><td>NC_005071 </td><td>50.7 </td><td>2 410 873 </td><td char=".">0.0169 </td><td>7592 </td><td>7075 </td><td char=".">1.07 </td></tr><tr><td><em>Propionibacterium acnes</em> KPA171202 </td><td>NC_006085 </td><td>60 </td><td>2 560 265 </td><td char=".">0.0105 </td><td>5250 </td><td>10 021 </td><td char=".">0.52 </td></tr><tr><td><em>Pseudomonas aeruginosa</em> PAO1 </td><td>NC_002516 </td><td>66.6 </td><td>6 264 403 </td><td char=".">0.0063 </td><td>7472 </td><td>35 572 </td><td char=".">0.21 </td></tr><tr><td><em>Pseudomonas putida</em> KT2440 </td><td>NC_002947 </td><td>61.5 </td><td>6 181 863 </td><td char=".">0.0103 </td><td>12 079 </td><td>26 178 </td><td char=".">0.46 </td></tr><tr><td><em>Pseudomonas syringae</em> DC3000 </td><td>NC_004578 </td><td>58.4 </td><td>6 397 126 </td><td char=".">0.0125 </td><td>14 975 </td><td>23 174 </td><td char=".">0.65 </td></tr><tr><td><em>Ralstonia solanacearum</em> GMI1000 </td><td>NC_003295 </td><td>67 </td><td>5 810 922 </td><td char=".">0.0077 </td><td>8463 </td><td>33 793 </td><td char=".">0.25 </td></tr><tr><td><em>Rhodopseudomonas palustris</em> CGA009 </td><td>NC_005296 </td><td>65 </td><td>5 459 213 </td><td char=".">0.0043 </td><td>4511 </td><td>28 200 </td><td char=".">0.16 </td></tr><tr><td><em>Rickettsia conorii</em> Malish 7 </td><td>NC_003103 </td><td>32.4 </td><td>1 268 755 </td><td char=".">0.0525 </td><td>12 127 </td><td>7647 </td><td char=".">1.59 </td></tr><tr><td><em>Rickettsia prowazekii</em> Madrid E </td><td>NC_000963 </td><td>29 </td><td>1 111 523 </td><td char=".">0.0560 </td><td>11 282 </td><td>8206 </td><td char=".">1.37 </td></tr><tr><td><em>Rickettsia typhi</em> Wilmington </td><td>NC_006142 </td><td>28.3 </td><td>1 111 496 </td><td char=".">0.0552 </td><td>11 128 </td><td>8552 </td><td char=".">1.30 </td></tr><tr><td><em>Salmonella enterica</em> CT18 </td><td>NC_003198 </td><td>52.1 </td><td>4 809 037 </td><td char=".">0.0240 </td><td>21 454 </td><td>14 296 </td><td char=".">1.50 </td></tr><tr><td><em>Salmonella typhimurium</em> LT2 </td><td>NC_003197 </td><td>52.2 </td><td>4 857 432 </td><td char=".">0.0239 </td><td>21 602 </td><td>14 460 </td><td char=".">1.49 </td></tr><tr><td><em>Serratia marcescens</em> Dbl 1 </td><td>Unpublished </td><td>59.5 </td><td>5 113 802 </td><td char=".">0.0155 </td><td>14 776 </td><td>19 521 </td><td char=".">0.76 </td></tr><tr><td><em>Shewanella oneidensis</em> MR-1 </td><td>NC_004347 </td><td>46 </td><td>4 969 803 </td><td char=".">0.0249 </td><td>23 135 </td><td>15 337 </td><td char=".">1.51 </td></tr><tr><td><em>Shigella flexneri</em> 2457T </td><td>NC_004741 </td><td>50.9 </td><td>4 599 354 </td><td char=".">0.0222 </td><td>19 024 </td><td>13 511 </td><td char=".">1.41 </td></tr><tr><td><em>Sinorhizobium meliloti</em> 1021 </td><td>NC_003047 </td><td>62.7 </td><td>3 654 135 </td><td char=".">0.0075 </td><td>5228 </td><td>16 534 </td><td char=".">0.32 </td></tr><tr><td><em>Staphylococcus aureus</em> MW2 </td><td>NC_003923 </td><td>32.8 </td><td>2 820 462 </td><td char=".">0.0389 </td><td>20 991 </td><td>16 599 </td><td char=".">1.26 </td></tr><tr><td><em>Staphylococcus epidermidis</em> ATCC 12228 </td><td>NC_004461 </td><td>32.1 </td><td>2 499 279 </td><td char=".">0.0424 </td><td>19 544 </td><td>15 337 </td><td char=".">1.27 </td></tr><tr><td><em>Streptococcus agalactiae</em> 2603V/R </td><td>NC_004416 </td><td>35.7 </td><td>2 160 267 </td><td char=".">0.0446 </td><td>17 709 </td><td>10 712 </td><td char=".">1.65 </td></tr><tr><td><em>Streptococcus mutans</em> UA159 </td><td>NC_004350 </td><td>36.8 </td><td>2 030 921 </td><td char=".">0.0483 </td><td>18 109 </td><td>9453 </td><td char=".">1.92 </td></tr><tr><td><em>Streptococcus pneumoniae</em> TIGR4 </td><td>NC_003028 </td><td>39.7 </td><td>2 160 837 </td><td char=".">0.0374 </td><td>15 003 </td><td>8588 </td><td char=".">1.75 </td></tr><tr><td><em>Streptococcus pyogenes</em> MGAS315 </td><td>NC_004070 </td><td>38.6 </td><td>1 900 521 </td><td char=".">0.0453 </td><td>15 743 </td><td>8005 </td><td char=".">1.97 </td></tr><tr><td><em>Streptomyces avermitilis</em> MA-4680 </td><td>NC_003155 </td><td>70.7 </td><td>9 025 608 </td><td char=".">0.0125 </td><td>21 960 </td><td>65 454 </td><td char=".">0.34 </td></tr><tr><td><em>Streptomyces coelicolor</em> A3 (2) </td><td>NC_003888 </td><td>72.1 </td><td>8 667 507 </td><td char=".">0.0128 </td><td>21 627 </td><td>68 279 </td><td char=".">0.32 </td></tr><tr><td><em>Synechococcus</em> sp. WH8102 </td><td>NC_005070 </td><td>59.4 </td><td>2 434 428 </td><td char=".">0.0153 </td><td>7002 </td><td>9247 </td><td char=".">0.76 </td></tr><tr><td><em>Synechocystis</em> sp. PCC 6803 </td><td>NC_000911 </td><td>47.7 </td><td>3 573 470 </td><td char=".">0.0457 </td><td>30 442 </td><td>10 654 </td><td char=".">2.86 </td></tr><tr><td><em>Thermoanaerobacter tengcongensis</em> </td><td>NC_003869 </td><td>37.6 </td><td>2 689 445 </td><td char=".">0.0568 </td><td>28 261 </td><td>11 966 </td><td char=".">2.36 </td></tr><tr><td><em>Thermosynechococcus elongatus</em> BP-1 </td><td>NC_004113 </td><td>53.9 </td><td>2 593 857 </td><td char=".">0.0325 </td><td>15 850 </td><td>7985 </td><td char=".">1.99 </td></tr><tr><td><em>Thermotoga maritima</em> MSB 8 </td><td>NC_000853 </td><td>46.2 </td><td>1 860 725 </td><td char=".">0.0298 </td><td>10 394 </td><td>5714 </td><td char=".">1.82 </td></tr><tr><td><em>Thermus thermophilus</em> HB27 </td><td>NC_005835 </td><td>69.4 </td><td>1 894 877 </td><td char=".">0.0533 </td><td>18 935 </td><td>12 718 </td><td char=".">1.49 </td></tr><tr><td><em>Treponema denticola</em> ATCC 35405 </td><td>NC_002967 </td><td>37.9 </td><td>2 843 201 </td><td char=".">0.0737 </td><td>37 216 </td><td>12 442 </td><td char=".">2.99 </td></tr><tr><td><em>Treponema pallidum</em> Nichols </td><td>NC_000919 </td><td>52.8 </td><td>1 138 011 </td><td char=".">0.0320 </td><td>6669 </td><td>3421 </td><td char=".">1.95 </td></tr><tr><td><em>Tropheryma whipplei</em> TW08/27 </td><td>NC_004551 </td><td>46.3 </td><td>925 938 </td><td char=".">0.0286 </td><td>4942 </td><td>2837 </td><td char=".">1.74 </td></tr><tr><td><em>Ureaplasma parvum</em> ATCC 700970 </td><td>NC_002162 </td><td>25.5 </td><td>751 719 </td><td char=".">0.0833 </td><td>11 146 </td><td>6809 </td><td char=".">1.64 </td></tr><tr><td><em>Vibrio cholerae</em> Nl696l </td><td>NC_002505 </td><td>47.5 </td><td>4 033 464 </td><td char=".">0.0238 </td><td>17 969 </td><td>12 063 </td><td char=".">1.49 </td></tr><tr><td><em>Vibrio parahaemolyticus</em> RMID 2210633 </td><td>NC_004603 </td><td>45.4 </td><td>5 165 770 </td><td char=".">0.0228 </td><td>22 140 </td><td>16 203 </td><td char=".">1.37 </td></tr><tr><td><em>Vibrio vulnificus</em> CMCP6 </td><td>NC_004459 </td><td>46.7 </td><td>5 211 578 </td><td char=".">0.0240 </td><td>23 472 </td><td>15 823 </td><td char=".">1.48 </td></tr><tr><td><em>Wigglesworthia glossinidia</em> </td><td>NC_004344 </td><td>22.5 </td><td>697 721 </td><td char=".">0.1241 </td><td>14 619 </td><td>7488 </td><td char=".">1.95 </td></tr><tr><td><em>Wolinella succinogenes</em> DSM 1740 </td><td>NC_005090 </td><td>48.5 </td><td>2 110 355 </td><td char=".">0.0434 </td><td>17 143 </td><td>6229 </td><td char=".">2.75 </td></tr><tr><td><em>Xanthomonas axonopodis</em> 306 </td><td>NC_003919 </td><td>64.8 </td><td>5 175 554 </td><td char=".">0.0060 </td><td>5855 </td><td>26 423 </td><td char=".">0.22 </td></tr><tr><td><em>Xanthomonas campestris</em> ATCC 33913 </td><td>NC_003902 </td><td>65.1 </td><td>5 076 188 </td><td char=".">0.0058 </td><td>5558 </td><td>26 376 </td><td char=".">0.21 </td></tr><tr><td><em>Xylella fastidiosa</em> 9a5c </td><td>NC_002488 </td><td>52.7 </td><td>2 679 306 </td><td char=".">0.0187 </td><td>9276 </td><td>8040 </td><td char=".">1.15 </td></tr><tr><td><em>Yersinia enterocolitica</em> 8081 </td><td>Unpublished </td><td>43.9 </td><td>4 615 899 </td><td char=".">0.0240 </td><td>20 592 </td><td>15 205 </td><td char=".">1.35 </td></tr><tr><td><em>Yersinia pestis</em> KIM </td><td>NC_004088 </td><td>47.6 </td><td>4 600 755 </td><td char=".">0.0270 </td><td>22 980 </td><td>13 738 </td><td char=".">1.67 </td></tr></tbody></table></div><div content-id="tbl2" class="table-modal"><table><thead name="thead" align=""><tr><th>Strain designation </th><th>Accession no. </th><th>mol% GC </th><th>Genome size (bp) </th><th>Repeat density </th><th>Observed no. of repeats </th><th>Expected (E) no. of repeats </th><th>O/E </th></tr></thead><tbody><tr><td><em>Acinetobacter</em> sp. ADP1 </td><td>NC_005966 </td><td>40.4 </td><td>3 598 621 </td><td char=".">0.0366 </td><td>24 075 </td><td>13 805 </td><td char=".">1.74 </td></tr><tr><td><em>Agrobacterium tumefaciens</em> C58 </td><td>NC_003304 </td><td>59.3 </td><td>4 917 050 </td><td char=".">0.0120 </td><td>11 180 </td><td>18 587 </td><td char=".">0.60 </td></tr><tr><td><em>Aquifex aeolicus</em> VF5 </td><td>NC_000918 </td><td>43.5 </td><td>1 551 335 </td><td char=".">0.0374 </td><td>10 932 </td><td>5187 </td><td char=".">2.11 </td></tr><tr><td><em>Bacillus anthracis Ames</em> </td><td>NC_003997 </td><td>35.4 </td><td>5 227 293 </td><td char=".">0.0454 </td><td>43 728 </td><td>26 377 </td><td char=".">1.66 </td></tr><tr><td><em>Bacillus cereus</em> ATCC 14579 </td><td>NC_004722 </td><td>35.3 </td><td>5 411 809 </td><td char=".">0.0455 </td><td>45 299 </td><td>27 468 </td><td char=".">1.65 </td></tr><tr><td><em>Bacillus halodurans</em> C-125 </td><td>NC_002570 </td><td>43.7 </td><td>4 202 353 </td><td char=".">0.0367 </td><td>28 385 </td><td>13 945 </td><td char=".">2.04 </td></tr><tr><td><em>Bacillus licheniformis</em> ATCC 14580 </td><td>NC_006270 </td><td>46.2 </td><td>4 222 336 </td><td char=".">0.0438 </td><td>33 815 </td><td>12 966 </td><td char=".">2.61 </td></tr><tr><td><em>Bacillus subtilis</em> 168 </td><td>NC_000964 </td><td>43.5 </td><td>4 214 630 </td><td char=".">0.0423 </td><td>32 621 </td><td>14 092 </td><td char=".">2.31 </td></tr><tr><td><em>Bacillus thuringiensis</em> 97-27 </td><td>NC_005957 </td><td>35.4 </td><td>5 327 682 </td><td char=".">0.0450 </td><td>44 116 </td><td>26 884 </td><td char=".">1.64 </td></tr><tr><td><em>Bacteriovorax marinus</em> </td><td>Unpublished<a reveal-id="tf2-1" data-open="tf2-1" class="link link-ref link-reveal xref-fn">a</a> </td><td>36.7 </td><td>3 435 933 </td><td char=".">0.0278 </td><td>17 861 </td><td>16 084 </td><td char=".">1.11 </td></tr><tr><td><em>Bacteroides fragilis</em> NCTC 9343 </td><td>Unpublished </td><td>43.1 </td><td>5 205 140 </td><td char=".">0.0269 </td><td>25 896 </td><td>17 679 </td><td char=".">1.46 </td></tr><tr><td><em>Bacteroides thetaiotamicron</em> VPI-5482 </td><td>NC_004663 </td><td>42.8 </td><td>6 260 361 </td><td char=".">0.0237 </td><td>27 662 </td><td>21 523 </td><td char=".">1.29 </td></tr><tr><td><em>Bartonella henselae</em> Houston- 1 </td><td>NC_005956 </td><td>38.2 </td><td>1 931 047 </td><td char=".">0.0568 </td><td>19 800 </td><td>8312 </td><td char=".">2.38 </td></tr><tr><td><em>Bartonella quintana</em> Toulose </td><td>NC_005955 </td><td>38.8 </td><td>1 581 384 </td><td char=".">0.0545 </td><td>15 587 </td><td>6589 </td><td char=".">2.37 </td></tr><tr><td><em>Bdellovibrio bacteriovorus</em> HD100 </td><td>NC_005363 </td><td>50.6 </td><td>3 782 950 </td><td char=".">0.0298 </td><td>21 025 </td><td>11 096 </td><td char=".">1.89 </td></tr><tr><td><em>Bifidobacterium longum</em> NCC 2705 </td><td>NC_004307 </td><td>60.1 </td><td>2 256 646 </td><td char=".">0.0064 </td><td>2761 </td><td>8877 </td><td char=".">0.31 </td></tr><tr><td><em>Blochmannia floridanus</em> </td><td>NC_005061 </td><td>27.4 </td><td>705 557 </td><td char=".">0.0595 </td><td>7353 </td><td>5724 </td><td char=".">1.28 </td></tr><tr><td><em>Bordetella avium</em> </td><td>Unpublished </td><td>69.2 </td><td>3 732 255 </td><td char=".">0.0144 </td><td>9986 </td><td>24 753 </td><td char=".">0.40 </td></tr><tr><td><em>Bordetella bronchiseptica</em> RB50 </td><td>NC_002927 </td><td>68.1 </td><td>5 339 179 </td><td char=".">0.0070 </td><td>7041 </td><td>33 158 </td><td char=".">0.21 </td></tr><tr><td><em>Bordetella parapertussis</em> 12822 </td><td>NC_002928 </td><td>68.1 </td><td>4 773 551 </td><td char=".">0.0068 </td><td>6134 </td><td>29 645 </td><td char=".">0.21 </td></tr><tr><td><em>Bordetella pertussis</em> Tohamal </td><td>NC_002929 </td><td>67.7 </td><td>4 086 189 </td><td char=".">0.0074 </td><td>5684 </td><td>24 777 </td><td char=".">0.23 </td></tr><tr><td><em>Borrelia burgdorferi</em> B3 1 </td><td>NC_001318 </td><td>28.6 </td><td>910 724 </td><td char=".">0.0922 </td><td>14 886 </td><td>6884 </td><td char=".">2.16 </td></tr><tr><td><em>Borrelia garinii</em> </td><td>NC_006156 </td><td>28.3 </td><td>904 246 </td><td char=".">0.0915 </td><td>14 614 </td><td>6957 </td><td char=".">2.10 </td></tr><tr><td><em>Bradyrhizobium japonicum</em> USDA 110 </td><td>NC_004463 </td><td>64.1 </td><td>9 105 828 </td><td char=".">0.0060 </td><td>10 568 </td><td>44 631 </td><td char=".">0.24 </td></tr><tr><td><em>Brucella melitensis</em> 16M </td><td>NC_003317 </td><td>57.2 </td><td>3 294 931 </td><td char=".">0.0132 </td><td>8324 </td><td>11 328 </td><td char=".">0.73 </td></tr><tr><td><em>Brucella suis</em> 1330 </td><td>NC_004310 </td><td>57.2 </td><td>3 315 173 </td><td char=".">0.0132 </td><td>8337 </td><td>11 398 </td><td char=".">0.73 </td></tr><tr><td><em>Buchnera aphidicola</em> Bp </td><td>NC_004545 </td><td>25.3 </td><td>615 980 </td><td char=".">0.0946 </td><td>10 043 </td><td>5644 </td><td char=".">1.78 </td></tr><tr><td><em>Burkholderia cenocepacia</em> J2315 </td><td>Unpublished </td><td>67.0 </td><td>7 963 121 </td><td char=".">0.0051 </td><td>7659 </td><td>46 309 </td><td char=".">0.17 </td></tr><tr><td><em>Burkholderia mallei</em> ATCC 23344 </td><td>NC_006349 </td><td>68.3 </td><td>5 835 527 </td><td char=".">0.0056 </td><td>6154 </td><td>36 676 </td><td char=".">0.17 </td></tr><tr><td><em>Burkholderia pseudomallei</em> K96243 </td><td>NC_006350 </td><td>68.1 </td><td>7 247 547 </td><td char=".">0.0058 </td><td>7897 </td><td>45 009 </td><td char=".">0.18 </td></tr><tr><td><em>Campylobacter jejuni</em> NCTC 11168 </td><td>NC_002163 </td><td>30.5 </td><td>1 641 481 </td><td char=".">0.0898 </td><td>26 642 </td><td>11 083 </td><td char=".">2.40 </td></tr><tr><td><em>Caulobacter crescentus</em> CB15 </td><td>NC_002996 </td><td>67.2 </td><td>4 016 947 </td><td char=".">0.0086 </td><td>6681 </td><td>23 641 </td><td char=".">0.28 </td></tr><tr><td><em>Chlamydia muridarum</em> MoPn </td><td>NC_002620 </td><td>40.3 </td><td>1 072 950 </td><td char=".">0.0460 </td><td>8978 </td><td>4137 </td><td char=".">2.17 </td></tr><tr><td><em>Chlamydia trachomatis</em> D/UW-3/CX </td><td>NC_000117 </td><td>41.3 </td><td>1 072 950 </td><td char=".">0.0383 </td><td>7473 </td><td>3941 </td><td char=".">1.90 </td></tr><tr><td><em>Chlamydophila abortus</em> S26/3 </td><td>Unpublished </td><td>39.9 </td><td>1 144 377 </td><td char=".">0.0408 </td><td>8530 </td><td>4502 </td><td char=".">1.89 </td></tr><tr><td><em>Chlamydophila caviae</em> GPIC </td><td>NC_003361 </td><td>39.2 </td><td>1 173 390 </td><td char=".">0.0405 </td><td>8712 </td><td>4786 </td><td char=".">1.82 </td></tr><tr><td><em>Chlamydophila pneumoniae</em> AR39 </td><td>NC_002179 </td><td>40.6 </td><td>1 229 853 </td><td char=".">0.0400 </td><td>8972 </td><td>4672 </td><td char=".">1.92 </td></tr><tr><td><em>Chlorobium tepidum</em> TLS </td><td>NC_002932 </td><td>56.5 </td><td>2 154 946 </td><td char=".">0.0213 </td><td>8449 </td><td>7205 </td><td char=".">1.17 </td></tr><tr><td><em>Chromobacterium violaceum</em> ATCC 12472 </td><td>NC_005085 </td><td>64.8 </td><td>4 751 080 </td><td char=".">0.0088 </td><td>7808 </td><td>24 256 </td><td char=".">0.32 </td></tr><tr><td><em>Clostridium acetobutylicum</em> ATCC 824 </td><td>NC_003030 </td><td>30.9 </td><td>3 940 880 </td><td char=".">0.0542 </td><td>38 709 </td><td>25 981 </td><td char=".">1.49 </td></tr><tr><td><em>Clostridium botulinum</em> ATCC 3502 </td><td>Unpublished </td><td>28.2 </td><td>3 886 916 </td><td char=".">0.0639 </td><td>45 111 </td><td>30 083 </td><td char=".">1.50 </td></tr><tr><td><em>Clostridium difficile</em> 630 </td><td>Unpublished </td><td>29.1 </td><td>4 290 252 </td><td char=".">0.0570 </td><td>44 286 </td><td>31 485 </td><td char=".">1.41 </td></tr><tr><td><em>Clostridium perfringens</em> 13 </td><td>NC_003366 </td><td>28.6 </td><td>3 031 430 </td><td char=".">0.0558 </td><td>31 018 </td><td>22 915 </td><td char=".">1.35 </td></tr><tr><td><em>Clostridium tetani</em> E88 </td><td>NC_004557 </td><td>28.7 </td><td>2 799 251 </td><td char=".">0.0612 </td><td>31 149 </td><td>21 035 </td><td char=".">1.48 </td></tr><tr><td><em>Corynebacterium diphtheriae</em> NCTC 13129 </td><td>NC_002935 </td><td>53.5 </td><td>2 488 635 </td><td char=".">0.0171 </td><td>8230 </td><td>7589 </td><td char=".">1.08 </td></tr><tr><td><em>Corynebacterium efficiens</em> YS-314 </td><td>NC_004369 </td><td>63.1 </td><td>3 147 090 </td><td char=".">0.0150 </td><td>9230 </td><td>14 565 </td><td char=".">0.63 </td></tr><tr><td><em>Corynebacterium glutamicum</em> ATCC 13032 </td><td>NC_003450 </td><td>53.8 </td><td>3 309 401 </td><td char=".">0.0142 </td><td>9101 </td><td>10 163 </td><td char=".">0.90 </td></tr><tr><td><em>Coxiella burnettii</em> RSA 493 </td><td>NC_002971 </td><td>42.7 </td><td>1 995 275 </td><td char=".">0.0510 </td><td>18 486 </td><td>6888 </td><td char=".">2.68 </td></tr><tr><td><em>Deinococcus radiodurans</em> Rl </td><td>NC_001263 </td><td>66.9 </td><td>3 060 986 </td><td char=".">0.0202 </td><td>11 805 </td><td>17 695 </td><td char=".">0.67 </td></tr><tr><td><em>Desulfotalea psychrophila</em> LSv54 </td><td>NC_006138 </td><td>46.8 </td><td>3 523 383 </td><td char=".">0.0396 </td><td>25 157 </td><td>10 675 </td><td char=".">2.36 </td></tr><tr><td><em>Desulfovibrio vulgaris</em> Hildenborough </td><td>NC_002937 </td><td>63.1 </td><td>3 570 858 </td><td char=".">0.0156 </td><td>10 447 </td><td>16 526 </td><td char=".">0.63 </td></tr><tr><td><em>Enterococcus faecalis</em> V583 </td><td>NC_004668 </td><td>37.5 </td><td>3 218 031 </td><td char=".">0.0547 </td><td>31 958 </td><td>14 398 </td><td char=".">2.22 </td></tr><tr><td><em>Erwinia carotovora</em> SCR11043 </td><td>NC_004547 </td><td>51 </td><td>5 064 019 </td><td char=".">0.0213 </td><td>20 025 </td><td>14 885 </td><td char=".">1.35 </td></tr><tr><td><em>Escherichia coli</em> K12 </td><td>NC_000913 </td><td>50.8 </td><td>4 639 221 </td><td char=".">0.0222 </td><td>19 237 </td><td>13 620 </td><td char=".">1.41 </td></tr><tr><td><em>Fusobacterium nucleatum</em> ATCC 25586 </td><td>NC_003454 </td><td>27.1 </td><td>2 174 500 </td><td char=".">0.0809 </td><td>31 627 </td><td>17 952 </td><td char=".">1.76 </td></tr><tr><td><em>Geobacter sulfurreducens</em> PCA </td><td>NC_002939 </td><td>60.9 </td><td>3 814 139 </td><td char=".">0.0235 </td><td>16 741 </td><td>15 641 </td><td char=".">1.07 </td></tr><tr><td><em>Gloeobacter violaceus</em> PCC7421 </td><td>NC_005125 </td><td>62 </td><td>4 659 019 </td><td char=".">0.0238 </td><td>20 847 </td><td>20 275 </td><td char=".">1.03 </td></tr><tr><td><em>Haemophilus ducreyi</em> 35000HP </td><td>NC_002940 </td><td>38.2 </td><td>1 698 955 </td><td char=".">0.0410 </td><td>12 790 </td><td>7313 </td><td char=".">1.75 </td></tr><tr><td><em>Haemophilus influenzae</em> Rd KW20 </td><td>NC_000907 </td><td>38.1 </td><td>1 830 138 </td><td char=".">0.0459 </td><td>15 499 </td><td>7921 </td><td char=".">1.96 </td></tr><tr><td><em>Helicobacter hepaticus</em> ATCC 51449 </td><td>NC_004917 </td><td>35.9 </td><td>1 799 146 </td><td char=".">0.0546 </td><td>17 933 </td><td>8818 </td><td char=".">2.03 </td></tr><tr><td><em>Helicobacter pylori</em> 26695 </td><td>NC_000915 </td><td>38.9 </td><td>1 667 867 </td><td char=".">0.0793 </td><td>24 106 </td><td>6913 </td><td char=".">3.49 </td></tr><tr><td><em>Helicobacter pylori</em> J99 </td><td>NC_000921 </td><td>39.2 </td><td>1 643 831 </td><td char=".">0.0785 </td><td>23 551 </td><td>6705 </td><td char=".">3.51 </td></tr><tr><td><em>Lactobacillus johnsonii</em> NCC 533 </td><td>NC_005362 </td><td>34.6 </td><td>1 992 676 </td><td char=".">0.0450 </td><td>16 570 </td><td>10 538 </td><td char=".">1.57 </td></tr><tr><td><em>Lactobacillus plantarum</em> WCFS1 </td><td>NC_004567 </td><td>44.5 </td><td>3 308 274 </td><td char=".">0.0227 </td><td>14 021 </td><td>10 672 </td><td char=".">1.31 </td></tr><tr><td><em>Lactococcus lactis</em> IL1403 </td><td>NC_002662 </td><td>35.3 </td><td>2 365 589 </td><td char=".">0.0573 </td><td>24 747 </td><td>12 007 </td><td char=".">2.06 </td></tr><tr><td><em>Leifsonia xyli</em> CTCB07 </td><td>NC_006087 </td><td>67.7 </td><td>2 584 158 </td><td char=".">0.0100 </td><td>4989 </td><td>15 669 </td><td char=".">0.32 </td></tr><tr><td><em>Leptospira interrogans</em> Fiocruz LI-130 </td><td>NC_004342 </td><td>35 </td><td>4 277 185 </td><td char=".">0.0755 </td><td>58 065 </td><td>22 094 </td><td char=".">2.63 </td></tr><tr><td><em>Listeria innocua</em> Clip 11262 </td><td>NC_003212 </td><td>37.4 </td><td>3 011 208 </td><td char=".">0.0513 </td><td>28 276 </td><td>13 549 </td><td char=".">2.09 </td></tr><tr><td><em>Listeria monocytogenes</em> EGD-e </td><td>NC_003210 </td><td>38 </td><td>2 944 528 </td><td char=".">0.0496 </td><td>26 798 </td><td>12 814 </td><td char=".">2.09 </td></tr><tr><td><em>Mesoplasma florum</em> LI </td><td>NC_006055 </td><td>27 </td><td>793 224 </td><td char=".">0.0738 </td><td>10 545 </td><td>6587 </td><td char=".">1.60 </td></tr><tr><td><em>Mesorhizobium loti</em> MAFF303099 </td><td>NC_002678 </td><td>62.7 </td><td>7 036 074 </td><td char=".">0.0070 </td><td>9388 </td><td>31 836 </td><td char=".">0.29 </td></tr><tr><td><em>Mycobacterium avium</em> k10 </td><td>NC_002944 </td><td>69.3 </td><td>4 829 781 </td><td char=".">0.0075 </td><td>7045 </td><td>32 224 </td><td char=".">0.22 </td></tr><tr><td><em>Mycobacterium bovis</em> AF2122/97 </td><td>NC_002945 </td><td>65.6 </td><td>4 345 492 </td><td char=".">0.0080 </td><td>6710 </td><td>23 254 </td><td char=".">0.29 </td></tr><tr><td><em>Mycobacterium leprae</em> TN </td><td>NC_002677 </td><td>57.8 </td><td>3 268 203 </td><td char=".">0.0095 </td><td>5925 </td><td>11 526 </td><td char=".">0.51 </td></tr><tr><td><em>Mycobacterium tuberculosis</em> H37Rv </td><td>NC_002755 </td><td>65.6 </td><td>4 411 532 </td><td char=".">0.0079 </td><td>6767 </td><td>23 608 </td><td char=".">0.29 </td></tr><tr><td><em>Mycoplasma gallisepticum</em> Rd KW20 </td><td>NC_004829 </td><td>31.5 </td><td>996 422 </td><td char=".">0.0518 </td><td>9347 </td><td>6338 </td><td char=".">1.47 </td></tr><tr><td><em>Mycoplasma genitalium</em> G-37 </td><td>NC_000908 </td><td>31.7 </td><td>580 074 </td><td char=".">0.0709 </td><td>7428 </td><td>3646 </td><td char=".">2.04 </td></tr><tr><td><em>Mycoplasma mobile</em> 163K </td><td>NC_006908 </td><td>25 </td><td>777 079 </td><td char=".">0.1034 </td><td>14 323 </td><td>7244 </td><td char=".">1.98 </td></tr><tr><td><em>Mycoplasma mycoides</em> PG1 </td><td>NC_005364 </td><td>24 </td><td>1 211 703 </td><td char=".">0.0914 </td><td>19 889 </td><td>11 956 </td><td char=".">1.66 </td></tr><tr><td><em>Mycoplasma penetrans</em> HF-2 </td><td>NC_004432 </td><td>25.7 </td><td>1 358 633 </td><td char=".">0.0764 </td><td>18 734 </td><td>12 166 </td><td char=".">1.54 </td></tr><tr><td><em>Mycoplasma pneumoniae</em> Ml29 </td><td>NC_000912 </td><td>40 </td><td>816 394 </td><td char=".">0.0432 </td><td>6584 </td><td>3195 </td><td char=".">2.06 </td></tr><tr><td><em>Mycoplasma pulmonis</em> UAB CTIP </td><td>NC_002771 </td><td>26.6 </td><td>963 879 </td><td char=".">0.1149 </td><td>19 740 </td><td>8193 </td><td char=".">2.41 </td></tr><tr><td><em>Neisseria gonorrhoeae</em> </td><td>Unpublished </td><td>52.7 </td><td>2 146 877 </td><td char=".">0.0326 </td><td>12 995 </td><td>6443 </td><td char=".">2.02 </td></tr><tr><td><em>Neisseria meningitidis</em> MC58 </td><td>NC_003112 </td><td>51.5 </td><td>2 272 351 </td><td char=".">0.0331 </td><td>14 001 </td><td>6707 </td><td char=".">2.09 </td></tr><tr><td><em>Neisseria meningitidis</em> Z2491 </td><td>NC_003116 </td><td>51.8 </td><td>2184 406 </td><td char=".">0.0328 </td><td>13 304 </td><td>6469 </td><td char=".">2.06 </td></tr><tr><td><em>Nitrosomonas europeae</em> ATCC 19718 </td><td>NC_004757 </td><td>50.7 </td><td>2 812094 </td><td char=".">0.0209 </td><td>10 934 </td><td>8252 </td><td char=".">1.33 </td></tr><tr><td><em>Nostoc</em> sp. PCC 7120 </td><td>NC_003272 </td><td>41.3 </td><td>6 413 771 </td><td char=".">0.0315 </td><td>37 738 </td><td>23 559 </td><td char=".">1.60 </td></tr><tr><td><em>Oceanobacillus iheyensis</em> HTE831 </td><td>NC_004193 </td><td>35.7 </td><td>3 630 528 </td><td char=".">0.0388 </td><td>25 862 </td><td>18 002 </td><td char=".">1.44 </td></tr><tr><td><em>Parachlamydia</em> sp. UWE25 </td><td>NC_005861 </td><td>34.7 </td><td>2 414 465 </td><td char=".">0.0649 </td><td>27 883 </td><td>12 694 </td><td char=".">2.20 </td></tr><tr><td><em>Pasteurella multocida</em> Pm70 </td><td>NC_002632 </td><td>40.4 </td><td>2 257 487 </td><td char=".">0.0417 </td><td>17 297 </td><td>8660 </td><td char=".">2.00 </td></tr><tr><td><em>Photorhabdus luminescens</em> TTO1 </td><td>NC_005126 </td><td>42.8 </td><td>5 688 987 </td><td char=".">0.0314 </td><td>32 937 </td><td>19 559 </td><td char=".">1.68 </td></tr><tr><td><em>Phytoplasma asteris</em> OY-M </td><td>NC_005303 </td><td>27.7 </td><td>860 631 </td><td char=".">0.1108 </td><td>16 757 </td><td>6860 </td><td char=".">2.44 </td></tr><tr><td><em>Pirelulla</em> sp. 1 </td><td>NC_005027 </td><td>55.4 </td><td>7 145 576 </td><td char=".">0.0101 </td><td>13 771 </td><td>22 974 </td><td char=".">0.60 </td></tr><tr><td><em>Porphyromonas gingivalis</em> W83 </td><td>NC_002950 </td><td>48.3 </td><td>2 343 476 </td><td char=".">0.0221 </td><td>9614 </td><td>6932 </td><td char=".">1.39 </td></tr><tr><td><em>Prochlorococcus marinus</em> MIT 9313 </td><td>NC_005071 </td><td>50.7 </td><td>2 410 873 </td><td char=".">0.0169 </td><td>7592 </td><td>7075 </td><td char=".">1.07 </td></tr><tr><td><em>Propionibacterium acnes</em> KPA171202 </td><td>NC_006085 </td><td>60 </td><td>2 560 265 </td><td char=".">0.0105 </td><td>5250 </td><td>10 021 </td><td char=".">0.52 </td></tr><tr><td><em>Pseudomonas aeruginosa</em> PAO1 </td><td>NC_002516 </td><td>66.6 </td><td>6 264 403 </td><td char=".">0.0063 </td><td>7472 </td><td>35 572 </td><td char=".">0.21 </td></tr><tr><td><em>Pseudomonas putida</em> KT2440 </td><td>NC_002947 </td><td>61.5 </td><td>6 181 863 </td><td char=".">0.0103 </td><td>12 079 </td><td>26 178 </td><td char=".">0.46 </td></tr><tr><td><em>Pseudomonas syringae</em> DC3000 </td><td>NC_004578 </td><td>58.4 </td><td>6 397 126 </td><td char=".">0.0125 </td><td>14 975 </td><td>23 174 </td><td char=".">0.65 </td></tr><tr><td><em>Ralstonia solanacearum</em> GMI1000 </td><td>NC_003295 </td><td>67 </td><td>5 810 922 </td><td char=".">0.0077 </td><td>8463 </td><td>33 793 </td><td char=".">0.25 </td></tr><tr><td><em>Rhodopseudomonas palustris</em> CGA009 </td><td>NC_005296 </td><td>65 </td><td>5 459 213 </td><td char=".">0.0043 </td><td>4511 </td><td>28 200 </td><td char=".">0.16 </td></tr><tr><td><em>Rickettsia conorii</em> Malish 7 </td><td>NC_003103 </td><td>32.4 </td><td>1 268 755 </td><td char=".">0.0525 </td><td>12 127 </td><td>7647 </td><td char=".">1.59 </td></tr><tr><td><em>Rickettsia prowazekii</em> Madrid E </td><td>NC_000963 </td><td>29 </td><td>1 111 523 </td><td char=".">0.0560 </td><td>11 282 </td><td>8206 </td><td char=".">1.37 </td></tr><tr><td><em>Rickettsia typhi</em> Wilmington </td><td>NC_006142 </td><td>28.3 </td><td>1 111 496 </td><td char=".">0.0552 </td><td>11 128 </td><td>8552 </td><td char=".">1.30 </td></tr><tr><td><em>Salmonella enterica</em> CT18 </td><td>NC_003198 </td><td>52.1 </td><td>4 809 037 </td><td char=".">0.0240 </td><td>21 454 </td><td>14 296 </td><td char=".">1.50 </td></tr><tr><td><em>Salmonella typhimurium</em> LT2 </td><td>NC_003197 </td><td>52.2 </td><td>4 857 432 </td><td char=".">0.0239 </td><td>21 602 </td><td>14 460 </td><td char=".">1.49 </td></tr><tr><td><em>Serratia marcescens</em> Dbl 1 </td><td>Unpublished </td><td>59.5 </td><td>5 113 802 </td><td char=".">0.0155 </td><td>14 776 </td><td>19 521 </td><td char=".">0.76 </td></tr><tr><td><em>Shewanella oneidensis</em> MR-1 </td><td>NC_004347 </td><td>46 </td><td>4 969 803 </td><td char=".">0.0249 </td><td>23 135 </td><td>15 337 </td><td char=".">1.51 </td></tr><tr><td><em>Shigella flexneri</em> 2457T </td><td>NC_004741 </td><td>50.9 </td><td>4 599 354 </td><td char=".">0.0222 </td><td>19 024 </td><td>13 511 </td><td char=".">1.41 </td></tr><tr><td><em>Sinorhizobium meliloti</em> 1021 </td><td>NC_003047 </td><td>62.7 </td><td>3 654 135 </td><td char=".">0.0075 </td><td>5228 </td><td>16 534 </td><td char=".">0.32 </td></tr><tr><td><em>Staphylococcus aureus</em> MW2 </td><td>NC_003923 </td><td>32.8 </td><td>2 820 462 </td><td char=".">0.0389 </td><td>20 991 </td><td>16 599 </td><td char=".">1.26 </td></tr><tr><td><em>Staphylococcus epidermidis</em> ATCC 12228 </td><td>NC_004461 </td><td>32.1 </td><td>2 499 279 </td><td char=".">0.0424 </td><td>19 544 </td><td>15 337 </td><td char=".">1.27 </td></tr><tr><td><em>Streptococcus agalactiae</em> 2603V/R </td><td>NC_004416 </td><td>35.7 </td><td>2 160 267 </td><td char=".">0.0446 </td><td>17 709 </td><td>10 712 </td><td char=".">1.65 </td></tr><tr><td><em>Streptococcus mutans</em> UA159 </td><td>NC_004350 </td><td>36.8 </td><td>2 030 921 </td><td char=".">0.0483 </td><td>18 109 </td><td>9453 </td><td char=".">1.92 </td></tr><tr><td><em>Streptococcus pneumoniae</em> TIGR4 </td><td>NC_003028 </td><td>39.7 </td><td>2 160 837 </td><td char=".">0.0374 </td><td>15 003 </td><td>8588 </td><td char=".">1.75 </td></tr><tr><td><em>Streptococcus pyogenes</em> MGAS315 </td><td>NC_004070 </td><td>38.6 </td><td>1 900 521 </td><td char=".">0.0453 </td><td>15 743 </td><td>8005 </td><td char=".">1.97 </td></tr><tr><td><em>Streptomyces avermitilis</em> MA-4680 </td><td>NC_003155 </td><td>70.7 </td><td>9 025 608 </td><td char=".">0.0125 </td><td>21 960 </td><td>65 454 </td><td char=".">0.34 </td></tr><tr><td><em>Streptomyces coelicolor</em> A3 (2) </td><td>NC_003888 </td><td>72.1 </td><td>8 667 507 </td><td char=".">0.0128 </td><td>21 627 </td><td>68 279 </td><td char=".">0.32 </td></tr><tr><td><em>Synechococcus</em> sp. WH8102 </td><td>NC_005070 </td><td>59.4 </td><td>2 434 428 </td><td char=".">0.0153 </td><td>7002 </td><td>9247 </td><td char=".">0.76 </td></tr><tr><td><em>Synechocystis</em> sp. PCC 6803 </td><td>NC_000911 </td><td>47.7 </td><td>3 573 470 </td><td char=".">0.0457 </td><td>30 442 </td><td>10 654 </td><td char=".">2.86 </td></tr><tr><td><em>Thermoanaerobacter tengcongensis</em> </td><td>NC_003869 </td><td>37.6 </td><td>2 689 445 </td><td char=".">0.0568 </td><td>28 261 </td><td>11 966 </td><td char=".">2.36 </td></tr><tr><td><em>Thermosynechococcus elongatus</em> BP-1 </td><td>NC_004113 </td><td>53.9 </td><td>2 593 857 </td><td char=".">0.0325 </td><td>15 850 </td><td>7985 </td><td char=".">1.99 </td></tr><tr><td><em>Thermotoga maritima</em> MSB 8 </td><td>NC_000853 </td><td>46.2 </td><td>1 860 725 </td><td char=".">0.0298 </td><td>10 394 </td><td>5714 </td><td char=".">1.82 </td></tr><tr><td><em>Thermus thermophilus</em> HB27 </td><td>NC_005835 </td><td>69.4 </td><td>1 894 877 </td><td char=".">0.0533 </td><td>18 935 </td><td>12 718 </td><td char=".">1.49 </td></tr><tr><td><em>Treponema denticola</em> ATCC 35405 </td><td>NC_002967 </td><td>37.9 </td><td>2 843 201 </td><td char=".">0.0737 </td><td>37 216 </td><td>12 442 </td><td char=".">2.99 </td></tr><tr><td><em>Treponema pallidum</em> Nichols </td><td>NC_000919 </td><td>52.8 </td><td>1 138 011 </td><td char=".">0.0320 </td><td>6669 </td><td>3421 </td><td char=".">1.95 </td></tr><tr><td><em>Tropheryma whipplei</em> TW08/27 </td><td>NC_004551 </td><td>46.3 </td><td>925 938 </td><td char=".">0.0286 </td><td>4942 </td><td>2837 </td><td char=".">1.74 </td></tr><tr><td><em>Ureaplasma parvum</em> ATCC 700970 </td><td>NC_002162 </td><td>25.5 </td><td>751 719 </td><td char=".">0.0833 </td><td>11 146 </td><td>6809 </td><td char=".">1.64 </td></tr><tr><td><em>Vibrio cholerae</em> Nl696l </td><td>NC_002505 </td><td>47.5 </td><td>4 033 464 </td><td char=".">0.0238 </td><td>17 969 </td><td>12 063 </td><td char=".">1.49 </td></tr><tr><td><em>Vibrio parahaemolyticus</em> RMID 2210633 </td><td>NC_004603 </td><td>45.4 </td><td>5 165 770 </td><td char=".">0.0228 </td><td>22 140 </td><td>16 203 </td><td char=".">1.37 </td></tr><tr><td><em>Vibrio vulnificus</em> CMCP6 </td><td>NC_004459 </td><td>46.7 </td><td>5 211 578 </td><td char=".">0.0240 </td><td>23 472 </td><td>15 823 </td><td char=".">1.48 </td></tr><tr><td><em>Wigglesworthia glossinidia</em> </td><td>NC_004344 </td><td>22.5 </td><td>697 721 </td><td char=".">0.1241 </td><td>14 619 </td><td>7488 </td><td char=".">1.95 </td></tr><tr><td><em>Wolinella succinogenes</em> DSM 1740 </td><td>NC_005090 </td><td>48.5 </td><td>2 110 355 </td><td char=".">0.0434 </td><td>17 143 </td><td>6229 </td><td char=".">2.75 </td></tr><tr><td><em>Xanthomonas axonopodis</em> 306 </td><td>NC_003919 </td><td>64.8 </td><td>5 175 554 </td><td char=".">0.0060 </td><td>5855 </td><td>26 423 </td><td char=".">0.22 </td></tr><tr><td><em>Xanthomonas campestris</em> ATCC 33913 </td><td>NC_003902 </td><td>65.1 </td><td>5 076 188 </td><td char=".">0.0058 </td><td>5558 </td><td>26 376 </td><td char=".">0.21 </td></tr><tr><td><em>Xylella fastidiosa</em> 9a5c </td><td>NC_002488 </td><td>52.7 </td><td>2 679 306 </td><td char=".">0.0187 </td><td>9276 </td><td>8040 </td><td char=".">1.15 </td></tr><tr><td><em>Yersinia enterocolitica</em> 8081 </td><td>Unpublished </td><td>43.9 </td><td>4 615 899 </td><td char=".">0.0240 </td><td>20 592 </td><td>15 205 </td><td char=".">1.35 </td></tr><tr><td><em>Yersinia pestis</em> KIM </td><td>NC_004088 </td><td>47.6 </td><td>4 600 755 </td><td char=".">0.0270 </td><td>22 980 </td><td>13 738 </td><td char=".">1.67 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Repeat density is shown as the percentage of the genome that is located in mononucleotide repeats &gt;5 nt.</p></div></div><div content-id="tf2-1"><div class="fn"><p><sup>a</sup>Unpublished genome sequences were produced by the Pathogen Sequencing Group at the Sanger Institute (<a class="link link-uri" href="http://www.sanger.ac.uk/Projects/Microbes/" target="_blank"></a>) (<em>Bacteriovorax marinus</em>, <em>Bacteroides fragilis</em>, <em>Bordetella avium</em>, <em>Burkholderia cenocepacia</em>, <em>Chlamydophila abortus</em>, <em>Clostridium botulinum</em>, <em>Clostridium difficile</em>, <em>Serratia marcescens</em> and <em>Yersinia enterocolitica</em>) or the University of Oklahoma (<a class="link link-uri" href="http://www.genome.ou.edu/gono.html" target="_blank"></a>) (<em>Neisseria gonorrhoeae</em>).</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/10424235" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=10424236 id="10424236" class="section-title" >2.2. Determination of the number of mononucleotide SSRs</h3>
<p>We used the software developed by Gur-Arie et al.<a reveal-id="b24" data-open="b24" class="link link-ref link-reveal xref-bibr">24</a> to screen the entire genome of all organisms listed in <a reveal-id="tbl1" data-open="tbl1" class="link link-reveal link-table xref-fig">Tables 1</a> and <a reveal-id="tbl2" data-open="tbl2" class="link link-reveal link-table xref-fig">2</a> for mononucleotide SSRs with a length between 5 and 13 bp and a minimal number of three repeats. This range was based on (i) the fact that SSRs &gt;3 nt can engage in illegitimate recombination, (ii) the significant threshold for the length of SSR elements reported by Rocha et al.<a reveal-id="b10" data-open="b10" class="link link-ref link-reveal xref-bibr">10</a> and (iii) the previous observations that larger repeats are rare in prokaryotes.<a reveal-id="b24  b26" data-open="b24  b26" class="link link-ref link-reveal xref-bibr">24–26</a> In addition, when we noticed that mononucleotide repeats &gt;13 nt were present, these were characterized as well. To determine the distribution of SSRs between coding and non-coding regions of the genome, all coding regions were extracted from the corresponding GenBank files and analysed separately.</p>                            <h3 scrollto-destination=10424238 id="10424238" class="section-title" >2.3. Calculation of the expected number of mononucleotide SSRs</h3>
<p>The expected number of homo-oligomer tracts of <em>t</em> bases in a sequence of length N was calculated as given by De Wachter<a reveal-id="b27" data-open="b27" class="link link-ref link-reveal xref-bibr">27</a>: <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>τ</mi><mrow><mn>1</mn><mo>,</mo><mi>t</mi></mrow></msub><mo>=</mo><mo>∑</mo><mrow><msubsup><mi>p</mi><mi>i</mi><mi>t</mi></msubsup><mo>(</mo><mn>1</mn><mo>−</mo><msub><mi>p</mi><mi>i</mi></msub><mo>)</mo></mrow><mo>×</mo><mo>[</mo><mo>(</mo><mi>N</mi><mo>−</mo><mi>t</mi><mo>−</mo><mn>1</mn><mo>)</mo><mo>(</mo><mn>1</mn><mo>−</mo><msub><mi>p</mi><mi>i</mi></msub><mo>)</mo><mo>+</mo><mn>2</mn><mo>]</mo></mrow></math></span> (summed over <em>i</em> = 1 − 4), with <em>p<sub>i</sub></em> being the frequency of each base in the sequence. Considering the large number of genomes involved in this study, we did not perform permutation tests (i.e. did not count the number of mononucleotide SSRs in permutated genomes in order to assess statistical significance) but assessed deviations from the expected number of mononucleotide SSRs by representation plots.<a reveal-id="b6" data-open="b6" class="link link-ref link-reveal xref-bibr">6</a></p>                            <h3 scrollto-destination=10424240 id="10424240" class="section-title" >2.4. Statistical analysis</h3>
<p>All statistical analyses were carried out using the SPSS 11.0.1. software (SPSS Inc.). Data were compared using one-way analysis of variance (ANOVA) by applying Levene's test of homogeneity of variance. When appropriate for paired samples, <em>t</em>-tests were used. The correlation coefficients calculated included the Pearson correlation coefficient (<em>r</em>) and the non-parametric Spearman's (ρ) and Kendall's (<em>T</em>) correlation coefficients. Whenever appropriate, we also calculated partial correlation coefficients (i.e. the correlation that remains between two variables after removing the correlation that is due to their mutual association with another variable). We calculated the correlation between the number of mononculeotide SSRs, the mononucleotide SSR density, the ratio (<em>R</em>) between the observed (<em>O</em>) and expected (<em>E</em>) number of mononucleotide repeats and a number of other features of the genomes. These features included genome size, GC content, coding density, fraction of the genome located in duplicated segments and fraction of the genome located in horizontally transferred segments. The genome size, the GC content and the coding density were determined using Artemis 4.0.<a reveal-id="b28" data-open="b28" class="link link-ref link-reveal xref-bibr">28</a> The number of putatively duplicated or horizontally transferred genes for each genome were taken from data published by Gevers et al.<a reveal-id="b29" data-open="b29" class="link link-ref link-reveal xref-bibr">29</a> and Nakamura et al.,<a reveal-id="b30" data-open="b30" class="link link-ref link-reveal xref-bibr">30</a> respectively.</p>                            <h2 scrollto-destination=10424242 id="10424242" class="section-title" >3. Results</h2>
                            <h3 scrollto-destination=10424243 id="10424243" class="section-title" >3.1. Number and density of mononucleotide SSRs in prokaryotic genomes</h3>
<p>An overview of the total number of mononucleotide SSRs observed and the repeat density (expressed as percentage of the genome that is occupied by mononucleotide repeats) for archaeal and bacterial genomes are shown in <a reveal-id="tbl1" data-open="tbl1" class="link link-reveal link-table xref-fig">Tables 1</a> and <a reveal-id="tbl2" data-open="tbl2" class="link link-reveal link-table xref-fig">2</a>, respectively. For archaeal genomes, the number of mononucleotide SSRs with repeat size &gt;5 nt ranged from 3797 (<em>Methanopyrus kandleri</em>) to 19 856 (<em>Methanosarcina acetivorans</em>). For bacterial genomes, the number of mononucleotide SSRs with repeat size &gt;5 nt ranged from 2761 (<em>Bifidobaterium longum</em>) to 58 065 (<em>Leptospira interrogans</em>). The mononucleotide SSR density ranged from 0.0114% (<em>Halobacterium</em> sp.) to 0.0741% (<em>Methanococcus maripaludis</em>) for archaea, and from 0.0043% (<em>Rhodopseudomonas palustris</em>) to 0.1241% (<em>Wigglesworthia glossinidia</em>) for bacteria. There were no significant differences between archaea and bacteria with regard to number of SSRs (<em>P</em> = 0.211) or SSR density (<em>P</em> = 0.592).</p>                            <h3 scrollto-destination=10424245 id="10424245" class="section-title" >3.2. Length distribution of mononucleotide SSRs</h3>
<p>The distribution of mononucleotide SSRs over different length categories was approximately the same in all prokaryotic genomes, with an almost linear or slightly sigmoid relationship between the logio of the number of repeats in a particular class and the size of the repeat unit of that class (<a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1</a>). Deviations from this general model were most obvious for longer repeats, but this is most likely due to the small number of these larger repeats (data not shown). Complete tables with the exact number of observed and expected number of mononucleotide SSRs for each class are available as Supplementary Data (Table S1).</p>                        <a id="10424249" scrollto-destination="10424249"></a>
<div id="fig1" data-id="fig1" class="fig fig-section"><div class="label fig-label"><strong>Figure 1</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/4/10.1093/dnares/dsi009/2/m_dsi009f1.jpeg?Expires=2147483647&Signature=3zvdZwOW2-yX7RyKooseopcPlHGu9PokhfCSaNDoX~qYdMSAbx5ec1i3ys1ucrXrOwzApvu3exRGYW0L4M1dDTU8XZRD2CQZ9Ewe0PWrYRigHg3fLqT558sJ-L79hir3Mc56NKWuAhtAklXnvXO8KD8kdhkNQ8vKuQFocLH97ZkDOJjVtO-KyPOiTCyizDhX8L9G23G~dx0CfBjwO8C7EyAvNh1ObWX3qTw6lLiKEoYW4uaPANWkhxFUTRq05FdMNNCIn1L6warErNQkJnonM0psCqMZSluarh~TQpisnhomqLjrGeqvrUs9sT-N4-4USLXmrOTN5~rwPygRX2TjWA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Correlation between the number of mononucleotide SSRs in a given class and the repeat size." path-from-xml="dsi009f1.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/10424249/dsi009f1.jpeg" path-from-xml="dsi009f1.jpeg" target="_blank">View large</a><a section="10424249" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/4/10.1093/dnares/dsi009/2/dsi009f1.jpeg?Expires=2147483647&Signature=dtT-P9pFBq4bwALvvLakQLz9tO44CjsOnzM7yOVk~EaxsgfCYXUr1GFTvst0HdJLuNTRyq4DQJtGJQRuSB0UI5CvZXigxrS8pAHjS2A4Oxt0BarmVveq-cTN6vDif1EDdxpRJSh8e3T99Wrk9q4Bs-jvqnyQVFceM3KG64rRXXeOwf1C0mwqeqV5rM2GqqWFrEIzJ-rNOY4Q~EFb~tu0XdwK~c3urAfNrirZ8GspfdbkOkhPOHvbdNAV352KZooo3lmTVvnjWkPP8T0oFOUuXp5Cd5ut-Nf5S2l9DKZjYsOieUkRYdxoKTjGZSBDLr05J40eztEXHxRoRys5wN73vg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=10424249&ar=621324&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi009f1.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Correlation between the number of mononucleotide SSRs in a given class and the repeat size.</p></div></div><div content-id="fig1" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 1</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/4/10.1093/dnares/dsi009/2/m_dsi009f1.jpeg?Expires=2147483647&Signature=3zvdZwOW2-yX7RyKooseopcPlHGu9PokhfCSaNDoX~qYdMSAbx5ec1i3ys1ucrXrOwzApvu3exRGYW0L4M1dDTU8XZRD2CQZ9Ewe0PWrYRigHg3fLqT558sJ-L79hir3Mc56NKWuAhtAklXnvXO8KD8kdhkNQ8vKuQFocLH97ZkDOJjVtO-KyPOiTCyizDhX8L9G23G~dx0CfBjwO8C7EyAvNh1ObWX3qTw6lLiKEoYW4uaPANWkhxFUTRq05FdMNNCIn1L6warErNQkJnonM0psCqMZSluarh~TQpisnhomqLjrGeqvrUs9sT-N4-4USLXmrOTN5~rwPygRX2TjWA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Correlation between the number of mononucleotide SSRs in a given class and the repeat size." path-from-xml="dsi009f1.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/10424249/dsi009f1.jpeg" path-from-xml="dsi009f1.jpeg" target="_blank">View large</a><a section="10424249" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/4/10.1093/dnares/dsi009/2/dsi009f1.jpeg?Expires=2147483647&Signature=dtT-P9pFBq4bwALvvLakQLz9tO44CjsOnzM7yOVk~EaxsgfCYXUr1GFTvst0HdJLuNTRyq4DQJtGJQRuSB0UI5CvZXigxrS8pAHjS2A4Oxt0BarmVveq-cTN6vDif1EDdxpRJSh8e3T99Wrk9q4Bs-jvqnyQVFceM3KG64rRXXeOwf1C0mwqeqV5rM2GqqWFrEIzJ-rNOY4Q~EFb~tu0XdwK~c3urAfNrirZ8GspfdbkOkhPOHvbdNAV352KZooo3lmTVvnjWkPP8T0oFOUuXp5Cd5ut-Nf5S2l9DKZjYsOieUkRYdxoKTjGZSBDLr05J40eztEXHxRoRys5wN73vg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=10424249&ar=621324&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi009f1.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Correlation between the number of mononucleotide SSRs in a given class and the repeat size.</p></div></div>                            <h3 scrollto-destination=10424253 id="10424253" class="section-title" >3.3. Distribution of mononucleotide SSRs over coding and non-coding regions</h3>
<p>No clear general trend can be observed in the distribution of mononucleotide SSRs over coding and non-coding regions in archaeal (Fig. S1, available in Supplementary Data) and bacterial (<a reveal-id="fig2" data-open="fig2" class="link link-reveal link-table xref-fig">Fig. 2</a>) genomes. Nevertheless, there appears to be a tendency in the majority of the genomes to exclude longer mononucleotide SSRs from coding regions. Notable exceptions include the <em>ɛ-Proteobacteria</em> (including <em>Helicobacter pylori</em> and <em>Campylobacter jejuni</em>), <em>Haemophilus ducreyi, Neisseria meningitidis</em> and <em>Synechocystis</em> sp., in which mononucleotide SSRs are almost exclusively located in coding regions, irrespective of length. The opposite is true for the genome of the [β-proteobacterium <em>Chromobacterium violaceum</em>: in this genome mononucleotide SSRs are almost exclusively located in non-coding regions, irrespective of the length of the repeat (<a reveal-id="fig2" data-open="fig2" class="link link-reveal link-table xref-fig">Fig. 2</a>)].</p>                        <a id="10424257" scrollto-destination="10424257"></a>
<div id="fig2" data-id="fig2" class="fig fig-section"><div class="label fig-label"><strong>Figure 2</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/4/10.1093/dnares/dsi009/2/m_dsi009f2.jpeg?Expires=2147483647&Signature=UP7tA6KLYj~ZBonv~cjYcQG60y2r6NOW4rH5E8BaKWKYJZGZm4IVBizowa4CPsDwqvEDNu07uS5mEP7neOxLRbo9R3LNocPKzZvLJLIB5IiY4h2h~Fn-~o4ZKJR8LC8d5vscxItXbFS7lMMMkzIzZj0JuNBM~HfTUXzUSkBM4pRUkIIu5L-lW3a0rdneTL-rJSO8~6eSMMCElh~LFRgFzYoqtYGr0qx8pZIPvlJPM7liNqiGNMH72GrzkaQNNvLoWFJ1KS91KeMHd~B6IhaUB719C7wZkFArQYME~YLvmTNsnGlonsJwao~N1IUB8CzH7si2Z6SkanDqVpQG10TyDw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Relationship between the size of the repeat unit and the % of mononucleotide SSRs found in coding regions in selected genomes of bacteria." path-from-xml="dsi009f2.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/10424257/dsi009f2.jpeg" path-from-xml="dsi009f2.jpeg" target="_blank">View large</a><a section="10424257" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/4/10.1093/dnares/dsi009/2/dsi009f2.jpeg?Expires=2147483647&Signature=UgsykYC9uJzuNeMkJbkxqalME3QD7B6VgU8AK3g2piIlHwJyIxLT3wLy0HobmlfiFXlNKwSqo~8S-Lnrp6qR4OLRMjWYztxvaI4u5bN116YKvHFqNv0Otva8pXxuX7Pg82PT~nFxLkbMWSTtmTX6tph-qBNUMFWlXcHNvhNqXTmODRnSQ5dAzQ~fu73DOPNLPs-fYR99TaAb7Y45MbcfrYxHMrDqRagbwdAqxqWDha2Yd5s9KNVten8npYbz-bdEevXjPH60LVoPnxlWCBba6cuRmZ1ESfKSLry26zhKvhTWVjwB0P62sYpYDeXZBkD7uUJ4KCmVV~1ekTmGBbs6Qw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=10424257&ar=621324&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi009f2.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Relationship between the size of the repeat unit and the % of mononucleotide SSRs found in coding regions in selected genomes of bacteria.</p></div></div><div content-id="fig2" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 2</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/4/10.1093/dnares/dsi009/2/m_dsi009f2.jpeg?Expires=2147483647&Signature=UP7tA6KLYj~ZBonv~cjYcQG60y2r6NOW4rH5E8BaKWKYJZGZm4IVBizowa4CPsDwqvEDNu07uS5mEP7neOxLRbo9R3LNocPKzZvLJLIB5IiY4h2h~Fn-~o4ZKJR8LC8d5vscxItXbFS7lMMMkzIzZj0JuNBM~HfTUXzUSkBM4pRUkIIu5L-lW3a0rdneTL-rJSO8~6eSMMCElh~LFRgFzYoqtYGr0qx8pZIPvlJPM7liNqiGNMH72GrzkaQNNvLoWFJ1KS91KeMHd~B6IhaUB719C7wZkFArQYME~YLvmTNsnGlonsJwao~N1IUB8CzH7si2Z6SkanDqVpQG10TyDw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Relationship between the size of the repeat unit and the % of mononucleotide SSRs found in coding regions in selected genomes of bacteria." path-from-xml="dsi009f2.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/10424257/dsi009f2.jpeg" path-from-xml="dsi009f2.jpeg" target="_blank">View large</a><a section="10424257" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/4/10.1093/dnares/dsi009/2/dsi009f2.jpeg?Expires=2147483647&Signature=UgsykYC9uJzuNeMkJbkxqalME3QD7B6VgU8AK3g2piIlHwJyIxLT3wLy0HobmlfiFXlNKwSqo~8S-Lnrp6qR4OLRMjWYztxvaI4u5bN116YKvHFqNv0Otva8pXxuX7Pg82PT~nFxLkbMWSTtmTX6tph-qBNUMFWlXcHNvhNqXTmODRnSQ5dAzQ~fu73DOPNLPs-fYR99TaAb7Y45MbcfrYxHMrDqRagbwdAqxqWDha2Yd5s9KNVten8npYbz-bdEevXjPH60LVoPnxlWCBba6cuRmZ1ESfKSLry26zhKvhTWVjwB0P62sYpYDeXZBkD7uUJ4KCmVV~1ekTmGBbs6Qw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=10424257&ar=621324&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi009f2.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Relationship between the size of the repeat unit and the % of mononucleotide SSRs found in coding regions in selected genomes of bacteria.</p></div></div>                            <h3 scrollto-destination=10424260 id="10424260" class="section-title" >3.4. Comparison between observed and expected number of mononucleotide repeats</h3>
<p>When we compared the total number of mononucleotide repeat observed with the total number expected based on the equation proposed by De Wachter<a reveal-id="b27" data-open="b27" class="link link-ref link-reveal xref-bibr">27</a>, we found that the ratio of the observed number of repeats/expected number of repeats (<em>R</em>) was highly variable. In archaeal genomes, <em>R</em> ranged from 0.36 (<em>Halobacterium</em> sp.) to 2.42 (<em>Methanosarcina acetivorans</em>) (<a reveal-id="tbl1" data-open="tbl1" class="link link-reveal link-table xref-fig">Table 1</a>). In bacterial genomes, <em>R</em> ranged from 0.16 (<em>Rhodopseudomonas palustris</em>) to 3.51 (<em>Helicobacter pylori</em> J99) (<a reveal-id="tbl2" data-open="tbl2" class="link link-reveal link-table xref-fig">Table 2</a>). There was no significant difference in <em>R</em> values between archaea and bacteria (<em>P</em> = 0.686). Representation plots for mononucleotide SSRs in selected archaeal and bacterial genomes are shown in <a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3</a>. We also observed that overall under- or overrepresentation does not necessarily mean that all types of mononucleotide SSRs are over- or underrepresented. This is illustrated by the representation plots for <em>Helicobacter pylori</em> J99 and <em>Methanosarcina acetivorans</em>: the total number of mononucleotide SSRs is much higher than expected by chance, but this is almost entirely due to the overrepresentation of smaller (&lt;9 nt) repeats, as larger repeats are generally underrepresented or present in numbers that are close to the expected numbers (<a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3</a>). An overview of the expected number of repeats for each individual repeat class, as well as the <em>R</em>-value for each repeat class is presented in Table S1.</p>                        <a id="10424264" scrollto-destination="10424264"></a>
<div id="fig3" data-id="fig3" class="fig fig-section"><div class="label fig-label"><strong>Figure 3</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/4/10.1093/dnares/dsi009/2/m_dsi009f3.jpeg?Expires=2147483647&Signature=oNCGhgMOh1P4AmXS1tbcyLr2pF423BkdujmzgUSDZOTQe0BxsbSfjj5aKT4MfZHH9Hz69JvddwEuQlwH0fr8juURr~TPL8jNEVmUuy7DZ2kS5zchToLiy-6qTaaqFvR494WZMOIQy4Bo-WwbmK9V2ki7t2R9kpa9V8ZfnCxcx1zDoUUgYCN8HZC7qVGknzMNabb2LTaannQCHVtWVhjI2MqEz9Ei4PRAZwvkAO7sSDVe4WrCSDF~neOrRKx~GMgtWDQCXt2IbMFB7hwL82uhctGbQaVINLY5t7212yb43h~C0-dtljDuCoPEeynG8oZBRny9zUug9Gq-Jhxywi4vig__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Representation plots for mononucleotide SSRs (size between 5 and 13 nt) in selected archaeal and bacterial genomes. x-axis: size of the repeat, y-axis: number of repeats observed and number of repeats expected." path-from-xml="dsi009f3.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/10424264/dsi009f3.jpeg" path-from-xml="dsi009f3.jpeg" target="_blank">View large</a><a section="10424264" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/4/10.1093/dnares/dsi009/2/dsi009f3.jpeg?Expires=2147483647&Signature=Zz6EGt6TCPISejG~HQvz4Lfcd8dizuLjkKc5UUZCvxW4TibrY03KGg5-B53nCfNgmsUvTGWZz6raXDnsYU-4PrVA-GH0EjS0Dpb65FhQffDwBFYatUvPFwbAhs-~9bULhmFVz30rNenopbVN70NuH9zo70QnyKx7gFxBes7rZz2Amz8m1ndEs6YnkVXKnl4TG8D2YzaQzBs7YYLZMBadGoEXhhuzXCFxbM19ffvEfmG6l-W7opNy9sTOmEx0qtx33slLsx7aiEuC-rP4pnc~vfX3HAT2s6YdHIdFKes6UzophnyDiHtJ~kbfOkhWPgghtUbuwLTfoN6t~~kFGg7PiA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=10424264&ar=621324&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi009f3.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Representation plots for mononucleotide SSRs (size between 5 and 13 nt) in selected archaeal and bacterial genomes. <em>x</em>-axis: size of the repeat, <em>y</em>-axis: number of repeats observed and number of repeats expected.</p></div></div><div content-id="fig3" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 3</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/4/10.1093/dnares/dsi009/2/m_dsi009f3.jpeg?Expires=2147483647&Signature=oNCGhgMOh1P4AmXS1tbcyLr2pF423BkdujmzgUSDZOTQe0BxsbSfjj5aKT4MfZHH9Hz69JvddwEuQlwH0fr8juURr~TPL8jNEVmUuy7DZ2kS5zchToLiy-6qTaaqFvR494WZMOIQy4Bo-WwbmK9V2ki7t2R9kpa9V8ZfnCxcx1zDoUUgYCN8HZC7qVGknzMNabb2LTaannQCHVtWVhjI2MqEz9Ei4PRAZwvkAO7sSDVe4WrCSDF~neOrRKx~GMgtWDQCXt2IbMFB7hwL82uhctGbQaVINLY5t7212yb43h~C0-dtljDuCoPEeynG8oZBRny9zUug9Gq-Jhxywi4vig__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Representation plots for mononucleotide SSRs (size between 5 and 13 nt) in selected archaeal and bacterial genomes. x-axis: size of the repeat, y-axis: number of repeats observed and number of repeats expected." path-from-xml="dsi009f3.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/10424264/dsi009f3.jpeg" path-from-xml="dsi009f3.jpeg" target="_blank">View large</a><a section="10424264" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/4/10.1093/dnares/dsi009/2/dsi009f3.jpeg?Expires=2147483647&Signature=Zz6EGt6TCPISejG~HQvz4Lfcd8dizuLjkKc5UUZCvxW4TibrY03KGg5-B53nCfNgmsUvTGWZz6raXDnsYU-4PrVA-GH0EjS0Dpb65FhQffDwBFYatUvPFwbAhs-~9bULhmFVz30rNenopbVN70NuH9zo70QnyKx7gFxBes7rZz2Amz8m1ndEs6YnkVXKnl4TG8D2YzaQzBs7YYLZMBadGoEXhhuzXCFxbM19ffvEfmG6l-W7opNy9sTOmEx0qtx33slLsx7aiEuC-rP4pnc~vfX3HAT2s6YdHIdFKes6UzophnyDiHtJ~kbfOkhWPgghtUbuwLTfoN6t~~kFGg7PiA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=10424264&ar=621324&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi009f3.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Representation plots for mononucleotide SSRs (size between 5 and 13 nt) in selected archaeal and bacterial genomes. <em>x</em>-axis: size of the repeat, <em>y</em>-axis: number of repeats observed and number of repeats expected.</p></div></div>                            <h3 scrollto-destination=10424268 id="10424268" class="section-title" >3.5. Presence of longer mononucleotide SSRs in prokaryotic genomes</h3>
<p>Sixty prokaryotic genomes (5 archaeal and 55 bacterial genomes) investigated contained mononucleotide repeats &gt;13 nt (Table S2, available in Supplementary Data). The total number of mononucleotide SSRs &gt;13 nt ranged from 1 (24 genomes) to 24 (<em>Helicobacter hepaticus</em>). The largest mononucleotide SSRs are found in the genome of <em>Thermoanaerobacter tengcongensis</em>, containing a poly(A) mononucleotide SSR of 54 nt and one of 58 nt. Only in 18 genomes, one or more of these longer mononucleotide SSRs were located in coding regions (Table S2). The maximum number of longer mononucleotide SSRs located in coding regions was five (in <em>Helicobacter hepaticus</em> and <em>Helicobacter pylori</em> 26695). The longest mononucleotide SSR located in a coding region is the 22 nt poly(G) repeat in the <em>Pirelulla</em> sp. genome. There was no significant difference in the number of long mononucleotide repeats between archaea and bacteria (<em>P</em> = 0.207).</p>                            <h3 scrollto-destination=10424271 id="10424271" class="section-title" >3.6. Correlation between number of mononucleotide SSRs and other genome parameters</h3>
<p>Pearson correlation coefficients between number of mononucleotide SSRs in a genome, repeat density, the ratio observed/expected number of mononucleotide SSRs, genome size and GC content are shown in Table S3 (available in Supplementary Data). Similar trends were observed when we calculated the non-parametric Spearman and Kendall correlations (data not shown). From these data, it appears that the number of mononucleotide SSRs is weakly correlated with genome size and more strongly, but negatively, correlated with GC content. The repeat density and <em>R</em> are both negatively correlated with genome size and GC content. Graphical representations of the relationship between a number of variables investigated are shown in <a reveal-id="fig4" data-open="fig4" class="link link-reveal link-table xref-fig">Fig. 4</a>. However, genome size and GC content are correlated, as larger genomes tend to be more GC-rich and smaller genomes tend to be more AT-rich (see also Table S3).<a reveal-id="b31" data-open="b31" class="link link-ref link-reveal xref-bibr">31</a> For that reason, we calculated partial correlations between a number of variables while removing the influence caused by the relation between genome size and GC content. We found that the number of mononucleotide SSRs observed correlated well with genome size (<em>r</em><sup>2</sup> = 0.6741, <em>P</em> &lt; 0.001) and GC content (<em>r</em><sup>2</sup> = −0.7301, <em>P</em> &lt; 0.001). The repeat density correlated very well with the GC content (<em>r</em><sup>2</sup> = −0.7712, <em>P</em> &lt; 0.001), while the relation between repeat density and genome size was less pronounced (<em>r</em><sup>2</sup> = −0.2260, <em>P</em> = 0.005). The correlation between <em>R</em> and GC content was also high (<em>r</em><sup>2</sup> = −0.5761, <em>P</em> &lt; 0.001), but no significant correlation was detected between <em>R</em> and genome size following the correction for the confounding effect of GC content. These data show that (i) larger genomes contain more mononucleotide SSRs than smaller genomes with the same GC content, (ii) larger genomes have a lower mononucleotide SSR density than smaller genomes with the same GC content, (iii) genomes with a high GC content contain less mononucleotide SSRs (and thus have a lower mononucleotide SSR density) than similar-sized genomes with a low GC content, and (iv) genomes with high GC content have a lower <em>R</em> value (i.e. contain less mononucleotide SSRs than expected) than similar-sized genomes with a low GC content. Also, the higher the mononucleotide SSR density, the larger the <em>R</em> value, meaning the larger the deviation from expected total number of mononucleotide SSRs.</p>                        <a id="10424276" scrollto-destination="10424276"></a>
<div id="fig4" data-id="fig4" class="fig fig-section"><div class="label fig-label"><strong>Figure 4</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/4/10.1093/dnares/dsi009/2/m_dsi009f4.jpeg?Expires=2147483647&Signature=AYtS8NcLEbbaw~B~tVo-eewfmFmATla3XHk8Bg26ARiG5QjAFy6dJq94Yb7Jyx08vcmbOBHYCIrE0QtZFbkhT3jvAKxlTezPzaND3~8P0DHbsgSVOusrSDAGfvp8eH8iT41F7jWdsjaoxpsG3a~~2r-uILqbyev1ZKUneOcpdqhYtfk-8PMnDaD-5itAgZ4UEfK3jpP5FVxXgtO~E9MY7ffHIu2SoFkVx4G~Pe6s29305BlitKQB03lzN2ISRBEWCSc1kZsxGUpKgRC37RPMJe2z7PCA1KeGuaLsatMA6ZXhZRt-kFEnKFqthvzSFfsLkR48lKMgKr6q6yB6WE-d1w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Correlation between a number of variables investigated." path-from-xml="dsi009f4.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/10424276/dsi009f4.jpeg" path-from-xml="dsi009f4.jpeg" target="_blank">View large</a><a section="10424276" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/4/10.1093/dnares/dsi009/2/dsi009f4.jpeg?Expires=2147483647&Signature=N7IiwSSEK48-5b8j9BcckoeMUQfUtoeI8freotXGMq0YyRqCYI58p0SWX-B67aSmPt79W7-wnc5P6CXlTtFtEW-0Od6X275RMMtVEnPtOR5QSxx81BU~5M1PMoJNPLWiBVFeuh3J0Ov7d5s8EIxIvOFWeT-KiNGxwYcyzFLmTJZemCtgc3VfI0G5f02eIbRC0nDkSwEJZdnZoT9Au8RiFRiQw22rxr59PMIXYLxD13wJcxqbpgHklkxiI2Ts2KCp~TdK9DspcOKqaS36D1h~-Gpm3rQNUpXbVIo9W9eOT1hZbIFstJYTGfAqTgegm583GqxemRCcVc1NDTPo-6QycQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=10424276&ar=621324&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi009f4.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Correlation between a number of variables investigated.</p></div></div><div content-id="fig4" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 4</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/4/10.1093/dnares/dsi009/2/m_dsi009f4.jpeg?Expires=2147483647&Signature=AYtS8NcLEbbaw~B~tVo-eewfmFmATla3XHk8Bg26ARiG5QjAFy6dJq94Yb7Jyx08vcmbOBHYCIrE0QtZFbkhT3jvAKxlTezPzaND3~8P0DHbsgSVOusrSDAGfvp8eH8iT41F7jWdsjaoxpsG3a~~2r-uILqbyev1ZKUneOcpdqhYtfk-8PMnDaD-5itAgZ4UEfK3jpP5FVxXgtO~E9MY7ffHIu2SoFkVx4G~Pe6s29305BlitKQB03lzN2ISRBEWCSc1kZsxGUpKgRC37RPMJe2z7PCA1KeGuaLsatMA6ZXhZRt-kFEnKFqthvzSFfsLkR48lKMgKr6q6yB6WE-d1w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Correlation between a number of variables investigated." path-from-xml="dsi009f4.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/10424276/dsi009f4.jpeg" path-from-xml="dsi009f4.jpeg" target="_blank">View large</a><a section="10424276" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/4/10.1093/dnares/dsi009/2/dsi009f4.jpeg?Expires=2147483647&Signature=N7IiwSSEK48-5b8j9BcckoeMUQfUtoeI8freotXGMq0YyRqCYI58p0SWX-B67aSmPt79W7-wnc5P6CXlTtFtEW-0Od6X275RMMtVEnPtOR5QSxx81BU~5M1PMoJNPLWiBVFeuh3J0Ov7d5s8EIxIvOFWeT-KiNGxwYcyzFLmTJZemCtgc3VfI0G5f02eIbRC0nDkSwEJZdnZoT9Au8RiFRiQw22rxr59PMIXYLxD13wJcxqbpgHklkxiI2Ts2KCp~TdK9DspcOKqaS36D1h~-Gpm3rQNUpXbVIo9W9eOT1hZbIFstJYTGfAqTgegm583GqxemRCcVc1NDTPo-6QycQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=10424276&ar=621324&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi009f4.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Correlation between a number of variables investigated.</p></div></div><p>We also investigated the correlation between the observed and the expected mononucleotide SSR density and the GC content. As can be seen from Fig. S2 (available in Supplementary Data), there is a strong relationship between the GC content and the expected repeat density. This can be explained by the fact that, as the GC content shifts away from 50%, the four-letter nucleotide alphabet changes towards a two-letter alphabet, thereby increasing the probability of repeated sequences.<a reveal-id="b32" data-open="b32" class="link link-ref link-reveal xref-bibr">32</a> However, we see that there is an entirely different relationship between the GC content and the actual observed mononucleotide SSR density (Fig. S2).</p><p>In addition, we calculated correlations between number and density of mononucleotide SSRs and <em>R</em>, and the fraction of the genome located in duplicated segments and in horizontally transferred segments. Following correction for genome size (as the fraction of duplicated genes in a genome correlates positively with genome size<a reveal-id="b29" data-open="b29" class="link link-ref link-reveal xref-bibr">29</a>), no correlation between number and density of mononucleotide SSRs and <em>R</em>, and the fraction of duplicated genes in the genome could be observed. However, we found a significant negative relationship between the fraction of the genome localized in putatively horizontally transferred genes and SSR density (<em>r</em><sup>2</sup> = −0.592, <em>P</em> &lt; 0.001). This correlation remained highly significant after correcting for the confounding effect of genome size (larger genomes tend to contain a higher fraction of putatively horizontally transferred genes<a reveal-id="b30" data-open="b30" class="link link-ref link-reveal xref-bibr">30</a>) (<em>r</em><sup>2</sup> = −0.4130, <em>P</em> &lt; 0.001) (<a reveal-id="fig5" data-open="fig5" class="link link-reveal link-table xref-fig">Fig. 5</a>).</p>                        <a id="10424284" scrollto-destination="10424284"></a>
<div id="fig5" data-id="fig5" class="fig fig-section"><div class="label fig-label"><strong>Figure 5</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/4/10.1093/dnares/dsi009/2/m_dsi009f5.jpeg?Expires=2147483647&Signature=YmxYS7wHA2GiX0IV0Cxasb5KgNxtyrf375A5UMHiZFu4UN55gIirimvVFvp4uAgF9Cz4J3FGtSG06pBlobSXOgaFI-oNYTSqPTFFFlaCkS9Qq6zYlr0CbYIXUps~TYRSEVyu5wu4QUqsQ9AjeD5Kg6X7o356Kr0-PaxlPrdUvTkNRbYILdsVoSjQwUBoxzfaDS~YuX3FtBkJPQPlLo4r2dof~9JvnXEncFPCFn0Wn9j8Y6AlOqXudCllt4zXSfP9xAkM7Lh~zlRAgHWOS5Mu-mnNwg50I5Ag-D8NFi26Xj-r5MPA-xTo2gWUR4BGkpxRifc29h3VNNNGvC1EVXgjCg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Correlation between mononucleotide SSR density and the fraction of putatively horizontally transferred genes." path-from-xml="dsi009f5.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/10424284/dsi009f5.jpeg" path-from-xml="dsi009f5.jpeg" target="_blank">View large</a><a section="10424284" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/4/10.1093/dnares/dsi009/2/dsi009f5.jpeg?Expires=2147483647&Signature=F6Vm9v5b76kdn~qO5Z7m2odcivZdWZfWpk1m~EMCEFjKEoT8R7XKCiyP2qrVA5SkGWhspwidJdPhGb~v2NcLcWFpT6MNPozwKpunnMjvGkFX7etOD95uB7vfD3q9lNxZd6tYKJPn1Zb40dSzuCfQl3DhCNQY3x4D7VzZHYmenpH7kvpdKcDuk9E0ynigAVU4EjGALw2yw4nEjjt0d48bDyrdH6SFAf-4p2Q7RzjqK1VtCIb9zE6WTvOQfJQ3MiOnJZh9oom2G8~MLRYZIuPmkDb4NrSCZKrJ48sC6jH68rccLoibt1MFH1~bEeS3OIfh7mmW1KAOZtn8qP9YlBuZwQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=10424284&ar=621324&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi009f5.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Correlation between mononucleotide SSR density and the fraction of putatively horizontally transferred genes.</p></div></div><div content-id="fig5" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 5</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/4/10.1093/dnares/dsi009/2/m_dsi009f5.jpeg?Expires=2147483647&Signature=YmxYS7wHA2GiX0IV0Cxasb5KgNxtyrf375A5UMHiZFu4UN55gIirimvVFvp4uAgF9Cz4J3FGtSG06pBlobSXOgaFI-oNYTSqPTFFFlaCkS9Qq6zYlr0CbYIXUps~TYRSEVyu5wu4QUqsQ9AjeD5Kg6X7o356Kr0-PaxlPrdUvTkNRbYILdsVoSjQwUBoxzfaDS~YuX3FtBkJPQPlLo4r2dof~9JvnXEncFPCFn0Wn9j8Y6AlOqXudCllt4zXSfP9xAkM7Lh~zlRAgHWOS5Mu-mnNwg50I5Ag-D8NFi26Xj-r5MPA-xTo2gWUR4BGkpxRifc29h3VNNNGvC1EVXgjCg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Correlation between mononucleotide SSR density and the fraction of putatively horizontally transferred genes." path-from-xml="dsi009f5.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/10424284/dsi009f5.jpeg" path-from-xml="dsi009f5.jpeg" target="_blank">View large</a><a section="10424284" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/4/10.1093/dnares/dsi009/2/dsi009f5.jpeg?Expires=2147483647&Signature=F6Vm9v5b76kdn~qO5Z7m2odcivZdWZfWpk1m~EMCEFjKEoT8R7XKCiyP2qrVA5SkGWhspwidJdPhGb~v2NcLcWFpT6MNPozwKpunnMjvGkFX7etOD95uB7vfD3q9lNxZd6tYKJPn1Zb40dSzuCfQl3DhCNQY3x4D7VzZHYmenpH7kvpdKcDuk9E0ynigAVU4EjGALw2yw4nEjjt0d48bDyrdH6SFAf-4p2Q7RzjqK1VtCIb9zE6WTvOQfJQ3MiOnJZh9oom2G8~MLRYZIuPmkDb4NrSCZKrJ48sC6jH68rccLoibt1MFH1~bEeS3OIfh7mmW1KAOZtn8qP9YlBuZwQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=10424284&ar=621324&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi009f5.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Correlation between mononucleotide SSR density and the fraction of putatively horizontally transferred genes.</p></div></div>                            <h3 scrollto-destination=10424288 id="10424288" class="section-title" >3.7. Correlation between the mean GC content of mononucleotide SSRs, the mean GC content of the genome and repeat density</h3>
<p>The relationship between genomic GC content and the mean GC content of mononucleotide SSRs in a given genome is shown in Fig. S3 (available in Supplementary Data). This relationship is best described by a least-squares fit with the following equation: GC<sub>SSR</sub> = 2E<sup>−07</sup> (GC<sub>genome</sub>)<sup>4.6365</sup> (<em>r</em><sup>2</sup> = 0.899). Our data clearly show that there is a tremendous overrepresentation of A and T in mononucleotide SSRs in AT-rich genomes (i.e. genomes in which the GC content is &lt;50 mol%), while G and C are only overrepresented in a few GC-rich genomes (GC content &gt;65 mol%). The relationship between repeat density and the mean GC content of mononucleotide SSRs in a given genome is also shown in Fig. S3. This relationship is best described by a least-squares fit with the following equation: GC<sub>SSR</sub> = 4890.7 (repeat density)<sup>−1.5515</sup> (<em>r</em><sup>2</sup> = 0.695). This indicates that the GC content of mononucleotide SSRs is highest when the repeat density is lowest.</p>                            <h2 scrollto-destination=10424291 id="10424291" class="section-title" >4. Discussion</h2>
<p>In the present study, we investigated the presence of mononucleotide SSRs in sequenced prokaryotic genomes. The data obtained in the present study show that (i) a large number of mononucleotide SSRs is present in all prokaryotic genomes investigated (belonging both to the bacteria and to the archaea), (ii) shorter repeats are much more abundant than longer repeats, and (iii) in the majority of the genomes, longer mononucleotide SSRs are excluded from coding regions (although there are several exceptions to this general rule). Bacterial genomes that contain much less (i.e. <em>R</em> &lt; 0.5) mononucleotide SSRs than expected are generally larger (mean genome size = 5 505 212 bp) and more GC-rich (mean GC content = 66.4 mol%), while bacterial genomes that contain much more (i.e. <em>R</em> &gt; 2.0) mononucleotide SSRs than expected are in general smaller (average genome size = 2 254 190 bp) and more AT-rich (average GC content = 39.3 mol%) (the average genome size for all bacterial genomes = 3 336 435 bp, average GC-content = 46.6%). We also noted that genomes that contain a high fraction of horizontally transferred genes, in general have a lower mononucleotide SSR density.</p><p>It is now generally accepted that SSRs, through a variety of molecular processes, are involved in generating genomic and phenotypic diversity in eukaryotes as well as in prokaryotes. Our analyses performed on a large number of prokaryotic genomes clearly indicate that, due to the presence of this large number of mononucleotide SSRs, prokaryotes have an enormous potential for generating this genomic and phenotypic diversity.</p><p>Longer mononucleotide SSRs have more opportunity to undergo slipped-strand mispairing and there will be more mutability in their length than in shorter mononucleotide SSRs. This could help to explain why these are overrepresented in non-coding regions of the genome as selection has ample opportunity to operate against these larger repeats that would cause frameshift and non-sense mutations in coding regions. The observation that in some genomes (including the genomes of the <em>ɛ-Proteobacteria Campylobacter jejuni</em>, <em>Helicobacter pylori</em>, <em>Helicobacter hepaticus</em>, <em>Wolinella succinogenes</em>, and those of <em>Haemophilus ducreyi</em>, <em>Neisseria meningitidis</em> and <em>Synechocystis</em> sp.) larger mononucleotide SSRs are not (or to a lesser extent) excluded from coding regions, suggest they may play an important role in phase variation<a reveal-id="b33" data-open="b33" class="link link-ref link-reveal xref-bibr">33</a> as this process has been observed in these organisms.<a reveal-id="b9" data-open="b9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="b34  b36" data-open="b34  b36" class="link link-ref link-reveal xref-bibr">34–36</a></p><p>Interestingly, we also observed that the repeat density was significantly higher (one-way ANOVA, <em>P</em> &lt; 0.001) in organisms with an intracellular or strictly parasitic lifestyle (including <em>Brucella</em> spp., <em>Treponema</em> spp., <em>Chlamydia</em> spp., <em>Mycoplasma</em> spp., <em>Bartonella</em> spp., <em>Mesoplasma florum, Phytoplasma asteris, Coxiella burnettii, Wigglesworthia glossinidia, Buchnera aphidicola</em> and <em>Blochmannia floridanus</em>) (see <a reveal-id="tbl2" data-open="tbl2" class="link link-reveal link-table xref-fig">Table 2</a>). This seems to be contradicting the consistent genome reduction observed in most of these organisms (e.g. there are no repeated sequences &gt;200 bp in members of the genus <em>Buchnera</em><a reveal-id="b37" data-open="b37" class="link link-ref link-reveal xref-bibr">37</a>). For at least some of these organisms (e.g. <em>Buchnera</em> species), this probably can be attributed to the loss of genes involved in DNA repair functions,<a reveal-id="b38" data-open="b38" class="link link-ref link-reveal xref-bibr">38</a>,<a reveal-id="b39" data-open="b39" class="link link-ref link-reveal xref-bibr">39</a> as it has been shown that the loss of DNA polymerase proofreading activity or post-replication mismatch repair can greatly enhance the rate of SSR tract alterations.<a reveal-id="b3" data-open="b3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="b40" data-open="b40" class="link link-ref link-reveal xref-bibr">40</a>,<a reveal-id="b41" data-open="b41" class="link link-ref link-reveal xref-bibr">41</a> However, other intracellular bacteria do contain proofreading activity, so most probably other (as yet unidentified) factors are involved as well.</p><p>Besides their likely involvement in phase variation, variation in mononucleotide SSR length at or near regulatory elements can have a significant influence on gene expression by affecting the binding of regulatory proteins, the distance between regulatory elements, the bendability, coiling, packaging and phasing of the DNA, or the formation of unusual DNA structures.<a reveal-id="b24" data-open="b24" class="link link-ref link-reveal xref-bibr">24</a>,<a reveal-id="b42  b44" data-open="b42  b44" class="link link-ref link-reveal xref-bibr">42–44</a> In contrast to phase variation, these processes can affect gene expression in a more subtle, quantitative way.<a reveal-id="b42" data-open="b42" class="link link-ref link-reveal xref-bibr">42</a>,<a reveal-id="b45" data-open="b45" class="link link-ref link-reveal xref-bibr">45</a> Computer simulations performed by King and co-workers<a reveal-id="b46" data-open="b46" class="link link-ref link-reveal xref-bibr">46</a>,<a reveal-id="b47" data-open="b47" class="link link-ref link-reveal xref-bibr">47</a> have shown that the mutational properties of SSRs could be exploited by evolving populations. The mutation rate in populations which are under selection pressure (e.g. imposed by gradually changing environmental conditions) can increase as an indirect consequence of selection. When mutations are suitably constrained, frequent mutations allow the population to adapt efficiently. In this light, it has been hypothesized that SSRs have become frequent because selection has favoured their intrinsic instability, which allows them to work as ‘adjustable tuning knobs’ that facilitate the process of evolutionary change.<a reveal-id="b46" data-open="b46" class="link link-ref link-reveal xref-bibr">46</a> The large differences between expected and observed number of mononucleotide SSRs could be attributable to the presence of alternative mechanisms for generating diversity in genomes with a low number of mononucleotide SSRs. The observation that there is a significant negative correlation between the fraction of the genome localized in putatively horizontally transferred genes and mononucleotide SSR density fits into this hypothesis as well.</p><p>We would like to point out that the number of putatively horizontally transferred genes for each genome were taken from data published by Nakamura et al.<a reveal-id="b30" data-open="b30" class="link link-ref link-reveal xref-bibr">30</a> In this study, putatively horizontally transferred genes were identified by a Bayesian method based on differences in nucleotide composition. While this method has several advantages over more direct phylogenetic methods, a disadvantage is that it may preferentially detect recent horizontally transferred genes, while anciently transferred genes may be indistinguishable because the nucleotide composition of the transferred gene is converging with that of the recipient genome by mutational pressure (a process called ‘amelioration’<a reveal-id="b48" data-open="b48" class="link link-ref link-reveal xref-bibr">48</a>). It will be interesting to see whether the intriguing relationship between the fraction of the genome localized in putatively horizontally transferred genes and SSR density holds up if more ancient horizontal gene transfer events are taken into account.</p><p>The overrepresentation of A and T in mononucleotide SSRs in the majority of prokaryotic genomes might be explained by the fact that slipped-strand mispairing is more likely for poly(A) or poly(T) repeats, as strand separation for these polynucleotides is energetically favourable compared with strand separation of poly(C) or poly(G) repeats. It is thus easier to generate variability with poly(A) and poly(T) mononucleotide SSRs than with poly(C) or poly(G) repeats. This is confirmed by the observation that the loss of DNA polymerase proofreading activity or post-replication mismatch repair specifically enhances the rate of poly (A) tract alterations.<a reveal-id="b49" data-open="b49" class="link link-ref link-reveal xref-bibr">49</a> However, our data also clearly indicate that the GC content of mononucleotide SSRs is highest when the repeat density is lowest. This suggest that there could be other reasons for the tremendous overrepresentation of poly(A) and poly(T) mononucleotide SSRs. It has been suggested that the higher energy cost of G and C over A and T/U could be the reason for the high variation seen in genomic G+C content.<a reveal-id="b31" data-open="b31" class="link link-ref link-reveal xref-bibr">31</a>,<a reveal-id="b50" data-open="b50" class="link link-ref link-reveal xref-bibr">50</a> Indeed, the synthesis of GTP requires an additional NAD compared with AMP, while the synthesis of CTP from UTP requires an additional ATP molecule.<a reveal-id="b50" data-open="b50" class="link link-ref link-reveal xref-bibr">50</a> In addition, due to its central role in metabolism, ATP is abundantly present in the cell. The observation that more GC-rich mononucleotide SSRs are predominantly found in genomes with a low mononucleotide SSR density suggest that the same underlying reason might be responsible for the marked differences observed in G+C content of these mononucleotide SSRs, as it would be too costly to have many poly(G) and/or poly(C) SSRs in genomes with a high density of mononucleotide SSRs. This hypothesis is corroborated by the observation that the G+C content of mononucleotide SSRs of intracellular bacteria is extremely low (i.e. the mononucleotide SSRs of these organisms are—just like the rest of the genome<a reveal-id="b50" data-open="b50" class="link link-ref link-reveal xref-bibr">50</a>—more AT-rich as a result of selection by competition for scarce resources). Another possible reason is the involvement of repeated sequences in the formation of non-canonical DNA structures, including triple-stranded H-DNA. These structures are more easily formed in GA-rich regions<a reveal-id="b51" data-open="b51" class="link link-ref link-reveal xref-bibr">51</a>,<a reveal-id="b52" data-open="b52" class="link link-ref link-reveal xref-bibr">52</a> and could block transcription by RNA polymerase.<a reveal-id="b53" data-open="b53" class="link link-ref link-reveal xref-bibr">53</a> As such, the presence of many G (or A) mononucleotide SSRs in bacteria with a high GC content would result in a selective disadvantage.</p>                            <h2 scrollto-destination=10424307 id="10424307" class="section-title" >Supplementary Data</h2>
<p>Supplementary Data are available online at <a class="link link-uri" href="http://users.ugent.be/~tcoenye/index_files/Page444.htm" target="_blank"></a> and at <a class="link link-uri" href="http://dnaresearch.oxfordjournals.org" target="_blank"></a> or can be requested from Tom Coenye.</p><p>T.C. and P.V. are indebted to the Fund for Scientific Research—Flanders (Belgium) for the position as postdoctoral fellow and research grants, respectively. We thank the anonymous reviewers for helpful comments that drastically improved our manuscript.</p>                            <h2 scrollto-destination=10424315 id="10424315" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="b1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">van Belkum</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Scherer</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">van Alphen</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Verbrugh</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Short-sequence DNA repeats in prokaryotic genomes</div>, <div class="source">Microbiol. Mol. Biol. Rev.</div> , <div class="year">1998</div>, vol. <div class="volume">62</div> (pg. <div class="fpage">275</div>-<div class="lpage">293</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Short-sequence%20DNA%20repeats%20in%20prokaryotic%20genomes&amp;author=A.%20van%20Belkum&amp;author=S.%20Scherer&amp;author=L.%20van%20Alphen&amp;author=H.%20Verbrugh&amp;publication_year=1998&amp;journal=Microbiol.%20Mol.%20Biol.%20Rev.&amp;volume=62&amp;pages=275-293" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9618442" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yeramian</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Buc</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Tandem repeats in complete bacterial genomes sequences. Sequence and structural analyses for comparative studies</div>, <div class="source">Res. Microbiol.</div> , <div class="year">1999</div>, vol. <div class="volume">150</div> (pg. <div class="fpage">745</div>-<div class="lpage">754</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Tandem%20repeats%20in%20complete%20bacterial%20genomes%20sequences.%20Sequence%20and%20structural%20analyses%20for%20comparative%20studies&amp;author=E.%20Yeramian&amp;author=H.%20Buc&amp;publication_year=1999&amp;journal=Res.%20Microbiol.&amp;volume=150&amp;pages=745-754" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0923-2508(99)00118-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0923-2508(99)00118-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0923-2508(99)00118-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10673012" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Levinson</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Gutman</div> <div class="given-names">G. A.</div></div>. </div><div class="article-title">Slipped-strand mispairing: a major mechanism for DNA sequence evolution</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">1987</div>, vol. <div class="volume">4</div> (pg. <div class="fpage">203</div>-<div class="lpage">221</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Slipped-strand%20mispairing%3A%20a%20major%20mechanism%20for%20DNA%20sequence%20evolution&amp;author=G.%20Levinson&amp;author=G.%20A.%20Gutman&amp;publication_year=1987&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=4&amp;pages=203-221" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3328815" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">van Belkum</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">van Leeuwen</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Scherer</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Verbrugh</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Occurrence and structure–function relationship of pentameric short sequence repeats in microbial genomes</div>, <div class="source">Res. Microbiol.</div> , <div class="year">1999</div>, vol. <div class="volume">150</div> (pg. <div class="fpage">617</div>-<div class="lpage">626</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Occurrence%20and%20structure%E2%80%93function%20relationship%20of%20pentameric%20short%20sequence%20repeats%20in%20microbial%20genomes&amp;author=A.%20van%20Belkum&amp;author=W.%20van%20Leeuwen&amp;author=S.%20Scherer&amp;author=H.%20Verbrugh&amp;publication_year=1999&amp;journal=Res.%20Microbiol.&amp;volume=150&amp;pages=617-626" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0923-2508(99)00129-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0923-2508(99)00129-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0923-2508(99)00129-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10673001" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhu</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Strassmann</div> <div class="given-names">J. E.</div></div>,  <div class="name"><div class="surname">Queller</div> <div class="given-names">D. C.</div></div>. </div><div class="article-title">Insertions, deletions, and the origin of microsatellites</div>, <div class="source">Genet. Res.</div> , <div class="year">2000</div>, vol. <div class="volume">76</div> (pg. <div class="fpage">227</div>-<div class="lpage">236</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Insertions%2C%20deletions%2C%20and%20the%20origin%20of%20microsatellites&amp;author=Y.%20Zhu&amp;author=J.%20E.%20Strassmann&amp;author=D.%20C.%20Queller&amp;publication_year=2000&amp;journal=Genet.%20Res.&amp;volume=76&amp;pages=227-236" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1017/S001667230000478X" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1017%2FS001667230000478X" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1017%2FS001667230000478X"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11204970" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dieringer</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Schlotterer</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Two distinct modes of microsatellite mutation processes: Evidence from the complete genomic sequences of nine species</div>, <div class="source">Genome Res.</div> , <div class="year">2003</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">2242</div>-<div class="lpage">2251</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Two%20distinct%20modes%20of%20microsatellite%20mutation%20processes%3A%20Evidence%20from%20the%20complete%20genomic%20sequences%20of%20nine%20species&amp;author=D.%20Dieringer&amp;author=C.%20Schlotterer&amp;publication_year=2003&amp;journal=Genome%20Res.&amp;volume=13&amp;pages=2242-2251" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.1416703" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.1416703" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.1416703"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14525926" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Perez-Martin</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Rojo</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">de Lorenzo</div> <div class="given-names">V.</div></div>. </div><div class="article-title">Promoters responsive to DNA bending: a common theme in prokaryotic gene expression</div>, <div class="source">Microbiol. Rev.</div> , <div class="year">1994</div>, vol. <div class="volume">58</div> (pg. <div class="fpage">268</div>-<div class="lpage">290</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Promoters%20responsive%20to%20DNA%20bending%3A%20a%20common%20theme%20in%20prokaryotic%20gene%20expression&amp;author=J.%20Perez-Martin&amp;author=F.%20Rojo&amp;author=V.%20de%20Lorenzo&amp;publication_year=1994&amp;journal=Microbiol.%20Rev.&amp;volume=58&amp;pages=268-290" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8078436" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Krasilnikova</div> <div class="given-names">M. M.</div></div>,  <div class="name"><div class="surname">Samadashwily</div> <div class="given-names">G. M.</div></div>,  <div class="name"><div class="surname">Krasilnikov</div> <div class="given-names">A. S.</div></div>,  <div class="name"><div class="surname">Mirkin</div> <div class="given-names">S. M.</div></div>. </div><div class="article-title">Transcription through a simple DNA repeat blocks replication elongation</div>, <div class="source">EMBO J.</div> , <div class="year">1998</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">5095</div>-<div class="lpage">5102</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcription%20through%20a%20simple%20DNA%20repeat%20blocks%20replication%20elongation&amp;author=M.%20M.%20Krasilnikova&amp;author=G.%20M.%20Samadashwily&amp;author=A.%20S.%20Krasilnikov&amp;author=S.%20M.%20Mirkin&amp;publication_year=1998&amp;journal=EMBO%20J.&amp;volume=17&amp;pages=5095-5102" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/emboj/17.17.5095" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Femboj%2F17.17.5095" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Femboj%2F17.17.5095"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9724645" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Henderson</div> <div class="given-names">I. R.</div></div>,  <div class="name"><div class="surname">Owen</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Nataro</div> <div class="given-names">J. P.</div></div>. </div><div class="article-title">Molecular switches—the ON and OFF of bacterial phase variation</div>, <div class="source">Mol. Microbiol.</div> , <div class="year">1999</div>, vol. <div class="volume">33</div> (pg. <div class="fpage">919</div>-<div class="lpage">932</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20switches%E2%80%94the%20ON%20and%20OFF%20of%20bacterial%20phase%20variation&amp;author=I.%20R.%20Henderson&amp;author=P.%20Owen&amp;author=J.%20P.%20Nataro&amp;publication_year=1999&amp;journal=Mol.%20Microbiol.&amp;volume=33&amp;pages=919-932" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-2958.1999.01555.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-2958.1999.01555.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-2958.1999.01555.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10476027" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rocha</div> <div class="given-names">E. P. C.</div></div>,  <div class="name"><div class="surname">Matic</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Taddei</div> <div class="given-names">F.</div></div>. </div><div class="article-title">Over-representation of repeats in stress response genes:a strategy to increase versatility under stressful conditions?</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2002</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">1886</div>-<div class="lpage">1894</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Over-representation%20of%20repeats%20in%20stress%20response%20genes%3Aa%20strategy%20to%20increase%20versatility%20under%20stressful%20conditions%3F&amp;author=E.%20P.%20C.%20Rocha&amp;author=I.%20Matic&amp;author=F.%20Taddei&amp;publication_year=2002&amp;journal=Nucleic%20Acids%20Res.&amp;volume=30&amp;pages=1886-1894" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/30.9.1886" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F30.9.1886" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F30.9.1886"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11972324" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bzymek</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Lovett</div> <div class="given-names">S. T.</div></div>. </div><div class="article-title">Instability of repetitive DNA sequences: The role of replication in multiple mechanisms</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2001</div>, vol. <div class="volume">98</div> (pg. <div class="fpage">8319</div>-<div class="lpage">8325</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Instability%20of%20repetitive%20DNA%20sequences%3A%20The%20role%20of%20replication%20in%20multiple%20mechanisms&amp;author=M.%20Bzymek&amp;author=S.%20T.%20Lovett&amp;publication_year=2001&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=98&amp;pages=8319-8325" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.111008398" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.111008398" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.111008398"> </span></div> </div></div></div></div></div><div content-id="b12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rocha</div> <div class="given-names">E. P. C.</div></div>. </div><div class="article-title">An appraisal of the potential for illegitimate recombination in bacterial genomes and its consequences: from duplications to genome reduction</div>, <div class="source">Genome Res.</div> , <div class="year">2003</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">1123</div>-<div class="lpage">1132</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20appraisal%20of%20the%20potential%20for%20illegitimate%20recombination%20in%20bacterial%20genomes%20and%20its%20consequences%3A%20from%20duplications%20to%20genome%20reduction&amp;author=E.%20P.%20C.%20Rocha&amp;publication_year=2003&amp;journal=Genome%20Res.&amp;volume=13&amp;pages=1123-1132" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.966203" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.966203" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.966203"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12743022" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Orti</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Pearse</div> <div class="given-names">D. E.</div></div>,  <div class="name"><div class="surname">Avise</div> <div class="given-names">J. C.</div></div>. </div><div class="article-title">Phylogenetic assignment of length variation at a microsatellite locus</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">1997</div>, vol. <div class="volume">94</div> (pg. <div class="fpage">10745</div>-<div class="lpage">10749</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phylogenetic%20assignment%20of%20length%20variation%20at%20a%20microsatellite%20locus&amp;author=G.%20Orti&amp;author=D.%20E.%20Pearse&amp;author=J.%20C.%20Avise&amp;publication_year=1997&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=94&amp;pages=10745-10749" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.94.20.10745" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.94.20.10745" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.94.20.10745"> </span></div> </div></div></div></div></div><div content-id="b14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sivolap</div> <div class="given-names">Y. M.</div></div>,  <div class="name"><div class="surname">Bonner</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Association of chromosomal RNA with repetitive DNA</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">1971</div>, vol. <div class="volume">68</div> (pg. <div class="fpage">387</div>-<div class="lpage">389</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Association%20of%20chromosomal%20RNA%20with%20repetitive%20DNA&amp;author=Y.%20M.%20Sivolap&amp;author=J.%20Bonner&amp;publication_year=1971&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=68&amp;pages=387-389" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.68.2.387" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.68.2.387" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.68.2.387"> </span></div> </div></div></div></div></div><div content-id="b15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Locker</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Rabinowitz</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Getz</div> <div class="given-names">G. S.</div></div>. </div><div class="article-title">Tandem inverted repeats in mitochondrial DNA of petite mutants of <em>Saccharomyces cerevisiae</em></div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">1974</div>, vol. <div class="volume">71</div> (pg. <div class="fpage">1366</div>-<div class="lpage">1370</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Tandem%20inverted%20repeats%20in%20mitochondrial%20DNA%20of%20petite%20mutants%20of%20Saccharomyces%20cerevisiae&amp;author=J.%20Locker&amp;author=M.%20Rabinowitz&amp;author=G.%20S.%20Getz&amp;publication_year=1974&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=71&amp;pages=1366-1370" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.71.4.1366" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.71.4.1366" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.71.4.1366"> </span></div> </div></div></div></div></div><div content-id="b16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hamada</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Petrino</div> <div class="given-names">M. G.</div></div>,  <div class="name"><div class="surname">Kakunaga</div> <div class="given-names">T.</div></div>. </div><div class="article-title">A novel repeated element with Z-DNA-forming potential is widely found in evolutionarily diverse eukaryotic genomes</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">1982</div>, vol. <div class="volume">79</div> (pg. <div class="fpage">6465</div>-<div class="lpage">6469</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20novel%20repeated%20element%20with%20Z-DNA-forming%20potential%20is%20widely%20found%20in%20evolutionarily%20diverse%20eukaryotic%20genomes&amp;author=H.%20Hamada&amp;author=M.%20G.%20Petrino&amp;author=T.%20Kakunaga&amp;publication_year=1982&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=79&amp;pages=6465-6469" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.79.21.6465" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.79.21.6465" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.79.21.6465"> </span></div> </div></div></div></div></div><div content-id="b17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Katti</div> <div class="given-names">M. V.</div></div>,  <div class="name"><div class="surname">Ranjekar</div> <div class="given-names">P. K.</div></div>,  <div class="name"><div class="surname">Gupta</div> <div class="given-names">V. S.</div></div>. </div><div class="article-title">Differential distribution of simple sequence repeats in eukaryotic genome sequences</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2001</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">1161</div>-<div class="lpage">1167</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Differential%20distribution%20of%20simple%20sequence%20repeats%20in%20eukaryotic%20genome%20sequences&amp;author=M.%20V.%20Katti&amp;author=P.%20K.%20Ranjekar&amp;author=V.%20S.%20Gupta&amp;publication_year=2001&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=18&amp;pages=1161-1167" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a003903" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Foxfordjournals.molbev.a003903" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Foxfordjournals.molbev.a003903"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11420357" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tautz</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Hypervariability of simple sequences as a general source for polymorphic DNA markers</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1989</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">6463</div>-<div class="lpage">6471</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Hypervariability%20of%20simple%20sequences%20as%20a%20general%20source%20for%20polymorphic%20DNA%20markers&amp;author=D.%20Tautz&amp;publication_year=1989&amp;journal=Nucleic%20Acids%20Res.&amp;volume=17&amp;pages=6463-6471" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/17.16.6463" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F17.16.6463" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F17.16.6463"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2780284" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tautz</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Schlotterer</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Simple sequences</div>, <div class="source">Curr. Opin. Genet. Dev.</div> , <div class="year">1994</div>, vol. <div class="volume">4</div> (pg. <div class="fpage">832</div>-<div class="lpage">837</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Simple%20sequences&amp;author=D.%20Tautz&amp;author=C.%20Schlotterer&amp;publication_year=1994&amp;journal=Curr.%20Opin.%20Genet.%20Dev.&amp;volume=4&amp;pages=832-837" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0959-437X(94)90067-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0959-437X(94)90067-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0959-437X(94)90067-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7888752" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cox</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Mirkin</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Characteristic enrichment of DNA repeats in different genomes</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">1997</div>, vol. <div class="volume">94</div> (pg. <div class="fpage">5237</div>-<div class="lpage">5242</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characteristic%20enrichment%20of%20DNA%20repeats%20in%20different%20genomes&amp;author=R.%20Cox&amp;author=S.%20Mirkin&amp;publication_year=1997&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=94&amp;pages=5237-5242" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.94.10.5237" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.94.10.5237" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.94.10.5237"> </span></div> </div></div></div></div></div><div content-id="b21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Field</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Wills</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Abundant microsatellite polymorphisms in <em>Saccharomyces cerevisiae</em>, and the different distributions of microsatellites in eight prokaryotes and <em>S. cerevisiae</em>, result from strong mutation pressures and a variety of selective forces</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">1998</div>, vol. <div class="volume">95</div> (pg. <div class="fpage">1647</div>-<div class="lpage">1652</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Abundant%20microsatellite%20polymorphisms%20in%20Saccharomyces%20cerevisiae%2C%20and%20the%20different%20distributions%20of%20microsatellites%20in%20eight%20prokaryotes%20and%20S.%20cerevisiae%2C%20result%20from%20strong%20mutation%20pressures%20and%20a%20variety%20of%20selective%20forces&amp;author=D.%20Field&amp;author=C.%20Wills&amp;publication_year=1998&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=95&amp;pages=1647-1652" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.95.4.1647" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.95.4.1647" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.95.4.1647"> </span></div> </div></div></div></div></div><div content-id="b22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tautz</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Trick</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Dover</div> <div class="given-names">G. A.</div></div>. </div><div class="article-title">Cryptic simplicity is a major source of genetic variation</div>, <div class="source">Nature</div> , <div class="year">1986</div>, vol. <div class="volume">322</div> (pg. <div class="fpage">652</div>-<div class="lpage">656</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cryptic%20simplicity%20is%20a%20major%20source%20of%20genetic%20variation&amp;author=D.%20Tautz&amp;author=M.%20Trick&amp;author=G.%20A.%20Dover&amp;publication_year=1986&amp;journal=Nature&amp;volume=322&amp;pages=652-656" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/322652a0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F322652a0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F322652a0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3748144" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dechering</div> <div class="given-names">K. J.</div></div>,  <div class="name"><div class="surname">Cuelenaere</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Konings</div> <div class="given-names">R. N. H.</div></div>,  <div class="name"><div class="surname">Leunissen</div> <div class="given-names">J. A. M.</div></div>. </div><div class="article-title">Distinct frequency-distributions of homopolymeric DNA tracts in different genomes</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1998</div>, vol. <div class="volume">26</div> (pg. <div class="fpage">4056</div>-<div class="lpage">4062</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Distinct%20frequency-distributions%20of%20homopolymeric%20DNA%20tracts%20in%20different%20genomes&amp;author=K.%20J.%20Dechering&amp;author=K.%20Cuelenaere&amp;author=R.%20N.%20H.%20Konings&amp;author=J.%20A.%20M.%20Leunissen&amp;publication_year=1998&amp;journal=Nucleic%20Acids%20Res.&amp;volume=26&amp;pages=4056-4062" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/26.17.4056" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F26.17.4056" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F26.17.4056"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9705519" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gur-Arie</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Cohen</div> <div class="given-names">C. J.</div></div>,  <div class="name"><div class="surname">Eitan</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Shelef</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Hallerman</div> <div class="given-names">E. M.</div></div>,  <div class="name"><div class="surname">Kashi</div> <div class="given-names">Y.</div></div>. </div><div class="article-title">Simple sequence repeats in <em>Escherichia coli</em>: abundance, distribution, composition, and polymorphism</div>, <div class="source">Genome Res.</div> , <div class="year">2000</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">62</div>-<div class="lpage">71</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Simple%20sequence%20repeats%20in%20Escherichia%20coli%3A%20abundance%2C%20distribution%2C%20composition%2C%20and%20polymorphism&amp;author=R.%20Gur-Arie&amp;author=C.%20J.%20Cohen&amp;author=Y.%20Eitan&amp;author=L.%20Shelef&amp;author=E.%20M.%20Hallerman&amp;author=Y.%20Kashi&amp;publication_year=2000&amp;journal=Genome%20Res.&amp;volume=10&amp;pages=62-71" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10645951" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Coenye</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Vandamme</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Simple sequence repeats and compositional bias in the bipartite <em>Ralstonia solanacearum</em> GMI1000 genome</div>, <div class="source">BMC Genomics</div> , <div class="year">2003</div>, vol. <div class="volume">4</div> pg. <div class="fpage">10</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Simple%20sequence%20repeats%20and%20compositional%20bias%20in%20the%20bipartite%20Ralstonia%20solanacearum%20GMI1000%20genome&amp;author=T.%20Coenye&amp;author=P.%20Vandamme&amp;publication_year=2003&amp;journal=BMC%20Genomics&amp;volume=4&amp;pages=10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-4-10" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-4-10" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-4-10"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12697060" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yang</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Yu</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Dong</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Yao</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Shen</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Jin</div> <div class="given-names">Q.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Identification and characterisation of simple sequence repeats in the genomes <em>of Shigella</em> species</div>, <div class="source">Gene</div> , <div class="year">2003</div>, vol. <div class="volume">322</div> (pg. <div class="fpage">85</div>-<div class="lpage">92</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20and%20characterisation%20of%20simple%20sequence%20repeats%20in%20the%20genomes%20of%20Shigella%20species&amp;author=J.%20Yang&amp;author=J.%20Wang&amp;author=L.%20Chen&amp;author=J.%20Yu&amp;author=J.%20Dong&amp;author=Z.%20Yao&amp;author=Y.%20Shen&amp;author=Q.%20Jin&amp;author=R.%20Chen&amp;publication_year=2003&amp;journal=Gene&amp;volume=322&amp;pages=85-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gene.2003.09.017" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gene.2003.09.017" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gene.2003.09.017"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14644500" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">De Wachter</div> <div class="given-names">R.</div></div>. </div><div class="article-title">The number of repeats expected in random nucleic acid sequences and found in genes</div>, <div class="source">J. Theor. Biol.</div> , <div class="year">1981</div>, vol. <div class="volume">91</div> (pg. <div class="fpage">71</div>-<div class="lpage">98</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20number%20of%20repeats%20expected%20in%20random%20nucleic%20acid%20sequences%20and%20found%20in%20genes&amp;author=R.%20De%20Wachter&amp;publication_year=1981&amp;journal=J.%20Theor.%20Biol.&amp;volume=91&amp;pages=71-98" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0022-5193(81)90375-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0022-5193(81)90375-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0022-5193(81)90375-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7029145" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rutherford</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Parkhill</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Crook</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Horsnell</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Rice</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Rajandream</div> <div class="given-names">M. A.</div></div>,  <div class="name"><div class="surname">Barell</div> <div class="given-names">B.</div></div>. </div><div class="article-title">Artemis Sequence visualisation and annotation</div>, <div class="source">Bioinformatics</div> , <div class="year">2000</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">944</div>-<div class="lpage">945</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Artemis%20Sequence%20visualisation%20and%20annotation&amp;author=K.%20Rutherford&amp;author=J.%20Parkhill&amp;author=J.%20Crook&amp;author=T.%20Horsnell&amp;author=P.%20Rice&amp;author=M.%20A.%20Rajandream&amp;author=B.%20Barell&amp;publication_year=2000&amp;journal=Bioinformatics&amp;volume=16&amp;pages=944-945" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/16.10.944" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2F16.10.944" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2F16.10.944"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11120685" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gevers</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Vandepoele</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Simillon</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Van de Peer</div> <div class="given-names">Y.</div></div>. </div><div class="article-title">Gene duplication and biased functional retention of paralogs in bacterial genomes</div>, <div class="source">Trends Microbiol.</div> , <div class="year">2004</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">148</div>-<div class="lpage">154</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20duplication%20and%20biased%20functional%20retention%20of%20paralogs%20in%20bacterial%20genomes&amp;author=D.%20Gevers&amp;author=K.%20Vandepoele&amp;author=C.%20Simillon&amp;author=Y.%20Van%20de%20Peer&amp;publication_year=2004&amp;journal=Trends%20Microbiol.&amp;volume=12&amp;pages=148-154" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tim.2004.02.007" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tim.2004.02.007" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tim.2004.02.007"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15116722" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nakamura</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Itoh</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Matsuda</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Gojobori</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Biased biological functions of horizontally transferred genes in prokaryotic genomes</div>, <div class="source">Nature Genet.</div> , <div class="year">2004</div>, vol. <div class="volume">36</div> (pg. <div class="fpage">760</div>-<div class="lpage">766</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Biased%20biological%20functions%20of%20horizontally%20transferred%20genes%20in%20prokaryotic%20genomes&amp;author=Y.%20Nakamura&amp;author=T.%20Itoh&amp;author=H.%20Matsuda&amp;author=T.%20Gojobori&amp;publication_year=2004&amp;journal=Nature%20Genet.&amp;volume=36&amp;pages=760-766" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng1381" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng1381" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng1381"> </span></div> </div></div></div></div></div><div content-id="b31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bentley</div> <div class="given-names">S. D.</div></div>,  <div class="name"><div class="surname">Parkhill</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Comparative genomic structure of prokaryotes</div>, <div class="source">Annu. Rev. Genet.</div> , <div class="year">2004</div>, vol. <div class="volume">38</div> (pg. <div class="fpage">771</div>-<div class="lpage">791</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20genomic%20structure%20of%20prokaryotes&amp;author=S.%20D.%20Bentley&amp;author=J.%20Parkhill&amp;publication_year=2004&amp;journal=Annu.%20Rev.%20Genet.&amp;volume=38&amp;pages=771-791" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.genet.38.072902.094318" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.genet.38.072902.094318" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.genet.38.072902.094318"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15568993" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hallin</div> <div class="given-names">P. P.</div></div>,  <div class="name"><div class="surname">Coenye</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Staerfeldt</div> <div class="given-names">H. H.</div></div>,  <div class="name"><div class="surname">Binnewies</div> <div class="given-names">T. T.</div></div>,  <div class="name"><div class="surname">Jarmer</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Ussery</div> <div class="given-names">D. W.</div></div>. </div><div class="article-title">Genome update: Correlation of bacterial genomic properties</div>, <div class="source">Microbiology</div> , <div class="year">2004</div>, vol. <div class="volume">150</div> (pg. <div class="fpage">3899</div>-<div class="lpage">3903</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20update%3A%20Correlation%20of%20bacterial%20genomic%20properties&amp;author=P.%20P.%20Hallin&amp;author=T.%20Coenye&amp;author=H.%20H.%20Staerfeldt&amp;author=T.%20T.%20Binnewies&amp;author=H.%20Jarmer&amp;author=D.%20W.%20Ussery&amp;publication_year=2004&amp;journal=Microbiology&amp;volume=150&amp;pages=3899-3903" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1099/mic.0.27720-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1099%2Fmic.0.27720-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1099%2Fmic.0.27720-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15583141" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">van der Woude</div> <div class="given-names">M. W.</div></div>,  <div class="name"><div class="surname">Baumler</div> <div class="given-names">A. J.</div></div>. </div><div class="article-title">Phase and antigenic variation in bacteria</div>, <div class="source">Clin. Microbiol. Rev.</div> , <div class="year">2004</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">581</div>-<div class="lpage">611</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phase%20and%20antigenic%20variation%20in%20bacteria&amp;author=M.%20W.%20van%20der%20Woude&amp;author=A.%20J.%20Baumler&amp;publication_year=2004&amp;journal=Clin.%20Microbiol.%20Rev.&amp;volume=17&amp;pages=581-611" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/CMR.17.3.581-611.2004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FCMR.17.3.581-611.2004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FCMR.17.3.581-611.2004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15258095" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Saunders</div> <div class="given-names">N. J.</div></div>,  <div class="name"><div class="surname">Peden</div> <div class="given-names">J. F.</div></div>,  <div class="name"><div class="surname">Hood</div> <div class="given-names">D. W.</div></div>,  <div class="name"><div class="surname">Moxon</div> <div class="given-names">E. R.</div></div>. </div><div class="article-title">Simple sequence repeats in the <em>Helicobacter pylori</em> genome</div>, <div class="source">Mol. Microbiol.</div> , <div class="year">1998</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">1091</div>-<div class="lpage">1098</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Simple%20sequence%20repeats%20in%20the%20Helicobacter%20pylori%20genome&amp;author=N.%20J.%20Saunders&amp;author=J.%20F.%20Peden&amp;author=D.%20W.%20Hood&amp;author=E.%20R.%20Moxon&amp;publication_year=1998&amp;journal=Mol.%20Microbiol.&amp;volume=27&amp;pages=1091-1098" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-2958.1998.00768.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-2958.1998.00768.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-2958.1998.00768.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9570395" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Salaun</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Linz</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Suerbaum</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Saunders</div> <div class="given-names">N. J.</div></div>. </div><div class="article-title">The diversity within an expanded and redefined repertoire of phase-variable genes in <em>Helicobacter pylori</em></div>, <div class="source">Microbiology</div> , <div class="year">2004</div>, vol. <div class="volume">150</div> (pg. <div class="fpage">817</div>-<div class="lpage">830</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20diversity%20within%20an%20expanded%20and%20redefined%20repertoire%20of%20phase-variable%20genes%20in%20Helicobacter%20pylori&amp;author=L.%20Salaun&amp;author=B.%20Linz&amp;author=S.%20Suerbaum&amp;author=N.%20J.%20Saunders&amp;publication_year=2004&amp;journal=Microbiology&amp;volume=150&amp;pages=817-830" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1099/mic.0.26993-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1099%2Fmic.0.26993-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1099%2Fmic.0.26993-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15073292" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Linton</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Karlyshev</div> <div class="given-names">A. V.</div></div>,  <div class="name"><div class="surname">Wren</div> <div class="given-names">B. W.</div></div>. </div><div class="article-title">Deciphering <em>Campylobacter jejuni</em> cell surface interactions from the genome sequence</div>, <div class="source">Curr. Opin. Microbiol.</div> , <div class="year">2001</div>, vol. <div class="volume">4</div> (pg. <div class="fpage">35</div>-<div class="lpage">40</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Deciphering%20Campylobacter%20jejuni%20cell%20surface%20interactions%20from%20the%20genome%20sequence&amp;author=D.%20Linton&amp;author=A.%20V.%20Karlyshev&amp;author=B.%20W.%20Wren&amp;publication_year=2001&amp;journal=Curr.%20Opin.%20Microbiol.&amp;volume=4&amp;pages=35-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S1369-5274(00)00161-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS1369-5274(00)00161-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS1369-5274(00)00161-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11173031" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Frank</div> <div class="given-names">A. C.</div></div>,  <div class="name"><div class="surname">Amiri</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Adersson</div> <div class="given-names">S. G.</div></div>. </div><div class="article-title">Genome deterioration: loss of repeated sequences and accumulation of junk DNA</div>, <div class="source">Genetica</div> , <div class="year">2002</div>, vol. <div class="volume">115</div> (pg. <div class="fpage">1</div>-<div class="lpage">12</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20deterioration%3A%20loss%20of%20repeated%20sequences%20and%20accumulation%20of%20junk%20DNA&amp;author=A.%20C.%20Frank&amp;author=H.%20Amiri&amp;author=S.%20G.%20Adersson&amp;publication_year=2002&amp;journal=Genetica&amp;volume=115&amp;pages=1-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1023/A:1016064511533" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1023%2FA:1016064511533" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1023%2FA:1016064511533"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12188042" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dybvig</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Voelker</div> <div class="given-names">L. L.</div></div>. </div><div class="article-title">Molecular biology of mycoplasmas</div>, <div class="source">Annu. Rev. Microbiol.</div> , <div class="year">1996</div>, vol. <div class="volume">50</div> (pg. <div class="fpage">25</div>-<div class="lpage">57</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20biology%20of%20mycoplasmas&amp;author=K.%20Dybvig&amp;author=L.%20L.%20Voelker&amp;publication_year=1996&amp;journal=Annu.%20Rev.%20Microbiol.&amp;volume=50&amp;pages=25-57" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.micro.50.1.25" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.micro.50.1.25" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.micro.50.1.25"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8905075" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wernegreen</div> <div class="given-names">J. J.</div></div>. </div><div class="article-title">Genome evolution in bacterial endosymbionts of insects</div>, <div class="source">Nature Rev. Gen.</div> , <div class="year">2002</div>, vol. <div class="volume">3</div> (pg. <div class="fpage">850</div>-<div class="lpage">861</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20evolution%20in%20bacterial%20endosymbionts%20of%20insects&amp;author=J.%20J.%20Wernegreen&amp;publication_year=2002&amp;journal=Nature%20Rev.%20Gen.&amp;volume=3&amp;pages=850-861" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg931" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg931" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg931"> </span></div> </div></div></div></div></div><div content-id="b40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wells</div> <div class="given-names">R. D.</div></div>. </div><div class="article-title">Molecular basis of genetic instability of triplet repeats</div>, <div class="source">J. Biol. Chem.</div> , <div class="year">1996</div>, vol. <div class="volume">271</div> (pg. <div class="fpage">2875</div>-<div class="lpage">2878</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20basis%20of%20genetic%20instability%20of%20triplet%20repeats&amp;author=R.%20D.%20Wells&amp;publication_year=1996&amp;journal=J.%20Biol.%20Chem.&amp;volume=271&amp;pages=2875-2878" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.271.6.2875" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.271.6.2875" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.271.6.2875"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8621672" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Iran</div> <div class="given-names">H. T.</div></div>,  <div class="name"><div class="surname">Keen</div> <div class="given-names">J. D.</div></div>,  <div class="name"><div class="surname">Kricker</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Resnick</div> <div class="given-names">M. A.</div></div>,  <div class="name"><div class="surname">Gordenin</div> <div class="given-names">D. A.</div></div>. </div><div class="article-title">Hypermutability of homonucleotide runs in mismatch repair and DNA polymerase proofreading yeast mutants</div>, <div class="source">Mol. Cell. Biol.</div> , <div class="year">1997</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">2859</div>-<div class="lpage">2865</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Hypermutability%20of%20homonucleotide%20runs%20in%20mismatch%20repair%20and%20DNA%20polymerase%20proofreading%20yeast%20mutants&amp;author=H.%20T.%20Iran&amp;author=J.%20D.%20Keen&amp;author=M.%20Kricker&amp;author=M.%20A.%20Resnick&amp;author=D.%20A.%20Gordenin&amp;publication_year=1997&amp;journal=Mol.%20Cell.%20Biol.&amp;volume=17&amp;pages=2859-2865" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/MCB.17.5.2859" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FMCB.17.5.2859" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FMCB.17.5.2859"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9111358" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kashi</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">King</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Soller</div> <div class="given-names">M. I.</div></div>. </div><div class="article-title">Simple sequence repeats as a source of quantitative genetic variation</div>, <div class="source">Trends Genet.</div> , <div class="year">1997</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">74</div>-<div class="lpage">78</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Simple%20sequence%20repeats%20as%20a%20source%20of%20quantitative%20genetic%20variation&amp;author=Y.%20Kashi&amp;author=D.%20King&amp;author=M.%20I.%20Soller&amp;publication_year=1997&amp;journal=Trends%20Genet.&amp;volume=13&amp;pages=74-78" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0168-9525(97)01008-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0168-9525(97)01008-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0168-9525(97)01008-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9055609" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pedersen</div> <div class="given-names">A. G.</div></div>,  <div class="name"><div class="surname">Jensen</div> <div class="given-names">L. J.</div></div>,  <div class="name"><div class="surname">Brunak</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Staerfeldt</div> <div class="given-names">H. H.</div></div>,  <div class="name"><div class="surname">Ussery</div> <div class="given-names">D. W.</div></div>. </div><div class="article-title">A DNA structural atlas for <em>Escherichia coli</em></div>, <div class="source">J. Mol. Biol.</div> , <div class="year">2000</div>, vol. <div class="volume">299</div> (pg. <div class="fpage">907</div>-<div class="lpage">930</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20DNA%20structural%20atlas%20for%20Escherichia%20coli&amp;author=A.%20G.%20Pedersen&amp;author=L.%20J.%20Jensen&amp;author=S.%20Brunak&amp;author=H.%20H.%20Staerfeldt&amp;author=D.%20W.%20Ussery&amp;publication_year=2000&amp;journal=J.%20Mol.%20Biol.&amp;volume=299&amp;pages=907-930" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/jmbi.2000.3787" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fjmbi.2000.3787" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fjmbi.2000.3787"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10843847" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ussery</div> <div class="given-names">D. W.</div></div>,  <div class="name"><div class="surname">Soumpasis</div> <div class="given-names">D. M.</div></div>,  <div class="name"><div class="surname">Brunak</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Staerfeldt</div> <div class="given-names">H. H.</div></div>,  <div class="name"><div class="surname">Worning</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Krogh</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Bias of purine stretches in sequenced chromosomes</div>, <div class="source">Comp. Chem.</div> , <div class="year">2002</div>, vol. <div class="volume">26</div> (pg. <div class="fpage">531</div>-<div class="lpage">541</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bias%20of%20purine%20stretches%20in%20sequenced%20chromosomes&amp;author=D.%20W.%20Ussery&amp;author=D.%20M.%20Soumpasis&amp;author=S.%20Brunak&amp;author=H.%20H.%20Staerfeldt&amp;author=P.%20Worning&amp;author=A.%20Krogh&amp;publication_year=2002&amp;journal=Comp.%20Chem.&amp;volume=26&amp;pages=531-541" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0097-8485(02)00013-X" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0097-8485(02)00013-X" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0097-8485(02)00013-X"> </span></div> </div></div></div></div></div><div content-id="b45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pennisi</div> <div class="given-names">E.</div></div>. </div><div class="article-title">How the genome readies itself for evolution</div>, <div class="source">Science</div> , <div class="year">1998</div>, vol. <div class="volume">281</div> (pg. <div class="fpage">1131</div>-<div class="lpage">1134</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=How%20the%20genome%20readies%20itself%20for%20evolution&amp;author=E.%20Pennisi&amp;publication_year=1998&amp;journal=Science&amp;volume=281&amp;pages=1131-1134" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.281.5380.1131" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.281.5380.1131" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.281.5380.1131"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9735027" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b46" class="ref-content "><div class="label -label">46</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">King</div> <div class="given-names">D. G.</div></div>,  <div class="name"><div class="surname">Soller</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Kashi</div> <div class="given-names">Y.</div></div>. </div><div class="article-title">Evolutionary tuning knob</div>, <div class="source">Endeavour</div> , <div class="year">1997</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">36</div>-<div class="lpage">40</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolutionary%20tuning%20knob&amp;author=D.%20G.%20King&amp;author=M.%20Soller&amp;author=Y.%20Kashi&amp;publication_year=1997&amp;journal=Endeavour&amp;volume=21&amp;pages=36-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0160-9327(97)01005-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0160-9327(97)01005-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0160-9327(97)01005-3"> </span></div> </div></div></div></div></div><div content-id="b47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b47" class="ref-content "><div class="label -label">47</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">King</div> <div class="given-names">D. G.</div></div>. </div><div class="person-group"><div class="name"><div class="surname">Wade</div> <div class="given-names">M. J.</div></div>. </div><div class="article-title">Indirect selection on the mutational landscape: An evolved role for the mutability of repetitive DNA?</div>, <div class="year">2000</div> <div class="comment">Evolution 2000, Indiana University Conferences, Bloomington, IN</div></div></div></div></div><div content-id="b48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b48" class="ref-content "><div class="label -label">48</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lawrence</div> <div class="given-names">J. G.</div></div>,  <div class="name"><div class="surname">Ochman</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Amelioration of bacterial genomes: Rates of change and exchange</div>, <div class="source">J. Mol. Evol.</div> , <div class="year">1997</div>, vol. <div class="volume">44</div> (pg. <div class="fpage">383</div>-<div class="lpage">397</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Amelioration%20of%20bacterial%20genomes%3A%20Rates%20of%20change%20and%20exchange&amp;author=J.%20G.%20Lawrence&amp;author=H.%20Ochman&amp;publication_year=1997&amp;journal=J.%20Mol.%20Evol.&amp;volume=44&amp;pages=383-397" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/PL00006158" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FPL00006158" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FPL00006158"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9089078" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b49" class="ref-content "><div class="label -label">49</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sia</div> <div class="given-names">E. A.</div></div>,  <div class="name"><div class="surname">Jinks-Robertson</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Petes</div> <div class="given-names">T. D.</div></div>. </div><div class="article-title">Genetic control of microsatellite stability</div>, <div class="source">Mutat.Res.</div> , <div class="year">1997</div>, vol. <div class="volume">383</div> (pg. <div class="fpage">61</div>-<div class="lpage">70</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20control%20of%20microsatellite%20stability&amp;author=E.%20A.%20Sia&amp;author=S.%20Jinks-Robertson&amp;author=T.%20D.%20Petes&amp;publication_year=1997&amp;journal=Mutat.Res.&amp;volume=383&amp;pages=61-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0921-8777(96)00046-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0921-8777(96)00046-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0921-8777(96)00046-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9042420" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b50" class="ref-content "><div class="label -label">50</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rocha</div> <div class="given-names">E. P.</div></div>,  <div class="name"><div class="surname">Danchin</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Base composition bias might result from competition for metabolic resources</div>, <div class="source">Trends Genet.</div> , <div class="year">2002</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">291</div>-<div class="lpage">294</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Base%20composition%20bias%20might%20result%20from%20competition%20for%20metabolic%20resources&amp;author=E.%20P.%20Rocha&amp;author=A.%20Danchin&amp;publication_year=2002&amp;journal=Trends%20Genet.&amp;volume=18&amp;pages=291-294" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0168-9525(02)02690-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0168-9525(02)02690-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0168-9525(02)02690-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12044357" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b51" class="ref-content "><div class="label -label">51</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schroth</div> <div class="given-names">G. P.</div></div>,  <div class="name"><div class="surname">Ho</div> <div class="given-names">P. S.</div></div>. </div><div class="article-title">Occurrence of potential cruciform and H-DNA forming sequences in genomic DNA</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1995</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">1977</div>-<div class="lpage">1983</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Occurrence%20of%20potential%20cruciform%20and%20H-DNA%20forming%20sequences%20in%20genomic%20DNA&amp;author=G.%20P.%20Schroth&amp;author=P.%20S.%20Ho&amp;publication_year=1995&amp;journal=Nucleic%20Acids%20Res.&amp;volume=23&amp;pages=1977-1983" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/23.11.1977" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F23.11.1977" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F23.11.1977"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7596826" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b52" class="ref-content "><div class="label -label">52</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hoyne</div> <div class="given-names">P. R.</div></div>,  <div class="name"><div class="surname">Edwards</div> <div class="given-names">L. M.</div></div>,  <div class="name"><div class="surname">Viari</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Maher</div> <div class="given-names">L. J.</div></div>. </div><div class="article-title">Searching genomes for sequences with the potential to form intrastrand triple helices</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">2000</div>, vol. <div class="volume">302</div> (pg. <div class="fpage">797</div>-<div class="lpage">809</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Searching%20genomes%20for%20sequences%20with%20the%20potential%20to%20form%20intrastrand%20triple%20helices&amp;author=P.%20R.%20Hoyne&amp;author=L.%20M.%20Edwards&amp;author=A.%20Viari&amp;author=L.%20J.%20Maher&amp;publication_year=2000&amp;journal=J.%20Mol.%20Biol.&amp;volume=302&amp;pages=797-809" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/jmbi.2000.4502" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fjmbi.2000.4502" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fjmbi.2000.4502"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10993724" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b53" class="ref-content "><div class="label -label">53</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ashely</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Lee</div> <div class="given-names">J. S.</div></div>. </div><div class="article-title">A triplex-mediated knot between separated polypurine-polypyrimidine tracts in circular DNA blocks transcription by <em>Escherichia coli</em> RNA polymerase</div>, <div class="source">DNA Cell Biol.</div> , <div class="year">2000</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">235</div>-<div class="lpage">341</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20triplex-mediated%20knot%20between%20separated%20polypurine-polypyrimidine%20tracts%20in%20circular%20DNA%20blocks%20transcription%20by%20Escherichia%20coli%20RNA%20polymerase&amp;author=C.%20Ashely&amp;author=J.%20S.%20Lee&amp;publication_year=2000&amp;journal=DNA%20Cell%20Biol.&amp;volume=19&amp;pages=235-341" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1089/104454900314500" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1089%2F104454900314500" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1089%2F104454900314500"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10798447" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2006. Kazusa DNA Research Institute</div><div class="license"><div class="license-p">The online version of this article has been published under an open access model. Users are entitled to use, reproduce, disseminate, or display the open access version of this article for non-commercial purposes provided that: the original authorship is properly and fully attributed; the Journal and Oxford University Press are attributed as the original place of publication with the correct citation details given; if an article is subsequently reproduced or disseminated not in its entirety but only in part or as a derivative work this must be clearly indicated. For commercial re-use, please contact journals.permissions@oxfordjournals.org</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=621324&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/4/10.1093/dnares/dsi009/2/dsi009-dsi009_TABLES3.doc?Expires=2147483647&amp;Signature=TCucMfdj9YCJeFrhQ1lB-1xUg9tv6DVEO6ts6HDvcmf9pMsS0Ei9v6TsS-QPyL33JTbU3imlNzupaKvYbP5hmu~HYvQe56NmqW10ZHVIUkhbjQ2Br2CCmwa9GvpUB-5r2nFS1vhEyEjm9cQ4w2PW~kYgF0OE-b90Y-62QlnYdJpmsfDxnAQCvJzxTyUABq-Txyezw6f5C3LTLQIemXM~yNCeVXT-pctkRoRqK1yf3rTk8ABK63xPlB6HEqW~smLSC5LkkPs8TzW2HsFMe6DzrIWYUUDhkyt6f90By1jIusxaZMSNoj~iPusSwDSQ1d5P5UJBRUb-1sBp9n-Yz-xW4w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-6</a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/4/10.1093/dnares/dsi009/2/dsi009-dsi009_TABLES2.doc?Expires=2147483647&amp;Signature=VjdvEK6lAA0yaSKZ-aynN1TWkKzNbxURwXVzitl9onZ27MWTFPmbIKEcJ1b5V1e9HqSThXgGKdyJ0dBWSeEdwStP22dCbg1-wTUIgGgaPiJS5XgGDOXBpzQTHCErm4cAdVXGn~o3kNcEDUJnPlebcJZRm1oWr4pYD~1saF4HttOsqbqjOJRzryQztco1T4qekHb~3Gd9DbrXk5nqwu05ALOtXlDlDBCe82hI40NnUJ6BcKbHPogWEYHPav4TOm4lwgiUg~yFqrRG-IljXfkZjJifKElxnXKSpkK72943uIzc~X28UBJAdb66yi9zU-mSX8dhP1eR2APthht4ZvzrqQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-5</a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/4/10.1093/dnares/dsi009/2/dsi009-dsi009_TableS1-SupData.xls?Expires=2147483647&amp;Signature=pOrU6JR2SWtUNHTe447~H~Q~b2NbBZXdHW0ZJvUnGrIV55J63OJpRZuwNZ63dAQG-XKnzyryAa5oI-qOYj3tMxvGizZ-BYxrSsqZTXktG2frmXANsCyIPng6-4CuNPK8VbBkZJrEomOfiVqrM4~xtEIvMy6lVj9ltCLP4MtRlMdl47APxnu3nbMjbXtuZA75c4QLdyB32~S8POhVSGkGx4yzOWxv0OXkWTZnbjcVfMrL4VHFEQGcLTAmjZkmLuCswjiKpCeSAIORLfwxoUymB7bKf6xRUXt5vJwAxByxt2cIgdWPRcij2YOsJXOnTym6Zfe2vPEWnsOW9cUJm51y2g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-4</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/4/10.1093/dnares/dsi009/2/dsi009-dsi009_Supplementary_FigS3.pdf?Expires=2147483647&amp;Signature=a6rquS7JdkfpvQzg6hWKizNKJ6QXQbc1HJNiv6iAyeB1UTzMKH57VcohMZ6aG8gBipDJrKTxIaFonIIvNgtwfpYygipCe4Sv5uvPV3Dx3AIQQkPi5D9NNwsEeiXSjWMW6jDhNxFJ9v0YvJsZcnKqqtIx3ZpTit9NAG~aE-Gax8dc6mz9RTrM07e39pyymdLzxADAUEKLUvQa7pNeN7qnCegrHF7exV6RGED6Ghm1le1XyTjhmWkeppq2igxjJ5qQCOxTO6i6jjnA~eY9Hg3vaeyV9JxiHt9c72fbRotNs2WOvvd8EYkO3JEIYI0HVPxxCLD955GZoAoHuy4HoVteJg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/4/10.1093/dnares/dsi009/2/dsi009-dsi009_Supplementary_FigS2.pdf?Expires=2147483647&amp;Signature=r-Ydo13kiWnK~x9OCNstXvgIZJepzAR7xb8K63KxBDsj18m91~S75k1OpvtU~Lyf3OQh1-99sapOkCevHvbViY7cpSUmPadsb4ITCV1LEn7-hYDv1KVji5XXid~NlOO8Xn7kbsmDaXckbOeXWpfAM7QrcEDihwk6g7KSdokaCKLMfSORTZg2Hxsw4EP-y5XZTwwffACZ7hCbJjkT0Qi-6eU7uuVqYOwgB2vqvyGhsww98GGjSFC-UMiya4OwujfApii7fetQz25sLPLoIepMO4qJOisnxolvNo-iZsJ1tvY32VcVGR~O3Q87tVg8stm9bxu3fQPq9Vk1grdJ-9V6FQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/4/10.1093/dnares/dsi009/2/dsi009-dsi009_Supplementary_FigS1.pdf?Expires=2147483647&amp;Signature=nhUFMIsxlvefvdPb6o~6QBMR3-AQbjfQ2LSxA9zAbit9LOx1STfENdhTcJbRT6MKqzNipHg~Tmx65QGKzuSgdkEGYo5GYMOzP2EMCaJ0dtqTIvRM2oPsOXWe7k3z~~Yh4sCeld8lGyh0krvvDnNX7Vuk~mUot~t4lQ7JCWw89v7W3NJq5FuOFKXC4pBq9ZLtLPe2zB77xD97LLXEC9n-aGvrVNRXLVHe1duChskOx7qmckw9ba9phx~4lUlCdYddU42tLVTCxXKEBj~RyfpcY4SZ-h3mX6knGRU8bQuCV9FD7qP-THc3R1VRZFr6KvuQSM9RzLduGTdu8SB1wEKTZw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">366</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000242119700001&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">38</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsi009"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="621324"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000242119700001&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f12%2f4%2f221%2f621324">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000242119700001&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (38)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F12%2F4%2F221%2F621324" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/621324'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Characterization of Mononucleotide Repeats in Sequenced Prokaryotic Genomes" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsi009");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f12%2f4%2f221%2f621324">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

