

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/24/2/117/2949521","@type":"ScholarlyArticle","name":"DNA double-strand break repair in  Penaeus monodon  is predominantly dependent on homologous recombination","datePublished":"2017-01-24","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/24/2","@type":"PublicationIssue","issueNumber":"2","datePublished":"2017-04-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsw059","keywords":["Nonhomologous DNA end joining","MMEJ","Microhomology","Double-strand break","homologous recombination"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/24/2/117/2949521","author":[{"name":"Srivastava, Shikha","affiliation":"1 Department of Biochemistry, Indian Institute of Science, Bangalore 560 012, India","@type":"Person"},{"name":"Dahal, Sumedha","affiliation":"1 Department of Biochemistry, Indian Institute of Science, Bangalore 560 012, India","@type":"Person"},{"name":"Naidu, Sharanya J.","affiliation":"1 Department of Biochemistry, Indian Institute of Science, Bangalore 560 012, India","@type":"Person"},{"name":"Anand, Deepika","affiliation":"2 ICAR-Central Institute of Fisheries Education, Mumbai 400 061, India","@type":"Person"},{"name":"Gopalakrishnan, Vidya","affiliation":"1 Department of Biochemistry, Indian Institute of Science, Bangalore 560 012, India","@type":"Person"},{"name":"Kooloth Valappil, Rajendran","affiliation":"2 ICAR-Central Institute of Fisheries Education, Mumbai 400 061, India","@type":"Person"},{"name":"Raghavan, Sathees C.","affiliation":"1 Department of Biochemistry, Indian Institute of Science, Bangalore 560 012, India","@type":"Person"}],"description":"Abstract.  DNA double-strand breaks (DSBs) are mostly repaired by nonhomologous end joining (NHEJ) and homologous recombination (HR) in higher eukaryotes. In co","pageStart":"117","pageEnd":"128","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/2/2/m_dnares_24_2cover.png?Expires=2147483647&Signature=C8sLyIyfepNN-Oy5BR2xLoVJLx2W1jQBt~48a2lQ~mjys3EWTAKP5k2kV~t7b8sG~Qk1B4iVdz8BINFP5bR-Hc0gvwqekGHM8qavh8kDaLAfN~qNxrDj7NNeLHJ-IURMi5v1ATu23u5wB01wP27BHknbA3W9JgGnq6AVGPqeLfhK7AJcTzOYzKpGYwaedh5crkYXYHp~9GCcHuTjoInf3sKaYC2Ew-HnuA-yIL45j8ocXrNQuIxJV8rFXwx1Tt0b7I1ZJg09R4-7m-GZB9vPgdehgAXUMUNQ4gDatBY22xUt9COqnlb7RZKNUXFDo7Qe0LNJgSHwNUOJvsh-SLpXkg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"DNA double-strand break repair in  Penaeus monodon  is predominantly dependent on homologous recombination","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/2/2/m_dnares_24_2cover.png?Expires=2147483647&Signature=C8sLyIyfepNN-Oy5BR2xLoVJLx2W1jQBt~48a2lQ~mjys3EWTAKP5k2kV~t7b8sG~Qk1B4iVdz8BINFP5bR-Hc0gvwqekGHM8qavh8kDaLAfN~qNxrDj7NNeLHJ-IURMi5v1ATu23u5wB01wP27BHknbA3W9JgGnq6AVGPqeLfhK7AJcTzOYzKpGYwaedh5crkYXYHp~9GCcHuTjoInf3sKaYC2Ew-HnuA-yIL45j8ocXrNQuIxJV8rFXwx1Tt0b7I1ZJg09R4-7m-GZB9vPgdehgAXUMUNQ4gDatBY22xUt9COqnlb7RZKNUXFDo7Qe0LNJgSHwNUOJvsh-SLpXkg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="DNA double-strand break repair in  Penaeus monodon  is predominantly dependent on homologous recombination" />
    <meta property="og:description" content="Abstract.  DNA double-strand breaks (DSBs) are mostly repaired by nonhomologous end joining (NHEJ) and homologous recombination (HR) in higher eukaryotes. In co" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/24/2/117/2949521" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2017-01-24" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Srivastava, Shikha" /><meta name="citation_author_institution" content="Department of Biochemistry, Indian Institute of Science, Bangalore 560 012, India" /><meta name="citation_author" content="Dahal, Sumedha" /><meta name="citation_author_institution" content="Department of Biochemistry, Indian Institute of Science, Bangalore 560 012, India" /><meta name="citation_author" content="Naidu, Sharanya J." /><meta name="citation_author_institution" content="Department of Biochemistry, Indian Institute of Science, Bangalore 560 012, India" /><meta name="citation_author" content="Anand, Deepika" /><meta name="citation_author_institution" content="ICAR-Central Institute of Fisheries Education, Mumbai 400 061, India" /><meta name="citation_author" content="Gopalakrishnan, Vidya" /><meta name="citation_author_institution" content="Department of Biochemistry, Indian Institute of Science, Bangalore 560 012, India" /><meta name="citation_author" content="Kooloth Valappil, Rajendran" /><meta name="citation_author_institution" content="ICAR-Central Institute of Fisheries Education, Mumbai 400 061, India" /><meta name="citation_author" content="Raghavan, Sathees C." /><meta name="citation_author_institution" content="Department of Biochemistry, Indian Institute of Science, Bangalore 560 012, India" /><meta name="citation_title" content="DNA double-strand break repair in Penaeus monodon is predominantly dependent on homologous recombination" /><meta name="citation_firstpage" content="117" /><meta name="citation_lastpage" content="128" /><meta name="citation_doi" content="10.1093/dnares/dsw059" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="24" /><meta name="citation_issue" content="2" /><meta name="citation_publication_date" content="2017/04/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=DNA double-strand break repair inhibitors as cancer therapeutics;  citation_journal_title=Chem. Biol;  citation_year=2015;  citation_volume=22;  citation_pages=17-29; " /><meta name="citation_reference" content="citation_title=DNA repair: from molecular mechanism to human disease;  citation_journal_title=DNA Repair;  citation_year=2006;  citation_volume=5;  citation_pages=986-96; " /><meta name="citation_reference" content="citation_title=Molecular aspects of DNA repair;  citation_journal_title=Mutation Res;  citation_year=1987;  citation_volume=184;  citation_pages=67-86; " /><meta name="citation_reference" content="citation_title=Chromosomal translocations among the healthy human population: implications in oncogenesis;  citation_journal_title=Cell. Mol. Life Sci;  citation_year=2013;  citation_volume=70;  citation_pages=1381-92; " /><meta name="citation_reference" content="citation_title=Nonhomologous DNA end joining in cell-free extracts;  citation_journal_title=J. Nucleic Acids;  citation_year=2010; " /><meta name="citation_reference" content="citation_title=Roles of nonhomologous DNA end joining, V(D)J recombination, and class switch recombination in chromosomal translocations;  citation_journal_title=DNA Repair;  citation_year=2006;  citation_volume=5;  citation_pages=1234-45; " /><meta name="citation_reference" content="citation_title=Alternative end-joining repair pathways are the ultimate backup for abrogated classical non-homologous end-joining and homologous recombination repair: implications for the formation of chromosome translocations;  citation_journal_title=Mutat. Res. Genet. Toxicol. Environ. Mutagen;  citation_year=2015;  citation_volume=793;  citation_pages=166-75; " /><meta name="citation_reference" content="citation_title=How does DNA break during chromosomal translocations?;  citation_journal_title=Nucleic Acids Res;  citation_year=2011;  citation_volume=39;  citation_pages=5813-25; " /><meta name="citation_reference" content="citation_title=Snaps and mends: DNA breaks and chromosomal translocations;  citation_journal_title=FEBS J;  citation_year=2015;  citation_volume=282;  citation_pages=2627-45; " /><meta name="citation_reference" content="citation_title=DNA double-strand break repair: all's well that ends well;  citation_journal_title=Annu. Rev. Genet;  citation_year=2006;  citation_volume=40;  citation_pages=363-83; " /><meta name="citation_reference" content="citation_title=Mechanistic aspects of lymphoid chromosomal translocations;  citation_journal_title=J. Natl. Cancer Inst. Monogr;  citation_year=2008;  citation_volume=39;  citation_pages=8-11; " /><meta name="citation_reference" content="citation_title=Mechanisms that promote and suppress chromosomal translocations in lymphocytes;  citation_journal_title=Ann. Rev. Immunol;  citation_year=2011;  citation_volume=29;  citation_pages=319-50; " /><meta name="citation_reference" content="citation_title=NHEJ and its backup pathways in chromosomal translocations;  citation_journal_title=Nat. Struct. Mol. Biol;  citation_year=2010;  citation_volume=17;  citation_pages=393-5; " /><meta name="citation_reference" content="citation_title=Playing the end game: DNA double-strand break repair pathway choice;  citation_journal_title=Mol. Cell;  citation_year=2012;  citation_volume=47;  citation_pages=497-510; " /><meta name="citation_reference" content="citation_title=Mechanism and regulation of human non-homologous DNA end-joining;  citation_journal_title=Nat. Rev. Mol. Cell. Biol;  citation_year=2003;  citation_volume=4;  citation_pages=712-20; " /><meta name="citation_reference" content="citation_title=The DNA damage response: making it safe to play with knives;  citation_journal_title=Mol. Cell;  citation_year=2010;  citation_volume=40;  citation_pages=179-204; " /><meta name="citation_reference" content="citation_title=Mitosis inhibits DNA double-strand break repair to guard against telomere fusions;  citation_journal_title=Science;  citation_year=2014;  citation_volume=344;  citation_pages=189-93; " /><meta name="citation_reference" content="citation_title=Alternative end-joining is suppressed by the canonical NHEJ component Xrcc4-ligase IV during chromosomal translocation formation;  citation_journal_title=Nat. Struct. Mol. Biol;  citation_year=2010;  citation_volume=17;  citation_pages=410-6; " /><meta name="citation_reference" content="citation_title=Backup pathways of NHEJ in cells of higher eukaryotes: cell cycle dependence;  citation_journal_title=Radiother. Oncol;  citation_year=2009;  citation_volume=92;  citation_pages=310-5; " /><meta name="citation_reference" content="citation_title=Enhanced use of backup pathways of NHEJ in G2 in Chinese hamster mutant cells with defects in the classical pathway of NHEJ;  citation_journal_title=Radiat. Res;  citation_year=2008;  citation_volume=170;  citation_pages=512-20; " /><meta name="citation_reference" content="citation_title=Time-dependent predominance of nonhomologous DNA end-joining pathways during embryonic development in mice;  citation_journal_title=J. Mol. Biol;  citation_year=2012;  citation_volume=417;  citation_pages=197-211; " /><meta name="citation_reference" content="citation_title=Homology and enzymatic requirements of microhomology-dependent alternative end joining;  citation_journal_title=Cell Death Dis;  citation_year=2015;  citation_volume=6;  citation_pages=e1697" /><meta name="citation_reference" content="citation_title=Disease will limit future food supply from the global crustacean fishery and aquaculture sectors;  citation_journal_title=J. Invertebr. Pathol;  citation_year=2012;  citation_volume=110;  citation_pages=141-57; " /><meta name="citation_reference" content="citation_title=Historic emergence, impact and current status of shrimp pathogens in Asia;  citation_journal_title=J. Invertebr. Pathol;  citation_year=2012;  citation_volume=110;  citation_pages=166-73; " /><meta name="citation_reference" content="citation_title=Deciphering the DNA repair protein, Rad23 from kuruma shrimp Marsupenaeus japonicus: full-length cDNA cloning and characterization;  citation_journal_title=Lett. Appl. Microbiol;  citation_year=2011;  citation_volume=53;  citation_pages=63-72; " /><meta name="citation_reference" content="citation_title=Expression pattern analysis of DNA repair-related and DNA damage response genes revealed by 55K oligomicroarray upon UV-B irradiation in the intertidal copepod, Tigriopus japonicus;  citation_journal_title=Comp. Biochem. Physiol. C. Toxicol. Pharmacol;  citation_year=2012;  citation_volume=155;  citation_pages=359-68; " /><meta name="citation_reference" content="citation_title=Temperature effects on survival and DNA repair in four freshwater cladoceran Daphnia species exposed to UV radiation;  citation_journal_title=Photochem. Photobiol;  citation_year=2009;  citation_volume=85;  citation_pages=144-52; " /><meta name="citation_reference" content="citation_title=DNA repair in Drosophila: insights from the Drosophila genome sequence;  citation_journal_title=J. Cell Biol;  citation_year=2000;  citation_volume=150;  citation_pages=F31-6; " /><meta name="citation_reference" content="citation_title=Drosophila IRBP/Ku p70 corresponds to the mutagen-sensitive mus309 gene and is involved in P-element excision in vivo;  citation_journal_title=Genes Dev;  citation_year=1996;  citation_volume=10;  citation_pages=921-33; " /><meta name="citation_reference" content="citation_title=Toll-pathway in tiger shrimp (Penaeus monodon) responds to white spot syndrome virus infection: evidence through molecular characterisation and expression profiles of MyD88, TRAF6 and TLR genes;  citation_journal_title=Fish Shellfish Immunol;  citation_year=2014;  citation_volume=41;  citation_pages=441-54; " /><meta name="citation_reference" content="citation_title=Formation of a G-quadruplex at the BCL2 major breakpoint region of the t(14;18) translocation in follicular lymphoma;  citation_journal_title=Nucleic Acids Res;  citation_year=2011;  citation_volume=39;  citation_pages=936-48; " /><meta name="citation_reference" content="citation_title=Mechanism of fragility at BCL2 gene minor breakpoint cluster region during t(14;18) chromosomal translocation;  citation_journal_title=J. Biol. Chem;  citation_year=2012;  citation_volume=287;  citation_pages=8688-701; " /><meta name="citation_reference" content="citation_title=Anti-apoptotic protein BCL2 down-regulates DNA end joining in cancer cells;  citation_journal_title=J. Biol. Chem;  citation_year=2010;  citation_volume=285;  citation_pages=32657-70; " /><meta name="citation_reference" content="citation_title=Microhomology-mediated end joining is the principal mediator of double-strand break repair during mitochondrial DNA lesions;  citation_journal_title=Mol. Biol. Cell;  citation_year=2016;  citation_volume=27;  citation_pages=223-35; " /><meta name="citation_reference" content="citation_title=Efficiency of nonhomologous DNA end joining varies among somatic tissues, despite similarity in mechanism;  citation_journal_title=Cell. Mol. Life Sci;  citation_year=2011;  citation_volume=68;  citation_pages=661-76; " /><meta name="citation_reference" content="citation_title=DNA end-joining catalyzed by human cell-free extracts;  citation_journal_title=Proc. Natl. Acad. Sci. U.S.A;  citation_year=1998;  citation_volume=95;  citation_pages=14066-70; " /><meta name="citation_reference" content="citation_title=Mouse testicular extracts process DNA double-strand breaks efficiently by DNA end-to-end joining;  citation_journal_title=Mutat. Res;  citation_year=1999;  citation_volume=433;  citation_pages=1-13; " /><meta name="citation_reference" content="citation_title=Nonhomologous end joining of complementary and noncomplementary DNA termini in mouse testicular extracts;  citation_journal_title=DNA Repair;  citation_year=2004;  citation_volume=3;  citation_pages=1297-310; " /><meta name="citation_reference" content="citation_title=An inhibitor of nonhomologous end-joining abrogates double-strand break repair and impedes cancer progression;  citation_journal_title=Cell;  citation_year=2012;  citation_volume=151;  citation_pages=1474-87; " /><meta name="citation_reference" content="citation_title=A plasmid system to monitor gene conversion and reciprocal recombination in vitro;  citation_journal_title=Mutation Res;  citation_year=1993;  citation_volume=291;  citation_pages=181-92; " /><meta name="citation_reference" content="citation_title=Homologous recombination-mediated double-strand break repair in mouse testicular extracts and comparison with different germ cell stages;  citation_journal_title=Cell Biochem. Funct;  citation_year=2007;  citation_volume=25;  citation_pages=75-86; " /><meta name="citation_reference" content="citation_title=Double-strand break repair assays determine pathway choice and structure of gene conversion events in Drosophila melanogaster;  citation_journal_title=G3 (Bethesda);  citation_year=2014;  citation_volume=4;  citation_pages=425-32; " /><meta name="citation_reference" content="citation_title=A recA-like gene in Drosophila melanogaster that is expressed at high levels in female but not male meiotic tissues;  citation_journal_title=Chromosoma;  citation_year=1996;  citation_volume=104;  citation_pages=479-488; " /><meta name="citation_reference" content="citation_title=okra and spindle-B encode components of the RAD52 DNA repair pathway and affect meiosis and patterning in Drosophila oogenesis;  citation_journal_title=Genes Dev;  citation_year=1998;  citation_volume=12;  citation_pages=2711-23; " /><meta name="citation_reference" content="citation_title=Dual roles for DNA polymerase theta in alternative end-joining repair of double-strand breaks in Drosophila;  citation_journal_title=PLoS Genet;  citation_year=2010;  citation_volume=6;  citation_pages=e1001005" /><meta name="citation_reference" content="citation_title=Synthesis-dependent microhomology-mediated end joining accounts for multiple types of repair junctions;  citation_journal_title=Nucleic Acids Res;  citation_year=2010;  citation_volume=38;  citation_pages=5706-17; " /><meta name="citation_reference" content="citation_title=Role of recombination activating genes in the generation of antigen receptor diversity and beyond;  citation_journal_title=Immunology;  citation_year=2012;  citation_volume=137;  citation_pages=271-81; " /><meta name="citation_reference" content="citation_title=Chromosomal translocations in cancer;  citation_journal_title=Biochim. Biophys. Acta;  citation_year=2008;  citation_volume=1786;  citation_pages=139-52; " /><meta name="citation_reference" content="citation_title=Cellular DNA repair cofactors affecting hepatitis B virus infection and replication;  citation_journal_title=World J. Gastroenterol;  citation_year=2008;  citation_volume=14;  citation_pages=5059-65; " /><meta name="citation_reference" content="citation_title=Multiple regulators of Ty1 transposition in Saccharomyces cerevisiae have conserved roles in genome maintenance;  citation_journal_title=Genetics;  citation_year=2001;  citation_volume=159;  citation_pages=1449-65; " /><meta name="citation_reference" content="citation_title=Retroviruses and yeast retrotransposons use overlapping sets of host genes;  citation_journal_title=Genome Res;  citation_year=2005;  citation_volume=15;  citation_pages=641-54; " /><meta name="citation_reference" content="citation_title=The DNA repair genes XPB and XPD defend cells from retroviral infection;  citation_journal_title=Proc. Natl. Acad. Sci. U.S.A;  citation_year=2006;  citation_volume=103;  citation_pages=4622-7; " /><meta name="citation_reference" content="citation_title=Double-strand break end resection and repair pathway choice;  citation_journal_title=Annu. Rev. Genet;  citation_year=2011;  citation_volume=45;  citation_pages=247-71; " /><meta name="citation_reference" content="citation_title=End-joining repair of double-strand breaks in Drosophila melanogaster is largely DNA ligase IV independent;  citation_journal_title=Genetics;  citation_year=2004;  citation_volume=168;  citation_pages=2067-76; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/24/2/117/13688944/dsw059.pdf" /><meta name="description" content="Abstract.  DNA double-strand breaks (DSBs) are mostly repaired by nonhomologous end joining (NHEJ) and homologous recombination (HR) in higher eukaryotes. In co" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/24/2/117/2949521" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/24/2/117/2949521" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/24/2/117/2949521");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsw059';
    </script>
<title>
	DNA double-strand break repair in Penaeus monodon is predominantly dependent on homologous recombination | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./2949521" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BfSBATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU3JpdmFzdGF2YSwgU2hpa2hhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBCaW9jaGVtaXN0cnksIEluZGlhbiBJbnN0aXR1dGUgb2YgU2NpZW5jZSwgQmFuZ2Fsb3JlIDU2MCAwMTIsIEluZGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iRGFoYWwsIFN1bWVkaGEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEJpb2NoZW1pc3RyeSwgSW5kaWFuIEluc3RpdHV0ZSBvZiBTY2llbmNlLCBCYW5nYWxvcmUgNTYwIDAxMiwgSW5kaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJOYWlkdSwgU2hhcmFueWEgSi4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEJpb2NoZW1pc3RyeSwgSW5kaWFuIEluc3RpdHV0ZSBvZiBTY2llbmNlLCBCYW5nYWxvcmUgNTYwIDAxMiwgSW5kaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJBbmFuZCwgRGVlcGlrYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IklDQVItQ2VudHJhbCBJbnN0aXR1dGUgb2YgRmlzaGVyaWVzIEVkdWNhdGlvbiwgTXVtYmFpIDQwMCAwNjEsIEluZGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iR29wYWxha3Jpc2huYW4sIFZpZHlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBCaW9jaGVtaXN0cnksIEluZGlhbiBJbnN0aXR1dGUgb2YgU2NpZW5jZSwgQmFuZ2Fsb3JlIDU2MCAwMTIsIEluZGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS29vbG90aCBWYWxhcHBpbCwgUmFqZW5kcmFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSUNBUi1DZW50cmFsIEluc3RpdHV0ZSBvZiBGaXNoZXJpZXMgRWR1Y2F0aW9uLCBNdW1iYWkgNDAwIDA2MSwgSW5kaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJSYWdoYXZhbiwgU2F0aGVlcyBDLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQmlvY2hlbWlzdHJ5LCBJbmRpYW4gSW5zdGl0dXRlIG9mIFNjaWVuY2UsIEJhbmdhbG9yZSA1NjAgMDEyLCBJbmRpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iRE5BIGRvdWJsZS1zdHJhbmQgYnJlYWsgcmVwYWlyIGluIFBlbmFldXMgbW9ub2RvbiBpcyBwcmVkb21pbmFudGx5IGRlcGVuZGVudCBvbiBob21vbG9nb3VzIHJlY29tYmluYXRpb24iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIxMTciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjEyOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzdzA1OSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIyNCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDE3LzA0LzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RE5BIGRvdWJsZS1zdHJhbmQgYnJlYWsgcmVwYWlyIGluaGliaXRvcnMgYXMgY2FuY2VyIHRoZXJhcGV1dGljczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q2hlbS4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0yMjsgIGNpdGF0aW9uX3BhZ2VzPTE3LTI5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1ETkEgcmVwYWlyOiBmcm9tIG1vbGVjdWxhciBtZWNoYW5pc20gdG8gaHVtYW4gZGlzZWFzZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RE5BIFJlcGFpcjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9OTg2LTk2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Nb2xlY3VsYXIgYXNwZWN0cyBvZiBETkEgcmVwYWlyOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1NdXRhdGlvbiBSZXM7ICBjaXRhdGlvbl95ZWFyPTE5ODc7ICBjaXRhdGlvbl92b2x1bWU9MTg0OyAgY2l0YXRpb25fcGFnZXM9NjctODY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNocm9tb3NvbWFsIHRyYW5zbG9jYXRpb25zIGFtb25nIHRoZSBoZWFsdGh5IGh1bWFuIHBvcHVsYXRpb246IGltcGxpY2F0aW9ucyBpbiBvbmNvZ2VuZXNpczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q2VsbC4gTW9sLiBMaWZlIFNjaTsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT03MDsgIGNpdGF0aW9uX3BhZ2VzPTEzODEtOTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5vbmhvbW9sb2dvdXMgRE5BIGVuZCBqb2luaW5nIGluIGNlbGwtZnJlZSBleHRyYWN0czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gTnVjbGVpYyBBY2lkczsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Um9sZXMgb2Ygbm9uaG9tb2xvZ291cyBETkEgZW5kIGpvaW5pbmcsIFYoRClKIHJlY29tYmluYXRpb24sIGFuZCBjbGFzcyBzd2l0Y2ggcmVjb21iaW5hdGlvbiBpbiBjaHJvbW9zb21hbCB0cmFuc2xvY2F0aW9uczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RE5BIFJlcGFpcjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9MTIzNC00NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWx0ZXJuYXRpdmUgZW5kLWpvaW5pbmcgcmVwYWlyIHBhdGh3YXlzIGFyZSB0aGUgdWx0aW1hdGUgYmFja3VwIGZvciBhYnJvZ2F0ZWQgY2xhc3NpY2FsIG5vbi1ob21vbG9nb3VzIGVuZC1qb2luaW5nIGFuZCBob21vbG9nb3VzIHJlY29tYmluYXRpb24gcmVwYWlyOiBpbXBsaWNhdGlvbnMgZm9yIHRoZSBmb3JtYXRpb24gb2YgY2hyb21vc29tZSB0cmFuc2xvY2F0aW9uczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TXV0YXQuIFJlcy4gR2VuZXQuIFRveGljb2wuIEVudmlyb24uIE11dGFnZW47ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9NzkzOyAgY2l0YXRpb25fcGFnZXM9MTY2LTc1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ib3cgZG9lcyBETkEgYnJlYWsgZHVyaW5nIGNocm9tb3NvbWFsIHRyYW5zbG9jYXRpb25zPzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9Mzk7ICBjaXRhdGlvbl9wYWdlcz01ODEzLTI1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TbmFwcyBhbmQgbWVuZHM6IEROQSBicmVha3MgYW5kIGNocm9tb3NvbWFsIHRyYW5zbG9jYXRpb25zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1GRUJTIEo7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MjgyOyAgY2l0YXRpb25fcGFnZXM9MjYyNy00NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RE5BIGRvdWJsZS1zdHJhbmQgYnJlYWsgcmVwYWlyOiBhbGwncyB3ZWxsIHRoYXQgZW5kcyB3ZWxsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Bbm51LiBSZXYuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTQwOyAgY2l0YXRpb25fcGFnZXM9MzYzLTgzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NZWNoYW5pc3RpYyBhc3BlY3RzIG9mIGx5bXBob2lkIGNocm9tb3NvbWFsIHRyYW5zbG9jYXRpb25zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBOYXRsLiBDYW5jZXIgSW5zdC4gTW9ub2dyOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTM5OyAgY2l0YXRpb25fcGFnZXM9OC0xMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWVjaGFuaXNtcyB0aGF0IHByb21vdGUgYW5kIHN1cHByZXNzIGNocm9tb3NvbWFsIHRyYW5zbG9jYXRpb25zIGluIGx5bXBob2N5dGVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Bbm4uIFJldi4gSW1tdW5vbDsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0yOTsgIGNpdGF0aW9uX3BhZ2VzPTMxOS01MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TkhFSiBhbmQgaXRzIGJhY2t1cCBwYXRod2F5cyBpbiBjaHJvbW9zb21hbCB0cmFuc2xvY2F0aW9uczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBTdHJ1Y3QuIE1vbC4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xNzsgIGNpdGF0aW9uX3BhZ2VzPTM5My01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QbGF5aW5nIHRoZSBlbmQgZ2FtZTogRE5BIGRvdWJsZS1zdHJhbmQgYnJlYWsgcmVwYWlyIHBhdGh3YXkgY2hvaWNlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NDc7ICBjaXRhdGlvbl9wYWdlcz00OTctNTEwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NZWNoYW5pc20gYW5kIHJlZ3VsYXRpb24gb2YgaHVtYW4gbm9uLWhvbW9sb2dvdXMgRE5BIGVuZC1qb2luaW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIFJldi4gTW9sLiBDZWxsLiBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTQ7ICBjaXRhdGlvbl9wYWdlcz03MTItMjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBETkEgZGFtYWdlIHJlc3BvbnNlOiBtYWtpbmcgaXQgc2FmZSB0byBwbGF5IHdpdGgga25pdmVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NDA7ICBjaXRhdGlvbl9wYWdlcz0xNzktMjA0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NaXRvc2lzIGluaGliaXRzIEROQSBkb3VibGUtc3RyYW5kIGJyZWFrIHJlcGFpciB0byBndWFyZCBhZ2FpbnN0IHRlbG9tZXJlIGZ1c2lvbnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MzQ0OyAgY2l0YXRpb25fcGFnZXM9MTg5LTkzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbHRlcm5hdGl2ZSBlbmQtam9pbmluZyBpcyBzdXBwcmVzc2VkIGJ5IHRoZSBjYW5vbmljYWwgTkhFSiBjb21wb25lbnQgWHJjYzQtbGlnYXNlIElWIGR1cmluZyBjaHJvbW9zb21hbCB0cmFuc2xvY2F0aW9uIGZvcm1hdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBTdHJ1Y3QuIE1vbC4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xNzsgIGNpdGF0aW9uX3BhZ2VzPTQxMC02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CYWNrdXAgcGF0aHdheXMgb2YgTkhFSiBpbiBjZWxscyBvZiBoaWdoZXIgZXVrYXJ5b3RlczogY2VsbCBjeWNsZSBkZXBlbmRlbmNlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1SYWRpb3RoZXIuIE9uY29sOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTkyOyAgY2l0YXRpb25fcGFnZXM9MzEwLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVuaGFuY2VkIHVzZSBvZiBiYWNrdXAgcGF0aHdheXMgb2YgTkhFSiBpbiBHMiBpbiBDaGluZXNlIGhhbXN0ZXIgbXV0YW50IGNlbGxzIHdpdGggZGVmZWN0cyBpbiB0aGUgY2xhc3NpY2FsIHBhdGh3YXkgb2YgTkhFSjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UmFkaWF0LiBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTcwOyAgY2l0YXRpb25fcGFnZXM9NTEyLTIwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaW1lLWRlcGVuZGVudCBwcmVkb21pbmFuY2Ugb2Ygbm9uaG9tb2xvZ291cyBETkEgZW5kLWpvaW5pbmcgcGF0aHdheXMgZHVyaW5nIGVtYnJ5b25pYyBkZXZlbG9wbWVudCBpbiBtaWNlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBNb2wuIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NDE3OyAgY2l0YXRpb25fcGFnZXM9MTk3LTIxMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SG9tb2xvZ3kgYW5kIGVuenltYXRpYyByZXF1aXJlbWVudHMgb2YgbWljcm9ob21vbG9neS1kZXBlbmRlbnQgYWx0ZXJuYXRpdmUgZW5kIGpvaW5pbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNlbGwgRGVhdGggRGlzOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz1lMTY5NyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpc2Vhc2Ugd2lsbCBsaW1pdCBmdXR1cmUgZm9vZCBzdXBwbHkgZnJvbSB0aGUgZ2xvYmFsIGNydXN0YWNlYW4gZmlzaGVyeSBhbmQgYXF1YWN1bHR1cmUgc2VjdG9yczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gSW52ZXJ0ZWJyLiBQYXRob2w7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTEwOyAgY2l0YXRpb25fcGFnZXM9MTQxLTU3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IaXN0b3JpYyBlbWVyZ2VuY2UsIGltcGFjdCBhbmQgY3VycmVudCBzdGF0dXMgb2Ygc2hyaW1wIHBhdGhvZ2VucyBpbiBBc2lhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBJbnZlcnRlYnIuIFBhdGhvbDsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMTA7ICBjaXRhdGlvbl9wYWdlcz0xNjYtNzM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURlY2lwaGVyaW5nIHRoZSBETkEgcmVwYWlyIHByb3RlaW4sIFJhZDIzIGZyb20ga3VydW1hIHNocmltcCBNYXJzdXBlbmFldXMgamFwb25pY3VzOiBmdWxsLWxlbmd0aCBjRE5BIGNsb25pbmcgYW5kIGNoYXJhY3Rlcml6YXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUxldHQuIEFwcGwuIE1pY3JvYmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT01MzsgIGNpdGF0aW9uX3BhZ2VzPTYzLTcyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FeHByZXNzaW9uIHBhdHRlcm4gYW5hbHlzaXMgb2YgRE5BIHJlcGFpci1yZWxhdGVkIGFuZCBETkEgZGFtYWdlIHJlc3BvbnNlIGdlbmVzIHJldmVhbGVkIGJ5IDU1SyBvbGlnb21pY3JvYXJyYXkgdXBvbiBVVi1CIGlycmFkaWF0aW9uIGluIHRoZSBpbnRlcnRpZGFsIGNvcGVwb2QsIFRpZ3Jpb3B1cyBqYXBvbmljdXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNvbXAuIEJpb2NoZW0uIFBoeXNpb2wuIEMuIFRveGljb2wuIFBoYXJtYWNvbDsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xNTU7ICBjaXRhdGlvbl9wYWdlcz0zNTktNjg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRlbXBlcmF0dXJlIGVmZmVjdHMgb24gc3Vydml2YWwgYW5kIEROQSByZXBhaXIgaW4gZm91ciBmcmVzaHdhdGVyIGNsYWRvY2VyYW4gRGFwaG5pYSBzcGVjaWVzIGV4cG9zZWQgdG8gVVYgcmFkaWF0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QaG90b2NoZW0uIFBob3RvYmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT04NTsgIGNpdGF0aW9uX3BhZ2VzPTE0NC01MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RE5BIHJlcGFpciBpbiBEcm9zb3BoaWxhOiBpbnNpZ2h0cyBmcm9tIHRoZSBEcm9zb3BoaWxhIGdlbm9tZSBzZXF1ZW5jZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQ2VsbCBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTE1MDsgIGNpdGF0aW9uX3BhZ2VzPUYzMS02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ecm9zb3BoaWxhIElSQlAvS3UgcDcwIGNvcnJlc3BvbmRzIHRvIHRoZSBtdXRhZ2VuLXNlbnNpdGl2ZSBtdXMzMDkgZ2VuZSBhbmQgaXMgaW52b2x2ZWQgaW4gUC1lbGVtZW50IGV4Y2lzaW9uIGluIHZpdm87ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbmVzIERldjsgIGNpdGF0aW9uX3llYXI9MTk5NjsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTkyMS0zMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VG9sbC1wYXRod2F5IGluIHRpZ2VyIHNocmltcCAoUGVuYWV1cyBtb25vZG9uKSByZXNwb25kcyB0byB3aGl0ZSBzcG90IHN5bmRyb21lIHZpcnVzIGluZmVjdGlvbjogZXZpZGVuY2UgdGhyb3VnaCBtb2xlY3VsYXIgY2hhcmFjdGVyaXNhdGlvbiBhbmQgZXhwcmVzc2lvbiBwcm9maWxlcyBvZiBNeUQ4OCwgVFJBRjYgYW5kIFRMUiBnZW5lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RmlzaCBTaGVsbGZpc2ggSW1tdW5vbDsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT00MTsgIGNpdGF0aW9uX3BhZ2VzPTQ0MS01NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Rm9ybWF0aW9uIG9mIGEgRy1xdWFkcnVwbGV4IGF0IHRoZSBCQ0wyIG1ham9yIGJyZWFrcG9pbnQgcmVnaW9uIG9mIHRoZSB0KDE0OzE4KSB0cmFuc2xvY2F0aW9uIGluIGZvbGxpY3VsYXIgbHltcGhvbWE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTM5OyAgY2l0YXRpb25fcGFnZXM9OTM2LTQ4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NZWNoYW5pc20gb2YgZnJhZ2lsaXR5IGF0IEJDTDIgZ2VuZSBtaW5vciBicmVha3BvaW50IGNsdXN0ZXIgcmVnaW9uIGR1cmluZyB0KDE0OzE4KSBjaHJvbW9zb21hbCB0cmFuc2xvY2F0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBCaW9sLiBDaGVtOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTI4NzsgIGNpdGF0aW9uX3BhZ2VzPTg2ODgtNzAxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbnRpLWFwb3B0b3RpYyBwcm90ZWluIEJDTDIgZG93bi1yZWd1bGF0ZXMgRE5BIGVuZCBqb2luaW5nIGluIGNhbmNlciBjZWxsczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQmlvbC4gQ2hlbTsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0yODU7ICBjaXRhdGlvbl9wYWdlcz0zMjY1Ny03MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWljcm9ob21vbG9neS1tZWRpYXRlZCBlbmQgam9pbmluZyBpcyB0aGUgcHJpbmNpcGFsIG1lZGlhdG9yIG9mIGRvdWJsZS1zdHJhbmQgYnJlYWsgcmVwYWlyIGR1cmluZyBtaXRvY2hvbmRyaWFsIEROQSBsZXNpb25zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEJpb2wuIENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9Mjc7ICBjaXRhdGlvbl9wYWdlcz0yMjMtMzU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVmZmljaWVuY3kgb2Ygbm9uaG9tb2xvZ291cyBETkEgZW5kIGpvaW5pbmcgdmFyaWVzIGFtb25nIHNvbWF0aWMgdGlzc3VlcywgZGVzcGl0ZSBzaW1pbGFyaXR5IGluIG1lY2hhbmlzbTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q2VsbC4gTW9sLiBMaWZlIFNjaTsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT02ODsgIGNpdGF0aW9uX3BhZ2VzPTY2MS03NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RE5BIGVuZC1qb2luaW5nIGNhdGFseXplZCBieSBodW1hbiBjZWxsLWZyZWUgZXh0cmFjdHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVS5TLkE7ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9OTU7ICBjaXRhdGlvbl9wYWdlcz0xNDA2Ni03MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW91c2UgdGVzdGljdWxhciBleHRyYWN0cyBwcm9jZXNzIEROQSBkb3VibGUtc3RyYW5kIGJyZWFrcyBlZmZpY2llbnRseSBieSBETkEgZW5kLXRvLWVuZCBqb2luaW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1NdXRhdC4gUmVzOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTQzMzsgIGNpdGF0aW9uX3BhZ2VzPTEtMTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5vbmhvbW9sb2dvdXMgZW5kIGpvaW5pbmcgb2YgY29tcGxlbWVudGFyeSBhbmQgbm9uY29tcGxlbWVudGFyeSBETkEgdGVybWluaSBpbiBtb3VzZSB0ZXN0aWN1bGFyIGV4dHJhY3RzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1ETkEgUmVwYWlyOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz0xMjk3LTMxMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW4gaW5oaWJpdG9yIG9mIG5vbmhvbW9sb2dvdXMgZW5kLWpvaW5pbmcgYWJyb2dhdGVzIGRvdWJsZS1zdHJhbmQgYnJlYWsgcmVwYWlyIGFuZCBpbXBlZGVzIGNhbmNlciBwcm9ncmVzc2lvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q2VsbDsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xNTE7ICBjaXRhdGlvbl9wYWdlcz0xNDc0LTg3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHBsYXNtaWQgc3lzdGVtIHRvIG1vbml0b3IgZ2VuZSBjb252ZXJzaW9uIGFuZCByZWNpcHJvY2FsIHJlY29tYmluYXRpb24gaW4gdml0cm87ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU11dGF0aW9uIFJlczsgIGNpdGF0aW9uX3llYXI9MTk5MzsgIGNpdGF0aW9uX3ZvbHVtZT0yOTE7ICBjaXRhdGlvbl9wYWdlcz0xODEtOTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhvbW9sb2dvdXMgcmVjb21iaW5hdGlvbi1tZWRpYXRlZCBkb3VibGUtc3RyYW5kIGJyZWFrIHJlcGFpciBpbiBtb3VzZSB0ZXN0aWN1bGFyIGV4dHJhY3RzIGFuZCBjb21wYXJpc29uIHdpdGggZGlmZmVyZW50IGdlcm0gY2VsbCBzdGFnZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNlbGwgQmlvY2hlbS4gRnVuY3Q7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz03NS04NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RG91YmxlLXN0cmFuZCBicmVhayByZXBhaXIgYXNzYXlzIGRldGVybWluZSBwYXRod2F5IGNob2ljZSBhbmQgc3RydWN0dXJlIG9mIGdlbmUgY29udmVyc2lvbiBldmVudHMgaW4gRHJvc29waGlsYSBtZWxhbm9nYXN0ZXI7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUczIChCZXRoZXNkYSk7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTQyNS0zMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSByZWNBLWxpa2UgZ2VuZSBpbiBEcm9zb3BoaWxhIG1lbGFub2dhc3RlciB0aGF0IGlzIGV4cHJlc3NlZCBhdCBoaWdoIGxldmVscyBpbiBmZW1hbGUgYnV0IG5vdCBtYWxlIG1laW90aWMgdGlzc3VlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q2hyb21vc29tYTsgIGNpdGF0aW9uX3llYXI9MTk5NjsgIGNpdGF0aW9uX3ZvbHVtZT0xMDQ7ICBjaXRhdGlvbl9wYWdlcz00NzktNDg4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1va3JhIGFuZCBzcGluZGxlLUIgZW5jb2RlIGNvbXBvbmVudHMgb2YgdGhlIFJBRDUyIEROQSByZXBhaXIgcGF0aHdheSBhbmQgYWZmZWN0IG1laW9zaXMgYW5kIHBhdHRlcm5pbmcgaW4gRHJvc29waGlsYSBvb2dlbmVzaXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbmVzIERldjsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTI3MTEtMjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUR1YWwgcm9sZXMgZm9yIEROQSBwb2x5bWVyYXNlIHRoZXRhIGluIGFsdGVybmF0aXZlIGVuZC1qb2luaW5nIHJlcGFpciBvZiBkb3VibGUtc3RyYW5kIGJyZWFrcyBpbiBEcm9zb3BoaWxhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz1lMTAwMTAwNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN5bnRoZXNpcy1kZXBlbmRlbnQgbWljcm9ob21vbG9neS1tZWRpYXRlZCBlbmQgam9pbmluZyBhY2NvdW50cyBmb3IgbXVsdGlwbGUgdHlwZXMgb2YgcmVwYWlyIGp1bmN0aW9uczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9Mzg7ICBjaXRhdGlvbl9wYWdlcz01NzA2LTE3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Sb2xlIG9mIHJlY29tYmluYXRpb24gYWN0aXZhdGluZyBnZW5lcyBpbiB0aGUgZ2VuZXJhdGlvbiBvZiBhbnRpZ2VuIHJlY2VwdG9yIGRpdmVyc2l0eSBhbmQgYmV5b25kOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1JbW11bm9sb2d5OyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTEzNzsgIGNpdGF0aW9uX3BhZ2VzPTI3MS04MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2hyb21vc29tYWwgdHJhbnNsb2NhdGlvbnMgaW4gY2FuY2VyOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9jaGltLiBCaW9waHlzLiBBY3RhOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTE3ODY7ICBjaXRhdGlvbl9wYWdlcz0xMzktNTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNlbGx1bGFyIEROQSByZXBhaXIgY29mYWN0b3JzIGFmZmVjdGluZyBoZXBhdGl0aXMgQiB2aXJ1cyBpbmZlY3Rpb24gYW5kIHJlcGxpY2F0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Xb3JsZCBKLiBHYXN0cm9lbnRlcm9sOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9NTA1OS02NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TXVsdGlwbGUgcmVndWxhdG9ycyBvZiBUeTEgdHJhbnNwb3NpdGlvbiBpbiBTYWNjaGFyb215Y2VzIGNlcmV2aXNpYWUgaGF2ZSBjb25zZXJ2ZWQgcm9sZXMgaW4gZ2Vub21lIG1haW50ZW5hbmNlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0xNTk7ICBjaXRhdGlvbl9wYWdlcz0xNDQ5LTY1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZXRyb3ZpcnVzZXMgYW5kIHllYXN0IHJldHJvdHJhbnNwb3NvbnMgdXNlIG92ZXJsYXBwaW5nIHNldHMgb2YgaG9zdCBnZW5lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTY0MS01NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIEROQSByZXBhaXIgZ2VuZXMgWFBCIGFuZCBYUEQgZGVmZW5kIGNlbGxzIGZyb20gcmV0cm92aXJhbCBpbmZlY3Rpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVS5TLkE7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MTAzOyAgY2l0YXRpb25fcGFnZXM9NDYyMi03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Eb3VibGUtc3RyYW5kIGJyZWFrIGVuZCByZXNlY3Rpb24gYW5kIHJlcGFpciBwYXRod2F5IGNob2ljZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QW5udS4gUmV2LiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT00NTsgIGNpdGF0aW9uX3BhZ2VzPTI0Ny03MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RW5kLWpvaW5pbmcgcmVwYWlyIG9mIGRvdWJsZS1zdHJhbmQgYnJlYWtzIGluIERyb3NvcGhpbGEgbWVsYW5vZ2FzdGVyIGlzIGxhcmdlbHkgRE5BIGxpZ2FzZSBJViBpbmRlcGVuZGVudDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTY4OyAgY2l0YXRpb25fcGFnZXM9MjA2Ny03NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzI0LzIvMTE3LzEzNjg4OTQ0L2RzdzA1OS5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgRE5BIGRvdWJsZS1zdHJhbmQgYnJlYWtzIChEU0JzKSBhcmUgbW9zdGx5IHJlcGFpcmVkIGJ5IG5vbmhvbW9sb2dvdXMgZW5kIGpvaW5pbmcgKE5IRUopIGFuZCBob21vbG9nb3VzIHJlY29tYmluYXRpb24gKEhSKSBpbiBoaWdoZXIgZXVrYXJ5b3Rlcy4gSW4gY28iIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVzLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTI5NDk1MjEmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2Rkfm9ijwsk/tqNBsqB47LZuJJx1RQ=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f2%2f117%2f2949521" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f2%2f117%2f2949521" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=97104" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/24/2"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/2/2/m_dnares_24_2cover.png?Expires=2147483647&amp;Signature=C8sLyIyfepNN-Oy5BR2xLoVJLx2W1jQBt~48a2lQ~mjys3EWTAKP5k2kV~t7b8sG~Qk1B4iVdz8BINFP5bR-Hc0gvwqekGHM8qavh8kDaLAfN~qNxrDj7NNeLHJ-IURMi5v1ATu23u5wB01wP27BHknbA3W9JgGnq6AVGPqeLfhK7AJcTzOYzKpGYwaedh5crkYXYHp~9GCcHuTjoInf3sKaYC2Ew-HnuA-yIL45j8ocXrNQuIxJV8rFXwx1Tt0b7I1ZJg09R4-7m-GZB9vPgdehgAXUMUNQ4gDatBY22xUt9COqnlb7RZKNUXFDo7Qe0LNJgSHwNUOJvsh-SLpXkg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/24/2">
                        <div class="volume trailing-comma">Volume 24</div>
                        <div class="issue">Issue 2</div>

            </a>

        </div>
        <div class="ii-pub-date">
April 2017        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="78055421">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#78055421">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="78055423">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#78055423">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="78055429">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#78055429">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="78055452">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#78055452">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="78055471">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#78055471">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="78055485">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#78055485">Authors’ contributions</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="78055487">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#78055487">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="78055491">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#78055491">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="78055493">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#78055493">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/24/2/103/2962881">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/24/2/129/2901809">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">DNA double-strand break repair in <em>Penaeus monodon</em> is predominantly dependent on homologous recombination
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shikha Srivastava</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Department of Biochemistry, Indian Institute of Science, Bangalore 560 012, India</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shikha+Srivastava">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Srivastava S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Srivastava S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Shikha Srivastava</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Sumedha Dahal</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Department of Biochemistry, Indian Institute of Science, Bangalore 560 012, India</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Sumedha+Dahal">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Dahal S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Dahal S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Sumedha Dahal</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Sharanya J. Naidu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Department of Biochemistry, Indian Institute of Science, Bangalore 560 012, India</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Sharanya+J.+Naidu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Naidu S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Naidu S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Sharanya J. Naidu</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Deepika Anand</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div>ICAR-Central Institute of Fisheries Education, Mumbai 400 061, India</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Deepika+Anand">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Anand D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Anand D&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Deepika Anand</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Vidya Gopalakrishnan</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Department of Biochemistry, Indian Institute of Science, Bangalore 560 012, India</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Vidya+Gopalakrishnan">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Gopalakrishnan V">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Gopalakrishnan V&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Vidya Gopalakrishnan</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Rajendran Kooloth Valappil</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div>ICAR-Central Institute of Fisheries Education, Mumbai 400 061, India</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsw059-cor1" data-open="dsw059-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. +91 22 2631 0657. Fax: +91 22 2631 0573. E-mail: <a href="mailto:rajendrankv@hotmail.com" target="_blank">rajendrankv@hotmail.com</a> (R.K.V.); Tel. +91 80 22932674. Fax. +91 80 23600814. Email: <a href="mailto:sathees@biochem.iisc.ernet.in" target="_blank">sathees@biochem.iisc.ernet.in</a> (S.C.R.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Rajendran+Kooloth+Valappil">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kooloth Valappil R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kooloth Valappil R&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Rajendran Kooloth Valappil</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Sathees C. Raghavan</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Department of Biochemistry, Indian Institute of Science, Bangalore 560 012, India</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsw059-cor1" data-open="dsw059-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. +91 22 2631 0657. Fax: +91 22 2631 0573. E-mail: <a href="mailto:rajendrankv@hotmail.com" target="_blank">rajendrankv@hotmail.com</a> (R.K.V.); Tel. +91 80 22932674. Fax. +91 80 23600814. Email: <a href="mailto:sathees@biochem.iisc.ernet.in" target="_blank">sathees@biochem.iisc.ernet.in</a> (S.C.R.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Sathees+C.+Raghavan">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Raghavan S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Raghavan S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Sathees C. Raghavan</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsw059-FM1"><div class="fn"><div class="footnote-content">Edited by Dr. Osamu Ohara</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 24, Issue 2, 1 April 2017, Pages 117–128, <a href='https://doi.org/10.1093/dnares/dsw059'>https://doi.org/10.1093/dnares/dsw059</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">24 January 2017</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">08 August 2016</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">08 December 2016</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="2949521" href="/dnaresearch/article-pdf/24/2/117/13688944/dsw059.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Shikha Srivastava, Sumedha Dahal, Sharanya J. Naidu, Deepika Anand, Vidya Gopalakrishnan, Rajendran Kooloth Valappil, Sathees C. Raghavan;  DNA double-strand break repair in <em>Penaeus monodon</em> is predominantly dependent on homologous recombination, <em>DNA Research</em>, Volume 24, Issue 2, 1 April 2017, Pages 117–128, <a href='https://doi.org/10.1093/dnares/dsw059'>https://doi.org/10.1093/dnares/dsw059</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/2949521?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/2949521?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/2949521?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/2949521?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/2949521?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/2949521?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/2949521?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=DNA%20double-strand%20break%20repair%20in%20%26lt%3Bem%26gt%3BPenaeus%20monodon%26lt%3B%2Fem%26gt%3B%20is%20predominantly%20dependent%20on%20homologous%20recombination&amp;publicationDate=2017-01-24&amp;volumeNum=24&amp;issueNum=2&amp;author=Srivastava%2C%20Shikha%3B%20Dahal%2C%20Sumedha&amp;startPage=117&amp;endPage=128&amp;contentId=10.1093%2Fdnares%2Fdsw059&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f2%2f117%2f2949521" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=78055421 id="78055421" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>DNA double-strand breaks (DSBs) are mostly repaired by nonhomologous end joining (NHEJ) and homologous recombination (HR) in higher eukaryotes. In contrast, HR-mediated DSB repair is the major double-strand break repair pathway in lower order organisms such as bacteria and yeast. <em>Penaeus monodon</em>, commonly known as black tiger shrimp, is one of the economically important crustaceans facing large-scale mortality due to exposure to infectious diseases. The animals can also get exposed to chemical mutagens under the culture conditions as well as in wild. Although DSB repair mechanisms have been described in mammals and some invertebrates, its mechanism is unknown in the shrimp species. In the present study, we show that HR-mediated DSB repair is the predominant mode of repair in <em>P. monodon</em>. Robust repair was observed at a temperature of 30 °C, when 2 µg of cell-free extract derived from hepatopancreas was used for the study. Although HR occurred through both reciprocal recombination and gene conversion, the latter was predominant when the bacterial colonies containing recombinants were evaluated. Unlike mammals, NHEJ-mediated DSB repair was undetectable in <em>P. monodon</em>. However, we could detect evidence for an alternative mode of NHEJ that uses microhomology, termed as microhomology-mediated end joining (MMEJ). Interestingly, unlike HR, MMEJ was predominant at lower temperatures. Therefore, the results suggest that, while HR is major DSB repair pathway in shrimp, MMEJ also plays a role in ensuring the continuity and stability of the genome.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Nonhomologous DNA end joining&quot;">Nonhomologous DNA end joining</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="MMEJ">MMEJ</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="Microhomology">Microhomology</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Double-strand break&quot;">Double-strand break</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;homologous recombination&quot;">homologous recombination</a></div>                            <h2 scrollto-destination=78055423 id="78055423" class="section-title" data-legacy-id=SEC1>1. Introduction</h2>
<p>DNA in every living organism is constantly exposed to a plethora of mutagens including radiation, chemicals or oxidative stress leading to induction of a variety of DNA damages. Among different DNA damages, single- and double-strand breaks are considered as most hazardous, if not repaired or misrepaired.<a reveal-id="dsw059-B1 dsw059-B2 dsw059-B3 dsw059-B4 dsw059-B5" data-open="dsw059-B1 dsw059-B2 dsw059-B3 dsw059-B4 dsw059-B5" class="link link-ref link-reveal xref-bibr"><sup>1–5</sup></a> Depending upon the multitude of genotoxic lesions, a variety of DNA repair pathways have been evolved, namely excision repair [nucleotide excision repair (NER), base excision repair (BER) and mismatch repair (MMR)], which deals with DNA damage at the level of base and nucleotides, and DNA strand break repair, which helps in the repair of both single- and double-strand breaks.<a reveal-id="dsw059-B1 dsw059-B2 dsw059-B3" data-open="dsw059-B1 dsw059-B2 dsw059-B3" class="link link-ref link-reveal xref-bibr"><sup>1–3</sup></a><sup>,</sup><a reveal-id="dsw059-B5 dsw059-B6 dsw059-B7" data-open="dsw059-B5 dsw059-B6 dsw059-B7" class="link link-ref link-reveal xref-bibr"><sup>5–7</sup></a></p><p>DNA double-strand breaks (DSBs) occur when both strands of DNA duplex are broken in close vicinity. Improper repair of DSBs could result in genome rearrangements, loss of genetic information etc., while no repair may lead to cell death.<a reveal-id="dsw059-B8 dsw059-B9 dsw059-B10 dsw059-B11 dsw059-B12" data-open="dsw059-B8 dsw059-B9 dsw059-B10 dsw059-B11 dsw059-B12" class="link link-ref link-reveal xref-bibr"><sup>8–12</sup></a> In higher eukaryotes, nonhomologous end joining (NHEJ) is one of the major double-strand break repair pathways.<a reveal-id="dsw059-B5" data-open="dsw059-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a><sup>,</sup><a reveal-id="dsw059-B10" data-open="dsw059-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a><sup>,</sup><a reveal-id="dsw059-B12" data-open="dsw059-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a><sup>,</sup><a reveal-id="dsw059-B13" data-open="dsw059-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> Homologous recombination (HR) pathway is helpful in error-free DSB repair and is active at late S and G2 phases of cell cycle.<a reveal-id="dsw059-B5" data-open="dsw059-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a><sup>,</sup><a reveal-id="dsw059-B10" data-open="dsw059-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a><sup>,</sup><a reveal-id="dsw059-B12 dsw059-B13 dsw059-B14" data-open="dsw059-B12 dsw059-B13 dsw059-B14" class="link link-ref link-reveal xref-bibr"><sup>12–14</sup></a> Unlike HR, NHEJ helps in the maintenance of genome stability throughout all phases of cell cycle.<a reveal-id="dsw059-B15 dsw059-B16 dsw059-B17" data-open="dsw059-B15 dsw059-B16 dsw059-B17" class="link link-ref link-reveal xref-bibr"><sup>15–17</sup></a> A backup pathway to NHEJ, often referred as microhomology-mediated end joining (MMEJ) also exists in mammals. Till recently, it was believed that alternative NHEJ operates only in the absence of classical NHEJ;<a reveal-id="dsw059-B7" data-open="dsw059-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a><sup>,</sup><a reveal-id="dsw059-B12" data-open="dsw059-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a><sup>,</sup><a reveal-id="dsw059-B18 dsw059-B19 dsw059-B20 dsw059-B21 dsw059-B22" data-open="dsw059-B18 dsw059-B19 dsw059-B20 dsw059-B21 dsw059-B22" class="link link-ref link-reveal xref-bibr"><sup>18–22</sup></a> however, recent studies suggest that MMEJ can operate in parallel to classical NHEJ, albeit with low efficiency.<a reveal-id="dsw059-B22" data-open="dsw059-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a></p><p>Penaeid species comprise the largest group of farmed crustaceans, with global production of approximately 3.4 million ton and Asia accounts for 90% of global shrimp aquaculture. Shrimp is the most important seafood product traded internationally; however, shrimp cultivation has been severely affected by a number of pathogens, mainly viruses, throughout the world. Based on the estimates, it was predicted that up to 40% of tropical shrimp production worth &gt;$3 billion is lost annually due to viral pathogens alone,<a reveal-id="dsw059-B23" data-open="dsw059-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> and ∼80% production losses are due to disease occurrence in Asia.<a reveal-id="dsw059-B24" data-open="dsw059-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> Besides, shrimp can also get exposed to various chemicals with potential mutagenic effect, which are either used intentionally for health management purposes or through contaminated water sources. Being invertebrates, it is believed that shrimp do not possess acquired immunity, and are dependent on innate immune response. Although attempts are on to identify defense related host genes, information on DNA repair pathways and DNA repair genes is largely unknown.</p><p>MjRad23, a Rad23 homolog of NER pathway protein was identified and characterized in <em>Marsupenaeus japonicus</em>, a crustacean.<a reveal-id="dsw059-B25" data-open="dsw059-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> UVB radiation-induced damage and repair was also investigated in another crustacean, <em>Tigriopus japonicus</em>. An increased expression of proteins associated with classical NHEJ, HR, BER and MMR was observed.<a reveal-id="dsw059-B26" data-open="dsw059-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> In <em>Daphnia</em>, the DNA damage caused by UVB radiation is repaired by photoenzymatic mechanism, while NER was observed only at lower temperature.<a reveal-id="dsw059-B27" data-open="dsw059-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> Complete genome sequence of <em>Drosophila</em> revealed presence of various critical DNA repair proteins depending on their sequence similarity to other species.<a reveal-id="dsw059-B28" data-open="dsw059-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a>. While important NHEJ proteins, Ku70 and Ku80 orthologs were present in <em>Drosophila</em>,<a reveal-id="dsw059-B29" data-open="dsw059-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> DNA-dependent protein kinase (DNA-PK) catalytic subunit was absent suggesting an aberrant NHEJ/nonclassical NHEJ pathway in <em>Drosophila</em>.<a reveal-id="dsw059-B28" data-open="dsw059-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a></p><p><em>Penaeus monodon</em>, the black tiger shrimp, is one of the most economically important crustaceans. In the present study, we investigated various DSB repair mechanisms in <em>P. monodon</em>. We report that HR-mediated DSB repair is the major DSB repair process in shrimp, while classical NHEJ is absent. Although HR occurred through both reciprocal recombination and gene conversion, the latter was predominant. To our surprise, we could detect evidence for an alternative mode of NHEJ named as MMEJ, which uses microhomology for joining. Hence, present study reveals that HR is the major DSB repair pathway in shrimps.</p>                            <h2 scrollto-destination=78055429 id="78055429" class="section-title" >2. Materials and methods</h2>
                            <h3 scrollto-destination=78055430 id="78055430" class="section-title" >2.1. Enzymes, chemicals and reagents</h3>
<p>Chemical reagents were obtained from Sigma Chemical Co. (St. Louis, MO, USA), Amresco (USA), SRL (India) and Himedia (India). Radioisotope-labelled nucleotides were purchased from BRIT (Hyderabad, India).</p>                            <h3 scrollto-destination=78055432 id="78055432" class="section-title" >2.2. Shrimp samples</h3>
<div class="block-child-p">Healthy shrimp, <em>P. monodon</em> (25–33 g), were obtained from Shakthi Aqua Farm Pvt. Ltd. Mumbai, Maharashtra (<a reveal-id="dsw059-F1" data-open="dsw059-F1" class="link link-reveal link-table xref-fig">Fig. 1A</a>). Shrimp were kept in 500-I FRP tanks containing sea water (14 ± 2 ppt salinity) with continuous aeration and maintained at 25–30°C. The collected animals were fed on commercially available pellet diet and acclimatized for 4–5 days before the experiment.<a reveal-id="dsw059-B30" data-open="dsw059-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> Hepatopancreas was collected from a group of three to four animals and pooled together for the experiments (<a reveal-id="dsw059-F1" data-open="dsw059-F1" class="link link-reveal link-table xref-fig">Fig. 1B</a>). The animals were screened for white spot syndrome virus (WSSV), the most pathogenic and prevalent pathogen infecting shrimp, using standard PCR method before starting the experiments. <div data-id="dsw059-F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw059/2/m_dsw059f1.png?Expires=2147483647&Signature=Pg2EhEcOo4eDx6Vb9lpov9B3wzAhuNp3mqRyQPGKXrqIvcTtb9mKVrUAp46A~OwI49zul0ZF59oCnCsq0j22AT4fA4WRqVruM-zuQ60fywmsiAcDi6cDDBuewVt1gJH81-Rf8hiyMoRMloP7dWY7r5noKReOU1DUn5mRKSQrX3gNFdRROefWeNxlLK0YjxLsVvufgD0p7j-R50r8x370qf4vss-OVhsBNVjtSeRwMr4k0EKrr5Dvi4Y4Ht8K0XBmGx4fGOv32QrjRqXKCuYviWqNoq-u0PqCKSlH7nBoRv1KByTH~oSQ9LTTnJfoFxSzAa3u5-iwL5YDFYpjT93CmA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Preparation of cell-free extract (CFE) of hepatopancreas of shrimp, P. monodon and evaluation of protein profile. (A) Image showing black tiger shrimp, Penaeus monodon. (B) Dorsal view of dissected out shrimp showing hepatopancreas. (C) Histological section of hepatopancreas showing the tubular architecture and different types of cells of hepatopancreatic tubules. (D) Schematic representation of steps involved in the preparation of cell-free extract from hepatopancreas of P. monodon. (E) CBB staining showing protein profile of cell-free extracts of hepatopancreas. Different concentrations of CFE (5, 10, 15 µg) were resolved on a denaturing SDS-PAGE and stained with CBB stain." path-from-xml="dsw059f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055433/dsw059f1.tif" path-from-xml="dsw059f1.tif" target="_blank">View large</a><a section="78055433" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw059/2/dsw059f1.png?Expires=2147483647&Signature=wlV9CZziLkd7zNShMZ0y73LLzuXXAAEM1nc2wsLH-iugKZ8f8CS7v8AJm6cT2YKTs7UGyY9GRpWUPMZeAUxr~chLwshzGRdatYKS95EvAYoKNXrQ-4hQt~XwJczKAR-20ZqrORy5iQ-gKUq~raCoBG0YcKg3RbRXOQh39k5gnLnRb1JMaUHPrdeXouEEDP6jYj~mCt7d68W6sRyAh2amJ9LHfx2Deh8dzNmMeFK2hSdQeVKay6X~x2cTBJc8-SVxmIIzH6FXxTj3V82SqdtnV34Ktm7uBdMggceJxiKIUBx7sPL422zCZKlA1th9ZlpLtxuJd9~v7ZuPZtqwd9tF4A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=78055433&ar=2949521&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw059f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Preparation of cell-free extract (CFE) of hepatopancreas of shrimp, <em>P. monodon</em> and evaluation of protein profile. (A) Image showing black tiger shrimp, <em>Penaeus monodon</em>. (B) Dorsal view of dissected out shrimp showing hepatopancreas. <strong>(C)</strong> Histological section of hepatopancreas showing the tubular architecture and different types of cells of hepatopancreatic tubules. (D) Schematic representation of steps involved in the preparation of cell-free extract from hepatopancreas of <em>P. monodon</em>. (E) CBB staining showing protein profile of cell-free extracts of hepatopancreas. Different concentrations of CFE (5, 10, 15 µg) were resolved on a denaturing SDS-PAGE and stained with CBB stain.</p></div></div><div content-id="dsw059-F1" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw059/2/m_dsw059f1.png?Expires=2147483647&Signature=Pg2EhEcOo4eDx6Vb9lpov9B3wzAhuNp3mqRyQPGKXrqIvcTtb9mKVrUAp46A~OwI49zul0ZF59oCnCsq0j22AT4fA4WRqVruM-zuQ60fywmsiAcDi6cDDBuewVt1gJH81-Rf8hiyMoRMloP7dWY7r5noKReOU1DUn5mRKSQrX3gNFdRROefWeNxlLK0YjxLsVvufgD0p7j-R50r8x370qf4vss-OVhsBNVjtSeRwMr4k0EKrr5Dvi4Y4Ht8K0XBmGx4fGOv32QrjRqXKCuYviWqNoq-u0PqCKSlH7nBoRv1KByTH~oSQ9LTTnJfoFxSzAa3u5-iwL5YDFYpjT93CmA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Preparation of cell-free extract (CFE) of hepatopancreas of shrimp, P. monodon and evaluation of protein profile. (A) Image showing black tiger shrimp, Penaeus monodon. (B) Dorsal view of dissected out shrimp showing hepatopancreas. (C) Histological section of hepatopancreas showing the tubular architecture and different types of cells of hepatopancreatic tubules. (D) Schematic representation of steps involved in the preparation of cell-free extract from hepatopancreas of P. monodon. (E) CBB staining showing protein profile of cell-free extracts of hepatopancreas. Different concentrations of CFE (5, 10, 15 µg) were resolved on a denaturing SDS-PAGE and stained with CBB stain." path-from-xml="dsw059f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055433/dsw059f1.tif" path-from-xml="dsw059f1.tif" target="_blank">View large</a><a section="78055433" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw059/2/dsw059f1.png?Expires=2147483647&Signature=wlV9CZziLkd7zNShMZ0y73LLzuXXAAEM1nc2wsLH-iugKZ8f8CS7v8AJm6cT2YKTs7UGyY9GRpWUPMZeAUxr~chLwshzGRdatYKS95EvAYoKNXrQ-4hQt~XwJczKAR-20ZqrORy5iQ-gKUq~raCoBG0YcKg3RbRXOQh39k5gnLnRb1JMaUHPrdeXouEEDP6jYj~mCt7d68W6sRyAh2amJ9LHfx2Deh8dzNmMeFK2hSdQeVKay6X~x2cTBJc8-SVxmIIzH6FXxTj3V82SqdtnV34Ktm7uBdMggceJxiKIUBx7sPL422zCZKlA1th9ZlpLtxuJd9~v7ZuPZtqwd9tF4A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=78055433&ar=2949521&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw059f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Preparation of cell-free extract (CFE) of hepatopancreas of shrimp, <em>P. monodon</em> and evaluation of protein profile. (A) Image showing black tiger shrimp, <em>Penaeus monodon</em>. (B) Dorsal view of dissected out shrimp showing hepatopancreas. <strong>(C)</strong> Histological section of hepatopancreas showing the tubular architecture and different types of cells of hepatopancreatic tubules. (D) Schematic representation of steps involved in the preparation of cell-free extract from hepatopancreas of <em>P. monodon</em>. (E) CBB staining showing protein profile of cell-free extracts of hepatopancreas. Different concentrations of CFE (5, 10, 15 µg) were resolved on a denaturing SDS-PAGE and stained with CBB stain.</p></div></div></div>                            <h3 scrollto-destination=78055434 id="78055434" class="section-title" >2.3. Oligomers</h3>
<p>The oligomers used in the study were purified using 8–15% denaturing PAGE as described before.<a reveal-id="dsw059-B31" data-open="dsw059-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> 5′-end labelling of the oligomer was performed as described previously.<a reveal-id="dsw059-B32" data-open="dsw059-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> T4 polynucleotide kinase was used and incubated in appropriate buffer containing [γ-<sup>32</sup>P] ATP at 37 °C for 1 h. The labelled substrates were purified using G-25 Sephadex columns and stored at −20 °C until use.</p>                            <h3 scrollto-destination=78055436 id="78055436" class="section-title" >2.4. Preparation of DNA substrates for NHEJ and MMEJ</h3>
<p> For NHEJ study, oligomeric DNA substrate (SCR19), of length 75 nt was radiolabelled with [γ-<sup>32</sup>P] ATP and annealed with unlabelled complementary strand, SCR20 in the presence of 100 mM NaCl and 1 mM EDTA as described earlier to prepare DSBs bearing 5′ compatible overhangs.<a reveal-id="dsw059-B33" data-open="dsw059-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a><sup>,</sup><a reveal-id="dsw059-B34" data-open="dsw059-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> The substrates containing 5′–5′ and 5′–3′ noncompatible overhangs were prepared by annealing radiolabelled SCR19 with cold oligomer, VK11 and VK13, respectively.<a reveal-id="dsw059-B35" data-open="dsw059-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> Blunt ended substrate was prepared by annealing radiolabelled VK7 and cold oligomer, VK8.<a reveal-id="dsw059-B35" data-open="dsw059-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a></p><p>For MMEJ studies, DNA substrates containing 10 nt microhomology regions were synthesized and prepared as described earlier.<a reveal-id="dsw059-B21" data-open="dsw059-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a><sup>,</sup><a reveal-id="dsw059-B22" data-open="dsw059-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> The complementary oligomers were annealed to generate double stranded DNA containing microhomology region. SS54 was annealed to SS62, and SS65 was annealed to SS66 to generate 10 nt microhomology region. MMEJ junctions were PCR amplified using radiolabelled SS60 and unlabelled SS61 primer.</p>                            <h3 scrollto-destination=78055439 id="78055439" class="section-title" >2.5. Preparation of shrimp hepatopancreatic cell-free extract (CFE)</h3>
<p> Cell-free extract was prepared from shrimp hepatopancreas in ice-cold conditions as described previously with modification.<a reveal-id="dsw059-B21" data-open="dsw059-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a><sup>,</sup><a reveal-id="dsw059-B22" data-open="dsw059-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a><sup>,</sup><a reveal-id="dsw059-B36" data-open="dsw059-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> Briefly, tissues were minced with sterile scalpel and washed in ice-cold phosphate buffer saline. The cells were then resuspended in hypotonic lysis buffer [10 mM Tris–HCl (pH 8.0), 1 mM EDTA, 5 mM DTT] and were homogenized in 100 µl of hypotonic lysis buffer in the presence of protease inhibitors (phenylmethylsulfonyl fluoride, 0.01 M; aprotinin, 2 μg/ml; pepstatin, 1 μg/ml; leupeptin, 1 μg/ml) and incubated on ice for 20 min. The extract was centrifuged at 42,000 rpm for 3 h at 4°C in Beckman TLA-100 rotor after addition of 50 µl of high salt buffer [50 mM Tris–HCl (pH 7.5), 1 M KCl, 2 mM EDTA, 2 mM DTT]. The supernatant was dialysed overnight against dialysis buffer [20 mM Tris–HCl (pH 8.0), 0.1 M KOAc, 20% glycerol, 0.5 mM EDTA and 1 mM DTT]. Protein concentration was determined by Bradford assay, aliquoted and stored at −80°C till use.</p>                            <h3 scrollto-destination=78055441 id="78055441" class="section-title" >2.6. NHEJ assay</h3>
<p>NHEJ assay was performed using plasmid as described previously.<a reveal-id="dsw059-B37 dsw059-B38 dsw059-B39" data-open="dsw059-B37 dsw059-B38 dsw059-B39" class="link link-ref link-reveal xref-bibr"><sup>37–39</sup></a> Supercoiled pUC18 plasmid was digested with <em>Eco</em>RI to generate DSBs with cohesive ends. The plasmid DNA (600 ng) was incubated with increasing concentration of cell-free extract (0.5, 1, 1.5, 2, 4 and 8 µg) and incubated at 30°C for 6 h in 1× NHEJ buffer containing 30 mM HEPES–KOH (pH 7.9), 7.5 mM MgCl<sub>2</sub>, 2 mM ATP, 1 mM DTT, 50 µM dNTPs, 100 mg/ml BSA, 10% PEG and 5% glycerol. Reaction was terminated (10 mM EDTA), products were purified by phenol–chloroform extraction and resolved on 1% agarose gel. NHEJ assay performed using rat testicular extract served as a positive control. The gel was visualized following staining with ethidium bromide and image was captured using gel documentation system (UVITEC, Cambridge, UK).</p><p>The NHEJ assay using oligomeric DNA was performed as described previously with modifications.<a reveal-id="dsw059-B33" data-open="dsw059-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a><sup>,</sup><a reveal-id="dsw059-B35" data-open="dsw059-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a><sup>,</sup><a reveal-id="dsw059-B39" data-open="dsw059-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> Cell-free extract prepared from shrimp hepatopancreas was incubated with radio-labelled DNA substrates along with (or without) cold oligomeric DNA substrates (2 µM) at 30°C for 1 h (or as specified) in 1× NHEJ buffer. The reaction was terminated and the products were purified as described above. The purified products were resolved on 8% denaturing PAGE. The gels were dried, exposed onto the screen, and scanned using phosphorImager (FLA9000, Fuji, Japan). Analyses of data were done using Multigauge software (version 3).</p><p>Radioactive PCR was carried out after end-joining reaction to improve the sensitivity of the assay. Following end joining of different oligomeric DNA substrates bearing DSBs either compatible or non-compatible ends, catalysed by cell-free extract of shrimp hepatopancreas (0.5 µg; 30°C for 1 h), NHEJ junctions were amplified using radiolabelled oligomer, VK24 (5′-CCGGTACTACTCGAGCC-3′) and unlabelled SS37 (5′-TGCAGCATCAAGCTAGCC-3'). The reaction products were resolved on 8% denaturing PAGE and radioactive signals were detected as described above.</p>                            <h3 scrollto-destination=78055445 id="78055445" class="section-title" >2.7. MMEJ assay</h3>
<p>MMEJ assay was performed as described earlier.<a reveal-id="dsw059-B21" data-open="dsw059-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a><sup>,</sup><a reveal-id="dsw059-B22" data-open="dsw059-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a><sup>,</sup><a reveal-id="dsw059-B34" data-open="dsw059-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> The reaction was carried out in a volume of 20 µl containing MMEJ buffer [50 mM Tris–HCl (pH 7.6), 20 mM MgCl<sub>2</sub>, 1 mM DTT and 1 mM ATP]. The DNA substrates bearing 10 nt microhomology were incubated along with hepatopancreatic CFE at 30°C for 2 h (or as specified). Reaction was terminated by heat denaturing at 65°C (for 20 min) and the end-joined products were detected by radioactive PCR using radiolabelled oligomer, SS60 and unlabelled SS61. The reaction products were resolved on 8% denaturing PAGE and radioactive signals were detected using phosphorImager, as described above.</p>                            <h3 scrollto-destination=78055447 id="78055447" class="section-title" >2.8. HR assay</h3>
<p><em>In vitro</em> HR assay was performed as described previously, using two independent plasmids.<a reveal-id="dsw059-B38" data-open="dsw059-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a><sup>,</sup><a reveal-id="dsw059-B40" data-open="dsw059-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a><sup>,</sup><a reveal-id="dsw059-B41" data-open="dsw059-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a> Two plasmid constructs, pTO223 and pTO231, were generated by introducing mutations to neomycin resistant gene on pTO221 as described.<a reveal-id="dsw059-B40" data-open="dsw059-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> Briefly, pTO223 was created by deleting a 245-bp (<em>NarI</em>) segment from 5′ region of <em>neomycin</em> (<em>neoΔ2</em>) gene and pTO231 was created by deleting 282 bp (<em>NaeI</em>) segment from 3′ of <em>neomycin</em> (<em>neoΔ1</em>). In pTO223, <em>neo</em> gene was flanked by <em>Eco</em>RI and <em>Sal</em>I restriction enzyme sites. <em>Hin</em>dIII site was also present close to <em>Eco</em>RI at 5′ end, which was absent in pTO231. pTO221 was generated using pBSKS where 1.5 kb of <em>Hin</em>dIII/SalI restriction fragment of pBR322::Tn5 containing the <em>neo</em> gene was inserted at the same restriction sites. pTO223 and pTO231 carried functional ampicillin resistant gene and served as substrates for the assay.</p><p>The HR assay was performed using 500 ng of each plasmids (pTO223 and pTO231) by incubating with increasing concentrations of shrimp hepatopancreatic extracts (0, 0.5, 1, 1.5, 2 and 4 µg) in a buffer containing 35 mM HEPES (pH 8.0), 10 mM MgCl<sub>2</sub>, 1 mM DTT, 2 mM ATP, 50 µM dNTPs, 1 mM NAD and 100 µg/ml BSA at 30°C (or as specified) for 30 min. Reaction was terminated by adding EDTA (20 mM) and reaction products were purified using phenol–chloroform extraction. Purified DNA was used for the transformation of <em>E.</em><em>coli</em> DH5α and plated over agar plates containing ampicillin and kanamycin antibiotics, and scored for transformants and recombinants, respectively. For all the experiments, no protein control served as negative control, which accounts for background recombination catalysed by <em>E.</em><em>coli</em>. The recombination frequency was calculated as ratio of number of recombinants to the number of transformants per microgram of DNA.</p>                            <h3 scrollto-destination=78055450 id="78055450" class="section-title" >2.9. Restriction enzyme digestion analysis to distinguish reciprocal exchange and gene conversion</h3>
<p> Kanamycin resistant colonies containing recombinants were confirmed for functional <em>neo</em> gene by restriction endonuclease digestion. Functional <em>neo</em> gene can be formed by reciprocal exchange or gene conversion. In reciprocal exchange, a crossing over within the region between deletions <em>neoΔ1</em> and <em>neoΔ2</em> takes place giving rise to functional <em>neo<sup>+</sup> </em>allele and a non-functional double mutant allele. In gene conversion a transfer of DNA from <em>neoΔ1</em> to <em>neoΔ2</em> takes place giving rise to functional <em>neo</em> gene. Restriction digestion of pTO223 and pTO231 will give rise to a 1.25- and 1.2-kb fragments, respectively, when digested with <em>Eco</em>RI<em>–</em><em>Sal</em>I and is indicative of <em>neoΔ2</em> and <em>neoΔ1</em> allele and a 2.9-kb fragment corresponding to vector DNA. In contrast, digestion of plasmids containing <em>neo</em> gene will release a 1.5-kb fragment that is indicative of the presence of functional <em>neo</em> gene.</p>                            <h2 scrollto-destination=78055452 id="78055452" class="section-title" >3. Results</h2>
                            <h3 scrollto-destination=78055453 id="78055453" class="section-title" >3.1. Cell-free system to study DNA repair pathways</h3>
<p>Cell-free extract was prepared from hepatopancreas of healthy tiger shrimp, <em>P. monodon</em> for studying DSB repair pathways (<a reveal-id="dsw059-F1" data-open="dsw059-F1" class="link link-reveal link-table xref-fig">Fig. 1A and B</a>). The hepatopancreas was chosen as the target tissue in the experiment, as this is the primary organ responsible for absorption and storage of ingested material and involved in the synthesis of digestive enzymes and detoxification of xenobiotics. The organ constitutes 2–6% of the body weight and consists of hepatopancreatic tubules (<a reveal-id="dsw059-F1" data-open="dsw059-F1" class="link link-reveal link-table xref-fig">Fig. 1C</a>). Cell-free extract was prepared based on previously published work,<a reveal-id="dsw059-B21" data-open="dsw059-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a><sup>,</sup><a reveal-id="dsw059-B22" data-open="dsw059-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a><sup>,</sup><a reveal-id="dsw059-B36" data-open="dsw059-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> protein profile was examined on a CBB gel, and used for DSB repair studies (<a reveal-id="dsw059-F1" data-open="dsw059-F1" class="link link-reveal link-table xref-fig">Fig. 1D and E</a>).</p>                            <h3 scrollto-destination=78055455 id="78055455" class="section-title" >3.2. Unlike mammals, classical NHEJ is absent in <em>P. monodon</em></h3>
<div class="block-child-p">To test for the presence of classical NHEJ in the extracts of hepatopancreas of <em>P. monodon</em>, an oligomeric DNA based assay system was employed in addition to plasmid-joining assay (<a reveal-id="dsw059-F2" data-open="dsw059-F2" class="link link-reveal link-table xref-fig">Fig. 2</a> and data not shown). In order to optimize the oligomer-based assay system, different parameters were tested using oligomeric DNA substrate bearing 5′–5′ compatible ends (<a reveal-id="dsw059-F2" data-open="dsw059-F2" class="link link-reveal link-table xref-fig">Fig. 2A</a>). [γ-<sup>32</sup>P] ATP-labelled oligonucleotides were incubated (30°C, 1 h) with increasing concentrations of hepatopancreas cell-free extract and purified products were resolved on denaturing PAGE (<a reveal-id="dsw059-F2" data-open="dsw059-F2" class="link link-reveal link-table xref-fig">Fig. 2B</a>). Results showed no joined products upon treatment with the hepatopancreas cell extracts, unlike the positive controls in which the testicular extract prepared from rat showed end to end joining of 75 mer DNA leading to both circular and linear multimeric products (<a reveal-id="dsw059-F2" data-open="dsw059-F2" class="link link-reveal link-table xref-fig">Fig. 2B</a>).<a reveal-id="dsw059-B21" data-open="dsw059-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a><sup>,</sup><a reveal-id="dsw059-B33" data-open="dsw059-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a><sup>,</sup><a reveal-id="dsw059-B35" data-open="dsw059-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a><sup>,</sup><a reveal-id="dsw059-B39" data-open="dsw059-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> Although hepatopancreatic extracts did not exhibit DNA end joining, it possessed efficient nuclease activity leading to degradation of substrate DNA, in a protein concentration-dependent manner (<a reveal-id="dsw059-F2" data-open="dsw059-F2" class="link link-reveal link-table xref-fig">Fig. 2B</a>). Furthermore, unlabelled random double-stranded DNA was added to the end-joining reaction mix to reduce the nuclease activity. Although this helped in reducing the nuclease activity on labelled DNA significantly, hepatopancreas cell extract did not exhibit joining of DSBs possessing compatible ends even after incubating with increasing concentrations of the extract (<a reveal-id="dsw059-F2" data-open="dsw059-F2" class="link link-reveal link-table xref-fig">Fig. 2C</a>). <div data-id="dsw059-F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw059/2/m_dsw059f2.png?Expires=2147483647&Signature=NaNX2P45Go~cvb9OLSWrAtpidOTxLMvtwhq35cD89yOwyQ8NberVSs5w95IAJbG1F~6k7--DGCjSRJOL-4YOUv4pMH2~i7DlS1q2oVmHonfxwCsaHp3CSXSmEKQDpQZgflkdV7r2ZJdWFohwds~DqvF8pXs45VbToSlEs-yQeueCmXEJsDdogSgySSDPU0PpfXJrnALH5N1vrNPuTKpuXKkY0BQ869UGGfplSFz9SGd8FlrnWZyyubuLWai-6lBA-5fRWgAe4PZfTFprwBtFKt5-bEtbLFNTdLmEHM5CNRWxYdaY1u3cx7OKI-348WPu7DSGOyJe4PvzHQnod4Leyg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Evaluation of compatible DNA end joining catalysed by extracts of hepatopancreas of shrimp. (A) Sequence of oligomeric DNA bearing 5′–5′ compatible ends used for the EJ study. (B, C) Evaluation of DNA end joining efficiency using increasing concentrations of CFE (0.01, 0.05, 0.1, 0.5, 1 and 2 µg) of hepatopancreas using 5′–5′ compatible end substrate (B) and in presence of additional cold oligomers (C). DNA incubated with rat testicular extract (RTE) was used as a positive control. Lane1 is no protein control and ‘M’ is 50 nt DNA ladder. Bar diagram showing quantitation of EJ efficiency along with the error bar (SEM). Intensity of the band was calculated and expressed in PSL units. Arrow indicates joined products; circular form I (monomer), linear dimer and circular form II (dimer) are shown." path-from-xml="dsw059f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055456/dsw059f2.tif" path-from-xml="dsw059f2.tif" target="_blank">View large</a><a section="78055456" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw059/2/dsw059f2.png?Expires=2147483647&Signature=c910zTl7K5ZwdmYzJOQ8cBRj~MJCV8lz9qmUwf7FjpW36iuQLMEfHdrpPonSfrlxyu5Rooa5FXZSmtagZRD1A0WOnZoUa43Ruc3O9eyDIpqjT~K9KoM1jRyV0I7sAhSSO01Q9nDnfydM~s0ivgUNAtYRlXCjaz8iJaC5rVPlokWDEDfIdpbt3h5MwyJnfvyqXh2ugfxn41P739SuejdsmFuAFZ9mu0rhW4SkymTRj7a-SO9M7Ww9whSmgagcRgIAOp0FdIhr139EQGBWr4aZffqtppEE-F935MMkCx1QxVdBS~VmmPOv56I0G5QKOmDD6aanGRVBs8BOW052tqj1Vw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=78055456&ar=2949521&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw059f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Evaluation of compatible DNA end joining catalysed by extracts of hepatopancreas of shrimp. (A) Sequence of oligomeric DNA bearing 5′–5′ compatible ends used for the EJ study. (B, C) Evaluation of DNA end joining efficiency using increasing concentrations of CFE (0.01, 0.05, 0.1, 0.5, 1 and 2 µg) of hepatopancreas using 5′–5′ compatible end substrate (B) and in presence of additional cold oligomers (C). DNA incubated with rat testicular extract (RTE) was used as a positive control. Lane1 is no protein control and ‘M’ is 50 nt DNA ladder. Bar diagram showing quantitation of EJ efficiency along with the error bar (SEM). Intensity of the band was calculated and expressed in PSL units. Arrow indicates joined products; circular form I (monomer), linear dimer and circular form II (dimer) are shown.</p></div></div><div content-id="dsw059-F2" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw059/2/m_dsw059f2.png?Expires=2147483647&Signature=NaNX2P45Go~cvb9OLSWrAtpidOTxLMvtwhq35cD89yOwyQ8NberVSs5w95IAJbG1F~6k7--DGCjSRJOL-4YOUv4pMH2~i7DlS1q2oVmHonfxwCsaHp3CSXSmEKQDpQZgflkdV7r2ZJdWFohwds~DqvF8pXs45VbToSlEs-yQeueCmXEJsDdogSgySSDPU0PpfXJrnALH5N1vrNPuTKpuXKkY0BQ869UGGfplSFz9SGd8FlrnWZyyubuLWai-6lBA-5fRWgAe4PZfTFprwBtFKt5-bEtbLFNTdLmEHM5CNRWxYdaY1u3cx7OKI-348WPu7DSGOyJe4PvzHQnod4Leyg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Evaluation of compatible DNA end joining catalysed by extracts of hepatopancreas of shrimp. (A) Sequence of oligomeric DNA bearing 5′–5′ compatible ends used for the EJ study. (B, C) Evaluation of DNA end joining efficiency using increasing concentrations of CFE (0.01, 0.05, 0.1, 0.5, 1 and 2 µg) of hepatopancreas using 5′–5′ compatible end substrate (B) and in presence of additional cold oligomers (C). DNA incubated with rat testicular extract (RTE) was used as a positive control. Lane1 is no protein control and ‘M’ is 50 nt DNA ladder. Bar diagram showing quantitation of EJ efficiency along with the error bar (SEM). Intensity of the band was calculated and expressed in PSL units. Arrow indicates joined products; circular form I (monomer), linear dimer and circular form II (dimer) are shown." path-from-xml="dsw059f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055456/dsw059f2.tif" path-from-xml="dsw059f2.tif" target="_blank">View large</a><a section="78055456" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw059/2/dsw059f2.png?Expires=2147483647&Signature=c910zTl7K5ZwdmYzJOQ8cBRj~MJCV8lz9qmUwf7FjpW36iuQLMEfHdrpPonSfrlxyu5Rooa5FXZSmtagZRD1A0WOnZoUa43Ruc3O9eyDIpqjT~K9KoM1jRyV0I7sAhSSO01Q9nDnfydM~s0ivgUNAtYRlXCjaz8iJaC5rVPlokWDEDfIdpbt3h5MwyJnfvyqXh2ugfxn41P739SuejdsmFuAFZ9mu0rhW4SkymTRj7a-SO9M7Ww9whSmgagcRgIAOp0FdIhr139EQGBWr4aZffqtppEE-F935MMkCx1QxVdBS~VmmPOv56I0G5QKOmDD6aanGRVBs8BOW052tqj1Vw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=78055456&ar=2949521&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw059f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Evaluation of compatible DNA end joining catalysed by extracts of hepatopancreas of shrimp. (A) Sequence of oligomeric DNA bearing 5′–5′ compatible ends used for the EJ study. (B, C) Evaluation of DNA end joining efficiency using increasing concentrations of CFE (0.01, 0.05, 0.1, 0.5, 1 and 2 µg) of hepatopancreas using 5′–5′ compatible end substrate (B) and in presence of additional cold oligomers (C). DNA incubated with rat testicular extract (RTE) was used as a positive control. Lane1 is no protein control and ‘M’ is 50 nt DNA ladder. Bar diagram showing quantitation of EJ efficiency along with the error bar (SEM). Intensity of the band was calculated and expressed in PSL units. Arrow indicates joined products; circular form I (monomer), linear dimer and circular form II (dimer) are shown.</p></div></div></div><div class="block-child-p">To further evaluate the end-joining efficiency of hepatopancreas extract, oligomeric DNA having different types of DSBs, such as substrates possessing 5′–5′ compatible end, 5′–5′ non-compatible end, 5′–3′ non-compatible end and blunt ends, were incubated with extracts (0.5 µg) (<a reveal-id="dsw059-F3" data-open="dsw059-F3" class="link link-reveal link-table xref-fig">Fig. 3A</a>) in the presence of cold double-stranded DNA at different time points (2, 5, 15, 30 min, 1, 2 and 6 h). Analysis of purified products on denaturing PAGE confirmed no end-to-end joining of any of the DNA substrates tested (<a reveal-id="dsw059-F3" data-open="dsw059-F3" class="link link-reveal link-table xref-fig">Fig. 3B–E</a>), although end joining was observed in the case of rat testicular extracts, as reported previously.<a reveal-id="dsw059-B35" data-open="dsw059-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a><div data-id="dsw059-F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw059/2/m_dsw059f3.png?Expires=2147483647&Signature=jqf3XU66sboFRRfYF7DxwFMrVJHCMwiOR61LcidvWzalqJNhynCA54QX-i90xGfGXgCNgIZbeS9UxlXLvAEPwD2v0jExju8ccsT41DaZJTIDnsrUfGEUsEf77ai-Fpq2Emk1-iu0xnIewP1UYOltdQPyGxrbYWWM1jzV26FPH-iBWeRGa0oHl08tIqp4z9TyaSfELusreg~p62iudhjwGv3CMXbbtkWJLInBZ03TvD6b8Hf6yBEQpvykMaPvvJe8k4FGD5~32RSz~E7b0v0svRlJwmwl0k2bjlEkXfqwzEkT8~cjero0q6eIuQ3ANGVAWQabBNxkb-zpIN~gJGxm3A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Efficacy of end joining catalysed by extracts of hepatopancreas of shrimp, compared with oligomeric DNA bearing DSBs with different end structures. (A) Sequence of different oligomeric DNA substrates used for the NHEJ reaction. (B–E) Time kinetics of NHEJ catalysed by shrimp hepatopancreatic extracts (0.5 µg), when treated with 5′–5′ compatible end, (B) 5′–5′ noncompatible end (C), 5′–3′ noncompatible end (D) and blunt end (E) DNA substrates incubated for 2, 5, 15, 30 min and 1, 2 and 6 h. Products were resolved on denaturing PAGE. RTE treated samples served as positive control for respective type of DNA breaks. Lane 1 is no protein control and ‘M’ is 50 nt DNA ladder. In each panel, bar diagram displaying quantitation of NHEJ products is shown with error bar (SEM; n = 3). Joining products are indicated by arrow as circular form I (monomer), linear dimer and circular form II (dimer)." path-from-xml="dsw059f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055457/dsw059f3.tif" path-from-xml="dsw059f3.tif" target="_blank">View large</a><a section="78055457" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw059/2/dsw059f3.png?Expires=2147483647&Signature=OoHr8dvEezFZ5tJLftf9yq4VsJw9VHdypwprb35kbClA2JnPJp4YpcxUp2XwY900qOi6-b0BFjCZ5GakieAiOaq-GEQi0-0~OsyTPSaFKpRvTpPx~FegB3tsnaA0LS50Aiit-M07gw62VGZpFCruNv1F7iaxLhgza5iO5w7Gp4ctWbBw7oX1Tx~WRutIqzna6ZiRSiOsjkSrE-m7A~0ZESmnUFioKSqDyoDFRVcWAuQOVX6QRdBMKN7nb-VJQ4rMeTH-eEAPE0xl-PEWTB1-NDhmfLEtD4k7v6kuF9fUsxVvAsdxy3WOKSRuL61p1cGxXBjOwHfTRadiy7QREx1Zog__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=78055457&ar=2949521&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw059f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Efficacy of end joining catalysed by extracts of hepatopancreas of shrimp, compared with oligomeric DNA bearing DSBs with different end structures. (A) Sequence of different oligomeric DNA substrates used for the NHEJ reaction. (B–E) Time kinetics of NHEJ catalysed by shrimp hepatopancreatic extracts (0.5 µg), when treated with 5′–5′ compatible end, (B) 5′–5′ noncompatible end (C), 5′–3′ noncompatible end (D) and blunt end (E) DNA substrates incubated for 2, 5, 15, 30 min and 1, 2 and 6 h. Products were resolved on denaturing PAGE. RTE treated samples served as positive control for respective type of DNA breaks. Lane 1 is no protein control and ‘M’ is 50 nt DNA ladder. In each panel, bar diagram displaying quantitation of NHEJ products is shown with error bar (SEM; <em>n</em> = 3). Joining products are indicated by arrow as circular form I (monomer), linear dimer and circular form II (dimer).</p></div></div><div content-id="dsw059-F3" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw059/2/m_dsw059f3.png?Expires=2147483647&Signature=jqf3XU66sboFRRfYF7DxwFMrVJHCMwiOR61LcidvWzalqJNhynCA54QX-i90xGfGXgCNgIZbeS9UxlXLvAEPwD2v0jExju8ccsT41DaZJTIDnsrUfGEUsEf77ai-Fpq2Emk1-iu0xnIewP1UYOltdQPyGxrbYWWM1jzV26FPH-iBWeRGa0oHl08tIqp4z9TyaSfELusreg~p62iudhjwGv3CMXbbtkWJLInBZ03TvD6b8Hf6yBEQpvykMaPvvJe8k4FGD5~32RSz~E7b0v0svRlJwmwl0k2bjlEkXfqwzEkT8~cjero0q6eIuQ3ANGVAWQabBNxkb-zpIN~gJGxm3A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Efficacy of end joining catalysed by extracts of hepatopancreas of shrimp, compared with oligomeric DNA bearing DSBs with different end structures. (A) Sequence of different oligomeric DNA substrates used for the NHEJ reaction. (B–E) Time kinetics of NHEJ catalysed by shrimp hepatopancreatic extracts (0.5 µg), when treated with 5′–5′ compatible end, (B) 5′–5′ noncompatible end (C), 5′–3′ noncompatible end (D) and blunt end (E) DNA substrates incubated for 2, 5, 15, 30 min and 1, 2 and 6 h. Products were resolved on denaturing PAGE. RTE treated samples served as positive control for respective type of DNA breaks. Lane 1 is no protein control and ‘M’ is 50 nt DNA ladder. In each panel, bar diagram displaying quantitation of NHEJ products is shown with error bar (SEM; n = 3). Joining products are indicated by arrow as circular form I (monomer), linear dimer and circular form II (dimer)." path-from-xml="dsw059f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055457/dsw059f3.tif" path-from-xml="dsw059f3.tif" target="_blank">View large</a><a section="78055457" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw059/2/dsw059f3.png?Expires=2147483647&Signature=OoHr8dvEezFZ5tJLftf9yq4VsJw9VHdypwprb35kbClA2JnPJp4YpcxUp2XwY900qOi6-b0BFjCZ5GakieAiOaq-GEQi0-0~OsyTPSaFKpRvTpPx~FegB3tsnaA0LS50Aiit-M07gw62VGZpFCruNv1F7iaxLhgza5iO5w7Gp4ctWbBw7oX1Tx~WRutIqzna6ZiRSiOsjkSrE-m7A~0ZESmnUFioKSqDyoDFRVcWAuQOVX6QRdBMKN7nb-VJQ4rMeTH-eEAPE0xl-PEWTB1-NDhmfLEtD4k7v6kuF9fUsxVvAsdxy3WOKSRuL61p1cGxXBjOwHfTRadiy7QREx1Zog__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=78055457&ar=2949521&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw059f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Efficacy of end joining catalysed by extracts of hepatopancreas of shrimp, compared with oligomeric DNA bearing DSBs with different end structures. (A) Sequence of different oligomeric DNA substrates used for the NHEJ reaction. (B–E) Time kinetics of NHEJ catalysed by shrimp hepatopancreatic extracts (0.5 µg), when treated with 5′–5′ compatible end, (B) 5′–5′ noncompatible end (C), 5′–3′ noncompatible end (D) and blunt end (E) DNA substrates incubated for 2, 5, 15, 30 min and 1, 2 and 6 h. Products were resolved on denaturing PAGE. RTE treated samples served as positive control for respective type of DNA breaks. Lane 1 is no protein control and ‘M’ is 50 nt DNA ladder. In each panel, bar diagram displaying quantitation of NHEJ products is shown with error bar (SEM; <em>n</em> = 3). Joining products are indicated by arrow as circular form I (monomer), linear dimer and circular form II (dimer).</p></div></div></div><div class="block-child-p">Since we failed to detect any end-joined products using <em>in vitro</em> NHEJ analysis, radioactive PCR was employed to improve the sensitivity of the assay. Oligomeric DNA substrates bearing DSBs (5′–5′ compatible, 5′–5′ non-compatible, 5′–3′ non-compatible or blunt ends) were incubated with shrimp hepatopancreas cell-free extract (0.5 µg) and resolved on denaturing PAGE following radioactive PCR (<a reveal-id="dsw059-F4" data-open="dsw059-F4" class="link link-reveal link-table xref-fig">Fig. 4A and B</a>). Results showed efficient joining in case of rat testicular extracts (positive control), while no amplification of joined products was observed in the case of shrimp hepatopancreas extract (<a reveal-id="dsw059-F4" data-open="dsw059-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>). Thus, we conclude that cell-free extract from <em>P. monodon</em> was unable to catalyse DSB joining through NHEJ, irrespective of the type of DSBs used for analysis suggesting the absence of functional classical NHEJ in shrimps. <div data-id="dsw059-F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw059/2/m_dsw059f4.png?Expires=2147483647&Signature=SQBHM9nWHJ5-04zUsJyBOVaDG0wDzPDL4DoO2YXDgAhjy0B-S87pGmTVRRX868WFnpEvzh6JLpUIOp-p0oRAuspGYiPa3~vf-hL9QMZHwLjm4vk5yGfTK8ADlkHoUn6R17k4qFP-PmWJGKNIXF6OROPzasNo2bf64DFJ~aKv8Jdi4pDh25Gm1dMxmCn7DmA8uYpCVsoHnjiB6fO7K7MuBVrRwUAbzMJ7hPemPIojGAq3bfOMI0QH~VUq~AANTeqkanI~u24iilCj8HFLMQZv6~2oegmg51EpAs52AwBrao-Y6SDxtNOHkL3QLTPt3lYrYng2Yp5Ni9qPillpMfRKdg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Evaluation of end joining catalysed by hepatopancreas extract of shrimp following radioactive PCR. (A) Different oligomeric DNA substrates (5′–5′ compatible, 5′–5′ noncompatible, 5′–3′ noncompatible and blunt end) were incubated with shrimp hepatopancreas CFE. The presence of end-joined products was examined through radioactive PCR. Amplified products were resolved on 8% denaturing PAGE. Lane 1, 4, 7, 10 are no protein control. Rat testicular extract treated samples served as positive controls. (B) Bar diagram displaying quantitation of joined products with error bar (SEM; n = 3) indicated. Band intensity was calculated and expressed in PSL units. Joined products are indicated by arrow. Multimers are shown in bracket. ‘M’ is 50 nt DNA ladder." path-from-xml="dsw059f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055458/dsw059f4.tif" path-from-xml="dsw059f4.tif" target="_blank">View large</a><a section="78055458" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw059/2/dsw059f4.png?Expires=2147483647&Signature=ZFTRplM~RmjoaYMyOR6MeHcEQW9tSpeUeYRvcz8pPVFmGlnsTLIl7VlXd3U1vh-jjCXlcjXJzUbcTfFlfSr-cBMW3uBtwmxlBg2uHyUZUBcrzy6HKGHGG9IJA1SqeteLonNw2dgRMa-~zXfyTe5sg1FRgecSan7kYVegc5LaPw8rOlmOVzFrmeWyLGrskosr3koh6idLXUk~rDThA~lonWE0EEDQpCDwy4o-fyfzQ6jO2dTKhGPUG6xKw8AExjAyfJWVu28Vc5nNyqVCTMMh4BCGLC7BfcEKDakOecjiuO2KpAAsCzMUlcTr9U22~3iMTYPba5Ms1bK0ARefIWrylA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=78055458&ar=2949521&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw059f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Evaluation of end joining catalysed by hepatopancreas extract of shrimp following radioactive PCR. (A) Different oligomeric DNA substrates (5′–5′ compatible, 5′–5′ noncompatible, 5′–3′ noncompatible and blunt end) were incubated with shrimp hepatopancreas CFE. The presence of end-joined products was examined through radioactive PCR. Amplified products were resolved on 8% denaturing PAGE. Lane 1, 4, 7, 10 are no protein control. Rat testicular extract treated samples served as positive controls. (B) Bar diagram displaying quantitation of joined products with error bar (SEM; <em>n</em> = 3) indicated. Band intensity was calculated and expressed in PSL units. Joined products are indicated by arrow. Multimers are shown in bracket. ‘M’ is 50 nt DNA ladder.</p></div></div><div content-id="dsw059-F4" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw059/2/m_dsw059f4.png?Expires=2147483647&Signature=SQBHM9nWHJ5-04zUsJyBOVaDG0wDzPDL4DoO2YXDgAhjy0B-S87pGmTVRRX868WFnpEvzh6JLpUIOp-p0oRAuspGYiPa3~vf-hL9QMZHwLjm4vk5yGfTK8ADlkHoUn6R17k4qFP-PmWJGKNIXF6OROPzasNo2bf64DFJ~aKv8Jdi4pDh25Gm1dMxmCn7DmA8uYpCVsoHnjiB6fO7K7MuBVrRwUAbzMJ7hPemPIojGAq3bfOMI0QH~VUq~AANTeqkanI~u24iilCj8HFLMQZv6~2oegmg51EpAs52AwBrao-Y6SDxtNOHkL3QLTPt3lYrYng2Yp5Ni9qPillpMfRKdg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Evaluation of end joining catalysed by hepatopancreas extract of shrimp following radioactive PCR. (A) Different oligomeric DNA substrates (5′–5′ compatible, 5′–5′ noncompatible, 5′–3′ noncompatible and blunt end) were incubated with shrimp hepatopancreas CFE. The presence of end-joined products was examined through radioactive PCR. Amplified products were resolved on 8% denaturing PAGE. Lane 1, 4, 7, 10 are no protein control. Rat testicular extract treated samples served as positive controls. (B) Bar diagram displaying quantitation of joined products with error bar (SEM; n = 3) indicated. Band intensity was calculated and expressed in PSL units. Joined products are indicated by arrow. Multimers are shown in bracket. ‘M’ is 50 nt DNA ladder." path-from-xml="dsw059f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055458/dsw059f4.tif" path-from-xml="dsw059f4.tif" target="_blank">View large</a><a section="78055458" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw059/2/dsw059f4.png?Expires=2147483647&Signature=ZFTRplM~RmjoaYMyOR6MeHcEQW9tSpeUeYRvcz8pPVFmGlnsTLIl7VlXd3U1vh-jjCXlcjXJzUbcTfFlfSr-cBMW3uBtwmxlBg2uHyUZUBcrzy6HKGHGG9IJA1SqeteLonNw2dgRMa-~zXfyTe5sg1FRgecSan7kYVegc5LaPw8rOlmOVzFrmeWyLGrskosr3koh6idLXUk~rDThA~lonWE0EEDQpCDwy4o-fyfzQ6jO2dTKhGPUG6xKw8AExjAyfJWVu28Vc5nNyqVCTMMh4BCGLC7BfcEKDakOecjiuO2KpAAsCzMUlcTr9U22~3iMTYPba5Ms1bK0ARefIWrylA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=78055458&ar=2949521&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw059f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Evaluation of end joining catalysed by hepatopancreas extract of shrimp following radioactive PCR. (A) Different oligomeric DNA substrates (5′–5′ compatible, 5′–5′ noncompatible, 5′–3′ noncompatible and blunt end) were incubated with shrimp hepatopancreas CFE. The presence of end-joined products was examined through radioactive PCR. Amplified products were resolved on 8% denaturing PAGE. Lane 1, 4, 7, 10 are no protein control. Rat testicular extract treated samples served as positive controls. (B) Bar diagram displaying quantitation of joined products with error bar (SEM; <em>n</em> = 3) indicated. Band intensity was calculated and expressed in PSL units. Joined products are indicated by arrow. Multimers are shown in bracket. ‘M’ is 50 nt DNA ladder.</p></div></div></div>                            <h3 scrollto-destination=78055459 id="78055459" class="section-title" >3.3. <em>P</em><em>enaeus</em><em>monodon</em> possesses detectable microhomology mediated DNA end joining (MMEJ)</h3>
<div class="block-child-p">Since classical NHEJ was not detected in <em>P. monodon</em>, we investigated whether DSB repair can occur using microhomology as reported recently in rodents.<a reveal-id="dsw059-B22" data-open="dsw059-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a><sup>,</sup><a reveal-id="dsw059-B34" data-open="dsw059-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> To test this hypothesis, we synthesized DNA with 10 nt microhomology region and experiment was carried out as outlined (<a reveal-id="dsw059-F5" data-open="dsw059-F5" class="link link-reveal link-table xref-fig">Fig. 5A and B</a>).<a reveal-id="dsw059-B22" data-open="dsw059-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> DNA substrates bearing microhomology were incubated with increasing concentrations of shrimp hepatopancreas extracts or with heat inactivated extracts for 2 h. Products were resolved on a denaturing PAGE, following radioactive PCR using labelled primer (<a reveal-id="dsw059-F5" data-open="dsw059-F5" class="link link-reveal link-table xref-fig">Fig. 5C</a>). Results showed a distinct product at 62 bp position (indicated by arrow) suggesting the joining through MMEJ (<a reveal-id="dsw059-F5" data-open="dsw059-F5" class="link link-reveal link-table xref-fig">Fig. 5C</a>). Although the efficiency of the joining catalysed by extracts of hepatopancreas was lesser compared with the positive control, testicular extract from rat; it was dependent on concentration of hepatopancreatic extracts (<a reveal-id="dsw059-F5" data-open="dsw059-F5" class="link link-reveal link-table xref-fig">Fig. 5D</a>). A distinct band due to MMEJ was observed, when as low as 0.1 µg of hepatopancreas extract was used. An increase in efficiency was observed with increasing concentration of CFE, and 0.5 µg of extract was noted as optimal (<a reveal-id="dsw059-F5" data-open="dsw059-F5" class="link link-reveal link-table xref-fig">Fig. 5D</a>). A decrease in joining efficiency was observed at high concentrations of CFE (1 and 2 µg), which could be due to increased levels of nuclease activity. <div data-id="dsw059-F5" class="fig fig-section"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw059/2/m_dsw059f5.png?Expires=2147483647&Signature=Hjay8s2P8I8WpvyXWDfhP8woA6UOssCAtq0ynjPWRX4c7xQmekmWfDw72TCKw9WLrgzuM0T9fCfJjAp5LdtdYWkKs-75G9r6PjKAae-lDPNquOY6ccmyEI-bsOcEArm0EVem61FAbv8amerebxkbmrSWDDX1ebnDMtxLUNAoQu1HRfxtEe9FdwsrcBcYXvwL~UG2IWPhIEmoaaq2Bd8HDeWeatgKstzwXMjVyz42NQ1vaZEGa5CeJo37QMwE2R-a1dgGcmS-kDLTkWr6ZA1KIXxfXmrwLD1uLevumIb0Vqh~8dSpKBeZzTlhgyiD9hlsRTz8CrzWeC1MUe-hD8vzkA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Efficiency of MMEJ catalysed by increasing concentrations of shrimp hepatopancreatic extract. (A) Schematic showing experimental strategy used for evaluating MMEJ in shrimp hepatopancreatic extract. Double-stranded oligomeric DNA possessing 10 nt microhomology region was used as DNA substrates as indicated and incubated with buffer and extracts at 30 °C for 2 h. Following termination of reaction, MMEJ products were amplified using radioactive PCR following labelling of one of the primers. PCR products were resolved on 8% denaturing PAGE and a characteristic 62 nt product indicates joining due to MMEJ. (B) DNA sequence of oligomers used for the study and 10 nt microhomology regions are indicated. (C) PAGE profile showing MMEJ products following incubation of DNA substrates with increasing concentrations of shrimp hepatopancreatic extracts (0.1, 0.25, 0.5, 1 and 2 µg). Lane 1 is no protein control and RTE treated sample was used as positive control. MMEJ product was indicated by arrow. (D) Bar diagram showing error bar (SEM; n = 3) based on the quantitation of multiple biological repeats of MMEJ reaction. The intensity of bands was calculated and expressed in PSL units. C-NHEJ products are shown in bracket. ‘M’ is 60 nt marker and ‘M’ is 50 nt DNA ladder." path-from-xml="dsw059f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055460/dsw059f5.tif" path-from-xml="dsw059f5.tif" target="_blank">View large</a><a section="78055460" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw059/2/dsw059f5.png?Expires=2147483647&Signature=1No6aPJ3gApx5DY9oclhtRrTQXbxihRHWRvZv-X2KBYD3F431VHDmDRsq76yyfIDAqUTTNJxjsv2v0TyQHQRFpP5r4xhuAtMz89XqCeTDumhZK8RktM0KImrtyoaAZCVeI2YEzk8auIGjlE-tGlKIJyhvk06e7gHljPweG6yFgdO68lrFfTRsuXJdi~3AuH0zMEqyWAulIe7JmEAI5s~B2jQc~a5AtWn1YZarJEv8~8mEma7RmMhtVIEM4XkSd3qCQJ-P3BNWxUbiIOC9tLi-3sjRNppyd5dH6O17kp8A1o73fjJiCPb8fp7UxfBkLJ7rPrb3v5OphdEKg0HgJ6Flw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=78055460&ar=2949521&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw059f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Efficiency of MMEJ catalysed by increasing concentrations of shrimp hepatopancreatic extract. (A) Schematic showing experimental strategy used for evaluating MMEJ in shrimp hepatopancreatic extract. Double-stranded oligomeric DNA possessing 10 nt microhomology region was used as DNA substrates as indicated and incubated with buffer and extracts at 30 °C for 2 h. Following termination of reaction, MMEJ products were amplified using radioactive PCR following labelling of one of the primers. PCR products were resolved on 8% denaturing PAGE and a characteristic 62 nt product indicates joining due to MMEJ. (B) DNA sequence of oligomers used for the study and 10 nt microhomology regions are indicated. (C) PAGE profile showing MMEJ products following incubation of DNA substrates with increasing concentrations of shrimp hepatopancreatic extracts (0.1, 0.25, 0.5, 1 and 2 µg). Lane 1 is no protein control and RTE treated sample was used as positive control. MMEJ product was indicated by arrow. (D) Bar diagram showing error bar (SEM; <em>n</em> = 3) based on the quantitation of multiple biological repeats of MMEJ reaction. The intensity of bands was calculated and expressed in PSL units. C-NHEJ products are shown in bracket. ‘M’ is 60 nt marker and ‘M’ is 50 nt DNA ladder.</p></div></div><div content-id="dsw059-F5" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw059/2/m_dsw059f5.png?Expires=2147483647&Signature=Hjay8s2P8I8WpvyXWDfhP8woA6UOssCAtq0ynjPWRX4c7xQmekmWfDw72TCKw9WLrgzuM0T9fCfJjAp5LdtdYWkKs-75G9r6PjKAae-lDPNquOY6ccmyEI-bsOcEArm0EVem61FAbv8amerebxkbmrSWDDX1ebnDMtxLUNAoQu1HRfxtEe9FdwsrcBcYXvwL~UG2IWPhIEmoaaq2Bd8HDeWeatgKstzwXMjVyz42NQ1vaZEGa5CeJo37QMwE2R-a1dgGcmS-kDLTkWr6ZA1KIXxfXmrwLD1uLevumIb0Vqh~8dSpKBeZzTlhgyiD9hlsRTz8CrzWeC1MUe-hD8vzkA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Efficiency of MMEJ catalysed by increasing concentrations of shrimp hepatopancreatic extract. (A) Schematic showing experimental strategy used for evaluating MMEJ in shrimp hepatopancreatic extract. Double-stranded oligomeric DNA possessing 10 nt microhomology region was used as DNA substrates as indicated and incubated with buffer and extracts at 30 °C for 2 h. Following termination of reaction, MMEJ products were amplified using radioactive PCR following labelling of one of the primers. PCR products were resolved on 8% denaturing PAGE and a characteristic 62 nt product indicates joining due to MMEJ. (B) DNA sequence of oligomers used for the study and 10 nt microhomology regions are indicated. (C) PAGE profile showing MMEJ products following incubation of DNA substrates with increasing concentrations of shrimp hepatopancreatic extracts (0.1, 0.25, 0.5, 1 and 2 µg). Lane 1 is no protein control and RTE treated sample was used as positive control. MMEJ product was indicated by arrow. (D) Bar diagram showing error bar (SEM; n = 3) based on the quantitation of multiple biological repeats of MMEJ reaction. The intensity of bands was calculated and expressed in PSL units. C-NHEJ products are shown in bracket. ‘M’ is 60 nt marker and ‘M’ is 50 nt DNA ladder." path-from-xml="dsw059f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055460/dsw059f5.tif" path-from-xml="dsw059f5.tif" target="_blank">View large</a><a section="78055460" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw059/2/dsw059f5.png?Expires=2147483647&Signature=1No6aPJ3gApx5DY9oclhtRrTQXbxihRHWRvZv-X2KBYD3F431VHDmDRsq76yyfIDAqUTTNJxjsv2v0TyQHQRFpP5r4xhuAtMz89XqCeTDumhZK8RktM0KImrtyoaAZCVeI2YEzk8auIGjlE-tGlKIJyhvk06e7gHljPweG6yFgdO68lrFfTRsuXJdi~3AuH0zMEqyWAulIe7JmEAI5s~B2jQc~a5AtWn1YZarJEv8~8mEma7RmMhtVIEM4XkSd3qCQJ-P3BNWxUbiIOC9tLi-3sjRNppyd5dH6O17kp8A1o73fjJiCPb8fp7UxfBkLJ7rPrb3v5OphdEKg0HgJ6Flw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=78055460&ar=2949521&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw059f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Efficiency of MMEJ catalysed by increasing concentrations of shrimp hepatopancreatic extract. (A) Schematic showing experimental strategy used for evaluating MMEJ in shrimp hepatopancreatic extract. Double-stranded oligomeric DNA possessing 10 nt microhomology region was used as DNA substrates as indicated and incubated with buffer and extracts at 30 °C for 2 h. Following termination of reaction, MMEJ products were amplified using radioactive PCR following labelling of one of the primers. PCR products were resolved on 8% denaturing PAGE and a characteristic 62 nt product indicates joining due to MMEJ. (B) DNA sequence of oligomers used for the study and 10 nt microhomology regions are indicated. (C) PAGE profile showing MMEJ products following incubation of DNA substrates with increasing concentrations of shrimp hepatopancreatic extracts (0.1, 0.25, 0.5, 1 and 2 µg). Lane 1 is no protein control and RTE treated sample was used as positive control. MMEJ product was indicated by arrow. (D) Bar diagram showing error bar (SEM; <em>n</em> = 3) based on the quantitation of multiple biological repeats of MMEJ reaction. The intensity of bands was calculated and expressed in PSL units. C-NHEJ products are shown in bracket. ‘M’ is 60 nt marker and ‘M’ is 50 nt DNA ladder.</p></div></div></div>                            <h3 scrollto-destination=78055461 id="78055461" class="section-title" >3.4. MMEJ occurs efficiently at lower temperature and starts within 2 min of addition of hepatopancreas extract</h3>
<div class="block-child-p">In order to determine optimal incubation time for efficient MMEJ catalysed by hepatopancreatic extracts, 0.5 µg of extract was incubated with DNA substrates possessing 10 nt microhomology for various time periods (2, 5, 15, 30 min, 1, 2, 6 h). Results showed that robust joining due to MMEJ was observed as early as 2 min, however, incubation up to 2 h and beyond resulted in reduced joining (<a reveal-id="dsw059-F6" data-open="dsw059-F6" class="link link-reveal link-table xref-fig">Fig. 6A and B</a>). To check the optimal temperature for MMEJ catalysed by the shrimp hepatopancreatic extracts, the DNA substrate and protein extracts were incubated at various temperatures (4, 16, 25, 30 and 37°C). Results showed robust MMEJ at lower temperature. An increase in temperature to 30°C or above resulted in reduced efficiency of joining, most likely due to loss of DNA because of nuclease activity (<a reveal-id="dsw059-F6" data-open="dsw059-F6" class="link link-reveal link-table xref-fig">Fig. 6C and D</a>). Therefore, various lines of experiments suggest that hepatopancreas extract of <em>P. monodon</em> possesses MMEJ, although classical NHEJ was undetectable. <div data-id="dsw059-F6" class="fig fig-section"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw059/2/m_dsw059f6.png?Expires=2147483647&Signature=YCJhSefy-vVaPyX-yLj3O3jLRX3DDsP3O5f1kUfvqOcV4MSYiEeRqG5-m1cOfUoexsmJ8znkY5nakfPBchJwIuGrWLE-L9ndt5IXAJ8-DlCsmuUU6lndtgE0dA9y-b5RW6th7XZ2L87R8LK6Xq9KSBIpeJD6eLsdb9f~GnU2COxnh~7CtjcdufluQKJAhlyu2QFIGsKC86BwrNhP7iZ2y1c6sUa~3O9bzy2~XZ7Eop8ug-Ncp0PMztDi~zUhcFef9adNbVVCahcYZOk0CWTcqfLI9cAxhNrUi~a3Pd1LINrOCzvMdY1jAq~wYCWi8PaaegTH-Ba4PtHgeb4LXyPvxg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Evaluation of MMEJ catalysed by hepatopancreatic extracts of shrimp at different incubation time and temperature. (A) PAGE profile showing time kinetics of MMEJ, when 10 nt microhomology DNA substrate was used. Shrimp hepatopancreatic extracts (0.5 µg) were incubated at 30 °C for different time points (2, 5, 15, 30 min, 1, 2, 6 h). (B) Bar diagram with error bar (SEM) showing quantitation of MMEJ products at different time points (n = 3). (C) PAGE profile showing efficiency of MMEJ catalysed by extracts of shrimp hepatopancreas (0.5 µg) at different temperature (4, 16, 25, 30 and 37 °C) incubated for 2 h. (D) Bar diagram showing error bar (SEM) of MMEJ products at different temperature (n = 3). C-NHEJ products are shown in bracket. ‘M’ is 60 nt marker and ‘M’ is 50 nt DNA ladder." path-from-xml="dsw059f6.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055462/dsw059f6.tif" path-from-xml="dsw059f6.tif" target="_blank">View large</a><a section="78055462" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw059/2/dsw059f6.png?Expires=2147483647&Signature=MYGvM34u8ThWBE2i1SrEQYvbMXvMfQhLYvXMFMgZ1lZ6COglNztJ8PrIerK89bMtq2RCYu7VhWe~vHfSL6lDRi9bOBKeLASVuIy1y4AdFQwJZz-POyyHlon0H8XfbTPbrRsTvsRJCLXXaKl-yNcn7aFOLkUTUbNRsOfsdqk1jj9hZ0J--Sw9dSdE6Hlwk0b43QAEjAz8sMQbCLd6EgMy08lGqKGKr0A8UnNn1oLwkKK0bmgPp6aeXFpCMDBDSuh01FD~PL~oFwya66AFsKDRbaulv5KCdOG~vpS3szZX8GL0tVwNG9jze2IZvWGUnP02Pzo6D5ZaF~wyge9Uc1NTkw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=78055462&ar=2949521&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw059f6.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Evaluation of MMEJ catalysed by hepatopancreatic extracts of shrimp at different incubation time and temperature. (A) PAGE profile showing time kinetics of MMEJ, when 10 nt microhomology DNA substrate was used. Shrimp hepatopancreatic extracts (0.5 µg) were incubated at 30 °C for different time points (2, 5, 15, 30 min, 1, 2, 6 h). (B) Bar diagram with error bar (SEM) showing quantitation of MMEJ products at different time points (<em>n</em> = 3). (C) PAGE profile showing efficiency of MMEJ catalysed by extracts of shrimp hepatopancreas (0.5 µg) at different temperature (4, 16, 25, 30 and 37 °C) incubated for 2 h. (D) Bar diagram showing error bar (SEM) of MMEJ products at different temperature (<em>n</em> = 3). C-NHEJ products are shown in bracket. ‘M’ is 60 nt marker and ‘M’ is 50 nt DNA ladder.</p></div></div><div content-id="dsw059-F6" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw059/2/m_dsw059f6.png?Expires=2147483647&Signature=YCJhSefy-vVaPyX-yLj3O3jLRX3DDsP3O5f1kUfvqOcV4MSYiEeRqG5-m1cOfUoexsmJ8znkY5nakfPBchJwIuGrWLE-L9ndt5IXAJ8-DlCsmuUU6lndtgE0dA9y-b5RW6th7XZ2L87R8LK6Xq9KSBIpeJD6eLsdb9f~GnU2COxnh~7CtjcdufluQKJAhlyu2QFIGsKC86BwrNhP7iZ2y1c6sUa~3O9bzy2~XZ7Eop8ug-Ncp0PMztDi~zUhcFef9adNbVVCahcYZOk0CWTcqfLI9cAxhNrUi~a3Pd1LINrOCzvMdY1jAq~wYCWi8PaaegTH-Ba4PtHgeb4LXyPvxg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Evaluation of MMEJ catalysed by hepatopancreatic extracts of shrimp at different incubation time and temperature. (A) PAGE profile showing time kinetics of MMEJ, when 10 nt microhomology DNA substrate was used. Shrimp hepatopancreatic extracts (0.5 µg) were incubated at 30 °C for different time points (2, 5, 15, 30 min, 1, 2, 6 h). (B) Bar diagram with error bar (SEM) showing quantitation of MMEJ products at different time points (n = 3). (C) PAGE profile showing efficiency of MMEJ catalysed by extracts of shrimp hepatopancreas (0.5 µg) at different temperature (4, 16, 25, 30 and 37 °C) incubated for 2 h. (D) Bar diagram showing error bar (SEM) of MMEJ products at different temperature (n = 3). C-NHEJ products are shown in bracket. ‘M’ is 60 nt marker and ‘M’ is 50 nt DNA ladder." path-from-xml="dsw059f6.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055462/dsw059f6.tif" path-from-xml="dsw059f6.tif" target="_blank">View large</a><a section="78055462" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw059/2/dsw059f6.png?Expires=2147483647&Signature=MYGvM34u8ThWBE2i1SrEQYvbMXvMfQhLYvXMFMgZ1lZ6COglNztJ8PrIerK89bMtq2RCYu7VhWe~vHfSL6lDRi9bOBKeLASVuIy1y4AdFQwJZz-POyyHlon0H8XfbTPbrRsTvsRJCLXXaKl-yNcn7aFOLkUTUbNRsOfsdqk1jj9hZ0J--Sw9dSdE6Hlwk0b43QAEjAz8sMQbCLd6EgMy08lGqKGKr0A8UnNn1oLwkKK0bmgPp6aeXFpCMDBDSuh01FD~PL~oFwya66AFsKDRbaulv5KCdOG~vpS3szZX8GL0tVwNG9jze2IZvWGUnP02Pzo6D5ZaF~wyge9Uc1NTkw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=78055462&ar=2949521&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw059f6.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Evaluation of MMEJ catalysed by hepatopancreatic extracts of shrimp at different incubation time and temperature. (A) PAGE profile showing time kinetics of MMEJ, when 10 nt microhomology DNA substrate was used. Shrimp hepatopancreatic extracts (0.5 µg) were incubated at 30 °C for different time points (2, 5, 15, 30 min, 1, 2, 6 h). (B) Bar diagram with error bar (SEM) showing quantitation of MMEJ products at different time points (<em>n</em> = 3). (C) PAGE profile showing efficiency of MMEJ catalysed by extracts of shrimp hepatopancreas (0.5 µg) at different temperature (4, 16, 25, 30 and 37 °C) incubated for 2 h. (D) Bar diagram showing error bar (SEM) of MMEJ products at different temperature (<em>n</em> = 3). C-NHEJ products are shown in bracket. ‘M’ is 60 nt marker and ‘M’ is 50 nt DNA ladder.</p></div></div></div>                            <h3 scrollto-destination=78055463 id="78055463" class="section-title" >3.5. HR-mediated DSB repair is efficient in <em>P. monodon</em></h3>
<div class="block-child-p">Since DSB repair mediated through MMEJ results in deletion of sequence between microhomology regions, this could result in deletions in genome and hence genomic instability. Therefore, we were interested in investigating the mechanism by which genomic stability is maintained. To test this, HR-mediated DSB repair was evaluated in hepatopancreas extract of <em>P. monodon</em> using plasmid-based assay system. Two plasmids with two independent mutations at neomycin gene (pTO223 and pTO231) were used to evaluate DSB repair catalysed through HR (<a reveal-id="dsw059-F7" data-open="dsw059-F7" class="link link-reveal link-table xref-fig">Fig. 7A</a>). The plasmids were incubated with hepatopancreatic extracts in HR buffer and purified DNA was transformed in <em>E. coli</em> DH5α. Recombinants were scored on kanamycin containing agar plates. Recombination catalysed by the extracts, will result in the restoration of functional <em>neo<sup>+ </sup></em>gene conferring resistance to kanamycin upon transformation in recombination deficient, <em>E.</em><em>coli</em> (<em>recA</em><sup><em>−</em></sup> strain of DH5αF<sup><em>−</em></sup>). <div data-id="dsw059-F7" class="fig fig-section"><div class="label fig-label">Figure 7</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw059/2/m_dsw059f7.png?Expires=2147483647&Signature=Vv35Cg-rIpKB-BgUzXlXLPzJt1jwepvO0MCQIPQortrpGwiS5IoLxmyNicIJddURUUiHRpAWkFi1jYxa~caOzOTjuMVorn0s0VuBjWax6jslzccZ7LcXMYx9KhJFvt2wAr4VrbYlpecyQlkB~HM5ZCHYbih6iPPzq-eg8GTot5Urnz~erWfS1-lUSD0~F4m5TKwNEg-RmM0bvpgoi0UjPINSIS45x2pmqNwNb2R-cCSlZT6sC~VSaUk6XSrYykre0JL1QnpFVEVfSD6DV~jKgCxoZHY80xZBEb1Idc220ItxVFEdaliimWsK1hEa3V9NqSVmr6GzozVbQ1g0IQT6TA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Evaluation of HR-mediated DSB repair catalysed by hepatopancreas extracts. (A) Schematic presentation showing plasmid based HR assay. Plasmids (pTO223 and pTO231) were incubated with shrimp hepatopancreas CFE in HR assay buffer at 30 °C for 30 min. Purified DNA was used for transformation of E. coli DH5α and plated on agar plates containing ampicillin to determine transformation efficiency and kanamycin to calculate recombination frequency. (B) Table showing in vitro recombination catalysed by hepatopancreas extract and its heat inactivated form. Mouse testicular extracts acted as a positive control. (C) Efficacy of HR-mediated DSB repair in shrimp hepatopancreatic extract when examined at different incubation temperatures. Hepatopancreatic extracts (0.5 µg) were incubated with substrate DNA at different temperature (4, 16, 24, 30, 37 °C). (D) Bar diagram showing recombination frequencies catalysed by increasing concentrations (0.5, 1, 1.5, 2, 4 µg) of shrimp hepatopancreas extract. ‘C’ is no protein control." path-from-xml="dsw059f7.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055464/dsw059f7.tif" path-from-xml="dsw059f7.tif" target="_blank">View large</a><a section="78055464" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw059/2/dsw059f7.png?Expires=2147483647&Signature=GbWRP2GbKmiO4S56GRi~GDif0hTnyH2gIuAQFq7SFfQDB8ebyDlb8ssqBYq-zODmXFAiVKCUttcNNv6vzJL71ZpFDyFxFqH6tCOMTVm5FyjsQZLwXa6Wymf5qBvcHKkSvUTUDLLyXDBQWO16TYWJZD-2Z4mItQFvEGDoMSOZX1S-KTVNsV-kUZ417TbRnEHcmAURE2JBXgHjPxngbwEuW79nIb~mPay0v97rehLv1mErRhFELx~BDacv~vMeRdjeDWd~3aA9W1KtO64FbgY1PjDcu3JbJGxjnL3jUUuoukZumdbPKmbv-vzFdpSpmsnfWzhu5jICh3zDXFwm7K~lSA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=78055464&ar=2949521&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw059f7.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Evaluation of HR-mediated DSB repair catalysed by hepatopancreas extracts. (A) Schematic presentation showing plasmid based HR assay. Plasmids (pTO223 and pTO231) were incubated with shrimp hepatopancreas CFE in HR assay buffer at 30 °C for 30 min. Purified DNA was used for transformation of <em>E. coli</em> DH5α and plated on agar plates containing ampicillin to determine transformation efficiency and kanamycin to calculate recombination frequency. (B) Table showing <em>in vitro</em> recombination catalysed by hepatopancreas extract and its heat inactivated form. Mouse testicular extracts acted as a positive control. (C) Efficacy of HR-mediated DSB repair in shrimp hepatopancreatic extract when examined at different incubation temperatures. Hepatopancreatic extracts (0.5 µg) were incubated with substrate DNA at different temperature (4, 16, 24, 30, 37 °C). (D) Bar diagram showing recombination frequencies catalysed by increasing concentrations (0.5, 1, 1.5, 2, 4 µg) of shrimp hepatopancreas extract. ‘C’ is no protein control.</p></div></div><div content-id="dsw059-F7" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 7</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw059/2/m_dsw059f7.png?Expires=2147483647&Signature=Vv35Cg-rIpKB-BgUzXlXLPzJt1jwepvO0MCQIPQortrpGwiS5IoLxmyNicIJddURUUiHRpAWkFi1jYxa~caOzOTjuMVorn0s0VuBjWax6jslzccZ7LcXMYx9KhJFvt2wAr4VrbYlpecyQlkB~HM5ZCHYbih6iPPzq-eg8GTot5Urnz~erWfS1-lUSD0~F4m5TKwNEg-RmM0bvpgoi0UjPINSIS45x2pmqNwNb2R-cCSlZT6sC~VSaUk6XSrYykre0JL1QnpFVEVfSD6DV~jKgCxoZHY80xZBEb1Idc220ItxVFEdaliimWsK1hEa3V9NqSVmr6GzozVbQ1g0IQT6TA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Evaluation of HR-mediated DSB repair catalysed by hepatopancreas extracts. (A) Schematic presentation showing plasmid based HR assay. Plasmids (pTO223 and pTO231) were incubated with shrimp hepatopancreas CFE in HR assay buffer at 30 °C for 30 min. Purified DNA was used for transformation of E. coli DH5α and plated on agar plates containing ampicillin to determine transformation efficiency and kanamycin to calculate recombination frequency. (B) Table showing in vitro recombination catalysed by hepatopancreas extract and its heat inactivated form. Mouse testicular extracts acted as a positive control. (C) Efficacy of HR-mediated DSB repair in shrimp hepatopancreatic extract when examined at different incubation temperatures. Hepatopancreatic extracts (0.5 µg) were incubated with substrate DNA at different temperature (4, 16, 24, 30, 37 °C). (D) Bar diagram showing recombination frequencies catalysed by increasing concentrations (0.5, 1, 1.5, 2, 4 µg) of shrimp hepatopancreas extract. ‘C’ is no protein control." path-from-xml="dsw059f7.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055464/dsw059f7.tif" path-from-xml="dsw059f7.tif" target="_blank">View large</a><a section="78055464" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw059/2/dsw059f7.png?Expires=2147483647&Signature=GbWRP2GbKmiO4S56GRi~GDif0hTnyH2gIuAQFq7SFfQDB8ebyDlb8ssqBYq-zODmXFAiVKCUttcNNv6vzJL71ZpFDyFxFqH6tCOMTVm5FyjsQZLwXa6Wymf5qBvcHKkSvUTUDLLyXDBQWO16TYWJZD-2Z4mItQFvEGDoMSOZX1S-KTVNsV-kUZ417TbRnEHcmAURE2JBXgHjPxngbwEuW79nIb~mPay0v97rehLv1mErRhFELx~BDacv~vMeRdjeDWd~3aA9W1KtO64FbgY1PjDcu3JbJGxjnL3jUUuoukZumdbPKmbv-vzFdpSpmsnfWzhu5jICh3zDXFwm7K~lSA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=78055464&ar=2949521&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw059f7.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Evaluation of HR-mediated DSB repair catalysed by hepatopancreas extracts. (A) Schematic presentation showing plasmid based HR assay. Plasmids (pTO223 and pTO231) were incubated with shrimp hepatopancreas CFE in HR assay buffer at 30 °C for 30 min. Purified DNA was used for transformation of <em>E. coli</em> DH5α and plated on agar plates containing ampicillin to determine transformation efficiency and kanamycin to calculate recombination frequency. (B) Table showing <em>in vitro</em> recombination catalysed by hepatopancreas extract and its heat inactivated form. Mouse testicular extracts acted as a positive control. (C) Efficacy of HR-mediated DSB repair in shrimp hepatopancreatic extract when examined at different incubation temperatures. Hepatopancreatic extracts (0.5 µg) were incubated with substrate DNA at different temperature (4, 16, 24, 30, 37 °C). (D) Bar diagram showing recombination frequencies catalysed by increasing concentrations (0.5, 1, 1.5, 2, 4 µg) of shrimp hepatopancreas extract. ‘C’ is no protein control.</p></div></div></div><p>HR assay was performed by incubating shrimp hepatopancreatic extracts (2 µg) along with pTO223 and pTO231. DNA treated with heat inactivated extracts served as the control (<a reveal-id="dsw059-F7" data-open="dsw059-F7" class="link link-reveal link-table xref-fig">Fig. 7B</a>). No protein control acted as negative control for bacterial recombination efficiency. Results showed efficient recombination catalysed by hepatopancreas extract, which was ∼200-fold higher than the recombination frequency of no protein control and heat inactivated extract control (<a reveal-id="dsw059-F7" data-open="dsw059-F7" class="link link-reveal link-table xref-fig">Fig. 7B</a>). Importantly, the observed HR efficiency was &gt;30-fold higher than that of HR efficiency of mouse testicular extract (<a reveal-id="dsw059-F7" data-open="dsw059-F7" class="link link-reveal link-table xref-fig">Fig. 7B</a>). Therefore, cell-free extract of hepatopancreas is proficient in HR-mediated DSB repair.</p><p>Further protein titration was performed to determine the optimal extract concentration for HR-mediated DSB repair activity. Results showed a concentration (0.5, 1, 1.5, 2, 4 µg) dependent increase in the recombination frequency (<a reveal-id="dsw059-F7" data-open="dsw059-F7" class="link link-reveal link-table xref-fig">Fig. 7D</a>). We have also checked the effect of temperature on the efficiency of HR-mediated DSB repair. Hepatopancreas extract (0.5 µg) was incubated with pTO223 and pTO231 at different temperatures (4, 16, 24, 30, 37°C) for 30 min (<a reveal-id="dsw059-F7" data-open="dsw059-F7" class="link link-reveal link-table xref-fig">Fig. 7C</a>). The purified plasmids were transformed in <em>E.</em><em>coli</em> and scored for recombinants as described above. Results showed the highest recombination frequency at 30 and 37°C. Efficiency of recombination was lowest at 4 and 42°C (data not shown), while it was moderate at 16 and 24°C. These results in conjunction with earlier results suggest that HR is predominant at the physiological temperature, while MMEJ efficiency is poor. In contrast, HR frequency was poor at lower temperature, whereas MMEJ efficiency was high.</p>                            <h3 scrollto-destination=78055467 id="78055467" class="section-title" >3.6. HR-mediated recombination in hepatopancreas extracts occurs through gene conversion and reciprocal exchange</h3>
<div class="block-child-p">Recombinant clones were evaluated for joining through reciprocal recombination and gene conversion by restriction enzyme digestion. During reciprocal exchange, a crossing over within the region between deleted neomycin gene, <em>neoΔ1</em> and <em>neoΔ2</em> takes place, giving rise to functional <em>neo<sup>+</sup> </em>allele and a nonfunctional double mutant allele. In contrast during gene conversion, a transfer of DNA from <em>neoΔ1</em> to <em>neoΔ2</em> takes place, giving rise to functional <em>neo</em> gene. When digested with <em>Eco</em>RI<em>–</em><em>Sal</em>I, pTO223 and pTO231 can give rise to a 1.25- and 1.2-kb fragments, respectively, and is indicative of <em>neoΔ2</em> and <em>neoΔ1</em> allele (<a reveal-id="dsw059-F8" data-open="dsw059-F8" class="link link-reveal link-table xref-fig">Fig. 8A</a>). In contrast, digestion of plasmids containing <em>neo</em> gene will result in release of a 1.5-kb fragment that is indicative of presence of functional <em>neo</em> gene. <div data-id="dsw059-F8" class="fig fig-section"><div class="label fig-label">Figure 8</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw059/2/m_dsw059f8.png?Expires=2147483647&Signature=epS1kEMBI6q664XDt2rNE2V7~ZB27IQXfxWiEgG6XTtnAR~yinyxGLZUzsXsHsaQSvFumFidvTp4~ESszZbixI4TPF9UDrrin0PpkVeB4jmdgEl1SCFQgkvu8MdKLJg7CrPvtOoXBlcChmUoOGd7Q6FERWRzTQCtdeuQxwECw9IdpXpfe2ewF2d1b~44UvwJnmtzub-9R~hz8762i8NLKgMREbBgYvlj4e9tbv8WSTzPqdtKlSt2hKnPZ6bpBllZnhWx1Ryqw11n5TsPEwVdm3ocnZjl5xqk2cVEaNX9AStE8bmka1QHA9Xis1oXPHMa-KbgIfBcNpx4pF8PPPhlQA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Evaluation of molecular mechanism of HR-mediated DSB repair in hepatopancreatic extract of shrimp. (A) Schematics showing reciprocal recombination and gene conversion-mediated HR that can lead to recreation of functional neo gene. neoΔ1 in pTO231 and neoΔ2 in pTO223 are nonfunctional neomycin genes. neoΔ1 lacks NaeI towards right side of the gene (shown in red) and neoΔ2 lacks NarI towards the left side of the gene (shown in red). A crossing over between two mutants will give rise to a functional neo gene and a nonfunctional allele containing both the deletions. In the case of gene conversion, transfer of DNA from neoΔ1 to neoΔ2 takes place resulting in functional neo gene. HindIII site present adjacent to EcoRI site was destroyed in neoΔ1, hence HindIII/SalI resulting in 1.5 kb fragment occurs only at the time of gene conversion. (B) Representative agarose gel profile showing EcoRI/SalI restriction digestion of recombinant clones. Lanes 1–7 show release of fragment size of 1.5 kb of functional neo gene; 223 and 231 are mutant plasmids. (C) Agarose gel profile showing HindIII/SalI restriction digestion of recombinants. Lanes 1–7 show gene conversion releasing 1.5 kb of functional neo gene, except in the case of lane 4. Lane 3 showed a partially digested clone. ‘M’ is 1 kb DNA ladder. (D) Bar diagram showing frequency of HR-mediated DSB repair through either gene conversion or reciprocal recombination." path-from-xml="dsw059f8.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055468/dsw059f8.tif" path-from-xml="dsw059f8.tif" target="_blank">View large</a><a section="78055468" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw059/2/dsw059f8.png?Expires=2147483647&Signature=YmtwqPHL90Jb~xYtPjr-1rpuxneDa-fTEtGZi07wkvpCi0s4uoCutAdl0UKpy9N3~C5tgjJVBA2GTvaEu9jDPZ6VjeuRA2k4AbQOUwpwnAYO8oh6RgG9z7XVGuhAfFMKzG6yubnz56GSZIjI3IwYOXaxXiiyGGHLviSyYG~BGiySdrITpdrFJ2zu1Lxk~LL2oR4rIZSvMGf3EwZAccs8PkqsF-d6lTVoF1-Aw2cCPU5iV3NEQ0QCPsUxTuGeW93IRSQ~x9u19XD~uyV~3ZOui8RNQz5b4G54i7yavnrzwMhuLCLQhW40-wvM3jStHesPZj32OMBBFvJeMvJ3Q~-1OQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=78055468&ar=2949521&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw059f8.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Evaluation of molecular mechanism of HR-mediated DSB repair in hepatopancreatic extract of shrimp. (A) Schematics showing reciprocal recombination and gene conversion-mediated HR that can lead to recreation of functional <em>neo</em> gene. <em>neoΔ1 in pTO231</em> and <em>neoΔ2</em> in pTO223 are nonfunctional <em>neomycin</em> genes. <em>neoΔ1</em> lacks <em>NaeI</em> towards right side of the gene (shown in red) and <em>neoΔ2</em> lacks <em>NarI</em> towards the left side of the gene (shown in red). A crossing over between two mutants will give rise to a functional <em>neo</em> gene and a nonfunctional allele containing both the deletions. In the case of gene conversion, transfer of DNA from <em>neoΔ1</em> to <em>neoΔ2</em> takes place resulting in functional <em>neo</em> gene. <em>Hin</em>dIII site present adjacent to <em>Eco</em>RI site was destroyed in <em>neoΔ1</em>, hence <em>Hin</em>dIII<em>/Sal</em>I resulting in 1.5 kb fragment occurs only at the time of gene conversion. (B) Representative agarose gel profile showing <em>Eco</em>RI<em>/Sal</em>I restriction digestion of recombinant clones. Lanes 1–7 show release of fragment size of 1.5 kb of functional <em>neo</em> gene; 223 and 231 are mutant plasmids. (C) Agarose gel profile showing <em>Hin</em>dIII<em>/Sal</em>I restriction digestion of recombinants. Lanes 1–7 show gene conversion releasing 1.5 kb of functional <em>neo</em> gene, except in the case of lane 4. Lane 3 showed a partially digested clone. ‘M’ is 1 kb DNA ladder. (D) Bar diagram showing frequency of HR-mediated DSB repair through either gene conversion or reciprocal recombination.</p></div></div><div content-id="dsw059-F8" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 8</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw059/2/m_dsw059f8.png?Expires=2147483647&Signature=epS1kEMBI6q664XDt2rNE2V7~ZB27IQXfxWiEgG6XTtnAR~yinyxGLZUzsXsHsaQSvFumFidvTp4~ESszZbixI4TPF9UDrrin0PpkVeB4jmdgEl1SCFQgkvu8MdKLJg7CrPvtOoXBlcChmUoOGd7Q6FERWRzTQCtdeuQxwECw9IdpXpfe2ewF2d1b~44UvwJnmtzub-9R~hz8762i8NLKgMREbBgYvlj4e9tbv8WSTzPqdtKlSt2hKnPZ6bpBllZnhWx1Ryqw11n5TsPEwVdm3ocnZjl5xqk2cVEaNX9AStE8bmka1QHA9Xis1oXPHMa-KbgIfBcNpx4pF8PPPhlQA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Evaluation of molecular mechanism of HR-mediated DSB repair in hepatopancreatic extract of shrimp. (A) Schematics showing reciprocal recombination and gene conversion-mediated HR that can lead to recreation of functional neo gene. neoΔ1 in pTO231 and neoΔ2 in pTO223 are nonfunctional neomycin genes. neoΔ1 lacks NaeI towards right side of the gene (shown in red) and neoΔ2 lacks NarI towards the left side of the gene (shown in red). A crossing over between two mutants will give rise to a functional neo gene and a nonfunctional allele containing both the deletions. In the case of gene conversion, transfer of DNA from neoΔ1 to neoΔ2 takes place resulting in functional neo gene. HindIII site present adjacent to EcoRI site was destroyed in neoΔ1, hence HindIII/SalI resulting in 1.5 kb fragment occurs only at the time of gene conversion. (B) Representative agarose gel profile showing EcoRI/SalI restriction digestion of recombinant clones. Lanes 1–7 show release of fragment size of 1.5 kb of functional neo gene; 223 and 231 are mutant plasmids. (C) Agarose gel profile showing HindIII/SalI restriction digestion of recombinants. Lanes 1–7 show gene conversion releasing 1.5 kb of functional neo gene, except in the case of lane 4. Lane 3 showed a partially digested clone. ‘M’ is 1 kb DNA ladder. (D) Bar diagram showing frequency of HR-mediated DSB repair through either gene conversion or reciprocal recombination." path-from-xml="dsw059f8.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055468/dsw059f8.tif" path-from-xml="dsw059f8.tif" target="_blank">View large</a><a section="78055468" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw059/2/dsw059f8.png?Expires=2147483647&Signature=YmtwqPHL90Jb~xYtPjr-1rpuxneDa-fTEtGZi07wkvpCi0s4uoCutAdl0UKpy9N3~C5tgjJVBA2GTvaEu9jDPZ6VjeuRA2k4AbQOUwpwnAYO8oh6RgG9z7XVGuhAfFMKzG6yubnz56GSZIjI3IwYOXaxXiiyGGHLviSyYG~BGiySdrITpdrFJ2zu1Lxk~LL2oR4rIZSvMGf3EwZAccs8PkqsF-d6lTVoF1-Aw2cCPU5iV3NEQ0QCPsUxTuGeW93IRSQ~x9u19XD~uyV~3ZOui8RNQz5b4G54i7yavnrzwMhuLCLQhW40-wvM3jStHesPZj32OMBBFvJeMvJ3Q~-1OQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=78055468&ar=2949521&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw059f8.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Evaluation of molecular mechanism of HR-mediated DSB repair in hepatopancreatic extract of shrimp. (A) Schematics showing reciprocal recombination and gene conversion-mediated HR that can lead to recreation of functional <em>neo</em> gene. <em>neoΔ1 in pTO231</em> and <em>neoΔ2</em> in pTO223 are nonfunctional <em>neomycin</em> genes. <em>neoΔ1</em> lacks <em>NaeI</em> towards right side of the gene (shown in red) and <em>neoΔ2</em> lacks <em>NarI</em> towards the left side of the gene (shown in red). A crossing over between two mutants will give rise to a functional <em>neo</em> gene and a nonfunctional allele containing both the deletions. In the case of gene conversion, transfer of DNA from <em>neoΔ1</em> to <em>neoΔ2</em> takes place resulting in functional <em>neo</em> gene. <em>Hin</em>dIII site present adjacent to <em>Eco</em>RI site was destroyed in <em>neoΔ1</em>, hence <em>Hin</em>dIII<em>/Sal</em>I resulting in 1.5 kb fragment occurs only at the time of gene conversion. (B) Representative agarose gel profile showing <em>Eco</em>RI<em>/Sal</em>I restriction digestion of recombinant clones. Lanes 1–7 show release of fragment size of 1.5 kb of functional <em>neo</em> gene; 223 and 231 are mutant plasmids. (C) Agarose gel profile showing <em>Hin</em>dIII<em>/Sal</em>I restriction digestion of recombinants. Lanes 1–7 show gene conversion releasing 1.5 kb of functional <em>neo</em> gene, except in the case of lane 4. Lane 3 showed a partially digested clone. ‘M’ is 1 kb DNA ladder. (D) Bar diagram showing frequency of HR-mediated DSB repair through either gene conversion or reciprocal recombination.</p></div></div></div><p>Upon <em>Hin</em>dIII<em>–</em><em>Sal</em>I double digestion, release of 1.25 kb fragment from pTO223 was observed, which was similar to that of <em>Eco</em>RI<em>–</em><em>Sal</em>I digestion, while a larger 4.1-kb fragment was released from pTO231 due to the absence of <em>Hin</em>dIII site (<a reveal-id="dsw059-F8" data-open="dsw059-F8" class="link link-reveal link-table xref-fig">Fig. 8A</a>) (when digested, pTO231 gets linearized with <em>Sal</em>I giving rise to single large 4.1 kb fragment). Upon HR, the <em>neo</em> gene comprising <em>neoΔ1</em> allele lacking <em>Hin</em>dIII site and right part of <em>neoΔ2</em> allele with <em>Sal</em>I site resulted in the absence of 1.5 kb fragment, which was seen upon <em>Eco</em>RI<em>–</em><em>Sal</em>I double digestion. In contrast, during gene conversion, transfer of DNA fragment from <em>neoΔ1</em> to <em>neoΔ2</em> allele takes place, and therefore, <em>Hin</em>dIII<em>–</em><em>Sal</em>I digestion resulted in the release of a 1.5-kb fragment. Thus, the release of a 1.5-kb fragment during <em>Eco</em>RI<em>–</em><em>Sal</em>I digestion is indicative of the presence of functional <em>neo</em> gene (<a reveal-id="dsw059-F8" data-open="dsw059-F8" class="link link-reveal link-table xref-fig">Fig. 8B</a>), while a release of 1.5 kb fragment due to <em>Hin</em>dIII<em>–</em><em>Sal</em>I digestion is indicative of wild-type <em>neo</em> gene due to gene conversion (<a reveal-id="dsw059-F8" data-open="dsw059-F8" class="link link-reveal link-table xref-fig">Fig. 8C</a>). However, it is important to note that gene conversion between <em>neoΔ2</em> to <em>neoΔ1</em>, cannot be scored upon <em>Hin</em>dIII<em>–</em><em>Sal</em>I digestion and therefore, observed frequency of gene conversion could be an under representation.</p><div class="block-child-p">These observations revealed that HR-mediated DSB repair occurred using both gene conversion and reciprocal exchange. Among the clones analysed, in &gt;60% of the cases HR was through gene conversion (<a reveal-id="dsw059-F8" data-open="dsw059-F8" class="link link-reveal link-table xref-fig">Fig. 8D</a>). Therefore, our results suggested that DSB repair in <em>P. monodon</em> is dependent on HR and microhomology-mediated DNA end joining (<a reveal-id="dsw059-F9" data-open="dsw059-F9" class="link link-reveal link-table xref-fig">Fig. 9</a>). However, unlike mammals classical NHEJ was undetectable (<a reveal-id="dsw059-F9" data-open="dsw059-F9" class="link link-reveal link-table xref-fig">Fig. 9</a>). <div data-id="dsw059-F9" class="fig fig-section"><div class="label fig-label">Figure 9</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw059/2/m_dsw059f9.png?Expires=2147483647&Signature=bZ~adIMyLvI2Otrtb3~EwZq0l-LrzlhVEBs91LCGo3uiy5dXWaCRiLxBuDUo7eNVVhmZNcEV4xU6WVs9Wf1q3FsQLCCyG8lEjBSDcRtPzExzwBviPUKuPRr6kkckzgQLwUYuezzsaJT47IyJs5PamB-6~h~ClcRMQ2A2IYohEwZZrfuGbq0qs7vc8XF0aeaeZHGDmE-bYLo8NJGe1RXf74GLoC7OVhvS9wkJHvu2Cw7WyjSSXKYhS3iyfS7KMHm1L-0jFgatgyNe~msBiroC19~F0B6U1DpCe10hd8CDVzEIIEweAw9-HP41CFSQIDowWsqKuJ6UVHjFuW2kwcrgJA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Cartoon depicting comparison of major DSB repair pathways in mammals and Penaeus monodon. Unlike mammals, HR and alternative NHEJ (MMEJ) are the major DNA break repair pathways in shrimp. Classical NHEJ and HR are the major DSB repair pathways in mammals, while MMEJ operates only at a low level." path-from-xml="dsw059f9.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055470/dsw059f9.tif" path-from-xml="dsw059f9.tif" target="_blank">View large</a><a section="78055470" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw059/2/dsw059f9.png?Expires=2147483647&Signature=prFgH6-dvIvDufUtNOCqlEPF~bI46qTCqjeq0UmvQ~Lx3sX0-sif3Nzsxu1FnxKCxP-Q6ZIpSPc0a2qnAbfFGpAG9yINtxdnoHNxKfe9asrXepy50zITZgtBd44YwCyjrnO45pL8anJOA7-abKBMe2gYVhd0El5h3vJ-tE9guf-KFoxY3VJFJc4dIy6PbMCq5KbWrAbbix3~LED5EIaAKzvXLAadsNYglpQlFNLduC27-mq59fxz6asUApRDCpu4IVi8vcQsfYfutua337F1S0Dqcs4q~QTUZ0UGb34JpNgkfSCT8LVJ6ijZJugbw0LiQkqeckBEIyNHrPIoglgZ1Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=78055470&ar=2949521&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw059f9.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Cartoon depicting comparison of major DSB repair pathways in mammals and <em>Penaeus monodon.</em> Unlike mammals, HR and alternative NHEJ (MMEJ) are the major DNA break repair pathways in shrimp. Classical NHEJ and HR are the major DSB repair pathways in mammals, while MMEJ operates only at a low level.</p></div></div><div content-id="dsw059-F9" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 9</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw059/2/m_dsw059f9.png?Expires=2147483647&Signature=bZ~adIMyLvI2Otrtb3~EwZq0l-LrzlhVEBs91LCGo3uiy5dXWaCRiLxBuDUo7eNVVhmZNcEV4xU6WVs9Wf1q3FsQLCCyG8lEjBSDcRtPzExzwBviPUKuPRr6kkckzgQLwUYuezzsaJT47IyJs5PamB-6~h~ClcRMQ2A2IYohEwZZrfuGbq0qs7vc8XF0aeaeZHGDmE-bYLo8NJGe1RXf74GLoC7OVhvS9wkJHvu2Cw7WyjSSXKYhS3iyfS7KMHm1L-0jFgatgyNe~msBiroC19~F0B6U1DpCe10hd8CDVzEIIEweAw9-HP41CFSQIDowWsqKuJ6UVHjFuW2kwcrgJA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Cartoon depicting comparison of major DSB repair pathways in mammals and Penaeus monodon. Unlike mammals, HR and alternative NHEJ (MMEJ) are the major DNA break repair pathways in shrimp. Classical NHEJ and HR are the major DSB repair pathways in mammals, while MMEJ operates only at a low level." path-from-xml="dsw059f9.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055470/dsw059f9.tif" path-from-xml="dsw059f9.tif" target="_blank">View large</a><a section="78055470" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw059/2/dsw059f9.png?Expires=2147483647&Signature=prFgH6-dvIvDufUtNOCqlEPF~bI46qTCqjeq0UmvQ~Lx3sX0-sif3Nzsxu1FnxKCxP-Q6ZIpSPc0a2qnAbfFGpAG9yINtxdnoHNxKfe9asrXepy50zITZgtBd44YwCyjrnO45pL8anJOA7-abKBMe2gYVhd0El5h3vJ-tE9guf-KFoxY3VJFJc4dIy6PbMCq5KbWrAbbix3~LED5EIaAKzvXLAadsNYglpQlFNLduC27-mq59fxz6asUApRDCpu4IVi8vcQsfYfutua337F1S0Dqcs4q~QTUZ0UGb34JpNgkfSCT8LVJ6ijZJugbw0LiQkqeckBEIyNHrPIoglgZ1Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=78055470&ar=2949521&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw059f9.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Cartoon depicting comparison of major DSB repair pathways in mammals and <em>Penaeus monodon.</em> Unlike mammals, HR and alternative NHEJ (MMEJ) are the major DNA break repair pathways in shrimp. Classical NHEJ and HR are the major DSB repair pathways in mammals, while MMEJ operates only at a low level.</p></div></div></div>                            <h2 scrollto-destination=78055471 id="78055471" class="section-title" >4. Discussion</h2>
<p>Multiple DNA repair pathways have evolved to combat DNA damage, ensuring survival of a cell.<a reveal-id="dsw059-B2" data-open="dsw059-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a><sup>,</sup><a reveal-id="dsw059-B3" data-open="dsw059-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a><sup>,</sup><a reveal-id="dsw059-B10" data-open="dsw059-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> From prokaryotes to lower eukaryotes and to higher eukaryotes, several of these DNA repair pathways are conserved. Several DNA repair proteins also show high degree of amino acid homology across the living world. However, there are several organisms, where DNA repair pathways are unexplored due to various reasons including non-conservation of polypeptides or lack of complete genome sequence making the study of DNA repair a challenge.</p><p>In the present study, we investigated DSB repair pathways in one of the crustaceans, <em>P.</em><em>monodon</em>, using a cell-free assay system of one of the most important organs, hepatopancreas. Using the cell free-extract, which contains almost all the cellular proteins, we observed that NHEJ, the major DSB repair pathway in mammals is undetectable in the crustaceans. Interestingly, the more precise and faithful, HR-mediated repair is predominant and efficient. Although classical NHEJ was absent, a recently described backup NHEJ, known as MMEJ is functional in this organism. However, the protein machinery involved in each of these DNA repair pathway needs to be investigated.</p>                            <h3 scrollto-destination=78055474 id="78055474" class="section-title" >4.1. HR is proficient in <em>P. monodon</em></h3>
<p>Unlike other DSB repair mechanisms, HR-mediated repair results in faithful joining of broken DNA without nucleotide loss. Although NHEJ is the predominant DSB repair mechanism in mammals, HR is the major DSB repair pathway in prokaryotes, and lower eukaryotes. Interestingly, we observed that HR-mediated DSB repair is highly efficient in <em>P. monodon.</em> Interestingly, we observed &gt;30-fold higher recombination frequency, when extracts of shrimp hepatopancreas was compared with testicular extract. This is indeed remarkable and underlines the importance of precise repair, and maintenance of integrity of the genome. Although a hepatopancreatic protein concentration-dependent increase in the recombination frequency was observed, recombination frequency was highest at 30 or 37°C and the efficiency was significantly less at lower temperature. There are several DNA repair proteins present in <em>Drosophila</em> that are reported to possess sequence similarity with other species. It was reported that HR is the predominant DSB repair pathway in mitotically dividing cells of <em>Drosophila</em>.<a reveal-id="dsw059-B42" data-open="dsw059-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a></p><p>We noted that although HR-mediated DSB repair could occur through reciprocal exchange and gene conversion, in the case of <em>P. monodon</em>, the most preferred pathway was gene conversion (&gt;60% of the cases). Considering that cell type used in the present study was of somatic cells, in hepatopancreas, this is understandable. In <em>E. coli</em>, RecA is the major component of DNA repair system. In <em>S. cerevisiae</em>, Rad51 family of proteins are present, which are similar to RecA proteins present in <em>E. coli</em>.<a reveal-id="dsw059-B28" data-open="dsw059-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> An ortholog of Rad51 has been found in <em>Drosophila</em><a reveal-id="dsw059-B43" data-open="dsw059-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a> and another one closely related to human Xrcc3,<a reveal-id="dsw059-B44" data-open="dsw059-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a> but <em>Drosophila</em> lacks Dmc1 ortholog. Although, we have not investigated the protein machinery involved in HR-mediated repair in shrimp, it is very much possible that RAD51 homologs will be playing a critical role. Similarly, homologs of MRE11, RAD50 and NBS1 also need to be investigated. Furthermore, it will be interesting to see whether RAD51 paralogues play any significant role in crustaceans.</p>                            <h3 scrollto-destination=78055477 id="78055477" class="section-title" >4.2. MMEJ is preferred over classical NHEJ</h3>
<p>The present study suggested that cell-free extracts prepared from hepatopancreas of <em>P. monodon</em> possess microhomology-mediated end joining, while classical NHEJ was undetectable. This is surprising, when one considers the fact that joining through MMEJ could lead to deletions in the genome culminating into genomic instability.<a reveal-id="dsw059-B12" data-open="dsw059-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a><sup>,</sup><a reveal-id="dsw059-B18" data-open="dsw059-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> It may be possible that NHEJ proteins such as Ligase IV/XRCC4, Ku70/Ku80, Artemis, DNA-PKcs etc., may be absent or expressed low in crustaceans, although it needs to be verified. Similarly, it will be interesting to look at the levels of MMEJ proteins such as Ligase III, PARP1, FEN1 in <em>P. monodon.</em></p><p>Previously, it has been suggested that MMEJ operates in those cells, which are devoid of C-NHEJ.<a reveal-id="dsw059-B12" data-open="dsw059-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a><sup>,</sup><a reveal-id="dsw059-B18" data-open="dsw059-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> Although, recent studies suggest that both classical NHEJ and MMEJ can coexist, this will be dependent on levels of protein expression. A synthesis-dependent microhomology-mediated end joining has been previously observed in <em>Drosophila</em>,<a reveal-id="dsw059-B45" data-open="dsw059-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a><sup>,</sup><a reveal-id="dsw059-B46" data-open="dsw059-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a> which is consistent with our present study.</p><p>In our study, we observed that unlike HR-mediated repair, which was very efficient at 30°C and above, MMEJ was optimal at 4 and 16°C. Although the relevance of this observation is unclear, it appears that this could be one way of promoting HR-mediated repair to ensure genomic stability. More studies are needed to decipher the importance of MMEJ in crustaceans, as it may facilitate evolution, since it could lead to changes in the DNA sequences.</p>                            <h3 scrollto-destination=78055481 id="78055481" class="section-title" >4.3. Does efficient repair protect crustaceans from infections?</h3>
<p>There is a direct connection between DSB repair and immunoglobulin diversity during adaptive immune response in mammals. NHEJ is critical during the rearrangement of V(D)J subexons.<a reveal-id="dsw059-B47" data-open="dsw059-B47" class="link link-ref link-reveal xref-bibr"><sup>47</sup></a><sup>,</sup><a reveal-id="dsw059-B48" data-open="dsw059-B48" class="link link-ref link-reveal xref-bibr"><sup>48</sup></a> Hence, NHEJ proteins such as KU complex and Ligase IV play a critical role in the generation of antibody diversity. Therefore, it will be interesting to evaluate, whether efficient repair system in crustaceans can protect these organisms from infections. Previous studies indicate that, DNA repair proteins may suppress viral infections, in higher organisms.<a reveal-id="dsw059-B49" data-open="dsw059-B49" class="link link-ref link-reveal xref-bibr"><sup>49</sup></a> Studies suggest that DNA repair proteins may inhibit and can have an effect on retrotransposons and viral infections in yeast and mammals.<a reveal-id="dsw059-B50" data-open="dsw059-B50" class="link link-ref link-reveal xref-bibr"><sup>50</sup></a><sup>,</sup><sup>–</sup><a reveal-id="dsw059-B52" data-open="dsw059-B52" class="link link-ref link-reveal xref-bibr"><sup>52</sup></a> Furthermore, both Rad52 and Rad18 have been reported to suppress HIV-1 infection.<a reveal-id="dsw059-B52" data-open="dsw059-B52" class="link link-ref link-reveal xref-bibr"><sup>52</sup></a> However, these aspects need to be further investigated in shrimp.</p><p>Choice of repair pathway takes place at several steps of DSB repair process. After initiation of processive end resection, NHEJ cannot take place,<a reveal-id="dsw059-B53" data-open="dsw059-B53" class="link link-ref link-reveal xref-bibr"><sup>53</sup></a> while a choice between HR and SSA (single-strand annealing), which is Rad51 dependent, takes place in <em>Drosophila</em>. P-element-induced DSBs were shown to be efficiently repaired by MMEJ mechanism in wild-type <em>Drosophila</em> and were unaffected in lig4 mutant flies.<a reveal-id="dsw059-B54" data-open="dsw059-B54" class="link link-ref link-reveal xref-bibr"><sup>54</sup></a></p><p>It will be very interesting and significant to understand whether these DNA repair machineries can protect crustaceans against infections. It is possible that certain mutations in the repair pathways make them susceptible to a particular infection. In other words, it will be important to look at the DNA repair genes of crustaceans following viral infection to determine the cause of heavy mortality due to some of the viruses.</p>                            <h2 scrollto-destination=78055485 id="78055485" class="section-title" >Authors’ contributions</h2>
<p>S.S., S.D., S.J.N., D.A., V.G., R.K.V. and S.C.R. conceived, designed and conducted the experiments, analysed the data and wrote the manuscript.</p>                            <h2 scrollto-destination=78055487 id="78055487" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank M. Nambiar, S. Vartak, R. Sebastian and SCR laboratory members for critical reading of the manuscript.</p>                            <h3 scrollto-destination=78055489 id="78055489" class="section-title" >Conflict of interest</h3>
<p>None declared.</p>                            <h2 scrollto-destination=78055491 id="78055491" class="backsection-title" >Funding</h2>
<p>Financial assistance from IISc-DBT partnership programme [DBT/BF/PR/INS/2011-12/IISc] for SCR is acknowledged. S.S. is supported by Dr D. S. Kothari Postdoctoral Fellowship from UGC, India and S.D. is supported by IISc fellowship, Bangalore (India).</p>                            <h2 scrollto-destination=78055493 id="78055493" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsw059-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw059-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Srivastava</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Raghavan</div> <div class="given-names">S.C.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">DNA double-strand break repair inhibitors as cancer therapeutics</div>. <div class="source">Chem. Biol</div> ., <div class="volume">22</div>, <div class="fpage">17</div>–<div class="lpage">29</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20double-strand%20break%20repair%20inhibitors%20as%20cancer%20therapeutics&amp;author=M.%20Srivastava&amp;author=S.C.%20Raghavan&amp;publication_year=2015&amp;journal=Chem.%20Biol&amp;volume=22&amp;pages=17-29" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.chembiol.2014.11.013" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.chembiol.2014.11.013" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.chembiol.2014.11.013"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25579208" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw059-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw059-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Friedberg</div> <div class="given-names">E.C.</div></div>, <div class="name string-name"><div class="surname">Aguilera</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Gellert</div> <div class="given-names">M.</div></div></div>, et al.  . <div class="year">2006</div>, 
<div class="article-title">DNA repair: from molecular mechanism to human disease</div>. <div class="source">DNA Repair</div> , <div class="volume">5</div>, <div class="fpage">986</div>–<div class="lpage">96</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20repair%3A%20from%20molecular%20mechanism%20to%20human%20disease&amp;author=E.C.%20Friedberg&amp;author=A.%20Aguilera&amp;author=M.%20Gellert&amp;publication_year=2006&amp;journal=DNA%20Repair&amp;volume=5&amp;pages=986-96" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.dnarep.2006.05.005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.dnarep.2006.05.005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.dnarep.2006.05.005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16955546" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw059-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw059-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Friedberg</div> <div class="given-names">E.C.</div></div>, <div class="name string-name"><div class="surname">Backendorf</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Burke</div> <div class="given-names">J.</div></div></div>, et al.  . <div class="year">1987</div>, 
<div class="article-title">Molecular aspects of DNA repair</div>. <div class="source">Mutation Res</div> , <div class="volume">184</div>, <div class="fpage">67</div>–<div class="lpage">86</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20aspects%20of%20DNA%20repair&amp;author=E.C.%20Friedberg&amp;author=C.%20Backendorf&amp;author=J.%20Burke&amp;publication_year=1987&amp;journal=Mutation%20Res&amp;volume=184&amp;pages=67-86" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0167-8817(87)90063-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0167-8817(87)90063-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0167-8817(87)90063-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3306357" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw059-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw059-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Nambiar</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Raghavan</div> <div class="given-names">S.C.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Chromosomal translocations among the healthy human population: implications in oncogenesis</div>. <div class="source">Cell. Mol. Life Sci</div> , <div class="volume">70</div>, <div class="fpage">1381</div>–<div class="lpage">92</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Chromosomal%20translocations%20among%20the%20healthy%20human%20population%3A%20implications%20in%20oncogenesis&amp;author=M.%20Nambiar&amp;author=S.C.%20Raghavan&amp;publication_year=2013&amp;journal=Cell.%20Mol.%20Life%20Sci&amp;volume=70&amp;pages=1381-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00018-012-1135-x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00018-012-1135-x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00018-012-1135-x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22948164" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw059-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw059-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sharma</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Raghavan</div> <div class="given-names">S.C.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Nonhomologous DNA end joining in cell-free extracts</div>. <div class="source">J. Nucleic Acids</div> .</div></div></div></div><div content-id="dsw059-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw059-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lieber</div> <div class="given-names">M.R.</div></div>, <div class="name string-name"><div class="surname">Yu</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Raghavan</div> <div class="given-names">S.C.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">Roles of nonhomologous DNA end joining, V(D)J recombination, and class switch recombination in chromosomal translocations</div>. <div class="source">DNA Repair</div> , <div class="volume">5</div>, <div class="fpage">1234</div>–<div class="lpage">45</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Roles%20of%20nonhomologous%20DNA%20end%20joining%2C%20V(D)J%20recombination%2C%20and%20class%20switch%20recombination%20in%20chromosomal%20translocations&amp;author=M.R.%20Lieber&amp;author=K.%20Yu&amp;author=S.C.%20Raghavan&amp;publication_year=2006&amp;journal=DNA%20Repair&amp;volume=5&amp;pages=1234-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.dnarep.2006.05.013" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.dnarep.2006.05.013" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.dnarep.2006.05.013"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16793349" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw059-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw059-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Iliakis</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Murmann</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Soni</div> <div class="given-names">A.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Alternative end-joining repair pathways are the ultimate backup for abrogated classical non-homologous end-joining and homologous recombination repair: implications for the formation of chromosome translocations</div>. <div class="source">Mutat. Res. Genet. Toxicol. Environ. Mutagen</div> ., <div class="volume">793</div>, <div class="fpage">166</div>–<div class="lpage">75</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Alternative%20end-joining%20repair%20pathways%20are%20the%20ultimate%20backup%20for%20abrogated%20classical%20non-homologous%20end-joining%20and%20homologous%20recombination%20repair%3A%20implications%20for%20the%20formation%20of%20chromosome%20translocations&amp;author=G.%20Iliakis&amp;author=T.%20Murmann&amp;author=A.%20Soni&amp;publication_year=2015&amp;journal=Mutat.%20Res.%20Genet.%20Toxicol.%20Environ.%20Mutagen&amp;volume=793&amp;pages=166-75" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.mrgentox.2015.07.001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.mrgentox.2015.07.001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.mrgentox.2015.07.001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26520387" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw059-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw059-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Nambiar</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Raghavan</div> <div class="given-names">S.C.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">How does DNA break during chromosomal translocations?</div> <div class="source">Nucleic Acids Res</div> ., <div class="volume">39</div>, <div class="fpage">5813</div>–<div class="lpage">25</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=How%20does%20DNA%20break%20during%20chromosomal%20translocations%3F&amp;author=M.%20Nambiar&amp;author=S.C.%20Raghavan&amp;publication_year=2011&amp;journal=Nucleic%20Acids%20Res&amp;volume=39&amp;pages=5813-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkr223" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr223" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkr223"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21498543" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw059-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw059-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Javadekar</div> <div class="given-names">S.M.</div></div>, <div class="name string-name"><div class="surname">Raghavan</div> <div class="given-names">S.C.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Snaps and mends: DNA breaks and chromosomal translocations</div>. <div class="source">FEBS J</div> ., <div class="volume">282</div>, <div class="fpage">2627</div>–<div class="lpage">45</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Snaps%20and%20mends%3A%20DNA%20breaks%20and%20chromosomal%20translocations&amp;author=S.M.%20Javadekar&amp;author=S.C.%20Raghavan&amp;publication_year=2015&amp;journal=FEBS%20J&amp;volume=282&amp;pages=2627-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/febs.2015.282.issue-14" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ffebs.2015.282.issue-14" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ffebs.2015.282.issue-14"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25913527" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw059-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw059-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wyman</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Kanaar</div> <div class="given-names">R.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">DNA double-strand break repair: all's well that ends well</div>. <div class="source">Annu. Rev. Genet</div> ., <div class="volume">40</div>, <div class="fpage">363</div>–<div class="lpage">83</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20double-strand%20break%20repair%3A%20all's%20well%20that%20ends%20well&amp;author=C.%20Wyman&amp;author=R.%20Kanaar&amp;publication_year=2006&amp;journal=Annu.%20Rev.%20Genet&amp;volume=40&amp;pages=363-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.genet.40.110405.090451" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.genet.40.110405.090451" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.genet.40.110405.090451"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16895466" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw059-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw059-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lieber</div> <div class="given-names">M.R.</div></div>, <div class="name string-name"><div class="surname">Raghavan</div> <div class="given-names">S.C.</div></div>, <div class="name string-name"><div class="surname">Yu</div> <div class="given-names">K.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">Mechanistic aspects of lymphoid chromosomal translocations</div>. <div class="source">J. Natl. Cancer Inst. Monogr</div> ., <div class="volume">39</div>, <div class="fpage">8</div>–<div class="lpage">11</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mechanistic%20aspects%20of%20lymphoid%20chromosomal%20translocations&amp;author=M.R.%20Lieber&amp;author=S.C.%20Raghavan&amp;author=K.%20Yu&amp;publication_year=2008&amp;journal=J.%20Natl.%20Cancer%20Inst.%20Monogr&amp;volume=39&amp;pages=8-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jncimonographs/lgn012" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjncimonographs%2Flgn012" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjncimonographs%2Flgn012"> </span></div> </div></div></div></div></div><div content-id="dsw059-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw059-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gostissa</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Alt</div> <div class="given-names">F.W.</div></div>, <div class="name string-name"><div class="surname">Chiarle</div> <div class="given-names">R.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Mechanisms that promote and suppress chromosomal translocations in lymphocytes</div>. <div class="source">Ann. Rev. Immunol</div> ., <div class="volume">29</div>, <div class="fpage">319</div>–<div class="lpage">50</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mechanisms%20that%20promote%20and%20suppress%20chromosomal%20translocations%20in%20lymphocytes&amp;author=M.%20Gostissa&amp;author=F.W.%20Alt&amp;author=R.%20Chiarle&amp;publication_year=2011&amp;journal=Ann.%20Rev.%20Immunol&amp;volume=29&amp;pages=319-50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev-immunol-031210-101329" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev-immunol-031210-101329" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev-immunol-031210-101329"> </span></div> </div></div></div></div></div><div content-id="dsw059-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw059-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lieber</div> <div class="given-names">M.R.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">NHEJ and its backup pathways in chromosomal translocations</div>. <div class="source">Nat. Struct. Mol. Biol</div> ., <div class="volume">17</div>, <div class="fpage">393</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=NHEJ%20and%20its%20backup%20pathways%20in%20chromosomal%20translocations&amp;author=M.R.%20Lieber&amp;publication_year=2010&amp;journal=Nat.%20Struct.%20Mol.%20Biol&amp;volume=17&amp;pages=393-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nsmb0410-393" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnsmb0410-393" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnsmb0410-393"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20368722" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw059-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw059-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chapman</div> <div class="given-names">J.R.</div></div>, <div class="name string-name"><div class="surname">Taylor</div> <div class="given-names">M.R.</div></div>, <div class="name string-name"><div class="surname">Boulton</div> <div class="given-names">S.J.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Playing the end game: DNA double-strand break repair pathway choice</div>. <div class="source">Mol. Cell</div> , <div class="volume">47</div>, <div class="fpage">497</div>–<div class="lpage">510</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Playing%20the%20end%20game%3A%20DNA%20double-strand%20break%20repair%20pathway%20choice&amp;author=J.R.%20Chapman&amp;author=M.R.%20Taylor&amp;author=S.J.%20Boulton&amp;publication_year=2012&amp;journal=Mol.%20Cell&amp;volume=47&amp;pages=497-510" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.molcel.2012.07.029" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.molcel.2012.07.029" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.molcel.2012.07.029"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22920291" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw059-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw059-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lieber</div> <div class="given-names">M.R.</div></div>, <div class="name string-name"><div class="surname">Ma</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Pannicke</div> <div class="given-names">U.</div></div>, <div class="name string-name"><div class="surname">Schwarz</div> <div class="given-names">K.</div></div></div> <div class="year">2003</div>, 
<div class="article-title">Mechanism and regulation of human non-homologous DNA end-joining</div>. <div class="source">Nat. Rev. Mol. Cell. Biol</div> ., <div class="volume">4</div>, <div class="fpage">712</div>–<div class="lpage">20</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mechanism%20and%20regulation%20of%20human%20non-homologous%20DNA%20end-joining&amp;author=M.R.%20Lieber&amp;author=Y.%20Ma&amp;author=U.%20Pannicke&amp;author=K.%20Schwarz&amp;publication_year=2003&amp;journal=Nat.%20Rev.%20Mol.%20Cell.%20Biol&amp;volume=4&amp;pages=712-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrm1202" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrm1202" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrm1202"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14506474" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw059-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw059-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ciccia</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Elledge</div> <div class="given-names">S.J.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">The DNA damage response: making it safe to play with knives</div>. <div class="source">Mol. Cell</div> , <div class="volume">40</div>, <div class="fpage">179</div>–<div class="lpage">204</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20DNA%20damage%20response%3A%20making%20it%20safe%20to%20play%20with%20knives&amp;author=A.%20Ciccia&amp;author=S.J.%20Elledge&amp;publication_year=2010&amp;journal=Mol.%20Cell&amp;volume=40&amp;pages=179-204" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.molcel.2010.09.019" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.molcel.2010.09.019" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.molcel.2010.09.019"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20965415" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw059-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw059-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Orthwein</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Fradet-Turcotte</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Noordermeer</div> <div class="given-names">S.M.</div></div></div>, et al.  . <div class="year">2014</div>, 
<div class="article-title">Mitosis inhibits DNA double-strand break repair to guard against telomere fusions</div>. <div class="source">Science</div> , <div class="volume">344</div>, <div class="fpage">189</div>–<div class="lpage">93</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mitosis%20inhibits%20DNA%20double-strand%20break%20repair%20to%20guard%20against%20telomere%20fusions&amp;author=A.%20Orthwein&amp;author=A.%20Fradet-Turcotte&amp;author=S.M.%20Noordermeer&amp;publication_year=2014&amp;journal=Science&amp;volume=344&amp;pages=189-93" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1248024" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1248024" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1248024"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24652939" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw059-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw059-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Simsek</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Jasin</div> <div class="given-names">M.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Alternative end-joining is suppressed by the canonical NHEJ component Xrcc4-ligase IV during chromosomal translocation formation</div>. <div class="source">Nat. Struct. Mol. Biol</div> ., <div class="volume">17</div>, <div class="fpage">410</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Alternative%20end-joining%20is%20suppressed%20by%20the%20canonical%20NHEJ%20component%20Xrcc4-ligase%20IV%20during%20chromosomal%20translocation%20formation&amp;author=D.%20Simsek&amp;author=M.%20Jasin&amp;publication_year=2010&amp;journal=Nat.%20Struct.%20Mol.%20Biol&amp;volume=17&amp;pages=410-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nsmb.1773" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnsmb.1773" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnsmb.1773"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20208544" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw059-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw059-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Iliakis</div> <div class="given-names">G.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Backup pathways of NHEJ in cells of higher eukaryotes: cell cycle dependence</div>. <div class="source">Radiother. Oncol</div> ., <div class="volume">92</div>, <div class="fpage">310</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Backup%20pathways%20of%20NHEJ%20in%20cells%20of%20higher%20eukaryotes%3A%20cell%20cycle%20dependence&amp;author=G.%20Iliakis&amp;publication_year=2009&amp;journal=Radiother.%20Oncol&amp;volume=92&amp;pages=310-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.radonc.2009.06.024" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.radonc.2009.06.024" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.radonc.2009.06.024"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19604590" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw059-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw059-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wu</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Mussfeldt</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Iliakis</div> <div class="given-names">G.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">Enhanced use of backup pathways of NHEJ in G2 in Chinese hamster mutant cells with defects in the classical pathway of NHEJ</div>. <div class="source">Radiat. Res</div> ., <div class="volume">170</div>, <div class="fpage">512</div>–<div class="lpage">20</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Enhanced%20use%20of%20backup%20pathways%20of%20NHEJ%20in%20G2%20in%20Chinese%20hamster%20mutant%20cells%20with%20defects%20in%20the%20classical%20pathway%20of%20NHEJ&amp;author=W.%20Wu&amp;author=M.%20Wang&amp;author=T.%20Mussfeldt&amp;author=G.%20Iliakis&amp;publication_year=2008&amp;journal=Radiat.%20Res&amp;volume=170&amp;pages=512-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1667/RR1456.1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1667%2FRR1456.1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1667%2FRR1456.1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19024658" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw059-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw059-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chiruvella</div> <div class="given-names">K.K.</div></div>, <div class="name string-name"><div class="surname">Sebastian</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Sharma</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Karande</div> <div class="given-names">A.A.</div></div>, <div class="name string-name"><div class="surname">Choudhary</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Raghavan</div> <div class="given-names">S.C.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Time-dependent predominance of nonhomologous DNA end-joining pathways during embryonic development in mice</div>. <div class="source">J. Mol. Biol</div> ., <div class="volume">417</div>, <div class="fpage">197</div>–<div class="lpage">211</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Time-dependent%20predominance%20of%20nonhomologous%20DNA%20end-joining%20pathways%20during%20embryonic%20development%20in%20mice&amp;author=K.K.%20Chiruvella&amp;author=R.%20Sebastian&amp;author=S.%20Sharma&amp;author=A.A.%20Karande&amp;author=B.%20Choudhary&amp;author=S.C.%20Raghavan&amp;publication_year=2012&amp;journal=J.%20Mol.%20Biol&amp;volume=417&amp;pages=197-211" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.jmb.2012.01.029" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.jmb.2012.01.029" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.jmb.2012.01.029"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22306462" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw059-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw059-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sharma</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Javadekar</div> <div class="given-names">S.M.</div></div>, <div class="name string-name"><div class="surname">Pandey</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Srivastava</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Kumari</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Raghavan</div> <div class="given-names">S.C.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Homology and enzymatic requirements of microhomology-dependent alternative end joining</div>. <div class="source">Cell Death Dis</div> ., <div class="volume">6</div>, <div class="fpage">e1697</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Homology%20and%20enzymatic%20requirements%20of%20microhomology-dependent%20alternative%20end%20joining&amp;author=S.%20Sharma&amp;author=S.M.%20Javadekar&amp;author=M.%20Pandey&amp;author=M.%20Srivastava&amp;author=R.%20Kumari&amp;author=S.C.%20Raghavan&amp;publication_year=2015&amp;journal=Cell%20Death%20Dis&amp;volume=6&amp;pages=e1697" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/cddis.2015.58" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fcddis.2015.58" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fcddis.2015.58"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25789972" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw059-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw059-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Stentiford</div> <div class="given-names">G.D.</div></div>, <div class="name string-name"><div class="surname">Neil</div> <div class="given-names">D.M.</div></div>, <div class="name string-name"><div class="surname">Peeler</div> <div class="given-names">E.J.</div></div></div>, et al.  . <div class="year">2012</div>, 
<div class="article-title">Disease will limit future food supply from the global crustacean fishery and aquaculture sectors</div>. <div class="source">J. Invertebr. Pathol</div> ., <div class="volume">110</div>, <div class="fpage">141</div>–<div class="lpage">57</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Disease%20will%20limit%20future%20food%20supply%20from%20the%20global%20crustacean%20fishery%20and%20aquaculture%20sectors&amp;author=G.D.%20Stentiford&amp;author=D.M.%20Neil&amp;author=E.J.%20Peeler&amp;publication_year=2012&amp;journal=J.%20Invertebr.%20Pathol&amp;volume=110&amp;pages=141-57" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.jip.2012.03.013" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.jip.2012.03.013" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.jip.2012.03.013"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22434002" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw059-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw059-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Flegel</div> <div class="given-names">T.W.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Historic emergence, impact and current status of shrimp pathogens in Asia</div> <div class="source">J. Invertebr. Pathol</div> ., <div class="volume">110</div>, <div class="fpage">166</div>–<div class="lpage">73</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Historic%20emergence%2C%20impact%20and%20current%20status%20of%20shrimp%20pathogens%20in%20Asia&amp;author=T.W.%20Flegel&amp;publication_year=2012&amp;journal=J.%20Invertebr.%20Pathol&amp;volume=110&amp;pages=166-73" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.jip.2012.03.004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.jip.2012.03.004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.jip.2012.03.004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22429834" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw059-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw059-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sudhakaran</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Okugawa</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Mekata</div> <div class="given-names">T.</div></div></div>, et al.  . <div class="year">2011</div>, 
<div class="article-title">Deciphering the DNA repair protein, Rad23 from kuruma shrimp <em>Marsupenaeus japonicus</em>: full-length cDNA cloning and characterization</div>. <div class="source">Lett. Appl. Microbiol</div> ., <div class="volume">53</div>, <div class="fpage">63</div>–<div class="lpage">72</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Deciphering%20the%20DNA%20repair%20protein%2C%20Rad23%20from%20kuruma%20shrimp%20Marsupenaeus%20japonicus%3A%20full-length%20cDNA%20cloning%20and%20characterization&amp;author=R.%20Sudhakaran&amp;author=S.%20Okugawa&amp;author=T.%20Mekata&amp;publication_year=2011&amp;journal=Lett.%20Appl.%20Microbiol&amp;volume=53&amp;pages=63-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/lam.2011.53.issue-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Flam.2011.53.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Flam.2011.53.issue-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21535050" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw059-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw059-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Rhee</div> <div class="given-names">J.S.</div></div>, <div class="name string-name"><div class="surname">Kim</div> <div class="given-names">B.M.</div></div>, <div class="name string-name"><div class="surname">Choi</div> <div class="given-names">B.S.</div></div>, <div class="name string-name"><div class="surname">Lee</div> <div class="given-names">J.S.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Expression pattern analysis of DNA repair-related and DNA damage response genes revealed by 55K oligomicroarray upon UV-B irradiation in the intertidal copepod, <em>Tigriopus japonicus</em></div>. <div class="source">Comp. Biochem. Physiol. C. Toxicol. Pharmacol</div> ., <div class="volume">155</div>, <div class="fpage">359</div>–<div class="lpage">68</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Expression%20pattern%20analysis%20of%20DNA%20repair-related%20and%20DNA%20damage%20response%20genes%20revealed%20by%2055K%20oligomicroarray%20upon%20UV-B%20irradiation%20in%20the%20intertidal%20copepod%2C%20Tigriopus%20japonicus&amp;author=J.S.%20Rhee&amp;author=B.M.%20Kim&amp;author=B.S.%20Choi&amp;author=J.S.%20Lee&amp;publication_year=2012&amp;journal=Comp.%20Biochem.%20Physiol.%20C.%20Toxicol.%20Pharmacol&amp;volume=155&amp;pages=359-68" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cbpc.2011.10.005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cbpc.2011.10.005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cbpc.2011.10.005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22051804" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw059-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw059-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Connelly</div> <div class="given-names">S.J.</div></div>, <div class="name string-name"><div class="surname">Moeller</div> <div class="given-names">R.E.</div></div>, <div class="name string-name"><div class="surname">Sanchez</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Mitchell</div> <div class="given-names">D.L.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Temperature effects on survival and DNA repair in four freshwater cladoceran Daphnia species exposed to UV radiation</div>. <div class="source">Photochem. Photobiol</div> ., <div class="volume">85</div>, <div class="fpage">144</div>–<div class="lpage">52</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Temperature%20effects%20on%20survival%20and%20DNA%20repair%20in%20four%20freshwater%20cladoceran%20Daphnia%20species%20exposed%20to%20UV%20radiation&amp;author=S.J.%20Connelly&amp;author=R.E.%20Moeller&amp;author=G.%20Sanchez&amp;author=D.L.%20Mitchell&amp;publication_year=2009&amp;journal=Photochem.%20Photobiol&amp;volume=85&amp;pages=144-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/php.2009.85.issue-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fphp.2009.85.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fphp.2009.85.issue-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18700864" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw059-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw059-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sekelsky</div> <div class="given-names">J.J.</div></div>, <div class="name string-name"><div class="surname">Brodsky</div> <div class="given-names">M.H.</div></div>, <div class="name string-name"><div class="surname">Burtis</div> <div class="given-names">K.C.</div></div></div> <div class="year">2000</div>, 
<div class="article-title">DNA repair in <em>Drosophila</em>: insights from the <em>Drosophila</em> genome sequence</div>. <div class="source">J. Cell Biol</div> ., <div class="volume">150</div>, <div class="fpage">F31</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20repair%20in%20Drosophila%3A%20insights%20from%20the%20Drosophila%20genome%20sequence&amp;author=J.J.%20Sekelsky&amp;author=M.H.%20Brodsky&amp;author=K.C.%20Burtis&amp;publication_year=2000&amp;journal=J.%20Cell%20Biol&amp;volume=150&amp;pages=F31-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1083/jcb.150.2.F31" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1083%2Fjcb.150.2.F31" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1083%2Fjcb.150.2.F31"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10908583" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw059-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw059-B29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Beall</div> <div class="given-names">E.L.</div></div>, <div class="name string-name"><div class="surname">Rio</div> <div class="given-names">D.C.</div></div></div> <div class="year">1996</div>, 
<div class="article-title"><em>Drosophila</em> IRBP/Ku p70 corresponds to the mutagen-sensitive mus309 gene and is involved in P-element excision <em>in vivo</em></div>. <div class="source">Genes Dev</div> ., <div class="volume">10</div>, <div class="fpage">921</div>–<div class="lpage">33</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Drosophila%20IRBP%2FKu%20p70%20corresponds%20to%20the%20mutagen-sensitive%20mus309%20gene%20and%20is%20involved%20in%20P-element%20excision%20in%20vivo&amp;author=E.L.%20Beall&amp;author=D.C.%20Rio&amp;publication_year=1996&amp;journal=Genes%20Dev&amp;volume=10&amp;pages=921-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gad.10.8.921" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgad.10.8.921" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgad.10.8.921"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8608940" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw059-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw059-B30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Deepika</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Sreedharan</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Paria</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Makesh</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Rajendran</div> <div class="given-names">K.V.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Toll-pathway in tiger shrimp (<em>Penaeus monodon</em>) responds to white spot syndrome virus infection: evidence through molecular characterisation and expression profiles of MyD88, TRAF6 and TLR genes</div>. <div class="source">Fish Shellfish Immunol</div> ., <div class="volume">41</div>, <div class="fpage">441</div>–<div class="lpage">54</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Toll-pathway%20in%20tiger%20shrimp%20(Penaeus%20monodon)%20responds%20to%20white%20spot%20syndrome%20virus%20infection%3A%20evidence%20through%20molecular%20characterisation%20and%20expression%20profiles%20of%20MyD88%2C%20TRAF6%20and%20TLR%20genes&amp;author=A.%20Deepika&amp;author=K.%20Sreedharan&amp;author=A.%20Paria&amp;author=M.%20Makesh&amp;author=K.V.%20Rajendran&amp;publication_year=2014&amp;journal=Fish%20Shellfish%20Immunol&amp;volume=41&amp;pages=441-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.fsi.2014.09.026" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.fsi.2014.09.026" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.fsi.2014.09.026"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25266891" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw059-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw059-B31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Nambiar</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Goldsmith</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Moorthy</div> <div class="given-names">B.T.</div></div></div> et al.  . <div class="year">2011</div>, 
<div class="article-title">Formation of a G-quadruplex at the BCL2 major breakpoint region of the t(14;18) translocation in follicular lymphoma</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">39</div>, <div class="fpage">936</div>–<div class="lpage">48</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Formation%20of%20a%20G-quadruplex%20at%20the%20BCL2%20major%20breakpoint%20region%20of%20the%20t(14%3B18)%20translocation%20in%20follicular%20lymphoma&amp;author=M.%20Nambiar&amp;author=G.%20Goldsmith&amp;author=B.T.%20Moorthy&amp;publication_year=2011&amp;journal=Nucleic%20Acids%20Res&amp;volume=39&amp;pages=936-48" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkq824" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkq824" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkq824"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20880994" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw059-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw059-B32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Nambiar</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Raghavan</div> <div class="given-names">S.C.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Mechanism of fragility at BCL2 gene minor breakpoint cluster region during t(14;18) chromosomal translocation</div>. <div class="source">J. Biol. Chem</div> ., <div class="volume">287</div>, <div class="fpage">8688</div>–<div class="lpage">701</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mechanism%20of%20fragility%20at%20BCL2%20gene%20minor%20breakpoint%20cluster%20region%20during%20t(14%3B18)%20chromosomal%20translocation&amp;author=M.%20Nambiar&amp;author=S.C.%20Raghavan&amp;publication_year=2012&amp;journal=J.%20Biol.%20Chem&amp;volume=287&amp;pages=8688-701" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.M111.307363" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.M111.307363" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.M111.307363"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22275374" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw059-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw059-B33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kumar</div> <div class="given-names">T.S.</div></div>, <div class="name string-name"><div class="surname">Kari</div> <div class="given-names">V.</div></div>, <div class="name string-name"><div class="surname">Choudhary</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Nambiar</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Akila</div> <div class="given-names">T.S.</div></div>, <div class="name string-name"><div class="surname">Raghavan</div> <div class="given-names">S.C.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Anti-apoptotic protein BCL2 down-regulates DNA end joining in cancer cells</div>. <div class="source">J. Biol. Chem</div> ., <div class="volume">285</div>, <div class="fpage">32657</div>–<div class="lpage">70</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Anti-apoptotic%20protein%20BCL2%20down-regulates%20DNA%20end%20joining%20in%20cancer%20cells&amp;author=T.S.%20Kumar&amp;author=V.%20Kari&amp;author=B.%20Choudhary&amp;author=M.%20Nambiar&amp;author=T.S.%20Akila&amp;author=S.C.%20Raghavan&amp;publication_year=2010&amp;journal=J.%20Biol.%20Chem&amp;volume=285&amp;pages=32657-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.M110.140350" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.M110.140350" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.M110.140350"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20699221" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw059-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw059-B34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tadi</div> <div class="given-names">S.K.</div></div>, <div class="name string-name"><div class="surname">Sebastian</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Dahal</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Babu</div> <div class="given-names">R.K.</div></div>, <div class="name string-name"><div class="surname">Choudhary</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Raghavan</div> <div class="given-names">S.C.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Microhomology-mediated end joining is the principal mediator of double-strand break repair during mitochondrial DNA lesions</div>. <div class="source">Mol. Biol. Cell</div> , <div class="volume">27</div>, <div class="fpage">223</div>–<div class="lpage">35</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microhomology-mediated%20end%20joining%20is%20the%20principal%20mediator%20of%20double-strand%20break%20repair%20during%20mitochondrial%20DNA%20lesions&amp;author=S.K.%20Tadi&amp;author=R.%20Sebastian&amp;author=S.%20Dahal&amp;author=R.K.%20Babu&amp;author=B.%20Choudhary&amp;author=S.C.%20Raghavan&amp;publication_year=2016&amp;journal=Mol.%20Biol.%20Cell&amp;volume=27&amp;pages=223-35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1091/mbc.E15-05-0260" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1091%2Fmbc.E15-05-0260" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1091%2Fmbc.E15-05-0260"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26609070" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw059-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw059-B35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sharma</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Choudhary</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Raghavan</div> <div class="given-names">S.C.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Efficiency of nonhomologous DNA end joining varies among somatic tissues, despite similarity in mechanism</div>. <div class="source">Cell. Mol. Life Sci</div> ., <div class="volume">68</div>, <div class="fpage">661</div>–<div class="lpage">76</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Efficiency%20of%20nonhomologous%20DNA%20end%20joining%20varies%20among%20somatic%20tissues%2C%20despite%20similarity%20in%20mechanism&amp;author=S.%20Sharma&amp;author=B.%20Choudhary&amp;author=S.C.%20Raghavan&amp;publication_year=2011&amp;journal=Cell.%20Mol.%20Life%20Sci&amp;volume=68&amp;pages=661-76" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00018-010-0472-x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00018-010-0472-x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00018-010-0472-x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20680388" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw059-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw059-B36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Baumann</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">West</div> <div class="given-names">S.C.</div></div></div> <div class="year">1998</div>, 
<div class="article-title">DNA end-joining catalyzed by human cell-free extracts</div>. <div class="source">Proc. Natl. Acad. Sci. U.S.A</div> ., <div class="volume">95</div>, <div class="fpage">14066</div>–<div class="lpage">70</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20end-joining%20catalyzed%20by%20human%20cell-free%20extracts&amp;author=P.%20Baumann&amp;author=S.C.%20West&amp;publication_year=1998&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A&amp;volume=95&amp;pages=14066-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.95.24.14066" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.95.24.14066" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.95.24.14066"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9826654" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw059-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw059-B37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sathees</div> <div class="given-names">C.R.</div></div>, <div class="name string-name"><div class="surname">Raman</div> <div class="given-names">M.J.</div></div></div> <div class="year">1999</div>, 
<div class="article-title">Mouse testicular extracts process DNA double-strand breaks efficiently by DNA end-to-end joining</div>. <div class="source">Mutat. Res</div> ., <div class="volume">433</div>, <div class="fpage">1</div>–<div class="lpage">13</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mouse%20testicular%20extracts%20process%20DNA%20double-strand%20breaks%20efficiently%20by%20DNA%20end-to-end%20joining&amp;author=C.R.%20Sathees&amp;author=M.J.%20Raman&amp;publication_year=1999&amp;journal=Mutat.%20Res&amp;volume=433&amp;pages=1-13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0921-8777(98)00055-X" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0921-8777(98)00055-X" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0921-8777(98)00055-X"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10047774" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw059-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw059-B38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Raghavan</div> <div class="given-names">S.C.</div></div>, <div class="name string-name"><div class="surname">Raman</div> <div class="given-names">M.J.</div></div></div> <div class="year">2004</div>, 
<div class="article-title">Nonhomologous end joining of complementary and noncomplementary DNA termini in mouse testicular extracts</div>. <div class="source">DNA Repair</div> , <div class="volume">3</div>, <div class="fpage">1297</div>–<div class="lpage">310</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nonhomologous%20end%20joining%20of%20complementary%20and%20noncomplementary%20DNA%20termini%20in%20mouse%20testicular%20extracts&amp;author=S.C.%20Raghavan&amp;author=M.J.%20Raman&amp;publication_year=2004&amp;journal=DNA%20Repair&amp;volume=3&amp;pages=1297-310" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.dnarep.2004.04.007" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.dnarep.2004.04.007" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.dnarep.2004.04.007"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15336625" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw059-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw059-B39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Srivastava</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Nambiar</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Sharma</div> <div class="given-names">S.</div></div></div> et al.  . <div class="year">2012</div>, 
<div class="article-title">An inhibitor of nonhomologous end-joining abrogates double-strand break repair and impedes cancer progression</div>. <div class="source">Cell</div> , <div class="volume">151</div>, <div class="fpage">1474</div>–<div class="lpage">87</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20inhibitor%20of%20nonhomologous%20end-joining%20abrogates%20double-strand%20break%20repair%20and%20impedes%20cancer%20progression&amp;author=M.%20Srivastava&amp;author=M.%20Nambiar&amp;author=S.%20Sharma&amp;publication_year=2012&amp;journal=Cell&amp;volume=151&amp;pages=1474-87" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2012.11.054" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2012.11.054" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2012.11.054"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23260137" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw059-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw059-B40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Oppliger</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Wurgler</div> <div class="given-names">F.E.</div></div>, <div class="name string-name"><div class="surname">Sengstag</div> <div class="given-names">C.</div></div></div> <div class="year">1993</div>, 
<div class="article-title">A plasmid system to monitor gene conversion and reciprocal recombination <em>in vitro</em></div>. <div class="source">Mutation Res</div> ., <div class="volume">291</div>, <div class="fpage">181</div>–<div class="lpage">92</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20plasmid%20system%20to%20monitor%20gene%20conversion%20and%20reciprocal%20recombination%20in%20vitro&amp;author=T.%20Oppliger&amp;author=F.E.%20Wurgler&amp;author=C.%20Sengstag&amp;publication_year=1993&amp;journal=Mutation%20Res&amp;volume=291&amp;pages=181-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0165-1161(93)90158-V" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0165-1161(93)90158-V" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0165-1161(93)90158-V"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7685059" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw059-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw059-B41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Srivastava</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Raman</div> <div class="given-names">M.J.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">Homologous recombination-mediated double-strand break repair in mouse testicular extracts and comparison with different germ cell stages</div>. <div class="source">Cell Biochem. Funct</div> ., <div class="volume">25</div>, <div class="fpage">75</div>–<div class="lpage">86</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Homologous%20recombination-mediated%20double-strand%20break%20repair%20in%20mouse%20testicular%20extracts%20and%20comparison%20with%20different%20germ%20cell%20stages&amp;author=N.%20Srivastava&amp;author=M.J.%20Raman&amp;publication_year=2007&amp;journal=Cell%20Biochem.%20Funct&amp;volume=25&amp;pages=75-86" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1099-0844" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1099-0844" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1099-0844"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16989005" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw059-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw059-B42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Do</div> <div class="given-names">A.T.</div></div>, <div class="name string-name"><div class="surname">Brooks</div> <div class="given-names">J.T.</div></div>, <div class="name string-name"><div class="surname">Le Neveu</div> <div class="given-names">M.K.</div></div>, <div class="name string-name"><div class="surname">LaRocque</div> <div class="given-names">J.R.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Double-strand break repair assays determine pathway choice and structure of gene conversion events in <em>Drosophila melanogaster</em></div>. <div class="source">G3 (Bethesda)</div> , <div class="volume">4</div>, <div class="fpage">425</div>–<div class="lpage">32</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Double-strand%20break%20repair%20assays%20determine%20pathway%20choice%20and%20structure%20of%20gene%20conversion%20events%20in%20Drosophila%20melanogaster&amp;author=A.T.%20Do&amp;author=J.T.%20Brooks&amp;author=M.K.%20Le%20Neveu&amp;author=J.R.%20LaRocque&amp;publication_year=2014&amp;journal=G3%20(Bethesda)&amp;volume=4&amp;pages=425-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/g3.113.010074" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fg3.113.010074" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fg3.113.010074"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24368780" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw059-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw059-B43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">McKee</div> <div class="given-names">B.D.</div></div>, <div class="name string-name"><div class="surname">Ren</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Hong</div> <div class="given-names">C.H.</div></div></div> <div class="year">1996</div>, 
<div class="article-title">A recA-like gene in <em>Drosophila melanogaster</em> that is expressed at high levels in female but not male meiotic tissues</div>. <div class="source">Chromosoma</div> , <div class="volume">104</div>, <div class="fpage">479</div>–<div class="lpage">488</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20recA-like%20gene%20in%20Drosophila%20melanogaster%20that%20is%20expressed%20at%20high%20levels%20in%20female%20but%20not%20male%20meiotic%20tissues&amp;author=B.D.%20McKee&amp;author=X.%20Ren&amp;author=C.H.%20Hong&amp;publication_year=1996&amp;journal=Chromosoma&amp;volume=104&amp;pages=479-488" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8625736" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw059-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw059-B44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ghabrial</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Ray</div> <div class="given-names">R.P.</div></div>, <div class="name string-name"><div class="surname">Schupbach</div> <div class="given-names">T.</div></div></div> <div class="year">1998</div>, 
<div class="article-title">okra and spindle-B encode components of the RAD52 DNA repair pathway and affect meiosis and patterning in <em>Drosophila oogenesis</em></div>. <div class="source">Genes Dev</div> ., <div class="volume">12</div>, <div class="fpage">2711</div>–<div class="lpage">23</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=okra%20and%20spindle-B%20encode%20components%20of%20the%20RAD52%20DNA%20repair%20pathway%20and%20affect%20meiosis%20and%20patterning%20in%20Drosophila%20oogenesis&amp;author=A.%20Ghabrial&amp;author=R.P.%20Ray&amp;author=T.%20Schupbach&amp;publication_year=1998&amp;journal=Genes%20Dev&amp;volume=12&amp;pages=2711-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gad.12.17.2711" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgad.12.17.2711" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgad.12.17.2711"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9732269" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw059-B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw059-B45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chan</div> <div class="given-names">S.H.</div></div>, <div class="name string-name"><div class="surname">Yu</div> <div class="given-names">A.M.</div></div>, <div class="name string-name"><div class="surname">McVey</div> <div class="given-names">M.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Dual roles for DNA polymerase theta in alternative end-joining repair of double-strand breaks in <em>Drosophila</em></div>. <div class="source">PLoS Genet</div> ., <div class="volume">6</div>, <div class="fpage">e1001005</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dual%20roles%20for%20DNA%20polymerase%20theta%20in%20alternative%20end-joining%20repair%20of%20double-strand%20breaks%20in%20Drosophila&amp;author=S.H.%20Chan&amp;author=A.M.%20Yu&amp;author=M.%20McVey&amp;publication_year=2010&amp;journal=PLoS%20Genet&amp;volume=6&amp;pages=e1001005" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1001005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1001005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1001005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20617203" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw059-B46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw059-B46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yu</div> <div class="given-names">A.M.</div></div>, <div class="name string-name"><div class="surname">McVey</div> <div class="given-names">M.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Synthesis-dependent microhomology-mediated end joining accounts for multiple types of repair junctions</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">38</div>, <div class="fpage">5706</div>–<div class="lpage">17</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Synthesis-dependent%20microhomology-mediated%20end%20joining%20accounts%20for%20multiple%20types%20of%20repair%20junctions&amp;author=A.M.%20Yu&amp;author=M.%20McVey&amp;publication_year=2010&amp;journal=Nucleic%20Acids%20Res&amp;volume=38&amp;pages=5706-17" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkq379" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkq379" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkq379"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20460465" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw059-B47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw059-B47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Nishana</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Raghavan</div> <div class="given-names">S.C.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Role of recombination activating genes in the generation of antigen receptor diversity and beyond</div>. <div class="source">Immunology</div> , <div class="volume">137</div>, <div class="fpage">271</div>–<div class="lpage">81</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Role%20of%20recombination%20activating%20genes%20in%20the%20generation%20of%20antigen%20receptor%20diversity%20and%20beyond&amp;author=M.%20Nishana&amp;author=S.C.%20Raghavan&amp;publication_year=2012&amp;journal=Immunology&amp;volume=137&amp;pages=271-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/imm.2012.137.issue-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fimm.2012.137.issue-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fimm.2012.137.issue-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23039142" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw059-B48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw059-B48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Nambiar</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Kari</div> <div class="given-names">V.</div></div>, <div class="name string-name"><div class="surname">Raghavan</div> <div class="given-names">S.C.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">Chromosomal translocations in cancer</div>. <div class="source">Biochim. Biophys. Acta</div> , <div class="volume">1786</div>, <div class="fpage">139</div>–<div class="lpage">52</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Chromosomal%20translocations%20in%20cancer&amp;author=M.%20Nambiar&amp;author=V.%20Kari&amp;author=S.C.%20Raghavan&amp;publication_year=2008&amp;journal=Biochim.%20Biophys.%20Acta&amp;volume=1786&amp;pages=139-52" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18718509" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw059-B49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw059-B49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zhao</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Hou</div> <div class="given-names">N.B.</div></div>, <div class="name string-name"><div class="surname">Song</div> <div class="given-names">T.</div></div></div> et al.  . <div class="year">2008</div>, 
<div class="article-title">Cellular DNA repair cofactors affecting hepatitis B virus infection and replication</div>. <div class="source">World J. Gastroenterol</div> ., <div class="volume">14</div>, <div class="fpage">5059</div>–<div class="lpage">65</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cellular%20DNA%20repair%20cofactors%20affecting%20hepatitis%20B%20virus%20infection%20and%20replication&amp;author=F.%20Zhao&amp;author=N.B.%20Hou&amp;author=T.%20Song&amp;publication_year=2008&amp;journal=World%20J.%20Gastroenterol&amp;volume=14&amp;pages=5059-65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3748/wjg.14.5059" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3748%2Fwjg.14.5059" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3748%2Fwjg.14.5059"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18763290" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw059-B50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw059-B50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Scholes</div> <div class="given-names">D.T.</div></div>, <div class="name string-name"><div class="surname">Banerjee</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Bowen</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Curcio</div> <div class="given-names">M.J.</div></div></div> <div class="year">2001</div>, 
<div class="article-title">Multiple regulators of Ty1 transposition in Saccharomyces cerevisiae have conserved roles in genome maintenance</div>. <div class="source">Genetics</div> , <div class="volume">159</div>, <div class="fpage">1449</div>–<div class="lpage">65</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Multiple%20regulators%20of%20Ty1%20transposition%20in%20Saccharomyces%20cerevisiae%20have%20conserved%20roles%20in%20genome%20maintenance&amp;author=D.T.%20Scholes&amp;author=M.%20Banerjee&amp;author=B.%20Bowen&amp;author=M.J.%20Curcio&amp;publication_year=2001&amp;journal=Genetics&amp;volume=159&amp;pages=1449-65" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11779788" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw059-B51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw059-B51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Irwin</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Aye</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Baldi</div> <div class="given-names">P.</div></div></div>, et al.  . <div class="year">2005</div>, 
<div class="article-title">Retroviruses and yeast retrotransposons use overlapping sets of host genes</div>. <div class="source">Genome Res</div> ., <div class="volume">15</div>, <div class="fpage">641</div>–<div class="lpage">54</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Retroviruses%20and%20yeast%20retrotransposons%20use%20overlapping%20sets%20of%20host%20genes&amp;author=B.%20Irwin&amp;author=M.%20Aye&amp;author=P.%20Baldi&amp;publication_year=2005&amp;journal=Genome%20Res&amp;volume=15&amp;pages=641-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.3739005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.3739005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.3739005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15837808" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw059-B52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw059-B52" class="ref-content "><div class="label -label">52</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yoder</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Sarasin</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Kraemer</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">McIlhatton</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Bushman</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Fishel</div> <div class="given-names">R.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">The DNA repair genes XPB and XPD defend cells from retroviral infection</div>. <div class="source">Proc. Natl. Acad. Sci. U.S.A</div> ., <div class="volume">103</div>, <div class="fpage">4622</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20DNA%20repair%20genes%20XPB%20and%20XPD%20defend%20cells%20from%20retroviral%20infection&amp;author=K.%20Yoder&amp;author=A.%20Sarasin&amp;author=K.%20Kraemer&amp;author=M.%20McIlhatton&amp;author=F.%20Bushman&amp;author=R.%20Fishel&amp;publication_year=2006&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A&amp;volume=103&amp;pages=4622-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0509828103" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0509828103" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0509828103"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16537383" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw059-B53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw059-B53" class="ref-content "><div class="label -label">53</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Symington</div> <div class="given-names">L.S.</div></div>, <div class="name string-name"><div class="surname">Gautier</div> <div class="given-names">J.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Double-strand break end resection and repair pathway choice</div>. <div class="source">Annu. Rev. Genet</div> ., <div class="volume">45</div>, <div class="fpage">247</div>–<div class="lpage">71</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Double-strand%20break%20end%20resection%20and%20repair%20pathway%20choice&amp;author=L.S.%20Symington&amp;author=J.%20Gautier&amp;publication_year=2011&amp;journal=Annu.%20Rev.%20Genet&amp;volume=45&amp;pages=247-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev-genet-110410-132435" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev-genet-110410-132435" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev-genet-110410-132435"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21910633" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw059-B54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw059-B54" class="ref-content "><div class="label -label">54</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">McVey</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Radut</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Sekelsky</div> <div class="given-names">J.J.</div></div></div> <div class="year">2004</div>, 
<div class="article-title">End-joining repair of double-strand breaks in Drosophila melanogaster is largely DNA ligase IV independent</div>. <div class="source">Genetics</div> , <div class="volume">168</div>, <div class="fpage">2067</div>–<div class="lpage">76</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=End-joining%20repair%20of%20double-strand%20breaks%20in%20Drosophila%20melanogaster%20is%20largely%20DNA%20ligase%20IV%20independent&amp;author=M.%20McVey&amp;author=D.%20Radut&amp;author=J.J.%20Sekelsky&amp;publication_year=2004&amp;journal=Genetics&amp;volume=168&amp;pages=2067-76" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.104.033902" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.104.033902" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.104.033902"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15611176" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2017. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=2949521&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">572</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
0                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsw059"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="2949521"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f24%2f2%2f117%2f2949521">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F24%2F2%2F117%2F2949521" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/2949521'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="DNA double-strand break repair in Penaeus monodon is predominantly dependent on homologous recombination" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsw059");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f24%2f2%2f117%2f2949521">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

