

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/25/1/39/4103403","@type":"ScholarlyArticle","name":"Multiple hybrid  de novo  genome assembly of finger millet, an orphan allotetraploid crop","datePublished":"2017-09-05","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/25/1","@type":"PublicationIssue","issueNumber":"1","datePublished":"2018-02-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsx036","keywords":["hybrid <italic>de novo</italic> assembly","finger millet","allotetraploid genome","whole-genome optical mapping"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/25/1/39/4103403","author":[{"name":"Hatakeyama, Masaomi","affiliation":"Department of Evolutionary Biology and Environmental Studies, University of Zurich, Winterthurerstrasse. 190, 8057 Zurich, Switzerland  Functional Genomics Center Zurich, ETH Zurich/University of Zurich, 8057 Zurich, Switzerland  Swiss Institute of Bioinformatics, Quartier Sorge - Batiment Genopode, 1015 Lausanne, Switzerland","@type":"Person"},{"name":"Aluri, Sirisha","affiliation":"Functional Genomics Center Zurich, ETH Zurich/University of Zurich, 8057 Zurich, Switzerland","@type":"Person"},{"name":"Balachadran, Mathi Thumilan","affiliation":"Department of Evolutionary Biology and Environmental Studies, University of Zurich, Winterthurerstrasse. 190, 8057 Zurich, Switzerland  Functional Genomics Center Zurich, ETH Zurich/University of Zurich, 8057 Zurich, Switzerland  Department of Crop Physiology, University of Agricultural Sciences, GKVK, Bangalore 560065, India","@type":"Person"},{"name":"Sivarajan, Sajeevan Radha","affiliation":"Department of Evolutionary Biology and Environmental Studies, University of Zurich, Winterthurerstrasse. 190, 8057 Zurich, Switzerland  Functional Genomics Center Zurich, ETH Zurich/University of Zurich, 8057 Zurich, Switzerland  Department of Crop Physiology, University of Agricultural Sciences, GKVK, Bangalore 560065, India","@type":"Person"},{"name":"Patrignani, Andrea","affiliation":"Functional Genomics Center Zurich, ETH Zurich/University of Zurich, 8057 Zurich, Switzerland","@type":"Person"},{"name":"Grüter, Simon","affiliation":"Functional Genomics Center Zurich, ETH Zurich/University of Zurich, 8057 Zurich, Switzerland","@type":"Person"},{"name":"Poveda, Lucy","affiliation":"Functional Genomics Center Zurich, ETH Zurich/University of Zurich, 8057 Zurich, Switzerland","@type":"Person"},{"name":"Shimizu-Inatsugi, Rie","affiliation":"Department of Evolutionary Biology and Environmental Studies, University of Zurich, Winterthurerstrasse. 190, 8057 Zurich, Switzerland","@type":"Person"},{"name":"Baeten, John","affiliation":"BioNano Genomics, San Diego, CA 92121, USA","@type":"Person"},{"name":"Francoijs, Kees-Jan","affiliation":"BioNano Genomics, San Diego, CA 92121, USA","@type":"Person"},{"name":"Nataraja, Karaba N","affiliation":"Department of Crop Physiology, University of Agricultural Sciences, GKVK, Bangalore 560065, India","@type":"Person"},{"name":"Reddy, Yellodu A Nanja","affiliation":"AICRP (Small Millets), ICAR-UAS, GKVK, Bangalore 560065, India","@type":"Person"},{"name":"Phadnis, Shamprasad","affiliation":"Department of Plant Biotechnology, University of Agricultural Sciences, GKVK, Bangalore 560065, India","@type":"Person"},{"name":"Ravikumar, Ramapura L","affiliation":"Department of Plant Biotechnology, University of Agricultural Sciences, GKVK, Bangalore 560065, India","@type":"Person"},{"name":"Schlapbach, Ralph","affiliation":"Functional Genomics Center Zurich, ETH Zurich/University of Zurich, 8057 Zurich, Switzerland","@type":"Person"},{"name":"Sreeman, Sheshshayee M","affiliation":"Department of Crop Physiology, University of Agricultural Sciences, GKVK, Bangalore 560065, India","@type":"Person"},{"name":"Shimizu, Kentaro K","affiliation":"Department of Evolutionary Biology and Environmental Studies, University of Zurich, Winterthurerstrasse. 190, 8057 Zurich, Switzerland  Kihara Institute for Biological Research, Yokohama City University, Yokohama, Japan","@type":"Person"}],"description":"Abstract.  Finger millet (Eleusine coracana (L.) Gaertn) is an important crop for food security because of its tolerance to drought, which is expected to be exa","pageStart":"39","pageEnd":"47","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/1/3/m_dnares_25_1cover.png?Expires=2147483647&Signature=SO6YCeydUh5yJs5n-IY9wmy~LoBoAx4P6Fvm-AjuGlXgQjk9YpdSZprn-w-uvSGcgqbsut967tdIfiY3dSqjk-kYgFErDTIMFh~0aK8LiNrwVszpCD29Va86wemvVduCLQhn8b79YN7Bcvi7qFB5E0wUNp-cxt7rNByaE14ePnoDwCgR5G58Pc5wx03IVhK9jft2n0g0lvAsEdQaWr3LG7PN4prc1KFhItNPOxm0fdg7HmXK23iCiP1EUMnFfqTR655CMN~bUB0aDDAZUT18HRZXXdjkjcMUCT~fjmwOlkGzU~71oDZYn-Ji-8-K8iuA-savjer0tQoSi784gQ8L6w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Multiple hybrid  de novo  genome assembly of finger millet, an orphan allotetraploid crop","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/1/3/m_dnares_25_1cover.png?Expires=2147483647&Signature=SO6YCeydUh5yJs5n-IY9wmy~LoBoAx4P6Fvm-AjuGlXgQjk9YpdSZprn-w-uvSGcgqbsut967tdIfiY3dSqjk-kYgFErDTIMFh~0aK8LiNrwVszpCD29Va86wemvVduCLQhn8b79YN7Bcvi7qFB5E0wUNp-cxt7rNByaE14ePnoDwCgR5G58Pc5wx03IVhK9jft2n0g0lvAsEdQaWr3LG7PN4prc1KFhItNPOxm0fdg7HmXK23iCiP1EUMnFfqTR655CMN~bUB0aDDAZUT18HRZXXdjkjcMUCT~fjmwOlkGzU~71oDZYn-Ji-8-K8iuA-savjer0tQoSi784gQ8L6w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Multiple hybrid  de novo  genome assembly of finger millet, an orphan allotetraploid crop" />
    <meta property="og:description" content="Abstract.  Finger millet (Eleusine coracana (L.) Gaertn) is an important crop for food security because of its tolerance to drought, which is expected to be exa" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/25/1/39/4103403" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2017-09-05" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Hatakeyama, Masaomi" /><meta name="citation_author_institution" content="Department of Evolutionary Biology and Environmental Studies, University of Zurich, Winterthurerstrasse. 190, 8057 Zurich, Switzerland" /><meta name="citation_author_institution" content="Functional Genomics Center Zurich, ETH Zurich/University of Zurich, 8057 Zurich, Switzerland" /><meta name="citation_author_institution" content="Swiss Institute of Bioinformatics, Quartier Sorge - Batiment Genopode, 1015 Lausanne, Switzerland" /><meta name="citation_author" content="Aluri, Sirisha" /><meta name="citation_author_institution" content="Functional Genomics Center Zurich, ETH Zurich/University of Zurich, 8057 Zurich, Switzerland" /><meta name="citation_author" content="Balachadran, Mathi Thumilan" /><meta name="citation_author_institution" content="Department of Evolutionary Biology and Environmental Studies, University of Zurich, Winterthurerstrasse. 190, 8057 Zurich, Switzerland" /><meta name="citation_author_institution" content="Functional Genomics Center Zurich, ETH Zurich/University of Zurich, 8057 Zurich, Switzerland" /><meta name="citation_author_institution" content="Department of Crop Physiology, University of Agricultural Sciences, GKVK, Bangalore 560065, India" /><meta name="citation_author" content="Sivarajan, Sajeevan Radha" /><meta name="citation_author_institution" content="Department of Evolutionary Biology and Environmental Studies, University of Zurich, Winterthurerstrasse. 190, 8057 Zurich, Switzerland" /><meta name="citation_author_institution" content="Functional Genomics Center Zurich, ETH Zurich/University of Zurich, 8057 Zurich, Switzerland" /><meta name="citation_author_institution" content="Department of Crop Physiology, University of Agricultural Sciences, GKVK, Bangalore 560065, India" /><meta name="citation_author" content="Patrignani, Andrea" /><meta name="citation_author_institution" content="Functional Genomics Center Zurich, ETH Zurich/University of Zurich, 8057 Zurich, Switzerland" /><meta name="citation_author" content="Grüter, Simon" /><meta name="citation_author_institution" content="Functional Genomics Center Zurich, ETH Zurich/University of Zurich, 8057 Zurich, Switzerland" /><meta name="citation_author" content="Poveda, Lucy" /><meta name="citation_author_institution" content="Functional Genomics Center Zurich, ETH Zurich/University of Zurich, 8057 Zurich, Switzerland" /><meta name="citation_author" content="Shimizu-Inatsugi, Rie" /><meta name="citation_author_institution" content="Department of Evolutionary Biology and Environmental Studies, University of Zurich, Winterthurerstrasse. 190, 8057 Zurich, Switzerland" /><meta name="citation_author" content="Baeten, John" /><meta name="citation_author_institution" content="BioNano Genomics, San Diego, CA 92121, USA" /><meta name="citation_author" content="Francoijs, Kees-Jan" /><meta name="citation_author_institution" content="BioNano Genomics, San Diego, CA 92121, USA" /><meta name="citation_author" content="Nataraja, Karaba N" /><meta name="citation_author_institution" content="Department of Crop Physiology, University of Agricultural Sciences, GKVK, Bangalore 560065, India" /><meta name="citation_author" content="Reddy, Yellodu A Nanja" /><meta name="citation_author_institution" content="AICRP (Small Millets), ICAR-UAS, GKVK, Bangalore 560065, India" /><meta name="citation_author" content="Phadnis, Shamprasad" /><meta name="citation_author_institution" content="Department of Plant Biotechnology, University of Agricultural Sciences, GKVK, Bangalore 560065, India" /><meta name="citation_author" content="Ravikumar, Ramapura L" /><meta name="citation_author_institution" content="Department of Plant Biotechnology, University of Agricultural Sciences, GKVK, Bangalore 560065, India" /><meta name="citation_author" content="Schlapbach, Ralph" /><meta name="citation_author_institution" content="Functional Genomics Center Zurich, ETH Zurich/University of Zurich, 8057 Zurich, Switzerland" /><meta name="citation_author" content="Sreeman, Sheshshayee M" /><meta name="citation_author_institution" content="Department of Crop Physiology, University of Agricultural Sciences, GKVK, Bangalore 560065, India" /><meta name="citation_author" content="Shimizu, Kentaro K" /><meta name="citation_author_institution" content="Department of Evolutionary Biology and Environmental Studies, University of Zurich, Winterthurerstrasse. 190, 8057 Zurich, Switzerland" /><meta name="citation_author_institution" content="Kihara Institute for Biological Research, Yokohama City University, Yokohama, Japan" /><meta name="citation_title" content="Multiple hybrid de novo genome assembly of finger millet, an orphan allotetraploid crop" /><meta name="citation_firstpage" content="39" /><meta name="citation_lastpage" content="47" /><meta name="citation_doi" content="10.1093/dnares/dsx036" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="25" /><meta name="citation_issue" content="1" /><meta name="citation_publication_date" content="2018/02/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Allotetraploid origin and divergence in Eleusine (Chloridoideae, Poaceae): evidence from low-copy nuclear gene phylogenies and a plastid gene chronogram;  citation_journal_title=Ann. Bot;  citation_year=2011;  citation_volume=108;  citation_pages=1287-98; " /><meta name="citation_reference" content="citation_title=Low-copy nuclear gene and McGISH resolves polyploid history of Eleusine coracana and morphological character evolution in Eleusine;  citation_journal_title=Turkish J. Bot;  citation_year=2014;  citation_volume=38;  citation_pages=1-12; " /><meta name="citation_reference" content="citation_title=Genomic in situ hybridization identifies genome donor of finger millet (Eleusine coracana);  citation_journal_title=Theor. Appl. Genet;  citation_year=2001;  citation_volume=102;  citation_pages=825-32; " /><meta name="citation_reference" content="citation_title=Identification of the “A” genome of finger millet using chloroplast DNA;  citation_journal_title=Genetics;  citation_year=1988;  citation_volume=118;  citation_pages=163-7; " /><meta name="citation_reference" content="citation_title=Review of finger millet (Eleusine coracana (L.) Gaertn): a power house of health benefiting nutrients;  citation_journal_title=Food Sci. Human Wellness;  citation_year=2016;  citation_volume=5;  citation_pages=149-55; " /><meta name="citation_reference" content="citation_title=Millet grains: nutritional quality, processing, and potential health benefits;  citation_journal_title=Compr. Rev. Food Sci. Food Safety;  citation_year=2013;  citation_volume=12;  citation_pages=281-95; " /><meta name="citation_reference" content="citation_title=Genetic diversity and genomic resources available for the small millet crops to accelerate a New Green Revolution;  citation_journal_title=Front. Plant Sci;  citation_year=2015;  citation_volume=6;  citation_pages=157" /><meta name="citation_reference" content="citation_title=Bioinformatics in the orphan crops;  citation_journal_title=Brief. Bioinform;  citation_year=2009;  citation_volume=10;  citation_pages=645-53; " /><meta name="citation_reference" content="citation_title=Nuclear DNA content in species of Eleusine (Gramineae): a critical re-evaluation using laser flow cytometry;  citation_journal_title=Plant Syst. Evol;  citation_year=1997;  citation_volume=207;  citation_pages=1-11; " /><meta name="citation_reference" content="citation_title=Expression of a finger millet transcription factor, EcNAC1, in tobacco confers abiotic stress-tolerance;  citation_journal_title=PLoS One;  citation_year=2012;  citation_volume=7;  citation_pages=e40397" /><meta name="citation_reference" content="citation_title=Expression analysis of stress responsive pathway genes linked to drought hardiness in an adapted crop, finger millet (Eleusine coracana);  citation_journal_title=J. Plant Biochem. Biotechnol;  citation_year=2013;  citation_volume=22;  citation_pages=193-201; " /><meta name="citation_reference" content="citation_title=Full-length cloning and characterization of abiotic stress responsive CIPK31-like gene from finger millet, a drought-tolerant crop;  citation_journal_title=Curr. Sci;  citation_year=2016;  citation_volume=111;  citation_pages=890" /><meta name="citation_reference" content="citation_title=Transcriptome analysis of salinity responsiveness in contrasting genotypes of finger millet (Eleusine coracana L.) through RNA-sequencing;  citation_journal_title=Plant Mol. Biol;  citation_year=2014;  citation_volume=85;  citation_pages=485-503; " /><meta name="citation_reference" content="citation_title=The genetic map of finger millet, Eleusine coracana;  citation_journal_title=Theor. Appl. Genet;  citation_year=2006;  citation_volume=114;  citation_pages=321-32; " /><meta name="citation_reference" content="citation_title=The frequency of polyploid speciation in vascular plants;  citation_journal_title=Proc. Natl. Acad. Sci;  citation_year=2009;  citation_volume=106;  citation_pages=13875-9; " /><meta name="citation_reference" content="citation_title=Recently formed polyploid plants diversify at lower rates;  citation_journal_title=Science;  citation_year=2011;  citation_volume=333;  citation_pages=1257" /><meta name="citation_reference" content="citation_title=Genomic plasticity and the diversity of polyploid plants;  citation_journal_title=Science;  citation_year=2008;  citation_volume=320;  citation_pages=481-3; " /><meta name="citation_reference" content="citation_title=Heterozygous genome assembly via binary classification of homologous sequence;  citation_journal_title=BMC Bioinform;  citation_year=2015;  citation_volume=16;  citation_pages=S5" /><meta name="citation_reference" content="citation_title=Redundans: an assembly pipeline for highly heterozygous genomes;  citation_journal_title=Nucleic Acids Res;  citation_year=2016;  citation_volume=44;  citation_pages=e113–3" /><meta name="citation_reference" content="citation_title=The tobacco genome sequence and its comparison with those of tomato and potato;  citation_journal_title=Nat. Commun;  citation_year=2014;  citation_volume=5;  citation_pages=3833" /><meta name="citation_reference" content="citation_title=Sequencing of allotetraploid cotton (Gossypium hirsutum L. acc. TM-1) provides a resource for fiber improvement;  citation_journal_title=Nat. Biotechnol;  citation_year=2015;  citation_volume=33;  citation_pages=531-7; " /><meta name="citation_reference" content="citation_title=A chromosome-based draft sequence of the hexaploid bread wheat (Triticum aestivum) genome;  citation_journal_title=Science;  citation_year=2014;  citation_volume=345;  citation_pages=1251788-8; " /><meta name="citation_reference" content="citation_title=The genome sequence of allopolyploid Brassica juncea and analysis of differential homoeolog gene expression influencing selection;  citation_journal_title=Nat. Genet;  citation_year=2016;  citation_volume=48;  citation_pages=1225-32; " /><meta name="citation_reference" content="citation_title=Conserved but attenuated parental gene expression in allopolyploids: constitutive zinc hyperaccumulation in the allotetraploid Arabidopsis kamchatica;  citation_journal_title=Mol. Biol. Evol;  citation_year=2016;  citation_volume=33;  citation_pages=2781-800; " /><meta name="citation_reference" content="citation_title=Genome-wide quantification of homeolog expression ratio revealed nonstochastic gene regulation in synthetic allopolyploid Arabidopsis;  citation_journal_title=Nucleic Acids Res;  citation_year=2014;  citation_volume=42;  citation_pages=e46" /><meta name="citation_reference" content="citation_title=De novo hybrid assembly of the rubber tree genome reveals evidence of paleotetraploidy in Hevea species;  citation_journal_title=Sci. Reports;  citation_year=2017;  citation_volume=7;  citation_pages=41457" /><meta name="citation_reference" content="citation_title=Comprehensive mapping of long-range interactions reveals folding principles of the human genome;  citation_journal_title=Science (New York, N.Y.);  citation_year=2009;  citation_volume=326;  citation_pages=289-93; " /><meta name="citation_reference" content="citation_title=A chromosome conformation capture ordered sequence of the barley genome;  citation_journal_title=Nature;  citation_year=2017;  citation_volume=544;  citation_pages=427-33; " /><meta name="citation_reference" content="citation_title=Improvements in genomic technologies: application to crop genomics;  citation_journal_title=Trends Biotechnol;  citation_year=2017;  citation_volume=35;  citation_pages=547-58; " /><meta name="citation_reference" content="citation_title=Identification of finger millet (Eleusine coracana Gaertn.) variety suitable to rainfed areas of north western zone of Tamil Nadu;  citation_journal_title=Indian J. Agric. Res;  citation_year=2012;  citation_volume=46;  citation_pages=60-4; " /><meta name="citation_reference" content="citation_title=Comparison of three DNA fluorochromes for flow cytometric estimation of nuclear DNA content in plants;  citation_journal_title=Physiologia Plantarum;  citation_year=1992;  citation_volume=85;  citation_pages=625-31; " /><meta name="citation_reference" content="citation_title=Trimmomatic: a flexible trimmer for Illumina sequence data;  citation_journal_title=Bioinformatics (Oxford, England);  citation_year=2014;  citation_volume=30;  citation_pages=2114-20; " /><meta name="citation_reference" content="citation_title=SUSHI: an exquisite recipe for fully documented, reproducible and reusable NGS data analysis;  citation_journal_title=BMC Bioinform;  citation_year=2016;  citation_volume=17;  citation_pages=228" /><meta name="citation_reference" content="citation_title=Efficient de novo assembly of highly heterozygous genomes from whole-genome shotgun short reads;  citation_journal_title=Genome Res;  citation_year=2014;  citation_volume=24;  citation_pages=1384-95; " /><meta name="citation_reference" content="citation_title=DBG2OLC: efficient assembly of large genomes using long erroneous reads of the third generation sequencing technologies;  citation_journal_title=Sci. Reports;  citation_year=2016;  citation_volume=6;  citation_pages=31900" /><meta name="citation_reference" content="citation_title=Pilon: an integrated tool for comprehensive microbial variant detection and genome assembly improvement;  citation_journal_title=PLoS One;  citation_year=2014;  citation_volume=9;  citation_pages=e112963" /><meta name="citation_reference" content="citation_title=Scaffolding pre-assembled contigs using SSPACE;  citation_journal_title=Bioinformatics;  citation_year=2011;  citation_volume=27;  citation_pages=578-9; " /><meta name="citation_reference" content="citation_title=GMcloser: closing gaps in assemblies accurately with a likelihood-based selection of contig or long-read alignments;  citation_journal_title=Bioinformatics;  citation_year=2015;  citation_volume=31;  citation_pages=btv465" /><meta name="citation_reference" content="citation_title=MAKER: An easy-to-use annotation pipeline designed for emerging model organism genomes;  citation_journal_title=Genome Res;  citation_year=2007;  citation_volume=18;  citation_pages=188-96; " /><meta name="citation_reference" content="citation_title=STAR: ultrafast universal RNA-seq aligner;  citation_journal_title=Bioinformatics;  citation_year=2013;  citation_volume=29;  citation_pages=15-21; " /><meta name="citation_reference" content="citation_title=StringTie enables improved reconstruction of a transcriptome from RNA-seq reads;  citation_journal_title=Nat. Biotechnol;  citation_year=2015;  citation_volume=33;  citation_pages=290-5; " /><meta name="citation_reference" content="citation_title=Repbase Update, a database of eukaryotic repetitive elements;  citation_journal_title=Cytogenet. Genome Res;  citation_year=2005;  citation_volume=110;  citation_pages=462-7; " /><meta name="citation_reference" content="citation_title=Empirical comparison of ab initio repeat finding programs;  citation_journal_title=Nucleic Acids Res;  citation_year=2008;  citation_volume=36;  citation_pages=2284-94; " /><meta name="citation_reference" content="citation_title=Using native and synthetically mapped cDNA alignments to improve de novo gene finding;  citation_journal_title=Bioinformatics;  citation_year=2008;  citation_volume=24;  citation_pages=637-44; " /><meta name="citation_reference" content="citation_title=BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs;  citation_journal_title=Bioinformatics;  citation_year=2015;  citation_volume=31;  citation_pages=3210-2; " /><meta name="citation_reference" content="citation_title=BioNano genome mapping of individual chromosomes supports physical mapping and sequence assembly in complex plant genomes;  citation_journal_title=Plant Biotechnol. J;  citation_year=2016;  citation_volume=14;  citation_pages=1523-31; " /><meta name="citation_reference" content="citation_title=Using linkage maps to correct and scaffold de novo genome assemblies: methods, challenges, and computational tools;  citation_journal_title=Front. Genet;  citation_year=2015;  citation_volume=6;  citation_pages=1-8; " /><meta name="citation_reference" content="citation_title=A combinatorial problem;  citation_journal_title=Proc. Sect. Sci;  citation_year=1946;  citation_volume=49;  citation_pages=758-64; " /><meta name="citation_reference" content="citation_title=How to apply de Bruijn graphs to genome assembly;  citation_journal_title=Nat. Biotechnol;  citation_year=2011;  citation_volume=29;  citation_pages=987-91; " /><meta name="citation_reference" content="citation_title=Separating homeologs by phasing in the tetraploid wheat transcriptome;  citation_journal_title=Genome Biol;  citation_year=2013;  citation_volume=14;  citation_pages=R66" /><meta name="citation_reference" content="citation_title=The evolution of gene duplications: classifying and distinguishing between models;  citation_journal_title=Nat. Rev. Genet;  citation_year=2010;  citation_volume=11;  citation_pages=4" /><meta name="citation_reference" content="citation_title=Homeolog loss and expression changes in natural populations of the recently and repeatedly formed allotetraploid Tragopogon mirus (Asteraceae);  citation_journal_title=BMC Genomics;  citation_year=2010;  citation_volume=11;  citation_pages=97" /><meta name="citation_reference" content="citation_title=Pfam: the protein families database;  citation_journal_title=Nucleic Acids Res;  citation_year=2014;  citation_volume=42;  citation_pages=D222-30; " /><meta name="citation_reference" content="citation_title=Basic local alignment search tool;  citation_journal_title=J. Mol. Biol;  citation_year=1990;  citation_volume=215;  citation_pages=403-10; " /><meta name="citation_reference" content="citation_title=MEGA4: Molecular Evolutionary Genetics Analysis (MEGA) Software Version 4.0;  citation_journal_title=Mol. Biol. Evol;  citation_year=2007;  citation_volume=24;  citation_pages=1596-9; " /><meta name="citation_reference" content="citation_title=Constraints and potentials of future irrigation water availability on agricultural production under climate change;  citation_journal_title=Proc. Natl. Acad. Sci;  citation_year=2014;  citation_volume=111;  citation_pages=3239-44; " /><meta name="citation_reference" content="citation_title=Climate change impacts on global food security;  citation_journal_title=Science;  citation_year=2013;  citation_volume=341;  citation_pages=508-13; " /><meta name="citation_reference" content="citation_title=Variation under domestication in plants: 1859 and today;  citation_journal_title=Philos. Trans. Roy. Soc. B Biol. Sci;  citation_year=2010;  citation_volume=365;  citation_pages=2523-30; " /><meta name="citation_reference" content="citation_title=Patterns and processes in crop domestication: an historical review and quantitative analysis of 203 global food crops;  citation_journal_title=New Phytol;  citation_year=2012;  citation_volume=196;  citation_pages=29-48; " /><meta name="citation_reference" content="citation_title=Genome and transcriptome sequence of finger millet (Eleusine coracana (L.) Gaertn.) provides insights into drought tolerance and nutraceutical properties;  citation_journal_title=BMC Genomics;  citation_year=2017;  citation_volume=18;  citation_pages=465" /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/25/1/39/24151495/dsx036.pdf" /><meta name="description" content="Abstract.  Finger millet (Eleusine coracana (L.) Gaertn) is an important crop for food security because of its tolerance to drought, which is expected to be exa" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/25/1/39/4103403" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/25/1/39/4103403" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/25/1/39/4103403");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsx036';
    </script>
<title>
	Multiple hybrid de novo genome assembly of finger millet, an orphan allotetraploid crop | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./4103403" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BYilATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSGF0YWtleWFtYSwgTWFzYW9taSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgRXZvbHV0aW9uYXJ5IEJpb2xvZ3kgYW5kIEVudmlyb25tZW50YWwgU3R1ZGllcywgVW5pdmVyc2l0eSBvZiBadXJpY2gsIFdpbnRlcnRodXJlcnN0cmFzc2UuIDE5MCwgODA1NyBadXJpY2gsIFN3aXR6ZXJsYW5kIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRnVuY3Rpb25hbCBHZW5vbWljcyBDZW50ZXIgWnVyaWNoLCBFVEggWnVyaWNoL1VuaXZlcnNpdHkgb2YgWnVyaWNoLCA4MDU3IFp1cmljaCwgU3dpdHplcmxhbmQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJTd2lzcyBJbnN0aXR1dGUgb2YgQmlvaW5mb3JtYXRpY3MsIFF1YXJ0aWVyIFNvcmdlIC0gQmF0aW1lbnQgR2Vub3BvZGUsIDEwMTUgTGF1c2FubmUsIFN3aXR6ZXJsYW5kIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQWx1cmksIFNpcmlzaGEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJGdW5jdGlvbmFsIEdlbm9taWNzIENlbnRlciBadXJpY2gsIEVUSCBadXJpY2gvVW5pdmVyc2l0eSBvZiBadXJpY2gsIDgwNTcgWnVyaWNoLCBTd2l0emVybGFuZCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkJhbGFjaGFkcmFuLCBNYXRoaSBUaHVtaWxhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgRXZvbHV0aW9uYXJ5IEJpb2xvZ3kgYW5kIEVudmlyb25tZW50YWwgU3R1ZGllcywgVW5pdmVyc2l0eSBvZiBadXJpY2gsIFdpbnRlcnRodXJlcnN0cmFzc2UuIDE5MCwgODA1NyBadXJpY2gsIFN3aXR6ZXJsYW5kIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRnVuY3Rpb25hbCBHZW5vbWljcyBDZW50ZXIgWnVyaWNoLCBFVEggWnVyaWNoL1VuaXZlcnNpdHkgb2YgWnVyaWNoLCA4MDU3IFp1cmljaCwgU3dpdHplcmxhbmQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIENyb3AgUGh5c2lvbG9neSwgVW5pdmVyc2l0eSBvZiBBZ3JpY3VsdHVyYWwgU2NpZW5jZXMsIEdLVkssIEJhbmdhbG9yZSA1NjAwNjUsIEluZGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2l2YXJhamFuLCBTYWplZXZhbiBSYWRoYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgRXZvbHV0aW9uYXJ5IEJpb2xvZ3kgYW5kIEVudmlyb25tZW50YWwgU3R1ZGllcywgVW5pdmVyc2l0eSBvZiBadXJpY2gsIFdpbnRlcnRodXJlcnN0cmFzc2UuIDE5MCwgODA1NyBadXJpY2gsIFN3aXR6ZXJsYW5kIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRnVuY3Rpb25hbCBHZW5vbWljcyBDZW50ZXIgWnVyaWNoLCBFVEggWnVyaWNoL1VuaXZlcnNpdHkgb2YgWnVyaWNoLCA4MDU3IFp1cmljaCwgU3dpdHplcmxhbmQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIENyb3AgUGh5c2lvbG9neSwgVW5pdmVyc2l0eSBvZiBBZ3JpY3VsdHVyYWwgU2NpZW5jZXMsIEdLVkssIEJhbmdhbG9yZSA1NjAwNjUsIEluZGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iUGF0cmlnbmFuaSwgQW5kcmVhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRnVuY3Rpb25hbCBHZW5vbWljcyBDZW50ZXIgWnVyaWNoLCBFVEggWnVyaWNoL1VuaXZlcnNpdHkgb2YgWnVyaWNoLCA4MDU3IFp1cmljaCwgU3dpdHplcmxhbmQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJHcsO8dGVyLCBTaW1vbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkZ1bmN0aW9uYWwgR2Vub21pY3MgQ2VudGVyIFp1cmljaCwgRVRIIFp1cmljaC9Vbml2ZXJzaXR5IG9mIFp1cmljaCwgODA1NyBadXJpY2gsIFN3aXR6ZXJsYW5kIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iUG92ZWRhLCBMdWN5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRnVuY3Rpb25hbCBHZW5vbWljcyBDZW50ZXIgWnVyaWNoLCBFVEggWnVyaWNoL1VuaXZlcnNpdHkgb2YgWnVyaWNoLCA4MDU3IFp1cmljaCwgU3dpdHplcmxhbmQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTaGltaXp1LUluYXRzdWdpLCBSaWUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEV2b2x1dGlvbmFyeSBCaW9sb2d5IGFuZCBFbnZpcm9ubWVudGFsIFN0dWRpZXMsIFVuaXZlcnNpdHkgb2YgWnVyaWNoLCBXaW50ZXJ0aHVyZXJzdHJhc3NlLiAxOTAsIDgwNTcgWnVyaWNoLCBTd2l0emVybGFuZCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkJhZXRlbiwgSm9obiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkJpb05hbm8gR2Vub21pY3MsIFNhbiBEaWVnbywgQ0EgOTIxMjEsIFVTQSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkZyYW5jb2lqcywgS2Vlcy1KYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJCaW9OYW5vIEdlbm9taWNzLCBTYW4gRGllZ28sIENBIDkyMTIxLCBVU0EiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJOYXRhcmFqYSwgS2FyYWJhIE4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIENyb3AgUGh5c2lvbG9neSwgVW5pdmVyc2l0eSBvZiBBZ3JpY3VsdHVyYWwgU2NpZW5jZXMsIEdLVkssIEJhbmdhbG9yZSA1NjAwNjUsIEluZGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iUmVkZHksIFllbGxvZHUgQSBOYW5qYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkFJQ1JQIChTbWFsbCBNaWxsZXRzKSwgSUNBUi1VQVMsIEdLVkssIEJhbmdhbG9yZSA1NjAwNjUsIEluZGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iUGhhZG5pcywgU2hhbXByYXNhZCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgUGxhbnQgQmlvdGVjaG5vbG9neSwgVW5pdmVyc2l0eSBvZiBBZ3JpY3VsdHVyYWwgU2NpZW5jZXMsIEdLVkssIEJhbmdhbG9yZSA1NjAwNjUsIEluZGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iUmF2aWt1bWFyLCBSYW1hcHVyYSBMIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBQbGFudCBCaW90ZWNobm9sb2d5LCBVbml2ZXJzaXR5IG9mIEFncmljdWx0dXJhbCBTY2llbmNlcywgR0tWSywgQmFuZ2Fsb3JlIDU2MDA2NSwgSW5kaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTY2hsYXBiYWNoLCBSYWxwaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkZ1bmN0aW9uYWwgR2Vub21pY3MgQ2VudGVyIFp1cmljaCwgRVRIIFp1cmljaC9Vbml2ZXJzaXR5IG9mIFp1cmljaCwgODA1NyBadXJpY2gsIFN3aXR6ZXJsYW5kIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU3JlZW1hbiwgU2hlc2hzaGF5ZWUgTSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQ3JvcCBQaHlzaW9sb2d5LCBVbml2ZXJzaXR5IG9mIEFncmljdWx0dXJhbCBTY2llbmNlcywgR0tWSywgQmFuZ2Fsb3JlIDU2MDA2NSwgSW5kaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTaGltaXp1LCBLZW50YXJvIEsiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEV2b2x1dGlvbmFyeSBCaW9sb2d5IGFuZCBFbnZpcm9ubWVudGFsIFN0dWRpZXMsIFVuaXZlcnNpdHkgb2YgWnVyaWNoLCBXaW50ZXJ0aHVyZXJzdHJhc3NlLiAxOTAsIDgwNTcgWnVyaWNoLCBTd2l0emVybGFuZCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IktpaGFyYSBJbnN0aXR1dGUgZm9yIEJpb2xvZ2ljYWwgUmVzZWFyY2gsIFlva29oYW1hIENpdHkgVW5pdmVyc2l0eSwgWW9rb2hhbWEsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJNdWx0aXBsZSBoeWJyaWQgZGUgbm92byBnZW5vbWUgYXNzZW1ibHkgb2YgZmluZ2VyIG1pbGxldCwgYW4gb3JwaGFuIGFsbG90ZXRyYXBsb2lkIGNyb3AiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIzOSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iNDciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3gwMzYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMjUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxOC8wMi8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFsbG90ZXRyYXBsb2lkIG9yaWdpbiBhbmQgZGl2ZXJnZW5jZSBpbiBFbGV1c2luZSAoQ2hsb3JpZG9pZGVhZSwgUG9hY2VhZSk6IGV2aWRlbmNlIGZyb20gbG93LWNvcHkgbnVjbGVhciBnZW5lIHBoeWxvZ2VuaWVzIGFuZCBhIHBsYXN0aWQgZ2VuZSBjaHJvbm9ncmFtOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Bbm4uIEJvdDsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDg7ICBjaXRhdGlvbl9wYWdlcz0xMjg3LTk4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Mb3ctY29weSBudWNsZWFyIGdlbmUgYW5kIE1jR0lTSCByZXNvbHZlcyBwb2x5cGxvaWQgaGlzdG9yeSBvZiBFbGV1c2luZSBjb3JhY2FuYSBhbmQgbW9ycGhvbG9naWNhbCBjaGFyYWN0ZXIgZXZvbHV0aW9uIGluIEVsZXVzaW5lOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UdXJraXNoIEouIEJvdDsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0zODsgIGNpdGF0aW9uX3BhZ2VzPTEtMTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9taWMgaW4gc2l0dSBoeWJyaWRpemF0aW9uIGlkZW50aWZpZXMgZ2Vub21lIGRvbm9yIG9mIGZpbmdlciBtaWxsZXQgKEVsZXVzaW5lIGNvcmFjYW5hKTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VGhlb3IuIEFwcGwuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTEwMjsgIGNpdGF0aW9uX3BhZ2VzPTgyNS0zMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZmljYXRpb24gb2YgdGhlIOKAnEHigJ0gZ2Vub21lIG9mIGZpbmdlciBtaWxsZXQgdXNpbmcgY2hsb3JvcGxhc3QgRE5BOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MTk4ODsgIGNpdGF0aW9uX3ZvbHVtZT0xMTg7ICBjaXRhdGlvbl9wYWdlcz0xNjMtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmV2aWV3IG9mIGZpbmdlciBtaWxsZXQgKEVsZXVzaW5lIGNvcmFjYW5hIChMLikgR2FlcnRuKTogYSBwb3dlciBob3VzZSBvZiBoZWFsdGggYmVuZWZpdGluZyBudXRyaWVudHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUZvb2QgU2NpLiBIdW1hbiBXZWxsbmVzczsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9MTQ5LTU1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NaWxsZXQgZ3JhaW5zOiBudXRyaXRpb25hbCBxdWFsaXR5LCBwcm9jZXNzaW5nLCBhbmQgcG90ZW50aWFsIGhlYWx0aCBiZW5lZml0czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q29tcHIuIFJldi4gRm9vZCBTY2kuIEZvb2QgU2FmZXR5OyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9MjgxLTk1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5ldGljIGRpdmVyc2l0eSBhbmQgZ2Vub21pYyByZXNvdXJjZXMgYXZhaWxhYmxlIGZvciB0aGUgc21hbGwgbWlsbGV0IGNyb3BzIHRvIGFjY2VsZXJhdGUgYSBOZXcgR3JlZW4gUmV2b2x1dGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RnJvbnQuIFBsYW50IFNjaTsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9MTU3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QmlvaW5mb3JtYXRpY3MgaW4gdGhlIG9ycGhhbiBjcm9wczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QnJpZWYuIEJpb2luZm9ybTsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTY0NS01MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TnVjbGVhciBETkEgY29udGVudCBpbiBzcGVjaWVzIG9mIEVsZXVzaW5lIChHcmFtaW5lYWUpOiBhIGNyaXRpY2FsIHJlLWV2YWx1YXRpb24gdXNpbmcgbGFzZXIgZmxvdyBjeXRvbWV0cnk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IFN5c3QuIEV2b2w7ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9MjA3OyAgY2l0YXRpb25fcGFnZXM9MS0xMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXhwcmVzc2lvbiBvZiBhIGZpbmdlciBtaWxsZXQgdHJhbnNjcmlwdGlvbiBmYWN0b3IsIEVjTkFDMSwgaW4gdG9iYWNjbyBjb25mZXJzIGFiaW90aWMgc3RyZXNzLXRvbGVyYW5jZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBPbmU7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPWU0MDM5NyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV4cHJlc3Npb24gYW5hbHlzaXMgb2Ygc3RyZXNzIHJlc3BvbnNpdmUgcGF0aHdheSBnZW5lcyBsaW5rZWQgdG8gZHJvdWdodCBoYXJkaW5lc3MgaW4gYW4gYWRhcHRlZCBjcm9wLCBmaW5nZXIgbWlsbGV0IChFbGV1c2luZSBjb3JhY2FuYSk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIFBsYW50IEJpb2NoZW0uIEJpb3RlY2hub2w7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MjI7ICBjaXRhdGlvbl9wYWdlcz0xOTMtMjAxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GdWxsLWxlbmd0aCBjbG9uaW5nIGFuZCBjaGFyYWN0ZXJpemF0aW9uIG9mIGFiaW90aWMgc3RyZXNzIHJlc3BvbnNpdmUgQ0lQSzMxLWxpa2UgZ2VuZSBmcm9tIGZpbmdlciBtaWxsZXQsIGEgZHJvdWdodC10b2xlcmFudCBjcm9wOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DdXJyLiBTY2k7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9MTExOyAgY2l0YXRpb25fcGFnZXM9ODkwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNjcmlwdG9tZSBhbmFseXNpcyBvZiBzYWxpbml0eSByZXNwb25zaXZlbmVzcyBpbiBjb250cmFzdGluZyBnZW5vdHlwZXMgb2YgZmluZ2VyIG1pbGxldCAoRWxldXNpbmUgY29yYWNhbmEgTC4pIHRocm91Z2ggUk5BLXNlcXVlbmNpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IE1vbC4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT04NTsgIGNpdGF0aW9uX3BhZ2VzPTQ4NS01MDM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBnZW5ldGljIG1hcCBvZiBmaW5nZXIgbWlsbGV0LCBFbGV1c2luZSBjb3JhY2FuYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VGhlb3IuIEFwcGwuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTExNDsgIGNpdGF0aW9uX3BhZ2VzPTMyMS0zMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGZyZXF1ZW5jeSBvZiBwb2x5cGxvaWQgc3BlY2lhdGlvbiBpbiB2YXNjdWxhciBwbGFudHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwuIEFjYWQuIFNjaTsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDY7ICBjaXRhdGlvbl9wYWdlcz0xMzg3NS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZWNlbnRseSBmb3JtZWQgcG9seXBsb2lkIHBsYW50cyBkaXZlcnNpZnkgYXQgbG93ZXIgcmF0ZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MzMzOyAgY2l0YXRpb25fcGFnZXM9MTI1NyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9taWMgcGxhc3RpY2l0eSBhbmQgdGhlIGRpdmVyc2l0eSBvZiBwb2x5cGxvaWQgcGxhbnRzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTMyMDsgIGNpdGF0aW9uX3BhZ2VzPTQ4MS0zOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IZXRlcm96eWdvdXMgZ2Vub21lIGFzc2VtYmx5IHZpYSBiaW5hcnkgY2xhc3NpZmljYXRpb24gb2YgaG9tb2xvZ291cyBzZXF1ZW5jZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEJpb2luZm9ybTsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPVM1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVkdW5kYW5zOiBhbiBhc3NlbWJseSBwaXBlbGluZSBmb3IgaGlnaGx5IGhldGVyb3p5Z291cyBnZW5vbWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT00NDsgIGNpdGF0aW9uX3BhZ2VzPWUxMTPigJMzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHRvYmFjY28gZ2Vub21lIHNlcXVlbmNlIGFuZCBpdHMgY29tcGFyaXNvbiB3aXRoIHRob3NlIG9mIHRvbWF0byBhbmQgcG90YXRvOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIENvbW11bjsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9MzgzMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNlcXVlbmNpbmcgb2YgYWxsb3RldHJhcGxvaWQgY290dG9uIChHb3NzeXBpdW0gaGlyc3V0dW0gTC4gYWNjLiBUTS0xKSBwcm92aWRlcyBhIHJlc291cmNlIGZvciBmaWJlciBpbXByb3ZlbWVudDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBCaW90ZWNobm9sOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTMzOyAgY2l0YXRpb25fcGFnZXM9NTMxLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgY2hyb21vc29tZS1iYXNlZCBkcmFmdCBzZXF1ZW5jZSBvZiB0aGUgaGV4YXBsb2lkIGJyZWFkIHdoZWF0IChUcml0aWN1bSBhZXN0aXZ1bSkgZ2Vub21lOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTM0NTsgIGNpdGF0aW9uX3BhZ2VzPTEyNTE3ODgtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGdlbm9tZSBzZXF1ZW5jZSBvZiBhbGxvcG9seXBsb2lkIEJyYXNzaWNhIGp1bmNlYSBhbmQgYW5hbHlzaXMgb2YgZGlmZmVyZW50aWFsIGhvbW9lb2xvZyBnZW5lIGV4cHJlc3Npb24gaW5mbHVlbmNpbmcgc2VsZWN0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTQ4OyAgY2l0YXRpb25fcGFnZXM9MTIyNS0zMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29uc2VydmVkIGJ1dCBhdHRlbnVhdGVkIHBhcmVudGFsIGdlbmUgZXhwcmVzc2lvbiBpbiBhbGxvcG9seXBsb2lkczogY29uc3RpdHV0aXZlIHppbmMgaHlwZXJhY2N1bXVsYXRpb24gaW4gdGhlIGFsbG90ZXRyYXBsb2lkIEFyYWJpZG9wc2lzIGthbWNoYXRpY2E7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gQmlvbC4gRXZvbDsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT0zMzsgIGNpdGF0aW9uX3BhZ2VzPTI3ODEtODAwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBxdWFudGlmaWNhdGlvbiBvZiBob21lb2xvZyBleHByZXNzaW9uIHJhdGlvIHJldmVhbGVkIG5vbnN0b2NoYXN0aWMgZ2VuZSByZWd1bGF0aW9uIGluIHN5bnRoZXRpYyBhbGxvcG9seXBsb2lkIEFyYWJpZG9wc2lzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT00MjsgIGNpdGF0aW9uX3BhZ2VzPWU0NiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURlIG5vdm8gaHlicmlkIGFzc2VtYmx5IG9mIHRoZSBydWJiZXIgdHJlZSBnZW5vbWUgcmV2ZWFscyBldmlkZW5jZSBvZiBwYWxlb3RldHJhcGxvaWR5IGluIEhldmVhIHNwZWNpZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaS4gUmVwb3J0czsgIGNpdGF0aW9uX3llYXI9MjAxNzsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9NDE0NTciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wcmVoZW5zaXZlIG1hcHBpbmcgb2YgbG9uZy1yYW5nZSBpbnRlcmFjdGlvbnMgcmV2ZWFscyBmb2xkaW5nIHByaW5jaXBsZXMgb2YgdGhlIGh1bWFuIGdlbm9tZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpZW5jZSAoTmV3IFlvcmssIE4uWS4pOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTMyNjsgIGNpdGF0aW9uX3BhZ2VzPTI4OS05MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBjaHJvbW9zb21lIGNvbmZvcm1hdGlvbiBjYXB0dXJlIG9yZGVyZWQgc2VxdWVuY2Ugb2YgdGhlIGJhcmxleSBnZW5vbWU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAxNzsgIGNpdGF0aW9uX3ZvbHVtZT01NDQ7ICBjaXRhdGlvbl9wYWdlcz00MjctMzM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUltcHJvdmVtZW50cyBpbiBnZW5vbWljIHRlY2hub2xvZ2llczogYXBwbGljYXRpb24gdG8gY3JvcCBnZW5vbWljczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VHJlbmRzIEJpb3RlY2hub2w7ICBjaXRhdGlvbl95ZWFyPTIwMTc7ICBjaXRhdGlvbl92b2x1bWU9MzU7ICBjaXRhdGlvbl9wYWdlcz01NDctNTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZpY2F0aW9uIG9mIGZpbmdlciBtaWxsZXQgKEVsZXVzaW5lIGNvcmFjYW5hIEdhZXJ0bi4pIHZhcmlldHkgc3VpdGFibGUgdG8gcmFpbmZlZCBhcmVhcyBvZiBub3J0aCB3ZXN0ZXJuIHpvbmUgb2YgVGFtaWwgTmFkdTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9SW5kaWFuIEouIEFncmljLiBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NDY7ICBjaXRhdGlvbl9wYWdlcz02MC00OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wYXJpc29uIG9mIHRocmVlIEROQSBmbHVvcm9jaHJvbWVzIGZvciBmbG93IGN5dG9tZXRyaWMgZXN0aW1hdGlvbiBvZiBudWNsZWFyIEROQSBjb250ZW50IGluIHBsYW50czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGh5c2lvbG9naWEgUGxhbnRhcnVtOyAgY2l0YXRpb25feWVhcj0xOTkyOyAgY2l0YXRpb25fdm9sdW1lPTg1OyAgY2l0YXRpb25fcGFnZXM9NjI1LTMxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmltbW9tYXRpYzogYSBmbGV4aWJsZSB0cmltbWVyIGZvciBJbGx1bWluYSBzZXF1ZW5jZSBkYXRhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljcyAoT3hmb3JkLCBFbmdsYW5kKTsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTIxMTQtMjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNVU0hJOiBhbiBleHF1aXNpdGUgcmVjaXBlIGZvciBmdWxseSBkb2N1bWVudGVkLCByZXByb2R1Y2libGUgYW5kIHJldXNhYmxlIE5HUyBkYXRhIGFuYWx5c2lzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgQmlvaW5mb3JtOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTE3OyAgY2l0YXRpb25fcGFnZXM9MjI4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RWZmaWNpZW50IGRlIG5vdm8gYXNzZW1ibHkgb2YgaGlnaGx5IGhldGVyb3p5Z291cyBnZW5vbWVzIGZyb20gd2hvbGUtZ2Vub21lIHNob3RndW4gc2hvcnQgcmVhZHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MjQ7ICBjaXRhdGlvbl9wYWdlcz0xMzg0LTk1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EQkcyT0xDOiBlZmZpY2llbnQgYXNzZW1ibHkgb2YgbGFyZ2UgZ2Vub21lcyB1c2luZyBsb25nIGVycm9uZW91cyByZWFkcyBvZiB0aGUgdGhpcmQgZ2VuZXJhdGlvbiBzZXF1ZW5jaW5nIHRlY2hub2xvZ2llczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpLiBSZXBvcnRzOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz0zMTkwMCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBpbG9uOiBhbiBpbnRlZ3JhdGVkIHRvb2wgZm9yIGNvbXByZWhlbnNpdmUgbWljcm9iaWFsIHZhcmlhbnQgZGV0ZWN0aW9uIGFuZCBnZW5vbWUgYXNzZW1ibHkgaW1wcm92ZW1lbnQ7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgT25lOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz1lMTEyOTYzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2NhZmZvbGRpbmcgcHJlLWFzc2VtYmxlZCBjb250aWdzIHVzaW5nIFNTUEFDRTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9Mjc7ICBjaXRhdGlvbl9wYWdlcz01NzgtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R01jbG9zZXI6IGNsb3NpbmcgZ2FwcyBpbiBhc3NlbWJsaWVzIGFjY3VyYXRlbHkgd2l0aCBhIGxpa2VsaWhvb2QtYmFzZWQgc2VsZWN0aW9uIG9mIGNvbnRpZyBvciBsb25nLXJlYWQgYWxpZ25tZW50czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MzE7ICBjaXRhdGlvbl9wYWdlcz1idHY0NjUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NQUtFUjogQW4gZWFzeS10by11c2UgYW5ub3RhdGlvbiBwaXBlbGluZSBkZXNpZ25lZCBmb3IgZW1lcmdpbmcgbW9kZWwgb3JnYW5pc20gZ2Vub21lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0xODsgIGNpdGF0aW9uX3BhZ2VzPTE4OC05NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U1RBUjogdWx0cmFmYXN0IHVuaXZlcnNhbCBSTkEtc2VxIGFsaWduZXI7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTI5OyAgY2l0YXRpb25fcGFnZXM9MTUtMjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN0cmluZ1RpZSBlbmFibGVzIGltcHJvdmVkIHJlY29uc3RydWN0aW9uIG9mIGEgdHJhbnNjcmlwdG9tZSBmcm9tIFJOQS1zZXEgcmVhZHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gQmlvdGVjaG5vbDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0zMzsgIGNpdGF0aW9uX3BhZ2VzPTI5MC01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZXBiYXNlIFVwZGF0ZSwgYSBkYXRhYmFzZSBvZiBldWthcnlvdGljIHJlcGV0aXRpdmUgZWxlbWVudHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUN5dG9nZW5ldC4gR2Vub21lIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMTA7ICBjaXRhdGlvbl9wYWdlcz00NjItNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RW1waXJpY2FsIGNvbXBhcmlzb24gb2YgYWIgaW5pdGlvIHJlcGVhdCBmaW5kaW5nIHByb2dyYW1zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0zNjsgIGNpdGF0aW9uX3BhZ2VzPTIyODQtOTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVVzaW5nIG5hdGl2ZSBhbmQgc3ludGhldGljYWxseSBtYXBwZWQgY0ROQSBhbGlnbm1lbnRzIHRvIGltcHJvdmUgZGUgbm92byBnZW5lIGZpbmRpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTI0OyAgY2l0YXRpb25fcGFnZXM9NjM3LTQ0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CVVNDTzogYXNzZXNzaW5nIGdlbm9tZSBhc3NlbWJseSBhbmQgYW5ub3RhdGlvbiBjb21wbGV0ZW5lc3Mgd2l0aCBzaW5nbGUtY29weSBvcnRob2xvZ3M7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTMxOyAgY2l0YXRpb25fcGFnZXM9MzIxMC0yOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CaW9OYW5vIGdlbm9tZSBtYXBwaW5nIG9mIGluZGl2aWR1YWwgY2hyb21vc29tZXMgc3VwcG9ydHMgcGh5c2ljYWwgbWFwcGluZyBhbmQgc2VxdWVuY2UgYXNzZW1ibHkgaW4gY29tcGxleCBwbGFudCBnZW5vbWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBCaW90ZWNobm9sLiBKOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9MTUyMy0zMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VXNpbmcgbGlua2FnZSBtYXBzIHRvIGNvcnJlY3QgYW5kIHNjYWZmb2xkIGRlIG5vdm8gZ2Vub21lIGFzc2VtYmxpZXM6IG1ldGhvZHMsIGNoYWxsZW5nZXMsIGFuZCBjb21wdXRhdGlvbmFsIHRvb2xzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Gcm9udC4gR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTEtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBjb21iaW5hdG9yaWFsIHByb2JsZW07ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIFNlY3QuIFNjaTsgIGNpdGF0aW9uX3llYXI9MTk0NjsgIGNpdGF0aW9uX3ZvbHVtZT00OTsgIGNpdGF0aW9uX3BhZ2VzPTc1OC02NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SG93IHRvIGFwcGx5IGRlIEJydWlqbiBncmFwaHMgdG8gZ2Vub21lIGFzc2VtYmx5OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIEJpb3RlY2hub2w7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9Mjk7ICBjaXRhdGlvbl9wYWdlcz05ODctOTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNlcGFyYXRpbmcgaG9tZW9sb2dzIGJ5IHBoYXNpbmcgaW4gdGhlIHRldHJhcGxvaWQgd2hlYXQgdHJhbnNjcmlwdG9tZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz1SNjYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZXZvbHV0aW9uIG9mIGdlbmUgZHVwbGljYXRpb25zOiBjbGFzc2lmeWluZyBhbmQgZGlzdGluZ3Vpc2hpbmcgYmV0d2VlbiBtb2RlbHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gUmV2LiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ib21lb2xvZyBsb3NzIGFuZCBleHByZXNzaW9uIGNoYW5nZXMgaW4gbmF0dXJhbCBwb3B1bGF0aW9ucyBvZiB0aGUgcmVjZW50bHkgYW5kIHJlcGVhdGVkbHkgZm9ybWVkIGFsbG90ZXRyYXBsb2lkIFRyYWdvcG9nb24gbWlydXMgKEFzdGVyYWNlYWUpOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz05NyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBmYW06IHRoZSBwcm90ZWluIGZhbWlsaWVzIGRhdGFiYXNlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT00MjsgIGNpdGF0aW9uX3BhZ2VzPUQyMjItMzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJhc2ljIGxvY2FsIGFsaWdubWVudCBzZWFyY2ggdG9vbDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gTW9sLiBCaW9sOyAgY2l0YXRpb25feWVhcj0xOTkwOyAgY2l0YXRpb25fdm9sdW1lPTIxNTsgIGNpdGF0aW9uX3BhZ2VzPTQwMy0xMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TUVHQTQ6IE1vbGVjdWxhciBFdm9sdXRpb25hcnkgR2VuZXRpY3MgQW5hbHlzaXMgKE1FR0EpIFNvZnR3YXJlIFZlcnNpb24gNC4wOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEJpb2wuIEV2b2w7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MjQ7ICBjaXRhdGlvbl9wYWdlcz0xNTk2LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbnN0cmFpbnRzIGFuZCBwb3RlbnRpYWxzIG9mIGZ1dHVyZSBpcnJpZ2F0aW9uIHdhdGVyIGF2YWlsYWJpbGl0eSBvbiBhZ3JpY3VsdHVyYWwgcHJvZHVjdGlvbiB1bmRlciBjbGltYXRlIGNoYW5nZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bC4gQWNhZC4gU2NpOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTExMTsgIGNpdGF0aW9uX3BhZ2VzPTMyMzktNDQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNsaW1hdGUgY2hhbmdlIGltcGFjdHMgb24gZ2xvYmFsIGZvb2Qgc2VjdXJpdHk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MzQxOyAgY2l0YXRpb25fcGFnZXM9NTA4LTEzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1WYXJpYXRpb24gdW5kZXIgZG9tZXN0aWNhdGlvbiBpbiBwbGFudHM6IDE4NTkgYW5kIHRvZGF5OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QaGlsb3MuIFRyYW5zLiBSb3kuIFNvYy4gQiBCaW9sLiBTY2k7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MzY1OyAgY2l0YXRpb25fcGFnZXM9MjUyMy0zMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGF0dGVybnMgYW5kIHByb2Nlc3NlcyBpbiBjcm9wIGRvbWVzdGljYXRpb246IGFuIGhpc3RvcmljYWwgcmV2aWV3IGFuZCBxdWFudGl0YXRpdmUgYW5hbHlzaXMgb2YgMjAzIGdsb2JhbCBmb29kIGNyb3BzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OZXcgUGh5dG9sOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTE5NjsgIGNpdGF0aW9uX3BhZ2VzPTI5LTQ4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUgYW5kIHRyYW5zY3JpcHRvbWUgc2VxdWVuY2Ugb2YgZmluZ2VyIG1pbGxldCAoRWxldXNpbmUgY29yYWNhbmEgKEwuKSBHYWVydG4uKSBwcm92aWRlcyBpbnNpZ2h0cyBpbnRvIGRyb3VnaHQgdG9sZXJhbmNlIGFuZCBudXRyYWNldXRpY2FsIHByb3BlcnRpZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxNzsgIGNpdGF0aW9uX3ZvbHVtZT0xODsgIGNpdGF0aW9uX3BhZ2VzPTQ2NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8yNS8xLzM5LzI0MTUxNDk1L2RzeDAzNi5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgRmluZ2VyIG1pbGxldCAoRWxldXNpbmUgY29yYWNhbmEgKEwuKSBHYWVydG4pIGlzIGFuIGltcG9ydGFudCBjcm9wIGZvciBmb29kIHNlY3VyaXR5IGJlY2F1c2Ugb2YgaXRzIHRvbGVyYW5jZSB0byBkcm91Z2h0LCB3aGljaCBpcyBleHBlY3RlZCB0byBiZSBleGEiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVzLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTQxMDM0MDMmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2RkP+xYDvGjXx6pFm+weROCnwGnNBY=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f1%2f39%2f4103403" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f1%2f39%2f4103403" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=107546" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/25/1"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/1/3/m_dnares_25_1cover.png?Expires=2147483647&amp;Signature=SO6YCeydUh5yJs5n-IY9wmy~LoBoAx4P6Fvm-AjuGlXgQjk9YpdSZprn-w-uvSGcgqbsut967tdIfiY3dSqjk-kYgFErDTIMFh~0aK8LiNrwVszpCD29Va86wemvVduCLQhn8b79YN7Bcvi7qFB5E0wUNp-cxt7rNByaE14ePnoDwCgR5G58Pc5wx03IVhK9jft2n0g0lvAsEdQaWr3LG7PN4prc1KFhItNPOxm0fdg7HmXK23iCiP1EUMnFfqTR655CMN~bUB0aDDAZUT18HRZXXdjkjcMUCT~fjmwOlkGzU~71oDZYn-Ji-8-K8iuA-savjer0tQoSi784gQ8L6w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/25/1">
                        <div class="volume trailing-comma">Volume 25</div>
                        <div class="issue">Issue 1</div>

            </a>

        </div>
        <div class="ii-pub-date">
February 2018        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="112482221">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#112482221">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="112482223">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#112482223">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="112482229">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#112482229">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="112482247">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#112482247">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="112482265">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#112482265">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="112482278">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#112482278">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/25/1/25/4161430">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/25/1/49/4107864">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
<span class="article-flag"><a href="https://academic.oup.com/dnaresearch/search-results?f_OUPSeries=Editor's+Choice">Editor's Choice</a></span>            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Multiple hybrid <em>de novo</em> genome assembly of finger millet, an orphan allotetraploid crop
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                            <div class="meta-authors--limited">
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Masaomi Hatakeyama</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Evolutionary Biology and Environmental Studies, University of Zurich, Winterthurerstrasse. 190, 8057 Zurich, Switzerland</div><div class="aff">Functional Genomics Center Zurich, ETH Zurich/University of Zurich, 8057 Zurich, Switzerland</div><div class="aff">Swiss Institute of Bioinformatics, Quartier Sorge - Batiment Genopode, 1015 Lausanne, Switzerland</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Masaomi+Hatakeyama">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hatakeyama M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hatakeyama M&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Masaomi Hatakeyama</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Sirisha Aluri</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Functional Genomics Center Zurich, ETH Zurich/University of Zurich, 8057 Zurich, Switzerland</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Sirisha+Aluri">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Aluri S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Aluri S&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Sirisha Aluri</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Mathi Thumilan Balachadran</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Evolutionary Biology and Environmental Studies, University of Zurich, Winterthurerstrasse. 190, 8057 Zurich, Switzerland</div><div class="aff">Functional Genomics Center Zurich, ETH Zurich/University of Zurich, 8057 Zurich, Switzerland</div><div class="aff">Department of Crop Physiology, University of Agricultural Sciences, GKVK, Bangalore 560065, India</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Mathi+Thumilan+Balachadran">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Balachadran M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Balachadran M&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Mathi Thumilan Balachadran</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Sajeevan Radha Sivarajan</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Evolutionary Biology and Environmental Studies, University of Zurich, Winterthurerstrasse. 190, 8057 Zurich, Switzerland</div><div class="aff">Functional Genomics Center Zurich, ETH Zurich/University of Zurich, 8057 Zurich, Switzerland</div><div class="aff">Department of Crop Physiology, University of Agricultural Sciences, GKVK, Bangalore 560065, India</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Sajeevan+Radha+Sivarajan">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sivarajan S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sivarajan S&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Sajeevan Radha Sivarajan</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Andrea Patrignani</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Functional Genomics Center Zurich, ETH Zurich/University of Zurich, 8057 Zurich, Switzerland</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Andrea+Patrignani">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Patrignani A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Patrignani A&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Andrea Patrignani</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Simon Gr&#252;ter</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Functional Genomics Center Zurich, ETH Zurich/University of Zurich, 8057 Zurich, Switzerland</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Simon+Gr%c3%bcter">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Gr&#252;ter S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Gr&#252;ter S&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Simon Grüter</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Lucy Poveda</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Functional Genomics Center Zurich, ETH Zurich/University of Zurich, 8057 Zurich, Switzerland</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Lucy+Poveda">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Poveda L">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Poveda L&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Lucy Poveda</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Rie Shimizu-Inatsugi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Evolutionary Biology and Environmental Studies, University of Zurich, Winterthurerstrasse. 190, 8057 Zurich, Switzerland</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Rie+Shimizu-Inatsugi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shimizu-Inatsugi R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Shimizu-Inatsugi R&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Rie Shimizu-Inatsugi</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">John Baeten</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">BioNano Genomics, San Diego, CA 92121, USA</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=John+Baeten">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Baeten J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Baeten J&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">John Baeten</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kees-Jan Francoijs</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">BioNano Genomics, San Diego, CA 92121, USA</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kees-Jan+Francoijs">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Francoijs K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Francoijs K&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Kees-Jan Francoijs</a>
                                    </span>
                                <a class="meta-authors--etal is-active" id="show-meta-authors" aria-controls="remaining-authors" href="#">
                                    <span>...</span> Show more</a>
                                <div id="remaining-authors" class="meta-authors--remaining" aria-hidden="true" aria-labeledby="show-meta-authors">
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Karaba N Nataraja</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Crop Physiology, University of Agricultural Sciences, GKVK, Bangalore 560065, India</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Karaba+N+Nataraja">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nataraja K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nataraja K&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Karaba N Nataraja</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yellodu A Nanja Reddy</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">AICRP (Small Millets), ICAR-UAS, GKVK, Bangalore 560065, India</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yellodu+A+Nanja+Reddy">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Reddy Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Reddy Y&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Yellodu A Nanja Reddy</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shamprasad Phadnis</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Plant Biotechnology, University of Agricultural Sciences, GKVK, Bangalore 560065, India</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shamprasad+Phadnis">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Phadnis S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Phadnis S&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Shamprasad Phadnis</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ramapura L Ravikumar</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Plant Biotechnology, University of Agricultural Sciences, GKVK, Bangalore 560065, India</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ramapura+L+Ravikumar">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ravikumar R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ravikumar R&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Ramapura L Ravikumar</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ralph Schlapbach</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Functional Genomics Center Zurich, ETH Zurich/University of Zurich, 8057 Zurich, Switzerland</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ralph+Schlapbach">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Schlapbach R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Schlapbach R&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Ralph Schlapbach</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Sheshshayee M Sreeman</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Crop Physiology, University of Agricultural Sciences, GKVK, Bangalore 560065, India</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsx036-cor1" data-open="dsx036-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Fax. +41 44 63 56821. Email: <a href="mailto:kentaro.shimizu@ieu.uzh.ch" target="_blank">kentaro.shimizu@ieu.uzh.ch</a> (K.K.S.); Tel. +91 80 23330153. Fax: +91 80 23330277. Email: <a href="mailto:msshesh1@uasbangalore.edu.in" target="_blank">msshesh1@uasbangalore.edu.in</a> (S.M.S.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Sheshshayee+M+Sreeman">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sreeman S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sreeman S&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Sheshshayee M Sreeman</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kentaro K Shimizu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Evolutionary Biology and Environmental Studies, University of Zurich, Winterthurerstrasse. 190, 8057 Zurich, Switzerland</div><div class="aff">Kihara Institute for Biological Research, Yokohama City University, Yokohama, Japan</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsx036-cor1" data-open="dsx036-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Fax. +41 44 63 56821. Email: <a href="mailto:kentaro.shimizu@ieu.uzh.ch" target="_blank">kentaro.shimizu@ieu.uzh.ch</a> (K.K.S.); Tel. +91 80 23330153. Fax: +91 80 23330277. Email: <a href="mailto:msshesh1@uasbangalore.edu.in" target="_blank">msshesh1@uasbangalore.edu.in</a> (S.M.S.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kentaro+K+Shimizu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shimizu K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Shimizu K&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Kentaro K Shimizu</a>
                                        </span>
                                </div>
                            </div>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsx036-FM1"><div class="fn"><div class="footnote-content">Edited by Dr. Sachiko Isobe</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 25, Issue 1, 1 February 2018, Pages 39–47, <a href='https://doi.org/10.1093/dnares/dsx036'>https://doi.org/10.1093/dnares/dsx036</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">05 September 2017</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">06 May 2017</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">23 August 2017</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="4103403" href="/dnaresearch/article-pdf/25/1/39/24151495/dsx036.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Masaomi Hatakeyama, Sirisha Aluri, Mathi Thumilan Balachadran, Sajeevan Radha Sivarajan, Andrea Patrignani, Simon Grüter, Lucy Poveda, Rie Shimizu-Inatsugi, John Baeten, Kees-Jan Francoijs, Karaba N Nataraja, Yellodu A Nanja Reddy, Shamprasad Phadnis, Ramapura L Ravikumar, Ralph Schlapbach, Sheshshayee M Sreeman, Kentaro K Shimizu;  Multiple hybrid <em>de novo</em> genome assembly of finger millet, an orphan allotetraploid crop, <em>DNA Research</em>, Volume 25, Issue 1, 1 February 2018, Pages 39–47, <a href='https://doi.org/10.1093/dnares/dsx036'>https://doi.org/10.1093/dnares/dsx036</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/4103403?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/4103403?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/4103403?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/4103403?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/4103403?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/4103403?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/4103403?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Multiple%20hybrid%20%26lt%3Bem%26gt%3Bde%20novo%26lt%3B%2Fem%26gt%3B%20genome%20assembly%20of%20finger%20millet%2C%20an%20orphan%20allotetraploid%20crop&amp;publicationDate=2017-09-05&amp;volumeNum=25&amp;issueNum=1&amp;author=Hatakeyama%2C%20Masaomi%3B%20Aluri%2C%20Sirisha&amp;startPage=39&amp;endPage=47&amp;contentId=10.1093%2Fdnares%2Fdsx036&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f1%2f39%2f4103403" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=112482221 id="112482221" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Finger millet (<em>Eleusine coracana</em> (L.) Gaertn) is an important crop for food security because of its tolerance to drought, which is expected to be exacerbated by global climate changes. Nevertheless, it is often classified as an orphan/underutilized crop because of the paucity of scientific attention. Among several small millets, finger millet is considered as an excellent source of essential nutrient elements, such as iron and zinc; hence, it has potential as an alternate coarse cereal. However, high-quality genome sequence data of finger millet are currently not available. One of the major problems encountered in the genome assembly of this species was its polyploidy, which hampers genome assembly compared with a diploid genome. To overcome this problem, we sequenced its genome using diverse technologies with sufficient coverage and assembled it via a novel multiple hybrid assembly workflow that combines next-generation with single-molecule sequencing, followed by whole-genome optical mapping using the Bionano Irys® system. The total number of scaffolds was 1,897 with an N50 length &gt;2.6 Mb and detection of 96% of the universal single-copy orthologs. The majority of the homeologs were assembled separately. This indicates that the proposed workflow is applicable to the assembly of other allotetraploid genomes.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;hybrid de novo assembly&quot;">hybrid <em>de novo</em> assembly</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;finger millet&quot;">finger millet</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;allotetraploid genome&quot;">allotetraploid genome</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;whole-genome optical mapping&quot;">whole-genome optical mapping</a></div>                            <h2 scrollto-destination=112482223 id="112482223" class="section-title" >1. Introduction</h2>
<p>After sorghum and pearl millet, finger millet (<em>Eleusine coracana</em> (L.) Gaertn) is the coarse cereal that is most widely cultivated in dry regions under rain-fed conditions. Finger millet is an allotetraploid (2<em>n</em> = 4× = 36) with an AABB genome that belongs to the family Poaceae and subfamily Chloridoideae. It was domesticated from the wild allotetraploid <em>E. africana</em> in Uganda to Ethiopia more than 5000 years ago. It is widely accepted that the A genome progenitor is the wild diploid species <em>E. indica</em> (2<em>n</em> = 2× = 18) or one of its close relatives, including <em>E. tristachya</em>, while the B genome progenitor is unknown or most likely extinct.<a reveal-id="dsx036-B1" data-open="dsx036-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a><sup>,</sup><a reveal-id="dsx036-B2" data-open="dsx036-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a><sup>,</sup><a reveal-id="dsx036-B3" data-open="dsx036-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a><sup>,</sup><a reveal-id="dsx036-B4" data-open="dsx036-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> Finger millet is one of the richest sources of mineral elements such as calcium, iron, manganese, and zinc, in addition to being a fairly good source of vitamins and several essential amino acids.<a reveal-id="dsx036-B5" data-open="dsx036-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> Among all millets, finger millet is often considered as the best source of lysine.<a reveal-id="dsx036-B6" data-open="dsx036-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a> With its high satiety value and low glycemic index, finger millet has excellent potential as a nutraceutical cereal. It is grown widely, including in India, many African countries, China, and, historically until the present day, even in Japan.<a reveal-id="dsx036-B7" data-open="dsx036-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a></p><p>Despite its importance, very little research emphasis has been given to finger millet; hence, this species is classified as an orphan or underutilized crop.<a reveal-id="dsx036-B7" data-open="dsx036-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a><sup>,</sup><a reveal-id="dsx036-B8" data-open="dsx036-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> The major reasons for this classification include a distant relationship to any of the major cereals, a large genome size, and particularly, its tetraploidy. Thus, limited efforts have been made toward the generation of genomic resources for this species, including sequencing of its genome. The genome size of finger millet was estimated to be 3.34–3.87 pg (2C),<a reveal-id="dsx036-B7" data-open="dsx036-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a><sup>,</sup><a reveal-id="dsx036-B9" data-open="dsx036-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> which corresponds to 1.63–1.89 Gb in haploid (1C) cells assuming that 1 pg DNA corresponds to 978 Mbp with a GC content of 50%. Several attempts have also been made to prospect novel genes linked to the drought-stress acclimation response,<a reveal-id="dsx036-B10 dsx036-B11 dsx036-B12 dsx036-B13" data-open="dsx036-B10 dsx036-B11 dsx036-B12 dsx036-B13" class="link link-ref link-reveal xref-bibr"><sup>10–13</sup></a> and a small number of SSR markers for the construction of genetic linkage maps have been reported.<a reveal-id="dsx036-B14" data-open="dsx036-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> The <em>EcNAC1</em> gene of <em>E. coracana</em>, which encodes a NAC family transcriptional factor, was shown to be induced by drought and to confer resistance to drought and other stresses when expressed in tobacco.<a reveal-id="dsx036-B10" data-open="dsx036-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> However, compared with the progress made in generating genomic information in sorghum and pearl millet, efforts in finger millet remain scarce. The absence of such genomic resources has been the major impediment in improving finger millet through the adoption of molecular breeding approaches.</p><p>The genome assembly and annotation of polyploid species has been a major challenge in genomics, because homeologous genes (i.e. duplicated genes derived from genome duplication) are by definition highly similar and their separate assembly is difficult. About 35% of vascular plants are estimated to be recent polyploids,<a reveal-id="dsx036-B15" data-open="dsx036-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a><sup>,</sup><a reveal-id="dsx036-B16" data-open="dsx036-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> and it has long been known that important crop species are polyploids.<a reveal-id="dsx036-B17" data-open="dsx036-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> However, the genome assemblies of only a handful of polyploid species have been reported. More importantly, the quality of the polyploid assembly, in particular the scaffold length, tends to be low in such reports. The presence of duplicated genes and polyploidy split the assembly, thus leading to shorter scaffolds and contigs.<a reveal-id="dsx036-B18" data-open="dsx036-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a><sup>,</sup><a reveal-id="dsx036-B19" data-open="dsx036-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a></p><p>Some polyploid species have, however, been assembled; e.g. <em>Nicotiana tabacum</em>,<a reveal-id="dsx036-B20" data-open="dsx036-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a><sup>,</sup><em>Gossypium hirsutum</em>,<a reveal-id="dsx036-B21" data-open="dsx036-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a><sup>,</sup><em>Triticum aestivum</em>,<a reveal-id="dsx036-B22" data-open="dsx036-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> and <em>Brassica juncea.</em><a reveal-id="dsx036-B23" data-open="dsx036-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> The genome assembly of hexaploid wheat, <em>Triticum aestivum</em> (TGACv1),<a reveal-id="dsx036-B22" data-open="dsx036-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> was performed via chromosome separation, albeit with an N50 of several kb.<a reveal-id="dsx036-B22" data-open="dsx036-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> A combined approach of PacBio long reads and Bionano Genomics optical genome mapping succeeded in assembling the genome of <em>Brassica juncea</em>, which is an allotetraploid, with an N50 scaffold that was 1.5 Mb long.<a reveal-id="dsx036-B23" data-open="dsx036-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> Subsequently, pseudochromosomes were constructed using the diploid genome and genetic linkage maps. However, in previous polyploid assemblies, little was reported on the validation of the separation of homeologs in cases with unknown parental species. Another potential assembly strategy that can be used when all the parental species are known is the assembly of the genome sequences based on the parental diploid species. This was employed for the allotetraploid <em>Arabidopsis kamchatica</em> using the two parental species <em>A. halleri</em> and <em>A. lyrata</em>;<a reveal-id="dsx036-B24" data-open="dsx036-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a><sup>,</sup><a reveal-id="dsx036-B25" data-open="dsx036-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> however, this cannot be simply applied to finger millet because one of the parental species is unknown. Recently, the rubber tree genome was assembled using Dovetail Chicago method<a reveal-id="dsx036-B26" data-open="dsx036-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> (Dovetail Genomics, LLC, Santa Cruz, CA, USA) with a scaffold N50 of 96.8 kb, and they revealed the paleotetraploidy of the genome. The Chicago library is a longer range technique than mate-pair library, which is based on chromosome conformation capture sequencing (Hi-C) protocol.<a reveal-id="dsx036-B27" data-open="dsx036-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> The barley genome was assembled in chromosome length scale with Hi-C library.<a reveal-id="dsx036-B28" data-open="dsx036-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> NRGene Ltd (Ness-Ziona, Israel) has succeeded recently in assembling some crop genomes, e.g. the genome of maize with a very high scaffold N50 of 35.5 Mb, using the DeNovoMAGIC<sup>TM</sup> software (<a class="link link-uri" href="http://nrgene.com/products-technology/denovomagic/" target="_blank">http://nrgene.com/products-technology/denovomagic/</a>).<a reveal-id="dsx036-B29" data-open="dsx036-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> They require some PCR-free paired-end libraries and mate-pair libraries for the DeNovoMAGIC<sup>TM</sup>, but the company has not reported its algorithm and strategy in detail.</p><p>Here, we report the whole-genome draft sequencing and assembly of a tetraploid finger millet, cultivar PR202 (IC: 479099), using a novel polyploid genome assembly workflow. This cultivar was chosen for this work considering the extent of its adaptability. A selection from the Godavari regions of Southern India was chosen for this study based on its resilience to drought and high-temperature stresses. This cultivar is being used as the national check in all of the multilocation yield-evaluation trials performed by the All India Coordinated Small Millets Improvement Project (AICSMIP) (<a class="link link-uri" href="http://millets.res.in/aicrp_small.php" target="_blank">http://millets.res.in/aicrp_small.php</a>).<a reveal-id="dsx036-B30" data-open="dsx036-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a></p>                            <h2 scrollto-destination=112482229 id="112482229" class="section-title" >2. Materials and methods</h2>
<p>Seeds of finger millet (variety name, PR202; IC number, 479099; the IC number is the indigenous collection number issued by the National Bureau of Plant Genetic Resources, India) that were stocked at the Project Coordinating Unit (Small Millets), University of Agricultural Sciences, GKVK, Bangalore, India) were used for germination. DNA was extracted from young leaves of finger millet plants grown in a plant chamber at 20 °C to 22 °C, 3 weeks later for sequencing and 2 months later for genome scanning. In this section, the methods used for DNA and RNA isolation and preparation of libraries for sequencing are described.</p>                            <h3 scrollto-destination=112482231 id="112482231" class="section-title" >2.1. Preparation of DNA libraries for sequencing</h3>
<p>Good-quality and high-molecular-weight genomic DNA (gDNA) was isolated using a combination of CTAB and Genomic tip 500/G column (Qiagen AG, Hombrechtikon, Switzerland) purification method, in accordance with the protocol ‘Preparing Arabidopsis Genomic DNA for Size-Selected ∼20 kb SMRTbell™ Libraries’ (<a class="link link-uri" href="http://www.pacb.com/wp-content/uploads/2015/09/Shared-Protocol-Preparing-Arabidopsis-DNA-for-20-kb-SMRTbell-Libraries.pdf" target="_blank">http://www.pacb.com/wp-content/uploads/2015/09/Shared-Protocol-Preparing-Arabidopsis-DNA-for-20-kb-SMRTbell-Libraries.pdf</a>). The DNA pellet obtained using the conventional CTAB method was dissolved in 1 ml of Dnase- and RNase-free distilled water (Life Technologies, Carlsbad, CA, USA) and purified by passing through Genomic tip 500/G columns, to obtain intact DNA with a size above 60 kb. For library preparation, the TruSeq DNA NanoSample Prep Kit v2 (Illumina Inc., San Diego, CA, USA) was used in the following steps. DNA samples (1 µg) were sonicated on a Covaris instrument (Covaris, Woburn, MA, USA) using settings specific for a fragment size of 550 bp. The fragmented DNA samples were size selected using AMPure beads (Beckman Coulter, Inc., Brea, CA, USA), end-repaired, and polyadenylated. TruSeq adapters containing the index for multiplexing were ligated to the fragmented DNA samples. Fragments containing TruSeq adapters at both ends were selectively enriched by PCR. The quality and quantity of the enriched libraries were validated using Qubit<sup>®</sup> 2.0 Fluorometer (Life Technologies) and TapeStation system (Agilent, Waldbronn, Germany). The products were a smear with an average fragment size of approximately 700 bp. The libraries were normalized to 10 nM in Tris-HCl 10 mM, pH 8.5 with 0.1% Tween 20. The 3–8 and 20 kb mate-pair libraries were prepared according to Lucigen’s NxSeq<sup>®</sup> Long Mate Pair Kit protocol and NxSeq<sup>®</sup> 20 kb Mate Pair Library kit protocol (Lucigen Corporation, Middleton, WI, USA), respectively. Briefly, genomic DNA was sheared using Covaris G-TUBES (Covaris, Woburn, MA, USA) to the desired size (3, 5, 8, and 20 kb), end-repaired, A-tailed, and ligated to adaptors. The insert was size selected and ligated to a unique coupler that contained encrypted Chimera Code™ sequences. The circularized inserts were treated with exonuclease for the removal of unwanted linear DNA and digested with a selection of endonucleases to produce correctly sized di-tags (400–900 bp). Biotin capture was used to remove unwanted DNA fragments prior to the addition of the Junction Code™ Reagent. The library was recircularized and PCR amplified and the quality was assessed using a Qubit<sup>®</sup> 2.0 Fluorometer (Life Technologies), and the TapeStation system (Agilent, Waldbronn, Germany). The final mate-pair libraries were normalized to 10 nM in Tris–HCl 10 mM, pH 8.5 with 0.1% Tween 20.</p>                            <h3 scrollto-destination=112482233 id="112482233" class="section-title" >2.2. Preparation of RNA libraries for sequencing</h3>
<p>RNA was extracted from young leaves of finger millet 1 month after germination. Freshly germinated finger millet plants were exposed to drought stress, in which the plants were not watered for 7 days. Tissue samples were taken from the young leaves before and after the exposure to drought conditions. RNA was extracted using an RNeasy plant kit (Qiagen AG, Hombrechtikon, Switzerland). The quantity and quality of the isolated RNA was determined using a Qubit<sup>®</sup> 2.0 Fluorometer (Life Technologies) and a Bioanalyzer 2100 (Agilent, Waldbronn). For library preparation, TruSeq Stranded mRNA Sample Prep Kit (Illumina) was used in the subsequent steps. Briefly, total RNA samples (1 µg) were ribosome depleted and then reverse transcribed into double-stranded cDNAs, with addition of actinomycin during first-strand synthesis. The cDNA samples were fragmented, end-repaired, and polyadenylated before the ligation of TruSeq adapters. The adapters contained the index for multiplexing. Fragments containing TruSeq adapters at both ends were selectively enriched with PCR. The quality and quantity of the enriched libraries were assessed using a Qubit<sup>®</sup> 2.0 Fluorometer and a Bioanalyzer 2100 (Agilent, Waldbronn). Products were a smear with an average fragment size of ∼360 bp. The libraries were normalized to 10 nM in Tris-HCl 10 mM, pH 8.5 with 0.1% Tween 20.</p>                            <h3 scrollto-destination=112482235 id="112482235" class="section-title" >2.3. Library preparation and sequencing for PacBio RS II</h3>
<p>The SMRT Bell library was produced using a DNA Template Prep Kit 1.0 (PacBio p/n 100-259-100). The concentration of the input gDNA was measured using a Qubit Fluorometer dsDNA Broad Range assay (Life Technologies p/n 32850). gDNA (10 µg) was mechanically sheared to an average size distribution of 10 kb using a Covaris gTube (Kbiosciences p/n 520079). A Bioanalyzer 2100 12K DNA Chip assay (Agilent p/n 5067-1508) was used to assess the fragment size distribution. Sheared gDNA (5 µg) was DNA-damage repaired and end-repaired using polishing enzymes. A blunt-end ligation reaction followed by exonuclease treatment was performed to create the SMRT Bell template. A Blue Pippin device (Sage Science, Inc., Beverly, MA, USA) was used to size select the SMRT Bell template and enrich for large fragments (&gt;10 kbp). The size-selected library was quality inspected and quantified on Agilent Bioanalyzer 12 kb DNA Chip and Qubit Fluorimeter (Life Technologies). A ready-to-sequence SMRT Bell-Polymerase Complex was created using a P6 DNA/Polymerase Binding Kit 2.0 (PacBio p/n 100-236-500), according to the manufacturer’s instructions. The PacBio RS II instrument was programmed to load and sequence the sample on PacBio SMRT cells v3.0 (PacBio p/n 100-171-800), acquiring one movie of 360 min per SMRT cell. A MagBead loading (PacBio p/n 100-133-600) method was chosen to improve the enrichment of the longer fragments. After the run, a sequencing report was generated for each cell via the SMRT portal, to assess adapter dimer contamination, sample-loading efficiency, average read length, and number of filtered sub-reads.</p>                            <h3 scrollto-destination=112482237 id="112482237" class="section-title" >2.4. Next-generation optical mapping</h3>
<p>A Bionano Prep™ Plant Tissue DNA Isolation Kit was used according to the Bionano Prep™ Plant Tissue DNA Isolation Base Protocol (Document number: 30068, Document revision: B, Bionano Genomics, Inc., San Diego, CA, USA). Briefly, 1 g of young leaves collected 2 months after germination was fixed in formaldehyde, to protect the nuclei and DNA against mechanical shearing. The leaves were then homogenized with a rotor-stator apparatus, to break the tough plant cell wall, followed by a density gradient purification for the recovery of intact nuclei. These samples were embedded in low-melt agarose plugs for proteinase K (Qiagen AG, Hombrechtikon, Switzerland) and RNase A (Qiagen AG, Hombrechtikon) digestion and subsequent purification of high-molecular-weight (HMW) DNA. The resulting HMW DNA was processed further according to the IrysPrep™ Labeling-NLRS User Guide (Document number: 30024, Document revision: G, Bionano Genomics) using an NLRS Kit (Bionano Genomics). In brief, 300 ng of HMW DNA was nicked by Nt.BspQI (New England Biolabs, Ipswich, MA, USA) and labeled. These reactions provided sequence specificity for an optimal label density of 8–15 labels per 100 kbp. The nick enzyme Nt.BspQI was selected by predicting the label density as 11.9 labels per 100 kbp using Bionano Genomics Knickers (ver. 1.5.5.0, <a class="link link-uri" href="http://www.bnxinstall.com/knickers/Knickers.htm" target="_blank">http://www.bnxinstall.com/knickers/Knickers.htm</a>) beforehand. The labeled nicks were repaired to restore strand integrity, and the labeled DNA was stained for backbone visualization. Labeled nicks were subsequently detected as dots on a string when run on a Bionano Irys<sup>®</sup> system instrument.</p>                            <h3 scrollto-destination=112482239 id="112482239" class="section-title" >2.5. Sequencing and optical mapping</h3>
<p>Sequencing was performed on an Illumina NextSeq500, MiSeq instrument (Illumina) and a PacBio RS II system (Pacific Bioscience of California, Inc., CA, USA). Genome optical mapping was carried out on a Bionano Irys<sup>®</sup> system (Bionano Genomics). The three paired-end libraries with insertion lengths of 250, 550, and 700 bases were sequenced on an Illumina NextSeq500 apparatus (reagent version 2 kits 2 × 150 cycles). In contrast, the four mate-pair libraries with insert sizes of 3, 5, 8, and 20 kb were sequenced on an Illumina MiSeq instrument (reagent version 3 kits 2 × 300 cycles). Long reads were obtained from PacBio RS II using a total of 23 SMRT cells. The nicked-labeled HMW DNA was run on a Bionano IrysChip<sup>®</sup> flowcell that scanned 6 times for 96.5× coverage.</p>                            <h3 scrollto-destination=112482241 id="112482241" class="section-title" >2.6. Genome size estimation</h3>
<p>Approximately 0.5 cm<sup>2</sup> of fresh leaf tissue of finger millet, together with an equal amount of leaf tissue of tomato (<em>Lycopersicon esculentum</em>; used as an internal standard [diploid genome size, 1.96 pg<a reveal-id="dsx036-B31" data-open="dsx036-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a>]), was chopped with a sharp razor blade in extraction buffer (CyStain PI Absolut P; Sysmex Europe GmbH, Norderstedt, Germany). The suspension of nuclei was filtrated and incubated in staining solution containing propidium iodide (PI) and RNase A (CyStain PI Absolut P; Sysmex). The suspension of stained nuclei was analyzed using a flow cytometer (CyFlow Space, Sysmex).</p>                            <h3 scrollto-destination=112482243 id="112482243" class="section-title" >2.7. Hybrid <em>de novo</em> assembly workflow and gene annotation</h3>
<p>The hybrid <em>de novo</em> assembly workflow is shown in <a reveal-id="dsx036-F1" data-open="dsx036-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>. Adapters that contaminated the sequenced paired-end reads and low-quality bases were trimmed using Trimmomatic (ver. 0.33)<a reveal-id="dsx036-B32" data-open="dsx036-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> on the SUSHI framework<a reveal-id="dsx036-B33" data-open="dsx036-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> after sequencing. Regarding the mate-pair libraries, first, chimera reads were filtered out, and chimera code and linker sequences from the 5′ end were trimmed using the script IlluminaChimera-Clean5.py provided by Lucigen Co., Wisconsin, USA, followed by another script, IlluminaNxSeqJunction-Split9.py, to segregate the reads into proper mate-pair reads and unsplit reads via the detection of junction sequences. Subsequently, Platanus (ver. 1.2.4)<a reveal-id="dsx036-B34" data-open="dsx036-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> was executed using the preprocessed paired-end reads to assemble contigs (<a reveal-id="dsx036-F1" data-open="dsx036-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>, step 1), which were then used as anchors to connect the PacBio long reads on DBG2OLC (commit id: 7f1712ae76b015b1f4efee81a5d4e1305701197a)<a reveal-id="dsx036-B35" data-open="dsx036-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> (<a reveal-id="dsx036-F1" data-open="dsx036-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>, step 2). Contigs from the Platanus assembly that were longer than 1 kb were used for the input of the DBG2OLC hybrid assembly. Note that many short contigs were filtered out to adopt a conservative approach aimed at avoiding misassembly, as short and fragmented contigs stemmed most probably from complex regions, such as repeats and duplicates. The contigs from the DBG2OLC hybrid assembly were polished using Pilon (ver. 1.21)<a reveal-id="dsx036-B36" data-open="dsx036-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> with the accurate Illumina short reads, to correct misassemblies (NGS contigs) (<a reveal-id="dsx036-F1" data-open="dsx036-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>, step 3). We used only half amount of the paired-end reads (250 base insert size) for the polishing. Subsequently, the polished contigs were scaffolded with the mate-pairs using SSPACE standard (ver. 3.0)<a reveal-id="dsx036-B37" data-open="dsx036-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> (NGS scaffolds) (<a reveal-id="dsx036-F1" data-open="dsx036-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>, step 4-1). We only used the proper mate-pair reads for scaffolding. Independently, optical genome mapping was performed on a Bionano Irys<sup>®</sup> system (<a reveal-id="dsx036-F1" data-open="dsx036-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>, step 4-2), followed by <em>de novo</em> assembly with the detected molecules using the IrysView<sup>®</sup> and Bionano IrysSolve<sup>®</sup> software (ver. 2.1) (BNG contigs) (<a reveal-id="dsx036-F1" data-open="dsx036-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>, step 5). The <em>de novo</em> assembly was performed twice: the first <em>de novo</em> assembly used the estimated nose parameters by mapping the detected molecules to the NGS contigs, and the second assembly used autonoise (autodetection of appropriate noise parameters). The hybrid scaffolding was performed using Bionano IrysSolve<sup>®</sup> (ver. 2.1) with both the NGS scaffolds and the BNG contigs. Finally, GMclose (ver. 1.6)<a reveal-id="dsx036-B38" data-open="dsx036-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> filled the gaps in the super-scaffolds using all the contigs assembled by Platanus. The actual executed commands as a shell script with configuration files and parameters are provided in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx036/3/dsx036_suppl_data.zip?Expires=2147483647&Signature=sUJU62LQXktRG1MSxA1tOfm80oYihgIlD3~3j6DFBg-PF7MIbxZL2BbNr87MXOg3420R9SbCs16c94jbq3Dxina1NARhf-Uq4mEvZzKPn6lwmVM7m5tk6ho1hnLHZ6JN2bjS8FV~KfE8r67EnD8GLAPUPEGsZwWEq2sxevStTFQZy2O8s943dIl1wU7pYWtkZ38frAPZ~G8tfKxhyXMfeyop~VcLT2xdgIYbKPB83FxOOim83rEqXMbojXZJ6dplzORh3kjKpl6F2a7Q2e~woJkk0ad-Inp-bpXemDGGUI4EUmV0BssMbU6NAMRvrqSS7s2-lqkUNsMd9TPf1F-C~Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary File S1</a></span>. </p>                        <a id="112482245" scrollto-destination="112482245"></a>
<div id="dsx036-F1" data-id="dsx036-F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx036/3/m_dsx036f1.png?Expires=2147483647&Signature=em1KEL9FNqAN5UATb5YbYs98eOSU0r39UL2oKyitu2LLEveBCS7DwMmkjPBsrqSnUO3h5GqTmg2QV10cZXF54LwzcfnZZ826UoQo0aSqGA693-~LDKbMRGrqsR5ITpryrvzyibO636f-QBnGGDa8jTzMsKrvqC7w~Ok1w3OyQr-0QiXNftZoc7mz06Ql3g9woWxPJKZk2S8r4WIjrg5exiB2l66l6aD5W0p984DE4QJkSm44qDLCSO8IoVWj1huKrBQeCIkAxzCKSnBrxIvAJfdrUJkVCkxCTMgicjan-yuKL0iE-5rGz-yzEt~3HdS5UyQE9oll5TSHaZkfdAY05Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Hybrid de novo assembly workflow. In the figure, the white box represents a process, the gray box represents a data entity, the blue solid arrow shows the process flow, and the black dashed arrow shows the data flow. The hybrid de novo assembly was executed as follows: (1) contig assembly using only paired-end reads (Platanus ver. 1.2.4), (2) hybrid contig assembly using contigs &gt;1 kb from the first step and long reads (DBG2OLC commit id: 7f1712ae76b015b1f4efee81a5d4e1305701197a), (3) polishing of the assembled contig with paired-end reads (Pilon ver. 1.21) (NGS contigs), (4) using the polished contigs in the two subsequent independent steps, (4-1) scaffolding of the polished contigs with mate-pair reads (NGS scaffolds), (4-2) after converting scanned molecules into a CMAP file and optical genome mapping to the NGS contigs, (5) de novo assembly of scanned molecules based on the parameters of the molecular-quality report and autonoise (autodetection of appropriate noise parameters) with the NGS contigs (BNG contigs), (6) hybrid scaffolding with both NGS scaffolds (step 4-1 output) and BNG contigs (step 5 output), and (7) gap closing (GMclose ver. 1.6) of the scaffolds with all the contigs generated in the first step (step 1 output). Finally, the workflow produced the super scaffolds. The raw data preprocesses, such as adapter trimming and removing chimera code from mate-pair reads, are not shown in this workflow." path-from-xml="dsx036f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112482245/dsx036f1.tif" path-from-xml="dsx036f1.tif" target="_blank">View large</a><a section="112482245" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx036/3/dsx036f1.png?Expires=2147483647&Signature=3HJeC4ppDaqkhvwiVtwkQzfEFxHPfxoggxBe3QJj3Z-iFthQXaslc5gDEhJ4S7f6HJQwr5hR8X71Lk5-naVJ74fO2r67MMrDvRmG31q2~a7iz~OtoyJh9IPvd5Qp0dTaiP4MlhTxr5erq-CaieJvlCMMGpjxSCVrX7pQUlwdgMvHmHrGYDGBRnHgGkZSJZg6MmoKui4SDFUlTptnxhwxJnYfb-2qOVkq6--Jy-eBY8J4ghzQXrIojIxP59Jr1mwTHaPvXuc8Yw6ikn6CGIsUuDag~cblh37rU0OJI2eDWFEMhoys97zIyR2LMWtGFiy00c25nIpBmImjVu8JEJoo7g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112482245&ar=4103403&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx036f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Hybrid <em>de novo</em> assembly workflow. In the figure, the white box represents a process, the gray box represents a data entity, the blue solid arrow shows the process flow, and the black dashed arrow shows the data flow. The hybrid <em>de novo</em> assembly was executed as follows: (1) contig assembly using only paired-end reads (Platanus ver. 1.2.4), (2) hybrid contig assembly using contigs &gt;1 kb from the first step and long reads (DBG2OLC commit id: 7f1712ae76b015b1f4efee81a5d4e1305701197a), (3) polishing of the assembled contig with paired-end reads (Pilon ver. 1.21) (NGS contigs), (4) using the polished contigs in the two subsequent independent steps, (4-1) scaffolding of the polished contigs with mate-pair reads (NGS scaffolds), (4-2) after converting scanned molecules into a CMAP file and optical genome mapping to the NGS contigs, (5) <em>de novo</em> assembly of scanned molecules based on the parameters of the molecular-quality report and autonoise (autodetection of appropriate noise parameters) with the NGS contigs (BNG contigs), (6) hybrid scaffolding with both NGS scaffolds (step 4-1 output) and BNG contigs (step 5 output), and (7) gap closing (GMclose ver. 1.6) of the scaffolds with all the contigs generated in the first step (step 1 output). Finally, the workflow produced the super scaffolds. The raw data preprocesses, such as adapter trimming and removing chimera code from mate-pair reads, are not shown in this workflow.</p></div></div><div content-id="dsx036-F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx036/3/m_dsx036f1.png?Expires=2147483647&Signature=em1KEL9FNqAN5UATb5YbYs98eOSU0r39UL2oKyitu2LLEveBCS7DwMmkjPBsrqSnUO3h5GqTmg2QV10cZXF54LwzcfnZZ826UoQo0aSqGA693-~LDKbMRGrqsR5ITpryrvzyibO636f-QBnGGDa8jTzMsKrvqC7w~Ok1w3OyQr-0QiXNftZoc7mz06Ql3g9woWxPJKZk2S8r4WIjrg5exiB2l66l6aD5W0p984DE4QJkSm44qDLCSO8IoVWj1huKrBQeCIkAxzCKSnBrxIvAJfdrUJkVCkxCTMgicjan-yuKL0iE-5rGz-yzEt~3HdS5UyQE9oll5TSHaZkfdAY05Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Hybrid de novo assembly workflow. In the figure, the white box represents a process, the gray box represents a data entity, the blue solid arrow shows the process flow, and the black dashed arrow shows the data flow. The hybrid de novo assembly was executed as follows: (1) contig assembly using only paired-end reads (Platanus ver. 1.2.4), (2) hybrid contig assembly using contigs &gt;1 kb from the first step and long reads (DBG2OLC commit id: 7f1712ae76b015b1f4efee81a5d4e1305701197a), (3) polishing of the assembled contig with paired-end reads (Pilon ver. 1.21) (NGS contigs), (4) using the polished contigs in the two subsequent independent steps, (4-1) scaffolding of the polished contigs with mate-pair reads (NGS scaffolds), (4-2) after converting scanned molecules into a CMAP file and optical genome mapping to the NGS contigs, (5) de novo assembly of scanned molecules based on the parameters of the molecular-quality report and autonoise (autodetection of appropriate noise parameters) with the NGS contigs (BNG contigs), (6) hybrid scaffolding with both NGS scaffolds (step 4-1 output) and BNG contigs (step 5 output), and (7) gap closing (GMclose ver. 1.6) of the scaffolds with all the contigs generated in the first step (step 1 output). Finally, the workflow produced the super scaffolds. The raw data preprocesses, such as adapter trimming and removing chimera code from mate-pair reads, are not shown in this workflow." path-from-xml="dsx036f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112482245/dsx036f1.tif" path-from-xml="dsx036f1.tif" target="_blank">View large</a><a section="112482245" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx036/3/dsx036f1.png?Expires=2147483647&Signature=3HJeC4ppDaqkhvwiVtwkQzfEFxHPfxoggxBe3QJj3Z-iFthQXaslc5gDEhJ4S7f6HJQwr5hR8X71Lk5-naVJ74fO2r67MMrDvRmG31q2~a7iz~OtoyJh9IPvd5Qp0dTaiP4MlhTxr5erq-CaieJvlCMMGpjxSCVrX7pQUlwdgMvHmHrGYDGBRnHgGkZSJZg6MmoKui4SDFUlTptnxhwxJnYfb-2qOVkq6--Jy-eBY8J4ghzQXrIojIxP59Jr1mwTHaPvXuc8Yw6ikn6CGIsUuDag~cblh37rU0OJI2eDWFEMhoys97zIyR2LMWtGFiy00c25nIpBmImjVu8JEJoo7g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112482245&ar=4103403&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx036f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Hybrid <em>de novo</em> assembly workflow. In the figure, the white box represents a process, the gray box represents a data entity, the blue solid arrow shows the process flow, and the black dashed arrow shows the data flow. The hybrid <em>de novo</em> assembly was executed as follows: (1) contig assembly using only paired-end reads (Platanus ver. 1.2.4), (2) hybrid contig assembly using contigs &gt;1 kb from the first step and long reads (DBG2OLC commit id: 7f1712ae76b015b1f4efee81a5d4e1305701197a), (3) polishing of the assembled contig with paired-end reads (Pilon ver. 1.21) (NGS contigs), (4) using the polished contigs in the two subsequent independent steps, (4-1) scaffolding of the polished contigs with mate-pair reads (NGS scaffolds), (4-2) after converting scanned molecules into a CMAP file and optical genome mapping to the NGS contigs, (5) <em>de novo</em> assembly of scanned molecules based on the parameters of the molecular-quality report and autonoise (autodetection of appropriate noise parameters) with the NGS contigs (BNG contigs), (6) hybrid scaffolding with both NGS scaffolds (step 4-1 output) and BNG contigs (step 5 output), and (7) gap closing (GMclose ver. 1.6) of the scaffolds with all the contigs generated in the first step (step 1 output). Finally, the workflow produced the super scaffolds. The raw data preprocesses, such as adapter trimming and removing chimera code from mate-pair reads, are not shown in this workflow.</p></div></div><p>The genes were predicted and annotated using MAKER (ver. 2.31.8)<a reveal-id="dsx036-B39" data-open="dsx036-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> based on the rice genome (<em>Oryza sativa</em> ssp<em>. japonica</em> IRGSP-1.0, ENA Assembly: GCA_001433935.1) and the RNAseq data that were obtained from young leaves. The rice cDNAs and RNAseq data for the EST evidence and the rice proteins for the protein homology evidence were used for the MAKER job. To render the GFF file, the input of MAKER, the RNAseq data were mapped to the assembled reference sequences using STAR aligner (ver. 2.5.2b)<a reveal-id="dsx036-B40" data-open="dsx036-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> and the GFF annotation file was constructed via transcriptome construction using StringTie (ver. 1.3.3b).<a reveal-id="dsx036-B41" data-open="dsx036-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a> The rice-specific repeat-making library obtained from Repbase database<a reveal-id="dsx036-B42" data-open="dsx036-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a> was used for RepeatMasker (ver. 4.0.5).<a reveal-id="dsx036-B43" data-open="dsx036-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a> The rice model was used for Augustus (ver. 3.2.0)<a reveal-id="dsx036-B44" data-open="dsx036-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a> gene prediction in the MAKER pipeline. The detail options are written in the <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx036/3/dsx036_suppl_data.zip?Expires=2147483647&Signature=sUJU62LQXktRG1MSxA1tOfm80oYihgIlD3~3j6DFBg-PF7MIbxZL2BbNr87MXOg3420R9SbCs16c94jbq3Dxina1NARhf-Uq4mEvZzKPn6lwmVM7m5tk6ho1hnLHZ6JN2bjS8FV~KfE8r67EnD8GLAPUPEGsZwWEq2sxevStTFQZy2O8s943dIl1wU7pYWtkZ38frAPZ~G8tfKxhyXMfeyop~VcLT2xdgIYbKPB83FxOOim83rEqXMbojXZJ6dplzORh3kjKpl6F2a7Q2e~woJkk0ad-Inp-bpXemDGGUI4EUmV0BssMbU6NAMRvrqSS7s2-lqkUNsMd9TPf1F-C~Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary File S1</a></span>. For assessing the quality of the genome assembly and annotation, we executed BUSCO (ver. 2.0.1)<a reveal-id="dsx036-B45" data-open="dsx036-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a> using the published plant universal single-copy orthologs. We also performed an RNAseq data alignment to the final assembled genome using STAR (ver. 2.5.2b)<a reveal-id="dsx036-B40" data-open="dsx036-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> and checked the mapping statistics.</p>                            <h2 scrollto-destination=112482247 id="112482247" class="section-title" >3. Results and discussion</h2>
<p>In this section, we report the genome assembly and discuss the results of its validation.</p>                            <h3 scrollto-destination=112482249 id="112482249" class="section-title" >3.1. Sequencing and assembly statistics</h3>
<p>Using the flow cytometry experiment, we estimated that the size of the finger millet genome is 1.5 Gb. In total, short reads amounting to 140× coverage and long reads amounting to 16× coverage were sequenced. We scanned the DNA molecules on a Bionano Irys<sup>®</sup> system and detected 144,761 Mb molecules in a total of six runs (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx036/3/dsx036_suppl_data.zip?Expires=2147483647&Signature=sUJU62LQXktRG1MSxA1tOfm80oYihgIlD3~3j6DFBg-PF7MIbxZL2BbNr87MXOg3420R9SbCs16c94jbq3Dxina1NARhf-Uq4mEvZzKPn6lwmVM7m5tk6ho1hnLHZ6JN2bjS8FV~KfE8r67EnD8GLAPUPEGsZwWEq2sxevStTFQZy2O8s943dIl1wU7pYWtkZ38frAPZ~G8tfKxhyXMfeyop~VcLT2xdgIYbKPB83FxOOim83rEqXMbojXZJ6dplzORh3kjKpl6F2a7Q2e~woJkk0ad-Inp-bpXemDGGUI4EUmV0BssMbU6NAMRvrqSS7s2-lqkUNsMd9TPf1F-C~Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). Based on the estimated genome size, the sequenced coverage depth for each library is reported in <a reveal-id="dsx036-T1" data-open="dsx036-T1" class="link link-reveal link-table xref-fig">Table 1</a> (PacBio subreads length distribution and calculated insert size of distribution are shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx036/3/dsx036_suppl_data.zip?Expires=2147483647&Signature=sUJU62LQXktRG1MSxA1tOfm80oYihgIlD3~3j6DFBg-PF7MIbxZL2BbNr87MXOg3420R9SbCs16c94jbq3Dxina1NARhf-Uq4mEvZzKPn6lwmVM7m5tk6ho1hnLHZ6JN2bjS8FV~KfE8r67EnD8GLAPUPEGsZwWEq2sxevStTFQZy2O8s943dIl1wU7pYWtkZ38frAPZ~G8tfKxhyXMfeyop~VcLT2xdgIYbKPB83FxOOim83rEqXMbojXZJ6dplzORh3kjKpl6F2a7Q2e~woJkk0ad-Inp-bpXemDGGUI4EUmV0BssMbU6NAMRvrqSS7s2-lqkUNsMd9TPf1F-C~Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figs S1 and S2</a></span>, respectively). <div content-id="dsx036-T1" class="table-modal"><div class="table-wrap"><div id="dsx036-T1" data-id="dsx036-T1" class="table-wrap-title"><div class="label -label">Table 1.</div><div class="caption"><p>Raw sequence data</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Library type </th><th>DNA|RNA </th><th>Sequencer </th><th>Insert length (bases) </th><th>Number of bases </th><th>Coverage depth </th></tr></thead><tbody><tr><td>Paired-end </td><td>DNA </td><td>Illumina NextSeq </td><td>250 </td><td>93,783,186,606 </td><td>63× </td></tr><tr><td>Paired-end </td><td>DNA </td><td>Illumina NextSeq </td><td>550 </td><td>63,519,282,198 </td><td>42× </td></tr><tr><td>Paired-end </td><td>DNA </td><td>Illumina NextSeq </td><td>700 </td><td>50,551,135,834 </td><td>34× </td></tr><tr><td>Mate-pair </td><td>DNA </td><td>Illumina MiSeq </td><td>3,000 </td><td>789,059,46 </td><td>0.053× </td></tr><tr><td>Mate-pair </td><td>DNA </td><td>Illumina MiSeq </td><td>5,000 </td><td>364,070,088 </td><td>0.24× </td></tr><tr><td>Mate-pair </td><td>DNA </td><td>Illumina MiSeq </td><td>8,000 </td><td>1,138,183,671 </td><td>0.76× </td></tr><tr><td>Mate-pair </td><td>DNA </td><td>Illumina MiSeq </td><td>20,000 </td><td>1,366,910,280 </td><td>0.91× </td></tr><tr><td>PacBio CLR </td><td>DNA </td><td>PacBio RS II </td><td>– </td><td>25,086,179,201 </td><td>16.7× </td></tr><tr><td>Paired-end </td><td>RNA </td><td>Illumina HiSeq4000 </td><td>– </td><td>148,856,244,082 </td><td>99.2× </td></tr></tbody></table></div><div content-id="dsx036-T1" class="table-modal"><table><thead name="thead" align=""><tr><th>Library type </th><th>DNA|RNA </th><th>Sequencer </th><th>Insert length (bases) </th><th>Number of bases </th><th>Coverage depth </th></tr></thead><tbody><tr><td>Paired-end </td><td>DNA </td><td>Illumina NextSeq </td><td>250 </td><td>93,783,186,606 </td><td>63× </td></tr><tr><td>Paired-end </td><td>DNA </td><td>Illumina NextSeq </td><td>550 </td><td>63,519,282,198 </td><td>42× </td></tr><tr><td>Paired-end </td><td>DNA </td><td>Illumina NextSeq </td><td>700 </td><td>50,551,135,834 </td><td>34× </td></tr><tr><td>Mate-pair </td><td>DNA </td><td>Illumina MiSeq </td><td>3,000 </td><td>789,059,46 </td><td>0.053× </td></tr><tr><td>Mate-pair </td><td>DNA </td><td>Illumina MiSeq </td><td>5,000 </td><td>364,070,088 </td><td>0.24× </td></tr><tr><td>Mate-pair </td><td>DNA </td><td>Illumina MiSeq </td><td>8,000 </td><td>1,138,183,671 </td><td>0.76× </td></tr><tr><td>Mate-pair </td><td>DNA </td><td>Illumina MiSeq </td><td>20,000 </td><td>1,366,910,280 </td><td>0.91× </td></tr><tr><td>PacBio CLR </td><td>DNA </td><td>PacBio RS II </td><td>– </td><td>25,086,179,201 </td><td>16.7× </td></tr><tr><td>Paired-end </td><td>RNA </td><td>Illumina HiSeq4000 </td><td>– </td><td>148,856,244,082 </td><td>99.2× </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>The number of sequenced raw data in each library and the coverage depth based on the genome size, 1.5 Gb as estimated by flow cytometry, are shown.</p></div></div></div></div></div><div class="table-wrap"><div id="dsx036-T1" data-id="dsx036-T1" class="table-wrap-title"><div class="label -label">Table 1.</div><div class="caption"><p>Raw sequence data</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Library type </th><th>DNA|RNA </th><th>Sequencer </th><th>Insert length (bases) </th><th>Number of bases </th><th>Coverage depth </th></tr></thead><tbody><tr><td>Paired-end </td><td>DNA </td><td>Illumina NextSeq </td><td>250 </td><td>93,783,186,606 </td><td>63× </td></tr><tr><td>Paired-end </td><td>DNA </td><td>Illumina NextSeq </td><td>550 </td><td>63,519,282,198 </td><td>42× </td></tr><tr><td>Paired-end </td><td>DNA </td><td>Illumina NextSeq </td><td>700 </td><td>50,551,135,834 </td><td>34× </td></tr><tr><td>Mate-pair </td><td>DNA </td><td>Illumina MiSeq </td><td>3,000 </td><td>789,059,46 </td><td>0.053× </td></tr><tr><td>Mate-pair </td><td>DNA </td><td>Illumina MiSeq </td><td>5,000 </td><td>364,070,088 </td><td>0.24× </td></tr><tr><td>Mate-pair </td><td>DNA </td><td>Illumina MiSeq </td><td>8,000 </td><td>1,138,183,671 </td><td>0.76× </td></tr><tr><td>Mate-pair </td><td>DNA </td><td>Illumina MiSeq </td><td>20,000 </td><td>1,366,910,280 </td><td>0.91× </td></tr><tr><td>PacBio CLR </td><td>DNA </td><td>PacBio RS II </td><td>– </td><td>25,086,179,201 </td><td>16.7× </td></tr><tr><td>Paired-end </td><td>RNA </td><td>Illumina HiSeq4000 </td><td>– </td><td>148,856,244,082 </td><td>99.2× </td></tr></tbody></table></div><div content-id="dsx036-T1" class="table-modal"><table><thead name="thead" align=""><tr><th>Library type </th><th>DNA|RNA </th><th>Sequencer </th><th>Insert length (bases) </th><th>Number of bases </th><th>Coverage depth </th></tr></thead><tbody><tr><td>Paired-end </td><td>DNA </td><td>Illumina NextSeq </td><td>250 </td><td>93,783,186,606 </td><td>63× </td></tr><tr><td>Paired-end </td><td>DNA </td><td>Illumina NextSeq </td><td>550 </td><td>63,519,282,198 </td><td>42× </td></tr><tr><td>Paired-end </td><td>DNA </td><td>Illumina NextSeq </td><td>700 </td><td>50,551,135,834 </td><td>34× </td></tr><tr><td>Mate-pair </td><td>DNA </td><td>Illumina MiSeq </td><td>3,000 </td><td>789,059,46 </td><td>0.053× </td></tr><tr><td>Mate-pair </td><td>DNA </td><td>Illumina MiSeq </td><td>5,000 </td><td>364,070,088 </td><td>0.24× </td></tr><tr><td>Mate-pair </td><td>DNA </td><td>Illumina MiSeq </td><td>8,000 </td><td>1,138,183,671 </td><td>0.76× </td></tr><tr><td>Mate-pair </td><td>DNA </td><td>Illumina MiSeq </td><td>20,000 </td><td>1,366,910,280 </td><td>0.91× </td></tr><tr><td>PacBio CLR </td><td>DNA </td><td>PacBio RS II </td><td>– </td><td>25,086,179,201 </td><td>16.7× </td></tr><tr><td>Paired-end </td><td>RNA </td><td>Illumina HiSeq4000 </td><td>– </td><td>148,856,244,082 </td><td>99.2× </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>The number of sequenced raw data in each library and the coverage depth based on the genome size, 1.5 Gb as estimated by flow cytometry, are shown.</p></div></div></div></div></p><p>The statistics of each step in the hybrid assembly are shown in <a reveal-id="dsx036-T2" data-open="dsx036-T2" class="link link-reveal link-table xref-fig">Table 2</a>. The total assembled size was 1,188,784,944 bases, which is 78.2% of the genome size estimated by flow cytometry. The hybrid contig assembly by DBG2OLC and the scaffolding by SSPACE enhanced the continuity from the Platanus contig assembly, although the mate-pair sequenced coverage was less than 10×, and the total assembly size decreased compared with the Platanus contig assembly. This could be explained by the low sequencing coverage of the PacBio long reads, rather than the filtering out of the shorter Platanus contigs. At the final hybrid scaffolding and gap closing, the N50 of the final super-scaffolds increased to 2.68 Mb, which is approximately twice as long as the NGS scaffolds. The hybrid scaffolding generated super-scaffolds that concatenated the NGS scaffolds based on the BNG contigs that were assembled from the optical genome mapping. Some conflicts were detected between NGS scaffolds and BNG contigs, which split the conflicting NGS scaffolds into smaller pieces. This had the major advantage of correcting the polyploid genome assembly, as scaffolding with only mate-pair short reads may fail to solve the long-range homeologous complexity and may sometimes lead to mis-scaffolding.<a reveal-id="dsx036-B46" data-open="dsx036-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a><sup>,</sup><a reveal-id="dsx036-B47" data-open="dsx036-B47" class="link link-ref link-reveal xref-bibr"><sup>47</sup></a><div content-id="dsx036-T2" class="table-modal"><div class="table-wrap"><div id="dsx036-T2" data-id="dsx036-T2" class="table-wrap-title"><div class="label -label">Table 2.</div><div class="caption"><p>Assembly statistics at each assembly step</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Statistics </th><th>Contig assembly </th><th>Hybrid assembly </th><th>Scaffold </th><th>Hybrid scaffold </th></tr></thead><tbody><tr><td>Total sequences </td><td>2,812,919 </td><td>6,374 </td><td>2,387 </td><td>1,897 </td></tr><tr><td>Total bases </td><td>1,307,217,455 </td><td>1,067,045,564 </td><td>1,069,478,541 </td><td>1,188,784,944 </td></tr><tr><td>Min sequence length </td><td>115 </td><td>3,408 </td><td>3,911 </td><td>1,244 </td></tr><tr><td>Max sequence length </td><td>27,802 </td><td>2,072,336 </td><td>5,237,708 </td><td>13,553,037 </td></tr><tr><td>Average sequence length </td><td>464.72 </td><td>167,405.96 </td><td>448,042.96 </td><td>626,665.76 </td></tr><tr><td>Median sequence length </td><td>154.00 </td><td>102,084.50 </td><td>237,045.00 </td><td>92,178.00 </td></tr><tr><td>N25 length </td><td>3,419 </td><td>520,874 </td><td>1,581,824 </td><td>5,029,714 </td></tr><tr><td>N50 length </td><td>1,410 </td><td>285,549 </td><td>905,318 </td><td>2,683,090 </td></tr><tr><td>N75 length </td><td>311 </td><td>145,770 </td><td>457,768 </td><td>1,232,573 </td></tr><tr><td>N90 length </td><td>133 </td><td>76,519 </td><td>216,067 </td><td>419,121 </td></tr><tr><td>(G + C)s </td><td>43.15% </td><td>43.83% </td><td>43.57% </td><td>40.98% </td></tr><tr><td>Ns </td><td>0.00% </td><td>0.00% </td><td>0.73% </td><td>6.63% </td></tr></tbody></table></div><div content-id="dsx036-T2" class="table-modal"><table><thead name="thead" align=""><tr><th>Statistics </th><th>Contig assembly </th><th>Hybrid assembly </th><th>Scaffold </th><th>Hybrid scaffold </th></tr></thead><tbody><tr><td>Total sequences </td><td>2,812,919 </td><td>6,374 </td><td>2,387 </td><td>1,897 </td></tr><tr><td>Total bases </td><td>1,307,217,455 </td><td>1,067,045,564 </td><td>1,069,478,541 </td><td>1,188,784,944 </td></tr><tr><td>Min sequence length </td><td>115 </td><td>3,408 </td><td>3,911 </td><td>1,244 </td></tr><tr><td>Max sequence length </td><td>27,802 </td><td>2,072,336 </td><td>5,237,708 </td><td>13,553,037 </td></tr><tr><td>Average sequence length </td><td>464.72 </td><td>167,405.96 </td><td>448,042.96 </td><td>626,665.76 </td></tr><tr><td>Median sequence length </td><td>154.00 </td><td>102,084.50 </td><td>237,045.00 </td><td>92,178.00 </td></tr><tr><td>N25 length </td><td>3,419 </td><td>520,874 </td><td>1,581,824 </td><td>5,029,714 </td></tr><tr><td>N50 length </td><td>1,410 </td><td>285,549 </td><td>905,318 </td><td>2,683,090 </td></tr><tr><td>N75 length </td><td>311 </td><td>145,770 </td><td>457,768 </td><td>1,232,573 </td></tr><tr><td>N90 length </td><td>133 </td><td>76,519 </td><td>216,067 </td><td>419,121 </td></tr><tr><td>(G + C)s </td><td>43.15% </td><td>43.83% </td><td>43.57% </td><td>40.98% </td></tr><tr><td>Ns </td><td>0.00% </td><td>0.00% </td><td>0.73% </td><td>6.63% </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn2"><div class="fn"><p>Assembly statistics at each step are summarized. Contig assembly represents the contigs that were assembled using Platanus with preprocessed paired-end reads. Hybrid assembly represents the contigs that were generated using DBG2OLC with the Platanus contigs (&gt; 1 kb long) and PacBio long reads. Scaffold is the result of the SSPACE standard scaffolding with mate-pair reads, and Hybrid scaffold is the hybrid scaffolding result with both Bionano Genomics de novo assembled contigs and the SSPACE scaffolds, followed by GMclose gap closing.</p></div></div></div></div></div><div class="table-wrap"><div id="dsx036-T2" data-id="dsx036-T2" class="table-wrap-title"><div class="label -label">Table 2.</div><div class="caption"><p>Assembly statistics at each assembly step</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Statistics </th><th>Contig assembly </th><th>Hybrid assembly </th><th>Scaffold </th><th>Hybrid scaffold </th></tr></thead><tbody><tr><td>Total sequences </td><td>2,812,919 </td><td>6,374 </td><td>2,387 </td><td>1,897 </td></tr><tr><td>Total bases </td><td>1,307,217,455 </td><td>1,067,045,564 </td><td>1,069,478,541 </td><td>1,188,784,944 </td></tr><tr><td>Min sequence length </td><td>115 </td><td>3,408 </td><td>3,911 </td><td>1,244 </td></tr><tr><td>Max sequence length </td><td>27,802 </td><td>2,072,336 </td><td>5,237,708 </td><td>13,553,037 </td></tr><tr><td>Average sequence length </td><td>464.72 </td><td>167,405.96 </td><td>448,042.96 </td><td>626,665.76 </td></tr><tr><td>Median sequence length </td><td>154.00 </td><td>102,084.50 </td><td>237,045.00 </td><td>92,178.00 </td></tr><tr><td>N25 length </td><td>3,419 </td><td>520,874 </td><td>1,581,824 </td><td>5,029,714 </td></tr><tr><td>N50 length </td><td>1,410 </td><td>285,549 </td><td>905,318 </td><td>2,683,090 </td></tr><tr><td>N75 length </td><td>311 </td><td>145,770 </td><td>457,768 </td><td>1,232,573 </td></tr><tr><td>N90 length </td><td>133 </td><td>76,519 </td><td>216,067 </td><td>419,121 </td></tr><tr><td>(G + C)s </td><td>43.15% </td><td>43.83% </td><td>43.57% </td><td>40.98% </td></tr><tr><td>Ns </td><td>0.00% </td><td>0.00% </td><td>0.73% </td><td>6.63% </td></tr></tbody></table></div><div content-id="dsx036-T2" class="table-modal"><table><thead name="thead" align=""><tr><th>Statistics </th><th>Contig assembly </th><th>Hybrid assembly </th><th>Scaffold </th><th>Hybrid scaffold </th></tr></thead><tbody><tr><td>Total sequences </td><td>2,812,919 </td><td>6,374 </td><td>2,387 </td><td>1,897 </td></tr><tr><td>Total bases </td><td>1,307,217,455 </td><td>1,067,045,564 </td><td>1,069,478,541 </td><td>1,188,784,944 </td></tr><tr><td>Min sequence length </td><td>115 </td><td>3,408 </td><td>3,911 </td><td>1,244 </td></tr><tr><td>Max sequence length </td><td>27,802 </td><td>2,072,336 </td><td>5,237,708 </td><td>13,553,037 </td></tr><tr><td>Average sequence length </td><td>464.72 </td><td>167,405.96 </td><td>448,042.96 </td><td>626,665.76 </td></tr><tr><td>Median sequence length </td><td>154.00 </td><td>102,084.50 </td><td>237,045.00 </td><td>92,178.00 </td></tr><tr><td>N25 length </td><td>3,419 </td><td>520,874 </td><td>1,581,824 </td><td>5,029,714 </td></tr><tr><td>N50 length </td><td>1,410 </td><td>285,549 </td><td>905,318 </td><td>2,683,090 </td></tr><tr><td>N75 length </td><td>311 </td><td>145,770 </td><td>457,768 </td><td>1,232,573 </td></tr><tr><td>N90 length </td><td>133 </td><td>76,519 </td><td>216,067 </td><td>419,121 </td></tr><tr><td>(G + C)s </td><td>43.15% </td><td>43.83% </td><td>43.57% </td><td>40.98% </td></tr><tr><td>Ns </td><td>0.00% </td><td>0.00% </td><td>0.73% </td><td>6.63% </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn2"><div class="fn"><p>Assembly statistics at each step are summarized. Contig assembly represents the contigs that were assembled using Platanus with preprocessed paired-end reads. Hybrid assembly represents the contigs that were generated using DBG2OLC with the Platanus contigs (&gt; 1 kb long) and PacBio long reads. Scaffold is the result of the SSPACE standard scaffolding with mate-pair reads, and Hybrid scaffold is the hybrid scaffolding result with both Bionano Genomics de novo assembled contigs and the SSPACE scaffolds, followed by GMclose gap closing.</p></div></div></div></div></p><p>Currently, the so-called <em>de novo</em> assembly method based on next-generation and third-generation sequencing technologies is classified mainly into two approaches: (i) de Bruijn graph (DBG) assembly and (ii) overlap-layout consensus (OLC) assembly.<a reveal-id="dsx036-B48" data-open="dsx036-B48" class="link link-ref link-reveal xref-bibr"><sup>48</sup></a><sup>,</sup><a reveal-id="dsx036-B49" data-open="dsx036-B49" class="link link-ref link-reveal xref-bibr"><sup>49</sup></a> Although the DBG method is more efficient computationally regarding both memory and calculation time, the fragment size parameter, usually called <em>k</em>-mer length, critically influences the results and often causes a misassembly because of the small fragment <em>k</em>-mers on homologous or repetitive genome regions. Some assemblers, such as Platanus,<a reveal-id="dsx036-B34" data-open="dsx036-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> solve this problem by using multiple <em>k</em>-mer sizes for the construction of the DBG. As another solution for the homeologous complexity, longer reads produced from a single-molecule sequencer combined with the OLC method may be beneficial. However, high-coverage sequencing of long reads may solve and correct the wrongly sequenced bases, albeit at a high cost. The hybrid <em>de novo</em> assembly described in this article, which uses both accurate short reads and inaccurate long reads with a low coverage depth, represents another solution regarding assembly cost and calculation efficiency; moreover, the beneficial feature of long reads can be applied to solve the homeologous complexity encountered during polyploid genome assembly. The classification of reads via homeolog phasing<a reveal-id="dsx036-B50" data-open="dsx036-B50" class="link link-ref link-reveal xref-bibr"><sup>50</sup></a> based on long reads or the parental diploid genome may improve the polyploid assembly further. In addition, further longer molecule information from such as Bionano Irys<sup>®</sup> system is very promising, not only for super-scaffolding, but also for solving the long-range homeologous complexity in polyploid genome assembly, as demonstrated by the results reported in this article.</p>                            <h3 scrollto-destination=112482253 id="112482253" class="section-title" >3.2. Gene annotation and assembly validation</h3>
<p>After the hybrid assembly, we conducted gene annotation prediction using MAKER and 62,348 genes were predicted. The distribution of Annotation Edit Distance (AED)<a reveal-id="dsx036-B39" data-open="dsx036-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> tagged by MAKER is shown in <a reveal-id="dsx036-F2" data-open="dsx036-F2" class="link link-reveal link-table xref-fig">Fig. 2A</a>, in which more than 91% of the annotated genes (57,066 genes among a total of 62,348 genes) show an AED &lt;0.5. This result indicates that the annotation is well supported by the evidence. For the assessment of the quality of the assembly and annotation, the assembled scaffolds and predicted genes were analyzed on BUSCO (<a reveal-id="dsx036-F2" data-open="dsx036-F2" class="link link-reveal link-table xref-fig">Fig. 2B</a>). In the assembled scaffolds (shown in <a reveal-id="dsx036-F2" data-open="dsx036-F2" class="link link-reveal link-table xref-fig">Fig. 2B</a>, ‘genome’), 96.5% of the universal single-copy genes (1,389 genes among a total of 1,440 genes) were identified, supporting the quality of the assembly. Among the 1,389 single-copy genes detected, 606 genes were detected as being complete single-copy genes, 783 genes were found to be duplicates, and 766 of the 783 duplicates were exactly twice found from two different scaffolds. An NCBI BLAST search of the complete single-copy genes (606) that were detected only once in the assembled genome found an additional 556 homologous partial genes with a shorter alignment length (40% of the 1,389 complete single-copy genes detected). This indicates that the majority of the homeologous genes were assembled separately (see section 3.3). In tetraploid species, two copies may be found for each ‘single-copy gene’, although duplicated genes may be gradually lost because of redundancy (called nonfunctionalization or homeolog loss).<a reveal-id="dsx036-B51" data-open="dsx036-B51" class="link link-ref link-reveal xref-bibr"><sup>51</sup></a><sup>,</sup><a reveal-id="dsx036-B52" data-open="dsx036-B52" class="link link-ref link-reveal xref-bibr"><sup>52</sup></a> Similar results were obtained from the BUSCO analysis of proteins and transcripts (<a reveal-id="dsx036-F2" data-open="dsx036-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>, ‘proteins’ and ‘transcripts’), but the missing and fragmented proportion was increased compared with the ‘genome’ analysis. It is well known that one of the homeologs may not be expressed (or silenced) after polyploidization, because of redundancy; thus, some genes were not annotated and the missing and fragmented proportion increased among transcripts and proteins because of the lack of evidence. Moreover, we searched the predicted proteins in the Pfam database<a reveal-id="dsx036-B53" data-open="dsx036-B53" class="link link-ref link-reveal xref-bibr"><sup>53</sup></a> and found that 74% of predicted proteins were present in this database (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx036/3/dsx036_suppl_data.zip?Expires=2147483647&Signature=sUJU62LQXktRG1MSxA1tOfm80oYihgIlD3~3j6DFBg-PF7MIbxZL2BbNr87MXOg3420R9SbCs16c94jbq3Dxina1NARhf-Uq4mEvZzKPn6lwmVM7m5tk6ho1hnLHZ6JN2bjS8FV~KfE8r67EnD8GLAPUPEGsZwWEq2sxevStTFQZy2O8s943dIl1wU7pYWtkZ38frAPZ~G8tfKxhyXMfeyop~VcLT2xdgIYbKPB83FxOOim83rEqXMbojXZJ6dplzORh3kjKpl6F2a7Q2e~woJkk0ad-Inp-bpXemDGGUI4EUmV0BssMbU6NAMRvrqSS7s2-lqkUNsMd9TPf1F-C~Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>). RNAseq data obtained using other tissues and conditions may improve the gene annotation process. </p>                        <a id="112482255" scrollto-destination="112482255"></a>
<div id="dsx036-F2" data-id="dsx036-F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx036/3/m_dsx036f2.png?Expires=2147483647&Signature=sCEwtCJ65hI2-ixCs~TEoDT1UcNLPNCDlDKtn85KYv-rFHVNW9a8Z6xo-dK0Mo41bWdQFSriFJokN1fzGtOgMwF71B3YumMKCSuu0p1GNMkgOKbK~AxVfqd~uIQxLrBgRTnJK6tMPWcmvLgmFH7DEpPtaOaztJBKVmZEr2qVuaDGQ9FcyOhBkLV44S60kvmJs1LOl-AXWKvj5rwdTcvnmh8Q1V61T4PyvSTbOy~wbNgxNVoy0zuyOzDem5hO8h9G7A4sgWZc9hTrdEtkH4yT3brm-vgNj5lKufna0ruNDko5Pf6priZVUKowUB1pv1DcHZeTqWN7HHIVlbjoqPDh7A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Annotation edit distance (AED) and benchmark of universal single-copy orthologs. (A) The distribution of AED tagged as a gene by MAKER. (B) Benchmark result of the plant set of universal single-copy orthologs produced by BUSCO. In each bar, the pale-blue color indicates the number of genes that were detected completely as single-copy genes in the genome, the dark-blue color indicates the number of genes that were detected as single-copy genes but were counted more than twice, the yellow color indicates the number of genes that were detected as single-copy genes but not completely, and the red color indicates the number of single-copy genes that were not detected among the plant universal single-copy orthologs. The ‘genome’ is the BUSCO result that was obtained by searching the assembled genome FASTA file, and the ‘transcripts’ and ‘proteins’ constitute the BUSCO result that was obtained by searching the transcript FASTA file and amino acid FASTA file produced by MAKER." path-from-xml="dsx036f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112482255/dsx036f2.tif" path-from-xml="dsx036f2.tif" target="_blank">View large</a><a section="112482255" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx036/3/dsx036f2.png?Expires=2147483647&Signature=k5xRtRADUhq942QXjbuFcyEtpp-ctPLThBX5h8V5DE0kPHafobjVj5sHBCtrMMLclx0fnztyzZRDHHiUdYNIFoSOo3PPQBW~1gEq6-2DuWk9RlBTRM01-~Y9U0~xim-j5ysT4PUf7ZtuKETZLfVrabodGzzPCVVVYRy1HQGvISjX-Byg8K74oILh54hKz6d6BoRPv3rYdbbG0VBB-4O0JEiVFSkcWxsiLDiCKL5i4s6XiI0RCrDXqvh33Hdg8PDTHyHv3yYnHFXsG6QrfjNm4rlcwCp~zo6LJoDugpVfnIKdgxTIuEdRFsfkxvzdze5JrQg-k2lHzqtYciam2YP1hw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112482255&ar=4103403&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx036f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Annotation edit distance (AED) and benchmark of universal single-copy orthologs. (A) The distribution of AED tagged as a gene by MAKER. (B) Benchmark result of the plant set of universal single-copy orthologs produced by BUSCO. In each bar, the pale-blue color indicates the number of genes that were detected completely as single-copy genes in the genome, the dark-blue color indicates the number of genes that were detected as single-copy genes but were counted more than twice, the yellow color indicates the number of genes that were detected as single-copy genes but not completely, and the red color indicates the number of single-copy genes that were not detected among the plant universal single-copy orthologs. The ‘genome’ is the BUSCO result that was obtained by searching the assembled genome FASTA file, and the ‘transcripts’ and ‘proteins’ constitute the BUSCO result that was obtained by searching the transcript FASTA file and amino acid FASTA file produced by MAKER.</p></div></div><div content-id="dsx036-F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx036/3/m_dsx036f2.png?Expires=2147483647&Signature=sCEwtCJ65hI2-ixCs~TEoDT1UcNLPNCDlDKtn85KYv-rFHVNW9a8Z6xo-dK0Mo41bWdQFSriFJokN1fzGtOgMwF71B3YumMKCSuu0p1GNMkgOKbK~AxVfqd~uIQxLrBgRTnJK6tMPWcmvLgmFH7DEpPtaOaztJBKVmZEr2qVuaDGQ9FcyOhBkLV44S60kvmJs1LOl-AXWKvj5rwdTcvnmh8Q1V61T4PyvSTbOy~wbNgxNVoy0zuyOzDem5hO8h9G7A4sgWZc9hTrdEtkH4yT3brm-vgNj5lKufna0ruNDko5Pf6priZVUKowUB1pv1DcHZeTqWN7HHIVlbjoqPDh7A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Annotation edit distance (AED) and benchmark of universal single-copy orthologs. (A) The distribution of AED tagged as a gene by MAKER. (B) Benchmark result of the plant set of universal single-copy orthologs produced by BUSCO. In each bar, the pale-blue color indicates the number of genes that were detected completely as single-copy genes in the genome, the dark-blue color indicates the number of genes that were detected as single-copy genes but were counted more than twice, the yellow color indicates the number of genes that were detected as single-copy genes but not completely, and the red color indicates the number of single-copy genes that were not detected among the plant universal single-copy orthologs. The ‘genome’ is the BUSCO result that was obtained by searching the assembled genome FASTA file, and the ‘transcripts’ and ‘proteins’ constitute the BUSCO result that was obtained by searching the transcript FASTA file and amino acid FASTA file produced by MAKER." path-from-xml="dsx036f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112482255/dsx036f2.tif" path-from-xml="dsx036f2.tif" target="_blank">View large</a><a section="112482255" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx036/3/dsx036f2.png?Expires=2147483647&Signature=k5xRtRADUhq942QXjbuFcyEtpp-ctPLThBX5h8V5DE0kPHafobjVj5sHBCtrMMLclx0fnztyzZRDHHiUdYNIFoSOo3PPQBW~1gEq6-2DuWk9RlBTRM01-~Y9U0~xim-j5ysT4PUf7ZtuKETZLfVrabodGzzPCVVVYRy1HQGvISjX-Byg8K74oILh54hKz6d6BoRPv3rYdbbG0VBB-4O0JEiVFSkcWxsiLDiCKL5i4s6XiI0RCrDXqvh33Hdg8PDTHyHv3yYnHFXsG6QrfjNm4rlcwCp~zo6LJoDugpVfnIKdgxTIuEdRFsfkxvzdze5JrQg-k2lHzqtYciam2YP1hw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112482255&ar=4103403&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx036f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Annotation edit distance (AED) and benchmark of universal single-copy orthologs. (A) The distribution of AED tagged as a gene by MAKER. (B) Benchmark result of the plant set of universal single-copy orthologs produced by BUSCO. In each bar, the pale-blue color indicates the number of genes that were detected completely as single-copy genes in the genome, the dark-blue color indicates the number of genes that were detected as single-copy genes but were counted more than twice, the yellow color indicates the number of genes that were detected as single-copy genes but not completely, and the red color indicates the number of single-copy genes that were not detected among the plant universal single-copy orthologs. The ‘genome’ is the BUSCO result that was obtained by searching the assembled genome FASTA file, and the ‘transcripts’ and ‘proteins’ constitute the BUSCO result that was obtained by searching the transcript FASTA file and amino acid FASTA file produced by MAKER.</p></div></div><p>Furthermore, we aligned the data from six RNAseq sequenced libraries to the assembled genome for validation: 95.9% of the reads were mapped, which indicates that the assembled genome may cover more than 95% of the coding region (STAR alignment statistics is shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx036/3/dsx036_suppl_data.zip?Expires=2147483647&Signature=sUJU62LQXktRG1MSxA1tOfm80oYihgIlD3~3j6DFBg-PF7MIbxZL2BbNr87MXOg3420R9SbCs16c94jbq3Dxina1NARhf-Uq4mEvZzKPn6lwmVM7m5tk6ho1hnLHZ6JN2bjS8FV~KfE8r67EnD8GLAPUPEGsZwWEq2sxevStTFQZy2O8s943dIl1wU7pYWtkZ38frAPZ~G8tfKxhyXMfeyop~VcLT2xdgIYbKPB83FxOOim83rEqXMbojXZJ6dplzORh3kjKpl6F2a7Q2e~woJkk0ad-Inp-bpXemDGGUI4EUmV0BssMbU6NAMRvrqSS7s2-lqkUNsMd9TPf1F-C~Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>).</p>                            <h3 scrollto-destination=112482257 id="112482257" class="section-title" >3.3. Polyploid assembly validation and phylogenetic analysis</h3>
<p>To validate the quality of the polyploid assembly, i.e. to determine whether the homeologs were assembled in different scaffolds, we analyzed our assembly data of three genes for which Sanger sequencing data of <em>E. coracana</em> had been already reported. First, we performed an NCBI BLAST (ver. 2.2.31+)<a reveal-id="dsx036-B54" data-open="dsx036-B54" class="link link-ref link-reveal xref-bibr"><sup>54</sup></a> search of the published low-copy-number marker phosphoenolpyruvate carboxylase 4 (<em>Pepc4</em>)<a reveal-id="dsx036-B1" data-open="dsx036-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> and granule-bound starch synthase I-like (<em>waxy</em>)<a reveal-id="dsx036-B2" data-open="dsx036-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> in the assembled scaffolds. The reciprocal BLAST best-hit search with a very low <em>E</em>-value threshold identified exactly two scaffolds with two different completely matching sequences from different clades (the NCBI BLAST search result is shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx036/3/dsx036_suppl_data.zip?Expires=2147483647&Signature=sUJU62LQXktRG1MSxA1tOfm80oYihgIlD3~3j6DFBg-PF7MIbxZL2BbNr87MXOg3420R9SbCs16c94jbq3Dxina1NARhf-Uq4mEvZzKPn6lwmVM7m5tk6ho1hnLHZ6JN2bjS8FV~KfE8r67EnD8GLAPUPEGsZwWEq2sxevStTFQZy2O8s943dIl1wU7pYWtkZ38frAPZ~G8tfKxhyXMfeyop~VcLT2xdgIYbKPB83FxOOim83rEqXMbojXZJ6dplzORh3kjKpl6F2a7Q2e~woJkk0ad-Inp-bpXemDGGUI4EUmV0BssMbU6NAMRvrqSS7s2-lqkUNsMd9TPf1F-C~Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary File S2</a></span>). Second, we conducted a molecular phylogenetic analysis using MEGA4 (ver. 7.0.26)<a reveal-id="dsx036-B55" data-open="dsx036-B55" class="link link-ref link-reveal xref-bibr"><sup>55</sup></a> based on the detected low-copy-number homeologs with the published sequences of close relatives<a reveal-id="dsx036-B1" data-open="dsx036-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx036/3/dsx036_suppl_data.zip?Expires=2147483647&Signature=sUJU62LQXktRG1MSxA1tOfm80oYihgIlD3~3j6DFBg-PF7MIbxZL2BbNr87MXOg3420R9SbCs16c94jbq3Dxina1NARhf-Uq4mEvZzKPn6lwmVM7m5tk6ho1hnLHZ6JN2bjS8FV~KfE8r67EnD8GLAPUPEGsZwWEq2sxevStTFQZy2O8s943dIl1wU7pYWtkZ38frAPZ~G8tfKxhyXMfeyop~VcLT2xdgIYbKPB83FxOOim83rEqXMbojXZJ6dplzORh3kjKpl6F2a7Q2e~woJkk0ad-Inp-bpXemDGGUI4EUmV0BssMbU6NAMRvrqSS7s2-lqkUNsMd9TPf1F-C~Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>). As shown in these figures, the homeologs detected were clearly separated in different clusters. This result indicates that, in these gene regions, the homeologs were successfully assembled, regardless of the presence of highly homologous base sequences. Moreover, the trees supported the contention that <em>E. indica</em> is the wild diploid progenitor of the A genome, because one of the homeologs clustered with the sequences of <em>E. indica</em>.</p><p>Next, we analyzed <em>EcNAC1</em>, which is the best-studied gene of <em>E. coracana</em> related to the drought response.<a reveal-id="dsx036-B10" data-open="dsx036-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> NAC is one of the largest families of plant-specific transcription factors and has been shown to be involved in diverse processes, such as growth, development, and abiotic stresses. The <em>EcNAC1</em> of the <em>E. coracana</em> GPU28 variety was induced by drought stresses and conferred tolerance to various abiotic stresses, such as simulated osmotic stress by polyethylene glycol (<em>PEG</em>) and mannitol and salinity stresses in the overexpression lines of tobacco.<a reveal-id="dsx036-B10" data-open="dsx036-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> Based on our assembly of PR202, we found a sequence that was similar to, and clustered with, the <em>EcNAC1</em> of GPU28, together with another sequence from the Trichy1 variety (GenBank accession ID: KU500625.1) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx036/3/dsx036_suppl_data.zip?Expires=2147483647&Signature=sUJU62LQXktRG1MSxA1tOfm80oYihgIlD3~3j6DFBg-PF7MIbxZL2BbNr87MXOg3420R9SbCs16c94jbq3Dxina1NARhf-Uq4mEvZzKPn6lwmVM7m5tk6ho1hnLHZ6JN2bjS8FV~KfE8r67EnD8GLAPUPEGsZwWEq2sxevStTFQZy2O8s943dIl1wU7pYWtkZ38frAPZ~G8tfKxhyXMfeyop~VcLT2xdgIYbKPB83FxOOim83rEqXMbojXZJ6dplzORh3kjKpl6F2a7Q2e~woJkk0ad-Inp-bpXemDGGUI4EUmV0BssMbU6NAMRvrqSS7s2-lqkUNsMd9TPf1F-C~Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span>). The assembly also had another copy, which is likely to be a homeolog in a different scaffold. It would be valuable to study this copy, which may well have drought-related functions.</p><p>In addition, we searched homologous genes as ‘duplicated genes’ in the predicted transcripts using NCBI BLAST and found 57,913 genes that had more than two copies in the genome (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx036/3/dsx036_suppl_data.zip?Expires=2147483647&Signature=sUJU62LQXktRG1MSxA1tOfm80oYihgIlD3~3j6DFBg-PF7MIbxZL2BbNr87MXOg3420R9SbCs16c94jbq3Dxina1NARhf-Uq4mEvZzKPn6lwmVM7m5tk6ho1hnLHZ6JN2bjS8FV~KfE8r67EnD8GLAPUPEGsZwWEq2sxevStTFQZy2O8s943dIl1wU7pYWtkZ38frAPZ~G8tfKxhyXMfeyop~VcLT2xdgIYbKPB83FxOOim83rEqXMbojXZJ6dplzORh3kjKpl6F2a7Q2e~woJkk0ad-Inp-bpXemDGGUI4EUmV0BssMbU6NAMRvrqSS7s2-lqkUNsMd9TPf1F-C~Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary File S3</a></span>). Among these duplicates, we found 34,678 genes as reciprocal BLAST best-hit genes located in different scaffolds. Assuming that these genes are homeologs, we found that 56% of all 62,348 genes were considered homeologs, which is consistent with the ratio (55% (766/1389)) of duplicated single-copy genes detected exactly twice in BUSCO (the analytical process and results are shown as a shell script in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx036/3/dsx036_suppl_data.zip?Expires=2147483647&Signature=sUJU62LQXktRG1MSxA1tOfm80oYihgIlD3~3j6DFBg-PF7MIbxZL2BbNr87MXOg3420R9SbCs16c94jbq3Dxina1NARhf-Uq4mEvZzKPn6lwmVM7m5tk6ho1hnLHZ6JN2bjS8FV~KfE8r67EnD8GLAPUPEGsZwWEq2sxevStTFQZy2O8s943dIl1wU7pYWtkZ38frAPZ~G8tfKxhyXMfeyop~VcLT2xdgIYbKPB83FxOOim83rEqXMbojXZJ6dplzORh3kjKpl6F2a7Q2e~woJkk0ad-Inp-bpXemDGGUI4EUmV0BssMbU6NAMRvrqSS7s2-lqkUNsMd9TPf1F-C~Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary File S3</a></span>). To check the location of these homeologs, we calculated the gene density (the number of genes per Mb) and the homeolog density (the number of homeologs per Mb) in each scaffold (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx036/3/dsx036_suppl_data.zip?Expires=2147483647&Signature=sUJU62LQXktRG1MSxA1tOfm80oYihgIlD3~3j6DFBg-PF7MIbxZL2BbNr87MXOg3420R9SbCs16c94jbq3Dxina1NARhf-Uq4mEvZzKPn6lwmVM7m5tk6ho1hnLHZ6JN2bjS8FV~KfE8r67EnD8GLAPUPEGsZwWEq2sxevStTFQZy2O8s943dIl1wU7pYWtkZ38frAPZ~G8tfKxhyXMfeyop~VcLT2xdgIYbKPB83FxOOim83rEqXMbojXZJ6dplzORh3kjKpl6F2a7Q2e~woJkk0ad-Inp-bpXemDGGUI4EUmV0BssMbU6NAMRvrqSS7s2-lqkUNsMd9TPf1F-C~Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6</a></span> A1–2, B1–2). Panels A1 and B1 of <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx036/3/dsx036_suppl_data.zip?Expires=2147483647&Signature=sUJU62LQXktRG1MSxA1tOfm80oYihgIlD3~3j6DFBg-PF7MIbxZL2BbNr87MXOg3420R9SbCs16c94jbq3Dxina1NARhf-Uq4mEvZzKPn6lwmVM7m5tk6ho1hnLHZ6JN2bjS8FV~KfE8r67EnD8GLAPUPEGsZwWEq2sxevStTFQZy2O8s943dIl1wU7pYWtkZ38frAPZ~G8tfKxhyXMfeyop~VcLT2xdgIYbKPB83FxOOim83rEqXMbojXZJ6dplzORh3kjKpl6F2a7Q2e~woJkk0ad-Inp-bpXemDGGUI4EUmV0BssMbU6NAMRvrqSS7s2-lqkUNsMd9TPf1F-C~Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6</a></span> show the density against the scaffold length of all genes and homeologs, respectively, and panels A2 and B2 show the density distribution of all genes and homeologs, respectively, as well. The genome-wide gene density was 52.4 genes per Mb and the genome-wide homeolog density was 29.2 homeologs per Mb. As shown in these figures, all genes and homeologs were distributed similarly over the genome. Nonetheless, there was a small number of scaffolds (&gt;1 kb long) with high homeolog density (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx036/3/dsx036_suppl_data.zip?Expires=2147483647&Signature=sUJU62LQXktRG1MSxA1tOfm80oYihgIlD3~3j6DFBg-PF7MIbxZL2BbNr87MXOg3420R9SbCs16c94jbq3Dxina1NARhf-Uq4mEvZzKPn6lwmVM7m5tk6ho1hnLHZ6JN2bjS8FV~KfE8r67EnD8GLAPUPEGsZwWEq2sxevStTFQZy2O8s943dIl1wU7pYWtkZ38frAPZ~G8tfKxhyXMfeyop~VcLT2xdgIYbKPB83FxOOim83rEqXMbojXZJ6dplzORh3kjKpl6F2a7Q2e~woJkk0ad-Inp-bpXemDGGUI4EUmV0BssMbU6NAMRvrqSS7s2-lqkUNsMd9TPf1F-C~Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6</a></span>, B1), which indicates that there are areas that are dense in homeologs compared with the genome-wide homeolog distribution.</p>                            <h3 scrollto-destination=112482261 id="112482261" class="section-title" >3.4. Summary of the assembly of the genome of polyploid finger millet</h3>
<p>Over half of the world’s freshwater resources are currently being used for irrigating agricultural crops. The domestic and industrial demands of the burgeoning population have led to an unprecedented withdrawal of freshwater.<a reveal-id="dsx036-B56" data-open="dsx036-B56" class="link link-ref link-reveal xref-bibr"><sup>56</sup></a> Combined with the erratic pattern of rainfall caused by the ensuing global climate change, agriculture production is expected to be severely affected in the years to come.<a reveal-id="dsx036-B57" data-open="dsx036-B57" class="link link-ref link-reveal xref-bibr"><sup>57</sup></a> This scenario urgently necessitates the cultivation of more water-productive crops and/or the development of improved cultivars with a superior water-use efficiency. Millets, with their C<sub>4</sub> photosynthetic metabolism, are generally considered to possess higher levels of drought tolerance and are generally cultivated under rain-fed conditions. Because of their higher resilience to harsh climatic conditions, millets exhibit a wider adaptation, and hence can be cultivated in a wider range of agro-ecosystems. Therefore, millets currently represent an integral component of human diets in most impoverished areas, especially in the tropical and subtropical regions of Africa and Southeast Asia.</p><p>We assembled the genome of the allotetraploid finger millet (<em>Eleusine coracana</em> (L.) Gaertn, variety: PR202, IC: 479099) via a novel pipeline using various types of sequencers. The size of the assembled genome was about 1.2 Gb, while the genome size measured by flow cytometry was 1.5 Gb. Considering that the majority of the grass genomes are likely to be composed of repetitive elements, the coverage of the genome is sufficiently high. MAKER predicted 62,348 genes, which is roughly the double of the 35,679 genes identified in diploid rice (<em>Oryza sativa Japonica</em> IRGSP-1.0); this is also consistent with a good coverage of the genome. Three lines of evidence supported the good coverage. First, we provided a polyploid assembly with a long scaffold size (N50 length &gt;2.5 Mb), while the assembly of polyploid genomes tends to be fragmented. It is true that the N50 may not mean correct assembly and scaffolding, but we avoided too-greedy scaffolding. Second, 96.5% of the 1,440 universal single-copy orthologs were identified, and 95.9% of RNAseq reads were mapped to the assembled genome, which supported the adequateness of the assembly. Third, the majority of these universal sets had two copies, suggesting the separation of homeologs was adequate in our assembly strategy. Concomitantly, however, we found a significant number of genes that had a single copy. There are several likely explanations for this finding. (i) One of the duplicated genes tends to be lost because of redundancy. Polyploidization was estimated to have occurred well before the domestication of millet (0.50–2.71 mya), in contrast with other crops, in which polyploidization occurred through domestication during the last thousand years.<a reveal-id="dsx036-B58" data-open="dsx036-B58" class="link link-ref link-reveal xref-bibr"><sup>58</sup></a><sup>,</sup><a reveal-id="dsx036-B59" data-open="dsx036-B59" class="link link-ref link-reveal xref-bibr"><sup>59</sup></a> Thus, it is possible that many duplicated genes were lost. (ii) The published plant universal single-copy gene set is not collected by random sampling from the entire plant genes in terms of the rate of gene loss. Despite the frequent occurrence of genome duplication events throughout angiosperms, these genes tend to be lost and become single copy, potentially because of dosage balance or redundancy. (iii) The assembly size is less than the genome size estimated by flow cytometry. Most of the lacking sequences are likely to be repetitive sequences, but it is also likely that parts of the genic regions were not assembled. (iv) The assembler could not deal with two homeologs; thus, one copy was missing or two copies were combined. We verified that the two copies of the <em>Pepc4</em> gene, <em>waxy</em> and <em>NAC</em>, were assembled separately. Currently, not many genome sequences from diploid <em>E. indica</em> are available, preventing the systematic validation of homeologs. Further improvement of assembly algorithms and validation would be valuable in this field of research. It is still possible to improve the assembly and annotation of this genome. Nevertheless, the allotetraploid finger millet genome information reported here is expected to be of much value for further genomic research of this millet at the molecular level, as well as for studies aimed at improving finger millet productivity and bioavailability. We have successfully shown the applicability of the multiple hybrid <em>de novo</em> assembly workflow to a complex allotetraploid genome.</p><p>Recently, the genome assembly of the <em>E. coracana</em> ML-365 variety was elucidated using Illumina and SOLiD sequencing data.<a reveal-id="dsx036-B60" data-open="dsx036-B60" class="link link-ref link-reveal xref-bibr"><sup>60</sup></a> We emphasize that our data and analyses were targeted at the analysis of complex polyploid genomes. The N50 of our assembly (&gt;2.5 Mb) was two orders of magnitude higher than that of ML-365 (about 24 kb) and would allow RNAseq analysis of each homeolog separately, as well as resequencing analyses. Our results illustrate the strength of single-molecule sequencing and optical genome mapping in assembling polyploid genomes.</p>                            <h2 scrollto-destination=112482265 id="112482265" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We wish to thank Dr. Scott Monsma at Lucigen Co., Wisconsin, USA, for mate-pair library preparation, Carmen Näf at the University of Zurich for RNA extraction and library preparation, Aki Morishima for plant growth and genome size estimation by flow cytometry, Dr. Misako Yamazaki and Dr. Lucas Mohn at the University of Zurich for sample handling, and Dr. Dario Copetti at the University of Arizona for advice on gene annotation, and Dr. Jun Sese and Dr. Tony Kuo at the National Institute of Advanced Industrial Science and Technology for discussion about polyploid genome assembly. We also thank Dr. Lilian Gilgen and Damaris O’Brien at the Indo-Swiss Collaboration in Biotechnology (ISCB) office at the Federal Institute of Technology in Lausanne (EPFL), Nangsa Kamtzi and Isavel Schöchli at the University of Zurich, and Lydia Imhof and Jana Ploszaj at the Functional Genomics Center Zurich for the administrative support. Moreover, we would like to thank the members of the Evolutionary and Ecological Genomics Group at the University of Zurich and the members of Genomics/Transcriptomics Group at the Functional Genomics Center Zurich for useful discussions and suggestions.</p>                            <h3 scrollto-destination=112482267 id="112482267" class="section-title" >Availability</h3>
<p>All sequenced raw data in FASTQ format, scanned molecular data in BNX format, <em>de novo</em> assembly results by Bionano IrysSolve<sup>®</sup> in CMAP format, and the final scaffolds and annotation in FASTA and GFF format are available in the DDBJ Sequence Read Archive (DRA) database with the BioProject accession number: PRJDB5606 and BioSample number: SAMD00076255.</p>                            <h3 scrollto-destination=112482269 id="112482269" class="section-title" >Supplementary data</h3>
<p><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx036/3/dsx036_suppl_data.zip?Expires=2147483647&Signature=sUJU62LQXktRG1MSxA1tOfm80oYihgIlD3~3j6DFBg-PF7MIbxZL2BbNr87MXOg3420R9SbCs16c94jbq3Dxina1NARhf-Uq4mEvZzKPn6lwmVM7m5tk6ho1hnLHZ6JN2bjS8FV~KfE8r67EnD8GLAPUPEGsZwWEq2sxevStTFQZy2O8s943dIl1wU7pYWtkZ38frAPZ~G8tfKxhyXMfeyop~VcLT2xdgIYbKPB83FxOOim83rEqXMbojXZJ6dplzORh3kjKpl6F2a7Q2e~woJkk0ad-Inp-bpXemDGGUI4EUmV0BssMbU6NAMRvrqSS7s2-lqkUNsMd9TPf1F-C~Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary data</a></span> are available at <em>DNARES</em> online.</p>                            <h3 scrollto-destination=112482271 id="112482271" class="section-title" >Conflict of interest</h3>
<p>John Baeten and Kees-Jan Francoijs are employees of Bionano Genomics.</p>                            <h3 scrollto-destination=112482273 id="112482273" class="section-title" >Accession numbers</h3>
<p>BioProject accession number: PRJDB5606</p><p>BioSample number: SAMD00076255, DRA accession number: DRA005897</p>                            <h3 scrollto-destination=112482276 id="112482276" class="section-title" >Funding</h3>
<p>This project is funded by the Indo-Swiss Collaboration in Biotechnology (ISCB), a JST CREST Grant (number JPMJCR16O3), and a KAKENHI Grant (number 16H06469, 16K21727), Japan.</p>                            <h2 scrollto-destination=112482278 id="112482278" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsx036-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx036-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Liu</div> <div class="given-names">Q.</div></div>, <div class="name string-name"><div class="surname">Triplett</div> <div class="given-names">J. K.</div></div>, <div class="name string-name"><div class="surname">Wen</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Peterson</div> <div class="given-names">P. M.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Allotetraploid origin and divergence in Eleusine (Chloridoideae, Poaceae): evidence from low-copy nuclear gene phylogenies and a plastid gene chronogram</div>. <div class="source">Ann. Bot</div> ., <div class="volume">108</div>, <div class="fpage">1287</div>–<div class="lpage">98</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Allotetraploid%20origin%20and%20divergence%20in%20Eleusine%20(Chloridoideae%2C%20Poaceae)%3A%20evidence%20from%20low-copy%20nuclear%20gene%20phylogenies%20and%20a%20plastid%20gene%20chronogram&amp;author=Q.%20Liu&amp;author=J.%20K.%20Triplett&amp;author=J.%20Wen&amp;author=P.%20M.%20Peterson&amp;publication_year=2011&amp;journal=Ann.%20Bot&amp;volume=108&amp;pages=1287-98" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/aob/mcr231" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Faob%2Fmcr231" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Faob%2Fmcr231"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21880659" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx036-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx036-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Liu</div> <div class="given-names">Q.</div></div>, <div class="name string-name"><div class="surname">Jiang</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Wen</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Peterson</div> <div class="given-names">P. M.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Low-copy nuclear gene and McGISH resolves polyploid history of <em>Eleusine coracana</em> and morphological character evolution in Eleusine</div>. <div class="source">Turkish J. Bot</div> ., <div class="volume">38</div>, <div class="fpage">1</div>–<div class="lpage">12</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Low-copy%20nuclear%20gene%20and%20McGISH%20resolves%20polyploid%20history%20of%20Eleusine%20coracana%20and%20morphological%20character%20evolution%20in%20Eleusine&amp;author=Q.%20Liu&amp;author=B.%20Jiang&amp;author=J.%20Wen&amp;author=P.%20M.%20Peterson&amp;publication_year=2014&amp;journal=Turkish%20J.%20Bot&amp;volume=38&amp;pages=1-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3906/bot-1305-12" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3906%2Fbot-1305-12" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3906%2Fbot-1305-12"> </span></div> </div></div></div></div></div><div content-id="dsx036-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx036-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bisht</div> <div class="given-names">M. S.</div></div>, <div class="name string-name"><div class="surname">Mukai</div> <div class="given-names">Y.</div></div></div> <div class="year">2001</div>, 
<div class="article-title">Genomic in situ hybridization identifies genome donor of finger millet (<em>Eleusine coracana</em>)</div>. <div class="source">Theor. Appl. Genet</div> ., <div class="volume">102</div>, <div class="fpage">825</div>–<div class="lpage">32</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20in%20situ%20hybridization%20identifies%20genome%20donor%20of%20finger%20millet%20(Eleusine%20coracana)&amp;author=M.%20S.%20Bisht&amp;author=Y.%20Mukai&amp;publication_year=2001&amp;journal=Theor.%20Appl.%20Genet&amp;volume=102&amp;pages=825-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s001220000497" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs001220000497" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs001220000497"> </span></div> </div></div></div></div></div><div content-id="dsx036-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx036-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hilu</div> <div class="given-names">K. W.</div></div></div> <div class="year">1988</div>, 
<div class="article-title">Identification of the “A” genome of finger millet using chloroplast DNA</div>. <div class="source">Genetics</div> , <div class="volume">118</div>, <div class="fpage">163</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20the%20%E2%80%9CA%E2%80%9D%20genome%20of%20finger%20millet%20using%20chloroplast%20DNA&amp;author=K.%20W.%20Hilu&amp;publication_year=1988&amp;journal=Genetics&amp;volume=118&amp;pages=163-7" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8608927" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx036-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx036-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chandra</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Chandra</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Arora</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Sharma</div> <div class="given-names">A. K.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Review of finger millet (<em>Eleusine coracana</em> (L.) Gaertn): a power house of health benefiting nutrients</div>. <div class="source">Food Sci. Human Wellness</div> , <div class="volume">5</div>, <div class="fpage">149</div>–<div class="lpage">55</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Review%20of%20finger%20millet%20(Eleusine%20coracana%20(L.)%20Gaertn)%3A%20a%20power%20house%20of%20health%20benefiting%20nutrients&amp;author=D.%20Chandra&amp;author=S.%20Chandra&amp;author=P.%20Arora&amp;author=A.%20K.%20Sharma&amp;publication_year=2016&amp;journal=Food%20Sci.%20Human%20Wellness&amp;volume=5&amp;pages=149-55" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.fshw.2016.05.004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.fshw.2016.05.004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.fshw.2016.05.004"> </span></div> </div></div></div></div></div><div content-id="dsx036-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx036-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Saleh</div> <div class="given-names">A. S.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">Q.</div></div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Shen</div> <div class="given-names">Q.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Millet grains: nutritional quality, processing, and potential health benefits</div>. <div class="source">Compr. Rev. Food Sci. Food Safety</div> , <div class="volume">12</div>, <div class="fpage">281</div>–<div class="lpage">95</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Millet%20grains%3A%20nutritional%20quality%2C%20processing%2C%20and%20potential%20health%20benefits&amp;author=A.%20S.%20Saleh&amp;author=Q.%20Zhang&amp;author=J.%20Chen&amp;author=Q.%20Shen&amp;publication_year=2013&amp;journal=Compr.%20Rev.%20Food%20Sci.%20Food%20Safety&amp;volume=12&amp;pages=281-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/crf3.2013.12.issue-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fcrf3.2013.12.issue-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fcrf3.2013.12.issue-3"> </span></div> </div></div></div></div></div><div content-id="dsx036-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx036-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Goron</div> <div class="given-names">T. L.</div></div>, <div class="name string-name"><div class="surname">Raizada</div> <div class="given-names">M. N.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Genetic diversity and genomic resources available for the small millet crops to accelerate a New Green Revolution</div>. <div class="source">Front. Plant Sci</div> ., <div class="volume">6</div>, <div class="fpage">157</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20diversity%20and%20genomic%20resources%20available%20for%20the%20small%20millet%20crops%20to%20accelerate%20a%20New%20Green%20Revolution&amp;author=T.%20L.%20Goron&amp;author=M.%20N.%20Raizada&amp;publication_year=2015&amp;journal=Front.%20Plant%20Sci&amp;volume=6&amp;pages=157" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25852710" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx036-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx036-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Armstead</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Huang</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Ravagnani</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Robson</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Ougham</div> <div class="given-names">H.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Bioinformatics in the orphan crops</div>. <div class="source">Brief. Bioinform</div> ., <div class="volume">10</div>, <div class="fpage">645</div>–<div class="lpage">53</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bioinformatics%20in%20the%20orphan%20crops&amp;author=I.%20Armstead&amp;author=L.%20Huang&amp;author=A.%20Ravagnani&amp;author=P.%20Robson&amp;author=H.%20Ougham&amp;publication_year=2009&amp;journal=Brief.%20Bioinform&amp;volume=10&amp;pages=645-53" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bib/bbp036" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbib%2Fbbp036" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbib%2Fbbp036"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19734255" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx036-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx036-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Mysore</div> <div class="given-names">K. S.</div></div>, <div class="name string-name"><div class="surname">Baird</div> <div class="given-names">V.</div></div></div> <div class="year">1997</div>, 
<div class="article-title">Nuclear DNA content in species of Eleusine (Gramineae): a critical re-evaluation using laser flow cytometry</div>. <div class="source">Plant Syst. Evol</div> ., <div class="volume">207</div>, <div class="fpage">1</div>–<div class="lpage">11</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nuclear%20DNA%20content%20in%20species%20of%20Eleusine%20(Gramineae)%3A%20a%20critical%20re-evaluation%20using%20laser%20flow%20cytometry&amp;author=K.%20S.%20Mysore&amp;author=V.%20Baird&amp;publication_year=1997&amp;journal=Plant%20Syst.%20Evol&amp;volume=207&amp;pages=1-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00985206" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00985206" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00985206"> </span></div> </div></div></div></div></div><div content-id="dsx036-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx036-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ramegowda</div> <div class="given-names">V.</div></div>, <div class="name string-name"><div class="surname">Senthil-Kumar</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Nataraja</div> <div class="given-names">K. N.</div></div>, <div class="name string-name"><div class="surname">Reddy</div> <div class="given-names">M. K.</div></div>, <div class="name string-name"><div class="surname">Mysore</div> <div class="given-names">K. S.</div></div>, <div class="name string-name"><div class="surname">Udayakumar</div> <div class="given-names">M.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Expression of a finger millet transcription factor, <em>EcNAC1</em>, in tobacco confers abiotic stress-tolerance</div>. <div class="source">PLoS One</div> , <div class="volume">7</div>, <div class="fpage">e40397</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Expression%20of%20a%20finger%20millet%20transcription%20factor%2C%20EcNAC1%2C%20in%20tobacco%20confers%20abiotic%20stress-tolerance&amp;author=V.%20Ramegowda&amp;author=M.%20Senthil-Kumar&amp;author=K.%20N.%20Nataraja&amp;author=M.%20K.%20Reddy&amp;author=K.%20S.%20Mysore&amp;author=M.%20Udayakumar&amp;publication_year=2012&amp;journal=PLoS%20One&amp;volume=7&amp;pages=e40397" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0040397" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0040397" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0040397"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22808152" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx036-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx036-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Parvathi</div> <div class="given-names">M. S.</div></div>, <div class="name string-name"><div class="surname">Nataraja</div> <div class="given-names">K. N.</div></div>, <div class="name string-name"><div class="surname">Yashoda</div> <div class="given-names">B. K.</div></div>, <div class="name string-name"><div class="surname">Ramegowda</div> <div class="given-names">H. V.</div></div>, <div class="name string-name"><div class="surname">Mamrutha</div> <div class="given-names">H. M.</div></div>, <div class="name string-name"><div class="surname">Rama</div> <div class="given-names">N.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Expression analysis of stress responsive pathway genes linked to drought hardiness in an adapted crop, finger millet (<em>Eleusine coracana</em>)</div>. <div class="source">J. Plant Biochem. Biotechnol</div> ., <div class="volume">22</div>, <div class="fpage">193</div>–<div class="lpage">201</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Expression%20analysis%20of%20stress%20responsive%20pathway%20genes%20linked%20to%20drought%20hardiness%20in%20an%20adapted%20crop%2C%20finger%20millet%20(Eleusine%20coracana)&amp;author=M.%20S.%20Parvathi&amp;author=K.%20N.%20Nataraja&amp;author=B.%20K.%20Yashoda&amp;author=H.%20V.%20Ramegowda&amp;author=H.%20M.%20Mamrutha&amp;author=N.%20Rama&amp;publication_year=2013&amp;journal=J.%20Plant%20Biochem.%20Biotechnol&amp;volume=22&amp;pages=193-201" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s13562-012-0135-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs13562-012-0135-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs13562-012-0135-0"> </span></div> </div></div></div></div></div><div content-id="dsx036-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx036-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Nagarjuna</div> <div class="given-names">K. N.</div></div>, <div class="name string-name"><div class="surname">Parvathi</div> <div class="given-names">M. S.</div></div>, <div class="name string-name"><div class="surname">Sajeevan</div> <div class="given-names">R. S.</div></div>, <div class="name string-name"><div class="surname">Pruthvi</div> <div class="given-names">V.</div></div>, <div class="name string-name"><div class="surname">Mamrutha</div> <div class="given-names">H. M.</div></div>, <div class="name string-name"><div class="surname">Nataraja</div> <div class="given-names">K. N.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Full-length cloning and characterization of abiotic stress responsive CIPK31-like gene from finger millet, a drought-tolerant crop</div>. <div class="source">Curr. Sci</div> ., <div class="volume">111</div>, <div class="fpage">890</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Full-length%20cloning%20and%20characterization%20of%20abiotic%20stress%20responsive%20CIPK31-like%20gene%20from%20finger%20millet%2C%20a%20drought-tolerant%20crop&amp;author=K.%20N.%20Nagarjuna&amp;author=M.%20S.%20Parvathi&amp;author=R.%20S.%20Sajeevan&amp;author=V.%20Pruthvi&amp;author=H.%20M.%20Mamrutha&amp;author=K.%20N.%20Nataraja&amp;publication_year=2016&amp;journal=Curr.%20Sci&amp;volume=111&amp;pages=890" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.18520/cs/v111/i5/890-894" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.18520%2Fcs%2Fv111%2Fi5%2F890-894" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.18520%2Fcs%2Fv111%2Fi5%2F890-894"> </span></div> </div></div></div></div></div><div content-id="dsx036-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx036-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Rahman</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Jagadeeshselvam</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Valarmathi</div> <div class="given-names">R.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Transcriptome analysis of salinity responsiveness in contrasting genotypes of finger millet (<em>Eleusine coracana</em> L.) through RNA-sequencing</div>. <div class="source">Plant Mol. Biol</div> ., <div class="volume">85</div>, <div class="fpage">485</div>–<div class="lpage">503</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptome%20analysis%20of%20salinity%20responsiveness%20in%20contrasting%20genotypes%20of%20finger%20millet%20(Eleusine%20coracana%20L.)%20through%20RNA-sequencing&amp;author=H.%20Rahman&amp;author=N.%20Jagadeeshselvam&amp;author=R.%20Valarmathi&amp;publication_year=2014&amp;journal=Plant%20Mol.%20Biol&amp;volume=85&amp;pages=485-503" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11103-014-0199-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11103-014-0199-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11103-014-0199-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24838653" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx036-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx036-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Dida</div> <div class="given-names">M. M.</div></div>, <div class="name string-name"><div class="surname">Srinivasachary</div></div>, <div class="name string-name"><div class="surname">Ramakrishnan</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Bennetzen</div> <div class="given-names">J. L.</div></div>, <div class="name string-name"><div class="surname">Gale</div> <div class="given-names">M. D.</div></div>, <div class="name string-name"><div class="surname">Devos</div> <div class="given-names">K. M.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">The genetic map of finger millet, Eleusine coracana</div>. <div class="source">Theor. Appl. Genet</div> ., <div class="volume">114</div>, <div class="fpage">321</div>–<div class="lpage">32</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genetic%20map%20of%20finger%20millet%2C%20Eleusine%20coracana&amp;author=M.%20M.%20Dida&amp;author=%20Srinivasachary&amp;author=S.%20Ramakrishnan&amp;author=J.%20L.%20Bennetzen&amp;author=M.%20D.%20Gale&amp;author=K.%20M.%20Devos&amp;publication_year=2006&amp;journal=Theor.%20Appl.%20Genet&amp;volume=114&amp;pages=321-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-006-0435-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-006-0435-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-006-0435-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17103137" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx036-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx036-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wood</div> <div class="given-names">T. E.</div></div>, <div class="name string-name"><div class="surname">Takebayashi</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Barker</div> <div class="given-names">M. S.</div></div>, <div class="name string-name"><div class="surname">Mayrose</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Greenspoon</div> <div class="given-names">P. B.</div></div>, <div class="name string-name"><div class="surname">Rieseberg</div> <div class="given-names">L. H.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">The frequency of polyploid speciation in vascular plants</div>. <div class="source">Proc. Natl. Acad. Sci</div> ., <div class="volume">106</div>, <div class="fpage">13875</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20frequency%20of%20polyploid%20speciation%20in%20vascular%20plants&amp;author=T.%20E.%20Wood&amp;author=N.%20Takebayashi&amp;author=M.%20S.%20Barker&amp;author=I.%20Mayrose&amp;author=P.%20B.%20Greenspoon&amp;author=L.%20H.%20Rieseberg&amp;publication_year=2009&amp;journal=Proc.%20Natl.%20Acad.%20Sci&amp;volume=106&amp;pages=13875-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0811575106" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0811575106" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0811575106"> </span></div> </div></div></div></div></div><div content-id="dsx036-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx036-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Mayrose</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Zhan</div> <div class="given-names">S. H.</div></div>, <div class="name string-name"><div class="surname">Rothfels</div> <div class="given-names">C. J.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">Recently formed polyploid plants diversify at lower rates</div>. <div class="source">Science</div> , <div class="volume">333</div>, <div class="fpage">1257</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Recently%20formed%20polyploid%20plants%20diversify%20at%20lower%20rates&amp;author=I.%20Mayrose&amp;author=S.%20H.%20Zhan&amp;author=C.%20J.%20Rothfels&amp;publication_year=2011&amp;journal=Science&amp;volume=333&amp;pages=1257" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1207205" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1207205" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1207205"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21852456" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx036-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx036-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Leitch</div> <div class="given-names">A. R.</div></div>, <div class="name string-name"><div class="surname">Leitch</div> <div class="given-names">I. J.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">Genomic plasticity and the diversity of polyploid plants</div>. <div class="source">Science</div> , <div class="volume">320</div>, <div class="fpage">481</div>–<div class="lpage">3</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20plasticity%20and%20the%20diversity%20of%20polyploid%20plants&amp;author=A.%20R.%20Leitch&amp;author=I.%20J.%20Leitch&amp;publication_year=2008&amp;journal=Science&amp;volume=320&amp;pages=481-3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1153585" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1153585" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1153585"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18436776" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx036-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx036-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bodily</div> <div class="given-names">P. M.</div></div>, <div class="name string-name"><div class="surname">Fujimoto</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Ortega</div> <div class="given-names">C.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Heterozygous genome assembly via binary classification of homologous sequence</div>. <div class="source">BMC Bioinform</div> ., <div class="volume">16</div>, <div class="fpage">S5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Heterozygous%20genome%20assembly%20via%20binary%20classification%20of%20homologous%20sequence&amp;author=P.%20M.%20Bodily&amp;author=M.%20Fujimoto&amp;author=C.%20Ortega&amp;publication_year=2015&amp;journal=BMC%20Bioinform&amp;volume=16&amp;pages=S5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-16-S7-S5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-16-S7-S5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-16-S7-S5"> </span></div> </div></div></div></div></div><div content-id="dsx036-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx036-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Pryszcz</div> <div class="given-names">L. P.</div></div>, <div class="name string-name"><div class="surname">Gabaldón</div> <div class="given-names">T.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Redundans: an assembly pipeline for highly heterozygous genomes</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">44</div>, <div class="fpage">e113–3</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Redundans%3A%20an%20assembly%20pipeline%20for%20highly%20heterozygous%20genomes&amp;author=L.%20P.%20Pryszcz&amp;author=T.%20Gabald%C3%B3n&amp;publication_year=2016&amp;journal=Nucleic%20Acids%20Res&amp;volume=44&amp;pages=e113–3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkw294" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkw294" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkw294"> </span></div> </div></div></div></div></div><div content-id="dsx036-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx036-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sierro</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Battey</div> <div class="given-names">J. N. D.</div></div>, <div class="name string-name"><div class="surname">Ouadi</div> <div class="given-names">S.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">The tobacco genome sequence and its comparison with those of tomato and potato</div>. <div class="source">Nat. Commun</div> ., <div class="volume">5</div>, <div class="fpage">3833</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20tobacco%20genome%20sequence%20and%20its%20comparison%20with%20those%20of%20tomato%20and%20potato&amp;author=N.%20Sierro&amp;author=J.%20N.%20D.%20Battey&amp;author=S.%20Ouadi&amp;publication_year=2014&amp;journal=Nat.%20Commun&amp;volume=5&amp;pages=3833" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ncomms4833" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fncomms4833" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fncomms4833"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24807620" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx036-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx036-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Hu</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Jiang</div> <div class="given-names">W.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Sequencing of allotetraploid cotton (<em>Gossypium hirsutum</em> L. acc. TM-1) provides a resource for fiber improvement</div>. <div class="source">Nat. Biotechnol</div> ., <div class="volume">33</div>, <div class="fpage">531</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequencing%20of%20allotetraploid%20cotton%20(Gossypium%20hirsutum%20L.%20acc.%20TM-1)%20provides%20a%20resource%20for%20fiber%20improvement&amp;author=T.%20Zhang&amp;author=Y.%20Hu&amp;author=W.%20Jiang&amp;publication_year=2015&amp;journal=Nat.%20Biotechnol&amp;volume=33&amp;pages=531-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.3207" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.3207" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.3207"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25893781" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx036-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx036-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Mayer</div> <div class="given-names">K. F. X.</div></div>, <div class="name string-name"><div class="surname">Rogers</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Dole el</div> <div class="given-names">J.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">A chromosome-based draft sequence of the hexaploid bread wheat (<em>Triticum aestivum</em>) genome</div>. <div class="source">Science</div> , <div class="volume">345</div>, <div class="fpage">1251788</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20chromosome-based%20draft%20sequence%20of%20the%20hexaploid%20bread%20wheat%20(Triticum%20aestivum)%20genome&amp;author=K.%20F.%20X.%20Mayer&amp;author=J.%20Rogers&amp;author=J.%20Dole%20el&amp;publication_year=2014&amp;journal=Science&amp;volume=345&amp;pages=1251788-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1251788" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1251788" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1251788"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25035500" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx036-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx036-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yang</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">X.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">The genome sequence of allopolyploid <em>Brassica juncea</em> and analysis of differential homoeolog gene expression influencing selection</div>. <div class="source">Nat. Genet</div> ., <div class="volume">48</div>, <div class="fpage">1225</div>–<div class="lpage">32</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20sequence%20of%20allopolyploid%20Brassica%20juncea%20and%20analysis%20of%20differential%20homoeolog%20gene%20expression%20influencing%20selection&amp;author=J.%20Yang&amp;author=D.%20Liu&amp;author=X.%20Wang&amp;publication_year=2016&amp;journal=Nat.%20Genet&amp;volume=48&amp;pages=1225-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.3657" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.3657" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.3657"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27595476" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx036-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx036-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Paape</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Hatakeyama</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Shimizu-Inatsugi</div> <div class="given-names">R.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">Conserved but attenuated parental gene expression in allopolyploids: constitutive zinc hyperaccumulation in the allotetraploid <em>Arabidopsis kamchatica</em></div>. <div class="source">Mol. Biol. Evol</div> ., <div class="volume">33</div>, <div class="fpage">2781</div>–<div class="lpage">800</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Conserved%20but%20attenuated%20parental%20gene%20expression%20in%20allopolyploids%3A%20constitutive%20zinc%20hyperaccumulation%20in%20the%20allotetraploid%20Arabidopsis%20kamchatica&amp;author=T.%20Paape&amp;author=M.%20Hatakeyama&amp;author=R.%20Shimizu-Inatsugi&amp;publication_year=2016&amp;journal=Mol.%20Biol.%20Evol&amp;volume=33&amp;pages=2781-800" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msw141" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsw141" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsw141"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27413047" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx036-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx036-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Akama</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Shimizu-Inatsugi</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Shimizu</div> <div class="given-names">K. K.</div></div>, <div class="name string-name"><div class="surname">Sese</div> <div class="given-names">J.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Genome-wide quantification of homeolog expression ratio revealed nonstochastic gene regulation in synthetic allopolyploid Arabidopsis</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">42</div>, <div class="fpage">e46</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20quantification%20of%20homeolog%20expression%20ratio%20revealed%20nonstochastic%20gene%20regulation%20in%20synthetic%20allopolyploid%20Arabidopsis&amp;author=S.%20Akama&amp;author=R.%20Shimizu-Inatsugi&amp;author=K.%20K.%20Shimizu&amp;author=J.%20Sese&amp;publication_year=2014&amp;journal=Nucleic%20Acids%20Res&amp;volume=42&amp;pages=e46" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkt1376" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkt1376" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkt1376"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24423873" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx036-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx036-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Pootakham</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Sonthirod</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Naktang</div> <div class="given-names">C.</div></div></div>, et al.   <div class="year">2017</div>, 
<div class="article-title">De novo hybrid assembly of the rubber tree genome reveals evidence of paleotetraploidy in Hevea species</div>. <div class="source">Sci. Reports</div> , <div class="volume">7</div>, <div class="fpage">41457</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=De%20novo%20hybrid%20assembly%20of%20the%20rubber%20tree%20genome%20reveals%20evidence%20of%20paleotetraploidy%20in%20Hevea%20species&amp;author=W.%20Pootakham&amp;author=C.%20Sonthirod&amp;author=C.%20Naktang&amp;publication_year=2017&amp;journal=Sci.%20Reports&amp;volume=7&amp;pages=41457" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/srep41457" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsrep41457" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsrep41457"> </span></div> </div></div></div></div></div><div content-id="dsx036-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx036-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lieberman-Aiden</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">van Berkum</div> <div class="given-names">N. L.</div></div>, <div class="name string-name"><div class="surname">Williams</div> <div class="given-names">L.</div></div></div>, et al.   <div class="year">2009</div>, 
<div class="article-title">Comprehensive mapping of long-range interactions reveals folding principles of the human genome</div>. <div class="source">Science (New York, N.Y.)</div> , <div class="volume">326</div>, <div class="fpage">289</div>–<div class="lpage">93</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comprehensive%20mapping%20of%20long-range%20interactions%20reveals%20folding%20principles%20of%20the%20human%20genome&amp;author=E.%20Lieberman-Aiden&amp;author=N.%20L.%20van%20Berkum&amp;author=L.%20Williams&amp;publication_year=2009&amp;journal=Science%20(New%20York%2C%20N.Y.)&amp;volume=326&amp;pages=289-93" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1181369" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1181369" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1181369"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19815776" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx036-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx036-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Mascher</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Gundlach</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Himmelbach</div> <div class="given-names">A.</div></div></div>, et al.   <div class="year">2017</div>, 
<div class="article-title">A chromosome conformation capture ordered sequence of the barley genome</div>. <div class="source">Nature</div> , <div class="volume">544</div>, <div class="fpage">427</div>–<div class="lpage">33</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20chromosome%20conformation%20capture%20ordered%20sequence%20of%20the%20barley%20genome&amp;author=M.%20Mascher&amp;author=H.%20Gundlach&amp;author=A.%20Himmelbach&amp;publication_year=2017&amp;journal=Nature&amp;volume=544&amp;pages=427-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature22043" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature22043" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature22043"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28447635" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx036-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx036-B29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yuan</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Bayer</div> <div class="given-names">P. E.</div></div>, <div class="name string-name"><div class="surname">Batley</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Edwards</div> <div class="given-names">D.</div></div></div> <div class="year">2017</div>, 
<div class="article-title">Improvements in genomic technologies: application to crop genomics</div>. <div class="source">Trends Biotechnol</div> ., <div class="volume">35</div>, <div class="fpage">547</div>–<div class="lpage">58</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Improvements%20in%20genomic%20technologies%3A%20application%20to%20crop%20genomics&amp;author=Y.%20Yuan&amp;author=P.%20E.%20Bayer&amp;author=J.%20Batley&amp;author=D.%20Edwards&amp;publication_year=2017&amp;journal=Trends%20Biotechnol&amp;volume=35&amp;pages=547-58" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tibtech.2017.02.009" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tibtech.2017.02.009" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tibtech.2017.02.009"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28284542" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx036-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx036-B30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Geetha</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Mani</div> <div class="given-names">A. K.</div></div>, <div class="name string-name"><div class="surname">Sur</div> <div class="given-names">M.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Identification of finger millet (<em>Eleusine coracana</em> Gaertn.) variety suitable to rainfed areas of north western zone of Tamil Nadu</div>. <div class="source">Indian J. Agric. Res</div> ., <div class="volume">46</div>, <div class="fpage">60</div>–<div class="lpage">4</div>.</div></div></div></div><div content-id="dsx036-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx036-B31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Dolezel</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Sgorbati</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Lucretti</div> <div class="given-names">S.</div></div></div> <div class="year">1992</div>, 
<div class="article-title">Comparison of three DNA fluorochromes for flow cytometric estimation of nuclear DNA content in plants</div>. <div class="source">Physiologia Plantarum</div> , <div class="volume">85</div>, <div class="fpage">625</div>–<div class="lpage">31</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparison%20of%20three%20DNA%20fluorochromes%20for%20flow%20cytometric%20estimation%20of%20nuclear%20DNA%20content%20in%20plants&amp;author=J.%20Dolezel&amp;author=S.%20Sgorbati&amp;author=S.%20Lucretti&amp;publication_year=1992&amp;journal=Physiologia%20Plantarum&amp;volume=85&amp;pages=625-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/ppl.1992.85.issue-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fppl.1992.85.issue-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fppl.1992.85.issue-4"> </span></div> </div></div></div></div></div><div content-id="dsx036-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx036-B32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bolger</div> <div class="given-names">A. M.</div></div>, <div class="name string-name"><div class="surname">Lohse</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Usadel</div> <div class="given-names">B.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Trimmomatic: a flexible trimmer for Illumina sequence data</div>. <div class="source">Bioinformatics (Oxford, England)</div> , <div class="volume">30</div>, <div class="fpage">2114</div>–<div class="lpage">20</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Trimmomatic%3A%20a%20flexible%20trimmer%20for%20Illumina%20sequence%20data&amp;author=A.%20M.%20Bolger&amp;author=M.%20Lohse&amp;author=B.%20Usadel&amp;publication_year=2014&amp;journal=Bioinformatics%20(Oxford%2C%20England)&amp;volume=30&amp;pages=2114-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btu170" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtu170" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtu170"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24695404" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx036-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx036-B33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hatakeyama</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Opitz</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Russo</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Qi</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Schlapbach</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Rehrauer</div> <div class="given-names">H.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">SUSHI: an exquisite recipe for fully documented, reproducible and reusable NGS data analysis</div>. <div class="source">BMC Bioinform</div> ., <div class="volume">17</div>, <div class="fpage">228</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SUSHI%3A%20an%20exquisite%20recipe%20for%20fully%20documented%2C%20reproducible%20and%20reusable%20NGS%20data%20analysis&amp;author=M.%20Hatakeyama&amp;author=L.%20Opitz&amp;author=G.%20Russo&amp;author=W.%20Qi&amp;author=R.%20Schlapbach&amp;author=H.%20Rehrauer&amp;publication_year=2016&amp;journal=BMC%20Bioinform&amp;volume=17&amp;pages=228" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12859-016-1104-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12859-016-1104-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12859-016-1104-8"> </span></div> </div></div></div></div></div><div content-id="dsx036-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx036-B34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kajitani</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Toshimoto</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Noguchi</div> <div class="given-names">H.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Efficient de novo assembly of highly heterozygous genomes from whole-genome shotgun short reads</div>. <div class="source">Genome Res</div> ., <div class="volume">24</div>, <div class="fpage">1384</div>–<div class="lpage">95</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Efficient%20de%20novo%20assembly%20of%20highly%20heterozygous%20genomes%20from%20whole-genome%20shotgun%20short%20reads&amp;author=R.%20Kajitani&amp;author=K.%20Toshimoto&amp;author=H.%20Noguchi&amp;publication_year=2014&amp;journal=Genome%20Res&amp;volume=24&amp;pages=1384-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.170720.113" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.170720.113" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.170720.113"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24755901" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx036-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx036-B35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ye</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Hill</div> <div class="given-names">C. M.</div></div>, <div class="name string-name"><div class="surname">Wu</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Ruan</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Ma</div> <div class="given-names">Z.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">DBG2OLC: efficient assembly of large genomes using long erroneous reads of the third generation sequencing technologies</div>. <div class="source">Sci. Reports</div> , <div class="volume">6</div>, <div class="fpage">31900</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DBG2OLC%3A%20efficient%20assembly%20of%20large%20genomes%20using%20long%20erroneous%20reads%20of%20the%20third%20generation%20sequencing%20technologies&amp;author=C.%20Ye&amp;author=C.%20M.%20Hill&amp;author=S.%20Wu&amp;author=J.%20Ruan&amp;author=Z.%20Ma&amp;publication_year=2016&amp;journal=Sci.%20Reports&amp;volume=6&amp;pages=31900" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/srep31900" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsrep31900" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsrep31900"> </span></div> </div></div></div></div></div><div content-id="dsx036-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx036-B36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Walker</div> <div class="given-names">B. J.</div></div>, <div class="name string-name"><div class="surname">Abeel</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Shea</div> <div class="given-names">T.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Pilon: an integrated tool for comprehensive microbial variant detection and genome assembly improvement</div>. <div class="source">PLoS One</div> , <div class="volume">9</div>, <div class="fpage">e112963</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Pilon%3A%20an%20integrated%20tool%20for%20comprehensive%20microbial%20variant%20detection%20and%20genome%20assembly%20improvement&amp;author=B.%20J.%20Walker&amp;author=T.%20Abeel&amp;author=T.%20Shea&amp;publication_year=2014&amp;journal=PLoS%20One&amp;volume=9&amp;pages=e112963" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0112963" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0112963" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0112963"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25409509" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx036-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx036-B37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Boetzer</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Henkel</div> <div class="given-names">C. V.</div></div>, <div class="name string-name"><div class="surname">Jansen</div> <div class="given-names">H. J.</div></div>, <div class="name string-name"><div class="surname">Butler</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Pirovano</div> <div class="given-names">W.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Scaffolding pre-assembled contigs using SSPACE</div>. <div class="source">Bioinformatics</div> , <div class="volume">27</div>, <div class="fpage">578</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Scaffolding%20pre-assembled%20contigs%20using%20SSPACE&amp;author=M.%20Boetzer&amp;author=C.%20V.%20Henkel&amp;author=H.%20J.%20Jansen&amp;author=D.%20Butler&amp;author=W.%20Pirovano&amp;publication_year=2011&amp;journal=Bioinformatics&amp;volume=27&amp;pages=578-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btq683" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtq683" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtq683"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21149342" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx036-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx036-B38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kosugi</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Hirakawa</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Tabata</div> <div class="given-names">S.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">GMcloser: closing gaps in assemblies accurately with a likelihood-based selection of contig or long-read alignments</div>. <div class="source">Bioinformatics</div> , <div class="volume">31</div>, <div class="fpage">btv465</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GMcloser%3A%20closing%20gaps%20in%20assemblies%20accurately%20with%20a%20likelihood-based%20selection%20of%20contig%20or%20long-read%20alignments&amp;author=S.%20Kosugi&amp;author=H.%20Hirakawa&amp;author=S.%20Tabata&amp;publication_year=2015&amp;journal=Bioinformatics&amp;volume=31&amp;pages=btv465" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btv465" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtv465" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtv465"> </span></div> </div></div></div></div></div><div content-id="dsx036-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx036-B39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Cantarel</div> <div class="given-names">B. L.</div></div>, <div class="name string-name"><div class="surname">Korf</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Robb</div> <div class="given-names">S. M.</div></div></div>, et al.   <div class="year">2007</div>, 
<div class="article-title">MAKER: An easy-to-use annotation pipeline designed for emerging model organism genomes</div>. <div class="source">Genome Res</div> ., <div class="volume">18</div>, <div class="fpage">188</div>–<div class="lpage">96</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MAKER%3A%20An%20easy-to-use%20annotation%20pipeline%20designed%20for%20emerging%20model%20organism%20genomes&amp;author=B.%20L.%20Cantarel&amp;author=I.%20Korf&amp;author=S.%20M.%20Robb&amp;publication_year=2007&amp;journal=Genome%20Res&amp;volume=18&amp;pages=188-96" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.6743907" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.6743907" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.6743907"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18025269" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx036-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx036-B40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Dobin</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Davis</div> <div class="given-names">C. A.</div></div>, <div class="name string-name"><div class="surname">Schlesinger</div> <div class="given-names">F.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">STAR: ultrafast universal RNA-seq aligner</div>. <div class="source">Bioinformatics</div> , <div class="volume">29</div>, <div class="fpage">15</div>–<div class="lpage">21</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=STAR%3A%20ultrafast%20universal%20RNA-seq%20aligner&amp;author=A.%20Dobin&amp;author=C.%20A.%20Davis&amp;author=F.%20Schlesinger&amp;publication_year=2013&amp;journal=Bioinformatics&amp;volume=29&amp;pages=15-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bts635" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbts635" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbts635"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23104886" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx036-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx036-B41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Pertea</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Pertea</div> <div class="given-names">G. M.</div></div>, <div class="name string-name"><div class="surname">Antonescu</div> <div class="given-names">C. M.</div></div>, <div class="name string-name"><div class="surname">Chang</div> <div class="given-names">T. C.</div></div>, <div class="name string-name"><div class="surname">Mendell</div> <div class="given-names">J. T.</div></div>, <div class="name string-name"><div class="surname">Salzberg</div> <div class="given-names">S. L.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">StringTie enables improved reconstruction of a transcriptome from RNA-seq reads</div>. <div class="source">Nat. Biotechnol</div> ., <div class="volume">33</div>, <div class="fpage">290</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=StringTie%20enables%20improved%20reconstruction%20of%20a%20transcriptome%20from%20RNA-seq%20reads&amp;author=M.%20Pertea&amp;author=G.%20M.%20Pertea&amp;author=C.%20M.%20Antonescu&amp;author=T.%20C.%20Chang&amp;author=J.%20T.%20Mendell&amp;author=S.%20L.%20Salzberg&amp;publication_year=2015&amp;journal=Nat.%20Biotechnol&amp;volume=33&amp;pages=290-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.3122" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.3122" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.3122"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25690850" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx036-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx036-B42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Jurka</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Kapitonov</div> <div class="given-names">V.</div></div>, <div class="name string-name"><div class="surname">Pavlicek</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Klonowski</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Kohany</div> <div class="given-names">O.</div></div>, <div class="name string-name"><div class="surname">Walichiewicz</div> <div class="given-names">J.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">Repbase Update, a database of eukaryotic repetitive elements</div>. <div class="source">Cytogenet. Genome Res</div> ., <div class="volume">110</div>, <div class="fpage">462</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Repbase%20Update%2C%20a%20database%20of%20eukaryotic%20repetitive%20elements&amp;author=J.%20Jurka&amp;author=V.%20Kapitonov&amp;author=A.%20Pavlicek&amp;author=P.%20Klonowski&amp;author=O.%20Kohany&amp;author=J.%20Walichiewicz&amp;publication_year=2005&amp;journal=Cytogenet.%20Genome%20Res&amp;volume=110&amp;pages=462-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1159/000084979" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1159%2F000084979" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1159%2F000084979"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16093699" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx036-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx036-B43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Saha</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Bridges</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Magbanua</div> <div class="given-names">Z. V.</div></div>, <div class="name string-name"><div class="surname">Peterson</div> <div class="given-names">D. G.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">Empirical comparison of ab initio repeat finding programs</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">36</div>, <div class="fpage">2284</div>–<div class="lpage">94</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Empirical%20comparison%20of%20ab%20initio%20repeat%20finding%20programs&amp;author=S.%20Saha&amp;author=S.%20Bridges&amp;author=Z.%20V.%20Magbanua&amp;author=D.%20G.%20Peterson&amp;publication_year=2008&amp;journal=Nucleic%20Acids%20Res&amp;volume=36&amp;pages=2284-94" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkn064" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkn064" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkn064"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18287116" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx036-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx036-B44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Stanke</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Diekhans</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Baertsch</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Haussler</div> <div class="given-names">D.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">Using native and synthetically mapped cDNA alignments to improve de novo gene finding</div>. <div class="source">Bioinformatics</div> , <div class="volume">24</div>, <div class="fpage">637</div>–<div class="lpage">44</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Using%20native%20and%20synthetically%20mapped%20cDNA%20alignments%20to%20improve%20de%20novo%20gene%20finding&amp;author=M.%20Stanke&amp;author=M.%20Diekhans&amp;author=R.%20Baertsch&amp;author=D.%20Haussler&amp;publication_year=2008&amp;journal=Bioinformatics&amp;volume=24&amp;pages=637-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btn013" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtn013" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtn013"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18218656" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx036-B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx036-B45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Simão</div> <div class="given-names">F. A.</div></div>, <div class="name string-name"><div class="surname">Waterhouse</div> <div class="given-names">R. M.</div></div>, <div class="name string-name"><div class="surname">Ioannidis</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Kriventseva</div> <div class="given-names">E. V.</div></div>, <div class="name string-name"><div class="surname">Zdobnov</div> <div class="given-names">E. M.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs</div>. <div class="source">Bioinformatics</div> , <div class="volume">31</div>, <div class="fpage">3210</div>–<div class="lpage">2</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=BUSCO%3A%20assessing%20genome%20assembly%20and%20annotation%20completeness%20with%20single-copy%20orthologs&amp;author=F.%20A.%20Sim%C3%A3o&amp;author=R.%20M.%20Waterhouse&amp;author=P.%20Ioannidis&amp;author=E.%20V.%20Kriventseva&amp;author=E.%20M.%20Zdobnov&amp;publication_year=2015&amp;journal=Bioinformatics&amp;volume=31&amp;pages=3210-2" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btv351" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtv351" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtv351"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26059717" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx036-B46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx036-B46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Staňková</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Hastie</div> <div class="given-names">A. R.</div></div>, <div class="name string-name"><div class="surname">Chan</div> <div class="given-names">S.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">BioNano genome mapping of individual chromosomes supports physical mapping and sequence assembly in complex plant genomes</div>. <div class="source">Plant Biotechnol. J</div> ., <div class="volume">14</div>, <div class="fpage">1523</div>–<div class="lpage">31</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=BioNano%20genome%20mapping%20of%20individual%20chromosomes%20supports%20physical%20mapping%20and%20sequence%20assembly%20in%20complex%20plant%20genomes&amp;author=H.%20Sta%C5%88kov%C3%A1&amp;author=A.%20R.%20Hastie&amp;author=S.%20Chan&amp;publication_year=2016&amp;journal=Plant%20Biotechnol.%20J&amp;volume=14&amp;pages=1523-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/pbi.2016.14.issue-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fpbi.2016.14.issue-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fpbi.2016.14.issue-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26801360" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx036-B47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx036-B47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Fierst</div> <div class="given-names">J. L.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Using linkage maps to correct and scaffold de novo genome assemblies: methods, challenges, and computational tools</div>. <div class="source">Front. Genet</div> ., <div class="volume">6</div>, <div class="fpage">1</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Using%20linkage%20maps%20to%20correct%20and%20scaffold%20de%20novo%20genome%20assemblies%3A%20methods%2C%20challenges%2C%20and%20computational%20tools&amp;author=J.%20L.%20Fierst&amp;publication_year=2015&amp;journal=Front.%20Genet&amp;volume=6&amp;pages=1-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3389/fgene.2015.00220" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3389%2Ffgene.2015.00220" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3389%2Ffgene.2015.00220"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25674101" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx036-B48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx036-B48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bruijn de</div> <div class="given-names">N. G.</div></div></div> <div class="year">1946</div>, 
<div class="article-title">A combinatorial problem</div>. <div class="source">Proc. Sect. Sci</div> ., <div class="volume">49</div>, <div class="fpage">758</div>–<div class="lpage">64</div>.</div></div></div></div><div content-id="dsx036-B49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx036-B49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Compeau</div> <div class="given-names">P. E. C.</div></div>, <div class="name string-name"><div class="surname">Pevzner</div> <div class="given-names">P. A.</div></div>, <div class="name string-name"><div class="surname">Tesler</div> <div class="given-names">G.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">How to apply de Bruijn graphs to genome assembly</div>. <div class="source">Nat. Biotechnol</div> ., <div class="volume">29</div>, <div class="fpage">987</div>–<div class="lpage">91</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=How%20to%20apply%20de%20Bruijn%20graphs%20to%20genome%20assembly&amp;author=P.%20E.%20C.%20Compeau&amp;author=P.%20A.%20Pevzner&amp;author=G.%20Tesler&amp;publication_year=2011&amp;journal=Nat.%20Biotechnol&amp;volume=29&amp;pages=987-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.2023" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.2023" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.2023"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22068540" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx036-B50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx036-B50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Krasileva</div> <div class="given-names">K. V.</div></div>, <div class="name string-name"><div class="surname">Buffalo</div> <div class="given-names">V.</div></div>, <div class="name string-name"><div class="surname">Bailey</div> <div class="given-names">P.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">Separating homeologs by phasing in the tetraploid wheat transcriptome</div>. <div class="source">Genome Biol</div> ., <div class="volume">14</div>, <div class="fpage">R66</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Separating%20homeologs%20by%20phasing%20in%20the%20tetraploid%20wheat%20transcriptome&amp;author=K.%20V.%20Krasileva&amp;author=V.%20Buffalo&amp;author=P.%20Bailey&amp;publication_year=2013&amp;journal=Genome%20Biol&amp;volume=14&amp;pages=R66" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2013-14-6-r66" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2013-14-6-r66" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2013-14-6-r66"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23800085" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx036-B51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx036-B51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Innan</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Kondrashov</div> <div class="given-names">F.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">The evolution of gene duplications: classifying and distinguishing between models</div>. <div class="source">Nat. Rev. Genet</div> ., <div class="volume">11</div>, <div class="fpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20evolution%20of%20gene%20duplications%3A%20classifying%20and%20distinguishing%20between%20models&amp;author=H.%20Innan&amp;author=F.%20Kondrashov&amp;publication_year=2010&amp;journal=Nat.%20Rev.%20Genet&amp;volume=11&amp;pages=4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg2689" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg2689" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg2689"> </span></div> </div></div></div></div></div><div content-id="dsx036-B52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx036-B52" class="ref-content "><div class="label -label">52</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Koh</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Soltis</div> <div class="given-names">P. S.</div></div>, <div class="name string-name"><div class="surname">Soltis</div> <div class="given-names">D. E.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Homeolog loss and expression changes in natural populations of the recently and repeatedly formed allotetraploid <em>Tragopogon mirus</em> (Asteraceae)</div>. <div class="source">BMC Genomics</div> , <div class="volume">11</div>, <div class="fpage">97</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Homeolog%20loss%20and%20expression%20changes%20in%20natural%20populations%20of%20the%20recently%20and%20repeatedly%20formed%20allotetraploid%20Tragopogon%20mirus%20(Asteraceae)&amp;author=J.%20Koh&amp;author=P.%20S.%20Soltis&amp;author=D.%20E.%20Soltis&amp;publication_year=2010&amp;journal=BMC%20Genomics&amp;volume=11&amp;pages=97" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-11-97" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-11-97" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-11-97"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20141639" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx036-B53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx036-B53" class="ref-content "><div class="label -label">53</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Finn</div> <div class="given-names">R. D.</div></div>, <div class="name string-name"><div class="surname">Bateman</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Clements</div> <div class="given-names">J.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Pfam: the protein families database</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">42</div>, <div class="fpage">D222</div>–<div class="lpage">30</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Pfam%3A%20the%20protein%20families%20database&amp;author=R.%20D.%20Finn&amp;author=A.%20Bateman&amp;author=J.%20Clements&amp;publication_year=2014&amp;journal=Nucleic%20Acids%20Res&amp;volume=42&amp;pages=D222-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkt1223" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkt1223" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkt1223"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24288371" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx036-B54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx036-B54" class="ref-content "><div class="label -label">54</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Altschul</div> <div class="given-names">S. F.</div></div>, <div class="name string-name"><div class="surname">Gish</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Miller</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Myers</div> <div class="given-names">E. W.</div></div>, <div class="name string-name"><div class="surname">Lipman</div> <div class="given-names">D. J.</div></div></div> <div class="year">1990</div>, 
<div class="article-title">Basic local alignment search tool</div>. <div class="source">J. Mol. Biol</div> ., <div class="volume">215</div>, <div class="fpage">403</div>–<div class="lpage">10</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Basic%20local%20alignment%20search%20tool&amp;author=S.%20F.%20Altschul&amp;author=W.%20Gish&amp;author=W.%20Miller&amp;author=E.%20W.%20Myers&amp;author=D.%20J.%20Lipman&amp;publication_year=1990&amp;journal=J.%20Mol.%20Biol&amp;volume=215&amp;pages=403-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0022-2836(05)80360-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0022-2836(05)80360-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0022-2836(05)80360-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2231712" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx036-B55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx036-B55" class="ref-content "><div class="label -label">55</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tamura</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Dudley</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Nei</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Kumar</div> <div class="given-names">S.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">MEGA4: Molecular Evolutionary Genetics Analysis (MEGA) Software Version 4.0</div>. <div class="source">Mol. Biol. Evol</div> ., <div class="volume">24</div>, <div class="fpage">1596</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MEGA4%3A%20Molecular%20Evolutionary%20Genetics%20Analysis%20(MEGA)%20Software%20Version%204.0&amp;author=K.%20Tamura&amp;author=J.%20Dudley&amp;author=M.%20Nei&amp;author=S.%20Kumar&amp;publication_year=2007&amp;journal=Mol.%20Biol.%20Evol&amp;volume=24&amp;pages=1596-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msm092" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsm092" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsm092"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17488738" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx036-B56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx036-B56" class="ref-content "><div class="label -label">56</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Elliott</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Deryng</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Müller</div> <div class="given-names">C.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Constraints and potentials of future irrigation water availability on agricultural production under climate change</div>. <div class="source">Proc. Natl. Acad. Sci</div> ., <div class="volume">111</div>, <div class="fpage">3239</div>–<div class="lpage">44</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Constraints%20and%20potentials%20of%20future%20irrigation%20water%20availability%20on%20agricultural%20production%20under%20climate%20change&amp;author=J.%20Elliott&amp;author=D.%20Deryng&amp;author=C.%20M%C3%BCller&amp;publication_year=2014&amp;journal=Proc.%20Natl.%20Acad.%20Sci&amp;volume=111&amp;pages=3239-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1222474110" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1222474110" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1222474110"> </span></div> </div></div></div></div></div><div content-id="dsx036-B57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx036-B57" class="ref-content "><div class="label -label">57</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wheeler</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Braun</div> <div class="given-names">J. von.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Climate change impacts on global food security</div>. <div class="source">Science</div> , <div class="volume">341</div>, <div class="fpage">508</div>–<div class="lpage">13</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Climate%20change%20impacts%20on%20global%20food%20security&amp;author=T.%20Wheeler&amp;author=J.%20von.%20Braun&amp;publication_year=2013&amp;journal=Science&amp;volume=341&amp;pages=508-13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1239402" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1239402" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1239402"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23908229" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx036-B58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx036-B58" class="ref-content "><div class="label -label">58</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Brown</div> <div class="given-names">A. H. D.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Variation under domestication in plants: 1859 and today</div>. <div class="source">Philos. Trans. Roy. Soc. B Biol. Sci</div> ., <div class="volume">365</div>, <div class="fpage">2523</div>–<div class="lpage">30</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Variation%20under%20domestication%20in%20plants%3A%201859%20and%20today&amp;author=A.%20H.%20D.%20Brown&amp;publication_year=2010&amp;journal=Philos.%20Trans.%20Roy.%20Soc.%20B%20Biol.%20Sci&amp;volume=365&amp;pages=2523-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1098/rstb.2010.0006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1098%2Frstb.2010.0006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1098%2Frstb.2010.0006"> </span></div> </div></div></div></div></div><div content-id="dsx036-B59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx036-B59" class="ref-content "><div class="label -label">59</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Meyer</div> <div class="given-names">R. S.</div></div>, <div class="name string-name"><div class="surname">DuVal</div> <div class="given-names">A. E.</div></div>, <div class="name string-name"><div class="surname">Jensen</div> <div class="given-names">H. R.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Patterns and processes in crop domestication: an historical review and quantitative analysis of 203 global food crops</div>. <div class="source">New Phytol</div> ., <div class="volume">196</div>, <div class="fpage">29</div>–<div class="lpage">48</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Patterns%20and%20processes%20in%20crop%20domestication%3A%20an%20historical%20review%20and%20quantitative%20analysis%20of%20203%20global%20food%20crops&amp;author=R.%20S.%20Meyer&amp;author=A.%20E.%20DuVal&amp;author=H.%20R.%20Jensen&amp;publication_year=2012&amp;journal=New%20Phytol&amp;volume=196&amp;pages=29-48" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1469-8137.2012.04253.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1469-8137.2012.04253.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1469-8137.2012.04253.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22889076" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx036-B60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx036-B60" class="ref-content "><div class="label -label">60</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hittalmani</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Mahesh</div> <div class="given-names">H. B.</div></div>, <div class="name string-name"><div class="surname">Shirke</div> <div class="given-names">M. D.</div></div></div>, et al.   <div class="year">2017</div>, 
<div class="article-title">Genome and transcriptome sequence of finger millet (<em>Eleusine coracana</em> (L.) Gaertn.) provides insights into drought tolerance and nutraceutical properties</div>. <div class="source">BMC Genomics</div> , <div class="volume">18</div>, <div class="fpage">465</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20and%20transcriptome%20sequence%20of%20finger%20millet%20(Eleusine%20coracana%20(L.)%20Gaertn.)%20provides%20insights%20into%20drought%20tolerance%20and%20nutraceutical%20properties&amp;author=S.%20Hittalmani&amp;author=H.%20B.%20Mahesh&amp;author=M.%20D.%20Shirke&amp;publication_year=2017&amp;journal=BMC%20Genomics&amp;volume=18&amp;pages=465" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12864-017-3850-z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12864-017-3850-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12864-017-3850-z"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28619070" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2017. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=4103403&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx036/3/dsx036_suppl_data.zip?Expires=2147483647&amp;Signature=sUJU62LQXktRG1MSxA1tOfm80oYihgIlD3~3j6DFBg-PF7MIbxZL2BbNr87MXOg3420R9SbCs16c94jbq3Dxina1NARhf-Uq4mEvZzKPn6lwmVM7m5tk6ho1hnLHZ6JN2bjS8FV~KfE8r67EnD8GLAPUPEGsZwWEq2sxevStTFQZy2O8s943dIl1wU7pYWtkZ38frAPZ~G8tfKxhyXMfeyop~VcLT2xdgIYbKPB83FxOOim83rEqXMbojXZJ6dplzORh3kjKpl6F2a7Q2e~woJkk0ad-Inp-bpXemDGGUI4EUmV0BssMbU6NAMRvrqSS7s2-lqkUNsMd9TPf1F-C~Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">2,542</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000425954100004&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">3</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsx036"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="4103403"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000425954100004&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f25%2f1%2f39%2f4103403">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000425954100004&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (3)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F25%2F1%2F39%2F4103403" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/4103403'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Multiple hybrid de novo genome assembly of finger millet, an orphan allotetraploid crop" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsx036");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f25%2f1%2f39%2f4103403">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

