

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" /><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info = {"beacon":"bam.nr-data.net","errorBeacon":"bam.nr-data.net","licenseKey":"8d76ca6f26","applicationID":"29565620","transactionName":"ZQBRMkVUCEtRVkINXlxKcjVnGgdKRFxVCFQcBEAWTw==","queueTime":0,"applicationTime":2094,"agent":"","atts":""}</script><script type="text/javascript">(window.NREUM||(NREUM={})).loader_config={xpid:"XAYBV15WGwEJUVdXAQMH"};window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{s.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(21),s={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(s.console=!0,o.indexOf("dev")!==-1&&(s.dev=!0),o.indexOf("nr_dev")!==-1&&(s.nrDev=!0))}catch(c){}s.nrDev&&i.on("internal-error",function(t){r(t.stack)}),s.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),s.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(s,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,s){try{l?l-=1:o(s||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,c.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:c.now();i("err",[t,n])}var i=t("handle"),a=t(22),s=t("ee"),c=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;c.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),c.xhrWrappable&&t(14),d=!0)}s.on("fn-start",function(t,e,n){d&&(l+=1)}),s.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),s.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),s.on("internal-error",function(t){i("ierr",[t,c.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){M++,N=y.hash,this[u]=g.now()}function o(){M--,y.hash!==N&&i(0,!0);var t=g.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+y,e])}function a(t,e){t.on(e,function(){this[e]=g.now()})}var s="-start",c="-end",f="-body",u="fn"+s,d="fn"+c,p="cb"+s,l="cb"+c,h="jsTime",m="fetch",v="addEventListener",w=window,y=w.location,g=t("loader");if(w[v]&&g.xhrWrappable){var b=t(10),x=t(11),E=t(8),O=t(6),P=t(13),R=t(7),T=t(14),L=t(9),j=t("ee"),S=j.get("tracer");t(15),g.features.spa=!0;var N,M=0;j.on(u,r),j.on(p,r),j.on(d,o),j.on(l,o),j.buffer([u,d,"xhr-done","xhr-resolved"]),O.buffer([u]),P.buffer(["setTimeout"+c,"clearTimeout"+s,u]),T.buffer([u,"new-xhr","send-xhr"+s]),R.buffer([m+s,m+"-done",m+f+s,m+f+c]),E.buffer(["newURL"]),b.buffer([u]),x.buffer(["propagate",p,l,"executor-err","resolve"+s]),S.buffer([u,"no-"+u]),L.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(T,"send-xhr"+s),a(j,"xhr-resolved"),a(j,"xhr-done"),a(R,m+s),a(R,m+"-done"),a(L,"new-jsonp"),a(L,"jsonp-end"),a(L,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,M>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),s=t(12),c="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",y="pushState",g=t("loader");g.features.stn=!0,t(8);var b=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof b&&(this.bstStart=g.now())}),o.on(v,function(t,e){var n=t[0];n instanceof b&&i("bst",[n,e,this.bstStart,g.now()])}),a.on(m,function(t,e,n){this.bstStart=g.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),this.bstType])}),s.on(m,function(){this.bstStart=g.now()}),s.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),"requestAnimationFrame"])}),o.on(y+l,function(t){this.time=g.now(),this.startPath=location.pathname+location.hash}),o.on(y+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+c]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+c]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+c]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){s.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),s=t(24)(a,!0),c=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=c(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?s(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=r.apply(this,arguments);return o.emit(n+"start",arguments,t),t.then(function(e){return o.emit(n+"end",[null,e],t),e},function(e){throw o.emit(n+"end",[e],t),e})})}var o=t("ee").get("fetch"),i=t(21);e.exports=o;var a=window,s="fetch-",c=s+"body-",f=["arrayBuffer","blob","json","text","formData"],u=a.Request,d=a.Response,p=a.fetch,l="prototype";u&&d&&p&&(i(f,function(t,e){r(u[l],e,c),r(d[l],e,c)}),r(a,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t(24)(r);e.exports=r,o.inPlace(window.history,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){c.emit("jsonp-error",[],p),c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=s(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),c.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function s(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var c=t("ee").get("jsonp"),f=t(24)(c);if(e.exports=c,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-"),c.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t(24)(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=s(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t(24),a=t("ee").get("promise"),s=i(a),c=t(21),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;c(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),s=f.resolve(i);return s}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),c(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=s(t[0],"resolve-",this),t[1]=s(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),s.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=s(t[0],"cb-",this),t[1]=s(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t(24)(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t(24)(i),s="setTimeout",c="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[s,"setImmediate"],s+d),a.inPlace(window,[c],c+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(c+u,r),i.on(s+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",s)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,y,"fn-",s)}function i(t){g.push(t),h&&(x?x.then(a):v?v(a):(E=-E,O.data=E))}function a(){for(var t=0;t<g.length;t++)r([],g[t]);g.length&&(g=[])}function s(t,e){return e}function c(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t(24)(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",y=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],g=[];e.exports=u;var b=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(c(l,b),b.prototype=l.prototype,d.inPlace(b.prototype,["open","send"],"-xhr-",s),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var x=m&&m.resolve();if(!v&&!m){var E=1,O=document.createTextNode(E);new h(a).observe(O,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<d;r++)t.removeEventListener(u[r],this.listener,!1);if(!e.aborted){if(n.duration=a.now()-this.startTime,4===t.readyState){e.status=t.status;var i=o(t,this.lastSize);if(i&&(n.rxSize=i),this.sameOrigin){var c=t.getResponseHeader("X-NewRelic-App-Data");c&&(e.cat=c.split(", ").pop())}}else e.status=0;n.cbTime=this.cbTime,f.emit("xhr-done",[t],t),s("xhr",[e,n,this.startTime])}}}function o(t,e){var n=t.responseType;if("json"===n&&null!==e)return e;var r="arraybuffer"===n||"blob"===n||"json"===n?t.response:t.responseText;return h(r)}function i(t,e){var n=c(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.sameOrigin=n.sameOrigin}var a=t("loader");if(a.xhrWrappable){var s=t("handle"),c=t(16),f=t("ee"),u=["load","error","abort","timeout"],d=u.length,p=t("id"),l=t(19),h=t(18),m=window.XMLHttpRequest;a.features.xhr=!0,t(14),f.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,l&&(l>34||l<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),f.on("open-xhr-start",function(t){this.params={method:t[0]},i(this,t[1]),this.metrics={}}),f.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid)}),f.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=h(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"===t.type&&(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{f.emit("internal-error",[n])}catch(r){}}};for(var s=0;s<d;s++)e.addEventListener(u[s],this.listener,!1)}),f.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),f.on("xhr-load-added",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),f.on("xhr-load-removed",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),f.on("addEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-added",[t[1],t[2]],e)}),f.on("removeEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-removed",[t[1],t[2]],e)}),f.on("fn-start",function(t,e,n){e instanceof m&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),f.on("fn-end",function(t,e){this.xhrCbStart&&f.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)})}},{}],16:[function(t,e,n){e.exports=function(t){var e=document.createElement("a"),n=window.location,r={};e.href=t,r.port=e.port;var o=e.href.split("://");!r.port&&o[1]&&(r.port=o[1].split("/")[0].split("@").pop().split(":")[1]),r.port&&"0"!==r.port||(r.port="https"===o[0]?"443":"80"),r.hostname=e.hostname||n.hostname,r.pathname=e.pathname,r.protocol=o[0],"/"!==r.pathname.charAt(0)&&(r.pathname="/"+r.pathname);var i=!e.protocol||":"===e.protocol||e.protocol===n.protocol,a=e.hostname===document.domain&&e.port===n.port;return r.sameOrigin=i&&(!e.hostname||a),r}},{}],17:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(s(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(21),s=t(22),c=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(c.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw c.emit("fn-err",[arguments,this,t],n),t}finally{c.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now()])}},{}],18:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],19:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],20:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var s=navigator.userAgent,c=s.match(a);c&&s.indexOf("Chrome")===-1&&s.indexOf("Chromium")===-1&&(o="Safari",i=c[1])}e.exports={agent:o,version:i,match:r}},{}],21:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],22:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],23:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],24:[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(22),a="nr@original",s=Object.prototype.hasOwnProperty,c=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,s,c;try{a=this,r=i(arguments),s="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],s])}u(e+"start",[r,a,o],s);try{return c=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],s),d}finally{u(e+"end",[r,a,c],s)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,s,c,f="-"===o.charAt(0);for(c=0;c<e.length;c++)s=e[c],a=t[s],r(a)||(t[s]=n(a,f?s+o:o,i,s))}function u(n,r,o){if(!c||e){var i=c;c=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}c=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)s.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?c(t,s,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),s=m(n),c=s.length,f=0;f<c;f++)s[f].apply(a,r);var d=u[g[n]];return d&&d.push([b,n,r,a]),a}}function l(t,e){y[t]=m(t).concat(e)}function h(t,e){var n=y[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return y[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",g[n]=e,e in u||(u[e]=[])})}var y={},g={},b={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return b}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var s="nr@context",c=t("gos"),f=t(21),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=x.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(g,function(e,n){t[e]||(t[e]=n)}),c("mark",["onload",a()+x.offset],null,"api");var n=l.createElement("script");n.src="https://"+t.agent,e.parentNode.insertBefore(n,e)}}function o(){"complete"===l.readyState&&i()}function i(){c("mark",["domContent",a()+x.offset],null,"api")}function a(){return O.exists&&performance.now?Math.round(performance.now()):(s=Math.max((new Date).getTime(),s))-x.offset}var s=(new Date).getTime(),c=t("handle"),f=t(21),u=t("ee"),d=t(20),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var y=""+location,g={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1099.min.js"},b=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),x=e.exports={offset:s,now:a,origin:y,features:{},xhrWrappable:b,userAgent:d};t(17),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),c("mark",["firstbyte",s],null,"api");var E=0,O=t(23)},{}]},{},["loader",2,15,5,3,4]);</script>
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/25/5/547/5059530","@type":"ScholarlyArticle","name":"The yellowtail ( Seriola quinqueradiata ) genome and transcriptome atlas of the digestive tract","datePublished":"2018-07-25","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/25/5","@type":"PublicationIssue","issueNumber":"5","datePublished":"2018-10-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsy024","keywords":["<italic>Seriola quinqueradiata</italic>","genome sequencing","protein-coding genes","transcriptome analysis","digestive enzymes"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/25/5/547/5059530","author":[{"name":"Yasuike, Motoshige","affiliation":"Research Center for Bioinformatics and Biosciences, National Research Institute of Fisheries Science, Japan Fisheries Research and Education Agency, Yokohama, Kanagawa, Japan","@type":"Person"},{"name":"Iwasaki, Yuki","affiliation":"Research Center for Bioinformatics and Biosciences, National Research Institute of Fisheries Science, Japan Fisheries Research and Education Agency, Yokohama, Kanagawa, Japan","@type":"Person"},{"name":"Nishiki, Issei","affiliation":"Research Center for Bioinformatics and Biosciences, National Research Institute of Fisheries Science, Japan Fisheries Research and Education Agency, Yokohama, Kanagawa, Japan","@type":"Person"},{"name":"Nakamura, Yoji","affiliation":"Research Center for Bioinformatics and Biosciences, National Research Institute of Fisheries Science, Japan Fisheries Research and Education Agency, Yokohama, Kanagawa, Japan","@type":"Person"},{"name":"Matsuura, Aiko","affiliation":"Research Center for Bioinformatics and Biosciences, National Research Institute of Fisheries Science, Japan Fisheries Research and Education Agency, Yokohama, Kanagawa, Japan","@type":"Person"},{"name":"Yoshida, Kazunori","affiliation":"Goto Laboratory, Stock Enhancement and Aquaculture Division, Seikai National Fisheries Research Institute Japan Fisheries Research and Education Agency, Tamanoura-cho, Goto, Nagasaki, Japan","@type":"Person"},{"name":"Noda, Tsutomu","affiliation":"Goto Laboratory, Stock Enhancement and Aquaculture Division, Seikai National Fisheries Research Institute Japan Fisheries Research and Education Agency, Tamanoura-cho, Goto, Nagasaki, Japan","@type":"Person"},{"name":"Andoh, Tadashi","affiliation":"Stock Enhancement and Aquaculture Division, Seikai National Fisheries Research Institute, Japan Fisheries Research and Education Agency, Nagasaki, Japan","@type":"Person"},{"name":"Fujiwara, Atushi","affiliation":"Research Center for Bioinformatics and Biosciences, National Research Institute of Fisheries Science, Japan Fisheries Research and Education Agency, Yokohama, Kanagawa, Japan","@type":"Person"}],"description":"Abstract.  Seriola quinqueradiata (yellowtail) is the most widely farmed and economically important fish in aquaculture in Japan. In this study, we used the gen","pageStart":"547","pageEnd":"560","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"The yellowtail ( Seriola quinqueradiata ) genome and transcriptome atlas of the digestive tract","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="yellowtail ( Seriola quinqueradiata ) genome and transcriptome atlas of the digestive tract" />
    <meta property="og:description" content="Abstract.  Seriola quinqueradiata (yellowtail) is the most widely farmed and economically important fish in aquaculture in Japan. In this study, we used the gen" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/25/5/547/5059530" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2018-07-25" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Yasuike, Motoshige" /><meta name="citation_author_institution" content="Research Center for Bioinformatics and Biosciences, National Research Institute of Fisheries Science, Japan Fisheries Research and Education Agency, Yokohama, Kanagawa, Japan" /><meta name="citation_author" content="Iwasaki, Yuki" /><meta name="citation_author_institution" content="Research Center for Bioinformatics and Biosciences, National Research Institute of Fisheries Science, Japan Fisheries Research and Education Agency, Yokohama, Kanagawa, Japan" /><meta name="citation_author" content="Nishiki, Issei" /><meta name="citation_author_institution" content="Research Center for Bioinformatics and Biosciences, National Research Institute of Fisheries Science, Japan Fisheries Research and Education Agency, Yokohama, Kanagawa, Japan" /><meta name="citation_author" content="Nakamura, Yoji" /><meta name="citation_author_institution" content="Research Center for Bioinformatics and Biosciences, National Research Institute of Fisheries Science, Japan Fisheries Research and Education Agency, Yokohama, Kanagawa, Japan" /><meta name="citation_author" content="Matsuura, Aiko" /><meta name="citation_author_institution" content="Research Center for Bioinformatics and Biosciences, National Research Institute of Fisheries Science, Japan Fisheries Research and Education Agency, Yokohama, Kanagawa, Japan" /><meta name="citation_author" content="Yoshida, Kazunori" /><meta name="citation_author_institution" content="Goto Laboratory, Stock Enhancement and Aquaculture Division, Seikai National Fisheries Research Institute Japan Fisheries Research and Education Agency, Tamanoura-cho, Goto, Nagasaki, Japan" /><meta name="citation_author" content="Noda, Tsutomu" /><meta name="citation_author_institution" content="Goto Laboratory, Stock Enhancement and Aquaculture Division, Seikai National Fisheries Research Institute Japan Fisheries Research and Education Agency, Tamanoura-cho, Goto, Nagasaki, Japan" /><meta name="citation_author" content="Andoh, Tadashi" /><meta name="citation_author_institution" content="Stock Enhancement and Aquaculture Division, Seikai National Fisheries Research Institute, Japan Fisheries Research and Education Agency, Nagasaki, Japan" /><meta name="citation_author" content="Fujiwara, Atushi" /><meta name="citation_author_institution" content="Research Center for Bioinformatics and Biosciences, National Research Institute of Fisheries Science, Japan Fisheries Research and Education Agency, Yokohama, Kanagawa, Japan" /><meta name="citation_title" content="The yellowtail (Seriola quinqueradiata) genome and transcriptome atlas of the digestive tract" /><meta name="citation_firstpage" content="547" /><meta name="citation_lastpage" content="560" /><meta name="citation_doi" content="10.1093/dnares/dsy024" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="25" /><meta name="citation_issue" content="5" /><meta name="citation_publication_date" content="2018/10/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Molecular systematics and biogeography of the circumglobally distributed genus Seriola (Pisces: carangidae);  citation_journal_title=Mol. Phylogenet. Evol;  citation_year=2015;  citation_volume=93;  citation_pages=274-80; " /><meta name="citation_reference" content="citation_title=The State of Seriola spp. Other Than Yellowtail (S. quinqueradiata) Farming in the World;  citation_journal_title=Rev. Fish Sci. Aquac;  citation_year=2016;  citation_volume=24;  citation_pages=314-25; " /><meta name="citation_reference" content="citation_author=Ministry of Agriculture, Forestry and Fisheries of Japan;  citation_year=2017; " /><meta name="citation_reference" content="citation_publisher=FAO Fisheries and Aquaculture Department, Rome;  citation_title=Cultured Aquatic Species Information Programme;  citation_year=2005;  citation_pages=1-11; " /><meta name="citation_reference" content="citation_author=World Bank; citation_publisher=World Bank Group, Washington DC;  citation_title=Fish to 2030: Prospects for Fisheries and Aquaculture. Agriculture and Environmental Services Discussion Paper, No. 3;  citation_year=2013;  citation_pages=1-102; " /><meta name="citation_reference" content="citation_title=Aquaculture genomics, genetics and breeding in the United States: current status, challenges, and priorities for future research;  citation_journal_title=BMC Genomics;  citation_year=2017;  citation_volume=18;  citation_pages=191" /><meta name="citation_reference" content="citation_title=Functional Annotation of All Salmonid Genomes (FAASG): an international initiative supporting future salmonid research, conservation and aquaculture;  citation_journal_title=BMC Genomics;  citation_year=2017;  citation_volume=18;  citation_pages=484" /><meta name="citation_reference" content="citation_title=Current status of genome sequencing and its applications in aquaculture;  citation_journal_title=Aquaculture;  citation_year=2017;  citation_volume=468;  citation_pages=337-47; " /><meta name="citation_reference" content="citation_title=Second generation physical and linkage maps of yellowtail (Seriola quinqueradiata) and comparison of synteny with four model fish;  citation_journal_title=BMC Genomics;  citation_year=2015;  citation_volume=16;  citation_pages=406" /><meta name="citation_reference" content="citation_title=Construction of a high-coverage bacterial artificial chromosome library and comprehensive genetic linkage map of yellowtail Seriola quinqueradiata;  citation_journal_title=BMC Res. Notes;  citation_year=2014;  citation_volume=7;  citation_pages=200" /><meta name="citation_reference" content="citation_title=Genetic linkage maps of two yellowtails (Seriola quinqueradiata and Seriola lalandi);  citation_journal_title=Aquaculture;  citation_year=2005;  citation_volume=244;  citation_pages=41-8; " /><meta name="citation_reference" content="citation_title=Construction of a radiation hybrid panel and the first yellowtail (Seriola quinqueradiata) radiation hybrid map using a nanofluidic dynamic array;  citation_journal_title=BMC Genomics;  citation_year=2014;  citation_volume=15;  citation_pages=165" /><meta name="citation_reference" content="citation_title=Quantitative trait loci (QTL) associated with resistance to a monogenean parasite (Benedenia seriolae) in yellowtail (Seriola quinqueradiata) through genome wide analysis;  citation_journal_title=PloS One;  citation_year=2013;  citation_volume=8;  citation_pages=e64987" /><meta name="citation_reference" content="citation_title=Identification of the sex-linked locus in yellowtail, Seriola quinqueradiata;  citation_journal_title=Aquaculture;  citation_year=2010;  citation_volume=308;  citation_pages=S51-5; " /><meta name="citation_reference" content="citation_title=Identification of sex-linked snps and sex-determining regions in the yellowtail genome;  citation_journal_title=Mar. Biotechnol.;  citation_year=2015;  citation_volume=17;  citation_pages=502-10; " /><meta name="citation_reference" content="citation_title=Effective de novo assembly of fish genome using haploid larvae;  citation_journal_title=Gene;  citation_year=2016;  citation_volume=576;  citation_pages=644-9; " /><meta name="citation_reference" content="citation_title=Aligning sequence reads, clone sequences and assembly contigs with BWA-MEM;  citation_journal_title=arXiv Preprint, arXiv;  citation_year=2013;  citation_volume=1303;  citation_pages=3997" /><meta name="citation_reference" content="citation_title=Efficient de novo assembly of highly heterozygous genomes from whole-genome shotgun short reads;  citation_journal_title=Genome Res;  citation_year=2014;  citation_volume=24;  citation_pages=1384-95; " /><meta name="citation_reference" content="citation_title=Basic local alignment search tool;  citation_journal_title=J. Mol. Biol;  citation_year=1990;  citation_volume=215;  citation_pages=403-10; " /><meta name="citation_reference" content="citation_title=A ddRAD-based genetic map and its integration with the genome assembly of Japanese eel (Anguilla japonica) provides insights into genome evolution after the teleost-specific genome duplication;  citation_journal_title=BMC Genomics;  citation_year=2014;  citation_volume=15;  citation_pages=233" /><meta name="citation_reference" content="citation_title=L_RNA_scaffolder: scaffolding genomes with transcripts;  citation_journal_title=BMC Genomics;  citation_year=2013;  citation_volume=14;  citation_pages=604" /><meta name="citation_reference" content="citation_title=A fast, lock-free approach for efficient parallel counting of occurrences of k-mers;  citation_journal_title=Bioinformatics;  citation_year=2011;  citation_volume=27;  citation_pages=764-70; " /><meta name="citation_reference" content="citation_title=Stacks: building and genotyping Loci de novo from short-read sequences;  citation_journal_title=G3 (Bethesda);  citation_year=2011;  citation_volume=1;  citation_pages=171-82; " /><meta name="citation_reference" content="citation_title=A framework for variation discovery and genotyping using next-generation DNA sequencing data;  citation_journal_title=Nat. Genet;  citation_year=2011;  citation_volume=43;  citation_pages=491-8; " /><meta name="citation_reference" content="citation_title=R/qtl: qTL mapping in experimental crosses;  citation_journal_title=Bioinformatics;  citation_year=2003;  citation_volume=19;  citation_pages=889-90; " /><meta name="citation_reference" content="citation_title=gVolante for standardizing completeness assessment of genome and transcriptome assemblies;  citation_journal_title=Bioinformatics;  citation_year=2017;  citation_volume=33;  citation_pages=3635-37; " /><meta name="citation_reference" content="citation_title=Assessing the gene space in draft genomes;  citation_journal_title=Nucleic Acids Res;  citation_year=2009;  citation_volume=37;  citation_pages=289-97; " /><meta name="citation_reference" content="citation_title=Optimizing and benchmarking de novo transcriptome sequencing: from library preparation to assembly evaluation;  citation_journal_title=BMC Genomics;  citation_year=2015;  citation_volume=16;  citation_pages=977" /><meta name="citation_reference" content="citation_title=BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs;  citation_journal_title=Bioinformatics;  citation_year=2015;  citation_volume=31;  citation_pages=3210-2; " /><meta name="citation_reference" content="citation_title=Sequencing and de novo analysis of a coral larval transcriptome using 454 GSFlx;  citation_journal_title=BMC Genomics;  citation_year=2009;  citation_volume=10;  citation_pages=219" /><meta name="citation_reference" content="citation_title=Using native and syntenically mapped cDNA alignments to improve de novo gene finding;  citation_journal_title=Bioinformatics;  citation_year=2008;  citation_volume=24;  citation_pages=637-44; " /><meta name="citation_reference" content="Smit A., Hubley R., Green P.  1996–2010 , RepeatMasker Open-3.0. http://www.repeatmasker.org. " /><meta name="citation_reference" content="Smit A., Hubley R.  2008–2010 , RepeatModeler Open-1.0. http://www.repeatmasker.org. " /><meta name="citation_reference" content="citation_title=TopHat2: accurate alignment of transcriptomes in the presence of insertions, deletions and gene fusions;  citation_journal_title=Genome Biol;  citation_year=2013;  citation_volume=14;  citation_pages=R36" /><meta name="citation_reference" content="citation_title=Transcript assembly and quantification by RNA-Seq reveals unannotated transcripts and isoform switching during cell differentiation;  citation_journal_title=Nat. Biotechnol;  citation_year=2010;  citation_volume=28;  citation_pages=511-5; " /><meta name="citation_reference" content="citation_title=Ensembl 2016;  citation_journal_title=Nucleic Acids Res;  citation_year=2016;  citation_volume=44;  citation_pages=D710-6; " /><meta name="citation_reference" content="citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs;  citation_journal_title=Nucleic Acids Res;  citation_year=1997;  citation_volume=25;  citation_pages=3389-402; " /><meta name="citation_reference" content="citation_title=InterProScan 5: genome-scale protein function classification;  citation_journal_title=Bioinformatics;  citation_year=2014;  citation_volume=30;  citation_pages=1236-40; " /><meta name="citation_reference" content="citation_title=High-throughput functional annotation and data mining with the Blast2GO suite;  citation_journal_title=Nucleic Acids Res;  citation_year=2008;  citation_volume=36;  citation_pages=3420-35; " /><meta name="citation_reference" content="citation_title=KAAS: an automatic genome annotation and pathway reconstruction server;  citation_journal_title=Nucleic Acids Res;  citation_year=2007;  citation_volume=35;  citation_pages=W182-5; " /><meta name="citation_reference" content="citation_title=Evolutionary changes of multiple visual pigment genes in the complete genome of Pacific bluefin tuna;  citation_journal_title=Proc. Natl. Acad. Sci. U. S. A;  citation_year=2013;  citation_volume=110;  citation_pages=11061-6; " /><meta name="citation_reference" content="citation_title=The genome of the miiuy croaker reveals well-developed innate immune and sensory systems;  citation_journal_title=Sci. Rep;  citation_year=2016;  citation_volume=6;  citation_pages=21902" /><meta name="citation_reference" content="citation_title=OrthoVenn: a web server for genome wide comparison and annotation of orthologous clusters across multiple species;  citation_journal_title=Nucleic Acids Res;  citation_year=2015;  citation_volume=43;  citation_pages=W78-84; " /><meta name="citation_reference" content="citation_title=Measurement of mRNA abundance using RNA-seq data: rPKM measure is inconsistent among samples;  citation_journal_title=Theory Biosci;  citation_year=2012;  citation_volume=131;  citation_pages=281-5; " /><meta name="citation_reference" content="citation_title=Large-scale gene function analysis with the PANTHER classification system;  citation_journal_title=Nat. Protoc;  citation_year=2013;  citation_volume=8;  citation_pages=1551-66; " /><meta name="citation_reference" content="citation_title=Dramatic improvement in genome assembly achieved using doubled-haploid genomes;  citation_journal_title=Sci. Rep;  citation_year=2015;  citation_volume=4;  citation_pages=6780" /><meta name="citation_reference" content="citation_title=Profiling of gene duplication patterns of sequenced teleost genomes: evidence for rapid lineage-specific genome expansion mediated by recent tandem duplications;  citation_journal_title=BMC Genomics;  citation_year=2012;  citation_volume=13;  citation_pages=246" /><meta name="citation_reference" content="citation_title=Cloning, tissue and ontogenetic expression of the taurine transporter in the flatfish Senegalese sole (Solea senegalensis);  citation_journal_title=Amino Acids;  citation_year=2012;  citation_volume=42;  citation_pages=1317-27; " /><meta name="citation_reference" content="citation_title=Progress on larval and juvenile nutrition to improve the quality and health of seawater fish: a review;  citation_journal_title=Fish. Sci;  citation_year=2014;  citation_volume=80;  citation_pages=389-403; " /><meta name="citation_reference" content="citation_title=The green liver syndrome is caused by taurine deficiency in yellowtail, Seriola quinqueradiata Fed diets without fishmeal;  citation_journal_title=Aquacult. Sci;  citation_year=2005;  citation_volume=53;  citation_pages=279-90; " /><meta name="citation_reference" content="citation_title=Full genome survey and dynamics of gene expression in the greater amberjack Seriola dumerili;  citation_journal_title=GigaScience;  citation_year=2017;  citation_volume=6;  citation_pages=1-13; " /><meta name="citation_reference" content="citation_title=Insights into teleost sex determination from the Seriola dorsalis genome assembly;  citation_journal_title=BMC Genomics;  citation_year=2018;  citation_volume=19;  citation_pages=31" /><meta name="citation_reference" content="citation_title=Evolution of the immune system influences speciation rates in teleost fishes;  citation_journal_title=Nat. Genet;  citation_year=2016;  citation_volume=48;  citation_pages=1204-10; " /><meta name="citation_reference" content="citation_title=Linking species habitat and past palaeoclimatic events to evolution of the teleost innate immune system;  citation_journal_title=Proc. R Soc. B;  citation_year=2017;  citation_volume=284;  citation_pages=20162810" /><meta name="citation_reference" content="citation_title=Human gastric (H+ + K+)-ATPase gene. Similarity to (Na+ + K+)-ATPase genes in exon/intron organization but difference in control region;  citation_journal_title=J. Biol. Chem;  citation_year=1990;  citation_volume=265;  citation_pages=9027-32; " /><meta name="citation_reference" content="citation_title=Availability of dietary phosphorus in carp and rainbow trout;  citation_journal_title=Nippon Suisan Gakkaishi;  citation_year=1979;  citation_volume=45;  citation_pages=1527-32; " /><meta name="citation_reference" content="citation_title=Regulation of apical H(+)-ATPase activity and intestinal HCO(3)(-) secretion in marine fish osmoregulation;  citation_journal_title=Am. J. Physiol. Regul. Integr. Comp. Physiol;  citation_year=2011;  citation_volume=301;  citation_pages=R1682-91; " /><meta name="citation_reference" content="citation_title=Gene expression profiling reveals functional specialization along the intestinal tract of a carnivorous teleostean fish (Dicentrarchus labrax);  citation_journal_title=Front. Physiol;  citation_year=2016;  citation_volume=7;  citation_pages=359" /><meta name="citation_reference" content="citation_title=A nonelectrogenic H+ pump in plasma membranes of hog stomach;  citation_journal_title=J. Biol. Chem;  citation_year=1976;  citation_volume=251;  citation_pages=7690-8; " /><meta name="citation_reference" content="citation_title=Gastric proteinases–structure, function, evolution and mechanism of action;  citation_journal_title=Essays Biochem;  citation_year=1981;  citation_volume=17;  citation_pages=52-84; " /><meta name="citation_reference" content="citation_title=Partial characterization and activities of proteases from the digestive tract of discus fish (Symphysodon aequifasciata);  citation_journal_title=Aquaculture;  citation_year=2002;  citation_volume=203;  citation_pages=321-33; " /><meta name="citation_reference" content="citation_title=Enzyme activities and pH variations in the digestive tract of gilthead sea bream;  citation_journal_title=J. Fish Biol;  citation_year=2003;  citation_volume=62;  citation_pages=1033-43; " /><meta name="citation_reference" content="citation_publisher=AOCS Press, Champaign, IL;  citation_title=Nutrition and Utilization Technology in Aqualture;  citation_year=1995;  citation_pages=199-219; " /><meta name="citation_reference" content="citation_title=Morphological and molecular evidence for functional organization along the rostrocaudal axis of the adult zebrafish intestine;  citation_journal_title=BMC Genomics;  citation_year=2010;  citation_volume=11;  citation_pages=392" /><meta name="citation_reference" content="citation_title=Biochemistry of fish stomach chitinase;  citation_journal_title=Int. J. Biol. Macromol;  citation_year=2017;  citation_volume=104;  citation_pages=1672-81; " /><meta name="citation_reference" content="citation_title=Distribution of chitinase and β-N-acetylhexosaminidase in the organs of several fishes;  citation_journal_title=Fish. Sci;  citation_year=1996;  citation_volume=62;  citation_pages=150-1; " /><meta name="citation_reference" content="citation_title=Searching the evolutionary origin of epithelial mucus protein components-mucins and FCGBP;  citation_journal_title=Mol. Biol. Evol;  citation_year=2016;  citation_volume=33;  citation_pages=1921-36; " /><meta name="citation_reference" content="citation_title=The role of cysteine-rich secretory proteins in male fertility;  citation_journal_title=Asian J. Androl;  citation_year=2011;  citation_volume=13;  citation_pages=111-7; " /><meta name="citation_reference" content="citation_title=Intestinal absorption of protein in teleost fish;  citation_journal_title=Comp. Biochem. Physiol. A Physiol;  citation_year=1992;  citation_volume=103;  citation_pages=771-81; " /><meta name="citation_reference" content="citation_title=Effects of diets containing soybean meal on trypsin mRNA expression and activity in Atlantic salmon (Salmo salar L);  citation_journal_title=Comp. Biochem. Physiol. A Mol. Integr. Physiol;  citation_year=2007;  citation_volume=147;  citation_pages=25-36; " /><meta name="citation_reference" content="citation_title=Molecular characterization and gene expression of six trypsinogens in the flatfish Senegalese sole (Solea senegalensis Kaup) during larval development and in tissues;  citation_journal_title=Comp. Biochem. Physiol. B, Biochem. Mol. Biol;  citation_year=2008;  citation_volume=149;  citation_pages=334-44; " /><meta name="citation_reference" content="citation_title=Effects of feeding with corn gluten meal on trypsin activity and mRNA expression in Fugu obscurus;  citation_journal_title=Fish Physiol. Biochem;  citation_year=2011;  citation_volume=37;  citation_pages=453-60; " /><meta name="citation_reference" content="citation_title=Isolation and characterization of some novel genes of the apolipoprotein A-I family in Japanese eel, Anguilla Japonica;  citation_journal_title=Cent. Eur. J. Biol;  citation_year=2011;  citation_volume=6;  citation_pages=545" /><meta name="citation_reference" content="citation_title=Isolation and characterization of the apolipoprotein multigene family in Hemibarbus mylodon (Teleostei: cypriniformes);  citation_journal_title=Comp. Biochem. Physiol. B Biochem. Mol. Biol;  citation_year=2009;  citation_volume=152;  citation_pages=38-46; " /><meta name="citation_reference" content="citation_title=Cloning and expression analysis of apolipoprotein A-I (ApoA-I) in the Hong Kong grouper (Epinephelus akaara);  citation_journal_title=Aquaculture;  citation_year=2014;  citation_volume=432;  citation_pages=85-96; " /><meta name="citation_reference" content="citation_title=Isolation and characterization of an antifreeze protein from the longhorn sculpin, Myoxocephalus octodecimspinosis;  citation_journal_title=Biochim. Biophys. Acta;  citation_year=1998;  citation_volume=1388;  citation_pages=305-14; " /><meta name="citation_reference" content="citation_title=Lipid nutrition in fish;  citation_journal_title=Fish Physiol. Biochem;  citation_year=1986;  citation_volume=2;  citation_pages=121" /><meta name="citation_reference" content="citation_title=Acidic mammalian chitinase is a proteases-resistant glycosidase in mouse digestive system;  citation_journal_title=Sci. Rep;  citation_year=2016;  citation_volume=6;  citation_pages=37756" /><meta name="citation_reference" content="citation_title=Acidic chitinase primes the protective immune response to gastrointestinal nematodes;  citation_journal_title=Nat. Immunol;  citation_year=2016;  citation_volume=17;  citation_pages=538-44; " /><meta name="citation_reference" content="citation_title=Transcriptomic responses in the fish intestine;  citation_journal_title=Dev. Comp. Immunol;  citation_year=2016;  citation_volume=64;  citation_pages=103-17; " /><meta name="citation_reference" content="citation_title=Adaptive immune responses at mucosal surfaces of teleost fish;  citation_journal_title=Fish Shellfish Immunol;  citation_year=2014;  citation_volume=40;  citation_pages=634-43; " /><meta name="citation_reference" content="citation_title=Local expression of apolipoprotein A-I gene and a possible role for HDL in primary defence in the carp skin;  citation_journal_title=Fish Shellfish Immunol;  citation_year=2003;  citation_volume=14;  citation_pages=259-73; " /><meta name="citation_reference" content="citation_title=Apolipoproteins A-I and A-II are potentially important effectors of innate immunity in the teleost fish,;  citation_journal_title=Cyprinus Carpio. Eur. J. Biochem;  citation_year=2004;  citation_volume=271;  citation_pages=2984-90; " /><meta name="citation_reference" content="citation_title=Apolipoprotein A-I, an antimicrobial protein in Oncorhynchus mykiss: evaluation of its expression in primary defence barriers and plasma levels in sick and healthy fish;  citation_journal_title=Fish Shellfish Immunol;  citation_year=2007;  citation_volume=23;  citation_pages=197-209; " /><meta name="citation_reference" content="citation_title=Graded effects of unregulated smooth muscle myosin on intestinal architecture, intestinal motility and vascular function in zebrafish;  citation_journal_title=Dis. Model. Mech;  citation_year=2016;  citation_volume=9;  citation_pages=529-40; " /><meta name="citation_reference" content="citation_title=Mechanical stretch modulates TGF-β1 and α1(I) collagen expression in fetal human intestinal smooth muscle cells;  citation_journal_title=Am. J. Physiol. Gastrointest. Liver Physiol;  citation_year=1999;  citation_volume=277;  citation_pages=G1074-80; " /><meta name="citation_reference" content="citation_title=The digestive tract of the amberjack Seriola dumerili, Risso: a light and scanning electron microscope study;  citation_journal_title=J. Fish Biol;  citation_year=1992;  citation_volume=41;  citation_pages=287-303; " /><meta name="citation_reference" content="citation_title=Endocytic receptors in the renal proximal tubule;  citation_journal_title=Physiology (Bethesda);  citation_year=2012;  citation_volume=27;  citation_pages=223-36; " /><meta name="citation_reference" content="citation_title=Tubular reabsorption of albumin: it's all about cubilin;  citation_journal_title=J. Am. Soc. Nephrol;  citation_year=2010;  citation_volume=21;  citation_pages=1810-2; " /><meta name="citation_reference" content="citation_title=A mannose-receptor is possibly involved in the phagocytosis of Saccharomyces cerevisiae by seabream (Sparus aurata L.) leucocytes;  citation_journal_title=Fish Shellfish Immunol;  citation_year=2003;  citation_volume=14;  citation_pages=375-88; " /><meta name="citation_reference" content="citation_title=Regulation of DMBT1 via NOD2 and TLR4 in intestinal epithelial cells modulates bacterial recognition and invasion;  citation_journal_title=J. Immunol;  citation_year=2007;  citation_volume=178;  citation_pages=8203-11; " /><meta name="citation_reference" content="citation_title=Structural organization and complete sequence of the human α-N-acetylgalactosaminidase gene: homology with the α-galactosidase A gene provides evidence for evolution from a common ancestral gene;  citation_journal_title=Genomics;  citation_year=1991;  citation_volume=10;  citation_pages=133-42; " /><meta name="citation_reference" content="citation_title=A quantitative study of some digestive enzymes in the rabbitfish, Siganus canaliculatus and the sea bass, Lates calcarifer;  citation_journal_title=J. Fish Biol;  citation_year=1993;  citation_volume=42;  citation_pages=595-602; " /><meta name="citation_reference" content="citation_publisher=Academic Press, London;  citation_title=Advances in Marine Biology;  citation_year=1976;  citation_pages=109-239; " /><meta name="citation_reference" content="citation_title=Comparative study of digestive enzymes in fish with different nutritional habits. Proteolytic and amylase activities;  citation_journal_title=Aquaculture;  citation_year=1999;  citation_volume=170;  citation_pages=267-83; " /><meta name="citation_reference" content="citation_title=Do digestive enzymes set a physiological limit on growth rate and food conversion efficiency in the Atlantic cod (Gadus morhua)?;  citation_journal_title=Fish Physiol. Biochem;  citation_year=1999;  citation_volume=20;  citation_pages=293-303; " /><meta name="citation_reference" content="citation_title=Comparative study of proteolytic enzymes in the digestive tracts of the european sea bass and hybrid striped bass reared in freshwater;  citation_journal_title=Comp. Biochem. Physiol. A Physiol;  citation_year=1993;  citation_volume=106;  citation_pages=627-34; " /><meta name="citation_reference" content="citation_title=Proteolytic digestive enzymes of carnivorous (Silurus glanis L.), herbivorous (Hypophthalmichthys molitrix Val.) and omnivorous (Cyprinus carpio L.) fishes;  citation_journal_title=Aquaculture;  citation_year=1983;  citation_volume=30;  citation_pages=145-54; " /><meta name="citation_reference" content="citation_title=Pepsinogen A and C genes in turbot (Scophthalmus maximus): characterization and expression in early development;  citation_journal_title=Comp. Biochem. Physiol. B, Biochem. Mol. Biol;  citation_year=2013;  citation_volume=165;  citation_pages=58-65; " /><meta name="citation_reference" content="citation_title=Atlantic cod trypsins: from basic research to practical applications;  citation_journal_title=Mar. Biotechnol;  citation_year=2005;  citation_volume=7;  citation_pages=77-88; " /><meta name="citation_reference" content="citation_title=Isolation and characterization of a cDNA encoding rat cationic trypsinogen;  citation_journal_title=Biochemistry;  citation_year=1987;  citation_volume=26;  citation_pages=3081-6; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/25/5/547/26076953/dsy024.pdf" /><meta name="description" content="Abstract.  Seriola quinqueradiata (yellowtail) is the most widely farmed and economically important fish in aquaculture in Japan. In this study, we used the gen" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/25/5/547/5059530" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/25/5/547/5059530" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/25/5/547/5059530");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2796125&account_list=2796125,2777753&authnips=128.235.12.133&doi=10.1093/dnares/dsy024';
    </script>
<title>
	yellowtail (Seriola quinqueradiata) genome and transcriptome atlas of the digestive tract | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./5059530" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BfDqATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iWWFzdWlrZSwgTW90b3NoaWdlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iUmVzZWFyY2ggQ2VudGVyIGZvciBCaW9pbmZvcm1hdGljcyBhbmQgQmlvc2NpZW5jZXMsIE5hdGlvbmFsIFJlc2VhcmNoIEluc3RpdHV0ZSBvZiBGaXNoZXJpZXMgU2NpZW5jZSwgSmFwYW4gRmlzaGVyaWVzIFJlc2VhcmNoIGFuZCBFZHVjYXRpb24gQWdlbmN5LCBZb2tvaGFtYSwgS2FuYWdhd2EsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSXdhc2FraSwgWXVraSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlJlc2VhcmNoIENlbnRlciBmb3IgQmlvaW5mb3JtYXRpY3MgYW5kIEJpb3NjaWVuY2VzLCBOYXRpb25hbCBSZXNlYXJjaCBJbnN0aXR1dGUgb2YgRmlzaGVyaWVzIFNjaWVuY2UsIEphcGFuIEZpc2hlcmllcyBSZXNlYXJjaCBhbmQgRWR1Y2F0aW9uIEFnZW5jeSwgWW9rb2hhbWEsIEthbmFnYXdhLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik5pc2hpa2ksIElzc2VpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iUmVzZWFyY2ggQ2VudGVyIGZvciBCaW9pbmZvcm1hdGljcyBhbmQgQmlvc2NpZW5jZXMsIE5hdGlvbmFsIFJlc2VhcmNoIEluc3RpdHV0ZSBvZiBGaXNoZXJpZXMgU2NpZW5jZSwgSmFwYW4gRmlzaGVyaWVzIFJlc2VhcmNoIGFuZCBFZHVjYXRpb24gQWdlbmN5LCBZb2tvaGFtYSwgS2FuYWdhd2EsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTmFrYW11cmEsIFlvamkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJSZXNlYXJjaCBDZW50ZXIgZm9yIEJpb2luZm9ybWF0aWNzIGFuZCBCaW9zY2llbmNlcywgTmF0aW9uYWwgUmVzZWFyY2ggSW5zdGl0dXRlIG9mIEZpc2hlcmllcyBTY2llbmNlLCBKYXBhbiBGaXNoZXJpZXMgUmVzZWFyY2ggYW5kIEVkdWNhdGlvbiBBZ2VuY3ksIFlva29oYW1hLCBLYW5hZ2F3YSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJNYXRzdXVyYSwgQWlrbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlJlc2VhcmNoIENlbnRlciBmb3IgQmlvaW5mb3JtYXRpY3MgYW5kIEJpb3NjaWVuY2VzLCBOYXRpb25hbCBSZXNlYXJjaCBJbnN0aXR1dGUgb2YgRmlzaGVyaWVzIFNjaWVuY2UsIEphcGFuIEZpc2hlcmllcyBSZXNlYXJjaCBhbmQgRWR1Y2F0aW9uIEFnZW5jeSwgWW9rb2hhbWEsIEthbmFnYXdhLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Illvc2hpZGEsIEthenVub3JpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iR290byBMYWJvcmF0b3J5LCBTdG9jayBFbmhhbmNlbWVudCBhbmQgQXF1YWN1bHR1cmUgRGl2aXNpb24sIFNlaWthaSBOYXRpb25hbCBGaXNoZXJpZXMgUmVzZWFyY2ggSW5zdGl0dXRlIEphcGFuIEZpc2hlcmllcyBSZXNlYXJjaCBhbmQgRWR1Y2F0aW9uIEFnZW5jeSwgVGFtYW5vdXJhLWNobywgR290bywgTmFnYXNha2ksIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTm9kYSwgVHN1dG9tdSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkdvdG8gTGFib3JhdG9yeSwgU3RvY2sgRW5oYW5jZW1lbnQgYW5kIEFxdWFjdWx0dXJlIERpdmlzaW9uLCBTZWlrYWkgTmF0aW9uYWwgRmlzaGVyaWVzIFJlc2VhcmNoIEluc3RpdHV0ZSBKYXBhbiBGaXNoZXJpZXMgUmVzZWFyY2ggYW5kIEVkdWNhdGlvbiBBZ2VuY3ksIFRhbWFub3VyYS1jaG8sIEdvdG8sIE5hZ2FzYWtpLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkFuZG9oLCBUYWRhc2hpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iU3RvY2sgRW5oYW5jZW1lbnQgYW5kIEFxdWFjdWx0dXJlIERpdmlzaW9uLCBTZWlrYWkgTmF0aW9uYWwgRmlzaGVyaWVzIFJlc2VhcmNoIEluc3RpdHV0ZSwgSmFwYW4gRmlzaGVyaWVzIFJlc2VhcmNoIGFuZCBFZHVjYXRpb24gQWdlbmN5LCBOYWdhc2FraSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJGdWppd2FyYSwgQXR1c2hpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iUmVzZWFyY2ggQ2VudGVyIGZvciBCaW9pbmZvcm1hdGljcyBhbmQgQmlvc2NpZW5jZXMsIE5hdGlvbmFsIFJlc2VhcmNoIEluc3RpdHV0ZSBvZiBGaXNoZXJpZXMgU2NpZW5jZSwgSmFwYW4gRmlzaGVyaWVzIFJlc2VhcmNoIGFuZCBFZHVjYXRpb24gQWdlbmN5LCBZb2tvaGFtYSwgS2FuYWdhd2EsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJUaGUgeWVsbG93dGFpbCAoU2VyaW9sYSBxdWlucXVlcmFkaWF0YSkgZ2Vub21lIGFuZCB0cmFuc2NyaXB0b21lIGF0bGFzIG9mIHRoZSBkaWdlc3RpdmUgdHJhY3QiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSI1NDciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjU2MCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzeTAyNCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIyNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDE4LzEwLzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9sZWN1bGFyIHN5c3RlbWF0aWNzIGFuZCBiaW9nZW9ncmFwaHkgb2YgdGhlIGNpcmN1bWdsb2JhbGx5IGRpc3RyaWJ1dGVkIGdlbnVzIFNlcmlvbGEgKFBpc2NlczogY2FyYW5naWRhZSk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gUGh5bG9nZW5ldC4gRXZvbDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT05MzsgIGNpdGF0aW9uX3BhZ2VzPTI3NC04MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIFN0YXRlIG9mIFNlcmlvbGEgc3BwLiBPdGhlciBUaGFuIFllbGxvd3RhaWwgKFMuIHF1aW5xdWVyYWRpYXRhKSBGYXJtaW5nIGluIHRoZSBXb3JsZDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UmV2LiBGaXNoIFNjaS4gQXF1YWM7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9MjQ7ICBjaXRhdGlvbl9wYWdlcz0zMTQtMjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX2F1dGhvcj1NaW5pc3RyeSBvZiBBZ3JpY3VsdHVyZSwgRm9yZXN0cnkgYW5kIEZpc2hlcmllcyBvZiBKYXBhbjsgIGNpdGF0aW9uX3llYXI9MjAxNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fcHVibGlzaGVyPUZBTyBGaXNoZXJpZXMgYW5kIEFxdWFjdWx0dXJlIERlcGFydG1lbnQsIFJvbWU7ICBjaXRhdGlvbl90aXRsZT1DdWx0dXJlZCBBcXVhdGljIFNwZWNpZXMgSW5mb3JtYXRpb24gUHJvZ3JhbW1lOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fcGFnZXM9MS0xMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fYXV0aG9yPVdvcmxkIEJhbms7IGNpdGF0aW9uX3B1Ymxpc2hlcj1Xb3JsZCBCYW5rIEdyb3VwLCBXYXNoaW5ndG9uIERDOyAgY2l0YXRpb25fdGl0bGU9RmlzaCB0byAyMDMwOiBQcm9zcGVjdHMgZm9yIEZpc2hlcmllcyBhbmQgQXF1YWN1bHR1cmUuIEFncmljdWx0dXJlIGFuZCBFbnZpcm9ubWVudGFsIFNlcnZpY2VzIERpc2N1c3Npb24gUGFwZXIsIE5vLiAzOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fcGFnZXM9MS0xMDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFxdWFjdWx0dXJlIGdlbm9taWNzLCBnZW5ldGljcyBhbmQgYnJlZWRpbmcgaW4gdGhlIFVuaXRlZCBTdGF0ZXM6IGN1cnJlbnQgc3RhdHVzLCBjaGFsbGVuZ2VzLCBhbmQgcHJpb3JpdGllcyBmb3IgZnV0dXJlIHJlc2VhcmNoOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTc7ICBjaXRhdGlvbl92b2x1bWU9MTg7ICBjaXRhdGlvbl9wYWdlcz0xOTEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GdW5jdGlvbmFsIEFubm90YXRpb24gb2YgQWxsIFNhbG1vbmlkIEdlbm9tZXMgKEZBQVNHKTogYW4gaW50ZXJuYXRpb25hbCBpbml0aWF0aXZlIHN1cHBvcnRpbmcgZnV0dXJlIHNhbG1vbmlkIHJlc2VhcmNoLCBjb25zZXJ2YXRpb24gYW5kIGFxdWFjdWx0dXJlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTc7ICBjaXRhdGlvbl92b2x1bWU9MTg7ICBjaXRhdGlvbl9wYWdlcz00ODQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DdXJyZW50IHN0YXR1cyBvZiBnZW5vbWUgc2VxdWVuY2luZyBhbmQgaXRzIGFwcGxpY2F0aW9ucyBpbiBhcXVhY3VsdHVyZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QXF1YWN1bHR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMTc7ICBjaXRhdGlvbl92b2x1bWU9NDY4OyAgY2l0YXRpb25fcGFnZXM9MzM3LTQ3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZWNvbmQgZ2VuZXJhdGlvbiBwaHlzaWNhbCBhbmQgbGlua2FnZSBtYXBzIG9mIHllbGxvd3RhaWwgKFNlcmlvbGEgcXVpbnF1ZXJhZGlhdGEpIGFuZCBjb21wYXJpc29uIG9mIHN5bnRlbnkgd2l0aCBmb3VyIG1vZGVsIGZpc2g7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTQwNiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbnN0cnVjdGlvbiBvZiBhIGhpZ2gtY292ZXJhZ2UgYmFjdGVyaWFsIGFydGlmaWNpYWwgY2hyb21vc29tZSBsaWJyYXJ5IGFuZCBjb21wcmVoZW5zaXZlIGdlbmV0aWMgbGlua2FnZSBtYXAgb2YgeWVsbG93dGFpbCBTZXJpb2xhIHF1aW5xdWVyYWRpYXRhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgUmVzLiBOb3RlczsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9MjAwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpYyBsaW5rYWdlIG1hcHMgb2YgdHdvIHllbGxvd3RhaWxzIChTZXJpb2xhIHF1aW5xdWVyYWRpYXRhIGFuZCBTZXJpb2xhIGxhbGFuZGkpOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1BcXVhY3VsdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0yNDQ7ICBjaXRhdGlvbl9wYWdlcz00MS04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db25zdHJ1Y3Rpb24gb2YgYSByYWRpYXRpb24gaHlicmlkIHBhbmVsIGFuZCB0aGUgZmlyc3QgeWVsbG93dGFpbCAoU2VyaW9sYSBxdWlucXVlcmFkaWF0YSkgcmFkaWF0aW9uIGh5YnJpZCBtYXAgdXNpbmcgYSBuYW5vZmx1aWRpYyBkeW5hbWljIGFycmF5OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz0xNjUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1RdWFudGl0YXRpdmUgdHJhaXQgbG9jaSAoUVRMKSBhc3NvY2lhdGVkIHdpdGggcmVzaXN0YW5jZSB0byBhIG1vbm9nZW5lYW4gcGFyYXNpdGUgKEJlbmVkZW5pYSBzZXJpb2xhZSkgaW4geWVsbG93dGFpbCAoU2VyaW9sYSBxdWlucXVlcmFkaWF0YSkgdGhyb3VnaCBnZW5vbWUgd2lkZSBhbmFseXNpczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxvUyBPbmU7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPWU2NDk4NyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZpY2F0aW9uIG9mIHRoZSBzZXgtbGlua2VkIGxvY3VzIGluIHllbGxvd3RhaWwsIFNlcmlvbGEgcXVpbnF1ZXJhZGlhdGE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFxdWFjdWx0dXJlOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTMwODsgIGNpdGF0aW9uX3BhZ2VzPVM1MS01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JZGVudGlmaWNhdGlvbiBvZiBzZXgtbGlua2VkIHNucHMgYW5kIHNleC1kZXRlcm1pbmluZyByZWdpb25zIGluIHRoZSB5ZWxsb3d0YWlsIGdlbm9tZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TWFyLiBCaW90ZWNobm9sLjsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0xNzsgIGNpdGF0aW9uX3BhZ2VzPTUwMi0xMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RWZmZWN0aXZlIGRlIG5vdm8gYXNzZW1ibHkgb2YgZmlzaCBnZW5vbWUgdXNpbmcgaGFwbG9pZCBsYXJ2YWU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbmU7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9NTc2OyAgY2l0YXRpb25fcGFnZXM9NjQ0LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFsaWduaW5nIHNlcXVlbmNlIHJlYWRzLCBjbG9uZSBzZXF1ZW5jZXMgYW5kIGFzc2VtYmx5IGNvbnRpZ3Mgd2l0aCBCV0EtTUVNOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1hclhpdiBQcmVwcmludCwgYXJYaXY7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTMwMzsgIGNpdGF0aW9uX3BhZ2VzPTM5OTciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FZmZpY2llbnQgZGUgbm92byBhc3NlbWJseSBvZiBoaWdobHkgaGV0ZXJvenlnb3VzIGdlbm9tZXMgZnJvbSB3aG9sZS1nZW5vbWUgc2hvdGd1biBzaG9ydCByZWFkczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTEzODQtOTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJhc2ljIGxvY2FsIGFsaWdubWVudCBzZWFyY2ggdG9vbDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gTW9sLiBCaW9sOyAgY2l0YXRpb25feWVhcj0xOTkwOyAgY2l0YXRpb25fdm9sdW1lPTIxNTsgIGNpdGF0aW9uX3BhZ2VzPTQwMy0xMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBkZFJBRC1iYXNlZCBnZW5ldGljIG1hcCBhbmQgaXRzIGludGVncmF0aW9uIHdpdGggdGhlIGdlbm9tZSBhc3NlbWJseSBvZiBKYXBhbmVzZSBlZWwgKEFuZ3VpbGxhIGphcG9uaWNhKSBwcm92aWRlcyBpbnNpZ2h0cyBpbnRvIGdlbm9tZSBldm9sdXRpb24gYWZ0ZXIgdGhlIHRlbGVvc3Qtc3BlY2lmaWMgZ2Vub21lIGR1cGxpY2F0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz0yMzMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1MX1JOQV9zY2FmZm9sZGVyOiBzY2FmZm9sZGluZyBnZW5vbWVzIHdpdGggdHJhbnNjcmlwdHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTYwNCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgZmFzdCwgbG9jay1mcmVlIGFwcHJvYWNoIGZvciBlZmZpY2llbnQgcGFyYWxsZWwgY291bnRpbmcgb2Ygb2NjdXJyZW5jZXMgb2Ygay1tZXJzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0yNzsgIGNpdGF0aW9uX3BhZ2VzPTc2NC03MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3RhY2tzOiBidWlsZGluZyBhbmQgZ2Vub3R5cGluZyBMb2NpIGRlIG5vdm8gZnJvbSBzaG9ydC1yZWFkIHNlcXVlbmNlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RzMgKEJldGhlc2RhKTsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xOyAgY2l0YXRpb25fcGFnZXM9MTcxLTgyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGZyYW1ld29yayBmb3IgdmFyaWF0aW9uIGRpc2NvdmVyeSBhbmQgZ2Vub3R5cGluZyB1c2luZyBuZXh0LWdlbmVyYXRpb24gRE5BIHNlcXVlbmNpbmcgZGF0YTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT00MzsgIGNpdGF0aW9uX3BhZ2VzPTQ5MS04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SL3F0bDogcVRMIG1hcHBpbmcgaW4gZXhwZXJpbWVudGFsIGNyb3NzZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTE5OyAgY2l0YXRpb25fcGFnZXM9ODg5LTkwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1nVm9sYW50ZSBmb3Igc3RhbmRhcmRpemluZyBjb21wbGV0ZW5lc3MgYXNzZXNzbWVudCBvZiBnZW5vbWUgYW5kIHRyYW5zY3JpcHRvbWUgYXNzZW1ibGllczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTc7ICBjaXRhdGlvbl92b2x1bWU9MzM7ICBjaXRhdGlvbl9wYWdlcz0zNjM1LTM3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Bc3Nlc3NpbmcgdGhlIGdlbmUgc3BhY2UgaW4gZHJhZnQgZ2Vub21lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9Mzc7ICBjaXRhdGlvbl9wYWdlcz0yODktOTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU9wdGltaXppbmcgYW5kIGJlbmNobWFya2luZyBkZSBub3ZvIHRyYW5zY3JpcHRvbWUgc2VxdWVuY2luZzogZnJvbSBsaWJyYXJ5IHByZXBhcmF0aW9uIHRvIGFzc2VtYmx5IGV2YWx1YXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTk3NyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJVU0NPOiBhc3Nlc3NpbmcgZ2Vub21lIGFzc2VtYmx5IGFuZCBhbm5vdGF0aW9uIGNvbXBsZXRlbmVzcyB3aXRoIHNpbmdsZS1jb3B5IG9ydGhvbG9nczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MzE7ICBjaXRhdGlvbl9wYWdlcz0zMjEwLTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNlcXVlbmNpbmcgYW5kIGRlIG5vdm8gYW5hbHlzaXMgb2YgYSBjb3JhbCBsYXJ2YWwgdHJhbnNjcmlwdG9tZSB1c2luZyA0NTQgR1NGbHg7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTIxOSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVVzaW5nIG5hdGl2ZSBhbmQgc3ludGVuaWNhbGx5IG1hcHBlZCBjRE5BIGFsaWdubWVudHMgdG8gaW1wcm92ZSBkZSBub3ZvIGdlbmUgZmluZGluZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MjQ7ICBjaXRhdGlvbl9wYWdlcz02MzctNDQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9IlNtaXQgQS4sIEh1YmxleSBSLiwgR3JlZW4gUC4gIDE5OTbigJMyMDEwICwgUmVwZWF0TWFza2VyIE9wZW4tMy4wLiBodHRwOi8vd3d3LnJlcGVhdG1hc2tlci5vcmcuICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9IlNtaXQgQS4sIEh1YmxleSBSLiAgMjAwOOKAkzIwMTAgLCBSZXBlYXRNb2RlbGVyIE9wZW4tMS4wLiBodHRwOi8vd3d3LnJlcGVhdG1hc2tlci5vcmcuICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRvcEhhdDI6IGFjY3VyYXRlIGFsaWdubWVudCBvZiB0cmFuc2NyaXB0b21lcyBpbiB0aGUgcHJlc2VuY2Ugb2YgaW5zZXJ0aW9ucywgZGVsZXRpb25zIGFuZCBnZW5lIGZ1c2lvbnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9UjM2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNjcmlwdCBhc3NlbWJseSBhbmQgcXVhbnRpZmljYXRpb24gYnkgUk5BLVNlcSByZXZlYWxzIHVuYW5ub3RhdGVkIHRyYW5zY3JpcHRzIGFuZCBpc29mb3JtIHN3aXRjaGluZyBkdXJpbmcgY2VsbCBkaWZmZXJlbnRpYXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gQmlvdGVjaG5vbDsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0yODsgIGNpdGF0aW9uX3BhZ2VzPTUxMS01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FbnNlbWJsIDIwMTY7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTQ0OyAgY2l0YXRpb25fcGFnZXM9RDcxMC02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HYXBwZWQgQkxBU1QgYW5kIFBTSS1CTEFTVDogYSBuZXcgZ2VuZXJhdGlvbiBvZiBwcm90ZWluIGRhdGFiYXNlIHNlYXJjaCBwcm9ncmFtczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0zMzg5LTQwMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW50ZXJQcm9TY2FuIDU6IGdlbm9tZS1zY2FsZSBwcm90ZWluIGZ1bmN0aW9uIGNsYXNzaWZpY2F0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTEyMzYtNDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhpZ2gtdGhyb3VnaHB1dCBmdW5jdGlvbmFsIGFubm90YXRpb24gYW5kIGRhdGEgbWluaW5nIHdpdGggdGhlIEJsYXN0MkdPIHN1aXRlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0zNjsgIGNpdGF0aW9uX3BhZ2VzPTM0MjAtMzU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUtBQVM6IGFuIGF1dG9tYXRpYyBnZW5vbWUgYW5ub3RhdGlvbiBhbmQgcGF0aHdheSByZWNvbnN0cnVjdGlvbiBzZXJ2ZXI7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTM1OyAgY2l0YXRpb25fcGFnZXM9VzE4Mi01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Fdm9sdXRpb25hcnkgY2hhbmdlcyBvZiBtdWx0aXBsZSB2aXN1YWwgcGlnbWVudCBnZW5lcyBpbiB0aGUgY29tcGxldGUgZ2Vub21lIG9mIFBhY2lmaWMgYmx1ZWZpbiB0dW5hOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFUuIFMuIEE7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTEwOyAgY2l0YXRpb25fcGFnZXM9MTEwNjEtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGdlbm9tZSBvZiB0aGUgbWlpdXkgY3JvYWtlciByZXZlYWxzIHdlbGwtZGV2ZWxvcGVkIGlubmF0ZSBpbW11bmUgYW5kIHNlbnNvcnkgc3lzdGVtczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpLiBSZXA7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTIxOTAyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9T3J0aG9WZW5uOiBhIHdlYiBzZXJ2ZXIgZm9yIGdlbm9tZSB3aWRlIGNvbXBhcmlzb24gYW5kIGFubm90YXRpb24gb2Ygb3J0aG9sb2dvdXMgY2x1c3RlcnMgYWNyb3NzIG11bHRpcGxlIHNwZWNpZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTQzOyAgY2l0YXRpb25fcGFnZXM9Vzc4LTg0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NZWFzdXJlbWVudCBvZiBtUk5BIGFidW5kYW5jZSB1c2luZyBSTkEtc2VxIGRhdGE6IHJQS00gbWVhc3VyZSBpcyBpbmNvbnNpc3RlbnQgYW1vbmcgc2FtcGxlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VGhlb3J5IEJpb3NjaTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMzE7ICBjaXRhdGlvbl9wYWdlcz0yODEtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TGFyZ2Utc2NhbGUgZ2VuZSBmdW5jdGlvbiBhbmFseXNpcyB3aXRoIHRoZSBQQU5USEVSIGNsYXNzaWZpY2F0aW9uIHN5c3RlbTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBQcm90b2M7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPTE1NTEtNjY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURyYW1hdGljIGltcHJvdmVtZW50IGluIGdlbm9tZSBhc3NlbWJseSBhY2hpZXZlZCB1c2luZyBkb3VibGVkLWhhcGxvaWQgZ2Vub21lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpLiBSZXA7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTY3ODAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qcm9maWxpbmcgb2YgZ2VuZSBkdXBsaWNhdGlvbiBwYXR0ZXJucyBvZiBzZXF1ZW5jZWQgdGVsZW9zdCBnZW5vbWVzOiBldmlkZW5jZSBmb3IgcmFwaWQgbGluZWFnZS1zcGVjaWZpYyBnZW5vbWUgZXhwYW5zaW9uIG1lZGlhdGVkIGJ5IHJlY2VudCB0YW5kZW0gZHVwbGljYXRpb25zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz0yNDYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DbG9uaW5nLCB0aXNzdWUgYW5kIG9udG9nZW5ldGljIGV4cHJlc3Npb24gb2YgdGhlIHRhdXJpbmUgdHJhbnNwb3J0ZXIgaW4gdGhlIGZsYXRmaXNoIFNlbmVnYWxlc2Ugc29sZSAoU29sZWEgc2VuZWdhbGVuc2lzKTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QW1pbm8gQWNpZHM7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NDI7ICBjaXRhdGlvbl9wYWdlcz0xMzE3LTI3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qcm9ncmVzcyBvbiBsYXJ2YWwgYW5kIGp1dmVuaWxlIG51dHJpdGlvbiB0byBpbXByb3ZlIHRoZSBxdWFsaXR5IGFuZCBoZWFsdGggb2Ygc2Vhd2F0ZXIgZmlzaDogYSByZXZpZXc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUZpc2guIFNjaTsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT04MDsgIGNpdGF0aW9uX3BhZ2VzPTM4OS00MDM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBncmVlbiBsaXZlciBzeW5kcm9tZSBpcyBjYXVzZWQgYnkgdGF1cmluZSBkZWZpY2llbmN5IGluIHllbGxvd3RhaWwsIFNlcmlvbGEgcXVpbnF1ZXJhZGlhdGEgRmVkIGRpZXRzIHdpdGhvdXQgZmlzaG1lYWw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFxdWFjdWx0LiBTY2k7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9NTM7ICBjaXRhdGlvbl9wYWdlcz0yNzktOTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZ1bGwgZ2Vub21lIHN1cnZleSBhbmQgZHluYW1pY3Mgb2YgZ2VuZSBleHByZXNzaW9uIGluIHRoZSBncmVhdGVyIGFtYmVyamFjayBTZXJpb2xhIGR1bWVyaWxpOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HaWdhU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAxNzsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9MS0xMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW5zaWdodHMgaW50byB0ZWxlb3N0IHNleCBkZXRlcm1pbmF0aW9uIGZyb20gdGhlIFNlcmlvbGEgZG9yc2FsaXMgZ2Vub21lIGFzc2VtYmx5OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTg7ICBjaXRhdGlvbl92b2x1bWU9MTk7ICBjaXRhdGlvbl9wYWdlcz0zMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2b2x1dGlvbiBvZiB0aGUgaW1tdW5lIHN5c3RlbSBpbmZsdWVuY2VzIHNwZWNpYXRpb24gcmF0ZXMgaW4gdGVsZW9zdCBmaXNoZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9NDg7ICBjaXRhdGlvbl9wYWdlcz0xMjA0LTEwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1MaW5raW5nIHNwZWNpZXMgaGFiaXRhdCBhbmQgcGFzdCBwYWxhZW9jbGltYXRpYyBldmVudHMgdG8gZXZvbHV0aW9uIG9mIHRoZSB0ZWxlb3N0IGlubmF0ZSBpbW11bmUgc3lzdGVtOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBSIFNvYy4gQjsgIGNpdGF0aW9uX3llYXI9MjAxNzsgIGNpdGF0aW9uX3ZvbHVtZT0yODQ7ICBjaXRhdGlvbl9wYWdlcz0yMDE2MjgxMCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUh1bWFuIGdhc3RyaWMgKEgrICsgSyspLUFUUGFzZSBnZW5lLiBTaW1pbGFyaXR5IHRvIChOYSsgKyBLKyktQVRQYXNlIGdlbmVzIGluIGV4b24vaW50cm9uIG9yZ2FuaXphdGlvbiBidXQgZGlmZmVyZW5jZSBpbiBjb250cm9sIHJlZ2lvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQmlvbC4gQ2hlbTsgIGNpdGF0aW9uX3llYXI9MTk5MDsgIGNpdGF0aW9uX3ZvbHVtZT0yNjU7ICBjaXRhdGlvbl9wYWdlcz05MDI3LTMyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BdmFpbGFiaWxpdHkgb2YgZGlldGFyeSBwaG9zcGhvcnVzIGluIGNhcnAgYW5kIHJhaW5ib3cgdHJvdXQ7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5pcHBvbiBTdWlzYW4gR2Fra2Fpc2hpOyAgY2l0YXRpb25feWVhcj0xOTc5OyAgY2l0YXRpb25fdm9sdW1lPTQ1OyAgY2l0YXRpb25fcGFnZXM9MTUyNy0zMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVndWxhdGlvbiBvZiBhcGljYWwgSCgrKS1BVFBhc2UgYWN0aXZpdHkgYW5kIGludGVzdGluYWwgSENPKDMpKC0pIHNlY3JldGlvbiBpbiBtYXJpbmUgZmlzaCBvc21vcmVndWxhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QW0uIEouIFBoeXNpb2wuIFJlZ3VsLiBJbnRlZ3IuIENvbXAuIFBoeXNpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MzAxOyAgY2l0YXRpb25fcGFnZXM9UjE2ODItOTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmUgZXhwcmVzc2lvbiBwcm9maWxpbmcgcmV2ZWFscyBmdW5jdGlvbmFsIHNwZWNpYWxpemF0aW9uIGFsb25nIHRoZSBpbnRlc3RpbmFsIHRyYWN0IG9mIGEgY2Fybml2b3JvdXMgdGVsZW9zdGVhbiBmaXNoIChEaWNlbnRyYXJjaHVzIGxhYnJheCk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUZyb250LiBQaHlzaW9sOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz0zNTkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIG5vbmVsZWN0cm9nZW5pYyBIKyBwdW1wIGluIHBsYXNtYSBtZW1icmFuZXMgb2YgaG9nIHN0b21hY2g7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEJpb2wuIENoZW07ICBjaXRhdGlvbl95ZWFyPTE5NzY7ICBjaXRhdGlvbl92b2x1bWU9MjUxOyAgY2l0YXRpb25fcGFnZXM9NzY5MC04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HYXN0cmljIHByb3RlaW5hc2Vz4oCTc3RydWN0dXJlLCBmdW5jdGlvbiwgZXZvbHV0aW9uIGFuZCBtZWNoYW5pc20gb2YgYWN0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Fc3NheXMgQmlvY2hlbTsgIGNpdGF0aW9uX3llYXI9MTk4MTsgIGNpdGF0aW9uX3ZvbHVtZT0xNzsgIGNpdGF0aW9uX3BhZ2VzPTUyLTg0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QYXJ0aWFsIGNoYXJhY3Rlcml6YXRpb24gYW5kIGFjdGl2aXRpZXMgb2YgcHJvdGVhc2VzIGZyb20gdGhlIGRpZ2VzdGl2ZSB0cmFjdCBvZiBkaXNjdXMgZmlzaCAoU3ltcGh5c29kb24gYWVxdWlmYXNjaWF0YSk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFxdWFjdWx0dXJlOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTIwMzsgIGNpdGF0aW9uX3BhZ2VzPTMyMS0zMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RW56eW1lIGFjdGl2aXRpZXMgYW5kIHBIIHZhcmlhdGlvbnMgaW4gdGhlIGRpZ2VzdGl2ZSB0cmFjdCBvZiBnaWx0aGVhZCBzZWEgYnJlYW07ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEZpc2ggQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT02MjsgIGNpdGF0aW9uX3BhZ2VzPTEwMzMtNDM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3B1Ymxpc2hlcj1BT0NTIFByZXNzLCBDaGFtcGFpZ24sIElMOyAgY2l0YXRpb25fdGl0bGU9TnV0cml0aW9uIGFuZCBVdGlsaXphdGlvbiBUZWNobm9sb2d5IGluIEFxdWFsdHVyZTsgIGNpdGF0aW9uX3llYXI9MTk5NTsgIGNpdGF0aW9uX3BhZ2VzPTE5OS0yMTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1vcnBob2xvZ2ljYWwgYW5kIG1vbGVjdWxhciBldmlkZW5jZSBmb3IgZnVuY3Rpb25hbCBvcmdhbml6YXRpb24gYWxvbmcgdGhlIHJvc3Ryb2NhdWRhbCBheGlzIG9mIHRoZSBhZHVsdCB6ZWJyYWZpc2ggaW50ZXN0aW5lOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz0zOTIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CaW9jaGVtaXN0cnkgb2YgZmlzaCBzdG9tYWNoIGNoaXRpbmFzZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9SW50LiBKLiBCaW9sLiBNYWNyb21vbDsgIGNpdGF0aW9uX3llYXI9MjAxNzsgIGNpdGF0aW9uX3ZvbHVtZT0xMDQ7ICBjaXRhdGlvbl9wYWdlcz0xNjcyLTgxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaXN0cmlidXRpb24gb2YgY2hpdGluYXNlIGFuZCDOsi1OLWFjZXR5bGhleG9zYW1pbmlkYXNlIGluIHRoZSBvcmdhbnMgb2Ygc2V2ZXJhbCBmaXNoZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUZpc2guIFNjaTsgIGNpdGF0aW9uX3llYXI9MTk5NjsgIGNpdGF0aW9uX3ZvbHVtZT02MjsgIGNpdGF0aW9uX3BhZ2VzPTE1MC0xOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZWFyY2hpbmcgdGhlIGV2b2x1dGlvbmFyeSBvcmlnaW4gb2YgZXBpdGhlbGlhbCBtdWN1cyBwcm90ZWluIGNvbXBvbmVudHMtbXVjaW5zIGFuZCBGQ0dCUDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBCaW9sLiBFdm9sOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTMzOyAgY2l0YXRpb25fcGFnZXM9MTkyMS0zNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHJvbGUgb2YgY3lzdGVpbmUtcmljaCBzZWNyZXRvcnkgcHJvdGVpbnMgaW4gbWFsZSBmZXJ0aWxpdHk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFzaWFuIEouIEFuZHJvbDsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTExMS03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbnRlc3RpbmFsIGFic29ycHRpb24gb2YgcHJvdGVpbiBpbiB0ZWxlb3N0IGZpc2g7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNvbXAuIEJpb2NoZW0uIFBoeXNpb2wuIEEgUGh5c2lvbDsgIGNpdGF0aW9uX3llYXI9MTk5MjsgIGNpdGF0aW9uX3ZvbHVtZT0xMDM7ICBjaXRhdGlvbl9wYWdlcz03NzEtODE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVmZmVjdHMgb2YgZGlldHMgY29udGFpbmluZyBzb3liZWFuIG1lYWwgb24gdHJ5cHNpbiBtUk5BIGV4cHJlc3Npb24gYW5kIGFjdGl2aXR5IGluIEF0bGFudGljIHNhbG1vbiAoU2FsbW8gc2FsYXIgTCk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNvbXAuIEJpb2NoZW0uIFBoeXNpb2wuIEEgTW9sLiBJbnRlZ3IuIFBoeXNpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTQ3OyAgY2l0YXRpb25fcGFnZXM9MjUtMzY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1vbGVjdWxhciBjaGFyYWN0ZXJpemF0aW9uIGFuZCBnZW5lIGV4cHJlc3Npb24gb2Ygc2l4IHRyeXBzaW5vZ2VucyBpbiB0aGUgZmxhdGZpc2ggU2VuZWdhbGVzZSBzb2xlIChTb2xlYSBzZW5lZ2FsZW5zaXMgS2F1cCkgZHVyaW5nIGxhcnZhbCBkZXZlbG9wbWVudCBhbmQgaW4gdGlzc3VlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q29tcC4gQmlvY2hlbS4gUGh5c2lvbC4gQiwgQmlvY2hlbS4gTW9sLiBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTE0OTsgIGNpdGF0aW9uX3BhZ2VzPTMzNC00NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RWZmZWN0cyBvZiBmZWVkaW5nIHdpdGggY29ybiBnbHV0ZW4gbWVhbCBvbiB0cnlwc2luIGFjdGl2aXR5IGFuZCBtUk5BIGV4cHJlc3Npb24gaW4gRnVndSBvYnNjdXJ1czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RmlzaCBQaHlzaW9sLiBCaW9jaGVtOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTM3OyAgY2l0YXRpb25fcGFnZXM9NDUzLTYwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Jc29sYXRpb24gYW5kIGNoYXJhY3Rlcml6YXRpb24gb2Ygc29tZSBub3ZlbCBnZW5lcyBvZiB0aGUgYXBvbGlwb3Byb3RlaW4gQS1JIGZhbWlseSBpbiBKYXBhbmVzZSBlZWwsIEFuZ3VpbGxhIEphcG9uaWNhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DZW50LiBFdXIuIEouIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTU0NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlzb2xhdGlvbiBhbmQgY2hhcmFjdGVyaXphdGlvbiBvZiB0aGUgYXBvbGlwb3Byb3RlaW4gbXVsdGlnZW5lIGZhbWlseSBpbiBIZW1pYmFyYnVzIG15bG9kb24gKFRlbGVvc3RlaTogY3lwcmluaWZvcm1lcyk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNvbXAuIEJpb2NoZW0uIFBoeXNpb2wuIEIgQmlvY2hlbS4gTW9sLiBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTE1MjsgIGNpdGF0aW9uX3BhZ2VzPTM4LTQ2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DbG9uaW5nIGFuZCBleHByZXNzaW9uIGFuYWx5c2lzIG9mIGFwb2xpcG9wcm90ZWluIEEtSSAoQXBvQS1JKSBpbiB0aGUgSG9uZyBLb25nIGdyb3VwZXIgKEVwaW5lcGhlbHVzIGFrYWFyYSk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFxdWFjdWx0dXJlOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTQzMjsgIGNpdGF0aW9uX3BhZ2VzPTg1LTk2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Jc29sYXRpb24gYW5kIGNoYXJhY3Rlcml6YXRpb24gb2YgYW4gYW50aWZyZWV6ZSBwcm90ZWluIGZyb20gdGhlIGxvbmdob3JuIHNjdWxwaW4sIE15b3hvY2VwaGFsdXMgb2N0b2RlY2ltc3Bpbm9zaXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2NoaW0uIEJpb3BoeXMuIEFjdGE7ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9MTM4ODsgIGNpdGF0aW9uX3BhZ2VzPTMwNS0xNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TGlwaWQgbnV0cml0aW9uIGluIGZpc2g7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUZpc2ggUGh5c2lvbC4gQmlvY2hlbTsgIGNpdGF0aW9uX3llYXI9MTk4NjsgIGNpdGF0aW9uX3ZvbHVtZT0yOyAgY2l0YXRpb25fcGFnZXM9MTIxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWNpZGljIG1hbW1hbGlhbiBjaGl0aW5hc2UgaXMgYSBwcm90ZWFzZXMtcmVzaXN0YW50IGdseWNvc2lkYXNlIGluIG1vdXNlIGRpZ2VzdGl2ZSBzeXN0ZW07ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaS4gUmVwOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz0zNzc1NiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFjaWRpYyBjaGl0aW5hc2UgcHJpbWVzIHRoZSBwcm90ZWN0aXZlIGltbXVuZSByZXNwb25zZSB0byBnYXN0cm9pbnRlc3RpbmFsIG5lbWF0b2RlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBJbW11bm9sOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTE3OyAgY2l0YXRpb25fcGFnZXM9NTM4LTQ0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2NyaXB0b21pYyByZXNwb25zZXMgaW4gdGhlIGZpc2ggaW50ZXN0aW5lOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1EZXYuIENvbXAuIEltbXVub2w7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9NjQ7ICBjaXRhdGlvbl9wYWdlcz0xMDMtMTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFkYXB0aXZlIGltbXVuZSByZXNwb25zZXMgYXQgbXVjb3NhbCBzdXJmYWNlcyBvZiB0ZWxlb3N0IGZpc2g7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUZpc2ggU2hlbGxmaXNoIEltbXVub2w7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9NDA7ICBjaXRhdGlvbl9wYWdlcz02MzQtNDM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxvY2FsIGV4cHJlc3Npb24gb2YgYXBvbGlwb3Byb3RlaW4gQS1JIGdlbmUgYW5kIGEgcG9zc2libGUgcm9sZSBmb3IgSERMIGluIHByaW1hcnkgZGVmZW5jZSBpbiB0aGUgY2FycCBza2luOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1GaXNoIFNoZWxsZmlzaCBJbW11bm9sOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9MjU5LTczOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BcG9saXBvcHJvdGVpbnMgQS1JIGFuZCBBLUlJIGFyZSBwb3RlbnRpYWxseSBpbXBvcnRhbnQgZWZmZWN0b3JzIG9mIGlubmF0ZSBpbW11bml0eSBpbiB0aGUgdGVsZW9zdCBmaXNoLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q3lwcmludXMgQ2FycGlvLiBFdXIuIEouIEJpb2NoZW07ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MjcxOyAgY2l0YXRpb25fcGFnZXM9Mjk4NC05MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXBvbGlwb3Byb3RlaW4gQS1JLCBhbiBhbnRpbWljcm9iaWFsIHByb3RlaW4gaW4gT25jb3JoeW5jaHVzIG15a2lzczogZXZhbHVhdGlvbiBvZiBpdHMgZXhwcmVzc2lvbiBpbiBwcmltYXJ5IGRlZmVuY2UgYmFycmllcnMgYW5kIHBsYXNtYSBsZXZlbHMgaW4gc2ljayBhbmQgaGVhbHRoeSBmaXNoOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1GaXNoIFNoZWxsZmlzaCBJbW11bm9sOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTIzOyAgY2l0YXRpb25fcGFnZXM9MTk3LTIwOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R3JhZGVkIGVmZmVjdHMgb2YgdW5yZWd1bGF0ZWQgc21vb3RoIG11c2NsZSBteW9zaW4gb24gaW50ZXN0aW5hbCBhcmNoaXRlY3R1cmUsIGludGVzdGluYWwgbW90aWxpdHkgYW5kIHZhc2N1bGFyIGZ1bmN0aW9uIGluIHplYnJhZmlzaDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RGlzLiBNb2RlbC4gTWVjaDsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9NTI5LTQwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NZWNoYW5pY2FsIHN0cmV0Y2ggbW9kdWxhdGVzIFRHRi3OsjEgYW5kIM6xMShJKSBjb2xsYWdlbiBleHByZXNzaW9uIGluIGZldGFsIGh1bWFuIGludGVzdGluYWwgc21vb3RoIG11c2NsZSBjZWxsczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QW0uIEouIFBoeXNpb2wuIEdhc3Ryb2ludGVzdC4gTGl2ZXIgUGh5c2lvbDsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT0yNzc7ICBjaXRhdGlvbl9wYWdlcz1HMTA3NC04MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGRpZ2VzdGl2ZSB0cmFjdCBvZiB0aGUgYW1iZXJqYWNrIFNlcmlvbGEgZHVtZXJpbGksIFJpc3NvOiBhIGxpZ2h0IGFuZCBzY2FubmluZyBlbGVjdHJvbiBtaWNyb3Njb3BlIHN0dWR5OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBGaXNoIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTE5OTI7ICBjaXRhdGlvbl92b2x1bWU9NDE7ICBjaXRhdGlvbl9wYWdlcz0yODctMzAzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FbmRvY3l0aWMgcmVjZXB0b3JzIGluIHRoZSByZW5hbCBwcm94aW1hbCB0dWJ1bGU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBoeXNpb2xvZ3kgKEJldGhlc2RhKTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0yNzsgIGNpdGF0aW9uX3BhZ2VzPTIyMy0zNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHVidWxhciByZWFic29ycHRpb24gb2YgYWxidW1pbjogaXQncyBhbGwgYWJvdXQgY3ViaWxpbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQW0uIFNvYy4gTmVwaHJvbDsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTE4MTAtMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBtYW5ub3NlLXJlY2VwdG9yIGlzIHBvc3NpYmx5IGludm9sdmVkIGluIHRoZSBwaGFnb2N5dG9zaXMgb2YgU2FjY2hhcm9teWNlcyBjZXJldmlzaWFlIGJ5IHNlYWJyZWFtIChTcGFydXMgYXVyYXRhIEwuKSBsZXVjb2N5dGVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1GaXNoIFNoZWxsZmlzaCBJbW11bm9sOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9Mzc1LTg4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZWd1bGF0aW9uIG9mIERNQlQxIHZpYSBOT0QyIGFuZCBUTFI0IGluIGludGVzdGluYWwgZXBpdGhlbGlhbCBjZWxscyBtb2R1bGF0ZXMgYmFjdGVyaWFsIHJlY29nbml0aW9uIGFuZCBpbnZhc2lvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gSW1tdW5vbDsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0xNzg7ICBjaXRhdGlvbl9wYWdlcz04MjAzLTExOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdHJ1Y3R1cmFsIG9yZ2FuaXphdGlvbiBhbmQgY29tcGxldGUgc2VxdWVuY2Ugb2YgdGhlIGh1bWFuIM6xLU4tYWNldHlsZ2FsYWN0b3NhbWluaWRhc2UgZ2VuZTogaG9tb2xvZ3kgd2l0aCB0aGUgzrEtZ2FsYWN0b3NpZGFzZSBBIGdlbmUgcHJvdmlkZXMgZXZpZGVuY2UgZm9yIGV2b2x1dGlvbiBmcm9tIGEgY29tbW9uIGFuY2VzdHJhbCBnZW5lOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MTk5MTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTEzMy00MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBxdWFudGl0YXRpdmUgc3R1ZHkgb2Ygc29tZSBkaWdlc3RpdmUgZW56eW1lcyBpbiB0aGUgcmFiYml0ZmlzaCwgU2lnYW51cyBjYW5hbGljdWxhdHVzIGFuZCB0aGUgc2VhIGJhc3MsIExhdGVzIGNhbGNhcmlmZXI7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEZpc2ggQmlvbDsgIGNpdGF0aW9uX3llYXI9MTk5MzsgIGNpdGF0aW9uX3ZvbHVtZT00MjsgIGNpdGF0aW9uX3BhZ2VzPTU5NS02MDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3B1Ymxpc2hlcj1BY2FkZW1pYyBQcmVzcywgTG9uZG9uOyAgY2l0YXRpb25fdGl0bGU9QWR2YW5jZXMgaW4gTWFyaW5lIEJpb2xvZ3k7ICBjaXRhdGlvbl95ZWFyPTE5NzY7ICBjaXRhdGlvbl9wYWdlcz0xMDktMjM5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wYXJhdGl2ZSBzdHVkeSBvZiBkaWdlc3RpdmUgZW56eW1lcyBpbiBmaXNoIHdpdGggZGlmZmVyZW50IG51dHJpdGlvbmFsIGhhYml0cy4gUHJvdGVvbHl0aWMgYW5kIGFteWxhc2UgYWN0aXZpdGllczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QXF1YWN1bHR1cmU7ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9MTcwOyAgY2l0YXRpb25fcGFnZXM9MjY3LTgzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EbyBkaWdlc3RpdmUgZW56eW1lcyBzZXQgYSBwaHlzaW9sb2dpY2FsIGxpbWl0IG9uIGdyb3d0aCByYXRlIGFuZCBmb29kIGNvbnZlcnNpb24gZWZmaWNpZW5jeSBpbiB0aGUgQXRsYW50aWMgY29kIChHYWR1cyBtb3JodWEpPzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RmlzaCBQaHlzaW9sLiBCaW9jaGVtOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9MjkzLTMwMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyYXRpdmUgc3R1ZHkgb2YgcHJvdGVvbHl0aWMgZW56eW1lcyBpbiB0aGUgZGlnZXN0aXZlIHRyYWN0cyBvZiB0aGUgZXVyb3BlYW4gc2VhIGJhc3MgYW5kIGh5YnJpZCBzdHJpcGVkIGJhc3MgcmVhcmVkIGluIGZyZXNod2F0ZXI7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNvbXAuIEJpb2NoZW0uIFBoeXNpb2wuIEEgUGh5c2lvbDsgIGNpdGF0aW9uX3llYXI9MTk5MzsgIGNpdGF0aW9uX3ZvbHVtZT0xMDY7ICBjaXRhdGlvbl9wYWdlcz02MjctMzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByb3Rlb2x5dGljIGRpZ2VzdGl2ZSBlbnp5bWVzIG9mIGNhcm5pdm9yb3VzIChTaWx1cnVzIGdsYW5pcyBMLiksIGhlcmJpdm9yb3VzIChIeXBvcGh0aGFsbWljaHRoeXMgbW9saXRyaXggVmFsLikgYW5kIG9tbml2b3JvdXMgKEN5cHJpbnVzIGNhcnBpbyBMLikgZmlzaGVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1BcXVhY3VsdHVyZTsgIGNpdGF0aW9uX3llYXI9MTk4MzsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTE0NS01NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGVwc2lub2dlbiBBIGFuZCBDIGdlbmVzIGluIHR1cmJvdCAoU2NvcGh0aGFsbXVzIG1heGltdXMpOiBjaGFyYWN0ZXJpemF0aW9uIGFuZCBleHByZXNzaW9uIGluIGVhcmx5IGRldmVsb3BtZW50OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Db21wLiBCaW9jaGVtLiBQaHlzaW9sLiBCLCBCaW9jaGVtLiBNb2wuIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTY1OyAgY2l0YXRpb25fcGFnZXM9NTgtNjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUF0bGFudGljIGNvZCB0cnlwc2luczogZnJvbSBiYXNpYyByZXNlYXJjaCB0byBwcmFjdGljYWwgYXBwbGljYXRpb25zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1NYXIuIEJpb3RlY2hub2w7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTc3LTg4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Jc29sYXRpb24gYW5kIGNoYXJhY3Rlcml6YXRpb24gb2YgYSBjRE5BIGVuY29kaW5nIHJhdCBjYXRpb25pYyB0cnlwc2lub2dlbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvY2hlbWlzdHJ5OyAgY2l0YXRpb25feWVhcj0xOTg3OyAgY2l0YXRpb25fdm9sdW1lPTI2OyAgY2l0YXRpb25fcGFnZXM9MzA4MS02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMjUvNS81NDcvMjYwNzY5NTMvZHN5MDI0LnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBTZXJpb2xhIHF1aW5xdWVyYWRpYXRhICh5ZWxsb3d0YWlsKSBpcyB0aGUgbW9zdCB3aWRlbHkgZmFybWVkIGFuZCBlY29ub21pY2FsbHkgaW1wb3J0YW50IGZpc2ggaW4gYXF1YWN1bHR1cmUgaW4gSmFwYW4uIEluIHRoaXMgc3R1ZHksIHdlIHVzZWQgdGhlIGdlbiIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXMvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9NTA1OTUzMCZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGQh+g24yQTxZ03/IqZdui+olvKh1g==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">New Jersey Institute of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

New Jersey Institute of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f5%2f547%2f5059530" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f5%2f547%2f5059530" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=110919" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/25/5"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/25/5">
                        <div class="volume trailing-comma">Volume 25</div>
                        <div class="issue">Issue 5</div>

            </a>

        </div>
        <div class="ii-pub-date">
October 2018        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="122671109">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#122671109">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="122671111">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#122671111">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="122671115">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#122671115">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="122671130">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#122671130">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="122671160">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#122671160">Availability</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="122671162">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#122671162">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="122671166">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#122671166">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/25/5/535/5055375">&lt; Previous</a>
        </li>        
            <li class="empty arrow">&nbsp;</li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">The yellowtail (<em>Seriola quinqueradiata</em>) genome and transcriptome atlas of the digestive tract
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Motoshige Yasuike</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Research Center for Bioinformatics and Biosciences, National Research Institute of Fisheries Science, Japan Fisheries Research and Education Agency, Yokohama, Kanagawa, Japan</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsy024-cor1" data-open="dsy024-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Tel. +81 045 788 7640. Fax. +81 045 788 7640. Email: <a href="mailto:yasuike@affrc.go.jp" target="_blank">yasuike@affrc.go.jp</a> (M.Y.); Tel. +81 045 788 7691. Fax. +81 045 788 7691. Email: <a href="mailto:jiwara@affrc.go.jp" target="_blank">jiwara@affrc.go.jp</a> (A.F.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Motoshige+Yasuike">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yasuike M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yasuike M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Motoshige Yasuike</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yuki Iwasaki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Research Center for Bioinformatics and Biosciences, National Research Institute of Fisheries Science, Japan Fisheries Research and Education Agency, Yokohama, Kanagawa, Japan</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsy024-Cor2" data-open="dsy024-Cor2" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            Present address: Center for Information Biology, National Institute of Genetics, 1111 Yata, Mishima, Shizuoka 411-8540, Japan.
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yuki+Iwasaki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Iwasaki Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Iwasaki Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yuki Iwasaki</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Issei Nishiki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Research Center for Bioinformatics and Biosciences, National Research Institute of Fisheries Science, Japan Fisheries Research and Education Agency, Yokohama, Kanagawa, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Issei+Nishiki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nishiki I">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nishiki I&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Issei Nishiki</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yoji Nakamura</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Research Center for Bioinformatics and Biosciences, National Research Institute of Fisheries Science, Japan Fisheries Research and Education Agency, Yokohama, Kanagawa, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yoji+Nakamura">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nakamura Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nakamura Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yoji Nakamura</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Aiko Matsuura</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Research Center for Bioinformatics and Biosciences, National Research Institute of Fisheries Science, Japan Fisheries Research and Education Agency, Yokohama, Kanagawa, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Aiko+Matsuura">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Matsuura A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Matsuura A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Aiko Matsuura</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kazunori Yoshida</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Goto Laboratory, Stock Enhancement and Aquaculture Division, Seikai National Fisheries Research Institute Japan Fisheries Research and Education Agency, Tamanoura-cho, Goto, Nagasaki, Japan</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsy024-Cor3" data-open="dsy024-Cor3" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            Present address: Kamiura Laboratory, Research Center for Aquatic Breeding, National Research Institute of Aquaculture, Japan Fisheries Research and Education Agency, Tsuiura, Kamiura, Saiki, Oita 879-2602, Japan.
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kazunori+Yoshida">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yoshida K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yoshida K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Kazunori Yoshida</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tsutomu Noda</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Goto Laboratory, Stock Enhancement and Aquaculture Division, Seikai National Fisheries Research Institute Japan Fisheries Research and Education Agency, Tamanoura-cho, Goto, Nagasaki, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tsutomu+Noda">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Noda T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Noda T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Tsutomu Noda</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tadashi Andoh</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Stock Enhancement and Aquaculture Division, Seikai National Fisheries Research Institute, Japan Fisheries Research and Education Agency, Nagasaki, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tadashi+Andoh">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Andoh T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Andoh T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Tadashi Andoh</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Atushi Fujiwara</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Research Center for Bioinformatics and Biosciences, National Research Institute of Fisheries Science, Japan Fisheries Research and Education Agency, Yokohama, Kanagawa, Japan</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsy024-cor1" data-open="dsy024-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Tel. +81 045 788 7640. Fax. +81 045 788 7640. Email: <a href="mailto:yasuike@affrc.go.jp" target="_blank">yasuike@affrc.go.jp</a> (M.Y.); Tel. +81 045 788 7691. Fax. +81 045 788 7691. Email: <a href="mailto:jiwara@affrc.go.jp" target="_blank">jiwara@affrc.go.jp</a> (A.F.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Atushi+Fujiwara">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Fujiwara A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Fujiwara A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Atushi Fujiwara</a>
                                </span>

                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 25, Issue 5, 1 October 2018, Pages 547–560, <a href='https://doi.org/10.1093/dnares/dsy024'>https://doi.org/10.1093/dnares/dsy024</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">25 July 2018</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">08 November 2017</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">28 June 2018</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="5059530" href="/dnaresearch/article-pdf/25/5/547/26076953/dsy024.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Motoshige Yasuike, Yuki Iwasaki, Issei Nishiki, Yoji Nakamura, Aiko Matsuura, Kazunori Yoshida, Tsutomu Noda, Tadashi Andoh, Atushi Fujiwara;  The yellowtail (<em>Seriola quinqueradiata</em>) genome and transcriptome atlas of the digestive tract, <em>DNA Research</em>, Volume 25, Issue 5, 1 October 2018, Pages 547–560, <a href='https://doi.org/10.1093/dnares/dsy024'>https://doi.org/10.1093/dnares/dsy024</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/5059530?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/5059530?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/5059530?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/5059530?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/5059530?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/5059530?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/5059530?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=The%20yellowtail%20(%26lt%3Bem%26gt%3BSeriola%20quinqueradiata%26lt%3B%2Fem%26gt%3B)%20genome%20and%20transcriptome%20atlas%20of%20the%20digestive%20tract&amp;publicationDate=2018-07-25&amp;volumeNum=25&amp;issueNum=5&amp;author=Yasuike%2C%20Motoshige%3B%20Iwasaki%2C%20Yuki&amp;startPage=547&amp;endPage=560&amp;contentId=10.1093%2Fdnares%2Fdsy024&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f5%2f547%2f5059530" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=122671109 id="122671109" class="abstract-title" >Abstract</h2>
<section class="abstract"><p><em>Seriola quinqueradiata</em> (yellowtail) is the most widely farmed and economically important fish in aquaculture in Japan. In this study, we used the genome of haploid yellowtail fish larvae for <em>de novo</em> assembly of whole-genome sequences, and built a high-quality draft genome for the yellowtail. The total length of the assembled sequences was 627.3 Mb, consisting of 1,394 scaffold sequences (&gt;2 kb) with an N50 length of 1.43 Mb. A total of 27,693 protein-coding genes were predicted for the draft genome, and among these, 25,832 predicted genes (93.3%) were functionally annotated. Given our lack of knowledge of the yellowtail digestive system, and using the annotated draft genome as a reference, we conducted an RNA-Seq analysis of its three digestive organs (stomach, intestine and rectum). The RNA-Seq results highlighted the importance of certain genes in encoding proteolytic enzymes necessary for digestion and absorption in the yellowtail gastrointestinal tract, and this finding will accelerate development of formulated feeds for this species. Since this study offers comprehensive annotation of predicted protein-coding genes, it has potential broad application to our understanding of yellowtail biology and aquaculture.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Seriola quinqueradiata&quot;"><em>Seriola quinqueradiata</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genome sequencing&quot;">genome sequencing</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;protein-coding genes&quot;">protein-coding genes</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;transcriptome analysis&quot;">transcriptome analysis</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;digestive enzymes&quot;">digestive enzymes</a></div>                            <h2 scrollto-destination=122671111 id="122671111" class="section-title" >1. Introduction</h2>
<p>The genus <em>Seriola</em> (family Carangidae) contains nine recognized species of carnivorous, marine fish, globally distributed in tropical, subtropical and temperate ocean regions.<a reveal-id="dsy024-B1" data-open="dsy024-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a><sup>,</sup><em>Seriola</em> species have a high market value and several species are farmed around the world.<a reveal-id="dsy024-B2" data-open="dsy024-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> Japan is the biggest producer of <em>Seriola</em> species, farming yellowtail (<em>S. quinqueradiata</em>), greater amberjack (<em>S. dumerili</em>) and yellowtail kingfish (<em>S. lalandi</em>). In 2016 the production of these three species was 141,000 tons, accounting for over 56% of marine finfish aquaculture in Japan.<a reveal-id="dsy024-B3" data-open="dsy024-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> Most of this production is accounted for by yellowtail, which has the longest history of aquaculture, having started in 1927 and expanded rapidly in the 1960s.<a reveal-id="dsy024-B4" data-open="dsy024-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> Although yellowtail aquaculture technologies have continued to improve, several challenges to the sustainable farming of this species remain to be addressed. The price of fish meal used for yellowtail feed continues to rise,<a reveal-id="dsy024-B5" data-open="dsy024-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> necessitating a better understanding of yellowtail digestive pathways and nutritional requirements in order to develop more cost-effective feeds.</p><p>To improve aquaculture production efficiency, product quality, sustainability and profitability, recent efforts have focussed on the genetics and genomics of economically important aquaculture species, particularly following the development of high-throughput next-generation sequencing (NGS) technologies.<a reveal-id="dsy024-B6 dsy024-B7 dsy024-B8" data-open="dsy024-B6 dsy024-B7 dsy024-B8" class="link link-ref link-reveal xref-bibr"><sup>6–8</sup></a> Genetic maps<a reveal-id="dsy024-B9 dsy024-B10 dsy024-B11" data-open="dsy024-B9 dsy024-B10 dsy024-B11" class="link link-ref link-reveal xref-bibr"><sup>9–11</sup></a> and a whole genome radiation hybrid panel<a reveal-id="dsy024-B12" data-open="dsy024-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> for yellowtail have been constructed, and the detection of quantitative trait loci (QTL) in yellowtail has recently been initiated for resistance to a monogenean parasite, <em>Benedenia seriolae</em>,<a reveal-id="dsy024-B13" data-open="dsy024-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> and sex-determination.<a reveal-id="dsy024-B14" data-open="dsy024-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a><sup>,</sup><a reveal-id="dsy024-B15" data-open="dsy024-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> However, the resources for genes and protein annotations for this species are limited. Only 257 protein sequences have been deposited at NCBI Genbank, taxonomy ID: 8161 (as of 11 July 2018). A comprehensive annotation of the overall protein-coding genes for the yellowtail genome will help identify important genes and pathways involved in the regulation of economically important traits. In addition, a well-annotated genome will provide a powerful tool for better understanding yellowtail biology, and will enhance physiological studies for optimization of breeding technologies, including the development of artificial diets.</p><p>Recently, we demonstrated that use of the genome of a haploid gynogenetic yellowtail larva leads to a significant improvement in <em>de novo</em> assembly because allelic variation does not impede contig extension as it does for assembly of diploid genomes.<a reveal-id="dsy024-B16" data-open="dsy024-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> Here, we present a draft genome sequence for yellowtail based on a haploid genome assembly and its annotation of protein-coding genes. Furthermore, to better understand molecular mechanisms of feed digestion and nutrient absorption in yellowtail, we performed RNA-Seq analysis of the three major organs of the gastrointestinal (GI) tract i.e. stomach, intestine and rectum, using the annotated draft yellowtail genome as a reference. The analysis suggested that certain proteolytic digestive enzymes play key roles in digestion and absorption, and could inform the choice of ingredients for the development of yellowtail-specific feeds for use in aquaculture.</p>                            <h2 scrollto-destination=122671115 id="122671115" class="section-title" >2. Materials and methods</h2>
                            <h3 scrollto-destination=122671116 id="122671116" class="section-title" >2.1. Genome sequencing and assembly</h3>
<p>Preparation of genomic DNA from haploid gynogenetic yellowtail larvae induced by ultraviolet-irradiated sperm and the blood of its dam (diploid genome) has been described in.<a reveal-id="dsy024-B16" data-open="dsy024-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> For contig construction, the genomic shotgun libraries were prepared from the haploid genomic DNA, and sequenced with the Ion PGM (two runs) and Proton (four runs) platforms (Life Technologies, Carlsbad, CA). These haploid reads were assembled using Newbler 3.0 (Roche Diagnostics, Basel, Switzerland). Four Illumina paired-end (PE) libraries (insert size: 240, 360, 480 and 720 bp) and three mate-pair (MP) libraries (insert size: 3–5, 10 and 20 kb) were constructed from the diploid dam genomic DNA and sequenced on a NextSeq 500 sequencer (Illumina, San Diego, CA, USA) using 2 × 150 bp chemistry. To improve sequence accuracy of the haploid contig sequences generated by Ion PGM and Proton sequencers, the Illumina PE reads were mapped against the haploid contigs using Burrows–Wheeler aligner-maximum exact matches (BWA-MEM),<a reveal-id="dsy024-B17" data-open="dsy024-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> and nucleotide mismatches or short indels were overridden by the Illumina read sequences. We also carried out a <em>de novo</em> assembly of the four diploid Illumina PE libraries (all merged 240 bp, and 10 Gb of 360, 480 and 720 bp inserts) with Platanus 1.2.4,<a reveal-id="dsy024-B18" data-open="dsy024-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> and performed a sequence comparison between the haploid and diploid contigs using BLASTN<a reveal-id="dsy024-B19" data-open="dsy024-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> (<em>E</em> value threshold of 1<em>E</em> – 20 and the bit score of the top hit with a more than 2-fold difference over that of the second hit),<a reveal-id="dsy024-B20" data-open="dsy024-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> and the diploid contigs absent in the haploid contigs were added to the haploid contigs. These merged contigs were subjected to scaffolding using Platanus 1.2.4<a reveal-id="dsy024-B18" data-open="dsy024-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> together with all the Illumina PE (unmerged 240, 360, 480 and 720 bp insert) and MP reads (3–5, 10 and 20 kb insert), and gaps in the scaffolds were filled by the Illumina PE reads (unmerged 240, 360, 480 and 720 bp inserts) with the gap-close step in Platanus. The resulting scaffolds were further assembled using RNA-Seq data sets generated in this study (described later), by L_RNA_scaffolder.<a reveal-id="dsy024-B21" data-open="dsy024-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a></p><p>To confirm ploidy level and estimate genome size, Jellyfish software<a reveal-id="dsy024-B22" data-open="dsy024-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> was used to produce a <em>k</em>-mer distribution (19-mer) with the Ion Proton haploid reads and with the Illumina PE diploid reads (all merged 240 bp, and 10 Gb of 360, 480 and 720 bp inserts).</p>                            <h3 scrollto-destination=122671119 id="122671119" class="section-title" >2.2. Linkage map construction and scaffold correction</h3>
<p>To detect and correct scaffold misassemblies, we generated a double-digest restriction-site associated DNA (ddRAD)-based genetic linkage map of the yellowtail using a previously described method<a reveal-id="dsy024-B20" data-open="dsy024-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> with slight modifications. Briefly, genomic DNA samples were extracted from 161 full-sib progeny and their parents’ muscles using the DNeasy Blood &amp; Tissue Kit (Qiagen, Venlo, Netherlands). For preparation of the ddRAD libraries, genomic DNA was digested using two restriction enzymes (<em>Pst</em>I and <em>Msp</em>I) and ligated to Ion Plus Fragment Library Adapters using an Ion Xpress Plus gDNA Fragment Library kit (Life technologies). The ligation products from ∼25 individuals were pooled in equimolar proportions and size-selected within a range of 200–260 bp using BluePippin (Sage Science, Beverly, MA, USA). After size selection, the ddRAD libraries were amplified by eight-cycle PCR, purified using Agencourt AMPure XP beads (Beckman Coulter Inc., Brea, CA, USA), and sequenced on the Ion Proton (Life technologies). Raw sequences were filtered to discard those of low quality and demultiplexed using the process_radtags programme in Stacks.<a reveal-id="dsy024-B23" data-open="dsy024-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> BWA-MEM<a reveal-id="dsy024-B17" data-open="dsy024-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> was used to map the filtered ddRAD-Seq reads to the scaffold sequences, and the output SAM file was filtered using custom Perl scripts based on the following criteria: mapping quality (MQ) of ≥ 30, length fraction of 0.95 and non-specific match handling of ignore. After local realignment, single nucleotide polymorphism (SNP) calling was performed using the Unified Genotyper module in the Genome Analysis Toolkit (GATK)<a reveal-id="dsy024-B24" data-open="dsy024-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> and the variants were filtered as follows: only bi-allelic sites, genotype quality (GQ) of ≥ 30, either reference (REF) or alternative (ALT) allele frequencies of ≥ 95% in homozygous sites and both REF and ALT allele frequencies of ≥ 30% in heterozygous sites. The R/qtl package<a reveal-id="dsy024-B25" data-open="dsy024-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> was employed for linkage analysis and linkage map construction with the same parameters as previously described in.<a reveal-id="dsy024-B20" data-open="dsy024-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> Linkage group (LG) numbering and orientation followed that of Fuji et al.<a reveal-id="dsy024-B10" data-open="dsy024-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> To detect the possibility of misassembled chimeric scaffolds, the linkage marker sequences were mapped to the scaffold sequences with BLASTN using the same criteria as previously described in<a reveal-id="dsy024-B20" data-open="dsy024-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a>; the scaffolds attributed to two or more LGs were split into consistent scaffolds based on the locations of markers. Scaffolds &gt;2 kb in length were used for subsequent analysis.</p>                            <h3 scrollto-destination=122671121 id="122671121" class="section-title" >2.3. Evaluation of completeness of the final assembly</h3>
<p>A completeness assessment was performed with gVolante<a reveal-id="dsy024-B26" data-open="dsy024-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> using the pipeline CEGMA (Core Eukaryotic Genes Mapping Approach)<a reveal-id="dsy024-B27" data-open="dsy024-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> and the reference dataset Core Vertebrate Genes (CVGs).<a reveal-id="dsy024-B28" data-open="dsy024-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> We also assessed the assembly quality by Benchmarking Universal Single-Copy Orthologues (BUSCO)<a reveal-id="dsy024-B29" data-open="dsy024-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> version 3.0.2 with an Actinopterygii-specific set of 4,584 single copy orthologs.</p>                            <h3 scrollto-destination=122671123 id="122671123" class="section-title" >2.4. cDNA sequencing</h3>
<p>For gene predictions, normalized, full-length enriched cDNA libraries were constructed as follows: Total RNAs were isolated from eighteen yellowtail tissues (gills, skin, fins, red muscle, white muscle, heart, kidney, spleen, stomach, intestine, pyloric caeca, liver, gallbladder, retina, cerebellum, optic lobe, olfactory lobe and ovary) using RNeasy Plus Universal Mini Kit (Qiagen). The isolated total RNA samples were treated with 2 U of TURBO DNase from the TURBO DNase-free Kit (Life Technologies) at 37°C for 30 min as recommended by the manufacturer’s instructions. RNA integrity was assessed with an Agilent Bioanalyzer 2100 with the RNA 6000 Nano Kit (Agilent Technologies, Santa Clara, CA, USA). Poly(A) + RNA (mRNA) was purified using the FastTrack MAG Micro mRNA Isolation Kit (Invitrogen, Carlsbad, CA, USA). Equal amounts of mRNA were pooled from each tissue and 500 ng of the pooled mRNA was used to construct a full-length enriched cDNA library using the SMART PCR cDNA Synthesis Kit (Clontech Laboratories, Inc. Palo Alto, CA, USA) with modifications as described by Meyer et al.<a reveal-id="dsy024-B30" data-open="dsy024-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> The primer used for first-strand cDNA synthesis was a modified 3’ SMART CDS Primer II A (AAGCAGTGGTATCAACGCAGAGTCGCAGTCGGTACTTTTTTCTTTTTTV) including a recognition site for the restriction enzyme <em>Bpm</em>I,<a reveal-id="dsy024-B30" data-open="dsy024-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> and the cDNA was amplified using the 5’ PCR Primer II A (AAGCAGTGGTATCAACGCAGAGT) with the KAPA library amplification kit (KAPA Biosystems, Boston, MA, USA). Amplified cDNA was purified using the QIAquick PCR Purification Kit (Qiagen) and normalized using a Trimmer-2 cDNA normalization kit (Evrogen, Moscow, Russia). The normalized cDNA was purified with the QIAquick PCR purification kit, and the purified cDNA was digested with <em>Bpm</em>I for 1 h at 37°C to remove poly (A) tails. Finally, the digested cDNA was purified using a QIAquick PCR purification kit. In addition to the 18-tissue normalized cDNA library, additional full-length cDNA libraries for pituitary, testis, ovary haploid cells and digestive organs from larval fish (8–10 days post-hatch) were prepared individually with the SMART PCR cDNA Synthesis Kit employing the same procedure as above. Among these, the cDNA libraries for the digestive organs from the larval fish were normalized using the methods above. These cDNA libraries were sequenced with the 454 GS FLX+ (Roche Diagnostics), Ion PGM or Proton platforms (Life Technologies). Detailed information on the cDNA libraries with corresponding sequencers and sequence statistics are summarized in <span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy024/1/dsy024_supplementary_table_s2.xlsx?Expires=2147483647&Signature=fRv2fXdIjcGOV0lUu4UfWW84KSrtuB-LMM349frfrVPuviBIPl2aX~5o2flP8nZd7osb1C0sdf~kMR8AE7ixasCoQAxcZzVYdq-14rHV2Rguwg9SUeUrokHoBL96NDNrLpC8SgmV6E5iD-WHxLJAKZHZPbQXEP5s0b96i1rPwl5D8IhOvY2vaZGWNsECnBEOEtdUjdYU9t6UMCfY1lJrwhlLk3NtY6PRm1E7Z51Wa2T1oWV4K~kvrKhVnbcra09KOLMKDuvPlvXbRbZOmeU64bF~47e27B1Xt3CFXcFOo9t2e~bHmJYRJYjO-kbYCAxUFL2K1VcaBZulFXeu6mtQjA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>.</p>                            <h3 scrollto-destination=122671125 id="122671125" class="section-title" >2.5. Gene prediction and annotation</h3>
<p>Protein-coding genes in the yellowtail genome were predicted using AUGUSTUS (version 3.1).<a reveal-id="dsy024-B31" data-open="dsy024-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> First, repeat regions in the yellowtail scaffold sequences (&gt;2 kb) were masked by RepeatMasker<a reveal-id="dsy024-B32" data-open="dsy024-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> according to the yellowtail repeat database constructed by RepeatModeler.<a reveal-id="dsy024-B33" data-open="dsy024-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> Next, the RNA-Seq reads of yellowtail sequenced in this study were mapped to the scaffolds using TopHat<a reveal-id="dsy024-B34" data-open="dsy024-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> and assembled with Cufflinks.<a reveal-id="dsy024-B35" data-open="dsy024-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> Then, the scaffold sequences were scanned with the Zebrafish model in AUGUSTUS, using the map information of RNA-Seq as hints. The predicted gene sequences (24,728 genes) were validated using BUSCO,<a reveal-id="dsy024-B29" data-open="dsy024-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> and loci attributed to ‘complete’ (792 loci) were used to construct a yellowtail gene training model. Moreover, protein sequences of 11 fish species: spotted gar (<em>Lepisosteus oculatus</em>), Mexican tetra (<em>Astyanax mexicanus</em>), zebrafish (<em>Danio rerio</em>), Atlantic cod (<em>Gadus morhua</em>), Nile tilapia (<em>Oreochromis niloticus</em>), platyfish (<em>Xiphophorus maculatus</em>), Amazon molly (<em>Poecilia formosa</em>), medaka (<em>Oryzias latipes</em>), stickleback (<em>Gasterosteus aculeatus</em>), green puffer (<em>Tetraodon nigroviridis</em>) and fugu (<em>Takifugu rubripes</em>), were downloaded from the Ensembl database (Release 84),<a reveal-id="dsy024-B36" data-open="dsy024-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> mapped to the yellowtail scaffolds by TBLASTN<a reveal-id="dsy024-B37" data-open="dsy024-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> with <em>E</em>-value &lt; 10<sup>−5</sup>, and the map information was used as hints in AUGUSTUS. Finally, the predicted gene sequences were analysed with InterProScan<a reveal-id="dsy024-B38" data-open="dsy024-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> and those matched by any domain or supported by any AUGUSTUS hint were collected as valid protein-coding genes.</p><p>The predicted amino acid sequences were compared using BLASTP<a reveal-id="dsy024-B19" data-open="dsy024-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> (<em>E</em> value threshold of 1<em>E</em> − 5) against those of well-annotated model fishes, medaka and zebrafish, in the Ensembl database (Release 84),<a reveal-id="dsy024-B36" data-open="dsy024-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> and against the NCBI non-redundant (nr) protein database. Functional annotation was performed using Blast2GO<a reveal-id="dsy024-B39" data-open="dsy024-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> and the Kyoto Encyclopedia of Genes and Genomes (KEGG) Automatic Annotation Server.<a reveal-id="dsy024-B40" data-open="dsy024-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> Orthologous gene cluster analysis of yellowtail, Pacific bluefin tuna (<em>Thunnus orientalis</em>),<a reveal-id="dsy024-B41" data-open="dsy024-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a> croaker (<em>Larimichthys crocea</em>),<a reveal-id="dsy024-B42" data-open="dsy024-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a> Nile tilapia, medaka and zebrafish was performed using the OrthoVenn<a reveal-id="dsy024-B43" data-open="dsy024-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a> web server with default parameters (<em>E</em>-value 1e–5, inflation value 1.5). OrthoVenn was also used to assign significantly enriched GO terms of species-specific gene clusters based on a hypergeometric test (<em>P</em>-value &lt; 0.05). The same analysis also applied to comparisons within the genus <em>Seriola</em> including yellowtail, <em>S. dumerili</em> (Genbank: <a class="link link-gen" href="http://www.ncbi.nlm.nih.gov/nuccore/BDQW01000000" target="_blank">BDQW01000000</a>) and <em>S. dorsalis</em> (Genbank: <a class="link link-gen" href="http://www.ncbi.nlm.nih.gov/nuccore/PEQF01000000" target="_blank">PEQF01000000</a>).</p>                            <h3 scrollto-destination=122671128 id="122671128" class="section-title" >2.6. RNA-Seq analysis of yellowtail GI tract</h3>
<p>The three major organs of the GI tract: stomach, intestine and rectum, were dissected from three healthy adult yellowtails (<em>n</em> = 3) obtained from brood stock maintained at the Goto laboratory of the Seikai National Fisheries Research Institute, Nagasaki, Japan and immediately immersed in RNAlater stabilized solution (Thermo Fisher Scientific, Waltham, MA, USA). Total RNA was extracted using the Maxwell RSC simply RNA Kit (Promega, Madison, WI, USA) and mRNA was purified from 3 µg total RNA using the Gene Read Pure mRNA kit (Qiagen). Sequencing libraries were constructed from 5 ng of each mRNA sample using the Ion Total RNA-Seq Kit v2 and sequenced with Ion Proton (Life technologies). RNA-Seq data analysis was carried out using CLC Genomics Workbench 9.5.2 software (CLC Bio, Aarhus, Denmark) as follows: The low-quality sequences were filtered with default parameters (quality limit = 0.05 and ambiguous limit = 2), and the filtered sequence reads were mapped to the yellowtail reference genome constructed in this study with default parameters (mismatch cost = 2, insertion cost = 3 and deletion cost = 3, length fraction = 0.8 and similarity fraction = 0.8). Subsequently, transcripts per millions (TPMs)<a reveal-id="dsy024-B44" data-open="dsy024-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a> values for each gene were calculated. The differentially expressed genes (DEGs) were considered significant for the following criteria: a 2-fold or greater change in expression with a false discovery rate <em>P</em>-value &lt; 0.05. Visualization of the DEGs was performed by constructing a hierarchical heatmap with Euclidean distance and complete linkage in the CLC Genomics Workbench. To compare the over-represented functional categories between the highly expressed genes (500 highest TPM values) in the three tissues, a GO enrichment analysis was conducted using the functional annotation tool, PANTHER version 12.0 (released 7 Ocotber 2017)<a reveal-id="dsy024-B45" data-open="dsy024-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a> using the zebrafish gene list. The Bonferroni correction for multiple testing was applied with the corrected <em>P</em>-value &lt; 0.05 considered significantly over-represented in the PANTHER analysis.</p>                            <h2 scrollto-destination=122671130 id="122671130" class="section-title" >3. Results and discussion</h2>
                            <h3 scrollto-destination=122671131 id="122671131" class="section-title" >3.1. Genome sequencing and assembly</h3>
<p>Recent advances in the speed, accuracy and cost effectiveness of NGS technologies have provided the opportunity for whole genome sequencing of even non-model organisms with no available reference genomes. However, <em>de novo</em> assembly of diploid species, including teleosts, remains challenging because of allelic variation. Recently, several teleost studies have used DNA samples with theoretically no allelic variations as biological starting material for genome assembly. Zhang et al.<a reveal-id="dsy024-B46" data-open="dsy024-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a> reported a marked improvement in genome assembly after employing doubled-haploid fugu individuals via mitotic gynogenesis. Although doubled haploid fishes are useful in genomic analysis, limited doubled haploid fish lines are available, and their establishment is labour-intensive, time-consuming and expensive. On the other hand, the use of haploid larvae is a convenient means of obtaining a non-allelic variation genome in fish if an artificial insemination technique is established, and we have already demonstrated the effectiveness of genomic DNA from a haploid gynogenetic yellowtail larva in genome assembly.<a reveal-id="dsy024-B16" data-open="dsy024-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> However, there are two concerns related to the use of haploid fish larvae for the construction of reference genome sequences. To overcome these concerns, we established an assembling strategy (<a reveal-id="dsy024-F1" data-open="dsy024-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>) as follows. First, defective regions may exist in the haploid genome sequence due to allelic variations. Taking this probability into account, we compared the haploid contigs with those generated from its dam (diploid), and then merged the diploid sequences absent in the haploid contigs (<a reveal-id="dsy024-F1" data-open="dsy024-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>). Second, because haploid larvae die during early larval stages, the amount of DNA available from a haploid fish larva is limited, and not sufficient for the preparation of PE and MP libraries for scaffolding. To make up for this limitation, the contigs were constructed from limited amounts of DNA from a haploid fish, and scaffolding was performed using sufficiently available DNA from its dam (<a reveal-id="dsy024-F1" data-open="dsy024-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>). </p>                        <a id="122671133" scrollto-destination="122671133"></a>
<div id="dsy024-F1" data-id="dsy024-F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy024/1/m_dsy024f1.png?Expires=2147483647&amp;Signature=nihAva5nNIofRR6GN4T3NacdEU56kZYFT13MxmQyrvonOqYgmvrdaaO5eMp6jCBCVmARVLe8fQpxUV5DNVqnHF9XTPi-jgaUEDbEN6wpEt6PeDUej7ruUxviL3-r8c1QXj6Rp1AHhSSY9O~opx0ggvWG6HeZ2R0Nj0C34vR5~XJ4YYZNvKx5mZTfIu4cpNjK01q~FNIGnIGcjF5~z3S5lIsxBxHh3xgxh9p1kvQMFO9EkjzkkE8o4QWpUous-7MEQPq6n~KnT~LAoDkyA6lL7nhN5DDrBC458E6OdYMZqmAify7zV6KA0U1Mv8fIN798vKoo7ko~3BWV~urcE3wqQQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Strategy for de novo assembly of the yellowtail (S. quinqueradiata) genome using the genome of haploid larvae. More detailed information on the sequencing libraries and the summary statistics of reads used in this study can be found in Supplementary Table S1, and contig and scaffold statistics are shown in Supplementary Tables S3 and S4, respectively." path-from-xml="dsy024f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/122671133/dsy024f1.tif" path-from-xml="dsy024f1.tif" target="_blank">View large</a><a section="122671133" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy024/1/dsy024f1.png?Expires=2147483647&amp;Signature=vioYBu~uYlGljrCxdK0vQ~J5uQojQid9PPLxNBJoUP8-rmFHxVyI3Y7IyN0lGTIVruPRoNCTTGqMwTX6RqgFUzu0BrYbfSBhPvQPEf0M~aG~xgdKX0cFc1PVHIpPbUaHGKHjO34hEIHz1JoyMXB4~sRMF-SaxuruLjNimjE~HvFeDtdcmChUZIvuxuhCLA1tN40AzBt05MdHDq6VM-F1Z6XPpd804VHsJUuk8QVCrG2QFtPtCIL1GUld30Vu4t8KAlkQpmSIS2NxZGTjfEC4OYuOByzk6~IiOTuVVZGO5slbyPw45Y7tQz2kMlT1UrC~MkP9wHMrKIT2qFyaNsjopA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=122671133&amp;ar=5059530&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsy024f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Strategy for <em>de novo</em> assembly of the yellowtail (<em>S. quinqueradiata</em>) genome using the genome of haploid larvae. More detailed information on the sequencing libraries and the summary statistics of reads used in this study can be found in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy024/1/dsy024_supplementary_table_s1.xlsx?Expires=2147483647&amp;Signature=ZXQ~~2kF~81JP0zehGivG8oAMg1khl~qDbt-C-bLvEVxLxrxOJSaPIwQqPZLMOVJt4No7RRIWEOprcg2SPhdW9Ff-k7d2aWDPuSUYwC1fLZmQnARZrGpyE6kRQAC6-ZEkAOrrK6zhvWGDyS~8mYTq03Cxd26KrDq1gDa~e4VdteZ7ZVtd~f7mXw1L3dBqpeMJxqLgN9xLo~-NBpzjIhLA9T7L-4GdMe1te91hqruD41lYLuJexPncWHzVBtvW-YtB~DsmVefGsf3YJbCP0Lyw4V2bz4UyY1MJinO6jYnkqmZI8wCem~WxVS9aBTgDzrXnYknd05d~tn5Stsfj5PPMA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>, and contig and scaffold statistics are shown in <span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy024/1/dsy024_supplementary_table_s3.xlsx?Expires=2147483647&amp;Signature=45PH9kh-uXRoTqKjAqxde9ye2hCyPt94vG5ifNzatLVuCUGjYqdfM9hEGa3pqdSa3nG7He10Ct-13XuLt4HAM9EGun~AXH6I00ygGZjOmxO7jh-9NWBMTN6JpUwG9ANRFploi12OSm35dxOMiDYljkD4KQ5nKHaJlYUVN1zWoSGVaWug2kzwn14g9EaRSNvOd506AzjBwSYzhYzBfHkyy7FFyS2X1JIcmx0Wzps7NQNqPHBY2MpmD5ZZahNJi37ss2bKr8siB~xyzXP7ZUcz00minundXllJoHczXyRJSGS7Cmg~btvXSIuS-gplKa1RU1jF4tNnOuh4TGzqf7vP~g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S3</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup4" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy024/1/dsy024_supplementary_table_s4.xlsx?Expires=2147483647&amp;Signature=S62dCvdyf6CiYkS4Obtb-LQjhorJVtDjwzDv~qWl9Dc3PFwrcUrxFjltK4-5OxVG2naeBupJVaVrtnzZbqULbnZkCYptSehiwx9~5eIL6~CxtqE7cAWpJiYKEVOhxLxB8v9f9LVHjqe9S1ZOc11zpub1hcuCvbTq9pfYBYz1x4RoeMHC~mbKG2apyZP1s07Ml15Aq8AspZmvGWQ3ow0ssIrj4ZCwoTRHFfITN6AhcpHdL8qFbvRoOpOIVlo3vaCYHkHK7h178o4LATWE8InMRdWV5tcHpRNxQzOXyxDvi6-3LLC9lthMZ6ImmiFtHYqNKcy3ftPQYVfHKK2j8EI2eA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">S4</a></span>, respectively.</p></div></div><div content-id="dsy024-F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy024/1/m_dsy024f1.png?Expires=2147483647&amp;Signature=nihAva5nNIofRR6GN4T3NacdEU56kZYFT13MxmQyrvonOqYgmvrdaaO5eMp6jCBCVmARVLe8fQpxUV5DNVqnHF9XTPi-jgaUEDbEN6wpEt6PeDUej7ruUxviL3-r8c1QXj6Rp1AHhSSY9O~opx0ggvWG6HeZ2R0Nj0C34vR5~XJ4YYZNvKx5mZTfIu4cpNjK01q~FNIGnIGcjF5~z3S5lIsxBxHh3xgxh9p1kvQMFO9EkjzkkE8o4QWpUous-7MEQPq6n~KnT~LAoDkyA6lL7nhN5DDrBC458E6OdYMZqmAify7zV6KA0U1Mv8fIN798vKoo7ko~3BWV~urcE3wqQQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Strategy for de novo assembly of the yellowtail (S. quinqueradiata) genome using the genome of haploid larvae. More detailed information on the sequencing libraries and the summary statistics of reads used in this study can be found in Supplementary Table S1, and contig and scaffold statistics are shown in Supplementary Tables S3 and S4, respectively." path-from-xml="dsy024f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/122671133/dsy024f1.tif" path-from-xml="dsy024f1.tif" target="_blank">View large</a><a section="122671133" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy024/1/dsy024f1.png?Expires=2147483647&amp;Signature=vioYBu~uYlGljrCxdK0vQ~J5uQojQid9PPLxNBJoUP8-rmFHxVyI3Y7IyN0lGTIVruPRoNCTTGqMwTX6RqgFUzu0BrYbfSBhPvQPEf0M~aG~xgdKX0cFc1PVHIpPbUaHGKHjO34hEIHz1JoyMXB4~sRMF-SaxuruLjNimjE~HvFeDtdcmChUZIvuxuhCLA1tN40AzBt05MdHDq6VM-F1Z6XPpd804VHsJUuk8QVCrG2QFtPtCIL1GUld30Vu4t8KAlkQpmSIS2NxZGTjfEC4OYuOByzk6~IiOTuVVZGO5slbyPw45Y7tQz2kMlT1UrC~MkP9wHMrKIT2qFyaNsjopA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=122671133&amp;ar=5059530&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsy024f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Strategy for <em>de novo</em> assembly of the yellowtail (<em>S. quinqueradiata</em>) genome using the genome of haploid larvae. More detailed information on the sequencing libraries and the summary statistics of reads used in this study can be found in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy024/1/dsy024_supplementary_table_s1.xlsx?Expires=2147483647&amp;Signature=ZXQ~~2kF~81JP0zehGivG8oAMg1khl~qDbt-C-bLvEVxLxrxOJSaPIwQqPZLMOVJt4No7RRIWEOprcg2SPhdW9Ff-k7d2aWDPuSUYwC1fLZmQnARZrGpyE6kRQAC6-ZEkAOrrK6zhvWGDyS~8mYTq03Cxd26KrDq1gDa~e4VdteZ7ZVtd~f7mXw1L3dBqpeMJxqLgN9xLo~-NBpzjIhLA9T7L-4GdMe1te91hqruD41lYLuJexPncWHzVBtvW-YtB~DsmVefGsf3YJbCP0Lyw4V2bz4UyY1MJinO6jYnkqmZI8wCem~WxVS9aBTgDzrXnYknd05d~tn5Stsfj5PPMA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>, and contig and scaffold statistics are shown in <span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy024/1/dsy024_supplementary_table_s3.xlsx?Expires=2147483647&amp;Signature=45PH9kh-uXRoTqKjAqxde9ye2hCyPt94vG5ifNzatLVuCUGjYqdfM9hEGa3pqdSa3nG7He10Ct-13XuLt4HAM9EGun~AXH6I00ygGZjOmxO7jh-9NWBMTN6JpUwG9ANRFploi12OSm35dxOMiDYljkD4KQ5nKHaJlYUVN1zWoSGVaWug2kzwn14g9EaRSNvOd506AzjBwSYzhYzBfHkyy7FFyS2X1JIcmx0Wzps7NQNqPHBY2MpmD5ZZahNJi37ss2bKr8siB~xyzXP7ZUcz00minundXllJoHczXyRJSGS7Cmg~btvXSIuS-gplKa1RU1jF4tNnOuh4TGzqf7vP~g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S3</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup4" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy024/1/dsy024_supplementary_table_s4.xlsx?Expires=2147483647&amp;Signature=S62dCvdyf6CiYkS4Obtb-LQjhorJVtDjwzDv~qWl9Dc3PFwrcUrxFjltK4-5OxVG2naeBupJVaVrtnzZbqULbnZkCYptSehiwx9~5eIL6~CxtqE7cAWpJiYKEVOhxLxB8v9f9LVHjqe9S1ZOc11zpub1hcuCvbTq9pfYBYz1x4RoeMHC~mbKG2apyZP1s07Ml15Aq8AspZmvGWQ3ow0ssIrj4ZCwoTRHFfITN6AhcpHdL8qFbvRoOpOIVlo3vaCYHkHK7h178o4LATWE8InMRdWV5tcHpRNxQzOXyxDvi6-3LLC9lthMZ6ImmiFtHYqNKcy3ftPQYVfHKK2j8EI2eA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">S4</a></span>, respectively.</p></div></div><p>In this study, a total of 11.2 million reads (3.7 Gb) and 259.4 million reads (51.9 Gb) from the haploid genome were generated from Ion PGM and Ion Proton, respectively, and a total of 858 million reads (129.1 Gb) were obtained from a series of Illumina PE (240 bp insert: 27.8 Gb, 360 bp insert: 31.5 Gb, 480 bp insert: 25.3 Gb and 720 bp insert: 9.5 Gb) and MP (3–5 kb insert: 10.8 Gb, 10 kb insert: 12.2 Gb and 20 kb insert: 12.0 Gb) libraries of the diploid genome (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy024/1/dsy024_supplementary_table_s1.xlsx?Expires=2147483647&amp;Signature=ZXQ~~2kF~81JP0zehGivG8oAMg1khl~qDbt-C-bLvEVxLxrxOJSaPIwQqPZLMOVJt4No7RRIWEOprcg2SPhdW9Ff-k7d2aWDPuSUYwC1fLZmQnARZrGpyE6kRQAC6-ZEkAOrrK6zhvWGDyS~8mYTq03Cxd26KrDq1gDa~e4VdteZ7ZVtd~f7mXw1L3dBqpeMJxqLgN9xLo~-NBpzjIhLA9T7L-4GdMe1te91hqruD41lYLuJexPncWHzVBtvW-YtB~DsmVefGsf3YJbCP0Lyw4V2bz4UyY1MJinO6jYnkqmZI8wCem~WxVS9aBTgDzrXnYknd05d~tn5Stsfj5PPMA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). The genome size of the yellowtail has been calculated to be ∼685 Mb based on the <em>c</em>-value,<a reveal-id="dsy024-B10" data-open="dsy024-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> while the genome size based on <em>k</em>-mer frequency was estimated to be 603.8 and 602.7 Mb from the haploid and diploid reads datasets, respectively. Thus, the generated sequence data (184.7 Gb) has 270- to 306-fold coverage of its genome size. <a reveal-id="dsy024-F1" data-open="dsy024-F1" class="link link-reveal link-table xref-fig">Figure 1</a> provides a flow diagram of the genome assembly procedure used for this study; information on the sequencing libraries and summary statistics of reads appears in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy024/1/dsy024_supplementary_table_s1.xlsx?Expires=2147483647&amp;Signature=ZXQ~~2kF~81JP0zehGivG8oAMg1khl~qDbt-C-bLvEVxLxrxOJSaPIwQqPZLMOVJt4No7RRIWEOprcg2SPhdW9Ff-k7d2aWDPuSUYwC1fLZmQnARZrGpyE6kRQAC6-ZEkAOrrK6zhvWGDyS~8mYTq03Cxd26KrDq1gDa~e4VdteZ7ZVtd~f7mXw1L3dBqpeMJxqLgN9xLo~-NBpzjIhLA9T7L-4GdMe1te91hqruD41lYLuJexPncWHzVBtvW-YtB~DsmVefGsf3YJbCP0Lyw4V2bz4UyY1MJinO6jYnkqmZI8wCem~WxVS9aBTgDzrXnYknd05d~tn5Stsfj5PPMA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>. Because our previous study demonstrated that the overlap-layout-consensus (OLC)-based <em>de novo</em> assembly (i.e. Newbler) was the most efficient method for haploid reads generated from the Ion Torrent sequencing platform,<a reveal-id="dsy024-B16" data-open="dsy024-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> the haploid reads (55.6 Gb) were assembled by Newbler and generated 90,827 contigs (&gt;500 bp) with a total length with 611.9 Mb (<a reveal-id="dsy024-F1" data-open="dsy024-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>, Contigs A and <span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy024/1/dsy024_supplementary_table_s3.xlsx?Expires=2147483647&amp;Signature=45PH9kh-uXRoTqKjAqxde9ye2hCyPt94vG5ifNzatLVuCUGjYqdfM9hEGa3pqdSa3nG7He10Ct-13XuLt4HAM9EGun~AXH6I00ygGZjOmxO7jh-9NWBMTN6JpUwG9ANRFploi12OSm35dxOMiDYljkD4KQ5nKHaJlYUVN1zWoSGVaWug2kzwn14g9EaRSNvOd506AzjBwSYzhYzBfHkyy7FFyS2X1JIcmx0Wzps7NQNqPHBY2MpmD5ZZahNJi37ss2bKr8siB~xyzXP7ZUcz00minundXllJoHczXyRJSGS7Cmg~btvXSIuS-gplKa1RU1jF4tNnOuh4TGzqf7vP~g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>). On the other hand, a 52.1 Gb of diploid reads from the four Illumina PE libraries (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy024/1/dsy024_supplementary_table_s1.xlsx?Expires=2147483647&amp;Signature=ZXQ~~2kF~81JP0zehGivG8oAMg1khl~qDbt-C-bLvEVxLxrxOJSaPIwQqPZLMOVJt4No7RRIWEOprcg2SPhdW9Ff-k7d2aWDPuSUYwC1fLZmQnARZrGpyE6kRQAC6-ZEkAOrrK6zhvWGDyS~8mYTq03Cxd26KrDq1gDa~e4VdteZ7ZVtd~f7mXw1L3dBqpeMJxqLgN9xLo~-NBpzjIhLA9T7L-4GdMe1te91hqruD41lYLuJexPncWHzVBtvW-YtB~DsmVefGsf3YJbCP0Lyw4V2bz4UyY1MJinO6jYnkqmZI8wCem~WxVS9aBTgDzrXnYknd05d~tn5Stsfj5PPMA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>) were assembled with Platanus, a <em>de novo</em> assembler specifically designed to reconstruct highly heterozygous genomes,<a reveal-id="dsy024-B18" data-open="dsy024-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> and the resulting assembly consists of 209,882 contigs (&gt;500 bp) with a total length with 596.7 Mb (<a reveal-id="dsy024-F1" data-open="dsy024-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>, Contigs B and <span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy024/1/dsy024_supplementary_table_s3.xlsx?Expires=2147483647&amp;Signature=45PH9kh-uXRoTqKjAqxde9ye2hCyPt94vG5ifNzatLVuCUGjYqdfM9hEGa3pqdSa3nG7He10Ct-13XuLt4HAM9EGun~AXH6I00ygGZjOmxO7jh-9NWBMTN6JpUwG9ANRFploi12OSm35dxOMiDYljkD4KQ5nKHaJlYUVN1zWoSGVaWug2kzwn14g9EaRSNvOd506AzjBwSYzhYzBfHkyy7FFyS2X1JIcmx0Wzps7NQNqPHBY2MpmD5ZZahNJi37ss2bKr8siB~xyzXP7ZUcz00minundXllJoHczXyRJSGS7Cmg~btvXSIuS-gplKa1RU1jF4tNnOuh4TGzqf7vP~g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>). Although different types of sequencers and assemblers were used for the haploid and diploid reads, the assembly of the haploid genome significantly reduced the total number of contigs with longer average and N50 contig lengths when compared with the diploid genome assembly (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy024/1/dsy024_supplementary_table_s3.xlsx?Expires=2147483647&amp;Signature=45PH9kh-uXRoTqKjAqxde9ye2hCyPt94vG5ifNzatLVuCUGjYqdfM9hEGa3pqdSa3nG7He10Ct-13XuLt4HAM9EGun~AXH6I00ygGZjOmxO7jh-9NWBMTN6JpUwG9ANRFploi12OSm35dxOMiDYljkD4KQ5nKHaJlYUVN1zWoSGVaWug2kzwn14g9EaRSNvOd506AzjBwSYzhYzBfHkyy7FFyS2X1JIcmx0Wzps7NQNqPHBY2MpmD5ZZahNJi37ss2bKr8siB~xyzXP7ZUcz00minundXllJoHczXyRJSGS7Cmg~btvXSIuS-gplKa1RU1jF4tNnOuh4TGzqf7vP~g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>). Furthermore, the <em>k</em>-mer distribution of the diploid reads showed an additional small peak which corresponds to heterozygous regions, i.e. a ‘hetero-peak’<a reveal-id="dsy024-B18" data-open="dsy024-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> (<span class="content-section supplementary-material"><a path-from-xml="sup10" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy024/1/dsy024_supplementary_fig_s1.pdf?Expires=2147483647&amp;Signature=P98bjJmlGdMLt4aSAVuqwH1z4EoLiwhGB-qK2W~-DT1it4BvpzxEdB9xdGtSuovXUG-U0x6pVosg-MAmHMFNXogIVXdMEs875iCE8ZwdJNflwm2oj8pG~SPUm3HNH7W4OvWmPq-bbVWh0UkEb4INgTSWL3dK4q0bC4oI9x8hPAfWCwQY9vq6z3VaFECAnasPXxTiqb77TPaGtzSU14tJlS1ziZZ8voWHW4jLykUddT8avxqgMsSbp-WVjyNmzeZGXmJs2RcU3857u7HFXOwgmCjHLxsjfvnT6JgQyMNj3uE2zGfNYnjO20pjUe-J-4thOjQW--XbIw8xskBCFz8e8g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>). These heterozygous regions may disrupt the extension of contigs during the assembly process.<a reveal-id="dsy024-B16" data-open="dsy024-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> Thus, these results support the argument for using the haploid genome for <em>de novo</em> assembly.</p><p>After sequence error correction of the haploid contigs using 94.1 Gb of Illumina PE reads (<a reveal-id="dsy024-F1" data-open="dsy024-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>, Contigs A’), the diploid contigs (<a reveal-id="dsy024-F1" data-open="dsy024-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>, Contigs B) absent in the haploid contigs (<a reveal-id="dsy024-F1" data-open="dsy024-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>, Contigs A’) were added to the haploid contigs. This process could add 26,916 contigs (4.5 Mb) to the haploid contigs, and generated 162,987 contigs (<a reveal-id="dsy024-F1" data-open="dsy024-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>, Contigs C and <span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy024/1/dsy024_supplementary_table_s3.xlsx?Expires=2147483647&amp;Signature=45PH9kh-uXRoTqKjAqxde9ye2hCyPt94vG5ifNzatLVuCUGjYqdfM9hEGa3pqdSa3nG7He10Ct-13XuLt4HAM9EGun~AXH6I00ygGZjOmxO7jh-9NWBMTN6JpUwG9ANRFploi12OSm35dxOMiDYljkD4KQ5nKHaJlYUVN1zWoSGVaWug2kzwn14g9EaRSNvOd506AzjBwSYzhYzBfHkyy7FFyS2X1JIcmx0Wzps7NQNqPHBY2MpmD5ZZahNJi37ss2bKr8siB~xyzXP7ZUcz00minundXllJoHczXyRJSGS7Cmg~btvXSIuS-gplKa1RU1jF4tNnOuh4TGzqf7vP~g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>). These merged contigs (<a reveal-id="dsy024-F1" data-open="dsy024-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>, Contigs C) were subjected to scaffolding with 71.5 Gb of Illumina PE and 35.0 Gb of Illumina MP reads (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy024/1/dsy024_supplementary_table_s1.xlsx?Expires=2147483647&amp;Signature=ZXQ~~2kF~81JP0zehGivG8oAMg1khl~qDbt-C-bLvEVxLxrxOJSaPIwQqPZLMOVJt4No7RRIWEOprcg2SPhdW9Ff-k7d2aWDPuSUYwC1fLZmQnARZrGpyE6kRQAC6-ZEkAOrrK6zhvWGDyS~8mYTq03Cxd26KrDq1gDa~e4VdteZ7ZVtd~f7mXw1L3dBqpeMJxqLgN9xLo~-NBpzjIhLA9T7L-4GdMe1te91hqruD41lYLuJexPncWHzVBtvW-YtB~DsmVefGsf3YJbCP0Lyw4V2bz4UyY1MJinO6jYnkqmZI8wCem~WxVS9aBTgDzrXnYknd05d~tn5Stsfj5PPMA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>), and the resulting scaffolds were filled by 71.5 Gb of Illumina PE reads (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy024/1/dsy024_supplementary_table_s1.xlsx?Expires=2147483647&amp;Signature=ZXQ~~2kF~81JP0zehGivG8oAMg1khl~qDbt-C-bLvEVxLxrxOJSaPIwQqPZLMOVJt4No7RRIWEOprcg2SPhdW9Ff-k7d2aWDPuSUYwC1fLZmQnARZrGpyE6kRQAC6-ZEkAOrrK6zhvWGDyS~8mYTq03Cxd26KrDq1gDa~e4VdteZ7ZVtd~f7mXw1L3dBqpeMJxqLgN9xLo~-NBpzjIhLA9T7L-4GdMe1te91hqruD41lYLuJexPncWHzVBtvW-YtB~DsmVefGsf3YJbCP0Lyw4V2bz4UyY1MJinO6jYnkqmZI8wCem~WxVS9aBTgDzrXnYknd05d~tn5Stsfj5PPMA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). As a result, 1,225 scaffolds (&gt;2 kb) were obtained (<a reveal-id="dsy024-F1" data-open="dsy024-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>, Scaffolds A and <span class="content-section supplementary-material"><a path-from-xml="sup4" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy024/1/dsy024_supplementary_table_s4.xlsx?Expires=2147483647&amp;Signature=S62dCvdyf6CiYkS4Obtb-LQjhorJVtDjwzDv~qWl9Dc3PFwrcUrxFjltK4-5OxVG2naeBupJVaVrtnzZbqULbnZkCYptSehiwx9~5eIL6~CxtqE7cAWpJiYKEVOhxLxB8v9f9LVHjqe9S1ZOc11zpub1hcuCvbTq9pfYBYz1x4RoeMHC~mbKG2apyZP1s07Ml15Aq8AspZmvGWQ3ow0ssIrj4ZCwoTRHFfITN6AhcpHdL8qFbvRoOpOIVlo3vaCYHkHK7h178o4LATWE8InMRdWV5tcHpRNxQzOXyxDvi6-3LLC9lthMZ6ImmiFtHYqNKcy3ftPQYVfHKK2j8EI2eA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>). Further scaffolding using RNA-Seq datasets generated in this study (<span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy024/1/dsy024_supplementary_table_s2.xlsx?Expires=2147483647&amp;Signature=fRv2fXdIjcGOV0lUu4UfWW84KSrtuB-LMM349frfrVPuviBIPl2aX~5o2flP8nZd7osb1C0sdf~kMR8AE7ixasCoQAxcZzVYdq-14rHV2Rguwg9SUeUrokHoBL96NDNrLpC8SgmV6E5iD-WHxLJAKZHZPbQXEP5s0b96i1rPwl5D8IhOvY2vaZGWNsECnBEOEtdUjdYU9t6UMCfY1lJrwhlLk3NtY6PRm1E7Z51Wa2T1oWV4K~kvrKhVnbcra09KOLMKDuvPlvXbRbZOmeU64bF~47e27B1Xt3CFXcFOo9t2e~bHmJYRJYjO-kbYCAxUFL2K1VcaBZulFXeu6mtQjA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>) was conducted and 1,155 scaffolds (&gt;2 kb) were constructed (<a reveal-id="dsy024-F1" data-open="dsy024-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>, Scaffolds B and <span class="content-section supplementary-material"><a path-from-xml="sup4" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy024/1/dsy024_supplementary_table_s4.xlsx?Expires=2147483647&amp;Signature=S62dCvdyf6CiYkS4Obtb-LQjhorJVtDjwzDv~qWl9Dc3PFwrcUrxFjltK4-5OxVG2naeBupJVaVrtnzZbqULbnZkCYptSehiwx9~5eIL6~CxtqE7cAWpJiYKEVOhxLxB8v9f9LVHjqe9S1ZOc11zpub1hcuCvbTq9pfYBYz1x4RoeMHC~mbKG2apyZP1s07Ml15Aq8AspZmvGWQ3ow0ssIrj4ZCwoTRHFfITN6AhcpHdL8qFbvRoOpOIVlo3vaCYHkHK7h178o4LATWE8InMRdWV5tcHpRNxQzOXyxDvi6-3LLC9lthMZ6ImmiFtHYqNKcy3ftPQYVfHKK2j8EI2eA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>). Finally, the linkage marker sequences from the ddRAD-based genetic linkage map of the yellowtail generated in this study were used to correct the scaffold misassemblies (<a reveal-id="dsy024-F1" data-open="dsy024-F1" class="link link-reveal link-table xref-fig">Fig 1</a>, Final scaffolds). The female and male maps (consisting of 1, 163 and 1, 147 ddRAD markers for female and male maps, respectively) for LGs (LG1–LG24) are shown in <span class="content-section supplementary-material"><a path-from-xml="sup11" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy024/1/dsy024_supplementary_fig_s2.pdf?Expires=2147483647&amp;Signature=iVkTlSCcoVqwNeU2kNLaQsq-8-~LKFbnkpJg2kuZVV9D6-~FbGHb6LdaN2CXbTA5HlctI~2Y1boxf0A1oGUEvI-PApOKQOLj9H7un1sUzz-fVnaRtVOtNbhPepCDptwq9HAqIGOdzsBnzTo1zO6-ElINKFAQWsN0Fb-8O1F19O8X572qQP~xN6~-ldPmFtG9o4WJIaJJqf0AqbTZ~oPz0xcvnHSTXIToE0c-xvGerbQy9I5Oc8lq6rGAe2ChyRyVOY8Rg~cH~QOq9NjvHWg2hMdl1xhILVm4VyZziGIii~c-Z4Cz~SdWpYoeHpJMv1GjAL-~JTOnrJ9AsTMWpmfyow__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>, and the locations of genetic markers are listed in <span class="content-section supplementary-material"><a path-from-xml="sup5" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy024/1/dsy024_supplementary_table_s5.xlsx?Expires=2147483647&amp;Signature=qj4p1YvFQwJIr4-aMiLTNSv1i7~sfNo~Pw3CJ9WTXq017Lp7xJqjexN83lK1IcUHC50s46eRih-uANzNwxuWw91sh4-i2Bx6WMNtyozXgt7g1NY6PiUoDnb2fxxTst4mk2FiP4una5Z4OqxZFpVOxVJAuWVmBSUuXUgIAalPOjW8dwxdm8FU0QZP-bfM~dpnBBVJiJpBIqoSO-jiX~pJhTU-mN4rBJtF6vRReJdsIebnc1myApt0J7n96vf6wAgsLO3np-va6~ELqLkZpv3H8WwLLme7jZOtrmhe4eRhXsqo6RcJCkk-qWxj7XAe4EcnTYnqVwdH0O6jAtoRmpVzsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>. The final assembly is composed of 1, 394 scaffolds (&gt;2 kb) and the total size of scaffolds is 627.3 Mb, with a G + C content of 40.9%, an N50 length of 1.43 Mb (number of N50 or longer: 127) and a largest scaffold size of 6.62 Mb (<a reveal-id="dsy024-T1" data-open="dsy024-T1" class="link link-reveal link-table xref-fig">Table 1</a>). <div content-id="dsy024-T1" class="table-modal"><div class="table-wrap"><div id="dsy024-T1" data-id="dsy024-T1" class="table-wrap-title"><div class="label -label">Table 1.</div><div class="caption"><p>Assembly statistics of yellowtail genome</p></div></div><div class="table-overflow"><table><tbody><tr><td>Number of scaffolds (&gt;2 kb) </td><td>1,394 </td></tr><tr><td>Total size of scaffolds (bp) </td><td>627,268,966 </td></tr><tr><td>Average scaffold length (bp) </td><td>449,978 </td></tr><tr><td>N50 scaffold length (bp) </td><td>1,433,177 </td></tr><tr><td>N50 scaffold number </td><td>127 </td></tr><tr><td>N90 scaffold length (bp) </td><td>404,933 </td></tr><tr><td>N90 scaffold number </td><td>453 </td></tr><tr><td>Longest scaffold (bp) </td><td>6,617,595 </td></tr><tr><td>GC (%) </td><td>40.9 </td></tr></tbody></table></div><div content-id="dsy024-T1" class="table-modal"><table><tbody><tr><td>Number of scaffolds (&gt;2 kb) </td><td>1,394 </td></tr><tr><td>Total size of scaffolds (bp) </td><td>627,268,966 </td></tr><tr><td>Average scaffold length (bp) </td><td>449,978 </td></tr><tr><td>N50 scaffold length (bp) </td><td>1,433,177 </td></tr><tr><td>N50 scaffold number </td><td>127 </td></tr><tr><td>N90 scaffold length (bp) </td><td>404,933 </td></tr><tr><td>N90 scaffold number </td><td>453 </td></tr><tr><td>Longest scaffold (bp) </td><td>6,617,595 </td></tr><tr><td>GC (%) </td><td>40.9 </td></tr></tbody></table></div></div></div><div class="table-wrap"><div id="dsy024-T1" data-id="dsy024-T1" class="table-wrap-title"><div class="label -label">Table 1.</div><div class="caption"><p>Assembly statistics of yellowtail genome</p></div></div><div class="table-overflow"><table><tbody><tr><td>Number of scaffolds (&gt;2 kb) </td><td>1,394 </td></tr><tr><td>Total size of scaffolds (bp) </td><td>627,268,966 </td></tr><tr><td>Average scaffold length (bp) </td><td>449,978 </td></tr><tr><td>N50 scaffold length (bp) </td><td>1,433,177 </td></tr><tr><td>N50 scaffold number </td><td>127 </td></tr><tr><td>N90 scaffold length (bp) </td><td>404,933 </td></tr><tr><td>N90 scaffold number </td><td>453 </td></tr><tr><td>Longest scaffold (bp) </td><td>6,617,595 </td></tr><tr><td>GC (%) </td><td>40.9 </td></tr></tbody></table></div><div content-id="dsy024-T1" class="table-modal"><table><tbody><tr><td>Number of scaffolds (&gt;2 kb) </td><td>1,394 </td></tr><tr><td>Total size of scaffolds (bp) </td><td>627,268,966 </td></tr><tr><td>Average scaffold length (bp) </td><td>449,978 </td></tr><tr><td>N50 scaffold length (bp) </td><td>1,433,177 </td></tr><tr><td>N50 scaffold number </td><td>127 </td></tr><tr><td>N90 scaffold length (bp) </td><td>404,933 </td></tr><tr><td>N90 scaffold number </td><td>453 </td></tr><tr><td>Longest scaffold (bp) </td><td>6,617,595 </td></tr><tr><td>GC (%) </td><td>40.9 </td></tr></tbody></table></div></div></p><p>The completeness of the generated genome assembly was evaluated using CEGMA analysis, which identified 97.86% complete and 98.28% partial genes from the 233 CVGs. This percentage of complete gene hits is higher than that of model fish genome assemblies such as medaka (93.56%), zebrafish (93.13%), stickleback (92.27%) and spotted green puffer (90.56%), according to the gVolante, Completeness Score Database (<a class="link link-uri" href="https://gvolante.riken.jp/script/database.cgi" target="_blank">https://gvolante.riken.jp/script/database.cgi</a> (11 July 2018, date last accessed)). Recently, two yellowtail genome assemblies (Squ_1.0 and Squ_2.0) have been deposited at DDBJ/EMBL/Genbank under accession numbers <a class="link link-gen" href="http://www.ncbi.nlm.nih.gov/nuccore/BDJM01000000" target="_blank">BDJM01000000</a> (Squ_1.0) and <a class="link link-gen" href="http://www.ncbi.nlm.nih.gov/nuccore/BDMU00000000" target="_blank">BDMU00000000</a> (Squ_2.0). According to assembly information at NCBI (<a class="link link-uri" href="https://www.ncbi.nlm.nih.gov/assembly/organism/8161/all/" target="_blank">https://www.ncbi.nlm.nih.gov/assembly/organism/8161/all/</a> (11 July 2018, date last accessed)), both assemblies were generated from diploid DNA samples with an Illumina Hiseq2500 platform and a long-read sequencer PacBio RSII. Squ_2.0 is an updated version of Squ_1.0 and is representative of the genome of this species at present. Although Squ_2.0 has fewer scaffolds (384) and a larger N50 scaffold size (5.6 Mb) than our assembly, the assembly quality based on BUSCO and CEGMA analysis showed that our assembly has higher scores than those of Squ_2.0 in both analyses. Our assembly captured 97.27% (4,459 out of 4,584) of complete BUSCOs, while Squ_2.0 captured 96.81% (4,438 out of 4,584). Similarly, CEGMA evaluation showed that the completeness of our assembly and that of Squ_2.0 was 97.85% (228 out of 233) and 97.42% (227 out of 233), respectively. Despite using relatively short sequence reads for the genome assembly, we obtained a high level of completeness in BUSCO and CEGMA analyses, suggesting the advantage of utilizing the haploid genome for <em>de novo</em> assembly. Thus, we concluded that our assembly has a more complete gene space than Squ_2.0 and is more suitable for conducting gene-prediction and annotation of yellowtail.</p>                            <h3 scrollto-destination=122671137 id="122671137" class="section-title" >3.2. Annotation of protein-coding genes</h3>
<p>A total of 27,693 protein-coding genes were predicted for the yellowtail draft genome, based on AUGUSTUS, the yellowtail training model and the transcript + protein hints. A summary of the functional annotation of the yellowtail predicted genes is shown in <a reveal-id="dsy024-T2" data-open="dsy024-T2" class="link link-reveal link-table xref-fig">Table 2</a> and the results of the functional annotation of the complete set of predicted protein-coding genes are available in <span class="content-section supplementary-material"><a path-from-xml="sup7" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy024/1/dsy024_supplementary_table_s7.xlsx?Expires=2147483647&Signature=B-UKJRuFk0CuOM1vqtJ8PJp4rsuY~rLtnkKLUuj02ZVm20F1H2JGCIpIvQW~EVHanNj3yexU-CI0nXMi7IezUHO0T474OIANzB47N9M6JThXj1II108-PCs2wVC3S3Jz9p6v6Qh7DhV1-Hp~Wlj1slC6NHfKCrDg5hp93hCzuKg0YyjY4zH7Zr5QLUtVoNaAH1Mcgmzo~yjA5DllrMCMvMFJlBYhjTACrKbpfzMKKJH8ybDnxahUwpTwGN175ncaVGdlVUexwOY5fNBHk6vajnotXmLvWcHmgaKZYVZsgB4vRIaMLPSGnulF70Sy4z-kwk~5whRHvp2qMaJ64JmNWg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S7</a></span>. Of the 27,693 predicted genes, 25,319 (91.4%) genes matched (<em>E</em> value threshold of 1 <em>E</em> − 5) at least one entry in the NCBI nr database, while 21,233 (76.7%) and 23,422 (84.6%) genes matched with model fishes, medaka and zebrafish, respectively (<a reveal-id="dsy024-T2" data-open="dsy024-T2" class="link link-reveal link-table xref-fig">Table 2</a>). A total of 18,575 (67.1%) genes mapped to at least one GO term (biological process: 12,606 genes, cellular component: 11,784 genes and molecular function: 13,094 genes). In addition, 13,880 (50.1%) genes were mapped to KEGG pathways, while 2,867 genes were identified to have at least one enzyme hit. The conserved domain search (InterPro) showed that 25,035 (90.4%) genes were identified to have at least one domain. Overall, 25,832 predicted genes (93.3%) of yellowtail were functionally annotated by at least one database, and only 1,861 predicted genes (6.7%) were unannotated (<a reveal-id="dsy024-T2" data-open="dsy024-T2" class="link link-reveal link-table xref-fig">Table 2</a>). This richness in functional annotation for yellowtail protein-coding genes will help us to better understand their biology in downstream studies, such as whole-transcriptome analysis. <div content-id="dsy024-T2" class="table-modal"><div class="table-wrap"><div id="dsy024-T2" data-id="dsy024-T2" class="table-wrap-title"><div class="label -label">Table 2.</div><div class="caption"><p>Functional annotation of the 27,693 yelowtail predicted protein-coding genes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Database </th><th>Number of the yellowtail genes mached </th><th>Percentage of annotated genes </th></tr></thead><tbody><tr><td>NCBI non-redundant (nr) database<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td>25,319 </td><td>91.4% </td></tr><tr><td>Medaka genes<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td>21,233 </td><td>76.7% </td></tr><tr><td>Zebrafish genes<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td>23,422 </td><td>84.6% </td></tr><tr><td>Gene ontrogy (GO) </td><td>18,575 </td><td>67.1% </td></tr><tr><td> Biological process </td><td>12,606 </td><td>45.5% </td></tr><tr><td> Cellular component </td><td>11,784 </td><td>42.6% </td></tr><tr><td> Molecular function </td><td>13,094 </td><td>47.3% </td></tr><tr><td>InterPro </td><td>25,035 </td><td>90.4% </td></tr><tr><td>Enzyme </td><td>2,867 </td><td>10.4% </td></tr><tr><td>KEGG pathway </td><td>13,880 </td><td>50.1% </td></tr><tr><td>At least one functional annotation </td><td>25,832 </td><td>93.3% </td></tr><tr><td>Unannotated </td><td>1, 861 </td><td>— </td></tr></tbody></table></div><div content-id="dsy024-T2" class="table-modal"><table><thead name="thead" align=""><tr><th>Database </th><th>Number of the yellowtail genes mached </th><th>Percentage of annotated genes </th></tr></thead><tbody><tr><td>NCBI non-redundant (nr) database<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td>25,319 </td><td>91.4% </td></tr><tr><td>Medaka genes<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td>21,233 </td><td>76.7% </td></tr><tr><td>Zebrafish genes<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td>23,422 </td><td>84.6% </td></tr><tr><td>Gene ontrogy (GO) </td><td>18,575 </td><td>67.1% </td></tr><tr><td> Biological process </td><td>12,606 </td><td>45.5% </td></tr><tr><td> Cellular component </td><td>11,784 </td><td>42.6% </td></tr><tr><td> Molecular function </td><td>13,094 </td><td>47.3% </td></tr><tr><td>InterPro </td><td>25,035 </td><td>90.4% </td></tr><tr><td>Enzyme </td><td>2,867 </td><td>10.4% </td></tr><tr><td>KEGG pathway </td><td>13,880 </td><td>50.1% </td></tr><tr><td>At least one functional annotation </td><td>25,832 </td><td>93.3% </td></tr><tr><td>Unannotated </td><td>1, 861 </td><td>— </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><div class="label -label">a</div><p>BLASTP hit of less than 1<em>E−</em>5.</p></div></div></div></div></div><div class="table-wrap"><div id="dsy024-T2" data-id="dsy024-T2" class="table-wrap-title"><div class="label -label">Table 2.</div><div class="caption"><p>Functional annotation of the 27,693 yelowtail predicted protein-coding genes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Database </th><th>Number of the yellowtail genes mached </th><th>Percentage of annotated genes </th></tr></thead><tbody><tr><td>NCBI non-redundant (nr) database<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td>25,319 </td><td>91.4% </td></tr><tr><td>Medaka genes<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td>21,233 </td><td>76.7% </td></tr><tr><td>Zebrafish genes<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td>23,422 </td><td>84.6% </td></tr><tr><td>Gene ontrogy (GO) </td><td>18,575 </td><td>67.1% </td></tr><tr><td> Biological process </td><td>12,606 </td><td>45.5% </td></tr><tr><td> Cellular component </td><td>11,784 </td><td>42.6% </td></tr><tr><td> Molecular function </td><td>13,094 </td><td>47.3% </td></tr><tr><td>InterPro </td><td>25,035 </td><td>90.4% </td></tr><tr><td>Enzyme </td><td>2,867 </td><td>10.4% </td></tr><tr><td>KEGG pathway </td><td>13,880 </td><td>50.1% </td></tr><tr><td>At least one functional annotation </td><td>25,832 </td><td>93.3% </td></tr><tr><td>Unannotated </td><td>1, 861 </td><td>— </td></tr></tbody></table></div><div content-id="dsy024-T2" class="table-modal"><table><thead name="thead" align=""><tr><th>Database </th><th>Number of the yellowtail genes mached </th><th>Percentage of annotated genes </th></tr></thead><tbody><tr><td>NCBI non-redundant (nr) database<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td>25,319 </td><td>91.4% </td></tr><tr><td>Medaka genes<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td>21,233 </td><td>76.7% </td></tr><tr><td>Zebrafish genes<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td>23,422 </td><td>84.6% </td></tr><tr><td>Gene ontrogy (GO) </td><td>18,575 </td><td>67.1% </td></tr><tr><td> Biological process </td><td>12,606 </td><td>45.5% </td></tr><tr><td> Cellular component </td><td>11,784 </td><td>42.6% </td></tr><tr><td> Molecular function </td><td>13,094 </td><td>47.3% </td></tr><tr><td>InterPro </td><td>25,035 </td><td>90.4% </td></tr><tr><td>Enzyme </td><td>2,867 </td><td>10.4% </td></tr><tr><td>KEGG pathway </td><td>13,880 </td><td>50.1% </td></tr><tr><td>At least one functional annotation </td><td>25,832 </td><td>93.3% </td></tr><tr><td>Unannotated </td><td>1, 861 </td><td>— </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><div class="label -label">a</div><p>BLASTP hit of less than 1<em>E−</em>5.</p></div></div></div></div></p><p>The genome-wide comparison of orthologous gene clusters was performed to identify the degree of commonality across yellowtail and five other teleost species: Pacific bluefin tuna, croaker, tilapia, medaka and zebrafish. Based on sequence similarity of proteins, all protein sequences from the six species were grouped into 21,658 clusters, from which 9,444 (43.6%) orthologous gene clusters were shared by all six species and 20,787 (96.0%) orthologous clusters contained at least two species (<a reveal-id="dsy024-F2" data-open="dsy024-F2" class="link link-reveal link-table xref-fig">Fig. 2A</a>). On the other hand, 871 (4.0%) single-species clusters: 114 clusters (336 genes) for yellowtail, 125 clusters (364 genes) for Pacific bluefin tuna, 105 clusters (236 genes) for croaker, 109 clusters (326 genes) for tilapia, 119 (490 genes) clusters for medaka and 299 (1,515 genes) clusters for zebrafish, were identified (<a reveal-id="dsy024-F2" data-open="dsy024-F2" class="link link-reveal link-table xref-fig">Fig. 2A</a>), suggesting a lineage-specific expansion of these clusters. As seen earlier, the number of zebrafish-specific clusters was 2.4- to 2.7-fold higher than those for the other five teleosts, including yellowtail, which may explain the notable gene duplication and retention of recent duplicates in zebrafish after a teleost-specific genome duplication event around 300 million years ago.<a reveal-id="dsy024-B47" data-open="dsy024-B47" class="link link-ref link-reveal xref-bibr"><sup>47</sup></a> A hypergeometric test for assessing enriched GO terms in the single-species clusters was performed. Overall, many immune-related GO terms were found in the significantly enriched GO terms of species-specific clusters, except for those in madaka, which showed that transposon-related GO terms were the most significant (<em>P</em> = 2.58<em>E</em>-4) (<span class="content-section supplementary-material"><a path-from-xml="sup8" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy024/1/dsy024_supplementary_table_s8.xlsx?Expires=2147483647&Signature=fpvj~xNmoLTfu4V4OZ9dRScoQVLIdGIkIpUMJZMziBQ9YSdhYIrQ4JItn5xBbjqF26bcMBXaNqBZuTooHTnXAA9SM91bVW1WeijhlLfIEZyL6Jq14daOOEjSXIY3pRhSak5BqZcfkjCuEUa8-dF3bGwxquWe-4JXs9V6ge9MCo~BAOwKuKJ-WQPziF6qJDBpmAx4N0Sj2lZKjJgsLOobKyCEYiF2w~z2W10lYFi-2BJDMiHkoQsmnhcIoOMKTREDiX96CuYPftnc3NPKkBTbe8JbuPUcEEmwzkXLFNNc5fzEJCU1s6mH7ytiDXuEGZCf03rWwORVEysSVqXFUAUMhw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S8</a></span>). </p>                        <a id="122671140" scrollto-destination="122671140"></a>
<div id="dsy024-F2" data-id="dsy024-F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy024/1/m_dsy024f2.png?Expires=2147483647&Signature=G4GfRP2nisg2fCD6~VIgxWOxJQ-2-VwEyvl29Zc-9djidoURsJ86vJXQZICx~-HRk843bqrpATvVVrk8N4E0FObhkNnVI1fAMRNaGTwg34KM-Fip0BuKdE-gJBXVPf5jHzuZIS1IBBpEtL1nTvTUURmF7zA-mJYmspwT7qYtsSpy11Pk82XqM9XARnQpBAR6I~3kcMYh1XL46SG-2cBE-l0H1q1tifs5gxAhCZvlZBs4ljoq53IYDEe3srfvrPmjwFDQu1fYGqLayoT6GUFi0V8IxRDdAw9xaoL5Vk0j-zlYYOQ3mFqynzxub72b5oWBKIUDKq2paBTfD7fbPO5snw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Venn diagram showing the number of unique and shared gene clusters in (A) six teleosts (yellowtail, Pacific bluefin tuna, croaker, tilapia, medaka and zebrafish) and in (B) three Seriola species: yellowtail, greater amberjack (S. dumerili) and California yellowtail (S. dorsalis). Orthologous gene clusters were identified and visualized using OrthoVenn.43 The bar charts below the Venn diagrams represent the total number of clusters that are unique to a single species or shared between 2 and 6 species." path-from-xml="dsy024f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/122671140/dsy024f2.tif" path-from-xml="dsy024f2.tif" target="_blank">View large</a><a section="122671140" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy024/1/dsy024f2.png?Expires=2147483647&Signature=Ca3LIswdphbUV~670vnfDnIc3tEnKxdeo87lvdDJRGu9d0v1mf6HeZVtsGhSI~RKjp6VwCP4OQ5TE8T1Ls-Wjqd3FKAHmnbHRFdiGRculs5QZl9i0rnjUiz2QPMcYtw4npGlWjHgeb4uzHkU2ToNNmwlylaIEDxoM15FUeegJZjdyXVQnYejuSjXB9oGjlwanX3xA04VfHL9kCLis7Jd9if6LnCvFlRoLwhTy46~5uvWLcdqGm-w65j-7dGqhfuBnwO3QgF1a4ZMG6-SY1NGk34dVN-YO24rxjxG1H6s76qerGVRhAF-FNY1DlThMjosjvC-TtJXWaJrONdXFDDcow__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=122671140&ar=5059530&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy024f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Venn diagram showing the number of unique and shared gene clusters in (A) six teleosts (yellowtail, Pacific bluefin tuna, croaker, tilapia, medaka and zebrafish) and in (B) three <em>Seriola</em> species: yellowtail, greater amberjack (<em>S. dumerili</em>) and California yellowtail (<em>S. dorsalis</em>). Orthologous gene clusters were identified and visualized using OrthoVenn.<a reveal-id="dsy024-B43" data-open="dsy024-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a> The bar charts below the Venn diagrams represent the total number of clusters that are unique to a single species or shared between 2 and 6 species.</p></div></div><div content-id="dsy024-F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy024/1/m_dsy024f2.png?Expires=2147483647&Signature=G4GfRP2nisg2fCD6~VIgxWOxJQ-2-VwEyvl29Zc-9djidoURsJ86vJXQZICx~-HRk843bqrpATvVVrk8N4E0FObhkNnVI1fAMRNaGTwg34KM-Fip0BuKdE-gJBXVPf5jHzuZIS1IBBpEtL1nTvTUURmF7zA-mJYmspwT7qYtsSpy11Pk82XqM9XARnQpBAR6I~3kcMYh1XL46SG-2cBE-l0H1q1tifs5gxAhCZvlZBs4ljoq53IYDEe3srfvrPmjwFDQu1fYGqLayoT6GUFi0V8IxRDdAw9xaoL5Vk0j-zlYYOQ3mFqynzxub72b5oWBKIUDKq2paBTfD7fbPO5snw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Venn diagram showing the number of unique and shared gene clusters in (A) six teleosts (yellowtail, Pacific bluefin tuna, croaker, tilapia, medaka and zebrafish) and in (B) three Seriola species: yellowtail, greater amberjack (S. dumerili) and California yellowtail (S. dorsalis). Orthologous gene clusters were identified and visualized using OrthoVenn.43 The bar charts below the Venn diagrams represent the total number of clusters that are unique to a single species or shared between 2 and 6 species." path-from-xml="dsy024f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/122671140/dsy024f2.tif" path-from-xml="dsy024f2.tif" target="_blank">View large</a><a section="122671140" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy024/1/dsy024f2.png?Expires=2147483647&Signature=Ca3LIswdphbUV~670vnfDnIc3tEnKxdeo87lvdDJRGu9d0v1mf6HeZVtsGhSI~RKjp6VwCP4OQ5TE8T1Ls-Wjqd3FKAHmnbHRFdiGRculs5QZl9i0rnjUiz2QPMcYtw4npGlWjHgeb4uzHkU2ToNNmwlylaIEDxoM15FUeegJZjdyXVQnYejuSjXB9oGjlwanX3xA04VfHL9kCLis7Jd9if6LnCvFlRoLwhTy46~5uvWLcdqGm-w65j-7dGqhfuBnwO3QgF1a4ZMG6-SY1NGk34dVN-YO24rxjxG1H6s76qerGVRhAF-FNY1DlThMjosjvC-TtJXWaJrONdXFDDcow__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=122671140&ar=5059530&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy024f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Venn diagram showing the number of unique and shared gene clusters in (A) six teleosts (yellowtail, Pacific bluefin tuna, croaker, tilapia, medaka and zebrafish) and in (B) three <em>Seriola</em> species: yellowtail, greater amberjack (<em>S. dumerili</em>) and California yellowtail (<em>S. dorsalis</em>). Orthologous gene clusters were identified and visualized using OrthoVenn.<a reveal-id="dsy024-B43" data-open="dsy024-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a> The bar charts below the Venn diagrams represent the total number of clusters that are unique to a single species or shared between 2 and 6 species.</p></div></div><p>Focussing on the yellowtail-specific clusters, one of the most significant GO terms was taurine: sodium symporter activity (GO: 0005369, <em>P</em> = 0.015), containing two taurine transporter (TauT/Slc6a6) genes, g27489 and g26790 (<span class="content-section supplementary-material"><a path-from-xml="sup8" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy024/1/dsy024_supplementary_table_s8.xlsx?Expires=2147483647&Signature=fpvj~xNmoLTfu4V4OZ9dRScoQVLIdGIkIpUMJZMziBQ9YSdhYIrQ4JItn5xBbjqF26bcMBXaNqBZuTooHTnXAA9SM91bVW1WeijhlLfIEZyL6Jq14daOOEjSXIY3pRhSak5BqZcfkjCuEUa8-dF3bGwxquWe-4JXs9V6ge9MCo~BAOwKuKJ-WQPziF6qJDBpmAx4N0Sj2lZKjJgsLOobKyCEYiF2w~z2W10lYFi-2BJDMiHkoQsmnhcIoOMKTREDiX96CuYPftnc3NPKkBTbe8JbuPUcEEmwzkXLFNNc5fzEJCU1s6mH7ytiDXuEGZCf03rWwORVEysSVqXFUAUMhw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S8</a></span>). TauT mediates cellular uptake of taurine (2-aminoethane sulphonic acid) which is found in high concentrations in animal tissues.<a reveal-id="dsy024-B48" data-open="dsy024-B48" class="link link-ref link-reveal xref-bibr"><sup>48</sup></a> Taurine is a vital nutrient for many fish species, particularly for carnivorous fish, due to their limited taurine biosynthesis abilities.<a reveal-id="dsy024-B49" data-open="dsy024-B49" class="link link-ref link-reveal xref-bibr"><sup>49</sup></a> It has been reported that taurine synthesis is markedly low or negligible in yellowtail.<a reveal-id="dsy024-B50" data-open="dsy024-B50" class="link link-ref link-reveal xref-bibr"><sup>50</sup></a> Therefore, the absorption of taurine from feeds via TauT is arguably important for yellowtail. Further study of the two TauT genes from the yellowtail-specific cluster will provide insight into taurine absorption in this species.</p><p>Draft genomes for greater amberjack (<em>S. dumerili</em>)<a reveal-id="dsy024-B51" data-open="dsy024-B51" class="link link-ref link-reveal xref-bibr"><sup>51</sup></a> and California yellowtail (<em>S. dorsalis</em>)<a reveal-id="dsy024-B52" data-open="dsy024-B52" class="link link-ref link-reveal xref-bibr"><sup>52</sup></a> have recently been published. We compared the predicted-protein sequences of yellowtail with those of these two <em>Seriola</em> species. The protein sequences from yellowtail (27,693), greater amberjack (22,005) and California yellowtail (25,789) were grouped into 20,499 clusters from which 17,197 (84%) clusters were shared across all three species, and 20,058 (97.8%) clusters contained at least two species (<a reveal-id="dsy024-F2" data-open="dsy024-F2" class="link link-reveal link-table xref-fig">Fig. 2B</a>). We found 441 single-species clusters (2.2%) including 221 clusters (814 genes) from yellowtail, 22 clusters (53 genes) from greater amberjack and 198 clusters (416 genes) from California yellowtail (<a reveal-id="dsy024-F2" data-open="dsy024-F2" class="link link-reveal link-table xref-fig">Fig. 2B</a>). As in the results of the comparisons with five teleosts, the most significant enriched GO terms in yellowtail- and greater amberjack-specific clusters were related to immune processes such as antigen binding (GO: 0003823, <em>P</em> = 5.1<em>E</em>-4) in yellowtail and positive regulation of interleukin-1 beta secretion (GO: 0050718, <em>P</em> = 3.9<em>E</em>-4) in greater amberjack (<span class="content-section supplementary-material"><a path-from-xml="sup9" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy024/1/dsy024_supplementary_table_s9.xlsx?Expires=2147483647&Signature=cS0UbfjEUPaQI9g9qIXizrNQIAyrq-dV2Zk2jBeWme8fFa6OOLe7rDIcbOlY6swrEGACl~L4QA1290oq3diNrpdN~0zWkAdYBhgM3fLMYNYQUYxE2AwIg2POSswYsZShUhcw07h6EFZvaxJyACX8By7ozBSrHKfqRv0LH58G1Fa3-CgOxBLG3TvDtyap1EC511WdKP7g1v~km9IQhxUJ~qq7hH2EV4pEyvsHGwokCYNlZ4wBihbUxJVROq7wT7jXNsM2aNkiGHXbJ8tgaND5BtAqU0xD5lorp7gpWvuXGt1la2i~7PGq9SIhCcOZrZlHzx~FQx5b0NPFyi28r9FO5g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S9</a></span>). California yellowtail-specific clusters showed regulation of osteoclast differentiation (GO: 0045670, <em>P</em> = 0.008) and regulation of RNA stability (GO: 0043487, <em>P</em> = 0.008) as the most enriched clusters, but the significance of these data is unclear (<span class="content-section supplementary-material"><a path-from-xml="sup9" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy024/1/dsy024_supplementary_table_s9.xlsx?Expires=2147483647&Signature=cS0UbfjEUPaQI9g9qIXizrNQIAyrq-dV2Zk2jBeWme8fFa6OOLe7rDIcbOlY6swrEGACl~L4QA1290oq3diNrpdN~0zWkAdYBhgM3fLMYNYQUYxE2AwIg2POSswYsZShUhcw07h6EFZvaxJyACX8By7ozBSrHKfqRv0LH58G1Fa3-CgOxBLG3TvDtyap1EC511WdKP7g1v~km9IQhxUJ~qq7hH2EV4pEyvsHGwokCYNlZ4wBihbUxJVROq7wT7jXNsM2aNkiGHXbJ8tgaND5BtAqU0xD5lorp7gpWvuXGt1la2i~7PGq9SIhCcOZrZlHzx~FQx5b0NPFyi28r9FO5g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S9</a></span>). Together with the results of the comparison across six teleosts, most of the species-specific clusters included immune-related GO terms, suggesting that lineage-specific expansion of immune-related genes is likely to play a key role in species diversification. This may be due to the remarkable evolutionary plasticity of the teleost immune system in response to habitat, specific environmental factors and lineage-specific pathogens, and which is linked to the survival and radiation of the teleost linnage.<a reveal-id="dsy024-B53" data-open="dsy024-B53" class="link link-ref link-reveal xref-bibr"><sup>53</sup></a><sup>,</sup><a reveal-id="dsy024-B54" data-open="dsy024-B54" class="link link-ref link-reveal xref-bibr"><sup>54</sup></a></p>                            <h3 scrollto-destination=122671143 id="122671143" class="section-title" >3.3. A snapshot of yellowtail GI tract transcriptome</h3>
<p>We used our yellowtail reference genome for RNA-Seq analysis of three major organs of the GI tract: stomach, intestine and rectum as a first step toward understanding digestion and nutrient absorption in yellowtail at the molecular level. This RNA-Seq generated 10,692,067–27,326,877 reads (mean read length: 135–167 bp) per individual tissue type (<span class="content-section supplementary-material"><a path-from-xml="sup6" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy024/1/dsy024_supplementary_table_s6.xlsx?Expires=2147483647&Signature=EBZiyUt24oYGvcwgZsxMQCd5FFkhgsmq2IsB~6mizLvynPiqk1Iv3UbMKtJ~nUEmNhqqQ8V1i6EsiGNcRqxdAbsNSKHVuDqi0WM6aVNm9RXNI~RObHfVmmRQ21Q0~wY8EdKIu9vPLY5JLRU5vFI3Pw8k4maNvyHj-Uy5xxEqQhZtKyWq-0rppLdN-VoqAR5-opYae3bapWTkdwld0nSIsi4tTeF~Sohtf~Q-Di6bWxYbIRWToOAZSMyOya29iIRr7aj2FBob4nepbqz9owHOAegySCW4baxYTFhn0z27jPL8VUufPNLuT5GzvIpeqTdAOnTm42setw1gMzNQyztSzA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6</a></span>).</p><p>First, we characterized the highly expressed genes (top 500) in each tissue using a GO enrichment analysis, because the highly expressed genes can be considered to serve functionally important roles in the respective tissues that will help us to understand the mode of operation of each tissue. This analysis showed that most of the enriched categories were shared between the intestine and rectum, and included: ‘metallopeptidase activity’ (GO: 0008237, nine genes each in intestine and rectum), ‘serine-type peptidase activity’ (GO: 0008236, 13 genes each in intestine and rectum), ‘peptidase activity’ (GO: 0008233, 29 genes each in intestine and rectum), ‘hydrolase activity’ (GO: 0016787, 62 genes in intestine and 66 genes in rectum) under the molecular function category, and ‘proteolysis’ (GO: 0006508, 31 genes in intestine and 33 genes in rectum) under the biological process category (<a reveal-id="dsy024-F3" data-open="dsy024-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>). These categories are involved in the hydrolysis of proteins into smaller polypeptides, suggesting that both the intestine and rectum function to digest protein in the yellowtail. We also found enriched GO terms common in the stomach and rectum that are related to proton transport activity, including ‘proton-transporting ATP synthase activity, rotational mechanism’ (GO: 0046933, five genes in stomach and six genes in rectum), ‘hydrogen ion transmembrane transporter activity’ (GO: 0015078, seven genes in stomach and eight genes in rectum) under the molecular function category, and ‘proton-transporting ATP synthase complex’ (GO: 0045259, five genes in stomach and six genes in rectum) under the cellular component category (<a reveal-id="dsy024-F3" data-open="dsy024-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>). These genes probably contribute to the acidity of the stomach necessary for acid hydrolysis of feeds.<a reveal-id="dsy024-B55" data-open="dsy024-B55" class="link link-ref link-reveal xref-bibr"><sup>55</sup></a> In general, carnivorous fish have well developed stomachs capable of an acid digestion phase, whereas in stomach-less fish such as carp, no such phase exists.<a reveal-id="dsy024-B56" data-open="dsy024-B56" class="link link-ref link-reveal xref-bibr"><sup>56</sup></a> In the rectum, these genes may play a role in osmoregulation.<a reveal-id="dsy024-B57" data-open="dsy024-B57" class="link link-ref link-reveal xref-bibr"><sup>57</sup></a> The stomach has more non-overlapping enriched GO terms than other organs, and overexpressed ‘oxidoreductase activity’ (GO: 0016491, 24 genes), ‘isomerase activity’ (GO: 0016853, 11 genes) under the molecular function category, ‘tricarboxylic acid cycle’ (GO: 0006099, five genes), ‘glycolysis’ (GO: 0006096, six genes), ‘generation of precursor metabolites and energy’ (GO: 0006091, 19 genes) under the biological process category, and ‘mitochondrial inner membrane’ (GO: 0005743, six genes) under the cellular component category (<a reveal-id="dsy024-F3" data-open="dsy024-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>). These GO terms suggest the role of the stomach in energy metabolism. Although interpretation of the roles of these genes is unclear, these results suggest that the digestive function of the stomach differs from that of the intestine and rectum in yellowtail. </p>                        <a id="122671146" scrollto-destination="122671146"></a>
<div id="dsy024-F3" data-id="dsy024-F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy024/1/m_dsy024f3.png?Expires=2147483647&Signature=BEJZpZdg-g1CaKutcB2guvQWGVaPzvau7DhBW2NCxhIBBJPnrm1gK~PZE5Z0ZL9jeimMfzKFK5zhesOmpX78wImhqiCbMYbe3CvuKaYoUEmhTyMxNWY5x3t7Cg~gGEKj77ZE9HeqqdMzO2urCEdk-OXKNXwi69Aiq-Iekn8VOawZfYIg7UG-AqD8WEY3~Jn9vahOaN0X6ptkeO2zkpNmg93NK7~X~qfu~olNFLwMgDHleo1eZrbOjZ5zZbmGBg9NjyoZ11f5kOrs66WVQBsN2hn0u~1ws3Zu8wiYkDI~6ZhlxxHcX~PFcmD-Rit1bmLN5W4490b2HTyFvKf1XH~jAA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="GO enrichment analysis of the top 500 highly expressed genes in the yellowtail stomach, intestine and rectum. This analysis was performed using PANTHER45 and the zebrafish gene list was selected. Values indicate fold enrichment (P-value &lt; 0.05 after Bonferroni correction) and NS means not significant." path-from-xml="dsy024f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/122671146/dsy024f3.tif" path-from-xml="dsy024f3.tif" target="_blank">View large</a><a section="122671146" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy024/1/dsy024f3.png?Expires=2147483647&Signature=yBKT-3r8lewSL-qyXLIHysLChwWf3cN0IawLPIE-dXcjtfRtL48Q12-ApXchP25nqepeAlPJO3C0g4Ag3~PAwML2YruTioQdDhBWFYaY4zpSr24Aw0GFRRQDI9YX0oOoPp4YOIW9fiMJdGONgNtqT9zD~VTIHqbUQu4f5W2g8z6nGslXDfbO6JPGJe9L5OhkLZb8ziafZF31VOBDq59Fd1js9W4xOX6dYTfSFk~9ZaIySC~Z9CSKabxO2d9f5vo9B15PpuoAQgma5seokHvZGZCiqRZK1FE-XoUKLp7dmUiHY0D5MHdtbWFBi3i-QuRQGJC-ajeg-dt8EmJInVnngw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=122671146&ar=5059530&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy024f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>GO enrichment analysis of the top 500 highly expressed genes in the yellowtail stomach, intestine and rectum. This analysis was performed using PANTHER<a reveal-id="dsy024-B45" data-open="dsy024-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a> and the zebrafish gene list was selected. Values indicate fold enrichment (<em>P</em>-value &lt; 0.05 after Bonferroni correction) and NS means not significant.</p></div></div><div content-id="dsy024-F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy024/1/m_dsy024f3.png?Expires=2147483647&Signature=BEJZpZdg-g1CaKutcB2guvQWGVaPzvau7DhBW2NCxhIBBJPnrm1gK~PZE5Z0ZL9jeimMfzKFK5zhesOmpX78wImhqiCbMYbe3CvuKaYoUEmhTyMxNWY5x3t7Cg~gGEKj77ZE9HeqqdMzO2urCEdk-OXKNXwi69Aiq-Iekn8VOawZfYIg7UG-AqD8WEY3~Jn9vahOaN0X6ptkeO2zkpNmg93NK7~X~qfu~olNFLwMgDHleo1eZrbOjZ5zZbmGBg9NjyoZ11f5kOrs66WVQBsN2hn0u~1ws3Zu8wiYkDI~6ZhlxxHcX~PFcmD-Rit1bmLN5W4490b2HTyFvKf1XH~jAA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="GO enrichment analysis of the top 500 highly expressed genes in the yellowtail stomach, intestine and rectum. This analysis was performed using PANTHER45 and the zebrafish gene list was selected. Values indicate fold enrichment (P-value &lt; 0.05 after Bonferroni correction) and NS means not significant." path-from-xml="dsy024f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/122671146/dsy024f3.tif" path-from-xml="dsy024f3.tif" target="_blank">View large</a><a section="122671146" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy024/1/dsy024f3.png?Expires=2147483647&Signature=yBKT-3r8lewSL-qyXLIHysLChwWf3cN0IawLPIE-dXcjtfRtL48Q12-ApXchP25nqepeAlPJO3C0g4Ag3~PAwML2YruTioQdDhBWFYaY4zpSr24Aw0GFRRQDI9YX0oOoPp4YOIW9fiMJdGONgNtqT9zD~VTIHqbUQu4f5W2g8z6nGslXDfbO6JPGJe9L5OhkLZb8ziafZF31VOBDq59Fd1js9W4xOX6dYTfSFk~9ZaIySC~Z9CSKabxO2d9f5vo9B15PpuoAQgma5seokHvZGZCiqRZK1FE-XoUKLp7dmUiHY0D5MHdtbWFBi3i-QuRQGJC-ajeg-dt8EmJInVnngw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=122671146&ar=5059530&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy024f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>GO enrichment analysis of the top 500 highly expressed genes in the yellowtail stomach, intestine and rectum. This analysis was performed using PANTHER<a reveal-id="dsy024-B45" data-open="dsy024-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a> and the zebrafish gene list was selected. Values indicate fold enrichment (<em>P</em>-value &lt; 0.05 after Bonferroni correction) and NS means not significant.</p></div></div><p>Next, we surveyed DEGs among the three organs to identify tissue-specific gene expression patterns. The number of DEGs in the stomach, when compared with those in the intestine and rectum were 6,001 and 6,040, respectively, indicating that the expression pattern of the stomach was different from those of the intestine and rectum, as expected by the GO enrichment analysis of the highly expressed genes (<a reveal-id="dsy024-F3" data-open="dsy024-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>). The comparison between the intestine and the rectum also showed considerable differences in the gene expression pattern, with 2,195 DEGs, suggesting that each part has a specialized function in digestive and absorption processes. Similarly, over 1,900 DEGs (fold-change cutoff of 2) were observed between anterior-middle and posterior-intestinal segments (including the rectum) in a carnivorous fish, European sea bass (<em>Dicentrarchus labrax</em>) using a custom oligo microarray.<a reveal-id="dsy024-B58" data-open="dsy024-B58" class="link link-ref link-reveal xref-bibr"><sup>58</sup></a> Using a cluster analysis of the DEGs, we extracted expression patterns that are characteristic of the three tissue types (<span class="content-section supplementary-material"><a path-from-xml="sup12" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy024/1/dsy024_supplementary_fig_s3.pdf?Expires=2147483647&amp;Signature=bougMt6J4RuoVZ4iaqMA9kJH87j3S0HhCgjM1Bdky0h5oELSlJRnYNV7wjUoaZwPmripKukY~7iWwF13W1km9eO8SzskxIlpkyk~Q~P9oY0gvAghF4fhwM2TdVi~1ClLPQdNkACoTBRy3oCriCnDn2ggdANaUE2RodCGAcA~qazuhx0kMhzM2C6J0~ukuPEGHbEPGiNVc6ensAFUrU7voLxidouEFul8nkfKDbVLzK76bzI0vBYUVKvSRBJYxi1-dZfJOfeoixe3-8y-Vg4GjsdJOywM827ZUXdQT764kMpKHsRAnBejNmYOGAwX4GfBGaF~mstWowY0KfNghYO~KQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>). <a reveal-id="dsy024-F4" data-open="dsy024-F4" class="link link-reveal link-table xref-fig">Figure 4</a> shows the heat map of 30 DEGs with the highest coefficient of variation among the three tissues. </p>                        <a id="122671148" scrollto-destination="122671148"></a>
<div id="dsy024-F4" data-id="dsy024-F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy024/1/m_dsy024f4.png?Expires=2147483647&Signature=4dN26owFyU8qCthuZvoA2CS2MO2vyAiaAFS0ceDgWVBeb~YJYf8Hy3i3JoAIWnMIfH8Ot~H2ZMPyay41r79UQfp9wctbDyP8kzb8r4rDobK12YMsDgBcnjN2KK~QYw5L23rgisyj1kRafBghaEdIEWodiMlindnX~ZfGg0GPBcDpp2rhOoxZIQExb8E0jF5OXSRW3JyV-QJ7osReYNkKK7hAZKUIGhH4NSvV~kj9D0kc0AubUIKzq8qlmW42DMSxQ0zU6joWPmKtGqjww7AliMVYl7YgZd3dWjFBUwknWxJMohJ78wXzFNo3XcanncujRaHD34o02S6wgJegi0sz2w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Heat map of 30 DEGs with the highest coefficient of variation among the yellowtail stomach (st), intestine (int) and rectum (rec)." path-from-xml="dsy024f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/122671148/dsy024f4.tif" path-from-xml="dsy024f4.tif" target="_blank">View large</a><a section="122671148" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy024/1/dsy024f4.png?Expires=2147483647&Signature=JOobs8D7oAp3Tify2bTIDeaQgvEXqTPD0exkAwv~Y6ydLhGRhX7lr9ZRCf5ozfMwqTZYvZeiQtEk2eTsA~zH8hT63g-AWO1QkAXgZLllrRCb~sxprGnB3Pi0zQnaqKqWGTMXIJ90bn9Mn7xLFenLo7kc1isacLOBsqZxO472t~Xm-UR7IMj-Q8SoQ5VnDKRngEB8WO-EgGuTMT7CK7MSEqEe6uWuNb7RRlKgV5ivoGS-TldV8teH8tbHPc7ZZB5~gMqJQPykLpKBHAaYCW8i2fgWDGYUZAnEQWVyx43WkJ9HGUHLKzZ~LhOOOg~PIvQlzIIevMRyblL~mARNABxnCw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=122671148&ar=5059530&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy024f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Heat map of 30 DEGs with the highest coefficient of variation among the yellowtail stomach (st), intestine (int) and rectum (rec).</p></div></div><div content-id="dsy024-F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy024/1/m_dsy024f4.png?Expires=2147483647&Signature=4dN26owFyU8qCthuZvoA2CS2MO2vyAiaAFS0ceDgWVBeb~YJYf8Hy3i3JoAIWnMIfH8Ot~H2ZMPyay41r79UQfp9wctbDyP8kzb8r4rDobK12YMsDgBcnjN2KK~QYw5L23rgisyj1kRafBghaEdIEWodiMlindnX~ZfGg0GPBcDpp2rhOoxZIQExb8E0jF5OXSRW3JyV-QJ7osReYNkKK7hAZKUIGhH4NSvV~kj9D0kc0AubUIKzq8qlmW42DMSxQ0zU6joWPmKtGqjww7AliMVYl7YgZd3dWjFBUwknWxJMohJ78wXzFNo3XcanncujRaHD34o02S6wgJegi0sz2w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Heat map of 30 DEGs with the highest coefficient of variation among the yellowtail stomach (st), intestine (int) and rectum (rec)." path-from-xml="dsy024f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/122671148/dsy024f4.tif" path-from-xml="dsy024f4.tif" target="_blank">View large</a><a section="122671148" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy024/1/dsy024f4.png?Expires=2147483647&Signature=JOobs8D7oAp3Tify2bTIDeaQgvEXqTPD0exkAwv~Y6ydLhGRhX7lr9ZRCf5ozfMwqTZYvZeiQtEk2eTsA~zH8hT63g-AWO1QkAXgZLllrRCb~sxprGnB3Pi0zQnaqKqWGTMXIJ90bn9Mn7xLFenLo7kc1isacLOBsqZxO472t~Xm-UR7IMj-Q8SoQ5VnDKRngEB8WO-EgGuTMT7CK7MSEqEe6uWuNb7RRlKgV5ivoGS-TldV8teH8tbHPc7ZZB5~gMqJQPykLpKBHAaYCW8i2fgWDGYUZAnEQWVyx43WkJ9HGUHLKzZ~LhOOOg~PIvQlzIIevMRyblL~mARNABxnCw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=122671148&ar=5059530&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy024f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Heat map of 30 DEGs with the highest coefficient of variation among the yellowtail stomach (st), intestine (int) and rectum (rec).</p></div></div><p>A gene encoding potassium-transporting ATPase alpha chain 1 (g269), which has a role in acid production in the mammalian stomach<a reveal-id="dsy024-B55" data-open="dsy024-B55" class="link link-ref link-reveal xref-bibr"><sup>55</sup></a><sup>,</sup><a reveal-id="dsy024-B59" data-open="dsy024-B59" class="link link-ref link-reveal xref-bibr"><sup>59</sup></a> was specifically expressed in the stomach (<a reveal-id="dsy024-F4" data-open="dsy024-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>). As mentioned above, this gene may facilitate the acidic environment of the yellowtail stomach. In addition, four genes (g6958, g17761, g17762 and g23012) encoding pepsinogens, and activated to the active form pepsin under acidic conditions,<a reveal-id="dsy024-B60" data-open="dsy024-B60" class="link link-ref link-reveal xref-bibr"><sup>60</sup></a> were specifically expressed in the stomach (<a reveal-id="dsy024-F4" data-open="dsy024-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>). In vertebrates, including fish, pepsin has been identified as the major acidic protease in the stomach, acting during the earliest stage of protein digestion in breaking down long-chain peptides.<a reveal-id="dsy024-B61" data-open="dsy024-B61" class="link link-ref link-reveal xref-bibr"><sup>61</sup></a><sup>,</sup><a reveal-id="dsy024-B62" data-open="dsy024-B62" class="link link-ref link-reveal xref-bibr"><sup>62</sup></a> In particular, carnivorous fish have higher pepsin levels in their stomachs than omnivorous fish with stomachs and stomach-less fish,<a reveal-id="dsy024-B63" data-open="dsy024-B63" class="link link-ref link-reveal xref-bibr"><sup>63</sup></a> such as zebrafish, which lack the pepsinogen gene.<a reveal-id="dsy024-B64" data-open="dsy024-B64" class="link link-ref link-reveal xref-bibr"><sup>64</sup></a> Another digestive enzyme, encoded by a chitinase gene (g6038), was also found in the stomach-specific cluster (<a reveal-id="dsy024-F4" data-open="dsy024-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>). It has been reported that the fish stomach exhibits chitinase activity, which is associated with the digestion of chitinous substances from crustaceans.<a reveal-id="dsy024-B65" data-open="dsy024-B65" class="link link-ref link-reveal xref-bibr"><sup>65</sup></a><sup>,</sup><a reveal-id="dsy024-B66" data-open="dsy024-B66" class="link link-ref link-reveal xref-bibr"><sup>66</sup></a> We found two stomach-specific genes, the Fc fragment of IgG binding protein (FCGBP; g2075) and cysteine-rich secretory protein 2 (CRISP2; g10639). FCGBP (g2075) is abundant in the mucus of humans and mice, and may be functionally related to the gel-forming mucins,<a reveal-id="dsy024-B67" data-open="dsy024-B67" class="link link-ref link-reveal xref-bibr"><sup>67</sup></a> while CRISP2 is strongly expressed in the mammalian male reproductive tract.<a reveal-id="dsy024-B68" data-open="dsy024-B68" class="link link-ref link-reveal xref-bibr"><sup>68</sup></a> Since the function of these two genes in fish has not been elucidated, further studies are required to identify the role of these genes in the yellowtail stomach.</p><p>As expected following the results of the GO enrichment analysis (<a reveal-id="dsy024-F3" data-open="dsy024-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>), proteolytic digestive enzyme genes, a trypsinogen (precursor of trypsin) gene (g15220), two chymotrypsinogen (precursor of chymotrypsin) genes (g21846 and g21847) and a carboxypeptidase B gene (g11916) were found to be specifically expressed in the intestine and the rectum (<a reveal-id="dsy024-F4" data-open="dsy024-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>). In vertebrates including fish, hydrolysis and absorption of proteins occurs primarily in the intestine.<a reveal-id="dsy024-B69" data-open="dsy024-B69" class="link link-ref link-reveal xref-bibr"><sup>69</sup></a> It has been reported that trypsin activity in fish is significantly higher in the intestine than in the stomach, which showed very little or no activity,<a reveal-id="dsy024-B61" data-open="dsy024-B61" class="link link-ref link-reveal xref-bibr"><sup>61</sup></a><sup>,</sup><a reveal-id="dsy024-B62" data-open="dsy024-B62" class="link link-ref link-reveal xref-bibr"><sup>62</sup></a> and high trypsinogen mRNA expression levels were also found in the intestine of Atlantic salmon (<em>Salmo salar</em>),<a reveal-id="dsy024-B70" data-open="dsy024-B70" class="link link-ref link-reveal xref-bibr"><sup>70</sup></a> Senegalese sole (<em>Solea senegalensis</em>)<a reveal-id="dsy024-B71" data-open="dsy024-B71" class="link link-ref link-reveal xref-bibr"><sup>71</sup></a> and pufferfish (<em>Takifugu obscurus</em>).<a reveal-id="dsy024-B72" data-open="dsy024-B72" class="link link-ref link-reveal xref-bibr"><sup>72</sup></a> Our RNA-Seq results suggest that both the intestine and rectum play major roles in hydrolysis and absorption of proteins in yellowtail. It should be noted that the gene expression of these proteolytic digestive enzymes was low in the intestine of one fish (<a reveal-id="dsy024-F4" data-open="dsy024-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>). Calduch-Giner et al.<a reveal-id="dsy024-B58" data-open="dsy024-B58" class="link link-ref link-reveal xref-bibr"><sup>58</sup></a> suggested that fish intestine transcriptomes are not static, but change spatially, seasonally and with diet. Since the samples of GI tract examined were obtained from cultured yellowtail maintained under the same conditions and sampled at the same time, our study suggests that the expression of proteolytic enzymes also varies between individuals, and that such differences may cause different physiological conditions in individuals.</p><p>Focussing on intestine-specific gene expression, three apolipoprotein genes, apolipo Eb (g15747), apolipo B-100 (g10622) and apolipo A-1 (g15231), were included in this cluster (<a reveal-id="dsy024-F4" data-open="dsy024-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>). Apolipoproteins play a crucial role in lipid transport and uptake in vertebrates and are synthesized mainly in the intestine and liver of most teleosts.<a reveal-id="dsy024-B73 dsy024-B74 dsy024-B75" data-open="dsy024-B73 dsy024-B74 dsy024-B75" class="link link-ref link-reveal xref-bibr"><sup>73–75</sup></a> In addition, another intestine-specific gene, Type-4 ice-structuring protein LS-12 (AFP4; g15894), which was first isolated as an antifreeze protein from the serum of the longhorn sculpin (<em>Myoxocephalus octodecimspinosis</em>),<a reveal-id="dsy024-B76" data-open="dsy024-B76" class="link link-ref link-reveal xref-bibr"><sup>76</sup></a> may also play a role in lipid metabolism because it has an apolipoprotein A-II domain (InterPro: IPR006801). It has also been reported that the expression of the AFP4 gene was higher in the anterior-middle intestinal segments than the posterior segment in European seabass.<a reveal-id="dsy024-B58" data-open="dsy024-B58" class="link link-ref link-reveal xref-bibr"><sup>58</sup></a> In contrast to mammals, which use carbohydrates as their main energy source, most fish predominantly make use of lipids.<a reveal-id="dsy024-B77" data-open="dsy024-B77" class="link link-ref link-reveal xref-bibr"><sup>77</sup></a> Therefore, these apolipoproteins and AFP4 may play important roles in lipid metabolism in the yellowtail intestine. In addition to these lipid metabolism-related genes, an acidic mammalian chitinase (AMCase) gene (g23028) was found in the intestine-specific cluster (<a reveal-id="dsy024-F4" data-open="dsy024-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>), and a higher expression of this gene in the anterior-middle intestinal area as opposed to posterior segments in European seabass has been found.<a reveal-id="dsy024-B58" data-open="dsy024-B58" class="link link-ref link-reveal xref-bibr"><sup>58</sup></a> In the mouse, acidic mammalian AMCase can act in digestion of chitin polymers even in the presence of pepsin C, trypsin and chymotrypsin.<a reveal-id="dsy024-B78" data-open="dsy024-B78" class="link link-ref link-reveal xref-bibr"><sup>78</sup></a> This enzyme also plays a critical role in the human immune response to pathogens.<a reveal-id="dsy024-B79" data-open="dsy024-B79" class="link link-ref link-reveal xref-bibr"><sup>79</sup></a> It should be noted that fish intestines function as immune barriers to pathogens<a reveal-id="dsy024-B80" data-open="dsy024-B80" class="link link-ref link-reveal xref-bibr"><sup>80</sup></a><sup>,</sup><a reveal-id="dsy024-B81" data-open="dsy024-B81" class="link link-ref link-reveal xref-bibr"><sup>81</sup></a> and that fish apolipoproteins play a role in this immune function.<a reveal-id="dsy024-B82 dsy024-B83 dsy024-B84" data-open="dsy024-B82 dsy024-B83 dsy024-B84" class="link link-ref link-reveal xref-bibr"><sup>82–84</sup></a> Thus, AMCase and apolipoproteins may have roles not only in digestion and absorption but also in defense against intestinal pathogens. Four muscle-related genes, myosin-11 (g20983), desmin (g10938), actin (g20707) and filamin A (g9142) were highly expressed in the intestine (<a reveal-id="dsy024-F4" data-open="dsy024-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>). The GO enrichment analysis (<a reveal-id="dsy024-F3" data-open="dsy024-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>) also showed the over-representation of muscle-related GO terms in the intestine such as ‘cytoskeleton’ (GO: 0005856, 22 genes), ‘actin cytoskeleton’ (GO: 0015629, 13 genes) and ‘intermediate filament cytoskeleton’ (GO: 0045111, 6 genes). Recently, Abrams et al.<a reveal-id="dsy024-B85" data-open="dsy024-B85" class="link link-ref link-reveal xref-bibr"><sup>85</sup></a> reported that zebrafish myosin-11 (myh11) gene mutations affected intestinal motility. In addition, a collagen α-1(I) chain gene (g10729) was found in the intestine-specific expression cluster (<a reveal-id="dsy024-F4" data-open="dsy024-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>), and its role in responding to stretching of foetal human intestinal smooth muscle cells has been investigated.<a reveal-id="dsy024-B86" data-open="dsy024-B86" class="link link-ref link-reveal xref-bibr"><sup>86</sup></a> Thus, these muscle-related genes of yellowtail may have functions related to intestinal motility. Grau et al.<a reveal-id="dsy024-B87" data-open="dsy024-B87" class="link link-ref link-reveal xref-bibr"><sup>87</sup></a> suggested the importance of intestinal motility for intestinal absorption in carnivorous fish with irregular intake of large quantities of food.</p><p>The rectum of yellowtail is separated from the posterior region of the intestine by the ileo-rectal valve. As observed in the rectum of the greater amberjack,<a reveal-id="dsy024-B87" data-open="dsy024-B87" class="link link-ref link-reveal xref-bibr"><sup>87</sup></a> the mucosa are more deeply folded than in those of the intestine, and the rectum is also considered to play a role in absorption.<a reveal-id="dsy024-B87" data-open="dsy024-B87" class="link link-ref link-reveal xref-bibr"><sup>87</sup></a> The rectum-specific cluster included two cathepsin genes, cathepsin L1 (g24904) and cathepsin Z (g6016) (<a reveal-id="dsy024-F4" data-open="dsy024-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>). Cathepsin L1 gene also showed a higher expression in the anterior segments than in the posterior segment of the intestinal tract in zebrafish<a reveal-id="dsy024-B64" data-open="dsy024-B64" class="link link-ref link-reveal xref-bibr"><sup>64</sup></a> and European seabass.<a reveal-id="dsy024-B58" data-open="dsy024-B58" class="link link-ref link-reveal xref-bibr"><sup>58</sup></a> Cathepsins are essential lysosomal proteolytic enzymes and the activity of cathepsin D has been shown to be involved primarily with the posterior intestine in rainbow trout (<em>Oncorhynchus mykiss</em>).<a reveal-id="dsy024-B69" data-open="dsy024-B69" class="link link-ref link-reveal xref-bibr"><sup>69</sup></a> However, little is known about the function of cathepsins in the fish rectum and studies are needed to elucidate their role in digestion and absorption processes. Two multiligand endocytic receptor genes, cubilin (g20710) and megalin (g13916), were also found as rectum-specific genes (<a reveal-id="dsy024-F4" data-open="dsy024-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>), and each shown to mediate reabsorption of proteins and vitamins in mammals. The megalin/cubilin complex delivers its ligands (proteins and vitamins) to lysosomes, where all proteins are degraded, and the amino acids and vitamins returned to circulation.<a reveal-id="dsy024-B88" data-open="dsy024-B88" class="link link-ref link-reveal xref-bibr"><sup>88</sup></a><sup>,</sup><a reveal-id="dsy024-B89" data-open="dsy024-B89" class="link link-ref link-reveal xref-bibr"><sup>89</sup></a> Two rectum-specific genes, deleted in malignant brain tumours 1 (DMBT1; g13722) and macrophage mannose receptor 1 (g428), are possibly involved in defense against bacterial pathogens.<a reveal-id="dsy024-B90" data-open="dsy024-B90" class="link link-ref link-reveal xref-bibr"><sup>90</sup></a><sup>,</sup><a reveal-id="dsy024-B91" data-open="dsy024-B91" class="link link-ref link-reveal xref-bibr"><sup>91</sup></a> The gene (g9672) encoding α-N-acetylgalactosaminidase, a lysosomal glycohydrolase,<a reveal-id="dsy024-B92" data-open="dsy024-B92" class="link link-ref link-reveal xref-bibr"><sup>92</sup></a> was also specifically expressed in the rectum (<a reveal-id="dsy024-F4" data-open="dsy024-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>), although the functions of this enzyme in the rectum are unknown.</p><p>The results of GO enrichment analysis of highly expressed genes and cluster analysis of the DEGs together showed that the acid secretion-related genes and acid-activated protease (pepsinogen) genes were specifically expressed in the stomach, while the proteolytic digestive enzymes, trypsin, chymotrypsin and carboxypeptidase B, were highly and commonly expressed in the intestine and the rectum. In addition, the RNA-Seq also revealed differences in gene expression patterns between the intestine and the rectum. The possible lipid metabolism-related genes (apolipoproteins and AFP4) were specifically expressed in the intestine, while the genes involved in lysosomal digestive and reabsorption processes (cathepsins, cubilin and megalin), were specifically expressed in the rectum. These transcriptional signatures exhibit the main features of the digestive tract of carnivorous fishes, which possess higher proteolytic enzyme activities than herbivorous or omnivorous species,<a reveal-id="dsy024-B93" data-open="dsy024-B93" class="link link-ref link-reveal xref-bibr"><sup>93</sup></a><sup>,</sup><a reveal-id="dsy024-B94" data-open="dsy024-B94" class="link link-ref link-reveal xref-bibr"><sup>94</sup></a> which in general exhibit higher α-amylase (carbohydrate enzyme) activities.<a reveal-id="dsy024-B95" data-open="dsy024-B95" class="link link-ref link-reveal xref-bibr"><sup>95</sup></a> It should be noted that two α-amylase genes (g13238 and g13239) were found in the yellowtail, but their expression was low or undetected in the three GI tissues. Efficiency of food absorption and conversion can depend on the availability of digestive enzymes.<a reveal-id="dsy024-B96" data-open="dsy024-B96" class="link link-ref link-reveal xref-bibr"><sup>96</sup></a> Thus, these findings suggest that proteolytic digestive enzymes play a key role in digestion and nutrient absorption in the yellowtail GI tract.</p>                            <h3 scrollto-destination=122671154 id="122671154" class="section-title" >3.4. Gene expression patterns of proteolytic digestive enzymes</h3>
<p>Because the RNA-Seq results suggested the importance of proteolytic processes in the yellowtail GI tract, we further analysed the expression patterns of proteolytic digestive enzymes in yellowtail. In the yellowtail genome, we found 75 candidate genes for proteolytic digestive enzymes: five pepsinogens, 19 trypsinogens, four chymotrypsinogens, 11 elastases, seven carboxypeptidases (A and B), four aminopeptidase Ns, 19 cathepsins and six collagenases (<a reveal-id="dsy024-F5" data-open="dsy024-F5" class="link link-reveal link-table xref-fig">Fig. 5A</a>). The expressional percentages of these categories were expressed as TPM values for each tissue (<a reveal-id="dsy024-F5" data-open="dsy024-F5" class="link link-reveal link-table xref-fig">Fig. 5B</a>). The distribution patterns of these enzymes varied among the three tissues. In the stomach, the expressional percentage was almost entirely occupied by pepsinogen genes, which accounted for 99.9% of the total. Surprisingly, the expression percentage of pepsinogens accounted for 45.6–65.0% of the overall stomach transcriptome in three individuals. In the intestine, the transcript levels of trypsinogens were very high (58.9%), followed by chymotrypsinogens (15.4%), elastases (14.6%) and carboxypeptidases (6.4%). These gene expression patterns are correlated with the intestinal protease activity in carnivorous fish as observed in previous studies. Eshel et al.<a reveal-id="dsy024-B97" data-open="dsy024-B97" class="link link-ref link-reveal xref-bibr"><sup>97</sup></a> estimated that trypsin contributes to 40–50% of the overall protein digestion in carnivorous fish intestines. Furthermore, it has been reported that trypsin activity in carnivorous fish is about four times (3.9 times) that of chymotrypsin activity, a relationship that is reversed in omnivorous and herbivorous species.<a reveal-id="dsy024-B98" data-open="dsy024-B98" class="link link-ref link-reveal xref-bibr"><sup>98</sup></a> Similarly, the expressional percentage of trypsinogen genes in yellowtail showed about four times (3.8 times) that of chymotrypsinogen genes. Trypsinogen genes were also expressed at the highest levels in the rectum (36.6%), but the expressional percentage was lower than that of the intestine (58.9%). Alternatively, cathepsin genes showed a high transcript level in the rectum which accounted for 32.8%, the highest among the three tissues. </p>                        <a id="122671156" scrollto-destination="122671156"></a>
<div id="dsy024-F5" data-id="dsy024-F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy024/1/m_dsy024f5.png?Expires=2147483647&amp;Signature=oR25fbgyhNqUd1rQAhD36CSkQQudK-PVGON42Bx0hfNOIh10sAqgZf5wQXc2vaA29pB89Y~tRvNrmwtVI0KItQ9BSFC-9OmeH5kFE9FeZmPdj7KlQnCcnFOS3Ab6vHV-FCbVGRZyS~P89BDXeJuBGM794it6fEVIosZz5feG8csvX1AznQJE2b~uyVUhvoXg9oikEyicA9GE6DHK7~cbCDF6NyJXJhVI8Nahf6oDQks53vm6B1mfNf5l-CgpU0blq7R3yWiMSWHMI5kuUL893CM4qcxtsqQbmorJok8GOB7Cre3w5ZbRKuXgpXq4kfyYCQTYYGXgE9ooPpeCPqFUCw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Transcript levels of proteolytic digestive enzyme genes in the yellowtail stomach, intestine and rectum. (A) The number of proteolytic digestive enzyme genes identified in the yellowtail genome. (B) Expressional percentages based on TPM values for different categories of proteolytic enzyme genes. The expression levels of individual genes in the highly expressed categories in each tissue are also shown in (C) pepsinogen genes in stomach, (D) trypsinogen genes in intestine and rectum and (E) cathepsin genes in the rectum. The amino acid sequence identities and similarities between these highly expressed genes can be found in Supplementary Fig. S4." path-from-xml="dsy024f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/122671156/dsy024f5.tif" path-from-xml="dsy024f5.tif" target="_blank">View large</a><a section="122671156" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy024/1/dsy024f5.png?Expires=2147483647&amp;Signature=ODZZqfK6UrIA8shHKBrB0vbkZzYJvKm8O9GB6rbaUURpSEicpzsIw6lJ~8iaf0TtXBNPjzYMl53wxOYAGP-ZYjgsrEk09vVobDEfhtkIYa3fEFQcnpyKWc6BsAc1ktGJyawbqTNmUeY3BHXQagnuKyHlzt8kuXMXihKoSfa-Kcyay~9nSx80i-qOJkm6lBeNmboFyF6oCLVkr6K9THhvImrkPu1NuJFRTfr-gvpx1eulr~fe356rsmNrCdPojMUVlcR5S3hd~5K0DTLjL-6yQMiBkamnp4Sl-8iFydcPVHJG8Iy-ZXGds5hwNIA3v5pEsECl6xQEpE8wcCYTchc0RA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=122671156&amp;ar=5059530&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsy024f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Transcript levels of proteolytic digestive enzyme genes in the yellowtail stomach, intestine and rectum. (A) The number of proteolytic digestive enzyme genes identified in the yellowtail genome. (B) Expressional percentages based on TPM values for different categories of proteolytic enzyme genes. The expression levels of individual genes in the highly expressed categories in each tissue are also shown in (C) pepsinogen genes in stomach, (D) trypsinogen genes in intestine and rectum and (E) cathepsin genes in the rectum. The amino acid sequence identities and similarities between these highly expressed genes can be found in <span class="content-section supplementary-material"><a path-from-xml="sup13" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy024/1/dsy024_supplementary_fig_s4.pdf?Expires=2147483647&amp;Signature=evZrhfEbNvNkd4Qc898wmTGyyaM3UZijdwWfrb0lgjfSL-M0LhjT5cLz9J4HkOR9rPt2L~NUv84hD5I5Xz5m-MN~7Wjxfb0bYwUydXwQXzZYXd7nm~sQzusa0KB4XLyTdTbG1O5cl0AzYDD6Way1qtRp1O0BK76Okx-tsDCfKjr0R36g5QRvQ~aKszhqe4-BFOXs2wSryt1txUs8G9Jte6mxQ7aSlkGn-QzDcsc3IpCudpB419e9-NmI2tXyI4f4pzoaKSuO3T~CmE8EuC01zmYR6cbKlyDO01MK3ha5aZ0I3lhCihFtjQJLBTsLL5Q00Moh6E8vk2YambWTFF4jdg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>.</p></div></div><div content-id="dsy024-F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy024/1/m_dsy024f5.png?Expires=2147483647&amp;Signature=oR25fbgyhNqUd1rQAhD36CSkQQudK-PVGON42Bx0hfNOIh10sAqgZf5wQXc2vaA29pB89Y~tRvNrmwtVI0KItQ9BSFC-9OmeH5kFE9FeZmPdj7KlQnCcnFOS3Ab6vHV-FCbVGRZyS~P89BDXeJuBGM794it6fEVIosZz5feG8csvX1AznQJE2b~uyVUhvoXg9oikEyicA9GE6DHK7~cbCDF6NyJXJhVI8Nahf6oDQks53vm6B1mfNf5l-CgpU0blq7R3yWiMSWHMI5kuUL893CM4qcxtsqQbmorJok8GOB7Cre3w5ZbRKuXgpXq4kfyYCQTYYGXgE9ooPpeCPqFUCw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Transcript levels of proteolytic digestive enzyme genes in the yellowtail stomach, intestine and rectum. (A) The number of proteolytic digestive enzyme genes identified in the yellowtail genome. (B) Expressional percentages based on TPM values for different categories of proteolytic enzyme genes. The expression levels of individual genes in the highly expressed categories in each tissue are also shown in (C) pepsinogen genes in stomach, (D) trypsinogen genes in intestine and rectum and (E) cathepsin genes in the rectum. The amino acid sequence identities and similarities between these highly expressed genes can be found in Supplementary Fig. S4." path-from-xml="dsy024f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/122671156/dsy024f5.tif" path-from-xml="dsy024f5.tif" target="_blank">View large</a><a section="122671156" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy024/1/dsy024f5.png?Expires=2147483647&amp;Signature=ODZZqfK6UrIA8shHKBrB0vbkZzYJvKm8O9GB6rbaUURpSEicpzsIw6lJ~8iaf0TtXBNPjzYMl53wxOYAGP-ZYjgsrEk09vVobDEfhtkIYa3fEFQcnpyKWc6BsAc1ktGJyawbqTNmUeY3BHXQagnuKyHlzt8kuXMXihKoSfa-Kcyay~9nSx80i-qOJkm6lBeNmboFyF6oCLVkr6K9THhvImrkPu1NuJFRTfr-gvpx1eulr~fe356rsmNrCdPojMUVlcR5S3hd~5K0DTLjL-6yQMiBkamnp4Sl-8iFydcPVHJG8Iy-ZXGds5hwNIA3v5pEsECl6xQEpE8wcCYTchc0RA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=122671156&amp;ar=5059530&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsy024f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Transcript levels of proteolytic digestive enzyme genes in the yellowtail stomach, intestine and rectum. (A) The number of proteolytic digestive enzyme genes identified in the yellowtail genome. (B) Expressional percentages based on TPM values for different categories of proteolytic enzyme genes. The expression levels of individual genes in the highly expressed categories in each tissue are also shown in (C) pepsinogen genes in stomach, (D) trypsinogen genes in intestine and rectum and (E) cathepsin genes in the rectum. The amino acid sequence identities and similarities between these highly expressed genes can be found in <span class="content-section supplementary-material"><a path-from-xml="sup13" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy024/1/dsy024_supplementary_fig_s4.pdf?Expires=2147483647&amp;Signature=evZrhfEbNvNkd4Qc898wmTGyyaM3UZijdwWfrb0lgjfSL-M0LhjT5cLz9J4HkOR9rPt2L~NUv84hD5I5Xz5m-MN~7Wjxfb0bYwUydXwQXzZYXd7nm~sQzusa0KB4XLyTdTbG1O5cl0AzYDD6Way1qtRp1O0BK76Okx-tsDCfKjr0R36g5QRvQ~aKszhqe4-BFOXs2wSryt1txUs8G9Jte6mxQ7aSlkGn-QzDcsc3IpCudpB419e9-NmI2tXyI4f4pzoaKSuO3T~CmE8EuC01zmYR6cbKlyDO01MK3ha5aZ0I3lhCihFtjQJLBTsLL5Q00Moh6E8vk2YambWTFF4jdg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>.</p></div></div><p>To determine whether there are differential gene usages within each category of proteolytic digestive enzymes, we further analysed the expression levels of individual genes in the highly expressed categories in each tissue (<a reveal-id="dsy024-F5" data-open="dsy024-F5" class="link link-reveal link-table xref-fig">Fig. 5B</a>): pepsinogen genes in the stomach (<a reveal-id="dsy024-F5" data-open="dsy024-F5" class="link link-reveal link-table xref-fig">Fig. 5C</a>), trypsinogen genes in the intestine and rectum (<a reveal-id="dsy024-F5" data-open="dsy024-F5" class="link link-reveal link-table xref-fig">Fig. 5D</a>) and cathepsin genes in the rectum (<a reveal-id="dsy024-F5" data-open="dsy024-F5" class="link link-reveal link-table xref-fig">Fig. 5E</a>). This analysis revealed that individual gene expression levels are different for each enzyme. Of the five pepsinogen genes in the stomach, g23012 occupied 72.1% of the total expression value, followed by g6958 which accounted for 16.0% (<a reveal-id="dsy024-F5" data-open="dsy024-F5" class="link link-reveal link-table xref-fig">Fig. 5C</a>). In teleosts, two types of pepsinogen have been identified, pepsinogen A and pepsinogen C.<a reveal-id="dsy024-B99" data-open="dsy024-B99" class="link link-ref link-reveal xref-bibr"><sup>99</sup></a> Both g23012 and g6958 genes encode pepsinogen A, and the amino acid identity and similarity between these two sequences are 64.6 and 78.1%, respectively (<span class="content-section supplementary-material"><a path-from-xml="sup13" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy024/1/dsy024_supplementary_fig_s4.pdf?Expires=2147483647&amp;Signature=evZrhfEbNvNkd4Qc898wmTGyyaM3UZijdwWfrb0lgjfSL-M0LhjT5cLz9J4HkOR9rPt2L~NUv84hD5I5Xz5m-MN~7Wjxfb0bYwUydXwQXzZYXd7nm~sQzusa0KB4XLyTdTbG1O5cl0AzYDD6Way1qtRp1O0BK76Okx-tsDCfKjr0R36g5QRvQ~aKszhqe4-BFOXs2wSryt1txUs8G9Jte6mxQ7aSlkGn-QzDcsc3IpCudpB419e9-NmI2tXyI4f4pzoaKSuO3T~CmE8EuC01zmYR6cbKlyDO01MK3ha5aZ0I3lhCihFtjQJLBTsLL5Q00Moh6E8vk2YambWTFF4jdg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4A</a></span>). Of 19 trypsinogen genes, only three, g15220, g121 and g15219, accounted for &gt;70% of the total in the intestine and rectum (<a reveal-id="dsy024-F5" data-open="dsy024-F5" class="link link-reveal link-table xref-fig">Fig. 5D</a>). Among them, g15220 showed about half of the total expression value in the intestine (46.9%) and rectum (51.7%). Based on amino acid sequence identities, most trypsins can be classified into three basic Groups I–III.<a reveal-id="dsy024-B100" data-open="dsy024-B100" class="link link-ref link-reveal xref-bibr"><sup>100</sup></a> g15220 and g15219 encode Trypsin I, and the amino acid identity and similarity between them was 84.3 and 92.6%, respectively. On the other hand, g121 encodes Trypsin III, and it has 69.0 and 82.7% amino acid identity and similarity with g15220, and 73.9 and 86.7% amino acid identity and similarity with g15219 (<span class="content-section supplementary-material"><a path-from-xml="sup13" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy024/1/dsy024_supplementary_fig_s4.pdf?Expires=2147483647&amp;Signature=evZrhfEbNvNkd4Qc898wmTGyyaM3UZijdwWfrb0lgjfSL-M0LhjT5cLz9J4HkOR9rPt2L~NUv84hD5I5Xz5m-MN~7Wjxfb0bYwUydXwQXzZYXd7nm~sQzusa0KB4XLyTdTbG1O5cl0AzYDD6Way1qtRp1O0BK76Okx-tsDCfKjr0R36g5QRvQ~aKszhqe4-BFOXs2wSryt1txUs8G9Jte6mxQ7aSlkGn-QzDcsc3IpCudpB419e9-NmI2tXyI4f4pzoaKSuO3T~CmE8EuC01zmYR6cbKlyDO01MK3ha5aZ0I3lhCihFtjQJLBTsLL5Q00Moh6E8vk2YambWTFF4jdg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4B</a></span>). Of 19 cathepsin genes in the rectum, the transcript levels of g24904 (cathepsin L1) were highest (40.5%), followed by g6016 (cathepsin Z, 29.3%) and g6015 (cathepsin Z, 18.2%). The protein homology values among them were low or showed no significant similarity (<span class="content-section supplementary-material"><a path-from-xml="sup13" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy024/1/dsy024_supplementary_fig_s4.pdf?Expires=2147483647&amp;Signature=evZrhfEbNvNkd4Qc898wmTGyyaM3UZijdwWfrb0lgjfSL-M0LhjT5cLz9J4HkOR9rPt2L~NUv84hD5I5Xz5m-MN~7Wjxfb0bYwUydXwQXzZYXd7nm~sQzusa0KB4XLyTdTbG1O5cl0AzYDD6Way1qtRp1O0BK76Okx-tsDCfKjr0R36g5QRvQ~aKszhqe4-BFOXs2wSryt1txUs8G9Jte6mxQ7aSlkGn-QzDcsc3IpCudpB419e9-NmI2tXyI4f4pzoaKSuO3T~CmE8EuC01zmYR6cbKlyDO01MK3ha5aZ0I3lhCihFtjQJLBTsLL5Q00Moh6E8vk2YambWTFF4jdg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4C</a></span>).</p><p>Overall, these results suggest that most proteolytic digestion in the yellowtail is related to the expression of specific genes, despite the large number of proteolytic digestive enzyme genes in the genome (<a reveal-id="dsy024-F5" data-open="dsy024-F5" class="link link-reveal link-table xref-fig">Fig. 5A</a>). The highly expressed proteolytic digestive enzyme genes identified in this study may play a key role in digestion in adult yellowtail. This finding suggests that the identification of proteins digestible by the products of these genes may lead to a yellowtail feed formulation with enhanced digestibility. Lemeiux et al.<a reveal-id="dsy024-B96" data-open="dsy024-B96" class="link link-ref link-reveal xref-bibr"><sup>96</sup></a> reported that among the activities of a number of enzymes, trypsin alone showed a significant correlation with food conversion efficiency and growth rate in the carnivorous Atlantic cod. Furthermore, it has been demonstrated that trypsin isoforms showed differing substrate preferences and distinct catalytic properties.<a reveal-id="dsy024-B100" data-open="dsy024-B100" class="link link-ref link-reveal xref-bibr"><sup>100</sup></a><sup>,</sup><a reveal-id="dsy024-B101" data-open="dsy024-B101" class="link link-ref link-reveal xref-bibr"><sup>101</sup></a> Therefore, further study of the substrate preferences of the products of the three highly expressed trypsinogen genes (g15220, g121 and g15219) may inform the makeup of yellowtail feed ingredients.</p><p>This study has focussed on adult yellowtail, and the gene expression patterns for digestive enzymes may be different for larval stages.<a reveal-id="dsy024-B71" data-open="dsy024-B71" class="link link-ref link-reveal xref-bibr"><sup>71</sup></a> Studies of gene expression patterns for digestive enzymes in larval stages of yellowtail will inform the development of formulated diets for larvae of this species. In addition, the digestive systems of fish show numerous structural and functional adaptations to their feeding habits. Thus, the transcriptomes of digestive tissues in fish are considered highly species-specific,<a reveal-id="dsy024-B58" data-open="dsy024-B58" class="link link-ref link-reveal xref-bibr"><sup>58</sup></a> suggesting that transcriptome profiling of these tissues may be instructive for the development of fish feed for species of interest in the aquaculture industry.</p>                            <h2 scrollto-destination=122671160 id="122671160" class="section-title" >Availability</h2>
<p>The draft genome sequence of <em>S. quinqueradiata</em> was submitted to DDBJ under accession no. BEWX01000001–BEWX01001394 (1394 entries). The complete set of predicted protein-coding genes are accessible at <a class="link link-uri" href="http://nrifs.fra.affrc.go.jp/ResearchCenter/5_BB/genomes/Yellowtail_genes/index.html" target="_blank">http://nrifs.fra.affrc.go.jp/ResearchCenter/5_BB/genomes/Yellowtail_genes/index.html</a>. The results of the functional annotation of the predicted protein-coding genes are available in <span class="content-section supplementary-material"><a path-from-xml="sup7" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy024/1/dsy024_supplementary_table_s7.xlsx?Expires=2147483647&Signature=B-UKJRuFk0CuOM1vqtJ8PJp4rsuY~rLtnkKLUuj02ZVm20F1H2JGCIpIvQW~EVHanNj3yexU-CI0nXMi7IezUHO0T474OIANzB47N9M6JThXj1II108-PCs2wVC3S3Jz9p6v6Qh7DhV1-Hp~Wlj1slC6NHfKCrDg5hp93hCzuKg0YyjY4zH7Zr5QLUtVoNaAH1Mcgmzo~yjA5DllrMCMvMFJlBYhjTACrKbpfzMKKJH8ybDnxahUwpTwGN175ncaVGdlVUexwOY5fNBHk6vajnotXmLvWcHmgaKZYVZsgB4vRIaMLPSGnulF70Sy4z-kwk~5whRHvp2qMaJ64JmNWg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S7</a></span>. Raw RNA-Seq datasets for the three digestive organs of yellowtail have been deposited at DDBJ Sequence Read Archive (DRA) under accession no. DRR107933–DRR107935 (stomach), DRR107936–DRR107938 (intestine) and DRR107939– DRR107941 (rectum). </p>                            <h2 scrollto-destination=122671162 id="122671162" class="backacknowledgements-title" >Acknowledgements</h2>
<p>This work was partly supported by an Adaptable and Seamless Technology transfer Programme through Target-driven R&amp;D (A-STEP: AS2815106U) from the Japan Science and Technology Agency. We thank Dr Shintaro Imamura (National Research Institute of Fisheries Science, Japan Fisheries Research and Education Agency) and Dr Susumu Uji (National Research Institute of Aquaculture, Japan Fisheries Research and Education Agency) for providing samples of larval yellowtail digestive organs and Dr Tsubasa Uchino (National Research Institute of Fisheries Science, Japan Fisheries Research and Education Agency) for his help in writing Section 2.2. Linkage map construction and scaffold correction. We also would like to thank Dr Kristian von Schalburg (Department of Molecular Biology and Biochemistry, S.F.U., Vancouver, BC, Canada) and Editage (<a class="link link-uri" href="http://www.editage.jp" target="_blank">www.editage.jp</a>) for English language editing.</p>                            <h3 scrollto-destination=122671164 id="122671164" class="section-title" >Conflict of interest</h3>
<p>None declared.</p>                            <h2 scrollto-destination=122671166 id="122671166" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsy024-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Swart</div> <div class="given-names">B.L.</div></div>, <div class="name string-name"><div class="surname">von der Heyden</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Bester-van der Merwe</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Roodt-Wilding</div> <div class="given-names">R.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Molecular systematics and biogeography of the circumglobally distributed genus <em>Seriola</em> (Pisces: carangidae)</div>, <div class="source">Mol. Phylogenet. Evol</div> ., <div class="volume">93</div>, <div class="fpage">274</div>–<div class="lpage">80</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20systematics%20and%20biogeography%20of%20the%20circumglobally%20distributed%20genus%20Seriola%20(Pisces%3A%20carangidae)&amp;author=B.L.%20Swart&amp;author=S.%20von%20der%20Heyden&amp;author=A.%20Bester-van%20der%20Merwe&amp;author=R.%20Roodt-Wilding&amp;publication_year=2015&amp;journal=Mol.%20Phylogenet.%20Evol&amp;volume=93&amp;pages=274-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ympev.2015.08.002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ympev.2015.08.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ympev.2015.08.002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26279346" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy024-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sicuro</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Luzzana</div> <div class="given-names">U.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">The State of Seriola spp. Other Than Yellowtail (<em>S. quinqueradiata</em>) Farming in the World</div>, <div class="source">Rev. Fish Sci. Aquac</div> ., <div class="volume">24</div>, <div class="fpage">314</div>–<div class="lpage">25</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20State%20of%20Seriola%20spp.%20Other%20Than%20Yellowtail%20(S.%20quinqueradiata)%20Farming%20in%20the%20World&amp;author=B.%20Sicuro&amp;author=U.%20Luzzana&amp;publication_year=2016&amp;journal=Rev.%20Fish%20Sci.%20Aquac&amp;volume=24&amp;pages=314-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1080/23308249.2016.1187583" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F23308249.2016.1187583" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1080%2F23308249.2016.1187583"> </span></div> </div></div></div></div></div><div content-id="dsy024-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="collab">Ministry of Agriculture, Forestry and Fisheries of Japan</div>. <div class="year">2017</div>, <em>Annual Statistics of Fishery and Aquaculture Production</em>. Statistics Department. <a class="link link-uri" href="http://www.maff.go.jp/j/tokei/kouhyou/kaimen_gyosei/attach/pdf/index-7.pdf" target="_blank">http://www.maff.go.jp/j/tokei/kouhyou/kaimen_gyosei/attach/pdf/index-7.pdf</a> (in Japanese).</div></div></div></div><div content-id="dsy024-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Dhirendra</div> <div class="given-names">P.T.</div></div></div> <div class="year">2005</div>, Cultured Aquatic Species Information Programme. Seriola quinqueradiata. In: <div class="source">Cultured Aquatic Species Information Programme</div> . 
<div class="publisher-loc">Rome</div>: 
<div class="publisher-name">FAO Fisheries and Aquaculture Department</div>, pp. <div class="fpage">1</div>–<div class="lpage">11</div>.</div></div></div></div><div content-id="dsy024-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="collab">World Bank</div>. <div class="year">2013</div>, <div class="source">Fish to 2030: Prospects for Fisheries and Aquaculture. Agriculture and Environmental Services Discussion Paper, No. 3</div> . 
<div class="publisher-loc">Washington DC</div>: 
<div class="publisher-name">World Bank Group</div>, pp. <div class="fpage">1</div>–<div class="lpage">102</div>.</div></div></div></div><div content-id="dsy024-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Abdelrahman</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">ElHady</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Alcivar-Warren</div> <div class="given-names">A.</div></div></div>, et al.   <div class="year">2017</div>, 
<div class="article-title">Aquaculture genomics, genetics and breeding in the United States: current status, challenges, and priorities for future research</div>, <div class="source">BMC Genomics</div> , <div class="volume">18</div>, <div class="fpage">191</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Aquaculture%20genomics%2C%20genetics%20and%20breeding%20in%20the%20United%20States%3A%20current%20status%2C%20challenges%2C%20and%20priorities%20for%20future%20research&amp;author=H.%20Abdelrahman&amp;author=M.%20ElHady&amp;author=A.%20Alcivar-Warren&amp;publication_year=2017&amp;journal=BMC%20Genomics&amp;volume=18&amp;pages=191" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12864-017-3557-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12864-017-3557-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12864-017-3557-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28219347" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy024-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Macqueen</div> <div class="given-names">D.J.</div></div>, <div class="name string-name"><div class="surname">Primmer</div> <div class="given-names">C.R.</div></div>, <div class="name string-name"><div class="surname">Houston</div> <div class="given-names">R.D.</div></div></div>, et al.   <div class="year">2017</div>, 
<div class="article-title">Functional Annotation of All Salmonid Genomes (FAASG): an international initiative supporting future salmonid research, conservation and aquaculture</div>, <div class="source">BMC Genomics</div> , <div class="volume">18</div>, <div class="fpage">484</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Functional%20Annotation%20of%20All%20Salmonid%20Genomes%20(FAASG)%3A%20an%20international%20initiative%20supporting%20future%20salmonid%20research%2C%20conservation%20and%20aquaculture&amp;author=D.J.%20Macqueen&amp;author=C.R.%20Primmer&amp;author=R.D.%20Houston&amp;publication_year=2017&amp;journal=BMC%20Genomics&amp;volume=18&amp;pages=484" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12864-017-3862-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12864-017-3862-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12864-017-3862-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28655320" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy024-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yue</div> <div class="given-names">G.H.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">L.</div></div></div> <div class="year">2017</div>, 
<div class="article-title">Current status of genome sequencing and its applications in aquaculture</div>, <div class="source">Aquaculture</div> , <div class="volume">468</div>, <div class="fpage">337</div>–<div class="lpage">47</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Current%20status%20of%20genome%20sequencing%20and%20its%20applications%20in%20aquaculture&amp;author=G.H.%20Yue&amp;author=L.%20Wang&amp;publication_year=2017&amp;journal=Aquaculture&amp;volume=468&amp;pages=337-47" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.aquaculture.2016.10.036" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.aquaculture.2016.10.036" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.aquaculture.2016.10.036"> </span></div> </div></div></div></div></div><div content-id="dsy024-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Aoki</div> <div class="given-names">J.Y.</div></div>, <div class="name string-name"><div class="surname">Kai</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Kawabata</div> <div class="given-names">Y.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Second generation physical and linkage maps of yellowtail (<em>Seriola quinqueradiata</em>) and comparison of synteny with four model fish</div>, <div class="source">BMC Genomics</div> . , <div class="volume">16</div>, <div class="fpage">406</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Second%20generation%20physical%20and%20linkage%20maps%20of%20yellowtail%20(Seriola%20quinqueradiata)%20and%20comparison%20of%20synteny%20with%20four%20model%20fish&amp;author=J.Y.%20Aoki&amp;author=W.%20Kai&amp;author=Y.%20Kawabata&amp;publication_year=2015&amp;journal=BMC%20Genomics&amp;volume=16&amp;pages=406" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12864-015-1600-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12864-015-1600-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12864-015-1600-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26003112" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy024-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Fuji</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Koyama</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Kai</div> <div class="given-names">W.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Construction of a high-coverage bacterial artificial chromosome library and comprehensive genetic linkage map of yellowtail <em>Seriola quinqueradiata</em></div>, <div class="source">BMC Res. Notes</div> , <div class="volume">7</div>, <div class="fpage">200</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20a%20high-coverage%20bacterial%20artificial%20chromosome%20library%20and%20comprehensive%20genetic%20linkage%20map%20of%20yellowtail%20Seriola%20quinqueradiata&amp;author=K.%20Fuji&amp;author=T.%20Koyama&amp;author=W.%20Kai&amp;publication_year=2014&amp;journal=BMC%20Res.%20Notes&amp;volume=7&amp;pages=200" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1756-0500-7-200" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1756-0500-7-200" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1756-0500-7-200"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24684753" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy024-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ohara</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Nishimura</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Nagakura</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Sakamoto</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Mushiake</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Okamoto</div> <div class="given-names">N.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">Genetic linkage maps of two yellowtails (<em>Seriola quinqueradiata</em> and <em>Seriola lalandi</em>)</div>, <div class="source">Aquaculture</div> , <div class="volume">244</div>, <div class="fpage">41</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20linkage%20maps%20of%20two%20yellowtails%20(Seriola%20quinqueradiata%20and%20Seriola%20lalandi)&amp;author=E.%20Ohara&amp;author=T.%20Nishimura&amp;author=Y.%20Nagakura&amp;author=T.%20Sakamoto&amp;author=K.%20Mushiake&amp;author=N.%20Okamoto&amp;publication_year=2005&amp;journal=Aquaculture&amp;volume=244&amp;pages=41-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.aquaculture.2004.10.022" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.aquaculture.2004.10.022" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.aquaculture.2004.10.022"> </span></div> </div></div></div></div></div><div content-id="dsy024-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Aoki</div> <div class="given-names">J.Y.</div></div>, <div class="name string-name"><div class="surname">Kai</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Kawabata</div> <div class="given-names">Y.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Construction of a radiation hybrid panel and the first yellowtail (<em>Seriola quinqueradiata</em>) radiation hybrid map using a nanofluidic dynamic array</div>, <div class="source">BMC Genomics</div> , <div class="volume">15</div>, <div class="fpage">165</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20a%20radiation%20hybrid%20panel%20and%20the%20first%20yellowtail%20(Seriola%20quinqueradiata)%20radiation%20hybrid%20map%20using%20a%20nanofluidic%20dynamic%20array&amp;author=J.Y.%20Aoki&amp;author=W.%20Kai&amp;author=Y.%20Kawabata&amp;publication_year=2014&amp;journal=BMC%20Genomics&amp;volume=15&amp;pages=165" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-15-165" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-15-165" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-15-165"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24571093" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy024-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ozaki</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Yoshida</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Fuji</div> <div class="given-names">K.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">Quantitative trait loci (QTL) associated with resistance to a monogenean parasite (<em>Benedenia seriolae</em>) in yellowtail (<em>Seriola quinqueradiata</em>) through genome wide analysis</div>, <div class="source">PloS One</div> , <div class="volume">8</div>, <div class="fpage">e64987</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Quantitative%20trait%20loci%20(QTL)%20associated%20with%20resistance%20to%20a%20monogenean%20parasite%20(Benedenia%20seriolae)%20in%20yellowtail%20(Seriola%20quinqueradiata)%20through%20genome%20wide%20analysis&amp;author=A.%20Ozaki&amp;author=K.%20Yoshida&amp;author=K.%20Fuji&amp;publication_year=2013&amp;journal=PloS%20One&amp;volume=8&amp;pages=e64987" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0064987" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0064987" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0064987"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23750223" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy024-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Fuji</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Yoshida</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Hattori</div> <div class="given-names">K.</div></div></div>, et al.   <div class="year">2010</div>, 
<div class="article-title">Identification of the sex-linked locus in yellowtail, <em>Seriola quinqueradiata</em></div>, <div class="source">Aquaculture</div> , <div class="volume">308</div>, <div class="fpage">S51</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20the%20sex-linked%20locus%20in%20yellowtail%2C%20Seriola%20quinqueradiata&amp;author=K.%20Fuji&amp;author=K.%20Yoshida&amp;author=K.%20Hattori&amp;publication_year=2010&amp;journal=Aquaculture&amp;volume=308&amp;pages=S51-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.aquaculture.2010.06.035" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.aquaculture.2010.06.035" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.aquaculture.2010.06.035"> </span></div> </div></div></div></div></div><div content-id="dsy024-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Koyama</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Ozaki</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Yoshida</div> <div class="given-names">K.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Identification of sex-linked snps and sex-determining regions in the yellowtail genome</div>, <div class="source">Mar. Biotechnol.</div>  , <div class="volume">17</div>, <div class="fpage">502</div>–<div class="lpage">10</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20sex-linked%20snps%20and%20sex-determining%20regions%20in%20the%20yellowtail%20genome&amp;author=T.%20Koyama&amp;author=A.%20Ozaki&amp;author=K.%20Yoshida&amp;publication_year=2015&amp;journal=Mar.%20Biotechnol.&amp;volume=17&amp;pages=502-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10126-015-9636-x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10126-015-9636-x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10126-015-9636-x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25975833" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy024-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Iwasaki</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Nishiki</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Nakamura</div> <div class="given-names">Y.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">Effective de novo assembly of fish genome using haploid larvae</div>, <div class="source">Gene</div> , <div class="volume">576</div>, <div class="fpage">644</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Effective%20de%20novo%20assembly%20of%20fish%20genome%20using%20haploid%20larvae&amp;author=Y.%20Iwasaki&amp;author=I.%20Nishiki&amp;author=Y.%20Nakamura&amp;publication_year=2016&amp;journal=Gene&amp;volume=576&amp;pages=644-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gene.2015.10.015" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gene.2015.10.015" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gene.2015.10.015"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26478467" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy024-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">H.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Aligning sequence reads, clone sequences and assembly contigs with BWA-MEM</div>, <div class="source">arXiv Preprint, arXiv</div> , <div class="volume">1303</div>, <div class="fpage">3997</div>.</div></div></div></div><div content-id="dsy024-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kajitani</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Toshimoto</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Noguchi</div> <div class="given-names">H.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Efficient <em>de novo</em> assembly of highly heterozygous genomes from whole-genome shotgun short reads</div>, <div class="source">Genome Res</div> ., <div class="volume">24</div>, <div class="fpage">1384</div>–<div class="lpage">95</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Efficient%20de%20novo%20assembly%20of%20highly%20heterozygous%20genomes%20from%20whole-genome%20shotgun%20short%20reads&amp;author=R.%20Kajitani&amp;author=K.%20Toshimoto&amp;author=H.%20Noguchi&amp;publication_year=2014&amp;journal=Genome%20Res&amp;volume=24&amp;pages=1384-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.170720.113" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.170720.113" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.170720.113"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24755901" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy024-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Altschul</div> <div class="given-names">S.F.</div></div>, <div class="name string-name"><div class="surname">Gish</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Miller</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Myers</div> <div class="given-names">E.W.</div></div>, <div class="name string-name"><div class="surname">Lipman</div> <div class="given-names">D.J.</div></div></div> <div class="year">1990</div>, 
<div class="article-title">Basic local alignment search tool</div>, <div class="source">J. Mol. Biol</div> ., <div class="volume">215</div>, <div class="fpage">403</div>–<div class="lpage">10</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Basic%20local%20alignment%20search%20tool&amp;author=S.F.%20Altschul&amp;author=W.%20Gish&amp;author=W.%20Miller&amp;author=E.W.%20Myers&amp;author=D.J.%20Lipman&amp;publication_year=1990&amp;journal=J.%20Mol.%20Biol&amp;volume=215&amp;pages=403-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0022-2836(05)80360-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0022-2836(05)80360-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0022-2836(05)80360-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2231712" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy024-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kai</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Nomura</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Fujiwara</div> <div class="given-names">A.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">A ddRAD-based genetic map and its integration with the genome assembly of Japanese eel (<em>Anguilla japonica</em>) provides insights into genome evolution after the teleost-specific genome duplication</div>, <div class="source">BMC Genomics</div> , <div class="volume">15</div>, <div class="fpage">233</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20ddRAD-based%20genetic%20map%20and%20its%20integration%20with%20the%20genome%20assembly%20of%20Japanese%20eel%20(Anguilla%20japonica)%20provides%20insights%20into%20genome%20evolution%20after%20the%20teleost-specific%20genome%20duplication&amp;author=W.%20Kai&amp;author=K.%20Nomura&amp;author=A.%20Fujiwara&amp;publication_year=2014&amp;journal=BMC%20Genomics&amp;volume=15&amp;pages=233" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-15-233" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-15-233" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-15-233"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24669946" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy024-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Xue</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">J.T.</div></div>, <div class="name string-name"><div class="surname">Zhu</div> <div class="given-names">Y.P.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">L_RNA_scaffolder: scaffolding genomes with transcripts</div>, <div class="source">BMC Genomics</div> ., <div class="volume">14</div>, <div class="fpage">604</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=L_RNA_scaffolder%3A%20scaffolding%20genomes%20with%20transcripts&amp;author=W.%20Xue&amp;author=J.T.%20Li&amp;author=Y.P.%20Zhu&amp;publication_year=2013&amp;journal=BMC%20Genomics&amp;volume=14&amp;pages=604" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-14-604" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-14-604" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-14-604"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24010822" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy024-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Marcais</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Kingsford</div> <div class="given-names">C.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">A fast, lock-free approach for efficient parallel counting of occurrences of k-mers</div>, <div class="source">Bioinformatics</div> , <div class="volume">27</div>, <div class="fpage">764</div>–<div class="lpage">70</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20fast%2C%20lock-free%20approach%20for%20efficient%20parallel%20counting%20of%20occurrences%20of%20k-mers&amp;author=G.%20Marcais&amp;author=C.%20Kingsford&amp;publication_year=2011&amp;journal=Bioinformatics&amp;volume=27&amp;pages=764-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btr011" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtr011" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtr011"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21217122" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy024-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Catchen</div> <div class="given-names">J.M.</div></div>, <div class="name string-name"><div class="surname">Amores</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Hohenlohe</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Cresko</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Postlethwait</div> <div class="given-names">J.H.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Stacks: building and genotyping Loci de novo from short-read sequences</div>, <div class="source">G3 (Bethesda)</div> , <div class="volume">1</div>, <div class="fpage">171</div>–<div class="lpage">82</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Stacks%3A%20building%20and%20genotyping%20Loci%20de%20novo%20from%20short-read%20sequences&amp;author=J.M.%20Catchen&amp;author=A.%20Amores&amp;author=P.%20Hohenlohe&amp;author=W.%20Cresko&amp;author=J.H.%20Postlethwait&amp;publication_year=2011&amp;journal=G3%20(Bethesda)&amp;volume=1&amp;pages=171-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/g3.111.000240" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fg3.111.000240" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fg3.111.000240"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22384329" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy024-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">DePristo</div> <div class="given-names">M.A.</div></div>, <div class="name string-name"><div class="surname">Banks</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Poplin</div> <div class="given-names">R.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">A framework for variation discovery and genotyping using next-generation DNA sequencing data</div>, <div class="source">Nat. Genet</div> ., <div class="volume">43</div>, <div class="fpage">491</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20framework%20for%20variation%20discovery%20and%20genotyping%20using%20next-generation%20DNA%20sequencing%20data&amp;author=M.A.%20DePristo&amp;author=E.%20Banks&amp;author=R.%20Poplin&amp;publication_year=2011&amp;journal=Nat.%20Genet&amp;volume=43&amp;pages=491-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.806" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.806" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.806"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21478889" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy024-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Broman</div> <div class="given-names">K.W.</div></div>, <div class="name string-name"><div class="surname">Wu</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Sen</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Churchill</div> <div class="given-names">G.A.</div></div></div> <div class="year">2003</div>, 
<div class="article-title">R/qtl: qTL mapping in experimental crosses</div>, <div class="source">Bioinformatics</div> , <div class="volume">19</div>, <div class="fpage">889</div>–<div class="lpage">90</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=R%2Fqtl%3A%20qTL%20mapping%20in%20experimental%20crosses&amp;author=K.W.%20Broman&amp;author=H.%20Wu&amp;author=S.%20Sen&amp;author=G.A.%20Churchill&amp;publication_year=2003&amp;journal=Bioinformatics&amp;volume=19&amp;pages=889-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btg112" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtg112" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtg112"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12724300" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy024-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Nishimura</div> <div class="given-names">O.</div></div>, <div class="name string-name"><div class="surname">Hara</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Kuraku</div> <div class="given-names">S.</div></div></div> <div class="year">2017</div>, 
<div class="article-title">gVolante for standardizing completeness assessment of genome and transcriptome assemblies</div>, <div class="source">Bioinformatics</div> , <div class="volume">33</div>, <div class="fpage">3635</div>–<div class="lpage">37</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=gVolante%20for%20standardizing%20completeness%20assessment%20of%20genome%20and%20transcriptome%20assemblies&amp;author=O.%20Nishimura&amp;author=Y.%20Hara&amp;author=S.%20Kuraku&amp;publication_year=2017&amp;journal=Bioinformatics&amp;volume=33&amp;pages=3635-37" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btx445" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtx445" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtx445"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29036533" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy024-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Parra</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Bradnam</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Ning</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Keane</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Korf</div> <div class="given-names">I.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Assessing the gene space in draft genomes</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">37</div>, <div class="fpage">289</div>–<div class="lpage">97</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Assessing%20the%20gene%20space%20in%20draft%20genomes&amp;author=G.%20Parra&amp;author=K.%20Bradnam&amp;author=Z.%20Ning&amp;author=T.%20Keane&amp;author=I.%20Korf&amp;publication_year=2009&amp;journal=Nucleic%20Acids%20Res&amp;volume=37&amp;pages=289-97" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkn916" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkn916" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkn916"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19042974" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy024-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hara</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Tatsumi</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Yoshida</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Kajikawa</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Kiyonari</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Kuraku</div> <div class="given-names">S.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Optimizing and benchmarking <em>de novo</em> transcriptome sequencing: from library preparation to assembly evaluation</div>, <div class="source">BMC Genomics</div> , <div class="volume">16</div>, <div class="fpage">977</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Optimizing%20and%20benchmarking%20de%20novo%20transcriptome%20sequencing%3A%20from%20library%20preparation%20to%20assembly%20evaluation&amp;author=Y.%20Hara&amp;author=K.%20Tatsumi&amp;author=M.%20Yoshida&amp;author=E.%20Kajikawa&amp;author=H.%20Kiyonari&amp;author=S.%20Kuraku&amp;publication_year=2015&amp;journal=BMC%20Genomics&amp;volume=16&amp;pages=977" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12864-015-2007-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12864-015-2007-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12864-015-2007-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26581708" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy024-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Simao</div> <div class="given-names">F.A.</div></div>, <div class="name string-name"><div class="surname">Waterhouse</div> <div class="given-names">R.M.</div></div>, <div class="name string-name"><div class="surname">Ioannidis</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Kriventseva</div> <div class="given-names">E.V.</div></div>, <div class="name string-name"><div class="surname">Zdobnov</div> <div class="given-names">E.M.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs</div>, <div class="source">Bioinformatics</div> , <div class="volume">31</div>, <div class="fpage">3210</div>–<div class="lpage">2</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=BUSCO%3A%20assessing%20genome%20assembly%20and%20annotation%20completeness%20with%20single-copy%20orthologs&amp;author=F.A.%20Simao&amp;author=R.M.%20Waterhouse&amp;author=P.%20Ioannidis&amp;author=E.V.%20Kriventseva&amp;author=E.M.%20Zdobnov&amp;publication_year=2015&amp;journal=Bioinformatics&amp;volume=31&amp;pages=3210-2" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btv351" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtv351" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtv351"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26059717" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy024-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Meyer</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Aglyamova</div> <div class="given-names">G.V.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">S.</div></div></div>, et al.   <div class="year">2009</div>, 
<div class="article-title">Sequencing and <em>de novo</em> analysis of a coral larval transcriptome using 454 GSFlx</div>, <div class="source">BMC Genomics</div> ., <div class="volume">10</div>, <div class="fpage">219</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequencing%20and%20de%20novo%20analysis%20of%20a%20coral%20larval%20transcriptome%20using%20454%20GSFlx&amp;author=E.%20Meyer&amp;author=G.V.%20Aglyamova&amp;author=S.%20Wang&amp;publication_year=2009&amp;journal=BMC%20Genomics&amp;volume=10&amp;pages=219" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-10-219" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-10-219" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-10-219"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19435504" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy024-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Stanke</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Diekhans</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Baertsch</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Haussler</div> <div class="given-names">D.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">Using native and syntenically mapped cDNA alignments to improve <em>de novo</em> gene finding</div>, <div class="source">Bioinformatics</div>  , <div class="volume">24</div>, <div class="fpage">637</div>–<div class="lpage">44</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Using%20native%20and%20syntenically%20mapped%20cDNA%20alignments%20to%20improve%20de%20novo%20gene%20finding&amp;author=M.%20Stanke&amp;author=M.%20Diekhans&amp;author=R.%20Baertsch&amp;author=D.%20Haussler&amp;publication_year=2008&amp;journal=Bioinformatics&amp;volume=24&amp;pages=637-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btn013" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtn013" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtn013"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18218656" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy024-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Smit</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Hubley</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Green</div> <div class="given-names">P.</div></div></div> <div class="year">1996–2010</div>, RepeatMasker Open-3.0. <a class="link link-uri" href="http://www.repeatmasker.org" target="_blank">http://www.repeatmasker.org</a>.</div></div></div></div><div content-id="dsy024-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Smit</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Hubley</div> <div class="given-names">R.</div></div></div> <div class="year">2008–2010</div>, RepeatModeler Open-1.0. <a class="link link-uri" href="http://www.repeatmasker.org" target="_blank">http://www.repeatmasker.org</a>.</div></div></div></div><div content-id="dsy024-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kim</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Pertea</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Trapnell</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Pimentel</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Kelley</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Salzberg</div> <div class="given-names">S.L.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">TopHat2: accurate alignment of transcriptomes in the presence of insertions, deletions and gene fusions</div>, <div class="source">Genome Biol</div> ., <div class="volume">14</div>, <div class="fpage">R36</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=TopHat2%3A%20accurate%20alignment%20of%20transcriptomes%20in%20the%20presence%20of%20insertions%2C%20deletions%20and%20gene%20fusions&amp;author=D.%20Kim&amp;author=G.%20Pertea&amp;author=C.%20Trapnell&amp;author=H.%20Pimentel&amp;author=R.%20Kelley&amp;author=S.L.%20Salzberg&amp;publication_year=2013&amp;journal=Genome%20Biol&amp;volume=14&amp;pages=R36" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2013-14-4-r36" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2013-14-4-r36" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2013-14-4-r36"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23618408" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy024-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Trapnell</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Williams</div> <div class="given-names">B.A.</div></div>, <div class="name string-name"><div class="surname">Pertea</div> <div class="given-names">G.</div></div></div>, et al.   <div class="year">2010</div>, 
<div class="article-title">Transcript assembly and quantification by RNA-Seq reveals unannotated transcripts and isoform switching during cell differentiation</div>, <div class="source">Nat. Biotechnol</div> ., <div class="volume">28</div>, <div class="fpage">511</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcript%20assembly%20and%20quantification%20by%20RNA-Seq%20reveals%20unannotated%20transcripts%20and%20isoform%20switching%20during%20cell%20differentiation&amp;author=C.%20Trapnell&amp;author=B.A.%20Williams&amp;author=G.%20Pertea&amp;publication_year=2010&amp;journal=Nat.%20Biotechnol&amp;volume=28&amp;pages=511-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.1621" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.1621" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.1621"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20436464" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy024-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yates</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Akanni</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Amode</div> <div class="given-names">M.R.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">Ensembl 2016</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">44</div>, <div class="fpage">D710</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ensembl%202016&amp;author=A.%20Yates&amp;author=W.%20Akanni&amp;author=M.R.%20Amode&amp;publication_year=2016&amp;journal=Nucleic%20Acids%20Res&amp;volume=44&amp;pages=D710-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkv1157" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkv1157" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkv1157"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26687719" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy024-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Altschul</div> <div class="given-names">S.F.</div></div>, <div class="name string-name"><div class="surname">Madden</div> <div class="given-names">T.L.</div></div>, <div class="name string-name"><div class="surname">Schaffer</div> <div class="given-names">A.A.</div></div></div>, et al.   <div class="year">1997</div>, 
<div class="article-title">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">25</div>, <div class="fpage">3389</div>–<div class="lpage">402</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gapped%20BLAST%20and%20PSI-BLAST%3A%20a%20new%20generation%20of%20protein%20database%20search%20programs&amp;author=S.F.%20Altschul&amp;author=T.L.%20Madden&amp;author=A.A.%20Schaffer&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res&amp;volume=25&amp;pages=3389-402" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.17.3389" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.17.3389" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.17.3389"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9254694" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy024-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Jones</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Binns</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Chang</div> <div class="given-names">H.Y.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">InterProScan 5: genome-scale protein function classification</div>, <div class="source">Bioinformatics</div> , <div class="volume">30</div>, <div class="fpage">1236</div>–<div class="lpage">40</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=InterProScan%205%3A%20genome-scale%20protein%20function%20classification&amp;author=P.%20Jones&amp;author=D.%20Binns&amp;author=H.Y.%20Chang&amp;publication_year=2014&amp;journal=Bioinformatics&amp;volume=30&amp;pages=1236-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btu031" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtu031" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtu031"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24451626" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy024-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gotz</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Garcia-Gomez</div> <div class="given-names">J.M.</div></div>, <div class="name string-name"><div class="surname">Terol</div> <div class="given-names">J.</div></div></div>, et al.   <div class="year">2008</div>, 
<div class="article-title">High-throughput functional annotation and data mining with the Blast2GO suite</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">36</div>, <div class="fpage">3420</div>–<div class="lpage">35</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-throughput%20functional%20annotation%20and%20data%20mining%20with%20the%20Blast2GO%20suite&amp;author=S.%20Gotz&amp;author=J.M.%20Garcia-Gomez&amp;author=J.%20Terol&amp;publication_year=2008&amp;journal=Nucleic%20Acids%20Res&amp;volume=36&amp;pages=3420-35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkn176" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkn176" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkn176"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18445632" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy024-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Moriya</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Itoh</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Okuda</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Yoshizawa</div> <div class="given-names">A.C.</div></div>, <div class="name string-name"><div class="surname">Kanehisa</div> <div class="given-names">M.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">KAAS: an automatic genome annotation and pathway reconstruction server</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">35</div>, <div class="fpage">W182</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=KAAS%3A%20an%20automatic%20genome%20annotation%20and%20pathway%20reconstruction%20server&amp;author=Y.%20Moriya&amp;author=M.%20Itoh&amp;author=S.%20Okuda&amp;author=A.C.%20Yoshizawa&amp;author=M.%20Kanehisa&amp;publication_year=2007&amp;journal=Nucleic%20Acids%20Res&amp;volume=35&amp;pages=W182-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkm321" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkm321" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkm321"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17526522" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy024-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Nakamura</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Mori</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Saitoh</div> <div class="given-names">K.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">Evolutionary changes of multiple visual pigment genes in the complete genome of Pacific bluefin tuna</div>, <div class="source">Proc. Natl. Acad. Sci. U. S. A</div> ., <div class="volume">110</div>, <div class="fpage">11061</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolutionary%20changes%20of%20multiple%20visual%20pigment%20genes%20in%20the%20complete%20genome%20of%20Pacific%20bluefin%20tuna&amp;author=Y.%20Nakamura&amp;author=K.%20Mori&amp;author=K.%20Saitoh&amp;publication_year=2013&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A&amp;volume=110&amp;pages=11061-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1302051110" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1302051110" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1302051110"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23781100" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy024-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Xu</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Xu</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Che</div> <div class="given-names">R.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">The genome of the miiuy croaker reveals well-developed innate immune and sensory systems</div>, <div class="source">Sci. Rep</div> ., <div class="volume">6</div>, <div class="fpage">21902</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20of%20the%20miiuy%20croaker%20reveals%20well-developed%20innate%20immune%20and%20sensory%20systems&amp;author=T.%20Xu&amp;author=G.%20Xu&amp;author=R.%20Che&amp;publication_year=2016&amp;journal=Sci.%20Rep&amp;volume=6&amp;pages=21902" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/srep21902" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsrep21902" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsrep21902"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26902509" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy024-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wang</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Coleman-Derr</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Gu</div> <div class="given-names">Y.Q.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">OrthoVenn: a web server for genome wide comparison and annotation of orthologous clusters across multiple species</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">43</div>, <div class="fpage">W78</div>–<div class="lpage">84</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=OrthoVenn%3A%20a%20web%20server%20for%20genome%20wide%20comparison%20and%20annotation%20of%20orthologous%20clusters%20across%20multiple%20species&amp;author=Y.%20Wang&amp;author=D.%20Coleman-Derr&amp;author=G.%20Chen&amp;author=Y.Q.%20Gu&amp;publication_year=2015&amp;journal=Nucleic%20Acids%20Res&amp;volume=43&amp;pages=W78-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkv487" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkv487" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkv487"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25964301" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy024-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wagner</div> <div class="given-names">G.P.</div></div>, <div class="name string-name"><div class="surname">Kin</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Lynch</div> <div class="given-names">V.J.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Measurement of mRNA abundance using RNA-seq data: rPKM measure is inconsistent among samples</div>, <div class="source">Theory Biosci</div> ., <div class="volume">131</div>, <div class="fpage">281</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Measurement%20of%20mRNA%20abundance%20using%20RNA-seq%20data%3A%20rPKM%20measure%20is%20inconsistent%20among%20samples&amp;author=G.P.%20Wagner&amp;author=K.%20Kin&amp;author=V.J.%20Lynch&amp;publication_year=2012&amp;journal=Theory%20Biosci&amp;volume=131&amp;pages=281-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s12064-012-0162-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs12064-012-0162-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs12064-012-0162-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22872506" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy024-B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Mi</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Muruganujan</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Casagrande</div> <div class="given-names">J.T.</div></div>, <div class="name string-name"><div class="surname">Thomas</div> <div class="given-names">P.D.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Large-scale gene function analysis with the PANTHER classification system</div>, <div class="source">Nat. Protoc</div> ., <div class="volume">8</div>, <div class="fpage">1551</div>–<div class="lpage">66</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Large-scale%20gene%20function%20analysis%20with%20the%20PANTHER%20classification%20system&amp;author=H.%20Mi&amp;author=A.%20Muruganujan&amp;author=J.T.%20Casagrande&amp;author=P.D.%20Thomas&amp;publication_year=2013&amp;journal=Nat.%20Protoc&amp;volume=8&amp;pages=1551-66" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nprot.2013.092" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnprot.2013.092" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnprot.2013.092"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23868073" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy024-B46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Tan</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Suzuki</div> <div class="given-names">Y.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Dramatic improvement in genome assembly achieved using doubled-haploid genomes</div>, <div class="source">Sci. Rep</div> ., <div class="volume">4</div>, <div class="fpage">6780</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dramatic%20improvement%20in%20genome%20assembly%20achieved%20using%20doubled-haploid%20genomes&amp;author=H.%20Zhang&amp;author=E.%20Tan&amp;author=Y.%20Suzuki&amp;publication_year=2015&amp;journal=Sci.%20Rep&amp;volume=4&amp;pages=6780" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/srep06780" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsrep06780" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsrep06780"> </span></div> </div></div></div></div></div><div content-id="dsy024-B47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lu</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Peatman</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Tang</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Lewis</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">Z.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Profiling of gene duplication patterns of sequenced teleost genomes: evidence for rapid lineage-specific genome expansion mediated by recent tandem duplications</div>, <div class="source">BMC Genomics</div> , <div class="volume">13</div>, <div class="fpage">246</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Profiling%20of%20gene%20duplication%20patterns%20of%20sequenced%20teleost%20genomes%3A%20evidence%20for%20rapid%20lineage-specific%20genome%20expansion%20mediated%20by%20recent%20tandem%20duplications&amp;author=J.%20Lu&amp;author=E.%20Peatman&amp;author=H.%20Tang&amp;author=J.%20Lewis&amp;author=Z.%20Liu&amp;publication_year=2012&amp;journal=BMC%20Genomics&amp;volume=13&amp;pages=246" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-13-246" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-13-246" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-13-246"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22702965" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy024-B48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Pinto</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Ronnestad</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Jordal</div> <div class="given-names">A.E.</div></div>, <div class="name string-name"><div class="surname">Gomes</div> <div class="given-names">A.S.</div></div>, <div class="name string-name"><div class="surname">Dinis</div> <div class="given-names">M.T.</div></div>, <div class="name string-name"><div class="surname">Aragao</div> <div class="given-names">C.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Cloning, tissue and ontogenetic expression of the taurine transporter in the flatfish Senegalese sole (<em>Solea senegalensis</em>)</div>, <div class="source">Amino Acids</div> , <div class="volume">42</div>, <div class="fpage">1317</div>–<div class="lpage">27</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cloning%2C%20tissue%20and%20ontogenetic%20expression%20of%20the%20taurine%20transporter%20in%20the%20flatfish%20Senegalese%20sole%20(Solea%20senegalensis)&amp;author=W.%20Pinto&amp;author=I.%20Ronnestad&amp;author=A.E.%20Jordal&amp;author=A.S.%20Gomes&amp;author=M.T.%20Dinis&amp;author=C.%20Aragao&amp;publication_year=2012&amp;journal=Amino%20Acids&amp;volume=42&amp;pages=1317-27" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00726-010-0826-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00726-010-0826-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00726-010-0826-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21210163" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy024-B49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Takeuchi</div> <div class="given-names">T.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Progress on larval and juvenile nutrition to improve the quality and health of seawater fish: a review</div>, <div class="source">Fish. Sci</div> ., <div class="volume">80</div>, <div class="fpage">389</div>–<div class="lpage">403</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Progress%20on%20larval%20and%20juvenile%20nutrition%20to%20improve%20the%20quality%20and%20health%20of%20seawater%20fish%3A%20a%20review&amp;author=T.%20Takeuchi&amp;publication_year=2014&amp;journal=Fish.%20Sci&amp;volume=80&amp;pages=389-403" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s12562-014-0744-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs12562-014-0744-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs12562-014-0744-8"> </span></div> </div></div></div></div></div><div content-id="dsy024-B50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Takagi</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Murata</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Goto</div> <div class="given-names">T.</div></div></div>, et al.   <div class="year">2005</div>, 
<div class="article-title">The green liver syndrome is caused by taurine deficiency in yellowtail, <em>Seriola quinqueradiata</em> Fed diets without fishmeal</div>, <div class="source">Aquacult. Sci</div> ., <div class="volume">53</div>, <div class="fpage">279</div>–<div class="lpage">90</div>.</div></div></div></div><div content-id="dsy024-B51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sarropoulou</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Sundaram</div> <div class="given-names">A.Y.M.</div></div>, <div class="name string-name"><div class="surname">Kaitetzidou</div> <div class="given-names">E.</div></div></div>, et al.   <div class="year">2017</div>, 
<div class="article-title">Full genome survey and dynamics of gene expression in the greater amberjack <em>Seriola dumerili</em></div>, <div class="source">GigaScience</div> , <div class="volume">6</div>, <div class="fpage">1</div>–<div class="lpage">13</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Full%20genome%20survey%20and%20dynamics%20of%20gene%20expression%20in%20the%20greater%20amberjack%20Seriola%20dumerili&amp;author=E.%20Sarropoulou&amp;author=A.Y.M.%20Sundaram&amp;author=E.%20Kaitetzidou&amp;publication_year=2017&amp;journal=GigaScience&amp;volume=6&amp;pages=1-13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/gigascience/gix108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fgigascience%2Fgix108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fgigascience%2Fgix108"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29126158" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy024-B52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B52" class="ref-content "><div class="label -label">52</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Purcell</div> <div class="given-names">C.M.</div></div>, <div class="name string-name"><div class="surname">Seetharam</div> <div class="given-names">A.S.</div></div>, <div class="name string-name"><div class="surname">Snodgrass</div> <div class="given-names">O.</div></div>, <div class="name string-name"><div class="surname">Ortega-Garcia</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Hyde</div> <div class="given-names">J.R.</div></div>, <div class="name string-name"><div class="surname">Severin</div> <div class="given-names">A.J.</div></div></div> <div class="year">2018</div>, 
<div class="article-title">Insights into teleost sex determination from the <em>Seriola dorsalis</em> genome assembly</div>, <div class="source">BMC Genomics</div> , <div class="volume">19</div>, <div class="fpage">31</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Insights%20into%20teleost%20sex%20determination%20from%20the%20Seriola%20dorsalis%20genome%20assembly&amp;author=C.M.%20Purcell&amp;author=A.S.%20Seetharam&amp;author=O.%20Snodgrass&amp;author=S.%20Ortega-Garcia&amp;author=J.R.%20Hyde&amp;author=A.J.%20Severin&amp;publication_year=2018&amp;journal=BMC%20Genomics&amp;volume=19&amp;pages=31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12864-017-4403-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12864-017-4403-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12864-017-4403-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29310588" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy024-B53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B53" class="ref-content "><div class="label -label">53</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Malmstrom</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Matschiner</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Torresen</div> <div class="given-names">O.K.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">Evolution of the immune system influences speciation rates in teleost fishes</div>, <div class="source">Nat. Genet</div> ., <div class="volume">48</div>, <div class="fpage">1204</div>–<div class="lpage">10</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20of%20the%20immune%20system%20influences%20speciation%20rates%20in%20teleost%20fishes&amp;author=M.%20Malmstrom&amp;author=M.%20Matschiner&amp;author=O.K.%20Torresen&amp;publication_year=2016&amp;journal=Nat.%20Genet&amp;volume=48&amp;pages=1204-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.3645" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.3645" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.3645"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27548311" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy024-B54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B54" class="ref-content "><div class="label -label">54</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Solbakken</div> <div class="given-names">M.H.</div></div>, <div class="name string-name"><div class="surname">Voje</div> <div class="given-names">K.L.</div></div>, <div class="name string-name"><div class="surname">Jakobsen</div> <div class="given-names">K.S.</div></div>, <div class="name string-name"><div class="surname">Jentoft</div> <div class="given-names">S.</div></div></div> <div class="year">2017</div>, 
<div class="article-title">Linking species habitat and past palaeoclimatic events to evolution of the teleost innate immune system</div>, <div class="source">Proc. R Soc. B</div> , <div class="volume">284</div>, <div class="fpage">20162810</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Linking%20species%20habitat%20and%20past%20palaeoclimatic%20events%20to%20evolution%20of%20the%20teleost%20innate%20immune%20system&amp;author=M.H.%20Solbakken&amp;author=K.L.%20Voje&amp;author=K.S.%20Jakobsen&amp;author=S.%20Jentoft&amp;publication_year=2017&amp;journal=Proc.%20R%20Soc.%20B&amp;volume=284&amp;pages=20162810" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1098/rspb.2016.2810" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1098%2Frspb.2016.2810" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1098%2Frspb.2016.2810"> </span></div> </div></div></div></div></div><div content-id="dsy024-B55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B55" class="ref-content "><div class="label -label">55</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Maeda</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Oshiman</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Tamura</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Futai</div> <div class="given-names">M.</div></div></div> <div class="year">1990</div>, 
<div class="article-title">Human gastric (H+ + K+)-ATPase gene. Similarity to (Na+ + K+)-ATPase genes in exon/intron organization but difference in control region</div>, <div class="source">J. Biol. Chem</div> ., <div class="volume">265</div>, <div class="fpage">9027</div>–<div class="lpage">32</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Human%20gastric%20(H%2B%20%2B%20K%2B)-ATPase%20gene.%20Similarity%20to%20(Na%2B%20%2B%20K%2B)-ATPase%20genes%20in%20exon%2Fintron%20organization%20but%20difference%20in%20control%20region&amp;author=M.%20Maeda&amp;author=K.%20Oshiman&amp;author=S.%20Tamura&amp;author=M.%20Futai&amp;publication_year=1990&amp;journal=J.%20Biol.%20Chem&amp;volume=265&amp;pages=9027-32" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2160952" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy024-B56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B56" class="ref-content "><div class="label -label">56</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ogino</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Takeuchi</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Takeda</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Watanabe</div> <div class="given-names">T.</div></div></div> <div class="year">1979</div>, 
<div class="article-title">Availability of dietary phosphorus in carp and rainbow trout</div>, <div class="source">Nippon Suisan Gakkaishi</div> , <div class="volume">45</div>, <div class="fpage">1527</div>–<div class="lpage">32</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Availability%20of%20dietary%20phosphorus%20in%20carp%20and%20rainbow%20trout&amp;author=C.%20Ogino&amp;author=L.%20Takeuchi&amp;author=H.%20Takeda&amp;author=T.%20Watanabe&amp;publication_year=1979&amp;journal=Nippon%20Suisan%20Gakkaishi&amp;volume=45&amp;pages=1527-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2331/suisan.45.1527" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2331%2Fsuisan.45.1527" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2331%2Fsuisan.45.1527"> </span></div> </div></div></div></div></div><div content-id="dsy024-B57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B57" class="ref-content "><div class="label -label">57</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Guffey</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Esbaugh</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Grosell</div> <div class="given-names">M.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Regulation of apical H(+)-ATPase activity and intestinal HCO(3)(-) secretion in marine fish osmoregulation</div>, <div class="source">Am. J. Physiol. Regul. Integr. Comp. Physiol</div> ., <div class="volume">301</div>, <div class="fpage">R1682</div>–<div class="lpage">91</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Regulation%20of%20apical%20H(%2B)-ATPase%20activity%20and%20intestinal%20HCO(3)(-)%20secretion%20in%20marine%20fish%20osmoregulation&amp;author=S.%20Guffey&amp;author=A.%20Esbaugh&amp;author=M.%20Grosell&amp;publication_year=2011&amp;journal=Am.%20J.%20Physiol.%20Regul.%20Integr.%20Comp.%20Physiol&amp;volume=301&amp;pages=R1682-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1152/ajpregu.00059.2011" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1152%2Fajpregu.00059.2011" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1152%2Fajpregu.00059.2011"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21865541" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy024-B58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B58" class="ref-content "><div class="label -label">58</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Calduch-Giner</div> <div class="given-names">J.A.</div></div>, <div class="name string-name"><div class="surname">Sitja-Bobadilla</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Perez-Sanchez</div> <div class="given-names">J.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Gene expression profiling reveals functional specialization along the intestinal tract of a carnivorous teleostean fish (<em>Dicentrarchus labrax</em>)</div>, <div class="source">Front. Physiol</div> ., <div class="volume">7</div>, <div class="fpage">359</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20expression%20profiling%20reveals%20functional%20specialization%20along%20the%20intestinal%20tract%20of%20a%20carnivorous%20teleostean%20fish%20(Dicentrarchus%20labrax)&amp;author=J.A.%20Calduch-Giner&amp;author=A.%20Sitja-Bobadilla&amp;author=J.%20Perez-Sanchez&amp;publication_year=2016&amp;journal=Front.%20Physiol&amp;volume=7&amp;pages=359" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3389/fphys.2016.00359" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3389%2Ffphys.2016.00359" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3389%2Ffphys.2016.00359"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27610085" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy024-B59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B59" class="ref-content "><div class="label -label">59</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sachs</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Chang</div> <div class="given-names">H.H.</div></div>, <div class="name string-name"><div class="surname">Rabon</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Schackman</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Lewin</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Saccomani</div> <div class="given-names">G.</div></div></div> <div class="year">1976</div>, 
<div class="article-title">A nonelectrogenic H+ pump in plasma membranes of hog stomach</div>, <div class="source">J. Biol. Chem</div> ., <div class="volume">251</div>, <div class="fpage">7690</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20nonelectrogenic%20H%2B%20pump%20in%20plasma%20membranes%20of%20hog%20stomach&amp;author=G.%20Sachs&amp;author=H.H.%20Chang&amp;author=E.%20Rabon&amp;author=R.%20Schackman&amp;author=M.%20Lewin&amp;author=G.%20Saccomani&amp;publication_year=1976&amp;journal=J.%20Biol.%20Chem&amp;volume=251&amp;pages=7690-8" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12175" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy024-B60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B60" class="ref-content "><div class="label -label">60</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Foltmann</div> <div class="given-names">B.</div></div></div> <div class="year">1981</div>, 
<div class="article-title">Gastric proteinases–structure, function, evolution and mechanism of action</div>, <div class="source">Essays Biochem</div> ., <div class="volume">17</div>, <div class="fpage">52</div>–<div class="lpage">84</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gastric%20proteinases%E2%80%93structure%2C%20function%2C%20evolution%20and%20mechanism%20of%20action&amp;author=B.%20Foltmann&amp;publication_year=1981&amp;journal=Essays%20Biochem&amp;volume=17&amp;pages=52-84" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6795036" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy024-B61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B61" class="ref-content "><div class="label -label">61</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chong</div> <div class="given-names">A.S.C.</div></div>, <div class="name string-name"><div class="surname">Hashim</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Chow-Yang</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Ali</div> <div class="given-names">A.B.</div></div></div> <div class="year">2002</div>, 
<div class="article-title">Partial characterization and activities of proteases from the digestive tract of discus fish (<em>Symphysodon aequifasciata</em>)</div>, <div class="source">Aquaculture</div> , <div class="volume">203</div>, <div class="fpage">321</div>–<div class="lpage">33</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Partial%20characterization%20and%20activities%20of%20proteases%20from%20the%20digestive%20tract%20of%20discus%20fish%20(Symphysodon%20aequifasciata)&amp;author=A.S.C.%20Chong&amp;author=R.%20Hashim&amp;author=L.%20Chow-Yang&amp;author=A.B.%20Ali&amp;publication_year=2002&amp;journal=Aquaculture&amp;volume=203&amp;pages=321-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0044-8486(01)00630-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0044-8486(01)00630-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0044-8486(01)00630-5"> </span></div> </div></div></div></div></div><div content-id="dsy024-B62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B62" class="ref-content "><div class="label -label">62</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Deguara</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Jauncey</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Agius</div> <div class="given-names">C.</div></div></div> <div class="year">2003</div>, 
<div class="article-title">Enzyme activities and pH variations in the digestive tract of gilthead sea bream</div>, <div class="source">J. Fish Biol</div> ., <div class="volume">62</div>, <div class="fpage">1033</div>–<div class="lpage">43</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Enzyme%20activities%20and%20pH%20variations%20in%20the%20digestive%20tract%20of%20gilthead%20sea%20bream&amp;author=S.%20Deguara&amp;author=K.%20Jauncey&amp;author=C.%20Agius&amp;publication_year=2003&amp;journal=J.%20Fish%20Biol&amp;volume=62&amp;pages=1033-43" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1095-8649.2003.00094.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1095-8649.2003.00094.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1095-8649.2003.00094.x"> </span></div> </div></div></div></div></div><div content-id="dsy024-B63" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B63" class="ref-content "><div class="label -label">63</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Haard</div> <div class="given-names">N.F.</div></div></div> <div class="year">1995</div>, Digestibility and in vitro evaluation of vegetable proteins for salmonid feed. In: <div class="person-group"><div class="name string-name"><div class="surname">Lim</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Sessa</div> <div class="given-names">D.J.</div></div></div> (eds), <div class="source">Nutrition and Utilization Technology in Aqualture</div> , 
<div class="publisher-loc">Champaign, IL</div>: 
<div class="publisher-name">AOCS Press</div>, pp. <div class="fpage">199</div>–<div class="lpage">219</div>.</div></div></div></div><div content-id="dsy024-B64" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B64" class="ref-content "><div class="label -label">64</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wang</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Du</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Lam</div> <div class="given-names">S.H.</div></div>, <div class="name string-name"><div class="surname">Mathavan</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Matsudaira</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Gong</div> <div class="given-names">Z.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Morphological and molecular evidence for functional organization along the rostrocaudal axis of the adult zebrafish intestine</div>, <div class="source">BMC Genomics</div> , <div class="volume">11</div>, <div class="fpage">392</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Morphological%20and%20molecular%20evidence%20for%20functional%20organization%20along%20the%20rostrocaudal%20axis%20of%20the%20adult%20zebrafish%20intestine&amp;author=Z.%20Wang&amp;author=J.%20Du&amp;author=S.H.%20Lam&amp;author=S.%20Mathavan&amp;author=P.%20Matsudaira&amp;author=Z.%20Gong&amp;publication_year=2010&amp;journal=BMC%20Genomics&amp;volume=11&amp;pages=392" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-11-392" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-11-392" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-11-392"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20565988" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy024-B65" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B65" class="ref-content "><div class="label -label">65</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ikeda</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Kakizaki</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Matsumiya</div> <div class="given-names">M.</div></div></div> <div class="year">2017</div>, 
<div class="article-title">Biochemistry of fish stomach chitinase</div>, <div class="source">Int. J. Biol. Macromol</div> ., <div class="volume">104</div>, <div class="fpage">1672</div>–<div class="lpage">81</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Biochemistry%20of%20fish%20stomach%20chitinase&amp;author=M.%20Ikeda&amp;author=H.%20Kakizaki&amp;author=M.%20Matsumiya&amp;publication_year=2017&amp;journal=Int.%20J.%20Biol.%20Macromol&amp;volume=104&amp;pages=1672-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ijbiomac.2017.03.118" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ijbiomac.2017.03.118" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ijbiomac.2017.03.118"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28365290" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy024-B66" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B66" class="ref-content "><div class="label -label">66</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Matsumiya</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Mochizuki</div> <div class="given-names">A.</div></div></div> <div class="year">1996</div>, 
<div class="article-title">Distribution of chitinase and β-N-acetylhexosaminidase in the organs of several fishes</div>, <div class="source">Fish. Sci</div> ., <div class="volume">62</div>, <div class="fpage">150</div>–<div class="lpage">1</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Distribution%20of%20chitinase%20and%20%CE%B2-N-acetylhexosaminidase%20in%20the%20organs%20of%20several%20fishes&amp;author=M.%20Matsumiya&amp;author=A.%20Mochizuki&amp;publication_year=1996&amp;journal=Fish.%20Sci&amp;volume=62&amp;pages=150-1" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2331/fishsci.62.150" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2331%2Ffishsci.62.150" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2331%2Ffishsci.62.150"> </span></div> </div></div></div></div></div><div content-id="dsy024-B67" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B67" class="ref-content "><div class="label -label">67</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lang</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Klasson</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Larsson</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Johansson</div> <div class="given-names">M.E.</div></div>, <div class="name string-name"><div class="surname">Hansson</div> <div class="given-names">G.C.</div></div>, <div class="name string-name"><div class="surname">Samuelsson</div> <div class="given-names">T.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Searching the evolutionary origin of epithelial mucus protein components-mucins and FCGBP</div>, <div class="source">Mol. Biol. Evol</div> ., <div class="volume">33</div>, <div class="fpage">1921</div>–<div class="lpage">36</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Searching%20the%20evolutionary%20origin%20of%20epithelial%20mucus%20protein%20components-mucins%20and%20FCGBP&amp;author=T.%20Lang&amp;author=S.%20Klasson&amp;author=E.%20Larsson&amp;author=M.E.%20Johansson&amp;author=G.C.%20Hansson&amp;author=T.%20Samuelsson&amp;publication_year=2016&amp;journal=Mol.%20Biol.%20Evol&amp;volume=33&amp;pages=1921-36" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msw066" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsw066" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsw066"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27189557" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy024-B68" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B68" class="ref-content "><div class="label -label">68</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Koppers</div> <div class="given-names">A.J.</div></div>, <div class="name string-name"><div class="surname">Reddy</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">O’Bryan</div> <div class="given-names">M.K.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">The role of cysteine-rich secretory proteins in male fertility</div>, <div class="source">Asian J. Androl</div> ., <div class="volume">13</div>, <div class="fpage">111</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20role%20of%20cysteine-rich%20secretory%20proteins%20in%20male%20fertility&amp;author=A.J.%20Koppers&amp;author=T.%20Reddy&amp;author=M.K.%20O%E2%80%99Bryan&amp;publication_year=2011&amp;journal=Asian%20J.%20Androl&amp;volume=13&amp;pages=111-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/aja.2010.77" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Faja.2010.77" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Faja.2010.77"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20972450" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy024-B69" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B69" class="ref-content "><div class="label -label">69</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sire</div> <div class="given-names">M.F.</div></div>, <div class="name string-name"><div class="surname">Vernier</div> <div class="given-names">J.-M.</div></div></div> <div class="year">1992</div>, 
<div class="article-title">Intestinal absorption of protein in teleost fish</div>, <div class="source">Comp. Biochem. Physiol. A Physiol</div> ., <div class="volume">103</div>, <div class="fpage">771</div>–<div class="lpage">81</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Intestinal%20absorption%20of%20protein%20in%20teleost%20fish&amp;author=M.F.%20Sire&amp;author=J.-M.%20Vernier&amp;publication_year=1992&amp;journal=Comp.%20Biochem.%20Physiol.%20A%20Physiol&amp;volume=103&amp;pages=771-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0300-9629(92)90180-X" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0300-9629(92)90180-X" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0300-9629(92)90180-X"> </span></div> </div></div></div></div></div><div content-id="dsy024-B70" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B70" class="ref-content "><div class="label -label">70</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lilleeng</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Froystad</div> <div class="given-names">M.K.</div></div>, <div class="name string-name"><div class="surname">Ostby</div> <div class="given-names">G.C.</div></div>, <div class="name string-name"><div class="surname">Valen</div> <div class="given-names">E.C.</div></div>, <div class="name string-name"><div class="surname">Krogdahl</div> <div class="given-names">A.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">Effects of diets containing soybean meal on trypsin mRNA expression and activity in Atlantic salmon (<em>Salmo salar</em> L)</div>, <div class="source">Comp. Biochem. Physiol. A Mol. Integr. Physiol</div> ., <div class="volume">147</div>, <div class="fpage">25</div>–<div class="lpage">36</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Effects%20of%20diets%20containing%20soybean%20meal%20on%20trypsin%20mRNA%20expression%20and%20activity%20in%20Atlantic%20salmon%20(Salmo%20salar%20L)&amp;author=E.%20Lilleeng&amp;author=M.K.%20Froystad&amp;author=G.C.%20Ostby&amp;author=E.C.%20Valen&amp;author=A.%20Krogdahl&amp;publication_year=2007&amp;journal=Comp.%20Biochem.%20Physiol.%20A%20Mol.%20Integr.%20Physiol&amp;volume=147&amp;pages=25-36" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cbpa.2006.10.043" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cbpa.2006.10.043" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cbpa.2006.10.043"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17293147" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy024-B71" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B71" class="ref-content "><div class="label -label">71</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Manchado</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Infante</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Asensio</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Crespo</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Zuasti</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Cañavate</div> <div class="given-names">J.P.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">Molecular characterization and gene expression of six trypsinogens in the flatfish Senegalese sole (<em>Solea senegalensis</em> Kaup) during larval development and in tissues</div>, <div class="source">Comp. Biochem. Physiol. B, Biochem. Mol. Biol</div> ., <div class="volume">149</div>, <div class="fpage">334</div>–<div class="lpage">44</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20characterization%20and%20gene%20expression%20of%20six%20trypsinogens%20in%20the%20flatfish%20Senegalese%20sole%20(Solea%20senegalensis%20Kaup)%20during%20larval%20development%20and%20in%20tissues&amp;author=M.%20Manchado&amp;author=C.%20Infante&amp;author=E.%20Asensio&amp;author=A.%20Crespo&amp;author=E.%20Zuasti&amp;author=J.P.%20Ca%C3%B1avate&amp;publication_year=2008&amp;journal=Comp.%20Biochem.%20Physiol.%20B%2C%20Biochem.%20Mol.%20Biol&amp;volume=149&amp;pages=334-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cbpb.2007.10.005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cbpb.2007.10.005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cbpb.2007.10.005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18006348" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy024-B72" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B72" class="ref-content "><div class="label -label">72</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zhong</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Qian</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Hua</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Zhou</div> <div class="given-names">H.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Effects of feeding with corn gluten meal on trypsin activity and mRNA expression in <em>Fugu obscurus</em></div>, <div class="source">Fish Physiol. Biochem</div> ., <div class="volume">37</div>, <div class="fpage">453</div>–<div class="lpage">60</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Effects%20of%20feeding%20with%20corn%20gluten%20meal%20on%20trypsin%20activity%20and%20mRNA%20expression%20in%20Fugu%20obscurus&amp;author=G.%20Zhong&amp;author=X.%20Qian&amp;author=X.%20Hua&amp;author=H.%20Zhou&amp;publication_year=2011&amp;journal=Fish%20Physiol.%20Biochem&amp;volume=37&amp;pages=453-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10695-010-9446-z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10695-010-9446-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10695-010-9446-z"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21063906" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy024-B73" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B73" class="ref-content "><div class="label -label">73</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Choudhury</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Oku</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Yamada</div> <div class="given-names">S.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">Isolation and characterization of some novel genes of the apolipoprotein A-I family in Japanese eel, Anguilla Japonica</div>, <div class="source">Cent. Eur. J. Biol</div> ., <div class="volume">6</div>, <div class="fpage">545</div>.</div></div></div></div><div content-id="dsy024-B74" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B74" class="ref-content "><div class="label -label">74</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kim</div> <div class="given-names">K.Y.</div></div>, <div class="name string-name"><div class="surname">Cho</div> <div class="given-names">Y.S.</div></div>, <div class="name string-name"><div class="surname">Bang</div> <div class="given-names">I.C.</div></div>, <div class="name string-name"><div class="surname">Nam</div> <div class="given-names">Y.K.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Isolation and characterization of the apolipoprotein multigene family in <em>Hemibarbus mylodon</em> (Teleostei: cypriniformes)</div>, <div class="source">Comp. Biochem. Physiol. B Biochem. Mol. Biol</div> ., <div class="volume">152</div>, <div class="fpage">38</div>–<div class="lpage">46</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Isolation%20and%20characterization%20of%20the%20apolipoprotein%20multigene%20family%20in%20Hemibarbus%20mylodon%20(Teleostei%3A%20cypriniformes)&amp;author=K.Y.%20Kim&amp;author=Y.S.%20Cho&amp;author=I.C.%20Bang&amp;author=Y.K.%20Nam&amp;publication_year=2009&amp;journal=Comp.%20Biochem.%20Physiol.%20B%20Biochem.%20Mol.%20Biol&amp;volume=152&amp;pages=38-46" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cbpb.2008.09.084" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cbpb.2008.09.084" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cbpb.2008.09.084"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18838126" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy024-B75" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B75" class="ref-content "><div class="label -label">75</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Qu</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Huang</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">Q.</div></div>, <div class="name string-name"><div class="surname">Ding</div> <div class="given-names">S.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Cloning and expression analysis of apolipoprotein A-I (ApoA-I) in the Hong Kong grouper (<em>Epinephelus akaara</em>)</div>, <div class="source">Aquaculture</div> , <div class="volume">432</div>, <div class="fpage">85</div>–<div class="lpage">96</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cloning%20and%20expression%20analysis%20of%20apolipoprotein%20A-I%20(ApoA-I)%20in%20the%20Hong%20Kong%20grouper%20(Epinephelus%20akaara)&amp;author=M.%20Qu&amp;author=X.%20Huang&amp;author=X.%20Zhang&amp;author=Q.%20Liu&amp;author=S.%20Ding&amp;publication_year=2014&amp;journal=Aquaculture&amp;volume=432&amp;pages=85-96" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.aquaculture.2014.04.023" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.aquaculture.2014.04.023" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.aquaculture.2014.04.023"> </span></div> </div></div></div></div></div><div content-id="dsy024-B76" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B76" class="ref-content "><div class="label -label">76</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Deng</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Laursen</div> <div class="given-names">R.A.</div></div></div> <div class="year">1998</div>, 
<div class="article-title">Isolation and characterization of an antifreeze protein from the longhorn sculpin, <em>Myoxocephalus octodecimspinosis</em></div>, <div class="source">Biochim. Biophys. Acta</div> , <div class="volume">1388</div>, <div class="fpage">305</div>–<div class="lpage">14</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Isolation%20and%20characterization%20of%20an%20antifreeze%20protein%20from%20the%20longhorn%20sculpin%2C%20Myoxocephalus%20octodecimspinosis&amp;author=G.%20Deng&amp;author=R.A.%20Laursen&amp;publication_year=1998&amp;journal=Biochim.%20Biophys.%20Acta&amp;volume=1388&amp;pages=305-14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0167-4838(98)00180-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0167-4838(98)00180-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0167-4838(98)00180-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9858755" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy024-B77" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B77" class="ref-content "><div class="label -label">77</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Luquet</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Watanabe</div> <div class="given-names">T.</div></div></div> <div class="year">1986</div>, 
<div class="article-title">Lipid nutrition in fish</div>, <div class="source">Fish Physiol. Biochem</div> ., <div class="volume">2</div>, <div class="fpage">121</div>.,<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Lipid%20nutrition%20in%20fish&amp;author=P.%20Luquet&amp;author=T.%20Watanabe&amp;publication_year=1986&amp;journal=Fish%20Physiol.%20Biochem&amp;volume=2&amp;pages=121" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF02264080" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF02264080" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF02264080"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24233174" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy024-B78" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B78" class="ref-content "><div class="label -label">78</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ohno</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Kimura</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Miyazaki</div> <div class="given-names">H.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">Acidic mammalian chitinase is a proteases-resistant glycosidase in mouse digestive system</div>, <div class="source">Sci. Rep</div> ., <div class="volume">6</div>, <div class="fpage">37756</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Acidic%20mammalian%20chitinase%20is%20a%20proteases-resistant%20glycosidase%20in%20mouse%20digestive%20system&amp;author=M.%20Ohno&amp;author=M.%20Kimura&amp;author=H.%20Miyazaki&amp;publication_year=2016&amp;journal=Sci.%20Rep&amp;volume=6&amp;pages=37756" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/srep37756" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsrep37756" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsrep37756"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27883045" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy024-B79" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B79" class="ref-content "><div class="label -label">79</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Vannella</div> <div class="given-names">K.M.</div></div>, <div class="name string-name"><div class="surname">Ramalingam</div> <div class="given-names">T.R.</div></div>, <div class="name string-name"><div class="surname">Hart</div> <div class="given-names">K.M.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">Acidic chitinase primes the protective immune response to gastrointestinal nematodes</div>, <div class="source">Nat. Immunol</div> ., <div class="volume">17</div>, <div class="fpage">538</div>–<div class="lpage">44</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Acidic%20chitinase%20primes%20the%20protective%20immune%20response%20to%20gastrointestinal%20nematodes&amp;author=K.M.%20Vannella&amp;author=T.R.%20Ramalingam&amp;author=K.M.%20Hart&amp;publication_year=2016&amp;journal=Nat.%20Immunol&amp;volume=17&amp;pages=538-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ni.3417" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fni.3417" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fni.3417"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27043413" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy024-B80" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B80" class="ref-content "><div class="label -label">80</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Martin</div> <div class="given-names">S.A.M.</div></div>, <div class="name string-name"><div class="surname">Dehler</div> <div class="given-names">C.E.</div></div>, <div class="name string-name"><div class="surname">Król</div> <div class="given-names">E.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Transcriptomic responses in the fish intestine</div>, <div class="source">Dev. Comp. Immunol</div> ., <div class="volume">64</div>, <div class="fpage">103</div>–<div class="lpage">17</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptomic%20responses%20in%20the%20fish%20intestine&amp;author=S.A.M.%20Martin&amp;author=C.E.%20Dehler&amp;author=E.%20Kr%C3%B3l&amp;publication_year=2016&amp;journal=Dev.%20Comp.%20Immunol&amp;volume=64&amp;pages=103-17" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.dci.2016.03.014" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.dci.2016.03.014" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.dci.2016.03.014"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26995769" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy024-B81" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B81" class="ref-content "><div class="label -label">81</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Rombout</div> <div class="given-names">J.H.W.M.</div></div>, <div class="name string-name"><div class="surname">Yang</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Kiron</div> <div class="given-names">V.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Adaptive immune responses at mucosal surfaces of teleost fish</div>, <div class="source">Fish Shellfish Immunol</div> ., <div class="volume">40</div>, <div class="fpage">634</div>–<div class="lpage">43</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Adaptive%20immune%20responses%20at%20mucosal%20surfaces%20of%20teleost%20fish&amp;author=J.H.W.M.%20Rombout&amp;author=G.%20Yang&amp;author=V.%20Kiron&amp;publication_year=2014&amp;journal=Fish%20Shellfish%20Immunol&amp;volume=40&amp;pages=634-43" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.fsi.2014.08.020" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.fsi.2014.08.020" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.fsi.2014.08.020"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25150451" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy024-B82" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B82" class="ref-content "><div class="label -label">82</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Concha</div> <div class="given-names">M.I.</div></div>, <div class="name string-name"><div class="surname">Molina</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Oyarzun</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Villanueva</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Amthauer</div> <div class="given-names">R.</div></div></div> <div class="year">2003</div>, 
<div class="article-title">Local expression of apolipoprotein A-I gene and a possible role for HDL in primary defence in the carp skin</div>, <div class="source">Fish Shellfish Immunol</div> ., <div class="volume">14</div>, <div class="fpage">259</div>–<div class="lpage">73</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Local%20expression%20of%20apolipoprotein%20A-I%20gene%20and%20a%20possible%20role%20for%20HDL%20in%20primary%20defence%20in%20the%20carp%20skin&amp;author=M.I.%20Concha&amp;author=S.%20Molina&amp;author=C.%20Oyarzun&amp;author=J.%20Villanueva&amp;author=R.%20Amthauer&amp;publication_year=2003&amp;journal=Fish%20Shellfish%20Immunol&amp;volume=14&amp;pages=259-73" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/fsim.2002.0435" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Ffsim.2002.0435" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Ffsim.2002.0435"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12681280" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy024-B83" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B83" class="ref-content "><div class="label -label">83</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Concha</div> <div class="given-names">M.I.</div></div>, <div class="name string-name"><div class="surname">Smith</div> <div class="given-names">V.J.</div></div>, <div class="name string-name"><div class="surname">Castro</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Bastias</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Romero</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Amthauer</div> <div class="given-names">R.J.</div></div></div> <div class="year">2004</div>, 
<div class="article-title">Apolipoproteins A-I and A-II are potentially important effectors of innate immunity in the teleost fish,</div> <div class="source">Cyprinus Carpio. Eur. J. Biochem</div> ., <div class="volume">271</div>, <div class="fpage">2984</div>–<div class="lpage">90</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Apolipoproteins%20A-I%20and%20A-II%20are%20potentially%20important%20effectors%20of%20innate%20immunity%20in%20the%20teleost%20fish%2C&amp;author=M.I.%20Concha&amp;author=V.J.%20Smith&amp;author=K.%20Castro&amp;author=A.%20Bastias&amp;author=A.%20Romero&amp;author=R.J.%20Amthauer&amp;publication_year=2004&amp;journal=Cyprinus%20Carpio.%20Eur.%20J.%20Biochem&amp;volume=271&amp;pages=2984-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/(ISSN)1432-1033" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2F(ISSN)1432-1033" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2F(ISSN)1432-1033"> </span></div> </div></div></div></div></div><div content-id="dsy024-B84" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B84" class="ref-content "><div class="label -label">84</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Villarroel</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Bastias</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Casado</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Amthauer</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Concha</div> <div class="given-names">M.I.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">Apolipoprotein A-I, an antimicrobial protein in <em>Oncorhynchus mykiss</em>: evaluation of its expression in primary defence barriers and plasma levels in sick and healthy fish</div>, <div class="source">Fish Shellfish Immunol</div> ., <div class="volume">23</div>, <div class="fpage">197</div>–<div class="lpage">209</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Apolipoprotein%20A-I%2C%20an%20antimicrobial%20protein%20in%20Oncorhynchus%20mykiss%3A%20evaluation%20of%20its%20expression%20in%20primary%20defence%20barriers%20and%20plasma%20levels%20in%20sick%20and%20healthy%20fish&amp;author=F.%20Villarroel&amp;author=A.%20Bastias&amp;author=A.%20Casado&amp;author=R.%20Amthauer&amp;author=M.I.%20Concha&amp;publication_year=2007&amp;journal=Fish%20Shellfish%20Immunol&amp;volume=23&amp;pages=197-209" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.fsi.2006.10.008" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.fsi.2006.10.008" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.fsi.2006.10.008"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17391986" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy024-B85" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B85" class="ref-content "><div class="label -label">85</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Abrams</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Einhorn</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Seiler</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Zong</div> <div class="given-names">A.B.</div></div>, <div class="name string-name"><div class="surname">Sweeney</div> <div class="given-names">H.L.</div></div>, <div class="name string-name"><div class="surname">Pack</div> <div class="given-names">M.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Graded effects of unregulated smooth muscle myosin on intestinal architecture, intestinal motility and vascular function in zebrafish</div>, <div class="source">Dis. Model. Mech</div> ., <div class="volume">9</div>, <div class="fpage">529</div>–<div class="lpage">40</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Graded%20effects%20of%20unregulated%20smooth%20muscle%20myosin%20on%20intestinal%20architecture%2C%20intestinal%20motility%20and%20vascular%20function%20in%20zebrafish&amp;author=J.%20Abrams&amp;author=Z.%20Einhorn&amp;author=C.%20Seiler&amp;author=A.B.%20Zong&amp;author=H.L.%20Sweeney&amp;author=M.%20Pack&amp;publication_year=2016&amp;journal=Dis.%20Model.%20Mech&amp;volume=9&amp;pages=529-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1242/dmm.023309" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1242%2Fdmm.023309" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1242%2Fdmm.023309"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26893369" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy024-B86" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B86" class="ref-content "><div class="label -label">86</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gutierrez</div> <div class="given-names">J.A.</div></div>, <div class="name string-name"><div class="surname">Perr</div> <div class="given-names">H.A.</div></div></div> <div class="year">1999</div>, 
<div class="article-title">Mechanical stretch modulates TGF-β1 and α1(I) collagen expression in fetal human intestinal smooth muscle cells</div>, <div class="source">Am. J. Physiol. Gastrointest. Liver Physiol</div> ., <div class="volume">277</div>, <div class="fpage">G1074</div>–<div class="lpage">80</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mechanical%20stretch%20modulates%20TGF-%CE%B21%20and%20%CE%B11(I)%20collagen%20expression%20in%20fetal%20human%20intestinal%20smooth%20muscle%20cells&amp;author=J.A.%20Gutierrez&amp;author=H.A.%20Perr&amp;publication_year=1999&amp;journal=Am.%20J.%20Physiol.%20Gastrointest.%20Liver%20Physiol&amp;volume=277&amp;pages=G1074-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1152/ajpgi.1999.277.5.G1074" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1152%2Fajpgi.1999.277.5.G1074" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1152%2Fajpgi.1999.277.5.G1074"> </span></div> </div></div></div></div></div><div content-id="dsy024-B87" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B87" class="ref-content "><div class="label -label">87</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Grau</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Crespo</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Sarasquete</div> <div class="given-names">M.C.</div></div>, <div class="name string-name"><div class="surname">de Canales</div> <div class="given-names">M.L.G.</div></div></div> <div class="year">1992</div>, 
<div class="article-title">The digestive tract of the amberjack <em>Seriola dumerili</em>, Risso: a light and scanning electron microscope study</div>, <div class="source">J. Fish Biol</div> ., <div class="volume">41</div>, <div class="fpage">287</div>–<div class="lpage">303</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20digestive%20tract%20of%20the%20amberjack%20Seriola%20dumerili%2C%20Risso%3A%20a%20light%20and%20scanning%20electron%20microscope%20study&amp;author=A.%20Grau&amp;author=S.%20Crespo&amp;author=M.C.%20Sarasquete&amp;author=M.L.G.%20de%20Canales&amp;publication_year=1992&amp;journal=J.%20Fish%20Biol&amp;volume=41&amp;pages=287-303" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/jfb.1992.41.issue-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fjfb.1992.41.issue-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fjfb.1992.41.issue-2"> </span></div> </div></div></div></div></div><div content-id="dsy024-B88" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B88" class="ref-content "><div class="label -label">88</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Christensen</div> <div class="given-names">E.I.</div></div>, <div class="name string-name"><div class="surname">Birn</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Storm</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Weyer</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Nielsen</div> <div class="given-names">R.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Endocytic receptors in the renal proximal tubule</div>, <div class="source">Physiology (Bethesda)</div> , <div class="volume">27</div>, <div class="fpage">223</div>–<div class="lpage">36</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Endocytic%20receptors%20in%20the%20renal%20proximal%20tubule&amp;author=E.I.%20Christensen&amp;author=H.%20Birn&amp;author=T.%20Storm&amp;author=K.%20Weyer&amp;author=R.%20Nielsen&amp;publication_year=2012&amp;journal=Physiology%20(Bethesda)&amp;volume=27&amp;pages=223-36" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22875453" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy024-B89" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B89" class="ref-content "><div class="label -label">89</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Haraldsson</div> <div class="given-names">B.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Tubular reabsorption of albumin: it's all about cubilin</div>, <div class="source">J. Am. Soc. Nephrol</div> ., <div class="volume">21</div>, <div class="fpage">1810</div>–<div class="lpage">2</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Tubular%20reabsorption%20of%20albumin%3A%20it's%20all%20about%20cubilin&amp;author=B.%20Haraldsson&amp;publication_year=2010&amp;journal=J.%20Am.%20Soc.%20Nephrol&amp;volume=21&amp;pages=1810-2" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1681/ASN.2010090984" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1681%2FASN.2010090984" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1681%2FASN.2010090984"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20947627" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy024-B90" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B90" class="ref-content "><div class="label -label">90</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Rodrı´guez</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Esteban</div> <div class="given-names">M.A.</div></div>, <div class="name string-name"><div class="surname">Meseguer</div> <div class="given-names">J.</div></div></div> <div class="year">2003</div>, 
<div class="article-title">A mannose-receptor is possibly involved in the phagocytosis of <em>Saccharomyces cerevisiae</em> by seabream (<em>Sparus aurata</em> L.) leucocytes</div>, <div class="source">Fish Shellfish Immunol</div> ., <div class="volume">14</div>, <div class="fpage">375</div>–<div class="lpage">88</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20mannose-receptor%20is%20possibly%20involved%20in%20the%20phagocytosis%20of%20Saccharomyces%20cerevisiae%20by%20seabream%20(Sparus%20aurata%20L.)%20leucocytes&amp;author=A.%20Rodr%C4%B1%C2%B4guez&amp;author=M.A.%20Esteban&amp;author=J.%20Meseguer&amp;publication_year=2003&amp;journal=Fish%20Shellfish%20Immunol&amp;volume=14&amp;pages=375-88" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/fsim.2002.0446" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Ffsim.2002.0446" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Ffsim.2002.0446"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12711272" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy024-B91" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B91" class="ref-content "><div class="label -label">91</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Rosenstiel</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Sina</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">End</div> <div class="given-names">C.</div></div></div>, et al.   <div class="year">2007</div>, 
<div class="article-title">Regulation of DMBT1 via NOD2 and TLR4 in intestinal epithelial cells modulates bacterial recognition and invasion</div>, <div class="source">J. Immunol</div> ., <div class="volume">178</div>, <div class="fpage">8203</div>–<div class="lpage">11</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Regulation%20of%20DMBT1%20via%20NOD2%20and%20TLR4%20in%20intestinal%20epithelial%20cells%20modulates%20bacterial%20recognition%20and%20invasion&amp;author=P.%20Rosenstiel&amp;author=C.%20Sina&amp;author=C.%20End&amp;publication_year=2007&amp;journal=J.%20Immunol&amp;volume=178&amp;pages=8203-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.4049/jimmunol.178.12.8203" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.4049%2Fjimmunol.178.12.8203" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.4049%2Fjimmunol.178.12.8203"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17548659" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy024-B92" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B92" class="ref-content "><div class="label -label">92</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wang</div> <div class="given-names">A.M.</div></div>, <div class="name string-name"><div class="surname">Desnick</div> <div class="given-names">R.J.</div></div></div> <div class="year">1991</div>, 
<div class="article-title">Structural organization and complete sequence of the human α-N-acetylgalactosaminidase gene: homology with the α-galactosidase A gene provides evidence for evolution from a common ancestral gene</div>, <div class="source">Genomics</div> , <div class="volume">10</div>, <div class="fpage">133</div>–<div class="lpage">42</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structural%20organization%20and%20complete%20sequence%20of%20the%20human%20%CE%B1-N-acetylgalactosaminidase%20gene%3A%20homology%20with%20the%20%CE%B1-galactosidase%20A%20gene%20provides%20evidence%20for%20evolution%20from%20a%20common%20ancestral%20gene&amp;author=A.M.%20Wang&amp;author=R.J.%20Desnick&amp;publication_year=1991&amp;journal=Genomics&amp;volume=10&amp;pages=133-42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0888-7543(91)90493-X" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0888-7543(91)90493-X" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0888-7543(91)90493-X"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1646157" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy024-B93" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B93" class="ref-content "><div class="label -label">93</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sabapathy</div> <div class="given-names">U.</div></div>, <div class="name string-name"><div class="surname">Teo</div> <div class="given-names">L.H.</div></div></div> <div class="year">1993</div>, 
<div class="article-title">A quantitative study of some digestive enzymes in the rabbitfish, <em>Siganus canaliculatus</em> and the sea bass, <em>Lates calcarifer</em></div>, <div class="source">J. Fish Biol</div> ., <div class="volume">42</div>, <div class="fpage">595</div>–<div class="lpage">602</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20quantitative%20study%20of%20some%20digestive%20enzymes%20in%20the%20rabbitfish%2C%20Siganus%20canaliculatus%20and%20the%20sea%20bass%2C%20Lates%20calcarifer&amp;author=U.%20Sabapathy&amp;author=L.H.%20Teo&amp;publication_year=1993&amp;journal=J.%20Fish%20Biol&amp;volume=42&amp;pages=595-602" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/jfb.1993.42.issue-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fjfb.1993.42.issue-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fjfb.1993.42.issue-4"> </span></div> </div></div></div></div></div><div content-id="dsy024-B94" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B94" class="ref-content "><div class="label -label">94</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kapoor</div> <div class="given-names">B.G.</div></div>, <div class="name string-name"><div class="surname">Smit</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Verighina</div> <div class="given-names">I.A.</div></div></div> <div class="year">1976</div>, The alimentary canal and digestion in teleosts. In: <div class="person-group"><div class="name string-name"><div class="surname">Russell</div> <div class="given-names">F.S.</div></div>, <div class="name string-name"><div class="surname">Yonge</div> <div class="given-names">M.</div></div></div> (eds), <div class="source">Advances in Marine Biology</div> , 
<div class="publisher-loc">London</div>: 
<div class="publisher-name">Academic Press</div>, pp. <div class="fpage">109</div>–<div class="lpage">239</div>.</div></div></div></div><div content-id="dsy024-B95" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B95" class="ref-content "><div class="label -label">95</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hidalgo</div> <div class="given-names">M.C.</div></div>, <div class="name string-name"><div class="surname">Urea</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Sanz</div> <div class="given-names">A.</div></div></div> <div class="year">1999</div>, 
<div class="article-title">Comparative study of digestive enzymes in fish with different nutritional habits. Proteolytic and amylase activities</div>, <div class="source">Aquaculture</div> , <div class="volume">170</div>, <div class="fpage">267</div>–<div class="lpage">83</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20study%20of%20digestive%20enzymes%20in%20fish%20with%20different%20nutritional%20habits.%20Proteolytic%20and%20amylase%20activities&amp;author=M.C.%20Hidalgo&amp;author=E.%20Urea&amp;author=A.%20Sanz&amp;publication_year=1999&amp;journal=Aquaculture&amp;volume=170&amp;pages=267-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0044-8486(98)00413-X" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0044-8486(98)00413-X" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0044-8486(98)00413-X"> </span></div> </div></div></div></div></div><div content-id="dsy024-B96" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B96" class="ref-content "><div class="label -label">96</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lemieux</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Blier</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Dutil</div> <div class="given-names">J.D.</div></div></div> <div class="year">1999</div>, 
<div class="article-title">Do digestive enzymes set a physiological limit on growth rate and food conversion efficiency in the Atlantic cod (<em>Gadus morhua</em>)?</div>, <div class="source">Fish Physiol. Biochem</div> ., <div class="volume">20</div>, <div class="fpage">293</div>–<div class="lpage">303</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Do%20digestive%20enzymes%20set%20a%20physiological%20limit%20on%20growth%20rate%20and%20food%20conversion%20efficiency%20in%20the%20Atlantic%20cod%20(Gadus%20morhua)%3F&amp;author=H.%20Lemieux&amp;author=P.%20Blier&amp;author=J.D.%20Dutil&amp;publication_year=1999&amp;journal=Fish%20Physiol.%20Biochem&amp;volume=20&amp;pages=293-303" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1023/A:1007791019523" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1023%2FA:1007791019523" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1023%2FA:1007791019523"> </span></div> </div></div></div></div></div><div content-id="dsy024-B97" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B97" class="ref-content "><div class="label -label">97</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Eshel</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Lindner</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Smirnoff</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Newton</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Harpaz</div> <div class="given-names">S.</div></div></div> <div class="year">1993</div>, 
<div class="article-title">Comparative study of proteolytic enzymes in the digestive tracts of the european sea bass and hybrid striped bass reared in freshwater</div>, <div class="source">Comp. Biochem. Physiol. A Physiol</div> ., <div class="volume">106</div>, <div class="fpage">627</div>–<div class="lpage">34</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20study%20of%20proteolytic%20enzymes%20in%20the%20digestive%20tracts%20of%20the%20european%20sea%20bass%20and%20hybrid%20striped%20bass%20reared%20in%20freshwater&amp;author=A.%20Eshel&amp;author=P.%20Lindner&amp;author=P.%20Smirnoff&amp;author=S.%20Newton&amp;author=S.%20Harpaz&amp;publication_year=1993&amp;journal=Comp.%20Biochem.%20Physiol.%20A%20Physiol&amp;volume=106&amp;pages=627-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0300-9629(93)90371-A" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0300-9629(93)90371-A" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0300-9629(93)90371-A"> </span></div> </div></div></div></div></div><div content-id="dsy024-B98" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B98" class="ref-content "><div class="label -label">98</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Jónás</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Rágyanszki</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Oláh</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Boross</div> <div class="given-names">L.</div></div></div> <div class="year">1983</div>, 
<div class="article-title">Proteolytic digestive enzymes of carnivorous (<em>Silurus glanis</em> L.), herbivorous (<em>Hypophthalmichthys molitrix</em> Val.) and omnivorous (<em>Cyprinus carpio</em> L.) fishes</div>, <div class="source">Aquaculture</div> , <div class="volume">30</div>, <div class="fpage">145</div>–<div class="lpage">54</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Proteolytic%20digestive%20enzymes%20of%20carnivorous%20(Silurus%20glanis%20L.)%2C%20herbivorous%20(Hypophthalmichthys%20molitrix%20Val.)%20and%20omnivorous%20(Cyprinus%20carpio%20L.)%20fishes&amp;author=E.%20J%C3%B3n%C3%A1s&amp;author=M.%20R%C3%A1gyanszki&amp;author=J.%20Ol%C3%A1h&amp;author=L.%20Boross&amp;publication_year=1983&amp;journal=Aquaculture&amp;volume=30&amp;pages=145-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0044-8486(83)90158-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0044-8486(83)90158-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0044-8486(83)90158-8"> </span></div> </div></div></div></div></div><div content-id="dsy024-B99" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B99" class="ref-content "><div class="label -label">99</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chi</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Xu</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Xiao</div> <div class="given-names">Z.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">Pepsinogen A and C genes in turbot (Scophthalmus maximus): characterization and expression in early development</div>, <div class="source">Comp. Biochem. Physiol. B, Biochem. Mol. Biol</div> ., <div class="volume">165</div>, <div class="fpage">58</div>–<div class="lpage">65</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Pepsinogen%20A%20and%20C%20genes%20in%20turbot%20(Scophthalmus%20maximus)%3A%20characterization%20and%20expression%20in%20early%20development&amp;author=L.%20Chi&amp;author=S.%20Xu&amp;author=Z.%20Xiao&amp;publication_year=2013&amp;journal=Comp.%20Biochem.%20Physiol.%20B%2C%20Biochem.%20Mol.%20Biol&amp;volume=165&amp;pages=58-65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cbpb.2013.03.002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cbpb.2013.03.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cbpb.2013.03.002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23499940" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy024-B100" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B100" class="ref-content "><div class="label -label">100</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gudmundsdottir</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Palsdottir</div> <div class="given-names">H.M.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">Atlantic cod trypsins: from basic research to practical applications</div>, <div class="source">Mar. Biotechnol</div> ., <div class="volume">7</div>, <div class="fpage">77</div>–<div class="lpage">88</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Atlantic%20cod%20trypsins%3A%20from%20basic%20research%20to%20practical%20applications&amp;author=A.%20Gudmundsdottir&amp;author=H.M.%20Palsdottir&amp;publication_year=2005&amp;journal=Mar.%20Biotechnol&amp;volume=7&amp;pages=77-88" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10126-004-0061-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10126-004-0061-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10126-004-0061-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15759084" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy024-B101" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy024-B101" class="ref-content "><div class="label -label">101</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Fletcher</div> <div class="given-names">T.S.</div></div>, <div class="name string-name"><div class="surname">Alhadeff</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Craik</div> <div class="given-names">C.S.</div></div>, <div class="name string-name"><div class="surname">Largman</div> <div class="given-names">C.</div></div></div> <div class="year">1987</div>, 
<div class="article-title">Isolation and characterization of a cDNA encoding rat cationic trypsinogen</div>, <div class="source">Biochemistry</div> , <div class="volume">26</div>, <div class="fpage">3081</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Isolation%20and%20characterization%20of%20a%20cDNA%20encoding%20rat%20cationic%20trypsinogen&amp;author=T.S.%20Fletcher&amp;author=M.%20Alhadeff&amp;author=C.S.%20Craik&amp;author=C.%20Largman&amp;publication_year=1987&amp;journal=Biochemistry&amp;volume=26&amp;pages=3081-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1021/bi00385a020" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1021%2Fbi00385a020" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1021%2Fbi00385a020"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3607011" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author(s) 2018. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        
        <div class="wi-editors">
            <div class="al-authors-list">

                    <div class="editor-tag-label">
Editor:
                            <span class="al-author-name">
                                <span id="info-18743829" class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_Article_EditorInformation_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Osamu Ohara</div>
            <div class="info-card-role">
                Editor
            </div>
    </div>






        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Osamu+Ohara">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ohara O">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ohara O&quot;">Google Scholar</a>
            </div>

</div>

                                </span>
                                <a class="linked-name" href="javascript:;">Osamu Ohara</a>
                            </span>
                        <br/>
                    </div>
            </div>
        </div>

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=5059530&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy024/1/dsy024_supplementary_table_s1.xlsx?Expires=2147483647&amp;Signature=ZXQ~~2kF~81JP0zehGivG8oAMg1khl~qDbt-C-bLvEVxLxrxOJSaPIwQqPZLMOVJt4No7RRIWEOprcg2SPhdW9Ff-k7d2aWDPuSUYwC1fLZmQnARZrGpyE6kRQAC6-ZEkAOrrK6zhvWGDyS~8mYTq03Cxd26KrDq1gDa~e4VdteZ7ZVtd~f7mXw1L3dBqpeMJxqLgN9xLo~-NBpzjIhLA9T7L-4GdMe1te91hqruD41lYLuJexPncWHzVBtvW-YtB~DsmVefGsf3YJbCP0Lyw4V2bz4UyY1MJinO6jYnkqmZI8wCem~WxVS9aBTgDzrXnYknd05d~tn5Stsfj5PPMA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S1</div></a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy024/1/dsy024_supplementary_table_s2.xlsx?Expires=2147483647&amp;Signature=fRv2fXdIjcGOV0lUu4UfWW84KSrtuB-LMM349frfrVPuviBIPl2aX~5o2flP8nZd7osb1C0sdf~kMR8AE7ixasCoQAxcZzVYdq-14rHV2Rguwg9SUeUrokHoBL96NDNrLpC8SgmV6E5iD-WHxLJAKZHZPbQXEP5s0b96i1rPwl5D8IhOvY2vaZGWNsECnBEOEtdUjdYU9t6UMCfY1lJrwhlLk3NtY6PRm1E7Z51Wa2T1oWV4K~kvrKhVnbcra09KOLMKDuvPlvXbRbZOmeU64bF~47e27B1Xt3CFXcFOo9t2e~bHmJYRJYjO-kbYCAxUFL2K1VcaBZulFXeu6mtQjA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S2</div></a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy024/1/dsy024_supplementary_table_s3.xlsx?Expires=2147483647&amp;Signature=45PH9kh-uXRoTqKjAqxde9ye2hCyPt94vG5ifNzatLVuCUGjYqdfM9hEGa3pqdSa3nG7He10Ct-13XuLt4HAM9EGun~AXH6I00ygGZjOmxO7jh-9NWBMTN6JpUwG9ANRFploi12OSm35dxOMiDYljkD4KQ5nKHaJlYUVN1zWoSGVaWug2kzwn14g9EaRSNvOd506AzjBwSYzhYzBfHkyy7FFyS2X1JIcmx0Wzps7NQNqPHBY2MpmD5ZZahNJi37ss2bKr8siB~xyzXP7ZUcz00minundXllJoHczXyRJSGS7Cmg~btvXSIuS-gplKa1RU1jF4tNnOuh4TGzqf7vP~g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S3</div></a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy024/1/dsy024_supplementary_table_s4.xlsx?Expires=2147483647&amp;Signature=S62dCvdyf6CiYkS4Obtb-LQjhorJVtDjwzDv~qWl9Dc3PFwrcUrxFjltK4-5OxVG2naeBupJVaVrtnzZbqULbnZkCYptSehiwx9~5eIL6~CxtqE7cAWpJiYKEVOhxLxB8v9f9LVHjqe9S1ZOc11zpub1hcuCvbTq9pfYBYz1x4RoeMHC~mbKG2apyZP1s07Ml15Aq8AspZmvGWQ3ow0ssIrj4ZCwoTRHFfITN6AhcpHdL8qFbvRoOpOIVlo3vaCYHkHK7h178o4LATWE8InMRdWV5tcHpRNxQzOXyxDvi6-3LLC9lthMZ6ImmiFtHYqNKcy3ftPQYVfHKK2j8EI2eA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S4</div></a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy024/1/dsy024_supplementary_table_s5.xlsx?Expires=2147483647&amp;Signature=qj4p1YvFQwJIr4-aMiLTNSv1i7~sfNo~Pw3CJ9WTXq017Lp7xJqjexN83lK1IcUHC50s46eRih-uANzNwxuWw91sh4-i2Bx6WMNtyozXgt7g1NY6PiUoDnb2fxxTst4mk2FiP4una5Z4OqxZFpVOxVJAuWVmBSUuXUgIAalPOjW8dwxdm8FU0QZP-bfM~dpnBBVJiJpBIqoSO-jiX~pJhTU-mN4rBJtF6vRReJdsIebnc1myApt0J7n96vf6wAgsLO3np-va6~ELqLkZpv3H8WwLLme7jZOtrmhe4eRhXsqo6RcJCkk-qWxj7XAe4EcnTYnqVwdH0O6jAtoRmpVzsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S5</div></a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy024/1/dsy024_supplementary_table_s6.xlsx?Expires=2147483647&amp;Signature=EBZiyUt24oYGvcwgZsxMQCd5FFkhgsmq2IsB~6mizLvynPiqk1Iv3UbMKtJ~nUEmNhqqQ8V1i6EsiGNcRqxdAbsNSKHVuDqi0WM6aVNm9RXNI~RObHfVmmRQ21Q0~wY8EdKIu9vPLY5JLRU5vFI3Pw8k4maNvyHj-Uy5xxEqQhZtKyWq-0rppLdN-VoqAR5-opYae3bapWTkdwld0nSIsi4tTeF~Sohtf~Q-Di6bWxYbIRWToOAZSMyOya29iIRr7aj2FBob4nepbqz9owHOAegySCW4baxYTFhn0z27jPL8VUufPNLuT5GzvIpeqTdAOnTm42setw1gMzNQyztSzA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S6</div></a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy024/1/dsy024_supplementary_table_s7.xlsx?Expires=2147483647&amp;Signature=B-UKJRuFk0CuOM1vqtJ8PJp4rsuY~rLtnkKLUuj02ZVm20F1H2JGCIpIvQW~EVHanNj3yexU-CI0nXMi7IezUHO0T474OIANzB47N9M6JThXj1II108-PCs2wVC3S3Jz9p6v6Qh7DhV1-Hp~Wlj1slC6NHfKCrDg5hp93hCzuKg0YyjY4zH7Zr5QLUtVoNaAH1Mcgmzo~yjA5DllrMCMvMFJlBYhjTACrKbpfzMKKJH8ybDnxahUwpTwGN175ncaVGdlVUexwOY5fNBHk6vajnotXmLvWcHmgaKZYVZsgB4vRIaMLPSGnulF70Sy4z-kwk~5whRHvp2qMaJ64JmNWg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S7</div></a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy024/1/dsy024_supplementary_table_s8.xlsx?Expires=2147483647&amp;Signature=fpvj~xNmoLTfu4V4OZ9dRScoQVLIdGIkIpUMJZMziBQ9YSdhYIrQ4JItn5xBbjqF26bcMBXaNqBZuTooHTnXAA9SM91bVW1WeijhlLfIEZyL6Jq14daOOEjSXIY3pRhSak5BqZcfkjCuEUa8-dF3bGwxquWe-4JXs9V6ge9MCo~BAOwKuKJ-WQPziF6qJDBpmAx4N0Sj2lZKjJgsLOobKyCEYiF2w~z2W10lYFi-2BJDMiHkoQsmnhcIoOMKTREDiX96CuYPftnc3NPKkBTbe8JbuPUcEEmwzkXLFNNc5fzEJCU1s6mH7ytiDXuEGZCf03rWwORVEysSVqXFUAUMhw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S8</div></a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy024/1/dsy024_supplementary_table_s9.xlsx?Expires=2147483647&amp;Signature=cS0UbfjEUPaQI9g9qIXizrNQIAyrq-dV2Zk2jBeWme8fFa6OOLe7rDIcbOlY6swrEGACl~L4QA1290oq3diNrpdN~0zWkAdYBhgM3fLMYNYQUYxE2AwIg2POSswYsZShUhcw07h6EFZvaxJyACX8By7ozBSrHKfqRv0LH58G1Fa3-CgOxBLG3TvDtyap1EC511WdKP7g1v~km9IQhxUJ~qq7hH2EV4pEyvsHGwokCYNlZ4wBihbUxJVROq7wT7jXNsM2aNkiGHXbJ8tgaND5BtAqU0xD5lorp7gpWvuXGt1la2i~7PGq9SIhCcOZrZlHzx~FQx5b0NPFyi28r9FO5g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S9</div></a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy024/1/dsy024_supplementary_fig_s1.pdf?Expires=2147483647&amp;Signature=P98bjJmlGdMLt4aSAVuqwH1z4EoLiwhGB-qK2W~-DT1it4BvpzxEdB9xdGtSuovXUG-U0x6pVosg-MAmHMFNXogIVXdMEs875iCE8ZwdJNflwm2oj8pG~SPUm3HNH7W4OvWmPq-bbVWh0UkEb4INgTSWL3dK4q0bC4oI9x8hPAfWCwQY9vq6z3VaFECAnasPXxTiqb77TPaGtzSU14tJlS1ziZZ8voWHW4jLykUddT8avxqgMsSbp-WVjyNmzeZGXmJs2RcU3857u7HFXOwgmCjHLxsjfvnT6JgQyMNj3uE2zGfNYnjO20pjUe-J-4thOjQW--XbIw8xskBCFz8e8g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Figure S1</div></a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy024/1/dsy024_supplementary_fig_s2.pdf?Expires=2147483647&amp;Signature=iVkTlSCcoVqwNeU2kNLaQsq-8-~LKFbnkpJg2kuZVV9D6-~FbGHb6LdaN2CXbTA5HlctI~2Y1boxf0A1oGUEvI-PApOKQOLj9H7un1sUzz-fVnaRtVOtNbhPepCDptwq9HAqIGOdzsBnzTo1zO6-ElINKFAQWsN0Fb-8O1F19O8X572qQP~xN6~-ldPmFtG9o4WJIaJJqf0AqbTZ~oPz0xcvnHSTXIToE0c-xvGerbQy9I5Oc8lq6rGAe2ChyRyVOY8Rg~cH~QOq9NjvHWg2hMdl1xhILVm4VyZziGIii~c-Z4Cz~SdWpYoeHpJMv1GjAL-~JTOnrJ9AsTMWpmfyow__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Figure S2</div></a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy024/1/dsy024_supplementary_fig_s3.pdf?Expires=2147483647&amp;Signature=bougMt6J4RuoVZ4iaqMA9kJH87j3S0HhCgjM1Bdky0h5oELSlJRnYNV7wjUoaZwPmripKukY~7iWwF13W1km9eO8SzskxIlpkyk~Q~P9oY0gvAghF4fhwM2TdVi~1ClLPQdNkACoTBRy3oCriCnDn2ggdANaUE2RodCGAcA~qazuhx0kMhzM2C6J0~ukuPEGHbEPGiNVc6ensAFUrU7voLxidouEFul8nkfKDbVLzK76bzI0vBYUVKvSRBJYxi1-dZfJOfeoixe3-8y-Vg4GjsdJOywM827ZUXdQT764kMpKHsRAnBejNmYOGAwX4GfBGaF~mstWowY0KfNghYO~KQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Figure S3</div></a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy024/1/dsy024_supplementary_fig_s4.pdf?Expires=2147483647&amp;Signature=evZrhfEbNvNkd4Qc898wmTGyyaM3UZijdwWfrb0lgjfSL-M0LhjT5cLz9J4HkOR9rPt2L~NUv84hD5I5Xz5m-MN~7Wjxfb0bYwUydXwQXzZYXd7nm~sQzusa0KB4XLyTdTbG1O5cl0AzYDD6Way1qtRp1O0BK76Okx-tsDCfKjr0R36g5QRvQ~aKszhqe4-BFOXs2wSryt1txUs8G9Jte6mxQ7aSlkGn-QzDcsc3IpCudpB419e9-NmI2tXyI4f4pzoaKSuO3T~CmE8EuC01zmYR6cbKlyDO01MK3ha5aZ0I3lhCihFtjQJLBTsLL5Q00Moh6E8vk2YambWTFF4jdg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Figure S4</div></a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">723</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
0                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsy024"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="5059530"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f25%2f5%2f547%2f5059530">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F25%2F5%2F547%2F5059530" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/5059530'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="yellowtail (Seriola quinqueradiata) genome and transcriptome atlas of the digestive tract" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsy024");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f25%2f5%2f547%2f5059530">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

