

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" /><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info = {"beacon":"bam.nr-data.net","errorBeacon":"bam.nr-data.net","licenseKey":"8d76ca6f26","applicationID":"29565620","transactionName":"ZQBRMkVUCEtRVkINXlxKcjVnGgdKRFxVCFQcBEAWTw==","queueTime":0,"applicationTime":1885,"agent":"","atts":""}</script><script type="text/javascript">(window.NREUM||(NREUM={})).loader_config={xpid:"XAYBV15WGwEJUVdXAQMH"};window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{s.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(21),s={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(s.console=!0,o.indexOf("dev")!==-1&&(s.dev=!0),o.indexOf("nr_dev")!==-1&&(s.nrDev=!0))}catch(c){}s.nrDev&&i.on("internal-error",function(t){r(t.stack)}),s.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),s.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(s,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,s){try{l?l-=1:o(s||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,c.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:c.now();i("err",[t,n])}var i=t("handle"),a=t(22),s=t("ee"),c=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;c.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),c.xhrWrappable&&t(14),d=!0)}s.on("fn-start",function(t,e,n){d&&(l+=1)}),s.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),s.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),s.on("internal-error",function(t){i("ierr",[t,c.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){M++,N=y.hash,this[u]=g.now()}function o(){M--,y.hash!==N&&i(0,!0);var t=g.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+y,e])}function a(t,e){t.on(e,function(){this[e]=g.now()})}var s="-start",c="-end",f="-body",u="fn"+s,d="fn"+c,p="cb"+s,l="cb"+c,h="jsTime",m="fetch",v="addEventListener",w=window,y=w.location,g=t("loader");if(w[v]&&g.xhrWrappable){var b=t(10),x=t(11),E=t(8),O=t(6),P=t(13),R=t(7),T=t(14),L=t(9),j=t("ee"),S=j.get("tracer");t(15),g.features.spa=!0;var N,M=0;j.on(u,r),j.on(p,r),j.on(d,o),j.on(l,o),j.buffer([u,d,"xhr-done","xhr-resolved"]),O.buffer([u]),P.buffer(["setTimeout"+c,"clearTimeout"+s,u]),T.buffer([u,"new-xhr","send-xhr"+s]),R.buffer([m+s,m+"-done",m+f+s,m+f+c]),E.buffer(["newURL"]),b.buffer([u]),x.buffer(["propagate",p,l,"executor-err","resolve"+s]),S.buffer([u,"no-"+u]),L.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(T,"send-xhr"+s),a(j,"xhr-resolved"),a(j,"xhr-done"),a(R,m+s),a(R,m+"-done"),a(L,"new-jsonp"),a(L,"jsonp-end"),a(L,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,M>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),s=t(12),c="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",y="pushState",g=t("loader");g.features.stn=!0,t(8);var b=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof b&&(this.bstStart=g.now())}),o.on(v,function(t,e){var n=t[0];n instanceof b&&i("bst",[n,e,this.bstStart,g.now()])}),a.on(m,function(t,e,n){this.bstStart=g.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),this.bstType])}),s.on(m,function(){this.bstStart=g.now()}),s.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),"requestAnimationFrame"])}),o.on(y+l,function(t){this.time=g.now(),this.startPath=location.pathname+location.hash}),o.on(y+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+c]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+c]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+c]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){s.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),s=t(24)(a,!0),c=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=c(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?s(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=r.apply(this,arguments);return o.emit(n+"start",arguments,t),t.then(function(e){return o.emit(n+"end",[null,e],t),e},function(e){throw o.emit(n+"end",[e],t),e})})}var o=t("ee").get("fetch"),i=t(21);e.exports=o;var a=window,s="fetch-",c=s+"body-",f=["arrayBuffer","blob","json","text","formData"],u=a.Request,d=a.Response,p=a.fetch,l="prototype";u&&d&&p&&(i(f,function(t,e){r(u[l],e,c),r(d[l],e,c)}),r(a,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t(24)(r);e.exports=r,o.inPlace(window.history,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){c.emit("jsonp-error",[],p),c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=s(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),c.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function s(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var c=t("ee").get("jsonp"),f=t(24)(c);if(e.exports=c,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-"),c.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t(24)(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=s(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t(24),a=t("ee").get("promise"),s=i(a),c=t(21),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;c(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),s=f.resolve(i);return s}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),c(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=s(t[0],"resolve-",this),t[1]=s(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),s.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=s(t[0],"cb-",this),t[1]=s(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t(24)(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t(24)(i),s="setTimeout",c="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[s,"setImmediate"],s+d),a.inPlace(window,[c],c+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(c+u,r),i.on(s+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",s)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,y,"fn-",s)}function i(t){g.push(t),h&&(x?x.then(a):v?v(a):(E=-E,O.data=E))}function a(){for(var t=0;t<g.length;t++)r([],g[t]);g.length&&(g=[])}function s(t,e){return e}function c(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t(24)(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",y=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],g=[];e.exports=u;var b=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(c(l,b),b.prototype=l.prototype,d.inPlace(b.prototype,["open","send"],"-xhr-",s),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var x=m&&m.resolve();if(!v&&!m){var E=1,O=document.createTextNode(E);new h(a).observe(O,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<d;r++)t.removeEventListener(u[r],this.listener,!1);if(!e.aborted){if(n.duration=a.now()-this.startTime,4===t.readyState){e.status=t.status;var i=o(t,this.lastSize);if(i&&(n.rxSize=i),this.sameOrigin){var c=t.getResponseHeader("X-NewRelic-App-Data");c&&(e.cat=c.split(", ").pop())}}else e.status=0;n.cbTime=this.cbTime,f.emit("xhr-done",[t],t),s("xhr",[e,n,this.startTime])}}}function o(t,e){var n=t.responseType;if("json"===n&&null!==e)return e;var r="arraybuffer"===n||"blob"===n||"json"===n?t.response:t.responseText;return h(r)}function i(t,e){var n=c(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.sameOrigin=n.sameOrigin}var a=t("loader");if(a.xhrWrappable){var s=t("handle"),c=t(16),f=t("ee"),u=["load","error","abort","timeout"],d=u.length,p=t("id"),l=t(19),h=t(18),m=window.XMLHttpRequest;a.features.xhr=!0,t(14),f.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,l&&(l>34||l<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),f.on("open-xhr-start",function(t){this.params={method:t[0]},i(this,t[1]),this.metrics={}}),f.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid)}),f.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=h(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"===t.type&&(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{f.emit("internal-error",[n])}catch(r){}}};for(var s=0;s<d;s++)e.addEventListener(u[s],this.listener,!1)}),f.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),f.on("xhr-load-added",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),f.on("xhr-load-removed",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),f.on("addEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-added",[t[1],t[2]],e)}),f.on("removeEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-removed",[t[1],t[2]],e)}),f.on("fn-start",function(t,e,n){e instanceof m&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),f.on("fn-end",function(t,e){this.xhrCbStart&&f.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)})}},{}],16:[function(t,e,n){e.exports=function(t){var e=document.createElement("a"),n=window.location,r={};e.href=t,r.port=e.port;var o=e.href.split("://");!r.port&&o[1]&&(r.port=o[1].split("/")[0].split("@").pop().split(":")[1]),r.port&&"0"!==r.port||(r.port="https"===o[0]?"443":"80"),r.hostname=e.hostname||n.hostname,r.pathname=e.pathname,r.protocol=o[0],"/"!==r.pathname.charAt(0)&&(r.pathname="/"+r.pathname);var i=!e.protocol||":"===e.protocol||e.protocol===n.protocol,a=e.hostname===document.domain&&e.port===n.port;return r.sameOrigin=i&&(!e.hostname||a),r}},{}],17:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(s(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(21),s=t(22),c=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(c.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw c.emit("fn-err",[arguments,this,t],n),t}finally{c.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now()])}},{}],18:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],19:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],20:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var s=navigator.userAgent,c=s.match(a);c&&s.indexOf("Chrome")===-1&&s.indexOf("Chromium")===-1&&(o="Safari",i=c[1])}e.exports={agent:o,version:i,match:r}},{}],21:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],22:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],23:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],24:[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(22),a="nr@original",s=Object.prototype.hasOwnProperty,c=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,s,c;try{a=this,r=i(arguments),s="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],s])}u(e+"start",[r,a,o],s);try{return c=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],s),d}finally{u(e+"end",[r,a,c],s)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,s,c,f="-"===o.charAt(0);for(c=0;c<e.length;c++)s=e[c],a=t[s],r(a)||(t[s]=n(a,f?s+o:o,i,s))}function u(n,r,o){if(!c||e){var i=c;c=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}c=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)s.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?c(t,s,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),s=m(n),c=s.length,f=0;f<c;f++)s[f].apply(a,r);var d=u[g[n]];return d&&d.push([b,n,r,a]),a}}function l(t,e){y[t]=m(t).concat(e)}function h(t,e){var n=y[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return y[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",g[n]=e,e in u||(u[e]=[])})}var y={},g={},b={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return b}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var s="nr@context",c=t("gos"),f=t(21),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=x.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(g,function(e,n){t[e]||(t[e]=n)}),c("mark",["onload",a()+x.offset],null,"api");var n=l.createElement("script");n.src="https://"+t.agent,e.parentNode.insertBefore(n,e)}}function o(){"complete"===l.readyState&&i()}function i(){c("mark",["domContent",a()+x.offset],null,"api")}function a(){return O.exists&&performance.now?Math.round(performance.now()):(s=Math.max((new Date).getTime(),s))-x.offset}var s=(new Date).getTime(),c=t("handle"),f=t(21),u=t("ee"),d=t(20),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var y=""+location,g={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1099.min.js"},b=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),x=e.exports={offset:s,now:a,origin:y,features:{},xhrWrappable:b,userAgent:d};t(17),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),c("mark",["firstbyte",s],null,"api");var E=0,O=t(23)},{}]},{},["loader",2,15,5,3,4]);</script>
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/18/2/125/365741","@type":"ScholarlyArticle","name":"Prediction of Directional Changes of Influenza A Virus Genome Sequences with Emphasis on Pandemic H1N1/09 as a Model Case","datePublished":"2011-03-28","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/18/2","@type":"PublicationIssue","issueNumber":"2","datePublished":"2011-04-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsr005","keywords":["influenza virus","pandemic","self-organizing map","oligonucleotide composition","codon usage"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/18/2/125/365741","author":[{"name":"Iwasaki, Yuki","affiliation":"Nagahama Institute of Bio-Science and Technology ,  Tamura-cho 1266, Nagahama-shi, Shiga-ken 526-0829 ,  Japan","@type":"Person"},{"name":"Abe, Takashi","affiliation":"Nagahama Institute of Bio-Science and Technology ,  Tamura-cho 1266, Nagahama-shi, Shiga-ken 526-0829 ,  Japan","@type":"Person"},{"name":"Wada, Kennosuke","affiliation":"Nagahama Institute of Bio-Science and Technology ,  Tamura-cho 1266, Nagahama-shi, Shiga-ken 526-0829 ,  Japan","@type":"Person"},{"name":"Itoh, Masae","affiliation":"Nagahama Institute of Bio-Science and Technology ,  Tamura-cho 1266, Nagahama-shi, Shiga-ken 526-0829 ,  Japan","@type":"Person"},{"name":"Ikemura, Toshimichi","affiliation":"Nagahama Institute of Bio-Science and Technology ,  Tamura-cho 1266, Nagahama-shi, Shiga-ken 526-0829 ,  Japan","@type":"Person"}],"description":"Abstract.  Influenza virus poses a significant threat to public health, as exemplified by the recent introduction of the new pandemic strain H1N1/09 into human ","pageStart":"125","pageEnd":"136","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/18/2/1/m_cover.gif?Expires=2147483647&Signature=zPrik-vQsfP8tHq7QCbk4-ZAM~7ALug60QU9E2a0HTM3jWpw4mhYvM5ZjIfk8gvh0~D~8ZHzMVsbLbVV9I~imi38eK1-WbEZj8UJvCZsVN9ocsj3MzknfjxMwhJRMN7OhfXTufW~YjTAGZRrjum43uomBF~AyUhUCSuUuosVsuEI7xRAk7wYC1Ni9rgonZdcOzQdfiuitUvT5RG~wLQZYhdUWA83IS9zyFUehXeSL1v2Ik301pcwodAx4daAPn-F6iCk6UwJJGptSyDThKiltCKPrcoMzV3V1WqixkUPlkjoJdqdq8RIeTplSpeuudNseKNAwRJXHEN70rQsx2BmAw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Prediction of Directional Changes of Influenza A Virus Genome Sequences with Emphasis on Pandemic H1N1/09 as a Model Case","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/18/2/1/m_cover.gif?Expires=2147483647&Signature=zPrik-vQsfP8tHq7QCbk4-ZAM~7ALug60QU9E2a0HTM3jWpw4mhYvM5ZjIfk8gvh0~D~8ZHzMVsbLbVV9I~imi38eK1-WbEZj8UJvCZsVN9ocsj3MzknfjxMwhJRMN7OhfXTufW~YjTAGZRrjum43uomBF~AyUhUCSuUuosVsuEI7xRAk7wYC1Ni9rgonZdcOzQdfiuitUvT5RG~wLQZYhdUWA83IS9zyFUehXeSL1v2Ik301pcwodAx4daAPn-F6iCk6UwJJGptSyDThKiltCKPrcoMzV3V1WqixkUPlkjoJdqdq8RIeTplSpeuudNseKNAwRJXHEN70rQsx2BmAw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Prediction of Directional Changes of Influenza A Virus Genome Sequences with Emphasis on Pandemic H1N1/09 as a Model Case" />
    <meta property="og:description" content="Abstract.  Influenza virus poses a significant threat to public health, as exemplified by the recent introduction of the new pandemic strain H1N1/09 into human " />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/18/2/125/365741" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2011-03-28" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Iwasaki, Yuki" /><meta name="citation_author_institution" content="Nagahama Institute of Bio-Science and Technology, Tamura-cho 1266, Nagahama-shi, Shiga-ken 526-0829, Japan" /><meta name="citation_author" content="Abe, Takashi" /><meta name="citation_author_institution" content="Nagahama Institute of Bio-Science and Technology, Tamura-cho 1266, Nagahama-shi, Shiga-ken 526-0829, Japan" /><meta name="citation_author" content="Wada, Kennosuke" /><meta name="citation_author_institution" content="Nagahama Institute of Bio-Science and Technology, Tamura-cho 1266, Nagahama-shi, Shiga-ken 526-0829, Japan" /><meta name="citation_author" content="Itoh, Masae" /><meta name="citation_author_institution" content="Nagahama Institute of Bio-Science and Technology, Tamura-cho 1266, Nagahama-shi, Shiga-ken 526-0829, Japan" /><meta name="citation_author" content="Ikemura, Toshimichi" /><meta name="citation_author_institution" content="Nagahama Institute of Bio-Science and Technology, Tamura-cho 1266, Nagahama-shi, Shiga-ken 526-0829, Japan" /><meta name="citation_title" content="Prediction of Directional Changes of Influenza A Virus Genome Sequences with Emphasis on Pandemic H1N1/09 as a Model Case" /><meta name="citation_firstpage" content="125" /><meta name="citation_lastpage" content="136" /><meta name="citation_doi" content="10.1093/dnares/dsr005" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="18" /><meta name="citation_issue" content="2" /><meta name="citation_publication_date" content="2011/04/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Predicting the evolution of human influenza A; Science;  citation_year=1999;  citation_volume=286;  citation_pages=1921-5; " /><meta name="citation_reference" content="citation_title=A method for detecting positive selection at single amino acid sites; Mol. Biol. Evol.;  citation_year=1999;  citation_volume=16;  citation_pages=1315-28; " /><meta name="citation_reference" content="citation_title=Predicting the antigenic structure of the pandemic (H1N1) 2009 influenza virus hemagglutinin; PLoS One;  citation_year=2010;  citation_volume=5;  citation_pages=e8553" /><meta name="citation_reference" content="citation_publisher=Columbia University Press, New York; Molecular Evolutionary Genetics;  citation_year=1987; " /><meta name="citation_reference" content="citation_title=MEGA: a biologist-centric software for evolutionary analysis of DNA and protein sequences; Brief Bioinform.;  citation_year=2008;  citation_volume=9;  citation_pages=299-306; " /><meta name="citation_reference" content="citation_title=Informatics for unveiling hidden genome signatures; Genome Res.;  citation_year=2003;  citation_volume=13;  citation_pages=693-702; " /><meta name="citation_reference" content="citation_title=Novel phylogenetic studies of genomic sequence fragments derived from uncultured microbe mixtures in environmental and clinical samples; DNA Res.;  citation_year=2005;  citation_volume=12;  citation_pages=281-90; " /><meta name="citation_reference" content="citation_title=Self-organized formation of topologically correct feature maps; Biol. Cybern.;  citation_year=1982;  citation_volume=43;  citation_pages=59-69; " /><meta name="citation_reference" content="citation_title=Engineering applications of the self-organizing map; Proc. IEEE;  citation_year=1996;  citation_volume=84;  citation_pages=1358-84; " /><meta name="citation_reference" content="citation_title=Comparative DNA analysis across diverse genomes; Annu. Rev. Genet.;  citation_year=1998;  citation_volume=32;  citation_pages=185-225; " /><meta name="citation_reference" content="citation_title=Substrate-induced gene-expression screening of environmental metagenome libraries for isolation of catabolic genes; Nat. Biotechnol.;  citation_year=2005;  citation_volume=23;  citation_pages=88-93; " /><meta name="citation_reference" content="citation_title=Direct cloning of genes encoding novel xylanases from human gut; Can. J. Microbiol.;  citation_year=2005;  citation_volume=51;  citation_pages=251-9; " /><meta name="citation_reference" content="citation_title=The genome of Pelotomaculum thermopropionicum reveals niche-associated evolution in anaerobic microbiota; Genome Res.;  citation_year=2008;  citation_volume=18;  citation_pages=442-8; " /><meta name="citation_reference" content="citation_title=Community proteogenomics highlights microbial strain-variant protein expression within activated sludge performing enhanced biological phosphorus removal; ISME J.;  citation_year=2008;  citation_volume=2;  citation_pages=853-64; " /><meta name="citation_reference" content="citation_title=Community-wide analysis of microbial genome sequence signatures; Genome Biol.;  citation_year=2009;  citation_volume=10;  citation_pages=R85" /><meta name="citation_reference" content="citation_title=Swine influenza A (H1N1) infection in two children—South California, March–April 2009; citation_author=Centers for Disease Control and Prevention; Morb. Mortal. Wkly Rep.;  citation_year=2009;  citation_volume=58;  citation_pages=400-2; " /><meta name="citation_reference" content="citation_title=Origins and evolutionary genomics of the 2009 swine-origin H1N1 influenza A epidemic; Nature;  citation_year=2009;  citation_volume=459;  citation_pages=1122-5; " /><meta name="citation_reference" content="citation_title=Antigenic and genetic characteristics of swine-origin 2009 A (H1N1) influenza viruses circulating in humans; Science;  citation_year=2009;  citation_volume=325;  citation_pages=197-201; " /><meta name="citation_reference" content="citation_title=Genome information broker for viruses; Nucl. Acids Res.;  citation_year=2006;  citation_volume=35;  citation_pages=D339-42; " /><meta name="citation_reference" content="citation_title=The influenza virus resource at the National Center for Biotechnology Information; J. Virol.;  citation_year=2008;  citation_volume=82;  citation_pages=596-601; " /><meta name="citation_reference" content="citation_title=Analysis of codon usage diversity of bacterial genes with a self-organizing map (SOM): characterization of horizontally transferred genes with emphasis on the E. coli O157 genome; Gene;  citation_year=2001;  citation_volume=276;  citation_pages=89-99; " /><meta name="citation_reference" content="citation_title=Inhibition of interferon-mediated antiviral responses by influenza A viruses and other negative-strand RNA viruses; Virology;  citation_year=2001;  citation_volume=279;  citation_pages=375-84; " /><meta name="citation_reference" content="citation_title=Induction and suppression of RNA silencing: insights from viral infections; Nat. Rev. Genet.;  citation_year=2005;  citation_volume=6;  citation_pages=206-20; " /><meta name="citation_reference" content="citation_title=The evolution of epidemic influenza; Nat. Rev. Genet.;  citation_year=2007;  citation_volume=8;  citation_pages=196-205; " /><meta name="citation_reference" content="citation_title=Dicer is involved in protection against influenza A virus infection; J. Gen. Virol.;  citation_year=2007;  citation_volume=88;  citation_pages=2627-35; " /><meta name="citation_reference" content="citation_title=Comparison of avian and human influenza A viruses reveals a mutational bias on the viral genomes; J. Virol.;  citation_year=2006;  citation_volume=80;  citation_pages=11887-91; " /><meta name="citation_reference" content="citation_title=Codon usage bias and the evolution of influenza A viruses; BMC Evol. Biol.;  citation_year=2010;  citation_volume=10;  citation_pages=253" /><meta name="citation_reference" content="citation_title=Correlation between the abundance of Escherichia coli transfer RNAs and the occurrence of the respective codons in its protein genes; J. Mol. Biol.;  citation_year=1981;  citation_volume=146;  citation_pages=1-21; " /><meta name="citation_reference" content="citation_title=Codon usage and transfer RNA content in unicellular and multicellular organisms; Mol. Biol. Evol.;  citation_year=1985;  citation_volume=2;  citation_pages=13-34; " /><meta name="citation_reference" content="citation_title=Codon usage and genome evolution; Curr. Opin. Gen. Dev.;  citation_year=1994;  citation_volume=4;  citation_pages=851-60; " /><meta name="citation_reference" content="citation_title=Intrastrand parity rules of DNA base composition and usage biases of synonymous codons; J. Mol. Evol.;  citation_year=1995;  citation_volume=40;  citation_pages=318-25; " /><meta name="citation_reference" content="citation_title=RNA virus mutations and fitness for survival; Annu. Rev. Microbiol.;  citation_year=1997;  citation_volume=51;  citation_pages=151-78; " /><meta name="citation_reference" content="citation_title=Patterns of evolution and host gene mimicry in influenza and other RNA viruses; PLoS Pathog.;  citation_year=2008;  citation_volume=4;  citation_pages=e1000079" /><meta name="citation_reference" content="citation_title=The war against influenza: discovery and development of sialidase inhibitors; Nat. Rev. Drug Discov.;  citation_year=2007;  citation_volume=6;  citation_pages=967-74; " /><meta name="citation_reference" content="citation_title=Avian flu: isolation of drug-resistant H5N1 virus; Nature;  citation_year=2005;  citation_volume=437;  citation_pages=1108" /><meta name="citation_reference" content="citation_title=Rescue of influenza A virus from recombinant DNA; J. Virol.;  citation_year=1999;  citation_volume=73;  citation_pages=9679-82; " /><meta name="citation_reference" content="citation_title=Biological and structural characterization of a host-adapting amino acid in influenza virus; PLoS Pathog.;  citation_year=2010;  citation_volume=6;  citation_pages=e1001034" /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/18/2/125/1044075/dsr005.pdf" /><meta name="description" content="Abstract.  Influenza virus poses a significant threat to public health, as exemplified by the recent introduction of the new pandemic strain H1N1/09 into human " />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/18/2/125/365741" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/18/2/125/365741" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/18/2/125/365741");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsr005';
    </script>
<title>
	Prediction of Directional Changes of Influenza A Virus Genome Sequences with Emphasis on Pandemic H1N1/09 as a Model Case | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./365741" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BftQPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJJd2FzYWtpLCBZdWtpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTmFnYWhhbWEgSW5zdGl0dXRlIG9mIEJpby1TY2llbmNlIGFuZCBUZWNobm9sb2d5LCBUYW11cmEtY2hvIDEyNjYsIE5hZ2FoYW1hLXNoaSwgU2hpZ2Eta2VuIDUyNi0wODI5LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkFiZSwgVGFrYXNoaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5hZ2FoYW1hIEluc3RpdHV0ZSBvZiBCaW8tU2NpZW5jZSBhbmQgVGVjaG5vbG9neSwgVGFtdXJhLWNobyAxMjY2LCBOYWdhaGFtYS1zaGksIFNoaWdhLWtlbiA1MjYtMDgyOSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJXYWRhLCBLZW5ub3N1a2UiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJOYWdhaGFtYSBJbnN0aXR1dGUgb2YgQmlvLVNjaWVuY2UgYW5kIFRlY2hub2xvZ3ksIFRhbXVyYS1jaG8gMTI2NiwgTmFnYWhhbWEtc2hpLCBTaGlnYS1rZW4gNTI2LTA4MjksIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSXRvaCwgTWFzYWUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJOYWdhaGFtYSBJbnN0aXR1dGUgb2YgQmlvLVNjaWVuY2UgYW5kIFRlY2hub2xvZ3ksIFRhbXVyYS1jaG8gMTI2NiwgTmFnYWhhbWEtc2hpLCBTaGlnYS1rZW4gNTI2LTA4MjksIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSWtlbXVyYSwgVG9zaGltaWNoaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5hZ2FoYW1hIEluc3RpdHV0ZSBvZiBCaW8tU2NpZW5jZSBhbmQgVGVjaG5vbG9neSwgVGFtdXJhLWNobyAxMjY2LCBOYWdhaGFtYS1zaGksIFNoaWdhLWtlbiA1MjYtMDgyOSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IlByZWRpY3Rpb24gb2YgRGlyZWN0aW9uYWwgQ2hhbmdlcyBvZiBJbmZsdWVuemEgQSBWaXJ1cyBHZW5vbWUgU2VxdWVuY2VzIHdpdGggRW1waGFzaXMgb24gUGFuZGVtaWMgSDFOMS8wOSBhcyBhIE1vZGVsIENhc2UiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIxMjUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjEzNiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzcjAwNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIxOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDExLzA0LzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJlZGljdGluZyB0aGUgZXZvbHV0aW9uIG9mIGh1bWFuIGluZmx1ZW56YSBBOyBTY2llbmNlOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTI4NjsgIGNpdGF0aW9uX3BhZ2VzPTE5MjEtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBtZXRob2QgZm9yIGRldGVjdGluZyBwb3NpdGl2ZSBzZWxlY3Rpb24gYXQgc2luZ2xlIGFtaW5vIGFjaWQgc2l0ZXM7IE1vbC4gQmlvbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz0xMzE1LTI4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QcmVkaWN0aW5nIHRoZSBhbnRpZ2VuaWMgc3RydWN0dXJlIG9mIHRoZSBwYW5kZW1pYyAoSDFOMSkgMjAwOSBpbmZsdWVuemEgdmlydXMgaGVtYWdnbHV0aW5pbjsgUExvUyBPbmU7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPWU4NTUzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fcHVibGlzaGVyPUNvbHVtYmlhIFVuaXZlcnNpdHkgUHJlc3MsIE5ldyBZb3JrOyBNb2xlY3VsYXIgRXZvbHV0aW9uYXJ5IEdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0xOTg3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NRUdBOiBhIGJpb2xvZ2lzdC1jZW50cmljIHNvZnR3YXJlIGZvciBldm9sdXRpb25hcnkgYW5hbHlzaXMgb2YgRE5BIGFuZCBwcm90ZWluIHNlcXVlbmNlczsgQnJpZWYgQmlvaW5mb3JtLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9Mjk5LTMwNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW5mb3JtYXRpY3MgZm9yIHVudmVpbGluZyBoaWRkZW4gZ2Vub21lIHNpZ25hdHVyZXM7IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9NjkzLTcwMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Tm92ZWwgcGh5bG9nZW5ldGljIHN0dWRpZXMgb2YgZ2Vub21pYyBzZXF1ZW5jZSBmcmFnbWVudHMgZGVyaXZlZCBmcm9tIHVuY3VsdHVyZWQgbWljcm9iZSBtaXh0dXJlcyBpbiBlbnZpcm9ubWVudGFsIGFuZCBjbGluaWNhbCBzYW1wbGVzOyBETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTI4MS05MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2VsZi1vcmdhbml6ZWQgZm9ybWF0aW9uIG9mIHRvcG9sb2dpY2FsbHkgY29ycmVjdCBmZWF0dXJlIG1hcHM7IEJpb2wuIEN5YmVybi47ICBjaXRhdGlvbl95ZWFyPTE5ODI7ICBjaXRhdGlvbl92b2x1bWU9NDM7ICBjaXRhdGlvbl9wYWdlcz01OS02OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RW5naW5lZXJpbmcgYXBwbGljYXRpb25zIG9mIHRoZSBzZWxmLW9yZ2FuaXppbmcgbWFwOyBQcm9jLiBJRUVFOyAgY2l0YXRpb25feWVhcj0xOTk2OyAgY2l0YXRpb25fdm9sdW1lPTg0OyAgY2l0YXRpb25fcGFnZXM9MTM1OC04NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyYXRpdmUgRE5BIGFuYWx5c2lzIGFjcm9zcyBkaXZlcnNlIGdlbm9tZXM7IEFubnUuIFJldi4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTMyOyAgY2l0YXRpb25fcGFnZXM9MTg1LTIyNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3Vic3RyYXRlLWluZHVjZWQgZ2VuZS1leHByZXNzaW9uIHNjcmVlbmluZyBvZiBlbnZpcm9ubWVudGFsIG1ldGFnZW5vbWUgbGlicmFyaWVzIGZvciBpc29sYXRpb24gb2YgY2F0YWJvbGljIGdlbmVzOyBOYXQuIEJpb3RlY2hub2wuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTIzOyAgY2l0YXRpb25fcGFnZXM9ODgtOTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpcmVjdCBjbG9uaW5nIG9mIGdlbmVzIGVuY29kaW5nIG5vdmVsIHh5bGFuYXNlcyBmcm9tIGh1bWFuIGd1dDsgQ2FuLiBKLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTUxOyAgY2l0YXRpb25fcGFnZXM9MjUxLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBnZW5vbWUgb2YgUGVsb3RvbWFjdWx1bSB0aGVybW9wcm9waW9uaWN1bSByZXZlYWxzIG5pY2hlLWFzc29jaWF0ZWQgZXZvbHV0aW9uIGluIGFuYWVyb2JpYyBtaWNyb2Jpb3RhOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xODsgIGNpdGF0aW9uX3BhZ2VzPTQ0Mi04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21tdW5pdHkgcHJvdGVvZ2Vub21pY3MgaGlnaGxpZ2h0cyBtaWNyb2JpYWwgc3RyYWluLXZhcmlhbnQgcHJvdGVpbiBleHByZXNzaW9uIHdpdGhpbiBhY3RpdmF0ZWQgc2x1ZGdlIHBlcmZvcm1pbmcgZW5oYW5jZWQgYmlvbG9naWNhbCBwaG9zcGhvcnVzIHJlbW92YWw7IElTTUUgSi47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MjsgIGNpdGF0aW9uX3BhZ2VzPTg1My02NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tbXVuaXR5LXdpZGUgYW5hbHlzaXMgb2YgbWljcm9iaWFsIGdlbm9tZSBzZXF1ZW5jZSBzaWduYXR1cmVzOyBHZW5vbWUgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz1SODUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Td2luZSBpbmZsdWVuemEgQSAoSDFOMSkgaW5mZWN0aW9uIGluIHR3byBjaGlsZHJlbuKAlFNvdXRoIENhbGlmb3JuaWEsIE1hcmNo4oCTQXByaWwgMjAwOTsgY2l0YXRpb25fYXV0aG9yPUNlbnRlcnMgZm9yIERpc2Vhc2UgQ29udHJvbCBhbmQgUHJldmVudGlvbjsgTW9yYi4gTW9ydGFsLiBXa2x5IFJlcC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9NTg7ICBjaXRhdGlvbl9wYWdlcz00MDAtMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9T3JpZ2lucyBhbmQgZXZvbHV0aW9uYXJ5IGdlbm9taWNzIG9mIHRoZSAyMDA5IHN3aW5lLW9yaWdpbiBIMU4xIGluZmx1ZW56YSBBIGVwaWRlbWljOyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9NDU5OyAgY2l0YXRpb25fcGFnZXM9MTEyMi01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbnRpZ2VuaWMgYW5kIGdlbmV0aWMgY2hhcmFjdGVyaXN0aWNzIG9mIHN3aW5lLW9yaWdpbiAyMDA5IEEgKEgxTjEpIGluZmx1ZW56YSB2aXJ1c2VzIGNpcmN1bGF0aW5nIGluIGh1bWFuczsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0zMjU7ICBjaXRhdGlvbl9wYWdlcz0xOTctMjAxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUgaW5mb3JtYXRpb24gYnJva2VyIGZvciB2aXJ1c2VzOyBOdWNsLiBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTM1OyAgY2l0YXRpb25fcGFnZXM9RDMzOS00MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGluZmx1ZW56YSB2aXJ1cyByZXNvdXJjZSBhdCB0aGUgTmF0aW9uYWwgQ2VudGVyIGZvciBCaW90ZWNobm9sb2d5IEluZm9ybWF0aW9uOyBKLiBWaXJvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9ODI7ICBjaXRhdGlvbl9wYWdlcz01OTYtNjAxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbmFseXNpcyBvZiBjb2RvbiB1c2FnZSBkaXZlcnNpdHkgb2YgYmFjdGVyaWFsIGdlbmVzIHdpdGggYSBzZWxmLW9yZ2FuaXppbmcgbWFwIChTT00pOiBjaGFyYWN0ZXJpemF0aW9uIG9mIGhvcml6b250YWxseSB0cmFuc2ZlcnJlZCBnZW5lcyB3aXRoIGVtcGhhc2lzIG9uIHRoZSBFLiBjb2xpIE8xNTcgZ2Vub21lOyBHZW5lOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTI3NjsgIGNpdGF0aW9uX3BhZ2VzPTg5LTk5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbmhpYml0aW9uIG9mIGludGVyZmVyb24tbWVkaWF0ZWQgYW50aXZpcmFsIHJlc3BvbnNlcyBieSBpbmZsdWVuemEgQSB2aXJ1c2VzIGFuZCBvdGhlciBuZWdhdGl2ZS1zdHJhbmQgUk5BIHZpcnVzZXM7IFZpcm9sb2d5OyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTI3OTsgIGNpdGF0aW9uX3BhZ2VzPTM3NS04NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW5kdWN0aW9uIGFuZCBzdXBwcmVzc2lvbiBvZiBSTkEgc2lsZW5jaW5nOiBpbnNpZ2h0cyBmcm9tIHZpcmFsIGluZmVjdGlvbnM7IE5hdC4gUmV2LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTIwNi0yMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGV2b2x1dGlvbiBvZiBlcGlkZW1pYyBpbmZsdWVuemE7IE5hdC4gUmV2LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPTE5Ni0yMDU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpY2VyIGlzIGludm9sdmVkIGluIHByb3RlY3Rpb24gYWdhaW5zdCBpbmZsdWVuemEgQSB2aXJ1cyBpbmZlY3Rpb247IEouIEdlbi4gVmlyb2wuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTg4OyAgY2l0YXRpb25fcGFnZXM9MjYyNy0zNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyaXNvbiBvZiBhdmlhbiBhbmQgaHVtYW4gaW5mbHVlbnphIEEgdmlydXNlcyByZXZlYWxzIGEgbXV0YXRpb25hbCBiaWFzIG9uIHRoZSB2aXJhbCBnZW5vbWVzOyBKLiBWaXJvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9ODA7ICBjaXRhdGlvbl9wYWdlcz0xMTg4Ny05MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29kb24gdXNhZ2UgYmlhcyBhbmQgdGhlIGV2b2x1dGlvbiBvZiBpbmZsdWVuemEgQSB2aXJ1c2VzOyBCTUMgRXZvbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz0yNTMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db3JyZWxhdGlvbiBiZXR3ZWVuIHRoZSBhYnVuZGFuY2Ugb2YgRXNjaGVyaWNoaWEgY29saSB0cmFuc2ZlciBSTkFzIGFuZCB0aGUgb2NjdXJyZW5jZSBvZiB0aGUgcmVzcGVjdGl2ZSBjb2RvbnMgaW4gaXRzIHByb3RlaW4gZ2VuZXM7IEouIE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5ODE7ICBjaXRhdGlvbl92b2x1bWU9MTQ2OyAgY2l0YXRpb25fcGFnZXM9MS0yMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29kb24gdXNhZ2UgYW5kIHRyYW5zZmVyIFJOQSBjb250ZW50IGluIHVuaWNlbGx1bGFyIGFuZCBtdWx0aWNlbGx1bGFyIG9yZ2FuaXNtczsgTW9sLiBCaW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MTk4NTsgIGNpdGF0aW9uX3ZvbHVtZT0yOyAgY2l0YXRpb25fcGFnZXM9MTMtMzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvZG9uIHVzYWdlIGFuZCBnZW5vbWUgZXZvbHV0aW9uOyBDdXJyLiBPcGluLiBHZW4uIERldi47ICBjaXRhdGlvbl95ZWFyPTE5OTQ7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTg1MS02MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW50cmFzdHJhbmQgcGFyaXR5IHJ1bGVzIG9mIEROQSBiYXNlIGNvbXBvc2l0aW9uIGFuZCB1c2FnZSBiaWFzZXMgb2Ygc3lub255bW91cyBjb2RvbnM7IEouIE1vbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTU7ICBjaXRhdGlvbl92b2x1bWU9NDA7ICBjaXRhdGlvbl9wYWdlcz0zMTgtMjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJOQSB2aXJ1cyBtdXRhdGlvbnMgYW5kIGZpdG5lc3MgZm9yIHN1cnZpdmFsOyBBbm51LiBSZXYuIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9NTE7ICBjaXRhdGlvbl9wYWdlcz0xNTEtNzg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBhdHRlcm5zIG9mIGV2b2x1dGlvbiBhbmQgaG9zdCBnZW5lIG1pbWljcnkgaW4gaW5mbHVlbnphIGFuZCBvdGhlciBSTkEgdmlydXNlczsgUExvUyBQYXRob2cuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTQ7ICBjaXRhdGlvbl9wYWdlcz1lMTAwMDA3OSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSB3YXIgYWdhaW5zdCBpbmZsdWVuemE6IGRpc2NvdmVyeSBhbmQgZGV2ZWxvcG1lbnQgb2Ygc2lhbGlkYXNlIGluaGliaXRvcnM7IE5hdC4gUmV2LiBEcnVnIERpc2Nvdi47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTk2Ny03NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXZpYW4gZmx1OiBpc29sYXRpb24gb2YgZHJ1Zy1yZXNpc3RhbnQgSDVOMSB2aXJ1czsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTQzNzsgIGNpdGF0aW9uX3BhZ2VzPTExMDgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZXNjdWUgb2YgaW5mbHVlbnphIEEgdmlydXMgZnJvbSByZWNvbWJpbmFudCBETkE7IEouIFZpcm9sLjsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT03MzsgIGNpdGF0aW9uX3BhZ2VzPTk2NzktODI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJpb2xvZ2ljYWwgYW5kIHN0cnVjdHVyYWwgY2hhcmFjdGVyaXphdGlvbiBvZiBhIGhvc3QtYWRhcHRpbmcgYW1pbm8gYWNpZCBpbiBpbmZsdWVuemEgdmlydXM7IFBMb1MgUGF0aG9nLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9ZTEwMDEwMzQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMTgvMi8xMjUvMTA0NDA3NS9kc3IwMDUucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIEluZmx1ZW56YSB2aXJ1cyBwb3NlcyBhIHNpZ25pZmljYW50IHRocmVhdCB0byBwdWJsaWMgaGVhbHRoLCBhcyBleGVtcGxpZmllZCBieSB0aGUgcmVjZW50IGludHJvZHVjdGlvbiBvZiB0aGUgbmV3IHBhbmRlbWljIHN0cmFpbiBIMU4xLzA5IGludG8gaHVtYW4gIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFci8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj0zNjU3NDEmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2RkZplyV7gfyZucn3x8xrLc+5pKqB8=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f18%2f2%2f125%2f365741" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f18%2f2%2f125%2f365741" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=14229" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/18/2"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/18/2/1/m_cover.gif?Expires=2147483647&amp;Signature=zPrik-vQsfP8tHq7QCbk4-ZAM~7ALug60QU9E2a0HTM3jWpw4mhYvM5ZjIfk8gvh0~D~8ZHzMVsbLbVV9I~imi38eK1-WbEZj8UJvCZsVN9ocsj3MzknfjxMwhJRMN7OhfXTufW~YjTAGZRrjum43uomBF~AyUhUCSuUuosVsuEI7xRAk7wYC1Ni9rgonZdcOzQdfiuitUvT5RG~wLQZYhdUWA83IS9zyFUehXeSL1v2Ik301pcwodAx4daAPn-F6iCk6UwJJGptSyDThKiltCKPrcoMzV3V1WqixkUPlkjoJdqdq8RIeTplSpeuudNseKNAwRJXHEN70rQsx2BmAw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/18/2">
                        <div class="volume trailing-comma">Volume 18</div>
                        <div class="issue">Issue 2</div>

            </a>

        </div>
        <div class="ii-pub-date">
April 2011        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="5307166">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5307166">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5307168">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5307168">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5307171">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5307171">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5307177">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5307177">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5307220">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5307220">Supplementary Data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5307222">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5307222">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5307224">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5307224">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5307226">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5307226">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/18/2/117/365636">&lt; Previous</a>
        </li>        
            <li class="empty arrow">&nbsp;</li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Prediction of Directional Changes of Influenza A Virus Genome Sequences with Emphasis on Pandemic H1N1/09 as a Model Case
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yuki Iwasaki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">Nagahama Institute of Bio-Science and Technology</div>, <div class="addr-line">Tamura-cho 1266, Nagahama-shi, Shiga-ken 526-0829</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yuki+Iwasaki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Iwasaki Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Iwasaki Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yuki Iwasaki</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Takashi Abe</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">Nagahama Institute of Bio-Science and Technology</div>, <div class="addr-line">Tamura-cho 1266, Nagahama-shi, Shiga-ken 526-0829</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81 749-64-8126. Fax. +81 749-64-8126. Email: <a href="mailto:takaabe@nagahama-i-bio.ac.jp" target="_blank">takaabe@nagahama-i-bio.ac.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Takashi+Abe">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Abe T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Abe T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Takashi Abe</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kennosuke Wada</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">Nagahama Institute of Bio-Science and Technology</div>, <div class="addr-line">Tamura-cho 1266, Nagahama-shi, Shiga-ken 526-0829</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kennosuke+Wada">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Wada K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Wada K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Kennosuke Wada</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Masae Itoh</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">Nagahama Institute of Bio-Science and Technology</div>, <div class="addr-line">Tamura-cho 1266, Nagahama-shi, Shiga-ken 526-0829</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Masae+Itoh">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Itoh M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Itoh M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Masae Itoh</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Toshimichi Ikemura</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">Nagahama Institute of Bio-Science and Technology</div>, <div class="addr-line">Tamura-cho 1266, Nagahama-shi, Shiga-ken 526-0829</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Toshimichi+Ikemura">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ikemura T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ikemura T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Toshimichi Ikemura</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">These two authors contributed equally to this work.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Edited by Katsumi Isono</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 18, Issue 2, 1 April 2011, Pages 125–136, <a href='https://doi.org/10.1093/dnares/dsr005'>https://doi.org/10.1093/dnares/dsr005</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">28 March 2011</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">24 December 2010</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">27 February 2011</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="365741" href="/dnaresearch/article-pdf/18/2/125/1044075/dsr005.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Yuki Iwasaki, Takashi Abe, Kennosuke Wada, Masae Itoh, Toshimichi Ikemura;  Prediction of Directional Changes of Influenza A Virus Genome Sequences with Emphasis on Pandemic H1N1/09 as a Model Case, <em>DNA Research</em>, Volume 18, Issue 2, 1 April 2011, Pages 125–136, <a href='https://doi.org/10.1093/dnares/dsr005'>https://doi.org/10.1093/dnares/dsr005</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/365741?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/365741?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/365741?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/365741?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/365741?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/365741?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/365741?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Prediction%20of%20Directional%20Changes%20of%20Influenza%20A%20Virus%20Genome%20Sequences%20with%20Emphasis%20on%20Pandemic%20H1N1%2F09%20as%20a%20Model%20Case&amp;publicationDate=2011-03-28&amp;volumeNum=18&amp;issueNum=2&amp;author=Iwasaki%2C%20Yuki%3B%20Abe%2C%20Takashi&amp;startPage=125&amp;endPage=136&amp;contentId=10.1093%2Fdnares%2Fdsr005&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f18%2f2%2f125%2f365741" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=5307166 id="5307166" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Influenza virus poses a significant threat to public health, as exemplified by the recent introduction of the new pandemic strain H1N1/09 into human populations. Pandemics have been initiated by the occurrence of novel changes in animal sources that eventually adapt to human. One important issue in studies of viral genomes, particularly those of influenza virus, is to predict possible changes in genomic sequence that will become hazardous. We previously established a clustering method termed ‘BLSOM’ (batch-learning self-organizing map) that does not depend on sequence alignment and can characterize and compare even 1 million genomic sequences in one run. Strategies for comparing a vast number of genomic sequences simultaneously become increasingly important in genome studies because of remarkable progresses in nucleotide sequencing. In this study, we have constructed BLSOMs based on the oligonucleotide and codon composition of all influenza A viral strains available. Without prior information with regard to their hosts, sequences derived from strains isolated from avian or human sources were successfully clustered according to the hosts. Notably, the pandemic H1N1/09 strains have oligonucleotide and codon compositions that are clearly different from those of human seasonal influenza A strains. This enables us to infer future directional changes in the influenza A viral genome.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;influenza virus&quot;">influenza virus</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="pandemic">pandemic</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;self-organizing map&quot;">self-organizing map</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;oligonucleotide composition&quot;">oligonucleotide composition</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;codon usage&quot;">codon usage</a></div>                            <h2 scrollto-destination=5307168 id="5307168" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>One important issue for bioinformatics studies of virus genomes, especially of influenza viruses, is to develop a strategy to predict genome sequence changes that will become hazardous.<sup><a reveal-id="DSR005C1  DSR005C3" data-open="DSR005C1  DSR005C3" class="link link-ref link-reveal xref-bibr">1–3</a></sup> The phylogenetic analysis based on sequence homology searches is a well-established and an irreplaceably important method for studying genomic sequences.<sup><a reveal-id="DSR005C4" data-open="DSR005C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DSR005C5" data-open="DSR005C5" class="link link-ref link-reveal xref-bibr">5</a></sup> However, it inevitably depends on alignments of sequences, which is potentially error-prone and troublesome especially for distantly related sequences. This difficulty becomes increasingly evident as the number of sequences obtained from a wide range of species, including novel species, increases dramatically because of the remarkable progress of the high-throughput DNA sequencing methods. To address the difficulty and complement the sequence homology searches, we previously established an alignment-free clustering method BLSOM (batch-learning self-organizing map)<sup><a reveal-id="DSR005C6" data-open="DSR005C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DSR005C7" data-open="DSR005C7" class="link link-ref link-reveal xref-bibr">7</a></sup> that can analyse a million sequences simultaneously on the basis of SOM originally developed by Kohonen and his colleagues.<sup><a reveal-id="DSR005C8" data-open="DSR005C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSR005C9" data-open="DSR005C9" class="link link-ref link-reveal xref-bibr">9</a></sup> SOM is a powerful unsupervised clustering method that provides an efficient interpretation of complex data, using visualization on one map. Unlike the SOM, the BLSOM depends on neither the data-input order nor the initial conditions, and therefore, is suitable for studying genomic sequences.</p><p>G + C% has been used for a long period as a fundamental parameter for phylogenetic classification of microbial genomes, including viral genomes, but the G + C% is apparently too simple a parameter to differentiate a wide variety of known microbial genomes. Oligonucleotide composition, on the other hand, can be used to distinguish the species even with the same G + C%, because the oligonucleotide composition varies significantly among the genomes and is called ‘genome signature’.<sup><a reveal-id="DSR005C10" data-open="DSR005C10" class="link link-ref link-reveal xref-bibr">10</a></sup> When we constructed a BLSOM for oligonucleotide frequencies in fragment sequences (e.g. 10 kb) from a wide variety of microbial species, sequences were clustered (self-organized) according to species. BLSOMs could recognize and visualize species-specific characteristics of oligonucleotide composition.<sup><a reveal-id="DSR005C6" data-open="DSR005C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DSR005C7" data-open="DSR005C7" class="link link-ref link-reveal xref-bibr">7</a></sup> The alignment-free clustering methods, BLSOM<sup><a reveal-id="DSR005C11  DSR005C13" data-open="DSR005C11  DSR005C13" class="link link-ref link-reveal xref-bibr">11–13</a></sup> and SOM,<sup><a reveal-id="DSR005C14" data-open="DSR005C14" class="link link-ref link-reveal xref-bibr">14</a>,<a reveal-id="DSR005C15" data-open="DSR005C15" class="link link-ref link-reveal xref-bibr">15</a></sup> have been successfully applied to phylogenetic classification of a large number of microbial sequences obtained by metagenome studies of environmental and clinical samples. Here, we have analysed influenza A viruses, including those of the pandemic H1N1/09,<sup><a reveal-id="DSR005C16  DSR005C18" data-open="DSR005C16  DSR005C18" class="link link-ref link-reveal xref-bibr">16–18</a></sup> with BLSOM and developed a strategy for predicting directional sequence changes in influenza A viruses. This strategy should be widely applicable for other zoonotic viruses. Introduction of BLSOM into viral genome studies can undoubtedly provide a powerful tool for efficiently extracting profound knowledge from a vast number of viral genomic sequences obtained by high-throughput sequencers currently available.</p>                            <h2 scrollto-destination=5307171 id="5307171" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=5307172 id="5307172" class="section-title" data-legacy-id=s2a>2.1. Viral genome sequences</h3>
<p>A total of 43 831 virus sequences analysed in Fig. <a reveal-id="DSR005F1" data-open="DSR005F1" class="link link-reveal link-table xref-fig">1</a>A were obtained from the DDBJ Genome Information Broker for Viruses (GIB-V; <a class="link link-uri" href="http://gib-v.genes.nig.ac.jp/">http://gib-v.genes.nig.ac.jp/</a>),<sup><a reveal-id="DSR005C19" data-open="DSR005C19" class="link link-ref link-reveal xref-bibr">19</a></sup> and a total of 59 512 segment sequences derived from 7439 influenza A virus strains were obtained from the NCBI Influenza Virus Resource (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/genomes/FLU/">http://www.ncbi.nlm.nih.gov/genomes/FLU/</a>).<sup><a reveal-id="DSR005C20" data-open="DSR005C20" class="link link-ref link-reveal xref-bibr">20</a></sup></p>                        <a id="5307174" scrollto-destination="5307174"></a>
<div id="DSR005F1" data-id="DSR005F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/2/10.1093/dnares/dsr005/2/m_dsr00501.jpeg?Expires=2147483647&Signature=OwL7ip9em6qdi-q3R3Y65kxw1SD8or3b1eJUkh331FLwXjAugjmdz-tLiUZCRtPr0-GgjS1tpYXScE559B0ZgwE9k~5Wm4f4-~~uAazu03gumR0ACUiD1stZS7XmH2AlBwN93EUntUngwUlsU36jM88Xxx8KxJFmZVdTet4kPPKytregM2j2XEhbxPlE8b6CZizmFO6GZx3VOMIwnHxy-I3M5N8qGPfpRjlpErG1PoOEkwnysfX~hwEw-76Z5ohU83Jk7TBoLhSJoaCO1dRHBxy769ufZyNvGUIOZz66MwL79TRINcVCWS0cc4q8TActw1qMOA1hl00kHWEe~lFdXw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Oligonucleotide-BLSOMs for virus genome sequences. (A) BLSOM was constructed for tri- or tetranucleotide composition (Tri or Tetra) in 1- or 0.5-kb fragment sequences from 42 957 virus genomes. Lattice points that include sequences from more than one phylogenetic family were indicated in black, and those containing sequences from a single phylotype were indicated in a colour representing the phylotype described by the DDBJ GIB-V (see the legend of Supplementary Fig. S1 for the phylotypes). (B) Influenza Di, Tri, or Tetra: di-, tri-, or tetranucleotide BLSOM was constructed for 5350 strains of influenza A virus including 167 H1N1/09 strains isolated in the early stage (April and May) of the 2009 pandemic. Lattice points containing sequences from strains isolated from more than one host were indicated in black, and those containing sequences from one host were indicated in a colour representing the host shown at the bottom of (B): avian (1948 strains), human (2788 strains), swine (249 strains), equine (68 strains), and other hosts (130 strains); human H1N1/09 (167 strains) was separately coloured and indicated with an arrow. (C) Tetra Hum Sub (human subtype): on the tetra-BLSOM presented in (B), each human virus subtype was specified in a colour representing the subtype (H1N1, H1N1/09, H2N2, H3N2, H5N1, H7, H9) shown at the bottom of Tetra Hum Sub, and territories of other hosts were achromatic. Avi Sub (avian subtype): each avian subtype was specified in a colour representing the subtype (H5N1, H5N2, H6N1, H7N2, H9N2, others) shown at the bottom of Avi Sub, and territories of other hosts were achromatic. The zone representing the minor human territory representing H1N1/09 (achromatic) was indicated with an arrow to help recognize its position for reference. Avi Sub near Hum Sub : avian subtype strains that were in a close proximity to human, swine, or pandemic H1N1/09 territory were specified by a colour representing both the subtype and the geographical information of isolation (H5N1 isolated in Russia, H5N2 in Minnesota (MN), H6N1 in Taiwan, H7N2 in New York (NY), H9N2 in Israel, other avian strains) shown at the bottom of Avi Sub near Hum Sub." path-from-xml="dsr00501.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5307174/dsr00501.jpeg" path-from-xml="dsr00501.jpeg" target="_blank">View large</a><a section="5307174" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/2/10.1093/dnares/dsr005/2/dsr00501.jpeg?Expires=2147483647&Signature=GGCwy4UZCcTMcxh~b9ldzyWl8KOjgvcDe6pXWczIpQxzR-qZR98S4me6eBS-I3sh0z2ys5uEUHrFe52pyg4uKxVNI167xjv--ZZxREDZc29mkzJG17sbW0sKCArAmTm~dOEn3CT3SIKSzKr9jA8lH9O8RgR2szsTTHiBgG7Fqp8rmqdyeEsE33GVI0eD7YfYHHkYEVp3Hqzo0Qwxq6Ui4f8lcaUM0KnTQUT9psDQGT3hxLgvSomcQXSTTbIgIh40xsnBL-QEEgQxtyEW5bu2eytCdNs8goZEpsRg1lBgTzsyjh7Qi0Hkn4b0tsyeZeC3N2GtjZIhgyw6YYKRXxUOZg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5307174&ar=365741&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr00501.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Oligonucleotide-BLSOMs for virus genome sequences. (A) BLSOM was constructed for tri- or tetranucleotide composition (Tri or Tetra) in 1- or 0.5-kb fragment sequences from 42 957 virus genomes. Lattice points that include sequences from more than one phylogenetic family were indicated in black, and those containing sequences from a single phylotype were indicated in a colour representing the phylotype described by the DDBJ GIB-V (see the legend of <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a> for the phylotypes). (B) Influenza Di, Tri, or Tetra: di-, tri-, or tetranucleotide BLSOM was constructed for 5350 strains of influenza A virus including 167 H1N1/09 strains isolated in the early stage (April and May) of the 2009 pandemic. Lattice points containing sequences from strains isolated from more than one host were indicated in black, and those containing sequences from one host were indicated in a colour representing the host shown at the bottom of (B): avian (1948 strains), human (2788 strains), swine (249 strains), equine (68 strains), and other hosts (130 strains); human H1N1/09 (167 strains) was separately coloured and indicated with an arrow. (C) Tetra Hum Sub (human subtype): on the tetra-BLSOM presented in (B), each human virus subtype was specified in a colour representing the subtype (H1N1, H1N1/09, H2N2, H3N2, H5N1, H7, H9) shown at the bottom of Tetra Hum Sub, and territories of other hosts were achromatic. Avi Sub (avian subtype): each avian subtype was specified in a colour representing the subtype (H5N1, H5N2, H6N1, H7N2, H9N2, others) shown at the bottom of Avi Sub, and territories of other hosts were achromatic. The zone representing the minor human territory representing H1N1/09 (achromatic) was indicated with an arrow to help recognize its position for reference. Avi Sub near Hum Sub : avian subtype strains that were in a close proximity to human, swine, or pandemic H1N1/09 territory were specified by a colour representing both the subtype and the geographical information of isolation (H5N1 isolated in Russia, H5N2 in Minnesota (MN), H6N1 in Taiwan, H7N2 in New York (NY), H9N2 in Israel, other avian strains) shown at the bottom of Avi Sub near Hum Sub.</p></div></div><div content-id="DSR005F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/2/10.1093/dnares/dsr005/2/m_dsr00501.jpeg?Expires=2147483647&Signature=OwL7ip9em6qdi-q3R3Y65kxw1SD8or3b1eJUkh331FLwXjAugjmdz-tLiUZCRtPr0-GgjS1tpYXScE559B0ZgwE9k~5Wm4f4-~~uAazu03gumR0ACUiD1stZS7XmH2AlBwN93EUntUngwUlsU36jM88Xxx8KxJFmZVdTet4kPPKytregM2j2XEhbxPlE8b6CZizmFO6GZx3VOMIwnHxy-I3M5N8qGPfpRjlpErG1PoOEkwnysfX~hwEw-76Z5ohU83Jk7TBoLhSJoaCO1dRHBxy769ufZyNvGUIOZz66MwL79TRINcVCWS0cc4q8TActw1qMOA1hl00kHWEe~lFdXw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Oligonucleotide-BLSOMs for virus genome sequences. (A) BLSOM was constructed for tri- or tetranucleotide composition (Tri or Tetra) in 1- or 0.5-kb fragment sequences from 42 957 virus genomes. Lattice points that include sequences from more than one phylogenetic family were indicated in black, and those containing sequences from a single phylotype were indicated in a colour representing the phylotype described by the DDBJ GIB-V (see the legend of Supplementary Fig. S1 for the phylotypes). (B) Influenza Di, Tri, or Tetra: di-, tri-, or tetranucleotide BLSOM was constructed for 5350 strains of influenza A virus including 167 H1N1/09 strains isolated in the early stage (April and May) of the 2009 pandemic. Lattice points containing sequences from strains isolated from more than one host were indicated in black, and those containing sequences from one host were indicated in a colour representing the host shown at the bottom of (B): avian (1948 strains), human (2788 strains), swine (249 strains), equine (68 strains), and other hosts (130 strains); human H1N1/09 (167 strains) was separately coloured and indicated with an arrow. (C) Tetra Hum Sub (human subtype): on the tetra-BLSOM presented in (B), each human virus subtype was specified in a colour representing the subtype (H1N1, H1N1/09, H2N2, H3N2, H5N1, H7, H9) shown at the bottom of Tetra Hum Sub, and territories of other hosts were achromatic. Avi Sub (avian subtype): each avian subtype was specified in a colour representing the subtype (H5N1, H5N2, H6N1, H7N2, H9N2, others) shown at the bottom of Avi Sub, and territories of other hosts were achromatic. The zone representing the minor human territory representing H1N1/09 (achromatic) was indicated with an arrow to help recognize its position for reference. Avi Sub near Hum Sub : avian subtype strains that were in a close proximity to human, swine, or pandemic H1N1/09 territory were specified by a colour representing both the subtype and the geographical information of isolation (H5N1 isolated in Russia, H5N2 in Minnesota (MN), H6N1 in Taiwan, H7N2 in New York (NY), H9N2 in Israel, other avian strains) shown at the bottom of Avi Sub near Hum Sub." path-from-xml="dsr00501.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5307174/dsr00501.jpeg" path-from-xml="dsr00501.jpeg" target="_blank">View large</a><a section="5307174" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/2/10.1093/dnares/dsr005/2/dsr00501.jpeg?Expires=2147483647&Signature=GGCwy4UZCcTMcxh~b9ldzyWl8KOjgvcDe6pXWczIpQxzR-qZR98S4me6eBS-I3sh0z2ys5uEUHrFe52pyg4uKxVNI167xjv--ZZxREDZc29mkzJG17sbW0sKCArAmTm~dOEn3CT3SIKSzKr9jA8lH9O8RgR2szsTTHiBgG7Fqp8rmqdyeEsE33GVI0eD7YfYHHkYEVp3Hqzo0Qwxq6Ui4f8lcaUM0KnTQUT9psDQGT3hxLgvSomcQXSTTbIgIh40xsnBL-QEEgQxtyEW5bu2eytCdNs8goZEpsRg1lBgTzsyjh7Qi0Hkn4b0tsyeZeC3N2GtjZIhgyw6YYKRXxUOZg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5307174&ar=365741&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr00501.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Oligonucleotide-BLSOMs for virus genome sequences. (A) BLSOM was constructed for tri- or tetranucleotide composition (Tri or Tetra) in 1- or 0.5-kb fragment sequences from 42 957 virus genomes. Lattice points that include sequences from more than one phylogenetic family were indicated in black, and those containing sequences from a single phylotype were indicated in a colour representing the phylotype described by the DDBJ GIB-V (see the legend of <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a> for the phylotypes). (B) Influenza Di, Tri, or Tetra: di-, tri-, or tetranucleotide BLSOM was constructed for 5350 strains of influenza A virus including 167 H1N1/09 strains isolated in the early stage (April and May) of the 2009 pandemic. Lattice points containing sequences from strains isolated from more than one host were indicated in black, and those containing sequences from one host were indicated in a colour representing the host shown at the bottom of (B): avian (1948 strains), human (2788 strains), swine (249 strains), equine (68 strains), and other hosts (130 strains); human H1N1/09 (167 strains) was separately coloured and indicated with an arrow. (C) Tetra Hum Sub (human subtype): on the tetra-BLSOM presented in (B), each human virus subtype was specified in a colour representing the subtype (H1N1, H1N1/09, H2N2, H3N2, H5N1, H7, H9) shown at the bottom of Tetra Hum Sub, and territories of other hosts were achromatic. Avi Sub (avian subtype): each avian subtype was specified in a colour representing the subtype (H5N1, H5N2, H6N1, H7N2, H9N2, others) shown at the bottom of Avi Sub, and territories of other hosts were achromatic. The zone representing the minor human territory representing H1N1/09 (achromatic) was indicated with an arrow to help recognize its position for reference. Avi Sub near Hum Sub : avian subtype strains that were in a close proximity to human, swine, or pandemic H1N1/09 territory were specified by a colour representing both the subtype and the geographical information of isolation (H5N1 isolated in Russia, H5N2 in Minnesota (MN), H6N1 in Taiwan, H7N2 in New York (NY), H9N2 in Israel, other avian strains) shown at the bottom of Avi Sub near Hum Sub.</p></div></div>                            <h3 scrollto-destination=5307175 id="5307175" class="section-title" data-legacy-id=s2b>2.2. Batch-learning self-organizing map</h3>
<p>SOM is an unsupervised neural network algorithm that implements a characteristic non-linear projection from the high-dimensional space of input data onto a two-dimensional array of weight vectors.<sup><a reveal-id="DSR005C8" data-open="DSR005C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSR005C9" data-open="DSR005C9" class="link link-ref link-reveal xref-bibr">9</a></sup> We modified the conventional SOM for genome informatics to make the learning process and resulting map independent of the order of data input, and established a BLSOM.<sup><a reveal-id="DSR005C6" data-open="DSR005C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DSR005C7" data-open="DSR005C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DSR005C21" data-open="DSR005C21" class="link link-ref link-reveal xref-bibr">21</a></sup> The initial weight vectors were defined by principal component analysis instead of random values. BLSOM learning for oligonucleotide composition was conducted as described previously.<sup><a reveal-id="DSR005C6" data-open="DSR005C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DSR005C7" data-open="DSR005C7" class="link link-ref link-reveal xref-bibr">7</a></sup> BLSOM learning for synonymous codon usage and visualization of diagnostic codons for the category separation were conducted as described by Kanaya <em>et al</em>.<sup><a reveal-id="DSR005C21" data-open="DSR005C21" class="link link-ref link-reveal xref-bibr">21</a></sup> BLSOM program was obtained from UNTROD, Inc. (<a href="mailto:y_wada@nagahama-i-bio.ac.jp" target="_blank">y_wada@nagahama-i-bio.ac.jp</a>).</p>                            <h2 scrollto-destination=5307177 id="5307177" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=5307178 id="5307178" class="section-title" data-legacy-id=s3a>3.1. Oligonucleotide BLSOM for all virus genome sequences</h3>
<p>To test the clustering powers of BLSOM for large numbers of sequences from a wide variety of virus genomes, we initially constructed BLSOM with tri- and tetranucleotide frequencies in all 1-kb fragment sequences (ca. 200 000 sequences) from 42 957 virus genomes, which have been compiled and classified into 79 phylogenetic families by DDBJ GIB-V<sup><a reveal-id="DSR005C19" data-open="DSR005C19" class="link link-ref link-reveal xref-bibr">19</a></sup> (Fig. <a reveal-id="DSR005F1" data-open="DSR005F1" class="link link-reveal link-table xref-fig">1</a>A). Lattice points that contained sequences from one phylogenetic family are indicated in colour, and those that included sequences from more than one family are indicated in black. A major portion was coloured, showing a major portion of sequences to be self-organized according to phylotype; ∼80 and 86% of lattice points had sequences from one phylotype on tri- and tetra-BLSOMs, respectively. Notably, no information in regard to phylotype was given during the BLSOM calculation. Tetra-BLSOM for 0.5-kb sequences (ca. 4 000 000 sequences) also showed a clear clustering according to phylotype, with a slight reduction (∼5%) in the separation (Fig. <a reveal-id="DSR005F1" data-open="DSR005F1" class="link link-reveal link-table xref-fig">1</a>A); tri-BLSOM for 0.5-kb sequences is presented in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>.</p>                            <h3 scrollto-destination=5307180 id="5307180" class="section-title" data-legacy-id=s3b>3.2. Oligonucleotide BLSOM for influenza A virus genomes</h3>
<p>We next analysed influenza A virus sequences available from Influenza Virus Resource<sup><a reveal-id="DSR005C20" data-open="DSR005C20" class="link link-ref link-reveal xref-bibr">20</a></sup> in NCBI. Influenza A virus genome is composed of eight segments, each of which encodes primarily one protein. Genome sequences from a total of approximately 7400 strains were available even when we focused on strains for which all eight segments were sequenced, and approximately 2300 strains corresponded to the new pandemic H1N1/09. The direct target of natural selection is a virion containing a full set of the eight genome segments, and analyses at the genome level should provide valuable information for characterizing individual strains. Viruses are inevitably dependent on many host factors for their growth (e.g. pools of nucleotides, amino acids and tRNAs), and at the same time have to escape from antiviral host mechanisms such as antibodies, cytotoxic T cells, interferons, and RNA interferences.<sup><a reveal-id="DSR005C22  DSR005C25" data-open="DSR005C22  DSR005C25" class="link link-ref link-reveal xref-bibr">22–25</a></sup> Actually, mononucleotide compositions and codon usage biases in influenza A virus genomes were shown to differ between strains isolated from human and avian sources.<sup><a reveal-id="DSR005C26" data-open="DSR005C26" class="link link-ref link-reveal xref-bibr">26</a>,<a reveal-id="DSR005C27" data-open="DSR005C27" class="link link-ref link-reveal xref-bibr">27</a></sup></p><p>To clarify the host-dependent characteristics of influenza A virus sequences with BLSOM, di-, tri-, or tetranucleotide frequencies in eight genome segments of virus strains were summed up for each strain, and BLSOM was constructed with the summed frequency for each strain. The merit of the present sequence data set of viral genomes was to include those from the H1N1/09 strains isolated in an early stage of the new pandemic, which was caused by a new virus invasion from other hosts. To specifically study sequence characteristics of the strains isolated in the early stages, we first analysed the data set only including approximately 170 H1N1/09 strains that were isolated from April to May in 2009 (Fig. <a reveal-id="DSR005F1" data-open="DSR005F1" class="link link-reveal link-table xref-fig">1</a>B). Lattice points containing virus strains isolated from one host species were indicated in a colour representing the host and those containing strains isolated from more than one host were in black. Without information with regard to the host during the BLSOM calculation, strains isolated from avian (red) or human (light green) were clustered, forming a large contiguous territory on each BLSOM. This showed host-specific characteristics of oligonucleotide composition and was consistent with the observation of Rabadan <em>et al</em>.<sup><a reveal-id="DSR005C2" data-open="DSR005C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DSR005C6" data-open="DSR005C6" class="link link-ref link-reveal xref-bibr">6</a></sup> that mononucleotide compositions in influenza A virus genes differed between strains isolated from human and avian sources.</p><p>Notably, a minor human territory (dark green and indicated by an arrow) appeared, which was separated from the major human territory representing human seasonal strains and surrounded by avian, equine, and swine territories on each BLSOM. This minor human territory was composed of the pandemic H1N1/09 strains, which have resulted from genetic re-assortment between the recently circulating swine H1 viruses in North America and the avian-like swine viruses in Europe.<sup><a reveal-id="DSR005C17" data-open="DSR005C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DSR005C18" data-open="DSR005C18" class="link link-ref link-reveal xref-bibr">18</a></sup></p><p>To further investigate human virus subtypes other than the H1N1/09, lattice points that contained human seasonal viruses of one subtype on the tetra-BLSOM (Tetra in Fig. <a reveal-id="DSR005F1" data-open="DSR005F1" class="link link-reveal link-table xref-fig">1</a>B) were specified with one colour representing the subtype (Hum Sub in Fig. <a reveal-id="DSR005F1" data-open="DSR005F1" class="link link-reveal link-table xref-fig">1</a>C). Human seasonal H1N1 (light blue) and H3N2 (dark blue) were clearly separated from each other. In contrast to the compact minor territory of H1N1/09 (dark green, arrowed), human H5N1 strains (red) were rather scattered within the avian territory (achromatic in Hum Sub in Fig. <a reveal-id="DSR005F1" data-open="DSR005F1" class="link link-reveal link-table xref-fig">1</a>C). This should show that these human H5N1 strains jumped to humans but were not able to spread from human to human,<sup><a reveal-id="DSR005C24" data-open="DSR005C24" class="link link-ref link-reveal xref-bibr">24</a></sup> and therefore, they had characteristics of avian viruses. These human H5N1 strains were more separated from the swine and human territories than H1N1/09 strains, and this difference may relate with their infection power in the human and swine populations.</p><p>In order to identify avian strains that were located near the swine and human territories, we next marked lattice points containing sequences from one avian subtype by one colour (Avi Sub in Fig. <a reveal-id="DSR005F1" data-open="DSR005F1" class="link link-reveal link-table xref-fig">1</a>C). Avian subtypes were clearly separated from each other. Some avian strains were in closer proximity to human or swine territory than the human H5N1 strains already known (Avi Sub near Hum Sub in Fig. <a reveal-id="DSR005F1" data-open="DSR005F1" class="link link-reveal link-table xref-fig">1</a>C); for example, H5N1 isolated in Russia (red), H6N1 in Taiwan (yellow), H9N2 in Israel (brown), H5N2 in Minnesota (violet), and H7N2 in New York (light blue). These strains should have oligonucleotide compositions more similar to those of human and/or swine viruses than the known human H5N1 strains. This similarity may reflect in part their evolutionary histories, during which the virus themselves or their segments have changed hosts. If so, the similarity is of particular interest with regard to potential infection powers in human and/or swine populations when these strains invade these populations. The finding that the H1N1/09 strains were located in a closer vicinity of both swine and human territories than the H5N1 strains isolated from humans on BLSOM was consistent with this view, although much more data of new pandemics of various subtypes should be needed before proving this view. Even when analysing a large number of strains, BLSOM could effectively visualize and characterize, from various angles, the strains with peculiar characteristics, on which experimental and/or medical research groups will focus their interests. In the present study, strains isolated from various avian hosts (e.g. chicken, duck, mallard and turkey) were grouped into one category. In the near future, a large number of strains isolated from various avian hosts will be sequenced because of importance both in medicine and in chicken and cattle industries, and difference in oligonucleotide composition might be detectable for different avian hosts on BLSOM.</p><p>In order to clarify an overall feature of the H1N1/09 pandemic, we next investigated the data set including all 2300 H1N1/09 strains available (Fig. <a reveal-id="DSR005F2" data-open="DSR005F2" class="link link-reveal link-table xref-fig">2</a>). Again, host-dependent separation of strains was observed, and 2300 H1N1/09 strains formed a large contiguous territory on all BLSOMs (Fig. <a reveal-id="DSR005F2" data-open="DSR005F2" class="link link-reveal link-table xref-fig">2</a>). This showed that even at the late stage of the 2009 pandemic, H1N1/09 strains had sequence characteristics distinct from human seasonal strains. To analyse possible changes that might occur during this pandemic period, strains isolated in the very early stage (April), the middle stage (August), and the later stage (December) in the 2009 pandemic were separately marked in pink in Fig. <a reveal-id="DSR005F2" data-open="DSR005F2" class="link link-reveal link-table xref-fig">2</a>. In the very early stage (April), a major portion of the strains were located in the vicinity of avian and swine territories, but in the later stage (December), strains near the avian territory became a minority and a major portion moved towards the human territory, indicating directional sequence changes. Dispersion within the H1N1/09 territory for strains isolated at the same pandemic stage appeared to reflect primarily their geographical differences. </p>                        <a id="5307187" scrollto-destination="5307187"></a>
<div id="DSR005F2" data-id="DSR005F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/2/10.1093/dnares/dsr005/2/m_dsr00502.jpeg?Expires=2147483647&Signature=xMBO1y-qnR3tfpOMPJWMaOaVxdWlBWtkSaMtvCYKaY5HdaHO~PUEeOAivkjeXpSx1tYY3MEprWGtEvyczONmYgKbMMbSYlPuvIfi91599~Oc0LPgQEN-H7ZBmJPXa0rKusBoyG-4XXovMG5WQtJ8o~lOUHJomeH0M2qTTbaPpO4zpe4bPXbUm5ODohqvKTHz0G6~yDwcvL7Aic6AGjJTOBEn7yVxRf9XHk7L4a608lPTOfokpvz7bTWiYC~2VvYqMgAmcf6YYrhfZDv6rGlbOoDNA0kO~nRDguMn6mmTyZLHs2736Bohi6AGayb131RuCObx5AJl3SLkinfQNcxxaw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Oligonucleotide-BLSOMs for all influenza A virus sequences. Di, Tri, or Tetra: BLSOMs were constructed for di-, tri-, or tetranucleotide composition in genome sequences from influenza A viruses including all 2256 H1N1/09 strains (arrowed), and lattice points were coloured as described in Fig. 1B. April, August, or December: lattice points containing the H1N1/09 strains isolated in April, August, or December were coloured in pink in the H1N1/09 territory, and lattice points in the territories other than the H1N1/09 territory were coloured as described in Fig. 1B." path-from-xml="dsr00502.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5307187/dsr00502.jpeg" path-from-xml="dsr00502.jpeg" target="_blank">View large</a><a section="5307187" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/2/10.1093/dnares/dsr005/2/dsr00502.jpeg?Expires=2147483647&Signature=IfT8kjxLbuUMHw9G7Ub3y64dAPZTKzZ1AOV4iOBH4aTZDOFmDFQlM3g81mTejMDz6Eb6tRlFG~NNMZnmLnYN9tRIdaeJkG0qq0yOmEsrxb2~sCo0qt1TZ7LcerKAI1hTD5yIoTZ8WTbsA1owG-BuMfHWVkVOo26H-Fh0UFRhhocgATuSeRuYiTinUp09cdq2dXJxg44b-KGpZlaCfwdxOi1uzhUP-HZ2IE8doc-3YM~1EFOGECcMV4jLOlBVtyNZ8-xTZENysnfcIMev3R~tIrgZUFzNdx3zgndgYTXIgsIAg7ZnP2iM5TKBl8D0Wc0madch7yKv4kG2cK56rZmBtw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5307187&ar=365741&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr00502.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Oligonucleotide-BLSOMs for all influenza A virus sequences. Di, Tri, or Tetra: BLSOMs were constructed for di-, tri-, or tetranucleotide composition in genome sequences from influenza A viruses including all 2256 H1N1/09 strains (arrowed), and lattice points were coloured as described in Fig. <a reveal-id="DSR005F1" data-open="DSR005F1" class="link link-reveal link-table xref-fig">1</a>B. April, August, or December: lattice points containing the H1N1/09 strains isolated in April, August, or December were coloured in pink in the H1N1/09 territory, and lattice points in the territories other than the H1N1/09 territory were coloured as described in Fig. <a reveal-id="DSR005F1" data-open="DSR005F1" class="link link-reveal link-table xref-fig">1</a>B.</p></div></div><div content-id="DSR005F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/2/10.1093/dnares/dsr005/2/m_dsr00502.jpeg?Expires=2147483647&Signature=xMBO1y-qnR3tfpOMPJWMaOaVxdWlBWtkSaMtvCYKaY5HdaHO~PUEeOAivkjeXpSx1tYY3MEprWGtEvyczONmYgKbMMbSYlPuvIfi91599~Oc0LPgQEN-H7ZBmJPXa0rKusBoyG-4XXovMG5WQtJ8o~lOUHJomeH0M2qTTbaPpO4zpe4bPXbUm5ODohqvKTHz0G6~yDwcvL7Aic6AGjJTOBEn7yVxRf9XHk7L4a608lPTOfokpvz7bTWiYC~2VvYqMgAmcf6YYrhfZDv6rGlbOoDNA0kO~nRDguMn6mmTyZLHs2736Bohi6AGayb131RuCObx5AJl3SLkinfQNcxxaw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Oligonucleotide-BLSOMs for all influenza A virus sequences. Di, Tri, or Tetra: BLSOMs were constructed for di-, tri-, or tetranucleotide composition in genome sequences from influenza A viruses including all 2256 H1N1/09 strains (arrowed), and lattice points were coloured as described in Fig. 1B. April, August, or December: lattice points containing the H1N1/09 strains isolated in April, August, or December were coloured in pink in the H1N1/09 territory, and lattice points in the territories other than the H1N1/09 territory were coloured as described in Fig. 1B." path-from-xml="dsr00502.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5307187/dsr00502.jpeg" path-from-xml="dsr00502.jpeg" target="_blank">View large</a><a section="5307187" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/2/10.1093/dnares/dsr005/2/dsr00502.jpeg?Expires=2147483647&Signature=IfT8kjxLbuUMHw9G7Ub3y64dAPZTKzZ1AOV4iOBH4aTZDOFmDFQlM3g81mTejMDz6Eb6tRlFG~NNMZnmLnYN9tRIdaeJkG0qq0yOmEsrxb2~sCo0qt1TZ7LcerKAI1hTD5yIoTZ8WTbsA1owG-BuMfHWVkVOo26H-Fh0UFRhhocgATuSeRuYiTinUp09cdq2dXJxg44b-KGpZlaCfwdxOi1uzhUP-HZ2IE8doc-3YM~1EFOGECcMV4jLOlBVtyNZ8-xTZENysnfcIMev3R~tIrgZUFzNdx3zgndgYTXIgsIAg7ZnP2iM5TKBl8D0Wc0madch7yKv4kG2cK56rZmBtw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5307187&ar=365741&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr00502.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Oligonucleotide-BLSOMs for all influenza A virus sequences. Di, Tri, or Tetra: BLSOMs were constructed for di-, tri-, or tetranucleotide composition in genome sequences from influenza A viruses including all 2256 H1N1/09 strains (arrowed), and lattice points were coloured as described in Fig. <a reveal-id="DSR005F1" data-open="DSR005F1" class="link link-reveal link-table xref-fig">1</a>B. April, August, or December: lattice points containing the H1N1/09 strains isolated in April, August, or December were coloured in pink in the H1N1/09 territory, and lattice points in the territories other than the H1N1/09 territory were coloured as described in Fig. <a reveal-id="DSR005F1" data-open="DSR005F1" class="link link-reveal link-table xref-fig">1</a>B.</p></div></div>                            <h3 scrollto-destination=5307188 id="5307188" class="section-title" data-legacy-id=s3c>3.3. Host-specific codon biases</h3>
<p>Synonymous codon choice sensitively reflects constraints imposed on genome sequences and thus provides a sensitive probe for searching for molecular mechanisms responsible for the constraints, e.g. genome G + C% and tRNA composition in the cases of micro-organisms.<sup><a reveal-id="DSR005C28" data-open="DSR005C28" class="link link-ref link-reveal xref-bibr">28</a>–<a reveal-id="DSR005C31" data-open="DSR005C31" class="link link-ref link-reveal xref-bibr">31</a></sup> We previously found that BLSOM efficiently detected species-specific codon-choice patterns of micro-organisms, resulting in self-organization of genes according to microbial species.<sup><a reveal-id="DSR005C21" data-open="DSR005C21" class="link link-ref link-reveal xref-bibr">21</a></sup> Furthermore, in the case of genes horizontally transferred relatively recently, codon choice reflected primarily that of the donor, but not the recipient, genome. We next constructed BLSOM for synonymous codon usage in influenza A virus genes including genes from the H1N1/09 strains isolated in the early stage of the 2009 pandemic (Fig. <a reveal-id="DSR005F3" data-open="DSR005F3" class="link link-reveal link-table xref-fig">3</a>A); in order to know codon biases for each strain, codon usages in eight genes were summed for each strain. Human and avian territories were again clearly separated from each other, and human H1N1/09 strains (arrowed in Fig. <a reveal-id="DSR005F3" data-open="DSR005F3" class="link link-reveal link-table xref-fig">3</a>) were again separated from the major human territory and surrounded by avian, equine, and swine territories. Synonymous codon-choice patterns of newly invading viruses, such as H1N1/09, should be close to those of the original host viruses, at least for a period immediately after the invasion. </p>                        <a id="5307190" scrollto-destination="5307190"></a>
<div id="DSR005F3" data-id="DSR005F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/2/10.1093/dnares/dsr005/2/m_dsr00503.jpeg?Expires=2147483647&Signature=ZMKauaVAnpEjYNCSwFE1UKbtFt-Q2lgptiamHmYe4NJF7OWLqovmU7Cy4czAjJHctNQc4XQmGJTxuq11cAuR3UlfWd-Jh-xnv~S3Sp6vCnDhqd0wxmtI72lB3L0qQ-UJJFeJQTpo86xKXMiWIl98YykQhvGWS5gJEWsx19xWv95z3cZjq0K0OYOFI~v-VNKAC5iqHyVz5dHVbI3fKZeesjhZZ9qf9INdSzqfR88Fe5UWxWQulYCdEQn6p8vzhqqz1E00akZ5KC2jrNqX--I8gLs8q1DQUuZTp3Wa1DfiS-RFzIp8WP7LxWYZxEyPznKjEothbkyUXmi0wHP23utUNA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Codon-BLSOM. (A) Codon: BLSOM was constructed for synonymous codon usage in approximately 34 000 genes from influenza A strains including H1N1/09 isolated in the early stage (April and May), as described by Kanaya et al.,21 and lattice points were indicated in a colour representing the host as described in Fig. 1B. Occurrence levels of seven codons, which were diagnostic for host-specific separation, were indicated with different levels of two colours: pink (high), green (low), and achromatic (intermediate) as described by Kanaya et al.21 Because the synonymous codon usage was analysed, two codons belonging to each two-codon box gave the complementary patterns, and therefore, the pattern of one codon was listed for those belonging to two-codon boxes. (B) Codon: BLSOM was constructed for synonymous codon usage in approximately 60 000 genes from influenza A strains including all H1N1/09strains, and lattice points were indicated in a colour representing the host as described in Fig. 1B. (C) Occurrence levels of four diagnostic codons in the Codon-BLSOM presented in (B) were indicated as described in (A). Scale for the occurrence levels in (A) and (C) was indicated in the colours shown at the bottom of (C). (D) Retrospective time-series changes for human subtype strains (H1N1 and H3N2) on the Codon-BLSOM presented in (A). A zone for H1N1/09 or equine strains was additionally marked for reference to help recognize the position in Codon-BLSOM in Fig. 1A. Each strain was indicated in the colours shown at the bottom of (D)." path-from-xml="dsr00503.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5307190/dsr00503.jpeg" path-from-xml="dsr00503.jpeg" target="_blank">View large</a><a section="5307190" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/2/10.1093/dnares/dsr005/2/dsr00503.jpeg?Expires=2147483647&Signature=cx7HDhvECwuMcR140u69ZzI6TKhXpf3N9~ywGtlHnRkOnjx6GCbZWAQ-BhJOa7Lq7ANaMYhSIyvoKgTEmWRakJ6fb9lE1Ab1cgkFyFZQCoNkfZVIsTmfr1q-vo1Qib1fzER1V5hVHlVJaAvNxZz5bPZj5TePf-6SELqoryTMdYeyG6z-YnHkreAKjVO3cdaaJpO65NQTbbbSAjzXneWMhktJ1o-CP0AFYrRAmBkIap~yesfxzA4RPrbl6gL5HWe-5cmVOkkEGWBWxVI~e~M5Y7ctwwxbU7csfJewY4KruYTsoMA0KzS0NCyuXSjLlbaQt0OtUWS3Ia2bNKns~v5iRQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5307190&ar=365741&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr00503.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Codon-BLSOM. (A) Codon: BLSOM was constructed for synonymous codon usage in approximately 34 000 genes from influenza A strains including H1N1/09 isolated in the early stage (April and May), as described by Kanaya <em>et al</em>.,<a reveal-id="DSR005C21" data-open="DSR005C21" class="link link-ref link-reveal xref-bibr">21</a> and lattice points were indicated in a colour representing the host as described in Fig. <a reveal-id="DSR005F1" data-open="DSR005F1" class="link link-reveal link-table xref-fig">1</a>B. Occurrence levels of seven codons, which were diagnostic for host-specific separation, were indicated with different levels of two colours: pink (high), green (low), and achromatic (intermediate) as described by Kanaya <em>et al</em>.<a reveal-id="DSR005C21" data-open="DSR005C21" class="link link-ref link-reveal xref-bibr">21</a> Because the synonymous codon usage was analysed, two codons belonging to each two-codon box gave the complementary patterns, and therefore, the pattern of one codon was listed for those belonging to two-codon boxes. (B) Codon: BLSOM was constructed for synonymous codon usage in approximately 60 000 genes from influenza A strains including all H1N1/09strains, and lattice points were indicated in a colour representing the host as described in Fig. <a reveal-id="DSR005F1" data-open="DSR005F1" class="link link-reveal link-table xref-fig">1</a>B. (C) Occurrence levels of four diagnostic codons in the Codon-BLSOM presented in (B) were indicated as described in (A). Scale for the occurrence levels in (A) and (C) was indicated in the colours shown at the bottom of (C). (D) Retrospective time-series changes for human subtype strains (H1N1 and H3N2) on the Codon-BLSOM presented in (A). A zone for H1N1/09 or equine strains was additionally marked for reference to help recognize the position in Codon-BLSOM in Fig. <a reveal-id="DSR005F1" data-open="DSR005F1" class="link link-reveal link-table xref-fig">1</a>A. Each strain was indicated in the colours shown at the bottom of (D).</p></div></div><div content-id="DSR005F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/2/10.1093/dnares/dsr005/2/m_dsr00503.jpeg?Expires=2147483647&Signature=ZMKauaVAnpEjYNCSwFE1UKbtFt-Q2lgptiamHmYe4NJF7OWLqovmU7Cy4czAjJHctNQc4XQmGJTxuq11cAuR3UlfWd-Jh-xnv~S3Sp6vCnDhqd0wxmtI72lB3L0qQ-UJJFeJQTpo86xKXMiWIl98YykQhvGWS5gJEWsx19xWv95z3cZjq0K0OYOFI~v-VNKAC5iqHyVz5dHVbI3fKZeesjhZZ9qf9INdSzqfR88Fe5UWxWQulYCdEQn6p8vzhqqz1E00akZ5KC2jrNqX--I8gLs8q1DQUuZTp3Wa1DfiS-RFzIp8WP7LxWYZxEyPznKjEothbkyUXmi0wHP23utUNA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Codon-BLSOM. (A) Codon: BLSOM was constructed for synonymous codon usage in approximately 34 000 genes from influenza A strains including H1N1/09 isolated in the early stage (April and May), as described by Kanaya et al.,21 and lattice points were indicated in a colour representing the host as described in Fig. 1B. Occurrence levels of seven codons, which were diagnostic for host-specific separation, were indicated with different levels of two colours: pink (high), green (low), and achromatic (intermediate) as described by Kanaya et al.21 Because the synonymous codon usage was analysed, two codons belonging to each two-codon box gave the complementary patterns, and therefore, the pattern of one codon was listed for those belonging to two-codon boxes. (B) Codon: BLSOM was constructed for synonymous codon usage in approximately 60 000 genes from influenza A strains including all H1N1/09strains, and lattice points were indicated in a colour representing the host as described in Fig. 1B. (C) Occurrence levels of four diagnostic codons in the Codon-BLSOM presented in (B) were indicated as described in (A). Scale for the occurrence levels in (A) and (C) was indicated in the colours shown at the bottom of (C). (D) Retrospective time-series changes for human subtype strains (H1N1 and H3N2) on the Codon-BLSOM presented in (A). A zone for H1N1/09 or equine strains was additionally marked for reference to help recognize the position in Codon-BLSOM in Fig. 1A. Each strain was indicated in the colours shown at the bottom of (D)." path-from-xml="dsr00503.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5307190/dsr00503.jpeg" path-from-xml="dsr00503.jpeg" target="_blank">View large</a><a section="5307190" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/2/10.1093/dnares/dsr005/2/dsr00503.jpeg?Expires=2147483647&Signature=cx7HDhvECwuMcR140u69ZzI6TKhXpf3N9~ywGtlHnRkOnjx6GCbZWAQ-BhJOa7Lq7ANaMYhSIyvoKgTEmWRakJ6fb9lE1Ab1cgkFyFZQCoNkfZVIsTmfr1q-vo1Qib1fzER1V5hVHlVJaAvNxZz5bPZj5TePf-6SELqoryTMdYeyG6z-YnHkreAKjVO3cdaaJpO65NQTbbbSAjzXneWMhktJ1o-CP0AFYrRAmBkIap~yesfxzA4RPrbl6gL5HWe-5cmVOkkEGWBWxVI~e~M5Y7ctwwxbU7csfJewY4KruYTsoMA0KzS0NCyuXSjLlbaQt0OtUWS3Ia2bNKns~v5iRQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5307190&ar=365741&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr00503.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Codon-BLSOM. (A) Codon: BLSOM was constructed for synonymous codon usage in approximately 34 000 genes from influenza A strains including H1N1/09 isolated in the early stage (April and May), as described by Kanaya <em>et al</em>.,<a reveal-id="DSR005C21" data-open="DSR005C21" class="link link-ref link-reveal xref-bibr">21</a> and lattice points were indicated in a colour representing the host as described in Fig. <a reveal-id="DSR005F1" data-open="DSR005F1" class="link link-reveal link-table xref-fig">1</a>B. Occurrence levels of seven codons, which were diagnostic for host-specific separation, were indicated with different levels of two colours: pink (high), green (low), and achromatic (intermediate) as described by Kanaya <em>et al</em>.<a reveal-id="DSR005C21" data-open="DSR005C21" class="link link-ref link-reveal xref-bibr">21</a> Because the synonymous codon usage was analysed, two codons belonging to each two-codon box gave the complementary patterns, and therefore, the pattern of one codon was listed for those belonging to two-codon boxes. (B) Codon: BLSOM was constructed for synonymous codon usage in approximately 60 000 genes from influenza A strains including all H1N1/09strains, and lattice points were indicated in a colour representing the host as described in Fig. <a reveal-id="DSR005F1" data-open="DSR005F1" class="link link-reveal link-table xref-fig">1</a>B. (C) Occurrence levels of four diagnostic codons in the Codon-BLSOM presented in (B) were indicated as described in (A). Scale for the occurrence levels in (A) and (C) was indicated in the colours shown at the bottom of (C). (D) Retrospective time-series changes for human subtype strains (H1N1 and H3N2) on the Codon-BLSOM presented in (A). A zone for H1N1/09 or equine strains was additionally marked for reference to help recognize the position in Codon-BLSOM in Fig. <a reveal-id="DSR005F1" data-open="DSR005F1" class="link link-reveal link-table xref-fig">1</a>A. Each strain was indicated in the colours shown at the bottom of (D).</p></div></div><p>Because viruses depend on many cellular factors for their growth, codon choice will most likely shift towards the pattern of seasonal human viruses during many infection cycles among humans. If so, the direction of sequence changes in H1N1/09 in the near future is predictable, and this should be testable because of the high mutation rate and short generation time of influenza A viruses.<sup><a reveal-id="DSR005C24" data-open="DSR005C24" class="link link-ref link-reveal xref-bibr">24</a>,<a reveal-id="DSR005C32" data-open="DSR005C32" class="link link-ref link-reveal xref-bibr">32</a></sup> By analysing retrospective changes of codon bias of human seasonal H1 and H3 strains isolated before 2006, Wong <em>et al</em>.<sup><a reveal-id="DSR005C27" data-open="DSR005C27" class="link link-ref link-reveal xref-bibr">27</a></sup> found the overall reduction of G + C% during course of their evolutions. To examine whether the codon choice in H1N1/09 strains also changed even within the 2009 pandemic, we constructed codon-BLSOM in which genes from all 2300 H1N1/09 strains were included and marked strains isolated in the very early stage (April), the middle stage (August), and the later stage (December) separately in pink (Fig. <a reveal-id="DSR005F3" data-open="DSR005F3" class="link link-reveal link-table xref-fig">3</a>B). A major portion of the strains isolated in the very early stage (April) was located in the vicinity of the avian territory but a major portion in the later stage (December) was apart from the avian territory, as found in the tetra-BLSOM (Fig. <a reveal-id="DSR005F2" data-open="DSR005F2" class="link link-reveal link-table xref-fig">2</a>), supporting the directional sequence changes.</p>                            <h3 scrollto-destination=5307192 id="5307192" class="section-title" data-legacy-id=s3d>3.4. Diagnostic codons for host-specific clustering</h3>
<p>We next attempted to identify codons that will change their occurrence levels during the course of the H1N1/09 evolution. BLSOM provides a powerful ability for visualizing diagnostic codons or oligonucleotides that contribute to self-organization of sequences according to hosts.<sup><a reveal-id="DSR005C6" data-open="DSR005C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DSR005C7" data-open="DSR005C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DSR005C21" data-open="DSR005C21" class="link link-ref link-reveal xref-bibr">21</a></sup> In Fig. <a reveal-id="DSR005F3" data-open="DSR005F3" class="link link-reveal link-table xref-fig">3</a>A, the frequency of each codon in the representative vector at each lattice point was calculated and sorted according to the frequency, and this rank order was represented at different levels in colours pink (high) and green (low) for each codon, as described previously.<sup><a reveal-id="DSR005C21" data-open="DSR005C21" class="link link-ref link-reveal xref-bibr">21</a></sup> Transitions between the high and low ranks often coincided with host territory borders, and seven examples of the codons diagnostic for host separation were presented (Fig. <a reveal-id="DSR005F3" data-open="DSR005F3" class="link link-reveal link-table xref-fig">3</a>A). Clear difference was observed between human and avian territories for these seven codons. Minor differences in the pattern between codons were observed mainly for swine and equine territories. For example, AAG was preferred in the avian territory but neither in a major portion of the human territory nor in the equine territory, which was specified in yellow in the ‘Codon' panel in Fig. <a reveal-id="DSR005F3" data-open="DSR005F3" class="link link-reveal link-table xref-fig">3</a>A. In the cases of CAG and UUC, these were preferred in both the avian and equine territories, but not in a major portion of the human territory.</p><p>In Table <a reveal-id="DSR005TB1" data-open="DSR005TB1" class="link link-reveal link-table xref-fig">1</a>, all diagnostic codons for the separation between human and avian territories were summarized. When we focused on diagnostic codons (Codon in Table <a reveal-id="DSR005TB1" data-open="DSR005TB1" class="link link-reveal link-table xref-fig">1</a>), one simple tendency was observed. Codons ending with G or C were more favourable for the avian strains than the human strains. This supported the previous observation of codon biases found by Wong <em>et al</em>.<sup><a reveal-id="DSR005C27" data-open="DSR005C27" class="link link-ref link-reveal xref-bibr">27</a></sup> in their analyses in which individual genes were analysed separately with Corresponding Analysis. Confirmation of the previous finding concerning codon usage with the BLSOM method showed the reliability of this new method. The G + C% effect was most apparent in two-codon boxes (Table <a reveal-id="DSR005TB1" data-open="DSR005TB1" class="link link-reveal link-table xref-fig">1</a>, Fig. <a reveal-id="DSR005F3" data-open="DSR005F3" class="link link-reveal link-table xref-fig">3</a>A). This was also observed for many codons in four- or six-codon boxes, but there were few exceptional cases, such as codons ‘GCA’ preferred in the avian territory and ‘UUG’ preferred in the human territory (Table <a reveal-id="DSR005TB1" data-open="DSR005TB1" class="link link-reveal link-table xref-fig">1</a>), indicating the presence of constraints other than the G + C% effect. </p>                        <a id="5307196" scrollto-destination="5307196"></a>
<div content-id="DSR005TB1" class="table-modal"><div class="table-wrap"><div id="DSR005TB1" class="table-wrap-title" data-id="DSR005TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Preferred codons and oligonucleotides in avian or human viruses</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Preferred in avian viruses </th><th>Preferred in human viruses </th></tr></thead><tbody><tr><td>Codon </td><td><strong><em>AAG</em></strong>, <strong><em>CAG</em></strong>, <strong><em>CUC</em></strong>, CUG, <strong><em>GCA</em></strong>, GCG, GUG, <strong><em>UCG</em></strong>, <strong><em>UUC</em></strong> </td><td><strong><em>AAA</em></strong>, <strong><em>ACU</em></strong>, AGA, <strong><em>CAA</em></strong>, <strong><em>CCU</em></strong>, <strong><em>GUU</em></strong>, UCA, <strong><em>UUA</em></strong>, <strong><em>UUG</em></strong>, <strong><em>UUU</em></strong> </td></tr><tr><td>Di </td><td><strong><em>AG</em></strong>, <strong><em>CG</em></strong>, CU, <strong><em>GA</em></strong>, <strong><em>GG</em></strong> </td><td>AA, <strong><em>UU</em></strong> </td></tr><tr><td>Tri </td><td>ACG, AGG, CAG, <strong><em>CCA</em></strong>, CGU, GAG, GCA, <strong><em>GCG</em></strong>, GGA, <strong><em>GUG</em></strong>, UCC, UCU </td><td>AAA, <strong><em>AUU</em></strong>, UAA, UCA, <strong><em>UUA</em></strong>, <strong><em>UUU</em></strong> </td></tr><tr><td>Tetra </td><td>ACGC, <strong><em>ACGG</em></strong>, <strong><em>AGAG</em></strong>, <strong><em>AGCG</em></strong>, <strong><em>CCAC</em></strong>, CGAG, CGGA, <strong><em>CGGC</em></strong>, CUUC, <strong><em>GACU</em></strong>, GAGC, GAGG, GCAG, GGAG, <strong><em>UCUU</em></strong>, UGUG, <strong><em>UUCG</em></strong> </td><td><strong><em>AAAA</em></strong>, AAAU, AAGU, AUUA, AUUU, <strong><em>CAAA</em></strong>, CCAU, CUUU, GGCC, <strong><em>GGGG</em></strong>, <strong><em>UGUA</em></strong>, <strong><em>UGUU</em></strong>, <strong><em>UUAA</em></strong>, UUAU, UUCA, <strong><em>UUGU</em></strong>, <strong><em>UUUC</em></strong>, <strong><em>UUUG</em></strong>, <strong><em>UUUU</em></strong> </td></tr></tbody></table></div><div content-id="DSR005TB1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Preferred in avian viruses </th><th>Preferred in human viruses </th></tr></thead><tbody><tr><td>Codon </td><td><strong><em>AAG</em></strong>, <strong><em>CAG</em></strong>, <strong><em>CUC</em></strong>, CUG, <strong><em>GCA</em></strong>, GCG, GUG, <strong><em>UCG</em></strong>, <strong><em>UUC</em></strong> </td><td><strong><em>AAA</em></strong>, <strong><em>ACU</em></strong>, AGA, <strong><em>CAA</em></strong>, <strong><em>CCU</em></strong>, <strong><em>GUU</em></strong>, UCA, <strong><em>UUA</em></strong>, <strong><em>UUG</em></strong>, <strong><em>UUU</em></strong> </td></tr><tr><td>Di </td><td><strong><em>AG</em></strong>, <strong><em>CG</em></strong>, CU, <strong><em>GA</em></strong>, <strong><em>GG</em></strong> </td><td>AA, <strong><em>UU</em></strong> </td></tr><tr><td>Tri </td><td>ACG, AGG, CAG, <strong><em>CCA</em></strong>, CGU, GAG, GCA, <strong><em>GCG</em></strong>, GGA, <strong><em>GUG</em></strong>, UCC, UCU </td><td>AAA, <strong><em>AUU</em></strong>, UAA, UCA, <strong><em>UUA</em></strong>, <strong><em>UUU</em></strong> </td></tr><tr><td>Tetra </td><td>ACGC, <strong><em>ACGG</em></strong>, <strong><em>AGAG</em></strong>, <strong><em>AGCG</em></strong>, <strong><em>CCAC</em></strong>, CGAG, CGGA, <strong><em>CGGC</em></strong>, CUUC, <strong><em>GACU</em></strong>, GAGC, GAGG, GCAG, GGAG, <strong><em>UCUU</em></strong>, UGUG, <strong><em>UUCG</em></strong> </td><td><strong><em>AAAA</em></strong>, AAAU, AAGU, AUUA, AUUU, <strong><em>CAAA</em></strong>, CCAU, CUUU, GGCC, <strong><em>GGGG</em></strong>, <strong><em>UGUA</em></strong>, <strong><em>UGUU</em></strong>, <strong><em>UUAA</em></strong>, UUAU, UUCA, <strong><em>UUGU</em></strong>, <strong><em>UUUC</em></strong>, <strong><em>UUUG</em></strong>, <strong><em>UUUU</em></strong> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>To specify the characteristic codon/oligonucleotide preference in the H1N1/09 strains, codons and oligonucleotides preferred in both H1N1/09 and avian strains are indicated in bold italic letters in the column ‘Preferred in avian viruses’. Similarly, those preferred in seasonal human viruses but not in H1N1/09 are indicated in bold italic letters in the column ‘Preferred in human viruses’.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5307196" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR005TB1" class="table-wrap-title" data-id="DSR005TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Preferred codons and oligonucleotides in avian or human viruses</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Preferred in avian viruses </th><th>Preferred in human viruses </th></tr></thead><tbody><tr><td>Codon </td><td><strong><em>AAG</em></strong>, <strong><em>CAG</em></strong>, <strong><em>CUC</em></strong>, CUG, <strong><em>GCA</em></strong>, GCG, GUG, <strong><em>UCG</em></strong>, <strong><em>UUC</em></strong> </td><td><strong><em>AAA</em></strong>, <strong><em>ACU</em></strong>, AGA, <strong><em>CAA</em></strong>, <strong><em>CCU</em></strong>, <strong><em>GUU</em></strong>, UCA, <strong><em>UUA</em></strong>, <strong><em>UUG</em></strong>, <strong><em>UUU</em></strong> </td></tr><tr><td>Di </td><td><strong><em>AG</em></strong>, <strong><em>CG</em></strong>, CU, <strong><em>GA</em></strong>, <strong><em>GG</em></strong> </td><td>AA, <strong><em>UU</em></strong> </td></tr><tr><td>Tri </td><td>ACG, AGG, CAG, <strong><em>CCA</em></strong>, CGU, GAG, GCA, <strong><em>GCG</em></strong>, GGA, <strong><em>GUG</em></strong>, UCC, UCU </td><td>AAA, <strong><em>AUU</em></strong>, UAA, UCA, <strong><em>UUA</em></strong>, <strong><em>UUU</em></strong> </td></tr><tr><td>Tetra </td><td>ACGC, <strong><em>ACGG</em></strong>, <strong><em>AGAG</em></strong>, <strong><em>AGCG</em></strong>, <strong><em>CCAC</em></strong>, CGAG, CGGA, <strong><em>CGGC</em></strong>, CUUC, <strong><em>GACU</em></strong>, GAGC, GAGG, GCAG, GGAG, <strong><em>UCUU</em></strong>, UGUG, <strong><em>UUCG</em></strong> </td><td><strong><em>AAAA</em></strong>, AAAU, AAGU, AUUA, AUUU, <strong><em>CAAA</em></strong>, CCAU, CUUU, GGCC, <strong><em>GGGG</em></strong>, <strong><em>UGUA</em></strong>, <strong><em>UGUU</em></strong>, <strong><em>UUAA</em></strong>, UUAU, UUCA, <strong><em>UUGU</em></strong>, <strong><em>UUUC</em></strong>, <strong><em>UUUG</em></strong>, <strong><em>UUUU</em></strong> </td></tr></tbody></table></div><div content-id="DSR005TB1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Preferred in avian viruses </th><th>Preferred in human viruses </th></tr></thead><tbody><tr><td>Codon </td><td><strong><em>AAG</em></strong>, <strong><em>CAG</em></strong>, <strong><em>CUC</em></strong>, CUG, <strong><em>GCA</em></strong>, GCG, GUG, <strong><em>UCG</em></strong>, <strong><em>UUC</em></strong> </td><td><strong><em>AAA</em></strong>, <strong><em>ACU</em></strong>, AGA, <strong><em>CAA</em></strong>, <strong><em>CCU</em></strong>, <strong><em>GUU</em></strong>, UCA, <strong><em>UUA</em></strong>, <strong><em>UUG</em></strong>, <strong><em>UUU</em></strong> </td></tr><tr><td>Di </td><td><strong><em>AG</em></strong>, <strong><em>CG</em></strong>, CU, <strong><em>GA</em></strong>, <strong><em>GG</em></strong> </td><td>AA, <strong><em>UU</em></strong> </td></tr><tr><td>Tri </td><td>ACG, AGG, CAG, <strong><em>CCA</em></strong>, CGU, GAG, GCA, <strong><em>GCG</em></strong>, GGA, <strong><em>GUG</em></strong>, UCC, UCU </td><td>AAA, <strong><em>AUU</em></strong>, UAA, UCA, <strong><em>UUA</em></strong>, <strong><em>UUU</em></strong> </td></tr><tr><td>Tetra </td><td>ACGC, <strong><em>ACGG</em></strong>, <strong><em>AGAG</em></strong>, <strong><em>AGCG</em></strong>, <strong><em>CCAC</em></strong>, CGAG, CGGA, <strong><em>CGGC</em></strong>, CUUC, <strong><em>GACU</em></strong>, GAGC, GAGG, GCAG, GGAG, <strong><em>UCUU</em></strong>, UGUG, <strong><em>UUCG</em></strong> </td><td><strong><em>AAAA</em></strong>, AAAU, AAGU, AUUA, AUUU, <strong><em>CAAA</em></strong>, CCAU, CUUU, GGCC, <strong><em>GGGG</em></strong>, <strong><em>UGUA</em></strong>, <strong><em>UGUU</em></strong>, <strong><em>UUAA</em></strong>, UUAU, UUCA, <strong><em>UUGU</em></strong>, <strong><em>UUUC</em></strong>, <strong><em>UUUG</em></strong>, <strong><em>UUUU</em></strong> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>To specify the characteristic codon/oligonucleotide preference in the H1N1/09 strains, codons and oligonucleotides preferred in both H1N1/09 and avian strains are indicated in bold italic letters in the column ‘Preferred in avian viruses’. Similarly, those preferred in seasonal human viruses but not in H1N1/09 are indicated in bold italic letters in the column ‘Preferred in human viruses’.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5307196" target="_blank">View Large</a></div></div><p>Notably, for many diagnostic codons, H1N1/09 strains (arrowed) had the avian-type preference rather than the human-type preference (Fig. <a reveal-id="DSR005F3" data-open="DSR005F3" class="link link-reveal link-table xref-fig">3</a>A). In Table <a reveal-id="DSR005TB1" data-open="DSR005TB1" class="link link-reveal link-table xref-fig">1</a>, to specify this characteristic codon preference in the H1N1/09 strains, codons preferred in both H1N1/09 and avian strains are indicated in bold italic letters in the column ‘Preferred in avian viruses’, and codons preferred in seasonal human viruses, but not in H1N1/09, are indicated in bold italic letters in the column ‘Preferred in human viruses’. Adaptation of codon choice to a new cellular environment (e.g. host body temperature and cellular factors) may be a process for a newly invading virus to establish continuous and efficient infection cycles among humans by increasing fitness.</p><p>Next, we analysed diagnostic codons for the host-specific separation found in the codon-BLSOM in Fig. <a reveal-id="DSR005F3" data-open="DSR005F3" class="link link-reveal link-table xref-fig">3</a>B, which included all 2300 H1N1/09 strains, and presented four diagnostic examples in Fig. <a reveal-id="DSR005F3" data-open="DSR005F3" class="link link-reveal link-table xref-fig">3</a>C. Codon preference in the H1N1/09 territory (dark green, arrowed) was again similar to that of the avian territory but not of the human territory, showing that even in the late stage of the 2009 pandemic, the codon bias in the H1N1/09 genes did not reach the level found for a major portion of human seasonal virus genes. Uneven pink and green levels were observed within the H1N1/09 territory, and this presumably represented in part the sequence changes accumulated during the 2009 pandemic.</p>                            <h3 scrollto-destination=5307200 id="5307200" class="section-title" data-legacy-id=s3e>3.5. Diagnostic oligonucleotides for host-specific clustering</h3>
<p>To identify the oligonucleotides that may change their occurrence levels after the present new virus invasion into human populations, we next identified diagnostic oligonucleotides for host-specific separation on the di-, tri-, and tetranucleotide BLSOMs, which were constructed for the data set including only the 170 H1N1/09 strains isolated in the early stage. Three examples of diagnostic dinucleotides for the host-specific separation on the dinucleotide BLSOM (Di in Fig. <a reveal-id="DSR005F1" data-open="DSR005F1" class="link link-reveal link-table xref-fig">1</a>B) are presented in Fig. <a reveal-id="DSR005F4" data-open="DSR005F4" class="link link-reveal link-table xref-fig">4</a>A, seven examples of diagnostic tetranucleotides are presented in Fig. <a reveal-id="DSR005F4" data-open="DSR005F4" class="link link-reveal link-table xref-fig">4</a>B, and seven examples of diagnostic trinucleotides are presented in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>. All diagnostic di-, tri-, and tetranucleotides for the separation between human and avian territories are summarized in Table <a reveal-id="DSR005TB1" data-open="DSR005TB1" class="link link-reveal link-table xref-fig">1</a>. Two sorts of distinct tendency were immediately apparent: (i) G- and C-rich oligonucleotides were more favourable in avians than in humans. This G + C% effect was previously reported by Rabadan <em>et al</em>.<sup><a reveal-id="DSR005C26" data-open="DSR005C26" class="link link-ref link-reveal xref-bibr">26</a></sup> (ii) Oligonucleotides containing AG, CG, or GA dinucleotides were more favourable in avians than in humans. The observation about the CG dinucleotide was consistent with the previous finding by Greenbaum <em>et al</em>.<sup><a reveal-id="DSR005C33" data-open="DSR005C33" class="link link-ref link-reveal xref-bibr">33</a></sup> that human viruses exhibit a lower CG dinucleotide content than avian viruses. Most of diagnostic di- and trinucleotides could be explained by the above two rules, but there were various exceptional cases for tetranucleotides, indicating the presence of factors other than the two rules. For example, as observed in Fig. <a reveal-id="DSR005F4" data-open="DSR005F4" class="link link-reveal link-table xref-fig">4</a>B, GGGG, a tetranucleotide composed only of G, was preferred mainly in the human territory, whereas UCUU, a tetranucleotide rich in U, was preferred mainly in the avian territory. In the case of interaction of viral components with host factors (e.g. host proteins), oligonucleotide compositions, rather than the mononucleotide composition, will become important, because interactions with host factors primarily depend on oligonucleotide sequences. This should also be true in considering escape processes from host antiviral mechanisms. BLSOM for oligonucleotide composition should provide valuable information to experimental studies of adaptation mechanisms, such as interactions between viral and host factors. </p>                        <a id="5307203" scrollto-destination="5307203"></a>
<div id="DSR005F4" data-id="DSR005F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/2/10.1093/dnares/dsr005/2/m_dsr00504.jpeg?Expires=2147483647&Signature=nFNlnBEvVp7-zB8NqJeni014bmqjoYwG1ushkoxRLnCM4pwvhYVKSPjiHxh1sM2wFI~2xvlWNbxPb5z6fU0RUV77nhKpRG-m4zdEr0MHLB35pmSy1wh1~abmFmrT6qMAXB2NdjNNoKjXgRCwqSV7zOWtloYuymK~TyaGFgN4ekCOiwq224Q7B1lywquVNNKvUkrjjSljdQcMMQ0XiGwbwIFiQJK9ASTk3lXdKWfSHDWyvvjn0KQ5WdrVkOiVe9QTkIuexh0hiUeNy0JmJ2mlAgFQ1htm6I1D5UVj9k7Ii9kOekUCLyBrSXZOtKAUtR3cD4aUe1WgLy54KBleHnumJg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Diagnostic di- or tetranucleotides for host-specific separation. (A) Di: dinucleotide-BLSOM listed in Fig. 1B. Occurrence levels of three dinucleotides, which were diagnostics for host-specific separation, were indicated with different levels of two colours as described in Fig. 3A. (B) Tetra: tetranucleotide-BLSOM listed in Fig. 1B. Occurrence levels of seven diagnostic tetranucleotides were indicated with different levels of two colours as described in Fig. 3A. Scale for the occurrence levels was indicated in the colours shown at the bottom of this figure." path-from-xml="dsr00504.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5307203/dsr00504.jpeg" path-from-xml="dsr00504.jpeg" target="_blank">View large</a><a section="5307203" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/2/10.1093/dnares/dsr005/2/dsr00504.jpeg?Expires=2147483647&Signature=rpVYZQTtAdj1Dy510ArKRfRmcfJwXAsblrVrfKaXQC6T9~WyC74QZKzi7EYz4sr6TY7s3rFMl8kKWKH~L7nXJPigQjnWu7vPIgxxPDRUMp9D1jQy4XvVuZBwOjN2a8gtMxlY0WJJ1o6R0L59hPczBsiZl471fxwWYMFadymjm00kxuWDgvk5sNWDDokhpoMrzHioAGDx3b69koFvvh-GPyXIbbRLAl2WWqRd5-0ywGAiX3-1yygn1Vq~TVDAHzqieJ2oX~tDFOekVNeS-Xa8Nrfj0sin7KcHDsJq8WpiTWu-VyQJhlJl1y4UKNLe-K9~jrD3yn7MM3rAVjV0HsEKUA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5307203&ar=365741&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr00504.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Diagnostic di- or tetranucleotides for host-specific separation. (A) Di: dinucleotide-BLSOM listed in Fig. <a reveal-id="DSR005F1" data-open="DSR005F1" class="link link-reveal link-table xref-fig">1</a>B. Occurrence levels of three dinucleotides, which were diagnostics for host-specific separation, were indicated with different levels of two colours as described in Fig. <a reveal-id="DSR005F3" data-open="DSR005F3" class="link link-reveal link-table xref-fig">3</a>A. (B) Tetra: tetranucleotide-BLSOM listed in Fig. <a reveal-id="DSR005F1" data-open="DSR005F1" class="link link-reveal link-table xref-fig">1</a>B. Occurrence levels of seven diagnostic tetranucleotides were indicated with different levels of two colours as described in Fig. <a reveal-id="DSR005F3" data-open="DSR005F3" class="link link-reveal link-table xref-fig">3</a>A. Scale for the occurrence levels was indicated in the colours shown at the bottom of this figure.</p></div></div><div content-id="DSR005F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/2/10.1093/dnares/dsr005/2/m_dsr00504.jpeg?Expires=2147483647&Signature=nFNlnBEvVp7-zB8NqJeni014bmqjoYwG1ushkoxRLnCM4pwvhYVKSPjiHxh1sM2wFI~2xvlWNbxPb5z6fU0RUV77nhKpRG-m4zdEr0MHLB35pmSy1wh1~abmFmrT6qMAXB2NdjNNoKjXgRCwqSV7zOWtloYuymK~TyaGFgN4ekCOiwq224Q7B1lywquVNNKvUkrjjSljdQcMMQ0XiGwbwIFiQJK9ASTk3lXdKWfSHDWyvvjn0KQ5WdrVkOiVe9QTkIuexh0hiUeNy0JmJ2mlAgFQ1htm6I1D5UVj9k7Ii9kOekUCLyBrSXZOtKAUtR3cD4aUe1WgLy54KBleHnumJg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Diagnostic di- or tetranucleotides for host-specific separation. (A) Di: dinucleotide-BLSOM listed in Fig. 1B. Occurrence levels of three dinucleotides, which were diagnostics for host-specific separation, were indicated with different levels of two colours as described in Fig. 3A. (B) Tetra: tetranucleotide-BLSOM listed in Fig. 1B. Occurrence levels of seven diagnostic tetranucleotides were indicated with different levels of two colours as described in Fig. 3A. Scale for the occurrence levels was indicated in the colours shown at the bottom of this figure." path-from-xml="dsr00504.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5307203/dsr00504.jpeg" path-from-xml="dsr00504.jpeg" target="_blank">View large</a><a section="5307203" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/2/10.1093/dnares/dsr005/2/dsr00504.jpeg?Expires=2147483647&Signature=rpVYZQTtAdj1Dy510ArKRfRmcfJwXAsblrVrfKaXQC6T9~WyC74QZKzi7EYz4sr6TY7s3rFMl8kKWKH~L7nXJPigQjnWu7vPIgxxPDRUMp9D1jQy4XvVuZBwOjN2a8gtMxlY0WJJ1o6R0L59hPczBsiZl471fxwWYMFadymjm00kxuWDgvk5sNWDDokhpoMrzHioAGDx3b69koFvvh-GPyXIbbRLAl2WWqRd5-0ywGAiX3-1yygn1Vq~TVDAHzqieJ2oX~tDFOekVNeS-Xa8Nrfj0sin7KcHDsJq8WpiTWu-VyQJhlJl1y4UKNLe-K9~jrD3yn7MM3rAVjV0HsEKUA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5307203&ar=365741&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr00504.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Diagnostic di- or tetranucleotides for host-specific separation. (A) Di: dinucleotide-BLSOM listed in Fig. <a reveal-id="DSR005F1" data-open="DSR005F1" class="link link-reveal link-table xref-fig">1</a>B. Occurrence levels of three dinucleotides, which were diagnostics for host-specific separation, were indicated with different levels of two colours as described in Fig. <a reveal-id="DSR005F3" data-open="DSR005F3" class="link link-reveal link-table xref-fig">3</a>A. (B) Tetra: tetranucleotide-BLSOM listed in Fig. <a reveal-id="DSR005F1" data-open="DSR005F1" class="link link-reveal link-table xref-fig">1</a>B. Occurrence levels of seven diagnostic tetranucleotides were indicated with different levels of two colours as described in Fig. <a reveal-id="DSR005F3" data-open="DSR005F3" class="link link-reveal link-table xref-fig">3</a>A. Scale for the occurrence levels was indicated in the colours shown at the bottom of this figure.</p></div></div><p>Notably again, H1N1/09 strains (arrowed in Fig. <a reveal-id="DSR005F4" data-open="DSR005F4" class="link link-reveal link-table xref-fig">4</a>) have characteristics of avian, rather than of human, strains. In the ‘Preferred in avian viruses’ column in Table <a reveal-id="DSR005TB1" data-open="DSR005TB1" class="link link-reveal link-table xref-fig">1</a>, oligonucleotides that were preferred in both avian and H1N1/09 strains, but not in human seasonal strains and thus have the potentiality for decrease in H1N1/09 genomes in the near future, are indicated in bold italic letters. Similarly, oligonucleotides that were preferred in human seasonal viruses, but not in H1N1/09 and thus have the potentiality for increase in H1N1/09 genomes in the near future, are indicated in bold italic letters in the ‘Preferred in human viruses’ column. Searches for nucleotide positions in H1N1/09 genomes with the elevated potentiality of directional changes within antigenic sites,<sup><a reveal-id="DSR005C1" data-open="DSR005C1" class="link link-ref link-reveal xref-bibr">1</a>–<a reveal-id="DSR005C3" data-open="DSR005C3" class="link link-ref link-reveal xref-bibr">3</a></sup> antiviral drug-binding sites<sup><a reveal-id="DSR005C34" data-open="DSR005C34" class="link link-ref link-reveal xref-bibr">34</a>,<a reveal-id="DSR005C35" data-open="DSR005C35" class="link link-ref link-reveal xref-bibr">35</a></sup> and sites affecting virus growth and virulence<sup><a reveal-id="DSR005C24" data-open="DSR005C24" class="link link-ref link-reveal xref-bibr">24</a>,<a reveal-id="DSR005C32" data-open="DSR005C32" class="link link-ref link-reveal xref-bibr">32</a></sup> will provide valuable information for predicting possible H1N1/09 descendant sequences that may present potential hazards. Although protein sequences evolve undoubtedly under the selection on protein sequence, consideration at the nucleotide sequence level is also important. For example, when a certain nucleotide position with an elevated potentiality of the directional changes is the position responsible for coding the amino acid in a certain antigenic site, the probability of this amino acid change is expected to be higher than that for other amino acids in this or other antigenic sites. Additionally, if a certain H1N1/09 strain isolated in a certain geographical area has, in an antiviral drug-binding site, an oligonucleotide with the elevated potentiality of the directional changes, the strains and the area may be counted as a potentially hazardous strain and area. This type of information obtained by informatics methods will become increasingly important in accord with the increase of nucleotide sequences obtained from a wide variety of virus strains.</p>                            <h3 scrollto-destination=5307207 id="5307207" class="section-title" data-legacy-id=s3f>3.6. Retrospective time-series changes visualized for human viruses</h3>
<p>Invader viruses will change their sequences on balance between a stochastic process of mutation and selection pressure derived from various constraints, including constraints from hosts. As a model case to study the virus evolution after changing hosts, the 2009 pandemic was very informative because genomes of a large number of strains isolated in various stages, including those isolated in the very early stage, were sequenced. The finding that even in the late stage in the 2009 pandemic the H1N1/09 strains had the characteristics appreciably distinct from human seasonal viruses (Figs. <a reveal-id="DSR005F2" data-open="DSR005F2" class="link link-reveal link-table xref-fig">2</a> and <a reveal-id="DSR005F3" data-open="DSR005F3" class="link link-reveal link-table xref-fig">3</a>B) indicated that the sequence changes had not reached the presumable equilibrium state, which represented the sequence characteristics common among human seasonal strains. It will take a more extended period to reach the hypothesized equilibrium. Actually, by analysing retrospective changes of codon biases of human seasonal H1 and H3 strains isolated from 1918 to 2006, Wong <em>et al</em>.<sup><a reveal-id="DSR005C27" data-open="DSR005C27" class="link link-ref link-reveal xref-bibr">27</a></sup> found the overall reduction of G + C% during the course of their evolutions with Corresponding Analysis. We thus examined whether BLSOM could detect the retrospective long-period changes in codon biases in human seasonal H1 and H3 strains. On the codon-BLSOM listed in Fig. <a reveal-id="DSR005F3" data-open="DSR005F3" class="link link-reveal link-table xref-fig">3</a>A, human H1N1 or H3N2 strains that were isolated in the five different periods were separately coloured in light or dark blue for H1N1 or H3N2 strains, respectively (Fig. <a reveal-id="DSR005F3" data-open="DSR005F3" class="link link-reveal link-table xref-fig">3</a>D). Strains isolated before 1975 (shown in the panel ‘–1974’) were located around the border between the human and avian territories, and pandemic descendants (shown in the panels specifying the time period) moved apart from the avian territory (achromatic in Fig. <a reveal-id="DSR005F3" data-open="DSR005F3" class="link link-reveal link-table xref-fig">3</a>D). If human viruses had changed their sequences solely by stochastic processes, pandemic descendants should move primarily in a non-directional way from year to year. Absence of such non-directional movements indicated a directional pressure during the course of establishment of human seasonal strains after the onset of their new pandemics. This finding also showed that BLSOM had a power to visualize evolutionary histories of various subtype viruses, and more importantly to visualize any categories of strains, in which experimental and medical groups will be interested.</p>                            <h3 scrollto-destination=5307209 id="5307209" class="section-title" data-legacy-id=s3g>3.7. Segments separately analysed</h3>
<p>At the onset of a new pandemic, re-assortment of virus genome segments in a certain host (e.g. swine) and successive invasion of the new re-assortant into the human population were often essential.<sup><a reveal-id="DSR005C17" data-open="DSR005C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DSR005C18" data-open="DSR005C18" class="link link-ref link-reveal xref-bibr">18</a>,<a reveal-id="DSR005C24" data-open="DSR005C24" class="link link-ref link-reveal xref-bibr">24</a>,<a reveal-id="DSR005C32" data-open="DSR005C32" class="link link-ref link-reveal xref-bibr">32</a></sup> Therefore, we next analysed sequences of eight segments separately, which were derived from approximately 5300 strains including 170 H1N1/09 strains isolated in the early stage. The length of the shortest segment (segment 8) is approximately 0.8 kb, and therefore, enough clustering power can be expected as already shown in Fig. <a reveal-id="DSR005F1" data-open="DSR005F1" class="link link-reveal link-table xref-fig">1</a>A. Tetra-BLSOMs for eight segments are presented separately in Fig. <a reveal-id="DSR005F5" data-open="DSR005F5" class="link link-reveal link-table xref-fig">5</a>A; clear clustering of sequences according to host was observed for all segments, and this was true also for di-, tri-, and codon-BLSOMs (data not shown). Segment 2 of H1N1/09 was in close proximity to the human territory, but some other segments (e.g. segments 1 and 3) were within the avian territory. This similarity of the oligonucleotide composition of H1N1/09 with that of human, swine, or avian viruses was consistent with the findings obtained by conventional phylogenetic studies.<sup><a reveal-id="DSR005C17" data-open="DSR005C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DSR005C18" data-open="DSR005C18" class="link link-ref link-reveal xref-bibr">18</a></sup> For example, segment 1 in H1N1/09 was shown to be derived originally from avian strains in the phylogenetic studies, and sequences of segment 1 in the H1N1/09 strains isolated in the early stage were located within the avian territory on BLSOM (Fig. <a reveal-id="DSR005F5" data-open="DSR005F5" class="link link-reveal link-table xref-fig">5</a>A), supporting their evolutionary origin disclosed previously by the phylogenetic studies. During the course of the H1N1/09 evolution, sequence characteristics in segment 1 may have changed towards the characteristics of seasonal human viruses in order to adapt to the new host cellular environments, and this will be examined in the last part of this paper. Sequences of segment 2, which was evolutionarily derived from human strains, were located between human and avian territories, again supporting their evolutionary origin. </p>                        <a id="5307211" scrollto-destination="5307211"></a>
<div id="DSR005F5" data-id="DSR005F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/2/10.1093/dnares/dsr005/2/m_dsr00505.jpeg?Expires=2147483647&Signature=4A-gU458vc-jzl9ELFYJx2tWg6hhD9WkpBCYOXCszGPALLHK~TL-PzmNhvs~v2To6CaG3nd6wkCYgp-TCAahAeLAbtY9hmePqGoNNW0cEjVxWsGePCO9~C0mdEja6zx5DqQE3YkvlRktlHL8Q3bK6AzsO0IUEXU-jLADHwfNrqHCat1mKLc0udfHFxr6Ni46HjUtK2Lri5pIMwG9LFFonFfLaa2eZ9E5WvXqqJioQugK8ylOrv0msVj4UYbvmIBS-krMydsEQj4gafns4U0VybuT21AKwHDsrzTyFQmPeB3TIELqPVJ-KpXqrAcT4yzfSe~ZyEYN2gSRRtE8MLH9qQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Tetra-BLSOMs for eight genome segments derived from human, avian, and swine viruses. (A) Gene product name was listed along with the segment number. Lattice points were indicated in colours representing the hosts, as described in Fig. 1B, and thus a zone for H1N1/09 was in dark green. (B) Examples of avian strains whose segments were located in close proximity to human and/or swine segments were marked in light green or blue to specify the geographical areas where the subtype strains were isolated: segment 2 of H7N2 strains and segment 4 of H5N2 strains that were isolated in New York (NY), segment 4 of H5N2 strains isolated in Minnesota (MN), H6N1 for segment 6 isolated in Taiwan and Hong Kong (HK), and segment 6 of H9N2 strains isolated in Wisconsin (WI) and Hong Kong (HK). Segments of other avian strains were in red and segments from other hosts were achromatic, but a zone for H1N1/09 was marked in dark green for reference to help recognize the position in (A)." path-from-xml="dsr00505.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5307211/dsr00505.jpeg" path-from-xml="dsr00505.jpeg" target="_blank">View large</a><a section="5307211" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/2/10.1093/dnares/dsr005/2/dsr00505.jpeg?Expires=2147483647&Signature=2DE1kIUWKd8t1q0FqF66OObhsqBiDvPaAzBDqkiHxnvZBeU-Ztual672afMdlg8Gyq0U5RTHwE95sFgD1FoRGNr~asdU8~y2LURIy2cU9DdC1ylL3YZuukmgL-83eb8Heasp7YyRk5xSZI2zWac8oUUeukLCBfTFOTIY-9NU4hmJfeAhN8QBTPUlj37CHLhANmC~yMNCCSx2CfhV153SOddSxwC5tgWMfNAeO3SVTnfjPcit0j9yV5Z8shvrkXDRwvosld9iDzl294iqoOunVIPg7hwenZ5L91~F7NqQKbmmBabEw3poOQPSscyMpVSShBMztd17GuHQYGTqmJXpIQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5307211&ar=365741&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr00505.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Tetra-BLSOMs for eight genome segments derived from human, avian, and swine viruses. (A) Gene product name was listed along with the segment number. Lattice points were indicated in colours representing the hosts, as described in Fig. <a reveal-id="DSR005F1" data-open="DSR005F1" class="link link-reveal link-table xref-fig">1</a>B, and thus a zone for H1N1/09 was in dark green. (B) Examples of avian strains whose segments were located in close proximity to human and/or swine segments were marked in light green or blue to specify the geographical areas where the subtype strains were isolated: segment 2 of H7N2 strains and segment 4 of H5N2 strains that were isolated in New York (NY), segment 4 of H5N2 strains isolated in Minnesota (MN), H6N1 for segment 6 isolated in Taiwan and Hong Kong (HK), and segment 6 of H9N2 strains isolated in Wisconsin (WI) and Hong Kong (HK). Segments of other avian strains were in red and segments from other hosts were achromatic, but a zone for H1N1/09 was marked in dark green for reference to help recognize the position in (A).</p></div></div><div content-id="DSR005F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/2/10.1093/dnares/dsr005/2/m_dsr00505.jpeg?Expires=2147483647&Signature=4A-gU458vc-jzl9ELFYJx2tWg6hhD9WkpBCYOXCszGPALLHK~TL-PzmNhvs~v2To6CaG3nd6wkCYgp-TCAahAeLAbtY9hmePqGoNNW0cEjVxWsGePCO9~C0mdEja6zx5DqQE3YkvlRktlHL8Q3bK6AzsO0IUEXU-jLADHwfNrqHCat1mKLc0udfHFxr6Ni46HjUtK2Lri5pIMwG9LFFonFfLaa2eZ9E5WvXqqJioQugK8ylOrv0msVj4UYbvmIBS-krMydsEQj4gafns4U0VybuT21AKwHDsrzTyFQmPeB3TIELqPVJ-KpXqrAcT4yzfSe~ZyEYN2gSRRtE8MLH9qQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Tetra-BLSOMs for eight genome segments derived from human, avian, and swine viruses. (A) Gene product name was listed along with the segment number. Lattice points were indicated in colours representing the hosts, as described in Fig. 1B, and thus a zone for H1N1/09 was in dark green. (B) Examples of avian strains whose segments were located in close proximity to human and/or swine segments were marked in light green or blue to specify the geographical areas where the subtype strains were isolated: segment 2 of H7N2 strains and segment 4 of H5N2 strains that were isolated in New York (NY), segment 4 of H5N2 strains isolated in Minnesota (MN), H6N1 for segment 6 isolated in Taiwan and Hong Kong (HK), and segment 6 of H9N2 strains isolated in Wisconsin (WI) and Hong Kong (HK). Segments of other avian strains were in red and segments from other hosts were achromatic, but a zone for H1N1/09 was marked in dark green for reference to help recognize the position in (A)." path-from-xml="dsr00505.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5307211/dsr00505.jpeg" path-from-xml="dsr00505.jpeg" target="_blank">View large</a><a section="5307211" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/2/10.1093/dnares/dsr005/2/dsr00505.jpeg?Expires=2147483647&Signature=2DE1kIUWKd8t1q0FqF66OObhsqBiDvPaAzBDqkiHxnvZBeU-Ztual672afMdlg8Gyq0U5RTHwE95sFgD1FoRGNr~asdU8~y2LURIy2cU9DdC1ylL3YZuukmgL-83eb8Heasp7YyRk5xSZI2zWac8oUUeukLCBfTFOTIY-9NU4hmJfeAhN8QBTPUlj37CHLhANmC~yMNCCSx2CfhV153SOddSxwC5tgWMfNAeO3SVTnfjPcit0j9yV5Z8shvrkXDRwvosld9iDzl294iqoOunVIPg7hwenZ5L91~F7NqQKbmmBabEw3poOQPSscyMpVSShBMztd17GuHQYGTqmJXpIQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5307211&ar=365741&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr00505.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Tetra-BLSOMs for eight genome segments derived from human, avian, and swine viruses. (A) Gene product name was listed along with the segment number. Lattice points were indicated in colours representing the hosts, as described in Fig. <a reveal-id="DSR005F1" data-open="DSR005F1" class="link link-reveal link-table xref-fig">1</a>B, and thus a zone for H1N1/09 was in dark green. (B) Examples of avian strains whose segments were located in close proximity to human and/or swine segments were marked in light green or blue to specify the geographical areas where the subtype strains were isolated: segment 2 of H7N2 strains and segment 4 of H5N2 strains that were isolated in New York (NY), segment 4 of H5N2 strains isolated in Minnesota (MN), H6N1 for segment 6 isolated in Taiwan and Hong Kong (HK), and segment 6 of H9N2 strains isolated in Wisconsin (WI) and Hong Kong (HK). Segments of other avian strains were in red and segments from other hosts were achromatic, but a zone for H1N1/09 was marked in dark green for reference to help recognize the position in (A).</p></div></div><p>Importantly, approximately 5300 sequences were effectively characterized and visualized on one map, supporting efficient knowledge discovery such as those described in what follows. In Fig. <a reveal-id="DSR005F1" data-open="DSR005F1" class="link link-reveal link-table xref-fig">1</a>C (Avi Sub near Hum Sub), we present examples of avian subtype strains that were in close proximity to human and/or swine territories for four subtypes, along with the geographical information of places where the subtype strains were isolated. This provided the information of their similarity at a strain level, potentially unveiling their evolutionary histories at a strain level. In contrast, BLSOM analyses of individual segments presented in Fig. <a reveal-id="DSR005F5" data-open="DSR005F5" class="link link-reveal link-table xref-fig">5</a>A may provide evolutionary histories at a segment level. Visualization and identification of avian- or swine-virus segments whose oligonucleotide and codon compositions were closely related to those of humans should be valuable for predicting candidate strains that may cause pandemics among humans after segment re-assortment. For example, by summarizing potentially hazardous segments, we may specify avian strains that will come to resemble the human or swine strains with re-assortment of only a few segments. This type of information should be very valuable for gaining new perspectives on systematic surveillance of viruses presenting potential hazards. By analysing a large number of virus sequences collectively, BLSOM can efficiently discern such hazardous segments and strains on the basis of its strong visualization power.</p><p>Identification of codons or oligonucleotides that are preferred in individual hosts (Table <a reveal-id="DSR005TB1" data-open="DSR005TB1" class="link link-reveal link-table xref-fig">1</a>) may also provide novel information to aid the design of vaccine candidate strains with high growth rates for supporting high-yield vaccine production and/or the design of generation of recombinant influenza viruses containing various mutations in their genes,<sup><a reveal-id="DSR005C36" data-open="DSR005C36" class="link link-ref link-reveal xref-bibr">36</a></sup> which will stably be maintained through infection cycles by avoiding unfavourable codons and oligonucleotides.</p>                            <h3 scrollto-destination=5307214 id="5307214" class="section-title" data-legacy-id=s3h>3.8. Confirmation of sequence changes on a basis of sequence homology searches</h3>
<p>To further test the feasibility of the present strategy for predicting directional changes in H1N1/09 sequences, we compared the gene sequences of the strain that was isolated in the very early stage of the pandemic (California/04/2009) with the sequences from approximately 100 H1N1/09 strains that were isolated in the latest stage of the 2009 pandemic (after 1 December 2009). As observed in time-series changes on oligonucleotide- and codon-BLSOMs in Figs. <a reveal-id="DSR005F2" data-open="DSR005F2" class="link link-reveal link-table xref-fig">2</a> and <a reveal-id="DSR005F3" data-open="DSR005F3" class="link link-reveal link-table xref-fig">3</a>B, a significant level of sequence changes appeared to have accumulated in the H1N1/09 strains during the course of the 2009 pandemic. Next, we analysed base changes at a gene sequence level after alignment with BLAST search, by focusing only on protein-coding sequences because one purpose of the analysis was to recognize changes in codon usage. Furthermore, the exclusion of UTR sequences enabled us to avoid mistakes and/or uncertainties with regard to sequence alignments, which increased significantly for UTR sequences. For each gene of the 100 strains isolated at the latest stage in the 2009 pandemic, nucleotide positions where the base was changed from the sequence of the very early isolated strain were searched for. Then, by keeping focus on the base-changed positions, the number of the codons or oligonucleotides indicated in bold italic letters in the ‘Preferred in avian viruses’ column in Table <a reveal-id="DSR005TB1" data-open="DSR005TB1" class="link link-reveal link-table xref-fig">1</a> (i.e. codons or oligonucleotides predicted to decrees in the near future) that were actually lost or gained in each strain isolated in the latest stage was calculated and summed for the 100 strains (Dec or Inc in Table <a reveal-id="DSR005TB2" data-open="DSR005TB2" class="link link-reveal link-table xref-fig">2</a>). In this summation, the same mutation found for more than one strain was treated as one change, because it was uncertain whether this base change occurred independently. The number of the lost codons or oligonucleotides of attention exceeded generically the number of those gained (Table <a reveal-id="DSR005TB2" data-open="DSR005TB2" class="link link-reveal link-table xref-fig">2</a>) (red vertical bars in Fig. <a reveal-id="DSR005F6" data-open="DSR005F6" class="link link-reveal link-table xref-fig">6</a>), supporting the view that the unfavourable codons or oligonucleotides predicted with BLSOM analyses actually had a higher tendency to be lost at the gene sequence level. Notably, this tendency differed among segments. For example, the tendency was evident in segment 1 harbouring PB2 gene (Fig. <a reveal-id="DSR005F6" data-open="DSR005F6" class="link link-reveal link-table xref-fig">6</a>): approximately two times more losses than gains in total (97/185: Inc/Dec in Table <a reveal-id="DSR005TB2" data-open="DSR005TB2" class="link link-reveal link-table xref-fig">2</a>). It should be mentioned that the PB2 of the H1N1/09 was derived from an avian virus.<sup><a reveal-id="DSR005C17" data-open="DSR005C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DSR005C18" data-open="DSR005C18" class="link link-ref link-reveal xref-bibr">18</a></sup> In the case of the segment-2-harbouring PB1gene, which was originally derived from the human seasonal virus, the tendency was less evident and slightly reversed (Fig. <a reveal-id="DSR005F6" data-open="DSR005F6" class="link link-reveal link-table xref-fig">6</a>) (147/139 in Table <a reveal-id="DSR005TB2" data-open="DSR005TB2" class="link link-reveal link-table xref-fig">2</a>), suggesting that a large portion of the base changes observed in this gene might represent non-directional, stochastic mutation processes. </p>                        <a id="5307216" scrollto-destination="5307216"></a>
<div content-id="DSR005TB2" class="table-modal"><div class="table-wrap"><div id="DSR005TB2" class="table-wrap-title" data-id="DSR005TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Increase or decrease of codons or oligonucleotides favourable in avian and H1N1/09 viruses</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Segment 1 (Inc/Dec) </th><th>Segment 2 (Inc/Dec) </th><th>Segment 3 (Inc/Dec) </th><th>Segment 4 (Inc/Dec) </th><th>Segment 5 (Inc/Dec) </th><th>Segment 6 (Inc/Dec) </th><th>Segment 7 (Inc/Dec) </th><th>Segment 8 (Inc/Dec) </th></tr></thead><tbody><tr><td>Codon </td><td>16/31 </td><td>27/24 </td><td>20/19 </td><td>13/32 </td><td>9/18 </td><td>10/12 </td><td>5/10 </td><td>6/6 </td></tr><tr><td>Di </td><td>53/95 </td><td>81/71 </td><td>61/80 </td><td>60/68 </td><td>39/36 </td><td>45/35 </td><td>16/22 </td><td>14/18 </td></tr><tr><td>Tri </td><td>13/26 </td><td>17/17 </td><td>23/24 </td><td>20/19 </td><td>14/19 </td><td>14/25 </td><td>4/10 </td><td>4/5 </td></tr><tr><td>Tetra </td><td>15/33 </td><td>22/27 </td><td>24/34 </td><td>22/18 </td><td>18/19 </td><td>12/12 </td><td>5/8 </td><td>12/13 </td></tr><tr><td>Sum </td><td>97/185 </td><td>147/139 </td><td>128/157 </td><td>115/137 </td><td>80/92 </td><td>81/84 </td><td>30/50 </td><td>36/42 </td></tr></tbody></table></div><div content-id="DSR005TB2" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Segment 1 (Inc/Dec) </th><th>Segment 2 (Inc/Dec) </th><th>Segment 3 (Inc/Dec) </th><th>Segment 4 (Inc/Dec) </th><th>Segment 5 (Inc/Dec) </th><th>Segment 6 (Inc/Dec) </th><th>Segment 7 (Inc/Dec) </th><th>Segment 8 (Inc/Dec) </th></tr></thead><tbody><tr><td>Codon </td><td>16/31 </td><td>27/24 </td><td>20/19 </td><td>13/32 </td><td>9/18 </td><td>10/12 </td><td>5/10 </td><td>6/6 </td></tr><tr><td>Di </td><td>53/95 </td><td>81/71 </td><td>61/80 </td><td>60/68 </td><td>39/36 </td><td>45/35 </td><td>16/22 </td><td>14/18 </td></tr><tr><td>Tri </td><td>13/26 </td><td>17/17 </td><td>23/24 </td><td>20/19 </td><td>14/19 </td><td>14/25 </td><td>4/10 </td><td>4/5 </td></tr><tr><td>Tetra </td><td>15/33 </td><td>22/27 </td><td>24/34 </td><td>22/18 </td><td>18/19 </td><td>12/12 </td><td>5/8 </td><td>12/13 </td></tr><tr><td>Sum </td><td>97/185 </td><td>147/139 </td><td>128/157 </td><td>115/137 </td><td>80/92 </td><td>81/84 </td><td>30/50 </td><td>36/42 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Inc/Dec: the number of increase or decrease of codons or oligonucleotides of attention.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5307216" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR005TB2" class="table-wrap-title" data-id="DSR005TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Increase or decrease of codons or oligonucleotides favourable in avian and H1N1/09 viruses</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Segment 1 (Inc/Dec) </th><th>Segment 2 (Inc/Dec) </th><th>Segment 3 (Inc/Dec) </th><th>Segment 4 (Inc/Dec) </th><th>Segment 5 (Inc/Dec) </th><th>Segment 6 (Inc/Dec) </th><th>Segment 7 (Inc/Dec) </th><th>Segment 8 (Inc/Dec) </th></tr></thead><tbody><tr><td>Codon </td><td>16/31 </td><td>27/24 </td><td>20/19 </td><td>13/32 </td><td>9/18 </td><td>10/12 </td><td>5/10 </td><td>6/6 </td></tr><tr><td>Di </td><td>53/95 </td><td>81/71 </td><td>61/80 </td><td>60/68 </td><td>39/36 </td><td>45/35 </td><td>16/22 </td><td>14/18 </td></tr><tr><td>Tri </td><td>13/26 </td><td>17/17 </td><td>23/24 </td><td>20/19 </td><td>14/19 </td><td>14/25 </td><td>4/10 </td><td>4/5 </td></tr><tr><td>Tetra </td><td>15/33 </td><td>22/27 </td><td>24/34 </td><td>22/18 </td><td>18/19 </td><td>12/12 </td><td>5/8 </td><td>12/13 </td></tr><tr><td>Sum </td><td>97/185 </td><td>147/139 </td><td>128/157 </td><td>115/137 </td><td>80/92 </td><td>81/84 </td><td>30/50 </td><td>36/42 </td></tr></tbody></table></div><div content-id="DSR005TB2" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Segment 1 (Inc/Dec) </th><th>Segment 2 (Inc/Dec) </th><th>Segment 3 (Inc/Dec) </th><th>Segment 4 (Inc/Dec) </th><th>Segment 5 (Inc/Dec) </th><th>Segment 6 (Inc/Dec) </th><th>Segment 7 (Inc/Dec) </th><th>Segment 8 (Inc/Dec) </th></tr></thead><tbody><tr><td>Codon </td><td>16/31 </td><td>27/24 </td><td>20/19 </td><td>13/32 </td><td>9/18 </td><td>10/12 </td><td>5/10 </td><td>6/6 </td></tr><tr><td>Di </td><td>53/95 </td><td>81/71 </td><td>61/80 </td><td>60/68 </td><td>39/36 </td><td>45/35 </td><td>16/22 </td><td>14/18 </td></tr><tr><td>Tri </td><td>13/26 </td><td>17/17 </td><td>23/24 </td><td>20/19 </td><td>14/19 </td><td>14/25 </td><td>4/10 </td><td>4/5 </td></tr><tr><td>Tetra </td><td>15/33 </td><td>22/27 </td><td>24/34 </td><td>22/18 </td><td>18/19 </td><td>12/12 </td><td>5/8 </td><td>12/13 </td></tr><tr><td>Sum </td><td>97/185 </td><td>147/139 </td><td>128/157 </td><td>115/137 </td><td>80/92 </td><td>81/84 </td><td>30/50 </td><td>36/42 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Inc/Dec: the number of increase or decrease of codons or oligonucleotides of attention.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5307216" target="_blank">View Large</a></div></div>                        <a id="5307217" scrollto-destination="5307217"></a>
<div id="DSR005F6" data-id="DSR005F6" class="fig fig-section"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/2/10.1093/dnares/dsr005/2/m_dsr00506.jpeg?Expires=2147483647&Signature=hbm1HchDexjdRZDceMgiBM6xgdiLJJSmxHSwiBqadajQMtnrnEYAGk1AvPWD7jC~dH5JTXCf2TbN-KbLwX2dR-zyWcFrjOajOSUiYl3ksww33-vjFlXhHeF3q~UDOpybG-qCPvHjITSIJn9mmbHPR8MSsJ-JFNSOU3iM8i9KK3XZLD-AB43sEOZU4mE97Rv7JQ3y4CfekNvqITHsKPHPyIfOAVR1NInfQ2S19u3vpHfy9yYh5kpxUdrwPfnPM5-xmtsuptXKkwy~HI3RfPkyxew-ml8XHDqP16~ihqBpY7OY8O7AEK33jJFOFYflLa-gJpDLRc-x8bjL-LOUxYe4Mg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Sequence changes observed in the strains isolated in the latest stage in the 2009 pandemic from an H1N1/09 strain isolated in a very early stage. The ratios ‘Inc/Dec’ for Sum (summation of di-, tri-, and tetranucleotides and codons) in Tables 2 and 3 were shown in red and green, respectively." path-from-xml="dsr00506.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5307217/dsr00506.jpeg" path-from-xml="dsr00506.jpeg" target="_blank">View large</a><a section="5307217" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/2/10.1093/dnares/dsr005/2/dsr00506.jpeg?Expires=2147483647&Signature=uZ4iSFu3CTQmcoy4McQD2rMxt6PrfuUkRQTe9dcbBSFY9YghgvfyvlFzJm0SHEjgBs~W4TOU5IrU6x7VmKxoXN5yeO6tzWEUZwlL8XG5nie7q-FFghcV4KvphmqVeAJNdLgyIsNFsVj9R3X~EzOQpYd9G3zmQNARVcUCWJd1MckYgug-NBf~mrBTbTyVeXmYO4CiEdKoyS8ZFUwHHZ9c4u7ygT3t3YcHCSIaU4X0nrbaFETLnZfqmCN-B5vEzOVdhbopZlPerSDTXcrDY6~amNA9Zy7257BCcKQALJZcT-864Jba2UVH4XZUzDOpZwBp4k9fMzvnCE3imVw3NpURSg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5307217&ar=365741&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr00506.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Sequence changes observed in the strains isolated in the latest stage in the 2009 pandemic from an H1N1/09 strain isolated in a very early stage. The ratios ‘Inc/Dec’ for Sum (summation of di-, tri-, and tetranucleotides and codons) in Tables <a reveal-id="DSR005TB2" data-open="DSR005TB2" class="link link-reveal link-table xref-fig">2</a> and <a reveal-id="DSR005TB3" data-open="DSR005TB3" class="link link-reveal link-table xref-fig">3</a> were shown in red and green, respectively.</p></div></div><div content-id="DSR005F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/2/10.1093/dnares/dsr005/2/m_dsr00506.jpeg?Expires=2147483647&Signature=hbm1HchDexjdRZDceMgiBM6xgdiLJJSmxHSwiBqadajQMtnrnEYAGk1AvPWD7jC~dH5JTXCf2TbN-KbLwX2dR-zyWcFrjOajOSUiYl3ksww33-vjFlXhHeF3q~UDOpybG-qCPvHjITSIJn9mmbHPR8MSsJ-JFNSOU3iM8i9KK3XZLD-AB43sEOZU4mE97Rv7JQ3y4CfekNvqITHsKPHPyIfOAVR1NInfQ2S19u3vpHfy9yYh5kpxUdrwPfnPM5-xmtsuptXKkwy~HI3RfPkyxew-ml8XHDqP16~ihqBpY7OY8O7AEK33jJFOFYflLa-gJpDLRc-x8bjL-LOUxYe4Mg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Sequence changes observed in the strains isolated in the latest stage in the 2009 pandemic from an H1N1/09 strain isolated in a very early stage. The ratios ‘Inc/Dec’ for Sum (summation of di-, tri-, and tetranucleotides and codons) in Tables 2 and 3 were shown in red and green, respectively." path-from-xml="dsr00506.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5307217/dsr00506.jpeg" path-from-xml="dsr00506.jpeg" target="_blank">View large</a><a section="5307217" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/2/10.1093/dnares/dsr005/2/dsr00506.jpeg?Expires=2147483647&Signature=uZ4iSFu3CTQmcoy4McQD2rMxt6PrfuUkRQTe9dcbBSFY9YghgvfyvlFzJm0SHEjgBs~W4TOU5IrU6x7VmKxoXN5yeO6tzWEUZwlL8XG5nie7q-FFghcV4KvphmqVeAJNdLgyIsNFsVj9R3X~EzOQpYd9G3zmQNARVcUCWJd1MckYgug-NBf~mrBTbTyVeXmYO4CiEdKoyS8ZFUwHHZ9c4u7ygT3t3YcHCSIaU4X0nrbaFETLnZfqmCN-B5vEzOVdhbopZlPerSDTXcrDY6~amNA9Zy7257BCcKQALJZcT-864Jba2UVH4XZUzDOpZwBp4k9fMzvnCE3imVw3NpURSg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5307217&ar=365741&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr00506.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Sequence changes observed in the strains isolated in the latest stage in the 2009 pandemic from an H1N1/09 strain isolated in a very early stage. The ratios ‘Inc/Dec’ for Sum (summation of di-, tri-, and tetranucleotides and codons) in Tables <a reveal-id="DSR005TB2" data-open="DSR005TB2" class="link link-reveal link-table xref-fig">2</a> and <a reveal-id="DSR005TB3" data-open="DSR005TB3" class="link link-reveal link-table xref-fig">3</a> were shown in red and green, respectively.</p></div></div>                        <a id="5307218" scrollto-destination="5307218"></a>
<div content-id="DSR005TB3" class="table-modal"><div class="table-wrap"><div id="DSR005TB3" class="table-wrap-title" data-id="DSR005TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Increase or decrease of codons or oligonucleotides favourable in seasonal human viruses but not in H1N1/09</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Segment 1 (Inc/Dec) </th><th>Segment 2 (Inc/Dec) </th><th>Segment 3 (Inc/Dec) </th><th>Segment 4 (Inc/Dec) </th><th>Segment 5 (Inc/Dec) </th><th>Segment 6 (Inc/Dec) </th><th>Segment 7 (Inc/Dec) </th><th>Segment 8 (Inc/Dec) </th></tr></thead><tbody><tr><td>Codon </td><td>40/27 </td><td>36/31 </td><td>32/26 </td><td>34/16 </td><td>18/13 </td><td>25/12 </td><td>16/4 </td><td>10/12 </td></tr><tr><td>Di </td><td>25/22 </td><td>26/18 </td><td>26/12 </td><td>40/25 </td><td>19/9 </td><td>25/14 </td><td>14/3 </td><td>9/8 </td></tr><tr><td>Tri </td><td>33/20 </td><td>32/25 </td><td>33/14 </td><td>47/38 </td><td>22/9 </td><td>28/17 </td><td>16/3 </td><td>17/11 </td></tr><tr><td>Tetra </td><td>55/28 </td><td>47/33 </td><td>24/30 </td><td>47/39 </td><td>16/17 </td><td>36/18 </td><td>17/6 </td><td>13/9 </td></tr><tr><td>Sum </td><td>153/97 </td><td>141/107 </td><td>115/82 </td><td>168/118 </td><td>75/48 </td><td>114/61 </td><td>63/16 </td><td>49/40 </td></tr></tbody></table></div><div content-id="DSR005TB3" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Segment 1 (Inc/Dec) </th><th>Segment 2 (Inc/Dec) </th><th>Segment 3 (Inc/Dec) </th><th>Segment 4 (Inc/Dec) </th><th>Segment 5 (Inc/Dec) </th><th>Segment 6 (Inc/Dec) </th><th>Segment 7 (Inc/Dec) </th><th>Segment 8 (Inc/Dec) </th></tr></thead><tbody><tr><td>Codon </td><td>40/27 </td><td>36/31 </td><td>32/26 </td><td>34/16 </td><td>18/13 </td><td>25/12 </td><td>16/4 </td><td>10/12 </td></tr><tr><td>Di </td><td>25/22 </td><td>26/18 </td><td>26/12 </td><td>40/25 </td><td>19/9 </td><td>25/14 </td><td>14/3 </td><td>9/8 </td></tr><tr><td>Tri </td><td>33/20 </td><td>32/25 </td><td>33/14 </td><td>47/38 </td><td>22/9 </td><td>28/17 </td><td>16/3 </td><td>17/11 </td></tr><tr><td>Tetra </td><td>55/28 </td><td>47/33 </td><td>24/30 </td><td>47/39 </td><td>16/17 </td><td>36/18 </td><td>17/6 </td><td>13/9 </td></tr><tr><td>Sum </td><td>153/97 </td><td>141/107 </td><td>115/82 </td><td>168/118 </td><td>75/48 </td><td>114/61 </td><td>63/16 </td><td>49/40 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Inc/Dec: the number of increase or decrease of codons or oligonucleotides of attention.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5307218" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR005TB3" class="table-wrap-title" data-id="DSR005TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Increase or decrease of codons or oligonucleotides favourable in seasonal human viruses but not in H1N1/09</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Segment 1 (Inc/Dec) </th><th>Segment 2 (Inc/Dec) </th><th>Segment 3 (Inc/Dec) </th><th>Segment 4 (Inc/Dec) </th><th>Segment 5 (Inc/Dec) </th><th>Segment 6 (Inc/Dec) </th><th>Segment 7 (Inc/Dec) </th><th>Segment 8 (Inc/Dec) </th></tr></thead><tbody><tr><td>Codon </td><td>40/27 </td><td>36/31 </td><td>32/26 </td><td>34/16 </td><td>18/13 </td><td>25/12 </td><td>16/4 </td><td>10/12 </td></tr><tr><td>Di </td><td>25/22 </td><td>26/18 </td><td>26/12 </td><td>40/25 </td><td>19/9 </td><td>25/14 </td><td>14/3 </td><td>9/8 </td></tr><tr><td>Tri </td><td>33/20 </td><td>32/25 </td><td>33/14 </td><td>47/38 </td><td>22/9 </td><td>28/17 </td><td>16/3 </td><td>17/11 </td></tr><tr><td>Tetra </td><td>55/28 </td><td>47/33 </td><td>24/30 </td><td>47/39 </td><td>16/17 </td><td>36/18 </td><td>17/6 </td><td>13/9 </td></tr><tr><td>Sum </td><td>153/97 </td><td>141/107 </td><td>115/82 </td><td>168/118 </td><td>75/48 </td><td>114/61 </td><td>63/16 </td><td>49/40 </td></tr></tbody></table></div><div content-id="DSR005TB3" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Segment 1 (Inc/Dec) </th><th>Segment 2 (Inc/Dec) </th><th>Segment 3 (Inc/Dec) </th><th>Segment 4 (Inc/Dec) </th><th>Segment 5 (Inc/Dec) </th><th>Segment 6 (Inc/Dec) </th><th>Segment 7 (Inc/Dec) </th><th>Segment 8 (Inc/Dec) </th></tr></thead><tbody><tr><td>Codon </td><td>40/27 </td><td>36/31 </td><td>32/26 </td><td>34/16 </td><td>18/13 </td><td>25/12 </td><td>16/4 </td><td>10/12 </td></tr><tr><td>Di </td><td>25/22 </td><td>26/18 </td><td>26/12 </td><td>40/25 </td><td>19/9 </td><td>25/14 </td><td>14/3 </td><td>9/8 </td></tr><tr><td>Tri </td><td>33/20 </td><td>32/25 </td><td>33/14 </td><td>47/38 </td><td>22/9 </td><td>28/17 </td><td>16/3 </td><td>17/11 </td></tr><tr><td>Tetra </td><td>55/28 </td><td>47/33 </td><td>24/30 </td><td>47/39 </td><td>16/17 </td><td>36/18 </td><td>17/6 </td><td>13/9 </td></tr><tr><td>Sum </td><td>153/97 </td><td>141/107 </td><td>115/82 </td><td>168/118 </td><td>75/48 </td><td>114/61 </td><td>63/16 </td><td>49/40 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Inc/Dec: the number of increase or decrease of codons or oligonucleotides of attention.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5307218" target="_blank">View Large</a></div></div><p>In Table <a reveal-id="DSR005TB3" data-open="DSR005TB3" class="link link-reveal link-table xref-fig">3</a>, the number of codons or oligonucleotides favourable for seasonal human viruses but not for both H1N1/09 and avian viruses (i.e. codons or oligonucleotides predicted to increase in the near future) that were actually gained or lost in the strains isolated in the latest stage was listed (Inc or Dec in Table <a reveal-id="DSR005TB3" data-open="DSR005TB3" class="link link-reveal link-table xref-fig">3</a>). In this case, the number of the gained codons or oligonucleotides exceeded the number of those lost (green vertical bars in Fig. <a reveal-id="DSR005F6" data-open="DSR005F6" class="link link-reveal link-table xref-fig">6</a>), supporting the view that the favourable codons or oligonucleotides predicted with BLSOM were actually favourable for growth in human cellular environments. The tendency again differed among segments, and this was evident in segments 1, 6, and 7 but less evident in segment 2 (Table <a reveal-id="DSR005TB3" data-open="DSR005TB3" class="link link-reveal link-table xref-fig">3</a>) (Fig. <a reveal-id="DSR005F6" data-open="DSR005F6" class="link link-reveal link-table xref-fig">6</a>). General trend of the difference among segments was similar with that observed in Table <a reveal-id="DSR005TB2" data-open="DSR005TB2" class="link link-reveal link-table xref-fig">2</a>, but the direction was reversed. Although one important factor contributing to the difference among segments appeared to be due to the difference in their evolutionary histories, there should exist other factors which are related to adaptation mechanisms of an invader virus in a new host. For example, the high level of losses of unfavourable codons and oligonucleotides and of gains of favourable codons and oligonucleotides in segment 1 suggests a possibility that the PB2 protein and/or RNA of the invader viruses may feel foreign in the human cellular environment because of its interaction with various host cellular factors.<sup><a reveal-id="DSR005C25" data-open="DSR005C25" class="link link-ref link-reveal xref-bibr">25</a>,<a reveal-id="DSR005C32" data-open="DSR005C32" class="link link-ref link-reveal xref-bibr">32</a>,<a reveal-id="DSR005C33" data-open="DSR005C33" class="link link-ref link-reveal xref-bibr">33</a>,<a reveal-id="DSR005C37" data-open="DSR005C37" class="link link-ref link-reveal xref-bibr">37</a></sup> Detailed inspection of differences among genes may provide information about molecular mechanisms of adaptation processes in a new host, and therefore, additional information useful for predicting sequence changes which will occur in the invader virus genome in the near future. Effects of these predicted sequence changes on protein functions, including bindings with antiviral drugs and antibodies, can be studied in detail. </p>                            <h2 scrollto-destination=5307220 id="5307220" class="section-title" data-legacy-id=s4>Supplementary Data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Data are available at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=5307222 id="5307222" class="section-title" data-legacy-id=s5>Funding</h2>
<p>This work was supported by the Integrated Database Project and Grant-in-Aid for Scientific Research (C) and for Young Scientists (B) from the Ministry of Education, Culture, Sports, Science and Technology of Japan. </p>                            <h2 scrollto-destination=5307224 id="5307224" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We wish to thank Dr Kimihito Ito (the Research Center for Zoonosis Control, Hokkaido University) for valuable suggestions and discussions, and the editor for valuable comments. The computation was done in part at the Earth Simulator of Japan Agency for Marine-Earth Science and Technology.</p>                            <h2 scrollto-destination=5307226 id="5307226" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSR005C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR005C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bush</div> <div class="given-names">R.M.</div></div>,  <div class="name"><div class="surname">Bender</div> <div class="given-names">C.A.</div></div>,  <div class="name"><div class="surname">Subbarao</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Cox</div> <div class="given-names">N.J.</div></div>,  <div class="name"><div class="surname">Fitch</div> <div class="given-names">W.M.</div></div>. </div><div class="article-title">Predicting the evolution of human influenza A</div>, <div class="source">Science</div> , <div class="year">1999</div>, vol. <div class="volume">286</div> (pg. <div class="fpage">1921</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Predicting%20the%20evolution%20of%20human%20influenza%20A&amp;author=R.M.%20Bush&amp;author=C.A.%20Bender&amp;author=K.%20Subbarao&amp;author=N.J.%20Cox&amp;author=W.M.%20Fitch&amp;publication_year=1999&amp;journal=Science&amp;volume=286&amp;pages=1921-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.286.5446.1921" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.286.5446.1921" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.286.5446.1921"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10583948" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR005C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR005C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Suzuki</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Gojobori</div> <div class="given-names">T.</div></div>. </div><div class="article-title">A method for detecting positive selection at single amino acid sites</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">1999</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">1315</div>-<div class="lpage">28</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20method%20for%20detecting%20positive%20selection%20at%20single%20amino%20acid%20sites&amp;author=Y.%20Suzuki&amp;author=T.%20Gojobori&amp;publication_year=1999&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=16&amp;pages=1315-28" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a026042" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Foxfordjournals.molbev.a026042" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Foxfordjournals.molbev.a026042"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10563013" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR005C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR005C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Igarashi</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Ito</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Yoshida</div> <div class="given-names">R.</div></div>, et al. </div><div class="article-title">Predicting the antigenic structure of the pandemic (H1N1) 2009 influenza virus hemagglutinin</div>, <div class="source">PLoS One</div> , <div class="year">2010</div>, vol. <div class="volume">5</div> pg. <div class="fpage">e8553</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Predicting%20the%20antigenic%20structure%20of%20the%20pandemic%20(H1N1)%202009%20influenza%20virus%20hemagglutinin&amp;author=M.%20Igarashi&amp;author=K.%20Ito&amp;author=R.%20Yoshida&amp;publication_year=2010&amp;journal=PLoS%20One&amp;volume=5&amp;pages=e8553" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0008553" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0008553" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0008553"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20049332" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR005C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR005C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nei</div> <div class="given-names">M.</div></div>. </div>, <div class="source">Molecular Evolutionary Genetics</div> , <div class="year">1987</div><div class="publisher-loc">New York</div><div class="publisher-name">Columbia University Press</div></div></div></div></div><div content-id="DSR005C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR005C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kumar</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Nei</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Dudley</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Tamura</div> <div class="given-names">K.</div></div>. </div><div class="article-title">MEGA: a biologist-centric software for evolutionary analysis of DNA and protein sequences</div>, <div class="source">Brief Bioinform.</div> , <div class="year">2008</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">299</div>-<div class="lpage">306</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MEGA%3A%20a%20biologist-centric%20software%20for%20evolutionary%20analysis%20of%20DNA%20and%20protein%20sequences&amp;author=S.%20Kumar&amp;author=M.%20Nei&amp;author=J.%20Dudley&amp;author=K.%20Tamura&amp;publication_year=2008&amp;journal=Brief%20Bioinform.&amp;volume=9&amp;pages=299-306" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bib/bbn017" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbib%2Fbbn017" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbib%2Fbbn017"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18417537" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR005C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR005C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Abe</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Kanaya</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Kinouchi</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Informatics for unveiling hidden genome signatures</div>, <div class="source">Genome Res.</div> , <div class="year">2003</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">693</div>-<div class="lpage">702</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Informatics%20for%20unveiling%20hidden%20genome%20signatures&amp;author=T.%20Abe&amp;author=S.%20Kanaya&amp;author=M.%20Kinouchi&amp;publication_year=2003&amp;journal=Genome%20Res.&amp;volume=13&amp;pages=693-702" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.634603" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.634603" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.634603"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12671005" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR005C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR005C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Abe</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Sugawara</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Kinouchi</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Kanaya</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Ikemura</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Novel phylogenetic studies of genomic sequence fragments derived from uncultured microbe mixtures in environmental and clinical samples</div>, <div class="source">DNA Res.</div> , <div class="year">2005</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">281</div>-<div class="lpage">90</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Novel%20phylogenetic%20studies%20of%20genomic%20sequence%20fragments%20derived%20from%20uncultured%20microbe%20mixtures%20in%20environmental%20and%20clinical%20samples&amp;author=T.%20Abe&amp;author=H.%20Sugawara&amp;author=M.%20Kinouchi&amp;author=S.%20Kanaya&amp;author=T.%20Ikemura&amp;publication_year=2005&amp;journal=DNA%20Res.&amp;volume=12&amp;pages=281-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsi015" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsi015" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsi015"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16769690" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR005C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR005C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kohonen</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Self-organized formation of topologically correct feature maps</div>, <div class="source">Biol. Cybern.</div> , <div class="year">1982</div>, vol. <div class="volume">43</div> (pg. <div class="fpage">59</div>-<div class="lpage">69</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Self-organized%20formation%20of%20topologically%20correct%20feature%20maps&amp;author=T.%20Kohonen&amp;publication_year=1982&amp;journal=Biol.%20Cybern.&amp;volume=43&amp;pages=59-69" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00337288" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00337288" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00337288"> </span></div> </div></div></div></div></div><div content-id="DSR005C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR005C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kohonen</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Oja</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Simula</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Visa</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Kangas</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Engineering applications of the self-organizing map</div>, <div class="source">Proc. IEEE</div> , <div class="year">1996</div>, vol. <div class="volume">84</div> (pg. <div class="fpage">1358</div>-<div class="lpage">84</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Engineering%20applications%20of%20the%20self-organizing%20map&amp;author=T.%20Kohonen&amp;author=E.%20Oja&amp;author=O.%20Simula&amp;author=A.%20Visa&amp;author=J.%20Kangas&amp;publication_year=1996&amp;journal=Proc.%20IEEE&amp;volume=84&amp;pages=1358-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1109/5.537105" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1109%2F5.537105" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1109%2F5.537105"> </span></div> </div></div></div></div></div><div content-id="DSR005C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR005C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Karlin</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Campbell</div> <div class="given-names">A.M.</div></div>,  <div class="name"><div class="surname">Mrazek</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Comparative DNA analysis across diverse genomes</div>, <div class="source">Annu. Rev. Genet.</div> , <div class="year">1998</div>, vol. <div class="volume">32</div> (pg. <div class="fpage">185</div>-<div class="lpage">225</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20DNA%20analysis%20across%20diverse%20genomes&amp;author=S.%20Karlin&amp;author=A.M.%20Campbell&amp;author=J.%20Mrazek&amp;publication_year=1998&amp;journal=Annu.%20Rev.%20Genet.&amp;volume=32&amp;pages=185-225" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.genet.32.1.185" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.genet.32.1.185" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.genet.32.1.185"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9928479" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR005C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR005C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Uchiyama</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Abe</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Ikemura</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Watanabe</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Substrate-induced gene-expression screening of environmental metagenome libraries for isolation of catabolic genes</div>, <div class="source">Nat. Biotechnol.</div> , <div class="year">2005</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">88</div>-<div class="lpage">93</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Substrate-induced%20gene-expression%20screening%20of%20environmental%20metagenome%20libraries%20for%20isolation%20of%20catabolic%20genes&amp;author=T.%20Uchiyama&amp;author=T.%20Abe&amp;author=T.%20Ikemura&amp;author=K.%20Watanabe&amp;publication_year=2005&amp;journal=Nat.%20Biotechnol.&amp;volume=23&amp;pages=88-93" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt1048" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt1048" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt1048"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15608629" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR005C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR005C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hayashi</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Abe</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Sakamoto</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Direct cloning of genes encoding novel xylanases from human gut</div>, <div class="source">Can. J. Microbiol.</div> , <div class="year">2005</div>, vol. <div class="volume">51</div> (pg. <div class="fpage">251</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Direct%20cloning%20of%20genes%20encoding%20novel%20xylanases%20from%20human%20gut&amp;author=H.%20Hayashi&amp;author=T.%20Abe&amp;author=M.%20Sakamoto&amp;publication_year=2005&amp;journal=Can.%20J.%20Microbiol.&amp;volume=51&amp;pages=251-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/w04-136" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2Fw04-136" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2Fw04-136"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15920623" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR005C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR005C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kosaka</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Kato</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Shimoyama</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">The genome of <em>Pelotomaculum thermopropionicum</em> reveals niche-associated evolution in anaerobic microbiota</div>, <div class="source">Genome Res.</div> , <div class="year">2008</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">442</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20of%20Pelotomaculum%20thermopropionicum%20reveals%20niche-associated%20evolution%20in%20anaerobic%20microbiota&amp;author=T.%20Kosaka&amp;author=S.%20Kato&amp;author=T.%20Shimoyama&amp;publication_year=2008&amp;journal=Genome%20Res.&amp;volume=18&amp;pages=442-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.7136508" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.7136508" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.7136508"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18218977" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR005C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR005C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wilmes</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Andersson</div> <div class="given-names">A.F.</div></div>,  <div class="name"><div class="surname">Lefsrud</div> <div class="given-names">M.G.</div></div>, et al. </div><div class="article-title">Community proteogenomics highlights microbial strain-variant protein expression within activated sludge performing enhanced biological phosphorus removal</div>, <div class="source">ISME J.</div> , <div class="year">2008</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">853</div>-<div class="lpage">64</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Community%20proteogenomics%20highlights%20microbial%20strain-variant%20protein%20expression%20within%20activated%20sludge%20performing%20enhanced%20biological%20phosphorus%20removal&amp;author=P.%20Wilmes&amp;author=A.F.%20Andersson&amp;author=M.G.%20Lefsrud&amp;publication_year=2008&amp;journal=ISME%20J.&amp;volume=2&amp;pages=853-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ismej.2008.38" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fismej.2008.38" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fismej.2008.38"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18449217" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR005C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR005C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dick</div> <div class="given-names">G.J.</div></div>,  <div class="name"><div class="surname">Andersson</div> <div class="given-names">A.F.</div></div>,  <div class="name"><div class="surname">Baker</div> <div class="given-names">B.J.</div></div>, et al. </div><div class="article-title">Community-wide analysis of microbial genome sequence signatures</div>, <div class="source">Genome Biol.</div> , <div class="year">2009</div>, vol. <div class="volume">10</div> pg. <div class="fpage">R85</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Community-wide%20analysis%20of%20microbial%20genome%20sequence%20signatures&amp;author=G.J.%20Dick&amp;author=A.F.%20Andersson&amp;author=B.J.%20Baker&amp;publication_year=2009&amp;journal=Genome%20Biol.&amp;volume=10&amp;pages=R85" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2009-10-8-r85" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2009-10-8-r85" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2009-10-8-r85"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19698104" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR005C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR005C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="collab">Centers for Disease Control and Prevention</div><div class="article-title">Swine influenza A (H1N1) infection in two children—South California, March–April 2009</div>, <div class="source">Morb. Mortal. Wkly Rep.</div> , <div class="year">2009</div>, vol. <div class="volume">58</div> (pg. <div class="fpage">400</div>-<div class="lpage">2</div>)</div></div></div></div><div content-id="DSR005C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR005C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Smith</div> <div class="given-names">G.J.</div></div>,  <div class="name"><div class="surname">Vijaykrishna</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Bahl</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Origins and evolutionary genomics of the 2009 swine-origin H1N1 influenza A epidemic</div>, <div class="source">Nature</div> , <div class="year">2009</div>, vol. <div class="volume">459</div> (pg. <div class="fpage">1122</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Origins%20and%20evolutionary%20genomics%20of%20the%202009%20swine-origin%20H1N1%20influenza%20A%20epidemic&amp;author=G.J.%20Smith&amp;author=D.%20Vijaykrishna&amp;author=J.%20Bahl&amp;publication_year=2009&amp;journal=Nature&amp;volume=459&amp;pages=1122-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature08182" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature08182" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature08182"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19516283" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR005C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR005C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Garten</div> <div class="given-names">R.J.</div></div>,  <div class="name"><div class="surname">Davis</div> <div class="given-names">C.T.</div></div>,  <div class="name"><div class="surname">Russell</div> <div class="given-names">C.A.</div></div>, et al. </div><div class="article-title">Antigenic and genetic characteristics of swine-origin 2009 A (H1N1) influenza viruses circulating in humans</div>, <div class="source">Science</div> , <div class="year">2009</div>, vol. <div class="volume">325</div> (pg. <div class="fpage">197</div>-<div class="lpage">201</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Antigenic%20and%20genetic%20characteristics%20of%20swine-origin%202009%20A%20(H1N1)%20influenza%20viruses%20circulating%20in%20humans&amp;author=R.J.%20Garten&amp;author=C.T.%20Davis&amp;author=C.A.%20Russell&amp;publication_year=2009&amp;journal=Science&amp;volume=325&amp;pages=197-201" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1176225" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1176225" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1176225"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19465683" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR005C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR005C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hirahata</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Abe</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Tanaka</div> <div class="given-names">N.</div></div>, et al. </div><div class="article-title">Genome information broker for viruses</div>, <div class="source">Nucl. Acids Res.</div> , <div class="year">2006</div>, vol. <div class="volume">35</div> (pg. <div class="fpage">D339</div>-<div class="lpage">42</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20information%20broker%20for%20viruses&amp;author=M.%20Hirahata&amp;author=T.%20Abe&amp;author=N.%20Tanaka&amp;publication_year=2006&amp;journal=Nucl.%20Acids%20Res.&amp;volume=35&amp;pages=D339-42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkl1004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkl1004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkl1004"> </span></div> </div></div></div></div></div><div content-id="DSR005C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR005C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bao</div> <div class="given-names">Y.</div></div>. </div><div class="article-title">The influenza virus resource at the National Center for Biotechnology Information</div>, <div class="source">J. Virol.</div> , <div class="year">2008</div>, vol. <div class="volume">82</div> (pg. <div class="fpage">596</div>-<div class="lpage">601</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20influenza%20virus%20resource%20at%20the%20National%20Center%20for%20Biotechnology%20Information&amp;author=Y.%20Bao&amp;publication_year=2008&amp;journal=J.%20Virol.&amp;volume=82&amp;pages=596-601" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JVI.02005-07" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJVI.02005-07" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJVI.02005-07"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17942553" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR005C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR005C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kanaya</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Kinouchi</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Abe</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Analysis of codon usage diversity of bacterial genes with a self-organizing map (SOM): characterization of horizontally transferred genes with emphasis on the <em>E. coli</em> O157 genome</div>, <div class="source">Gene</div> , <div class="year">2001</div>, vol. <div class="volume">276</div> (pg. <div class="fpage">89</div>-<div class="lpage">99</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20codon%20usage%20diversity%20of%20bacterial%20genes%20with%20a%20self-organizing%20map%20(SOM)%3A%20characterization%20of%20horizontally%20transferred%20genes%20with%20emphasis%20on%20the%20E.%20coli%20O157%20genome&amp;author=S.%20Kanaya&amp;author=M.%20Kinouchi&amp;author=T.%20Abe&amp;publication_year=2001&amp;journal=Gene&amp;volume=276&amp;pages=89-99" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0378-1119(01)00673-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0378-1119(01)00673-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0378-1119(01)00673-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11591475" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR005C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR005C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">García-Sastre</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Inhibition of interferon-mediated antiviral responses by influenza A viruses and other negative-strand RNA viruses</div>, <div class="source">Virology</div> , <div class="year">2001</div>, vol. <div class="volume">279</div> (pg. <div class="fpage">375</div>-<div class="lpage">84</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Inhibition%20of%20interferon-mediated%20antiviral%20responses%20by%20influenza%20A%20viruses%20and%20other%20negative-strand%20RNA%20viruses&amp;author=A.%20Garc%C3%ADa-Sastre&amp;publication_year=2001&amp;journal=Virology&amp;volume=279&amp;pages=375-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/viro.2000.0756" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fviro.2000.0756" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fviro.2000.0756"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11162793" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR005C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR005C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Voinnet</div> <div class="given-names">O.</div></div>. </div><div class="article-title">Induction and suppression of RNA silencing: insights from viral infections</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="year">2005</div>, vol. <div class="volume">6</div> (pg. <div class="fpage">206</div>-<div class="lpage">20</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Induction%20and%20suppression%20of%20RNA%20silencing%3A%20insights%20from%20viral%20infections&amp;author=O.%20Voinnet&amp;publication_year=2005&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=6&amp;pages=206-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg1555" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg1555" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg1555"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15703763" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR005C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR005C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nelson</div> <div class="given-names">M.I.</div></div>,  <div class="name"><div class="surname">Holmes</div> <div class="given-names">E.C.</div></div>. </div><div class="article-title">The evolution of epidemic influenza</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="year">2007</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">196</div>-<div class="lpage">205</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20evolution%20of%20epidemic%20influenza&amp;author=M.I.%20Nelson&amp;author=E.C.%20Holmes&amp;publication_year=2007&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=8&amp;pages=196-205" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg2053" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg2053" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg2053"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17262054" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR005C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR005C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Alexey</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Moelling</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Dicer is involved in protection against influenza A virus infection</div>, <div class="source">J. Gen. Virol.</div> , <div class="year">2007</div>, vol. <div class="volume">88</div> (pg. <div class="fpage">2627</div>-<div class="lpage">35</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dicer%20is%20involved%20in%20protection%20against%20influenza%20A%20virus%20infection&amp;author=A.%20Alexey&amp;author=K.%20Moelling&amp;publication_year=2007&amp;journal=J.%20Gen.%20Virol.&amp;volume=88&amp;pages=2627-35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1099/vir.0.83103-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1099%2Fvir.0.83103-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1099%2Fvir.0.83103-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17872512" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR005C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR005C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rabadan</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Levine</div> <div class="given-names">A.J.</div></div>,  <div class="name"><div class="surname">Robins</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Comparison of avian and human influenza A viruses reveals a mutational bias on the viral genomes</div>, <div class="source">J. Virol.</div> , <div class="year">2006</div>, vol. <div class="volume">80</div> (pg. <div class="fpage">11887</div>-<div class="lpage">91</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparison%20of%20avian%20and%20human%20influenza%20A%20viruses%20reveals%20a%20mutational%20bias%20on%20the%20viral%20genomes&amp;author=R.%20Rabadan&amp;author=A.J.%20Levine&amp;author=H.%20Robins&amp;publication_year=2006&amp;journal=J.%20Virol.&amp;volume=80&amp;pages=11887-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JVI.01414-06" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJVI.01414-06" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJVI.01414-06"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16987977" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR005C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR005C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wong</div> <div class="given-names">E.H.</div></div>,  <div class="name"><div class="surname">Smith</div> <div class="given-names">D.K.</div></div>,  <div class="name"><div class="surname">Rabadan</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Peiris</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Poon</div> <div class="given-names">L.L.</div></div>. </div><div class="article-title">Codon usage bias and the evolution of influenza A viruses</div>, <div class="source">BMC Evol. Biol.</div> , <div class="year">2010</div>, vol. <div class="volume">10</div> pg. <div class="fpage">253</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20usage%20bias%20and%20the%20evolution%20of%20influenza%20A%20viruses&amp;author=E.H.%20Wong&amp;author=D.K.%20Smith&amp;author=R.%20Rabadan&amp;author=M.%20Peiris&amp;author=L.L.%20Poon&amp;publication_year=2010&amp;journal=BMC%20Evol.%20Biol.&amp;volume=10&amp;pages=253" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2148-10-253" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2148-10-253" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2148-10-253"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20723216" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR005C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR005C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ikemura</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Correlation between the abundance of <em>Escherichia coli</em> transfer RNAs and the occurrence of the respective codons in its protein genes</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">1981</div>, vol. <div class="volume">146</div> (pg. <div class="fpage">1</div>-<div class="lpage">21</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Correlation%20between%20the%20abundance%20of%20Escherichia%20coli%20transfer%20RNAs%20and%20the%20occurrence%20of%20the%20respective%20codons%20in%20its%20protein%20genes&amp;author=T.%20Ikemura&amp;publication_year=1981&amp;journal=J.%20Mol.%20Biol.&amp;volume=146&amp;pages=1-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0022-2836(81)90363-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0022-2836(81)90363-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0022-2836(81)90363-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6167728" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR005C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR005C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ikemura</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Codon usage and transfer RNA content in unicellular and multicellular organisms</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">1985</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">13</div>-<div class="lpage">34</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20usage%20and%20transfer%20RNA%20content%20in%20unicellular%20and%20multicellular%20organisms&amp;author=T.%20Ikemura&amp;publication_year=1985&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=2&amp;pages=13-34" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3916708" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR005C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR005C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sharp</div> <div class="given-names">P.M.</div></div>,  <div class="name"><div class="surname">Matassi</div> <div class="given-names">G.</div></div>. </div><div class="article-title">Codon usage and genome evolution</div>, <div class="source">Curr. Opin. Gen. Dev.</div> , <div class="year">1994</div>, vol. <div class="volume">4</div> (pg. <div class="fpage">851</div>-<div class="lpage">60</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20usage%20and%20genome%20evolution&amp;author=P.M.%20Sharp&amp;author=G.%20Matassi&amp;publication_year=1994&amp;journal=Curr.%20Opin.%20Gen.%20Dev.&amp;volume=4&amp;pages=851-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0959-437X(94)90070-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0959-437X(94)90070-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0959-437X(94)90070-1"> </span></div> </div></div></div></div></div><div content-id="DSR005C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR005C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sueoka</div> <div class="given-names">N.</div></div>. </div><div class="article-title">Intrastrand parity rules of DNA base composition and usage biases of synonymous codons</div>, <div class="source">J. Mol. Evol.</div> , <div class="year">1995</div>, vol. <div class="volume">40</div> (pg. <div class="fpage">318</div>-<div class="lpage">25</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Intrastrand%20parity%20rules%20of%20DNA%20base%20composition%20and%20usage%20biases%20of%20synonymous%20codons&amp;author=N.%20Sueoka&amp;publication_year=1995&amp;journal=J.%20Mol.%20Evol.&amp;volume=40&amp;pages=318-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00163236" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00163236" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00163236"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7723058" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR005C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR005C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Domingo</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Holland</div> <div class="given-names">J.J.</div></div>. </div><div class="article-title">RNA virus mutations and fitness for survival</div>, <div class="source">Annu. Rev. Microbiol.</div> , <div class="year">1997</div>, vol. <div class="volume">51</div> (pg. <div class="fpage">151</div>-<div class="lpage">78</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RNA%20virus%20mutations%20and%20fitness%20for%20survival&amp;author=E.%20Domingo&amp;author=J.J.%20Holland&amp;publication_year=1997&amp;journal=Annu.%20Rev.%20Microbiol.&amp;volume=51&amp;pages=151-78" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.micro.51.1.151" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.micro.51.1.151" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.micro.51.1.151"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9343347" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR005C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR005C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Greenbaum</div> <div class="given-names">B.D.</div></div>,  <div class="name"><div class="surname">Levine</div> <div class="given-names">A.J.</div></div>,  <div class="name"><div class="surname">Bhanot</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Rabadan</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Patterns of evolution and host gene mimicry in influenza and other RNA viruses</div>, <div class="source">PLoS Pathog.</div> , <div class="year">2008</div>, vol. <div class="volume">4</div> pg. <div class="fpage">e1000079</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Patterns%20of%20evolution%20and%20host%20gene%20mimicry%20in%20influenza%20and%20other%20RNA%20viruses&amp;author=B.D.%20Greenbaum&amp;author=A.J.%20Levine&amp;author=G.%20Bhanot&amp;author=R.%20Rabadan&amp;publication_year=2008&amp;journal=PLoS%20Pathog.&amp;volume=4&amp;pages=e1000079" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.ppat.1000079" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.ppat.1000079" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.ppat.1000079"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18535658" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR005C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR005C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Itzstein</div> <div class="given-names">M.V.</div></div>. </div><div class="article-title">The war against influenza: discovery and development of sialidase inhibitors</div>, <div class="source">Nat. Rev. Drug Discov.</div> , <div class="year">2007</div>, vol. <div class="volume">6</div> (pg. <div class="fpage">967</div>-<div class="lpage">74</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20war%20against%20influenza%3A%20discovery%20and%20development%20of%20sialidase%20inhibitors&amp;author=M.V.%20Itzstein&amp;publication_year=2007&amp;journal=Nat.%20Rev.%20Drug%20Discov.&amp;volume=6&amp;pages=967-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrd2400" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrd2400" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrd2400"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18049471" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR005C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR005C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Le</div> <div class="given-names">Q.M.</div></div>,  <div class="name"><div class="surname">Kiso</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Someya</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">Avian flu: isolation of drug-resistant H5N1 virus</div>, <div class="source">Nature</div> , <div class="year">2005</div>, vol. <div class="volume">437</div> pg. <div class="fpage">1108</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Avian%20flu%3A%20isolation%20of%20drug-resistant%20H5N1%20virus&amp;author=Q.M.%20Le&amp;author=M.%20Kiso&amp;author=K.%20Someya&amp;publication_year=2005&amp;journal=Nature&amp;volume=437&amp;pages=1108" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/4371108a" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F4371108a" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F4371108a"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16228009" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR005C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR005C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fodor</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Devenish</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Engelhardt</div> <div class="given-names">O.G.</div></div>, et al. </div><div class="article-title">Rescue of influenza A virus from recombinant DNA</div>, <div class="source">J. Virol.</div> , <div class="year">1999</div>, vol. <div class="volume">73</div> (pg. <div class="fpage">9679</div>-<div class="lpage">82</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rescue%20of%20influenza%20A%20virus%20from%20recombinant%20DNA&amp;author=E.%20Fodor&amp;author=L.%20Devenish&amp;author=O.G.%20Engelhardt&amp;publication_year=1999&amp;journal=J.%20Virol.&amp;volume=73&amp;pages=9679-82" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10516084" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR005C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR005C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yamada</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Hatta</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Staker</div> <div class="given-names">B. L.</div></div>, et al. </div><div class="article-title">Biological and structural characterization of a host-adapting amino acid in influenza virus</div>, <div class="source">PLoS Pathog.</div> , <div class="year">2010</div>, vol. <div class="volume">6</div> pg. <div class="fpage">e1001034</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Biological%20and%20structural%20characterization%20of%20a%20host-adapting%20amino%20acid%20in%20influenza%20virus&amp;author=S.%20Yamada&amp;author=M.%20Hatta&amp;author=B.%20L.%20Staker&amp;publication_year=2010&amp;journal=PLoS%20Pathog.&amp;volume=6&amp;pages=e1001034" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.ppat.1001034" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.ppat.1001034" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.ppat.1001034"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20700447" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2011. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/2.5), which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=365741&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/2/10.1093/dnares/dsr005/2/dsr005-dsr005supp.doc?Expires=2147483647&amp;Signature=dT7rTsU-XutPBcrcRjLVdTi8U8bHndJj-2-mlqDDLSqmQLBaop3hWTit1I-JdYrt8C4Ul47ejzMbeiIOuRdpylhycXZoudvMk6Cdq3CHwr0Uc7acK3ZgYvlBpRmI-UQZ6B~it3hn3q5I1dYmwOctsE4HjCzVVyrChaIzZs2XSt9-SQ1xtCvJhXdXCWdqircsEsL4nLzpSMsOoZF1t2NGRSy~tLw9y6jrKVHWpWcfFr9gOy5dfiUYuXWEPa6lDneq5FLSDYC-0h71kF6GPE4zdcVSvTp1KQmXCsvRF4UyaMWVsl-hB17xf9vBQpNwvIdTfQaV3qP-j6qmbDgMFOd3sA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - doc file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">262</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000290061400005&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">6</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsr005"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="365741"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000290061400005&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f18%2f2%2f125%2f365741">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000290061400005&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (6)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F18%2F2%2F125%2F365741" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/365741'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Prediction of Directional Changes of Influenza A Virus Genome Sequences with Emphasis on Pandemic H1N1/09 as a Model Case" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsr005");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f18%2f2%2f125%2f365741">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

