

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/25/2/183/4665695","@type":"ScholarlyArticle","name":"Construction of  PRDM9  allele-specific recombination maps in cattle using large-scale pedigree analysis and genome-wide single sperm genomics","datePublished":"2017-11-27","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/25/2","@type":"PublicationIssue","issueNumber":"2","datePublished":"2018-04-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsx048","keywords":["PRDM9","recombination","cattle","allele","single sperm genomics"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/25/2/183/4665695","author":[{"name":"Zhou, Yang","affiliation":"Animal Genomics and Improvement Laboratory, BARC, USDA-ARS, Beltsville, MD 20705, USA  Shaanxi Key Laboratory of Agricultural Molecular Biology, College of Animal Science and Technology, Northwest A&amp;F University, Yangling, Shaanxi 712100, China  Key Laboratory of Agricultural Animal Genetics, Breeding and Reproduction, Education Ministry of China, Huazhong Agricultural University, Wuhan, Hubei 430070, China","@type":"Person"},{"name":"Shen, Botong","affiliation":"Department of Animal and Avian Sciences, University of Maryland, College Park, MD 20742, USA","@type":"Person"},{"name":"Jiang, Jicai","affiliation":"Department of Animal and Avian Sciences, University of Maryland, College Park, MD 20742, USA","@type":"Person"},{"name":"Padhi, Abinash","affiliation":"Department of Animal and Avian Sciences, University of Maryland, College Park, MD 20742, USA","@type":"Person"},{"name":"Park, Ki-Eun","affiliation":"Department of Animal and Avian Sciences, University of Maryland, College Park, MD 20742, USA","@type":"Person"},{"name":"Oswalt, Adam","affiliation":"Select Sires Inc. 11740 U.S. 42 North, Plain City, Ohio 43064, USA","@type":"Person"},{"name":"Sattler, Charles G","affiliation":"Select Sires Inc. 11740 U.S. 42 North, Plain City, Ohio 43064, USA","@type":"Person"},{"name":"Telugu, Bhanu P","affiliation":"Department of Animal and Avian Sciences, University of Maryland, College Park, MD 20742, USA","@type":"Person"},{"name":"Chen, Hong","affiliation":"Shaanxi Key Laboratory of Agricultural Molecular Biology, College of Animal Science and Technology, Northwest A&amp;F University, Yangling, Shaanxi 712100, China","@type":"Person"},{"name":"Cole, John B","affiliation":"Animal Genomics and Improvement Laboratory, BARC, USDA-ARS, Beltsville, MD 20705, USA","@type":"Person"},{"name":"Liu, George E","affiliation":"Animal Genomics and Improvement Laboratory, BARC, USDA-ARS, Beltsville, MD 20705, USA","@type":"Person"},{"name":"Ma, Li","affiliation":"Department of Animal and Avian Sciences, University of Maryland, College Park, MD 20742, USA","@type":"Person"}],"description":"Abstract.  PRDM9 contributes to hybrid sterility and species evolution. However, its role is to be confirmed in cattle, a major domesticated livestock species. ","pageStart":"183","pageEnd":"194","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/2/1/m_dnares_25_2cover.png?Expires=2147483647&Signature=yfSnSDuEb8Ru1iEiczh9QiVIRujFlv5eQUGA2PixxhIP2ifTZWTC6Dh4YwZbNdfMcc6NjI5nmXhBBnXl~MO-9TlTyrmfE~hJJOwCiRzc0uAKzzN1WszCYLPzg4YPUGBBBvmjvYjtWeuxzT6HMUtRCVQ1Lykx1-FxL9MPbgabfK3LfM0lfiO9VFwha2rRByA4174pANV1EEHkTLYoj9VZAqG9NluBapcroduGQ77QCNh6n6sU6cwRapmFfSHO~HcCIdSas62HDXNUy3sCegaV5-lnkgDK34Vs8bE~hWbIr-4dZ69GanJYFX-rhZ4a7s6UpCy38dZKnOnrMjKCPyIBeg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Construction of  PRDM9  allele-specific recombination maps in cattle using large-scale pedigree analysis and genome-wide single sperm genomics","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/2/1/m_dnares_25_2cover.png?Expires=2147483647&Signature=yfSnSDuEb8Ru1iEiczh9QiVIRujFlv5eQUGA2PixxhIP2ifTZWTC6Dh4YwZbNdfMcc6NjI5nmXhBBnXl~MO-9TlTyrmfE~hJJOwCiRzc0uAKzzN1WszCYLPzg4YPUGBBBvmjvYjtWeuxzT6HMUtRCVQ1Lykx1-FxL9MPbgabfK3LfM0lfiO9VFwha2rRByA4174pANV1EEHkTLYoj9VZAqG9NluBapcroduGQ77QCNh6n6sU6cwRapmFfSHO~HcCIdSas62HDXNUy3sCegaV5-lnkgDK34Vs8bE~hWbIr-4dZ69GanJYFX-rhZ4a7s6UpCy38dZKnOnrMjKCPyIBeg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Construction of  PRDM9  allele-specific recombination maps in cattle using large-scale pedigree analysis and genome-wide single sperm genomics" />
    <meta property="og:description" content="Abstract.  PRDM9 contributes to hybrid sterility and species evolution. However, its role is to be confirmed in cattle, a major domesticated livestock species. " />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/25/2/183/4665695" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2017-11-27" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Zhou, Yang" /><meta name="citation_author_institution" content="Animal Genomics and Improvement Laboratory, BARC, USDA-ARS, Beltsville, MD 20705, USA" /><meta name="citation_author_institution" content="Shaanxi Key Laboratory of Agricultural Molecular Biology, College of Animal Science and Technology, Northwest A&amp;F University, Yangling, Shaanxi 712100, China" /><meta name="citation_author_institution" content="Key Laboratory of Agricultural Animal Genetics, Breeding and Reproduction, Education Ministry of China, Huazhong Agricultural University, Wuhan, Hubei 430070, China" /><meta name="citation_author" content="Shen, Botong" /><meta name="citation_author_institution" content="Department of Animal and Avian Sciences, University of Maryland, College Park, MD 20742, USA" /><meta name="citation_author" content="Jiang, Jicai" /><meta name="citation_author_institution" content="Department of Animal and Avian Sciences, University of Maryland, College Park, MD 20742, USA" /><meta name="citation_author" content="Padhi, Abinash" /><meta name="citation_author_institution" content="Department of Animal and Avian Sciences, University of Maryland, College Park, MD 20742, USA" /><meta name="citation_author" content="Park, Ki-Eun" /><meta name="citation_author_institution" content="Department of Animal and Avian Sciences, University of Maryland, College Park, MD 20742, USA" /><meta name="citation_author" content="Oswalt, Adam" /><meta name="citation_author_institution" content="Select Sires Inc. 11740 U.S. 42 North, Plain City, Ohio 43064, USA" /><meta name="citation_author" content="Sattler, Charles G" /><meta name="citation_author_institution" content="Select Sires Inc. 11740 U.S. 42 North, Plain City, Ohio 43064, USA" /><meta name="citation_author" content="Telugu, Bhanu P" /><meta name="citation_author_institution" content="Department of Animal and Avian Sciences, University of Maryland, College Park, MD 20742, USA" /><meta name="citation_author" content="Chen, Hong" /><meta name="citation_author_institution" content="Shaanxi Key Laboratory of Agricultural Molecular Biology, College of Animal Science and Technology, Northwest A&amp;F University, Yangling, Shaanxi 712100, China" /><meta name="citation_author" content="Cole, John B" /><meta name="citation_author_institution" content="Animal Genomics and Improvement Laboratory, BARC, USDA-ARS, Beltsville, MD 20705, USA" /><meta name="citation_author" content="Liu, George E" /><meta name="citation_author_institution" content="Animal Genomics and Improvement Laboratory, BARC, USDA-ARS, Beltsville, MD 20705, USA" /><meta name="citation_author" content="Ma, Li" /><meta name="citation_author_institution" content="Department of Animal and Avian Sciences, University of Maryland, College Park, MD 20742, USA" /><meta name="citation_title" content="Construction of PRDM9 allele-specific recombination maps in cattle using large-scale pedigree analysis and genome-wide single sperm genomics" /><meta name="citation_firstpage" content="183" /><meta name="citation_lastpage" content="194" /><meta name="citation_doi" content="10.1093/dnares/dsx048" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="25" /><meta name="citation_issue" content="2" /><meta name="citation_publication_date" content="2018/04/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Why sex and recombination?,;  citation_journal_title=Science;  citation_year=1998;  citation_volume=281;  citation_pages=1986-90; " /><meta name="citation_reference" content="citation_title=Estimating recombination rates from population-genetic data;  citation_journal_title=Nat. Rev. Genet;  citation_year=2003;  citation_volume=4;  citation_pages=959-68; " /><meta name="citation_reference" content="citation_title=Where the crossovers are: recombination distributions in mammals;  citation_journal_title=Nat. Rev. Genet.;  citation_year=2004;  citation_volume=5;  citation_pages=413-24; " /><meta name="citation_reference" content="citation_title=An evolutionary view of human recombination;  citation_journal_title=Nat. Rev. Genet.;  citation_year=2007;  citation_volume=8;  citation_pages=23-34; " /><meta name="citation_reference" content="citation_title=Mammalian recombination hot spots: properties, control and evolution;  citation_journal_title=Nat. Rev. Genet;  citation_year=2010;  citation_volume=11;  citation_pages=221-33; " /><meta name="citation_reference" content="citation_title=Prdm9 controls activation of mammalian recombination hotspots;  citation_journal_title=Science;  citation_year=2010;  citation_volume=327;  citation_pages=835" /><meta name="citation_reference" content="citation_title=PRDM9 is a major determinant of meiotic recombination hotspots in humans and mice;  citation_journal_title=Science;  citation_year=2010;  citation_volume=327;  citation_pages=836-840; " /><meta name="citation_reference" content="citation_title=PRDM9 variation strongly influences recombination hot-spot activity and meiotic instability in humans;  citation_journal_title=Nat. Genet;  citation_year=2010;  citation_volume=42;  citation_pages=859-863; " /><meta name="citation_reference" content="citation_title=Drive against hotspot motifs in primates implicates the PRDM9 gene in meiotic recombination;  citation_journal_title=Science;  citation_year=2010;  citation_volume=327;  citation_pages=876-879; " /><meta name="citation_reference" content="citation_title=Prdm9 polymorphism unveils mouse evolutionary tracks;  citation_journal_title=DNA Res;  citation_year=2014;  citation_volume=21;  citation_pages=315-326; " /><meta name="citation_reference" content="citation_title=Primate evolution of the recombination regulator PRDM9;  citation_journal_title=Nat. Commun.;  citation_year=2014;  citation_volume=5;  citation_pages=4370" /><meta name="citation_reference" content="citation_title=Evidence of positive selection and concerted evolution in the rapidly evolving PRDM9 zinc finger domain in goats and sheep;  citation_journal_title=Anim. Genet.;  citation_year=2016;  citation_volume=47;  citation_pages=740-751; " /><meta name="citation_reference" content="citation_title=What are the genomic drivers of the rapid evolution of PRDM9?;  citation_journal_title=Trends Genet.;  citation_year=2011;  citation_volume=27;  citation_pages=165-171; " /><meta name="citation_reference" content="citation_title=The Red Queen model of recombination hotspots evolution in the light of archaic and modern human genomes;  citation_journal_title=PLoS Genet;  citation_year=2014;  citation_volume=10;  citation_pages=e1004790" /><meta name="citation_reference" content="citation_title=Diversity of Prdm9 zinc finger array in wild mice unravels new facets of the evolutionary turnover of this coding minisatellite;  citation_journal_title=PLoS One;  citation_year=2014;  citation_volume=9;  citation_pages=e85021" /><meta name="citation_reference" content="citation_title=Death of PRDM9 coincides with stabilization of the recombination landscape in the dog genome;  citation_journal_title=Genome Res;  citation_year=2012;  citation_volume=22;  citation_pages=51-63; " /><meta name="citation_reference" content="citation_title=Genetic recombination is targeted towards gene promoter regions in dogs;  citation_journal_title=PLoS Genet;  citation_year=2013;  citation_volume=9;  citation_pages=e1003984" /><meta name="citation_reference" content="citation_title=Zinc finger domain of the PRDM9 gene on chromosome 1 exhibits high diversity in ruminants but its paralog PRDM7 contains multiple disruptive mutations;  citation_journal_title=PLoS One;  citation_year=2016;  citation_volume=11;  citation_pages=e0156159" /><meta name="citation_reference" content="citation_title=Ruminant-specific multiple duplication events of PRDM9 before speciation;  citation_journal_title=BMC Evol. Biol.,;  citation_year=2017;  citation_volume=17;  citation_pages=79" /><meta name="citation_reference" content="citation_title=Evidence for two independent domestications of cattle;  citation_journal_title=Proc. Natl. Acad. Sci;  citation_year=1994;  citation_volume=91;  citation_pages=2757-2761; " /><meta name="citation_reference" content="citation_title=Inferring population size history from large samples of genome-wide molecular data-an approximate Bayesian computation approach;  citation_journal_title=PLoS Genet;  citation_year=2016;  citation_volume=12;  citation_pages=e1005877" /><meta name="citation_reference" content="citation_title=Fine-scale recombination rate differences between sexes, populations and individuals;  citation_journal_title=Nature;  citation_year=2010;  citation_volume=467;  citation_pages=1099-1103; " /><meta name="citation_reference" content="citation_title=A high-resolution single nucleotide polymorphism genetic map of the mouse genome;  citation_journal_title=PLoS Biol.;  citation_year=2006;  citation_volume=4;  citation_pages=e395" /><meta name="citation_reference" content="citation_title=The genetic architecture of natural variation in recombination rate in Drosophila melanogaster;  citation_journal_title=PLoS Genet.;  citation_year=2016;  citation_volume=12;  citation_pages=e1005951" /><meta name="citation_reference" content="citation_title=Recombination rate variation and speciation: theoretical predictions and empirical results from rabbits and mice;  citation_journal_title=Philos. Trans. R. Soc. B;  citation_year=2012;  citation_volume=367;  citation_pages=409-421; " /><meta name="citation_reference" content="citation_title=Hybrid sterility locus on chromosome X controls meiotic recombination rate in mouse;  citation_journal_title=PLoS Genet.;  citation_year=2016;  citation_volume=12;  citation_pages=e1005906" /><meta name="citation_reference" content="citation_title=Recombination initiation maps of individual human genomes;  citation_journal_title=Science;  citation_year=2014;  citation_volume=346;  citation_pages=1256442" /><meta name="citation_reference" content="citation_title=PRDM9 drives evolutionary erosion of hotspots in Mus musculus through haplotype-specific initiation of meiotic recombination;  citation_journal_title=PLoS Genet;  citation_year=2015;  citation_volume=11;  citation_pages=e1004916" /><meta name="citation_reference" content="citation_title=Genetic links between recombination and speciation;  citation_journal_title=PLoS Genet.;  citation_year=2016;  citation_volume=12;  citation_pages=e1006066" /><meta name="citation_reference" content="citation_title=Re-engineering the zinc fingers of PRDM9 reverses hybrid sterility in mice;  citation_journal_title=Nature;  citation_year=2016;  citation_volume=530;  citation_pages=171-176; " /><meta name="citation_reference" content="citation_title=Cattle sex-specific recombination and genetic control from a large pedigree analysis;  citation_journal_title=PLoS Genet.;  citation_year=2015;  citation_volume=11;  citation_pages=e1005387" /><meta name="citation_reference" content="citation_title=Comparative recombination rates in the rat, mouse, and human genomes;  citation_journal_title=Genome Res;  citation_year=2004;  citation_volume=14;  citation_pages=528-538; " /><meta name="citation_reference" content="citation_title=The recombinational anatomy of a mouse chromosome;  citation_journal_title=PLoS Genet;  citation_year=2008;  citation_volume=4;  citation_pages=e1000119" /><meta name="citation_reference" content="citation_title=Coding and noncoding variants in HFM1, MLH3, MSH4, MSH5, RNF212, and RNF212B affect recombination rate in cattle;  citation_journal_title=Genome Res.;  citation_year=2016;  citation_volume=26;  citation_pages=1323-1332; " /><meta name="citation_reference" content="citation_title=High resolution localization of recombination hot spots using sperm typing;  citation_journal_title=Nat. Genet.;  citation_year=1994;  citation_volume=7;  citation_pages=420-424; " /><meta name="citation_reference" content="citation_title=Genome-wide single-cell analysis of recombination activity and de novo mutation rates in human sperm;  citation_journal_title=Cell;  citation_year=2012;  citation_volume=150;  citation_pages=402-412; " /><meta name="citation_reference" content="citation_title=Contrasting methods of quantifying fine structure of human recombination;  citation_journal_title=Annu. Rev. Genom. Hum. Genet.;  citation_year=2010;  citation_volume=11;  citation_pages=45" /><meta name="citation_reference" content="citation_title=Probing meiotic recombination and aneuploidy of single sperm cells by whole-genome sequencing;  citation_journal_title=Science;  citation_year=2012;  citation_volume=338;  citation_pages=1627-1630; " /><meta name="citation_reference" content="citation_title=The Ensembl genome database project;  citation_journal_title=Nucl. Acids Res;  citation_year=2002;  citation_volume=30;  citation_pages=38-41; " /><meta name="citation_reference" content="citation_title=Primer-BLAST: a tool to design target-specific primers for polymerase chain reaction;  citation_journal_title=BMC Bioinformatics;  citation_year=2012;  citation_volume=13;  citation_pages=1" /><meta name="citation_reference" content="citation_title=A whole-genome assembly of the domestic cow, Bos taurus;  citation_journal_title=Genome Biol.;  citation_year=2009;  citation_volume=10;  citation_issue=1;  citation_pages=R42" /><meta name="citation_reference" content="citation_title=Proteomic analysis of GLUT4 storage vesicles reveals tumor suppressor candidate 5 (TUSC5) as a novel regulator of insulin action in adipocytes;  citation_journal_title=J. Biol. Chem.;  citation_year=2015;  citation_volume=290;  citation_pages=23528-23542; " /><meta name="citation_reference" content="citation_title=R: a language and environment for statistical computing; citation_author=R, CT;  citation_journal_title=Vienna, Austria: R Foundation Stat. Comput;  citation_year=2014;  citation_volume=5;  citation_pages=299-314; " /><meta name="citation_reference" content="citation_title=Accelerated evolution of the Prdm9 speciation gene across diverse metazoan taxa;  citation_journal_title=PLoS Genet;  citation_year=2009;  citation_volume=5;  citation_pages=e1000753" /><meta name="citation_reference" content="citation_title=De novo prediction of DNA-binding specificities for Cys2His2 zinc finger proteins;  citation_journal_title=Nucl. Acids Res;  citation_year=2014;  citation_volume=42;  citation_pages=97-108; " /><meta name="citation_reference" content="citation_title=The evolutionary turnover of recombination hot spots contributes to speciation in mice;  citation_journal_title=Genes Dev.;  citation_year=2016;  citation_volume=30;  citation_pages=266-280; " /><meta name="citation_reference" content="citation_title=Genetic variants in REC8, RNF212, and PRDM9 influence male recombination in cattle;  citation_journal_title=PLoS Genet.;  citation_year=2012;  citation_volume=8;  citation_pages=e1002854" /><meta name="citation_reference" content="citation_title=The case of the fickle fingers: how the PRDM9 zinc finger protein specifies meiotic recombination hotspots in humans;  citation_journal_title=PLoS Biol;  citation_year=2011;  citation_volume=9;  citation_pages=e1001211" /><meta name="citation_reference" content="citation_title=Mouse PRDM9 DNA-binding specificity determines sites of histone H3 lysine 4 trimethylation for initiation of meiotic recombination;  citation_journal_title=PLoS Biol;  citation_year=2011;  citation_volume=9;  citation_pages=e1001176" /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/25/2/183/24662208/dsx048.pdf" /><meta name="description" content="Abstract.  PRDM9 contributes to hybrid sterility and species evolution. However, its role is to be confirmed in cattle, a major domesticated livestock species. " />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/25/2/183/4665695" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/25/2/183/4665695" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/25/2/183/4665695");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2796125&account_list=2796125,2777753&authnips=128.235.12.133&doi=10.1093/dnares/dsx048';
    </script>
<title>
	Construction of PRDM9 allele-specific recombination maps in cattle using large-scale pedigree analysis and genome-wide single sperm genomics | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./4665695" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0Bf59PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJaaG91LCBZYW5nIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQW5pbWFsIEdlbm9taWNzIGFuZCBJbXByb3ZlbWVudCBMYWJvcmF0b3J5LCBCQVJDLCBVU0RBLUFSUywgQmVsdHN2aWxsZSwgTUQgMjA3MDUsIFVTQSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlNoYWFueGkgS2V5IExhYm9yYXRvcnkgb2YgQWdyaWN1bHR1cmFsIE1vbGVjdWxhciBCaW9sb2d5LCBDb2xsZWdlIG9mIEFuaW1hbCBTY2llbmNlIGFuZCBUZWNobm9sb2d5LCBOb3J0aHdlc3QgQSZhbXA7RiBVbml2ZXJzaXR5LCBZYW5nbGluZywgU2hhYW54aSA3MTIxMDAsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2V5IExhYm9yYXRvcnkgb2YgQWdyaWN1bHR1cmFsIEFuaW1hbCBHZW5ldGljcywgQnJlZWRpbmcgYW5kIFJlcHJvZHVjdGlvbiwgRWR1Y2F0aW9uIE1pbmlzdHJ5IG9mIENoaW5hLCBIdWF6aG9uZyBBZ3JpY3VsdHVyYWwgVW5pdmVyc2l0eSwgV3VoYW4sIEh1YmVpIDQzMDA3MCwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTaGVuLCBCb3RvbmciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEFuaW1hbCBhbmQgQXZpYW4gU2NpZW5jZXMsIFVuaXZlcnNpdHkgb2YgTWFyeWxhbmQsIENvbGxlZ2UgUGFyaywgTUQgMjA3NDIsIFVTQSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkppYW5nLCBKaWNhaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQW5pbWFsIGFuZCBBdmlhbiBTY2llbmNlcywgVW5pdmVyc2l0eSBvZiBNYXJ5bGFuZCwgQ29sbGVnZSBQYXJrLCBNRCAyMDc0MiwgVVNBIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iUGFkaGksIEFiaW5hc2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEFuaW1hbCBhbmQgQXZpYW4gU2NpZW5jZXMsIFVuaXZlcnNpdHkgb2YgTWFyeWxhbmQsIENvbGxlZ2UgUGFyaywgTUQgMjA3NDIsIFVTQSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlBhcmssIEtpLUV1biIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQW5pbWFsIGFuZCBBdmlhbiBTY2llbmNlcywgVW5pdmVyc2l0eSBvZiBNYXJ5bGFuZCwgQ29sbGVnZSBQYXJrLCBNRCAyMDc0MiwgVVNBIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iT3N3YWx0LCBBZGFtIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iU2VsZWN0IFNpcmVzIEluYy4gMTE3NDAgVS5TLiA0MiBOb3J0aCwgUGxhaW4gQ2l0eSwgT2hpbyA0MzA2NCwgVVNBIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2F0dGxlciwgQ2hhcmxlcyBHIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iU2VsZWN0IFNpcmVzIEluYy4gMTE3NDAgVS5TLiA0MiBOb3J0aCwgUGxhaW4gQ2l0eSwgT2hpbyA0MzA2NCwgVVNBIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iVGVsdWd1LCBCaGFudSBQIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBBbmltYWwgYW5kIEF2aWFuIFNjaWVuY2VzLCBVbml2ZXJzaXR5IG9mIE1hcnlsYW5kLCBDb2xsZWdlIFBhcmssIE1EIDIwNzQyLCBVU0EiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJDaGVuLCBIb25nIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iU2hhYW54aSBLZXkgTGFib3JhdG9yeSBvZiBBZ3JpY3VsdHVyYWwgTW9sZWN1bGFyIEJpb2xvZ3ksIENvbGxlZ2Ugb2YgQW5pbWFsIFNjaWVuY2UgYW5kIFRlY2hub2xvZ3ksIE5vcnRod2VzdCBBJmFtcDtGIFVuaXZlcnNpdHksIFlhbmdsaW5nLCBTaGFhbnhpIDcxMjEwMCwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJDb2xlLCBKb2huIEIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJBbmltYWwgR2Vub21pY3MgYW5kIEltcHJvdmVtZW50IExhYm9yYXRvcnksIEJBUkMsIFVTREEtQVJTLCBCZWx0c3ZpbGxlLCBNRCAyMDcwNSwgVVNBIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTGl1LCBHZW9yZ2UgRSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkFuaW1hbCBHZW5vbWljcyBhbmQgSW1wcm92ZW1lbnQgTGFib3JhdG9yeSwgQkFSQywgVVNEQS1BUlMsIEJlbHRzdmlsbGUsIE1EIDIwNzA1LCBVU0EiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJNYSwgTGkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEFuaW1hbCBhbmQgQXZpYW4gU2NpZW5jZXMsIFVuaXZlcnNpdHkgb2YgTWFyeWxhbmQsIENvbGxlZ2UgUGFyaywgTUQgMjA3NDIsIFVTQSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iQ29uc3RydWN0aW9uIG9mIFBSRE05IGFsbGVsZS1zcGVjaWZpYyByZWNvbWJpbmF0aW9uIG1hcHMgaW4gY2F0dGxlIHVzaW5nIGxhcmdlLXNjYWxlIHBlZGlncmVlIGFuYWx5c2lzIGFuZCBnZW5vbWUtd2lkZSBzaW5nbGUgc3Blcm0gZ2Vub21pY3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIxODMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjE5NCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzeDA0OCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIyNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDE4LzA0LzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9V2h5IHNleCBhbmQgcmVjb21iaW5hdGlvbj8sOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2llbmNlOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTI4MTsgIGNpdGF0aW9uX3BhZ2VzPTE5ODYtOTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVzdGltYXRpbmcgcmVjb21iaW5hdGlvbiByYXRlcyBmcm9tIHBvcHVsYXRpb24tZ2VuZXRpYyBkYXRhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIFJldi4gR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTk1OS02ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9V2hlcmUgdGhlIGNyb3Nzb3ZlcnMgYXJlOiByZWNvbWJpbmF0aW9uIGRpc3RyaWJ1dGlvbnMgaW4gbWFtbWFsczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBSZXYuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9NDEzLTI0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBldm9sdXRpb25hcnkgdmlldyBvZiBodW1hbiByZWNvbWJpbmF0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIFJldi4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz0yMy0zNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWFtbWFsaWFuIHJlY29tYmluYXRpb24gaG90IHNwb3RzOiBwcm9wZXJ0aWVzLCBjb250cm9sIGFuZCBldm9sdXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gUmV2LiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTIyMS0zMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJkbTkgY29udHJvbHMgYWN0aXZhdGlvbiBvZiBtYW1tYWxpYW4gcmVjb21iaW5hdGlvbiBob3RzcG90czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0zMjc7ICBjaXRhdGlvbl9wYWdlcz04MzUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QUkRNOSBpcyBhIG1ham9yIGRldGVybWluYW50IG9mIG1laW90aWMgcmVjb21iaW5hdGlvbiBob3RzcG90cyBpbiBodW1hbnMgYW5kIG1pY2U7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MzI3OyAgY2l0YXRpb25fcGFnZXM9ODM2LTg0MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UFJETTkgdmFyaWF0aW9uIHN0cm9uZ2x5IGluZmx1ZW5jZXMgcmVjb21iaW5hdGlvbiBob3Qtc3BvdCBhY3Rpdml0eSBhbmQgbWVpb3RpYyBpbnN0YWJpbGl0eSBpbiBodW1hbnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NDI7ICBjaXRhdGlvbl9wYWdlcz04NTktODYzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ecml2ZSBhZ2FpbnN0IGhvdHNwb3QgbW90aWZzIGluIHByaW1hdGVzIGltcGxpY2F0ZXMgdGhlIFBSRE05IGdlbmUgaW4gbWVpb3RpYyByZWNvbWJpbmF0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTMyNzsgIGNpdGF0aW9uX3BhZ2VzPTg3Ni04Nzk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByZG05IHBvbHltb3JwaGlzbSB1bnZlaWxzIG1vdXNlIGV2b2x1dGlvbmFyeSB0cmFja3M7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUROQSBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MjE7ICBjaXRhdGlvbl9wYWdlcz0zMTUtMzI2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QcmltYXRlIGV2b2x1dGlvbiBvZiB0aGUgcmVjb21iaW5hdGlvbiByZWd1bGF0b3IgUFJETTk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gQ29tbXVuLjsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9NDM3MCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2aWRlbmNlIG9mIHBvc2l0aXZlIHNlbGVjdGlvbiBhbmQgY29uY2VydGVkIGV2b2x1dGlvbiBpbiB0aGUgcmFwaWRseSBldm9sdmluZyBQUkRNOSB6aW5jIGZpbmdlciBkb21haW4gaW4gZ29hdHMgYW5kIHNoZWVwOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1BbmltLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9NDc7ICBjaXRhdGlvbl9wYWdlcz03NDAtNzUxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1XaGF0IGFyZSB0aGUgZ2Vub21pYyBkcml2ZXJzIG9mIHRoZSByYXBpZCBldm9sdXRpb24gb2YgUFJETTk/OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UcmVuZHMgR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTI3OyAgY2l0YXRpb25fcGFnZXM9MTY1LTE3MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIFJlZCBRdWVlbiBtb2RlbCBvZiByZWNvbWJpbmF0aW9uIGhvdHNwb3RzIGV2b2x1dGlvbiBpbiB0aGUgbGlnaHQgb2YgYXJjaGFpYyBhbmQgbW9kZXJuIGh1bWFuIGdlbm9tZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz1lMTAwNDc5MCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpdmVyc2l0eSBvZiBQcmRtOSB6aW5jIGZpbmdlciBhcnJheSBpbiB3aWxkIG1pY2UgdW5yYXZlbHMgbmV3IGZhY2V0cyBvZiB0aGUgZXZvbHV0aW9uYXJ5IHR1cm5vdmVyIG9mIHRoaXMgY29kaW5nIG1pbmlzYXRlbGxpdGU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgT25lOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz1lODUwMjEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZWF0aCBvZiBQUkRNOSBjb2luY2lkZXMgd2l0aCBzdGFiaWxpemF0aW9uIG9mIHRoZSByZWNvbWJpbmF0aW9uIGxhbmRzY2FwZSBpbiB0aGUgZG9nIGdlbm9tZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0yMjsgIGNpdGF0aW9uX3BhZ2VzPTUxLTYzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5ldGljIHJlY29tYmluYXRpb24gaXMgdGFyZ2V0ZWQgdG93YXJkcyBnZW5lIHByb21vdGVyIHJlZ2lvbnMgaW4gZG9nczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9ZTEwMDM5ODQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1aaW5jIGZpbmdlciBkb21haW4gb2YgdGhlIFBSRE05IGdlbmUgb24gY2hyb21vc29tZSAxIGV4aGliaXRzIGhpZ2ggZGl2ZXJzaXR5IGluIHJ1bWluYW50cyBidXQgaXRzIHBhcmFsb2cgUFJETTcgY29udGFpbnMgbXVsdGlwbGUgZGlzcnVwdGl2ZSBtdXRhdGlvbnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgT25lOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9ZTAxNTYxNTkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SdW1pbmFudC1zcGVjaWZpYyBtdWx0aXBsZSBkdXBsaWNhdGlvbiBldmVudHMgb2YgUFJETTkgYmVmb3JlIHNwZWNpYXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBFdm9sLiBCaW9sLiw7ICBjaXRhdGlvbl95ZWFyPTIwMTc7ICBjaXRhdGlvbl92b2x1bWU9MTc7ICBjaXRhdGlvbl9wYWdlcz03OSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2aWRlbmNlIGZvciB0d28gaW5kZXBlbmRlbnQgZG9tZXN0aWNhdGlvbnMgb2YgY2F0dGxlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsLiBBY2FkLiBTY2k7ICBjaXRhdGlvbl95ZWFyPTE5OTQ7ICBjaXRhdGlvbl92b2x1bWU9OTE7ICBjaXRhdGlvbl9wYWdlcz0yNzU3LTI3NjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUluZmVycmluZyBwb3B1bGF0aW9uIHNpemUgaGlzdG9yeSBmcm9tIGxhcmdlIHNhbXBsZXMgb2YgZ2Vub21lLXdpZGUgbW9sZWN1bGFyIGRhdGEtYW4gYXBwcm94aW1hdGUgQmF5ZXNpYW4gY29tcHV0YXRpb24gYXBwcm9hY2g7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz1lMTAwNTg3NyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZpbmUtc2NhbGUgcmVjb21iaW5hdGlvbiByYXRlIGRpZmZlcmVuY2VzIGJldHdlZW4gc2V4ZXMsIHBvcHVsYXRpb25zIGFuZCBpbmRpdmlkdWFsczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTQ2NzsgIGNpdGF0aW9uX3BhZ2VzPTEwOTktMTEwMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBoaWdoLXJlc29sdXRpb24gc2luZ2xlIG51Y2xlb3RpZGUgcG9seW1vcnBoaXNtIGdlbmV0aWMgbWFwIG9mIHRoZSBtb3VzZSBnZW5vbWU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPWUzOTUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZ2VuZXRpYyBhcmNoaXRlY3R1cmUgb2YgbmF0dXJhbCB2YXJpYXRpb24gaW4gcmVjb21iaW5hdGlvbiByYXRlIGluIERyb3NvcGhpbGEgbWVsYW5vZ2FzdGVyOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPWUxMDA1OTUxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVjb21iaW5hdGlvbiByYXRlIHZhcmlhdGlvbiBhbmQgc3BlY2lhdGlvbjogdGhlb3JldGljYWwgcHJlZGljdGlvbnMgYW5kIGVtcGlyaWNhbCByZXN1bHRzIGZyb20gcmFiYml0cyBhbmQgbWljZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGhpbG9zLiBUcmFucy4gUi4gU29jLiBCOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTM2NzsgIGNpdGF0aW9uX3BhZ2VzPTQwOS00MjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUh5YnJpZCBzdGVyaWxpdHkgbG9jdXMgb24gY2hyb21vc29tZSBYIGNvbnRyb2xzIG1laW90aWMgcmVjb21iaW5hdGlvbiByYXRlIGluIG1vdXNlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPWUxMDA1OTA2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVjb21iaW5hdGlvbiBpbml0aWF0aW9uIG1hcHMgb2YgaW5kaXZpZHVhbCBodW1hbiBnZW5vbWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTM0NjsgIGNpdGF0aW9uX3BhZ2VzPTEyNTY0NDIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QUkRNOSBkcml2ZXMgZXZvbHV0aW9uYXJ5IGVyb3Npb24gb2YgaG90c3BvdHMgaW4gTXVzIG11c2N1bHVzIHRocm91Z2ggaGFwbG90eXBlLXNwZWNpZmljIGluaXRpYXRpb24gb2YgbWVpb3RpYyByZWNvbWJpbmF0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9ZTEwMDQ5MTYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5ldGljIGxpbmtzIGJldHdlZW4gcmVjb21iaW5hdGlvbiBhbmQgc3BlY2lhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz1lMTAwNjA2NiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlLWVuZ2luZWVyaW5nIHRoZSB6aW5jIGZpbmdlcnMgb2YgUFJETTkgcmV2ZXJzZXMgaHlicmlkIHN0ZXJpbGl0eSBpbiBtaWNlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9NTMwOyAgY2l0YXRpb25fcGFnZXM9MTcxLTE3NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2F0dGxlIHNleC1zcGVjaWZpYyByZWNvbWJpbmF0aW9uIGFuZCBnZW5ldGljIGNvbnRyb2wgZnJvbSBhIGxhcmdlIHBlZGlncmVlIGFuYWx5c2lzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPWUxMDA1Mzg3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyYXRpdmUgcmVjb21iaW5hdGlvbiByYXRlcyBpbiB0aGUgcmF0LCBtb3VzZSwgYW5kIGh1bWFuIGdlbm9tZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz01MjgtNTM4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgcmVjb21iaW5hdGlvbmFsIGFuYXRvbXkgb2YgYSBtb3VzZSBjaHJvbW9zb21lOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTQ7ICBjaXRhdGlvbl9wYWdlcz1lMTAwMDExOSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvZGluZyBhbmQgbm9uY29kaW5nIHZhcmlhbnRzIGluIEhGTTEsIE1MSDMsIE1TSDQsIE1TSDUsIFJORjIxMiwgYW5kIFJORjIxMkIgYWZmZWN0IHJlY29tYmluYXRpb24gcmF0ZSBpbiBjYXR0bGU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTI2OyAgY2l0YXRpb25fcGFnZXM9MTMyMy0xMzMyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IaWdoIHJlc29sdXRpb24gbG9jYWxpemF0aW9uIG9mIHJlY29tYmluYXRpb24gaG90IHNwb3RzIHVzaW5nIHNwZXJtIHR5cGluZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTE5OTQ7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTQyMC00MjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIHNpbmdsZS1jZWxsIGFuYWx5c2lzIG9mIHJlY29tYmluYXRpb24gYWN0aXZpdHkgYW5kIGRlIG5vdm8gbXV0YXRpb24gcmF0ZXMgaW4gaHVtYW4gc3Blcm07ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTUwOyAgY2l0YXRpb25fcGFnZXM9NDAyLTQxMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29udHJhc3RpbmcgbWV0aG9kcyBvZiBxdWFudGlmeWluZyBmaW5lIHN0cnVjdHVyZSBvZiBodW1hbiByZWNvbWJpbmF0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Bbm51LiBSZXYuIEdlbm9tLiBIdW0uIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTQ1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJvYmluZyBtZWlvdGljIHJlY29tYmluYXRpb24gYW5kIGFuZXVwbG9pZHkgb2Ygc2luZ2xlIHNwZXJtIGNlbGxzIGJ5IHdob2xlLWdlbm9tZSBzZXF1ZW5jaW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTMzODsgIGNpdGF0aW9uX3BhZ2VzPTE2MjctMTYzMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIEVuc2VtYmwgZ2Vub21lIGRhdGFiYXNlIHByb2plY3Q7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2wuIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTM4LTQxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QcmltZXItQkxBU1Q6IGEgdG9vbCB0byBkZXNpZ24gdGFyZ2V0LXNwZWNpZmljIHByaW1lcnMgZm9yIHBvbHltZXJhc2UgY2hhaW4gcmVhY3Rpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHdob2xlLWdlbm9tZSBhc3NlbWJseSBvZiB0aGUgZG9tZXN0aWMgY293LCBCb3MgdGF1cnVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9pc3N1ZT0xOyAgY2l0YXRpb25fcGFnZXM9UjQyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJvdGVvbWljIGFuYWx5c2lzIG9mIEdMVVQ0IHN0b3JhZ2UgdmVzaWNsZXMgcmV2ZWFscyB0dW1vciBzdXBwcmVzc29yIGNhbmRpZGF0ZSA1IChUVVNDNSkgYXMgYSBub3ZlbCByZWd1bGF0b3Igb2YgaW5zdWxpbiBhY3Rpb24gaW4gYWRpcG9jeXRlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQmlvbC4gQ2hlbS47ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MjkwOyAgY2l0YXRpb25fcGFnZXM9MjM1MjgtMjM1NDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVI6IGEgbGFuZ3VhZ2UgYW5kIGVudmlyb25tZW50IGZvciBzdGF0aXN0aWNhbCBjb21wdXRpbmc7IGNpdGF0aW9uX2F1dGhvcj1SLCBDVDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Vmllbm5hLCBBdXN0cmlhOiBSIEZvdW5kYXRpb24gU3RhdC4gQ29tcHV0OyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz0yOTktMzE0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BY2NlbGVyYXRlZCBldm9sdXRpb24gb2YgdGhlIFByZG05IHNwZWNpYXRpb24gZ2VuZSBhY3Jvc3MgZGl2ZXJzZSBtZXRhem9hbiB0YXhhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz1lMTAwMDc1MyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURlIG5vdm8gcHJlZGljdGlvbiBvZiBETkEtYmluZGluZyBzcGVjaWZpY2l0aWVzIGZvciBDeXMySGlzMiB6aW5jIGZpbmdlciBwcm90ZWluczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbC4gQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTQyOyAgY2l0YXRpb25fcGFnZXM9OTctMTA4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZXZvbHV0aW9uYXJ5IHR1cm5vdmVyIG9mIHJlY29tYmluYXRpb24gaG90IHNwb3RzIGNvbnRyaWJ1dGVzIHRvIHNwZWNpYXRpb24gaW4gbWljZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZXMgRGV2LjsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTI2Ni0yODA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmV0aWMgdmFyaWFudHMgaW4gUkVDOCwgUk5GMjEyLCBhbmQgUFJETTkgaW5mbHVlbmNlIG1hbGUgcmVjb21iaW5hdGlvbiBpbiBjYXR0bGU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz1lMTAwMjg1NCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBjYXNlIG9mIHRoZSBmaWNrbGUgZmluZ2VyczogaG93IHRoZSBQUkRNOSB6aW5jIGZpbmdlciBwcm90ZWluIHNwZWNpZmllcyBtZWlvdGljIHJlY29tYmluYXRpb24gaG90c3BvdHMgaW4gaHVtYW5zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPWUxMDAxMjExIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW91c2UgUFJETTkgRE5BLWJpbmRpbmcgc3BlY2lmaWNpdHkgZGV0ZXJtaW5lcyBzaXRlcyBvZiBoaXN0b25lIEgzIGx5c2luZSA0IHRyaW1ldGh5bGF0aW9uIGZvciBpbml0aWF0aW9uIG9mIG1laW90aWMgcmVjb21iaW5hdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz1lMTAwMTE3NiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8yNS8yLzE4My8yNDY2MjIwOC9kc3gwNDgucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIFBSRE05IGNvbnRyaWJ1dGVzIHRvIGh5YnJpZCBzdGVyaWxpdHkgYW5kIHNwZWNpZXMgZXZvbHV0aW9uLiBIb3dldmVyLCBpdHMgcm9sZSBpcyB0byBiZSBjb25maXJtZWQgaW4gY2F0dGxlLCBhIG1ham9yIGRvbWVzdGljYXRlZCBsaXZlc3RvY2sgc3BlY2llcy4gIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFcy8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj00NjY1Njk1JnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZLhXVTJKFjeBkAhSSF9kPTbgaTmF" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">New Jersey Institute of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

New Jersey Institute of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f2%2f183%2f4665695" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f2%2f183%2f4665695" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=108668" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/25/2"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/2/1/m_dnares_25_2cover.png?Expires=2147483647&amp;Signature=yfSnSDuEb8Ru1iEiczh9QiVIRujFlv5eQUGA2PixxhIP2ifTZWTC6Dh4YwZbNdfMcc6NjI5nmXhBBnXl~MO-9TlTyrmfE~hJJOwCiRzc0uAKzzN1WszCYLPzg4YPUGBBBvmjvYjtWeuxzT6HMUtRCVQ1Lykx1-FxL9MPbgabfK3LfM0lfiO9VFwha2rRByA4174pANV1EEHkTLYoj9VZAqG9NluBapcroduGQ77QCNh6n6sU6cwRapmFfSHO~HcCIdSas62HDXNUy3sCegaV5-lnkgDK34Vs8bE~hWbIr-4dZ69GanJYFX-rhZ4a7s6UpCy38dZKnOnrMjKCPyIBeg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/25/2">
                        <div class="volume trailing-comma">Volume 25</div>
                        <div class="issue">Issue 2</div>

            </a>

        </div>
        <div class="ii-pub-date">
April 2018        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="115733708">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#115733708">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="115733710">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#115733710">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="115733716">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#115733716">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="115733732">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#115733732">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="115733756">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#115733756">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="115733760">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#115733760">5. Conclusions</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="115733762">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#115733762">Declarations</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="115733765">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#115733765">Consent for publication</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="115733767">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#115733767">Availability of data and material</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="115733769">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#115733769">Authors' contributions</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="115733771">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#115733771">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="115733781">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#115733781">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/25/2/173/4633668">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/25/2/195/4638432">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Construction of <em>PRDM9</em> allele-specific recombination maps in cattle using large-scale pedigree analysis and genome-wide single sperm genomics
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                            <div class="meta-authors--limited">
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yang Zhou</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Animal Genomics and Improvement Laboratory, BARC, USDA-ARS, Beltsville, MD 20705, USA</div><div class="aff">Shaanxi Key Laboratory of Agricultural Molecular Biology, College of Animal Science and Technology, Northwest A&amp;F University, Yangling, Shaanxi 712100, China</div><div class="aff">Key Laboratory of Agricultural Animal Genetics, Breeding and Reproduction, Education Ministry of China, Huazhong Agricultural University, Wuhan, Hubei 430070, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yang+Zhou">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Zhou Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Zhou Y&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Yang Zhou</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Botong Shen</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Animal and Avian Sciences, University of Maryland, College Park, MD 20742, USA</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Botong+Shen">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shen B">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Shen B&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Botong Shen</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jicai Jiang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Animal and Avian Sciences, University of Maryland, College Park, MD 20742, USA</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jicai+Jiang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Jiang J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Jiang J&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Jicai Jiang</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Abinash Padhi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Animal and Avian Sciences, University of Maryland, College Park, MD 20742, USA</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Abinash+Padhi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Padhi A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Padhi A&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Abinash Padhi</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ki-Eun Park</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Animal and Avian Sciences, University of Maryland, College Park, MD 20742, USA</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ki-Eun+Park">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Park K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Park K&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Ki-Eun Park</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Adam Oswalt</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Select Sires Inc. 11740 U.S. 42 North, Plain City, Ohio 43064, USA</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Adam+Oswalt">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Oswalt A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Oswalt A&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Adam Oswalt</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Charles G Sattler</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Select Sires Inc. 11740 U.S. 42 North, Plain City, Ohio 43064, USA</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Charles+G+Sattler">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sattler C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sattler C&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Charles G Sattler</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Bhanu P Telugu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Animal and Avian Sciences, University of Maryland, College Park, MD 20742, USA</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Bhanu+P+Telugu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Telugu B">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Telugu B&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Bhanu P Telugu</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hong Chen</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Shaanxi Key Laboratory of Agricultural Molecular Biology, College of Animal Science and Technology, Northwest A&amp;F University, Yangling, Shaanxi 712100, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hong+Chen">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Chen H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Chen H&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Hong Chen</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">John B Cole</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Animal Genomics and Improvement Laboratory, BARC, USDA-ARS, Beltsville, MD 20705, USA</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=John+B+Cole">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Cole J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Cole J&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">John B Cole</a>
                                    </span>
                                <a class="meta-authors--etal is-active" id="show-meta-authors" aria-controls="remaining-authors" href="#">
                                    <span>...</span> Show more</a>
                                <div id="remaining-authors" class="meta-authors--remaining" aria-hidden="true" aria-labeledby="show-meta-authors">
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">George E Liu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Animal Genomics and Improvement Laboratory, BARC, USDA-ARS, Beltsville, MD 20705, USA</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsx048-cor1" data-open="dsx048-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Tel. +301 405 1389. Fax. 301 405 7980. Email: <a href="mailto:lima@umd.edu" target="_blank">lima@umd.edu</a> (L.M.); Tel. +1 301 504 9843. Fax. +1 301 504 8414. (G.E.L.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=George+E+Liu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Liu G">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Liu G&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">George E Liu</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Li Ma</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Animal and Avian Sciences, University of Maryland, College Park, MD 20742, USA</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsx048-cor1" data-open="dsx048-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Tel. +301 405 1389. Fax. 301 405 7980. Email: <a href="mailto:lima@umd.edu" target="_blank">lima@umd.edu</a> (L.M.); Tel. +1 301 504 9843. Fax. +1 301 504 8414. (G.E.L.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Li+Ma">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ma L">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ma L&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Li Ma</a>
                                        </span>
                                </div>
                            </div>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsx048-FM1"><div class="fn"><div class="footnote-content">Yang Zhou and Botong Shen contributed equally to this work.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsx048-FM2"><div class="fn"><div class="footnote-content">Edited by Dr. Osamu Ohara</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 25, Issue 2, 1 April 2018, Pages 183–194, <a href='https://doi.org/10.1093/dnares/dsx048'>https://doi.org/10.1093/dnares/dsx048</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">27 November 2017</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">12 July 2017</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">09 November 2017</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="4665695" href="/dnaresearch/article-pdf/25/2/183/24662208/dsx048.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Yang Zhou, Botong Shen, Jicai Jiang, Abinash Padhi, Ki-Eun Park, Adam Oswalt, Charles G Sattler, Bhanu P Telugu, Hong Chen, John B Cole, George E Liu, Li Ma;  Construction of <em>PRDM9</em> allele-specific recombination maps in cattle using large-scale pedigree analysis and genome-wide single sperm genomics, <em>DNA Research</em>, Volume 25, Issue 2, 1 April 2018, Pages 183–194, <a href='https://doi.org/10.1093/dnares/dsx048'>https://doi.org/10.1093/dnares/dsx048</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/4665695?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/4665695?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/4665695?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/4665695?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/4665695?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/4665695?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/4665695?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Construction%20of%20%26lt%3Bem%26gt%3BPRDM9%26lt%3B%2Fem%26gt%3B%20allele-specific%20recombination%20maps%20in%20cattle%20using%20large-scale%20pedigree%20analysis%20and%20genome-wide%20single%20sperm%20genomics&amp;publicationDate=2017-11-27&amp;volumeNum=25&amp;issueNum=2&amp;author=Zhou%2C%20Yang%3B%20Shen%2C%20Botong&amp;startPage=183&amp;endPage=194&amp;contentId=10.1093%2Fdnares%2Fdsx048&amp;oa=&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f2%2f183%2f4665695" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=115733708 id="115733708" class="abstract-title" >Abstract</h2>
<section class="abstract"><p><em>PRDM</em>9 contributes to hybrid sterility and species evolution. However, its role is to be confirmed in cattle, a major domesticated livestock species. We previously found an association near <em>PRDM</em>9 with cattle recombination features, but the causative variants are still unknown. Using millions of genotyped cattle with pedigree information, we characterized five <em>PRDM</em>9 alleles and generated allele-specific recombination maps. By examining allele-specific recombination patterns, we observed the impact of <em>PRDM</em>9 on global distribution of recombination, especially in the two ends of chromosomes. We also showed strong associations between recombination hotspot regions and functional mutations within <em>PRDM9</em> zinc finger domain. More importantly, we found one allele of <em>PRDM</em>9 to be very different from others in both protein composition and recombination landscape, indicating the causative role of this allele on the association between <em>PRDM</em>9 and cattle recombination. When comparing recombination maps from sperm and pedigree data, we observed similar genome-wide recombination patterns, validating the quality of pedigree-based results. Collectively, these evidence supported <em>PRDM</em>9 alleles as causal variants for the reported association with cattle recombination. Our study comprehensively surveyed the bovine <em>PRDM</em>9 alleles, generated allele-specific recombination maps, and expanded our understanding of the role of <em>PRDM</em>9 on genome distribution of recombination.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="PRDM9">PRDM9</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="recombination">recombination</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="cattle">cattle</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="allele">allele</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;single sperm genomics&quot;">single sperm genomics</a></div>                            <h2 scrollto-destination=115733710 id="115733710" class="section-title" >1. Introduction</h2>
<p>Meiotic recombination promotes population diversity by reshuffling parental genetic variants into the next generation and providing novel combinations of genes for selection and evolution.<a reveal-id="dsx048-B1 dsx048-B2 dsx048-B3 dsx048-B4 dsx048-B5" data-open="dsx048-B1 dsx048-B2 dsx048-B3 dsx048-B4 dsx048-B5" class="link link-ref link-reveal xref-bibr"><sup>1–5</sup></a> Meiotic recombination is also important in determining proper chromosomal segregation.<a reveal-id="dsx048-B4" data-open="dsx048-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> Recombination hotspots are usually clustered into narrow genomic regions that are specified by the PR domain-containing 9 (<em>PRDM</em>9) gene in human and mouse.<a reveal-id="dsx048-B6 dsx048-B7 dsx048-B8 dsx048-B9" data-open="dsx048-B6 dsx048-B7 dsx048-B8 dsx048-B9" class="link link-ref link-reveal xref-bibr"><sup>6–9</sup></a> The high polymorphism level (number and type) in the tandem-repeat zinc finger (ZnF) regions of <em>PRDM</em>9 has drawn wide interest and attention.<a reveal-id="dsx048-B10 dsx048-B11 dsx048-B12" data-open="dsx048-B10 dsx048-B11 dsx048-B12" class="link link-ref link-reveal xref-bibr"><sup>10–12</sup></a> In mammals, the number of <em>PRDM</em>9 ZnF varies from 6 to 19 with highly diverse ZnF components between and within species, likely evolving under strong positive selection.<a reveal-id="dsx048-B4" data-open="dsx048-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a><sup>,</sup><a reveal-id="dsx048-B10" data-open="dsx048-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a><sup>,</sup><a reveal-id="dsx048-B13 dsx048-B14 dsx048-B15" data-open="dsx048-B13 dsx048-B14 dsx048-B15" class="link link-ref link-reveal xref-bibr"><sup>13–15</sup></a> Some species, like canids, carry inert versions of <em>PRDM</em>9 genes with multiple disruptive mutations.<a reveal-id="dsx048-B16" data-open="dsx048-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a><sup>,</sup><a reveal-id="dsx048-B17" data-open="dsx048-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> Although the polymorphism level of <em>PRDM</em>9 is dramatically higher than other genes in many mammalian species, the diversity of <em>PRDM</em>9 has only recently been documented in cattle (<em>Bos taurus</em>),<a reveal-id="dsx048-B18" data-open="dsx048-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a><sup>,</sup><a reveal-id="dsx048-B19" data-open="dsx048-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> which has been domesticated since the Neolithic period<a reveal-id="dsx048-B20" data-open="dsx048-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> and whose effective population size continued to shrink from tens of thousands prior to domestication to hundreds at present.<a reveal-id="dsx048-B21" data-open="dsx048-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a></p><p>Considerable variation in recombination rate between individuals has been documented in mammals and other species.<a reveal-id="dsx048-B22 dsx048-B23 dsx048-B24 dsx048-B25 dsx048-B26" data-open="dsx048-B22 dsx048-B23 dsx048-B24 dsx048-B25 dsx048-B26" class="link link-ref link-reveal xref-bibr"><sup>22–26</sup></a> A recent study reported different locations of double-strand breaks between <em>PRDM</em>9 alleles in humans, indicating a critical role of <em>PRDM</em>9 in individual-level variations of recombination.<a reveal-id="dsx048-B27" data-open="dsx048-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a><sup>,</sup><em>PRDM</em>9 has also been shown to drive evolutionary erosion of hotspots in <em>Mus musculus</em> through haplotype-specific initiation of meiotic recombination.<a reveal-id="dsx048-B28" data-open="dsx048-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> In a further study of <em>Mus musculus PRDM9</em>, Smagulova et al. found hotspot erosion governed the preferential usage of <em>PRDM9</em> alleles and increased sequence diversity at hotspots that become active in the hybrids.<a reveal-id="dsx048-B28" data-open="dsx048-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> Because crossovers were disfavored at such hotspots, it was assumed that sequence divergence generated by hotspot turnover may create an impediment for recombination in hybrids, potentially leading to reduced fertility and, eventually, speciation. Through these mechanisms, <em>PRDM</em>9 has been considered as an important player in speciation.<a reveal-id="dsx048-B29" data-open="dsx048-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> Moreover, re-engineering the ZnFs of <em>PRDM9</em> with human alleles reversed hybrid sterility in mouse.<a reveal-id="dsx048-B30" data-open="dsx048-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> Despite the important role of <em>PRDM</em>9 on recombination, a genome-wide evaluation of the recombination patterns of different <em>PRDM</em>9 alleles has been lacking in mammals, mainly due to the limited pedigree data in mammalian species.</p><p>Large-scale pedigree data are crucial for reconstructing fine-scale recombination maps and for studying patterns of recombination. Genomic evaluations in the cattle industry have accumulated tons of genotype data with pedigree information. The USDA-ARS Animal Genomics and Improvement Laboratory (AGIL) maintains a large database that includes millions of genotyped cattle, a unique resource for studying meiotic recombination with unprecedented power. In a large-scale study of cattle pedigree, recently, we have reported strikingly different recombination patterns between males and females, and identified several loci associated with recombination rate and hotspot usage in both sexes, including the <em>PRDM</em>9 gene on chromosome 1.<a reveal-id="dsx048-B31" data-open="dsx048-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> Additionally, in contrast to human and mouse studies that reported a male to female ratio of 0.6:0.9 in genome-wide recombination rate,<a reveal-id="dsx048-B22" data-open="dsx048-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a><sup>,</sup><a reveal-id="dsx048-B32" data-open="dsx048-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a><sup>,</sup><a reveal-id="dsx048-B33" data-open="dsx048-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> a much higher ratio of 1.1:1.2 was found in cattle,<a reveal-id="dsx048-B31" data-open="dsx048-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a><sup>,</sup><a reveal-id="dsx048-B34" data-open="dsx048-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> suggesting marked divergence in the sex-specific recombination rate in these placental mammals. However, it remains unclear what the causative mutations are for the cattle <em>PRDM</em>9 association and how <em>PRDM</em>9 alleles impact recombination features.</p><p>While pedigree-based studies have been widely applied, there are two other methods for measuring recombination based on either linkage disequilibrium (LD) patterns or sperm typing. Sperm typing and single-sperm genomics evaluate recombination at either a regional scale or genome-wide level.<a reveal-id="dsx048-B35" data-open="dsx048-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a><sup>,</sup><a reveal-id="dsx048-B36" data-open="dsx048-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> Although global recombination maps obtained by these methods are more or less consistent with one another,<a reveal-id="dsx048-B37" data-open="dsx048-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> measurable local differences that attributed to different genetic features cannot be ruled out. Using single sperm genomics approach, Wang et al. reported an average of 22.8 recombination events, 5 to 15 gene conversion events, and 25 to 36 de novo mutations in human sperm cells.<a reveal-id="dsx048-B36" data-open="dsx048-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> Using the same method, another study reported aneuploidy in 4% of the cells and 26 recombination events per single-sperm cell in humans.<a reveal-id="dsx048-B38" data-open="dsx048-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> These studies affirmed the robustness of single sperm genomics as alternative to pedigree-based approaches in recombination research.</p><p>Using large-scale pedigree analysis and genome-wide single sperm genomics, the objective of this study was to (1) Characterize <em>PRDM9</em> ZnF sequence variations in two cattle breeds; (2) Correlate different <em>PRDM9</em> alleles with recombination features and generate <em>PRDM9</em> allele-specific recombination maps in two sexes of cattle; and (3) Assess the reliability of pedigree and single sperm-typing based approaches and compare recombination patterns between sperms and live-born offspring.</p>                            <h2 scrollto-destination=115733716 id="115733716" class="section-title" >2. Materials and methods</h2>
                            <h3 scrollto-destination=115733717 id="115733717" class="section-title" >2.1. Validation of bovine <em>PRDM</em>9 gene structure</h3>
<p>The Ensembl database has two coding sequences for the Zinc finger (ZnF) repeat of the bovine <em>PRDM</em>9 gene, located respectively at Chr1: 45, 034, 069-45, 034, 571 (ZnF1) and Chr1: 45, 078, 067-45, 078, 685 (ZnF2).<a reveal-id="dsx048-B39" data-open="dsx048-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> Using evidence from multiple sources, we confirmed ZnF1 as the correct ZnF coding sequence for <em>PRDM</em>9. First, we separately amplified the two ZnF repeat regions using genomic DNA of eight bulls from two cattle breeds (Holstein and Hereford). Due to the high diversity in the ZnF regions of <em>PRDM</em>9 in other species, we expected to observe polymorphisms in the ZnF region of bovine <em>PRDM</em>9. However, we found nucleotide variation in only the ZnF1 region; no polymorphisms were observed in the ZnF2 region. To further validate the two ZnF sequences for <em>PRDM</em>9, we attempted to amplify them from the cDNA of bovine testis tissue using two specifically designed primers. Only ZnF1 was successfully amplified (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx048/1/dsx048_supp.zip?Expires=2147483647&Signature=cB7LewD99XIfXtilEHTinjygYWutIBSPgTSSy6xH4XO0UhC59pZBSl92WUktO2BqH2Hl3-GTWAYw6LtDm7lW3YE-dlS2MMAJV4XEQXCo7TifcPEQEtn9LH0s2AbatwLORuKdw7mxT~nLTak89qTAjVAIM7KkWkbB~m9AEtyVLoWKEPXNGaKxuHqdS616H4n9373orlSfnWCzhmqoBYwtgyfQLPB02zulnR7CrM97FtAfZQI7dtJtAAxSr2-Z3~J8WnVzzJL1ozAZMGjaHP3tY2mkF3halnzqJJ4cFtpVZWGJH7catGGXeTJ8UIeICXKqNKHC8OxySuj2kP00XSGRuA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>). The reverse primer for ZnF1 is located in a non-coding region, suggesting that ZnF1 and ZnF2 are disconnected in the mRNA level. These results were used to confirm the gene structure of bovine <em>PRDM</em>9, and we used the ZnF1 region of <em>PRDM</em>9 in all following analyses.</p>                            <h3 scrollto-destination=115733719 id="115733719" class="section-title" >2.2. Amplification and sequencing of cattle <em>PRDM</em>9 ZnF repeats</h3>
<p>Cattle testis tissues were collected as per the ethical guidelines of USDA-ARS animal use and care protocol. The testis was snap-frozen in liquid nitrogen immediately after excision and kept at −80 °C until further use. Total RNA was isolated and reverse-transcribed to cDNA. Frozen semen and somatic tissue of animal HOUSA000072190767 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx048/1/dsx048_supp.zip?Expires=2147483647&Signature=cB7LewD99XIfXtilEHTinjygYWutIBSPgTSSy6xH4XO0UhC59pZBSl92WUktO2BqH2Hl3-GTWAYw6LtDm7lW3YE-dlS2MMAJV4XEQXCo7TifcPEQEtn9LH0s2AbatwLORuKdw7mxT~nLTak89qTAjVAIM7KkWkbB~m9AEtyVLoWKEPXNGaKxuHqdS616H4n9373orlSfnWCzhmqoBYwtgyfQLPB02zulnR7CrM97FtAfZQI7dtJtAAxSr2-Z3~J8WnVzzJL1ozAZMGjaHP3tY2mkF3halnzqJJ4cFtpVZWGJH7catGGXeTJ8UIeICXKqNKHC8OxySuj2kP00XSGRuA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>), together with its parent somatic tissues were donated by Selected Sires, Inc (Plain City, OH, USA). Other Holstein and Jersey cattle DNA samples were obtained from the Cooperative Dairy DNA Repository (CDDR) at USDA-ARS. The PCR and sequencing primers were designed using Primer-BLAST,<a reveal-id="dsx048-B40" data-open="dsx048-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> and the <em>PRDM</em>9 gene in the cattle reference genome (UMD 3.1.1) was used as template.<a reveal-id="dsx048-B41" data-open="dsx048-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a> We confirmed the specificity of primers by comparing with the newest version of the cattle genome (Btau_5.0.1, GCA_000003205.6). The primer pairs for PCR amplification using cDNA as the template were designed by crossing two or more exons to avoid potential DNA contamination. The primer pairs for <em>PRDM</em>9 ZnF repeat amplification were designed with 579 bp (5′ end) and 157 bp (3′ end) of unique, non-repetitive flanking sequences around the ZnF repeat. All the primers used in the present study are listed in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx048/1/dsx048_supp.zip?Expires=2147483647&Signature=cB7LewD99XIfXtilEHTinjygYWutIBSPgTSSy6xH4XO0UhC59pZBSl92WUktO2BqH2Hl3-GTWAYw6LtDm7lW3YE-dlS2MMAJV4XEQXCo7TifcPEQEtn9LH0s2AbatwLORuKdw7mxT~nLTak89qTAjVAIM7KkWkbB~m9AEtyVLoWKEPXNGaKxuHqdS616H4n9373orlSfnWCzhmqoBYwtgyfQLPB02zulnR7CrM97FtAfZQI7dtJtAAxSr2-Z3~J8WnVzzJL1ozAZMGjaHP3tY2mkF3halnzqJJ4cFtpVZWGJH7catGGXeTJ8UIeICXKqNKHC8OxySuj2kP00XSGRuA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>.</p><p>The PCR amplification was performed with 50 μL reaction volume according to Taq DNA polymerase manufacturer’s protocol (Taq PCR Master Mix Kit, Qiagen, Hilden, Germany), and the genomic DNA was amplified on a bioRad MyIQ thermocycler. The PCR cycle for <em>PRDM</em>9 ZnF repeat amplification was as follows: initial denaturation at 95 °C for 5 min; followed by 40 cycles of 95 °C for 45 s, annealing at 63 °C for 40 s; primer extension at 72 °C for 1 min 20 s; and final extension at 72 °C for 10 min. All the amplified products were run in 1.5% agarose gel, the bands were cut, and DNA was purified with the MinElute Gel Extraction Kit provided by the Qiagen (Hilden, Germany). Purified PCR products were ligated to the pGEM-T Easy vector using the pGEM-T Vector System I (Promega, Madison, WI, USA) and transformed into DH5α (subcloning efficiency) competent cells (Invitrogen/Life Technologies, Carlsbad, CA, USA). Examples of Agarose gel electrophoresis of PCR products are shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx048/1/dsx048_supp.zip?Expires=2147483647&Signature=cB7LewD99XIfXtilEHTinjygYWutIBSPgTSSy6xH4XO0UhC59pZBSl92WUktO2BqH2Hl3-GTWAYw6LtDm7lW3YE-dlS2MMAJV4XEQXCo7TifcPEQEtn9LH0s2AbatwLORuKdw7mxT~nLTak89qTAjVAIM7KkWkbB~m9AEtyVLoWKEPXNGaKxuHqdS616H4n9373orlSfnWCzhmqoBYwtgyfQLPB02zulnR7CrM97FtAfZQI7dtJtAAxSr2-Z3~J8WnVzzJL1ozAZMGjaHP3tY2mkF3halnzqJJ4cFtpVZWGJH7catGGXeTJ8UIeICXKqNKHC8OxySuj2kP00XSGRuA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>. At least 10 single colonies for each PCR product were randomly picked. Plasmid was extracted using Plasmid Miniprep System (Promega, Madison, WI, USA) and sequenced using the T7 and SP6 primers at two companies, GENEWIZ (South Plainfield, NJ, USA) and Macrogene (Rockville, MD, USA).</p><p>To account for the short tandem repetitive nature of ZnF repeats, only forward and reverse sequences with sufficient overlap (at least 168 nt sequences covering two ZnF repeats) were used to prepare the contig and the consensus sequence for each clone was obtained. We used at least three clones with identical sequence to support a valid consensus sequence. To get both alleles with the same length from an individual, we generated at least six assembled sequences for each PCR product and used more clones for those individuals whose two alleles showed an unbalanced ratio. The detailed clone number and allele information for all animals were listed in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx048/1/dsx048_supp.zip?Expires=2147483647&Signature=cB7LewD99XIfXtilEHTinjygYWutIBSPgTSSy6xH4XO0UhC59pZBSl92WUktO2BqH2Hl3-GTWAYw6LtDm7lW3YE-dlS2MMAJV4XEQXCo7TifcPEQEtn9LH0s2AbatwLORuKdw7mxT~nLTak89qTAjVAIM7KkWkbB~m9AEtyVLoWKEPXNGaKxuHqdS616H4n9373orlSfnWCzhmqoBYwtgyfQLPB02zulnR7CrM97FtAfZQI7dtJtAAxSr2-Z3~J8WnVzzJL1ozAZMGjaHP3tY2mkF3halnzqJJ4cFtpVZWGJH7catGGXeTJ8UIeICXKqNKHC8OxySuj2kP00XSGRuA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S2 and S3</a></span>.</p>                            <h3 scrollto-destination=115733723 id="115733723" class="section-title" >2.3. Estimation of recombination rate in cattle pedigree</h3>
<p>We used similar approaches as describe before.<a reveal-id="dsx048-B42" data-open="dsx048-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a> In brief, we extracted three-generation families from Holstein and Jersey pedigree. Within a family, we require that the offspring, at least one parent, and at least one grandparent were genotyped by single nucleotide polymophism (SNP) arrays. In a three-generation family, we phased the two haplotypes of an animal (second and third generations) based on the parental genotypes, and crossover locations were identified by comparing either a paternal or maternal haplotype of an offspring (third generation) to its corresponding parent’s two haplotypes (second generation). Based on the location of a crossover, a recombination event was assigned to an interval flanked by two SNPs that are informative (phased heterozygote in the second generation). To construct recombination maps of SNPs, we estimated recombination rate between consecutive SNPs based on the identified crossover events by assigning a recombination event as equal probabilities to all consecutive SNP intervals between the two informative SNPs. When constructing recombination maps, we only included high-quality data where all members of a three-generation family were genotyped by at least 50 K SNP chips. For quality control purposes, we also removed animals (&lt;1%) that have more than 45 crossover events genome-wide, based on the distribution of crossover events in all data. The sex chromosomes were excluded from all analyses due to the poor quality of the genome assembly.</p>                            <h3 scrollto-destination=115733725 id="115733725" class="section-title" >2.4. Global and local comparisons of recombination maps</h3>
<p>To show the global distribution of recombination rates along the chromosomes, we adopted a smooth spline model of recombination rates against relative physical locations on chromosomes using the smooth spline function implemented in R 3.2.4.<a reveal-id="dsx048-B42" data-open="dsx048-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a><sup>,</sup><a reveal-id="dsx048-B43" data-open="dsx048-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a> We divided the recombination data into subgroups based on the <em>PRDM</em>9 genotype and sex and generated global recombination maps for each of the <em>PRDM</em>9 alleles in both males and females.</p><p>To identify <em>PRDM</em>9 dependent hotspot regions, we compared recombination rate locally in a SNP interval between three <em>PRDM</em>9 genotypes of allele 5 (allele 5 homozygote, allele 5 heterozygote and non-allele 5 genotypes). Given a SNP interval and a pair of groups, we generated a 2 × 2 table of recombination data (number of meioses w/o recombination in two genotype groups) and applied a χ<sup>2</sup> test to determine whether recombination rate is different between the two groups. Since the frequency of allele 5 in Holsteins is about 9%, there are unbalanced numbers of animals for the three genotype groups for allele 5 (allele 5 homozygote, heterozygote, and non-allele 5 homozygote). Male and female data also have different sample sizes. To account for the issue of unbalanced sample sizes, we used evidence from multiple sources to select the most significant allele-specific hotspots for allele 5 based on the following rules: (1) Same direction of the difference in recombination rate for males and females; (2) <em>P</em>-value for the comparison between allele 5 homozygote and non-allele 5 genotypes is less than 0.05; (3) <em>P</em>-value in the comparison of allele 5 heterozygote versus allele 5 homozygote or non-allele 5 genotypes is less than 0.05; and (4) At least one <em>P</em>-value from (2) or (3) is less than 2 × 10<sup>−6</sup>. For non-allele 5, we excluded the comparison between allele 5 homozygote and non-allele 5 genotypes due to the very limited sample sizes and low statistical power.</p>                            <h3 scrollto-destination=115733728 id="115733728" class="section-title" >2.5. Whole genome amplification, genotyping and quality control of single sperm DNA</h3>
<p>Sperm cells were obtained from semen samples stored in liquid nitrogen. Frozen semen was thawed in 37 °C water for 30–45 s. Sperms were diluted by PBS + 1% BSA and washed twice. Then the sperms were further diluted to a proper resolution using PBS + 1% BSA on a petri-dish, and active single sperms were picked up manually by pipetting into a reaction tube under micromanipulator. Whole-genome amplification was performed according to the manufacturer’s protocol using a Single Cell Whole Genome Amplification Kit (Yikon Genomics, Shanghai, China) based on the MALBAC (Multiple Annealing and Looping Based Amplification Cycles) technology. In brief, a single sperm was initially lyzed and pre-amplified by primers supplied in the kit with eight cycles with multiple annealing steps. Fragments with variable length at random starting positions were generated by polymerase extension for multiple cycles by exponential amplification step. PCR products were purified and sent to Neogen/GeneSeek (Lansing, MI, USA) for genotyping by the Illumina<sup>®</sup> BovineHD Genotyping BeadChip assay. PCR amplification quality was confirmed by showing the percentage of SNP successfully genotyped per Mb evenly distributed along the whole genome (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx048/1/dsx048_supp.zip?Expires=2147483647&Signature=cB7LewD99XIfXtilEHTinjygYWutIBSPgTSSy6xH4XO0UhC59pZBSl92WUktO2BqH2Hl3-GTWAYw6LtDm7lW3YE-dlS2MMAJV4XEQXCo7TifcPEQEtn9LH0s2AbatwLORuKdw7mxT~nLTak89qTAjVAIM7KkWkbB~m9AEtyVLoWKEPXNGaKxuHqdS616H4n9373orlSfnWCzhmqoBYwtgyfQLPB02zulnR7CrM97FtAfZQI7dtJtAAxSr2-Z3~J8WnVzzJL1ozAZMGjaHP3tY2mkF3halnzqJJ4cFtpVZWGJH7catGGXeTJ8UIeICXKqNKHC8OxySuj2kP00XSGRuA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>). To improve the genotyping accuracy for single sperms, we applied a stringent cut-off on the raw genotyping quality score to call genotypes, eliminated heterozygous SNP calls, and removed SNPs that had conflict with the sire genotype. In total, we performed whole genome amplification for 97 single sperms.</p>                            <h3 scrollto-destination=115733730 id="115733730" class="section-title" >2.6. Phasing and inferring crossovers in single sperms</h3>
<p>Genomic DNA of Animal 102 and the parents were extracted from ear tissues and genotyped together with single sperm DNA at GeneSeek using Illumina<sup>®</sup> BovineHD Genotyping BeadChip assay. Heterozygous SNPs (hetSNPs) of the bull were phased to two haploids (paternal and maternal) based on the genotypes of two parents. In total, over 76.33% of the 193,066 hetSNPs were phased. Before inferring crossovers in sperms, we applied additional quality control procedures, including general call score ≥ 0.7 for a SNP and homozygote rate ≥ 0.8 for a sample (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx048/1/dsx048_supp.zip?Expires=2147483647&Signature=cB7LewD99XIfXtilEHTinjygYWutIBSPgTSSy6xH4XO0UhC59pZBSl92WUktO2BqH2Hl3-GTWAYw6LtDm7lW3YE-dlS2MMAJV4XEQXCo7TifcPEQEtn9LH0s2AbatwLORuKdw7mxT~nLTak89qTAjVAIM7KkWkbB~m9AEtyVLoWKEPXNGaKxuHqdS616H4n9373orlSfnWCzhmqoBYwtgyfQLPB02zulnR7CrM97FtAfZQI7dtJtAAxSr2-Z3~J8WnVzzJL1ozAZMGjaHP3tY2mkF3halnzqJJ4cFtpVZWGJH7catGGXeTJ8UIeICXKqNKHC8OxySuj2kP00XSGRuA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx048/1/dsx048_supp.zip?Expires=2147483647&Signature=cB7LewD99XIfXtilEHTinjygYWutIBSPgTSSy6xH4XO0UhC59pZBSl92WUktO2BqH2Hl3-GTWAYw6LtDm7lW3YE-dlS2MMAJV4XEQXCo7TifcPEQEtn9LH0s2AbatwLORuKdw7mxT~nLTak89qTAjVAIM7KkWkbB~m9AEtyVLoWKEPXNGaKxuHqdS616H4n9373orlSfnWCzhmqoBYwtgyfQLPB02zulnR7CrM97FtAfZQI7dtJtAAxSr2-Z3~J8WnVzzJL1ozAZMGjaHP3tY2mkF3halnzqJJ4cFtpVZWGJH7catGGXeTJ8UIeICXKqNKHC8OxySuj2kP00XSGRuA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>). After quality control (QC), single sperm samples had phased hetSNP numbers ranging from 11,762 to 87,648. To improve accuracy, we applied the Viterbi algorithm in a hidden Markov model in R to identify crossovers in the sperm haplotype as a transition between paternal and maternal status.<a reveal-id="dsx048-B43" data-open="dsx048-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a> After filtering samples with abnormal numbers of crossovers (&gt;45), we obtained 1,526 autosomal crossover events from 56 high quality single sperms. Using pedigree data of the same bull, we identified 12,089 crossovers from 556 offspring that were genotyped with 7 K, 10 K or 50 K SNP chips.</p>                            <h2 scrollto-destination=115733732 id="115733732" class="section-title" >3. Results</h2>
                            <h3 scrollto-destination=115733733 id="115733733" class="section-title" >3.1. ZnF variants of <em>PRDM</em>9 in two cattle breeds</h3>
<p>Due to the highly repetitive nature of <em>PRDM</em>9 ZnFs, next-generation sequencing is unable to accurately measure the target sequence because one read cannot capture the full length of the ZnF region. To fully characterize genetic variation of cattle <em>PRDM</em>9, we PCR-amplified, cloned, sequenced, and phased the ZnFs of <em>PRDM9</em> for 25 and 17 influential bulls, respectively, for Holstein and Jersey, the two most common breeds of dairy cattle in the US. Based on amino acid composition, a total of ten different ZnF arrays (color coded in <a reveal-id="dsx048-F1" data-open="dsx048-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>) and five different alleles (combination of multiple ZnF arrays) were recovered from the 42 bulls that are representative of the dairy population because these chosen bulls typically had thousands of daughters and sons (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx048/1/dsx048_supp.zip?Expires=2147483647&Signature=cB7LewD99XIfXtilEHTinjygYWutIBSPgTSSy6xH4XO0UhC59pZBSl92WUktO2BqH2Hl3-GTWAYw6LtDm7lW3YE-dlS2MMAJV4XEQXCo7TifcPEQEtn9LH0s2AbatwLORuKdw7mxT~nLTak89qTAjVAIM7KkWkbB~m9AEtyVLoWKEPXNGaKxuHqdS616H4n9373orlSfnWCzhmqoBYwtgyfQLPB02zulnR7CrM97FtAfZQI7dtJtAAxSr2-Z3~J8WnVzzJL1ozAZMGjaHP3tY2mkF3halnzqJJ4cFtpVZWGJH7catGGXeTJ8UIeICXKqNKHC8OxySuj2kP00XSGRuA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). The five alleles of <em>PRDM</em>9 were named by the order of allele frequency in the Holstein sample. In contrast to primate and mouse <em>PRDM</em>9 that has 6–19 ZnF repeats,<a reveal-id="dsx048-B10" data-open="dsx048-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a><sup>,</sup><a reveal-id="dsx048-B11" data-open="dsx048-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> the cattle <em>PRDM</em>9 appears to have fewer ZnF repeats, ranging from 6 to 7 for each allele (<a reveal-id="dsx048-F1" data-open="dsx048-F1" class="link link-reveal link-table xref-fig">Fig. 1a</a>). With exception of allele 3 that possesses seven ZnF repeats, all the remaining alleles consist of six repeats. Despite the small effective population size of domestic cattle,<a reveal-id="dsx048-B21" data-open="dsx048-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a>the amino acid residues at positions –1, 3 and 6 of the ZnF α helix, which were predicted to be in contact with DNA motifs,<a reveal-id="dsx048-B11" data-open="dsx048-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a><sup>,</sup><a reveal-id="dsx048-B44" data-open="dsx048-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a> are highly polymorphic. Note that the ten different ZnF arrays may have the same composition at these three positions as ZnF arrarys were defined by the full length amino acid sequence. At the nucleotide level, all of the changes at these three sites are non-synonymous. The amino acid alignments of the cattle ZnF repeats with the corresponding repeats of other species, including human, rhesus monkey, rat and mouse, are shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx048/1/dsx048_supp.zip?Expires=2147483647&Signature=cB7LewD99XIfXtilEHTinjygYWutIBSPgTSSy6xH4XO0UhC59pZBSl92WUktO2BqH2Hl3-GTWAYw6LtDm7lW3YE-dlS2MMAJV4XEQXCo7TifcPEQEtn9LH0s2AbatwLORuKdw7mxT~nLTak89qTAjVAIM7KkWkbB~m9AEtyVLoWKEPXNGaKxuHqdS616H4n9373orlSfnWCzhmqoBYwtgyfQLPB02zulnR7CrM97FtAfZQI7dtJtAAxSr2-Z3~J8WnVzzJL1ozAZMGjaHP3tY2mkF3halnzqJJ4cFtpVZWGJH7catGGXeTJ8UIeICXKqNKHC8OxySuj2kP00XSGRuA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span>. While the ZnFs of cattle <em>PRDM</em>9 show consistent patterns with other species, several amino acid residues are unique to cattle. For example, cattle have Serine and Glycine at positions –4 and –9, respectively, but the corresponding positions in all other species are Glycine and Arginine (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx048/1/dsx048_supp.zip?Expires=2147483647&Signature=cB7LewD99XIfXtilEHTinjygYWutIBSPgTSSy6xH4XO0UhC59pZBSl92WUktO2BqH2Hl3-GTWAYw6LtDm7lW3YE-dlS2MMAJV4XEQXCo7TifcPEQEtn9LH0s2AbatwLORuKdw7mxT~nLTak89qTAjVAIM7KkWkbB~m9AEtyVLoWKEPXNGaKxuHqdS616H4n9373orlSfnWCzhmqoBYwtgyfQLPB02zulnR7CrM97FtAfZQI7dtJtAAxSr2-Z3~J8WnVzzJL1ozAZMGjaHP3tY2mkF3halnzqJJ4cFtpVZWGJH7catGGXeTJ8UIeICXKqNKHC8OxySuj2kP00XSGRuA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span>). </p>                        <a id="115733735" scrollto-destination="115733735"></a>
<div id="dsx048-F1" data-id="dsx048-F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx048/1/m_dsx048f1.png?Expires=2147483647&Signature=tPYHgYdhmf7qNiHJVHk1Nox4VnLKGDYYRJHZd4eVPC7UApkfY94a-aOtD4PlHMBbiiwGLTlsDo3Wphgb4JH4XwYm4sg4XuhWRpawK3gkxwz1yo7lLWfUTgnp0qZKpNvzb2m-tNSuXH0moutSUgQasi3XCQD9~ggawetL599-zre2gnXmOxavpupblspn-j0ISrh47z7KncQt2TrvOajkTnIr88rkJe9jnZIdNWm9w1Mzn35sVNTEaHYjy1GIgptUaEZ03ysE4eEB7nKt0ZB83IB76MnL~Ay9wYH41iN3zDZFyNfC77F5~YtNHhhFB7PTTijDvawt5e6CijedytpIUQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="PRDM9 alleles and ZnF arrays in Holstein and Jersey cattle. (a) Allele and ZnF array information in Holstein and Jersey cattle. ZnF arrays were coded from A to J according to the full length amino acid composition, and the three important sites were shown (–1, 3 and 6) that may be same for different ZnF arrays. Alleles were coded from allele 1 to allele 5 based on their ZnF array composition. (b) PRDM9 gene structure and comparison between non-allele 5 and allele 5. Four domains (KRAB, SET, single ZnF, 6 to 7 ZnF repeats) were present in bovine PRDM9. Allele 5 was perfectly linked with the minor allele of the SNP at Chr1: 45112924 (R2 = 1). Comparing between non-allele 5 and allele 5, amino acid frequencies at each positions were shown as proportional to the size of the letters (one ZnF includes 28 amino acids). Positions with different amino acid components between allele 5 and other alleles were highlighted by “*”." path-from-xml="dsx048f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/115733735/dsx048f1.tif" path-from-xml="dsx048f1.tif" target="_blank">View large</a><a section="115733735" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx048/1/dsx048f1.png?Expires=2147483647&Signature=E0rcf6svyp0AZLcfChyMYwQ1Om6yQwyE5Hb1GAePzszjYqJRYMXTNhP~teqhjBdr2TeLm4g6QyYxccDWvgC8FmPhqgkC6r0VMuC8Kk5BCpOVEGttOK1T2Q0UXGsD-t6LtGYCMmFrsEx0XntmGl9ZdQK~PCCnigYF~XaljA-d067dq7kiH7rVai3a20ZSgLNJyPCo3CtLHu0nK4RCgaEDq1qtYXH-Ww0W6at~2uOpzvVLOQoXHazz9c5QschpDcyCTzcY7q11SrNPrY-We6uLY2k4lsKH9kjUfxJ5JQYaSUP-RkkbPqF6s1FsKbL5hgAwC9nEVcWQAlrMkqsMedB20Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=115733735&ar=4665695&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx048f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p><strong><em>PRDM</em>9 alleles and ZnF arrays in Holstein and Jersey cattle.</strong> (<strong>a</strong>) Allele and ZnF array information in Holstein and Jersey cattle. ZnF arrays were coded from A to J according to the full length amino acid composition, and the three important sites were shown (–1, 3 and 6) that may be same for different ZnF arrays. Alleles were coded from allele 1 to allele 5 based on their ZnF array composition. (<strong>b</strong>) <em>PRDM9</em> gene structure and comparison between non-allele 5 and allele 5. Four domains (KRAB, SET, single ZnF, 6 to 7 ZnF repeats) were present in bovine <em>PRDM9</em>. Allele 5 was perfectly linked with the minor allele of the SNP at Chr1: 45112924 (<em>R</em><sup>2</sup> = 1). Comparing between non-allele 5 and allele 5, amino acid frequencies at each positions were shown as proportional to the size of the letters (one ZnF includes 28 amino acids). Positions with different amino acid components between allele 5 and other alleles were highlighted by “*”.</p></div></div><div content-id="dsx048-F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx048/1/m_dsx048f1.png?Expires=2147483647&Signature=tPYHgYdhmf7qNiHJVHk1Nox4VnLKGDYYRJHZd4eVPC7UApkfY94a-aOtD4PlHMBbiiwGLTlsDo3Wphgb4JH4XwYm4sg4XuhWRpawK3gkxwz1yo7lLWfUTgnp0qZKpNvzb2m-tNSuXH0moutSUgQasi3XCQD9~ggawetL599-zre2gnXmOxavpupblspn-j0ISrh47z7KncQt2TrvOajkTnIr88rkJe9jnZIdNWm9w1Mzn35sVNTEaHYjy1GIgptUaEZ03ysE4eEB7nKt0ZB83IB76MnL~Ay9wYH41iN3zDZFyNfC77F5~YtNHhhFB7PTTijDvawt5e6CijedytpIUQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="PRDM9 alleles and ZnF arrays in Holstein and Jersey cattle. (a) Allele and ZnF array information in Holstein and Jersey cattle. ZnF arrays were coded from A to J according to the full length amino acid composition, and the three important sites were shown (–1, 3 and 6) that may be same for different ZnF arrays. Alleles were coded from allele 1 to allele 5 based on their ZnF array composition. (b) PRDM9 gene structure and comparison between non-allele 5 and allele 5. Four domains (KRAB, SET, single ZnF, 6 to 7 ZnF repeats) were present in bovine PRDM9. Allele 5 was perfectly linked with the minor allele of the SNP at Chr1: 45112924 (R2 = 1). Comparing between non-allele 5 and allele 5, amino acid frequencies at each positions were shown as proportional to the size of the letters (one ZnF includes 28 amino acids). Positions with different amino acid components between allele 5 and other alleles were highlighted by “*”." path-from-xml="dsx048f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/115733735/dsx048f1.tif" path-from-xml="dsx048f1.tif" target="_blank">View large</a><a section="115733735" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx048/1/dsx048f1.png?Expires=2147483647&Signature=E0rcf6svyp0AZLcfChyMYwQ1Om6yQwyE5Hb1GAePzszjYqJRYMXTNhP~teqhjBdr2TeLm4g6QyYxccDWvgC8FmPhqgkC6r0VMuC8Kk5BCpOVEGttOK1T2Q0UXGsD-t6LtGYCMmFrsEx0XntmGl9ZdQK~PCCnigYF~XaljA-d067dq7kiH7rVai3a20ZSgLNJyPCo3CtLHu0nK4RCgaEDq1qtYXH-Ww0W6at~2uOpzvVLOQoXHazz9c5QschpDcyCTzcY7q11SrNPrY-We6uLY2k4lsKH9kjUfxJ5JQYaSUP-RkkbPqF6s1FsKbL5hgAwC9nEVcWQAlrMkqsMedB20Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=115733735&ar=4665695&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx048f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p><strong><em>PRDM</em>9 alleles and ZnF arrays in Holstein and Jersey cattle.</strong> (<strong>a</strong>) Allele and ZnF array information in Holstein and Jersey cattle. ZnF arrays were coded from A to J according to the full length amino acid composition, and the three important sites were shown (–1, 3 and 6) that may be same for different ZnF arrays. Alleles were coded from allele 1 to allele 5 based on their ZnF array composition. (<strong>b</strong>) <em>PRDM9</em> gene structure and comparison between non-allele 5 and allele 5. Four domains (KRAB, SET, single ZnF, 6 to 7 ZnF repeats) were present in bovine <em>PRDM9</em>. Allele 5 was perfectly linked with the minor allele of the SNP at Chr1: 45112924 (<em>R</em><sup>2</sup> = 1). Comparing between non-allele 5 and allele 5, amino acid frequencies at each positions were shown as proportional to the size of the letters (one ZnF includes 28 amino acids). Positions with different amino acid components between allele 5 and other alleles were highlighted by “*”.</p></div></div><p>Although we identified five alleles in the Holstein cattle (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx048/1/dsx048_supp.zip?Expires=2147483647&Signature=cB7LewD99XIfXtilEHTinjygYWutIBSPgTSSy6xH4XO0UhC59pZBSl92WUktO2BqH2Hl3-GTWAYw6LtDm7lW3YE-dlS2MMAJV4XEQXCo7TifcPEQEtn9LH0s2AbatwLORuKdw7mxT~nLTak89qTAjVAIM7KkWkbB~m9AEtyVLoWKEPXNGaKxuHqdS616H4n9373orlSfnWCzhmqoBYwtgyfQLPB02zulnR7CrM97FtAfZQI7dtJtAAxSr2-Z3~J8WnVzzJL1ozAZMGjaHP3tY2mkF3halnzqJJ4cFtpVZWGJH7catGGXeTJ8UIeICXKqNKHC8OxySuj2kP00XSGRuA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>), only four alleles (alleles 1–4) were observed in Jerseys (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx048/1/dsx048_supp.zip?Expires=2147483647&Signature=cB7LewD99XIfXtilEHTinjygYWutIBSPgTSSy6xH4XO0UhC59pZBSl92WUktO2BqH2Hl3-GTWAYw6LtDm7lW3YE-dlS2MMAJV4XEQXCo7TifcPEQEtn9LH0s2AbatwLORuKdw7mxT~nLTak89qTAjVAIM7KkWkbB~m9AEtyVLoWKEPXNGaKxuHqdS616H4n9373orlSfnWCzhmqoBYwtgyfQLPB02zulnR7CrM97FtAfZQI7dtJtAAxSr2-Z3~J8WnVzzJL1ozAZMGjaHP3tY2mkF3halnzqJJ4cFtpVZWGJH7catGGXeTJ8UIeICXKqNKHC8OxySuj2kP00XSGRuA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>). As the two breeds are closely related, we observed minor differences in allele frequency between Holstein and Jersey (<a reveal-id="dsx048-F1" data-open="dsx048-F1" class="link link-reveal link-table xref-fig">Fig. 1a</a>). For instance, allele 1 is most frequent in Holstein (38%), but allele 2 appears to be dominant in Jersey (41.2%). The most striking difference between the two breeds is the uniqueness of allele 5 in Holstein (8%). Interestingly, allele 5 is the most distinct allele compared to other alleles with no shared ZnF arrays between allele 5 and non-allele 5 (<a reveal-id="dsx048-F1" data-open="dsx048-F1" class="link link-reveal link-table xref-fig">Fig. 1b</a>). The amino acid alignments between allele 5 and the rest of the alleles revealed unique amino acid substitutions at the DNA-contact sites (i.e. –1, 3 and 6) and nearby positions (–2 and 2). Consistently, allele 5 and non-allele 5 (referred to as the rest of the alleles) were predicted to preferentially recognize distinct DNA motifs (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx048/1/dsx048_supp.zip?Expires=2147483647&Signature=cB7LewD99XIfXtilEHTinjygYWutIBSPgTSSy6xH4XO0UhC59pZBSl92WUktO2BqH2Hl3-GTWAYw6LtDm7lW3YE-dlS2MMAJV4XEQXCo7TifcPEQEtn9LH0s2AbatwLORuKdw7mxT~nLTak89qTAjVAIM7KkWkbB~m9AEtyVLoWKEPXNGaKxuHqdS616H4n9373orlSfnWCzhmqoBYwtgyfQLPB02zulnR7CrM97FtAfZQI7dtJtAAxSr2-Z3~J8WnVzzJL1ozAZMGjaHP3tY2mkF3halnzqJJ4cFtpVZWGJH7catGGXeTJ8UIeICXKqNKHC8OxySuj2kP00XSGRuA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6</a></span>). In a previous genome-wide association study (GWAS) study of recombination in cattle, we identified a SNP (rs110661033, Chr1: 45113934) downstream of the <em>PRDM</em>9 gene that was associated with genome-wide hotspot usage and recombination rate.<a reveal-id="dsx048-B42" data-open="dsx048-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a> Intriguingly, the minor allele ‘A’ of SNP rs110661033 was perfectly linked with <em>PRDM</em>9 allele 5 in the current study (<em>R</em><sup>2</sup> = 1; <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx048/1/dsx048_supp.zip?Expires=2147483647&Signature=cB7LewD99XIfXtilEHTinjygYWutIBSPgTSSy6xH4XO0UhC59pZBSl92WUktO2BqH2Hl3-GTWAYw6LtDm7lW3YE-dlS2MMAJV4XEQXCo7TifcPEQEtn9LH0s2AbatwLORuKdw7mxT~nLTak89qTAjVAIM7KkWkbB~m9AEtyVLoWKEPXNGaKxuHqdS616H4n9373orlSfnWCzhmqoBYwtgyfQLPB02zulnR7CrM97FtAfZQI7dtJtAAxSr2-Z3~J8WnVzzJL1ozAZMGjaHP3tY2mkF3halnzqJJ4cFtpVZWGJH7catGGXeTJ8UIeICXKqNKHC8OxySuj2kP00XSGRuA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). All the Holstein bulls that carry one copy of allele 5 are heterozygous at SNP rs110661033 (G/A). We further confirmed this linkage between <em>PRDM</em>9 (non-allele 5/allele 5) and SNP rs110661033 (G/A) by sequencing <em>PRDM</em>9 in an additional five Holstein bulls selected by the SNP genotype, in which two animals are heterozygous at rs110661033 and carry one copy of allele 5 at <em>PRDM9</em>, and three bulls are homozygous (A/A) at rs110661033 and carry two copies of allele 5 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx048/1/dsx048_supp.zip?Expires=2147483647&Signature=cB7LewD99XIfXtilEHTinjygYWutIBSPgTSSy6xH4XO0UhC59pZBSl92WUktO2BqH2Hl3-GTWAYw6LtDm7lW3YE-dlS2MMAJV4XEQXCo7TifcPEQEtn9LH0s2AbatwLORuKdw7mxT~nLTak89qTAjVAIM7KkWkbB~m9AEtyVLoWKEPXNGaKxuHqdS616H4n9373orlSfnWCzhmqoBYwtgyfQLPB02zulnR7CrM97FtAfZQI7dtJtAAxSr2-Z3~J8WnVzzJL1ozAZMGjaHP3tY2mkF3halnzqJJ4cFtpVZWGJH7catGGXeTJ8UIeICXKqNKHC8OxySuj2kP00XSGRuA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). Altogether, these results demonstrate that <em>PRDM</em>9 ZnF alleles are likely the causative mutations associated with the recombination features in cattle, with <em>PRDM</em>9 allele 5/non-allele 5 combinations fully explaining the association between rs110661033 and recombination rate and hotspot usage that we previously reported.<a reveal-id="dsx048-B42" data-open="dsx048-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a></p>                            <h3 scrollto-destination=115733737 id="115733737" class="section-title" >3.2. <em>PRDM9</em> allele-specific recombination maps for holstein</h3>
<p>Superior bulls have been extensively used in dairy cattle breeding through artificial insemination. In the national dairy database maintained at AGIL, a bull typically has hundreds to thousands of daughters genotyped. The 25 Holstein bulls selected for sequencing of <em>PRDM</em>9 were chosen because they have large numbers of genotyped offspring to study recombination (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx048/1/dsx048_supp.zip?Expires=2147483647&Signature=cB7LewD99XIfXtilEHTinjygYWutIBSPgTSSy6xH4XO0UhC59pZBSl92WUktO2BqH2Hl3-GTWAYw6LtDm7lW3YE-dlS2MMAJV4XEQXCo7TifcPEQEtn9LH0s2AbatwLORuKdw7mxT~nLTak89qTAjVAIM7KkWkbB~m9AEtyVLoWKEPXNGaKxuHqdS616H4n9373orlSfnWCzhmqoBYwtgyfQLPB02zulnR7CrM97FtAfZQI7dtJtAAxSr2-Z3~J8WnVzzJL1ozAZMGjaHP3tY2mkF3halnzqJJ4cFtpVZWGJH7catGGXeTJ8UIeICXKqNKHC8OxySuj2kP00XSGRuA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). To have an overview of the recombination patterns of <em>PRDM</em>9 alleles, we divided the 25 Holstein bulls into six groups based on their <em>PRDM</em>9 genotypes: allele 1 homozygote, allele 2 homozygote, allele 1/2 heterozygote, allele 3 carrier (homozygote or heterozygote), allele 4 carrier (homozygote or heterozygote), and allele 5 carrier (heterozygote). After splitting, each group has 3 to 6 bulls and more than 2, 300 offspring genotyped by 50 K SNP chips, allowing us to generate <em>PRDM9</em>-specific recombination maps for individual groups (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx048/1/dsx048_supp.zip?Expires=2147483647&Signature=cB7LewD99XIfXtilEHTinjygYWutIBSPgTSSy6xH4XO0UhC59pZBSl92WUktO2BqH2Hl3-GTWAYw6LtDm7lW3YE-dlS2MMAJV4XEQXCo7TifcPEQEtn9LH0s2AbatwLORuKdw7mxT~nLTak89qTAjVAIM7KkWkbB~m9AEtyVLoWKEPXNGaKxuHqdS616H4n9373orlSfnWCzhmqoBYwtgyfQLPB02zulnR7CrM97FtAfZQI7dtJtAAxSr2-Z3~J8WnVzzJL1ozAZMGjaHP3tY2mkF3halnzqJJ4cFtpVZWGJH7catGGXeTJ8UIeICXKqNKHC8OxySuj2kP00XSGRuA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S7</a></span>). The six recombination maps exhibited similar global patterns of recombination with two peak regions, one near the 10% from the beginning of a chromosome and the other to the end. However, allele 5 carrier group showed a larger recombination rate than other alleles at both peak regions. Note that the difference between allele 5 carrier group and other groups is small due to the limitation of sample size and mixed grouping of heterozygote and homozygote individuals. Still, this observation is consistent with the observed differences in protein sequences between the five alleles, where allele 5 has the most distinct ZnFs compared to other alleles.</p><p>Using an imputation-based approach, we next aimed at generating allele-specific recombination maps for each of the five <em>PRDM</em>9 alleles. Based on the LD patterns between <em>PRDM</em>9 alleles and nearby SNPs or haplotypes in the 25 Holstein bulls, we identified markers tagging each of the five <em>PRDM</em>9 alleles (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx048/1/dsx048_supp.zip?Expires=2147483647&Signature=cB7LewD99XIfXtilEHTinjygYWutIBSPgTSSy6xH4XO0UhC59pZBSl92WUktO2BqH2Hl3-GTWAYw6LtDm7lW3YE-dlS2MMAJV4XEQXCo7TifcPEQEtn9LH0s2AbatwLORuKdw7mxT~nLTak89qTAjVAIM7KkWkbB~m9AEtyVLoWKEPXNGaKxuHqdS616H4n9373orlSfnWCzhmqoBYwtgyfQLPB02zulnR7CrM97FtAfZQI7dtJtAAxSr2-Z3~J8WnVzzJL1ozAZMGjaHP3tY2mkF3halnzqJJ4cFtpVZWGJH7catGGXeTJ8UIeICXKqNKHC8OxySuj2kP00XSGRuA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>). Using this tagging information, we extracted all available recombination data and constructed recombination maps for each of the five <em>PRDM</em>9 alleles in both males and females (<a reveal-id="dsx048-F2" data-open="dsx048-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>). In total, we extracted 1,369,139 three-generation families in Holstein, with each family including one offspring, at least one parent (maternal or paternal), and at least one grandparent. A total of 239,116 three-generation families were genotyped by chips with at least 50 K SNPs. We inferred over 3.7 million paternal and over 2 million maternal crossover events from these three-generation families. As mentioned previously, SNP rs110661033 was in perfect LD with allele 5 with a maximum <em>R</em><sup>2</sup> value of 1. Using this SNP (MAF = 9.4%) as a tagging marker, we extracted all animals (bulls and cows) carrying allele 5 and the corresponding three-generation families, with the number of families for allele 5 homozygote, allele 5 heterozygote and non-allele 5 homozygote genotypes being 637, 28, 759 and 120, 990 in males and 719, 15, 548 and 72, 513 in females, respectively. Similarly, using tagging haplotypes or SNPs, we extracted animals and three-generation families for each of the other four alleles (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx048/1/dsx048_supp.zip?Expires=2147483647&Signature=cB7LewD99XIfXtilEHTinjygYWutIBSPgTSSy6xH4XO0UhC59pZBSl92WUktO2BqH2Hl3-GTWAYw6LtDm7lW3YE-dlS2MMAJV4XEQXCo7TifcPEQEtn9LH0s2AbatwLORuKdw7mxT~nLTak89qTAjVAIM7KkWkbB~m9AEtyVLoWKEPXNGaKxuHqdS616H4n9373orlSfnWCzhmqoBYwtgyfQLPB02zulnR7CrM97FtAfZQI7dtJtAAxSr2-Z3~J8WnVzzJL1ozAZMGjaHP3tY2mkF3halnzqJJ4cFtpVZWGJH7catGGXeTJ8UIeICXKqNKHC8OxySuj2kP00XSGRuA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>). </p>                        <a id="115733740" scrollto-destination="115733740"></a>
<div id="dsx048-F2" data-id="dsx048-F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx048/1/m_dsx048f2.png?Expires=2147483647&Signature=qAH2CScGraO2Jo4G7c5QFzRZdM4g2mg8GEl-FFc~e67XrZYRP-2QjYtJjR11XAYGAHE5GfgDBJDgUgiBIoolMtGD4KiVogkyIKH6yJHYYJuTUqGggHZ93OIZZX4ZEf3vZTEnRp-N6zCk7~e-snWvL91nsq6tF3LspTabab2txMsMGiJHFMdE9BDVOC9mOvUXFllMqOoyAJogzSnOSFFw-piKOHdy6gG~ezkwU1GJaMfJFY~UzoSaMVYJGaSKD3yRBVlkMB8NJy0~TCbvsyG8m-Xgc-RKMnB0FCnGBzhUQSCVGaRCCluQ51Muy0mbPI3~r1yDVbbniZxQ3o96cZXkYg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="PRDM9 allele-specific distribution of recombination rate along a chromosome in males and females in Holstein. (a) Recombination patterns of five alleles in males. (b) Recombination patterns of five alleles in females. (c) Recombination patterns of three genotypes of allele 5 in males. (d) Recombination patterns of three genotypes of allele 5 in females. The relative physical position on a chromosome is used, where zero corresponding to the beginning of a chromosome and one the end. The smooth spline model was fitted across all of the 29 autosomes." path-from-xml="dsx048f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/115733740/dsx048f2.tif" path-from-xml="dsx048f2.tif" target="_blank">View large</a><a section="115733740" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx048/1/dsx048f2.png?Expires=2147483647&Signature=uv1ZTSz0Ih6JfVTeMO6eH4a383XzRUsVqtCLSO9Yb9lKyWneo40U3yqtSA6lGPgrhxH1qrOtFzF0Ethh3NNvN8HPA2x1BbK9YbxJX9tRPl0kbQ8pOxHnAo-x1e2lSGsuX4Q~ULjfw~34ugZrMyl4UsKgJEUp2n72umClwjQfYC5nzh-p7594PL8wbS2rgYpfInO-plcSuKm1RSw7B8aBn0Bcn06f2-MS18LABL194CJmLzWtSmLf3vGYahsPPphj7xVroxEQpqsfRZCG1dPjRFRekKiQwb4N7ea5FxrjgmgLX0Z9-hNqtupmke0VKi-Cgp3wRbUwLyWzzs14cUJUbA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=115733740&ar=4665695&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx048f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p><strong><em>PRDM9</em> allele-specific distribution of recombination rate along a chromosome in males and females in Holstein.</strong> (<strong>a</strong>) Recombination patterns of five alleles in males. (<strong>b</strong>) Recombination patterns of five alleles in females. (<strong>c</strong>) Recombination patterns of three genotypes of allele 5 in males. (<strong>d</strong>) Recombination patterns of three genotypes of allele 5 in females. The relative physical position on a chromosome is used, where zero corresponding to the beginning of a chromosome and one the end. The smooth spline model was fitted across all of the 29 autosomes.</p></div></div><div content-id="dsx048-F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx048/1/m_dsx048f2.png?Expires=2147483647&Signature=qAH2CScGraO2Jo4G7c5QFzRZdM4g2mg8GEl-FFc~e67XrZYRP-2QjYtJjR11XAYGAHE5GfgDBJDgUgiBIoolMtGD4KiVogkyIKH6yJHYYJuTUqGggHZ93OIZZX4ZEf3vZTEnRp-N6zCk7~e-snWvL91nsq6tF3LspTabab2txMsMGiJHFMdE9BDVOC9mOvUXFllMqOoyAJogzSnOSFFw-piKOHdy6gG~ezkwU1GJaMfJFY~UzoSaMVYJGaSKD3yRBVlkMB8NJy0~TCbvsyG8m-Xgc-RKMnB0FCnGBzhUQSCVGaRCCluQ51Muy0mbPI3~r1yDVbbniZxQ3o96cZXkYg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="PRDM9 allele-specific distribution of recombination rate along a chromosome in males and females in Holstein. (a) Recombination patterns of five alleles in males. (b) Recombination patterns of five alleles in females. (c) Recombination patterns of three genotypes of allele 5 in males. (d) Recombination patterns of three genotypes of allele 5 in females. The relative physical position on a chromosome is used, where zero corresponding to the beginning of a chromosome and one the end. The smooth spline model was fitted across all of the 29 autosomes." path-from-xml="dsx048f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/115733740/dsx048f2.tif" path-from-xml="dsx048f2.tif" target="_blank">View large</a><a section="115733740" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx048/1/dsx048f2.png?Expires=2147483647&Signature=uv1ZTSz0Ih6JfVTeMO6eH4a383XzRUsVqtCLSO9Yb9lKyWneo40U3yqtSA6lGPgrhxH1qrOtFzF0Ethh3NNvN8HPA2x1BbK9YbxJX9tRPl0kbQ8pOxHnAo-x1e2lSGsuX4Q~ULjfw~34ugZrMyl4UsKgJEUp2n72umClwjQfYC5nzh-p7594PL8wbS2rgYpfInO-plcSuKm1RSw7B8aBn0Bcn06f2-MS18LABL194CJmLzWtSmLf3vGYahsPPphj7xVroxEQpqsfRZCG1dPjRFRekKiQwb4N7ea5FxrjgmgLX0Z9-hNqtupmke0VKi-Cgp3wRbUwLyWzzs14cUJUbA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=115733740&ar=4665695&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx048f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p><strong><em>PRDM9</em> allele-specific distribution of recombination rate along a chromosome in males and females in Holstein.</strong> (<strong>a</strong>) Recombination patterns of five alleles in males. (<strong>b</strong>) Recombination patterns of five alleles in females. (<strong>c</strong>) Recombination patterns of three genotypes of allele 5 in males. (<strong>d</strong>) Recombination patterns of three genotypes of allele 5 in females. The relative physical position on a chromosome is used, where zero corresponding to the beginning of a chromosome and one the end. The smooth spline model was fitted across all of the 29 autosomes.</p></div></div><p>With an expanded data set using imputation, we assembled enough data to generate recombination maps for each of the five <em>PRDM</em>9 alleles in two sexes (<a reveal-id="dsx048-F2" data-open="dsx048-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>). Across the five alleles, the average number of crossovers for males and females are 25.10 and 22.74, respectively, which is consistent with previously reported higher recombination rate in bulls than in cows.<a reveal-id="dsx048-B42" data-open="dsx048-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a> While males and females exhibited different recombination maps across the genome, most notably near the end of chromosomes, <em>PRDM</em>9 alleles were associated with differences in global recombination maps within each sex (<a reveal-id="dsx048-F2" data-open="dsx048-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>). One striking observation was that among the five <em>PRDM</em>9 alleles, allele 5 showed the largest recombination maps in both sexes (<a reveal-id="dsx048-F2" data-open="dsx048-F2" class="link link-reveal link-table xref-fig">Fig. 2a and b</a>). Bulls with allele 5 homozygote, allele 5 heterozygote and non-allele 5 homozygote genotypes had a genetic map length of 27.0, 25.7 and 24.9 Morgans, respectively; and for cows, the map length dropped to 23.6, 23.4 and 22.6 Morgans for the three genotypes, respectively. These results also confirmed the previous association of SNP rs110661033 with genome-wide recombination rates with the minor allele (linked with allele 5) increasing recombination rate.<a reveal-id="dsx048-B42" data-open="dsx048-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a> For both males and females, the largest difference in recombination rate between <em>PRDM9</em> alleles fell into the two recombination peak regions, with animals carrying two copies of allele 5 having highest recombination rate and animals carrying other alleles showing lower recombination rates (<a reveal-id="dsx048-F2" data-open="dsx048-F2" class="link link-reveal link-table xref-fig">Fig. 2c and d</a>). Interestingly, animals carrying one copy of allele 5 (heterozygote) have recombination rates in between the two homozygotes, but are closer to allele 5 homozygous animals, especially in the recombination peak regions. Since allele 5 clearly stands out from the rest <em>PRDM</em>9 alleles in protein composition and recombination patterns, we generated three recombination maps for the three genotypes of allele 5 and non-allele 5 in each of the two sexes (Supp_Data_1).</p>                            <h3 scrollto-destination=115733742 id="115733742" class="section-title" >3.3. <em>PRDM9</em> allele-specific recombination maps in Jersey</h3>
<p>The 17 sequenced Jerseys had four <em>PRDM</em>9 alleles (alleles 1–4) and seven genotype combinations (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx048/1/dsx048_supp.zip?Expires=2147483647&Signature=cB7LewD99XIfXtilEHTinjygYWutIBSPgTSSy6xH4XO0UhC59pZBSl92WUktO2BqH2Hl3-GTWAYw6LtDm7lW3YE-dlS2MMAJV4XEQXCo7TifcPEQEtn9LH0s2AbatwLORuKdw7mxT~nLTak89qTAjVAIM7KkWkbB~m9AEtyVLoWKEPXNGaKxuHqdS616H4n9373orlSfnWCzhmqoBYwtgyfQLPB02zulnR7CrM97FtAfZQI7dtJtAAxSr2-Z3~J8WnVzzJL1ozAZMGjaHP3tY2mkF3halnzqJJ4cFtpVZWGJH7catGGXeTJ8UIeICXKqNKHC8OxySuj2kP00XSGRuA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>). Similarly, we evaluated the LD patterns between <em>PRDM</em>9 alleles and nearby SNPs or haplotypes in the Jersey samples and identified allele-tagging markers for Jersey (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx048/1/dsx048_supp.zip?Expires=2147483647&Signature=cB7LewD99XIfXtilEHTinjygYWutIBSPgTSSy6xH4XO0UhC59pZBSl92WUktO2BqH2Hl3-GTWAYw6LtDm7lW3YE-dlS2MMAJV4XEQXCo7TifcPEQEtn9LH0s2AbatwLORuKdw7mxT~nLTak89qTAjVAIM7KkWkbB~m9AEtyVLoWKEPXNGaKxuHqdS616H4n9373orlSfnWCzhmqoBYwtgyfQLPB02zulnR7CrM97FtAfZQI7dtJtAAxSr2-Z3~J8WnVzzJL1ozAZMGjaHP3tY2mkF3halnzqJJ4cFtpVZWGJH7catGGXeTJ8UIeICXKqNKHC8OxySuj2kP00XSGRuA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6</a></span>). As the genotyped pedigree is smaller in Jersey than in Holstein, we focused on allele 2 that had enough data to study allele-specific recombination maps. In total, 11 of 17 animals with allele 2 carry at least one minor allele of SNP Hapmap26498-BTA-33060. An association test further confirmed the correlation between this SNP and genome-wide hotspot usage in Jersey (<em>P</em>  = 5.4 × 10<sup>−3</sup>). Note that this SNP was not linked with allele 2 in Holsteins, possibly due to different LD patterns between cattle breeds. Using this tagging marker of allele 2 (MAF = 41%), we assembled 26, 945, 71, 726 and 52, 870 three-generation families for allele 2 homozygote, allele 2 heterozygote and non-allele 2 homozygote genotypes, respectively. Among these families, 3,959, 5,660 and 2,106 were genotyped by 50 K SNP chips in males, and 1,016, 1,214 and 262 genotyped by 50 K chips in females. The male map lengths are 23.3, 23.6 and 24.1 Morgans for allele 2 homozygote, allele 2 heterozygote and non-allele 2 genotypes, respectively; while female maps are 22.0, 22.0 and 22.6 Morgans in length, respectively (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx048/1/dsx048_supp.zip?Expires=2147483647&Signature=cB7LewD99XIfXtilEHTinjygYWutIBSPgTSSy6xH4XO0UhC59pZBSl92WUktO2BqH2Hl3-GTWAYw6LtDm7lW3YE-dlS2MMAJV4XEQXCo7TifcPEQEtn9LH0s2AbatwLORuKdw7mxT~nLTak89qTAjVAIM7KkWkbB~m9AEtyVLoWKEPXNGaKxuHqdS616H4n9373orlSfnWCzhmqoBYwtgyfQLPB02zulnR7CrM97FtAfZQI7dtJtAAxSr2-Z3~J8WnVzzJL1ozAZMGjaHP3tY2mkF3halnzqJJ4cFtpVZWGJH7catGGXeTJ8UIeICXKqNKHC8OxySuj2kP00XSGRuA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S8</a></span>). Consistent with the patterns found in Holstein, main differences between <em>PRDM9</em> alleles in Jersey were also found in the two recombination peak regions: in the centromere peak region, allele 2 homozygote map had the lowest recombination rate in both males and females; and in the telomere region, allele 2 carriers had the lowest recombination rate in females and was close to the lowest in males. Note that recombination rate in Jersey is in general slightly smaller than that in Holstein, likely due to lower polymorphism levels of markers in Jersey as the SNP chips were originally designed for Holstein.</p>                            <h3 scrollto-destination=115733744 id="115733744" class="section-title" >3.4. <em>PRDM</em>9 allele-specific hotspot regions</h3>
<p>Recombination rates were calculated between consecutive SNPs for animals that carry different <em>PRDM</em>9 alleles, showing allele-specific distributions of recombination across the cattle genome (<a reveal-id="dsx048-F3" data-open="dsx048-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>). We here used the term ‘hotspot region’ instead of ‘hotspot’ because our SNP intervals were much larger (average 44 Kb) than typical recombination hotspot regions in human and mouse studies. We focused on the most distinct <em>PRDM</em>9 alleles in this analysis, allele 5 and non-allele 5. To find allele-specific hotspot regions for allele 5 and non-allele 5, we applied a χ<sup>2</sup> test to identify SNP intervals with significantly different recombination rate between the three genotypes of allele 5 and non-allele 5 in both sexes of Holstein (<a reveal-id="dsx048-F3" data-open="dsx048-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>; <a reveal-id="dsx048-T1" data-open="dsx048-T1" class="link link-reveal link-table xref-fig">Tables 1</a> and <a reveal-id="dsx048-T2" data-open="dsx048-T2" class="link link-reveal link-table xref-fig">2</a>). Based on the genome-wide significance level of 8.3 × 10<sup>−7</sup> after Bonferroni correction, in males, we identified seven SNP intervals with different recombination rate between allele 5 homozygote and non-allele 5 genotypes, 369 intervals between allele 5 heterozygote and non-allele 5 genotypes, and only 1 interval between allele 5 homozygote and allele 5 heterozygote, respectively. In females, the numbers of intervals with different recombination rate between the three genotypes were 2, 36 and 0, respectively. The different number of identified intervals for these comparisons reflected different sample sizes and statistical power, because the recombination data involved more bulls than cows and had more animals carrying non-allele 5 than allele 5. To account for the issue of unbalanced sample sizes, we adopted multiple evidence to select the most significant allele-specific hotspot regions of allele 5:(1) Same direction of the difference in recombination rate for males and females, (2) Small <em>P</em>-value for the comparison between allele 5 homozygote and non-allele 5 genotype, and (3) Small <em>P</em>-value in the comparison of allele 5 heterozygote versus allele 5 homozygote or non-allele 5 genotype. For non-allele 5, we dropped the comparison between allele 5 homozygote and non-allele 5 genotype due to the limited sample sizes and low statistical power. </p>                        <a id="115733746" scrollto-destination="115733746"></a>
<div id="dsx048-F3" data-id="dsx048-F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx048/1/m_dsx048f3.png?Expires=2147483647&Signature=fa7mCHxfI1lx1S3CQcu1zEREqiMQN-GJlpXXX29Q6ZeQfBqMnbmRMJMRPNvC14Vzd8E5i6DEwv~KA1NDPjzTYczV2Bw6ny56wB~o4hZkkStywyAwGP~9seim1YhF1IBhymXPz0UrTeRobIIslQuPvp9qRpGsotGUTPbE9~I7UxZq8Dlmo8LezRVYYZS1LZtEYhlB1yKtGnxrMG5eFK48imeFS7sIofyHB2a1H7FuwbadXrQbn5jNzvEc~DRVxpTI~g0E9AzMDkDEOwu8ZN9cED5B1UDCWUO8TfPUT4qW4K4hL0J3Ol9c9w7gycEKf8e2lrR-sNFryCZcnIvXDld3Gw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="PRDM9 allele 5 dependent recombination hotspots in two sexes. (a) Non-allele 5 v.s. allele 5 homozygote in males. (b) Non-allele 5 v.s. allele 5 homozygote in females. (c) Non-allele 5 v.s. allele 5 heterozygote in males. (d) Non-allele 5 v.s. allele 5 heterozygote in females. (e) Allele 5 heterozygote v.s. allele 5 homozygote in males. (f) Allele 5 heterozygote v.s. allele 5 homozygote in females. For each panel, recombination rates in each SNP intervals of two groups were shown in the top half and corresponding P-values were shown in the bottom half. Different colors were used to distinguish the 29 chromosomes." path-from-xml="dsx048f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/115733746/dsx048f3.tif" path-from-xml="dsx048f3.tif" target="_blank">View large</a><a section="115733746" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx048/1/dsx048f3.png?Expires=2147483647&Signature=Apd8jcP3xha-jhgsITGZor2u8SRqUVYiCW350uuO3-YWIEWzwW8uRTUDPElKJI271ei-iT3MhmxhvWfDwCd0Ykc1sHJrBGT3mnlHmD3VcAtizUoKXMYUo4KqT14e5dnN4wlEKv6mFKM002Ej5EDdBTF5gPGTIsbKX2m0bf3jI~ssa1JR17xuGOGsOP08vBdAIbzua~ixqpioW-4dGOA~7ZdUuelaVrz7vRz~gTj9P1Ikw0O~GrFMQHsabzmSWPtJGQ3WSxx3fc4rPz2JtdgNrBBv3eQlJcc-1YMgBjSWbs6zUJZXv7PbU8XHa-mtRDnaf6p35NqhBfw7eZIc3QEsGQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=115733746&ar=4665695&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx048f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p><strong><em>PRDM9</em> allele 5 dependent recombination hotspots in two sexes.</strong> (<strong>a</strong>) Non-allele 5 v.s. allele 5 homozygote in males. (<strong>b</strong>) Non-allele 5 v.s. allele 5 homozygote in females. (<strong>c</strong>) Non-allele 5 v.s. allele 5 heterozygote in males. (<strong>d</strong>) Non-allele 5 v.s. allele 5 heterozygote in females. (<strong>e</strong>) Allele 5 heterozygote v.s. allele 5 homozygote in males. (<strong>f</strong>) Allele 5 heterozygote v.s. allele 5 homozygote in females. For each panel, recombination rates in each SNP intervals of two groups were shown in the top half and corresponding <em>P</em>-values were shown in the bottom half. Different colors were used to distinguish the 29 chromosomes.</p></div></div><div content-id="dsx048-F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx048/1/m_dsx048f3.png?Expires=2147483647&Signature=fa7mCHxfI1lx1S3CQcu1zEREqiMQN-GJlpXXX29Q6ZeQfBqMnbmRMJMRPNvC14Vzd8E5i6DEwv~KA1NDPjzTYczV2Bw6ny56wB~o4hZkkStywyAwGP~9seim1YhF1IBhymXPz0UrTeRobIIslQuPvp9qRpGsotGUTPbE9~I7UxZq8Dlmo8LezRVYYZS1LZtEYhlB1yKtGnxrMG5eFK48imeFS7sIofyHB2a1H7FuwbadXrQbn5jNzvEc~DRVxpTI~g0E9AzMDkDEOwu8ZN9cED5B1UDCWUO8TfPUT4qW4K4hL0J3Ol9c9w7gycEKf8e2lrR-sNFryCZcnIvXDld3Gw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="PRDM9 allele 5 dependent recombination hotspots in two sexes. (a) Non-allele 5 v.s. allele 5 homozygote in males. (b) Non-allele 5 v.s. allele 5 homozygote in females. (c) Non-allele 5 v.s. allele 5 heterozygote in males. (d) Non-allele 5 v.s. allele 5 heterozygote in females. (e) Allele 5 heterozygote v.s. allele 5 homozygote in males. (f) Allele 5 heterozygote v.s. allele 5 homozygote in females. For each panel, recombination rates in each SNP intervals of two groups were shown in the top half and corresponding P-values were shown in the bottom half. Different colors were used to distinguish the 29 chromosomes." path-from-xml="dsx048f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/115733746/dsx048f3.tif" path-from-xml="dsx048f3.tif" target="_blank">View large</a><a section="115733746" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx048/1/dsx048f3.png?Expires=2147483647&Signature=Apd8jcP3xha-jhgsITGZor2u8SRqUVYiCW350uuO3-YWIEWzwW8uRTUDPElKJI271ei-iT3MhmxhvWfDwCd0Ykc1sHJrBGT3mnlHmD3VcAtizUoKXMYUo4KqT14e5dnN4wlEKv6mFKM002Ej5EDdBTF5gPGTIsbKX2m0bf3jI~ssa1JR17xuGOGsOP08vBdAIbzua~ixqpioW-4dGOA~7ZdUuelaVrz7vRz~gTj9P1Ikw0O~GrFMQHsabzmSWPtJGQ3WSxx3fc4rPz2JtdgNrBBv3eQlJcc-1YMgBjSWbs6zUJZXv7PbU8XHa-mtRDnaf6p35NqhBfw7eZIc3QEsGQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=115733746&ar=4665695&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx048f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p><strong><em>PRDM9</em> allele 5 dependent recombination hotspots in two sexes.</strong> (<strong>a</strong>) Non-allele 5 v.s. allele 5 homozygote in males. (<strong>b</strong>) Non-allele 5 v.s. allele 5 homozygote in females. (<strong>c</strong>) Non-allele 5 v.s. allele 5 heterozygote in males. (<strong>d</strong>) Non-allele 5 v.s. allele 5 heterozygote in females. (<strong>e</strong>) Allele 5 heterozygote v.s. allele 5 homozygote in males. (<strong>f</strong>) Allele 5 heterozygote v.s. allele 5 homozygote in females. For each panel, recombination rates in each SNP intervals of two groups were shown in the top half and corresponding <em>P</em>-values were shown in the bottom half. Different colors were used to distinguish the 29 chromosomes.</p></div></div><p>Among the top 5 hotspot regions of allele 5, one hotspot region, located at chr13: 14923596-15017558, is shared between males and females (<a reveal-id="dsx048-T1" data-open="dsx048-T1" class="link link-reveal link-table xref-fig">Table 1</a>). In males, the recombination rate increased 13 fold from 0.0006 to 0.009 between animals with non-allele 5 and two copies of allele 5. In females, animals carrying two copies of allele 5 showed a 5-fold increase. To further investigate the recombination patterns of allele-specific hotspot regions, we zoomed into the local recombination maps of the three <em>PRDM</em>9 genotypes of allele 5 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx048/1/dsx048_supp.zip?Expires=2147483647&Signature=cB7LewD99XIfXtilEHTinjygYWutIBSPgTSSy6xH4XO0UhC59pZBSl92WUktO2BqH2Hl3-GTWAYw6LtDm7lW3YE-dlS2MMAJV4XEQXCo7TifcPEQEtn9LH0s2AbatwLORuKdw7mxT~nLTak89qTAjVAIM7KkWkbB~m9AEtyVLoWKEPXNGaKxuHqdS616H4n9373orlSfnWCzhmqoBYwtgyfQLPB02zulnR7CrM97FtAfZQI7dtJtAAxSr2-Z3~J8WnVzzJL1ozAZMGjaHP3tY2mkF3halnzqJJ4cFtpVZWGJH7catGGXeTJ8UIeICXKqNKHC8OxySuj2kP00XSGRuA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S9</a></span>). Generally, for the allele 5 favored hotspot regions, we found a striking peak of the recombination rates of allele 5 compared to other genotypes. As for non-allele 5 hotspot regions, we also observed higher recombination rates for animals carrying alleles other than 5. In both cases, recombination rates of allele 5 heterozygote would mimic either allele 5 or non-allele 5 since the heterozygote animals had both alleles and binding motifs. <div content-id="dsx048-T1" class="table-modal"><div class="table-wrap"><div id="dsx048-T1" data-id="dsx048-T1" class="table-wrap-title"><div class="label -label">Table 1.</div><div class="caption"><p>Most significant allele 5 specific hotspot regions for males and females in Holstein cattle</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Chr </th><th rowspan="2">Start </th><th rowspan="2">End </th><th colspan="6">Recombination rate<hr /> </th><th colspan="2">Comparison<hr /> </th></tr><tr><th>M0 </th><th>M1 </th><th>M2 </th><th>F0 </th><th>F1 </th><th>F2 </th><th><em>P</em>-value1 </th><th><em>P</em>-value2 </th></tr></thead><tbody><tr><td>23 </td><td>40,203,865 </td><td>40,236,175 </td><td>0.00061 </td><td>0.00200 </td><td>0.01006 </td><td>0.00061 </td><td>0.00089 </td><td>0.00076 </td><td>1.9 × 10<sup>−17</sup> </td><td>9.8 × 10<sup>−5</sup> </td></tr><tr><td>13 </td><td>14,923,596 </td><td>15,017,558 </td><td>0.00067 </td><td>0.00281 </td><td>0.00905 </td><td>0.00078 </td><td>0.00314 </td><td>0.00383 </td><td>9.4 × 10<sup>−13</sup> </td><td>1.2 × 10<sup>−2</sup> </td></tr><tr><td>4 </td><td>21,160,109 </td><td>21,179,159 </td><td>0.00047 </td><td>0.00050 </td><td>0.00733 </td><td>0.00056 </td><td>0.00062 </td><td>0.00082 </td><td>1.0 × 10<sup>−11</sup> </td><td>3.1 × 10<sup>−09</sup> </td></tr><tr><td>1 </td><td>152,566,977 </td><td>152,592,454 </td><td>0.00126 </td><td>0.00252 </td><td>0.00975 </td><td>0.00023 </td><td>0.00030 </td><td>0.00017 </td><td>7.6 × 10<sup>−8</sup> </td><td>1.9 × 10<sup>−3</sup> </td></tr><tr><td>18 </td><td>55,956,772 </td><td>55,983,042 </td><td>0.00041 </td><td>0.00073 </td><td>0.00544 </td><td>0.00048 </td><td>0.00044 </td><td>0.00023 </td><td>3.6 × 10<sup>−7</sup> </td><td>7.1 × 10<sup>−4</sup> </td></tr><tr><td>13 </td><td>14,923,596 </td><td>15,017,558 </td><td>0.00067 </td><td>0.00281 </td><td>0.00905 </td><td>0.00078 </td><td>0.00314 </td><td>0.00383 </td><td>2.7 × 10<sup>−2</sup> </td><td>2.4 × 10<sup>−14</sup> </td></tr><tr><td>18 </td><td>7,014,208 </td><td>7,041,735 </td><td>0.00048 </td><td>0.00220 </td><td>0.00114 </td><td>0.00032 </td><td>0.00157 </td><td>0.00308 </td><td>3.4 × 10<sup>−3</sup> </td><td>4.2 × 10<sup>−9</sup> </td></tr><tr><td>23 </td><td>40,366,428 </td><td>40,416,708 </td><td>0.00062 </td><td>0.00193 </td><td>0.00079 </td><td>0.00059 </td><td>0.00192 </td><td>0.00346 </td><td>2.0 × 10<sup>−2</sup> </td><td>4.1 × 10<sup>−7</sup> </td></tr><tr><td>11 </td><td>24,175,823 </td><td>24,193,422 </td><td>0.00025 </td><td>0.00056 </td><td>0.00031 </td><td>0.00029 </td><td>0.00084 </td><td>0.00441 </td><td>5.5 × 10<sup>−7</sup> </td><td>3.2 × 10<sup>−3</sup> </td></tr><tr><td>7 </td><td>34,678,536 </td><td>34,709,628 </td><td>0.00062 </td><td>0.00091 </td><td>0.00075 </td><td>0.00074 </td><td>0.00211 </td><td>0.00349 </td><td>4.8 × 10<sup>−2</sup> </td><td>1.5 × 10<sup>−6</sup> </td></tr></tbody></table></div><div content-id="dsx048-T1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Chr </th><th rowspan="2">Start </th><th rowspan="2">End </th><th colspan="6">Recombination rate<hr /> </th><th colspan="2">Comparison<hr /> </th></tr><tr><th>M0 </th><th>M1 </th><th>M2 </th><th>F0 </th><th>F1 </th><th>F2 </th><th><em>P</em>-value1 </th><th><em>P</em>-value2 </th></tr></thead><tbody><tr><td>23 </td><td>40,203,865 </td><td>40,236,175 </td><td>0.00061 </td><td>0.00200 </td><td>0.01006 </td><td>0.00061 </td><td>0.00089 </td><td>0.00076 </td><td>1.9 × 10<sup>−17</sup> </td><td>9.8 × 10<sup>−5</sup> </td></tr><tr><td>13 </td><td>14,923,596 </td><td>15,017,558 </td><td>0.00067 </td><td>0.00281 </td><td>0.00905 </td><td>0.00078 </td><td>0.00314 </td><td>0.00383 </td><td>9.4 × 10<sup>−13</sup> </td><td>1.2 × 10<sup>−2</sup> </td></tr><tr><td>4 </td><td>21,160,109 </td><td>21,179,159 </td><td>0.00047 </td><td>0.00050 </td><td>0.00733 </td><td>0.00056 </td><td>0.00062 </td><td>0.00082 </td><td>1.0 × 10<sup>−11</sup> </td><td>3.1 × 10<sup>−09</sup> </td></tr><tr><td>1 </td><td>152,566,977 </td><td>152,592,454 </td><td>0.00126 </td><td>0.00252 </td><td>0.00975 </td><td>0.00023 </td><td>0.00030 </td><td>0.00017 </td><td>7.6 × 10<sup>−8</sup> </td><td>1.9 × 10<sup>−3</sup> </td></tr><tr><td>18 </td><td>55,956,772 </td><td>55,983,042 </td><td>0.00041 </td><td>0.00073 </td><td>0.00544 </td><td>0.00048 </td><td>0.00044 </td><td>0.00023 </td><td>3.6 × 10<sup>−7</sup> </td><td>7.1 × 10<sup>−4</sup> </td></tr><tr><td>13 </td><td>14,923,596 </td><td>15,017,558 </td><td>0.00067 </td><td>0.00281 </td><td>0.00905 </td><td>0.00078 </td><td>0.00314 </td><td>0.00383 </td><td>2.7 × 10<sup>−2</sup> </td><td>2.4 × 10<sup>−14</sup> </td></tr><tr><td>18 </td><td>7,014,208 </td><td>7,041,735 </td><td>0.00048 </td><td>0.00220 </td><td>0.00114 </td><td>0.00032 </td><td>0.00157 </td><td>0.00308 </td><td>3.4 × 10<sup>−3</sup> </td><td>4.2 × 10<sup>−9</sup> </td></tr><tr><td>23 </td><td>40,366,428 </td><td>40,416,708 </td><td>0.00062 </td><td>0.00193 </td><td>0.00079 </td><td>0.00059 </td><td>0.00192 </td><td>0.00346 </td><td>2.0 × 10<sup>−2</sup> </td><td>4.1 × 10<sup>−7</sup> </td></tr><tr><td>11 </td><td>24,175,823 </td><td>24,193,422 </td><td>0.00025 </td><td>0.00056 </td><td>0.00031 </td><td>0.00029 </td><td>0.00084 </td><td>0.00441 </td><td>5.5 × 10<sup>−7</sup> </td><td>3.2 × 10<sup>−3</sup> </td></tr><tr><td>7 </td><td>34,678,536 </td><td>34,709,628 </td><td>0.00062 </td><td>0.00091 </td><td>0.00075 </td><td>0.00074 </td><td>0.00211 </td><td>0.00349 </td><td>4.8 × 10<sup>−2</sup> </td><td>1.5 × 10<sup>−6</sup> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>Start, End = physical position of hotspot regions on chromosome. M0, M1 and M2 = recombination rates of males with non-allele 5, allele 5 heterozygote and allele 5 homozygote genotypes, respectively. F0, F1 and F2 = female recombination rates. <em>P</em>-value 1= allele 5 homozygote versus non-allele 5 genotype. <em>P</em>-value 2 = allele 5 heterozygote versus non-allele 5 genotype. Top 5 rows are from male results and bottom 5 rows are female hotspots.</p></div></div></div></div></div><div class="table-wrap"><div id="dsx048-T1" data-id="dsx048-T1" class="table-wrap-title"><div class="label -label">Table 1.</div><div class="caption"><p>Most significant allele 5 specific hotspot regions for males and females in Holstein cattle</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Chr </th><th rowspan="2">Start </th><th rowspan="2">End </th><th colspan="6">Recombination rate<hr /> </th><th colspan="2">Comparison<hr /> </th></tr><tr><th>M0 </th><th>M1 </th><th>M2 </th><th>F0 </th><th>F1 </th><th>F2 </th><th><em>P</em>-value1 </th><th><em>P</em>-value2 </th></tr></thead><tbody><tr><td>23 </td><td>40,203,865 </td><td>40,236,175 </td><td>0.00061 </td><td>0.00200 </td><td>0.01006 </td><td>0.00061 </td><td>0.00089 </td><td>0.00076 </td><td>1.9 × 10<sup>−17</sup> </td><td>9.8 × 10<sup>−5</sup> </td></tr><tr><td>13 </td><td>14,923,596 </td><td>15,017,558 </td><td>0.00067 </td><td>0.00281 </td><td>0.00905 </td><td>0.00078 </td><td>0.00314 </td><td>0.00383 </td><td>9.4 × 10<sup>−13</sup> </td><td>1.2 × 10<sup>−2</sup> </td></tr><tr><td>4 </td><td>21,160,109 </td><td>21,179,159 </td><td>0.00047 </td><td>0.00050 </td><td>0.00733 </td><td>0.00056 </td><td>0.00062 </td><td>0.00082 </td><td>1.0 × 10<sup>−11</sup> </td><td>3.1 × 10<sup>−09</sup> </td></tr><tr><td>1 </td><td>152,566,977 </td><td>152,592,454 </td><td>0.00126 </td><td>0.00252 </td><td>0.00975 </td><td>0.00023 </td><td>0.00030 </td><td>0.00017 </td><td>7.6 × 10<sup>−8</sup> </td><td>1.9 × 10<sup>−3</sup> </td></tr><tr><td>18 </td><td>55,956,772 </td><td>55,983,042 </td><td>0.00041 </td><td>0.00073 </td><td>0.00544 </td><td>0.00048 </td><td>0.00044 </td><td>0.00023 </td><td>3.6 × 10<sup>−7</sup> </td><td>7.1 × 10<sup>−4</sup> </td></tr><tr><td>13 </td><td>14,923,596 </td><td>15,017,558 </td><td>0.00067 </td><td>0.00281 </td><td>0.00905 </td><td>0.00078 </td><td>0.00314 </td><td>0.00383 </td><td>2.7 × 10<sup>−2</sup> </td><td>2.4 × 10<sup>−14</sup> </td></tr><tr><td>18 </td><td>7,014,208 </td><td>7,041,735 </td><td>0.00048 </td><td>0.00220 </td><td>0.00114 </td><td>0.00032 </td><td>0.00157 </td><td>0.00308 </td><td>3.4 × 10<sup>−3</sup> </td><td>4.2 × 10<sup>−9</sup> </td></tr><tr><td>23 </td><td>40,366,428 </td><td>40,416,708 </td><td>0.00062 </td><td>0.00193 </td><td>0.00079 </td><td>0.00059 </td><td>0.00192 </td><td>0.00346 </td><td>2.0 × 10<sup>−2</sup> </td><td>4.1 × 10<sup>−7</sup> </td></tr><tr><td>11 </td><td>24,175,823 </td><td>24,193,422 </td><td>0.00025 </td><td>0.00056 </td><td>0.00031 </td><td>0.00029 </td><td>0.00084 </td><td>0.00441 </td><td>5.5 × 10<sup>−7</sup> </td><td>3.2 × 10<sup>−3</sup> </td></tr><tr><td>7 </td><td>34,678,536 </td><td>34,709,628 </td><td>0.00062 </td><td>0.00091 </td><td>0.00075 </td><td>0.00074 </td><td>0.00211 </td><td>0.00349 </td><td>4.8 × 10<sup>−2</sup> </td><td>1.5 × 10<sup>−6</sup> </td></tr></tbody></table></div><div content-id="dsx048-T1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Chr </th><th rowspan="2">Start </th><th rowspan="2">End </th><th colspan="6">Recombination rate<hr /> </th><th colspan="2">Comparison<hr /> </th></tr><tr><th>M0 </th><th>M1 </th><th>M2 </th><th>F0 </th><th>F1 </th><th>F2 </th><th><em>P</em>-value1 </th><th><em>P</em>-value2 </th></tr></thead><tbody><tr><td>23 </td><td>40,203,865 </td><td>40,236,175 </td><td>0.00061 </td><td>0.00200 </td><td>0.01006 </td><td>0.00061 </td><td>0.00089 </td><td>0.00076 </td><td>1.9 × 10<sup>−17</sup> </td><td>9.8 × 10<sup>−5</sup> </td></tr><tr><td>13 </td><td>14,923,596 </td><td>15,017,558 </td><td>0.00067 </td><td>0.00281 </td><td>0.00905 </td><td>0.00078 </td><td>0.00314 </td><td>0.00383 </td><td>9.4 × 10<sup>−13</sup> </td><td>1.2 × 10<sup>−2</sup> </td></tr><tr><td>4 </td><td>21,160,109 </td><td>21,179,159 </td><td>0.00047 </td><td>0.00050 </td><td>0.00733 </td><td>0.00056 </td><td>0.00062 </td><td>0.00082 </td><td>1.0 × 10<sup>−11</sup> </td><td>3.1 × 10<sup>−09</sup> </td></tr><tr><td>1 </td><td>152,566,977 </td><td>152,592,454 </td><td>0.00126 </td><td>0.00252 </td><td>0.00975 </td><td>0.00023 </td><td>0.00030 </td><td>0.00017 </td><td>7.6 × 10<sup>−8</sup> </td><td>1.9 × 10<sup>−3</sup> </td></tr><tr><td>18 </td><td>55,956,772 </td><td>55,983,042 </td><td>0.00041 </td><td>0.00073 </td><td>0.00544 </td><td>0.00048 </td><td>0.00044 </td><td>0.00023 </td><td>3.6 × 10<sup>−7</sup> </td><td>7.1 × 10<sup>−4</sup> </td></tr><tr><td>13 </td><td>14,923,596 </td><td>15,017,558 </td><td>0.00067 </td><td>0.00281 </td><td>0.00905 </td><td>0.00078 </td><td>0.00314 </td><td>0.00383 </td><td>2.7 × 10<sup>−2</sup> </td><td>2.4 × 10<sup>−14</sup> </td></tr><tr><td>18 </td><td>7,014,208 </td><td>7,041,735 </td><td>0.00048 </td><td>0.00220 </td><td>0.00114 </td><td>0.00032 </td><td>0.00157 </td><td>0.00308 </td><td>3.4 × 10<sup>−3</sup> </td><td>4.2 × 10<sup>−9</sup> </td></tr><tr><td>23 </td><td>40,366,428 </td><td>40,416,708 </td><td>0.00062 </td><td>0.00193 </td><td>0.00079 </td><td>0.00059 </td><td>0.00192 </td><td>0.00346 </td><td>2.0 × 10<sup>−2</sup> </td><td>4.1 × 10<sup>−7</sup> </td></tr><tr><td>11 </td><td>24,175,823 </td><td>24,193,422 </td><td>0.00025 </td><td>0.00056 </td><td>0.00031 </td><td>0.00029 </td><td>0.00084 </td><td>0.00441 </td><td>5.5 × 10<sup>−7</sup> </td><td>3.2 × 10<sup>−3</sup> </td></tr><tr><td>7 </td><td>34,678,536 </td><td>34,709,628 </td><td>0.00062 </td><td>0.00091 </td><td>0.00075 </td><td>0.00074 </td><td>0.00211 </td><td>0.00349 </td><td>4.8 × 10<sup>−2</sup> </td><td>1.5 × 10<sup>−6</sup> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>Start, End = physical position of hotspot regions on chromosome. M0, M1 and M2 = recombination rates of males with non-allele 5, allele 5 heterozygote and allele 5 homozygote genotypes, respectively. F0, F1 and F2 = female recombination rates. <em>P</em>-value 1= allele 5 homozygote versus non-allele 5 genotype. <em>P</em>-value 2 = allele 5 heterozygote versus non-allele 5 genotype. Top 5 rows are from male results and bottom 5 rows are female hotspots.</p></div></div></div></div><div content-id="dsx048-T2" class="table-modal"><div class="table-wrap"><div id="dsx048-T2" data-id="dsx048-T2" class="table-wrap-title"><div class="label -label">Table 2.</div><div class="caption"><p>Most significant non-allele 5 hotspot regions for males and females in Holstein cattle</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Chr </th><th rowspan="2">Start </th><th rowspan="2">End </th><th colspan="6">Recombination rate<hr /> </th><th>Comparison </th></tr><tr><th>M0 </th><th>M1 </th><th>M2 </th><th>F0 </th><th>F1 </th><th>F2 </th><th><em>P</em>-value </th></tr></thead><tbody><tr><td>21 </td><td>67,159,217 </td><td>67,195,140 </td><td>0.00196 </td><td>0.00072 </td><td>0.00011 </td><td>0.00084 </td><td>0.00038 </td><td>0.00035 </td><td>7.4 × 10<sup>−6</sup> </td></tr><tr><td>19 </td><td>54,920,324 </td><td>54,947,863 </td><td>0.00216 </td><td>0.00069 </td><td>0.00059 </td><td>0.00072 </td><td>0.00049 </td><td>0.00063 </td><td>3.4 × 10<sup>−7</sup> </td></tr><tr><td>4 </td><td>18,380,952 </td><td>18,447,308 </td><td>0.002 </td><td>0.00108 </td><td>0.00057 </td><td>0.0016 </td><td>0.00122 </td><td>0.00095 </td><td>1.3 × 10<sup>−3</sup> </td></tr><tr><td>18 </td><td>624,6055 </td><td>6,275,418 </td><td>0.00155 </td><td>0.00069 </td><td>0.00020 </td><td>0.00092 </td><td>0.00030 </td><td>0.00034 </td><td>6.4 × 10<sup>−4</sup> </td></tr><tr><td>2 </td><td>12,415,870 </td><td>12,451,708 </td><td>0.00126 </td><td>0.00068 </td><td>9.45E-05 </td><td>0.00072 </td><td>0.00057 </td><td>0.00014 </td><td>1.2 × 10<sup>−2</sup> </td></tr><tr><td>22 </td><td>33,218,085 </td><td>33,245,588 </td><td>0.00048 </td><td>0.00025 </td><td>0.00027 </td><td>0.00171 </td><td>0.00072 </td><td>0.00038 </td><td>6.1 × 10<sup>−3</sup> </td></tr><tr><td>5 </td><td>8,435,987 </td><td>8,476,871 </td><td>0.00079 </td><td>0.00036 </td><td>0.00040 </td><td>0.00119 </td><td>0.00034 </td><td>5.28E-05 </td><td>4.5 × 10<sup>−3</sup> </td></tr><tr><td>7 </td><td>34,269,024 </td><td>34,324,708 </td><td>0.00084 </td><td>0.00047 </td><td>0.00075 </td><td>0.00129 </td><td>0.00062 </td><td>0.00018 </td><td>3.9 × 10<sup>−2</sup> </td></tr><tr><td>26 </td><td>26,340,616 </td><td>26,369,697 </td><td>0.00054 </td><td>0.00033 </td><td>0.00038 </td><td>0.00159 </td><td>0.00064 </td><td>0.00051 </td><td>6.3 × 10<sup>−3</sup> </td></tr><tr><td>10 </td><td>7,791,755 </td><td>7,830,003 </td><td>0.00148 </td><td>0.00072 </td><td>0.00043 </td><td>0.0017 </td><td>0.00083 </td><td>0.00068 </td><td>1.6 × 10<sup>−2</sup> </td></tr></tbody></table></div><div content-id="dsx048-T2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Chr </th><th rowspan="2">Start </th><th rowspan="2">End </th><th colspan="6">Recombination rate<hr /> </th><th>Comparison </th></tr><tr><th>M0 </th><th>M1 </th><th>M2 </th><th>F0 </th><th>F1 </th><th>F2 </th><th><em>P</em>-value </th></tr></thead><tbody><tr><td>21 </td><td>67,159,217 </td><td>67,195,140 </td><td>0.00196 </td><td>0.00072 </td><td>0.00011 </td><td>0.00084 </td><td>0.00038 </td><td>0.00035 </td><td>7.4 × 10<sup>−6</sup> </td></tr><tr><td>19 </td><td>54,920,324 </td><td>54,947,863 </td><td>0.00216 </td><td>0.00069 </td><td>0.00059 </td><td>0.00072 </td><td>0.00049 </td><td>0.00063 </td><td>3.4 × 10<sup>−7</sup> </td></tr><tr><td>4 </td><td>18,380,952 </td><td>18,447,308 </td><td>0.002 </td><td>0.00108 </td><td>0.00057 </td><td>0.0016 </td><td>0.00122 </td><td>0.00095 </td><td>1.3 × 10<sup>−3</sup> </td></tr><tr><td>18 </td><td>624,6055 </td><td>6,275,418 </td><td>0.00155 </td><td>0.00069 </td><td>0.00020 </td><td>0.00092 </td><td>0.00030 </td><td>0.00034 </td><td>6.4 × 10<sup>−4</sup> </td></tr><tr><td>2 </td><td>12,415,870 </td><td>12,451,708 </td><td>0.00126 </td><td>0.00068 </td><td>9.45E-05 </td><td>0.00072 </td><td>0.00057 </td><td>0.00014 </td><td>1.2 × 10<sup>−2</sup> </td></tr><tr><td>22 </td><td>33,218,085 </td><td>33,245,588 </td><td>0.00048 </td><td>0.00025 </td><td>0.00027 </td><td>0.00171 </td><td>0.00072 </td><td>0.00038 </td><td>6.1 × 10<sup>−3</sup> </td></tr><tr><td>5 </td><td>8,435,987 </td><td>8,476,871 </td><td>0.00079 </td><td>0.00036 </td><td>0.00040 </td><td>0.00119 </td><td>0.00034 </td><td>5.28E-05 </td><td>4.5 × 10<sup>−3</sup> </td></tr><tr><td>7 </td><td>34,269,024 </td><td>34,324,708 </td><td>0.00084 </td><td>0.00047 </td><td>0.00075 </td><td>0.00129 </td><td>0.00062 </td><td>0.00018 </td><td>3.9 × 10<sup>−2</sup> </td></tr><tr><td>26 </td><td>26,340,616 </td><td>26,369,697 </td><td>0.00054 </td><td>0.00033 </td><td>0.00038 </td><td>0.00159 </td><td>0.00064 </td><td>0.00051 </td><td>6.3 × 10<sup>−3</sup> </td></tr><tr><td>10 </td><td>7,791,755 </td><td>7,830,003 </td><td>0.00148 </td><td>0.00072 </td><td>0.00043 </td><td>0.0017 </td><td>0.00083 </td><td>0.00068 </td><td>1.6 × 10<sup>−2</sup> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn2"><div class="fn"><p>Start, End = physical position of hotspot regions on chromosome. M0, M1 and M2 = recombination rates of males with non-allele 5, allele 5 heterozygote and allele 5 homozygote genotypes, respectively. F0, F1 and F2 = female recombination rates. <em>P</em>-value = allele 5 heterozygote versus non-allele 5 genotype. Top 5 rows are from male results and bottom 5 rows are female hotspots.</p></div></div></div></div></div><div class="table-wrap"><div id="dsx048-T2" data-id="dsx048-T2" class="table-wrap-title"><div class="label -label">Table 2.</div><div class="caption"><p>Most significant non-allele 5 hotspot regions for males and females in Holstein cattle</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Chr </th><th rowspan="2">Start </th><th rowspan="2">End </th><th colspan="6">Recombination rate<hr /> </th><th>Comparison </th></tr><tr><th>M0 </th><th>M1 </th><th>M2 </th><th>F0 </th><th>F1 </th><th>F2 </th><th><em>P</em>-value </th></tr></thead><tbody><tr><td>21 </td><td>67,159,217 </td><td>67,195,140 </td><td>0.00196 </td><td>0.00072 </td><td>0.00011 </td><td>0.00084 </td><td>0.00038 </td><td>0.00035 </td><td>7.4 × 10<sup>−6</sup> </td></tr><tr><td>19 </td><td>54,920,324 </td><td>54,947,863 </td><td>0.00216 </td><td>0.00069 </td><td>0.00059 </td><td>0.00072 </td><td>0.00049 </td><td>0.00063 </td><td>3.4 × 10<sup>−7</sup> </td></tr><tr><td>4 </td><td>18,380,952 </td><td>18,447,308 </td><td>0.002 </td><td>0.00108 </td><td>0.00057 </td><td>0.0016 </td><td>0.00122 </td><td>0.00095 </td><td>1.3 × 10<sup>−3</sup> </td></tr><tr><td>18 </td><td>624,6055 </td><td>6,275,418 </td><td>0.00155 </td><td>0.00069 </td><td>0.00020 </td><td>0.00092 </td><td>0.00030 </td><td>0.00034 </td><td>6.4 × 10<sup>−4</sup> </td></tr><tr><td>2 </td><td>12,415,870 </td><td>12,451,708 </td><td>0.00126 </td><td>0.00068 </td><td>9.45E-05 </td><td>0.00072 </td><td>0.00057 </td><td>0.00014 </td><td>1.2 × 10<sup>−2</sup> </td></tr><tr><td>22 </td><td>33,218,085 </td><td>33,245,588 </td><td>0.00048 </td><td>0.00025 </td><td>0.00027 </td><td>0.00171 </td><td>0.00072 </td><td>0.00038 </td><td>6.1 × 10<sup>−3</sup> </td></tr><tr><td>5 </td><td>8,435,987 </td><td>8,476,871 </td><td>0.00079 </td><td>0.00036 </td><td>0.00040 </td><td>0.00119 </td><td>0.00034 </td><td>5.28E-05 </td><td>4.5 × 10<sup>−3</sup> </td></tr><tr><td>7 </td><td>34,269,024 </td><td>34,324,708 </td><td>0.00084 </td><td>0.00047 </td><td>0.00075 </td><td>0.00129 </td><td>0.00062 </td><td>0.00018 </td><td>3.9 × 10<sup>−2</sup> </td></tr><tr><td>26 </td><td>26,340,616 </td><td>26,369,697 </td><td>0.00054 </td><td>0.00033 </td><td>0.00038 </td><td>0.00159 </td><td>0.00064 </td><td>0.00051 </td><td>6.3 × 10<sup>−3</sup> </td></tr><tr><td>10 </td><td>7,791,755 </td><td>7,830,003 </td><td>0.00148 </td><td>0.00072 </td><td>0.00043 </td><td>0.0017 </td><td>0.00083 </td><td>0.00068 </td><td>1.6 × 10<sup>−2</sup> </td></tr></tbody></table></div><div content-id="dsx048-T2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Chr </th><th rowspan="2">Start </th><th rowspan="2">End </th><th colspan="6">Recombination rate<hr /> </th><th>Comparison </th></tr><tr><th>M0 </th><th>M1 </th><th>M2 </th><th>F0 </th><th>F1 </th><th>F2 </th><th><em>P</em>-value </th></tr></thead><tbody><tr><td>21 </td><td>67,159,217 </td><td>67,195,140 </td><td>0.00196 </td><td>0.00072 </td><td>0.00011 </td><td>0.00084 </td><td>0.00038 </td><td>0.00035 </td><td>7.4 × 10<sup>−6</sup> </td></tr><tr><td>19 </td><td>54,920,324 </td><td>54,947,863 </td><td>0.00216 </td><td>0.00069 </td><td>0.00059 </td><td>0.00072 </td><td>0.00049 </td><td>0.00063 </td><td>3.4 × 10<sup>−7</sup> </td></tr><tr><td>4 </td><td>18,380,952 </td><td>18,447,308 </td><td>0.002 </td><td>0.00108 </td><td>0.00057 </td><td>0.0016 </td><td>0.00122 </td><td>0.00095 </td><td>1.3 × 10<sup>−3</sup> </td></tr><tr><td>18 </td><td>624,6055 </td><td>6,275,418 </td><td>0.00155 </td><td>0.00069 </td><td>0.00020 </td><td>0.00092 </td><td>0.00030 </td><td>0.00034 </td><td>6.4 × 10<sup>−4</sup> </td></tr><tr><td>2 </td><td>12,415,870 </td><td>12,451,708 </td><td>0.00126 </td><td>0.00068 </td><td>9.45E-05 </td><td>0.00072 </td><td>0.00057 </td><td>0.00014 </td><td>1.2 × 10<sup>−2</sup> </td></tr><tr><td>22 </td><td>33,218,085 </td><td>33,245,588 </td><td>0.00048 </td><td>0.00025 </td><td>0.00027 </td><td>0.00171 </td><td>0.00072 </td><td>0.00038 </td><td>6.1 × 10<sup>−3</sup> </td></tr><tr><td>5 </td><td>8,435,987 </td><td>8,476,871 </td><td>0.00079 </td><td>0.00036 </td><td>0.00040 </td><td>0.00119 </td><td>0.00034 </td><td>5.28E-05 </td><td>4.5 × 10<sup>−3</sup> </td></tr><tr><td>7 </td><td>34,269,024 </td><td>34,324,708 </td><td>0.00084 </td><td>0.00047 </td><td>0.00075 </td><td>0.00129 </td><td>0.00062 </td><td>0.00018 </td><td>3.9 × 10<sup>−2</sup> </td></tr><tr><td>26 </td><td>26,340,616 </td><td>26,369,697 </td><td>0.00054 </td><td>0.00033 </td><td>0.00038 </td><td>0.00159 </td><td>0.00064 </td><td>0.00051 </td><td>6.3 × 10<sup>−3</sup> </td></tr><tr><td>10 </td><td>7,791,755 </td><td>7,830,003 </td><td>0.00148 </td><td>0.00072 </td><td>0.00043 </td><td>0.0017 </td><td>0.00083 </td><td>0.00068 </td><td>1.6 × 10<sup>−2</sup> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn2"><div class="fn"><p>Start, End = physical position of hotspot regions on chromosome. M0, M1 and M2 = recombination rates of males with non-allele 5, allele 5 heterozygote and allele 5 homozygote genotypes, respectively. F0, F1 and F2 = female recombination rates. <em>P</em>-value = allele 5 heterozygote versus non-allele 5 genotype. Top 5 rows are from male results and bottom 5 rows are female hotspots.</p></div></div></div></div></p>                            <h3 scrollto-destination=115733748 id="115733748" class="section-title" >3.5. Dominant effect of allele 5 on recombination in two sexes</h3>
<p>Using both sharing of recombination hotspot region and correlation in recombination rate, we found animals carrying one copy of allele 5 were more similar to animals with two copies of allele 5 than to animals carrying non-allele 5. To evaluate the sharing of recombination hotspot region, we tentatively defined hotspot regions as the SNP intervals with recombination rate 2.5 S.D greater than the genome-wide average. For Holstein males, a total of 1361, 1353 and 1365 hotspot regions were identified for allele 5 homozygote, allele 5 heterozygote and non-allele 5 animals, respectively. Consistent with observations in human studies,<a reveal-id="dsx048-B27" data-open="dsx048-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> each <em>PRDM</em>9 genotype had its unique hotspot regions (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx048/1/dsx048_supp.zip?Expires=2147483647&Signature=cB7LewD99XIfXtilEHTinjygYWutIBSPgTSSy6xH4XO0UhC59pZBSl92WUktO2BqH2Hl3-GTWAYw6LtDm7lW3YE-dlS2MMAJV4XEQXCo7TifcPEQEtn9LH0s2AbatwLORuKdw7mxT~nLTak89qTAjVAIM7KkWkbB~m9AEtyVLoWKEPXNGaKxuHqdS616H4n9373orlSfnWCzhmqoBYwtgyfQLPB02zulnR7CrM97FtAfZQI7dtJtAAxSr2-Z3~J8WnVzzJL1ozAZMGjaHP3tY2mkF3halnzqJJ4cFtpVZWGJH7catGGXeTJ8UIeICXKqNKHC8OxySuj2kP00XSGRuA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S10</a></span>). Allele 5 homozygote, allele 5 heterozygote and non-allele 5 genotypes each had 887, 737 and 1, 032 unique hotspot regions, with only 97 hotspot regions shared by all three genotypes. Excluding the 97 hotspot regions that were shared by all three genotypes, allele 5 and non-allele 5 had only 47 hotspot regions in common. Consistent with the patterns observed in the global recombination maps, allele 5 heterozygote shared more hotspot regions with allele 5 homozygote than with non-allele 5 genotype (330 vs 189), indicating an observed dominant effect of allele 5. A similar pattern was found in the Holstein cows (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx048/1/dsx048_supp.zip?Expires=2147483647&Signature=cB7LewD99XIfXtilEHTinjygYWutIBSPgTSSy6xH4XO0UhC59pZBSl92WUktO2BqH2Hl3-GTWAYw6LtDm7lW3YE-dlS2MMAJV4XEQXCo7TifcPEQEtn9LH0s2AbatwLORuKdw7mxT~nLTak89qTAjVAIM7KkWkbB~m9AEtyVLoWKEPXNGaKxuHqdS616H4n9373orlSfnWCzhmqoBYwtgyfQLPB02zulnR7CrM97FtAfZQI7dtJtAAxSr2-Z3~J8WnVzzJL1ozAZMGjaHP3tY2mkF3halnzqJJ4cFtpVZWGJH7catGGXeTJ8UIeICXKqNKHC8OxySuj2kP00XSGRuA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S10</a></span>). Allele 5 homozygote, allele 5 heterozygote and non-allele 5 genotypes each had 801, 583 and 855 allele-specific hotspot regions, with 101 hotspot regions shared across all three genotypes. Excluding the common hotspots, allele 5 homozygote and allele 5 heterozygote genotypes shared 314 hotspot regions, but allele 5 heterozygote and non-allele 5 genotypes had only 260 hotspot regions in common. Using a correlation analysis of recombination rates across three <em>PRDM</em>9 genotypes and two sexes, we found higher correlations between allele 5 homozygote and heterozygote than between allele 5 heterozygote and non-allele 5 in both males and females (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx048/1/dsx048_supp.zip?Expires=2147483647&Signature=cB7LewD99XIfXtilEHTinjygYWutIBSPgTSSy6xH4XO0UhC59pZBSl92WUktO2BqH2Hl3-GTWAYw6LtDm7lW3YE-dlS2MMAJV4XEQXCo7TifcPEQEtn9LH0s2AbatwLORuKdw7mxT~nLTak89qTAjVAIM7KkWkbB~m9AEtyVLoWKEPXNGaKxuHqdS616H4n9373orlSfnWCzhmqoBYwtgyfQLPB02zulnR7CrM97FtAfZQI7dtJtAAxSr2-Z3~J8WnVzzJL1ozAZMGjaHP3tY2mkF3halnzqJJ4cFtpVZWGJH7catGGXeTJ8UIeICXKqNKHC8OxySuj2kP00XSGRuA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S11</a></span>), confirming the observed dominant effect of allele 5 to other <em>PRDM</em>9 alleles.</p>                            <h3 scrollto-destination=115733750 id="115733750" class="section-title" >3.6. Enrichment of PRDM9 binding motifs in recombination hotspot regions</h3>
<p>Using a computational approach designed for ZnF proteins,<a reveal-id="dsx048-B45" data-open="dsx048-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a> we predicted the binding motifs of the <em>PRDM</em>9 alleles (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx048/1/dsx048_supp.zip?Expires=2147483647&Signature=cB7LewD99XIfXtilEHTinjygYWutIBSPgTSSy6xH4XO0UhC59pZBSl92WUktO2BqH2Hl3-GTWAYw6LtDm7lW3YE-dlS2MMAJV4XEQXCo7TifcPEQEtn9LH0s2AbatwLORuKdw7mxT~nLTak89qTAjVAIM7KkWkbB~m9AEtyVLoWKEPXNGaKxuHqdS616H4n9373orlSfnWCzhmqoBYwtgyfQLPB02zulnR7CrM97FtAfZQI7dtJtAAxSr2-Z3~J8WnVzzJL1ozAZMGjaHP3tY2mkF3halnzqJJ4cFtpVZWGJH7catGGXeTJ8UIeICXKqNKHC8OxySuj2kP00XSGRuA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6</a></span>). Since the predicted motifs are almost the same for alleles 1–4, we evaluated the enrichment of PRDM9 binding motifs in recombination hotspot regions by focusing on allele 1 and allele 5. Based on the position weight matrices of the predicted motifs, we extracted the degenerated 17-bp motifs ‘ANNANNANNANNANGGC’ and ‘CGNNANNAGCANNANNA’ for allele 1 and allele 5, respectively. Here, we used allele 1 to represent non-allele 5 alleles because allele 1 was the most frequent and there was little difference in binding motif between allele 1 and other non-allele 5 alleles. In the whole bovine reference genome (UMD 3.1.1), allele 5 motif is 1.17 times more prevalent than allele 1 motif. Compared to this genome-wide baseline ratio of 1.17, non-allele 5 hotspot regions had a decreased ratio of 1.14, and allele 5 hotspot regions had an increased ratio of 1.20 in Holstein males (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx048/1/dsx048_supp.zip?Expires=2147483647&Signature=cB7LewD99XIfXtilEHTinjygYWutIBSPgTSSy6xH4XO0UhC59pZBSl92WUktO2BqH2Hl3-GTWAYw6LtDm7lW3YE-dlS2MMAJV4XEQXCo7TifcPEQEtn9LH0s2AbatwLORuKdw7mxT~nLTak89qTAjVAIM7KkWkbB~m9AEtyVLoWKEPXNGaKxuHqdS616H4n9373orlSfnWCzhmqoBYwtgyfQLPB02zulnR7CrM97FtAfZQI7dtJtAAxSr2-Z3~J8WnVzzJL1ozAZMGjaHP3tY2mkF3halnzqJJ4cFtpVZWGJH7catGGXeTJ8UIeICXKqNKHC8OxySuj2kP00XSGRuA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S7</a></span>). When measured in various subsets of hotspot regions, this ratio showed a consistent trend, 1.14 in non-allele 5 and allele 5 heterozygous shared hotspot regions, 1.19 in allele 5 homozygous and allele 5 heterozygous shared hotspot regions, 1.14 in non-allele 5 specific hotspot regions, and 1.21 in allele 5 specific hotspot regions. Similar enrichment trends were observed in females (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx048/1/dsx048_supp.zip?Expires=2147483647&Signature=cB7LewD99XIfXtilEHTinjygYWutIBSPgTSSy6xH4XO0UhC59pZBSl92WUktO2BqH2Hl3-GTWAYw6LtDm7lW3YE-dlS2MMAJV4XEQXCo7TifcPEQEtn9LH0s2AbatwLORuKdw7mxT~nLTak89qTAjVAIM7KkWkbB~m9AEtyVLoWKEPXNGaKxuHqdS616H4n9373orlSfnWCzhmqoBYwtgyfQLPB02zulnR7CrM97FtAfZQI7dtJtAAxSr2-Z3~J8WnVzzJL1ozAZMGjaHP3tY2mkF3halnzqJJ4cFtpVZWGJH7catGGXeTJ8UIeICXKqNKHC8OxySuj2kP00XSGRuA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S7</a></span>). Collectively, although we used computationally predicted motifs and our hotspot regions were larger than typical recombination hotspot regions, we did observe an expected trend that allele 5 motif was enriched in allele 5 hotspot regions and allele 1 motif enriched in non-allele 5 hotspot regions.</p>                            <h3 scrollto-destination=115733752 id="115733752" class="section-title" >3.7. Comparison of recombination maps from single sperm-typing and pedigree data</h3>
<p>The large pedigree and widely used artificial insemination in cattle provided an opportunity to compare recombination patterns between sperms and live-born offspring. To check the consistency of the recombination patterns inferred from pedigree data, we characterized the recombination patterns of a single Holstein bull using both pedigree and single sperm-typing. The comparison involved 56 high-quality single sperms genotyped on BovineHD (770 K) SNP chip and 556 live-born offspring genotyped with various chips of more than 50 K SNPs (see Materials and methods). Using the same approach of pedigree analysis, we calculated recombination rates in single sperm data between adjacent SNPs on a 50 K SNP chip, so the two recombination maps could be compared with the same number of SNPs. Although sperm data have more SNPs than pedigree data to begin with, the sperm-based recombination map supported the differences between different <em>PRDM9</em> alleles identified from pedigree data (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx048/1/dsx048_supp.zip?Expires=2147483647&Signature=cB7LewD99XIfXtilEHTinjygYWutIBSPgTSSy6xH4XO0UhC59pZBSl92WUktO2BqH2Hl3-GTWAYw6LtDm7lW3YE-dlS2MMAJV4XEQXCo7TifcPEQEtn9LH0s2AbatwLORuKdw7mxT~nLTak89qTAjVAIM7KkWkbB~m9AEtyVLoWKEPXNGaKxuHqdS616H4n9373orlSfnWCzhmqoBYwtgyfQLPB02zulnR7CrM97FtAfZQI7dtJtAAxSr2-Z3~J8WnVzzJL1ozAZMGjaHP3tY2mkF3halnzqJJ4cFtpVZWGJH7catGGXeTJ8UIeICXKqNKHC8OxySuj2kP00XSGRuA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S12</a></span>). Overall, recombination rates from the single sperm data (allele1/allele2) showed the highest correlation with the pedigree-based rates from the same bull, and this correlation continuously decreased with recombination maps from animals carrying zero, one and two copies of allele 5 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx048/1/dsx048_supp.zip?Expires=2147483647&Signature=cB7LewD99XIfXtilEHTinjygYWutIBSPgTSSy6xH4XO0UhC59pZBSl92WUktO2BqH2Hl3-GTWAYw6LtDm7lW3YE-dlS2MMAJV4XEQXCo7TifcPEQEtn9LH0s2AbatwLORuKdw7mxT~nLTak89qTAjVAIM7KkWkbB~m9AEtyVLoWKEPXNGaKxuHqdS616H4n9373orlSfnWCzhmqoBYwtgyfQLPB02zulnR7CrM97FtAfZQI7dtJtAAxSr2-Z3~J8WnVzzJL1ozAZMGjaHP3tY2mkF3halnzqJJ4cFtpVZWGJH7catGGXeTJ8UIeICXKqNKHC8OxySuj2kP00XSGRuA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S12a</a></span>). As expected, the sperm recombination rates are more similar to male recombination rates than that of females in all groups. In addition, we observed the same trend using the sharing of top 5% recombination intervals between sperm and pedigree data (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx048/1/dsx048_supp.zip?Expires=2147483647&Signature=cB7LewD99XIfXtilEHTinjygYWutIBSPgTSSy6xH4XO0UhC59pZBSl92WUktO2BqH2Hl3-GTWAYw6LtDm7lW3YE-dlS2MMAJV4XEQXCo7TifcPEQEtn9LH0s2AbatwLORuKdw7mxT~nLTak89qTAjVAIM7KkWkbB~m9AEtyVLoWKEPXNGaKxuHqdS616H4n9373orlSfnWCzhmqoBYwtgyfQLPB02zulnR7CrM97FtAfZQI7dtJtAAxSr2-Z3~J8WnVzzJL1ozAZMGjaHP3tY2mkF3halnzqJJ4cFtpVZWGJH7catGGXeTJ8UIeICXKqNKHC8OxySuj2kP00XSGRuA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S12b</a></span>).</p><p>By comparing recombination maps from sperms and pedigree of the same bull, we found the same number but slightly different preferred locations of recombination between sperms and live-born offspring. At the chromosome level, the number of crossovers from pedigree and sperm data were correlated with <em>r</em> = 0.77 across 29 autosomes (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx048/1/dsx048_supp.zip?Expires=2147483647&Signature=cB7LewD99XIfXtilEHTinjygYWutIBSPgTSSy6xH4XO0UhC59pZBSl92WUktO2BqH2Hl3-GTWAYw6LtDm7lW3YE-dlS2MMAJV4XEQXCo7TifcPEQEtn9LH0s2AbatwLORuKdw7mxT~nLTak89qTAjVAIM7KkWkbB~m9AEtyVLoWKEPXNGaKxuHqdS616H4n9373orlSfnWCzhmqoBYwtgyfQLPB02zulnR7CrM97FtAfZQI7dtJtAAxSr2-Z3~J8WnVzzJL1ozAZMGjaHP3tY2mkF3halnzqJJ4cFtpVZWGJH7catGGXeTJ8UIeICXKqNKHC8OxySuj2kP00XSGRuA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S13</a></span>). Since the sperm data had a higher density of SNPs, we manually decreased the SNP number of sperm data to levels comparable to pedigree data. When using similar numbers of SNPs in both samples (10 K in sperms and 7 K–50 K in pedigree), we saw no difference in the total number of recombination events: 21.91 ± 0.5 SE (±3.8 SD) in sperms and 21.65 ± 0.12 SE (±3.8 SD) in pedigree data (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx048/1/dsx048_supp.zip?Expires=2147483647&Signature=cB7LewD99XIfXtilEHTinjygYWutIBSPgTSSy6xH4XO0UhC59pZBSl92WUktO2BqH2Hl3-GTWAYw6LtDm7lW3YE-dlS2MMAJV4XEQXCo7TifcPEQEtn9LH0s2AbatwLORuKdw7mxT~nLTak89qTAjVAIM7KkWkbB~m9AEtyVLoWKEPXNGaKxuHqdS616H4n9373orlSfnWCzhmqoBYwtgyfQLPB02zulnR7CrM97FtAfZQI7dtJtAAxSr2-Z3~J8WnVzzJL1ozAZMGjaHP3tY2mkF3halnzqJJ4cFtpVZWGJH7catGGXeTJ8UIeICXKqNKHC8OxySuj2kP00XSGRuA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S14</a></span>). To compare the patterns of recombination in pedigree and sperm data, we generated global recombination maps by collapsing all 29 autosomes into a standardized chromosome (<a reveal-id="dsx048-F4" data-open="dsx048-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>). Overall, the pedigree and sperm recombination maps showed a very similar pattern except near the two ends of chromosomes (standardized locations 10%–20% and &gt;80% on a chromosome). At the beginning of a chromosome (10%–20%), the pedigree-based map exhibited a higher recombination rate than the sperm-based map. To the end of a chromosome (&gt;80%), the recombination rate of sperms reached a peak near the 85% of a chromosome, whereas the pedigree-based recombination rate continuously increased to the end of chromosomes. To avoid potential biases from SNP coverage in sperms, we checked the distribution of SNP numbers along the genome and found no clear deficiency of SNPs to the two ends of chromosomes (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx048/1/dsx048_supp.zip?Expires=2147483647&Signature=cB7LewD99XIfXtilEHTinjygYWutIBSPgTSSy6xH4XO0UhC59pZBSl92WUktO2BqH2Hl3-GTWAYw6LtDm7lW3YE-dlS2MMAJV4XEQXCo7TifcPEQEtn9LH0s2AbatwLORuKdw7mxT~nLTak89qTAjVAIM7KkWkbB~m9AEtyVLoWKEPXNGaKxuHqdS616H4n9373orlSfnWCzhmqoBYwtgyfQLPB02zulnR7CrM97FtAfZQI7dtJtAAxSr2-Z3~J8WnVzzJL1ozAZMGjaHP3tY2mkF3halnzqJJ4cFtpVZWGJH7catGGXeTJ8UIeICXKqNKHC8OxySuj2kP00XSGRuA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S15</a></span>). However, a χ<sup>2</sup> test found no significant differences (<em>P</em> &gt; 0.05) at both locations, possibly due to the small number of sperms tested. </p>                        <a id="115733755" scrollto-destination="115733755"></a>
<div id="dsx048-F4" data-id="dsx048-F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx048/1/m_dsx048f4.png?Expires=2147483647&Signature=iFXtIYysUaWrkFTTWHgrHn-wHEwjGrmTCjQ3N7j1drkeP5EeLPDySOcSg8HjcpThWjGZm2DV6CUdttARFkAGY1swtzNn7byBlV3sWpIQvVNKBTtrCJWkMJbLvJUC0SD4twXngvzHWYJv2Q-9qHqAPHNuQ22~dIroz58i3P-oOwG7oaucWg0Et~XLkGKcP7IwC-Py6NoKdkIAyyGXHujEqypWMddX0sOtVuSblrTrxr0Lx0PZOtxwS2nVvvcco8wcNXe20HuFRF3nfbtYhT0sOPkgWtcNKyBhId17CGkk9TPPODaXgNMCV27gN3GheDSuHxIjC1pU9TypLmb6KzPSXw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Spline-smoother plot of recombination rate along the chromosome from single sperm data and pedigree data. The relative physical position on a chromosome is used, where zero corresponding to the beginning of a chromosome and one the end. The smooth spline model was fitted across all of the 29 chromosomes. Due to the differences in SNP density, the single sperm and pedigree recombination rates were plotted in different scales." path-from-xml="dsx048f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/115733755/dsx048f4.tif" path-from-xml="dsx048f4.tif" target="_blank">View large</a><a section="115733755" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx048/1/dsx048f4.png?Expires=2147483647&Signature=H9nuMnr8w4l11IOetRFM6N0ARVUBJ-qlfW0lF6Da6P1ZC2f750xYNSEQeebLbYPq92aIzJ9EcUJ~XyzD9jWYXP5KgSMeIbNMhjd15Jaki~~lSYrD6PO3LkH~9nFzbZUht6QHuEBk-Oeg41KPsm9krNvB-iKq~VuSlqWm4rXeDQikpvvhQRIeUztJi5KQPV9TfxDWy~v7HSwktGBrZWrB5INJQcaYSgXvbNfseI0fzNqY1~7TlhKJmqAcdUXPOhxfXLx~20Jbmg-yQHDgMC04BiSWLDxoRIHqnqCY5OZ64dxo~K1vkHQd5OVbfaqHBU77vTHXEa3cs-ruXjDGoCp5oA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=115733755&ar=4665695&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx048f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p><strong>Spline-smoother plot of recombination rate along the chromosome from single sperm data and pedigree data.</strong> The relative physical position on a chromosome is used, where zero corresponding to the beginning of a chromosome and one the end. The smooth spline model was fitted across all of the 29 chromosomes. Due to the differences in SNP density, the single sperm and pedigree recombination rates were plotted in different scales.</p></div></div><div content-id="dsx048-F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx048/1/m_dsx048f4.png?Expires=2147483647&Signature=iFXtIYysUaWrkFTTWHgrHn-wHEwjGrmTCjQ3N7j1drkeP5EeLPDySOcSg8HjcpThWjGZm2DV6CUdttARFkAGY1swtzNn7byBlV3sWpIQvVNKBTtrCJWkMJbLvJUC0SD4twXngvzHWYJv2Q-9qHqAPHNuQ22~dIroz58i3P-oOwG7oaucWg0Et~XLkGKcP7IwC-Py6NoKdkIAyyGXHujEqypWMddX0sOtVuSblrTrxr0Lx0PZOtxwS2nVvvcco8wcNXe20HuFRF3nfbtYhT0sOPkgWtcNKyBhId17CGkk9TPPODaXgNMCV27gN3GheDSuHxIjC1pU9TypLmb6KzPSXw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Spline-smoother plot of recombination rate along the chromosome from single sperm data and pedigree data. The relative physical position on a chromosome is used, where zero corresponding to the beginning of a chromosome and one the end. The smooth spline model was fitted across all of the 29 chromosomes. Due to the differences in SNP density, the single sperm and pedigree recombination rates were plotted in different scales." path-from-xml="dsx048f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/115733755/dsx048f4.tif" path-from-xml="dsx048f4.tif" target="_blank">View large</a><a section="115733755" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx048/1/dsx048f4.png?Expires=2147483647&Signature=H9nuMnr8w4l11IOetRFM6N0ARVUBJ-qlfW0lF6Da6P1ZC2f750xYNSEQeebLbYPq92aIzJ9EcUJ~XyzD9jWYXP5KgSMeIbNMhjd15Jaki~~lSYrD6PO3LkH~9nFzbZUht6QHuEBk-Oeg41KPsm9krNvB-iKq~VuSlqWm4rXeDQikpvvhQRIeUztJi5KQPV9TfxDWy~v7HSwktGBrZWrB5INJQcaYSgXvbNfseI0fzNqY1~7TlhKJmqAcdUXPOhxfXLx~20Jbmg-yQHDgMC04BiSWLDxoRIHqnqCY5OZ64dxo~K1vkHQd5OVbfaqHBU77vTHXEa3cs-ruXjDGoCp5oA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=115733755&ar=4665695&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx048f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p><strong>Spline-smoother plot of recombination rate along the chromosome from single sperm data and pedigree data.</strong> The relative physical position on a chromosome is used, where zero corresponding to the beginning of a chromosome and one the end. The smooth spline model was fitted across all of the 29 chromosomes. Due to the differences in SNP density, the single sperm and pedigree recombination rates were plotted in different scales.</p></div></div>                            <h2 scrollto-destination=115733756 id="115733756" class="section-title" >4. Discussion</h2>
<p>The purpose of this study was to characterize <em>PRDM</em>9 allele-specific localizations of meiotic recombination in widely commercialized cattle breeds.<a reveal-id="dsx048-B20" data-open="dsx048-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a><sup>,</sup><a reveal-id="dsx048-B21" data-open="dsx048-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> Meiotic recombination is known to enhance genetic and phenotypic variations in sexually reproducing organisms, and <em>PRDM</em>9 regulates the location of double-stranded breaks and thus recombination events in most of the placental mammals.<a reveal-id="dsx048-B6" data-open="dsx048-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a><sup>,</sup><a reveal-id="dsx048-B7" data-open="dsx048-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> Recent studies in humans showed that the <em>PRDM</em>9 alleles have dramatic influences on the localization and turnover of the recombination hotspot via increased sequence diversity;<a reveal-id="dsx048-B27" data-open="dsx048-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a><sup>,</sup><a reveal-id="dsx048-B46" data-open="dsx048-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a> however, it is unclear whether these patterns can also be observed in cattle, which has unique demographic histories and more importantly, persistently succumbed to intense selective pressures in the past 10,000 years after domestication. In addition, multiple recent GWAS studies have identified candidate genes associated with recombination features in cattle, including <em>PRDM</em>9.<a reveal-id="dsx048-B31" data-open="dsx048-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a><sup>,</sup><a reveal-id="dsx048-B34" data-open="dsx048-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a><sup>,</sup><a reveal-id="dsx048-B47" data-open="dsx048-B47" class="link link-ref link-reveal xref-bibr"><sup>47</sup></a> The present study not only identified the <em>PRDM</em>9 allele-specific variations including the amino acid substitutions at three functionally important sites (i.e., −1, 3, and 6) of the ZnFs but also provided strong evidence of allele-specific localizations of recombination events with most pronounced differences at the two ends of chromosomes in cattle. Importantly, these evidence suggested a specific allele (allele 5) as the causal variant for the <em>PRDM</em>9 association with cattle recombination.</p><p>While four out of five <em>PRDM</em>9 alleles were shared between the two cattle breeds and showed similar global recombination patterns, one allele (allele 5) that was unique to Holstein exhibited the highest recombination rates at two recombination peak regions. Interestingly, this pattern is correlated with the observed amino acid substitutions at the functional sites of the <em>PRDM</em>9 ZnFs. In addition, previous studies have reported strong associations between the allele-specific localization and turnover of recombination hotspot regions and the polymorphisms of <em>PRDM</em>9 in humans.<a reveal-id="dsx048-B8" data-open="dsx048-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a><sup>,</sup><a reveal-id="dsx048-B27" data-open="dsx048-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a><sup>,</sup><a reveal-id="dsx048-B46" data-open="dsx048-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a> Although the diversity levels in <em>PRDM</em>9 alleles and number of ZnF repeats were relatively lower in cattle than primates and rodents,<a reveal-id="dsx048-B10" data-open="dsx048-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a><sup>,</sup><a reveal-id="dsx048-B11" data-open="dsx048-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> alleles with unique amino acid substitutions appeared to have dramatic differences in the binding-specificity and the distribution of recombination events. For instance, as observed in this study, animals that carry allele 5 have unique recombination hotspot regions that are distinguishable from hotspot regions modulated by non-allele 5, consistent with studies in humans.<a reveal-id="dsx048-B27" data-open="dsx048-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a><sup>,</sup><a reveal-id="dsx048-B46" data-open="dsx048-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a> As the PRDM9 ZnFs were predicted to bind sequence motifs that are enriched in recombination hotspot,<a reveal-id="dsx048-B48" data-open="dsx048-B48" class="link link-ref link-reveal xref-bibr"><sup>48</sup></a><sup>,</sup><a reveal-id="dsx048-B49" data-open="dsx048-B49" class="link link-ref link-reveal xref-bibr"><sup>49</sup></a> one might speculate uneven distributions of binding affinities of the <em>PRDM</em>9 alleles across the cattle genome. Computationally, we predicted two17-bp motifs ‘CGNNANNAGNANNANNA’ and ‘ANNANNANNANNANGGC’ of the most common allele (i.e. allele 1) and allele 5, respectively. We also reported consistent enrichment patterns of these binding motifs in corresponding recombination hotspot regions. Collectively, these variations in <em>PRDM</em>9 ZnFs are associated with the location and intensity of recombination in cattle. However, due to the limited resolution of SNP densities, the strength of enrichment is relatively low and the predicted binding motifs need to be further validated in future experiments.</p><p>Given the fact that sampling and genotype errors may potentially bias the pedigree-based results, we further confirmed our findings using a single-sperm genomics approach.<a reveal-id="dsx048-B36" data-open="dsx048-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a><sup>,</sup><a reveal-id="dsx048-B38" data-open="dsx048-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> Although the genome-wide recombination rates from these two approaches were consistent, we found some differences, especially at the two locations of recombination peaks, between the pedigree and sperm-based recombination maps. These findings were in agreement with the previous studies in humans, which showed that although the recombination maps from the pedigree and sperm-typing methods are largely consistent, considerable differences were observed at a higher resolution. Since the sperms used in the present study were active and viable, the differences in fitness before fertilization are small between the sperm samples and the sperms that ended up in live-born offspring. However, different fitness between sperms and live-born offspring may still lie in the selection process between sperm-egg fertilization and embryo development till birth. Although it is intuitively unclear as to what factors drive such differences, based on our results and previous reports,<a reveal-id="dsx048-B40" data-open="dsx048-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> we postulate the differences in sperm fitness during and after fertilization to be one of the plausible explanations.</p>                            <h2 scrollto-destination=115733760 id="115733760" class="section-title" >5. Conclusions</h2>
<p>Taken together, in this study, we characterized the <em>PRDM</em>9 sequence diversity in multiple cattle breeds and generated <em>PRDM</em>9 allele-specific global and local recombination maps in individual bulls. The large cattle pedigree provided us the power to show differences in genome-wide recombination maps of <em>PRDM</em>9 alleles. For the first time, we showed the impact of <em>PRDM</em>9 on the global distribution of recombination on the genome, particularly in the telomere and centromere regions. Using genome-wide single sperm genotyping, we validated the quality of pedigree-based recombination maps. Collectively, these results will provide new insights into the regulatory functions of <em>PRDM</em>9 on meiotic recombination, which further contribute to our understanding of genome evolution in mammals.</p>                            <h2 scrollto-destination=115733762 id="115733762" class="section-title" >Declarations</h2>
<p><strong>Ethics approval and consent to participate</strong></p><p>The need for ethics approval was waived as the current study didn’t involve whole animals.</p>                            <h2 scrollto-destination=115733765 id="115733765" class="section-title" >Consent for publication</h2>
<p>Not applicable.</p>                            <h2 scrollto-destination=115733767 id="115733767" class="section-title" >Availability of data and material</h2>
<p>The data that support the results of this research are available within the article and its <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx048/1/dsx048_supp.zip?Expires=2147483647&Signature=cB7LewD99XIfXtilEHTinjygYWutIBSPgTSSy6xH4XO0UhC59pZBSl92WUktO2BqH2Hl3-GTWAYw6LtDm7lW3YE-dlS2MMAJV4XEQXCo7TifcPEQEtn9LH0s2AbatwLORuKdw7mxT~nLTak89qTAjVAIM7KkWkbB~m9AEtyVLoWKEPXNGaKxuHqdS616H4n9373orlSfnWCzhmqoBYwtgyfQLPB02zulnR7CrM97FtAfZQI7dtJtAAxSr2-Z3~J8WnVzzJL1ozAZMGjaHP3tY2mkF3halnzqJJ4cFtpVZWGJH7catGGXeTJ8UIeICXKqNKHC8OxySuj2kP00XSGRuA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Information</a></span> files. All other sequence data can be tracked in a Supplementary file (Supp_Data_2). The single sperm microarray data are submitted to GEO under the accession number GSE99620.</p>                            <h2 scrollto-destination=115733769 id="115733769" class="section-title" >Authors' contributions</h2>
<p>GL and LM conceived the study. YZ, BS, JJ and AP analyzed and interpreted data. YZ, BS and LM wrote the manuscript. KP, BPT, AO, CGS, HC and JBC contributed tools and materials. All authors read and approved the final manuscript.</p>                            <h2 scrollto-destination=115733771 id="115733771" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank two anonymous reviewers for the constructive comments that have improved the manuscript. We thank Reuben Anderson and Alexandre Dimitriv for technical assistance. The Council on Dairy Cattle Breeding (Bowie, MD) and the Cooperative Dairy DNA Repository (Columbia, MO) are also acknowledged for providing data used in this study.</p>                            <h3 scrollto-destination=115733773 id="115733773" class="section-title" >Accession number</h3>
<p>GSE99620</p>                            <h3 scrollto-destination=115733775 id="115733775" class="section-title" >Conflict of interest</h3>
<p>AO and CS are employees of Select Sires, Inc. All other authors declare that they have no competing interests.</p>                            <h3 scrollto-destination=115733777 id="115733777" class="section-title" >Funding</h3>
<p>This work was supported in part by AFRI grant Nos. 2011-67015-30183, 2013-67015-20951 and 2016-67015-24886 from the USDA NIFA and grant No. US-4997-17 from the US-Israel Binational Agricultural Research and Development Fund. JBC and GEL were also supported by appropriated projects 1265-31000-096-00, ‘Improving Genetic Predictions in Dairy Animals Using Phenotypic and Genomic Information’, and 8042-31000-104-00, ‘Enhancing Genetic Merit of Ruminants Through Genome Selection and Analysis’, of the Agricultural Research Service of the United States Department of Agriculture, respectively. Mention of trade names or commercial products in this article is solely for the purpose of providing specific information and does not imply recommendation or endorsement by the US Department of Agriculture. The USDA is an equal opportunity provider and employer. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p>                            <h3 scrollto-destination=115733779 id="115733779" class="section-title" >Supplementary data</h3>
<p><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx048/1/dsx048_supp.zip?Expires=2147483647&Signature=cB7LewD99XIfXtilEHTinjygYWutIBSPgTSSy6xH4XO0UhC59pZBSl92WUktO2BqH2Hl3-GTWAYw6LtDm7lW3YE-dlS2MMAJV4XEQXCo7TifcPEQEtn9LH0s2AbatwLORuKdw7mxT~nLTak89qTAjVAIM7KkWkbB~m9AEtyVLoWKEPXNGaKxuHqdS616H4n9373orlSfnWCzhmqoBYwtgyfQLPB02zulnR7CrM97FtAfZQI7dtJtAAxSr2-Z3~J8WnVzzJL1ozAZMGjaHP3tY2mkF3halnzqJJ4cFtpVZWGJH7catGGXeTJ8UIeICXKqNKHC8OxySuj2kP00XSGRuA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary data</a></span> are available at <em>DNARES</em> online.</p>                            <h2 scrollto-destination=115733781 id="115733781" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsx048-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx048-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Barton</div> <div class="given-names">N.H.</div></div>, <div class="name string-name"><div class="surname">Charlesworth</div> <div class="given-names">B.</div></div></div> <div class="year">1998</div>, 
<div class="article-title">Why sex and recombination?,</div> <div class="source">Science</div> , <div class="volume">281</div>, <div class="fpage">1986</div>–<div class="lpage">90</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Why%20sex%20and%20recombination%3F%2C&amp;author=N.H.%20Barton&amp;author=B.%20Charlesworth&amp;publication_year=1998&amp;journal=Science&amp;volume=281&amp;pages=1986-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.281.5385.1986" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.281.5385.1986" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.281.5385.1986"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9748151" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx048-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx048-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Stumpf</div> <div class="given-names">M.P.</div></div>, <div class="name string-name"><div class="surname">McVean</div> <div class="given-names">G.A.</div></div></div> <div class="year">2003</div>, 
<div class="article-title">Estimating recombination rates from population-genetic data</div>, <div class="source">Nat. Rev. Genet</div> ., <div class="volume">4</div>, <div class="fpage">959</div>–<div class="lpage">68</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Estimating%20recombination%20rates%20from%20population-genetic%20data&amp;author=M.P.%20Stumpf&amp;author=G.A.%20McVean&amp;publication_year=2003&amp;journal=Nat.%20Rev.%20Genet&amp;volume=4&amp;pages=959-68" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg1227" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg1227" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg1227"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14631356" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx048-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx048-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kauppi</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Jeffreys</div> <div class="given-names">A.J.</div></div>, <div class="name string-name"><div class="surname">Keeney</div> <div class="given-names">S.</div></div></div> <div class="year">2004</div>, 
<div class="article-title">Where the crossovers are: recombination distributions in mammals</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="volume">5</div>, <div class="fpage">413</div>–<div class="lpage">24</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Where%20the%20crossovers%20are%3A%20recombination%20distributions%20in%20mammals&amp;author=L.%20Kauppi&amp;author=A.J.%20Jeffreys&amp;author=S.%20Keeney&amp;publication_year=2004&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=5&amp;pages=413-24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg1346" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg1346" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg1346"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15153994" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx048-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx048-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Coop</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Przeworski</div> <div class="given-names">M.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">An evolutionary view of human recombination</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="volume">8</div>, <div class="fpage">23</div>–<div class="lpage">34</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20evolutionary%20view%20of%20human%20recombination&amp;author=G.%20Coop&amp;author=M.%20Przeworski&amp;publication_year=2007&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=8&amp;pages=23-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg1947" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg1947" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg1947"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17146469" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx048-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx048-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Paigen</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Petkov</div> <div class="given-names">P.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Mammalian recombination hot spots: properties, control and evolution</div>, <div class="source">Nat. Rev. Genet</div> ., <div class="volume">11</div>, <div class="fpage">221</div>–<div class="lpage">33</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mammalian%20recombination%20hot%20spots%3A%20properties%2C%20control%20and%20evolution&amp;author=K.%20Paigen&amp;author=P.%20Petkov&amp;publication_year=2010&amp;journal=Nat.%20Rev.%20Genet&amp;volume=11&amp;pages=221-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg2712" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg2712" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg2712"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20168297" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx048-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx048-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Parvanov</div> <div class="given-names">E.D.</div></div>, <div class="name string-name"><div class="surname">Petkov</div> <div class="given-names">P.M.</div></div>, <div class="name string-name"><div class="surname">Paigen</div> <div class="given-names">K.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Prdm9 controls activation of mammalian recombination hotspots</div>, <div class="source">Science</div> , <div class="volume">327</div>, <div class="fpage">835</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Prdm9%20controls%20activation%20of%20mammalian%20recombination%20hotspots&amp;author=E.D.%20Parvanov&amp;author=P.M.%20Petkov&amp;author=K.%20Paigen&amp;publication_year=2010&amp;journal=Science&amp;volume=327&amp;pages=835" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1181495" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1181495" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1181495"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20044538" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx048-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx048-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Baudat</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Buard</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Grey</div> <div class="given-names">C.</div></div></div>, et al.   <div class="year">2010</div>, 
<div class="article-title">PRDM9 is a major determinant of meiotic recombination hotspots in humans and mice</div>, <div class="source">Science</div> , <div class="volume">327</div>, <div class="fpage">836</div>–<div class="lpage">840</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PRDM9%20is%20a%20major%20determinant%20of%20meiotic%20recombination%20hotspots%20in%20humans%20and%20mice&amp;author=F.%20Baudat&amp;author=J.%20Buard&amp;author=C.%20Grey&amp;publication_year=2010&amp;journal=Science&amp;volume=327&amp;pages=836-840" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1183439" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1183439" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1183439"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20044539" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx048-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx048-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Berg</div> <div class="given-names">I.L.</div></div>, <div class="name string-name"><div class="surname">Rita</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Lam</div> <div class="given-names">K.W.G.</div></div></div>, et al.   <div class="year">2010</div>, 
<div class="article-title">PRDM9 variation strongly influences recombination hot-spot activity and meiotic instability in humans</div>, <div class="source">Nat. Genet</div> ., <div class="volume">42</div>, <div class="fpage">859</div>–<div class="lpage">863</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PRDM9%20variation%20strongly%20influences%20recombination%20hot-spot%20activity%20and%20meiotic%20instability%20in%20humans&amp;author=I.L.%20Berg&amp;author=N.%20Rita&amp;author=K.W.G.%20Lam&amp;publication_year=2010&amp;journal=Nat.%20Genet&amp;volume=42&amp;pages=859-863" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.658" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.658" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.658"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20818382" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx048-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx048-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Myers</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Bowden</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Tumian</div> <div class="given-names">A.</div></div></div>, et al.   <div class="year">2010</div>, 
<div class="article-title">Drive against hotspot motifs in primates implicates the PRDM9 gene in meiotic recombination</div>, <div class="source">Science</div> , <div class="volume">327</div>, <div class="fpage">876</div>–<div class="lpage">879</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Drive%20against%20hotspot%20motifs%20in%20primates%20implicates%20the%20PRDM9%20gene%20in%20meiotic%20recombination&amp;author=S.%20Myers&amp;author=R.%20Bowden&amp;author=A.%20Tumian&amp;publication_year=2010&amp;journal=Science&amp;volume=327&amp;pages=876-879" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1182363" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1182363" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1182363"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20044541" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx048-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx048-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kono</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Tamura</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Osada</div> <div class="given-names">N.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Prdm9 polymorphism unveils mouse evolutionary tracks</div>, <div class="source">DNA Res</div> ., <div class="volume">21</div>, <div class="fpage">315</div>–<div class="lpage">326</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Prdm9%20polymorphism%20unveils%20mouse%20evolutionary%20tracks&amp;author=H.%20Kono&amp;author=M.%20Tamura&amp;author=N.%20Osada&amp;publication_year=2014&amp;journal=DNA%20Res&amp;volume=21&amp;pages=315-326" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dst059" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdst059" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdst059"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24449848" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx048-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx048-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Schwartz</div> <div class="given-names">J.J.</div></div>, <div class="name string-name"><div class="surname">Roach</div> <div class="given-names">D.J.</div></div>, <div class="name string-name"><div class="surname">Thomas</div> <div class="given-names">J.H.</div></div>, <div class="name string-name"><div class="surname">Shendure</div> <div class="given-names">J.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Primate evolution of the recombination regulator PRDM9</div>, <div class="source">Nat. Commun.</div> , <div class="volume">5</div>, <div class="fpage">4370</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Primate%20evolution%20of%20the%20recombination%20regulator%20PRDM9&amp;author=J.J.%20Schwartz&amp;author=D.J.%20Roach&amp;author=J.H.%20Thomas&amp;author=J.%20Shendure&amp;publication_year=2014&amp;journal=Nat.%20Commun.&amp;volume=5&amp;pages=4370" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ncomms5370" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fncomms5370" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fncomms5370"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25001002" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx048-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx048-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ahlawat</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Sharma</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Sharma</div> <div class="given-names">R.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">Evidence of positive selection and concerted evolution in the rapidly evolving PRDM9 zinc finger domain in goats and sheep</div>, <div class="source">Anim. Genet.</div> , <div class="volume">47</div>, <div class="fpage">740</div>–<div class="lpage">751</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evidence%20of%20positive%20selection%20and%20concerted%20evolution%20in%20the%20rapidly%20evolving%20PRDM9%20zinc%20finger%20domain%20in%20goats%20and%20sheep&amp;author=S.%20Ahlawat&amp;author=P.%20Sharma&amp;author=R.%20Sharma&amp;publication_year=2016&amp;journal=Anim.%20Genet.&amp;volume=47&amp;pages=740-751" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/age.2016.47.issue-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fage.2016.47.issue-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fage.2016.47.issue-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27621101" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx048-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx048-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ponting</div> <div class="given-names">C.P.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">What are the genomic drivers of the rapid evolution of PRDM9?</div> <div class="source">Trends Genet.</div> , <div class="volume">27</div>, <div class="fpage">165</div>–<div class="lpage">171</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=What%20are%20the%20genomic%20drivers%20of%20the%20rapid%20evolution%20of%20PRDM9%3F&amp;author=C.P.%20Ponting&amp;publication_year=2011&amp;journal=Trends%20Genet.&amp;volume=27&amp;pages=165-171" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tig.2011.02.001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tig.2011.02.001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tig.2011.02.001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21388701" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx048-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx048-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lesecque</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Glémin</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Lartillot</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Mouchiroud</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Duret</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Coop</div> <div class="given-names">G.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">The Red Queen model of recombination hotspots evolution in the light of archaic and modern human genomes</div>, <div class="source">PLoS Genet</div> ., <div class="volume">10</div>, <div class="fpage">e1004790</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Red%20Queen%20model%20of%20recombination%20hotspots%20evolution%20in%20the%20light%20of%20archaic%20and%20modern%20human%20genomes&amp;author=Y.%20Lesecque&amp;author=S.%20Gl%C3%A9min&amp;author=N.%20Lartillot&amp;author=D.%20Mouchiroud&amp;author=L.%20Duret&amp;author=G.%20Coop&amp;publication_year=2014&amp;journal=PLoS%20Genet&amp;volume=10&amp;pages=e1004790" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1004790" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1004790" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1004790"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25393762" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx048-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx048-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Buard</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Rivals</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Dunoyer de Segonzac</div> <div class="given-names">D.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Diversity of Prdm9 zinc finger array in wild mice unravels new facets of the evolutionary turnover of this coding minisatellite</div>, <div class="source">PLoS One</div> , <div class="volume">9</div>, <div class="fpage">e85021</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Diversity%20of%20Prdm9%20zinc%20finger%20array%20in%20wild%20mice%20unravels%20new%20facets%20of%20the%20evolutionary%20turnover%20of%20this%20coding%20minisatellite&amp;author=J.%20Buard&amp;author=E.%20Rivals&amp;author=D.%20Dunoyer%20de%20Segonzac&amp;publication_year=2014&amp;journal=PLoS%20One&amp;volume=9&amp;pages=e85021" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0085021" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0085021" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0085021"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24454780" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx048-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx048-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Axelsson</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Webster</div> <div class="given-names">M.T.</div></div>, <div class="name string-name"><div class="surname">Ratnakumar</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Ponting</div> <div class="given-names">C.P.</div></div>, <div class="name string-name"><div class="surname">Lindblad-Toh</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Consortium</div> <div class="given-names">L.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Death of PRDM9 coincides with stabilization of the recombination landscape in the dog genome</div>, <div class="source">Genome Res</div> ., <div class="volume">22</div>, <div class="fpage">51</div>–<div class="lpage">63</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Death%20of%20PRDM9%20coincides%20with%20stabilization%20of%20the%20recombination%20landscape%20in%20the%20dog%20genome&amp;author=E.%20Axelsson&amp;author=M.T.%20Webster&amp;author=A.%20Ratnakumar&amp;author=C.P.%20Ponting&amp;author=K.%20Lindblad-Toh&amp;author=L.%20Consortium&amp;publication_year=2012&amp;journal=Genome%20Res&amp;volume=22&amp;pages=51-63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.124123.111" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.124123.111" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.124123.111"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22006216" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx048-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx048-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Auton</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">Y.R.</div></div>, <div class="name string-name"><div class="surname">Kidd</div> <div class="given-names">J.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">Genetic recombination is targeted towards gene promoter regions in dogs</div>, <div class="source">PLoS Genet</div> ., <div class="volume">9</div>, <div class="fpage">e1003984</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20recombination%20is%20targeted%20towards%20gene%20promoter%20regions%20in%20dogs&amp;author=A.%20Auton&amp;author=Y.R.%20Li&amp;author=J.%20Kidd&amp;publication_year=2013&amp;journal=PLoS%20Genet&amp;volume=9&amp;pages=e1003984" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1003984" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1003984" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1003984"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24348265" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx048-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx048-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ahlawat</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Sharma</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Sharma</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Arora</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">De</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Fugmann</div> <div class="given-names">S. D.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Zinc finger domain of the PRDM9 gene on chromosome 1 exhibits high diversity in ruminants but its paralog PRDM7 contains multiple disruptive mutations</div>, <div class="source">PLoS One</div> , <div class="volume">11</div>, <div class="fpage">e0156159</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Zinc%20finger%20domain%20of%20the%20PRDM9%20gene%20on%20chromosome%201%20exhibits%20high%20diversity%20in%20ruminants%20but%20its%20paralog%20PRDM7%20contains%20multiple%20disruptive%20mutations&amp;author=S.%20Ahlawat&amp;author=P.%20Sharma&amp;author=R.%20Sharma&amp;author=R.%20Arora&amp;author=S.%20De&amp;author=S.%20D.%20Fugmann&amp;publication_year=2016&amp;journal=PLoS%20One&amp;volume=11&amp;pages=e0156159" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0156159" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0156159" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0156159"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27203728" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx048-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx048-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Padhi</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Shen</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Jiang</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Zhou</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">G.E.</div></div>, <div class="name string-name"><div class="surname">Ma</div> <div class="given-names">L.</div></div></div> <div class="year">2017</div>, 
<div class="article-title">Ruminant-specific multiple duplication events of PRDM9 before speciation</div>, <div class="source">BMC Evol. Biol.,</div>  <div class="volume">17</div>, <div class="fpage">79</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ruminant-specific%20multiple%20duplication%20events%20of%20PRDM9%20before%20speciation&amp;author=A.%20Padhi&amp;author=B.%20Shen&amp;author=J.%20Jiang&amp;author=Y.%20Zhou&amp;author=G.E.%20Liu&amp;author=L.%20Ma&amp;publication_year=2017&amp;journal=BMC%20Evol.%20Biol.%2C&amp;volume=17&amp;pages=79" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12862-017-0892-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12862-017-0892-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12862-017-0892-4"> </span></div> </div></div></div></div></div><div content-id="dsx048-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx048-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Loftus</div> <div class="given-names">R.T.</div></div>, <div class="name string-name"><div class="surname">MacHugh</div> <div class="given-names">D.E.</div></div>, <div class="name string-name"><div class="surname">Bradley</div> <div class="given-names">D.G.</div></div>, <div class="name string-name"><div class="surname">Sharp</div> <div class="given-names">P.M.</div></div>, <div class="name string-name"><div class="surname">Cunningham</div> <div class="given-names">P.</div></div></div> <div class="year">1994</div>, 
<div class="article-title">Evidence for two independent domestications of cattle</div>, <div class="source">Proc. Natl. Acad. Sci</div> ., <div class="volume">91</div>, <div class="fpage">2757</div>–<div class="lpage">2761</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evidence%20for%20two%20independent%20domestications%20of%20cattle&amp;author=R.T.%20Loftus&amp;author=D.E.%20MacHugh&amp;author=D.G.%20Bradley&amp;author=P.M.%20Sharp&amp;author=P.%20Cunningham&amp;publication_year=1994&amp;journal=Proc.%20Natl.%20Acad.%20Sci&amp;volume=91&amp;pages=2757-2761" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.91.7.2757" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.91.7.2757" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.91.7.2757"> </span></div> </div></div></div></div></div><div content-id="dsx048-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx048-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Boitard</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Rodriguez</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Jay</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Mona</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Austerlitz</div> <div class="given-names">F.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Inferring population size history from large samples of genome-wide molecular data-an approximate Bayesian computation approach</div>, <div class="source">PLoS Genet</div> ., <div class="volume">12</div>, <div class="fpage">e1005877</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Inferring%20population%20size%20history%20from%20large%20samples%20of%20genome-wide%20molecular%20data-an%20approximate%20Bayesian%20computation%20approach&amp;author=S.%20Boitard&amp;author=W.%20Rodriguez&amp;author=F.%20Jay&amp;author=S.%20Mona&amp;author=F.%20Austerlitz&amp;publication_year=2016&amp;journal=PLoS%20Genet&amp;volume=12&amp;pages=e1005877" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1005877" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1005877" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1005877"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26943927" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx048-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx048-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kong</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Thorleifsson</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Gudbjartsson</div> <div class="given-names">D.F.</div></div></div>, et al.   <div class="year">2010</div>, 
<div class="article-title">Fine-scale recombination rate differences between sexes, populations and individuals</div>, <div class="source">Nature</div> , <div class="volume">467</div>, <div class="fpage">1099</div>–<div class="lpage">1103</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fine-scale%20recombination%20rate%20differences%20between%20sexes%2C%20populations%20and%20individuals&amp;author=A.%20Kong&amp;author=G.%20Thorleifsson&amp;author=D.F.%20Gudbjartsson&amp;publication_year=2010&amp;journal=Nature&amp;volume=467&amp;pages=1099-1103" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature09525" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature09525" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature09525"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20981099" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx048-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx048-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Shifman</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Bell</div> <div class="given-names">J.T.</div></div>, <div class="name string-name"><div class="surname">Copley</div> <div class="given-names">R.R.</div></div></div>, et al.   <div class="year">2006</div>, 
<div class="article-title">A high-resolution single nucleotide polymorphism genetic map of the mouse genome</div>, <div class="source">PLoS Biol.</div> , <div class="volume">4</div>, <div class="fpage">e395</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20high-resolution%20single%20nucleotide%20polymorphism%20genetic%20map%20of%20the%20mouse%20genome&amp;author=S.%20Shifman&amp;author=J.T.%20Bell&amp;author=R.R.%20Copley&amp;publication_year=2006&amp;journal=PLoS%20Biol.&amp;volume=4&amp;pages=e395" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pbio.0040395" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pbio.0040395" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pbio.0040395"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17105354" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx048-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx048-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hunter</div> <div class="given-names">C.M.</div></div>, <div class="name string-name"><div class="surname">Huang</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Mackay</div> <div class="given-names">T.F.C.</div></div>, <div class="name string-name"><div class="surname">Singh</div> <div class="given-names">N.D.</div></div>, <div class="name string-name"><div class="surname">Sekelsky</div> <div class="given-names">J.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">The genetic architecture of natural variation in recombination rate in Drosophila melanogaster</div>, <div class="source">PLoS Genet.</div> , <div class="volume">12</div>, <div class="fpage">e1005951</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genetic%20architecture%20of%20natural%20variation%20in%20recombination%20rate%20in%20Drosophila%20melanogaster&amp;author=C.M.%20Hunter&amp;author=W.%20Huang&amp;author=T.F.C.%20Mackay&amp;author=N.D.%20Singh&amp;author=J.%20Sekelsky&amp;publication_year=2016&amp;journal=PLoS%20Genet.&amp;volume=12&amp;pages=e1005951" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1005951" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1005951" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1005951"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27035832" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx048-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx048-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Nachman</div> <div class="given-names">M.W.</div></div>, <div class="name string-name"><div class="surname">Payseur</div> <div class="given-names">B.A.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Recombination rate variation and speciation: theoretical predictions and empirical results from rabbits and mice</div>, <div class="source">Philos. Trans. R. Soc. B</div> , <div class="volume">367</div>, <div class="fpage">409</div>–<div class="lpage">421</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Recombination%20rate%20variation%20and%20speciation%3A%20theoretical%20predictions%20and%20empirical%20results%20from%20rabbits%20and%20mice&amp;author=M.W.%20Nachman&amp;author=B.A.%20Payseur&amp;publication_year=2012&amp;journal=Philos.%20Trans.%20R.%20Soc.%20B&amp;volume=367&amp;pages=409-421" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1098/rstb.2011.0249" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1098%2Frstb.2011.0249" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1098%2Frstb.2011.0249"> </span></div> </div></div></div></div></div><div content-id="dsx048-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx048-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Balcova</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Faltusova</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Gergelits</div> <div class="given-names">V.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">Hybrid sterility locus on chromosome X controls meiotic recombination rate in mouse</div>, <div class="source">PLoS Genet.</div> , <div class="volume">12</div>, <div class="fpage">e1005906</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Hybrid%20sterility%20locus%20on%20chromosome%20X%20controls%20meiotic%20recombination%20rate%20in%20mouse&amp;author=M.%20Balcova&amp;author=B.%20Faltusova&amp;author=V.%20Gergelits&amp;publication_year=2016&amp;journal=PLoS%20Genet.&amp;volume=12&amp;pages=e1005906" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1005906" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1005906" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1005906"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27104744" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx048-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx048-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Pratto</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Brick</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Khil</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Smagulova</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Petukhova</div> <div class="given-names">G.V.</div></div>, <div class="name string-name"><div class="surname">Camerini-Otero</div> <div class="given-names">R.D.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Recombination initiation maps of individual human genomes</div>, <div class="source">Science</div> , <div class="volume">346</div>, <div class="fpage">1256442</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Recombination%20initiation%20maps%20of%20individual%20human%20genomes&amp;author=F.%20Pratto&amp;author=K.%20Brick&amp;author=P.%20Khil&amp;author=F.%20Smagulova&amp;author=G.V.%20Petukhova&amp;author=R.D.%20Camerini-Otero&amp;publication_year=2014&amp;journal=Science&amp;volume=346&amp;pages=1256442" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1256442" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1256442" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1256442"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25395542" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx048-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx048-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Baker</div> <div class="given-names">C.L.</div></div>, <div class="name string-name"><div class="surname">Kajita</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Walker</div> <div class="given-names">M.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">PRDM9 drives evolutionary erosion of hotspots in Mus musculus through haplotype-specific initiation of meiotic recombination</div>, <div class="source">PLoS Genet</div> ., <div class="volume">11</div>, <div class="fpage">e1004916</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PRDM9%20drives%20evolutionary%20erosion%20of%20hotspots%20in%20Mus%20musculus%20through%20haplotype-specific%20initiation%20of%20meiotic%20recombination&amp;author=C.L.%20Baker&amp;author=S.%20Kajita&amp;author=M.%20Walker&amp;publication_year=2015&amp;journal=PLoS%20Genet&amp;volume=11&amp;pages=e1004916" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1004916" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1004916" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1004916"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25568937" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx048-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx048-B29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Payseur</div> <div class="given-names">B.A.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Genetic links between recombination and speciation</div>, <div class="source">PLoS Genet.</div> , <div class="volume">12</div>, <div class="fpage">e1006066</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20links%20between%20recombination%20and%20speciation&amp;author=B.A.%20Payseur&amp;publication_year=2016&amp;journal=PLoS%20Genet.&amp;volume=12&amp;pages=e1006066" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1006066" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1006066" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1006066"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27280290" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx048-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx048-B30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Davies</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Hatton</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Altemose</div> <div class="given-names">N.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">Re-engineering the zinc fingers of PRDM9 reverses hybrid sterility in mice</div>, <div class="source">Nature</div> , <div class="volume">530</div>, <div class="fpage">171</div>–<div class="lpage">176</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Re-engineering%20the%20zinc%20fingers%20of%20PRDM9%20reverses%20hybrid%20sterility%20in%20mice&amp;author=B.%20Davies&amp;author=E.%20Hatton&amp;author=N.%20Altemose&amp;publication_year=2016&amp;journal=Nature&amp;volume=530&amp;pages=171-176" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature16931" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature16931" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature16931"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26840484" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx048-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx048-B31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ma</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">O'Connell</div> <div class="given-names">J.R.</div></div>, <div class="name string-name"><div class="surname">VanRaden</div> <div class="given-names">P.M.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Cattle sex-specific recombination and genetic control from a large pedigree analysis</div>, <div class="source">PLoS Genet.</div> , <div class="volume">11</div>, <div class="fpage">e1005387</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cattle%20sex-specific%20recombination%20and%20genetic%20control%20from%20a%20large%20pedigree%20analysis&amp;author=L.%20Ma&amp;author=J.R.%20O'Connell&amp;author=P.M.%20VanRaden&amp;publication_year=2015&amp;journal=PLoS%20Genet.&amp;volume=11&amp;pages=e1005387" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1005387" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1005387" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1005387"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26540184" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx048-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx048-B32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Jensen-Seaman</div> <div class="given-names">M.I.</div></div>, <div class="name string-name"><div class="surname">Furey</div> <div class="given-names">T.S.</div></div>, <div class="name string-name"><div class="surname">Payseur</div> <div class="given-names">B.A.</div></div></div>, et al.   <div class="year">2004</div>, 
<div class="article-title">Comparative recombination rates in the rat, mouse, and human genomes</div>, <div class="source">Genome Res</div> ., <div class="volume">14</div>, <div class="fpage">528</div>–<div class="lpage">538</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20recombination%20rates%20in%20the%20rat%2C%20mouse%2C%20and%20human%20genomes&amp;author=M.I.%20Jensen-Seaman&amp;author=T.S.%20Furey&amp;author=B.A.%20Payseur&amp;publication_year=2004&amp;journal=Genome%20Res&amp;volume=14&amp;pages=528-538" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.1970304" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.1970304" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.1970304"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15059993" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx048-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx048-B33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Paigen</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Szatkiewicz</div> <div class="given-names">J.P.</div></div>, <div class="name string-name"><div class="surname">Sawyer</div> <div class="given-names">K.</div></div></div>, et al.   <div class="year">2008</div>, 
<div class="article-title">The recombinational anatomy of a mouse chromosome</div>, <div class="source">PLoS Genet</div> ., <div class="volume">4</div>, <div class="fpage">e1000119</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20recombinational%20anatomy%20of%20a%20mouse%20chromosome&amp;author=K.%20Paigen&amp;author=J.P.%20Szatkiewicz&amp;author=K.%20Sawyer&amp;publication_year=2008&amp;journal=PLoS%20Genet&amp;volume=4&amp;pages=e1000119" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1000119" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1000119" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1000119"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18617997" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx048-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx048-B34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kadri</div> <div class="given-names">N.K.</div></div>, <div class="name string-name"><div class="surname">Harland</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Faux</div> <div class="given-names">P.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">Coding and noncoding variants in HFM1, MLH3, MSH4, MSH5, RNF212, and RNF212B affect recombination rate in cattle</div>, <div class="source">Genome Res.</div> , <div class="volume">26</div>, <div class="fpage">1323</div>–<div class="lpage">1332</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Coding%20and%20noncoding%20variants%20in%20HFM1%2C%20MLH3%2C%20MSH4%2C%20MSH5%2C%20RNF212%2C%20and%20RNF212B%20affect%20recombination%20rate%20in%20cattle&amp;author=N.K.%20Kadri&amp;author=C.%20Harland&amp;author=P.%20Faux&amp;publication_year=2016&amp;journal=Genome%20Res.&amp;volume=26&amp;pages=1323-1332" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.204214.116" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.204214.116" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.204214.116"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27516620" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx048-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx048-B35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hubert</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">MacDonald</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Gusella</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Arnheim</div> <div class="given-names">N.</div></div></div> <div class="year">1994</div>, 
<div class="article-title">High resolution localization of recombination hot spots using sperm typing</div>, <div class="source">Nat. Genet.</div> , <div class="volume">7</div>, <div class="fpage">420</div>–<div class="lpage">424</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High%20resolution%20localization%20of%20recombination%20hot%20spots%20using%20sperm%20typing&amp;author=R.%20Hubert&amp;author=M.%20MacDonald&amp;author=J.%20Gusella&amp;author=N.%20Arnheim&amp;publication_year=1994&amp;journal=Nat.%20Genet.&amp;volume=7&amp;pages=420-424" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng0794-420" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng0794-420" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng0794-420"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7920662" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx048-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx048-B36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wang</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Fan</div> <div class="given-names">H.C.</div></div>, <div class="name string-name"><div class="surname">Behr</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Quake</div> <div class="given-names">S. R.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Genome-wide single-cell analysis of recombination activity and de novo mutation rates in human sperm</div>, <div class="source">Cell</div> , <div class="volume">150</div>, <div class="fpage">402</div>–<div class="lpage">412</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20single-cell%20analysis%20of%20recombination%20activity%20and%20de%20novo%20mutation%20rates%20in%20human%20sperm&amp;author=J.%20Wang&amp;author=H.C.%20Fan&amp;author=B.%20Behr&amp;author=S.%20R.%20Quake&amp;publication_year=2012&amp;journal=Cell&amp;volume=150&amp;pages=402-412" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2012.06.030" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2012.06.030" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2012.06.030"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22817899" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx048-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx048-B37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Clark</div> <div class="given-names">A.G.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Matise</div> <div class="given-names">T.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Contrasting methods of quantifying fine structure of human recombination</div>, <div class="source">Annu. Rev. Genom. Hum. Genet.</div> , <div class="volume">11</div>, <div class="fpage">45</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Contrasting%20methods%20of%20quantifying%20fine%20structure%20of%20human%20recombination&amp;author=A.G.%20Clark&amp;author=X.%20Wang&amp;author=T.%20Matise&amp;publication_year=2010&amp;journal=Annu.%20Rev.%20Genom.%20Hum.%20Genet.&amp;volume=11&amp;pages=45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev-genom-082908-150031" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev-genom-082908-150031" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev-genom-082908-150031"> </span></div> </div></div></div></div></div><div content-id="dsx048-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx048-B38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lu</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Zong</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Fan</div> <div class="given-names">W.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">Probing meiotic recombination and aneuploidy of single sperm cells by whole-genome sequencing</div>, <div class="source">Science</div> , <div class="volume">338</div>, <div class="fpage">1627</div>–<div class="lpage">1630</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Probing%20meiotic%20recombination%20and%20aneuploidy%20of%20single%20sperm%20cells%20by%20whole-genome%20sequencing&amp;author=S.%20Lu&amp;author=C.%20Zong&amp;author=W.%20Fan&amp;publication_year=2012&amp;journal=Science&amp;volume=338&amp;pages=1627-1630" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1229112" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1229112" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1229112"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23258895" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx048-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx048-B39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hubbard</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Barker</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Birney</div> <div class="given-names">E.</div></div></div>, et al.   <div class="year">2002</div>, 
<div class="article-title">The Ensembl genome database project</div>, <div class="source">Nucl. Acids Res</div> ., <div class="volume">30</div>, <div class="fpage">38</div>–<div class="lpage">41</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Ensembl%20genome%20database%20project&amp;author=T.%20Hubbard&amp;author=D.%20Barker&amp;author=E.%20Birney&amp;publication_year=2002&amp;journal=Nucl.%20Acids%20Res&amp;volume=30&amp;pages=38-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/30.1.38" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F30.1.38" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F30.1.38"> </span></div> </div></div></div></div></div><div content-id="dsx048-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx048-B40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ye</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Coulouris</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Zaretskaya</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Cutcutache</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Rozen</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Madden</div> <div class="given-names">T.L.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Primer-BLAST: a tool to design target-specific primers for polymerase chain reaction</div>, <div class="source">BMC Bioinformatics</div> , <div class="volume">13</div>, <div class="fpage">1</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Primer-BLAST%3A%20a%20tool%20to%20design%20target-specific%20primers%20for%20polymerase%20chain%20reaction&amp;author=J.%20Ye&amp;author=G.%20Coulouris&amp;author=I.%20Zaretskaya&amp;author=I.%20Cutcutache&amp;author=S.%20Rozen&amp;author=T.L.%20Madden&amp;publication_year=2012&amp;journal=BMC%20Bioinformatics&amp;volume=13&amp;pages=1" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-13-S6-S1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-13-S6-S1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-13-S6-S1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22214541" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx048-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx048-B41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zimin</div> <div class="given-names">A.V.</div></div>, <div class="name string-name"><div class="surname">Delcher</div> <div class="given-names">A.L.</div></div>, <div class="name string-name"><div class="surname">Florea</div> <div class="given-names">L.</div></div></div>, et al.   <div class="year">2009</div>, 
<div class="article-title">A whole-genome assembly of the domestic cow, Bos taurus</div>, <div class="source">Genome Biol.</div> , <div class="volume">10</div>(<div class="issue">1</div>), <div class="fpage">R42</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20whole-genome%20assembly%20of%20the%20domestic%20cow%2C%20Bos%20taurus&amp;author=A.V.%20Zimin&amp;author=A.L.%20Delcher&amp;author=L.%20Florea&amp;publication_year=2009&amp;journal=Genome%20Biol.&amp;volume=10&amp;pages=R42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2009-10-4-r42" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2009-10-4-r42" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2009-10-4-r42"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19393038" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx048-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx048-B42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Fazakerley</div> <div class="given-names">D.J.</div></div>, <div class="name string-name"><div class="surname">Naghiloo</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Chaudhuri</div> <div class="given-names">R.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Proteomic analysis of GLUT4 storage vesicles reveals tumor suppressor candidate 5 (TUSC5) as a novel regulator of insulin action in adipocytes</div>, <div class="source">J. Biol. Chem.</div> , <div class="volume">290</div>, <div class="fpage">23528</div>–<div class="lpage">23542</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Proteomic%20analysis%20of%20GLUT4%20storage%20vesicles%20reveals%20tumor%20suppressor%20candidate%205%20(TUSC5)%20as%20a%20novel%20regulator%20of%20insulin%20action%20in%20adipocytes&amp;author=D.J.%20Fazakerley&amp;author=S.%20Naghiloo&amp;author=R.%20Chaudhuri&amp;publication_year=2015&amp;journal=J.%20Biol.%20Chem.&amp;volume=290&amp;pages=23528-23542" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.M115.657361" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.M115.657361" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.M115.657361"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26240143" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx048-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx048-B43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation"><div class="collab">R, CT</div>. <div class="year">2014</div>, 
<div class="article-title">R: a language and environment for statistical computing</div>, <div class="source">Vienna, Austria: R Foundation Stat. Comput</div> ., <div class="volume">5</div>, <div class="fpage">299</div>–<div class="lpage">314</div>.</div></div></div></div><div content-id="dsx048-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx048-B44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Oliver</div> <div class="given-names">P.L.</div></div>, <div class="name string-name"><div class="surname">Goodstadt</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Bayes</div> <div class="given-names">J.J.</div></div></div>, et al.   <div class="year">2009</div>, 
<div class="article-title">Accelerated evolution of the Prdm9 speciation gene across diverse metazoan taxa</div>, <div class="source">PLoS Genet</div> ., <div class="volume">5</div>, <div class="fpage">e1000753</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Accelerated%20evolution%20of%20the%20Prdm9%20speciation%20gene%20across%20diverse%20metazoan%20taxa&amp;author=P.L.%20Oliver&amp;author=L.%20Goodstadt&amp;author=J.J.%20Bayes&amp;publication_year=2009&amp;journal=PLoS%20Genet&amp;volume=5&amp;pages=e1000753" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1000753" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1000753" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1000753"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19997497" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx048-B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx048-B45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Persikov</div> <div class="given-names">A.V.</div></div>, <div class="name string-name"><div class="surname">Singh</div> <div class="given-names">M.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">De novo prediction of DNA-binding specificities for Cys2His2 zinc finger proteins</div>, <div class="source">Nucl. Acids Res</div> ., <div class="volume">42</div>, <div class="fpage">97</div>–<div class="lpage">108</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=De%20novo%20prediction%20of%20DNA-binding%20specificities%20for%20Cys2His2%20zinc%20finger%20proteins&amp;author=A.V.%20Persikov&amp;author=M.%20Singh&amp;publication_year=2014&amp;journal=Nucl.%20Acids%20Res&amp;volume=42&amp;pages=97-108" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkt890" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkt890" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkt890"> </span></div> </div></div></div></div></div><div content-id="dsx048-B46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx048-B46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Smagulova</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Brick</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Pu</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Camerini-Otero</div> <div class="given-names">R.D.</div></div>, <div class="name string-name"><div class="surname">Petukhova</div> <div class="given-names">G.V.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">The evolutionary turnover of recombination hot spots contributes to speciation in mice</div>, <div class="source">Genes Dev.</div> , <div class="volume">30</div>, <div class="fpage">266</div>–<div class="lpage">280</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20evolutionary%20turnover%20of%20recombination%20hot%20spots%20contributes%20to%20speciation%20in%20mice&amp;author=F.%20Smagulova&amp;author=K.%20Brick&amp;author=Y.%20Pu&amp;author=R.D.%20Camerini-Otero&amp;author=G.V.%20Petukhova&amp;publication_year=2016&amp;journal=Genes%20Dev.&amp;volume=30&amp;pages=266-280" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gad.270009.115" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgad.270009.115" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgad.270009.115"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26833728" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx048-B47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx048-B47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sandor</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Coppieters</div> <div class="given-names">W.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">Genetic variants in REC8, RNF212, and PRDM9 influence male recombination in cattle</div>, <div class="source">PLoS Genet.</div> , <div class="volume">8</div>, <div class="fpage">e1002854</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20variants%20in%20REC8%2C%20RNF212%2C%20and%20PRDM9%20influence%20male%20recombination%20in%20cattle&amp;author=C.%20Sandor&amp;author=W.%20Li&amp;author=W.%20Coppieters&amp;publication_year=2012&amp;journal=PLoS%20Genet.&amp;volume=8&amp;pages=e1002854" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1002854" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1002854" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1002854"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22844258" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx048-B48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx048-B48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ségurel</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Leffler</div> <div class="given-names">E.M.</div></div>, <div class="name string-name"><div class="surname">Przeworski</div> <div class="given-names">M.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">The case of the fickle fingers: how the PRDM9 zinc finger protein specifies meiotic recombination hotspots in humans</div>, <div class="source">PLoS Biol</div> ., <div class="volume">9</div>, <div class="fpage">e1001211</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20case%20of%20the%20fickle%20fingers%3A%20how%20the%20PRDM9%20zinc%20finger%20protein%20specifies%20meiotic%20recombination%20hotspots%20in%20humans&amp;author=L.%20S%C3%A9gurel&amp;author=E.M.%20Leffler&amp;author=M.%20Przeworski&amp;publication_year=2011&amp;journal=PLoS%20Biol&amp;volume=9&amp;pages=e1001211" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pbio.1001211" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pbio.1001211" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pbio.1001211"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22162947" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx048-B49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx048-B49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Grey</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Barthès</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Chauveau-Le Friec</div> <div class="given-names">G.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">Mouse PRDM9 DNA-binding specificity determines sites of histone H3 lysine 4 trimethylation for initiation of meiotic recombination</div>, <div class="source">PLoS Biol</div> ., <div class="volume">9</div>, <div class="fpage">e1001176</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mouse%20PRDM9%20DNA-binding%20specificity%20determines%20sites%20of%20histone%20H3%20lysine%204%20trimethylation%20for%20initiation%20of%20meiotic%20recombination&amp;author=C.%20Grey&amp;author=P.%20Barth%C3%A8s&amp;author=G.%20Chauveau-Le%20Friec&amp;publication_year=2011&amp;journal=PLoS%20Biol&amp;volume=9&amp;pages=e1001176" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pbio.1001176" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pbio.1001176" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pbio.1001176"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22028627" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">Published by Oxford University Press on behalf of Kazusa DNA Research Institute 2017. This work is written by US Government employees and is in the public domain in the US.</div><div class="license"><div class="license-p">This article is published and distributed under the terms of the Oxford University Press, Standard Journals Publication Model (<a class="link link-uri" href="https://academic.oup.com/journals/pages/about_us/legal/notices" target="_blank">https://academic.oup.com/journals/pages/about_us/legal/notices</a>)</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=4665695&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx048/1/dsx048_supp.zip?Expires=2147483647&amp;Signature=cB7LewD99XIfXtilEHTinjygYWutIBSPgTSSy6xH4XO0UhC59pZBSl92WUktO2BqH2Hl3-GTWAYw6LtDm7lW3YE-dlS2MMAJV4XEQXCo7TifcPEQEtn9LH0s2AbatwLORuKdw7mxT~nLTak89qTAjVAIM7KkWkbB~m9AEtyVLoWKEPXNGaKxuHqdS616H4n9373orlSfnWCzhmqoBYwtgyfQLPB02zulnR7CrM97FtAfZQI7dtJtAAxSr2-Z3~J8WnVzzJL1ozAZMGjaHP3tY2mkF3halnzqJJ4cFtpVZWGJH7catGGXeTJ8UIeICXKqNKHC8OxySuj2kP00XSGRuA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">1,014</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
0                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsx048"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="4665695"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f25%2f2%2f183%2f4665695">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F25%2F2%2F183%2F4665695" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/4665695'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Construction of PRDM9 allele-specific recombination maps in cattle using large-scale pedigree analysis and genome-wide single sperm genomics" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsx048");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f25%2f2%2f183%2f4665695">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

