

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/15/5/323/359995","@type":"ScholarlyArticle","name":"A High-density Linkage Map of  Lotus japonicus  Based on AFLP and SSR Markers","datePublished":"2008-09-14","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/15/5","@type":"PublicationIssue","issueNumber":"5","datePublished":"2008-10-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsn022","keywords":["<italic>Lotus japonicus</italic>","AFLP","SSR","linkage map","HEGS (high efficiency genome scanning)"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/15/5/323/359995","author":[{"name":"Wang, Xinwang","affiliation":"1  National Institute of Agrobiological Sciences ,  Kannon-dai 2-1-2, Tsukuba, Ibaraki 305-8602 ,  Japan","@type":"Person"},{"name":"Sato, Shusei","affiliation":"2  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Tabata, Satoshi","affiliation":"2  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Kawasaki, Shinji","affiliation":"1  National Institute of Agrobiological Sciences ,  Kannon-dai 2-1-2, Tsukuba, Ibaraki 305-8602 ,  Japan","@type":"Person"}],"description":"Abstract.  A collection of 94 F6 individuals derived from crosses between Lotus japonicus, Gifu B-129 (G) and Miyakojima MG-20 (M) were used for mapping. By usi","pageStart":"323","pageEnd":"332","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/15/5/1/m_cover.gif?Expires=2147483647&Signature=w0UhL1170rQznOI4EBmUQXu1ZOwiDy5D9yvI-xItfA5gEGy2mkrpZiqzDFtXeoJ~7GciRt1Jb~wykd96BMcXNK2oJMQKCvan9OGOux6ILlOeg1peDGvihwkEZeSlIEQRWsuy-Os5b2zaUJLN2EMIjSPATvDcEQeiR8VJm~tHNehbKW~jZGwxEfbLVfvvbsN-HCrfd6zxxRmciZJddcZFQjaH25Oc8zwhd9TcubBub~UwfgV64C6hjwxGjWayYeyC6503xComjsunKItnp20PuervJsOqbPEquDoZ8SVxa8ULqEXM-U2u8h~RKynzXZGyUfUdPZZhfeDp4GDC653uKg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"A High-density Linkage Map of  Lotus japonicus  Based on AFLP and SSR Markers","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/15/5/1/m_cover.gif?Expires=2147483647&Signature=w0UhL1170rQznOI4EBmUQXu1ZOwiDy5D9yvI-xItfA5gEGy2mkrpZiqzDFtXeoJ~7GciRt1Jb~wykd96BMcXNK2oJMQKCvan9OGOux6ILlOeg1peDGvihwkEZeSlIEQRWsuy-Os5b2zaUJLN2EMIjSPATvDcEQeiR8VJm~tHNehbKW~jZGwxEfbLVfvvbsN-HCrfd6zxxRmciZJddcZFQjaH25Oc8zwhd9TcubBub~UwfgV64C6hjwxGjWayYeyC6503xComjsunKItnp20PuervJsOqbPEquDoZ8SVxa8ULqEXM-U2u8h~RKynzXZGyUfUdPZZhfeDp4GDC653uKg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="High-density Linkage Map of  Lotus japonicus  Based on AFLP and SSR Markers" />
    <meta property="og:description" content="Abstract.  A collection of 94 F6 individuals derived from crosses between Lotus japonicus, Gifu B-129 (G) and Miyakojima MG-20 (M) were used for mapping. By usi" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/15/5/323/359995" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2008-09-14" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Wang, Xinwang" /><meta name="citation_author_institution" content="National Institute of Agrobiological Sciences, Kannon-dai 2-1-2, Tsukuba, Ibaraki 305-8602, Japan" /><meta name="citation_author" content="Sato, Shusei" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Tabata, Satoshi" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Kawasaki, Shinji" /><meta name="citation_author_institution" content="National Institute of Agrobiological Sciences, Kannon-dai 2-1-2, Tsukuba, Ibaraki 305-8602, Japan" /><meta name="citation_title" content="A High-density Linkage Map of Lotus japonicus Based on AFLP and SSR Markers" /><meta name="citation_firstpage" content="323" /><meta name="citation_lastpage" content="332" /><meta name="citation_doi" content="10.1093/dnares/dsn022" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="15" /><meta name="citation_issue" content="5" /><meta name="citation_publication_date" content="2008/10/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=AFLP: a new technique for DNA fingerprinting; Nucleic Acids Res.;  citation_year=1995;  citation_volume=23;  citation_pages=4407-4414; " /><meta name="citation_reference" content="citation_title=Level of polymorphism and genetic mapping of AFLP markers in rice; Genome;  citation_year=1996;  citation_volume=39;  citation_pages=969-977; " /><meta name="citation_reference" content="citation_title=A high-density soybean genetic map based on AFLP markers; Crop. Sci.;  citation_year=1997;  citation_volume=37;  citation_pages=537-543; " /><meta name="citation_reference" content="citation_title=Construction of an AFLP genetic map with nearly complete genome coverage in Pinus taeda; Theor. Appl. Genet.;  citation_year=1999;  citation_volume=98;  citation_pages=1279-1292; " /><meta name="citation_reference" content="citation_title=The development of a genetic map for meadowfoam comprised of amplified fragment length polymorphisms; Theor. Appl. Genet.;  citation_year=2002;  citation_volume=104;  citation_pages=92-96; " /><meta name="citation_reference" content="citation_title=A genetic linkage map of sweetpotato [Ipomoea batatas (L.) Lam.] based on AFLP markers; Mol. Breed.;  citation_year=2003;  citation_volume=11;  citation_pages=169-185; " /><meta name="citation_reference" content="citation_title=Lotus japonicus, an autogamous, diploid legume species for classical and molecular genetics; Plant J.;  citation_year=1992;  citation_volume=2;  citation_pages=487-496; " /><meta name="citation_reference" content="citation_title=Classical and molecular genetics of the model legume Lotus japonicus; Mol. Plant Microbe Interact.;  citation_year=1997;  citation_volume=10;  citation_pages=59-68; " /><meta name="citation_reference" content="citation_title=Genome and chromosome dimensions of Lotus japonicus; J. Plant Res.;  citation_year=2000;  citation_volume=113;  citation_pages=435-442; " /><meta name="citation_reference" content="citation_title=Genome analysis of Lotus japonicus; J. Plant Res.;  citation_year=2000;  citation_volume=113;  citation_pages=497-506; " /><meta name="citation_reference" content="citation_title=High frequency transformation and regeneration of transgenic plants in the model legume Lotus japonicus; J. Exp. Bot.;  citation_year=1997;  citation_volume=48;  citation_pages=1357-1365; " /><meta name="citation_reference" content="citation_title=Providing the basis for genomics in Lotus japonicus: the accessions Miyakojima MG-20 and Gifu B-129 are appropriate crossing partners for genetic analyses; Mol. Gen. Genet.;  citation_year=2001;  citation_volume=266;  citation_pages=157-166; " /><meta name="citation_reference" content="citation_title=Construction of a genetic linkage map of the model legume Lotus japonicus using an intraspecific F2 population; DNA Res.;  citation_year=2001;  citation_volume=8;  citation_pages=301-310; " /><meta name="citation_reference" content="citation_title=A genetic linkage map of the model legume Lotus japonicus and strategies for fast mapping of new loci; Genetics;  citation_year=2002;  citation_volume=161;  citation_pages=1673-1683; " /><meta name="citation_reference" content="citation_title=Structural analysis of a Lotus japonicus genome. I. Sequence features and mapping of fifty-six TAC clones which cover the 5.4 Mb regions of the genome; DNA Res.;  citation_year=2001;  citation_volume=8;  citation_pages=311-318; " /><meta name="citation_reference" content="citation_title=JoinMapÂ® 3.0, Software for the calculation of genetic linkage maps, Plant Research International; citation_publisher=the Netherlands,, Wageningen;  citation_year=2001; " /><meta name="citation_reference" content="citation_title=The estimation of map distances from recombination values; Ann. Eugen.;  citation_year=1944;  citation_volume=12;  citation_pages=172-175; " /><meta name="citation_reference" content="citation_title=MapChart: Software for the graphical presentation of linkage maps and QTLs; J. Hered.;  citation_year=2002;  citation_volume=93;  citation_pages=77-78; " /><meta name="citation_reference" content="citation_title=Precision mapping of quantitative trait loci; Genetics;  citation_year=1994;  citation_volume=136;  citation_pages=1457-1466; " /><meta name="citation_reference" content="citation_title=Windows QTL Cartographer 2.5;  citation_url=http://statgen.ncsu.edu/qtlcart/WQTLCart.htm; citation_publisher=Department of Statistics, North Carolina State University, Raleigh,, NC;  citation_year=2006; " /><meta name="citation_reference" content="citation_title=High-density molecular linkage maps of the tomato and potato genomes; Genetics;  citation_year=1992;  citation_volume=132;  citation_pages=1141-1160; " /><meta name="citation_reference" content="citation_title=Use of locus-specific AFLP markers to construct a high-density molecular map in barley; Theor. Appl. Genet.;  citation_year=1998;  citation_volume=96;  citation_pages=376-384; " /><meta name="citation_reference" content="citation_title=An integrated RFLP-AFLP linkage map for cultivated sunflower; Genome;  citation_year=2001;  citation_volume=44;  citation_pages=213-221; " /><meta name="citation_reference" content="citation_title=AFLP maps of Petunia hybrida: building maps when markers cluster; Theor. Appl. Genet.;  citation_year=2002;  citation_volume=105;  citation_pages=1000-1009; " /><meta name="citation_reference" content="citation_title=RFLP-based genetic map of the homoeologous group-3 chromosomes of wheat and rye; Theor. Appl. Genet.;  citation_year=1992;  citation_volume=83;  citation_pages=931-939; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/15/5/323/1030691/dsn022.pdf" /><meta name="description" content="Abstract.  A collection of 94 F6 individuals derived from crosses between Lotus japonicus, Gifu B-129 (G) and Miyakojima MG-20 (M) were used for mapping. By usi" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/15/5/323/359995" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/15/5/323/359995" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/15/5/323/359995");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsn022';
    </script>
<title>
	High-density Linkage Map of Lotus japonicus Based on AFLP and SSR Markers | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./359995" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0Bbg6PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJXYW5nLCBYaW53YW5nIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTmF0aW9uYWwgSW5zdGl0dXRlIG9mIEFncm9iaW9sb2dpY2FsIFNjaWVuY2VzLCBLYW5ub24tZGFpIDItMS0yLCBUc3VrdWJhLCBJYmFyYWtpIDMwNS04NjAyLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlNhdG8sIFNodXNlaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCAyLTYtNyBLYXp1c2Eta2FtYXRhcmksIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJUYWJhdGEsIFNhdG9zaGkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZSwgMi02LTcgS2F6dXNhLWthbWF0YXJpLCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS2F3YXNha2ksIFNoaW5qaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5hdGlvbmFsIEluc3RpdHV0ZSBvZiBBZ3JvYmlvbG9naWNhbCBTY2llbmNlcywgS2Fubm9uLWRhaSAyLTEtMiwgVHN1a3ViYSwgSWJhcmFraSAzMDUtODYwMiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IkEgSGlnaC1kZW5zaXR5IExpbmthZ2UgTWFwIG9mIExvdHVzIGphcG9uaWN1cyBCYXNlZCBvbiBBRkxQIGFuZCBTU1IgTWFya2VycyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjMyMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iMzMyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHNuMDIyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjE1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSI1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMDgvMTAvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BRkxQOiBhIG5ldyB0ZWNobmlxdWUgZm9yIEROQSBmaW5nZXJwcmludGluZzsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTk1OyAgY2l0YXRpb25fdm9sdW1lPTIzOyAgY2l0YXRpb25fcGFnZXM9NDQwNy00NDE0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1MZXZlbCBvZiBwb2x5bW9ycGhpc20gYW5kIGdlbmV0aWMgbWFwcGluZyBvZiBBRkxQIG1hcmtlcnMgaW4gcmljZTsgR2Vub21lOyAgY2l0YXRpb25feWVhcj0xOTk2OyAgY2l0YXRpb25fdm9sdW1lPTM5OyAgY2l0YXRpb25fcGFnZXM9OTY5LTk3NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBoaWdoLWRlbnNpdHkgc295YmVhbiBnZW5ldGljIG1hcCBiYXNlZCBvbiBBRkxQIG1hcmtlcnM7IENyb3AuIFNjaS47ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9Mzc7ICBjaXRhdGlvbl9wYWdlcz01MzctNTQzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db25zdHJ1Y3Rpb24gb2YgYW4gQUZMUCBnZW5ldGljIG1hcCB3aXRoIG5lYXJseSBjb21wbGV0ZSBnZW5vbWUgY292ZXJhZ2UgaW4gUGludXMgdGFlZGE7IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9OTg7ICBjaXRhdGlvbl9wYWdlcz0xMjc5LTEyOTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBkZXZlbG9wbWVudCBvZiBhIGdlbmV0aWMgbWFwIGZvciBtZWFkb3dmb2FtIGNvbXByaXNlZCBvZiBhbXBsaWZpZWQgZnJhZ21lbnQgbGVuZ3RoIHBvbHltb3JwaGlzbXM7IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MTA0OyAgY2l0YXRpb25fcGFnZXM9OTItOTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgZ2VuZXRpYyBsaW5rYWdlIG1hcCBvZiBzd2VldHBvdGF0byBbSXBvbW9lYSBiYXRhdGFzIChMLikgTGFtLl0gYmFzZWQgb24gQUZMUCBtYXJrZXJzOyBNb2wuIEJyZWVkLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTE2OS0xODU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxvdHVzIGphcG9uaWN1cywgYW4gYXV0b2dhbW91cywgZGlwbG9pZCBsZWd1bWUgc3BlY2llcyBmb3IgY2xhc3NpY2FsIGFuZCBtb2xlY3VsYXIgZ2VuZXRpY3M7IFBsYW50IEouOyAgY2l0YXRpb25feWVhcj0xOTkyOyAgY2l0YXRpb25fdm9sdW1lPTI7ICBjaXRhdGlvbl9wYWdlcz00ODctNDk2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DbGFzc2ljYWwgYW5kIG1vbGVjdWxhciBnZW5ldGljcyBvZiB0aGUgbW9kZWwgbGVndW1lIExvdHVzIGphcG9uaWN1czsgTW9sLiBQbGFudCBNaWNyb2JlIEludGVyYWN0LjsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTU5LTY4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUgYW5kIGNocm9tb3NvbWUgZGltZW5zaW9ucyBvZiBMb3R1cyBqYXBvbmljdXM7IEouIFBsYW50IFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MTEzOyAgY2l0YXRpb25fcGFnZXM9NDM1LTQ0MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lIGFuYWx5c2lzIG9mIExvdHVzIGphcG9uaWN1czsgSi4gUGxhbnQgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTM7ICBjaXRhdGlvbl9wYWdlcz00OTctNTA2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IaWdoIGZyZXF1ZW5jeSB0cmFuc2Zvcm1hdGlvbiBhbmQgcmVnZW5lcmF0aW9uIG9mIHRyYW5zZ2VuaWMgcGxhbnRzIGluIHRoZSBtb2RlbCBsZWd1bWUgTG90dXMgamFwb25pY3VzOyBKLiBFeHAuIEJvdC47ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9NDg7ICBjaXRhdGlvbl9wYWdlcz0xMzU3LTEzNjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByb3ZpZGluZyB0aGUgYmFzaXMgZm9yIGdlbm9taWNzIGluIExvdHVzIGphcG9uaWN1czogdGhlIGFjY2Vzc2lvbnMgTWl5YWtvamltYSBNRy0yMCBhbmQgR2lmdSBCLTEyOSBhcmUgYXBwcm9wcmlhdGUgY3Jvc3NpbmcgcGFydG5lcnMgZm9yIGdlbmV0aWMgYW5hbHlzZXM7IE1vbC4gR2VuLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MjY2OyAgY2l0YXRpb25fcGFnZXM9MTU3LTE2NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29uc3RydWN0aW9uIG9mIGEgZ2VuZXRpYyBsaW5rYWdlIG1hcCBvZiB0aGUgbW9kZWwgbGVndW1lIExvdHVzIGphcG9uaWN1cyB1c2luZyBhbiBpbnRyYXNwZWNpZmljIEYyIHBvcHVsYXRpb247IEROQSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz0zMDEtMzEwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGdlbmV0aWMgbGlua2FnZSBtYXAgb2YgdGhlIG1vZGVsIGxlZ3VtZSBMb3R1cyBqYXBvbmljdXMgYW5kIHN0cmF0ZWdpZXMgZm9yIGZhc3QgbWFwcGluZyBvZiBuZXcgbG9jaTsgR2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MTYxOyAgY2l0YXRpb25fcGFnZXM9MTY3My0xNjgzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdHJ1Y3R1cmFsIGFuYWx5c2lzIG9mIGEgTG90dXMgamFwb25pY3VzIGdlbm9tZS4gSS4gU2VxdWVuY2UgZmVhdHVyZXMgYW5kIG1hcHBpbmcgb2YgZmlmdHktc2l4IFRBQyBjbG9uZXMgd2hpY2ggY292ZXIgdGhlIDUuNCBNYiByZWdpb25zIG9mIHRoZSBnZW5vbWU7IEROQSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz0zMTEtMzE4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Kb2luTWFwwq4gMy4wLCBTb2Z0d2FyZSBmb3IgdGhlIGNhbGN1bGF0aW9uIG9mIGdlbmV0aWMgbGlua2FnZSBtYXBzLCBQbGFudCBSZXNlYXJjaCBJbnRlcm5hdGlvbmFsOyBjaXRhdGlvbl9wdWJsaXNoZXI9dGhlIE5ldGhlcmxhbmRzLCwgV2FnZW5pbmdlbjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGVzdGltYXRpb24gb2YgbWFwIGRpc3RhbmNlcyBmcm9tIHJlY29tYmluYXRpb24gdmFsdWVzOyBBbm4uIEV1Z2VuLjsgIGNpdGF0aW9uX3llYXI9MTk0NDsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTE3Mi0xNzU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1hcENoYXJ0OiBTb2Z0d2FyZSBmb3IgdGhlIGdyYXBoaWNhbCBwcmVzZW50YXRpb24gb2YgbGlua2FnZSBtYXBzIGFuZCBRVExzOyBKLiBIZXJlZC47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9OTM7ICBjaXRhdGlvbl9wYWdlcz03Ny03ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJlY2lzaW9uIG1hcHBpbmcgb2YgcXVhbnRpdGF0aXZlIHRyYWl0IGxvY2k7IEdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0xOTk0OyAgY2l0YXRpb25fdm9sdW1lPTEzNjsgIGNpdGF0aW9uX3BhZ2VzPTE0NTctMTQ2NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9V2luZG93cyBRVEwgQ2FydG9ncmFwaGVyIDIuNTsgIGNpdGF0aW9uX3VybD1odHRwOi8vc3RhdGdlbi5uY3N1LmVkdS9xdGxjYXJ0L1dRVExDYXJ0Lmh0bTsgY2l0YXRpb25fcHVibGlzaGVyPURlcGFydG1lbnQgb2YgU3RhdGlzdGljcywgTm9ydGggQ2Fyb2xpbmEgU3RhdGUgVW5pdmVyc2l0eSwgUmFsZWlnaCwsIE5DOyAgY2l0YXRpb25feWVhcj0yMDA2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IaWdoLWRlbnNpdHkgbW9sZWN1bGFyIGxpbmthZ2UgbWFwcyBvZiB0aGUgdG9tYXRvIGFuZCBwb3RhdG8gZ2Vub21lczsgR2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTE5OTI7ICBjaXRhdGlvbl92b2x1bWU9MTMyOyAgY2l0YXRpb25fcGFnZXM9MTE0MS0xMTYwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Vc2Ugb2YgbG9jdXMtc3BlY2lmaWMgQUZMUCBtYXJrZXJzIHRvIGNvbnN0cnVjdCBhIGhpZ2gtZGVuc2l0eSBtb2xlY3VsYXIgbWFwIGluIGJhcmxleTsgVGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT05NjsgIGNpdGF0aW9uX3BhZ2VzPTM3Ni0zODQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuIGludGVncmF0ZWQgUkZMUC1BRkxQIGxpbmthZ2UgbWFwIGZvciBjdWx0aXZhdGVkIHN1bmZsb3dlcjsgR2Vub21lOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTQ0OyAgY2l0YXRpb25fcGFnZXM9MjEzLTIyMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QUZMUCBtYXBzIG9mIFBldHVuaWEgaHlicmlkYTogYnVpbGRpbmcgbWFwcyB3aGVuIG1hcmtlcnMgY2x1c3RlcjsgVGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMDU7ICBjaXRhdGlvbl9wYWdlcz0xMDAwLTEwMDk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJGTFAtYmFzZWQgZ2VuZXRpYyBtYXAgb2YgdGhlIGhvbW9lb2xvZ291cyBncm91cC0zIGNocm9tb3NvbWVzIG9mIHdoZWF0IGFuZCByeWU7IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTE5OTI7ICBjaXRhdGlvbl92b2x1bWU9ODM7ICBjaXRhdGlvbl9wYWdlcz05MzEtOTM5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMTUvNS8zMjMvMTAzMDY5MS9kc24wMjIucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIEEgY29sbGVjdGlvbiBvZiA5NCBGNiBpbmRpdmlkdWFscyBkZXJpdmVkIGZyb20gY3Jvc3NlcyBiZXR3ZWVuIExvdHVzIGphcG9uaWN1cywgR2lmdSBCLTEyOSAoRykgYW5kIE1peWFrb2ppbWEgTUctMjAgKE0pIHdlcmUgdXNlZCBmb3IgbWFwcGluZy4gQnkgdXNpIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFci8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj0zNTk5OTUmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2Rk7vEAn/1oJdaErJR+5kxjS9SgVz0=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f15%2f5%2f323%2f359995" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f15%2f5%2f323%2f359995" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=13976" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/15/5"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/15/5/1/m_cover.gif?Expires=2147483647&amp;Signature=w0UhL1170rQznOI4EBmUQXu1ZOwiDy5D9yvI-xItfA5gEGy2mkrpZiqzDFtXeoJ~7GciRt1Jb~wykd96BMcXNK2oJMQKCvan9OGOux6ILlOeg1peDGvihwkEZeSlIEQRWsuy-Os5b2zaUJLN2EMIjSPATvDcEQeiR8VJm~tHNehbKW~jZGwxEfbLVfvvbsN-HCrfd6zxxRmciZJddcZFQjaH25Oc8zwhd9TcubBub~UwfgV64C6hjwxGjWayYeyC6503xComjsunKItnp20PuervJsOqbPEquDoZ8SVxa8ULqEXM-U2u8h~RKynzXZGyUfUdPZZhfeDp4GDC653uKg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/15/5">
                        <div class="volume trailing-comma">Volume 15</div>
                        <div class="issue">Issue 5</div>

            </a>

        </div>
        <div class="ii-pub-date">
October 2008        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="5193856">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5193856">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5193858">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5193858">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5193862">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5193862">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5193878">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5193878">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5193903">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5193903">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5193914">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5193914">Supplementary Data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5193916">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5193916">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5193918">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5193918">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/15/5/309/359388">&lt; Previous</a>
        </li>        
            <li class="empty arrow">&nbsp;</li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">A High-density Linkage Map of <em>Lotus japonicus</em> Based on AFLP and SSR Markers
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Xinwang Wang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">National Institute of Agrobiological Sciences</div>, <div class="addr-line">Kannon-dai 2-1-2, Tsukuba, Ibaraki 305-8602</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +1 865-974-1070. Fax. <div class="fax">+1 865-974-4744</div>. E-mail: <a href="mailto:xinwangwang@hotmail.com" target="_blank">xinwangwang@hotmail.com</a> (X.W.) or <a href="mailto:kawasa@nias.affrc.go.jp" target="_blank">kawasa@nias.affrc.go.jp</a> (S.K.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Xinwang+Wang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Wang X">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Wang X&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Xinwang Wang</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shusei Sato</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shusei+Sato">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sato S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sato S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Shusei Sato</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Satoshi Tabata</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Satoshi+Tabata">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tabata S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tabata S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Satoshi Tabata</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shinji Kawasaki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">National Institute of Agrobiological Sciences</div>, <div class="addr-line">Kannon-dai 2-1-2, Tsukuba, Ibaraki 305-8602</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +1 865-974-1070. Fax. <div class="fax">+1 865-974-4744</div>. E-mail: <a href="mailto:xinwangwang@hotmail.com" target="_blank">xinwangwang@hotmail.com</a> (X.W.) or <a href="mailto:kawasa@nias.affrc.go.jp" target="_blank">kawasa@nias.affrc.go.jp</a> (S.K.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shinji+Kawasaki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kawasaki S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kawasaki S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Shinji Kawasaki</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Edited by Katsumi Isono</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="FN1"><div class="fn"><div class="label -label">â </div><div class="footnote-content">Present address: The University of Tennessee, Knoxville, TN 37996, USA.</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 15, Issue 5, 1 October 2008, Pages 323â332, <a href='https://doi.org/10.1093/dnares/dsn022'>https://doi.org/10.1093/dnares/dsn022</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">14 September 2008</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">18 June 2008</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">18 August 2008</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="359995" href="/dnaresearch/article-pdf/15/5/323/1030691/dsn022.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Xinwang Wang, Shusei Sato, Satoshi Tabata, Shinji Kawasaki;  A High-density Linkage Map of <em>Lotus japonicus</em> Based on AFLP and SSR Markers, <em>DNA Research</em>, Volume 15, Issue 5, 1 October 2008, Pages 323â332, <a href='https://doi.org/10.1093/dnares/dsn022'>https://doi.org/10.1093/dnares/dsn022</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/359995?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/359995?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/359995?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/359995?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/359995?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/359995?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/359995?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=A%20High-density%20Linkage%20Map%20of%20%26lt%3Bem%26gt%3BLotus%20japonicus%26lt%3B%2Fem%26gt%3B%20Based%20on%20AFLP%20and%20SSR%20Markers&amp;publicationDate=2008-09-14&amp;volumeNum=15&amp;issueNum=5&amp;author=Wang%2C%20Xinwang%3B%20Sato%2C%20Shusei&amp;startPage=323&amp;endPage=332&amp;contentId=10.1093%2Fdnares%2Fdsn022&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f15%2f5%2f323%2f359995" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=5193856 id="5193856" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>A collection of 94 F<sub>6</sub> individuals derived from crosses between <em>Lotus japonicus</em>, Gifu B-129 (G) and Miyakojima MG-20 (M) were used for mapping. By using the HEGS running system, 427 <em>Eco</em>RI/<em>Mse</em>I primer pairs were selected to generate a total of 2053 markers, consisting of 739 G-associated dominant markers, 674 M-associated dominant markers, 640 co-dominant markers, 95 SSR markers and 2 dCAPS markers. Excluding heavily distorted markers, 1588 were mapped to six chromosomes of the <em>L</em>. <em>japonicus</em> genome based on the 97 reference markers. This linkage map consisted of 1023 unique markers (excluding duplicated markers) and covered a total of 508.5 cM of the genome with an average chromosome length of 84.7 cM and interval distance of 0.50 cM. Fifteen quantitative traits loci for eight morphological traits were also mapped. This linkage map will provide a useful framework for physical map construction in <em>L</em>. <em>japonicus</em> in the near future.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Lotus japonicus&quot;"><em>Lotus japonicus</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="AFLP">AFLP</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="SSR">SSR</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;linkage map&quot;">linkage map</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;HEGS (high efficiency genome scanning)&quot;">HEGS (high efficiency genome scanning)</a></div>                            <h2 scrollto-destination=5193858 id="5193858" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Genetic mapping is a basic tool of eukaryotic genomic research. Molecular linkage maps provide information about the organization of the genome and can be used for genetic studies and breeding applications. A high-density genetic linkage map is essential to physical map construction and also a powerful tool for the location and map-based cloning of desired gene(s). PCR-based DNA markers make such a linkage map possible. Of these PCR-based markers, AFLP markers were demonstrated to be a powerful new class of markers making it feasible to develop linkage maps for plants with a large genome.<sup><a reveal-id="DSN022C1" data-open="DSN022C1" class="link link-ref link-reveal xref-bibr">1</a></sup> There are many linkage maps based on AFLP markers reported for various plants.<sup><a reveal-id="DSN022C2  DSN022C6" data-open="DSN022C2  DSN022C6" class="link link-ref link-reveal xref-bibr">2â6</a></sup></p><p><em>Lotus japonicus</em> is an autogamous diploid legume species.<sup><a reveal-id="DSN022C7" data-open="DSN022C7" class="link link-ref link-reveal xref-bibr">7</a></sup> As a model legume, it has many characteristics that make it a candidate for genomic research. These attributes are an autogamous diploid (2<em>n</em> = 12) and small genome (432â¼494 Mb),<sup><a reveal-id="DSN022C8  DSN022C10" data-open="DSN022C8  DSN022C10" class="link link-ref link-reveal xref-bibr">8â10</a></sup> short life cycle and transformation ability,<sup><a reveal-id="DSN022C6" data-open="DSN022C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DSN022C11" data-open="DSN022C11" class="link link-ref link-reveal xref-bibr">11</a></sup> making <em>L</em>. <em>japonicus</em> a model legume plant that can be used for molecular genetics and physiological studies. Genome synteny will help in marker preparation and gene cloning for other legume crops. To date, the cloning of various nodulation genes has been the subject of heated international competition. Therefore, the need for construction of a physical map covering the genome of <em>L</em>. <em>japonicus</em> is especially important. A primary genetic linkage map based on DAF (DNA Amplification Fingerprinting) markers with an F<sub>2</sub> population from a cross of <em>L</em>. <em>japonicus</em> accessions, Gifu B-129 and Funakura B-581 has been reported.<sup><a reveal-id="DSN022C8" data-open="DSN022C8" class="link link-ref link-reveal xref-bibr">8</a></sup> However, due to low polymorphism observed in this cross, the number of linkage groups did not cover all chromosomes of the genome. Crosses made with âMiyakojima MG-20â showed the highest-level of polymorphisms relative to Gifu B-129 (&gt;4%).<sup><a reveal-id="DSN022C12" data-open="DSN022C12" class="link link-ref link-reveal xref-bibr">12</a></sup> Although this recombination rate is still low when compared with other plants,<sup><a reveal-id="DSN022C2  DSN022C5" data-open="DSN022C2  DSN022C5" class="link link-ref link-reveal xref-bibr">2â5</a></sup> a fast and simple high efficiency genome scanning (HEGS)<sup><a reveal-id="DSN022C10" data-open="DSN022C10" class="link link-ref link-reveal xref-bibr">10</a></sup> AFLP protocol system can overcome this disadvantage. Although traditional AFLP protocols simultaneously assay for large numbers of polymorphic bands on a single gel, developing more than 2000 AFLP markers is time consuming and laborious. The HEGS system allows the development of AFLP markers in a short time.<sup><a reveal-id="DSN022C10" data-open="DSN022C10" class="link link-ref link-reveal xref-bibr">10</a></sup> The HEGS gel running apparatus is composed of a set of 24 cm Ã 26 cm glass plates. One hundred samples can be analyzed on a two-layer gel that is composed of 13% bis: arcylamide (19:1) separating gel and 5% stocking gel at 350 V for 4 h. As a result, 800 individuals can be analyzed on eight sets of plates per day by one person. By using the HEGS/AFLP system, some linkage maps have been developed for <em>L</em>. <em>japonicus</em>.<sup><a reveal-id="DSN022C13" data-open="DSN022C13" class="link link-ref link-reveal xref-bibr">13</a>,<a reveal-id="DSN022C14" data-open="DSN022C14" class="link link-ref link-reveal xref-bibr">14</a></sup> For example, Hayashi et al.<sup><a reveal-id="DSN022C13" data-open="DSN022C13" class="link link-ref link-reveal xref-bibr">13</a></sup> constructed a linkage map consisting of 287 markers (AFLP, SSR, dCAPS and other PCR-based markers) that spanned a total length of 487.3 cM and corresponded to six chromosomes in the <em>Lotus</em> genome using an F<sub>2</sub> population from a cross of âGifu B-129â and âMiyakojima MG-20â. This study will use the map of Hayashi et al.<sup><a reveal-id="DSN022C13" data-open="DSN022C13" class="link link-ref link-reveal xref-bibr">13</a></sup> as reference to build a high-density linkage map in <em>L</em>. <em>japonicus</em>.</p><p>To construct a fine physical map, however, requires a linkage map of <em>L</em>. <em>japonicus</em> with sufficient high marker density (&gt;1000 markers with less than 1 cM interval distance). This linkage map will also be used for gene cloning in future research. The resulting high-density linkage map created in this study will serve as a framework for building a genome physical map that will be suitable for map-based cloning in <em>L</em>. <em>japonicus</em> genetic research.</p>                            <h2 scrollto-destination=5193862 id="5193862" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=5193863 id="5193863" class="section-title" data-legacy-id=s2a>2.1. Plant materials and DNA extraction</h3>
<p>A cross of <em>L</em>. <em>japonicus</em> accessions âGifu B-129â and âMiyakojima MG-20â was made at Kazusa DNA Institute, Japan. Gifu B-129 has a crawling habit with a red stem and Miyakojima MG-20 has erect habit with a green stem. A mapping population of 194 F<sub>6</sub> recombinant inbred lines (RILs) was obtained by single-seed descent method. A collection of 94 from the 194 RILs was randomly selected as a mapping population for convenient analysis of the HEGS/AFLP running system.</p><p>Genomic DNA was extracted from young leaves of the 94 individuals. In brief, 0.3 g of young leaves were collected in a 50 ml Falcon tube including five metal beads, immersed in liquid nitrogen and ground with a shaker (EYELA cute Mixer CM-100), 250 rpm, 30 s, twice. Immediately, the powder was incubated with 2 ml extraction buffer (10% CTAB: 10% SDS and sodium lanroylsarcosine = 1:2, preheated) at 65Â°C for 1 h with gentle shaking. About 2 ml of aqueous phase were transferred to a 6-tube strips and DNA was extracted automatically using KURABO NA-2000 (Japan) following the manufacturer's instructions. The DNA pellet was dried and dissolved in 50â100 Âµl 0.1 Ã TE (10 mM TrisâHCl and 0.1 mM EDTA, pH 8.0). The dissolved DNA samples were treated with RNase A (50 Âµg/ml) for 3 h at 37Â°C.</p>                            <h3 scrollto-destination=5193866 id="5193866" class="section-title" data-legacy-id=s2b>2.2. SSR analysis</h3>
<p>Microsatellite (SSRs) and dCAPS analysis were performed based on the methods of Sato et al.<sup><a reveal-id="DSN022C15" data-open="DSN022C15" class="link link-ref link-reveal xref-bibr">15</a></sup> with the minor modifications. Genomic DNA (0.2 ng) was used in a total volume of 5 Âµl containing 0.0125 U Extaq polymerase (TaKaRa, Japan). The annealing temperature was set to 60Â°C. The sequence information of all SSR and dCAPS primers (written in TM prefix with serial number) can be found in Sato et al.<sup><a reveal-id="DSN022C15" data-open="DSN022C15" class="link link-ref link-reveal xref-bibr">15</a></sup> All the TM primers were synthesized from Invitrogen, Life Technologies, Japan.</p>                            <h3 scrollto-destination=5193868 id="5193868" class="section-title" data-legacy-id=s2c>2.3. AFLP analysis</h3>
<p>The AFLP assays were performed as described by Vos et al.<sup><a reveal-id="DSN022C1" data-open="DSN022C1" class="link link-ref link-reveal xref-bibr">1</a></sup> with the following modifications. Genomic DNA samples (250 ng) were digested with 8 U of <em>Eco</em>RI and 5 U of <em>Mse</em>I (Biolabs Inc., New England) in a reaction volume of 25 Âµl with 1Ã <em>Not</em> I preservation buffer (10 mM TrisâHCl, pH 7.5, 100 mM KCl, 1 mM DTT, 0.1 mM EDTA, pH 8.0, 0.01% BSA, 0.15% Triton X-100, 50% glycerol), mixed briefly and incubated at 37Â°C for 3 h. Five microliters of the digestion solution were mixed with 1 Âµl <em>Eco</em>RI adapter (5 pmol/Âµl), 1 Âµl <em>Mse</em>I adapter (50 pmol/Âµl), 1 Âµl 10Ã ligation buffer and 28 U of T4 DNA ligase (TaKaRa) in a total of 10 Âµl of reaction volume and incubated at 37Â°C overnight. This digestionâligation solution was diluted 10-fold in 0.1Ã TE buffer and used for pre-amplification. Pre-amplification was conducted in 25 Âµl reaction volume containing 2.5 Âµl diluted adaptor-ligated DNA, 1 Âµl <em>Eco</em>RI+A primer (5 ng/Âµl), 1 Âµl <em>Mse</em>I+C primer (30 ng/Âµl), 2 Âµl 2.5 mM dNTP each, 2.5 Âµl ExTaq buffer and 0.5 U ExTaq DNA polymerase (TaKaRa). The PCR profile was 94Â°C for 30 s, 56Â°C for 1 min, 72Â°C for 1 min and 20 cycles with a 10 min final extension at 72Â°C. The PCR products were diluted 100-fold in 0.1Ã TE buffer and stored at â20Â°C until further use. Selective amplification was performed in 5 Âµl reaction volume containing 2.45 Âµl diluted pre-amplification products, 0.5 Âµl <em>Eco</em>RI +3 primer (7 ng/Âµl) and 1.125 Âµl <em>Mse</em>I+3 primer (7 ng/Âµl), 0.4 Âµl 2.5 mM dNTP each, 0.5 Âµl ExTaq buffer and 0.025 U ExTaq DNA polymerase (TaKaRa). The touchdown PCR profile was one cycle at 94Â°C for 30 s, 68Â°C for 30 s and 72Â°C for 60 s, 17 cycles with the annealing temperature reduced 0.7Â°C/cycle, and 23 cycles with an annealing temperature of 56Â°C and with final extension at 72Â°C for 10 min.</p>                            <h3 scrollto-destination=5193870 id="5193870" class="section-title" data-legacy-id=s2d>2.4. Gel running with HEGS running system</h3>
<p>Five microliters of PCR samples were mixed with 1 Âµl loading buffer (0.25% xylene cyanol, 0.25% bromophenol blue, 1 mM EDTA, pH 8.0, 40% glycerol) and separated using the HEGS running system.<sup><a reveal-id="DSN022C5" data-open="DSN022C5" class="link link-ref link-reveal xref-bibr">5</a></sup> In brief, 6 Âµl of each sample per lane were loaded into polyacrylamide gels consisting of stacking gel (upper-side gel, 2.5 cm in depth, 5% bis-polyacrylamide contained 0.5 M TrisâHCl, pH 6.8) and running gel (lower-side gel, 13% bis-polyacrylamide contained 1.5 M TrisâHCl, pH 8.8). A total of 94 F<sub>6</sub> individuals and two parents and <em>Î¦</em>X174-<em>Hae III</em> (50 ng/lane) in one gel was run in 1Ã Trisâglycine buffer (25 mM TrisâHCl, pH 8.3, 1.92 M glycine) at 100 V for 80 min followed 350 V for 4.5 h. The gels were stained in 1/10 000 volume Vistra Green dye solution (Amersham Pharmacia Biotech) for 10â20 min, washed in water for 5â10 min and scanned in FluorImager 575 (Molecular Dynamics, Amersham Pharmacia Biotech). We therefore employed the HEGS/AFLP analysis with 94 F<sub>6</sub> recombination lines from a cross of âGifu B-129â and âMiyakojima MG-20â to develop a large number of markers. This approach attempts to construct a high-density linkage map containing â¼2000 markers with an average of &lt;1 cM interval distance between markers.</p>                            <h3 scrollto-destination=5193872 id="5193872" class="section-title" data-legacy-id=s2e>2.5. Data collection</h3>
<p>Clear polymorphic bands were selected using the F<sub>2</sub> generation genome analysis software (Kazusa DNA Research Institute, Japan), and the bands were confirmed visually. This software also automatically assigned molecular weights to the fragments, distinguished the single polymorphic band from parents before manually setting paternal and maternal alleles, and generated reports of fragment presence/absence in 1/0 binary type. Only clearly visible markers were scored. For F<sub>6</sub> progeny, band presence associated with the Gifu B-129 allele was coded as A; band presence with Miyakojima MG-20 allele was coded as B, and those bands with both female and male parent were coded as H for heterozygote. Each AFLP marker was identified by a code referring to the primer combination, EM (<em>Eco</em>RI/<em>Mse</em>I) and character G (associated with Gifu B-129 allele), M (associated with Miyakojima MG-20 allele) or C (co-dominant marker), followed by the estimated size of the DNA fragment in nucleotides. The heterozygote (for the co-dominant markers) and missing data were coded as â-â.</p><p>Eight morphological traits, plant type (PT), stem pigment (SP), leaflet (LL), stipule (ST), petiole (PE), trichome (TR), seed color (SC) and seed size (SS), were scored as qualitative traits. We scored grade 1â5 for all traits. PT: 1âcrawl stem and 5âstand stem; SP: 1âlight color (green) stem and 5âdark color (red) stem; LF: 1âno LL and 5âlong LL; ST: 1âno ST on the petal and 5âmost STs; PE: 1ânone and 5âlongest; TR: 1âno TR and 5âmost; SC: 1âlightest color and 5âdark (brown); SS: 1âsmallest and 5âlargest.</p>                            <h3 scrollto-destination=5193875 id="5193875" class="section-title" data-legacy-id=s2f>2.6. Linkage map construction</h3>
<p>Before linkage analysis, chi-square tests (<em>Ï</em><sup>2</sup>) were performed on both SSR and AFLP markers for goodness of fit to the expected Mendelian 1:1 segregation ratio of each marker. Distorted loci that deviated significantly at <em>P</em> &lt; 0.01 were excluded from map construction. Linkage analysis was performed with the Joinmap 3.0.<sup><a reveal-id="DSN022C16" data-open="DSN022C16" class="link link-ref link-reveal xref-bibr">16</a></sup> Initially, an LOD score of 14.0 was used to identify six linkage groups, corresponding to six chromosomes of <em>Lotus</em> genome based on the previously mapped SSR/dCAPS markers<sup><a reveal-id="DSN022C13" data-open="DSN022C13" class="link link-ref link-reveal xref-bibr">13</a></sup> (details can also be found from <a class="link link-uri" href="http://www.kazusa.or.jp/lotus/markerdb_index.html">http://www.kazusa.or.jp/lotus/markerdb_index.html</a>). The Kosambi mapping function<sup><a reveal-id="DSN022C17" data-open="DSN022C17" class="link link-ref link-reveal xref-bibr">17</a></sup> was used to convert recombination frequencies into map distances. Linkage maps were drawn using MapChart 2.1 software.<sup><a reveal-id="DSN022C18" data-open="DSN022C18" class="link link-ref link-reveal xref-bibr">18</a></sup></p><p>A subset of markers spanned across the linkage map with even distance of 5 cM was selected and used for composite interval mapping.<sup><a reveal-id="DSN022C19" data-open="DSN022C19" class="link link-ref link-reveal xref-bibr">19</a></sup> Quantitative traits loci (QTL) analysis was carried out using WinQTL Cartographer 2.5.<sup><a reveal-id="DSN022C20" data-open="DSN022C20" class="link link-ref link-reveal xref-bibr">20</a></sup></p>                            <h2 scrollto-destination=5193878 id="5193878" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=5193880 id="5193880" class="section-title" data-legacy-id=s3a>3.1. AFLP markers generated in HEGS system</h3>
<p>Ninety-four F<sub>6</sub> individuals and two parents were analyzed on a single gel (Fig.Â <a reveal-id="DSN022F1" data-open="DSN022F1" class="link link-reveal link-table xref-fig">1</a>); 4096 <em>Eco</em>RIâ<em>Mse</em>I primer combinations were first screened on the parental DNAs (data not shown). Of the primer pairs (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplement 1</a>) that produced the most polymorphic bands, 427 were selected for further AFLP analysis for 94 random individuals of F<sub>6</sub> population. Initially, a total of 2053 diagnostic AFLP markers, which included 739 Gifu B-129 dominant, 674 Miyakojima MG-20 dominant and 640 co-dominant markers, were scored (Fig.Â <a reveal-id="DSN022F2" data-open="DSN022F2" class="link link-reveal link-table xref-fig">2</a>, TableÂ <a reveal-id="DSN022TB1" data-open="DSN022TB1" class="link link-reveal link-table xref-fig">1</a>). On average, 4.8 AFLP markers and 1.5 co-dominant markers were generated per primer combination, with a range of 2â8 visible markers (data not shown). The polymorphism ratio per primer pair combination agreed with that of Kawaguchi et al.<sup><a reveal-id="DSN022C12" data-open="DSN022C12" class="link link-ref link-reveal xref-bibr">12</a></sup> The electrophoresis profile (Fig.Â <a reveal-id="DSN022F1" data-open="DSN022F1" class="link link-reveal link-table xref-fig">1</a>) scanned by the FluorImager 575 provided sufficient resolution to distinguish fragment mobility from 70 to 5000 bp, indicating a wide range of the amplification fragments with high resolution using the HEGS/AFLP-SSR running system (in â¼24 cm long gel). A total of 95 SSR and 2 dCAPS markers that mapped on the previous linkage map<sup><a reveal-id="DSN022C13" data-open="DSN022C13" class="link link-ref link-reveal xref-bibr">13</a></sup> was also separated using HEGS running system.</p>                        <a id="5193883" scrollto-destination="5193883"></a>
<div id="DSN022F1" data-id="DSN022F1" class="fig fig-section"><div class="label fig-label">FigureÂ 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn022/2/m_dsn02201.jpeg?Expires=2147483647&Signature=VElFYUNc~fshv0ACgSOkdm3Uvl8atBiy1N5ABl2DvtgCT5ZPLNIVQKZOUfhQKULH26Hg6ZN9lzCNLUFUNItX2-auKRXbfsI5F9mKhfReTkU9aIwutLJVSTAcPRpLG4dyDJKvg69fx11dKQss-gv7hgQKbc9t0sFuiFIy09m57hIQq0nCvP50gSTlCf3SVV1v4jyqsMbjfMyjU8EwwrHF3pLtIOiuRg80HOlmI7obgtFkXe5WXcQAUnrFIs2EmAcb1P0BQu5h2o54UNv5iQFJJmSdZxZKVOBluQlJJPkjPbB4YJvmRHPU0dUf5OBngbUOw3BU25SFk3OKvmkyFkFX5Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="A sample of AFLP gel profile with a combination of E-ATG I/M-ACC primers for 94 F6 individuals from a cross of L. japonicus accessions, âGifu B-129â and âMiyakojima MG-20â with HEGS system. Lanes from left: lane 1, size marker Î¦X174-Hae III; lane 2, maternal âGifu B-129â (associated allele was assigned as âGâ, showed as arrow); lane 3, paternal âMiyakojima MG-20â (associated allele was assigned as âMâ, showed as arrow); lanes 4â97, F6 individuals. Co-dominant allele was assigned as âCâ, showed as arrow." path-from-xml="dsn02201.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5193883/dsn02201.jpeg" path-from-xml="dsn02201.jpeg" target="_blank">View large</a><a section="5193883" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn022/2/dsn02201.jpeg?Expires=2147483647&Signature=r5pC8CpOl9CFP3E0otwAybdO7AviZkyTeTyGwbTeJkN0vh3niNltj5DjGnvrofwEAY3ZOpDT4fTvvzxWF19yHHIUtKYKS2RJkDoGdBtLUj8EwKJFqqvU5TiC0vdmCRvYjdXw116ifSnEYBwTZH0PwJXRJIbISCmlkpVjkRYuL-vGfL7JhIPQ2uf9fg3VxOVj1sjm0DD276oWY4vqwaMDH7fdMkD1lWOE7-wfolbZTWj1vTLg79a3FeddB4LMVJxeX6xq8lB0MDc1Lxz9eLbtf5yNcMeqFLXaPEHRZAZRl0dGXaLTiowxJ30jq~83GqqvG8-kqH6k4Hb4kzenAxzSwQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5193883&ar=359995&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn02201.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>A sample of AFLP gel profile with a combination of E-ATG I/M-ACC primers for 94 F<sub>6</sub> individuals from a cross of <em>L</em>. <em>japonicus</em> accessions, âGifu B-129â and âMiyakojima MG-20â with HEGS system. Lanes from left: lane 1, size marker <em>Î¦</em>X174-<em>Hae</em> III; lane 2, maternal âGifu B-129â (associated allele was assigned as âGâ, showed as arrow); lane 3, paternal âMiyakojima MG-20â (associated allele was assigned as âMâ, showed as arrow); lanes 4â97, F<sub>6</sub> individuals. Co-dominant allele was assigned as âCâ, showed as arrow.</p></div></div><div content-id="DSN022F1" class="fig fig-modal reveal-modal"><div class="label fig-label">FigureÂ 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn022/2/m_dsn02201.jpeg?Expires=2147483647&Signature=VElFYUNc~fshv0ACgSOkdm3Uvl8atBiy1N5ABl2DvtgCT5ZPLNIVQKZOUfhQKULH26Hg6ZN9lzCNLUFUNItX2-auKRXbfsI5F9mKhfReTkU9aIwutLJVSTAcPRpLG4dyDJKvg69fx11dKQss-gv7hgQKbc9t0sFuiFIy09m57hIQq0nCvP50gSTlCf3SVV1v4jyqsMbjfMyjU8EwwrHF3pLtIOiuRg80HOlmI7obgtFkXe5WXcQAUnrFIs2EmAcb1P0BQu5h2o54UNv5iQFJJmSdZxZKVOBluQlJJPkjPbB4YJvmRHPU0dUf5OBngbUOw3BU25SFk3OKvmkyFkFX5Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="A sample of AFLP gel profile with a combination of E-ATG I/M-ACC primers for 94 F6 individuals from a cross of L. japonicus accessions, âGifu B-129â and âMiyakojima MG-20â with HEGS system. Lanes from left: lane 1, size marker Î¦X174-Hae III; lane 2, maternal âGifu B-129â (associated allele was assigned as âGâ, showed as arrow); lane 3, paternal âMiyakojima MG-20â (associated allele was assigned as âMâ, showed as arrow); lanes 4â97, F6 individuals. Co-dominant allele was assigned as âCâ, showed as arrow." path-from-xml="dsn02201.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5193883/dsn02201.jpeg" path-from-xml="dsn02201.jpeg" target="_blank">View large</a><a section="5193883" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn022/2/dsn02201.jpeg?Expires=2147483647&Signature=r5pC8CpOl9CFP3E0otwAybdO7AviZkyTeTyGwbTeJkN0vh3niNltj5DjGnvrofwEAY3ZOpDT4fTvvzxWF19yHHIUtKYKS2RJkDoGdBtLUj8EwKJFqqvU5TiC0vdmCRvYjdXw116ifSnEYBwTZH0PwJXRJIbISCmlkpVjkRYuL-vGfL7JhIPQ2uf9fg3VxOVj1sjm0DD276oWY4vqwaMDH7fdMkD1lWOE7-wfolbZTWj1vTLg79a3FeddB4LMVJxeX6xq8lB0MDc1Lxz9eLbtf5yNcMeqFLXaPEHRZAZRl0dGXaLTiowxJ30jq~83GqqvG8-kqH6k4Hb4kzenAxzSwQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5193883&ar=359995&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn02201.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>A sample of AFLP gel profile with a combination of E-ATG I/M-ACC primers for 94 F<sub>6</sub> individuals from a cross of <em>L</em>. <em>japonicus</em> accessions, âGifu B-129â and âMiyakojima MG-20â with HEGS system. Lanes from left: lane 1, size marker <em>Î¦</em>X174-<em>Hae</em> III; lane 2, maternal âGifu B-129â (associated allele was assigned as âGâ, showed as arrow); lane 3, paternal âMiyakojima MG-20â (associated allele was assigned as âMâ, showed as arrow); lanes 4â97, F<sub>6</sub> individuals. Co-dominant allele was assigned as âCâ, showed as arrow.</p></div></div>                        <a id="5193892" scrollto-destination="5193892"></a>
<div id="DSN022F2" data-id="DSN022F2" class="fig fig-section"><div class="label fig-label">FigureÂ 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn022/2/m_dsn02202.jpeg?Expires=2147483647&Signature=AvNzAqShFvFjTjj9v0rGKIp8UgeC3EHGrRKR9Zt0IMRlADqZn6w0NHd7R8bogG6GH2JNKppaNEYhHw3v20BSXDtEugB0c0Swfbs3~BNoeBM3sfXjWA3KvFtcL5NHXPLFJrh7edX9KidjjRtbtHkMpclSiw-n9NQzhIaqhNvk8DLzz6z3-uVE7TLQYoRWPMJUUo3cuv2Ozuo5zq60IP7KobPfpNQ-rB9uRcOPTIE8b2sws4dkaN8TbHxmlgqnh8vgGYUsTZPeJiXbPXyErDTrG8NJrVx6PE9PWJgtg4PKiNIOtQbiQiXutpq2erCFrDJa2ZSUyMZpoqnZdEHf9kiNrw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distributions of 2053 AFLP alleles from 94 F6 individuals derived from a cross of L. japonicus, âGifu B-129â and âMiyakojima MG-20â." path-from-xml="dsn02202.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5193892/dsn02202.jpeg" path-from-xml="dsn02202.jpeg" target="_blank">View large</a><a section="5193892" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn022/2/dsn02202.jpeg?Expires=2147483647&Signature=JftJHIWHlMc6qPCkED-qO0g0TaCLus~IlUb0c4RVmqaf8YxWi1SPxtOeOQdXy2k7JsdDnesJlsyX4ZBwFncqRZKMw0d5-eDvQQk9lbP2WBJRawuKB6WvsXl8pln3PtcN0Jdg6nnDc4rxZ1P628x1rhS9jORtcwp9dLCI1SmPH46KNiea3weixuUB6lq6wHNNZY0Pzk5RaLC6JAfzGkrqHAiT~XhDpKDPq8CsQ5~-3zXt1iCBNj0e3dnPcVGv80NlKNtb53seZoRP68r5W3910pre3YMNFl~29i49e-7fDv93zmqJqS01FFHdF4khGFHYoNa~KO02uP4Ee4wSjzmXUw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5193892&ar=359995&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn02202.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distributions of 2053 AFLP alleles from 94 F<sub>6</sub> individuals derived from a cross of <em>L</em>. <em>japonicus</em>, âGifu B-129â and âMiyakojima MG-20â.</p></div></div><div content-id="DSN022F2" class="fig fig-modal reveal-modal"><div class="label fig-label">FigureÂ 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn022/2/m_dsn02202.jpeg?Expires=2147483647&Signature=AvNzAqShFvFjTjj9v0rGKIp8UgeC3EHGrRKR9Zt0IMRlADqZn6w0NHd7R8bogG6GH2JNKppaNEYhHw3v20BSXDtEugB0c0Swfbs3~BNoeBM3sfXjWA3KvFtcL5NHXPLFJrh7edX9KidjjRtbtHkMpclSiw-n9NQzhIaqhNvk8DLzz6z3-uVE7TLQYoRWPMJUUo3cuv2Ozuo5zq60IP7KobPfpNQ-rB9uRcOPTIE8b2sws4dkaN8TbHxmlgqnh8vgGYUsTZPeJiXbPXyErDTrG8NJrVx6PE9PWJgtg4PKiNIOtQbiQiXutpq2erCFrDJa2ZSUyMZpoqnZdEHf9kiNrw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distributions of 2053 AFLP alleles from 94 F6 individuals derived from a cross of L. japonicus, âGifu B-129â and âMiyakojima MG-20â." path-from-xml="dsn02202.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5193892/dsn02202.jpeg" path-from-xml="dsn02202.jpeg" target="_blank">View large</a><a section="5193892" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn022/2/dsn02202.jpeg?Expires=2147483647&Signature=JftJHIWHlMc6qPCkED-qO0g0TaCLus~IlUb0c4RVmqaf8YxWi1SPxtOeOQdXy2k7JsdDnesJlsyX4ZBwFncqRZKMw0d5-eDvQQk9lbP2WBJRawuKB6WvsXl8pln3PtcN0Jdg6nnDc4rxZ1P628x1rhS9jORtcwp9dLCI1SmPH46KNiea3weixuUB6lq6wHNNZY0Pzk5RaLC6JAfzGkrqHAiT~XhDpKDPq8CsQ5~-3zXt1iCBNj0e3dnPcVGv80NlKNtb53seZoRP68r5W3910pre3YMNFl~29i49e-7fDv93zmqJqS01FFHdF4khGFHYoNa~KO02uP4Ee4wSjzmXUw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5193892&ar=359995&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn02202.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distributions of 2053 AFLP alleles from 94 F<sub>6</sub> individuals derived from a cross of <em>L</em>. <em>japonicus</em>, âGifu B-129â and âMiyakojima MG-20â.</p></div></div>                        <a id="5193893" scrollto-destination="5193893"></a>
<div content-id="DSN022TB1" class="table-modal"><div class="table-wrap"><div id="DSN022TB1" class="table-wrap-title" data-id="DSN022TB1"><div class="label -label">TableÂ 1</div><div class="caption"><p>AFLP and SSR markers generated in the 94 F<sub>6</sub> individuals from a cross between <em>L</em>. <em>japonicus</em> accessions, âGifu B-129â (G) and âMiyakojima MG-20â (M)</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Â </th><th>TotalÂ </th><th>Mapped markersÂ </th><th>Distorted at <em>P</em> â¤ 0.05Â </th><th>Distorted at <em>P</em> â¤ 0.01Â </th></tr></thead><tbody><tr><td><em>Eco</em>RI/<em>Mse</em>I primer pairsÂ </td><td>427Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>G associated dominant markersÂ </td><td>739Â </td><td>504Â </td><td>194Â </td><td>44Â </td></tr><tr><td>M associated dominant markersÂ </td><td>674Â </td><td>454Â </td><td>202Â </td><td>55Â </td></tr><tr><td>Co-dominant markersÂ </td><td>640Â </td><td>533Â </td><td>179Â </td><td>32Â </td></tr><tr><td>Total AFLP markersÂ </td><td>2053Â </td><td>1491Â </td><td>575 (28%)Â </td><td>131 (6.4%)Â </td></tr><tr><td>SSR markers*Â </td><td>97Â </td><td>97Â </td><td>23 (23.7%)Â </td><td>5 (5.1%)Â </td></tr></tbody></table></div><div content-id="DSN022TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Â </th><th>TotalÂ </th><th>Mapped markersÂ </th><th>Distorted at <em>P</em> â¤ 0.05Â </th><th>Distorted at <em>P</em> â¤ 0.01Â </th></tr></thead><tbody><tr><td><em>Eco</em>RI/<em>Mse</em>I primer pairsÂ </td><td>427Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>G associated dominant markersÂ </td><td>739Â </td><td>504Â </td><td>194Â </td><td>44Â </td></tr><tr><td>M associated dominant markersÂ </td><td>674Â </td><td>454Â </td><td>202Â </td><td>55Â </td></tr><tr><td>Co-dominant markersÂ </td><td>640Â </td><td>533Â </td><td>179Â </td><td>32Â </td></tr><tr><td>Total AFLP markersÂ </td><td>2053Â </td><td>1491Â </td><td>575 (28%)Â </td><td>131 (6.4%)Â </td></tr><tr><td>SSR markers*Â </td><td>97Â </td><td>97Â </td><td>23 (23.7%)Â </td><td>5 (5.1%)Â </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>The numbers in parentheses show the percentage of the markers stated to the total AFLP markers.</p></div></div><div content-id=""><div class="fn"><p>*Included two dCAPS markers.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5193893" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSN022TB1" class="table-wrap-title" data-id="DSN022TB1"><div class="label -label">TableÂ 1</div><div class="caption"><p>AFLP and SSR markers generated in the 94 F<sub>6</sub> individuals from a cross between <em>L</em>. <em>japonicus</em> accessions, âGifu B-129â (G) and âMiyakojima MG-20â (M)</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Â </th><th>TotalÂ </th><th>Mapped markersÂ </th><th>Distorted at <em>P</em> â¤ 0.05Â </th><th>Distorted at <em>P</em> â¤ 0.01Â </th></tr></thead><tbody><tr><td><em>Eco</em>RI/<em>Mse</em>I primer pairsÂ </td><td>427Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>G associated dominant markersÂ </td><td>739Â </td><td>504Â </td><td>194Â </td><td>44Â </td></tr><tr><td>M associated dominant markersÂ </td><td>674Â </td><td>454Â </td><td>202Â </td><td>55Â </td></tr><tr><td>Co-dominant markersÂ </td><td>640Â </td><td>533Â </td><td>179Â </td><td>32Â </td></tr><tr><td>Total AFLP markersÂ </td><td>2053Â </td><td>1491Â </td><td>575 (28%)Â </td><td>131 (6.4%)Â </td></tr><tr><td>SSR markers*Â </td><td>97Â </td><td>97Â </td><td>23 (23.7%)Â </td><td>5 (5.1%)Â </td></tr></tbody></table></div><div content-id="DSN022TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Â </th><th>TotalÂ </th><th>Mapped markersÂ </th><th>Distorted at <em>P</em> â¤ 0.05Â </th><th>Distorted at <em>P</em> â¤ 0.01Â </th></tr></thead><tbody><tr><td><em>Eco</em>RI/<em>Mse</em>I primer pairsÂ </td><td>427Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>G associated dominant markersÂ </td><td>739Â </td><td>504Â </td><td>194Â </td><td>44Â </td></tr><tr><td>M associated dominant markersÂ </td><td>674Â </td><td>454Â </td><td>202Â </td><td>55Â </td></tr><tr><td>Co-dominant markersÂ </td><td>640Â </td><td>533Â </td><td>179Â </td><td>32Â </td></tr><tr><td>Total AFLP markersÂ </td><td>2053Â </td><td>1491Â </td><td>575 (28%)Â </td><td>131 (6.4%)Â </td></tr><tr><td>SSR markers*Â </td><td>97Â </td><td>97Â </td><td>23 (23.7%)Â </td><td>5 (5.1%)Â </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>The numbers in parentheses show the percentage of the markers stated to the total AFLP markers.</p></div></div><div content-id=""><div class="fn"><p>*Included two dCAPS markers.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5193893" target="_blank">View Large</a></div></div><p>An F<sub>6</sub> population is considered a RIL and presents an exception of 1:1 ratio for allele segregation among individuals, but a theoretical ratio of 3% of individuals should show as heterozygous. In this case, each segregating marker was tested with a <em>Ï</em><sup>2</sup> test for goodness-of-fit to the expected 1:1 Mendelian segregating ratio. As a result, 131 of the 2053 markers (6.4%) were distributed to a skewed segregation with significance at <em>P</em> â¤ 0.01, and 575 markers (28%) were distorted at <em>P</em> â¤ 0.05. Most skewed segregating markers deviated to Miyakojima MG-20 alleles. In this study, the distributions of 82 of these 131 markers inclined to Miyakojima MG-20 and only 49 to Gifu B-129 (TableÂ <a reveal-id="DSN022TB1" data-open="DSN022TB1" class="link link-reveal link-table xref-fig">1</a>). Five SSR markers were distorted significantly at the 1% level, of which three were distorted to Gifu B-129 and two to Miyakojima MG-20. However, these five SSR markers were retained in the mapping analysis because their distribution frequencies were near to <em>P</em> = 0.01. The AFLP markers skewed at the 1% level were discarded and excluded in the data analysis. About 24% of the SSR markers were skewed at the 5% level and included in the map analysis (TableÂ <a reveal-id="DSN022TB1" data-open="DSN022TB1" class="link link-reveal link-table xref-fig">1</a>). As a result, a total of 1588 AFLP markers and 97 SSR including 2 dCAPS markers were mapped on the linkage map of <em>L</em>. <em>japonicus</em>.</p>                            <h3 scrollto-destination=5193895 id="5193895" class="section-title" data-legacy-id=s3b>3.2. Construction of linkage map</h3>
<p>All AFLP markers and 95 SSR and 2 dCAPS markers were run on JoinMapÂ® 4 to generate six groups at an LOD value of 14.0, with a maximum distance of 30 cM. Using the SSR markers on each group as references, these six groups were assigned to chromosome 1â6 of <em>L</em>. <em>japonicus</em>.<sup><a reveal-id="DSN022C13" data-open="DSN022C13" class="link link-ref link-reveal xref-bibr">13</a></sup> A high-density linkage map for <em>L</em>. <em>japonicus</em> was generated; 533 co-dominant AFLP markers and 97 SSR (including 2 dCAPS markers) were mapped on six chromosomes of <em>L</em>. <em>japonicus</em>, with an additional 958 AFLP markers (504 Gifu associated and 454 Miyakojima associated markers, TableÂ <a reveal-id="DSN022TB2" data-open="DSN022TB2" class="link link-reveal link-table xref-fig">2</a>). A total of 1588 markers were mapped on the six chromosome maps (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplement 2</a>). This linkage map consisted of 1013 unique markers (excluded duplicated markers) and spanned a total length of 508.5 cM with an average of 0.50 cM between markers. The range of the length of each chromosome varies from 50.1 cM (chromosome 2) to 131.7 cM (chromosome 1), with a mean of 84.7 cM (TableÂ <a reveal-id="DSN022TB2" data-open="DSN022TB2" class="link link-reveal link-table xref-fig">2</a>, Fig.Â <a reveal-id="DSN022F3" data-open="DSN022F3" class="link link-reveal link-table xref-fig">3</a>).</p>                        <a id="5193897" scrollto-destination="5193897"></a>
<div id="DSN022F3" data-id="DSN022F3" class="fig fig-section"><div class="label fig-label">FigureÂ 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn022/2/m_dsn02203a.jpeg?Expires=2147483647&amp;Signature=alqsFzPHpmNxKFbk1Eg5VfDw2uj8RE4OlNvjiMLfO0I2~MP9ds~ZGGvxZlg8f0OcXlFVu0Yx83G1CIhx~Wp5PIw56VM1aLAboHbIqMhR-WsKhZuKsU3o~YJj37IciV7T3Z0NVoIh42~TISjaMgbzMCaGAPZ0w3TMULb9YlCYG9JAJpgk7ID8SjMx3YAilaX8OK13oFuC1sKgjFx1i6akA7EaQL8ZltqdJHPSetDBWMjqc46-g8CjZDH3uCzcUXNC1ut69MMZNBjffCOMEPD9WzjajWCnORtow3bJwzEYxLgUXliJT4R75irmFHWNKq87F3ugOiHlP3z2GG3BDwAe9g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Linkage maps of chromosome 1â6 of Ljaponicus genome Marker names were assigned based on the combination of EcoRI and MseI AFLP primers with generated marker size (in base pair). Letter G stands for maternal âGifu B-129â associated marker; M for paternal âMiyakojimaâ associated marker and C for co-dominant markers. All SSR/dCAPS markers assigned as TM (Hayashi et al.13 and visit at http://www.kazusa.or.jp/lotus/markerdb_index.html). Morphological QTL: SS stands for seed size; LL for leaflet; SC for seed color; ST for stipule; PE for petiole; TR for trichrome; PT for plant type and SP for stem pigment." path-from-xml="dsn02203a.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5193897/dsn02203a.jpeg" path-from-xml="dsn02203a.jpeg" target="_blank">View large</a><a section="5193897" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn022/2/dsn02203a.jpeg?Expires=2147483647&amp;Signature=eCCK02jp4u7y8ZAPEBKz3QSm~Q-U1CvuyIoGQmvM7UZ5YrxQ-Y-f3T9RvlkAjSfvJqoEVFjBWV5uJ7IMGazQgZb92zsiogh32zaHrYQFtxvBBttlyDuvmB267gOBb0pUy0r2B6HwVS06fiNMqX9RnWD67qiIgjQC9665FPmb93ayPFrsnJ-h1ay2JXRu9vfPxldpbQp2sp95IOpPcF44x~8iAnuVbLIgzbTlp-FK082ZNhRroUSZI-LQfjl5zmrBn915g2yI6z~hPR9aGbfSXOGyOo6EufM9NwBs4O~4Xf81mkfRgXs5Wao-62VqzjTWsf21Il8-G~rniviGq75pKg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=5193897&amp;ar=359995&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsn02203a.jpeg" target="_blank">Download slide</a></div></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn022/2/m_dsn02203b.jpeg?Expires=2147483647&amp;Signature=sc6O-3zPRqAN36TFcM-j95xwOwlOqelBODlb9UV3Oh3tGL-AccNevbac2t~~jYV820Ee~6RlWxubE7sE3r7~17YY6no~D6-npxApTrUypLWSHLYxtPKbUEm~3oWjkBx3ptgcs2dWa~wqnN1~tbJpdM8BvqsN1TeOfWX8jicsDqmhc~S8x5o9NWD-72Y~L-cjsuRIDhCAmNoFECiR9naaMnYJ3~TEM4fRaYxkM1vksCYcmF9hDz5B2v90vJ3-s3jNaWpGO~jRYugj2gQHgX8mriUrRU1IUlrvLivJK4mGH3oK-Djdpe3~qXWKKo2tJa86AwkkaoQHn74qg8teAf0SFQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Linkage maps of chromosome 1â6 of Ljaponicus genome Marker names were assigned based on the combination of EcoRI and MseI AFLP primers with generated marker size (in base pair). Letter G stands for maternal âGifu B-129â associated marker; M for paternal âMiyakojimaâ associated marker and C for co-dominant markers. All SSR/dCAPS markers assigned as TM (Hayashi et al.13 and visit at http://www.kazusa.or.jp/lotus/markerdb_index.html). Morphological QTL: SS stands for seed size; LL for leaflet; SC for seed color; ST for stipule; PE for petiole; TR for trichrome; PT for plant type and SP for stem pigment." path-from-xml="dsn02203b.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5193897/dsn02203b.jpeg" path-from-xml="dsn02203b.jpeg" target="_blank">View large</a><a section="5193897" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn022/2/dsn02203b.jpeg?Expires=2147483647&amp;Signature=Jt55pciZPz9rii0~RV7TfTFiTugS5dWRwjSfVBG-3dl15jISrwpbaivgfnbykXwelGRD8Yb~F~utCMlVBtZmW5lGhReZvR4pYvw8Reb3-IfFuiNf7pYD9QnYlBe09seDANYbJ7Zf4eMnQ3iL5qaWYZtqWsM~nIK7X9TryUyF-Cs0yRvBY80-h2yZ7uUqE9Wypq5ZgwcSDXbgoT7aPc-FLE~xkO4sVCGbj7xOc7jJpWm2mkKGA9ldTzV0Sk7WgTrWmhGMAZWDZHe992Froi86p0r9uTuuVUGnBtiBHuVEAF~b1EWY7IRrELdynha7Zj6030ME3MCT3cHnvfzn-vefMg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=5193897&amp;ar=359995&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsn02203b.jpeg" target="_blank">Download slide</a></div></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn022/2/m_dsn02203c.jpeg?Expires=2147483647&amp;Signature=IKiSgyLjw9WMadxszoHpp7aTcqmeBYHbRugBcaPuVaGhQJYQXItmwE6mMCw5qcUqMn6UEMG60A4HYT9eP~ZovZv2Jxj2pYf7LihdMQH3dgTmHyfdFvc593f3~EePN28i0p73P8ll-BH-~~jdmf1YubaDcjzABYMyR~BPFB1KQhiTIifpRSLGsy9zlXXbPQ278vmOcTCkjyrMmzjZM2q7dih7eWXQiiHy9hlZab2-5zrjBwfZbh3xM3TuaiD3F9LdoFGZ7tThtFbKtDZGd1zCH4WKKw5PoZ3VpuLSJT0C9DuoL-KzydAKpszR-Epd78MH5xuz043b0-PLz2zH9DS-9g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Linkage maps of chromosome 1â6 of Ljaponicus genome Marker names were assigned based on the combination of EcoRI and MseI AFLP primers with generated marker size (in base pair). Letter G stands for maternal âGifu B-129â associated marker; M for paternal âMiyakojimaâ associated marker and C for co-dominant markers. All SSR/dCAPS markers assigned as TM (Hayashi et al.13 and visit at http://www.kazusa.or.jp/lotus/markerdb_index.html). Morphological QTL: SS stands for seed size; LL for leaflet; SC for seed color; ST for stipule; PE for petiole; TR for trichrome; PT for plant type and SP for stem pigment." path-from-xml="dsn02203c.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5193897/dsn02203c.jpeg" path-from-xml="dsn02203c.jpeg" target="_blank">View large</a><a section="5193897" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn022/2/dsn02203c.jpeg?Expires=2147483647&amp;Signature=UPspNe08eAOLBDD2B~N3bc61kLpukow8g~zDTmGCp71a~1J18t1ig-vaCzmlq5pvm78OY7Vju5wJ2ZvlYGP31sOFDS7YK6AZGaBopr63lVhjM4mF5Ao~PdeNgew5QoT7yioiVzGuMkPwbElzqvt2jn8D1uN1nuzOz4JFHXq3LdPBnCILnkO2VW7v6zsxQV-a0a3QnC-NDMFW6vsgC4PmAu1aKemVknaiduK0f58epmCe5XFztS-SyDylgPmigZ9nWatkF2EhSFDt8VVwiXMcEJ3oR8RwZQuRTNgB0E8aCJZOKBzKE0Rjo3QzAVlhk2e~bGLVRzLx7-KiGZkUM-3dhg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=5193897&amp;ar=359995&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsn02203c.jpeg" target="_blank">Download slide</a></div></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn022/2/m_dsn02203d.jpeg?Expires=2147483647&amp;Signature=JVx0nB4ibNz0ruZPqelfX4RlJqjG7gRtIOzZTmqN88~SzJrWAZt~QjwygXPPdkIlA6VN3rgK-eYEBvbYtvQ4jv8JACf3wJi0~pXhHW6ZiCzmZqHZ8TqkmIxjc3nxlQIH-IaxCHSAGDVVoLD3uruXQNSixCI6bogLyLGbBuI5RjQth9o8e7EAsG1YrUAD1J9LwPTsdLAnMZxjk3Covop3I2lXVyc4QEreuDZzRD-1x9trBqoLYOEgKHTgaIaFpg0tPof5ZhAoMguBD17o4dW~p4F6fCTtnjKKz9RSCaKZsYotdESwlW8aRleiIM4C7MU~jD1pGgWyOjZaFQkxFO6y7A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Linkage maps of chromosome 1â6 of Ljaponicus genome Marker names were assigned based on the combination of EcoRI and MseI AFLP primers with generated marker size (in base pair). Letter G stands for maternal âGifu B-129â associated marker; M for paternal âMiyakojimaâ associated marker and C for co-dominant markers. All SSR/dCAPS markers assigned as TM (Hayashi et al.13 and visit at http://www.kazusa.or.jp/lotus/markerdb_index.html). Morphological QTL: SS stands for seed size; LL for leaflet; SC for seed color; ST for stipule; PE for petiole; TR for trichrome; PT for plant type and SP for stem pigment." path-from-xml="dsn02203d.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5193897/dsn02203d.jpeg" path-from-xml="dsn02203d.jpeg" target="_blank">View large</a><a section="5193897" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn022/2/dsn02203d.jpeg?Expires=2147483647&amp;Signature=twMnU5LjsGs6WMHTH3zAHqgsCIdayS-GBR8TVL7cDhETG-ut~YIbP5YkUsf4rI98FQipeiMwNiT9PemBpm6pnzlCZqgsPdpYR~CEGKObwP-dDTCQZBlrX6vBTcLxnsVsynJSDwZPaWTIFq5I6BThhuokkkkXxXIp8xo0XGwh64vfiPndXLGCUnJYtBcWpyuW2Bfn4~ci9DbX8aK3S96jdFeb0kwPeHePPRwlqGo50eGkdCREgGXXZx77DVNjuEr2c7kTIpFkd~8RwptlvrMa51iF~21scfQuR0HW7Ehko6PqEoHMcr~aAvZlepFTNiBR85EzCAPO8tvxprwmcQLwdw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=5193897&amp;ar=359995&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsn02203d.jpeg" target="_blank">Download slide</a></div></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn022/2/m_dsn02203e.jpeg?Expires=2147483647&amp;Signature=YyVS4CSXSWmA8PrneY8YWFFte4LPtPQ4IyHiUEx7q-RSeh9BdH-ZtJ3M9L9iKZZtGB30HQpmtQZwE6~Sxtts6Qb1LOv6m-6ftHveRR43N0Ak2WdemB63B4TzCeBHVlETwn2aIHfBr4QlhR64aT53kgA96IL8FaslmmDKQq~X2nxTz5nB5N5ps0MdIvec7lERtCllPLkmfJj2K3OVjaqZX-fPFFjBkUpr3GNmP6WVIZXVeGzISShXVRRs-HeMfy7YHLrNIeYAd-uim~6vJLKi48IXZHl9gVazmFBkOVpOHbgIA5qNU6d7I6DJT2YL5~DwgwlZZAMRY1Z-U-omg1Euag__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Linkage maps of chromosome 1â6 of Ljaponicus genome Marker names were assigned based on the combination of EcoRI and MseI AFLP primers with generated marker size (in base pair). Letter G stands for maternal âGifu B-129â associated marker; M for paternal âMiyakojimaâ associated marker and C for co-dominant markers. All SSR/dCAPS markers assigned as TM (Hayashi et al.13 and visit at http://www.kazusa.or.jp/lotus/markerdb_index.html). Morphological QTL: SS stands for seed size; LL for leaflet; SC for seed color; ST for stipule; PE for petiole; TR for trichrome; PT for plant type and SP for stem pigment." path-from-xml="dsn02203e.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5193897/dsn02203e.jpeg" path-from-xml="dsn02203e.jpeg" target="_blank">View large</a><a section="5193897" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn022/2/dsn02203e.jpeg?Expires=2147483647&amp;Signature=PiuZs4j3wFXSHPipw~gYQ8HhUAcz~rh7dSEAaah8oqOBVhjuM24MNi-JVC8SXERHcfoR6CRfxFBrvKL2TGLgeUNeY4qhDweujZAdyE9qeQyaEbshRtcYZtmJ-qWof9U6OmM5FhqTIHqS66MIyM8WNcJV8B-RuQviiM6oMqMrr3g2CrfToULRSabLYYKnYmpjaSV3qLv7PCXl9QyLGcyJIRcFV~Et1-qdmnaomG9qW4AwRM4CwcwWD-sdSTAD6Uy9oVEnd-feXa7LIbtHb6shrjhMhF2Gd7WpAIX-Vhxrdsig9ZjTvMrtQq1azV-blS0WsyOeNULFxmxPM0vFoXC7sA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=5193897&amp;ar=359995&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsn02203e.jpeg" target="_blank">Download slide</a></div></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn022/2/m_dsn02203f.jpeg?Expires=2147483647&amp;Signature=x51x5Qgqm9K8~8Hdzz4zvOjUWels7Pjk7cLT9t~2Z-p8PtAlykGyPIqiG71fiBg1QuwuVg7PlpSEWRnBSRU5CO3roP74qhero7DK3Fx-jYsGIUR2V5KEUQuk0ovhuAjBfZtYfwEKQ7l9EqdC-ZYluzPwvj3avPVHxI~WXpiv8fFouXbcrRC096urkJS2NvYcRaJCII3-ojOJ8t9QnynnaIQs~npahUOaSVPpXwWwPZs5kptv174AMNs6z5mPb7QQKRhkEqW-HDhVp0tIdKqjfJU7npqXowVKDOwJaJ0Up9QFNk3tIeMsCEWuKZ8b5KvTwCOhCCPa7bC9nmIWj88Jsg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Linkage maps of chromosome 1â6 of Ljaponicus genome Marker names were assigned based on the combination of EcoRI and MseI AFLP primers with generated marker size (in base pair). Letter G stands for maternal âGifu B-129â associated marker; M for paternal âMiyakojimaâ associated marker and C for co-dominant markers. All SSR/dCAPS markers assigned as TM (Hayashi et al.13 and visit at http://www.kazusa.or.jp/lotus/markerdb_index.html). Morphological QTL: SS stands for seed size; LL for leaflet; SC for seed color; ST for stipule; PE for petiole; TR for trichrome; PT for plant type and SP for stem pigment." path-from-xml="dsn02203f.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5193897/dsn02203f.jpeg" path-from-xml="dsn02203f.jpeg" target="_blank">View large</a><a section="5193897" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn022/2/dsn02203f.jpeg?Expires=2147483647&amp;Signature=kU-DNS6-MrXDfMZLbP2-QoMwwyP75hqy6yrYi-0aMJ0Ib3vg0mMuBt2ZGIP5OUAH~1LNcPTaYFC09S1ZR5kKwZ~IhF1vmkQUSbDpVGC7LMAzNL2tOwUsRcU5KHQ~m5z3dlPGyr4XHWkGMFtf-cOhT6cPi32-Hb0D1xP8u9pQIGeEHVu-WsM4s9qKHEcyCCx6BqwuU7F3fL~xUZ0BNgporf3iwLJlyaK17e-BZYgIqFWUY9KJOgeBcGkouui6TfOyG-qet8f1Lz~mMuyq-~KlITVmu6KRLqWdqOCRIS88IP7ooUUzxZ-H88TeMtHeE0SXYsxPVnxmqzXtF0m-Yw2tKA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=5193897&amp;ar=359995&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsn02203f.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Linkage maps of chromosome 1â6 of <em>L</em><em>japonicus</em> genome Marker names were assigned based on the combination of <em>Eco</em>RI and <em>Mse</em>I AFLP primers with generated marker size (in base pair). Letter G stands for maternal âGifu B-129â associated marker; M for paternal âMiyakojimaâ associated marker and C for co-dominant markers. All SSR/dCAPS markers assigned as TM (Hayashi et al.<sup><a reveal-id="DSN022C13" data-open="DSN022C13" class="link link-ref link-reveal xref-bibr">13</a></sup> and visit at <a class="link link-uri" href="http://www.kazusa.or.jp/lotus/markerdb_index.html">http://www.kazusa.or.jp/lotus/markerdb_index.html</a>). Morphological QTL: SS stands for seed size; LL for leaflet; SC for seed color; ST for stipule; PE for petiole; TR for trichrome; PT for plant type and SP for stem pigment.</p></div></div><div content-id="DSN022F3" class="fig fig-modal reveal-modal"><div class="label fig-label">FigureÂ 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn022/2/m_dsn02203a.jpeg?Expires=2147483647&amp;Signature=alqsFzPHpmNxKFbk1Eg5VfDw2uj8RE4OlNvjiMLfO0I2~MP9ds~ZGGvxZlg8f0OcXlFVu0Yx83G1CIhx~Wp5PIw56VM1aLAboHbIqMhR-WsKhZuKsU3o~YJj37IciV7T3Z0NVoIh42~TISjaMgbzMCaGAPZ0w3TMULb9YlCYG9JAJpgk7ID8SjMx3YAilaX8OK13oFuC1sKgjFx1i6akA7EaQL8ZltqdJHPSetDBWMjqc46-g8CjZDH3uCzcUXNC1ut69MMZNBjffCOMEPD9WzjajWCnORtow3bJwzEYxLgUXliJT4R75irmFHWNKq87F3ugOiHlP3z2GG3BDwAe9g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Linkage maps of chromosome 1â6 of Ljaponicus genome Marker names were assigned based on the combination of EcoRI and MseI AFLP primers with generated marker size (in base pair). Letter G stands for maternal âGifu B-129â associated marker; M for paternal âMiyakojimaâ associated marker and C for co-dominant markers. All SSR/dCAPS markers assigned as TM (Hayashi et al.13 and visit at http://www.kazusa.or.jp/lotus/markerdb_index.html). Morphological QTL: SS stands for seed size; LL for leaflet; SC for seed color; ST for stipule; PE for petiole; TR for trichrome; PT for plant type and SP for stem pigment." path-from-xml="dsn02203a.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5193897/dsn02203a.jpeg" path-from-xml="dsn02203a.jpeg" target="_blank">View large</a><a section="5193897" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn022/2/dsn02203a.jpeg?Expires=2147483647&amp;Signature=eCCK02jp4u7y8ZAPEBKz3QSm~Q-U1CvuyIoGQmvM7UZ5YrxQ-Y-f3T9RvlkAjSfvJqoEVFjBWV5uJ7IMGazQgZb92zsiogh32zaHrYQFtxvBBttlyDuvmB267gOBb0pUy0r2B6HwVS06fiNMqX9RnWD67qiIgjQC9665FPmb93ayPFrsnJ-h1ay2JXRu9vfPxldpbQp2sp95IOpPcF44x~8iAnuVbLIgzbTlp-FK082ZNhRroUSZI-LQfjl5zmrBn915g2yI6z~hPR9aGbfSXOGyOo6EufM9NwBs4O~4Xf81mkfRgXs5Wao-62VqzjTWsf21Il8-G~rniviGq75pKg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=5193897&amp;ar=359995&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsn02203a.jpeg" target="_blank">Download slide</a></div></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn022/2/m_dsn02203b.jpeg?Expires=2147483647&amp;Signature=sc6O-3zPRqAN36TFcM-j95xwOwlOqelBODlb9UV3Oh3tGL-AccNevbac2t~~jYV820Ee~6RlWxubE7sE3r7~17YY6no~D6-npxApTrUypLWSHLYxtPKbUEm~3oWjkBx3ptgcs2dWa~wqnN1~tbJpdM8BvqsN1TeOfWX8jicsDqmhc~S8x5o9NWD-72Y~L-cjsuRIDhCAmNoFECiR9naaMnYJ3~TEM4fRaYxkM1vksCYcmF9hDz5B2v90vJ3-s3jNaWpGO~jRYugj2gQHgX8mriUrRU1IUlrvLivJK4mGH3oK-Djdpe3~qXWKKo2tJa86AwkkaoQHn74qg8teAf0SFQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Linkage maps of chromosome 1â6 of Ljaponicus genome Marker names were assigned based on the combination of EcoRI and MseI AFLP primers with generated marker size (in base pair). Letter G stands for maternal âGifu B-129â associated marker; M for paternal âMiyakojimaâ associated marker and C for co-dominant markers. All SSR/dCAPS markers assigned as TM (Hayashi et al.13 and visit at http://www.kazusa.or.jp/lotus/markerdb_index.html). Morphological QTL: SS stands for seed size; LL for leaflet; SC for seed color; ST for stipule; PE for petiole; TR for trichrome; PT for plant type and SP for stem pigment." path-from-xml="dsn02203b.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5193897/dsn02203b.jpeg" path-from-xml="dsn02203b.jpeg" target="_blank">View large</a><a section="5193897" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn022/2/dsn02203b.jpeg?Expires=2147483647&amp;Signature=Jt55pciZPz9rii0~RV7TfTFiTugS5dWRwjSfVBG-3dl15jISrwpbaivgfnbykXwelGRD8Yb~F~utCMlVBtZmW5lGhReZvR4pYvw8Reb3-IfFuiNf7pYD9QnYlBe09seDANYbJ7Zf4eMnQ3iL5qaWYZtqWsM~nIK7X9TryUyF-Cs0yRvBY80-h2yZ7uUqE9Wypq5ZgwcSDXbgoT7aPc-FLE~xkO4sVCGbj7xOc7jJpWm2mkKGA9ldTzV0Sk7WgTrWmhGMAZWDZHe992Froi86p0r9uTuuVUGnBtiBHuVEAF~b1EWY7IRrELdynha7Zj6030ME3MCT3cHnvfzn-vefMg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=5193897&amp;ar=359995&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsn02203b.jpeg" target="_blank">Download slide</a></div></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn022/2/m_dsn02203c.jpeg?Expires=2147483647&amp;Signature=IKiSgyLjw9WMadxszoHpp7aTcqmeBYHbRugBcaPuVaGhQJYQXItmwE6mMCw5qcUqMn6UEMG60A4HYT9eP~ZovZv2Jxj2pYf7LihdMQH3dgTmHyfdFvc593f3~EePN28i0p73P8ll-BH-~~jdmf1YubaDcjzABYMyR~BPFB1KQhiTIifpRSLGsy9zlXXbPQ278vmOcTCkjyrMmzjZM2q7dih7eWXQiiHy9hlZab2-5zrjBwfZbh3xM3TuaiD3F9LdoFGZ7tThtFbKtDZGd1zCH4WKKw5PoZ3VpuLSJT0C9DuoL-KzydAKpszR-Epd78MH5xuz043b0-PLz2zH9DS-9g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Linkage maps of chromosome 1â6 of Ljaponicus genome Marker names were assigned based on the combination of EcoRI and MseI AFLP primers with generated marker size (in base pair). Letter G stands for maternal âGifu B-129â associated marker; M for paternal âMiyakojimaâ associated marker and C for co-dominant markers. All SSR/dCAPS markers assigned as TM (Hayashi et al.13 and visit at http://www.kazusa.or.jp/lotus/markerdb_index.html). Morphological QTL: SS stands for seed size; LL for leaflet; SC for seed color; ST for stipule; PE for petiole; TR for trichrome; PT for plant type and SP for stem pigment." path-from-xml="dsn02203c.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5193897/dsn02203c.jpeg" path-from-xml="dsn02203c.jpeg" target="_blank">View large</a><a section="5193897" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn022/2/dsn02203c.jpeg?Expires=2147483647&amp;Signature=UPspNe08eAOLBDD2B~N3bc61kLpukow8g~zDTmGCp71a~1J18t1ig-vaCzmlq5pvm78OY7Vju5wJ2ZvlYGP31sOFDS7YK6AZGaBopr63lVhjM4mF5Ao~PdeNgew5QoT7yioiVzGuMkPwbElzqvt2jn8D1uN1nuzOz4JFHXq3LdPBnCILnkO2VW7v6zsxQV-a0a3QnC-NDMFW6vsgC4PmAu1aKemVknaiduK0f58epmCe5XFztS-SyDylgPmigZ9nWatkF2EhSFDt8VVwiXMcEJ3oR8RwZQuRTNgB0E8aCJZOKBzKE0Rjo3QzAVlhk2e~bGLVRzLx7-KiGZkUM-3dhg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=5193897&amp;ar=359995&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsn02203c.jpeg" target="_blank">Download slide</a></div></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn022/2/m_dsn02203d.jpeg?Expires=2147483647&amp;Signature=JVx0nB4ibNz0ruZPqelfX4RlJqjG7gRtIOzZTmqN88~SzJrWAZt~QjwygXPPdkIlA6VN3rgK-eYEBvbYtvQ4jv8JACf3wJi0~pXhHW6ZiCzmZqHZ8TqkmIxjc3nxlQIH-IaxCHSAGDVVoLD3uruXQNSixCI6bogLyLGbBuI5RjQth9o8e7EAsG1YrUAD1J9LwPTsdLAnMZxjk3Covop3I2lXVyc4QEreuDZzRD-1x9trBqoLYOEgKHTgaIaFpg0tPof5ZhAoMguBD17o4dW~p4F6fCTtnjKKz9RSCaKZsYotdESwlW8aRleiIM4C7MU~jD1pGgWyOjZaFQkxFO6y7A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Linkage maps of chromosome 1â6 of Ljaponicus genome Marker names were assigned based on the combination of EcoRI and MseI AFLP primers with generated marker size (in base pair). Letter G stands for maternal âGifu B-129â associated marker; M for paternal âMiyakojimaâ associated marker and C for co-dominant markers. All SSR/dCAPS markers assigned as TM (Hayashi et al.13 and visit at http://www.kazusa.or.jp/lotus/markerdb_index.html). Morphological QTL: SS stands for seed size; LL for leaflet; SC for seed color; ST for stipule; PE for petiole; TR for trichrome; PT for plant type and SP for stem pigment." path-from-xml="dsn02203d.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5193897/dsn02203d.jpeg" path-from-xml="dsn02203d.jpeg" target="_blank">View large</a><a section="5193897" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn022/2/dsn02203d.jpeg?Expires=2147483647&amp;Signature=twMnU5LjsGs6WMHTH3zAHqgsCIdayS-GBR8TVL7cDhETG-ut~YIbP5YkUsf4rI98FQipeiMwNiT9PemBpm6pnzlCZqgsPdpYR~CEGKObwP-dDTCQZBlrX6vBTcLxnsVsynJSDwZPaWTIFq5I6BThhuokkkkXxXIp8xo0XGwh64vfiPndXLGCUnJYtBcWpyuW2Bfn4~ci9DbX8aK3S96jdFeb0kwPeHePPRwlqGo50eGkdCREgGXXZx77DVNjuEr2c7kTIpFkd~8RwptlvrMa51iF~21scfQuR0HW7Ehko6PqEoHMcr~aAvZlepFTNiBR85EzCAPO8tvxprwmcQLwdw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=5193897&amp;ar=359995&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsn02203d.jpeg" target="_blank">Download slide</a></div></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn022/2/m_dsn02203e.jpeg?Expires=2147483647&amp;Signature=YyVS4CSXSWmA8PrneY8YWFFte4LPtPQ4IyHiUEx7q-RSeh9BdH-ZtJ3M9L9iKZZtGB30HQpmtQZwE6~Sxtts6Qb1LOv6m-6ftHveRR43N0Ak2WdemB63B4TzCeBHVlETwn2aIHfBr4QlhR64aT53kgA96IL8FaslmmDKQq~X2nxTz5nB5N5ps0MdIvec7lERtCllPLkmfJj2K3OVjaqZX-fPFFjBkUpr3GNmP6WVIZXVeGzISShXVRRs-HeMfy7YHLrNIeYAd-uim~6vJLKi48IXZHl9gVazmFBkOVpOHbgIA5qNU6d7I6DJT2YL5~DwgwlZZAMRY1Z-U-omg1Euag__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Linkage maps of chromosome 1â6 of Ljaponicus genome Marker names were assigned based on the combination of EcoRI and MseI AFLP primers with generated marker size (in base pair). Letter G stands for maternal âGifu B-129â associated marker; M for paternal âMiyakojimaâ associated marker and C for co-dominant markers. All SSR/dCAPS markers assigned as TM (Hayashi et al.13 and visit at http://www.kazusa.or.jp/lotus/markerdb_index.html). Morphological QTL: SS stands for seed size; LL for leaflet; SC for seed color; ST for stipule; PE for petiole; TR for trichrome; PT for plant type and SP for stem pigment." path-from-xml="dsn02203e.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5193897/dsn02203e.jpeg" path-from-xml="dsn02203e.jpeg" target="_blank">View large</a><a section="5193897" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn022/2/dsn02203e.jpeg?Expires=2147483647&amp;Signature=PiuZs4j3wFXSHPipw~gYQ8HhUAcz~rh7dSEAaah8oqOBVhjuM24MNi-JVC8SXERHcfoR6CRfxFBrvKL2TGLgeUNeY4qhDweujZAdyE9qeQyaEbshRtcYZtmJ-qWof9U6OmM5FhqTIHqS66MIyM8WNcJV8B-RuQviiM6oMqMrr3g2CrfToULRSabLYYKnYmpjaSV3qLv7PCXl9QyLGcyJIRcFV~Et1-qdmnaomG9qW4AwRM4CwcwWD-sdSTAD6Uy9oVEnd-feXa7LIbtHb6shrjhMhF2Gd7WpAIX-Vhxrdsig9ZjTvMrtQq1azV-blS0WsyOeNULFxmxPM0vFoXC7sA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=5193897&amp;ar=359995&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsn02203e.jpeg" target="_blank">Download slide</a></div></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn022/2/m_dsn02203f.jpeg?Expires=2147483647&amp;Signature=x51x5Qgqm9K8~8Hdzz4zvOjUWels7Pjk7cLT9t~2Z-p8PtAlykGyPIqiG71fiBg1QuwuVg7PlpSEWRnBSRU5CO3roP74qhero7DK3Fx-jYsGIUR2V5KEUQuk0ovhuAjBfZtYfwEKQ7l9EqdC-ZYluzPwvj3avPVHxI~WXpiv8fFouXbcrRC096urkJS2NvYcRaJCII3-ojOJ8t9QnynnaIQs~npahUOaSVPpXwWwPZs5kptv174AMNs6z5mPb7QQKRhkEqW-HDhVp0tIdKqjfJU7npqXowVKDOwJaJ0Up9QFNk3tIeMsCEWuKZ8b5KvTwCOhCCPa7bC9nmIWj88Jsg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Linkage maps of chromosome 1â6 of Ljaponicus genome Marker names were assigned based on the combination of EcoRI and MseI AFLP primers with generated marker size (in base pair). Letter G stands for maternal âGifu B-129â associated marker; M for paternal âMiyakojimaâ associated marker and C for co-dominant markers. All SSR/dCAPS markers assigned as TM (Hayashi et al.13 and visit at http://www.kazusa.or.jp/lotus/markerdb_index.html). Morphological QTL: SS stands for seed size; LL for leaflet; SC for seed color; ST for stipule; PE for petiole; TR for trichrome; PT for plant type and SP for stem pigment." path-from-xml="dsn02203f.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5193897/dsn02203f.jpeg" path-from-xml="dsn02203f.jpeg" target="_blank">View large</a><a section="5193897" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn022/2/dsn02203f.jpeg?Expires=2147483647&amp;Signature=kU-DNS6-MrXDfMZLbP2-QoMwwyP75hqy6yrYi-0aMJ0Ib3vg0mMuBt2ZGIP5OUAH~1LNcPTaYFC09S1ZR5kKwZ~IhF1vmkQUSbDpVGC7LMAzNL2tOwUsRcU5KHQ~m5z3dlPGyr4XHWkGMFtf-cOhT6cPi32-Hb0D1xP8u9pQIGeEHVu-WsM4s9qKHEcyCCx6BqwuU7F3fL~xUZ0BNgporf3iwLJlyaK17e-BZYgIqFWUY9KJOgeBcGkouui6TfOyG-qet8f1Lz~mMuyq-~KlITVmu6KRLqWdqOCRIS88IP7ooUUzxZ-H88TeMtHeE0SXYsxPVnxmqzXtF0m-Yw2tKA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=5193897&amp;ar=359995&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsn02203f.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Linkage maps of chromosome 1â6 of <em>L</em><em>japonicus</em> genome Marker names were assigned based on the combination of <em>Eco</em>RI and <em>Mse</em>I AFLP primers with generated marker size (in base pair). Letter G stands for maternal âGifu B-129â associated marker; M for paternal âMiyakojimaâ associated marker and C for co-dominant markers. All SSR/dCAPS markers assigned as TM (Hayashi et al.<sup><a reveal-id="DSN022C13" data-open="DSN022C13" class="link link-ref link-reveal xref-bibr">13</a></sup> and visit at <a class="link link-uri" href="http://www.kazusa.or.jp/lotus/markerdb_index.html">http://www.kazusa.or.jp/lotus/markerdb_index.html</a>). Morphological QTL: SS stands for seed size; LL for leaflet; SC for seed color; ST for stipule; PE for petiole; TR for trichrome; PT for plant type and SP for stem pigment.</p></div></div>                        <a id="5193898" scrollto-destination="5193898"></a>
<div content-id="DSN022TB2" class="table-modal"><div class="table-wrap"><div id="DSN022TB2" class="table-wrap-title" data-id="DSN022TB2"><div class="label -label">TableÂ 2</div><div class="caption"><p>The mapping characterization of six chromosomes of <em>L</em>. <em>japonicus</em> genome</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Â </th><th>SSRÂ </th><th colspan="3">AFLP markers<hr />Â </th><th>Total markersÂ </th><th>Length (cM)Â </th><th>Unique markers*Â </th><th>Mean intervalsÂ </th><th>QTLÂ </th></tr><tr><th>Â </th><th>Â </th><th>Co-dominantÂ </th><th>G DominantÂ </th><th>M DominantÂ </th><th>Â </th><th>Â </th><th>Â </th><th>Â </th><th>Â </th></tr></thead><tbody><tr><td>Chromosome 1Â </td><td>41Â </td><td>137Â </td><td>95Â </td><td>100Â </td><td>373Â </td><td>131.7Â </td><td>204Â </td><td>0.65Â </td><td>3Â </td></tr><tr><td>Chromosome 2Â </td><td>7Â </td><td>102Â </td><td>83Â </td><td>67Â </td><td>259Â </td><td>50.1Â </td><td>154Â </td><td>0.33Â </td><td>2Â </td></tr><tr><td>Chromosome 3Â </td><td>17Â </td><td>115Â </td><td>92Â </td><td>114Â </td><td>338Â </td><td>98.9Â </td><td>206Â </td><td>0.48Â </td><td>1Â </td></tr><tr><td>Chromosome 4Â </td><td>12Â </td><td>71Â </td><td>100Â </td><td>58Â </td><td>241Â </td><td>82.5Â </td><td>157Â </td><td>0.53Â </td><td>6Â </td></tr><tr><td>Chromosome 5Â </td><td>10Â </td><td>60Â </td><td>65Â </td><td>49Â </td><td>184Â </td><td>75.2Â </td><td>146Â </td><td>0.52Â </td><td>1Â </td></tr><tr><td>Chromosome 6Â </td><td>10Â </td><td>48Â </td><td>69Â </td><td>66Â </td><td>193Â </td><td>70.1Â </td><td>146Â </td><td>0.48Â </td><td>1Â </td></tr><tr><td>Total IIIÂ </td><td>97Â </td><td>533Â </td><td>504Â </td><td>454Â </td><td>1588Â </td><td>508.5Â </td><td>1013Â </td><td>0.50Â </td><td>14Â </td></tr></tbody></table></div><div content-id="DSN022TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Â </th><th>SSRÂ </th><th colspan="3">AFLP markers<hr />Â </th><th>Total markersÂ </th><th>Length (cM)Â </th><th>Unique markers*Â </th><th>Mean intervalsÂ </th><th>QTLÂ </th></tr><tr><th>Â </th><th>Â </th><th>Co-dominantÂ </th><th>G DominantÂ </th><th>M DominantÂ </th><th>Â </th><th>Â </th><th>Â </th><th>Â </th><th>Â </th></tr></thead><tbody><tr><td>Chromosome 1Â </td><td>41Â </td><td>137Â </td><td>95Â </td><td>100Â </td><td>373Â </td><td>131.7Â </td><td>204Â </td><td>0.65Â </td><td>3Â </td></tr><tr><td>Chromosome 2Â </td><td>7Â </td><td>102Â </td><td>83Â </td><td>67Â </td><td>259Â </td><td>50.1Â </td><td>154Â </td><td>0.33Â </td><td>2Â </td></tr><tr><td>Chromosome 3Â </td><td>17Â </td><td>115Â </td><td>92Â </td><td>114Â </td><td>338Â </td><td>98.9Â </td><td>206Â </td><td>0.48Â </td><td>1Â </td></tr><tr><td>Chromosome 4Â </td><td>12Â </td><td>71Â </td><td>100Â </td><td>58Â </td><td>241Â </td><td>82.5Â </td><td>157Â </td><td>0.53Â </td><td>6Â </td></tr><tr><td>Chromosome 5Â </td><td>10Â </td><td>60Â </td><td>65Â </td><td>49Â </td><td>184Â </td><td>75.2Â </td><td>146Â </td><td>0.52Â </td><td>1Â </td></tr><tr><td>Chromosome 6Â </td><td>10Â </td><td>48Â </td><td>69Â </td><td>66Â </td><td>193Â </td><td>70.1Â </td><td>146Â </td><td>0.48Â </td><td>1Â </td></tr><tr><td>Total IIIÂ </td><td>97Â </td><td>533Â </td><td>504Â </td><td>454Â </td><td>1588Â </td><td>508.5Â </td><td>1013Â </td><td>0.50Â </td><td>14Â </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>*The duplicated markers were excluded.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5193898" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSN022TB2" class="table-wrap-title" data-id="DSN022TB2"><div class="label -label">TableÂ 2</div><div class="caption"><p>The mapping characterization of six chromosomes of <em>L</em>. <em>japonicus</em> genome</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Â </th><th>SSRÂ </th><th colspan="3">AFLP markers<hr />Â </th><th>Total markersÂ </th><th>Length (cM)Â </th><th>Unique markers*Â </th><th>Mean intervalsÂ </th><th>QTLÂ </th></tr><tr><th>Â </th><th>Â </th><th>Co-dominantÂ </th><th>G DominantÂ </th><th>M DominantÂ </th><th>Â </th><th>Â </th><th>Â </th><th>Â </th><th>Â </th></tr></thead><tbody><tr><td>Chromosome 1Â </td><td>41Â </td><td>137Â </td><td>95Â </td><td>100Â </td><td>373Â </td><td>131.7Â </td><td>204Â </td><td>0.65Â </td><td>3Â </td></tr><tr><td>Chromosome 2Â </td><td>7Â </td><td>102Â </td><td>83Â </td><td>67Â </td><td>259Â </td><td>50.1Â </td><td>154Â </td><td>0.33Â </td><td>2Â </td></tr><tr><td>Chromosome 3Â </td><td>17Â </td><td>115Â </td><td>92Â </td><td>114Â </td><td>338Â </td><td>98.9Â </td><td>206Â </td><td>0.48Â </td><td>1Â </td></tr><tr><td>Chromosome 4Â </td><td>12Â </td><td>71Â </td><td>100Â </td><td>58Â </td><td>241Â </td><td>82.5Â </td><td>157Â </td><td>0.53Â </td><td>6Â </td></tr><tr><td>Chromosome 5Â </td><td>10Â </td><td>60Â </td><td>65Â </td><td>49Â </td><td>184Â </td><td>75.2Â </td><td>146Â </td><td>0.52Â </td><td>1Â </td></tr><tr><td>Chromosome 6Â </td><td>10Â </td><td>48Â </td><td>69Â </td><td>66Â </td><td>193Â </td><td>70.1Â </td><td>146Â </td><td>0.48Â </td><td>1Â </td></tr><tr><td>Total IIIÂ </td><td>97Â </td><td>533Â </td><td>504Â </td><td>454Â </td><td>1588Â </td><td>508.5Â </td><td>1013Â </td><td>0.50Â </td><td>14Â </td></tr></tbody></table></div><div content-id="DSN022TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Â </th><th>SSRÂ </th><th colspan="3">AFLP markers<hr />Â </th><th>Total markersÂ </th><th>Length (cM)Â </th><th>Unique markers*Â </th><th>Mean intervalsÂ </th><th>QTLÂ </th></tr><tr><th>Â </th><th>Â </th><th>Co-dominantÂ </th><th>G DominantÂ </th><th>M DominantÂ </th><th>Â </th><th>Â </th><th>Â </th><th>Â </th><th>Â </th></tr></thead><tbody><tr><td>Chromosome 1Â </td><td>41Â </td><td>137Â </td><td>95Â </td><td>100Â </td><td>373Â </td><td>131.7Â </td><td>204Â </td><td>0.65Â </td><td>3Â </td></tr><tr><td>Chromosome 2Â </td><td>7Â </td><td>102Â </td><td>83Â </td><td>67Â </td><td>259Â </td><td>50.1Â </td><td>154Â </td><td>0.33Â </td><td>2Â </td></tr><tr><td>Chromosome 3Â </td><td>17Â </td><td>115Â </td><td>92Â </td><td>114Â </td><td>338Â </td><td>98.9Â </td><td>206Â </td><td>0.48Â </td><td>1Â </td></tr><tr><td>Chromosome 4Â </td><td>12Â </td><td>71Â </td><td>100Â </td><td>58Â </td><td>241Â </td><td>82.5Â </td><td>157Â </td><td>0.53Â </td><td>6Â </td></tr><tr><td>Chromosome 5Â </td><td>10Â </td><td>60Â </td><td>65Â </td><td>49Â </td><td>184Â </td><td>75.2Â </td><td>146Â </td><td>0.52Â </td><td>1Â </td></tr><tr><td>Chromosome 6Â </td><td>10Â </td><td>48Â </td><td>69Â </td><td>66Â </td><td>193Â </td><td>70.1Â </td><td>146Â </td><td>0.48Â </td><td>1Â </td></tr><tr><td>Total IIIÂ </td><td>97Â </td><td>533Â </td><td>504Â </td><td>454Â </td><td>1588Â </td><td>508.5Â </td><td>1013Â </td><td>0.50Â </td><td>14Â </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>*The duplicated markers were excluded.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5193898" target="_blank">View Large</a></div></div><p>Eight morphological traits were recorded as QTL. After the map of each chromosome for each parent was constructed, several markers with almost an equal distance interval of &lt;10 cM were used as frame markers to scan eight phenotypic markers within the linkage map of each chromosome. As a result, 14 QTLs were detected on six chromosomes for the eight morphological traits. There were six QTLs detected on chromosome 4, only one QTL on chromosomes 3, 5 and 6 (TableÂ <a reveal-id="DSN022TB2" data-open="DSN022TB2" class="link link-reveal link-table xref-fig">2</a>). SS showed five QTLs and was distributed to five chromosomes except for chromosome 5. LL and SC presented two QTLs. SP, PT, ST, PE and TR showed only one QTL, indicating control by a single gene (TableÂ <a reveal-id="DSN022TB2" data-open="DSN022TB2" class="link link-reveal link-table xref-fig">2</a>, Fig.Â <a reveal-id="DSN022F3" data-open="DSN022F3" class="link link-reveal link-table xref-fig">3</a>).</p><p>The duplicated markers presented in six chromosomes, resulting in significant clustering in the whole genome, especially in the center region of each chromosome. To decrease the numbers of markers, all duplicated markers will be excluded on the each chromosome map (Fig.Â <a reveal-id="DSN022F3" data-open="DSN022F3" class="link link-reveal link-table xref-fig">3</a>). So Fig.Â <a reveal-id="DSN022F3" data-open="DSN022F3" class="link link-reveal link-table xref-fig">3</a> presented only unique markers. All requirements about the details of mapped markers should be address to the corresponding authors.</p>                            <h3 scrollto-destination=5193901 id="5193901" class="section-title" data-legacy-id=s3c>3.3. Characterization of the linkage map</h3>
<p>Chromosome 1 had the largest number of markers (373) and longest genetic distance (131.7 cM). Chromosome 5 had the fewest number of markers (184) and chromosome 2 has shortest genetic distance (50.1 cM). The average interval distance between markers (excluded the duplicated markers) was 0.50 cM for all chromosomes. Chromosome 1 (0.64 cM) had the longest and chromosome 2 (0.33 cM) had the shortest average interval distance between markers. The other four chromosomes had similar average interval distances (TableÂ <a reveal-id="DSN022TB2" data-open="DSN022TB2" class="link link-reveal link-table xref-fig">2</a>). The distorted markers were mainly mapped on distal parts of chromosome 5 (marked with asterisk on chromosome 5 in Fig.Â <a reveal-id="DSN022F3" data-open="DSN022F3" class="link link-reveal link-table xref-fig">3</a>). Chromosome 6 also showed some distorted markers, dispersed through the chromosome. The clustering of markers occurred on the center region of all six chromosomes (Fig.Â <a reveal-id="DSN022F3" data-open="DSN022F3" class="link link-reveal link-table xref-fig">3</a>).</p>                            <h2 scrollto-destination=5193903 id="5193903" class="section-title" data-legacy-id=s4>4. Discussion</h2>
                            <h3 scrollto-destination=5193904 id="5193904" class="section-title" data-legacy-id=s4a>4.1. Determination of the linkage map of each chromosome</h3>
<p>Before determining the chromosome of the genome, all AFLP and SSR markers were combined and analyzed with JoinMap<sup>Â®</sup> 3.0.<sup><a reveal-id="DSN022C16" data-open="DSN022C16" class="link link-ref link-reveal xref-bibr">16</a></sup> At the LOD of 14.0, six big groups were generated. On the basis of the distribution of SSR markers<sup><a reveal-id="DSN022C13" data-open="DSN022C13" class="link link-ref link-reveal xref-bibr">13</a></sup> within each group, we were able to locate six groups to six corresponding chromosomes of <em>L</em>. <em>japonicus</em>. After running the JoinMap<sup>Â®</sup> 3.0 program for each group, a linkage map on each chromosome was determined. The orders of these SSR markers on each chromosome were the same as those in the map of created by Hayashi et al.<sup><a reveal-id="DSN022C13" data-open="DSN022C13" class="link link-ref link-reveal xref-bibr">13</a></sup> except for some minor differences for some markers. Therefore, we confirmed the six groups correspond to the six chromosomes of <em>Lotus</em> genome.</p>                            <h3 scrollto-destination=5193906 id="5193906" class="section-title" data-legacy-id=s4b>4.2. The length of linkage map</h3>
<p>This linkage map spans a total genetic distance of 508.5 cM in <em>L</em>. <em>japonicus</em> genome, slightly longer than those of previous maps.<sup><a reveal-id="DSN022C13" data-open="DSN022C13" class="link link-ref link-reveal xref-bibr">13</a>,<a reveal-id="DSN022C14" data-open="DSN022C14" class="link link-ref link-reveal xref-bibr">14</a></sup> This can be explained by the marker numbers dramatically increased in the present mapping population. The larger number of markers within one linkage group may enlarge the genetic intercrossing value between markers. Additionally, the small size of the F<sub>6</sub> population (94 individuals) compared with 127 F<sub>2</sub> individuals in Hayashi et al.<sup><a reveal-id="DSN022C13" data-open="DSN022C13" class="link link-ref link-reveal xref-bibr">13</a></sup> may not be enough for allele segregation and cause allele partial distribution.</p><p>The lengths of chromosome 3â6 were very close to the lengths of Hayashi's results.<sup><a reveal-id="DSN022C13" data-open="DSN022C13" class="link link-ref link-reveal xref-bibr">13</a></sup> However, they were nearly 30 cM longer for chromosome 1 and 30 cM shorter for chromosome 2 than the reference map.<sup><a reveal-id="DSN022C13" data-open="DSN022C13" class="link link-ref link-reveal xref-bibr">13</a></sup> Hayashi et al.<sup><a reveal-id="DSN022C13" data-open="DSN022C13" class="link link-ref link-reveal xref-bibr">13</a></sup> reported that there was a translocation region between the chromosome 1 and 2 in both parents' map. This translocation could have caused unequal crossover in the second generation and inherited to sixth generation. Fragment deletion may also occur during the translocation. In the present map, chromosome 1 is 131.7 cM in length and chromosome 2 is only 50.1 cM in length. The difference of 30 cM is supposed to be the translocation fragment, compared with Hayashi et al.<sup><a reveal-id="DSN022C13" data-open="DSN022C13" class="link link-ref link-reveal xref-bibr">13</a></sup> chromosome 1 and 2 map. However, the map length of chromosome 1 and 2 are very close to that of Sandal et al.<sup><a reveal-id="DSN022C14" data-open="DSN022C14" class="link link-ref link-reveal xref-bibr">14</a></sup> map of <em>L</em>. <em>japonicus</em> from a cross of <em>L.filicaulis</em><em>Ã</em><em>L. japonicus</em>.</p><p>To further confirm the reliability of this linkage map, we developed a total of 300 AFLP co-dominant markers with the same primer-pair combinations and 97 SSR markers with 94 F<sub>9</sub> individuals derived from same F<sub>6</sub> individuals. With the same calculation, these 397 markers were located on six chromosomes and the order was generally the same within each chromosome, although there is a slight difference for some distances between markers. So this linkage map is reliable to use as a framework for physical map construction and map-based cloning in <em>L. japonicas</em>.</p>                            <h3 scrollto-destination=5193910 id="5193910" class="section-title" data-legacy-id=s4c>4.3. Clustering of the markers</h3>
<p>The clustering of markers on each chromosome occurred significantly in this study. AFLP markers characteristically cluster in centromeric and/or telomeric regions in plant species with large genome.<sup><a reveal-id="DSN022C21  DSN022C24" data-open="DSN022C21  DSN022C24" class="link link-ref link-reveal xref-bibr">21â24</a></sup> Clustering of markers occurred mainly at heterochromatin-rich centromeric regions that ascribed to the great portions of repetitive sequences frequently present, and these repetitive sequences suppressed recombination between chromosomes.<sup><a reveal-id="DSN022C21" data-open="DSN022C21" class="link link-ref link-reveal xref-bibr">21</a></sup> Also, a high degree of clustering of markers in the AFLP map is much more pronounced than in the RFLP map.<sup><a reveal-id="DSN022C25" data-open="DSN022C25" class="link link-ref link-reveal xref-bibr">25</a></sup> There may be some very small variation, possibly, 1 bp deletion/insertion in repetitive sequences that can be detected by the AFLP technique, but not by RFLP technique. Thus, AFLP markers can be relatively easy to generate in highly repetitive regions near centromere.<sup><a reveal-id="DSN022C22" data-open="DSN022C22" class="link link-ref link-reveal xref-bibr">22</a></sup> In the present study, of the 1588 mapped markers, 575 duplicated markers will be excluded in the Fig.Â <a reveal-id="DSN022F3" data-open="DSN022F3" class="link link-reveal link-table xref-fig">3</a>. Most significant clustering of markers located near the centromeric region.</p><p>Although the significant clusters presented, map gaps were found in whole genome, but much smaller than previous map. The biggest gap of 13.7 cM was found on chromosome 4. The gaps on other chromosomes ranged from 3.8 to 7.7 cM.</p><p>In this study, the rapid and efficient development of the linkage map with high resolution of <em>L</em>. <em>japonicus</em> was facilitated by the HEGS/AFLP system, by which a total of 1588 AFLP markers was mapped on the <em>L</em>. <em>japonicus</em> genome in 6 months. This map created a framework for anchoring EST, SSR and other sequence-based markers, and built the foundation for physical map construction in <em>L</em>. <em>japonicus</em> and gene cloning in other legume crops.</p>                            <h2 scrollto-destination=5193914 id="5193914" class="section-title" data-legacy-id=s5>Supplementary Data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available online at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=5193916 id="5193916" class="section-title" data-legacy-id=s6>Funding</h2>
<p>This work was supported by the fund for the Promotion of Basic Research Activities for Innovative Biosciences (BRAIN), Japan.</p>                            <h2 scrollto-destination=5193918 id="5193918" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSN022C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN022C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Vos</div> <div class="given-names">P.</div></div>,Â  <div class="name"><div class="surname">Hogers</div> <div class="given-names">R.</div></div>,Â  <div class="name"><div class="surname">Bleeker</div> <div class="given-names">M.</div></div>,Â et al.Â </div><div class="article-title">AFLP: a new technique for DNA fingerprinting</div>,Â <div class="source">Nucleic Acids Res.</div>Â ,Â <div class="year">1995</div>,Â vol.Â <div class="volume">23</div>Â (pg.Â <div class="fpage">4407</div>-<div class="lpage">4414</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=AFLP%3A%20a%20new%20technique%20for%20DNA%20fingerprinting&amp;author=P.%20Vos&amp;author=R.%20Hogers&amp;author=M.%20Bleeker&amp;publication_year=1995&amp;journal=Nucleic%20Acids%20Res.&amp;volume=23&amp;pages=4407-4414" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/23.21.4407" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F23.21.4407" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F23.21.4407"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7501463" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSN022C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN022C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mackill</div> <div class="given-names">D. J.</div></div>,Â  <div class="name"><div class="surname">Zhang</div> <div class="given-names">Z.</div></div>,Â  <div class="name"><div class="surname">Redona</div> <div class="given-names">E. D.</div></div>,Â  <div class="name"><div class="surname">Colowit</div> <div class="given-names">P. M.</div></div>.Â </div><div class="article-title">Level of polymorphism and genetic mapping of AFLP markers in rice</div>,Â <div class="source">Genome</div>Â ,Â <div class="year">1996</div>,Â vol.Â <div class="volume">39</div>Â (pg.Â <div class="fpage">969</div>-<div class="lpage">977</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Level%20of%20polymorphism%20and%20genetic%20mapping%20of%20AFLP%20markers%20in%20rice&amp;author=D.%20J.%20Mackill&amp;author=Z.%20Zhang&amp;author=E.%20D.%20Redona&amp;author=P.%20M.%20Colowit&amp;publication_year=1996&amp;journal=Genome&amp;volume=39&amp;pages=969-977" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/g96-121" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2Fg96-121" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2Fg96-121"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8890522" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSN022C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN022C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Keim</div> <div class="given-names">P.</div></div>,Â  <div class="name"><div class="surname">Schupp</div> <div class="given-names">J. M.</div></div>,Â  <div class="name"><div class="surname">Travis</div> <div class="given-names">S. E.</div></div>,Â et al.Â </div><div class="article-title">A high-density soybean genetic map based on AFLP markers</div>,Â <div class="source">Crop. Sci.</div>Â ,Â <div class="year">1997</div>,Â vol.Â <div class="volume">37</div>Â (pg.Â <div class="fpage">537</div>-<div class="lpage">543</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20high-density%20soybean%20genetic%20map%20based%20on%20AFLP%20markers&amp;author=P.%20Keim&amp;author=J.%20M.%20Schupp&amp;author=S.%20E.%20Travis&amp;publication_year=1997&amp;journal=Crop.%20Sci.&amp;volume=37&amp;pages=537-543" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2135/cropsci1997.0011183X003700020038x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2135%2Fcropsci1997.0011183X003700020038x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2135%2Fcropsci1997.0011183X003700020038x"> </span></div>Â </div></div></div></div></div><div content-id="DSN022C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN022C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Remington</div> <div class="given-names">D. L.</div></div>,Â  <div class="name"><div class="surname">Whetten</div> <div class="given-names">R. W.</div></div>,Â  <div class="name"><div class="surname">Liu</div> <div class="given-names">B. H.</div></div>,Â  <div class="name"><div class="surname">O'Malley</div> <div class="given-names">D. M.</div></div>.Â </div><div class="article-title">Construction of an AFLP genetic map with nearly complete genome coverage in <em>Pinus taeda</em></div>,Â <div class="source">Theor. Appl. Genet.</div>Â ,Â <div class="year">1999</div>,Â vol.Â <div class="volume">98</div>Â (pg.Â <div class="fpage">1279</div>-<div class="lpage">1292</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20an%20AFLP%20genetic%20map%20with%20nearly%20complete%20genome%20coverage%20in%20Pinus%20taeda&amp;author=D.%20L.%20Remington&amp;author=R.%20W.%20Whetten&amp;author=B.%20H.%20Liu&amp;author=D.%20M.%20O'Malley&amp;publication_year=1999&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=98&amp;pages=1279-1292" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s001220051194" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs001220051194" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs001220051194"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12238515" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSN022C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN022C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Katengam</div> <div class="given-names">S.</div></div>,Â  <div class="name"><div class="surname">Crane</div> <div class="given-names">J. M.</div></div>,Â  <div class="name"><div class="surname">Knapp</div> <div class="given-names">S. J.</div></div>.Â </div><div class="article-title">The development of a genetic map for meadowfoam comprised of amplified fragment length polymorphisms</div>,Â <div class="source">Theor. Appl. Genet.</div>Â ,Â <div class="year">2002</div>,Â vol.Â <div class="volume">104</div>Â (pg.Â <div class="fpage">92</div>-<div class="lpage">96</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20development%20of%20a%20genetic%20map%20for%20meadowfoam%20comprised%20of%20amplified%20fragment%20length%20polymorphisms&amp;author=S.%20Katengam&amp;author=J.%20M.%20Crane&amp;author=S.%20J.%20Knapp&amp;publication_year=2002&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=104&amp;pages=92-96" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s001220200010" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs001220200010" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs001220200010"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12579432" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSN022C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN022C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kriegner</div> <div class="given-names">A.</div></div>,Â  <div class="name"><div class="surname">Cervantes</div> <div class="given-names">J. C.</div></div>,Â  <div class="name"><div class="surname">Burg</div> <div class="given-names">K.</div></div>,Â  <div class="name"><div class="surname">Mwanga</div> <div class="given-names">R. O. M.</div></div>,Â  <div class="name"><div class="surname">Zhang</div> <div class="given-names">D.</div></div>.Â </div><div class="article-title">A genetic linkage map of sweetpotato [<em>Ipomoea batatas</em> (L.) Lam.] based on AFLP markers</div>,Â <div class="source">Mol. Breed.</div>Â ,Â <div class="year">2003</div>,Â vol.Â <div class="volume">11</div>Â (pg.Â <div class="fpage">169</div>-<div class="lpage">185</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20genetic%20linkage%20map%20of%20sweetpotato%20%5BIpomoea%20batatas%20(L.)%20Lam.%5D%20based%20on%20AFLP%20markers&amp;author=A.%20Kriegner&amp;author=J.%20C.%20Cervantes&amp;author=K.%20Burg&amp;author=R.%20O.%20M.%20Mwanga&amp;author=D.%20Zhang&amp;publication_year=2003&amp;journal=Mol.%20Breed.&amp;volume=11&amp;pages=169-185" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1023/A:1022870917230" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1023%2FA:1022870917230" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1023%2FA:1022870917230"> </span></div>Â </div></div></div></div></div><div content-id="DSN022C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN022C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Handberg</div> <div class="given-names">K.</div></div>,Â  <div class="name"><div class="surname">Stougaard</div> <div class="given-names">J.</div></div>.Â </div><div class="article-title"><em>Lotus japonicus</em>, an autogamous, diploid legume species for classical and molecular genetics</div>,Â <div class="source">Plant J.</div>Â ,Â <div class="year">1992</div>,Â vol.Â <div class="volume">2</div>Â (pg.Â <div class="fpage">487</div>-<div class="lpage">496</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Lotus%20japonicus%2C%20an%20autogamous%2C%20diploid%20legume%20species%20for%20classical%20and%20molecular%20genetics&amp;author=K.%20Handberg&amp;author=J.%20Stougaard&amp;publication_year=1992&amp;journal=Plant%20J.&amp;volume=2&amp;pages=487-496" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.1992.2.issue-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.1992.2.issue-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.1992.2.issue-4"> </span></div>Â </div></div></div></div></div><div content-id="DSN022C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN022C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jiang</div> <div class="given-names">Q.</div></div>,Â  <div class="name"><div class="surname">Gressholf</div> <div class="given-names">P. M.</div></div>.Â </div><div class="article-title">Classical and molecular genetics of the model legume <em>Lotus japonicus</em></div>,Â <div class="source">Mol. Plant Microbe Interact.</div>Â ,Â <div class="year">1997</div>,Â vol.Â <div class="volume">10</div>Â (pg.Â <div class="fpage">59</div>-<div class="lpage">68</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Classical%20and%20molecular%20genetics%20of%20the%20model%20legume%20Lotus%20japonicus&amp;author=Q.%20Jiang&amp;author=P.%20M.%20Gressholf&amp;publication_year=1997&amp;journal=Mol.%20Plant%20Microbe%20Interact.&amp;volume=10&amp;pages=59-68" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1094/MPMI.1997.10.1.59" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1094%2FMPMI.1997.10.1.59" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1094%2FMPMI.1997.10.1.59"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9002271" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSN022C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN022C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ito</div> <div class="given-names">M.</div></div>,Â  <div class="name"><div class="surname">Miyamoto</div> <div class="given-names">J.</div></div>,Â  <div class="name"><div class="surname">Mori</div> <div class="given-names">Y.</div></div>,Â  <div class="name"><div class="surname">Fujimoto</div> <div class="given-names">S.</div></div>,Â  <div class="name"><div class="surname">Uchiumi</div> <div class="given-names">T.</div></div>,Â  <div class="name"><div class="surname">Abe</div> <div class="given-names">M.</div></div>,Â  <div class="name"><div class="surname">Suzuki</div> <div class="given-names">A.</div></div>,Â  <div class="name"><div class="surname">Tabata</div> <div class="given-names">S.</div></div>,Â  <div class="name"><div class="surname">Fukui</div> <div class="given-names">K.</div></div>.Â </div><div class="article-title">Genome and chromosome dimensions of <em>Lotus japonicus</em></div>,Â <div class="source">J. Plant Res.</div>Â ,Â <div class="year">2000</div>,Â vol.Â <div class="volume">113</div>Â (pg.Â <div class="fpage">435</div>-<div class="lpage">442</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20and%20chromosome%20dimensions%20of%20Lotus%20japonicus&amp;author=M.%20Ito&amp;author=J.%20Miyamoto&amp;author=Y.%20Mori&amp;author=S.%20Fujimoto&amp;author=T.%20Uchiumi&amp;author=M.%20Abe&amp;author=A.%20Suzuki&amp;author=S.%20Tabata&amp;author=K.%20Fukui&amp;publication_year=2000&amp;journal=J.%20Plant%20Res.&amp;volume=113&amp;pages=435-442" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/PL00013952" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FPL00013952" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FPL00013952"> </span></div>Â </div></div></div></div></div><div content-id="DSN022C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN022C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kawasaki</div> <div class="given-names">S.</div></div>,Â  <div class="name"><div class="surname">Murakami</div> <div class="given-names">Y.</div></div>.Â </div><div class="article-title">Genome analysis of <em>Lotus japonicus</em></div>,Â <div class="source">J. Plant Res.</div>Â ,Â <div class="year">2000</div>,Â vol.Â <div class="volume">113</div>Â (pg.Â <div class="fpage">497</div>-<div class="lpage">506</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20analysis%20of%20Lotus%20japonicus&amp;author=S.%20Kawasaki&amp;author=Y.%20Murakami&amp;publication_year=2000&amp;journal=J.%20Plant%20Res.&amp;volume=113&amp;pages=497-506" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/PL00013960" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FPL00013960" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FPL00013960"> </span></div>Â </div></div></div></div></div><div content-id="DSN022C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN022C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Stiller</div> <div class="given-names">J.</div></div>,Â  <div class="name"><div class="surname">Martirani</div> <div class="given-names">L.</div></div>,Â  <div class="name"><div class="surname">Tuppale</div> <div class="given-names">S.</div></div>,Â  <div class="name"><div class="surname">Chian</div> <div class="given-names">R. J.</div></div>,Â  <div class="name"><div class="surname">Chiurazzi</div> <div class="given-names">M.</div></div>,Â  <div class="name"><div class="surname">Gresshoff</div> <div class="given-names">P. M.</div></div>.Â </div><div class="article-title">High frequency transformation and regeneration of transgenic plants in the model legume <em>Lotus japonicus</em></div>,Â <div class="source">J. Exp. Bot.</div>Â ,Â <div class="year">1997</div>,Â vol.Â <div class="volume">48</div>Â (pg.Â <div class="fpage">1357</div>-<div class="lpage">1365</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High%20frequency%20transformation%20and%20regeneration%20of%20transgenic%20plants%20in%20the%20model%20legume%20Lotus%20japonicus&amp;author=J.%20Stiller&amp;author=L.%20Martirani&amp;author=S.%20Tuppale&amp;author=R.%20J.%20Chian&amp;author=M.%20Chiurazzi&amp;author=P.%20M.%20Gresshoff&amp;publication_year=1997&amp;journal=J.%20Exp.%20Bot.&amp;volume=48&amp;pages=1357-1365" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jxb/48.7.1357" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjxb%2F48.7.1357" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjxb%2F48.7.1357"> </span></div>Â </div></div></div></div></div><div content-id="DSN022C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN022C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kawaguchi</div> <div class="given-names">M.</div></div>,Â  <div class="name"><div class="surname">Motomura</div> <div class="given-names">T.</div></div>,Â  <div class="name"><div class="surname">Imaizumi-Anraku</div> <div class="given-names">H.</div></div>,Â  <div class="name"><div class="surname">Akao</div> <div class="given-names">S.</div></div>,Â  <div class="name"><div class="surname">Kawasaki</div> <div class="given-names">S.</div></div>.Â </div><div class="article-title">Providing the basis for genomics in <em>Lotus japonicus</em>: the accessions Miyakojima MG-20 and Gifu B-129 are appropriate crossing partners for genetic analyses</div>,Â <div class="source">Mol. Gen. Genet.</div>Â ,Â <div class="year">2001</div>,Â vol.Â <div class="volume">266</div>Â (pg.Â <div class="fpage">157</div>-<div class="lpage">166</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Providing%20the%20basis%20for%20genomics%20in%20Lotus%20japonicus%3A%20the%20accessions%20Miyakojima%20MG-20%20and%20Gifu%20B-129%20are%20appropriate%20crossing%20partners%20for%20genetic%20analyses&amp;author=M.%20Kawaguchi&amp;author=T.%20Motomura&amp;author=H.%20Imaizumi-Anraku&amp;author=S.%20Akao&amp;author=S.%20Kawasaki&amp;publication_year=2001&amp;journal=Mol.%20Gen.%20Genet.&amp;volume=266&amp;pages=157-166" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s004380100540" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs004380100540" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs004380100540"> </span></div>Â </div></div></div></div></div><div content-id="DSN022C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN022C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hayashi</div> <div class="given-names">M.</div></div>,Â  <div class="name"><div class="surname">Miyahara</div> <div class="given-names">A.</div></div>,Â  <div class="name"><div class="surname">Sato</div> <div class="given-names">S.</div></div>,Â et al.Â </div><div class="article-title">Construction of a genetic linkage map of the model legume <em>Lotus japonicus</em> using an intraspecific F<sub>2</sub> population</div>,Â <div class="source">DNA Res.</div>Â ,Â <div class="year">2001</div>,Â vol.Â <div class="volume">8</div>Â (pg.Â <div class="fpage">301</div>-<div class="lpage">310</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20a%20genetic%20linkage%20map%20of%20the%20model%20legume%20Lotus%20japonicus%20using%20an%20intraspecific%20F2%20population&amp;author=M.%20Hayashi&amp;author=A.%20Miyahara&amp;author=S.%20Sato&amp;publication_year=2001&amp;journal=DNA%20Res.&amp;volume=8&amp;pages=301-310" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/8.6.301" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F8.6.301" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2F8.6.301"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11853317" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSN022C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN022C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sandal</div> <div class="given-names">N.</div></div>,Â  <div class="name"><div class="surname">Krusell</div> <div class="given-names">L.</div></div>,Â  <div class="name"><div class="surname">Radutoiu</div> <div class="given-names">S.</div></div>,Â et al.Â </div><div class="article-title">A genetic linkage map of the model legume <em>Lotus japonicus</em> and strategies for fast mapping of new loci</div>,Â <div class="source">Genetics</div>Â ,Â <div class="year">2002</div>,Â vol.Â <div class="volume">161</div>Â (pg.Â <div class="fpage">1673</div>-<div class="lpage">1683</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20genetic%20linkage%20map%20of%20the%20model%20legume%20Lotus%20japonicus%20and%20strategies%20for%20fast%20mapping%20of%20new%20loci&amp;author=N.%20Sandal&amp;author=L.%20Krusell&amp;author=S.%20Radutoiu&amp;publication_year=2002&amp;journal=Genetics&amp;volume=161&amp;pages=1673-1683" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12196410" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSN022C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN022C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sato</div> <div class="given-names">S.</div></div>,Â  <div class="name"><div class="surname">Kaneko</div> <div class="given-names">T.</div></div>,Â  <div class="name"><div class="surname">Nakamura</div> <div class="given-names">Y.</div></div>,Â  <div class="name"><div class="surname">Asamizu</div> <div class="given-names">E.</div></div>,Â  <div class="name"><div class="surname">Kato</div> <div class="given-names">T.</div></div>,Â  <div class="name"><div class="surname">Tabata</div> <div class="given-names">S.</div></div>.Â </div><div class="article-title">Structural analysis of a <em>Lotus japonicus</em> genome. I. Sequence features and mapping of fifty-six TAC clones which cover the 5.4 Mb regions of the genome</div>,Â <div class="source">DNA Res.</div>Â ,Â <div class="year">2001</div>,Â vol.Â <div class="volume">8</div>Â (pg.Â <div class="fpage">311</div>-<div class="lpage">318</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structural%20analysis%20of%20a%20Lotus%20japonicus%20genome.%20I.%20Sequence%20features%20and%20mapping%20of%20fifty-six%20TAC%20clones%20which%20cover%20the%205.4%20Mb%20regions%20of%20the%20genome&amp;author=S.%20Sato&amp;author=T.%20Kaneko&amp;author=Y.%20Nakamura&amp;author=E.%20Asamizu&amp;author=T.%20Kato&amp;author=S.%20Tabata&amp;publication_year=2001&amp;journal=DNA%20Res.&amp;volume=8&amp;pages=311-318" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/8.6.311" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F8.6.311" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2F8.6.311"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11853318" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSN022C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN022C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Van Ooijen</div> <div class="given-names">J. W.</div></div>,Â  <div class="name"><div class="surname">Voorrips</div> <div class="given-names">R. E.</div></div>.Â </div><div class="article-title">JoinMap<sup>Â®</sup> 3.0, Software for the calculation of genetic linkage maps, Plant Research International</div>,Â <div class="year">2001</div><div class="publisher-loc">Wageningen</div><div class="publisher-name">the Netherlands,</div></div></div></div></div><div content-id="DSN022C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN022C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kosambi</div> <div class="given-names">D. D.</div></div>.Â </div><div class="article-title">The estimation of map distances from recombination values</div>,Â <div class="source">Ann. Eugen.</div>Â ,Â <div class="year">1944</div>,Â vol.Â <div class="volume">12</div>Â (pg.Â <div class="fpage">172</div>-<div class="lpage">175</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20estimation%20of%20map%20distances%20from%20recombination%20values&amp;author=D.%20D.%20Kosambi&amp;publication_year=1944&amp;journal=Ann.%20Eugen.&amp;volume=12&amp;pages=172-175" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1469-1809.1943.tb02321.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1469-1809.1943.tb02321.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1469-1809.1943.tb02321.x"> </span></div>Â </div></div></div></div></div><div content-id="DSN022C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN022C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Voorrips</div> <div class="given-names">R. E.</div></div>.Â </div><div class="article-title">MapChart: Software for the graphical presentation of linkage maps and QTLs</div>,Â <div class="source">J. Hered.</div>Â ,Â <div class="year">2002</div>,Â vol.Â <div class="volume">93</div>Â (pg.Â <div class="fpage">77</div>-<div class="lpage">78</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MapChart%3A%20Software%20for%20the%20graphical%20presentation%20of%20linkage%20maps%20and%20QTLs&amp;author=R.%20E.%20Voorrips&amp;publication_year=2002&amp;journal=J.%20Hered.&amp;volume=93&amp;pages=77-78" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jhered/93.1.77" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjhered%2F93.1.77" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjhered%2F93.1.77"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12011185" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSN022C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN022C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zeng</div> <div class="given-names">B.</div></div>.Â </div><div class="article-title">Precision mapping of quantitative trait loci</div>,Â <div class="source">Genetics</div>Â ,Â <div class="year">1994</div>,Â vol.Â <div class="volume">136</div>Â (pg.Â <div class="fpage">1457</div>-<div class="lpage">1466</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Precision%20mapping%20of%20quantitative%20trait%20loci&amp;author=B.%20Zeng&amp;publication_year=1994&amp;journal=Genetics&amp;volume=136&amp;pages=1457-1466" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8013918" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSN022C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN022C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">S.</div></div>,Â  <div class="name"><div class="surname">Basten</div> <div class="given-names">C. J.</div></div>,Â  <div class="name"><div class="surname">Zeng</div> <div class="given-names">B.</div></div>.Â </div><div class="article-title">Windows QTL Cartographer 2.5</div>,Â <div class="year">2006</div><div class="publisher-loc">NC</div><div class="publisher-name">Department of Statistics, North Carolina State University, Raleigh,</div>Â <div class="comment"><a class="link link-uri" href="http://statgen.ncsu.edu/qtlcart/WQTLCart.htm">http://statgen.ncsu.edu/qtlcart/WQTLCart.htm</a></div></div></div></div></div><div content-id="DSN022C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN022C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tanksley</div> <div class="given-names">S. D.</div></div>,Â  <div class="name"><div class="surname">Ganal</div> <div class="given-names">M. W.</div></div>,Â  <div class="name"><div class="surname">Prince</div> <div class="given-names">J. P.</div></div>,Â et al.Â </div><div class="article-title">High-density molecular linkage maps of the tomato and potato genomes</div>,Â <div class="source">Genetics</div>Â ,Â <div class="year">1992</div>,Â vol.Â <div class="volume">132</div>Â (pg.Â <div class="fpage">1141</div>-<div class="lpage">1160</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-density%20molecular%20linkage%20maps%20of%20the%20tomato%20and%20potato%20genomes&amp;author=S.%20D.%20Tanksley&amp;author=M.%20W.%20Ganal&amp;author=J.%20P.%20Prince&amp;publication_year=1992&amp;journal=Genetics&amp;volume=132&amp;pages=1141-1160" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1360934" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSN022C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN022C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Qi</div> <div class="given-names">X.</div></div>,Â  <div class="name"><div class="surname">Stam</div> <div class="given-names">P.</div></div>,Â  <div class="name"><div class="surname">lindhout</div> <div class="given-names">P.</div></div>.Â </div><div class="article-title">Use of locus-specific AFLP markers to construct a high-density molecular map in barley</div>,Â <div class="source">Theor. Appl. Genet.</div>Â ,Â <div class="year">1998</div>,Â vol.Â <div class="volume">96</div>Â (pg.Â <div class="fpage">376</div>-<div class="lpage">384</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Use%20of%20locus-specific%20AFLP%20markers%20to%20construct%20a%20high-density%20molecular%20map%20in%20barley&amp;author=X.%20Qi&amp;author=P.%20Stam&amp;author=P.%20lindhout&amp;publication_year=1998&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=96&amp;pages=376-384" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s001220050752" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs001220050752" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs001220050752"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24710875" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSN022C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN022C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gedil</div> <div class="given-names">M. A.</div></div>,Â  <div class="name"><div class="surname">Wye</div> <div class="given-names">C.</div></div>,Â  <div class="name"><div class="surname">Berry</div></div>,Â et al.Â </div><div class="article-title">An integrated RFLP-AFLP linkage map for cultivated sunflower</div>,Â <div class="source">Genome</div>Â ,Â <div class="year">2001</div>,Â vol.Â <div class="volume">44</div>Â (pg.Â <div class="fpage">213</div>-<div class="lpage">221</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20integrated%20RFLP-AFLP%20linkage%20map%20for%20cultivated%20sunflower&amp;author=M.%20A.%20Gedil&amp;author=C.%20Wye&amp;author=%20Berry&amp;publication_year=2001&amp;journal=Genome&amp;volume=44&amp;pages=213-221" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/g00-111" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2Fg00-111" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2Fg00-111"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11341731" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSN022C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN022C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Strommer</div> <div class="given-names">J.</div></div>,Â  <div class="name"><div class="surname">Peters</div> <div class="given-names">J.</div></div>,Â  <div class="name"><div class="surname">Zethof</div> <div class="given-names">J.</div></div>,Â  <div class="name"><div class="surname">de Keukekeire</div> <div class="given-names">P.</div></div>.Â </div><div class="article-title">AFLP maps of <em>Petunia hybrida</em>: building maps when markers cluster</div>,Â <div class="source">Theor. Appl. Genet.</div>Â ,Â <div class="year">2002</div>,Â vol.Â <div class="volume">105</div>Â (pg.Â <div class="fpage">1000</div>-<div class="lpage">1009</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=AFLP%20maps%20of%20Petunia%20hybrida%3A%20building%20maps%20when%20markers%20cluster&amp;author=J.%20Strommer&amp;author=J.%20Peters&amp;author=J.%20Zethof&amp;author=P.%20de%20Keukekeire&amp;publication_year=2002&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=105&amp;pages=1000-1009" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-002-1009-y" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-002-1009-y" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-002-1009-y"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12582927" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSN022C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN022C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Devos</div> <div class="given-names">K. M.</div></div>,Â  <div class="name"><div class="surname">Atkinson</div> <div class="given-names">M. D.</div></div>,Â  <div class="name"><div class="surname">Chinoy</div> <div class="given-names">C. N.</div></div>,Â  <div class="name"><div class="surname">Liu</div> <div class="given-names">C. J.</div></div>,Â  <div class="name"><div class="surname">Gale</div> <div class="given-names">M. D.</div></div>.Â </div><div class="article-title">RFLP-based genetic map of the homoeologous group-3 chromosomes of wheat and rye</div>,Â <div class="source">Theor. Appl. Genet.</div>Â ,Â <div class="year">1992</div>,Â vol.Â <div class="volume">83</div>Â (pg.Â <div class="fpage">931</div>-<div class="lpage">939</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RFLP-based%20genetic%20map%20of%20the%20homoeologous%20group-3%20chromosomes%20of%20wheat%20and%20rye&amp;author=K.%20M.%20Devos&amp;author=M.%20D.%20Atkinson&amp;author=C.%20N.%20Chinoy&amp;author=C.%20J.%20Liu&amp;author=M.%20D.%20Gale&amp;publication_year=1992&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=83&amp;pages=931-939" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00232953" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00232953" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00232953"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24202916" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">Â© The Author 2008. Kazusa DNA Research Institute</div><div class="license"><div class="license-p">The online version of this article has been published under an open access model. Users are entitled to use, reproduce, disseminate, or display the open access version of this article for non-commercial purposes provided that: the original authorship is properly and fully attributed; the Journal and Oxford University Press are attributed as the original place of publication with the correct citation details given; if an article is subsequently reproduced or disseminated not in its entirety but only in part or as a derivative work this must be clearly indicated. For commercial re-use, please contact journals.permissions@oxfordjournals.org</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=359995&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn022/2/dsn022-dsn022supp_Table2.xls?Expires=2147483647&amp;Signature=P~fZGa2Y~MBX2BTOc3eTRPvJL8sdEZPl-jjhsStvWr2HJcsdJgBxTzNmgGib612UMTtDvTnXN7XvHGs4MlP6cSfFX-lgwQTgl5zOcvtoMyUcqz9t~vtg6zR7bDjsQ5to3yQEy4mi8BQBNDtyX6mg3dJ6UxePv2BbeTTKooRr-dqt-C1qREB5V9IL9U0pdN3POGky431irCQnB3y~mq0P8mZ38ahX-foQZNsQ85oehOSqMRCZX9wNpiJnyZgc9eVYVdqlyZ14IWPblCf0O1sv5Ae-O8fNxQrqEqa7FfluVF-IMsj6Ur6NHPU460VJKrBnbQECZVuwRRCqyiB4bl43FA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn022/2/dsn022-dsn022supp_Table1.xls?Expires=2147483647&amp;Signature=zBT~oCnGFPrwsqVGbF-CDzTahUObXHEPbCvOLLtWVFwc1XEPL9sPe4vz-168J33AmJFfHWphqTIg4TOm7WVlQrSlizZ3nHiYap4djK9IdD0Pytc7jE1LaBUYAlIR~-L3dVab6w5bHAygzMQO7vaxQtCV96hEEk7fnKlOe5wi~HYBv2HTVSSUeGCSeR7N5DAPkF3oiWZiF3vtwkmbg2Ylm-rUozlo0FhGZ-sQi5OPy~u2TnPh60c5PWHW7Jl6NwcmFTOyFNkG9cm8VeqeeBIgQjxQSlGa~51khDyft1zlVMfqjJKbPfsga9DS2rzJqtu54YN8VYOtPOv3q2lzL-sN2Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - xls file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">163</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000260843400006&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">13</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsn022"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="359995"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000260843400006&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f15%2f5%2f323%2f359995">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000260843400006&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (13)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F15%2F5%2F323%2F359995" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/359995'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transpositionâduplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="High-density Linkage Map of Lotus japonicus Based on AFLP and SSR Markers" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsn022");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f15%2f5%2f323%2f359995">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

