

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/19/2/143/419384","@type":"ScholarlyArticle","name":"Detection of Base Substitution-Type Somatic Mosaicism of the  NLRP3  Gene with &gt;99.9% Statistical Confidence by Massively Parallel Sequencing","datePublished":"2012-01-24","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/19/2","@type":"PublicationIssue","issueNumber":"2","datePublished":"2012-04-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsr047","keywords":["next generation sequencing","mosaicism","DNA diagnosis","chronic infantile neurological cutaneous and articular syndrome"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/19/2/143/419384","author":[{"name":"Izawa, Kazushi","affiliation":"1  Department of Pediatrics ,  Kyoto University Graduate School of Medicine ,  54 Shogoin Sakyo, Kyoto 606-8507 ,  Japan","@type":"Person"},{"name":"Hijikata, Atsushi","affiliation":"2  Laboratory for Immunogenomics, RIKEN Research Center for Allergy and Immunology ,  RIKEN Yokohama Institute ,  1-7-22 Suehiro-cho Tarumi-ku, Yokohama, Kanagawa  230-0045 ,  Japan","@type":"Person"},{"name":"Tanaka, Naoko","affiliation":"1  Department of Pediatrics ,  Kyoto University Graduate School of Medicine ,  54 Shogoin Sakyo, Kyoto 606-8507 ,  Japan","@type":"Person"},{"name":"Kawai, Tomoki","affiliation":"1  Department of Pediatrics ,  Kyoto University Graduate School of Medicine ,  54 Shogoin Sakyo, Kyoto 606-8507 ,  Japan","@type":"Person"},{"name":"Saito, Megumu K","affiliation":"3  Clinical Application Department,  Center for iPS Cell Research and Application (CiRA), Kyoto University ,  Kyoto ,  Japan","@type":"Person"},{"name":"Goldbach-Mansky, Raphaela","affiliation":"4  Translational Autoinflammatory Disease Section NIH/NIAMS ,  Bethesda, MD ,  USA","@type":"Person"},{"name":"Aksentijevich, Ivona","affiliation":"5  The National Human Genome Research Institute ,  Bethesda, MD ,  USA","@type":"Person"},{"name":"Yasumi, Takahiro","affiliation":"1  Department of Pediatrics ,  Kyoto University Graduate School of Medicine ,  54 Shogoin Sakyo, Kyoto 606-8507 ,  Japan","@type":"Person"},{"name":"Nakahata, Tatsutoshi","affiliation":"3  Clinical Application Department,  Center for iPS Cell Research and Application (CiRA), Kyoto University ,  Kyoto ,  Japan","@type":"Person"},{"name":"Heike, Toshio","affiliation":"1  Department of Pediatrics ,  Kyoto University Graduate School of Medicine ,  54 Shogoin Sakyo, Kyoto 606-8507 ,  Japan","@type":"Person"},{"name":"Nishikomori, Ryuta","affiliation":"1  Department of Pediatrics ,  Kyoto University Graduate School of Medicine ,  54 Shogoin Sakyo, Kyoto 606-8507 ,  Japan","@type":"Person"},{"name":"Ohara, Osamu","affiliation":"2  Laboratory for Immunogenomics, RIKEN Research Center for Allergy and Immunology ,  RIKEN Yokohama Institute ,  1-7-22 Suehiro-cho Tarumi-ku, Yokohama, Kanagawa  230-0045 ,  Japan    6  Department of Human Genome Research ,  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba  292-0818 ,  Japan","@type":"Person"}],"description":"Abstract.  Chronic infantile neurological cutaneous and articular syndrome (CINCA), also known as neonatal-onset multisystem inflammatory disease (NOMID), is a ","pageStart":"143","pageEnd":"152","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/19/2/1/m_cover.gif?Expires=2147483647&Signature=EMTi8MiwskfL8whcuG7gPpMQVkoTywhsN9k2fcs1qMWyjLc9OmU3zzfeux1LcC8edMnaqIc6tYNqgsUJ8hCH0UQnGgbUVvjHa69ZDqc3rwPQg3T0QzS9XdepkqXV57OA92AfnyIdwEzEggsZ18zDGYx5fC~exu1TTRcnk2bBo6TdYsVgw52BF-63NakC5-nPvHQVSqNzjaFosams6g1DZ~UchQ1c7lmvgwn6cnAg013zUUb9XUUhijVu63SQWmhw0ByRocoE2U4pz7BLDSieaJZqPdVUaKjHeYkUlMHEhZhGv0eccckPuOqL2UhMZ0FzVdPiNB6hKa39ET8qYgw8~Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Detection of Base Substitution-Type Somatic Mosaicism of the  NLRP3  Gene with &gt;99.9% Statistical Confidence by Massively Parallel Sequencing","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/19/2/1/m_cover.gif?Expires=2147483647&Signature=EMTi8MiwskfL8whcuG7gPpMQVkoTywhsN9k2fcs1qMWyjLc9OmU3zzfeux1LcC8edMnaqIc6tYNqgsUJ8hCH0UQnGgbUVvjHa69ZDqc3rwPQg3T0QzS9XdepkqXV57OA92AfnyIdwEzEggsZ18zDGYx5fC~exu1TTRcnk2bBo6TdYsVgw52BF-63NakC5-nPvHQVSqNzjaFosams6g1DZ~UchQ1c7lmvgwn6cnAg013zUUb9XUUhijVu63SQWmhw0ByRocoE2U4pz7BLDSieaJZqPdVUaKjHeYkUlMHEhZhGv0eccckPuOqL2UhMZ0FzVdPiNB6hKa39ET8qYgw8~Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Detection of Base Substitution-Type Somatic Mosaicism of the  NLRP3  Gene with &amp;gt;99.9% Statistical Confidence by Massively Parallel Sequencing" />
    <meta property="og:description" content="Abstract.  Chronic infantile neurological cutaneous and articular syndrome (CINCA), also known as neonatal-onset multisystem inflammatory disease (NOMID), is a " />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/19/2/143/419384" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2012-01-24" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Izawa, Kazushi" /><meta name="citation_author_institution" content="Department of Pediatrics, Kyoto University Graduate School of Medicine, 54 Shogoin Sakyo, Kyoto 606-8507, Japan" /><meta name="citation_author" content="Hijikata, Atsushi" /><meta name="citation_author_institution" content="Laboratory for Immunogenomics, RIKEN Research Center for Allergy and Immunology, RIKEN Yokohama Institute, 1-7-22 Suehiro-cho Tarumi-ku, Yokohama, Kanagawa230-0045, Japan" /><meta name="citation_author" content="Tanaka, Naoko" /><meta name="citation_author_institution" content="Department of Pediatrics, Kyoto University Graduate School of Medicine, 54 Shogoin Sakyo, Kyoto 606-8507, Japan" /><meta name="citation_author" content="Kawai, Tomoki" /><meta name="citation_author_institution" content="Department of Pediatrics, Kyoto University Graduate School of Medicine, 54 Shogoin Sakyo, Kyoto 606-8507, Japan" /><meta name="citation_author" content="Saito, Megumu K" /><meta name="citation_author_institution" content="Clinical Application Department,Center for iPS Cell Research and Application (CiRA), Kyoto University, Kyoto, Japan" /><meta name="citation_author" content="Goldbach-Mansky, Raphaela" /><meta name="citation_author_institution" content="Translational Autoinflammatory Disease Section NIH/NIAMS, Bethesda, MD, USA" /><meta name="citation_author" content="Aksentijevich, Ivona" /><meta name="citation_author_institution" content="The National Human Genome Research Institute, Bethesda, MD, USA" /><meta name="citation_author" content="Yasumi, Takahiro" /><meta name="citation_author_institution" content="Department of Pediatrics, Kyoto University Graduate School of Medicine, 54 Shogoin Sakyo, Kyoto 606-8507, Japan" /><meta name="citation_author" content="Nakahata, Tatsutoshi" /><meta name="citation_author_institution" content="Clinical Application Department,Center for iPS Cell Research and Application (CiRA), Kyoto University, Kyoto, Japan" /><meta name="citation_author" content="Heike, Toshio" /><meta name="citation_author_institution" content="Department of Pediatrics, Kyoto University Graduate School of Medicine, 54 Shogoin Sakyo, Kyoto 606-8507, Japan" /><meta name="citation_author" content="Nishikomori, Ryuta" /><meta name="citation_author_institution" content="Department of Pediatrics, Kyoto University Graduate School of Medicine, 54 Shogoin Sakyo, Kyoto 606-8507, Japan" /><meta name="citation_author" content="Ohara, Osamu" /><meta name="citation_author_institution" content="Laboratory for Immunogenomics, RIKEN Research Center for Allergy and Immunology, RIKEN Yokohama Institute, 1-7-22 Suehiro-cho Tarumi-ku, Yokohama, Kanagawa230-0045, Japan" /><meta name="citation_author_institution" content="Department of Human Genome Research, Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba292-0818, Japan" /><meta name="citation_title" content="Detection of Base Substitution-Type Somatic Mosaicism of the NLRP3 Gene with &gt;99.9% Statistical Confidence by Massively Parallel Sequencing" /><meta name="citation_firstpage" content="143" /><meta name="citation_lastpage" content="152" /><meta name="citation_doi" content="10.1093/dnares/dsr047" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="19" /><meta name="citation_issue" content="2" /><meta name="citation_publication_date" content="2012/04/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Arthropathy with rash, chronic meningitis, eye lesions, and mental retardation; J. Pediatr.;  citation_year=1981;  citation_volume=99;  citation_pages=79-83; " /><meta name="citation_reference" content="citation_title=Neonatal onset multisystem inflammatory disease; Arthritis Rheum.;  citation_year=1983;  citation_volume=26;  citation_pages=668-73; " /><meta name="citation_reference" content="citation_title=NOMID–a neonatal syndrome of multisystem inflammation; Skeletal Radiol.;  citation_year=1989;  citation_volume=18;  citation_pages=359-64; " /><meta name="citation_reference" content="citation_title=Chronic infantile neurological cutaneous and articular syndrome is caused by mutations in CIAS1, a gene highly expressed in polymorphonuclear cells and chondrocytes; Am. J. Hum. Genet. United States;  citation_year=2002;  citation_pages=198-203; " /><meta name="citation_reference" content="citation_title=De novo CIAS1 mutations, cytokine activation, and evidence for genetic heterogeneity in patients with neonatal-onset multisystem inflammatory disease (NOMID)—a new member of the expanding family of pyrin-associated autoinflammatory diseases; Arthritis Rheum.;  citation_year=2002;  citation_volume=46;  citation_pages=3340-8; " /><meta name="citation_reference" content="citation_title=Mutation of a new gene encoding a putative pyrin-like protein causes familial cold autoinflammatory syndrome and Muckle-Wells syndrome; Nat. Genet.;  citation_year=2001;  citation_volume=29;  citation_pages=301-5; " /><meta name="citation_reference" content="citation_title=Current status of understanding the pathogenesis and management of patients with NOMID/CINCA; Curr. Rheumatol. Rep.;  citation_year=2011;  citation_volume=13;  citation_pages=123-31; " /><meta name="citation_reference" content="citation_title=The infevers autoinflammatory mutation online registry: Update with new genes and functions; Hum. Mutat.;  citation_year=2008;  citation_volume=29;  citation_pages=803-8; " /><meta name="citation_reference" content="citation_title=High incidence of NLRP3 somatic mosaicism in patients with chronic infantile neurologic, cutaneous, articular syndrome: results of an International Multicenter Collaborative Study; Arthritis Rheum.;  citation_year=2011;  citation_volume=63;  citation_pages=3625-32; " /><meta name="citation_reference" content="citation_title=Ultra deep sequencing detects a low rate of mosaic mutations in tuberous sclerosis complex; Hum. Genet.;  citation_year=2010;  citation_volume=127;  citation_pages=573-82; " /><meta name="citation_reference" content="citation_title=Subclonal phylogenetic structures in cancer revealed by ultra-deep sequencing; Proc. Natl. Acad. Sci. USA;  citation_year=2008;  citation_volume=105;  citation_pages=13081-6; " /><meta name="citation_reference" content="citation_title=Parallel sequencing used in detection of mosaic mutations: Comparison with four diagnostic DNA screening techniques; Hum. Mutat.;  citation_year=2009;  citation_volume=30;  citation_pages=1012-20; " /><meta name="citation_reference" content="citation_title=Disease-associated CIAS1 mutations induce monocyte death, revealing low-level mosaicism in mutation-negative cryopyrin-associated periodic syndrome patients; Blood;  citation_year=2008;  citation_volume=111;  citation_pages=2132-41; " /><meta name="citation_reference" content="citation_title=BLAT—the BLAST-like alignment tool; Genome Res.;  citation_year=2002;  citation_volume=12;  citation_pages=656-64; " /><meta name="citation_reference" content="citation_title=Disease-associated mutations in CIAS1 induce cathepsin B-dependent rapid cell death of human THP-1 monocytic cells; Blood;  citation_year=2007;  citation_volume=109;  citation_pages=2903-11; " /><meta name="citation_reference" content="citation_title=Accuracy and quality assessment of 454 GS-FLX Titanium pyrosequencing; BMC Genomics;  citation_year=2011;  citation_volume=12;  citation_pages=245" /><meta name="citation_reference" content="citation_title=Genome sequencing in microfabricated high-density picolitre reactors; Nature;  citation_year=2005;  citation_volume=437;  citation_pages=376-80; " /><meta name="citation_reference" content="citation_title=vipR: variant identification in pooled DNA using R; Bioinformatics;  citation_year=2011;  citation_volume=27;  citation_pages=i77-84; " /><meta name="citation_reference" content="citation_title=Substantial biases in ultra-short read data sets from high-throughput DNA sequencing; Nucleic Acids Res.;  citation_year=2008;  citation_volume=36;  citation_pages=e105" /><meta name="citation_reference" content="citation_title=The mutation spectrum revealed by paired genome sequences from a lung cancer patient; Nature;  citation_year=2010;  citation_volume=465;  citation_pages=473-7; " /><meta name="citation_reference" content="citation_title=Direct Measure of the de novo mutation rate in Autism and Schizophrenia Cohorts; Am. J. Hum. Genet.;  citation_year=2010;  citation_volume=87;  citation_pages=316-24; " /><meta name="citation_reference" content="citation_title=Variation in genome-wide mutation rates within and between human families; Nat. Genet.;  citation_year=2011;  citation_volume=43;  citation_pages=712-4; " /><meta name="citation_reference" content="citation_title=Specificity, efficiency, and fidelity of PCR; PCR Methods Appl.;  citation_year=1993;  citation_volume=3;  citation_pages=S18-29; " /><meta name="citation_reference" content="citation_title=Minor variant detection in amplicons using 454 massive parallel pyrosequencing: experiences and considerations for successful applications; BioTechniques;  citation_year=2011;  citation_volume=51;  citation_pages=167-77; " /><meta name="citation_reference" content="citation_title=A comprehensive catalogue of somatic mutations from a human cancer genome; Nature;  citation_year=2010;  citation_volume=463;  citation_pages=191-6; " /><meta name="citation_reference" content="citation_title=SNP discovery in pooled samples with mismatch repair detection; Genome Res.;  citation_year=2004;  citation_volume=14;  citation_pages=1404-12; " /><meta name="citation_reference" content="citation_title=A statistical method for the detection of variants from next-generation resequencing of DNA pools; Bioinformatics;  citation_year=2010;  citation_volume=26;  citation_pages=i318-24; " /><meta name="citation_reference" content="citation_title=Ultrasensitive detection of rare mutations using next-generation targeted resequencing; Nucleic Acids Res.;  citation_year=2012;  citation_volume=40;  citation_pages=e2" /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/19/2/143/1163062/dsr047.pdf" /><meta name="description" content="Abstract.  Chronic infantile neurological cutaneous and articular syndrome (CINCA), also known as neonatal-onset multisystem inflammatory disease (NOMID), is a " />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/19/2/143/419384" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/19/2/143/419384" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/19/2/143/419384");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsr047';
    </script>
<title>
	Detection of Base Substitution-Type Somatic Mosaicism of the NLRP3 Gene with &gt;99.9% Statistical Confidence by Massively Parallel Sequencing | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./419384" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BapTPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJJemF3YSwgS2F6dXNoaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgUGVkaWF0cmljcywgS3lvdG8gVW5pdmVyc2l0eSBHcmFkdWF0ZSBTY2hvb2wgb2YgTWVkaWNpbmUsIDU0IFNob2dvaW4gU2FreW8sIEt5b3RvIDYwNi04NTA3LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkhpamlrYXRhLCBBdHN1c2hpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTGFib3JhdG9yeSBmb3IgSW1tdW5vZ2Vub21pY3MsIFJJS0VOIFJlc2VhcmNoIENlbnRlciBmb3IgQWxsZXJneSBhbmQgSW1tdW5vbG9neSwgUklLRU4gWW9rb2hhbWEgSW5zdGl0dXRlLCAxLTctMjIgU3VlaGlyby1jaG8gVGFydW1pLWt1LCBZb2tvaGFtYSwgS2FuYWdhd2EyMzAtMDA0NSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJUYW5ha2EsIE5hb2tvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBQZWRpYXRyaWNzLCBLeW90byBVbml2ZXJzaXR5IEdyYWR1YXRlIFNjaG9vbCBvZiBNZWRpY2luZSwgNTQgU2hvZ29pbiBTYWt5bywgS3lvdG8gNjA2LTg1MDcsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS2F3YWksIFRvbW9raSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgUGVkaWF0cmljcywgS3lvdG8gVW5pdmVyc2l0eSBHcmFkdWF0ZSBTY2hvb2wgb2YgTWVkaWNpbmUsIDU0IFNob2dvaW4gU2FreW8sIEt5b3RvIDYwNi04NTA3LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlNhaXRvLCBNZWd1bXUgSyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkNsaW5pY2FsIEFwcGxpY2F0aW9uIERlcGFydG1lbnQsQ2VudGVyIGZvciBpUFMgQ2VsbCBSZXNlYXJjaCBhbmQgQXBwbGljYXRpb24gKENpUkEpLCBLeW90byBVbml2ZXJzaXR5LCBLeW90bywgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJHb2xkYmFjaC1NYW5za3ksIFJhcGhhZWxhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iVHJhbnNsYXRpb25hbCBBdXRvaW5mbGFtbWF0b3J5IERpc2Vhc2UgU2VjdGlvbiBOSUgvTklBTVMsIEJldGhlc2RhLCBNRCwgVVNBIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQWtzZW50aWpldmljaCwgSXZvbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJUaGUgTmF0aW9uYWwgSHVtYW4gR2Vub21lIFJlc2VhcmNoIEluc3RpdHV0ZSwgQmV0aGVzZGEsIE1ELCBVU0EiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJZYXN1bWksIFRha2FoaXJvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBQZWRpYXRyaWNzLCBLeW90byBVbml2ZXJzaXR5IEdyYWR1YXRlIFNjaG9vbCBvZiBNZWRpY2luZSwgNTQgU2hvZ29pbiBTYWt5bywgS3lvdG8gNjA2LTg1MDcsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTmFrYWhhdGEsIFRhdHN1dG9zaGkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJDbGluaWNhbCBBcHBsaWNhdGlvbiBEZXBhcnRtZW50LENlbnRlciBmb3IgaVBTIENlbGwgUmVzZWFyY2ggYW5kIEFwcGxpY2F0aW9uIChDaVJBKSwgS3lvdG8gVW5pdmVyc2l0eSwgS3lvdG8sIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSGVpa2UsIFRvc2hpbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgUGVkaWF0cmljcywgS3lvdG8gVW5pdmVyc2l0eSBHcmFkdWF0ZSBTY2hvb2wgb2YgTWVkaWNpbmUsIDU0IFNob2dvaW4gU2FreW8sIEt5b3RvIDYwNi04NTA3LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik5pc2hpa29tb3JpLCBSeXV0YSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgUGVkaWF0cmljcywgS3lvdG8gVW5pdmVyc2l0eSBHcmFkdWF0ZSBTY2hvb2wgb2YgTWVkaWNpbmUsIDU0IFNob2dvaW4gU2FreW8sIEt5b3RvIDYwNi04NTA3LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik9oYXJhLCBPc2FtdSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkxhYm9yYXRvcnkgZm9yIEltbXVub2dlbm9taWNzLCBSSUtFTiBSZXNlYXJjaCBDZW50ZXIgZm9yIEFsbGVyZ3kgYW5kIEltbXVub2xvZ3ksIFJJS0VOIFlva29oYW1hIEluc3RpdHV0ZSwgMS03LTIyIFN1ZWhpcm8tY2hvIFRhcnVtaS1rdSwgWW9rb2hhbWEsIEthbmFnYXdhMjMwLTAwNDUsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBIdW1hbiBHZW5vbWUgUmVzZWFyY2gsIEthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCAyLTYtNyBLYXp1c2Eta2FtYXRhcmksIEtpc2FyYXp1LCBDaGliYTI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iRGV0ZWN0aW9uIG9mIEJhc2UgU3Vic3RpdHV0aW9uLVR5cGUgU29tYXRpYyBNb3NhaWNpc20gb2YgdGhlIE5MUlAzIEdlbmUgd2l0aCAmZ3Q7OTkuOSUgU3RhdGlzdGljYWwgQ29uZmlkZW5jZSBieSBNYXNzaXZlbHkgUGFyYWxsZWwgU2VxdWVuY2luZyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjE0MyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iMTUyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHNyMDQ3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjE5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSIyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTIvMDQvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BcnRocm9wYXRoeSB3aXRoIHJhc2gsIGNocm9uaWMgbWVuaW5naXRpcywgZXllIGxlc2lvbnMsIGFuZCBtZW50YWwgcmV0YXJkYXRpb247IEouIFBlZGlhdHIuOyAgY2l0YXRpb25feWVhcj0xOTgxOyAgY2l0YXRpb25fdm9sdW1lPTk5OyAgY2l0YXRpb25fcGFnZXM9NzktODM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5lb25hdGFsIG9uc2V0IG11bHRpc3lzdGVtIGluZmxhbW1hdG9yeSBkaXNlYXNlOyBBcnRocml0aXMgUmhldW0uOyAgY2l0YXRpb25feWVhcj0xOTgzOyAgY2l0YXRpb25fdm9sdW1lPTI2OyAgY2l0YXRpb25fcGFnZXM9NjY4LTczOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OT01JROKAk2EgbmVvbmF0YWwgc3luZHJvbWUgb2YgbXVsdGlzeXN0ZW0gaW5mbGFtbWF0aW9uOyBTa2VsZXRhbCBSYWRpb2wuOyAgY2l0YXRpb25feWVhcj0xOTg5OyAgY2l0YXRpb25fdm9sdW1lPTE4OyAgY2l0YXRpb25fcGFnZXM9MzU5LTY0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaHJvbmljIGluZmFudGlsZSBuZXVyb2xvZ2ljYWwgY3V0YW5lb3VzIGFuZCBhcnRpY3VsYXIgc3luZHJvbWUgaXMgY2F1c2VkIGJ5IG11dGF0aW9ucyBpbiBDSUFTMSwgYSBnZW5lIGhpZ2hseSBleHByZXNzZWQgaW4gcG9seW1vcnBob251Y2xlYXIgY2VsbHMgYW5kIGNob25kcm9jeXRlczsgQW0uIEouIEh1bS4gR2VuZXQuIFVuaXRlZCBTdGF0ZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl9wYWdlcz0xOTgtMjAzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZSBub3ZvIENJQVMxIG11dGF0aW9ucywgY3l0b2tpbmUgYWN0aXZhdGlvbiwgYW5kIGV2aWRlbmNlIGZvciBnZW5ldGljIGhldGVyb2dlbmVpdHkgaW4gcGF0aWVudHMgd2l0aCBuZW9uYXRhbC1vbnNldCBtdWx0aXN5c3RlbSBpbmZsYW1tYXRvcnkgZGlzZWFzZSAoTk9NSUQp4oCUYSBuZXcgbWVtYmVyIG9mIHRoZSBleHBhbmRpbmcgZmFtaWx5IG9mIHB5cmluLWFzc29jaWF0ZWQgYXV0b2luZmxhbW1hdG9yeSBkaXNlYXNlczsgQXJ0aHJpdGlzIFJoZXVtLjsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT00NjsgIGNpdGF0aW9uX3BhZ2VzPTMzNDAtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TXV0YXRpb24gb2YgYSBuZXcgZ2VuZSBlbmNvZGluZyBhIHB1dGF0aXZlIHB5cmluLWxpa2UgcHJvdGVpbiBjYXVzZXMgZmFtaWxpYWwgY29sZCBhdXRvaW5mbGFtbWF0b3J5IHN5bmRyb21lIGFuZCBNdWNrbGUtV2VsbHMgc3luZHJvbWU7IE5hdC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTI5OyAgY2l0YXRpb25fcGFnZXM9MzAxLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUN1cnJlbnQgc3RhdHVzIG9mIHVuZGVyc3RhbmRpbmcgdGhlIHBhdGhvZ2VuZXNpcyBhbmQgbWFuYWdlbWVudCBvZiBwYXRpZW50cyB3aXRoIE5PTUlEL0NJTkNBOyBDdXJyLiBSaGV1bWF0b2wuIFJlcC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz0xMjMtMzE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBpbmZldmVycyBhdXRvaW5mbGFtbWF0b3J5IG11dGF0aW9uIG9ubGluZSByZWdpc3RyeTogVXBkYXRlIHdpdGggbmV3IGdlbmVzIGFuZCBmdW5jdGlvbnM7IEh1bS4gTXV0YXQuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTI5OyAgY2l0YXRpb25fcGFnZXM9ODAzLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhpZ2ggaW5jaWRlbmNlIG9mIE5MUlAzIHNvbWF0aWMgbW9zYWljaXNtIGluIHBhdGllbnRzIHdpdGggY2hyb25pYyBpbmZhbnRpbGUgbmV1cm9sb2dpYywgY3V0YW5lb3VzLCBhcnRpY3VsYXIgc3luZHJvbWU6IHJlc3VsdHMgb2YgYW4gSW50ZXJuYXRpb25hbCBNdWx0aWNlbnRlciBDb2xsYWJvcmF0aXZlIFN0dWR5OyBBcnRocml0aXMgUmhldW0uOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTYzOyAgY2l0YXRpb25fcGFnZXM9MzYyNS0zMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VWx0cmEgZGVlcCBzZXF1ZW5jaW5nIGRldGVjdHMgYSBsb3cgcmF0ZSBvZiBtb3NhaWMgbXV0YXRpb25zIGluIHR1YmVyb3VzIHNjbGVyb3NpcyBjb21wbGV4OyBIdW0uIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMjc7ICBjaXRhdGlvbl9wYWdlcz01NzMtODI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN1YmNsb25hbCBwaHlsb2dlbmV0aWMgc3RydWN0dXJlcyBpbiBjYW5jZXIgcmV2ZWFsZWQgYnkgdWx0cmEtZGVlcCBzZXF1ZW5jaW5nOyBQcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xMDU7ICBjaXRhdGlvbl9wYWdlcz0xMzA4MS02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QYXJhbGxlbCBzZXF1ZW5jaW5nIHVzZWQgaW4gZGV0ZWN0aW9uIG9mIG1vc2FpYyBtdXRhdGlvbnM6IENvbXBhcmlzb24gd2l0aCBmb3VyIGRpYWdub3N0aWMgRE5BIHNjcmVlbmluZyB0ZWNobmlxdWVzOyBIdW0uIE11dGF0LjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTEwMTItMjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpc2Vhc2UtYXNzb2NpYXRlZCBDSUFTMSBtdXRhdGlvbnMgaW5kdWNlIG1vbm9jeXRlIGRlYXRoLCByZXZlYWxpbmcgbG93LWxldmVsIG1vc2FpY2lzbSBpbiBtdXRhdGlvbi1uZWdhdGl2ZSBjcnlvcHlyaW4tYXNzb2NpYXRlZCBwZXJpb2RpYyBzeW5kcm9tZSBwYXRpZW50czsgQmxvb2Q7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTExOyAgY2l0YXRpb25fcGFnZXM9MjEzMi00MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QkxBVOKAlHRoZSBCTEFTVC1saWtlIGFsaWdubWVudCB0b29sOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTY1Ni02NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlzZWFzZS1hc3NvY2lhdGVkIG11dGF0aW9ucyBpbiBDSUFTMSBpbmR1Y2UgY2F0aGVwc2luIEItZGVwZW5kZW50IHJhcGlkIGNlbGwgZGVhdGggb2YgaHVtYW4gVEhQLTEgbW9ub2N5dGljIGNlbGxzOyBCbG9vZDsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0xMDk7ICBjaXRhdGlvbl9wYWdlcz0yOTAzLTExOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BY2N1cmFjeSBhbmQgcXVhbGl0eSBhc3Nlc3NtZW50IG9mIDQ1NCBHUy1GTFggVGl0YW5pdW0gcHlyb3NlcXVlbmNpbmc7IEJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTI0NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZSBzZXF1ZW5jaW5nIGluIG1pY3JvZmFicmljYXRlZCBoaWdoLWRlbnNpdHkgcGljb2xpdHJlIHJlYWN0b3JzOyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9NDM3OyAgY2l0YXRpb25fcGFnZXM9Mzc2LTgwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT12aXBSOiB2YXJpYW50IGlkZW50aWZpY2F0aW9uIGluIHBvb2xlZCBETkEgdXNpbmcgUjsgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9Mjc7ICBjaXRhdGlvbl9wYWdlcz1pNzctODQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN1YnN0YW50aWFsIGJpYXNlcyBpbiB1bHRyYS1zaG9ydCByZWFkIGRhdGEgc2V0cyBmcm9tIGhpZ2gtdGhyb3VnaHB1dCBETkEgc2VxdWVuY2luZzsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTM2OyAgY2l0YXRpb25fcGFnZXM9ZTEwNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBtdXRhdGlvbiBzcGVjdHJ1bSByZXZlYWxlZCBieSBwYWlyZWQgZ2Vub21lIHNlcXVlbmNlcyBmcm9tIGEgbHVuZyBjYW5jZXIgcGF0aWVudDsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTQ2NTsgIGNpdGF0aW9uX3BhZ2VzPTQ3My03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaXJlY3QgTWVhc3VyZSBvZiB0aGUgZGUgbm92byBtdXRhdGlvbiByYXRlIGluIEF1dGlzbSBhbmQgU2NoaXpvcGhyZW5pYSBDb2hvcnRzOyBBbS4gSi4gSHVtLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9ODc7ICBjaXRhdGlvbl9wYWdlcz0zMTYtMjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVZhcmlhdGlvbiBpbiBnZW5vbWUtd2lkZSBtdXRhdGlvbiByYXRlcyB3aXRoaW4gYW5kIGJldHdlZW4gaHVtYW4gZmFtaWxpZXM7IE5hdC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTQzOyAgY2l0YXRpb25fcGFnZXM9NzEyLTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNwZWNpZmljaXR5LCBlZmZpY2llbmN5LCBhbmQgZmlkZWxpdHkgb2YgUENSOyBQQ1IgTWV0aG9kcyBBcHBsLjsgIGNpdGF0aW9uX3llYXI9MTk5MzsgIGNpdGF0aW9uX3ZvbHVtZT0zOyAgY2l0YXRpb25fcGFnZXM9UzE4LTI5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NaW5vciB2YXJpYW50IGRldGVjdGlvbiBpbiBhbXBsaWNvbnMgdXNpbmcgNDU0IG1hc3NpdmUgcGFyYWxsZWwgcHlyb3NlcXVlbmNpbmc6IGV4cGVyaWVuY2VzIGFuZCBjb25zaWRlcmF0aW9ucyBmb3Igc3VjY2Vzc2Z1bCBhcHBsaWNhdGlvbnM7IEJpb1RlY2huaXF1ZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NTE7ICBjaXRhdGlvbl9wYWdlcz0xNjctNzc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgY29tcHJlaGVuc2l2ZSBjYXRhbG9ndWUgb2Ygc29tYXRpYyBtdXRhdGlvbnMgZnJvbSBhIGh1bWFuIGNhbmNlciBnZW5vbWU7IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT00NjM7ICBjaXRhdGlvbl9wYWdlcz0xOTEtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U05QIGRpc2NvdmVyeSBpbiBwb29sZWQgc2FtcGxlcyB3aXRoIG1pc21hdGNoIHJlcGFpciBkZXRlY3Rpb247IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9MTQwNC0xMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBzdGF0aXN0aWNhbCBtZXRob2QgZm9yIHRoZSBkZXRlY3Rpb24gb2YgdmFyaWFudHMgZnJvbSBuZXh0LWdlbmVyYXRpb24gcmVzZXF1ZW5jaW5nIG9mIEROQSBwb29sczsgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MjY7ICBjaXRhdGlvbl9wYWdlcz1pMzE4LTI0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1VbHRyYXNlbnNpdGl2ZSBkZXRlY3Rpb24gb2YgcmFyZSBtdXRhdGlvbnMgdXNpbmcgbmV4dC1nZW5lcmF0aW9uIHRhcmdldGVkIHJlc2VxdWVuY2luZzsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTQwOyAgY2l0YXRpb25fcGFnZXM9ZTIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMTkvMi8xNDMvMTE2MzA2Mi9kc3IwNDcucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIENocm9uaWMgaW5mYW50aWxlIG5ldXJvbG9naWNhbCBjdXRhbmVvdXMgYW5kIGFydGljdWxhciBzeW5kcm9tZSAoQ0lOQ0EpLCBhbHNvIGtub3duIGFzIG5lb25hdGFsLW9uc2V0IG11bHRpc3lzdGVtIGluZmxhbW1hdG9yeSBkaXNlYXNlIChOT01JRCksIGlzIGEgIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFci8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj00MTkzODQmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2Rkcz1NE26dlXdtVKoxvleO8TRYQWo=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f19%2f2%2f143%2f419384" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f19%2f2%2f143%2f419384" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=16354" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/19/2"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/19/2/1/m_cover.gif?Expires=2147483647&amp;Signature=EMTi8MiwskfL8whcuG7gPpMQVkoTywhsN9k2fcs1qMWyjLc9OmU3zzfeux1LcC8edMnaqIc6tYNqgsUJ8hCH0UQnGgbUVvjHa69ZDqc3rwPQg3T0QzS9XdepkqXV57OA92AfnyIdwEzEggsZ18zDGYx5fC~exu1TTRcnk2bBo6TdYsVgw52BF-63NakC5-nPvHQVSqNzjaFosams6g1DZ~UchQ1c7lmvgwn6cnAg013zUUb9XUUhijVu63SQWmhw0ByRocoE2U4pz7BLDSieaJZqPdVUaKjHeYkUlMHEhZhGv0eccckPuOqL2UhMZ0FzVdPiNB6hKa39ET8qYgw8~Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/19/2">
                        <div class="volume trailing-comma">Volume 19</div>
                        <div class="issue">Issue 2</div>

            </a>

        </div>
        <div class="ii-pub-date">
April 2012        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="6211033">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6211033">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6211036">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6211036">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6211040">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6211040">2. Patients and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6211054">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6211054">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6211077">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6211077">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6211083">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6211083">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6211085">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6211085">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6211087">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6211087">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6211089">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6211089">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/19/2/131/419166">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/19/2/153/419549">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Detection of Base Substitution-Type Somatic Mosaicism of the <em>NLRP3</em> Gene with &gt;99.9% Statistical Confidence by Massively Parallel Sequencing
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                            <div class="meta-authors--limited">
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kazushi Izawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Pediatrics</div>, <div class="insititution">Kyoto University Graduate School of Medicine</div>, <div class="addr-line">54 Shogoin Sakyo, Kyoto 606-8507</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kazushi+Izawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Izawa K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Izawa K&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Kazushi Izawa</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Atsushi Hijikata</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Laboratory for Immunogenomics, RIKEN Research Center for Allergy and Immunology</div>, <div class="insititution">RIKEN Yokohama Institute</div>, <div class="addr-line">1-7-22 Suehiro-cho Tarumi-ku, Yokohama, Kanagawa</div><div class="addr-line">230-0045</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Atsushi+Hijikata">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hijikata A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hijikata A&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Atsushi Hijikata</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Naoko Tanaka</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Pediatrics</div>, <div class="insititution">Kyoto University Graduate School of Medicine</div>, <div class="addr-line">54 Shogoin Sakyo, Kyoto 606-8507</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Naoko+Tanaka">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tanaka N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tanaka N&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Naoko Tanaka</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tomoki Kawai</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Pediatrics</div>, <div class="insititution">Kyoto University Graduate School of Medicine</div>, <div class="addr-line">54 Shogoin Sakyo, Kyoto 606-8507</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tomoki+Kawai">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kawai T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kawai T&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Tomoki Kawai</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Megumu K Saito</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Clinical Application Department,</div><div class="insititution">Center for iPS Cell Research and Application (CiRA), Kyoto University</div>, <div class="addr-line">Kyoto</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Megumu+K+Saito">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Saito M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Saito M&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Megumu K Saito</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Raphaela Goldbach-Mansky</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="insititution">Translational Autoinflammatory Disease Section NIH/NIAMS</div>, <div class="addr-line">Bethesda, MD</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Raphaela+Goldbach-Mansky">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Goldbach-Mansky R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Goldbach-Mansky R&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Raphaela Goldbach-Mansky</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ivona Aksentijevich</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">5</div><div class="insititution">The National Human Genome Research Institute</div>, <div class="addr-line">Bethesda, MD</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ivona+Aksentijevich">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Aksentijevich I">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Aksentijevich I&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Ivona Aksentijevich</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Takahiro Yasumi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Pediatrics</div>, <div class="insititution">Kyoto University Graduate School of Medicine</div>, <div class="addr-line">54 Shogoin Sakyo, Kyoto 606-8507</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Takahiro+Yasumi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yasumi T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yasumi T&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Takahiro Yasumi</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tatsutoshi Nakahata</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Clinical Application Department,</div><div class="insititution">Center for iPS Cell Research and Application (CiRA), Kyoto University</div>, <div class="addr-line">Kyoto</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tatsutoshi+Nakahata">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nakahata T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nakahata T&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Tatsutoshi Nakahata</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Toshio Heike</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Pediatrics</div>, <div class="insititution">Kyoto University Graduate School of Medicine</div>, <div class="addr-line">54 Shogoin Sakyo, Kyoto 606-8507</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Toshio+Heike">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Heike T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Heike T&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Toshio Heike</a>
                                    </span>
                                <a class="meta-authors--etal is-active" id="show-meta-authors" aria-controls="remaining-authors" href="#">
                                    <span>...</span> Show more</a>
                                <div id="remaining-authors" class="meta-authors--remaining" aria-hidden="true" aria-labeledby="show-meta-authors">
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ryuta Nishikomori</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Pediatrics</div>, <div class="insititution">Kyoto University Graduate School of Medicine</div>, <div class="addr-line">54 Shogoin Sakyo, Kyoto 606-8507</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. <strong>+</strong>81 75-751-3291 (R.N.); +81 438-52-3913/+81 45-503-9696 (O.O.). Fax. +81 75-752-2361 (R.N.); +81 438-52-3914/+81 45-503-9694 (O.O.). Email: <a href="mailto:rnishiko@kuhp.kyoto-u.ac.jp" target="_blank">rnishiko@kuhp.kyoto-u.ac.jp</a> (R.N.); <a href="mailto:ohara@kazusa.or.jp" target="_blank">ohara@kazusa.or.jp</a>/<a href="mailto:oosamu@rcai.riken.jp" target="_blank">oosamu@rcai.riken.jp</a> (O.O.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ryuta+Nishikomori">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nishikomori R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nishikomori R&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Ryuta Nishikomori</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Osamu Ohara</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Laboratory for Immunogenomics, RIKEN Research Center for Allergy and Immunology</div>, <div class="insititution">RIKEN Yokohama Institute</div>, <div class="addr-line">1-7-22 Suehiro-cho Tarumi-ku, Yokohama, Kanagawa</div><div class="addr-line">230-0045</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">6</div><div class="addr-line">Department of Human Genome Research</div>, <div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba</div><div class="addr-line">292-0818</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. <strong>+</strong>81 75-751-3291 (R.N.); +81 438-52-3913/+81 45-503-9696 (O.O.). Fax. +81 75-752-2361 (R.N.); +81 438-52-3914/+81 45-503-9694 (O.O.). Email: <a href="mailto:rnishiko@kuhp.kyoto-u.ac.jp" target="_blank">rnishiko@kuhp.kyoto-u.ac.jp</a> (R.N.); <a href="mailto:ohara@kazusa.or.jp" target="_blank">ohara@kazusa.or.jp</a>/<a href="mailto:oosamu@rcai.riken.jp" target="_blank">oosamu@rcai.riken.jp</a> (O.O.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Osamu+Ohara">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ohara O">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ohara O&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Osamu Ohara</a>
                                        </span>
                                </div>
                            </div>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">These authors contributed equally to this work.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Edited by Mitsuo Oshimura</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 19, Issue 2, 1 April 2012, Pages 143–152, <a href='https://doi.org/10.1093/dnares/dsr047'>https://doi.org/10.1093/dnares/dsr047</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">24 January 2012</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">18 November 2011</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">26 December 2011</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="419384" href="/dnaresearch/article-pdf/19/2/143/1163062/dsr047.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Kazushi Izawa, Atsushi Hijikata, Naoko Tanaka, Tomoki Kawai, Megumu K Saito, Raphaela Goldbach-Mansky, Ivona Aksentijevich, Takahiro Yasumi, Tatsutoshi Nakahata, Toshio Heike, Ryuta Nishikomori, Osamu Ohara;  Detection of Base Substitution-Type Somatic Mosaicism of the <em>NLRP3</em> Gene with &gt;99.9% Statistical Confidence by Massively Parallel Sequencing, <em>DNA Research</em>, Volume 19, Issue 2, 1 April 2012, Pages 143–152, <a href='https://doi.org/10.1093/dnares/dsr047'>https://doi.org/10.1093/dnares/dsr047</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/419384?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/419384?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/419384?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/419384?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/419384?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/419384?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/419384?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Detection%20of%20Base%20Substitution-Type%20Somatic%20Mosaicism%20of%20the%20%26lt%3Bem%26gt%3BNLRP3%26lt%3B%2Fem%26gt%3B%20Gene%20with%20%26amp%3Bgt%3B99.9%25%20Statistical%20Confidence%20by%20Massively%20Parallel%20Sequencing&amp;publicationDate=2012-01-24&amp;volumeNum=19&amp;issueNum=2&amp;author=Izawa%2C%20Kazushi%3B%20Hijikata%2C%20Atsushi&amp;startPage=143&amp;endPage=152&amp;contentId=10.1093%2Fdnares%2Fdsr047&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f19%2f2%2f143%2f419384" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=6211033 id="6211033" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Chronic infantile neurological cutaneous and articular syndrome (CINCA), also known as neonatal-onset multisystem inflammatory disease (NOMID), is a dominantly inherited systemic autoinflammatory disease and is caused by a heterozygous germline gain-of-function mutation in the <em>NLRP3</em> gene. We recently found a high incidence of <em>NLRP3</em> somatic mosaicism in apparently mutation-negative CINCA/NOMID patients using subcloning and subsequent capillary DNA sequencing. It is important to rapidly diagnose somatic <em>NLRP3</em> mosaicism to ensure proper treatment. However, this approach requires large investments of time, cost, and labour that prevent routine genetic diagnosis of low-level somatic <em>NLRP3</em> mosaicism. We developed a routine pipeline to detect even a low-level allele of <em>NLRP3</em> with statistical significance using massively parallel DNA sequencing. To address the critical concern of discriminating a low-level allele from sequencing errors, we first constructed error rate maps of 14 polymerase chain reaction products covering the entire coding <em>NLRP3</em> exons on a Roche 454 GS-FLX sequencer from 50 control samples without mosaicism. Based on these results, we formulated a statistical confidence value for each sequence variation in each strand to discriminate sequencing errors from real genetic variation even in a low-level allele, and thereby detected base substitutions at an allele frequency as low as 1% with 99.9% or higher confidence.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;next generation sequencing&quot;">next generation sequencing</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="mosaicism">mosaicism</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;DNA diagnosis&quot;">DNA diagnosis</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;chronic infantile neurological cutaneous and articular syndrome&quot;">chronic infantile neurological cutaneous and articular syndrome</a></div>                            <h2 scrollto-destination=6211036 id="6211036" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Chronic infantile neurological cutaneous and articular syndrome (CINCA; MIM #607115), also known as neonatal-onset multisystem inflammatory disease (NOMID), is a dominantly inherited autoinflammatory disease that is characterized by neonatal onset and a triad of symptoms, including an urticarial-like skin rash, neurological manifestations, and arthritis/arthropathy.<sup><a reveal-id="DSR047C1  DSR047C3" data-open="DSR047C1  DSR047C3" class="link link-ref link-reveal xref-bibr">1–3</a></sup> Patients often experience recurrent fever and systemic inflammation. CINCA/NOMID is the most severe clinical phenotype in the spectrum of cryopyrin-associated periodic syndromes (CAPS), which also include two less severe but phenotypically similar syndromes, familial cold autoinflammatory syndrome (FCAS; MIM #120100), and Muckle−Wells syndrome (MWS; MIM #191900). CAPS are caused by mutations in the <em>NLRP3</em> gene, which is a member of the Nod-like receptor (NLR) family of the innate immune system.<sup><a reveal-id="DSR047C4  DSR047C6" data-open="DSR047C4  DSR047C6" class="link link-ref link-reveal xref-bibr">4–6</a></sup></p><p>Approximately 60% of CINCA/NOMID patients carry heterozygous germline missense mutations in the <em>NLRP3</em> coding region (mutation-positive patients).<sup><a reveal-id="DSR047C7" data-open="DSR047C7" class="link link-ref link-reveal xref-bibr">7</a></sup> More than 80 different disease-causing mutations have been reported to date.<sup><a reveal-id="DSR047C8" data-open="DSR047C8" class="link link-ref link-reveal xref-bibr">8</a></sup> However, the remaining clinically diagnosed CINCA/NOMID patients (∼40%) show no heterozygous germline <em>NLRP3</em> mutation based on conventional DNA sequencing-based genetic analyses (mutation-negative patients). In a previous international collaborative study, we found that there was a high incidence of somatic <em>NLRP3</em> mosaicism in mutation-negative CINCA/NOMID patients worldwide.<sup><a reveal-id="DSR047C9" data-open="DSR047C9" class="link link-ref link-reveal xref-bibr">9</a></sup> The level of mosaicism ranges from 4.2 to 35.8% (median = 10.2%). Rapidly diagnosing somatic <em>NLRP3</em> mosaicism is important to ensure proper treatment. However, the conventional approach used to identify somatic mosaicism of the <em>NLRP3</em> gene is time and labour intensive due to the subcloning of the <em>NLRP3</em> exon polymerase chain reaction (PCR) products, hereafter designated as amplicons, followed by capillary DNA sequencing of more than 100 subclones for each patient. Thus, this approach is not suitable to routinely diagnose somatic mosaicism of the <em>NLRP3</em> gene and additional labour and time will be required to reliably identify somatic mosaicism that occurs at a lower rate. The aim of the present study was to establish a new method that can be used to reliably diagnose somatic mosaicism using the <em>NLRP3</em> gene as a model. Massively parallel DNA sequencing (MPS) technology is an obvious method of choice to identify somatic mosaicism, and this approach has been already reported by other groups.<sup><a reveal-id="DSR047C10  DSR047C12" data-open="DSR047C10  DSR047C12" class="link link-ref link-reveal xref-bibr">10–12</a></sup> However, a well-known caveat of MPS is the high rate of sequencing errors, which cannot be disregarded when identifying low-level somatic mosaicism. To our knowledge, there have been no reports of a reliable method to discriminate MPS sequencing errors from somatic mosaicism with statistical confidence.</p><p>In this study, we first analysed the patterns of sequencing errors in <em>NLRP3</em> coding exons at a single-residue resolution by MPS using a Roche 454 GS-FLX sequencer and then constructed an error rate map for each base position in the <em>NLRP3</em> exons. Based on the error rate map, we could formulate a discrimination pipeline of somatic mosaicism from sequencing errors and thereby detect new somatic mosaicism in mutation-negative CINCA/NOMID patients, whose somatic mutations were subsequently confirmed by subcloning and Sanger sequencing. This approach can also be generally used to identify low-level somatic mosaicism in other genes.</p>                            <h2 scrollto-destination=6211040 id="6211040" class="section-title" data-legacy-id=s2>2. Patients and methods</h2>
                            <h3 scrollto-destination=6211041 id="6211041" class="section-title" data-legacy-id=s2a>2.1. Patients and DNA materials</h3>
<p>Patients were clinically diagnosed with CAPS by their referring physicians and the <em>NLRP3</em> gene was examined using the conventional Sanger sequencing method. DNA samples were obtained from Japanese <em>NLRP3</em> somatic mosaic patients (<em>n</em> = 5) who have been previously described,<sup><a reveal-id="DSR047C9" data-open="DSR047C9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="DSR047C13" data-open="DSR047C13" class="link link-ref link-reveal xref-bibr">13</a></sup> CAPS patients (<em>n</em> = 5) with heterozygous <em>NLRP3</em> mutations, and healthy donors (<em>n</em> = 50). Genomic DNA samples from mutation-negative CINCA/NOMID patients (<em>n</em> = 10) were obtained from the National Institute of Health, Bethesda, USA. To generate DNA samples with no mosaicism, we constructed a set of subcloned plasmids containing each exon and its flanking intronic regions in the <em>NLRP3</em> gene from healthy donor genomic DNA using a Topo TA cloning kit (Invitrogen, San Diego, CA, USA). The cloned plasmids containing each exon and the flanking regions were validated by Sanger sequencing. Written informed consent was obtained from all the patients and their families. The study was approved by the ethical committees of Kyoto University and Kazusa DNA Research Institute and was conducted in accordance with the Helsinki Declaration.</p>                            <h3 scrollto-destination=6211043 id="6211043" class="section-title" data-legacy-id=s2b>2.2. MPS of <em>NLRP3</em> gene amplicons</h3>
<p>Genomic DNA samples were extracted from whole blood or peripheral blood mononuclear cells as previously described. We used a two-step PCR assay and pooled sample libraries for MPS. To cover the entire <em>NLRP3</em> coding exonic regions and flanking intronic regions, 14 amplicons were designed to be as long as an average read length for a 454 GS-FLX sequencer (up to 450 bases) and then amplified from each genomic DNA sample (Fig. <a reveal-id="DSR047F1" data-open="DSR047F1" class="link link-reveal link-table xref-fig">1</a>A). The sequences of the PCR primers that were used to generate these 14 amplicons are provided in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>. The upper and lower amplicon-specific primer sequences were flanked by common 15-base adapter sequences (TGTAAAACGACGGCC and GGAAACAGCTATGAC for the upper and lower primers, respectively) at the 5′ end in order to fuse the primer-binding sequence for MPS in the second-step PCR. The first PCR amplifications were performed in 50-μl reactions using 30 ng of genomic DNA, 1× PrimeSTAR GXL buffer, 0.2 mM of each dNTP, 12.5 pmol of each forward and reverse primer, and 1.25 U of PrimeSTAR GXL DNA polymerase (Takara Bio, Shiga, Japan). The thermal cycling profile consisted of an initial denaturation step at 98°C for 1 min, followed by 28–32 cycles of 10 s denaturation at 98°C, 15 s of annealing at 60°C, and a 30 s extension at 68°C. The lengths of the PCR products ranged from 291 to 421 bp. The second PCR amplifications were performed using primers with adapter sequences at the 3′ end and Multiplex Identiﬁer (MID) sequences at the 5′ end (Fig. <a reveal-id="DSR047F1" data-open="DSR047F1" class="link link-reveal link-table xref-fig">1</a>B), which was used as a tag for each sample. The PCR reactions were performed in 50-μl volumes using 0.5 μl of the first PCR products, 1× PrimeSTAR GXL buffer, 0.2 mM of each dNTP, 12.5 pmol of each forward and reverse primer, and 1.25 U of PrimeSTAR GXL DNA polymerase to attach the anchor sequences for MPS. The thermal cycling profile consisted of an initial denaturation step at 98°C for 20 s, followed by 20 cycles of 10 s denaturation at 98°C, 15 s of annealing at 60°C, and a 40 s extension at 68°C. </p>                        <a id="6211045" scrollto-destination="6211045"></a>
<div id="DSR047F1" data-id="DSR047F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dsr047/2/m_dsr04701.jpeg?Expires=2147483647&Signature=yoL0-hiXj~6Zk7mXLQaM5jaYIne7~~WS3Q1gfY7Ue4jtmpNpGCOXOMptWvdRn3MeRAFZN2Xm6785T9qSjVRcloR2QpUwBUShpA-Ocj~GVdvyr7WN4JQgoWoMoj3WALrlZJnOjwYuEm7rV49Mcpe8gTd4PoYeIq~tfnxegt28UCeJLsuS4G9zrKomOOrm2Py7ORCV-u9UcuaZLnNlgjSr6F5lk6q2WycuJUnYMmzuMBfpOQesZDMW5nEmY41ipTJUFxP0NuGLPGAv2AnPe6Cj~Z9S2EmHkCfrIKYH2wta787NtTvXaMHJZpEWF6OUiLvY3EtSES~bqptN8IUbXeAs~g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The amplicon analysis for NLRP3 exons and its error rate. (A) Exon–intron structure of the NLPR3 gene. Thick and thin rectangles depict exons and introns, respectively. Blue thick rectangles indicate the CDS region. The 14 designed amplicons (red) for nine exons are shown under the exon–intron structure. (B) Amplicon design schema. (C) Error rate for each error category in the region of entire amplicon (pale blue), that without designed primer regions (light blue), and the target regions (CDS + flanking intron; dark blue), respectively. (D) Strand-wise error rate for each amplicon. (E) Error rates along the amplicon sequence of exon 1 in each strand for insertions and deletions in the upper panel and mismatches and ambiguous base calls in the lower panel. The orange and blue lines depict the primer and target regions, respectively. The yellow shaded area depicts the homonucleotide (n &gt; 3) region. The colour representation for the bars is the same as (D). (F) Co-occurrence error rate in both strands. The fraction of positions where a certain error occurred with the error rate for insertions, deletions, and mismatches. The colour representation is the same as in (D) and (E)." path-from-xml="dsr04701.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6211045/dsr04701.jpeg" path-from-xml="dsr04701.jpeg" target="_blank">View large</a><a section="6211045" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dsr047/2/dsr04701.jpeg?Expires=2147483647&Signature=x8b1Rk7LJWEAj69f8z-tz2X6haRMK-7Y8J8tj7QE6~U7Pl~tlf-4602vDJ240Lm0I9r3g7bdboYZw9BuWy0iEwJcT1WY23kYruV37RVjqxFQ5ZopZG9QEzqQuHCMgagaQIPgzHMH7NssO01wTTN-YEgDt1g3i7kuoeGuYA7~XCeEnnvn4EXdzk3d5RycQQ9N7~IvdyK4GgQTwGROqeaTUk3kuYVrWDdFab5EWSTLcQi5-mfQrZ7FZ32h1~YboLDBo1Oi1fGb15ZryjcpFxMB81-5lONCEYLzRY1AL-jhZfQa9j1F7VzdI5h3XZ3cZro9RLQwJe9Xsv4BTfYs1ngcNA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6211045&ar=419384&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr04701.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The amplicon analysis for <em>NLRP3</em> exons and its error rate. (A) Exon–intron structure of the <em>NLPR3</em> gene. Thick and thin rectangles depict exons and introns, respectively. Blue thick rectangles indicate the CDS region. The 14 designed amplicons (red) for nine exons are shown under the exon–intron structure. (B) Amplicon design schema. (C) Error rate for each error category in the region of entire amplicon (pale blue), that without designed primer regions (light blue), and the target regions (CDS + flanking intron; dark blue), respectively. (D) Strand-wise error rate for each amplicon. (E) Error rates along the amplicon sequence of exon 1 in each strand for insertions and deletions in the upper panel and mismatches and ambiguous base calls in the lower panel. The orange and blue lines depict the primer and target regions, respectively. The yellow shaded area depicts the homonucleotide (<em>n</em> &gt; 3) region. The colour representation for the bars is the same as (D). (F) Co-occurrence error rate in both strands. The fraction of positions where a certain error occurred with the error rate for insertions, deletions, and mismatches. The colour representation is the same as in (D) and (E).</p></div></div><div content-id="DSR047F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dsr047/2/m_dsr04701.jpeg?Expires=2147483647&Signature=yoL0-hiXj~6Zk7mXLQaM5jaYIne7~~WS3Q1gfY7Ue4jtmpNpGCOXOMptWvdRn3MeRAFZN2Xm6785T9qSjVRcloR2QpUwBUShpA-Ocj~GVdvyr7WN4JQgoWoMoj3WALrlZJnOjwYuEm7rV49Mcpe8gTd4PoYeIq~tfnxegt28UCeJLsuS4G9zrKomOOrm2Py7ORCV-u9UcuaZLnNlgjSr6F5lk6q2WycuJUnYMmzuMBfpOQesZDMW5nEmY41ipTJUFxP0NuGLPGAv2AnPe6Cj~Z9S2EmHkCfrIKYH2wta787NtTvXaMHJZpEWF6OUiLvY3EtSES~bqptN8IUbXeAs~g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The amplicon analysis for NLRP3 exons and its error rate. (A) Exon–intron structure of the NLPR3 gene. Thick and thin rectangles depict exons and introns, respectively. Blue thick rectangles indicate the CDS region. The 14 designed amplicons (red) for nine exons are shown under the exon–intron structure. (B) Amplicon design schema. (C) Error rate for each error category in the region of entire amplicon (pale blue), that without designed primer regions (light blue), and the target regions (CDS + flanking intron; dark blue), respectively. (D) Strand-wise error rate for each amplicon. (E) Error rates along the amplicon sequence of exon 1 in each strand for insertions and deletions in the upper panel and mismatches and ambiguous base calls in the lower panel. The orange and blue lines depict the primer and target regions, respectively. The yellow shaded area depicts the homonucleotide (n &gt; 3) region. The colour representation for the bars is the same as (D). (F) Co-occurrence error rate in both strands. The fraction of positions where a certain error occurred with the error rate for insertions, deletions, and mismatches. The colour representation is the same as in (D) and (E)." path-from-xml="dsr04701.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6211045/dsr04701.jpeg" path-from-xml="dsr04701.jpeg" target="_blank">View large</a><a section="6211045" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dsr047/2/dsr04701.jpeg?Expires=2147483647&Signature=x8b1Rk7LJWEAj69f8z-tz2X6haRMK-7Y8J8tj7QE6~U7Pl~tlf-4602vDJ240Lm0I9r3g7bdboYZw9BuWy0iEwJcT1WY23kYruV37RVjqxFQ5ZopZG9QEzqQuHCMgagaQIPgzHMH7NssO01wTTN-YEgDt1g3i7kuoeGuYA7~XCeEnnvn4EXdzk3d5RycQQ9N7~IvdyK4GgQTwGROqeaTUk3kuYVrWDdFab5EWSTLcQi5-mfQrZ7FZ32h1~YboLDBo1Oi1fGb15ZryjcpFxMB81-5lONCEYLzRY1AL-jhZfQa9j1F7VzdI5h3XZ3cZro9RLQwJe9Xsv4BTfYs1ngcNA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6211045&ar=419384&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr04701.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The amplicon analysis for <em>NLRP3</em> exons and its error rate. (A) Exon–intron structure of the <em>NLPR3</em> gene. Thick and thin rectangles depict exons and introns, respectively. Blue thick rectangles indicate the CDS region. The 14 designed amplicons (red) for nine exons are shown under the exon–intron structure. (B) Amplicon design schema. (C) Error rate for each error category in the region of entire amplicon (pale blue), that without designed primer regions (light blue), and the target regions (CDS + flanking intron; dark blue), respectively. (D) Strand-wise error rate for each amplicon. (E) Error rates along the amplicon sequence of exon 1 in each strand for insertions and deletions in the upper panel and mismatches and ambiguous base calls in the lower panel. The orange and blue lines depict the primer and target regions, respectively. The yellow shaded area depicts the homonucleotide (<em>n</em> &gt; 3) region. The colour representation for the bars is the same as (D). (F) Co-occurrence error rate in both strands. The fraction of positions where a certain error occurred with the error rate for insertions, deletions, and mismatches. The colour representation is the same as in (D) and (E).</p></div></div><p>After confirming the amount and integrity of the PCR products by agarose gel electrophoresis, we mixed virtually equal amounts of the respective PCR amplicons that were generated using the same genomic DNA and applied the samples to a 454 Genome Sequencer (GS)-FLX system (Roche Diagnostics Corp., USA). All amplicons were ampliﬁed by emPCR and sequenced together in a multiplex fashion. MPS on this platform was performed as instructed by Roche. The sequencing reads from each of the pooled libraries were identified by their MID tags.</p>                            <h3 scrollto-destination=6211047 id="6211047" class="section-title" data-legacy-id=s2c>2.3. Sequence data analysis</h3>
<p>The sequence read data were generated using GS RunProcessor ver.2.5.3 with default settings. Reads were sorted according to the MID tag sequences and were mapped to the reference amplicon sequences using the BLAT program<sup><a reveal-id="DSR047C14" data-open="DSR047C14" class="link link-ref link-reveal xref-bibr">14</a></sup> with the ‘-fine’ option. In order to identify positions where the bases in a read differed from those in the reference sequence, each read was aligned to its reference sequence with the dpAlign module in the BioPerl package (<a class="link link-uri" href="http://www.bioperl.org/">http://www.bioperl.org/</a>). The 454 pyrosequencing-related errors were categorized as insertions, deletions, mismatches, or ambiguous base calls. When aligning sequences, insertions/deletions are allocated based on the sequence context and strand orientation. To eliminate alignment artefacts due to insertion/deletion positions, the lower strand reads were converted to the reverse complement sequence, i.e. keeping the same strandness as the upper strand reads, when aligned with the reference sequence. A sequence error was defined as discordance in an equivalent position between the reference and control (from the 49 healthy individuals and a cloned plasmid vector). The error rate for a specified category was defined as the number of errors divided by the total number of bases in a read. The error rates of a base position on each strand were calculated from 50 control samples.</p>                            <h3 scrollto-destination=6211049 id="6211049" class="section-title" data-legacy-id=s2d>2.4. Confirmation of somatic mosaicism of the <em>NLRP3</em> gene by subcloning and subsequent capillary DNA sequencing</h3>
<p>To confirm the somatic mutational frequency that was identified based on the 454 sequencing data, we subcloned the PCR products and performed capillary DNA sequencing as previously described.<sup><a reveal-id="DSR047C9" data-open="DSR047C9" class="link link-ref link-reveal xref-bibr">9</a></sup> A Topo TA cloning kit (Invitrogen, San Diego, CA, USA) was used to subclone each of the 14 amplicons.</p>                            <h3 scrollto-destination=6211051 id="6211051" class="section-title" data-legacy-id=s2e>2.5. Functional analysis</h3>
<p>To determine whether the identified <em>NLRP3</em> mutants are disease-causing, we assessed both ASC [apoptosis-associated speck-like protein containing a caspase recruitment domain; PYCARD, an approved symbol from the HUGO Gene Nomenclature Committee (HGNC) database]-dependent NF-κB activation in HEK293FT cells and transfection-induced cell death in THP-1 cells, a human monocytic cell line, as previously described.<sup><a reveal-id="DSR047C9" data-open="DSR047C9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="DSR047C13" data-open="DSR047C13" class="link link-ref link-reveal xref-bibr">13</a>,<a reveal-id="DSR047C15" data-open="DSR047C15" class="link link-ref link-reveal xref-bibr">15</a></sup> cDNAs encoding carboxy-terminal green fluorescent protein (GFP)-tagged NLRP3 and its mutants were subcloned into pcDNA5/TO (Invitrogen). Before being introduced into THP-1 cells (10<sup>6</sup>) using a Cell Line Nucleofector Kit V (Amaxa Biosystems, Cologne, Germany), phorbol myristate acetate (10 ng/ml) was added to enhance transient expression of <em>NLRP3</em> gene with minimizing spontaneous cell death.<sup><a reveal-id="DSR047C15" data-open="DSR047C15" class="link link-ref link-reveal xref-bibr">15</a></sup> Four hours after the introduction of plasmids (0.5 μg), cell death of GFP-positive THP-1 cells was measured by flow cytometry.</p><p>Expression plasmids for NLRP3 and ASC in the pEF-BOS vector background have been previously described.<sup><a reveal-id="DSR047C13" data-open="DSR047C13" class="link link-ref link-reveal xref-bibr">13</a></sup> HEK293FT cells (10<sup>5</sup>) were transfected using TransIT-293 Transfection Reagent (Milus Bio, Madison, WI, USA) with an NF-κB reporter construct (pNF-κB-luc; 20 ng; BD Biosciences Clontech, Palo Alto, CA, USA), an internal control construct (pRL-TK; 5 ng; Toyo Ink, Tokyo, Japan), and wild-type or mutant NLRP3 expression plasmid (20 ng) in the presence or absence of ASC expression plasmid (20 ng). The amounts of total plasmid DNA used for transfection experiments were kept constant by adding pEF-BOS vector DNA. Twenty-four hours later, the transfected cells were harvested and subjected to dual luciferase assay by which the ability of each construct to induce NF-κB activation was assessed as previously described.<sup><a reveal-id="DSR047C9" data-open="DSR047C9" class="link link-ref link-reveal xref-bibr">9</a></sup></p>                            <h2 scrollto-destination=6211054 id="6211054" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=6211055 id="6211055" class="section-title" data-legacy-id=s3a>3.1. Construction of base- and strand-specific error rate maps of <em>NLRP3</em> exons from the MPS data of 50 control samples</h3>
<p>Errors in sequence reads generated by a Roche 454 GS-FLX sequencer are not randomly distributed along the sequence and depend on various factors.<sup><a reveal-id="DSR047C16" data-open="DSR047C16" class="link link-ref link-reveal xref-bibr">16</a></sup> Although this is a well-known characteristic of 454 sequencing, the occurrence pattern of these errors has not been explored in detail simply because these sequencing errors are considered noise that can be filtered out in most cases. However, it is highly critical to understand the occurrence pattern of sequencing errors on the MPS platform because low-level somatic mosaicism might appear at a rate close to that of sequencing errors. To address this, we collected ∼1 million sequence reads using the 454 GS-FLX sequencer for 14 amplicons of <em>NLRP3</em> exons from 50 control samples that were thought to be free from somatic mosaicism, and ∼94% of those reads were mapped to one of the reference <em>NLRP3</em> exon sequences. The number of sequencing depths for each amplicon of each sample on each strand was between 65 and 2139 (mean = 565.3, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). We found that the average error rate for each mutation category (insertion, deletion, mismatch, and ambiguous base calls) at each base position on each strand of the amplicons in the control samples was 0.22, 0.16, 0.036, and 0.014%, respectively (Fig. <a reveal-id="DSR047F1" data-open="DSR047F1" class="link link-reveal link-table xref-fig">1</a>C). These values were consistent with those reported in a recent study on the error rates with 454 sequencing data.<sup><a reveal-id="DSR047C16" data-open="DSR047C16" class="link link-ref link-reveal xref-bibr">16</a></sup> The sequencing error in the 454 GS-FLX system tends to occur at the beginning and end of the reads,<sup><a reveal-id="DSR047C11" data-open="DSR047C11" class="link link-ref link-reveal xref-bibr">11</a>,<a reveal-id="DSR047C16" data-open="DSR047C16" class="link link-ref link-reveal xref-bibr">16</a></sup> and we confirmed this trend in our amplicon sequencing data (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). Moreover, after removing the end regions of the read sequences, we found that the error rates of the target regions for each category were 0.20, 0.134, 0.023, and 0.014%, respectively (Fig. <a reveal-id="DSR047F1" data-open="DSR047F1" class="link link-reveal link-table xref-fig">1</a>C and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). When generating the amplicon sequences for the <em>NLRP3</em> exons, the target sequence (CDS region and flanking intron in 10-bp length) was designed to be 300−400 bp and not adjacent to primer sequences in order to obtain relatively low sequencing error rates (Fig. <a reveal-id="DSR047F1" data-open="DSR047F1" class="link link-reveal link-table xref-fig">1</a>C). However, when the base- and strand-specific error rates of the respective amplicons were compared, we noticed that there were large variations in the error rate among amplicons in a strand-specific manner (Fig. <a reveal-id="DSR047F1" data-open="DSR047F1" class="link link-reveal link-table xref-fig">1</a>D). We further examined the occurrence pattern of sequencing errors, as shown in Fig. <a reveal-id="DSR047F1" data-open="DSR047F1" class="link link-reveal link-table xref-fig">1</a>E; the average sequencing error rates at each base in the 50 control amplicons are shown in a bar graph, where the bars in the upper or lower direction show the sequence error rates at the base position on the upper or lower strand of the amplicons, respectively. As evident in Fig. <a reveal-id="DSR047F1" data-open="DSR047F1" class="link link-reveal link-table xref-fig">1</a>E, the error rates at most residues were low (&lt;1%) with some hotspots for each type of error. Most of the insertion/deletion errors preferentially occurred at a homonucleotide region (yellow regions in Fig. <a reveal-id="DSR047F1" data-open="DSR047F1" class="link link-reveal link-table xref-fig">1</a>E) as previously described,<sup><a reveal-id="DSR047C17" data-open="DSR047C17" class="link link-ref link-reveal xref-bibr">17</a></sup> but it was not always the case for all of homonucleotide regions. We could not find any tight relationship between other sequence patterns and the error rate. In addition, there was almost no position where sequencing errors occurred at a similar rate on both strands. This is more clearly shown in Fig. <a reveal-id="DSR047F1" data-open="DSR047F1" class="link link-reveal link-table xref-fig">1</a>F, which indicates the numbers of positions with sequence variations in both strands that were higher than the threshold along the horizontal axis. These results indicate that the sequence errors can be discriminated from real genetic alterations when the sequence is read in both directions. However, it is important to keep in mind that PCR errors are not distinct from real genetic alterations. We did not observe any base substitution at a rate higher than 1% in our experiments (Fig. <a reveal-id="DSR047F1" data-open="DSR047F1" class="link link-reveal link-table xref-fig">1</a>F), and the overall PCR error rate under MPS conditions was lower than 1% as long as a high-fidelity DNA polymerase was used to generate the amplicons.</p><p>Because Gilles <em>et al.</em><sup><a reveal-id="DSR047C16" data-open="DSR047C16" class="link link-ref link-reveal xref-bibr">16</a></sup> recently reported that the occurrence of sequencing errors using the Roche 454 GS-FLX DNA sequencer depends on various factors, we first examined variations in the sequencing error rates of <em>NLRP3</em> exons among samples in the same run. For each mutation category, we found a similar trend in the error distribution rate in the amplicon sequences among the control samples (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Figs S2–S4</a>). We confirmed that, for almost all residues, the error rate distributions among the 50 control samples fitted a Poisson distribution (data not shown). We next examined the run-to-run variation of the sequencing error rate for <em>NLRP3</em> exons. For this purpose, we performed an additional MPS run with seven amplicons (exons 3, 4, and 6) that were newly prepared and compared the number and positions of the sequencing errors between two independent sequencing runs. Out of 1993 base positions in the target regions, there was a low occurrence rate of mismatch errors in both runs and this seemed to fit a Poisson distribution. However, insertion/deletion errors (&gt;1% error rate) were observed at ∼100 base positions (&lt;5% in the target regions) in each run, and only a half of these errors were shared between both runs (Table <a reveal-id="DSR047TB1" data-open="DSR047TB1" class="link link-reveal link-table xref-fig">1</a>). This indicated that the occurrence of insertion/deletion errors was considerably affected by the run conditions (probably due to variations in the absolute signal strengths of pyrosequencing). Thus, as previously reported, the detection of insertion/deletion mutations by MPS on the 454 GS-FLX system was quite error-prone at least at a limited number of residues. However, the results also implied that false-positive mosaic mutations could be avoided by considering the sequencing data for both strands because these run-dependent insertion/deletion errors occur only in a single strand. Taken together, we conclude that the obtained sequence error map is stable and sufficiently robust to discriminate substitution sequencing errors from low-level mosaicism. </p>                        <a id="6211058" scrollto-destination="6211058"></a>
<div content-id="DSR047TB1" class="table-modal"><div class="table-wrap"><div id="DSR047TB1" class="table-wrap-title" data-id="DSR047TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Run-to-run variations in the error occurrence (&gt;1% frequency)</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Error category </th><th colspan="3">Upper strand<hr /> </th><th colspan="3">Lower strand<hr /> </th><th>All<sup>a</sup> </th></tr><tr><th> </th><th>First run </th><th>Second run </th><th>Overlap </th><th>First run </th><th>Second run </th><th>Overlap </th><th> </th></tr></thead><tbody><tr><td>Insertions </td><td>63 </td><td>73 </td><td>42 </td><td>76 </td><td>96 </td><td>52 </td><td>10 </td></tr><tr><td>Deletions </td><td>36 </td><td>44 </td><td>24 </td><td>29 </td><td>65 </td><td>20 </td><td>2 </td></tr><tr><td>Mismatches </td><td>0 </td><td>0 </td><td>0 </td><td>3 </td><td>0 </td><td>0 </td><td>0 </td></tr><tr><td>Ambiguous base calls </td><td>6 </td><td>8 </td><td>6 </td><td>12 </td><td>10 </td><td>10 </td><td>0 </td></tr></tbody></table></div><div content-id="DSR047TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Error category </th><th colspan="3">Upper strand<hr /> </th><th colspan="3">Lower strand<hr /> </th><th>All<sup>a</sup> </th></tr><tr><th> </th><th>First run </th><th>Second run </th><th>Overlap </th><th>First run </th><th>Second run </th><th>Overlap </th><th> </th></tr></thead><tbody><tr><td>Insertions </td><td>63 </td><td>73 </td><td>42 </td><td>76 </td><td>96 </td><td>52 </td><td>10 </td></tr><tr><td>Deletions </td><td>36 </td><td>44 </td><td>24 </td><td>29 </td><td>65 </td><td>20 </td><td>2 </td></tr><tr><td>Mismatches </td><td>0 </td><td>0 </td><td>0 </td><td>3 </td><td>0 </td><td>0 </td><td>0 </td></tr><tr><td>Ambiguous base calls </td><td>6 </td><td>8 </td><td>6 </td><td>12 </td><td>10 </td><td>10 </td><td>0 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>The number of positions where the error rates in each category were commonly &gt;1% for both strands in two independent runs.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6211058" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR047TB1" class="table-wrap-title" data-id="DSR047TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Run-to-run variations in the error occurrence (&gt;1% frequency)</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Error category </th><th colspan="3">Upper strand<hr /> </th><th colspan="3">Lower strand<hr /> </th><th>All<sup>a</sup> </th></tr><tr><th> </th><th>First run </th><th>Second run </th><th>Overlap </th><th>First run </th><th>Second run </th><th>Overlap </th><th> </th></tr></thead><tbody><tr><td>Insertions </td><td>63 </td><td>73 </td><td>42 </td><td>76 </td><td>96 </td><td>52 </td><td>10 </td></tr><tr><td>Deletions </td><td>36 </td><td>44 </td><td>24 </td><td>29 </td><td>65 </td><td>20 </td><td>2 </td></tr><tr><td>Mismatches </td><td>0 </td><td>0 </td><td>0 </td><td>3 </td><td>0 </td><td>0 </td><td>0 </td></tr><tr><td>Ambiguous base calls </td><td>6 </td><td>8 </td><td>6 </td><td>12 </td><td>10 </td><td>10 </td><td>0 </td></tr></tbody></table></div><div content-id="DSR047TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Error category </th><th colspan="3">Upper strand<hr /> </th><th colspan="3">Lower strand<hr /> </th><th>All<sup>a</sup> </th></tr><tr><th> </th><th>First run </th><th>Second run </th><th>Overlap </th><th>First run </th><th>Second run </th><th>Overlap </th><th> </th></tr></thead><tbody><tr><td>Insertions </td><td>63 </td><td>73 </td><td>42 </td><td>76 </td><td>96 </td><td>52 </td><td>10 </td></tr><tr><td>Deletions </td><td>36 </td><td>44 </td><td>24 </td><td>29 </td><td>65 </td><td>20 </td><td>2 </td></tr><tr><td>Mismatches </td><td>0 </td><td>0 </td><td>0 </td><td>3 </td><td>0 </td><td>0 </td><td>0 </td></tr><tr><td>Ambiguous base calls </td><td>6 </td><td>8 </td><td>6 </td><td>12 </td><td>10 </td><td>10 </td><td>0 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>The number of positions where the error rates in each category were commonly &gt;1% for both strands in two independent runs.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6211058" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=6211059 id="6211059" class="section-title" data-legacy-id=s3b>3.2. Discrimination formula for detection of somatic mosaicism with statistical confidence</h3>
<p>We next examined known SNPs, known heterozygous mutations and somatic mosaic mutations of CAPS patients using MPS. All of these variations appeared on both strands at the expected allele frequencies as shown in Fig. <a reveal-id="DSR047F2" data-open="DSR047F2" class="link link-reveal link-table xref-fig">2</a>, again indicating that filtering the strand-specific sequence variations is unlikely to eliminate real genetic variations. </p>                        <a id="6211061" scrollto-destination="6211061"></a>
<div id="DSR047F2" data-id="DSR047F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dsr047/2/m_dsr04702.jpeg?Expires=2147483647&Signature=BTSgwTo5XRnSQLXvLp2MWiFkKs1xy1IVNLea4dDnyErn2VEOGqOMw-15EWpyk2mbRjBq17h0c9-CJQaCX5SSP2iTpr~kR~YL~QIJdSH7LlfE2qOZadjR-AMwlc~5uiHOQpHHOLRgavVxzkIFDbA~UBHYq2YRG4eafd2qOzUrOqjAWgsKCFmBCCvjof9eeDXC~nXDAn98jw86PN9Ca1sS1-~dtTzvPm8-~yf1xpyM4YIcScNJoHkgo4ZqlXzqQwvGMwXxNtJV0TvtSs5uw1qyADtr9J-EHDcJ~FC33Ib21UWZDURL-9aziewqLfOkrEYy7GTv-2uoaecBM3v3lF~kug__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Scatter plot of the observed frequency variation in both strands. The colours depict known SNPs (green), heterozygous and mosaic mutations (orange) and errors (grey)." path-from-xml="dsr04702.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6211061/dsr04702.jpeg" path-from-xml="dsr04702.jpeg" target="_blank">View large</a><a section="6211061" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dsr047/2/dsr04702.jpeg?Expires=2147483647&Signature=ctMU45Zozsa0Z53ixpwZbrVaGEPOLxm3XZNxubaSfmeU-cEDdoco6BEkttNeq23hU0B-dB9XX~v2ADfuF~YXfIIIT7I~3-UcCcLW-IG7KhQtyxlA1SGf0fDgOw-8BBdCXfIO7VL1g39ELv~9WcuAP9czsp2c4XfXN2Np2yUgVRZ2jSsdNVscJnuSULhZ6GKfM~QvJcjLDUiSnczRt3vVh0cY6uKHs~jkd7kWy0RwYoDcJ0qNnQThl1LZ6tZtZKDywAEHjQYc5Feg9Muim5DDaOOV~XqlfNSYAIG4iTDA-UOFu~RC3POkYT2MPVwUSSzcq6fkIAZQRngh0TPjyewZLw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6211061&ar=419384&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr04702.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Scatter plot of the observed frequency variation in both strands. The colours depict known SNPs (green), heterozygous and mosaic mutations (orange) and errors (grey).</p></div></div><div content-id="DSR047F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dsr047/2/m_dsr04702.jpeg?Expires=2147483647&Signature=BTSgwTo5XRnSQLXvLp2MWiFkKs1xy1IVNLea4dDnyErn2VEOGqOMw-15EWpyk2mbRjBq17h0c9-CJQaCX5SSP2iTpr~kR~YL~QIJdSH7LlfE2qOZadjR-AMwlc~5uiHOQpHHOLRgavVxzkIFDbA~UBHYq2YRG4eafd2qOzUrOqjAWgsKCFmBCCvjof9eeDXC~nXDAn98jw86PN9Ca1sS1-~dtTzvPm8-~yf1xpyM4YIcScNJoHkgo4ZqlXzqQwvGMwXxNtJV0TvtSs5uw1qyADtr9J-EHDcJ~FC33Ib21UWZDURL-9aziewqLfOkrEYy7GTv-2uoaecBM3v3lF~kug__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Scatter plot of the observed frequency variation in both strands. The colours depict known SNPs (green), heterozygous and mosaic mutations (orange) and errors (grey)." path-from-xml="dsr04702.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6211061/dsr04702.jpeg" path-from-xml="dsr04702.jpeg" target="_blank">View large</a><a section="6211061" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dsr047/2/dsr04702.jpeg?Expires=2147483647&Signature=ctMU45Zozsa0Z53ixpwZbrVaGEPOLxm3XZNxubaSfmeU-cEDdoco6BEkttNeq23hU0B-dB9XX~v2ADfuF~YXfIIIT7I~3-UcCcLW-IG7KhQtyxlA1SGf0fDgOw-8BBdCXfIO7VL1g39ELv~9WcuAP9czsp2c4XfXN2Np2yUgVRZ2jSsdNVscJnuSULhZ6GKfM~QvJcjLDUiSnczRt3vVh0cY6uKHs~jkd7kWy0RwYoDcJ0qNnQThl1LZ6tZtZKDywAEHjQYc5Feg9Muim5DDaOOV~XqlfNSYAIG4iTDA-UOFu~RC3POkYT2MPVwUSSzcq6fkIAZQRngh0TPjyewZLw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6211061&ar=419384&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr04702.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Scatter plot of the observed frequency variation in both strands. The colours depict known SNPs (green), heterozygous and mosaic mutations (orange) and errors (grey).</p></div></div><div class="block-child-p">Based on the experimentally observed sequencing errors with the 454 GS-FLX system described above, we established a discrimination formula to detect low-level somatic mosaicism as follows. In previous studies, the number of reads with the sequence error of a certain category in a sequence position was modelled based on the Poisson distribution with two parameters <em>λ</em> and <em>k</em> where the expected number of reads containing an error and the observed number of reads containing a sequence alteration, respectively, are as shown below<sup><a reveal-id="DSR047C18" data-open="DSR047C18" class="link link-ref link-reveal xref-bibr">18</a></sup>: <a id="DSR047M1" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSR047M1"><div class="fig-graphic"><a class="jumplink-placeholder" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dsr047/2/dsr047eq1.gif?Expires=2147483647&Signature=CgqjUp4P2M85BLGMj2YKTdug1Cxxd6a4ShchuJC1x5-roVXcmjtSzlxdSIFGowQAut8wMncq6i6p81fJTYKVIM8W3WbRwbUBjmqhg~fDIJ21ddNfGtvuLc4StW2nf5MDARbZvkL1WuouQ7i~01p0M9QxVEBMQhXrLVytbGu8jo-mPP-3DYMXm2aW1hAM0quKIyn6eleb0uF5euDH1~6dZ7zX97iHfsdeMAHtX3HNmSi2kizt9WgIWDhIjNo8YpcJbxu5PXd2Vbv3pcGxDOYZ1EJlMpCPJ4dT3xPogXzXLlfy8WjeJOTAJZ-~E~xdoPafOaVkPkDXygxT1R6cOaMzmg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" id="dsr047eq1.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dsr047/2/m_dsr047eq1.gif?Expires=2147483647&Signature=tbY5SokPTtTqJrredru1JDcJDuBEyLdfK7OMTq~Psh-KHplv5J5jYFgGt9ZIDe2ZHmm6SWOK7X1JeAFHRsXOAv13jz2~CsoeGmYdDCrW6vIjOCog0eJ6RVj6awIIutimPnL074lN4JP0xRA07L9InOZBBR6N4g07HrnJHjPUvxCW617wOH9g1JjJr-jm4myOuPT6YfyFD6j7GLLVKyVifXz-uP0lta4rc8zzOVGUEeYAz4lI2w-WVn2R4qPjgOANzBcDQqL0TP95aFBosJMjrzJwbcC98yM4ayj93uOYm7SP~bPQ1-VY8LrqoO7lAoQ3Ov0MXaQLhobzPC4kcsidDA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsr047eq1.gif" /></a></div></div><div class="label">(1)</div></div></div><div class="block-child-p">This model assumes that the error rate is constant across the different sequence regions but our data described above pointed out that the sequence error rate varies with the sequence content.<sup><a reveal-id="DSR047C19" data-open="DSR047C19" class="link link-ref link-reveal xref-bibr">19</a></sup> Thus, we introduced a position- and strand-specific error rate <em>q<sub>i,j,d</sub></em> for a certain error category <em>j</em> in amplicon position <em>i</em> with strand <em>d</em> based on the sequencing data from 50 control samples. With the error rate <em>q<sub>i,j,d,</sub></em> the upper probability (<em>P</em>) that the number of reads (<em>R</em>) with a certain sequence alteration of category <em>j</em> in position <em>i</em> is equal or greater than the number of observed reads <em>r</em> out of <em>N</em> reads with a sequenced direction <em>d</em> for an unknown sample was defined as: <a id="DSR047M2" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSR047M2"><div class="fig-graphic"><a class="jumplink-placeholder" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dsr047/2/dsr047eq2.gif?Expires=2147483647&Signature=kdGoIVRCl-JvB~B1gOJ1e~dYEIucOD6mBgk1j7v9AU0dlRSFsYNBrDX60wzlKp-Ypgt5kYWCnIXwXtVhExTiOmmIPIu9uVdadlmiIoBd2Lzghat~PfPzx0PSnnj9KxE5zXMhhiO75KdwTzajPqFl5PF~PTAM5z8fGu2HGR07W94BT454i5TcKmY9enRR4KIYf8RrXi1k0ankcmD4gdldJrVDyOh2BbdgfH0b5PFpG5e9C~9uA2y4omM6d8E6FKWpwcNi36oAnujo~T9ynuAenT0CbYX89egk0DTXTZrVxVT8WL8Bk7UdHjRvidxsSoCtm3uLkxRlRBzKc2Ky59cQQg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" id="dsr047eq2.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dsr047/2/m_dsr047eq2.gif?Expires=2147483647&Signature=XiMYrrKC6x3QbfrApjE4JrVCDcm2IWGEaAbqecuMR2DiMzsd44zuulI5QvO-Bbc9Ng4Ft3w8wqdWRNzfatxcjwkywyiBjpJSonxtQIeAMtt~LgEIHTjV4HhcsksH5RrrseGYtwoAd06pYssXS52TWxR6ETwjqGPX~MH2abGvviDiuYbdcSBMD4LQnZqAsIEtzqYC3-ZtJ2SaIB-BqBuN6gVoR2OhG5ImnOLCkWRh3NKk8dbkqFo21nyzbwYaZaquN2rGMAt3pOYFxR97nvaSvz-oAgKIDhx9bGSPCPjHdZik8IglAzXLH31st67FwCFKp13G7qBOhn7Vi4CH2rbh4Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsr047eq2.gif" /></a></div></div><div class="label">(2)</div></div> where, <em>λ</em><sub>i,j,d</sub> = <em>N<sub>i,d</sub></em> × <em>q<sub>i,j,d</sub></em>.</div><p>For the mismatch error rate, we did not consider the type of base substituted in an amplicon position in this study. We took (1 − <em>P</em>) as a measure of the statistical confidence of the data and conventionally set a threshold of the statistical confidence to be 99.9%. In other words, if <em>P</em>-value was &lt;0.001, the sequence alteration was considered to be a real sequence variation, not an error. For the final identification of real genetic variation with low-level somatic mosaicism, we determined that both of the <em>P</em>-values for the <em>i</em>th residue in the upper and lower strands must be smaller than the threshold.</p><p>To evaluate the lower detection limit for the allele frequencies of somatic mosaicism based on the statistical formulation shown above, we generated a series of known allele frequencies by diluting DNA from CAPS patients carrying heterozygous <em>NLRP3</em> mutations (c.1043C&gt;T, c.1316C&gt;T, and c.1985T&gt;C) with DNA from normal donors carrying the wild-type <em>NLRP3</em> gene. In the dilution series, the mutant allele frequencies were adjusted to be 10, 5, 3, 2, 1, and 0.5% (Table <a reveal-id="DSR047TB2" data-open="DSR047TB2" class="link link-reveal link-table xref-fig">2</a>). The data indicated that somatic mosaicism at these sites and at an allele frequency ≥1% could be convincingly detected with statistical significance (<em>P</em>&lt; 0.001) if more than 350 reads for each strand were obtained for an amplicon. We also applied this statistical method to detect somatic mosaicism in patients with known low-level mosaic mutations described above and confirmed that all of the mutations could be detected with statistical significance without any false positives (data not shown). </p>                        <a id="6211069" scrollto-destination="6211069"></a>
<div content-id="DSR047TB2" class="table-modal"><div class="table-wrap"><div id="DSR047TB2" class="table-wrap-title" data-id="DSR047TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Evaluation of the lower detection limit for mosaicism with three sets of dilution series</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Mutation </th><th>Dilution (%) </th><th colspan="4">Upper strand<hr /> </th><th colspan="4">Lower strand<hr /> </th></tr><tr><th> </th><th> </th><th>Totalreads </th><th>Mutantreads </th><th>%Mutant </th><th><em>P</em>-value </th><th>Totalreads </th><th>Mutantreads </th><th>%Mutant </th><th><em>P</em>-value </th></tr></thead><tbody><tr><td>c.1043C&gt;T; p.Thr348Met </td><td>10.0 </td><td>724 </td><td>61 </td><td>8.43 </td><td>8.62E−130 </td><td>520 </td><td>57 </td><td>10.96 </td><td>1.73E−117 </td></tr><tr><td> </td><td>5.0 </td><td>453 </td><td>24 </td><td>5.30 </td><td>2.86E−47 </td><td>372 </td><td>15 </td><td>4.03 </td><td>1.26E−25 </td></tr><tr><td> </td><td>3.0 </td><td>876 </td><td>27 </td><td>3.08 </td><td>1.16E−46 </td><td>757 </td><td>21 </td><td>2.77 </td><td>6.83E−32 </td></tr><tr><td> </td><td>2.0 </td><td>737 </td><td>10 </td><td>1.36 </td><td>1.05E−14 </td><td>645 </td><td>7 </td><td>1.09 </td><td>8.68E−09 </td></tr><tr><td> </td><td>1.0 </td><td>715 </td><td>9 </td><td>1.26 </td><td>4.73E−13 </td><td>624 </td><td>4 </td><td>0.64 </td><td>1.11E−04 </td></tr><tr><td> </td><td>0.5 </td><td>1025 </td><td>7 </td><td>0.68 </td><td>1.15E−14 </td><td>756 </td><td>3 </td><td>0.40 </td><td>3.22E−03<sup>a</sup> </td></tr><tr><td>c.1431C&gt;A; p.Asn477Lys </td><td>10.0 </td><td>542 </td><td>65 </td><td>11.99 </td><td>1.22E−113 </td><td>346 </td><td>24 </td><td>6.94 </td><td>6.84E−49 </td></tr><tr><td> </td><td>5.0 </td><td>491 </td><td>30 </td><td>6.11 </td><td>1.13E−44 </td><td>356 </td><td>17 </td><td>4.78 </td><td>2.42E−32 </td></tr><tr><td> </td><td>3.0 </td><td>487 </td><td>21 </td><td>4.31 </td><td>1.26E−28 </td><td>374 </td><td>19 </td><td>5.08 </td><td>1.78E−36 </td></tr><tr><td> </td><td>2.0 </td><td>577 </td><td>18 </td><td>3.12 </td><td>2.78E−22 </td><td>495 </td><td>9 </td><td>1.82 </td><td>4.57E−14 </td></tr><tr><td> </td><td>1.0 </td><td>491 </td><td>4 </td><td>0.82 </td><td>9.17E−04 </td><td>354 </td><td>5 </td><td>1.41 </td><td>7.34E−08 </td></tr><tr><td> </td><td>0.5 </td><td>483 </td><td>0 </td><td>0 </td><td>NA </td><td>424 </td><td>3 </td><td>0.71 </td><td>NA </td></tr><tr><td>c.1985T&gt;C; p.Met662Thr </td><td>10.0 </td><td>658 </td><td>79 </td><td>12.01 </td><td>1.13E−179 </td><td>643 </td><td>74 </td><td>11.51 </td><td>4.64E−167 </td></tr><tr><td> </td><td>5.0 </td><td>643 </td><td>31 </td><td>4.82 </td><td>2.56E−59 </td><td>608 </td><td>33 </td><td>5.43 </td><td>9.96E−65 </td></tr><tr><td> </td><td>3.0 </td><td>777 </td><td>27 </td><td>3.48 </td><td>4.65E−48 </td><td>704 </td><td>29 </td><td>4.12 </td><td>1.26E−53 </td></tr><tr><td> </td><td>2.0 </td><td>929 </td><td>21 </td><td>2.26 </td><td>7.59E−34 </td><td>835 </td><td>15 </td><td>1.80 </td><td>3.92E−23 </td></tr><tr><td> </td><td>1.0 </td><td>735 </td><td>17 </td><td>1.09 </td><td>2.74E−11 </td><td>709 </td><td>9 </td><td>1.27 </td><td>4.06E−13 </td></tr><tr><td> </td><td>0.5 </td><td>702 </td><td>2 </td><td>0.29 </td><td>3.90E−03<sup>a</sup> </td><td>590 </td><td>1 </td><td>0.17 </td><td>1.37E−01<sup>a</sup> </td></tr></tbody></table></div><div content-id="DSR047TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Mutation </th><th>Dilution (%) </th><th colspan="4">Upper strand<hr /> </th><th colspan="4">Lower strand<hr /> </th></tr><tr><th> </th><th> </th><th>Totalreads </th><th>Mutantreads </th><th>%Mutant </th><th><em>P</em>-value </th><th>Totalreads </th><th>Mutantreads </th><th>%Mutant </th><th><em>P</em>-value </th></tr></thead><tbody><tr><td>c.1043C&gt;T; p.Thr348Met </td><td>10.0 </td><td>724 </td><td>61 </td><td>8.43 </td><td>8.62E−130 </td><td>520 </td><td>57 </td><td>10.96 </td><td>1.73E−117 </td></tr><tr><td> </td><td>5.0 </td><td>453 </td><td>24 </td><td>5.30 </td><td>2.86E−47 </td><td>372 </td><td>15 </td><td>4.03 </td><td>1.26E−25 </td></tr><tr><td> </td><td>3.0 </td><td>876 </td><td>27 </td><td>3.08 </td><td>1.16E−46 </td><td>757 </td><td>21 </td><td>2.77 </td><td>6.83E−32 </td></tr><tr><td> </td><td>2.0 </td><td>737 </td><td>10 </td><td>1.36 </td><td>1.05E−14 </td><td>645 </td><td>7 </td><td>1.09 </td><td>8.68E−09 </td></tr><tr><td> </td><td>1.0 </td><td>715 </td><td>9 </td><td>1.26 </td><td>4.73E−13 </td><td>624 </td><td>4 </td><td>0.64 </td><td>1.11E−04 </td></tr><tr><td> </td><td>0.5 </td><td>1025 </td><td>7 </td><td>0.68 </td><td>1.15E−14 </td><td>756 </td><td>3 </td><td>0.40 </td><td>3.22E−03<sup>a</sup> </td></tr><tr><td>c.1431C&gt;A; p.Asn477Lys </td><td>10.0 </td><td>542 </td><td>65 </td><td>11.99 </td><td>1.22E−113 </td><td>346 </td><td>24 </td><td>6.94 </td><td>6.84E−49 </td></tr><tr><td> </td><td>5.0 </td><td>491 </td><td>30 </td><td>6.11 </td><td>1.13E−44 </td><td>356 </td><td>17 </td><td>4.78 </td><td>2.42E−32 </td></tr><tr><td> </td><td>3.0 </td><td>487 </td><td>21 </td><td>4.31 </td><td>1.26E−28 </td><td>374 </td><td>19 </td><td>5.08 </td><td>1.78E−36 </td></tr><tr><td> </td><td>2.0 </td><td>577 </td><td>18 </td><td>3.12 </td><td>2.78E−22 </td><td>495 </td><td>9 </td><td>1.82 </td><td>4.57E−14 </td></tr><tr><td> </td><td>1.0 </td><td>491 </td><td>4 </td><td>0.82 </td><td>9.17E−04 </td><td>354 </td><td>5 </td><td>1.41 </td><td>7.34E−08 </td></tr><tr><td> </td><td>0.5 </td><td>483 </td><td>0 </td><td>0 </td><td>NA </td><td>424 </td><td>3 </td><td>0.71 </td><td>NA </td></tr><tr><td>c.1985T&gt;C; p.Met662Thr </td><td>10.0 </td><td>658 </td><td>79 </td><td>12.01 </td><td>1.13E−179 </td><td>643 </td><td>74 </td><td>11.51 </td><td>4.64E−167 </td></tr><tr><td> </td><td>5.0 </td><td>643 </td><td>31 </td><td>4.82 </td><td>2.56E−59 </td><td>608 </td><td>33 </td><td>5.43 </td><td>9.96E−65 </td></tr><tr><td> </td><td>3.0 </td><td>777 </td><td>27 </td><td>3.48 </td><td>4.65E−48 </td><td>704 </td><td>29 </td><td>4.12 </td><td>1.26E−53 </td></tr><tr><td> </td><td>2.0 </td><td>929 </td><td>21 </td><td>2.26 </td><td>7.59E−34 </td><td>835 </td><td>15 </td><td>1.80 </td><td>3.92E−23 </td></tr><tr><td> </td><td>1.0 </td><td>735 </td><td>17 </td><td>1.09 </td><td>2.74E−11 </td><td>709 </td><td>9 </td><td>1.27 </td><td>4.06E−13 </td></tr><tr><td> </td><td>0.5 </td><td>702 </td><td>2 </td><td>0.29 </td><td>3.90E−03<sup>a</sup> </td><td>590 </td><td>1 </td><td>0.17 </td><td>1.37E−01<sup>a</sup> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Not significant.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6211069" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR047TB2" class="table-wrap-title" data-id="DSR047TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Evaluation of the lower detection limit for mosaicism with three sets of dilution series</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Mutation </th><th>Dilution (%) </th><th colspan="4">Upper strand<hr /> </th><th colspan="4">Lower strand<hr /> </th></tr><tr><th> </th><th> </th><th>Totalreads </th><th>Mutantreads </th><th>%Mutant </th><th><em>P</em>-value </th><th>Totalreads </th><th>Mutantreads </th><th>%Mutant </th><th><em>P</em>-value </th></tr></thead><tbody><tr><td>c.1043C&gt;T; p.Thr348Met </td><td>10.0 </td><td>724 </td><td>61 </td><td>8.43 </td><td>8.62E−130 </td><td>520 </td><td>57 </td><td>10.96 </td><td>1.73E−117 </td></tr><tr><td> </td><td>5.0 </td><td>453 </td><td>24 </td><td>5.30 </td><td>2.86E−47 </td><td>372 </td><td>15 </td><td>4.03 </td><td>1.26E−25 </td></tr><tr><td> </td><td>3.0 </td><td>876 </td><td>27 </td><td>3.08 </td><td>1.16E−46 </td><td>757 </td><td>21 </td><td>2.77 </td><td>6.83E−32 </td></tr><tr><td> </td><td>2.0 </td><td>737 </td><td>10 </td><td>1.36 </td><td>1.05E−14 </td><td>645 </td><td>7 </td><td>1.09 </td><td>8.68E−09 </td></tr><tr><td> </td><td>1.0 </td><td>715 </td><td>9 </td><td>1.26 </td><td>4.73E−13 </td><td>624 </td><td>4 </td><td>0.64 </td><td>1.11E−04 </td></tr><tr><td> </td><td>0.5 </td><td>1025 </td><td>7 </td><td>0.68 </td><td>1.15E−14 </td><td>756 </td><td>3 </td><td>0.40 </td><td>3.22E−03<sup>a</sup> </td></tr><tr><td>c.1431C&gt;A; p.Asn477Lys </td><td>10.0 </td><td>542 </td><td>65 </td><td>11.99 </td><td>1.22E−113 </td><td>346 </td><td>24 </td><td>6.94 </td><td>6.84E−49 </td></tr><tr><td> </td><td>5.0 </td><td>491 </td><td>30 </td><td>6.11 </td><td>1.13E−44 </td><td>356 </td><td>17 </td><td>4.78 </td><td>2.42E−32 </td></tr><tr><td> </td><td>3.0 </td><td>487 </td><td>21 </td><td>4.31 </td><td>1.26E−28 </td><td>374 </td><td>19 </td><td>5.08 </td><td>1.78E−36 </td></tr><tr><td> </td><td>2.0 </td><td>577 </td><td>18 </td><td>3.12 </td><td>2.78E−22 </td><td>495 </td><td>9 </td><td>1.82 </td><td>4.57E−14 </td></tr><tr><td> </td><td>1.0 </td><td>491 </td><td>4 </td><td>0.82 </td><td>9.17E−04 </td><td>354 </td><td>5 </td><td>1.41 </td><td>7.34E−08 </td></tr><tr><td> </td><td>0.5 </td><td>483 </td><td>0 </td><td>0 </td><td>NA </td><td>424 </td><td>3 </td><td>0.71 </td><td>NA </td></tr><tr><td>c.1985T&gt;C; p.Met662Thr </td><td>10.0 </td><td>658 </td><td>79 </td><td>12.01 </td><td>1.13E−179 </td><td>643 </td><td>74 </td><td>11.51 </td><td>4.64E−167 </td></tr><tr><td> </td><td>5.0 </td><td>643 </td><td>31 </td><td>4.82 </td><td>2.56E−59 </td><td>608 </td><td>33 </td><td>5.43 </td><td>9.96E−65 </td></tr><tr><td> </td><td>3.0 </td><td>777 </td><td>27 </td><td>3.48 </td><td>4.65E−48 </td><td>704 </td><td>29 </td><td>4.12 </td><td>1.26E−53 </td></tr><tr><td> </td><td>2.0 </td><td>929 </td><td>21 </td><td>2.26 </td><td>7.59E−34 </td><td>835 </td><td>15 </td><td>1.80 </td><td>3.92E−23 </td></tr><tr><td> </td><td>1.0 </td><td>735 </td><td>17 </td><td>1.09 </td><td>2.74E−11 </td><td>709 </td><td>9 </td><td>1.27 </td><td>4.06E−13 </td></tr><tr><td> </td><td>0.5 </td><td>702 </td><td>2 </td><td>0.29 </td><td>3.90E−03<sup>a</sup> </td><td>590 </td><td>1 </td><td>0.17 </td><td>1.37E−01<sup>a</sup> </td></tr></tbody></table></div><div content-id="DSR047TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Mutation </th><th>Dilution (%) </th><th colspan="4">Upper strand<hr /> </th><th colspan="4">Lower strand<hr /> </th></tr><tr><th> </th><th> </th><th>Totalreads </th><th>Mutantreads </th><th>%Mutant </th><th><em>P</em>-value </th><th>Totalreads </th><th>Mutantreads </th><th>%Mutant </th><th><em>P</em>-value </th></tr></thead><tbody><tr><td>c.1043C&gt;T; p.Thr348Met </td><td>10.0 </td><td>724 </td><td>61 </td><td>8.43 </td><td>8.62E−130 </td><td>520 </td><td>57 </td><td>10.96 </td><td>1.73E−117 </td></tr><tr><td> </td><td>5.0 </td><td>453 </td><td>24 </td><td>5.30 </td><td>2.86E−47 </td><td>372 </td><td>15 </td><td>4.03 </td><td>1.26E−25 </td></tr><tr><td> </td><td>3.0 </td><td>876 </td><td>27 </td><td>3.08 </td><td>1.16E−46 </td><td>757 </td><td>21 </td><td>2.77 </td><td>6.83E−32 </td></tr><tr><td> </td><td>2.0 </td><td>737 </td><td>10 </td><td>1.36 </td><td>1.05E−14 </td><td>645 </td><td>7 </td><td>1.09 </td><td>8.68E−09 </td></tr><tr><td> </td><td>1.0 </td><td>715 </td><td>9 </td><td>1.26 </td><td>4.73E−13 </td><td>624 </td><td>4 </td><td>0.64 </td><td>1.11E−04 </td></tr><tr><td> </td><td>0.5 </td><td>1025 </td><td>7 </td><td>0.68 </td><td>1.15E−14 </td><td>756 </td><td>3 </td><td>0.40 </td><td>3.22E−03<sup>a</sup> </td></tr><tr><td>c.1431C&gt;A; p.Asn477Lys </td><td>10.0 </td><td>542 </td><td>65 </td><td>11.99 </td><td>1.22E−113 </td><td>346 </td><td>24 </td><td>6.94 </td><td>6.84E−49 </td></tr><tr><td> </td><td>5.0 </td><td>491 </td><td>30 </td><td>6.11 </td><td>1.13E−44 </td><td>356 </td><td>17 </td><td>4.78 </td><td>2.42E−32 </td></tr><tr><td> </td><td>3.0 </td><td>487 </td><td>21 </td><td>4.31 </td><td>1.26E−28 </td><td>374 </td><td>19 </td><td>5.08 </td><td>1.78E−36 </td></tr><tr><td> </td><td>2.0 </td><td>577 </td><td>18 </td><td>3.12 </td><td>2.78E−22 </td><td>495 </td><td>9 </td><td>1.82 </td><td>4.57E−14 </td></tr><tr><td> </td><td>1.0 </td><td>491 </td><td>4 </td><td>0.82 </td><td>9.17E−04 </td><td>354 </td><td>5 </td><td>1.41 </td><td>7.34E−08 </td></tr><tr><td> </td><td>0.5 </td><td>483 </td><td>0 </td><td>0 </td><td>NA </td><td>424 </td><td>3 </td><td>0.71 </td><td>NA </td></tr><tr><td>c.1985T&gt;C; p.Met662Thr </td><td>10.0 </td><td>658 </td><td>79 </td><td>12.01 </td><td>1.13E−179 </td><td>643 </td><td>74 </td><td>11.51 </td><td>4.64E−167 </td></tr><tr><td> </td><td>5.0 </td><td>643 </td><td>31 </td><td>4.82 </td><td>2.56E−59 </td><td>608 </td><td>33 </td><td>5.43 </td><td>9.96E−65 </td></tr><tr><td> </td><td>3.0 </td><td>777 </td><td>27 </td><td>3.48 </td><td>4.65E−48 </td><td>704 </td><td>29 </td><td>4.12 </td><td>1.26E−53 </td></tr><tr><td> </td><td>2.0 </td><td>929 </td><td>21 </td><td>2.26 </td><td>7.59E−34 </td><td>835 </td><td>15 </td><td>1.80 </td><td>3.92E−23 </td></tr><tr><td> </td><td>1.0 </td><td>735 </td><td>17 </td><td>1.09 </td><td>2.74E−11 </td><td>709 </td><td>9 </td><td>1.27 </td><td>4.06E−13 </td></tr><tr><td> </td><td>0.5 </td><td>702 </td><td>2 </td><td>0.29 </td><td>3.90E−03<sup>a</sup> </td><td>590 </td><td>1 </td><td>0.17 </td><td>1.37E−01<sup>a</sup> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Not significant.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6211069" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=6211071 id="6211071" class="section-title" data-legacy-id=s3c>3.3. Detection and characterization of <em>NLRP3</em> somatic mosaicism using the MPS platform</h3>
<p>To demonstrate the power of this approach in practice, we applied our new pipeline for 10 CINCA/NOMID patients in whom we failed to detect mutations in the <em>NLRP3</em> gene using a conventional direct DNA sequencing approach. The mutations detected by the analysis formulated using the MPS platform in this study are listed in Table <a reveal-id="DSR047TB3" data-open="DSR047TB3" class="link link-reveal link-table xref-fig">3</a>. We successfully identified four out of the 10 patients with <em>NLRP3</em> somatic mosaicism, which was confirmed by subcloning and Sanger sequencing. The nucleotide substitutions were as follows (parentheses indicate the corresponding amino acid change): c.907G&gt;C (p.Asp303His), c.1699G&gt;A (p.Glu567Lys) in two patients, and c.906C&gt;A (p.Phe302Leu). The frequencies of mosaicism identified in these patients by the MPS approach were consistent with those that were identified by the subcloning and subsequent capillary DNA sequencing method (data not shown). Both c.907G&gt;C and c.1699G&gt;A variants were reported as CINCA/NOMID-associated mutations in Infevers database (<a class="link link-uri" href="http://fmf.igh.cnrs.fr/ISSAID/infevers/">http://fmf.igh.cnrs.fr/ISSAID/infevers/</a>) and in the dbSNP database (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/projects/SNP/">http://www.ncbi.nlm.nih.gov/projects/SNP/</a>).<sup><a reveal-id="DSR047C8" data-open="DSR047C8" class="link link-ref link-reveal xref-bibr">8</a></sup></p>                        <a id="6211074" scrollto-destination="6211074"></a>
<div content-id="DSR047TB3" class="table-modal"><div class="table-wrap"><div id="DSR047TB3" class="table-wrap-title" data-id="DSR047TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Potential mosaic mutations detected in patients with unknown mutations</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Patient ID </th><th>Amplicon # </th><th colspan="2">Variation<hr /> </th><th colspan="2">% Variationfrequency<hr /> </th><th colspan="2"><em>P</em>-value<hr /> </th><th>dbSNP </th><th>State </th></tr><tr><th> </th><th> </th><th> </th><th> </th><th>Forward </th><th>Reverse </th><th>Forward </th><th>Reverse </th><th> </th><th> </th></tr></thead><tbody><tr><td>P1 </td><td>Exon3_2 </td><td>c.907G&gt;C </td><td>p.Asp303His </td><td>7.12 </td><td>11.56 </td><td>3.0E−44 </td><td>1.7E−84 </td><td>rs121908153 </td><td>Known </td></tr><tr><td>P2 </td><td>Exon3_5 </td><td>c.1699G&gt;A </td><td>p.Glu567Lys </td><td>5.94 </td><td>5.79 </td><td>2.0E−69 </td><td>8.9E−47 </td><td>— </td><td>Known </td></tr><tr><td>P3 </td><td>Exon3_5 </td><td>c.1699G&gt;A </td><td>p.Glu567Lys </td><td>18.28 </td><td>15.33 </td><td>0.0E + 00 </td><td>1.0E−312 </td><td>— </td><td>Known </td></tr><tr><td>P4 </td><td>Exon3_2 </td><td>c.906C&gt;A </td><td>p.Phe302Leu </td><td>9.78 </td><td>9.70 </td><td>1.7E−86 </td><td>2.2E−122 </td><td>— </td><td>Novel </td></tr></tbody></table></div><div content-id="DSR047TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Patient ID </th><th>Amplicon # </th><th colspan="2">Variation<hr /> </th><th colspan="2">% Variationfrequency<hr /> </th><th colspan="2"><em>P</em>-value<hr /> </th><th>dbSNP </th><th>State </th></tr><tr><th> </th><th> </th><th> </th><th> </th><th>Forward </th><th>Reverse </th><th>Forward </th><th>Reverse </th><th> </th><th> </th></tr></thead><tbody><tr><td>P1 </td><td>Exon3_2 </td><td>c.907G&gt;C </td><td>p.Asp303His </td><td>7.12 </td><td>11.56 </td><td>3.0E−44 </td><td>1.7E−84 </td><td>rs121908153 </td><td>Known </td></tr><tr><td>P2 </td><td>Exon3_5 </td><td>c.1699G&gt;A </td><td>p.Glu567Lys </td><td>5.94 </td><td>5.79 </td><td>2.0E−69 </td><td>8.9E−47 </td><td>— </td><td>Known </td></tr><tr><td>P3 </td><td>Exon3_5 </td><td>c.1699G&gt;A </td><td>p.Glu567Lys </td><td>18.28 </td><td>15.33 </td><td>0.0E + 00 </td><td>1.0E−312 </td><td>— </td><td>Known </td></tr><tr><td>P4 </td><td>Exon3_2 </td><td>c.906C&gt;A </td><td>p.Phe302Leu </td><td>9.78 </td><td>9.70 </td><td>1.7E−86 </td><td>2.2E−122 </td><td>— </td><td>Novel </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6211074" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR047TB3" class="table-wrap-title" data-id="DSR047TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Potential mosaic mutations detected in patients with unknown mutations</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Patient ID </th><th>Amplicon # </th><th colspan="2">Variation<hr /> </th><th colspan="2">% Variationfrequency<hr /> </th><th colspan="2"><em>P</em>-value<hr /> </th><th>dbSNP </th><th>State </th></tr><tr><th> </th><th> </th><th> </th><th> </th><th>Forward </th><th>Reverse </th><th>Forward </th><th>Reverse </th><th> </th><th> </th></tr></thead><tbody><tr><td>P1 </td><td>Exon3_2 </td><td>c.907G&gt;C </td><td>p.Asp303His </td><td>7.12 </td><td>11.56 </td><td>3.0E−44 </td><td>1.7E−84 </td><td>rs121908153 </td><td>Known </td></tr><tr><td>P2 </td><td>Exon3_5 </td><td>c.1699G&gt;A </td><td>p.Glu567Lys </td><td>5.94 </td><td>5.79 </td><td>2.0E−69 </td><td>8.9E−47 </td><td>— </td><td>Known </td></tr><tr><td>P3 </td><td>Exon3_5 </td><td>c.1699G&gt;A </td><td>p.Glu567Lys </td><td>18.28 </td><td>15.33 </td><td>0.0E + 00 </td><td>1.0E−312 </td><td>— </td><td>Known </td></tr><tr><td>P4 </td><td>Exon3_2 </td><td>c.906C&gt;A </td><td>p.Phe302Leu </td><td>9.78 </td><td>9.70 </td><td>1.7E−86 </td><td>2.2E−122 </td><td>— </td><td>Novel </td></tr></tbody></table></div><div content-id="DSR047TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Patient ID </th><th>Amplicon # </th><th colspan="2">Variation<hr /> </th><th colspan="2">% Variationfrequency<hr /> </th><th colspan="2"><em>P</em>-value<hr /> </th><th>dbSNP </th><th>State </th></tr><tr><th> </th><th> </th><th> </th><th> </th><th>Forward </th><th>Reverse </th><th>Forward </th><th>Reverse </th><th> </th><th> </th></tr></thead><tbody><tr><td>P1 </td><td>Exon3_2 </td><td>c.907G&gt;C </td><td>p.Asp303His </td><td>7.12 </td><td>11.56 </td><td>3.0E−44 </td><td>1.7E−84 </td><td>rs121908153 </td><td>Known </td></tr><tr><td>P2 </td><td>Exon3_5 </td><td>c.1699G&gt;A </td><td>p.Glu567Lys </td><td>5.94 </td><td>5.79 </td><td>2.0E−69 </td><td>8.9E−47 </td><td>— </td><td>Known </td></tr><tr><td>P3 </td><td>Exon3_5 </td><td>c.1699G&gt;A </td><td>p.Glu567Lys </td><td>18.28 </td><td>15.33 </td><td>0.0E + 00 </td><td>1.0E−312 </td><td>— </td><td>Known </td></tr><tr><td>P4 </td><td>Exon3_2 </td><td>c.906C&gt;A </td><td>p.Phe302Leu </td><td>9.78 </td><td>9.70 </td><td>1.7E−86 </td><td>2.2E−122 </td><td>— </td><td>Novel </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6211074" target="_blank">View Large</a></div></div><p>Because the NLRP3 p.Phe302Leu mutation was novel and not detected in the 50 healthy controls, we performed an <em>in vitro</em> functional analysis to see the effect of p.Phe302Leu on the protein function. We used two different <em>in vitro</em> transfection experiments, the rapid cell death in transfected THP-1 cells and the ASC-dependent NF-κB activation in transfected HEK293FT cells (Fig. <a reveal-id="DSR047F3" data-open="DSR047F3" class="link link-reveal link-table xref-fig">3</a>A and B, respectively). Both assays clearly showed that p.Phe302Leu was a disease-causing mutation similar to known CINCA/NOMID-associated pathogenic mutations (p.Asp303His and p.Glu567Lys).<sup><a reveal-id="DSR047C9" data-open="DSR047C9" class="link link-ref link-reveal xref-bibr">9</a></sup></p>                        <a id="6211076" scrollto-destination="6211076"></a>
<div id="DSR047F3" data-id="DSR047F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dsr047/2/m_dsr04703.jpeg?Expires=2147483647&Signature=e90krs1Ikt~I36vZiR4~yvcrsTfCNdrjNIPQRBo2o6EmSYvX135hGDpMFKRSlREjPL-4onNwm~WG1U5Nw~RUGqJobplXGaNgDRWGA66d5LZ-8qRnh45EDAxmZ-ohtwZjXXOlnqcd8SaO39eHtz-C9ZVYkL7yAu67NaFR8iHK1~IVNvUIlS5Udq7RSAj3ZNnZO9raZ4CO~qyygLp5AyOSZmJrGdDt1MrYfvNeoKOU2jqZFisYjcgbNLreuu607FgEudocKw7u-ysh~I9iTvOl8gRsttbQmnomMKRR2fIAFvdOjSrkO-1-VoPBYQdtnE2cwNXeyRJDsbfza55n4nJwkg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="In vitro functional analysis of the identified NLRP3 mosaic mutations. (A) Rapid cell death in transfected THP-1 cells. A GFP-fused wild-type or mutant NLRP3 was transfected into THP-1 cells and incubated with PMA (10 ng/ml) for 4 h. The percentage of dead cells (7-amino-Actinomycin D [7-AAD]-positive) among the GFP-positive cells is shown. Data represent the means ± SD of triplicate experiments and are representative of two independent experiments. The data for previously reported mutations as well as the mutations found in this study are shown. (B) ACS-dependent NF-κB activation in transfected HEK293FT cells. HEK293FT cells were co-transfected with wild-type or mutant NLRP3 in the presence or absence of ASC. NF-κB induction is shown as the fold-change compared with cells that were transfected with a control vector without ASC (set equal to one). Values are the means ± SD of triplicate experiments, and the data are representative of three independent experiments. The data for previously reported mutations (p.Arg260Trp and p.Tyr570Cys) and the mutations found in this study are shown. For each mutation, the data obtained in the presence and absence of ASC are shown. These findings identified p.Phe302Leu as a novel disease-causing mutation." path-from-xml="dsr04703.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6211076/dsr04703.jpeg" path-from-xml="dsr04703.jpeg" target="_blank">View large</a><a section="6211076" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dsr047/2/dsr04703.jpeg?Expires=2147483647&Signature=KmpqQhzdkMSiFRKacFiYR3mo3Yvhd-u-2WgVbXORLzpwXpf-VOY~~yLvvrpUxz8lgHfPUHbdvqGHOKQcXP7Cpz5wtARjqVnvGfhqVWuY2-aYCp3Zeo~8IsojOXBmk69hnAJV-yb~wRwvJ2A67ydty1BxgAiAi2BZswbkA~O8fT9zIve7FQarxGhxLZJiQbOdaEjmePRDdpMmfxl90d4P81wio5~8repJ8SC2IgF~kP471jlrzB7QHlsna1fYBmWRv~vUojtHCgsS7v9oZ7RFB1ihhgjcZNRPtvj-R3QY0eWAr6Z~JPh9Yo43bn9meRRpCJ6QHUHR9w6V7ySsfNYqKQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6211076&ar=419384&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr04703.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p><em>In vitro</em> functional analysis of the identified <em>NLRP3</em> mosaic mutations. (A) Rapid cell death in transfected THP-1 cells. A GFP-fused wild-type or mutant <em>NLRP3</em> was transfected into THP-1 cells and incubated with PMA (10 ng/ml) for 4 h. The percentage of dead cells (7-amino-Actinomycin D [7-AAD]-positive) among the GFP-positive cells is shown. Data represent the means ± SD of triplicate experiments and are representative of two independent experiments. The data for previously reported mutations as well as the mutations found in this study are shown. (B) ACS-dependent NF-κB activation in transfected HEK293FT cells. HEK293FT cells were co-transfected with wild-type or mutant <em>NLRP3</em> in the presence or absence of ASC. NF-κB induction is shown as the fold-change compared with cells that were transfected with a control vector without ASC (set equal to one). Values are the means ± SD of triplicate experiments, and the data are representative of three independent experiments. The data for previously reported mutations (p.Arg260Trp and p.Tyr570Cys) and the mutations found in this study are shown. For each mutation, the data obtained in the presence and absence of ASC are shown. These findings identified p.Phe302Leu as a novel disease-causing mutation.</p></div></div><div content-id="DSR047F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dsr047/2/m_dsr04703.jpeg?Expires=2147483647&Signature=e90krs1Ikt~I36vZiR4~yvcrsTfCNdrjNIPQRBo2o6EmSYvX135hGDpMFKRSlREjPL-4onNwm~WG1U5Nw~RUGqJobplXGaNgDRWGA66d5LZ-8qRnh45EDAxmZ-ohtwZjXXOlnqcd8SaO39eHtz-C9ZVYkL7yAu67NaFR8iHK1~IVNvUIlS5Udq7RSAj3ZNnZO9raZ4CO~qyygLp5AyOSZmJrGdDt1MrYfvNeoKOU2jqZFisYjcgbNLreuu607FgEudocKw7u-ysh~I9iTvOl8gRsttbQmnomMKRR2fIAFvdOjSrkO-1-VoPBYQdtnE2cwNXeyRJDsbfza55n4nJwkg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="In vitro functional analysis of the identified NLRP3 mosaic mutations. (A) Rapid cell death in transfected THP-1 cells. A GFP-fused wild-type or mutant NLRP3 was transfected into THP-1 cells and incubated with PMA (10 ng/ml) for 4 h. The percentage of dead cells (7-amino-Actinomycin D [7-AAD]-positive) among the GFP-positive cells is shown. Data represent the means ± SD of triplicate experiments and are representative of two independent experiments. The data for previously reported mutations as well as the mutations found in this study are shown. (B) ACS-dependent NF-κB activation in transfected HEK293FT cells. HEK293FT cells were co-transfected with wild-type or mutant NLRP3 in the presence or absence of ASC. NF-κB induction is shown as the fold-change compared with cells that were transfected with a control vector without ASC (set equal to one). Values are the means ± SD of triplicate experiments, and the data are representative of three independent experiments. The data for previously reported mutations (p.Arg260Trp and p.Tyr570Cys) and the mutations found in this study are shown. For each mutation, the data obtained in the presence and absence of ASC are shown. These findings identified p.Phe302Leu as a novel disease-causing mutation." path-from-xml="dsr04703.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6211076/dsr04703.jpeg" path-from-xml="dsr04703.jpeg" target="_blank">View large</a><a section="6211076" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dsr047/2/dsr04703.jpeg?Expires=2147483647&Signature=KmpqQhzdkMSiFRKacFiYR3mo3Yvhd-u-2WgVbXORLzpwXpf-VOY~~yLvvrpUxz8lgHfPUHbdvqGHOKQcXP7Cpz5wtARjqVnvGfhqVWuY2-aYCp3Zeo~8IsojOXBmk69hnAJV-yb~wRwvJ2A67ydty1BxgAiAi2BZswbkA~O8fT9zIve7FQarxGhxLZJiQbOdaEjmePRDdpMmfxl90d4P81wio5~8repJ8SC2IgF~kP471jlrzB7QHlsna1fYBmWRv~vUojtHCgsS7v9oZ7RFB1ihhgjcZNRPtvj-R3QY0eWAr6Z~JPh9Yo43bn9meRRpCJ6QHUHR9w6V7ySsfNYqKQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6211076&ar=419384&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr04703.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p><em>In vitro</em> functional analysis of the identified <em>NLRP3</em> mosaic mutations. (A) Rapid cell death in transfected THP-1 cells. A GFP-fused wild-type or mutant <em>NLRP3</em> was transfected into THP-1 cells and incubated with PMA (10 ng/ml) for 4 h. The percentage of dead cells (7-amino-Actinomycin D [7-AAD]-positive) among the GFP-positive cells is shown. Data represent the means ± SD of triplicate experiments and are representative of two independent experiments. The data for previously reported mutations as well as the mutations found in this study are shown. (B) ACS-dependent NF-κB activation in transfected HEK293FT cells. HEK293FT cells were co-transfected with wild-type or mutant <em>NLRP3</em> in the presence or absence of ASC. NF-κB induction is shown as the fold-change compared with cells that were transfected with a control vector without ASC (set equal to one). Values are the means ± SD of triplicate experiments, and the data are representative of three independent experiments. The data for previously reported mutations (p.Arg260Trp and p.Tyr570Cys) and the mutations found in this study are shown. For each mutation, the data obtained in the presence and absence of ASC are shown. These findings identified p.Phe302Leu as a novel disease-causing mutation.</p></div></div>                            <h2 scrollto-destination=6211077 id="6211077" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>Although the somatic mutation rate at the nucleotide level <em>in vivo</em> was difficult to quantitatively measure due to the complexity of the genome and laborious molecular detection processes, recent advances in MPS technologies have allowed us to directly quantitate somatic mutations in human genome.<sup><a reveal-id="DSR047C20  DSR047C22" data-open="DSR047C20  DSR047C22" class="link link-ref link-reveal xref-bibr">20–22</a></sup> The current estimate for the somatic (<em>de novo</em>) mutation rate is 1–2 × 10<sup>−8</sup> residues/generation/haploid, and this estimate is sufficiently low that we would expect to never observe somatic mosaicism in the <em>NLRP3</em> gene by chance; although the error rate of the high-fidelity DNA polymerase used to produce the amplicons is two orders of magnitude larger than the somatic mutation rate,<sup><a reveal-id="DSR047C23" data-open="DSR047C23" class="link link-ref link-reveal xref-bibr">23</a>,<a reveal-id="DSR047C24" data-open="DSR047C24" class="link link-ref link-reveal xref-bibr">24</a></sup> we could not detect PCR-generated mosaicism higher than 1% in the 454 sequencing error maps. Based on the literature, the single base substitutions are the most frequent type of somatic mutations (∼500 times more frequent than short insertions/deletions)<sup><a reveal-id="DSR047C25" data-open="DSR047C25" class="link link-ref link-reveal xref-bibr">25</a></sup> and protein-coding sequences are less mutagenic than sequences in non-coding regions, assuming that the somatic mutation spectrum in malignant cells is the same as in normal cells. Somatic mosaicism is thought to result from <em>de novo</em> gain-of-function-type mutations that are introduced at a very early and limited stage of development, and it is reasonable to focus our efforts on detecting base substitutions for somatic mosaicism in the <em>NLRP3</em> gene.</p><p>It is challenging but highly important in many areas of research, such as cancer, to detect low-level somatic mutations, which we designated as somatic mosaicism in this study, from apparently mutation-negative samples by conventional sequencing. Subcloning followed by the capillary DNA sequencing has been a <em>de facto</em> standard to identify somatic mosaicism, but this is not the method of choice for routine diagnostics because it is laborious, time consuming, and costly. Thus, it is reasonable for us to explore MPS as a new tool for this purpose. Although previous studies have used MPS technology to detect somatic mosaicism, it was unclear how sensitive this method is to detect a low-level somatic mosaicism using the MPS platform because this platform is generally error-prone. To address this challenge, we developed a new pipeline to detect low-level somatic mosaicism with statistical confidence using base position- and strand-specific error rate maps for the <em>NLRP3</em> amplicons to be studied. Whereas the detection limit of somatic mosaicism depends on the base position and the read depth of the amplicons, the limit of detection could be as low as 1% allele frequency with no false positives for substitutions (the precision is higher than 99.9%). Our error map shows that 98.1% of base positions (3343 out of 3407 target positions) in the <em>NLRP3</em> exonic amplicons can be detected with ∼1% mosaicism when more than ∼350 reads were accumulated for each strand. Although the remaining region (64 base positions out of 3407 target positions) was too error-prone (the error rate ranged from 0.1 to 1.7% in either the upper or lower strand) to detect low-level mosaicism by MPS, medium-level mosaicism (5% or high) could be identified in all base positions in the target region with the same significance level. Based on this pipeline, we successfully identified four cases of somatic mosaicism among 10 apparently mutation-negative CINCA/NOMID patients. These results were subsequently confirmed by functional analysis and subcloning followed by capillary DNA sequencing method.</p><p>As described above, we revealed that a read depth of ∼350 for each strand of each amplicon would be sufficient to detect somatic mosaicism as low as 1% with statistical confidence. This means that an analysis of somatic mosaicism (detection limit of 1% allele frequency) of the <em>NLRP3</em> gene for one sample requires 350 × 2 × 14 = 9800 reads with the 454 GS-FLX sequencer, which has a capacity to obtain 1 000 000 reads per run. Thus, we could analyse ∼100 patient samples with a single run (∼10 h) using this MPS platform. For this purpose, a miniaturized 454 sequencer might be more convenient because it could analyse 10 patient samples at once with a reasonably reduced running cost.</p><p>The approach used to detect somatic mosaicism is very similar to that for low-frequency alleles in pooled DNA samples, for which MPS applications have been reported by many groups.<sup><a reveal-id="DSR047C18" data-open="DSR047C18" class="link link-ref link-reveal xref-bibr">18</a>,<a reveal-id="DSR047C26" data-open="DSR047C26" class="link link-ref link-reveal xref-bibr">26</a>,<a reveal-id="DSR047C27" data-open="DSR047C27" class="link link-ref link-reveal xref-bibr">27</a></sup> However, the main aim of these previous studies was to screen for a rare allele in a population. Thus, the discovery phase on the MPS platform must be followed by an evaluation phase using conventional methods. Therefore, when diagnosing somatic mosaicism of the <em>NLRP3</em> gene based solely on the MPS platform, we could not use the same approach to detect rare alleles in a population due to its low accuracy. The sequencing error rate on the Roche MPS platform was sufficiently stable and low enough as shown in this study. Using our pipeline, we were able to detect 1% somatic mosaicism in the <em>NLRP3</em> gene with 99.9% confidence. Although another research group recently used a similar approach with a short-read MPS,<sup><a reveal-id="DSR047C28" data-open="DSR047C28" class="link link-ref link-reveal xref-bibr">28</a></sup> the Roche long-read MPS is more suitable as a diagnostic tool mainly because of the short run time. If we could diagnose somatic mosaicism of the <em>NLRP3</em> gene within a reasonable time with low labour and costs as shown in this study, the success rate of CINCA/NOMID genetic diagnosis will increase from 60 to 80% or higher,<sup><a reveal-id="DSR047C9" data-open="DSR047C9" class="link link-ref link-reveal xref-bibr">9</a></sup> which will greatly advance the health and care of these patients and prevent irreversible bone and neurological complications of disease.</p><p>This pipeline would also be efficient to detect somatic mosaicism in mutation-negative patients with other diseases, including cancer. The error rate map for a given gene should be constructed from authentic plasmids, and used to detect somatic mosaicism of other genes as well as rare alleles in various populations.</p>                            <h2 scrollto-destination=6211083 id="6211083" class="section-title" data-legacy-id=s5>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available at</a><a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org">www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=6211085 id="6211085" class="section-title" data-legacy-id=s6>Funding</h2>
<p>This study was supported by the Japanese Ministry of Education, Science, Sports, and Culture, and the Japanese Ministry of Health, Labor, and Welfare.</p>                            <h2 scrollto-destination=6211087 id="6211087" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank all patients who participated in the study. We are grateful to Ms. Yuki Takaoka at the Department of Pediatrics, Kyoto University Graduate School of Medicine and Mr. Takashi Watanabe at the Department of Human Genome Research, Kazusa DNA Research Institute for their technical assistance.</p>                            <h2 scrollto-destination=6211089 id="6211089" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSR047C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR047C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Prieur</div> <div class="given-names">A.M.</div></div>,  <div class="name"><div class="surname">Griscelli</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Arthropathy with rash, chronic meningitis, eye lesions, and mental retardation</div>, <div class="source">J. Pediatr.</div> , <div class="year">1981</div>, vol. <div class="volume">99</div> (pg. <div class="fpage">79</div>-<div class="lpage">83</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/S0022-3476(81)80961-4" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/S0022-3476(81)80961-4</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Arthropathy%20with%20rash%2C%20chronic%20meningitis%2C%20eye%20lesions%2C%20and%20mental%20retardation&amp;author=A.M.%20Prieur&amp;author=C.%20Griscelli&amp;publication_year=1981&amp;journal=J.%20Pediatr.&amp;volume=99&amp;pages=79-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0022-3476(81)80961-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0022-3476(81)80961-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0022-3476(81)80961-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7252669" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR047C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR047C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hassink</div> <div class="given-names">S.G.</div></div>,  <div class="name"><div class="surname">Goldsmith</div> <div class="given-names">D.P.</div></div>. </div><div class="article-title">Neonatal onset multisystem inflammatory disease</div>, <div class="source">Arthritis Rheum.</div> , <div class="year">1983</div>, vol. <div class="volume">26</div> (pg. <div class="fpage">668</div>-<div class="lpage">73</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1002/art.1780260515" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1002/art.1780260515</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Neonatal%20onset%20multisystem%20inflammatory%20disease&amp;author=S.G.%20Hassink&amp;author=D.P.%20Goldsmith&amp;publication_year=1983&amp;journal=Arthritis%20Rheum.&amp;volume=26&amp;pages=668-73" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1529-0131" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1529-0131" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1529-0131"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6847730" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR047C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR047C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Torbiak</div> <div class="given-names">R.P.</div></div>,  <div class="name"><div class="surname">Dent</div> <div class="given-names">P.B.</div></div>,  <div class="name"><div class="surname">Cockshott</div> <div class="given-names">W.P.</div></div>. </div><div class="article-title">NOMID–a neonatal syndrome of multisystem inflammation</div>, <div class="source">Skeletal Radiol.</div> , <div class="year">1989</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">359</div>-<div class="lpage">64</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/BF00361425" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/BF00361425</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=NOMID%E2%80%93a%20neonatal%20syndrome%20of%20multisystem%20inflammation&amp;author=R.P.%20Torbiak&amp;author=P.B.%20Dent&amp;author=W.P.%20Cockshott&amp;publication_year=1989&amp;journal=Skeletal%20Radiol.&amp;volume=18&amp;pages=359-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00361425" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00361425" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00361425"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2781338" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR047C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR047C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Feldmann</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Prieur</div> <div class="given-names">A.M.</div></div>,  <div class="name"><div class="surname">Quartier</div> <div class="given-names">P.</div></div>, et al. </div><div class="article-title">Chronic infantile neurological cutaneous and articular syndrome is caused by mutations in CIAS1, a gene highly expressed in polymorphonuclear cells and chondrocytes</div>, <div class="source">Am. J. Hum. Genet. United States</div> , <div class="year">2002</div>(pg. <div class="fpage">198</div>-<div class="lpage">203</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1086/341357" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1086/341357</a></div></div></div></div></div><div content-id="DSR047C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR047C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Aksentijevich</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Nowak</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Mallah</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">De novo CIAS1 mutations, cytokine activation, and evidence for genetic heterogeneity in patients with neonatal-onset multisystem inflammatory disease (NOMID)—a new member of the expanding family of pyrin-associated autoinflammatory diseases</div>, <div class="source">Arthritis Rheum.</div> , <div class="year">2002</div>, vol. <div class="volume">46</div> (pg. <div class="fpage">3340</div>-<div class="lpage">8</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1002/art.10688" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1002/art.10688</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=De%20novo%20CIAS1%20mutations%2C%20cytokine%20activation%2C%20and%20evidence%20for%20genetic%20heterogeneity%20in%20patients%20with%20neonatal-onset%20multisystem%20inflammatory%20disease%20(NOMID)%E2%80%94a%20new%20member%20of%20the%20expanding%20family%20of%20pyrin-associated%20autoinflammatory%20diseases&amp;author=I.%20Aksentijevich&amp;author=M.%20Nowak&amp;author=M.%20Mallah&amp;publication_year=2002&amp;journal=Arthritis%20Rheum.&amp;volume=46&amp;pages=3340-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1529-0131" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1529-0131" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1529-0131"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12483741" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR047C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR047C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hoffman</div> <div class="given-names">H.M.</div></div>,  <div class="name"><div class="surname">Mueller</div> <div class="given-names">J.L.</div></div>,  <div class="name"><div class="surname">Broide</div> <div class="given-names">D.H.</div></div>,  <div class="name"><div class="surname">Wanderer</div> <div class="given-names">A.A.</div></div>,  <div class="name"><div class="surname">Kolodner</div> <div class="given-names">R.D.</div></div>. </div><div class="article-title">Mutation of a new gene encoding a putative pyrin-like protein causes familial cold autoinflammatory syndrome and Muckle-Wells syndrome</div>, <div class="source">Nat. Genet.</div> , <div class="year">2001</div>, vol. <div class="volume">29</div> (pg. <div class="fpage">301</div>-<div class="lpage">5</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/ng756" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/ng756</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mutation%20of%20a%20new%20gene%20encoding%20a%20putative%20pyrin-like%20protein%20causes%20familial%20cold%20autoinflammatory%20syndrome%20and%20Muckle-Wells%20syndrome&amp;author=H.M.%20Hoffman&amp;author=J.L.%20Mueller&amp;author=D.H.%20Broide&amp;author=A.A.%20Wanderer&amp;author=R.D.%20Kolodner&amp;publication_year=2001&amp;journal=Nat.%20Genet.&amp;volume=29&amp;pages=301-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng756" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng756" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng756"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11687797" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR047C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR047C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Goldbach-Mansky</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Current status of understanding the pathogenesis and management of patients with NOMID/CINCA</div>, <div class="source">Curr. Rheumatol. Rep.</div> , <div class="year">2011</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">123</div>-<div class="lpage">31</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s11926-011-0165-y" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s11926-011-0165-y</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Current%20status%20of%20understanding%20the%20pathogenesis%20and%20management%20of%20patients%20with%20NOMID%2FCINCA&amp;author=R.%20Goldbach-Mansky&amp;publication_year=2011&amp;journal=Curr.%20Rheumatol.%20Rep.&amp;volume=13&amp;pages=123-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11926-011-0165-y" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11926-011-0165-y" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11926-011-0165-y"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21538043" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR047C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR047C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Milhavet</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Cuisset</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Hoffman</div> <div class="given-names">H.M.</div></div>, et al. </div><div class="article-title">The infevers autoinflammatory mutation online registry: Update with new genes and functions</div>, <div class="source">Hum. Mutat.</div> , <div class="year">2008</div>, vol. <div class="volume">29</div> (pg. <div class="fpage">803</div>-<div class="lpage">8</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1002/humu.20720" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1002/humu.20720</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20infevers%20autoinflammatory%20mutation%20online%20registry%3A%20Update%20with%20new%20genes%20and%20functions&amp;author=F.%20Milhavet&amp;author=L.%20Cuisset&amp;author=H.M.%20Hoffman&amp;publication_year=2008&amp;journal=Hum.%20Mutat.&amp;volume=29&amp;pages=803-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/humu.v29:6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fhumu.v29:6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Fhumu.v29:6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18409191" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR047C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR047C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tanaka</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Izawa</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Saito</div> <div class="given-names">M.K.</div></div>, et al. </div><div class="article-title">High incidence of NLRP3 somatic mosaicism in patients with chronic infantile neurologic, cutaneous, articular syndrome: results of an International Multicenter Collaborative Study</div>, <div class="source">Arthritis Rheum.</div> , <div class="year">2011</div>, vol. <div class="volume">63</div> (pg. <div class="fpage">3625</div>-<div class="lpage">32</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1002/art.30512" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1002/art.30512</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High%20incidence%20of%20NLRP3%20somatic%20mosaicism%20in%20patients%20with%20chronic%20infantile%20neurologic%2C%20cutaneous%2C%20articular%20syndrome%3A%20results%20of%20an%20International%20Multicenter%20Collaborative%20Study&amp;author=N.%20Tanaka&amp;author=K.%20Izawa&amp;author=M.K.%20Saito&amp;publication_year=2011&amp;journal=Arthritis%20Rheum.&amp;volume=63&amp;pages=3625-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/art.v63.11" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fart.v63.11" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Fart.v63.11"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21702021" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR047C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR047C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Qin</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Kozlowski</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Taillon</div> <div class="given-names">B.E.</div></div>, et al. </div><div class="article-title">Ultra deep sequencing detects a low rate of mosaic mutations in tuberous sclerosis complex</div>, <div class="source">Hum. Genet.</div> , <div class="year">2010</div>, vol. <div class="volume">127</div> (pg. <div class="fpage">573</div>-<div class="lpage">82</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s00439-010-0801-z" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s00439-010-0801-z</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ultra%20deep%20sequencing%20detects%20a%20low%20rate%20of%20mosaic%20mutations%20in%20tuberous%20sclerosis%20complex&amp;author=W.%20Qin&amp;author=P.%20Kozlowski&amp;author=B.E.%20Taillon&amp;publication_year=2010&amp;journal=Hum.%20Genet.&amp;volume=127&amp;pages=573-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00439-010-0801-z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00439-010-0801-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00439-010-0801-z"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20165957" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR047C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR047C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Campbell</div> <div class="given-names">P.J.</div></div>,  <div class="name"><div class="surname">Pleasance</div> <div class="given-names">E.D.</div></div>,  <div class="name"><div class="surname">Stephens</div> <div class="given-names">P.J.</div></div>, et al. </div><div class="article-title">Subclonal phylogenetic structures in cancer revealed by ultra-deep sequencing</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2008</div>, vol. <div class="volume">105</div> (pg. <div class="fpage">13081</div>-<div class="lpage">6</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1073/pnas.0801523105" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1073/pnas.0801523105</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Subclonal%20phylogenetic%20structures%20in%20cancer%20revealed%20by%20ultra-deep%20sequencing&amp;author=P.J.%20Campbell&amp;author=E.D.%20Pleasance&amp;author=P.J.%20Stephens&amp;publication_year=2008&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=105&amp;pages=13081-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0801523105" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0801523105" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0801523105"> </span></div> </div></div></div></div></div><div content-id="DSR047C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR047C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rohlin</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Wernersson</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Engwall</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Wiklund</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Bjoerk</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Nordling</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Parallel sequencing used in detection of mosaic mutations: Comparison with four diagnostic DNA screening techniques</div>, <div class="source">Hum. Mutat.</div> , <div class="year">2009</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">1012</div>-<div class="lpage">20</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1002/humu.20980" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1002/humu.20980</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Parallel%20sequencing%20used%20in%20detection%20of%20mosaic%20mutations%3A%20Comparison%20with%20four%20diagnostic%20DNA%20screening%20techniques&amp;author=A.%20Rohlin&amp;author=J.%20Wernersson&amp;author=Y.%20Engwall&amp;author=L.%20Wiklund&amp;author=J.%20Bjoerk&amp;author=M.%20Nordling&amp;publication_year=2009&amp;journal=Hum.%20Mutat.&amp;volume=30&amp;pages=1012-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/humu.v30:6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fhumu.v30:6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Fhumu.v30:6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19347965" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR047C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR047C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Saito</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Nishikomori</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Kambe</div> <div class="given-names">N.</div></div>, et al. </div><div class="article-title">Disease-associated CIAS1 mutations induce monocyte death, revealing low-level mosaicism in mutation-negative cryopyrin-associated periodic syndrome patients</div>, <div class="source">Blood</div> , <div class="year">2008</div>, vol. <div class="volume">111</div> (pg. <div class="fpage">2132</div>-<div class="lpage">41</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1182/blood-2007-06-094201" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1182/blood-2007-06-094201</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Disease-associated%20CIAS1%20mutations%20induce%20monocyte%20death%2C%20revealing%20low-level%20mosaicism%20in%20mutation-negative%20cryopyrin-associated%20periodic%20syndrome%20patients&amp;author=M.%20Saito&amp;author=R.%20Nishikomori&amp;author=N.%20Kambe&amp;publication_year=2008&amp;journal=Blood&amp;volume=111&amp;pages=2132-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1182/blood-2007-06-094201" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1182%2Fblood-2007-06-094201" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1182%2Fblood-2007-06-094201"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18063752" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR047C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR047C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kent</div> <div class="given-names">W.J.</div></div>. </div><div class="article-title">BLAT—the BLAST-like alignment tool</div>, <div class="source">Genome Res.</div> , <div class="year">2002</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">656</div>-<div class="lpage">64</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=BLAT%E2%80%94the%20BLAST-like%20alignment%20tool&amp;author=W.J.%20Kent&amp;publication_year=2002&amp;journal=Genome%20Res.&amp;volume=12&amp;pages=656-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.229202. Article published online before March 2002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.229202. Article published online before March 2002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.229202. Article published online before March 2002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11932250" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR047C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR047C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fujisawa</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Kambe</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Saito</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Disease-associated mutations in CIAS1 induce cathepsin B-dependent rapid cell death of human THP-1 monocytic cells</div>, <div class="source">Blood</div> , <div class="year">2007</div>, vol. <div class="volume">109</div> (pg. <div class="fpage">2903</div>-<div class="lpage">11</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Disease-associated%20mutations%20in%20CIAS1%20induce%20cathepsin%20B-dependent%20rapid%20cell%20death%20of%20human%20THP-1%20monocytic%20cells&amp;author=A.%20Fujisawa&amp;author=N.%20Kambe&amp;author=M.%20Saito&amp;publication_year=2007&amp;journal=Blood&amp;volume=109&amp;pages=2903-11" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17164343" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR047C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR047C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gilles</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Meglecz</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Pech</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Ferreira</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Malausa</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Martin</div> <div class="given-names">J.F.</div></div>. </div><div class="article-title">Accuracy and quality assessment of 454 GS-FLX Titanium pyrosequencing</div>, <div class="source">BMC Genomics</div> , <div class="year">2011</div>, vol. <div class="volume">12</div> pg. <div class="fpage">245</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2164-12-245" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2164-12-245</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Accuracy%20and%20quality%20assessment%20of%20454%20GS-FLX%20Titanium%20pyrosequencing&amp;author=A.%20Gilles&amp;author=E.%20Meglecz&amp;author=N.%20Pech&amp;author=S.%20Ferreira&amp;author=T.%20Malausa&amp;author=J.F.%20Martin&amp;publication_year=2011&amp;journal=BMC%20Genomics&amp;volume=12&amp;pages=245" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-12-245" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-12-245" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-12-245"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21592414" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR047C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR047C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Margulies</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Egholm</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Altman</div> <div class="given-names">W.E.</div></div>, et al. </div><div class="article-title">Genome sequencing in microfabricated high-density picolitre reactors</div>, <div class="source">Nature</div> , <div class="year">2005</div>, vol. <div class="volume">437</div> (pg. <div class="fpage">376</div>-<div class="lpage">80</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20sequencing%20in%20microfabricated%20high-density%20picolitre%20reactors&amp;author=M.%20Margulies&amp;author=M.%20Egholm&amp;author=W.E.%20Altman&amp;publication_year=2005&amp;journal=Nature&amp;volume=437&amp;pages=376-80" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16056220" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR047C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR047C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Altmann</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Weber</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Quast</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Rex-Haffner</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Binder</div> <div class="given-names">E.B.</div></div>,  <div class="name"><div class="surname">Muller-Myhsok</div> <div class="given-names">B.</div></div>. </div><div class="article-title">vipR: variant identification in pooled DNA using R</div>, <div class="source">Bioinformatics</div> , <div class="year">2011</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">i77</div>-<div class="lpage">84</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/bioinformatics/btr205" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/bioinformatics/btr205</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=vipR%3A%20variant%20identification%20in%20pooled%20DNA%20using%20R&amp;author=A.%20Altmann&amp;author=P.%20Weber&amp;author=C.%20Quast&amp;author=M.%20Rex-Haffner&amp;author=E.B.%20Binder&amp;author=B.%20Muller-Myhsok&amp;publication_year=2011&amp;journal=Bioinformatics&amp;volume=27&amp;pages=i77-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btr205" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtr205" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtr205"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21685105" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR047C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR047C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dohm</div> <div class="given-names">J.C.</div></div>,  <div class="name"><div class="surname">Lottaz</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Borodina</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Himmelbauer</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Substantial biases in ultra-short read data sets from high-throughput DNA sequencing</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2008</div>, vol. <div class="volume">36</div> pg. <div class="fpage">e105</div> <div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/gkn425" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/gkn425</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Substantial%20biases%20in%20ultra-short%20read%20data%20sets%20from%20high-throughput%20DNA%20sequencing&amp;author=J.C.%20Dohm&amp;author=C.%20Lottaz&amp;author=T.%20Borodina&amp;author=H.%20Himmelbauer&amp;publication_year=2008&amp;journal=Nucleic%20Acids%20Res.&amp;volume=36&amp;pages=e105" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkn425" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkn425" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkn425"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18660515" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR047C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR047C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lee</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Jiang</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Liu</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">The mutation spectrum revealed by paired genome sequences from a lung cancer patient</div>, <div class="source">Nature</div> , <div class="year">2010</div>, vol. <div class="volume">465</div> (pg. <div class="fpage">473</div>-<div class="lpage">7</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nature09004" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nature09004</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20mutation%20spectrum%20revealed%20by%20paired%20genome%20sequences%20from%20a%20lung%20cancer%20patient&amp;author=W.%20Lee&amp;author=Z.%20Jiang&amp;author=J.%20Liu&amp;publication_year=2010&amp;journal=Nature&amp;volume=465&amp;pages=473-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature09004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature09004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature09004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20505728" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR047C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR047C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Awadalla</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Gauthier</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Myers</div> <div class="given-names">R.A.</div></div>, et al. </div><div class="article-title">Direct Measure of the de novo mutation rate in Autism and Schizophrenia Cohorts</div>, <div class="source">Am. J. Hum. Genet.</div> , <div class="year">2010</div>, vol. <div class="volume">87</div> (pg. <div class="fpage">316</div>-<div class="lpage">24</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.ajhg.2010.07.019" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.ajhg.2010.07.019</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Direct%20Measure%20of%20the%20de%20novo%20mutation%20rate%20in%20Autism%20and%20Schizophrenia%20Cohorts&amp;author=P.%20Awadalla&amp;author=J.%20Gauthier&amp;author=R.A.%20Myers&amp;publication_year=2010&amp;journal=Am.%20J.%20Hum.%20Genet.&amp;volume=87&amp;pages=316-24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ajhg.2010.07.019" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ajhg.2010.07.019" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ajhg.2010.07.019"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20797689" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR047C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR047C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Conrad</div> <div class="given-names">D.F.</div></div>,  <div class="name"><div class="surname">Keebler</div> <div class="given-names">J.E.M.</div></div>,  <div class="name"><div class="surname">DePristo</div> <div class="given-names">M.A.</div></div>, et al. </div><div class="article-title">Variation in genome-wide mutation rates within and between human families</div>, <div class="source">Nat. Genet.</div> , <div class="year">2011</div>, vol. <div class="volume">43</div> (pg. <div class="fpage">712</div>-<div class="lpage">4</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/ng.862" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/ng.862</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Variation%20in%20genome-wide%20mutation%20rates%20within%20and%20between%20human%20families&amp;author=D.F.%20Conrad&amp;author=J.E.M.%20Keebler&amp;author=M.A.%20DePristo&amp;publication_year=2011&amp;journal=Nat.%20Genet.&amp;volume=43&amp;pages=712-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.862" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.862" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.862"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21666693" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR047C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR047C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cha</div> <div class="given-names">R.S.</div></div>,  <div class="name"><div class="surname">Thilly</div> <div class="given-names">W.G.</div></div>. </div><div class="article-title">Specificity, efficiency, and fidelity of PCR</div>, <div class="source">PCR Methods Appl.</div> , <div class="year">1993</div>, vol. <div class="volume">3</div> (pg. <div class="fpage">S18</div>-<div class="lpage">29</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Specificity%2C%20efficiency%2C%20and%20fidelity%20of%20PCR&amp;author=R.S.%20Cha&amp;author=W.G.%20Thilly&amp;publication_year=1993&amp;journal=PCR%20Methods%20Appl.&amp;volume=3&amp;pages=S18-29" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.3.3.S18" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.3.3.S18" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.3.3.S18"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8118393" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR047C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR047C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Vandenbroucke</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Marck</div> <div class="given-names">H.V.</div></div>,  <div class="name"><div class="surname">Verhasselt</div> <div class="given-names">P.</div></div>, et al. </div><div class="article-title">Minor variant detection in amplicons using 454 massive parallel pyrosequencing: experiences and considerations for successful applications</div>, <div class="source">BioTechniques</div> , <div class="year">2011</div>, vol. <div class="volume">51</div> (pg. <div class="fpage">167</div>-<div class="lpage">77</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Minor%20variant%20detection%20in%20amplicons%20using%20454%20massive%20parallel%20pyrosequencing%3A%20experiences%20and%20considerations%20for%20successful%20applications&amp;author=I.%20Vandenbroucke&amp;author=H.V.%20Marck&amp;author=P.%20Verhasselt&amp;publication_year=2011&amp;journal=BioTechniques&amp;volume=51&amp;pages=167-77" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2144/000113733" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2144%2F000113733" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2144%2F000113733"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21906038" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR047C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR047C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pleasance</div> <div class="given-names">E.D.</div></div>,  <div class="name"><div class="surname">Cheetham</div> <div class="given-names">R.K.</div></div>,  <div class="name"><div class="surname">Stephens</div> <div class="given-names">P.J.</div></div>, et al. </div><div class="article-title">A comprehensive catalogue of somatic mutations from a human cancer genome</div>, <div class="source">Nature</div> , <div class="year">2010</div>, vol. <div class="volume">463</div> (pg. <div class="fpage">191</div>-<div class="lpage">6</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nature08658" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nature08658</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20comprehensive%20catalogue%20of%20somatic%20mutations%20from%20a%20human%20cancer%20genome&amp;author=E.D.%20Pleasance&amp;author=R.K.%20Cheetham&amp;author=P.J.%20Stephens&amp;publication_year=2010&amp;journal=Nature&amp;volume=463&amp;pages=191-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature08658" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature08658" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature08658"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20016485" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR047C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR047C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fakhrai-Rad</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Zheng</div> <div class="given-names">J.B.</div></div>,  <div class="name"><div class="surname">Willis</div> <div class="given-names">T.D.</div></div>, et al. </div><div class="article-title">SNP discovery in pooled samples with mismatch repair detection</div>, <div class="source">Genome Res.</div> , <div class="year">2004</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">1404</div>-<div class="lpage">12</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1101/gr.2373904" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1101/gr.2373904</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SNP%20discovery%20in%20pooled%20samples%20with%20mismatch%20repair%20detection&amp;author=H.%20Fakhrai-Rad&amp;author=J.B.%20Zheng&amp;author=T.D.%20Willis&amp;publication_year=2004&amp;journal=Genome%20Res.&amp;volume=14&amp;pages=1404-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.2373904" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.2373904" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.2373904"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15231755" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR047C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR047C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bansal</div> <div class="given-names">V.</div></div>. </div><div class="article-title">A statistical method for the detection of variants from next-generation resequencing of DNA pools</div>, <div class="source">Bioinformatics</div> , <div class="year">2010</div>, vol. <div class="volume">26</div> (pg. <div class="fpage">i318</div>-<div class="lpage">24</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/bioinformatics/btq214" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/bioinformatics/btq214</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20statistical%20method%20for%20the%20detection%20of%20variants%20from%20next-generation%20resequencing%20of%20DNA%20pools&amp;author=V.%20Bansal&amp;publication_year=2010&amp;journal=Bioinformatics&amp;volume=26&amp;pages=i318-24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btq214" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtq214" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtq214"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20529923" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR047C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR047C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Flaherty</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Natsoulis</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Muralidharan</div> <div class="given-names">O.</div></div>, et al. </div><div class="article-title">Ultrasensitive detection of rare mutations using next-generation targeted resequencing</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2012</div>, vol. <div class="volume">40</div> pg. <div class="fpage">e2</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ultrasensitive%20detection%20of%20rare%20mutations%20using%20next-generation%20targeted%20resequencing&amp;author=P.%20Flaherty&amp;author=G.%20Natsoulis&amp;author=O.%20Muralidharan&amp;publication_year=2012&amp;journal=Nucleic%20Acids%20Res.&amp;volume=40&amp;pages=e2" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkr861" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr861" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkr861"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22013163" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2012. Published by Oxford University Press on behalf of Kazusa DNA Research Institute</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/3.0/), which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=419384&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dsr047/2/dsr047-dsr047supp_tables.xlsx?Expires=2147483647&amp;Signature=Jf~9HjQHrXvz1ATPjhoGNy4hvInk7ul1NGxWTUcHPBi1B0z3-JyYMDljKO7rPOVSHV22kVk40QEIIGYSTEgJd7refJD3~IIJ10OBnBkVLMqLToILOFx7Ug14S1y0IRDNOJCpkn7eQs07d3QXZ6CuN9hNWxO06BhokpO~R2~wOFy~7TJibCKs0~GsvTRoaKFurAUQbxNJkCUscTuWQ8qp283vTqZR~SpmcDzJyGkpV4SOZH3c7Cc7BWIzHawlnaouTQO~5C6AThWrfNceogbQDvKqCg1vtYQBnJQ4rxScWxI3Fgj~NfO7s8AMojefPRY1fgMm2PzrShrES5bkY05pmQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-6</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dsr047/2/dsr047-dsr047supp_fig4.pdf?Expires=2147483647&amp;Signature=tf0xLmAXqhYNBEuWdXP5u0KWm3ZySx9TGyf0XNQp2JJ5-39uVyT0rMhzNC6D9zB8uo0bnz0SJU6Tv401WbGl~che6xZ8ExabCu9nGTmgv7fzgCVliuriBdBXzk2iiGpkwy8HpdkCOE-R7ugUJNM9O0-egUUwtx09TwdZWjFYTd3-nPsnx1sA2PnqhubA-L6VDPXSn2vdYxOHe7Wi7e28WeuhqR7Am4HuZxwoJ-mncWPjlB4T4xbqYtyI1OIekSFAnBeCEUz4yOcH-AKJsE-FIHI5CK3xBL1E1Tw3CTlc4jKJNty~~yuxu2xumKMurped9JFEq2dj0MfPEQNj13wKIQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-5</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dsr047/2/dsr047-dsr047supp_fig3.pdf?Expires=2147483647&amp;Signature=hUPRRK8H8x0bdJfuGzv6RUl6dTyRg9M0f6Fq6UNPynT56yLGyelSRVL3RfTPeRl~qWiGfFo3mAprZPVxzKj~XaSe1cZjC8BYBgL93KhlaBwxLxfNtdnea2HWVw6gEsaoubJ6vhH6LDod53kCNO-LC0wNnxFSRZn2ITNDbYx2q3I8VzVwNNG~pimaQl6w0Z55BjjwQx34EMbIM5Iiqh3wZ1Dk8p4-gi5sVaU6qHcTjvCYMiw2f2ubnXZbAP3j80Hp-mf26GbQd6qlkMBlhrGhyA4tSpUB9roEkjW5wkhqkp-WPsR7E9jfaCw6gn1YGMsEsllHviPgNCETwLOIoSPF2g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-4</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dsr047/2/dsr047-dsr047supp_fig2.pdf?Expires=2147483647&amp;Signature=j-M6oK4cCLlg8qq7dKzEerRLfMEEOx0bixMWT5hoBXal2VhZXex6j1R8AwpWQxwbxSYlnLrbt7AOuWZUQAaL7b0XSW2eqDRlny7EduIyLs1HTVJD3Zm9SlBjdMxEZcFP4WgBd~ukkrmnx92f3s0rsBfSC9pwk2Zl-2WYDb9UY5GrxjWThTN8ADLiaCmtUZ5RC5p2r52R8ub1-S-u9hcPfC3n0n453-UtwKJeKQYLeSXvaTRybML1Pj0k4h63G8WjjYu2Z-GgW-ymIHSmU0oLbFNT-GmfB1HguUtIyl0NeOrQ2r-8k5mS44fVH2bLmrdz5ITJCK08xHwXvnVe5bZIGg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dsr047/2/dsr047-dsr047supp_fig1.pdf?Expires=2147483647&amp;Signature=Hqw9g5FJiMRhtCHiOTZqDwo2tqLwmWoTRp2qXv2FwEZnw0Pof~ru2i8hFGg17egPmz8SuDxtLMd2lz3eLzdlrTEOwFk3WszRvO0Mh9xtiHL-5hk68tSED8hEltfd5p63H2zAyC-C6QzI7oj0O35-3icMlsRsoNvBRSe46VgC1-DhQR4JxJEiQ4Nwe0f2oVjWWCxoQkKgYYGER~~u0czxR~CDMHCxAJuW39Cr1eMdUKTnyvvUWezHpMYhGM3wWJj5iMpVCV9STv9Bggt6eQrGGj77Yfn4Hk7rtDYkJQTdnyij3PkYir1zCdabuwIX0to2tsjqnjA8Zbd~a5NNePyhjg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dsr047/2/dsr047-dsr047supp.doc?Expires=2147483647&amp;Signature=IHufnORBKkvOARAZdRh7uL~ihVVW~KP7muRvAixIakNcaNwK11MabdaxR29Yo2TRXV61WZU1tas6qVSxrwSU3A9T54BOr5p8rcQ57PjxkiBHXKUWZvQVg6wnalg2RrjXh9hPQsNCelFn-3AkORJ5dc8rDWr~e-rQnaIvpcBGaF5pcMryKXwzAT~1QGA0Sf4Qqv5UYwYR-5mpl~vM475asUyydinfxdGHtNJz5p4tzPlstBPY9rq2bwt-ni9okomqtG~QCirFszmOrFN~niNEEJpVNou7noUBVPZROrBW0NR5dowHkdQW51C4ZNCzUtWp58KGUaff2mtojG1Px1g7nw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - doc file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">381</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000303290100004&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">24</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsr047"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="419384"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000303290100004&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f19%2f2%2f143%2f419384">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000303290100004&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (24)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F19%2F2%2F143%2F419384" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/419384'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsr047");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f19%2f2%2f143%2f419384">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

