

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/20/1/79/383877","@type":"ScholarlyArticle","name":"Construction of an Integrated High Density Simple Sequence Repeat Linkage Map in Cultivated Strawberry ( Fragaria  ×  ananassa ) and its Applicability","datePublished":"2012-12-17","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/20/1","@type":"PublicationIssue","issueNumber":"1","datePublished":"2013-02-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dss035","keywords":["<italic>Fragaria</italic> × <italic>ananassa</italic>","SSR marker","integrated linkage map","comparative mapping"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/20/1/79/383877","author":[{"name":"Isobe, Sachiko N.","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Hirakawa, Hideki","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Sato, Shusei","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Maeda, Fumi","affiliation":"2  Crop Breeding Institute  , Chiba Prefectural Agriculture and Forestry Research Center ,  Mizoguchi 1-7-8, Chousei, Daizenno-cyou 808, Midori, Chiba 299-4335 ,  Japan","@type":"Person"},{"name":"Ishikawa, Masami","affiliation":"2  Crop Breeding Institute  , Chiba Prefectural Agriculture and Forestry Research Center ,  Mizoguchi 1-7-8, Chousei, Daizenno-cyou 808, Midori, Chiba 299-4335 ,  Japan","@type":"Person"},{"name":"Mori, Toshiki","affiliation":"3  Mie Prefecture Agricultural Research Institute ,  Ureshino Kawakitamachi 530, Matsuzaka, Mie 515-2316 ,  Japan","@type":"Person"},{"name":"Yamamoto, Yuko","affiliation":"3  Mie Prefecture Agricultural Research Institute ,  Ureshino Kawakitamachi 530, Matsuzaka, Mie 515-2316 ,  Japan","@type":"Person"},{"name":"Shirasawa, Kenta","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Kimura, Mitsuhiro","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Fukami, Masanobu","affiliation":"2  Crop Breeding Institute  , Chiba Prefectural Agriculture and Forestry Research Center ,  Mizoguchi 1-7-8, Chousei, Daizenno-cyou 808, Midori, Chiba 299-4335 ,  Japan","@type":"Person"},{"name":"Hashizume, Fujio","affiliation":"3  Mie Prefecture Agricultural Research Institute ,  Ureshino Kawakitamachi 530, Matsuzaka, Mie 515-2316 ,  Japan","@type":"Person"},{"name":"Tsuji, Tomoko","affiliation":"3  Mie Prefecture Agricultural Research Institute ,  Ureshino Kawakitamachi 530, Matsuzaka, Mie 515-2316 ,  Japan","@type":"Person"},{"name":"Sasamoto, Shigemi","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Kato, Midori","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Nanri, Keiko","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Tsuruoka, Hisano","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Minami, Chiharu","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Takahashi, Chika","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Wada, Tsuyuko","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Ono, Akiko","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Kawashima, Kumiko","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Nakazaki, Naomi","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Kishida, Yoshie","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Kohara, Mitsuyo","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Nakayama, Shinobu","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Yamada, Manabu","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Fujishiro, Tsunakazu","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Watanabe, Akiko","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Tabata, Satoshi","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"}],"description":"Abstract.  The cultivated strawberry (Fragaria× ananassa) is an octoploid (2n = 8x = 56) of the Rosaceae family whose genomic architecture is still controversia","pageStart":"79","pageEnd":"92","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/1/1/m_cover.gif?Expires=2147483647&Signature=K1v8cGQnstZYdECB9GA3oc5Ko3ohhQ9lF83XAKOVlOLlyAqR0JX0Vn18cJ3Fh6Ni4coRVii1nVVwFuzd9baa8EpiNPdDbUxg77BFyv3S5jO4sVroBj5PfHgZd5D14mPzL5KK6FHcweE5FD2RLiGEj~ct-3TIgAM5vP7r7dob4djMEW-dmZ2mnE0M4g6gvpwd4meZtXx6YM6PJ-qadpYPQyOJp7E7APzkRfsaUptwFx9azq6qh9BweylqnGS-k05la9PhL7T57M1047efztU-bcJDc0lmA7wiGh4dEYA5ldVPoM0OpeZ~9-tJ2gANZpruMQldUUZX51aSqOwvf70tSQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Construction of an Integrated High Density Simple Sequence Repeat Linkage Map in Cultivated Strawberry ( Fragaria  ×  ananassa ) and its Applicability","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/1/1/m_cover.gif?Expires=2147483647&Signature=K1v8cGQnstZYdECB9GA3oc5Ko3ohhQ9lF83XAKOVlOLlyAqR0JX0Vn18cJ3Fh6Ni4coRVii1nVVwFuzd9baa8EpiNPdDbUxg77BFyv3S5jO4sVroBj5PfHgZd5D14mPzL5KK6FHcweE5FD2RLiGEj~ct-3TIgAM5vP7r7dob4djMEW-dmZ2mnE0M4g6gvpwd4meZtXx6YM6PJ-qadpYPQyOJp7E7APzkRfsaUptwFx9azq6qh9BweylqnGS-k05la9PhL7T57M1047efztU-bcJDc0lmA7wiGh4dEYA5ldVPoM0OpeZ~9-tJ2gANZpruMQldUUZX51aSqOwvf70tSQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Construction of an Integrated High Density Simple Sequence Repeat Linkage Map in Cultivated Strawberry ( Fragaria  &#215;  ananassa ) and its Applicability" />
    <meta property="og:description" content="Abstract.  The cultivated strawberry (Fragaria&#215; ananassa) is an octoploid (2n = 8x = 56) of the Rosaceae family whose genomic architecture is still controversia" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/20/1/79/383877" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2012-12-17" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Isobe, Sachiko N." /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Hirakawa, Hideki" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Sato, Shusei" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Maeda, Fumi" /><meta name="citation_author_institution" content="Crop Breeding Institute, Chiba Prefectural Agriculture and Forestry Research Center, Mizoguchi 1-7-8, Chousei, Daizenno-cyou 808, Midori, Chiba 299-4335, Japan" /><meta name="citation_author" content="Ishikawa, Masami" /><meta name="citation_author_institution" content="Crop Breeding Institute, Chiba Prefectural Agriculture and Forestry Research Center, Mizoguchi 1-7-8, Chousei, Daizenno-cyou 808, Midori, Chiba 299-4335, Japan" /><meta name="citation_author" content="Mori, Toshiki" /><meta name="citation_author_institution" content="Mie Prefecture Agricultural Research Institute, Ureshino Kawakitamachi 530, Matsuzaka, Mie 515-2316, Japan" /><meta name="citation_author" content="Yamamoto, Yuko" /><meta name="citation_author_institution" content="Mie Prefecture Agricultural Research Institute, Ureshino Kawakitamachi 530, Matsuzaka, Mie 515-2316, Japan" /><meta name="citation_author" content="Shirasawa, Kenta" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Kimura, Mitsuhiro" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Fukami, Masanobu" /><meta name="citation_author_institution" content="Crop Breeding Institute, Chiba Prefectural Agriculture and Forestry Research Center, Mizoguchi 1-7-8, Chousei, Daizenno-cyou 808, Midori, Chiba 299-4335, Japan" /><meta name="citation_author" content="Hashizume, Fujio" /><meta name="citation_author_institution" content="Mie Prefecture Agricultural Research Institute, Ureshino Kawakitamachi 530, Matsuzaka, Mie 515-2316, Japan" /><meta name="citation_author" content="Tsuji, Tomoko" /><meta name="citation_author_institution" content="Mie Prefecture Agricultural Research Institute, Ureshino Kawakitamachi 530, Matsuzaka, Mie 515-2316, Japan" /><meta name="citation_author" content="Sasamoto, Shigemi" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Kato, Midori" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Nanri, Keiko" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Tsuruoka, Hisano" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Minami, Chiharu" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Takahashi, Chika" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Wada, Tsuyuko" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Ono, Akiko" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Kawashima, Kumiko" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Nakazaki, Naomi" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Kishida, Yoshie" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Kohara, Mitsuyo" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Nakayama, Shinobu" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Yamada, Manabu" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Fujishiro, Tsunakazu" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Watanabe, Akiko" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Tabata, Satoshi" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_title" content="Construction of an Integrated High Density Simple Sequence Repeat Linkage Map in Cultivated Strawberry (Fragaria × ananassa) and its Applicability" /><meta name="citation_firstpage" content="79" /><meta name="citation_lastpage" content="92" /><meta name="citation_doi" content="10.1093/dnares/dss035" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="20" /><meta name="citation_issue" content="1" /><meta name="citation_publication_date" content="2013/02/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="FAOSTAT http://faostat3.fao.org. " /><meta name="citation_reference" content="UPOV http://www.upov.int " /><meta name="citation_reference" content="citation_publisher=Holt, Rinehart and Winston, New York, USA; The Strawberry. History, Breeding and Physiology;  citation_year=1966; " /><meta name="citation_reference" content="citation_title=Cross ability and phylogenetic relationships in the main European species of Fragaria; Natl. Acad. Sci. USSR;  citation_year=1946;  citation_volume=52;  citation_pages=545-7; " /><meta name="citation_reference" content="citation_title=Origin of the Fragaria polyploids. I. Cytological evidence; Am. J. Bot.;  citation_year=1967;  citation_volume=54;  citation_pages=221-8; " /><meta name="citation_reference" content="citation_title=Cytogenetics and evolution in American Fragaria; Hortic. Sci.;  citation_year=1990;  citation_volume=25;  citation_pages=879-81; " /><meta name="citation_reference" content="citation_title=CAPS markers improved by cluster-specific amplification for identification of octoploid strawberry (Fragaria ×ananassa Duch.) cultivars, and their disomic inheritance; citation_text=1410–1418; Theor. Appl. Genet.;  citation_year=2005;  citation_volume=110; " /><meta name="citation_reference" content="citation_title=Comparative genetic mapping between octoploid and diploid Fragaria species reveals a high level of colinearity between their genomes and the essentially disomic behavior of the cultivated octoploid strawberry; Genetics;  citation_year=2008;  citation_volume=179;  citation_pages=2045-60; " /><meta name="citation_reference" content="citation_title=A genetic linkage map of the cultivated strawberry (Fragaria times ananassa) and its comparison to the diploid Fragaria reference map; Mol. Breed.;  citation_year=2009;  citation_volume=24;  citation_pages=293-303; " /><meta name="citation_reference" content="citation_title=A microsatellite linkage map for the cultivated strawberry (Fragaria times ananassa) suggests extensive regions of homozygosity in the genome that may have resulted from breeding and selection; Theor. Appl. Genet.;  citation_year=2012;  citation_volume=124;  citation_pages=1229-40; " /><meta name="citation_reference" content="citation_title=Genetic dissection of fruit quality traits in the octoploid cultivated strawberry highlights the role of homoeo-QTL in their control; Theor. Appl. Genet.;  citation_year=2012;  citation_volume=124;  citation_pages=1059-77; " /><meta name="citation_reference" content="citation_title=The species of Fragaria, their taxonomy and geographic distribution; Acta Hort.;  citation_year=1989;  citation_volume=265;  citation_pages=23-33; " /><meta name="citation_reference" content="citation_publisher=CABI Publishing, New York, USA; Strawberries;  citation_year=1999; " /><meta name="citation_reference" content="citation_title=Tracking the evolutionary history of polyploidy in Fragaria L. (strawberry): new insights from phylogenetic analyses of low-copy nuclear genes; Mol. Phylogenet. Evol.;  citation_year=2009;  citation_volume=51;  citation_pages=515-30; " /><meta name="citation_reference" content="citation_title=Isolation and characterization of polymorphic microsatellites in diploid strawberry (Fragaria vesca L.) for mapping, diversity studies and clone identification; Mol. Ecol. Notes;  citation_year=2003;  citation_volume=3;  citation_pages=171-3; " /><meta name="citation_reference" content="citation_title=The development of ISSR-derived SCAR markers around the SEASONAL FLOWERING LOCUS (SFL) in Fragaria vesca; Theor. Appl. Genet.;  citation_year=2004;  citation_volume=109;  citation_pages=571-9; " /><meta name="citation_reference" content="citation_title=Isolation and characterization of microsatellite loci in Fragaria; Mol. Ecol. Notes;  citation_year=2004;  citation_volume=4;  citation_pages=366-8; " /><meta name="citation_reference" content="citation_title=Development of microsatellite markers in Fragaria, their use in genetic diversity analysis, and their potential for genetic linkage mapping; Genome;  citation_year=2004;  citation_volume=47;  citation_pages=429-38; " /><meta name="citation_reference" content="citation_title=A new set of polymorphic simple sequence repeat (SSR) markers from a wild strawberry (Fragaria vesca) are transferable to other diploid Fragaria species and to Fragaria times ananassa; Mol. Ecol. Notes;  citation_year=2006;  citation_volume=6;  citation_pages=197-200; " /><meta name="citation_reference" content="citation_title=A genetic linkage map of microsatellite, gene-specific and morphological markers in diploid Fragaria; Theor. Appl. Genet.;  citation_year=2004;  citation_volume=7;  citation_pages=1385-91; " /><meta name="citation_reference" content="citation_title=An enhanced microsatellite map of diploid Fragaria; Theor. Appl. Genet.;  citation_year=2006;  citation_volume=112;  citation_pages=1349-59; " /><meta name="citation_reference" content="citation_title=The development and mapping of functional markers in Fragaria and their transferability and potential for mapping in other genera; Theor. Appl. Genet.;  citation_year=2007;  citation_volume=114;  citation_pages=373-84; " /><meta name="citation_reference" content="citation_title=The development of a bin mapping population and the selective mapping of 103 markers in the diploid Fragaria reference map; Genome;  citation_year=2008;  citation_volume=51;  citation_pages=120-7; " /><meta name="citation_reference" content="citation_title=The development and characterisation of a bacterial artificial chromosome library for Fragaria vesca; BMC Res. Notes;  citation_year=2009;  citation_volume=2;  citation_pages=188" /><meta name="citation_reference" content="citation_title=Gene content and distribution in the nuclear genome of Fragaria vesca; Plant Genome;  citation_year=2009;  citation_volume=2;  citation_pages=93-101; " /><meta name="citation_reference" content="citation_title=The genome of woodland strawberry (Fragaria vesca); Nat. Genet.;  citation_year=2011;  citation_volume=43;  citation_pages=109-16; " /><meta name="citation_reference" content="citation_title=Development and bin mapping of a Rosaceae conserved ortholog set (COS) of markers; BMC Genomics;  citation_year=2009;  citation_volume=10;  citation_pages=562" /><meta name="citation_reference" content="citation_title=Whole genome comparisons of Fragaria, Prunus and Malus reveal different modes of evolution between Rosaceous subfamilies; BMC Genomics;  citation_year=2012;  citation_volume=13;  citation_pages=129" /><meta name="citation_reference" content="citation_title=Characterization of mixed disomic and polysomic inheritance in the octoploid strawberry (Fragaria times ananassa) using AFLP mapping; Theor. Appl. Genet.;  citation_year=2003;  citation_volume=107;  citation_pages=619-28; " /><meta name="citation_reference" content="citation_title=Quantitative trait loci and underlying candidate genes controlling agronomical and fruit quality traits in octoploid strawberry (Fragaria times ananassa); Theor. Appl. Genet.;  citation_year=2011;  citation_volume=123;  citation_pages=755-78; " /><meta name="citation_reference" content="citation_title=Primer3 on the WWW for general users and for biologist programmers; citation_publisher=Humana Press, Totowa, NJ, USA; Bioinformatics Methods and Protocols: Methods in Molecular Biology;  citation_year=2000;  citation_pages=365-86; " /><meta name="citation_reference" content="citation_title=High variability and disomic segregation of microsatellites in the octoploid Fragaria virginiana Mill. (Rosaceae); Theor. Appl. Genet.;  citation_year=2003;  citation_volume=107;  citation_pages=1201-7; " /><meta name="citation_reference" content="citation_title=Development and characterization of polymorphic microsatellite markers from Fragaria viridis, a wild diploid strawberry; Mol. Ecol. Notes;  citation_year=2003;  citation_volume=3;  citation_pages=550-2; " /><meta name="citation_reference" content="citation_title=Strawberry GenBank derived and genomic simple sequence repeat (SSR) markers and their utility with strawberry, blackberry, and red and black raspberry; J. Am. Soc. Hortic. Sci.;  citation_year=2005;  citation_volume=130;  citation_pages=102-15; " /><meta name="citation_reference" content="citation_title=Microsatellite markers for Fragaria from ‘Strawberry Festival’ expressed sequence tags; Mol. Ecol. Notes;  citation_year=2006;  citation_volume=6;  citation_pages=473-76; " /><meta name="citation_reference" content="citation_title=EST-derived polymorphic microsatellites from cultivated strawberry (Fragaria times ananassa) are useful for diversity studies and varietal identification among Fragaria species; Mol. Ecol. Notes;  citation_year=2006;  citation_volume=6;  citation_pages=1195-7; " /><meta name="citation_reference" content="citation_title=Identification and characterization of simple sequence repeat (SSR) markers from Fragaria times ananassa expressed sequences; Mol. Ecol. Notes;  citation_year=2006;  citation_volume=6;  citation_pages=319-22; " /><meta name="citation_reference" content="citation_title=Genetic mapping of sex determination in a wild strawberry, Fragaria virginiana, reveals earliest form of sex chromosome; Heredity;  citation_year=2008;  citation_volume=101;  citation_pages=507-17; " /><meta name="citation_reference" content="citation_title=Simple sequence repeat marker development and mapping targeted to previously unmapped regions of the strawberry genome sequence; Plant Genome;  citation_year=2011;  citation_volume=4;  citation_pages=1-13; " /><meta name="citation_reference" content="citation_title=Using the miraEST assembler for reliable and automated mRNA transcript assembly and SNP detection in sequenced ESTs; Genome Res.;  citation_year=2004;  citation_volume=14;  citation_pages=1147-59; " /><meta name="citation_reference" content="citation_title=Comprehensive structural analysis of the genome of red clover (Trifolium pretense L.); DNA Res.;  citation_year=2005;  citation_volume=12;  citation_pages=301-64; " /><meta name="citation_reference" content="citation_publisher=Kyazma B.V.,, Wageningen, Netherlands; JoinMAP® 4, Software for the Calculation of Genetic Linkage Maps in Experimental Populations;  citation_year=2006; " /><meta name="citation_reference" content="citation_title=Colormapping; a non-mathematical procedure for genetic mapping; Acta Biol. Hung.;  citation_year=1998;  citation_volume=19;  citation_pages=125-42; " /><meta name="citation_reference" content="citation_title=Studies using DNA markers in F. times ananassa: genetic analysis, genome structure, and cultivar identification; J. Jpn. Soc. Hortic. Sci.;  citation_year=2011;  citation_volume=80;  citation_pages=231-43; " /><meta name="citation_reference" content="citation_title=GGT 2.0: versatile Software for visualization and analysis of genetic data; J. Hered.;  citation_year=2008;  citation_volume=99;  citation_pages=232-236; " /><meta name="citation_reference" content="citation_title=MEGA5: Molecular Evolutionary Genetics Analysis using likelihood, distance, and parsimony methods; Mol. Biol. Evol.;  citation_year=2011;  citation_volume=28;  citation_pages=2731-9; " /><meta name="citation_reference" content="citation_title=Karyotype analysis in octoploid and decaploid wild strawberries in Fragaria (Rosaceae); Cytologia;  citation_year=2010;  citation_volume=75;  citation_pages=277-88; " /><meta name="citation_reference" content="citation_title=Chromosome numbers and morphology in Japanese strawberry cultivars; Hortic. Res.;  citation_year=2008;  citation_volume=7;  citation_pages=195" /><meta name="citation_reference" content="citation_text=(in Japanese) ISBN 4-540-03284-4; citation_publisher=Nou-Bunkyou, Japan; History of Japanese Strawberry Varieties and their Adaptability, Yasai-Engei Dai-Hyakka;  citation_year=2004;  citation_pages=116-7; " /><meta name="citation_reference" content="citation_title=A reliable multiplexed microsatellite set for genotyping Fragaria and its use in a survey of 60 F. ×ananassa cultivars; Mol. Breeding;  citation_year=2008;  citation_volume=22;  citation_pages=649-61; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/20/1/79/1087588/dss035.pdf" /><meta name="description" content="Abstract.  The cultivated strawberry (Fragaria× ananassa) is an octoploid (2n = 8x = 56) of the Rosaceae family whose genomic architecture is still controversia" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/20/1/79/383877" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/20/1/79/383877" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/20/1/79/383877");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dss035';
    </script>
<title>
	Construction of an Integrated High Density Simple Sequence Repeat Linkage Map in Cultivated Strawberry (Fragaria × ananassa) and its Applicability | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./383877" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BZCXATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSXNvYmUsIFNhY2hpa28gTi4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZSwgMi02LTcgS2F6dXNhLWthbWF0YXJpLCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSGlyYWthd2EsIEhpZGVraSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCAyLTYtNyBLYXp1c2Eta2FtYXRhcmksIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTYXRvLCBTaHVzZWkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZSwgMi02LTcgS2F6dXNhLWthbWF0YXJpLCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTWFlZGEsIEZ1bWkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJDcm9wIEJyZWVkaW5nIEluc3RpdHV0ZSwgQ2hpYmEgUHJlZmVjdHVyYWwgQWdyaWN1bHR1cmUgYW5kIEZvcmVzdHJ5IFJlc2VhcmNoIENlbnRlciwgTWl6b2d1Y2hpIDEtNy04LCBDaG91c2VpLCBEYWl6ZW5uby1jeW91IDgwOCwgTWlkb3JpLCBDaGliYSAyOTktNDMzNSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJJc2hpa2F3YSwgTWFzYW1pIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQ3JvcCBCcmVlZGluZyBJbnN0aXR1dGUsIENoaWJhIFByZWZlY3R1cmFsIEFncmljdWx0dXJlIGFuZCBGb3Jlc3RyeSBSZXNlYXJjaCBDZW50ZXIsIE1pem9ndWNoaSAxLTctOCwgQ2hvdXNlaSwgRGFpemVubm8tY3lvdSA4MDgsIE1pZG9yaSwgQ2hpYmEgMjk5LTQzMzUsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTW9yaSwgVG9zaGlraSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik1pZSBQcmVmZWN0dXJlIEFncmljdWx0dXJhbCBSZXNlYXJjaCBJbnN0aXR1dGUsIFVyZXNoaW5vIEthd2FraXRhbWFjaGkgNTMwLCBNYXRzdXpha2EsIE1pZSA1MTUtMjMxNiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJZYW1hbW90bywgWXVrbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik1pZSBQcmVmZWN0dXJlIEFncmljdWx0dXJhbCBSZXNlYXJjaCBJbnN0aXR1dGUsIFVyZXNoaW5vIEthd2FraXRhbWFjaGkgNTMwLCBNYXRzdXpha2EsIE1pZSA1MTUtMjMxNiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTaGlyYXNhd2EsIEtlbnRhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2F6dXNhIEROQSBSZXNlYXJjaCBJbnN0aXR1dGUsIDItNi03IEthenVzYS1rYW1hdGFyaSwgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IktpbXVyYSwgTWl0c3VoaXJvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2F6dXNhIEROQSBSZXNlYXJjaCBJbnN0aXR1dGUsIDItNi03IEthenVzYS1rYW1hdGFyaSwgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkZ1a2FtaSwgTWFzYW5vYnUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJDcm9wIEJyZWVkaW5nIEluc3RpdHV0ZSwgQ2hpYmEgUHJlZmVjdHVyYWwgQWdyaWN1bHR1cmUgYW5kIEZvcmVzdHJ5IFJlc2VhcmNoIENlbnRlciwgTWl6b2d1Y2hpIDEtNy04LCBDaG91c2VpLCBEYWl6ZW5uby1jeW91IDgwOCwgTWlkb3JpLCBDaGliYSAyOTktNDMzNSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJIYXNoaXp1bWUsIEZ1amlvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTWllIFByZWZlY3R1cmUgQWdyaWN1bHR1cmFsIFJlc2VhcmNoIEluc3RpdHV0ZSwgVXJlc2hpbm8gS2F3YWtpdGFtYWNoaSA1MzAsIE1hdHN1emFrYSwgTWllIDUxNS0yMzE2LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlRzdWppLCBUb21va28iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJNaWUgUHJlZmVjdHVyZSBBZ3JpY3VsdHVyYWwgUmVzZWFyY2ggSW5zdGl0dXRlLCBVcmVzaGlubyBLYXdha2l0YW1hY2hpIDUzMCwgTWF0c3V6YWthLCBNaWUgNTE1LTIzMTYsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2FzYW1vdG8sIFNoaWdlbWkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZSwgMi02LTcgS2F6dXNhLWthbWF0YXJpLCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS2F0bywgTWlkb3JpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2F6dXNhIEROQSBSZXNlYXJjaCBJbnN0aXR1dGUsIDItNi03IEthenVzYS1rYW1hdGFyaSwgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik5hbnJpLCBLZWlrbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCAyLTYtNyBLYXp1c2Eta2FtYXRhcmksIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJUc3VydW9rYSwgSGlzYW5vIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2F6dXNhIEROQSBSZXNlYXJjaCBJbnN0aXR1dGUsIDItNi03IEthenVzYS1rYW1hdGFyaSwgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik1pbmFtaSwgQ2hpaGFydSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCAyLTYtNyBLYXp1c2Eta2FtYXRhcmksIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJUYWthaGFzaGksIENoaWthIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2F6dXNhIEROQSBSZXNlYXJjaCBJbnN0aXR1dGUsIDItNi03IEthenVzYS1rYW1hdGFyaSwgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IldhZGEsIFRzdXl1a28iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZSwgMi02LTcgS2F6dXNhLWthbWF0YXJpLCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iT25vLCBBa2lrbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCAyLTYtNyBLYXp1c2Eta2FtYXRhcmksIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLYXdhc2hpbWEsIEt1bWlrbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCAyLTYtNyBLYXp1c2Eta2FtYXRhcmksIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJOYWthemFraSwgTmFvbWkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZSwgMi02LTcgS2F6dXNhLWthbWF0YXJpLCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS2lzaGlkYSwgWW9zaGllIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2F6dXNhIEROQSBSZXNlYXJjaCBJbnN0aXR1dGUsIDItNi03IEthenVzYS1rYW1hdGFyaSwgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IktvaGFyYSwgTWl0c3V5byIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCAyLTYtNyBLYXp1c2Eta2FtYXRhcmksIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJOYWtheWFtYSwgU2hpbm9idSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCAyLTYtNyBLYXp1c2Eta2FtYXRhcmksIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJZYW1hZGEsIE1hbmFidSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCAyLTYtNyBLYXp1c2Eta2FtYXRhcmksIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJGdWppc2hpcm8sIFRzdW5ha2F6dSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCAyLTYtNyBLYXp1c2Eta2FtYXRhcmksIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJXYXRhbmFiZSwgQWtpa28iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZSwgMi02LTcgS2F6dXNhLWthbWF0YXJpLCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iVGFiYXRhLCBTYXRvc2hpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2F6dXNhIEROQSBSZXNlYXJjaCBJbnN0aXR1dGUsIDItNi03IEthenVzYS1rYW1hdGFyaSwgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iQ29uc3RydWN0aW9uIG9mIGFuIEludGVncmF0ZWQgSGlnaCBEZW5zaXR5IFNpbXBsZSBTZXF1ZW5jZSBSZXBlYXQgTGlua2FnZSBNYXAgaW4gQ3VsdGl2YXRlZCBTdHJhd2JlcnJ5IChGcmFnYXJpYSDDlyBhbmFuYXNzYSkgYW5kIGl0cyBBcHBsaWNhYmlsaXR5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iNzkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjkyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHNzMDM1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjIwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSIxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTMvMDIvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJGQU9TVEFUIGh0dHA6Ly9mYW9zdGF0My5mYW8ub3JnLiAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJVUE9WIGh0dHA6Ly93d3cudXBvdi5pbnQgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fcHVibGlzaGVyPUhvbHQsIFJpbmVoYXJ0IGFuZCBXaW5zdG9uLCBOZXcgWW9yaywgVVNBOyBUaGUgU3RyYXdiZXJyeS4gSGlzdG9yeSwgQnJlZWRpbmcgYW5kIFBoeXNpb2xvZ3k7ICBjaXRhdGlvbl95ZWFyPTE5NjY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNyb3NzIGFiaWxpdHkgYW5kIHBoeWxvZ2VuZXRpYyByZWxhdGlvbnNoaXBzIGluIHRoZSBtYWluIEV1cm9wZWFuIHNwZWNpZXMgb2YgRnJhZ2FyaWE7IE5hdGwuIEFjYWQuIFNjaS4gVVNTUjsgIGNpdGF0aW9uX3llYXI9MTk0NjsgIGNpdGF0aW9uX3ZvbHVtZT01MjsgIGNpdGF0aW9uX3BhZ2VzPTU0NS03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1PcmlnaW4gb2YgdGhlIEZyYWdhcmlhIHBvbHlwbG9pZHMuIEkuIEN5dG9sb2dpY2FsIGV2aWRlbmNlOyBBbS4gSi4gQm90LjsgIGNpdGF0aW9uX3llYXI9MTk2NzsgIGNpdGF0aW9uX3ZvbHVtZT01NDsgIGNpdGF0aW9uX3BhZ2VzPTIyMS04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DeXRvZ2VuZXRpY3MgYW5kIGV2b2x1dGlvbiBpbiBBbWVyaWNhbiBGcmFnYXJpYTsgSG9ydGljLiBTY2kuOyAgY2l0YXRpb25feWVhcj0xOTkwOyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9ODc5LTgxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DQVBTIG1hcmtlcnMgaW1wcm92ZWQgYnkgY2x1c3Rlci1zcGVjaWZpYyBhbXBsaWZpY2F0aW9uIGZvciBpZGVudGlmaWNhdGlvbiBvZiBvY3RvcGxvaWQgc3RyYXdiZXJyeSAoRnJhZ2FyaWEgw5dhbmFuYXNzYSBEdWNoLikgY3VsdGl2YXJzLCBhbmQgdGhlaXIgZGlzb21pYyBpbmhlcml0YW5jZTsgY2l0YXRpb25fdGV4dD0xNDEw4oCTMTQxODsgVGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBhcmF0aXZlIGdlbmV0aWMgbWFwcGluZyBiZXR3ZWVuIG9jdG9wbG9pZCBhbmQgZGlwbG9pZCBGcmFnYXJpYSBzcGVjaWVzIHJldmVhbHMgYSBoaWdoIGxldmVsIG9mIGNvbGluZWFyaXR5IGJldHdlZW4gdGhlaXIgZ2Vub21lcyBhbmQgdGhlIGVzc2VudGlhbGx5IGRpc29taWMgYmVoYXZpb3Igb2YgdGhlIGN1bHRpdmF0ZWQgb2N0b3Bsb2lkIHN0cmF3YmVycnk7IEdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTE3OTsgIGNpdGF0aW9uX3BhZ2VzPTIwNDUtNjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgZ2VuZXRpYyBsaW5rYWdlIG1hcCBvZiB0aGUgY3VsdGl2YXRlZCBzdHJhd2JlcnJ5IChGcmFnYXJpYSB0aW1lcyBhbmFuYXNzYSkgYW5kIGl0cyBjb21wYXJpc29uIHRvIHRoZSBkaXBsb2lkIEZyYWdhcmlhIHJlZmVyZW5jZSBtYXA7IE1vbC4gQnJlZWQuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTI0OyAgY2l0YXRpb25fcGFnZXM9MjkzLTMwMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBtaWNyb3NhdGVsbGl0ZSBsaW5rYWdlIG1hcCBmb3IgdGhlIGN1bHRpdmF0ZWQgc3RyYXdiZXJyeSAoRnJhZ2FyaWEgdGltZXMgYW5hbmFzc2EpIHN1Z2dlc3RzIGV4dGVuc2l2ZSByZWdpb25zIG9mIGhvbW96eWdvc2l0eSBpbiB0aGUgZ2Vub21lIHRoYXQgbWF5IGhhdmUgcmVzdWx0ZWQgZnJvbSBicmVlZGluZyBhbmQgc2VsZWN0aW9uOyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTEyNDsgIGNpdGF0aW9uX3BhZ2VzPTEyMjktNDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmV0aWMgZGlzc2VjdGlvbiBvZiBmcnVpdCBxdWFsaXR5IHRyYWl0cyBpbiB0aGUgb2N0b3Bsb2lkIGN1bHRpdmF0ZWQgc3RyYXdiZXJyeSBoaWdobGlnaHRzIHRoZSByb2xlIG9mIGhvbW9lby1RVEwgaW4gdGhlaXIgY29udHJvbDsgVGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMjQ7ICBjaXRhdGlvbl9wYWdlcz0xMDU5LTc3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgc3BlY2llcyBvZiBGcmFnYXJpYSwgdGhlaXIgdGF4b25vbXkgYW5kIGdlb2dyYXBoaWMgZGlzdHJpYnV0aW9uOyBBY3RhIEhvcnQuOyAgY2l0YXRpb25feWVhcj0xOTg5OyAgY2l0YXRpb25fdm9sdW1lPTI2NTsgIGNpdGF0aW9uX3BhZ2VzPTIzLTMzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9wdWJsaXNoZXI9Q0FCSSBQdWJsaXNoaW5nLCBOZXcgWW9yaywgVVNBOyBTdHJhd2JlcnJpZXM7ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYWNraW5nIHRoZSBldm9sdXRpb25hcnkgaGlzdG9yeSBvZiBwb2x5cGxvaWR5IGluIEZyYWdhcmlhIEwuIChzdHJhd2JlcnJ5KTogbmV3IGluc2lnaHRzIGZyb20gcGh5bG9nZW5ldGljIGFuYWx5c2VzIG9mIGxvdy1jb3B5IG51Y2xlYXIgZ2VuZXM7IE1vbC4gUGh5bG9nZW5ldC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9NTE7ICBjaXRhdGlvbl9wYWdlcz01MTUtMzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlzb2xhdGlvbiBhbmQgY2hhcmFjdGVyaXphdGlvbiBvZiBwb2x5bW9ycGhpYyBtaWNyb3NhdGVsbGl0ZXMgaW4gZGlwbG9pZCBzdHJhd2JlcnJ5IChGcmFnYXJpYSB2ZXNjYSBMLikgZm9yIG1hcHBpbmcsIGRpdmVyc2l0eSBzdHVkaWVzIGFuZCBjbG9uZSBpZGVudGlmaWNhdGlvbjsgTW9sLiBFY29sLiBOb3RlczsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0zOyAgY2l0YXRpb25fcGFnZXM9MTcxLTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBkZXZlbG9wbWVudCBvZiBJU1NSLWRlcml2ZWQgU0NBUiBtYXJrZXJzIGFyb3VuZCB0aGUgU0VBU09OQUwgRkxPV0VSSU5HIExPQ1VTIChTRkwpIGluIEZyYWdhcmlhIHZlc2NhOyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTEwOTsgIGNpdGF0aW9uX3BhZ2VzPTU3MS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Jc29sYXRpb24gYW5kIGNoYXJhY3Rlcml6YXRpb24gb2YgbWljcm9zYXRlbGxpdGUgbG9jaSBpbiBGcmFnYXJpYTsgTW9sLiBFY29sLiBOb3RlczsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9MzY2LTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldmVsb3BtZW50IG9mIG1pY3Jvc2F0ZWxsaXRlIG1hcmtlcnMgaW4gRnJhZ2FyaWEsIHRoZWlyIHVzZSBpbiBnZW5ldGljIGRpdmVyc2l0eSBhbmFseXNpcywgYW5kIHRoZWlyIHBvdGVudGlhbCBmb3IgZ2VuZXRpYyBsaW5rYWdlIG1hcHBpbmc7IEdlbm9tZTsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT00NzsgIGNpdGF0aW9uX3BhZ2VzPTQyOS0zODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBuZXcgc2V0IG9mIHBvbHltb3JwaGljIHNpbXBsZSBzZXF1ZW5jZSByZXBlYXQgKFNTUikgbWFya2VycyBmcm9tIGEgd2lsZCBzdHJhd2JlcnJ5IChGcmFnYXJpYSB2ZXNjYSkgYXJlIHRyYW5zZmVyYWJsZSB0byBvdGhlciBkaXBsb2lkIEZyYWdhcmlhIHNwZWNpZXMgYW5kIHRvIEZyYWdhcmlhIHRpbWVzIGFuYW5hc3NhOyBNb2wuIEVjb2wuIE5vdGVzOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz0xOTctMjAwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGdlbmV0aWMgbGlua2FnZSBtYXAgb2YgbWljcm9zYXRlbGxpdGUsIGdlbmUtc3BlY2lmaWMgYW5kIG1vcnBob2xvZ2ljYWwgbWFya2VycyBpbiBkaXBsb2lkIEZyYWdhcmlhOyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz0xMzg1LTkxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBlbmhhbmNlZCBtaWNyb3NhdGVsbGl0ZSBtYXAgb2YgZGlwbG9pZCBGcmFnYXJpYTsgVGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0xMTI7ICBjaXRhdGlvbl9wYWdlcz0xMzQ5LTU5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZGV2ZWxvcG1lbnQgYW5kIG1hcHBpbmcgb2YgZnVuY3Rpb25hbCBtYXJrZXJzIGluIEZyYWdhcmlhIGFuZCB0aGVpciB0cmFuc2ZlcmFiaWxpdHkgYW5kIHBvdGVudGlhbCBmb3IgbWFwcGluZyBpbiBvdGhlciBnZW5lcmE7IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTE0OyAgY2l0YXRpb25fcGFnZXM9MzczLTg0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZGV2ZWxvcG1lbnQgb2YgYSBiaW4gbWFwcGluZyBwb3B1bGF0aW9uIGFuZCB0aGUgc2VsZWN0aXZlIG1hcHBpbmcgb2YgMTAzIG1hcmtlcnMgaW4gdGhlIGRpcGxvaWQgRnJhZ2FyaWEgcmVmZXJlbmNlIG1hcDsgR2Vub21lOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTUxOyAgY2l0YXRpb25fcGFnZXM9MTIwLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBkZXZlbG9wbWVudCBhbmQgY2hhcmFjdGVyaXNhdGlvbiBvZiBhIGJhY3RlcmlhbCBhcnRpZmljaWFsIGNocm9tb3NvbWUgbGlicmFyeSBmb3IgRnJhZ2FyaWEgdmVzY2E7IEJNQyBSZXMuIE5vdGVzOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTI7ICBjaXRhdGlvbl9wYWdlcz0xODgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5lIGNvbnRlbnQgYW5kIGRpc3RyaWJ1dGlvbiBpbiB0aGUgbnVjbGVhciBnZW5vbWUgb2YgRnJhZ2FyaWEgdmVzY2E7IFBsYW50IEdlbm9tZTsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0yOyAgY2l0YXRpb25fcGFnZXM9OTMtMTAxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZ2Vub21lIG9mIHdvb2RsYW5kIHN0cmF3YmVycnkgKEZyYWdhcmlhIHZlc2NhKTsgTmF0LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NDM7ICBjaXRhdGlvbl9wYWdlcz0xMDktMTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldmVsb3BtZW50IGFuZCBiaW4gbWFwcGluZyBvZiBhIFJvc2FjZWFlIGNvbnNlcnZlZCBvcnRob2xvZyBzZXQgKENPUykgb2YgbWFya2VyczsgQk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9NTYyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9V2hvbGUgZ2Vub21lIGNvbXBhcmlzb25zIG9mIEZyYWdhcmlhLCBQcnVudXMgYW5kIE1hbHVzIHJldmVhbCBkaWZmZXJlbnQgbW9kZXMgb2YgZXZvbHV0aW9uIGJldHdlZW4gUm9zYWNlb3VzIHN1YmZhbWlsaWVzOyBCTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz0xMjkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaGFyYWN0ZXJpemF0aW9uIG9mIG1peGVkIGRpc29taWMgYW5kIHBvbHlzb21pYyBpbmhlcml0YW5jZSBpbiB0aGUgb2N0b3Bsb2lkIHN0cmF3YmVycnkgKEZyYWdhcmlhIHRpbWVzIGFuYW5hc3NhKSB1c2luZyBBRkxQIG1hcHBpbmc7IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MTA3OyAgY2l0YXRpb25fcGFnZXM9NjE5LTI4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1RdWFudGl0YXRpdmUgdHJhaXQgbG9jaSBhbmQgdW5kZXJseWluZyBjYW5kaWRhdGUgZ2VuZXMgY29udHJvbGxpbmcgYWdyb25vbWljYWwgYW5kIGZydWl0IHF1YWxpdHkgdHJhaXRzIGluIG9jdG9wbG9pZCBzdHJhd2JlcnJ5IChGcmFnYXJpYSB0aW1lcyBhbmFuYXNzYSk7IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTIzOyAgY2l0YXRpb25fcGFnZXM9NzU1LTc4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QcmltZXIzIG9uIHRoZSBXV1cgZm9yIGdlbmVyYWwgdXNlcnMgYW5kIGZvciBiaW9sb2dpc3QgcHJvZ3JhbW1lcnM7IGNpdGF0aW9uX3B1Ymxpc2hlcj1IdW1hbmEgUHJlc3MsIFRvdG93YSwgTkosIFVTQTsgQmlvaW5mb3JtYXRpY3MgTWV0aG9kcyBhbmQgUHJvdG9jb2xzOiBNZXRob2RzIGluIE1vbGVjdWxhciBCaW9sb2d5OyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fcGFnZXM9MzY1LTg2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IaWdoIHZhcmlhYmlsaXR5IGFuZCBkaXNvbWljIHNlZ3JlZ2F0aW9uIG9mIG1pY3Jvc2F0ZWxsaXRlcyBpbiB0aGUgb2N0b3Bsb2lkIEZyYWdhcmlhIHZpcmdpbmlhbmEgTWlsbC4gKFJvc2FjZWFlKTsgVGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMDc7ICBjaXRhdGlvbl9wYWdlcz0xMjAxLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldmVsb3BtZW50IGFuZCBjaGFyYWN0ZXJpemF0aW9uIG9mIHBvbHltb3JwaGljIG1pY3Jvc2F0ZWxsaXRlIG1hcmtlcnMgZnJvbSBGcmFnYXJpYSB2aXJpZGlzLCBhIHdpbGQgZGlwbG9pZCBzdHJhd2JlcnJ5OyBNb2wuIEVjb2wuIE5vdGVzOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz01NTAtMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3RyYXdiZXJyeSBHZW5CYW5rIGRlcml2ZWQgYW5kIGdlbm9taWMgc2ltcGxlIHNlcXVlbmNlIHJlcGVhdCAoU1NSKSBtYXJrZXJzIGFuZCB0aGVpciB1dGlsaXR5IHdpdGggc3RyYXdiZXJyeSwgYmxhY2tiZXJyeSwgYW5kIHJlZCBhbmQgYmxhY2sgcmFzcGJlcnJ5OyBKLiBBbS4gU29jLiBIb3J0aWMuIFNjaS47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTMwOyAgY2l0YXRpb25fcGFnZXM9MTAyLTE1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NaWNyb3NhdGVsbGl0ZSBtYXJrZXJzIGZvciBGcmFnYXJpYSBmcm9tIOKAmFN0cmF3YmVycnkgRmVzdGl2YWzigJkgZXhwcmVzc2VkIHNlcXVlbmNlIHRhZ3M7IE1vbC4gRWNvbC4gTm90ZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTQ3My03NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RVNULWRlcml2ZWQgcG9seW1vcnBoaWMgbWljcm9zYXRlbGxpdGVzIGZyb20gY3VsdGl2YXRlZCBzdHJhd2JlcnJ5IChGcmFnYXJpYSB0aW1lcyBhbmFuYXNzYSkgYXJlIHVzZWZ1bCBmb3IgZGl2ZXJzaXR5IHN0dWRpZXMgYW5kIHZhcmlldGFsIGlkZW50aWZpY2F0aW9uIGFtb25nIEZyYWdhcmlhIHNwZWNpZXM7IE1vbC4gRWNvbC4gTm90ZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTExOTUtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZmljYXRpb24gYW5kIGNoYXJhY3Rlcml6YXRpb24gb2Ygc2ltcGxlIHNlcXVlbmNlIHJlcGVhdCAoU1NSKSBtYXJrZXJzIGZyb20gRnJhZ2FyaWEgdGltZXMgYW5hbmFzc2EgZXhwcmVzc2VkIHNlcXVlbmNlczsgTW9sLiBFY29sLiBOb3RlczsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9MzE5LTIyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5ldGljIG1hcHBpbmcgb2Ygc2V4IGRldGVybWluYXRpb24gaW4gYSB3aWxkIHN0cmF3YmVycnksIEZyYWdhcmlhIHZpcmdpbmlhbmEsIHJldmVhbHMgZWFybGllc3QgZm9ybSBvZiBzZXggY2hyb21vc29tZTsgSGVyZWRpdHk7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTAxOyAgY2l0YXRpb25fcGFnZXM9NTA3LTE3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TaW1wbGUgc2VxdWVuY2UgcmVwZWF0IG1hcmtlciBkZXZlbG9wbWVudCBhbmQgbWFwcGluZyB0YXJnZXRlZCB0byBwcmV2aW91c2x5IHVubWFwcGVkIHJlZ2lvbnMgb2YgdGhlIHN0cmF3YmVycnkgZ2Vub21lIHNlcXVlbmNlOyBQbGFudCBHZW5vbWU7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTEtMTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVVzaW5nIHRoZSBtaXJhRVNUIGFzc2VtYmxlciBmb3IgcmVsaWFibGUgYW5kIGF1dG9tYXRlZCBtUk5BIHRyYW5zY3JpcHQgYXNzZW1ibHkgYW5kIFNOUCBkZXRlY3Rpb24gaW4gc2VxdWVuY2VkIEVTVHM7IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9MTE0Ny01OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcHJlaGVuc2l2ZSBzdHJ1Y3R1cmFsIGFuYWx5c2lzIG9mIHRoZSBnZW5vbWUgb2YgcmVkIGNsb3ZlciAoVHJpZm9saXVtIHByZXRlbnNlIEwuKTsgRE5BIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz0zMDEtNjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3B1Ymxpc2hlcj1LeWF6bWEgQi5WLiwsIFdhZ2VuaW5nZW4sIE5ldGhlcmxhbmRzOyBKb2luTUFQwq4gNCwgU29mdHdhcmUgZm9yIHRoZSBDYWxjdWxhdGlvbiBvZiBHZW5ldGljIExpbmthZ2UgTWFwcyBpbiBFeHBlcmltZW50YWwgUG9wdWxhdGlvbnM7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbG9ybWFwcGluZzsgYSBub24tbWF0aGVtYXRpY2FsIHByb2NlZHVyZSBmb3IgZ2VuZXRpYyBtYXBwaW5nOyBBY3RhIEJpb2wuIEh1bmcuOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTE5OyAgY2l0YXRpb25fcGFnZXM9MTI1LTQyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdHVkaWVzIHVzaW5nIEROQSBtYXJrZXJzIGluIEYuIHRpbWVzIGFuYW5hc3NhOiBnZW5ldGljIGFuYWx5c2lzLCBnZW5vbWUgc3RydWN0dXJlLCBhbmQgY3VsdGl2YXIgaWRlbnRpZmljYXRpb247IEouIEpwbi4gU29jLiBIb3J0aWMuIFNjaS47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9ODA7ICBjaXRhdGlvbl9wYWdlcz0yMzEtNDM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdHVCAyLjA6IHZlcnNhdGlsZSBTb2Z0d2FyZSBmb3IgdmlzdWFsaXphdGlvbiBhbmQgYW5hbHlzaXMgb2YgZ2VuZXRpYyBkYXRhOyBKLiBIZXJlZC47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9OTk7ICBjaXRhdGlvbl9wYWdlcz0yMzItMjM2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NRUdBNTogTW9sZWN1bGFyIEV2b2x1dGlvbmFyeSBHZW5ldGljcyBBbmFseXNpcyB1c2luZyBsaWtlbGlob29kLCBkaXN0YW5jZSwgYW5kIHBhcnNpbW9ueSBtZXRob2RzOyBNb2wuIEJpb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTI4OyAgY2l0YXRpb25fcGFnZXM9MjczMS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1LYXJ5b3R5cGUgYW5hbHlzaXMgaW4gb2N0b3Bsb2lkIGFuZCBkZWNhcGxvaWQgd2lsZCBzdHJhd2JlcnJpZXMgaW4gRnJhZ2FyaWEgKFJvc2FjZWFlKTsgQ3l0b2xvZ2lhOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTc1OyAgY2l0YXRpb25fcGFnZXM9Mjc3LTg4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaHJvbW9zb21lIG51bWJlcnMgYW5kIG1vcnBob2xvZ3kgaW4gSmFwYW5lc2Ugc3RyYXdiZXJyeSBjdWx0aXZhcnM7IEhvcnRpYy4gUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9MTk1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGV4dD0oaW4gSmFwYW5lc2UpIElTQk4gNC01NDAtMDMyODQtNDsgY2l0YXRpb25fcHVibGlzaGVyPU5vdS1CdW5reW91LCBKYXBhbjsgSGlzdG9yeSBvZiBKYXBhbmVzZSBTdHJhd2JlcnJ5IFZhcmlldGllcyBhbmQgdGhlaXIgQWRhcHRhYmlsaXR5LCBZYXNhaS1FbmdlaSBEYWktSHlha2thOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fcGFnZXM9MTE2LTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgcmVsaWFibGUgbXVsdGlwbGV4ZWQgbWljcm9zYXRlbGxpdGUgc2V0IGZvciBnZW5vdHlwaW5nIEZyYWdhcmlhIGFuZCBpdHMgdXNlIGluIGEgc3VydmV5IG9mIDYwIEYuIMOXYW5hbmFzc2EgY3VsdGl2YXJzOyBNb2wuIEJyZWVkaW5nOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTIyOyAgY2l0YXRpb25fcGFnZXM9NjQ5LTYxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMjAvMS83OS8xMDg3NTg4L2RzczAzNS5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgVGhlIGN1bHRpdmF0ZWQgc3RyYXdiZXJyeSAoRnJhZ2FyaWHDlyBhbmFuYXNzYSkgaXMgYW4gb2N0b3Bsb2lkICgybiA9IDh4ID0gNTYpIG9mIHRoZSBSb3NhY2VhZSBmYW1pbHkgd2hvc2UgZ2Vub21pYyBhcmNoaXRlY3R1cmUgaXMgc3RpbGwgY29udHJvdmVyc2lhIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFci8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj0zODM4NzcmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2RkWwNfzaX2K2u+wqZoyd8n7ThC1/g=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f1%2f79%2f383877" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f1%2f79%2f383877" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=14916" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/20/1"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/1/1/m_cover.gif?Expires=2147483647&amp;Signature=K1v8cGQnstZYdECB9GA3oc5Ko3ohhQ9lF83XAKOVlOLlyAqR0JX0Vn18cJ3Fh6Ni4coRVii1nVVwFuzd9baa8EpiNPdDbUxg77BFyv3S5jO4sVroBj5PfHgZd5D14mPzL5KK6FHcweE5FD2RLiGEj~ct-3TIgAM5vP7r7dob4djMEW-dmZ2mnE0M4g6gvpwd4meZtXx6YM6PJ-qadpYPQyOJp7E7APzkRfsaUptwFx9azq6qh9BweylqnGS-k05la9PhL7T57M1047efztU-bcJDc0lmA7wiGh4dEYA5ldVPoM0OpeZ~9-tJ2gANZpruMQldUUZX51aSqOwvf70tSQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/20/1">
                        <div class="volume trailing-comma">Volume 20</div>
                        <div class="issue">Issue 1</div>

            </a>

        </div>
        <div class="ii-pub-date">
February 2013        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="5643879">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5643879">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5643882">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5643882">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5643888">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5643888">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5643906">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5643906">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5643932">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5643932">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5643945">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5643945">Supplementary Data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5643947">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5643947">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5643949">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5643949">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5643951">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5643951">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/20/1/67/383475">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/20/1/93/384291">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Construction of an Integrated High Density Simple Sequence Repeat Linkage Map in Cultivated Strawberry (<em>Fragaria</em> × <em>ananassa</em>) and its Applicability
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                            <div class="meta-authors--limited">
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Sachiko N. Isobe</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81 438-52-3928. Fax. +81 438-52-3934. Email: <a href="mailto:sisobe@kazusa.or.jp" target="_blank">sisobe@kazusa.or.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Sachiko+N.+Isobe">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Isobe S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Isobe S&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Sachiko N. Isobe</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hideki Hirakawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hideki+Hirakawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hirakawa H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hirakawa H&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Hideki Hirakawa</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shusei Sato</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shusei+Sato">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sato S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sato S&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Shusei Sato</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Fumi Maeda</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Crop Breeding Institute</div><div class="insititution">, Chiba Prefectural Agriculture and Forestry Research Center</div>, <div class="addr-line">Mizoguchi 1-7-8, Chousei, Daizenno-cyou 808, Midori, Chiba 299-4335</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Fumi+Maeda">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Maeda F">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Maeda F&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Fumi Maeda</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Masami Ishikawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Crop Breeding Institute</div><div class="insititution">, Chiba Prefectural Agriculture and Forestry Research Center</div>, <div class="addr-line">Mizoguchi 1-7-8, Chousei, Daizenno-cyou 808, Midori, Chiba 299-4335</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Masami+Ishikawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ishikawa M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ishikawa M&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Masami Ishikawa</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Toshiki Mori</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="insititution">Mie Prefecture Agricultural Research Institute</div>, <div class="addr-line">Ureshino Kawakitamachi 530, Matsuzaka, Mie 515-2316</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Toshiki+Mori">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Mori T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Mori T&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Toshiki Mori</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yuko Yamamoto</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="insititution">Mie Prefecture Agricultural Research Institute</div>, <div class="addr-line">Ureshino Kawakitamachi 530, Matsuzaka, Mie 515-2316</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yuko+Yamamoto">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yamamoto Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yamamoto Y&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Yuko Yamamoto</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kenta Shirasawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kenta+Shirasawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shirasawa K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Shirasawa K&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Kenta Shirasawa</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Mitsuhiro Kimura</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Mitsuhiro+Kimura">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kimura M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kimura M&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Mitsuhiro Kimura</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Masanobu Fukami</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Crop Breeding Institute</div><div class="insititution">, Chiba Prefectural Agriculture and Forestry Research Center</div>, <div class="addr-line">Mizoguchi 1-7-8, Chousei, Daizenno-cyou 808, Midori, Chiba 299-4335</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Masanobu+Fukami">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Fukami M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Fukami M&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Masanobu Fukami</a>
                                    </span>
                                <a class="meta-authors--etal is-active" id="show-meta-authors" aria-controls="remaining-authors" href="#">
                                    <span>...</span> Show more</a>
                                <div id="remaining-authors" class="meta-authors--remaining" aria-hidden="true" aria-labeledby="show-meta-authors">
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Fujio Hashizume</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="insititution">Mie Prefecture Agricultural Research Institute</div>, <div class="addr-line">Ureshino Kawakitamachi 530, Matsuzaka, Mie 515-2316</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Fujio+Hashizume">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hashizume F">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hashizume F&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Fujio Hashizume</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tomoko Tsuji</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="insititution">Mie Prefecture Agricultural Research Institute</div>, <div class="addr-line">Ureshino Kawakitamachi 530, Matsuzaka, Mie 515-2316</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tomoko+Tsuji">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tsuji T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tsuji T&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Tomoko Tsuji</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shigemi Sasamoto</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shigemi+Sasamoto">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sasamoto S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sasamoto S&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Shigemi Sasamoto</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Midori Kato</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Midori+Kato">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kato M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kato M&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Midori Kato</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Keiko Nanri</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Keiko+Nanri">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nanri K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nanri K&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Keiko Nanri</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hisano Tsuruoka</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hisano+Tsuruoka">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tsuruoka H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tsuruoka H&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Hisano Tsuruoka</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Chiharu Minami</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Chiharu+Minami">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Minami C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Minami C&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Chiharu Minami</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Chika Takahashi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Chika+Takahashi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Takahashi C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Takahashi C&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Chika Takahashi</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tsuyuko Wada</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tsuyuko+Wada">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Wada T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Wada T&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Tsuyuko Wada</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Akiko Ono</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Akiko+Ono">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ono A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ono A&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Akiko Ono</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kumiko Kawashima</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kumiko+Kawashima">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kawashima K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kawashima K&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Kumiko Kawashima</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Naomi Nakazaki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Naomi+Nakazaki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nakazaki N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nakazaki N&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Naomi Nakazaki</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yoshie Kishida</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yoshie+Kishida">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kishida Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kishida Y&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Yoshie Kishida</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Mitsuyo Kohara</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Mitsuyo+Kohara">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kohara M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kohara M&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Mitsuyo Kohara</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shinobu Nakayama</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shinobu+Nakayama">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nakayama S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nakayama S&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Shinobu Nakayama</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Manabu Yamada</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Manabu+Yamada">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yamada M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yamada M&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Manabu Yamada</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tsunakazu Fujishiro</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tsunakazu+Fujishiro">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Fujishiro T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Fujishiro T&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Tsunakazu Fujishiro</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Akiko Watanabe</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Akiko+Watanabe">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Watanabe A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Watanabe A&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Akiko Watanabe</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Satoshi Tabata</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Satoshi+Tabata">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tabata S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tabata S&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Satoshi Tabata</a>
                                        </span>
                                </div>
                            </div>

                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 20, Issue 1, 1 February 2013, Pages 79–92, <a href='https://doi.org/10.1093/dnares/dss035'>https://doi.org/10.1093/dnares/dss035</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">17 December 2012</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">25 September 2012</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">21 November 2012</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="383877" href="/dnaresearch/article-pdf/20/1/79/1087588/dss035.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Sachiko N. Isobe, Hideki Hirakawa, Shusei Sato, Fumi Maeda, Masami Ishikawa, Toshiki Mori, Yuko Yamamoto, Kenta Shirasawa, Mitsuhiro Kimura, Masanobu Fukami, Fujio Hashizume, Tomoko Tsuji, Shigemi Sasamoto, Midori Kato, Keiko Nanri, Hisano Tsuruoka, Chiharu Minami, Chika Takahashi, Tsuyuko Wada, Akiko Ono, Kumiko Kawashima, Naomi Nakazaki, Yoshie Kishida, Mitsuyo Kohara, Shinobu Nakayama, Manabu Yamada, Tsunakazu Fujishiro, Akiko Watanabe, Satoshi Tabata;  Construction of an Integrated High Density Simple Sequence Repeat Linkage Map in Cultivated Strawberry (<em>Fragaria</em> × <em>ananassa</em>) and its Applicability, <em>DNA Research</em>, Volume 20, Issue 1, 1 February 2013, Pages 79–92, <a href='https://doi.org/10.1093/dnares/dss035'>https://doi.org/10.1093/dnares/dss035</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/383877?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/383877?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/383877?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/383877?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/383877?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/383877?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/383877?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Construction%20of%20an%20Integrated%20High%20Density%20Simple%20Sequence%20Repeat%20Linkage%20Map%20in%20Cultivated%20Strawberry%20(%26lt%3Bem%26gt%3BFragaria%26lt%3B%2Fem%26gt%3B%20%C3%97%20%26lt%3Bem%26gt%3Bananassa%26lt%3B%2Fem%26gt%3B)%20and%20its%20Applicability&amp;publicationDate=2012-12-17&amp;volumeNum=20&amp;issueNum=1&amp;author=Isobe%2C%20Sachiko%20N.%3B%20Hirakawa%2C%20Hideki&amp;startPage=79&amp;endPage=92&amp;contentId=10.1093%2Fdnares%2Fdss035&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f1%2f79%2f383877" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=5643879 id="5643879" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>The cultivated strawberry (<em>Fragaria</em>× <em>ananassa</em>) is an octoploid (2<em>n</em> = 8<em>x</em> = 56) of the Rosaceae family whose genomic architecture is still controversial. Several recent studies support the AAA′A′BBB′B′ model, but its complexity has hindered genetic and genomic analysis of this important crop. To overcome this difficulty and to assist genome-wide analysis of <em>F.</em> × <em>ananassa</em>, we constructed an integrated linkage map by organizing a total of 4474 of simple sequence repeat (SSR) markers collected from published <em>Fragaria</em> sequences, including 3746 SSR markers [<em>Fragaria vesca</em> expressed sequence tag (EST)-derived SSR markers] derived from <em>F. vesca</em> ESTs, 603 markers (<em>F.</em> × <em>ananassa</em> EST-derived SSR markers) from <em>F.</em> × <em>ananassa</em> ESTs, and 125 markers (<em>F.</em> × <em>ananassa</em> transcriptome-derived SSR markers) from <em>F.</em> × <em>ananassa</em> transcripts. Along with the previously published SSR markers, these markers were mapped onto five parent-specific linkage maps derived from three mapping populations, which were then assembled into an integrated linkage map. The constructed map consists of 1856 loci in 28 linkage groups (LGs) that total 2364.1 cM in length. Macrosynteny at the chromosome level was observed between the LGs of <em>F.</em> × <em>ananassa</em> and the genome of <em>F. vesca.</em> Variety distinction on 129 <em>F.</em> × <em>ananassa</em> lines was demonstrated using 45 selected SSR markers.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Fragaria × ananassa&quot;"><em>Fragaria</em> × <em>ananassa</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;SSR marker&quot;">SSR marker</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;integrated linkage map&quot;">integrated linkage map</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;comparative mapping&quot;">comparative mapping</a></div>                            <h2 scrollto-destination=5643882 id="5643882" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>The cultivated strawberry (<em>Fragaria</em> × <em>ananassa</em>) is one of the most popular and globally consumed fruit crops. It is cultivated in various regions throughout the world; in 2010, 39% of the cultivated strawberry crop was produced in North and South America, followed by Europe (33%), Asia (18%), and Africa (9%).<sup><a reveal-id="DSS035C1" data-open="DSS035C1" class="link link-ref link-reveal xref-bibr">1</a></sup> Because of the economic importance of this species, breeding programmes of cultivated strawberry are underway in many countries. To date, more than 3000 varieties bred in 41 countries have been registered in the International Union for the Protection and New Varieties of Plants variety database.<sup><a reveal-id="DSS035C2" data-open="DSS035C2" class="link link-ref link-reveal xref-bibr">2</a></sup> However, despite intensive use of <em>F.</em> × <em>ananassa</em> in the industry, the progress of genetic and genomic research of this crop has lagged behind that of many other economically important plant species, because of its complex genome structure.</p><p><em>Fragaria</em> × <em>ananassa</em> is an octoploid (2<em>n</em> = 8<em>x</em> = 56) species that originated from a natural hybridization between <em>Fragaria virginiana</em> and <em>Fragaria chiloensis</em>.<sup><a reveal-id="DSS035C3" data-open="DSS035C3" class="link link-ref link-reveal xref-bibr">3</a></sup> The genome composition of <em>F.</em> × <em>ananassa</em> was initially proposed as AABBBBCC<sup><a reveal-id="DSS035C4" data-open="DSS035C4" class="link link-ref link-reveal xref-bibr">4</a></sup> or AAA′A′BBBB,<sup><a reveal-id="DSS035C5" data-open="DSS035C5" class="link link-ref link-reveal xref-bibr">5</a></sup> based on results of cytological studies. Later, Bringhurst<sup><a reveal-id="DSS035C6" data-open="DSS035C6" class="link link-ref link-reveal xref-bibr">6</a></sup> proposed the AAA′A′BBB′B′ model in light of cytological and genetic evidence. Unlike the first two models, the last model suggests disomic inheritance in entire chromosomes. Although the genome composition of <em>F.</em> × <em>ananassa</em> has not yet been confirmed, the AAA′A′BBB′B′ model has been supported by recent molecular genetic studies.<sup><a reveal-id="DSS035C7  DSS035C11" data-open="DSS035C7  DSS035C11" class="link link-ref link-reveal xref-bibr">7–11</a></sup> The genus <em>Fragaria</em> belongs to the Rosaceae family. Unlike other genera in the Rosaceae family, <em>Fragaria</em> comprises a limited number of species (approximately 22).<sup><a reveal-id="DSS035C12" data-open="DSS035C12" class="link link-ref link-reveal xref-bibr">12</a></sup> Several species have been nominated as candidate diploid ancestors such as <em>Fragaria vesca</em>, <em>Fragaria iinumae</em>, and <em>Fragaria daltoniana</em>.<sup><a reveal-id="DSS035C13" data-open="DSS035C13" class="link link-ref link-reveal xref-bibr">13</a>,<a reveal-id="DSS035C14" data-open="DSS035C14" class="link link-ref link-reveal xref-bibr">14</a></sup> Of these candidates, <em>F. vesca</em> is considered to be the most likely diploid ancestor, and it serves as a model species of <em>F.</em> × <em>ananassa</em>.<sup><a reveal-id="DSS035C14" data-open="DSS035C14" class="link link-ref link-reveal xref-bibr">14</a></sup> Therefore, genomic and genetic studies in <em>F. vesca</em> have been performed prior to those of <em>F.</em> × <em>ananassa</em>.</p><p>In molecular genetics studies, several types of <em>F. vesca</em> sequence-derived DNA markers, such as simple sequence repeat (SSR) and sequence-characterized amplified region (SCAR) markers, have been developed since 2003<sup><a reveal-id="DSS035C15  DSS035C19" data-open="DSS035C15  DSS035C19" class="link link-ref link-reveal xref-bibr">15–19</a></sup> (in this article, ‘DNA marker’ is defined as ‘unique DNA sequence(s) identified by specific primers’). Linkage maps were then constructed with a population derived from an interspecific cross between <em>F. vesca</em> and <em>Fragaria nubicola</em>.<sup><a reveal-id="DSS035C20  DSS035C23" data-open="DSS035C20  DSS035C23" class="link link-ref link-reveal xref-bibr">20–23</a></sup> In genomic studies, bacterial artificial chromosome and fosmid libraries were constructed to investigate features of genome sequences of <em>F. vesca</em>.<sup><a reveal-id="DSS035C24" data-open="DSS035C24" class="link link-ref link-reveal xref-bibr">24</a>,<a reveal-id="DSS035C25" data-open="DSS035C25" class="link link-ref link-reveal xref-bibr">25</a></sup> Subsequently, a whole genome sequence for <em>F. vesca</em> was published in 2011, and the results have greatly contributed to advances in genomic and genetic analysis of the genus <em>Fragaria</em>.<sup><a reveal-id="DSS035C26" data-open="DSS035C26" class="link link-ref link-reveal xref-bibr">26</a></sup> In parallel, comparative genomics studies within the Rosaceae family have also been performed. Cabrera et al<em>.</em><sup><a reveal-id="DSS035C27" data-open="DSS035C27" class="link link-ref link-reveal xref-bibr">27</a></sup> developed a total of 1039 Rosaceae-conserved orthologous set (RosCOS) markers. Primer pairs of the RosCOS markers were designed on intron-flanking regions of orthologous genes commonly conserved among the genera <em>Malus</em>, <em>Fragaria</em>, and <em>Prunus</em> and mapped onto a <em>Prunus</em> reference linkage map. After the genome sequence in <em>F. vesca</em> was published,<sup><a reveal-id="DSS035C26" data-open="DSS035C26" class="link link-ref link-reveal xref-bibr">26</a></sup> whole genome comparisons between <em>Fragaria</em>, <em>Prunus</em>, and <em>Malus</em> were performed, and major orthologous regions were identified across the genus.<sup><a reveal-id="DSS035C28" data-open="DSS035C28" class="link link-ref link-reveal xref-bibr">28</a></sup></p><p>In <em>F.</em> × <em>ananassa,</em> the first linkage map was constructed with 235 or 285 amplified fragment length polymorphism (AFLP) markers on 28 or 30 linkage groups (LGs) based on a 2-way pseudo-testcross strategy.<sup><a reveal-id="DSS035C29" data-open="DSS035C29" class="link link-ref link-reveal xref-bibr">29</a></sup> Several linkage maps were subsequently constructed with AFLP, SCAR, sequence-tagged site, random amplified polymorphic DNA, and SSR markers.<sup><a reveal-id="DSS035C8  DSS035C11" data-open="DSS035C8  DSS035C11" class="link link-ref link-reveal xref-bibr">8–11</a>,<a reveal-id="DSS035C30" data-open="DSS035C30" class="link link-ref link-reveal xref-bibr">30</a></sup> All the previously published linkage maps were generated based on single F<sub>1</sub> mapping populations, and integrated linkage maps were developed in each mapping population in all four studies.<sup><a reveal-id="DSS035C8" data-open="DSS035C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSS035C10" data-open="DSS035C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DSS035C11" data-open="DSS035C11" class="link link-ref link-reveal xref-bibr">11</a>,<a reveal-id="DSS035C30" data-open="DSS035C30" class="link link-ref link-reveal xref-bibr">30</a></sup> Of the previously developed linkage maps, the densest map was constructed by Sargent et al<em>.</em><sup><a reveal-id="DSS035C10" data-open="DSS035C10" class="link link-ref link-reveal xref-bibr">10</a></sup>; a total of 549 loci were mapped onto 28 LGs, with a total length of 2140.3 cM. The density of the map was greatly enhanced over previous maps, but several unsaturated LGs were still observed such as the unintegrated LG pair of FG2DA and FG2DB and LG6B that contained large gaps (36.7 cM). These results suggest the need to develop a denser linkage map in <em>F.</em> × <em>ananassa</em> to reveal the complex genome structure of this species.</p><p>In this study, we performed SSR marker development and constructed an integrated high density linkage map to accelerate the advancement of genomic and genetic studies in <em>F.</em> × <em>ananassa</em>. Three types of SSR markers were developed, namely <em>F. vesca</em> expressed sequence tag (EST)-derived SSR markers (FVES markers), <em>F.</em> × <em>ananassa</em> EST-derived SSR markers (FAES markers), and <em>F.</em> × <em>ananassa</em> transcriptome-derived SSR markers (FATS markers), using public genome sequence data. The markers were mapped onto five parent-specific linkage maps, along with previously published DNA markers, and the five parent-specific maps were then integrated into one map. The applicability of the integrated map and the markers developed in this study were also demonstrated by comparative analysis of <em>F.</em> × <em>ananassa</em> and <em>F. vesca</em> and by variety distinction on 129 <em>F.</em> × <em>ananassa</em> lines with selected markers. The markers and integrated linkage map described in this study are valuable resources for future studies that will help to elucidate the genome structure and evolutionary process in <em>F.</em> × <em>ananassa</em> and whole genome sequencing, genetic mapping, and molecular breeding of this species.</p>                            <h2 scrollto-destination=5643888 id="5643888" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=5643889 id="5643889" class="section-title" data-legacy-id=s2a>2.1. Plant material</h3>
<p>An integrated linkage map was constructed using three mapping populations originating from a total of five parental lines. ‘02–19’ × ‘Sachinoka’ is an F<sub>1</sub> mapping population of 188 individuals derived from a one-way pseudo-testcross. The female parent ‘02–19’ is a breeding line developed in Chiba Prefectural Agriculture and Forestry Research Center, which is resistant to powdery mildew and <em>Fusarium</em> wilt. The male parent ‘Sachinoka’ is a Japanese variety bred in the National Agricultural Research Center for Kyusyu and Okinawa Region, which was derived from a cross between ‘Toyonoka’ and ‘Aiberry’. ‘Kaorino’ × ‘Akihime’ consisted of 140 F<sub>1</sub> individuals derived from a one-way pseudo-testcross between the parental lines. ‘Kaorino’ and ‘Akihime’ were bred in Mie Prefecture Agricultural Research Institute and by Mr Kazuhiro Ogiwara in Japan, respectively. One of the ancestor lines of ‘Kaorino’ is ‘Akihime’. ‘Kaorino’ exhibits crown rot resistance, whereas ‘Akihime’ is susceptible to the disease. The ‘0212921’ mapping population consisted of 169 S<sub>1</sub> individuals of ‘0212921’. ‘0212921’ was generated from a cross between two breeding lines developed in Mie Prefecture Agricultural Research Institute, and the male parent of ‘0212921’ was identical to the female parent of ‘Kaorino’.</p>                            <h3 scrollto-destination=5643891 id="5643891" class="section-title" data-legacy-id=s2b>2.2. Development of EST-derived SSR markers</h3>
<p>SSR regions were identified from the <em>F. vesca</em> and <em>F.</em> × <em>ananassa</em> ESTs registered in public databases, namely NCBI's nucleotide database (<a class="link link-uri" href="http://www.ncbi.nih.gov">http://www.ncbi.nih.gov</a>). The numbers of ESTs that were investigated include 45 739 and 6117 in <em>F. vesca</em> and <em>F.</em> × <em>ananassa</em>, respectively. SSRs longer than 14 bases, which contained all possible combinations of dinucleotide (NN), trinucleotide (NNN) and tetranucleotide (NNNN) repeats, were identified using the FINDPATTERNS module in the GCG software package (Accelrys Inc., USA). Oligonucleotides for polymerase chain reaction (PCR) primers were designed based on the flanking regions of the identified SSRs using the Primer3 program<sup><a reveal-id="DSS035C31" data-open="DSS035C31" class="link link-ref link-reveal xref-bibr">31</a></sup> in such a way that the amplified products ranged from 90 to 300 bp in length. Markers corresponding to previously published SSR markers in the <em>Fragaria</em> spp and RosCOS of markers<sup><a reveal-id="DSS035C8" data-open="DSS035C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSS035C10" data-open="DSS035C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DSS035C15" data-open="DSS035C15" class="link link-ref link-reveal xref-bibr">15</a>,<a reveal-id="DSS035C17  DSS035C23" data-open="DSS035C17  DSS035C23" class="link link-ref link-reveal xref-bibr">17–23</a>,<a reveal-id="DSS035C27" data-open="DSS035C27" class="link link-ref link-reveal xref-bibr">27</a>,<a reveal-id="DSS035C32  DSS035C39" data-open="DSS035C32  DSS035C39" class="link link-ref link-reveal xref-bibr">32–39</a></sup> were identified based on primer sequences and were excluded from the collection of FVES and FAES markers.</p>                            <h3 scrollto-destination=5643893 id="5643893" class="section-title" data-legacy-id=s2c>2.3. Development of transcriptome-derived SSR markers</h3>
<p>A total of 1 188 226 <em>F.</em> × <em>ananassa</em> transcript sequence reads registered in NCBI's Sequence Read Achive (SRA, Accession number: SRX16008) were used for identification of the SSR regions. All reads were generated using the Roche 454 sequencing system (Roche, Basel, Switzerland). The MIRA 3.2.0 program was used for assembling non-redundant contigs.<sup><a reveal-id="DSS035C40" data-open="DSS035C40" class="link link-ref link-reveal xref-bibr">40</a></sup> Methods for identification of SSR regions and primer design were the same as those used for FVES and FAES marker development, except that penta- and hexanucleotide repeats were also identified and used for primer design. After designing the primer sequences, the redundancy between the FVES, FAES, and published markers described in the above section was confirmed. The newly developed markers were designated FATS markers.</p>                            <h3 scrollto-destination=5643895 id="5643895" class="section-title" data-legacy-id=s2d>2.4. Polymorphic analysis of the DNA markers</h3>
<p>DNA was extracted from the young leaves of plants using a DNeasy Plant Mini Kit (Qiagen Inc., Germany). DNA quantification and quality checks were performed using an ND1000 NanoDrop spectrophotometer (Nanodrop Technologies, DE, USA) and 0.8% agarose gels. In addition to the FVES, FAES, and FATS markers, a total of 1114 primer pairs of previously published SSR markers developed from <em>Fragaria</em> sequences and RosCOS markers were used for polymorphic analysis of the ‘02–19’ × ‘Sachinoka’ mapping population (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>).<sup><a reveal-id="DSS035C8" data-open="DSS035C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSS035C15" data-open="DSS035C15" class="link link-ref link-reveal xref-bibr">15</a>,<a reveal-id="DSS035C17  DSS035C21" data-open="DSS035C17  DSS035C21" class="link link-ref link-reveal xref-bibr">17–21</a>,<a reveal-id="DSS035C23" data-open="DSS035C23" class="link link-ref link-reveal xref-bibr">23</a>,<a reveal-id="DSS035C27" data-open="DSS035C27" class="link link-ref link-reveal xref-bibr">27</a>,<a reveal-id="DSS035C32  DSS035C38" data-open="DSS035C32  DSS035C38" class="link link-ref link-reveal xref-bibr">32–38</a></sup> Polymorphic analysis of the other two populations was performed without using published markers. PCR was performed in a 5-µl reaction volume using 0.6 ng of genomic DNA in 1X PCR buffer (Bioline, UK), 3 mM MgCl<sub>2</sub>, 0.08 U of BIOTAQ DNA polymerase (Bioline, UK), 0.8 mM dNTPs, and 0.4 µM of each primer. A modified touchdown PCR protocol was followed as described by Sato et al.<sup><a reveal-id="DSS035C41" data-open="DSS035C41" class="link link-ref link-reveal xref-bibr">41</a></sup> The PCR products were separated by 10% polyacrylamide gel electrophoresis in tris-borate-ethylenediaminetetraacetic acid (TBE) buffer according to the standard protocol or with an ABI 3730xl fluorescent fragment analyzer (Applied Biosystems, USA), according to the polymorphic fragment sizes of the PCR amplicons. In the former case, the data were analysed using the Polyans software (<a class="link link-uri" href="http://www.polyans.kazusa.or.jp">http://www.polyans.kazusa.or.jp</a>), whereas in the latter case, polymorphisms were investigated using the GeneMapper software (Applied Biosystems, USA).</p>                            <h3 scrollto-destination=5643897 id="5643897" class="section-title" data-legacy-id=s2e>2.5. Linkage analysis</h3>
<p>In this study, it was assumed that <em>F.</em> × <em>ananassa</em> is an allo-octoploid species and that polymorphic loci in entire chromosomes showed disomic inheritance. Therefore, linkage analysis was performed using the methodology employed with diploid and outcrossing species. The segregated data of the two pseudo-testcross mapping populations were categorized into two parent-specific data sets by comparing the sizes of polymorphic bands of the parents and progeny. The segregation data were rescored using the ‘HAP1’ or ‘F2’ population type codes employed in JoinMap analysis.<sup><a reveal-id="DSS035C42" data-open="DSS035C42" class="link link-ref link-reveal xref-bibr">42</a></sup> ‘HAP1’ codes were used for the four parental lines of the ‘02–19’ × ‘Sachinoka’ and ‘Kaorino’ × ‘Akihime’, whereas ‘F2’ codes were employed for the parents of the S<sub>1</sub> population ‘0212921’. As a result, a total of five parent-specific data sets were generated. The segregation data from each parent-specific data set were then classified into multiple LGs using the colour map method<sup><a reveal-id="DSS035C43" data-open="DSS035C43" class="link link-ref link-reveal xref-bibr">43</a></sup> that employed a comparison of graphical genotypes of the segregation data. During the process of colour mapping, reciprocal genotypes were converted to coupling genotypes. The robustness of the data sets for each LG was then confirmed using the Grouping Module of the JoinMap program, version 4, with a logarithm of odds (LOD) threshold of 10. The homeologous LGs within each parent-specific map were assumed based on corresponding positions on the <em>F. vesca</em> genome, which were predicted by comparative analysis (described below).</p><p>The locus orders in each parent-specific map were calculated using the Regression Mapping Module of JoinMap. ‘0212921’-specific data were handled as an ‘F2’ population type, whereas other parent-specific data sets were calculated as ‘HAP1’ population types. The following parameters were used for the calculation: Kosambi's mapping function, LOD ≥ 1.0, REC frequency ≤0.4, goodness of fit jump threshold for removal of loci = 5.0, number of added loci after which a ripple is performed = 1, and third round = yes. Each LG in parent-specific maps was named according to the following rule: numbers after LG (1–7) showed corresponding chromosome numbers of <em>F. vesca</em> predicted by comparative analysis (described below). The capital letters A–D distinguished four LGs in each homeologous group (HG), which corresponded to LGs in an integrated map. The A–D suffixes corresponded to the lengths of the LGs on the integrated map. When multiple LGs were integrated into a single LG of the integrated map, they were numbered using capital letters with underbars (e.g. LG1B_1 and LG1B_2). For unintegrated LGs, X or Y was used as a suffix.</p><p>For construction of an integrated linkage map, corresponding LGs between the parent-specific maps were assumed by identification of commonly mapped markers on each LG. Prior to integration, genotypes of dominant loci in each parent-specific map were imputed to co-dominant genotypes, according to the flanking genotypes of co-dominant loci. That is, alleles of dominant loci were converted to co-dominant alleles, according to the flanking genotypes of co-dominant loci. Parent-specific locus genotype data sets were then integrated into one dataset in each LG using the Combine Groups for Mapping Integration Module, followed by locus ordering by the Regression Mapping Module of JoinMap. The parameters used for the mapping module of an integrated map were the same as those used for parent-specific mapping. After construction of an integrated map, the mapped loci were classified into two groups, i.e. loci generated from single-locus diagnostic (SLD) and multi-loci diagnostic (MLD) markers. SLD markers were defined as markers that detected single segregation bands mapped onto single positions on an integrated map, and not amplified other loci, such as monomorphic loci, whereas MLD markers were defined as markers that amplified more than one locus, including monomorphic loci.</p>                            <h3 scrollto-destination=5643901 id="5643901" class="section-title" data-legacy-id=s2f>2.6. Comparative mapping</h3>
<p>Syntenic regions between the genomes of <em>F.</em> × <em>ananassa</em> and <em>F. vesca</em> were detected by identifying the conservation of the relative locations of genes and genomic regions. The source of the genome sequences of <em>F. vesca</em> was obtained from NCBI [Accession numbers: CM001053.1-CM001059.1 (LG1-7), GG775183.1- GG775301.1 (unplaced)]. The EST sequences adjacent to the mapped markers on the integrated <em>F.</em> × <em>ananassa</em> map were compared with the gene sequences in the reference genomes using the BLASTX program with a cutoff <em>E</em>-value ≤ 1e-10. The syntenic regions defined by the top hits of the homology search were plotted using the Cicros program (<a class="link link-uri" href="http://circos.ca">http://circos.ca</a>).</p>                            <h3 scrollto-destination=5643903 id="5643903" class="section-title" data-legacy-id=s2g>2.7. Distinguishing the varieties of 129 <em>F</em>. × <em>ananassa</em> lines using selected EST-SSR markers</h3>
<div class="block-child-p">In order to distinguish the varieties of 129 <em>F</em>. × <em>ananassa</em> lines, a total of 22 <em>F</em>. × <em>ananassa</em> varieties and breeding lines, most of which were bred in Japan, were used in pre-polymorphic analysis, including the following: ‘Sachinoka’, ‘Fusanoka’, ‘Asuka Wave’, ‘Kaorino’, ‘Akihime’, ‘Miyoshi’, ‘Dover’, ‘Strawberry Parental Line Nou-2’, ‘Karenberry’, ‘Ohkimi’, ‘Sanchiigo’, ‘Toyonoka’, ‘Nyohou’, Tochiotome’, ‘Nou-Hime’, ‘Asuka ruby’, ‘Sagahonoka’, ‘Beni Hoppe’, ‘Yayoihime’, ‘Sanukihime’, ‘02-19’, and ‘0212921’. PCR was performed with all the FVES and FAES markers developed in this study. The PCR protocol was same as that used for polymorphic analysis of the DNA markers (see above)<em>.</em> The PCR products were separated by 10% polyacrylamide gel electrophoresis in TBE buffer using the standard protocol. SSR markers showing solid and polymorphic amplification were selected, and the robustness of PCR, i.e. the repeatability of results and the absence of noise peaks, was investigated for the selected markers using a fragment analyzer (ABI 3730xl, Applied Biosystems, USA) for 22 <em>F.</em> × <em>ananassa</em> lines. PCR was performed in a 5-µl reaction volume using 1 ng of genomic DNA in 1X PCR buffer II (Applied Biosystems, USA), 2.5 mM MgCl<sub>2</sub>, 0.125 U of AmpliTaq Gold DNA polymerase (Applied Biosystems, USA), 0.2 mM dNTPs, and 0.4 µM of each primer. The thermal cycling conditions were as follows: 7 min denaturation at 95°C; 30 cycles of 30 s denaturation at 95°C, 30 s annealing at 57°C, and 1 min extension at 72°C, with a final 10 min extension at 72°C. A total of 100 markers showing high repeatability were screened, and the robustness of PCR was again confirmed for 129 <em>F.</em> × <em>ananassa</em> lines, 119 of which were previously investigated polymorphisms identified using CAPS markers<sup><a reveal-id="DSS035C44" data-open="DSS035C44" class="link link-ref link-reveal xref-bibr">44</a></sup> (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). The 45 best SSR markers were thereby selected to distinguish the varieties of 129 strawberry lines. The allelic data were converted into a binary matrix using the scores 1 or 0 for the presence or absence of the peak. The expected heterozygosity (HZ) of each identified peak was calculated using the following formula:<a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><div class="fig-graphic"><a class="jumplink-placeholder" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss035/2/dss035ueq1.gif?Expires=2147483647&Signature=A9ArVrff~n93u2fJvCLJVFTShkcGGTHAB2B1SBzmm0YwCFFG2O4zGS50WbAprAynetA57EE~lZmD-nQWZFwBwIUhSY2ZcT6f3Tz8gwiWh-seVANoe~vNVGPHLahiAmofA7QHqqRAQYYFf3LFjjImHMHBRy5~VALIySr66aHP69MFXcRlQPmGt1WxH-KY93cTuX2HuFjrlmmKNshzjG-WrrA4bydAKrNU-fs4SLnBT3BHQ~EODU6D79sL72Di0y4JptJcyT7sMNct4FhCBWS3LBDtirZ1CGmCPnvU7sYb5rm~crtPpihEq9b8kF8uP-J3OK1DeV9~48jdZ38xPQtosw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" id="dss035ueq1.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss035/2/m_dss035ueq1.gif?Expires=2147483647&Signature=asKNWqwG3B8UlVREgHGb7Ep~jjYBvbV0WpENjWIlGW9K4DZpfb4Ksh2d-9l8diSUiyGJTd816BUNYKJY3uz8Oa3rHV3dIlw4df26lvpqKJTQ9f0T~dwH0PsPX3TLa1GbrYiv0MPlzeUoWRI3pVYT-u-gcKI6JiQuiRIg1LrJu6JDlhrFT5kFIyxOgJcMmVoq4P2g99nFMoSE3E8uSZZa6bMhea18yDssXdcRRr59cUrj~LKxo1j7jA1F69h3dVr6dDxl96p9HzAXNKoX7taNoeeOzh~TcK0G3DHUCKfNPDDLKAfG5rGRk0I~2wLIt1yGPcV68yetsVarvEYJrBvzvw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dss035ueq1.gif" /></a></div></div></div></div><p>where <em>P<sub>ip</sub></em> and <em>P<sub>ia</sub></em> are the frequency of presence and absence of the <em>i</em>th peak. In <em>F.</em> × <em>ananassa,</em> SSR markers often identify multi loci due to polyploidy, and it is often difficult to investigate the exact number of loci identified in each marker in an unrelated population. Therefore, the mean HZ value of identified peaks generated by a marker was substituted for the HZ of each marker. The allelic binary data were also analysed using GGT 2.0<sup><a reveal-id="DSS035C45" data-open="DSS035C45" class="link link-ref link-reveal xref-bibr">45</a></sup> for the investigation of genetic similarity between the varieties using a Jaccard similarity coefficient. A unweighted pair group method (UPGMA) with arithmetic average dendrogram was constructed using MEGA version 5.05.<sup><a reveal-id="DSS035C46" data-open="DSS035C46" class="link link-ref link-reveal xref-bibr">46</a></sup></p>                            <h2 scrollto-destination=5643906 id="5643906" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=5643907 id="5643907" class="section-title" data-legacy-id=s3a>3.1. Development of EST-SSR markers</h3>
<p>A total of 2748 and 324 SSRs were identified in non-redundant 15 203 <em>F. vesca</em> and 1029 <em>F</em>. × <em>ananassa</em> ESTs, respectively. Of the identified SSRs, 562 primers were designed based on the flanking regions of 453 <em>F. vesca-</em> and 109 <em>F.</em> × <em>ananassa-</em>derived EST-SSRs and designated FVES and FAES markers. To increase the number of EST-SSR markers, additional primer pairs were designed that allowed either single-base mismatches (779 and 107 primer pairs for FVES and FAES markers, respectively) or two-base mismatches (2514 and 387 primer pairs for FVES and FAES markers, respectively) in the SSR regions. Of the markers that were generated, those corresponding to previously published markers were excluded.<sup><a reveal-id="DSS035C8" data-open="DSS035C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSS035C10" data-open="DSS035C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DSS035C15" data-open="DSS035C15" class="link link-ref link-reveal xref-bibr">15</a>,<a reveal-id="DSS035C17  DSS035C23" data-open="DSS035C17  DSS035C23" class="link link-ref link-reveal xref-bibr">17–23</a>,<a reveal-id="DSS035C27" data-open="DSS035C27" class="link link-ref link-reveal xref-bibr">27</a>,<a reveal-id="DSS035C32  DSS035C39" data-open="DSS035C32  DSS035C39" class="link link-ref link-reveal xref-bibr">32–39</a></sup> As a result, a total of 3746 FVES and 603 FAES markers were developed. Design details of the FVES and FAES markers, including primer sequences, corresponding SSR motifs, expected product sizes, and GenBank IDs of the template EST sequences, are listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S3 and S4</a> and on the web at <a class="link link-uri" href="http://marker.kazusa.or.jp/strawberry/">http://marker.kazusa.or.jp/strawberry/</a>.</p><p>Of the FVES and FAES markers that were developed, 2975 (79.4%) and 431 (71.5%) were trinucleotide repeats while 456 (12.2%) and 117 (19.4%) were dinucleotide repeats, and 315 (8.4%) and 55 (9.1%) were tetranucleotide repeats, respectively (Table <a reveal-id="DSS035TB1" data-open="DSS035TB1" class="link link-reveal link-table xref-fig">1</a>). In the FVES markers, the poly (AAG)<sub>n</sub> motif was most abundant in the trinucleotide repeats (755 SSRs, 20.1%), followed by poly (GGA)<sub>n</sub> (492 SSRs, 13.1%), poly (ATC)<sub>n</sub> (352 SSRs, 9.4%), poly (AG)<sub>n</sub> (347 SSRs, 9.3%), and poly (AGC)<sub>n</sub> (321 SSRs, 8.6%). Like the FVES markers, the poly (AAG)<sub>n</sub> motif was the most frequently observed motif in the FAES markers (132 SSRs, 21.9%). This was followed by poly (AG)<sub>n</sub> (81 SSRs, 13.4%), poly (ATC)<sub>n</sub> (68 SSRs, 11.3%), and poly (AGC)<sub>n</sub> (48 SSRs, 8.0%). Among the tetranucleotide repeats, AT-rich motifs, namely poly (AAAG)<sub>n</sub>, and poly (AAAC)<sub>n</sub>, were more frequently observed than other motifs in both the FVES and FAES markers. </p>                        <a id="5643910" scrollto-destination="5643910"></a>
<div content-id="DSS035TB1" class="table-modal"><div class="table-wrap"><div id="DSS035TB1" class="table-wrap-title" data-id="DSS035TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Numbers of SSR motifs in the FVES, FAES, and FATS markers</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Motif </th><th colspan="4">FVES<hr /> </th><th colspan="4">FAES<hr /> </th><th colspan="4">FATS<hr /> </th></tr><tr><th colspan="2">Designed (%) </th><th colspan="2">Mapped (%) </th><th colspan="2">Designed (%) </th><th colspan="2">Mapped (%) </th><th colspan="2">Designed (%) </th><th colspan="2">Mapped (%) </th></tr></thead><tbody><tr><td colspan="13">Dinucleotide </td></tr><tr><td> AG </td><td>347 </td><td>(9.3) </td><td>120 </td><td>(11.0) </td><td>81 </td><td>(13.4) </td><td>34 </td><td>(17.7) </td><td>10 </td><td>(8.0) </td><td>4 </td><td>(10.3) </td></tr><tr><td> AT </td><td>60 </td><td>(1.6) </td><td>21 </td><td>(1.9) </td><td>22 </td><td>(3.6) </td><td>12 </td><td>(6.3) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td></tr><tr><td> AC </td><td>49 </td><td>(1.3) </td><td>18 </td><td>(1.7) </td><td>14 </td><td>(2.3) </td><td>5 </td><td>(2.6) </td><td>1 </td><td>(0.8) </td><td>0 </td><td>(0.0) </td></tr><tr><td> TC </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td><td>7 </td><td>(5.6) </td><td>4 </td><td>(10.3) </td></tr><tr><td> Others </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td><td>7<sup>a</sup> </td><td>(5.6) </td><td>0 </td><td>(0.0) </td></tr><tr><td>Subtotal </td><td>456 </td><td>(12.2) </td><td>159 </td><td>(14.6) </td><td>117 </td><td>(19.4) </td><td>51 </td><td>(26.6) </td><td>25 </td><td>(20.0) </td><td>8 </td><td>(20.5) </td></tr><tr><td colspan="13">Trinucleotide </td></tr><tr><td> AAG </td><td>755 </td><td>(20.1) </td><td>210 </td><td>(19.3) </td><td>132 </td><td>(21.9) </td><td>41 </td><td>(21.4) </td><td>5 </td><td>(4.0) </td><td>1 </td><td>(2.6) </td></tr><tr><td> GGA </td><td>492 </td><td>(13.1) </td><td>138 </td><td>(12.7) </td><td>45 </td><td>(7.5) </td><td>6 </td><td>(3.1) </td><td>4 </td><td>(3.2) </td><td>3 </td><td>(7.7) </td></tr><tr><td> ATC </td><td>352 </td><td>(9.4) </td><td>104 </td><td>(9.6) </td><td>68 </td><td>(11.3) </td><td>16 </td><td>(8.3) </td><td>2 </td><td>(1.6) </td><td>1 </td><td>(2.6) </td></tr><tr><td> AGC </td><td>321 </td><td>(8.6) </td><td>92 </td><td>(8.4) </td><td>48 </td><td>(8.0) </td><td>20 </td><td>(10.4) </td><td>3 </td><td>(2.4) </td><td>2 </td><td>(5.1) </td></tr><tr><td> GGC </td><td>291 </td><td>(7.8) </td><td>77 </td><td>(7.1) </td><td>27 </td><td>(4.5) </td><td>11 </td><td>(5.7) </td><td>1 </td><td>(0.8) </td><td>0 </td><td>(0.0) </td></tr><tr><td> GGT </td><td>268 </td><td>(7.2) </td><td>78 </td><td>(7.2) </td><td>40 </td><td>(6.6) </td><td>7 </td><td>(3.6) </td><td>1 </td><td>(0.8) </td><td>0 </td><td>(0.0) </td></tr><tr><td> AAC </td><td>207 </td><td>(5.5) </td><td>58 </td><td>(5.3) </td><td>34 </td><td>(5.6) </td><td>11 </td><td>(5.7) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td></tr><tr><td> ACG </td><td>168 </td><td>(4.5) </td><td>50 </td><td>(4.6) </td><td>16 </td><td>(2.7) </td><td>5 </td><td>(2.6) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td></tr><tr><td> ACT </td><td>62 </td><td>(1.7) </td><td>8 </td><td>(0.7) </td><td>5 </td><td>(0.8) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td></tr><tr><td> AAT </td><td>59 </td><td>(1.6) </td><td>17 </td><td>(1.6) </td><td>16 </td><td>(2.7) </td><td>6 </td><td>(3.1) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td></tr><tr><td> CTT </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td><td>4 </td><td>(3.2) </td><td>1 </td><td>(2.6) </td></tr><tr><td> Others </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td><td>36<sup>a</sup> </td><td>(28.8) </td><td>12 </td><td>(30.8) </td></tr><tr><td>Subtotal </td><td>2975 </td><td>(79.4) </td><td>832 </td><td>(76.4) </td><td>431 </td><td>(71.5) </td><td>123 </td><td>(64.1) </td><td>56 </td><td>(44.8) </td><td>20 </td><td>(51.3) </td></tr><tr><td colspan="13">Tetranucleotide </td></tr><tr><td> AAAG </td><td>100 </td><td>(2.7) </td><td>37 </td><td>(3.4) </td><td>15 </td><td>(2.5) </td><td>4 </td><td>(2.1) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td></tr><tr><td> AAAC </td><td>54 </td><td>(1.4) </td><td>14 </td><td>(1.3) </td><td>7 </td><td>(1.2) </td><td>1 </td><td>(0.5) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td></tr><tr><td> AAAT </td><td>43 </td><td>(1.1) </td><td>11 </td><td>(1.0) </td><td>14 </td><td>(2.3) </td><td>6 </td><td>(3.1) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td></tr><tr><td> GGGA </td><td>26 </td><td>(0.7) </td><td>8 </td><td>(0.7) </td><td>3 </td><td>(0.5) </td><td>2 </td><td>(1.0) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td></tr><tr><td> AAGC </td><td>21 </td><td>(0.6) </td><td>6 </td><td>(0.6) </td><td>2 </td><td>(0.3) </td><td>1 </td><td>(0.5) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td></tr><tr><td> AATC </td><td>17 </td><td>(0.5) </td><td>4 </td><td>(0.4) </td><td>5 </td><td>(0.8) </td><td>1 </td><td>(0.5) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td></tr><tr><td> AATG </td><td>15 </td><td>(0.4) </td><td>7 </td><td>(0.6) </td><td>1 </td><td>(0.2) </td><td>1 </td><td>(0.5) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td></tr><tr><td> Others </td><td>39 </td><td>(1.0) </td><td>11 </td><td>(1.0) </td><td>8 </td><td>(1.3) </td><td>2 </td><td>(1.0) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td></tr><tr><td>Subtotal </td><td>315 </td><td>(8.4) </td><td>98 </td><td>(9.0) </td><td>55 </td><td>(9.1) </td><td>18 </td><td>(9.4) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td></tr><tr><td colspan="13">Pentanucleotide </td></tr><tr><td>Subtotal </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>7<sup>a</sup> </td><td>(5.6) </td><td>1 </td><td>(2.6) </td></tr><tr><td colspan="13">Hexanucleotide </td></tr><tr><td>Subtotal </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>37<sup>a</sup> </td><td>(29.6) </td><td>10 </td><td>(25.6) </td></tr><tr><td>Total </td><td>3746 </td><td>(100) </td><td>1089 </td><td>(100) </td><td>603 </td><td>(100) </td><td>192 </td><td>(100) </td><td>125 </td><td>(100) </td><td>39 </td><td>(100) </td></tr></tbody></table></div><div content-id="DSS035TB1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Motif </th><th colspan="4">FVES<hr /> </th><th colspan="4">FAES<hr /> </th><th colspan="4">FATS<hr /> </th></tr><tr><th colspan="2">Designed (%) </th><th colspan="2">Mapped (%) </th><th colspan="2">Designed (%) </th><th colspan="2">Mapped (%) </th><th colspan="2">Designed (%) </th><th colspan="2">Mapped (%) </th></tr></thead><tbody><tr><td colspan="13">Dinucleotide </td></tr><tr><td> AG </td><td>347 </td><td>(9.3) </td><td>120 </td><td>(11.0) </td><td>81 </td><td>(13.4) </td><td>34 </td><td>(17.7) </td><td>10 </td><td>(8.0) </td><td>4 </td><td>(10.3) </td></tr><tr><td> AT </td><td>60 </td><td>(1.6) </td><td>21 </td><td>(1.9) </td><td>22 </td><td>(3.6) </td><td>12 </td><td>(6.3) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td></tr><tr><td> AC </td><td>49 </td><td>(1.3) </td><td>18 </td><td>(1.7) </td><td>14 </td><td>(2.3) </td><td>5 </td><td>(2.6) </td><td>1 </td><td>(0.8) </td><td>0 </td><td>(0.0) </td></tr><tr><td> TC </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td><td>7 </td><td>(5.6) </td><td>4 </td><td>(10.3) </td></tr><tr><td> Others </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td><td>7<sup>a</sup> </td><td>(5.6) </td><td>0 </td><td>(0.0) </td></tr><tr><td>Subtotal </td><td>456 </td><td>(12.2) </td><td>159 </td><td>(14.6) </td><td>117 </td><td>(19.4) </td><td>51 </td><td>(26.6) </td><td>25 </td><td>(20.0) </td><td>8 </td><td>(20.5) </td></tr><tr><td colspan="13">Trinucleotide </td></tr><tr><td> AAG </td><td>755 </td><td>(20.1) </td><td>210 </td><td>(19.3) </td><td>132 </td><td>(21.9) </td><td>41 </td><td>(21.4) </td><td>5 </td><td>(4.0) </td><td>1 </td><td>(2.6) </td></tr><tr><td> GGA </td><td>492 </td><td>(13.1) </td><td>138 </td><td>(12.7) </td><td>45 </td><td>(7.5) </td><td>6 </td><td>(3.1) </td><td>4 </td><td>(3.2) </td><td>3 </td><td>(7.7) </td></tr><tr><td> ATC </td><td>352 </td><td>(9.4) </td><td>104 </td><td>(9.6) </td><td>68 </td><td>(11.3) </td><td>16 </td><td>(8.3) </td><td>2 </td><td>(1.6) </td><td>1 </td><td>(2.6) </td></tr><tr><td> AGC </td><td>321 </td><td>(8.6) </td><td>92 </td><td>(8.4) </td><td>48 </td><td>(8.0) </td><td>20 </td><td>(10.4) </td><td>3 </td><td>(2.4) </td><td>2 </td><td>(5.1) </td></tr><tr><td> GGC </td><td>291 </td><td>(7.8) </td><td>77 </td><td>(7.1) </td><td>27 </td><td>(4.5) </td><td>11 </td><td>(5.7) </td><td>1 </td><td>(0.8) </td><td>0 </td><td>(0.0) </td></tr><tr><td> GGT </td><td>268 </td><td>(7.2) </td><td>78 </td><td>(7.2) </td><td>40 </td><td>(6.6) </td><td>7 </td><td>(3.6) </td><td>1 </td><td>(0.8) </td><td>0 </td><td>(0.0) </td></tr><tr><td> AAC </td><td>207 </td><td>(5.5) </td><td>58 </td><td>(5.3) </td><td>34 </td><td>(5.6) </td><td>11 </td><td>(5.7) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td></tr><tr><td> ACG </td><td>168 </td><td>(4.5) </td><td>50 </td><td>(4.6) </td><td>16 </td><td>(2.7) </td><td>5 </td><td>(2.6) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td></tr><tr><td> ACT </td><td>62 </td><td>(1.7) </td><td>8 </td><td>(0.7) </td><td>5 </td><td>(0.8) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td></tr><tr><td> AAT </td><td>59 </td><td>(1.6) </td><td>17 </td><td>(1.6) </td><td>16 </td><td>(2.7) </td><td>6 </td><td>(3.1) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td></tr><tr><td> CTT </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td><td>4 </td><td>(3.2) </td><td>1 </td><td>(2.6) </td></tr><tr><td> Others </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td><td>36<sup>a</sup> </td><td>(28.8) </td><td>12 </td><td>(30.8) </td></tr><tr><td>Subtotal </td><td>2975 </td><td>(79.4) </td><td>832 </td><td>(76.4) </td><td>431 </td><td>(71.5) </td><td>123 </td><td>(64.1) </td><td>56 </td><td>(44.8) </td><td>20 </td><td>(51.3) </td></tr><tr><td colspan="13">Tetranucleotide </td></tr><tr><td> AAAG </td><td>100 </td><td>(2.7) </td><td>37 </td><td>(3.4) </td><td>15 </td><td>(2.5) </td><td>4 </td><td>(2.1) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td></tr><tr><td> AAAC </td><td>54 </td><td>(1.4) </td><td>14 </td><td>(1.3) </td><td>7 </td><td>(1.2) </td><td>1 </td><td>(0.5) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td></tr><tr><td> AAAT </td><td>43 </td><td>(1.1) </td><td>11 </td><td>(1.0) </td><td>14 </td><td>(2.3) </td><td>6 </td><td>(3.1) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td></tr><tr><td> GGGA </td><td>26 </td><td>(0.7) </td><td>8 </td><td>(0.7) </td><td>3 </td><td>(0.5) </td><td>2 </td><td>(1.0) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td></tr><tr><td> AAGC </td><td>21 </td><td>(0.6) </td><td>6 </td><td>(0.6) </td><td>2 </td><td>(0.3) </td><td>1 </td><td>(0.5) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td></tr><tr><td> AATC </td><td>17 </td><td>(0.5) </td><td>4 </td><td>(0.4) </td><td>5 </td><td>(0.8) </td><td>1 </td><td>(0.5) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td></tr><tr><td> AATG </td><td>15 </td><td>(0.4) </td><td>7 </td><td>(0.6) </td><td>1 </td><td>(0.2) </td><td>1 </td><td>(0.5) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td></tr><tr><td> Others </td><td>39 </td><td>(1.0) </td><td>11 </td><td>(1.0) </td><td>8 </td><td>(1.3) </td><td>2 </td><td>(1.0) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td></tr><tr><td>Subtotal </td><td>315 </td><td>(8.4) </td><td>98 </td><td>(9.0) </td><td>55 </td><td>(9.1) </td><td>18 </td><td>(9.4) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td></tr><tr><td colspan="13">Pentanucleotide </td></tr><tr><td>Subtotal </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>7<sup>a</sup> </td><td>(5.6) </td><td>1 </td><td>(2.6) </td></tr><tr><td colspan="13">Hexanucleotide </td></tr><tr><td>Subtotal </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>37<sup>a</sup> </td><td>(29.6) </td><td>10 </td><td>(25.6) </td></tr><tr><td>Total </td><td>3746 </td><td>(100) </td><td>1089 </td><td>(100) </td><td>603 </td><td>(100) </td><td>192 </td><td>(100) </td><td>125 </td><td>(100) </td><td>39 </td><td>(100) </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>The numbers of types of observed ‘other’ SSR motifs in di- and trinucleotide repeats and all penta- and hexa- nucleotide repeats were 4, 24, 6, and 37, respectively.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5643910" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSS035TB1" class="table-wrap-title" data-id="DSS035TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Numbers of SSR motifs in the FVES, FAES, and FATS markers</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Motif </th><th colspan="4">FVES<hr /> </th><th colspan="4">FAES<hr /> </th><th colspan="4">FATS<hr /> </th></tr><tr><th colspan="2">Designed (%) </th><th colspan="2">Mapped (%) </th><th colspan="2">Designed (%) </th><th colspan="2">Mapped (%) </th><th colspan="2">Designed (%) </th><th colspan="2">Mapped (%) </th></tr></thead><tbody><tr><td colspan="13">Dinucleotide </td></tr><tr><td> AG </td><td>347 </td><td>(9.3) </td><td>120 </td><td>(11.0) </td><td>81 </td><td>(13.4) </td><td>34 </td><td>(17.7) </td><td>10 </td><td>(8.0) </td><td>4 </td><td>(10.3) </td></tr><tr><td> AT </td><td>60 </td><td>(1.6) </td><td>21 </td><td>(1.9) </td><td>22 </td><td>(3.6) </td><td>12 </td><td>(6.3) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td></tr><tr><td> AC </td><td>49 </td><td>(1.3) </td><td>18 </td><td>(1.7) </td><td>14 </td><td>(2.3) </td><td>5 </td><td>(2.6) </td><td>1 </td><td>(0.8) </td><td>0 </td><td>(0.0) </td></tr><tr><td> TC </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td><td>7 </td><td>(5.6) </td><td>4 </td><td>(10.3) </td></tr><tr><td> Others </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td><td>7<sup>a</sup> </td><td>(5.6) </td><td>0 </td><td>(0.0) </td></tr><tr><td>Subtotal </td><td>456 </td><td>(12.2) </td><td>159 </td><td>(14.6) </td><td>117 </td><td>(19.4) </td><td>51 </td><td>(26.6) </td><td>25 </td><td>(20.0) </td><td>8 </td><td>(20.5) </td></tr><tr><td colspan="13">Trinucleotide </td></tr><tr><td> AAG </td><td>755 </td><td>(20.1) </td><td>210 </td><td>(19.3) </td><td>132 </td><td>(21.9) </td><td>41 </td><td>(21.4) </td><td>5 </td><td>(4.0) </td><td>1 </td><td>(2.6) </td></tr><tr><td> GGA </td><td>492 </td><td>(13.1) </td><td>138 </td><td>(12.7) </td><td>45 </td><td>(7.5) </td><td>6 </td><td>(3.1) </td><td>4 </td><td>(3.2) </td><td>3 </td><td>(7.7) </td></tr><tr><td> ATC </td><td>352 </td><td>(9.4) </td><td>104 </td><td>(9.6) </td><td>68 </td><td>(11.3) </td><td>16 </td><td>(8.3) </td><td>2 </td><td>(1.6) </td><td>1 </td><td>(2.6) </td></tr><tr><td> AGC </td><td>321 </td><td>(8.6) </td><td>92 </td><td>(8.4) </td><td>48 </td><td>(8.0) </td><td>20 </td><td>(10.4) </td><td>3 </td><td>(2.4) </td><td>2 </td><td>(5.1) </td></tr><tr><td> GGC </td><td>291 </td><td>(7.8) </td><td>77 </td><td>(7.1) </td><td>27 </td><td>(4.5) </td><td>11 </td><td>(5.7) </td><td>1 </td><td>(0.8) </td><td>0 </td><td>(0.0) </td></tr><tr><td> GGT </td><td>268 </td><td>(7.2) </td><td>78 </td><td>(7.2) </td><td>40 </td><td>(6.6) </td><td>7 </td><td>(3.6) </td><td>1 </td><td>(0.8) </td><td>0 </td><td>(0.0) </td></tr><tr><td> AAC </td><td>207 </td><td>(5.5) </td><td>58 </td><td>(5.3) </td><td>34 </td><td>(5.6) </td><td>11 </td><td>(5.7) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td></tr><tr><td> ACG </td><td>168 </td><td>(4.5) </td><td>50 </td><td>(4.6) </td><td>16 </td><td>(2.7) </td><td>5 </td><td>(2.6) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td></tr><tr><td> ACT </td><td>62 </td><td>(1.7) </td><td>8 </td><td>(0.7) </td><td>5 </td><td>(0.8) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td></tr><tr><td> AAT </td><td>59 </td><td>(1.6) </td><td>17 </td><td>(1.6) </td><td>16 </td><td>(2.7) </td><td>6 </td><td>(3.1) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td></tr><tr><td> CTT </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td><td>4 </td><td>(3.2) </td><td>1 </td><td>(2.6) </td></tr><tr><td> Others </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td><td>36<sup>a</sup> </td><td>(28.8) </td><td>12 </td><td>(30.8) </td></tr><tr><td>Subtotal </td><td>2975 </td><td>(79.4) </td><td>832 </td><td>(76.4) </td><td>431 </td><td>(71.5) </td><td>123 </td><td>(64.1) </td><td>56 </td><td>(44.8) </td><td>20 </td><td>(51.3) </td></tr><tr><td colspan="13">Tetranucleotide </td></tr><tr><td> AAAG </td><td>100 </td><td>(2.7) </td><td>37 </td><td>(3.4) </td><td>15 </td><td>(2.5) </td><td>4 </td><td>(2.1) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td></tr><tr><td> AAAC </td><td>54 </td><td>(1.4) </td><td>14 </td><td>(1.3) </td><td>7 </td><td>(1.2) </td><td>1 </td><td>(0.5) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td></tr><tr><td> AAAT </td><td>43 </td><td>(1.1) </td><td>11 </td><td>(1.0) </td><td>14 </td><td>(2.3) </td><td>6 </td><td>(3.1) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td></tr><tr><td> GGGA </td><td>26 </td><td>(0.7) </td><td>8 </td><td>(0.7) </td><td>3 </td><td>(0.5) </td><td>2 </td><td>(1.0) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td></tr><tr><td> AAGC </td><td>21 </td><td>(0.6) </td><td>6 </td><td>(0.6) </td><td>2 </td><td>(0.3) </td><td>1 </td><td>(0.5) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td></tr><tr><td> AATC </td><td>17 </td><td>(0.5) </td><td>4 </td><td>(0.4) </td><td>5 </td><td>(0.8) </td><td>1 </td><td>(0.5) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td></tr><tr><td> AATG </td><td>15 </td><td>(0.4) </td><td>7 </td><td>(0.6) </td><td>1 </td><td>(0.2) </td><td>1 </td><td>(0.5) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td></tr><tr><td> Others </td><td>39 </td><td>(1.0) </td><td>11 </td><td>(1.0) </td><td>8 </td><td>(1.3) </td><td>2 </td><td>(1.0) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td></tr><tr><td>Subtotal </td><td>315 </td><td>(8.4) </td><td>98 </td><td>(9.0) </td><td>55 </td><td>(9.1) </td><td>18 </td><td>(9.4) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td></tr><tr><td colspan="13">Pentanucleotide </td></tr><tr><td>Subtotal </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>7<sup>a</sup> </td><td>(5.6) </td><td>1 </td><td>(2.6) </td></tr><tr><td colspan="13">Hexanucleotide </td></tr><tr><td>Subtotal </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>37<sup>a</sup> </td><td>(29.6) </td><td>10 </td><td>(25.6) </td></tr><tr><td>Total </td><td>3746 </td><td>(100) </td><td>1089 </td><td>(100) </td><td>603 </td><td>(100) </td><td>192 </td><td>(100) </td><td>125 </td><td>(100) </td><td>39 </td><td>(100) </td></tr></tbody></table></div><div content-id="DSS035TB1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Motif </th><th colspan="4">FVES<hr /> </th><th colspan="4">FAES<hr /> </th><th colspan="4">FATS<hr /> </th></tr><tr><th colspan="2">Designed (%) </th><th colspan="2">Mapped (%) </th><th colspan="2">Designed (%) </th><th colspan="2">Mapped (%) </th><th colspan="2">Designed (%) </th><th colspan="2">Mapped (%) </th></tr></thead><tbody><tr><td colspan="13">Dinucleotide </td></tr><tr><td> AG </td><td>347 </td><td>(9.3) </td><td>120 </td><td>(11.0) </td><td>81 </td><td>(13.4) </td><td>34 </td><td>(17.7) </td><td>10 </td><td>(8.0) </td><td>4 </td><td>(10.3) </td></tr><tr><td> AT </td><td>60 </td><td>(1.6) </td><td>21 </td><td>(1.9) </td><td>22 </td><td>(3.6) </td><td>12 </td><td>(6.3) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td></tr><tr><td> AC </td><td>49 </td><td>(1.3) </td><td>18 </td><td>(1.7) </td><td>14 </td><td>(2.3) </td><td>5 </td><td>(2.6) </td><td>1 </td><td>(0.8) </td><td>0 </td><td>(0.0) </td></tr><tr><td> TC </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td><td>7 </td><td>(5.6) </td><td>4 </td><td>(10.3) </td></tr><tr><td> Others </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td><td>7<sup>a</sup> </td><td>(5.6) </td><td>0 </td><td>(0.0) </td></tr><tr><td>Subtotal </td><td>456 </td><td>(12.2) </td><td>159 </td><td>(14.6) </td><td>117 </td><td>(19.4) </td><td>51 </td><td>(26.6) </td><td>25 </td><td>(20.0) </td><td>8 </td><td>(20.5) </td></tr><tr><td colspan="13">Trinucleotide </td></tr><tr><td> AAG </td><td>755 </td><td>(20.1) </td><td>210 </td><td>(19.3) </td><td>132 </td><td>(21.9) </td><td>41 </td><td>(21.4) </td><td>5 </td><td>(4.0) </td><td>1 </td><td>(2.6) </td></tr><tr><td> GGA </td><td>492 </td><td>(13.1) </td><td>138 </td><td>(12.7) </td><td>45 </td><td>(7.5) </td><td>6 </td><td>(3.1) </td><td>4 </td><td>(3.2) </td><td>3 </td><td>(7.7) </td></tr><tr><td> ATC </td><td>352 </td><td>(9.4) </td><td>104 </td><td>(9.6) </td><td>68 </td><td>(11.3) </td><td>16 </td><td>(8.3) </td><td>2 </td><td>(1.6) </td><td>1 </td><td>(2.6) </td></tr><tr><td> AGC </td><td>321 </td><td>(8.6) </td><td>92 </td><td>(8.4) </td><td>48 </td><td>(8.0) </td><td>20 </td><td>(10.4) </td><td>3 </td><td>(2.4) </td><td>2 </td><td>(5.1) </td></tr><tr><td> GGC </td><td>291 </td><td>(7.8) </td><td>77 </td><td>(7.1) </td><td>27 </td><td>(4.5) </td><td>11 </td><td>(5.7) </td><td>1 </td><td>(0.8) </td><td>0 </td><td>(0.0) </td></tr><tr><td> GGT </td><td>268 </td><td>(7.2) </td><td>78 </td><td>(7.2) </td><td>40 </td><td>(6.6) </td><td>7 </td><td>(3.6) </td><td>1 </td><td>(0.8) </td><td>0 </td><td>(0.0) </td></tr><tr><td> AAC </td><td>207 </td><td>(5.5) </td><td>58 </td><td>(5.3) </td><td>34 </td><td>(5.6) </td><td>11 </td><td>(5.7) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td></tr><tr><td> ACG </td><td>168 </td><td>(4.5) </td><td>50 </td><td>(4.6) </td><td>16 </td><td>(2.7) </td><td>5 </td><td>(2.6) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td></tr><tr><td> ACT </td><td>62 </td><td>(1.7) </td><td>8 </td><td>(0.7) </td><td>5 </td><td>(0.8) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td></tr><tr><td> AAT </td><td>59 </td><td>(1.6) </td><td>17 </td><td>(1.6) </td><td>16 </td><td>(2.7) </td><td>6 </td><td>(3.1) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td></tr><tr><td> CTT </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td><td>4 </td><td>(3.2) </td><td>1 </td><td>(2.6) </td></tr><tr><td> Others </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td><td>36<sup>a</sup> </td><td>(28.8) </td><td>12 </td><td>(30.8) </td></tr><tr><td>Subtotal </td><td>2975 </td><td>(79.4) </td><td>832 </td><td>(76.4) </td><td>431 </td><td>(71.5) </td><td>123 </td><td>(64.1) </td><td>56 </td><td>(44.8) </td><td>20 </td><td>(51.3) </td></tr><tr><td colspan="13">Tetranucleotide </td></tr><tr><td> AAAG </td><td>100 </td><td>(2.7) </td><td>37 </td><td>(3.4) </td><td>15 </td><td>(2.5) </td><td>4 </td><td>(2.1) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td></tr><tr><td> AAAC </td><td>54 </td><td>(1.4) </td><td>14 </td><td>(1.3) </td><td>7 </td><td>(1.2) </td><td>1 </td><td>(0.5) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td></tr><tr><td> AAAT </td><td>43 </td><td>(1.1) </td><td>11 </td><td>(1.0) </td><td>14 </td><td>(2.3) </td><td>6 </td><td>(3.1) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td></tr><tr><td> GGGA </td><td>26 </td><td>(0.7) </td><td>8 </td><td>(0.7) </td><td>3 </td><td>(0.5) </td><td>2 </td><td>(1.0) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td></tr><tr><td> AAGC </td><td>21 </td><td>(0.6) </td><td>6 </td><td>(0.6) </td><td>2 </td><td>(0.3) </td><td>1 </td><td>(0.5) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td></tr><tr><td> AATC </td><td>17 </td><td>(0.5) </td><td>4 </td><td>(0.4) </td><td>5 </td><td>(0.8) </td><td>1 </td><td>(0.5) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td></tr><tr><td> AATG </td><td>15 </td><td>(0.4) </td><td>7 </td><td>(0.6) </td><td>1 </td><td>(0.2) </td><td>1 </td><td>(0.5) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td></tr><tr><td> Others </td><td>39 </td><td>(1.0) </td><td>11 </td><td>(1.0) </td><td>8 </td><td>(1.3) </td><td>2 </td><td>(1.0) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td></tr><tr><td>Subtotal </td><td>315 </td><td>(8.4) </td><td>98 </td><td>(9.0) </td><td>55 </td><td>(9.1) </td><td>18 </td><td>(9.4) </td><td>0 </td><td>(0.0) </td><td>0 </td><td>(0.0) </td></tr><tr><td colspan="13">Pentanucleotide </td></tr><tr><td>Subtotal </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>7<sup>a</sup> </td><td>(5.6) </td><td>1 </td><td>(2.6) </td></tr><tr><td colspan="13">Hexanucleotide </td></tr><tr><td>Subtotal </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>37<sup>a</sup> </td><td>(29.6) </td><td>10 </td><td>(25.6) </td></tr><tr><td>Total </td><td>3746 </td><td>(100) </td><td>1089 </td><td>(100) </td><td>603 </td><td>(100) </td><td>192 </td><td>(100) </td><td>125 </td><td>(100) </td><td>39 </td><td>(100) </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>The numbers of types of observed ‘other’ SSR motifs in di- and trinucleotide repeats and all penta- and hexa- nucleotide repeats were 4, 24, 6, and 37, respectively.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5643910" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=5643911 id="5643911" class="section-title" data-legacy-id=s3b>3.2. Development of transcriptome-SSR markers</h3>
<p>A total of 1 188 226 SRA sequences were assembled into 80 430 contigs by the MIRA 3.2.0 program.<sup><a reveal-id="DSS035C38" data-open="DSS035C38" class="link link-ref link-reveal xref-bibr">38</a></sup> On these contigs, 34 993 SSRs were identified. Primers that targeted the flanking regions of 129 of the 34 993 SSRs were designed and synthesized. All the 129 SSRs were located on non-redundant contigs, and a total of 125 primers, which did not overlap with the FVES, FAES, and previously published markers,<sub>,</sub><sup><a reveal-id="DSS035C8" data-open="DSS035C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSS035C10" data-open="DSS035C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DSS035C15" data-open="DSS035C15" class="link link-ref link-reveal xref-bibr">15</a>,<a reveal-id="DSS035C17  DSS035C23" data-open="DSS035C17  DSS035C23" class="link link-ref link-reveal xref-bibr">17–23</a>,<a reveal-id="DSS035C27" data-open="DSS035C27" class="link link-ref link-reveal xref-bibr">27</a>,<a reveal-id="DSS035C32  DSS035C39" data-open="DSS035C32  DSS035C39" class="link link-ref link-reveal xref-bibr">32–39</a></sup> were designed as FATS markers. The primer sequences of the FATS markers, along with the corresponding SSR motifs, product sizes, primer sequences, and template contigs, are provided on the web at <a class="link link-uri" href="http://marker.kazusa.or.jp/strawberry/">http://marker.kazusa.or.jp/strawberry/</a> and in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a>. Of the 125 FATS markers, trinucleotide repeats were the most frequently observed (56 SSRs, 44.8%), followed by hexanucleotide repeats (37 SSRs, 29.6%), and dinucleotide repeats (25 SSRs, 20.0%, Table <a reveal-id="DSS035TB1" data-open="DSS035TB1" class="link link-reveal link-table xref-fig">1</a>). No tetranucleotide repeats were observed, whereas seven pentanucleotide repeats were identified. Of the SSR motifs, the poly (AG)<sub>n</sub> motif was the most abundant (10 SSRs, 8.0%), followed by poly (TC)<sub>n</sub> (7 SSRs, 5.6%), and poly (AAG)<sub>n</sub> (5 SSRs, 4.0%). In the penta- and hexanucleotide repeats, each single SSR was identified for all the observed motifs except poly (GCTGT)<sub>n</sub> (two SSRs, data not shown).</p>                            <h3 scrollto-destination=5643913 id="5643913" class="section-title" data-legacy-id=s3c>3.3. Construction of parent-specific linkage maps</h3>
                            <h4 scrollto-destination=5643914 id="5643914" class="section-title" data-legacy-id=s3c1>3.3.1. The ‘0212921’ S<sub>1</sub> mapping population</h4>
<p>A total of 4474 primer pairs of SSR markers, including 3746 FVES, 603 FAES, and 125 FATS, were investigated with 8 randomly chosen S<sub>1</sub> individuals of the ‘0212921’ mapping population. Polymorphisms were observed on 605 FVES, 135 FAES, and 29 FATS markers, and segregation analysis was performed for 169 S<sub>1</sub> individuals with 769 primer pairs. A total of 881 segregation locus genotypes were generated from the 769 primer pairs, and 822 of the 881 loci were mapped onto 34 LGs (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S6 and S7</a>). The length of each LG ranged from 1.5 cM (LG4B) to 94.5 cM (LG1A), representing a total length of 1508.3 cM. The mean locus density and segregation distortion (<em>P</em> &lt; 0.05) were 1.83 cM locus<sup>−1</sup> and 22.4%, respectively, ranging from 0.22 cM locus<sup>−1</sup> (LG6D) to 4.26 cM locus<sup>−1</sup> (LG2A_2) and from 0.0% (LG4B) to 85.7% (LG5A_1), respectively.</p>                            <h4 scrollto-destination=5643916 id="5643916" class="section-title" data-legacy-id=s3c2>3.3.2. The ‘02–19’ × ‘Sachinoka’ mapping population</h4>
<p>Polymorphic analysis was performed with a total of 5588 primer pairs of SSR markers, including 4474 SSR markers developed in this study and 1114 previously published markers (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). A total of 1299 markers, i.e. 853 FVES, 131 FAES, 37 FATS, and 278 published markers, showed polymorphisms between the parental lines. By performing segregation analysis of the 188 individuals of the mapping population, 1078 polymorphic loci were generated from 881 of the 1299 primer pairs that were tested. Of the 1078 segregation loci, 260 showed biparental polymorphisms while the other 818 were parent specific. A total of 575 and 556 loci were mapped onto ‘02–19’ and ‘Sachinoka’ specific maps, respectively (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S6 and S7</a>). In the ‘02–19’ specific map, 32 LGs were constructed on 1668.9 cM, with lengths ranging from 2.0 cM (LG2X) to 129.2 cM (LG1A). The mean locus density and segregation distortion (<em>P</em> &lt; 0.05) were 2.90 cM locus<sup>−1</sup> and 30.2%, respectively, ranging from 0.34 cM locus<sup>−1</sup> (LG5A_2) to 11.63 cM locus<sup>−1</sup> (LG6C) and from 0.0% (LG3B_2, 5A_2 and 7D_2) to 100% (LG2X and 4B), respectively. In the ‘Sachinoka’ map, 34 LGs were developed, totalling 2166.6 cM in length, with a mean locus density of 3.90 cM locus<sup>−1</sup>. The length and locus density of each LG ranged from 1.6 cM (LG7C_2) to 120.0 cM (LG7A) and from 0.80 cM locus<sup>−1</sup> (LG7C_2) to 16.08 cM locus<sup>−1</sup> (LG3X), respectively. Segregation distortion (<em>P</em> &lt; 0.05) of each LG ranged from 0.0% (LG7C_2, 7D_1 and 7X) to 85.7% (LG2X), with a mean value of 25.4%.</p>                            <h4 scrollto-destination=5643918 id="5643918" class="section-title" data-legacy-id=s3c3>3.3.3. The ‘Kaorino’ × ‘Akihime’ mapping population</h4>
<p>A total of 4474 primer pairs of SSR markers developed in this study were used for polymorphic analysis with the parental lines. Polymorphisms were observed in 438 FVES, 95 FAES, and 33 FATS markers, and segregation analysis was performed with the 140 mapping plants, for a total of 566 primer pairs. A total of 612 segregation locus genotypes were generated from 537 of the 566 primer pairs that were tested. Of the 612 segregation loci, 140 showed biparental polymorphisms while the other 472 were parent specific. A total of 294 and 318 loci were mapped onto ‘Kaorino’ and ‘Akihime’ specific maps, respectively (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S6 and S7</a>). The ‘Kaorino’ map consisted of 32 LGs, representing a total length of 1103.4 cM. The mean locus density and segregation distortion (<em>P</em> &lt; 0.05) were 3.75 cM locus<sup>−1</sup> and 24.8%, respectively, ranging from 0.45 cM locus<sup>−1</sup> (LG1B_2) to 7.61 cM locus<sup>−1</sup> (LG1A) and from 0.0% (LG1B_2, 1X, 2A, 2D, 4X, 5A_1, 5X, 6A_2, 6A_3 and 7D) to 100% (LG4D_2), respectively. In the ‘Akihime’ map, 33 LGs were developed, with a total length of 951.4 cM and a mean locus density of 2.99 cM locus<sup>−1</sup>. The length and locus density of each LG ranged from 2.2 cM (LG5X) to 77.7 cM (LG7A) and from 0.55 cM locus<sup>−1</sup> (LG5X) to 12.72 cM locus<sup>−1</sup> (LG6A_4), respectively. Segregation distortion (<em>P</em> &lt; 0.05) of each LG ranged from 0.0% (LG3X, 4A_2, 4C, 5A, 5C, 5X, 6A_1, 6B_1, 6B_2, and 7D_2) to 100% (LG1A_1), with a 14.8% mean value.</p>                            <h3 scrollto-destination=5643920 id="5643920" class="section-title" data-legacy-id=s3d>3.4. Construction of an integrated linkage map</h3>
<p>Prior to integration, the subsets of parent-specific LGs to be integrated were determined by the numbers of commonly mapped markers across parent-specific maps, that is, pairs of LGs showing the largest number of commonly mapped markers were assembled into the same subset. When one LG showed the same numbers of commonly mapped markers in different pairs, the LG was excluded from the integration. Locus genotype data of each subset of LG were integrated using the Combine Groups for Mapping Integration Module in JoinMap, and the locus orders of an integrated map were then calculated. When grouped LG subsets were inadequate, the segregation loci of incorrect LGs were excluded during the process of locus ordering. When a LG subset was misassembled with incorrect pairs of parent-specific LGs, the loci of the mis-integrated LG overlapped with loci onto other integrated LGs which correctly paired LGs were integrated. Therefore, the correct assembly of a LG subset was determined by confirming the number of overlapping loci on each integrated LG using the Mapping module in JoinMap.</p><p>The integrated linkage map consisted of 1856 loci on 28 LGs, totalling 2364.1 cM in length (Table <a reveal-id="DSS035TB2" data-open="DSS035TB2" class="link link-reveal link-table xref-fig">2</a>, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S7 and Fig. S1</a>). The length of each LG varied from 34.2 (LG1D) to 123.7 cM (LG1A). The mean locus density was 1.27 cM locus<sup>−1</sup>, ranging from 0.68 cM locus<sup>−1</sup> (LG6B) to 5.08 cM locus<sup>−1</sup> (LG4B). The largest gap was 27.3 cM, between FVES1598_7a and FVES1351_7a on LG7A, followed by a 23.4 cM gap between FAES0326_4b and FAES3462_4b on LG4B, and a 21.6 cM gap between FVES0576_4c and FAES0045 on LG4C. The number of integrated LGs ranged from 1 (LG1D) to 16 (LG6A) parent-specific LGs (Table <a reveal-id="DSS035TB2" data-open="DSS035TB2" class="link link-reveal link-table xref-fig">2</a>). The number of unintegrated parent-specific LGs, which totalled 17, ranged from 1 (‘0212921’) to 5 (‘Akihime’) in each parental map. The ratios of mapped FVES, FAES, and FATS loci were 77.5, 15, and 3.4%, respectively (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>). The ratio of mapped FVES markers in each LG ranged from 62.5 to 93.3%. </p>                        <a id="5643923" scrollto-destination="5643923"></a>
<div content-id="DSS035TB2" class="table-modal"><div class="table-wrap"><div id="DSS035TB2" class="table-wrap-title" data-id="DSS035TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Number of mapped loci, length, locus density in an integrated map, and numbers of integrated LGs and loci of parental specific maps</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="3">LG </th><th colspan="4" rowspan="2">Interspecific map<hr /> </th><th colspan="10">Parental specific maps<hr /> </th></tr><tr><th colspan="2">‘0212921’<hr /> </th><th colspan="2">‘02–19’<hr /> </th><th colspan="2">‘Sachinoka’<hr /> </th><th colspan="2">‘Kaorino’<hr /> </th><th colspan="2">‘Akihime’<hr /> </th></tr><tr><th>Number of mapped loci </th><th>Length (cM) </th><th>Locus density (cM) </th><th>Single loci<sup>a</sup> (%) </th><th>LGs<sup>b</sup> </th><th>Loci<sup>c</sup> </th><th>LGs<sup>b</sup> </th><th>Loci<sup>c</sup> </th><th>LGs<sup>b</sup> </th><th>Loci<sup>c</sup> </th><th>LGs<sup>b</sup> </th><th>Loci<sup>c</sup> </th><th>LGs<sup>b</sup> </th><th>Loci<sup>c</sup> </th></tr></thead><tbody><tr><td>1A </td><td>91 </td><td>123.7 </td><td>1.36 </td><td>16 (17.5) </td><td>1 </td><td>48 </td><td>1 </td><td>40 </td><td>1 </td><td>13 </td><td>1 </td><td>7 </td><td>2 </td><td>10 </td></tr><tr><td>1B </td><td>81 </td><td>80.7 </td><td>1.00 </td><td>25 (30.8) </td><td>1 </td><td>25 </td><td>1 </td><td>41 </td><td>2 </td><td>42 </td><td>2 </td><td>14 </td><td>0 </td><td>0 </td></tr><tr><td>1C </td><td>37 </td><td>77.2 </td><td>2.09 </td><td>11 (29.7) </td><td>1 </td><td>24 </td><td>1 </td><td>9 </td><td>1 </td><td>13 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td></tr><tr><td>1D </td><td>24 </td><td>34.2 </td><td>1.43 </td><td>5 (20.8) </td><td>1 </td><td>24 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td></tr><tr><td>2A </td><td>123 </td><td>113.9 </td><td>0.93 </td><td>41 (33.3) </td><td>2 </td><td>25 </td><td>2 </td><td>50 </td><td>1 </td><td>67 </td><td>1 </td><td>8 </td><td>2 </td><td>22 </td></tr><tr><td>2B </td><td>71 </td><td>95.1 </td><td>1.34 </td><td>15 (21.1) </td><td>1 </td><td>42 </td><td>2 </td><td>20 </td><td>1 </td><td>4 </td><td>1 </td><td>11 </td><td>1 </td><td>15 </td></tr><tr><td>2C </td><td>55 </td><td>93.8 </td><td>1.71 </td><td>13 (23.6) </td><td>1 </td><td>28 </td><td>0 </td><td>0 </td><td>1 </td><td>18 </td><td>1 </td><td>24 </td><td>0 </td><td>0 </td></tr><tr><td>2D </td><td>64 </td><td>81.1 </td><td>1.27 </td><td>19 (29.6) </td><td>1 </td><td>37 </td><td>1 </td><td>28 </td><td>1 </td><td>7 </td><td>1 </td><td>8 </td><td>0 </td><td>0 </td></tr><tr><td>3A </td><td>114 </td><td>96.0 </td><td>0.84 </td><td>47 (41.2) </td><td>1 </td><td>33 </td><td>2 </td><td>70 </td><td>2 </td><td>27 </td><td>1 </td><td>23 </td><td>1 </td><td>18 </td></tr><tr><td>3B </td><td>89 </td><td>82.7 </td><td>0.93 </td><td>19 (21.3) </td><td>1 </td><td>40 </td><td>2 </td><td>37 </td><td>1 </td><td>24 </td><td>0 </td><td>0 </td><td>1 </td><td>11 </td></tr><tr><td>3C </td><td>51 </td><td>70.8 </td><td>1.39 </td><td>13 (25.4) </td><td>1 </td><td>36 </td><td>0 </td><td>0 </td><td>1 </td><td>13 </td><td>1 </td><td>11 </td><td>0 </td><td>0 </td></tr><tr><td>3D </td><td>38 </td><td>65.8 </td><td>1.73 </td><td>5 (13.1) </td><td>1 </td><td>30 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>1 </td><td>9 </td><td>1 </td><td>3 </td></tr><tr><td>4A </td><td>88 </td><td>112.2 </td><td>1.28 </td><td>31 (35.2) </td><td>1 </td><td>24 </td><td>1 </td><td>45 </td><td>2 </td><td>19 </td><td>0 </td><td>0 </td><td>2 </td><td>42 </td></tr><tr><td>4B </td><td>18 </td><td>91.4 </td><td>5.08 </td><td>2 (11.1) </td><td>1 </td><td>3 </td><td>1 </td><td>5 </td><td>1 </td><td>13 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td></tr><tr><td>4C </td><td>64 </td><td>87.9 </td><td>1.37 </td><td>6 (9.37) </td><td>2 </td><td>32 </td><td>2 </td><td>16 </td><td>1 </td><td>12 </td><td>1 </td><td>15 </td><td>1 </td><td>13 </td></tr><tr><td>4D </td><td>45 </td><td>53.0 </td><td>1.18 </td><td>7 (15.5) </td><td>1 </td><td>27 </td><td>0 </td><td>0 </td><td>1 </td><td>13 </td><td>2 </td><td>14 </td><td>1 </td><td>5 </td></tr><tr><td>5A </td><td>130 </td><td>119.0 </td><td>0.92 </td><td>40 (30.7) </td><td>2 </td><td>25 </td><td>2 </td><td>53 </td><td>2 </td><td>77 </td><td>2 </td><td>9 </td><td>1 </td><td>9 </td></tr><tr><td>5B </td><td>57 </td><td>79.3 </td><td>1.39 </td><td>13 (22.8) </td><td>1 </td><td>35 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>1 </td><td>26 </td><td>2 </td><td>15 </td></tr><tr><td>5C </td><td>18 </td><td>74.8 </td><td>4.16 </td><td>3 (16.6) </td><td>0 </td><td>0 </td><td>1 </td><td>12 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>1 </td><td>10 </td></tr><tr><td>5D </td><td>23 </td><td>46.0 </td><td>2.00 </td><td>5 (21.7) </td><td>1 </td><td>10 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>1 </td><td>15 </td><td>0 </td><td>0 </td></tr><tr><td>6A </td><td>131 </td><td>109.8 </td><td>0.84 </td><td>31 (23.6) </td><td>3 </td><td>37 </td><td>4 </td><td>37 </td><td>2 </td><td>44 </td><td>3 </td><td>16 </td><td>4 </td><td>49 </td></tr><tr><td>6B </td><td>129 </td><td>87.9 </td><td>0.68 </td><td>32 (24.8) </td><td>2 </td><td>86 </td><td>1 </td><td>29 </td><td>2 </td><td>31 </td><td>2 </td><td>15 </td><td>2 </td><td>9 </td></tr><tr><td>6C </td><td>59 </td><td>87.4 </td><td>1.48 </td><td>11 (18.6) </td><td>1 </td><td>43 </td><td>1 </td><td>4 </td><td>1 </td><td>18 </td><td>1 </td><td>6 </td><td>0 </td><td>0 </td></tr><tr><td>6D </td><td>49 </td><td>76.4 </td><td>1.56 </td><td>10 (20.4) </td><td>1 </td><td>11 </td><td>0 </td><td>0 </td><td>1 </td><td>15 </td><td>2 </td><td>24 </td><td>2 </td><td>23 </td></tr><tr><td>7A </td><td>80 </td><td>115.7 </td><td>1.45 </td><td>24 (30.0) </td><td>1 </td><td>27 </td><td>2 </td><td>25 </td><td>1 </td><td>33 </td><td>1 </td><td>17 </td><td>1 </td><td>24 </td></tr><tr><td>7B </td><td>37 </td><td>72.4 </td><td>1.96 </td><td>11 (29.7) </td><td>1 </td><td>30 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>1 </td><td>5 </td><td>1 </td><td>9 </td></tr><tr><td>7C </td><td>24 </td><td>68.6 </td><td>2.86 </td><td>6 (25.0) </td><td>1 </td><td>14 </td><td>0 </td><td>0 </td><td>2 </td><td>15 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td></tr><tr><td>7D </td><td>66 </td><td>67.3 </td><td>1.02 </td><td>20 (30.3) </td><td>1 </td><td>21 </td><td>2 </td><td>35 </td><td>2 </td><td>14 </td><td>1 </td><td>3 </td><td>2 </td><td>14 </td></tr><tr><td>Unintegrated </td><td>— </td><td>— </td><td>— </td><td>— </td><td>1 </td><td>5 </td><td>3 </td><td>20 </td><td>4 </td><td>24 </td><td>4 </td><td>14 </td><td>5 </td><td>17 </td></tr><tr><td>Total </td><td>1856 </td><td>2364.1 </td><td>1.27 </td><td>481 (25.9) </td><td>34 </td><td>822 </td><td>32 </td><td>576 </td><td>34 </td><td>556 </td><td>32 </td><td>294 </td><td>33 </td><td>318 </td></tr></tbody></table></div><div content-id="DSS035TB2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="3">LG </th><th colspan="4" rowspan="2">Interspecific map<hr /> </th><th colspan="10">Parental specific maps<hr /> </th></tr><tr><th colspan="2">‘0212921’<hr /> </th><th colspan="2">‘02–19’<hr /> </th><th colspan="2">‘Sachinoka’<hr /> </th><th colspan="2">‘Kaorino’<hr /> </th><th colspan="2">‘Akihime’<hr /> </th></tr><tr><th>Number of mapped loci </th><th>Length (cM) </th><th>Locus density (cM) </th><th>Single loci<sup>a</sup> (%) </th><th>LGs<sup>b</sup> </th><th>Loci<sup>c</sup> </th><th>LGs<sup>b</sup> </th><th>Loci<sup>c</sup> </th><th>LGs<sup>b</sup> </th><th>Loci<sup>c</sup> </th><th>LGs<sup>b</sup> </th><th>Loci<sup>c</sup> </th><th>LGs<sup>b</sup> </th><th>Loci<sup>c</sup> </th></tr></thead><tbody><tr><td>1A </td><td>91 </td><td>123.7 </td><td>1.36 </td><td>16 (17.5) </td><td>1 </td><td>48 </td><td>1 </td><td>40 </td><td>1 </td><td>13 </td><td>1 </td><td>7 </td><td>2 </td><td>10 </td></tr><tr><td>1B </td><td>81 </td><td>80.7 </td><td>1.00 </td><td>25 (30.8) </td><td>1 </td><td>25 </td><td>1 </td><td>41 </td><td>2 </td><td>42 </td><td>2 </td><td>14 </td><td>0 </td><td>0 </td></tr><tr><td>1C </td><td>37 </td><td>77.2 </td><td>2.09 </td><td>11 (29.7) </td><td>1 </td><td>24 </td><td>1 </td><td>9 </td><td>1 </td><td>13 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td></tr><tr><td>1D </td><td>24 </td><td>34.2 </td><td>1.43 </td><td>5 (20.8) </td><td>1 </td><td>24 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td></tr><tr><td>2A </td><td>123 </td><td>113.9 </td><td>0.93 </td><td>41 (33.3) </td><td>2 </td><td>25 </td><td>2 </td><td>50 </td><td>1 </td><td>67 </td><td>1 </td><td>8 </td><td>2 </td><td>22 </td></tr><tr><td>2B </td><td>71 </td><td>95.1 </td><td>1.34 </td><td>15 (21.1) </td><td>1 </td><td>42 </td><td>2 </td><td>20 </td><td>1 </td><td>4 </td><td>1 </td><td>11 </td><td>1 </td><td>15 </td></tr><tr><td>2C </td><td>55 </td><td>93.8 </td><td>1.71 </td><td>13 (23.6) </td><td>1 </td><td>28 </td><td>0 </td><td>0 </td><td>1 </td><td>18 </td><td>1 </td><td>24 </td><td>0 </td><td>0 </td></tr><tr><td>2D </td><td>64 </td><td>81.1 </td><td>1.27 </td><td>19 (29.6) </td><td>1 </td><td>37 </td><td>1 </td><td>28 </td><td>1 </td><td>7 </td><td>1 </td><td>8 </td><td>0 </td><td>0 </td></tr><tr><td>3A </td><td>114 </td><td>96.0 </td><td>0.84 </td><td>47 (41.2) </td><td>1 </td><td>33 </td><td>2 </td><td>70 </td><td>2 </td><td>27 </td><td>1 </td><td>23 </td><td>1 </td><td>18 </td></tr><tr><td>3B </td><td>89 </td><td>82.7 </td><td>0.93 </td><td>19 (21.3) </td><td>1 </td><td>40 </td><td>2 </td><td>37 </td><td>1 </td><td>24 </td><td>0 </td><td>0 </td><td>1 </td><td>11 </td></tr><tr><td>3C </td><td>51 </td><td>70.8 </td><td>1.39 </td><td>13 (25.4) </td><td>1 </td><td>36 </td><td>0 </td><td>0 </td><td>1 </td><td>13 </td><td>1 </td><td>11 </td><td>0 </td><td>0 </td></tr><tr><td>3D </td><td>38 </td><td>65.8 </td><td>1.73 </td><td>5 (13.1) </td><td>1 </td><td>30 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>1 </td><td>9 </td><td>1 </td><td>3 </td></tr><tr><td>4A </td><td>88 </td><td>112.2 </td><td>1.28 </td><td>31 (35.2) </td><td>1 </td><td>24 </td><td>1 </td><td>45 </td><td>2 </td><td>19 </td><td>0 </td><td>0 </td><td>2 </td><td>42 </td></tr><tr><td>4B </td><td>18 </td><td>91.4 </td><td>5.08 </td><td>2 (11.1) </td><td>1 </td><td>3 </td><td>1 </td><td>5 </td><td>1 </td><td>13 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td></tr><tr><td>4C </td><td>64 </td><td>87.9 </td><td>1.37 </td><td>6 (9.37) </td><td>2 </td><td>32 </td><td>2 </td><td>16 </td><td>1 </td><td>12 </td><td>1 </td><td>15 </td><td>1 </td><td>13 </td></tr><tr><td>4D </td><td>45 </td><td>53.0 </td><td>1.18 </td><td>7 (15.5) </td><td>1 </td><td>27 </td><td>0 </td><td>0 </td><td>1 </td><td>13 </td><td>2 </td><td>14 </td><td>1 </td><td>5 </td></tr><tr><td>5A </td><td>130 </td><td>119.0 </td><td>0.92 </td><td>40 (30.7) </td><td>2 </td><td>25 </td><td>2 </td><td>53 </td><td>2 </td><td>77 </td><td>2 </td><td>9 </td><td>1 </td><td>9 </td></tr><tr><td>5B </td><td>57 </td><td>79.3 </td><td>1.39 </td><td>13 (22.8) </td><td>1 </td><td>35 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>1 </td><td>26 </td><td>2 </td><td>15 </td></tr><tr><td>5C </td><td>18 </td><td>74.8 </td><td>4.16 </td><td>3 (16.6) </td><td>0 </td><td>0 </td><td>1 </td><td>12 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>1 </td><td>10 </td></tr><tr><td>5D </td><td>23 </td><td>46.0 </td><td>2.00 </td><td>5 (21.7) </td><td>1 </td><td>10 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>1 </td><td>15 </td><td>0 </td><td>0 </td></tr><tr><td>6A </td><td>131 </td><td>109.8 </td><td>0.84 </td><td>31 (23.6) </td><td>3 </td><td>37 </td><td>4 </td><td>37 </td><td>2 </td><td>44 </td><td>3 </td><td>16 </td><td>4 </td><td>49 </td></tr><tr><td>6B </td><td>129 </td><td>87.9 </td><td>0.68 </td><td>32 (24.8) </td><td>2 </td><td>86 </td><td>1 </td><td>29 </td><td>2 </td><td>31 </td><td>2 </td><td>15 </td><td>2 </td><td>9 </td></tr><tr><td>6C </td><td>59 </td><td>87.4 </td><td>1.48 </td><td>11 (18.6) </td><td>1 </td><td>43 </td><td>1 </td><td>4 </td><td>1 </td><td>18 </td><td>1 </td><td>6 </td><td>0 </td><td>0 </td></tr><tr><td>6D </td><td>49 </td><td>76.4 </td><td>1.56 </td><td>10 (20.4) </td><td>1 </td><td>11 </td><td>0 </td><td>0 </td><td>1 </td><td>15 </td><td>2 </td><td>24 </td><td>2 </td><td>23 </td></tr><tr><td>7A </td><td>80 </td><td>115.7 </td><td>1.45 </td><td>24 (30.0) </td><td>1 </td><td>27 </td><td>2 </td><td>25 </td><td>1 </td><td>33 </td><td>1 </td><td>17 </td><td>1 </td><td>24 </td></tr><tr><td>7B </td><td>37 </td><td>72.4 </td><td>1.96 </td><td>11 (29.7) </td><td>1 </td><td>30 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>1 </td><td>5 </td><td>1 </td><td>9 </td></tr><tr><td>7C </td><td>24 </td><td>68.6 </td><td>2.86 </td><td>6 (25.0) </td><td>1 </td><td>14 </td><td>0 </td><td>0 </td><td>2 </td><td>15 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td></tr><tr><td>7D </td><td>66 </td><td>67.3 </td><td>1.02 </td><td>20 (30.3) </td><td>1 </td><td>21 </td><td>2 </td><td>35 </td><td>2 </td><td>14 </td><td>1 </td><td>3 </td><td>2 </td><td>14 </td></tr><tr><td>Unintegrated </td><td>— </td><td>— </td><td>— </td><td>— </td><td>1 </td><td>5 </td><td>3 </td><td>20 </td><td>4 </td><td>24 </td><td>4 </td><td>14 </td><td>5 </td><td>17 </td></tr><tr><td>Total </td><td>1856 </td><td>2364.1 </td><td>1.27 </td><td>481 (25.9) </td><td>34 </td><td>822 </td><td>32 </td><td>576 </td><td>34 </td><td>556 </td><td>32 </td><td>294 </td><td>33 </td><td>318 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Number of mapped loci generated from SLD markers. Numbers in parentheses show percentage of all the mapped loci.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>Number of integrated LGs.</p></div></div><div content-id=""><div class="fn"><p><sup>c</sup>Number of integrated loci.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5643923" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSS035TB2" class="table-wrap-title" data-id="DSS035TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Number of mapped loci, length, locus density in an integrated map, and numbers of integrated LGs and loci of parental specific maps</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="3">LG </th><th colspan="4" rowspan="2">Interspecific map<hr /> </th><th colspan="10">Parental specific maps<hr /> </th></tr><tr><th colspan="2">‘0212921’<hr /> </th><th colspan="2">‘02–19’<hr /> </th><th colspan="2">‘Sachinoka’<hr /> </th><th colspan="2">‘Kaorino’<hr /> </th><th colspan="2">‘Akihime’<hr /> </th></tr><tr><th>Number of mapped loci </th><th>Length (cM) </th><th>Locus density (cM) </th><th>Single loci<sup>a</sup> (%) </th><th>LGs<sup>b</sup> </th><th>Loci<sup>c</sup> </th><th>LGs<sup>b</sup> </th><th>Loci<sup>c</sup> </th><th>LGs<sup>b</sup> </th><th>Loci<sup>c</sup> </th><th>LGs<sup>b</sup> </th><th>Loci<sup>c</sup> </th><th>LGs<sup>b</sup> </th><th>Loci<sup>c</sup> </th></tr></thead><tbody><tr><td>1A </td><td>91 </td><td>123.7 </td><td>1.36 </td><td>16 (17.5) </td><td>1 </td><td>48 </td><td>1 </td><td>40 </td><td>1 </td><td>13 </td><td>1 </td><td>7 </td><td>2 </td><td>10 </td></tr><tr><td>1B </td><td>81 </td><td>80.7 </td><td>1.00 </td><td>25 (30.8) </td><td>1 </td><td>25 </td><td>1 </td><td>41 </td><td>2 </td><td>42 </td><td>2 </td><td>14 </td><td>0 </td><td>0 </td></tr><tr><td>1C </td><td>37 </td><td>77.2 </td><td>2.09 </td><td>11 (29.7) </td><td>1 </td><td>24 </td><td>1 </td><td>9 </td><td>1 </td><td>13 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td></tr><tr><td>1D </td><td>24 </td><td>34.2 </td><td>1.43 </td><td>5 (20.8) </td><td>1 </td><td>24 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td></tr><tr><td>2A </td><td>123 </td><td>113.9 </td><td>0.93 </td><td>41 (33.3) </td><td>2 </td><td>25 </td><td>2 </td><td>50 </td><td>1 </td><td>67 </td><td>1 </td><td>8 </td><td>2 </td><td>22 </td></tr><tr><td>2B </td><td>71 </td><td>95.1 </td><td>1.34 </td><td>15 (21.1) </td><td>1 </td><td>42 </td><td>2 </td><td>20 </td><td>1 </td><td>4 </td><td>1 </td><td>11 </td><td>1 </td><td>15 </td></tr><tr><td>2C </td><td>55 </td><td>93.8 </td><td>1.71 </td><td>13 (23.6) </td><td>1 </td><td>28 </td><td>0 </td><td>0 </td><td>1 </td><td>18 </td><td>1 </td><td>24 </td><td>0 </td><td>0 </td></tr><tr><td>2D </td><td>64 </td><td>81.1 </td><td>1.27 </td><td>19 (29.6) </td><td>1 </td><td>37 </td><td>1 </td><td>28 </td><td>1 </td><td>7 </td><td>1 </td><td>8 </td><td>0 </td><td>0 </td></tr><tr><td>3A </td><td>114 </td><td>96.0 </td><td>0.84 </td><td>47 (41.2) </td><td>1 </td><td>33 </td><td>2 </td><td>70 </td><td>2 </td><td>27 </td><td>1 </td><td>23 </td><td>1 </td><td>18 </td></tr><tr><td>3B </td><td>89 </td><td>82.7 </td><td>0.93 </td><td>19 (21.3) </td><td>1 </td><td>40 </td><td>2 </td><td>37 </td><td>1 </td><td>24 </td><td>0 </td><td>0 </td><td>1 </td><td>11 </td></tr><tr><td>3C </td><td>51 </td><td>70.8 </td><td>1.39 </td><td>13 (25.4) </td><td>1 </td><td>36 </td><td>0 </td><td>0 </td><td>1 </td><td>13 </td><td>1 </td><td>11 </td><td>0 </td><td>0 </td></tr><tr><td>3D </td><td>38 </td><td>65.8 </td><td>1.73 </td><td>5 (13.1) </td><td>1 </td><td>30 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>1 </td><td>9 </td><td>1 </td><td>3 </td></tr><tr><td>4A </td><td>88 </td><td>112.2 </td><td>1.28 </td><td>31 (35.2) </td><td>1 </td><td>24 </td><td>1 </td><td>45 </td><td>2 </td><td>19 </td><td>0 </td><td>0 </td><td>2 </td><td>42 </td></tr><tr><td>4B </td><td>18 </td><td>91.4 </td><td>5.08 </td><td>2 (11.1) </td><td>1 </td><td>3 </td><td>1 </td><td>5 </td><td>1 </td><td>13 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td></tr><tr><td>4C </td><td>64 </td><td>87.9 </td><td>1.37 </td><td>6 (9.37) </td><td>2 </td><td>32 </td><td>2 </td><td>16 </td><td>1 </td><td>12 </td><td>1 </td><td>15 </td><td>1 </td><td>13 </td></tr><tr><td>4D </td><td>45 </td><td>53.0 </td><td>1.18 </td><td>7 (15.5) </td><td>1 </td><td>27 </td><td>0 </td><td>0 </td><td>1 </td><td>13 </td><td>2 </td><td>14 </td><td>1 </td><td>5 </td></tr><tr><td>5A </td><td>130 </td><td>119.0 </td><td>0.92 </td><td>40 (30.7) </td><td>2 </td><td>25 </td><td>2 </td><td>53 </td><td>2 </td><td>77 </td><td>2 </td><td>9 </td><td>1 </td><td>9 </td></tr><tr><td>5B </td><td>57 </td><td>79.3 </td><td>1.39 </td><td>13 (22.8) </td><td>1 </td><td>35 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>1 </td><td>26 </td><td>2 </td><td>15 </td></tr><tr><td>5C </td><td>18 </td><td>74.8 </td><td>4.16 </td><td>3 (16.6) </td><td>0 </td><td>0 </td><td>1 </td><td>12 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>1 </td><td>10 </td></tr><tr><td>5D </td><td>23 </td><td>46.0 </td><td>2.00 </td><td>5 (21.7) </td><td>1 </td><td>10 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>1 </td><td>15 </td><td>0 </td><td>0 </td></tr><tr><td>6A </td><td>131 </td><td>109.8 </td><td>0.84 </td><td>31 (23.6) </td><td>3 </td><td>37 </td><td>4 </td><td>37 </td><td>2 </td><td>44 </td><td>3 </td><td>16 </td><td>4 </td><td>49 </td></tr><tr><td>6B </td><td>129 </td><td>87.9 </td><td>0.68 </td><td>32 (24.8) </td><td>2 </td><td>86 </td><td>1 </td><td>29 </td><td>2 </td><td>31 </td><td>2 </td><td>15 </td><td>2 </td><td>9 </td></tr><tr><td>6C </td><td>59 </td><td>87.4 </td><td>1.48 </td><td>11 (18.6) </td><td>1 </td><td>43 </td><td>1 </td><td>4 </td><td>1 </td><td>18 </td><td>1 </td><td>6 </td><td>0 </td><td>0 </td></tr><tr><td>6D </td><td>49 </td><td>76.4 </td><td>1.56 </td><td>10 (20.4) </td><td>1 </td><td>11 </td><td>0 </td><td>0 </td><td>1 </td><td>15 </td><td>2 </td><td>24 </td><td>2 </td><td>23 </td></tr><tr><td>7A </td><td>80 </td><td>115.7 </td><td>1.45 </td><td>24 (30.0) </td><td>1 </td><td>27 </td><td>2 </td><td>25 </td><td>1 </td><td>33 </td><td>1 </td><td>17 </td><td>1 </td><td>24 </td></tr><tr><td>7B </td><td>37 </td><td>72.4 </td><td>1.96 </td><td>11 (29.7) </td><td>1 </td><td>30 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>1 </td><td>5 </td><td>1 </td><td>9 </td></tr><tr><td>7C </td><td>24 </td><td>68.6 </td><td>2.86 </td><td>6 (25.0) </td><td>1 </td><td>14 </td><td>0 </td><td>0 </td><td>2 </td><td>15 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td></tr><tr><td>7D </td><td>66 </td><td>67.3 </td><td>1.02 </td><td>20 (30.3) </td><td>1 </td><td>21 </td><td>2 </td><td>35 </td><td>2 </td><td>14 </td><td>1 </td><td>3 </td><td>2 </td><td>14 </td></tr><tr><td>Unintegrated </td><td>— </td><td>— </td><td>— </td><td>— </td><td>1 </td><td>5 </td><td>3 </td><td>20 </td><td>4 </td><td>24 </td><td>4 </td><td>14 </td><td>5 </td><td>17 </td></tr><tr><td>Total </td><td>1856 </td><td>2364.1 </td><td>1.27 </td><td>481 (25.9) </td><td>34 </td><td>822 </td><td>32 </td><td>576 </td><td>34 </td><td>556 </td><td>32 </td><td>294 </td><td>33 </td><td>318 </td></tr></tbody></table></div><div content-id="DSS035TB2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="3">LG </th><th colspan="4" rowspan="2">Interspecific map<hr /> </th><th colspan="10">Parental specific maps<hr /> </th></tr><tr><th colspan="2">‘0212921’<hr /> </th><th colspan="2">‘02–19’<hr /> </th><th colspan="2">‘Sachinoka’<hr /> </th><th colspan="2">‘Kaorino’<hr /> </th><th colspan="2">‘Akihime’<hr /> </th></tr><tr><th>Number of mapped loci </th><th>Length (cM) </th><th>Locus density (cM) </th><th>Single loci<sup>a</sup> (%) </th><th>LGs<sup>b</sup> </th><th>Loci<sup>c</sup> </th><th>LGs<sup>b</sup> </th><th>Loci<sup>c</sup> </th><th>LGs<sup>b</sup> </th><th>Loci<sup>c</sup> </th><th>LGs<sup>b</sup> </th><th>Loci<sup>c</sup> </th><th>LGs<sup>b</sup> </th><th>Loci<sup>c</sup> </th></tr></thead><tbody><tr><td>1A </td><td>91 </td><td>123.7 </td><td>1.36 </td><td>16 (17.5) </td><td>1 </td><td>48 </td><td>1 </td><td>40 </td><td>1 </td><td>13 </td><td>1 </td><td>7 </td><td>2 </td><td>10 </td></tr><tr><td>1B </td><td>81 </td><td>80.7 </td><td>1.00 </td><td>25 (30.8) </td><td>1 </td><td>25 </td><td>1 </td><td>41 </td><td>2 </td><td>42 </td><td>2 </td><td>14 </td><td>0 </td><td>0 </td></tr><tr><td>1C </td><td>37 </td><td>77.2 </td><td>2.09 </td><td>11 (29.7) </td><td>1 </td><td>24 </td><td>1 </td><td>9 </td><td>1 </td><td>13 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td></tr><tr><td>1D </td><td>24 </td><td>34.2 </td><td>1.43 </td><td>5 (20.8) </td><td>1 </td><td>24 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td></tr><tr><td>2A </td><td>123 </td><td>113.9 </td><td>0.93 </td><td>41 (33.3) </td><td>2 </td><td>25 </td><td>2 </td><td>50 </td><td>1 </td><td>67 </td><td>1 </td><td>8 </td><td>2 </td><td>22 </td></tr><tr><td>2B </td><td>71 </td><td>95.1 </td><td>1.34 </td><td>15 (21.1) </td><td>1 </td><td>42 </td><td>2 </td><td>20 </td><td>1 </td><td>4 </td><td>1 </td><td>11 </td><td>1 </td><td>15 </td></tr><tr><td>2C </td><td>55 </td><td>93.8 </td><td>1.71 </td><td>13 (23.6) </td><td>1 </td><td>28 </td><td>0 </td><td>0 </td><td>1 </td><td>18 </td><td>1 </td><td>24 </td><td>0 </td><td>0 </td></tr><tr><td>2D </td><td>64 </td><td>81.1 </td><td>1.27 </td><td>19 (29.6) </td><td>1 </td><td>37 </td><td>1 </td><td>28 </td><td>1 </td><td>7 </td><td>1 </td><td>8 </td><td>0 </td><td>0 </td></tr><tr><td>3A </td><td>114 </td><td>96.0 </td><td>0.84 </td><td>47 (41.2) </td><td>1 </td><td>33 </td><td>2 </td><td>70 </td><td>2 </td><td>27 </td><td>1 </td><td>23 </td><td>1 </td><td>18 </td></tr><tr><td>3B </td><td>89 </td><td>82.7 </td><td>0.93 </td><td>19 (21.3) </td><td>1 </td><td>40 </td><td>2 </td><td>37 </td><td>1 </td><td>24 </td><td>0 </td><td>0 </td><td>1 </td><td>11 </td></tr><tr><td>3C </td><td>51 </td><td>70.8 </td><td>1.39 </td><td>13 (25.4) </td><td>1 </td><td>36 </td><td>0 </td><td>0 </td><td>1 </td><td>13 </td><td>1 </td><td>11 </td><td>0 </td><td>0 </td></tr><tr><td>3D </td><td>38 </td><td>65.8 </td><td>1.73 </td><td>5 (13.1) </td><td>1 </td><td>30 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>1 </td><td>9 </td><td>1 </td><td>3 </td></tr><tr><td>4A </td><td>88 </td><td>112.2 </td><td>1.28 </td><td>31 (35.2) </td><td>1 </td><td>24 </td><td>1 </td><td>45 </td><td>2 </td><td>19 </td><td>0 </td><td>0 </td><td>2 </td><td>42 </td></tr><tr><td>4B </td><td>18 </td><td>91.4 </td><td>5.08 </td><td>2 (11.1) </td><td>1 </td><td>3 </td><td>1 </td><td>5 </td><td>1 </td><td>13 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td></tr><tr><td>4C </td><td>64 </td><td>87.9 </td><td>1.37 </td><td>6 (9.37) </td><td>2 </td><td>32 </td><td>2 </td><td>16 </td><td>1 </td><td>12 </td><td>1 </td><td>15 </td><td>1 </td><td>13 </td></tr><tr><td>4D </td><td>45 </td><td>53.0 </td><td>1.18 </td><td>7 (15.5) </td><td>1 </td><td>27 </td><td>0 </td><td>0 </td><td>1 </td><td>13 </td><td>2 </td><td>14 </td><td>1 </td><td>5 </td></tr><tr><td>5A </td><td>130 </td><td>119.0 </td><td>0.92 </td><td>40 (30.7) </td><td>2 </td><td>25 </td><td>2 </td><td>53 </td><td>2 </td><td>77 </td><td>2 </td><td>9 </td><td>1 </td><td>9 </td></tr><tr><td>5B </td><td>57 </td><td>79.3 </td><td>1.39 </td><td>13 (22.8) </td><td>1 </td><td>35 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>1 </td><td>26 </td><td>2 </td><td>15 </td></tr><tr><td>5C </td><td>18 </td><td>74.8 </td><td>4.16 </td><td>3 (16.6) </td><td>0 </td><td>0 </td><td>1 </td><td>12 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>1 </td><td>10 </td></tr><tr><td>5D </td><td>23 </td><td>46.0 </td><td>2.00 </td><td>5 (21.7) </td><td>1 </td><td>10 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>1 </td><td>15 </td><td>0 </td><td>0 </td></tr><tr><td>6A </td><td>131 </td><td>109.8 </td><td>0.84 </td><td>31 (23.6) </td><td>3 </td><td>37 </td><td>4 </td><td>37 </td><td>2 </td><td>44 </td><td>3 </td><td>16 </td><td>4 </td><td>49 </td></tr><tr><td>6B </td><td>129 </td><td>87.9 </td><td>0.68 </td><td>32 (24.8) </td><td>2 </td><td>86 </td><td>1 </td><td>29 </td><td>2 </td><td>31 </td><td>2 </td><td>15 </td><td>2 </td><td>9 </td></tr><tr><td>6C </td><td>59 </td><td>87.4 </td><td>1.48 </td><td>11 (18.6) </td><td>1 </td><td>43 </td><td>1 </td><td>4 </td><td>1 </td><td>18 </td><td>1 </td><td>6 </td><td>0 </td><td>0 </td></tr><tr><td>6D </td><td>49 </td><td>76.4 </td><td>1.56 </td><td>10 (20.4) </td><td>1 </td><td>11 </td><td>0 </td><td>0 </td><td>1 </td><td>15 </td><td>2 </td><td>24 </td><td>2 </td><td>23 </td></tr><tr><td>7A </td><td>80 </td><td>115.7 </td><td>1.45 </td><td>24 (30.0) </td><td>1 </td><td>27 </td><td>2 </td><td>25 </td><td>1 </td><td>33 </td><td>1 </td><td>17 </td><td>1 </td><td>24 </td></tr><tr><td>7B </td><td>37 </td><td>72.4 </td><td>1.96 </td><td>11 (29.7) </td><td>1 </td><td>30 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>1 </td><td>5 </td><td>1 </td><td>9 </td></tr><tr><td>7C </td><td>24 </td><td>68.6 </td><td>2.86 </td><td>6 (25.0) </td><td>1 </td><td>14 </td><td>0 </td><td>0 </td><td>2 </td><td>15 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td></tr><tr><td>7D </td><td>66 </td><td>67.3 </td><td>1.02 </td><td>20 (30.3) </td><td>1 </td><td>21 </td><td>2 </td><td>35 </td><td>2 </td><td>14 </td><td>1 </td><td>3 </td><td>2 </td><td>14 </td></tr><tr><td>Unintegrated </td><td>— </td><td>— </td><td>— </td><td>— </td><td>1 </td><td>5 </td><td>3 </td><td>20 </td><td>4 </td><td>24 </td><td>4 </td><td>14 </td><td>5 </td><td>17 </td></tr><tr><td>Total </td><td>1856 </td><td>2364.1 </td><td>1.27 </td><td>481 (25.9) </td><td>34 </td><td>822 </td><td>32 </td><td>576 </td><td>34 </td><td>556 </td><td>32 </td><td>294 </td><td>33 </td><td>318 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Number of mapped loci generated from SLD markers. Numbers in parentheses show percentage of all the mapped loci.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>Number of integrated LGs.</p></div></div><div content-id=""><div class="fn"><p><sup>c</sup>Number of integrated loci.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5643923" target="_blank">View Large</a></div></div><p>Of the mapped loci, 481 (25.9%) were generated from SLD markers, whereas the other 1375 were from MLD markers. The ratio of mapped loci generated from SLD markers in each LG varied from 9.4% (LG4C) to 41.2% (LG3A). These loci were mapped randomly onto many of the LGs while several clusters were observed on parts of LGs (LG1A, 1C, 2C, 4C, 4D, 6C, 6D, 7B, and 7C, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). Loci generated from MLD markers were classified into four types; multi loci mapped onto homoeologous LGs (Multi_H in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S7</a>), non-homeologous LGs (Multi_NH), the same LGs (Multi_S), and single position (Multi_NM). For the Multi_NM loci, all the observed multiple bands were monomorphic except for mapped ones. Of the four types of multi loci, Multi_H was most frequently observed (684 loci), followed by Multi_NM (523), Multi_S (206), and Multi_NH (77). Of the 684 Multi_H loci, 16 and 99 were observed, corresponding to loci on non-homeologous LGs (Multi_H&amp;NH) and the same LGs (Multi_H&amp;S), respectively. The Multi_H loci were randomly distributed along the entire integrated linkage map, whereas Multi_S loci were not observed on several LGs, i.e. LG2B, 2D, 3C, 4D, 5A, 5B, 7B, and 7C (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). In each HG, the homeologous regions differed depending on the paired LGs (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S3</a>). For example, most homeologous regions of LG2A and LG2C were observed on 20–40 cM while those of LG2B and LG2D were identified on 60–80 cM. In the case of HG1, homeologous regions were not observed between LG1C and LG1D. Such biases of homeologous pairs were identified across the genome.</p>                            <h3 scrollto-destination=5643925 id="5643925" class="section-title" data-legacy-id=s3e>3.5. Comparison with the genomes of a wild relative, <em>F. vesca</em></h3>
<p>Of the 3746 ESTs that corresponded to the designed FVES markers, 3743 showed significant similarities to the genome sequences of <em>F. vesca</em>, while 3 ESTs, corresponding to FVES1248, FVES2637, and FVES2807, did not (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). Of the 3743 ESTs, 3608 showed similarities to genome sequences placed on 7 chromosomes of <em>F. vesca</em>, and the other 135 were mapped onto unplaced genomic scaffolds. In <em>F.</em> × <em>ananassa</em> sequence-derived markers, significant similarities to the <em>F. vesca</em> genome sequences were observed in all 603 ESTs and 124 of the 125 transcript contigs from which FAES and FATS markers were designed, respectively (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4 and S5</a>). Of these, 593 and 120 sequences corresponding to FAES and FATS markers, respectively, were mapped onto <em>F. vesca</em> genome sequences placed on the 7 chromosomes, whereas the other 14 ESTs or transcript contigs were mapped onto unplaced scaffolds. Similarity searches were also performed for the previously published markers that were located on the integrated map. Of the 91 mapped previously published markers, 68 ESTs or genome sequence are available on the NCBI dbGSS database (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/projects/dbGSS">http://www.ncbi.nlm.nih.gov/projects/dbGSS</a>, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>) from which the markers were designed. Of the 68 sequences, 62 showed significant similarities to the <em>F. vesca</em> genome sequences placed onto the 7 chromosomes.</p><p>A total of 1354 markers that generated 1783 mapped loci onto the integrated linkage map showed significant similarity with <em>F. vesca</em> genome sequences placed onto the 7 chromosomes. By considering the sequences with highest similarity scores to be putative orthologs, the map locations of the SSR markers and the corresponding <em>F. vesca</em> genome sequences were compared. As shown in Fig. <a reveal-id="DSS035F1" data-open="DSS035F1" class="link link-reveal link-table xref-fig">1</a>, the alignment of homologous sequence pairs along each LG revealed an obvious syntenic relationship between corresponding HGs in <em>F.</em> × <em>ananassa</em> (Fa-HG) and chromosomes (Chr) in <em>F. vesca</em> (Fv-Chr). Most of the regions did not show synteny between non-corresponding Fa-HGs and Fv-Chr, except between Fa-HG3 and 25–30 Mb on Fv-Chr2. In some LGs in <em>F.</em> × <em>ananassa</em>, syntenic regions were observed on whole corresponding chromosomes in <em>F. vesca</em>, whereas segmental syntenic blocks were identified between pairs of the other Fa-LGs and Fv-Chr (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S3-1</a>). For example, Fa-LG1A displayed syntenies against entire regions of Fv-Chr1, whereas segmental syntenic blocks were observed between 0 and 11 Mb in Fv-Chr1 and Fa-LG1B or Fa-LG1D, as well as between 8 and 23 Mb in Fv-Chr1 and Fa-LG1C. </p>                        <a id="5643928" scrollto-destination="5643928"></a>
<div id="DSS035F1" data-id="DSS035F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss035/2/m_dss03501.jpeg?Expires=2147483647&Signature=H2hkpA7M11oiUy-ZiFPHct2tvlpHvGCa-F1fd3uDc1n7l9rh1UOCRINWpMDitpbu3PS8uYXtlLwt56O0Lz6bEV4rLtMTqzrWiGa1ZUzIBsEbB1T9P1obsVlW0oQI3mBo8BdbDSFHNnYNw2lRxE7UrdvWHRojlxmBzCSJoZUq60NHKLaLVeQEyYUlFLeGsabI8mVatYKERHLiVFGSU8ftjyKUVY4dqFg~1BKEZgnAit0NsgaPl7lKQJ9wORCBGsWgsfd7QzGb7zxVfwf56Iek7PwikUGzIH-vnpgz7Le6wedr2~M2O8QaW5VywQwJM0Lm2VaUMHs6NY7JIXmdQhCsdw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Graphical view of syntenic relationship between F. × ananassa and F. vesca. Red and blue bars show LGs of F. × ananassa and chromosomes of F. vesca, respectively. Syntenic regions between the two species are connected by coloured lines." path-from-xml="dss03501.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5643928/dss03501.jpeg" path-from-xml="dss03501.jpeg" target="_blank">View large</a><a section="5643928" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss035/2/dss03501.jpeg?Expires=2147483647&Signature=1TIqiKsCZcylNolCO3uQUV-TxJfyA8~LfcJh919ouN0685ztMj0EnMaGQwUU4cB8eFsEsvMSORATDoDKSawmoQZ-AP8JngtQ3BJ81x-ftBgHGgtysV78mR7d4-5NAVhuSmwyyP73bwGST8uaQqd4~~6fkdt8fxx5-wP5kMvbvzz6Dc9VqlbpCK1CgbJIjiU~WzrpdG38dSlk0KBSnCCvCxVDEpyJL-xTpH0VyaOyTl1xhz2XseaY73R86CRL7FRnRqpzxThC9~94P8Onrf9tNvwBGExP3NcRJbDIef9gq9fFmIH~GrgByJlG26fcXjMdXnMk0nonz~LnrTxQkyaVwQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5643928&ar=383877&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss03501.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Graphical view of syntenic relationship between <em>F.</em> × <em>ananassa</em> and <em>F. vesca</em>. Red and blue bars show LGs of <em>F.</em> × <em>ananassa</em> and chromosomes of <em>F. vesca</em>, respectively. Syntenic regions between the two species are connected by coloured lines.</p></div></div><div content-id="DSS035F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss035/2/m_dss03501.jpeg?Expires=2147483647&Signature=H2hkpA7M11oiUy-ZiFPHct2tvlpHvGCa-F1fd3uDc1n7l9rh1UOCRINWpMDitpbu3PS8uYXtlLwt56O0Lz6bEV4rLtMTqzrWiGa1ZUzIBsEbB1T9P1obsVlW0oQI3mBo8BdbDSFHNnYNw2lRxE7UrdvWHRojlxmBzCSJoZUq60NHKLaLVeQEyYUlFLeGsabI8mVatYKERHLiVFGSU8ftjyKUVY4dqFg~1BKEZgnAit0NsgaPl7lKQJ9wORCBGsWgsfd7QzGb7zxVfwf56Iek7PwikUGzIH-vnpgz7Le6wedr2~M2O8QaW5VywQwJM0Lm2VaUMHs6NY7JIXmdQhCsdw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Graphical view of syntenic relationship between F. × ananassa and F. vesca. Red and blue bars show LGs of F. × ananassa and chromosomes of F. vesca, respectively. Syntenic regions between the two species are connected by coloured lines." path-from-xml="dss03501.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5643928/dss03501.jpeg" path-from-xml="dss03501.jpeg" target="_blank">View large</a><a section="5643928" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss035/2/dss03501.jpeg?Expires=2147483647&Signature=1TIqiKsCZcylNolCO3uQUV-TxJfyA8~LfcJh919ouN0685ztMj0EnMaGQwUU4cB8eFsEsvMSORATDoDKSawmoQZ-AP8JngtQ3BJ81x-ftBgHGgtysV78mR7d4-5NAVhuSmwyyP73bwGST8uaQqd4~~6fkdt8fxx5-wP5kMvbvzz6Dc9VqlbpCK1CgbJIjiU~WzrpdG38dSlk0KBSnCCvCxVDEpyJL-xTpH0VyaOyTl1xhz2XseaY73R86CRL7FRnRqpzxThC9~94P8Onrf9tNvwBGExP3NcRJbDIef9gq9fFmIH~GrgByJlG26fcXjMdXnMk0nonz~LnrTxQkyaVwQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5643928&ar=383877&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss03501.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Graphical view of syntenic relationship between <em>F.</em> × <em>ananassa</em> and <em>F. vesca</em>. Red and blue bars show LGs of <em>F.</em> × <em>ananassa</em> and chromosomes of <em>F. vesca</em>, respectively. Syntenic regions between the two species are connected by coloured lines.</p></div></div>                            <h3 scrollto-destination=5643929 id="5643929" class="section-title" data-legacy-id=s3f>3.6. Distinguishing the varieties of 129 <em>F.</em> × <em>ananassa</em> lines using selected EST-SSR markers</h3>
<p>To select the best marker set to determine the varieties of the <em>F.</em> × <em>ananassa</em> lines, polymorphic analysis of the 22 cultivated strawberry lines was performed using primer pairs targeting 3746 FVES and 603 FAES markers. In the primer pairs of FVES markers, 2949 resulted in solid amplification, whereas of the remaining primer pairs, 431, 263, and 103 resulted in no amplification, multiple bands, and rare bands, respectively (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). In the FAES markers, solid amplification was observed for 460 primer pairs, whereas of the remaining primer pairs, 88, 52, and 3 resulted in no amplification, multiple, and rare bands, respectively (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>). A total of 751 primer pairs, including 650 FVES and 101 FAES, showed solid and polymorphic amplification and were used for the subsequent investigation of PCR, i.e. the stability of PCR and detection of noise peaks, using a fragment analyzer for all 22 lines. A total of 100 of the 751 primer pairs showing high stability and few noise peaks were selected and used for polymorphic analysis with 129 strawberry lines to reconfirm the robustness of PCR in a more diversified collection. From this experiment, the 45 best markers, which showed high stability in PCR and few noise peaks, were selected. PCR was then performed twice more with the 129 strawberry lines to exclude genotyping errors (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>).</p><p>The primer pairs of 45 selected SSR markers, including 4 FAES and 41 FVES markers, generated a total of 158 peaks in the 129 strawberry lines that were tested (<a class="link link-uri" href="http://vim.kazusa.or.jp/Strawberry/">http://vim.kazusa.or.jp/Strawberry/</a>). The number of identified peaks per a primer pair ranged from two to seven, with a mean value of 3.51 (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S4</a>). The HZ value of each identified peak was calculated based on a score of 1 or 0 for the presence and absence of the peak and ranged from 0 to 0.5. The mean HZ value of each marker ranged from 0.11 to 0.43, with an average value of 0.25. Similarity coefficients were used to examine the genetic relationships between the 129 strawberry lines. All possible genotypes showed similarity coefficients ranging from 0.00 to 0.41 (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). No genetic differences were identified with the 45 markers between ‘Nyoho’ and ‘Shinnyoho’ or between ‘Himatsuri’ and ‘Toyonoka’. The highest similarity coefficient, 0.41, was found between the UK variety ‘Serenata’ and the Japanese variety ‘Summer berry’.</p>                            <h2 scrollto-destination=5643932 id="5643932" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>In this study, a total of 4474 SSR markers, including 3746 FVES, 603 FAES, and 125 FATS, were designed from public EST and transcript sequences of <em>F. vesca</em> and <em>F.</em> × <em>ananassa</em>. Of the 4474 SSR markers, 672 resulted in rare or no amplification, and the remaining 3802 SSR markers resulted in amplification. To our knowledge, 441 SSR markers have previously been published for the genus <em>Fragaria</em>.<sup><a reveal-id="DSS035C8" data-open="DSS035C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSS035C10" data-open="DSS035C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DSS035C15" data-open="DSS035C15" class="link link-ref link-reveal xref-bibr">15</a>,<a reveal-id="DSS035C17  DSS035C23" data-open="DSS035C17  DSS035C23" class="link link-ref link-reveal xref-bibr">17–23</a>,<a reveal-id="DSS035C27" data-open="DSS035C27" class="link link-ref link-reveal xref-bibr">27</a>,<a reveal-id="DSS035C32  DSS035C39" data-open="DSS035C32  DSS035C39" class="link link-ref link-reveal xref-bibr">32–39</a></sup> The number of SSR markers that were developed in this study is approximately 8.6 times that of previously published markers. Recent advances in genome sequencing technology have enabled the large-scale development of single nucleotide polymorphism (SNP) markers in many of plant species. However, SNP discovery and genotyping are still difficult for polyploidy species because of the difficulty in distinguishing between homeologous allelic SNPs. Therefore, SSR markers are currently the most rapid and conclusive marker system for most polyploid species, including <em>F.</em> × <em>ananassa</em>. The numerous SSR markers developed in this study are an important resource for genetic and genomic studies of this species<em>.</em></p><p>In this study, linkage analysis was performed using a methodology previously employed in diploid and outcrossing species with the assumption that <em>F.</em> × <em>ananassa</em> is an allo-octoploid species with an AAA′A′BBB′B′ genome structure. The large number of mapped loci that showed disomic inheritance supported this assumption.</p><p>In the ‘02–19’ × ‘Sachinoka’ and ‘Kaorino’ × ‘Akihime’ populations, polymorphic marker screening was only performed with parental lines that caused failure of screening polymorphic markers showing a &lt;ab × ab&gt; segregation pattern. Meanwhile, eight randomly selected S<sub>1</sub> individuals were used in polymorphic marker screening of the ‘0212921’ population, and all possible polymorphic markers were selected. Therefore, we considered that the large differences of the number of mapped loci in the parent-specific maps were due to the steps in polymorphic marker screening and the genetic distances of haplotypes within parents. All the previously published maps in <em>F.</em> × <em>ananassa</em> were derived from F<sub>1</sub> mapping populations, and the ‘0212921’ specific map is the first parent-specific map derived from an S<sub>1</sub> population. The density of the parent-specific map ‘0212921’ suggests that an S<sub>1</sub> population is available for linkage map construction in <em>F.</em> × <em>ananassa</em>.</p><p>According to a previous study<sup><a reveal-id="DSS035C10" data-open="DSS035C10" class="link link-ref link-reveal xref-bibr">10</a></sup> and the integrated map developed in this study, the length of a saturated linkage map in <em>F.</em> × <em>ananassa</em> would exceed 2000 cM. The lengths of all the parent-specific maps were shorter than 2000 cM, except the ‘Sachinoka’ map that was 2166.6 cM. In addition, no parent-specific map had 28 LGs, i.e. the number of chromosomes in the haploid genome of <em>F.</em> × <em>ananassa.</em> Therefore, it was concluded that all parental specific maps were unsaturated.</p><p>The integrated linkage map comprises 1856 loci on 28 LGs. The total length of the map, 2364 cM, is slightly longer than the previously reported densest map, 2140 cM, by Sargent et al.<sup><a reveal-id="DSS035C10" data-open="DSS035C10" class="link link-ref link-reveal xref-bibr">10</a></sup> The mean locus density of our integrated map was 1.27 cM locus<sup>−1</sup>, which was three times denser than that of the map of Sargent et al. Moreover, the integrated map presented in this study is the first linkage map derived from multiple mapping populations of <em>F.</em> × <em>ananassa</em>, suggesting that the map reflects wider genetic diversity in the species<em>.</em> Several LGs of the integrated map seemed to be saturated, whereas some of others were not, such as LG1D, LG4B, LG5C, LG5D, LG7B, and LG7C. In addition, the parent-specific LGs were not evenly merged into the LGs in the integrated map. For example, LG6A in the integrated map consists of 16 parent-specific LGs, whereas LG1D was derived from a single parent-specific LG of the ‘0212921’-specific map. One of the causes for the uneven positions of the mapped loci and the number of merged parent-specific LGs may be the bias of heterozygous (polymorphic) genomic regions within each parental line. Moreover, the source of EST-SSR markers might affect the evenness of the mapping. Of the mapped loci on the integrated map, 82% were derived from <em>F. vesca</em> sequences, whereas 14 and 4% markers were generated from <em>F.</em> × <em>ananassa</em> and other species, respectively. Previous molecular and cytogenetic studies suggested that <em>F. vesca</em> and <em>F. iinumae</em> or <em>F. daltoniana</em> were candidate ancestral species of <em>F.</em> × <em>ananassa</em>.<sup><a reveal-id="DSS035C13" data-open="DSS035C13" class="link link-ref link-reveal xref-bibr">13</a>,<a reveal-id="DSS035C14" data-open="DSS035C14" class="link link-ref link-reveal xref-bibr">14</a></sup> This theory suggests that partial genome regions in <em>F.</em> × <em>ananassa</em> were derived from <em>F. iinumae</em> or <em>F. daltoniana</em> and may show non-homology with the genomes of <em>F. vesca</em>. In addition, the ratio of mapped FVES loci ranged from 62.5 to 93.3% (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>). The large contribution of <em>F. vesca</em> sequence-derived SSR markers on the integrated map may affect the biased position of the mapped loci.</p><p>Although a total of 107 loci derived from previously published markers were mapped onto the integrated linkage map, we did not employ the corresponding names used in previously published maps. This was due to the insufficient number of commonly mapped markers across linkage maps, along with the large number of mapped multi loci. In this study, ‘A’ to ‘D’ suffixes were added to the LG names, which corresponded to the lengths of the LGs; A was used for the longest LG, whereas D was added to the shortest. Therefore, there are no biological relationships among the LGs designated with the same capital letter suffix. We request the replacing of the capital letters as to represent the genome structure in <em>F.</em> × <em>ananassa</em>, for example AAA′A′BBB′B′, after corresponding LGs to each genome, will be identified in future.</p><p>Loci generated from SLD markers should play an important role in the identification of specific LGs and chromosomes in <em>F.</em> × <em>ananassa.</em> In this study, we mapped both SLD and MLD markers, and 25.9% (481) of the mapped loci were derived from SLD markers. Indeed, the karyotypes of octoploid <em>Fragaria</em> species were already reported in <em>F. chiloensis</em> and <em>F. virginiana</em>,<sup><a reveal-id="DSS035C47" data-open="DSS035C47" class="link link-ref link-reveal xref-bibr">47</a></sup> as well as in <em>F.</em> × <em>ananassa</em>.<sup><a reveal-id="DSS035C48" data-open="DSS035C48" class="link link-ref link-reveal xref-bibr">48</a></sup> Using genomic region mapped single loci as probes for cytological analysis such as fluorescence <em>in situ</em> hybridization, corresponding karyotypes of <em>F.</em> × <em>ananassa</em> could be identified on each LG in the integrated map. In addition, the genome structure of <em>F.</em> × <em>ananassa</em> could be resolved by comparative mapping with candidate ancestral <em>Fragaria</em> species using markers detecting single loci.</p><p>While 25.9% of the mapped loci were generated from SLD markers, the remaining 74.1% (1375) were derived from MLD markers, comprising 569 Multi_H, 16 Multi_H&amp;NH, 99 Multi H&amp;S, 61 Multi_NH, 107 Multi_S, and 523 Multi_NM. The ratio of all Multi_H (including Multi_H&amp;NS and H&amp;S) to the sum of mapped loci (except Multi_NM) was 0.51 [(569 + 16 + 99)/(1856–523)]; that is approximately half of the mapped loci on the integrated map were Multi_H. The Multi_H loci were mapped along the entire genomes. However, corresponding homeologous positions differed depending on the paired LGs. The uneven homeologous regions between each pair of LGs might represent a feature of the genome composition of <em>F.</em> × <em>ananassa</em>.</p><p>Comparative mapping studies between <em>F.</em> × <em>ananassa</em> and <em>F. vesca</em> have been reported.<sup><a reveal-id="DSS035C8  DSS035C11" data-open="DSS035C8  DSS035C11" class="link link-ref link-reveal xref-bibr">8–11</a></sup> According to previous results, most of the genomic regions are conserved between the two species, with the rare exception of chromosome rearrangement on HG1, HG3, and HG6. Results from our comparative mapping study generally agreed with those of previous studies, i.e. the identification of clear macrosynteny over both entire genomes. However, our results suggest that partial rearrangement occurred more frequently between homeologous genomes in <em>F.</em> × <em>ananassa</em> and <em>F. vesca</em> than previously reported. In addition, genomic rearrangement across HGs was first observed in this study, i.e. between Fv-Chr2 and Fa-HG3 (Fig. <a reveal-id="DSS035F1" data-open="DSS035F1" class="link link-reveal link-table xref-fig">1</a>). Of the LGs belonging to HG3, LG3A, 3B and 3C showed the genomic rearrangement, whereas it was not observed on LG3D.</p><p>The varieties of 129 <em>F.</em> × <em>ananassa</em> lines were distinguished to demonstrate the practicable utility of the markers developed in this study. Ninety-one percent (118 of 129) of the tested lines were developed in Japan. Because most Japanese lines were developed from a limited number of ancestral lines, such as ‘Haward17’ and ‘General Chanzy’, genetic diversity is considered to be generally narrow in Japanese germplasm.<sup><a reveal-id="DSS035C49" data-open="DSS035C49" class="link link-ref link-reveal xref-bibr">49</a></sup> Despite the expected narrow genetic diversity, the 45 SSR markers employed in this study identified most of the tested lines except for 2 pairs, ‘Nyoho’ and ‘Shin-Nyoho’ and ‘Toyonoka’ and ‘Himatsuri’. ‘Shin-Nyoho’ and ‘Himatsuri’ were developed from mutation lines of ‘Toyonika’ and ‘Nyoho’, respectively. Therefore, it was assumed that the genetic diversities were quite narrow in each pair of undistinguished varieties. Meanwhile, it is notable that the ‘Anter’, ‘Pihyaradondon’, and ‘Nyoho’ lines and the ‘Akita Berry’ and ‘Morioka 16’ lines could be distinguished from each other, although the former two and one of the varieties are mutant lines of ‘Nyoho’ and ‘Morioka 16’, respectively.</p><p>Variety identification is one of the major practical uses of DNA markers. Such identification should protect breeders' rights and inhibit the contamination of clonal seedlings during propagation. Because unknown samples are often used in variety identification, the accurate detection of targeted peaks is essential. In addition, it is well known that PCR results can vary depending on experimental conditions such as the type of thermal cycler and Taq polymerase employed during PCR analysis. In a previous study, Kunihisa<sup><a reveal-id="DSS035C44" data-open="DSS035C44" class="link link-ref link-reveal xref-bibr">44</a></sup> investigated the stability of polymorphic analysis of CAPS markers by comparing the results obtained in 14 laboratories to verify the adequacy of the makers used for variety identification tests in <em>F.</em> × <em>ananassa</em>. Govan et al.<sup><a reveal-id="DSS035C50" data-open="DSS035C50" class="link link-ref link-reveal xref-bibr">50</a></sup> also screened 32 SSRs that produced reliable PCR results for the identification of 60 varieties. In the present study, the 45 SSR markers were carefully screened by confirming the results of 3 independent PCRs. The 36 out of the 45 markers were mapped onto a total of 20 LGs of the integrated map. It suggested that the polymorphisms of selected markers roughly reflect genetic diversity in entire genome. Therefore, the markers were found to be reliable for variety identification.</p><p>In this study, we developed large numbers of SSR markers and constructed the densest linkage map for <em>F.</em> × <em>ananassa.</em> By performing comparative mapping and variety distinction, the resources that we developed were shown to be useful for genetic and genomic analysis as well as practical applications. Our results should contribute to the acceleration of advances in the study of <em>F.</em> × <em>ananassa</em>, along with the genus <em>Fragaria</em>.</p>                            <h2 scrollto-destination=5643945 id="5643945" class="section-title" data-legacy-id=s5>Supplementary Data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Data are available at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=5643947 id="5643947" class="section-title" data-legacy-id=s6>Funding</h2>
<p>This work was supported by the Kazusa DNA Research Institute Foundation, research and development projects for application in promoting new policies of agriculture, forestry and fisheries (21010) funded by the Ministry of Agriculture, Forestry and Fisheries.</p>                            <h2 scrollto-destination=5643949 id="5643949" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank Dr Satoru Matsumoto at the National Agriculture and Food Research Organization, Institute of Fruit Tree Science, Dr Toshi Nishimoto at Nara Prefectural Agriculture Research Center, and Toshihiro Tanaka at Wakayama Research Center of Agriculture, Forestry and Fisheries, for providing <em>F.</em> × <em>ananassa</em> DNA or plant material.</p>                            <h2 scrollto-destination=5643951 id="5643951" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSS035C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS035C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="article-title">FAOSTAT</div> <div class="comment"><a class="link link-uri" href="http://faostat3.fao.org">http://faostat3.fao.org</a>.</div></div></div></div></div><div content-id="DSS035C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS035C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="article-title">UPOV</div> <div class="comment"><a class="link link-uri" href="http://www.upov.int">http://www.upov.int</a></div></div></div></div></div><div content-id="DSS035C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS035C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Darrow</div> <div class="given-names">G.</div></div>. </div>, <div class="source">The Strawberry. History, Breeding and Physiology</div> , <div class="year">1966</div><div class="publisher-loc">New York, USA</div><div class="publisher-name">Holt, Rinehart and Winston</div></div></div></div></div><div content-id="DSS035C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS035C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Federova</div> <div class="given-names">N.J.</div></div>. </div><div class="article-title">Cross ability and phylogenetic relationships in the main European species of <em>Fragaria</em></div>, <div class="source">Natl. Acad. Sci. USSR</div> , <div class="year">1946</div>, vol. <div class="volume">52</div> (pg. <div class="fpage">545</div>-<div class="lpage">7</div>)</div></div></div></div><div content-id="DSS035C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS035C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Senanayake</div> <div class="given-names">Y.D.A.</div></div>,  <div class="name"><div class="surname">Bringhurst</div> <div class="given-names">R.S.</div></div>. </div><div class="article-title">Origin of the <em>Fragaria</em> polyploids. I. Cytological evidence</div>, <div class="source">Am. J. Bot.</div> , <div class="year">1967</div>, vol. <div class="volume">54</div> (pg. <div class="fpage">221</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Origin%20of%20the%20Fragaria%20polyploids.%20I.%20Cytological%20evidence&amp;author=Y.D.A.%20Senanayake&amp;author=R.S.%20Bringhurst&amp;publication_year=1967&amp;journal=Am.%20J.%20Bot.&amp;volume=54&amp;pages=221-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2307/2440801" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2307%2F2440801" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2307%2F2440801"> </span></div> </div></div></div></div></div><div content-id="DSS035C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS035C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bringhurst</div> <div class="given-names">R.S.</div></div>. </div><div class="article-title">Cytogenetics and evolution in American <em>Fragaria</em></div>, <div class="source">Hortic. Sci.</div> , <div class="year">1990</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">879</div>-<div class="lpage">81</div>)</div></div></div></div><div content-id="DSS035C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS035C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kunihisa</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Fukino</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Matsumoto</div> <div class="given-names">S.</div></div>. </div><div class="article-title">CAPS markers improved by cluster-specific amplification for identification of octoploid strawberry (<em>Fragaria</em> ×ananassa Duch.) cultivars, and their disomic inheritance</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2005</div>, vol. <div class="volume">110</div>  <div class="comment">1410–1418</div></div></div></div></div><div content-id="DSS035C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS035C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rousseau-Gueutin</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Lerceteau-Kohler</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Barrot</div> <div class="given-names">L.</div></div>, et al. </div><div class="article-title">Comparative genetic mapping between octoploid and diploid <em>Fragaria</em> species reveals a high level of colinearity between their genomes and the essentially disomic behavior of the cultivated octoploid strawberry</div>, <div class="source">Genetics</div> , <div class="year">2008</div>, vol. <div class="volume">179</div> (pg. <div class="fpage">2045</div>-<div class="lpage">60</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20genetic%20mapping%20between%20octoploid%20and%20diploid%20Fragaria%20species%20reveals%20a%20high%20level%20of%20colinearity%20between%20their%20genomes%20and%20the%20essentially%20disomic%20behavior%20of%20the%20cultivated%20octoploid%20strawberry&amp;author=M.%20Rousseau-Gueutin&amp;author=E.%20Lerceteau-Kohler&amp;author=L.%20Barrot&amp;publication_year=2008&amp;journal=Genetics&amp;volume=179&amp;pages=2045-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.107.083840" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.107.083840" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.107.083840"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18660542" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS035C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS035C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sargent</div> <div class="given-names">D.J.</div></div>,  <div class="name"><div class="surname">Fernandéz-Fernandéz</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Ruiz-Roja</div> <div class="given-names">J.J.</div></div>, et al. </div><div class="article-title">A genetic linkage map of the cultivated strawberry (<em>Fragaria</em> times <em>ananassa</em>) and its comparison to the diploid <em>Fragaria</em> reference map</div>, <div class="source">Mol. Breed.</div> , <div class="year">2009</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">293</div>-<div class="lpage">303</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20genetic%20linkage%20map%20of%20the%20cultivated%20strawberry%20(Fragaria%20times%20ananassa)%20and%20its%20comparison%20to%20the%20diploid%20Fragaria%20reference%20map&amp;author=D.J.%20Sargent&amp;author=F.%20Fernand%C3%A9z-Fernand%C3%A9z&amp;author=J.J.%20Ruiz-Roja&amp;publication_year=2009&amp;journal=Mol.%20Breed.&amp;volume=24&amp;pages=293-303" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11032-009-9292-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11032-009-9292-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11032-009-9292-9"> </span></div> </div></div></div></div></div><div content-id="DSS035C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS035C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sargent</div> <div class="given-names">D.J.</div></div>,  <div class="name"><div class="surname">Passey</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Šurbanovski</div> <div class="given-names">N.</div></div>, et al. </div><div class="article-title">A microsatellite linkage map for the cultivated strawberry (<em>Fragaria</em> times <em>ananassa</em>) suggests extensive regions of homozygosity in the genome that may have resulted from breeding and selection</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2012</div>, vol. <div class="volume">124</div> (pg. <div class="fpage">1229</div>-<div class="lpage">40</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20microsatellite%20linkage%20map%20for%20the%20cultivated%20strawberry%20(Fragaria%20times%20ananassa)%20suggests%20extensive%20regions%20of%20homozygosity%20in%20the%20genome%20that%20may%20have%20resulted%20from%20breeding%20and%20selection&amp;author=D.J.%20Sargent&amp;author=T.%20Passey&amp;author=N.%20%C5%A0urbanovski&amp;publication_year=2012&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=124&amp;pages=1229-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-011-1782-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-011-1782-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-011-1782-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22218676" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS035C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS035C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lerceteau-Köhler</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Moing</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Guérin</div> <div class="given-names">G.</div></div>, et al. </div><div class="article-title">Genetic dissection of fruit quality traits in the octoploid cultivated strawberry highlights the role of homoeo-QTL in their control</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2012</div>, vol. <div class="volume">124</div> (pg. <div class="fpage">1059</div>-<div class="lpage">77</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20dissection%20of%20fruit%20quality%20traits%20in%20the%20octoploid%20cultivated%20strawberry%20highlights%20the%20role%20of%20homoeo-QTL%20in%20their%20control&amp;author=E.%20Lerceteau-K%C3%B6hler&amp;author=A.%20Moing&amp;author=G.%20Gu%C3%A9rin&amp;publication_year=2012&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=124&amp;pages=1059-77" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-011-1769-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-011-1769-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-011-1769-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22215248" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS035C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS035C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Staudt</div> <div class="given-names">G.</div></div>. </div><div class="article-title">The species of <em>Fragaria</em>, their taxonomy and geographic distribution</div>, <div class="source">Acta Hort.</div> , <div class="year">1989</div>, vol. <div class="volume">265</div> (pg. <div class="fpage">23</div>-<div class="lpage">33</div>)</div></div></div></div><div content-id="DSS035C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS035C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hancock</div> <div class="given-names">J.F.</div></div>. </div>, <div class="source">Strawberries</div> , <div class="year">1999</div><div class="publisher-loc">New York, USA</div><div class="publisher-name">CABI Publishing</div></div></div></div></div><div content-id="DSS035C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS035C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rousseau-Gueutin</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Gaston</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Aïnouche</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Tracking the evolutionary history of polyploidy in <em>Fragaria</em> L. (strawberry): new insights from phylogenetic analyses of low-copy nuclear genes</div>, <div class="source">Mol. Phylogenet. Evol.</div> , <div class="year">2009</div>, vol. <div class="volume">51</div> (pg. <div class="fpage">515</div>-<div class="lpage">30</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Tracking%20the%20evolutionary%20history%20of%20polyploidy%20in%20Fragaria%20L.%20(strawberry)%3A%20new%20insights%20from%20phylogenetic%20analyses%20of%20low-copy%20nuclear%20genes&amp;author=M.%20Rousseau-Gueutin&amp;author=A.%20Gaston&amp;author=A.%20A%C3%AFnouche&amp;publication_year=2009&amp;journal=Mol.%20Phylogenet.%20Evol.&amp;volume=51&amp;pages=515-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ympev.2008.12.024" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ympev.2008.12.024" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ympev.2008.12.024"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19166953" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS035C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS035C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">James</div> <div class="given-names">C.M.</div></div>,  <div class="name"><div class="surname">Wilson</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Hadonou</div> <div class="given-names">A.M.</div></div>,  <div class="name"><div class="surname">Tobutt</div> <div class="given-names">K.R.</div></div>. </div><div class="article-title">Isolation and characterization of polymorphic microsatellites in diploid strawberry (<em>Fragaria vesca</em> L.) for mapping, diversity studies and clone identification</div>, <div class="source">Mol. Ecol. Notes</div> , <div class="year">2003</div>, vol. <div class="volume">3</div> (pg. <div class="fpage">171</div>-<div class="lpage">3</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Isolation%20and%20characterization%20of%20polymorphic%20microsatellites%20in%20diploid%20strawberry%20(Fragaria%20vesca%20L.)%20for%20mapping%2C%20diversity%20studies%20and%20clone%20identification&amp;author=C.M.%20James&amp;author=F.%20Wilson&amp;author=A.M.%20Hadonou&amp;author=K.R.%20Tobutt&amp;publication_year=2003&amp;journal=Mol.%20Ecol.%20Notes&amp;volume=3&amp;pages=171-3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1471-8286.2003.00365.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1471-8286.2003.00365.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1471-8286.2003.00365.x"> </span></div> </div></div></div></div></div><div content-id="DSS035C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS035C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Albani</div> <div class="given-names">M.C.</div></div>,  <div class="name"><div class="surname">Battey</div> <div class="given-names">N.H.</div></div>,  <div class="name"><div class="surname">Wilkinson</div> <div class="given-names">M.J.</div></div>. </div><div class="article-title">The development of ISSR-derived SCAR markers around the SEASONAL FLOWERING LOCUS (SFL) in <em>Fragaria vesca</em></div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2004</div>, vol. <div class="volume">109</div> (pg. <div class="fpage">571</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20development%20of%20ISSR-derived%20SCAR%20markers%20around%20the%20SEASONAL%20FLOWERING%20LOCUS%20(SFL)%20in%20Fragaria%20vesca&amp;author=M.C.%20Albani&amp;author=N.H.%20Battey&amp;author=M.J.%20Wilkinson&amp;publication_year=2004&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=109&amp;pages=571-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-004-1654-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-004-1654-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-004-1654-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15292991" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS035C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS035C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cipriani</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Testolin</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Isolation and characterization of microsatellite loci in <em>Fragaria</em></div>, <div class="source">Mol. Ecol. Notes</div> , <div class="year">2004</div>, vol. <div class="volume">4</div> (pg. <div class="fpage">366</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Isolation%20and%20characterization%20of%20microsatellite%20loci%20in%20Fragaria&amp;author=G.%20Cipriani&amp;author=R.%20Testolin&amp;publication_year=2004&amp;journal=Mol.%20Ecol.%20Notes&amp;volume=4&amp;pages=366-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/men.2004.4.issue-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmen.2004.4.issue-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmen.2004.4.issue-3"> </span></div> </div></div></div></div></div><div content-id="DSS035C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS035C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hadonou</div> <div class="given-names">A.M.</div></div>,  <div class="name"><div class="surname">Sargent</div> <div class="given-names">D.J.</div></div>,  <div class="name"><div class="surname">Wilson</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">James</div> <div class="given-names">C.M.</div></div>,  <div class="name"><div class="surname">Simpson</div> <div class="given-names">D.W.</div></div>. </div><div class="article-title">Development of microsatellite markers in <em>Fragaria</em>, their use in genetic diversity analysis, and their potential for genetic linkage mapping</div>, <div class="source">Genome</div> , <div class="year">2004</div>, vol. <div class="volume">47</div> (pg. <div class="fpage">429</div>-<div class="lpage">38</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20microsatellite%20markers%20in%20Fragaria%2C%20their%20use%20in%20genetic%20diversity%20analysis%2C%20and%20their%20potential%20for%20genetic%20linkage%20mapping&amp;author=A.M.%20Hadonou&amp;author=D.J.%20Sargent&amp;author=F.%20Wilson&amp;author=C.M.%20James&amp;author=D.W.%20Simpson&amp;publication_year=2004&amp;journal=Genome&amp;volume=47&amp;pages=429-38" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/g03-142" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2Fg03-142" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2Fg03-142"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15190360" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS035C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS035C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Monfort</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Vilanova</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Davis</div> <div class="given-names">T.M.</div></div>,  <div class="name"><div class="surname">Arús</div> <div class="given-names">P.</div></div>. </div><div class="article-title">A new set of polymorphic simple sequence repeat (SSR) markers from a wild strawberry (<em>Fragaria vesca</em>) are transferable to other diploid <em>Fragaria</em> species and to <em>Fragaria</em> times <em>ananassa</em></div>, <div class="source">Mol. Ecol. Notes</div> , <div class="year">2006</div>, vol. <div class="volume">6</div> (pg. <div class="fpage">197</div>-<div class="lpage">200</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20new%20set%20of%20polymorphic%20simple%20sequence%20repeat%20(SSR)%20markers%20from%20a%20wild%20strawberry%20(Fragaria%20vesca)%20are%20transferable%20to%20other%20diploid%20Fragaria%20species%20and%20to%20Fragaria%20times%20ananassa&amp;author=A.%20Monfort&amp;author=S.%20Vilanova&amp;author=T.M.%20Davis&amp;author=P.%20Ar%C3%BAs&amp;publication_year=2006&amp;journal=Mol.%20Ecol.%20Notes&amp;volume=6&amp;pages=197-200" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/men.2006.6.issue-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmen.2006.6.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmen.2006.6.issue-1"> </span></div> </div></div></div></div></div><div content-id="DSS035C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS035C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sargent</div> <div class="given-names">D.J.</div></div>,  <div class="name"><div class="surname">Davis</div> <div class="given-names">T.M.</div></div>,  <div class="name"><div class="surname">Tobutt</div> <div class="given-names">K.R.</div></div>,  <div class="name"><div class="surname">Wilkinson</div> <div class="given-names">M.J.</div></div>,  <div class="name"><div class="surname">Battey</div> <div class="given-names">N.H.</div></div>,  <div class="name"><div class="surname">Simpson</div> <div class="given-names">D.W.</div></div>. </div><div class="article-title">A genetic linkage map of microsatellite, gene-specific and morphological markers in diploid <em>Fragaria</em></div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2004</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">1385</div>-<div class="lpage">91</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20genetic%20linkage%20map%20of%20microsatellite%2C%20gene-specific%20and%20morphological%20markers%20in%20diploid%20Fragaria&amp;author=D.J.%20Sargent&amp;author=T.M.%20Davis&amp;author=K.R.%20Tobutt&amp;author=M.J.%20Wilkinson&amp;author=N.H.%20Battey&amp;author=D.W.%20Simpson&amp;publication_year=2004&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=7&amp;pages=1385-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-004-1767-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-004-1767-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-004-1767-9"> </span></div> </div></div></div></div></div><div content-id="DSS035C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS035C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sargent</div> <div class="given-names">D.J.</div></div>,  <div class="name"><div class="surname">Clarke</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Simpson</div> <div class="given-names">D.W.</div></div>, et al. </div><div class="article-title">An enhanced microsatellite map of diploid <em>Fragaria</em></div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2006</div>, vol. <div class="volume">112</div> (pg. <div class="fpage">1349</div>-<div class="lpage">59</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20enhanced%20microsatellite%20map%20of%20diploid%20Fragaria&amp;author=D.J.%20Sargent&amp;author=J.%20Clarke&amp;author=D.W.%20Simpson&amp;publication_year=2006&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=112&amp;pages=1349-59" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-006-0237-y" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-006-0237-y" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-006-0237-y"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16505996" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS035C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS035C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sargent</div> <div class="given-names">D.J.</div></div>,  <div class="name"><div class="surname">Rys</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Nier</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Simpson</div> <div class="given-names">D.W.</div></div>,  <div class="name"><div class="surname">Tobutt</div> <div class="given-names">K.R.</div></div>. </div><div class="article-title">The development and mapping of functional markers in <em>Fragaria</em> and their transferability and potential for mapping in other genera</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2007</div>, vol. <div class="volume">114</div> (pg. <div class="fpage">373</div>-<div class="lpage">84</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20development%20and%20mapping%20of%20functional%20markers%20in%20Fragaria%20and%20their%20transferability%20and%20potential%20for%20mapping%20in%20other%20genera&amp;author=D.J.%20Sargent&amp;author=A.%20Rys&amp;author=S.%20Nier&amp;author=D.W.%20Simpson&amp;author=K.R.%20Tobutt&amp;publication_year=2007&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=114&amp;pages=373-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-006-0441-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-006-0441-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-006-0441-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17091261" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS035C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS035C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sargent</div> <div class="given-names">D.J.</div></div>,  <div class="name"><div class="surname">Cipriani</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Vilanova</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">The development of a bin mapping population and the selective mapping of 103 markers in the diploid <em>Fragaria</em> reference map</div>, <div class="source">Genome</div> , <div class="year">2008</div>, vol. <div class="volume">51</div> (pg. <div class="fpage">120</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20development%20of%20a%20bin%20mapping%20population%20and%20the%20selective%20mapping%20of%20103%20markers%20in%20the%20diploid%20Fragaria%20reference%20map&amp;author=D.J.%20Sargent&amp;author=G.%20Cipriani&amp;author=S.%20Vilanova&amp;publication_year=2008&amp;journal=Genome&amp;volume=51&amp;pages=120-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/G07-107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2FG07-107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2FG07-107"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18356946" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS035C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS035C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bonet</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Girona</div> <div class="given-names">E.L.</div></div>,  <div class="name"><div class="surname">Sargent</div> <div class="given-names">D.J.</div></div>, et al. </div><div class="article-title">The development and characterisation of a bacterial artificial chromosome library for <em>Fragaria vesca</em></div>, <div class="source">BMC Res. Notes</div> , <div class="year">2009</div>, vol. <div class="volume">2</div> pg. <div class="fpage">188</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20development%20and%20characterisation%20of%20a%20bacterial%20artificial%20chromosome%20library%20for%20Fragaria%20vesca&amp;author=J.%20Bonet&amp;author=E.L.%20Girona&amp;author=D.J.%20Sargent&amp;publication_year=2009&amp;journal=BMC%20Res.%20Notes&amp;volume=2&amp;pages=188" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1756-0500-2-188" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1756-0500-2-188" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1756-0500-2-188"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19772672" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS035C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS035C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pontaroli</div> <div class="given-names">A.C.</div></div>,  <div class="name"><div class="surname">Rogers</div> <div class="given-names">R.L.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">Q.</div></div>, et al. </div><div class="article-title">Gene content and distribution in the nuclear genome of <em>Fragaria vesca</em></div>, <div class="source">Plant Genome</div> , <div class="year">2009</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">93</div>-<div class="lpage">101</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20content%20and%20distribution%20in%20the%20nuclear%20genome%20of%20Fragaria%20vesca&amp;author=A.C.%20Pontaroli&amp;author=R.L.%20Rogers&amp;author=Q.%20Zhang&amp;publication_year=2009&amp;journal=Plant%20Genome&amp;volume=2&amp;pages=93-101" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3835/plantgenome2008.09.0007" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3835%2Fplantgenome2008.09.0007" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3835%2Fplantgenome2008.09.0007"> </span></div> </div></div></div></div></div><div content-id="DSS035C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS035C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shulaev</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Sargent</div> <div class="given-names">D.J.</div></div>,  <div class="name"><div class="surname">Crowhurst</div> <div class="given-names">R.N.</div></div>, et al. </div><div class="article-title">The genome of woodland strawberry (<em>Fragaria vesca</em>)</div>, <div class="source">Nat. Genet.</div> , <div class="year">2011</div>, vol. <div class="volume">43</div> (pg. <div class="fpage">109</div>-<div class="lpage">16</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20of%20woodland%20strawberry%20(Fragaria%20vesca)&amp;author=V.%20Shulaev&amp;author=D.J.%20Sargent&amp;author=R.N.%20Crowhurst&amp;publication_year=2011&amp;journal=Nat.%20Genet.&amp;volume=43&amp;pages=109-16" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.740" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.740" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.740"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21186353" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS035C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS035C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cabrera</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Kozik</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Howad</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Arus</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Iezzoni</div> <div class="given-names">A.F.</div></div>,  <div class="name"><div class="surname">Knaap</div> <div class="given-names">E.</div></div>. </div><div class="article-title">Development and bin mapping of a Rosaceae conserved ortholog set (COS) of markers</div>, <div class="source">BMC Genomics</div> , <div class="year">2009</div>, vol. <div class="volume">10</div> pg. <div class="fpage">562</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20and%20bin%20mapping%20of%20a%20Rosaceae%20conserved%20ortholog%20set%20(COS)%20of%20markers&amp;author=A.%20Cabrera&amp;author=A.%20Kozik&amp;author=W.%20Howad&amp;author=P.%20Arus&amp;author=A.F.%20Iezzoni&amp;author=E.%20Knaap&amp;publication_year=2009&amp;journal=BMC%20Genomics&amp;volume=10&amp;pages=562" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-10-562" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-10-562" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-10-562"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19943965" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS035C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS035C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jung</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Cestaro</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Troggio</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Whole genome comparisons of <em>Fragaria</em>, <em>Prunus</em> and <em>Malus</em> reveal different modes of evolution between Rosaceous subfamilies</div>, <div class="source">BMC Genomics</div> , <div class="year">2012</div>, vol. <div class="volume">13</div> pg. <div class="fpage">129</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Whole%20genome%20comparisons%20of%20Fragaria%2C%20Prunus%20and%20Malus%20reveal%20different%20modes%20of%20evolution%20between%20Rosaceous%20subfamilies&amp;author=S.%20Jung&amp;author=A.%20Cestaro&amp;author=M.%20Troggio&amp;publication_year=2012&amp;journal=BMC%20Genomics&amp;volume=13&amp;pages=129" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-13-129" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-13-129" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-13-129"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22475018" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS035C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS035C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lerceteau-Köhler</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Guérin</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Laigret</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Denoyes-Rothan</div> <div class="given-names">B.</div></div>. </div><div class="article-title">Characterization of mixed disomic and polysomic inheritance in the octoploid strawberry (<em>Fragaria</em> times <em>ananassa</em>) using AFLP mapping</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2003</div>, vol. <div class="volume">107</div> (pg. <div class="fpage">619</div>-<div class="lpage">28</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20mixed%20disomic%20and%20polysomic%20inheritance%20in%20the%20octoploid%20strawberry%20(Fragaria%20times%20ananassa)%20using%20AFLP%20mapping&amp;author=E.%20Lerceteau-K%C3%B6hler&amp;author=G.%20Gu%C3%A9rin&amp;author=F.%20Laigret&amp;author=B.%20Denoyes-Rothan&amp;publication_year=2003&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=107&amp;pages=619-28" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-003-1300-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-003-1300-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-003-1300-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12768242" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS035C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS035C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zorrilla-Fontanesi</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Cabeza</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Domínguez</div> <div class="given-names">P.</div></div>, et al. </div><div class="article-title">Quantitative trait loci and underlying candidate genes controlling agronomical and fruit quality traits in octoploid strawberry (<em>Fragaria</em> times <em>ananassa</em>)</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2011</div>, vol. <div class="volume">123</div> (pg. <div class="fpage">755</div>-<div class="lpage">78</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Quantitative%20trait%20loci%20and%20underlying%20candidate%20genes%20controlling%20agronomical%20and%20fruit%20quality%20traits%20in%20octoploid%20strawberry%20(Fragaria%20times%20ananassa)&amp;author=Y.%20Zorrilla-Fontanesi&amp;author=A.%20Cabeza&amp;author=P.%20Dom%C3%ADnguez&amp;publication_year=2011&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=123&amp;pages=755-78" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-011-1624-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-011-1624-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-011-1624-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21667037" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS035C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS035C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rozen</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Skaletsky</div> <div class="given-names">H.</div></div>. </div><div class="person-group"><div class="name"><div class="surname">Krawetz</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Misener</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Primer3 on the WWW for general users and for biologist programmers</div>, <div class="source">Bioinformatics Methods and Protocols: Methods in Molecular Biology</div> , <div class="year">2000</div><div class="publisher-loc">Totowa, NJ, USA</div><div class="publisher-name">Humana Press</div>(pg. <div class="fpage">365</div>-<div class="lpage">86</div>)</div></div></div></div><div content-id="DSS035C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS035C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ashley</div> <div class="given-names">M.V.</div></div>,  <div class="name"><div class="surname">Wilk</div> <div class="given-names">J.A.</div></div>,  <div class="name"><div class="surname">Styan</div> <div class="given-names">S.M.N.</div></div>, et al. </div><div class="article-title">High variability and disomic segregation of microsatellites in the octoploid <em>Fragaria virginiana</em> Mill. (Rosaceae)</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2003</div>, vol. <div class="volume">107</div> (pg. <div class="fpage">1201</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High%20variability%20and%20disomic%20segregation%20of%20microsatellites%20in%20the%20octoploid%20Fragaria%20virginiana%20Mill.%20(Rosaceae)&amp;author=M.V.%20Ashley&amp;author=J.A.%20Wilk&amp;author=S.M.N.%20Styan&amp;publication_year=2003&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=107&amp;pages=1201-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-003-1370-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-003-1370-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-003-1370-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12908097" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS035C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS035C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sargent</div> <div class="given-names">D.J.</div></div>,  <div class="name"><div class="surname">Hadonou</div> <div class="given-names">A.M.</div></div>,  <div class="name"><div class="surname">Simpson</div> <div class="given-names">D.W.</div></div>. </div><div class="article-title">Development and characterization of polymorphic microsatellite markers from <em>Fragaria viridis</em>, a wild diploid strawberry</div>, <div class="source">Mol. Ecol. Notes</div> , <div class="year">2003</div>, vol. <div class="volume">3</div> (pg. <div class="fpage">550</div>-<div class="lpage">2</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20and%20characterization%20of%20polymorphic%20microsatellite%20markers%20from%20Fragaria%20viridis%2C%20a%20wild%20diploid%20strawberry&amp;author=D.J.%20Sargent&amp;author=A.M.%20Hadonou&amp;author=D.W.%20Simpson&amp;publication_year=2003&amp;journal=Mol.%20Ecol.%20Notes&amp;volume=3&amp;pages=550-2" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1471-8286.2003.00507.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1471-8286.2003.00507.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1471-8286.2003.00507.x"> </span></div> </div></div></div></div></div><div content-id="DSS035C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS035C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lewers</div> <div class="given-names">K.S.</div></div>,  <div class="name"><div class="surname">Styan</div> <div class="given-names">S.M.N.</div></div>,  <div class="name"><div class="surname">Hokanson</div> <div class="given-names">S.C.</div></div>. </div><div class="article-title">Strawberry GenBank derived and genomic simple sequence repeat (SSR) markers and their utility with strawberry, blackberry, and red and black raspberry</div>, <div class="source">J. Am. Soc. Hortic. Sci.</div> , <div class="year">2005</div>, vol. <div class="volume">130</div> (pg. <div class="fpage">102</div>-<div class="lpage">15</div>)</div></div></div></div><div content-id="DSS035C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS035C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bassil</div> <div class="given-names">N.V.</div></div>,  <div class="name"><div class="surname">Gunn</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Folta</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Lewers</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Microsatellite markers for <em>Fragaria</em> from ‘Strawberry Festival’ expressed sequence tags</div>, <div class="source">Mol. Ecol. Notes</div> , <div class="year">2006</div>, vol. <div class="volume">6</div> (pg. <div class="fpage">473</div>-<div class="lpage">76</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microsatellite%20markers%20for%20Fragaria%20from%20%E2%80%98Strawberry%20Festival%E2%80%99%20expressed%20sequence%20tags&amp;author=N.V.%20Bassil&amp;author=M.%20Gunn&amp;author=K.%20Folta&amp;author=K.%20Lewers&amp;publication_year=2006&amp;journal=Mol.%20Ecol.%20Notes&amp;volume=6&amp;pages=473-76" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/men.2006.6.issue-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmen.2006.6.issue-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmen.2006.6.issue-2"> </span></div> </div></div></div></div></div><div content-id="DSS035C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS035C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gil-Ariza</div> <div class="given-names">D.J.</div></div>,  <div class="name"><div class="surname">Amaya</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Botella</div> <div class="given-names">M.A.</div></div>, et al. </div><div class="article-title">EST-derived polymorphic microsatellites from cultivated strawberry (<em>Fragaria</em> times <em>ananassa</em>) are useful for diversity studies and varietal identification among <em>Fragaria</em> species</div>, <div class="source">Mol. Ecol. Notes</div> , <div class="year">2006</div>, vol. <div class="volume">6</div> (pg. <div class="fpage">1195</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=EST-derived%20polymorphic%20microsatellites%20from%20cultivated%20strawberry%20(Fragaria%20times%20ananassa)%20are%20useful%20for%20diversity%20studies%20and%20varietal%20identification%20among%20Fragaria%20species&amp;author=D.J.%20Gil-Ariza&amp;author=I.%20Amaya&amp;author=M.A.%20Botella&amp;publication_year=2006&amp;journal=Mol.%20Ecol.%20Notes&amp;volume=6&amp;pages=1195-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/men.2006.6.issue-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmen.2006.6.issue-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmen.2006.6.issue-4"> </span></div> </div></div></div></div></div><div content-id="DSS035C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS035C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Keniry</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Hopkins</div> <div class="given-names">C.J.</div></div>,  <div class="name"><div class="surname">Jewell</div> <div class="given-names">E.</div></div>, et al. </div><div class="article-title">Identification and characterization of simple sequence repeat (SSR) markers from <em>Fragaria</em> times <em>ananassa</em> expressed sequences</div>, <div class="source">Mol. Ecol. Notes</div> , <div class="year">2006</div>, vol. <div class="volume">6</div> (pg. <div class="fpage">319</div>-<div class="lpage">22</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20and%20characterization%20of%20simple%20sequence%20repeat%20(SSR)%20markers%20from%20Fragaria%20times%20ananassa%20expressed%20sequences&amp;author=A.%20Keniry&amp;author=C.J.%20Hopkins&amp;author=E.%20Jewell&amp;publication_year=2006&amp;journal=Mol.%20Ecol.%20Notes&amp;volume=6&amp;pages=319-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/men.2006.6.issue-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmen.2006.6.issue-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmen.2006.6.issue-2"> </span></div> </div></div></div></div></div><div content-id="DSS035C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS035C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Spigler</div> <div class="given-names">R.B.</div></div>,  <div class="name"><div class="surname">Lewers</div> <div class="given-names">K.S.</div></div>,  <div class="name"><div class="surname">Main</div> <div class="given-names">D.S.</div></div>,  <div class="name"><div class="surname">Ashman</div> <div class="given-names">T.L.</div></div>. </div><div class="article-title">Genetic mapping of sex determination in a wild strawberry, <em>Fragaria virginiana</em>, reveals earliest form of sex chromosome</div>, <div class="source">Heredity</div> , <div class="year">2008</div>, vol. <div class="volume">101</div> (pg. <div class="fpage">507</div>-<div class="lpage">17</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20mapping%20of%20sex%20determination%20in%20a%20wild%20strawberry%2C%20Fragaria%20virginiana%2C%20reveals%20earliest%20form%20of%20sex%20chromosome&amp;author=R.B.%20Spigler&amp;author=K.S.%20Lewers&amp;author=D.S.%20Main&amp;author=T.L.%20Ashman&amp;publication_year=2008&amp;journal=Heredity&amp;volume=101&amp;pages=507-17" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/hdy.2008.100" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fhdy.2008.100" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fhdy.2008.100"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18797475" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS035C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS035C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sargent</div> <div class="given-names">D.J.</div></div>,  <div class="name"><div class="surname">Kuchta</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Lopez Girona</div> <div class="given-names">E.</div></div>, et al. </div><div class="article-title">Simple sequence repeat marker development and mapping targeted to previously unmapped regions of the strawberry genome sequence</div>, <div class="source">Plant Genome</div> , <div class="year">2011</div>, vol. <div class="volume">4</div> (pg. <div class="fpage">1</div>-<div class="lpage">13</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Simple%20sequence%20repeat%20marker%20development%20and%20mapping%20targeted%20to%20previously%20unmapped%20regions%20of%20the%20strawberry%20genome%20sequence&amp;author=D.J.%20Sargent&amp;author=P.%20Kuchta&amp;author=E.%20Lopez%20Girona&amp;publication_year=2011&amp;journal=Plant%20Genome&amp;volume=4&amp;pages=1-13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3835/plantgenome2011.05.0014" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3835%2Fplantgenome2011.05.0014" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3835%2Fplantgenome2011.05.0014"> </span></div> </div></div></div></div></div><div content-id="DSS035C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS035C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chevreux</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Pfisterer</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Drescher</div> <div class="given-names">B.</div></div>, et al. </div><div class="article-title">Using the miraEST assembler for reliable and automated mRNA transcript assembly and SNP detection in sequenced ESTs</div>, <div class="source">Genome Res.</div> , <div class="year">2004</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">1147</div>-<div class="lpage">59</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Using%20the%20miraEST%20assembler%20for%20reliable%20and%20automated%20mRNA%20transcript%20assembly%20and%20SNP%20detection%20in%20sequenced%20ESTs&amp;author=B.%20Chevreux&amp;author=T.%20Pfisterer&amp;author=B.%20Drescher&amp;publication_year=2004&amp;journal=Genome%20Res.&amp;volume=14&amp;pages=1147-59" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.1917404" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.1917404" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.1917404"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15140833" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS035C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS035C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sato</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Isobe</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Asamizu</div> <div class="given-names">E.</div></div>, et al. </div><div class="article-title">Comprehensive structural analysis of the genome of red clover (<em>Trifolium pretense</em> L.)</div>, <div class="source">DNA Res.</div> , <div class="year">2005</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">301</div>-<div class="lpage">64</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comprehensive%20structural%20analysis%20of%20the%20genome%20of%20red%20clover%20(Trifolium%20pretense%20L.)&amp;author=S.%20Sato&amp;author=S.%20Isobe&amp;author=E.%20Asamizu&amp;publication_year=2005&amp;journal=DNA%20Res.&amp;volume=12&amp;pages=301-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsi018" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsi018" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsi018"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16769692" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS035C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS035C42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Van Ooijen</div> <div class="given-names">J.W.</div></div>. </div>, <div class="source">JoinMAP® 4, Software for the Calculation of Genetic Linkage Maps in Experimental Populations</div> , <div class="year">2006</div><div class="publisher-loc">Wageningen, Netherlands</div><div class="publisher-name">Kyazma B.V.,</div></div></div></div></div><div content-id="DSS035C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS035C43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kiss</div> <div class="given-names">G.B.</div></div>,  <div class="name"><div class="surname">Kereszt</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Kiss</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Endre</div> <div class="given-names">G.</div></div>. </div><div class="article-title">Colormapping; a non-mathematical procedure for genetic mapping</div>, <div class="source">Acta Biol. Hung.</div> , <div class="year">1998</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">125</div>-<div class="lpage">42</div>)</div></div></div></div><div content-id="DSS035C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS035C44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kunihisa</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Studies using DNA markers in <em>F</em>. times <em>ananassa</em>: genetic analysis, genome structure, and cultivar identification</div>, <div class="source">J. Jpn. Soc. Hortic. Sci.</div> , <div class="year">2011</div>, vol. <div class="volume">80</div> (pg. <div class="fpage">231</div>-<div class="lpage">43</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Studies%20using%20DNA%20markers%20in%20F.%20times%20ananassa%3A%20genetic%20analysis%2C%20genome%20structure%2C%20and%20cultivar%20identification&amp;author=M.%20Kunihisa&amp;publication_year=2011&amp;journal=J.%20Jpn.%20Soc.%20Hortic.%20Sci.&amp;volume=80&amp;pages=231-43" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2503/jjshs1.80.231" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2503%2Fjjshs1.80.231" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2503%2Fjjshs1.80.231"> </span></div> </div></div></div></div></div><div content-id="DSS035C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS035C45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">van Berloo</div> <div class="given-names">R.</div></div>. </div><div class="article-title">GGT 2.0: versatile Software for visualization and analysis of genetic data</div>, <div class="source">J. Hered.</div> , <div class="year">2008</div>, vol. <div class="volume">99</div> (pg. <div class="fpage">232</div>-<div class="lpage">236</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GGT%202.0%3A%20versatile%20Software%20for%20visualization%20and%20analysis%20of%20genetic%20data&amp;author=R.%20van%20Berloo&amp;publication_year=2008&amp;journal=J.%20Hered.&amp;volume=99&amp;pages=232-236" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jhered/esm109" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjhered%2Fesm109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjhered%2Fesm109"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18222930" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS035C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS035C46" class="ref-content "><div class="label -label">46</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tamura</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Peterson</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Peterson</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Stecher</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Nei</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Kumar</div> <div class="given-names">S.</div></div>. </div><div class="article-title">MEGA5: Molecular Evolutionary Genetics Analysis using likelihood, distance, and parsimony methods</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2011</div>, vol. <div class="volume">28</div> (pg. <div class="fpage">2731</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MEGA5%3A%20Molecular%20Evolutionary%20Genetics%20Analysis%20using%20likelihood%2C%20distance%2C%20and%20parsimony%20methods&amp;author=K.%20Tamura&amp;author=D.%20Peterson&amp;author=N.%20Peterson&amp;author=G.%20Stecher&amp;author=M.%20Nei&amp;author=S.%20Kumar&amp;publication_year=2011&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=28&amp;pages=2731-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msr121" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsr121" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsr121"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21546353" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS035C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS035C47" class="ref-content "><div class="label -label">47</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nathewet</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Yanagi</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Iwatsubo</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Sone</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Tamura</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Okuda</div> <div class="given-names">N.</div></div>. </div><div class="article-title">Karyotype analysis in octoploid and decaploid wild strawberries in <em>Fragaria</em> (Rosaceae)</div>, <div class="source">Cytologia</div> , <div class="year">2010</div>, vol. <div class="volume">75</div> (pg. <div class="fpage">277</div>-<div class="lpage">88</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Karyotype%20analysis%20in%20octoploid%20and%20decaploid%20wild%20strawberries%20in%20Fragaria%20(Rosaceae)&amp;author=P.%20Nathewet&amp;author=T.%20Yanagi&amp;author=Y.%20Iwatsubo&amp;author=K.%20Sone&amp;author=T.%20Tamura&amp;author=N.%20Okuda&amp;publication_year=2010&amp;journal=Cytologia&amp;volume=75&amp;pages=277-88" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1508/cytologia.75.277" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1508%2Fcytologia.75.277" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1508%2Fcytologia.75.277"> </span></div> </div></div></div></div></div><div content-id="DSS035C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS035C48" class="ref-content "><div class="label -label">48</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nathewet</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Iwatsubo</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Sone</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Hummer</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Yanagi</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Chromosome numbers and morphology in Japanese strawberry cultivars</div>, <div class="source">Hortic. Res.</div> , <div class="year">2008</div>, vol. <div class="volume">7</div> pg. <div class="fpage">195</div> </div></div></div></div><div content-id="DSS035C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS035C49" class="ref-content "><div class="label -label">49</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kimura</div> <div class="given-names">M.</div></div>. </div>, <div class="source">History of Japanese Strawberry Varieties and their Adaptability, Yasai-Engei Dai-Hyakka</div> , <div class="year">2004</div><div class="publisher-loc">Japan</div><div class="publisher-name">Nou-Bunkyou</div>(pg. <div class="fpage">116</div>-<div class="lpage">7</div>) <div class="comment">(in Japanese) ISBN 4-540-03284-4</div></div></div></div></div><div content-id="DSS035C50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS035C50" class="ref-content "><div class="label -label">50</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Govan</div> <div class="given-names">C.L.</div></div>,  <div class="name"><div class="surname">Simpson</div> <div class="given-names">D.W.</div></div>,  <div class="name"><div class="surname">Johnson</div> <div class="given-names">A.W.</div></div>,  <div class="name"><div class="surname">Tobutt</div> <div class="given-names">K.R.</div></div>,  <div class="name"><div class="surname">Sargent</div> <div class="given-names">D.J.</div></div>. </div><div class="article-title">A reliable multiplexed microsatellite set for genotyping Fragaria and its use in a survey of 60 <em>F</em>. ×<em>ananassa</em> cultivars</div>, <div class="source">Mol. Breeding</div> , <div class="year">2008</div>, vol. <div class="volume">22</div> (pg. <div class="fpage">649</div>-<div class="lpage">61</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20reliable%20multiplexed%20microsatellite%20set%20for%20genotyping%20Fragaria%20and%20its%20use%20in%20a%20survey%20of%2060%20F.%20%C3%97ananassa%20cultivars&amp;author=C.L.%20Govan&amp;author=D.W.%20Simpson&amp;author=A.W.%20Johnson&amp;author=K.R.%20Tobutt&amp;author=D.J.%20Sargent&amp;publication_year=2008&amp;journal=Mol.%20Breeding&amp;volume=22&amp;pages=649-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11032-008-9206-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11032-008-9206-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11032-008-9206-2"> </span></div> </div></div></div></div></div></div><div content-id=""><div class="fn"><div class="footnote-content">Edited by Dr Masahiro Yano</div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2012. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/3.0/), which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=383877&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss035/2/dss035-dss035supp_table7.xlsx?Expires=2147483647&amp;Signature=iqJKB8HERgXQW3uwU8XkpuNVpnh3hxpQufz7J4uVVrrRIueAVyAdX05ZrYYynYxd0aFzZZ5poK5g9T9HZEBL-s3womBo1j1wBeQOsRTSMv-w0fuPafxL6r7Ml7VHxS45l1cMkpALDpH7Oo6r7BJ7VjGLsXMaQ2lZiiVsWdnNEwfXgP~xsFmb-wMxRBf9pxEi3qX06BOtPfyVDuBMy7D0r4EuSFhtEfcLfeyV60IFrfBLfBhzjiL3YSufch~U3vqS7VkMShkR322so5N1wqnJcWxDhCN~gRP49MajFINPMIm10hfE~7VflYyEEmdMAFWXPeOQxz3-DH8pKK~gJQ1bVQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-13</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss035/2/dss035-dss035supp_table6.xls?Expires=2147483647&amp;Signature=Rna2RJJaorsICooz9yUJzBw66IwhODGNZFYX~l1lBMeCS~1FWM8FD-F3Wj6UHORkn4w7Lud4lUx~ph0QL-3JWnfAJtD~D~87JQAxGKJ5yp4vvPQ4SfqPdh-OC1Y3bKwwszn-wttLX2bAlgDk2xGz3PUzEGLMuQLpVdZ-B9lnYTUUV1tubIwu1KINyIA7oGAZ-29gPuA5HYIVkR3ESd-qm4CxNuOQ5yVXEvHTDAixO42E0rmY7JJm~8MVjACXbyUCqknIxx480HrOmvagWkxb~sFBGeWnyXNTIPqJMsQM55MycOFeHW2XScJArjHEBJI2xEJwfnb6LCwdp~4nXmh4IQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-12</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss035/2/dss035-dss035supp_table5.xlsx?Expires=2147483647&amp;Signature=nurqR3~VbYQkBR0fS-31-9QQ04zLWnFPB6-3MQ3ILila~~wlKW7Z1KXOErfAdbzOz5TYksU1gv8LlSg4vnpIZbqMW0T3x~Aik9iErgKZgcZSRSw6RdHnVt7-f6PdY6-RijYbEqqbPA78TT28hqEB~QHxQmbYKvCXanAGuSPDaY7E0DU5p8iB0UpTaaVBklM2eyBIMx31WiiGq5PT7e5EKvRqAzS5xO-VuOespXrELUrZKjt7g9iwCa0qgnj7hTTpRPU7bUVzCXMqN2Dn200uGUBIRa6BXRybVu~~x3wHU4mEKY8pMNYXMABDf34Ndlq47BrYSrlWT15HSNw~Wb2Abw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-11</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss035/2/dss035-dss035supp_table4.xlsx?Expires=2147483647&amp;Signature=IxObcIArdog-7dRR57TqWfZlRJkZcF~XzqVWwxyCLw1H6dYiz-6L3sdiz2m1yp-fOCINlNMKI9OtegboyABclakbZXDuLo2h7sklHSR2CzEIHD9S~wUQqZ-2yX6V5xDUmfWim~4BhMFgqqbI-viVZnSfq3sIGz4TfAbYkrxk1adSCg7SeYGr81wN8gIGPrqensE~KeOlDU8TqfGeyknR3VowukfZ0vW50lQPbqRi5qye~2b1HBJYmpEAgvZa2-CXV1ZANOXVQsM53uJ~-UT~c1qJhmWdsgEuJKzjpFcH7NHP-SqLVR6koh8gbqQv5ihvn8rZM2mL~FkxI2iyTW11~w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-10</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss035/2/dss035-dss035supp_table3.xlsx?Expires=2147483647&amp;Signature=h6Y6-G6hnFZfxI0OfmmQvPzJ1ND1J7CYfBcFyBDHWTGFnBEQXiIQLu9EppewVyksQaPLWZTxDNyZDmG3KDkAaa0Bm0YoVP0evxaHR4kXmqd5L5MiP9uIOZDRJeq~xNSNtfIk36hUJReixGUa2XMkp5kba1-Xe7zyTCf19zlFFBshVqdFgAhAtodc4wMBi-dyInwPqSbck1XUNi3RmRm4IG1YADcnsetYv97cKdnhLkXtg-VJiOHWoxUec706BEe2qssQCYxzXbt~a5fOLjPh9femHdGYmr7uo0iBFARryULSloENoVpaKfrIN8n2OCdlB1M8xI89YwGN8y3ZfyMovQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-9</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss035/2/dss035-dss035supp_table2.xlsx?Expires=2147483647&amp;Signature=3PiM~AChhAqZFXzU7D4L5m3f5OLQXa~Y7jSzemcdYyVprIOpRsxNbYOYsaxH6JhwFhpZkGMrR~BG5FMHKT~ZwLjWSxMeH4URoyFnX9GyJlJgvRE5WW9bmx1TJ2MKIsXx5WTUl-LRiiVjpjDQVd9e4zApfzorf7GaJU~vOVQvvrYsSDnPPDZPH2wiZ2hzsZgESJMRuEgDEpvEbmWDwQqcnz6KFjEO1gzw5lDZSXEG6UrzogCFHiWA~TV4O3mlUZU-1vWAh5hX0bV14S1GR929f9GhgykMwXaIixdBM6Z1TFuYdmOZ18B0X2hpxO6cZTJYitlm8VUj8N2M5-l38MT2bA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-8</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss035/2/dss035-dss035supp_table1.xlsx?Expires=2147483647&amp;Signature=oP21OflwJ7iRrvLPYHzxCWj9HFq0NTLzCnDc3KlFmq1w5LYOSeSfTrbtGzKlNJ9AubWCe4S6ROSWwm26ffrXMUYO9eXDwUJsW9GjO1QYgOmzwXhokGwYA9CUpi0aZYLJKQhwER2rB210fJEbWtMgRlG6SH8SSXw49~uZy1kt89B-vYcJFDEubKOYT3Z0I1HJwbhFrf~leHfqD4HaY80gSPQqjO05pxw9y-DNU3t6qLKj57YgkAVycrsZjEvvQELJPms0ROg04Mlvmcjg-tLxT0EqCU7dK7~yeHujerGHzeuqRoanXYddNWZYcDcCfaQLmnph9uoYo38w8ExQ1pepUg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-7</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss035/2/dss035-dss035supp_fig5.ppt?Expires=2147483647&amp;Signature=4ANokS9fS1065hCloE2prPgzZqRuTpozOzcbvnNdl~xoqVxeuFmP~ccSLjLVH-rm0tqyjUssDQjuXZBmKnEomyXDMBDfh7prOxgPBeT17Yq2~9pAaIXxokBDuz8s5MChn8TNmAwOyexOMbuKfivsd3nlchCN7Js~TS2SMEVdTYE-w62TzNCEMwexUk1~oZIaDUDM2xJx3qOPxDQhZ9hO0d31sN5PHOXV1KU~L4vWuLIUafHHWOduivyfUeXUvROb1ivIsTHThGAJk8rsaky0VNrI8R7~iSeUZZEnH1Y74fOV2-hWti49Qyv9zcOYLGQjCFOZBn7CBrBnGLUdGnLmnA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-6</a> - ppt file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss035/2/dss035-dss035supp_fig4.ppt?Expires=2147483647&amp;Signature=M5ZWvfNQXwtuTbMXv00DeHrVI8toY58ZnTaApSPL7bioHFQ0tio4fJiaMOYcytCdBlwV8dFaogHdMk4FNRcVzIkO2P4zPdTFQrZhVr4JmaLw7R5rRO1g4Ouo8tHQtzCylgbp30wtPQpRKOzF6MRjN7ecsZfQ3FhJhN4eoeNrFQGmj7BwBrCdKLbikylzJ5SDYbrEbPGtrj~YLXSK1JHrpibaSJTpa77rTfn3adIWv~Dcr7dj8Nmt1u6KWvxLTThSyGQmo3Z8xEDO4zVhgNC6R-IomFQ3WrjRwrROccTV2boll6YXH5rxs9ba1WKUnr~cU8p0Jc9mybf4W1og9~PkmQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-5</a> - ppt file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss035/2/dss035-dss035supp_fig3b.pdf?Expires=2147483647&amp;Signature=PouumpRU~iSadxfdP2bg5o1KV~JQxmkEc0TD-3mb5S2pudD9IBsypTI0WlTHKO1f9Bw0TpKK2QvU1JYkIvhqqmN~MMfdvztB7EZnDR69IpI9s11q8IqHttGYA2hJk0~WUwAbcKM3LRaT0b7dig5bZ5HphDY2mECC40jLoFZ-cQSDkRZB6t0ykAu~1TlVVPyVs~wSVPAt6XDm3iHn8nFNSmsQJiL0h7M583eS~NpcCpKQgMGSo8nzMuKoJIXUlYUpad8gqX4B07Ls~o-2MUSl8hTqjpu8kwXGxciui2tNSGKSUb9Vmdlowx8hc9x1JqcEn5Asei1D4YQdMgnBFgm5Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-4</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss035/2/dss035-dss035supp_fig3a.pdf?Expires=2147483647&amp;Signature=ES6eWgmJFh0M4QrPVJvI66JUHvKDAK8oNph9aICJhQ8LacnN8OIbqlFR7BKRRyNvM6KX1Wr9BJWrF8x-D0Lsx7Z-72MAGdnvwrGgp1g3qZ-1bWVfp7DjZNCp7vpa9ExS9QKTfSk3zEhjYdmOnJGVBSUMBz-JCRNdwBDzUjuqdgKYWhtSW3rKtUKG4ndhom8EoIH-NnyU5njXmi9xwM1xRgBa-YCM5QpD4I9s9Alqwn9qQ6NpfOlqy2Nz4Mpnq2M7Ay7MWk20KkxFgQsuc1hOmpapNMg7zrj9NiI9nUwQUZl6ityHIQSrGo3Y~83quwImG5o5MnVAm7rIwVX9jK-w6w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss035/2/dss035-dss035supp_fig2.ppt?Expires=2147483647&amp;Signature=ld4sjhdBvXNDrmQ8qC5YPDsea5nYI1l3kpEDM~8iklgzG~5EydKEjeib1gxySSri46Pxf2UUXAI93FTejpERSImvqT~KacGdV8UiJl~LgDwyys-ICosuamlkkcSXjCdt9bioLtaMJKwoqdjabYBvidCsU8cZvb9ZMNhcIagFaujXNaSjgPECuD0Vo4VWRJPtoS1S8GbXmEo9EyFvjM5GOnzS2D6cycIJGvfUvK-P9k14u31di55V3sDsZIgRkWANBtm7H205NK5wPgP1E6hYlDwV60HuVIP4jVnOSSndedtQ6WWOU6MrPjnBRMkyRmXA16W7Bau0LOPEd9c3yN5O1w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - ppt file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss035/2/dss035-dss035supp_fig1.pdf?Expires=2147483647&amp;Signature=y3YD4yEbPgI5-VhfTOaoQcnDASIdvSKMuXW28ROXVX7Lh86Qw-E1wIS4M5fK0bCSGxpBW9gT7uUvQDk6t--~qyeQidhXJ4lHwKZZwoYGOLqftuXuetCi5NCqHTwfEGwwfcgoz6cyV6QLbNkwksvzghehiIuZDDCiDPRf0tIkLq6uJ4E6whcVSp845nBmGAq2kj-8NMfzd7Elv-lffBg1dNj6Ygcanuq7-t8Kq~u2ZIT2VDz-uYys29HrrfEm3H4ncSAHK8fLBi1gj4DQOzBO02CU9rT44ZuChK6HCFSemsRLxAISAsikqHC5QXmOZxedcDQ60XWS0rjzRnHtjFRbEw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">357</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000315666200007&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">37</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dss035"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="383877"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000315666200007&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f20%2f1%2f79%2f383877">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000315666200007&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (37)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F20%2F1%2F79%2F383877" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/383877'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dss035");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f20%2f1%2f79%2f383877">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

