

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/17/2/73/344357","@type":"ScholarlyArticle","name":"Aligning a New Reference Genetic Map of  Lupinus angustifolius  with the Genome Sequence of the Model Legume,  Lotus japonicus ","datePublished":"2010-02-04","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/17/2","@type":"PublicationIssue","issueNumber":"2","datePublished":"2010-04-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsq001","keywords":["synteny","genome evolution","narrow-leafed lupin","Fabaceae"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/17/2/73/344357","author":[{"name":"Nelson, Matthew N.","affiliation":"1  School of Plant Biology and International Centre for Plant Breeding Education and Research, The University of Western Australia ,  35 Stirling Highway, Crawley, WA 6009 ,  Australia","@type":"Person"},{"name":"Moolhuijzen, Paula M.","affiliation":"2  Centre for Comparative Genomics, Murdoch University ,  Perth, WA 6150 ,  Australia","@type":"Person"},{"name":"Boersma, Jeffrey G.","affiliation":"3  School of Earth and Environment, The University of Western Australia ,  35 Stirling Highway, Crawley, WA 6009 ,  Australia    4  Department of Agriculture and Food Western Australia ,  South Perth, WA 6151 ,  Australia","@type":"Person"},{"name":"Chudy, Magdalena","affiliation":"5  Institute of Plant Genetics, Polish Academy of Sciences ,  60-479 Poznan ,  Poland","@type":"Person"},{"name":"Lesniewska, Karolina","affiliation":"5  Institute of Plant Genetics, Polish Academy of Sciences ,  60-479 Poznan ,  Poland","@type":"Person"},{"name":"Bellgard, Matthew","affiliation":"2  Centre for Comparative Genomics, Murdoch University ,  Perth, WA 6150 ,  Australia","@type":"Person"},{"name":"Oliver, Richard P.","affiliation":"6  Australian Centre for Necrotrophic Fungal Pathogens, WA State Agricultural Biotechnology Centre, Murdoch University ,  Perth, WA 6150 ,  Australia","@type":"Person"},{"name":"Święcicki, Wojciech","affiliation":"5  Institute of Plant Genetics, Polish Academy of Sciences ,  60-479 Poznan ,  Poland","@type":"Person"},{"name":"Wolko, Bogdan","affiliation":"5  Institute of Plant Genetics, Polish Academy of Sciences ,  60-479 Poznan ,  Poland","@type":"Person"},{"name":"Cowling, Wallace A.","affiliation":"1  School of Plant Biology and International Centre for Plant Breeding Education and Research, The University of Western Australia ,  35 Stirling Highway, Crawley, WA 6009 ,  Australia","@type":"Person"},{"name":"Ellwood, Simon R.","affiliation":"6  Australian Centre for Necrotrophic Fungal Pathogens, WA State Agricultural Biotechnology Centre, Murdoch University ,  Perth, WA 6150 ,  Australia","@type":"Person"}],"description":"Abstract.  We have developed a dense reference genetic map of Lupinus angustifolius (2n = 40) based on a set of 106 publicly available recombinant inbred lines ","pageStart":"73","pageEnd":"83","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/17/2/1/m_cover.gif?Expires=2147483647&Signature=IwfbTE7xRHpYPiWcYGpjFczNWogtXNkuUzyZoZWesXrHyOZIOeBkj1ptgmT6KB08Jodccr-wVIzmNzuprSuMFXnqyC7YwVUcQszfCHplBi2Y15gmfPlURCcr6cDrmuXI2RQ8kVYqbpwpkwU~IwAx-IxlD-SxvRJNg-rOnz1kFBZg6~ril-XCQUH57KgRmoLRdZOtq3kyqIqmR0akjn~EEqO1Zof8u6SDW3r-Givc~Q~FFDVgycai8wbeHX4jSo~RgtJ5fJEZ3leTfm3ID7Dx-QOBdgGBsB4Cf3hQvrSTBbbP5crdkbS1Y16Wy8VT~3dXnXyQP4~X-n136OwpQnKewg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Aligning a New Reference Genetic Map of  Lupinus angustifolius  with the Genome Sequence of the Model Legume,  Lotus japonicus ","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/17/2/1/m_cover.gif?Expires=2147483647&Signature=IwfbTE7xRHpYPiWcYGpjFczNWogtXNkuUzyZoZWesXrHyOZIOeBkj1ptgmT6KB08Jodccr-wVIzmNzuprSuMFXnqyC7YwVUcQszfCHplBi2Y15gmfPlURCcr6cDrmuXI2RQ8kVYqbpwpkwU~IwAx-IxlD-SxvRJNg-rOnz1kFBZg6~ril-XCQUH57KgRmoLRdZOtq3kyqIqmR0akjn~EEqO1Zof8u6SDW3r-Givc~Q~FFDVgycai8wbeHX4jSo~RgtJ5fJEZ3leTfm3ID7Dx-QOBdgGBsB4Cf3hQvrSTBbbP5crdkbS1Y16Wy8VT~3dXnXyQP4~X-n136OwpQnKewg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Aligning a New Reference Genetic Map of  Lupinus angustifolius  with the Genome Sequence of the Model Legume,  Lotus japonicus " />
    <meta property="og:description" content="Abstract.  We have developed a dense reference genetic map of Lupinus angustifolius (2n = 40) based on a set of 106 publicly available recombinant inbred lines " />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/17/2/73/344357" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2010-02-04" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Nelson, Matthew N." /><meta name="citation_author_institution" content="School of Plant Biology and International Centre for Plant Breeding Education and Research, The University of Western Australia, 35 Stirling Highway, Crawley, WA 6009, Australia" /><meta name="citation_author" content="Moolhuijzen, Paula M." /><meta name="citation_author_institution" content="Centre for Comparative Genomics, Murdoch University, Perth, WA 6150, Australia" /><meta name="citation_author" content="Boersma, Jeffrey G." /><meta name="citation_author_institution" content="School of Earth and Environment, The University of Western Australia, 35 Stirling Highway, Crawley, WA 6009, Australia" /><meta name="citation_author_institution" content="Department of Agriculture and Food Western Australia, South Perth, WA 6151, Australia" /><meta name="citation_author" content="Chudy, Magdalena" /><meta name="citation_author_institution" content="Institute of Plant Genetics, Polish Academy of Sciences, 60-479 Poznan, Poland" /><meta name="citation_author" content="Lesniewska, Karolina" /><meta name="citation_author_institution" content="Institute of Plant Genetics, Polish Academy of Sciences, 60-479 Poznan, Poland" /><meta name="citation_author" content="Bellgard, Matthew" /><meta name="citation_author_institution" content="Centre for Comparative Genomics, Murdoch University, Perth, WA 6150, Australia" /><meta name="citation_author" content="Oliver, Richard P." /><meta name="citation_author_institution" content="Australian Centre for Necrotrophic Fungal Pathogens, WA State Agricultural Biotechnology Centre, Murdoch University, Perth, WA 6150, Australia" /><meta name="citation_author" content="Święcicki, Wojciech" /><meta name="citation_author_institution" content="Institute of Plant Genetics, Polish Academy of Sciences, 60-479 Poznan, Poland" /><meta name="citation_author" content="Wolko, Bogdan" /><meta name="citation_author_institution" content="Institute of Plant Genetics, Polish Academy of Sciences, 60-479 Poznan, Poland" /><meta name="citation_author" content="Cowling, Wallace A." /><meta name="citation_author_institution" content="School of Plant Biology and International Centre for Plant Breeding Education and Research, The University of Western Australia, 35 Stirling Highway, Crawley, WA 6009, Australia" /><meta name="citation_author" content="Ellwood, Simon R." /><meta name="citation_author_institution" content="Australian Centre for Necrotrophic Fungal Pathogens, WA State Agricultural Biotechnology Centre, Murdoch University, Perth, WA 6150, Australia" /><meta name="citation_title" content="Aligning a New Reference Genetic Map of Lupinus angustifolius with the Genome Sequence of the Model Legume, Lotus japonicus" /><meta name="citation_firstpage" content="73" /><meta name="citation_lastpage" content="83" /><meta name="citation_doi" content="10.1093/dnares/dsq001" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="17" /><meta name="citation_issue" content="2" /><meta name="citation_publication_date" content="2010/04/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Construction of a comparative genetic map in faba bean (Vicia faba L.); conservation of genome structure with Lens culinaris; BMC Genomics;  citation_year=2008;  citation_volume=9;  citation_pages=380" /><meta name="citation_reference" content="citation_title=Extensive macrosynteny between Medicago truncatula and Lens culinaris ssp. culinaris; Theor. Appl. Genet.;  citation_year=2007;  citation_volume=114;  citation_pages=549-58; " /><meta name="citation_reference" content="citation_title=Sequencing the genespaces of Medicago truncatula and Lotus japonicus; Plant Physiol.;  citation_year=2005;  citation_volume=137;  citation_pages=1174-81; " /><meta name="citation_reference" content="citation_title=Estimating genome conservation between crop and model legume species; Proc. Natl Acad. Sci. USA;  citation_year=2004;  citation_volume=101;  citation_pages=15289-94; " /><meta name="citation_reference" content="citation_title=Evolutionary rates analysis of Leguminosae implicates a rapid diversification of lineages during the Tertiary; Syst. Biol.;  citation_year=2005;  citation_volume=54;  citation_pages=575-94; " /><meta name="citation_reference" content="citation_title=Nuclear DNA content variation and species relationships in the genus Lupinus (Fabaceae); Ann. Bot.;  citation_year=2003;  citation_volume=92;  citation_pages=349-55; " /><meta name="citation_reference" content="citation_title=The first gene-based map of Lupinus angustifolius L.—location of domestication genes and conserved synteny with Medicago truncatula; Theor. Appl. Genet.;  citation_year=2006;  citation_volume=113;  citation_pages=225-38; " /><meta name="citation_reference" content="citation_title=Estimation of Lupinus genome polyploidy on the basis of isozymic loci number; Genet. Pol.;  citation_year=1989;  citation_volume=30;  citation_pages=165-71; " /><meta name="citation_reference" content="citation_title=Construction of a genetic linkage map using MFLP and identification of molecular markers linked to domestication genes in narrow-leafed lupin (Lupinus angustifolius L.); Cell. Mol. Biol. Lett.;  citation_year=2005;  citation_volume=10;  citation_pages=331-44; " /><meta name="citation_reference" content="citation_title=Development of sequence-specific PCR markers linked to the Tardus gene that reduces pod shattering in narrow-leafed lupin (Lupinus angustifolius L.); Mol. Breed.;  citation_year=2009;  citation_volume=23;  citation_pages=259-67; " /><meta name="citation_reference" content="citation_title=Development of two sequence-specific PCR markers linked to the le gene that reduces pod shattering in narrow-leafed Lupin (Lupinus angustifolius L.); Genet. Mol. Biol.;  citation_year=2007;  citation_volume=30;  citation_pages=623-9; " /><meta name="citation_reference" content="citation_title=Development of a sequence-specific PCR marker linked to the Ku gene which removes the vernalization requirement in narrow-leafed lupin; Plant Breed.;  citation_year=2007;  citation_volume=126;  citation_pages=306-9; " /><meta name="citation_reference" content="citation_title=Development of a PCR marker tightly linked to mollis, the gene that controls seed dormancy in Lupinus angustifolius L; Plant Breed.;  citation_year=2007;  citation_volume=126;  citation_pages=612-6; " /><meta name="citation_reference" content="citation_title=A PCR-based molecular marker applicable for marker-assisted selection for anthracnose disease resistance in lupin breeding; Cell. Mol. Biol. Lett.;  citation_year=2005;  citation_volume=10;  citation_pages=123-34; " /><meta name="citation_reference" content="citation_title=Karyotyping of the narrow-leafed lupin (Lupinus angustifolius L.) by using FISH, PRINS and computer measurements of chromosomes; J. Appl. Genet.;  citation_year=2009;  citation_volume=50;  citation_pages=77-82; " /><meta name="citation_reference" content="citation_title=Physical mapping of 18S-25S rDNA and 5S rDNA in Lupinus via fluorescent in situ hybridization; Cell. Mol. Biol. Lett.;  citation_year=2002;  citation_volume=7;  citation_pages=665-70; " /><meta name="citation_reference" content="citation_title=Genome structure of the legume, Lotus japonicus; DNA Res.;  citation_year=2008;  citation_volume=15;  citation_pages=227-39; " /><meta name="citation_reference" content="citation_title=Development and implementation of a sequence-specific PCR marker linked to a gene conferring resistance to anthracnose disease in narrow-leafed lupin (Lupinus angustifolius L.); Mol. Breed.;  citation_year=2004;  citation_volume=14;  citation_pages=145-51; " /><meta name="citation_reference" content="citation_title=Linkage map of isozymes and RAPD markers for L. angustifolius L; citation_publisher=Evora, Portugal;  citation_year=1993; " /><meta name="citation_reference" content="citation_title=The bacterial artificial chromosome (BAC) library of the narrow-leafed lupin (Lupinus angustifolius L.); Cell. Mol. Biol. Lett.;  citation_year=2006;  citation_volume=11;  citation_pages=396-407; " /><meta name="citation_reference" content="citation_title=AFLP: A new technique for DNA fingerprinting; Nucleic Acids Res.;  citation_year=1995;  citation_volume=23;  citation_pages=4407-14; " /><meta name="citation_reference" content="citation_title=DNA fingerprinting based on microsatellite-anchored fragment length polymorphisms, and isolation of sequence-specific PCR markers in lupin (Lupinus angustifolius L.); Mol. Breed.;  citation_year=2001;  citation_volume=7;  citation_pages=203-9; " /><meta name="citation_reference" content="citation_title=Constructing large-scale genetic maps using an evolutionary strategy algorithm; Genetics;  citation_year=2003;  citation_volume=165;  citation_pages=2269-82; " /><meta name="citation_reference" content="citation_title=Basic local alignment search tool; J. Mol. Biol.;  citation_year=1990;  citation_volume=215;  citation_pages=403-10; " /><meta name="citation_reference" content="citation_title=A bioinformatics Web Service-based architecture for accessing HPC resources; Australasian Workshop on Grid Computing and e-Research;  citation_year=2005; " /><meta name="citation_reference" content="citation_title=LegumeDB bioinformatics resource: Comparative genomic analysis and novel cross-genera marker identification in lupin and pasture legume species; Genome;  citation_year=2006;  citation_volume=49;  citation_pages=689-99; " /><meta name="citation_reference" content="citation_title=New evidence from Sinapis alba L. for ancestral triplication in a crucifer genome; Genome;  citation_year=2006;  citation_volume=49;  citation_pages=230-8; " /><meta name="citation_reference" content="citation_title=An analysis of synteny of Arachis with Lotus and Medicago sheds new light on the structure, stability and evolution of legume genomes; BMC Genomics;  citation_year=2009;  citation_volume=10;  citation_pages=45" /><meta name="citation_reference" content="citation_title=Mechanisms of chromosome number reduction in Arabidopsis thaliana and related Brassicaceae species; Proc. Natl Acad. Sci. USA;  citation_year=2006;  citation_volume=103;  citation_pages=5224-9; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/17/2/73/984812/dsq001.pdf" /><meta name="description" content="Abstract.  We have developed a dense reference genetic map of Lupinus angustifolius (2n = 40) based on a set of 106 publicly available recombinant inbred lines " />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/17/2/73/344357" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/17/2/73/344357" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/17/2/73/344357");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsq001';
    </script>
<title>
	Aligning a New Reference Genetic Map of Lupinus angustifolius with the Genome Sequence of the Model Legume, Lotus japonicus | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./344357" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BY1VPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJOZWxzb24sIE1hdHRoZXcgTi4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJTY2hvb2wgb2YgUGxhbnQgQmlvbG9neSBhbmQgSW50ZXJuYXRpb25hbCBDZW50cmUgZm9yIFBsYW50IEJyZWVkaW5nIEVkdWNhdGlvbiBhbmQgUmVzZWFyY2gsIFRoZSBVbml2ZXJzaXR5IG9mIFdlc3Rlcm4gQXVzdHJhbGlhLCAzNSBTdGlybGluZyBIaWdod2F5LCBDcmF3bGV5LCBXQSA2MDA5LCBBdXN0cmFsaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJNb29saHVpanplbiwgUGF1bGEgTS4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJDZW50cmUgZm9yIENvbXBhcmF0aXZlIEdlbm9taWNzLCBNdXJkb2NoIFVuaXZlcnNpdHksIFBlcnRoLCBXQSA2MTUwLCBBdXN0cmFsaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJCb2Vyc21hLCBKZWZmcmV5IEcuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iU2Nob29sIG9mIEVhcnRoIGFuZCBFbnZpcm9ubWVudCwgVGhlIFVuaXZlcnNpdHkgb2YgV2VzdGVybiBBdXN0cmFsaWEsIDM1IFN0aXJsaW5nIEhpZ2h3YXksIENyYXdsZXksIFdBIDYwMDksIEF1c3RyYWxpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQWdyaWN1bHR1cmUgYW5kIEZvb2QgV2VzdGVybiBBdXN0cmFsaWEsIFNvdXRoIFBlcnRoLCBXQSA2MTUxLCBBdXN0cmFsaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJDaHVkeSwgTWFnZGFsZW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSW5zdGl0dXRlIG9mIFBsYW50IEdlbmV0aWNzLCBQb2xpc2ggQWNhZGVteSBvZiBTY2llbmNlcywgNjAtNDc5IFBvem5hbiwgUG9sYW5kIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTGVzbmlld3NrYSwgS2Fyb2xpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJJbnN0aXR1dGUgb2YgUGxhbnQgR2VuZXRpY3MsIFBvbGlzaCBBY2FkZW15IG9mIFNjaWVuY2VzLCA2MC00NzkgUG96bmFuLCBQb2xhbmQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJCZWxsZ2FyZCwgTWF0dGhldyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkNlbnRyZSBmb3IgQ29tcGFyYXRpdmUgR2Vub21pY3MsIE11cmRvY2ggVW5pdmVyc2l0eSwgUGVydGgsIFdBIDYxNTAsIEF1c3RyYWxpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik9saXZlciwgUmljaGFyZCBQLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkF1c3RyYWxpYW4gQ2VudHJlIGZvciBOZWNyb3Ryb3BoaWMgRnVuZ2FsIFBhdGhvZ2VucywgV0EgU3RhdGUgQWdyaWN1bHR1cmFsIEJpb3RlY2hub2xvZ3kgQ2VudHJlLCBNdXJkb2NoIFVuaXZlcnNpdHksIFBlcnRoLCBXQSA2MTUwLCBBdXN0cmFsaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSLFmndpxJljaWNraSwgV29qY2llY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJJbnN0aXR1dGUgb2YgUGxhbnQgR2VuZXRpY3MsIFBvbGlzaCBBY2FkZW15IG9mIFNjaWVuY2VzLCA2MC00NzkgUG96bmFuLCBQb2xhbmQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJXb2xrbywgQm9nZGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSW5zdGl0dXRlIG9mIFBsYW50IEdlbmV0aWNzLCBQb2xpc2ggQWNhZGVteSBvZiBTY2llbmNlcywgNjAtNDc5IFBvem5hbiwgUG9sYW5kIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQ293bGluZywgV2FsbGFjZSBBLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlNjaG9vbCBvZiBQbGFudCBCaW9sb2d5IGFuZCBJbnRlcm5hdGlvbmFsIENlbnRyZSBmb3IgUGxhbnQgQnJlZWRpbmcgRWR1Y2F0aW9uIGFuZCBSZXNlYXJjaCwgVGhlIFVuaXZlcnNpdHkgb2YgV2VzdGVybiBBdXN0cmFsaWEsIDM1IFN0aXJsaW5nIEhpZ2h3YXksIENyYXdsZXksIFdBIDYwMDksIEF1c3RyYWxpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkVsbHdvb2QsIFNpbW9uIFIuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQXVzdHJhbGlhbiBDZW50cmUgZm9yIE5lY3JvdHJvcGhpYyBGdW5nYWwgUGF0aG9nZW5zLCBXQSBTdGF0ZSBBZ3JpY3VsdHVyYWwgQmlvdGVjaG5vbG9neSBDZW50cmUsIE11cmRvY2ggVW5pdmVyc2l0eSwgUGVydGgsIFdBIDYxNTAsIEF1c3RyYWxpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iQWxpZ25pbmcgYSBOZXcgUmVmZXJlbmNlIEdlbmV0aWMgTWFwIG9mIEx1cGludXMgYW5ndXN0aWZvbGl1cyB3aXRoIHRoZSBHZW5vbWUgU2VxdWVuY2Ugb2YgdGhlIE1vZGVsIExlZ3VtZSwgTG90dXMgamFwb25pY3VzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iNzMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjgzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHNxMDAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjE3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSIyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTAvMDQvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db25zdHJ1Y3Rpb24gb2YgYSBjb21wYXJhdGl2ZSBnZW5ldGljIG1hcCBpbiBmYWJhIGJlYW4gKFZpY2lhIGZhYmEgTC4pOyBjb25zZXJ2YXRpb24gb2YgZ2Vub21lIHN0cnVjdHVyZSB3aXRoIExlbnMgY3VsaW5hcmlzOyBCTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTM4MCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV4dGVuc2l2ZSBtYWNyb3N5bnRlbnkgYmV0d2VlbiBNZWRpY2FnbyB0cnVuY2F0dWxhIGFuZCBMZW5zIGN1bGluYXJpcyBzc3AuIGN1bGluYXJpczsgVGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0xMTQ7ICBjaXRhdGlvbl9wYWdlcz01NDktNTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNlcXVlbmNpbmcgdGhlIGdlbmVzcGFjZXMgb2YgTWVkaWNhZ28gdHJ1bmNhdHVsYSBhbmQgTG90dXMgamFwb25pY3VzOyBQbGFudCBQaHlzaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMzc7ICBjaXRhdGlvbl9wYWdlcz0xMTc0LTgxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Fc3RpbWF0aW5nIGdlbm9tZSBjb25zZXJ2YXRpb24gYmV0d2VlbiBjcm9wIGFuZCBtb2RlbCBsZWd1bWUgc3BlY2llczsgUHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMDE7ICBjaXRhdGlvbl9wYWdlcz0xNTI4OS05NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXZvbHV0aW9uYXJ5IHJhdGVzIGFuYWx5c2lzIG9mIExlZ3VtaW5vc2FlIGltcGxpY2F0ZXMgYSByYXBpZCBkaXZlcnNpZmljYXRpb24gb2YgbGluZWFnZXMgZHVyaW5nIHRoZSBUZXJ0aWFyeTsgU3lzdC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9NTQ7ICBjaXRhdGlvbl9wYWdlcz01NzUtOTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU51Y2xlYXIgRE5BIGNvbnRlbnQgdmFyaWF0aW9uIGFuZCBzcGVjaWVzIHJlbGF0aW9uc2hpcHMgaW4gdGhlIGdlbnVzIEx1cGludXMgKEZhYmFjZWFlKTsgQW5uLiBCb3QuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTkyOyAgY2l0YXRpb25fcGFnZXM9MzQ5LTU1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZmlyc3QgZ2VuZS1iYXNlZCBtYXAgb2YgTHVwaW51cyBhbmd1c3RpZm9saXVzIEwu4oCUbG9jYXRpb24gb2YgZG9tZXN0aWNhdGlvbiBnZW5lcyBhbmQgY29uc2VydmVkIHN5bnRlbnkgd2l0aCBNZWRpY2FnbyB0cnVuY2F0dWxhOyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTExMzsgIGNpdGF0aW9uX3BhZ2VzPTIyNS0zODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXN0aW1hdGlvbiBvZiBMdXBpbnVzIGdlbm9tZSBwb2x5cGxvaWR5IG9uIHRoZSBiYXNpcyBvZiBpc296eW1pYyBsb2NpIG51bWJlcjsgR2VuZXQuIFBvbC47ICBjaXRhdGlvbl95ZWFyPTE5ODk7ICBjaXRhdGlvbl92b2x1bWU9MzA7ICBjaXRhdGlvbl9wYWdlcz0xNjUtNzE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbnN0cnVjdGlvbiBvZiBhIGdlbmV0aWMgbGlua2FnZSBtYXAgdXNpbmcgTUZMUCBhbmQgaWRlbnRpZmljYXRpb24gb2YgbW9sZWN1bGFyIG1hcmtlcnMgbGlua2VkIHRvIGRvbWVzdGljYXRpb24gZ2VuZXMgaW4gbmFycm93LWxlYWZlZCBsdXBpbiAoTHVwaW51cyBhbmd1c3RpZm9saXVzIEwuKTsgQ2VsbC4gTW9sLiBCaW9sLiBMZXR0LjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTMzMS00NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGV2ZWxvcG1lbnQgb2Ygc2VxdWVuY2Utc3BlY2lmaWMgUENSIG1hcmtlcnMgbGlua2VkIHRvIHRoZSBUYXJkdXMgZ2VuZSB0aGF0IHJlZHVjZXMgcG9kIHNoYXR0ZXJpbmcgaW4gbmFycm93LWxlYWZlZCBsdXBpbiAoTHVwaW51cyBhbmd1c3RpZm9saXVzIEwuKTsgTW9sLiBCcmVlZC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MjM7ICBjaXRhdGlvbl9wYWdlcz0yNTktNjc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldmVsb3BtZW50IG9mIHR3byBzZXF1ZW5jZS1zcGVjaWZpYyBQQ1IgbWFya2VycyBsaW5rZWQgdG8gdGhlIGxlIGdlbmUgdGhhdCByZWR1Y2VzIHBvZCBzaGF0dGVyaW5nIGluIG5hcnJvdy1sZWFmZWQgTHVwaW4gKEx1cGludXMgYW5ndXN0aWZvbGl1cyBMLik7IEdlbmV0LiBNb2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTMwOyAgY2l0YXRpb25fcGFnZXM9NjIzLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldmVsb3BtZW50IG9mIGEgc2VxdWVuY2Utc3BlY2lmaWMgUENSIG1hcmtlciBsaW5rZWQgdG8gdGhlIEt1IGdlbmUgd2hpY2ggcmVtb3ZlcyB0aGUgdmVybmFsaXphdGlvbiByZXF1aXJlbWVudCBpbiBuYXJyb3ctbGVhZmVkIGx1cGluOyBQbGFudCBCcmVlZC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTI2OyAgY2l0YXRpb25fcGFnZXM9MzA2LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldmVsb3BtZW50IG9mIGEgUENSIG1hcmtlciB0aWdodGx5IGxpbmtlZCB0byBtb2xsaXMsIHRoZSBnZW5lIHRoYXQgY29udHJvbHMgc2VlZCBkb3JtYW5jeSBpbiBMdXBpbnVzIGFuZ3VzdGlmb2xpdXMgTDsgUGxhbnQgQnJlZWQuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTEyNjsgIGNpdGF0aW9uX3BhZ2VzPTYxMi02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIFBDUi1iYXNlZCBtb2xlY3VsYXIgbWFya2VyIGFwcGxpY2FibGUgZm9yIG1hcmtlci1hc3Npc3RlZCBzZWxlY3Rpb24gZm9yIGFudGhyYWNub3NlIGRpc2Vhc2UgcmVzaXN0YW5jZSBpbiBsdXBpbiBicmVlZGluZzsgQ2VsbC4gTW9sLiBCaW9sLiBMZXR0LjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTEyMy0zNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9S2FyeW90eXBpbmcgb2YgdGhlIG5hcnJvdy1sZWFmZWQgbHVwaW4gKEx1cGludXMgYW5ndXN0aWZvbGl1cyBMLikgYnkgdXNpbmcgRklTSCwgUFJJTlMgYW5kIGNvbXB1dGVyIG1lYXN1cmVtZW50cyBvZiBjaHJvbW9zb21lczsgSi4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTUwOyAgY2l0YXRpb25fcGFnZXM9NzctODI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBoeXNpY2FsIG1hcHBpbmcgb2YgMThTLTI1UyByRE5BIGFuZCA1UyByRE5BIGluIEx1cGludXMgdmlhIGZsdW9yZXNjZW50IGluIHNpdHUgaHlicmlkaXphdGlvbjsgQ2VsbC4gTW9sLiBCaW9sLiBMZXR0LjsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9NjY1LTcwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUgc3RydWN0dXJlIG9mIHRoZSBsZWd1bWUsIExvdHVzIGphcG9uaWN1czsgRE5BIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz0yMjctMzk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldmVsb3BtZW50IGFuZCBpbXBsZW1lbnRhdGlvbiBvZiBhIHNlcXVlbmNlLXNwZWNpZmljIFBDUiBtYXJrZXIgbGlua2VkIHRvIGEgZ2VuZSBjb25mZXJyaW5nIHJlc2lzdGFuY2UgdG8gYW50aHJhY25vc2UgZGlzZWFzZSBpbiBuYXJyb3ctbGVhZmVkIGx1cGluIChMdXBpbnVzIGFuZ3VzdGlmb2xpdXMgTC4pOyBNb2wuIEJyZWVkLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTE0NS01MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TGlua2FnZSBtYXAgb2YgaXNvenltZXMgYW5kIFJBUEQgbWFya2VycyBmb3IgTC4gYW5ndXN0aWZvbGl1cyBMOyBjaXRhdGlvbl9wdWJsaXNoZXI9RXZvcmEsIFBvcnR1Z2FsOyAgY2l0YXRpb25feWVhcj0xOTkzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgYmFjdGVyaWFsIGFydGlmaWNpYWwgY2hyb21vc29tZSAoQkFDKSBsaWJyYXJ5IG9mIHRoZSBuYXJyb3ctbGVhZmVkIGx1cGluIChMdXBpbnVzIGFuZ3VzdGlmb2xpdXMgTC4pOyBDZWxsLiBNb2wuIEJpb2wuIExldHQuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9Mzk2LTQwNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QUZMUDogQSBuZXcgdGVjaG5pcXVlIGZvciBETkEgZmluZ2VycHJpbnRpbmc7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5NTsgIGNpdGF0aW9uX3ZvbHVtZT0yMzsgIGNpdGF0aW9uX3BhZ2VzPTQ0MDctMTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUROQSBmaW5nZXJwcmludGluZyBiYXNlZCBvbiBtaWNyb3NhdGVsbGl0ZS1hbmNob3JlZCBmcmFnbWVudCBsZW5ndGggcG9seW1vcnBoaXNtcywgYW5kIGlzb2xhdGlvbiBvZiBzZXF1ZW5jZS1zcGVjaWZpYyBQQ1IgbWFya2VycyBpbiBsdXBpbiAoTHVwaW51cyBhbmd1c3RpZm9saXVzIEwuKTsgTW9sLiBCcmVlZC47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTIwMy05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db25zdHJ1Y3RpbmcgbGFyZ2Utc2NhbGUgZ2VuZXRpYyBtYXBzIHVzaW5nIGFuIGV2b2x1dGlvbmFyeSBzdHJhdGVneSBhbGdvcml0aG07IEdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTE2NTsgIGNpdGF0aW9uX3BhZ2VzPTIyNjktODI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJhc2ljIGxvY2FsIGFsaWdubWVudCBzZWFyY2ggdG9vbDsgSi4gTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5MDsgIGNpdGF0aW9uX3ZvbHVtZT0yMTU7ICBjaXRhdGlvbl9wYWdlcz00MDMtMTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgYmlvaW5mb3JtYXRpY3MgV2ViIFNlcnZpY2UtYmFzZWQgYXJjaGl0ZWN0dXJlIGZvciBhY2Nlc3NpbmcgSFBDIHJlc291cmNlczsgQXVzdHJhbGFzaWFuIFdvcmtzaG9wIG9uIEdyaWQgQ29tcHV0aW5nIGFuZCBlLVJlc2VhcmNoOyAgY2l0YXRpb25feWVhcj0yMDA1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1MZWd1bWVEQiBiaW9pbmZvcm1hdGljcyByZXNvdXJjZTogQ29tcGFyYXRpdmUgZ2Vub21pYyBhbmFseXNpcyBhbmQgbm92ZWwgY3Jvc3MtZ2VuZXJhIG1hcmtlciBpZGVudGlmaWNhdGlvbiBpbiBsdXBpbiBhbmQgcGFzdHVyZSBsZWd1bWUgc3BlY2llczsgR2Vub21lOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTQ5OyAgY2l0YXRpb25fcGFnZXM9Njg5LTk5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OZXcgZXZpZGVuY2UgZnJvbSBTaW5hcGlzIGFsYmEgTC4gZm9yIGFuY2VzdHJhbCB0cmlwbGljYXRpb24gaW4gYSBjcnVjaWZlciBnZW5vbWU7IEdlbm9tZTsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT00OTsgIGNpdGF0aW9uX3BhZ2VzPTIzMC04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBhbmFseXNpcyBvZiBzeW50ZW55IG9mIEFyYWNoaXMgd2l0aCBMb3R1cyBhbmQgTWVkaWNhZ28gc2hlZHMgbmV3IGxpZ2h0IG9uIHRoZSBzdHJ1Y3R1cmUsIHN0YWJpbGl0eSBhbmQgZXZvbHV0aW9uIG9mIGxlZ3VtZSBnZW5vbWVzOyBCTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz00NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1lY2hhbmlzbXMgb2YgY2hyb21vc29tZSBudW1iZXIgcmVkdWN0aW9uIGluIEFyYWJpZG9wc2lzIHRoYWxpYW5hIGFuZCByZWxhdGVkIEJyYXNzaWNhY2VhZSBzcGVjaWVzOyBQcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTEwMzsgIGNpdGF0aW9uX3BhZ2VzPTUyMjQtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzE3LzIvNzMvOTg0ODEyL2RzcTAwMS5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgV2UgaGF2ZSBkZXZlbG9wZWQgYSBkZW5zZSByZWZlcmVuY2UgZ2VuZXRpYyBtYXAgb2YgTHVwaW51cyBhbmd1c3RpZm9saXVzICgybiA9IDQwKSBiYXNlZCBvbiBhIHNldCBvZiAxMDYgcHVibGljbHkgYXZhaWxhYmxlIHJlY29tYmluYW50IGluYnJlZCBsaW5lcyAiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVyLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTM0NDM1NyZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGQMo7aZQFBGz6pZm8pFyxI5qXkUyw==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f17%2f2%2f73%2f344357" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f17%2f2%2f73%2f344357" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=13234" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/17/2"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/17/2/1/m_cover.gif?Expires=2147483647&amp;Signature=IwfbTE7xRHpYPiWcYGpjFczNWogtXNkuUzyZoZWesXrHyOZIOeBkj1ptgmT6KB08Jodccr-wVIzmNzuprSuMFXnqyC7YwVUcQszfCHplBi2Y15gmfPlURCcr6cDrmuXI2RQ8kVYqbpwpkwU~IwAx-IxlD-SxvRJNg-rOnz1kFBZg6~ril-XCQUH57KgRmoLRdZOtq3kyqIqmR0akjn~EEqO1Zof8u6SDW3r-Givc~Q~FFDVgycai8wbeHX4jSo~RgtJ5fJEZ3leTfm3ID7Dx-QOBdgGBsB4Cf3hQvrSTBbbP5crdkbS1Y16Wy8VT~3dXnXyQP4~X-n136OwpQnKewg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/17/2">
                        <div class="volume trailing-comma">Volume 17</div>
                        <div class="issue">Issue 2</div>

            </a>

        </div>
        <div class="ii-pub-date">
April 2010        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="4838769">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4838769">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4838771">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4838771">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4838780">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4838780">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4838816">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4838816">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4838843">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4838843">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4838845">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4838845">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4838847">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4838847">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4838849">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4838849">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/17/2/61/343980">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/17/2/85/344628">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Aligning a New Reference Genetic Map of <em>Lupinus angustifolius</em> with the Genome Sequence of the Model Legume, <em>Lotus japonicus</em>
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                            <div class="meta-authors--limited">
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Matthew N. Nelson</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">School of Plant Biology and International Centre for Plant Breeding Education and Research, The University of Western Australia</div>, <div class="addr-line">35 Stirling Highway, Crawley, WA 6009</div>, <div class="country">Australia</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +61 8-6488-3671. Fax. <div class="fax">+61 8-6488-1108</div>. Email: <a href="mailto:mnelson@plants.uwa.edu.au" target="_blank">mnelson@plants.uwa.edu.au</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Matthew+N.+Nelson">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nelson M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nelson M&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Matthew N. Nelson</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Paula M. Moolhuijzen</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="insititution">Centre for Comparative Genomics, Murdoch University</div>, <div class="addr-line">Perth, WA 6150</div>, <div class="country">Australia</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Paula+M.+Moolhuijzen">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Moolhuijzen P">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Moolhuijzen P&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Paula M. Moolhuijzen</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jeffrey G. Boersma</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="insititution">School of Earth and Environment, The University of Western Australia</div>, <div class="addr-line">35 Stirling Highway, Crawley, WA 6009</div>, <div class="country">Australia</div></div><div class="aff"><div class="label -label">4</div><div class="insititution">Department of Agriculture and Food Western Australia</div>, <div class="addr-line">South Perth, WA 6151</div>, <div class="country">Australia</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jeffrey+G.+Boersma">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Boersma J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Boersma J&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Jeffrey G. Boersma</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Magdalena Chudy</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">5</div><div class="insititution">Institute of Plant Genetics, Polish Academy of Sciences</div>, <div class="addr-line">60-479 Poznan</div>, <div class="country">Poland</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Magdalena+Chudy">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Chudy M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Chudy M&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Magdalena Chudy</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Karolina Lesniewska</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">5</div><div class="insititution">Institute of Plant Genetics, Polish Academy of Sciences</div>, <div class="addr-line">60-479 Poznan</div>, <div class="country">Poland</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Karolina+Lesniewska">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Lesniewska K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Lesniewska K&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Karolina Lesniewska</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Matthew Bellgard</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="insititution">Centre for Comparative Genomics, Murdoch University</div>, <div class="addr-line">Perth, WA 6150</div>, <div class="country">Australia</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Matthew+Bellgard">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Bellgard M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Bellgard M&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Matthew Bellgard</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Richard P. Oliver</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">6</div><div class="insititution">Australian Centre for Necrotrophic Fungal Pathogens, WA State Agricultural Biotechnology Centre, Murdoch University</div>, <div class="addr-line">Perth, WA 6150</div>, <div class="country">Australia</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Richard+P.+Oliver">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Oliver R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Oliver R&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Richard P. Oliver</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Wojciech Święcicki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">5</div><div class="insititution">Institute of Plant Genetics, Polish Academy of Sciences</div>, <div class="addr-line">60-479 Poznan</div>, <div class="country">Poland</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Wojciech+%c5%9awi%c4%99cicki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Święcicki W">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Święcicki W&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Wojciech Święcicki</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Bogdan Wolko</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">5</div><div class="insititution">Institute of Plant Genetics, Polish Academy of Sciences</div>, <div class="addr-line">60-479 Poznan</div>, <div class="country">Poland</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Bogdan+Wolko">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Wolko B">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Wolko B&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Bogdan Wolko</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Wallace A. Cowling</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">School of Plant Biology and International Centre for Plant Breeding Education and Research, The University of Western Australia</div>, <div class="addr-line">35 Stirling Highway, Crawley, WA 6009</div>, <div class="country">Australia</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Wallace+A.+Cowling">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Cowling W">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Cowling W&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Wallace A. Cowling</a>
                                    </span>
                                <a class="meta-authors--etal is-active" id="show-meta-authors" aria-controls="remaining-authors" href="#">
                                    <span>...</span> Show more</a>
                                <div id="remaining-authors" class="meta-authors--remaining" aria-hidden="true" aria-labeledby="show-meta-authors">
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Simon R. Ellwood</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">6</div><div class="insititution">Australian Centre for Necrotrophic Fungal Pathogens, WA State Agricultural Biotechnology Centre, Murdoch University</div>, <div class="addr-line">Perth, WA 6150</div>, <div class="country">Australia</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Simon+R.+Ellwood">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ellwood S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ellwood S&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Simon R. Ellwood</a>
                                        </span>
                                </div>
                            </div>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">Present address: Department of Plant Agriculture, Crop Science Bldg., University of Guelph, Guelph, ON, Canada N1G 2W1.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN2"><div class="fn"><div class="label -label">‡</div><div class="footnote-content">Present address: Department of Plant Anatomy and Cytology, Faculty of Biology and Environmental Protection, University of Silesia, Jagiellonska 28, 40-032 Katowice, Poland.</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 17, Issue 2, 1 April 2010, Pages 73–83, <a href='https://doi.org/10.1093/dnares/dsq001'>https://doi.org/10.1093/dnares/dsq001</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">04 February 2010</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">24 November 2009</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">28 December 2009</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="344357" href="/dnaresearch/article-pdf/17/2/73/984812/dsq001.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Matthew N. Nelson, Paula M. Moolhuijzen, Jeffrey G. Boersma, Magdalena Chudy, Karolina Lesniewska, Matthew Bellgard, Richard P. Oliver, Wojciech Święcicki, Bogdan Wolko, Wallace A. Cowling, Simon R. Ellwood;  Aligning a New Reference Genetic Map of <em>Lupinus angustifolius</em> with the Genome Sequence of the Model Legume, <em>Lotus japonicus</em>, <em>DNA Research</em>, Volume 17, Issue 2, 1 April 2010, Pages 73–83, <a href='https://doi.org/10.1093/dnares/dsq001'>https://doi.org/10.1093/dnares/dsq001</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/344357?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/344357?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/344357?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/344357?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/344357?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/344357?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/344357?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Aligning%20a%20New%20Reference%20Genetic%20Map%20of%20%26lt%3Bem%26gt%3BLupinus%20angustifolius%26lt%3B%2Fem%26gt%3B%20with%20the%20Genome%20Sequence%20of%20the%20Model%20Legume%2C%20%26lt%3Bem%26gt%3BLotus%20japonicus%26lt%3B%2Fem%26gt%3B&amp;publicationDate=2010-02-04&amp;volumeNum=17&amp;issueNum=2&amp;author=Nelson%2C%20Matthew%20N.%3B%20Moolhuijzen%2C%20Paula%20M.&amp;startPage=73&amp;endPage=83&amp;contentId=10.1093%2Fdnares%2Fdsq001&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f17%2f2%2f73%2f344357" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=4838769 id="4838769" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>We have developed a dense reference genetic map of <em>Lupinus angustifolius</em> (2<em>n</em> = 40) based on a set of 106 publicly available recombinant inbred lines derived from a cross between domesticated and wild parental lines. The map comprised 1090 loci in 20 linkage groups and three small clusters, drawing together data from several previous mapping publications plus almost 200 new markers, of which 63 were gene-based markers. A total of 171 mainly gene-based, sequence-tagged site loci served as bridging points for comparing the <em>Lu. angustifolius</em> genome with the genome sequence of the model legume, <em>Lotus japonicus</em> via BLASTn homology searching. Comparative analysis indicated that the genomes of <em>Lu. angustifolius</em> and <em>Lo. japonicus</em> are highly diverged structurally but with significant regions of conserved synteny including the region of the <em>Lu. angustifolius</em> genome containing the pod-shatter resistance gene, <em>lentus</em>. We discuss the potential of synteny analysis for identifying candidate genes for domestication traits in <em>Lu. angustifolius</em> and in improving our understanding of Fabaceae genome evolution.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="synteny">synteny</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genome evolution&quot;">genome evolution</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;narrow-leafed lupin&quot;">narrow-leafed lupin</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="Fabaceae">Fabaceae</a></div>                            <h2 scrollto-destination=4838771 id="4838771" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Comparative genetics and genomics use knowledge from different species to define important evolutionary relationships, an important aspect of which is the physical and genetic synteny (two or more homologous loci found on a single chromosome) and collinearity (conserved linear order of loci) between any two species that can be used to explain chromosomal rearrangements. Synteny and collinearity have been widely studied in legumes (Fabaceae family), and these studies have shown direct correspondences between evolutionary distance and the degree of synteny and collinearity. These include whole chromosome levels of synteny between members of the closely related Viceae<sup><a reveal-id="DSQ001C1" data-open="DSQ001C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSQ001C2" data-open="DSQ001C2" class="link link-ref link-reveal xref-bibr">2</a></sup> with successively more fractionated and rearranged chromosomal relationships moving out to the Loteae,<sup><a reveal-id="DSQ001C3" data-open="DSQ001C3" class="link link-ref link-reveal xref-bibr">3</a></sup> Cicereae and Phaseoleae.<sup><a reveal-id="DSQ001C4" data-open="DSQ001C4" class="link link-ref link-reveal xref-bibr">4</a></sup></p><p>The <em>Lupinus</em> genus—part of the Genistoid clade of the Papilionoid legumes—includes a number of agricultural crop species, including <em>Lupinus angustifolius</em> L. (narrow-leafed lupin). The Genistoid clade is thought to be one of the first clades to diverge from the rest of the Papilionoid legumes about 56.4 million years ago and is quite distinct from the other clades containing crop or pasture species.<sup><a reveal-id="DSQ001C5" data-open="DSQ001C5" class="link link-ref link-reveal xref-bibr">5</a></sup> The <em>Lupinus</em> genus therefore represents a useful out-group for understanding genome evolution within the legume family.</p><p><em>Lupinus angustifolius</em> is diploid (2<em>n</em> = 40 chromosomes) with a nuclear genome size of 2C = 1.89 pg.<sup><a reveal-id="DSQ001C6" data-open="DSQ001C6" class="link link-ref link-reveal xref-bibr">6</a></sup> Cytogenetic and linkage analyses found evidence of ancient polyploidy in <em>Lu. angustifolius</em>,<sup><a reveal-id="DSQ001C6  DSQ001C8" data-open="DSQ001C6  DSQ001C8" class="link link-ref link-reveal xref-bibr">6–8</a></sup> but it is functionally diploid. Alignment of a low-density genetic map of <em>Lu. angustifolius</em> to an early draft genome sequence of the model legume species <em>Medicago truncatula</em> revealed short regions of conserved synteny between these highly divergent legume species.<sup><a reveal-id="DSQ001C7" data-open="DSQ001C7" class="link link-ref link-reveal xref-bibr">7</a></sup></p><p>Genetic map resources for <em>Lu. angustifolius</em> are modest, comprising two incomplete maps,<sup><a reveal-id="DSQ001C7" data-open="DSQ001C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DSQ001C9" data-open="DSQ001C9" class="link link-ref link-reveal xref-bibr">9</a></sup> several gene-tagging reports<sup><a reveal-id="DSQ001C10  DSQ001C14" data-open="DSQ001C10  DSQ001C14" class="link link-ref link-reveal xref-bibr">10–14</a></sup> and physical descriptions of the chromosomes by cytogenetic approaches.<sup><a reveal-id="DSQ001C15" data-open="DSQ001C15" class="link link-ref link-reveal xref-bibr">15</a>,<a reveal-id="DSQ001C16" data-open="DSQ001C16" class="link link-ref link-reveal xref-bibr">16</a></sup> The purpose of this current study was to bring together marker data from previous mapping studies, add almost 200 new markers and develop a high-density consensus reference genetic map for <em>Lu. angustifolius</em> with all chromosomes richly populated with gene-based markers. We then used this map to compare the genome structure of <em>Lu. angustifolius</em> with the recently released genome sequence of the model legume, <em>Lotus japonicus</em>.<sup><a reveal-id="DSQ001C17" data-open="DSQ001C17" class="link link-ref link-reveal xref-bibr">17</a></sup> By this approach, we hoped to gain further insight into the evolution of Papilionoid legumes by comparing the genomes of a Genistoid species and a Loteae species. We also sought to develop a resource for map alignment-based identification of candidate genes from <em>Lo. japonicus</em> for use in <em>Lu. angustifolius</em> genetic studies.</p>                            <h2 scrollto-destination=4838780 id="4838780" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=4838781 id="4838781" class="section-title" data-legacy-id=s2a>2.1. Characteristics of the Lu. angustifolius mapping population</h3>
<p>The <em>Lu. angustifolius</em> recombinant inbred line (RIL) mapping population used in this study was developed at the Department of Agriculture and Food Western Australia (Perth, Australia) and comprised 106 RILs derived from a cross between a domesticated line (breeding line 83A:476) and a wild landrace from Morocco (P27255).<sup><a reveal-id="DSQ001C7" data-open="DSQ001C7" class="link link-ref link-reveal xref-bibr">7</a></sup> These two crossing parents differed in six major genes for key domestication traits used in all current Australian cultivars: <em>Ku</em> (early flowering); <em>iucundis</em> (low seed alkaloid); <em>tardus</em> and <em>lentus</em> (pod shatter resistance); <em>mollis</em> (water permeable seed) and <em>leucospermus</em> (pigmented flowers, seeds and cotyledons, used as a visible marker to indicate when undesirable cross-pollination to wild types has occurred). Two previous mapping studies used subsets of this population: 89 RILs were used by Boersma <em>et al</em>.<sup><a reveal-id="DSQ001C9" data-open="DSQ001C9" class="link link-ref link-reveal xref-bibr">9</a></sup> and 93 RILs were used by Nelson <em>et al</em>.<sup><a reveal-id="DSQ001C7" data-open="DSQ001C7" class="link link-ref link-reveal xref-bibr">7</a></sup> These studies shared 76 RILs in common, which allowed the straightforward combining of data sets from these previous studies.</p>                            <h3 scrollto-destination=4838786 id="4838786" class="section-title" data-legacy-id=s2b>2.2. Previously published markers</h3>
<p>Genotyping data for 522 microsatellite-anchored fragment length polymorphism (MFLP) loci and five domestication genes (<em>Ku</em>, <em>iucundis</em>, <em>lentus</em>, <em>mollis</em> and <em>leucospermus</em>) were previously reported by Boersma <em>et al</em>.<sup><a reveal-id="DSQ001C9" data-open="DSQ001C9" class="link link-ref link-reveal xref-bibr">9</a></sup> The anthracnose resistance gene and linked marker (<em>Lanr1</em> and AntjM2, respectively) were reported by Yang <em>et al</em>.<sup><a reveal-id="DSQ001C18" data-open="DSQ001C18" class="link link-ref link-reveal xref-bibr">18</a></sup> and You <em>et al</em>.<sup><a reveal-id="DSQ001C14" data-open="DSQ001C14" class="link link-ref link-reveal xref-bibr">14</a></sup> Genotyping at 298 mainly gene-based sequence tagged site (STS) loci, 74 amplified fragment length fragment (AFLP) loci and five domestication genes (<em>Ku</em>, <em>iucundis</em>, <em>lentus</em>, <em>mollis</em> and <em>leucospermus</em>) was reported in Nelson <em>et al</em>.<sup><a reveal-id="DSQ001C7" data-open="DSQ001C7" class="link link-ref link-reveal xref-bibr">7</a></sup> A sixth domestication gene (<em>Tardus</em>), along with three linked markers (TaM1, TaM2 and TaM3), was reported by Boersma <em>et al</em>.<sup><a reveal-id="DSQ001C10" data-open="DSQ001C10" class="link link-ref link-reveal xref-bibr">10</a></sup> Four further markers tagging domestication traits (MoA, LeM1, LeM2 and KuHM1) were reported by Boersma <em>et al</em>.<sup><a reveal-id="DSQ001C11  DSQ001C13" data-open="DSQ001C11  DSQ001C13" class="link link-ref link-reveal xref-bibr">11–13</a></sup> Six isozyme markers were used to screen the population using the methods of Wolko and Weeden.<sup><a reveal-id="DSQ001C8" data-open="DSQ001C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSQ001C19" data-open="DSQ001C19" class="link link-ref link-reveal xref-bibr">19</a></sup></p>                            <h3 scrollto-destination=4838789 id="4838789" class="section-title" data-legacy-id=s2c>2.3. New PCR-based STS markers</h3>
<p>Forty-two STS primer pairs were developed within the framework of the 6th EU FP Grain Legumes Integrated Project (GLIP) and were provided by the Laboratory of Plant Genetics and Breeding at the Agricultural Biotechnology Center, Godollo, Hungary. The primer pairs were designed on the basis of information on <em>M. truncatula</em> and <em>Pisum sativum</em> consensus sequences using an intron-targeted strategy to amplify single or low copy genes. The primer information is available on the following website: <a class="link link-uri" href="http://bioweb.abc.hu/cgi-mt/pisprim/pisprim.pl">http://bioweb.abc.hu/cgi-mt/pisprim/pisprim.pl</a>.</p><p>Nineteen previously unmapped <em>Lupinus</em>-derived STS primer pairs based on polymorphism screening conducted in previous comparative mapping studies were used to genotype the RIL population.<sup><a reveal-id="DSQ001C1" data-open="DSQ001C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSQ001C2" data-open="DSQ001C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DSQ001C7" data-open="DSQ001C7" class="link link-ref link-reveal xref-bibr">7</a></sup></p><p>Two cross-legume primer pairs (PPE and SGR) were provided by Prof. Norman Weeden (Department of Plant Sciences and Plant Pathology at Montana State University, Bozeman, MT, USA). The design strategy was similar to that used for the GLIP marker generation (N. Weeden, pers. comm.).</p><p>One BAC-end sequence tag (BEST) marker was developed from the clone 15L10 from the BAC library of <em>Lu. angustifolius</em> reported by Kasprzak <em>et al</em>.<sup><a reveal-id="DSQ001C20" data-open="DSQ001C20" class="link link-ref link-reveal xref-bibr">20</a></sup> BAC DNA was isolated by QIAprep Spin Miniprep Kit (Qiagen, Doncaster, VIC, Australia) and the insert ends were sequenced by BigDye Terminator v3.1 Cycle Sequencing Kit (Applied Biosystems, Scoresby, VIC, Australia) using an AB PRISM 3130<em>xl</em> Genetic Analyzer. On the basis of the BAC-end sequence, primer pairs were designed with aid of Primer3 software (<a class="link link-uri" href="http://frodo.wi.mit.edu/primer3">http://frodo.wi.mit.edu/primer3</a>).</p><p>An improved dCAPS assay (d212Len) was developed to replace the CAPS marker 212Len reported by Nelson <em>et al</em>.<sup><a reveal-id="DSQ001C7" data-open="DSQ001C7" class="link link-ref link-reveal xref-bibr">7</a></sup></p><p>STS primer pairs from the above sources were screened on <em>Lu. angustifolius</em> parental DNA. Primer pairs showing large length polymorphisms were used directly to genotype the RIL population. Amplicons of primer pairs showing no visible length polymorphisms were purified and directly sequenced. DNA polymorphisms were identified by manual inspection of alignments and chromatograms and suitable SNaPshot (AB), CAPS (cleaved amplified polymorphic sequence) or dCAPS (derived-CAPS) assays were developed to genotype the RIL population.</p>                            <h3 scrollto-destination=4838800 id="4838800" class="section-title" data-legacy-id=s2d>2.4. New MFLP markers</h3>
<p>An additional 134 MFLP markers were developed using the approach described by Boersma <em>et al</em>.<sup><a reveal-id="DSQ001C9" data-open="DSQ001C9" class="link link-ref link-reveal xref-bibr">9</a></sup> Briefly, DNA from each RIL was digested by the restriction enzyme <em>Tru</em>91 (Roche Diagnostics Australia Pty Ltd, Kew, Australia), an isoschizomer of <em>Mse</em>I. An AFLP <em>Mse</em>I-adaptor<sup><a reveal-id="DSQ001C21" data-open="DSQ001C21" class="link link-ref link-reveal xref-bibr">21</a></sup> was ligated to the restriction fragments using T4 DNA-ligase (Roche). The DNA was then digested a second time with <em>Hpa</em>II (Gene Works Pty Ltd, Australia) as described by Yang <em>et al</em>.<sup><a reveal-id="DSQ001C22" data-open="DSQ001C22" class="link link-ref link-reveal xref-bibr">22</a></sup> Pre-selective amplification of fragments using the simple sequence repeat primers listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a> in combination with an <em>Mse</em>I primer with one selective nucleotide C was followed by a second round of amplification using 16 <em>Mse</em>I primers having two additional selective nucleotides (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). The PCR products were resolved on 5% denaturing sequencing gels and polymorphisms visualized by autoradiography.<sup><a reveal-id="DSQ001C22" data-open="DSQ001C22" class="link link-ref link-reveal xref-bibr">22</a></sup></p>                            <h3 scrollto-destination=4838804 id="4838804" class="section-title" data-legacy-id=s2e>2.5. Linkage map construction</h3>
<p>Linkage mapping was conducted with the aid of MultiPoint 1.2 (MultiQTL Ltd, Haifa, Israel), a mapping software package that used the ‘evolutionary optimization strategy’<sup><a reveal-id="DSQ001C23" data-open="DSQ001C23" class="link link-ref link-reveal xref-bibr">23</a></sup> to resolve locus order. Of the 1118 marker and trait loci inputted to MultiPoint 1.2, 16 marker loci showed severe segregation distortion (χ<sup>2</sup><em>P</em> &lt; 0.001). These were excluded from further analyses due to their tendency to introduce false linkages into the analysis. Because a large number of loci were included in this analysis, moderately distorted loci (0.001 &lt; <em>P</em> &lt; 0.01) were retained but kept under observation in the event that they became implicated in false linkages at a later stage.</p><p>Initial clustering into linkage groups was performed at a maximum observable recombination frequency (rf) of 0.10. Multipoint marker ordering was performed separately for each linkage group, and the robustness of marker order was assessed using jack-knife re-sampling. Markers ordered with jack-knife values &gt;90% were considered highly robust and designated as ‘framework’ markers. Markers ordered with jack-knife values &lt;90% (including redundant markers that mapped to the same location) were initially excluded from the map construction due to their destabilizing effect on locus order. The same set of procedures was carried out serially at gradually increasing maximum recombination frequencies (rf = 0.15, 0.20, 0.25, 0.28, 0.30 and 0.31). At each new clustering cycle, manual inspection of proposed new clusters assisted the identification of valid and invalid clusters. A new cluster was considered valid if its two progenitor clusters were most closely related to each other via their terminal loci. A new cluster was considered invalid if its two progenitor clusters were most closely related by non-terminal loci. Such invalid clusters were associated with problematic loci that showed either high degree of missing data points and/or moderately severe segregation distortion (0.001 &lt; <em>P</em> &lt; 0.01) of alleles towards one of the two founding parents of the population. Only clusters judged valid were permitted during linkage mapping.</p><p>Once the framework map construction was completed, the interval sizes were transformed to account for multiple meioses involved in the development of the RIL population and were expressed in Kosambi centiMorgans (cM). Markers that had earlier been removed from the map due to their destabilizing effect on locus order (i.e. those with jack-knife values &lt;90%) were assigned to the most likely intervals on the framework map and were referred to as ‘attached’ markers.</p>                            <h3 scrollto-destination=4838811 id="4838811" class="section-title" data-legacy-id=s2f>2.6. Comparison of Lu. angustifolius and Lo. japonicus genomes</h3>
<p>The <em>Lo. japonicus</em> genome sequence version 4 was used to generate pseudomolecules representing the six chromosomes of <em>Lo. japonicus</em>. The pseudomolecules were assembled based on the Kazusa clone lists (<a class="link link-uri" href="http://www.kazusa.or.jp/lotus/clonelist.html">http://www.kazusa.or.jp/lotus/clonelist.html</a>) using custom Perl scripts (<a class="link link-uri" href="http://www.perl.org/">http://www.perl.org/</a>). Comparison of the genetic map of <em>Lu. angustifolius</em> and the genome sequence of <em>Lo. japonicus</em> was achieved via blastall BLASTn homology search<sup><a reveal-id="DSQ001C24" data-open="DSQ001C24" class="link link-ref link-reveal xref-bibr">24</a>,<a reveal-id="DSQ001C25" data-open="DSQ001C25" class="link link-ref link-reveal xref-bibr">25</a></sup> (at expected alignment values of 1e<sup>−20</sup>) and visualized using CMAP hosted at LegumeDB<sup><a reveal-id="DSQ001C26" data-open="DSQ001C26" class="link link-ref link-reveal xref-bibr">26</a></sup> and GridMap 3.0 as described by Nelson and Lydiate.<sup><a reveal-id="DSQ001C27" data-open="DSQ001C27" class="link link-ref link-reveal xref-bibr">27</a></sup></p>                            <h2 scrollto-destination=4838816 id="4838816" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=4838818 id="4838818" class="section-title" data-legacy-id=s3a>3.1. A new reference map for Lu. angustifolius</h3>
<p>The new genetic map of <em>Lu. angustifolius</em> comprised 1090 loci arranged in 20 linkage groups and 3 small clusters (Fig. <a reveal-id="DSQ001F1" data-open="DSQ001F1" class="link link-reveal link-table xref-fig">1</a>; larger scale diagrams are provided in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). The genotype data and map positions are provided in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>. The total length of the map was 2361.8 cM, with linkage groups ranging from 69.7 to 168.1 cM. Table <a reveal-id="DSQ001TB1" data-open="DSQ001TB1" class="link link-reveal link-table xref-fig">1</a> gives summary details of each linkage group, including their size, the number of framework and total markers, and the equivalent linkage groups from the two previous maps of <em>Lu. angustifolius</em> reported by Boersma <em>et al</em>.<sup><a reveal-id="DSQ001C9" data-open="DSQ001C9" class="link link-ref link-reveal xref-bibr">9</a></sup> and Nelson <em>et al</em>.<sup><a reveal-id="DSQ001C7" data-open="DSQ001C7" class="link link-ref link-reveal xref-bibr">7</a></sup> These two previous studies lacked both the critical marker density and sophisticated mapping methodology of the current mapping study, which led to significant differences between the maps. Boersma <em>et al</em>.<sup><a reveal-id="DSQ001C9" data-open="DSQ001C9" class="link link-ref link-reveal xref-bibr">9</a></sup> reported 21 linkage groups; comparison with the new map revealed that two pairs of the Boersma <em>et al</em>.<sup><a reveal-id="DSQ001C9" data-open="DSQ001C9" class="link link-ref link-reveal xref-bibr">9</a></sup> linkage groups (LG13 and LG21, and LG17 and LG20) coalesced into two linkage groups in this new map (NLL-08 and NLL-10, respectively) (Table <a reveal-id="DSQ001TB1" data-open="DSQ001TB1" class="link link-reveal link-table xref-fig">1</a>). One linkage group (NLL-12) in the new map was entirely absent from the Boersma <em>et al</em>.<sup><a reveal-id="DSQ001C9" data-open="DSQ001C9" class="link link-ref link-reveal xref-bibr">9</a></sup> map. The Nelson <em>et al</em>.<sup><a reveal-id="DSQ001C7" data-open="DSQ001C7" class="link link-ref link-reveal xref-bibr">7</a></sup> map had 20 linkage groups (the same number as the current map); comparison with the new map revealed that five linkage groups of Nelson <em>et al</em>.<sup><a reveal-id="DSQ001C7" data-open="DSQ001C7" class="link link-ref link-reveal xref-bibr">7</a></sup> (LG07, LG19, LG05, LG14 and LG20) were illegitimately joined and were divided among 10 of the new linkage groups (Table <a reveal-id="DSQ001TB1" data-open="DSQ001TB1" class="link link-reveal link-table xref-fig">1</a>).</p>                        <a id="4838823" scrollto-destination="4838823"></a>
<div id="DSQ001F1" data-id="DSQ001F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/2/10.1093/dnares/dsq001/2/m_dsq00101a.jpeg?Expires=2147483647&amp;Signature=dswuPdOoHD2237MYvAid3oRskWD0REpjjRRzSiiIWdrkbqWZqvTsB9LFtdLSSBsnI4CMw7q7jjPkdxITfxoDnBptkFRzfBsFt8RqH~c68EGUfxGqdN8Php-eM0rNl7LE-S3zaDd3g231GNUj1ejN2wYle92i17ud-WwMcjH-Ib0DeMFbBRaoasdkJRo03abbbpuCBOxrZ9Kkt-D1spubmPFWbVBpcJgvtBZkCbgvYnauKFhRkn1TSIT2A2j73FoLseX3Fdlq0IODsH9HdW8hKvBA5hp~ZYbpJDZWrsHbi41i~emqoq3q-QuTqHJEnvgRhSuKHqcdYSk~zGq92~Bt9w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Linkage map of the Lu. angustifolius genome comprising 1090 loci distributed among 20 linkage groups (NLL-01 to NLL-20) and three small clusters (Cluster-1 to Cluster-3). Loci on the left of each linkage group are ‘framework’ markers used to construct the linkage group. Loci on the right of each linkage group are ‘attached’ markers whose approximate location between flanking markers is shown. Locus positions are in Kosambi centiMorgans. Major gene trait loci are shown in bold text." path-from-xml="dsq00101a.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4838823/dsq00101a.jpeg" path-from-xml="dsq00101a.jpeg" target="_blank">View large</a><a section="4838823" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/2/10.1093/dnares/dsq001/2/dsq00101a.jpeg?Expires=2147483647&amp;Signature=RfdZ~pEpFFA52Ih9e6QrRzLzFMlKdmAshtAL2VhUaFgmNX5NOumayO0CWMwR1-Ri2z7bEjL2feUc1iUNNcQF8Sr7Wl63u6Iinvb0XyW17GTsc8RthyT-I~e5fHdf7cctPKPoMf4YyuSIXEYxDAm2Cl1j3YfB1OENwJNh3wzhcMDv9CFUatt-Z~Qo3MUb~GajHFoQt2tGi9MY066rMlLkUpcoN1pVYB429AlkvIY4v8qpsaDye5ujLweZLERr57nt5y4MQsgVerkPkmJiFl7AaXkjUvGvI2nwhH0-BsJ9OL63t59TJzyFlaKYzU8sOmO9RI~DvTIDL38yDoIXljFdtw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=4838823&amp;ar=344357&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsq00101a.jpeg" target="_blank">Download slide</a></div></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/2/10.1093/dnares/dsq001/2/m_dsq00101b.jpeg?Expires=2147483647&amp;Signature=M4GMwjqqFLQM1sfZ-OOqp8HgwMP0p2x4b7wh0qzVlh~mLncpoHhpOlwXsLmosAoVPTig4WiLwOB0sCLeHoq9KRhHnDQVCupUBFs7QkQsblQ83Tdn8ACF9bNXU7ndIN33lFRl9IE9vyc88mEoNCjwN~wScX~5a9er6UnUZcPR2iqq8jH0duWCNzdXYwSHiKjI2os0UUaAC~XIMt2nEtr9jI-eKu-LptdQS24bp4gEfXBKy3Q6A4muqm1REpDLPF6ZHmYmDTcbb4aw-ENQzQsQRD3YhRiMvigfGXprI~ny5hyzKb43r-ihSPrquG2KtCWlTtAM~PC79-dP24e7xOy2PA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Linkage map of the Lu. angustifolius genome comprising 1090 loci distributed among 20 linkage groups (NLL-01 to NLL-20) and three small clusters (Cluster-1 to Cluster-3). Loci on the left of each linkage group are ‘framework’ markers used to construct the linkage group. Loci on the right of each linkage group are ‘attached’ markers whose approximate location between flanking markers is shown. Locus positions are in Kosambi centiMorgans. Major gene trait loci are shown in bold text." path-from-xml="dsq00101b.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4838823/dsq00101b.jpeg" path-from-xml="dsq00101b.jpeg" target="_blank">View large</a><a section="4838823" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/2/10.1093/dnares/dsq001/2/dsq00101b.jpeg?Expires=2147483647&amp;Signature=ZTKhegMyso-4lTSuI5wVow1p2o7gx28eVwLE8rDyus1cH2bPNzknsOFC5pskqLpInVWSormC-6TZn66dzHE7OHXhf1jb9ovsY2UFWVKYpLf0Usr~o~ZLoW16zlV9lG2p~-wPzCroR5XZ2S0xvwEgzoEvkbBQBh1u5dkCpygJbvyRo-rDtTi5Mbuf0KpNRHs0yTGRBRqs~D9TXPaK86iBctlz4sfUUd4m9wXe1OAV5LppaQLC3WFgSsR1E6ivzOIVV4wyF0Q65UJ~cwkibenaSwgbC2IcdoN5mH3wkT~O51BYxB-nMnw7f8AMGFlrUibAgU0eA9mZQ~56rxdFcxOzOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=4838823&amp;ar=344357&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsq00101b.jpeg" target="_blank">Download slide</a></div></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/2/10.1093/dnares/dsq001/2/m_dsq00101c.jpeg?Expires=2147483647&amp;Signature=fLtyNFaB~ESgU~zu9A7Da6onXBDCMw4jZSP~mfBxmswwmPUBSiT3TO~B0-fEAbWpJlygEgJedDjflKwJve9NYUmVqRkLCEebak-pFJXutuEQnhO1OsqvF09l60tjZnTGjf0YGJaBhNgMtjgeEjlRXYI1oml5dpgd2kkJtL~3ABaFcFhr3uXq4Mg6LhYr~veM9SmlP1vyVNvpv2hg6h2mJbwxeKS4zYjJhLDtgOv5dyMT0KjNO5NEDOIUaTiySXLM3a1tJ~ZuXJnPYLop6kjpN5RvhJsrDce4YeYPPhPTjMzNCEj8fAXna9fsrfSPHmZY5mFOxEEqrOIIrMAh2djl9Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Linkage map of the Lu. angustifolius genome comprising 1090 loci distributed among 20 linkage groups (NLL-01 to NLL-20) and three small clusters (Cluster-1 to Cluster-3). Loci on the left of each linkage group are ‘framework’ markers used to construct the linkage group. Loci on the right of each linkage group are ‘attached’ markers whose approximate location between flanking markers is shown. Locus positions are in Kosambi centiMorgans. Major gene trait loci are shown in bold text." path-from-xml="dsq00101c.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4838823/dsq00101c.jpeg" path-from-xml="dsq00101c.jpeg" target="_blank">View large</a><a section="4838823" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/2/10.1093/dnares/dsq001/2/dsq00101c.jpeg?Expires=2147483647&amp;Signature=JgsQVdE8a-kWhWYBb8ocLWV58cSDFmCS42NJ9fLAT0zlr9pR1MhVukoj6cyb4bg~Y22-AkKHlTF5ZLmmMaWr-eSwWoonMcQHk~kFGyQoO9ZFkz7ppaUubU5atZL9PiS9EUF3cYRNDX9ZOUZuWt1IOUR5mTAjWvjJerNmB7AcsU8KJeT-1qqTTy1A8mQY0ACBdoHPZq99h7wGFyFP~Jz61oKssSWYNaJmOmI8uBJ01mCZuJt6AlNo4iGL04G9fnoF3LwMpxErWCbyDKssWGtfdhIWIdGhVFJcelIps0e03QRO1b8ozDiSoqgT5g7nHYaVqMhsW3u~4rOv-DxaBnA9zA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=4838823&amp;ar=344357&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsq00101c.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Linkage map of the <em>Lu. angustifolius</em> genome comprising 1090 loci distributed among 20 linkage groups (NLL-01 to NLL-20) and three small clusters (Cluster-1 to Cluster-3). Loci on the left of each linkage group are ‘framework’ markers used to construct the linkage group. Loci on the right of each linkage group are ‘attached’ markers whose approximate location between flanking markers is shown. Locus positions are in Kosambi centiMorgans. Major gene trait loci are shown in bold text.</p></div></div><div content-id="DSQ001F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/2/10.1093/dnares/dsq001/2/m_dsq00101a.jpeg?Expires=2147483647&amp;Signature=dswuPdOoHD2237MYvAid3oRskWD0REpjjRRzSiiIWdrkbqWZqvTsB9LFtdLSSBsnI4CMw7q7jjPkdxITfxoDnBptkFRzfBsFt8RqH~c68EGUfxGqdN8Php-eM0rNl7LE-S3zaDd3g231GNUj1ejN2wYle92i17ud-WwMcjH-Ib0DeMFbBRaoasdkJRo03abbbpuCBOxrZ9Kkt-D1spubmPFWbVBpcJgvtBZkCbgvYnauKFhRkn1TSIT2A2j73FoLseX3Fdlq0IODsH9HdW8hKvBA5hp~ZYbpJDZWrsHbi41i~emqoq3q-QuTqHJEnvgRhSuKHqcdYSk~zGq92~Bt9w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Linkage map of the Lu. angustifolius genome comprising 1090 loci distributed among 20 linkage groups (NLL-01 to NLL-20) and three small clusters (Cluster-1 to Cluster-3). Loci on the left of each linkage group are ‘framework’ markers used to construct the linkage group. Loci on the right of each linkage group are ‘attached’ markers whose approximate location between flanking markers is shown. Locus positions are in Kosambi centiMorgans. Major gene trait loci are shown in bold text." path-from-xml="dsq00101a.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4838823/dsq00101a.jpeg" path-from-xml="dsq00101a.jpeg" target="_blank">View large</a><a section="4838823" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/2/10.1093/dnares/dsq001/2/dsq00101a.jpeg?Expires=2147483647&amp;Signature=RfdZ~pEpFFA52Ih9e6QrRzLzFMlKdmAshtAL2VhUaFgmNX5NOumayO0CWMwR1-Ri2z7bEjL2feUc1iUNNcQF8Sr7Wl63u6Iinvb0XyW17GTsc8RthyT-I~e5fHdf7cctPKPoMf4YyuSIXEYxDAm2Cl1j3YfB1OENwJNh3wzhcMDv9CFUatt-Z~Qo3MUb~GajHFoQt2tGi9MY066rMlLkUpcoN1pVYB429AlkvIY4v8qpsaDye5ujLweZLERr57nt5y4MQsgVerkPkmJiFl7AaXkjUvGvI2nwhH0-BsJ9OL63t59TJzyFlaKYzU8sOmO9RI~DvTIDL38yDoIXljFdtw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=4838823&amp;ar=344357&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsq00101a.jpeg" target="_blank">Download slide</a></div></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/2/10.1093/dnares/dsq001/2/m_dsq00101b.jpeg?Expires=2147483647&amp;Signature=M4GMwjqqFLQM1sfZ-OOqp8HgwMP0p2x4b7wh0qzVlh~mLncpoHhpOlwXsLmosAoVPTig4WiLwOB0sCLeHoq9KRhHnDQVCupUBFs7QkQsblQ83Tdn8ACF9bNXU7ndIN33lFRl9IE9vyc88mEoNCjwN~wScX~5a9er6UnUZcPR2iqq8jH0duWCNzdXYwSHiKjI2os0UUaAC~XIMt2nEtr9jI-eKu-LptdQS24bp4gEfXBKy3Q6A4muqm1REpDLPF6ZHmYmDTcbb4aw-ENQzQsQRD3YhRiMvigfGXprI~ny5hyzKb43r-ihSPrquG2KtCWlTtAM~PC79-dP24e7xOy2PA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Linkage map of the Lu. angustifolius genome comprising 1090 loci distributed among 20 linkage groups (NLL-01 to NLL-20) and three small clusters (Cluster-1 to Cluster-3). Loci on the left of each linkage group are ‘framework’ markers used to construct the linkage group. Loci on the right of each linkage group are ‘attached’ markers whose approximate location between flanking markers is shown. Locus positions are in Kosambi centiMorgans. Major gene trait loci are shown in bold text." path-from-xml="dsq00101b.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4838823/dsq00101b.jpeg" path-from-xml="dsq00101b.jpeg" target="_blank">View large</a><a section="4838823" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/2/10.1093/dnares/dsq001/2/dsq00101b.jpeg?Expires=2147483647&amp;Signature=ZTKhegMyso-4lTSuI5wVow1p2o7gx28eVwLE8rDyus1cH2bPNzknsOFC5pskqLpInVWSormC-6TZn66dzHE7OHXhf1jb9ovsY2UFWVKYpLf0Usr~o~ZLoW16zlV9lG2p~-wPzCroR5XZ2S0xvwEgzoEvkbBQBh1u5dkCpygJbvyRo-rDtTi5Mbuf0KpNRHs0yTGRBRqs~D9TXPaK86iBctlz4sfUUd4m9wXe1OAV5LppaQLC3WFgSsR1E6ivzOIVV4wyF0Q65UJ~cwkibenaSwgbC2IcdoN5mH3wkT~O51BYxB-nMnw7f8AMGFlrUibAgU0eA9mZQ~56rxdFcxOzOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=4838823&amp;ar=344357&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsq00101b.jpeg" target="_blank">Download slide</a></div></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/2/10.1093/dnares/dsq001/2/m_dsq00101c.jpeg?Expires=2147483647&amp;Signature=fLtyNFaB~ESgU~zu9A7Da6onXBDCMw4jZSP~mfBxmswwmPUBSiT3TO~B0-fEAbWpJlygEgJedDjflKwJve9NYUmVqRkLCEebak-pFJXutuEQnhO1OsqvF09l60tjZnTGjf0YGJaBhNgMtjgeEjlRXYI1oml5dpgd2kkJtL~3ABaFcFhr3uXq4Mg6LhYr~veM9SmlP1vyVNvpv2hg6h2mJbwxeKS4zYjJhLDtgOv5dyMT0KjNO5NEDOIUaTiySXLM3a1tJ~ZuXJnPYLop6kjpN5RvhJsrDce4YeYPPhPTjMzNCEj8fAXna9fsrfSPHmZY5mFOxEEqrOIIrMAh2djl9Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Linkage map of the Lu. angustifolius genome comprising 1090 loci distributed among 20 linkage groups (NLL-01 to NLL-20) and three small clusters (Cluster-1 to Cluster-3). Loci on the left of each linkage group are ‘framework’ markers used to construct the linkage group. Loci on the right of each linkage group are ‘attached’ markers whose approximate location between flanking markers is shown. Locus positions are in Kosambi centiMorgans. Major gene trait loci are shown in bold text." path-from-xml="dsq00101c.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4838823/dsq00101c.jpeg" path-from-xml="dsq00101c.jpeg" target="_blank">View large</a><a section="4838823" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/2/10.1093/dnares/dsq001/2/dsq00101c.jpeg?Expires=2147483647&amp;Signature=JgsQVdE8a-kWhWYBb8ocLWV58cSDFmCS42NJ9fLAT0zlr9pR1MhVukoj6cyb4bg~Y22-AkKHlTF5ZLmmMaWr-eSwWoonMcQHk~kFGyQoO9ZFkz7ppaUubU5atZL9PiS9EUF3cYRNDX9ZOUZuWt1IOUR5mTAjWvjJerNmB7AcsU8KJeT-1qqTTy1A8mQY0ACBdoHPZq99h7wGFyFP~Jz61oKssSWYNaJmOmI8uBJ01mCZuJt6AlNo4iGL04G9fnoF3LwMpxErWCbyDKssWGtfdhIWIdGhVFJcelIps0e03QRO1b8ozDiSoqgT5g7nHYaVqMhsW3u~4rOv-DxaBnA9zA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=4838823&amp;ar=344357&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsq00101c.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Linkage map of the <em>Lu. angustifolius</em> genome comprising 1090 loci distributed among 20 linkage groups (NLL-01 to NLL-20) and three small clusters (Cluster-1 to Cluster-3). Loci on the left of each linkage group are ‘framework’ markers used to construct the linkage group. Loci on the right of each linkage group are ‘attached’ markers whose approximate location between flanking markers is shown. Locus positions are in Kosambi centiMorgans. Major gene trait loci are shown in bold text.</p></div></div>                        <a id="4838830" scrollto-destination="4838830"></a>
<div content-id="DSQ001TB1" class="table-modal"><div class="table-wrap"><div id="DSQ001TB1" class="table-wrap-title" data-id="DSQ001TB1"><div class="label -label">Table 1</div><div class="caption"><p>Summary of linkage groups and small clusters for <em>Lu. angustifolius</em> (narrow-leafed lupin) in this study (NLL-01 to NLL-20 and Cluster-1 to Cluster-3, respectively) and the equivalent linkage groups (LG) in two previous maps</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Linkage group </th><th>Length (cM)<sup><a reveal-id="DSQ001C1" data-open="DSQ001C1" class="link link-ref link-reveal xref-bibr">1</a></sup> </th><th>Framework loci </th><th>Total loci<sup>a</sup> </th><th>Linkage group in Boersma <em>et al</em>.<sup><a reveal-id="DSQ001C9" data-open="DSQ001C9" class="link link-ref link-reveal xref-bibr">9</a></sup> </th><th>Linkage group in Nelson <em>et al</em>.<sup><a reveal-id="DSQ001C7" data-open="DSQ001C7" class="link link-ref link-reveal xref-bibr">7</a></sup> </th></tr></thead><tbody><tr><td>NLL-01 </td><td>168.1 </td><td>52 </td><td>94 </td><td>LG1 </td><td>LG07 (top) and LG18 </td></tr><tr><td>NLL-02 </td><td>155.6 </td><td>32 </td><td>60 </td><td>LG6 </td><td>LG19 (top) and LG15 </td></tr><tr><td>NLL-03 </td><td>153.9 </td><td>43 </td><td>65 </td><td>LG11 </td><td>LG02 </td></tr><tr><td>NLL-04 </td><td>143.9 </td><td>34 </td><td>56 </td><td>LG5 </td><td>LG10 </td></tr><tr><td>NLL-05 </td><td>137.1 </td><td>34 </td><td>62 </td><td>LG4 </td><td>LG13 </td></tr><tr><td>NLL-06 </td><td>133.4 </td><td>42 </td><td>82 </td><td>LG3 </td><td>LG16 and LG14 (top) </td></tr><tr><td>NLL-07 </td><td>121.8 </td><td>40 </td><td>66 </td><td>LG9 </td><td>LG04 </td></tr><tr><td>NLL-08 </td><td>121.7 </td><td>34 </td><td>55 </td><td>LG13 and LG21 </td><td>LG05 (top) </td></tr><tr><td>NLL-09 </td><td>119.4 </td><td>32 </td><td>45 </td><td>LG18 </td><td>LG11 </td></tr><tr><td>NLL-10 </td><td>117.8 </td><td>27 </td><td>54 </td><td>LG17 and LG20 </td><td>LG01, LG19 (bottom) and Triplet-1 </td></tr><tr><td>NLL-11 </td><td>112.9 </td><td>36 </td><td>78 </td><td>LG2 </td><td>LG06 and LG20(bottom) </td></tr><tr><td>NLL-12 </td><td>110.2 </td><td>20 </td><td>31 </td><td>Unlinked clusters </td><td>LG09 </td></tr><tr><td>NLL-13 </td><td>102.9 </td><td>25 </td><td>44 </td><td>LG16 </td><td>LG07(bottom) and Pair-3 </td></tr><tr><td>NLL-14 </td><td>102.9 </td><td>25 </td><td>36 </td><td>LG15 </td><td>LG14 (bottom) </td></tr><tr><td>NLL-15 </td><td>101.6 </td><td>30 </td><td>42 </td><td>LG12 </td><td>LG17 and Triplet-2 </td></tr><tr><td>NLL-16 </td><td>100.1 </td><td>17 </td><td>34 </td><td>LG19 </td><td>LG05 (bottom) and Pair-2 </td></tr><tr><td>NLL-17 </td><td>99.7 </td><td>27 </td><td>49 </td><td>LG8 </td><td>LG03 </td></tr><tr><td>NLL-18 </td><td>90.8 </td><td>29 </td><td>45 </td><td>LG14 </td><td>LG12 </td></tr><tr><td>NLL-19 </td><td>85.6 </td><td>22 </td><td>40 </td><td>LG7 </td><td>LG08 </td></tr><tr><td>NLL-20 </td><td>69.7 </td><td>27 </td><td>41 </td><td>LG10 </td><td>LG20 (top) </td></tr><tr><td>Cluster-1 </td><td>9.5 </td><td>4 </td><td>5 </td><td>Unlinked cluster </td><td>Pair-1 </td></tr><tr><td>Cluster-2 </td><td>1.8 </td><td>3 </td><td>3 </td><td>Unlinked cluster </td><td>— </td></tr><tr><td>Cluster-3 </td><td>1.4 </td><td>2 </td><td>2 </td><td>Unlinked </td><td>Unlinked </td></tr><tr><td>Unlinked </td><td>— </td><td>— </td><td>12 </td><td> </td><td> </td></tr><tr><td>Total </td><td>2361.8 </td><td>637 </td><td>1101 </td><td> </td><td> </td></tr></tbody></table></div><div content-id="DSQ001TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Linkage group </th><th>Length (cM)<sup><a reveal-id="DSQ001C1" data-open="DSQ001C1" class="link link-ref link-reveal xref-bibr">1</a></sup> </th><th>Framework loci </th><th>Total loci<sup>a</sup> </th><th>Linkage group in Boersma <em>et al</em>.<sup><a reveal-id="DSQ001C9" data-open="DSQ001C9" class="link link-ref link-reveal xref-bibr">9</a></sup> </th><th>Linkage group in Nelson <em>et al</em>.<sup><a reveal-id="DSQ001C7" data-open="DSQ001C7" class="link link-ref link-reveal xref-bibr">7</a></sup> </th></tr></thead><tbody><tr><td>NLL-01 </td><td>168.1 </td><td>52 </td><td>94 </td><td>LG1 </td><td>LG07 (top) and LG18 </td></tr><tr><td>NLL-02 </td><td>155.6 </td><td>32 </td><td>60 </td><td>LG6 </td><td>LG19 (top) and LG15 </td></tr><tr><td>NLL-03 </td><td>153.9 </td><td>43 </td><td>65 </td><td>LG11 </td><td>LG02 </td></tr><tr><td>NLL-04 </td><td>143.9 </td><td>34 </td><td>56 </td><td>LG5 </td><td>LG10 </td></tr><tr><td>NLL-05 </td><td>137.1 </td><td>34 </td><td>62 </td><td>LG4 </td><td>LG13 </td></tr><tr><td>NLL-06 </td><td>133.4 </td><td>42 </td><td>82 </td><td>LG3 </td><td>LG16 and LG14 (top) </td></tr><tr><td>NLL-07 </td><td>121.8 </td><td>40 </td><td>66 </td><td>LG9 </td><td>LG04 </td></tr><tr><td>NLL-08 </td><td>121.7 </td><td>34 </td><td>55 </td><td>LG13 and LG21 </td><td>LG05 (top) </td></tr><tr><td>NLL-09 </td><td>119.4 </td><td>32 </td><td>45 </td><td>LG18 </td><td>LG11 </td></tr><tr><td>NLL-10 </td><td>117.8 </td><td>27 </td><td>54 </td><td>LG17 and LG20 </td><td>LG01, LG19 (bottom) and Triplet-1 </td></tr><tr><td>NLL-11 </td><td>112.9 </td><td>36 </td><td>78 </td><td>LG2 </td><td>LG06 and LG20(bottom) </td></tr><tr><td>NLL-12 </td><td>110.2 </td><td>20 </td><td>31 </td><td>Unlinked clusters </td><td>LG09 </td></tr><tr><td>NLL-13 </td><td>102.9 </td><td>25 </td><td>44 </td><td>LG16 </td><td>LG07(bottom) and Pair-3 </td></tr><tr><td>NLL-14 </td><td>102.9 </td><td>25 </td><td>36 </td><td>LG15 </td><td>LG14 (bottom) </td></tr><tr><td>NLL-15 </td><td>101.6 </td><td>30 </td><td>42 </td><td>LG12 </td><td>LG17 and Triplet-2 </td></tr><tr><td>NLL-16 </td><td>100.1 </td><td>17 </td><td>34 </td><td>LG19 </td><td>LG05 (bottom) and Pair-2 </td></tr><tr><td>NLL-17 </td><td>99.7 </td><td>27 </td><td>49 </td><td>LG8 </td><td>LG03 </td></tr><tr><td>NLL-18 </td><td>90.8 </td><td>29 </td><td>45 </td><td>LG14 </td><td>LG12 </td></tr><tr><td>NLL-19 </td><td>85.6 </td><td>22 </td><td>40 </td><td>LG7 </td><td>LG08 </td></tr><tr><td>NLL-20 </td><td>69.7 </td><td>27 </td><td>41 </td><td>LG10 </td><td>LG20 (top) </td></tr><tr><td>Cluster-1 </td><td>9.5 </td><td>4 </td><td>5 </td><td>Unlinked cluster </td><td>Pair-1 </td></tr><tr><td>Cluster-2 </td><td>1.8 </td><td>3 </td><td>3 </td><td>Unlinked cluster </td><td>— </td></tr><tr><td>Cluster-3 </td><td>1.4 </td><td>2 </td><td>2 </td><td>Unlinked </td><td>Unlinked </td></tr><tr><td>Unlinked </td><td>— </td><td>— </td><td>12 </td><td> </td><td> </td></tr><tr><td>Total </td><td>2361.8 </td><td>637 </td><td>1101 </td><td> </td><td> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Framework markers were used to form the linkage groups. Total loci includes both framework and attached loci (see text for details).</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4838830" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSQ001TB1" class="table-wrap-title" data-id="DSQ001TB1"><div class="label -label">Table 1</div><div class="caption"><p>Summary of linkage groups and small clusters for <em>Lu. angustifolius</em> (narrow-leafed lupin) in this study (NLL-01 to NLL-20 and Cluster-1 to Cluster-3, respectively) and the equivalent linkage groups (LG) in two previous maps</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Linkage group </th><th>Length (cM)<sup><a reveal-id="DSQ001C1" data-open="DSQ001C1" class="link link-ref link-reveal xref-bibr">1</a></sup> </th><th>Framework loci </th><th>Total loci<sup>a</sup> </th><th>Linkage group in Boersma <em>et al</em>.<sup><a reveal-id="DSQ001C9" data-open="DSQ001C9" class="link link-ref link-reveal xref-bibr">9</a></sup> </th><th>Linkage group in Nelson <em>et al</em>.<sup><a reveal-id="DSQ001C7" data-open="DSQ001C7" class="link link-ref link-reveal xref-bibr">7</a></sup> </th></tr></thead><tbody><tr><td>NLL-01 </td><td>168.1 </td><td>52 </td><td>94 </td><td>LG1 </td><td>LG07 (top) and LG18 </td></tr><tr><td>NLL-02 </td><td>155.6 </td><td>32 </td><td>60 </td><td>LG6 </td><td>LG19 (top) and LG15 </td></tr><tr><td>NLL-03 </td><td>153.9 </td><td>43 </td><td>65 </td><td>LG11 </td><td>LG02 </td></tr><tr><td>NLL-04 </td><td>143.9 </td><td>34 </td><td>56 </td><td>LG5 </td><td>LG10 </td></tr><tr><td>NLL-05 </td><td>137.1 </td><td>34 </td><td>62 </td><td>LG4 </td><td>LG13 </td></tr><tr><td>NLL-06 </td><td>133.4 </td><td>42 </td><td>82 </td><td>LG3 </td><td>LG16 and LG14 (top) </td></tr><tr><td>NLL-07 </td><td>121.8 </td><td>40 </td><td>66 </td><td>LG9 </td><td>LG04 </td></tr><tr><td>NLL-08 </td><td>121.7 </td><td>34 </td><td>55 </td><td>LG13 and LG21 </td><td>LG05 (top) </td></tr><tr><td>NLL-09 </td><td>119.4 </td><td>32 </td><td>45 </td><td>LG18 </td><td>LG11 </td></tr><tr><td>NLL-10 </td><td>117.8 </td><td>27 </td><td>54 </td><td>LG17 and LG20 </td><td>LG01, LG19 (bottom) and Triplet-1 </td></tr><tr><td>NLL-11 </td><td>112.9 </td><td>36 </td><td>78 </td><td>LG2 </td><td>LG06 and LG20(bottom) </td></tr><tr><td>NLL-12 </td><td>110.2 </td><td>20 </td><td>31 </td><td>Unlinked clusters </td><td>LG09 </td></tr><tr><td>NLL-13 </td><td>102.9 </td><td>25 </td><td>44 </td><td>LG16 </td><td>LG07(bottom) and Pair-3 </td></tr><tr><td>NLL-14 </td><td>102.9 </td><td>25 </td><td>36 </td><td>LG15 </td><td>LG14 (bottom) </td></tr><tr><td>NLL-15 </td><td>101.6 </td><td>30 </td><td>42 </td><td>LG12 </td><td>LG17 and Triplet-2 </td></tr><tr><td>NLL-16 </td><td>100.1 </td><td>17 </td><td>34 </td><td>LG19 </td><td>LG05 (bottom) and Pair-2 </td></tr><tr><td>NLL-17 </td><td>99.7 </td><td>27 </td><td>49 </td><td>LG8 </td><td>LG03 </td></tr><tr><td>NLL-18 </td><td>90.8 </td><td>29 </td><td>45 </td><td>LG14 </td><td>LG12 </td></tr><tr><td>NLL-19 </td><td>85.6 </td><td>22 </td><td>40 </td><td>LG7 </td><td>LG08 </td></tr><tr><td>NLL-20 </td><td>69.7 </td><td>27 </td><td>41 </td><td>LG10 </td><td>LG20 (top) </td></tr><tr><td>Cluster-1 </td><td>9.5 </td><td>4 </td><td>5 </td><td>Unlinked cluster </td><td>Pair-1 </td></tr><tr><td>Cluster-2 </td><td>1.8 </td><td>3 </td><td>3 </td><td>Unlinked cluster </td><td>— </td></tr><tr><td>Cluster-3 </td><td>1.4 </td><td>2 </td><td>2 </td><td>Unlinked </td><td>Unlinked </td></tr><tr><td>Unlinked </td><td>— </td><td>— </td><td>12 </td><td> </td><td> </td></tr><tr><td>Total </td><td>2361.8 </td><td>637 </td><td>1101 </td><td> </td><td> </td></tr></tbody></table></div><div content-id="DSQ001TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Linkage group </th><th>Length (cM)<sup><a reveal-id="DSQ001C1" data-open="DSQ001C1" class="link link-ref link-reveal xref-bibr">1</a></sup> </th><th>Framework loci </th><th>Total loci<sup>a</sup> </th><th>Linkage group in Boersma <em>et al</em>.<sup><a reveal-id="DSQ001C9" data-open="DSQ001C9" class="link link-ref link-reveal xref-bibr">9</a></sup> </th><th>Linkage group in Nelson <em>et al</em>.<sup><a reveal-id="DSQ001C7" data-open="DSQ001C7" class="link link-ref link-reveal xref-bibr">7</a></sup> </th></tr></thead><tbody><tr><td>NLL-01 </td><td>168.1 </td><td>52 </td><td>94 </td><td>LG1 </td><td>LG07 (top) and LG18 </td></tr><tr><td>NLL-02 </td><td>155.6 </td><td>32 </td><td>60 </td><td>LG6 </td><td>LG19 (top) and LG15 </td></tr><tr><td>NLL-03 </td><td>153.9 </td><td>43 </td><td>65 </td><td>LG11 </td><td>LG02 </td></tr><tr><td>NLL-04 </td><td>143.9 </td><td>34 </td><td>56 </td><td>LG5 </td><td>LG10 </td></tr><tr><td>NLL-05 </td><td>137.1 </td><td>34 </td><td>62 </td><td>LG4 </td><td>LG13 </td></tr><tr><td>NLL-06 </td><td>133.4 </td><td>42 </td><td>82 </td><td>LG3 </td><td>LG16 and LG14 (top) </td></tr><tr><td>NLL-07 </td><td>121.8 </td><td>40 </td><td>66 </td><td>LG9 </td><td>LG04 </td></tr><tr><td>NLL-08 </td><td>121.7 </td><td>34 </td><td>55 </td><td>LG13 and LG21 </td><td>LG05 (top) </td></tr><tr><td>NLL-09 </td><td>119.4 </td><td>32 </td><td>45 </td><td>LG18 </td><td>LG11 </td></tr><tr><td>NLL-10 </td><td>117.8 </td><td>27 </td><td>54 </td><td>LG17 and LG20 </td><td>LG01, LG19 (bottom) and Triplet-1 </td></tr><tr><td>NLL-11 </td><td>112.9 </td><td>36 </td><td>78 </td><td>LG2 </td><td>LG06 and LG20(bottom) </td></tr><tr><td>NLL-12 </td><td>110.2 </td><td>20 </td><td>31 </td><td>Unlinked clusters </td><td>LG09 </td></tr><tr><td>NLL-13 </td><td>102.9 </td><td>25 </td><td>44 </td><td>LG16 </td><td>LG07(bottom) and Pair-3 </td></tr><tr><td>NLL-14 </td><td>102.9 </td><td>25 </td><td>36 </td><td>LG15 </td><td>LG14 (bottom) </td></tr><tr><td>NLL-15 </td><td>101.6 </td><td>30 </td><td>42 </td><td>LG12 </td><td>LG17 and Triplet-2 </td></tr><tr><td>NLL-16 </td><td>100.1 </td><td>17 </td><td>34 </td><td>LG19 </td><td>LG05 (bottom) and Pair-2 </td></tr><tr><td>NLL-17 </td><td>99.7 </td><td>27 </td><td>49 </td><td>LG8 </td><td>LG03 </td></tr><tr><td>NLL-18 </td><td>90.8 </td><td>29 </td><td>45 </td><td>LG14 </td><td>LG12 </td></tr><tr><td>NLL-19 </td><td>85.6 </td><td>22 </td><td>40 </td><td>LG7 </td><td>LG08 </td></tr><tr><td>NLL-20 </td><td>69.7 </td><td>27 </td><td>41 </td><td>LG10 </td><td>LG20 (top) </td></tr><tr><td>Cluster-1 </td><td>9.5 </td><td>4 </td><td>5 </td><td>Unlinked cluster </td><td>Pair-1 </td></tr><tr><td>Cluster-2 </td><td>1.8 </td><td>3 </td><td>3 </td><td>Unlinked cluster </td><td>— </td></tr><tr><td>Cluster-3 </td><td>1.4 </td><td>2 </td><td>2 </td><td>Unlinked </td><td>Unlinked </td></tr><tr><td>Unlinked </td><td>— </td><td>— </td><td>12 </td><td> </td><td> </td></tr><tr><td>Total </td><td>2361.8 </td><td>637 </td><td>1101 </td><td> </td><td> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Framework markers were used to form the linkage groups. Total loci includes both framework and attached loci (see text for details).</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4838830" target="_blank">View Large</a></div></div><p>This newly constructed genetic map of <em>Lu. angustifolius</em> involved a significantly greater number of markers and a more sophisticated mapping approach (the evolutionary optimization strategy) compared with the two previous maps. Jack-knife re-sampling was highly effective in identifying framework markers that were used to construct stable linkage groups onto which lower quality (and redundant) markers were later attached. This approach is particularly helpful in studies where marker data have been pooled from several sources since the quality of marker genotyping is likely to vary due to differing genotyping technologies and/or technical expertise. These factors led to a greatly improved map with the number of linkage groups equalling the haploid number for this species (<em>n</em> = 20).</p><p>This map cannot yet be considered saturated with markers since 18 intervals exceed 15 cM and 1 interval exceeds 20 cM (Fig. <a reveal-id="DSQ001F1" data-open="DSQ001F1" class="link link-reveal link-table xref-fig">1</a>, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). However, the marker density is ample for most mapping purposes. Eighteen markers showing significant (<em>P</em> &lt; 0.01) segregation distortion were distributed across nine linkage groups, with the majority (14) being dominant MFLP markers. Therefore, these distorted loci are more likely to have arisen from marker genotyping errors than being an indication of imbalance in the RIL mapping population. The population of RILs used to construct this map is available to the research community on request. With these resources, this map can be considered the new reference genetic map for <em>Lu. angustifolius</em>.</p>                            <h3 scrollto-destination=4838835 id="4838835" class="section-title" data-legacy-id=s3b>3.2. Comparing the genomes of Lu. angustifolius and Lo. japonicus</h3>
<p>The new genetic map of <em>Lu. angustifolius</em> was compared with the recently released genome sequence of the model legume <em>Lo. japonicus</em>.<sup><a reveal-id="DSQ001C17" data-open="DSQ001C17" class="link link-ref link-reveal xref-bibr">17</a></sup> This was achieved by comparing the DNA sequences of 311 STS markers (representing 363 mapped loci in <em>Lu. angustifolius</em>) to the genome sequence of <em>Lo. japonicus</em> using BLASTn homology searching. At the significance threshold of <em>P</em> &lt; 1e<sup>−20</sup>, 159 markers (detecting 187 <em>Lu. angustifolius</em> loci) matched one or more locations in the <em>Lo. japonicus</em> genome. Seven markers (UWA023, UWA097, UWA158, UWA160, UWA270, UWA300 and LSSR18) representing 16 mapped loci in <em>Lu. angustifolius</em> detected repetitive sequences in the <em>Lo. japonicus</em> genome and were removed from further analysis. The remaining 152 markers (representing 171 mapped loci in <em>Lu. angustifolius</em>) detected one or more correspondences in the <em>Lo. japonicus</em> genome. In total, there were 242 correspondences between the two genomes, and these are plotted graphically in Fig. <a reveal-id="DSQ001F2" data-open="DSQ001F2" class="link link-reveal link-table xref-fig">2</a> and summarized in Table <a reveal-id="DSQ001TB2" data-open="DSQ001TB2" class="link link-reveal link-table xref-fig">2</a>.</p>                        <a id="4838837" scrollto-destination="4838837"></a>
<div id="DSQ001F2" data-id="DSQ001F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/2/10.1093/dnares/dsq001/2/m_dsq00102.jpeg?Expires=2147483647&Signature=UnPdYTLCu4DnYlO9vmn92UIh~-Jd3aXoGAK-6YkXRfgmoBo~hKWxXnkhXWHNMYgbqrGZstiwg0DhIaT1nr8lGvwutrUjO6mgyh880-ARyEIKefGS0buSuNaz-MkD4iRJdD6zAX0nwlDnRO25GAoQC6giBkzx1WUbZEw04SNiVo~YiGVcfNGeLWG25oRUDvwH97pLQnpiTqY1jF69nTuiL42QHF8U4rHeNUnj05hsdcG9tcHkSbucf3vYhqzXSx3iYdgvumUthpZKMv0jZAiRam7WF42bK5hVKCTqgn9dQmR2515FxYwYYv7s4fQDvlvpOlDRxJz5ipI1Qt0EoGhD3w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Global distribution of synteny between 20 linkage groups of Lu. angustifolius (NLL-01 to NLL-20) and 6 chromosomes of Lo. japonicus (Lj1–Lj6). Loci showing homology between the two genomes at P &lt; 1e−20 significance threshold are indicated by dots." path-from-xml="dsq00102.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4838837/dsq00102.jpeg" path-from-xml="dsq00102.jpeg" target="_blank">View large</a><a section="4838837" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/2/10.1093/dnares/dsq001/2/dsq00102.jpeg?Expires=2147483647&Signature=UikNmnwnGM-MIUYzIvfqjnaMj-slh66IFjCESR68XTQvWwgTvg0bxo5pcQ0vJoRQGnnwbQnt-Ny4DfCoHTwJRZiegty4dl-V2AT3lYn8JsU-Tn91FvQ2kiKNDfbqra6T3LTXcv48uwEmp4CJfSGvnNTsFU4crv6XP6-aXQMruo8etjccrNn4oYHxVpJOj7gbGNdNVEwb4Bk5H4gkV8xLH2IA~jaqYnciG2AqxQx6RtN4F3~Nuvn7p0h-ZZAcsbzdhSdAAjzMbDWN6nHtatzEfn6wCCEoqYkFepqVGDflD159VTWk6RZk09shxhRjnpGDE5gT8TH47zFvQ1VsJPxieA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4838837&ar=344357&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq00102.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Global distribution of synteny between 20 linkage groups of <em>Lu. angustifolius</em> (NLL-01 to NLL-20) and 6 chromosomes of <em>Lo. japonicus</em> (Lj1–Lj6). Loci showing homology between the two genomes at <em>P</em> &lt; 1e<sup>−20</sup> significance threshold are indicated by dots.</p></div></div><div content-id="DSQ001F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/2/10.1093/dnares/dsq001/2/m_dsq00102.jpeg?Expires=2147483647&Signature=UnPdYTLCu4DnYlO9vmn92UIh~-Jd3aXoGAK-6YkXRfgmoBo~hKWxXnkhXWHNMYgbqrGZstiwg0DhIaT1nr8lGvwutrUjO6mgyh880-ARyEIKefGS0buSuNaz-MkD4iRJdD6zAX0nwlDnRO25GAoQC6giBkzx1WUbZEw04SNiVo~YiGVcfNGeLWG25oRUDvwH97pLQnpiTqY1jF69nTuiL42QHF8U4rHeNUnj05hsdcG9tcHkSbucf3vYhqzXSx3iYdgvumUthpZKMv0jZAiRam7WF42bK5hVKCTqgn9dQmR2515FxYwYYv7s4fQDvlvpOlDRxJz5ipI1Qt0EoGhD3w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Global distribution of synteny between 20 linkage groups of Lu. angustifolius (NLL-01 to NLL-20) and 6 chromosomes of Lo. japonicus (Lj1–Lj6). Loci showing homology between the two genomes at P &lt; 1e−20 significance threshold are indicated by dots." path-from-xml="dsq00102.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4838837/dsq00102.jpeg" path-from-xml="dsq00102.jpeg" target="_blank">View large</a><a section="4838837" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/2/10.1093/dnares/dsq001/2/dsq00102.jpeg?Expires=2147483647&Signature=UikNmnwnGM-MIUYzIvfqjnaMj-slh66IFjCESR68XTQvWwgTvg0bxo5pcQ0vJoRQGnnwbQnt-Ny4DfCoHTwJRZiegty4dl-V2AT3lYn8JsU-Tn91FvQ2kiKNDfbqra6T3LTXcv48uwEmp4CJfSGvnNTsFU4crv6XP6-aXQMruo8etjccrNn4oYHxVpJOj7gbGNdNVEwb4Bk5H4gkV8xLH2IA~jaqYnciG2AqxQx6RtN4F3~Nuvn7p0h-ZZAcsbzdhSdAAjzMbDWN6nHtatzEfn6wCCEoqYkFepqVGDflD159VTWk6RZk09shxhRjnpGDE5gT8TH47zFvQ1VsJPxieA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4838837&ar=344357&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq00102.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Global distribution of synteny between 20 linkage groups of <em>Lu. angustifolius</em> (NLL-01 to NLL-20) and 6 chromosomes of <em>Lo. japonicus</em> (Lj1–Lj6). Loci showing homology between the two genomes at <em>P</em> &lt; 1e<sup>−20</sup> significance threshold are indicated by dots.</p></div></div>                        <a id="4838838" scrollto-destination="4838838"></a>
<div content-id="DSQ001TB2" class="table-modal"><div class="table-wrap"><div id="DSQ001TB2" class="table-wrap-title" data-id="DSQ001TB2"><div class="label -label">Table 2</div><div class="caption"><p>Numbers of <em>Lu. angustifolius</em> loci with BLASTn (<em>P</em> &lt; 1e<sup>−20</sup>) correspondences in six <em>Lo. japonicus</em> chromosomes (Lj1–Lj6)</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th><em>Lotus japonicus</em> chromosomes </th><th>Primary correspondences </th><th>All correspondences<sup>a</sup> </th></tr></thead><tbody><tr><td>Lj1 </td><td>40 </td><td>60 </td></tr><tr><td>Lj2 </td><td>36 </td><td>52 </td></tr><tr><td>Lj3 </td><td>31 </td><td>47 </td></tr><tr><td>Lj4 </td><td>36 </td><td>44 </td></tr><tr><td>Lj5 </td><td>15 </td><td>22 </td></tr><tr><td>Lj6 </td><td>13 </td><td>17 </td></tr><tr><td>Total </td><td>171 </td><td>242 </td></tr></tbody></table></div><div content-id="DSQ001TB2" class="table-modal"><table><thead name="thead" align=""><tr><th><em>Lotus japonicus</em> chromosomes </th><th>Primary correspondences </th><th>All correspondences<sup>a</sup> </th></tr></thead><tbody><tr><td>Lj1 </td><td>40 </td><td>60 </td></tr><tr><td>Lj2 </td><td>36 </td><td>52 </td></tr><tr><td>Lj3 </td><td>31 </td><td>47 </td></tr><tr><td>Lj4 </td><td>36 </td><td>44 </td></tr><tr><td>Lj5 </td><td>15 </td><td>22 </td></tr><tr><td>Lj6 </td><td>13 </td><td>17 </td></tr><tr><td>Total </td><td>171 </td><td>242 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Primary, secondary and tertiary BLASTn correspondences.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4838838" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSQ001TB2" class="table-wrap-title" data-id="DSQ001TB2"><div class="label -label">Table 2</div><div class="caption"><p>Numbers of <em>Lu. angustifolius</em> loci with BLASTn (<em>P</em> &lt; 1e<sup>−20</sup>) correspondences in six <em>Lo. japonicus</em> chromosomes (Lj1–Lj6)</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th><em>Lotus japonicus</em> chromosomes </th><th>Primary correspondences </th><th>All correspondences<sup>a</sup> </th></tr></thead><tbody><tr><td>Lj1 </td><td>40 </td><td>60 </td></tr><tr><td>Lj2 </td><td>36 </td><td>52 </td></tr><tr><td>Lj3 </td><td>31 </td><td>47 </td></tr><tr><td>Lj4 </td><td>36 </td><td>44 </td></tr><tr><td>Lj5 </td><td>15 </td><td>22 </td></tr><tr><td>Lj6 </td><td>13 </td><td>17 </td></tr><tr><td>Total </td><td>171 </td><td>242 </td></tr></tbody></table></div><div content-id="DSQ001TB2" class="table-modal"><table><thead name="thead" align=""><tr><th><em>Lotus japonicus</em> chromosomes </th><th>Primary correspondences </th><th>All correspondences<sup>a</sup> </th></tr></thead><tbody><tr><td>Lj1 </td><td>40 </td><td>60 </td></tr><tr><td>Lj2 </td><td>36 </td><td>52 </td></tr><tr><td>Lj3 </td><td>31 </td><td>47 </td></tr><tr><td>Lj4 </td><td>36 </td><td>44 </td></tr><tr><td>Lj5 </td><td>15 </td><td>22 </td></tr><tr><td>Lj6 </td><td>13 </td><td>17 </td></tr><tr><td>Total </td><td>171 </td><td>242 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Primary, secondary and tertiary BLASTn correspondences.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4838838" target="_blank">View Large</a></div></div><p>Of the 120 pairwise chromosome comparisons (i.e. 20 <em>Lu. angustifolius</em> × 6 <em>Lo. japonicus</em> chromosomes), 50 had at least two correspondences (the minimum for detecting synteny) and 34 had at least three correspondences (the minimum for detecting collinearity). There were clear differences in the degree of conserved collinearity on <em>Lu. angustifolius</em> and <em>Lo. japonicus</em> chromosomes with some regions showing good conservation whereas other regions were highly rearranged with respect to each other (Fig. <a reveal-id="DSQ001F2" data-open="DSQ001F2" class="link link-reveal link-table xref-fig">2</a>). This heterogeneous pattern of genome collinearity was also observed in an earlier genome comparison between <em>Lu. angustifolius</em> (a Genistoid species) and <em>M. truncatula</em> (a Galegoid species),<sup><a reveal-id="DSQ001C7" data-open="DSQ001C7" class="link link-ref link-reveal xref-bibr">7</a></sup> and provides additional support to the wide evolutionary distance believed to separate the Galegoids and Genistoids.<sup><a reveal-id="DSQ001C5" data-open="DSQ001C5" class="link link-ref link-reveal xref-bibr">5</a></sup> A similarly heterogeneous pattern was observed between <em>Arachis hypogea</em> (groundnut, in the Dalbergioid clade, which is also considered relatively basal in the evolution of Papilionoid legumes) and the Galegoid species <em>Lo. japonicus</em> and <em>M. truncatula</em>.<sup><a reveal-id="DSQ001C28" data-open="DSQ001C28" class="link link-ref link-reveal xref-bibr">28</a></sup> It is interesting to note that arguably the best conserved section between the <em>Arachis</em> and <em>Lo. japonicus</em> genomes on chromosomes Ar6 and Lj1 (synteny block 5, delineated by contigs CM0222–CM0105), respectively, appears to shows no conservation in the <em>Lu. angustifolius</em> genome. A more thorough analysis will be required to make firm conclusions, but on first inspection it would appear that regions of conserved gene order may not be generalized across wide evolutionary distance within the Papilionoid legumes. This is in contrast to the crucifer family where large variations in chromosome numbers and frequent rearrangements exist, but where the integrity of many chromosomal blocks has been maintained over a similarly long evolutionary timeframe.<sup><a reveal-id="DSQ001C29" data-open="DSQ001C29" class="link link-ref link-reveal xref-bibr">29</a></sup></p><p>Three regions showing relatively good conservation of locus order between <em>Lu. angustifolius</em> and <em>Lo. japonicus</em> are shown in Fig. <a reveal-id="DSQ001F3" data-open="DSQ001F3" class="link link-reveal link-table xref-fig">3</a>. Even these relatively well-conserved regions show evidence of chromosome translocations and/or inversions, which will make exploitation of synteny by lupin researchers more difficult. One of the potential applications of synteny is map alignment-based identification of candidate genes for important traits in the crop species by searching for likely candidates in the equivalent region of the model genome. For example, the pod shatter resistance gene <em>lentus</em> is located on <em>Lu. angustifolius</em> linkage group NLL-08 in a region with relatively well-conserved gene order with <em>Lo. japonicus</em> chromosome 5 (Fig. <a reveal-id="DSQ001F3" data-open="DSQ001F3" class="link link-reveal link-table xref-fig">3</a>). Unfortunately, there appears to be an inversion in the interval containing <em>lentus</em> such that the equivalent region of <em>Lo. japonicus</em> is somewhat ambiguous and is only delimited to the lower half of chromosome 5. Clearly, more bridging points will be required to make such map-based inferences in <em>Lu. angustifolius</em> using <em>Lo. japonicus</em> as a model genome and therefore more gene-based markers are required in <em>Lu. angustifolius</em>.</p>                        <a id="4838841" scrollto-destination="4838841"></a>
<div id="DSQ001F3" data-id="DSQ001F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/2/10.1093/dnares/dsq001/2/m_dsq00103.jpeg?Expires=2147483647&Signature=N-HfLyOih~pC4GM0cREr~PEl~16MAjzitQSXbjpsiv4YRDfAoGgvSlTmwrB0ykU4-BMkCuyMiaRTwlt9sfRo9mSw-ccQXwPBvcbZdCM7--UyBDrcQRiUyXrgNX7JudtEl6hs0paJ0VLDZ57jLRBfiO4iAUwOcrfZK02O3ZZVv-ZF4ZlSxcy219K4LjNlzN~KWNHCUm1wH9YEmyEpJdCCuqkkX19lYW20AWX~jvWQq40gyf-GcanVW5LzpO75nUdadTMRT6ytZXmwOVsiEmhR25GLt3n5tEfbgxy7tvOoqTthR9lNj7fEB6jgtr~z15Hp7QHNqypVWtKAXcMoxWDoKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Three Lu. angustifolius linkage groups (NLL-08, NLL-06 and NLL-12) showing conserved gene order with Lo. japonicus chromosomes Lj2, Lj3, Lj4 and Lj5. The pod shatter resistance gene, Lentus, is located on NLL-08. Names of loci showing conserved synteny between the genomes are shown next to the Lu. angustifolius linkage groups (scaled in Kosambi centiMorgans, cM). The start position of homologous sequences on Lo. japonicus pseudomolecules is shown next to Lo. japonicus chromosomes (scaled in megabases, Mb)." path-from-xml="dsq00103.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4838841/dsq00103.jpeg" path-from-xml="dsq00103.jpeg" target="_blank">View large</a><a section="4838841" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/2/10.1093/dnares/dsq001/2/dsq00103.jpeg?Expires=2147483647&Signature=4LwRvPJS7T9XQ8C4FSwBxuWiOhJT-XwFa3e5Jy2T7Jk5aC99e7CZOBGyonrSoOuSCZ4uB3tO9FrucWUeIrNxEv41xjrRUKrSNECBk~KJbjWS5G1RP~TAoJAUdXZozaDwSHszyX-VZLZI0GNgpLkaBI18stcfeIAFLER0r2Scvk5J~oaKjkg2iJfV-Res-aysfgvPcAKw-7yEBOUhHCyQesF-LnF2Ecc~xhmr3kc~MQzKBdvLxaQk2zP8OCv1c3Vm9J3JqjArFIukKa-pQCu9TnQhXy6~dmSaRu0Z2Nb6u~NXY~aYcr~pOK-iS0QQne1YR0qI-TbcZ-wHtBDy~VofeQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4838841&ar=344357&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq00103.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Three <em>Lu. angustifolius</em> linkage groups (NLL-08, NLL-06 and NLL-12) showing conserved gene order with <em>Lo. japonicus</em> chromosomes Lj2, Lj3, Lj4 and Lj5. The pod shatter resistance gene, <em>Lentus</em>, is located on NLL-08. Names of loci showing conserved synteny between the genomes are shown next to the <em>Lu. angustifolius</em> linkage groups (scaled in Kosambi centiMorgans, cM). The start position of homologous sequences on <em>Lo. japonicus</em> pseudomolecules is shown next to <em>Lo. japonicus</em> chromosomes (scaled in megabases, Mb).</p></div></div><div content-id="DSQ001F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/2/10.1093/dnares/dsq001/2/m_dsq00103.jpeg?Expires=2147483647&Signature=N-HfLyOih~pC4GM0cREr~PEl~16MAjzitQSXbjpsiv4YRDfAoGgvSlTmwrB0ykU4-BMkCuyMiaRTwlt9sfRo9mSw-ccQXwPBvcbZdCM7--UyBDrcQRiUyXrgNX7JudtEl6hs0paJ0VLDZ57jLRBfiO4iAUwOcrfZK02O3ZZVv-ZF4ZlSxcy219K4LjNlzN~KWNHCUm1wH9YEmyEpJdCCuqkkX19lYW20AWX~jvWQq40gyf-GcanVW5LzpO75nUdadTMRT6ytZXmwOVsiEmhR25GLt3n5tEfbgxy7tvOoqTthR9lNj7fEB6jgtr~z15Hp7QHNqypVWtKAXcMoxWDoKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Three Lu. angustifolius linkage groups (NLL-08, NLL-06 and NLL-12) showing conserved gene order with Lo. japonicus chromosomes Lj2, Lj3, Lj4 and Lj5. The pod shatter resistance gene, Lentus, is located on NLL-08. Names of loci showing conserved synteny between the genomes are shown next to the Lu. angustifolius linkage groups (scaled in Kosambi centiMorgans, cM). The start position of homologous sequences on Lo. japonicus pseudomolecules is shown next to Lo. japonicus chromosomes (scaled in megabases, Mb)." path-from-xml="dsq00103.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4838841/dsq00103.jpeg" path-from-xml="dsq00103.jpeg" target="_blank">View large</a><a section="4838841" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/2/10.1093/dnares/dsq001/2/dsq00103.jpeg?Expires=2147483647&Signature=4LwRvPJS7T9XQ8C4FSwBxuWiOhJT-XwFa3e5Jy2T7Jk5aC99e7CZOBGyonrSoOuSCZ4uB3tO9FrucWUeIrNxEv41xjrRUKrSNECBk~KJbjWS5G1RP~TAoJAUdXZozaDwSHszyX-VZLZI0GNgpLkaBI18stcfeIAFLER0r2Scvk5J~oaKjkg2iJfV-Res-aysfgvPcAKw-7yEBOUhHCyQesF-LnF2Ecc~xhmr3kc~MQzKBdvLxaQk2zP8OCv1c3Vm9J3JqjArFIukKa-pQCu9TnQhXy6~dmSaRu0Z2Nb6u~NXY~aYcr~pOK-iS0QQne1YR0qI-TbcZ-wHtBDy~VofeQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4838841&ar=344357&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq00103.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Three <em>Lu. angustifolius</em> linkage groups (NLL-08, NLL-06 and NLL-12) showing conserved gene order with <em>Lo. japonicus</em> chromosomes Lj2, Lj3, Lj4 and Lj5. The pod shatter resistance gene, <em>Lentus</em>, is located on NLL-08. Names of loci showing conserved synteny between the genomes are shown next to the <em>Lu. angustifolius</em> linkage groups (scaled in Kosambi centiMorgans, cM). The start position of homologous sequences on <em>Lo. japonicus</em> pseudomolecules is shown next to <em>Lo. japonicus</em> chromosomes (scaled in megabases, Mb).</p></div></div><p>In conclusion, we present a high-resolution reference map for <em>Lu. angustifolius</em> which will serve as a shared resource for the legume genetic community. We discovered that the <em>Lu. angustifolius</em> genome has numerous chromosomal rearrangements relative to the <em>Lo. japonicus</em> genome, though widespread but small sections of conserved gene order are present. As an outlying species compared with other legume model and crop species, <em>Lu. angustifolius</em> serves as a useful reference for gaining better understanding of legume genome evolution.</p>                            <h2 scrollto-destination=4838843 id="4838843" class="section-title" data-legacy-id=s4>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=4838845 id="4838845" class="section-title" data-legacy-id=s5>Funding</h2>
<p>Generation of the GLIP STS markers was supported by the EU 6th Framework Programme Grain Legumes Integrated Project, no. FP-2002-FOOD-506223. Generation of isozyme and BEST markers was financed from the project of Polish Ministry of Sciences and Higher Education no. PBZ-MniSW-2/3/2006/3.</p>                            <h2 scrollto-destination=4838847 id="4838847" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank Dr G.B. Kiss (Laboratory of Plant Genetics and Breeding, Agricultural Biotechnology Center, Godollo, Hungary) and Prof. N.F. Weeden (Department of Plant Sciences and Plant Pathology, Montana State University, Bozeman, USA) for providing primer pairs for STS markers.</p>                            <h2 scrollto-destination=4838849 id="4838849" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSQ001C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ001C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ellwood</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Phan</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Jordan</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Construction of a comparative genetic map in faba bean (<em>Vicia faba</em> L.); conservation of genome structure with <em>Lens culinaris</em></div>, <div class="source">BMC Genomics</div> , <div class="year">2008</div>, vol. <div class="volume">9</div> pg. <div class="fpage">380</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20a%20comparative%20genetic%20map%20in%20faba%20bean%20(Vicia%20faba%20L.)%3B%20conservation%20of%20genome%20structure%20with%20Lens%20culinaris&amp;author=S.%20Ellwood&amp;author=H.%20Phan&amp;author=M.%20Jordan&amp;publication_year=2008&amp;journal=BMC%20Genomics&amp;volume=9&amp;pages=380" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-9-380" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-9-380" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-9-380"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18691425" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ001C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ001C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Phan</div> <div class="given-names">H.T.T.</div></div>,  <div class="name"><div class="surname">Ellwood</div> <div class="given-names">S.R.</div></div>,  <div class="name"><div class="surname">Hane</div> <div class="given-names">J.K.</div></div>,  <div class="name"><div class="surname">Ford</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Materne</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Oliver</div> <div class="given-names">R.P.</div></div>. </div><div class="article-title">Extensive macrosynteny between <em>Medicago truncatula</em> and <em>Lens culinaris</em> ssp. <em>culinaris</em></div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2007</div>, vol. <div class="volume">114</div> (pg. <div class="fpage">549</div>-<div class="lpage">58</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Extensive%20macrosynteny%20between%20Medicago%20truncatula%20and%20Lens%20culinaris%20ssp.%20culinaris&amp;author=H.T.T.%20Phan&amp;author=S.R.%20Ellwood&amp;author=J.K.%20Hane&amp;author=R.%20Ford&amp;author=M.%20Materne&amp;author=R.P.%20Oliver&amp;publication_year=2007&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=114&amp;pages=549-58" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-006-0455-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-006-0455-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-006-0455-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17119911" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ001C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ001C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Young</div> <div class="given-names">N.D.</div></div>,  <div class="name"><div class="surname">Cannon</div> <div class="given-names">S.B.</div></div>,  <div class="name"><div class="surname">Sato</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Sequencing the genespaces of <em>Medicago truncatula</em> and <em>Lotus japonicus</em></div>, <div class="source">Plant Physiol.</div> , <div class="year">2005</div>, vol. <div class="volume">137</div> (pg. <div class="fpage">1174</div>-<div class="lpage">81</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequencing%20the%20genespaces%20of%20Medicago%20truncatula%20and%20Lotus%20japonicus&amp;author=N.D.%20Young&amp;author=S.B.%20Cannon&amp;author=S.%20Sato&amp;publication_year=2005&amp;journal=Plant%20Physiol.&amp;volume=137&amp;pages=1174-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.104.057034" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.104.057034" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.104.057034"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15824279" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ001C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ001C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Choi</div> <div class="given-names">H.-K.</div></div>,  <div class="name"><div class="surname">Mun</div> <div class="given-names">J.-H.</div></div>,  <div class="name"><div class="surname">Kim</div> <div class="given-names">D.-J.</div></div>, et al. </div><div class="article-title">Estimating genome conservation between crop and model legume species</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2004</div>, vol. <div class="volume">101</div> (pg. <div class="fpage">15289</div>-<div class="lpage">94</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Estimating%20genome%20conservation%20between%20crop%20and%20model%20legume%20species&amp;author=H.-K.%20Choi&amp;author=J.-H.%20Mun&amp;author=D.-J.%20Kim&amp;publication_year=2004&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=101&amp;pages=15289-94" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0402251101" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0402251101" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0402251101"> </span></div> </div></div></div></div></div><div content-id="DSQ001C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ001C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lavin</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Herendeen</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Wojchiechowski</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Evolutionary rates analysis of Leguminosae implicates a rapid diversification of lineages during the Tertiary</div>, <div class="source">Syst. Biol.</div> , <div class="year">2005</div>, vol. <div class="volume">54</div> (pg. <div class="fpage">575</div>-<div class="lpage">94</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolutionary%20rates%20analysis%20of%20Leguminosae%20implicates%20a%20rapid%20diversification%20of%20lineages%20during%20the%20Tertiary&amp;author=M.%20Lavin&amp;author=P.%20Herendeen&amp;author=M.%20Wojchiechowski&amp;publication_year=2005&amp;journal=Syst.%20Biol.&amp;volume=54&amp;pages=575-94" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1080/10635150590947131" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F10635150590947131" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1080%2F10635150590947131"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16085576" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ001C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ001C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Naganowska</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Wolko</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Sliwinska</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Kaczmarek</div> <div class="given-names">Z.</div></div>. </div><div class="article-title">Nuclear DNA content variation and species relationships in the genus <em>Lupinus</em> (Fabaceae)</div>, <div class="source">Ann. Bot.</div> , <div class="year">2003</div>, vol. <div class="volume">92</div> (pg. <div class="fpage">349</div>-<div class="lpage">55</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nuclear%20DNA%20content%20variation%20and%20species%20relationships%20in%20the%20genus%20Lupinus%20(Fabaceae)&amp;author=B.%20Naganowska&amp;author=B.%20Wolko&amp;author=E.%20Sliwinska&amp;author=Z.%20Kaczmarek&amp;publication_year=2003&amp;journal=Ann.%20Bot.&amp;volume=92&amp;pages=349-55" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/aob/mcg145" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Faob%2Fmcg145" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Faob%2Fmcg145"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12853281" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ001C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ001C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nelson</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Phan</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Ellwood</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">The first gene-based map of <em>Lupinus angustifolius</em> L.—location of domestication genes and conserved synteny with <em>Medicago truncatula</em></div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2006</div>, vol. <div class="volume">113</div> (pg. <div class="fpage">225</div>-<div class="lpage">38</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20first%20gene-based%20map%20of%20Lupinus%20angustifolius%20L.%E2%80%94location%20of%20domestication%20genes%20and%20conserved%20synteny%20with%20Medicago%20truncatula&amp;author=M.%20Nelson&amp;author=H.%20Phan&amp;author=S.%20Ellwood&amp;publication_year=2006&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=113&amp;pages=225-38" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-006-0288-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-006-0288-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-006-0288-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16791689" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ001C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ001C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wolko</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Weeden</div> <div class="given-names">N.</div></div>. </div><div class="article-title">Estimation of <em>Lupinus</em> genome polyploidy on the basis of isozymic loci number</div>, <div class="source">Genet. Pol.</div> , <div class="year">1989</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">165</div>-<div class="lpage">71</div>)</div></div></div></div><div content-id="DSQ001C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ001C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Boersma</div> <div class="given-names">J.G.</div></div>,  <div class="name"><div class="surname">Pallotta</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">C.D.</div></div>,  <div class="name"><div class="surname">Buirchell</div> <div class="given-names">B.J.</div></div>,  <div class="name"><div class="surname">Sivasithamparam</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Yang</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Construction of a genetic linkage map using MFLP and identification of molecular markers linked to domestication genes in narrow-leafed lupin (<em>Lupinus angustifolius</em> L.)</div>, <div class="source">Cell. Mol. Biol. Lett.</div> , <div class="year">2005</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">331</div>-<div class="lpage">44</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20a%20genetic%20linkage%20map%20using%20MFLP%20and%20identification%20of%20molecular%20markers%20linked%20to%20domestication%20genes%20in%20narrow-leafed%20lupin%20(Lupinus%20angustifolius%20L.)&amp;author=J.G.%20Boersma&amp;author=M.%20Pallotta&amp;author=C.D.%20Li&amp;author=B.J.%20Buirchell&amp;author=K.%20Sivasithamparam&amp;author=H.%20Yang&amp;publication_year=2005&amp;journal=Cell.%20Mol.%20Biol.%20Lett.&amp;volume=10&amp;pages=331-44" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16010297" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ001C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ001C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Boersma</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Nelson</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Sivasithamparam</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Yang</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Development of sequence-specific PCR markers linked to the <em>Tardus</em> gene that reduces pod shattering in narrow-leafed lupin (<em>Lupinus angustifolius</em> L.)</div>, <div class="source">Mol. Breed.</div> , <div class="year">2009</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">259</div>-<div class="lpage">67</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20sequence-specific%20PCR%20markers%20linked%20to%20the%20Tardus%20gene%20that%20reduces%20pod%20shattering%20in%20narrow-leafed%20lupin%20(Lupinus%20angustifolius%20L.)&amp;author=J.%20Boersma&amp;author=M.%20Nelson&amp;author=K.%20Sivasithamparam&amp;author=H.%20Yang&amp;publication_year=2009&amp;journal=Mol.%20Breed.&amp;volume=23&amp;pages=259-67" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11032-008-9230-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11032-008-9230-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11032-008-9230-2"> </span></div> </div></div></div></div></div><div content-id="DSQ001C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ001C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Boersma</div> <div class="given-names">J.G.</div></div>,  <div class="name"><div class="surname">Buirchell</div> <div class="given-names">B.J.</div></div>,  <div class="name"><div class="surname">Sivasithamparam</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Yang</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Development of two sequence-specific PCR markers linked to the <em>le</em> gene that reduces pod shattering in narrow-leafed Lupin (<em>Lupinus angustifolius</em> L.)</div>, <div class="source">Genet. Mol. Biol.</div> , <div class="year">2007</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">623</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20two%20sequence-specific%20PCR%20markers%20linked%20to%20the%20le%20gene%20that%20reduces%20pod%20shattering%20in%20narrow-leafed%20Lupin%20(Lupinus%20angustifolius%20L.)&amp;author=J.G.%20Boersma&amp;author=B.J.%20Buirchell&amp;author=K.%20Sivasithamparam&amp;author=H.%20Yang&amp;publication_year=2007&amp;journal=Genet.%20Mol.%20Biol.&amp;volume=30&amp;pages=623-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1590/S1415-47572007000400020" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1590%2FS1415-47572007000400020" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1590%2FS1415-47572007000400020"> </span></div> </div></div></div></div></div><div content-id="DSQ001C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ001C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Boersma</div> <div class="given-names">J.G.</div></div>,  <div class="name"><div class="surname">Buirchell</div> <div class="given-names">B.J.</div></div>,  <div class="name"><div class="surname">Sivasithamparam</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Yang</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Development of a sequence-specific PCR marker linked to the <em>Ku</em> gene which removes the vernalization requirement in narrow-leafed lupin</div>, <div class="source">Plant Breed.</div> , <div class="year">2007</div>, vol. <div class="volume">126</div> (pg. <div class="fpage">306</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20a%20sequence-specific%20PCR%20marker%20linked%20to%20the%20Ku%20gene%20which%20removes%20the%20vernalization%20requirement%20in%20narrow-leafed%20lupin&amp;author=J.G.%20Boersma&amp;author=B.J.%20Buirchell&amp;author=K.%20Sivasithamparam&amp;author=H.%20Yang&amp;publication_year=2007&amp;journal=Plant%20Breed.&amp;volume=126&amp;pages=306-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/pbr.2007.126.issue-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fpbr.2007.126.issue-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fpbr.2007.126.issue-3"> </span></div> </div></div></div></div></div><div content-id="DSQ001C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ001C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Boersma</div> <div class="given-names">J.G.</div></div>,  <div class="name"><div class="surname">Buirchell</div> <div class="given-names">B.J.</div></div>,  <div class="name"><div class="surname">Sivasithamparam</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Yang</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Development of a PCR marker tightly linked to <em>mollis</em>, the gene that controls seed dormancy in <em>Lupinus angustifolius</em> L</div>, <div class="source">Plant Breed.</div> , <div class="year">2007</div>, vol. <div class="volume">126</div> (pg. <div class="fpage">612</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20a%20PCR%20marker%20tightly%20linked%20to%20mollis%2C%20the%20gene%20that%20controls%20seed%20dormancy%20in%20Lupinus%20angustifolius%20L&amp;author=J.G.%20Boersma&amp;author=B.J.%20Buirchell&amp;author=K.%20Sivasithamparam&amp;author=H.%20Yang&amp;publication_year=2007&amp;journal=Plant%20Breed.&amp;volume=126&amp;pages=612-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/pbr.2007.126.issue-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fpbr.2007.126.issue-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fpbr.2007.126.issue-6"> </span></div> </div></div></div></div></div><div content-id="DSQ001C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ001C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">You</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Boersma</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Buirchell</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Sweetingham</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Siddique</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Yang</div> <div class="given-names">H.</div></div>. </div><div class="article-title">A PCR-based molecular marker applicable for marker-assisted selection for anthracnose disease resistance in lupin breeding</div>, <div class="source">Cell. Mol. Biol. Lett.</div> , <div class="year">2005</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">123</div>-<div class="lpage">34</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20PCR-based%20molecular%20marker%20applicable%20for%20marker-assisted%20selection%20for%20anthracnose%20disease%20resistance%20in%20lupin%20breeding&amp;author=M.%20You&amp;author=J.%20Boersma&amp;author=B.%20Buirchell&amp;author=M.%20Sweetingham&amp;author=K.%20Siddique&amp;author=H.%20Yang&amp;publication_year=2005&amp;journal=Cell.%20Mol.%20Biol.%20Lett.&amp;volume=10&amp;pages=123-34" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15809684" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ001C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ001C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kaczmarek</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Naganowska</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Wolko</div> <div class="given-names">B.</div></div>. </div><div class="article-title">Karyotyping of the narrow-leafed lupin (<em>Lupinus angustifolius</em> L.) by using FISH, PRINS and computer measurements of chromosomes</div>, <div class="source">J. Appl. Genet.</div> , <div class="year">2009</div>, vol. <div class="volume">50</div> (pg. <div class="fpage">77</div>-<div class="lpage">82</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Karyotyping%20of%20the%20narrow-leafed%20lupin%20(Lupinus%20angustifolius%20L.)%20by%20using%20FISH%2C%20PRINS%20and%20computer%20measurements%20of%20chromosomes&amp;author=A.%20Kaczmarek&amp;author=B.%20Naganowska&amp;author=B.%20Wolko&amp;publication_year=2009&amp;journal=J.%20Appl.%20Genet.&amp;volume=50&amp;pages=77-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF03195657" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF03195657" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF03195657"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19433903" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ001C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ001C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Naganowska</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Zielinska</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Physical mapping of 18S-25S rDNA and 5S rDNA in <em>Lupinus</em> via fluorescent in situ hybridization</div>, <div class="source">Cell. Mol. Biol. Lett.</div> , <div class="year">2002</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">665</div>-<div class="lpage">70</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Physical%20mapping%20of%2018S-25S%20rDNA%20and%205S%20rDNA%20in%20Lupinus%20via%20fluorescent%20in%20situ%20hybridization&amp;author=B.%20Naganowska&amp;author=A.%20Zielinska&amp;publication_year=2002&amp;journal=Cell.%20Mol.%20Biol.%20Lett.&amp;volume=7&amp;pages=665-70" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12378225" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ001C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ001C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sato</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Nakamura</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Kaneko</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Genome structure of the legume, <em>Lotus japonicus</em></div>, <div class="source">DNA Res.</div> , <div class="year">2008</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">227</div>-<div class="lpage">39</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20structure%20of%20the%20legume%2C%20Lotus%20japonicus&amp;author=S.%20Sato&amp;author=Y.%20Nakamura&amp;author=T.%20Kaneko&amp;publication_year=2008&amp;journal=DNA%20Res.&amp;volume=15&amp;pages=227-39" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsn008" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsn008" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsn008"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18511435" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ001C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ001C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yang</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Boersma</div> <div class="given-names">J.G.</div></div>,  <div class="name"><div class="surname">You</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Buirchell</div> <div class="given-names">B.J.</div></div>,  <div class="name"><div class="surname">Sweetingham</div> <div class="given-names">M.W.</div></div>. </div><div class="article-title">Development and implementation of a sequence-specific PCR marker linked to a gene conferring resistance to anthracnose disease in narrow-leafed lupin (<em>Lupinus angustifolius</em> L.)</div>, <div class="source">Mol. Breed.</div> , <div class="year">2004</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">145</div>-<div class="lpage">51</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20and%20implementation%20of%20a%20sequence-specific%20PCR%20marker%20linked%20to%20a%20gene%20conferring%20resistance%20to%20anthracnose%20disease%20in%20narrow-leafed%20lupin%20(Lupinus%20angustifolius%20L.)&amp;author=H.%20Yang&amp;author=J.G.%20Boersma&amp;author=M.%20You&amp;author=B.J.%20Buirchell&amp;author=M.W.%20Sweetingham&amp;publication_year=2004&amp;journal=Mol.%20Breed.&amp;volume=14&amp;pages=145-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1023/B:MOLB.0000038003.49638.97" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1023%2FB:MOLB.0000038003.49638.97" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1023%2FB:MOLB.0000038003.49638.97"> </span></div> </div></div></div></div></div><div content-id="DSQ001C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ001C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wolko</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Weeden</div> <div class="given-names">N.</div></div>. </div><div class="article-title">Linkage map of isozymes and RAPD markers for <em>L. angustifolius</em> L</div>, <div class="year">1993</div><div class="conf-name">Proceedings of the VIIth International Lupin Conference</div><div class="publisher-loc">Evora, Portugal</div></div></div></div></div><div content-id="DSQ001C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ001C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kasprzak</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Šafar</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Janda</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Dolezel</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Wolko</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Naganowska</div> <div class="given-names">B.</div></div>. </div><div class="article-title">The bacterial artificial chromosome (BAC) library of the narrow-leafed lupin (<em>Lupinus angustifolius</em> L.)</div>, <div class="source">Cell. Mol. Biol. Lett.</div> , <div class="year">2006</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">396</div>-<div class="lpage">407</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20bacterial%20artificial%20chromosome%20(BAC)%20library%20of%20the%20narrow-leafed%20lupin%20(Lupinus%20angustifolius%20L.)&amp;author=A.%20Kasprzak&amp;author=J.%20%C5%A0afar&amp;author=J.%20Janda&amp;author=J.%20Dolezel&amp;author=B.%20Wolko&amp;author=B.%20Naganowska&amp;publication_year=2006&amp;journal=Cell.%20Mol.%20Biol.%20Lett.&amp;volume=11&amp;pages=396-407" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2478/s11658-006-0033-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2478%2Fs11658-006-0033-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2478%2Fs11658-006-0033-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16847554" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ001C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ001C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Vos</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Hogers</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Bleeker</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">AFLP: A new technique for DNA fingerprinting</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1995</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">4407</div>-<div class="lpage">14</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=AFLP%3A%20A%20new%20technique%20for%20DNA%20fingerprinting&amp;author=P.%20Vos&amp;author=R.%20Hogers&amp;author=M.%20Bleeker&amp;publication_year=1995&amp;journal=Nucleic%20Acids%20Res.&amp;volume=23&amp;pages=4407-14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/23.21.4407" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F23.21.4407" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F23.21.4407"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7501463" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ001C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ001C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yang</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Sweetingham</div> <div class="given-names">M.W.</div></div>,  <div class="name"><div class="surname">Cowling</div> <div class="given-names">W.A.</div></div>,  <div class="name"><div class="surname">Smith</div> <div class="given-names">P.M.C.</div></div>. </div><div class="article-title">DNA fingerprinting based on microsatellite-anchored fragment length polymorphisms, and isolation of sequence-specific PCR markers in lupin (<em>Lupinus angustifolius</em> L.)</div>, <div class="source">Mol. Breed.</div> , <div class="year">2001</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">203</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20fingerprinting%20based%20on%20microsatellite-anchored%20fragment%20length%20polymorphisms%2C%20and%20isolation%20of%20sequence-specific%20PCR%20markers%20in%20lupin%20(Lupinus%20angustifolius%20L.)&amp;author=H.%20Yang&amp;author=M.W.%20Sweetingham&amp;author=W.A.%20Cowling&amp;author=P.M.C.%20Smith&amp;publication_year=2001&amp;journal=Mol.%20Breed.&amp;volume=7&amp;pages=203-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1023/A:1011363205557" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1023%2FA:1011363205557" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1023%2FA:1011363205557"> </span></div> </div></div></div></div></div><div content-id="DSQ001C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ001C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mester</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Ronin</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Minkov</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Nevo</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Korol</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Constructing large-scale genetic maps using an evolutionary strategy algorithm</div>, <div class="source">Genetics</div> , <div class="year">2003</div>, vol. <div class="volume">165</div> (pg. <div class="fpage">2269</div>-<div class="lpage">82</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Constructing%20large-scale%20genetic%20maps%20using%20an%20evolutionary%20strategy%20algorithm&amp;author=D.%20Mester&amp;author=Y.%20Ronin&amp;author=D.%20Minkov&amp;author=E.%20Nevo&amp;author=A.%20Korol&amp;publication_year=2003&amp;journal=Genetics&amp;volume=165&amp;pages=2269-82" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14704202" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ001C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ001C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Altschul</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Gish</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Miller</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Myers</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Lipman</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Basic local alignment search tool</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">1990</div>, vol. <div class="volume">215</div> (pg. <div class="fpage">403</div>-<div class="lpage">10</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Basic%20local%20alignment%20search%20tool&amp;author=S.%20Altschul&amp;author=W.%20Gish&amp;author=W.%20Miller&amp;author=E.%20Myers&amp;author=D.%20Lipman&amp;publication_year=1990&amp;journal=J.%20Mol.%20Biol.&amp;volume=215&amp;pages=403-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0022-2836(05)80360-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0022-2836(05)80360-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0022-2836(05)80360-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2231712" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ001C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ001C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hunter</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Schibeci</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Hiew</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Grendel</div> <div class="given-names">B.</div></div>. </div><div class="article-title">A bioinformatics Web Service-based architecture for accessing HPC resources</div>, <div class="source">Australasian Workshop on Grid Computing and e-Research</div> , <div class="year">2005</div></div></div></div></div><div content-id="DSQ001C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ001C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Moolhuijzen</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Cakir</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Macgregor</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">LegumeDB bioinformatics resource: Comparative genomic analysis and novel cross-genera marker identification in lupin and pasture legume species</div>, <div class="source">Genome</div> , <div class="year">2006</div>, vol. <div class="volume">49</div> (pg. <div class="fpage">689</div>-<div class="lpage">99</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=LegumeDB%20bioinformatics%20resource%3A%20Comparative%20genomic%20analysis%20and%20novel%20cross-genera%20marker%20identification%20in%20lupin%20and%20pasture%20legume%20species&amp;author=P.%20Moolhuijzen&amp;author=M.%20Cakir&amp;author=A.%20Macgregor&amp;publication_year=2006&amp;journal=Genome&amp;volume=49&amp;pages=689-99" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/G06-009" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2FG06-009" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2FG06-009"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16936848" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ001C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ001C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nelson</div> <div class="given-names">M.N.</div></div>,  <div class="name"><div class="surname">Lydiate</div> <div class="given-names">D.</div></div>. </div><div class="article-title">New evidence from <em>Sinapis alba</em> L. for ancestral triplication in a crucifer genome</div>, <div class="source">Genome</div> , <div class="year">2006</div>, vol. <div class="volume">49</div> (pg. <div class="fpage">230</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=New%20evidence%20from%20Sinapis%20alba%20L.%20for%20ancestral%20triplication%20in%20a%20crucifer%20genome&amp;author=M.N.%20Nelson&amp;author=D.%20Lydiate&amp;publication_year=2006&amp;journal=Genome&amp;volume=49&amp;pages=230-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/G05-099" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2FG05-099" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2FG05-099"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16604105" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ001C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ001C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bertioli</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Moretzsohn</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Madsen</div> <div class="given-names">L.</div></div>, et al. </div><div class="article-title">An analysis of synteny of <em>Arachis</em> with <em>Lotus</em> and <em>Medicago</em> sheds new light on the structure, stability and evolution of legume genomes</div>, <div class="source">BMC Genomics</div> , <div class="year">2009</div>, vol. <div class="volume">10</div> pg. <div class="fpage">45</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20analysis%20of%20synteny%20of%20Arachis%20with%20Lotus%20and%20Medicago%20sheds%20new%20light%20on%20the%20structure%2C%20stability%20and%20evolution%20of%20legume%20genomes&amp;author=D.%20Bertioli&amp;author=M.%20Moretzsohn&amp;author=L.%20Madsen&amp;publication_year=2009&amp;journal=BMC%20Genomics&amp;volume=10&amp;pages=45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-10-45" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-10-45" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-10-45"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19166586" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ001C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ001C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lysak</div> <div class="given-names">M.A.</div></div>,  <div class="name"><div class="surname">Berr</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Pecinka</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Schmidt</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">McBreen</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Schubert</div> <div class="given-names">I.</div></div>. </div><div class="article-title">Mechanisms of chromosome number reduction in <em>Arabidopsis thaliana</em> and related Brassicaceae species</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2006</div>, vol. <div class="volume">103</div> (pg. <div class="fpage">5224</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mechanisms%20of%20chromosome%20number%20reduction%20in%20Arabidopsis%20thaliana%20and%20related%20Brassicaceae%20species&amp;author=M.A.%20Lysak&amp;author=A.%20Berr&amp;author=A.%20Pecinka&amp;author=R.%20Schmidt&amp;author=K.%20McBreen&amp;author=I.%20Schubert&amp;publication_year=2006&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=103&amp;pages=5224-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0510791103" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0510791103" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0510791103"> </span></div> </div></div></div></div></div></div><div content-id=""><div class="fn"><div class="footnote-content">Edited by Satoshi Tabata</div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2010. Published by Oxford University Press on behalf of Kazusa DNA Research Institute</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/2.5), which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=344357&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/2/10.1093/dnares/dsq001/2/dsq001-dsq001_supp_table_2.xls?Expires=2147483647&amp;Signature=FmQCcQYPLkVBW8YehvcFwYvgKwyAJZOLgLiqiXBNM9b53T1YfY2K6SEQacrkB0kEhkopPBtYMcAzjgaLVYuFV4nvCe9qT8fnL2hD5MTePv1CV1-D-jOVJOcGtkwAS~DouG9wjXiDxPXOZ39up2FRW-Espv0sTnZPutrFucamSts5~bWn083hQnpCzMi1Y53K0lND4iABwwfzPkUBCJQxG8jiIoRS-g6Vhy8G7JRYMSOJhTs~0vgOMV9fN8HJs0uncjat3gzrx21RH1JspkpaUdpa94cPGlm6-Jz0qvsiqprW2yPyaEw3~sk5FZyXXcorL3Q9~1Wl6WL6RkFlPdvVDQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/2/10.1093/dnares/dsq001/2/dsq001-dsq001supp.pdf?Expires=2147483647&amp;Signature=AlalutK0cN8CJlgQDvOfk9fG6aJoY~i26q08v1D4DoymFEqWnMf9GFkpw9JaD4gxCaoKEA1TVq4t6ZhXRCQyqtyQKhWs3WjB51Y~~ojvQVKtKcrO0weizcpXY5cWyycTsMlGgTR80feHEnnXlDiBJF4V8iBf5GFe8REuco4SZ8kkILFrBhG5ZUtcyloBbpmGZovNVbdu~k5BqhxMYtHtPEv9AyWPRKxODNUWWwNuLOfuHYKr426VqY-nRD9c5ojKdr55UsBJD3DypisQkj09LNvwZwl29xRt5q1ZBwN26Ljk2vOoVDwN2GU42VIOAkvg0Mr-gja7kJIpkbbXHYRRcw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">289</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000277166500003&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">48</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsq001"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="344357"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000277166500003&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f17%2f2%2f73%2f344357">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000277166500003&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (48)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F17%2F2%2F73%2F344357" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/344357'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Aligning a New Reference Genetic Map of Lupinus angustifolius with the Genome Sequence of the Model Legume, Lotus japonicus" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsq001");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f17%2f2%2f73%2f344357">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

