

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/16/5/275/421566","@type":"ScholarlyArticle","name":"Development of a Novel Output Value for Quantitative Assessment in Methylated DNA Immunoprecipitation-CpG Island Microarray Analysis","datePublished":"2009-09-18","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/16/5","@type":"PublicationIssue","issueNumber":"5","datePublished":"2009-10-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsp017","keywords":["epigenetics","CpG island microarray","5-aza-2′-deoxycytidine","methylation silencing","gastric cancer"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/16/5/275/421566","author":[{"name":"Yamashita, Satoshi","affiliation":"Carcinogenesis Division ,  National Cancer Center Research Institute ,  1-1 Tsukiji 5-chome, Chuo-ku, Tokyo 104-0045 ,  Japan","@type":"Person"},{"name":"Hosoya, Kosuke","affiliation":"Carcinogenesis Division ,  National Cancer Center Research Institute ,  1-1 Tsukiji 5-chome, Chuo-ku, Tokyo 104-0045 ,  Japan","@type":"Person"},{"name":"Gyobu, Ken","affiliation":"Carcinogenesis Division ,  National Cancer Center Research Institute ,  1-1 Tsukiji 5-chome, Chuo-ku, Tokyo 104-0045 ,  Japan","@type":"Person"},{"name":"Takeshima, Hideyuki","affiliation":"Carcinogenesis Division ,  National Cancer Center Research Institute ,  1-1 Tsukiji 5-chome, Chuo-ku, Tokyo 104-0045 ,  Japan","@type":"Person"},{"name":"Ushijima, Toshikazu","affiliation":"Carcinogenesis Division ,  National Cancer Center Research Institute ,  1-1 Tsukiji 5-chome, Chuo-ku, Tokyo 104-0045 ,  Japan","@type":"Person"}],"description":"Abstract.  In DNA methylation microarray analysis, quantitative assessment of intermediate methylation levels in samples with various global methylation levels ","pageStart":"275","pageEnd":"286","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/16/5/1/m_cover.gif?Expires=2147483647&Signature=vHjcTqPezdHoAGnGzycKsSlFc8Gej6TP70uQwBn0qFpZszwo~DxESYey6jXrFQdL3azkQleuWxsU9Gx6KgyZrhpkI-FY8mT2FNSaEQgMVQBUD5PJfR5Xh4MSBpg01w16~XqS5ONKzDqUEjOWBe6gjnlQEKg5ezp6b7HYSOApJ92p6HDMJ7E-oFo80vPo7Cj5zQocrpOPrViJaukjgYrI1SV9FCGcupU2HBYOlUaGdNIt9AncDj7Iog4iBwhVbtB5erQ~pCAdLkoVAnXEM1p3hlOv1w~m1J04HZNiAF0MD5UIZZKnBlqALRrMuf1inukdVJZ6nbB7tWN485~NoYhI0w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Development of a Novel Output Value for Quantitative Assessment in Methylated DNA Immunoprecipitation-CpG Island Microarray Analysis","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/16/5/1/m_cover.gif?Expires=2147483647&Signature=vHjcTqPezdHoAGnGzycKsSlFc8Gej6TP70uQwBn0qFpZszwo~DxESYey6jXrFQdL3azkQleuWxsU9Gx6KgyZrhpkI-FY8mT2FNSaEQgMVQBUD5PJfR5Xh4MSBpg01w16~XqS5ONKzDqUEjOWBe6gjnlQEKg5ezp6b7HYSOApJ92p6HDMJ7E-oFo80vPo7Cj5zQocrpOPrViJaukjgYrI1SV9FCGcupU2HBYOlUaGdNIt9AncDj7Iog4iBwhVbtB5erQ~pCAdLkoVAnXEM1p3hlOv1w~m1J04HZNiAF0MD5UIZZKnBlqALRrMuf1inukdVJZ6nbB7tWN485~NoYhI0w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Development of a Novel Output Value for Quantitative Assessment in Methylated DNA Immunoprecipitation-CpG Island Microarray Analysis" />
    <meta property="og:description" content="Abstract.  In DNA methylation microarray analysis, quantitative assessment of intermediate methylation levels in samples with various global methylation levels " />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/16/5/275/421566" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2009-09-18" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Yamashita, Satoshi" /><meta name="citation_author_institution" content="Carcinogenesis Division, National Cancer Center Research Institute, 1-1 Tsukiji 5-chome, Chuo-ku, Tokyo 104-0045, Japan" /><meta name="citation_author" content="Hosoya, Kosuke" /><meta name="citation_author_institution" content="Carcinogenesis Division, National Cancer Center Research Institute, 1-1 Tsukiji 5-chome, Chuo-ku, Tokyo 104-0045, Japan" /><meta name="citation_author" content="Gyobu, Ken" /><meta name="citation_author_institution" content="Carcinogenesis Division, National Cancer Center Research Institute, 1-1 Tsukiji 5-chome, Chuo-ku, Tokyo 104-0045, Japan" /><meta name="citation_author" content="Takeshima, Hideyuki" /><meta name="citation_author_institution" content="Carcinogenesis Division, National Cancer Center Research Institute, 1-1 Tsukiji 5-chome, Chuo-ku, Tokyo 104-0045, Japan" /><meta name="citation_author" content="Ushijima, Toshikazu" /><meta name="citation_author_institution" content="Carcinogenesis Division, National Cancer Center Research Institute, 1-1 Tsukiji 5-chome, Chuo-ku, Tokyo 104-0045, Japan" /><meta name="citation_title" content="Development of a Novel Output Value for Quantitative Assessment in Methylated DNA Immunoprecipitation-CpG Island Microarray Analysis" /><meta name="citation_firstpage" content="275" /><meta name="citation_lastpage" content="286" /><meta name="citation_doi" content="10.1093/dnares/dsp017" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="16" /><meta name="citation_issue" content="5" /><meta name="citation_publication_date" content="2009/10/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=The fundamental role of epigenetic events in cancer; Nat. Rev. Genet.;  citation_year=2002;  citation_volume=3;  citation_pages=415-28; " /><meta name="citation_reference" content="citation_title=Fidelity of the methylation pattern and its variation in the genome; Genome Res.;  citation_year=2003;  citation_volume=13;  citation_pages=868-74; " /><meta name="citation_reference" content="citation_title=The epigenomics of cancer; Cell;  citation_year=2007;  citation_volume=128;  citation_pages=683-92; " /><meta name="citation_reference" content="citation_title=DNA methylation and human disease; Nat. Rev. Genet.;  citation_year=2005;  citation_volume=6;  citation_pages=597-610; " /><meta name="citation_reference" content="citation_title=DNA methylation landscapes: provocative insights from epigenomics; Nat. Rev. Genet.;  citation_year=2008;  citation_volume=9;  citation_pages=465-76; " /><meta name="citation_reference" content="citation_title=Detection and interpretation of altered methylation patterns in cancer cells; Nat. Rev. Cancer;  citation_year=2005;  citation_volume=5;  citation_pages=223-31; " /><meta name="citation_reference" content="citation_title=Chromosome-wide and promoter-specific analyses identify sites of differential DNA methylation in normal and transformed human cells; Nat. Genet.;  citation_year=2005;  citation_volume=37;  citation_pages=853-62; " /><meta name="citation_reference" content="citation_title=Genome-wide high-resolution mapping and functional analysis of DNA methylation in arabidopsis; Cell;  citation_year=2006;  citation_volume=126;  citation_pages=1189-201; " /><meta name="citation_reference" content="citation_title=Evidence for an instructive mechanism of de novo methylation in cancer cells; Nat. Genet.;  citation_year=2006;  citation_volume=38;  citation_pages=149-53; " /><meta name="citation_reference" content="citation_title=Distribution, silencing potential and evolutionary impact of promoter DNA methylation in the human genome; Nat. Genet.;  citation_year=2007;  citation_volume=39;  citation_pages=457-66; " /><meta name="citation_reference" content="citation_title=Genome-wide analysis of Arabidopsis thaliana DNA methylation uncovers an interdependence between methylation and transcription; Nat. Genet.;  citation_year=2007;  citation_volume=39;  citation_pages=61-9; " /><meta name="citation_reference" content="citation_title=High-resolution mapping of DNA methylation in human genome using oligonucleotide tiling array; Hum. Genet.;  citation_year=2007;  citation_volume=120;  citation_pages=701-11; " /><meta name="citation_reference" content="citation_title=Discovery of epigenetically silenced genes by methylated DNA immunoprecipitation in colon cancer cells; Cancer Res.;  citation_year=2007;  citation_volume=67;  citation_pages=11481-6; " /><meta name="citation_reference" content="citation_title=A Bayesian deconvolution strategy for immunoprecipitation-based DNA methylome analysis; Nat. Biotechnol.;  citation_year=2008;  citation_volume=26;  citation_pages=779-85; " /><meta name="citation_reference" content="citation_title=Epithelial progeny of estrogen-exposed breast progenitor cells display a cancer-like methylome; Cancer Res.;  citation_year=2008;  citation_volume=68;  citation_pages=1786-96; " /><meta name="citation_reference" content="citation_title=MEDME: an experimental and analytical methodology for the estimation of DNA methylation levels based on microarray derived MeDIP-enrichment; Genome Res.;  citation_year=2008;  citation_volume=18;  citation_pages=1652-9; " /><meta name="citation_reference" content="citation_title=Developmental programming of CpG island methylation profiles in the human genome; Nat. Struct. Mol. Biol.;  citation_year=2009;  citation_volume=16;  citation_pages=564-71; " /><meta name="citation_reference" content="citation_title=ChIP-chip: considerations for the design, analysis, and application of genome-wide chromatin immunoprecipitation experiments; Genomics;  citation_year=2004;  citation_volume=83;  citation_pages=349-60; " /><meta name="citation_reference" content="citation_title=Chemical genomic screening for methylation-silenced genes in gastric cancer cell lines using 5-aza-2′-deoxycytidine treatment and oligonucleotide microarray; Cancer Sci.;  citation_year=2006;  citation_volume=97;  citation_pages=64-71; " /><meta name="citation_reference" content="citation_title=The presence of a methylation fingerprint of Helicobacter pylori infection in human gastric mucosae; Int. J. Cancer;  citation_year=2009;  citation_volume=124;  citation_pages=905-10; " /><meta name="citation_reference" content="citation_title=Transcriptional regulatory networks in Saccharomyces cerevisiae; Science;  citation_year=2002;  citation_volume=298;  citation_pages=799-804; " /><meta name="citation_reference" content="citation_title=Genome-wide map of nucleosome acetylation and methylation in yeast; Cell;  citation_year=2005;  citation_volume=122;  citation_pages=517-27; " /><meta name="citation_reference" content="citation_title=Methylation of polycomb target genes in intestinal cancer is mediated by inflammation; Cancer Res.;  citation_year=2008;  citation_volume=68;  citation_pages=10280-9; " /><meta name="citation_reference" content="citation_title=Identification of thyroid hormone receptor binding sites and target genes using ChIP-on-chip in developing mouse cerebellum; PLoS One;  citation_year=2009;  citation_volume=4;  citation_pages=e4610" /><meta name="citation_reference" content="citation_title=Genome-wide profiling of histone h3 lysine 4 and lysine 27 trimethylation reveals an epigenetic signature in prostate carcinogenesis; PLoS One;  citation_year=2009;  citation_volume=4;  citation_pages=e4687" /><meta name="citation_reference" content="citation_title=Larger numbers of silenced genes in cancer cell lines with increased de novo methylation of scattered CpG sites; Cancer Lett.;  citation_year=2007;  citation_volume=249;  citation_pages=178-87; " /><meta name="citation_reference" content="citation_title=Identification of silencing of nine genes in human gastric cancers; Cancer Res.;  citation_year=2002;  citation_volume=62;  citation_pages=6645-50; " /><meta name="citation_reference" content="citation_title=Methylation-associated silencing of heparan sulfate d-glucosaminyl 3-O-sulfotransferase-2 (3-OST-2) in human breast, colon, lung and pancreatic cancers; Oncogene.;  citation_year=2003;  citation_volume=22;  citation_pages=274-80; " /><meta name="citation_reference" content="citation_title=Identification of genes targeted by CpG island methylator phenotype in neuroblastomas, and their possible integrative involvement in poor prognosis; Oncology;  citation_year=2008;  citation_volume=74;  citation_pages=50-60; " /><meta name="citation_reference" content="citation_title=Methylation silencing of transforming growth factor-beta receptor type II in rat prostate cancers; Cancer Res.;  citation_year=2008;  citation_volume=68;  citation_pages=2112-21; " /><meta name="citation_reference" content="citation_title=DNA methylation profiling of human chromosomes 6, 20 and 22; Nat. Genet.;  citation_year=2006;  citation_volume=38;  citation_pages=1378-85; " /><meta name="citation_reference" content="citation_title=Microarray analysis of promoter methylation in lung cancers; J. Hum. Genet.;  citation_year=2006;  citation_volume=51;  citation_pages=368-74; " /><meta name="citation_reference" content="citation_title=Regulation of mammalian gene expression by retroelements and non-coding tandem repeats; Bioessays;  citation_year=2008;  citation_volume=30;  citation_pages=338-48; " /><meta name="citation_reference" content="citation_title=Gene body-specific methylation on the active X chromosome; Science;  citation_year=2007;  citation_volume=315;  citation_pages=1141-3; " /><meta name="citation_reference" content="citation_title=Identification of DNA methylation in 3′ genomic regions that are associated with upregulation of gene expression in colorectal cancer; Epigenetics;  citation_year=2007;  citation_volume=2;  citation_pages=161-72; " /><meta name="citation_reference" content="citation_title=Targeted and genome-scale strategies reveal gene-body methylation signatures in human cells; Nat. Biotechnol.;  citation_year=2009;  citation_volume=27;  citation_pages=361-8; " /><meta name="citation_reference" content="citation_title=A human B cell methylome at 100-base pair resolution; Proc. Natl Acad. Sci. USA;  citation_year=2009;  citation_volume=106;  citation_pages=671-8; " /><meta name="citation_reference" content="citation_title=Hypermethylation trigger of the glutathione-S-transferase gene (GSTP1) in prostate cancer cells; Oncogene;  citation_year=2002;  citation_volume=21;  citation_pages=1048-61; " /><meta name="citation_reference" content="citation_title=Promoter-dependent mechanism leading to selective hypomethylation within the 5′ region of gene MAGE-A1 in tumor cells; Mol. Cell Biol.;  citation_year=2004;  citation_volume=24;  citation_pages=4781-90; " /><meta name="citation_reference" content="citation_title=The human colon cancer methylome shows similar hypo- and hypermethylation at conserved tissue-specific CpG island shores; Nat. Genet.;  citation_year=2009;  citation_volume=41;  citation_pages=178-86; " /><meta name="citation_reference" content="citation_title=MIRA-assisted microarray analysis, a new technology for the determination of DNA methylation patterns, identifies frequent methylation of homeodomain-containing genes in lung cancer cells; Cancer Res.;  citation_year=2006;  citation_volume=66;  citation_pages=7939-47; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/16/5/275/1167270/dsp017.pdf" /><meta name="description" content="Abstract.  In DNA methylation microarray analysis, quantitative assessment of intermediate methylation levels in samples with various global methylation levels " />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/16/5/275/421566" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/16/5/275/421566" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/16/5/275/421566");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsp017';
    </script>
<title>
	Development of a Novel Output Value for Quantitative Assessment in Methylated DNA Immunoprecipitation-CpG Island Microarray Analysis | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./421566" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BcFdPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJZYW1hc2hpdGEsIFNhdG9zaGkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJDYXJjaW5vZ2VuZXNpcyBEaXZpc2lvbiwgTmF0aW9uYWwgQ2FuY2VyIENlbnRlciBSZXNlYXJjaCBJbnN0aXR1dGUsIDEtMSBUc3VraWppIDUtY2hvbWUsIENodW8ta3UsIFRva3lvIDEwNC0wMDQ1LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ikhvc295YSwgS29zdWtlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQ2FyY2lub2dlbmVzaXMgRGl2aXNpb24sIE5hdGlvbmFsIENhbmNlciBDZW50ZXIgUmVzZWFyY2ggSW5zdGl0dXRlLCAxLTEgVHN1a2lqaSA1LWNob21lLCBDaHVvLWt1LCBUb2t5byAxMDQtMDA0NSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJHeW9idSwgS2VuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQ2FyY2lub2dlbmVzaXMgRGl2aXNpb24sIE5hdGlvbmFsIENhbmNlciBDZW50ZXIgUmVzZWFyY2ggSW5zdGl0dXRlLCAxLTEgVHN1a2lqaSA1LWNob21lLCBDaHVvLWt1LCBUb2t5byAxMDQtMDA0NSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJUYWtlc2hpbWEsIEhpZGV5dWtpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQ2FyY2lub2dlbmVzaXMgRGl2aXNpb24sIE5hdGlvbmFsIENhbmNlciBDZW50ZXIgUmVzZWFyY2ggSW5zdGl0dXRlLCAxLTEgVHN1a2lqaSA1LWNob21lLCBDaHVvLWt1LCBUb2t5byAxMDQtMDA0NSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJVc2hpamltYSwgVG9zaGlrYXp1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQ2FyY2lub2dlbmVzaXMgRGl2aXNpb24sIE5hdGlvbmFsIENhbmNlciBDZW50ZXIgUmVzZWFyY2ggSW5zdGl0dXRlLCAxLTEgVHN1a2lqaSA1LWNob21lLCBDaHVvLWt1LCBUb2t5byAxMDQtMDA0NSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IkRldmVsb3BtZW50IG9mIGEgTm92ZWwgT3V0cHV0IFZhbHVlIGZvciBRdWFudGl0YXRpdmUgQXNzZXNzbWVudCBpbiBNZXRoeWxhdGVkIEROQSBJbW11bm9wcmVjaXBpdGF0aW9uLUNwRyBJc2xhbmQgTWljcm9hcnJheSBBbmFseXNpcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjI3NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iMjg2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHNwMDE3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjE2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSI1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMDkvMTAvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZnVuZGFtZW50YWwgcm9sZSBvZiBlcGlnZW5ldGljIGV2ZW50cyBpbiBjYW5jZXI7IE5hdC4gUmV2LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MzsgIGNpdGF0aW9uX3BhZ2VzPTQxNS0yODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RmlkZWxpdHkgb2YgdGhlIG1ldGh5bGF0aW9uIHBhdHRlcm4gYW5kIGl0cyB2YXJpYXRpb24gaW4gdGhlIGdlbm9tZTsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz04NjgtNzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBlcGlnZW5vbWljcyBvZiBjYW5jZXI7IENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTI4OyAgY2l0YXRpb25fcGFnZXM9NjgzLTkyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1ETkEgbWV0aHlsYXRpb24gYW5kIGh1bWFuIGRpc2Vhc2U7IE5hdC4gUmV2LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTU5Ny02MTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUROQSBtZXRoeWxhdGlvbiBsYW5kc2NhcGVzOiBwcm92b2NhdGl2ZSBpbnNpZ2h0cyBmcm9tIGVwaWdlbm9taWNzOyBOYXQuIFJldi4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz00NjUtNzY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldGVjdGlvbiBhbmQgaW50ZXJwcmV0YXRpb24gb2YgYWx0ZXJlZCBtZXRoeWxhdGlvbiBwYXR0ZXJucyBpbiBjYW5jZXIgY2VsbHM7IE5hdC4gUmV2LiBDYW5jZXI7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPTIyMy0zMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2hyb21vc29tZS13aWRlIGFuZCBwcm9tb3Rlci1zcGVjaWZpYyBhbmFseXNlcyBpZGVudGlmeSBzaXRlcyBvZiBkaWZmZXJlbnRpYWwgRE5BIG1ldGh5bGF0aW9uIGluIG5vcm1hbCBhbmQgdHJhbnNmb3JtZWQgaHVtYW4gY2VsbHM7IE5hdC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTM3OyAgY2l0YXRpb25fcGFnZXM9ODUzLTYyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBoaWdoLXJlc29sdXRpb24gbWFwcGluZyBhbmQgZnVuY3Rpb25hbCBhbmFseXNpcyBvZiBETkEgbWV0aHlsYXRpb24gaW4gYXJhYmlkb3BzaXM7IENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MTI2OyAgY2l0YXRpb25fcGFnZXM9MTE4OS0yMDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2aWRlbmNlIGZvciBhbiBpbnN0cnVjdGl2ZSBtZWNoYW5pc20gb2YgZGUgbm92byBtZXRoeWxhdGlvbiBpbiBjYW5jZXIgY2VsbHM7IE5hdC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTM4OyAgY2l0YXRpb25fcGFnZXM9MTQ5LTUzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaXN0cmlidXRpb24sIHNpbGVuY2luZyBwb3RlbnRpYWwgYW5kIGV2b2x1dGlvbmFyeSBpbXBhY3Qgb2YgcHJvbW90ZXIgRE5BIG1ldGh5bGF0aW9uIGluIHRoZSBodW1hbiBnZW5vbWU7IE5hdC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTM5OyAgY2l0YXRpb25fcGFnZXM9NDU3LTY2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBhbmFseXNpcyBvZiBBcmFiaWRvcHNpcyB0aGFsaWFuYSBETkEgbWV0aHlsYXRpb24gdW5jb3ZlcnMgYW4gaW50ZXJkZXBlbmRlbmNlIGJldHdlZW4gbWV0aHlsYXRpb24gYW5kIHRyYW5zY3JpcHRpb247IE5hdC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTM5OyAgY2l0YXRpb25fcGFnZXM9NjEtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGlnaC1yZXNvbHV0aW9uIG1hcHBpbmcgb2YgRE5BIG1ldGh5bGF0aW9uIGluIGh1bWFuIGdlbm9tZSB1c2luZyBvbGlnb251Y2xlb3RpZGUgdGlsaW5nIGFycmF5OyBIdW0uIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0xMjA7ICBjaXRhdGlvbl9wYWdlcz03MDEtMTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpc2NvdmVyeSBvZiBlcGlnZW5ldGljYWxseSBzaWxlbmNlZCBnZW5lcyBieSBtZXRoeWxhdGVkIEROQSBpbW11bm9wcmVjaXBpdGF0aW9uIGluIGNvbG9uIGNhbmNlciBjZWxsczsgQ2FuY2VyIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9Njc7ICBjaXRhdGlvbl9wYWdlcz0xMTQ4MS02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIEJheWVzaWFuIGRlY29udm9sdXRpb24gc3RyYXRlZ3kgZm9yIGltbXVub3ByZWNpcGl0YXRpb24tYmFzZWQgRE5BIG1ldGh5bG9tZSBhbmFseXNpczsgTmF0LiBCaW90ZWNobm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0yNjsgIGNpdGF0aW9uX3BhZ2VzPTc3OS04NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXBpdGhlbGlhbCBwcm9nZW55IG9mIGVzdHJvZ2VuLWV4cG9zZWQgYnJlYXN0IHByb2dlbml0b3IgY2VsbHMgZGlzcGxheSBhIGNhbmNlci1saWtlIG1ldGh5bG9tZTsgQ2FuY2VyIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9Njg7ICBjaXRhdGlvbl9wYWdlcz0xNzg2LTk2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NRURNRTogYW4gZXhwZXJpbWVudGFsIGFuZCBhbmFseXRpY2FsIG1ldGhvZG9sb2d5IGZvciB0aGUgZXN0aW1hdGlvbiBvZiBETkEgbWV0aHlsYXRpb24gbGV2ZWxzIGJhc2VkIG9uIG1pY3JvYXJyYXkgZGVyaXZlZCBNZURJUC1lbnJpY2htZW50OyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xODsgIGNpdGF0aW9uX3BhZ2VzPTE2NTItOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGV2ZWxvcG1lbnRhbCBwcm9ncmFtbWluZyBvZiBDcEcgaXNsYW5kIG1ldGh5bGF0aW9uIHByb2ZpbGVzIGluIHRoZSBodW1hbiBnZW5vbWU7IE5hdC4gU3RydWN0LiBNb2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9NTY0LTcxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaElQLWNoaXA6IGNvbnNpZGVyYXRpb25zIGZvciB0aGUgZGVzaWduLCBhbmFseXNpcywgYW5kIGFwcGxpY2F0aW9uIG9mIGdlbm9tZS13aWRlIGNocm9tYXRpbiBpbW11bm9wcmVjaXBpdGF0aW9uIGV4cGVyaW1lbnRzOyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT04MzsgIGNpdGF0aW9uX3BhZ2VzPTM0OS02MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2hlbWljYWwgZ2Vub21pYyBzY3JlZW5pbmcgZm9yIG1ldGh5bGF0aW9uLXNpbGVuY2VkIGdlbmVzIGluIGdhc3RyaWMgY2FuY2VyIGNlbGwgbGluZXMgdXNpbmcgNS1hemEtMuKAsi1kZW94eWN5dGlkaW5lIHRyZWF0bWVudCBhbmQgb2xpZ29udWNsZW90aWRlIG1pY3JvYXJyYXk7IENhbmNlciBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTk3OyAgY2l0YXRpb25fcGFnZXM9NjQtNzE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBwcmVzZW5jZSBvZiBhIG1ldGh5bGF0aW9uIGZpbmdlcnByaW50IG9mIEhlbGljb2JhY3RlciBweWxvcmkgaW5mZWN0aW9uIGluIGh1bWFuIGdhc3RyaWMgbXVjb3NhZTsgSW50LiBKLiBDYW5jZXI7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTI0OyAgY2l0YXRpb25fcGFnZXM9OTA1LTEwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2NyaXB0aW9uYWwgcmVndWxhdG9yeSBuZXR3b3JrcyBpbiBTYWNjaGFyb215Y2VzIGNlcmV2aXNpYWU7IFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9Mjk4OyAgY2l0YXRpb25fcGFnZXM9Nzk5LTgwNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lLXdpZGUgbWFwIG9mIG51Y2xlb3NvbWUgYWNldHlsYXRpb24gYW5kIG1ldGh5bGF0aW9uIGluIHllYXN0OyBDZWxsOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTEyMjsgIGNpdGF0aW9uX3BhZ2VzPTUxNy0yNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWV0aHlsYXRpb24gb2YgcG9seWNvbWIgdGFyZ2V0IGdlbmVzIGluIGludGVzdGluYWwgY2FuY2VyIGlzIG1lZGlhdGVkIGJ5IGluZmxhbW1hdGlvbjsgQ2FuY2VyIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9Njg7ICBjaXRhdGlvbl9wYWdlcz0xMDI4MC05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JZGVudGlmaWNhdGlvbiBvZiB0aHlyb2lkIGhvcm1vbmUgcmVjZXB0b3IgYmluZGluZyBzaXRlcyBhbmQgdGFyZ2V0IGdlbmVzIHVzaW5nIENoSVAtb24tY2hpcCBpbiBkZXZlbG9waW5nIG1vdXNlIGNlcmViZWxsdW07IFBMb1MgT25lOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTQ7ICBjaXRhdGlvbl9wYWdlcz1lNDYxMCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIHByb2ZpbGluZyBvZiBoaXN0b25lIGgzIGx5c2luZSA0IGFuZCBseXNpbmUgMjcgdHJpbWV0aHlsYXRpb24gcmV2ZWFscyBhbiBlcGlnZW5ldGljIHNpZ25hdHVyZSBpbiBwcm9zdGF0ZSBjYXJjaW5vZ2VuZXNpczsgUExvUyBPbmU7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPWU0Njg3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TGFyZ2VyIG51bWJlcnMgb2Ygc2lsZW5jZWQgZ2VuZXMgaW4gY2FuY2VyIGNlbGwgbGluZXMgd2l0aCBpbmNyZWFzZWQgZGUgbm92byBtZXRoeWxhdGlvbiBvZiBzY2F0dGVyZWQgQ3BHIHNpdGVzOyBDYW5jZXIgTGV0dC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MjQ5OyAgY2l0YXRpb25fcGFnZXM9MTc4LTg3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JZGVudGlmaWNhdGlvbiBvZiBzaWxlbmNpbmcgb2YgbmluZSBnZW5lcyBpbiBodW1hbiBnYXN0cmljIGNhbmNlcnM7IENhbmNlciBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTYyOyAgY2l0YXRpb25fcGFnZXM9NjY0NS01MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWV0aHlsYXRpb24tYXNzb2NpYXRlZCBzaWxlbmNpbmcgb2YgaGVwYXJhbiBzdWxmYXRlIGQtZ2x1Y29zYW1pbnlsIDMtTy1zdWxmb3RyYW5zZmVyYXNlLTIgKDMtT1NULTIpIGluIGh1bWFuIGJyZWFzdCwgY29sb24sIGx1bmcgYW5kIHBhbmNyZWF0aWMgY2FuY2VyczsgT25jb2dlbmUuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTIyOyAgY2l0YXRpb25fcGFnZXM9Mjc0LTgwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JZGVudGlmaWNhdGlvbiBvZiBnZW5lcyB0YXJnZXRlZCBieSBDcEcgaXNsYW5kIG1ldGh5bGF0b3IgcGhlbm90eXBlIGluIG5ldXJvYmxhc3RvbWFzLCBhbmQgdGhlaXIgcG9zc2libGUgaW50ZWdyYXRpdmUgaW52b2x2ZW1lbnQgaW4gcG9vciBwcm9nbm9zaXM7IE9uY29sb2d5OyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTc0OyAgY2l0YXRpb25fcGFnZXM9NTAtNjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1ldGh5bGF0aW9uIHNpbGVuY2luZyBvZiB0cmFuc2Zvcm1pbmcgZ3Jvd3RoIGZhY3Rvci1iZXRhIHJlY2VwdG9yIHR5cGUgSUkgaW4gcmF0IHByb3N0YXRlIGNhbmNlcnM7IENhbmNlciBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTY4OyAgY2l0YXRpb25fcGFnZXM9MjExMi0yMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RE5BIG1ldGh5bGF0aW9uIHByb2ZpbGluZyBvZiBodW1hbiBjaHJvbW9zb21lcyA2LCAyMCBhbmQgMjI7IE5hdC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTM4OyAgY2l0YXRpb25fcGFnZXM9MTM3OC04NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWljcm9hcnJheSBhbmFseXNpcyBvZiBwcm9tb3RlciBtZXRoeWxhdGlvbiBpbiBsdW5nIGNhbmNlcnM7IEouIEh1bS4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTUxOyAgY2l0YXRpb25fcGFnZXM9MzY4LTc0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZWd1bGF0aW9uIG9mIG1hbW1hbGlhbiBnZW5lIGV4cHJlc3Npb24gYnkgcmV0cm9lbGVtZW50cyBhbmQgbm9uLWNvZGluZyB0YW5kZW0gcmVwZWF0czsgQmlvZXNzYXlzOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTMwOyAgY2l0YXRpb25fcGFnZXM9MzM4LTQ4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5lIGJvZHktc3BlY2lmaWMgbWV0aHlsYXRpb24gb24gdGhlIGFjdGl2ZSBYIGNocm9tb3NvbWU7IFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MzE1OyAgY2l0YXRpb25fcGFnZXM9MTE0MS0zOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JZGVudGlmaWNhdGlvbiBvZiBETkEgbWV0aHlsYXRpb24gaW4gM+KAsiBnZW5vbWljIHJlZ2lvbnMgdGhhdCBhcmUgYXNzb2NpYXRlZCB3aXRoIHVwcmVndWxhdGlvbiBvZiBnZW5lIGV4cHJlc3Npb24gaW4gY29sb3JlY3RhbCBjYW5jZXI7IEVwaWdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTI7ICBjaXRhdGlvbl9wYWdlcz0xNjEtNzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRhcmdldGVkIGFuZCBnZW5vbWUtc2NhbGUgc3RyYXRlZ2llcyByZXZlYWwgZ2VuZS1ib2R5IG1ldGh5bGF0aW9uIHNpZ25hdHVyZXMgaW4gaHVtYW4gY2VsbHM7IE5hdC4gQmlvdGVjaG5vbC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9Mjc7ICBjaXRhdGlvbl9wYWdlcz0zNjEtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBodW1hbiBCIGNlbGwgbWV0aHlsb21lIGF0IDEwMC1iYXNlIHBhaXIgcmVzb2x1dGlvbjsgUHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDY7ICBjaXRhdGlvbl9wYWdlcz02NzEtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SHlwZXJtZXRoeWxhdGlvbiB0cmlnZ2VyIG9mIHRoZSBnbHV0YXRoaW9uZS1TLXRyYW5zZmVyYXNlIGdlbmUgKEdTVFAxKSBpbiBwcm9zdGF0ZSBjYW5jZXIgY2VsbHM7IE9uY29nZW5lOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTIxOyAgY2l0YXRpb25fcGFnZXM9MTA0OC02MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJvbW90ZXItZGVwZW5kZW50IG1lY2hhbmlzbSBsZWFkaW5nIHRvIHNlbGVjdGl2ZSBoeXBvbWV0aHlsYXRpb24gd2l0aGluIHRoZSA14oCyIHJlZ2lvbiBvZiBnZW5lIE1BR0UtQTEgaW4gdHVtb3IgY2VsbHM7IE1vbC4gQ2VsbCBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTQ3ODEtOTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBodW1hbiBjb2xvbiBjYW5jZXIgbWV0aHlsb21lIHNob3dzIHNpbWlsYXIgaHlwby0gYW5kIGh5cGVybWV0aHlsYXRpb24gYXQgY29uc2VydmVkIHRpc3N1ZS1zcGVjaWZpYyBDcEcgaXNsYW5kIHNob3JlczsgTmF0LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9NDE7ICBjaXRhdGlvbl9wYWdlcz0xNzgtODY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1JUkEtYXNzaXN0ZWQgbWljcm9hcnJheSBhbmFseXNpcywgYSBuZXcgdGVjaG5vbG9neSBmb3IgdGhlIGRldGVybWluYXRpb24gb2YgRE5BIG1ldGh5bGF0aW9uIHBhdHRlcm5zLCBpZGVudGlmaWVzIGZyZXF1ZW50IG1ldGh5bGF0aW9uIG9mIGhvbWVvZG9tYWluLWNvbnRhaW5pbmcgZ2VuZXMgaW4gbHVuZyBjYW5jZXIgY2VsbHM7IENhbmNlciBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTY2OyAgY2l0YXRpb25fcGFnZXM9NzkzOS00NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzE2LzUvMjc1LzExNjcyNzAvZHNwMDE3LnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBJbiBETkEgbWV0aHlsYXRpb24gbWljcm9hcnJheSBhbmFseXNpcywgcXVhbnRpdGF0aXZlIGFzc2Vzc21lbnQgb2YgaW50ZXJtZWRpYXRlIG1ldGh5bGF0aW9uIGxldmVscyBpbiBzYW1wbGVzIHdpdGggdmFyaW91cyBnbG9iYWwgbWV0aHlsYXRpb24gbGV2ZWxzICIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXIvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9NDIxNTY2JnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZIFTD9+H3Kmc1spzjUJLZEdDSIX2" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f16%2f5%2f275%2f421566" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f16%2f5%2f275%2f421566" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=16431" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/16/5"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/16/5/1/m_cover.gif?Expires=2147483647&amp;Signature=vHjcTqPezdHoAGnGzycKsSlFc8Gej6TP70uQwBn0qFpZszwo~DxESYey6jXrFQdL3azkQleuWxsU9Gx6KgyZrhpkI-FY8mT2FNSaEQgMVQBUD5PJfR5Xh4MSBpg01w16~XqS5ONKzDqUEjOWBe6gjnlQEKg5ezp6b7HYSOApJ92p6HDMJ7E-oFo80vPo7Cj5zQocrpOPrViJaukjgYrI1SV9FCGcupU2HBYOlUaGdNIt9AncDj7Iog4iBwhVbtB5erQ~pCAdLkoVAnXEM1p3hlOv1w~m1J04HZNiAF0MD5UIZZKnBlqALRrMuf1inukdVJZ6nbB7tWN485~NoYhI0w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/16/5">
                        <div class="volume trailing-comma">Volume 16</div>
                        <div class="issue">Issue 5</div>

            </a>

        </div>
        <div class="ii-pub-date">
October 2009        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="6247669">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6247669">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6247671">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6247671">1 Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6247679">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6247679">2 Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6247701">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6247701">3 Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6247734">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6247734">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6247736">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6247736">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6247738">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6247738">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6247740">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6247740">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/16/5/261/420676">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/16/5/287/421945">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Development of a Novel Output Value for Quantitative Assessment in Methylated DNA Immunoprecipitation-CpG Island Microarray Analysis
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Satoshi Yamashita</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Carcinogenesis Division</div>, <div class="insititution">National Cancer Center Research Institute</div>, <div class="addr-line">1-1 Tsukiji 5-chome, Chuo-ku, Tokyo 104-0045</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Satoshi+Yamashita">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yamashita S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yamashita S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Satoshi Yamashita</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kosuke Hosoya</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Carcinogenesis Division</div>, <div class="insititution">National Cancer Center Research Institute</div>, <div class="addr-line">1-1 Tsukiji 5-chome, Chuo-ku, Tokyo 104-0045</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kosuke+Hosoya">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hosoya K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hosoya K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Kosuke Hosoya</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ken Gyobu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Carcinogenesis Division</div>, <div class="insititution">National Cancer Center Research Institute</div>, <div class="addr-line">1-1 Tsukiji 5-chome, Chuo-ku, Tokyo 104-0045</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ken+Gyobu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Gyobu K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Gyobu K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Ken Gyobu</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hideyuki Takeshima</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Carcinogenesis Division</div>, <div class="insititution">National Cancer Center Research Institute</div>, <div class="addr-line">1-1 Tsukiji 5-chome, Chuo-ku, Tokyo 104-0045</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hideyuki+Takeshima">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Takeshima H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Takeshima H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hideyuki Takeshima</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Toshikazu Ushijima</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Carcinogenesis Division</div>, <div class="insititution">National Cancer Center Research Institute</div>, <div class="addr-line">1-1 Tsukiji 5-chome, Chuo-ku, Tokyo 104-0045</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81 3-3542-2511. Fax. <div class="fax">+81 3-5565-1753</div>. E-mail: <a href="mailto:tushijim@ncc.go.jp" target="_blank">tushijim@ncc.go.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Toshikazu+Ushijima">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ushijima T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ushijima T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Toshikazu Ushijima</a>
                                </span>

                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 16, Issue 5, 1 October 2009, Pages 275–286, <a href='https://doi.org/10.1093/dnares/dsp017'>https://doi.org/10.1093/dnares/dsp017</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">18 September 2009</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">10 July 2009</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">13 August 2009</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="421566" href="/dnaresearch/article-pdf/16/5/275/1167270/dsp017.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Satoshi Yamashita, Kosuke Hosoya, Ken Gyobu, Hideyuki Takeshima, Toshikazu Ushijima;  Development of a Novel Output Value for Quantitative Assessment in Methylated DNA Immunoprecipitation-CpG Island Microarray Analysis, <em>DNA Research</em>, Volume 16, Issue 5, 1 October 2009, Pages 275–286, <a href='https://doi.org/10.1093/dnares/dsp017'>https://doi.org/10.1093/dnares/dsp017</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/421566?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/421566?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/421566?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/421566?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/421566?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/421566?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/421566?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Development%20of%20a%20Novel%20Output%20Value%20for%20Quantitative%20Assessment%20in%20Methylated%20DNA%20Immunoprecipitation-CpG%20Island%20Microarray%20Analysis&amp;publicationDate=2009-09-18&amp;volumeNum=16&amp;issueNum=5&amp;author=Yamashita%2C%20Satoshi%3B%20Hosoya%2C%20Kosuke&amp;startPage=275&amp;endPage=286&amp;contentId=10.1093%2Fdnares%2Fdsp017&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f16%2f5%2f275%2f421566" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=6247669 id="6247669" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>In DNA methylation microarray analysis, quantitative assessment of intermediate methylation levels in samples with various global methylation levels is still difficult. Here, specifically for methylated DNA immunoprecipitation-CpG island (CGI) microarray analysis, we developed a new output value. The signal log ratio reflected the global methylation levels, but had only moderate linear correlation (<em>r</em> = 0.72) with the fraction of DNA molecules immunoprecipitated. By multiplying the signal log ratio using a coefficient obtained from the probability value that took account of signals in neighbouring probes, its linearity was markedly improved (<em>r</em> = 0.94). The new output value, Me value, reflected the global methylation level, had a strong correlation also with the fraction of methylated CpG sites obtained by bisulphite sequencing (<em>r</em> = 0.88), and had an accuracy of 71.8 and 83.8% in detecting completely methylated and unmethylated CGIs. Analysis of gastric cancer cell lines using the Me value showed that methylation of CGIs in promoters and gene bodies was associated with low and high, respectively, gene expression. The degree of demethylation of promoter CGIs after 5-aza-2'-deoxycytidine treatment had no association with that of induction of gene expression. The Me value was considered to be useful for analysis of intermediate methylation levels of CGIs.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="epigenetics">epigenetics</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;CpG island microarray&quot;">CpG island microarray</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="5-aza-2′-deoxycytidine">5-aza-2′-deoxycytidine</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;methylation silencing&quot;">methylation silencing</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;gastric cancer&quot;">gastric cancer</a></div>                            <h2 scrollto-destination=6247671 id="6247671" class="section-title" data-legacy-id=s1>1 Introduction</h2>
<p>DNA methylation plays a critical role during mammalian development and differentiation. Methylation of a CpG island (CGI) in a gene promoter region has been known to repress transcription of its downstream gene.<sup><a reveal-id="DSP017C1" data-open="DSP017C1" class="link link-ref link-reveal xref-bibr">1</a></sup> At the same time, DNA methylation statuses of CGIs are faithfully inherited upon cell replication,<sup><a reveal-id="DSP017C2" data-open="DSP017C2" class="link link-ref link-reveal xref-bibr">2</a></sup> and are considered to work as a stable switch of gene transcription.<sup><a reveal-id="DSP017C1" data-open="DSP017C1" class="link link-ref link-reveal xref-bibr">1</a></sup> Once a promoter CGI is aberrantly methylated, it leads to permanent aberrant silencing of its downstream gene. Aberrant DNA methylation is deeply involved in human cancers,<sup><a reveal-id="DSP017C3" data-open="DSP017C3" class="link link-ref link-reveal xref-bibr">3</a></sup> and is also likely to be involved in other human-acquired disorders.<sup><a reveal-id="DSP017C4" data-open="DSP017C4" class="link link-ref link-reveal xref-bibr">4</a></sup></p><p>There is a great interest in genome-wide analysis of DNA methylation, and new technologies involving microarrays and next-generation sequencers are being developed,<sup><a reveal-id="DSP017C5" data-open="DSP017C5" class="link link-ref link-reveal xref-bibr">5</a></sup> replacing traditional techniques.<sup><a reveal-id="DSP017C6" data-open="DSP017C6" class="link link-ref link-reveal xref-bibr">6</a></sup> In comparison with techniques using next-generation sequencers, microarray techniques are cost-effective and do not need complex bioinformatics. Their hybridization probes can be prepared by bisulphite modification of unmethylated cytosines, use of methylation-sensitive restriction enzymes, and affinity purification. The affinity purification can be performed by an antibody against 5-methylcytidine or by methylated DNA binding domains (MBDs). It has an advantage over the use of restriction enzymes, since genomic regions analysed by affinity purification are not limited to restriction sites of methylation-sensitive enzymes.</p><p>Methylated DNA immunoprecipitation (MeDIP)-microarray analysis has been used to obtain a high-resolution whole-genome DNA methylation profile of various genomes.<sup><a reveal-id="DSP017C7  DSP017C17" data-open="DSP017C7  DSP017C17" class="link link-ref link-reveal xref-bibr">7–17</a></sup> However, quantitative assessment of intermediate methylation levels has been hampered by the difficulty in appropriate normalization. Methylation levels have a unique distribution pattern that is essentially different from gene transcription and is likely to be bimodal.<sup><a reveal-id="DSP017C17" data-open="DSP017C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DSP017C18" data-open="DSP017C18" class="link link-ref link-reveal xref-bibr">18</a></sup> Also, global methylation levels in different samples are highly variable, and there are few reference genes that have consistent methylation levels across various samples. To overcome these issues, two methods (Batman and MEDME) were recently developed.<sup><a reveal-id="DSP017C14" data-open="DSP017C14" class="link link-ref link-reveal xref-bibr">14</a>,<a reveal-id="DSP017C16" data-open="DSP017C16" class="link link-ref link-reveal xref-bibr">16</a></sup></p><p>Batman (Bayesian tool for methylation analysis) transforms a signal log ratio of an individual probe to a value of methylation level taking account of the methylation levels of nearby CpG sites using standard Bayesian techniques. It is capable of processing data obtained by microarray and by next-generation sequencers. The method was validated by bisulphite sequencing of sperm samples,<sup><a reveal-id="DSP017C14" data-open="DSP017C14" class="link link-ref link-reveal xref-bibr">14</a></sup> and its validity in samples with different global methylation levels remains to be established. MEDME (modelling experimental data with MeDIP enrichment) weighs signal log ratios of individual probes using a logistic model and signals obtained by neighbouring probes and by using completely methylated DNA samples.<sup><a reveal-id="DSP017C16" data-open="DSP017C16" class="link link-ref link-reveal xref-bibr">16</a></sup> Both Batman and MEDME had good correlation with methylation levels obtained by bisulphite sequencing (<em>R</em><sup>2</sup> = 0.82 and 0.75, respectively). Also, both are capable of processing data from both CpG-rich and -poor regions, and this made their conversion algorithm complex as a trade-off.</p><p>In this study, we developed a novel output value, the ‘Me value’, that can be calculated from raw output values, and confirmed that the value had a linear correlation with methylation levels of genomic regions using samples with various global methylation levels. The Me value was used to clarify how methylation of CGIs in various positions against transcription start sites (TSSs) is associated with gene expression, and how demethylation of CGIs is associated with re-expression of genes.</p>                            <h2 scrollto-destination=6247679 id="6247679" class="section-title" data-legacy-id=s2>2 Materials and methods</h2>
                            <h3 scrollto-destination=6247680 id="6247680" class="section-title" data-legacy-id=s2a>2.1 Cell lines and tissue samples</h3>
<p>AGS and KATOIII gastric cancer cell lines were obtained from the American Type Culture Collection (Manassas, VA, USA) and the Japanese Collection of Research Bioresources (Tokyo, Japan), respectively. HSC39 and HSC57 gastric cancer cell lines were gifted by Dr K. Yanagihara, National Cancer Center Research Institute, Tokyo, Japan. Treatment with a demethylating agent, 5-aza-2'-deoxycytidine (5-aza-dC, Sigma, St Louis, MO, USA), was performed as in our previous study (AGS, 3 days, 1 µM).<sup><a reveal-id="DSP017C19" data-open="DSP017C19" class="link link-ref link-reveal xref-bibr">19</a></sup> A normal gastric tissue sample was prepared by pooling endoscopic biopsy specimens from three healthy volunteers with informed consents. High-molecular-weight DNA was extracted by the phenol/chloroform method with RNase A treatment.</p>                            <h3 scrollto-destination=6247683 id="6247683" class="section-title" data-legacy-id=s2b>2.2 MeDIP and quantification of the number of immunoprecipitated DNA molecules</h3>
<p>Five micrograms of genomic DNA were sonicated by a VP-5s homogenizer (TAITEC, Saitama, Japan) to fragment lengths between 200 and 800 bp. The mode of fragment length was about 300 bp. After heat denaturation at 95°C for 10 min, DNA was incubated with 5 µg antibody against 5-methylcytidine (Diagnode, Liége, Belgium) in 1× IP buffer [10 mM Na-phosphate, pH 7.0, 140 mM NaCl, 0.05% (w/v) Triton X-100] at 4°C overnight. Immune complexes were collected with Dynabeads Protein A (Invitrogen Dynal AS, Oslo, Norway), washed with 1× IP buffer four times, treated with Proteinase K, and purified by phenol and chloroform extraction and isopropanol precipitation.</p><p>To assess the fraction of immunoprecipitated (IP) DNA molecules among that of the total DNA (whole cell extract DNA, WCE) molecules, the number of IP and WCE molecules was quantified by real-time PCR using SYBR® Green I (BioWhittaker Molecular Applications, Rockland, ME, USA) and an iCycler Thermal Cycler (Bio-Rad Laboratories, Hercules, CA, USA) as described previously.<sup><a reveal-id="DSP017C20" data-open="DSP017C20" class="link link-ref link-reveal xref-bibr">20</a></sup> All primers used in this study are listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>.</p>                            <h3 scrollto-destination=6247686 id="6247686" class="section-title" data-legacy-id=s2c>2.3 CGI microarray analysis</h3>
<p>Methylation microarray analysis was carried out using a human CGI oligonucleotide microarray (Agilent Technologies, Santa Clara, CA, USA) that contained 237 220 probes in or within 95 bp either side of a CGI and covered 27 800 CGIs with an average probe spacing of 100 bp. IP from 4.5 µg of sonicated DNA and 1.0 µg of WCE, without any amplification, were labelled with Cy5 and Cy3, respectively, using an Agilent Genomic DNA Labeling Kit PLUS (Agilent Technologies). Labelled DNA was hybridized to the microarray at 67°C for 40 h with constant rotation (20 rpm), and then scanned with an Agilent G2565BA microarray scanner (Agilent Technologies).</p><p>From the scanned data, signal values of the IP and WCE were obtained using Feature Extraction Ver.9.1 (Agilent Technologies). These two signal values were normalized using background subtraction, and signal ratio (IP/WCE), signal log ratio [log<sub>2</sub>(IP/WCE)], <em>P</em>[<em>X</em>], and <em>P</em><span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp017/2/m_dsp017IL0001.gif?Expires=2147483647&amp;Signature=rOFH6gTvww7vvZPy2ARciI1oNgK22kkUK6HX7i6ZJfHW4jYKE~XqMmrCp1tLDnQHGH4CmIvO8wOg1GeWQIA-tLNlpeg3tDq63lEqtAskMTAp50PrPnN3ZqvjJO9jtzDaHFztc8A-wtWIZH2c0gAjlE3oE18pabDAeuA-qY-KSOOxK5x4RP32zH0IowzxWRl6V6IIgTv03GB2NmEGoWs~pNR7tar~xdG6euGTP8Nlk4Nv5GUCTgkyB1ptDRPn9XFdAxT57xFrc6KfEvNLch-1ev4KWaX7HAu5RQgSesSoKjjPt0e3h1PNOhKvQPbIm4R5U3h3JtRHVv3Hx3TMc19Tug__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsp017IL0001.gif" /></span> were obtained using Agilent G4477AA ChIP Analytics 1.3 software (Agilent Technologies). The <em>P</em>[<em>X</em>] and <em>P</em><span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp017/2/m_dsp017IL0002.gif?Expires=2147483647&amp;Signature=tMa6Z~NeFMdAzk7f-Ty4xTjWC8eODcD8qZsGOP9RA6tw~Kqau-bQbpZ6q-8cdXMReZFM4qvtddJhgECCX3F9MUep99Waw9nRKrTY6uI2Mdu3aeuPvW8vJPnt5-U6XibmBjtDeyudCqMpp6~d1T8kYyoQKaQ6QqT8~4Zm77gFUxTBjM7coTbcQd5fSfW~qOcwMB4uKy5rEssshHNAAeSZZdfIrbkqWef69psm8WaYi9uHqvdOfkMysVAmQr29YUY2BokcCbjorc1AH5qlESwikF1v7tovCkBDtBHbjF3Hfkfwfh-junvdiOckJ-7t-71a11uBdAJtZsbLp8NNmkeNBA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsp017IL0002.gif" /></span> values, which are used in chromatin immunoprecipitation (ChIP)-on-chip analysis to obtain a binding call,<sup><a reveal-id="DSP017C21  DSP017C25" data-open="DSP017C21  DSP017C25" class="link link-ref link-reveal xref-bibr">21–25</a></sup> were defined as the probability how the <em>X</em> (<span class="inline-formula no-formula-id">⁠<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp017/2/m_dsp017IL0003.gif?Expires=2147483647&amp;Signature=agnF6hOhnp4UUWiKo1vcYbiwd-U-RRBeJ3IJMjLj2aCDD985qQc8Itebg9nTfneVcGQtvKppSK9SUhayL~nRg8V77ro-~iazQtCmE3c5y3MQDHDtgvrvbDzgbfWHyKFc4BgucfnfGhik7XchmDC7zl53ZsOS4y1070zJIxbceMmPbI2dbFitIxJ9p7NCf4jIyVBdulAqolgaPoCSt2Yk8gS9lYE7uT7ybFdSxBn-4s-~o2II0I8dWbdLLj3-xff2Fxp5A8Ot0175MKtAinOS7NCWDWyzq2rNSNPqS~-4ge8YCOWA0AthtnDnnFuOV~xkXU8doIuBK3j5KP~S5hqDhQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsp017IL0003.gif" />⁠</span>) value deviates from Gaussian distribution of <em>X</em> (<span class="inline-formula no-formula-id">⁠<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp017/2/m_dsp017IL0004.gif?Expires=2147483647&amp;Signature=fjH-PmRS3BwcztRBz2wSWLGni~0LYJvYKdfeIge3bgiqdsllE052-YA8~yQNMFX5jiX4A3RAGr09DXaAt7B8jzW2X1RbYNSFCFiTXXnAMWnlcrv0nfgiyTwDrHWy58rBBgE7vsHDYFMgJ3Oj5I2CuMrbaSkOeMSUwU0gpufvT6RqcrdJUooEZ-UkjFdRGNVtPwgsJ9VCdoJzeWvIb6NhszX0xbu-FdLiXpbJsALGtS6oa~Y2I9mxSO3DC~DyQk56cMOioiJ7wfiQsLoMDmwflD3azaRIqtGUVW2ejPys-r4VfvNynB1TeSObcCibmSMkItfxBWxIeSAeCwVERPG8Gw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsp017IL0004.gif" />⁠</span>) values of the entire genome of a sample. Here, the <em>X</em> value for a probe was obtained as the difference between the IP and the WCE signals after adjusting the symmetry of its distribution. The <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp017/2/m_dsp017IL0005.gif?Expires=2147483647&amp;Signature=uNc5ca3vyGUQiq29YNlR1BhMIbLC4VhXDF14Eio5TeqIUQ5fjPYXuJdw67jOd9LvkEsFkDjPPmRF46XIlM5PMmCrSDUUzrpRwGsoGbmfs-yHjfm94y6AbvaojRZfJ9Qfl2yB0g3crWqN9rkdhWJpXha9O2KHv~scWHL~KCdOFrMRrKwyjperJX6AWTxw4f8q9RAIFrKi-WfZImk30qOkadRYX3ajuOAqUVuHpH4loyynqJymhnzlec-6YYEdRnNcfJXj00rVfP5DMp-y-Un3JQ5qMaKfH1ZPrr0fRsfc2PIVZs6MW4NnHp9Hc6xb3xo03VhJtQXWdKkKG3nqinwqkg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsp017IL0005.gif" /></span> value for a probe was calculated as an average <em>X</em>, taking account of signals for neighbouring probes (within 1 kb of the probe). In addition, to calculate signal log ratio in experiments specifically referred to, the two signal values were also normalized by the Median and the Lowess normalization methods. The microarray results were submitted to the GEO database (GSE15291).</p>                            <h3 scrollto-destination=6247689 id="6247689" class="section-title" data-legacy-id=s2d>2.4 Calculation of the Me value</h3>
<p>The Me value of each probe (site Me value) was calculated as Me value = [signal log ratio × (1 − <em>P</em><span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp017/2/m_dsp017IL0006.gif?Expires=2147483647&amp;Signature=0K~4va9yPWvs-scGKygV66f3CD0Z1tx31kDl30nUtjuh66mW22w6-pmtsqfLxevOLp944zYXd~uBfuo~w3qjSjKxF1x~Os32WnpmdI62kTGuU5QVkt-Vne2DH3ovqeqaJBRWiBUGQ0H~lpGrWfPsdwRA3cqUQagfzXIWwyK7rK5ixjn4jE1FLYj5wZAAhQJnecdcuH1UYEUJFMjWu5ELZBgLHT0e7RLBP8Kt~enMUGIqUt0DbvUvot0jOvkgE9PnHsSyEmMXLjRVTlRBcCDH7YYwGnrhtHpr74gEVMw45YlklHMXtu-wT5Nhm-GqhF5IjCwqx7~juLrxN43BE9WuTg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsp017IL0006.gif" />⁠</span>) − <em>k</em>]/<em>l</em> + 0.5. The <em>P</em><span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp017/2/m_dsp017IL0007.gif?Expires=2147483647&amp;Signature=G2qlA~rX6vGJX3q3-Bw~rQDw7~YxQ-Uc~EkpxtIqnl2wPldFF6j4eCKGUYCKJtV7Wnop1Z10pGFT6YcF4ck-7Rk4Hv5Om5UQcsCzIXJs4B3c6MojjbKsy1ZFlBdwOl1pNJ-y0nqj~9C-exmnPZPoM2q7GSyM58Fyv5y7UhRKHSRWZ7K4tqP4ux5dL1te0j9nOtVkBcUV4BmPnwaX7msCkNXS0gycBZP-Q2pdVcceaACyAyfRvUbZ~utBMlyrTc3lHmQYx98sP6XlajV719coOQ~BGsOK4JY5HpeqF7ZUK0sVnLkjMKYg3h6N56nlvBorBc20c9Bc1pJCPKL5R9EWGQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsp017IL0007.gif" /></span> value and signal log ratio normalized using background subtraction were used for this formula. The [signal log ratio × (1 − <em>P</em><span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp017/2/m_dsp017IL0008.gif?Expires=2147483647&amp;Signature=LDD8ZYbMNsRgrlZtLlI0rBMzilKB6dnu0VmGUwpLuZ92pKucRY6c7z4oI~4id8MLHek30bGK-S5X3ad0qbekNHC2M8ArDFFXmeH6GxxNQjvhW7zJQ2wbRLDY3wC3qEPjJRq-E5wuu1-mbAKpOP49PltwWUzLiEA~rKeXWlz8bSr8hHLn3SSLuewEr2t7Rzas2qIAjn2E1l8BG316Tjsda850pS5W~cXoCAblNoXCtYPmTc5-l6Tksa7TPJqX-lQARGq0Z1ogexuF8Fjf881DSJrpnr27bEDXjCgYJPJe1EU1hnEw6mS31gruHw9sFEuFO3IFMZO1g9jBlA-bUmTFjQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsp017IL0008.gif" />⁠</span>)] value mostly ranged from 0 to 2.6 in this study, and in general, the distribution depends on the microarray platform. Accordingly, the constant <em>l</em> was fixed at 2.6 in this study, so that the Me value would be within a range between 0 and 1. Me values larger than 1 and those smaller than 0, which were occasionally produced after calculation, were corrected to 1 and 0, respectively. The constant <em>k</em> was calculated as [the signal log ratio of CGIs that had a 50% fraction of DNA molecules IP (1.7 in this study) – 0.4], which equalled to 1.3 in this study. The signal log ratio of CGIs with 50% methylation depends on the microarray platform, labelling method, and mixture rate of IP and WCE, but does not need to be changed once established to suit a protocol.</p><p>The Me value was calculated only for probes with high reliability. To select such probes, first, probes that yielded extremely high signal intensities (5-fold higher than average) for the WCE (Cy-3) were excluded. Since the signals obtained for the WCE should be the same theoretically for all the probes, extremely high signals were considered to be due to cross-hybridization. Then, continuity of signal log ratios of neighbouring probes was enforced. If the value of a probe was higher than those of neighbouring probes on both sides, it was corrected to their average because the value was likely to be an error. In addition, efficiency in labelling and hybridization in each microarray analysis was monitored by the signal log ratio and the fraction of DNA molecules IP by MeDIP at 10 probe loci. The data processing for the Me value was performed by Excel 2007 (Microsoft, Redmond, WA, USA), and the templates are available upon request.</p>                            <h3 scrollto-destination=6247693 id="6247693" class="section-title" data-legacy-id=s2e>2.5 Definitions of genomic regions</h3>
<p>The position of each probe against a TSS was determined using UCSC hg18 (NCBI Build 36.1, March 2006). A single CGI was defined as an assembly of probes in the CGI microarray with intervals &lt;500 bp. CGIs were classified into four categories: upstream CGIs (within 10 kb upstream of the TSS), divergent CGIs (within 10 kb upstream of the TSSs of two genes that are transcribed in opposite directions), gene body CGIs, and downstream CGIs (within 10 kb downstream of genes). A CGI spanning both an upstream region and a gene body was split into an upstream CGI and a gene body CGI. A putative promoter region (promoter) was defined as a region between a TSS, determined by UCSC hg18 (NCBI Build 36.1, March 2006) and its 200 bp upstream. According to these definitions, 34 697 assemblies of probes were defined as CGIs, and 9624 assemblies were defined as promoters. Genes with multiple promoters were analysed as different genes because of their multiple TSSs. CGIs that could not be classified by these criteria (4164 CGIs) were omitted from the following analysis. An average number of probes that covered a single CGI (or a single promoter) was 6.8 (2.0), and the distribution of the numbers is shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>.</p>                            <h3 scrollto-destination=6247695 id="6247695" class="section-title" data-legacy-id=s2f>2.6 Expression microarray analysis</h3>
<p>Microarray analysis of gene expression was performed using GeneChip (Affymetrix) as described previously,<sup><a reveal-id="DSP017C19" data-open="DSP017C19" class="link link-ref link-reveal xref-bibr">19</a>,<a reveal-id="DSP017C26" data-open="DSP017C26" class="link link-ref link-reveal xref-bibr">26</a></sup> and the signal intensities were normalized, so that the average intensity of all the genes on a microarray would be 500. The average signal intensity of all the probes for a gene was used as its expression level. Genes with signal intensities of 1000 or more and of 250 or less were defined as those with high and low expression, respectively.</p>                            <h3 scrollto-destination=6247697 id="6247697" class="section-title" data-legacy-id=s2g>2.7 Bisulphite treatment, methylation-specific PCR, and bisulphite sequencing</h3>
<p>Bisulphite modification was performed using <em>Bam</em>HI-digested genomic DNA as descried previously.<sup><a reveal-id="DSP017C26" data-open="DSP017C26" class="link link-ref link-reveal xref-bibr">26</a></sup> Methylation-specific PCR (MSP) was performed using bisulphite-treated DNA as descried previously.<sup><a reveal-id="DSP017C26" data-open="DSP017C26" class="link link-ref link-reveal xref-bibr">26</a></sup> Bisulphite sequencing was performed after cloning the PCR product (10 clones or more for each sample). We used the data of methylation status previously analysed.<sup><a reveal-id="DSP017C19" data-open="DSP017C19" class="link link-ref link-reveal xref-bibr">19</a>,<a reveal-id="DSP017C26  DSP017C30" data-open="DSP017C26  DSP017C30" class="link link-ref link-reveal xref-bibr">26–30</a></sup></p>                            <h3 scrollto-destination=6247699 id="6247699" class="section-title" data-legacy-id=s2h>2.8 Determination of methylation levels of a genomic region and CGI (or promoter) using the microarray data</h3>
<p>A methylation level of a genomic region analysed by quantitative PCR of IP DNA molecules was assessed by an output value of a probe within the PCR product and closest to the forward primer, or a probe closest to the PCR product when no probes were present within the PCR product. A methylation level of a region analysed by bisulphite sequencing (200 bp) was assessed by an output value of a probe in the centre of the region. This was possible because bisulphite sequencing was performed for a region larger than 100 bp upstream and downstream of a probe, and methylation statuses of CpG sites within the 200 bp region were scored. A methylation level of a CGI (or promoter) was assessed by an average of site Me values of the probes located within the CGI (or promoter), which was defined as the CGI (or promoter) Me value.</p>                            <h2 scrollto-destination=6247701 id="6247701" class="section-title" data-legacy-id=s3>3 Results and discussion</h2>
                            <h3 scrollto-destination=6247702 id="6247702" class="section-title" data-legacy-id=s3a>3.1 Assessment of current output values for methylation levels</h3>
<p>We first examined whether or not distribution of the available output values reflected the global methylation levels. The output values analysed were: (i) the signal log ratio, which is most frequently used in microarray analysis, (ii) the <em>P</em>[<em>X</em>] value, and (iii) <em>P</em><span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp017/2/m_dsp017IL0009.gif?Expires=2147483647&amp;Signature=Zlu1lNagHrdLTCSVQ~FgymjgnvI6d8xEdUmnq5XmlspkiV8OD--pZJajwc44LjU7QoZVGSOzq20Aw2aq59wJb8uwaX9cA80-5cZIJ-OOK5pWMLgKYAZ06BSVQ6XgV7InZ196wDYYNKx0cLc9L2L1MbZFs3WrS58km7v~uGmaFNee1IJpal9R-NVDsTcloP-vlxyyL2cCFSjpRHHntfEeHjbq3xJPLvVjdoM5cc~C~JKHYb6bc5~gEnAO8H1AHI8FX-PeTgfDQHqONU6fx-QyaqLQEy2WxgVLdsCN6r07znCtq0YgqSQ9lxTYdOnwRfDpARWgQ9MUw0wxLbd28UlvoQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsp017IL0009.gif" /></span> value, which is often used in ChIP-on-chip analysis. Their distribution was analysed in two samples, a cell line (AGS) with frequent CGI methylation<sup><a reveal-id="DSP017C26" data-open="DSP017C26" class="link link-ref link-reveal xref-bibr">26</a></sup> and the same cell line after treatment with 5-aza-dC. Our previous study showed that AGS after 5-aza-dC treatment has demethylation of at least 421 promoter CGIs,<sup><a reveal-id="DSP017C19" data-open="DSP017C19" class="link link-ref link-reveal xref-bibr">19</a></sup> and its global methylation level was expected to shift towards unmethylated ranges. Among the three values, the signal log ratio showed such a shift (Fig. <a reveal-id="DSP017F1" data-open="DSP017F1" class="link link-reveal link-table xref-fig">1</a>A). On the other hand, the <em>P</em>[X] and <em>P</em><span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp017/2/m_dsp017IL0010.gif?Expires=2147483647&amp;Signature=fwJHTm1iaASeeiNKERfXIov86X-qK699gbJc3HCPJnZN0ltuGRfPuiTyFkPQRXmx49CKlCOOMaPjmLnFJ~-JTBcvAADStnhjqG66zq5aT-jZrLh6jfxi0P-oVD1rnkdx1klmBppxuy5NhnJuuFmhR4l4PNoQFw8BVyc-Y35oyH9bRkPVV2MNK0~IQ~2xcy1JCyD2cLDN5f0xEzuoxNIJXFmF9C1dlJroxZFoOqtRJJw1GkcYn96EOsUvzoHXO3~pyJHzt2DGdS-p6gPYyq~ZfomWG1NAxFwd8JFuJyVTwNJ-4wXMaePqzajbybAqOfP4Gpg2fNx99SxzlL-395RmwQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsp017IL0010.gif" /></span> values did not show such a shift (Fig. <a reveal-id="DSP017F1" data-open="DSP017F1" class="link link-reveal link-table xref-fig">1</a>B and C).</p>                        <a id="6247704" scrollto-destination="6247704"></a>
<div id="DSP017F1" data-id="DSP017F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp017/2/m_dsp01701.jpeg?Expires=2147483647&amp;Signature=ERCo1gPlWfAdEsUug2KdTl2fmnL39uyPg~aVU2rFZo50C4jvHDd-ZEEUblygzMK3e14jrGJQ-tkOWNG5CrcbBU3kvEFiEfawJCBj2NmZ457oEUxJ6V2K6tWAXAv1m7NIPobCJsStxoodcZkI61SDTkkOusa99Rc8bqeiLZmYgB2hLvH74Q5hQ4tnWIqaX-jA-FUZbo0PU006e91licsk0YJGJPLRrFMQViJ-Enqj3LQ-wuHU1jhL~E1xmeOcsd1IiVwoeEOhSnavyd8b~afBVck-38F4ItgG6Ecr0ss9NY5aktHqni~In3BV5ix290tOYBLZYfcTp-GTnvzJTgqXkA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of methylation levels assessed by the available output values and the Me value in a gastric cancer cell line, AGS, and the same cell line after 5-aza-dC treatment. The analysis was performed for the signal log ratio with background subtraction normalization (A), the P[X] value (B), the P value (C), and the Me value (D). Lines between the two bar graphs indicate values of individual probes (randomly selected representative 20 values). Distributions of the signal log ratio and the Me value reflected the global methylation level, as shown by their shift towards smaller values in cells with 5-aza-dC treatment. In contrast, those of the P[X] and P values did not show the shift." path-from-xml="dsp01701.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6247704/dsp01701.jpeg" path-from-xml="dsp01701.jpeg" target="_blank">View large</a><a section="6247704" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp017/2/dsp01701.jpeg?Expires=2147483647&amp;Signature=gczl-GMxdd1q21nPwap3zl29mMdch7nD7q9Kmq7deMH8o1oJpgs~4Y0AEC5TlGwe4VzBJgGqSLtzJ3OPXgcxm6XBQKxK6E5zH29rsk6WWPbyq~vemiQxkD1SlhcvcT305-k677wlfeMntjPWf01ukQ6JM-DE6WuygbP6fq9VNtXY8SP1NiC6ILeP0lv0RU3AFd2od8JF~6Av0Idwg5y3QXt0CWzHQGC3mt3ChWTVNWNtcEF9upLpHqkjKyZIZb5ueM~lsJuntRPfVQ7OMsHnCD95R1En2uSGBrWZGporrXHIj3KfrKgqnr~TbOUYp0dXpyRioninsa~KeF0hwHd0hQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=6247704&amp;ar=421566&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsp01701.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of methylation levels assessed by the available output values and the Me value in a gastric cancer cell line, AGS, and the same cell line after 5-aza-dC treatment. The analysis was performed for the signal log ratio with background subtraction normalization (A), the <em>P</em>[<em>X</em>] value (B), the <em>P</em><span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp017/2/m_dsp017IL0020.gif?Expires=2147483647&amp;Signature=XXScFf8kPUkKEUpzGo1RZKFJaoEyFNE5u3h~81haTIhsn21shGfQbrsptgTVihsl2n4qLxBPoY3tBnji-9ozoymDgN0H4MMHQLy~MwdD5mHHVGNMone7OGln6Y~sxKR6hWQWNjgsQX2WzEsXaCpwHxy2r7VAuVO0mZGt2xKt6FVVR0ivhZ~t7Wo5dsNPXzqIcJYLxi3K97NTW~dpGvWUQ-BKrXrS3cra3QPFpsMAp8-4y5pGiEod8feBp7Bd~hpandfFXj07NoRMWfUjPznOoa7YDOFXoxOYqans930XIa6NpzfKzg2WDLeB9vCCf7XzMteyHu8w5siIRlbnltQDfQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsp017IL0020.gif" /></span> value (C), and the Me value (D). Lines between the two bar graphs indicate values of individual probes (randomly selected representative 20 values). Distributions of the signal log ratio and the Me value reflected the global methylation level, as shown by their shift towards smaller values in cells with 5-aza-dC treatment. In contrast, those of the <em>P</em>[<em>X</em>] and <em>P</em><span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp017/2/m_dsp017IL0021.gif?Expires=2147483647&amp;Signature=hNxVvveDbkVdEHMmg7o10px4arbjyzdF5LhiMotXmjfrr6CLH0nbNyvLT1D-6RiSTI8knFcTorxh789Soj-mhwfc~oLimeJn~xS~j9Mf-h6ilJ8biPKqmzQlJHUPGy6DtL59Bgl6tGYfRkfqD9Th4qWNXEvHwogYw7~3RFuHXSrancefyrn0P-O9-i~y-2TVwKgM5ClWXFgQVyyCjFZ~0x2fy82E0SrzaEMajw~m3bc7UosfE048AA8I4SZR8y5sfV6MVNJiS7GaR719I5YTJZ22Z9WQva7X2yiXsf01EFkpURanvl5EwE2jcmnw64YVRarCiKlcbilbdA~JB0Y~Og__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsp017IL0021.gif" /></span> values did not show the shift.</p></div></div><div content-id="DSP017F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp017/2/m_dsp01701.jpeg?Expires=2147483647&amp;Signature=ERCo1gPlWfAdEsUug2KdTl2fmnL39uyPg~aVU2rFZo50C4jvHDd-ZEEUblygzMK3e14jrGJQ-tkOWNG5CrcbBU3kvEFiEfawJCBj2NmZ457oEUxJ6V2K6tWAXAv1m7NIPobCJsStxoodcZkI61SDTkkOusa99Rc8bqeiLZmYgB2hLvH74Q5hQ4tnWIqaX-jA-FUZbo0PU006e91licsk0YJGJPLRrFMQViJ-Enqj3LQ-wuHU1jhL~E1xmeOcsd1IiVwoeEOhSnavyd8b~afBVck-38F4ItgG6Ecr0ss9NY5aktHqni~In3BV5ix290tOYBLZYfcTp-GTnvzJTgqXkA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of methylation levels assessed by the available output values and the Me value in a gastric cancer cell line, AGS, and the same cell line after 5-aza-dC treatment. The analysis was performed for the signal log ratio with background subtraction normalization (A), the P[X] value (B), the P value (C), and the Me value (D). Lines between the two bar graphs indicate values of individual probes (randomly selected representative 20 values). Distributions of the signal log ratio and the Me value reflected the global methylation level, as shown by their shift towards smaller values in cells with 5-aza-dC treatment. In contrast, those of the P[X] and P values did not show the shift." path-from-xml="dsp01701.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6247704/dsp01701.jpeg" path-from-xml="dsp01701.jpeg" target="_blank">View large</a><a section="6247704" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp017/2/dsp01701.jpeg?Expires=2147483647&amp;Signature=gczl-GMxdd1q21nPwap3zl29mMdch7nD7q9Kmq7deMH8o1oJpgs~4Y0AEC5TlGwe4VzBJgGqSLtzJ3OPXgcxm6XBQKxK6E5zH29rsk6WWPbyq~vemiQxkD1SlhcvcT305-k677wlfeMntjPWf01ukQ6JM-DE6WuygbP6fq9VNtXY8SP1NiC6ILeP0lv0RU3AFd2od8JF~6Av0Idwg5y3QXt0CWzHQGC3mt3ChWTVNWNtcEF9upLpHqkjKyZIZb5ueM~lsJuntRPfVQ7OMsHnCD95R1En2uSGBrWZGporrXHIj3KfrKgqnr~TbOUYp0dXpyRioninsa~KeF0hwHd0hQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=6247704&amp;ar=421566&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsp01701.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of methylation levels assessed by the available output values and the Me value in a gastric cancer cell line, AGS, and the same cell line after 5-aza-dC treatment. The analysis was performed for the signal log ratio with background subtraction normalization (A), the <em>P</em>[<em>X</em>] value (B), the <em>P</em><span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp017/2/m_dsp017IL0020.gif?Expires=2147483647&amp;Signature=XXScFf8kPUkKEUpzGo1RZKFJaoEyFNE5u3h~81haTIhsn21shGfQbrsptgTVihsl2n4qLxBPoY3tBnji-9ozoymDgN0H4MMHQLy~MwdD5mHHVGNMone7OGln6Y~sxKR6hWQWNjgsQX2WzEsXaCpwHxy2r7VAuVO0mZGt2xKt6FVVR0ivhZ~t7Wo5dsNPXzqIcJYLxi3K97NTW~dpGvWUQ-BKrXrS3cra3QPFpsMAp8-4y5pGiEod8feBp7Bd~hpandfFXj07NoRMWfUjPznOoa7YDOFXoxOYqans930XIa6NpzfKzg2WDLeB9vCCf7XzMteyHu8w5siIRlbnltQDfQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsp017IL0020.gif" /></span> value (C), and the Me value (D). Lines between the two bar graphs indicate values of individual probes (randomly selected representative 20 values). Distributions of the signal log ratio and the Me value reflected the global methylation level, as shown by their shift towards smaller values in cells with 5-aza-dC treatment. In contrast, those of the <em>P</em>[<em>X</em>] and <em>P</em><span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp017/2/m_dsp017IL0021.gif?Expires=2147483647&amp;Signature=hNxVvveDbkVdEHMmg7o10px4arbjyzdF5LhiMotXmjfrr6CLH0nbNyvLT1D-6RiSTI8knFcTorxh789Soj-mhwfc~oLimeJn~xS~j9Mf-h6ilJ8biPKqmzQlJHUPGy6DtL59Bgl6tGYfRkfqD9Th4qWNXEvHwogYw7~3RFuHXSrancefyrn0P-O9-i~y-2TVwKgM5ClWXFgQVyyCjFZ~0x2fy82E0SrzaEMajw~m3bc7UosfE048AA8I4SZR8y5sfV6MVNJiS7GaR719I5YTJZ22Z9WQva7X2yiXsf01EFkpURanvl5EwE2jcmnw64YVRarCiKlcbilbdA~JB0Y~Og__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsp017IL0021.gif" /></span> values did not show the shift.</p></div></div><p>Next, a linear correlation with methylation level, represented here by the fraction of DNA molecules IP by the anti-5-methylcytidine antibody, was examined for individual output values using 31 genomic regions located within various CGIs. In addition to the three output values, (iv) the signal ratio (background subtraction normalization), (v) the signal log ratio with Median normalization, and (vi) the signal log ratio with the Lowess normalization were analysed. Although <em>P</em><span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp017/2/m_dsp017IL0011.gif?Expires=2147483647&Signature=ha9b3MdGGq7QzgAOMYkexRoI746SleGZuMhbnYkk33LC6X~dOk9A6O-LM3aHIT~TgN6Y7x4DcHrKa48VIDg~7XGQNhQ8hTsHBF4jOGc3LjUKwxLcFd4Zv~3a7CbbWo887gMMTVZJrpbCXH6DMCNtrfou~eCsTJvE9AQNjU52pobX6H99kQ9DxIKb3XmEktEyOOYCR4daXAqae1b2RW6BpurEoqZVLJ4fFBlaai1BCIFZuBEpoQlJ~RGgR0gKLKtaZOPmGs0BU5nuyMTGih~PzaQEKd4yip0JeqoIMxaX9-LFHUQhOLxhEb-VEqLT9jybYwD~WCAllxMCO~Wb93ISTQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsp017IL0011.gif" /></span> gave a correlation coefficient of −0.91 (<em>r</em>, Pearson's), the absolute values of correlation coefficients using other output values were smaller than 0.72 (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>). Median and Lowess normalization did not improve the linear correlation.</p>                            <h3 scrollto-destination=6247707 id="6247707" class="section-title" data-legacy-id=s3b>3.2 Development of a novel output value ‘Me value’</h3>
<p>To improve the linearity of the signal log ratio, which reflected the global methylation levels, <em>P</em><span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp017/2/m_dsp017IL0012.gif?Expires=2147483647&amp;Signature=1N0qFt~8OTp6EwaNvORC46Xjs7mwVNyqhnvymNeN0ZsLzAlXQctHdVSOOQlum2AP-cn9w1Fht19tPyN8cmP4FuiycrH1fGMeWqzGCpFhZ3e1UMy5Yi1stMOaE909hMy5w~mbRTDEVJjFa0-9eH1UDjhe1Q9ow5M8LZickSWVGqaWT~RJHk7IEYy7z1qQI6qQH2EGrUmcve5g2IPciUa6oR4TM7DC8IAMMELX88fnVYuKFj~0~TyV0j5iMznRfODIl696KCIhrGElIAgYNjYlDScL4MuFp0uKD3sps92PlOjKnn1S8CbGKlXTie97aSgcQG5mHhYXT~agtSnE14RUyQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsp017IL0012.gif" />⁠</span>, which had a strong linear correlation, was used as a coefficient to multiply it. Because <em>P</em><span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp017/2/m_dsp017IL0013.gif?Expires=2147483647&amp;Signature=R96Tk5J6Pt0kR94047AxpjAxH8gGTRjqyRMJwa2pE13WZF2GWdnx7bK-WiTwEeXSdy6YcPy8Givw9T5FOANedZDKnDJe7i~ozMJFAy5U4fSu3dQ3PoCgYW2ZeUFdZeVzVgPblYq-lYR-T~tYENMzr3CAXjirFZiyXq9Hl31WFh30dasVHsa0m~poEl0rVYHfCRw1qJbAy4rh6CPN0mQK7EAXjKZn8gs4QWEE3TMhN4r-ciuotsf41y6T7L7GxMw-2LZRTAEo2oz-xS~fwtgaq~AeNmcjw5uQ4-eoERRgBlnp6fJn0S3fB4pY8CR0AUIanuixMRMkkz9efj46xwtPiA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsp017IL0013.gif" /></span> showed an inverse correlation, (1 − <em>P</em><span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp017/2/m_dsp017IL0014.gif?Expires=2147483647&amp;Signature=V~-va9rBTFsp8rHgpD7jSxyyZYL1Vxez8saJxI0mISdMkeUS8O~LtofzaP86gPOxE~QCbJitFfR8Oq8O1rZHVDqkYyrB~4kPgHtx-Kl9aPUMofc6A7DT0ZLNAFuHQxDYsHe0gkP3xbxK6mYh0bkDRKVk~9idZAH0NQAEAsjjpFfy5g8vdS8zK2Exn9AiaG4sb~yO1YK8iGWYjfW12CGheHaC0AQY7vRAFaqD3WJU~8-XeirUFSGZob8g1koEKQVVdRl4po1itbV9A96oMM32GjnNuSf7mjneEhPy9ERKCRl~3K-NFgiPPik37jcxq6S6xms1SS2pou40JFGGvo-9QA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsp017IL0014.gif" />⁠</span>) was used as a coefficient to multiply the signal log ratio ‘=(1 − <em>P</em><span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp017/2/m_dsp017IL0015.gif?Expires=2147483647&amp;Signature=HVZ3i61wATg1UMjD~MhbPU5RMG3dTbD1r2w53333mAyRVACxja2kmTpwI2p~54MjPE6zio2rOHglbMEbPFbu7sqYAV5lT9RYayRQA9701dP08sAAiofdo8h7U3rAe~3YQmstTAeOV9u8cmBcfny4irAYGVLT36nKo-qlKp3oOvOMTJPF5MgU0JQf0nhGG7LREvFrqLtT19GYJSQYDaUPNYRYbio99j36XJ4ftok8OY44-3RfBdBhWfnNQ-Kw7usoEheZKeZBKizNo0CbnP7kY4gGqy0mLhJB7m54qwi9pY3DrDQETx7bwlGbNPXkTHzvnRO5jdTHssE-g~bsPRwDug__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsp017IL0015.gif" />⁠</span>) × signal log ratio’. This value showed a higher correlation coefficient (0.93) than the other output values (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>). This value was scaled to a value with a minimum value of 0 and a maximum value of 1 using the constants in Section 2, and the scaled output value was designated as the ‘Me value’. The Me value had the largest correlation coefficient (0.94) among all of the output values analysed. The distribution of the Me value reflected the global methylation levels by showing a shift towards smaller values after demethylation (Fig. <a reveal-id="DSP017F1" data-open="DSP017F1" class="link link-reveal link-table xref-fig">1</a>D).</p><p>The Me value was generated taking advantage of the signal log ratio and <em>P</em><span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp017/2/m_dsp017IL0016.gif?Expires=2147483647&amp;Signature=1Id0Wl6OVbaPLXkSTknxIvs1-Z4WUERc0~tObYCpPP7I7S~qIRLbdIAvih2wYxIdhB5-kwLHn8-pl7nn02LsUUdb2XjsYwnYV7aESfy1egCh1OiKGjgQmhvxeOAU8BCAfpnh78fjCbqwBln74UeHlWHr-D5CyZPSIqdFEhz5lYt5hM-OwwLUl4qCrPumqDfCJY~Ji~wg4oEpHbxsRXimKtlD8toNf6niM6zfSCQZqVO6Kfm1MivB7F5nePY4Y7Y9J7uLpkq7I84LeXdyRTPQCouU76G2S1J9fBFPt-jnmwLtNjbIuw8Cfnkeea0ckLrRmZbRi66w3sTqkRYc-FmqOg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsp017IL0016.gif" />⁠</span>. The signal log ratio was not quantitative but reflected the global methylation levels. On the other hand, <em>P</em><span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp017/2/m_dsp017IL0017.gif?Expires=2147483647&amp;Signature=ZcpqVint~V4b3alxO8j-5beG0cy1FYw8trsn7YrSxA4Ov5k8kKLyNqJkYJ4lxs7Pm0Sq8J4M3ynpuwwqwcjwysa8sIgKnipADANkrVZ8yyk998KV-H2HJxMGKjp2ewARSQq4~wMFfBXFMTQFEIYxb0tbCjJOBBZ0SOkaeyMydUdSXy~J4TGJNzzcxAv2rfSjmN2wnOsVw~f8Ja88hfuMB6y~oQrCF~obwTJSJirCD-N8zq9vQnEzIBi44G-fUixQea2ypfFfv~odu1-N87rkpufxM84bB5chkk3XbVkN--gs70B1sNY-l81BLd0A2VD~gsXGrHHCHPCrSFzAaB177w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsp017IL0017.gif" /></span> had a high linear correlation with the fraction of DNA molecules IP by the anti-5-methylcytidine antibody. The <em>P</em><span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp017/2/m_dsp017IL0018.gif?Expires=2147483647&amp;Signature=kn0G3O-VE~efuWyO6jwR0L9iiJS8X3UoiAnFaMwLCaaPJQq3eo3s~i4-SRrsDq44d2Twhq-z~zv799Vx-vv1roRJ~5RWqomv9BLiJ~pzNiutlVU7rRCH7pYw46j1hlqaKXtbSU2AMA4Jwk5n4a1~45J01d2sRkiXuxw5LG11buBQS6P9Gl1wWxMTWbpj33Z8qAaBGS9OlEHXz3Z7k7KJU-m8IfKJZ7vgJ7sr1VpfqQR1yef5t9k42k7mbYRovrT1JTj3wvvv1NxdrY14PoowRauaSO9pgCLfKkIaq9s7SO69LwiqJL1RywuIBnrCP7ASb1XMizVlbXp~R7enDwsHfA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsp017IL0018.gif" /></span> value is obtained as a probability value that takes account of neighbouring probes, and reflects the methylation level of a small local region. Since the vast majority of CpG sites within a CGI are (un)methylated when the CGI is (un)methylated,<sup><a reveal-id="DSP017C31" data-open="DSP017C31" class="link link-ref link-reveal xref-bibr">31</a></sup> the <em>P</em><span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp017/2/m_dsp017IL0019.gif?Expires=2147483647&amp;Signature=JkAJUG4k~ciNlRgP1l79cRGbUsRXC7OQkWucJ9do-TzUr9d0ZFGJXbm8brxfXddMhZOHDQLgm9yatEqFttrGgjD3mWv38AZpCR3sOQoKwZ3vW8Wb3uXPLMLxzhnQQNXTXeV5uHB5nUx8RxyQc53HB3WWZG7xhxnSm5-9LXpQUYeH9ZHzDmYFr-LV~vTpKPPGtr1aHvLpSNKS6mwdOm5cK3h4fpTkVbgHE4T1Ahz9owo1~9GGZsnmLoiBKP6NAl2nJrcmB393cTqcawPtJRZ0oQR0pTZugtP~WBLVFWOEm736XhV1GN2qpot2j3nNOb07lWKkNA4AzZJGliP7F16K3w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsp017IL0019.gif" /></span> value was considered to have an advantage in faithful reflection of the local methylation status.</p>                            <h3 scrollto-destination=6247710 id="6247710" class="section-title" data-legacy-id=s3c>3.3 High accuracy of MeDIP-CGI microarray with Me value</h3>
<p>In addition to the linear correlation between the Me value and the fraction of DNA molecules IP, a linear correlation between the Me value and the fraction of methylated CpG sites was analysed. Fractions of methylated CpG sites of 11 genomic regions (each 200 bp) with a variety of methylation levels were obtained by bisulphite sequencing in four different cell lines with different global methylation levels<sup><a reveal-id="DSP017C26" data-open="DSP017C26" class="link link-ref link-reveal xref-bibr">26</a></sup> (44 values in total). The Me value was obtained for a probe (site Me value) in the centre of the genomic region analysed. A strong correlation (<em>r</em> = 0.88, Fig. <a reveal-id="DSP017F2" data-open="DSP017F2" class="link link-reveal link-table xref-fig">2</a>) was observed. The correlation was stronger than any of those obtained by the other output values. When the analysis was limited to genomic regions with intermediate Me values, the correlation coefficients were 0.75 (Me value: 0.1–0.9); 0.53 (0.2–0.8); 0.47 (0.3–0.7); and 0.19 (0.4–0.6). These data further supported that a site Me value reflects a methylation level of a small genomic region (200 bp), even if it is within an intermediate range, and that the Me value is useful for the analysis of various biological samples, such as tissue samples.</p>                        <a id="6247712" scrollto-destination="6247712"></a>
<div id="DSP017F2" data-id="DSP017F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp017/2/m_dsp01702.jpeg?Expires=2147483647&Signature=pkWpAWD3u5MUXnY1tdXjvicq20rK84dTKUaNuja89biGUyn4-G1fIjmTcqjsmd3ttxW18f9ljPCe7VJ8scsjY1ys8CiMnRXzIi-A8A4rvFJ-4ZbZMr9d4T1tVYq8Dk-jX6c0v~OX8s7su2AkLorVicxthz2qmKOoe6G6Divk2IrBzCb8OX-3CR-gZkiPBd0qalnSQJgtwmlldc~9wJb4Q2n~ZwJGRS1RX4S69Ksrn1KV0Kn9it94uXdjc9aB80tFHCbPl2Om2yqxXaBCimuKFUTDIr8G-Ws8Nkb3SSHLe1-wNPki9fpNWyYbAmhqOtrRXPTK71g6-0GFkMWW62KPmQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Correlation between an output value and the fraction of methylated CpG sites by bisulphite sequencing. Forty samples with variable methylation levels (11 loci in four cell lines) were analysed by bisulphite sequencing, and the fraction of methylated CpG sites among all the CpG sites within 200 bp of the microarray probes was obtained. The numbers of CpG sites in this region ranged from 7 to 21. The Me value had a linear correlation with the fraction (r = 0.88, Pearson's correlation coefficient)." path-from-xml="dsp01702.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6247712/dsp01702.jpeg" path-from-xml="dsp01702.jpeg" target="_blank">View large</a><a section="6247712" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp017/2/dsp01702.jpeg?Expires=2147483647&Signature=na7o80ZSmJA2bUOVUQYUqt4fKCofeRn9e88OogjP463XHAjGC9eajugxz-bk-U6S19aMe74FLdJ7UFUB-JaD28zMN1k6PzxCh0ZvrwuoX3TGDVNQ0-C0XKhAA~g0e2SR63UJMmdMTGwPPitHORO9V9hvBiXc3UXMQL0VZAgfVpk24HVpfhtrGutRRIxzHg4PKryuEMtrdoc~t2vPRkzNl9iaU1ahC96PTwwFXLeh9IiJLxzcrqkoqm6nh-rUWTL6mbizKclmt393hLP9K9etAXLlG9G41~mQQkMjsRiX8ysFplE3akL~L0Pi13sVMpfJQsmseBGfo3Za9L1dc256kw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6247712&ar=421566&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp01702.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Correlation between an output value and the fraction of methylated CpG sites by bisulphite sequencing. Forty samples with variable methylation levels (11 loci in four cell lines) were analysed by bisulphite sequencing, and the fraction of methylated CpG sites among all the CpG sites within 200 bp of the microarray probes was obtained. The numbers of CpG sites in this region ranged from 7 to 21. The Me value had a linear correlation with the fraction (<em>r</em> = 0.88, Pearson's correlation coefficient).</p></div></div><div content-id="DSP017F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp017/2/m_dsp01702.jpeg?Expires=2147483647&Signature=pkWpAWD3u5MUXnY1tdXjvicq20rK84dTKUaNuja89biGUyn4-G1fIjmTcqjsmd3ttxW18f9ljPCe7VJ8scsjY1ys8CiMnRXzIi-A8A4rvFJ-4ZbZMr9d4T1tVYq8Dk-jX6c0v~OX8s7su2AkLorVicxthz2qmKOoe6G6Divk2IrBzCb8OX-3CR-gZkiPBd0qalnSQJgtwmlldc~9wJb4Q2n~ZwJGRS1RX4S69Ksrn1KV0Kn9it94uXdjc9aB80tFHCbPl2Om2yqxXaBCimuKFUTDIr8G-Ws8Nkb3SSHLe1-wNPki9fpNWyYbAmhqOtrRXPTK71g6-0GFkMWW62KPmQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Correlation between an output value and the fraction of methylated CpG sites by bisulphite sequencing. Forty samples with variable methylation levels (11 loci in four cell lines) were analysed by bisulphite sequencing, and the fraction of methylated CpG sites among all the CpG sites within 200 bp of the microarray probes was obtained. The numbers of CpG sites in this region ranged from 7 to 21. The Me value had a linear correlation with the fraction (r = 0.88, Pearson's correlation coefficient)." path-from-xml="dsp01702.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6247712/dsp01702.jpeg" path-from-xml="dsp01702.jpeg" target="_blank">View large</a><a section="6247712" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp017/2/dsp01702.jpeg?Expires=2147483647&Signature=na7o80ZSmJA2bUOVUQYUqt4fKCofeRn9e88OogjP463XHAjGC9eajugxz-bk-U6S19aMe74FLdJ7UFUB-JaD28zMN1k6PzxCh0ZvrwuoX3TGDVNQ0-C0XKhAA~g0e2SR63UJMmdMTGwPPitHORO9V9hvBiXc3UXMQL0VZAgfVpk24HVpfhtrGutRRIxzHg4PKryuEMtrdoc~t2vPRkzNl9iaU1ahC96PTwwFXLeh9IiJLxzcrqkoqm6nh-rUWTL6mbizKclmt393hLP9K9etAXLlG9G41~mQQkMjsRiX8ysFplE3akL~L0Pi13sVMpfJQsmseBGfo3Za9L1dc256kw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6247712&ar=421566&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp01702.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Correlation between an output value and the fraction of methylated CpG sites by bisulphite sequencing. Forty samples with variable methylation levels (11 loci in four cell lines) were analysed by bisulphite sequencing, and the fraction of methylated CpG sites among all the CpG sites within 200 bp of the microarray probes was obtained. The numbers of CpG sites in this region ranged from 7 to 21. The Me value had a linear correlation with the fraction (<em>r</em> = 0.88, Pearson's correlation coefficient).</p></div></div><p>Cancer tissues sometimes show mixtures of methylated and unmethylated CpG sites on the same DNA molecule (mosaic pattern). Even in this case, multiple CpG sites are usually located within the average size of shearing DNA (300 bp) because there are 9–53 CpG sites in 300 bp regions of promoter CGIs.<sup><a reveal-id="DSP017C20" data-open="DSP017C20" class="link link-ref link-reveal xref-bibr">20</a></sup> If two or more CpG sites are methylated, such DNA molecules are reported to be efficiently IP.<sup><a reveal-id="DSP017C9" data-open="DSP017C9" class="link link-ref link-reveal xref-bibr">9</a></sup> Therefore, it is expected that, for most CGIs, the Me value will work even in samples with mosaic pattern methylation, such as cancer tissues.</p><p>Next, detection of completely methylated and unmethylated statuses of CGIs was attempted. Using output values other than the Me value, this has been achieved by optimizing cut-off values depending upon samples because their global methylation levels were highly variable. Since the Me value well reflected the global methylation levels, we tried to use cut-off CGI Me values common to different samples. Methylation statuses of 113 CGIs in four cell lines with different global methylation levels<sup><a reveal-id="DSP017C26" data-open="DSP017C26" class="link link-ref link-reveal xref-bibr">26</a></sup> (452 values in total) were scored using various cut-off CGI Me values. A high specificity with little compromise of sensitivity was achieved with the cut-off values of 0.6 and 0.4 for highly methylated and unmethylated CGIs, respectively (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S3</a>). Accuracies for methylated and unmethylated CGIs with these cut-off values were 71.8 and 83.8%, respectively (Table <a reveal-id="DSP017TB1" data-open="DSP017TB1" class="link link-reveal link-table xref-fig">1</a>). Since these cut-off CGI Me values worked in the four samples with different global methylation levels, Me values of 0.6 and 0.4 were considered to be suitable to identify highly methylated and unmethylated CGIs, respectively, with reasonable accuracies.</p>                        <a id="6247715" scrollto-destination="6247715"></a>
<div content-id="DSP017TB1" class="table-modal"><div class="table-wrap"><div id="DSP017TB1" class="table-wrap-title" data-id="DSP017TB1"><div class="label -label">Table 1</div><div class="caption"><p>Comparison between methylation statuses determined using the CGI Me value and those by MSP</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>MSP </th><th>MeDIP-CGI microarray </th><th>AGS </th><th>HSC39 </th><th>HSC57 </th><th>KATOIII </th><th>Total </th></tr></thead><tbody><tr><td rowspan="3">Methylated </td><td>Highly methylated </td><td>64 </td><td>17 </td><td>12 </td><td>36 </td><td>129 </td></tr><tr><td>Moderately methylated </td><td>29 </td><td>13 </td><td>6 </td><td>23 </td><td>71 </td></tr><tr><td>Unmethylated </td><td>4 </td><td>3 </td><td>6 </td><td>8 </td><td>21 </td></tr><tr><td rowspan="3">Unmethylated </td><td>Highly methylated </td><td>1 </td><td>6 </td><td>7 </td><td>4 </td><td>18 </td></tr><tr><td>Moderately methylated </td><td>2 </td><td>8 </td><td>11 </td><td>4 </td><td>25 </td></tr><tr><td>Unmethylated </td><td>11 </td><td>56 </td><td>61 </td><td>21 </td><td>149 </td></tr><tr><td rowspan="3">Methylated/unmethylated </td><td>Highly methylated </td><td>0 </td><td>4 </td><td>5 </td><td>8 </td><td>17 </td></tr><tr><td>Moderately methylated </td><td>1 </td><td>4 </td><td>2 </td><td>5 </td><td>12 </td></tr><tr><td>Unmethylated </td><td>1 </td><td>2 </td><td>2 </td><td>4 </td><td>9 </td></tr><tr><td>Not amplified by MSP </td><td> </td><td>0 </td><td>0 </td><td>1 </td><td>0 </td><td>1 </td></tr><tr><td rowspan="3">Methylated </td><td>Sensitivity </td><td>0.660 </td><td>0.515 </td><td>0.500 </td><td>0.537 </td><td>0.584 </td></tr><tr><td>Specificity </td><td>0.938 </td><td>0.875 </td><td>0.864 </td><td>0.739 </td><td>0.848 </td></tr><tr><td>Accuracy </td><td>0.699 </td><td>0.770 </td><td>0.786 </td><td>0.619 </td><td>0.718 </td></tr><tr><td rowspan="3">Unmethylated </td><td>Sensitivity </td><td>0.786 </td><td>0.800 </td><td>0.772 </td><td>0.724 </td><td>0.776 </td></tr><tr><td>Specificity </td><td>0.949 </td><td>0.884 </td><td>0.758 </td><td>0.857 </td><td>0.884 </td></tr><tr><td>Accuracy </td><td>0.929 </td><td>0.832 </td><td>0.768 </td><td>0.823 </td><td>0.838 </td></tr></tbody></table></div><div content-id="DSP017TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>MSP </th><th>MeDIP-CGI microarray </th><th>AGS </th><th>HSC39 </th><th>HSC57 </th><th>KATOIII </th><th>Total </th></tr></thead><tbody><tr><td rowspan="3">Methylated </td><td>Highly methylated </td><td>64 </td><td>17 </td><td>12 </td><td>36 </td><td>129 </td></tr><tr><td>Moderately methylated </td><td>29 </td><td>13 </td><td>6 </td><td>23 </td><td>71 </td></tr><tr><td>Unmethylated </td><td>4 </td><td>3 </td><td>6 </td><td>8 </td><td>21 </td></tr><tr><td rowspan="3">Unmethylated </td><td>Highly methylated </td><td>1 </td><td>6 </td><td>7 </td><td>4 </td><td>18 </td></tr><tr><td>Moderately methylated </td><td>2 </td><td>8 </td><td>11 </td><td>4 </td><td>25 </td></tr><tr><td>Unmethylated </td><td>11 </td><td>56 </td><td>61 </td><td>21 </td><td>149 </td></tr><tr><td rowspan="3">Methylated/unmethylated </td><td>Highly methylated </td><td>0 </td><td>4 </td><td>5 </td><td>8 </td><td>17 </td></tr><tr><td>Moderately methylated </td><td>1 </td><td>4 </td><td>2 </td><td>5 </td><td>12 </td></tr><tr><td>Unmethylated </td><td>1 </td><td>2 </td><td>2 </td><td>4 </td><td>9 </td></tr><tr><td>Not amplified by MSP </td><td> </td><td>0 </td><td>0 </td><td>1 </td><td>0 </td><td>1 </td></tr><tr><td rowspan="3">Methylated </td><td>Sensitivity </td><td>0.660 </td><td>0.515 </td><td>0.500 </td><td>0.537 </td><td>0.584 </td></tr><tr><td>Specificity </td><td>0.938 </td><td>0.875 </td><td>0.864 </td><td>0.739 </td><td>0.848 </td></tr><tr><td>Accuracy </td><td>0.699 </td><td>0.770 </td><td>0.786 </td><td>0.619 </td><td>0.718 </td></tr><tr><td rowspan="3">Unmethylated </td><td>Sensitivity </td><td>0.786 </td><td>0.800 </td><td>0.772 </td><td>0.724 </td><td>0.776 </td></tr><tr><td>Specificity </td><td>0.949 </td><td>0.884 </td><td>0.758 </td><td>0.857 </td><td>0.884 </td></tr><tr><td>Accuracy </td><td>0.929 </td><td>0.832 </td><td>0.768 </td><td>0.823 </td><td>0.838 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Using 113 CGIs, we compared methylation statuses determined using the Me value and those by MSP.</p></div></div><div content-id=""><div class="fn"><p>Methylated/unmethylated, both methylated and unmethylated DNA molecules were detected by MSP.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6247715" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSP017TB1" class="table-wrap-title" data-id="DSP017TB1"><div class="label -label">Table 1</div><div class="caption"><p>Comparison between methylation statuses determined using the CGI Me value and those by MSP</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>MSP </th><th>MeDIP-CGI microarray </th><th>AGS </th><th>HSC39 </th><th>HSC57 </th><th>KATOIII </th><th>Total </th></tr></thead><tbody><tr><td rowspan="3">Methylated </td><td>Highly methylated </td><td>64 </td><td>17 </td><td>12 </td><td>36 </td><td>129 </td></tr><tr><td>Moderately methylated </td><td>29 </td><td>13 </td><td>6 </td><td>23 </td><td>71 </td></tr><tr><td>Unmethylated </td><td>4 </td><td>3 </td><td>6 </td><td>8 </td><td>21 </td></tr><tr><td rowspan="3">Unmethylated </td><td>Highly methylated </td><td>1 </td><td>6 </td><td>7 </td><td>4 </td><td>18 </td></tr><tr><td>Moderately methylated </td><td>2 </td><td>8 </td><td>11 </td><td>4 </td><td>25 </td></tr><tr><td>Unmethylated </td><td>11 </td><td>56 </td><td>61 </td><td>21 </td><td>149 </td></tr><tr><td rowspan="3">Methylated/unmethylated </td><td>Highly methylated </td><td>0 </td><td>4 </td><td>5 </td><td>8 </td><td>17 </td></tr><tr><td>Moderately methylated </td><td>1 </td><td>4 </td><td>2 </td><td>5 </td><td>12 </td></tr><tr><td>Unmethylated </td><td>1 </td><td>2 </td><td>2 </td><td>4 </td><td>9 </td></tr><tr><td>Not amplified by MSP </td><td> </td><td>0 </td><td>0 </td><td>1 </td><td>0 </td><td>1 </td></tr><tr><td rowspan="3">Methylated </td><td>Sensitivity </td><td>0.660 </td><td>0.515 </td><td>0.500 </td><td>0.537 </td><td>0.584 </td></tr><tr><td>Specificity </td><td>0.938 </td><td>0.875 </td><td>0.864 </td><td>0.739 </td><td>0.848 </td></tr><tr><td>Accuracy </td><td>0.699 </td><td>0.770 </td><td>0.786 </td><td>0.619 </td><td>0.718 </td></tr><tr><td rowspan="3">Unmethylated </td><td>Sensitivity </td><td>0.786 </td><td>0.800 </td><td>0.772 </td><td>0.724 </td><td>0.776 </td></tr><tr><td>Specificity </td><td>0.949 </td><td>0.884 </td><td>0.758 </td><td>0.857 </td><td>0.884 </td></tr><tr><td>Accuracy </td><td>0.929 </td><td>0.832 </td><td>0.768 </td><td>0.823 </td><td>0.838 </td></tr></tbody></table></div><div content-id="DSP017TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>MSP </th><th>MeDIP-CGI microarray </th><th>AGS </th><th>HSC39 </th><th>HSC57 </th><th>KATOIII </th><th>Total </th></tr></thead><tbody><tr><td rowspan="3">Methylated </td><td>Highly methylated </td><td>64 </td><td>17 </td><td>12 </td><td>36 </td><td>129 </td></tr><tr><td>Moderately methylated </td><td>29 </td><td>13 </td><td>6 </td><td>23 </td><td>71 </td></tr><tr><td>Unmethylated </td><td>4 </td><td>3 </td><td>6 </td><td>8 </td><td>21 </td></tr><tr><td rowspan="3">Unmethylated </td><td>Highly methylated </td><td>1 </td><td>6 </td><td>7 </td><td>4 </td><td>18 </td></tr><tr><td>Moderately methylated </td><td>2 </td><td>8 </td><td>11 </td><td>4 </td><td>25 </td></tr><tr><td>Unmethylated </td><td>11 </td><td>56 </td><td>61 </td><td>21 </td><td>149 </td></tr><tr><td rowspan="3">Methylated/unmethylated </td><td>Highly methylated </td><td>0 </td><td>4 </td><td>5 </td><td>8 </td><td>17 </td></tr><tr><td>Moderately methylated </td><td>1 </td><td>4 </td><td>2 </td><td>5 </td><td>12 </td></tr><tr><td>Unmethylated </td><td>1 </td><td>2 </td><td>2 </td><td>4 </td><td>9 </td></tr><tr><td>Not amplified by MSP </td><td> </td><td>0 </td><td>0 </td><td>1 </td><td>0 </td><td>1 </td></tr><tr><td rowspan="3">Methylated </td><td>Sensitivity </td><td>0.660 </td><td>0.515 </td><td>0.500 </td><td>0.537 </td><td>0.584 </td></tr><tr><td>Specificity </td><td>0.938 </td><td>0.875 </td><td>0.864 </td><td>0.739 </td><td>0.848 </td></tr><tr><td>Accuracy </td><td>0.699 </td><td>0.770 </td><td>0.786 </td><td>0.619 </td><td>0.718 </td></tr><tr><td rowspan="3">Unmethylated </td><td>Sensitivity </td><td>0.786 </td><td>0.800 </td><td>0.772 </td><td>0.724 </td><td>0.776 </td></tr><tr><td>Specificity </td><td>0.949 </td><td>0.884 </td><td>0.758 </td><td>0.857 </td><td>0.884 </td></tr><tr><td>Accuracy </td><td>0.929 </td><td>0.832 </td><td>0.768 </td><td>0.823 </td><td>0.838 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Using 113 CGIs, we compared methylation statuses determined using the Me value and those by MSP.</p></div></div><div content-id=""><div class="fn"><p>Methylated/unmethylated, both methylated and unmethylated DNA molecules were detected by MSP.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6247715" target="_blank">View Large</a></div></div><p>These data showed that, for quantification of methylation levels, the Me value had linearity similar to Batman and MEDME.<sup><a reveal-id="DSP017C14" data-open="DSP017C14" class="link link-ref link-reveal xref-bibr">14</a>,<a reveal-id="DSP017C16" data-open="DSP017C16" class="link link-ref link-reveal xref-bibr">16</a></sup> Although samples analysed were different, the coefficient of determination (<em>R</em><sup>2</sup>, Pearson's) obtained by the Me value was 0.77, being in the same range as Batman (0.82) and MEDME (0.75). The linearity of the Me value was validated also using four samples with different global methylation levels, which has not been done with Batman (one sample) and MEDME (two samples). Further, the CGI Me value enabled us to score completely methylated and unmethylated statuses of CGIs in samples with variable global methylation levels using common cut-off values. It is also of note that the Me value can be conveniently calculated using a spreadsheet and a commercially available software, such as Excel (Microsoft).</p>                            <h3 scrollto-destination=6247717 id="6247717" class="section-title" data-legacy-id=s3d>3.4 Application of the Me value to analysis of the association between CGI methylation and gene expression</h3>
<p>Using the Me value, methylation profiles of CGIs and promoters were analysed in four gastric cancer cell lines, AGS cells after 5-aza-dC treatment, and one normal gastric tissue sample (a pool of four samples from four individuals). CGIs (promoters) with CGI (promoter) Me values more than 0.6 were classified as ‘methylated’. Of the 30 533 CGIs analysed, 3768–7310 CGIs were methylated in the cancer cell lines, and 3393 CGIs were methylated in the normal sample (Table <a reveal-id="DSP017TB2" data-open="DSP017TB2" class="link link-reveal link-table xref-fig">2</a>). However, methylation was infrequent in promoters (Table <a reveal-id="DSP017TB2" data-open="DSP017TB2" class="link link-reveal link-table xref-fig">2</a>, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S4</a>A and B). The ratio of methylated promoters in cancer cell lines (6.7–12.5%) was in accordance with that in a previous report (10.9% in lung cancer cell lines).<sup><a reveal-id="DSP017C32" data-open="DSP017C32" class="link link-ref link-reveal xref-bibr">32</a></sup> The difference between overall CGIs and promoters was clearer in the normal gastric tissue than in the human gastric cancer cell lines. Interestingly, CGIs in the vicinity of the LINE and SINE repetitive elements had lower Me values than those further away from them in cancer cell lines (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S4</a>C and D). A small peak was observed at 300 bp from SINE, and CGIs closer to SINE and LINE were less methylated than those further apart from SINE and LINE. There is a possibility that a boundary function of these repetitive elements prevents methylation beyond the boundary spread across into CGIs. Actually, SINE is reported to have binding sites for the TFIIIC transcription factor, and to protect promoter CGIs from repressive chromatin modifications.<sup><a reveal-id="DSP017C33" data-open="DSP017C33" class="link link-ref link-reveal xref-bibr">33</a></sup></p>                        <a id="6247719" scrollto-destination="6247719"></a>
<div content-id="DSP017TB2" class="table-modal"><div class="table-wrap"><div id="DSP017TB2" class="table-wrap-title" data-id="DSP017TB2"><div class="label -label">Table 2</div><div class="caption"><p>DNA methylation profiles of CGIs according to their positions against TSSs obtained by MeDIP-CGI microarray</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2"> </th><th rowspan="2">Total number analysed </th><th colspan="6">Number of methylated CpG islands and promoters<hr /> </th></tr><tr><th>AGS </th><th>HSC39 </th><th>HSC57 </th><th>KATOIII </th><th>AGS + 5-aza-dC (1 µM) </th><th>Normal gastric tissue </th></tr></thead><tbody><tr><td>CpG islands </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> All CGI </td><td>30 533 </td><td>7310 (23.9%) </td><td>3768 (12.3%) </td><td>4663 (15.3%) </td><td>6460 (21.2%) </td><td>4 (0.0%) </td><td>3393 (11.1%) </td></tr><tr><td>  Upstream </td><td>10 709 </td><td>1911 (17.8%) </td><td>937 (8.7%) </td><td>794 (7.4%) </td><td>1556 (14.5%) </td><td>1 (0.0%) </td><td>260 (2.4%) </td></tr><tr><td>  Gene body (+1 to +1 k) </td><td>10 654 </td><td>1607 (15.1%) </td><td>867 (8.1%) </td><td>665 (6.2%) </td><td>1328 (12.5%) </td><td>0 (0.0%) </td><td>194 (1.8%) </td></tr><tr><td>  Gene body (+1 to +5 k) </td><td>2050 </td><td>694 (47.0%) </td><td>508 (24.8%) </td><td>598 (29.2%) </td><td>809 (39.5%) </td><td>1 (0.0%) </td><td>353 (17.2%) </td></tr><tr><td>  Gene body (more than +5 k) </td><td>4431 </td><td>2438 (55.0%) </td><td>1099 (24.8%) </td><td>2171 (49.0%) </td><td>2155 (48.6%) </td><td>2 (0.0%) </td><td>2360 (53.3%) </td></tr><tr><td>  Downstream </td><td>1186 </td><td>561 (47.3%) </td><td>310 (26.1%) </td><td>376 (31.7%) </td><td>509 (42.9%) </td><td>0 (0.0%) </td><td>216 (18.2%) </td></tr><tr><td>  Divergent </td><td>1503 </td><td>99 (6.6%) </td><td>47 (3.1%) </td><td>59 (3.9%) </td><td>103 (6.9%) </td><td>0 (0.0%) </td><td>10 (0.7%) </td></tr><tr><td>Promoters </td><td>9624 </td><td>1205 (12.5%) </td><td>792 (8.2%) </td><td>641 (6.7%) </td><td>1142 (11.9%) </td><td>3 (0.0%) </td><td>113 (1.2%) </td></tr><tr><td>Probes </td><td>237 202 </td><td>70 027 (29.5%) </td><td>43 825 (18.5%) </td><td>48 292 (20.4%) </td><td>65 192 (27.5%) </td><td>723 (0.3%) </td><td>27 017 (11.4%) </td></tr></tbody></table></div><div content-id="DSP017TB2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2"> </th><th rowspan="2">Total number analysed </th><th colspan="6">Number of methylated CpG islands and promoters<hr /> </th></tr><tr><th>AGS </th><th>HSC39 </th><th>HSC57 </th><th>KATOIII </th><th>AGS + 5-aza-dC (1 µM) </th><th>Normal gastric tissue </th></tr></thead><tbody><tr><td>CpG islands </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> All CGI </td><td>30 533 </td><td>7310 (23.9%) </td><td>3768 (12.3%) </td><td>4663 (15.3%) </td><td>6460 (21.2%) </td><td>4 (0.0%) </td><td>3393 (11.1%) </td></tr><tr><td>  Upstream </td><td>10 709 </td><td>1911 (17.8%) </td><td>937 (8.7%) </td><td>794 (7.4%) </td><td>1556 (14.5%) </td><td>1 (0.0%) </td><td>260 (2.4%) </td></tr><tr><td>  Gene body (+1 to +1 k) </td><td>10 654 </td><td>1607 (15.1%) </td><td>867 (8.1%) </td><td>665 (6.2%) </td><td>1328 (12.5%) </td><td>0 (0.0%) </td><td>194 (1.8%) </td></tr><tr><td>  Gene body (+1 to +5 k) </td><td>2050 </td><td>694 (47.0%) </td><td>508 (24.8%) </td><td>598 (29.2%) </td><td>809 (39.5%) </td><td>1 (0.0%) </td><td>353 (17.2%) </td></tr><tr><td>  Gene body (more than +5 k) </td><td>4431 </td><td>2438 (55.0%) </td><td>1099 (24.8%) </td><td>2171 (49.0%) </td><td>2155 (48.6%) </td><td>2 (0.0%) </td><td>2360 (53.3%) </td></tr><tr><td>  Downstream </td><td>1186 </td><td>561 (47.3%) </td><td>310 (26.1%) </td><td>376 (31.7%) </td><td>509 (42.9%) </td><td>0 (0.0%) </td><td>216 (18.2%) </td></tr><tr><td>  Divergent </td><td>1503 </td><td>99 (6.6%) </td><td>47 (3.1%) </td><td>59 (3.9%) </td><td>103 (6.9%) </td><td>0 (0.0%) </td><td>10 (0.7%) </td></tr><tr><td>Promoters </td><td>9624 </td><td>1205 (12.5%) </td><td>792 (8.2%) </td><td>641 (6.7%) </td><td>1142 (11.9%) </td><td>3 (0.0%) </td><td>113 (1.2%) </td></tr><tr><td>Probes </td><td>237 202 </td><td>70 027 (29.5%) </td><td>43 825 (18.5%) </td><td>48 292 (20.4%) </td><td>65 192 (27.5%) </td><td>723 (0.3%) </td><td>27 017 (11.4%) </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Definitions of the CGI positions are described in Section 2.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6247719" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSP017TB2" class="table-wrap-title" data-id="DSP017TB2"><div class="label -label">Table 2</div><div class="caption"><p>DNA methylation profiles of CGIs according to their positions against TSSs obtained by MeDIP-CGI microarray</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2"> </th><th rowspan="2">Total number analysed </th><th colspan="6">Number of methylated CpG islands and promoters<hr /> </th></tr><tr><th>AGS </th><th>HSC39 </th><th>HSC57 </th><th>KATOIII </th><th>AGS + 5-aza-dC (1 µM) </th><th>Normal gastric tissue </th></tr></thead><tbody><tr><td>CpG islands </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> All CGI </td><td>30 533 </td><td>7310 (23.9%) </td><td>3768 (12.3%) </td><td>4663 (15.3%) </td><td>6460 (21.2%) </td><td>4 (0.0%) </td><td>3393 (11.1%) </td></tr><tr><td>  Upstream </td><td>10 709 </td><td>1911 (17.8%) </td><td>937 (8.7%) </td><td>794 (7.4%) </td><td>1556 (14.5%) </td><td>1 (0.0%) </td><td>260 (2.4%) </td></tr><tr><td>  Gene body (+1 to +1 k) </td><td>10 654 </td><td>1607 (15.1%) </td><td>867 (8.1%) </td><td>665 (6.2%) </td><td>1328 (12.5%) </td><td>0 (0.0%) </td><td>194 (1.8%) </td></tr><tr><td>  Gene body (+1 to +5 k) </td><td>2050 </td><td>694 (47.0%) </td><td>508 (24.8%) </td><td>598 (29.2%) </td><td>809 (39.5%) </td><td>1 (0.0%) </td><td>353 (17.2%) </td></tr><tr><td>  Gene body (more than +5 k) </td><td>4431 </td><td>2438 (55.0%) </td><td>1099 (24.8%) </td><td>2171 (49.0%) </td><td>2155 (48.6%) </td><td>2 (0.0%) </td><td>2360 (53.3%) </td></tr><tr><td>  Downstream </td><td>1186 </td><td>561 (47.3%) </td><td>310 (26.1%) </td><td>376 (31.7%) </td><td>509 (42.9%) </td><td>0 (0.0%) </td><td>216 (18.2%) </td></tr><tr><td>  Divergent </td><td>1503 </td><td>99 (6.6%) </td><td>47 (3.1%) </td><td>59 (3.9%) </td><td>103 (6.9%) </td><td>0 (0.0%) </td><td>10 (0.7%) </td></tr><tr><td>Promoters </td><td>9624 </td><td>1205 (12.5%) </td><td>792 (8.2%) </td><td>641 (6.7%) </td><td>1142 (11.9%) </td><td>3 (0.0%) </td><td>113 (1.2%) </td></tr><tr><td>Probes </td><td>237 202 </td><td>70 027 (29.5%) </td><td>43 825 (18.5%) </td><td>48 292 (20.4%) </td><td>65 192 (27.5%) </td><td>723 (0.3%) </td><td>27 017 (11.4%) </td></tr></tbody></table></div><div content-id="DSP017TB2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2"> </th><th rowspan="2">Total number analysed </th><th colspan="6">Number of methylated CpG islands and promoters<hr /> </th></tr><tr><th>AGS </th><th>HSC39 </th><th>HSC57 </th><th>KATOIII </th><th>AGS + 5-aza-dC (1 µM) </th><th>Normal gastric tissue </th></tr></thead><tbody><tr><td>CpG islands </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> All CGI </td><td>30 533 </td><td>7310 (23.9%) </td><td>3768 (12.3%) </td><td>4663 (15.3%) </td><td>6460 (21.2%) </td><td>4 (0.0%) </td><td>3393 (11.1%) </td></tr><tr><td>  Upstream </td><td>10 709 </td><td>1911 (17.8%) </td><td>937 (8.7%) </td><td>794 (7.4%) </td><td>1556 (14.5%) </td><td>1 (0.0%) </td><td>260 (2.4%) </td></tr><tr><td>  Gene body (+1 to +1 k) </td><td>10 654 </td><td>1607 (15.1%) </td><td>867 (8.1%) </td><td>665 (6.2%) </td><td>1328 (12.5%) </td><td>0 (0.0%) </td><td>194 (1.8%) </td></tr><tr><td>  Gene body (+1 to +5 k) </td><td>2050 </td><td>694 (47.0%) </td><td>508 (24.8%) </td><td>598 (29.2%) </td><td>809 (39.5%) </td><td>1 (0.0%) </td><td>353 (17.2%) </td></tr><tr><td>  Gene body (more than +5 k) </td><td>4431 </td><td>2438 (55.0%) </td><td>1099 (24.8%) </td><td>2171 (49.0%) </td><td>2155 (48.6%) </td><td>2 (0.0%) </td><td>2360 (53.3%) </td></tr><tr><td>  Downstream </td><td>1186 </td><td>561 (47.3%) </td><td>310 (26.1%) </td><td>376 (31.7%) </td><td>509 (42.9%) </td><td>0 (0.0%) </td><td>216 (18.2%) </td></tr><tr><td>  Divergent </td><td>1503 </td><td>99 (6.6%) </td><td>47 (3.1%) </td><td>59 (3.9%) </td><td>103 (6.9%) </td><td>0 (0.0%) </td><td>10 (0.7%) </td></tr><tr><td>Promoters </td><td>9624 </td><td>1205 (12.5%) </td><td>792 (8.2%) </td><td>641 (6.7%) </td><td>1142 (11.9%) </td><td>3 (0.0%) </td><td>113 (1.2%) </td></tr><tr><td>Probes </td><td>237 202 </td><td>70 027 (29.5%) </td><td>43 825 (18.5%) </td><td>48 292 (20.4%) </td><td>65 192 (27.5%) </td><td>723 (0.3%) </td><td>27 017 (11.4%) </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Definitions of the CGI positions are described in Section 2.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6247719" target="_blank">View Large</a></div></div><p>The numbers of methylated CGIs were larger in AGS and KATOIII than in HSC39 and HSC57, which was in accordance with our previous findings.<sup><a reveal-id="DSP017C26" data-open="DSP017C26" class="link link-ref link-reveal xref-bibr">26</a></sup> The difference among the four cell lines was observed not only at the overall genome level but also on individual chromosomes (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S5</a>). In contrast, the numbers of methylated promoters had the same difference at the overall genome level, but some distortions were present on specific chromosomes. For example, HSC39 had the largest numbers of methylated promoters on chromosomes 15, 18, and 22, and HSC57 had the largest number on chromosome 21. This result suggested that promoters have unique mechanisms to be methylated in cancer cells, such as functional selection of a cell with methylation of specific promoters.</p><p>Gene expression profiles were also obtained by expression microarray analysis of the same four cancer cell lines and the normal gastric tissue sample. When genes were classified by their expression levels, genes with high expression had lower methylation levels in their promoters than those with low expression in all cell lines (Fig. <a reveal-id="DSP017F3" data-open="DSP017F3" class="link link-reveal link-table xref-fig">3</a>A). When genes were classified by methylation levels (unmethylated, moderately methylated, and highly methylated), genes with high methylation in their promoters had lower expression than those with low methylation (left panel in Fig. <a reveal-id="DSP017F3" data-open="DSP017F3" class="link link-reveal link-table xref-fig">3</a>B). In contrast, genes with high methylation in their gene bodies (5 kb or more downstream of TSSs) had slightly, but significantly, higher expression levels (right panel). These results clearly showed that methylation of gene body CGIs was associated with increased gene expression, as in previous reports.<sup><a reveal-id="DSP017C34" data-open="DSP017C34" class="link link-ref link-reveal xref-bibr">34</a>–<a reveal-id="DSP017C37" data-open="DSP017C37" class="link link-ref link-reveal xref-bibr">37</a></sup> Finally, genes with low expression in normal gastric tissue had higher methylation levels of promoters in cancer cell lines (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S6</a>). This result was in line with the fact that genes with low expression are susceptible to DNA methylation.<sup><a reveal-id="DSP017C20" data-open="DSP017C20" class="link link-ref link-reveal xref-bibr">20</a>,<a reveal-id="DSP017C38" data-open="DSP017C38" class="link link-ref link-reveal xref-bibr">38</a>,<a reveal-id="DSP017C39" data-open="DSP017C39" class="link link-ref link-reveal xref-bibr">39</a></sup></p>                        <a id="6247722" scrollto-destination="6247722"></a>
<div id="DSP017F3" data-id="DSP017F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp017/2/m_dsp01703.jpeg?Expires=2147483647&Signature=v4THd6gvRX9Mz-FSji3PqOzZfbBZIUvJnBhUAJTR0w2vhASjyP~fzGdw6h90yIwp8Uqf1vEAN5X-dOT3kddPXUZ52Lx02YXJhKPVxhRSpNmqgCRHvpM0nClzOjviNXODkYBZahGfkTHYvlVubiAkmU-f~fUKfgLTMLmYUMJEW0Sh2~Sy~LzoOspRZx3c7E9pPvtd6M7Ve3g4jmh5AYq5QJB~FTAIGzMZYHd18LM2EPfO-iP6AMSMbK~8SVUjQBPrIsyaRVTJDlnCMO0JV8e7ObS--ChVLn8MflWK79F~YGlryGWlvpd1M5vWBJ~MSz1ginzqX6D6cHZpKkwntNCn9w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Association between DNA methylation level and gene expression. (A) Average methylation levels (site Me values) of genes with high and low expression. Genes with low expression (dashed line) showed significantly higher methylation levels than those with high expression (solid line) in positions close to TSSs. (B) Gene expression levels according to methylation statuses of the promoters (left panel) and gene bodies (right panel). Genes were classified into unmethylated (CGI Me value &lt;0.4, dotted line), moderately methylated (0.4 ≤ CGI Me value ≤ 0.6, dashed line), and highly methylated (CGI Me value &gt;0.6, solid line). Methylation of promoters was strongly associated with low expression (P = 8 × 10−203 in AGS, t-test of expression of methylated genes and unmethylated genes), and methylation of CGIs in gene bodies was associated with higher expression (P = 6 × 10−4 in AGS)." path-from-xml="dsp01703.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6247722/dsp01703.jpeg" path-from-xml="dsp01703.jpeg" target="_blank">View large</a><a section="6247722" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp017/2/dsp01703.jpeg?Expires=2147483647&Signature=3ieKmyAFteXC7S~93uw2wYKOENlpXFIv8rZx7MtZ95m8JWrfL4kXhx-qtnqERss3OImFr0NashnKO81bAkypSeb~TQCdVjI7-61fnTM3JzjklTFQZvDow~BvIBciO0G1pKK9J-rSZNndOcZW9A65yKZ~mLdnB4CWBXhRr~t~vuhDdO~GtdoJx~i8X2zv6-3ttNzE-XHgbPMbbwIDUd8uKjWK9IqaJNJsc55WVOvRT5VGdkqcuIn5JRAL4vrlImxDbx8pV~BpssN2BHYgB1srikimRiSvgI2QYWYuZ2yBZEUB2nWnAdUOCephsVYLMoB09QEzfGCT9jkkwJxck4CLfA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6247722&ar=421566&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp01703.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Association between DNA methylation level and gene expression. (A) Average methylation levels (site Me values) of genes with high and low expression. Genes with low expression (dashed line) showed significantly higher methylation levels than those with high expression (solid line) in positions close to TSSs. (B) Gene expression levels according to methylation statuses of the promoters (left panel) and gene bodies (right panel). Genes were classified into unmethylated (CGI Me value &lt;0.4, dotted line), moderately methylated (0.4 ≤ CGI Me value ≤ 0.6, dashed line), and highly methylated (CGI Me value &gt;0.6, solid line). Methylation of promoters was strongly associated with low expression (<em>P</em> = 8 × 10<sup>−203</sup> in AGS, <em>t</em>-test of expression of methylated genes and unmethylated genes), and methylation of CGIs in gene bodies was associated with higher expression (<em>P</em> = 6 × 10<sup>−4</sup> in AGS).</p></div></div><div content-id="DSP017F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp017/2/m_dsp01703.jpeg?Expires=2147483647&Signature=v4THd6gvRX9Mz-FSji3PqOzZfbBZIUvJnBhUAJTR0w2vhASjyP~fzGdw6h90yIwp8Uqf1vEAN5X-dOT3kddPXUZ52Lx02YXJhKPVxhRSpNmqgCRHvpM0nClzOjviNXODkYBZahGfkTHYvlVubiAkmU-f~fUKfgLTMLmYUMJEW0Sh2~Sy~LzoOspRZx3c7E9pPvtd6M7Ve3g4jmh5AYq5QJB~FTAIGzMZYHd18LM2EPfO-iP6AMSMbK~8SVUjQBPrIsyaRVTJDlnCMO0JV8e7ObS--ChVLn8MflWK79F~YGlryGWlvpd1M5vWBJ~MSz1ginzqX6D6cHZpKkwntNCn9w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Association between DNA methylation level and gene expression. (A) Average methylation levels (site Me values) of genes with high and low expression. Genes with low expression (dashed line) showed significantly higher methylation levels than those with high expression (solid line) in positions close to TSSs. (B) Gene expression levels according to methylation statuses of the promoters (left panel) and gene bodies (right panel). Genes were classified into unmethylated (CGI Me value &lt;0.4, dotted line), moderately methylated (0.4 ≤ CGI Me value ≤ 0.6, dashed line), and highly methylated (CGI Me value &gt;0.6, solid line). Methylation of promoters was strongly associated with low expression (P = 8 × 10−203 in AGS, t-test of expression of methylated genes and unmethylated genes), and methylation of CGIs in gene bodies was associated with higher expression (P = 6 × 10−4 in AGS)." path-from-xml="dsp01703.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6247722/dsp01703.jpeg" path-from-xml="dsp01703.jpeg" target="_blank">View large</a><a section="6247722" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp017/2/dsp01703.jpeg?Expires=2147483647&Signature=3ieKmyAFteXC7S~93uw2wYKOENlpXFIv8rZx7MtZ95m8JWrfL4kXhx-qtnqERss3OImFr0NashnKO81bAkypSeb~TQCdVjI7-61fnTM3JzjklTFQZvDow~BvIBciO0G1pKK9J-rSZNndOcZW9A65yKZ~mLdnB4CWBXhRr~t~vuhDdO~GtdoJx~i8X2zv6-3ttNzE-XHgbPMbbwIDUd8uKjWK9IqaJNJsc55WVOvRT5VGdkqcuIn5JRAL4vrlImxDbx8pV~BpssN2BHYgB1srikimRiSvgI2QYWYuZ2yBZEUB2nWnAdUOCephsVYLMoB09QEzfGCT9jkkwJxck4CLfA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6247722&ar=421566&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp01703.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Association between DNA methylation level and gene expression. (A) Average methylation levels (site Me values) of genes with high and low expression. Genes with low expression (dashed line) showed significantly higher methylation levels than those with high expression (solid line) in positions close to TSSs. (B) Gene expression levels according to methylation statuses of the promoters (left panel) and gene bodies (right panel). Genes were classified into unmethylated (CGI Me value &lt;0.4, dotted line), moderately methylated (0.4 ≤ CGI Me value ≤ 0.6, dashed line), and highly methylated (CGI Me value &gt;0.6, solid line). Methylation of promoters was strongly associated with low expression (<em>P</em> = 8 × 10<sup>−203</sup> in AGS, <em>t</em>-test of expression of methylated genes and unmethylated genes), and methylation of CGIs in gene bodies was associated with higher expression (<em>P</em> = 6 × 10<sup>−4</sup> in AGS).</p></div></div>                            <h3 scrollto-destination=6247723 id="6247723" class="section-title" data-legacy-id=s3e>3.5 Application of the Me value to analyse the effect of a demethylating agent</h3>
<p>Treatment of cells with 5-aza-dC induces demethylation of various CGIs, but the degree depending upon their positions against TSSs has not been clarified. The relationship between the degree of demethylation of promoters and that of expression induction has not been clarified, either. To address these two issues, we analysed methylation and gene expression levels in AGS before and after 5-aza-dC treatment.</p><p>Demethylation of the genes with methylated CGIs or promoters (Me value &gt; 0.6) before 5-aza-dC treatment was analysed. The average degree of demethylation was not influenced by the positions of CGIs against their TSS (Fig. <a reveal-id="DSP017F4" data-open="DSP017F4" class="link link-reveal link-table xref-fig">4</a>A), whereas the degree of demethylation of individual CGIs was highly variable (representative genes in Fig. <a reveal-id="DSP017F4" data-open="DSP017F4" class="link link-reveal link-table xref-fig">4</a>B). The average degree was not influenced by the distance between a CGI and repetitive elements (LINE, SINE), either (data not shown). There was no correlation (<em>r</em> = 0.12, Fig. <a reveal-id="DSP017F4" data-open="DSP017F4" class="link link-reveal link-table xref-fig">4</a>C) between the degree of demethylation of promoters (decrease of Me value in MeDIP-CGI microarray) and that of induction of gene expression (fold increase of signal log ratio in expression microarray analysis). The majority of genes with methylation of their promoters showed little or no increase of expression after 5-aza-dC treatment. The number of methylated promoters identified by MeDIP-microarray analysis was much larger than that of genes identified as silenced by expression microarray analysis after 5-aza-dC treatment (Table <a reveal-id="DSP017TB3" data-open="DSP017TB3" class="link link-reveal link-table xref-fig">3</a>).</p>                        <a id="6247726" scrollto-destination="6247726"></a>
<div id="DSP017F4" data-id="DSP017F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp017/2/m_dsp01704.jpeg?Expires=2147483647&Signature=3hEkwIt8-7w2BcGaS6nPd8Xevj0VuMUeRnV4B~p1W2QpCRzBCm6SE2LyHZmDH94MlWC8n1wQatdazxQ~qLhNvZG8fmIrjogxfSDMG5N4OJpNAevUSmx1aC7IFN8RkkvDMz~q0iNulwPEFvu-bTBA54UNoTPKV83vSHLxeeHjFJMpTBTipxIJcnlG7QzslqWnMCiym6dh~KaeDLOhKPwX6KlLYgMxals68cM0asLXZN1hcbxu42dWwf5TF5U5~w13jCvw~P8Z8HFc62SglMtg4wkxGZ9rcxJGLlsKkFefMIckMceHoP35EMf-ZsDdVwy8FzpW6Gpn2yL7dr9jsZHqzQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Degree of demethylation according to the positions against TSSs, and the lack of association between the degree of demethylation and that of re-expression. (A) Degree of demethylation according to the positions against TSSs. Methylation levels were analysed for CGIs methylated in AGS without 5-aza-dC treatment. 5-Aza-dC treatment induced demethylation of upstream CGIs, gene body CGIs, downstream CGIs, divergent CGIs and promoters to similar degrees. (B) Site Me value of TFAP2C and KISS1R in AGS and AGS treated with 5-aza-dC. In 1400–1800 bp against TSS, the degree of demethylation was high in TFAP2C and low in KISS1R, which representatively showed variation among individual genes. (C) Degree of demethylation of promoters and degree of re-expression in AGS. The degree of demethylation of promoters was not associated with that of re-expression of methylated genes after 5-aza-dC treatment." path-from-xml="dsp01704.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6247726/dsp01704.jpeg" path-from-xml="dsp01704.jpeg" target="_blank">View large</a><a section="6247726" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp017/2/dsp01704.jpeg?Expires=2147483647&Signature=brBMq1oyvVYsonfv61MqbnnSjni02lxxnoxX42O1JH1NcYX-x6Kujb07rI6RMGdN3RinBgWhVnqgElY5qZN-5Fsbsyyx3qkDYC~F81~T~hdco63KJlyM1Hz~GkQMWnnTY8zHWfaREtRhsd8zYO8nS8BA9npTxemWn9YP~Co8u6h8KS205mvnYIrjTRXorAsjMx6IROr3SAi7~12PjaU2-T1Z32lkkqhtc1M3I8mTXbPfpb4v9-vXLg0M9DfJ14VCWxSWYoTuU244zS9OWJ1EuqDVU3FIXnvEUAsplUZvetjV7VDb5AL5boxzG1-QlukW4dCKArRLc4eSHqvCExeO4g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6247726&ar=421566&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp01704.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Degree of demethylation according to the positions against TSSs, and the lack of association between the degree of demethylation and that of re-expression. (A) Degree of demethylation according to the positions against TSSs. Methylation levels were analysed for CGIs methylated in AGS without 5-aza-dC treatment. 5-Aza-dC treatment induced demethylation of upstream CGIs, gene body CGIs, downstream CGIs, divergent CGIs and promoters to similar degrees. (B) Site Me value of <em>TFAP2C</em> and <em>KISS1R</em> in AGS and AGS treated with 5-aza-dC. In 1400–1800 bp against TSS, the degree of demethylation was high in <em>TFAP2C</em> and low in <em>KISS1R</em>, which representatively showed variation among individual genes. (C) Degree of demethylation of promoters and degree of re-expression in AGS. The degree of demethylation of promoters was not associated with that of re-expression of methylated genes after 5-aza-dC treatment.</p></div></div><div content-id="DSP017F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp017/2/m_dsp01704.jpeg?Expires=2147483647&Signature=3hEkwIt8-7w2BcGaS6nPd8Xevj0VuMUeRnV4B~p1W2QpCRzBCm6SE2LyHZmDH94MlWC8n1wQatdazxQ~qLhNvZG8fmIrjogxfSDMG5N4OJpNAevUSmx1aC7IFN8RkkvDMz~q0iNulwPEFvu-bTBA54UNoTPKV83vSHLxeeHjFJMpTBTipxIJcnlG7QzslqWnMCiym6dh~KaeDLOhKPwX6KlLYgMxals68cM0asLXZN1hcbxu42dWwf5TF5U5~w13jCvw~P8Z8HFc62SglMtg4wkxGZ9rcxJGLlsKkFefMIckMceHoP35EMf-ZsDdVwy8FzpW6Gpn2yL7dr9jsZHqzQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Degree of demethylation according to the positions against TSSs, and the lack of association between the degree of demethylation and that of re-expression. (A) Degree of demethylation according to the positions against TSSs. Methylation levels were analysed for CGIs methylated in AGS without 5-aza-dC treatment. 5-Aza-dC treatment induced demethylation of upstream CGIs, gene body CGIs, downstream CGIs, divergent CGIs and promoters to similar degrees. (B) Site Me value of TFAP2C and KISS1R in AGS and AGS treated with 5-aza-dC. In 1400–1800 bp against TSS, the degree of demethylation was high in TFAP2C and low in KISS1R, which representatively showed variation among individual genes. (C) Degree of demethylation of promoters and degree of re-expression in AGS. The degree of demethylation of promoters was not associated with that of re-expression of methylated genes after 5-aza-dC treatment." path-from-xml="dsp01704.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6247726/dsp01704.jpeg" path-from-xml="dsp01704.jpeg" target="_blank">View large</a><a section="6247726" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp017/2/dsp01704.jpeg?Expires=2147483647&Signature=brBMq1oyvVYsonfv61MqbnnSjni02lxxnoxX42O1JH1NcYX-x6Kujb07rI6RMGdN3RinBgWhVnqgElY5qZN-5Fsbsyyx3qkDYC~F81~T~hdco63KJlyM1Hz~GkQMWnnTY8zHWfaREtRhsd8zYO8nS8BA9npTxemWn9YP~Co8u6h8KS205mvnYIrjTRXorAsjMx6IROr3SAi7~12PjaU2-T1Z32lkkqhtc1M3I8mTXbPfpb4v9-vXLg0M9DfJ14VCWxSWYoTuU244zS9OWJ1EuqDVU3FIXnvEUAsplUZvetjV7VDb5AL5boxzG1-QlukW4dCKArRLc4eSHqvCExeO4g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6247726&ar=421566&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp01704.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Degree of demethylation according to the positions against TSSs, and the lack of association between the degree of demethylation and that of re-expression. (A) Degree of demethylation according to the positions against TSSs. Methylation levels were analysed for CGIs methylated in AGS without 5-aza-dC treatment. 5-Aza-dC treatment induced demethylation of upstream CGIs, gene body CGIs, downstream CGIs, divergent CGIs and promoters to similar degrees. (B) Site Me value of <em>TFAP2C</em> and <em>KISS1R</em> in AGS and AGS treated with 5-aza-dC. In 1400–1800 bp against TSS, the degree of demethylation was high in <em>TFAP2C</em> and low in <em>KISS1R</em>, which representatively showed variation among individual genes. (C) Degree of demethylation of promoters and degree of re-expression in AGS. The degree of demethylation of promoters was not associated with that of re-expression of methylated genes after 5-aza-dC treatment.</p></div></div>                        <a id="6247727" scrollto-destination="6247727"></a>
<div content-id="DSP017TB3" class="table-modal"><div class="table-wrap"><div id="DSP017TB3" class="table-wrap-title" data-id="DSP017TB3"><div class="label -label">Table 3</div><div class="caption"><p>The number of methylated promoters by MeDIP-microarray analysis and silenced genes by expression microarray after 5-aza-dC treatment</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2"> </th><th colspan="4">Number of methylated promoter CGIs<hr /> </th></tr><tr><th>AGS </th><th>HSC39 </th><th>HSC57 </th><th>KATOIII </th></tr></thead><tbody><tr><td>MeDIP-CGI microarray<sup>a</sup> </td><td>1205 </td><td>792 </td><td>641 </td><td>1142 </td></tr><tr><td>Expression microarray after 5-aza-dC treatment<sup>b</sup> (Moriguchi et al.<sup><a reveal-id="DSP017C26" data-open="DSP017C26" class="link link-ref link-reveal xref-bibr">26</a></sup>) </td><td>25 </td><td>4 </td><td>1 </td><td>41 </td></tr></tbody></table></div><div content-id="DSP017TB3" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2"> </th><th colspan="4">Number of methylated promoter CGIs<hr /> </th></tr><tr><th>AGS </th><th>HSC39 </th><th>HSC57 </th><th>KATOIII </th></tr></thead><tbody><tr><td>MeDIP-CGI microarray<sup>a</sup> </td><td>1205 </td><td>792 </td><td>641 </td><td>1142 </td></tr><tr><td>Expression microarray after 5-aza-dC treatment<sup>b</sup> (Moriguchi et al.<sup><a reveal-id="DSP017C26" data-open="DSP017C26" class="link link-ref link-reveal xref-bibr">26</a></sup>) </td><td>25 </td><td>4 </td><td>1 </td><td>41 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Promoters identified as methylated by MeDIP-microarray analysis, among 9624 genes arrayed on CpG island microarray (Agilent).</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>The genes in which a 16-fold increase of expression after treatment of 5-aza-dC (optimized concentrations in each cell line) was observed by expression microarray, which confirmed its methylation in promoter by MSP, among 19 421 genes arrayed on GeneChip Human Genome 133 Plus 2.0 (Affymetrix).</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6247727" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSP017TB3" class="table-wrap-title" data-id="DSP017TB3"><div class="label -label">Table 3</div><div class="caption"><p>The number of methylated promoters by MeDIP-microarray analysis and silenced genes by expression microarray after 5-aza-dC treatment</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2"> </th><th colspan="4">Number of methylated promoter CGIs<hr /> </th></tr><tr><th>AGS </th><th>HSC39 </th><th>HSC57 </th><th>KATOIII </th></tr></thead><tbody><tr><td>MeDIP-CGI microarray<sup>a</sup> </td><td>1205 </td><td>792 </td><td>641 </td><td>1142 </td></tr><tr><td>Expression microarray after 5-aza-dC treatment<sup>b</sup> (Moriguchi et al.<sup><a reveal-id="DSP017C26" data-open="DSP017C26" class="link link-ref link-reveal xref-bibr">26</a></sup>) </td><td>25 </td><td>4 </td><td>1 </td><td>41 </td></tr></tbody></table></div><div content-id="DSP017TB3" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2"> </th><th colspan="4">Number of methylated promoter CGIs<hr /> </th></tr><tr><th>AGS </th><th>HSC39 </th><th>HSC57 </th><th>KATOIII </th></tr></thead><tbody><tr><td>MeDIP-CGI microarray<sup>a</sup> </td><td>1205 </td><td>792 </td><td>641 </td><td>1142 </td></tr><tr><td>Expression microarray after 5-aza-dC treatment<sup>b</sup> (Moriguchi et al.<sup><a reveal-id="DSP017C26" data-open="DSP017C26" class="link link-ref link-reveal xref-bibr">26</a></sup>) </td><td>25 </td><td>4 </td><td>1 </td><td>41 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Promoters identified as methylated by MeDIP-microarray analysis, among 9624 genes arrayed on CpG island microarray (Agilent).</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>The genes in which a 16-fold increase of expression after treatment of 5-aza-dC (optimized concentrations in each cell line) was observed by expression microarray, which confirmed its methylation in promoter by MSP, among 19 421 genes arrayed on GeneChip Human Genome 133 Plus 2.0 (Affymetrix).</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6247727" target="_blank">View Large</a></div></div><p>These results showed that expression cannot be induced for the majority of genes with methylation of their promoters even with a demethylating agent, possibly due to the lack of transcriptional factors or the presence of inactive histone modifications. Genes with low transcription tend to become methylated,<sup><a reveal-id="DSP017C20" data-open="DSP017C20" class="link link-ref link-reveal xref-bibr">20</a>,<a reveal-id="DSP017C38" data-open="DSP017C38" class="link link-ref link-reveal xref-bibr">38</a>,<a reveal-id="DSP017C39" data-open="DSP017C39" class="link link-ref link-reveal xref-bibr">39</a></sup> and such genes are unlikely to be expressed even if the methylation is removed. Caution is necessary when the relationship between methylation of the promoter and the expression of a gene is interpreted.</p>                            <h3 scrollto-destination=6247729 id="6247729" class="section-title" data-legacy-id=s3f>3.6 Application of the Me value in future studies</h3>
<p>The differential role of DNA methylation in promoters and gene bodies and the lack of association between the degree of demethylation of promoters and that of induction of gene expression were clearly shown due to the accuracy of the Me value. We here focused on CGIs using a CGI microarray. Roles of methylation of CpG-poor genomic regions in genome regulation are not as clear as those of CGIs,<sup><a reveal-id="DSP017C10" data-open="DSP017C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DSP017C40" data-open="DSP017C40" class="link link-ref link-reveal xref-bibr">40</a></sup> and focusing on CpG-rich regions is advantageous to isolate genes of interest. Also, the collection efficiency of methylated DNA by MeDIP or MBD is low in CpG-poor genomic regions,<sup><a reveal-id="DSP017C7" data-open="DSP017C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DSP017C9" data-open="DSP017C9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="DSP017C41" data-open="DSP017C41" class="link link-ref link-reveal xref-bibr">41</a></sup> and analysis of both CpG-poor and -rich regions leads to a complicated algorithm for normalization. The selective analysis of CGIs enabled us to develop a simple, but reliable output value.</p><p>The use of MeDIP allowed us to analyse methylation levels of any genomic regions regardless of the presence of restriction sites, and we were able to focus on promoters in some part of this study. We did not amplify the IP DNA during preparation of hybridization probes to avoid any amplification bias, and this was enabled by the use of a microarray platform with a high sensitivity and low signal/noise ratio. Under these optimal conditions, we achieved a high reproducibility (<em>r</em> = 0.98) and simplicity for the quantitative methylation microarray analysis.</p><p>In conclusion, we developed a new output value for microarray analysis suitable for quantitative assessment of DNA methylation levels. The Me value will be useful in genome-wide screening using heterogeneous samples.</p>                            <h2 scrollto-destination=6247734 id="6247734" class="section-title" data-legacy-id=s4>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available at www.dnaresearch.oxfordjournals.org.</a></p>                            <h2 scrollto-destination=6247736 id="6247736" class="section-title" data-legacy-id=s5>Funding</h2>
<p>This work was supported by a Grant-in-Aid for the Third-term Cancer Control Strategy Program from the Ministry of Health, Labour and Welfare, Japan. K.G. and H.T. are recipients of a Research Resident Fellowship from the Foundation for Promotion of Cancer Research.</p>                            <h2 scrollto-destination=6247738 id="6247738" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank Dr Toshiki Taya (Agilent Technologies) for his knowledgeable and devoted technical assistance.</p>                            <h2 scrollto-destination=6247740 id="6247740" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSP017C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP017C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jones</div> <div class="given-names">P.A.</div></div>,  <div class="name"><div class="surname">Baylin</div> <div class="given-names">S.B.</div></div>. </div><div class="article-title">The fundamental role of epigenetic events in cancer</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="year">2002</div>, vol. <div class="volume">3</div> (pg. <div class="fpage">415</div>-<div class="lpage">28</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20fundamental%20role%20of%20epigenetic%20events%20in%20cancer&amp;author=P.A.%20Jones&amp;author=S.B.%20Baylin&amp;publication_year=2002&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=3&amp;pages=415-28" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg962" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg962" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg962"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12042769" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP017C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP017C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ushijima</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Watanabe</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Okochi</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Kaneda</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Sugimura</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Miyamoto</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Fidelity of the methylation pattern and its variation in the genome</div>, <div class="source">Genome Res.</div> , <div class="year">2003</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">868</div>-<div class="lpage">74</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fidelity%20of%20the%20methylation%20pattern%20and%20its%20variation%20in%20the%20genome&amp;author=T.%20Ushijima&amp;author=N.%20Watanabe&amp;author=E.%20Okochi&amp;author=A.%20Kaneda&amp;author=T.%20Sugimura&amp;author=K.%20Miyamoto&amp;publication_year=2003&amp;journal=Genome%20Res.&amp;volume=13&amp;pages=868-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.969603" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.969603" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.969603"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12727906" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP017C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP017C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jones</div> <div class="given-names">P.A.</div></div>,  <div class="name"><div class="surname">Baylin</div> <div class="given-names">S.B.</div></div>. </div><div class="article-title">The epigenomics of cancer</div>, <div class="source">Cell</div> , <div class="year">2007</div>, vol. <div class="volume">128</div> (pg. <div class="fpage">683</div>-<div class="lpage">92</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20epigenomics%20of%20cancer&amp;author=P.A.%20Jones&amp;author=S.B.%20Baylin&amp;publication_year=2007&amp;journal=Cell&amp;volume=128&amp;pages=683-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2007.01.029" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2007.01.029" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2007.01.029"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17320506" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP017C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP017C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Robertson</div> <div class="given-names">K.D.</div></div>. </div><div class="article-title">DNA methylation and human disease</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="year">2005</div>, vol. <div class="volume">6</div> (pg. <div class="fpage">597</div>-<div class="lpage">610</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20methylation%20and%20human%20disease&amp;author=K.D.%20Robertson&amp;publication_year=2005&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=6&amp;pages=597-610" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg1655" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg1655" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg1655"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16136652" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP017C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP017C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Suzuki</div> <div class="given-names">M.M.</div></div>,  <div class="name"><div class="surname">Bird</div> <div class="given-names">A.</div></div>. </div><div class="article-title">DNA methylation landscapes: provocative insights from epigenomics</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="year">2008</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">465</div>-<div class="lpage">76</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20methylation%20landscapes%3A%20provocative%20insights%20from%20epigenomics&amp;author=M.M.%20Suzuki&amp;author=A.%20Bird&amp;publication_year=2008&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=9&amp;pages=465-76" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg2341" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg2341" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg2341"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18463664" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP017C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP017C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ushijima</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Detection and interpretation of altered methylation patterns in cancer cells</div>, <div class="source">Nat. Rev. Cancer</div> , <div class="year">2005</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">223</div>-<div class="lpage">31</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Detection%20and%20interpretation%20of%20altered%20methylation%20patterns%20in%20cancer%20cells&amp;author=T.%20Ushijima&amp;publication_year=2005&amp;journal=Nat.%20Rev.%20Cancer&amp;volume=5&amp;pages=223-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrc1571" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrc1571" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrc1571"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15719030" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP017C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP017C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Weber</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Davies</div> <div class="given-names">J.J.</div></div>,  <div class="name"><div class="surname">Wittig</div> <div class="given-names">D.</div></div>, et al. </div><div class="article-title">Chromosome-wide and promoter-specific analyses identify sites of differential DNA methylation in normal and transformed human cells</div>, <div class="source">Nat. Genet.</div> , <div class="year">2005</div>, vol. <div class="volume">37</div> (pg. <div class="fpage">853</div>-<div class="lpage">62</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Chromosome-wide%20and%20promoter-specific%20analyses%20identify%20sites%20of%20differential%20DNA%20methylation%20in%20normal%20and%20transformed%20human%20cells&amp;author=M.%20Weber&amp;author=J.J.%20Davies&amp;author=D.%20Wittig&amp;publication_year=2005&amp;journal=Nat.%20Genet.&amp;volume=37&amp;pages=853-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng1598" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng1598" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng1598"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16007088" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP017C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP017C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Yazaki</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Sundaresan</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Genome-wide high-resolution mapping and functional analysis of DNA methylation in arabidopsis</div>, <div class="source">Cell</div> , <div class="year">2006</div>, vol. <div class="volume">126</div> (pg. <div class="fpage">1189</div>-<div class="lpage">201</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20high-resolution%20mapping%20and%20functional%20analysis%20of%20DNA%20methylation%20in%20arabidopsis&amp;author=X.%20Zhang&amp;author=J.%20Yazaki&amp;author=A.%20Sundaresan&amp;publication_year=2006&amp;journal=Cell&amp;volume=126&amp;pages=1189-201" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2006.08.003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2006.08.003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2006.08.003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16949657" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP017C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP017C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Keshet</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Schlesinger</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Farkash</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Evidence for an instructive mechanism of de novo methylation in cancer cells</div>, <div class="source">Nat. Genet.</div> , <div class="year">2006</div>, vol. <div class="volume">38</div> (pg. <div class="fpage">149</div>-<div class="lpage">53</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evidence%20for%20an%20instructive%20mechanism%20of%20de%20novo%20methylation%20in%20cancer%20cells&amp;author=I.%20Keshet&amp;author=Y.%20Schlesinger&amp;author=S.%20Farkash&amp;publication_year=2006&amp;journal=Nat.%20Genet.&amp;volume=38&amp;pages=149-53" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng1719" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng1719" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng1719"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16444255" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP017C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP017C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Weber</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Hellmann</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Stadler</div> <div class="given-names">M.B.</div></div>, et al. </div><div class="article-title">Distribution, silencing potential and evolutionary impact of promoter DNA methylation in the human genome</div>, <div class="source">Nat. Genet.</div> , <div class="year">2007</div>, vol. <div class="volume">39</div> (pg. <div class="fpage">457</div>-<div class="lpage">66</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Distribution%2C%20silencing%20potential%20and%20evolutionary%20impact%20of%20promoter%20DNA%20methylation%20in%20the%20human%20genome&amp;author=M.%20Weber&amp;author=I.%20Hellmann&amp;author=M.B.%20Stadler&amp;publication_year=2007&amp;journal=Nat.%20Genet.&amp;volume=39&amp;pages=457-66" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng1990" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng1990" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng1990"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17334365" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP017C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP017C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zilberman</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Gehring</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Tran</div> <div class="given-names">R.K.</div></div>,  <div class="name"><div class="surname">Ballinger</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Henikoff</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Genome-wide analysis of <em>Arabidopsis thaliana</em> DNA methylation uncovers an interdependence between methylation and transcription</div>, <div class="source">Nat. Genet.</div> , <div class="year">2007</div>, vol. <div class="volume">39</div> (pg. <div class="fpage">61</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20analysis%20of%20Arabidopsis%20thaliana%20DNA%20methylation%20uncovers%20an%20interdependence%20between%20methylation%20and%20transcription&amp;author=D.%20Zilberman&amp;author=M.%20Gehring&amp;author=R.K.%20Tran&amp;author=T.%20Ballinger&amp;author=S.%20Henikoff&amp;publication_year=2007&amp;journal=Nat.%20Genet.&amp;volume=39&amp;pages=61-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng1929" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng1929" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng1929"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17128275" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP017C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP017C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hayashi</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Nagae</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Tsutsumi</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">High-resolution mapping of DNA methylation in human genome using oligonucleotide tiling array</div>, <div class="source">Hum. Genet.</div> , <div class="year">2007</div>, vol. <div class="volume">120</div> (pg. <div class="fpage">701</div>-<div class="lpage">11</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-resolution%20mapping%20of%20DNA%20methylation%20in%20human%20genome%20using%20oligonucleotide%20tiling%20array&amp;author=H.%20Hayashi&amp;author=G.%20Nagae&amp;author=S.%20Tsutsumi&amp;publication_year=2007&amp;journal=Hum.%20Genet.&amp;volume=120&amp;pages=701-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00439-006-0254-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00439-006-0254-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00439-006-0254-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17024368" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP017C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP017C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jacinto</div> <div class="given-names">F.V.</div></div>,  <div class="name"><div class="surname">Ballestar</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Ropero</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Esteller</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Discovery of epigenetically silenced genes by methylated DNA immunoprecipitation in colon cancer cells</div>, <div class="source">Cancer Res.</div> , <div class="year">2007</div>, vol. <div class="volume">67</div> (pg. <div class="fpage">11481</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Discovery%20of%20epigenetically%20silenced%20genes%20by%20methylated%20DNA%20immunoprecipitation%20in%20colon%20cancer%20cells&amp;author=F.V.%20Jacinto&amp;author=E.%20Ballestar&amp;author=S.%20Ropero&amp;author=M.%20Esteller&amp;publication_year=2007&amp;journal=Cancer%20Res.&amp;volume=67&amp;pages=11481-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1158/0008-5472.CAN-07-2687" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1158%2F0008-5472.CAN-07-2687" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1158%2F0008-5472.CAN-07-2687"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18089774" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP017C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP017C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Down</div> <div class="given-names">T.A.</div></div>,  <div class="name"><div class="surname">Rakyan</div> <div class="given-names">V.K.</div></div>,  <div class="name"><div class="surname">Turner</div> <div class="given-names">D.J.</div></div>, et al. </div><div class="article-title">A Bayesian deconvolution strategy for immunoprecipitation-based DNA methylome analysis</div>, <div class="source">Nat. Biotechnol.</div> , <div class="year">2008</div>, vol. <div class="volume">26</div> (pg. <div class="fpage">779</div>-<div class="lpage">85</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20Bayesian%20deconvolution%20strategy%20for%20immunoprecipitation-based%20DNA%20methylome%20analysis&amp;author=T.A.%20Down&amp;author=V.K.%20Rakyan&amp;author=D.J.%20Turner&amp;publication_year=2008&amp;journal=Nat.%20Biotechnol.&amp;volume=26&amp;pages=779-85" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt1414" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt1414" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt1414"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18612301" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP017C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP017C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cheng</div> <div class="given-names">A.S.</div></div>,  <div class="name"><div class="surname">Culhane</div> <div class="given-names">A.C.</div></div>,  <div class="name"><div class="surname">Chan</div> <div class="given-names">M.W.</div></div>, et al. </div><div class="article-title">Epithelial progeny of estrogen-exposed breast progenitor cells display a cancer-like methylome</div>, <div class="source">Cancer Res.</div> , <div class="year">2008</div>, vol. <div class="volume">68</div> (pg. <div class="fpage">1786</div>-<div class="lpage">96</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Epithelial%20progeny%20of%20estrogen-exposed%20breast%20progenitor%20cells%20display%20a%20cancer-like%20methylome&amp;author=A.S.%20Cheng&amp;author=A.C.%20Culhane&amp;author=M.W.%20Chan&amp;publication_year=2008&amp;journal=Cancer%20Res.&amp;volume=68&amp;pages=1786-96" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1158/0008-5472.CAN-07-5547" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1158%2F0008-5472.CAN-07-5547" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1158%2F0008-5472.CAN-07-5547"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18339859" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP017C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP017C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pelizzola</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Koga</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Urban</div> <div class="given-names">A.E.</div></div>, et al. </div><div class="article-title">MEDME: an experimental and analytical methodology for the estimation of DNA methylation levels based on microarray derived MeDIP-enrichment</div>, <div class="source">Genome Res.</div> , <div class="year">2008</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">1652</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MEDME%3A%20an%20experimental%20and%20analytical%20methodology%20for%20the%20estimation%20of%20DNA%20methylation%20levels%20based%20on%20microarray%20derived%20MeDIP-enrichment&amp;author=M.%20Pelizzola&amp;author=Y.%20Koga&amp;author=A.E.%20Urban&amp;publication_year=2008&amp;journal=Genome%20Res.&amp;volume=18&amp;pages=1652-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.080721.108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.080721.108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.080721.108"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18765822" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP017C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP017C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Straussman</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Nejman</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Roberts</div> <div class="given-names">D.</div></div>, et al. </div><div class="article-title">Developmental programming of CpG island methylation profiles in the human genome</div>, <div class="source">Nat. Struct. Mol. Biol.</div> , <div class="year">2009</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">564</div>-<div class="lpage">71</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Developmental%20programming%20of%20CpG%20island%20methylation%20profiles%20in%20the%20human%20genome&amp;author=R.%20Straussman&amp;author=D.%20Nejman&amp;author=D.%20Roberts&amp;publication_year=2009&amp;journal=Nat.%20Struct.%20Mol.%20Biol.&amp;volume=16&amp;pages=564-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nsmb.1594" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnsmb.1594" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnsmb.1594"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19377480" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP017C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP017C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Buck</div> <div class="given-names">M.J.</div></div>,  <div class="name"><div class="surname">Lieb</div> <div class="given-names">J.D.</div></div>. </div><div class="article-title">ChIP-chip: considerations for the design, analysis, and application of genome-wide chromatin immunoprecipitation experiments</div>, <div class="source">Genomics</div> , <div class="year">2004</div>, vol. <div class="volume">83</div> (pg. <div class="fpage">349</div>-<div class="lpage">60</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=ChIP-chip%3A%20considerations%20for%20the%20design%2C%20analysis%2C%20and%20application%20of%20genome-wide%20chromatin%20immunoprecipitation%20experiments&amp;author=M.J.%20Buck&amp;author=J.D.%20Lieb&amp;publication_year=2004&amp;journal=Genomics&amp;volume=83&amp;pages=349-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ygeno.2003.11.004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ygeno.2003.11.004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ygeno.2003.11.004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14986705" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP017C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP017C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yamashita</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Tsujino</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Moriguchi</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Tatematsu</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Ushijima</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Chemical genomic screening for methylation-silenced genes in gastric cancer cell lines using 5-aza-2′-deoxycytidine treatment and oligonucleotide microarray</div>, <div class="source">Cancer Sci.</div> , <div class="year">2006</div>, vol. <div class="volume">97</div> (pg. <div class="fpage">64</div>-<div class="lpage">71</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Chemical%20genomic%20screening%20for%20methylation-silenced%20genes%20in%20gastric%20cancer%20cell%20lines%20using%205-aza-2%E2%80%B2-deoxycytidine%20treatment%20and%20oligonucleotide%20microarray&amp;author=S.%20Yamashita&amp;author=Y.%20Tsujino&amp;author=K.%20Moriguchi&amp;author=M.%20Tatematsu&amp;author=T.%20Ushijima&amp;publication_year=2006&amp;journal=Cancer%20Sci.&amp;volume=97&amp;pages=64-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/cas.2006.97.issue-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fcas.2006.97.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fcas.2006.97.issue-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16367923" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP017C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP017C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nakajima</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Yamashita</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Maekita</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Niwa</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Nakazawa</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Ushijima</div> <div class="given-names">T.</div></div>. </div><div class="article-title">The presence of a methylation fingerprint of Helicobacter pylori infection in human gastric mucosae</div>, <div class="source">Int. J. Cancer</div> , <div class="year">2009</div>, vol. <div class="volume">124</div> (pg. <div class="fpage">905</div>-<div class="lpage">10</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20presence%20of%20a%20methylation%20fingerprint%20of%20Helicobacter%20pylori%20infection%20in%20human%20gastric%20mucosae&amp;author=T.%20Nakajima&amp;author=S.%20Yamashita&amp;author=T.%20Maekita&amp;author=T.%20Niwa&amp;author=K.%20Nakazawa&amp;author=T.%20Ushijima&amp;publication_year=2009&amp;journal=Int.%20J.%20Cancer&amp;volume=124&amp;pages=905-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/ijc.v124:4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fijc.v124:4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Fijc.v124:4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19035455" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP017C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP017C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lee</div> <div class="given-names">T.I.</div></div>,  <div class="name"><div class="surname">Rinaldi</div> <div class="given-names">N.J.</div></div>,  <div class="name"><div class="surname">Robert</div> <div class="given-names">F.</div></div>, et al. </div><div class="article-title">Transcriptional regulatory networks in <em>Saccharomyces cerevisiae</em></div>, <div class="source">Science</div> , <div class="year">2002</div>, vol. <div class="volume">298</div> (pg. <div class="fpage">799</div>-<div class="lpage">804</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptional%20regulatory%20networks%20in%20Saccharomyces%20cerevisiae&amp;author=T.I.%20Lee&amp;author=N.J.%20Rinaldi&amp;author=F.%20Robert&amp;publication_year=2002&amp;journal=Science&amp;volume=298&amp;pages=799-804" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1075090" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1075090" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1075090"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12399584" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP017C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP017C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pokholok</div> <div class="given-names">D.K.</div></div>,  <div class="name"><div class="surname">Harbison</div> <div class="given-names">C.T.</div></div>,  <div class="name"><div class="surname">Levine</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Genome-wide map of nucleosome acetylation and methylation in yeast</div>, <div class="source">Cell</div> , <div class="year">2005</div>, vol. <div class="volume">122</div> (pg. <div class="fpage">517</div>-<div class="lpage">27</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20map%20of%20nucleosome%20acetylation%20and%20methylation%20in%20yeast&amp;author=D.K.%20Pokholok&amp;author=C.T.%20Harbison&amp;author=S.%20Levine&amp;publication_year=2005&amp;journal=Cell&amp;volume=122&amp;pages=517-27" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2005.06.026" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2005.06.026" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2005.06.026"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16122420" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP017C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP017C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hahn</div> <div class="given-names">M.A.</div></div>,  <div class="name"><div class="surname">Hahn</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Lee</div> <div class="given-names">D.H.</div></div>, et al. </div><div class="article-title">Methylation of polycomb target genes in intestinal cancer is mediated by inflammation</div>, <div class="source">Cancer Res.</div> , <div class="year">2008</div>, vol. <div class="volume">68</div> (pg. <div class="fpage">10280</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Methylation%20of%20polycomb%20target%20genes%20in%20intestinal%20cancer%20is%20mediated%20by%20inflammation&amp;author=M.A.%20Hahn&amp;author=T.%20Hahn&amp;author=D.H.%20Lee&amp;publication_year=2008&amp;journal=Cancer%20Res.&amp;volume=68&amp;pages=10280-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1158/0008-5472.CAN-08-1957" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1158%2F0008-5472.CAN-08-1957" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1158%2F0008-5472.CAN-08-1957"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19074896" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP017C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP017C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dong</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Yauk</div> <div class="given-names">C.L.</div></div>,  <div class="name"><div class="surname">Rowan-Carroll</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Identification of thyroid hormone receptor binding sites and target genes using ChIP-on-chip in developing mouse cerebellum</div>, <div class="source">PLoS One</div> , <div class="year">2009</div>, vol. <div class="volume">4</div> pg. <div class="fpage">e4610</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20thyroid%20hormone%20receptor%20binding%20sites%20and%20target%20genes%20using%20ChIP-on-chip%20in%20developing%20mouse%20cerebellum&amp;author=H.%20Dong&amp;author=C.L.%20Yauk&amp;author=A.%20Rowan-Carroll&amp;publication_year=2009&amp;journal=PLoS%20One&amp;volume=4&amp;pages=e4610" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0004610" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0004610" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0004610"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19240802" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP017C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP017C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ke</div> <div class="given-names">X.S.</div></div>,  <div class="name"><div class="surname">Qu</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Rostad</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">Genome-wide profiling of histone h3 lysine 4 and lysine 27 trimethylation reveals an epigenetic signature in prostate carcinogenesis</div>, <div class="source">PLoS One</div> , <div class="year">2009</div>, vol. <div class="volume">4</div> pg. <div class="fpage">e4687</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20profiling%20of%20histone%20h3%20lysine%204%20and%20lysine%2027%20trimethylation%20reveals%20an%20epigenetic%20signature%20in%20prostate%20carcinogenesis&amp;author=X.S.%20Ke&amp;author=Y.%20Qu&amp;author=K.%20Rostad&amp;publication_year=2009&amp;journal=PLoS%20One&amp;volume=4&amp;pages=e4687" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0004687" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0004687" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0004687"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19262738" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP017C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP017C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Moriguchi</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Yamashita</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Tsujino</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Tatematsu</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Ushijima</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Larger numbers of silenced genes in cancer cell lines with increased de novo methylation of scattered CpG sites</div>, <div class="source">Cancer Lett.</div> , <div class="year">2007</div>, vol. <div class="volume">249</div> (pg. <div class="fpage">178</div>-<div class="lpage">87</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Larger%20numbers%20of%20silenced%20genes%20in%20cancer%20cell%20lines%20with%20increased%20de%20novo%20methylation%20of%20scattered%20CpG%20sites&amp;author=K.%20Moriguchi&amp;author=S.%20Yamashita&amp;author=Y.%20Tsujino&amp;author=M.%20Tatematsu&amp;author=T.%20Ushijima&amp;publication_year=2007&amp;journal=Cancer%20Lett.&amp;volume=249&amp;pages=178-87" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.canlet.2006.08.014" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.canlet.2006.08.014" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.canlet.2006.08.014"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16971039" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP017C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP017C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kaneda</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Kaminishi</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Yanagihara</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Sugimura</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Ushijima</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Identification of silencing of nine genes in human gastric cancers</div>, <div class="source">Cancer Res.</div> , <div class="year">2002</div>, vol. <div class="volume">62</div> (pg. <div class="fpage">6645</div>-<div class="lpage">50</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20silencing%20of%20nine%20genes%20in%20human%20gastric%20cancers&amp;author=A.%20Kaneda&amp;author=M.%20Kaminishi&amp;author=K.%20Yanagihara&amp;author=T.%20Sugimura&amp;author=T.%20Ushijima&amp;publication_year=2002&amp;journal=Cancer%20Res.&amp;volume=62&amp;pages=6645-50" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12438262" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP017C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP017C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Miyamoto</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Asada</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Fukutomi</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Methylation-associated silencing of heparan sulfate <span class="small-caps">d</span>-glucosaminyl 3-<em>O</em>-sulfotransferase-2 (3-OST-2) in human breast, colon, lung and pancreatic cancers</div>, <div class="source">Oncogene.</div> , <div class="year">2003</div>, vol. <div class="volume">22</div> (pg. <div class="fpage">274</div>-<div class="lpage">80</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Methylation-associated%20silencing%20of%20heparan%20sulfate%20d-glucosaminyl%203-O-sulfotransferase-2%20(3-OST-2)%20in%20human%20breast%2C%20colon%2C%20lung%20and%20pancreatic%20cancers&amp;author=K.%20Miyamoto&amp;author=K.%20Asada&amp;author=T.%20Fukutomi&amp;publication_year=2003&amp;journal=Oncogene.&amp;volume=22&amp;pages=274-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/sj.onc.1206146" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsj.onc.1206146" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsj.onc.1206146"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12527896" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP017C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP017C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Abe</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Watanabe</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">McDonell</div> <div class="given-names">N.</div></div>, et al. </div><div class="article-title">Identification of genes targeted by CpG island methylator phenotype in neuroblastomas, and their possible integrative involvement in poor prognosis</div>, <div class="source">Oncology</div> , <div class="year">2008</div>, vol. <div class="volume">74</div> (pg. <div class="fpage">50</div>-<div class="lpage">60</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20genes%20targeted%20by%20CpG%20island%20methylator%20phenotype%20in%20neuroblastomas%2C%20and%20their%20possible%20integrative%20involvement%20in%20poor%20prognosis&amp;author=M.%20Abe&amp;author=N.%20Watanabe&amp;author=N.%20McDonell&amp;publication_year=2008&amp;journal=Oncology&amp;volume=74&amp;pages=50-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1159/000139124" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1159%2F000139124" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1159%2F000139124"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18544995" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP017C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP017C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yamashita</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Takahashi</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">McDonell</div> <div class="given-names">N.</div></div>, et al. </div><div class="article-title">Methylation silencing of transforming growth factor-beta receptor type II in rat prostate cancers</div>, <div class="source">Cancer Res.</div> , <div class="year">2008</div>, vol. <div class="volume">68</div> (pg. <div class="fpage">2112</div>-<div class="lpage">21</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Methylation%20silencing%20of%20transforming%20growth%20factor-beta%20receptor%20type%20II%20in%20rat%20prostate%20cancers&amp;author=S.%20Yamashita&amp;author=S.%20Takahashi&amp;author=N.%20McDonell&amp;publication_year=2008&amp;journal=Cancer%20Res.&amp;volume=68&amp;pages=2112-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1158/0008-5472.CAN-07-5282" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1158%2F0008-5472.CAN-07-5282" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1158%2F0008-5472.CAN-07-5282"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18381416" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP017C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP017C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Eckhardt</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Lewin</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Cortese</div> <div class="given-names">R.</div></div>, et al. </div><div class="article-title">DNA methylation profiling of human chromosomes 6, 20 and 22</div>, <div class="source">Nat. Genet.</div> , <div class="year">2006</div>, vol. <div class="volume">38</div> (pg. <div class="fpage">1378</div>-<div class="lpage">85</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20methylation%20profiling%20of%20human%20chromosomes%206%2C%2020%20and%2022&amp;author=F.%20Eckhardt&amp;author=J.%20Lewin&amp;author=R.%20Cortese&amp;publication_year=2006&amp;journal=Nat.%20Genet.&amp;volume=38&amp;pages=1378-85" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng1909" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng1909" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng1909"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17072317" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP017C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP017C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fukasawa</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Kimura</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Morita</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Microarray analysis of promoter methylation in lung cancers</div>, <div class="source">J. Hum. Genet.</div> , <div class="year">2006</div>, vol. <div class="volume">51</div> (pg. <div class="fpage">368</div>-<div class="lpage">74</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microarray%20analysis%20of%20promoter%20methylation%20in%20lung%20cancers&amp;author=M.%20Fukasawa&amp;author=M.%20Kimura&amp;author=S.%20Morita&amp;publication_year=2006&amp;journal=J.%20Hum.%20Genet.&amp;volume=51&amp;pages=368-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10038-005-0355-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10038-005-0355-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10038-005-0355-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16435073" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP017C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP017C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tomilin</div> <div class="given-names">N.V.</div></div>. </div><div class="article-title">Regulation of mammalian gene expression by retroelements and non-coding tandem repeats</div>, <div class="source">Bioessays</div> , <div class="year">2008</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">338</div>-<div class="lpage">48</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Regulation%20of%20mammalian%20gene%20expression%20by%20retroelements%20and%20non-coding%20tandem%20repeats&amp;author=N.V.%20Tomilin&amp;publication_year=2008&amp;journal=Bioessays&amp;volume=30&amp;pages=338-48" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1521-1878" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1521-1878" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1521-1878"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18348251" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP017C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP017C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hellman</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Chess</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Gene body-specific methylation on the active X chromosome</div>, <div class="source">Science</div> , <div class="year">2007</div>, vol. <div class="volume">315</div> (pg. <div class="fpage">1141</div>-<div class="lpage">3</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20body-specific%20methylation%20on%20the%20active%20X%20chromosome&amp;author=A.%20Hellman&amp;author=A.%20Chess&amp;publication_year=2007&amp;journal=Science&amp;volume=315&amp;pages=1141-3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1136352" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1136352" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1136352"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17322062" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP017C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP017C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Smith</div> <div class="given-names">J.F.</div></div>,  <div class="name"><div class="surname">Mahmood</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Song</div> <div class="given-names">F.</div></div>, et al. </div><div class="article-title">Identification of DNA methylation in 3′ genomic regions that are associated with upregulation of gene expression in colorectal cancer</div>, <div class="source">Epigenetics</div> , <div class="year">2007</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">161</div>-<div class="lpage">72</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20DNA%20methylation%20in%203%E2%80%B2%20genomic%20regions%20that%20are%20associated%20with%20upregulation%20of%20gene%20expression%20in%20colorectal%20cancer&amp;author=J.F.%20Smith&amp;author=S.%20Mahmood&amp;author=F.%20Song&amp;publication_year=2007&amp;journal=Epigenetics&amp;volume=2&amp;pages=161-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.4161/epi.2.3.4805" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.4161%2Fepi.2.3.4805" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.4161%2Fepi.2.3.4805"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17965620" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP017C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP017C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ball</div> <div class="given-names">M.P.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">J.B.</div></div>,  <div class="name"><div class="surname">Gao</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">Targeted and genome-scale strategies reveal gene-body methylation signatures in human cells</div>, <div class="source">Nat. Biotechnol.</div> , <div class="year">2009</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">361</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Targeted%20and%20genome-scale%20strategies%20reveal%20gene-body%20methylation%20signatures%20in%20human%20cells&amp;author=M.P.%20Ball&amp;author=J.B.%20Li&amp;author=Y.%20Gao&amp;publication_year=2009&amp;journal=Nat.%20Biotechnol.&amp;volume=27&amp;pages=361-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.1533" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.1533" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.1533"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19329998" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP017C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP017C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rauch</div> <div class="given-names">T.A.</div></div>,  <div class="name"><div class="surname">Wu</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Zhong</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Riggs</div> <div class="given-names">A.D.</div></div>,  <div class="name"><div class="surname">Pfeifer</div> <div class="given-names">G.P.</div></div>. </div><div class="article-title">A human B cell methylome at 100-base pair resolution</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2009</div>, vol. <div class="volume">106</div> (pg. <div class="fpage">671</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20human%20B%20cell%20methylome%20at%20100-base%20pair%20resolution&amp;author=T.A.%20Rauch&amp;author=X.%20Wu&amp;author=X.%20Zhong&amp;author=A.D.%20Riggs&amp;author=G.P.%20Pfeifer&amp;publication_year=2009&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=106&amp;pages=671-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0812399106" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0812399106" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0812399106"> </span></div> </div></div></div></div></div><div content-id="DSP017C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP017C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Song</div> <div class="given-names">J.Z.</div></div>,  <div class="name"><div class="surname">Stirzaker</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Harrison</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Melki</div> <div class="given-names">J.R.</div></div>,  <div class="name"><div class="surname">Clark</div> <div class="given-names">S.J.</div></div>. </div><div class="article-title">Hypermethylation trigger of the glutathione-S-transferase gene (GSTP1) in prostate cancer cells</div>, <div class="source">Oncogene</div> , <div class="year">2002</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">1048</div>-<div class="lpage">61</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Hypermethylation%20trigger%20of%20the%20glutathione-S-transferase%20gene%20(GSTP1)%20in%20prostate%20cancer%20cells&amp;author=J.Z.%20Song&amp;author=C.%20Stirzaker&amp;author=J.%20Harrison&amp;author=J.R.%20Melki&amp;author=S.J.%20Clark&amp;publication_year=2002&amp;journal=Oncogene&amp;volume=21&amp;pages=1048-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/sj.onc.1205153" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsj.onc.1205153" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsj.onc.1205153"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11850822" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP017C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP017C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">de Smet</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Loriot</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Boon</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Promoter-dependent mechanism leading to selective hypomethylation within the 5′ region of gene MAGE-A1 in tumor cells</div>, <div class="source">Mol. Cell Biol.</div> , <div class="year">2004</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">4781</div>-<div class="lpage">90</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Promoter-dependent%20mechanism%20leading%20to%20selective%20hypomethylation%20within%20the%205%E2%80%B2%20region%20of%20gene%20MAGE-A1%20in%20tumor%20cells&amp;author=C.%20de%20Smet&amp;author=A.%20Loriot&amp;author=T.%20Boon&amp;publication_year=2004&amp;journal=Mol.%20Cell%20Biol.&amp;volume=24&amp;pages=4781-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/MCB.24.11.4781-4790.2004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FMCB.24.11.4781-4790.2004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FMCB.24.11.4781-4790.2004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15143172" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP017C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP017C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Irizarry</div> <div class="given-names">R.A.</div></div>,  <div class="name"><div class="surname">Ladd-Acosta</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Wen</div> <div class="given-names">B.</div></div>, et al. </div><div class="article-title">The human colon cancer methylome shows similar hypo- and hypermethylation at conserved tissue-specific CpG island shores</div>, <div class="source">Nat. Genet.</div> , <div class="year">2009</div>, vol. <div class="volume">41</div> (pg. <div class="fpage">178</div>-<div class="lpage">86</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20human%20colon%20cancer%20methylome%20shows%20similar%20hypo-%20and%20hypermethylation%20at%20conserved%20tissue-specific%20CpG%20island%20shores&amp;author=R.A.%20Irizarry&amp;author=C.%20Ladd-Acosta&amp;author=B.%20Wen&amp;publication_year=2009&amp;journal=Nat.%20Genet.&amp;volume=41&amp;pages=178-86" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.298" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.298" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.298"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19151715" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP017C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP017C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rauch</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Wu</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Pfeifer</div> <div class="given-names">G.P.</div></div>. </div><div class="article-title">MIRA-assisted microarray analysis, a new technology for the determination of DNA methylation patterns, identifies frequent methylation of homeodomain-containing genes in lung cancer cells</div>, <div class="source">Cancer Res.</div> , <div class="year">2006</div>, vol. <div class="volume">66</div> (pg. <div class="fpage">7939</div>-<div class="lpage">47</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MIRA-assisted%20microarray%20analysis%2C%20a%20new%20technology%20for%20the%20determination%20of%20DNA%20methylation%20patterns%2C%20identifies%20frequent%20methylation%20of%20homeodomain-containing%20genes%20in%20lung%20cancer%20cells&amp;author=T.%20Rauch&amp;author=H.%20Li&amp;author=X.%20Wu&amp;author=G.P.%20Pfeifer&amp;publication_year=2006&amp;journal=Cancer%20Res.&amp;volume=66&amp;pages=7939-47" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1158/0008-5472.CAN-06-1888" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1158%2F0008-5472.CAN-06-1888" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1158%2F0008-5472.CAN-06-1888"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16912168" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div><div content-id=""><div class="fn"><div class="footnote-content">Edited by Minoru Yoshida</div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2009. Published by Oxford University Press on behalf of Kazusa DNA Research Institute</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/2.5/uk/) which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=421566&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp017/2/dsp017-dsp017supp.pdf?Expires=2147483647&amp;Signature=TpeXuiSS3rQVxYk0h4b3CYa6rybbqqW~HivtmUyxENIXsdhWxI50zH5EnMoF-Hq~HoJivTrJrIeebv2uA7FmEc4EBWZbHWyUPBPSF0hHbJGev7nHrvWIUjlN9lngLB3w7GLiFyhkt9nw3dzxePoXw4rv-Gb1j71YsMD-bnxilVxKMKxndEq1f0moY7cIMBoiSbzXZAb357MT1ek-baZa3EK8UU0H4idW0lPqyhBaiaiPMYW4b9nH9zf7X31IQleh~W2BifVZeXOA7vpDo0BzF6DAg6NrSKZHSqEnZ7ib1FGXfVS79iXR4vwmvtWjgTmSiU0MbahLSRB4t5YCxwzymA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">129</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000271610400003&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">31</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsp017"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="421566"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000271610400003&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f16%2f5%2f275%2f421566">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000271610400003&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (31)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F16%2F5%2F275%2F421566" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/421566'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Development of a Novel Output Value for Quantitative Assessment in Methylated DNA Immunoprecipitation-CpG Island Microarray Analysis" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsp017");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f16%2f5%2f275%2f421566">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

